
<html lang="en"     class="pb-page"  data-request-id="660eabd5-e888-4ab1-9d0c-fb2f39a7d854"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2013.56.issue-18;wgroup:string:ACHS website Group;article:article:10.1021/jm400919p;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Small-Molecule Ligands of Methyl-Lysine Binding Proteins: Optimization of Selectivity for L3MBTL3" /></meta><meta name="dc.Creator" content="Lindsey I.  James" /></meta><meta name="dc.Creator" content="Victoria K.  Korboukh" /></meta><meta name="dc.Creator" content="Liubov  Krichevsky" /></meta><meta name="dc.Creator" content="Brandi M.  Baughman" /></meta><meta name="dc.Creator" content="J. Martin  Herold" /></meta><meta name="dc.Creator" content="Jacqueline L.  Norris" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Creator" content="Dmitri B.  Kireev" /></meta><meta name="dc.Creator" content="William P.  Janzen" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Stephen V.  Frye" /></meta><meta name="dc.Description" content="Lysine methylation is a key epigenetic mark, the dysregulation of which is linked to many diseases. Small-molecule antagonism of methyl-lysine (Kme) binding proteins that recognize such epigenetic ..." /></meta><meta name="Description" content="Lysine methylation is a key epigenetic mark, the dysregulation of which is linked to many diseases. Small-molecule antagonism of methyl-lysine (Kme) binding proteins that recognize such epigenetic ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 16, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm400919p" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm400919p" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm400919p" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm400919p" /></link>
        
    
    

<title>Small-Molecule Ligands of Methyl-Lysine Binding Proteins: Optimization of Selectivity for L3MBTL3 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm400919p" /></meta><meta property="og:title" content="Small-Molecule Ligands of Methyl-Lysine Binding Proteins: Optimization of Selectivity for L3MBTL3" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0019.jpeg" /></meta><meta property="og:description" content="Lysine methylation is a key epigenetic mark, the dysregulation of which is linked to many diseases. Small-molecule antagonism of methyl-lysine (Kme) binding proteins that recognize such epigenetic marks can improve our understanding of these regulatory mechanisms and potentially validate Kme binding proteins as drug-discovery targets. We previously reported the discovery of 1 (UNC1215), the first potent and selective small-molecule chemical probe of a methyl-lysine reader protein, L3MBTL3, which antagonizes the mono- and dimethyl-lysine reading function of L3MBTL3. The design, synthesis, and structure–activity relationship studies that led to the discovery of 1 are described herein. These efforts established the requirements for potent L3MBTL3 binding and enabled the design of novel antagonists, such as compound 2 (UNC1679), that maintain in vitro and cellular potency with improved selectivity against other MBT-containing proteins. The antagonists described were also found to effectively interact with unlabeled endogenous L3MBTL3 in cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm400919p"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm400919p">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm400919p&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm400919p&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm400919p&amp;href=/doi/10.1021/jm400919p" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 7358-7371</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm4008906" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm400939k" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Small-Molecule Ligands of Methyl-Lysine Binding Proteins: Optimization of Selectivity for L3MBTL3</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lindsey+I.++James">Lindsey I. James</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Victoria+K.++Korboukh">Victoria K. Korboukh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Liubov++Krichevsky">Liubov Krichevsky</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brandi+M.++Baughman">Brandi M. Baughman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Martin++Herold">J. Martin Herold</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline+L.++Norris">Jacqueline L. Norris</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dmitri+B.++Kireev">Dmitri B. Kireev</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=William+P.++Janzen">William P. Janzen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+V.++Frye">Stephen V. Frye</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Center for Integrative Chemical Biology and Drug Discovery, Division of Chemical Biology and Medicinal Chemistry, UNC Eshelman School of Pharmacy, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599, United States</span></div><div class="aff-info" id="aff2"><span class="aff-text"><sup>‡</sup>Structural Genomics Consortium, University of Toronto, <sup>§</sup>Department of Medical Biophysics, University of Toronto, and <sup>∥</sup>Princess Margaret Cancer Centre, 101 College Street, Toronto, Ontario M5G 1L7, Canada</span></div><div class="corresp-info"><strong>*</strong>Phone: 919-843-5486. Fax: 919-843-8465. E-mail: <a href="/cdn-cgi/l/email-protection#02717464707b6742676f636b6e2c776c612c676677"><span class="__cf_email__" data-cfemail="f3808595818a96b3969e929a9fdd869d90dd969786">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm400919p&amp;href=/doi/10.1021%2Fjm400919p" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 7358–7371</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 16, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 June 2013</li><li><span class="item_label"><b>Published</b> online</span>16 September 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 September 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm400919p" title="DOI URL">https://doi.org/10.1021/jm400919p</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7358%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLindsey%2BI.%2BJames%252C%2BVictoria%2BK.%2BKorboukh%252C%2BLiubov%2BKrichevsky%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D18%26contentID%3Djm400919p%26title%3DSmall-Molecule%2BLigands%2Bof%2BMethyl-Lysine%2BBinding%2BProteins%253A%2BOptimization%2Bof%2BSelectivity%2Bfor%2BL3MBTL3%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7371%26publicationDate%3DSeptember%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm400919p"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2263</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">55</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm400919p" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Small-Molecule Ligands of Methyl-Lysine Binding Proteins: Optimization of Selectivity for L3MBTL3&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Lindsey&quot;,&quot;last_name&quot;:&quot;I. James&quot;},{&quot;first_name&quot;:&quot;Victoria&quot;,&quot;last_name&quot;:&quot;K. Korboukh&quot;},{&quot;first_name&quot;:&quot;Liubov&quot;,&quot;last_name&quot;:&quot;Krichevsky&quot;},{&quot;first_name&quot;:&quot;Brandi&quot;,&quot;last_name&quot;:&quot;M. Baughman&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Martin Herold&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;L. Norris&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;},{&quot;first_name&quot;:&quot;Dmitri&quot;,&quot;last_name&quot;:&quot;B. Kireev&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;P. Janzen&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;V. Frye&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;16&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;7358-7371&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm400919p&quot;},&quot;abstract&quot;:&quot;Lysine methylation is a key epigenetic mark, the dysregulation of which is linked to many diseases. Small-molecule antagonism of methyl-lysine (Kme) binding proteins that recognize such epigenetic marks can improve our understanding of these regulatory mechanisms and potentially validate Kme binding proteins as drug-discovery targets. We previously reported the discovery of 1 (UNC1215), the first potent and selective small-molecule chemical probe of a methyl-lysine reader protein, L3MBTL3, which antagonizes the mono- and dimethyl-lysine reading function of L3MBTL3. The design, synthesis, and structure–activity relationship studies that led to the discovery of 1 are described herein. These efforts established the requirements for potent L3MBTL3 binding and enabled the design of novel antagonists, such as compound 2 (UNC1679), that maintain in vitro and cellular potency with improved selectivity against other MBT-containing proteins. The antagonists described were also found to effectively interact with unlabel&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400919p&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400919p" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400919p&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400919p" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400919p&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400919p" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm400919p&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400919p&amp;href=/doi/10.1021/jm400919p" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm400919p" /></input><a href="/doi/pdf/10.1021/jm400919p" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm400919p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm400919p%26sid%3Dliteratum%253Aachs%26pmid%3D24040942%26genre%3Darticle%26aulast%3DJames%26date%3D2013%26atitle%3DSmall-Molecule%2BLigands%2Bof%2BMethyl-Lysine%2BBinding%2BProteins%253A%2BOptimization%2Bof%2BSelectivity%2Bfor%2BL3MBTL3%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D18%26spage%3D7358%26epage%3D7371%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291181" title="Amines">Amines</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jmcmar.2013.56.issue-18/production/jmcmar.2013.56.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Lysine methylation is a key epigenetic mark, the dysregulation of which is linked to many diseases. Small-molecule antagonism of methyl-lysine (Kme) binding proteins that recognize such epigenetic marks can improve our understanding of these regulatory mechanisms and potentially validate Kme binding proteins as drug-discovery targets. We previously reported the discovery of <b>1</b> (UNC1215), the first potent and selective small-molecule chemical probe of a methyl-lysine reader protein, L3MBTL3, which antagonizes the mono- and dimethyl-lysine reading function of L3MBTL3. The design, synthesis, and structure–activity relationship studies that led to the discovery of <b>1</b> are described herein. These efforts established the requirements for potent L3MBTL3 binding and enabled the design of novel antagonists, such as compound <b>2</b> (UNC1679), that maintain in vitro and cellular potency with improved selectivity against other MBT-containing proteins. The antagonists described were also found to effectively interact with unlabeled endogenous L3MBTL3 in cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Expression of the genetic code is largely regulated by cell-type specific transcription factors and chemical modifications of histone proteins and DNA. The intricate and dynamic set of post-translational modifications (PTMs), of which methylation is one of the most significant, controls access of the transcriptional machinery to DNA, which in turn determines protein expression and cell function. The methylation state of lysine residues in histones is regulated by families of enzymes that can either ‘write’ (create a PTM) or ‘erase’ (chemically remove) such PTMs, which are known as methyltransferases and demethylases, respectively, whereas ‘readers’ recognize PTMs via protein–protein interactions.</div><div class="NLM_p">Lysine residues can be mono-, di-, or trimethylated. Both the location of this PTM within a histone and the degree of methylation dictate the transcriptional outcome (activation versus repression) as well as the recruitment of Kme modulators (readers, writers, and erasers) that work collectively to maintain an appropriate level of methylation within the cell. Methyl-lysine acts as docking site for specific reader proteins that can in turn alter chromatin structure and direct various cellular processes, often by attracting additional regulatory proteins in a highly coordinated manner.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In addition, histone methylation is dynamic and has been shown to play an important role in cell-cycle regulation, DNA damage and stress response, and cell fate during development and differentiation.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Aberrant methylation levels and ensuing changes in gene expression patterns resuling from the mutation or altered expression of Kme regulators is one mechanism by which such epigenetic factors can contribute to disease.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> There is increasing evidence that many epigenetic regulators are critical proteins dysregulated in cancer, as the levels of histone marks are often altered within cancer epigenomes.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> However, the underlying mechanisms of chromatin regulation in oncogenesis via miswriting, misreading, and/or miserasing methyl-lysine are not understood. One approach to increase our knowledge of these regulatory mechanisms is through small-molecule perturbation. High-quality potent, selective, and cell-penetrant chemical probes serve as excellent tools for improving our understanding of their molecular targets and the broader biological and therapeutic consequences of modulating these targets.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Accordingly, chemical biology efforts focused on deciphering the function of lysine methylation with small-molecule tools have gained momentum, resulting in a number of freely available high-quality chemical probes.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> The methyltransferases were an initial focus of this effort, which has resulted in chemical probes for enzymes including G9a/GLP,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> EZH2,<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14 ref15 ref16">(10-16)</a> and DOT1L,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> all of which have been implicated in tumorgenesis.</div><div class="NLM_p">Kme readers have recently emerged as less precedented epigenetic targets,<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> and antagonism of reader domains may result in cellular effects that are distinct from enzyme inhibitors. We recently reported a first-in-class chemical probe, <b>1</b>, (UNC1215, Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>a) that selectively binds L3MBTL3, a member of the MBT (malignant brain tumor) family of methyl-lysine reader proteins, validating this class of proteins as tractable for probe discovery.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Compound <b>1</b> provided valuable insight into the binding mechanism of L3MBTL3 and enabled the identification of a nonhistone L3MBTL3 Kme substrate, BCLAF1. Although the role of L3MBTL3 in chromatin biology is largely unexplored to date, proteins containing MBT domains have generally been functionally associated with transcriptional repression, chromatin compaction, and significant developmental biology because of their presence in Polycomb complexes.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> Furthermore, it has been reported that L3MBTL3 knockout mice are embryonic lethal because of defects in myeloid lineage differentiation.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Structure of L3MBTL3 chemical probe <b>1</b>. (b) Cocrystal structure of <b>1</b> (green) bound to two molecules of L3MBTL3. The amine meta to the aniline substituent (domain 2 amine) binds in the Kme binding pocket of MBT domain 2 of one L3MBTL3 molecule (magenta), and the amine ortho to the aniline substituent (domain 1 amine) binds to MBT domain 1 of a second L3MBTL3 molecule (cyan). (c) Structures of other L3MBTL3 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Herein, we summarize the structure–activity relationship (SAR) studies that led to the discovery of compound <b>1</b>. Binding affinities for L3MBTL3 were determined for multiple series of compounds by an AlphaScreen assay, and these binding trends were subsequently confirmed in an orthogonal LANCE time-resolved fluorescence resonance energy transfer (TR-FRET) assay, which, to the best of our knowledge, is the first report of this assay in the evaluation of inhibitors of epigenetic protein–protein interactions. To assess the selectivity of these compounds and their affinity for other Kme reader targets, each ligand was screened against a small panel of Kme readers including related proteins from the MBT family as well as representative proteins from other Kme reader families including tudor domains, PHD fingers, and chromodomains. These studies established the requirements for potent L3MBTL3 binding and enabled the design of novel antagonists (such as <b>2</b>, UNC1679; Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>c) that maintain in vitro and cellular potency with improved selectivity against other MBT-containing proteins. Lastly, we have demonstrated that the described inhibitors interact with unlabeled endogenous L3MBTL3 in the context of protein pulldown assays.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23625" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23625" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In an attempt to design potent small-molecule inhibitors of the tudor domain-containing protein p53-binding protein 1 (53BP1) that recognizes dimethyl-lysine (Kme2) in sequences with an adjacent basic residue (e.g., histone 4 R19K20me2<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and p53 K381K382me2<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>), we initially synthesized compounds containing two basic amines: one that we hypothesized would function as a specific Kme mimic and anchor the compound in the methyl-lysine pocket, and another that would interact with the protein in a similar manner as that of the adjacent unmethylated lysine or arginine in the native substrates. On the basis of prior work toward the discovery of an antagonist of L3MBTL1 (<b>3</b>, UNC669; Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>c), a reader of the H4K20me1-2 mark, we found that a 4-(pyrrolidin-1-yl)piperidine amine effectively filled the lysine binding cavity of this domain, enhancing the number of van der Waals contacts in the pocket as compared to Kme2 while also positioning the basic amine in close enough proximity to make a key hydrogen bond in the second MBT domain.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> Anticipating that this amine may be well suited to bind other H4K20me1-2 readers, compound <b>4</b> (UNC928; Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>c) was prepared, containing two meta-4-(pyrrolidin-1-yl)piperidine amines. Although the affinity of <b>4</b> appeared to be weak for 53BP1, our cross-screening strategy<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> revealed submicromolar inhibition of L3MBTL3 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). In comparison to our previous work with analogs of <b>3</b> containing only a single amine,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> it was apparent that the expansion of this scaffold to include a second basic site resulted in a significant boost in potency and selectivity for L3MBTL3. Intrigued by this result, we set out to optimize the inhibitory activity of this compound for L3MBTL3, with the hope that this early lead would evolve into a potent and selective L3MBTL3 antagonist.</div><div class="NLM_p">In the course of these studies, we solved the cocrystal structure of <b>1</b> and L3MBTL3 (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>b, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FL6">4FL6</a>), which helped to elucidate the compound’s mechanism of action and rationalize many of the SAR trends observed.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Surprisingly, a unique 2:2 polyvalent mode of interaction was identified whereby two different L3MBTL3 molecules interact with two molecules of <b>1</b>. L3MBTL3 is a multidomain protein containing three MBT domains, of which MBT domain 1 and MBT domain 2 were found to participate in binding to <b>1</b>. The pyrrolidine amine meta to the aniline substituent of <b>1</b> binds in the Kme reader pocket of MBT domain 2, whereas the pyrrolidine amine ortho to the aniline bridges the two L3MBTL3 proteins by binding more shallowly in MBT domain 1 of a second L3MBTL3 molecule. Site-directed mutagenesis demonstrated that domain 2 interactions were dominant in the binding of <b>1</b>, whereas binding to MBT domain 1 was essential for dimerization as well as the observed boost in potency versus monovalent binding. This is in contrast to the binding of compounds <b>1</b> and <b>3</b> to L3MBTL1, which are monovalent interactions involving only the second MBT domain Kme pocket, as confirmed by crystallography (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P8H">3P8H</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UWN">3UWN</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Although our understanding of this binding mode was not contemporaneous with all of the SAR studies that we present herein, this binding mode provides a rational explanation for the trends observed, and on the basis of this structural model, the amines described will generally be referred to as “domain 1 binding” or “domain 2 binding” (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>a).</div><div class="NLM_p">Compounds were initially designed to explore the SAR of the domain 1 binding amine, the position of the two amines, the linker between the domain 1 amine and the aromatic core, and modifications to the aromatic core. Significant efforts had previously been made to optimize the amine that binds in the dominant Kme binding pocket (domain 2 binding amine) in the context of monobasic scaffolds; however, little improvement was achieved over <b>3</b> for L3MBTL3.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> As a result, the domain 2 binding amine was held constant as 4-(pyrrolidin-1-yl)piperidine throughout most of these SAR studies.</div><div class="NLM_p">The inhibitory activity of analogs of <b>4</b> versus histone peptide binding to L3MBTL3 as well as a small panel of various other Kme reader proteins was evaluated using an AlphaScreen competition assay reported previously.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The direct binding affinity of compounds of particular interest was also confirmed by isothermal titration calorimetry (ITC). Although signal amplification and savings on protein supplies are an advantage of the AlphaScreen assay,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> we observed variation in the results from day to day. Therefore, to most accurately evaluate the SAR of these analogs and their relative activities, about 100 analogs of compound <b>4</b> that demonstrate the most important trends were prepared in serial dilution screening plates and tested simultaneously. This allowed for a more precise comparison of the compounds’ activities and an accurate depiction of the observed binding trends. The serial dilution screening was performed up to 10 μM; therefore, the SAR trends were most closely evaluated for those compounds that were active within the concentration range explored. In addition, a counterscreen assay was conducted in the absence of protein with a His<sub>6</sub>-biotin fusion-peptide substrate to ensure that the compounds did not interfere with the assay by serving as metal chelators, biotin mimetics, or singlet-oxygen quenchers.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">To further validate the SAR trends observed by AlphaScreen, a secondary L3MBTL3 assay was developed that was comparable to AlphaScreen in throughput and efficiency. This LANCE TR-FRET assay is based on the interaction of a biotinylated H4K20me2 peptide bound to a streptavidin-dye conjugate (U<i>Light</i> streptavidin) with a His-tagged L3MBTL3 molecule recognized by an anti-6X His antibody labeled with a europium donor fluorophore. When L3MBTL3 binds the H4K20me2 peptide, the donor and acceptor fluorophores are in close enough proximity to produce a FRET signal, which in turn can be disrupted by small-molecule inhibitors. U<i>Light</i> is a PerkinElmer proprietary dye, chosen for its small size, brightness, and reduced steric hindrance. Similarly to AlphaScreen, this assay was easily miniaturized and automated in a 384-well format and proved to be equally efficient and cost effective. Because of some of the shortcomings in the day-to-day reproducibility that we have encountered with the AlphaScreen assay, we are currently investigating the expansion of TR-FRET-based assays for screening Kme reader proteins more broadly. To compare the screening results of the AlphaScreen and LANCE TR-FRET assays following at least three biological replicates in each format, a scatter plot was generated to compare the two data sets (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). This revealed an overall linear regression with an <i>R</i><sup>2</sup> value of 0.8 that, when combined with ITC confirmations of key compounds, gave us high confidence in interpreting the reported SAR trends.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Scatter plot showing the excellent linear correlation between the L3MBTL3 AlphaScreen data and L3MBTL3 TR-FRET data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our first step toward the optimization of compound <b>4</b> involved the preparation of various structural isomers to determine whether the position of the two amines influenced compound potency. The synthesis of these compounds was straightforward because of their symmetry, and they were easily prepared in one step from the appropriate phthalic acids. All three dibasic compounds (<b>4</b>, <b>5</b>, and <b>6</b>) showed improved potency for L3MBTL3 over compound <b>3</b> and maintained comparable potency as that of <b>3</b> against L3MBTL1 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). This revealed that this series of compounds had the potential to be developed as selective L3MBTL3 inhibitors. The 1,4-bis-benzamide compound, <b>5</b> (UNC1021),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> demonstrated about a 5-fold improvement in potency relative to <b>4</b> (1,3-bis-benzamide), whereas the 1,2-bis-benzamide (<b>6</b>) showed weaker in vitro potency. These geometrical preferences led us to believe that the improved inhibition of L3MBTL3 relative to L3MBTL1 was not purely a nonspecific interaction resulting from the addition of a second basic amine or a change in the compound’s physical properties. Furthermore, the addition of a third 4-(pyrrolidin-1-yl)piperidine amine in compound <b>7</b> did not result in any additional improvement in potency against L3MBTL3. Compounds <b>4</b>–<b>7</b> all are weakly potent against L3MBTL1, which was anticipated; however, compound <b>5</b> demonstrated more selective inhibition of L3MBTL3 over the other panel members, whereas <b>6</b> and <b>7</b> show modest inhibition of 53BP1. The affinity of <b>5</b> for L3MBTL3 was confirmed by ITC, resulting in an average <i>K</i><sub>d</sub> of 390 nM, which is about 7-fold more potent than compound <b>4</b> by ITC. This ITC data closely agrees with the activity trends observed by both high-throughput assays.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Effect of Varying the Substitution Pattern of the 4-(Pyrrolidin-1-yl)piperidine Amines<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0006.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">To explore the nature of the linker between the domain 1 amine and the aromatic core, one amide of compound <b>5</b> was varied to generate asymmetric compounds <b>8</b>, <b>9</b>, and <b>10</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Although the conversion to a sulfonamide linker in compound <b>8</b> was well tolerated, this modification did not further improve the potency against L3MBTL3. Similarly, attaching the amine to the aromatic core directly in compound <b>9</b> to give a more rigid structure and replacing the amide with a methylene group in compound <b>10</b> to increase flexibility resulted in a 2- to 3-fold decrease in potency in each case relative to that of <b>5</b>. Compound <b>5</b> is also the only inhibitor from this series demonstrating IC<sub>50</sub> values greater than 10 μM versus all other panel members (except L3MBTL1), making it the preferred candidate for future optimization. Interestingly, compound <b>9</b> is the only analog screened that shows weak affinity for L3MBTL4, another member of the MBT family, suggesting that the phenyl-4-(pyrrolidin-1-yl)piperidine moiety may be a good starting point for L3MBTL4 ligand development.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Effect of Varying the Linker between the Domain 1 Amine and the Phenyl Core<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0007.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">As compound <b>5</b> showed the most promising activity versus L3MBTL3, we next desymmetrized the molecule to explore the domain 1 binding amine (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). This series of compounds was prepared by reacting 4-(<i>tert</i>-butoxycarbonyl)benzoic acid with 4-(1-pyrrolidinyl)piperidine followed by deprotection with TFA and coupling with the desired amine. Expansion of the domain 1 amine to either a piperidine or azepane in compounds <b>11</b> and <b>12</b>, respectively, decreased potency by about 3- to 4-fold, indicating that increasing the size of this ring is not beneficial. Converting the piperidine amine in compound <b>11</b> to a morpholine (<b>13</b>) or cyclohexyl (<b>14</b>) group, where the basicity of the ring is either diminished or eliminated, decreases the potency by greater than 20-fold, demonstrating that two basic amines are required for nanomolar in vitro potency. In contrast, a smaller dimethyl amino group in compound <b>15</b> results in only about a 4-fold loss in potency relative to that of compound <b>5</b>. This suggests that although the presence of a basic nitrogen in the domain 1 amine is essential for high potency, a variety of substitutents or ring systems are well tolerated.</div><div class="NLM_p">Methylation of the pyrrolidine nitrogen to form a quaternary amine in the domain 1 amine position of compound <b>16</b> is well tolerated, as is the installation of 4-(pyrrolidin-2-yl)piperidine in compound <b>17</b>, resulting in a secondary amine slightly further removed from the aromatic core. Neither compound shows improved potency over compound <b>5</b> for L3MBTL3, but compound <b>17</b> does show a modest affinity for MBTD1 (IC<sub>50</sub> = 5.5 μM). Further increasing the distance between the secondary amine and the aromatic ring in compound <b>18</b> additionally decreases L3MBTL3 potency by greater than 10-fold. Compound <b>19</b> demonstrated a more than a 50-fold loss in affinity by simply swapping the position of the pyrrolidine nitrogen with the adjacent tertiary carbon. This suggests that the nitrogen must be positioned appropriately to make a key interaction(s) and to facilitate binding to the first MBT domain of L3MBTL3. Compounds such as <b>20</b> and <b>21</b>, containing additional basic amines, also did not improve potency.</div><div class="NLM_p">The role of the piperidine ring of the domain 1 binding amine was also investigated. Although replacement with a larger 3-azabicyclo[3.2.1]octane (<b>22</b>) or azepane (<b>23</b>) only resulted in a minimal decrease in L3MBTL3 in vitro potency, the smaller azetidine (<b>24</b>) and pyrrolidine (<b>25</b>) ring systems at this position resulted in a decrease in activity. Increasing the flexibility of the molecule by linking the pyrrolidine to the phenyl core via an alkyl amine as in compound <b>26</b> was also not favorable. We last investigated various spirocycles (<b>27</b>–<b>29</b>) or fused bicyclic amines (<b>30</b> and <b>31</b>). Although compound <b>29</b>, containing the largest spirocycle amine, binds L3MBTL3 about 10-fold weaker than <b>5</b>, the remaining compounds were micromolar ligands, indicating that they are not likely forming favorable interactions with the first MBT domain and therefore not facilitating L3MBTL3 dimerization.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Effect of Varying the Domain 1 Amine<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0008.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0009.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0010.gif" alt="" id="fx4.1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">To determine whether the symmetric nature of compound <b>5</b> enhanced binding to L3MBTL3 relative to the asymmetric compounds in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, a number of other symmetric 1,4-bis-benzamide compounds were synthesized (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). In agreement with the trends from Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, L3MBTL3 binding is affected more significantly by modification of the pyrrolidine rings than by changing the nature of the piperidine rings in compound <b>5</b>. Consequently, compound <b>32</b> with symmetric 4-(pyrrolidin-1-yl)azepane amines shows diminished affinity for L3MBTL3 by about 2-fold, whereas compound <b>33</b> with (1<i>R</i>,5<i>S</i>)-8-(pyrrolidin-1-yl)-3-azabicyclo[3.2.1]octane amines binds about 6-fold weaker. Both of these compounds also show a somewhat diminished affinity for L3MBTL1, indicating that the larger rings are not ideally suited for binding in the domain 2 pocket of L3MBTL1, yet this can be overcome to some extent with L3MBTL3 because of the added interactions of the second amine binding to domain 1. Compounds <b>34</b> and <b>35</b>, where the pyrrolidines are replaced by smaller azetidine rings or the pyrrolidine nitrogens are moved to the 2 position of the ring, respectively, both show reduced activity against L3MBTL3 and L3MBTL1, whereas the methylation of compound <b>35</b> to give tertiary amines (<b>36</b>) binds L3MBTL3 with micromolar affinity. The most significant changes are seen with compounds <b>37</b> and <b>38</b>; although the 1,4′-bipiperidine and <i>N</i>,<i>N</i>-dimethylpiperidin-4-amine are reasonably well tolerated in the domain 1 amine position, these amines are not well suited to bind the Kme pocket of domain 2, resulting in affinities of ca. 10 μM or greater. Because compound <b>37</b> is structurally similar to <b>1</b> but substantially less potent as a L3MBTL3 antagonist, it was used as a negative-control compound in prior cellular studies.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> None of the compounds in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a> were equipotent to <b>5</b>, indicating that the specific nature of the 4-(pyrrolidin-1-yl)piperidine amines contributes more significantly to L3MBTL3 binding than the inherent symmetry of the molecule. Symmetric compounds <b>35</b> and <b>37</b> also are weak antagonists of MBTD1, which correlates well with the observed MBTD1 binding of their asymmetric analogs in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> (compounds <b>17</b> and <b>11</b>, respectively).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Potency of Symmetric Dibasic Compounds<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0011.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0012.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">Because extensive exploration of the amine groups did not result in an L3MBTL3 ligand that is more potent than compound <b>5</b>, we next evaluated whether modification of the phenyl core would serve to further improve L3MBTL3 affinity (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). Overall, a variety of modifications to compound <b>5</b> were reasonably well tolerated, as all ligands retained submicromolar affinity for L3MBTL3, and their affinity for L3MBTL1 was virtually unaffected. The introduction of a pyridine (<b>39</b>), bromophenyl (<b>40</b>), or nitrophenyl (<b>41</b>) core yielded IC<sub>50</sub> values less than 2-fold weaker than <b>5</b>, whereas the aniline (<b>42</b>), tetrafluorophenyl (<b>43</b>), and 2,5-dimethylphenyl (<b>44</b>) analogs were only slightly less potent, with IC<sub>50</sub> values ranging from 3- to 5-fold weaker. This indicates that neither the presence of electron-donating nor electron-withdrawing substituents serves to enhance affinity for L3MBTL3. Furthermore, expansion of the phenyl core to a 1,4-substituted naphthalene (<b>45</b>) results in a potency comparable to <b>5</b> and demonstrates a slightly enhanced affinity for MBTD1 and 53BP1. In contrast, a 2,6-substituted naphthalene core (<b>46</b>), which increases the distance between the two amines, is less favorable, yielding a 10-fold decrease in affinity for L3MBTL3. Lastly, substitution of the phenyl ring for a smaller 2,5-substituted furan ring (<b>47</b>) is less potent by about 7-fold, whereas a more isosteric replacement by a thiophene core (<b>48</b>) is virtually equipotent to compound <b>5</b>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Effect of Modifications to the Aromatic Core<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0013.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0014.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t5fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">Compounds were next synthesized with bulkier substituents on the aromatic core (Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>) under Buchwald, Suzuki, or reductive amination conditions from compounds <b>40</b> or <b>42</b>. Both compound <b>1</b> with a 2-phenylamino substituent and compound <b>49</b> with a 2-benzylamino substituent bind L3MBTL3 virtually identically to compound <b>5</b> in vitro, albeit with a decrease in ligand efficiency. Interestingly, <b>49</b> also displays submicromolar potency against L3MBTL1 as determined by AlphaScreen. However, follow-up ITC studies revealed that <b>49</b> binds L3MBTL1 with micromolar affinity, which is comparable to the <i>K</i><sub>d</sub> values determined previously for compounds <b>1</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and <b>3</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The 2-benzyl (<b>50</b>) and 2-phenoxy (<b>51</b>) analogs of compound <b>1</b> were also found to be equipotent, as was the biphenyl derivative (<b>52</b>). The increased hydrophobicity of this series of inhibitors also resulted in a modest increase in affinity for MBTD1. Although compound <b>1</b> did not show improved potency over compound <b>5</b> in vitro, simultaneous cellular studies revealed that <b>1</b> was a more potent L3MBTL3 inhibitor in cells. As reported previously, compound <b>1</b> showed potent effects on the subnuclear localization of a GFP fusion protein containing the three MBT domains of L3MBTL3 (GFP-3MBT), with an EC<sub>50</sub> of approximately 500 nM, relative to 3 μM for compound <b>5</b>.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> One reason for this difference in cellular activities may be due to the increased lipophilicity of compound <b>1</b>, which has a cLogP value of 2.4 relative to a cLogP of 0.55 for compound <b>5</b>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Effect of Phenyl Substituents on the Aromatic Core<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0015.gif" alt="" id="fx9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t6fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">Despite the fact that compound <b>1</b> demonstrated potent cellular activity and selectivity against more than 250 chromatin-associated effector proteins,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> we were interested in designing potent L3MBTL3 inhibitors with enhanced selectivity over L3MBTL1. On the basis of the SAR presented and the binding mode shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>b, this should be achievable by replacing each 4-(pyrrolidin-1-yl)piperidine amine of <b>1</b> with amines that bind L3MBTL1 more weakly than compound <b>3</b>. We first sought to identify a monobasic ligand (therefore limiting binding to domain 2 of each MBT protein) that selectivity antagonizes histone peptide binding to L3MBTL3 versus L3MBTL1 (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Extensive screening of compounds that could effectively serve as methyl-lysine mimics led to the identification of compound <b>58</b>, which binds L3MBTL3 with an IC<sub>50</sub> of 1.2 μM yet has an IC<sub>50</sub> of greater than 30 μM for L3MBTL1. This is in contrast to compound <b>57</b>, which binds L3MBTL3 and L3MBTL1 equally well. Accordingly, this 2-ethylisoindoline scaffold seemed promising as a Kme mimic to bind the second domain of L3MBTL3 and to enhance selectivity by decreasing binding to the second domain of L3MBTL1. In choosing an amine to interact with the first domain of L3MBTL3, we started with those compounds in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> that still have L3MBTL3 activity within 3-fold of that of compound <b>5</b> but that display diminished potency for L3MBTL1 as a monobasic species relative to <b>3</b>. For example, compound <b>22</b> is only about 2-fold weaker than <b>5</b> against L3MBTL3, whereas monobasic compound <b>59</b> binds L3MBTL1 weakly with an IC<sub>50</sub> of 20 μM, suggesting that 3-azabicyclo[3.2.1]octane may be a good choice for the domain 1 amine position. Similarly, 4-(pyrrolidin-2-yl)piperidine seemed promising, as <b>17</b> binds L3MBTL3 potently, yet compound <b>60</b> binds both L3MBTL1 and L3MBTL3 with an IC<sub>50</sub> greater than 30 μM.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of monobasic MBT domain inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>53</b> was synthesized and, not surprisingly, binds L3MBTL1 with an IC<sub>50</sub> of 4.0 μM (Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>). When the 4-(pyrrolidin-1-yl)piperidine amine was replaced with 3-azabicyclo[3.2.1]octane as in compound <b>54</b>, weak potency for L3MBTL1 was maintained, whereas compound <b>2</b> containing 4-(pyrrolidin-2-yl)piperidine showed an IC<sub>50</sub> of greater than 10 μM, as presumably neither amine in compound <b>2</b> binds the domain 2 Kme pocket of L3MBTL1 very well, as intended. However, both compounds <b>54</b> and <b>2</b> retained nanomolar potency versus L3MBTL3, within 3-fold of that of <b>5</b>. Furthermore, they both demonstrated IC<sub>50</sub> values greater than 10 μM against the remaining Kme reader proteins in the panel, maintaining the overall selectivity of compound <b>5</b>. As compound <b>2</b> appeared to be the weakest L3MBTL1 antagonist, we next prepared aniline derivatives analogous to <b>1</b>, with the aniline substituent both ortho and meta to the 2-ethylisoindoline Kme mimic (<b>55</b> and <b>56</b>). Although both of these compounds were equipotent to <b>2</b> against L3MBTL3, they gained some activity against L3MBTL1, MBTD1, and 53BP1 in the case of compound <b>55</b>. Overall, this suggests that the aniline substituent may increase nonspecific binding and that its location relative to the 2-ethylisoindoline is not significant in L3MBTL3 binding. Although neither <b>55</b> nor <b>56</b> showed improved in vitro selectivity properties over compound <b>2</b>, we expect that similarly to <b>1</b>, these compounds could have a higher potency in cells.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Potency of Compounds Containing Novel Methyl-Lysine Mimics<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0016.gif" alt="" id="fx10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t7fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">Because compounds <b>2</b> and <b>56</b> showed promise as more selective L3MBTL3 inhibitors, we sought to quantify further the level of selectivity by ITC (Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a>). Compound <b>2</b> binds L3MBTL3 with a <i>K</i><sub>d</sub> of 0.47 ± 0.14 μM and L3MBTL1 with a <i>K</i><sub>d</sub> of 68 ± 18 μM, resulting in ca. 150-fold selectivity, whereas compound <b>56</b> is even more selective for L3MBTL3, demonstrating approximately 400-fold selectivity by ITC. In comparison, ITC data for compound <b>5</b> revealed that the compound was 16-fold more selective for L3MBTL3 than L3MBTL1, whereas compound <b>1</b> is 78-fold more selective for L3MBTL3 than L3MBTL1 because of its slightly higher L3MBTL3 potency by ITC.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> These data suggest that it is possible to improve the selectivity of potent L3MBTL3 inhibitors by varying the amines that bind to both the first and second MBT domain in the dimeric binding mode (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>b), and they highlight the feasibility of generating novel amine scaffolds that function as Kme mimetics.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Inhibitor Selectivity for L3MBTL3 over L3MBTL1 as Determined by AlphaScreen and ITC<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0017.gif" alt="" id="fx11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0018.gif" alt="" id="fx12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup>Table a</sup><p class="last">The data for the average IC<sub>50</sub> values was calculated from the number of replicate runs indicated in parentheses.</p></div><div class="footnote" id="t8fn2"><sup>Table b</sup><p class="last">IC<sub>50</sub> values were calculated by averaging the data points for each compound concentration and plotted using four-parameter curve fitting when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</p></div></div><div></div></div><div class="NLM_p">In an attempt to better understand the preference of the 2-ethylisoindoline Kme mimic for L3MBTL3, we performed a structural comparison of the Kme binding pockets of L3MBTL1 and L3MBTL3. Both proteins are highly similar, with the major exception being a phenylalanine near the L3MBTL3 Kme reader pocket (F387) that is a leucine in L3MBTL1 (L361). Compound <b>58</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>) was docked into L3MBTL1 by superimposing it with compound <b>3</b> in the cocrystal structure of L3MBTL1 and <b>3</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P8H">3P8H</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The docked ligand was then transferred into L3MBTL3 by protein–protein alignment. Subsequent energy optimizations of compound <b>58</b> docked to both proteins suggested that there are favorable interactions between F387 of L3MBTL3 and the 2-ethylisoindoline group of <b>58</b>, which are stabilized by either π–π or CH−π interactions. To evaluate experimentally the contribution of F387 to binding, an L3MBTL1 L361F mutant was prepared in an attempt to engineer a more ‘L3MBTL3-like’ binding pocket, and the binding of the mutant protein to compounds <b>1</b>, <b>2</b>, and <b>57</b> was evaluated by ITC. Although compounds <b>1</b> and <b>57</b> were virtually unaffected by the mutation, binding L3MBTL1 L361F within about 2-fold of wildtype L3MBTL1, compound <b>2</b> demonstrated a 12-fold increase in affinity for L3MBTL1 L361F relative to wildtype. This data supports the notion that the enhanced selectivity of 2-ethylisoindoline containing compounds is due to specific interactions with F387 in L3MBTL3 that are absent in L3MBTL1 binding. In contrast, the binding of 4-(pyrrolidin-1-yl)piperidine Kme mimics does not seem to be significantly influenced by the presence of a phenylalanine at this position.</div><div class="NLM_p">The cytotoxicity of compound <b>2</b> was also evaluated in comparison to <b>1</b> and <b>5</b>. All compounds showed no observable level of cytotoxicity up to 100 μM, as measured by a CellTiter-Glo luminescent cell-viability assay. Furthermore, although compound <b>1</b> was previously reported to have weak antagonist activity against the M<sub>1</sub> muscarinic receptor (IC<sub>50</sub> = 3.5 μM),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> compound <b>2</b> had no appreciable antagonist activity against M<sub>1</sub> (less than 20% inhibition at 30 μM).</div><div class="NLM_p">We next wanted to confirm the cellular activity of compounds <b>2</b> and <b>56</b>. To demonstrate the ability of <b>1</b> to bind full-length L3MBTL3 in cells, we previously reported the use of a fluorescent conjugate, mero76-<b>1</b>, to visualize the colocalization of the fluorescent antagonist with an N-terminal GFP fusion of the full-length L3MBTL3 protein (GFP-FLMBT).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Therefore, we treated HEK293 cells containing GFP-FLMBT and mero76-<b>1</b> with <b>1</b> or <b>5</b> equiv (relative to mero76-<b>1</b>) of the untagged competitor and observed the displacement of mero76-<b>1</b> from L3MBTL3 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). As expected, treatment with compound <b>1</b> largely displaces mero76-<b>1</b> from GFP-FLMBT, eliminating mero76-<b>1</b> foci at both concentrations tested. Compounds <b>2</b> and <b>56</b> were also found to be potent cellular L3MBTL3 antagonists, abolishing mero76-<b>1</b> foci when the cells were treated with 5 equiv of compound. When treating the cells with only 1 equiv of compound, both <b>1</b> and <b>56</b> appear slightly more effective than <b>2</b> at displacing mero76-<b>1</b> from L3MBTL3. Furthermore, mero76 alone (not conjugated to <b>1</b>) does not colocalize with GFP-FLMBT, confirming that foci formation is dependent on binding to compound <b>1</b> and mero76-<b>1</b> also does not form nuclear foci in the absence of GFP-FLMBT.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mero76-<b>1</b> colocalizes with GFP-L3MBTL3 in the nucleus in HEK293 cells (row 1), and compounds <b>1</b>, <b>2</b>, and <b>56</b> (1 equiv relative to mero76-<b>1</b>; rows 2–4) all displace mero76-<b>1</b> from GFP-L3MBTL3 (red is mero76-<b>1</b>, green is GFP-L3MBTL3, and blue is Hoechst). The scale bar is 10 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Encouraged by the observed cellular potency of these compounds, we sought to demonstrate for the first time that L3MBTL3 antagonists bind endogenous L3MBTL3. Because of the generally low levels of endogenous L3MBTL3 expression, we investigated L3MBTL3 mRNA levels in a variety of cell lines. G-401 cells were identified as having higher levels (by 5-fold or greater) of endogenous L3MBTL3 than other cell lines, including HEK293, U2OS, and MCF10A. G-401 cells are derived from a rhabdoid tumor of the kidney; however, the relevance of this higher expression level is not known. Through traditional pulldown assays, we explored the ability of a biotinylated analog of <b>1</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> to affinity purify L3MBTL3 from G-401 cell lysates (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). Indeed, endogenous L3MBTL3 could be efficiently purified from whole-cell lysates with this biotinylated reagent as well as competed off with soluble <b>1</b>. In addition, compound <b>56</b> also demonstrated potent binding to endogenous L3MBTL3 by effectively competing with biotin-<b>1</b>, whereas the negative-control compound (<b>37</b>) failed to compete for binding to endogenous L3MBTL3. These results together strongly suggest that compounds <b>1</b> and <b>56</b> are potent cellular inhibitors of the Kme binding function of L3MBTL3 and that neither artificial tagging of L3MBTL3 or L3MBTL3 overexpression are required for this interaction.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western analysis of biotin-<b>1</b> pulldown experiments in G-401 cell lysates. Biotin-<b>1</b> effectively binds endogenous L3MBTL3. Compounds <b>1</b> and <b>56</b> inhibit this interaction, whereas the inactive control compound, <b>37</b>, cannot successfully block L3MBTL3 binding.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18282" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18282" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">To improve the in vitro potency of a preliminary submicromolar inhibitor of L3MBTL3 (<b>4</b>), define the properties required to achieve potent L3MBTL3 antagonism, and explore the mechanism of action of our L3MBTL3 chemical probe (<b>1</b>), we designed several series of novel analogs and evaluated their SARs. All compounds were screened against a panel of 11 Kme reader proteins by AlphaScreen, and an orthogonal L3MBTL3 TR-FRET assay was developed that further validated the observed SAR trends. Through the knowledge gained in these studies, we were able to design novel antagonists (<b>2</b>) that maintain in vitro and cellular potency with improved selectivity against the closely related MBT-containing protein, L3MBTL1. The ability of subtle modifications, steric and electronic in nature, to the amine groups that occupy the Kme pockets of L3MBTL3 to significantly modulate the affinity bodes well for the design of selective ligands for other Kme domains. It was also demonstrated that compounds <b>1</b> and <b>56</b> interact with endogenous L3MBTL3 in the context of protein pulldown assays, which is essential for the validation of tools to study the function of L3MBTL3 endogenously. Having demonstrated the chemical tractability of this target class, increased efforts toward the development of small-molecule tools for additional Kme reader domains will provide new opportunities to investigate their role in chromatin biology and drug discovery.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71577" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71577" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> General Chemistry Procedures</h3><div class="NLM_p last">Analytical LCMS data for all compounds were acquired using an Agilent 6110 series system with the UV detector set to 220 and 254 nm. Samples were injected (<10 μL) onto an Agilent Eclipse Plus 4.6 × 50 mm, 1.8 μm, C18 column at room temperature. A mobile phase of A (H<sub>2</sub>O + 0.1% acetic acid) and B (MeOH + 0.1% acetic acid) was used with a linear gradient from 10 to 100% B in 5.0 min followed by a flush at 100% B for another 2 min with a flow rate of 1.0 mL/min. Mass spectra data were acquired in positive-ion mode using an Agilent 6110 single quadrupole mass spectrometer with an electrospray ionization source. Nuclear magnetic resonance (NMR) spectra were recorded on a Varian Mercury spectrometer at 400 MHz for proton (<sup>1</sup>H NMR) and 100 MHz for carbon (<sup>13</sup>C NMR); chemical shifts are reported in ppm (δ). Preparative HPLC was performed using an Agilent Prep 1200 series with the UV detector set to 220 and 254 nm. Samples were injected onto a Phenomenex Luna 250 × 30 mm, 5 μm, C18 column at room temperature. Mobile phases of A (H<sub>2</sub>O + 0.1% TFA) and B (MeOH) were used with a flow rate of 40 mL/min. A general gradient of 0–5 min at 5% B, 5–20 min increasing from 5 to 40% B, and 20–25 min increasing from 40 to 80% B was used followed by a 100% B flush for another 3 min. Small variations in this purification method were made as needed to achieve the ideal separation for each compound. All compounds that were evaluated in biochemical and biophysical assays had >95% purity as determined by <sup>1</sup>H NMR and LCMS.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> AlphaScreen</h3><div class="NLM_p">The AlphaScreen assay was generally performed as previously described.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> In brief, compound plates (1 μL at 10 or 30 mM highest concentration; 3-fold, 10-point dilutions in DMSO) were diluted in 1× assay buffer (20 mM TRIS pH 8.0, 25 mM NaCl, 2 mM DTT, and 0.05% Tween-20) over two steps to 100 or 300 μM using a Multimek robotic pipettor (Nanoscreen), and 1 μL was spotted into the wells of 384-well low-volume Proxiplates (PerkinElmer). To these plates, 9 μL of protein-peptide mix in 1× assay buffer was added by Multidrop (Thermo) and incubated for 30 min at room temperature. Next, 2 μL of a 1:1 mixture of streptavidin-conjugate donor and nickel-chelate acceptor beads (45 μg/mL in 1× assay buffer) were added, and the plates were allowed to incubate for an additional 30 min in the dark at room temperature. After incubation, the plates were read on an EnVision multilabel reader equipped with an HTS AlphaScreen laser (PerkinElmer). The screens reported were performed at concentrations of up to either 10 or 30 μM as indicated; therefore, it should be noted that those compounds referred to as inactive are indeed inactive only within the concentration range tested. PHF23 and JARID1A are GST tagged and consequently GST-acceptor beads were used for these assays.</div><div class="NLM_p last">The IC<sub>50</sub> values reported are the average of at least three values ± the standard deviation. When specified in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>, the IC<sub>50</sub> values were calculated using four-parameter curve fitting (GraphPad Prism 5) from replicate runs using the averaged response values for each compound concentration. This was required only when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> LANCE TR-FRET</h3><div class="NLM_p">Compound plates were treated as described for the AlphaScreen experiments. Compound plates (1 μL at 10 mM highest concentration; 3-fold, 10-point dilutions in DMSO) were diluted in 1× assay buffer (20 mM TRIS pH 8.0, 25 mM NaCl, 2 mM DTT, and 0.05% Tween-20) over two steps to 100 μM using a Multimek robotic pipettor (Nanoscreen), and 1 μL was spotted into the wells of 384-well low-volume Proxiplates (PerkinElmer). To these plates, 4.5 μL of a 1:1 L3MBTL3/H4K20me2 mixture (555 nM each) in 1× assay buffer was added by Multidrop (Thermo) and incubated for 30 min at room temperature. Next, 4.5 μL of a mixture of Eu–W1024-Anti6His (2.2 nM) and U<i>Light</i>-streptavidin (137.5 nM) were added, and the plates were allowed to incubate for an additional 30 min in the dark at room temperature. After incubation, the plates were read on an EnVision multilabel reader (PerkinElmer) equipped with a LANCE/DELFIA D400/630 dual mirror, UV2(TRF) 320 nm excitation filter, and two emission filters (APC 665 nm and Europium 615 nM).</div><div class="NLM_p last">The IC<sub>50</sub> values reported are the average of at least three values ± the standard deviation. When specified in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>, the IC<sub>50</sub> values were calculated using four-parameter curve fitting (GraphPad Prism 5) from replicate runs using the averaged response values for each compound concentration. This was required only when the IC<sub>50</sub> values for a single compound were not all active (<10 μM) or inactive (>10 μM).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">All ITC measurements were recorded at 25 °C with an AutoITC<sub>200</sub> microcalorimeter (MicroCal Inc.). All protein and compound stock samples were in the target buffer (25 mM Tris-HCl, pH 8, 75–150 mM NaCl, and 2 mM β-mercaptoethanol) and then diluted in the same buffer to achieve the desired concentrations: 30–90 μM protein and 0.3–2 mM compound, depending on the expected dissociation constant. The concentration of protein stock solutions were established using the Edelhoch method, whereas 10 mM compound stock solutions were prepared gravimetrically. A typical experiment included a single 0.2 μL compound injection into a 200 μL cell filled with protein followed by 26 subsequent 1.5 μL injections of compound. Injections were performed with a spacing of 180 s and a reference power of 8 μcal/sec. Control experiments were performed by titrating each compound into buffer under identical conditions to determine the heat signals, if any, that arise from diluting the compound. If applicable, the heats of dilution generated were then subtracted from the protein–compound binding curves. The titration data was analyzed using Origin Software (MicroCal Inc.) by nonlinear least squares, fitting the heats of binding as a function of the compound/protein ratio. The data were fit on the basis of a one set of sites model. L3MBTL3 binding was also fit with a one site model because of the fact that two binding events were not observed and therefore the data could not be fit on the basis of two independent or sequential binding events.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Immunofluorescence Experiments</h3><div class="NLM_p last"> HEK293 cells (150 000) were seeded on coverslips in 12-well plates. After 24 h, the cells were transfected with GFP-L3MBTL3, and after another 16 h they were treated with mero76-<b>1</b> (5 μM). After a 3 h incubation period, the cells were treated with 5 or 25 μM of compound <b>1</b>, <b>2</b>, or <b>56</b> for an additional 3 h. Prior to imaging, the cells were stained with Hoechst 33342 dye, washed with PBS, and incubated with DMEM/F-12 media without phenol red (11039). Images were acquired using a Quorum Spinning Disk Confocal microscope equipped with 405, 491, 561, and 642 nm lasers. Image analysis was performed using Volocity 5.4.1.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Affinity Purification Experiments</h3><div class="NLM_p last">Frozen G-401 cell pellets were lysed in lysis buffer containing 50 mM Tris-HCl, pH 8.0, 500 mM NaCl, 2 mM EDTA, 1% Triton X-100, and complete protease-inhibitor cocktail (Roche). After vortexing and centrifugation at 13 000 rpm at 4 °C for 5 min to pellet the DNA, the supernatant was diluted to 150 mM NaCl (while keeping the concentrations of the other reagents constant). The lysate concentration was determined by a BCA colorimetric assay (Pierce). Approximately 1 mg of cell lysate was used per immunoprecipitation. Lysate was incubated in the absence or presence of compound (1 or 20 equiv relative to biotin-<b>1</b>; 1.75 or 35 nmol, respectively) at 4 °C for 1 h. Magnetic streptavidin beads (50 μL, binding capacity is 1.75 nmol; Pierce) prewashed with IP buffer (50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 2 mM EDTA, 1% Triton X-100, and protease-inhibitor cocktail (Roche)) were saturated with biotin-<b>1</b> (or biotin as a negative control). The beads were incubated with biotin-<b>1</b> or biotin (35 nmol) in 200 μL of IP buffer for 30 min at room temperature. Three washes were performed to remove all excess biotin-<b>1</b> (or biotin), and the beads were resuspended in the corresponding lysate and rotated overnight at 4 °C. Three washes were performed with IP buffer followed by elution with 1× NuPAGE Sample Reducing Agent (Invitrogen) and 1× NuPAGE LDS Sample Buffer (Invitrogen), analysis by SDS-PAGE, and immunoblotting.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19"><a href="/doi/suppl/10.1021/jm400919p">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Synthesis of compounds <b>1</b>–<b>60</b>; protein purification and expression details; CellTiter-Glo luminescent cell viability assay details and results; details and results of the M1 and M2 assays; AlphaScreen proteins and corresponding peptide substrates; representative ITC binding curves; model of compound <b>58</b> in complex with L3MBTL3 and L3MBTL1; immunofluorescence results of <b>1</b>, <b>2</b>, and <b>56</b> in HEK293 cells; L3MBTL3 mRNA levels in various cell lines; and <sup>1</sup>H NMR spectra of compounds <b>11</b>–<b>31</b>, <b>39</b>, and <b>41</b>–<b>48</b>. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400919p/suppl_file/jm400919p_si_001.pdf">jm400919p_si_001.pdf (2.87 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm400919p" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53327" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53327" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen V. Frye</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#691a1f0f1b100c290c04080005471c070a470c0d1c"><span class="__cf_email__" data-cfemail="5e2d28382c273b1e3b333f3732702b303d703b3a2b">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lindsey I. James</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Victoria K. Korboukh</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liubov Krichevsky</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">‡Structural
Genomics Consortium, University of Toronto, §Department of Medical
Biophysics, University of Toronto, and ∥Princess Margaret
Cancer Centre, 101 College Street, Toronto, Ontario M5G
1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brandi M. Baughman</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Martin Herold</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline L. Norris</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dmitri B. Kireev</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William P. Janzen</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Center for
Integrative Chemical
Biology and Drug Discovery, Division of Chemical Biology and Medicinal
Chemistry, UNC Eshelman School of Pharmacy, University
of North Carolina at Chapel Hill, Chapel Hill, North
Carolina 27599, United States</span>; 
    <span class="hlFld-Affiliation affiliation">‡Structural
Genomics Consortium, University of Toronto, §Department of Medical
Biophysics, University of Toronto, and ∥Princess Margaret
Cancer Centre, 101 College Street, Toronto, Ontario M5G
1L7, Canada</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d1114e1700-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72753" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72753" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Emily Hull-Ryde for support with the CellTiter-Glo cell viability assay, Chris MacNevin and Klaus Hahn for providing mero76, Xi-Ping for support with the M<sub>1</sub> muscarinic receptor assay, Greg Wang for providing PHF23 and JARID1, and Timothy Willson and John Moore for helpful discussions and access to resources for the endogenous L3MBTL3 pulldown assays. The research described here was supported by the National Institute of General Medical Sciences, U.S. National Institutes of Health (NIH, grants RC1GM090732 and R01GM100919), the Carolina Partnership and the University Cancer Research Fund, University of North Carolina at Chapel Hill, the Ontario Research Fund (grant ORF-GL2), the Natural Sciences and Engineering Research Council of Canada, and the Structural Genomics Consortium, which is a registered charity (no. 1097737) that receives funds from AbbVie, Boehringer Ingelheim, the Canada Foundation for Innovation, the Canadian Institutes for Health Research (CIHR), Genome Canada through the Ontario Genomics Institute [OGI-055], GlaxoSmithKline, Janssen, Lilly Canada, the Novartis Research Foundation, the Ontario Ministry of Economic Development and Innovation, Pfizer, Takeda, and the Wellcome Trust (092809/Z/10/Z). L.K. holds a Frederick Banting and Charles Best Canada Graduate Scholarship from the CIHR, and C.H.A. holds a Canada Research Chair in Structural Genomics.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i23" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i23"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i24" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i24"> Abbreviations</h2><tr><td class="NLM_term">Kme</td><td class="NLM_def"><p class="first last">methyl-lysine</p></td></tr><tr><td class="NLM_term">Kme2</td><td class="NLM_def"><p class="first last">dimethyl-lysine</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">MBT</td><td class="NLM_def"><p class="first last">malignant brain tumor</p></td></tr><tr><td class="NLM_term">L3MBTL3</td><td class="NLM_def"><p class="first last">Lethal(3)malignant brain tumor-like protein 3</p></td></tr><tr><td class="NLM_term">L3MBTL1</td><td class="NLM_def"><p class="first last">Lethal(3)malignant brain tumor-like protein 1</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">H4K20me1-2</td><td class="NLM_def"><p class="first last">mono- or dimethylation of lysine 20 on histone 4</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30781" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Multivalent engagement of chromatin modifications by linked binding modules</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">994</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnrm2298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=18037899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=983-994&author=A.+J.+Ruthenburgauthor=H.+Liauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Multivalent+engagement+of+chromatin+modifications+by+linked+binding+modules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Multivalent engagement of chromatin modifications by linked binding modules</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Li, Haitao; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">983-994</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone post-translational modifications (PTMs) play crucial roles in genome management, in part by recruiting specific factors that alter the structural properties of chromatin.  These so-called effector complexes often comprise multiple histone-binding modules that may act in concert to regulate chromatin structure and DNA-related activities.  Various chem. modifications on histones and regions of assocd. DNA play crucial roles in genome management by binding specific factors that, in turn, serve to alter the structural properties of chromatin.  These so-called effector proteins have typically been studied with the biochemist's paring knife; the capacity to recognize specific chromatin PTMs has been mapped to an increasing no. of domains that frequently appear in the nuclear subset of the proteome, often present in large, multisubunit complexes that bristle with modification-dependent binding potential.  The authors propose that multivalent interactions on a single histone tail and beyond may have a significant, if not dominant, role in chromatin transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkIrlnyvt64LVg90H21EOLACvtfcHk0ljuq_zinytpfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L&md5=0d27fc9ba2e2a009e0a9903e1c65bdf7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm2298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2298%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMultivalent%2520engagement%2520of%2520chromatin%2520modifications%2520by%2520linked%2520binding%2520modules%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D983%26epage%3D994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0ljuq_zinytpfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discov.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lj5Xot2atsFvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The epigenomics of cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0lj5Xot2atsFvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Whyte, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlando, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hnisz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagey, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span> </span><span class="NLM_article-title">Master transcription factors and mediator establish super-enhancers at key cell identity genes</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">319</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=307-319&author=W.+A.+Whyteauthor=D.+A.+Orlandoauthor=D.+Hniszauthor=B.+J.+Abrahamauthor=C.+Y.+Linauthor=M.+H.+Kageyauthor=P.+B.+Rahlauthor=T.+I.+Leeauthor=R.+A.+Young&title=Master+transcription+factors+and+mediator+establish+super-enhancers+at+key+cell+identity+genes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhyte%26aufirst%3DW.%2BA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DKagey%26aufirst%3DM.%2BH.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DMaster%2520transcription%2520factors%2520and%2520mediator%2520establish%2520super-enhancers%2520at%2520key%2520cell%2520identity%2520genes%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D307%26epage%3D319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lj5Xot2atsFvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span>Structural Genomics Consortium. <a href="http://www.thesgc.org/scientists/chemical_probes" class="extLink">http://www.thesgc.org/scientists/chemical_probes</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Structural+Genomics+Consortium.+http%3A%2F%2Fwww.thesgc.org%2Fscientists%2Fchemical_probes."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Small-molecular modulators of cancer-associated epigenetic mechanisms</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1039%2Fc3mb25410k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23511667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=873-896&author=Y.+Itohauthor=T.+Suzukiauthor=N.+Miyata&title=Small-molecular+modulators+of+cancer-associated+epigenetic+mechanisms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecular modulators of cancer-associated epigenetic mechanisms</span></div><div class="casAuthors">Itoh, Yukihiro; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">873-896</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  DNA methylation and posttranslational histone modifications regulate expression of various genes independently of changes in the DNA sequence.  Such epigenetic mechanisms play important roles in controlling cellular functions, including the cell cycle, immunoresponses and signal transduction.  On the other hand, epigenetic aberrations are assocd. with oncogenesis and proliferation of cancer cells, and epigenetic alterations have been identified in many human cancer cells.  Furthermore, chem.-biol. approaches have uncovered relationships between epigenetic dysregulation and cancer, and several small mols. that modulate epigenetic mechanisms have already been approved for cancer therapy.  In this review, we deal with chem. epigenetics relevant to cancer therapy, focusing esp. on small mols. that regulate epigenetic mechanisms related to DNA methylation and histone modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo76xFXo9kCe7Vg90H21EOLACvtfcHk0lgHQKcfhwljpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D&md5=ecc7cd46637f6010872510329d51ac90</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2Fc3mb25410k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3mb25410k%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSmall-molecular%2520modulators%2520of%2520cancer-associated%2520epigenetic%2520mechanisms%26jtitle%3DMol.%2520BioSyst.%26date%3D2013%26volume%3D9%26spage%3D873%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.+C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.+P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lgHQKcfhwljpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lhpgFL_PinF2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Della Pietra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+Della+Pietraauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+R.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDella%2BPietra%26aufirst%3DA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%2BR.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of potent, selective, cell-active inhibitors of the histone lysine methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+potent%2C+selective%2C+cell-active+inhibitors+of+the+histone+lysine+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0lhRMjnjwall_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520potent%252C%2520selective%252C%2520cell-active%2520inhibitors%2520of%2520the%2520histone%2520lysine%2520methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Qi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardayfio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span> </span><span class="NLM_article-title">Selective inhibition of Ezh2 by a small molecule inhibitor blocks tumor cells proliferation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">21360</span><span class="NLM_x">–</span> <span class="NLM_lpage">21365</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=21360-21365&author=W.+Qiauthor=H.+Chanauthor=L.+Tengauthor=L.+Liauthor=S.+Chuaiauthor=R.+Zhangauthor=J.+Zengauthor=M.+Liauthor=H.+Fanauthor=Y.+Linauthor=J.+Guauthor=O.+Ardayfioauthor=J.+H.+Zhangauthor=X.+Yanauthor=J.+Fangauthor=Y.+Miauthor=M.+Zhangauthor=T.+Zhouauthor=G.+Fengauthor=Z.+Chenauthor=G.+Liauthor=T.+Yangauthor=K.+Zhaoauthor=X.+Liuauthor=Z.+Yuauthor=C.+X.+Luauthor=P.+Atadjaauthor=E.+Li&title=Selective+inhibition+of+Ezh2+by+a+small+molecule+inhibitor+blocks+tumor+cells+proliferation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DArdayfio%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DC.%2BX.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DSelective%2520inhibition%2520of%2520Ezh2%2520by%2520a%2520small%2520molecule%2520inhibitor%2520blocks%2520tumor%2520cells%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D21360%26epage%3D21365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macnevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the lysine methyltransferases EZH2 and EZH1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1324</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+Macnevinauthor=F.+Liuauthor=C.+Gaoauthor=X.+P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+lysine+methyltransferases+EZH2+and+EZH1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0lhRMjnjwall_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacnevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7922</span><span class="NLM_x">–</span> <span class="NLM_lpage">7927</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0lhgQl2wa6b0fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Beguelin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teater, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezponda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez-Garcia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoyne, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdel-Wahab, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Licht, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A. M.</span><span> </span><span class="NLM_article-title">EZH2 is required for germinal center formation and somatic EZH2 mutations promote lymphoid transformation</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=677-692&author=W.+Beguelinauthor=R.+Popovicauthor=M.+Teaterauthor=Y.+Jiangauthor=K.+L.+Buntingauthor=M.+Rosenauthor=H.+Shenauthor=S.+N.+Yangauthor=L.+Wangauthor=T.+Ezpondaauthor=E.+Martinez-Garciaauthor=H.+Zhangauthor=Y.+Zhengauthor=S.+K.+Vermaauthor=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+S.+Van+Allerauthor=R.+G.+Krugerauthor=Y.+Liuauthor=C.+F.+McHughauthor=D.+W.+Scottauthor=Y.+R.+Chungauthor=N.+Kelleherauthor=R.+Shaknovichauthor=C.+L.+Creasyauthor=R.+D.+Gascoyneauthor=K.+K.+Wongauthor=L.+Cerchiettiauthor=R.+L.+Levineauthor=O.+Abdel-Wahabauthor=J.+D.+Lichtauthor=O.+Elementoauthor=A.+M.+Melnick&title=EZH2+is+required+for+germinal+center+formation+and+somatic+EZH2+mutations+promote+lymphoid+transformation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DTeater%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DRosen%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DEzponda%26aufirst%3DT.%26aulast%3DMartinez-Garcia%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DScott%26aufirst%3DD.%2BW.%26aulast%3DChung%26aufirst%3DY.%2BR.%26aulast%3DKelleher%26aufirst%3DN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DEZH2%2520is%2520required%2520for%2520germinal%2520center%2520formation%2520and%2520somatic%2520EZH2%2520mutations%2520promote%2520lymphoid%2520transformation%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D677%26epage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chory, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wernimont, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scopton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federation, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wienholds, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fncomms2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23250418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1288&author=W.+Yuauthor=E.+J.+Choryauthor=A.+K.+Wernimontauthor=W.+Tempelauthor=A.+Scoptonauthor=A.+Federationauthor=J.+J.+Marineauauthor=J.+Qiauthor=D.+Barsyte-Lovejoyauthor=J.+Yiauthor=R.+Marcellusauthor=R.+E.+Iacobauthor=J.+R.+Engenauthor=C.+Griffinauthor=A.+Amanauthor=E.+Wienholdsauthor=F.+Liauthor=J.+Pinedaauthor=G.+Estiuauthor=T.+Shatsevaauthor=T.+Hajianauthor=R.+Al-Awarauthor=J.+E.+Dickauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+E.+Bradnerauthor=M.+Schapira&title=Catalytic+site+remodelling+of+the+DOT1L+methyltransferase+by+selective+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span></div><div class="casAuthors">Yu Wenyu; Chory Emma J; Wernimont Amy K; Tempel Wolfram; Scopton Alex; Federation Alexander; Marineau Jason J; Qi Jun; Barsyte-Lovejoy Dalia; Yi Joanna; Marcellus Richard; Iacob Roxana E; Engen John R; Griffin Carly; Aman Ahmed; Wienholds Erno; Li Fengling; Pineda Javier; Estiu Guillermina; Shatseva Tatiana; Hajian Taraneh; Al-Awar Rima; Dick John E; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Bradner James E; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1288</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Selective inhibition of protein methyltransferases is a promising new approach to drug discovery.  An attractive strategy towards this goal is the development of compounds that selectively inhibit binding of the cofactor, S-adenosylmethionine, within specific protein methyltransferases.  Here we report the three-dimensional structure of the protein methyltransferase DOT1L bound to EPZ004777, the first S-adenosylmethionine-competitive inhibitor of a protein methyltransferase with in vivo efficacy.  This structure and those of four new analogues reveal remodelling of the catalytic site.  EPZ004777 and a brominated analogue, SGC0946, inhibit DOT1L in vitro and selectively kill mixed lineage leukaemia cells, in which DOT1L is aberrantly localized via interaction with an oncogenic MLL fusion protein.  These data provide important new insight into mechanisms of cell-active S-adenosylmethionine-competitive protein methyltransferase inhibitors, and establish a foundation for the further development of drug-like inhibitors of DOT1L for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcST2GJQO5XUmjDnmmyUf5qOfW6udTcc2eb46_3u48cVgbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D&md5=f7b501ac4188b14b48fc435637e0e192</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2304%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChory%26aufirst%3DE.%2BJ.%26aulast%3DWernimont%26aufirst%3DA.%2BK.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DScopton%26aufirst%3DA.%26aulast%3DFederation%26aufirst%3DA.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DYi%26aufirst%3DJ.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DWienholds%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPineda%26aufirst%3DJ.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DCatalytic%2520site%2520remodelling%2520of%2520the%2520DOT1L%2520methyltransferase%2520by%2520selective%2520inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D1288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lgKsrHYQBIJcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingerman, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Drug discovery toward antagonists of methyl-lysine binding proteins</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.2174%2F1875397301005010051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=22145013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=51-61&author=J.+M.+Heroldauthor=L.+A.+Ingermanauthor=C.+Gaoauthor=S.+V.+Frye&title=Drug+discovery+toward+antagonists+of+methyl-lysine+binding+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery toward antagonists of methyl-lysine binding proteins</span></div><div class="casAuthors">Herold, J. Martin; Ingerman, Lindsey A.; Gao, Cen; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The recognition of methyl-lysine and -arginine residues on both histone and other proteins by specific "reader" elements is important for chromatin regulation, gene expression, and control of cell-cycle progression.  Recently the crucial role of these reader proteins in cancer development and dedifferentiation has emerged, owing to the increased interest among the scientific community.  The methyl-lysine and -arginine readers are a large and very diverse set of effector proteins and targeting them with small mol. probes in drug discovery will inevitably require a detailed understanding of their structural biol. and mechanism of binding.  In the following review, the crit. elements of methyl-lysine and -arginine recognition will be summarized with respect to each protein family and initial results in assay development, probe design, and drug discovery will be highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-fC8z74GQSLVg90H21EOLACvtfcHk0lgKsrHYQBIJcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nN&md5=845f1233f9ea44c2706fc7737ca34c08</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010051%26sid%3Dliteratum%253Aachs%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DIngerman%26aufirst%3DL.%2BA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDrug%2520discovery%2520toward%2520antagonists%2520of%2520methyl-lysine%2520binding%2520proteins%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D51%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Targeting chromatin readers</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">314</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fclpt.2013.6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23403847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVKrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2013&pages=312-314&author=L.+I.+Jamesauthor=S.+V.+Frye&title=Targeting+chromatin+readers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Chromatin Readers</span></div><div class="casAuthors">James, L. I.; Frye, S. V.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-314</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Modulation of gene expression through epigenetic signaling has recently emerged as a novel approach in treating human disease.  Specifically, chromatin reader proteins, which mediate protein-protein interactions via binding to modified lysine residues, are gaining traction as potential therapeutic targets.  Herein, we review recent efforts to understand and modulate the activity of chromatin reader proteins with small-mol. ligands.  Clin. Pharmacol. & Therapeutics (2013); 93 4, 312-314. doi:10.1038/clpt.2013.6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAp554d8hcQLVg90H21EOLACvtfcHk0lhrx8ptnU93FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVKrsbg%253D&md5=4ec2bd0426dba17a52ad7174dc534351</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2013.6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2013.6%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DTargeting%2520chromatin%2520readers%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2013%26volume%3D93%26spage%3D312%26epage%3D314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krichevsky, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macnevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sagum, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emili, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">184</span><span class="NLM_x">–</span> <span class="NLM_lpage">191</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.1157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23292653" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=184-191&author=L.+I.+Jamesauthor=D.+Barsyte-Lovejoyauthor=N.+Zhongauthor=L.+Krichevskyauthor=V.+K.+Korboukhauthor=J.+M.+Heroldauthor=C.+J.+Macnevinauthor=J.+L.+Norrisauthor=C.+A.+Sagumauthor=W.+Tempelauthor=E.+Marconauthor=H.+Guoauthor=C.+Gaoauthor=X.+P.+Huangauthor=S.+Duanauthor=A.+Emiliauthor=J.+F.+Greenblattauthor=D.+B.+Kireevauthor=J.+Jinauthor=W.+P.+Janzenauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=C.+H.+Arrowsmithauthor=S.+V.+Frye&title=Discovery+of+a+chemical+probe+for+the+L3MBTL3+methyllysine+reader+domain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span></div><div class="casAuthors">James, Lindsey I.; Barsyte-Lovejoy, Dalia; Zhong, Nan; Krichevsky, Liubov; Korboukh, Victoria K.; Herold, J. Martin; MacNevin, Christopher J.; Norris, Jacqueline L.; Sagum, Cari A.; Tempel, Wolfram; Marcon, Edyta; Guo, Hongbo; Gao, Cen; Huang, Xi-Ping; Duan, Shili; Emili, Andrew; Greenblatt, Jack F.; Kireev, Dmitri B.; Jin, Jian; Janzen, William P.; Brown, Peter J.; Bedford, Mark T.; Arrowsmith, Cheryl H.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">184-191</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe the discovery of UNC1215, a potent and selective chem. probe for the methyllysine (Kme) reading function of L3MBTL3, a member of the malignant brain tumor (MBT) family of chromatin-interacting transcriptional repressors.  UNC1215 binds L3MBTL3 with a Kd of 120 nM, competitively displacing mono- or dimethyllysine-contg. peptides, and is greater than 50-fold more potent toward L3MBTL3 than other members of the MBT family while also demonstrating selectivity against more than 200 other reader domains examd.  X-ray crystallog. identified a unique 2:2 polyvalent mode of interaction between UNC1215 and L3MBTL3.  In cells, UNC1215 is nontoxic and directly binds L3MBTL3 via the Kme-binding pocket of the MBT domains.  UNC1215 increases the cellular mobility of GFP-L3MBTL3 fusion proteins, and point mutants that disrupt the Kme-binding function of GFP-L3MBTL3 phenocopy the effects of UNC1215 on localization.  Finally, UNC1215 was used to reveal a new Kme-dependent interaction of L3MBTL3 with BCLAF1 (BCL2-assocd. transcription factor 1), a protein implicated in DNA damage repair and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqta8WQQkm04rVg90H21EOLACvtfcHk0lhrx8ptnU93FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D&md5=ea8779ddceed07b5e4c822f2eb888acb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1157%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DN.%26aulast%3DKrichevsky%26aufirst%3DL.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DMacnevin%26aufirst%3DC.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DSagum%26aufirst%3DC.%2BA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DMarcon%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DEmili%26aufirst%3DA.%26aulast%3DGreenblatt%26aufirst%3DJ.%2BF.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDiscovery%2520of%2520a%2520chemical%2520probe%2520for%2520the%2520L3MBTL3%2520methyllysine%2520reader%2520domain%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D184%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Bonasio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecona, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">MBT domain proteins in development and disease</span> <span class="citation_source-journal">Semin. Cell Dev. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">221</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=221-230&author=R.+Bonasioauthor=E.+Leconaauthor=D.+Reinberg&title=MBT+domain+proteins+in+development+and+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBonasio%26aufirst%3DR.%26aulast%3DLecona%26aufirst%3DE.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DMBT%2520domain%2520proteins%2520in%2520development%2520and%2520disease%26jtitle%3DSemin.%2520Cell%2520Dev.%2520Biol.%26date%3D2010%26volume%3D21%26spage%3D221%26epage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaquero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalakonda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boccuni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimer, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">L3MBTL1, a histone-methylation-dependent chromatin lock</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">928</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=915-928&author=P.+Trojerauthor=G.+Liauthor=R.+J.+Simsauthor=A.+Vaqueroauthor=N.+Kalakondaauthor=P.+Boccuniauthor=D.+Leeauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+D.+Nimerauthor=Y.+H.+Wangauthor=D.+Reinberg&title=L3MBTL1%2C+a+histone-methylation-dependent+chromatin+lock"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DVaquero%26aufirst%3DA.%26aulast%3DKalakonda%26aufirst%3DN.%26aulast%3DBoccuni%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DL3MBTL1%252C%2520a%2520histone-methylation-dependent%2520chromatin%2520lock%26jtitle%3DCell%26date%3D2007%26volume%3D129%26spage%3D915%26epage%3D928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Arai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyazaki, T.</span><span> </span><span class="NLM_article-title">Impaired maturation of myeloid progenitors in mice lacking novel polycomb group protein MBT-1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1863</span><span class="NLM_x">–</span> <span class="NLM_lpage">1873</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=1863-1873&author=S.+Araiauthor=T.+Miyazaki&title=Impaired+maturation+of+myeloid+progenitors+in+mice+lacking+novel+polycomb+group+protein+MBT-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DArai%26aufirst%3DS.%26aulast%3DMiyazaki%26aufirst%3DT.%26atitle%3DImpaired%2520maturation%2520of%2520myeloid%2520progenitors%2520in%2520mice%2520lacking%2520novel%2520polycomb%2520group%2520protein%2520MBT-1%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D1863%26epage%3D1873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Botuyan, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mer, G.</span><span> </span><span class="NLM_article-title">Structural basis for the methylation state-specific recognition of histone H4-K20 by 53BP1 and Crb2 in DNA repair</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">1361</span><span class="NLM_x">–</span> <span class="NLM_lpage">1373</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2006&pages=1361-1373&author=M.+V.+Botuyanauthor=J.+Leeauthor=I.+M.+Wardauthor=J.+E.+Kimauthor=J.+R.+Thompsonauthor=J.+Chenauthor=G.+Mer&title=Structural+basis+for+the+methylation+state-specific+recognition+of+histone+H4-K20+by+53BP1+and+Crb2+in+DNA+repair"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBotuyan%26aufirst%3DM.%2BV.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DWard%26aufirst%3DI.%2BM.%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DThompson%26aufirst%3DJ.%2BR.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMer%26aufirst%3DG.%26atitle%3DStructural%2520basis%2520for%2520the%2520methylation%2520state-specific%2520recognition%2520of%2520histone%2520H4-K20%2520by%252053BP1%2520and%2520Crb2%2520in%2520DNA%2520repair%26jtitle%3DCell%26date%3D2006%26volume%3D127%26spage%3D1361%26epage%3D1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Roy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musselman, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kachirskaia, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayashi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glass, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nix, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appella, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutateladze, T. G.</span><span> </span><span class="NLM_article-title">Structural insight into p53 recognition by the 53BP1 tandem tudor domain</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">398</span><span class="NLM_x">, </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=398&publication_year=2010&pages=489-496&author=S.+Royauthor=C.+A.+Musselmanauthor=I.+Kachirskaiaauthor=R.+Hayashiauthor=K.+C.+Glassauthor=J.+C.+Nixauthor=O.+Gozaniauthor=E.+Appellaauthor=T.+G.+Kutateladze&title=Structural+insight+into+p53+recognition+by+the+53BP1+tandem+tudor+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DKachirskaia%26aufirst%3DI.%26aulast%3DHayashi%26aufirst%3DR.%26aulast%3DGlass%26aufirst%3DK.%2BC.%26aulast%3DNix%26aufirst%3DJ.%2BC.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DStructural%2520insight%2520into%2520p53%2520recognition%2520by%2520the%252053BP1%2520tandem%2520tudor%2520domain%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D398%26spage%3D489%26epage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coil, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bua, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Structure–activity relationships of methyl-lysine reader antagonists</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">45</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=45-51&author=J.+M.+Heroldauthor=L.+I.+Jamesauthor=V.+K.+Korboukhauthor=C.+Gaoauthor=K.+E.+Coilauthor=D.+J.+Buaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=P.+J.+Brownauthor=J.+Jinauthor=W.+P.+Janzenauthor=O.+Gozaniauthor=S.+V.+Frye&title=Structure%E2%80%93activity+relationships+of+methyl-lysine+reader+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DCoil%26aufirst%3DK.%2BE.%26aulast%3DBua%26aufirst%3DD.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DStructure%25E2%2580%2593activity%2520relationships%2520of%2520methyl-lysine%2520reader%2520antagonists%26jtitle%3DMedChemComm%26date%3D2012%26volume%3D3%26spage%3D45%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingerman, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Small-molecule ligands of methyl-lysine binding proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2504</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200045v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2504-2511&author=J.+M.+Heroldauthor=T.+J.+Wigleauthor=J.+L.+Norrisauthor=R.+Lamauthor=V.+K.+Korboukhauthor=C.+Gaoauthor=L.+A.+Ingermanauthor=D.+B.+Kireevauthor=G.+Senisterraauthor=M.+Vedadiauthor=A.+Tripathyauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+Jinauthor=W.+P.+Janzenauthor=S.+V.+Frye&title=Small-molecule+ligands+of+methyl-lysine+binding+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm200045v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200045v%26sid%3Dliteratum%253Aachs%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DLam%26aufirst%3DR.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DIngerman%26aufirst%3DL.%2BA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DSmall-molecule%2520ligands%2520of%2520methyl-lysine%2520binding%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2504%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span> </span><span class="NLM_article-title">Screening for inhibitors of low-affinity epigenetic peptide-protein interactions: an AlphaScreen-based assay for antagonists of methyl-lysine binding proteins</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">62</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1177%2F1087057109352902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=20008125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFWmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=62-71&author=T.+J.+Wigleauthor=J.+M.+Heroldauthor=G.+A.+Senisterraauthor=M.+Vedadiauthor=D.+B.+Kireevauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=W.+P.+Janzen&title=Screening+for+inhibitors+of+low-affinity+epigenetic+peptide-protein+interactions%3A+an+AlphaScreen-based+assay+for+antagonists+of+methyl-lysine+binding+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Screening for inhibitors of low-affinity epigenetic peptide-protein interactions: an AlphaScreen-based assay for antagonists of methyl-lysine binding proteins</span></div><div class="casAuthors">Wigle, Tim J.; Herold, J. Martin; Senisterra, Guillermo A.; Vedadi, Masoud; Kireev, Dmitri B.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Janzen, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-71</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone code comprises many posttranslational modifications that occur mainly in histone tail peptides.  The identity and location of these marks are read by a variety of histone-binding proteins that are emerging as important regulators of cellular differentiation and development and are increasingly being implicated in numerous disease states.  The authors describe the development of the first high-throughput screening assay for the discovery of inhibitors of methyl-lysine binding proteins that will be used to initiate a full-scale discovery effort for this broad target class.  They focus on the development of an AlphaScreen-based assay for malignant brain tumor (MBT) domain-contg. proteins, which bind to the lower methylation states of lysine residues present in histone tail peptides.  This assay takes advantage of the avidity of the AlphaScreen beads to clear the hurdle to assay development presented by the low micromolar binding consts. of the histone binding proteins for their cognate peptides.  The assay is applicable to other families of methyl-lysine binding proteins, and it has the potential to be used in screening efforts toward the discovery of novel small mols. with utility as research tools for cellular reprogramming and ultimately drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKUKuGjBHvdLVg90H21EOLACvtfcHk0lj5V_bdXkcERg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFWmu7s%253D&md5=ff7396785ae549d1e0d2e7197d6c34e7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1177%2F1087057109352902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109352902%26sid%3Dliteratum%253Aachs%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26atitle%3DScreening%2520for%2520inhibitors%2520of%2520low-affinity%2520epigenetic%2520peptide-protein%2520interactions%253A%2520an%2520AlphaScreen-based%2520assay%2520for%2520antagonists%2520of%2520methyl-lysine%2520binding%2520proteins%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2010%26volume%3D15%26spage%3D62%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormann, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">459</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">851</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnature08036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=19430464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2009&pages=847-851&author=G.+G.+Wangauthor=J.+Songauthor=Z.+Wangauthor=H.+L.+Dormannauthor=F.+Casadioauthor=H.+Liauthor=J.+L.+Luoauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Haematopoietic+malignancies+caused+by+dysregulation+of+a+chromatin-binding+PHD+finger"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span></div><div class="casAuthors">Wang, Gang G.; Song, Jikui; Wang, Zhanxin; Dormann, Holger L.; Casadio, Fabio; Li, Haitao; Luo, Jun-Li; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">459</span>
        (<span class="NLM_cas:issue">7248</span>),
    <span class="NLM_cas:pages">847-851</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 lysine 4 methylation (H3K4me) has been proposed as a crit. component in regulating gene expression, epigenetic states, and cellular identities.  The biol. meaning of H3K4me is interpreted by conserved modules including plant homeodomain (PHD) fingers that recognize varied H3K4me states.  The dysregulation of PHD fingers has been implicated in several human diseases, including cancers and immune or neurol. disorders.  Here we report that fusing an H3K4-trimethylation (H3K4me3)-binding PHD finger, such as the carboxy-terminal PHD finger of PHF23 or JARID1A (also known as KDM5A or RBBP2), to a common fusion partner nucleoporin-98 (NUP98) as identified in human leukemias, generated potent oncoproteins that arrested hematopoietic differentiation and induced acute myeloid leukemia in murine models.  In these processes, a PHD finger that specifically recognizes H3K4me3/2 marks was essential for leukemogenesis.  Mutations in PHD fingers that abrogated H3K4me3 binding also abolished leukemic transformation.  NUP98-PHD fusion prevented the differentiation-assocd. removal of H3K4me3 at many loci encoding lineage-specific transcription factors (Hox(s), Gata3, Meis1, Eya1 and Pbx1), and enforced their active gene transcription in murine hematopoietic stem/progenitor cells.  Mechanistically, NUP98-PHD fusions act as 'chromatin boundary factors', dominating over polycomb-mediated gene silencing to 'lock' developmentally crit. loci into an active chromatin state (H3K4me3 with induced histone acetylation), a state that defined leukemia stem cells.  Collectively, our studies represent, to our knowledge, the first report that deregulation of the PHD finger, an 'effector' of specific histone modification, perturbs the epigenetic dynamics on developmentally crit. loci, catastrophizes cellular fate decision-making, and even causes oncogenesis during mammalian development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR1S9VoRYMRrVg90H21EOLACvtfcHk0ljEuKDqF2Wa_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D&md5=3a647643c8a73639fcb332a774f1155e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature08036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08036%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DDormann%26aufirst%3DH.%2BL.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DJ.%2BL.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DHaematopoietic%2520malignancies%2520caused%2520by%2520dysregulation%2520of%2520a%2520chromatin-binding%2520PHD%2520finger%26jtitle%3DNature%26date%3D2009%26volume%3D459%26spage%3D847%26epage%3D851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 55 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jian-Yuan Li, Gabriella Miklossy, Ram K. Modukuri, Kurt M. Bohren, Zhifeng Yu, Murugesan Palaniappan, John C. Faver, Kevin Riehle, Martin M. Matzuk, <span class="NLM_string-name hlFld-ContribAuthor">Nicholas Simmons</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium-Catalyzed Hydroxycarbonylation of (Hetero)aryl Halides for DNA-Encoded Chemical Library Synthesis. </span><span class="cited-content_cbyCitation_journal-name">Bioconjugate Chemistry</span><span> <strong>2019,</strong> <em>30 </em>
                                    (8)
                                     , 2209-2215. <a href="https://doi.org/10.1021/acs.bioconjchem.9b00447" title="DOI URL">https://doi.org/10.1021/acs.bioconjchem.9b00447</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.bioconjchem.9b00447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.bioconjchem.9b00447%26sid%3Dliteratum%253Aachs%26jtitle%3DBioconjugate%2520Chemistry%26atitle%3DPalladium-Catalyzed%252BHydroxycarbonylation%252Bof%252B%252528Hetero%252529aryl%252BHalides%252Bfor%252BDNA-Encoded%252BChemical%252BLibrary%252BSynthesis%26aulast%3DLi%26aufirst%3DJian-Yuan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D25062019%26date%3D17072019%26date%3D31072019%26date%3D22072019%26volume%3D30%26issue%3D8%26spage%3D2209%26epage%3D2215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joydev K.  Laha</span>, <span class="hlFld-ContribAuthor ">Ketul V.  Patel</span>, and <span class="hlFld-ContribAuthor ">Sheetal  Sharma</span>  . </span><span class="cited-content_cbyCitation_article-title">Palladium-Catalyzed Decarboxylative Ortho-Acylation of Tertiary Benzamides with Arylglyoxylic Acids. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (7)
                                     , 3806-3815. <a href="https://doi.org/10.1021/acsomega.7b00717" title="DOI URL">https://doi.org/10.1021/acsomega.7b00717</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00717%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DPalladium-Catalyzed%252BDecarboxylative%252BOrtho-Acylation%252Bof%252BTertiary%252BBenzamides%252Bwith%252BArylglyoxylic%252BAcids%26aulast%3DLaha%26aufirst%3DJoydev%2BK.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D02062017%26date%3D11072017%26date%3D31072017%26volume%3D2%26issue%3D7%26spage%3D3806%26epage%3D3815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacob I.  Stuckey</span>, <span class="hlFld-ContribAuthor ">Catherine  Simpson</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris-Drouin</span>, <span class="hlFld-ContribAuthor ">Stephanie H.  Cholensky</span>, <span class="hlFld-ContribAuthor ">Junghyun  Lee</span>, <span class="hlFld-ContribAuthor ">Ryan  Pasca</span>, <span class="hlFld-ContribAuthor ">Nancy  Cheng</span>, <span class="hlFld-ContribAuthor ">Bradley M.  Dickson</span>, <span class="hlFld-ContribAuthor ">Kenneth H.  Pearce</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, and <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationships and Kinetic Studies of Peptidic Antagonists of CBX Chromodomains. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (19)
                                     , 8913-8923. <a href="https://doi.org/10.1021/acs.jmedchem.6b00801" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationships%252Band%252BKinetic%252BStudies%252Bof%252BPeptidic%252BAntagonists%252Bof%252BCBX%252BChromodomains%26aulast%3DStuckey%26aufirst%3DJacob%2BI.%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D27052016%26date%3D19092016%26date%3D13102016%26date%3D29082016%26volume%3D59%26issue%3D19%26spage%3D8913%26epage%3D8923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pradeep  Nareddy</span>, <span class="hlFld-ContribAuthor ">Frank  Jordan</span>, <span class="hlFld-ContribAuthor ">Stacey E.  Brenner-Moyer</span>, and <span class="hlFld-ContribAuthor ">Michal  Szostak</span>  . </span><span class="cited-content_cbyCitation_article-title">Ruthenium(II)-Catalyzed Regioselective C–H Arylation of Cyclic and N,N-Dialkyl Benzamides with Boronic Acids by Weak Coordination. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2016,</strong> <em>6 </em>
                                    (7)
                                     , 4755-4759. <a href="https://doi.org/10.1021/acscatal.6b01360" title="DOI URL">https://doi.org/10.1021/acscatal.6b01360</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.6b01360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.6b01360%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DRuthenium%252528II%252529-Catalyzed%252BRegioselective%252BC%2525E2%252580%252593H%252BArylation%252Bof%252BCyclic%252Band%252BN%25252CN-Dialkyl%252BBenzamides%252Bwith%252BBoronic%252BAcids%252Bby%252BWeak%252BCoordination%26aulast%3DNareddy%26aufirst%3DPradeep%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D15052016%26date%3D11062016%26date%3D22062016%26date%3D01072016%26volume%3D6%26issue%3D7%26spage%3D4755%26epage%3D4759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia  Milosevich</span> and <span class="hlFld-ContribAuthor ">Fraser  Hof</span>  . </span><span class="cited-content_cbyCitation_article-title">Chemical Inhibitors of Epigenetic Methyllysine Reader Proteins. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2016,</strong> <em>55 </em>
                                    (11)
                                     , 1570-1583. <a href="https://doi.org/10.1021/acs.biochem.5b01073" title="DOI URL">https://doi.org/10.1021/acs.biochem.5b01073</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.5b01073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.5b01073%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DChemical%252BInhibitors%252Bof%252BEpigenetic%252BMethyllysine%252BReader%252BProteins%26aulast%3DMilosevich%26aufirst%3DNatalia%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D01102015%26date%3D07122015%26date%3D28122015%26date%3D22032016%26date%3D09122015%26volume%3D55%26issue%3D11%26spage%3D1570%26epage%3D1583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandi M.  Baughman</span>, <span class="hlFld-ContribAuthor ">Samantha G.  Pattenden</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, and <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>  . </span><span class="cited-content_cbyCitation_article-title">The L3MBTL3 Methyl-Lysine Reader Domain Functions As a Dimer. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2016,</strong> <em>11 </em>
                                    (3)
                                     , 722-728. <a href="https://doi.org/10.1021/acschembio.5b00632" title="DOI URL">https://doi.org/10.1021/acschembio.5b00632</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.5b00632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.5b00632%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BL3MBTL3%252BMethyl-Lysine%252BReader%252BDomain%252BFunctions%252BAs%252Ba%252BDimer%26aulast%3DBaughman%26aufirst%3DBrandi%2BM.%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D2017%26date%3D19122017%26date%3D22082014%26date%3D28082015%26date%3D02092015%26date%3D18032016%26date%3D28082015%26volume%3D11%26issue%3D3%26spage%3D722%26epage%3D728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vineet  Pande</span>  . </span><span class="cited-content_cbyCitation_article-title">Understanding the Complexity of Epigenetic Target Space. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1299-1307. <a href="https://doi.org/10.1021/acs.jmedchem.5b01507" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01507</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01507%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DUnderstanding%252Bthe%252BComplexity%252Bof%252BEpigenetic%252BTarget%252BSpace%26aulast%3DPande%26aufirst%3DVineet%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D29092015%26date%3D28012016%26date%3D25022016%26date%3D21012016%26volume%3D59%26issue%3D4%26spage%3D1299%26epage%3D1307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia  Milosevich</span>, <span class="hlFld-ContribAuthor ">Michael C.  Gignac</span>, <span class="hlFld-ContribAuthor ">James  McFarlane</span>, <span class="hlFld-ContribAuthor ">Chakravarthi  Simhadri</span>, <span class="hlFld-ContribAuthor ">Shanti  Horvath</span>, <span class="hlFld-ContribAuthor ">Kevin D.  Daze</span>, <span class="hlFld-ContribAuthor ">Caitlin S.  Croft</span>, <span class="hlFld-ContribAuthor ">Aman  Dheri</span>, <span class="hlFld-ContribAuthor ">Taylor T. H.  Quon</span>, <span class="hlFld-ContribAuthor ">Sarah F.  Douglas</span>, <span class="hlFld-ContribAuthor ">Jeremy E.  Wulff</span>, <span class="hlFld-ContribAuthor ">Irina  Paci</span>, and <span class="hlFld-ContribAuthor ">Fraser  Hof</span>  . </span><span class="cited-content_cbyCitation_article-title">Selective Inhibition of CBX6: A Methyllysine Reader Protein in the Polycomb Family. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>7 </em>
                                    (2)
                                     , 139-144. <a href="https://doi.org/10.1021/acsmedchemlett.5b00378" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.5b00378</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.5b00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.5b00378%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSelective%252BInhibition%252Bof%252BCBX6%25253A%252BA%252BMethyllysine%252BReader%252BProtein%252Bin%252Bthe%252BPolycomb%252BFamily%26aulast%3DMilosevich%26aufirst%3DNatalia%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D29092015%26date%3D07122015%26date%3D21122015%26date%3D11022016%26date%3D07122015%26volume%3D7%26issue%3D2%26spage%3D139%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael T.  Perfetti</span>, <span class="hlFld-ContribAuthor ">Brandi M.  Baughman</span>, <span class="hlFld-ContribAuthor ">Bradley M.  Dickson</span>, <span class="hlFld-ContribAuthor ">Yunxiang  Mu</span>, <span class="hlFld-ContribAuthor ">Gaofeng  Cui</span>, <span class="hlFld-ContribAuthor ">Pavel  Mader</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris</span>, <span class="hlFld-ContribAuthor ">Scott B.  Rothbart</span>, <span class="hlFld-ContribAuthor ">Brian D.  Strahl</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">William P.  Janzen</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Georges  Mer</span>, <span class="hlFld-ContribAuthor ">Kevin M.  McBride</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, and <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of a Fragment-like Small Molecule Ligand for the Methyl-lysine Binding Protein, 53BP1. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2015,</strong> <em>10 </em>
                                    (4)
                                     , 1072-1081. <a href="https://doi.org/10.1021/cb500956g" title="DOI URL">https://doi.org/10.1021/cb500956g</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb500956g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb500956g%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DIdentification%252Bof%252Ba%252BFragment-like%252BSmall%252BMolecule%252BLigand%252Bfor%252Bthe%252BMethyl-lysine%252BBinding%252BProtein%25252C%252B53BP1%26aulast%3DPerfetti%26aufirst%3DMichael%2BT.%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D24112014%26date%3D15012015%26date%3D28012015%26date%3D17042015%26date%3D15012015%26volume%3D10%26issue%3D4%26spage%3D1072%26epage%3D1081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robyn J.  Eisert</span>, <span class="hlFld-ContribAuthor ">Sarah A.  Kennedy</span>, and <span class="hlFld-ContribAuthor ">Marcey L.  Waters</span>  . </span><span class="cited-content_cbyCitation_article-title">Investigation of the β-Sheet Interactions between dHP1 Chromodomain and Histone 3. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2015,</strong> <em>54 </em>
                                    (14)
                                     , 2314-2322. <a href="https://doi.org/10.1021/acs.biochem.5b00024" title="DOI URL">https://doi.org/10.1021/acs.biochem.5b00024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.5b00024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.5b00024%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DInvestigation%252Bof%252Bthe%252B%2525CE%2525B2-Sheet%252BInteractions%252Bbetween%252BdHP1%252BChromodomain%252Band%252BHistone%252B3%26aulast%3DEisert%26aufirst%3DRobyn%2BJ.%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D10012015%26date%3D17032015%26date%3D01042015%26date%3D14042015%26date%3D19032015%26volume%3D54%26issue%3D14%26spage%3D2314%26epage%3D2322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chakravarthi  Simhadri</span>, <span class="hlFld-ContribAuthor ">Kevin D.  Daze</span>, <span class="hlFld-ContribAuthor ">Sarah F.  Douglas</span>, <span class="hlFld-ContribAuthor ">Taylor T. H.  Quon</span>, <span class="hlFld-ContribAuthor ">Amarjot  Dev</span>, <span class="hlFld-ContribAuthor ">Michael C.  Gignac</span>, <span class="hlFld-ContribAuthor ">Fangni  Peng</span>, <span class="hlFld-ContribAuthor ">Markus  Heller</span>, <span class="hlFld-ContribAuthor ">Martin J.  Boulanger</span>, <span class="hlFld-ContribAuthor ">Jeremy E.  Wulff</span>, and <span class="hlFld-ContribAuthor ">Fraser  Hof</span>  . </span><span class="cited-content_cbyCitation_article-title">Chromodomain Antagonists That Target the Polycomb-Group Methyllysine Reader Protein Chromobox Homolog 7 (CBX7). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (7)
                                     , 2874-2883. <a href="https://doi.org/10.1021/jm401487x" title="DOI URL">https://doi.org/10.1021/jm401487x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm401487x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm401487x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DChromodomain%252BAntagonists%252BThat%252BTarget%252Bthe%252BPolycomb-Group%252BMethyllysine%252BReader%252BProtein%252BChromobox%252BHomolog%252B7%252B%252528CBX7%252529%26aulast%3DSimhadri%26aufirst%3DChakravarthi%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D25092013%26date%3D26032014%26date%3D10042014%26date%3D13032014%26volume%3D57%26issue%3D7%26spage%3D2874%26epage%3D2883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ammar D.  Elmezayen</span>, <span class="hlFld-ContribAuthor ">Yelekçi  Kemal</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based virtual screening for novel potential selective inhibitors of class IIa histone deacetylases for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2021,</strong> <em>31 </em>, 107491. <a href="https://doi.org/10.1016/j.compbiolchem.2021.107491" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2021.107491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2021.107491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2021.107491%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DStructure-based%252Bvirtual%252Bscreening%252Bfor%252Bnovel%252Bpotential%252Bselective%252Binhibitors%252Bof%252Bclass%252BIIa%252Bhistone%252Bdeacetylases%252Bfor%252Bcancer%252Btreatment%26aulast%3DElmezayen%26aufirst%3DAmmar%2BD.%26date%3D2021%26volume%3D31%26spage%3D107491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Soňa  Krajčovičová</span>, <span class="hlFld-ContribAuthor ">Jan  Hlaváč</span>, <span class="hlFld-ContribAuthor ">Kristýna  Vychodilová</span>. </span><span class="cited-content_cbyCitation_article-title">Polymer-supported synthesis of
              N
              -substituted anthranilates as the building blocks for preparation of
              N
              -arylated 3-hydroxyquinolin-4(1
              H
              )-ones. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (16)
                                     , 9362-9365. <a href="https://doi.org/10.1039/D1RA01308D" title="DOI URL">https://doi.org/10.1039/D1RA01308D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1RA01308D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1RA01308D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DPolymer-supported%252Bsynthesis%252Bof%252BN%252B-substituted%252Banthranilates%252Bas%252Bthe%252Bbuilding%252Bblocks%252Bfor%252Bpreparation%252Bof%252BN%252B-arylated%252B3-hydroxyquinolin-4%2525281%252BH%252B%252529-ones%26aulast%3DKraj%25C4%258Dovi%25C4%258Dov%25C3%25A1%26aufirst%3DSo%25C5%2588a%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D16%26spage%3D9362%26epage%3D9365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Il-Geun  Park</span>, <span class="hlFld-ContribAuthor ">Minsol  Jeon</span>, <span class="hlFld-ContribAuthor ">Hyunkyung  Kim</span>, <span class="hlFld-ContribAuthor ">Ji Min  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Coordinated methyl readers: Functional communications in cancer. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2021,</strong> <em>5 </em><a href="https://doi.org/10.1016/j.semcancer.2021.03.015" title="DOI URL">https://doi.org/10.1016/j.semcancer.2021.03.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2021.03.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2021.03.015%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DCoordinated%252Bmethyl%252Breaders%25253A%252BFunctional%252Bcommunications%252Bin%252Bcancer%26aulast%3DPark%26aufirst%3DIl-Geun%26date%3D2021%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack‐Sjodin</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Proteins as Emerging Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 423-448. <a href="https://doi.org/10.1002/9781118681121.ch18" title="DOI URL">https://doi.org/10.1002/9781118681121.ch18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch18%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BProteins%252Bas%252BEmerging%252BDrug%252BTargets%26aulast%3DBoriack%25E2%2580%2590Sjodin%26aufirst%3DP.%2BAnn%26date%3D2020%26date%3D2020%26spage%3D423%26epage%3D448%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomohiro  Kozako</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Shin-ichiro  Honda</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Control Using Small Molecules in Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 111-148. <a href="https://doi.org/10.1007/978-3-030-32857-3_6" title="DOI URL">https://doi.org/10.1007/978-3-030-32857-3_6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-32857-3_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-32857-3_6%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BControl%252BUsing%252BSmall%252BMolecules%252Bin%252BCancer%26aulast%3DKozako%26aufirst%3DTomohiro%26date%3D2020%26date%3D2020%26spage%3D111%26epage%3D148%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DApproaching%252BComplex%252BDiseases%26aulast%3DBizzarri%26aufirst%3DMariano%26date%3D2020%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kun  Jing</span>, <span class="hlFld-ContribAuthor ">Peng-Cheng  Cui</span>, <span class="hlFld-ContribAuthor ">Guan-Wu  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium-catalyzed decarboxylative
              ortho
              -amidation of
              O
              -methyl ketoximes with oxamic acids. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (83)
                                     , 12551-12554. <a href="https://doi.org/10.1039/C9CC06460E" title="DOI URL">https://doi.org/10.1039/C9CC06460E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CC06460E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CC06460E%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DPalladium-catalyzed%252Bdecarboxylative%252Bortho%252B-amidation%252Bof%252BO%252B-methyl%252Bketoximes%252Bwith%252Boxamic%252Bacids%26aulast%3DJing%26aufirst%3DKun%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D83%26spage%3D12551%26epage%3D12554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anshika N.  Singh</span>, <span class="hlFld-ContribAuthor ">Neeti  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Modulators as Potential Multi-targeted Drugs Against Hedgehog Pathway for Treatment of Cancer. </span><span class="cited-content_cbyCitation_journal-name">The Protein Journal</span><span> <strong>2019,</strong> <em>38 </em>
                                    (5)
                                     , 537-550. <a href="https://doi.org/10.1007/s10930-019-09832-9" title="DOI URL">https://doi.org/10.1007/s10930-019-09832-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10930-019-09832-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10930-019-09832-9%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Protein%2520Journal%26atitle%3DEpigenetic%252BModulators%252Bas%252BPotential%252BMulti-targeted%252BDrugs%252BAgainst%252BHedgehog%252BPathway%252Bfor%252BTreatment%252Bof%252BCancer%26aulast%3DSingh%26aufirst%3DAnshika%2BN.%26date%3D2019%26date%3D2019%26volume%3D38%26issue%3D5%26spage%3D537%26epage%3D550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chakravarthi  Simhadri</span>, <span class="hlFld-ContribAuthor ">Kevin D.  Daze</span>, <span class="hlFld-ContribAuthor ">Sarah F.  Douglas</span>, <span class="hlFld-ContribAuthor ">Natalia  Milosevich</span>, <span class="hlFld-ContribAuthor ">Leticia  Monjas</span>, <span class="hlFld-ContribAuthor ">Amarjot  Dev</span>, <span class="hlFld-ContribAuthor ">Tyler M.  Brown</span>, <span class="hlFld-ContribAuthor ">Anna K. H.  Hirsch</span>, <span class="hlFld-ContribAuthor ">Jeremy E.  Wulff</span>, <span class="hlFld-ContribAuthor ">Fraser  Hof</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Adaptation of L3MBTL1 Inhibitors to Create Small‐Molecule Cbx7 Antagonists. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (15)
                                     , 1444-1456. <a href="https://doi.org/10.1002/cmdc.201900021" title="DOI URL">https://doi.org/10.1002/cmdc.201900021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900021%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DRational%252BAdaptation%252Bof%252BL3MBTL1%252BInhibitors%252Bto%252BCreate%252BSmall%2525E2%252580%252590Molecule%252BCbx7%252BAntagonists%26aulast%3DSimhadri%26aufirst%3DChakravarthi%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D15%26spage%3D1444%26epage%3D1456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fatima  Arshad</span>, <span class="hlFld-ContribAuthor ">Mohemmed Faraz  Khan</span>, <span class="hlFld-ContribAuthor ">Wasim  Akhtar</span>, <span class="hlFld-ContribAuthor ">Mohammad Mumtaz  Alam</span>, <span class="hlFld-ContribAuthor ">Lalit Mohan  Nainwal</span>, <span class="hlFld-ContribAuthor ">Sumit Kumar  Kaushik</span>, <span class="hlFld-ContribAuthor ">Mymoona  Akhter</span>, <span class="hlFld-ContribAuthor ">Suhel  Parvez</span>, <span class="hlFld-ContribAuthor ">Syed Misbahul  Hasan</span>, <span class="hlFld-ContribAuthor ">Mohammad  Shaquiquzzaman</span>. </span><span class="cited-content_cbyCitation_article-title">Revealing quinquennial anticancer journey of morpholine: A SAR based review. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 324-356. <a href="https://doi.org/10.1016/j.ejmech.2019.02.015" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.015%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRevealing%252Bquinquennial%252Banticancer%252Bjourney%252Bof%252Bmorpholine%25253A%252BA%252BSAR%252Bbased%252Breview%26aulast%3DArshad%26aufirst%3DFatima%26date%3D2019%26volume%3D167%26spage%3D324%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Helai P.  Mohammad</span>, <span class="hlFld-ContribAuthor ">Olena  Barbash</span>, <span class="hlFld-ContribAuthor ">Caretha L.  Creasy</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic modifications in cancer therapy: erasing the roadmap to cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Medicine</span><span> <strong>2019,</strong> <em>25 </em>
                                    (3)
                                     , 403-418. <a href="https://doi.org/10.1038/s41591-019-0376-8" title="DOI URL">https://doi.org/10.1038/s41591-019-0376-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41591-019-0376-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41591-019-0376-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Medicine%26atitle%3DTargeting%252Bepigenetic%252Bmodifications%252Bin%252Bcancer%252Btherapy%25253A%252Berasing%252Bthe%252Broadmap%252Bto%252Bcancer%26aulast%3DMohammad%26aufirst%3DHelai%2BP.%26date%3D2019%26date%3D2019%26volume%3D25%26issue%3D3%26spage%3D403%26epage%3D418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Johannes  Bacher</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Reader Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 383-419. <a href="https://doi.org/10.1002/9783527809257.ch14" title="DOI URL">https://doi.org/10.1002/9783527809257.ch14</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch14%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BReader%252BProteins%26aulast%3DBacher%26aufirst%3DJohannes%26date%3D2019%26date%3D2019%26spage%3D383%26epage%3D419%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Methyl-Readers and Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 339-399. <a href="https://doi.org/10.1007/7355_2019_78" title="DOI URL">https://doi.org/10.1007/7355_2019_78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_78%26sid%3Dliteratum%253Aachs%26atitle%3DMethyl-Readers%252Band%252BInhibitors%26aulast%3DSbardella%26aufirst%3DGianluca%26date%3D2019%26date%3D2019%26spage%3D339%26epage%3D399%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunxiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Feng  Leng</span>, <span class="hlFld-ContribAuthor ">Lovely  Saxena</span>, <span class="hlFld-ContribAuthor ">Nam  Hoang</span>, <span class="hlFld-ContribAuthor ">Jiekai  Yu</span>, <span class="hlFld-ContribAuthor ">Salvador  Alejo</span>, <span class="hlFld-ContribAuthor ">Logan  Lee</span>, <span class="hlFld-ContribAuthor ">Dandan  Qi</span>, <span class="hlFld-ContribAuthor ">Fei  Lu</span>, <span class="hlFld-ContribAuthor ">Hong  Sun</span>, <span class="hlFld-ContribAuthor ">Hui  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis of methylated SOX2 protein is regulated by L3MBTL3 and CRL4DCAF5 ubiquitin ligase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2019,</strong> <em>294 </em>
                                    (2)
                                     , 476-489. <a href="https://doi.org/10.1074/jbc.RA118.005336" title="DOI URL">https://doi.org/10.1074/jbc.RA118.005336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA118.005336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA118.005336%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DProteolysis%252Bof%252Bmethylated%252BSOX2%252Bprotein%252Bis%252Bregulated%252Bby%252BL3MBTL3%252Band%252BCRL4DCAF5%252Bubiquitin%252Bligase%26aulast%3DZhang%26aufirst%3DChunxiao%26date%3D2019%26volume%3D294%26issue%3D2%26spage%3D476%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander-Thomas  Hauser</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic small molecule modulators of histone and DNA methylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>45 </em>, 73-85. <a href="https://doi.org/10.1016/j.cbpa.2018.03.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.03.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.03.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DEpigenetic%252Bsmall%252Bmolecule%252Bmodulators%252Bof%252Bhistone%252Band%252BDNA%252Bmethylation%26aulast%3DHauser%26aufirst%3DAlexander-Thomas%26date%3D2018%26volume%3D45%26spage%3D73%26epage%3D85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hui  Liu</span>, <span class="hlFld-ContribAuthor ">Zhuoming  Li</span>, <span class="hlFld-ContribAuthor ">Liang  Li</span>. </span><span class="cited-content_cbyCitation_article-title">The molecular selectivity of UNC3866 inhibitor for Polycomb CBX7 protein from molecular dynamics simulation. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2018,</strong> <em>74 </em>, 339-346. <a href="https://doi.org/10.1016/j.compbiolchem.2018.04.005" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2018.04.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2018.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2018.04.005%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DThe%252Bmolecular%252Bselectivity%252Bof%252BUNC3866%252Binhibitor%252Bfor%252BPolycomb%252BCBX7%252Bprotein%252Bfrom%252Bmolecular%252Bdynamics%252Bsimulation%26aulast%3DLiu%26aufirst%3DHui%26date%3D2018%26volume%3D74%26spage%3D339%26epage%3D346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Scott  Ribich</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>. </span><span class="cited-content_cbyCitation_article-title">RNA-modifying proteins as anticancer drug targets. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2018,</strong> <em>17 </em>
                                    (6)
                                     , 435-453. <a href="https://doi.org/10.1038/nrd.2018.71" title="DOI URL">https://doi.org/10.1038/nrd.2018.71</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2018.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2018.71%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DRNA-modifying%252Bproteins%252Bas%252Banticancer%252Bdrug%252Btargets%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BAnn%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D6%26spage%3D435%26epage%3D453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiuyang  Liu</span>, <span class="hlFld-ContribAuthor ">Shuya  Zhang</span>, <span class="hlFld-ContribAuthor ">Mingqing  Liu</span>, <span class="hlFld-ContribAuthor ">Yaqian  Liu</span>, <span class="hlFld-ContribAuthor ">Gilbert  Nshogoza</span>, <span class="hlFld-ContribAuthor ">Jia  Gao</span>, <span class="hlFld-ContribAuthor ">Rongsheng  Ma</span>, <span class="hlFld-ContribAuthor ">Yang  Yang</span>, <span class="hlFld-ContribAuthor ">Jihui  Wu</span>, <span class="hlFld-ContribAuthor ">Jiahai  Zhang</span>, <span class="hlFld-ContribAuthor ">Fudong  Li</span>, <span class="hlFld-ContribAuthor ">Ke  Ruan</span>. </span><span class="cited-content_cbyCitation_article-title">Structural plasticity of the TDRD3 Tudor domain probed by a fragment screening hit. </span><span class="cited-content_cbyCitation_journal-name">The FEBS Journal</span><span> <strong>2018,</strong> <em>285 </em>
                                    (11)
                                     , 2091-2103. <a href="https://doi.org/10.1111/febs.14469" title="DOI URL">https://doi.org/10.1111/febs.14469</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.14469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.14469%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FEBS%2520Journal%26atitle%3DStructural%252Bplasticity%252Bof%252Bthe%252BTDRD3%252BTudor%252Bdomain%252Bprobed%252Bby%252Ba%252Bfragment%252Bscreening%252Bhit%26aulast%3DLiu%26aufirst%3DJiuyang%26date%3D2018%26date%3D2018%26volume%3D285%26issue%3D11%26spage%3D2091%26epage%3D2103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>. </span><span class="cited-content_cbyCitation_article-title">Application of Virtual Screening Approaches for the Identification of Small Molecule Inhibitors of the Methyllysine Reader Protein Spindlin1. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 347-370. <a href="https://doi.org/10.1007/978-1-4939-8630-9_21" title="DOI URL">https://doi.org/10.1007/978-1-4939-8630-9_21</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-8630-9_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-8630-9_21%26sid%3Dliteratum%253Aachs%26atitle%3DApplication%252Bof%252BVirtual%252BScreening%252BApproaches%252Bfor%252Bthe%252BIdentification%252Bof%252BSmall%252BMolecule%252BInhibitors%252Bof%252Bthe%252BMethyllysine%252BReader%252BProtein%252BSpindlin1%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2018%26date%3D2018%26spage%3D347%26epage%3D370%26pub%3DSpringer%2520New%2520York%26atitle%3DRational%252BDrug%252BDesign%26aulast%3DMavromoustakos%26aufirst%3DThomas%26date%3D2018%26volume%3D1824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyan  Yue</span>, <span class="hlFld-ContribAuthor ">Wei  Wan</span>, <span class="hlFld-ContribAuthor ">Pan  Xu</span>, <span class="hlFld-ContribAuthor ">Linjuan  Li</span>, <span class="hlFld-ContribAuthor ">Chen  Wang</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Heng  Xu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Junchi  Hu</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Inhibitors Targeting New Targets of Protein-Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 179-211. <a href="https://doi.org/10.1007/978-981-13-0773-7_7" title="DOI URL">https://doi.org/10.1007/978-981-13-0773-7_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-13-0773-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-13-0773-7_7%26sid%3Dliteratum%253Aachs%26atitle%3DSmall%252BMolecule%252BInhibitors%252BTargeting%252BNew%252BTargets%252Bof%252BProtein-Protein%252BInteractions%26aulast%3DYue%26aufirst%3DLiyan%26date%3D2018%26date%3D2018%26spage%3D179%26epage%3D211%26pub%3DSpringer%2520Singapore%26atitle%3DTargeting%252BProtein-Protein%252BInteractions%252Bby%252BSmall%252BMolecules%26aulast%3DSheng%26aufirst%3DChunquan%26date%3D2018%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">, , , , , , <span class="hlFld-ContribAuthor ">Roger L  Milne</span>, <span class="hlFld-ContribAuthor ">Karoline B  Kuchenbaecker</span>, <span class="hlFld-ContribAuthor ">Kyriaki  Michailidou</span>, <span class="hlFld-ContribAuthor ">Jonathan  Beesley</span>, <span class="hlFld-ContribAuthor ">Siddhartha  Kar</span>, <span class="hlFld-ContribAuthor ">Sara  Lindström</span>, <span class="hlFld-ContribAuthor ">Shirley  Hui</span>, <span class="hlFld-ContribAuthor ">Audrey  Lemaçon</span>, <span class="hlFld-ContribAuthor ">Penny  Soucy</span>, <span class="hlFld-ContribAuthor ">Joe  Dennis</span>, <span class="hlFld-ContribAuthor ">Xia  Jiang</span>, <span class="hlFld-ContribAuthor ">Asha  Rostamianfar</span>, <span class="hlFld-ContribAuthor ">Hilary  Finucane</span>, <span class="hlFld-ContribAuthor ">Manjeet K  Bolla</span>, <span class="hlFld-ContribAuthor ">Lesley  McGuffog</span>, <span class="hlFld-ContribAuthor ">Qin  Wang</span>, <span class="hlFld-ContribAuthor ">Cora M  Aalfs</span>, <span class="hlFld-ContribAuthor ">Marcia  Adams</span>, <span class="hlFld-ContribAuthor ">Julian  Adlard</span>, <span class="hlFld-ContribAuthor ">Simona  Agata</span>, <span class="hlFld-ContribAuthor ">Shahana  Ahmed</span>, <span class="hlFld-ContribAuthor ">Habibul  Ahsan</span>, <span class="hlFld-ContribAuthor ">Kristiina  Aittomäki</span>, <span class="hlFld-ContribAuthor ">Fares  Al-Ejeh</span>, <span class="hlFld-ContribAuthor ">Jamie  Allen</span>, <span class="hlFld-ContribAuthor ">Christine B  Ambrosone</span>, <span class="hlFld-ContribAuthor ">Christopher I  Amos</span>, <span class="hlFld-ContribAuthor ">Irene L  Andrulis</span>, <span class="hlFld-ContribAuthor ">Hoda  Anton-Culver</span>, <span class="hlFld-ContribAuthor ">Natalia N  Antonenkova</span>, <span class="hlFld-ContribAuthor ">Volker  Arndt</span>, <span class="hlFld-ContribAuthor ">Norbert  Arnold</span>, <span class="hlFld-ContribAuthor ">Kristan J  Aronson</span>, <span class="hlFld-ContribAuthor ">Bernd  Auber</span>, <span class="hlFld-ContribAuthor ">Paul L  Auer</span>, <span class="hlFld-ContribAuthor ">Margreet G E M  Ausems</span>, <span class="hlFld-ContribAuthor ">Jacopo  Azzollini</span>, <span class="hlFld-ContribAuthor ">François  Bacot</span>, <span class="hlFld-ContribAuthor ">Judith  Balmaña</span>, <span class="hlFld-ContribAuthor ">Monica  Barile</span>, <span class="hlFld-ContribAuthor ">Laure  Barjhoux</span>, <span class="hlFld-ContribAuthor ">Rosa B  Barkardottir</span>, <span class="hlFld-ContribAuthor ">Myrto  Barrdahl</span>, <span class="hlFld-ContribAuthor ">Daniel  Barnes</span>, <span class="hlFld-ContribAuthor ">Daniel  Barrowdale</span>, <span class="hlFld-ContribAuthor ">Caroline  Baynes</span>, <span class="hlFld-ContribAuthor ">Matthias W  Beckmann</span>, <span class="hlFld-ContribAuthor ">Javier  Benitez</span>, <span class="hlFld-ContribAuthor ">Marina  Bermisheva</span>, <span class="hlFld-ContribAuthor ">Leslie  Bernstein</span>, <span class="hlFld-ContribAuthor ">Yves-Jean  Bignon</span>, <span class="hlFld-ContribAuthor ">Kathleen R  Blazer</span>, <span class="hlFld-ContribAuthor ">Marinus J  Blok</span>, <span class="hlFld-ContribAuthor ">Carl  Blomqvist</span>, <span class="hlFld-ContribAuthor ">William  Blot</span>, <span class="hlFld-ContribAuthor ">Kristie  Bobolis</span>, <span class="hlFld-ContribAuthor ">Bram  Boeckx</span>, <span class="hlFld-ContribAuthor ">Natalia V  Bogdanova</span>, <span class="hlFld-ContribAuthor ">Anders  Bojesen</span>, <span class="hlFld-ContribAuthor ">Stig E  Bojesen</span>, <span class="hlFld-ContribAuthor ">Bernardo  Bonanni</span>, <span class="hlFld-ContribAuthor ">Anne-Lise  Børresen-Dale</span>, <span class="hlFld-ContribAuthor ">Aniko  Bozsik</span>, <span class="hlFld-ContribAuthor ">Angela R  Bradbury</span>, <span class="hlFld-ContribAuthor ">Judith S  Brand</span>, <span class="hlFld-ContribAuthor ">Hiltrud  Brauch</span>, <span class="hlFld-ContribAuthor ">Hermann  Brenner</span>, <span class="hlFld-ContribAuthor ">Brigitte  Bressac-de Paillerets</span>, <span class="hlFld-ContribAuthor ">Carole  Brewer</span>, <span class="hlFld-ContribAuthor ">Louise  Brinton</span>, <span class="hlFld-ContribAuthor ">Per  Broberg</span>, <span class="hlFld-ContribAuthor ">Angela  Brooks-Wilson</span>, <span class="hlFld-ContribAuthor ">Joan  Brunet</span>, <span class="hlFld-ContribAuthor ">Thomas  Brüning</span>, <span class="hlFld-ContribAuthor ">Barbara  Burwinkel</span>, <span class="hlFld-ContribAuthor ">Saundra S  Buys</span>, <span class="hlFld-ContribAuthor ">Jinyoung  Byun</span>, <span class="hlFld-ContribAuthor ">Qiuyin  Cai</span>, <span class="hlFld-ContribAuthor ">Trinidad  Caldés</span>, <span class="hlFld-ContribAuthor ">Maria A  Caligo</span>, <span class="hlFld-ContribAuthor ">Ian  Campbell</span>, <span class="hlFld-ContribAuthor ">Federico  Canzian</span>, <span class="hlFld-ContribAuthor ">Olivier  Caron</span>, <span class="hlFld-ContribAuthor ">Angel  Carracedo</span>, <span class="hlFld-ContribAuthor ">Brian D  Carter</span>, <span class="hlFld-ContribAuthor ">J Esteban  Castelao</span>, <span class="hlFld-ContribAuthor ">Laurent  Castera</span>, <span class="hlFld-ContribAuthor ">Virginie  Caux-Moncoutier</span>, <span class="hlFld-ContribAuthor ">Salina B  Chan</span>, <span class="hlFld-ContribAuthor ">Jenny  Chang-Claude</span>, <span class="hlFld-ContribAuthor ">Stephen J  Chanock</span>, <span class="hlFld-ContribAuthor ">Xiaoqing  Chen</span>, <span class="hlFld-ContribAuthor ">Ting-Yuan David  Cheng</span>, <span class="hlFld-ContribAuthor ">Jocelyne  Chiquette</span>, <span class="hlFld-ContribAuthor ">Hans  Christiansen</span>, <span class="hlFld-ContribAuthor ">Kathleen B M  Claes</span>, <span class="hlFld-ContribAuthor ">Christine L  Clarke</span>, <span class="hlFld-ContribAuthor ">Thomas  Conner</span>, <span class="hlFld-ContribAuthor ">Don M  Conroy</span>, <span class="hlFld-ContribAuthor ">Jackie  Cook</span>, <span class="hlFld-ContribAuthor ">Emilie  Cordina-Duverger</span>, <span class="hlFld-ContribAuthor ">Sten  Cornelissen</span>, <span class="hlFld-ContribAuthor ">Isabelle  Coupier</span>, <span class="hlFld-ContribAuthor ">Angela  Cox</span>, <span class="hlFld-ContribAuthor ">David G  Cox</span>, <span class="hlFld-ContribAuthor ">Simon S  Cross</span>, <span class="hlFld-ContribAuthor ">Katarina  Cuk</span>, <span class="hlFld-ContribAuthor ">Julie M  Cunningham</span>, <span class="hlFld-ContribAuthor ">Kamila  Czene</span>, <span class="hlFld-ContribAuthor ">Mary B  Daly</span>, <span class="hlFld-ContribAuthor ">Francesca  Damiola</span>, <span class="hlFld-ContribAuthor ">Hatef  Darabi</span>, <span class="hlFld-ContribAuthor ">Rosemarie  Davidson</span>, <span class="hlFld-ContribAuthor ">Kim  De Leeneer</span>, <span class="hlFld-ContribAuthor ">Peter  Devilee</span>, <span class="hlFld-ContribAuthor ">Ed  Dicks</span>, <span class="hlFld-ContribAuthor ">Orland  Diez</span>, <span class="hlFld-ContribAuthor ">Yuan Chun  Ding</span>, <span class="hlFld-ContribAuthor ">Nina  Ditsch</span>, <span class="hlFld-ContribAuthor ">Kimberly F  Doheny</span>, <span class="hlFld-ContribAuthor ">Susan M  Domchek</span>, <span class="hlFld-ContribAuthor ">Cecilia M  Dorfling</span>, <span class="hlFld-ContribAuthor ">Thilo  Dörk</span>, <span class="hlFld-ContribAuthor ">Isabel  dos-Santos-Silva</span>, <span class="hlFld-ContribAuthor ">Stéphane  Dubois</span>, <span class="hlFld-ContribAuthor ">Pierre-Antoine  Dugué</span>, <span class="hlFld-ContribAuthor ">Martine  Dumont</span>, <span class="hlFld-ContribAuthor ">Alison M  Dunning</span>, <span class="hlFld-ContribAuthor ">Lorraine  Durcan</span>, <span class="hlFld-ContribAuthor ">Miriam  Dwek</span>, <span class="hlFld-ContribAuthor ">Bernd  Dworniczak</span>, <span class="hlFld-ContribAuthor ">Diana  Eccles</span>, <span class="hlFld-ContribAuthor ">Ros  Eeles</span>, <span class="hlFld-ContribAuthor ">Hans  Ehrencrona</span>, <span class="hlFld-ContribAuthor ">Ursula  Eilber</span>, <span class="hlFld-ContribAuthor ">Bent  Ejlertsen</span>, <span class="hlFld-ContribAuthor ">Arif B  Ekici</span>, <span class="hlFld-ContribAuthor ">A Heather  Eliassen</span>, <span class="hlFld-ContribAuthor ">Christoph  Engel</span>, <span class="hlFld-ContribAuthor ">Mikael  Eriksson</span>, <span class="hlFld-ContribAuthor ">Laura  Fachal</span>, <span class="hlFld-ContribAuthor ">Laurence  Faivre</span>, <span class="hlFld-ContribAuthor ">Peter A  Fasching</span>, <span class="hlFld-ContribAuthor ">Ulrike  Faust</span>, <span class="hlFld-ContribAuthor ">Jonine  Figueroa</span>, <span class="hlFld-ContribAuthor ">Dieter  Flesch-Janys</span>, <span class="hlFld-ContribAuthor ">Olivia  Fletcher</span>, <span class="hlFld-ContribAuthor ">Henrik  Flyger</span>, <span class="hlFld-ContribAuthor ">William D  Foulkes</span>, <span class="hlFld-ContribAuthor ">Eitan  Friedman</span>, <span class="hlFld-ContribAuthor ">Lin  Fritschi</span>, <span class="hlFld-ContribAuthor ">Debra  Frost</span>, <span class="hlFld-ContribAuthor ">Marike  Gabrielson</span>, <span class="hlFld-ContribAuthor ">Pragna  Gaddam</span>, <span class="hlFld-ContribAuthor ">Marilie D  Gammon</span>, <span class="hlFld-ContribAuthor ">Patricia A  Ganz</span>, <span class="hlFld-ContribAuthor ">Susan M  Gapstur</span>, <span class="hlFld-ContribAuthor ">Judy  Garber</span>, <span class="hlFld-ContribAuthor ">Vanesa  Garcia-Barberan</span>, <span class="hlFld-ContribAuthor ">José A  García-Sáenz</span>, <span class="hlFld-ContribAuthor ">Mia M  Gaudet</span>, <span class="hlFld-ContribAuthor ">Marion  Gauthier-Villars</span>, <span class="hlFld-ContribAuthor ">Andrea  Gehrig</span>, <span class="hlFld-ContribAuthor ">Vassilios  Georgoulias</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Gerdes</span>, <span class="hlFld-ContribAuthor ">Graham G  Giles</span>, <span class="hlFld-ContribAuthor ">Gord  Glendon</span>, <span class="hlFld-ContribAuthor ">Andrew K  Godwin</span>, <span class="hlFld-ContribAuthor ">Mark S  Goldberg</span>, <span class="hlFld-ContribAuthor ">David E  Goldgar</span>, <span class="hlFld-ContribAuthor ">Anna  González-Neira</span>, <span class="hlFld-ContribAuthor ">Paul  Goodfellow</span>, <span class="hlFld-ContribAuthor ">Mark H  Greene</span>, <span class="hlFld-ContribAuthor ">Grethe I Grenaker  Alnæs</span>, <span class="hlFld-ContribAuthor ">Mervi  Grip</span>, <span class="hlFld-ContribAuthor ">Jacek  Gronwald</span>, <span class="hlFld-ContribAuthor ">Anne  Grundy</span>, <span class="hlFld-ContribAuthor ">Daphne  Gschwantler-Kaulich</span>, <span class="hlFld-ContribAuthor ">Pascal  Guénel</span>, <span class="hlFld-ContribAuthor ">Qi  Guo</span>, <span class="hlFld-ContribAuthor ">Lothar  Haeberle</span>, <span class="hlFld-ContribAuthor ">Eric  Hahnen</span>, <span class="hlFld-ContribAuthor ">Christopher A  Haiman</span>, <span class="hlFld-ContribAuthor ">Niclas  Håkansson</span>, <span class="hlFld-ContribAuthor ">Emily  Hallberg</span>, <span class="hlFld-ContribAuthor ">Ute  Hamann</span>, <span class="hlFld-ContribAuthor ">Nathalie  Hamel</span>, <span class="hlFld-ContribAuthor ">Susan  Hankinson</span>, <span class="hlFld-ContribAuthor ">Thomas V O  Hansen</span>, <span class="hlFld-ContribAuthor ">Patricia  Harrington</span>, <span class="hlFld-ContribAuthor ">Steven N  Hart</span>, <span class="hlFld-ContribAuthor ">Jaana M  Hartikainen</span>, <span class="hlFld-ContribAuthor ">Catherine S  Healey</span>, <span class="hlFld-ContribAuthor ">Alexander  Hein</span>, <span class="hlFld-ContribAuthor ">Sonja  Helbig</span>, <span class="hlFld-ContribAuthor ">Alex  Henderson</span>, <span class="hlFld-ContribAuthor ">Jane  Heyworth</span>, <span class="hlFld-ContribAuthor ">Belynda  Hicks</span>, <span class="hlFld-ContribAuthor ">Peter  Hillemanns</span>, <span class="hlFld-ContribAuthor ">Shirley  Hodgson</span>, <span class="hlFld-ContribAuthor ">Frans B  Hogervorst</span>, <span class="hlFld-ContribAuthor ">Antoinette  Hollestelle</span>, <span class="hlFld-ContribAuthor ">Maartje J  Hooning</span>, <span class="hlFld-ContribAuthor ">Bob  Hoover</span>, <span class="hlFld-ContribAuthor ">John L  Hopper</span>, <span class="hlFld-ContribAuthor ">Chunling  Hu</span>, <span class="hlFld-ContribAuthor ">Guanmengqian  Huang</span>, <span class="hlFld-ContribAuthor ">Peter J  Hulick</span>, <span class="hlFld-ContribAuthor ">Keith  Humphreys</span>, <span class="hlFld-ContribAuthor ">David J  Hunter</span>, <span class="hlFld-ContribAuthor ">Evgeny N  Imyanitov</span>, <span class="hlFld-ContribAuthor ">Claudine  Isaacs</span>, <span class="hlFld-ContribAuthor ">Motoki  Iwasaki</span>, <span class="hlFld-ContribAuthor ">Louise  Izatt</span>, <span class="hlFld-ContribAuthor ">Anna  Jakubowska</span>, <span class="hlFld-ContribAuthor ">Paul  James</span>, <span class="hlFld-ContribAuthor ">Ramunas  Janavicius</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Janni</span>, <span class="hlFld-ContribAuthor ">Uffe Birk  Jensen</span>, <span class="hlFld-ContribAuthor ">Esther M  John</span>, <span class="hlFld-ContribAuthor ">Nichola  Johnson</span>, <span class="hlFld-ContribAuthor ">Kristine  Jones</span>, <span class="hlFld-ContribAuthor ">Michael  Jones</span>, <span class="hlFld-ContribAuthor ">Arja  Jukkola-Vuorinen</span>, <span class="hlFld-ContribAuthor ">Rudolf  Kaaks</span>, <span class="hlFld-ContribAuthor ">Maria  Kabisch</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kaczmarek</span>, <span class="hlFld-ContribAuthor ">Daehee  Kang</span>, <span class="hlFld-ContribAuthor ">Karin  Kast</span>, <span class="hlFld-ContribAuthor ">Renske  Keeman</span>, <span class="hlFld-ContribAuthor ">Michael J  Kerin</span>, <span class="hlFld-ContribAuthor ">Carolien M  Kets</span>, <span class="hlFld-ContribAuthor ">Machteld  Keupers</span>, <span class="hlFld-ContribAuthor ">Sofia  Khan</span>, <span class="hlFld-ContribAuthor ">Elza  Khusnutdinova</span>, <span class="hlFld-ContribAuthor ">Johanna I  Kiiski</span>, <span class="hlFld-ContribAuthor ">Sung-Won  Kim</span>, <span class="hlFld-ContribAuthor ">Julia A  Knight</span>, <span class="hlFld-ContribAuthor ">Irene  Konstantopoulou</span>, <span class="hlFld-ContribAuthor ">Veli-Matti  Kosma</span>, <span class="hlFld-ContribAuthor ">Vessela N  Kristensen</span>, <span class="hlFld-ContribAuthor ">Torben A  Kruse</span>, <span class="hlFld-ContribAuthor ">Ava  Kwong</span>, <span class="hlFld-ContribAuthor ">Anne-Vibeke  Lænkholm</span>, <span class="hlFld-ContribAuthor ">Yael  Laitman</span>, <span class="hlFld-ContribAuthor ">Fiona  Lalloo</span>, <span class="hlFld-ContribAuthor ">Diether  Lambrechts</span>, <span class="hlFld-ContribAuthor ">Keren  Landsman</span>, <span class="hlFld-ContribAuthor ">Christine  Lasset</span>, <span class="hlFld-ContribAuthor ">Conxi  Lazaro</span>, <span class="hlFld-ContribAuthor ">Loic  Le Marchand</span>, <span class="hlFld-ContribAuthor ">Julie  Lecarpentier</span>, <span class="hlFld-ContribAuthor ">Andrew  Lee</span>, <span class="hlFld-ContribAuthor ">Eunjung  Lee</span>, <span class="hlFld-ContribAuthor ">Jong Won  Lee</span>, <span class="hlFld-ContribAuthor ">Min Hyuk  Lee</span>, <span class="hlFld-ContribAuthor ">Flavio  Lejbkowicz</span>, <span class="hlFld-ContribAuthor ">Fabienne  Lesueur</span>, <span class="hlFld-ContribAuthor ">Jingmei  Li</span>, <span class="hlFld-ContribAuthor ">Jenna  Lilyquist</span>, <span class="hlFld-ContribAuthor ">Anne  Lincoln</span>, <span class="hlFld-ContribAuthor ">Annika  Lindblom</span>, <span class="hlFld-ContribAuthor ">Jolanta  Lissowska</span>, <span class="hlFld-ContribAuthor ">Wing-Yee  Lo</span>, <span class="hlFld-ContribAuthor ">Sibylle  Loibl</span>, <span class="hlFld-ContribAuthor ">Jirong  Long</span>, <span class="hlFld-ContribAuthor ">Jennifer T  Loud</span>, <span class="hlFld-ContribAuthor ">Jan  Lubinski</span>, <span class="hlFld-ContribAuthor ">Craig  Luccarini</span>, <span class="hlFld-ContribAuthor ">Michael  Lush</span>, <span class="hlFld-ContribAuthor ">Robert J  MacInnis</span>, <span class="hlFld-ContribAuthor ">Tom  Maishman</span>, <span class="hlFld-ContribAuthor ">Enes  Makalic</span>, <span class="hlFld-ContribAuthor ">Ivana Maleva  Kostovska</span>, <span class="hlFld-ContribAuthor ">Kathleen E  Malone</span>, <span class="hlFld-ContribAuthor ">Siranoush  Manoukian</span>, <span class="hlFld-ContribAuthor ">JoAnn E  Manson</span>, <span class="hlFld-ContribAuthor ">Sara  Margolin</span>, <span class="hlFld-ContribAuthor ">John W M  Martens</span>, <span class="hlFld-ContribAuthor ">Maria Elena  Martinez</span>, <span class="hlFld-ContribAuthor ">Keitaro  Matsuo</span>, <span class="hlFld-ContribAuthor ">Dimitrios  Mavroudis</span>, <span class="hlFld-ContribAuthor ">Sylvie  Mazoyer</span>, <span class="hlFld-ContribAuthor ">Catriona  McLean</span>, <span class="hlFld-ContribAuthor ">Hanne  Meijers-Heijboer</span>, <span class="hlFld-ContribAuthor ">Primitiva  Menéndez</span>, <span class="hlFld-ContribAuthor ">Jeffery  Meyer</span>, <span class="hlFld-ContribAuthor ">Hui  Miao</span>, <span class="hlFld-ContribAuthor ">Austin  Miller</span>, <span class="hlFld-ContribAuthor ">Nicola  Miller</span>, <span class="hlFld-ContribAuthor ">Gillian  Mitchell</span>, <span class="hlFld-ContribAuthor ">Marco  Montagna</span>, <span class="hlFld-ContribAuthor ">Kenneth  Muir</span>, <span class="hlFld-ContribAuthor ">Anna Marie  Mulligan</span>, <span class="hlFld-ContribAuthor ">Claire  Mulot</span>, <span class="hlFld-ContribAuthor ">Sue  Nadesan</span>, <span class="hlFld-ContribAuthor ">Katherine L  Nathanson</span>, <span class="hlFld-ContribAuthor ">Susan L  Neuhausen</span>, <span class="hlFld-ContribAuthor ">Heli  Nevanlinna</span>, <span class="hlFld-ContribAuthor ">Ines  Nevelsteen</span>, <span class="hlFld-ContribAuthor ">Dieter  Niederacher</span>, <span class="hlFld-ContribAuthor ">Sune F  Nielsen</span>, <span class="hlFld-ContribAuthor ">Børge G  Nordestgaard</span>, <span class="hlFld-ContribAuthor ">Aaron  Norman</span>, <span class="hlFld-ContribAuthor ">Robert L  Nussbaum</span>, <span class="hlFld-ContribAuthor ">Edith  Olah</span>, <span class="hlFld-ContribAuthor ">Olufunmilayo I  Olopade</span>, <span class="hlFld-ContribAuthor ">Janet E  Olson</span>, <span class="hlFld-ContribAuthor ">Curtis  Olswold</span>, <span class="hlFld-ContribAuthor ">Kai-ren  Ong</span>, <span class="hlFld-ContribAuthor ">Jan C  Oosterwijk</span>, <span class="hlFld-ContribAuthor ">Nick  Orr</span>, <span class="hlFld-ContribAuthor ">Ana  Osorio</span>, <span class="hlFld-ContribAuthor ">V Shane  Pankratz</span>, <span class="hlFld-ContribAuthor ">Laura  Papi</span>, <span class="hlFld-ContribAuthor ">Tjoung-Won  Park-Simon</span>, <span class="hlFld-ContribAuthor ">Ylva  Paulsson-Karlsson</span>, <span class="hlFld-ContribAuthor ">Rachel  Lloyd</span>, <span class="hlFld-ContribAuthor ">Inge Søkilde  Pedersen</span>, <span class="hlFld-ContribAuthor ">Bernard  Peissel</span>, <span class="hlFld-ContribAuthor ">Ana  Peixoto</span>, <span class="hlFld-ContribAuthor ">Jose I A  Perez</span>, <span class="hlFld-ContribAuthor ">Paolo  Peterlongo</span>, <span class="hlFld-ContribAuthor ">Julian  Peto</span>, <span class="hlFld-ContribAuthor ">Georg  Pfeiler</span>, <span class="hlFld-ContribAuthor ">Catherine M  Phelan</span>, <span class="hlFld-ContribAuthor ">Mila  Pinchev</span>, <span class="hlFld-ContribAuthor ">Dijana  Plaseska-Karanfilska</span>, <span class="hlFld-ContribAuthor ">Bruce  Poppe</span>, <span class="hlFld-ContribAuthor ">Mary E  Porteous</span>, <span class="hlFld-ContribAuthor ">Ross  Prentice</span>, <span class="hlFld-ContribAuthor ">Nadege  Presneau</span>, <span class="hlFld-ContribAuthor ">Darya  Prokofieva</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Pugh</span>, <span class="hlFld-ContribAuthor ">Miquel Angel  Pujana</span>, <span class="hlFld-ContribAuthor ">Katri  Pylkäs</span>, <span class="hlFld-ContribAuthor ">Brigitte  Rack</span>, <span class="hlFld-ContribAuthor ">Paolo  Radice</span>, <span class="hlFld-ContribAuthor ">Nazneen  Rahman</span>, <span class="hlFld-ContribAuthor ">Johanna  Rantala</span>, <span class="hlFld-ContribAuthor ">Christine  Rappaport-Fuerhauser</span>, <span class="hlFld-ContribAuthor ">Gad  Rennert</span>, <span class="hlFld-ContribAuthor ">Hedy S  Rennert</span>, <span class="hlFld-ContribAuthor ">Valerie  Rhenius</span>, <span class="hlFld-ContribAuthor ">Kerstin  Rhiem</span>, <span class="hlFld-ContribAuthor ">Andrea  Richardson</span>, <span class="hlFld-ContribAuthor ">Gustavo C  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Atocha  Romero</span>, <span class="hlFld-ContribAuthor ">Jane  Romm</span>, <span class="hlFld-ContribAuthor ">Matti A  Rookus</span>, <span class="hlFld-ContribAuthor ">Anja  Rudolph</span>, <span class="hlFld-ContribAuthor ">Thomas  Ruediger</span>, <span class="hlFld-ContribAuthor ">Emmanouil  Saloustros</span>, <span class="hlFld-ContribAuthor ">Joyce  Sanders</span>, <span class="hlFld-ContribAuthor ">Dale P  Sandler</span>, <span class="hlFld-ContribAuthor ">Suleeporn  Sangrajrang</span>, <span class="hlFld-ContribAuthor ">Elinor J  Sawyer</span>, <span class="hlFld-ContribAuthor ">Daniel F  Schmidt</span>, <span class="hlFld-ContribAuthor ">Minouk J  Schoemaker</span>, <span class="hlFld-ContribAuthor ">Fredrick  Schumacher</span>, <span class="hlFld-ContribAuthor ">Peter  Schürmann</span>, <span class="hlFld-ContribAuthor ">Lukas  Schwentner</span>, <span class="hlFld-ContribAuthor ">Christopher  Scott</span>, <span class="hlFld-ContribAuthor ">Rodney J  Scott</span>, <span class="hlFld-ContribAuthor ">Sheila  Seal</span>, <span class="hlFld-ContribAuthor ">Leigha  Senter</span>, <span class="hlFld-ContribAuthor ">Caroline  Seynaeve</span>, <span class="hlFld-ContribAuthor ">Mitul  Shah</span>, <span class="hlFld-ContribAuthor ">Priyanka  Sharma</span>, <span class="hlFld-ContribAuthor ">Chen-Yang  Shen</span>, <span class="hlFld-ContribAuthor ">Xin  Sheng</span>, <span class="hlFld-ContribAuthor ">Hermela  Shimelis</span>, <span class="hlFld-ContribAuthor ">Martha J  Shrubsole</span>, <span class="hlFld-ContribAuthor ">Xiao-Ou  Shu</span>, <span class="hlFld-ContribAuthor ">Lucy E  Side</span>, <span class="hlFld-ContribAuthor ">Christian F  Singer</span>, <span class="hlFld-ContribAuthor ">Christof  Sohn</span>, <span class="hlFld-ContribAuthor ">Melissa C  Southey</span>, <span class="hlFld-ContribAuthor ">John J  Spinelli</span>, <span class="hlFld-ContribAuthor ">Amanda B  Spurdle</span>, <span class="hlFld-ContribAuthor ">Christa  Stegmaier</span>, <span class="hlFld-ContribAuthor ">Dominique  Stoppa-Lyonnet</span>, <span class="hlFld-ContribAuthor ">Grzegorz  Sukiennicki</span>, <span class="hlFld-ContribAuthor ">Harald  Surowy</span>, <span class="hlFld-ContribAuthor ">Christian  Sutter</span>, <span class="hlFld-ContribAuthor ">Anthony  Swerdlow</span>, <span class="hlFld-ContribAuthor ">Csilla I  Szabo</span>, <span class="hlFld-ContribAuthor ">Rulla M  Tamimi</span>, <span class="hlFld-ContribAuthor ">Yen Y  Tan</span>, <span class="hlFld-ContribAuthor ">Jack A  Taylor</span>, <span class="hlFld-ContribAuthor ">Maria-Isabel  Tejada</span>, <span class="hlFld-ContribAuthor ">Maria  Tengström</span>, <span class="hlFld-ContribAuthor ">Soo H  Teo</span>, <span class="hlFld-ContribAuthor ">Mary B  Terry</span>, <span class="hlFld-ContribAuthor ">Daniel C  Tessier</span>, <span class="hlFld-ContribAuthor ">Alex  Teulé</span>, <span class="hlFld-ContribAuthor ">Kathrin  Thöne</span>, <span class="hlFld-ContribAuthor ">Darcy L  Thull</span>, <span class="hlFld-ContribAuthor ">Maria Grazia  Tibiletti</span>, <span class="hlFld-ContribAuthor ">Laima  Tihomirova</span>, <span class="hlFld-ContribAuthor ">Marc  Tischkowitz</span>, <span class="hlFld-ContribAuthor ">Amanda E  Toland</span>, <span class="hlFld-ContribAuthor ">Rob A E M  Tollenaar</span>, <span class="hlFld-ContribAuthor ">Ian  Tomlinson</span>, <span class="hlFld-ContribAuthor ">Ling  Tong</span>, <span class="hlFld-ContribAuthor ">Diana  Torres</span>, <span class="hlFld-ContribAuthor ">Martine  Tranchant</span>, <span class="hlFld-ContribAuthor ">Thérèse  Truong</span>, <span class="hlFld-ContribAuthor ">Kathy  Tucker</span>, <span class="hlFld-ContribAuthor ">Nadine  Tung</span>, <span class="hlFld-ContribAuthor ">Jonathan  Tyrer</span>, <span class="hlFld-ContribAuthor ">Hans-Ulrich  Ulmer</span>, <span class="hlFld-ContribAuthor ">Celine  Vachon</span>, <span class="hlFld-ContribAuthor ">Christi J  van Asperen</span>, <span class="hlFld-ContribAuthor ">David  Van Den Berg</span>, <span class="hlFld-ContribAuthor ">Ans M W  van den Ouweland</span>, <span class="hlFld-ContribAuthor ">Elizabeth J  van Rensburg</span>, <span class="hlFld-ContribAuthor ">Liliana  Varesco</span>, <span class="hlFld-ContribAuthor ">Raymonda  Varon-Mateeva</span>, <span class="hlFld-ContribAuthor ">Ana  Vega</span>, <span class="hlFld-ContribAuthor ">Alessandra  Viel</span>, <span class="hlFld-ContribAuthor ">Joseph  Vijai</span>, <span class="hlFld-ContribAuthor ">Daniel  Vincent</span>, <span class="hlFld-ContribAuthor ">Jason  Vollenweider</span>, <span class="hlFld-ContribAuthor ">Lisa  Walker</span>, <span class="hlFld-ContribAuthor ">Zhaoming  Wang</span>, <span class="hlFld-ContribAuthor ">Shan  Wang-Gohrke</span>, <span class="hlFld-ContribAuthor ">Barbara  Wappenschmidt</span>, <span class="hlFld-ContribAuthor ">Clarice R  Weinberg</span>, <span class="hlFld-ContribAuthor ">Jeffrey N  Weitzel</span>, <span class="hlFld-ContribAuthor ">Camilla  Wendt</span>, <span class="hlFld-ContribAuthor ">Jelle  Wesseling</span>, <span class="hlFld-ContribAuthor ">Alice S  Whittemore</span>, <span class="hlFld-ContribAuthor ">Juul T  Wijnen</span>, <span class="hlFld-ContribAuthor ">Walter  Willett</span>, <span class="hlFld-ContribAuthor ">Robert  Winqvist</span>, <span class="hlFld-ContribAuthor ">Alicja  Wolk</span>, <span class="hlFld-ContribAuthor ">Anna H  Wu</span>, <span class="hlFld-ContribAuthor ">Lucy  Xia</span>, <span class="hlFld-ContribAuthor ">Xiaohong R  Yang</span>, <span class="hlFld-ContribAuthor ">Drakoulis  Yannoukakos</span>, <span class="hlFld-ContribAuthor ">Daniela  Zaffaroni</span>, <span class="hlFld-ContribAuthor ">Wei  Zheng</span>, <span class="hlFld-ContribAuthor ">Bin  Zhu</span>, <span class="hlFld-ContribAuthor ">Argyrios  Ziogas</span>, <span class="hlFld-ContribAuthor ">Elad  Ziv</span>, <span class="hlFld-ContribAuthor ">Kristin K  Zorn</span>, <span class="hlFld-ContribAuthor ">Manuela  Gago-Dominguez</span>, <span class="hlFld-ContribAuthor ">Arto  Mannermaa</span>, <span class="hlFld-ContribAuthor ">Håkan  Olsson</span>, <span class="hlFld-ContribAuthor ">Manuel R  Teixeira</span>, <span class="hlFld-ContribAuthor ">Jennifer  Stone</span>, <span class="hlFld-ContribAuthor ">Kenneth  Offit</span>, <span class="hlFld-ContribAuthor ">Laura  Ottini</span>, <span class="hlFld-ContribAuthor ">Sue K  Park</span>, <span class="hlFld-ContribAuthor ">Mads  Thomassen</span>, <span class="hlFld-ContribAuthor ">Per  Hall</span>, <span class="hlFld-ContribAuthor ">Alfons  Meindl</span>, <span class="hlFld-ContribAuthor ">Rita K  Schmutzler</span>, <span class="hlFld-ContribAuthor ">Arnaud  Droit</span>, <span class="hlFld-ContribAuthor ">Gary D  Bader</span>, <span class="hlFld-ContribAuthor ">Paul D P  Pharoah</span>, <span class="hlFld-ContribAuthor ">Fergus J  Couch</span>, <span class="hlFld-ContribAuthor ">Douglas F  Easton</span>, <span class="hlFld-ContribAuthor ">Peter  Kraft</span>, <span class="hlFld-ContribAuthor ">Georgia  Chenevix-Trench</span>, <span class="hlFld-ContribAuthor ">Montserrat  García-Closas</span>, <span class="hlFld-ContribAuthor ">Marjanka K  Schmidt</span>, <span class="hlFld-ContribAuthor ">Antonis C  Antoniou</span>, <span class="hlFld-ContribAuthor ">Jacques  Simard</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of ten variants associated with risk of estrogen-receptor-negative breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Genetics</span><span> <strong>2017,</strong> <em>49 </em>
                                    (12)
                                     , 1767-1778. <a href="https://doi.org/10.1038/ng.3785" title="DOI URL">https://doi.org/10.1038/ng.3785</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/ng.3785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fng.3785%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Genetics%26atitle%3DIdentification%252Bof%252Bten%252Bvariants%252Bassociated%252Bwith%252Brisk%252Bof%252Bestrogen-receptor-negative%252Bbreast%252Bcancer%26aulast%3DMilne%26aufirst%3DRoger%2BL%26date%3D2017%26date%3D2017%26volume%3D49%26issue%3D12%26spage%3D1767%26epage%3D1778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kelly A.  Teske</span>, <span class="hlFld-ContribAuthor ">M. Kyle  Hadden</span>. </span><span class="cited-content_cbyCitation_article-title">Methyllysine binding domains: Structural insight and small molecule probe development. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>136 </em>, 14-35. <a href="https://doi.org/10.1016/j.ejmech.2017.04.047" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.04.047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.04.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.04.047%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMethyllysine%252Bbinding%252Bdomains%25253A%252BStructural%252Binsight%252Band%252Bsmall%252Bmolecule%252Bprobe%252Bdevelopment%26aulast%3DTeske%26aufirst%3DKelly%2BA.%26date%3D2017%26volume%3D136%26spage%3D14%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Holger  Greschik</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Thomas  Günther</span>. </span><span class="cited-content_cbyCitation_article-title">Selective targeting of epigenetic reader domains. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 449-463. <a href="https://doi.org/10.1080/17460441.2017.1303474" title="DOI URL">https://doi.org/10.1080/17460441.2017.1303474</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2017.1303474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2017.1303474%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DSelective%252Btargeting%252Bof%252Bepigenetic%252Breader%252Bdomains%26aulast%3DGreschik%26aufirst%3DHolger%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D449%26epage%3D463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jake  Shortt</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Ott</span>, <span class="hlFld-ContribAuthor ">Ricky W.  Johnstone</span>, <span class="hlFld-ContribAuthor ">James E.  Bradner</span>. </span><span class="cited-content_cbyCitation_article-title">A chemical probe toolbox for dissecting the cancer epigenome. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2017,</strong> <em>17 </em>
                                    (3)
                                     , 160-183. <a href="https://doi.org/10.1038/nrc.2016.148" title="DOI URL">https://doi.org/10.1038/nrc.2016.148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrc.2016.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrc.2016.148%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DA%252Bchemical%252Bprobe%252Btoolbox%252Bfor%252Bdissecting%252Bthe%252Bcancer%252Bepigenome%26aulast%3DShortt%26aufirst%3DJake%26date%3D2017%26date%3D2017%26volume%3D17%26issue%3D3%26spage%3D160%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">John E.  Campbell</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Modulators. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 227-227. <a href="https://doi.org/10.1007/7355_2017_30" title="DOI URL">https://doi.org/10.1007/7355_2017_30</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2017_30&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2017_30%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BModulators%26aulast%3DDuncan%26aufirst%3DKenneth%2BW.%26date%3D2017%26spage%3D227%26epage%3D227%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DCancer%252BII%26aulast%3DWaring%26aufirst%3DMichael%2BJ.%26date%3D2018%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.M.  Woster</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Therapeutics Targeting Epigenetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 297-328. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12325-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12325-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12325-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12325-X%26sid%3Dliteratum%253Aachs%26atitle%3DNovel%252BTherapeutics%252BTargeting%252BEpigenetics%26aulast%3DWoster%26aufirst%3DP.M.%26date%3D2017%26spage%3D297%26epage%3D328%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Tobias  Wagner</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Luca  Carlino</span>, <span class="hlFld-ContribAuthor ">Joel  McMillan</span>, <span class="hlFld-ContribAuthor ">Ralf  Flaig</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Structure-Activity Relationship Studies of Small-Molecule Inhibitors of the Methyllysine Reader Protein Spindlin1. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (20)
                                     , 2327-2338. <a href="https://doi.org/10.1002/cmdc.201600362" title="DOI URL">https://doi.org/10.1002/cmdc.201600362</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201600362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201600362%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DIdentification%252Band%252BStructure-Activity%252BRelationship%252BStudies%252Bof%252BSmall-Molecule%252BInhibitors%252Bof%252Bthe%252BMethyllysine%252BReader%252BProtein%252BSpindlin1%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D20%26spage%3D2327%26epage%3D2338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia  Milosevich</span>, <span class="hlFld-ContribAuthor ">Zoey  Warmerdam</span>, <span class="hlFld-ContribAuthor ">Fraser  Hof</span>. </span><span class="cited-content_cbyCitation_article-title">Structural aspects of small-molecule inhibition of methyllysine reader proteins. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (13)
                                     , 1681-1702. <a href="https://doi.org/10.4155/fmc-2016-0082" title="DOI URL">https://doi.org/10.4155/fmc-2016-0082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0082%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DStructural%252Baspects%252Bof%252Bsmall-molecule%252Binhibition%252Bof%252Bmethyllysine%252Breader%252Bproteins%26aulast%3DMilosevich%26aufirst%3DNatalia%26date%3D2016%26volume%3D8%26issue%3D13%26spage%3D1681%26epage%3D1702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">R  Kumar</span>, <span class="hlFld-ContribAuthor ">D-Q  Li</span>, <span class="hlFld-ContribAuthor ">S  Müller</span>, <span class="hlFld-ContribAuthor ">S  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenomic regulation of oncogenesis by chromatin remodeling. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2016,</strong> <em>35 </em>
                                    (34)
                                     , 4423-4436. <a href="https://doi.org/10.1038/onc.2015.513" title="DOI URL">https://doi.org/10.1038/onc.2015.513</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/onc.2015.513&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fonc.2015.513%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DEpigenomic%252Bregulation%252Bof%252Boncogenesis%252Bby%252Bchromatin%252Bremodeling%26aulast%3DKumar%26aufirst%3DR%26date%3D2016%26date%3D2016%26volume%3D35%26issue%3D34%26spage%3D4423%26epage%3D4436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lindsey I  James</span>, <span class="hlFld-ContribAuthor ">Stephen V  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical probes for methyl lysine reader domains. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2016,</strong> <em>33 </em>, 135-141. <a href="https://doi.org/10.1016/j.cbpa.2016.06.004" title="DOI URL">https://doi.org/10.1016/j.cbpa.2016.06.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2016.06.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2016.06.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DChemical%252Bprobes%252Bfor%252Bmethyl%252Blysine%252Breader%252Bdomains%26aulast%3DJames%26aufirst%3DLindsey%2BI%26date%3D2016%26volume%3D33%26spage%3D135%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin E.  Paluch</span>, <span class="hlFld-ContribAuthor ">Abdul R.  Naqash</span>, <span class="hlFld-ContribAuthor ">Zachary  Brumberger</span>, <span class="hlFld-ContribAuthor ">Michael J.  Nemeth</span>, <span class="hlFld-ContribAuthor ">Elizabeth A.  Griffiths</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics: A primer for clinicians. </span><span class="cited-content_cbyCitation_journal-name">Blood Reviews</span><span> <strong>2016,</strong> <em>30 </em>
                                    (4)
                                     , 285-295. <a href="https://doi.org/10.1016/j.blre.2016.02.002" title="DOI URL">https://doi.org/10.1016/j.blre.2016.02.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.blre.2016.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.blre.2016.02.002%26sid%3Dliteratum%253Aachs%26jtitle%3DBlood%2520Reviews%26atitle%3DEpigenetics%25253A%252BA%252Bprimer%252Bfor%252Bclinicians%26aulast%3DPaluch%26aufirst%3DBenjamin%2BE.%26date%3D2016%26volume%3D30%26issue%3D4%26spage%3D285%26epage%3D295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tobias  Wagner</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Readers Interpreting the Lysine Methylome—Biological Roles and Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 273-304. <a href="https://doi.org/10.1016/B978-0-12-802208-5.00011-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-802208-5.00011-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802208-5.00011-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802208-5.00011-4%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BReaders%252BInterpreting%252Bthe%252BLysine%252BMethylome%2525E2%252580%252594Biological%252BRoles%252Band%252BDrug%252BDiscovery%26aulast%3DWagner%26aufirst%3DTobias%26date%3D2016%26spage%3D273%26epage%3D304%26pub%3DElsevier%26atitle%3DDrug%252BDiscovery%252Bin%252BCancer%252BEpigenetics%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William J.  Zuercher</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Elkins</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">The Intersection of Structural and Chemical Biology - An Essential Synergy. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2016,</strong> <em>23 </em>
                                    (1)
                                     , 173-182. <a href="https://doi.org/10.1016/j.chembiol.2015.12.005" title="DOI URL">https://doi.org/10.1016/j.chembiol.2015.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2015.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2015.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DThe%252BIntersection%252Bof%252BStructural%252Band%252BChemical%252BBiology%252B-%252BAn%252BEssential%252BSynergy%26aulast%3DZuercher%26aufirst%3DWilliam%25C2%25A0J.%26date%3D2016%26volume%3D23%26issue%3D1%26spage%3D173%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Ning</span>, <span class="hlFld-ContribAuthor ">Wenyuan  Li</span>, <span class="hlFld-ContribAuthor ">Wei  Zhao</span>, <span class="hlFld-ContribAuthor ">Rongfu  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic regulations in cancer. </span><span class="cited-content_cbyCitation_journal-name">Acta Biochimica et Biophysica Sinica</span><span> <strong>2015,</strong> <em>61 </em>, gmv116. <a href="https://doi.org/10.1093/abbs/gmv116" title="DOI URL">https://doi.org/10.1093/abbs/gmv116</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/abbs/gmv116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fabbs%2Fgmv116%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Biochimica%2520et%2520Biophysica%2520Sinica%26atitle%3DTargeting%252Bepigenetic%252Bregulations%252Bin%252Bcancer%26aulast%3DNing%26aufirst%3DBo%26date%3D2015%26date%3D2015%26volume%3D61%26spage%3Dgmv116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen V  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Unlocking the potential of chemical probes for methyl-lysine reader proteins. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2015,</strong> <em>7 </em>
                                    (14)
                                     , 1831-1833. <a href="https://doi.org/10.4155/fmc.15.119" title="DOI URL">https://doi.org/10.4155/fmc.15.119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.119%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DUnlocking%252Bthe%252Bpotential%252Bof%252Bchemical%252Bprobes%252Bfor%252Bmethyl-lysine%252Breader%252Bproteins%26aulast%3DFrye%26aufirst%3DStephen%2BV%26date%3D2015%26volume%3D7%26issue%3D14%26spage%3D1831%26epage%3D1833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefan  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging Target Families: Intractable Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 43-58. <a href="https://doi.org/10.1007/164_2015_28" title="DOI URL">https://doi.org/10.1007/164_2015_28</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/164_2015_28&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F164_2015_28%26sid%3Dliteratum%253Aachs%26atitle%3DEmerging%252BTarget%252BFamilies%25253A%252BIntractable%252BTargets%26aulast%3DKnapp%26aufirst%3DStefan%26date%3D2015%26spage%3D43%26epage%3D58%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DNew%252BApproaches%252Bto%252BDrug%252BDiscovery%26aulast%3DNielsch%26aufirst%3DUlrich%26date%3D2016%26volume%3D232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Modulation of Methyl-Lysine-Mediated Interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 243-271. <a href="https://doi.org/10.1007/978-3-319-18102-8_12" title="DOI URL">https://doi.org/10.1007/978-3-319-18102-8_12</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-18102-8_12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-18102-8_12%26sid%3Dliteratum%253Aachs%26atitle%3DSmall-Molecule%252BModulation%252Bof%252BMethyl-Lysine-Mediated%252BInteractions%26aulast%3DFrye%26aufirst%3DStephen%2BV.%26date%3D2015%26date%3D2015%26spage%3D243%26epage%3D271%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHistone%252BRecognition%26aulast%3DZhou%26aufirst%3DMing-Ming%26date%3D2015%26volume%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Zhao</span>, <span class="hlFld-ContribAuthor ">Haitao  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Crystallography-Based Mechanistic Insights into Epigenetic Regulation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 125-147. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00007-7" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00007-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00007-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00007-7%26sid%3Dliteratum%253Aachs%26atitle%3DCrystallography-Based%252BMechanistic%252BInsights%252Binto%252BEpigenetic%252BRegulation%26aulast%3DZhao%26aufirst%3DShuai%26date%3D2015%26spage%3D125%26epage%3D147%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dashyant  Dhanak</span>, <span class="hlFld-ContribAuthor ">Paul  Jackson</span>. </span><span class="cited-content_cbyCitation_article-title">Development and classes of epigenetic drugs for cancer. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2014,</strong> <em>455 </em>
                                    (1-2)
                                     , 58-69. <a href="https://doi.org/10.1016/j.bbrc.2014.07.006" title="DOI URL">https://doi.org/10.1016/j.bbrc.2014.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2014.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2014.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DDevelopment%252Band%252Bclasses%252Bof%252Bepigenetic%252Bdrugs%252Bfor%252Bcancer%26aulast%3DDhanak%26aufirst%3DDashyant%26date%3D2014%26volume%3D455%26issue%3D1-2%26spage%3D58%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Schnekenburger</span>, <span class="hlFld-ContribAuthor ">Mario  Dicato</span>, <span class="hlFld-ContribAuthor ">Marc  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic modulators from “The Big Blue”: A treasure to fight against cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2014,</strong> <em>351 </em>
                                    (2)
                                     , 182-197. <a href="https://doi.org/10.1016/j.canlet.2014.06.005" title="DOI URL">https://doi.org/10.1016/j.canlet.2014.06.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2014.06.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DEpigenetic%252Bmodulators%252Bfrom%252B%2525E2%252580%25259CThe%252BBig%252BBlue%2525E2%252580%25259D%25253A%252BA%252Btreasure%252Bto%252Bfight%252Bagainst%252Bcancer%26aulast%3DSchnekenburger%26aufirst%3DMichael%26date%3D2014%26volume%3D351%26issue%3D2%26spage%3D182%26epage%3D197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanli  Liu</span>, <span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Su  Qin</span>, <span class="hlFld-ContribAuthor ">Chao  Xu</span>, <span class="hlFld-ContribAuthor ">Jinrong  Min</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic targets and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2014,</strong> <em>143 </em>
                                    (3)
                                     , 275-294. <a href="https://doi.org/10.1016/j.pharmthera.2014.03.007" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2014.03.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2014.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2014.03.007%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DEpigenetic%252Btargets%252Band%252Bdrug%252Bdiscovery%26aulast%3DLiu%26aufirst%3DYanli%26date%3D2014%26volume%3D143%26issue%3D3%26spage%3D275%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sitaram  Gayatri</span>, <span class="hlFld-ContribAuthor ">Mark T.  Bedford</span>. </span><span class="cited-content_cbyCitation_article-title">Readers of histone methylarginine marks. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span> <strong>2014,</strong> <em>1839 </em>
                                    (8)
                                     , 702-710. <a href="https://doi.org/10.1016/j.bbagrm.2014.02.015" title="DOI URL">https://doi.org/10.1016/j.bbagrm.2014.02.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagrm.2014.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagrm.2014.02.015%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Gene%2520Regulatory%2520Mechanisms%26atitle%3DReaders%252Bof%252Bhistone%252Bmethylarginine%252Bmarks%26aulast%3DGayatri%26aufirst%3DSitaram%26date%3D2014%26volume%3D1839%26issue%3D8%26spage%3D702%26epage%3D710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine A.  Musselman</span>, <span class="hlFld-ContribAuthor ">Sepideh  Khorasanizadeh</span>, <span class="hlFld-ContribAuthor ">Tatiana G.  Kutateladze</span>. </span><span class="cited-content_cbyCitation_article-title">Towards understanding methyllysine readout. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span> <strong>2014,</strong> <em>1839 </em>
                                    (8)
                                     , 686-693. <a href="https://doi.org/10.1016/j.bbagrm.2014.04.001" title="DOI URL">https://doi.org/10.1016/j.bbagrm.2014.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagrm.2014.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagrm.2014.04.001%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Gene%2520Regulatory%2520Mechanisms%26atitle%3DTowards%252Bunderstanding%252Bmethyllysine%252Breadout%26aulast%3DMusselman%26aufirst%3DCatherine%2BA.%26date%3D2014%26volume%3D1839%26issue%3D8%26spage%3D686%26epage%3D693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tobias  Wagner</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Mind the Methyl: Methyllysine Binding Proteins in Epigenetic Regulation. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2014,</strong> <em>9 </em>
                                    (3)
                                     , 466-483. <a href="https://doi.org/10.1002/cmdc.201300422" title="DOI URL">https://doi.org/10.1002/cmdc.201300422</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201300422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201300422%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DMind%252Bthe%252BMethyl%25253A%252BMethyllysine%252BBinding%252BProteins%252Bin%252BEpigenetic%252BRegulation%26aulast%3DWagner%26aufirst%3DTobias%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D3%26spage%3D466%26epage%3D483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Boobalan  Pachaiyappan</span>, <span class="hlFld-ContribAuthor ">Patrick M.  Woster</span>. </span><span class="cited-content_cbyCitation_article-title">Design of small molecule epigenetic modulators. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (1)
                                     , 21-32. <a href="https://doi.org/10.1016/j.bmcl.2013.11.001" title="DOI URL">https://doi.org/10.1016/j.bmcl.2013.11.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2013.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2013.11.001%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%252Bof%252Bsmall%252Bmolecule%252Bepigenetic%252Bmodulators%26aulast%3DPachaiyappan%26aufirst%3DBoobalan%26date%3D2014%26volume%3D24%26issue%3D1%26spage%3D21%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Structure of L3MBTL3 chemical probe <b>1</b>. (b) Cocrystal structure of <b>1</b> (green) bound to two molecules of L3MBTL3. The amine meta to the aniline substituent (domain 2 amine) binds in the Kme binding pocket of MBT domain 2 of one L3MBTL3 molecule (magenta), and the amine ortho to the aniline substituent (domain 1 amine) binds to MBT domain 1 of a second L3MBTL3 molecule (cyan). (c) Structures of other L3MBTL3 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Scatter plot showing the excellent linear correlation between the L3MBTL3 AlphaScreen data and L3MBTL3 TR-FRET data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of monobasic MBT domain inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mero76-<b>1</b> colocalizes with GFP-L3MBTL3 in the nucleus in HEK293 cells (row 1), and compounds <b>1</b>, <b>2</b>, and <b>56</b> (1 equiv relative to mero76-<b>1</b>; rows 2–4) all displace mero76-<b>1</b> from GFP-L3MBTL3 (red is mero76-<b>1</b>, green is GFP-L3MBTL3, and blue is Hoechst). The scale bar is 10 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/medium/jm-2013-00919p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western analysis of biotin-<b>1</b> pulldown experiments in G-401 cell lysates. Biotin-<b>1</b> effectively binds endogenous L3MBTL3. Compounds <b>1</b> and <b>56</b> inhibit this interaction, whereas the inactive control compound, <b>37</b>, cannot successfully block L3MBTL3 binding.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-18/jm400919p/production/images/large/jm-2013-00919p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400919p&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36002" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36002" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Multivalent engagement of chromatin modifications by linked binding modules</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">983</span><span class="NLM_x">–</span> <span class="NLM_lpage">994</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnrm2298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=18037899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=983-994&author=A.+J.+Ruthenburgauthor=H.+Liauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Multivalent+engagement+of+chromatin+modifications+by+linked+binding+modules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Multivalent engagement of chromatin modifications by linked binding modules</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Li, Haitao; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">983-994</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone post-translational modifications (PTMs) play crucial roles in genome management, in part by recruiting specific factors that alter the structural properties of chromatin.  These so-called effector complexes often comprise multiple histone-binding modules that may act in concert to regulate chromatin structure and DNA-related activities.  Various chem. modifications on histones and regions of assocd. DNA play crucial roles in genome management by binding specific factors that, in turn, serve to alter the structural properties of chromatin.  These so-called effector proteins have typically been studied with the biochemist's paring knife; the capacity to recognize specific chromatin PTMs has been mapped to an increasing no. of domains that frequently appear in the nuclear subset of the proteome, often present in large, multisubunit complexes that bristle with modification-dependent binding potential.  The authors propose that multivalent interactions on a single histone tail and beyond may have a significant, if not dominant, role in chromatin transactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkIrlnyvt64LVg90H21EOLACvtfcHk0ljiSaLHuaY35Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrsb3L&md5=0d27fc9ba2e2a009e0a9903e1c65bdf7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm2298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2298%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DMultivalent%2520engagement%2520of%2520chromatin%2520modifications%2520by%2520linked%2520binding%2520modules%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2007%26volume%3D8%26spage%3D983%26epage%3D994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0liKC0srn_FSJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discov.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">384</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0liKC0srn_FSJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The epigenomics of cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0liKC0srn_FSJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Whyte, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlando, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hnisz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagey, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span> </span><span class="NLM_article-title">Master transcription factors and mediator establish super-enhancers at key cell identity genes</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">153</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">319</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=307-319&author=W.+A.+Whyteauthor=D.+A.+Orlandoauthor=D.+Hniszauthor=B.+J.+Abrahamauthor=C.+Y.+Linauthor=M.+H.+Kageyauthor=P.+B.+Rahlauthor=T.+I.+Leeauthor=R.+A.+Young&title=Master+transcription+factors+and+mediator+establish+super-enhancers+at+key+cell+identity+genes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhyte%26aufirst%3DW.%2BA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DKagey%26aufirst%3DM.%2BH.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DMaster%2520transcription%2520factors%2520and%2520mediator%2520establish%2520super-enhancers%2520at%2520key%2520cell%2520identity%2520genes%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D307%26epage%3D319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">The art of the chemical probe</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">159</span><span class="NLM_x">–</span> <span class="NLM_lpage">161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=20154659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=159-161&author=S.+V.+Frye&title=The+art+of+the+chemical+probe"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The art of the chemical probe</span></div><div class="casAuthors">Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">159-161</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. biologists frequently aim to create small-mol. probes that interact with a specific protein in vitro in order to explore the role of the protein in a broader biol. context (cells or organisms), but a common understanding of what makes a high-quality probe is lacking.  Here I propose a set of principles to guide probe qualification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7vDfsTLhqWrVg90H21EOLACvtfcHk0lhJjhPYVreyiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbg%253D&md5=b7b48c591a611b8686e11b3fc4f1b85c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.296%26sid%3Dliteratum%253Aachs%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DThe%2520art%2520of%2520the%2520chemical%2520probe%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D159%26epage%3D161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span>Structural Genomics Consortium. <a href="http://www.thesgc.org/scientists/chemical_probes" class="extLink">http://www.thesgc.org/scientists/chemical_probes</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Structural+Genomics+Consortium.+http%3A%2F%2Fwww.thesgc.org%2Fscientists%2Fchemical_probes."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Small-molecular modulators of cancer-associated epigenetic mechanisms</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1039%2Fc3mb25410k" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23511667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=873-896&author=Y.+Itohauthor=T.+Suzukiauthor=N.+Miyata&title=Small-molecular+modulators+of+cancer-associated+epigenetic+mechanisms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecular modulators of cancer-associated epigenetic mechanisms</span></div><div class="casAuthors">Itoh, Yukihiro; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">873-896</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  DNA methylation and posttranslational histone modifications regulate expression of various genes independently of changes in the DNA sequence.  Such epigenetic mechanisms play important roles in controlling cellular functions, including the cell cycle, immunoresponses and signal transduction.  On the other hand, epigenetic aberrations are assocd. with oncogenesis and proliferation of cancer cells, and epigenetic alterations have been identified in many human cancer cells.  Furthermore, chem.-biol. approaches have uncovered relationships between epigenetic dysregulation and cancer, and several small mols. that modulate epigenetic mechanisms have already been approved for cancer therapy.  In this review, we deal with chem. epigenetics relevant to cancer therapy, focusing esp. on small mols. that regulate epigenetic mechanisms related to DNA methylation and histone modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo76xFXo9kCe7Vg90H21EOLACvtfcHk0lhJjhPYVreyiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVOlsr4%253D&md5=ecc7cd46637f6010872510329d51ac90</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1039%2Fc3mb25410k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3mb25410k%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSmall-molecular%2520modulators%2520of%2520cancer-associated%2520epigenetic%2520mechanisms%26jtitle%3DMol.%2520BioSyst.%26date%3D2013%26volume%3D9%26spage%3D873%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.+C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.+P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lhJjhPYVreyiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lj8erTK3vXaKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Della Pietra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+Della+Pietraauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+R.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDella%2BPietra%26aufirst%3DA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%2BR.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of potent, selective, cell-active inhibitors of the histone lysine methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span></span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+potent%2C+selective%2C+cell-active+inhibitors+of+the+histone+lysine+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0lj8erTK3vXaKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520potent%252C%2520selective%252C%2520cell-active%2520inhibitors%2520of%2520the%2520histone%2520lysine%2520methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Qi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chuai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardayfio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span> </span><span class="NLM_article-title">Selective inhibition of Ezh2 by a small molecule inhibitor blocks tumor cells proliferation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">21360</span><span class="NLM_x">–</span> <span class="NLM_lpage">21365</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=21360-21365&author=W.+Qiauthor=H.+Chanauthor=L.+Tengauthor=L.+Liauthor=S.+Chuaiauthor=R.+Zhangauthor=J.+Zengauthor=M.+Liauthor=H.+Fanauthor=Y.+Linauthor=J.+Guauthor=O.+Ardayfioauthor=J.+H.+Zhangauthor=X.+Yanauthor=J.+Fangauthor=Y.+Miauthor=M.+Zhangauthor=T.+Zhouauthor=G.+Fengauthor=Z.+Chenauthor=G.+Liauthor=T.+Yangauthor=K.+Zhaoauthor=X.+Liuauthor=Z.+Yuauthor=C.+X.+Luauthor=P.+Atadjaauthor=E.+Li&title=Selective+inhibition+of+Ezh2+by+a+small+molecule+inhibitor+blocks+tumor+cells+proliferation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW.%26aulast%3DChan%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChuai%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DArdayfio%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DYan%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DC.%2BX.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DSelective%2520inhibition%2520of%2520Ezh2%2520by%2520a%2520small%2520molecule%2520inhibitor%2520blocks%2520tumor%2520cells%2520proliferation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D21360%26epage%3D21365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macnevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the lysine methyltransferases EZH2 and EZH1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1324</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span></span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+Macnevinauthor=F.+Liuauthor=C.+Gaoauthor=X.+P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+lysine+methyltransferases+EZH2+and+EZH1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0lgouyDBoanULw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacnevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7922</span><span class="NLM_x">–</span> <span class="NLM_lpage">7927</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0lj0i7hJE8n3eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Beguelin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popovic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teater, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezponda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez-Garcia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaknovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoyne, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerchietti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdel-Wahab, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Licht, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A. M.</span><span> </span><span class="NLM_article-title">EZH2 is required for germinal center formation and somatic EZH2 mutations promote lymphoid transformation</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=677-692&author=W.+Beguelinauthor=R.+Popovicauthor=M.+Teaterauthor=Y.+Jiangauthor=K.+L.+Buntingauthor=M.+Rosenauthor=H.+Shenauthor=S.+N.+Yangauthor=L.+Wangauthor=T.+Ezpondaauthor=E.+Martinez-Garciaauthor=H.+Zhangauthor=Y.+Zhengauthor=S.+K.+Vermaauthor=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+S.+Van+Allerauthor=R.+G.+Krugerauthor=Y.+Liuauthor=C.+F.+McHughauthor=D.+W.+Scottauthor=Y.+R.+Chungauthor=N.+Kelleherauthor=R.+Shaknovichauthor=C.+L.+Creasyauthor=R.+D.+Gascoyneauthor=K.+K.+Wongauthor=L.+Cerchiettiauthor=R.+L.+Levineauthor=O.+Abdel-Wahabauthor=J.+D.+Lichtauthor=O.+Elementoauthor=A.+M.+Melnick&title=EZH2+is+required+for+germinal+center+formation+and+somatic+EZH2+mutations+promote+lymphoid+transformation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeguelin%26aufirst%3DW.%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DTeater%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DRosen%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DEzponda%26aufirst%3DT.%26aulast%3DMartinez-Garcia%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DScott%26aufirst%3DD.%2BW.%26aulast%3DChung%26aufirst%3DY.%2BR.%26aulast%3DKelleher%26aufirst%3DN.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DCerchietti%26aufirst%3DL.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26atitle%3DEZH2%2520is%2520required%2520for%2520germinal%2520center%2520formation%2520and%2520somatic%2520EZH2%2520mutations%2520promote%2520lymphoid%2520transformation%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D677%26epage%3D692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chory, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wernimont, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scopton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federation, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wienholds, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fncomms2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23250418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1288&author=W.+Yuauthor=E.+J.+Choryauthor=A.+K.+Wernimontauthor=W.+Tempelauthor=A.+Scoptonauthor=A.+Federationauthor=J.+J.+Marineauauthor=J.+Qiauthor=D.+Barsyte-Lovejoyauthor=J.+Yiauthor=R.+Marcellusauthor=R.+E.+Iacobauthor=J.+R.+Engenauthor=C.+Griffinauthor=A.+Amanauthor=E.+Wienholdsauthor=F.+Liauthor=J.+Pinedaauthor=G.+Estiuauthor=T.+Shatsevaauthor=T.+Hajianauthor=R.+Al-Awarauthor=J.+E.+Dickauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+E.+Bradnerauthor=M.+Schapira&title=Catalytic+site+remodelling+of+the+DOT1L+methyltransferase+by+selective+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span></div><div class="casAuthors">Yu Wenyu; Chory Emma J; Wernimont Amy K; Tempel Wolfram; Scopton Alex; Federation Alexander; Marineau Jason J; Qi Jun; Barsyte-Lovejoy Dalia; Yi Joanna; Marcellus Richard; Iacob Roxana E; Engen John R; Griffin Carly; Aman Ahmed; Wienholds Erno; Li Fengling; Pineda Javier; Estiu Guillermina; Shatseva Tatiana; Hajian Taraneh; Al-Awar Rima; Dick John E; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Bradner James E; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1288</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Selective inhibition of protein methyltransferases is a promising new approach to drug discovery.  An attractive strategy towards this goal is the development of compounds that selectively inhibit binding of the cofactor, S-adenosylmethionine, within specific protein methyltransferases.  Here we report the three-dimensional structure of the protein methyltransferase DOT1L bound to EPZ004777, the first S-adenosylmethionine-competitive inhibitor of a protein methyltransferase with in vivo efficacy.  This structure and those of four new analogues reveal remodelling of the catalytic site.  EPZ004777 and a brominated analogue, SGC0946, inhibit DOT1L in vitro and selectively kill mixed lineage leukaemia cells, in which DOT1L is aberrantly localized via interaction with an oncogenic MLL fusion protein.  These data provide important new insight into mechanisms of cell-active S-adenosylmethionine-competitive protein methyltransferase inhibitors, and establish a foundation for the further development of drug-like inhibitors of DOT1L for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcST2GJQO5XUmjDnmmyUf5qOfW6udTcc2eYpxWEH4WvLCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D&md5=f7b501ac4188b14b48fc435637e0e192</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fncomms2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2304%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChory%26aufirst%3DE.%2BJ.%26aulast%3DWernimont%26aufirst%3DA.%2BK.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DScopton%26aufirst%3DA.%26aulast%3DFederation%26aufirst%3DA.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DYi%26aufirst%3DJ.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DWienholds%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPineda%26aufirst%3DJ.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DCatalytic%2520site%2520remodelling%2520of%2520the%2520DOT1L%2520methyltransferase%2520by%2520selective%2520inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D1288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0ljhQuUil4EiKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingerman, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Drug discovery toward antagonists of methyl-lysine binding proteins</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">51</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.2174%2F1875397301005010051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=22145013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=51-61&author=J.+M.+Heroldauthor=L.+A.+Ingermanauthor=C.+Gaoauthor=S.+V.+Frye&title=Drug+discovery+toward+antagonists+of+methyl-lysine+binding+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery toward antagonists of methyl-lysine binding proteins</span></div><div class="casAuthors">Herold, J. Martin; Ingerman, Lindsey A.; Gao, Cen; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The recognition of methyl-lysine and -arginine residues on both histone and other proteins by specific "reader" elements is important for chromatin regulation, gene expression, and control of cell-cycle progression.  Recently the crucial role of these reader proteins in cancer development and dedifferentiation has emerged, owing to the increased interest among the scientific community.  The methyl-lysine and -arginine readers are a large and very diverse set of effector proteins and targeting them with small mol. probes in drug discovery will inevitably require a detailed understanding of their structural biol. and mechanism of binding.  In the following review, the crit. elements of methyl-lysine and -arginine recognition will be summarized with respect to each protein family and initial results in assay development, probe design, and drug discovery will be highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-fC8z74GQSLVg90H21EOLACvtfcHk0ljhQuUil4EiKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nN&md5=845f1233f9ea44c2706fc7737ca34c08</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010051%26sid%3Dliteratum%253Aachs%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DIngerman%26aufirst%3DL.%2BA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDrug%2520discovery%2520toward%2520antagonists%2520of%2520methyl-lysine%2520binding%2520proteins%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D51%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Targeting chromatin readers</span> <span class="citation_source-journal">Clin. Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">314</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fclpt.2013.6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23403847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVKrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2013&pages=312-314&author=L.+I.+Jamesauthor=S.+V.+Frye&title=Targeting+chromatin+readers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Chromatin Readers</span></div><div class="casAuthors">James, L. I.; Frye, S. V.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacology & Therapeutics (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-314</span>CODEN:
                <span class="NLM_cas:coden">CLPTAT</span>;
        ISSN:<span class="NLM_cas:issn">0009-9236</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Modulation of gene expression through epigenetic signaling has recently emerged as a novel approach in treating human disease.  Specifically, chromatin reader proteins, which mediate protein-protein interactions via binding to modified lysine residues, are gaining traction as potential therapeutic targets.  Herein, we review recent efforts to understand and modulate the activity of chromatin reader proteins with small-mol. ligands.  Clin. Pharmacol. & Therapeutics (2013); 93 4, 312-314. doi:10.1038/clpt.2013.6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAp554d8hcQLVg90H21EOLACvtfcHk0ljhQuUil4EiKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVKrsbg%253D&md5=4ec2bd0426dba17a52ad7174dc534351</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fclpt.2013.6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fclpt.2013.6%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DTargeting%2520chromatin%2520readers%26jtitle%3DClin.%2520Pharmacol.%2520Ther.%26date%3D2013%26volume%3D93%26spage%3D312%26epage%3D314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krichevsky, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macnevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sagum, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcon, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emili, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">184</span><span class="NLM_x">–</span> <span class="NLM_lpage">191</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnchembio.1157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=23292653" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=184-191&author=L.+I.+Jamesauthor=D.+Barsyte-Lovejoyauthor=N.+Zhongauthor=L.+Krichevskyauthor=V.+K.+Korboukhauthor=J.+M.+Heroldauthor=C.+J.+Macnevinauthor=J.+L.+Norrisauthor=C.+A.+Sagumauthor=W.+Tempelauthor=E.+Marconauthor=H.+Guoauthor=C.+Gaoauthor=X.+P.+Huangauthor=S.+Duanauthor=A.+Emiliauthor=J.+F.+Greenblattauthor=D.+B.+Kireevauthor=J.+Jinauthor=W.+P.+Janzenauthor=P.+J.+Brownauthor=M.+T.+Bedfordauthor=C.+H.+Arrowsmithauthor=S.+V.+Frye&title=Discovery+of+a+chemical+probe+for+the+L3MBTL3+methyllysine+reader+domain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a chemical probe for the L3MBTL3 methyllysine reader domain</span></div><div class="casAuthors">James, Lindsey I.; Barsyte-Lovejoy, Dalia; Zhong, Nan; Krichevsky, Liubov; Korboukh, Victoria K.; Herold, J. Martin; MacNevin, Christopher J.; Norris, Jacqueline L.; Sagum, Cari A.; Tempel, Wolfram; Marcon, Edyta; Guo, Hongbo; Gao, Cen; Huang, Xi-Ping; Duan, Shili; Emili, Andrew; Greenblatt, Jack F.; Kireev, Dmitri B.; Jin, Jian; Janzen, William P.; Brown, Peter J.; Bedford, Mark T.; Arrowsmith, Cheryl H.; Frye, Stephen V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">184-191</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe the discovery of UNC1215, a potent and selective chem. probe for the methyllysine (Kme) reading function of L3MBTL3, a member of the malignant brain tumor (MBT) family of chromatin-interacting transcriptional repressors.  UNC1215 binds L3MBTL3 with a Kd of 120 nM, competitively displacing mono- or dimethyllysine-contg. peptides, and is greater than 50-fold more potent toward L3MBTL3 than other members of the MBT family while also demonstrating selectivity against more than 200 other reader domains examd.  X-ray crystallog. identified a unique 2:2 polyvalent mode of interaction between UNC1215 and L3MBTL3.  In cells, UNC1215 is nontoxic and directly binds L3MBTL3 via the Kme-binding pocket of the MBT domains.  UNC1215 increases the cellular mobility of GFP-L3MBTL3 fusion proteins, and point mutants that disrupt the Kme-binding function of GFP-L3MBTL3 phenocopy the effects of UNC1215 on localization.  Finally, UNC1215 was used to reveal a new Kme-dependent interaction of L3MBTL3 with BCLAF1 (BCL2-assocd. transcription factor 1), a protein implicated in DNA damage repair and apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqta8WQQkm04rVg90H21EOLACvtfcHk0lj43kgtd87bTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjslGjsw%253D%253D&md5=ea8779ddceed07b5e4c822f2eb888acb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1157%26sid%3Dliteratum%253Aachs%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DN.%26aulast%3DKrichevsky%26aufirst%3DL.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DMacnevin%26aufirst%3DC.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DSagum%26aufirst%3DC.%2BA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DMarcon%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DDuan%26aufirst%3DS.%26aulast%3DEmili%26aufirst%3DA.%26aulast%3DGreenblatt%26aufirst%3DJ.%2BF.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DDiscovery%2520of%2520a%2520chemical%2520probe%2520for%2520the%2520L3MBTL3%2520methyllysine%2520reader%2520domain%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D184%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Bonasio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lecona, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">MBT domain proteins in development and disease</span> <span class="citation_source-journal">Semin. Cell Dev. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">221</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=221-230&author=R.+Bonasioauthor=E.+Leconaauthor=D.+Reinberg&title=MBT+domain+proteins+in+development+and+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBonasio%26aufirst%3DR.%26aulast%3DLecona%26aufirst%3DE.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DMBT%2520domain%2520proteins%2520in%2520development%2520and%2520disease%26jtitle%3DSemin.%2520Cell%2520Dev.%2520Biol.%26date%3D2010%26volume%3D21%26spage%3D221%26epage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sims, R. J.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaquero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalakonda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boccuni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimer, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">L3MBTL1, a histone-methylation-dependent chromatin lock</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">915</span><span class="NLM_x">–</span> <span class="NLM_lpage">928</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=915-928&author=P.+Trojerauthor=G.+Liauthor=R.+J.+Simsauthor=A.+Vaqueroauthor=N.+Kalakondaauthor=P.+Boccuniauthor=D.+Leeauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+D.+Nimerauthor=Y.+H.+Wangauthor=D.+Reinberg&title=L3MBTL1%2C+a+histone-methylation-dependent+chromatin+lock"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DVaquero%26aufirst%3DA.%26aulast%3DKalakonda%26aufirst%3DN.%26aulast%3DBoccuni%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DD.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DL3MBTL1%252C%2520a%2520histone-methylation-dependent%2520chromatin%2520lock%26jtitle%3DCell%26date%3D2007%26volume%3D129%26spage%3D915%26epage%3D928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Arai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyazaki, T.</span><span> </span><span class="NLM_article-title">Impaired maturation of myeloid progenitors in mice lacking novel polycomb group protein MBT-1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1863</span><span class="NLM_x">–</span> <span class="NLM_lpage">1873</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=1863-1873&author=S.+Araiauthor=T.+Miyazaki&title=Impaired+maturation+of+myeloid+progenitors+in+mice+lacking+novel+polycomb+group+protein+MBT-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DArai%26aufirst%3DS.%26aulast%3DMiyazaki%26aufirst%3DT.%26atitle%3DImpaired%2520maturation%2520of%2520myeloid%2520progenitors%2520in%2520mice%2520lacking%2520novel%2520polycomb%2520group%2520protein%2520MBT-1%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D1863%26epage%3D1873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Botuyan, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mer, G.</span><span> </span><span class="NLM_article-title">Structural basis for the methylation state-specific recognition of histone H4-K20 by 53BP1 and Crb2 in DNA repair</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">1361</span><span class="NLM_x">–</span> <span class="NLM_lpage">1373</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2006&pages=1361-1373&author=M.+V.+Botuyanauthor=J.+Leeauthor=I.+M.+Wardauthor=J.+E.+Kimauthor=J.+R.+Thompsonauthor=J.+Chenauthor=G.+Mer&title=Structural+basis+for+the+methylation+state-specific+recognition+of+histone+H4-K20+by+53BP1+and+Crb2+in+DNA+repair"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBotuyan%26aufirst%3DM.%2BV.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DWard%26aufirst%3DI.%2BM.%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DThompson%26aufirst%3DJ.%2BR.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMer%26aufirst%3DG.%26atitle%3DStructural%2520basis%2520for%2520the%2520methylation%2520state-specific%2520recognition%2520of%2520histone%2520H4-K20%2520by%252053BP1%2520and%2520Crb2%2520in%2520DNA%2520repair%26jtitle%3DCell%26date%3D2006%26volume%3D127%26spage%3D1361%26epage%3D1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Roy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musselman, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kachirskaia, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayashi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glass, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nix, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appella, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutateladze, T. G.</span><span> </span><span class="NLM_article-title">Structural insight into p53 recognition by the 53BP1 tandem tudor domain</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">398</span><span class="NLM_x">, </span> <span class="NLM_fpage">489</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=398&publication_year=2010&pages=489-496&author=S.+Royauthor=C.+A.+Musselmanauthor=I.+Kachirskaiaauthor=R.+Hayashiauthor=K.+C.+Glassauthor=J.+C.+Nixauthor=O.+Gozaniauthor=E.+Appellaauthor=T.+G.+Kutateladze&title=Structural+insight+into+p53+recognition+by+the+53BP1+tandem+tudor+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DMusselman%26aufirst%3DC.%2BA.%26aulast%3DKachirskaia%26aufirst%3DI.%26aulast%3DHayashi%26aufirst%3DR.%26aulast%3DGlass%26aufirst%3DK.%2BC.%26aulast%3DNix%26aufirst%3DJ.%2BC.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DAppella%26aufirst%3DE.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26atitle%3DStructural%2520insight%2520into%2520p53%2520recognition%2520by%2520the%252053BP1%2520tandem%2520tudor%2520domain%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D398%26spage%3D489%26epage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coil, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bua, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Structure–activity relationships of methyl-lysine reader antagonists</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">45</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=45-51&author=J.+M.+Heroldauthor=L.+I.+Jamesauthor=V.+K.+Korboukhauthor=C.+Gaoauthor=K.+E.+Coilauthor=D.+J.+Buaauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=P.+J.+Brownauthor=J.+Jinauthor=W.+P.+Janzenauthor=O.+Gozaniauthor=S.+V.+Frye&title=Structure%E2%80%93activity+relationships+of+methyl-lysine+reader+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DCoil%26aufirst%3DK.%2BE.%26aulast%3DBua%26aufirst%3DD.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DStructure%25E2%2580%2593activity%2520relationships%2520of%2520methyl-lysine%2520reader%2520antagonists%26jtitle%3DMedChemComm%26date%3D2012%26volume%3D3%26spage%3D45%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ingerman, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Small-molecule ligands of methyl-lysine binding proteins</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2504</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm200045v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2504-2511&author=J.+M.+Heroldauthor=T.+J.+Wigleauthor=J.+L.+Norrisauthor=R.+Lamauthor=V.+K.+Korboukhauthor=C.+Gaoauthor=L.+A.+Ingermanauthor=D.+B.+Kireevauthor=G.+Senisterraauthor=M.+Vedadiauthor=A.+Tripathyauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+Jinauthor=W.+P.+Janzenauthor=S.+V.+Frye&title=Small-molecule+ligands+of+methyl-lysine+binding+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm200045v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200045v%26sid%3Dliteratum%253Aachs%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DLam%26aufirst%3DR.%26aulast%3DKorboukh%26aufirst%3DV.%2BK.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DIngerman%26aufirst%3DL.%2BA.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DSmall-molecule%2520ligands%2520of%2520methyl-lysine%2520binding%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2504%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span> </span><span class="NLM_article-title">Screening for inhibitors of low-affinity epigenetic peptide-protein interactions: an AlphaScreen-based assay for antagonists of methyl-lysine binding proteins</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">62</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1177%2F1087057109352902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=20008125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFWmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=62-71&author=T.+J.+Wigleauthor=J.+M.+Heroldauthor=G.+A.+Senisterraauthor=M.+Vedadiauthor=D.+B.+Kireevauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=W.+P.+Janzen&title=Screening+for+inhibitors+of+low-affinity+epigenetic+peptide-protein+interactions%3A+an+AlphaScreen-based+assay+for+antagonists+of+methyl-lysine+binding+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Screening for inhibitors of low-affinity epigenetic peptide-protein interactions: an AlphaScreen-based assay for antagonists of methyl-lysine binding proteins</span></div><div class="casAuthors">Wigle, Tim J.; Herold, J. Martin; Senisterra, Guillermo A.; Vedadi, Masoud; Kireev, Dmitri B.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Janzen, William P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-71</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The histone code comprises many posttranslational modifications that occur mainly in histone tail peptides.  The identity and location of these marks are read by a variety of histone-binding proteins that are emerging as important regulators of cellular differentiation and development and are increasingly being implicated in numerous disease states.  The authors describe the development of the first high-throughput screening assay for the discovery of inhibitors of methyl-lysine binding proteins that will be used to initiate a full-scale discovery effort for this broad target class.  They focus on the development of an AlphaScreen-based assay for malignant brain tumor (MBT) domain-contg. proteins, which bind to the lower methylation states of lysine residues present in histone tail peptides.  This assay takes advantage of the avidity of the AlphaScreen beads to clear the hurdle to assay development presented by the low micromolar binding consts. of the histone binding proteins for their cognate peptides.  The assay is applicable to other families of methyl-lysine binding proteins, and it has the potential to be used in screening efforts toward the discovery of novel small mols. with utility as research tools for cellular reprogramming and ultimately drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKUKuGjBHvdLVg90H21EOLACvtfcHk0ljG72VGhPHTdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFWmu7s%253D&md5=ff7396785ae549d1e0d2e7197d6c34e7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1177%2F1087057109352902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109352902%26sid%3Dliteratum%253Aachs%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26atitle%3DScreening%2520for%2520inhibitors%2520of%2520low-affinity%2520epigenetic%2520peptide-protein%2520interactions%253A%2520an%2520AlphaScreen-based%2520assay%2520for%2520antagonists%2520of%2520methyl-lysine%2520binding%2520proteins%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2010%26volume%3D15%26spage%3D62%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormann, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casadio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">459</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">851</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm400919p&amp;key=10.1038%2Fnature08036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm400919p&amp;key=19430464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm400919p&amp;key=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=459&publication_year=2009&pages=847-851&author=G.+G.+Wangauthor=J.+Songauthor=Z.+Wangauthor=H.+L.+Dormannauthor=F.+Casadioauthor=H.+Liauthor=J.+L.+Luoauthor=D.+J.+Patelauthor=C.+D.+Allis&title=Haematopoietic+malignancies+caused+by+dysregulation+of+a+chromatin-binding+PHD+finger"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Haematopoietic malignancies caused by dysregulation of a chromatin-binding PHD finger</span></div><div class="casAuthors">Wang, Gang G.; Song, Jikui; Wang, Zhanxin; Dormann, Holger L.; Casadio, Fabio; Li, Haitao; Luo, Jun-Li; Patel, Dinshaw J.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">459</span>
        (<span class="NLM_cas:issue">7248</span>),
    <span class="NLM_cas:pages">847-851</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 lysine 4 methylation (H3K4me) has been proposed as a crit. component in regulating gene expression, epigenetic states, and cellular identities.  The biol. meaning of H3K4me is interpreted by conserved modules including plant homeodomain (PHD) fingers that recognize varied H3K4me states.  The dysregulation of PHD fingers has been implicated in several human diseases, including cancers and immune or neurol. disorders.  Here we report that fusing an H3K4-trimethylation (H3K4me3)-binding PHD finger, such as the carboxy-terminal PHD finger of PHF23 or JARID1A (also known as KDM5A or RBBP2), to a common fusion partner nucleoporin-98 (NUP98) as identified in human leukemias, generated potent oncoproteins that arrested hematopoietic differentiation and induced acute myeloid leukemia in murine models.  In these processes, a PHD finger that specifically recognizes H3K4me3/2 marks was essential for leukemogenesis.  Mutations in PHD fingers that abrogated H3K4me3 binding also abolished leukemic transformation.  NUP98-PHD fusion prevented the differentiation-assocd. removal of H3K4me3 at many loci encoding lineage-specific transcription factors (Hox(s), Gata3, Meis1, Eya1 and Pbx1), and enforced their active gene transcription in murine hematopoietic stem/progenitor cells.  Mechanistically, NUP98-PHD fusions act as 'chromatin boundary factors', dominating over polycomb-mediated gene silencing to 'lock' developmentally crit. loci into an active chromatin state (H3K4me3 with induced histone acetylation), a state that defined leukemia stem cells.  Collectively, our studies represent, to our knowledge, the first report that deregulation of the PHD finger, an 'effector' of specific histone modification, perturbs the epigenetic dynamics on developmentally crit. loci, catastrophizes cellular fate decision-making, and even causes oncogenesis during mammalian development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR1S9VoRYMRrVg90H21EOLACvtfcHk0ljG72VGhPHTdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXls1CmtL0%253D&md5=3a647643c8a73639fcb332a774f1155e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnature08036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08036%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DDormann%26aufirst%3DH.%2BL.%26aulast%3DCasadio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DJ.%2BL.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DHaematopoietic%2520malignancies%2520caused%2520by%2520dysregulation%2520of%2520a%2520chromatin-binding%2520PHD%2520finger%26jtitle%3DNature%26date%3D2009%26volume%3D459%26spage%3D847%26epage%3D851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FL6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FL6','PDB','4FL6'); return false;">PDB: 4FL6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P8H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3P8H','PDB','3P8H'); return false;">PDB: 3P8H</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UWN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UWN','PDB','3UWN'); return false;">PDB: 3UWN</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i19"><a href="/doi/suppl/10.1021/jm400919p">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_45281"></div></div></div></div></div><hr /></hr><p class="last">Synthesis of compounds <b>1</b>–<b>60</b>; protein purification and expression details; CellTiter-Glo luminescent cell viability assay details and results; details and results of the M1 and M2 assays; AlphaScreen proteins and corresponding peptide substrates; representative ITC binding curves; model of compound <b>58</b> in complex with L3MBTL3 and L3MBTL1; immunofluorescence results of <b>1</b>, <b>2</b>, and <b>56</b> in HEK293 cells; L3MBTL3 mRNA levels in various cell lines; and <sup>1</sup>H NMR spectra of compounds <b>11</b>–<b>31</b>, <b>39</b>, and <b>41</b>–<b>48</b>. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400919p/suppl_file/jm400919p_si_001.pdf">jm400919p_si_001.pdf (2.87 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm400919p&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-18%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Fjm400919p%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm400919p" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799774f9ecc3c6a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
