
<html lang="en"     class="pb-page"  data-request-id="27595fc4-835b-416b-ae39-aaa250ed28c2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.6b00070;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2016.59.issue-17;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)" /></meta><meta name="dc.Creator" content="Robert H.  Bradbury" /></meta><meta name="dc.Creator" content="Rowena  Callis" /></meta><meta name="dc.Creator" content="Gregory R.  Carr" /></meta><meta name="dc.Creator" content="Huawei  Chen" /></meta><meta name="dc.Creator" content="Edwin  Clark" /></meta><meta name="dc.Creator" content="Lyman  Feron" /></meta><meta name="dc.Creator" content="Steve  Glossop" /></meta><meta name="dc.Creator" content="Mark A.  Graham" /></meta><meta name="dc.Creator" content="Maureen  Hattersley" /></meta><meta name="dc.Creator" content="Chris  Jones" /></meta><meta name="dc.Creator" content="Scott G.  Lamont" /></meta><meta name="dc.Creator" content="Gilles  Ouvry" /></meta><meta name="dc.Creator" content="Anil  Patel" /></meta><meta name="dc.Creator" content="Joe  Patel" /></meta><meta name="dc.Creator" content="Alfred A.  Rabow" /></meta><meta name="dc.Creator" content="Craig A.  Roberts" /></meta><meta name="dc.Creator" content="Stephen  Stokes" /></meta><meta name="dc.Creator" content="Natalie  Stratton" /></meta><meta name="dc.Creator" content="Graeme E.  Walker" /></meta><meta name="dc.Creator" content="Lara  Ward" /></meta><meta name="dc.Creator" content="David  Whalley" /></meta><meta name="dc.Creator" content="David  Whittaker" /></meta><meta name="dc.Creator" content="Gail  Wrigley" /></meta><meta name="dc.Creator" content="Michael J.  Waring" /></meta><meta name="dc.Description" content="Here we report the discovery and optimization of a series of bivalent bromodomain and extraterminal inhibitors. Starting with the observation of BRD4 activity of compounds from a previous program, ..." /></meta><meta name="Description" content="Here we report the discovery and optimization of a series of bivalent bromodomain and extraterminal inhibitors. Starting with the observation of BRD4 activity of compounds from a previous program, ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 24, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00070" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00070" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00070" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00070" /></link>
        
    
    

<title>Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00070" /></meta><meta property="og:title" content="Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0019.jpeg" /></meta><meta property="og:description" content="Here we report the discovery and optimization of a series of bivalent bromodomain and extraterminal inhibitors. Starting with the observation of BRD4 activity of compounds from a previous program, the compounds were optimized for BRD4 potency and physical properties. The optimized compound from this campaign exhibited excellent pharmacokinetic profile and exhibited high potency in vitro and in vivo effecting c-Myc downregulation and tumor growth inhibition in xenograft studies. This compound was selected as the development candidate, AZD5153. The series showed enhanced potency as a result of bivalent binding and a clear correlation between BRD4 activity and cellular potency." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00070"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00070">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00070&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00070&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00070&amp;href=/doi/10.1021/acs.jmedchem.6b00070" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 7801-7817</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00041" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00425" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3<i>R</i>)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Robert+H.++Bradbury">Robert H. Bradbury</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rowena++Callis">Rowena Callis</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gregory+R.++Carr">Gregory R. Carr</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Huawei++Chen">Huawei Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Edwin++Clark">Edwin Clark</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lyman++Feron">Lyman Feron</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steve++Glossop">Steve Glossop</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mark+A.++Graham">Mark A. Graham</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maureen++Hattersley">Maureen Hattersley</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chris++Jones">Chris Jones</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Scott+G.++Lamont">Scott G. Lamont</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gilles++Ouvry">Gilles Ouvry</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anil++Patel">Anil Patel</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Joe++Patel">Joe Patel</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alfred+A.++Rabow">Alfred A. Rabow</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Craig+A.++Roberts">Craig A. Roberts</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen++Stokes">Stephen Stokes</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Natalie++Stratton">Natalie Stratton</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Graeme+E.++Walker">Graeme E. Walker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lara++Ward">Lara Ward</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Whalley">David Whalley</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Whittaker">David Whittaker</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gail++Wrigley">Gail Wrigley</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+J.++Waring">Michael J. Waring</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">AstraZeneca, Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">AstraZeneca, Chemin du Moulin de Vrilly, 51100 Reims, France</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Northern Institute for Cancer Research, School of Chemistry, Newcastle University, Bedson Building, Newcastle upon Tyne NE1 7RU, U.K.</span></div><div class="corresp-info"><strong>*</strong>Prof. Michael J. Waring (for correspondence related to the manuscript). Tel. +44 (0)191 208 8591. E-mail: <a href="/cdn-cgi/l/email-protection#d5b8bcbeb0fba2b4a7bcbbb295bbb0a2b6b4a6a1b9b0fbb4b6fba0be"><span class="__cf_email__" data-cfemail="365b5f5d53184157445f585176585341555745425a5318575518435d">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Dr. Edwin Clark (for correspondence relating to the AstraZeneca programme). Tel. +1 781 839 4983. E-mail: <a href="/cdn-cgi/l/email-protection#3e5b5a495750105d525f4c557e5f4d4a4c5f445b505b5d5f105d5153"><span class="__cf_email__" data-cfemail="a8cdccdfc1c686cbc4c9dac3e8c9dbdcdac9d2cdc6cdcbc986cbc7c5">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00070&amp;href=/doi/10.1021%2Facs.jmedchem.6b00070" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 7801–7817</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 15, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 January 2016</li><li><span class="item_label"><b>Published</b> online</span>24 August 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 September 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00070" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00070</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7801%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRobert%2BH.%2BBradbury%252C%2BRowena%2BCallis%252C%2BGregory%2BR.%2BCarr%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D17%26contentID%3Dacs.jmedchem.6b00070%26title%3DOptimization%2Bof%2Ba%2BSeries%2Bof%2BBivalent%2BTriazolopyridazine%2BBased%2BBromodomain%2Band%2BExtraterminal%2BInhibitors%253A%2BThe%2BDiscovery%2Bof%2B%25283R%2529-4-%255B2-%255B4-%255B1-%25283-Methoxy-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-b%255Dpyridazin-6-yl%2529-4-piperidyl%255Dphenoxy%255Dethyl%255D-1%252C3-dimethyl-piperazin-2-one%2B%2528AZD5153%2529%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7817%26publicationDate%3DSeptember%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00070"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3209</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">36</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00070" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;H. Bradbury&quot;},{&quot;first_name&quot;:&quot;Rowena&quot;,&quot;last_name&quot;:&quot;Callis&quot;},{&quot;first_name&quot;:&quot;Gregory&quot;,&quot;last_name&quot;:&quot;R. Carr&quot;},{&quot;first_name&quot;:&quot;Huawei&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Edwin&quot;,&quot;last_name&quot;:&quot;Clark&quot;},{&quot;first_name&quot;:&quot;Lyman&quot;,&quot;last_name&quot;:&quot;Feron&quot;},{&quot;first_name&quot;:&quot;Steve&quot;,&quot;last_name&quot;:&quot;Glossop&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;A. Graham&quot;},{&quot;first_name&quot;:&quot;Maureen&quot;,&quot;last_name&quot;:&quot;Hattersley&quot;},{&quot;first_name&quot;:&quot;Chris&quot;,&quot;last_name&quot;:&quot;Jones&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;G. Lamont&quot;},{&quot;first_name&quot;:&quot;Gilles&quot;,&quot;last_name&quot;:&quot;Ouvry&quot;},{&quot;first_name&quot;:&quot;Anil&quot;,&quot;last_name&quot;:&quot;Patel&quot;},{&quot;first_name&quot;:&quot;Joe&quot;,&quot;last_name&quot;:&quot;Patel&quot;},{&quot;first_name&quot;:&quot;Alfred&quot;,&quot;last_name&quot;:&quot;A. Rabow&quot;},{&quot;first_name&quot;:&quot;Craig&quot;,&quot;last_name&quot;:&quot;A. Roberts&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;Stokes&quot;},{&quot;first_name&quot;:&quot;Natalie&quot;,&quot;last_name&quot;:&quot;Stratton&quot;},{&quot;first_name&quot;:&quot;Graeme&quot;,&quot;last_name&quot;:&quot;E. Walker&quot;},{&quot;first_name&quot;:&quot;Lara&quot;,&quot;last_name&quot;:&quot;Ward&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Whalley&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Whittaker&quot;},{&quot;first_name&quot;:&quot;Gail&quot;,&quot;last_name&quot;:&quot;Wrigley&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;J. Waring&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;7801-7817&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00070&quot;},&quot;abstract&quot;:&quot;Here we report the discovery and optimization of a series of bivalent bromodomain and extraterminal inhibitors. Starting with the observation of BRD4 activity of compounds from a previous program, the compounds were optimized for BRD4 potency and physical properties. The optimized compound from this campaign exhibited excellent pharmacokinetic profile and exhibited high potency in vitro and in vivo effecting c-Myc downregulation and tumor growth inhibition in xenograft studies. This compound was selected as the development candidate, AZD5153. The series showed enhanced potency as a result of bivalent binding and a clear correlation between BRD4 activity and cellular potency.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00070&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00070" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00070&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00070" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00070&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00070" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00070&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00070&amp;href=/doi/10.1021/acs.jmedchem.6b00070" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00070" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00070" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00070%26sid%3Dliteratum%253Aachs%26pmid%3D27528113%26genre%3Darticle%26aulast%3DBradbury%26date%3D2016%26atitle%3DOptimization%2Bof%2Ba%2BSeries%2Bof%2BBivalent%2BTriazolopyridazine%2BBased%2BBromodomain%2Band%2BExtraterminal%2BInhibitors%253A%2BThe%2BDiscovery%2Bof%2B%25283R%2529-4-%255B2-%255B4-%255B1-%25283-Methoxy-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-b%255Dpyridazin-6-yl%2529-4-piperidyl%255Dphenoxy%255Dethyl%255D-1%252C3-dimethyl-piperazin-2-one%2B%2528AZD5153%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D17%26spage%3D7801%26epage%3D7817%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290723" title="Solutions">Solutions</a>,</li><li><a href="/action/doSearch?ConceptID=291594" title="Silica">Silica</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292593" title="Liquid chromatography-mass spectrometry">Liquid chromatography-mass spectrometry</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/jmcmar.2016.59.issue-17/20160908/jmcmar.2016.59.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Here we report the discovery and optimization of a series of bivalent bromodomain and extraterminal inhibitors. Starting with the observation of BRD4 activity of compounds from a previous program, the compounds were optimized for BRD4 potency and physical properties. The optimized compound from this campaign exhibited excellent pharmacokinetic profile and exhibited high potency in vitro and in vivo effecting c-Myc downregulation and tumor growth inhibition in xenograft studies. This compound was selected as the development candidate, AZD5153. The series showed enhanced potency as a result of bivalent binding and a clear correlation between BRD4 activity and cellular potency.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56512" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56512" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomains (BRD) are a family of proteins responsible for binding acetylated lysine residues of proteins. The BRDs are rapidly emerging as a target class amenable to pharmacological intervention with small molecules.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The bromodomain and extraterminal (BET) subfamily are perhaps the most widely explored of the class.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Initially identified as a result of phenotypic screening for modulation of ApoA1, subsequent target identification in tandem with optimization has revealed multiple small molecule inhibitors of BET bromodomains, which have progressed to preclinical and clinical evaluation (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> The BET family consists of four proteins, termed BRD2, BRD3, BRD4, and BRDT. Each of these proteins contains two separate bromodomains. BRD4 in particular has been lauded as a potential drug target,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> although it is worth remembering that there is a high degree of homology within this class and the majority of compounds, particularly those studied most extensively in biological elucidation, do not exhibit selectivity within the class.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The BET family proteins regulate gene expression through binding of acetylated lysine residues on histone proteins followed by activation of transcription elongation driven by RNA-PolII. BET inhibitors inhibit this gene transcription and, as a consequence, exhibit growth inhibition in a number of hematological and solid tumor models. This action is believed, at least in part, to be due to downregulation of critical oncogenes such as c-Myc, leading to great interest in BET inhibitors in the oncology field, and a number are currently progressing in early clinical trials.</div><div class="NLM_p">From the initial ApoA1 screen, pioneering work at GlaxoSmithkline led to the discovery of the triazolo-benzodiazepine <b>1</b> (iBET-762, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>),<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> which has become one of the leading clinical BET inhibitors. Work carried out by Bradner and the Structural Genomics Consortium identified the structurally related compound <b>2</b> (JQ1).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Compounds <b>1</b> and <b>2</b> have become key standard compounds in the field with many studies using these compounds as probes to understand BET biology. More recently, OTX015, <b>3</b>,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> the corresponding <i>p</i>-hydroxyanilide derivative of <b>2</b> has been reported to show encouraging clinical results in AML patients.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p last">We have recently described the discovery that compounds from the androgen receptor (AR) downregulator (DR) program<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> were found to be potent BET inhibitors that are demonstrated to act through an in-cis bivalent binding mode in which a single inhibitor molecule simultaneously engages both bromodomains of a single protein via two acetyl lysine mimicking moieties.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> This activity is postulated to explain the effects of the compounds observed in AR and estrogen receptor (ER) DR assays, as well as their effects on c-Myc levels. Here we describe the medicinal chemistry program that led to the identification of these compounds and their optimization, leading to the discovery of a clinical candidate bivalent BET inhibitor.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53375" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53375" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The campaign of optimization was initiated with the objective of improving the potency of the previous clinical AR downregulator AZD3514, <b>4</b>, and the corresponding unsaturated analogue <b>5</b> while maintaining their excellent ADMET properties.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> These compounds exhibited modest cellular AR DR potencies (pIC<sub>50</sub> 5.8 and 6.5 for <b>4</b> and <b>5</b>, respectively, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>), and subsequent chemistry was focused on cellular AR DR potency to guide the chemistry campaign. The BET inhibitory activity of the compounds and resulting effects on ER and c-Myc DR were discovered during the course of this work.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Later chemistry focused on the BRD4 potency and ER DR data to develop SAR and was generated retrospectively on the earlier compounds. The AR, ER, and c-Myc DR data (measured in LNCaP, MCF7, and MM1.S cell lines, respectively) are tightly correlated and can be used reasonably interchangeably to interpret the SAR. The data from these assays are also predictive of growth inhibitory effects measured in MM1.S cells (see later and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_001.pdf" class="ext-link">Supporting Information</a> for more detail). In order to assess the effect of the bivalent binding, an assay employing a protein construct containing both bromodomains was required. Hence, a fluorescence polarization (FP) assay for tandem domain BRD4 protein (residues 44–460) was developed using a fluorescently labeled small molecule probe derived from <b>2</b>. It should be noted throughout that while data for BRD4 are quoted, no selectivity of these molecules within the BET family is anticipated due to their close homology.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The synthetic route that could be used to prepare analogues with modifications to the acylpiperazine is exemplified with the synthesis of <b>5</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). S<sub>N</sub>Ar addition of 4-(4-hydroxyphenyl)piperidine to chlorotriazolopyridazine afforded the corresponding phenol. The phenol was alkylated with ethylene carbonate to give the ethyl alcohol, which was derivatized as the mesylate, and this was used to alkylate the required amine, in this case acetylpiperazine, to give <b>5</b> in 35% overall yield for the four steps. Compounds <b>6</b> and <b>7</b> were prepared in an analogous manner.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) DIPEA, MeCN, rt, 66%; (ii) ethylene carbonate, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 72%; (iii) mesyl chloride, Et<sub>3</sub>N, DCM, 0 °C, 100%; (iv) <i>N</i>-acetylpiperazine, DIPEA, DMA, 110 °C, 73%. Compounds <b>6</b> and <b>7</b> were prepared in an analogous manner.</p></p></figure><div class="NLM_p">Alternatively, the amine substituent could be introduced before the triazolopyridazine allowing more effcient synthesis of compounds with different triazolopyridazine substituents. This is exemplified by the synthesis of <b>8</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). BOC protection of 4-(4-hydroxyphenyl)piperidine was followed by alkylation of the phenol with 1-bromo-2-chloroethane to give the alkyl halide. This underwent displacement with <i>R</i>-1,3-dimethylpiperazin-2-one to afford the BOC protected intermediate. Removal of the BOC group with ethereal HCl afforded the piperazine, which could be arylated with a variety of substituted chlorotriazolopyridazines, in this case 6-chloro-3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine to give the final compound <b>8</b> in five steps and 52% overall yield. Compounds <b>9</b>–<b>20</b> were prepared in a similar fashion.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Et<sub>3</sub>N, BOC<sub>2</sub>O, DCM, 0 °C, 88%; (ii) 1-bromo-2-chloroethane, K<sub>2</sub>CO<sub>3</sub>, acetone, 90 °C, 97%; (iii) DIPEA, KI, DMA, 110 °C, 90%; (iv) HCl, Et<sub>2</sub>O, quant; (v) DIPEA, DMA, 80 °C, 68%. Compounds <b>9</b> to <b>20</b> were prepared in a similar fashion.</p></p></figure><div class="NLM_p">Analogues with variations to the ethoxy linker of the piperazinone were prepared by initial alkylation of the piperazinone by reductive amination with an appropriate ketone, exemplified with <i>N</i>-BOC-azetidine-3-one (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Removal of the BOC group with trifluoroacetic acid gave the intermediate amine. Arylation of 4-(4-bromophenyl)piperidine with 6-methoxy-3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine gave the aryl bromide, which was coupled with the amine under Buchwald–Hartwig conditions to give the final compound <b>22</b> (10% overall yield for the lowest yielding sequence). Compounds <b>23</b> and <b>24</b> were prepared in an analogous manner.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Et<sub>3</sub>N, NaBH(OAc)<sub>3</sub>, DCM, 56%; (ii) TFA, DCM, quant; (iii) DIPEA, DMF, 80 °C, 34%; (iv) Ruphos palladium(II) phenethylamine chloride, dicyclohexyl(2′,6′-diisopropoxy-[1,1′-biphenyl]-2-yl)phosphine, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 90 °C, 28%. Compounds <b>9</b>–<b>20</b> were prepared in a similar fashion.</p></p></figure><div class="NLM_p">In exploring the structure–activity relationships in relation to the potential bromodomain inhibitory activity, we considered that both the triazolopyridazine and acylpiperazine moieties of the molecule could act as acetyl lysine mimetics (subsequently, we discovered that in fact both do, resulting in the in-cis binding mode and accounting for the SAR we observed).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Modifications to both of these regions were designed with consideration to the overlay of the compounds with acetyl lysine in the binding site. Changing the acyl piperazine of <b>5</b> for <i>N</i>-methylpiperazin-2-one (<b>6</b>) resulted in retained potency (AR DR pIC<sub>50</sub> 6.5 for both compounds; <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). The isomeric piperazin-3-one <b>7</b> was slightly less potent (ER DR pIC<sub>50</sub> 6.0 compared with 6.3 for <b>6</b>). 3-Methyl substitution of the piperazin-2-one, <b>8</b>, led to an increase in ER DR potency (pIC<sub>50</sub> 7.6 compared with 6.3 for <b>6</b>). In this series, the piperazinones had improved solubility but also increased metabolism, as measured by human hepatocyte turnover, compared with the acylpiperazine <b>5</b>. BRD4 assay data reveleaved that compounds <b>5</b>–<b>7</b> had weak BRD4 potency (pIC<sub>50</sub> range 5.6–6.7) and that introduction of the piperazinone 3-methyl substituent led to increased BRD4 activity (pIC<sub>50</sub> 8.1) consistent with the increase in the cellular potency.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Effects of Structural Modifications of the Acylpiperazine</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0012.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="rowsep0"><th class="colsep0 rowsep0" align="center">compd no.</th><th class="colsep0 rowsep0" align="center" char=".">BRD4 FP p<i>K</i><sub>i</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">AR DR pIC<sub>50</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">ER DR pIC<sub>50</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">MycDR pIC<sub>50</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">MM1.S pGI<sub>50</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">solubility (μM)</th><th class="rowsep0" align="center" char=".">Hu heps Cl<sub>int</sub> (mL min<sup>−1</sup> × 10<sup>−6</sup> cells)</th></tr></thead><tbody><tr valign="top" class="rowsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="rowsep0" align="char" char=".">12</td></tr><tr valign="top" class="rowsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">65</td><td class="rowsep0" align="char" char=".">18</td></tr><tr valign="top" class="rowsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="rowsep0" align="char" char=".">26</td></tr><tr valign="top" class="rowsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">8.1</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="rowsep0" align="char" char=".">70</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Measurements reported are the mean of at least three determinations. The standard deviation is not shown, because all three assays were found to be highly reproducible on repeat testing (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for more detail).</p></div></div></div><div class="NLM_p">Next, the effect of changing the triazolopyridazine trifluoromethyl substituent of <b>8</b> was explored (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Removal of the substituent (<b>9</b>) led to a loss of BRD4 and cellular potency. Replacement of the trifluoromethyl group with methyl, chloro, thiomethyl, methoxy, and dimethylamino substituents (<b>10</b>–<b>14</b>, respectively) resulted in retained activity. Of these compounds, the methoxy derivative <b>13</b> had the lowest log <i>D</i><sub>7.4</sub> value and showed very high solubility and a significantly improved human hepatocyte stability, resulting in a clear improvement in overall profile. Hence, the methoxy substituent appeared optimal and was retained in subsequent analogues.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Effect of Changing the Triazolopyridazine Trifluoromethyl Substituent</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0014.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd no.</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">BRD4 p<i>K</i><sub>i</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">AR DR pIC<sub>50</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">ER DR pIC<sub>50</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">Myc DR pIC<sub>50</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">MM1.S pGI<sub>50</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">solubility (μM)</th><th class="colsep0 rowsep0" align="center" char=".">Hu heps Cl<sub>int</sub> (mL min<sup>–1</sup> × 10<sup>–6</sup> cells)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">8.1</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">8.0</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">>4.2</td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="char" char=".">70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">-H</td><td class="colsep0 rowsep0" align="char" char="."><6.0</td><td class="colsep0 rowsep0" align="char" char=".">5.8</td><td class="colsep0 rowsep0" align="char" char="."><5.5</td><td class="colsep0 rowsep0" align="char" char=".">5.9</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">1000</td><td class="colsep0 rowsep0" align="char" char=".">180</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">-Me</td><td class="colsep0 rowsep0" align="char" char=".">7.8</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">670</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">-Cl</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="char" char=".">24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">-SMe</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char=".">660</td><td class="colsep0 rowsep0" align="char" char=".">21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">-OMe</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">>1600</td><td class="colsep0 rowsep0" align="char" char="."><3.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">-NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">7.8</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">>1600</td><td class="colsep0 rowsep0" align="char" char=".">4.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Measurements reported are the mean of at least three determinations. The standard deviation is not shown, because all three assays were found to be highly reproducible on repeat testing (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for more detail).</p></div></div></div><div class="NLM_p">Further exploration of changes to the piperazinone group (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) showed that the chirality of the 3-methyl substituent was important, with the <i>S</i>-enantiomer <b>15</b> showing a loss of potency (1.6 log units relative to <b>13</b>), suggesting that a specific interaction was formed with the methyl group. The 3,5-dimethyl-1-acetylpiperazine analogue <b>16</b> showed an increase in potency relative to piperazinone <b>13</b> (0.5 log units vs BRD4 but 1.2 units in ER DR). It is possible that this apparent discrepancy arises due to the potency limit in the FP assay. Unfortunately, this change also resulted in a decrease in human hepatocyte stability. Again, the methyl groups appeared to be important for potency; for instance, the corresponding 2,6-dimethyl isomer <b>16</b> showed a loss of potency (1.4 log units in ER DR) relative to <b>15</b>. Further methylation of the piperazinone, for example, <b>18</b>, led to similar potency to <b>13</b> but with a marked decrease in human hepatocyte stability. These data suggested that balancing potency and metabolic stability in these compounds was very challenging and the (<i>R</i>)-1,3-dimethylpiperazin-2-one group appeared to strike the best balance.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Further Exploration of Changes to the Piperazinone Group</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0015.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top"><th class="colsep0 rowsep0" align="center">compd no.</th><th class="colsep0 rowsep0" align="center" char=".">BRD4 p<i>K</i><sub>i</sub><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">ER DR pIC<sub>50</sub><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">Myc DR pIC<sub>50</sub><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">MM1.S pGI<sub>50</sub><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">solubility (μM)</th><th class="rowsep0" align="center" char=".">Hu heps Cl<sub>int</sub> (mL min<sup>−1</sup> × 10<sup>−6</sup> cells)</th></tr></thead><tbody><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">>1600</td><td class="rowsep0" align="char" char="."><3.5</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">900</td><td class="rowsep0" align="char" char="."><1.4</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">>8.7</td><td class="colsep0 rowsep0" align="char" char=".">>8.7</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="rowsep0" align="char" char=".">12</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">280</td><td class="rowsep0" align="char" char=".">48</td></tr><tr valign="top"><td class="colsep0" align="left"><b>18</b></td><td class="colsep0" align="char" char=".">8.1</td><td class="colsep0" align="char" char=".">7.7</td><td class="colsep0" align="char" char=".">7.9</td><td class="colsep0" align="char" char=".">8.2</td><td class="colsep0" align="char" char=".">2.3</td><td class="colsep0" align="char" char=".">750</td><td align="char" char=".">80</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Measurements reported are the mean of at least three determinations. The standard deviation is not shown, because all three assays were found to be highly reproducible on repeat testing (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for more detail).</p></div></div></div><div class="NLM_p">Modifications to the linker between the piperazinone and the central phenyl ring were also explored (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Extending the ethoxy linker to propoxy (<b>19</b>) led to increased cellular potency but higher hepatocyte Cl<sub>int</sub> (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Further extension to butoxy (<b>20</b>) led to similar potency but further increased Cl<sub>int</sub>. Deletion of the ether oxygen leading to an ethyl linker (<b>21</b>) resulted in similar potency and Cl<sub>int</sub> to <b>13</b>. Rigidification of the linker by incorporation of an azetidine (<b>22</b>) or piperidine (<b>23</b>) resulted in some loss of potency, whereas the more flexible azetidinylmethyl linker (<b>24</b>) resulted in an increase in cellular potency relative to <b>13</b> while maintaining good solubility and Cl<sub>int</sub> values.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Modifications to the Linker between the Piperazinone and the Central Phenyl Ring</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0017.gif" alt="" id="fx6" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="bottom"><th class="colsep0 rowsep0" align="center" valign="bottom">compd no.</th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">BRD4 p<i>K</i><sub>i</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">ER DR pIC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">cMyc DR pIC<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">MM1.S pGI<sub>50</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" valign="bottom" char=".">solubility (μM)</th><th class="rowsep0" align="center" valign="bottom" char=".">heps Cl<sub>int</sub> (mL min<sup>–1</sup> × 10<sup>–6</sup> cells)</th></tr></thead><tbody><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">>1600</td><td class="rowsep0" align="char" char="."><3.5</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">8.4</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">>8.8</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="rowsep0" align="char" char=".">5.1</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">8.2</td><td class="colsep0 rowsep0" align="char" char=".">7.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">470</td><td class="rowsep0" align="char" char=".">12</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">570</td><td class="rowsep0" align="char" char="."><2.2</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">870</td><td class="rowsep0" align="char" char="."><1.2</td></tr><tr valign="top"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">760</td><td class="rowsep0" align="char" char=".">2.7</td></tr><tr valign="top"><td class="colsep0" align="left"><b>24</b></td><td class="colsep0" align="char" char=".">8</td><td class="colsep0" align="char" char=".">8.4</td><td class="colsep0" align="char" char=".">>8.7</td><td class="colsep0" align="char" char=".">8.8</td><td class="colsep0" align="char" char=".">1.7</td><td class="colsep0" align="char" char=".">>1000</td><td align="char" char=".">4.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Measurements reported are the mean of at least three determinations. The standard deviation is not shown, because all three assays were found to be highly reproducible on repeat testing (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for more detail).</p></div></div></div><div class="NLM_p">Solving the X-ray crystal structure of <b>13</b> in complex with the first bromodomain of BRD4 [BRD4(1)] revealed the formation of a dimer as previously observed,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> in which both the triazolopyridazine and <i>N</i>-methylpiperazinone engage the acetyl lysine sites of different bromodomains (<a class="ref internalNav" href="#fig2a" aria-label="Figure 3">Figure 3</a>). In the first bromodomain, critical hydrogen bonds were observed between the triazolopyridazine nitrogen-1 and Asn140 with the nitrogen-2 and the methoxy substituent acting as the acetyl lysine mimic. In the second bromodomain, the <i>N</i>-methylpiperazinone methyl and carbonyl groups acted as the acetyl lysine mimic with the carbonyl also engaging Asn140. The 3-methyl group filled a small lipophilic hole, explaining the increase in potency arising from its introduction. While this structure is homodimeric, showing the interaction of both ends of the molecule with BRD4(1), given the close homology between the first and the second bromodomains, it is reasonable to anticipate that similar interactions would be made with each bromodomain in an in-cis complex with full length BRD4. This is fully consistent with the observed SAR and explains, for instance, the increase in potency for <b>16</b> relative to <b>13</b>, <b>17</b>, and <b>18</b> since the additional methyl group of 16 is able to occupy a vacant lipophilic cavity in the binding site. In this model, the diminished potency of <b>22</b> and <b>23</b> relative to the other exemplified linkers could be explained by the linear constraints of the linkers of these molecules requiring an energetically less favored orientation of the two bromodomains in order to achieve bivalent binding.</div><figure id="fig2a" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of <b>13</b> in complex with BRD4(1) showing (left) the formation of a dimeric structure with two bromodomains. (middle) Engagement of the triazolopyridazine of <b>13</b> with the first bromodomain (blue ribbon) acetyl lysine binding site. (right) Engagement of the <i>N</i>-methylpiperazinone of <b>13</b> with the second bromodomain (red ribbon) acetyl lysine site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig2a"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To provide further evidence of the bivalent binding mode, compounds were tested against the N140A and N433A mutant forms of BRD4 in the FP assay, and the p<i>K</i><sub>i</sub> values obtained were compared with the wild-type protein (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). N140 in BD1 and N433 in BD2 of BRD4 have been shown to be important in the binding of potent BRD ligands.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Mutation of these residues was expected to provide insights into the binding mechanism. Monovalent ligand <b>1</b> showed consistent <i>K</i><sub>i</sub> values across the wild-type and both mutant proteins as would be expected. Compounds <b>5</b>, <b>6</b>, and <b>7</b>, which have the potential for bivalent binding but have an unoptimized second site binding moiety showed a decrease in affinity for the N140A mutant but an increase in affinity for the N433A mutant relative to wild-type. This suggests that the triazolopyridazine moiety binds to BD1 and the acetyl piperazine or piperazinone group binds to BD2 although this has not been definitively established. The reason for the increase in affinity for the N433A mutant relative to wild-type for these compounds is unclear. It may suggest that the unoptimized second site binding groups employed can interact more strongly with mutated BD2 than for wild-type; indeed, these changes would be expected to disrupt the critical interaction with the asparagine residue. Compounds <b>8</b>–<b>24</b>, which are all optimized bivalent compounds, consistently show a decrease in affinity of approximately 1 order of magnitude for both mutant proteins relative to wild-type. These structure–activity relationships provide further evidence of the bivalent binding mechanism of the series as a whole.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. pK<sub>i</sub> Values for BD1 and BD2 Mutant Proteins Compared with Wild Type</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd no.</th><th class="colsep0 rowsep0" align="center" char=".">WT FP p<i>K</i><sub>i</sub><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">N140A p<i>K</i><sub>i</sub><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">N433A p<i>K</i><sub>i</sub><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">7.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">6.2</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">8.1</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char="."><6.0</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">7.8</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">8.8</td><td class="colsep0 rowsep0" align="char" char=".">7.8</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">8.1</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">8.4</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">8.2</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">7.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">7.2</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="char" char=".">8.0</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Measurements reported are the mean of at least three determinations. The standard deviation is not shown, as all three assays were found to be highly reproducible on repeat testing (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for more detail).</p></div></div></div><div class="NLM_p">The wild-type p<i>K</i><sub>i</sub> values were shown to accurately predict the cellular potencies such as ER DR (<a class="ref internalNav" href="#fig3" aria-label="Figure 4">Figure 4</a>). This provides confidence that the observed pharmacology is BRD4 driven, although we would expect the compounds to have similar affinities for other members of the BET family due to their close homology and the lack of selectivity within the BET family for <b>13</b> in selectivity screens (see later). Moreover, the observation of this correlation and lack of drop off between in vitro BRD4 inhibition and cellular potency, in contrast to monovalent inhibitors, which show significantly lower cellular potency,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> suggests that the bivalent mechanism is operative in a cellular environment.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of BRD4 WT p<i>K</i><sub>i</sub> vs ER downregulation. Bivalent triazolopyridazine are shown with black dots. Line Fit: ER DR pIC<sub>50</sub> = 0.046 + 0.93[BRD4 WT FP p<i>K</i><sub>i</sub>], <i>r</i><sup>2</sup> = 0.84, <i>n</i> = 20.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The compounds with the best balance of potency and in vitro ADME properties (favorable solubility, Caco-2 <i>P</i><sub>app</sub>, and metabolic stability) and attractive preclinical pharmacokinetics were selected for further profiling (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). A preliminary human dose prediction<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> was calculated based on achieving 24 h free cover above a cellular GI<sub>50</sub> (potency in MM1.S cells) from a twice-daily dosing schedule (BID, 12 h apart). The maximum absorbable dose (MAD) and human <i>F</i><sub>abs</sub> were estimated in GI-Sim<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> using measured Caco-2 <i>P</i><sub>app</sub>, solubility, and p<i>K</i><sub>a</sub> data.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Tabulated PK Data<a class="ref internalNav" href="#tbl6-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">property</th><th class="colsep0 rowsep0" align="center"><b>13</b></th><th class="colsep0 rowsep0" align="center"><b>16</b></th><th class="colsep0 rowsep0" align="center"><b>24</b></th><th class="colsep0 rowsep0" align="center"><b>21</b></th><th class="colsep0 rowsep0" align="center"><b>14</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MM1.S GI<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.002</td><td class="colsep0 rowsep0" align="left">0.002</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">log <i>D</i>/basic p<i>K</i><sub>a</sub></td><td class="colsep0 rowsep0" align="left">2.3/4.8</td><td class="colsep0 rowsep0" align="left">2.5/6.2</td><td class="colsep0 rowsep0" align="left">1.7/5.9</td><td class="colsep0 rowsep0" align="left">2.5/6.3</td><td class="colsep0 rowsep0" align="left">2.4/-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solubility, pH<sub>7.4</sub> (μM)</td><td class="colsep0 rowsep0" align="left">>1650</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">>1000</td><td class="colsep0 rowsep0" align="left">570</td><td class="colsep0 rowsep0" align="left">>1681</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Caco-2 <i>P</i><sub>app</sub> (nm s<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">400</td><td class="colsep0 rowsep0" align="left">300</td><td class="colsep0 rowsep0" align="left">460</td><td class="colsep0 rowsep0" align="left">280</td><td class="colsep0 rowsep0" align="left">-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">plasma protein binding (% free) (rat/dog/human)</td><td class="colsep0 rowsep0" align="left">4/24/23</td><td class="colsep0 rowsep0" align="left">6/16/19</td><td class="colsep0 rowsep0" align="left">5/27/32</td><td class="colsep0 rowsep0" align="left">2/24/25</td><td class="colsep0 rowsep0" align="left">4/9/16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatocyte CL<sub>int</sub> (μL min<sup>–1</sup> per 10<sup>6</sup> cells) (rat/dog/human)</td><td class="colsep0 rowsep0" align="left">5/<1/<4</td><td class="colsep0 rowsep0" align="left">13/7/12</td><td class="colsep0 rowsep0" align="left"><1/3/4</td><td class="colsep0 rowsep0" align="left"><1/<1/<2</td><td class="colsep0 rowsep0" align="left">13/18/4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat predicted clearance<a class="ref internalNav" href="#t6fn1" aria-label="b">b</a> (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat in vivo clearance (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat <i>V</i><sub>ss</sub> (L kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">1.61</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat <i>F</i><sub>oral</sub> (%)</td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left">72</td><td class="colsep0 rowsep0" align="left">42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog predicted clearance<a class="ref internalNav" href="#t6fn1" aria-label="b">b</a> (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog in vivo clearance (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog <i>V</i><sub>ss</sub> (L kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dog <i>F</i><sub>oral</sub> (%)</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">73</td><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left">-</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">predicted human clearance<a class="ref internalNav" href="#t6fn1" aria-label="b">b</a> (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left"><6.2</td><td class="colsep0 rowsep0" align="left">9.4</td><td class="colsep0 rowsep0" align="left">8.1</td><td class="colsep0 rowsep0" align="left"><5.7</td><td class="colsep0 rowsep0" align="left">6.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human dose prediction<a class="ref internalNav" href="#t6fn2" aria-label="c">c</a> (mg)</td><td class="colsep0 rowsep0" align="left"><20</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left"><13</td><td class="colsep0 rowsep0" align="left"><13</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">predicted maximum absorbable dose<a class="ref internalNav" href="#t6fn2" aria-label="c">c</a> (mg)</td><td class="colsep0 rowsep0" align="left">9300</td><td class="colsep0 rowsep0" align="left">8565</td><td class="colsep0 rowsep0" align="left">9935</td><td class="colsep0 rowsep0" align="left">8732</td><td class="colsep0 rowsep0" align="left">8968</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">predicted human <i>F</i><sub>abs</sub></td><td class="colsep0 rowsep0" align="left">0.98</td><td class="colsep0 rowsep0" align="left">0.93</td><td class="colsep0 rowsep0" align="left">1.0</td><td class="colsep0 rowsep0" align="left">0.99</td><td class="colsep0 rowsep0" align="left">0.92</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl6-fn1"><div class="footnote" id="tbl6-fn1"><sup>a</sup><p class="last">Preclinical PK parameters were derived from plasma concentration–time data using noncompartmental analysis.</p></div><div class="footnote" id="t6fn1"><sup>b</sup><p class="last">Clearance predictions were made from hepatocyte and plasma protein binding data using the well-stirred model.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a></p></div><div class="footnote" id="t6fn2"><sup>c</sup><p class="last">Preliminary human dose and maximum absorbable dose prediction methods are described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</p></div></div></div><div class="NLM_p">Several compounds had low predicted doses (<20 mg/dose) and good margins against the MAD (>100-fold). Compound <b>13</b> was also attractive in that its in vivo plasma clearance appeared well predicted from in vitro data across species, providing greater certainty in the human clearance and plasma half-life predictions. Compound <b>13</b> also had improved metabolic stability in hepatic microsomal fractions compared with other examples (data not shown). As a result, <b>13</b> was selected for further development and given the internal identifier AZD5153.</div><div class="NLM_p">Compound <b>13</b> (10 μM) was tested for activity across 32 bromodomains in the Bromoscan panel (<a class="ref internalNav" href="#fig4" aria-label="Figure 5">Figure 5</a>). This showed it to have inherent selectivity for the BET family isolated bromodomains; the next most potent BRD was TAF1(2) (p<i>K</i><sub>i</sub> 5.9). This selectivity is further increased when considering the increase in potency due to bivalent binding to full-length BET proteins, representing a margin of 2 orders of magnitude to its full length BRD4 potency as measured in our FP assay and to cellular potency as determined by c-Myc DR, making the assumption that bivalent binding is not possible for the other BRDs.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity of <b>13</b> in the Bromoscan panel. Values above the bars indicate p<i>K</i><sub>i</sub>. Inhibition measured at 10 μM compound concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>13</b> was administered orally to mice bearing MV-4-11 xenografts, and pharmacodynamic activity (intratumoral levels of c-Myc) was measured at 2, 4, and 8 h postdose (<a class="ref internalNav" href="#fig5" aria-label="Figure 6">Figure 6</a>). A considerable decrease in c-Myc expression was observed out to 8 h post dose at free plasma levels of compound <0.2 μM. This decrease in c-Myc expression after treatment with <b>13</b> is consistent with other published BET inhibitors.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Pharmacodynamic data following single oral dose of 10 mg/kg <b>13</b> to MV-4-11 tumor-bearing mice. Measured levels of c-Myc RNA in tumors 2, 4, or 8 h postdose plotted against free plasma exposure of <b>13</b>. c-Myc levels for the vehicle group are shown on the plot for comparison at an arbitrary position on the <i>x</i>-axis. (b) Corresponding downregulation of c-Myc protein and loading control measured by Western blot. Each lane represents expression in a tumor from one mouse, <i>n</i> = 3 mice per group. Vehicle-treated mice from early and late time points only.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having demonstrated effects of c-Myc inhibition in tumors, the ability of <b>13</b> to exert antitumor activity in the MV-4-11 xenograft model was assessed (<a class="ref internalNav" href="#fig6" aria-label="Figure 7">Figure 7</a>). Subcutaneously implanted tumors were allowed to grow to ∼200 mm<sup>3</sup> before administering <b>13</b> daily for 14 days. When dosed at 10 mg kg<sup>–1</sup>, tumor regressions occurred (>100% TGI), with nearly complete regression for the majority of tumors. Compound <b>13</b> was tolerated for 14 days with an average body weight loss of 8%, which is indicative of acceptable tolerability; further detailed in vivo pharmacology will be communicated in subsequent publications.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Antitumor activity of <b>13</b> in the MV-4-11 xenograft efficacy model. Tumors were allowed to grow to an average size of 200 mm<sup>3</sup>; then mice were treated daily with either vehicle (<i>n</i> = 8) or <b>13</b> (<i>n</i> = 8) for 2 weeks via oral gavage. Tumor volumes were monitored twice a week for 14 days and are plotted as the geometric mean volumes ± standard error (SE) for both vehicle and treated groups (vehicle group SE bars are too small to see).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Through careful analysis of structure–activity relationships, we have been able to elucidate the basis for secondary pharmacology observed for a series of AR downregulators and ascribe it to BRD4 inhibition. Subsequent optimization of potency and ADMET properties led to the development of potent bivalent BET ligands, culminating in the clinical candidate <b>13</b>, a potent and selective BET inhibitor with a novel bivalent binding mode. The in vitro potency of <b>13</b> translated directly into c-Myc modulation and tumor growth inhibition in vivo in xenograft studies at low doses. Further studies describing the pharmacology of <b>13</b> will be the subject of further publications.</div><div class="NLM_p last">In addition to the discovery of the clinical candidate, this work demonstrates the optimization of bivalent inhibitors with maintained drug-like properties. We have previously highlighted the potential advantages of bivalent binding as a general concept for improved potency and specificity in ligand discovery,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> but commonly, this concept has been approached by the linking of two monovalent ligands with a flexible linker leading to molecules that are large and lipophilic and hence have compromised drug-likeness. Indeed, this approach has been taken for BET inhibitors,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and it has been shown that bivalent binding may also be a strategy for overcoming drug resistance, as exemplified with mTOR inhibitors.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19, 20)</a> The concept described here shows that by partially fusing the two binding elements within a bivalent inhibitor, compounds of clinical candidate quality can be developed. We anticipate that this will be an attractive strategy for targeting other multidomain proteins with high potency and specificity in the future.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52276" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52276" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> General Experimental</h3><div class="NLM_p last">Proton nuclear magnetic resonance spectra were recorded on a Bruker 700, 500, or 400 MHz instrument. All <sup>1</sup>H NMR spectra were measured in parts per million (ppm) relative to the signal for residual dimethyl sulfoxide (DMSO) in deuterated DMSO (2.49 ppm) or chloroform in deuterated chloroform (7.26 ppm). Data for <sup>1</sup>H NMR spectra were reported as chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, h = heptet, m = multiplet), coupling constants, and integration. Carbon nuclear magnetic resonance spectra were recorded on a Bruker 176 MHz instrument. All <sup>13</sup>C NMR spectra were measured in parts per million (ppm) relative to the signal for residual dimethyl sulfoxide (DMSO) in deuterated DMSO (39.5 ppm) and were obtained with complete <sup>1</sup>H decoupling. Final compound purity was assessed by LS-MS analysis, and compounds are ≥95% unless otherwise stated. All LC-MS analyses were performed on a Waters 2795 separations module, using a Gemini-NX 5 μ C18 column with a Waters Micromass detector. High resolution mass spectra were obtained on either a Thermo LTQ-FT/Accela/CTC/PDA or Waters Xevo G2 Qtof instrument in ESI positive ion mode.</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> 1-(4-(2-(4-(1-(3-(Trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-1-yl)ethanone (<b>5</b>)</h3><div class="NLM_p">DIPEA (14 mL, 8.2 mmol) was added to 2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethylmethanesulfonate (<b>27</b>; 13 g, 27 mmol) and <i>N</i>-acetylpiperazine (3.9 g, 30 mmol) in DMA (65 mL). The resulting solution was stirred at 110 °C for 1 h. The reaction mixture was cooled to room temperature, and the solvents were evaporated. The residue was diluted with DCM (250 mL) and washed with water (250 mL) (emulsion formed). The aqueous was re-extracted with DCM (250 mL), and the combined organics were washed with brine (250 mL), dried over MgSO<sub>4</sub>, filtered, and evaporated to give crude product. The crude product was purified by flash silica chromatography, elution gradient 0–3% MeOH in DCM then 5% MeOH in DCM. Pure fractions were evaporated to give 1-(4-(2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-1-yl)ethanone (<b>5</b>; 10 g, 73%) as a white solid. <sup>1</sup>H NMR (700 MHz, DMSO) δ 1.64 (qd, <i>J</i> = 4.0, 12.8 Hz, 2H), 1.82–1.89 (m, 2H), 1.96 (s, 3H), 2.38–2.42 (m, 2H), 2.45–2.48 (m, 2H), 2.69 (t, <i>J</i> = 5.8 Hz, 2H), 2.78 (tt, <i>J</i> = 3.4, 12.0 Hz, 1H), 3.08 (td, <i>J</i> = 2.0, 13.1 Hz, 2H), 3.40 (dt, <i>J</i> = 5.2, 10.4 Hz, 3H), 4.04 (t, <i>J</i> = 5.8 Hz, 2H), 4.38 (d, <i>J</i> = 13.4 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.63 (d, <i>J</i> = 10.3 Hz, 1H), 8.21 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 21.03, 32.31, 40.51, 40.75, 45.57, 45.82, 52.66, 53.15, 56.40, 65.38, 114.34, 116.78, 118.61 (q, <i>J</i> = 269.2 Hz), 124.05, 127.53, 137.15 (q, <i>J</i> = 39.6 Hz), 137.49, 144.10, 155.43, 156.78, 167.96. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 518. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>30</sub>F<sub>3</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 518.2486; Found 518.2490.</div><div class="NLM_p">The following compounds were synthesized using analogous procedures.</div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 1-Methyl-4-(2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>6</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.64 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.86 (d, <i>J</i> = 11.5 Hz, 2H), 2.73 (dd, <i>J</i> = 4.9, 5.6 Hz, 4H), 2.75–2.79 (m, 1H), 2.80 (s, 3H), 3.04–3.1 (m, 4H), 3.22–3.26 (m, 2H), 4.05 (t, <i>J</i> = 5.6 Hz, 2H), 4.38 (d, <i>J</i> = 13.3 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.63 (d, <i>J</i> = 10.3 Hz, 1H), 8.21 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 32.31, 32.88, 40.52, 45.81, 47.78, 49.50, 55.29, 57.13, 65.42, 114.33, 116.77, 118.61 (q, <i>J</i> = 269.2), 124.04, 127.54, 137.15 (q, <i>J</i> = 39.6), 137.54, 144.09, 155.42, 156.72, 165.74. Purity 99% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 504. HRMS (ESI) Calcd for C<sub>24</sub>H<sub>28</sub>F<sub>3</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 504.2329; Found 504.2329.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 4-Methyl-1-(2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>7</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.64 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.86 (d, <i>J</i> = 12.4 Hz, 2H), 2.18 (s, 3H), 2.51–2.54 (m, 2H), 2.78 (tt, <i>J</i> = 3.3, 12.0 Hz, 1H), 2.90 (s, 2H), 3.07 (td, <i>J</i> = 2.0, 13.1 Hz, 2H), 3.39–3.42 (m, 2H), 3.61 (t, <i>J</i> = 5.7 Hz, 2H), 4.06 (t, <i>J</i> = 5.7 Hz, 2H), 4.38 (d, <i>J</i> = 13.4 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.63 (d, <i>J</i> = 10.3 Hz, 1H), 8.21 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 32.86, 41.08, 44.92, 45.75, 46.37, 47.95, 51.71, 59.28, 65.74, 114.89, 117.33, 119.17 (q, <i>J</i> = 269.3), 124.60, 128.14, 137.71 (q, <i>J</i> = 39.6), 138.27, 144.66, 155.98, 157.18, 166.64. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 504. HRMS (ESI) Calcd for C<sub>24</sub>H<sub>28</sub>F<sub>3</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 504.2329; Found 504.2327.</div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> (<i>R</i>)-1,3-Dimethyl-4-(2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>8</b>)</h3><div class="NLM_p last">DIPEA (0.23 mL, 1.4 mmol) was added to (<i>R</i>)-1,3-dimethyl-4-(2-(4-(piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>31</b>; 0.14 g, 0.34 mmol) and 6-chloro-3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (75 mg, 0.34 mmol) in DMA (2.0 mL) at 20 °C under nitrogen. The resulting solution was stirred at 80 °C for 1 h then cooled to room temperature. The cooled reaction mixture was then partitioned between DCM (50 mL) and water (100 mL) and extracted. The aqueous was extracted twice more with DCM (2 × 20 mL). The combined organic layers were passed through a phase separating cartridge and evaporated to give a yellow gum. The crude product was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μ silica, 19 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford (<i>R</i>)-1,3-dimethyl-4-(2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>8</b>; 0.12 g, 68%) as a brown solid. <sup>1</sup>H NMR (700 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 2H), 1.64 (qd, <i>J</i> = 3.7, 12.8 Hz, 2H), 1.86 (d, <i>J</i> = 12.4 Hz, 2H), 2.67 (ddd, <i>J</i> = 4.8, 7.0, 12.1 Hz, 1H), 2.72–2.78 (m, 1H), 2.79 (s, 3H), 2.90 (dt, <i>J</i> = 5.9, 13.5 Hz, 1H), 3.02–3.1 (m, 2H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.19–3.26 (m, 2H), 4.03 (t, <i>J</i> = 5.8 Hz, 1H), 4.38 (d, <i>J</i> = 13.4 Hz, 2H), 6.85 (d, <i>J</i> = 8.6 Hz, 2H), 7.15 (d, <i>J</i> = 8.6 Hz, 2H), 7.63 (d, <i>J</i> = 10.3 Hz, 1H), 8.21 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.97, 32.32, 33.62, 40.52, 45.81, 45.84, 47.39, 52.01, 59.61, 65.87, 114.34, 116.77, 118.61 (q, <i>J</i> = 269.3), 124.04, 127.53, 137.16 (q, <i>J</i> = 39.7), 137.51, 144.10, 155.42, 156.78, 169.23. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 518. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>30</sub>F<sub>3</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 518.2486; Found 518.2488.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> (<i>R</i>)-4-(2-(4-(1-([1,2,4]Triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>9</b>)</h3><div class="NLM_p">Zinc powder (65 mg, 0.99 mmol) was added to (<i>R</i>)-4-(2-(4-(1-(3-bromo-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>32</b>; 0.11 g, 0.20 mmol) in AcOH (4.0 mL). The resulting mixture was stirred for 20 min. The reaction mixture was evaporated to dryness, redissolved in EtOAc (25 mL), and washed sequentially with saturated NaHCO<sub>3</sub> (2 × 25 mL), water (25 mL), and saturated brine (25 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. This was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μ silica, 19 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford (<i>R</i>)-4-(2-(4-(1-(3-bromo-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (25.00 mg, 23.81%) as a white solid and (<i>R</i>)-4-(2-(4-(1-([1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>9</b>; 27 mg, 30%) as a white solid. <sup>1</sup>H NMR (700 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.64 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.83 (d, <i>J</i> = 12.8 Hz, 2H), 2.67 (ddd, <i>J</i> = 4.8, 7.0, 12.1 Hz, 1H), 2.73–2.79 (m, 2H), 2.79 (s, 3H), 2.90 (dt, <i>J</i> = 5.9, 13.6 Hz, 1H), 3–3.06 (m, 3H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.2–3.26 (m, 2H), 4.03 (t, <i>J</i> = 5.8 Hz, 2H), 4.33 (d, <i>J</i> = 13.3 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.40 (d, <i>J</i> = 10.2 Hz, 1H), 8.02–8.06 (m, 1H), 9.17 (d, <i>J</i> = 0.7 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.98, 32.42, 33.63, 40.65, 45.83, 46.10, 47.39, 52.01, 59.60, 65.87, 114.34, 115.17, 123.90, 127.53, 137.66, 138.21, 141.56, 155.08, 156.75, 169.24. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 450. HRMS (ESI) Calcd for C<sub>24</sub>H<sub>31</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 450.2612; Found 450.2612.</div><div class="NLM_p">The following compounds were synthesized using analogous procedures.</div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (<i>R</i>)-1,3-Dimethyl-4-(2-(4-(1-(3-methyl-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>10</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.64 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.84 (d, <i>J</i> = 12.5 Hz, 2H), 2.54 (s, 3H), 2.67 (ddd, <i>J</i> = 4.8, 7.1, 12.1 Hz, 1H), 2.72–2.78 (m, 2H), 2.79 (s, 3H), 2.90 (dt, <i>J</i> = 5.9, 13.6 Hz, 1H), 2.98–3.06 (m, 3H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.19–3.26 (m, 2H), 4.01–4.04 (m, 2H), 4.35 (d, <i>J</i> = 13.3 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.35 (d, <i>J</i> = 10.2 Hz, 1H), 7.97 (d, <i>J</i> = 10.2 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 9.14, 14.97, 32.42, 33.62, 40.63, 45.84, 46.04, 47.39, 52.01, 59.61, 65.87, 114.00, 114.33, 124.01, 127.52, 137.67, 141.76, 145.16, 154.73, 156.75, 169.23. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 464. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>33</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 464.2769; Found 464.2766.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (<i>R</i>)-4-(2-(4-(1-(3-Chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>11</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.64 (qd, <i>J</i> = 3.7, 12.7, 12.8, 12.8 Hz, 2H), 1.85 (d, <i>J</i> = 12.5 Hz, 2H), 2.67 (ddd, <i>J</i> = 4.8, 6.9, 12.1 Hz, 1H), 2.72–2.78 (m, 2H), 2.79 (s, 3H), 2.87–2.93 (m, 1H), 3.01–3.08 (m, 3H), 3.13 (q, <i>J</i> = 6.8, 6.8, 6.8 Hz, 1H), 3.19–3.26 (m, 2H), 4.03 (t, <i>J</i> = 5.7, 5.7 Hz, 2H), 4.39 (d, <i>J</i> = 13.3 Hz, 2H), 6.85 (d, <i>J</i> = 8.6 Hz, 2H), 7.15 (d, <i>J</i> = 8.6 Hz, 2H), 7.48 (d, <i>J</i> = 10.2 Hz, 1H), 8.06 (d, <i>J</i> = 10.2 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.97, 32.40, 33.62, 40.57, 45.83, 45.88, 47.39, 52.00, 59.60, 65.87, 114.33, 115.49, 124.19, 127.52, 134.33, 137.56, 143.31, 155.22, 156.76, 169.22. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 484. HRMS (ESI) Calcd for C<sub>24</sub>H<sub>30</sub>ClN<sub>3</sub>O<sub>2</sub> [<sup>35</sup> M+H]<sup>+</sup> 484.2222; Found 484.2226.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (<i>R</i>)-1,3-Dimethyl-4-(2-(4-(1-(3-(methylthio)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>12</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.64 (qd, <i>J</i> = 3.8, 12.8 Hz, 2H), 1.85 (d, <i>J</i> = 12.6 Hz, 2H), 2.53 (s, 3H), 2.65–2.69 (m, 2H), 2.73–2.78 (m, 1H), 2.79 (s, 3H), 2.91 (dt, <i>J</i> = 5.9, 13.6 Hz, 1H), 3.01–3.06 (m, 3H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.19–3.26 (m, 2H), 4.03 (t, <i>J</i> = 6.2 Hz, 2H), 4.35 (d, <i>J</i> = 13.2 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.16 (d, <i>J</i> = 8.7 Hz, 2H), 7.39 (d, <i>J</i> = 10.2 Hz, 1H), 8.01 (d, <i>J</i> = 10.2 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 13.41, 14.99, 32.39, 33.63, 40.41, 40.61, 45.84, 45.99, 47.40, 52.01, 59.61, 65.88, 114.35, 114.61, 124.10, 127.54, 137.65, 143.41, 144.16, 154.79, 156.76, 169.24. Purity 97% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 496. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>33</sub>N<sub>7</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 496.2489; Found 496.2488.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (<i>R</i>)-4-(2-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>13</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.62 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.82 (d, <i>J</i> = 11.3 Hz, 2H), 2.67 (ddd, <i>J</i> = 4.7, 7.1, 12.1 Hz, 1H), 2.7–2.78 (m, 2H), 2.79 (s, 3H), 2.90 (dt, <i>J</i> = 5.9, 13.6 Hz, 1H), 2.97 (td, <i>J</i> = 2.1, 13.0 Hz, 2H), 3.04 (dt, <i>J</i> = 5.0, 12.4 Hz, 1H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.19–3.26 (m, 2H), 4.01–4.04 (m, 2H), 4.17 (s, 3H), 4.28 (d, <i>J</i> = 13.3 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.14 (d, <i>J</i> = 8.7 Hz, 2H), 7.27 (d, <i>J</i> = 10.3 Hz, 1H), 7.84 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.97, 32.42, 33.62, 40.63, 45.84, 46.09, 47.39, 52.01, 57.67, 59.61, 65.87, 114.33, 114.38, 124.31, 127.50, 137.66, 140.39, 153.61, 154.43, 156.74, 169.22. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 480. Enantiomeric purity >98% as measured by chiral HPLC. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>33</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup> 480.2718; Found 480.2718.</div></div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> (<i>R</i>)-4-(2-(4-(1-(3-(Dimethylamino)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>14</b>)</h3><div class="NLM_p">A solution of (<i>R</i>)-1,3-dimethyl-4-(2-(4-(piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>31</b>; 0.21 g, 0.63 mmol) and 3,6-dichloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (0.12 g, 0.63 mmol) dissolved in DMF (3.0 mL) was treated with <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (0.17 mL, 0.95 mmol) under nitrogen. The resulting mixture was stirred at 20 °C for 16 h. The mixture was treated with dimethylamine in THF (2.0 M, 1.3 mL, 2.5 mmol) and heated in a sealed tube in microwave at 140 °C for 72 h. The reaction mixture was evaporated to afford crude product. The crude product was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μ silica, 50 mm diameter, 150 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford (<i>R</i>)-4-(2-(4-(1-(3-(dimethylamino)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>14</b>; 0.15 g, 48%) as a pale yellow solid. <sup>1</sup>H NMR (700 MHz, DMSO) 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.64 (qd, <i>J</i> = 3.8, 12.7 Hz, 2H), 1.83 (d, <i>J</i> = 12.1 Hz, 2H), 2.67 (ddd, <i>J</i> = 4.8, 7.0, 12.0 Hz, 1H), 2.71–2.78 (m, 2H), 2.79 (s, 3H), 2.90 (dt, <i>J</i> = 5.9, 13.6 Hz, 1H), 2.99 (t, <i>J</i> = 11.9 Hz, 2H), 3.01–3.06 (m, 1H), 3.04 (s, 6H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.19–3.26 (m, 2H), 4.02 (t, <i>J</i> = 5.7 Hz, 2H), 4.25 (d, <i>J</i> = 13.2 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.21 (d, <i>J</i> = 10.2 Hz, 1H), 7.82 (d, <i>J</i> = 10.2 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) 14.98, 32.36, 33.62, 39.58, 40.41, 40.63, 45.84, 46.26, 47.39, 52.01, 59.61, 65.87, 113.09, 114.34, 124.44, 127.51, 137.69, 140.85, 152.01, 153.75, 156.74, 169.23. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 493. HRMS (ESI) Calcd for C<sub>26</sub>H<sub>36</sub>N<sub>8</sub>O<sub>2</sub> [M + H]<sup>+</sup> 493.3034; Found 493.3032.</div><div class="NLM_p">The following compounds were synthesized using analogous procedures.</div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (<i>S</i>)-4-(2-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3- dimethylpiperazin-2-one (<b>15</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.62 (qd, <i>J</i> = 3.8, 12.7 Hz, 2H), 1.82 (d, <i>J</i> = 12.2 Hz, 2H), 2.67 (ddd, <i>J</i> = 4.8, 6.9, 12.1 Hz, 1H), 2.7–2.77 (m, 2H), 2.79 (s, 3H), 2.90 (dt, <i>J</i> = 5.9, 13.4 Hz, 1H), 2.97 (t, <i>J</i> = 11.9 Hz, 2H), 3.04 (dt, <i>J</i> = 5.0, 12.2 Hz, 1H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.19–3.26 (m, 2H), 4.03 (t, <i>J</i> = 5.7 Hz, 2H), 4.17 (s, 3H), 4.28 (d, <i>J</i> = 13.1 Hz, 2H), 6.85 (d, <i>J</i> = 8.6 Hz, 2H), 7.14 (d, <i>J</i> = 8.6 Hz, 2H), 7.28 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.2 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) 14.98, 32.42, 33.63, 40.63, 45.84, 46.09, 47.39, 52.01, 57.68, 59.61, 65.87, 114.33, 114.40, 124.31, 127.51, 137.67, 140.40, 153.61, 154.43, 156.74, 169.23. Purity 98% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 480. Enantiomeric purity >98% as measured by chiral HPLC. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>33</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 480.2718; Found 480.2722.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1-((3<i>S</i>,5<i>R</i>)-4-(2-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-3,5-dimethylpiperazin-1-yl)ethanone (<b>16</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.04 (d, <i>J</i> = 6.2 Hz, 3H), 1.05 (d, <i>J</i> = 6.1 Hz, 3H), 1.61 (qd, <i>J</i> = 12.7, 3.6 Hz, 2H), 1.82 (d, <i>J</i> = 12.3 Hz, 2H), 1.96 (s, 3H), 2.22–2.27 (m, 1H), 2.45–2.49 (m, 1H), 2.55–2.61 (m, 1H), 2.69–2.75 (m, 2H), 2.94–3 (m, 4H), 3.62 (d, <i>J</i> = 13.0 Hz, 1H), 3.93 (t, <i>J</i> = 6.3 Hz, 2H), 4.14 (d, <i>J</i> = 12.7 Hz, 1H), 4.17 (s, 3H), 4.27 (d, <i>J</i> = 13.1 Hz, 2H), 6.83 (d, <i>J</i> = 8.6 Hz, 2H), 7.14 (d, <i>J</i> = 8.6 Hz, 2H), 7.27 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 17.52, 17.74, 20.89, 32.42, 40.63, 46.08, 46.74, 47.48, 52.24, 54.22, 54.76, 57.67, 65.26, 114.20, 114.38, 124.31, 127.50, 137.61, 140.39, 153.60, 154.42, 156.76, 167.44. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 508. HRMS (ESI) Calcd for C<sub>27</sub>H<sub>37</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup> 508.3031; Found 508.3030.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 1-((2<i>S</i>,6<i>R</i>)-4-(2-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-2,6-dimethylpiperazin-1-yl)ethanone (<b>17</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.20 (br s, 6H), 1.62 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.82 (d, <i>J</i> = 11.3 Hz, 2H), 1.98 (s, 3H), 2.14 (br s, 2H), 2.68 (t, <i>J</i> = 5.7 Hz, 2H), 2.71–2.77 (m, 3H), 2.97 (td, <i>J</i> = 2.1, 13.0 Hz, 2H), 4.07 (t, <i>J</i> = 5.7 Hz, 2H), 4.17 (s, 3H), 4.28 (d, <i>J</i> = 13.0 Hz, 2H), 6.86 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.28 (d, <i>J</i> = 10.3 Hz, 1H), 7.84 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 21.17, 32.43, 40.64, 46.09, 56.43, 57.67, 57.79, 65.89, 114.39, 114.49, 124.31, 127.51, 137.69, 140.40, 153.61, 154.43, 156.86, 168.38. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 508. HRMS (ESI) Calcd for C<sub>27</sub>H<sub>37</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup> 508.3031; Found 508.3030.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (3<i>R</i>,5<i>R</i>)-4-(2-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3,5-trimethylpiperazin-2-one (<b>18</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.02 (d, <i>J</i> = 6.5 Hz, 3H), 1.21 (d, <i>J</i> = 7.0 Hz, 3H), 1.61 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.82 (d, <i>J</i> = 12.6 Hz, 2H), 2.69–2.76 (m, 2H), 2.78 (s, 3H), 2.87 (dt, <i>J</i> = 6.1, 12.6 Hz, 1H), 2.94–3.00 (m, 2H), 3.06 (dd, <i>J</i> = 7.6, 11.9 Hz, 1H), 3.23 (dd, <i>J</i> = 4.2, 11.9 Hz, 1H), 3.25–3.32 (m, 2H), 3.92–3.98 (m, 2H), 4.17 (s, 3H), 4.28 (d, <i>J</i> = 13.3 Hz, 2H), 6.83 (d, <i>J</i> = 8.7 Hz, 2H), 7.14 (d, <i>J</i> = 8.7 Hz, 2H), 7.28 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 13.49, 16.27, 32.42, 33.71, 40.63, 46.10, 46.68, 47.03, 52.90, 57.67, 57.74, 67.16, 114.31, 114.40, 124.31, 127.50, 137.62, 140.40, 153.61, 154.44, 156.79, 169.58. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 494. HRMS (ESI) Calcd for C<sub>26</sub>H<sub>35</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup> 494.2874; Found 494.2871.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (<i>R</i>)-4-(3-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)propyl)-1,3-dimethylpiperazin-2-one (<b>19</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.18 (d, <i>J</i> = 6.8 Hz, 3H), 1.61 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.78–1.86 (m, 3H), 2.41 (ddd, <i>J</i> = 5.7, 7.4, 12.9 Hz, 1H), 2.46–2.51 (m, 3H), 2.68–2.75 (m, 2H), 2.79 (s, 3H), 2.94–3.01 (m, 3H), 3.18–3.26 (m, 2H), 3.95 (t, <i>J</i> = 6.3 Hz, 2H), 4.17 (s, 3H), 4.28 (d, <i>J</i> = 13.3 Hz, 2H), 6.83 (d, <i>J</i> = 8.7 Hz, 2H), 7.14 (d, <i>J</i> = 8.7 Hz, 2H), 7.28 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.87, 26.13, 32.42, 33.62, 40.62, 45.10, 46.09, 47.39, 49.53, 57.67, 59.64, 65.41, 114.28, 114.39, 124.31, 127.48, 137.51, 140.39, 153.61, 154.43, 156.95, 169.29, 170.19. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 494. HRMS (ESI) Calcd for C<sub>26</sub>H<sub>35</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup> 494.2874; Found 494.2874.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>R</i>)-4-(4-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)butyl)-1,3-dimethylpiperazin-2-one (<b>20</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.18 (d, <i>J</i> = 6.8 Hz, 3H), 1.49–1.57 (m, 2H), 1.61 (qd, <i>J</i> = 4.1, 13.0 Hz, 2H), 1.65–1.74 (m, 2H), 1.82 (d, <i>J</i> = 12.3 Hz, 2H), 2.33 (ddd, <i>J</i> = 5.6, 7.9, 12.9 Hz, 1H), 2.45 (dt, <i>J</i> = 5.8, 12.0 Hz, 1H), 2.58 (dt, <i>J</i> = 7.6, 12.6 Hz, 1H), 2.72 (tt, <i>J</i> = 3.5, 12.0 Hz, 1H), 2.78 (s, 3H), 2.91–3 (m, 4H), 3.20 (t, <i>J</i> = 5.5 Hz, 2H), 3.92 (t, <i>J</i> = 6.5 Hz, 2H), 4.17 (s, 3H), 4.27 (d, <i>J</i> = 13.3 Hz, 2H), 6.83 (d, <i>J</i> = 8.7 Hz, 2H), 7.13 (d, <i>J</i> = 8.7 Hz, 2H), 7.28 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.73, 22.68, 26.41, 32.43, 33.61, 40.62, 44.81, 46.10, 47.37, 52.47, 57.67, 59.55, 67.16, 114.27, 114.39, 124.31, 127.47, 137.45, 140.40, 153.61, 154.44, 156.96, 169.38. Purity 99% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 508. HRMS (ESI) Calcd for C<sub>27</sub>H<sub>37</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup> 508.3031; Found 508.3038.</div></div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (<i>R</i>)-4-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenethyl)-1,3-dimethylpiperazin-2-one (<b>21</b>)</h3><div class="NLM_p last">DIPEA (1.4 mL, 7.9 mmol) was added to (<i>R</i>)-1,3-dimethyl-4-(4-(piperidin-4-yl)phenethyl)piperazin-2-one (<b>48</b>; 0.83 g, 2.6 mmol) and 6-chloro-3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>36</b>; 0.58 g, 3.2 mmol) in ethanol (25 mL). The resulting solution was stirred at 50 °C for 20 h. The reaction mixture was evaporated to dryness, redissolved in MeOH, and purified by ion exchange chromatography, using an SCX column. The desired product was eluted from the column using 1 M NH<sub>3</sub>/MeOH and evaporated to dryness to give an orange gum. This was further purified by flash silica chromatography, elution gradient 0–8% 7 M NH<sub>3</sub>/MeOH in EtOAc. Fractions containing product were evaporated to dryness to afford a yellow gum. This was further purified by flash alumina chromatography, elution gradient 0–10% MeOH in MeCN. Pure fractions were evaporated to dryness to afford (<i>R</i>)-4-(4-(1-(3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenethyl)-1,3-dimethylpiperazin-2-one (<b>21</b>; 0.44 g, 36%) as a white solid. <sup>1</sup>H NMR (700 MHz, DMSO) δ 1.18 (d, <i>J</i> = 6.8 Hz, 3H), 1.64 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.83 (d, <i>J</i> = 12.5 Hz, 2H), 2.53–2.66 (m, 3H), 2.67–2.78 (m, 3H), 2.79 (s, 3H), 2.95–3.02 (m, 3H), 3.07 (q, <i>J</i> = 6.8 Hz, 1H), 3.22 (t, <i>J</i> = 5.5 Hz, 2H), 4.17 (s, 3H), 4.28 (d, <i>J</i> = 13.3 Hz, 2H), 7.12–7.16 (m, 4H), 7.28 (d, <i>J</i> = 10.3 Hz, 1H), 7.84 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.68, 31.96, 32.24, 33.63, 41.11, 45.02, 46.07, 47.36, 54.85, 57.67, 59.16, 114.38, 124.32, 126.46, 128.58, 137.92, 140.40, 143.03, 153.61, 154.43, 169.34. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 464. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>33</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 464.2769; Found 464.2768.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (<i>R</i>)-4-(1-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenyl)azetidin-3-yl)-1,3-dimethylpiperazin-2-one (<b>22</b>)</h3><div class="NLM_p">To a mixture of (<i>R</i>)-4-(azetidin-3-yl)-1,3-dimethylpiperazin-2-one (<b>49</b>; 56 mg, 0.31 mmol) and 6-(4-(4-bromophenyl)piperidin-1-yl)-3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>50</b>; 0.12 g, 0.30 mmol) in toluene (3.0 mL) was added Ruphos palladium(II) phenethylamine chloride (18 mg, 0.020 mmol), dicyclohexyl(2′,6′-diisopropoxy-[1,1′-biphenyl]-2-yl)phosphine (9.0 mg, 0.020 mmol), and cesium carbonate (0.31 g, 0.96 mmol). The reaction was sparged with nitrogen for 5 min and then heated to 90 °C, stirring under nitrogen for 16 h. The reaction was cooled before being diluted with toluene (15 mL). Insoluble material was removed by filtration, and the filtrate was evaporated to dryness. The residue was partitioned between DCM (20 mL) and water (20 mL), and the organic layer was separated by elution through a phase separating cartridge. The aqueous layer was extracted with more DCM (20 mL), and the combined organics were evaporated to afford a yellow gum, which was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μ silica, 19 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 1% NH<sub>3</sub>) and MeCN as eluents. Fractions containing the desired compound were evaporated to dryness to afford a gum (42 mg). This was taken up in a small amount of DCM and re-evaporated to afford (<i>R</i>)-4-(1-(4-(1-(3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenyl)azetidin-3-yl)-1,3-dimethylpiperazin-2-one (<b>22</b>; 42 mg, 28%) as an off white foam. <sup>1</sup>H NMR (700 MHz, DMSO) δ 1.16 (d, <i>J</i> = 7.0 Hz, 3H), 1.59 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.80 (d, <i>J</i> = 11.6 Hz, 2H), 2.60 (dt, <i>J</i> = 4.7, 12.5 Hz, 1H), 2.66 (tt, <i>J</i> = 3.4, 12.0 Hz, 1H), 2.79 (s, 3H), 2.9–2.99 (m, 3H), 3.16–3.2 (m, 2H), 3.26–3.31 (m, 1H), 3.48–3.52 (m, 1H), 3.51–3.55 (m, 1H), 3.68 (p, <i>J</i> = 6.3 Hz, 1H), 3.88 (t, <i>J</i> = 7.1 Hz, 1H), 3.92 (t, <i>J</i> = 6.9 Hz, 1H), 4.17 (s, 3H), 4.27 (d, <i>J</i> = 13.3 Hz, 2H), 6.37 (d, <i>J</i> = 8.5 Hz, 2H), 7.04 (d, <i>J</i> = 8.5 Hz, 2H), 7.27 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.2 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.33, 32.55, 33.59, 40.23, 40.70, 46.14, 47.35, 50.99, 55.45, 57.10, 57.67, 57.86, 111.49, 114.39, 124.29, 126.86, 134.28, 140.39, 150.19, 153.61, 154.43, 169.14. Purity 97% by LC-MS, <i>m</i>/<i>z</i>: ES+ [M + H]+ 491. HRMS (ESI) Calcd for C<sub>25</sub>H<sub>33</sub>N<sub>7</sub>O<sub>2</sub> [M + H]<sup>+</sup> 491.2878; Found 491.2876.</div><div class="NLM_p">The following compounds were synthesized using analogous procedures.</div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>R</i>)-4-(1-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenyl)piperidin-4-yl)-1,3-dimethylpiperazin-2-one (<b>23</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.38 (qd, <i>J</i> = 3.9, 12.1 Hz, 1H), 1.54–1.64 (m, 3H), 1.7–1.75 (m, 1H), 1.81 (d, <i>J</i> = 13.1 Hz, 3H), 2.57–2.7 (m, 5H), 2.79 (s, 3H), 2.89 (ddd, <i>J</i> = 4.5, 6.4, 11.1 Hz, 1H), 2.93–2.99 (m, 2H), 3.14–3.22 (m, 2H), 3.33 (q, <i>J</i> = 6.8 Hz, 1H), 3.62 (d, <i>J</i> = 12.4 Hz, 2H), 4.17 (s, 3H), 4.27 (d, <i>J</i> = 13.3 Hz, 2H), 6.84 (d, <i>J</i> = 8.7 Hz, 2H), 7.06 (d, <i>J</i> = 8.7 Hz, 2H), 7.27 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.88, 25.71, 29.87, 32.41, 33.59, 40.00, 40.54, 46.11, 47.99, 48.26, 48.36, 55.31, 56.44, 57.67, 114.39, 115.86, 124.30, 126.97, 135.66, 140.39, 149.36, 153.61, 154.44, 169.75. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 519. HRMS (ESI) Calcd for C<sub>28</sub>H<sub>38</sub>N<sub>8</sub>O<sub>2</sub> [M + H]<sup>+</sup> 519.3191; Found 519.3183.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>R</i>)-4-((1-(4-(1-(3-Methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenyl)azetidin-3-yl)methyl)-1,3-dimethylpiperazin-2-one (<b>24</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO) δ 1.20 (d, <i>J</i> = 6.8 Hz, 3H), 1.59 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.80 (d, <i>J</i> = 11.5 Hz, 2H), 2.49–2.52 (m, 1H), 2.56 (dd, <i>J</i> = 6.6, 12.6 Hz, 1H), 2.63–2.69 (m, 1H), 2.79 (s, 3H), 2.8–2.82 (m, 1H), 2.85–2.9 (m, 1H), 2.92–3.02 (m, 4H), 3.22 (t, <i>J</i> = 5.5 Hz, 2H), 3.38 (dt, <i>J</i> = 6.3, 11.7 Hz, 2H), 3.84 (dt, <i>J</i> = 7.3, 17.2 Hz, 2H), 4.16 (s, 3H), 4.27 (d, <i>J</i> = 13.3 Hz, 2H), 6.34 (d, <i>J</i> = 8.5 Hz, 2H), 7.03 (d, <i>J</i> = 8.5 Hz, 2H), 7.28 (d, <i>J</i> = 10.3 Hz, 1H), 7.83 (d, <i>J</i> = 10.3 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO, 30 °C) δ 14.93, 27.54, 32.58, 33.61, 40.72, 45.09, 46.15, 47.30, 56.08, 56.17, 57.05, 57.68, 59.63, 111.22, 113.68, 123.81, 126.83, 134.00, 140.39, 150.48, 153.61, 154.44, 169.23. Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 505. HRMS (ESI) Calcd for C<sub>27</sub>H<sub>36</sub>N<sub>8</sub>O<sub>2</sub> [M + H]<sup>+</sup> 505.3034; Found 505.3038.</div></div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 4-(1-(3-(Trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenol (<b>25</b>)</h3><div class="NLM_p last">6-Chloro-3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (0.862 g, 3.87 mmol) was added to 4-(piperidin-4-yl)phenol hydrobromide (1 g, 3.87 mmol) and DIPEA (2.024 mL, 11.62 mmol) in MeCN (40 mL). The resulting mixture was stirred for 18 h. The reaction mixture was evaporated to dryness, redissolved in EtOAc (100 mL), and washed sequentially with water (100 mL) and saturated brine (100 mL). The organic layer was dried with MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 0–5% MeOH in DCM. Pure fractions were evaporated to dryness to give a pale yellow semisolid. This was triturated with DCM to give a solid, which was collected by filtration and dried under vacuum to give 4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenol (<b>25</b>; 0.928 g, 65.9%) as a white powder. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.62 (qd, <i>J</i> = 3.9, 12.8 Hz, 2H), 1.84 (d, <i>J</i> = 11.0 Hz, 2H), 2.73 (tt, <i>J</i> = 3.4, 11.9 Hz, 1H), 3.06 (td, <i>J</i> = 2.1, 13.0 Hz, 2H), 4.37 (d, <i>J</i> = 13.4 Hz, 2H), 6.65–6.69 (m, 2H), 7.01–7.05 (m, 2H), 7.62 (d, <i>J</i> = 10.4 Hz, 1H), 8.20 (d, <i>J</i> = 10.4 Hz, 1H), 9.11 (br s, 1H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 364.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 2-(4-(1-(3-(Trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethanol (<b>26</b>)</h3><div class="NLM_p last">A solution of ethylene carbonate (18.18 g, 206.4 mmol) in DMF (30 mL) was added dropwise to a stirred suspension of 4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenol (<b>25</b>; 15.00 g, 41.28 mmol) and potassium carbonate (11.41 g, 82.57 mmol) in DMF (30 mL) at 80 °C over a period of 10 min under nitrogen. The resulting mixture was stirred at 80 °C for 20 h. The reaction mixture was cooled to room temperature then concentrated, diluted with DCM (500 mL), and washed sequentially with water (500 mL) and saturated brine (250 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution gradient 70–100% EtOAc in isohexane. Pure fractions were evaporated to dryness to afford 2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethanol (<b>26</b>; 12.04 g, 71.6%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.65 (qd, <i>J</i> = 3.9, 12.7 Hz, 2H), 1.87 (d, <i>J</i> = 10.8 Hz, 2H), 2.74–2.84 (m, 1H), 3.04–3.13 (m, 2H), 3.68 (q, <i>J</i> = 5.2 Hz, 2H), 3.94 (t, <i>J</i> = 5.1 Hz, 2H), 4.39 (d, <i>J</i> = 13.4 Hz, 2H), 4.77 (t, <i>J</i> = 5.5 Hz, 1H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.15 (d, <i>J</i> = 8.7 Hz, 2H), 7.63 (d, <i>J</i> = 10.4 Hz, 1H), 8.21 (d, <i>J</i> = 10.4 Hz, 1H). Purity 99% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 408.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 2-(4-(1-(3-(Trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethylmethanesulfonate (<b>27</b>)</h3><div class="NLM_p last">A solution of methanesulfonyl chloride (2.7 mL, 35 mmol) in DCM (40 mL) was added dropwise to a stirred solution of 2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethanol (<b>26</b>; 12 g, 30 mmol) and triethylamine (8.2 mL, 59 mmol) in DCM (120 mL) cooled to 0 °C under nitrogen. The resulting solution was stirred at 0 °C for 15 min then warmed to room temperature and stirred for a further 15 min. The reaction mixture was diluted with DCM (100 mL) and washed with water (250 mL) and saturated brine (100 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford 2-(4-(1-(3-(trifluoromethyl)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethylmethanesulfonate (<b>27</b>; 14 g, 100%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.70 (qd, <i>J</i> = 3.9, 12.7 Hz, 2H), 1.92 (d, <i>J</i> = 10.8 Hz, 2H), 2.68–2.75 (m, 1H), 2.96–3.10 (m, 5H), 4.17 (t, <i>J</i> = 5.1 Hz, 2H), 4.30 (d, <i>J</i> = 10.2 Hz, 2H), 4.50 (t, <i>J</i> = 5.1 Hz, 2H), 6.78–6.81 (m, 2H), 7.04–7.10 (m, 3H), 7.86 (d, <i>J</i> = 8.7 Hz, 1H). Purity 96% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 486.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>tert</i>-Butyl 4-(4-hydroxyphenyl)piperidine-1-carboxylate (<b>28</b>)</h3><div class="NLM_p last">Triethylamine (48 mL, 0.340 mol) was added slowly to 4-(piperidin-4-yl)phenol hydrobromide (80 g, 0.310 mol) in DCM (1.5 L) at 0 °C. The resulting mixture was stirred for 20 min, and then di-<i>tert</i>-butyl dicarbonate (74 g, 0.34 mol) was added. The ice bath was removed, and the reaction was stirred at ambient temperature for 2 h. The reaction mixture was washed sequentially with water (2 × 1 L) and saturated brine (1 L). The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. This was purified by flash silica chromatography, elution gradient 0–5% MeOH in DCM. Pure fractions were evaporated to dryness to afford <i>tert</i>-butyl 4-(4-hydroxyphenyl)piperidine-1-carboxylate (<b>28</b>; 76 g, 88%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.36 (qd, <i>J</i> = 4.3, 12.7 Hz, 2H), 1.40 (s, 9H), 1.68 (d, <i>J</i> = 12.9 Hz, 2H), 2.53 (tt, <i>J</i> = 3.5, 12.1 Hz, 1H), 2.69–2.82 (m, 2H), 4.03 (d, <i>J</i> = 12.6 Hz, 2H), 6.66 (d, <i>J</i> = 8.5 Hz, 2H), 7.00 (d, <i>J</i> = 8.5 Hz, 2H), 9.10 (br s, 1H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>–</sup> [M-H]<sup>−</sup> 276.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>tert</i>-Butyl 4-(4-(2-chloroethoxy)phenyl)piperidine-1-carboxylate (<b>29</b>)</h3><div class="NLM_p last">Potassium carbonate (78 g, 0.56 mol) was added to a stirred solution of <i>tert</i>-butyl 4-(4-hydroxyphenyl)piperidine-1-carboxylate (<b>28</b>; 26 g, 94 mmol) and 1-bromo-2-chloroethane (47 mL, 0.56 mol) in acetone (200 mL). The reaction was stirred at 90 °C for 40 h. The mixture was filtered and evaporated to dryness to afford crude <i>tert</i>-butyl 4-(4-(2-chloroethoxy)phenyl)piperidine-1-carboxylate (<b>29</b>; 31 g, 97%) as a pale yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.40 (s, 9H), 1.45 (qd, <i>J</i> = 4.1, 12.6 Hz, 2H), 1.70 (d, <i>J</i> = 12.7 Hz, 2H), 2.60 (td, <i>J</i> = 3.3, 12.0 Hz, 1H), 2.67–2.86 (m, 2H), 3.89–3.93 (m, 2H), 4.04 (d, <i>J</i> = 10.4 Hz, 2H), 4.18–4.22 (m, 2H), 6.87 (d, <i>J</i> = 8.7 Hz, 2H), 7.14 (d, <i>J</i> = 8.6 Hz, 2H). Purity 91% by LC-MS.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> (<i>R</i>)-<i>tert</i>-Butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethoxy)phenyl)piperidine-1-carboxylate (<b>30</b>)</h3><div class="NLM_p last">DIPEA (72 mL, 0.41 mmol) was added to a suspension of (<i>R</i>)-1,3-dimethylpiperazin-2-one hydrochloride (21 g, 0.13 mol) in DMA. <i>tert</i>-Butyl 4-(4-(2-chloroethoxy)phenyl)piperidine-1-carboxylate (<b>29</b>; 31 g, 91 mmol) and potassium iodide (18 g, 0.11 mol) were added, and the solution was heated to 110 °C for 24 h. The reaction mixture was diluted with water (400 mL) and extracted with DCM (2 × 300 mL). The combined organic layers were dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography, elution 10% MeOH in EtOAc. Pure fractions were evaporated to dryness to afford (<i>R</i>)-<i>tert</i>-butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethoxy)phenyl)piperidine-1-carboxylate (<b>30</b>; 36 g, 90%) as a brown oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.40 (d, <i>J</i> = 6.8 Hz, 3H), 1.48 (s, 9H), 1.57 (qd, <i>J</i> = 4.6, 13.5 Hz, 2H), 1.79 (d, <i>J</i> = 12.9 Hz, 2H), 2.59 (tt, <i>J</i> = 3.4, 12.1 Hz, 1H), 2.78 (ddd, <i>J</i> = 4.3, 7.6, 12.1 Hz, 2H), 2.86 (dt, <i>J</i> = 5.6, 13.8 Hz, 1H), 2.94 (s, 3H), 3–3.07 (1H, m), 3.15 (dt, <i>J</i> = 4.6, 12.4 Hz, 1H), 3.23–3.4 (m, 4H), 4.02–4.12 (m, 2H), 4.16–4.31 (m, 2H), 6.84 (d, <i>J</i> = 8.7 Hz, 2H), 7.11 (d, <i>J</i> = 8.7 Hz, 2H). Purity 98% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 432.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> (<i>R</i>)-1,3-Dimethyl-4-(2-(4-(piperidin-4-yl)phenoxy)ethyl)piperazin-2-one dihydrochloride (<b>31</b>)</h3><div class="NLM_p last">Hydrogen chloride in dioxane (4M, 79 mL, 0.32 mol) was added to a suspension of (<i>R</i>)-<i>tert</i>-butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethoxy)phenyl)piperidine-1-carboxylate (<b>30</b>; 34 g, 79 mmol) in Et<sub>2</sub>O (500 mL). The solution was stirred at 20 °C for 2 h. The reaction mixture was evaporated to afford (<i>R</i>)-1,3-dimethyl-4-(2-(4-(piperidin-4-yl)phenoxy)ethyl)piperazin-2-one dihydrochloride (<b>31</b>; 33 g) as a beige solid, which was used without purification. <sup>1</sup>H NMR (500 MHz, DMSO, 100 °C) δ 1.53 (3H, d), 1.89–1.95 (3H, m), 2.77–2.85 (1H, m), 2.88 (3H, s), 2.91–3.02 (2H, m), 3.29–3.41 (3H, m), 3.4–3.58 (2H, m), 3.6–3.7 (1H, m), 3.78–3.86 (1H, m), 4.33–4.45 (2H, m), 6.93–6.99 (2H, m), 7.15–7.21 (2H, m), 9.09 (2H, br s). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 332.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> (<i>R</i>)-4-(2-(4-(1-(3-Bromo-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>32</b>)</h3><div class="NLM_p last">DIPEA (0.16 mL, 0.90 mmol) was added to 3-bromo-6-chloro-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (0.14 g, 0.60 mmol) and (<i>R</i>)-1,3-dimethyl-4-(2-(4-(piperidin-4-yl)phenoxy)ethyl)piperazin-2-one (<b>31</b>; 0.20 g, 0.60 mmol) in DMF (3.0 mL). The resulting solution was stirred for 18 h and then concentrated to dryness. The crude product was purified by flash silica chromatography, elution gradient 0–5% 7 M NH<sub>3</sub>/MeOH in DCM. Pure fractions were evaporated to dryness to afford (<i>R</i>)-4-(2-(4-(1-(3-bromo-[1,2,4]triazolo[4,3-<i>b</i>]pyridazin-6-yl)piperidin-4-yl)phenoxy)ethyl)-1,3-dimethylpiperazin-2-one (<b>32</b>; 0.22 g, 69%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.65 (qd, <i>J</i> = 3.8, 12.7 Hz, 2H), 1.86 (d, <i>J</i> = 11.4 Hz, 2H), 2.63–2.78 (m, 4H), 2.79 (s, 3H), 2.91 (dt, <i>J</i> = 5.9, 13.5 Hz, 1H), 3.04 (dt, <i>J</i> = 5.0, 12.3 Hz, 2H), 3.13 (q, <i>J</i> = 6.8 Hz, 1H), 3.2–3.25 (m, 2H), 4.03 (t, <i>J</i> = 5.8 Hz, 2H), 4.40 (d, <i>J</i> = 13.3 Hz, 2H), 6.85 (d, <i>J</i> = 8.7 Hz, 2H), 7.16 (d, <i>J</i> = 8.7 Hz, 2H), 7.49 (d, <i>J</i> = 10.3 Hz, 1H), 8.06 (d, <i>J</i> = 10.3 Hz, 1H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 530.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>′-(6-Chloropyridazin-3-yl)-1<i>H</i>-imidazole-1-carbothiohydrazide (<b>33</b>)</h3><div class="NLM_p last">3-Chloro-6-hydrazinylpyridazine (8.5 g, 59 mmol) was added portionwise to di(1<i>H</i>-imidazol-1-yl)methanethione (12 g, 65 mmol) in MeCN (80 mL) and water (8.9 mL) over a period of 30 min. The resulting solution was stirred at 24 °C for 1 h whereupon a precipitate had formed. This was filtered, and the residue was identified as potential product. The filtrate was acidified to pH 3 with 2 M HCl; the resulting orange solid was filtered, washed with water, and dried to give further product <i>N</i>′-(6-chloropyridazin-3-yl)-1<i>H</i>-imidazole-1-carbothiohydrazide (<b>33</b>; 16 g, 100%). The residues were combined and used without further purification in the next reaction. <sup>1</sup>H NMR (400 MHz, DMSO) δ 7.58 (s, 1H), 7.59 (s, 1H), 8.26 (s, 1H), 9.44, (br s, 1H), 9.46 (s, 1H). Purity 91% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>–</sup> [M-H]<sup>−</sup> 253.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (<i>E</i>)-Methyl <i>N</i>′-(6-chloropyridazin-3-yl)-1<i>H</i>-pyrazole-1-carbohydrazonothioate (<b>34</b>)</h3><div class="NLM_p last">Iodomethane (1.9 mL, 30 mmol) was added dropwise to <i>N</i>′-(6-chloropyridazin-3-yl)-1<i>H</i>-pyrazole-1-carbothiohydrazide (<b>33</b>; 9.5 g, 30 mmol) and potassium carbonate (8.3 g, 60 mmol) in THF (130 mL) and water (13 mL) over a period of 30 min. The resulting suspension was stirred at 24 °C for 3 h then concentrated <i>in vacuo</i>, and the crude was diluted with DCM (300 mL) and washed sequentially with water (100 mL) and saturated brine (100 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford the crude product, which was triturated with Et<sub>2</sub>O, and the solid was filtered to afford (<i>E</i>)-methyl <i>N</i>′-(6-chloropyridazin-3-yl)-1<i>H</i>-pyrazole-1-carbohydrazonothioate (<b>34</b>; 7.7 g, 97%). <sup>1</sup>H NMR (400 MHz, DMSO) δ 2.33 (s, 3H), 7.06–7.1 (m, 1H), 7.60 (s, 1H), 7.65 (s, 1H), 7.67–7.72 (m, 1H), 8.16 (s, 1H), 10.41 (br s, 1H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>–</sup> [M-H]<sup>−</sup> 267.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 6-Chloro-3-(methylthio)-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>35</b>)</h3><div class="NLM_p last">A slurry of (<i>E</i>)-methyl <i>N</i>′-(6-chloropyridazin-3-yl)-1<i>H</i>-pyrazole-1-carbohydrazonothioate (<b>34</b>; 5.1 g, 13.21 mmol) and hydrochloric acid (2 M, 50 mL, 1.0 mmol) in butan-1-ol (1.0 mL) was stirred at 70 °C for 1.5 weeks. The reaction was left to stand at room temperature overnight, whereupon solid precipitated out of solution. The reaction mixture was cooled in an ice bath, and further precipitation occurred. The reaction mixture was filtered, and the residue was identified as product. This was air-dried and collected to afford 6-chloro-3-(methylthio)-[1,2,4]trazolo[4,3-<i>b</i>]pyridazine (<b>35</b>; 0.54 g, 2.7 mmol, 20%). The filtrate contained product (by LCMS), so the mixture was concentrated in vacuo, and the crude product was purified by flash silica chromatography, elution gradient 0–100% EtOAc in DCM. Pure fractions were evaporated to dryness to afford 6-chloro-3-(methylthio)-[1,2,4]trazolo[4,3-<i>b</i>]pyridazine (<b>35</b>; 1.7 g, 8.6 mmol, 65%) as a cream solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 2.76 (s, 3H), 7.46 (d, <i>J</i> = 9.7 Hz, 1H), 8.42 (d, <i>J</i> = 9.7 Hz, 1H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 201.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 6-Chloro-3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>36</b>)</h3><div class="NLM_p last">3-Chloro-6-hydrazinylpyridazine (10 g, 69 mmol) was suspended in DME (200 mL) and treated with tetramethoxymethane (15 mL, 0.11 mol), and the resulting mixture was stirred at 90 °C for 24 h. The DME was evaporated off, and the residue was dissolved in 5% MeOH/DCM and then filtered through a silica plug. The filtrate was evaporated to dryness and then taken up in MTBE and slurried for 1 h. The solid was filtered and pulled dry to afford 6-chloro-3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>36</b>; 12 g, 91%) as a cream powder. <sup>1</sup>H NMR (400 MHz, DMSO) δ 4.25 (s, 3H), 7.30 (d, <i>J</i> = 9.8 Hz, 1H), 8.22 (d, <i>J</i> = 9.8 Hz, 1H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 185.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <i>N</i>-Acetyl-<i>N</i>-(2-(trifluoromethyl)phenyl)acetamide (<b>37</b>)</h3><div class="NLM_p last">Acetyl chloride (0.13 L, 1.9 mol) was added dropwise to a mixture of 2-(trifluoromethyl)aniline (0.10 kg, 0.62 mmol) and pyridine (200 mL, 2.48 mol) in toluene (500 mL) at ambient temperature. The reaction was then heated to 50 °C for 18 h. The pyridine and acetic acid byproducts were removed by aqueous extraction with 1 M citric acid solution (2 × 250 mL), and the toluene solution was concentrated via distillation. Crystallization was induced by addition of heptane (500 mL), and the mixture was cooled to 5 °C for isolation via vacuum filtration. This gave <i>N</i>-acetyl-<i>N</i>-(2-(trifluoromethyl)phenyl)acetamide (<b>37</b>; 93 g, 59%) as a beige powder. <sup>1</sup>H NMR (400 MHz, DMSO, 30 °C) δ 2.18 (6H, s), 7.58–7.93 (4H, m). Purity 78% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 246.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 1-((3<i>R</i>,5<i>S</i>)-3,5-Dimethylpiperazin-1-yl)ethanone (<b>38</b>)</h3><div class="NLM_p last"><i>N</i>-Acetyl-<i>N</i>-(2-(trifluoromethyl)phenyl)acetamide (<b>37</b>; 63 g, 0.25 mol) was added portionwise to (2<i>R</i>,6<i>S</i>)-2,6-dimethylpiperazine (27 g, 0.24 mol) in DCM (400 mL), and the reaction was stirred at ambient temperature for 18 h. The product was extracted into hydrochloric acid (2 M, 125 mL, 250 mmol), and the organic phase was discarded. The pH of the aqueous phase was adjusted to ∼14 by addition of sodium hydroxide solution (10 M). The product was then extracted into DCM (2 × 300 mL). The organic phases were combined, dried over MgSO<sub>4</sub>, and concentrated to dryness to afford 1-((3<i>R</i>,5<i>S</i>)-3,5-dimethylpiperazin-1-yl)ethanone (<b>38</b>; 35 g, 93%) as a yellow liquid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 0.93 (d, <i>J</i> = 6.5 Hz, 3H), 0.95 (d, <i>J</i> = 6.4 Hz, 3H), 1.92–1.98 (m, 1H), 1.96 (s, 3H), 2.47–2.51 (m, 2H), 2.57–2.66 (m, 1H), 3.60 (dt, <i>J</i> = 12.5, 2.5 Hz, 1H), 4.21 (dt, <i>J</i> = 12.5, 2.5 Hz, 1H).</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> (<i>R</i>)-<i>tert</i>-Butyl (1-(benzyl(methyl)amino)-1-oxopropan-2-yl)carbamate (<b>39</b>)</h3><div class="NLM_p last">Isobutyl chloroformate (32 mL, 250 mmol) in THF (150 mL) was added dropwise to (<i>R</i>)-2-((<i>tert</i>-butoxycarbonyl)amino)propanoic acid (45 g, 0.24 mmol) and triethylamine (38 mL, 0.27 mmol) in THF (750 mL) cooled to −30 °C over a period of 90 min under nitrogen. The resulting mixture was stirred at −30 °C for 30 min and then allowed to warm to ambient temperature and stirred for 16 h. The mixture was then cooled to 0 °C, and <i>N</i>-methyl-1-phenylmethanamine (33 mL, 0.26 mol) and triethylamine (42 mL, 300 mmol) in THF (150 mL) were added dropwise over 90 min under nitrogen. The resulting mixture was then allowed to warm to room temperature and stirred for 4 days. The reaction mixture was diluted with EtOAc (1.0 L) and washed sequentially with saturated NaHCO<sub>3</sub> (500 mL), water (500 mL), and saturated brine (500 mL). The combined aqueous washings were extracted with further EtOAc (1.0 L). The combined organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. This was purified by flash silica chromatography, elution gradient 10–35% EtOAc in heptane. Pure fractions were evaporated to dryness to afford (<i>R</i>)-<i>tert</i>-butyl (1-(benzyl(methyl)amino)-1-oxopropan-2-yl)carbamate (<b>39</b>; 33 g, 47%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.40 (s, 9H), 2.92 (s, 3H), 4.52 (q, <i>J</i> = 6.8 Hz, 1H), 4.55 (s, 2H), 6.17 (br s, 1H), 7.21–7.37 (m, 5H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M – BOC + H]<sup>+</sup> 193.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> (<i>R</i>)-2-Amino-<i>N</i>-benzyl-<i>N</i>-methylpropanamide (<b>40</b>)</h3><div class="NLM_p last">Trifluoroacetic acid (0.11 L) was added dropwise to (<i>R</i>)-<i>tert</i>-butyl (1-(benzyl(methyl)amino)-1-oxopropan-2-yl)carbamate (<b>39</b>; 33 g, 0.11 mol) in DCM (110 mL) cooled to 0 °C over a period of 1 h. The resulting solution was stirred at room temperature for 18 h and then evaporated to dryness. The residue was diluted with DCM (1.0 L) and stirred with saturated NaHCO<sub>3</sub> (1.0 L). Further solid NaHCO<sub>3</sub> was added to the stirring mixture until all effervescence had ceased. The organic layer was washed with water (200 mL) and then dried over MgSO<sub>4</sub>, filtered, and evaporated to afford desired product. This gave (<i>R</i>)-2-amino-<i>N</i>-benzyl-<i>N</i>-methylpropanamide (<b>40</b>; 8.7 g, 41%) as a pale yellow oil, which was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.15 (d, <i>J</i> = 6.6 Hz, 3H), 1.57 (br s, 2H), 2.91 (s, 3H), 3.80 (q, <i>J</i> = 6.6 Hz, 1H), 4.52 (d, <i>J</i> = 15.4 Hz, 1H), 4.59 (d, <i>J</i> = 15.4 Hz, 1H), 7.2–7.28 (m, 3H), 7.30–7.36 (2H, m).</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> (<i>R</i>)-<i>N</i>1-Benzyl-<i>N</i>1-methylpropane-1,2-diamine (<b>41</b>)</h3><div class="NLM_p last">Borane dimethylsulfide complex (5 M in diethyl ether, 0.14 L, 0.68 mol) was added dropwise to (<i>R</i>)-2-amino-<i>N</i>-benzyl-<i>N</i>-methylpropanamide (<b>40</b>; 8.7 g, 45 mmol) in THF (200 mL) at room temperature over a period of 90 min under nitrogen. The resulting solution was stirred at room temperature for 4 days. The mixture was then cooled to 0 °C and slowly quenched with dropwise addition of HCl (10 M) until pH 1 was achieved. The mixture was stirred for 1 h and then neutralized with sodium hydroxide solution (10 M). KOH (75 g) was then added, and the mixture was heated to 80 °C for 18 h. The reaction mixture was diluted with EtOAc (200 mL), and the layers were separated. The aqueous layer was extracted with further EtOAc (2 × 250 mL). The combined organic layer was washed with saturated brine (500 mL) and dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. This was purified by flash silica chromatography, eluting with 10% 3.5 M NH<sub>3</sub>/MeOH in DCM. Pure fractions were evaporated to dryness to afford (<i>R</i>)-<i>N</i>1-benzyl-<i>N</i>1-methylpropane-1,2-diamine (<b>41</b>; 7.3 g, 90%) as a pale yellow liquid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 0.90 (d, <i>J</i> = 6.3 Hz, 3H), 2.89–2.99 (2H, m), 2.09 (s, 3H), 2.89–2.99 (m, 1H), 3.40 (d, <i>J</i> = 13.3 Hz, 1H), 3.49 (d, <i>J</i> = 13.3 Hz, 1H), 7.19–7.25 (m, 1H), 7.26–7.34 (m, 4H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 179.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> (<i>R</i>)-Methyl 2-(((<i>R</i>)-1-(benzyl(methyl)amino)propan-2-yl)amino)propanoate (<b>42</b>)</h3><div class="NLM_p last">Trifluoromethanesulfonic anhydride (8.5 mL, 51 mmol) was added dropwise to (<i>S</i>)-methyl 2-hydroxypropanoate (4.7 mL, 49 mmol) in DCM (200 mL) cooled to 0 °C over a period of 40 min under nitrogen. The resulting solution was stirred at 0 °C for 10 min, and then 2,6-dimethylpyridine (6.6 mL, 57 mmol) in DCM (20 mL) was added dropwise. The resulting solution was stirred at 0 °C for a further 10 min, and then (<i>R</i>)-<i>N</i>1-benzyl-<i>N</i>1-methylpropane-1,2-diamine (7.3 g, 41 mmol) and triethylamine (9.1 mL, 66 mmol) in DCM (40 mL) were added dropwise over 90 min at 0 °C. The reaction mixture was then allowed to warm to room temperature and stirred for 18 h. The reaction mixture was diluted with DCM (400 mL) and washed sequentially with saturated NaHCO<sub>3</sub> (200 mL), water (2 × 200 mL), and saturated brine (200 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. This was purified by flash silica chromatography, eluting with EtOAc. Pure fractions were evaporated to dryness to afford (<i>R</i>)-methyl 2-(((<i>R</i>)-1-(benzyl(methyl)amino)propan-2-yl)amino)propanoate (<b>42</b>; 9.1 g, 84%) as a pale yellow liquid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 0.95 (d, <i>J</i> = 6.2 Hz, 3H), 1.18 (d, <i>J</i> = 6.8 Hz, 3H), 2.17 (s, 3H), 2.21 (dd, <i>J</i> = 5.7, 12.5 Hz, 1H), 2.35 (dd, <i>J</i> = 7.1, 12.5 Hz, 1H), 2.81 (dq, <i>J</i> = 6.2, 12.4 Hz, 2H), 3.44–3.52 (m, 3H), 3.64 (s, 3H), 7.18–7.25 (m, 1H), 7.26–7.34 (m, 4H). Purity 74% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 265.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> (3<i>R</i>,5<i>R</i>)-1,3,5-Trimethylpiperazin-2-one (<b>43</b>)</h3><div class="NLM_p last">(<i>R</i>)-Methyl 2-(((<i>R</i>)-1-(benzyl(methyl)amino)propan-2-yl)amino)propanoate (<b>42</b>; 9.1 g, 34 mmol) and 20% dihydroxypalladium on carbon (2.4 g, 3.4 mmol) in ethanol (170 mL) were stirred under an atmosphere of hydrogen for 3 days (NMR analysis of a dried aliquot after 4 h showed complete loss of the benzyl group, but the mixture was left for 3 days). The mixture was then filtered and evaporated to dryness. The crude product was purified by flash silica chromatography, elution gradient 0–10% 1 M NH<sub>3</sub>/MeOH in DCM. Pure fractions were evaporated to dryness to afford (3<i>R</i>,5<i>R</i>)-1,3,5-trimethylpiperazin-2-one (<b>43</b>; 3.0 g, 61%) as a colorless gum. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.06 (d, <i>J</i> = 6.2 Hz, 3H), 1.24 (d, <i>J</i> = 7.1 Hz, 3H), 2.15 (br s, 1H), 2.80 (s, 3H), 2.94–3.01 (m, 1H), 3.19–3.28 (m, 2H), 3.40 (q, <i>J</i> = 7.1 Hz, 1H). HRMS (ESI) Calcd for C<sub>7</sub>H<sub>14</sub>N<sub>2</sub>O [M + H]<sup>+</sup> 143.1179; Found 143.1179.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 4-Bromophenethylmethanesulfonate (<b>44</b>)</h3><div class="NLM_p last">Methanesulfonyl chloride (3.5 mL, 44 mmol) was added dropwise to 2-(4-bromophenyl)ethanol (7.5 g, 37 mmol) and DIPEA (9.7 mL, 56 mmol) in DCM (180 mL) cooled to 0 °C over a period of 10 min. The resulting solution was stirred at 0 °C for 2 h. The reaction mixture was washed with water (2 × 150 mL), dried over MgSO<sub>4</sub>, filtered, and evaporated to afford desired product. This gave 4-bromophenethylmethanesulfonate (<b>44</b>; 10 g, 99%) as an orange oil that crystallized on standing. <sup>1</sup>H NMR (400 MHz, DMSO) δ 2.97 (t, <i>J</i> = 6.6 Hz, 2H), 3.10 (s, 3H), 4.39 (t, <i>J</i> = 6.6 Hz, 2H), 7.26 (d, <i>J</i> = 8.5 Hz, 2H), 7.50 (d, <i>J</i> = 8.5 Hz, 2H). Purity 100% by LC-MS.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> (<i>R</i>)-4-(4-Bromophenethyl)-1,3-dimethylpiperazin-2-one (<b>45</b>)</h3><div class="NLM_p last">DIPEA (16 mL, 89 mmol) was added to 4-bromophenethylmethanesulfonate (<b>44</b>; 10 g, 36 mmol) and (<i>R</i>)-1,3-dimethylpiperazin-2-one hydrochloride (7.1 g, 43 mmol) in MeCN (100 mL). The resulting solution was stirred at 80 °C for 20 h. The reaction mixture was evaporated to dryness, redissolved in EtOAc (200 mL), and washed sequentially with water (100 mL) and saturated brine (100 mL). The combined aqueous washings were re-extracted with further EtOAc (200 mL). The combined organic layer was dried over MgSO<sub>4</sub>, filtered, and evaporated to afford crude product. This was purified by flash silica chromatography, elution gradient 0–5% 1 M NH<sub>3</sub>/MeOH in DCM. Pure fractions were evaporated to dryness to afford (<i>R</i>)-4-(4-bromophenethyl)-1,3-dimethylpiperazin-2-one (<b>45</b>; 1.1 g, 9.8%) as an orange oil. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.16 (d, <i>J</i> = 6.8 Hz, 3H), 2.53–2.78 (m, 5H), 2.79 (s, 3H), 2.97–3.08 (m, 2H), 3.22 (t, <i>J</i> = 5.5 Hz, 2H), 7.20 (d, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 8.4 Hz, 2H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 311.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> (<i>R</i>)-<i>tert</i>-Butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethyl)phenyl)-5,6-dihydropyridine-1(2<i>H</i>)-carboxylate (<b>46</b>)</h3><div class="NLM_p last">(Tetrakistriphenylphosphine)palladium(0) (0.41 g, 0.35 mmol) was added to (<i>R</i>)-4-(4-bromophenethyl)-1,3-dimethylpiperazin-2-one (<b>45</b>; 1.1 g, 3.5 mmol), <i>tert</i>-butyl 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-5,6-dihydropyridine-1(2<i>H</i>)-carboxylate (1.2 g, 3.9 mmol), and cesium carbonate (2.3 g, 7.0 mmol) in degassed 1,4-dioxane (10 mL) and water (3.0 mL). The resulting mixture was stirred at 100 °C for 18 h. The reaction mixture was evaporated to dryness, redissolved in EtOAc (100 mL), and washed sequentially with water (50 mL) and saturated brine (50 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered, and purified by by ion exchange chromatography, using an SCX column. The desired product was eluted from the column using 1 M NH<sub>3</sub>/MeOH and evaporated to dryness to give a yellow gum. This was further purified by flash silica chromatography, elution gradient 0–3% 1 M NH<sub>3</sub>/MeOH in DCM. Pure fractions were evaporated to dryness to afford (<i>R</i>)-<i>tert</i>-butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethyl)phenyl)-5,6-dihydropyridine-1(2<i>H</i>)-carboxylate (<b>46</b>; 1.2 g, 85%) as a pale yellow gum. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.45 (s, 9H), 2.44–2.48 (m, 2H), 2.6–2.77 (m, 4H), 2.78–2.87 (m, 3H), 3.03 (dt, <i>J</i> = 5.2, 12.4 Hz, 1H), 3.11 (q, <i>J</i> = 6.8 Hz, 1H), 3.25 (t, <i>J</i> = 5.5 Hz, 2H), 3.56 (t, <i>J</i> = 5.8 Hz, 2H), 4.00 (q, <i>J</i> = 2.7 Hz, 2H), 5.62 (s, 1H), 6.06 (tt, <i>J</i> = 1.5, 3.4 Hz, 1H), 7.19 (d, <i>J</i> = 8.3 Hz, 2H), 7.32 (d, <i>J</i> = 8.3 Hz, 2H). Purity 99% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 414.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> (<i>R</i>)-<i>tert</i>-Butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethyl)phenyl)piperidine-1-carboxylate (<b>47</b>)</h3><div class="NLM_p last">(<i>R</i>)-<i>tert</i>-Butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethyl)phenyl)-5,6-dihydropyridine-1(2<i>H</i>)-carboxylate (<b>46</b>; 1.2 g, 3.0 mmol) and 10% palladium on carbon (0.16 g, 0.15 mmol) in MeOH (15 mL) were stirred under an atmosphere of hydrogen for 16 h. The mixture was then filtered and evaporated to dryness to give (<i>R</i>)-<i>tert</i>-butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethyl)phenyl)piperidine-1-carboxylate (<b>47</b>; 1.1 g, 89%) as a pale green gum. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.44 (s, 9H), 1.45–1.57 (m, 2H), 1.78 (d, <i>J</i> = 9.9 Hz, 2H), 2.59–2.74 (m, 5H), 2.75–2.88 (m, 6H), 3.03 (dt, <i>J</i> = 5.2, 12.4 Hz, 1H), 3.11 (q, <i>J</i> = 6.8 Hz, 1H), 3.24 (q, <i>J</i> = 4.9, 5.3 Hz, 2H), 4.07 (dt, <i>J</i> = 2.2, 13.0 Hz, 2H), 7.1–7.17 (m, 4H). Purity 100% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 416.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> (<i>R</i>)-1,3-Dimethyl-4-(4-(piperidin-4-yl)phenethyl)piperazin-2-one (<b>48</b>)</h3><div class="NLM_p last">A solution of (<i>R</i>)-<i>tert</i>-butyl 4-(4-(2-(2,4-dimethyl-3-oxopiperazin-1-yl)ethyl)phenyl)piperidine-1-carboxylate (<b>47</b>; 1.1 g, 2.7 mmol) in DCM (5.0 mL) and TFA (5.0 mL) was stirred for 3 h. The crude product mixture was then evaporated to dryness, redissolved in MeOH, and purified by ion exchange chromatography, using an SCX column. The desired product was eluted from the column using 1 M NH<sub>3</sub>/MeOH, and pure fractions were evaporated to dryness to afford (<i>R</i>)-1,3-dimethyl-4-(4-(piperidin-4-yl)phenethyl)piperazin-2-one (<b>48</b>; 0.83 g, 99%) as a colorless gum. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.22 (d, <i>J</i> = 6.8 Hz, 3H), 1.52 (qd, <i>J</i> = 4.0, 12.3 Hz, 2H), 1.72 (d, <i>J</i> = 12.6 Hz, 2H), 2.52–2.8 (m, 9H), 2.82 (s, 3H), 2.99–3.14 (m, 4H), 3.25 (t, <i>J</i> = 5.5 Hz, 2H), 7.07–7.18 (m, 4H). Purity 92% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 316.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> (<i>R</i>)-<i>tert</i>-Butyl 3-(2,4-dimethyl-3-oxopiperazin-1-yl)azetidine-1-carboxylate (<b>48</b>)</h3><div class="NLM_p last">To a solution of <i>tert</i>-butyl 3-oxoazetidine-1-carboxylate (0.88 g, 5.2 mmol) in dichloromethane (20 mL) was added (<i>R</i>)-1,3-dimethylpiperazin-2-one hydrochloride (1.0 g, 6.3 mmol), followed by triethylamine (0.95 mL, 6.8 mmol). The yellow solution was stirred, under nitrogen, at room temperature for 90 min. Sodium triacetoxyborohydride (1.6 mg, 7.8 mmol) was added portionwise, at 5 min intervals, over 45 min, and the resulting mixture was stirred at room temperature for 16 h. The reaction mixture was cautiously quenched with MeOH (5 mL) and saturated aqueous NaHCO<sub>3</sub> (5 mL). The mixture was stirred for 1 h before partitioning between DCM (100 mL) and saturated NaHCO<sub>3</sub> solution (100 mL). The organics were separated, and the aqueous was further extracted with DCM (100 mL). The combined organics were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a yellow oil, which was purified by flash silica chromatography, elution gradient 0–5% 2 M NH<sub>3</sub>/MeOH in DCM. Product containing fractions were evaporated to dryness to afford (<i>R</i>)-<i>tert</i>-butyl 3-(2,4-dimethyl-3-oxopiperazin-1-yl)azetidine-1-carboxylate (1.1 mg, 75%) as a colorless gum, which required further purification by ion exchange chromatography, using an SCX-2 column. The column was first eluted through with methanol (∼75 mL), before the desired product was eluted from the column using 2 M NH<sub>3</sub>/MeOH (∼75 mL) and evaporated to dryness to afford (<i>R</i>)-<i>tert</i>-butyl 3-(2,4-dimethyl-3-oxopiperazin-1-yl)azetidine-1-carboxylate (<b>48</b>; 0.82 g, 55.8%) as a straw colored gum. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.28 (d, <i>J</i> = 7.0 Hz, 3H), 1.44 (s, 9H), 2.61–2.68 (m, 1H), 2.95 (s, 3H), 3.00 (ddd, <i>J</i> = 4.4, 9.0, 13.2 Hz, 1H), 3.23 (dt, <i>J</i> = 4.4, 11.8 Hz, 1H), 3.29–3.42 (m, 2H), 3.56 (tt, <i>J</i> = 5.6, 7.3 Hz, 1H), 3.81 (td, <i>J</i> = 5.6, 8.2 Hz, 2H), 3.96 (ddd, <i>J</i> = 7.6, 8.8, 11.0 Hz, 2H). HRMS (ESI) Calcd for C<sub>14</sub>H<sub>25</sub>N<sub>3</sub>O<sub>3</sub> [M + H] 228.1343 (−C<sub>4</sub>H<sub>8</sub>); Found 228.1343 (−C<sub>4</sub>H<sub>8</sub>).</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> (<i>R</i>)-4-(Azetidin-3-yl)-1,3-dimethylpiperazin-2-one (<b>49</b>)</h3><div class="NLM_p last">A solution of (<i>R</i>)-<i>tert</i>-butyl 3-(2,4-dimethyl-3-oxopiperazin-1-yl)azetidine-1-carboxylate (<b>48</b>; 0.81 g, 2.9 mmol) in dichloromethane (8.0 mL) was treated with trifluoroacetic acid (3.5 mL) and stirred at room temperature for 2 h. The reaction mixture was then poured onto an SCX-2 cartridge. The cartridge was eluted through with DCM (75 mL) and methanol (75 mL). The product was then eluted from the column with 2 M NH<sub>3</sub>/MeOH (75 mL) and evaporated to dryness to afford (<i>R</i>)-4-(azetidin-3-yl)-1,3-dimethylpiperazin-2-one (<b>49</b>; 0.54 mg, 100%) as a yellow oil, which was used without further purification. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.27 (d, <i>J</i> = 7.0 Hz, 3H), 2.57 (dt, <i>J</i> = 4.9, 12.4 Hz, 1H), 2.9–2.98 (m, 1H), 2.94, (s, 3H), 3.21–3.29 (m, 2H), 3.34 (ddd, <i>J</i> = 4.7, 8.0, 12.6 Hz, 1H), 3.51–3.67 (m, 4H). HRMS (ESI) Calcd for C<sub>9</sub>H<sub>17</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 184.1444; Found 184.1443.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 6-(4-(4-Bromophenyl)piperidin-1-yl)-3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>50</b>)</h3><div class="NLM_p last">A mixture of 6-chloro-3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>36</b>; 0.45 g, 2.5 mmol) and 4-(4-bromophenyl)piperidine hydrochloride (0.54 g, 2.0 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (10 mL) was treated with DIPEA (0.73 mL, 4.2 mmol). The resulting mixture was heated to 80 °C and stirred under nitrogen for 16 h. After cooling to room temperature, the reaction mixture was evapoarted to remove the bulk of the solvent. The residue was then drowned with water (30 mL) and stirred; no precipitate was observed, so the mixture was extracted with DCM (2 × 30 mL), and the combined extracts were washed with brine, dried over MgSO<sub>4</sub>, filtered, and evaporated. The crude product was purified by flash silica chromatography, elution gradient 0–10% MeOH in DCM. Product containing fractions were evaporated to dryness to afford 6-(4-(4-bromophenyl)piperidin-1-yl)-3-methoxy-[1,2,4]triazolo[4,3-<i>b</i>]pyridazine (<b>50</b>; 0.26 mg, 34%) as a peach colored solid. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.64 (qd, <i>J</i> = 3.9, 12.6 Hz, 2H), 1.85 (d, <i>J</i> = 13.5 Hz, 2H), 2.77–2.86 (m, 1H), 2.94–3.05 (m, 2H), 4.17 (s, 3H), 4.30 (d, <i>J</i> = 13.3 Hz, 2H), 7.23 (d, <i>J</i> = 8.4 Hz, 2H), 7.29 (d, <i>J</i> = 10.3 Hz, 1H), 7.47 (d, <i>J</i> = 8.5 Hz, 2H), 7.84 (d, <i>J</i> = 10.2 Hz, 1H). Purity 94% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [<sup>79</sup>M + H]<sup>+</sup> 390.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> (<i>R</i>)-<i>tert</i>-Butyl 4-(2,4-dimethyl-3-oxopiperazin-1-yl)piperidine-1-carboxylate (<b>51</b>)</h3><div class="NLM_p last">Sodium triacetoxyborohydride (3.1 g, 14 mmol) was added portionwise to (<i>R</i>)-1,3-dimethylpiperazin-2-one hydrochloride (1.9 g, 11 mmol) and <i>tert</i>-butyl 4-oxopiperidine-1-carboxylate (2.1 g, 10 mmol) in DCM (51 mL) at room temperature. The resulting mixture was stirred at room temperature for 16 h. The reaction mixture was cautiously quenched with MeOH, and the reaction mixture was diluted with DCM (50 mL) and washed with saturated NaHCO<sub>3</sub> solution. The organics were removed ,and the aqueous was further extracted with DCM (50 mL). The combined organics were washed with water (100 mL) and saturated brine (100 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a yellow oil. The crude product was purified by flash silica chromatography, elution gradient 0–5% 7 N NH<sub>3</sub>/MeOH in DCM. Pure fractions were evaporated to dryness to afford (<i>R</i>)-<i>tert</i>-butyl 4-(2,4-dimethyl-3-oxopiperazin-1-yl)piperidine-1-carboxylate (<b>51</b>; 2.2 g, 67%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.36 (d, <i>J</i> = 6.8 Hz, 3H), 1.46 (s, 9H), 1.65–1.89 (m, 2H), 2.64–2.82 (m, 4H), 2.94 (s, 3H), 2.95–3.08 (m, 2H), 3.19–3.32 (m, 2H), 3.48 (q, <i>J</i> = 6.8 Hz, 1H), 3.78–3.9 (m, 1H), 4.12 (d, <i>J</i> = 10.0 Hz, 2H).</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> (<i>R</i>)-1,3-Dimethyl-4-(piperidin-4-yl)piperazin-2-one (<b>52</b>)</h3><div class="NLM_p last">Trifluoroacetic acid (2.0 mL) was added to a stirred solution of (<i>R</i>)-<i>tert</i>-butyl 4-(2,4-dimethyl-3-oxopiperazin-1-yl)piperidine-1-carboxylate (<b>51</b>; 2.2 g, 6.9 mmol) in DCM. The reaction mixture was stirred at room temperature for 1 h, and then MeOH (0.20 mL) was added. The reaction mixture was stirred for a further 1 h, and then more trifluoroacetic acid (2.0 mL) was added and stirring continued for a further 3 days. The reaction mixture was purified by ion exchange chromatography, using an SCX column. The desired product was eluted from the column using 7 M NH<sub>3</sub>/MeOH and evaporated to dryness to afford a pale yellow oil. The crude product was purified by flash silica chromatography, elution gradient 0–10% 7 M NH<sub>3</sub>/MeOH in DCM. Pure fractions were evaporated to dryness to afford (<i>R</i>)-1,3-dimethyl-4-(piperidin-4-yl)piperazin-2-one (<b>52</b>; 0.96 g, 66%) as a colorless oil. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.25–1.35 (m, 1H), 1.36 (d, <i>J</i> = 6.8 Hz, 3H), 1.49 (qd, <i>J</i> = 4.1, 12.0 Hz, 1H), 1.7–1.84 (m, 2H), 2.56 (td, <i>J</i> = 2.6, 12.2 Hz, 1H), 2.6–2.78 (m, 3H), 2.93 (s, 3H), 2.98 (ddd, <i>J</i> = 4.4, 6.0, 12.4 Hz, 1H), 3.14 (d, <i>J</i> = 13.1 Hz, 2H), 3.19–3.33 (m, 2H), 3.45–3.52 (m, 1H).</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> (1-Benzhydrylazetidin-3-yl)methylmethanesulfonate (<b>53</b>)</h3><div class="NLM_p last">A solution of (1-benzhydrylazetidin-3-yl)methanol (1.4 g, 5.4 mmol) in DCM (35 mL) was treated with DIPEA (2.7 mL, 16 mmol). Methanesulfonyl chloride (0.60 mL, 7.7 mmol) was then added dropwise over 60 s, and the reaction was left to stir for 4 h. More DCM (50 mL) and water (75 mL) were then added, and the mixture was stirred for 10 min. The organic layer was then separated, washed with brine (75 mL), dried over MgSO<sub>4</sub>, filtered, and evaporated to afford the crude product, (1-benzhydrylazetidin-3-yl)methylmethanesulfonate (<b>53</b>) as a dark amber gum. This was purified by flash silica chromatography, elution gradient 25–75% EtOAc in heptane. Pure fractions were evaporated to dryness to afford (1-benzhydrylazetidin-3-yl)methylmethanesulfonate (<b>53</b>; 0.69 g, 38%) as a near colorless gum. <sup>1</sup>H NMR (400 MHz, DMSO) δ 2.74–2.86 (m, 1H), 2.95 (dd, <i>J</i> = 5.5, 8.1 Hz, 2H), 2.99 (s, 3H), 3.27 (t, <i>J</i> = 7.9 Hz, 2H), 4.33 (s, 1H), 4.39 (d, <i>J</i> = 7.0 Hz, 2H), 7.15–7.2 (m, 2H), 7.23–7.29 (m, 4H), 7.36–7.41 (m, 4H). Purity 96% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 332.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> (<i>R</i>)-4-((1-Benzhydrylazetidin-3-yl)methyl)-1,3-dimethylpiperazin-2-one (<b>54</b>)</h3><div class="NLM_p last">A mixture of (1-benzhydrylazetidin-3-yl)methylmethanesulfonate (<b>53</b>; 0.69 g, 2.1 mmol) and (<i>R</i>)-1,3-dimethylpiperazin-2-one (0.25 g, 2.0 mmol) in DMF (8.0 mL) was treated with DIPEA (0.7.0 mL, 4.0 mmol). Sodium iodide (0.29 g, 2.0 mmol) was then added, and reaction mixture was sealed into a microwave tube and heated in the microwave reactor to 100 °C for a total of 13 h. After cooling to room temperature, the reaction mixture was poured into water (50 mL) and extracted with ethyl acetate (3 × 40 mL). The combined extracts were washed with brine, dried over MgSO<sub>4</sub>, filtered, and evaporated. The crude product was purified by flash silica chromatography, elution gradient 0–10% MeOH in DCM. Product containing fractions were evaporated to dryness to afford (<i>R</i>)-4-((1-benzhydrylazetidin-3-yl)methyl)-1,3-dimethylpiperazin-2-one (<b>54</b>; 0.52 g, 73%) as an amber gum. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.33 (d, <i>J</i> = 6.8 Hz, 3H), 2.48 (ddd, <i>J</i> = 4.4, 7.5, 12.0 Hz, 1H), 2.54 (dd, <i>J</i> = 6.6, 12.4 Hz, 1H), 2.64 (hept, <i>J</i> = 6.8 Hz, 1H), 2.72–2.85 (m, 3H), 2.86–2.89 (m, 1H), 2.90 (s, 3H), 3.09 (q, <i>J</i> = 6.8 Hz, 1H), 3.14–3.21 (m, 1H), 3.26 (ddd, <i>J</i> = 4.4, 7.4, 11.7 Hz, 1H), 3.34 (dt, <i>J</i> = 7.1, 9.3 Hz, 2H), 4.30 (s, 1H), 7.14–7.2 (m, 2H), 7.22–7.29 (m, 4H), 7.36–7.42 (m, 4H). Purity 98% by LC-MS, <i>m</i>/<i>z</i>: ES<sup>+</sup> [M + H]<sup>+</sup> 364.</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> (<i>R</i>)-4-(Azetidin-3-ylmethyl)-1,3-dimethylpiperazin-2-one (<b>55</b>)</h3><div class="NLM_p last">A solution of (<i>R</i>)-4-((1-benzhydrylazetidin-3-yl)methyl)-1,3-dimethylpiperazin-2-one (<b>54</b>; 0.52 g, 1.4 mmol) in EtOH (15 mL) was treated with ammonium formate (0.63 g, 10 mmol). The mixture was evacuated and backfilled with nitrogen before addition of palladium 5 wt % on activated carbon (Wet, Degussa type E105CA/W; 0.34 g, 0.16 mmol). The reaction was then evacuated and backfilled with nitrogen once more and then heated to 50 °C. The reaction was incomplete, so the temperature was increased to 60 °C, and the reaction mixture was stirred for 18 h. After cooling to room temperature, the catalyst was removed by filtration, and the filtrate was evaporated and purified by ion exchange chromatography, using an SCX-2 column. The column was first eluted with methanol (75 mL), before the desired product was eluted from the column using 2 M NH<sub>3</sub>/MeOH (75 mL) and evaporated to dryness to afford (<i>R</i>)-4-(azetidin-3-ylmethyl)-1,3-dimethylpiperazin-2-one (<b>55</b>; 0.20 g, 72%) as a pale yellow oil. <sup>1</sup>H NMR (400 MHz, DMSO) δ 1.35 (d, <i>J</i> = 6.9 Hz, 3H), 2.52 (ddd, <i>J</i> = 4.4, 7.3, 12.0 Hz, 1H), 2.58 (q, <i>J</i> = 6.0 Hz, 1H), 2.83–2.92 (m, 3H), 2.93 (s, 3H), 3.12 (q, <i>J</i> = 6.8 Hz, 1H), 3.19–3.33 (m, 2H), 3.33–3.41 (m, 2H), 3.68 (dt, <i>J</i> = 7.6, 10.3 Hz, 2H). HRMS (ESI) Calcd for C<sub>10</sub>H<sub>19</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 198.1601; Found 198.1600.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> BRD4 Fluorescence Polarization Assay</h3><div class="NLM_p">Binding to the isolated BRD4 tandem domain constructs (WT, N140A and N433A, residues 44–460) was assessed using an in-house competitive displacement fluorescence polarization (FP) assay, measuring displacement of an Alexafluor 647-labeled probe derived from <b>2</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_001.pdf" class="ext-link">Supporting Information</a>). Test compounds were dissolved in DMSO to form 10 mM stock solutions then acoustically dispensed into black 384-well non binding small volume plates (Greiner Bio-One, Stonehouse, UK, Cat No. 784076) using an Echo 555 (Labcyte Inc., Sunnyvale, CA). Test compounds were screened in a 12-point concentration response format over a final concentration range of 33.3, 10.4, 4.2, 2.1, 0.33, 0.104, 0.042, 0.021, 0.0033, 0.00104, 0.00042, and 0.000033 μM. A final DMSO concentration of 0.3% was achieved after reagent additions. The assay was performed by dispensing 6 μL of isolated BRD4 and 6 μL of probe simultaneously into assay ready plates using the BioRaptr FRD Microfluidic workstation (Beckman Coulter, High Wycombe, UK) to give final BRD4 and probe concentrations of 3 nM and 2 nM, respectively. Plates were then spun at 1000 rpm for 2 min and incubated at room temperature for 1 h protected from light. Milli-polarization (mP) values were acquired by reading plates in FP mode (590 nm excitation and 675 nm emission wavelengths, 50 nm bandwidth) on a BMG Pherastar FS multimode platereader (BMG Labtech, Offenburg, Germany). Data were exported to Genedata Screener software package (Genedata, Basel, Switzerland) to generate sigmoidal dose–response curves, and competitive BRD4 binding was expressed as an IC<sub>50</sub> value generated by determining the compound concentration at which there was a 50% reduction in probe binding. IC<sub>50</sub> values were then converted to <i>K</i><sub>i</sub> values.</div><div class="NLM_p last">Compounds were tested in triplicate as a minimum, and the results were found to be highly reproducible. For example, <b>13</b> showed a standard deviation in p<i>K</i><sub>i</sub> values of 0.10, 0.046, and 0.11 across three measurements for WT, N140A, and N433A respectively.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Androgen Receptor Downregulation Assay</h3><div class="NLM_p last">The AR downregulation assay was carried out as described previously.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Compounds were tested in triplicate as a minimum and were found to be highly reproducible. For example, <b>13</b> showed a standard deviation in pEC<sub>50</sub> values of 0.14 across three measurements.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Estrogen Receptor Downregulation Assay</h3><div class="NLM_p last">The ER downregulation assay was carried out as described previously.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Compounds were tested in triplicate as a minimum and were found to be highly reproducible. For example, <b>13</b> showed a standard deviation in pEC<sub>50</sub> values of 0.20 across seven measurements.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> c-Myc Downregulation Assay</h3><div class="NLM_p">MM1.S cells were plated in 96-well plates at 200000 cells/well and treated with serial concentrations of compound 1 h after plating and grown for 16 h at 37 °C. Cells were transferred into a 96-well V-bottom plate and fixed sequentially with 2% paraformaldehyde for 10 min at 37 °C followed with 90% cold methanol for 10 min on ice. Cells were washed and blocked in incubation buffer (PBS + 0.5% BSA) for 10 min and incubated with anti-cMyc primary antibody (1:200, Cell Signaling Technology no. 5605) at RT for 1 h. Cells were washed twice and then stained with Alexa 488 conjugated anti-rabbit IgG (1:1000, Cell Signaling technology no. 4412) for 30 min at RT and washed again. The pellet was resuspended in incubation buffer and analyzed by flow cytometer (BD FACSCalibur) using the FL-1 detector. Data was analyzed using FlowJo software V7. Live cell population was selected by gating on FCS/SSC, and the geomean fluorescence of FL-1 signal was used to calculate pEC<sub>50</sub> values.</div><div class="NLM_p last">Compounds were tested in triplicate as a minimum and were found to be highly reproducible. For example, <b>13</b> showed a standard deviation in pEC<sub>50</sub> values of 0.15 across 12 measurements.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Cell Viability Assay</h3><div class="NLM_p">MM1.S cells were cultured in RPMI, 10% FBS, and 1% <span class="smallcaps smallerCapital">l</span>-glutamine. MV4-11 cells were cultured in IMDM, 10% FBS, and 1% <span class="smallcaps smallerCapital">l</span>-glutamine. Optimal 384 well seeding densities were premeasured for linear growth over 72 h. Thirty microliters of cells/well were seeded in black, flat bottom 384-well plates and treated with compounds for 72 h. The viable cells were measured using Alamar Blue reagents following manufacture’s protocol (Invitrogen), and the % viable cells were calculated by comparing to no-drug controls. Dose–response curves were plotted in GraphPad Prism and fitted using sigmoidal nonlinear regression to determine pGI<sub>50</sub> values.</div><div class="NLM_p last">Compounds were tested in triplicate as a minimum and were found to be highly reproducible. For example, <b>13</b> showed a standard deviation in pEC<sub>50</sub> values of 0.24 across nine measurements.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Pharmacokinetic Predictions</h3><div class="NLM_p">An early assessment of human dose from in vitro data can be achieved using the pharmacokinetic model shown in <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>. This relies on an estimate of <i>C</i><sub>min</sub> after the first dose as an input and assumes that exposure exceeding the free potency for the entire dosing interval is required for efficacy and is attained at steady state.<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/jm-2016-00070g_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span><span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/jm-2016-00070g_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span><span class="NLM_disp-formula" id="eq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/jm-2016-00070g_m003.gif" alt="" /></img><span class="labelSpan">(3)</span></span><span class="NLM_disp-formula" id="eq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/jm-2016-00070g_m004.gif" alt="" /></img><span class="labelSpan">(4)</span></span>where MEC<sub>ss</sub>= minimum effective concentration achieved at steady state, also known as <i>C</i><sub>min</sub> (potency corrected for binding to human plasma protein), τ = dosing interval (12 h part in this case), AR = accumulation ratio defined by <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>; <i>k</i><sub>elim</sub> = elimination rate (clearance/<i>V</i><sub>ss</sub>), <i>F</i> = oral bioavailability defined by <a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>, <i>F</i><sub>abs</sub> = fraction absorbed from the GI tract, <i>F</i><sub>G</sub> = fraction escaping first pass metabolism in the gut-wall (assumed to be 1), <i>k</i><sub>a</sub> = absorption rate estimated from <a class="ref internalNav" href="#eq4" aria-label="eq 4">eq 4</a>, and <i>Q</i><sub>h</sub> = liver blood flow (20 mL min<sup>–1</sup> kg<sup>–1</sup> for humans).</div><div class="NLM_p last">The <i>F</i><sub>abs</sub> and maximum absorbable dose (MAD) were estimated from physicochemical properties (thermodynamic solubility at pH 7.4, p<i>K</i><sub>a</sub>, and Caco-2 permeability in the apical to basolateral direction) using a simplified version of the GI-Sim algorithm and a predicted dosed from <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Xenograft Studies</h3><div class="NLM_p last">All animals were maintained under specific-pathogen-free conditions in an AAALAC-accredited facility, and irradiated food and autoclaved water were provided <i>ad libitum</i>. Animal protocols were approved by the AstraZeneca R&D Boston Institutional Animal Care and Use committee. All animal work was conducted in accordance with ARRIVE guidelines.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Female SCID mice aged 5–6 weeks were obtained from Charles River Laboratories (Wilmington, MA) and MV-4-11 human AML tumor cells (10<sup>7</sup>/mouse) were implanted subcutaneously in each mouse. Animals were randomized into groups of three for PD and eight for efficacy studies based on tumor volume and treated with either vehicle or <b>13</b> at 10 mg/kg. Compound <b>13</b> was formulated in 0.5% HPMC/0.1% Tween80 and dosed orally either once for PD or daily for 14 days for efficacy. Tumor length and width were measured by caliper, and tumor volume was calculated using the formula volume = (length × width<sup>2</sup>) × <i>P</i>/6.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Pharmacodynamic Assays</h3><div class="NLM_p last">Tumors were collected at 2 or 8 h post single dose from vehicle-treated mice and were collected from <b>13</b>-treated mice at 2, 4, or 8 h after a single dose. All tumors were cut in half and snap frozen for protein and RNA extraction, respectively.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Quantitative PCR</h3><div class="NLM_p last">Tumors were homogenized, and RNA was extracted and purified using an RNAeasy Mini kit according to manufacturer’s protocol (Qiagen, no. 74104). RNA was quantified on a Nanodrop 1000 spectrophotometer (Thermo Scientific), concentration was normalized across samples, and RNA was converted to cDNA with a High Capacity cDNA RT kit (Applied Biosystems, no. 4368814). cDNA was then quantified and diluted down to ∼100 ng/μL in DEPC treated RNase free water (Ambion, no. AM9915G). For each reaction in a 384-well Microamp Optical rection plate (Applied Biosystems, no. 4309849), 10 μL of 2× Taqman gene expression master mix (Applied Biosystems, no. 4369016) was mixed with 6 μL of RNase free DEPC water and 1 μL of 20× Myc Taqman probe (Applied Biosystems, Hs00153408_m1) or GAPDH Taqman probe (Applied Biosystems, no. 4352665). Sample cDNA (3μL, 300 ng total) was then added to each well of the plate containing the mixture. qPCR was run on an ABI Prism 7900HT machine (Applied Biosystems). Myc expression was measured and quantified relative to GAPDH expression for each sample and reported as relative expression. Each sample reaction was run in duplicate, and the average CT value was used to calculate expression relative to GAPDH.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Western Blot Analysis</h3><div class="NLM_p last">Tumors were homogenized in Swedish buffer (20 mM Tris pH 8.0, 150 mM NaCl, 1% NP-40, in 10 mL of lysis buffer, add 100 μL of Phosphatase Inhibitor Cocktail2, Sigma Cat no. P5726–5 ML, 100 μL of Phosphatase Inhibitor Cocktail 3, Sigma Cat no. P0044–5 ML, and 1 tablet of complete protease inhibitor cocktail, Roche no. 11836153001). Samples were analyzed under reduced conditions and loaded onto a 4–12% bis-Tris gel (Invitrogen). Electrophoresis was carried out, and proteins were transferred to nitrocellulose membranes, 0.45 μM pore size (Invitrogen LC2001). Nonspecific binding sites were blocked in Tris-buffered saline/0.1% Tween with 10% nonfat milk. Primary antibodies were incubated in Tris-buffered saline/0.1% Tween with 5% nonfat milk overnight at 4 °C. Membranes were washed five times for 10 min and incubated with a 1:5000 dilution of horseradish peroxidase-conjugated anti-mouse or anti-rabbit antibodies for 2 h. Blots were washed with TBST 5 times and developed with the ECL system (Amersham Biosciences) according to the manufacturer’s protocols.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="NOTES-d45e7377-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00070">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00070" class="ext-link">10.1021/acs.jmedchem.6b00070</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystallization and structure of BRD4(1) in complex with <b>13</b>, correlations of potency values between BRD4 FP potency, ER downregulation, c-Myc downregulation and growth inhibition, and Alexa647-labeled probe (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">List of compounds with key data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_001.pdf">jm6b00070_si_001.pdf (307.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_002.csv">jm6b00070_si_002.csv (0.76 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b00070" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20306" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20306" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edwin Clark</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c3a6a7b4aaadeda0afa2b1a883a2b0b7b1a2b9a6ada6a0a2eda0acae"><span class="__cf_email__" data-cfemail="197c7d6e7077377a75786b7259786a6d6b78637c777c7a78377a7674">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael J. Waring</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Northern
Institute for Cancer Research, School of Chemistry, Newcastle University, Bedson Building, Newcastle upon Tyne NE1 7RU, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#533e3a38367d2432213a3d34133d3624303220273f367d32307d2638"><span class="__cf_email__" data-cfemail="5a3733313f742d3b2833343d1a343f2d393b292e363f743b39742f31">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert H. Bradbury</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rowena Callis</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gregory R. Carr</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huawei Chen</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lyman Feron</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steve Glossop</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark A. Graham</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maureen Hattersley</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chris Jones</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott G. Lamont</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gilles Ouvry</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca, Chemin du Moulin de Vrilly, 51100 Reims, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anil Patel</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joe Patel</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca, Gatehouse Park, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alfred A. Rabow</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Craig A. Roberts</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen Stokes</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalie Stratton</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Graeme E. Walker</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lara Ward</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Whalley</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Whittaker</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gail Wrigley</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca,
Mereside, Alderley Park, Macclesfield, Cheshire SK10 4TG, U.K.</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i78" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i78"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i79" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i79"> Abbreviations</h2><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">BD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extraterminal</p></td></tr><tr><td class="NLM_term">BRD</td><td class="NLM_def"><p class="first last">bromodomain containing protein</p></td></tr><tr><td class="NLM_term">DR</td><td class="NLM_def"><p class="first last">downregulation</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor</p></td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicology</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">MAD</td><td class="NLM_def"><p class="first last">maximum absorbable dose</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMA</td><td class="NLM_def"><p class="first last">dimethylamine</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">MTBE</td><td class="NLM_def"><p class="first last">methyl <i>tert</i>-butyl ether</p></td></tr><tr><td class="NLM_term">DEPC</td><td class="NLM_def"><p class="first last">diethylpyrocarbonate</p></td></tr><tr><td class="NLM_term">TBST</td><td class="NLM_def"><p class="first last">Tris-buffered saline, 0.1% Tween 20</p></td></tr><tr><td class="NLM_term">BOC</td><td class="NLM_def"><p class="first last">di-<i>tert</i>-butyl dicarbonate</p></td></tr><tr><td class="NLM_term">SCX</td><td class="NLM_def"><p class="first last">strong cation exchange</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i80">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42414" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42414" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Druggability and structural classification of bromodomain acetyl-lysine sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7346</span><span class="NLM_x">–</span> <span class="NLM_lpage">7359</span><span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+and+structural+classification+of+bromodomain+acetyl-lysine+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0liYhwHTIBi0VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Nicodème, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-w.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicod%C3%A8meauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosmini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clément, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnathan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mordaunt, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimes, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajakane, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daugan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithers, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-w.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uings, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodème, E.</span><span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7501</span><span class="NLM_x">–</span> <span class="NLM_lpage">7515</span><span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0liYhwHTIBi0VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lh5oJQoZLZlZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Wu, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, C.-M.</span><span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">13141</span><span class="NLM_x">–</span> <span class="NLM_lpage">13145</span><span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.-Y.+Wuauthor=C.-M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lh5oJQoZLZlZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Coudé, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braun, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berrou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupont, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raffoux, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itzykson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delord, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riveiro, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herait, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baruchel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombret, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardin, C.</span><span> </span><span class="NLM_article-title">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-Myc in acute leukemia cells</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">17698</span><span class="NLM_x">–</span> <span class="NLM_lpage">17712</span><span class="refDoi"> DOI: 10.18632/oncotarget.4131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.18632%2Foncotarget.4131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=25989842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17698-17712&author=M.+M.+Coud%C3%A9author=T.+Braunauthor=J.+Berrouauthor=M.+Dupontauthor=S.+Bertrandauthor=A.+Masseauthor=E.+Raffouxauthor=R.+Itzyksonauthor=M.+Delordauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=A.+Baruchelauthor=H.+Dombretauthor=C.+Gardin&title=BET+inhibitor+OTX015+targets+BRD2+and+BRD4+and+decreases+c-Myc+in+acute+leukemia+cells&doi=10.18632%2Foncotarget.4131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span></div><div class="casAuthors">Coude Marie-Magdelaine; Braun Thorsten; Berrou Jeannig; Dupont Melanie; Bertrand Sibyl; Masse Aline; Raffoux Emmanuel; Itzykson Raphael; Baruchel Andre; Dombret Herve; Gardin Claude; Coude Marie-Magdelaine; Braun Thorsten; Gardin Claude; Raffoux Emmanuel; Itzykson Raphael; Dombret Herve; Delord Marc; Riveiro Maria E; Herait Patrice; Baruchel Andre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17698-712</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain (BRD) and extraterminal (BET) proteins including BRD2, BRD3 and BRD4 have been identified as key targets for leukemia maintenance.  A novel oral inhibitor of BRD2/3/4, the thienotriazolodiazepine compound OTX015, suitable for human use, is available.  Here we report its biological effects in AML and ALL cell lines and leukemic samples.  Exposure to OTX015 lead to cell growth inhibition, cell cycle arrest and apoptosis at submicromolar concentrations in acute leukemia cell lines and patient-derived leukemic cells, as described with the canonical JQ1 BET inhibitor.  Treatment with JQ1 and OTX15 induces similar gene expression profiles in sensitive cell lines, including a c-MYC decrease and an HEXIM1 increase.  OTX015 exposure also induced a strong decrease of BRD2, BRD4 and c-MYC and increase of HEXIM1 proteins, while BRD3 expression was unchanged. c-MYC, BRD2, BRD3, BRD4 and HEXIM1 mRNA levels did not correlate however with viability following exposure to OTX015.  Sequential combinations of OTX015 with other epigenetic modifying drugs, panobinostat and azacitidine have a synergic effect on growth of the KASUMI cell line.  Our results indicate that OTX015 and JQ1 have similar biological effects in leukemic cells, supporting OTX015 evaluation in a Phase Ib trial in relapsed/refractory leukemia patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSlHRu3Td8hIa_dlEzl_fBafW6udTcc2ebf3CSep8e2cbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D&md5=0db953a0a24b12ec2918a02d1a2df00b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4131%26sid%3Dliteratum%253Aachs%26aulast%3DCoud%25C3%25A9%26aufirst%3DM.%2BM.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DBerrou%26aufirst%3DJ.%26aulast%3DDupont%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DMasse%26aufirst%3DA.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DItzykson%26aufirst%3DR.%26aulast%3DDelord%26aufirst%3DM.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DBaruchel%26aufirst%3DA.%26aulast%3DDombret%26aufirst%3DH.%26aulast%3DGardin%26aufirst%3DC.%26atitle%3DBET%2520inhibitor%2520OTX015%2520targets%2520BRD2%2520and%2520BRD4%2520and%2520decreases%2520c-Myc%2520in%2520acute%2520leukemia%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17698%26epage%3D17712%26doi%3D10.18632%2Foncotarget.4131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Herait, P.</span><span> </span><span class="NLM_article-title">BET-bromodomain (BRD) inhibitor OTX015: final results of the dose-finding part of a phase I study in hematologic malignancies</span>. Presented at the 13th International Congress on Targeted Anticancer Therapies, March<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">, </span>Paris.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=P.+Herait&title=BET-bromodomain+%28BRD%29+inhibitor+OTX015%3A+final+results+of+the+dose-finding+part+of+a+phase+I+study+in+hematologic+malignancies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DHerait%26aufirst%3DP.%26atitle%3DBET-bromodomain%2520%2528BRD%2529%2520inhibitor%2520OTX015%253A%2520final%2520results%2520of%2520the%2520dose-finding%2520part%2520of%2520a%2520phase%2520I%2520study%2520in%2520hematologic%2520malignancies%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Bradbury, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acton, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broadbent, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatter, g.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayter, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jude, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loddick, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parveen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma-Singh, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomason, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trueman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. M.</span><span> </span><span class="NLM_article-title">Discovery of AZD3514, a small-molecule androgen receptor downregulator for treatment of advanced prostate cancer</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1945</span><span class="NLM_x">–</span> <span class="NLM_lpage">1948</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.02.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.bmcl.2013.02.056" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1945-1948&author=R.+H.+Bradburyauthor=D.+G.+Actonauthor=N.+L.+Broadbentauthor=A.+N.+Brooksauthor=G.+R.+Carrauthor=g.+Hatterauthor=B.+R.+Hayterauthor=K.+J.+Hillauthor=N.+J.+Howeauthor=R.+D.+Jonesauthor=D.+Judeauthor=S.+G.+Lamontauthor=S.+A.+Loddickauthor=H.+L.+McFarlandauthor=Z.+Parveenauthor=A.+A.+Rabowauthor=G.+Sharma-Singhauthor=N.+C.+Strattonauthor=A.+G.+Thomasonauthor=D.+Truemanauthor=G.+E.+Walkerauthor=S.+L.+Wellsauthor=J.+Wilsonauthor=J.+M.+Wood&title=Discovery+of+AZD3514%2C+a+small-molecule+androgen+receptor+downregulator+for+treatment+of+advanced+prostate+cancer&doi=10.1016%2Fj.bmcl.2013.02.056"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.02.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.02.056%26sid%3Dliteratum%253Aachs%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DActon%26aufirst%3DD.%2BG.%26aulast%3DBroadbent%26aufirst%3DN.%2BL.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DCarr%26aufirst%3DG.%2BR.%26aulast%3DHatter%26aufirst%3Dg.%26aulast%3DHayter%26aufirst%3DB.%2BR.%26aulast%3DHill%26aufirst%3DK.%2BJ.%26aulast%3DHowe%26aufirst%3DN.%2BJ.%26aulast%3DJones%26aufirst%3DR.%2BD.%26aulast%3DJude%26aufirst%3DD.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DLoddick%26aufirst%3DS.%2BA.%26aulast%3DMcFarland%26aufirst%3DH.%2BL.%26aulast%3DParveen%26aufirst%3DZ.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DSharma-Singh%26aufirst%3DG.%26aulast%3DStratton%26aufirst%3DN.%2BC.%26aulast%3DThomason%26aufirst%3DA.%2BG.%26aulast%3DTrueman%26aufirst%3DD.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWells%26aufirst%3DS.%2BL.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26atitle%3DDiscovery%2520of%2520AZD3514%252C%2520a%2520small-molecule%2520androgen%2520receptor%2520downregulator%2520for%2520treatment%2520of%2520advanced%2520prostate%2520cancer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1945%26epage%3D1948%26doi%3D10.1016%2Fj.bmcl.2013.02.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Loddick, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomason, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunkley, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brave, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broadbent, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trueman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouchet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaheen, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cumberbatch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradbury, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaughan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Critchlow, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wedge, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, A. N.</span><span> </span><span class="NLM_article-title">AZD3514: a small molecule that modulates androgen receptor signaling and function <i>in vitro</i> and <i>in vivo</i></span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1715</span><span class="NLM_x">–</span> <span class="NLM_lpage">1727</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-12-1174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1158%2F1535-7163.MCT-12-1174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=23861347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2ltbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=1715-1727&author=S.+A.+Loddickauthor=S.+J.+Rossauthor=A.+G.+Thomasonauthor=D.+M.+Robinsonauthor=G.+E.+Walkerauthor=T.+P.+Dunkleyauthor=S.+R.+Braveauthor=N.+Broadbentauthor=N.+C.+Strattonauthor=D.+Truemanauthor=E.+Mouchetauthor=F.+S.+Shaheenauthor=V.+N.+Jacobsauthor=M.+Cumberbatchauthor=J.+Wilsonauthor=R.+D.+Jonesauthor=R.+H.+Bradburyauthor=A.+A.+Rabowauthor=L.+Gaughanauthor=C.+Womackauthor=S.+T.+Barryauthor=C.+N.+Robsonauthor=S.+E.+Critchlowauthor=S.+R.+Wedgeauthor=A.+N.+Brooks&title=AZD3514%3A+a+small+molecule+that+modulates+androgen+receptor+signaling+and+function+in+vitro+and+in+vivo&doi=10.1158%2F1535-7163.MCT-12-1174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AZD3514: A Small Molecule That Modulates Androgen Receptor Signaling and Function In Vitro and In Vivo</span></div><div class="casAuthors">Loddick, Sarah A.; Ross, Sarah J.; Thomason, Andrew G.; Robinson, David M.; Walker, Graeme E.; Dunkley, Tom P. J.; Brave, Sandra R.; Broadbent, Nicola; Stratton, Natalie C.; Trueman, Dawn; Mouchet, Elizabeth; Shaheen, Fadhel S.; Jacobs, Vivien N.; Cumberbatch, Marie; Wilson, Joanne; Jones, Rhys D. O.; Bradbury, Robert H.; Rabow, Alfred; Gaughan, Luke; Womack, Chris; Barry, Simon T.; Robson, Craig N.; Critchlow, Susan E.; Wedge, Stephen R.; Brooks, A. Nigel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1715-1727</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Continued androgen receptor (AR) expression and signaling is a key driver in castration-resistant prostate cancer (CRPC) after classical androgen ablation therapies have failed, and therefore remains a target for the treatment of progressive disease.  Here, we describe the biol. characterization of AZD3514, an orally bioavailable drug that inhibits androgen-dependent and -independent AR signaling.  AZD3514 modulates AR signaling through two distinct mechanisms, an inhibition of ligand-driven nuclear translocation of AR and a downregulation of receptor levels, both of which were obsd. in vitro and in vivo.  AZD3514 inhibited testosterone-driven seminal vesicle development in juvenile male rats and the growth of androgen-dependent Dunning R3327H prostate tumors in adult rats.  Furthermore, this class of compd. showed antitumor activity in the HID28 mouse model of CRPC in vivo.  AZD3514 is currently in phase I clin. evaluation.  Mol Cancer Ther; 12(9); 1715-27. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruUjB28XcSVrVg90H21EOLACvtfcHk0lhW2VLoa3hY_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2ltbfI&md5=553f0953189e9e5982a08c7933566ce0</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-12-1174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-12-1174%26sid%3Dliteratum%253Aachs%26aulast%3DLoddick%26aufirst%3DS.%2BA.%26aulast%3DRoss%26aufirst%3DS.%2BJ.%26aulast%3DThomason%26aufirst%3DA.%2BG.%26aulast%3DRobinson%26aufirst%3DD.%2BM.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DDunkley%26aufirst%3DT.%2BP.%26aulast%3DBrave%26aufirst%3DS.%2BR.%26aulast%3DBroadbent%26aufirst%3DN.%26aulast%3DStratton%26aufirst%3DN.%2BC.%26aulast%3DTrueman%26aufirst%3DD.%26aulast%3DMouchet%26aufirst%3DE.%26aulast%3DShaheen%26aufirst%3DF.%2BS.%26aulast%3DJacobs%26aufirst%3DV.%2BN.%26aulast%3DCumberbatch%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DR.%2BD.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DGaughan%26aufirst%3DL.%26aulast%3DWomack%26aufirst%3DC.%26aulast%3DBarry%26aufirst%3DS.%2BT.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26aulast%3DCritchlow%26aufirst%3DS.%2BE.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26atitle%3DAZD3514%253A%2520a%2520small%2520molecule%2520that%2520modulates%2520androgen%2520receptor%2520signaling%2520and%2520function%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26volume%3D12%26spage%3D1715%26epage%3D1727%26doi%3D10.1158%2F1535-7163.MCT-12-1174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Waring, M. J.; Chen, H.; Rabow, A. A.; Walker, G.; Bobby, R.; Boiko, S.; Bradbury, R. H.; Callis, R.; Clark, E.; Dale, I.; Daniels, D. L.; Dulak, A.; Flavell, L.; Holdgate, G.; Jowitt, T. A.; Kikhney, A.; McAlister, M.; Méndez, J.; Ogg, D.; Patel, J.; Petteruti, P.; Robb, G. R.; Robers, M. B.; Stratton, N.; Svergun, D. I.; Wang, W.; Whittaker, D.; Wilson, D. M.; Yao, Y.</span><span> </span><span class="NLM_article-title">Potent and selective bivalent inhibitors of BET bromodomains</span>. Nat. Chem. Biol.<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">, </span>submitted for publication</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+J.+Waring&author=H.+Chen&author=A.+A.+Rabow&author=G.+Walker&author=R.+Bobby&author=S.+Boiko&author=R.+H.+Bradbury&author=R.+Callis&author=E.+Clark&author=I.+Dale&author=D.+L.+Daniels&author=A.+Dulak&author=L.+Flavell&author=G.+Holdgate&author=T.+A.+Jowitt&author=A.+Kikhney&author=M.+McAlister&author=J.+M%C3%A9ndez&author=D.+Ogg&author=J.+Patel&author=P.+Petteruti&author=G.+R.+Robb&author=M.+B.+Robers&author=N.+Stratton&author=D.+I.+Svergun&author=W.+Wang&author=D.+Whittaker&author=D.+M.+Wilson&author=Y.+Yao&title=Potent+and+selective+bivalent+inhibitors+of+BET+bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DPotent%2520and%2520selective%2520bivalent%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strain-Damerell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess-Brown, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span> </span><span class="NLM_article-title">Assessing cellular efficacy of bromodomain inhibitors using fluorescence recovery after photobleaching</span> <span class="citation_source-journal">Epigenet. Chromatin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1186/1756-8935-7-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1186%2F1756-8935-7-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=25097667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=14-25&author=M.+Philpottauthor=C.+M.+Rogersauthor=C.+Yappauthor=C.+Wellsauthor=J.-P.+Lambertauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=A.-C.+Gingrasauthor=S.+Knappauthor=S.+M%C3%BCller&title=Assessing+cellular+efficacy+of+bromodomain+inhibitors+using+fluorescence+recovery+after+photobleaching&doi=10.1186%2F1756-8935-7-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing cellular efficacy of bromodomain inhibitors using fluorescence recovery after photobleaching</span></div><div class="casAuthors">Philpott, Martin; Rogers, Catherine M.; Yapp, Clarence; Wells, Chris; Lambert, Jean-Philippe; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Gingras, Anne-Claude; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14/1-14/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Acetylation of lysine residues in histone tails plays an important role in the regulation of gene transcription.  Bromdomains are the readers of acetylated histone marks, and, consequently, bromodomain-contg. proteins have a variety of chromatin-related functions.  Moreover, they are increasingly being recognized as important mediators of a wide range of diseases.  The first potent and selective bromodomain inhibitors are beginning to be described, but the diverse or unknown functions of bromodomain-contg. proteins present challenges to systematically demonstrating cellular efficacy and selectivity for these inhibitors.  Here we assess the viability of fluorescence recovery after photobleaching (FRAP) assays as a target agnostic method for the direct visualisation of an on-target effect of bromodomain inhibitors in living cells.  Results: Mutation of a conserved asparagine crucial for binding to acetylated lysines in the bromodomains of BRD3, BRD4 and TRIM24 all resulted in redn. of FRAP recovery times, indicating loss of or significantly reduced binding to acetylated chromatin, as did the addn. of known inhibitors.  Significant differences between wild type and bromodomain mutants for ATAD2, BAZ2A, BRD1, BRD7, GCN5L2, SMARCA2 and ZMYND11 required the addn. of the histone deacetylase inhibitor suberoylanilide hydroxamic acid (SAHA) to amplify the binding contribution of the bromodomain.  Under these conditions, known inhibitors decreased FRAP recovery times back to mutant control levels.  Mutation of the bromodomain did not alter FRAP recovery times for full-length CREBBP, even in the presence of SAHA, indicating that other domains are primarily responsible for anchoring CREBBP to chromatin.  However, FRAP assays with multimerised CREBBP bromodomains resulted in a good assay to assess the efficacy of bromodomain inhibitors to this target.  The bromodomain and extraterminal protein inhibitor PFI-1 was inactive against other bromodomain targets, demonstrating the specificity of the method.  Conclusions: Viable FRAP assays were established for 11 representative bromodomain-contg. proteins that broadly cover the bromodomain phylogenetic tree.  Addn. of SAHA can overcome weak binding to chromatin, and the use of tandem bromodomain constructs can eliminate masking effects of other chromatin binding domains.  Together, these results demonstrate that FRAP assays offer a potentially pan-bromodomain method for generating cell-based assays, allowing the testing of compds. with respect to cell permeability, on-target efficacy and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0eRoqpM6LTbVg90H21EOLACvtfcHk0liaVleAR7cZrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE&md5=966744a9e7b3cba00776b76d65594fc6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-7-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-7-14%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DAssessing%2520cellular%2520efficacy%2520of%2520bromodomain%2520inhibitors%2520using%2520fluorescence%2520recovery%2520after%2520photobleaching%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2014%26volume%3D7%26spage%3D14%26epage%3D25%26doi%3D10.1186%2F1756-8935-7-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brassil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolgos, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webborn, P. J.</span><span> </span><span class="NLM_article-title">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.3109/03602532.2012.691099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.3109%2F03602532.2012.691099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=22697420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=224-252&author=P.+Ballardauthor=P.+Brassilauthor=K.+H.+Buiauthor=H.+Dolgosauthor=C.+Peterssonauthor=A.+Tunekauthor=P.+J.+Webborn&title=The+right+compound+in+the+right+assay+at+the+right+time%3A+an+integrated+discovery+DMPK+strategy&doi=10.3109%2F03602532.2012.691099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span></div><div class="casAuthors">Ballard, Peter; Brassil, Patrick; Bui, Khanh H.; Dolgos, Hugues; Petersson, Carl; Tunek, Anders; Webborn, Peter J. H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">224-252</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The high rate of attrition during drug development and its assocd. high research and development (R&D) cost have put pressure on pharmaceutical companies to ensure that candidate drugs going to clin. testing have the appropriate quality such that the biol. hypothesis could be evaluated.  To help achieve this ambition, drug metab. and pharmacokinetic (DMPK) science and increasing investment have been deployed earlier in the R&D process.  To gain max. return on investment, it is essential that DMPK concepts are both appropriately integrated into the compd. design process and that compd. selection is focused on accurate prediction of likely outcomes in patients.  This article describes key principles that underpin the contribution of DMPK science for small-mol. research based on 15 years of discovery support in a major pharmaceutical company.  It does not aim to describe the breadth and depth of DMPK science, but more the practical application for decision making in real-world situations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolEqf17mivQbVg90H21EOLACvtfcHk0liaVleAR7cZrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF&md5=105fbbcfba6213caaadf41c8b10cf408</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3109%2F03602532.2012.691099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2012.691099%26sid%3Dliteratum%253Aachs%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBrassil%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DK.%2BH.%26aulast%3DDolgos%26aufirst%3DH.%26aulast%3DPetersson%26aufirst%3DC.%26aulast%3DTunek%26aufirst%3DA.%26aulast%3DWebborn%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520right%2520compound%2520in%2520the%2520right%2520assay%2520at%2520the%2520right%2520time%253A%2520an%2520integrated%2520discovery%2520DMPK%2520strategy%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2012%26volume%3D44%26spage%3D224%26epage%3D252%26doi%3D10.3109%2F03602532.2012.691099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Sjögren, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westergren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanisch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindfors, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennernäs, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abrahamsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannergren, C.</span><span> </span><span class="NLM_article-title">In silico predictions of gastrointestinal drug absorption in pharmaceutical product development: application of the mechanistic absorption model GI-Sim</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1016/j.ejps.2013.05.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.ejps.2013.05.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=23727464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSlsr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=679-698&author=E.+Sj%C3%B6grenauthor=J.+Westergrenauthor=I.+Grantauthor=G.+Hanischauthor=L.+Lindforsauthor=H.+Lennern%C3%A4sauthor=B.+Abrahamssonauthor=C.+Tannergren&title=In+silico+predictions+of+gastrointestinal+drug+absorption+in+pharmaceutical+product+development%3A+application+of+the+mechanistic+absorption+model+GI-Sim&doi=10.1016%2Fj.ejps.2013.05.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In silico predictions of gastrointestinal drug absorption in pharmaceutical product development: Application of the mechanistic absorption model GI-Sim</span></div><div class="casAuthors">Sjogren, Erik; Westergren, Jan; Grant, Iain; Hanisch, Gunilla; Lindfors, Lennart; Lennernas, Hans; Abrahamsson, Bertil; Tannergren, Christer</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">679-698</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Oral drug delivery is the predominant administration route for a major part of the pharmaceutical products used worldwide.  Further understanding and improvement of gastrointestinal drug absorption predictions is currently a highly prioritized area of research within the pharmaceutical industry.  The fraction absorbed (fabs) of an oral dose after administration of a solid dosage form is a key parameter in the estn. of the in vivo performance of an orally administrated drug formulation.  This study discloses an evaluation of the predictive performance of the mechanistic physiol. based absorption model GI-Sim.  GI-Sim deploys a compartmental gastrointestinal absorption and transit model as well as algorithms describing permeability, dissoln. rate, salt effects, partitioning into micelles, particle and micelle drifting in the aq. boundary layer, particle growth and amorphous or cryst. pptn.  Twelve APIs with reported or expected absorption limitations in humans, due to permeability, dissoln. and/or soly., were investigated.  Predictions of the intestinal absorption for different doses and formulations were performed based on physicochem. and biopharmaceutical properties, such as soly. in buffer and simulated intestinal fluid, mol. wt., pKa, diffusivity and mol. d., measured or estd. human effective permeability and particle size distribution.  The performance of GI-Sim was evaluated by comparing predicted plasma concn.-time profiles along with oral pharmacokinetic parameters originating from clin. studies in healthy individuals.  The capability of GI-Sim to correctly predict impact of dose and particle size as well as the in vivo performance of nanoformulations was also investigated.  The overall predictive performance of GI-Sim was good as >95% of the predicted pharmacokinetic parameters (Cmax and AUC) were within a 2-fold deviation from the clin. observations and the predicted plasma AUC was within one std. deviation of the obsd. mean plasma AUC in 74% of the simulations.  GI-Sim was also able to correctly capture the trends in dose- and particle size dependent absorption for the study drugs with soly. and dissoln. limited absorption, resp.  In addn., GI-Sim was also shown to be able to predict the increase in absorption and plasma exposure achieved with nanoformulations.  Based on the results, the performance of GI-Sim was shown to be suitable for early risk assessment as well as to guide decision making in pharmaceutical formulation development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriPOOmNPmHhrVg90H21EOLACvtfcHk0ljrzjE9UMPGOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSlsr%252FJ&md5=b6eb6e66b7642463fd1191cfe3bdbc51</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2013.05.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2013.05.019%26sid%3Dliteratum%253Aachs%26aulast%3DSj%25C3%25B6gren%26aufirst%3DE.%26aulast%3DWestergren%26aufirst%3DJ.%26aulast%3DGrant%26aufirst%3DI.%26aulast%3DHanisch%26aufirst%3DG.%26aulast%3DLindfors%26aufirst%3DL.%26aulast%3DLennern%25C3%25A4s%26aufirst%3DH.%26aulast%3DAbrahamsson%26aufirst%3DB.%26aulast%3DTannergren%26aufirst%3DC.%26atitle%3DIn%2520silico%2520predictions%2520of%2520gastrointestinal%2520drug%2520absorption%2520in%2520pharmaceutical%2520product%2520development%253A%2520application%2520of%2520the%2520mechanistic%2520absorption%2520model%2520GI-Sim%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2013%26volume%3D49%26spage%3D679%26epage%3D698%26doi%3D10.1016%2Fj.ejps.2013.05.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sohlenius-Sternbeck, A.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afzelius, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neelissen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoogstraate, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bengtsson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gissberg, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersson, C.</span><span> </span><span class="NLM_article-title">Evaluation of the human prediction of clearance from hepatocyte and microsome intrinsic clearance for 52 drug compounds</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">637</span><span class="NLM_x">–</span> <span class="NLM_lpage">649</span><span class="refDoi"> DOI: 10.3109/00498254.2010.500407</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.3109%2F00498254.2010.500407" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=20624033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVajsLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2010&pages=637-649&author=A.-K.+Sohlenius-Sternbeckauthor=L.+Afzeliusauthor=P.+Prusisauthor=J.+Neelissenauthor=J.+Hoogstraateauthor=J.+Johanssonauthor=E.+Flobyauthor=A.+Bengtssonauthor=O.+Gissbergauthor=J.+Sternbeckauthor=C.+Petersson&title=Evaluation+of+the+human+prediction+of+clearance+from+hepatocyte+and+microsome+intrinsic+clearance+for+52+drug+compounds&doi=10.3109%2F00498254.2010.500407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the human prediction of clearance from hepatocyte and microsome intrinsic clearance for 52 drug compounds</span></div><div class="casAuthors">Sohlenius-Sternbeck, A.-K.; Afzelius, L.; Prusis, P.; Neelissen, J.; Hoogstraate, J.; Johansson, J.; Floby, E.; Bengtsson, A.; Gissberg, O.; Sternbeck, J.; Petersson, C.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">637-649</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">We compare 3 different approaches to scale clearance (CL) from human hepatocyte and microsome CLint (intrinsic CL) for 52 drug compds.  By using the well-stirred model with protein binding included only 11 and 30% of the compds. were predicted within 2-fold and the av. abs. fold errors (AAFE) for the predictions were 5.9 and 4.1 for hepatocytes and microsomes, resp.  When predictions were performed without protein binding, 59% of the compds. were predicted within 2-fold using either hepatocytes or microsomes and the AAFE was 2.2 and 2.3, resp.  For hepatocytes and microsomes there were significant correlations between predicted CLint in vivo (obtained from in vitro CLint) and measured CLint in vivo (obtained using the well-stirred model).  When CL was calcd. from the regression, 76 and 70% of the compds. were predicted within 2-fold and the AAFE was 1.6 and 1.8 for hepatocytes and microsomes, resp.  We demonstrate that microsomes and hepatocytes are in many cases comparable when scaling of CL is performed from regression.  By using the hepatocyte regression, CL for 82% of the compds. in an independent test set (n=11) were predicted within 2-fold (AAFE 1.4).  We suggest that a regression line that adjusts for systematic under-predictions should be the 1st-hand choice for scaling of CL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEy5gZ7yCu7bVg90H21EOLACvtfcHk0ljrzjE9UMPGOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVajsLnI&md5=ef98aa46451331023795cd3f56843eba</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3109%2F00498254.2010.500407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2010.500407%26sid%3Dliteratum%253Aachs%26aulast%3DSohlenius-Sternbeck%26aufirst%3DA.-K.%26aulast%3DAfzelius%26aufirst%3DL.%26aulast%3DPrusis%26aufirst%3DP.%26aulast%3DNeelissen%26aufirst%3DJ.%26aulast%3DHoogstraate%26aufirst%3DJ.%26aulast%3DJohansson%26aufirst%3DJ.%26aulast%3DFloby%26aufirst%3DE.%26aulast%3DBengtsson%26aufirst%3DA.%26aulast%3DGissberg%26aufirst%3DO.%26aulast%3DSternbeck%26aufirst%3DJ.%26aulast%3DPetersson%26aufirst%3DC.%26atitle%3DEvaluation%2520of%2520the%2520human%2520prediction%2520of%2520clearance%2520from%2520hepatocyte%2520and%2520microsome%2520intrinsic%2520clearance%2520for%252052%2520drug%2520compounds%26jtitle%3DXenobiotica%26date%3D2010%26volume%3D40%26spage%3D637%26epage%3D649%26doi%3D10.3109%2F00498254.2010.500407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Sohlenius-Sternbeck, A.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Projean, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bylund, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afzelius, L.</span><span> </span><span class="NLM_article-title">Practical use of the regression offset approach for the prediction of in vivo intrinsic clearance from hepatocytes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span><span class="refDoi"> DOI: 10.3109/00498254.2012.669080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.3109%2F00498254.2012.669080" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2012&pages=841-853&author=A.-K.+Sohlenius-Sternbeckauthor=C.+Jonesauthor=D.+Fergusonauthor=B.+J.+Middletonauthor=D.+Projeanauthor=E.+Flobyauthor=J.+Bylundauthor=L.+Afzelius&title=Practical+use+of+the+regression+offset+approach+for+the+prediction+of+in+vivo+intrinsic+clearance+from+hepatocytes&doi=10.3109%2F00498254.2012.669080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3109%2F00498254.2012.669080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2012.669080%26sid%3Dliteratum%253Aachs%26aulast%3DSohlenius-Sternbeck%26aufirst%3DA.-K.%26aulast%3DJones%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DMiddleton%26aufirst%3DB.%2BJ.%26aulast%3DProjean%26aufirst%3DD.%26aulast%3DFloby%26aufirst%3DE.%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DAfzelius%26aufirst%3DL.%26atitle%3DPractical%2520use%2520of%2520the%2520regression%2520offset%2520approach%2520for%2520the%2520prediction%2520of%2520in%2520vivo%2520intrinsic%2520clearance%2520from%2520hepatocytes%26jtitle%3DXenobiotica%26date%3D2012%26volume%3D42%26spage%3D841%26epage%3D853%26doi%3D10.3109%2F00498254.2012.669080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Delmore, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastritis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilpatrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paranal, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinzel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergsagel, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghobrial, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">146</span><span class="NLM_x">, </span> <span class="NLM_fpage">904</span><span class="NLM_x">–</span> <span class="NLM_lpage">917</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lgPHNjTVRZqZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Rhyasen, G. W.; Hattersley, M.; Yao, Y.; Dulak, A.; Wang, W.; Petteruti, P.; Dale, I.; Boiko, S.; Cheung, T.; Zhang, J.; Wen, S.; Castriotta, L.; Lawson, D.; Collins, M.; Ahdesmaki, M. J.; Walker, G.; O’Connor, G.; Yeh, T.; Rabow, A. A.; Dry, J.; Reimer, C.; Lyne, P.; Mills, G. B.; Fawell, S. E.; Waring, M. J.; Zinda, M.; Clark, E.; Chen, H.</span><span> </span><span class="NLM_article-title">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span>. Mol. Cancer Ther.<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">, </span>submitted for publication</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=27573426" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=G.+W.+Rhyasen&author=M.+Hattersley&author=Y.+Yao&author=A.+Dulak&author=W.+Wang&author=P.+Petteruti&author=I.+Dale&author=S.+Boiko&author=T.+Cheung&author=J.+Zhang&author=S.+Wen&author=L.+Castriotta&author=D.+Lawson&author=M.+Collins&author=M.+J.+Ahdesmaki&author=G.+Walker&author=G.+O%E2%80%99Connor&author=T.+Yeh&author=A.+A.+Rabow&author=J.+Dry&author=C.+Reimer&author=P.+Lyne&author=G.+B.+Mills&author=S.+E.+Fawell&author=M.+J.+Waring&author=M.+Zinda&author=E.+Clark&author=H.+Chen&title=AZD5153%3A+a+novel+bivalent+BET+bromodomain+inhibitor+highly+active+against+hematologic+malignancies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26atitle%3DAZD5153%253A%2520a%2520novel%2520bivalent%2520BET%2520bromodomain%2520inhibitor%2520highly%2520active%2520against%2520hematologic%2520malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Arnold, D. L.; Foreman, K. W.; Jin, M.; Wanner, J.; Werner, D. S.</span><span> </span><span class="NLM_article-title">Preparation of bivalent bromodomain ligands for treating a disease associated with a protein having tandem bromodomains</span>. PCT Int. Appl. WO 2013033268, March 7th,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=D.+L.+Arnold&author=K.+W.+Foreman&author=M.+Jin&author=J.+Wanner&author=D.+S.+Werner&title=Preparation+of+bivalent+bromodomain+ligands+for+treating+a+disease+associated+with+a+protein+having+tandem+bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DD.%2BL.%26atitle%3DPreparation%2520of%2520bivalent%2520bromodomain%2520ligands%2520for%2520treating%2520a%2520disease%2520associated%2520with%2520a%2520protein%2520having%2520tandem%2520bromodomains%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Rodrik-Outmezguine, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okaniwa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novotny, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McWhirter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banaji, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Won, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Stanchina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barratt, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosulich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Overcoming mTOR resistance mutations with a new-generation mTOR inhibitor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">534</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1038/nature17963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1038%2Fnature17963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=27279227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFOktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=272-276&author=V.+S.+Rodrik-Outmezguineauthor=M.+Okaniwaauthor=Z.+Yaoauthor=C.+J.+Novotnyauthor=C.+McWhirterauthor=A.+Banajiauthor=H.+Wonauthor=W.+Wongauthor=M.+Bergerauthor=E.+de+Stanchinaauthor=D.+G.+Barrattauthor=S.+Cosulichauthor=T.+Klinowskaauthor=N.+Rosenauthor=K.+M.+Shokat&title=Overcoming+mTOR+resistance+mutations+with+a+new-generation+mTOR+inhibitor&doi=10.1038%2Fnature17963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming mTOR resistance mutations with a new-generation mTOR inhibitor</span></div><div class="casAuthors">Rodrik-Outmezguine, Vanessa S.; Okaniwa, Masanori; Yao, Zhan; Novotny, Chris J.; McWhirter, Claire; Banaji, Arpitha; Won, Helen; Wong, Wai; Berger, Mike; de Stanchina, Elisa; Barratt, Derek G.; Cosulich, Sabina; Klinowska, Teresa; Rosen, Neal; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7606</span>),
    <span class="NLM_cas:pages">272-276</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Precision medicines exert selective pressure on tumor cells that leads to the preferential growth of resistant subpopulations, necessitating the development of next-generation therapies to treat the evolving cancer.  The PIK3CA-AKT-mTOR pathway is one of the most commonly activated pathways in human cancers, which has led to the development of small-mol. inhibitors that target various nodes in the pathway.  Among these agents, first-generation mTOR inhibitors (rapalogs) have caused responses in 'N-of-1' cases, and second-generation mTOR kinase inhibitors (TORKi) are currently in clin. trials.  Here the authors sought to delineate the likely resistance mechanisms to existing mTOR inhibitors in human cell lines, as a guide for next-generation therapies.  The mechanism of resistance to the TORKi was unusual in that intrinsic kinase activity of mTOR was increased, rather than a direct active-site mutation interfering with drug binding.  Indeed, identical drug-resistant mutations have been also identified in drug-naive patients, suggesting that tumors with activating MTOR mutations will be intrinsically resistant to second-generation mTOR inhibitors.  The authors report the development of a new class of mTOR inhibitors that overcomes resistance to existing first- and second-generation inhibitors.  The third-generation mTOR inhibitor exploits the unique juxtaposition of two drug-binding pockets to create a bivalent interaction that allows inhibition of these resistant mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyillD_NPGSLVg90H21EOLACvtfcHk0lgPHNjTVRZqZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFOktLo%253D&md5=8e9c2bb6b53e5e97b9f48d6ec2ed91d3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature17963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17963%26sid%3Dliteratum%253Aachs%26aulast%3DRodrik-Outmezguine%26aufirst%3DV.%2BS.%26aulast%3DOkaniwa%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DNovotny%26aufirst%3DC.%2BJ.%26aulast%3DMcWhirter%26aufirst%3DC.%26aulast%3DBanaji%26aufirst%3DA.%26aulast%3DWon%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DW.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3Dde%2BStanchina%26aufirst%3DE.%26aulast%3DBarratt%26aufirst%3DD.%2BG.%26aulast%3DCosulich%26aufirst%3DS.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DOvercoming%2520mTOR%2520resistance%2520mutations%2520with%2520a%2520new-generation%2520mTOR%2520inhibitor%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D272%26epage%3D276%26doi%3D10.1038%2Fnature17963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Callis, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradbury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Challinor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G.</span><span> </span><span class="NLM_article-title">A screening assay cascade to identify and characterize novel selective estrogen receptor downregulators (SERDs)</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">748</span><span class="NLM_x">–</span> <span class="NLM_lpage">759</span><span class="refDoi"> DOI: 10.1177/1087057115580298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1177%2F1087057115580298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=25851036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=748-759&author=R.+Callisauthor=A.+Rabowauthor=M.+Tongeauthor=R.+Bradburyauthor=M.+Challinorauthor=K.+Robertsauthor=K.+Jonesauthor=G.+Walker&title=A+screening+assay+cascade+to+identify+and+characterize+novel+selective+estrogen+receptor+downregulators+%28SERDs%29&doi=10.1177%2F1087057115580298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A screening assay cascade to identify and characterize novel selective estrogen receptor downregulators (SERDs)</span></div><div class="casAuthors">Callis, Rowena; Rabow, Alfred; Tonge, Michael; Bradbury, Robert; Challinor, Mairi; Roberts, Karen; Jones, Karen; Walker, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">748-759</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Here, we describe an approach to identify novel selective estrogen receptor downregulator (SERD) compds. with improved properties such as oral bioavailability and the potential of increased efficacy compared to currently marketed drug treatments.  Previously, methodologies such as Western blotting and transient cell reporter assays have been used to identify and characterize SERD compds., but such approaches can be limited due to low throughput and sensitivity, resp.  We have used an endogenous cell-imaging strategy that has both the throughput and sensitivity to support a large-scale hit-to-lead program to identify novel compds.  A screening cascade with a suite of assays has been developed to characterize compds. that modulate estrogen receptor α (ERα)-mediated signaling or downregulate ERα levels in cells.  Initially, from a focused high-throughput screening, novel ERα binders were identified that could be modified chem. into ERα downregulators.  Following this, cellular assays helped det. the mechanism of action of compds. to distinguish between on-target and off-target compds. and differentiate SERDs, selective estrogen receptor modulator (SERM) compds., and agonist ERα ligands.  Data are shown to exemplify the characterization of ERα-mediated signaling inhibitors using a selection of literature compds. and illustrate how this cascade has been used to drive the chem. design of novel SERD compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqNNWMPPJDNrVg90H21EOLACvtfcHk0ljE9frH6s0JSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE&md5=13c27dbf08f315d469f81b2bcefb4499</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1087057115580298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115580298%26sid%3Dliteratum%253Aachs%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DRabow%26aufirst%3DA.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DBradbury%26aufirst%3DR.%26aulast%3DChallinor%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWalker%26aufirst%3DG.%26atitle%3DA%2520screening%2520assay%2520cascade%2520to%2520identify%2520and%2520characterize%2520novel%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D748%26epage%3D759%26doi%3D10.1177%2F1087057115580298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kilkenny, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browne, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuthill, I. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, D. G.</span><span> </span><span class="NLM_article-title">Improving bioscience research reporting: the ARRIVE guidelines for reporting animal research</span> <span class="citation_source-journal">PLoS Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000412</span><span class="refDoi"> DOI: 10.1371/journal.pbio.1000412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1371%2Fjournal.pbio.1000412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=20613859" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=e1000412&author=C.+Kilkennyauthor=W.+J.+Browneauthor=I.+C.+Cuthillauthor=M.+Emersonauthor=D.+G.+Altman&title=Improving+bioscience+research+reporting%3A+the+ARRIVE+guidelines+for+reporting+animal+research&doi=10.1371%2Fjournal.pbio.1000412"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.1000412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.1000412%26sid%3Dliteratum%253Aachs%26aulast%3DKilkenny%26aufirst%3DC.%26aulast%3DBrowne%26aufirst%3DW.%2BJ.%26aulast%3DCuthill%26aufirst%3DI.%2BC.%26aulast%3DEmerson%26aufirst%3DM.%26aulast%3DAltman%26aufirst%3DD.%2BG.%26atitle%3DImproving%2520bioscience%2520research%2520reporting%253A%2520the%2520ARRIVE%2520guidelines%2520for%2520reporting%2520animal%2520research%26jtitle%3DPLoS%2520Biol.%26date%3D2010%26volume%3D8%26spage%3De1000412%26doi%3D10.1371%2Fjournal.pbio.1000412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 36 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Farrukh Vohidov, Jannik N. Andersen, Kyriakos D. Economides, Michail V. Shipitsin, Olga Burenkova, James C. Ackley, Bhavatarini Vangamudi, Hung V.-T. Nguyen, Nolan M. Gallagher, Peyton Shieh, Matthew R. Golder, Jenny Liu, William K. Dahlberg, Deborah J. C. Ehrlich, Julie Kim, Samantha L. Kristufek, Sung Jin Huh, Allison M. Neenan, Joelle Baddour, Sattanathan Paramasivan, Elisa de Stanchina, Gaurab KC, David J. Turnquist, Jennifer K. Saucier-Sawyer, Paul W. Kopesky, Samantha W. Brady, Michael J. Jessel, Lawrence A. Reiter, Donald E. Chickering, Jeremiah A. Johnson, <span class="NLM_string-name hlFld-ContribAuthor">Peter Blume-Jensen</span>. </span><span class="cited-content_cbyCitation_article-title">Design of BET Inhibitor Bottlebrush Prodrugs with Superior Efficacy and Devoid of Systemic Toxicities. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (12)
                                     , 4714-4724. <a href="https://doi.org/10.1021/jacs.1c00312" title="DOI URL">https://doi.org/10.1021/jacs.1c00312</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c00312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c00312%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DDesign%252Bof%252BBET%252BInhibitor%252BBottlebrush%252BProdrugs%252Bwith%252BSuperior%252BEfficacy%252Band%252BDevoid%252Bof%252BSystemic%252BToxicities%26aulast%3DVohidov%26aufirst%3DFarrukh%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D10012021%26date%3D19032021%26volume%3D143%26issue%3D12%26spage%3D4714%26epage%3D4724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yifei Yang, Lincheng Fang, Pan Chen, Huibin Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Jinpei Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 3,5-Dimethylisoxazole Derivatives as BRD4 Inhibitors for the Treatment of Colorectal Cancer. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (11)
                                     , 2174-2181. <a href="https://doi.org/10.1021/acsmedchemlett.0c00294" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00294</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00294%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252B3%25252C5-Dimethylisoxazole%252BDerivatives%252Bas%252BBRD4%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BColorectal%252BCancer%26aulast%3DYang%26aufirst%3DYifei%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29052020%26date%3D31082020%26date%3D02092020%26date%3D31082020%26volume%3D11%26issue%3D11%26spage%3D2174%26epage%3D2181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yangfeng Li, Jiong Zhao, Lauren M. Gutgesell, Zhengnan Shen, Kiira Ratia, Katherine Dye, Oleksii Dubrovskyi, Huiping Zhao, Fei Huang, Debra A. Tonetti, Gregory R. J. Thatcher, <span class="NLM_string-name hlFld-ContribAuthor">Rui Xiong</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Pyrrolopyridone Bromodomain and Extra-Terminal Motif (BET) Inhibitors Effective in Endocrine-Resistant ER+ Breast Cancer with Acquired Resistance to Fulvestrant and Palbociclib. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (13)
                                     , 7186-7210. <a href="https://doi.org/10.1021/acs.jmedchem.0c00456" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00456</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00456%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BPyrrolopyridone%252BBromodomain%252Band%252BExtra-Terminal%252BMotif%252B%252528BET%252529%252BInhibitors%252BEffective%252Bin%252BEndocrine-Resistant%252BER%25252B%252BBreast%252BCancer%252Bwith%252BAcquired%252BResistance%252Bto%252BFulvestrant%252Band%252BPalbociclib%26aulast%3DLi%26aufirst%3DYangfeng%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18032020%26date%3D15062020%26date%3D26052020%26volume%3D63%26issue%3D13%26spage%3D7186%26epage%3D7210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dean G. Brown, <span class="NLM_string-name hlFld-ContribAuthor">Jonas Boström</span>. </span><span class="cited-content_cbyCitation_article-title">Where Do Recent Small Molecule Clinical Development Candidates Come From?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (21)
                                     , 9442-9468. <a href="https://doi.org/10.1021/acs.jmedchem.8b00675" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00675</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00675%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DWhere%252BDo%252BRecent%252BSmall%252BMolecule%252BClinical%252BDevelopment%252BCandidates%252BCome%252BFrom%25253F%26aulast%3DBrown%26aufirst%3DDean%2BG.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D27042018%26date%3D09072018%26date%3D19062018%26volume%3D61%26issue%3D21%26spage%3D9442%26epage%3D9468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yujun  Zhao</span>, <span class="hlFld-ContribAuthor ">Longchuan  Bai</span>, <span class="hlFld-ContribAuthor ">Liu  Liu</span>, <span class="hlFld-ContribAuthor ">Donna  McEachern</span>, <span class="hlFld-ContribAuthor ">Jeanne A.  Stuckey</span>, <span class="hlFld-ContribAuthor ">Jennifer L.  Meagher</span>, <span class="hlFld-ContribAuthor ">Chao-Yie  Yang</span>, <span class="hlFld-ContribAuthor ">Xu  Ran</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>, <span class="hlFld-ContribAuthor ">Yang  Hu</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Li</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Ting  Zhao</span>, <span class="hlFld-ContribAuthor ">Siwei  Li</span>, <span class="hlFld-ContribAuthor ">Duxin  Sun</span>, and <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Discovery of 4-(6-Methoxy-2-methyl-4-(quinolin-4-yl)-9H-pyrimido[4,5-b]indol-7-yl)-3,5-dimethylisoxazole (CD161) as a Potent and Orally Bioavailable BET Bromodomain Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (9)
                                     , 3887-3901. <a href="https://doi.org/10.1021/acs.jmedchem.7b00193" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00193</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00193%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDiscovery%252Bof%252B4-%2525286-Methoxy-2-methyl-4-%252528quinolin-4-yl%252529-9H-pyrimido%25255B4%25252C5-b%25255Dindol-7-yl%252529-3%25252C5-dimethylisoxazole%252B%252528CD161%252529%252Bas%252Ba%252BPotent%252Band%252BOrally%252BBioavailable%252BBET%252BBromodomain%252BInhibitor%26aulast%3DZhao%26aufirst%3DYujun%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D04022017%26date%3D02052017%26date%3D11052017%26volume%3D60%26issue%3D9%26spage%3D3887%26epage%3D3901" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fatemeh  Amjadi-Moheb</span>, <span class="hlFld-ContribAuthor ">Alireza  Paniri</span>, <span class="hlFld-ContribAuthor ">Haleh  Akhavan-Niaki</span>. </span><span class="cited-content_cbyCitation_article-title">Insights into the Links between MYC and 3D Chromatin Structure and Epigenetics Regulation: Implications for Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2021,</strong> <em>81 </em>
                                    (8)
                                     , 1925-1936. <a href="https://doi.org/10.1158/0008-5472.CAN-20-3613" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-20-3613</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-20-3613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-20-3613%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DInsights%252Binto%252Bthe%252BLinks%252Bbetween%252BMYC%252Band%252B3D%252BChromatin%252BStructure%252Band%252BEpigenetics%252BRegulation%25253A%252BImplications%252Bfor%252BCancer%252BTherapy%26aulast%3DAmjadi-Moheb%26aufirst%3DFatemeh%26date%3D2021%26date%3D2021%26volume%3D81%26issue%3D8%26spage%3D1925%26epage%3D1936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam P.  Cribbs</span>, <span class="hlFld-ContribAuthor ">Panagis  Filippakopoulos</span>, <span class="hlFld-ContribAuthor ">Martin  Philpott</span>, <span class="hlFld-ContribAuthor ">Graham  Wells</span>, <span class="hlFld-ContribAuthor ">Henry  Penn</span>, <span class="hlFld-ContribAuthor ">Henrik  Oerum</span>, <span class="hlFld-ContribAuthor ">Viia  Valge-Archer</span>, <span class="hlFld-ContribAuthor ">Marc  Feldmann</span>, <span class="hlFld-ContribAuthor ">Udo  Oppermann</span>. </span><span class="cited-content_cbyCitation_article-title">Dissecting the Role of BET Bromodomain Proteins BRD2 and BRD4 in Human NK Cell Function. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Immunology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fimmu.2021.626255" title="DOI URL">https://doi.org/10.3389/fimmu.2021.626255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fimmu.2021.626255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffimmu.2021.626255%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Immunology%26atitle%3DDissecting%252Bthe%252BRole%252Bof%252BBET%252BBromodomain%252BProteins%252BBRD2%252Band%252BBRD4%252Bin%252BHuman%252BNK%252BCell%252BFunction%26aulast%3DCribbs%26aufirst%3DAdam%2BP.%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Yuan</span>, <span class="hlFld-ContribAuthor ">Natasha M.  Flores</span>, <span class="hlFld-ContribAuthor ">Simone  Hausmann</span>, <span class="hlFld-ContribAuthor ">Shane M.  Lofgren</span>, <span class="hlFld-ContribAuthor ">Vladlena  Kharchenko</span>, <span class="hlFld-ContribAuthor ">Maria  Angulo-Ibanez</span>, <span class="hlFld-ContribAuthor ">Deepanwita  Sengupta</span>, <span class="hlFld-ContribAuthor ">Xiaoyin  Lu</span>, <span class="hlFld-ContribAuthor ">Iwona  Czaban</span>, <span class="hlFld-ContribAuthor ">Dulat  Azhibek</span>, <span class="hlFld-ContribAuthor ">Silvestre  Vicent</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Fischle</span>, <span class="hlFld-ContribAuthor ">Mariusz  Jaremko</span>, <span class="hlFld-ContribAuthor ">Bingliang  Fang</span>, <span class="hlFld-ContribAuthor ">Ignacio I.  Wistuba</span>, <span class="hlFld-ContribAuthor ">Katrin F.  Chua</span>, <span class="hlFld-ContribAuthor ">Jack A.  Roth</span>, <span class="hlFld-ContribAuthor ">John D.  Minna</span>, <span class="hlFld-ContribAuthor ">Ning-Yi  Shao</span>, <span class="hlFld-ContribAuthor ">Łukasz  Jaremko</span>, <span class="hlFld-ContribAuthor ">Pawel K.  Mazur</span>, <span class="hlFld-ContribAuthor ">Or  Gozani</span>. </span><span class="cited-content_cbyCitation_article-title">Elevated NSD3 histone methylation activity drives squamous cell lung cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2021,</strong> <em>590 </em>
                                    (7846)
                                     , 504-508. <a href="https://doi.org/10.1038/s41586-020-03170-y" title="DOI URL">https://doi.org/10.1038/s41586-020-03170-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41586-020-03170-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41586-020-03170-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DElevated%252BNSD3%252Bhistone%252Bmethylation%252Bactivity%252Bdrives%252Bsquamous%252Bcell%252Blung%252Bcancer%26aulast%3DYuan%26aufirst%3DGang%26date%3D2021%26date%3D2021%26volume%3D590%26issue%3D7846%26spage%3D504%26epage%3D508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaohua  Liu</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yalei  Li</span>, <span class="hlFld-ContribAuthor ">Juan  Wang</span>, <span class="hlFld-ContribAuthor ">Huaqian  Ding</span>, <span class="hlFld-ContribAuthor ">Wenjing  Huang</span>, <span class="hlFld-ContribAuthor ">Chunyong  Ding</span>, <span class="hlFld-ContribAuthor ">Hongchun  Liu</span>, <span class="hlFld-ContribAuthor ">Wenfu  Tan</span>, <span class="hlFld-ContribAuthor ">Ao  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of hedgehog pathway inhibitors by epigenetically targeting GLI through BET bromodomain for the treatment of medulloblastoma. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 488-504. <a href="https://doi.org/10.1016/j.apsb.2020.07.007" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDevelopment%252Bof%252Bhedgehog%252Bpathway%252Binhibitors%252Bby%252Bepigenetically%252Btargeting%252BGLI%252Bthrough%252BBET%252Bbromodomain%252Bfor%252Bthe%252Btreatment%252Bof%252Bmedulloblastoma%26aulast%3DLiu%26aufirst%3DXiaohua%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D488%26epage%3D504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohd.  Muddassir</span>, <span class="hlFld-ContribAuthor ">Kunjal  Soni</span>, <span class="hlFld-ContribAuthor ">Chetan B.  Sangani</span>, <span class="hlFld-ContribAuthor ">Abdullah  Alarifi</span>, <span class="hlFld-ContribAuthor ">Mohd.  Afzal</span>, <span class="hlFld-ContribAuthor ">Naaser A. Y.  Abduh</span>, <span class="hlFld-ContribAuthor ">Yongtao  Duan</span>, <span class="hlFld-ContribAuthor ">Poonam  Bhadja</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain and BET family proteins as epigenetic targets in cancer therapy: their degradation, present drugs, and possible PROTACs. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 612-636. <a href="https://doi.org/10.1039/D0RA07971E" title="DOI URL">https://doi.org/10.1039/D0RA07971E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA07971E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA07971E%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DBromodomain%252Band%252BBET%252Bfamily%252Bproteins%252Bas%252Bepigenetic%252Btargets%252Bin%252Bcancer%252Btherapy%25253A%252Btheir%252Bdegradation%25252C%252Bpresent%252Bdrugs%25252C%252Band%252Bpossible%252BPROTACs%26aulast%3DMuddassir%26aufirst%3DMohd.%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D612%26epage%3D636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Massimo  Petretich</span>, <span class="hlFld-ContribAuthor ">Emmanuel H.  Demont</span>, <span class="hlFld-ContribAuthor ">Paola  Grandi</span>. </span><span class="cited-content_cbyCitation_article-title">Domain-selective targeting of BET proteins in cancer and immunological diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 184-193. <a href="https://doi.org/10.1016/j.cbpa.2020.02.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDomain-selective%252Btargeting%252Bof%252BBET%252Bproteins%252Bin%252Bcancer%252Band%252Bimmunological%252Bdiseases%26aulast%3DPetretich%26aufirst%3DMassimo%26date%3D2020%26volume%3D57%26spage%3D184%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sravani  Sana</span>, <span class="hlFld-ContribAuthor ">Velma Ganga  Reddy</span>, <span class="hlFld-ContribAuthor ">Sonal  Bhandari</span>, <span class="hlFld-ContribAuthor ">T. Srinivasa  Reddy</span>, <span class="hlFld-ContribAuthor ">Ramya  Tokala</span>, <span class="hlFld-ContribAuthor ">Akash P.  Sakla</span>, <span class="hlFld-ContribAuthor ">Suresh K.  Bhargava</span>, <span class="hlFld-ContribAuthor ">Nagula  Shankaraiah</span>. </span><span class="cited-content_cbyCitation_article-title">Exploration of carbamide derived pyrimidine-thioindole conjugates as potential VEGFR-2 inhibitors with anti-angiogenesis effect. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112457. <a href="https://doi.org/10.1016/j.ejmech.2020.112457" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112457</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112457%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExploration%252Bof%252Bcarbamide%252Bderived%252Bpyrimidine-thioindole%252Bconjugates%252Bas%252Bpotential%252BVEGFR-2%252Binhibitors%252Bwith%252Banti-angiogenesis%252Beffect%26aulast%3DSana%26aufirst%3DSravani%26date%3D2020%26volume%3D200%26spage%3D112457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yoshitomo  Shiroma</span>, <span class="hlFld-ContribAuthor ">Ryou‐u  Takahashi</span>, <span class="hlFld-ContribAuthor ">Yuki  Yamamoto</span>, <span class="hlFld-ContribAuthor ">Hidetoshi  Tahara</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting DNA binding proteins for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Cancer Science</span><span> <strong>2020,</strong> <em>111 </em>
                                    (4)
                                     , 1058-1064. <a href="https://doi.org/10.1111/cas.14355" title="DOI URL">https://doi.org/10.1111/cas.14355</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cas.14355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcas.14355%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Science%26atitle%3DTargeting%252BDNA%252Bbinding%252Bproteins%252Bfor%252Bcancer%252Btherapy%26aulast%3DShiroma%26aufirst%3DYoshitomo%26date%3D2020%26date%3D2020%26volume%3D111%26issue%3D4%26spage%3D1058%26epage%3D1064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xi  Li</span>, <span class="hlFld-ContribAuthor ">Yu  Fu</span>, <span class="hlFld-ContribAuthor ">Bin  Yang</span>, <span class="hlFld-ContribAuthor ">Ensong  Guo</span>, <span class="hlFld-ContribAuthor ">Yifan  Wu</span>, <span class="hlFld-ContribAuthor ">Jia  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoxiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Rourou  Xiao</span>, <span class="hlFld-ContribAuthor ">Kezhen  Li</span>, <span class="hlFld-ContribAuthor ">Beibei  Wang</span>, <span class="hlFld-ContribAuthor ">Junbo  Hu</span>, <span class="hlFld-ContribAuthor ">Chaoyang  Sun</span>, <span class="hlFld-ContribAuthor ">Gang  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">BRD4 Inhibition by AZD5153 Promotes Antitumor Immunity via Depolarizing M2 Macrophages. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Immunology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fimmu.2020.00089" title="DOI URL">https://doi.org/10.3389/fimmu.2020.00089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fimmu.2020.00089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffimmu.2020.00089%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Immunology%26atitle%3DBRD4%252BInhibition%252Bby%252BAZD5153%252BPromotes%252BAntitumor%252BImmunity%252Bvia%252BDepolarizing%252BM2%252BMacrophages%26aulast%3DLi%26aufirst%3DXi%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemische Epigenetik: der Einfluss chemischer und chemo‐biologischer Techniken auf die Zielstruktur‐Validierung von Bromodomänen. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2019,</strong> <em>131 </em>
                                    (50)
                                     , 18096-18120. <a href="https://doi.org/10.1002/ange.201812164" title="DOI URL">https://doi.org/10.1002/ange.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DChemische%252BEpigenetik%25253A%252Bder%252BEinfluss%252Bchemischer%252Bund%252Bchemo%2525E2%252580%252590biologischer%252BTechniken%252Bauf%252Bdie%252BZielstruktur%2525E2%252580%252590Validierung%252Bvon%252BBromodom%2525C3%2525A4nen%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26date%3D2019%26volume%3D131%26issue%3D50%26spage%3D18096%26epage%3D18120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Mustafa  Moroglu</span>, <span class="hlFld-ContribAuthor ">David M. H.  Ascough</span>, <span class="hlFld-ContribAuthor ">Anna E. R.  Chamberlain</span>, <span class="hlFld-ContribAuthor ">Jos J. A. G.  Kamps</span>, <span class="hlFld-ContribAuthor ">Angelina R.  Sekirnik</span>, <span class="hlFld-ContribAuthor ">Stuart J.  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical Epigenetics: The Impact of Chemical and Chemical Biology Techniques on Bromodomain Target Validation. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2019,</strong> <em>58 </em>
                                    (50)
                                     , 17930-17952. <a href="https://doi.org/10.1002/anie.201812164" title="DOI URL">https://doi.org/10.1002/anie.201812164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201812164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201812164%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DChemical%252BEpigenetics%25253A%252BThe%252BImpact%252Bof%252BChemical%252Band%252BChemical%252BBiology%252BTechniques%252Bon%252BBromodomain%252BTarget%252BValidation%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2019%26volume%3D58%26issue%3D50%26spage%3D17930%26epage%3D17952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deheng  Chen</span>, <span class="hlFld-ContribAuthor ">Tian  Lu</span>, <span class="hlFld-ContribAuthor ">Ziqin  Yan</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Feilong  Zhou</span>, <span class="hlFld-ContribAuthor ">Xilin  Lyu</span>, <span class="hlFld-ContribAuthor ">Biling  Xu</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Yujun  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>182 </em>, 111633. <a href="https://doi.org/10.1016/j.ejmech.2019.111633" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111633</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%25252C%252Bstructural%252Binsight%25252C%252Band%252Bbioactivities%252Bof%252BBY27%252Bas%252Ba%252Bselective%252Binhibitor%252Bof%252Bthe%252Bsecond%252Bbromodomains%252Bof%252BBET%252Bproteins%26aulast%3DChen%26aufirst%3DDeheng%26date%3D2019%26volume%3D182%26spage%3D111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brittany L.  Allen-Petersen</span>, <span class="hlFld-ContribAuthor ">Rosalie C.  Sears</span>. </span><span class="cited-content_cbyCitation_article-title">Mission Possible: Advances in MYC Therapeutic Targeting in Cancer. </span><span class="cited-content_cbyCitation_journal-name">BioDrugs</span><span> <strong>2019,</strong> <em>33 </em>
                                    (5)
                                     , 539-553. <a href="https://doi.org/10.1007/s40259-019-00370-5" title="DOI URL">https://doi.org/10.1007/s40259-019-00370-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s40259-019-00370-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs40259-019-00370-5%26sid%3Dliteratum%253Aachs%26jtitle%3DBioDrugs%26atitle%3DMission%252BPossible%25253A%252BAdvances%252Bin%252BMYC%252BTherapeutic%252BTargeting%252Bin%252BCancer%26aulast%3DAllen-Petersen%26aufirst%3DBrittany%2BL.%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D5%26spage%3D539%26epage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Maniaci</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Bifunctional chemical probes inducing protein–protein interactions. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2019,</strong> <em>52 </em>, 145-156. <a href="https://doi.org/10.1016/j.cbpa.2019.07.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2019.07.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2019.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2019.07.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DBifunctional%252Bchemical%252Bprobes%252Binducing%252Bprotein%2525E2%252580%252593protein%252Binteractions%26aulast%3DManiaci%26aufirst%3DChiara%26date%3D2019%26volume%3D52%26spage%3D145%26epage%3D156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Franca-Maria  Klingler</span>, <span class="hlFld-ContribAuthor ">Marcus  Gastreich</span>, <span class="hlFld-ContribAuthor ">Oleksandr  Grygorenko</span>, <span class="hlFld-ContribAuthor ">Olena  Savych</span>, <span class="hlFld-ContribAuthor ">Petro  Borysko</span>, <span class="hlFld-ContribAuthor ">Anastasia  Griniukova</span>, <span class="hlFld-ContribAuthor ">Kateryna  Gubina</span>, <span class="hlFld-ContribAuthor ">Christian  Lemmen</span>, <span class="hlFld-ContribAuthor ">Yurii  Moroz</span>. </span><span class="cited-content_cbyCitation_article-title">SAR by Space: Enriching Hit Sets from the Chemical Space. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (17)
                                     , 3096. <a href="https://doi.org/10.3390/molecules24173096" title="DOI URL">https://doi.org/10.3390/molecules24173096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24173096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24173096%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSAR%252Bby%252BSpace%25253A%252BEnriching%252BHit%252BSets%252Bfrom%252Bthe%252BChemical%252BSpace%26aulast%3DKlingler%26aufirst%3DFranca-Maria%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D17%26spage%3D3096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Minsun  Jung</span>, <span class="hlFld-ContribAuthor ">Soyeon  Kim</span>, <span class="hlFld-ContribAuthor ">June‐Koo  Lee</span>, <span class="hlFld-ContribAuthor ">Sun Och  Yoon</span>, <span class="hlFld-ContribAuthor ">Heae Surng  Park</span>, <span class="hlFld-ContribAuthor ">Soon Won  Hong</span>, <span class="hlFld-ContribAuthor ">Weon‐Seo  Park</span>, <span class="hlFld-ContribAuthor ">Ji Eun  Kim</span>, <span class="hlFld-ContribAuthor ">Joon  Kim</span>, <span class="hlFld-ContribAuthor ">Bhumsuk  Keam</span>, <span class="hlFld-ContribAuthor ">Hyun Jik  Kim</span>, <span class="hlFld-ContribAuthor ">Hyoung Jin  Kang</span>, <span class="hlFld-ContribAuthor ">Dong‐Wan  Kim</span>, <span class="hlFld-ContribAuthor ">Kyeong Cheon  Jung</span>, <span class="hlFld-ContribAuthor ">Young Tae  Kim</span>, <span class="hlFld-ContribAuthor ">Dae Seog  Heo</span>, <span class="hlFld-ContribAuthor ">Tae Min  Kim</span>, <span class="hlFld-ContribAuthor ">Yoon Kyung  Jeon</span>. </span><span class="cited-content_cbyCitation_article-title">Clinicopathological and Preclinical Findings of NUT Carcinoma: A Multicenter Study. </span><span class="cited-content_cbyCitation_journal-name">The Oncologist</span><span> <strong>2019,</strong> <em>24 </em>
                                    (8)
                                     <a href="https://doi.org/10.1634/theoncologist.2018-0477" title="DOI URL">https://doi.org/10.1634/theoncologist.2018-0477</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1634/theoncologist.2018-0477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1634%2Ftheoncologist.2018-0477%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Oncologist%26atitle%3DClinicopathological%252Band%252BPreclinical%252BFindings%252Bof%252BNUT%252BCarcinoma%25253A%252BA%252BMulticenter%252BStudy%26aulast%3DJung%26aufirst%3DMinsun%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shyh-Ming  Yang</span>, <span class="hlFld-ContribAuthor ">Makoto  Yoshioka</span>, <span class="hlFld-ContribAuthor ">Jeffrey W.  Strovel</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Urban</span>, <span class="hlFld-ContribAuthor ">Xin  Hu</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Hall</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">David J.  Maloney</span>. </span><span class="cited-content_cbyCitation_article-title">Lead optimization and efficacy evaluation of quinazoline-based BET family inhibitors for potential treatment of cancer and inflammatory diseases. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (10)
                                     , 1220-1226. <a href="https://doi.org/10.1016/j.bmcl.2019.03.014" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.03.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.03.014%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLead%252Boptimization%252Band%252Befficacy%252Bevaluation%252Bof%252Bquinazoline-based%252BBET%252Bfamily%252Binhibitors%252Bfor%252Bpotential%252Btreatment%252Bof%252Bcancer%252Band%252Binflammatory%252Bdiseases%26aulast%3DYang%26aufirst%3DShyh-Ming%26date%3D2019%26volume%3D29%26issue%3D10%26spage%3D1220%26epage%3D1226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoko  Takimoto-Shimomura</span>, <span class="hlFld-ContribAuthor ">Taku  Tsukamoto</span>, <span class="hlFld-ContribAuthor ">Saori  Maegawa</span>, <span class="hlFld-ContribAuthor ">Yuto  Fujibayashi</span>, <span class="hlFld-ContribAuthor ">Yayoi  Matsumura-Kimoto</span>, <span class="hlFld-ContribAuthor ">Yoshimi  Mizuno</span>, <span class="hlFld-ContribAuthor ">Yoshiaki  Chinen</span>, <span class="hlFld-ContribAuthor ">Yuji  Shimura</span>, <span class="hlFld-ContribAuthor ">Shinsuke  Mizutani</span>, <span class="hlFld-ContribAuthor ">Shigeo  Horiike</span>, <span class="hlFld-ContribAuthor ">Masafumi  Taniwaki</span>, <span class="hlFld-ContribAuthor ">Tsutomu  Kobayashi</span>, <span class="hlFld-ContribAuthor ">Junya  Kuroda</span>. </span><span class="cited-content_cbyCitation_article-title">Dual targeting of bromodomain-containing 4 by AZD5153 and BCL2 by AZD4320 against B-cell lymphomas concomitantly overexpressing c-MYC and BCL2. </span><span class="cited-content_cbyCitation_journal-name">Investigational New Drugs</span><span> <strong>2019,</strong> <em>37 </em>
                                    (2)
                                     , 210-222. <a href="https://doi.org/10.1007/s10637-018-0623-8" title="DOI URL">https://doi.org/10.1007/s10637-018-0623-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10637-018-0623-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10637-018-0623-8%26sid%3Dliteratum%253Aachs%26jtitle%3DInvestigational%2520New%2520Drugs%26atitle%3DDual%252Btargeting%252Bof%252Bbromodomain-containing%252B4%252Bby%252BAZD5153%252Band%252BBCL2%252Bby%252BAZD4320%252Bagainst%252BB-cell%252Blymphomas%252Bconcomitantly%252Boverexpressing%252Bc-MYC%252Band%252BBCL2%26aulast%3DTakimoto-Shimomura%26aufirst%3DTomoko%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D2%26spage%3D210%26epage%3D222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clinton G. L.  Veale</span>. </span><span class="cited-content_cbyCitation_article-title">Unpacking the Pathogen Box—An Open Source Tool for Fighting Neglected Tropical Disease. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (4)
                                     , 386-453. <a href="https://doi.org/10.1002/cmdc.201800755" title="DOI URL">https://doi.org/10.1002/cmdc.201800755</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800755%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DUnpacking%252Bthe%252BPathogen%252BBox%2525E2%252580%252594An%252BOpen%252BSource%252BTool%252Bfor%252BFighting%252BNeglected%252BTropical%252BDisease%26aulast%3DVeale%26aufirst%3DClinton%2BG.%2BL.%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D4%26spage%3D386%26epage%3D453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Xing</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiangrui  Jiang</span>, <span class="hlFld-ContribAuthor ">Tianwen  Hu</span>, <span class="hlFld-ContribAuthor ">Xinjun  Wang</span>, <span class="hlFld-ContribAuthor ">Gang  Qiao</span>, <span class="hlFld-ContribAuthor ">Junjian  Wang</span>, <span class="hlFld-ContribAuthor ">Fengling  Yang</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Jingshan  Shen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Rational design of 5-((1H-imidazol-1-yl)methyl)quinolin-8-ol derivatives as novel bromodomain-containing protein 4 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>163 </em>, 281-294. <a href="https://doi.org/10.1016/j.ejmech.2018.11.018" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.11.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.11.018%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252Bdesign%252Bof%252B5-%252528%2525281H-imidazol-1-yl%252529methyl%252529quinolin-8-ol%252Bderivatives%252Bas%252Bnovel%252Bbromodomain-containing%252Bprotein%252B4%252Binhibitors%26aulast%3DXing%26aufirst%3DJing%26date%3D2019%26volume%3D163%26spage%3D281%26epage%3D294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nathan J.  Dupper</span>, <span class="hlFld-ContribAuthor ">Yingsheng  Zhou</span>, <span class="hlFld-ContribAuthor ">Jérôme  Govin</span>, <span class="hlFld-ContribAuthor ">Charles E.  McKenna</span>. </span><span class="cited-content_cbyCitation_article-title">Bromodomain Inhibition and Its Application to Human Disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 475-492. <a href="https://doi.org/10.1016/B978-0-12-813939-4.00011-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-813939-4.00011-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-813939-4.00011-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-813939-4.00011-5%26sid%3Dliteratum%253Aachs%26atitle%3DBromodomain%252BInhibition%252Band%252BIts%252BApplication%252Bto%252BHuman%252BDisease%26aulast%3DDupper%26aufirst%3DNathan%2BJ.%26date%3D2019%26spage%3D475%26epage%3D492%26pub%3DElsevier%26atitle%3DPharmacoepigenetics%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingchao  Duan</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Guan</span>, <span class="hlFld-ContribAuthor ">Wenping  Qin</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhai</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Brd4 for cancer therapy: inhibitors and degraders. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2018,</strong> <em>9 </em>
                                    (11)
                                     , 1779-1802. <a href="https://doi.org/10.1039/C8MD00198G" title="DOI URL">https://doi.org/10.1039/C8MD00198G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8MD00198G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8MD00198G%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DTargeting%252BBrd4%252Bfor%252Bcancer%252Btherapy%25253A%252Binhibitors%252Band%252Bdegraders%26aulast%3DDuan%26aufirst%3DYingchao%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D11%26spage%3D1779%26epage%3D1802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Schiedel</span>, <span class="hlFld-ContribAuthor ">Stuart J  Conway</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecules as tools to study the chemical epigenetics of lysine acetylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>45 </em>, 166-178. <a href="https://doi.org/10.1016/j.cbpa.2018.06.015" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.06.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.06.015%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DSmall%252Bmolecules%252Bas%252Btools%252Bto%252Bstudy%252Bthe%252Bchemical%252Bepigenetics%252Bof%252Blysine%252Bacetylation%26aulast%3DSchiedel%26aufirst%3DMatthias%26date%3D2018%26volume%3D45%26spage%3D166%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunyan  Ren</span>, <span class="hlFld-ContribAuthor ">Guangtao  Zhang</span>, <span class="hlFld-ContribAuthor ">Fangbin  Han</span>, <span class="hlFld-ContribAuthor ">Shibo  Fu</span>, <span class="hlFld-ContribAuthor ">Yingdi  Cao</span>, <span class="hlFld-ContribAuthor ">Fan  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Zhang</span>, <span class="hlFld-ContribAuthor ">Jamel  Meslamani</span>, <span class="hlFld-ContribAuthor ">Yaoyao  Xu</span>, <span class="hlFld-ContribAuthor ">Donglei  Ji</span>, <span class="hlFld-ContribAuthor ">Lingling  Cao</span>, <span class="hlFld-ContribAuthor ">Qian  Zhou</span>, <span class="hlFld-ContribAuthor ">Ka-lung  Cheung</span>, <span class="hlFld-ContribAuthor ">Rajal  Sharma</span>, <span class="hlFld-ContribAuthor ">Nicolas  Babault</span>, <span class="hlFld-ContribAuthor ">Zhengzi  Yi</span>, <span class="hlFld-ContribAuthor ">Weijia  Zhang</span>, <span class="hlFld-ContribAuthor ">Martin J.  Walsh</span>, <span class="hlFld-ContribAuthor ">Lei  Zeng</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Spatially constrained tandem bromodomain inhibition bolsters sustained repression of BRD4 transcriptional activity for TNBC cell growth. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2018,</strong> <em>115 </em>
                                    (31)
                                     , 7949-7954. <a href="https://doi.org/10.1073/pnas.1720000115" title="DOI URL">https://doi.org/10.1073/pnas.1720000115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1720000115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1720000115%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DSpatially%252Bconstrained%252Btandem%252Bbromodomain%252Binhibition%252Bbolsters%252Bsustained%252Brepression%252Bof%252BBRD4%252Btranscriptional%252Bactivity%252Bfor%252BTNBC%252Bcell%252Bgrowth%26aulast%3DRen%26aufirst%3DChunyan%26date%3D2018%26date%3D2018%26volume%3D115%26issue%3D31%26spage%3D7949%26epage%3D7954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingwen  Zhang</span>, <span class="hlFld-ContribAuthor ">Austin M.  Dulak</span>, <span class="hlFld-ContribAuthor ">Maureen M.  Hattersley</span>, <span class="hlFld-ContribAuthor ">Brandon S.  Willis</span>, <span class="hlFld-ContribAuthor ">Jenni  Nikkilä</span>, <span class="hlFld-ContribAuthor ">Anderson  Wang</span>, <span class="hlFld-ContribAuthor ">Alan  Lau</span>, <span class="hlFld-ContribAuthor ">Corinne  Reimer</span>, <span class="hlFld-ContribAuthor ">Michael  Zinda</span>, <span class="hlFld-ContribAuthor ">Stephen E.  Fawell</span>, <span class="hlFld-ContribAuthor ">Gordon B.  Mills</span>, <span class="hlFld-ContribAuthor ">Huawei  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">BRD4 facilitates replication stress-induced DNA damage response. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2018,</strong> <em>37 </em>
                                    (28)
                                     , 3763-3777. <a href="https://doi.org/10.1038/s41388-018-0194-3" title="DOI URL">https://doi.org/10.1038/s41388-018-0194-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-018-0194-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-018-0194-3%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DBRD4%252Bfacilitates%252Breplication%252Bstress-induced%252BDNA%252Bdamage%252Bresponse%26aulast%3DZhang%26aufirst%3DJingwen%26date%3D2018%26date%3D2018%26volume%3D37%26issue%3D28%26spage%3D3763%26epage%3D3777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kun  Xu</span>, <span class="hlFld-ContribAuthor ">Dexuan  Chen</span>, <span class="hlFld-ContribAuthor ">Dong  Qian</span>, <span class="hlFld-ContribAuthor ">Shihu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yi  Zhang</span>, <span class="hlFld-ContribAuthor ">Song  Guo</span>, <span class="hlFld-ContribAuthor ">Zhaoqun  Ma</span>, <span class="hlFld-ContribAuthor ">Shui  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">AZD5153, a novel BRD4 inhibitor, suppresses human thyroid carcinoma cell growth in vitro and in vivo. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2018,</strong> <em>499 </em>
                                    (3)
                                     , 531-537. <a href="https://doi.org/10.1016/j.bbrc.2018.03.184" title="DOI URL">https://doi.org/10.1016/j.bbrc.2018.03.184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2018.03.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2018.03.184%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DAZD5153%25252C%252Ba%252Bnovel%252BBRD4%252Binhibitor%25252C%252Bsuppresses%252Bhuman%252Bthyroid%252Bcarcinoma%252Bcell%252Bgrowth%252Bin%2525C2%2525A0vitro%252Band%252Bin%2525C2%2525A0vivo%26aulast%3DXu%26aufirst%3DKun%26date%3D2018%26volume%3D499%26issue%3D3%26spage%3D531%26epage%3D537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Morgan</span>, <span class="hlFld-ContribAuthor ">Dean G.  Brown</span>, <span class="hlFld-ContribAuthor ">Simon  Lennard</span>, <span class="hlFld-ContribAuthor ">Mark J.  Anderton</span>, <span class="hlFld-ContribAuthor ">J. Carl  Barrett</span>, <span class="hlFld-ContribAuthor ">Ulf  Eriksson</span>, <span class="hlFld-ContribAuthor ">Mark  Fidock</span>, <span class="hlFld-ContribAuthor ">Bengt  Hamrén</span>, <span class="hlFld-ContribAuthor ">Anthony  Johnson</span>, <span class="hlFld-ContribAuthor ">Ruth E.  March</span>, <span class="hlFld-ContribAuthor ">James  Matcham</span>, <span class="hlFld-ContribAuthor ">Jerome  Mettetal</span>, <span class="hlFld-ContribAuthor ">David J.  Nicholls</span>, <span class="hlFld-ContribAuthor ">Stefan  Platz</span>, <span class="hlFld-ContribAuthor ">Steve  Rees</span>, <span class="hlFld-ContribAuthor ">Michael A.  Snowden</span>, <span class="hlFld-ContribAuthor ">Menelas N.  Pangalos</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of a five-dimensional framework on R&D productivity at AstraZeneca. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2018,</strong> <em>17 </em>
                                    (3)
                                     , 167-181. <a href="https://doi.org/10.1038/nrd.2017.244" title="DOI URL">https://doi.org/10.1038/nrd.2017.244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2017.244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2017.244%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DImpact%252Bof%252Ba%252Bfive-dimensional%252Bframework%252Bon%252BR%252526D%252Bproductivity%252Bat%252BAstraZeneca%26aulast%3DMorgan%26aufirst%3DPaul%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D3%26spage%3D167%26epage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">TA  Collins</span>, <span class="hlFld-ContribAuthor ">MM  Hattersley</span>, <span class="hlFld-ContribAuthor ">JWT  Yates</span>, <span class="hlFld-ContribAuthor ">E  Clark</span>, <span class="hlFld-ContribAuthor ">M  Mondal</span>, <span class="hlFld-ContribAuthor ">JT  Mettetal</span>. </span><span class="cited-content_cbyCitation_article-title">Translational Modeling of Drug-Induced Myelosuppression and Effect of Pretreatment Myelosuppression for AZD5153, a Selective BRD4 Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">CPT: Pharmacometrics & Systems Pharmacology</span><span> <strong>2017,</strong> <em>6 </em>
                                    (6)
                                     , 357-364. <a href="https://doi.org/10.1002/psp4.12194" title="DOI URL">https://doi.org/10.1002/psp4.12194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/psp4.12194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpsp4.12194%26sid%3Dliteratum%253Aachs%26jtitle%3DCPT%253A%2520Pharmacometrics%2520%2526%2520Systems%2520Pharmacology%26atitle%3DTranslational%252BModeling%252Bof%252BDrug-Induced%252BMyelosuppression%252Band%252BEffect%252Bof%252BPretreatment%252BMyelosuppression%252Bfor%252BAZD5153%25252C%252Ba%252BSelective%252BBRD4%252BInhibitor%26aulast%3DCollins%26aufirst%3DTA%26date%3D2017%26date%3D2017%26volume%3D6%26issue%3D6%26spage%3D357%26epage%3D364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tammie C.  Yeh</span>, <span class="hlFld-ContribAuthor ">Greg  O'Connor</span>, <span class="hlFld-ContribAuthor ">Philip  Petteruti</span>, <span class="hlFld-ContribAuthor ">Austin  Dulak</span>, <span class="hlFld-ContribAuthor ">Maureen  Hattersley</span>, <span class="hlFld-ContribAuthor ">J. Carl  Barrett</span>, <span class="hlFld-ContribAuthor ">Huawei  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of CCR2 and CD180 as Robust Pharmacodynamic Tumor and Blood Biomarkers for Clinical Use with BRD4/BET Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Clinical Cancer Research</span><span> <strong>2017,</strong> <em>23 </em>
                                    (4)
                                     , 1025-1035. <a href="https://doi.org/10.1158/1078-0432.CCR-16-1658" title="DOI URL">https://doi.org/10.1158/1078-0432.CCR-16-1658</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1078-0432.CCR-16-1658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1078-0432.CCR-16-1658%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Cancer%2520Research%26atitle%3DIdentification%252Bof%252BCCR2%252Band%252BCD180%252Bas%252BRobust%252BPharmacodynamic%252BTumor%252Band%252BBlood%252BBiomarkers%252Bfor%252BClinical%252BUse%252Bwith%252BBRD4%25252FBET%252BInhibitors%26aulast%3DYeh%26aufirst%3DTammie%2BC.%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D4%26spage%3D1025%26epage%3D1035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael J  Waring</span>, <span class="hlFld-ContribAuthor ">Huawei  Chen</span>, <span class="hlFld-ContribAuthor ">Alfred A  Rabow</span>, <span class="hlFld-ContribAuthor ">Graeme  Walker</span>, <span class="hlFld-ContribAuthor ">Romel  Bobby</span>, <span class="hlFld-ContribAuthor ">Scott  Boiko</span>, <span class="hlFld-ContribAuthor ">Rob H  Bradbury</span>, <span class="hlFld-ContribAuthor ">Rowena  Callis</span>, <span class="hlFld-ContribAuthor ">Edwin  Clark</span>, <span class="hlFld-ContribAuthor ">Ian  Dale</span>, <span class="hlFld-ContribAuthor ">Danette L  Daniels</span>, <span class="hlFld-ContribAuthor ">Austin  Dulak</span>, <span class="hlFld-ContribAuthor ">Liz  Flavell</span>, <span class="hlFld-ContribAuthor ">Geoff  Holdgate</span>, <span class="hlFld-ContribAuthor ">Thomas A  Jowitt</span>, <span class="hlFld-ContribAuthor ">Alexey  Kikhney</span>, <span class="hlFld-ContribAuthor ">Mark  McAlister</span>, <span class="hlFld-ContribAuthor ">Jacqui  Méndez</span>, <span class="hlFld-ContribAuthor ">Derek  Ogg</span>, <span class="hlFld-ContribAuthor ">Joe  Patel</span>, <span class="hlFld-ContribAuthor ">Philip  Petteruti</span>, <span class="hlFld-ContribAuthor ">Graeme R  Robb</span>, <span class="hlFld-ContribAuthor ">Matthew B  Robers</span>, <span class="hlFld-ContribAuthor ">Sakina  Saif</span>, <span class="hlFld-ContribAuthor ">Natalie  Stratton</span>, <span class="hlFld-ContribAuthor ">Dmitri I  Svergun</span>, <span class="hlFld-ContribAuthor ">Wenxian  Wang</span>, <span class="hlFld-ContribAuthor ">David  Whittaker</span>, <span class="hlFld-ContribAuthor ">David M  Wilson</span>, <span class="hlFld-ContribAuthor ">Yi  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Potent and selective bivalent inhibitors of BET bromodomains. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2016,</strong> <em>12 </em>
                                    (12)
                                     , 1097-1104. <a href="https://doi.org/10.1038/nchembio.2210" title="DOI URL">https://doi.org/10.1038/nchembio.2210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nchembio.2210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnchembio.2210%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DPotent%252Band%252Bselective%252Bbivalent%252Binhibitors%252Bof%252BBET%252Bbromodomains%26aulast%3DWaring%26aufirst%3DMichael%2BJ%26date%3D2016%26date%3D2016%26volume%3D12%26issue%3D12%26spage%3D1097%26epage%3D1104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Garrett W.  Rhyasen</span>, <span class="hlFld-ContribAuthor ">Maureen M.  Hattersley</span>, <span class="hlFld-ContribAuthor ">Yi  Yao</span>, <span class="hlFld-ContribAuthor ">Austin  Dulak</span>, <span class="hlFld-ContribAuthor ">Wenxian  Wang</span>, <span class="hlFld-ContribAuthor ">Philip  Petteruti</span>, <span class="hlFld-ContribAuthor ">Ian L.  Dale</span>, <span class="hlFld-ContribAuthor ">Scott  Boiko</span>, <span class="hlFld-ContribAuthor ">Tony  Cheung</span>, <span class="hlFld-ContribAuthor ">Jingwen  Zhang</span>, <span class="hlFld-ContribAuthor ">Shenghua  Wen</span>, <span class="hlFld-ContribAuthor ">Lillian  Castriotta</span>, <span class="hlFld-ContribAuthor ">Deborah  Lawson</span>, <span class="hlFld-ContribAuthor ">Michael  Collins</span>, <span class="hlFld-ContribAuthor ">Larry  Bao</span>, <span class="hlFld-ContribAuthor ">Miika J.  Ahdesmaki</span>, <span class="hlFld-ContribAuthor ">Graeme  Walker</span>, <span class="hlFld-ContribAuthor ">Greg  O'Connor</span>, <span class="hlFld-ContribAuthor ">Tammie C.  Yeh</span>, <span class="hlFld-ContribAuthor ">Alfred A.  Rabow</span>, <span class="hlFld-ContribAuthor ">Jonathan R.  Dry</span>, <span class="hlFld-ContribAuthor ">Corinne  Reimer</span>, <span class="hlFld-ContribAuthor ">Paul  Lyne</span>, <span class="hlFld-ContribAuthor ">Gordon B.  Mills</span>, <span class="hlFld-ContribAuthor ">Stephen E.  Fawell</span>, <span class="hlFld-ContribAuthor ">Michael J.  Waring</span>, <span class="hlFld-ContribAuthor ">Michael  Zinda</span>, <span class="hlFld-ContribAuthor ">Edwin  Clark</span>, <span class="hlFld-ContribAuthor ">Huawei  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">AZD5153: A Novel Bivalent BET Bromodomain Inhibitor Highly Active against Hematologic Malignancies. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2016,</strong> <em>15 </em>
                                    (11)
                                     , 2563-2574. <a href="https://doi.org/10.1158/1535-7163.MCT-16-0141" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-16-0141</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-16-0141%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DAZD5153%25253A%252BA%252BNovel%252BBivalent%252BBET%252BBromodomain%252BInhibitor%252BHighly%252BActive%252Bagainst%252BHematologic%252BMalignancies%26aulast%3DRhyasen%26aufirst%3DGarrett%2BW.%26date%3D2016%26date%3D2016%26volume%3D15%26issue%3D11%26spage%3D2563%26epage%3D2574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) DIPEA, MeCN, rt, 66%; (ii) ethylene carbonate, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 72%; (iii) mesyl chloride, Et<sub>3</sub>N, DCM, 0 °C, 100%; (iv) <i>N</i>-acetylpiperazine, DIPEA, DMA, 110 °C, 73%. Compounds <b>6</b> and <b>7</b> were prepared in an analogous manner.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Et<sub>3</sub>N, BOC<sub>2</sub>O, DCM, 0 °C, 88%; (ii) 1-bromo-2-chloroethane, K<sub>2</sub>CO<sub>3</sub>, acetone, 90 °C, 97%; (iii) DIPEA, KI, DMA, 110 °C, 90%; (iv) HCl, Et<sub>2</sub>O, quant; (v) DIPEA, DMA, 80 °C, 68%. Compounds <b>9</b> to <b>20</b> were prepared in a similar fashion.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) Et<sub>3</sub>N, NaBH(OAc)<sub>3</sub>, DCM, 56%; (ii) TFA, DCM, quant; (iii) DIPEA, DMF, 80 °C, 34%; (iv) Ruphos palladium(II) phenethylamine chloride, dicyclohexyl(2′,6′-diisopropoxy-[1,1′-biphenyl]-2-yl)phosphine, Cs<sub>2</sub>CO<sub>3</sub>, toluene, 90 °C, 28%. Compounds <b>9</b>–<b>20</b> were prepared in a similar fashion.</p></p></figure><figure data-id="fig2a" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of <b>13</b> in complex with BRD4(1) showing (left) the formation of a dimeric structure with two bromodomains. (middle) Engagement of the triazolopyridazine of <b>13</b> with the first bromodomain (blue ribbon) acetyl lysine binding site. (right) Engagement of the <i>N</i>-methylpiperazinone of <b>13</b> with the second bromodomain (red ribbon) acetyl lysine site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig2a"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of BRD4 WT p<i>K</i><sub>i</sub> vs ER downregulation. Bivalent triazolopyridazine are shown with black dots. Line Fit: ER DR pIC<sub>50</sub> = 0.046 + 0.93[BRD4 WT FP p<i>K</i><sub>i</sub>], <i>r</i><sup>2</sup> = 0.84, <i>n</i> = 20.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity of <b>13</b> in the Bromoscan panel. Values above the bars indicate p<i>K</i><sub>i</sub>. Inhibition measured at 10 μM compound concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Pharmacodynamic data following single oral dose of 10 mg/kg <b>13</b> to MV-4-11 tumor-bearing mice. Measured levels of c-Myc RNA in tumors 2, 4, or 8 h postdose plotted against free plasma exposure of <b>13</b>. c-Myc levels for the vehicle group are shown on the plot for comparison at an arbitrary position on the <i>x</i>-axis. (b) Corresponding downregulation of c-Myc protein and loading control measured by Western blot. Each lane represents expression in a tumor from one mouse, <i>n</i> = 3 mice per group. Vehicle-treated mice from early and late time points only.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/medium/jm-2016-00070g_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Antitumor activity of <b>13</b> in the MV-4-11 xenograft efficacy model. Tumors were allowed to grow to an average size of 200 mm<sup>3</sup>; then mice were treated daily with either vehicle (<i>n</i> = 8) or <b>13</b> (<i>n</i> = 8) for 2 weeks via oral gavage. Tumor volumes were monitored twice a week for 14 days and are plotted as the geometric mean volumes ± standard error (SE) for both vehicle and treated groups (vehicle group SE bars are too small to see).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-17/acs.jmedchem.6b00070/20160901/images/large/jm-2016-00070g_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00070&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i80">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03202" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03202" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Druggability and structural classification of bromodomain acetyl-lysine sites</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7346</span><span class="NLM_x">–</span> <span class="NLM_lpage">7359</span><span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+and+structural+classification+of+bromodomain+acetyl-lysine+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span> </span><span class="NLM_article-title">Place your BETs: the therapeutic potential of bromodomains</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">153</span><span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+the+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0ljsMI8xgsmFXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520the%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Nicodème, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beinke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dewell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-w.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandwani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marazzi, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coste, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirilovsky, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarakhovsky, A.</span><span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1123</span><span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicod%C3%A8meauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.-w.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Mirguet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gosmini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toum, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clément, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnathan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brusq, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mordaunt, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimes, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineau, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajakane, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daugan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cutler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smithers, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C.-w.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bamborough, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uings, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witherington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinjha, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicodème, E.</span><span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7501</span><span class="NLM_x">–</span> <span class="NLM_lpage">7515</span><span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0ljjXg350lOuug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morse, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keates, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hickman, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felletar, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christie, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">468</span><span class="NLM_x">, </span> <span class="NLM_fpage">1067</span><span class="NLM_x">–</span> <span class="NLM_lpage">1073</span><span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lhimAzxERnlkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Wu, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, C.-M.</span><span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">13141</span><span class="NLM_x">–</span> <span class="NLM_lpage">13145</span><span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.-Y.+Wuauthor=C.-M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lhimAzxERnlkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Coudé, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braun, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berrou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dupont, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masse, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raffoux, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itzykson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delord, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riveiro, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herait, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baruchel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombret, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardin, C.</span><span> </span><span class="NLM_article-title">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-Myc in acute leukemia cells</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">17698</span><span class="NLM_x">–</span> <span class="NLM_lpage">17712</span><span class="refDoi"> DOI: 10.18632/oncotarget.4131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.18632%2Foncotarget.4131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=25989842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17698-17712&author=M.+M.+Coud%C3%A9author=T.+Braunauthor=J.+Berrouauthor=M.+Dupontauthor=S.+Bertrandauthor=A.+Masseauthor=E.+Raffouxauthor=R.+Itzyksonauthor=M.+Delordauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=A.+Baruchelauthor=H.+Dombretauthor=C.+Gardin&title=BET+inhibitor+OTX015+targets+BRD2+and+BRD4+and+decreases+c-Myc+in+acute+leukemia+cells&doi=10.18632%2Foncotarget.4131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span></div><div class="casAuthors">Coude Marie-Magdelaine; Braun Thorsten; Berrou Jeannig; Dupont Melanie; Bertrand Sibyl; Masse Aline; Raffoux Emmanuel; Itzykson Raphael; Baruchel Andre; Dombret Herve; Gardin Claude; Coude Marie-Magdelaine; Braun Thorsten; Gardin Claude; Raffoux Emmanuel; Itzykson Raphael; Dombret Herve; Delord Marc; Riveiro Maria E; Herait Patrice; Baruchel Andre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17698-712</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain (BRD) and extraterminal (BET) proteins including BRD2, BRD3 and BRD4 have been identified as key targets for leukemia maintenance.  A novel oral inhibitor of BRD2/3/4, the thienotriazolodiazepine compound OTX015, suitable for human use, is available.  Here we report its biological effects in AML and ALL cell lines and leukemic samples.  Exposure to OTX015 lead to cell growth inhibition, cell cycle arrest and apoptosis at submicromolar concentrations in acute leukemia cell lines and patient-derived leukemic cells, as described with the canonical JQ1 BET inhibitor.  Treatment with JQ1 and OTX15 induces similar gene expression profiles in sensitive cell lines, including a c-MYC decrease and an HEXIM1 increase.  OTX015 exposure also induced a strong decrease of BRD2, BRD4 and c-MYC and increase of HEXIM1 proteins, while BRD3 expression was unchanged. c-MYC, BRD2, BRD3, BRD4 and HEXIM1 mRNA levels did not correlate however with viability following exposure to OTX015.  Sequential combinations of OTX015 with other epigenetic modifying drugs, panobinostat and azacitidine have a synergic effect on growth of the KASUMI cell line.  Our results indicate that OTX015 and JQ1 have similar biological effects in leukemic cells, supporting OTX015 evaluation in a Phase Ib trial in relapsed/refractory leukemia patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSlHRu3Td8hIa_dlEzl_fBafW6udTcc2ebV1z76PgwvqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D&md5=0db953a0a24b12ec2918a02d1a2df00b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4131%26sid%3Dliteratum%253Aachs%26aulast%3DCoud%25C3%25A9%26aufirst%3DM.%2BM.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DBerrou%26aufirst%3DJ.%26aulast%3DDupont%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DMasse%26aufirst%3DA.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DItzykson%26aufirst%3DR.%26aulast%3DDelord%26aufirst%3DM.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DBaruchel%26aufirst%3DA.%26aulast%3DDombret%26aufirst%3DH.%26aulast%3DGardin%26aufirst%3DC.%26atitle%3DBET%2520inhibitor%2520OTX015%2520targets%2520BRD2%2520and%2520BRD4%2520and%2520decreases%2520c-Myc%2520in%2520acute%2520leukemia%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17698%26epage%3D17712%26doi%3D10.18632%2Foncotarget.4131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Herait, P.</span><span> </span><span class="NLM_article-title">BET-bromodomain (BRD) inhibitor OTX015: final results of the dose-finding part of a phase I study in hematologic malignancies</span>. Presented at the 13th International Congress on Targeted Anticancer Therapies, March<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">, </span>Paris.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=P.+Herait&title=BET-bromodomain+%28BRD%29+inhibitor+OTX015%3A+final+results+of+the+dose-finding+part+of+a+phase+I+study+in+hematologic+malignancies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DHerait%26aufirst%3DP.%26atitle%3DBET-bromodomain%2520%2528BRD%2529%2520inhibitor%2520OTX015%253A%2520final%2520results%2520of%2520the%2520dose-finding%2520part%2520of%2520a%2520phase%2520I%2520study%2520in%2520hematologic%2520malignancies%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Bradbury, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acton, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broadbent, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatter, g.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayter, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howe, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jude, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamont, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loddick, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parveen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma-Singh, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomason, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trueman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, J. M.</span><span> </span><span class="NLM_article-title">Discovery of AZD3514, a small-molecule androgen receptor downregulator for treatment of advanced prostate cancer</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1945</span><span class="NLM_x">–</span> <span class="NLM_lpage">1948</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.02.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.bmcl.2013.02.056" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1945-1948&author=R.+H.+Bradburyauthor=D.+G.+Actonauthor=N.+L.+Broadbentauthor=A.+N.+Brooksauthor=G.+R.+Carrauthor=g.+Hatterauthor=B.+R.+Hayterauthor=K.+J.+Hillauthor=N.+J.+Howeauthor=R.+D.+Jonesauthor=D.+Judeauthor=S.+G.+Lamontauthor=S.+A.+Loddickauthor=H.+L.+McFarlandauthor=Z.+Parveenauthor=A.+A.+Rabowauthor=G.+Sharma-Singhauthor=N.+C.+Strattonauthor=A.+G.+Thomasonauthor=D.+Truemanauthor=G.+E.+Walkerauthor=S.+L.+Wellsauthor=J.+Wilsonauthor=J.+M.+Wood&title=Discovery+of+AZD3514%2C+a+small-molecule+androgen+receptor+downregulator+for+treatment+of+advanced+prostate+cancer&doi=10.1016%2Fj.bmcl.2013.02.056"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.02.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.02.056%26sid%3Dliteratum%253Aachs%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DActon%26aufirst%3DD.%2BG.%26aulast%3DBroadbent%26aufirst%3DN.%2BL.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DCarr%26aufirst%3DG.%2BR.%26aulast%3DHatter%26aufirst%3Dg.%26aulast%3DHayter%26aufirst%3DB.%2BR.%26aulast%3DHill%26aufirst%3DK.%2BJ.%26aulast%3DHowe%26aufirst%3DN.%2BJ.%26aulast%3DJones%26aufirst%3DR.%2BD.%26aulast%3DJude%26aufirst%3DD.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DLoddick%26aufirst%3DS.%2BA.%26aulast%3DMcFarland%26aufirst%3DH.%2BL.%26aulast%3DParveen%26aufirst%3DZ.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DSharma-Singh%26aufirst%3DG.%26aulast%3DStratton%26aufirst%3DN.%2BC.%26aulast%3DThomason%26aufirst%3DA.%2BG.%26aulast%3DTrueman%26aufirst%3DD.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWells%26aufirst%3DS.%2BL.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26atitle%3DDiscovery%2520of%2520AZD3514%252C%2520a%2520small-molecule%2520androgen%2520receptor%2520downregulator%2520for%2520treatment%2520of%2520advanced%2520prostate%2520cancer%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1945%26epage%3D1948%26doi%3D10.1016%2Fj.bmcl.2013.02.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Loddick, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomason, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunkley, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brave, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broadbent, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trueman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouchet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaheen, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cumberbatch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradbury, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaughan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Womack, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Critchlow, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wedge, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, A. N.</span><span> </span><span class="NLM_article-title">AZD3514: a small molecule that modulates androgen receptor signaling and function <i>in vitro</i> and <i>in vivo</i></span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">1715</span><span class="NLM_x">–</span> <span class="NLM_lpage">1727</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-12-1174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1158%2F1535-7163.MCT-12-1174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=23861347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2ltbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=1715-1727&author=S.+A.+Loddickauthor=S.+J.+Rossauthor=A.+G.+Thomasonauthor=D.+M.+Robinsonauthor=G.+E.+Walkerauthor=T.+P.+Dunkleyauthor=S.+R.+Braveauthor=N.+Broadbentauthor=N.+C.+Strattonauthor=D.+Truemanauthor=E.+Mouchetauthor=F.+S.+Shaheenauthor=V.+N.+Jacobsauthor=M.+Cumberbatchauthor=J.+Wilsonauthor=R.+D.+Jonesauthor=R.+H.+Bradburyauthor=A.+A.+Rabowauthor=L.+Gaughanauthor=C.+Womackauthor=S.+T.+Barryauthor=C.+N.+Robsonauthor=S.+E.+Critchlowauthor=S.+R.+Wedgeauthor=A.+N.+Brooks&title=AZD3514%3A+a+small+molecule+that+modulates+androgen+receptor+signaling+and+function+in+vitro+and+in+vivo&doi=10.1158%2F1535-7163.MCT-12-1174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">AZD3514: A Small Molecule That Modulates Androgen Receptor Signaling and Function In Vitro and In Vivo</span></div><div class="casAuthors">Loddick, Sarah A.; Ross, Sarah J.; Thomason, Andrew G.; Robinson, David M.; Walker, Graeme E.; Dunkley, Tom P. J.; Brave, Sandra R.; Broadbent, Nicola; Stratton, Natalie C.; Trueman, Dawn; Mouchet, Elizabeth; Shaheen, Fadhel S.; Jacobs, Vivien N.; Cumberbatch, Marie; Wilson, Joanne; Jones, Rhys D. O.; Bradbury, Robert H.; Rabow, Alfred; Gaughan, Luke; Womack, Chris; Barry, Simon T.; Robson, Craig N.; Critchlow, Susan E.; Wedge, Stephen R.; Brooks, A. Nigel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1715-1727</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Continued androgen receptor (AR) expression and signaling is a key driver in castration-resistant prostate cancer (CRPC) after classical androgen ablation therapies have failed, and therefore remains a target for the treatment of progressive disease.  Here, we describe the biol. characterization of AZD3514, an orally bioavailable drug that inhibits androgen-dependent and -independent AR signaling.  AZD3514 modulates AR signaling through two distinct mechanisms, an inhibition of ligand-driven nuclear translocation of AR and a downregulation of receptor levels, both of which were obsd. in vitro and in vivo.  AZD3514 inhibited testosterone-driven seminal vesicle development in juvenile male rats and the growth of androgen-dependent Dunning R3327H prostate tumors in adult rats.  Furthermore, this class of compd. showed antitumor activity in the HID28 mouse model of CRPC in vivo.  AZD3514 is currently in phase I clin. evaluation.  Mol Cancer Ther; 12(9); 1715-27. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruUjB28XcSVrVg90H21EOLACvtfcHk0lgr124wW4wBZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2ltbfI&md5=553f0953189e9e5982a08c7933566ce0</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-12-1174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-12-1174%26sid%3Dliteratum%253Aachs%26aulast%3DLoddick%26aufirst%3DS.%2BA.%26aulast%3DRoss%26aufirst%3DS.%2BJ.%26aulast%3DThomason%26aufirst%3DA.%2BG.%26aulast%3DRobinson%26aufirst%3DD.%2BM.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DDunkley%26aufirst%3DT.%2BP.%26aulast%3DBrave%26aufirst%3DS.%2BR.%26aulast%3DBroadbent%26aufirst%3DN.%26aulast%3DStratton%26aufirst%3DN.%2BC.%26aulast%3DTrueman%26aufirst%3DD.%26aulast%3DMouchet%26aufirst%3DE.%26aulast%3DShaheen%26aufirst%3DF.%2BS.%26aulast%3DJacobs%26aufirst%3DV.%2BN.%26aulast%3DCumberbatch%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DR.%2BD.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DGaughan%26aufirst%3DL.%26aulast%3DWomack%26aufirst%3DC.%26aulast%3DBarry%26aufirst%3DS.%2BT.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26aulast%3DCritchlow%26aufirst%3DS.%2BE.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26atitle%3DAZD3514%253A%2520a%2520small%2520molecule%2520that%2520modulates%2520androgen%2520receptor%2520signaling%2520and%2520function%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2013%26volume%3D12%26spage%3D1715%26epage%3D1727%26doi%3D10.1158%2F1535-7163.MCT-12-1174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Waring, M. J.; Chen, H.; Rabow, A. A.; Walker, G.; Bobby, R.; Boiko, S.; Bradbury, R. H.; Callis, R.; Clark, E.; Dale, I.; Daniels, D. L.; Dulak, A.; Flavell, L.; Holdgate, G.; Jowitt, T. A.; Kikhney, A.; McAlister, M.; Méndez, J.; Ogg, D.; Patel, J.; Petteruti, P.; Robb, G. R.; Robers, M. B.; Stratton, N.; Svergun, D. I.; Wang, W.; Whittaker, D.; Wilson, D. M.; Yao, Y.</span><span> </span><span class="NLM_article-title">Potent and selective bivalent inhibitors of BET bromodomains</span>. Nat. Chem. Biol.<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">, </span>submitted for publication</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+J.+Waring&author=H.+Chen&author=A.+A.+Rabow&author=G.+Walker&author=R.+Bobby&author=S.+Boiko&author=R.+H.+Bradbury&author=R.+Callis&author=E.+Clark&author=I.+Dale&author=D.+L.+Daniels&author=A.+Dulak&author=L.+Flavell&author=G.+Holdgate&author=T.+A.+Jowitt&author=A.+Kikhney&author=M.+McAlister&author=J.+M%C3%A9ndez&author=D.+Ogg&author=J.+Patel&author=P.+Petteruti&author=G.+R.+Robb&author=M.+B.+Robers&author=N.+Stratton&author=D.+I.+Svergun&author=W.+Wang&author=D.+Whittaker&author=D.+M.+Wilson&author=Y.+Yao&title=Potent+and+selective+bivalent+inhibitors+of+BET+bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DPotent%2520and%2520selective%2520bivalent%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Philpott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wells, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strain-Damerell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess-Brown, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, S.</span><span> </span><span class="NLM_article-title">Assessing cellular efficacy of bromodomain inhibitors using fluorescence recovery after photobleaching</span> <span class="citation_source-journal">Epigenet. Chromatin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">14</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1186/1756-8935-7-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1186%2F1756-8935-7-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=25097667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2014&pages=14-25&author=M.+Philpottauthor=C.+M.+Rogersauthor=C.+Yappauthor=C.+Wellsauthor=J.-P.+Lambertauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=A.-C.+Gingrasauthor=S.+Knappauthor=S.+M%C3%BCller&title=Assessing+cellular+efficacy+of+bromodomain+inhibitors+using+fluorescence+recovery+after+photobleaching&doi=10.1186%2F1756-8935-7-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing cellular efficacy of bromodomain inhibitors using fluorescence recovery after photobleaching</span></div><div class="casAuthors">Philpott, Martin; Rogers, Catherine M.; Yapp, Clarence; Wells, Chris; Lambert, Jean-Philippe; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Gingras, Anne-Claude; Knapp, Stefan; Muller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14/1-14/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Acetylation of lysine residues in histone tails plays an important role in the regulation of gene transcription.  Bromdomains are the readers of acetylated histone marks, and, consequently, bromodomain-contg. proteins have a variety of chromatin-related functions.  Moreover, they are increasingly being recognized as important mediators of a wide range of diseases.  The first potent and selective bromodomain inhibitors are beginning to be described, but the diverse or unknown functions of bromodomain-contg. proteins present challenges to systematically demonstrating cellular efficacy and selectivity for these inhibitors.  Here we assess the viability of fluorescence recovery after photobleaching (FRAP) assays as a target agnostic method for the direct visualisation of an on-target effect of bromodomain inhibitors in living cells.  Results: Mutation of a conserved asparagine crucial for binding to acetylated lysines in the bromodomains of BRD3, BRD4 and TRIM24 all resulted in redn. of FRAP recovery times, indicating loss of or significantly reduced binding to acetylated chromatin, as did the addn. of known inhibitors.  Significant differences between wild type and bromodomain mutants for ATAD2, BAZ2A, BRD1, BRD7, GCN5L2, SMARCA2 and ZMYND11 required the addn. of the histone deacetylase inhibitor suberoylanilide hydroxamic acid (SAHA) to amplify the binding contribution of the bromodomain.  Under these conditions, known inhibitors decreased FRAP recovery times back to mutant control levels.  Mutation of the bromodomain did not alter FRAP recovery times for full-length CREBBP, even in the presence of SAHA, indicating that other domains are primarily responsible for anchoring CREBBP to chromatin.  However, FRAP assays with multimerised CREBBP bromodomains resulted in a good assay to assess the efficacy of bromodomain inhibitors to this target.  The bromodomain and extraterminal protein inhibitor PFI-1 was inactive against other bromodomain targets, demonstrating the specificity of the method.  Conclusions: Viable FRAP assays were established for 11 representative bromodomain-contg. proteins that broadly cover the bromodomain phylogenetic tree.  Addn. of SAHA can overcome weak binding to chromatin, and the use of tandem bromodomain constructs can eliminate masking effects of other chromatin binding domains.  Together, these results demonstrate that FRAP assays offer a potentially pan-bromodomain method for generating cell-based assays, allowing the testing of compds. with respect to cell permeability, on-target efficacy and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0eRoqpM6LTbVg90H21EOLACvtfcHk0liC1A-WHMUZIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVaqtrzE&md5=966744a9e7b3cba00776b76d65594fc6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1186%2F1756-8935-7-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-8935-7-14%26sid%3Dliteratum%253Aachs%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DAssessing%2520cellular%2520efficacy%2520of%2520bromodomain%2520inhibitors%2520using%2520fluorescence%2520recovery%2520after%2520photobleaching%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2014%26volume%3D7%26spage%3D14%26epage%3D25%26doi%3D10.1186%2F1756-8935-7-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brassil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bui, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolgos, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webborn, P. J.</span><span> </span><span class="NLM_article-title">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span> <span class="citation_source-journal">Drug Metab. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">252</span><span class="refDoi"> DOI: 10.3109/03602532.2012.691099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.3109%2F03602532.2012.691099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=22697420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=224-252&author=P.+Ballardauthor=P.+Brassilauthor=K.+H.+Buiauthor=H.+Dolgosauthor=C.+Peterssonauthor=A.+Tunekauthor=P.+J.+Webborn&title=The+right+compound+in+the+right+assay+at+the+right+time%3A+an+integrated+discovery+DMPK+strategy&doi=10.3109%2F03602532.2012.691099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The right compound in the right assay at the right time: an integrated discovery DMPK strategy</span></div><div class="casAuthors">Ballard, Peter; Brassil, Patrick; Bui, Khanh H.; Dolgos, Hugues; Petersson, Carl; Tunek, Anders; Webborn, Peter J. H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">224-252</span>CODEN:
                <span class="NLM_cas:coden">DMTRAR</span>;
        ISSN:<span class="NLM_cas:issn">0360-2532</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  The high rate of attrition during drug development and its assocd. high research and development (R&D) cost have put pressure on pharmaceutical companies to ensure that candidate drugs going to clin. testing have the appropriate quality such that the biol. hypothesis could be evaluated.  To help achieve this ambition, drug metab. and pharmacokinetic (DMPK) science and increasing investment have been deployed earlier in the R&D process.  To gain max. return on investment, it is essential that DMPK concepts are both appropriately integrated into the compd. design process and that compd. selection is focused on accurate prediction of likely outcomes in patients.  This article describes key principles that underpin the contribution of DMPK science for small-mol. research based on 15 years of discovery support in a major pharmaceutical company.  It does not aim to describe the breadth and depth of DMPK science, but more the practical application for decision making in real-world situations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolEqf17mivQbVg90H21EOLACvtfcHk0liC1A-WHMUZIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVelsLzF&md5=105fbbcfba6213caaadf41c8b10cf408</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.3109%2F03602532.2012.691099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F03602532.2012.691099%26sid%3Dliteratum%253Aachs%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBrassil%26aufirst%3DP.%26aulast%3DBui%26aufirst%3DK.%2BH.%26aulast%3DDolgos%26aufirst%3DH.%26aulast%3DPetersson%26aufirst%3DC.%26aulast%3DTunek%26aufirst%3DA.%26aulast%3DWebborn%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520right%2520compound%2520in%2520the%2520right%2520assay%2520at%2520the%2520right%2520time%253A%2520an%2520integrated%2520discovery%2520DMPK%2520strategy%26jtitle%3DDrug%2520Metab.%2520Rev.%26date%3D2012%26volume%3D44%26spage%3D224%26epage%3D252%26doi%3D10.3109%2F03602532.2012.691099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Sjögren, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westergren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanisch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindfors, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennernäs, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abrahamsson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannergren, C.</span><span> </span><span class="NLM_article-title">In silico predictions of gastrointestinal drug absorption in pharmaceutical product development: application of the mechanistic absorption model GI-Sim</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1016/j.ejps.2013.05.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.ejps.2013.05.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=23727464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSlsr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=679-698&author=E.+Sj%C3%B6grenauthor=J.+Westergrenauthor=I.+Grantauthor=G.+Hanischauthor=L.+Lindforsauthor=H.+Lennern%C3%A4sauthor=B.+Abrahamssonauthor=C.+Tannergren&title=In+silico+predictions+of+gastrointestinal+drug+absorption+in+pharmaceutical+product+development%3A+application+of+the+mechanistic+absorption+model+GI-Sim&doi=10.1016%2Fj.ejps.2013.05.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In silico predictions of gastrointestinal drug absorption in pharmaceutical product development: Application of the mechanistic absorption model GI-Sim</span></div><div class="casAuthors">Sjogren, Erik; Westergren, Jan; Grant, Iain; Hanisch, Gunilla; Lindfors, Lennart; Lennernas, Hans; Abrahamsson, Bertil; Tannergren, Christer</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">679-698</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Oral drug delivery is the predominant administration route for a major part of the pharmaceutical products used worldwide.  Further understanding and improvement of gastrointestinal drug absorption predictions is currently a highly prioritized area of research within the pharmaceutical industry.  The fraction absorbed (fabs) of an oral dose after administration of a solid dosage form is a key parameter in the estn. of the in vivo performance of an orally administrated drug formulation.  This study discloses an evaluation of the predictive performance of the mechanistic physiol. based absorption model GI-Sim.  GI-Sim deploys a compartmental gastrointestinal absorption and transit model as well as algorithms describing permeability, dissoln. rate, salt effects, partitioning into micelles, particle and micelle drifting in the aq. boundary layer, particle growth and amorphous or cryst. pptn.  Twelve APIs with reported or expected absorption limitations in humans, due to permeability, dissoln. and/or soly., were investigated.  Predictions of the intestinal absorption for different doses and formulations were performed based on physicochem. and biopharmaceutical properties, such as soly. in buffer and simulated intestinal fluid, mol. wt., pKa, diffusivity and mol. d., measured or estd. human effective permeability and particle size distribution.  The performance of GI-Sim was evaluated by comparing predicted plasma concn.-time profiles along with oral pharmacokinetic parameters originating from clin. studies in healthy individuals.  The capability of GI-Sim to correctly predict impact of dose and particle size as well as the in vivo performance of nanoformulations was also investigated.  The overall predictive performance of GI-Sim was good as >95% of the predicted pharmacokinetic parameters (Cmax and AUC) were within a 2-fold deviation from the clin. observations and the predicted plasma AUC was within one std. deviation of the obsd. mean plasma AUC in 74% of the simulations.  GI-Sim was also able to correctly capture the trends in dose- and particle size dependent absorption for the study drugs with soly. and dissoln. limited absorption, resp.  In addn., GI-Sim was also shown to be able to predict the increase in absorption and plasma exposure achieved with nanoformulations.  Based on the results, the performance of GI-Sim was shown to be suitable for early risk assessment as well as to guide decision making in pharmaceutical formulation development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriPOOmNPmHhrVg90H21EOLACvtfcHk0liC1A-WHMUZIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSlsr%252FJ&md5=b6eb6e66b7642463fd1191cfe3bdbc51</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2013.05.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2013.05.019%26sid%3Dliteratum%253Aachs%26aulast%3DSj%25C3%25B6gren%26aufirst%3DE.%26aulast%3DWestergren%26aufirst%3DJ.%26aulast%3DGrant%26aufirst%3DI.%26aulast%3DHanisch%26aufirst%3DG.%26aulast%3DLindfors%26aufirst%3DL.%26aulast%3DLennern%25C3%25A4s%26aufirst%3DH.%26aulast%3DAbrahamsson%26aufirst%3DB.%26aulast%3DTannergren%26aufirst%3DC.%26atitle%3DIn%2520silico%2520predictions%2520of%2520gastrointestinal%2520drug%2520absorption%2520in%2520pharmaceutical%2520product%2520development%253A%2520application%2520of%2520the%2520mechanistic%2520absorption%2520model%2520GI-Sim%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2013%26volume%3D49%26spage%3D679%26epage%3D698%26doi%3D10.1016%2Fj.ejps.2013.05.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sohlenius-Sternbeck, A.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afzelius, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prusis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neelissen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoogstraate, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bengtsson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gissberg, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sternbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petersson, C.</span><span> </span><span class="NLM_article-title">Evaluation of the human prediction of clearance from hepatocyte and microsome intrinsic clearance for 52 drug compounds</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">637</span><span class="NLM_x">–</span> <span class="NLM_lpage">649</span><span class="refDoi"> DOI: 10.3109/00498254.2010.500407</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.3109%2F00498254.2010.500407" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=20624033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVajsLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2010&pages=637-649&author=A.-K.+Sohlenius-Sternbeckauthor=L.+Afzeliusauthor=P.+Prusisauthor=J.+Neelissenauthor=J.+Hoogstraateauthor=J.+Johanssonauthor=E.+Flobyauthor=A.+Bengtssonauthor=O.+Gissbergauthor=J.+Sternbeckauthor=C.+Petersson&title=Evaluation+of+the+human+prediction+of+clearance+from+hepatocyte+and+microsome+intrinsic+clearance+for+52+drug+compounds&doi=10.3109%2F00498254.2010.500407"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the human prediction of clearance from hepatocyte and microsome intrinsic clearance for 52 drug compounds</span></div><div class="casAuthors">Sohlenius-Sternbeck, A.-K.; Afzelius, L.; Prusis, P.; Neelissen, J.; Hoogstraate, J.; Johansson, J.; Floby, E.; Bengtsson, A.; Gissberg, O.; Sternbeck, J.; Petersson, C.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">637-649</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">We compare 3 different approaches to scale clearance (CL) from human hepatocyte and microsome CLint (intrinsic CL) for 52 drug compds.  By using the well-stirred model with protein binding included only 11 and 30% of the compds. were predicted within 2-fold and the av. abs. fold errors (AAFE) for the predictions were 5.9 and 4.1 for hepatocytes and microsomes, resp.  When predictions were performed without protein binding, 59% of the compds. were predicted within 2-fold using either hepatocytes or microsomes and the AAFE was 2.2 and 2.3, resp.  For hepatocytes and microsomes there were significant correlations between predicted CLint in vivo (obtained from in vitro CLint) and measured CLint in vivo (obtained using the well-stirred model).  When CL was calcd. from the regression, 76 and 70% of the compds. were predicted within 2-fold and the AAFE was 1.6 and 1.8 for hepatocytes and microsomes, resp.  We demonstrate that microsomes and hepatocytes are in many cases comparable when scaling of CL is performed from regression.  By using the hepatocyte regression, CL for 82% of the compds. in an independent test set (n=11) were predicted within 2-fold (AAFE 1.4).  We suggest that a regression line that adjusts for systematic under-predictions should be the 1st-hand choice for scaling of CL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEy5gZ7yCu7bVg90H21EOLACvtfcHk0lhikTJ_qrGxHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVajsLnI&md5=ef98aa46451331023795cd3f56843eba</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3109%2F00498254.2010.500407&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2010.500407%26sid%3Dliteratum%253Aachs%26aulast%3DSohlenius-Sternbeck%26aufirst%3DA.-K.%26aulast%3DAfzelius%26aufirst%3DL.%26aulast%3DPrusis%26aufirst%3DP.%26aulast%3DNeelissen%26aufirst%3DJ.%26aulast%3DHoogstraate%26aufirst%3DJ.%26aulast%3DJohansson%26aufirst%3DJ.%26aulast%3DFloby%26aufirst%3DE.%26aulast%3DBengtsson%26aufirst%3DA.%26aulast%3DGissberg%26aufirst%3DO.%26aulast%3DSternbeck%26aufirst%3DJ.%26aulast%3DPetersson%26aufirst%3DC.%26atitle%3DEvaluation%2520of%2520the%2520human%2520prediction%2520of%2520clearance%2520from%2520hepatocyte%2520and%2520microsome%2520intrinsic%2520clearance%2520for%252052%2520drug%2520compounds%26jtitle%3DXenobiotica%26date%3D2010%26volume%3D40%26spage%3D637%26epage%3D649%26doi%3D10.3109%2F00498254.2010.500407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Sohlenius-Sternbeck, A.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferguson, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Middleton, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Projean, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floby, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bylund, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afzelius, L.</span><span> </span><span class="NLM_article-title">Practical use of the regression offset approach for the prediction of in vivo intrinsic clearance from hepatocytes</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">841</span><span class="NLM_x">–</span> <span class="NLM_lpage">853</span><span class="refDoi"> DOI: 10.3109/00498254.2012.669080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.3109%2F00498254.2012.669080" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2012&pages=841-853&author=A.-K.+Sohlenius-Sternbeckauthor=C.+Jonesauthor=D.+Fergusonauthor=B.+J.+Middletonauthor=D.+Projeanauthor=E.+Flobyauthor=J.+Bylundauthor=L.+Afzelius&title=Practical+use+of+the+regression+offset+approach+for+the+prediction+of+in+vivo+intrinsic+clearance+from+hepatocytes&doi=10.3109%2F00498254.2012.669080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3109%2F00498254.2012.669080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2012.669080%26sid%3Dliteratum%253Aachs%26aulast%3DSohlenius-Sternbeck%26aufirst%3DA.-K.%26aulast%3DJones%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DMiddleton%26aufirst%3DB.%2BJ.%26aulast%3DProjean%26aufirst%3DD.%26aulast%3DFloby%26aufirst%3DE.%26aulast%3DBylund%26aufirst%3DJ.%26aulast%3DAfzelius%26aufirst%3DL.%26atitle%3DPractical%2520use%2520of%2520the%2520regression%2520offset%2520approach%2520for%2520the%2520prediction%2520of%2520in%2520vivo%2520intrinsic%2520clearance%2520from%2520hepatocytes%26jtitle%3DXenobiotica%26date%3D2012%26volume%3D42%26spage%3D841%26epage%3D853%26doi%3D10.3109%2F00498254.2012.669080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Delmore, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Issa, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastritis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilpatrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paranal, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schinzel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKeown, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heffernan, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vakoc, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergsagel, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghobrial, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span> </span><span class="NLM_article-title">BET bromodomain inhibition as a therapeutic strategy to target c-Myc</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">146</span><span class="NLM_x">, </span> <span class="NLM_fpage">904</span><span class="NLM_x">–</span> <span class="NLM_lpage">917</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1016%2Fj.cell.2011.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=21889194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2011&pages=904-917&author=J.+E.+Delmoreauthor=G.+C.+Issaauthor=M.+E.+Lemieuxauthor=P.+B.+Rahlauthor=J.+Shiauthor=H.+M.+Jacobsauthor=E.+Kastritisauthor=T.+Gilpatrickauthor=R.+M.+Paranalauthor=J.+Qiauthor=M.+Chesiauthor=A.+Schinzelauthor=M.+R.+McKeownauthor=T.+P.+Heffernanauthor=C.+R.+Vakocauthor=P.+L.+Bergsagelauthor=I.+M.+Ghobrialauthor=P.+G.+Richardsonauthor=R.+A.+Youngauthor=W.+C.+Hahnauthor=K.+C.+Andersonauthor=A.+L.+Kungauthor=J.+E.+Bradnerauthor=C.+S.+Mitsiades&title=BET+bromodomain+inhibition+as+a+therapeutic+strategy+to+target+c-Myc&doi=10.1016%2Fj.cell.2011.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibition as a Therapeutic Strategy to Target c-Myc</span></div><div class="casAuthors">Delmore, Jake E.; Issa, Ghayas C.; Lemieux, Madeleine E.; Rahl, Peter B.; Shi, Jun-Wei; Jacobs, Hannah M.; Kastritis, Efstathios; Gilpatrick, Timothy; Paranal, Ronald M.; Qi, Jun; Chesi, Marta; Schinzel, Anna C.; McKeown, Michael R.; Heffernan, Timothy P.; Vakoc, Christopher R.; Bergsagel, P. Leif; Ghobrial, Irene M.; Richardson, Paul G.; Young, Richard A.; Hahn, William C.; Anderson, Kenneth C.; Kung, Andrew L.; Bradner, James E.; Mitsiades, Constantine S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">904-917</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: MYC contributes to the pathogenesis of a majority of human cancers, yet strategies to modulate the function of the c-Myc oncoprotein do not exist.  Toward this objective, we have targeted MYC transcription by interfering with chromatin-dependent signal transduction to RNA polymerase, specifically by inhibiting the acetyl-lysine recognition domains (bromodomains) of putative coactivator proteins implicated in transcriptional initiation and elongation.  Using a selective small-mol. bromodomain inhibitor, JQ1, we identify BET bromodomain proteins as regulatory factors for c-Myc.  BET inhibition by JQ1 downregulates MYC transcription, followed by genome-wide downregulation of Myc-dependent target genes.  In exptl. models of multiple myeloma, a Myc-dependent hematol. malignancy, JQ1 produces a potent antiproliferative effect assocd. with cell-cycle arrest and cellular senescence.  Efficacy of JQ1 in three murine models of multiple myeloma establishes the therapeutic rationale for BET bromodomain inhibition in this disease and other malignancies characterized by pathol. activation of c-Myc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpevD1QHKmOhLVg90H21EOLACvtfcHk0lhikTJ_qrGxHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2ht7fF&md5=013674b8224214a9a1c8f0b75583a5a6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DDelmore%26aufirst%3DJ.%2BE.%26aulast%3DIssa%26aufirst%3DG.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJacobs%26aufirst%3DH.%2BM.%26aulast%3DKastritis%26aufirst%3DE.%26aulast%3DGilpatrick%26aufirst%3DT.%26aulast%3DParanal%26aufirst%3DR.%2BM.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChesi%26aufirst%3DM.%26aulast%3DSchinzel%26aufirst%3DA.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DHeffernan%26aufirst%3DT.%2BP.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBergsagel%26aufirst%3DP.%2BL.%26aulast%3DGhobrial%26aufirst%3DI.%2BM.%26aulast%3DRichardson%26aufirst%3DP.%2BG.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26atitle%3DBET%2520bromodomain%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520c-Myc%26jtitle%3DCell%26date%3D2011%26volume%3D146%26spage%3D904%26epage%3D917%26doi%3D10.1016%2Fj.cell.2011.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Rhyasen, G. W.; Hattersley, M.; Yao, Y.; Dulak, A.; Wang, W.; Petteruti, P.; Dale, I.; Boiko, S.; Cheung, T.; Zhang, J.; Wen, S.; Castriotta, L.; Lawson, D.; Collins, M.; Ahdesmaki, M. J.; Walker, G.; O’Connor, G.; Yeh, T.; Rabow, A. A.; Dry, J.; Reimer, C.; Lyne, P.; Mills, G. B.; Fawell, S. E.; Waring, M. J.; Zinda, M.; Clark, E.; Chen, H.</span><span> </span><span class="NLM_article-title">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span>. Mol. Cancer Ther.<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">, </span>submitted for publication</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=27573426" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=G.+W.+Rhyasen&author=M.+Hattersley&author=Y.+Yao&author=A.+Dulak&author=W.+Wang&author=P.+Petteruti&author=I.+Dale&author=S.+Boiko&author=T.+Cheung&author=J.+Zhang&author=S.+Wen&author=L.+Castriotta&author=D.+Lawson&author=M.+Collins&author=M.+J.+Ahdesmaki&author=G.+Walker&author=G.+O%E2%80%99Connor&author=T.+Yeh&author=A.+A.+Rabow&author=J.+Dry&author=C.+Reimer&author=P.+Lyne&author=G.+B.+Mills&author=S.+E.+Fawell&author=M.+J.+Waring&author=M.+Zinda&author=E.+Clark&author=H.+Chen&title=AZD5153%3A+a+novel+bivalent+BET+bromodomain+inhibitor+highly+active+against+hematologic+malignancies"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26atitle%3DAZD5153%253A%2520a%2520novel%2520bivalent%2520BET%2520bromodomain%2520inhibitor%2520highly%2520active%2520against%2520hematologic%2520malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Arnold, D. L.; Foreman, K. W.; Jin, M.; Wanner, J.; Werner, D. S.</span><span> </span><span class="NLM_article-title">Preparation of bivalent bromodomain ligands for treating a disease associated with a protein having tandem bromodomains</span>. PCT Int. Appl. WO 2013033268, March 7th,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=D.+L.+Arnold&author=K.+W.+Foreman&author=M.+Jin&author=J.+Wanner&author=D.+S.+Werner&title=Preparation+of+bivalent+bromodomain+ligands+for+treating+a+disease+associated+with+a+protein+having+tandem+bromodomains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DArnold%26aufirst%3DD.%2BL.%26atitle%3DPreparation%2520of%2520bivalent%2520bromodomain%2520ligands%2520for%2520treating%2520a%2520disease%2520associated%2520with%2520a%2520protein%2520having%2520tandem%2520bromodomains%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Rodrik-Outmezguine, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okaniwa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novotny, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McWhirter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banaji, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Won, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Stanchina, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barratt, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosulich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Overcoming mTOR resistance mutations with a new-generation mTOR inhibitor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">534</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1038/nature17963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1038%2Fnature17963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=27279227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFOktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=272-276&author=V.+S.+Rodrik-Outmezguineauthor=M.+Okaniwaauthor=Z.+Yaoauthor=C.+J.+Novotnyauthor=C.+McWhirterauthor=A.+Banajiauthor=H.+Wonauthor=W.+Wongauthor=M.+Bergerauthor=E.+de+Stanchinaauthor=D.+G.+Barrattauthor=S.+Cosulichauthor=T.+Klinowskaauthor=N.+Rosenauthor=K.+M.+Shokat&title=Overcoming+mTOR+resistance+mutations+with+a+new-generation+mTOR+inhibitor&doi=10.1038%2Fnature17963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming mTOR resistance mutations with a new-generation mTOR inhibitor</span></div><div class="casAuthors">Rodrik-Outmezguine, Vanessa S.; Okaniwa, Masanori; Yao, Zhan; Novotny, Chris J.; McWhirter, Claire; Banaji, Arpitha; Won, Helen; Wong, Wai; Berger, Mike; de Stanchina, Elisa; Barratt, Derek G.; Cosulich, Sabina; Klinowska, Teresa; Rosen, Neal; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7606</span>),
    <span class="NLM_cas:pages">272-276</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Precision medicines exert selective pressure on tumor cells that leads to the preferential growth of resistant subpopulations, necessitating the development of next-generation therapies to treat the evolving cancer.  The PIK3CA-AKT-mTOR pathway is one of the most commonly activated pathways in human cancers, which has led to the development of small-mol. inhibitors that target various nodes in the pathway.  Among these agents, first-generation mTOR inhibitors (rapalogs) have caused responses in 'N-of-1' cases, and second-generation mTOR kinase inhibitors (TORKi) are currently in clin. trials.  Here the authors sought to delineate the likely resistance mechanisms to existing mTOR inhibitors in human cell lines, as a guide for next-generation therapies.  The mechanism of resistance to the TORKi was unusual in that intrinsic kinase activity of mTOR was increased, rather than a direct active-site mutation interfering with drug binding.  Indeed, identical drug-resistant mutations have been also identified in drug-naive patients, suggesting that tumors with activating MTOR mutations will be intrinsically resistant to second-generation mTOR inhibitors.  The authors report the development of a new class of mTOR inhibitors that overcomes resistance to existing first- and second-generation inhibitors.  The third-generation mTOR inhibitor exploits the unique juxtaposition of two drug-binding pockets to create a bivalent interaction that allows inhibition of these resistant mutants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyillD_NPGSLVg90H21EOLACvtfcHk0lgqgxumnHHDZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFOktLo%253D&md5=8e9c2bb6b53e5e97b9f48d6ec2ed91d3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature17963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17963%26sid%3Dliteratum%253Aachs%26aulast%3DRodrik-Outmezguine%26aufirst%3DV.%2BS.%26aulast%3DOkaniwa%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DNovotny%26aufirst%3DC.%2BJ.%26aulast%3DMcWhirter%26aufirst%3DC.%26aulast%3DBanaji%26aufirst%3DA.%26aulast%3DWon%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DW.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3Dde%2BStanchina%26aufirst%3DE.%26aulast%3DBarratt%26aufirst%3DD.%2BG.%26aulast%3DCosulich%26aufirst%3DS.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DOvercoming%2520mTOR%2520resistance%2520mutations%2520with%2520a%2520new-generation%2520mTOR%2520inhibitor%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D272%26epage%3D276%26doi%3D10.1038%2Fnature17963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Callis, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradbury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Challinor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, G.</span><span> </span><span class="NLM_article-title">A screening assay cascade to identify and characterize novel selective estrogen receptor downregulators (SERDs)</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">748</span><span class="NLM_x">–</span> <span class="NLM_lpage">759</span><span class="refDoi"> DOI: 10.1177/1087057115580298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1177%2F1087057115580298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=25851036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=748-759&author=R.+Callisauthor=A.+Rabowauthor=M.+Tongeauthor=R.+Bradburyauthor=M.+Challinorauthor=K.+Robertsauthor=K.+Jonesauthor=G.+Walker&title=A+screening+assay+cascade+to+identify+and+characterize+novel+selective+estrogen+receptor+downregulators+%28SERDs%29&doi=10.1177%2F1087057115580298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A screening assay cascade to identify and characterize novel selective estrogen receptor downregulators (SERDs)</span></div><div class="casAuthors">Callis, Rowena; Rabow, Alfred; Tonge, Michael; Bradbury, Robert; Challinor, Mairi; Roberts, Karen; Jones, Karen; Walker, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">748-759</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Here, we describe an approach to identify novel selective estrogen receptor downregulator (SERD) compds. with improved properties such as oral bioavailability and the potential of increased efficacy compared to currently marketed drug treatments.  Previously, methodologies such as Western blotting and transient cell reporter assays have been used to identify and characterize SERD compds., but such approaches can be limited due to low throughput and sensitivity, resp.  We have used an endogenous cell-imaging strategy that has both the throughput and sensitivity to support a large-scale hit-to-lead program to identify novel compds.  A screening cascade with a suite of assays has been developed to characterize compds. that modulate estrogen receptor α (ERα)-mediated signaling or downregulate ERα levels in cells.  Initially, from a focused high-throughput screening, novel ERα binders were identified that could be modified chem. into ERα downregulators.  Following this, cellular assays helped det. the mechanism of action of compds. to distinguish between on-target and off-target compds. and differentiate SERDs, selective estrogen receptor modulator (SERM) compds., and agonist ERα ligands.  Data are shown to exemplify the characterization of ERα-mediated signaling inhibitors using a selection of literature compds. and illustrate how this cascade has been used to drive the chem. design of novel SERD compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqNNWMPPJDNrVg90H21EOLACvtfcHk0lgqgxumnHHDZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsF2nsbzE&md5=13c27dbf08f315d469f81b2bcefb4499</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1087057115580298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057115580298%26sid%3Dliteratum%253Aachs%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DRabow%26aufirst%3DA.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DBradbury%26aufirst%3DR.%26aulast%3DChallinor%26aufirst%3DM.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DWalker%26aufirst%3DG.%26atitle%3DA%2520screening%2520assay%2520cascade%2520to%2520identify%2520and%2520characterize%2520novel%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2015%26volume%3D20%26spage%3D748%26epage%3D759%26doi%3D10.1177%2F1087057115580298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kilkenny, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browne, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cuthill, I. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, D. G.</span><span> </span><span class="NLM_article-title">Improving bioscience research reporting: the ARRIVE guidelines for reporting animal research</span> <span class="citation_source-journal">PLoS Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000412</span><span class="refDoi"> DOI: 10.1371/journal.pbio.1000412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=10.1371%2Fjournal.pbio.1000412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;key=20613859" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=e1000412&author=C.+Kilkennyauthor=W.+J.+Browneauthor=I.+C.+Cuthillauthor=M.+Emersonauthor=D.+G.+Altman&title=Improving+bioscience+research+reporting%3A+the+ARRIVE+guidelines+for+reporting+animal+research&doi=10.1371%2Fjournal.pbio.1000412"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.1000412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.1000412%26sid%3Dliteratum%253Aachs%26aulast%3DKilkenny%26aufirst%3DC.%26aulast%3DBrowne%26aufirst%3DW.%2BJ.%26aulast%3DCuthill%26aufirst%3DI.%2BC.%26aulast%3DEmerson%26aufirst%3DM.%26aulast%3DAltman%26aufirst%3DD.%2BG.%26atitle%3DImproving%2520bioscience%2520research%2520reporting%253A%2520the%2520ARRIVE%2520guidelines%2520for%2520reporting%2520animal%2520research%26jtitle%3DPLoS%2520Biol.%26date%3D2010%26volume%3D8%26spage%3De1000412%26doi%3D10.1371%2Fjournal.pbio.1000412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-NOTES-d45e7377-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i77"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00070">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_41506"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00070">10.1021/acs.jmedchem.6b00070</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystallization and structure of BRD4(1) in complex with <b>13</b>, correlations of potency values between BRD4 FP potency, ER downregulation, c-Myc downregulation and growth inhibition, and Alexa647-labeled probe (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_001.pdf">PDF</a>)</p></li><li><p class="inline">List of compounds with key data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_001.pdf">jm6b00070_si_001.pdf (307.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00070/suppl_file/jm6b00070_si_002.csv">jm6b00070_si_002.csv (0.76 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00070%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-17%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00070" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993283c911248d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
