
<html lang="en"     class="pb-page"  data-request-id="5f9999a7-73d8-4d7f-917d-1f00afeb61bb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.1c00414;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A Druglike Small Molecule that Targets r(CCUG) Repeats in Myotonic Dystrophy Type 2 Facilitates Degradation by RNA Quality Control Pathways" /></meta><meta name="dc.Creator" content="Sarah  Wagner-Griffin" /></meta><meta name="dc.Creator" content="Masahito  Abe" /></meta><meta name="dc.Creator" content="Raphael I.  Benhamou" /></meta><meta name="dc.Creator" content="Alicia J.  Angelbello" /></meta><meta name="dc.Creator" content="Kamalakannan  Vishnu" /></meta><meta name="dc.Creator" content="Jonathan L.  Chen" /></meta><meta name="dc.Creator" content="Jessica L.  Childs-Disney" /></meta><meta name="dc.Creator" content="Matthew D.  Disney" /></meta><meta name="dc.Description" content="Myotonic dystrophy type 2 (DM2) is one of &gt;40 microsatellite disorders caused by RNA repeat expansions. The DM2 repeat expansion, r(CCUG)exp (where “exp” denotes expanded repeating nucleotides), is..." /></meta><meta name="Description" content="Myotonic dystrophy type 2 (DM2) is one of &gt;40 microsatellite disorders caused by RNA repeat expansions. The DM2 repeat expansion, r(CCUG)exp (where “exp” denotes expanded repeating nucleotides), is..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 8, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00414" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00414" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00414" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00414" /></link>
        
    
    

<title>A Druglike Small Molecule that Targets r(CCUG) Repeats in Myotonic Dystrophy Type 2 Facilitates Degradation by RNA Quality Control Pathways | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00414" /></meta><meta property="og:title" content="A Druglike Small Molecule that Targets r(CCUG) Repeats in Myotonic Dystrophy Type 2 Facilitates Degradation by RNA Quality Control Pathways" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0006.jpeg" /></meta><meta property="og:description" content="Myotonic dystrophy type 2 (DM2) is one of &gt;40 microsatellite disorders caused by RNA repeat expansions. The DM2 repeat expansion, r(CCUG)exp (where “exp” denotes expanded repeating nucleotides), is harbored in intron 1 of the CCHC-type zinc finger nucleic acid binding protein (CNBP). The expanded RNA repeat causes disease by a gain-of-function mechanism, sequestering various RNA-binding proteins including the pre-mRNA splicing regulator MBNL1. Sequestration of MBNL1 results in its loss-of-function and concomitant deregulation of the alternative splicing of its native substrates. Notably, this r(CCUG)exp causes retention of intron 1 in the mature CNBP mRNA. Herein, we report druglike small molecules that bind the structure adopted by r(CCUG)exp and improve DM2-associated defects. These small molecules were optimized from screening hits from an RNA-focused small-molecule library to afford a compound that binds r(CCUG)exp specifically and with nanomolar affinity, facilitates endogenous degradation of the aberrantly retained intron in which it is harbored, and rescues alternative splicing defects." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00414"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00414">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00414&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00414&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00414&amp;href=/doi/10.1021/acs.jmedchem.1c00414" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8474-8485</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00413" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00420" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Druglike Small Molecule that Targets r(CCUG) Repeats in Myotonic Dystrophy Type 2 Facilitates Degradation by RNA Quality Control Pathways</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sarah Wagner-Griffin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sarah Wagner-Griffin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sarah++Wagner-Griffin">Sarah Wagner-Griffin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masahito Abe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masahito Abe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masahito++Abe">Masahito Abe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Raphael I. Benhamou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Raphael I. Benhamou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Raphael+I.++Benhamou">Raphael I. Benhamou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alicia J. Angelbello</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alicia J. Angelbello</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alicia+J.++Angelbello">Alicia J. Angelbello</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kamalakannan Vishnu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kamalakannan Vishnu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kamalakannan++Vishnu">Kamalakannan Vishnu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jonathan L. Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan L. Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan+L.++Chen">Jonathan L. Chen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7907-9732" title="Orcid link">https://orcid.org/0000-0001-7907-9732</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jessica L. Childs-Disney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jessica L. Childs-Disney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jessica+L.++Childs-Disney">Jessica L. Childs-Disney</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Disney</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Disney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#fc98958f929985bc8f9f8e958c8c8fd2999889"><span class="__cf_email__" data-cfemail="74101d071a110d340717061d0404075a111001">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Disney">Matthew D. Disney</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-8486-1796" title="Orcid link">https://orcid.org/0000-0001-8486-1796</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00414&amp;href=/doi/10.1021%2Facs.jmedchem.1c00414" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8474–8485</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 8, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 March 2021</li><li><span class="item_label"><b>Published</b> online</span>8 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00414" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00414</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8474%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSarah%2BWagner-Griffin%252C%2BMasahito%2BAbe%252C%2BRaphael%2BI.%2BBenhamou%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D12%26contentID%3Dacs.jmedchem.1c00414%26title%3DA%2BDruglike%2BSmall%2BMolecule%2Bthat%2BTargets%2Br%2528CCUG%2529%2BRepeats%2Bin%2BMyotonic%2BDystrophy%2BType%2B2%2BFacilitates%2BDegradation%2Bby%2BRNA%2BQuality%2BControl%2BPathways%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8485%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00414"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">889</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00414" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Druglike Small Molecule that Targets r(CCUG) Repeats in Myotonic Dystrophy Type 2 Facilitates Degradation by RNA Quality Control Pathways&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;Wagner-Griffin&quot;},{&quot;first_name&quot;:&quot;Masahito&quot;,&quot;last_name&quot;:&quot;Abe&quot;},{&quot;first_name&quot;:&quot;Raphael&quot;,&quot;last_name&quot;:&quot;I. Benhamou&quot;},{&quot;first_name&quot;:&quot;Alicia&quot;,&quot;last_name&quot;:&quot;J. Angelbello&quot;},{&quot;first_name&quot;:&quot;Kamalakannan&quot;,&quot;last_name&quot;:&quot;Vishnu&quot;},{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;L. Chen&quot;},{&quot;first_name&quot;:&quot;Jessica&quot;,&quot;last_name&quot;:&quot;L. Childs-Disney&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Disney&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8474-8485&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00414&quot;},&quot;abstract&quot;:&quot;Myotonic dystrophy type 2 (DM2) is one of &gt;40 microsatellite disorders caused by RNA repeat expansions. The DM2 repeat expansion, r(CCUG)exp (where “exp” denotes expanded repeating nucleotides), is harbored in intron 1 of the CCHC-type zinc finger nucleic acid binding protein (CNBP). The expanded RNA repeat causes disease by a gain-of-function mechanism, sequestering various RNA-binding proteins including the pre-mRNA splicing regulator MBNL1. Sequestration of MBNL1 results in its loss-of-function and concomitant deregulation of the alternative splicing of its native substrates. Notably, this r(CCUG)exp causes retention of intron 1 in the mature CNBP mRNA. Herein, we report druglike small molecules that bind the structure adopted by r(CCUG)exp and improve DM2-associated defects. These small molecules were optimized from screening hits from an RNA-focused small-molecule library to afford a compound that binds r(CCUG)exp specifically and with nanomolar affinity, facilitates endogenous degradation of the aberran&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00414&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00414" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00414&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00414" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00414&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00414" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00414&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00414&amp;href=/doi/10.1021/acs.jmedchem.1c00414" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00414" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00414" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00414%26sid%3Dliteratum%253Aachs%26pmid%3D34101465%26genre%3Darticle%26aulast%3DWagner-Griffin%26date%3D2021%26atitle%3DA%2BDruglike%2BSmall%2BMolecule%2Bthat%2BTargets%2Br%2528CCUG%2529%2BRepeats%2Bin%2BMyotonic%2BDystrophy%2BType%2B2%2BFacilitates%2BDegradation%2Bby%2BRNA%2BQuality%2BControl%2BPathways%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8474%26epage%3D8485%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291356" title="Defects">Defects</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Myotonic dystrophy type 2 (DM2) is one of >40 microsatellite disorders caused by RNA repeat expansions. The DM2 repeat expansion, r(CCUG)<sup>exp</sup> (where “exp” denotes expanded repeating nucleotides), is harbored in intron 1 of the CCHC-type zinc finger nucleic acid binding protein (CNBP). The expanded RNA repeat causes disease by a gain-of-function mechanism, sequestering various RNA-binding proteins including the pre-mRNA splicing regulator MBNL1. Sequestration of MBNL1 results in its loss-of-function and concomitant deregulation of the alternative splicing of its native substrates. Notably, this r(CCUG)<sup>exp</sup> causes retention of intron 1 in the mature CNBP mRNA. Herein, we report druglike small molecules that bind the structure adopted by r(CCUG)<sup>exp</sup> and improve DM2-associated defects. These small molecules were optimized from screening hits from an RNA-focused small-molecule library to afford a compound that binds r(CCUG)<sup>exp</sup> specifically and with nanomolar affinity, facilitates endogenous degradation of the aberrantly retained intron in which it is harbored, and rescues alternative splicing defects.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60255" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60255" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">RNA repeat expansions cause >40 microsatellite disorders, including Huntington’s disease [HD, r(CAG)<sup>exp</sup>], C9orf72 amyotrophic lateral sclerosis/frontotemporal dementia [c9ALS/FTD, r(G<sub>4</sub>C<sub>2</sub>)<sup>exp</sup>], and myotonic dystrophy types 1 and 2 [DM1, r(CUG)<sup>exp</sup> and DM2, r(CCUG)<sup>exp</sup>, respectively].<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Repeat expansions cause disease through various mechanisms dependent upon its location within a gene. For example, an RNA gain-of-function mechanism occurs when repeat sequesters functionally inactivate proteins that regulate alternative pre-mRNA splicing.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Repeat expansions can also contribute to disease via an aberrant translational mechanism that generates toxic proteins, named repeat-associated non-ATG translation.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Recently, it was discovered that GC-rich RNA repeat expansions located in introns, such as r(CCUG)<sup>exp</sup>, r(CUG)<sup>exp</sup>, and r(G<sub>4</sub>C<sub>2</sub>)<sup>exp</sup>, cause retention of the intron in which they are harbored in mature mRNA species.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">The DM2 repeat expansion, r(CCUG)<sup>exp</sup>, is harbored in intron 1 of CCHC-type zinc finger nucleic acid binding protein gene (CNBP).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> This r(CCUG)<sup>exp</sup> folds into a structure containing repeating 2 × 2 CU/UC internal loops (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A),<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> which bind and sequester muscleblind-like 1 (MBNL1), an important regulator of alternative pre-mRNA splicing, in nuclear foci.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Sequestration of MBNL1 by r(CCUG)<sup>exp</sup> results in pre-mRNA splicing defects of MBNL1-regulated genes. For example, insulin receptor (<i>IR</i>) exon 11 is excluded too frequently in DM2 cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> r(CCUG)<sup>exp</sup> can also cause dysfunction through intron retention, where CNBP itself is mis-spliced, and the r(CCUG)<sup>exp</sup>-containing intron 1 is retained in mature mRNA species (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. r(CCUG)<sup>exp</sup>-mediated defects in DM2. (A) DM2 is caused by r(CCUG)<sup>exp</sup> in CNBP intron 1, which folds into a structure with repeating 2 × 2 nucleotide internal loops that sequester regulatory proteins involved in pre-mRNA splicing such as MBNL1. (B) Sequestration of MBNL1 by r(CCUG)<sup>exp</sup> results in splicing defects in its native pre-mRNAs substrates. For example, <i>IR</i> exon 11 is included ∼70% of the time in healthy cells, but only ∼50% in DM2-affected cells. (C) r(CCUG)<sup>exp</sup> also causes intron retention where intron 1 is aberrantly retained in CNBP mature RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We have previously shown that intron retention is due to MBNL1 binding; knock-down of MBNL1 reduces intron retention, whereas forced expression of MBNL1 increases intron retenion.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Thus, small-molecule recognition of r(CCUG)<sup>exp</sup> liberates MBNL1 and rescues deregulated splicing events and intron retention. In particular, we have shown that kanamycin A derivatives that bind r(CCUG)<sup>exp</sup> can specifically improve DM2-associated defects and reduce the levels of intron 1 containing r(CCUG)<sup>exp</sup>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In another example of an intron-retained RNA repeat [r(CUG)<sup>exp</sup> in intron 3 of transcription factor 4 mRNA (<i>TCF</i>4) in Fuchs endothelial corneal dystrophy (FECD)],<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> small-molecule binding results in decay of the r(CUG)<sup>exp</sup>-containing intron 3 through the RNA exosome.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Thus, small-molecule recognition of RNA repeat expansions in retained introns can allow for targeted RNA degradation through stimulation of RNA quality control mechanisms. In addition to interfacing of disease-causing RNAs with endogenous decay pathways, other methods for targeted RNA degradation rely on chimeric compounds comprising an RNA-binding ligand attached to a cleaving module. However, this strategy increases compound molecular weight and can affect various pharmacological and physicochemical properties such as membrane permeability and solubility.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17">(13−17)</a></div><div class="NLM_p last">Although our previous studies targeting r(CCUG)<sup>exp</sup> showed that small molecules can indeed interface the toxic RNA with endogenous degradation pathways, the aminoglycosides from which they are derived are not particularly druglike and provide little opportunity for optimization.<a onclick="showRef(event, 'ref10 ref18'); return false;" href="javascript:void(0);" class="ref ref10 ref18">(10,18)</a> Herein, we studied a collection of RNA-focused, druglike small molecules to target r(CCUG)<sup>exp</sup>. Medicinal chemistry optimization of screening hits afforded a druglike small molecule that binds r(CCUG)<sup>exp</sup> with nanomolar affinity, specifically degrades r(CCUG)<sup>exp</sup>-containing intron 1, and improves DM2-associated defects.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23678" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23678" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Studying an RNA-Focused Small-Molecule Compound Collection for Targeting r(CCUG)</h3><div class="NLM_p">To identify novel small molecules that bind r(CCUG)<sup>exp</sup>, a druglike RNA-focused small-molecule library containing 3271 compounds was employed.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This library was previously designed to contain structurally diverse small molecules that have chemotypes that confer avidity toward RNA.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> These chemotypes include benzimidazoles, indoles, thiazoles, and quinazolines. Furthermore, the compounds in this library are druglike as defined by satisfying Lipinksi’s Rule of 5,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> where Log <i>P</i> is < 5, molecular weight is < 500, the number of hydrogen bond donors is <5, and the number of hydrogen bond acceptors is <10 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Historically, druglikeness has been defined by satisfying the Rule of 5; however, an increasing number of approved orally bioavailable drugs fall outside of these guidelines, and parameters such as molecular weight are increasing.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> On average, the library also contains similar physicochemical properties to known drugs in DrugBank (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Thus, this library can be used to identify druglike small molecules that bind to RNA structures.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Small molecules that inhibit r(CCUG)<sup>exp</sup>–MBNL1 complex formation <i>in vitro</i>. (A) Characteristics of an RNA-focused small molecule library compared to characteristics of known drugs in DrugBank. (B) IC<sub>50</sub> values of hit compounds from the RNA-focused library screen for disrupting the r(CCUG)<sub>12</sub>–MBNL1 complex (<i>n</i> = 2). Molecules in red have IC<sub>50</sub>s < 50 μM. (C) Chemical structures of compounds with IC<sub>50</sub>s < 50 μM. (D) Rescue of the <i>IR</i> exon 11 splicing defect by <b>3</b>, <b>4</b>, <b>5</b>, and <b>7</b> in DM2 patient-derived fibroblasts (<i>n</i> = 2). Error bars represent standard deviation (SD) for all panels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To identify compounds that bind r(CCUG)<sup>exp</sup>, a previously reported time-resolved fluorescence resonance energy transfer (TR-FRET) assay was used that measures disruption of a preformed r(CCUG)<sub>12</sub>–MBNL1 complex by small molecules or other modalities.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> The 3271 member library was screened at 200 μM, and 44 compounds disrupted complex formation by >60%, which corresponds to three standard deviations from the mean (or 3σ), affording a hit rate of 1.34% (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S1 and Table S1</a>). Of the 44 hits, 38 contained a similar substituted N2-phenylquinazoline-2,4-diamine core, while the other hits contained benzimidazoles and indazoles (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S2</a>). Importantly, these 44 compounds are druglike as they have similar physicochemical properties to the rest of the library and to known drugs in DrugBank (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). On average, the 44 hit compounds have a molecular weight of 327 ± 35 g/mol, a c Log <i>P</i> of 3.2 ± 0.9, TPSA of 76 ± 15 Å, and 2.5 ± 0.9 hydrogen bond donors. The <i>in vitro</i> potencies (IC<sub>50</sub>s) of the 44 compounds were then measured in the TR-FRET assay, affording seven compounds (<b>1</b>–<b>7</b>) with IC<sub>50</sub>s < 50 μM (highlighted in red in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Table S1</a>). Interestingly, all seven of these compounds contained the substituted N2-phenylquinazoline-2,4-diamine core.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Analysis of Hit Compounds</h3><div class="NLM_p">The binding of <b>1</b>–<b>7</b> to an RNA containing a single 2 × 2 CU/UC was further studied by NMR spectroscopy. Compounds <b>1</b>, <b>2</b>, and <b>6</b> showed significant aggregation in NMR experiments and were excluded from further analysis. In contrast, waterLOGSY spectra showed that <b>3</b>, <b>4</b>, <b>5</b>, and <b>7</b> bound to the RNA, based on positive phase signals in the presence of RNA (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figures S3–S6</a>). To confirm these interactions, one-dimensional (1D) NMR spectra of <b>3</b>, <b>4</b>, <b>5</b>, and <b>7</b> bound to the RNA containing one 2×2 CU/UC internal loop were analyzed. Aromatic protons of the unbound RNA were assigned via a two-dimensional (2D) nuclear over-Hauser effect spectroscopy (NOESY) spectrum (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S7</a>). Significant shifts in resonances corresponding to H6 and H8 protons of the RNA were observed in the presence of these four compounds, indicating binding to the RNA (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S8</a>).</div><div class="NLM_p last">After confirming binding to r(CCUG) repeats by NMR spectroscopy, the compounds were evaluated for their ability to improve the deregulation of the <i>IR</i> exon 11 alternative splicing in DM2 patient-derived fibroblasts. All four compounds were well tolerated in the patient-derived fibroblasts, as no significant toxicity was observed up to a 20 μM dose (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S9</a>). As previously mentioned, <i>IR</i> exon 11 is frequently excluded in DM2 patient-derived fibroblasts as compared to wild-type cells. Interestingly, the most potent compound, <b>7</b>, rescued splicing by ∼50% at 5 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis and <i>In Vitro</i> Analysis of Derivatives of <b>7</b></h3><div class="NLM_p">To improve the potency of <b>7</b>, a library of derivatives that replace the functional groups on the N2-phenylquinazoline-2,4-diamine core was synthesized and then evaluated for disrupting the r(CCUG)<sub>12</sub>–MBNL1 complex <i>in vitro</i>. As an overview of medicinal chemistry effort, analogues were synthesized to explore the quinazoline core (<b>54</b>–<b>61</b>), the aniline moiety (<b>45</b>–<b>53</b>), and combinations thereof (<b>62</b>–<b>64</b>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S10</a>). Synthetically, compounds were prepared starting from 2,4-dichloroquinazoline or 2,4-dichloropyrimidine analogue, followed by the addition of 3-aminopropan-1-ol in a 4-position selective manner. The aniline derivatives at the N2 position were then installed in a microwave-assisted reaction (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Scheme 1</a>, Supporting Information). Of the nine compounds in which N2-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine was replaced with various aniline analogues, three compounds containing 2-methylbenzooxazole (<b>49</b>), 2-methylbenzothiazole (<b>50</b>), and 1-methyl-1<i>H</i>-indazole (<b>51</b>) substituents had IC<sub>50</sub>s that were ∼2–8 times lower than <b>7</b> (IC<sub>50</sub> = 48.2 ± 0.07 μM) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S11, and Table S1</a>). Eight derivatives (<b>54</b>–<b>61</b>) with various substitution patterns of quinazolines and fused pyrimidines were then evaluated, and one compound containing 6-methylquinazoline (<b>58</b>) was identified with an IC<sub>50</sub> of 12.7 ± 0.2 μM, about 4 times more potent than <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S11, and Table S1</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Lead optimization of <b>7</b> and activity of analogues <i>in vitro</i> and in DM2 patient-derived fibroblasts. (A) Synthetic optimization scheme of compound <b>7</b>. (B) Structures of derivatives of <b>7</b> where “R” indicates the position(s) that were varied. IC<sub>50</sub> values for disrupting the r(CCUG)<sub>12</sub>–MBNL1 complex for each derivative (<i>n</i> = 3). (C) Structures of compounds with IC<sub>50</sub>s < 5 μM. (D) Rescue of <i>IR</i> exon 11 mis-splicing by nontoxic compounds <b>63</b>, <b>79</b>, and <b>80</b> in DM2 patient-derived fibroblasts (<i>n</i> = 3). ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, as determined by a one-way analysis of variance (ANOVA) relative to untreated (“0”). (E) Binding affinity curve of <b>63</b> and r(CCUG)<sub>12</sub> (<i>K</i><sub>d</sub> = 192 ± 20 nM). (F) Binding affinity curve of <b>63</b> and a base pair control RNA (<i>K</i><sub>d</sub> > 20 μM). Error bars represent standard deviation (SD) for all panels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To combine features of the derivates with improved IC<sub>50</sub>s, three additional compounds were synthesized that contained 6-methylquinazoline and 2-methylbenzooxazole (<b>62</b>), 2-methylbenzothiazole (<b>63</b>), and 1-methyl-1<i>H</i>-indazole (<b>64</b>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S11, and Table S1</a>). Compounds <b>62</b> and <b>63</b> had slightly lower IC<sub>50</sub>s than corresponding analogues without a methyl substituent at the 6 position (<b>49</b> and <b>50</b>, respectively), and <b>63</b> was the most potent compound synthesized thus far with an IC<sub>50</sub> of 2.2 ± 0.5 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S11, and Table S1</a>).</div><div class="NLM_p last">To explore further SAR around the benzothiazole moiety of <b>63</b>, 18 analogues with extensions built onto benzothiazole were synthesized (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Schemes 2 and 3</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S10</a>). Six of these analogues were synthesized via Suzuki coupling using a bromobenzothiazole intermediate of <b>63</b> (Intermediate D, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Scheme 2</a>). Notably, an aromatic ring was added to derivatives <b>65</b>–<b>70</b> to potentially add π-stacking interactions with r(CCUG)<sup>exp</sup>’s internal loops and thereby enhance binding interactions. The other 12 analogues were synthesized via thermal amine replacement toward the corresponding bromobenzothiazole derivative of <b>63</b> (Intermediate D, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Scheme 3</a>). This series of compounds was designed to add functional groups that could potentially interact with the RNA such as those with a positive charge (<b>71</b>, <b>74</b>, <b>79</b>, and <b>82</b>), additional hydrogen bond donors and acceptors (<b>71</b>, <b>72</b>, <b>74</b>–<b>82</b>), or those with aromatic rings to form π–π interactions (<b>75</b>, <b>76</b>, <b>77</b>, and <b>81</b>). Within these analogues, seven compounds (<b>71</b>, <b>72</b>, <b>73</b>, <b>74</b>, <b>78</b>, <b>79</b>, and <b>82</b>; <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C) had IC<sub>50</sub>s < 5 μM and were chosen for further investigation, along with <b>63</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Interestingly, four compounds (<b>71</b>–<b>74</b>) have IC<sub>50</sub>s < 500 nM, which represents a 100-fold improvement in <i>in vitro</i> potency compared to the starting compound, <b>7</b>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Evaluation of the Cellular Activity of Compounds</h3><div class="NLM_p">The eight compounds with <i>in vitro</i> IC<sub>50</sub>s < 5 μM were studied in DM2 patient-derived fibroblasts, both for their effects on cell viability and for rescue of <i>IR</i> exon 11 splicing. For the latter, we compared the percentage of exon 11 excluded upon compound treatment to untreated DM2 patient-derived fibroblasts and untreated WT fibroblasts, as measured by RT-PCR analysis. The percent rescue of a compound that restores the splicing pattern to that observed in WT cells is set to 100. Compounds <b>64</b>, <b>65</b>, <b>71</b>, <b>72</b>, <b>73</b>, <b>74</b>, and <b>82</b> displayed significant toxicity at 5 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S11</a>) and were not further evaluated. Of compounds with no toxicity at 5 μM (<b>63</b>, <b>78</b>, and <b>79</b>), <b>63</b> and <b>78</b> significantly improved <i>IR</i> splicing at 5 μM, while <b>63</b> also rescued splicing, by ∼30%, at 1 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S12</a>). Compound <b>79</b> did not significantly rescue splicing at any concentration studied (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">S12</a>). Interestingly, this derivative (<b>79</b>) is structurally similar to <b>64</b>; however, the addition of the di-<i>N</i>-methylated functional group ablated its cellular activity, which could be due to reduced cellular uptake or changes to subcellar localization. At 1 μM dose, <b>63</b> rescues <i>IR</i> splicing more potently than <b>7</b> (33 ± 5 vs 12 ± 5; <i>p</i> = 0.026), and thus <b>63</b> is the most potent compound identified in these studies for improving DM2-associated splicing defects in cells.</div><div class="NLM_p">The affinity and specificity of <b>63</b> for r(CCUG)<sup>exp</sup> were measured by microscale thermophoresis (MST). Compound <b>63</b> binds to r(CCUG)<sub>12</sub> with a <i>K</i><sub>d,app</sub> of 192 ± 20 nM, while no binding was observed to a base-paired RNA that does not contain internal loops (<i>K</i><sub>d</sub> > 20 μM) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E,F).</div><div class="NLM_p">As <b>63</b> selectively bound to r(CCUG)<sub>12</sub> over a base-paired control, its ability to improve other DM2-associated defects was assessed in DM2 patient-derived fibroblasts. Compound <b>63</b> (5 μM) reduced the number of r(CCUG)<sup>exp</sup>–MBNL1 nuclear foci per cell from 7 ± 1 in untreated patient-derived fibroblasts to 5 ± 0.5 in treated cells, or an ∼30% reduction, as determined by RNA fluorescence <i>in situ</i> hybridization (FISH) and MBNL1 immunofluorescence (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). This decrease in the number of nuclear foci correlates with the improvement in <i>IR</i> receptor splicing observed at 5 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D) as MBNL1 is no longer sequestered in foci and can resume its normal function. Importantly, <b>63</b> did not affect <i>MAP</i>4<i>K</i>4 exon 22a splicing, a NOVA-dependent splicing event (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S13</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>63</b> rescues disease-associated defects in DM2 patient-derived fibroblasts. (A) Representative RNA FISH and MBNL1 immunofluorescence images of r(CCUG)<sup>exp</sup>–MBNL1 foci. (B) Quantification of the number of nuclear foci/cell (<i>n</i> = 3, 40 nuclei counted/replicate). *** <i>P</i> < 0.001, as determined by a Student’s <i>t</i>-test. (C) Analysis of CNBP intron 1 levels upon treatment with <b>63</b> via RT-qPCR (<i>n</i> = 3).* <i>P</i> < 0.05, *** <i>P</i> < 0.001, as determined by a one-way ANOVA. (D) Analysis of CNBP mature mRNA levels upon treatment with <b>63</b> via RT-qPCR (<i>n</i> = 3). Error bars represent standard deviation (SD) for all panels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As mentioned above, r(CCUG)<sup>exp</sup> causes the aberrant retention of the intron in which it is harbored in the mature mRNA. We have previously shown that small molecules that bind r(CCUG)<sup>exp</sup> can rescue defects in intron retention by facilitating removal of the retained intron, which is subsequently degraded.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Thus, we evaluated <b>63</b>’s ability to rescue r(CCUG)<sup>exp</sup>-mediated intron retention. Indeed, <b>63</b> significantly reduced the abundance of CNBP intron 1 in DM2 patient-derived fibroblasts at doses of 5 and 1 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C), as determined by RT-qPCR. Importantly, <b>63</b> did not affect the levels of CNBP mature mRNA (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). The effects on intron 1 levels were also specific to mutant CNBP containing r(CCUG)<sup>exp</sup>, as no effects were observed in wild-type fibroblasts (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S14</a>). Furthermore, <b>63</b> did not affect <i>IR</i> exon 11 inclusion in wild-type fibroblasts (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Figure S14</a>). Thus, <b>63</b> specifically improved r(CCUG)<sup>exp</sup>-mediated defects that are dysregulated in DM2 including nuclear foci, <i>IR</i> splicing, and intron retention.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Implications for Targeted Degradation of r(CCUG)<sup>exp</sup></h3><div class="NLM_p">Importantly, all of the bioactive compounds identified herein possess druglike properties. The optimized compound, <b>63</b>, has a low molecular weight (379.48 g/mol) and has a similar number of hydrogen bond donors (n = 3) to known drugs in DrugBank (n = 5). Furthermore, <b>63</b> possesses a favorable c Log <i>P</i> (3.75) and TPSA (83 Å) and satisfies the Rule of 5 criteria for druglikeness. Through medicinal chemistry optimization of screening hits, we identified compounds with >10-fold improvement in <i>in vitro</i> activity for disruption of an r(CCUG)<sub>12</sub>–MBNL1 complex. The most bioactive compound from lead optimization showed ∼5-fold improvement in its ability to improve DM2-associated splicing defects. Importantly, this lead-optimized compound (<b>63</b>) also rescued r(CCUG)<sup>exp</sup>-containing intron 1 retention, resulting in degradation of the expanded repeat but not the mature transcript.</div><div class="NLM_p">Endogenous decay of r(CCUG)<sup>exp</sup> activated by a small molecule could occur in the nucleus or the cytoplasm. If the small molecule bound the mutant allele in the cytoplasm, any MBNL1 bound to the cytoplasmic repeat expansion would be liberated and would return to the nucleus. Since r(CCUG)<sup>exp</sup>–MBNL1 foci are not observed in the cytoplasm in the absence of compound in our imaging studies (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), such a mechanism is unlikely. Alternatively, the small molecule could bind the repeat expansion in the nucleus, freeing MBNL1, increasing intron 1’s accessibility to the splicing machinery, reducing nuclear foci, and rescuing splicing defects. The liberated intron would then be shuttled through endogenous degradation pathways. In such a mechanism, intron 1 levels would be reduced while levels of the mature CNBP transcript would be unchanged. Collectively, our data (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>) suggest that the latter, rather than the former, is the primary mode of action for <b>63</b>.</div><div class="NLM_p last">Targeted degradation is an emerging field for affecting the biological function of disease-causing RNAs. Thus far, targeted RNA degradation can be achieved through: (i) direct cleavage of RNAs through conjugation of an RNA-binding module to bleomycin or derivatives of bleomycin;<a onclick="showRef(event, 'ref13 ref26'); return false;" href="javascript:void(0);" class="ref ref13 ref26">(13,26)</a> (ii) recruitment of a ribonuclease through ribonuclease targeting chimeras (RIBOTACs), comprising an RNA-binding module and a compound that recruits RNaseL;<a onclick="showRef(event, 'ref14 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref27">(14,27)</a> and (iii) small molecules that degrade repeat-containing introns, likely by increasing the accessibility of the disease-causing RNA to endogenous decay mechanisms.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> As the former two approaches require chimeric compounds, the molecular weights of such modalities are larger and can negatively alter physicochemical properties. Targeted degradation through endogenous RNA decay mechanisms, however, can be accomplished with monomeric small molecules. Herein, we show that it is indeed possible to degrade r(CCUG)<sup>exp</sup> with druglike monomeric small molecules. Targeted degradation of RNA repeat expansions with monomeric compounds can have broad applications as other repeat expansions, such as r(G<sub>4</sub>C<sub>2</sub>)<sup>exp</sup> that causes c9ALS/FTD or r(CUG)<sup>exp</sup> that causes FECD, which are both found in retained introns.<a onclick="showRef(event, 'ref4 ref28'); return false;" href="javascript:void(0);" class="ref ref4 ref28">(4,28)</a> Furthermore, targeted degradation of RNA through RNA quality control mechanisms may be broadly applicable with monomeric, druglike small molecules that recognize RNA structures.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18736" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18736" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We identified novel compounds that bind r(CCUG)<sup>exp</sup> and improve DM2-associated defects in patient-derived cells by high-throughput screening of a druglike RNA-focused small molecule library. Lead chemical optimization of small molecule binders afforded a compound with 20-fold improvement <i>in vitro</i> activity and 5-fold improvement in cellular activity. The lead-optimized small molecule selectively improves various DM2-associated defects including retention of r(CCUG)<sup>exp</sup>-containing intron 1, which is decayed in cells upon treatment with the compound. The physicochemical properties of the compounds identified herein indicate that druglike monomeric small molecules can indeed selectively bind RNA structures. The ability to selectively degrade introns containing RNA repeat expansions with druglike small molecules will have broad implications in drug discovery efforts toward targeting RNAs.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40010" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40010" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> General Synthetic Procedures</h3><div class="NLM_p">Reagents and solvents were purchased from commercial sources and used without further purification. Microwave (MW)-assisted reactions were performed by an Initiator + (Biotage).</div><div class="NLM_p">Compounds were purified: (i) by Isolera One flash chromatography system (Biotage) using prepacked C18 columns (spherical 20–35 μm, Agela Technologies) or prepacked silica gel columns (spherical 20–35 μm, Agela Technologies) or (ii) by high-performance liquid chromatography (HPLC) (Waters 2489 and 1525) using a SunFire Prep C18 OBD 5 μm column (19 mm × 150 mm) with a 5 mL/min flow. HPLC purity analysis was performed using a SunFire C18 3.5 μm column (4.6 mm × 150 mm) with a linear gradient (0–100% methanol (MeOH) + 0.1% (v/v) trifluoracetic acid (TFA) and water + 0.1% (v/v) TFA) over 60 min at a flow rate of 1 mL/min. The purity of all derivatives was evaluated via analytical HPLC and was >95% in all cases.</div><div class="NLM_p">NMR spectra were collected on a 400 UltraShield (Bruker) (400 MHz for <sup>1</sup>H and 100 MHz for <sup>13</sup>C) or Ascend 600 (Bruker) (600 MHz for <sup>1</sup>H and 150 MHz for <sup>13</sup>C). Chemical shifts are reported in ppm relative to tetramethylsilane (TMS) for <sup>1</sup>H and residual solvent for <sup>13</sup>C as internal standards. Coupling constants (<i>J</i> values) are expressed in hertz.</div><div class="NLM_p last">High-resolution mass spectra were recorded on an Agilent 1260 Infinity LC system coupled to an Agilent 6230 TOF (HR-ESI) with a Poroshell 120 EC-C18 column (Agilent, 50 mm × 4.6 mm, 2.7 μm). Liquid chromatography–mass spectrometry (LCMS) analysis was performed by Agilent 1260 Infinity LC system coupled to an Agilent 6130 quadrupole LC/MS (ESI) with a ZORBAX SB-C18 column (Agilent, 50 mm × 2.1 mm, 1.8 μm). Compounds <b>1</b>–<b>44</b> were purchased from ChemBridge and used without further purification.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General Synthetic Procedure for <b>7</b>, <b>45</b>–<b>64</b> (General Synthetic Procedure 1)</h3><div class="NLM_p">For <b>step 1</b>, a mixture of dichloroquinazoline (<b>A</b>) (0.469 mmol, 1.0 equiv) and 3-aminopropan-1-ol (1.52 mmol, 3.2 equiv) in isopropanol (IPA) (0.40 M) was heated at 85 °C for 30 min under MW irradiation. The reaction was cooled to room temperature, and product formation was confirmed by LCMS. The reaction mixture was then concentrated <i>in vacuo</i>,  washed with H<sub>2</sub>O (4 mL × 2), and dried to afford intermediate <b>B</b>. The material was used in the next reaction without further purification. Then in <b>step 2</b>, a mixture of <b>B</b> (79.5 μmol, 1.0 equiv) and a corresponding aniline (<b>C</b>) (159 μmol, 2.0 equiv) in ethanol (EtOH) was heated at 150 °C for 30–60 min under MW irradiation. The reaction was cooled to room temperature, and product formation was confirmed by LCMS. The reaction mixture was then concentrated <i>in vacuo</i> and purified by column chromatography [Agela Technologies, Silica, 20 g, 0–30% MeOH in dichloromethane (DCM)] to afford compounds <b>45</b>–<b>64</b>.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Synthesis of <b>7</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>7</b>) (12.2 mg, 44%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.08 (d, <i>J</i> = 8.2 Hz, 1H), 7.78 (d, <i>J</i> = 7.4 Hz, 1H), 7.52 (d, <i>J</i> = 8.2 Hz, 1H), 7.45 (d, <i>J</i> = 7.4 Hz, 1H), 7.11–6.86 (m, 3H), 4.28 (s, 4H), 3.76 (t, <i>J</i> = 7.1 Hz, 2H), 3.67 (t, <i>J</i> = 6.1 Hz, 2H), and 2.00–1.90 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.4, 154.7, 145.4, 143.5, 140.0, 134.2, 133.1, 123.9, 123.3, 121.7, 117.2, 114.4, 111.4, 110.1, 64.7, 64.4, 59.0, 38.9, and 32.1; HR-MS (ESI): calcd for C<sub>19</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 351.1463; found, 351.1463.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Synthesis of <b>45</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and benzo[<i>d</i>][1,3]dioxol-5-amine (21.8 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4]oxazin-7-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>45</b>) (16.6 mg, 62%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.4 (br s, 1H), 9.89 (br s, 1H), 8.48–8.37 (m, 1H), 7.83–7.73 (m, 1H), 7.55–7.47 (m, 1H), 7.45–7.29 (m, 2H), 7.05–6.97 (m, 1H), 6.97–6.91 (m, 1H), 6.06 (s, 2H), 4.75–4.57 (m, 1H), 3.65–3.59 (m, 2H), 3.53–3.47 (m, 2H), and 1.87–1.78 (m, 2H); HR-MS (ESI): calcd for C<sub>18</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 337.1306; found, 337.1321.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Synthesis of <b>46</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 2,2-difluorobenzo[<i>d</i>][1,3]dioxol-5-amine (27.5 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,2-difluorobenzo[<i>d</i>][1,3]dioxol-5-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>46</b>) (12.4 mg, 42%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.5 (br s, 1H), 9.77 (br s, 1H), 8.40–8.33 (m, 1H), 7.91–7.75 (m, 2H), 7.60–7.56 (m, 1H), 7.51–7.41 (m, 2H), 7.37–7.31 (m, 1H), 4.79–4.43 (m, 1H), 3.69–3.58 (m, 2H), 3.57–3.46 (m, 2H), and 1.87–1.78 (m, 2H); HR-MS (ESI): calcd for C<sub>18</sub>H<sub>15</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 373.1118; found, 373.1119.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Synthesis of <b>47</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (26.3 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((5-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>47</b>) (29.0 mg, 100%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86–8.69 (m, 2H), 8.24 (d, <i>J</i> = 7.8 Hz, 1H), 7.72–7.65 (m, 1H), 7.46 (d, <i>J</i> = 7.6 Hz, 1H), 7.35–7.27 (m, 1H), 6.96 (d, <i>J</i> = 8.6 Hz, 1H), 6.75 (d, <i>J</i> = 8.6 Hz, 1H), 4.58–4.54 (m, 1H), 4.33–4.28 (m, 2H), 4.27–4.22 (m, 2H), 3.59–3.49 (m, 2H), 3.49–3.43 (m, 2H), 2.06 (s, 3H), and 1.80–1.73 (m, 2H); HR-MS (ESI): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 365.1619; found, 365.1622.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Synthesis of <b>48</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4]oxazin-7-amine (26.1 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((4-methyl-3,4-dihydro-2<i>H</i>-benzo[<i>b</i>][1,4]oxazin-7-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>48</b>) (6.3 mg, 22%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.86–7.82 (m, 1H), 7.86–7.82 (m, 1H), 7.57–7.52 (m, 1H), 7.39–7.35 (m, 1H), 7.20 (d, <i>J</i> = 2.4 Hz, 1H), 7.16–7.11 (m, 1H), 7.01 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 6.70 (d, <i>J</i> = 8.7 Hz, 1H), 4.31–4.27 (m, 1H), 3.72–3.65 (m, 4H), 3.21–3.17 (m, 2H), 2.84 (s, 3H), and 1.97–1.88 (m, 2H); HR-MS (ESI): calcd for C<sub>20</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub><sup>–</sup> [M – H]<sup>−</sup>; 364.1779; found, 364.1786.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Synthesis of <b>49</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (23.5 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2-methylbenzo[<i>d</i>]oxazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>49</b>) (8.2 mg, 30%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.29 (d, <i>J</i> = 1.8 Hz, 1H), 7.94 (dd, <i>J</i> = 8.2, 1.0 Hz, 1H), 7.68–7.62 (m, 1H), 7.54–7.45 (m, 2H), 7.43 (dd, <i>J</i> = 8.6, 2.0 Hz, 1H), 7.29–7.23 (m, 1H), 3.75 (t, <i>J</i> = 7.0 Hz, 2H), 3.69 (t, <i>J</i> = 6.2 Hz, 2H), 2.63 (s, 3H), and 2.00–1.92 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1, 160.6, 151.2, 138.6, 135.8, 133.6, 128.5, 126.0, 124.3, 123.5, 122.7, 118.8, 117.0, 112.0, 101.6, 59.1, 38.7, 32.3, and 14.6; HR-MS (ESI): calcd for C<sub>19</sub>H<sub>18</sub>N<sub>5</sub>O<sub>2</sub><sup>–</sup> [M – H]<sup>−</sup>; 348.1466; found, 348.1470.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of <b>50</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 2-methylbenzo[<i>d</i>]thiazol-6-amine (26.1 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2-methylbenzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>50</b>) (7.7 mg, 27%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.56 (d, <i>J</i> = 2.0 Hz, 1H), 7.93–7.88 (m, 1H), 7.70 (d, <i>J</i> = 8.8 Hz, 1H), 7.66–7.63 (m, 1H), 7.63–7.57 (m, 1H), 7.49–7.45 (m, 1H), 7.24–7.17 (m, 1H), 3.74 (t, <i>J</i> = 7.0 Hz, 2H), 3.70 (t, <i>J</i> = 6.2 Hz, 2H), 2.79 (s, 3H), and 2.00–1.92 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.8, 160.6, 157.4, 151.4, 147.6, 139.5, 136.3, 133.0, 125.8, 123.2, 122.0, 121.9, 118.7, 112.2, 110.1, 59.2, 38.5, 32.4, and 20.1; HR-MS (ESI): calcd for C<sub>19</sub>H<sub>18</sub>N<sub>5</sub>OS<sup>–</sup> [M – H]<sup>−</sup>; 364.1238; found, 364.1245.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Synthesis of <b>51</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 1-methyl-1<i>H</i>-indazol-6-amine (23.4 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((1-methyl-1<i>H</i>-indazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>51</b>) (25.4 mg, 92%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.31 (br s, 1H), 7.97–7.93 (m, 1H), 7.89 (d, <i>J</i> = 0.9 Hz, 1H), 7.67–7.61 (m, 2H), 7.52–7.48 (m, 1H), 7.28–7.21 (m, 2H), 3.10 (s, 3H), 3.80 (t, <i>J</i> = 6.9 Hz, 2H), 3.69 (t, <i>J</i> = 6.1 Hz, 2H), and 2.02–1.94 (m, 2H); HR-MS (ESI): calcd for C<sub>19</sub>H<sub>19</sub>N<sub>6</sub>O<sup>–</sup> [M – H]<sup>−</sup>; 347.1626; found, 347.1630.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Synthesis of <b>52</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and 1-methyl-1<i>H</i>-indazol-3-amine (23.4 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((1-methyl-1<i>H</i>-indazol-3-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>52</b>) (5.2 mg, 19%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.8 (br s, 0.5H), 11.5 (br s, 0.5H), 9.87 (br s, 1H), 8.51 (d, <i>J</i> = 8.1 Hz, 1H), 8.02 (d, <i>J</i> = 8.1 Hz, 1H), 7.90–7.82 (m, 1H), 7.79–7.72 (m, 1H), 7.68 (d, <i>J</i> = 8.6 Hz, 1H), 7.55–7.45 (m, 2H), 7.23–7.15 (m, 1H), 4.72–4.56 (m, 1H), 4.10 (3H, s), 3.75–3.56 (m, 2H), 3.52–3.39 (m, 2H), and 1.88–1.70 (m, 2H); HR-MS (ESI): calcd for C<sub>19</sub>H<sub>19</sub>N<sub>6</sub>O<sup>–</sup> [M – H]<sup>−</sup>; 347.1626; found, 347.1635.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Synthesis of <b>53</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloroquinazoline (93.3 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 105.0 mg, 94%) and <i>N</i>-methyl-2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (26.3 mg, 159 μmol) in <b>step 2</b> (18.9 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)(methyl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>53</b>) (7.5 mg, 26%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.63 (br s, 1H), 8.36 (d, <i>J</i> = 8.2 Hz, 1H), 7.75–7.69 (m, 2H), 7.40–7.34 (m, 1H), 7.04 (d, <i>J</i> = 2.2 Hz, 1H), 7.00–6.91 (m, 2H), 4.65–4.58 (m, 1H), 4.32–4.28 (m, 4H), 3.58–3.49 (m, 2H), 3.52 (s, 3H), 3.48–3.41 (m, 2H), and 1.81–1.70 (m, 2H); HR-MS (ESI): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 365.1619; found, 365.1636.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Synthesis of <b>54</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-5,6,7,8-tetrahydroquinazoline (95.2 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 86.3 mg, 76%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (19.2 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-5,6,7,8-tetrahydroquinazolin-4-yl)amino)propan-1-ol (<b>54</b>) (15.4 mg, 54%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.40 (br s, 1H), 7.60 (br s, 1H), 7.30 (d, <i>J</i> = 2.2 Hz, 1H), 7.01 (dd, <i>J</i> = 8.8, 2.2 Hz, 1H), 6.77 (d, <i>J</i> = 8.8 Hz, 1H), 4.54 (t, <i>J</i> = 4.7 Hz, 1H), 4.26–4.17 (m, 4H), 3.51–3.42 (m, 4H), 2.54–2.46 (m, 2H), 2.28–2.21 (m, 2H), and 1.79–1.66 (m, 6H); HR-MS (ESI): calcd for C<sub>19</sub>H<sub>23</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 355.1778; found, 355.1773.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Synthesis of <b>55</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-5,7-dihydrofuro[3,4-<i>d</i>]pyrimidine (89.6 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 88.8 mg, 82%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (18.2 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-5,7-dihydrofuro[3,4-<i>d</i>]pyrimidin-4-yl)amino)propan-1-ol (<b>55</b>) (7.8 mg, 29%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.86 (br s, 1H), 7.45 (d, <i>J</i> = 2.5 Hz, 1H), 7.12 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.02 (t, <i>J</i> = 5.2 Hz, 1H), 6.69 (d, <i>J</i> = 8.8 Hz, 1H), 4.80 (br s, 2H), 4.68–4.63 (m, 2H), 4.45 (t, <i>J</i> = 4.6 Hz, 1H), 4.23–4.13 (m, 4H), 3.53–3.38 (m, 4H), and 1.77–1.68 (m, 2H); HR-MS (ESI): calcd for C<sub>17</sub>H<sub>19</sub>N<sub>4</sub>O<sub>4</sub><sup>–</sup> [M – H]<sup>−</sup>; 343.1412; found, 343.1418.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Synthesis of <b>56</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 4,6-dichloro-3-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidine (95.2 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 44.0 mg, 39%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (19.2 mg, 79.5 μmol of monochloro intermediate) afforded 3-((6-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-3-methyl-1<i>H</i>-pyrazolo[3,4-<i>d</i>]pyrimidin-4-yl)amino)propan-1-ol (<b>56</b>) (12.5 mg, 44%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.4 (br s, 1H), 8.79 (br s, 1H), 7.62–7.41 (m, 1H), 7.20–7.05 (m, 1H), 6.91 (br s, 1H), 6.79–6.65 (m, 1H), 4.84–4.49 (m, 1H), 4.28–4.13 (m, 4H), 3.64–3.50 (m, 4H), 2.45 (br s, 3H), and 1.85–1.74 (m, 2H); HR-MS (ESI): calcd for C<sub>17</sub>H<sub>19</sub>N<sub>6</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 355.1524; found, 355.1537.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Synthesis of <b>57</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-8-methylquinazoline (100.0 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 104.6 mg, 89%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (20.0 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-8-methylquinazolin-4-yl)amino)propan-1-ol (<b>57</b>) (4.2 mg, 14%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.95 (d, <i>J</i> = 7.9 Hz, 1H), 7.34 (d, <i>J</i> = 7.3 Hz, 1H), 7.39–7.35 (m, 1H), 7.29–7.24 (m, 1H), 6.99 (d, <i>J</i> = 8.7, 2.6 Hz, 1H), 6.89 (d, <i>J</i> = 8.7 Hz, 1H), 4.31–4.25 (m, 4H), 3.78–3.74 (m, 2H), 3.67 (t, <i>J</i> = 6.2 Hz, 2H), 2.51 (s, 3H), and 1.99–1.91 (m, 2H); HR-MS (ESI): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 365.1619; found, 365.1628.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Synthesis of <b>58</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-6-methylquinazoline (100.0 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 103.8 mg, 88%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (20.0 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>58</b>) (9.0 mg, 31%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.90 (br s, 1H), 7.65–7.60 (m, 1H), 7.41 (d, <i>J</i> = 8.5 Hz, 1H), 7.06 (br s, 1H), 6.95–6.88 (m, 2H), 4.27 (s, 4H), 3.75 (t, <i>J</i> = 7.1 Hz, 2H), 3.67 (t, <i>J</i> = 6.2 Hz, 2H), 2.47 (s, 3H), 1.99–1.89 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.2, 152.4, 143.7, 140.9, 138.7, 136.4, 134.1, 131.3, 123.8, 118.6, 117.5, 115.6, 111.3, 110.7, 64.7, 64.5, 58.8, 39.3, 31.9, and 21.2; HR-MS (ESI): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 365.1619; found, 365.1622.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Synthesis of <b>59</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-6-fluoroquinazoline (101.8 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 93.8 mg, 78%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (20.3 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-6-fluoroquinazolin-4-yl)amino)propan-1-ol (<b>59</b>) (8.0 mg, 27%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.59 (br s, 1H), 8.92 (br s, 1H), 8.19–8.08 (m, 1H), 7.65–7.56 (m, 1H), 7.56–7.48 (m, 1H), 7.42–7.28 (m, 1H), 7.16–7.05 (m, 1H), 6.82 (d, <i>J</i> = 8.7 Hz, 1H), 4.64–4.50 (m, 1H), 4.29–4.18 (m, 4H), 3.65–3.56 (m, 2H), 3.56–3.49 (m, 2H), and 1.87–1.78 (m, 2H); HR-MS (ESI): calcd for C<sub>19</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 369.1368; found, 369.1375.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Synthesis of <b>60</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-7-fluoroquinazoline (101.8 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 46.1 mg, 38%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (20.3 mg, 79.5 μmol of monochloro intermediate) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-7-fluoroquinazolin-4-yl)amino)propan-1-ol (<b>60</b>) (3.0 mg, 10%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.21–8.12 (m, 1H), 7.31–7.19 (m, 2H), 7.11–7.00 (m, 1H), 6.98–6.87 (m, 2H), 4.28 (s, 4H), 3.75 (t, <i>J</i> = 7.1 Hz, 2H), 3.67 (t, <i>J</i> = 6.1 Hz, 2H), and 1.99–1.90 (m, 2H); HR-MS (ESI): calcd for C<sub>19</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 369.1368; found, 369.1383.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Synthesis of <b>61</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-8-fluoroquinazoline (101.8 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 76.5 mg, 64%) and 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-amine (24.0 mg, 159 μmol) in <b>step 2</b> (20.3 mg, 79.5 μmol of monochloro intermediate was used) afforded 3-((2-((2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)amino)-8-fluoroquinazolin-4-yl)amino)propan-1-ol (<b>61</b>) (11.5 mg, 39%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.21 (br s, 1H), 8.38 (br s, 1H), 7.91 (d, <i>J</i> = 8.2 Hz, 1H), 7.73–7.56 (m, 1H), 7.51–7.40 (m, 1H), 7.30–7.21 (m, 1H), 7.17–7.07 (m, 1H), 6.75 (d, <i>J</i> = 8.7 Hz, 1H), 4.64–4.47 (m, 1H), 4.30–4.15 (m, 4H), 3.66–3.48 (m, 4H), and 1.90–1.78 (m, 2H); HR-MS (ESI): calcd for C<sub>19</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>3</sub><sup>–</sup> [M – H]<sup>−</sup>; 369.1368; found, 369.1383.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Synthesis of <b>62</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-6-methylquinazoline (100.0 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 103.8 mg, 88%) and 2-methylbenzo[<i>d</i>]oxazol-6-amine (23.5 mg, 159 μmol) in <b>step 2</b> (20.0 mg, 79.5 μmol of monochloro intermediate) afforded 3-((6-methyl-2-((2-methylbenzo[<i>d</i>]oxazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>62</b>) (6.2 mg, 21%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.11 (d, <i>J</i> = 1.8 Hz, 1H), 7.84 (br s, 1H), 7.60–7.55 (m, 2H), 7.44–7.38 (m, 2H), 3.75 (t, <i>J</i> = 7.0 Hz, 2H), 3.67 (t, <i>J</i> = 6.1 Hz, 2H), 2.64 (s, 3H), 2.46 (s, 3H), and 2.00–1.92 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.8, 160.3, 154.1, 151.1, 136.9, 135.8, 133.2, 128.6, 126.0, 123.3, 121.5, 119.1, 117.9, 111.3, 102.9, 58.9, 39.0, 32.0, 21.3, and 14.6; HR-MS (ESI): calcd for C<sub>20</sub>H<sub>20</sub>N<sub>5</sub>O<sub>2</sub><sup>–</sup> [M – H]<sup>−</sup>; 362.1623; found, 362.1624.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Synthesis of <b>63</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-6-methylquinazoline (100.0 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 103.8 mg 88%) and 2-methylbenzo[<i>d</i>]thiazol-6-amine (26.1 mg, 159 μmol) in <b>step 2</b> (20.0 mg, 79.5 μmol of monochloro intermediate) afforded 3-((6-methyl-2-((2-methylbenzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>63</b>) (15.4 mg, 56%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.31 (d, <i>J</i> = 2.0 Hz, 1H), 7.91–7.85 (m, 2H), 7.62–7.58 (m, 2H), 7.41 (d, <i>J</i> = 8.5 Hz, 1H), 3.76 (t, <i>J</i> = 7.1 Hz, 2H), 3.66 (t, <i>J</i> = 6.1 Hz, 2H), 2.83 (s, 3H), 2.47 (s, 3H), and 1.98–1.90 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.7, 160.3, 155.4, 148.5, 145.9, 138.0, 136.2, 135.2, 132.2, 123.5, 122.8, 122.0, 119.6, 111.6, 59.1, 38.8, 32.3, 21.3, and 20.1; HR-MS (ESI): calcd for C<sub>20</sub>H<sub>22</sub>N<sub>5</sub>OS<sup>–</sup> [M – H]<sup>−</sup>; 378.1394; found, 378.1403.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Synthesis of <b>64</b></h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-6-methylquinazoline (100.0 mg, 0.469 mmol) in <b>step 1</b> (obtained monochloro intermediate; 103.8 mg, 88%) and 1-methyl-1<i>H</i>-indazol-6-amine (23.4 mg, 159 μmol) in <b>step 2</b> (20.0 mg, 79.5 μmol of monochloro intermediate) afforded 3-((6-methyl-2-((1-methyl-1<i>H</i>-indazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>64</b>) (7.9 mg, 27%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.27 (br s, 1H), 7.89 (d, <i>J</i> = 0.9 Hz, 1H), 7.78 (br s, 1H), 7.67–7.63 (m, 1H), 7.53–7.48 (m, 1H), 7.41 (d, <i>J</i> = 8.5 Hz, 1H), 7.24–7.20 (m, 1H), 4.04 (s, 3H), 3.79 (t, <i>J</i> = 6.9 Hz, 2H), 3.69 (t, <i>J</i> = 6.1 Hz, 2H), 2.45 (s, 3H), and 2.02–1.93 (m, 2H); HR-MS (ESI): calcd for C<sub>20</sub>H<sub>21</sub>N<sub>6</sub>O<sup>–</sup> [M – H]<sup>−</sup>; 361.1782; found, 361.1778.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Synthesis of intermediate D</h4><div class="NLM_p last">Following general synthetic procedure 1 using 2,4-dichloro-6-methylquinazoline (1.50 g, 7.04 mmol) in step 1 (obtained monochloro intermediate; 1.56 g, 88%) and 2-bromobenzo[<i>d</i>]thiazol-6-amine (500 mg, 2.18 mmol) in <b>step 2</b> (1.10 g, 4.37 mmol of monochloro intermediate) afforded 3-((2-((2-bromobenzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>D</b>) (1.00 g, quant). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.32–8.25 (m, 1H), 8.02–7.94 (m, 2H), 7.69 (dd, <i>J</i> = 8.5, 1.5 Hz, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.2 Hz, 1H), 7.44 (d, <i>J</i> = 8.5 Hz, 1H), 3.80–3.74 (m, 2H), 3.65 (t, <i>J</i> = 6.1 Hz, 2H), 2.49 (s, 3H), and 1.99–1.89 (m, 2H); HR-MS (ESI): calcd for C<sub>19</sub>H<sub>18</sub>BrN<sub>5</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 444.0488; found, 444.04777.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> General Synthetic Procedure for 65–70 (General Synthetic Procedure 2)</h3><div class="NLM_p">A mixture of 3-((2-((2-bromobenzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>D</b>) (20.0 mg, 45.0 μmol, 1.0 equiv), Ar-boronic acid (67.5 μmol, 1.5 equiv), Pd(PPh<sub>3</sub>)<sub>4</sub> (5.2 mg, 4.5 μmol, 0.1 equiv), and K<sub>3</sub>PO<sub>4</sub> (19.1 mg, 90.0 μmol, 2.0 equiv) in dioxane/H<sub>2</sub>O (2/1, 0.50 mL) was heated at 120 °C overnight. The reaction was cooled to room temperature, and product formation was confirmed by LCMS. The reaction mixture was then concentrated <i>in vacuo</i> and purified by column chromatography as described above (Agela Technologies, Silica, 20 g, 0–20% MeOH in DCM) and HPLC to afford compounds <b>65</b>–<b>70</b>.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Synthesis of <b>65</b></h4><div class="NLM_p last">Following general synthetic procedure 2 using <b>D</b> (20.0 mg, 45.0 μmol) and (4-fluorophenyl)boronic acid (9.4 mg, 67.5 μmol) afforded 3-((2-((2-(4-fluorophenyl)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>65</b>) (5.0 mg, 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.6 (br s, 1H), 10.6 (br s, 1H), 9.55 (br s, 1H), 8.51 (br s, 1H), 8.19–8.06 (m, 4H), 7.73–7.65 (m, 2H), 7.54–7.40 (m, 3H), 4.79–4.49 (m, 1H), 3.72–3.63 (m, 2H), 3.57–3.49 (m, 2H), 2.43 (s, 3H), and 1.90–1.80 (m, 2H); HR-MS (ESI): calcd for C<sub>25</sub>H<sub>23</sub>FN<sub>5</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 460.1602; found, 460.1613.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Synthesis of <b>66</b></h4><div class="NLM_p last">Following general synthetic procedure 2 using <b>D</b> (20.0 mg, 45.0 μmol) and (3-fluorophenyl)boronic acid (9.4 mg, 67.5 μmol) afforded 3-((2-((2-(3-fluorophenyl)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>66</b>) (5.0 mg, 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.7 (br s, 1H), 10.6 (br s, 1H), 9.58 (br s, 1H), 8.57 (br s, 1H), 8.15–8.07 (m, 2H), 7.96–7.87 (m, 2H), 7.75–7.61 (m, 3H), 7.53–7.42 (m, 2H), 4.77–4.56 (m, 1H), 3.72–3.63 (m, 2H), 3.57–3.49 (m, 2H), 2.43 (s, 3H), and 1.90–1.80 (m, 2H); HR-MS (ESI): calcd for C<sub>25</sub>H<sub>23</sub>FN<sub>5</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 460.1602; found, 460.1621.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Synthesis of <b>67</b></h4><div class="NLM_p last">Following general synthetic procedure 2 using <b>D</b> (20.0 mg, 45.0 μmol) and <i>o</i>-tolylboronic acid (9.2 mg, 67.5 μmol) afforded 3-((6-methyl-2-((2-(<i>o</i>-tolyl)benzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>67</b>) (9.4 mg, 46%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.6 (br s, 1H), 10.6 (br s, 1H), 9.60 (br s, 1H), 8.53 (br s, 1H), 8.16–8.07 (m, 2H), 7.83 (d, <i>J</i> = 7.3 Hz, 1H), 7.75–7.63 (m, 2H), 7.53–7.38 (m, 4H), 4.75–4.52 (m, 1H), 3.73–3.63 (m, 2H), 3.58–3.50 (m, 2H), 2.65 (s, 3H), 2.43 (s, 3H), and 1.91–1.80 (m, 2H); HR-MS (ESI): calcd for C<sub>26</sub>H<sub>26</sub>N<sub>5</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 456.1853; found, 456.1872.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Synthesis of <b>68</b></h4><div class="NLM_p last">Following general synthetic procedure 2 using <b>D</b> (20.0 mg, 45.0 μmol) and (1H-pyrazol-3-yl)boronic acid (7.6 mg, 67.5 μmol) afforded 3-((2-((2-(1H-pyrazol-3-yl)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>68</b>) (18.5 mg, 95%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.4 (s, 1H), 12.5 (br s, 1H), 10.6 (br s, 1H), 9.66 (br s, 1H), 8.42 (br s, 1H), 8.19–7.93 (m, 3H), 7.71–7.60 (m, 2H), 7.54–7.46 (m, 1H), 6.93 (s, 1H), 4.73–4.57 (m, 1H), 3.72–3.62 (m, 2H), 3.57–3.49 (m, 2H), 2.43 (s, 3H), and 1.89–1.82 (m, 2H); HR-MS (ESI): calcd for C<sub>22</sub>H<sub>22</sub>N<sub>7</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 432.1601; found, 432.1613.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Synthesis of <b>69</b></h4><div class="NLM_p last">Following general synthetic procedure 2 using <b>D</b> (20.0 mg, 45.0 μmol) and (4-carbamoylphenyl)boronic acid (11.1 mg, 67.5 μmol) afforded 4-(6-((4-((3-hydroxypropyl)amino)-6-methylquinazolin-2-yl)amino)benzo[<i>d</i>]thiazol-2-yl)benzamide (<b>69</b>) (12.5 mg, 57%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.9 (br s, 1H), 10.7 (br s, 1H), 9.56 (br s, 1H), 8.59 (br s, 1H), 8.22–8.02 (m, 6H), 7.76–7.64 (m, 2H), 7.57 (s, 1H), 7.49 (d, <i>J</i> = 8.4 Hz, 1H), 6.93 (s, 1H), 4.78–4.52 (m, 1H), 3.76–3.62 (m, 2H), 3.60–3.51 (m, 2H), 2.43 (s, 3H), and 1.91–1.81 (m, 2H); HR-MS (ESI): calcd for C<sub>26</sub>H<sub>25</sub>N<sub>6</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 485.1754; found, 485.1778.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Synthesis of <b>70</b></h4><div class="NLM_p last">Following general synthetic procedure 2 using <b>D</b> (20.0 mg, 45.0 μmol) and (1H-indol-5-yl)boronic acid (10.9 mg, 67.5 μmol) afforded 3-((2-((2-(1H-indol-5-yl)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>70</b>) (19.6 mg, 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.6 (br s, 1H), 11.5 (s, 1H), 10.6 (br s, 1H), 9.60 (br s, 1H), 8.43 (br s, 1H), 8.32 (d, <i>J</i> = 1.7 Hz, 1H), 8.13 (s, 1H), 8.03 (d, <i>J</i> = 8.7 Hz, 1H), 7.87 (dd, <i>J</i> = 8.5, 1.8 Hz, 1H), 7.71–7.61 (m, 2H), 7.59–7.48 (m, 3H), 6.64–6.61 (m, 1H), 4.83–4.44 (m, 1H), 3.72–3.63 (m, 2H), 3.57–3.51 (m, 2H), 2.43 (s, 3H), and 1.90–1.81 (m, 2H); HR-MS (ESI): calcd for C<sub>27</sub>H<sub>28</sub>N<sub>6</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 481.1805; found, 481.1829.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> General Synthetic Procedure for <b>71</b>–<b>82</b> (General Synthetic Procedure 3)</h3><div class="NLM_p">A mixture of 3-((6-bromo-2-((2-methylbenzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>D</b>) (15.0 mg, 33.8 μmol, 1.0 equiv), <i>N,N</i>-diisopropylethylamine (DIPEA; 35.4 μL, 203 μmol, 6.0 equiv) and amine (203 μmol, 6.0 equiv) in 3-pentanol (0.10 M) was heated in a microwave vial at 120 °C overnight. The reaction was cooled to room temperature, and product formation was confirmed by LCMS. The reaction mixture was then concentrated <i>in vacuo</i> and purified by column chromatography (Biotage SNAP cartridge, KP-NH, 11g, 2%–30% MeOH in DCM) to afford compounds <b>71</b>–<b>82</b>.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Synthesis of <b>71</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 1-benzylpiperidin-4-amine (38.6 mg, 203 μmol) afforded 3-((2-((2-((1-benzylpiperidin-4-yl)amino)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>71</b>) (13.8 mg, 74%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.11 (d, <i>J</i> = 1.9 Hz, 1H), 7.71–7.67 (m, 1H), 7.46–7.25 (m, 9H), 3.81–3.64 (m, 5H), 3.56 (s, 2H), 2.97–2.86 (m, 2H), 2.42 (s, 3H), 2.28–2.19 (m, 2H), 2.13–2.04 (m, 2H), 1.96–1.89 (m, 2H), and 1.69–1.55 (m, 2H); HR-MS (ESI): calcd for C<sub>31</sub>H<sub>36</sub>N<sub>7</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 554.2697; found, 554.2722.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Synthesis of <b>72</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and morpholine (17.7 mg, 203 μmol) afforded 3-((6-methyl-2-((2-morpholinobenzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>72</b>) (7.3 mg, 48%).<sup>1</sup>H NMR (400 MHz, MeOD) δ 8.26 (d, <i>J</i> = 1.6 Hz, 1H), 7.71–7.67 (m, 1H), 7.48–7.40 (m, 3H), 7.33 (d, <i>J</i> = 8.5 Hz, 1H), 3.84–3.79 (m, 4H), 3.74–3.64 (m, 4H), 3.60–3.54 (m, 4H), 2.42 (s, 3H), and 2.00–1.89 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 160.2, 157.0, 149.7, 146.5, 136.9, 134.5, 131.0, 130.7, 125.6, 122.4, 118.9, 118.3, 111.9, 110.6, 66.0 (2C), 59.2, 48.7 (2C), 38.4, 32.4, and 21.3; HR-MS (ESI): calcd for C<sub>23</sub>H<sub>27</sub>N<sub>6</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 451.1911; found, 451.1929.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Synthesis of <b>73</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 2-methylpropan-1-amine (14.8 mg, 203 μmol) afforded 3-((2-((2-(isobutylamino)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>73</b>) (11.6 mg, 79%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.11 (d, <i>J</i> = 2.0 Hz, 1H), 7.70–7.67 (m, 1H), 7.45–7.38 (m, 2H), 7.35–7.31 (m, 2H), 3.74–3.65 (m, 4H), 3.23 (d, <i>J</i> = 7.0 Hz, 2H), 2.42 (s, 3H), 2.03–1.89 (m, 3H), 1.02 (s, 3H), and 1.00 (s, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.0, 160.2, 157.1, 149.8, 147.1, 136.2, 134.4, 130.7, 130.6, 125.6, 122.4, 117.9, 117.7, 111.8, 110.8, 59.3, 52.1, 38.4, 32.5, 28.2, 21.3, and 20.6 (2C); HR-MS (ESI): calcd for C<sub>23</sub>H<sub>27</sub>N<sub>6</sub>OS<sup>–</sup> [M – H]<sup>−</sup>; 435.1973; found, 435.1989.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Synthesis of <b>74</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 2-morpholinoethan-1-amine (26.4 mg, 203 μmol) afforded 3-((6-methyl-2-((2-((2-morpholinoethyl)amino)benzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>74</b>) (12.6 mg, 75%).<sup>1</sup>H NMR (400 MHz, MeOD) δ 8.14–8.09 (m, 1H), 7.68 (br s, 1H), 7.45–7.29 (m, 4H), 3.77–3.63 (m, 8H), 3.57 (d, <i>J</i> = 6.4 Hz, 2H), 2.66 (d, <i>J</i> = 6.4 Hz, 2H), 2.60–2.49 (m, 4H), 2.41(s, 3H), and 1.98–1.88 (m, 2H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.8, 160.2, 157.1, 149.8, 147.0, 136.3, 134.4, 130.8, 130.6, 125.6, 122.4, 118.0, 117.7, 111.9, 110.8, 66.6(2C), 59.3, 57.6, 53.8(2C), 41.5, 38.4, 32.5, 21.3; HR-MS (ESI): calcd for C<sub>25</sub>H<sub>32</sub>N<sub>7</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 494.2333; found, 494.2357.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Synthesis of <b>75</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and benzo[<i>d</i>][1,3]dioxol-5-ylmethanamine (29.1 mg, 203 μmol) afforded 3-((2-((2-((benzo[<i>d</i>][1,3]dioxol-5-ylmethyl)amino)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>75</b>) (8.9 mg, 51%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.89 (s, 1H), 7.83 (d, <i>J</i> = 2.0 Hz, 1H), 7.60 (dd, <i>J</i> = 8.5, 1.4 Hz, 1H), 7.48 (d, <i>J</i> = 8.5 Hz, 1H), 7.40 (d, <i>J</i> = 8.5 Hz, 1H), 7.37–7.32 (m, 1H), 6.91–6.86 (m, 2H), 6.79 (d, <i>J</i> = 11.8 Hz, 1H), 5.93 (s, 2H), 4.54 (s, 2H), 3.74 (t, <i>J</i> = 7.1 Hz, 2H), 3.65 (t, <i>J</i> = 6.1 Hz, 2H), 2.46 (s, 3H), 1.98–1.88 (m, 2H); HR-MS (ESI): calcd for C<sub>27</sub>H<sub>27</sub>N<sub>6</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 515.1860; found, 515.1883.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Synthesis of <b>76</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 2-phenoxyethan-1-amine (27.8 mg, 203 μmol) afforded 3-((6-methyl-2-((2-((2-phenoxyethyl)amino)benzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>76</b>) (6.3 mg, 37%).<sup>1</sup>H NMR (400 MHz, MeOD) δ 7.95–7.89 (m, 2H), 7.68–7.63 (m, 1H), 7.56–7.51 (m, 1H), 7.48–7.41 (m, 2H), 7.31–7.24 (m, 2H), 6.99–6.90 (m, 3H), 4.25 (t, <i>J</i> = 5.2 Hz, 2H), 3.90 (t, <i>J</i> = 5.2 Hz, 2H), 3.76 (t, <i>J</i> = 7.1 Hz, 2H), 3.65 (t, <i>J</i> = 6.1 Hz, 2H), 2.47 (s, 3H), and 1.98–1.88 (m, 2H); HR-MS (ESI): calcd for C<sub>27</sub>H<sub>27</sub>N<sub>6</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 515.1860; found, 515.1883.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Synthesis of <b>77</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 2-(benzyloxy)ethan-1-amine (30.7 mg, 203 μmol) afforded 3-((2-((2-((2-(benzyloxy)ethyl)amino)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>77</b>) (7.7 mg, 44%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.98–7.91 (m, 2H), 7.68–7.63 (m, 1H), 7.52–7.46 (m, 2H), 7.44 (d, <i>J</i> = 8.5 Hz, 1H), 7.36–7.21 (m, 5H), 4.59 (s, 2H), 3.79–3.70 (m, 6H), 3.65 (t, <i>J</i> = 6.1 Hz, 2H), 2.48 (s, 3H), and 1.98–1.88 (m, 2H); HR-MS (ESI): calcd for C<sub>28</sub>H<sub>31</sub>N<sub>6</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 515.2224; found, 515.2246.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Synthesis of <b>78</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 1-amino-2-methylpropan-2-ol (18.1 mg, 203 μmol) afforded 3-((2-((2-((2-hydroxy-2-methylpropyl)amino)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>78</b>) (14.0 mg, 91%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.12 (d, <i>J</i> = 2.1 Hz, 1H), 7.68 (br s, 1H), 7.44–7.37 (m, 2H), 7.35–7.30 (m, 2H), 3.73–3.64 (m, 4H), 3.44 (s, 2H), 2.41 (s, 3H), 1.98–1.87 (m, 2H), and 1.27 (s, 6H); HR-MS (ESI): calcd for C<sub>23</sub>H<sub>29</sub>N<sub>6</sub>O<sub>2</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 453.2067; found, 453.2069.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Synthesis of <b>79</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and N1,N1-dimethylethane-1,2-diamine (17.9 mg, 203 μmol) afforded 3-((2-((2-((2-(dimethylamino)ethyl)amino)benzo[<i>d</i>]thiazol-6-yl)amino)-6-methylquinazolin-4-yl)amino)propan-1-ol (<b>79</b>) (12.9 mg, 84%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.14 (d, <i>J</i> = 2.0 Hz, 1H), 7.68 (br s, 1H), 7.44–7.38 (m, 2H), 7.38–7.30 (m, 2H), 3.73–3.65 (m, 4H), 3.56 (t, <i>J</i> = 6.7 Hz, 2H), 2.64 (t, <i>J</i> = 6.7 Hz, 2H), 2.42 (s, 3H), 2.32 (s, 6H), and 1.98–1.89 (m, 2H); HR-MS (ESI): calcd for C<sub>23</sub>H<sub>30</sub>N<sub>7</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 452.2227; found, 452.2242.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Synthesis of <b>80</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 2,2′-azanediylbis(ethan-1-ol) (21.3 mg, 203 μmol) afforded 2,2′-((6-((4-((3-hydroxypropyl)amino)-6-methylquinazolin-2-yl)amino)benzo[<i>d</i>]thiazol-2-yl)azanediyl)bis(ethan-1-ol) (<b>80</b>) (5.2 mg, 33%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.10–8.02 (m, 1H), 7.95–7.90 (m, 1H), 7.65 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.56–7.50 (m, 2H), 7.44 (d, <i>J</i> = 8.5 Hz, 1H), 3.95–3.88 (m, 4H), 3.88–3.82 (m, 4H), 3.76 (t, <i>J</i> = 7.2 Hz, 2H), 3.66 (t, <i>J</i> = 6.1 Hz, 2H), 2.47 (s, 3H), and 1.98–1.88 (m, 2H); HR-MS (ESI): calcd for C<sub>23</sub>H<sub>29</sub>N<sub>6</sub>O<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup>; 469.2016; found, 469.2031.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Synthesis of <b>81</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 4-(2-aminoethyl)benzenesulfonamide (40.7 mg, 203 μmol) afforded 4-(2-((6-((4-((3-hydroxypropyl)amino)-6-methylquinazolin-2-yl)amino)benzo[<i>d</i>]thiazol-2-yl)amino)ethyl)benzenesulfonamide (<b>81</b>) (7.5 mg, 39%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.94–7.91 (m, 1H), 7.91–7.82 (m, 3H), 7.67 (dd, <i>J</i> = 8.5, 1.4 Hz, 1H), 7.53–7.40 (m, 5H), 3.82–3.71 (m, 4H), 3.65 (t, <i>J</i> = 6.1 Hz, 2H), 3.11 (t, <i>J</i> = 7.0 Hz, 2H), 2.48 (s, 3H), and 1.99–1.89 (m, 2H); HR-MS (ESI): calcd for C<sub>27</sub>H<sub>30</sub>N<sub>7</sub>O<sub>3</sub>S<sub>2</sub><sup>+</sup> [M + H]<sup>+</sup>; 564.1846; found, 564.1856.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Synthesis of <b>82</b></h4><div class="NLM_p last">Following general synthetic procedure 3 using <b>D</b> (15.0 mg, 33.8 μmol) and 1-methylpiperazine (20.3 mg, 203 μmol) afforded 3-((6-methyl-2-((2-(4-methylpiperazin-1-yl)benzo[<i>d</i>]thiazol-6-yl)amino)quinazolin-4-yl)amino)propan-1-ol (<b>82</b>) (2.9 mg, 18%). <sup>1</sup>H NMR (400 MHz, MeOD) δ 8.25 (d, <i>J</i> = 1.7 Hz, 1H), 7.69 (br s, 1H), 7.47–7.39 (m, 3H), 7.37–7.30 (m, 1H), 3.75–3.58 (m, 8H), 2.65 (t, <i>J</i> = 10.2 Hz, 4H), 2.42 (s, 3H), 2.36 (s, 3H), and 1.98–1.88 (m, 2H); HR-MS (ESI): calcd for C<sub>24</sub>H<sub>30</sub>N<sub>7</sub>OS<sup>+</sup> [M + H]<sup>+</sup>; 464.2227; found, 464.2235.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> High-Throughput Screen of the RNA-Focused Small-Molecule Library</h3><div class="NLM_p last">Measurements for disruption of the r(CCUG)<sub>12</sub>–MBNL1 complex were completed using a previously reported TR-FRET assay with minor modifications.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> Briefly, 5′-biotinylated r(CCUG)<sub>12</sub> was folded in 1 × Folding Buffer (20 mM HEPES, pH 7.5, 110 mM KCl, and 10 mM NaCl) at 60 °C for 5 min then cooled to room temperature. The buffer was adjusted to 1 × Assay Buffer (20 mM HEPES, pH 7.5, 110 mM KCl, 10 mM NaCl, 2 mM MgCl<sub>2</sub>, 2 mM CaCl<sub>2</sub>, 5 mM DTT, 0.1% BSA, and 0.5% Tween-20). Then, MBNL1-His<sub>6</sub> was added, and the samples were incubated at room temperature for 15 min and added to a white 384-well plate using an Aurora FRD-IB reagent dispenser. Compounds from the RNA-focused small molecule library were then delivered to each well using a Beckman Biomek NXP liquid handler with a pin tool, and the samples were incubated for another 15 min at room temperature. The final concentration of r(CCUG)<sub>12</sub> was 80 nM, while the final concentration of MBNL1 was 60 nM. A solution of streptavidin-XL665 and anti-His<sub>6</sub>-Tb antibody was then added to a total volume of 10 μL using an Aurora FRD-IB reagent dispenser, with final concentrations of 40 nM and 0.44 ng/μL, respectively. The sample plate was incubated for 30 min at room temperature and TR-FRET was measured on a Molecular Devices SpectraMax M5 plate reader using an excitation wavelength of 345 and a 420 nm cutoff. The ratios of fluorescence intensity at 545 and 665 nm were calculated, and ratios in the absence of a compound and in the absence of RNA were used to calculate percent disruption.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <i>In Vitro</i> IC<sub>50</sub> Measurements</h3><div class="NLM_p last">The IC<sub>50</sub>s for disruption of r(CCUG)<sub>12</sub>–MBNL1 by hit compounds from the RNA-focused small-molecule library were completed as described above except compounds were added manually to wells at varying concentrations to afford a dose response. The resulting curves were fit to <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a> to determine IC<sub>50</sub> values<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_m001.gif" alt="" id="_i57" /></img><span class="labelSpan">(1)</span></span>where <i>y</i> is the ratio of fluorescence intensities at 545 and 665 nm (F545/F665), <i>x</i> is the concentration of a compound, <i>B</i> is F545/F665 value at max FRET effect (solution has RNA and protein but no compound added); <i>A</i> is F545/F665 value at min FRET effect (solution has antibodies but no RNA, protein, or compound); and IC<sub>50</sub> is the concentration of a compound where half of the protein is displaced by a compound.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> NMR Sample Preparation</h3><div class="NLM_p last">A self-complementary RNA construct, r(5′-GACAGC<u class="uu">CU</u>GCUGUC-3′)<sub>,</sub> was purchased from GE Healthcare Dharmacon, Inc., deprotected according to the manufacturer’s recommended protocol, and desalted with PD-10 columns (GE Healthcare, cat: 17-0851-01) also per the manufacturer’s protocol. RNA samples were dissolved in NMR Buffer [10 mM KH<sub>2</sub>PO<sub>4</sub>/K<sub>2</sub>HPO<sub>4</sub> and 0.05 mM ethylenediamine tetraacetic acid (EDTA) (pH 6.0)] and folded by heating to 95 °C for 3 min and slowly cooling to room temperature.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> NMR Spectroscopy</h3><div class="NLM_p last">NMR spectra were acquired at 25 °C on Bruker Avance III 600 and 700 MHz spectrometers equipped with cryoprobes. One-dimensional NMR spectra were acquired on samples containing 100 μM of RNA alone in 100% D<sub>2</sub>O. Compounds were each titrated into separate samples at 0.5, 1.0, 1.5, and 2.0 compound: RNA molar ratios. WaterLOGSY (water-ligand observed via gradient spectroscopy) spectra<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> were acquired on samples containing 300 μM of each compound alone or in the presence of 15 μM of RNA at a 20:1 compound: RNA ratio in H<sub>2</sub>O, to which D<sub>2</sub>O was added to 5% by volume. WaterLOGSY spectra were phased to give negative NOEs for nonbinders. Two-dimensional NOESY and DQF-COSY spectra were acquired on samples containing 400 μM of RNA alone in 100% D<sub>2</sub>O. Two-dimensional NMR spectra were processed with nmrPipe<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and assigned with SPARKY.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Cell Lines and Cell Culture</h3><div class="NLM_p last">Compounds were tested in two cell lines: (i) DM2 patient-derived fibroblasts (generous gift from University of Florida, Center for NeuroGenetics) and (ii) fibroblasts from a healthy donor (wild-type; WT GM07492; Coriell Institute). Cells were maintained at 37 °C with 5% CO<sub>2</sub>. DM2 fibroblasts were cultured in Dulbecco’s modified Eagle’s medium (DMEM)/high glucose (HyClone) supplemented with 20% (v/v) fetal bovine serum (FBS) (Sigma) and 1% (v/v) antibiotic–antimycotic solution (Corning). Wild-type fibroblasts were cultured in MEM (Corning) supplemented with 10% (v/v) FBS, 1% (v/v) Glutagro (Corning), and 1% (v/v) antibiotic–antimycotic solution.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Evaluation of Cell Viability</h3><div class="NLM_p last">Compound toxicity in DM2 fibroblasts was evaluated using a CellTiter-Glo Kit (Promega) as described in the manufacturer’s protocol. Briefly, after 48 h treatment in 96-well plates, 100 μL of the CellTiter-Glo reagent was added to each well. The plate was then incubated at room temperature for 10 min, and luminescence was measured using BioTek FLX-800 luminescence plate reader (<i>n</i> = 6 replicates; 1 independent experiment).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Evaluation of Pre-mRNA Splicing via RT-PCR</h3><div class="NLM_p last">Pre-mRNA splicing was analyzed as previously described.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Briefly, cells were grown in 6-well plates and treated with the desired compound in growth medium at ∼40% confluency. After 48 h, the cells were lysed, and total RNA was harvested using a Zymo Quick RNA Miniprep Kit. Approximately 1 μg of total RNA was reverse transcribed using a qScript cDNA synthesis kit (20 μL of total reaction volume, Quanta BioSciences); 2 μL of the RT reaction was used for PCR using GoTaq DNA polymerase (Promega). RT-PCR products were observed after 35 cycles of 95 °C for 30 s, 58 °C for 30 s, 72 °C for 1 min, and a final extension at 72 °C for 5 min. Products were separated on a 2% agarose gel (110 V for 1 h in 1 × TBE buffer), visualized by staining with ethidium bromide, and imaged using a Typhoon 9410 variable mode imager. Gels were quantified using ImageJ. Percent rescue was calculated by dividing the difference between treated and untreated DM2 samples by the difference between untreated DM2 and WT samples (<a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>).<span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_m002.gif" alt="" id="_i63" /></img><span class="labelSpan">(2)</span></span></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Evaluation of CNBP Abundance via RT-qPCR</h3><div class="NLM_p last">CNBP abundance was evaluated as previously described.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Briefly, cells were grown in 6-well plates and treated with the desired compound in growth medium at 40% confluency. After 48 h, the cells were lysed, and total RNA was harvested using a Zymo Quick RNA Miniprep Kit per the manufacturer’s protocol. Approximately 1 μg of total RNA was reverse transcribed using a qScript cDNA synthesis kit (20 μL of total reaction volume, Quanta BioSciences); 2 μL of the reverse transcription (RT) reaction was used for each primer pair (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">Table S2</a>) for quantitative (q)PCR with SYBR Green Master Mix, performed on a QuantStudio 5, 384-well Block Real-Time PCR System (Applied Biosciences). Relative abundance was determined by normalizing to <i>GAPDH</i> (<i>n</i> = 6 replicates per concentration; 2 independent experiments).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Evaluation of Nuclear Foci via RNA Fluorescence <i>In Situ</i> Hybridization (FISH)</h3><div class="NLM_p last">RNA FISH was used to determine the small molecules’ effects on the number of nuclear foci as previously described.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The number of foci was counted in 40 nuclei/replicate (120 total nuclei counted); <i>n</i> = 3 replicates; 1 independent experiment.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Affinity Measurements</h3><div class="NLM_p last">Binding affinity measurements were performed via microscale thermophoresis (MST) on a Monolith NT.115 system (NanoTemper Technologies) with Cy5-labeled (CCUG)<sub>12</sub> (5′-Cy5-GCG(CCUG)<sub>12</sub>CGC; Dharmacon) and Cy5-labeled base pair control (BP; 5′-Cy5-GCG(CCUG)<sub>7</sub>(GCAG)<sub>5</sub>CGC; Dharmacon), which were deprotected according to the manufacturer’s protocol and then desalted using a PD-10 column (GE LifeSciences) per the manufacturer’s recommended procedure. RNA (5 nM) was prepared in 8 mM Na<sub>2</sub>HPO<sub>4</sub>, pH 7.0, 185 mM NaCl, and 1 mM EDTA and folded by heating at 60 °C for 5 min and slowly cooling to room temperature. Compound dilutions (1:1) were prepared using 1 × MST Buffer (8 mM Na<sub>2</sub>HPO<sub>4</sub>, pH 7.0, 185 mM NaCl, 1 mM EDTA) and 0.1% (v/v) Tween-20 was added. After cooling, the RNA solution was added to the compound solution (0.05% (v/v) Tween-20 final concentration). Samples were incubated for 30 min at room temperature and then loaded into standard capillaries (NanoTemper Technologies). The following parameters were used to acquire thermophoretic data: 5–20% LED, 80% MST power, laser-on time = 30 s, laser-off time = 5 s. Fluorescence was detected using excitation wavelengths of 605–645 nm and emission wavelengths of 680–685 nm. The resulting data were analyzed by thermophoresis analysis and fitted using a quadratic binding equation in the MST analysis software (NanoTemper Technologies). Dissociation constants were then determined using <a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>. The reported <i>K</i><sub>d</sub> values are an average of two independent sets of experiments.<span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_m003.gif" alt="" id="_i67" /></img><span class="labelSpan">(3)</span></span>where <i>c</i> is the ligand concentration, <i>c</i><sub>T</sub> is the concentration of the RNA, and <i>K</i><sub>d</sub> is the dissociation constant.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> SMILES and Physicochemical Properties</h3><div class="NLM_p last">Marvin 20.8.0 (ChemAxon; <a href="https://www.chemaxon.com" class="extLink">https://www.chemaxon.com</a>) was used to generate SMILES and to calculate physicochemical properties.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00414" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00414?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00414</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Approximate IC<sub>50</sub>s for the disruption of the r(CCUG)<sup>exp</sup>–MBNL1 complex (Table S1); Sequences of primers (Table S2); Results of a high-throughput screen (Figure S1); Structures of compounds <b>1</b>–<b>44</b> (Figure S2); WaterLOGSY NMR analysis of <b>3</b> (Figure S3); WaterLOGSY NMR analysis of <b>4</b> (Figure S4); WaterLOGSY NMR analysis of <b>5</b> (Figure S5); WaterLOGSY NMR analysis of <b>7</b> (Figure S6); Two-dimensional NOESY NMR analysis (Figure S7); One-dimensional NMR analysis of compounds (Figure S8); Cell viability of DM2 fibroblasts <b>t</b>reated with <b>3</b>, <b>4</b>, <b>5</b>, and <b>7</b> (Figure S9); Structures of compounds <b>45</b>–<b>82</b> (Figure S10); Cell viability of DM2 fibroblasts treated with lead-optimized compounds (Figure S11); Cellular activity of compounds <b>63</b>, <b>78</b>, and <b>79</b> (Figure S12); Compound <b>63</b> does not affect MAP4K4 alternative splicing (Figure S13); Compound <b>63</b> has no effect on CNBP levels or IR exon 11 splicing (Figure S14); synthetic schemes, compound characterization; HPLC traces; and NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">All DM2 hit compounds (SMILES + DATA) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf">jm1c00414_si_001.pdf (5.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_002.csv">jm1c00414_si_002.csv (8.13 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00414" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Disney</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-8486-1796" title="Orcid link">https://orcid.org/0000-0001-8486-1796</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f09499839e9589b083938299808083de959485"><span class="__cf_email__" data-cfemail="b3d7dac0ddd6caf3c0d0c1dac3c3c09dd6d7c6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah Wagner-Griffin</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masahito Abe</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raphael I. Benhamou</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alicia J. Angelbello</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kamalakannan Vishnu</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan L. Chen</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7907-9732" title="Orcid link">https://orcid.org/0000-0001-7907-9732</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jessica L. Childs-Disney</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, The Scripps Research Institute, 130 Scripps Way, Jupiter, Florida 33458, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.W.-G. and M.A. contributed equally.</p><span class="author-information-subsection-header">Author Contributions</span><p>M.D.D. and J.C.-D. directed the study, conceived ideas, and designed experiments. S.W.-G., R.I.B., A.J.A., and K.V. performed in vitro and cellular experiments. M.A. performed chemical synthesis. J.L.C. performed NMR experiments.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): M.D.D. is a founder of Expansion Therapeutics, and K.V. and A.J.A are current employees of Expansion Therapeutics.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4162-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank the University of Florida, Center for NeuroGenetics for the generous gift of the DM2 fibroblast cell line used in this paper. The authors also thank the agencies that funded this work including the National Institutes of Health (R35 NS116846 to M.D.D. and F31-NS110269 to A.J.A., and S10-OD021550 to The Scripps Research Institute), the Muscular Dystrophy Association (grant 380467 to M.D.D.), the Myotonic US Fellowship Research grant (to R.I.B.), and to the National Ataxia Foundation Fellowship Research grant (to R.I.B.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CNBP</td><td class="NLM_def"><p class="first last">CCHC-type zinc finger nucleic acid binding protein</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N,N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DM2</td><td class="NLM_def"><p class="first last">myotonic dystrophy type 2</p></td></tr><tr><td class="NLM_term">HCl</td><td class="NLM_def"><p class="first last">hydrochloric acid</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">FISH</td><td class="NLM_def"><p class="first last">fluorescence <i>in situ</i> hybridization</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">IPA</td><td class="NLM_def"><p class="first last">isopropanol</p></td></tr><tr><td class="NLM_term">IR</td><td class="NLM_def"><p class="first last">insulin receptor</p></td></tr><tr><td class="NLM_term">LCMS</td><td class="NLM_def"><p class="first last">liquid chromatography mass spectrometry</p></td></tr><tr><td class="NLM_term">MBNL1</td><td class="NLM_def"><p class="first last">muscleblind-like 1</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">microwave</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77552" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77552" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 31 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulson, H.</span></span> <span> </span><span class="NLM_article-title">Repeat expansion diseases</span>.  <i>Handbook of Clinical Neurology</i>; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2018</span>; Vol.  <span class="NLM_volume">147</span>, pp  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">123</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=105-123&author=H.+Paulson&title=Handbook+of+Clinical+Neurology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPaulson%26aufirst%3DH.%26atitle%3DRepeat%2520expansion%2520diseases%26btitle%3DHandbook%2520of%2520Clinical%2520Neurology%26pub%3DElsevier%26date%3D2018%26volume%3D147%26spage%3D105%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, J. A.</span></span> <span> </span><span class="NLM_article-title">Repeat-associated RNA structure and aberrant splicing</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1862</i></span>,  <span class="NLM_fpage">194405</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2019.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.bbagrm.2019.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=31323433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVajtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1862&publication_year=2019&pages=194405&author=M.+A.+Haleauthor=N.+E.+Johnsonauthor=J.+A.+Berglund&title=Repeat-associated+RNA+structure+and+aberrant+splicing&doi=10.1016%2Fj.bbagrm.2019.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Repeat-associated RNA structure and aberrant splicing</span></div><div class="casAuthors">Hale, Melissa A.; Johnson, Nicholas E.; Berglund, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1862</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">194405</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Over 30 hereditary disorders attributed to the expansion of microsatellite repeats have been identified.  Despite variant nucleotide content, no. of consecutive repeats, and different locations in the genome, many of these diseases have pathogenic RNA gain-of-function mechanisms.  The repeat-contg. RNAs can form structures in vitro predicted to contribute to the disease through assembly of intracellular RNA aggregates termed foci.  The expanded repeat RNAs within these foci sequester RNA binding proteins (RBPs) with important roles in the regulation of RNA metab., most notably alternative splicing (AS).  These deleterious interactions lead to downstream alterations in transcriptome-wide AS directly linked with disease symptoms.  This review summarizes existing knowledge about the assocn. between the repeat RNA structures and RBPs as well as the resulting aberrant AS patterns, specifically in the context of myotonic dystrophy.  The connection between toxic, structured RNAs and dysregulation of AS in other repeat expansion diseases is also discussed.  This article is part of a Special Issue entitled: RNA structure and splicing regulation edited by Francisco Baralle, Ravindra Singh and Stefan Stamm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5rGsBoLD24bVg90H21EOLACvtfcHk0li-Xw2G5yhPww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVajtr7P&md5=15361fe11a4e56e887948d4f12d505f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2019.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2019.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DHale%26aufirst%3DM.%2BA.%26aulast%3DJohnson%26aufirst%3DN.%2BE.%26aulast%3DBerglund%26aufirst%3DJ.%2BA.%26atitle%3DRepeat-associated%2520RNA%2520structure%2520and%2520aberrant%2520splicing%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2019%26volume%3D1862%26spage%3D194405%26doi%3D10.1016%2Fj.bbagrm.2019.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattamatta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranum, L. P. W.</span></span> <span> </span><span class="NLM_article-title">Repeat-associated non-ATG (RAN) translation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">16127</span>– <span class="NLM_lpage">16141</span>, <span class="refDoi"> DOI: 10.1074/jbc.R118.003237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1074%2Fjbc.R118.003237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=30213863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGhsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=16127-16141&author=J.+D.+Clearyauthor=A.+Pattamattaauthor=L.+P.+W.+Ranum&title=Repeat-associated+non-ATG+%28RAN%29+translation&doi=10.1074%2Fjbc.R118.003237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Repeat-associated non-ATG (RAN) translation</span></div><div class="casAuthors">Cleary, John Douglas; Pattamatta, Amrutha; Ranum, Laura P. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">16127-16141</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Microsatellite expansions cause more than 40 neurol. disorders, including Huntington's disease, myotonic dystrophy, and C9ORF72 amyotrophic lateral sclerosis/frontotemporal dementia (ALS/FTD).  These repeat expansion mutations can produce repeat-assocd. non-ATG (RAN) proteins in all three reading frames, which accumulate in disease-relevant tissues.  There has been considerable interest in RAN protein products and their downstream consequences, particularly for the dipeptide proteins found in C9ORF72 ALS/FTD.  Understanding how RAN translation occurs, what cellular factors contribute to RAN protein accumulation, and how these proteins contribute to disease should lead to a better understanding of the basic mechanisms of gene expression and human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKEAHr0bJIWLVg90H21EOLACvtfcHk0li-Xw2G5yhPww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGhsr%252FO&md5=38418351501e2ef2176a031fbafeff74</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R118.003237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R118.003237%26sid%3Dliteratum%253Aachs%26aulast%3DCleary%26aufirst%3DJ.%2BD.%26aulast%3DPattamatta%26aufirst%3DA.%26aulast%3DRanum%26aufirst%3DL.%2BP.%2BW.%26atitle%3DRepeat-associated%2520non-ATG%2520%2528RAN%2529%2520translation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D16127%26epage%3D16141%26doi%3D10.1074%2Fjbc.R118.003237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sznajder, Ł. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrell, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nutter, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobczak, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Intron retention induced by microsatellite expansions as a disease biomarker</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4234</span>, <span class="refDoi"> DOI: 10.1073/pnas.1716617115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1073%2Fpnas.1716617115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=29610297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Glur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=4234&author=%C5%81.+J.+Sznajderauthor=J.+D.+Thomasauthor=E.+M.+Carrellauthor=T.+Reidauthor=K.+N.+McFarlandauthor=J.+D.+Clearyauthor=R.+Oliveiraauthor=C.+A.+Nutterauthor=K.+Bhattauthor=K.+Sobczak&title=Intron+retention+induced+by+microsatellite+expansions+as+a+disease+biomarker&doi=10.1073%2Fpnas.1716617115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Intron retention induced by microsatellite expansions as a disease biomarker</span></div><div class="casAuthors">Sznajder, Lukasz J.; Thomas, James D.; Carrell, Ellie M.; Reid, Tammy; McFarland, Karen N.; Cleary, John D.; Oliveira, Ruan; Nutter, Curtis A.; Bhatt, Kirti; Sobczak, Krzysztof; Ashizawa, Tetsuo; Thornton, Charles A.; Ranum, Laura P. W.; Swanson, Maurice S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4234-4239</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Expansions of simple sequence repeats, or microsatellites, have been linked to ∼30 neurol.-neuromuscular diseases.  While these expansions occur in coding and noncoding regions, microsatellite sequence and repeat length diversity is more prominent in introns with eight different trinucleotide to hexanucleotide repeats, causing hereditary diseases such as myotonic dystrophy type 2 (DM2), Fuchs endothelial corneal dystrophy (FECD), and C9orf72 amyotrophic lateral sclerosis and frontotemporal dementia (C9-ALS/FTD).  Here, we test the hypothesis that these GC-rich intronic microsatellite expansions selectively trigger host intron retention (IR).  Using DM2, FECD, and C9-ALS/FTD as examples, we demonstrate that retention is readily detectable in affected tissues and peripheral blood lymphocytes and conclude that IR screening constitutes a rapid and inexpensive biomarker for intronic repeat expansion disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN1s3EDKLhlrVg90H21EOLACvtfcHk0lgBckBpzhtSYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Glur%252FM&md5=2d57239bf60d4d13a77c05b52d4de8cd</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1716617115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1716617115%26sid%3Dliteratum%253Aachs%26aulast%3DSznajder%26aufirst%3D%25C5%2581.%2BJ.%26aulast%3DThomas%26aufirst%3DJ.%2BD.%26aulast%3DCarrell%26aufirst%3DE.%2BM.%26aulast%3DReid%26aufirst%3DT.%26aulast%3DMcFarland%26aufirst%3DK.%2BN.%26aulast%3DCleary%26aufirst%3DJ.%2BD.%26aulast%3DOliveira%26aufirst%3DR.%26aulast%3DNutter%26aufirst%3DC.%2BA.%26aulast%3DBhatt%26aufirst%3DK.%26aulast%3DSobczak%26aufirst%3DK.%26atitle%3DIntron%2520retention%2520induced%2520by%2520microsatellite%2520expansions%2520as%2520a%2520disease%2520biomarker%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26spage%3D4234%26doi%3D10.1073%2Fpnas.1716617115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liquori, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kress, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranum, L. P. W.</span></span> <span> </span><span class="NLM_article-title">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">864</span>, <span class="refDoi"> DOI: 10.1126/science.1062125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1126%2Fscience.1062125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11486088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=864&author=C.+L.+Liquoriauthor=K.+Rickerauthor=M.+L.+Moseleyauthor=J.+F.+Jacobsenauthor=W.+Kressauthor=S.+L.+Naylorauthor=J.+W.+Dayauthor=L.+P.+W.+Ranum&title=Myotonic+dystrophy+type+2+caused+by+a+CCTG+expansion+in+intron+1+of+ZNF9&doi=10.1126%2Fscience.1062125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span></div><div class="casAuthors">Liquori, Christina L.; Ricker, Kenneth; Moseley, Melinda L.; Jacobsen, Jennifer F.; Kress, Wolfram; Naylor, Susan L.; Day, John W.; Ranum, Laura P. W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5531</span>),
    <span class="NLM_cas:pages">864-867</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Myotonic dystrophy (DM), the most common form of muscular dystrophy in adults, can be caused by a mutation on either chromosome 19q13 (DM1) or 3q21 (DMZ/PROMM).  DM1 is caused by a CTG expansion in the 3' untranslated region of the dystrophia myotonica-protein kinase gene (DMPK).  Several mechanisms have been invoked to explain how this mutation, which does not alter the protein-coding portion of a gene, causes the specific constellation of clin. features characteristic of DM.  We now report that DMZ is caused by a CCTG expansion (mean ∼5000 repeats) located in intron 1 of the zinc finger protein 9 (ZNF9) gene.  Parallels between these mutations indicate that microsatellite expansions in RNA can be pathogenic and cause the multisystemic features of DM1 and DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeKIAHv1r8_bVg90H21EOLACvtfcHk0lgBckBpzhtSYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D&md5=812d852f480461b722162114fcb906f2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1062125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1062125%26sid%3Dliteratum%253Aachs%26aulast%3DLiquori%26aufirst%3DC.%2BL.%26aulast%3DRicker%26aufirst%3DK.%26aulast%3DMoseley%26aufirst%3DM.%2BL.%26aulast%3DJacobsen%26aufirst%3DJ.%2BF.%26aulast%3DKress%26aufirst%3DW.%26aulast%3DNaylor%26aufirst%3DS.%2BL.%26aulast%3DDay%26aufirst%3DJ.%2BW.%26aulast%3DRanum%26aufirst%3DL.%2BP.%2BW.%26atitle%3DMyotonic%2520dystrophy%2520type%25202%2520caused%2520by%2520a%2520CCTG%2520expansion%2520in%2520intron%25201%2520of%2520ZNF9%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D864%26doi%3D10.1126%2Fscience.1062125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schatz, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Structure of the myotonic dystrophy type 2 RNA and designed small molecules that reduce toxicity</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1021/cb4007387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4007387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2ku7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=538-550&author=J.+L.+Childs-Disneyauthor=I.+Yildirimauthor=H.+Parkauthor=J.+R.+Lohmanauthor=L.+Guanauthor=T.+Tranauthor=P.+Sarkarauthor=G.+C.+Schatzauthor=M.+D.+Disney&title=Structure+of+the+myotonic+dystrophy+type+2+RNA+and+designed+small+molecules+that+reduce+toxicity&doi=10.1021%2Fcb4007387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Myotonic Dystrophy Type 2 RNA and Designed Small Molecules That Reduce Toxicity</span></div><div class="casAuthors">Childs-Disney, Jessica L.; Yildirim, Ilyas; Park, Ha Jeung; Lohman, Jeremy R.; Guan, Lirui; Tran, Tuan; Sarkar, Partha; Schatz, George C.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">538-550</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is an incurable neuromuscular disorder caused by a r-(CCUG) expansion (r-(CCUG)exp) that folds into an extended hairpin with periodically repeating 2 × 2 nucleotide internal loops (5'CCUG/3'GUCC).  We designed multivalent compds. that improve DM2-assocd. defects using information about RNA-small mol. interactions.  We also report the first crystal structure of r-(CCUG) repeats refined to 2.35 Å.  Structural anal. of the three 5'CCUG/3'GUCC repeat internal loops (L) reveals that the CU pairs in L1 are each stabilized by one hydrogen bond and a water-mediated hydrogen bond, while CU pairs in L2 and L3 are stabilized by two hydrogen bonds.  Mol. dynamics (MD) simulations reveal that the CU pairs are dynamic and stabilized by Na+ and water mols.  MD simulations of the binding of the small mol. to r-(CCUG) repeats reveal that the lowest free energy binding mode occurs via the major groove, in which one C residue is unstacked and the cross-strand nucleotides are displaced.  Moreover, we modeled the binding of our dimeric compd. to two 5'CCUG/3'GUCC motifs, which shows that the scaffold on which the RNA-binding modules are displayed provides an optimal distance to span two adjacent loops.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpShDc5fRNy6bVg90H21EOLACvtfcHk0lgBckBpzhtSYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2ku7zK&md5=8696f289e41fb8c984c167e3e64aa311</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcb4007387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4007387%26sid%3Dliteratum%253Aachs%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLohman%26aufirst%3DJ.%2BR.%26aulast%3DGuan%26aufirst%3DL.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DSarkar%26aufirst%3DP.%26aulast%3DSchatz%26aufirst%3DG.%2BC.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DStructure%2520of%2520the%2520myotonic%2520dystrophy%2520type%25202%2520RNA%2520and%2520designed%2520small%2520molecules%2520that%2520reduce%2520toxicity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D538%26epage%3D550%26doi%3D10.1021%2Fcb4007387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fardaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamshere, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brook, J. D.</span></span> <span> </span><span class="NLM_article-title">Three proteins, MBNL, MBLL and MBXL, co-localize in vivo with nuclear foci of expanded-repeat transcripts in DM1 and DM2 cells</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">814</span>, <span class="refDoi"> DOI: 10.1093/hmg/11.7.805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1093%2Fhmg%2F11.7.805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11929853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt1GrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=805-814&author=M.+Fardaeiauthor=M.+T.+Rogersauthor=H.+M.+Thorpeauthor=K.+Larkinauthor=M.+G.+Hamshereauthor=P.+S.+Harperauthor=J.+D.+Brook&title=Three+proteins%2C+MBNL%2C+MBLL+and+MBXL%2C+co-localize+in+vivo+with+nuclear+foci+of+expanded-repeat+transcripts+in+DM1+and+DM2+cells&doi=10.1093%2Fhmg%2F11.7.805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Three proteins, MBNL, MBLL and MBXL, co-localize in vivo with nuclear foci of expanded-repeat transcripts in DM1 and DM2 cells</span></div><div class="casAuthors">Fardaei, Majid; Rogers, Mark T.; Thorpe, Helena M.; Larkin, Kenneth; Hamshere, Marion G.; Harper, Peter S.; Brook, J. David</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-814</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy is a complex neuromuscular disorder assocd. with DNA expansion mutations in two different genes.  In DM1 a CTG repeat in the 3'-untranslated region of DMPK is expanded, whereas in DM2 an intronic CCTG expansion occurs in the gene ZNF9.  Transcripts contg. expanded repeats form foci in the nuclei of DM1 and DM2 cells.  Recent work using antibodies has shown that proteins related to Drosophila muscleblind co-localize with repeat foci in DM1 and DM2 cells.  We show that rather than there being a single human muscleblind gene producing multiple proteins through alternative splicing, there are in fact three different muscleblind genes, MBNL, MBLL and MBXL, which map to chromosomes 3, 13 and X, resp., and which show extensive alternative splicing.  Two of the genes, MBNL and MBLL, are expressed in many adult tissues whereas MBXL is expressed predominantly in the placenta.  Green fluorescent protein-tagged versions of MBNL, MBLL and MBXL co-localize with nuclear foci in DM1 and DM2 cells, suggesting that all three proteins may play a role in DM pathophysiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDPPadt35dOrVg90H21EOLACvtfcHk0lgEyrx6cly8yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt1GrsbY%253D&md5=c02d0ee9b6f306e26fd004493f0b6cff</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F11.7.805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F11.7.805%26sid%3Dliteratum%253Aachs%26aulast%3DFardaei%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DM.%2BT.%26aulast%3DThorpe%26aufirst%3DH.%2BM.%26aulast%3DLarkin%26aufirst%3DK.%26aulast%3DHamshere%26aufirst%3DM.%2BG.%26aulast%3DHarper%26aufirst%3DP.%2BS.%26aulast%3DBrook%26aufirst%3DJ.%2BD.%26atitle%3DThree%2520proteins%252C%2520MBNL%252C%2520MBLL%2520and%2520MBXL%252C%2520co-localize%2520in%2520vivo%2520with%2520nuclear%2520foci%2520of%2520expanded-repeat%2520transcripts%2520in%2520DM1%2520and%2520DM2%2520cells%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2002%26volume%3D11%26spage%3D805%26epage%3D814%26doi%3D10.1093%2Fhmg%2F11.7.805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mankodi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbinati, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Q. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moxley, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krym, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schalling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, C. A.</span></span> <span> </span><span class="NLM_article-title">Muscleblind localizes to nuclear foci of aberrant RNA in myotonic dystrophy types 1 and 2</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2165</span>– <span class="NLM_lpage">2170</span>, <span class="refDoi"> DOI: 10.1093/hmg/10.19.2165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1093%2Fhmg%2F10.19.2165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11590133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsl2ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2001&pages=2165-2170&author=A.+Mankodiauthor=C.+R.+Urbinatiauthor=Q.+P.+Yuanauthor=R.+T.+Moxleyauthor=V.+Sansoneauthor=M.+Krymauthor=D.+Hendersonauthor=M.+Schallingauthor=M.+S.+Swansonauthor=C.+A.+Thornton&title=Muscleblind+localizes+to+nuclear+foci+of+aberrant+RNA+in+myotonic+dystrophy+types+1+and+2&doi=10.1093%2Fhmg%2F10.19.2165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Muscleblind localizes to nuclear foci of aberrant RNA in myotonic dystrophy types 1 and 2</span></div><div class="casAuthors">Mankodi, Ami; Urbinati, Carl R.; Yuan, Qiu-Ping; Moxley, Richard T.; Sansone, Valeria; Krym, Matt; Henderson, Donald; Schalling, Martin; Swanson, Maurice S.; Thornton, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2165-2170</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The phenotypes in myotonic dystrophy types 1 and 2 (DM1 and DM2) are similar, suggesting a shared pathophysiol. mechanism.  DM1 is caused by expansion of a CTG repeat in the DMPK gene.  Pathogenic effects of this mutation are likely to be mediated, at least in part, by the expanded CUG repeat in mutant mRNA.  The mutant transcripts are retained in the nucleus in multiple discrete foci.  We investigated the possibility that DM2 is also caused by expansion of a CTG repeat or related sequence.  Anal. of DNA by repeat expansion detection methods, and RNA by RNase protection, did not show an expanded CTG or CUG repeat in DM2.  However, hybridization of muscle sections with fluorescence-labeled CAG-repeat oligonucleotides showed nuclear foci in DM2 similar to those seen in DM1.  Nuclear foci were present in all patients with symptomatic DM1 (n = 9) or DM2 (n = 9) but not in any disease controls or healthy subjects (n = 23).  The foci were not seen with CUG- or GUC-repeat probes.  Foci in DM2 were distinguished from DM1 by lower stability of the probe-target duplex, suggesting that a sequence related to the DM1 CUG expansion accumulates in the DM2 nucleus.  Muscleblind proteins, which interact with expanded CUG repeats in vitro, localized to the nuclear foci in both DM1 and DM2.  These results support the idea that nuclear accumulation of mutant RNA is pathogenic in DM1, suggest that a similar disease process occurs in DM2, and point to a role for muscleblind in the pathogenesis of both disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYY4FvhukXZrVg90H21EOLACvtfcHk0lgEyrx6cly8yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsl2ktbg%253D&md5=1d00123bf238f8b70406aec84699a176</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F10.19.2165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F10.19.2165%26sid%3Dliteratum%253Aachs%26aulast%3DMankodi%26aufirst%3DA.%26aulast%3DUrbinati%26aufirst%3DC.%2BR.%26aulast%3DYuan%26aufirst%3DQ.%2BP.%26aulast%3DMoxley%26aufirst%3DR.%2BT.%26aulast%3DSansone%26aufirst%3DV.%26aulast%3DKrym%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DD.%26aulast%3DSchalling%26aufirst%3DM.%26aulast%3DSwanson%26aufirst%3DM.%2BS.%26aulast%3DThornton%26aufirst%3DC.%2BA.%26atitle%3DMuscleblind%2520localizes%2520to%2520nuclear%2520foci%2520of%2520aberrant%2520RNA%2520in%2520myotonic%2520dystrophy%2520types%25201%2520and%25202%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2001%26volume%3D10%26spage%3D2165%26epage%3D2170%26doi%3D10.1093%2Fhmg%2F10.19.2165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savkur, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span> <span> </span><span class="NLM_article-title">Aberrant regulation of insulin receptor alternative splicing is associated with insulin resistance in myotonic dystrophy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1038/ng704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1038%2Fng704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11528389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFGmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=40-47&author=R.+S.+Savkurauthor=A.+V.+Philipsauthor=T.+A.+Cooper&title=Aberrant+regulation+of+insulin+receptor+alternative+splicing+is+associated+with+insulin+resistance+in+myotonic+dystrophy&doi=10.1038%2Fng704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Aberrant regulation of insulin receptor alternative splicing is associated with insulin resistance in myotonic dystrophy</span></div><div class="casAuthors">Savkur, Rajesh S.; Philips, Anne V.; Cooper, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-47</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is caused by a CTG trinucleotide expansion in the 3' untranslated region of the DM protein kinase gene.  People with DM1 have an unusual form of insulin resistance caused by a defect in skeletal muscle.  Here we demonstrate that alternative splicing of the insulin receptor (IR) pre-mRNA is aberrantly regulated in DM1 skeletal muscle tissue, resulting in predominant expression of the lower-signaling nonmuscle isoform (IR-A).  IR-A also predominates in DM1 skeletal muscle cultures, which exhibit a decreased metabolic response to insulin relative to cultures from normal controls.  Steady-state levels of CUG-BP, a regulator of pre-mRNA splicing proposed to mediate some aspects of DM1 pathogenesis, are increased in DM1 skeletal muscle; overexpression of CUG-BP in normal cells induces a switch to IR-A.  The CUG-BP protein mediates this switch through an intronic element located upstream of the alternatively spliced exon 11, and specifically binds within this element in vitro.  These results support a model in which increased expression of a splicing regulator contributes to insulin resistance in DM1 by affecting IR alternative splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVrM-Glm4je7Vg90H21EOLACvtfcHk0lgEyrx6cly8yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFGmsbg%253D&md5=544636c361fe07f81b60beb9d3b3a671</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fng704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng704%26sid%3Dliteratum%253Aachs%26aulast%3DSavkur%26aufirst%3DR.%2BS.%26aulast%3DPhilips%26aufirst%3DA.%2BV.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26atitle%3DAberrant%2520regulation%2520of%2520insulin%2520receptor%2520alternative%2520splicing%2520is%2520associated%2520with%2520insulin%2520resistance%2520in%2520myotonic%2520dystrophy%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D29%26spage%3D40%26epage%3D47%26doi%3D10.1038%2Fng704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA catalyzes the cellular synthesis of its own inhibitor, shunting It to endogenous decay pathways</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.chembiol.2020.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=31981476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFShu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=223-231&author=R.+I.+Benhamouauthor=A.+J.+Angelbelloauthor=E.+T.+Wangauthor=M.+D.+Disney&title=A+toxic+RNA+catalyzes+the+cellular+synthesis+of+its+own+inhibitor%2C+shunting+It+to+endogenous+decay+pathways&doi=10.1016%2Fj.chembiol.2020.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A Toxic RNA Catalyzes the Cellular Synthesis of Its Own Inhibitor, Shunting It to Endogenous Decay Pathways</span></div><div class="casAuthors">Benhamou, Raphael I.; Angelbello, Alicia J.; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">223-231.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is a genetically defined disease caused by a toxic expanded repeat of r(CCUG) [r(CCUG)exp], harbored in intron 1 of CCHC-type zinc-finger nucleic acid binding protein (CNBP) pre-mRNA.  This r(CCUG)exp causes toxicity via a gain-of-function mechanism, resulting in three pathol. hallmarks: aggregation into nuclear foci; sequestration of muscleblind-like-1 (MBNL1) protein, leading to splicing defects; and retention of CNBP intron 1.  We studied two types of small mols. with different modes of action, ones that simply bind and ones that are templated by r(CCUG)exp in cells, i.e., the RNA synthesizes its own drug.  Indeed, our studies completed in DM2 patient-derived fibroblasts showed that the compds. disrupt the r(CCUG)exp-MBNL1 complex, reduce intron retention, subjecting the liberated intronic r(CCUG)exp to native decay pathways, and rescue other DM2-assocd. cellular defects.  Importantly, this study shows that small mols. can modulate RNA biol. by shunting toxic transcripts toward native decay pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraOFp22Q9xcLVg90H21EOLACvtfcHk0ljV1kQfoZyLuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFShu7c%253D&md5=9c8623dd04af21511ce6d3cc4deb2a9e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520catalyzes%2520the%2520cellular%2520synthesis%2520of%2520its%2520own%2520inhibitor%252C%2520shunting%2520It%2520to%2520endogenous%2520decay%2520pathways%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D223%26epage%3D231%26doi%3D10.1016%2Fj.chembiol.2020.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wieben, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleff, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tosakulwong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Highsmith, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratz, K. H.</span></span> <span> </span><span class="NLM_article-title">A common trinucleotide repeat expansion within the transcription factor 4 (TCF4, E2-2) gene predicts Fuchs corneal dystrophy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e49083</span>– <span class="NLM_lpage">e49086</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0049083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1371%2Fjournal.pone.0049083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=23185296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVagsrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e49083-e49086&author=E.+D.+Wiebenauthor=R.+A.+Aleffauthor=N.+Tosakulwongauthor=M.+L.+Butzauthor=W.+E.+Highsmithauthor=A.+O.+Edwardsauthor=K.+H.+Baratz&title=A+common+trinucleotide+repeat+expansion+within+the+transcription+factor+4+%28TCF4%2C+E2-2%29+gene+predicts+Fuchs+corneal+dystrophy&doi=10.1371%2Fjournal.pone.0049083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A common trinucleotide repeat expansion within the transcription factor 4 (TCF4, E2-2) gene predicts fuchs corneal dystrophy</span></div><div class="casAuthors">Wieben, Eric D.; Aleff, Ross A.; Tosakulwong, Nirubol; Butz, Malinda L.; Highsmith, W. Edward; Edwards, Albert O.; Baratz, Keith H.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e49083</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Fuchs endothelial corneal dystrophy (FECD) is a common, familial disease of the corneal endothelium and is the leading indication for corneal transplantation.  Variation in the transcription factor 4 (TCF4) gene has been identified as a major contributor to the disease.  We tested for an assocn. between an intronic TGC trinucleotide repeat in TCF4 and FECD by detg. repeat length in 66 affected participants with severe FECD and 63 participants with normal corneas in a 3-stage discovery/replication/validation study.  PCR primers flanking the TGC repeat were used to amplify leukocyte-derived genomic DNA.  Repeat length was detd. by direct sequencing, short tandem repeat (STR) assay and Southern blotting.  Genomic Southern blots were used to evaluate samples for which only a single allele was identified by STR anal.  Compiling data for 3 arms of the study, a TGC repeat length >50 was present in 79% of FECD cases and in 3% of normal controls cases (p<0.001).  Among cases, 52 of 66 (79%) subjects had >50 TGC repeats, 13 (20%) had <40 repeats and 1 (2%) had an intermediate repeat length.  In comparison, only 2 of 63 (3%) unaffected control subjects had >50 repeats, 60 (95%) had <40 repeats and 1 (2%) had an intermediate repeat length.  The repeat length was greater than 1000 in 4 FECD cases.  The sensitivity and specificity of >50 TGC repeats identifying FECD in this patient cohort was 79% and 96%, resp. Expanded TGC repeat was more specific for FECD cases than the previously identified, highly assocd., single nucleotide polymorphism, rs613872 (specificity = 79%).  The TGC trinucleotide repeat expansion in TCF4 is strongly assocd. with FECD, and a repeat length >50 is highly specific for the disease.  This assocn. suggests that trinucleotide expansion may play a pathogenic role in the majority of FECD cases and is a predictor of disease risk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsRjy7lgVHc7Vg90H21EOLACvtfcHk0ljV1kQfoZyLuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVagsrvF&md5=98afb8d0791873502710d4716679d298</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0049083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0049083%26sid%3Dliteratum%253Aachs%26aulast%3DWieben%26aufirst%3DE.%2BD.%26aulast%3DAleff%26aufirst%3DR.%2BA.%26aulast%3DTosakulwong%26aufirst%3DN.%26aulast%3DButz%26aufirst%3DM.%2BL.%26aulast%3DHighsmith%26aufirst%3DW.%2BE.%26aulast%3DEdwards%26aufirst%3DA.%2BO.%26aulast%3DBaratz%26aufirst%3DK.%2BH.%26atitle%3DA%2520common%2520trinucleotide%2520repeat%2520expansion%2520within%2520the%2520transcription%2520factor%25204%2520%2528TCF4%252C%2520E2-2%2529%2520gene%2520predicts%2520Fuchs%2520corneal%2520dystrophy%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De49083%26epage%3De49086%26doi%3D10.1371%2Fjournal.pone.0049083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jun, A. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A small molecule that binds an RNA repeat expansion stimulates its decay via the exosome complex</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.chembiol.2020.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=33157036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1GqurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2021&pages=34-45&author=A.+J.+Angelbelloauthor=R.+I.+Benhamouauthor=S.+G.+Rzuczekauthor=S.+Choudharyauthor=Z.+Tangauthor=J.+L.+Chenauthor=M.+Royauthor=K.+W.+Wangauthor=I.+Yildirimauthor=A.+S.+Jun&title=A+small+molecule+that+binds+an+RNA+repeat+expansion+stimulates+its+decay+via+the+exosome+complex&doi=10.1016%2Fj.chembiol.2020.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule that Binds an RNA Repeat Expansion Stimulates Its Decay via the Exosome Complex</span></div><div class="casAuthors">Angelbello, Alicia J.; Benhamou, Raphael I.; Rzuczek, Suzanne G.; Choudhary, Shruti; Tang, Zhenzhi; Chen, Jonathan L.; Roy, Madhuparna; Wang, Kye Won; Yildirim, Ilyas; Jun, Albert S.; Thornton, Charles A.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-45.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Many diseases are caused by toxic RNA repeats.  Herein, we designed a lead small mol. that binds the structure of the r(CUG) repeat expansion [r(CUG)exp] that causes myotonic dystrophy type 1 (DM1) and Fuchs endothelial corneal dystrophy (FECD) and rescues disease biol. in patient-derived cells and in vivo.  Interestingly, the compds. downstream effects are different in the two diseases, owing to the location of the repeat expansion.  In DM1, r(CUG)exp is harbored in the 3'untranslated region, and the compd. has no effect on the mRNA's abundance.  In FECD, however, r(CUG)exp is located in an intron, and the small mol. facilitates excision of the intron, which is then degraded by the RNA exosome complex.  Thus, structure-specific, RNA-targeting small mols. can act disease specifically to affect biol., either by disabling the gain-of-function mechanism (DM1) or by stimulating quality control pathways to rid a disease-affected cell of a toxic RNA (FECD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfxuyDTqH07rVg90H21EOLACvtfcHk0ljV1kQfoZyLuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1GqurfM&md5=3f5e20b94dbb375234c9931657897d47</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DChoudhary%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DRoy%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DK.%2BW.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DJun%26aufirst%3DA.%2BS.%26atitle%3DA%2520small%2520molecule%2520that%2520binds%2520an%2520RNA%2520repeat%2520expansion%2520stimulates%2520its%2520decay%2520via%2520the%2520exosome%2520complex%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2021%26volume%3D28%26spage%3D34%26epage%3D45%26doi%3D10.1016%2Fj.chembiol.2020.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckee, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olafson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small-molecule cleavage of an r(CUG) repeat expansion in a myotonic dystrophy mouse model</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">7799</span>– <span class="NLM_lpage">7804</span>, <span class="refDoi"> DOI: 10.1073/pnas.1901484116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1073%2Fpnas.1901484116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=30926669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=7799-7804&author=A.+J.+Angelbelloauthor=S.+G.+Rzuczekauthor=K.+K.+Mckeeauthor=J.+L.+Chenauthor=H.+Olafsonauthor=M.+D.+Cameronauthor=W.+N.+Mossauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Precise+small-molecule+cleavage+of+an+r%28CUG%29+repeat+expansion+in+a+myotonic+dystrophy+mouse+model&doi=10.1073%2Fpnas.1901484116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule cleavage of an r(CUG) repeat expansion in a myotonic dystrophy mouse model</span></div><div class="casAuthors">Angelbello, Alicia J.; Rzuczek, Suzanne G.; Mckee, Kendra K.; Chen, Jonathan L.; Olafson, Hailey; Cameron, Michael D.; Moss, Walter N.; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7799-7804</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is an incurable neuromuscular disorder caused by an expanded CTG repeat that is transcribed into r(CUG)exp.  The RNA repeat expansion sequesters regulatory proteins such as Muscleblind-like protein 1 (MBNL1), which causes pre-mRNA splicing defects.  The disease-causing r(CUG)exp has been targeted by antisense oligonucleotides, CRISPR-based approaches, and RNA-targeting small mols.  Herein, we describe a designer small mol., Cugamycin, that recognizes the structure of r(CUG)exp and cleaves it in both DM1 patient-derived myotubes and a DM1 mouse model, leaving short repeats of r(CUG) untouched.  In contrast, oligonucleotides that recognize r(CUG) sequence rather than structure cleave both long and short r(CUG)-contg. transcripts.  Transcriptomic, histol., and phenotypic studies demonstrate that Cugamycin broadly and specifically relieves DM1-assocd. defects in vivo without detectable off-targets.  Thus, small mols. that bind and cleave RNA have utility as lead chem. probes and medicines and can selectively target disease-causing RNA structures to broadly improve defects in preclin. animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg0da_Q8Pd37Vg90H21EOLACvtfcHk0liQVebO1UZcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D&md5=9dfa637039d1b532cbcbbaaccaed40b1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1901484116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1901484116%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DMckee%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DOlafson%26aufirst%3DH.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small-molecule%2520cleavage%2520of%2520an%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520a%2520myotonic%2520dystrophy%2520mouse%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D7799%26epage%3D7804%26doi%3D10.1073%2Fpnas.1901484116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeep
Velagapudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">2406</span>– <span class="NLM_lpage">2411</span>, <span class="refDoi"> DOI: 10.1073/pnas.1914286117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1073%2Fpnas.1914286117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=31964809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2406-2411&author=M.+G.+Costalesauthor=H.+Aikawaauthor=Y.+Liauthor=J.+L.+Childs-Disneyauthor=D.+Abeggauthor=D.+G.+Hochauthor=S.+Pradeep%0AVelagapudiauthor=Y.+Nakaiauthor=T.+Khanauthor=K.+W.+Wang&title=Small-molecule+targeted+recruitment+of+a+nuclease+to+cleave+an+oncogenic+RNA+in+a+mouse+model+of+metastatic+cancer&doi=10.1073%2Fpnas.1914286117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span></div><div class="casAuthors">Costales, Matthew G.; Aikawa, Haruo; Li, Yue; Childs-Disney, Jessica L.; Abegg, Daniel; Hoch, Dominic G.; Velagapudi, Sai Pradeep; Nakai, Yoshio; Khan, Tanya; Wang, Kye Won; Yildirim, Ilyas; Adibekian, Alexander; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2406-2411</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the area of small mols. interacting with RNA advances, general routes to provide bioactive compds. are needed as ligands can bind RNA avidly to sites that will not affect function.  Small-mol. targeted RNA degrdn. will thus provide a general route to affect RNA biol.  A non-oligonucleotide-contg. compd. was designed from sequence to target the precursor to oncogenic microRNA-21 (pre-miR-21) for enzymic destruction with selectivity that can exceed that for protein-targeted medicines.  The compd. specifically binds the target and contains a heterocycle that recruits and activates a RNase to pre-miR-21 to substoichiometrically effect its cleavage and subsequently impede metastasis of breast cancer to lung in a mouse model.  Transcriptomic and proteomic analyses demonstrate that the compd. is potent and selective, specifically modulating oncogenic pathways.  Thus, small mols. can be designed from sequence to have all of the functional repertoire of oligonucleotides, including inducing enzymic degrdn., and to selectively and potently modulate RNA function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolzufxJ0-LLLVg90H21EOLACvtfcHk0liQVebO1UZcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D&md5=450996be5aadaa9e13acb5c2cfb03c62</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1914286117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1914286117%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DPradeep%2BVelagapudi%26aufirst%3DS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DK.%2BW.%26atitle%3DSmall-molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520cleave%2520an%2520oncogenic%2520RNA%2520in%2520a%2520mouse%2520model%2520of%2520metastatic%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2020%26volume%3D117%26spage%3D2406%26epage%3D2411%26doi%3D10.1073%2Fpnas.1914286117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Targeting RNA with small molecules to capture opportunities at the intersection of chemistry, biology, and medicine</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">6776</span>– <span class="NLM_lpage">6790</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslaqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=6776-6790&author=M.+D.+Disney&title=Targeting+RNA+with+small+molecules+to+capture+opportunities+at+the+intersection+of+chemistry%2C+biology%2C+and+medicine&doi=10.1021%2Fjacs.8b13419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting RNA with Small Molecules To Capture Opportunities at the Intersection of Chemistry, Biology, and Medicine</span></div><div class="casAuthors">Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6776-6790</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The biol. of healthy and diseased cells is often mediated by RNA structures, desirable targets for small mol. chem. probes and lead medicines.  Although structured regions are found throughout the transcriptome, some even with demonstrated functionality, human RNAs are considered recalcitrant to small mol. targeting.  However, targeting structured regions with small mols. provides an important alternative to oligonucleotides that target sequence.  In this Perspective, the authors describe challenges and progress in developing small mols. interacting with RNA (SMIRNAs) to capture their significant opportunities at the intersection of chem., biol., and medicine.  Key to establishing a new paradigm in chem. biol. and medicine is the development of methods to obtain, preferably by design, bioactive compds. that modulate RNA targets and the companion methods that validate their direct effects in cells and preclin. models.  While difficult, demonstration of direct target engagement in the complex cellular milieu, along with methods to establish modes of action, are required to push this field forward The authors also describe frameworks for accelerated advancements in this burgeoning area, their implications, key new technologies for development of SMIRNAs, and milestones that have led to broader acceptance of RNA as a small mol. druggable target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9jiNQMcQjbLVg90H21EOLACvtfcHk0liQVebO1UZcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslaqtL4%253D&md5=bcf0d58b321ee12c921650d3b1bc388c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13419%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTargeting%2520RNA%2520with%2520small%2520molecules%2520to%2520capture%2520opportunities%2520at%2520the%2520intersection%2520of%2520chemistry%252C%2520biology%252C%2520and%2520medicine%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D6776%26epage%3D6790%26doi%3D10.1021%2Fjacs.8b13419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Structure-specific cleavage of an RNA repeat expansion with a dimeric small molecule is advantageous over sequence-specific recognition by an oligonucleotide</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">493</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00958</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00958" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtlegtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=485-493&author=R.+I.+Benhamouauthor=A.+J.+Angelbelloauthor=R.+J.+Andrewsauthor=E.+T.+Wangauthor=W.+N.+Mossauthor=M.+D.+Disney&title=Structure-specific+cleavage+of+an+RNA+repeat+expansion+with+a+dimeric+small+molecule+is+advantageous+over+sequence-specific+recognition+by+an+oligonucleotide&doi=10.1021%2Facschembio.9b00958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Specific Cleavage of an RNA Repeat Expansion with a Dimeric Small Molecule Is Advantageous over Sequence-Specific Recognition by an Oligonucleotide</span></div><div class="casAuthors">Benhamou, Raphael I.; Angelbello, Alicia J.; Andrews, Ryan J.; Wang, Eric T.; Moss, Walter N.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">485-493</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is a genetically defined muscular dystrophy that is caused by an expanded repeat of r(CCUG) [r(CCUG)exp] in intron 1 of a CHC-Type Zinc Finger Nucleic Acid Binding Protein (CNBP) pre-mRNA.  Various mechanisms contribute to DM2 pathol. including pre-mRNA splicing defects caused by sequestration of the RNA splicing regulator muscleblind-like-1 (MBNL1) by r(CCUG)exp.  Herein, we study the biol. impacts of the mol. recognition of r(CCUG)exp's structure by a designer dimeric small mol. that directly cleaves the RNA in patient-derived cells.  The compd. is comprised of two RNA-binding modules conjugated to a deriv. of the natural product bleomycin.  Careful design of the chimera affords RNA-specific cleavage as attachment of the bleomycin cleaving module was done in a manner that disables DNA cleavage.  The chimeric cleaver is more potent than the parent binding compd. for alleviating DM2-assocd. defects.  Importantly, oligonucleotides targeting r(CCUG)exp sequence for cleavage exacerbate DM2 defects due to recognition of a short r(CCUG) sequence that is embedded in both CNBP, argonaute-1 (AGO1), and MBNL1, reducing these targets.  The latter event causes a greater depletion of functional MBNL1 than the amt. already sequestered by r(CCUG)exp.  Thus, compds. targeting RNA structures can have functional advantages over oligonucleotides that target sequence in some disease settings, particularly in DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDuGRa8-OiobVg90H21EOLACvtfcHk0lhhTN_860RqRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtlegtA%253D%253D&md5=5a1952891f4660a3353722d1a881cee6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00958%26sid%3Dliteratum%253Aachs%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DStructure-specific%2520cleavage%2520of%2520an%2520RNA%2520repeat%2520expansion%2520with%2520a%2520dimeric%2520small%2520molecule%2520is%2520advantageous%2520over%2520sequence-specific%2520recognition%2520by%2520an%2520oligonucleotide%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D485%26epage%3D493%26doi%3D10.1021%2Facschembio.9b00958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Optimization of the linker domain in a dimeric compound that degrades an r(CUG) repeat expansion in cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7827</span>– <span class="NLM_lpage">7839</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlGksbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7827-7839&author=R.+I.+Benhamouauthor=M.+Abeauthor=S.+Choudharyauthor=S.+M.+Meyerauthor=A.+J.+Angelbelloauthor=M.+D.+Disney&title=Optimization+of+the+linker+domain+in+a+dimeric+compound+that+degrades+an+r%28CUG%29+repeat+expansion+in+cells&doi=10.1021%2Facs.jmedchem.0c00558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the Linker Domain in a Dimeric Compound that Degrades an r(CUG) Repeat Expansion in Cells</span></div><div class="casAuthors">Benhamou, Raphael I.; Abe, Masahito; Choudhary, Shruti; Meyer, Samantha M.; Angelbello, Alicia J.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7827-7839</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA repeat expansions are responsible for more than 30 incurable diseases.  Among them is myotonic dystrophy type 1 (DM1), the most common form of adult on-set muscular dystrophy.  DM1 is caused by an r(CUG) repeat expansion [r(CUG)exp] located in the 3' untranslated region (UTR) of the dystrophia myotonica protein kinase gene.  This repeat expansion is highly structured, forming a periodic array of 5'CUG/3'GUC internal loop motifs.  We therefore designed dimeric compds. that simultaneously bind two of these motifs by connecting two RNA-binding modules with peptoid linkers of different geometries and lengths.  The optimal linker contains two proline residues and enhances compd. affinity.  Equipping this mol. with a bleomycin A5 cleaving module converts the simple binding compd. into a potent allele-selective cleaver of r(CUG)exp.  This study shows that the linker in modularly assembled ligands targeting RNA can be optimized to afford potent biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXG7D_mmEozrVg90H21EOLACvtfcHk0lhhTN_860RqRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlGksbrO&md5=aba7ecdeb9aa4aad9b1bdcde1f03ff8a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00558%26sid%3Dliteratum%253Aachs%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DAbe%26aufirst%3DM.%26aulast%3DChoudhary%26aufirst%3DS.%26aulast%3DMeyer%26aufirst%3DS.%2BM.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DOptimization%2520of%2520the%2520linker%2520domain%2520in%2520a%2520dimeric%2520compound%2520that%2520degrades%2520an%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7827%26epage%3D7839%26doi%3D10.1021%2Facs.jmedchem.0c00558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramisetty, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S. C.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of MBNL1–CCUG interaction by small molecules toward potential therapeutic agents for myotonic dystrophy type 2 (DM2)</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">8881</span>– <span class="NLM_lpage">8890</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1093%2Fnar%2Fgkr415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=21768123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqt7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=8881-8890&author=C.-H.+Wongauthor=Y.+Fuauthor=S.+R.+Ramisettyauthor=A.+M.+Barangerauthor=S.+C.+Zimmerman&title=Selective+inhibition+of+MBNL1%E2%80%93CCUG+interaction+by+small+molecules+toward+potential+therapeutic+agents+for+myotonic+dystrophy+type+2+%28DM2%29&doi=10.1093%2Fnar%2Fgkr415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of MBNL1-CCUG interaction by small molecules toward potential therapeutic agents for myotonic dystrophy type 2 (DM2)</span></div><div class="casAuthors">Wong, Chun-Ho; Fu, Yuan; Ramisetty, Sreenivasa Rao; Baranger, Anne M.; Zimmerman, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8881-8890</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is an incurable neuromuscular disease caused by expanded CCUG repeats that may exhibit toxicity by sequestering the splicing regulator MBNL1.  A series of triaminotriazine- and triaminopyrimidine-based small mols. (ligands 1-3) were designed, synthesized and tested as inhibitors of the MBNL1-CCUG interaction.  Despite the structural similarities of the triaminotriazine and triaminopyrimidine units, the triaminopyrimidine-based ligands bind with low micromolar affinity to CCUG repeats (Kd ∼ 0.1-3.6 μM) whereas the triaminotriazine ligands do not bind CCUG repeats.  Importantly, these simple and small triaminopyrimidine ligands exhibit both strong inhibition (Ki ∼ 2 μM) of the MBNL1-CCUG interaction and high selectivity for CCUG repeats over other RNA targets.  These expts. suggest these compds. are potential lead agents for the treatment of DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4kGlJSj32ZLVg90H21EOLACvtfcHk0lhhTN_860RqRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqt7jJ&md5=01cf07b5db37cc4f72c9ddf18e9dd39d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr415%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.-H.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DRamisetty%26aufirst%3DS.%2BR.%26aulast%3DBaranger%26aufirst%3DA.%2BM.%26aulast%3DZimmerman%26aufirst%3DS.%2BC.%26atitle%3DSelective%2520inhibition%2520of%2520MBNL1%25E2%2580%2593CCUG%2520interaction%2520by%2520small%2520molecules%2520toward%2520potential%2520therapeutic%2520agents%2520for%2520myotonic%2520dystrophy%2520type%25202%2520%2528DM2%2529%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3D8881%26epage%3D8890%26doi%3D10.1093%2Fnar%2Fgkr415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule recognition of RNA base pairs</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">491</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00049</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00049" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cqu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=482-491&author=H.+S.+Haniffauthor=A.+Gravesauthor=M.+D.+Disney&title=Selective+small+molecule+recognition+of+RNA+base+pairs&doi=10.1021%2Facscombsci.8b00049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Recognition of RNA Base Pairs</span></div><div class="casAuthors">Haniff, Hafeez S.; Graves, Amanda; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">482-491</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many types of RNAs exist in the human transcriptome, yet only the bacterial ribosome has been exploited as a small mol. drug target.  Aside from rRNA, other cellular RNAs such as non-coding RNAs have primarily secondary structure and limited tertiary structure.  Within these secondary structures of non-canonically paired and unpaired regions, >50% are base-paired, with most efforts to target these structures focused on looped regions.  A void exists in the availability of small mols. capable of targeting RNA base-pairs.  Here, using chemoinformatics, an RNA focused library enriched for nitrogen-rich heterocycles was developed and tested for binding RNA base-pairs, leading to the identification of 6 selective and previously unknown binders.  While all binders were derivs. of benzimidazoles, those with expanded arom. polycycles bound selectively to AU pairs, while those with flexible urea side-chains bound selectively to GC pairs.  Interestingly, 2 of the 3 selective GC pair binders distinguished between the orientation of 5'GG/3'CC and 5'GC/3'CG pairs.  Furthermore, all 6 mols. showed >50-fold selectivity for RNA vs. DNA.  These studies provide foundational knowledge to better exploit RNA as targets for small mol. chem. probes or lead therapeutics by using modules that target RNA base-pairs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYvpJn4HevkrVg90H21EOLACvtfcHk0lipAcZViSt1HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cqu7zK&md5=8e9941315667412e41ad587a0e02efea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00049%26sid%3Dliteratum%253Aachs%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DGraves%26aufirst%3DA.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSelective%2520small%2520molecule%2520recognition%2520of%2520RNA%2520base%2520pairs%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2018%26volume%3D20%26spage%3D482%26epage%3D491%26doi%3D10.1021%2Facscombsci.8b00049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lipAcZViSt1HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the Rule of Five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+Rule+of+Five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0lipAcZViSt1HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520Rule%2520of%2520Five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the Rule of 5: lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.-J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+Rule+of+5%3A+lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lh9NYovFNIaWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.-J.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520Rule%2520of%25205%253A%2520lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2605</span>– <span class="NLM_lpage">2609</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.bmcl.2016.04.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=27117425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2605-2609&author=M.+G.+Costalesauthor=S.+G.+Rzuczekauthor=M.+D.+Disney&title=Comparison+of+small+molecules+and+oligonucleotides+that+target+a+toxic%2C+non-coding+RNA&doi=10.1016%2Fj.bmcl.2016.04.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span></div><div class="casAuthors">Costales, Matthew G.; Rzuczek, Suzanne G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2605-2609</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Potential RNA targets for chem. probes and therapeutic modalities are pervasive in the transcriptome.  Oligonucleotide-based therapeutics are commonly used to target RNA sequence.  Small mols. are emerging as a modality to target RNA structures selectively, but their development is still in its infancy.  The authors compare the activity of oligonucleotides and several classes of small mols. that target the non-coding r(CCUG) repeat expansion (r(CCUG)exp) that causes myotonic dystrophy type 2 (DM2), an incurable disease that is the second-most common cause of adult onset muscular dystrophy.  Small mol. types investigated include monomers, dimers, and multivalent compds. synthesized on-site by using RNA-templated click chem.  Oligonucleotides investigated include phosphorothioates that cleave their target and vivo-morpholinos that modulate target RNA activity via binding.  The authors show that compds. assembled on-site that recognize structure have the highest potencies amongst small mols. and are similar in potency to a vivo-morpholino modified oligonucleotide that targets sequence.  These studies are likely to impact the design of therapeutic modalities targeting other repeats expansions that cause fragile X syndrome and amyotrophic lateral sclerosis, for example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hWVZ6e6s27Vg90H21EOLACvtfcHk0lh9NYovFNIaWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D&md5=540a6ffdd24b0205cfe7fdf702832174</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DComparison%2520of%2520small%2520molecules%2520and%2520oligonucleotides%2520that%2520target%2520a%2520toxic%252C%2520non-coding%2520RNA%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2605%26epage%3D2609%26doi%3D10.1016%2Fj.bmcl.2016.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobczak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoskins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span> <span> </span><span class="NLM_article-title">Two high-throughput screening assays for aberrant RNA-protein interactions in myotonic dystrophy type 1</span>. <i>Anal. Bioanal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>402</i></span>,  <span class="NLM_fpage">1889</span>– <span class="NLM_lpage">1898</span>, <span class="refDoi"> DOI: 10.1007/s00216-011-5604-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1007%2Fs00216-011-5604-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=22218462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC38XisV2jsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=402&publication_year=2012&pages=1889-1898&author=C.+Z.+Chenauthor=K.+Sobczakauthor=J.+Hoskinsauthor=N.+Southallauthor=J.+J.+Maruganauthor=W.+Zhengauthor=C.+A.+Thorntonauthor=C.+P.+Austin&title=Two+high-throughput+screening+assays+for+aberrant+RNA-protein+interactions+in+myotonic+dystrophy+type+1&doi=10.1007%2Fs00216-011-5604-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Two high-throughput screening assays for aberrant RNA-protein interactions in myotonic dystrophy type 1</span></div><div class="casAuthors">Chen, Catherine Z.; Sobczak, Krzysztof; Hoskins, Jason; Southall, Noel; Marugan, Juan J.; Zheng, Wei; Thornton, Charles A.; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">402</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1889-1898</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1), the most prevalent form of adult muscular dystrophy, is caused by expansion of a CTG repeat in the 3' untranslated region of the DM protein kinase (DMPK) gene.  The pathogenic effects of the CTG expansion arise from the deleterious effects of the mutant transcript.  RNA with expanded CUG tracts alters the activities of several RNA binding proteins, including muscleblind-like 1 (MBNL1).  MBNL1 becomes sequestered in nuclear foci in complex with the expanded CUG-repeat RNA.  The resulting loss of MBNL1 activity causes misregulated alternative splicing of multiple genes, leading to symptoms of DM1.  The binding interaction between MBNL1 and mutant RNA could be a key step in the pathogenesis of DM1 and serves as a potential target for therapeutic intervention.  We have developed two high-throughput screens suitable assays using both homogeneous time-resolved fluorescence energy transfer and AlphaScreen technologies to detect the binding of a C-terminally His-tagged MBNL1 and a biotinylated (CUG)12 RNA.  These assays are homogeneous and successfully miniaturized to 1536-well plate format.  Both assays were validated and show robust signal-to-basal ratios and Z' factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7szE3EThrrLVg90H21EOLACvtfcHk0lh9NYovFNIaWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisV2jsA%253D%253D&md5=9342df23fdd0e60c0c932a816248d003</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs00216-011-5604-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00216-011-5604-0%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DSobczak%26aufirst%3DK.%26aulast%3DHoskins%26aufirst%3DJ.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DThornton%26aufirst%3DC.%2BA.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26atitle%3DTwo%2520high-throughput%2520screening%2520assays%2520for%2520aberrant%2520RNA-protein%2520interactions%2520in%2520myotonic%2520dystrophy%2520type%25201%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2012%26volume%3D402%26spage%3D1889%26epage%3D1898%26doi%3D10.1007%2Fs00216-011-5604-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ule, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ule, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggiu, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Nova regulates brain-specific splicing to shape the synapse</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">844</span>– <span class="NLM_lpage">852</span>, <span class="refDoi"> DOI: 10.1038/ng1610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1038%2Fng1610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=16041372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsFKlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2005&pages=844-852&author=J.+Uleauthor=A.+Uleauthor=J.+Spencerauthor=A.+Williamsauthor=J.+S.+Huauthor=M.+Clineauthor=H.+Wangauthor=T.+Clarkauthor=C.+Fraserauthor=M.+Ruggiu&title=Nova+regulates+brain-specific+splicing+to+shape+the+synapse&doi=10.1038%2Fng1610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Nova regulates brain-specific splicing to shape the synapse</span></div><div class="casAuthors">Ule, Jernej; Ule, Aljaz; Spencer, Joanna; Williams, Alan; Hu, Jing-Shan; Cline, Melissa; Wang, Hui; Clark, Tyson; Fraser, Claire; Ruggiu, Matteo; Zeeberg, Barry R.; Kane, David; Weinstein, John N.; Blume, John; Darnell, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">844-852</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Alternative RNA splicing greatly increases proteome diversity and may thereby contribute to tissue-specific functions.  We carried out genome-wide quant. anal. of alternative splicing using a custom Affymetrix microarray to assess the role of the neuronal splicing factor Nova in the brain.  We used a stringent algorithm to identify 591 exons that were differentially spliced in the brain relative to immune tissues, and 6.6% of these showed major splicing defects in the neocortex of Nova2-/- mice.  We tested 49 exons with the largest predicted Nova-dependent splicing changes and validated all 49 by RT-PCR.  We analyzed the encoded proteins and found that all those with defined brain functions acted in the synapse (34 of 40, including neurotransmitter receptors, cation channels, adhesion and scaffold proteins) or in axon guidance (8 of 40).  Moreover, of the 35 proteins with known interaction partners, 74% (26) interact with each other.  Validating a large set of Nova RNA targets has led us to identify a multi-tiered network in which Nova regulates the exon content of RNAs encoding proteins that interact in the synapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1yFzoRIc5lbVg90H21EOLACvtfcHk0lh9NYovFNIaWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsFKlsLs%253D&md5=a4516efec2e0a43655211586bc3137cb</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fng1610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1610%26sid%3Dliteratum%253Aachs%26aulast%3DUle%26aufirst%3DJ.%26aulast%3DUle%26aufirst%3DA.%26aulast%3DSpencer%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DJ.%2BS.%26aulast%3DCline%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DFraser%26aufirst%3DC.%26aulast%3DRuggiu%26aufirst%3DM.%26atitle%3DNova%2520regulates%2520brain-specific%2520splicing%2520to%2520shape%2520the%2520synapse%26jtitle%3DNat.%2520Genet.%26date%3D2005%26volume%3D37%26spage%3D844%26epage%3D852%26doi%3D10.1038%2Fng1610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeFeo, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glinkerman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise targeted cleavage of a r(CUG) repeat expansion in cells by using a small-molecule–deglycobleomycin conjugate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">849</span>– <span class="NLM_lpage">855</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00036</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltFajsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=849-855&author=A.+J.+Angelbelloauthor=M.+E.+DeFeoauthor=C.+M.+Glinkermanauthor=D.+L.+Bogerauthor=M.+D.+Disney&title=Precise+targeted+cleavage+of+a+r%28CUG%29+repeat+expansion+in+cells+by+using+a+small-molecule%E2%80%93deglycobleomycin+conjugate&doi=10.1021%2Facschembio.0c00036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Precise Targeted Cleavage of a r(CUG) Repeat Expansion in Cells by Using a Small-Molecule-Deglycobleomycin Conjugate</span></div><div class="casAuthors">Angelbello, Alicia J.; DeFeo, Mary E.; Glinkerman, Christopher M.; Boger, Dale L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">849-855</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA repeat expansions cause more than 30 neurol. and neuromuscular diseases with no known cures.  Since repeat expansions operate via diverse pathomechanisms, one potential therapeutic strategy is to rid them from disease-affected cells, using bifunctional small mols. that cleave the aberrant RNA.  Such an approach has been previously implemented for the RNA repeat that causes myotonic dystrophy type 1 [DM1, r(CUG)exp] with Cugamycin, which is a small mol. that selectively binds r(CUG)exp conjugated to a bleomycin A5 cleaving module.  Herein, we demonstrate that, by replacing bleomycin A5 with deglycobleomycin, an analog in which the carbohydrate domain of bleomycin A5 is removed, the selectivity of the resulting small-mol. conjugate (DeglycoCugamycin) was enhanced, while maintaining potent and allele-selective cleavage of r(CUG)exp and rescue of DM1-assocd. defects.  In particular, DeglycoCugamycin did not induce the DNA damage that is obsd. with high concns. (25μM) of Cugamycin, while selectively cleaving the disease-causing allele and improving DM1 defects at 1μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_tmyLI4HRs7Vg90H21EOLACvtfcHk0lim7aB2u4KcNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltFajsrs%253D&md5=aa69cedc0d15fe97e70fd92eb3153535</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00036%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DDeFeo%26aufirst%3DM.%2BE.%26aulast%3DGlinkerman%26aufirst%3DC.%2BM.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520targeted%2520cleavage%2520of%2520a%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520cells%2520by%2520using%2520a%2520small-molecule%25E2%2580%2593deglycobleomycin%2520conjugate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D849%26epage%3D855%26doi%3D10.1021%2Facschembio.0c00036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting the SARS-CoV-2 RNA genome with small molecule binders and ribonuclease targeting chimera (RIBOTAC) degraders</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1721</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.0c00984</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00984" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFeitr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1713-1721&author=H.+S.+Haniffauthor=Y.+Tongauthor=X.+Liuauthor=J.+L.+Chenauthor=B.+M.+Sureshauthor=R.+J.+Andrewsauthor=J.+M.+Petersonauthor=C.+A.+O%E2%80%99Learyauthor=R.+I.+Benhamouauthor=W.+N.+Moss&title=Targeting+the+SARS-CoV-2+RNA+genome+with+small+molecule+binders+and+ribonuclease+targeting+chimera+%28RIBOTAC%29+degraders&doi=10.1021%2Facscentsci.0c00984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the SARS-CoV-2 RNA Genome with Small Molecule Binders and Ribonuclease Targeting Chimera (RIBOTAC) Degraders</span></div><div class="casAuthors">Haniff, Hafeez S.; Tong, Yuquan; Liu, Xiaohui; Chen, Jonathan L.; Suresh, Blessy M.; Andrews, Ryan J.; Peterson, Jake M.; O'Leary, Collin A.; Benhamou, Raphael I.; Moss, Walter N.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1713-1721</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">COVID-19 is a global pandemic, thus requiring multiple strategies to develop modalities against it.  We designed multiple bioactive small mols. that target a functional structure within the SARS-CoV-2's RNA genome, the causative agent of COVID-19.  An anal. to characterize the structure of the RNA genome provided a revised model of the SARS-CoV-2 frameshifting element including its attenuator hairpin.  By studying an RNA-focused small mol. collection, we identified a drug-like small mol. (I) that avidly binds to the revised attenuator hairpin structure with a Kd of 11 nM.  I stabilizes the hairpin's folded state and impairs frameshifting in cells.  I was further elaborated into a RNase targeting chimera (RIBOTAC) to recruit a cellular RNase to destroy the viral genome (C5-RIBOTAC) and into a covalent mol. (C5-Chem-CLIP) that validated direct target engagement and demonstrated its specificity for the viral RNA as compared to highly expressed host mRNAs.  The RIBOTAC lead optimization strategy improved the bioactivity of the compd. ≥10-fold.  Collectively, these studies demonstrate that the SARS-CoV-2 RNA genome should be considered druggable.  Covid-19 is a global pandemic caused by the SARS-CoV-2 virus, which has an RNA genome.  We designed small mols. that bind or degrade a structured, functional element in the virus to inhibit a process essential for viral propagation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwLN6QE_grALVg90H21EOLACvtfcHk0lim7aB2u4KcNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFeitr3O&md5=af035b9ac467ed21f0f69d54147284e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00984%26sid%3Dliteratum%253Aachs%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DSuresh%26aufirst%3DB.%2BM.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DPeterson%26aufirst%3DJ.%2BM.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DC.%2BA.%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26atitle%3DTargeting%2520the%2520SARS-CoV-2%2520RNA%2520genome%2520with%2520small%2520molecule%2520binders%2520and%2520ribonuclease%2520targeting%2520chimera%2520%2528RIBOTAC%2529%2520degraders%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2020%26volume%3D6%26spage%3D1713%26epage%3D1721%26doi%3D10.1021%2Facscentsci.0c00984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeJesus-Hernandez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackenzie, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeve, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamson, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2011.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.neuron.2011.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=21944778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A280%3ADC%252BC3MbjtFOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2011&pages=245-256&author=M.+DeJesus-Hernandezauthor=I.+R.+Mackenzieauthor=B.+F.+Boeveauthor=A.+L.+Boxerauthor=M.+Bakerauthor=N.+J.+Rutherfordauthor=A.+M.+Nicholsonauthor=N.+A.+Finchauthor=H.+Flynnauthor=J.+Adamson&title=Expanded+GGGGCC+hexanucleotide+repeat+in+noncoding+region+of+C9ORF72+causes+chromosome+9p-linked+FTD+and+ALS&doi=10.1016%2Fj.neuron.2011.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS</span></div><div class="casAuthors">DeJesus-Hernandez Mariely; Mackenzie Ian R; Boeve Bradley F; Boxer Adam L; Baker Matt; Rutherford Nicola J; Nicholson Alexandra M; Finch NiCole A; Flynn Heather; Adamson Jennifer; Kouri Naomi; Wojtas Aleksandra; Sengdy Pheth; Hsiung Ging-Yuek R; Karydas Anna; Seeley William W; Josephs Keith A; Coppola Giovanni; Geschwind Daniel H; Wszolek Zbigniew K; Feldman Howard; Knopman David S; Petersen Ronald C; Miller Bruce L; Dickson Dennis W; Boylan Kevin B; Graff-Radford Neill R; Rademakers Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">245-56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several families have been reported with autosomal-dominant frontotemporal dementia (FTD) and amyotrophic lateral sclerosis (ALS), genetically linked to chromosome 9p21.  Here, we report an expansion of a noncoding GGGGCC hexanucleotide repeat in the gene C9ORF72 that is strongly associated with disease in a large FTD/ALS kindred, previously reported to be conclusively linked to chromosome 9p.  This same repeat expansion was identified in the majority of our families with a combined FTD/ALS phenotype and TDP-43-based pathology.  Analysis of extended clinical series found the C9ORF72 repeat expansion to be the most common genetic abnormality in both familial FTD (11.7%) and familial ALS (23.5%).  The repeat expansion leads to the loss of one alternatively spliced C9ORF72 transcript and to formation of nuclear RNA foci, suggesting multiple disease mechanisms.  Our findings indicate that repeat expansion in C9ORF72 is a major cause of both FTD and ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8x2wRTCeVlWlSCSgBhwJFfW6udTcc2ebMRinGgrx99Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbjtFOrsg%253D%253D&md5=f5918aa4c4021cb33e6d6414ed0c4b44</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2011.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2011.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DDeJesus-Hernandez%26aufirst%3DM.%26aulast%3DMackenzie%26aufirst%3DI.%2BR.%26aulast%3DBoeve%26aufirst%3DB.%2BF.%26aulast%3DBoxer%26aufirst%3DA.%2BL.%26aulast%3DBaker%26aufirst%3DM.%26aulast%3DRutherford%26aufirst%3DN.%2BJ.%26aulast%3DNicholson%26aufirst%3DA.%2BM.%26aulast%3DFinch%26aufirst%3DN.%2BA.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DAdamson%26aufirst%3DJ.%26atitle%3DExpanded%2520GGGGCC%2520hexanucleotide%2520repeat%2520in%2520noncoding%2520region%2520of%2520C9ORF72%2520causes%2520chromosome%25209p-linked%2520FTD%2520and%2520ALS%26jtitle%3DNeuron%26date%3D2011%26volume%3D72%26spage%3D245%26epage%3D256%26doi%3D10.1016%2Fj.neuron.2011.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevarello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulpetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, M.</span></span> <span> </span><span class="NLM_article-title">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1023/A:1008354229396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1023%2FA%3A1008354229396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11061229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=65-68&author=C.+Dalvitauthor=P.+Pevarelloauthor=M.+Tatoauthor=M.+Veronesiauthor=A.+Vulpettiauthor=M.+Sundstrom&title=Identification+of+compounds+with+binding+affinity+to+proteins+via+magnetization+transfer+from+bulk+water&doi=10.1023%2FA%3A1008354229396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span></div><div class="casAuthors">Dalvit, Claudio; Pevarello, Paolo; Tato, Marco; Veronesi, Marina; Vulpetti, Anna; Sundstrom, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-68</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A powerful screening by NMR methodol. (WaterLOGSY), based on transfer of magnetization from bulk water, for the identification of compds. that interact with target biomols. (proteins, RNA and DNA fragments) is described.  The method exploits efficiently the large reservoir of H2O magnetization.  The high sensitivity of the technique reduces the amt. of biomol. and ligands needed for the screening, which constitutes an important requirement for high throughput screening by NMR of large libraries of compds.  Application of the method to a compd. mixt. against the cyclin-dependent kinase 2 (cdk2) protein is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8Jo0PHYdALVg90H21EOLACvtfcHk0ljEzYFd1rPl7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D&md5=e0fd0e5815f9bd20ed09d8033882feb6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008354229396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008354229396%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DPevarello%26aufirst%3DP.%26aulast%3DTato%26aufirst%3DM.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DSundstrom%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520compounds%2520with%2520binding%2520affinity%2520to%2520proteins%2520via%2520magnetization%2520transfer%2520from%2520bulk%2520water%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2000%26volume%3D18%26spage%3D65%26epage%3D68%26doi%3D10.1023%2FA%3A1008354229396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuister, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">NMRPipe: A multidimensional spectral processing system based on UNIX pipes</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+A+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0ljEzYFd1rPl7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520A%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span>; <span class="NLM_string-name">Kneller, D. G.</span></span> <i>SPARKY, NMR Assignment and Integration Software, 3</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=T.+D.+Goddard&author=D.+G.+Kneller&title=SPARKY%2C+NMR+Assignment+and+Integration+Software%2C+3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26btitle%3DSPARKY%252C%2520NMR%2520Assignment%2520and%2520Integration%2520Software%252C%25203%26pub%3DUniversity%2520of%2520California%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. r(CCUG)<sup>exp</sup>-mediated defects in DM2. (A) DM2 is caused by r(CCUG)<sup>exp</sup> in CNBP intron 1, which folds into a structure with repeating 2 × 2 nucleotide internal loops that sequester regulatory proteins involved in pre-mRNA splicing such as MBNL1. (B) Sequestration of MBNL1 by r(CCUG)<sup>exp</sup> results in splicing defects in its native pre-mRNAs substrates. For example, <i>IR</i> exon 11 is included ∼70% of the time in healthy cells, but only ∼50% in DM2-affected cells. (C) r(CCUG)<sup>exp</sup> also causes intron retention where intron 1 is aberrantly retained in CNBP mature RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Small molecules that inhibit r(CCUG)<sup>exp</sup>–MBNL1 complex formation <i>in vitro</i>. (A) Characteristics of an RNA-focused small molecule library compared to characteristics of known drugs in DrugBank. (B) IC<sub>50</sub> values of hit compounds from the RNA-focused library screen for disrupting the r(CCUG)<sub>12</sub>–MBNL1 complex (<i>n</i> = 2). Molecules in red have IC<sub>50</sub>s < 50 μM. (C) Chemical structures of compounds with IC<sub>50</sub>s < 50 μM. (D) Rescue of the <i>IR</i> exon 11 splicing defect by <b>3</b>, <b>4</b>, <b>5</b>, and <b>7</b> in DM2 patient-derived fibroblasts (<i>n</i> = 2). Error bars represent standard deviation (SD) for all panels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Lead optimization of <b>7</b> and activity of analogues <i>in vitro</i> and in DM2 patient-derived fibroblasts. (A) Synthetic optimization scheme of compound <b>7</b>. (B) Structures of derivatives of <b>7</b> where “R” indicates the position(s) that were varied. IC<sub>50</sub> values for disrupting the r(CCUG)<sub>12</sub>–MBNL1 complex for each derivative (<i>n</i> = 3). (C) Structures of compounds with IC<sub>50</sub>s < 5 μM. (D) Rescue of <i>IR</i> exon 11 mis-splicing by nontoxic compounds <b>63</b>, <b>79</b>, and <b>80</b> in DM2 patient-derived fibroblasts (<i>n</i> = 3). ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, as determined by a one-way analysis of variance (ANOVA) relative to untreated (“0”). (E) Binding affinity curve of <b>63</b> and r(CCUG)<sub>12</sub> (<i>K</i><sub>d</sub> = 192 ± 20 nM). (F) Binding affinity curve of <b>63</b> and a base pair control RNA (<i>K</i><sub>d</sub> > 20 μM). Error bars represent standard deviation (SD) for all panels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/medium/jm1c00414_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>63</b> rescues disease-associated defects in DM2 patient-derived fibroblasts. (A) Representative RNA FISH and MBNL1 immunofluorescence images of r(CCUG)<sup>exp</sup>–MBNL1 foci. (B) Quantification of the number of nuclear foci/cell (<i>n</i> = 3, 40 nuclei counted/replicate). *** <i>P</i> < 0.001, as determined by a Student’s <i>t</i>-test. (C) Analysis of CNBP intron 1 levels upon treatment with <b>63</b> via RT-qPCR (<i>n</i> = 3).* <i>P</i> < 0.05, *** <i>P</i> < 0.001, as determined by a one-way ANOVA. (D) Analysis of CNBP mature mRNA levels upon treatment with <b>63</b> via RT-qPCR (<i>n</i> = 3). Error bars represent standard deviation (SD) for all panels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00414/20210616/images/large/jm1c00414_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00414&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i74">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47100" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47100" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 31 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulson, H.</span></span> <span> </span><span class="NLM_article-title">Repeat expansion diseases</span>.  <i>Handbook of Clinical Neurology</i>; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2018</span>; Vol.  <span class="NLM_volume">147</span>, pp  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">123</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=105-123&author=H.+Paulson&title=Handbook+of+Clinical+Neurology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPaulson%26aufirst%3DH.%26atitle%3DRepeat%2520expansion%2520diseases%26btitle%3DHandbook%2520of%2520Clinical%2520Neurology%26pub%3DElsevier%26date%3D2018%26volume%3D147%26spage%3D105%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hale, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, J. A.</span></span> <span> </span><span class="NLM_article-title">Repeat-associated RNA structure and aberrant splicing</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1862</i></span>,  <span class="NLM_fpage">194405</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2019.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.bbagrm.2019.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=31323433" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVajtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1862&publication_year=2019&pages=194405&author=M.+A.+Haleauthor=N.+E.+Johnsonauthor=J.+A.+Berglund&title=Repeat-associated+RNA+structure+and+aberrant+splicing&doi=10.1016%2Fj.bbagrm.2019.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Repeat-associated RNA structure and aberrant splicing</span></div><div class="casAuthors">Hale, Melissa A.; Johnson, Nicholas E.; Berglund, J. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1862</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">194405</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Over 30 hereditary disorders attributed to the expansion of microsatellite repeats have been identified.  Despite variant nucleotide content, no. of consecutive repeats, and different locations in the genome, many of these diseases have pathogenic RNA gain-of-function mechanisms.  The repeat-contg. RNAs can form structures in vitro predicted to contribute to the disease through assembly of intracellular RNA aggregates termed foci.  The expanded repeat RNAs within these foci sequester RNA binding proteins (RBPs) with important roles in the regulation of RNA metab., most notably alternative splicing (AS).  These deleterious interactions lead to downstream alterations in transcriptome-wide AS directly linked with disease symptoms.  This review summarizes existing knowledge about the assocn. between the repeat RNA structures and RBPs as well as the resulting aberrant AS patterns, specifically in the context of myotonic dystrophy.  The connection between toxic, structured RNAs and dysregulation of AS in other repeat expansion diseases is also discussed.  This article is part of a Special Issue entitled: RNA structure and splicing regulation edited by Francisco Baralle, Ravindra Singh and Stefan Stamm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5rGsBoLD24bVg90H21EOLACvtfcHk0lhsXuNG-fx0sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVajtr7P&md5=15361fe11a4e56e887948d4f12d505f7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2019.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2019.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DHale%26aufirst%3DM.%2BA.%26aulast%3DJohnson%26aufirst%3DN.%2BE.%26aulast%3DBerglund%26aufirst%3DJ.%2BA.%26atitle%3DRepeat-associated%2520RNA%2520structure%2520and%2520aberrant%2520splicing%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2019%26volume%3D1862%26spage%3D194405%26doi%3D10.1016%2Fj.bbagrm.2019.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattamatta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranum, L. P. W.</span></span> <span> </span><span class="NLM_article-title">Repeat-associated non-ATG (RAN) translation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">16127</span>– <span class="NLM_lpage">16141</span>, <span class="refDoi"> DOI: 10.1074/jbc.R118.003237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1074%2Fjbc.R118.003237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=30213863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGhsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=16127-16141&author=J.+D.+Clearyauthor=A.+Pattamattaauthor=L.+P.+W.+Ranum&title=Repeat-associated+non-ATG+%28RAN%29+translation&doi=10.1074%2Fjbc.R118.003237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Repeat-associated non-ATG (RAN) translation</span></div><div class="casAuthors">Cleary, John Douglas; Pattamatta, Amrutha; Ranum, Laura P. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">16127-16141</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Microsatellite expansions cause more than 40 neurol. disorders, including Huntington's disease, myotonic dystrophy, and C9ORF72 amyotrophic lateral sclerosis/frontotemporal dementia (ALS/FTD).  These repeat expansion mutations can produce repeat-assocd. non-ATG (RAN) proteins in all three reading frames, which accumulate in disease-relevant tissues.  There has been considerable interest in RAN protein products and their downstream consequences, particularly for the dipeptide proteins found in C9ORF72 ALS/FTD.  Understanding how RAN translation occurs, what cellular factors contribute to RAN protein accumulation, and how these proteins contribute to disease should lead to a better understanding of the basic mechanisms of gene expression and human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKEAHr0bJIWLVg90H21EOLACvtfcHk0lhsXuNG-fx0sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGhsr%252FO&md5=38418351501e2ef2176a031fbafeff74</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R118.003237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R118.003237%26sid%3Dliteratum%253Aachs%26aulast%3DCleary%26aufirst%3DJ.%2BD.%26aulast%3DPattamatta%26aufirst%3DA.%26aulast%3DRanum%26aufirst%3DL.%2BP.%2BW.%26atitle%3DRepeat-associated%2520non-ATG%2520%2528RAN%2529%2520translation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D16127%26epage%3D16141%26doi%3D10.1074%2Fjbc.R118.003237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sznajder, Ł. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrell, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleary, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nutter, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobczak, K.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Intron retention induced by microsatellite expansions as a disease biomarker</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4234</span>, <span class="refDoi"> DOI: 10.1073/pnas.1716617115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1073%2Fpnas.1716617115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=29610297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Glur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=4234&author=%C5%81.+J.+Sznajderauthor=J.+D.+Thomasauthor=E.+M.+Carrellauthor=T.+Reidauthor=K.+N.+McFarlandauthor=J.+D.+Clearyauthor=R.+Oliveiraauthor=C.+A.+Nutterauthor=K.+Bhattauthor=K.+Sobczak&title=Intron+retention+induced+by+microsatellite+expansions+as+a+disease+biomarker&doi=10.1073%2Fpnas.1716617115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Intron retention induced by microsatellite expansions as a disease biomarker</span></div><div class="casAuthors">Sznajder, Lukasz J.; Thomas, James D.; Carrell, Ellie M.; Reid, Tammy; McFarland, Karen N.; Cleary, John D.; Oliveira, Ruan; Nutter, Curtis A.; Bhatt, Kirti; Sobczak, Krzysztof; Ashizawa, Tetsuo; Thornton, Charles A.; Ranum, Laura P. W.; Swanson, Maurice S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4234-4239</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Expansions of simple sequence repeats, or microsatellites, have been linked to ∼30 neurol.-neuromuscular diseases.  While these expansions occur in coding and noncoding regions, microsatellite sequence and repeat length diversity is more prominent in introns with eight different trinucleotide to hexanucleotide repeats, causing hereditary diseases such as myotonic dystrophy type 2 (DM2), Fuchs endothelial corneal dystrophy (FECD), and C9orf72 amyotrophic lateral sclerosis and frontotemporal dementia (C9-ALS/FTD).  Here, we test the hypothesis that these GC-rich intronic microsatellite expansions selectively trigger host intron retention (IR).  Using DM2, FECD, and C9-ALS/FTD as examples, we demonstrate that retention is readily detectable in affected tissues and peripheral blood lymphocytes and conclude that IR screening constitutes a rapid and inexpensive biomarker for intronic repeat expansion disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpN1s3EDKLhlrVg90H21EOLACvtfcHk0ljLhnUAOBLB_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Glur%252FM&md5=2d57239bf60d4d13a77c05b52d4de8cd</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1716617115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1716617115%26sid%3Dliteratum%253Aachs%26aulast%3DSznajder%26aufirst%3D%25C5%2581.%2BJ.%26aulast%3DThomas%26aufirst%3DJ.%2BD.%26aulast%3DCarrell%26aufirst%3DE.%2BM.%26aulast%3DReid%26aufirst%3DT.%26aulast%3DMcFarland%26aufirst%3DK.%2BN.%26aulast%3DCleary%26aufirst%3DJ.%2BD.%26aulast%3DOliveira%26aufirst%3DR.%26aulast%3DNutter%26aufirst%3DC.%2BA.%26aulast%3DBhatt%26aufirst%3DK.%26aulast%3DSobczak%26aufirst%3DK.%26atitle%3DIntron%2520retention%2520induced%2520by%2520microsatellite%2520expansions%2520as%2520a%2520disease%2520biomarker%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26spage%3D4234%26doi%3D10.1073%2Fpnas.1716617115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liquori, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moseley, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kress, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranum, L. P. W.</span></span> <span> </span><span class="NLM_article-title">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">864</span>, <span class="refDoi"> DOI: 10.1126/science.1062125</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1126%2Fscience.1062125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11486088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=864&author=C.+L.+Liquoriauthor=K.+Rickerauthor=M.+L.+Moseleyauthor=J.+F.+Jacobsenauthor=W.+Kressauthor=S.+L.+Naylorauthor=J.+W.+Dayauthor=L.+P.+W.+Ranum&title=Myotonic+dystrophy+type+2+caused+by+a+CCTG+expansion+in+intron+1+of+ZNF9&doi=10.1126%2Fscience.1062125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Myotonic dystrophy type 2 caused by a CCTG expansion in intron 1 of ZNF9</span></div><div class="casAuthors">Liquori, Christina L.; Ricker, Kenneth; Moseley, Melinda L.; Jacobsen, Jennifer F.; Kress, Wolfram; Naylor, Susan L.; Day, John W.; Ranum, Laura P. W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5531</span>),
    <span class="NLM_cas:pages">864-867</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Myotonic dystrophy (DM), the most common form of muscular dystrophy in adults, can be caused by a mutation on either chromosome 19q13 (DM1) or 3q21 (DMZ/PROMM).  DM1 is caused by a CTG expansion in the 3' untranslated region of the dystrophia myotonica-protein kinase gene (DMPK).  Several mechanisms have been invoked to explain how this mutation, which does not alter the protein-coding portion of a gene, causes the specific constellation of clin. features characteristic of DM.  We now report that DMZ is caused by a CCTG expansion (mean ∼5000 repeats) located in intron 1 of the zinc finger protein 9 (ZNF9) gene.  Parallels between these mutations indicate that microsatellite expansions in RNA can be pathogenic and cause the multisystemic features of DM1 and DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeKIAHv1r8_bVg90H21EOLACvtfcHk0ljLhnUAOBLB_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlvVKktbc%253D&md5=812d852f480461b722162114fcb906f2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1062125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1062125%26sid%3Dliteratum%253Aachs%26aulast%3DLiquori%26aufirst%3DC.%2BL.%26aulast%3DRicker%26aufirst%3DK.%26aulast%3DMoseley%26aufirst%3DM.%2BL.%26aulast%3DJacobsen%26aufirst%3DJ.%2BF.%26aulast%3DKress%26aufirst%3DW.%26aulast%3DNaylor%26aufirst%3DS.%2BL.%26aulast%3DDay%26aufirst%3DJ.%2BW.%26aulast%3DRanum%26aufirst%3DL.%2BP.%2BW.%26atitle%3DMyotonic%2520dystrophy%2520type%25202%2520caused%2520by%2520a%2520CCTG%2520expansion%2520in%2520intron%25201%2520of%2520ZNF9%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D864%26doi%3D10.1126%2Fscience.1062125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schatz, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Structure of the myotonic dystrophy type 2 RNA and designed small molecules that reduce toxicity</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1021/cb4007387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4007387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2ku7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=538-550&author=J.+L.+Childs-Disneyauthor=I.+Yildirimauthor=H.+Parkauthor=J.+R.+Lohmanauthor=L.+Guanauthor=T.+Tranauthor=P.+Sarkarauthor=G.+C.+Schatzauthor=M.+D.+Disney&title=Structure+of+the+myotonic+dystrophy+type+2+RNA+and+designed+small+molecules+that+reduce+toxicity&doi=10.1021%2Fcb4007387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Myotonic Dystrophy Type 2 RNA and Designed Small Molecules That Reduce Toxicity</span></div><div class="casAuthors">Childs-Disney, Jessica L.; Yildirim, Ilyas; Park, Ha Jeung; Lohman, Jeremy R.; Guan, Lirui; Tran, Tuan; Sarkar, Partha; Schatz, George C.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">538-550</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is an incurable neuromuscular disorder caused by a r-(CCUG) expansion (r-(CCUG)exp) that folds into an extended hairpin with periodically repeating 2 × 2 nucleotide internal loops (5'CCUG/3'GUCC).  We designed multivalent compds. that improve DM2-assocd. defects using information about RNA-small mol. interactions.  We also report the first crystal structure of r-(CCUG) repeats refined to 2.35 Å.  Structural anal. of the three 5'CCUG/3'GUCC repeat internal loops (L) reveals that the CU pairs in L1 are each stabilized by one hydrogen bond and a water-mediated hydrogen bond, while CU pairs in L2 and L3 are stabilized by two hydrogen bonds.  Mol. dynamics (MD) simulations reveal that the CU pairs are dynamic and stabilized by Na+ and water mols.  MD simulations of the binding of the small mol. to r-(CCUG) repeats reveal that the lowest free energy binding mode occurs via the major groove, in which one C residue is unstacked and the cross-strand nucleotides are displaced.  Moreover, we modeled the binding of our dimeric compd. to two 5'CCUG/3'GUCC motifs, which shows that the scaffold on which the RNA-binding modules are displayed provides an optimal distance to span two adjacent loops.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpShDc5fRNy6bVg90H21EOLACvtfcHk0ljLhnUAOBLB_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2ku7zK&md5=8696f289e41fb8c984c167e3e64aa311</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fcb4007387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4007387%26sid%3Dliteratum%253Aachs%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLohman%26aufirst%3DJ.%2BR.%26aulast%3DGuan%26aufirst%3DL.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DSarkar%26aufirst%3DP.%26aulast%3DSchatz%26aufirst%3DG.%2BC.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DStructure%2520of%2520the%2520myotonic%2520dystrophy%2520type%25202%2520RNA%2520and%2520designed%2520small%2520molecules%2520that%2520reduce%2520toxicity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D538%26epage%3D550%26doi%3D10.1021%2Fcb4007387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fardaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamshere, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brook, J. D.</span></span> <span> </span><span class="NLM_article-title">Three proteins, MBNL, MBLL and MBXL, co-localize in vivo with nuclear foci of expanded-repeat transcripts in DM1 and DM2 cells</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">814</span>, <span class="refDoi"> DOI: 10.1093/hmg/11.7.805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1093%2Fhmg%2F11.7.805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11929853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjt1GrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=805-814&author=M.+Fardaeiauthor=M.+T.+Rogersauthor=H.+M.+Thorpeauthor=K.+Larkinauthor=M.+G.+Hamshereauthor=P.+S.+Harperauthor=J.+D.+Brook&title=Three+proteins%2C+MBNL%2C+MBLL+and+MBXL%2C+co-localize+in+vivo+with+nuclear+foci+of+expanded-repeat+transcripts+in+DM1+and+DM2+cells&doi=10.1093%2Fhmg%2F11.7.805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Three proteins, MBNL, MBLL and MBXL, co-localize in vivo with nuclear foci of expanded-repeat transcripts in DM1 and DM2 cells</span></div><div class="casAuthors">Fardaei, Majid; Rogers, Mark T.; Thorpe, Helena M.; Larkin, Kenneth; Hamshere, Marion G.; Harper, Peter S.; Brook, J. David</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-814</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy is a complex neuromuscular disorder assocd. with DNA expansion mutations in two different genes.  In DM1 a CTG repeat in the 3'-untranslated region of DMPK is expanded, whereas in DM2 an intronic CCTG expansion occurs in the gene ZNF9.  Transcripts contg. expanded repeats form foci in the nuclei of DM1 and DM2 cells.  Recent work using antibodies has shown that proteins related to Drosophila muscleblind co-localize with repeat foci in DM1 and DM2 cells.  We show that rather than there being a single human muscleblind gene producing multiple proteins through alternative splicing, there are in fact three different muscleblind genes, MBNL, MBLL and MBXL, which map to chromosomes 3, 13 and X, resp., and which show extensive alternative splicing.  Two of the genes, MBNL and MBLL, are expressed in many adult tissues whereas MBXL is expressed predominantly in the placenta.  Green fluorescent protein-tagged versions of MBNL, MBLL and MBXL co-localize with nuclear foci in DM1 and DM2 cells, suggesting that all three proteins may play a role in DM pathophysiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDPPadt35dOrVg90H21EOLACvtfcHk0lgm5e4orVw2jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjt1GrsbY%253D&md5=c02d0ee9b6f306e26fd004493f0b6cff</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F11.7.805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F11.7.805%26sid%3Dliteratum%253Aachs%26aulast%3DFardaei%26aufirst%3DM.%26aulast%3DRogers%26aufirst%3DM.%2BT.%26aulast%3DThorpe%26aufirst%3DH.%2BM.%26aulast%3DLarkin%26aufirst%3DK.%26aulast%3DHamshere%26aufirst%3DM.%2BG.%26aulast%3DHarper%26aufirst%3DP.%2BS.%26aulast%3DBrook%26aufirst%3DJ.%2BD.%26atitle%3DThree%2520proteins%252C%2520MBNL%252C%2520MBLL%2520and%2520MBXL%252C%2520co-localize%2520in%2520vivo%2520with%2520nuclear%2520foci%2520of%2520expanded-repeat%2520transcripts%2520in%2520DM1%2520and%2520DM2%2520cells%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2002%26volume%3D11%26spage%3D805%26epage%3D814%26doi%3D10.1093%2Fhmg%2F11.7.805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mankodi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbinati, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Q. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moxley, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansone, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krym, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schalling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, C. A.</span></span> <span> </span><span class="NLM_article-title">Muscleblind localizes to nuclear foci of aberrant RNA in myotonic dystrophy types 1 and 2</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2165</span>– <span class="NLM_lpage">2170</span>, <span class="refDoi"> DOI: 10.1093/hmg/10.19.2165</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1093%2Fhmg%2F10.19.2165" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11590133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsl2ktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2001&pages=2165-2170&author=A.+Mankodiauthor=C.+R.+Urbinatiauthor=Q.+P.+Yuanauthor=R.+T.+Moxleyauthor=V.+Sansoneauthor=M.+Krymauthor=D.+Hendersonauthor=M.+Schallingauthor=M.+S.+Swansonauthor=C.+A.+Thornton&title=Muscleblind+localizes+to+nuclear+foci+of+aberrant+RNA+in+myotonic+dystrophy+types+1+and+2&doi=10.1093%2Fhmg%2F10.19.2165"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Muscleblind localizes to nuclear foci of aberrant RNA in myotonic dystrophy types 1 and 2</span></div><div class="casAuthors">Mankodi, Ami; Urbinati, Carl R.; Yuan, Qiu-Ping; Moxley, Richard T.; Sansone, Valeria; Krym, Matt; Henderson, Donald; Schalling, Martin; Swanson, Maurice S.; Thornton, Charles A.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2165-2170</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The phenotypes in myotonic dystrophy types 1 and 2 (DM1 and DM2) are similar, suggesting a shared pathophysiol. mechanism.  DM1 is caused by expansion of a CTG repeat in the DMPK gene.  Pathogenic effects of this mutation are likely to be mediated, at least in part, by the expanded CUG repeat in mutant mRNA.  The mutant transcripts are retained in the nucleus in multiple discrete foci.  We investigated the possibility that DM2 is also caused by expansion of a CTG repeat or related sequence.  Anal. of DNA by repeat expansion detection methods, and RNA by RNase protection, did not show an expanded CTG or CUG repeat in DM2.  However, hybridization of muscle sections with fluorescence-labeled CAG-repeat oligonucleotides showed nuclear foci in DM2 similar to those seen in DM1.  Nuclear foci were present in all patients with symptomatic DM1 (n = 9) or DM2 (n = 9) but not in any disease controls or healthy subjects (n = 23).  The foci were not seen with CUG- or GUC-repeat probes.  Foci in DM2 were distinguished from DM1 by lower stability of the probe-target duplex, suggesting that a sequence related to the DM1 CUG expansion accumulates in the DM2 nucleus.  Muscleblind proteins, which interact with expanded CUG repeats in vitro, localized to the nuclear foci in both DM1 and DM2.  These results support the idea that nuclear accumulation of mutant RNA is pathogenic in DM1, suggest that a similar disease process occurs in DM2, and point to a role for muscleblind in the pathogenesis of both disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYY4FvhukXZrVg90H21EOLACvtfcHk0lgm5e4orVw2jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsl2ktbg%253D&md5=1d00123bf238f8b70406aec84699a176</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F10.19.2165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F10.19.2165%26sid%3Dliteratum%253Aachs%26aulast%3DMankodi%26aufirst%3DA.%26aulast%3DUrbinati%26aufirst%3DC.%2BR.%26aulast%3DYuan%26aufirst%3DQ.%2BP.%26aulast%3DMoxley%26aufirst%3DR.%2BT.%26aulast%3DSansone%26aufirst%3DV.%26aulast%3DKrym%26aufirst%3DM.%26aulast%3DHenderson%26aufirst%3DD.%26aulast%3DSchalling%26aufirst%3DM.%26aulast%3DSwanson%26aufirst%3DM.%2BS.%26aulast%3DThornton%26aufirst%3DC.%2BA.%26atitle%3DMuscleblind%2520localizes%2520to%2520nuclear%2520foci%2520of%2520aberrant%2520RNA%2520in%2520myotonic%2520dystrophy%2520types%25201%2520and%25202%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2001%26volume%3D10%26spage%3D2165%26epage%3D2170%26doi%3D10.1093%2Fhmg%2F10.19.2165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savkur, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philips, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, T. A.</span></span> <span> </span><span class="NLM_article-title">Aberrant regulation of insulin receptor alternative splicing is associated with insulin resistance in myotonic dystrophy</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1038/ng704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1038%2Fng704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11528389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFGmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=40-47&author=R.+S.+Savkurauthor=A.+V.+Philipsauthor=T.+A.+Cooper&title=Aberrant+regulation+of+insulin+receptor+alternative+splicing+is+associated+with+insulin+resistance+in+myotonic+dystrophy&doi=10.1038%2Fng704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Aberrant regulation of insulin receptor alternative splicing is associated with insulin resistance in myotonic dystrophy</span></div><div class="casAuthors">Savkur, Rajesh S.; Philips, Anne V.; Cooper, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-47</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is caused by a CTG trinucleotide expansion in the 3' untranslated region of the DM protein kinase gene.  People with DM1 have an unusual form of insulin resistance caused by a defect in skeletal muscle.  Here we demonstrate that alternative splicing of the insulin receptor (IR) pre-mRNA is aberrantly regulated in DM1 skeletal muscle tissue, resulting in predominant expression of the lower-signaling nonmuscle isoform (IR-A).  IR-A also predominates in DM1 skeletal muscle cultures, which exhibit a decreased metabolic response to insulin relative to cultures from normal controls.  Steady-state levels of CUG-BP, a regulator of pre-mRNA splicing proposed to mediate some aspects of DM1 pathogenesis, are increased in DM1 skeletal muscle; overexpression of CUG-BP in normal cells induces a switch to IR-A.  The CUG-BP protein mediates this switch through an intronic element located upstream of the alternatively spliced exon 11, and specifically binds within this element in vitro.  These results support a model in which increased expression of a splicing regulator contributes to insulin resistance in DM1 by affecting IR alternative splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVrM-Glm4je7Vg90H21EOLACvtfcHk0lgm5e4orVw2jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFGmsbg%253D&md5=544636c361fe07f81b60beb9d3b3a671</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fng704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng704%26sid%3Dliteratum%253Aachs%26aulast%3DSavkur%26aufirst%3DR.%2BS.%26aulast%3DPhilips%26aufirst%3DA.%2BV.%26aulast%3DCooper%26aufirst%3DT.%2BA.%26atitle%3DAberrant%2520regulation%2520of%2520insulin%2520receptor%2520alternative%2520splicing%2520is%2520associated%2520with%2520insulin%2520resistance%2520in%2520myotonic%2520dystrophy%26jtitle%3DNat.%2520Genet.%26date%3D2001%26volume%3D29%26spage%3D40%26epage%3D47%26doi%3D10.1038%2Fng704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">A toxic RNA catalyzes the cellular synthesis of its own inhibitor, shunting It to endogenous decay pathways</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.chembiol.2020.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=31981476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFShu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=223-231&author=R.+I.+Benhamouauthor=A.+J.+Angelbelloauthor=E.+T.+Wangauthor=M.+D.+Disney&title=A+toxic+RNA+catalyzes+the+cellular+synthesis+of+its+own+inhibitor%2C+shunting+It+to+endogenous+decay+pathways&doi=10.1016%2Fj.chembiol.2020.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A Toxic RNA Catalyzes the Cellular Synthesis of Its Own Inhibitor, Shunting It to Endogenous Decay Pathways</span></div><div class="casAuthors">Benhamou, Raphael I.; Angelbello, Alicia J.; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">223-231.e4</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is a genetically defined disease caused by a toxic expanded repeat of r(CCUG) [r(CCUG)exp], harbored in intron 1 of CCHC-type zinc-finger nucleic acid binding protein (CNBP) pre-mRNA.  This r(CCUG)exp causes toxicity via a gain-of-function mechanism, resulting in three pathol. hallmarks: aggregation into nuclear foci; sequestration of muscleblind-like-1 (MBNL1) protein, leading to splicing defects; and retention of CNBP intron 1.  We studied two types of small mols. with different modes of action, ones that simply bind and ones that are templated by r(CCUG)exp in cells, i.e., the RNA synthesizes its own drug.  Indeed, our studies completed in DM2 patient-derived fibroblasts showed that the compds. disrupt the r(CCUG)exp-MBNL1 complex, reduce intron retention, subjecting the liberated intronic r(CCUG)exp to native decay pathways, and rescue other DM2-assocd. cellular defects.  Importantly, this study shows that small mols. can modulate RNA biol. by shunting toxic transcripts toward native decay pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraOFp22Q9xcLVg90H21EOLACvtfcHk0lg1BbdckTld7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFShu7c%253D&md5=9c8623dd04af21511ce6d3cc4deb2a9e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DA%2520toxic%2520RNA%2520catalyzes%2520the%2520cellular%2520synthesis%2520of%2520its%2520own%2520inhibitor%252C%2520shunting%2520It%2520to%2520endogenous%2520decay%2520pathways%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D223%26epage%3D231%26doi%3D10.1016%2Fj.chembiol.2020.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wieben, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aleff, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tosakulwong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Highsmith, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratz, K. H.</span></span> <span> </span><span class="NLM_article-title">A common trinucleotide repeat expansion within the transcription factor 4 (TCF4, E2-2) gene predicts Fuchs corneal dystrophy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e49083</span>– <span class="NLM_lpage">e49086</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0049083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1371%2Fjournal.pone.0049083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=23185296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVagsrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e49083-e49086&author=E.+D.+Wiebenauthor=R.+A.+Aleffauthor=N.+Tosakulwongauthor=M.+L.+Butzauthor=W.+E.+Highsmithauthor=A.+O.+Edwardsauthor=K.+H.+Baratz&title=A+common+trinucleotide+repeat+expansion+within+the+transcription+factor+4+%28TCF4%2C+E2-2%29+gene+predicts+Fuchs+corneal+dystrophy&doi=10.1371%2Fjournal.pone.0049083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A common trinucleotide repeat expansion within the transcription factor 4 (TCF4, E2-2) gene predicts fuchs corneal dystrophy</span></div><div class="casAuthors">Wieben, Eric D.; Aleff, Ross A.; Tosakulwong, Nirubol; Butz, Malinda L.; Highsmith, W. Edward; Edwards, Albert O.; Baratz, Keith H.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e49083</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Fuchs endothelial corneal dystrophy (FECD) is a common, familial disease of the corneal endothelium and is the leading indication for corneal transplantation.  Variation in the transcription factor 4 (TCF4) gene has been identified as a major contributor to the disease.  We tested for an assocn. between an intronic TGC trinucleotide repeat in TCF4 and FECD by detg. repeat length in 66 affected participants with severe FECD and 63 participants with normal corneas in a 3-stage discovery/replication/validation study.  PCR primers flanking the TGC repeat were used to amplify leukocyte-derived genomic DNA.  Repeat length was detd. by direct sequencing, short tandem repeat (STR) assay and Southern blotting.  Genomic Southern blots were used to evaluate samples for which only a single allele was identified by STR anal.  Compiling data for 3 arms of the study, a TGC repeat length >50 was present in 79% of FECD cases and in 3% of normal controls cases (p<0.001).  Among cases, 52 of 66 (79%) subjects had >50 TGC repeats, 13 (20%) had <40 repeats and 1 (2%) had an intermediate repeat length.  In comparison, only 2 of 63 (3%) unaffected control subjects had >50 repeats, 60 (95%) had <40 repeats and 1 (2%) had an intermediate repeat length.  The repeat length was greater than 1000 in 4 FECD cases.  The sensitivity and specificity of >50 TGC repeats identifying FECD in this patient cohort was 79% and 96%, resp. Expanded TGC repeat was more specific for FECD cases than the previously identified, highly assocd., single nucleotide polymorphism, rs613872 (specificity = 79%).  The TGC trinucleotide repeat expansion in TCF4 is strongly assocd. with FECD, and a repeat length >50 is highly specific for the disease.  This assocn. suggests that trinucleotide expansion may play a pathogenic role in the majority of FECD cases and is a predictor of disease risk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsRjy7lgVHc7Vg90H21EOLACvtfcHk0lg1BbdckTld7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVagsrvF&md5=98afb8d0791873502710d4716679d298</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0049083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0049083%26sid%3Dliteratum%253Aachs%26aulast%3DWieben%26aufirst%3DE.%2BD.%26aulast%3DAleff%26aufirst%3DR.%2BA.%26aulast%3DTosakulwong%26aufirst%3DN.%26aulast%3DButz%26aufirst%3DM.%2BL.%26aulast%3DHighsmith%26aufirst%3DW.%2BE.%26aulast%3DEdwards%26aufirst%3DA.%2BO.%26aulast%3DBaratz%26aufirst%3DK.%2BH.%26atitle%3DA%2520common%2520trinucleotide%2520repeat%2520expansion%2520within%2520the%2520transcription%2520factor%25204%2520%2528TCF4%252C%2520E2-2%2529%2520gene%2520predicts%2520Fuchs%2520corneal%2520dystrophy%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De49083%26epage%3De49086%26doi%3D10.1371%2Fjournal.pone.0049083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jun, A. S.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">A small molecule that binds an RNA repeat expansion stimulates its decay via the exosome complex</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.chembiol.2020.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=33157036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1GqurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2021&pages=34-45&author=A.+J.+Angelbelloauthor=R.+I.+Benhamouauthor=S.+G.+Rzuczekauthor=S.+Choudharyauthor=Z.+Tangauthor=J.+L.+Chenauthor=M.+Royauthor=K.+W.+Wangauthor=I.+Yildirimauthor=A.+S.+Jun&title=A+small+molecule+that+binds+an+RNA+repeat+expansion+stimulates+its+decay+via+the+exosome+complex&doi=10.1016%2Fj.chembiol.2020.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule that Binds an RNA Repeat Expansion Stimulates Its Decay via the Exosome Complex</span></div><div class="casAuthors">Angelbello, Alicia J.; Benhamou, Raphael I.; Rzuczek, Suzanne G.; Choudhary, Shruti; Tang, Zhenzhi; Chen, Jonathan L.; Roy, Madhuparna; Wang, Kye Won; Yildirim, Ilyas; Jun, Albert S.; Thornton, Charles A.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-45.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Many diseases are caused by toxic RNA repeats.  Herein, we designed a lead small mol. that binds the structure of the r(CUG) repeat expansion [r(CUG)exp] that causes myotonic dystrophy type 1 (DM1) and Fuchs endothelial corneal dystrophy (FECD) and rescues disease biol. in patient-derived cells and in vivo.  Interestingly, the compds. downstream effects are different in the two diseases, owing to the location of the repeat expansion.  In DM1, r(CUG)exp is harbored in the 3'untranslated region, and the compd. has no effect on the mRNA's abundance.  In FECD, however, r(CUG)exp is located in an intron, and the small mol. facilitates excision of the intron, which is then degraded by the RNA exosome complex.  Thus, structure-specific, RNA-targeting small mols. can act disease specifically to affect biol., either by disabling the gain-of-function mechanism (DM1) or by stimulating quality control pathways to rid a disease-affected cell of a toxic RNA (FECD).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfxuyDTqH07rVg90H21EOLACvtfcHk0lg1BbdckTld7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1GqurfM&md5=3f5e20b94dbb375234c9931657897d47</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DChoudhary%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DRoy%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DK.%2BW.%26aulast%3DYildirim%26aufirst%3DI.%26aulast%3DJun%26aufirst%3DA.%2BS.%26atitle%3DA%2520small%2520molecule%2520that%2520binds%2520an%2520RNA%2520repeat%2520expansion%2520stimulates%2520its%2520decay%2520via%2520the%2520exosome%2520complex%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2021%26volume%3D28%26spage%3D34%26epage%3D45%26doi%3D10.1016%2Fj.chembiol.2020.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckee, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olafson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise small-molecule cleavage of an r(CUG) repeat expansion in a myotonic dystrophy mouse model</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">7799</span>– <span class="NLM_lpage">7804</span>, <span class="refDoi"> DOI: 10.1073/pnas.1901484116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1073%2Fpnas.1901484116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=30926669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=7799-7804&author=A.+J.+Angelbelloauthor=S.+G.+Rzuczekauthor=K.+K.+Mckeeauthor=J.+L.+Chenauthor=H.+Olafsonauthor=M.+D.+Cameronauthor=W.+N.+Mossauthor=E.+T.+Wangauthor=M.+D.+Disney&title=Precise+small-molecule+cleavage+of+an+r%28CUG%29+repeat+expansion+in+a+myotonic+dystrophy+mouse+model&doi=10.1073%2Fpnas.1901484116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule cleavage of an r(CUG) repeat expansion in a myotonic dystrophy mouse model</span></div><div class="casAuthors">Angelbello, Alicia J.; Rzuczek, Suzanne G.; Mckee, Kendra K.; Chen, Jonathan L.; Olafson, Hailey; Cameron, Michael D.; Moss, Walter N.; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7799-7804</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1) is an incurable neuromuscular disorder caused by an expanded CTG repeat that is transcribed into r(CUG)exp.  The RNA repeat expansion sequesters regulatory proteins such as Muscleblind-like protein 1 (MBNL1), which causes pre-mRNA splicing defects.  The disease-causing r(CUG)exp has been targeted by antisense oligonucleotides, CRISPR-based approaches, and RNA-targeting small mols.  Herein, we describe a designer small mol., Cugamycin, that recognizes the structure of r(CUG)exp and cleaves it in both DM1 patient-derived myotubes and a DM1 mouse model, leaving short repeats of r(CUG) untouched.  In contrast, oligonucleotides that recognize r(CUG) sequence rather than structure cleave both long and short r(CUG)-contg. transcripts.  Transcriptomic, histol., and phenotypic studies demonstrate that Cugamycin broadly and specifically relieves DM1-assocd. defects in vivo without detectable off-targets.  Thus, small mols. that bind and cleave RNA have utility as lead chem. probes and medicines and can selectively target disease-causing RNA structures to broadly improve defects in preclin. animal models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg0da_Q8Pd37Vg90H21EOLACvtfcHk0lhd7N6pe90J9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVeks70%253D&md5=9dfa637039d1b532cbcbbaaccaed40b1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1901484116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1901484116%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DMckee%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DOlafson%26aufirst%3DH.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520small-molecule%2520cleavage%2520of%2520an%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520a%2520myotonic%2520dystrophy%2520mouse%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D7799%26epage%3D7804%26doi%3D10.1073%2Fpnas.1901484116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aikawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeep
Velagapudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. W.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">2406</span>– <span class="NLM_lpage">2411</span>, <span class="refDoi"> DOI: 10.1073/pnas.1914286117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1073%2Fpnas.1914286117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=31964809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=2406-2411&author=M.+G.+Costalesauthor=H.+Aikawaauthor=Y.+Liauthor=J.+L.+Childs-Disneyauthor=D.+Abeggauthor=D.+G.+Hochauthor=S.+Pradeep%0AVelagapudiauthor=Y.+Nakaiauthor=T.+Khanauthor=K.+W.+Wang&title=Small-molecule+targeted+recruitment+of+a+nuclease+to+cleave+an+oncogenic+RNA+in+a+mouse+model+of+metastatic+cancer&doi=10.1073%2Fpnas.1914286117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule targeted recruitment of a nuclease to cleave an oncogenic RNA in a mouse model of metastatic cancer</span></div><div class="casAuthors">Costales, Matthew G.; Aikawa, Haruo; Li, Yue; Childs-Disney, Jessica L.; Abegg, Daniel; Hoch, Dominic G.; Velagapudi, Sai Pradeep; Nakai, Yoshio; Khan, Tanya; Wang, Kye Won; Yildirim, Ilyas; Adibekian, Alexander; Wang, Eric T.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2406-2411</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the area of small mols. interacting with RNA advances, general routes to provide bioactive compds. are needed as ligands can bind RNA avidly to sites that will not affect function.  Small-mol. targeted RNA degrdn. will thus provide a general route to affect RNA biol.  A non-oligonucleotide-contg. compd. was designed from sequence to target the precursor to oncogenic microRNA-21 (pre-miR-21) for enzymic destruction with selectivity that can exceed that for protein-targeted medicines.  The compd. specifically binds the target and contains a heterocycle that recruits and activates a RNase to pre-miR-21 to substoichiometrically effect its cleavage and subsequently impede metastasis of breast cancer to lung in a mouse model.  Transcriptomic and proteomic analyses demonstrate that the compd. is potent and selective, specifically modulating oncogenic pathways.  Thus, small mols. can be designed from sequence to have all of the functional repertoire of oligonucleotides, including inducing enzymic degrdn., and to selectively and potently modulate RNA function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolzufxJ0-LLLVg90H21EOLACvtfcHk0lhd7N6pe90J9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVWisL0%253D&md5=450996be5aadaa9e13acb5c2cfb03c62</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1914286117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1914286117%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DAikawa%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DPradeep%2BVelagapudi%26aufirst%3DS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DKhan%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DK.%2BW.%26atitle%3DSmall-molecule%2520targeted%2520recruitment%2520of%2520a%2520nuclease%2520to%2520cleave%2520an%2520oncogenic%2520RNA%2520in%2520a%2520mouse%2520model%2520of%2520metastatic%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2020%26volume%3D117%26spage%3D2406%26epage%3D2411%26doi%3D10.1073%2Fpnas.1914286117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Targeting RNA with small molecules to capture opportunities at the intersection of chemistry, biology, and medicine</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">6776</span>– <span class="NLM_lpage">6790</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlslaqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=6776-6790&author=M.+D.+Disney&title=Targeting+RNA+with+small+molecules+to+capture+opportunities+at+the+intersection+of+chemistry%2C+biology%2C+and+medicine&doi=10.1021%2Fjacs.8b13419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting RNA with Small Molecules To Capture Opportunities at the Intersection of Chemistry, Biology, and Medicine</span></div><div class="casAuthors">Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6776-6790</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The biol. of healthy and diseased cells is often mediated by RNA structures, desirable targets for small mol. chem. probes and lead medicines.  Although structured regions are found throughout the transcriptome, some even with demonstrated functionality, human RNAs are considered recalcitrant to small mol. targeting.  However, targeting structured regions with small mols. provides an important alternative to oligonucleotides that target sequence.  In this Perspective, the authors describe challenges and progress in developing small mols. interacting with RNA (SMIRNAs) to capture their significant opportunities at the intersection of chem., biol., and medicine.  Key to establishing a new paradigm in chem. biol. and medicine is the development of methods to obtain, preferably by design, bioactive compds. that modulate RNA targets and the companion methods that validate their direct effects in cells and preclin. models.  While difficult, demonstration of direct target engagement in the complex cellular milieu, along with methods to establish modes of action, are required to push this field forward The authors also describe frameworks for accelerated advancements in this burgeoning area, their implications, key new technologies for development of SMIRNAs, and milestones that have led to broader acceptance of RNA as a small mol. druggable target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9jiNQMcQjbLVg90H21EOLACvtfcHk0lhd7N6pe90J9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlslaqtL4%253D&md5=bcf0d58b321ee12c921650d3b1bc388c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13419%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DTargeting%2520RNA%2520with%2520small%2520molecules%2520to%2520capture%2520opportunities%2520at%2520the%2520intersection%2520of%2520chemistry%252C%2520biology%252C%2520and%2520medicine%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D6776%26epage%3D6790%26doi%3D10.1021%2Fjacs.8b13419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Structure-specific cleavage of an RNA repeat expansion with a dimeric small molecule is advantageous over sequence-specific recognition by an oligonucleotide</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">493</span>, <span class="refDoi"> DOI: 10.1021/acschembio.9b00958</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.9b00958" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtlegtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=485-493&author=R.+I.+Benhamouauthor=A.+J.+Angelbelloauthor=R.+J.+Andrewsauthor=E.+T.+Wangauthor=W.+N.+Mossauthor=M.+D.+Disney&title=Structure-specific+cleavage+of+an+RNA+repeat+expansion+with+a+dimeric+small+molecule+is+advantageous+over+sequence-specific+recognition+by+an+oligonucleotide&doi=10.1021%2Facschembio.9b00958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Specific Cleavage of an RNA Repeat Expansion with a Dimeric Small Molecule Is Advantageous over Sequence-Specific Recognition by an Oligonucleotide</span></div><div class="casAuthors">Benhamou, Raphael I.; Angelbello, Alicia J.; Andrews, Ryan J.; Wang, Eric T.; Moss, Walter N.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">485-493</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is a genetically defined muscular dystrophy that is caused by an expanded repeat of r(CCUG) [r(CCUG)exp] in intron 1 of a CHC-Type Zinc Finger Nucleic Acid Binding Protein (CNBP) pre-mRNA.  Various mechanisms contribute to DM2 pathol. including pre-mRNA splicing defects caused by sequestration of the RNA splicing regulator muscleblind-like-1 (MBNL1) by r(CCUG)exp.  Herein, we study the biol. impacts of the mol. recognition of r(CCUG)exp's structure by a designer dimeric small mol. that directly cleaves the RNA in patient-derived cells.  The compd. is comprised of two RNA-binding modules conjugated to a deriv. of the natural product bleomycin.  Careful design of the chimera affords RNA-specific cleavage as attachment of the bleomycin cleaving module was done in a manner that disables DNA cleavage.  The chimeric cleaver is more potent than the parent binding compd. for alleviating DM2-assocd. defects.  Importantly, oligonucleotides targeting r(CCUG)exp sequence for cleavage exacerbate DM2 defects due to recognition of a short r(CCUG) sequence that is embedded in both CNBP, argonaute-1 (AGO1), and MBNL1, reducing these targets.  The latter event causes a greater depletion of functional MBNL1 than the amt. already sequestered by r(CCUG)exp.  Thus, compds. targeting RNA structures can have functional advantages over oligonucleotides that target sequence in some disease settings, particularly in DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDuGRa8-OiobVg90H21EOLACvtfcHk0liHavG8sQSzWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtlegtA%253D%253D&md5=5a1952891f4660a3353722d1a881cee6</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facschembio.9b00958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.9b00958%26sid%3Dliteratum%253Aachs%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DE.%2BT.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DStructure-specific%2520cleavage%2520of%2520an%2520RNA%2520repeat%2520expansion%2520with%2520a%2520dimeric%2520small%2520molecule%2520is%2520advantageous%2520over%2520sequence-specific%2520recognition%2520by%2520an%2520oligonucleotide%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D485%26epage%3D493%26doi%3D10.1021%2Facschembio.9b00958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Optimization of the linker domain in a dimeric compound that degrades an r(CUG) repeat expansion in cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">7827</span>– <span class="NLM_lpage">7839</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlGksbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=7827-7839&author=R.+I.+Benhamouauthor=M.+Abeauthor=S.+Choudharyauthor=S.+M.+Meyerauthor=A.+J.+Angelbelloauthor=M.+D.+Disney&title=Optimization+of+the+linker+domain+in+a+dimeric+compound+that+degrades+an+r%28CUG%29+repeat+expansion+in+cells&doi=10.1021%2Facs.jmedchem.0c00558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the Linker Domain in a Dimeric Compound that Degrades an r(CUG) Repeat Expansion in Cells</span></div><div class="casAuthors">Benhamou, Raphael I.; Abe, Masahito; Choudhary, Shruti; Meyer, Samantha M.; Angelbello, Alicia J.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">7827-7839</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA repeat expansions are responsible for more than 30 incurable diseases.  Among them is myotonic dystrophy type 1 (DM1), the most common form of adult on-set muscular dystrophy.  DM1 is caused by an r(CUG) repeat expansion [r(CUG)exp] located in the 3' untranslated region (UTR) of the dystrophia myotonica protein kinase gene.  This repeat expansion is highly structured, forming a periodic array of 5'CUG/3'GUC internal loop motifs.  We therefore designed dimeric compds. that simultaneously bind two of these motifs by connecting two RNA-binding modules with peptoid linkers of different geometries and lengths.  The optimal linker contains two proline residues and enhances compd. affinity.  Equipping this mol. with a bleomycin A5 cleaving module converts the simple binding compd. into a potent allele-selective cleaver of r(CUG)exp.  This study shows that the linker in modularly assembled ligands targeting RNA can be optimized to afford potent biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXG7D_mmEozrVg90H21EOLACvtfcHk0liHavG8sQSzWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlGksbrO&md5=aba7ecdeb9aa4aad9b1bdcde1f03ff8a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00558%26sid%3Dliteratum%253Aachs%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DAbe%26aufirst%3DM.%26aulast%3DChoudhary%26aufirst%3DS.%26aulast%3DMeyer%26aufirst%3DS.%2BM.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DOptimization%2520of%2520the%2520linker%2520domain%2520in%2520a%2520dimeric%2520compound%2520that%2520degrades%2520an%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D7827%26epage%3D7839%26doi%3D10.1021%2Facs.jmedchem.0c00558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramisetty, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranger, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, S. C.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of MBNL1–CCUG interaction by small molecules toward potential therapeutic agents for myotonic dystrophy type 2 (DM2)</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">8881</span>– <span class="NLM_lpage">8890</span>, <span class="refDoi"> DOI: 10.1093/nar/gkr415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1093%2Fnar%2Fgkr415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=21768123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqt7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=8881-8890&author=C.-H.+Wongauthor=Y.+Fuauthor=S.+R.+Ramisettyauthor=A.+M.+Barangerauthor=S.+C.+Zimmerman&title=Selective+inhibition+of+MBNL1%E2%80%93CCUG+interaction+by+small+molecules+toward+potential+therapeutic+agents+for+myotonic+dystrophy+type+2+%28DM2%29&doi=10.1093%2Fnar%2Fgkr415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of MBNL1-CCUG interaction by small molecules toward potential therapeutic agents for myotonic dystrophy type 2 (DM2)</span></div><div class="casAuthors">Wong, Chun-Ho; Fu, Yuan; Ramisetty, Sreenivasa Rao; Baranger, Anne M.; Zimmerman, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8881-8890</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 2 (DM2) is an incurable neuromuscular disease caused by expanded CCUG repeats that may exhibit toxicity by sequestering the splicing regulator MBNL1.  A series of triaminotriazine- and triaminopyrimidine-based small mols. (ligands 1-3) were designed, synthesized and tested as inhibitors of the MBNL1-CCUG interaction.  Despite the structural similarities of the triaminotriazine and triaminopyrimidine units, the triaminopyrimidine-based ligands bind with low micromolar affinity to CCUG repeats (Kd ∼ 0.1-3.6 μM) whereas the triaminotriazine ligands do not bind CCUG repeats.  Importantly, these simple and small triaminopyrimidine ligands exhibit both strong inhibition (Ki ∼ 2 μM) of the MBNL1-CCUG interaction and high selectivity for CCUG repeats over other RNA targets.  These expts. suggest these compds. are potential lead agents for the treatment of DM2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4kGlJSj32ZLVg90H21EOLACvtfcHk0liHavG8sQSzWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqt7jJ&md5=01cf07b5db37cc4f72c9ddf18e9dd39d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr415%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.-H.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DRamisetty%26aufirst%3DS.%2BR.%26aulast%3DBaranger%26aufirst%3DA.%2BM.%26aulast%3DZimmerman%26aufirst%3DS.%2BC.%26atitle%3DSelective%2520inhibition%2520of%2520MBNL1%25E2%2580%2593CCUG%2520interaction%2520by%2520small%2520molecules%2520toward%2520potential%2520therapeutic%2520agents%2520for%2520myotonic%2520dystrophy%2520type%25202%2520%2528DM2%2529%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3D8881%26epage%3D8890%26doi%3D10.1093%2Fnar%2Fgkr415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule recognition of RNA base pairs</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">491</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00049</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00049" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Cqu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=482-491&author=H.+S.+Haniffauthor=A.+Gravesauthor=M.+D.+Disney&title=Selective+small+molecule+recognition+of+RNA+base+pairs&doi=10.1021%2Facscombsci.8b00049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Recognition of RNA Base Pairs</span></div><div class="casAuthors">Haniff, Hafeez S.; Graves, Amanda; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">482-491</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many types of RNAs exist in the human transcriptome, yet only the bacterial ribosome has been exploited as a small mol. drug target.  Aside from rRNA, other cellular RNAs such as non-coding RNAs have primarily secondary structure and limited tertiary structure.  Within these secondary structures of non-canonically paired and unpaired regions, >50% are base-paired, with most efforts to target these structures focused on looped regions.  A void exists in the availability of small mols. capable of targeting RNA base-pairs.  Here, using chemoinformatics, an RNA focused library enriched for nitrogen-rich heterocycles was developed and tested for binding RNA base-pairs, leading to the identification of 6 selective and previously unknown binders.  While all binders were derivs. of benzimidazoles, those with expanded arom. polycycles bound selectively to AU pairs, while those with flexible urea side-chains bound selectively to GC pairs.  Interestingly, 2 of the 3 selective GC pair binders distinguished between the orientation of 5'GG/3'CC and 5'GC/3'CG pairs.  Furthermore, all 6 mols. showed >50-fold selectivity for RNA vs. DNA.  These studies provide foundational knowledge to better exploit RNA as targets for small mol. chem. probes or lead therapeutics by using modules that target RNA base-pairs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYvpJn4HevkrVg90H21EOLACvtfcHk0lifR7RVg93tNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cqu7zK&md5=8e9941315667412e41ad587a0e02efea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00049%26sid%3Dliteratum%253Aachs%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DGraves%26aufirst%3DA.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSelective%2520small%2520molecule%2520recognition%2520of%2520RNA%2520base%2520pairs%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2018%26volume%3D20%26spage%3D482%26epage%3D491%26doi%3D10.1021%2Facscombsci.8b00049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(96)00423-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2896%2900423-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0lifR7RVg93tNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25%26doi%3D10.1016%2FS0169-409X%2896%2900423-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span> <span> </span><span class="NLM_article-title">Two decades under the influence of the Rule of Five and the changing properties of approved oral drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1701</span>– <span class="NLM_lpage">1714</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslait73K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1701-1714&author=M.+D.+Shultz&title=Two+decades+under+the+influence+of+the+Rule+of+Five+and+the+changing+properties+of+approved+oral+drugs&doi=10.1021%2Facs.jmedchem.8b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Two Decades under the Influence of the Rule of Five and the Changing Properties of Approved Oral Drugs</span></div><div class="casAuthors">Shultz, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1701-1714</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two decades have passed since the rule of five ushered in the concept of "drug-like" properties.  Attempts to quantify, correlate, and categorize mols. based on Ro5 parameters evolved into the introduction of efficiency metrics with far reaching consequences in decision making by industry leaders and scientists seeking to discover new medicines.  Examn. of oral drug parameters approved before and after the original Ro5 anal. demonstrates that some parameters such as clogP and HBD remained const. while the cutoffs for parameters such as mol. wt. and HBA have increased substantially over the past 20 years.  The time dependent increase in the mol. wt. of oral drugs during the past 20 years provides compelling evidence to disprove the hypothesis that mol. wt. is a "drug-like" property.  This anal. does not validate parameters that have not changed as being "drug-like" but instead calls into question the entire hypothesis that "drug-like" properties exist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAnoxpQsv2kbVg90H21EOLACvtfcHk0lifR7RVg93tNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslait73K&md5=d3c8b1107133dc0b7b277d22fa029597</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00686%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26atitle%3DTwo%2520decades%2520under%2520the%2520influence%2520of%2520the%2520Rule%2520of%2520Five%2520and%2520the%2520changing%2520properties%2520of%2520approved%2520oral%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1701%26epage%3D1714%26doi%3D10.1021%2Facs.jmedchem.8b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeGoey, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, M. D.</span></span> <span> </span><span class="NLM_article-title">Beyond the Rule of 5: lessons learned from AbbVie’s drugs and compound collection</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00717</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00717" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2636-2651&author=D.+A.+DeGoeyauthor=H.-J.+Chenauthor=P.+B.+Coxauthor=M.+D.+Wendt&title=Beyond+the+Rule+of+5%3A+lessons+learned+from+AbbVie%E2%80%99s+drugs+and+compound+collection&doi=10.1021%2Facs.jmedchem.7b00717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the Rule of 5: Lessons Learned from AbbVie's Drugs and Compound Collection</span></div><div class="casAuthors">DeGoey, David A.; Chen, Hui-Ju; Cox, Philip B.; Wendt, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2636-2651</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing focus on the pursuit of targets considered to be less druggable that offer potential for development of promising new therapeutic agents for the treatment of diseases with large unmet medical need, particularly in the areas of oncol. and virol.  However, conducting drug discovery campaigns in "beyond rule of 5" (bRo5) chem. space presents a significant drug design and development challenge to medicinal chemists to achieve acceptable oral pharmacokinetics.  Retrospective anal. of past successes and failures in drug discovery bRo5 may shed light on the key principles that contribute to the oral bioavailability of successful bRo5 compds. and improve the efficiency of drug design for future projects.  We present here highlights and case studies of lessons learned from discovery of bRo5 compds.  A simple multiparametric scoring function (AB-MPS) was devised that correlated preclin. PK results with cLogD, no. of rotatable bonds, and no. of arom. rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHuaXbtjlALVg90H21EOLACvtfcHk0lgIW_Fem0SB5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeitbjJ&md5=48984dc65410d83d4ac8ddaca030b024</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00717%26sid%3Dliteratum%253Aachs%26aulast%3DDeGoey%26aufirst%3DD.%2BA.%26aulast%3DChen%26aufirst%3DH.-J.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DWendt%26aufirst%3DM.%2BD.%26atitle%3DBeyond%2520the%2520Rule%2520of%25205%253A%2520lessons%2520learned%2520from%2520AbbVie%25E2%2580%2599s%2520drugs%2520and%2520compound%2520collection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2636%26epage%3D2651%26doi%3D10.1021%2Facs.jmedchem.7b00717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2605</span>– <span class="NLM_lpage">2609</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.bmcl.2016.04.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=27117425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2605-2609&author=M.+G.+Costalesauthor=S.+G.+Rzuczekauthor=M.+D.+Disney&title=Comparison+of+small+molecules+and+oligonucleotides+that+target+a+toxic%2C+non-coding+RNA&doi=10.1016%2Fj.bmcl.2016.04.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of small molecules and oligonucleotides that target a toxic, non-coding RNA</span></div><div class="casAuthors">Costales, Matthew G.; Rzuczek, Suzanne G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2605-2609</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Potential RNA targets for chem. probes and therapeutic modalities are pervasive in the transcriptome.  Oligonucleotide-based therapeutics are commonly used to target RNA sequence.  Small mols. are emerging as a modality to target RNA structures selectively, but their development is still in its infancy.  The authors compare the activity of oligonucleotides and several classes of small mols. that target the non-coding r(CCUG) repeat expansion (r(CCUG)exp) that causes myotonic dystrophy type 2 (DM2), an incurable disease that is the second-most common cause of adult onset muscular dystrophy.  Small mol. types investigated include monomers, dimers, and multivalent compds. synthesized on-site by using RNA-templated click chem.  Oligonucleotides investigated include phosphorothioates that cleave their target and vivo-morpholinos that modulate target RNA activity via binding.  The authors show that compds. assembled on-site that recognize structure have the highest potencies amongst small mols. and are similar in potency to a vivo-morpholino modified oligonucleotide that targets sequence.  These studies are likely to impact the design of therapeutic modalities targeting other repeats expansions that cause fragile X syndrome and amyotrophic lateral sclerosis, for example.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hWVZ6e6s27Vg90H21EOLACvtfcHk0lgIW_Fem0SB5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVCmtrY%253D&md5=540a6ffdd24b0205cfe7fdf702832174</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.025%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DComparison%2520of%2520small%2520molecules%2520and%2520oligonucleotides%2520that%2520target%2520a%2520toxic%252C%2520non-coding%2520RNA%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2605%26epage%3D2609%26doi%3D10.1016%2Fj.bmcl.2016.04.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobczak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoskins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C. P.</span></span> <span> </span><span class="NLM_article-title">Two high-throughput screening assays for aberrant RNA-protein interactions in myotonic dystrophy type 1</span>. <i>Anal. Bioanal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>402</i></span>,  <span class="NLM_fpage">1889</span>– <span class="NLM_lpage">1898</span>, <span class="refDoi"> DOI: 10.1007/s00216-011-5604-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1007%2Fs00216-011-5604-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=22218462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BC38XisV2jsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=402&publication_year=2012&pages=1889-1898&author=C.+Z.+Chenauthor=K.+Sobczakauthor=J.+Hoskinsauthor=N.+Southallauthor=J.+J.+Maruganauthor=W.+Zhengauthor=C.+A.+Thorntonauthor=C.+P.+Austin&title=Two+high-throughput+screening+assays+for+aberrant+RNA-protein+interactions+in+myotonic+dystrophy+type+1&doi=10.1007%2Fs00216-011-5604-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Two high-throughput screening assays for aberrant RNA-protein interactions in myotonic dystrophy type 1</span></div><div class="casAuthors">Chen, Catherine Z.; Sobczak, Krzysztof; Hoskins, Jason; Southall, Noel; Marugan, Juan J.; Zheng, Wei; Thornton, Charles A.; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical and Bioanalytical Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">402</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1889-1898</span>CODEN:
                <span class="NLM_cas:coden">ABCNBP</span>;
        ISSN:<span class="NLM_cas:issn">1618-2642</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Myotonic dystrophy type 1 (DM1), the most prevalent form of adult muscular dystrophy, is caused by expansion of a CTG repeat in the 3' untranslated region of the DM protein kinase (DMPK) gene.  The pathogenic effects of the CTG expansion arise from the deleterious effects of the mutant transcript.  RNA with expanded CUG tracts alters the activities of several RNA binding proteins, including muscleblind-like 1 (MBNL1).  MBNL1 becomes sequestered in nuclear foci in complex with the expanded CUG-repeat RNA.  The resulting loss of MBNL1 activity causes misregulated alternative splicing of multiple genes, leading to symptoms of DM1.  The binding interaction between MBNL1 and mutant RNA could be a key step in the pathogenesis of DM1 and serves as a potential target for therapeutic intervention.  We have developed two high-throughput screens suitable assays using both homogeneous time-resolved fluorescence energy transfer and AlphaScreen technologies to detect the binding of a C-terminally His-tagged MBNL1 and a biotinylated (CUG)12 RNA.  These assays are homogeneous and successfully miniaturized to 1536-well plate format.  Both assays were validated and show robust signal-to-basal ratios and Z' factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7szE3EThrrLVg90H21EOLACvtfcHk0lgIW_Fem0SB5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisV2jsA%253D%253D&md5=9342df23fdd0e60c0c932a816248d003</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs00216-011-5604-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00216-011-5604-0%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BZ.%26aulast%3DSobczak%26aufirst%3DK.%26aulast%3DHoskins%26aufirst%3DJ.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DThornton%26aufirst%3DC.%2BA.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26atitle%3DTwo%2520high-throughput%2520screening%2520assays%2520for%2520aberrant%2520RNA-protein%2520interactions%2520in%2520myotonic%2520dystrophy%2520type%25201%26jtitle%3DAnal.%2520Bioanal.%2520Chem.%26date%3D2012%26volume%3D402%26spage%3D1889%26epage%3D1898%26doi%3D10.1007%2Fs00216-011-5604-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ule, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ule, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cline, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggiu, M.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Nova regulates brain-specific splicing to shape the synapse</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">844</span>– <span class="NLM_lpage">852</span>, <span class="refDoi"> DOI: 10.1038/ng1610</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1038%2Fng1610" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=16041372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsFKlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2005&pages=844-852&author=J.+Uleauthor=A.+Uleauthor=J.+Spencerauthor=A.+Williamsauthor=J.+S.+Huauthor=M.+Clineauthor=H.+Wangauthor=T.+Clarkauthor=C.+Fraserauthor=M.+Ruggiu&title=Nova+regulates+brain-specific+splicing+to+shape+the+synapse&doi=10.1038%2Fng1610"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Nova regulates brain-specific splicing to shape the synapse</span></div><div class="casAuthors">Ule, Jernej; Ule, Aljaz; Spencer, Joanna; Williams, Alan; Hu, Jing-Shan; Cline, Melissa; Wang, Hui; Clark, Tyson; Fraser, Claire; Ruggiu, Matteo; Zeeberg, Barry R.; Kane, David; Weinstein, John N.; Blume, John; Darnell, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">844-852</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Alternative RNA splicing greatly increases proteome diversity and may thereby contribute to tissue-specific functions.  We carried out genome-wide quant. anal. of alternative splicing using a custom Affymetrix microarray to assess the role of the neuronal splicing factor Nova in the brain.  We used a stringent algorithm to identify 591 exons that were differentially spliced in the brain relative to immune tissues, and 6.6% of these showed major splicing defects in the neocortex of Nova2-/- mice.  We tested 49 exons with the largest predicted Nova-dependent splicing changes and validated all 49 by RT-PCR.  We analyzed the encoded proteins and found that all those with defined brain functions acted in the synapse (34 of 40, including neurotransmitter receptors, cation channels, adhesion and scaffold proteins) or in axon guidance (8 of 40).  Moreover, of the 35 proteins with known interaction partners, 74% (26) interact with each other.  Validating a large set of Nova RNA targets has led us to identify a multi-tiered network in which Nova regulates the exon content of RNAs encoding proteins that interact in the synapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1yFzoRIc5lbVg90H21EOLACvtfcHk0liGQSJu9xSw6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsFKlsLs%253D&md5=a4516efec2e0a43655211586bc3137cb</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fng1610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng1610%26sid%3Dliteratum%253Aachs%26aulast%3DUle%26aufirst%3DJ.%26aulast%3DUle%26aufirst%3DA.%26aulast%3DSpencer%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DJ.%2BS.%26aulast%3DCline%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DT.%26aulast%3DFraser%26aufirst%3DC.%26aulast%3DRuggiu%26aufirst%3DM.%26atitle%3DNova%2520regulates%2520brain-specific%2520splicing%2520to%2520shape%2520the%2520synapse%26jtitle%3DNat.%2520Genet.%26date%3D2005%26volume%3D37%26spage%3D844%26epage%3D852%26doi%3D10.1038%2Fng1610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeFeo, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glinkerman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise targeted cleavage of a r(CUG) repeat expansion in cells by using a small-molecule–deglycobleomycin conjugate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">849</span>– <span class="NLM_lpage">855</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00036</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00036" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltFajsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=849-855&author=A.+J.+Angelbelloauthor=M.+E.+DeFeoauthor=C.+M.+Glinkermanauthor=D.+L.+Bogerauthor=M.+D.+Disney&title=Precise+targeted+cleavage+of+a+r%28CUG%29+repeat+expansion+in+cells+by+using+a+small-molecule%E2%80%93deglycobleomycin+conjugate&doi=10.1021%2Facschembio.0c00036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Precise Targeted Cleavage of a r(CUG) Repeat Expansion in Cells by Using a Small-Molecule-Deglycobleomycin Conjugate</span></div><div class="casAuthors">Angelbello, Alicia J.; DeFeo, Mary E.; Glinkerman, Christopher M.; Boger, Dale L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">849-855</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA repeat expansions cause more than 30 neurol. and neuromuscular diseases with no known cures.  Since repeat expansions operate via diverse pathomechanisms, one potential therapeutic strategy is to rid them from disease-affected cells, using bifunctional small mols. that cleave the aberrant RNA.  Such an approach has been previously implemented for the RNA repeat that causes myotonic dystrophy type 1 [DM1, r(CUG)exp] with Cugamycin, which is a small mol. that selectively binds r(CUG)exp conjugated to a bleomycin A5 cleaving module.  Herein, we demonstrate that, by replacing bleomycin A5 with deglycobleomycin, an analog in which the carbohydrate domain of bleomycin A5 is removed, the selectivity of the resulting small-mol. conjugate (DeglycoCugamycin) was enhanced, while maintaining potent and allele-selective cleavage of r(CUG)exp and rescue of DM1-assocd. defects.  In particular, DeglycoCugamycin did not induce the DNA damage that is obsd. with high concns. (25μM) of Cugamycin, while selectively cleaving the disease-causing allele and improving DM1 defects at 1μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_tmyLI4HRs7Vg90H21EOLACvtfcHk0liGQSJu9xSw6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltFajsrs%253D&md5=aa69cedc0d15fe97e70fd92eb3153535</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00036%26sid%3Dliteratum%253Aachs%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26aulast%3DDeFeo%26aufirst%3DM.%2BE.%26aulast%3DGlinkerman%26aufirst%3DC.%2BM.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520targeted%2520cleavage%2520of%2520a%2520r%2528CUG%2529%2520repeat%2520expansion%2520in%2520cells%2520by%2520using%2520a%2520small-molecule%25E2%2580%2593deglycobleomycin%2520conjugate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D849%26epage%3D855%26doi%3D10.1021%2Facschembio.0c00036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, W. N.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting the SARS-CoV-2 RNA genome with small molecule binders and ribonuclease targeting chimera (RIBOTAC) degraders</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1721</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.0c00984</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.0c00984" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFeitr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1713-1721&author=H.+S.+Haniffauthor=Y.+Tongauthor=X.+Liuauthor=J.+L.+Chenauthor=B.+M.+Sureshauthor=R.+J.+Andrewsauthor=J.+M.+Petersonauthor=C.+A.+O%E2%80%99Learyauthor=R.+I.+Benhamouauthor=W.+N.+Moss&title=Targeting+the+SARS-CoV-2+RNA+genome+with+small+molecule+binders+and+ribonuclease+targeting+chimera+%28RIBOTAC%29+degraders&doi=10.1021%2Facscentsci.0c00984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the SARS-CoV-2 RNA Genome with Small Molecule Binders and Ribonuclease Targeting Chimera (RIBOTAC) Degraders</span></div><div class="casAuthors">Haniff, Hafeez S.; Tong, Yuquan; Liu, Xiaohui; Chen, Jonathan L.; Suresh, Blessy M.; Andrews, Ryan J.; Peterson, Jake M.; O'Leary, Collin A.; Benhamou, Raphael I.; Moss, Walter N.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1713-1721</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">COVID-19 is a global pandemic, thus requiring multiple strategies to develop modalities against it.  We designed multiple bioactive small mols. that target a functional structure within the SARS-CoV-2's RNA genome, the causative agent of COVID-19.  An anal. to characterize the structure of the RNA genome provided a revised model of the SARS-CoV-2 frameshifting element including its attenuator hairpin.  By studying an RNA-focused small mol. collection, we identified a drug-like small mol. (I) that avidly binds to the revised attenuator hairpin structure with a Kd of 11 nM.  I stabilizes the hairpin's folded state and impairs frameshifting in cells.  I was further elaborated into a RNase targeting chimera (RIBOTAC) to recruit a cellular RNase to destroy the viral genome (C5-RIBOTAC) and into a covalent mol. (C5-Chem-CLIP) that validated direct target engagement and demonstrated its specificity for the viral RNA as compared to highly expressed host mRNAs.  The RIBOTAC lead optimization strategy improved the bioactivity of the compd. ≥10-fold.  Collectively, these studies demonstrate that the SARS-CoV-2 RNA genome should be considered druggable.  Covid-19 is a global pandemic caused by the SARS-CoV-2 virus, which has an RNA genome.  We designed small mols. that bind or degrade a structured, functional element in the virus to inhibit a process essential for viral propagation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwLN6QE_grALVg90H21EOLACvtfcHk0liGQSJu9xSw6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFeitr3O&md5=af035b9ac467ed21f0f69d54147284e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.0c00984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.0c00984%26sid%3Dliteratum%253Aachs%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%2BL.%26aulast%3DSuresh%26aufirst%3DB.%2BM.%26aulast%3DAndrews%26aufirst%3DR.%2BJ.%26aulast%3DPeterson%26aufirst%3DJ.%2BM.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DC.%2BA.%26aulast%3DBenhamou%26aufirst%3DR.%2BI.%26aulast%3DMoss%26aufirst%3DW.%2BN.%26atitle%3DTargeting%2520the%2520SARS-CoV-2%2520RNA%2520genome%2520with%2520small%2520molecule%2520binders%2520and%2520ribonuclease%2520targeting%2520chimera%2520%2528RIBOTAC%2529%2520degraders%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2020%26volume%3D6%26spage%3D1713%26epage%3D1721%26doi%3D10.1021%2Facscentsci.0c00984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeJesus-Hernandez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackenzie, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeve, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finch, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamson, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2011.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1016%2Fj.neuron.2011.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=21944778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A280%3ADC%252BC3MbjtFOrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2011&pages=245-256&author=M.+DeJesus-Hernandezauthor=I.+R.+Mackenzieauthor=B.+F.+Boeveauthor=A.+L.+Boxerauthor=M.+Bakerauthor=N.+J.+Rutherfordauthor=A.+M.+Nicholsonauthor=N.+A.+Finchauthor=H.+Flynnauthor=J.+Adamson&title=Expanded+GGGGCC+hexanucleotide+repeat+in+noncoding+region+of+C9ORF72+causes+chromosome+9p-linked+FTD+and+ALS&doi=10.1016%2Fj.neuron.2011.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS</span></div><div class="casAuthors">DeJesus-Hernandez Mariely; Mackenzie Ian R; Boeve Bradley F; Boxer Adam L; Baker Matt; Rutherford Nicola J; Nicholson Alexandra M; Finch NiCole A; Flynn Heather; Adamson Jennifer; Kouri Naomi; Wojtas Aleksandra; Sengdy Pheth; Hsiung Ging-Yuek R; Karydas Anna; Seeley William W; Josephs Keith A; Coppola Giovanni; Geschwind Daniel H; Wszolek Zbigniew K; Feldman Howard; Knopman David S; Petersen Ronald C; Miller Bruce L; Dickson Dennis W; Boylan Kevin B; Graff-Radford Neill R; Rademakers Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">245-56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Several families have been reported with autosomal-dominant frontotemporal dementia (FTD) and amyotrophic lateral sclerosis (ALS), genetically linked to chromosome 9p21.  Here, we report an expansion of a noncoding GGGGCC hexanucleotide repeat in the gene C9ORF72 that is strongly associated with disease in a large FTD/ALS kindred, previously reported to be conclusively linked to chromosome 9p.  This same repeat expansion was identified in the majority of our families with a combined FTD/ALS phenotype and TDP-43-based pathology.  Analysis of extended clinical series found the C9ORF72 repeat expansion to be the most common genetic abnormality in both familial FTD (11.7%) and familial ALS (23.5%).  The repeat expansion leads to the loss of one alternatively spliced C9ORF72 transcript and to formation of nuclear RNA foci, suggesting multiple disease mechanisms.  Our findings indicate that repeat expansion in C9ORF72 is a major cause of both FTD and ALS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8x2wRTCeVlWlSCSgBhwJFfW6udTcc2eZojT_XMfIajbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbjtFOrsg%253D%253D&md5=f5918aa4c4021cb33e6d6414ed0c4b44</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2011.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2011.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DDeJesus-Hernandez%26aufirst%3DM.%26aulast%3DMackenzie%26aufirst%3DI.%2BR.%26aulast%3DBoeve%26aufirst%3DB.%2BF.%26aulast%3DBoxer%26aufirst%3DA.%2BL.%26aulast%3DBaker%26aufirst%3DM.%26aulast%3DRutherford%26aufirst%3DN.%2BJ.%26aulast%3DNicholson%26aufirst%3DA.%2BM.%26aulast%3DFinch%26aufirst%3DN.%2BA.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DAdamson%26aufirst%3DJ.%26atitle%3DExpanded%2520GGGGCC%2520hexanucleotide%2520repeat%2520in%2520noncoding%2520region%2520of%2520C9ORF72%2520causes%2520chromosome%25209p-linked%2520FTD%2520and%2520ALS%26jtitle%3DNeuron%26date%3D2011%26volume%3D72%26spage%3D245%26epage%3D256%26doi%3D10.1016%2Fj.neuron.2011.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevarello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronesi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulpetti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, M.</span></span> <span> </span><span class="NLM_article-title">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1023/A:1008354229396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1023%2FA%3A1008354229396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=11061229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2000&pages=65-68&author=C.+Dalvitauthor=P.+Pevarelloauthor=M.+Tatoauthor=M.+Veronesiauthor=A.+Vulpettiauthor=M.+Sundstrom&title=Identification+of+compounds+with+binding+affinity+to+proteins+via+magnetization+transfer+from+bulk+water&doi=10.1023%2FA%3A1008354229396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of compounds with binding affinity to proteins via magnetization transfer from bulk water</span></div><div class="casAuthors">Dalvit, Claudio; Pevarello, Paolo; Tato, Marco; Veronesi, Marina; Vulpetti, Anna; Sundstrom, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-68</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A powerful screening by NMR methodol. (WaterLOGSY), based on transfer of magnetization from bulk water, for the identification of compds. that interact with target biomols. (proteins, RNA and DNA fragments) is described.  The method exploits efficiently the large reservoir of H2O magnetization.  The high sensitivity of the technique reduces the amt. of biomol. and ligands needed for the screening, which constitutes an important requirement for high throughput screening by NMR of large libraries of compds.  Application of the method to a compd. mixt. against the cyclin-dependent kinase 2 (cdk2) protein is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8Jo0PHYdALVg90H21EOLACvtfcHk0ljKnLUrr3znUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGit7o%253D&md5=e0fd0e5815f9bd20ed09d8033882feb6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008354229396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008354229396%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DPevarello%26aufirst%3DP.%26aulast%3DTato%26aufirst%3DM.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DVulpetti%26aufirst%3DA.%26aulast%3DSundstrom%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520compounds%2520with%2520binding%2520affinity%2520to%2520proteins%2520via%2520magnetization%2520transfer%2520from%2520bulk%2520water%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2000%26volume%3D18%26spage%3D65%26epage%3D68%26doi%3D10.1023%2FA%3A1008354229396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vuister, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">NMRPipe: A multidimensional spectral processing system based on UNIX pipes</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+A+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0ljKnLUrr3znUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520A%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span>; <span class="NLM_string-name">Kneller, D. G.</span></span> <i>SPARKY, NMR Assignment and Integration Software, 3</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=T.+D.+Goddard&author=D.+G.+Kneller&title=SPARKY%2C+NMR+Assignment+and+Integration+Software%2C+3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26btitle%3DSPARKY%252C%2520NMR%2520Assignment%2520and%2520Integration%2520Software%252C%25203%26pub%3DUniversity%2520of%2520California%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i69"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00414">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_61473"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00414?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00414</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Approximate IC<sub>50</sub>s for the disruption of the r(CCUG)<sup>exp</sup>–MBNL1 complex (Table S1); Sequences of primers (Table S2); Results of a high-throughput screen (Figure S1); Structures of compounds <b>1</b>–<b>44</b> (Figure S2); WaterLOGSY NMR analysis of <b>3</b> (Figure S3); WaterLOGSY NMR analysis of <b>4</b> (Figure S4); WaterLOGSY NMR analysis of <b>5</b> (Figure S5); WaterLOGSY NMR analysis of <b>7</b> (Figure S6); Two-dimensional NOESY NMR analysis (Figure S7); One-dimensional NMR analysis of compounds (Figure S8); Cell viability of DM2 fibroblasts <b>t</b>reated with <b>3</b>, <b>4</b>, <b>5</b>, and <b>7</b> (Figure S9); Structures of compounds <b>45</b>–<b>82</b> (Figure S10); Cell viability of DM2 fibroblasts treated with lead-optimized compounds (Figure S11); Cellular activity of compounds <b>63</b>, <b>78</b>, and <b>79</b> (Figure S12); Compound <b>63</b> does not affect MAP4K4 alternative splicing (Figure S13); Compound <b>63</b> has no effect on CNBP levels or IR exon 11 splicing (Figure S14); synthetic schemes, compound characterization; HPLC traces; and NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf">PDF</a>)</p></li><li><p class="inline">All DM2 hit compounds (SMILES + DATA) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_001.pdf">jm1c00414_si_001.pdf (5.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00414/suppl_file/jm1c00414_si_002.csv">jm1c00414_si_002.csv (8.13 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00414&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00414%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00414" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799937c6e8821a6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
