
<html lang="en"     class="pb-page"  data-request-id="8ad03a87-0123-4d58-9b5d-7677a7ee70e8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2012.55.issue-15;ctype:string:Journal Content;article:article:10.1021/jm300576q;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Predicting New Indications for Approved Drugs Using a Proteochemometric Method" /></meta><meta name="dc.Creator" content="Sivanesan  Dakshanamurthy" /></meta><meta name="dc.Creator" content="Naiem T.  Issa" /></meta><meta name="dc.Creator" content="Shahin  Assefnia" /></meta><meta name="dc.Creator" content="Ashwini  Seshasayee" /></meta><meta name="dc.Creator" content="Oakland J.  Peters" /></meta><meta name="dc.Creator" content="Subha  Madhavan" /></meta><meta name="dc.Creator" content="Aykut  Uren" /></meta><meta name="dc.Creator" content="Milton L.  Brown" /></meta><meta name="dc.Creator" content="Stephen W.  Byers" /></meta><meta name="dc.Description" content="The most effective way to move from target identification to the clinic is to identify already approved drugs with the potential for activating or inhibiting unintended targets (repurposing or repo..." /></meta><meta name="Description" content="The most effective way to move from target identification to the clinic is to identify already approved drugs with the potential for activating or inhibiting unintended targets (repurposing or repo..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 25, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm300576q" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2012 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm300576q" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm300576q" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm300576q" /></link>
        
    
    

<title>Predicting New Indications for Approved Drugs Using a Proteochemometric Method | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm300576q" /></meta><meta property="og:title" content="Predicting New Indications for Approved Drugs Using a Proteochemometric Method" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0015.jpeg" /></meta><meta property="og:description" content="The most effective way to move from target identification to the clinic is to identify already approved drugs with the potential for activating or inhibiting unintended targets (repurposing or repositioning). This is usually achieved by high throughput chemical screening, transcriptome matching, or simple in silico ligand docking. We now describe a novel rapid computational proteochemometric method called “train, match, fit, streamline” (TMFS) to map new drug–target interaction space and predict new uses. The TMFS method combines shape, topology, and chemical signatures, including docking score and functional contact points of the ligand, to predict potential drug–target interactions with remarkable accuracy. Using the TMFS method, we performed extensive molecular fit computations on 3671 FDA approved drugs across 2335 human protein crystal structures. The TMFS method predicts drug–target associations with 91% accuracy for the majority of drugs. Over 58% of the known best ligands for each target were correctly predicted as top ranked, followed by 66%, 76%, 84%, and 91% for agents ranked in the top 10, 20, 30, and 40, respectively, out of all 3671 drugs. Drugs ranked in the top 1–40 that have not been experimentally validated for a particular target now become candidates for repositioning. Furthermore, we used the TMFS method to discover that mebendazole, an antiparasitic with recently discovered and unexpected anticancer properties, has the structural potential to inhibit VEGFR2. We confirmed experimentally that mebendazole inhibits VEGFR2 kinase activity and angiogenesis at doses comparable with its known effects on hookworm. TMFS also predicted, and was confirmed with surface plasmon resonance, that dimethyl celecoxib and the anti-inflammatory agent celecoxib can bind cadherin-11, an adhesion molecule important in rheumatoid arthritis and poor prognosis malignancies for which no targeted therapies exist. We anticipate that expanding our TMFS method to the &gt;27 000 clinically active agents available worldwide across all targets will be most useful in the repositioning of existing drugs for new therapeutic targets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm300576q"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm300576q">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm300576q&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm300576q&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm300576q&amp;href=/doi/10.1021/jm300576q" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 6832-6848</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm300571h" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm3006965" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Predicting New Indications for Approved Drugs Using a Proteochemometric Method</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sivanesan++Dakshanamurthy">Sivanesan Dakshanamurthy</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Naiem+T.++Issa">Naiem T. Issa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shahin++Assefnia">Shahin Assefnia</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ashwini++Seshasayee">Ashwini Seshasayee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oakland+J.++Peters">Oakland J. Peters</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Subha++Madhavan">Subha Madhavan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aykut++Uren">Aykut Uren</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Milton+L.++Brown">Milton L. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+W.++Byers">Stephen W. Byers</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C. 20057, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Biochemistry and Molecular Biology, Georgetown University, Washington, D.C. 20057, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 202-687-2347. E-mail: <a href="/cdn-cgi/l/email-protection#4536217776760522202a372220312a322b6b202130"><span class="__cf_email__" data-cfemail="671403555454270002081500021308100949020312">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm300576q&amp;href=/doi/10.1021%2Fjm300576q" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 6832–6848</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 10, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 April 2012</li><li><span class="item_label"><b>Published</b> online</span>25 July 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 August 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm300576q" title="DOI URL">https://doi.org/10.1021/jm300576q</a></div><div class="article_header-article-copyright"><strong>Copyright © 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6832%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSivanesan%2BDakshanamurthy%252C%2BNaiem%2BT.%2BIssa%252C%2BShahin%2BAssefnia%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D55%26issueNum%3D15%26contentID%3Djm300576q%26title%3DPredicting%2BNew%2BIndications%2Bfor%2BApproved%2BDrugs%2BUsing%2Ba%2BProteochemometric%2BMethod%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6848%26publicationDate%3DAugust%2B2012">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm300576q"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3237</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">87</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm300576q" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Predicting New Indications for Approved Drugs Using a Proteochemometric Method&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sivanesan&quot;,&quot;last_name&quot;:&quot;Dakshanamurthy&quot;},{&quot;first_name&quot;:&quot;Naiem&quot;,&quot;last_name&quot;:&quot;T. Issa&quot;},{&quot;first_name&quot;:&quot;Shahin&quot;,&quot;last_name&quot;:&quot;Assefnia&quot;},{&quot;first_name&quot;:&quot;Ashwini&quot;,&quot;last_name&quot;:&quot;Seshasayee&quot;},{&quot;first_name&quot;:&quot;Oakland&quot;,&quot;last_name&quot;:&quot;J. Peters&quot;},{&quot;first_name&quot;:&quot;Subha&quot;,&quot;last_name&quot;:&quot;Madhavan&quot;},{&quot;first_name&quot;:&quot;Aykut&quot;,&quot;last_name&quot;:&quot;Uren&quot;},{&quot;first_name&quot;:&quot;Milton&quot;,&quot;last_name&quot;:&quot;L. Brown&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;W. Byers&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;6832-6848&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm300576q&quot;},&quot;abstract&quot;:&quot;The most effective way to move from target identification to the clinic is to identify already approved drugs with the potential for activating or inhibiting unintended targets (repurposing or repositioning). This is usually achieved by high throughput chemical screening, transcriptome matching, or simple in silico ligand docking. We now describe a novel rapid computational proteochemometric method called “train, match, fit, streamline” (TMFS) to map new drug–target interaction space and predict new uses. The TMFS method combines shape, topology, and chemical signatures, including docking score and functional contact points of the ligand, to predict potential drug–target interactions with remarkable accuracy. Using the TMFS method, we performed extensive molecular fit computations on 3671 FDA approved drugs across 2335 human protein crystal structures. The TMFS method predicts drug–target associations with 91% accuracy for the majority of drugs. Over 58% of the known best ligands for each target were correctl&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300576q&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300576q" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300576q&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300576q" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300576q&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300576q" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm300576q&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300576q&amp;href=/doi/10.1021/jm300576q" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm300576q" /></input><a href="/doi/pdf/10.1021/jm300576q" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm300576q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm300576q%26sid%3Dliteratum%253Aachs%26pmid%3D22780961%26genre%3Darticle%26aulast%3DDakshanamurthy%26date%3D2012%26atitle%3DPredicting%2BNew%2BIndications%2Bfor%2BApproved%2BDrugs%2BUsing%2Ba%2BProteochemometric%2BMethod%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D15%26spage%3D6832%26epage%3D6848%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jmcmar.2012.55.issue-15/production/jmcmar.2012.55.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_null" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0015.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The most effective way to move from target identification to the clinic is to identify already approved drugs with the potential for activating or inhibiting unintended targets (repurposing or repositioning). This is usually achieved by high throughput chemical screening, transcriptome matching, or simple in silico ligand docking. We now describe a novel rapid computational proteochemometric method called “train, match, fit, streamline” (TMFS) to map new drug–target interaction space and predict new uses. The TMFS method combines shape, topology, and chemical signatures, including docking score and functional contact points of the ligand, to predict potential drug–target interactions with remarkable accuracy. Using the TMFS method, we performed extensive molecular fit computations on 3671 FDA approved drugs across 2335 human protein crystal structures. The TMFS method predicts drug–target associations with 91% accuracy for the majority of drugs. Over 58% of the known best ligands for each target were correctly predicted as top ranked, followed by 66%, 76%, 84%, and 91% for agents ranked in the top 10, 20, 30, and 40, respectively, out of all 3671 drugs. Drugs ranked in the top 1–40 that have not been experimentally validated for a particular target now become candidates for repositioning. Furthermore, we used the TMFS method to discover that mebendazole, an antiparasitic with recently discovered and unexpected anticancer properties, has the structural potential to inhibit VEGFR2. We confirmed experimentally that mebendazole inhibits VEGFR2 kinase activity and angiogenesis at doses comparable with its known effects on hookworm. TMFS also predicted, and was confirmed with surface plasmon resonance, that dimethyl celecoxib and the anti-inflammatory agent celecoxib can bind cadherin-11, an adhesion molecule important in rheumatoid arthritis and poor prognosis malignancies for which no targeted therapies exist. We anticipate that expanding our TMFS method to the >27 000 clinically active agents available worldwide across all targets will be most useful in the repositioning of existing drugs for new therapeutic targets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55781" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Traditional methods of drug discovery face formidable scientific and regulatory obstacles resulting in the passage of many years and many failures from the discovery of a target to the clinical application of a novel patentable drug designed to inhibit or activate its function. Not surprisingly, there has been a marked decline in the willingness of the pharmaceutical industry to invest in drug discovery programs.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7 ref8">(1-8)</a> With the emergence of systems biology approaches many more new drug targets have been identified and validated. However, drug development for these new targets is time-consuming and prohibitively expensive leading to the concept of drug repositioning in which existing approved compounds are repurposed for another target/disease. There are clear advantages to this approach including a dramatic reduction in time, expense, and safety concerns.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">A number of existing approved drugs can be effective therapy for diseases other than those for which they were approved.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8-10)</a> Recently, the National Institutes of Health (NIH) has emphasized the importance of drug repositioning and deposited more than 27 000 active pharmaceutical ingredients in its Chemical Genomics Center (NCGC) database to encourage public screening.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> To date, screening is usually achieved by high throughput chemical screening or transcriptome matching. Other methods include phenotypic screening, protein–protein interaction assays, drug annotation technologies, high-throughput screening using cell-based disease models, gene activity mapping, ligand-based cheminformatics approaches, and in vivo animal models of diseases.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a> However, experimentally testing all approved drugs against all targets is extremely expensive and technically unrealizable. An additional challenge of these screening studies is that after one gets a “hit”, the rational mechanism of action must still be deduced and tested. To address this, computational approaches based on drug regulated gene expression, side effect profile, and protein or chemical similarity have been developed.<a onclick="showRef(event, 'ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29">(13-29)</a> By use of high performance computing, high-throughput computational drug–target docking and screening are now also feasible, but current methods are only able to predict a rough estimation of the free energy of binding and further suffer from high false positive and low accuracy rates of drug–target association prediction.<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34">(27-34)</a></div><div class="NLM_p last">Given the aforementioned challenges, we directed our efforts in this study to better predict “molecule of best fit” and have developed a comprehensive prediction method called “train, match, fit, streamline” (TMFS) that reduces false positive predictions and enriches for the highest confidence drug–target interactions. Previous studies screened FDA drugs using either chemical similarity or docking with stringent scoring criteria.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> In contrast, our TMFS method combines 11 different descriptors, which include shape and topology signatures, physicochemical functional descriptors, contact points of the ligand and the target protein, chemical similarity, and docking score. In the TMFS method, descriptors are trained with template knowledge, match and fit of the signatures are identified, and the data are streamlined. Using this method, we report in silico screening of 3671 FDA approved and investigational drugs across 2335 protein structures. Our directed efforts led to the identification of known drug–target interactions with remarkable accuracy and experimental confirmation of new activities for two well-known drugs.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23323" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23323" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The TMFS method is outlined in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>, and the following sections detail the steps within each module.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Protein Target Collection, Processing, and Preparation</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> Protein Collection</h4><div class="NLM_p last">We performed an extensive search of the PDB database (<a href="http://www.rcsb.org" class="extLink">www.rcsb.org</a>) with the following parameter filters to obtain juman PDB holo structures: (a) source organism, <i>Homo sapiens</i>; (b) macromolecule type, only contains protein/enzyme; (c) has ligands, yes; (d) experimental method, X-ray with experimental data; (e) do not include proteins that have sequence similarity of >90%. This filtered query resulted in 11 100 structures, which were subsequently downloaded.</div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Protein Processing</h4><div class="NLM_p last">The PDB structures were filtered to eliminate structures that contained only metals or other ions noted without ligands. The retained set was further filtered by removing structures containing “modified residues” as ligands using a PERL script. Using another PERL script, we cleaned the remaining protein PDB files so that they contained only the correct chains, those that are biologically relevant, interact with ligand, and contain all necessary cofactors. Next, the script was formatted as a list that contained the RCSB two- and three-letter codes corresponding to the cofactors and metals. These records were then searched against HETATM lines for matches. All matches were retained along with the corresponding CHAIN records, and nonmatching HETATM lines were deleted. The resulting modified PDB entries were devoid of all solvent molecules, salts, and noncofactor ligands. A total of 2335 modified PDB structures were subjected to virtual protein preparation. Modified PDB structures were converted into the Schrodinger software’s native MAE format for protein preparation. The “protein prep” command line utility was used in a C-shell script to automate the process of adding explicit hydrogen atoms and fixing the correct protonated states and disulfide bonds.</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Ligand Collection, Processing, and Preparation</h3><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Crystal Structures of Reference Ligands and FDA-Approved and Experimental Drug Set Collection</h4><div class="NLM_p last">For the set of protein structures obtained prior to the protein preparation procedure, we gathered their corresponding ligand crystal structures from the PDB database. These ligands served as template coordinates for receptor grid generation, a docking control, and references for the ligand-centric rescoring. To achieve this, we used a C-shell script that prepared a list of PDB IDs with their corresponding ligand three-letter codes and substituted the paired strings into a template hyperlink using the cURL command to retrieve the appropriate SDF files. This automation allowed for the retrieval of individual ligands that retained their bioactive conformations and coordinates with respect to each chain of their corresponding proteins. FDA-approved and experimental drug structures were obtained from the Drug Bank (<a href="http://www.drugbank.ca" class="extLink">www.drugbank.ca</a>), FDA (<a href="http://www.FDA.gov" class="extLink">www.FDA.gov</a>), and BindingDB (<a href="http://www.bindingdb.org" class="extLink">www.bindingdb.org</a>).</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Reference Ligand Processing</h4><div class="NLM_p last">The SDF files downloaded from the PDB database contained one or more instances of the ligand depending on whether or not the corresponding proteins were crystallized as multimeric structures. Since the PDB structures were processed such that only the biologically relevant chain is retained, we processed the SDF files so that each one contained only a single instance of the ligand that corresponds to the biologically active PDB chain. Using a PERL script, we extracted chain identifiers from the PDB files and used them to match the ligand chain IDs. The resulting SDF files were then subjected to ligand preparation procedures using Schrodinger’s LigPrep application.</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Ligand Preparation</h3><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Reference Ligands</h4><div class="NLM_p last">Since the reference ligand crystal structures were downloaded as three-dimensional structures in their bioactive conformations from the PDB database, we used the “applyhtreatment” command via a C-Shell script. This command allowed us to retain these conformations while adding hydrogen atoms and neutralizing the ligands for use in docking control, shape calculations, and the generation of ligand-based descriptors using Schrodinger’s QikProp application.</div></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> FDA-Approved/Investigational Drugs</h4><div class="NLM_p last">We first acquired these molecules in their two-dimensional SDF format. For conversion to three-dimensional energy-minimized structures and neutralization, a “ligprep” command was automated using a C-Shell script. This set of neutralized molecules was also subjected to QikProp descriptor generation. After neutralization, the neutralized structures were ionized at a pH of 7.0 using the “ionizer” utility to generate ionized states that retained the minimized conformations at physiological pH. These ionized structures were later used for shape calculations and docking.</div></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Generation of Ligand Descriptors</h4><div class="NLM_p last">Ligand descriptors for the ligand-centric descriptor similarity approach were calculated using Schrodinger’s QikProp application. The following descriptors were computed for the reference ligands and FDA-approved drugs: (1) number of hydrogen-bond acceptors, (2) dipole moment, (3) number of hydrogen-bond donors, (4) electron affinity, (5) globularity, (6) molecular weight, (7) predicted log of the octanol/water partition coefficient (ClogP), (8) number of rotatable bonds, (9) solvent-accessible surface area (SASA), and (10) volume.</div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Shape Quantification: Ligand and Protein Binding Pockets</h3><div class="NLM_p last">Shape descriptors for the ligand and protein binding pockets were generated using a Java software package provided by the Thornton group.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The spherical harmonics expansion approach was used to describe the shape of ligands and binding sites using protomol information of those sites obtained from the sc-PDB database.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> For PDB files missing in sc-PDB, we computed the protomols using SurFlex Protomol generator within SYBYL X.1 (Tripos International, St. Louis, MO, U.S.). Binding site protomols were stripped of all atoms except hydrogen and carbon so that the final pocket shape is as refined as possible. The methodological application of the expansion with real spherical harmonic functions to approximate the surface function was stimulated by the work by Kharaman et al.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Equation <a class="ref internalNav" href="#eq1" aria-label="1">1</a> shows the function for the spherical harmonics shape calculations.<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Grid Generation and Docking</h3><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Generation of Receptor Grids for Docking</h4><div class="NLM_p last">Grids were generated using Schrodinger’s Glide module. Grid center points were determined from the centroid of each protein’s cognate ligand. To obtain the centroid, we extracted the Cartesian coordinates for each atom in the ligand and took the average for each dimension. To determine the size of the grids, we took a trial-and-error approach to determine the smallest grid size that would allow for the re-docking of all reference ligands. We chose the largest reference ligand as our upper size limit and found that a grid size of 20 Å on each side was the minimum to allow for it to dock. Thus, the grid size for docking simulations was set at 20 Å.</div></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Creation of an FDA-Approved/Investigational Drugs Data Set for Docking</h4><div class="NLM_p last">Since the FDA drugs prepared using LigPrep were energetically minimized, their final 3D shapes may deviate significantly from those of the reference ligands and their native binding pockets. In order to predict more accurate poses, we sought to create unique conformer sets of FDA drugs with respect to each protein. To do so, we first performed an “exhaustive” conformational search using Schrodinger’s ConfGen module for each drug to obtain a library of more than 100 000 conformers. The shape of each conformer, along with the active conformation of the reference ligands, was then calculated using the spherical harmonics expansion approach.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Subsequently, shape similarities were quantified using the Euclidean distance metric between each reference ligand and all the drug conformers (eq <a class="ref internalNav" href="#eq2" aria-label="2">2</a>). The drug data set for docking was assembled by choosing the conformer for each drug whose shape had the smallest Euclidean distance to the shape of the reference cognate ligand for a given protein. Thus, each protein had a unique drug data set whose conformers more closely resembled the shape of that protein’s reference ligand.</div></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Positive Docking Control: Choosing a Scoring Function</h4><div class="NLM_p">Because a large number of protein–ligand complexes needed to be scored and our computational resources being limited, we sought a scoring function in Glide that would give reasonable docking results most efficiently. To determine this, we re-docked all the reference ligands, with their crystal structure conformations conserved, to their native targets to confirm that we were able to reproduce their bioactive conformations. We chose the XP scoring function with a 10-pose postminimization procedure to determine the final pose.</div><div class="NLM_p last">We determined reproducibility of the bioactive conformations using visual inspection. That is, we superimposed the PDB crystal structure reference containing the crystallized ligand to the docking output conformation. Upon superimposing, we visually inspected the positions of the crystallized ligand compared to the docking output ligand to note any deviations. From thorough inspection of 100 protein–ligand complexes, we found ∼70% of them to have reproduced the bioactive conformations. With this outcome, we decided to proceed with the XP scoring function. Each unique FDA drug was docked to its respective protein using the aforementioned parameter options.</div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Analysis</h3><div id="sec2_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Shape Similarities</h4><div class="NLM_p last">To quantify shape similarity, we employed the Euclidean distance metric.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Euclidean distances allowed for a fine-resolution comparison of ligand and pocket shapes through comparison of their spherical harmonics expansion coefficients, as well as the shape of multiple conformers for a single molecule. Equation <a class="ref internalNav" href="#eq2" aria-label="2">2</a> depicts the Euclidean distance function.<span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span>The smaller the Euclidean distance between two coefficients (vectors <i>a</i> and <i>b</i> having <i>n</i> = (<i>l</i><sub>max</sub> + 1)<sup>2</sup>), the more similar they are with regards to shape. As a general rule, shapes are identical if the Euclidean distance is less than 3, similar if it is less than 5, and increasingly dissimilar if it is greater than 6.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The ligand-to-ligand and ligand-to-pocket (protein binding site) Euclidean distance scores were normalized and implemented into the final ranking equation. Euclidean distances were also calculated for protein pocket-to-pocket shape comparisons, which were used for a binding site similarity analysis outside the “ligand of good fit” question.</div></div><div id="sec2_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Ligand-Based Descriptor Similarities</h4><div class="NLM_p last">Since our approach in determining the “ligand of good fit” depends partially on the cocrystallized ligand, we sought to perform analyses of similarities of the query molecules to the reference ligands using the ligand-based descriptors described earlier. The QikProp module in Schrodinger provides quantification of ligand-based descriptors, such as molecule globularity, which further allows for statistical similarity analysis. To do so, we used the Strike module in Schrodinger to calculate Tanimoto and Manhattan similarity scores for each probe molecule’s descriptor value against all of the reference ligands. A Tanimoto score of 1.0 or a Manhattan score of 0.0 for a particular descriptor signifies that a given probe molecule is practically identical to the reference ligand based on that descriptor property. The discrepancy in the usage of Tanimoto or Manhattan scores is due to whether or not the variable at hand is continuous or discrete, respectively. These similarity scores were later normalized and substituted for the final ranking procedure in eq <a class="ref internalNav" href="#eq8" aria-label="8">8</a>.</div></div><div id="sec2_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Data Normalization</h4><div class="NLM_p last">Next, we normalized the docking scores, shape similarity, Euclidean distance scores, and ligand-based descriptor similarity scores to create a common scoring scheme whose values ranged from 0 to 1, with 1 being either the most favorable docked conformation or greatest similarity based on the shape and ligand-based descriptors. Equations <a class="ref internalNav" href="#eq3" aria-label="3">3</a> and <a class="ref internalNav" href="#eq4" aria-label="4">4</a> depict the schema of normalization.<span class="NLM_disp-formula" id="eq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m003.gif" alt="" /></img><span class="labelSpan">(3)</span></span><span class="NLM_disp-formula" id="eq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m004.gif" alt="" /></img><span class="labelSpan">(4)</span></span>Equation <a class="ref internalNav" href="#eq3" aria-label="3">3</a> was used for data whose best score was the maximum score (i.e., Tanimoto scores for ligand-based descriptors), while eq <a class="ref internalNav" href="#eq4" aria-label="4">4</a> was used for data whose best score was the minimum score (i.e., docking scores where the most negative score is the best). The final normalized scores were substituted in eq <a class="ref internalNav" href="#eq8" aria-label="8">8</a>.</div></div><div id="sec2_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Ligand Interaction Correction Score</h4><div class="NLM_p last">To get a better estimate of ligand–protein interactions on the reference binding site, we introduced another correction term called “optimal ligand interaction correction” (OLIC). This correction assumes that ligands will have similar experimental activity if their interaction involves similar binding site residues and makes similar interaction patterns compared to the reference ligand. The nature of the interactions and their interacting residue motifs are then input to the OLIC algorithm, which calculated a score for each reference and test set ligand contact point using eqs <a class="ref internalNav" href="#eq5" aria-label="5">5</a> and <a class="ref internalNav" href="#eq6" aria-label="6">6</a>. The final contact point score was calculated using eq <a class="ref internalNav" href="#eq7" aria-label="7">7</a>. To be consistent with the other scores calculated above, the corrected score was normalized in the range of 0 to 1 using eqs <a class="ref internalNav" href="#eq3" aria-label="3">3</a> and <a class="ref internalNav" href="#eq4" aria-label="4">4</a>. If the corrected score is “0”, then the test set ligand has a similar interaction pattern and similar activity and is therefore considered as the molecule of best fit when compared to the reference ligand. If the corrected score is “1”, then the test set ligand is considered as a nonbinder.<span class="NLM_disp-formula" id="eq5"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m005.gif" alt="" /></img><span class="labelSpan">(5)</span></span><span class="NLM_disp-formula" id="eq6"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m006.gif" alt="" /></img><span class="labelSpan">(6)</span></span><span class="NLM_disp-formula" id="eq7"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m007.gif" alt="" /></img><span class="labelSpan">(7)</span></span>With respect to eqs <a class="ref internalNav" href="#eq5" aria-label="5">5</a>, <a class="ref internalNav" href="#eq6" aria-label="6">6</a>, and <a class="ref internalNav" href="#eq7" aria-label="7">7</a>, “<i>S</i>(OLIC-R)” represents the score for reference ligand, “<i>S</i>(OLIC-T)” represents the score for test set ligands, “CS (OLIC)” is the corrected score, where “<i>n</i>” is the total number of contact points extending from 1 to <i>∞,</i> “σ” represents contact point, “σ<i><sub>ij</sub></i>” is the contact points of <i>i</i>th ligand with <i>j</i>th protein, and “σ<sub><i>ji</i></sub>” is the contact points of <i>j</i>th protein with <i>i</i>th ligand.</div></div><div id="sec2_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Prediction of Molecule of Best-Fit</h4><div class="NLM_p">Molecules of best fit were calculated by the TMFS method comprehensive score “<i>Z</i>” given by the following equation:<span class="NLM_disp-formula" id="eq8"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m008.gif" alt="" /></img><span class="labelSpan">(8)</span></span>The “<i>Y</i>” term represents the normalized docking score of a ligand (σ<sub>l</sub>) to a particular protein (σ<sub>p</sub>), along with its designated weight (ω<sub><i>k</i></sub>). The first summation term, where <i>l</i> = 1, represents the combination of normalized Euclidean distance scores for protein pocket–ligand (<i>f</i>(σ<sub>p</sub>,σ<sub>l</sub>)) and reference ligand (σ<sub>c</sub>)–ligand (<i>f</i>(σ<sub>c</sub>,σ<sub>l</sub>)) comparisons with designated weights (ω<sub><i>i</i></sub>) and (ω<sub><i>i</i>+1</sub>), respectively. The second summation term, where <i>j</i> = 8, represents the combination of ligand-based descriptor terms (eight total) and their respective values. Normalized Tanimoto or Manhattan scores for each descriptor are represented by <i>X</i><sub><i>n</i></sub>, where the value of <i>n</i> corresponds to a particular descriptor such as solvent-accessible surface area (SASA) or number of rotatable bonds (Rotor). Note that the identities of σ<sub>p</sub>, σ<sub>c</sub>, and σ<sub>l</sub> (protein, reference ligand crystal structure for that protein, and probe molecule, respectively), are constant across all terms of the equation.</div><div class="NLM_p">The values for weights ω<sub><i>k</i></sub>, ω<sub><i>i</i></sub>, and ω<sub><i>i</i>+1</sub> are 4, 2, and 2, respectively. The final ranking of protein target–ligand complexes comprises 11 total descriptors, 8 of which are solely ligand-based. In order to reduce this bias, we provided weights to the protein-oriented descriptors (i.e., docking score and protein shape), as well as to the ligand shape score because we believe that the shape parameters are more accurate approximations of the true protein–ligand interactions. The weights of 4, 2, and 2 for docking, protein shape, and ligand shape scores, respectively, provided the final ranking equation with a good balance among the descriptors that led to the accurate predictions noted in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B.</div><div class="NLM_p last">The last term represents the “CS(OLIC)” correction term for protein–ligand interaction points calculated using eq <a class="ref internalNav" href="#eq7" aria-label="7">7</a>. The sum of all these terms provides a comprehensive TMFS <i>Z</i>-score for a single ligand with respect to a protein that takes into account receptor-centric features (docking score), ligand-centric nonstructural descriptors (QikProp descriptors from Schrodinger), and shape-based features (protein pocket–ligand and ligand–ligand).</div></div><div id="sec2_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Validation of TMFS Method</h4><div class="NLM_p last">As will be explained in the <a class="ref internalNav" href="#sec3" aria-label="Results and Discussion">Results and Discussion</a>, a proper validation of the TMFS predictions across our large protein target data set (2335 targets) using the currently available literature poses many challenges. In order to account for these challenges, we devised the following equation to calculate the percent correctly predicted (PCP) targets:<span class="NLM_disp-formula" id="eq9"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/jm-2012-00576q_m009.gif" alt="" /></img><span class="labelSpan">(9)</span></span>where “<i>n</i>” represents total number of targets; “<i>A</i>, <i>B</i>, <i>Y</i>, <i>B</i>, <i>X</i>, <i>Y</i>, <i>Z</i>, and <i>E</i>” represent targets; “<i>A</i><sub><i>ji</i></sub>” is the number of predicted targets “<i>j</i>” for drug “<i>i</i>”; “<i>B</i><sub><i>ji</i></sub>” is the number of experimentally validated targets “<i>j</i>” for drug “<i>i</i>”; “<i>X</i><sub><i>ji</i></sub>” is the number of correctly predicted targets “<i>j</i>” from these experimental results for drug “<i>i</i>”; “<i>Y</i><sub><i>ji</i></sub>” is the number of targets not validated experimentally within the predicted lists for drug “<i>i</i>”; “<i>Z</i><sub><i>ji</i></sub>” is the number of experimentally validated targets “<i>j</i>” for drug “<i>i</i>” that are not included in the target data set; and “<i>E</i><sub><i>ji</i></sub>” is number of experimentally validated targets “<i>j</i>” for drug “<i>i</i>” that were not target hits but are included in the target data set.</div></div><div id="sec2_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Cadherin-11 Surface Plasmon Resonance (SPR) Assay</h4><div class="NLM_p last">Surface plasmon resonance experiments were carried out with a Biacore T100 equipped with a CM5 sensor chip. Briefly, mouse extracellular domain 1-2 (EC 1-2) C-terminally cysteine-tagged cadherin-11 recombinant protein<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> was immobilized on flow cell (FC) 2 in HEPES buffered saline (10 mM Hepes, pH 7.4, and 150 mM NaCl, 3 mM CaCl2) using a thiol-coupling kit according to the manufacturer’s protocol, resulting in immobilization levels of 4580 response units (RU). FC1 was only activated and inactivated and used as a reference. Celecoxib and dimethyl celecoxib stock solution was diluted to a final concentration of 200, 100, 50, 25, 12 μM and injected in 10 mM Hepes, 150 mM NaCl, 3 mM CaCl<sub>2</sub>, 1% DMSO, and 0.5% P20. Each injection was repeated three times for 60 s. FC1 signals were deducted from FC2 for background noise elimination.</div></div><div id="sec2_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Growth Inhibition of MDA-MB-231 Invasive Breast Cancer Cell Line Using MTS Assay</h4><div class="NLM_p last">MDA-MB-231 cells were seeded at 4000 cells/well in a 96-well plate. Stock of celecoxib and dimethyl celecoxib was diluted in DMEM + 10% FBS to make final concentrations used for treatment, and all concentrations were prepared to have the same amount of DMSO. Three wells per concentration were treated 24 h after seeding, and the MTS assay was performed 48 h after treatment. The CellTiter96 aqueous nonradioactive cell proliferation assay kit (Promega) was used according to the manufacturer’s recommendations. The absorbance values were measured at 490 nm and viable cells presented as a percentage of the absorbance of DMSO-only treated cells. IC<sub>50</sub> and <i>R</i><sup>2</sup> values were calculated with the Graphpad Prism software.</div></div><div id="sec2_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> VEGFR2 Kinase Assay</h4><div class="NLM_p">The VEGFR2 kinase assay was performed by using the Caliper LabChip 3000 and a 12-sipper LabChip. LabChip assays are separations-based, wherein the product and substrate are electrophoretically separated, thereby minimizing interference and yielding high data quality. <i>Z</i>′ factors for both the EZ reader and LC3000 enzymatic assays are routinely between 0.8 and 0.9. The off-chip incubation mobility-shift kinase assay uses a microfluidic chip to measure the conversion of a fluorescent peptide substrate to a phosphorylated product. The reaction mixture, from a microtiter well plate, is introduced through a capillary sipper onto the chip, where the nonphosphorylated substrate and phosphorylated product are separated by electrophoresis and detected via laser-induced fluorescence. The signature of the fluorescence signal over time reveals the extent of the reaction. The precision of microfluidics allows for the detection of subtle interactions between drug candidates and therapeutic targets.</div><div class="NLM_p last">The assay reaction is fluorescein-peptide + ATP → fluorescein-phosphopeptide + ADP, which is measured by charge separation of phosphorylated product and unphosphorylated substrate. The assay incubation is carried out in 100 mM HEPES (pH 7.5), 10 mM MgCl<sub>2</sub>, 1 mM DTT, 0.05% CHAPSO, 1.5 μM peptide substrate, 450 μM ATP, nine different concentrations of mebendazole, and staurosporine used as the positive control. Following addition of ATP, samples were incubated for 60 min at room temperature and the reaction was terminated by the addition of stop buffer containing 100 mM HEPES (pH 7.5), 7 mM EDTA, 0.015% Brij-35, 4% DMSO. Phosphorylated product and unphosphorylated substrate were separated by charge using electrophoretic mobility shift. The product formed is compared to control wells to determine inhibition or enhancement of enzyme activity.</div></div><div id="sec2_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Angiogenesis Assay</h4><div class="NLM_p last">Mebendazole was dissolved in 50 μL of DMSO and diluted with endothelial growth medium (EGM) to a final concentration of 1 mM. The highest concentration of DMSO is 0.1%. Human umbilical vein endothelial cells (HUVECs) were purchased from Cambrex Co. (East Rutherford, NJ, U.S.) and maintained in endothelial growth medium (EGM) supplemented with 2% FBS, 0.1% EGF, 0.1% hydrocortisone, 0.1% GA-1000, and 0.4% BBE. Morphogenesis assay on Matrigel was performed according to the manufacturer’s instructions (Chemicon International). The ECMatrix kit consists of laminin, collagen type IV, heparan sulfate, proteoglycans, entactin, and nidogen. It also contains various growth factors (TGF-β, FGF) and proteolytic enzymes (plasminogen, tPA, and MMPs) that are normally produced in EHS tumors. The incubation condition was optimized for maximal tube formation as follows: 50 μL of ECMatrix was suitably diluted in a 9:1 ratio with 10× diluent buffer and used to coat the 96-well plate. The coated plates were incubated at 37 °C for 1 h to allow the matrix solution to solidify. HUVECs were cultured for 24 h in EGM with 2% FBS, trypsinized, and resuspended in the growth medium. After 1 h of preincubation of the plate with Matrix solution, the HUVECs were plated at 5 × 10<sup>5</sup> cells/well in the absence or in the presence of mebendazole (1–100 μM). After 24 h of incubation at 37 °C, the three-dimensional organization (cellular network structures) was examined under an inverted photomicroscope. Each treatment was performed in triplicate.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have developed a new proteochemometric method called “TMFS” that utilizes a comprehensive receptor- and ligand-centric approach to predict molecules of “good fit”. The detailed scheme is displayed in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. Briefly, in the receptor-centric approach, we collected ∼11 000 X-ray 3D structures (human–liganded proteins) from the PDB database (<a href="http://www.rcsb.org" class="extLink">www.rcsb.org</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0003.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Graphical summary of the TMFS method work flow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">After filtering (see <a class="ref internalNav" href="#sec2" aria-label="Materials and Methods">Materials and Methods</a> for details), we included 2335 unique protein structures for the receptor- and ligand-centric simulations. We docked 3671 FDA approved and clinical trial drugs as reported in DrugBank, BindingDB, and FDA.org. Given that an important factor in determining a molecule of “good fit” is the similarity of its shape to those of the bioactive conformation of the reference ligand and ligand-binding pocket, we integrated receptor and ligand shape descriptors and similarity quantification into TMFS scoring. To quantify a molecule’s shape similarity, we employed the Euclidean distance metric as described in Kharaman et al.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> We calculated the binding pocket, ligand, and drugs shape using sc-PDB<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and SurFlexDock Protomol generator within SYBYL X.1.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Then we collected and integrated the docking, shape, and ligand descriptor similarity score data. Independently, the ligand–protein contact point scores were calculated using our “OLIC” method. Each data set score was normalized between 0 and 1. Then, using eq <a class="ref internalNav" href="#eq8" aria-label="8">8</a>, we computed a final ranking score, the comprehensive TMFS score “<i>Z</i>” that gives molecules of “good fit”.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Analysis of Accuracy of the TMFS Method Using ROC</h3><div class="NLM_p">Next we examined the precision of the TMFS method to ascertain if it substantially enriches the number of active compounds detected at the top of the ranking list. Since our study involved 2335 unique proteins and 3671 drugs, the docked output has ∼8.4 million protein–ligand complexes for each docking protocol. We were interested in applying the TMFS method in conjunction with the most efficient docking algorithm to produce reliable results in the quickest time possible. To do this, we obtained a database of actives and decoys for estrogen receptor alpha (ERα) from the DUD (<a href="http://dud.docking.org/" class="extLink">http://dud.docking.org/</a>), which contains ∼3000 compounds.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Then we performed our computational prediction protocol on the crystal structure of the agonist conformation of estrogen receptor (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ERD">3ERD</a>) to determine if our TMFS method significantly enriches the number of known active compounds within the top 20 positions compared to the options provided solely in the Schrodinger software.</div><div class="NLM_p">First, we performed simple docking as our control. The Glide docking score yielded a high false positive rate (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). We then repeated the procedure with the “atom-pair similarity” of the probe ligands. This procedure reranks docked compounds according to their atom-pair similarity to a template ligand, in this case the cocrystallized ligand in its active conformation. The conformers of the probe molecules in these two steps were simply the minimized structures from the LigPrep application. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A, atom-pair similarity reranking did not provide significant enrichment over the pure Glide docking score. Next, we wanted to determine if choosing probe molecule conformers whose shapes most closely matched that of the reference ligand would result in greater enrichment. We prepared an exhaustive conformer library for each drug and chose the conformer whose shape most closely matched that of the active conformation of the crystallized ligand for each protein. With the new, individualized “conformer set” of FDA drugs for ERα, we repeated the Glide docking procedure with atom-pair similarity. This method significantly enriched the number of active compounds in the top 20 positions. Interestingly, when the shape parameters were considered in isolation, the enrichment was also significant. Finally, since these three procedures are receptor-centric, we added the ligand-centric approach to the last procedure to see if our combined receptor–ligand centric method would result in maximal enrichment. That is, we incorporated the ligand-based descriptor similarities to the reference ligand as well as probe-ligand-to-reference-ligand and probe-ligand-to-ligand-pocket shape (protein binding site) similarities to the docking score of the refined molecule database with atom-pair similarity. We applied computed values of all the receptor- and ligand-centric results unique to the TMFS method to solve the comprehensive ranking score “<i>Z</i>” (eq <a class="ref internalNav" href="#eq8" aria-label="8">8</a>). The resulting top 20 rankings from each of these four procedures were plotted on a receiver operator curve (ROC). From the ROC (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A), we were able to determine the effect of our prediction method on the enrichment of true positives over false positives. In comparison to all the individual descriptor approaches (i.e., ligand-centric or shape descriptors only), this combined approach showed the greatest enrichment of active compounds versus decoys and served to validate our TMFS method.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Accuracy of the TMFS method represented by ROC curves. We examined the TMFS method accuracy against the Glide docking scoring function. Here we report, in increasing order of enrichment of true bioactive compounds, the performance of each scoring method via their respective AUC: Glide score (0.3889, red), Glide score + atom pair (AP) similarity (0.3889, yellow), shape descriptors only (0.6905, teal), ligand-centric descriptors only (0.7500, blue), Glide score + AP similarity + post-shape (0.8167, green), and TMFS score (0.8810, purple). (B) Validation of predicted drug–target associations for FDA approved drugs. Predicted drug–target associations for each FDA drug in the top 1 through top 40 ranked hits were individually matched against the publically available experimental binding and functional data. Percent correctly predicted (PCP) targets were then calculated using eq <a class="ref internalNav" href="#eq9" aria-label="9">9</a> for each category of the top rank lists (i.e., top 1 to top 40). The histogram (filled bars) represents the percent correctly predicted (PCP) targets (<i>y</i>-axis) for each category of top rank lists. Error bars and percentages are highlighted on each histogram bar.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">While we understand that ROC enrichment performance analysis was performed on a single instance of the DUD, the purpose of using this well-established target (ERα) was for a proof-of-concept for our TMFS approach. As will be explained in the subsequent section, the application of TMFS to the 2335 human protein crystal structures is to validate our method across the largest and most diverse data set we could obtain. In fact, our protein target set included 21 out of the 40 total DUD targets to date with respect to protein nomenclature. Furthermore, as depicted in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 1">Supporting Information Table 1</a>, our protein target set also includes many different instances of those DUD targets. For example, PPARγ is represented in our protein target set through five unique PDB crystal structures: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KNU">1KNU</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NYX">1NYX</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS8">3CS8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUR">3FUR</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K8S">3K8S</a>. Although ROC enrichment performance analyses were not conducted for these other DUD targets, they are included in the large validation, which contributed to the final accuracy score (see next section). We therefore found the incorporation of these DUD targets in this fashion to be of greater value for our validation than a smaller number of individual ROC comparisons.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Validation of the TMFS Method Using Publically Available Literature</h3><div class="NLM_p">To validate predicted drug–target signatures, we used manual, structural, and automated text curation to exhaustively search DrugBank, BindingDB, ChemBL, PubChem, PDSP, and other published literature where experimental binding/functional data are available for FDA drugs. Since our drug data sets are associated with the DrugBank ID, we downloaded the most recently updated “drugcard” from DrugBank, which included information for each drug with respect to their targets and PDB IDs if available. This file was parsed into individual “drugcards” that correspond to each individual compound in the database. We subsequently took DrugBank ID/PDB ID combinations for each FDA molecule found in the top 1 through top 40 lists for each target and searched for matches across every individual “drugcard”. We recorded a successful prediction for every match occurrence and annotated the data to the corresponding drug in the top 1, top 10, top 20, top 30, and top 40 ranked lists. Then we used “significance analysis” (eq <a class="ref internalNav" href="#eq9" aria-label="9">9</a>) to obtain the percent correctly predicted (PCP) drug–target signatures. A similar procedure was used to curate and validate other experimental bioassay databases (see following section).</div><div class="NLM_p">One caveat of our approach is that this validation method only contains a subset of the overall potential successful predictions. This is because many of the predicted hits have not been tested experimentally; some of the hits listed in the databases are not included in our target data sets, and although a protein target may have many crystal structures deposited in the PDB, experimental data sources report only a single PDB ID for a target associated with a drug. To our knowledge, there is no reconciliatory method in the PDB to match PDB IDs to a single protein name, since each PDB file contains a different variation of the target name. In other words, we were unable to aggregate all the PDB IDs for a single protein target. Although it may be possible to perform a “fuzzy” text search, this process would decrease accuracy and is beyond the scope of our work. In addition our FDA-approved drug database is only a subset (54%) of the entire DrugBank database, which results in a skewed representation of this validation process. Similarly, we do not have all of the PDB IDs that are found in the entire DrugBank database. This is because our final protein target database was contingent upon our compiled target data set of 2335. Furthermore, we have not included universal target binding component proteins such as the multidrug resistant (MDR) protein cytochrome p450. These caveats pose an interesting challenge in performing an optimal validation check. To address these concerns in the validation, we have developed a statistical procedure specifically for this kind of validation task; eq <a class="ref internalNav" href="#eq9" aria-label="9">9</a> calculates the percent correctly predicted (PCP) targets.</div><div class="NLM_p last">We repeated the structural and automated text curation to exhaustively search DrugBank, BindingDB, ChemBL, PubChem, PDSP, and other published literatures. Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B depicts the percentage of targets correctly predicted by the TMFS method across all the databases. To obtain this percentage, we counted the number of matched and unmatched pairs and also determined the excluded/included missing targets in terms of their protein target name, drug name or structure, or PDB code. Then we substituted this number in eq <a class="ref internalNav" href="#eq9" aria-label="9">9</a>. This number was used as the total possible validations for each drug. Upon analysis of the results and validation generated using eq <a class="ref internalNav" href="#eq9" aria-label="9">9</a>, we reliably reproduced many experimentally validated drug–target associations (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B). We achieved >91% correct predictions for the majority of drugs. A classical example of some of the literature-validated (>95% target hits) include staurosporine, genistein, paricalcitol, ethoxzolamide, alitretinoin, and drospirenone. The first ranked drug for each target is 58.5% correctly predicted followed by 65%, 77%, 85%, and 91% for the top 10, 20, 30, and 40, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B). Prediction of a first-ranked drug is more sensitive to descriptor prediction value error such that a slight variation in the descriptor error will affect the correct prediction. For reference, we have included the data for all top-ranked staurosporine–protein target interactions in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table 2">Supporting Information Table 2</a>. In contrast, ranking within the top 40 is the least sensitive, and this error is more or less balanced. This is quite remarkable considering the completely in silico nature of the screening and the experimental vagaries for many of the interactions.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Performance of the TMFS Method</h3><div class="NLM_p">To determine how well the receptor- and ligand-centric component descriptors (11 in total) correlate in obtaining a reliable prediction of hits for a given protein target, we performed a principle component analysis (PCA). The data set for PCA comprised the normalized descriptor values for the top 40 hits with respect to their predicted targets (<i>n</i> = 2335). To reduce the dimensionality of the data, we first performed a “scree plot” to determine how many principle components explain most of the variance in the data. We found that the first three principle components account for approximately 76% of the data variance (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Thus, we plotted the transformed eigenvalue coefficients of the above descriptors against the first three principle components (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure 1">Supporting Information Figure 1</a>). All descriptors, with the exception of docking score and hydrogen bond donor (DonorHB), exhibit a positive correlation across all three components. This implies that most of the ligand-centric descriptors and ligand/protein-centric shape descriptors correlate well with each other in determining a “molecule of good fit” for a given target. Furthermore, the docking score descriptor deviates from the rest of the descriptors. In other words, the protein–ligand complex that has the lowest-energy pose is not necessarily (or even likely to be) the one with the best fit. The docking score is a raw energetic term that takes into account the energetics of interactions between a ligand and protein where important parameters such as solvent, entropy, and enthalpy are absent. In contrast, the TMFS method includes protein and ligand topology descriptors in addition to energetic terms. Hence, a reliable prediction algorithm would benefit from a comprehensive approach that takes into account both receptor- and ligand-centric descriptors.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Principle component analysis (PCA) of individual protein- and ligand-based descriptor variables for determination of descriptor correlation with obtaining reliable predictions. Scree plot depicting the first three principal components accounting for the majority of the data variance. The first three principle components account for the majority of the data variance; hence, the transformed eigenvalue coefficients of the above descriptor variables were plotted against the first three principle components in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure 1">Supporting Information Figure 1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Analysis of Drug–Target Associations</h3><div id="sec3_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Drug Frequency Propensity</h4><div class="NLM_p">Next we analyzed predicted drug–target associations (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) in terms of drug frequency propensity for each target (i.e., drugs that interact with multiple targets with potentially good or bad effects). If a side effect is desirable, the drug might be repurposed for an additional indication. Targets are considered as hits if the TMFS rank places it in the top 1 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A) to top 40 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B). The broad-spectrum kinase inhibitor staurosporine was predicted to hit the most protein targets in the top 1 position. This finding is consistent with previous reports that staurosporine is a prototypical ATP-competitive multikinase inhibitor, and 8% of our PDB data set comprises kinase-like structures.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Some clinical drugs with IDs denoted by DrugBank as DB02197, DB02916, and DB03376 are also predicted to hit many targets in the top 40 (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B). These structures are also ATP-like, which is not surprising, as the ATP-/GTP-like small molecules bind naturally to many proteins. In addition, the PDB database is biased toward kinases where many of those kinase structures are cocrystallized with ATP, GTP, or closely related analogues. In an effort to further enrich our prediction paradigm, we included an rmsd term for ligand shape, where the rmsd of the docked ligand is compared to the active conformation of the cocrystallized ligand. Since this was a computational-intensive process, we randomly chose 100 protein targets. In this case, indomethacin, a nonsteroidal anti-inflammatory drug, was predicted to hit the highest number of protein targets in the top 1 and top 40 rankings (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Analysis of FDA drug–target association. Frequency histogram depicting the number of protein target hits (<i>y</i>-axis) for each FDA drug (<i>x</i>-axis). Targets are considered hits for a particular molecule if the final ranking (<i>Z</i>-score) of the molecule places it in the top 1 position or somewhere in the top 40 positions. (A) Frequency histogram depicting the number of protein targets hit (<i>y</i>-aixs) for each FDA drugs (<i>x</i>-axis) in the top 1 position. The 2D structure of staurosporine, the drug with the most hits, is also displayed. (B) Frequency histogram depicting the number of protein target hits (<i>y</i>-axis) for each FDA drug (<i>x</i>-axis) in the top 40 position. The top 40 provides a more relaxed criterion for protein targets to be considered as hits. As such, for those molecules that survive the final cut-offs and are found in the top 40 rank list for a particular protein, we predict that they have a good potential to bind to that given target. DB02197, DB03376, and DB02916, drugs with the most predicted hits are depicted. (C) To further enrich our prediction paradigm, we included one more term corresponding to ligand shape. The value for this term is the rmsd of the docked ligand compared to the active conformation of the cocrystallized ligand for a particular protein crystal structure, which is derived from a set of 100 protein targets. The histogram portrays the frequency of hits of each FDA drug along with the 2D structure of the drug with most target hits (indomethacin).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Predicted Drug–Target Associations in Terms of Drug Frequency Propensity for Each Target<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0001.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0002.gif" alt="" id="GRAPHIC-d2376e1650-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">The top 10 occurring molecules and the PDB identification codes corresponding to them for the top 1 cohort.</p></div></div><div></div></div></div><div id="sec3_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> FDA “Blockbuster” Drug–Target Associations</h4><div class="NLM_p">Next we predicted the top 200 “blockbuster” drug–target associations and determined their frequency of occurrence across the 2335 human protein targets within the top 1 to top 40 hits (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). Several “blockbuster” drugs are predicted to target proteins across multiple families. Sutent is predicted to hit the greatest number of protein targets followed by Alimta, Lescol, Celebrex, Premarin, Zetia, and Blopress (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). Sutent, the drug predicted to be the most promiscuous, is a multikinase inhibitor prescribed to treat various cancers. Remarkably, Prograf, Valcote, Concerta, Sifrol, Niaspan, Exelon, Evodart, Sevorane, and Klacid have no hits in our protein data set (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C). Out of 2335 targets, a blockbuster drug is ranked first for 79 targets (3.2%), ranked in the top 5 for 243 targets (10.9%), and ranked in the top 20 for 816 targets (36.5%) (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>D). Ninety-six targets (4.3%) are predicted to bind to three or more “blockbuster drugs”. Taken together, these results may imply either toxicity by cross-target effects or potentially beneficial effects that might indicate new uses. In addition to utility in drug repurposing, these data may also provide clues for the synthesis of analogues with higher specificity for particular targets if the repurposed drug shows weak affinity toward a particular target.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Analysis of FDA blockbuster drug-target association. (A) Heat map depicting hit frequencies of the top 200 “blockbuster” FDA drugs across each top-rank category. Each box shows the number of occurrences, while the color scheme illustrates high frequencies as red and low frequencies as blue. (B) Heat map showing FDA approved drugs predicted to hit the greatest number of protein targets: Sutent, Alimta, Lescol, Celebrex, Premarin, Zetia, and Blopress. (C) Heat map portraying FDA approved drugs that have no hits in our protein data set: Prograf, Valcote, Concerta, Sifrol, Niaspan, Exelon, Evodart, Sevorane, and Klacid. (D) Histogram showing the percentage of total protein targets in our data set that have a FDA blockbuster drug in their top 1, 5, and 20 rank lists.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Drug Promiscuity and Overlap of Protein Family and Fold</h4><div class="NLM_p">In drug development, it is important that molecules reach and interact with their desired targets while minimizing cross-target interactions. However, many FDA approved drugs have notable side effects that consumers are warned about prior to their administration. Thus, we were interested in investigating whether our method could more formally predict the extent of drug promiscuity/nonspecificity. We evaluated the extent of promiscuity in terms of protein family and fold classifications. We used the entire SCOP database and parsed it to create a CSV file that matches PDB IDs with their corresponding fold and family keys.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> For each molecule in the drug data set, we then determined the targets for which they are considered the top 1 hit and used those PDB IDs to determine the folds and families they correspond to. Using this information, we were able to determine the numbers of unique folds and families that the drugs are targeting. To objectively quantify the “promiscuity” of a molecule, we devised a numerical score to create the “value of promiscuity”. This value is the combined sum of the number of unique folds and the number of unique families that a particular molecule is predicted to hit. The greater this value is, the greater the extent of promiscuity. According to Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>, the three most promiscuous compounds (DB02197, DB03869, and staurosporine) are kinase inhibitors. As indicated above, staurosporine is a “broad-specificity kinase inhibitor” targeting multiple families especially kinases.<a onclick="showRef(event, 'ref40 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref42">(40, 42)</a> Furthermore, Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> show that the five most promiscuous drugs are predicted to interact with proteins that have many overlapping folds/families. Intrigued by this result, we further explored whether shape similarities of protein binding pockets may exhibit drug promiscuity.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Specific Protein Folds of Targets That TMFS Predicted for the Top Five Most Promiscuous Drugs</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep1" align="center">molecule</th><th class="colsep0 rowsep1" align="center">folds targeted</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DB02197</td><td class="colsep0 rowsep0" align="left">protein kinase-like, ATPase domain of HSP90 chaperone/DNA topoisomerase II, nudix, zincin-like, phosphotyrosine protein phosphatases II, P-loop containing nucleoside triphosphate hydrolases, NAD(P)-binding Rossman fold domains, TIM β/α-barrel, concanavalin A-likelectin/glucanases, prealbumin-like</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep1" align="left">DB03869</td><td class="colsep0 rowsep1" align="left">HD-domain/PDEase-like, protein kinase-like, ATPase domain of HSP90 chaperone/DNA topoisomerase II, P-loop containing nucleoside triphosphate hydrolases, TIM β/α-barrel, ferredoxin-like, anticodon-binding domain-like, carbonic anhydrase, trypsin-like serine proteases, nuclear receptor ligand-binding domain</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep1" align="left">DB02010</td><td class="colsep0 rowsep1" align="left">GRIP domain, P-loop containing nucleoside triphosphate hydrolases, TIM β/α-barrel, protein kinase-like, phosphotyrosine protein phosphatases II, trypsin-like serine proteases, HAD-like, SH2-like, ribonuclease H-like motif, eight-bladed β-propeller</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep1" align="left">DB00686</td><td class="colsep0 rowsep1" align="left">protein kinase-like, ATPase domain of HSP90 chaperone/DNA topoisomerase II, phosphotyrosine protein phosphatases II, P-loop containing nucleoside triphosphate hydrolases, phosphoglycerate mutase-like, lipocalins, cyclin-like, trypsin-like serine proteases, nuclear receptor ligand-binding domain</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep1" align="left">DB04700</td><td class="colsep0 rowsep1" align="left">protein kinase-like, ATPase domain of HSP90 chaperone/DNA topoisomerase II, carbonic anhydrase, nuclear receptor ligand-binding domain, phosphorylase/hydrolase-like, α/α toroid, transducin (α subunit), GST C-terminal domain-like, DNA/RNA-binding three-helical bundle</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Specific Protein Families of Targets That TMFS Predicted for the Top Five Most Promiscuous Drugs</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">molecule</th><th class="colsep0 rowsep0" align="center">families targeted</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DB02197</td><td class="colsep0 rowsep0" align="left">protein kinases (catalytic subunit), HSP90 (N-terminal domain), MutT-like, matrix metalloproteinases (catalytic domain), higher-molecular-weight phosphotyrosine protein phosphatases, motor proteins, phosphoribulokinase/pantothenate kinase, tyrosine-dependent oxidoreductases, aldo–keto reductases (NADP), galectin (animal S-lectin), transthyretin (prealbumin)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DB03869</td><td class="colsep0 rowsep0" align="left">protein kinases (catalytic subunit), HSP90 (N-terminal domain), aldo–keto reductases (NADP), NAD-binding domain of HMG-CoA reductase, ITPase (HAM1), G proteins, carbonic anhydrase, eukaryotic proteases, PDEase, nuclear receptor ligand-binding domain</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DB02010</td><td class="colsep0 rowsep0" align="left">protein kinases (catalytic subunit), higher-molecular-weight phosphotyrosine protein phosphatases, G proteins,aldo–keto reductases (NADP), eukaryotic proteases, GRIP domain, 5′(3′)-deoxyribonucleotidase(dNT-2), DPP6 N-terminal domain-like, BadF/BadG/BcrA/BcrD-like, SH2 domain</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DB00686</td><td class="colsep0 rowsep0" align="left">protein kinases (catalytic subunit), HSP90 (N-terminal domain), higher-molecular-weight phosphotyrosine protein phosphatases, G proteins,phosphoribulokinase/pantothenate kinase, eukaryotic proteases, nuclear receptor ligand-binding domain, cofactor-dependent phosphoglycerate mutase, fatty acid binding protein-like, cyclin</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DB04700</td><td class="colsep0 rowsep0" align="left">protein kinases (catalytic subunit), HSP90 (N-terminal domain), carbonic anhydrase, nuclear receptor ligand-binding domain, purine and uridine phosphorylases, terpene synthases, transducin (alpha subunit), glutathione <i>S</i>-transferase(C-terminal domain), methionine aminopeptidase</td></tr></tbody></table></div></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Analysis of drug promiscuity. The “value of promiscuity (nonspecificity) for each drug is represented as a numerical score from the combined sum of the number of unique folds and the number of unique families that a particular molecule is predicted to hit. The drug with the greatest “value of nonspecificity” is considered to be the most promiscuous molecule. The histogram depicts the “values of nonspecificity” (<i>y</i>-axis) for each drug (<i>x</i>-axis) that had been ranked in the top 1 position, along with the 2D structures of the three most promiscuous compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Similarly Shaped Protein Pockets Bind Similarly Shaped Molecules</h4><div class="NLM_p">In drug development, it is commonly asked if protein targets with similar binding sites bind similar molecules. Given the central dogma of “form fits function”, it is usually acceptable to assume this. To determine the similarity of binding sites between proteins, we calculated the Euclidean distances between the spherical harmonics (SH) expansion coefficients of the binding pockets at a 6 Å radius from neighboring residues. Binding site protomols were utilized from the sc-PDB database<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and SurFlexDock within SYBYL X.1.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> These protomols provide a space-filled 3D structure of the binding sites using carbon, hydrogen, and oxygen atoms. To get a more refined sense of the pockets, we removed the oxygen atoms because of their large atomic radii and proceeded with the SH calculations on the remaining carbon and hydrogen atoms. With the binding pocket shapes quantified, we then calculated the Euclidean distances between target binding pockets in which we took each target as a template pocket and calculated Euclidean distances against all the other probe target binding pockets. The most similar pockets were those whose Euclidean distances were closest to zero. Then to determine if similarly shaped pockets bind similar molecules, we took three approaches. First, we took drugs DB2010 (staurosporine) and DB02197 (the most frequent hits using the top 1 and top 40 rankings, respectively) and determined if the number of proteins with similarly shaped binding sites was greater than the number of proteins with lesser similarity. Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A is a clustered histogram illustrating this relationship and shows that more similarly shaped pockets exist for both drugs. Next, we analyzed how many targets intersect with the top five reprofiled drugs for each target. We defined this intersection to be the number of times a top five profiled drug for a reference target also appears in the top five rank lists with respect to the rest of the targets. We found that many targets have drugs that are common across the total protein data set (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B). Last, we analyzed the estrogen receptor α protein (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ERD">3ERD</a>) and vascular endothelial growth factor receptor (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P2H">2P2H</a>) binding pocket commonality with respect to the 2335 targets (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C and Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>D). We calculated Euclidean distances of the 2335 target pockets against these two template pocket structures and subsequently counted how many of the top 20 ranked molecules for each protein target were common to each template. We found that those pockets with the greatest shape similarity have the greatest number of drug hits in common with the templates (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C and Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>D). These results suggest that similarly shaped protein pockets bind similar molecules and indicate that a drug can be repurposed to other indications based in part upon similarly shaped targets.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0009.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Similarly shaped protein binding pockets bind similar molecules. (A) Histogram where the left-most protein target on the <i>x</i>-axis corresponded to the protein target whose pocket was most similar to the template. If these histograms tapered off to the right, this indicates that protein target–ligand commonality is highly correlated to the three-dimensional spatial similarity of their binding pockets. (B) Commonality of the top-ranked drugs. The predicted top 5 ranked drugs were counted for each target. Commonality is defined as the number of times a molecule from the top-rank list for a reference protein target also shows up in the corresponding top-rank list for the rest of the targets. The histogram depicts the “commonality score” for molecules within the top 5 rank list for each protein target data set. The top 5 protein targets (with respect to commonality score), which were cocrystallized with a nucleotide (4 out of 5 are GDP, one is adenosine), are highlighted with their PDB codes and name. (C) Histogram depicting the number of molecules in common for all protein targets ordered from greatest to least with respect to pocket shape similarity to VEGFR2. (D) Histogram depicting the number of molecules in common for all protein targets ordered from greatest to least with respect to pocket shape similarity to ERα.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Experimental Validation</h3><div id="sec3_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Mebendazole Binds to VEGFR-2 and Inhibits Angiogenesis</h4><div class="NLM_p">The TMFS method predicted that mebendazole, an antiparasitic with unexpected anticancer properties in animals and humans, had a strong likelihood of binding to and inhibiting the function of VEGFR2.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43, 44)</a> According to the final rank list of the 3671 drugs against VEGFR2, mebendazole was ranked higher than the related inhibitor albendazole. We therefore elected to proceed with this experimental validation based on this particular finding.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0010.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Mebendazole binds directly to VEGFR2 kinase assay and also inhibits angiogenesis. (A) Mebendazole binds directly to VEGFR2 and affects VEGFR2 kinase activity with an IC<sub>50</sub> of 3.6 μM. IC<sub>50</sub> curves were generated using GraphPad 5 and a standard four-parameter nonlinear regression model (log [inhibitor] vs response – variable slope). Data points correspond to the averages of duplicate wells, and error bars represent the mean ± replicate % activity. The graphical representation shows dashed lines at the IC<sub>50</sub> values, where the vertical line is at log <i>x</i> = −5.4437. Solving the equation log <i>x</i> = −5.4437 results in an IC<sub>50</sub> of 3.6 μM. (B) Control: Staurosporine binds to VEGFR2 and affects kinase activity with an IC<sub>50</sub> of 8 nM. (C) Mebendazole, predicted to act as a VEGFR2 inhibitor by TMFS, inhibits angiogenesis in a HUVEC cell based assay. Mebendazole significantly inhibited network formation with an IC<sub>50</sub> of 8.8 μM, which is implicated by the lack of cellular migration, alignment, and branching.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Indeed, our in vitro studies show that mebendazole binds to VEGFR-2 and inhibits kinase activity at 3.6 μM (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>A) with staurosporine serving as the control (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>B). We also found that mebendazole blocks angiogenesis in a human umbilical vein endothelial cell (HUVEC) based angiogenesis functional assay. The efficiency of mebendazole to inhibit the VEGFR2 kinase was measured by monitoring the ability of HUVECs to form networks. In the HUVEC angiogenesis assay, formation of the cellular network progresses in a stepwise manner with an initial migration and alignment of cells, followed by development of capillary tubelike structures, sprouting of new branches, and finally formation of a cellular network.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45, 46)</a> Cells treated with mebendazole did not migrate and align, sprout branches, or form networks (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>C) with an IC<sub>50</sub> of 8.8 μM. Albendazole, a close analogue of mebendazole, was previously demonstrated to inhibit angiogenesis.<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45-47)</a> In both assays, mebendazole is active at a concentration similar to that approved for use in preventing hookworm infection.</div></div><div id="sec3_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Celecoxib Binds to Cadherin-11 (CDH11)</h4><div class="NLM_p">To our complete surprise, an anti-inflammatory cycloxgenase-2 inhibitor celecoxib and its COX-2 inactive analogue dimethyl celecoxib (DMC) were ranked as top hits for interaction with CDH11, an adhesion molecule important in the inflammatory disease rheumatoid arthritis and in several poor prognosis malignancies. Consistent with its known role as a COX-2 inhibitor, celecoxib was rank-ordered as top 1 for COX-2. Celecoxib is already in use as an anti-inflammatory agent in arthritis where its activity is not solely related to inhibition of COX-2. We assessed the ability of dimethyl celecoxib and celecoxib to bind CDH11 using surface plasmon resonance (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>). Both celecoxib and, importantly, the closely related but inactive (with respect to COX-2 inhibition) dimethyl celecoxib interacted with CDH11 as measured by SPR. We further calculated the growth inhibition of the MDA-MB-231 invasive breast cancer cell line by celecoxib and DMC (Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a>A and Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a>B, respectively). We found that celecoxib and DMC cause inhibition with an IC<sub>50</sub> of 40 and 36 μM, respectively. These findings correlate well with experimental and clinical studies where dimethyl celecoxib works as well as celecoxib, which points to a COX-2-independent mode of action.<a onclick="showRef(event, 'ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50 ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59">(48-59)</a> The measured IC<sub>50</sub> values of celecoxib and DMC are comparable to known celecoxib plasma concentrations in patients. Circulating levels in rats and in humans are in the 1–10 μM range for celecoxib and are not known for dimethyl celecoxib. Importantly, the <i>K</i><sub>d</sub> for the known celecoxib target Cox2 is in the low nanomolar range for in vitro measurement of enzyme inhibition, and yet its effects on inflammation and cancer cell growth are in the micromolar range. Taken together with the fact that DMC has no effect on Cox2 and yet is equally effective as an anticancer agent and in some cases as an anti-inflammatory, these discrepancies strongly point to a Cox2-independent mode of action. Though the affinity of dimethyl celecoxib for CDH11 is weak enough, this is a potential starting point for further optimization.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0011.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Celecoxib (CCB) and dimethyl celecoxib (DMC) bind directly to immobilized cadherin-11 (CDH11) in surface plasmon resonance (SPR) assay. (A) CCB and DMC bind to recombinant mouse extracellular domain (EC) 1-2 of CDH11 protein immobilized on the surface of the chip via similar patterns, as evident in the sensogram. CCB and DMC were separately injected three times on the CM5 chip at 25 μM. (B, C) CCB and DMC bind in a dose-dependent manner. (B) Lower magnification of the sensogram showing the signals generated from 200 and 100 μM DMC bound to cadherin-11. (C) Higher magnification of the compacted signals from panel A showing the binding levels of 50, 25, and 12 μM DMC to cadherin-11. Assays were performed in triplicate for each DMC concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0012.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Growth inhibition of MDA-MB-231 invasive breast cancer cell line by celecoxib (CCB) and its COX-2 inactive analogue dimethyl celecoxib (DMC). MTS assays demonstrate concentration-dependent cell growth inhibition when MDA-MB-231 cells were exposed to increasing doses of CCB or DMC for 48 h. Data are presented as the mean ± SEM. (A) CCB causes inhibition with an IC<sub>50</sub> of 40 μM, and (B) DMC causes inhibition with an IC<sub>50</sub> of 36 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have developed a new computational method called “TMFS” that includes a docking score, ligand and receptor shape/topology descriptor scores, and ligand–receptor contact point scores to predict “molecules of best fit” and filter out most false positive interactions. Using our TMFS method, we reprofiled 3671 FDA approved/experimental drugs against 2335 human protein targets. We predicted several drug–target associations that could potentially be applied to new disease indications. Literature validation using public databases reveals that the TMFS method predicts drug–target associations with greater than 91% accuracy for the majority of drugs. We predicted and experimentally validated that the anti-hookworm medication mebendazole can inhibit VEGFR2 activity and angiogenesis and that the anti-inflammatory drug celcoxib and its analogue DMC can bind to CDH11, a biomolecule that is very important in rheumatoid arthritis and poor prognosis malignancies and for which no targeted therapies currently exist. TMFS-predicted drug–target associations not only reveal potential drug candidates for new indications but also provide structural insight into their mechanism of action and cross-target effects.</div><div class="NLM_p">Despite these promising results, it is imperative that we discuss the current limitations of our method. TMFS is reliant on the presence of a crystallized protein–ligand complex, where the bioactive conformations of both the protein pocket and ligand are known. This information is important for obtaining descriptor values, such as shape, that are as close to the natural occurrence as possible. However, we understand that it may be difficult to apply TMFS to emerging targets that do not have an X-ray structure or known low molecular weight compounds that modulate them. In an attempt to address these issues, we are currently working on fine-tuning TMFS so that it may reliably predict ligand–protein interactions for targets crystallized without a known associated ligand. Since the shape of the binding pocket will be known in this scenario, TMFS would be able to rely mostly on shape differences between the binding pocket and query ligands, for, as we have shown in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A, the shape descriptor alone is able to provide significant enrichment.</div><div class="NLM_p last">Since we started our work, the NIH’s Chemical Genomics Center (NCGC) opened its Pharmaceutical Collection database for public screening of nearly 27 000 active pharmaceutical ingredients, including 2750 approved small-molecule drugs and all compounds registered for human clinical trials.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In this study, we screened only 3671 compounds. We are currently extending our computational screening to include these 27 000 active pharmaceutical ingredients to all human proteins and other species, including infectious diseases. An obvious advantage of drug repositioning is that existing drugs approved for human use have known pharmacokinetics, toxicity, and safety profiles. Hence, any newly identified use can be rapidly evaluated directly in phase II clinical trials, thus reducing time and cost. Although clinical studies will be needed before a drug can be approved for a new indication, this work shows that computational screening of approved drugs can uncover additional uses for other targets/diseases.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46"><a href="/doi/suppl/10.1021/jm300576q">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25764" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25764" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">PCA plot, table of PDB codes, and table of raw score values.  This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300576q/suppl_file/jm300576q_si_001.pdf">jm300576q_si_001.pdf (297.56 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm300576q" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sivanesan Dakshanamurthy</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1e6d7a2c2d2d5e797b716c797b6a716970307b7a6b"><span class="__cf_email__" data-cfemail="0271663031314265676d706567766d756c2c676677">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Naiem T. Issa</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shahin Assefnia</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ashwini Seshasayee</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Georgetown University, Washington, D.C. 20057, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oakland J. Peters</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Subha Madhavan</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aykut Uren</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Milton L. Brown</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen W. Byers</span> - <span class="hlFld-Affiliation affiliation">Department of Oncology, Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, D.C.
20057, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2376e2093-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15520" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15520" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This project has been funded in part with federal funds from the National Institutes of Health Grants NIH R01-CA129813, NIH R01-DK58196, NIH P01CA130821, NIH Contract No. HHSN261200800001E, No. NIH 1KL2RR031974, and the Department of Defense Grants BC062416-01 and W81XWH-10-1-0437. We acknowledge the Georgetown-Lombardi Cancer Center Grant Support Grant and the Surface Plasmon Resonance Shared Resource and Caliper Life Sciences for the kinase assay. We also thank reviewers for their fruitful comments.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62032" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62032" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 59 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Paul, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mytelka, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunwiddie, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Persinger, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munos, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindborg, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schacht, A. L.</span><span> </span><span class="NLM_article-title">How to improve R&D productivity: the pharmaceutical industry’s grand challenge</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">214</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnrd3078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=20168317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=203-214&author=S.+M.+Paulauthor=D.+S.+Mytelkaauthor=C.+T.+Dunwiddieauthor=C.+C.+Persingerauthor=B.+H.+Munosauthor=S.+R.+Lindborgauthor=A.+L.+Schacht&title=How+to+improve+R%26D+productivity%3A+the+pharmaceutical+industry%E2%80%99s+grand+challenge"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">How to improve R&D productivity: the pharmaceutical industry's grand challenge</span></div><div class="casAuthors">Paul, Steven M.; Mytelka, Daniel S.; Dunwiddie, Christopher T.; Persinger, Charles C.; Munos, Bernard H.; Lindborg, Stacy R.; Schacht, Aaron L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">203-214</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is under growing pressure from a range of environmental issues, including major losses of revenue owing to patent expirations, increasingly cost-constrained healthcare systems and more demanding regulatory requirements.  In our view, the key to tackling the challenges such issues pose to both the future viability of the pharmaceutical industry and advances in healthcare is to substantially increase the no. and quality of innovative, cost-effective new medicines, without incurring unsustainable R&D costs.  However, it is widely acknowledged that trends in industry R&D productivity have been moving in the opposite direction for a no. of years.  Here, we present a detailed anal. based on comprehensive, recent, industry-wide data to identify the relative contributions of each of the steps in the drug discovery and development process to overall R&D productivity.  We then propose specific strategies that could have the most substantial impact in improving R&D productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY67PCIhaBVLVg90H21EOLACvtfcHk0ljw-hLfrV_JrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFemsbg%253D&md5=2f32bcc48c869290eef18ff9400afcc5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3078%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DS.%2BM.%26aulast%3DMytelka%26aufirst%3DD.%2BS.%26aulast%3DDunwiddie%26aufirst%3DC.%2BT.%26aulast%3DPersinger%26aufirst%3DC.%2BC.%26aulast%3DMunos%26aufirst%3DB.%2BH.%26aulast%3DLindborg%26aufirst%3DS.%2BR.%26aulast%3DSchacht%26aufirst%3DA.%2BL.%26atitle%3DHow%2520to%2520improve%2520R%2526D%2520productivity%253A%2520the%2520pharmaceutical%2520industry%25E2%2580%2599s%2520grand%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D203%26epage%3D214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Lawrence, S.</span><span> </span><span class="NLM_article-title">Drug output slows in 2006</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1073</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1073&author=S.+Lawrence&title=Drug+output+slows+in+2006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLawrence%26aufirst%3DS.%26atitle%3DDrug%2520output%2520slows%2520in%25202006%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasgar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span> </span><span class="NLM_article-title">The NCGC Pharmaceutical Collection: a comprehensive resource of clinically approved drugs enabling repurposing and chemical genomics</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">80</span><span class="NLM_x">) </span> <span class="NLM_fpage">80ps16</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscitranslmed.3001862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21525397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntleksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=80ps16&issue=80&author=R.+Huangauthor=N.+Southallauthor=Y.+Wangauthor=A.+Yasgarauthor=P.+Shinnauthor=A.+Jadhavauthor=D.+Nguyenauthor=C.+Austin&title=The+NCGC+Pharmaceutical+Collection%3A+a+comprehensive+resource+of+clinically+approved+drugs+enabling+repurposing+and+chemical+genomics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The NCGC Pharmaceutical Collection: a comprehensive resource of clinically approved drugs enabling repurposing and chemical genomics</span></div><div class="casAuthors">Huang, Ruili; Southall, Noel; Wang, Yuhong; Yasgar, Adam; Shinn, Paul; Jadhav, Ajit; Nguyen, Dac-Trung; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">80</span>),
    <span class="NLM_cas:pages">80ps16, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Small-mol. compds. approved for use as drugs may be "repurposed" for new indications and studied to det. the mechanisms of their beneficial and adverse effects.  A comprehensive collection of all small-mol. drugs approved for human use would be invaluable for systematic repurposing across human diseases, particularly for rare and neglected diseases, for which the cost and time required for development of a new chem. entity are often prohibitive.  Previous efforts to build such a comprehensive collection have been limited by the complexities, redundancies, and semantic inconsistencies of drug naming within and among regulatory agencies worldwide; a lack of clear conceptualization of what constitutes a drug; and a lack of access to phys. samples.  We report here the creation of a definitive, complete, and nonredundant list of all approved mol. entities as a freely available electronic resource and a phys. collection of small mols. amenable to high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPSN4jSGue67Vg90H21EOLACvtfcHk0ljw-hLfrV_JrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntleksr4%253D&md5=32b2767a0d1e74c000ca55853aec37e4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3001862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3001862%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DAustin%26aufirst%3DC.%26atitle%3DThe%2520NCGC%2520Pharmaceutical%2520Collection%253A%2520a%2520comprehensive%2520resource%2520of%2520clinically%2520approved%2520drugs%2520enabling%2520repurposing%2520and%2520chemical%2520genomics%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26issue%3D80%26spage%3D80ps16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Collins, F. S.</span><span> </span><span class="NLM_article-title">Mining for therapeutic gold</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">395</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=395&issue=6&author=F.+S.+Collins&title=Mining+for+therapeutic+gold"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DF.%2BS.%26atitle%3DMining%2520for%2520therapeutic%2520gold%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26issue%3D6%26spage%3D395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">DiMasi, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grabowski, H. G.</span><span> </span><span class="NLM_article-title">The price of innovation: new estimates of drug development costs</span> <span class="citation_source-journal">J. Health Econ.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS0167-6296%2802%2900126-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12606142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A280%3ADC%252BD3s%252FpsFSisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=151-185&issue=2&author=J.+A.+DiMasiauthor=R.+W.+Hansenauthor=H.+G.+Grabowski&title=The+price+of+innovation%3A+new+estimates+of+drug+development+costs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The price of innovation: new estimates of drug development costs</span></div><div class="casAuthors">DiMasi Joseph A; Hansen Ronald W; Grabowski Henry G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of health economics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">151-85</span>
        ISSN:<span class="NLM_cas:issn">0167-6296</span>.
    </div><div class="casAbstract">The research and development costs of 68 randomly selected new drugs were obtained from a survey of 10 pharmaceutical firms.  These data were used to estimate the average pre-tax cost of new drug development.  The costs of compounds abandoned during testing were linked to the costs of compounds that obtained marketing approval.  The estimated average out-of-pocket cost per new drug is 403 million US dollars (2000 dollars).  Capitalizing out-of-pocket costs to the point of marketing approval at a real discount rate of 11% yields a total pre-approval cost estimate of 802 million US dollars (2000 dollars).  When compared to the results of an earlier study with a similar methodology, total capitalized costs were shown to have increased at an annual rate of 7.4% above general price inflation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcReRsWHw2uElYhLSh9fo81bfW6udTcc2eYzx5BFODt79rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s%252FpsFSisQ%253D%253D&md5=814fa4d06a749145df19ae7492bf323d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0167-6296%2802%2900126-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0167-6296%252802%252900126-1%26sid%3Dliteratum%253Aachs%26aulast%3DDiMasi%26aufirst%3DJ.%2BA.%26aulast%3DHansen%26aufirst%3DR.%2BW.%26aulast%3DGrabowski%26aufirst%3DH.%2BG.%26atitle%3DThe%2520price%2520of%2520innovation%253A%2520new%2520estimates%2520of%2520drug%2520development%2520costs%26jtitle%3DJ.%2520Health%2520Econ.%26date%3D2003%26volume%3D22%26issue%3D2%26spage%3D151%26epage%3D185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Buehler, L. K.</span><span> </span><span class="NLM_article-title">Advancing drug discovery—beyond design</span> <span class="citation_source-journal">Pharmagenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnslagu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=24-26&author=L.+K.+Buehler&title=Advancing+drug+discovery%E2%80%94beyond+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Advancing drug discovery - beyond design</span></div><div class="casAuthors">Buehler, Lukas K.</div><div class="citationInfo"><span class="NLM_cas:title">PharmaGenomics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">24, 26</span>CODEN:
                <span class="NLM_cas:coden">PHARCV</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Advanstar Communications, Inc.</span>)
        </div><div class="casAbstract">A review.  Drug discovery based upon high-throughput screening, combinatorial chem. and rational drug design faces not a practical but a conceptual hurdle: the redundancy inherent to biol. systems.  This conceptual hurdle is in the form of a structural and functional redundancy of metabolic pathways, signaling networks and protein complexes in living organisms.  All are based upon the combinatorial use of a limited set of structural motifs and domains in proteins.  The functional redundancy easily can be traced back to the way protein structures are formed.  At the mol. level, function follows form.  Single mutations often can change a similarity into a different one.  Often, proteins might not be highly specific and can bind many ligands and substrates, although with varying efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRhLatpc6utLVg90H21EOLACvtfcHk0li-htZ9nPWS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnslagu7s%253D&md5=56a1813f7a55420bfbd1d1de606a29b9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBuehler%26aufirst%3DL.%2BK.%26atitle%3DAdvancing%2520drug%2520discovery%25E2%2580%2594beyond%2520design%26jtitle%3DPharmagenomics%26date%3D2004%26volume%3D4%26spage%3D24%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Hughes, B.</span><span> </span><span class="NLM_article-title">2007 FDA drug approvals: a year of flux</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=107-109&issue=7&author=B.+Hughes&title=2007+FDA+drug+approvals%3A+a+year+of+flux"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DB.%26atitle%3D2007%2520FDA%2520drug%2520approvals%253A%2520a%2520year%2520of%2520flux%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26issue%3D7%26spage%3D107%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Ashburn, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thor, K. B.</span><span> </span><span class="NLM_article-title">Drug repositioning: identifying and developing new uses for existing drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">683</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnrd1468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=15286734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=673-683&author=T.+T.+Ashburnauthor=K.+B.+Thor&title=Drug+repositioning%3A+identifying+and+developing+new+uses+for+existing+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repositioning: identifying and developing new uses for existing drugs</span></div><div class="casAuthors">Ashburn, Ted T.; Thor, Karl B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">673-683</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biopharmaceutical companies attempting to increase productivity through novel discovery technologies have fallen short of achieving the desired results.  Repositioning existing drugs for new indications could deliver the productivity increases that the industry needs while shifting the locus of prodn. to biotechnol. companies.  More and more companies are scanning the existing pharmacopeia for repositioning candidates, and the no. of repositioning success stories is increasing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLkWl3DnePhrVg90H21EOLACvtfcHk0li-htZ9nPWS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D&md5=0145c02423ab81ace6e3b3d92585cccc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd1468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1468%26sid%3Dliteratum%253Aachs%26aulast%3DAshburn%26aufirst%3DT.%2BT.%26aulast%3DThor%26aufirst%3DK.%2BB.%26atitle%3DDrug%2520repositioning%253A%2520identifying%2520and%2520developing%2520new%2520uses%2520for%2520existing%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D673%26epage%3D683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Druker, B.</span><span> </span><span class="NLM_article-title">Imatinib as a paradigm of targeted therapies</span> <span class="citation_source-journal">Adv. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS0065-230X%2804%2991001-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=15327887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVent7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2004&pages=1-30&author=B.+Druker&title=Imatinib+as+a+paradigm+of+targeted+therapies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Imatinib as a paradigm of targeted therapies</span></div><div class="casAuthors">Druker, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-30</span>CODEN:
                <span class="NLM_cas:coden">ACRSAJ</span>;
        ISSN:<span class="NLM_cas:issn">0065-230X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Imatinib (Gleevec) exemplifies the successful development of a rationally designed, molecularly targeted therapy for the treatment of a specific cancer.  This article reviews the identification of the BCR-ABL tyrosine kinase as a therapeutic target in chronic myeloid leukemia and the steps in the development of an agent to specifically inactivate this abnormality.  The clin. trials results are reviewed along with a description of resistance mechanisms.  As imatinib also inhibits the tyrosine kinase activity of KIT and the platelet-derived growth factor receptors, the extension of imatinib to malignancies driven by these kinases will be described.  Issues related to clin. trials of molecularly targeted agents are discussed, including patient and dose selection.  Last, the translation of this paradigm to other malignancies is explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorfaGCiPz14bVg90H21EOLACvtfcHk0lhHdvIjmr1dew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVent7bP&md5=b9d2fd0bcc46afbf0dbe13e1c27100d8</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS0065-230X%2804%2991001-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-230X%252804%252991001-9%26sid%3Dliteratum%253Aachs%26aulast%3DDruker%26aufirst%3DB.%26atitle%3DImatinib%2520as%2520a%2520paradigm%2520of%2520targeted%2520therapies%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D2004%26volume%3D91%26spage%3D1%26epage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Young, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoyt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McWhinnie, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirn, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallarico, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span> </span><span class="NLM_article-title">Integrating high-content screening and ligand-target prediction to identify mechanism of action</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnchembio.2007.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18066055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVegtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=59-68&author=D.+Youngauthor=A.+Benderauthor=J.+Hoytauthor=E.+McWhinnieauthor=G.+Chirnauthor=C.+Taoauthor=J.+Tallaricoauthor=M.+Labowauthor=J.+Jenkinsauthor=T.+Mitchisonauthor=Y.+Feng&title=Integrating+high-content+screening+and+ligand-target+prediction+to+identify+mechanism+of+action"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating high-content screening and ligand-target prediction to identify mechanism of action</span></div><div class="casAuthors">Young, Daniel W.; Bender, Andreas; Hoyt, Jonathan; McWhinnie, Elizabeth; Chirn, Gung-Wei; Tao, Charles Y.; Tallarico, John A.; Labow, Mark; Jenkins, Jeremy L.; Mitchison, Timothy J.; Feng, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-content screening is transforming drug discovery by enabling simultaneous measurement of multiple features of cellular phenotype that are relevant to therapeutic and toxic activities of compds.  High-content screening studies typically generate immense datasets of image-based phenotypic information, and how best to mine relevant phenotypic data is an unsolved challenge.  Here, the authors introduce factor anal. as a data-driven tool for defining cell phenotypes and profiling compd. activities.  This method allows a large data redn. while retaining relevant information, and the data-derived factors used to quantify phenotype have discernable biol. meaning.  The authors used factor anal. of cells stained with fluorescent markers of cell cycle state to profile a compd. library and cluster the hits into seven phenotypic categories.  The authors then compared phenotypic profiles, chem. similarity and predicted protein binding activities of active compds.  By integrating these different descriptors of measured and potential biol. activity, the authors can effectively draw mechanism-of-action inferences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqIzrMvgKTaLVg90H21EOLACvtfcHk0lhHdvIjmr1dew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVegtbnE&md5=0456c454ed40d4c52382e5bf786a8346</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007.53%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DD.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DHoyt%26aufirst%3DJ.%26aulast%3DMcWhinnie%26aufirst%3DE.%26aulast%3DChirn%26aufirst%3DG.%26aulast%3DTao%26aufirst%3DC.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DLabow%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%26aulast%3DMitchison%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DIntegrating%2520high-content%2520screening%2520and%2520ligand-target%2520prediction%2520to%2520identify%2520mechanism%2520of%2520action%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D59%26epage%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Wagner, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peck, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramanathan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mootha, V.</span><span> </span><span class="NLM_article-title">Large-scale chemical dissection of mitochondrial function</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=343-351&author=B.+Wagnerauthor=T.+Kitamiauthor=T.+Gilbertauthor=D.+Peckauthor=A.+Ramanathanauthor=S.+Schreiberauthor=T.+Golubauthor=V.+Mootha&title=Large-scale+chemical+dissection+of+mitochondrial+function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DKitami%26aufirst%3DT.%26aulast%3DGilbert%26aufirst%3DT.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DRamanathan%26aufirst%3DA.%26aulast%3DSchreiber%26aufirst%3DS.%26aulast%3DGolub%26aufirst%3DT.%26aulast%3DMootha%26aufirst%3DV.%26atitle%3DLarge-scale%2520chemical%2520dissection%2520of%2520mitochondrial%2520function%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D343%26epage%3D351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Computational analysis of ligand relationships within target families</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">358</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2Fj.cbpa.2008.01.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18312862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ers7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=352-358&author=J.+Bajorath&title=Computational+analysis+of+ligand+relationships+within+target+families"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of ligand relationships within target families</span></div><div class="casAuthors">Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">352-358</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computational tools for the large-scale anal. and prediction of ligand-target interactions and the identification of small mols. having different selectivity profiles within target protein families complement research in chem. genetics and chemogenomics.  For computational anal. and design, such tasks require a departure from the traditional focus on single targets, hit identification, and lead optimization.  Recently, studies were reported that profile compds. in silico against arrays of targets or systematically map ligand-target space.  To identify small mol. probes that are suitable for chem. genetics applications, mol. diversity needs to be viewed in a way that partly differs from principles guiding conventional library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGFA5t6J8v3rVg90H21EOLACvtfcHk0lg6Vfw3I4ee3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ers7k%253D&md5=1d645ef300d472939c2d062ebad9f86e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.01.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.01.044%26sid%3Dliteratum%253Aachs%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520analysis%2520of%2520ligand%2520relationships%2520within%2520target%2520families%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D352%26epage%3D358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Oprea, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rintoul, M.</span><span> </span><span class="NLM_article-title">Systems chemical biology</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">447</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnchembio0807-447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17637771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvFaqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=447-450&author=T.+Opreaauthor=A.+Tropshaauthor=J.+Faulonauthor=M.+Rintoul&title=Systems+chemical+biology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Systems chemical biology</span></div><div class="casAuthors">Oprea, Tudor I.; Tropsha, Alexander; Faulon, Jean-Loup; Rintoul, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">447-450</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The increasing availability of data related to genes, proteins and their modulation by small mols. has provided a vast amt. of biol. information leading to the emergence of systems biol. and the broad use of simulation tools for data anal.  However, there is a crit. need to develop cheminformatics tools that can integrate chem. knowledge with these biol. databases and simulation approaches, with the goal of creating systems chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlQRRfUwh1ebVg90H21EOLACvtfcHk0lg6Vfw3I4ee3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvFaqs7Y%253D&md5=d47ca3c6aa344bfab403262e449645f2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0807-447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0807-447%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DFaulon%26aufirst%3DJ.%26aulast%3DRintoul%26aufirst%3DM.%26atitle%3DSystems%2520chemical%2520biology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D447%26epage%3D450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Siegel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Drugs in other drugs: a new look at drugs as fragments</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=71-79&author=M.+Siegelauthor=M.+Vieth&title=Drugs+in+other+drugs%3A+a+new+look+at+drugs+as+fragments"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DM.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DDrugs%2520in%2520other%2520drugs%253A%2520a%2520new%2520look%2520at%2520drugs%2520as%2520fragments%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D71%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Miller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochalkin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banotai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunkle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanna, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harwood, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huband, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karnovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limberakis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehrens, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narasimhan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skerlos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sulavik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VanderRoest, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stover, C.</span><span> </span><span class="NLM_article-title">A class of selective antibacterials derived from a protein kinase inhibitor pharmacophore</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">1737</span><span class="NLM_x">–</span> <span class="NLM_lpage">1742</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1073%2Fpnas.0811275106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19164768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1737-1742&author=J.+Millerauthor=S.+Dunhamauthor=I.+Mochalkinauthor=C.+Banotaiauthor=M.+Bowmanauthor=S.+Buistauthor=B.+Dunkleauthor=D.+Hannaauthor=H.+Harwoodauthor=M.+Hubandauthor=A.+Karnovskyauthor=M.+Kuhnauthor=C.+Limberakisauthor=J.+Liuauthor=S.+Mehrensauthor=W.+Muellerauthor=L.+Narasimhanauthor=A.+Ogdenauthor=J.+Ohrenauthor=J.+Prasadauthor=J.+Shellyauthor=L.+Skerlosauthor=M.+Sulavikauthor=V.+Thomasauthor=S.+VanderRoestauthor=L.+Wangauthor=Z.+Wangauthor=A.+Whittonauthor=T.+Zhuauthor=C.+Stover&title=A+class+of+selective+antibacterials+derived+from+a+protein+kinase+inhibitor+pharmacophore"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A class of selective antibacterials derived from a protein kinase inhibitor pharmacophore</span></div><div class="casAuthors">Miller, J. Richard; Dunham, Steve; Mochalkin, Igor; Banotai, Craig; Bowman, Matthew; Buist, Susan; Dunkle, Bill; Hanna, Debra; Harwood, James; Huband, Michael D.; Karnovsky, Alla; Kuhn, Michael; Limberakis, Chris; Liu, Jia Y.; Mehrens, Shawn; Mueller, W. Thomas; Narasimhan, Lakshmi; Ogden, Adam; Ohren, Jeff; Vara Prasad, J. V. N.; Shelly, John A.; Skerlos, Laura; Sulavik, Mark; Thomas, V. Hayden; VanderRoest, Steve; Wang, LiAnn; Wang, Zhigang; Whitton, Amy; Zhu, Tong; Stover, C. Kendall</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1737-1742</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the need for novel antibiotic classes to combat bacterial drug resistance increases, the paucity of leads resulting from target-based antibacterial screening of pharmaceutical compd. libraries is of major concern.  One explanation for this lack of success is that antibacterial screening efforts have not leveraged the eukaryotic bias resulting from more extensive chem. efforts targeting eukaryotic gene families such as G protein-coupled receptors and protein kinases.  Consistent with a focus on antibacterial target space resembling these eukaryotic targets, we used whole-cell screening to identify a series of antibacterial pyridopyrimidines derived from a protein kinase inhibitor pharmacophore.  In bacteria, the pyridopyrimidines target the ATP-binding site of biotin carboxylase (BC), which catalyzes the first enzymic step of fatty acid biosynthesis.  These inhibitors are effective in vitro and in vivo against fastidious Gram-neg. pathogens including Haemophilus influenzae.  Although the BC active site has architectural similarity to those of eukaryotic protein kinases, inhibitor binding to the BC ATP-binding site is distinct from the protein kinase-binding mode, such that the inhibitors are selective for bacterial BC.  In summary, the authors have discovered a promising class of potent antibacterials with a previously undescribed mechanism of action.  In consideration of the eukaryotic bias of pharmaceutical libraries, the findings also suggest that pursuit of a novel inhibitor leads for antibacterial targets with active-site structural similarity to known human targets will likely be more fruitful than the traditional focus on unique bacterial target space, particularly when structure-based and computational methodologies are applied to ensure bacterial selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8jpj8EOtRo7Vg90H21EOLACvtfcHk0lhNNHNGor8AWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D&md5=06758ee5befbe847825ec35adb177006</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811275106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811275106%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DDunham%26aufirst%3DS.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DBowman%26aufirst%3DM.%26aulast%3DBuist%26aufirst%3DS.%26aulast%3DDunkle%26aufirst%3DB.%26aulast%3DHanna%26aufirst%3DD.%26aulast%3DHarwood%26aufirst%3DH.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKarnovsky%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DMehrens%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DW.%26aulast%3DNarasimhan%26aufirst%3DL.%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DOhren%26aufirst%3DJ.%26aulast%3DPrasad%26aufirst%3DJ.%26aulast%3DShelly%26aufirst%3DJ.%26aulast%3DSkerlos%26aufirst%3DL.%26aulast%3DSulavik%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DV.%26aulast%3DVanderRoest%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWhitton%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DStover%26aufirst%3DC.%26atitle%3DA%2520class%2520of%2520selective%2520antibacterials%2520derived%2520from%2520a%2520protein%2520kinase%2520inhibitor%2520pharmacophore%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D1737%26epage%3D1742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Walsh, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischbach, M. A.</span><span> </span><span class="NLM_article-title">Repurposing libraries of eukaryotic protein kinase inhibitors for antibiotic discovery</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1690</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1689-1690&author=C.+T.+Walshauthor=M.+A.+Fischbach&title=Repurposing+libraries+of+eukaryotic+protein+kinase+inhibitors+for+antibiotic+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%2BT.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26atitle%3DRepurposing%2520libraries%2520of%2520eukaryotic%2520protein%2520kinase%2520inhibitors%2520for%2520antibiotic%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D1689%26epage%3D1690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Keiser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-182&author=M.+Keiserauthor=V.+Setolaauthor=J.+Irwinauthor=C.+Laggnerauthor=A.+Abbasauthor=S.+Hufeisenauthor=N.+Jensenauthor=M.+Kuijerauthor=R.+Matosauthor=T.+Tranauthor=R.+Whaleyauthor=R.+Glennonauthor=J.+Hertauthor=K.+Thomasauthor=D.+Edwardsauthor=B.+Shoichetauthor=B.+Roth&title=Predicting+new+molecular+targets+for+known+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0lhNNHNGor8AWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%26aulast%3DHufeisen%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DN.%26aulast%3DKuijer%26aufirst%3DM.%26aulast%3DMatos%26aufirst%3DR.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%26aulast%3DEdwards%26aufirst%3DD.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DRoth%26aufirst%3DB.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span> </span><span class="NLM_article-title">A computational approach to finding novel targets for existing drugs</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=1-13&author=Y.+Liauthor=J.+Anauthor=S.+Jones&title=A+computational+approach+to+finding+novel+targets+for+existing+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DS.%26atitle%3DA%2520computational%2520approach%2520to%2520finding%2520novel%2520targets%2520for%2520existing%2520drugs%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D1%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Cho, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeire, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merkel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lolis, E.</span><span> </span><span class="NLM_article-title">Drug repositioning and pharmacophore identification in the discovery of hookworm MIF inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1101</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2Fj.chembiol.2011.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21944748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1eju7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1089-1101&author=Y.+Choauthor=J.+Vermeireauthor=J.+Merkelauthor=L.+Lengauthor=X.+Duauthor=R.+Bucalaauthor=M.+Cappelloauthor=E.+Lolis&title=Drug+repositioning+and+pharmacophore+identification+in+the+discovery+of+hookworm+MIF+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Repositioning and Pharmacophore Identification in the Discovery of Hookworm MIF Inhibitors</span></div><div class="casAuthors">Cho, Yoonsang; Vermeire, Jon J.; Merkel, Jane S.; Leng, Lin; Du, Xin; Bucala, Richard; Cappello, Michael; Lolis, Elias</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1089-1101</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The screening of bioactive compd. libraries can be an effective approach for repositioning FDA-approved drugs or discovering new pharmacophores.  Hookworms are blood-feeding, intestinal nematode parasites that infect up to 600 million people worldwide.  Vaccination with recombinant Ancylostoma ceylanicum macrophage migration inhibitory factor (rAceMIF) provided partial protection from disease, thus establishing a "proof-of-concept" for targeting AceMIF to prevent or treat infection.  A high-throughput screen (HTS) against rAceMIF identified six AceMIF-specific inhibitors.  A nonsteroidal anti-inflammatory drug (NSAID), sodium meclofenamate, could be tested in an animal model to assess the therapeutic efficacy in treating hookworm disease.  Furosemide, an FDA-approved diuretic, exhibited submicromolar inhibition of rAceMIF tautomerase activity.  Structure-activity relationships of a pharmacophore based on furosemide included one analog that binds similarly to the active site, yet does not inhibit the Na-K-Cl symporter (NKCC1) responsible for diuretic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT30vfvK39a7Vg90H21EOLACvtfcHk0lhCGWNXVvESfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1eju7nL&md5=16e9c33e27c43f6f657c91e23f686114</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DY.%26aulast%3DVermeire%26aufirst%3DJ.%26aulast%3DMerkel%26aufirst%3DJ.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DCappello%26aufirst%3DM.%26aulast%3DLolis%26aufirst%3DE.%26atitle%3DDrug%2520repositioning%2520and%2520pharmacophore%2520identification%2520in%2520the%2520discovery%2520of%2520hookworm%2520MIF%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1089%26epage%3D1101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span> </span><span class="NLM_article-title">Fragment-based drug design and drug repositioning using multiple ligand simultaneous docking (MLSD): identifying celecoxib and template compounds as novel inhibitors of signal transducer and activator of transcription 3 (STAT3)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5592</span><span class="NLM_x">–</span> <span class="NLM_lpage">5596</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm101330h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5592-5596&author=H.+Liauthor=A.+Liuauthor=Z.+Zhaoauthor=Y.+Xuauthor=J.+Linauthor=D.+Jouauthor=C.+Li&title=Fragment-based+drug+design+and+drug+repositioning+using+multiple+ligand+simultaneous+docking+%28MLSD%29%3A+identifying+celecoxib+and+template+compounds+as+novel+inhibitors+of+signal+transducer+and+activator+of+transcription+3+%28STAT3%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm101330h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101330h%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DJou%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DFragment-based%2520drug%2520design%2520and%2520drug%2520repositioning%2520using%2520multiple%2520ligand%2520simultaneous%2520docking%2520%2528MLSD%2529%253A%2520identifying%2520celecoxib%2520and%2520template%2520compounds%2520as%2520novel%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520%2528STAT3%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5592%26epage%3D5596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Moriaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adcock, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chanas-Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgand, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jelloul, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delfaud, F.</span><span> </span><span class="NLM_article-title">Identify drug repurposing candidates by mining the Protein Data Bank</span> <span class="citation_source-journal">Briefings Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">336</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1093%2Fbib%2Fbbr017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21768131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=336-340&author=F.+Moriaudauthor=S.+Richardauthor=S.+Adcockauthor=L.+Chanas-Martinauthor=J.+Surgandauthor=M.+Jelloulauthor=F.+Delfaud&title=Identify+drug+repurposing+candidates+by+mining+the+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identify drug repurposing candidates by mining the Protein Data Bank</span></div><div class="casAuthors">Moriaud, Fabrice; Richard, Stephane B.; Adcock, Stewart A.; Chanas-Martin, Laetitia; Surgand, Jean-Sebastien; Ben Jelloul, Marouane; Delfaud, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">336-340</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Predicting off-targets by computational methods is gaining increasing interest in early-stage drug discovery.  Here, we present a computational method based on full 3D comparisons of 3D structures.  When a similar binding site is detected in the Protein Data Bank (PDB) (or any protein structure database), it is possible that the corresponding ligand also binds to that similar site.  On one hand, this target hopping case is probably rare because it requires a high similarity between the binding sites.  On the other hand, it could be a strong rational evidence to highlight possible off-target reactions and possibly a potential undesired side effect.  This target-based drug repurposing can be extended a significant step further with the capability of searching the full surface of all proteins in the PDB, and therefore not relying on pocket detection.  Using this approach, we describe how MED-SuMo reproduces the repurposing of tadalafil from PDE5A to PDE4A and a structure of PDE4A with tadalafil.  Searching for local protein similarities generates more hits than for whole binding site similarities and therefore fragment repurposing is more likely to occur than for drug-sized compds.  In this work, we illustrate that by mining the PDB for proteins sharing similarities with the hinge region of protein kinases.  The exptl. validated examples, biotin carboxylase and synapsin, are retrieved.  Further to fragment repurposing, this approach can be applied to the detection of druggable sites from 3D structures.  This is illustrated with detection of the protein kinase hinge motif in the HIV-RT non-nucleosidic allosteric site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCPSI-y0H9RbVg90H21EOLACvtfcHk0lhCGWNXVvESfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrY%253D&md5=d171b0356ec3b9931d15ff1f7610a5d4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbr017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbr017%26sid%3Dliteratum%253Aachs%26aulast%3DMoriaud%26aufirst%3DF.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DAdcock%26aufirst%3DS.%26aulast%3DChanas-Martin%26aufirst%3DL.%26aulast%3DSurgand%26aufirst%3DJ.%26aulast%3DJelloul%26aufirst%3DM.%26aulast%3DDelfaud%26aufirst%3DF.%26atitle%3DIdentify%2520drug%2520repurposing%2520candidates%2520by%2520mining%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DBriefings%2520Bioinf.%26date%3D2011%26volume%3D12%26spage%3D336%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Kinnings, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchmeier, N</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: repositioning the safe medicine comtan to treat multi-drug and extensively drug resistant tuberculosis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1000423</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000423&issue=7&author=S.+Kinningsauthor=N+Liuauthor=N+Buchmeierauthor=P.+Tongeauthor=L.+Xieauthor=P.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+repositioning+the+safe+medicine+comtan+to+treat+multi-drug+and+extensively+drug+resistant+tuberculosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DN%26aulast%3DBuchmeier%26aufirst%3DN%26aulast%3DTonge%26aufirst%3DP.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520repositioning%2520the%2520safe%2520medicine%2520comtan%2520to%2520treat%2520multi-drug%2520and%2520extensively%2520drug%2520resistant%2520tuberculosis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26issue%3D7%26spage%3De1000423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Campillos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gavin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span> </span><span class="NLM_article-title">Drug target identification using side-effect similarity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscience.1158140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18621671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=263-266&author=M.+Campillosauthor=M.+Kuhnauthor=A.+Gavinauthor=L.+Jensenauthor=P.+Bork&title=Drug+target+identification+using+side-effect+similarity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Target Identification Using Side-Effect Similarity</span></div><div class="casAuthors">Campillos, Monica; Kuhn, Michael; Gavin, Anne-Claude; Jensen, Lars Juhl; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5886</span>),
    <span class="NLM_cas:pages">263-266</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Targets for drugs have so far been predicted on the basis of mol. or cellular features, for example, by exploiting similarity in chem. structure or in activity across cell lines.  We used phenotypic side-effect similarities to infer whether two drugs share a target.  Applied to 746 marketed drugs, a network of 1018 side effect-driven drug-drug relations became apparent, 261 of which are formed by chem. dissimilar drugs from different therapeutic indications.  We exptl. tested 20 of these unexpected drug-drug relations and validated 13 implied drug-target relations by in vitro binding assays, of which 11 reveal inhibition consts. equal to less than 10 micromolar.  Nine of these were tested and confirmed in cell assays, documenting the feasibility of using phenotypic information to infer mol. interactions and hinting at new uses of marketed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbiFcs_camL7Vg90H21EOLACvtfcHk0lgO23Ymhlk9LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D&md5=d8fe131adeb39750cb97ee4e147de355</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1158140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1158140%26sid%3Dliteratum%253Aachs%26aulast%3DCampillos%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DGavin%26aufirst%3DA.%26aulast%3DJensen%26aufirst%3DL.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DDrug%2520target%2520identification%2520using%2520side-effect%2520similarity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D263%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Dudley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirota, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenoy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roedder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarwal, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasricha, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butte, A.</span><span> </span><span class="NLM_article-title">Computational repositioning of the anticonvulsant topiramate for inflammatory bowel disease</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">96</span><span class="NLM_x">) </span> <span class="NLM_fpage">96ra76</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscitranslmed.3002648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21849664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=96ra76&issue=96&author=J.+Dudleyauthor=M.+Sirotaauthor=M.+Shenoyauthor=R.+Paiauthor=S.+Roedderauthor=A.+Chiangauthor=A.+Morganauthor=M.+Sarwalauthor=P.+Pasrichaauthor=A.+Butte&title=Computational+repositioning+of+the+anticonvulsant+topiramate+for+inflammatory+bowel+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Computational repositioning of the anticonvulsant topiramate for inflammatory bowel disease</span></div><div class="casAuthors">Dudley, Joel T.; Sirota, Marina; Shenoy, Mohan; Pai, Reetesh K.; Roedder, Silke; Chiang, Annie P.; Morgan, Alex A.; Sarwal, Minnie M.; Pasricha, Pankaj Jay; Butte, Atul J.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">96</span>),
    <span class="NLM_cas:pages">96ra76</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Inflammatory bowel disease (IBD) is a chronic inflammatory disorder of the gastrointestinal tract for which there are few safe and effective therapeutic options for long-term treatment and disease maintenance.  Here, we applied a computational approach to discover new drug therapies for IBD in silico, using publicly available mol. data reporting gene expression in IBD samples and 164 small-mol. drug compds.  Among the top compds. predicted to be therapeutic for IBD by our approach were prednisolone, a corticosteroid used to treat IBD, and topiramate, an anticonvulsant drug not previously described to have efficacy for IBD or any related disorders of inflammation or the gastrointestinal tract.  Using a trinitrobenzenesulfonic acid (TNBS)-induced rodent model of IBD, we exptl. validated our topiramate prediction in vivo.  Oral administration of topiramate significantly reduced gross pathol. signs and microscopic damage in primary affected colon tissue in the TNBS-induced rodent model of IBD.  These findings suggest that topiramate might serve as a therapeutic option for IBD in humans and support the use of public mol. data and computational approaches to discover new therapeutic options for disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmF-DmOnq3dLVg90H21EOLACvtfcHk0lgO23Ymhlk9LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3J&md5=42dd418de306427e93735180e40288a9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3002648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3002648%26sid%3Dliteratum%253Aachs%26aulast%3DDudley%26aufirst%3DJ.%26aulast%3DSirota%26aufirst%3DM.%26aulast%3DShenoy%26aufirst%3DM.%26aulast%3DPai%26aufirst%3DR.%26aulast%3DRoedder%26aufirst%3DS.%26aulast%3DChiang%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DSarwal%26aufirst%3DM.%26aulast%3DPasricha%26aufirst%3DP.%26aulast%3DButte%26aufirst%3DA.%26atitle%3DComputational%2520repositioning%2520of%2520the%2520anticonvulsant%2520topiramate%2520for%2520inflammatory%2520bowel%2520disease%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26issue%3D96%26spage%3D96ra76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iskar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeller, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noort, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span> </span><span class="NLM_article-title">Prediction of drug combinations by integrating molecular and pharmacological data</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002323</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002323&issue=12&author=X.+Zhaoauthor=M.+Iskarauthor=G.+Zellerauthor=M.+Kuhnauthor=V.+Noortauthor=P.+Bork&title=Prediction+of+drug+combinations+by+integrating+molecular+and+pharmacological+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DIskar%26aufirst%3DM.%26aulast%3DZeller%26aufirst%3DG.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DNoort%26aufirst%3DV.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DPrediction%2520of%2520drug%2520combinations%2520by%2520integrating%2520molecular%2520and%2520pharmacological%2520data%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D12%26spage%3De1002323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Gottlieb, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruppin, E</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharan, R.</span><span> </span><span class="NLM_article-title">PREDICT: a method for inferring novel drug indications with application to personalized medicine</span> <span class="citation_source-journal">Mol. Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">496</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=496&author=A.+Gottliebauthor=G.+Steinauthor=E+Ruppinauthor=R.+Sharan&title=PREDICT%3A+a+method+for+inferring+novel+drug+indications+with+application+to+personalized+medicine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DG.%26aulast%3DRuppin%26aufirst%3DE%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DPREDICT%253A%2520a%2520method%2520for%2520inferring%2520novel%2520drug%2520indications%2520with%2520application%2520to%2520personalized%2520medicine%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Sirota, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet-Cordero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butte, A.</span><span> </span><span class="NLM_article-title">Discovery and preclinical validation of drug indications using compendia of public gene expression data</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">96</span><span class="NLM_x">) </span> <span class="NLM_fpage">96ra77</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscitranslmed.3001318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21849665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=96ra77&issue=96&author=M.+Sirotaauthor=J.+Dudleyauthor=J.+Kimauthor=A.+Chiangauthor=A.+Morganauthor=A.+Sweet-Corderoauthor=J.+Sageauthor=A.+Butte&title=Discovery+and+preclinical+validation+of+drug+indications+using+compendia+of+public+gene+expression+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and preclinical validation of drug indications using compendia of public gene expression data</span></div><div class="casAuthors">Sirota, Marina; Dudley, Joel T.; Kim, Jeewon; Chiang, Annie P.; Morgan, Alex A.; Sweet-Cordero, Alejandro; Sage, Julien; Butte, Atul J.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">96</span>),
    <span class="NLM_cas:pages">96ra77</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The application of established drug compds. to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval.  Recent approaches to drug repositioning use high-throughput exptl. approaches to assess a compd.'s potential therapeutic qualities.  Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of mol. signatures in drug-disease pairs.  We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compds., yielding predicted therapeutic potentials for these drugs.  We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs.  We exptl. validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models.  This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVAoZxuzFvS7Vg90H21EOLACvtfcHk0lgqRdxaW5BJeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3K&md5=73a95f1af7e6c2202eacb5e0f95eda9d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3001318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3001318%26sid%3Dliteratum%253Aachs%26aulast%3DSirota%26aufirst%3DM.%26aulast%3DDudley%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DChiang%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DSweet-Cordero%26aufirst%3DA.%26aulast%3DSage%26aufirst%3DJ.%26aulast%3DButte%26aufirst%3DA.%26atitle%3DDiscovery%2520and%2520preclinical%2520validation%2520of%2520drug%2520indications%2520using%2520compendia%2520of%2520public%2520gene%2520expression%2520data%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26issue%3D96%26spage%3D96ra77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J.</span><span> </span><span class="NLM_article-title">Docking screens: right for the right reasons?</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">770</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.2174%2F156802609789207091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19754393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=755-770&author=P.+Kolbauthor=J.+Irwin&title=Docking+screens%3A+right+for+the+right+reasons%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Docking screens: right for the right reasons?</span></div><div class="casAuthors">Kolb, Peter; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">755-770</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Whereas docking screens have emerged as the most practical way to use protein structure for ligand discovery, an inconsistent track record raises questions about how well docking actually works.  In its favor, a growing no. of publications report the successful discovery of new ligands, often supported by exptl. affinity data and controls for artifacts.  Few reports, however, actually test the underlying structural hypotheses that docking makes.  To be successful and not just lucky, prospective docking must not only rank a true ligand among the top scoring compds., it must also correctly orient the ligand so the score it receives is biophys. sound.  If the correct binding pose is not predicted, a skeptic might well infer that the discovery was serendipitous.  Surveying over 15 years of the docking literature, we were surprised to discover how rarely sufficient evidence is presented to establish whether docking actually worked for the right reasons.  The paucity of exptl. tests of theor. predicted poses undermines confidence in a technique that has otherwise become widely accepted.  Of course, solving a crystal structure is not always possible, and even when it is, it can be a lot of work, and is not readily accessible to all groups.  Even when a structure can be detd., investigators may prefer to gloss over an erroneous structural prediction to better focus on their discovery.  Still, the absence of a direct test of theory by expt. is a loss for method developers seeking to understand and improve docking methods.  We hope this review will motivate investigators to solve structures and compare them with their predictions whenever possible, to advance the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXLK_ZZmntsrVg90H21EOLACvtfcHk0lgqRdxaW5BJeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP&md5=3e6c0073795fae06c0d5a1175af8e8e5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2174%2F156802609789207091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609789207091%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%26atitle%3DDocking%2520screens%253A%2520right%2520for%2520the%2520right%2520reasons%253F%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D755%26epage%3D770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Taylor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewsbury, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J.</span><span> </span><span class="NLM_article-title">A review of protein–small molecule docking methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=151-166&author=R.+Taylorauthor=P.+Jewsburyauthor=J.+Essex&title=A+review+of+protein%E2%80%93small+molecule+docking+methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DJewsbury%26aufirst%3DP.%26aulast%3DEssex%26aufirst%3DJ.%26atitle%3DA%2520review%2520of%2520protein%25E2%2580%2593small%2520molecule%2520docking%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D151%26epage%3D166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">High-throughput docking for lead generation</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">382</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS1367-5931%2800%2900217-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=11470599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=375-382&issue=4&author=R.+Abagyanauthor=M.+Totrov&title=High-throughput+docking+for+lead+generation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput docking for lead generation</span></div><div class="casAuthors">Abagyan, R.; Totrov, M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-382</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Recent improvements in flexible docking technol. may lead to a bigger role for computational methods in lead discovery.  Although fast and accurate computational prediction of binding affinities for an arbitrary mol. is still beyond the limits of current methods, the docking and screening procedures can select small sets of likely lead candidates from large libraries of either com. or synthetically available compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9_FjizU9XdbVg90H21EOLACvtfcHk0lgqRdxaW5BJeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D&md5=c9bfaa493becdadb4386522865b9e7fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900217-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900217-9%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DHigh-throughput%2520docking%2520for%2520lead%2520generation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26issue%3D4%26spage%3D375%26epage%3D382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5912</span><span class="NLM_x">–</span> <span class="NLM_lpage">5931</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0lhV5wXmaV3PaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span> </span><span class="NLM_article-title">Prediction of protein–ligand interactions. Docking and scoring: successes and gaps</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5851</span><span class="NLM_x">–</span> <span class="NLM_lpage">5855</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060999m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5851-5855&author=A.+R.+Leachauthor=B.+K.+Shoichetauthor=C.+E.+Peishoff&title=Prediction+of+protein%E2%80%93ligand+interactions.+Docking+and+scoring%3A+successes+and+gaps"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm060999m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060999m%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26atitle%3DPrediction%2520of%2520protein%25E2%2580%2593ligand%2520interactions.%2520Docking%2520and%2520scoring%253A%2520successes%2520and%2520gaps%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5851%26epage%3D5855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Coupez, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. A.</span><span> </span><span class="NLM_article-title">Docking and scoring: theoretically easy, practically impossible?</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">2995</span><span class="NLM_x">–</span> <span class="NLM_lpage">3003</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.2174%2F092986706778521797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17073642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVamsrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=2995-3003&author=B.+Coupezauthor=R.+A.+Lewis&title=Docking+and+scoring%3A+theoretically+easy%2C+practically+impossible%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring - theoretically easy, practically impossible?</span></div><div class="casAuthors">Coupez, B.; Lewis, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">2995-3003</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based Drug Design (SBDD) is an essential part of the modern medicinal chem., and has led to the acceleration of many projects, and even to drugs on the market.  Programs that perform docking and scoring of ligands to receptors are powerful tools in the drug designer's armoury that enhance the process of SBDD.  They are even deployed on the desktop of many bench chemists.  It is timely to review the state of the art, to understand how good our docking programs are, and what are the issues.  In this review we would like to provide a guide around the reliable aspects of docking and scoring and the assocd. pitfalls aiming at an audience of medicinal chemists rather than modelers.  For convenience, we will divide the review into two parts: docking and scoring.  Docking concerns the prepn. of the receptor and the ligand(s), the sampling of conformational space and stereochem. (if appropriate).  Scoring concerns the evaluation of all of the ligand-receptor poses generated by docking.  The two processes are not truly independent, and this will be discussed here in detail.  The prepn. of the receptor and ligand(s) before docking requires great care.  For the receptor, issues of protonation, tautomerization and hydration are key, and we will discuss current approaches to these issues.  Even more important is the degree of sampling: can the algorithms reproduce what is obsd. exptl..  If they can, are the scoring algorithms good enough to recognize this pose as the best.  Do the scores correlate with obsd. binding affinity.  How does local knowledge of the target (for example hinge-binding to a kinase) affect the accuracy of the predictions.  We will review the key findings from several evaluation studies and present conclusions about when and how to interpret and trust the results of docking and scoring.  Finally, we will present an outline of some of the latest developments in the area of scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-ow8pkDVcLbVg90H21EOLACvtfcHk0lhV5wXmaV3PaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVamsrfF&md5=3ea8bd5273435881362163ca49f11d9e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.2174%2F092986706778521797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986706778521797%26sid%3Dliteratum%253Aachs%26aulast%3DCoupez%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26atitle%3DDocking%2520and%2520scoring%253A%2520theoretically%2520easy%252C%2520practically%2520impossible%253F%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2006%26volume%3D13%26spage%3D2995%26epage%3D3003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Kroemer, R. T.</span><span> </span><span class="NLM_article-title">Structure-based drug design: docking and scoring</span> <span class="citation_source-journal">Curr. Protein Pept. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.2174%2F138920307781369382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17696866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=312-328&issue=4&author=R.+T.+Kroemer&title=Structure-based+drug+design%3A+docking+and+scoring"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design: docking and scoring</span></div><div class="casAuthors">Kroemer, Romano T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-328</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review gives an introduction into ligand - receptor docking and illustrates the basic underlying concepts.  An overview of different approaches and algorithms is provided.  Although the application of docking and scoring has led to some remarkable successes, there are still some major challenges ahead, which are outlined here as well.  Approaches to address some of these challenges and the latest developments in the area are presented.  Some aspects of the assessment of docking program performance are discussed.  A no. of successful applications of structure-based virtual screening are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbXWKN0_d9g7Vg90H21EOLACvtfcHk0liIdBy6Z9mbPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D&md5=5bfb61ca9c4b405a98f50ddc2bb00ed7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138920307781369382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369382%26sid%3Dliteratum%253Aachs%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26atitle%3DStructure-based%2520drug%2520design%253A%2520docking%2520and%2520scoring%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26issue%3D4%26spage%3D312%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Kahraman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J.</span><span> </span><span class="NLM_article-title">Shape variation in protein binding pockets and their ligands</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">368</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2Fj.jmb.2007.01.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17337005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVeqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2007&pages=283-301&author=A.+Kahramanauthor=R.+Morrisauthor=R.+Laskowskiauthor=J.+Thornton&title=Shape+variation+in+protein+binding+pockets+and+their+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Shape Variation in Protein Binding Pockets and their Ligands</span></div><div class="casAuthors">Kahraman, Abdullah; Morris, Richard J.; Laskowski, Roman A.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">283-301</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A common assumption about the shape of protein binding pockets is that they are related to the shape of the small ligand mols. that can bind there.  But to what extent is that assumption true Here we use a recently developed shape matching method to compare the shapes of protein binding pockets to the shapes of their ligands.  We find that pockets binding the same ligand show greater variation in their shapes than can be accounted for by the conformational variability of the ligand.  This suggests that geometrical complementarity in general is not sufficient to drive mol. recognition.  Nevertheless, we show when considering only shape and size that a significant proportion of the recognition power of a binding pocket for its ligand resides in its shape.  Addnl., we observe a "buffer zone" or a region of free space between the ligand and protein, which results in binding pockets being on av. three times larger than the ligand that they bind.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoInd-ruAMm17Vg90H21EOLACvtfcHk0liIdBy6Z9mbPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVeqt78%253D&md5=3e1766868247fcc0baa99e55a21f8885</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.01.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.01.086%26sid%3Dliteratum%253Aachs%26aulast%3DKahraman%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DR.%26aulast%3DLaskowski%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%26atitle%3DShape%2520variation%2520in%2520protein%2520binding%2520pockets%2520and%2520their%2520ligands%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D368%26spage%3D283%26epage%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Patel, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciatto, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahna, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajebhosale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkus, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieren, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jessell, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, L.</span><span> </span><span class="NLM_article-title">Type II cadherin ectodomain structures: implications for classical cadherin specificity</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1268</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=1255-1268&issue=6&author=S.+D.+Patelauthor=C.+Ciattoauthor=C.+P.+Chenauthor=F.+Bahnaauthor=M.+Rajebhosaleauthor=N.+Arkusauthor=I.+Schierenauthor=T.+M.+Jessellauthor=B.+Honigauthor=S.+R.+Priceauthor=L.+Shapiro&title=Type+II+cadherin+ectodomain+structures%3A+implications+for+classical+cadherin+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DS.%2BD.%26aulast%3DCiatto%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DC.%2BP.%26aulast%3DBahna%26aufirst%3DF.%26aulast%3DRajebhosale%26aufirst%3DM.%26aulast%3DArkus%26aufirst%3DN.%26aulast%3DSchieren%26aufirst%3DI.%26aulast%3DJessell%26aufirst%3DT.%2BM.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DPrice%26aufirst%3DS.%2BR.%26aulast%3DShapiro%26aufirst%3DL.%26atitle%3DType%2520II%2520cadherin%2520ectodomain%2520structures%253A%2520implications%2520for%2520classical%2520cadherin%2520specificity%26jtitle%3DCell%26date%3D2006%26volume%3D124%26issue%3D6%26spage%3D1255%26epage%3D1268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span>Sc-PDB Home Page. <a href="http://cheminfo.u-strasbg.fr:8080/scPDB/2011/db_search/ acceuil.jsp&#xA;?uid=3807447192798425088" class="extLink">http://cheminfo.u-strasbg.fr:8080/scPDB/2011/db_search/ acceuil.jsp ?uid=3807447192798425088</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Sc-PDB+Home+Page.+http%3A%2F%2Fcheminfo.u-strasbg.fr%3A8080%2FscPDB%2F2011%2Fdb_search%2F+acceuil.jsp+%3Fuid%3D3807447192798425088."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span> <span class="citation_source-book">SYBYL X</span>; <span class="NLM_publisher-name">Tripos International</span>: <span class="NLM_publisher-loc">St. Louis, MO, U.S.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SYBYL+X%3B+Tripos+International%3A+St.+Louis%2C+MO%2C+U.S."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSYBYL%2520X%26pub%3DTripos%2520International" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+Shoichetauthor=J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0lgq8ZUYLMZdWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DIrwin%26aufirst%3DJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Karaman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&issue=1&author=M.+Karamanauthor=S.+Herrgardauthor=D.+Treiberauthor=P.+Gallantauthor=C.+Atteridgeauthor=B.+Campbellauthor=K.+Chanauthor=P.+Ciceriauthor=M.+Davisauthor=P.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+Huntauthor=D.+Lockhartauthor=Z.+Milanovauthor=M.+Morrisonauthor=G.+Pallaresauthor=H.+Patelauthor=S.+Pritchardauthor=L.+Wodickaauthor=P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lgxfu12aUL-ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DB.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%26aulast%3DEdeen%26aufirst%3DP.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%26aulast%3DLockhart%26aufirst%3DD.%26aulast%3DMilanov%26aufirst%3DZ.%26aulast%3DMorrison%26aufirst%3DM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%26aulast%3DZarrinkar%26aufirst%3DP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26issue%3D1%26spage%3D127%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Murzin, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chothia, C.</span><span> </span><span class="NLM_article-title">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">247</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS0022-2836%2805%2980134-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=7723011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1995&pages=536-540&author=A.+G.+Murzinauthor=S.+E.+Brennerauthor=T.+Hubbardauthor=C.+Chothia&title=SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span></div><div class="casAuthors">Murzin, Alexey G.; Brenner, Steven E.; Hubbard, Tim; Chothia, Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">536-40</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">To facilitate understanding of, and access to, the information available for protein structures, we have constructed the Structural Classification of Proteins (SCOP) database.  This database provides a detailed and comprehensive description of the structural and evolutionary relationships of the proteins of known structure.  It also provides for each entry links to coordinates, images of the structure, interactive viewers, sequence data and literature refs.  Two search facilitates are available.  The homol. search permits users to enter a sequence and obtain a list of any structures to which it has significant levels of sequence similarity.  The key word search finds, for a word entered by the user, matches from both the text of the SCOP database and the headers of Brookhaven Protein Databank structure files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-9_Oh4CusCbVg90H21EOLACvtfcHk0lgxfu12aUL-ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D&md5=65de3f2464aa75ba7efabbd183507913</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980134-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980134-2%26sid%3Dliteratum%253Aachs%26aulast%3DMurzin%26aufirst%3DA.%2BG.%26aulast%3DBrenner%26aufirst%3DS.%2BE.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DChothia%26aufirst%3DC.%26atitle%3DSCOP%253A%2520a%2520structural%2520classification%2520of%2520proteins%2520database%2520for%2520the%2520investigation%2520of%2520sequences%2520and%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D247%26spage%3D536%26epage%3D540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Cho, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chain, B. M.</span><span> </span><span class="NLM_article-title">Staurosporine induces rapid homotypic intercellular adhesion of U937 cells via multiple kinase activation</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fsj.bjp.0705436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12970105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotFOmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2003&pages=269-276&author=J.+Y.+Choauthor=D.+R.+Katzauthor=B.+M.+Chain&title=Staurosporine+induces+rapid+homotypic+intercellular+adhesion+of+U937+cells+via+multiple+kinase+activation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Staurosporine induces rapid homotypic intercellular adhesion of U937 cells via multiple kinase activation</span></div><div class="casAuthors">Cho, Jae Youl; Katz, David R.; Chain, Benjamin M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1 Staurosporine is a broad-specificity kinase inhibitor, which has acted as lead compd. for the development of some novel cytotoxic compds. for treatment of cancer.  This study investigates the unexpected observation that staurosporine can also induce homotypic cellular aggregation.  2 In this study, staurosporine is shown to activate rapid homotypic aggregation of U937 cells, at concns. below those required to induce cell death.  This activity is a particular feature of staurosporine, and is not shared by a no. of other kinase inhibitors.  The pro-aggregating activity of staurosporine is inhibited by deoxyglucose, cytochalasin B and colchicine.  Staurosporine-induced aggregation can be distinguished from that induced by the phorbol 12-myristate 13-acetate by faster kinetics and insensitivity to cycloheximide.  Staurosporine induces translocation of conventional and novel, but not atypical isoforms of protein kinase C (PKC).  Aggregation induced by staurosporine is inhibited by a no. of inhibitors of PKC isoforms, and by inhibitors of protein tyrosine kinases.  Staurosporine also induces rapid phosphorylation of ERK and p38, and inhibitors of both these enzymes block aggregation.  Staurosporine induces dis-regulated activation of multiple kinase signaling pathways in U937 cells, and the combined activity of several of these pathways is essential for the induction of aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokwzrAAO9guLVg90H21EOLACvtfcHk0ljPs2M2-9aEeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotFOmsbg%253D&md5=56377325fbb9c5947392ae3729223bda</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705436%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DJ.%2BY.%26aulast%3DKatz%26aufirst%3DD.%2BR.%26aulast%3DChain%26aufirst%3DB.%2BM.%26atitle%3DStaurosporine%2520induces%2520rapid%2520homotypic%2520intercellular%2520adhesion%2520of%2520U937%2520cells%2520via%2520multiple%2520kinase%2520activation%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D140%26spage%3D269%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Martarelli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompei, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzoni, G.</span><span> </span><span class="NLM_article-title">Mebendazole inhibits growth of human adrenocortical carcinoma cell lines implanted in nude mice</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">809</span><span class="NLM_x">–</span> <span class="NLM_lpage">817</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00280-007-0538-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17581752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitleru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2008&pages=809-817&issue=5&author=D.+Martarelliauthor=P.+Pompeiauthor=C.+Baldiauthor=G.+Mazzoni&title=Mebendazole+inhibits+growth+of+human+adrenocortical+carcinoma+cell+lines+implanted+in+nude+mice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Mebendazole inhibits growth of human adrenocortical carcinoma cell lines implanted in nude mice</span></div><div class="casAuthors">Martarelli, Daniele; Pompei, Pierluigi; Baldi, Caterina; Mazzoni, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">809-817</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Adrenocortical carcinoma is a rare tumor of the adrenal gland which requires new therapeutic approaches as its early diagnosis is difficult and prognosis poor despite therapies used.  Recently, mebendazole has been proved to be effective against different cancers.  The aim of our study was to evaluate whether mebendazole may result therapeutically useful in the treatment of human adrenocortical carcinoma.  We analyzed the effect of mebendazole on human adrenocortical carcinoma cells in vitro and after implantation in nude mice.  In order to clarify mechanisms of mebendazole action, metastases formation, apoptosis and angiogenesis were also investigated.  Mebendazole significantly inhibited cancer cells growth, both in vitro and in vivo, the effects being due to the induction of apoptosis.  Moreover, mebendazole inhibited invasion and migration of cancer cells in vitro, and metastases formation in vivo.  Overall, these data suggest that treatment with mebendazole, also in combination with std. therapies, could provide a new protocol for the inhibition of adrenocortical carcinoma growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwQ0oo1WtCLVg90H21EOLACvtfcHk0ljPs2M2-9aEeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitleru7c%253D&md5=723cfa4b786f346aa6a8789ee6a9b30e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2Fs00280-007-0538-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-007-0538-0%26sid%3Dliteratum%253Aachs%26aulast%3DMartarelli%26aufirst%3DD.%26aulast%3DPompei%26aufirst%3DP.%26aulast%3DBaldi%26aufirst%3DC.%26aulast%3DMazzoni%26aufirst%3DG.%26atitle%3DMebendazole%2520inhibits%2520growth%2520of%2520human%2520adrenocortical%2520carcinoma%2520cell%2520lines%2520implanted%2520in%2520nude%2520mice%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2008%26volume%3D61%26issue%3D5%26spage%3D809%26epage%3D817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Bai, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staedtke, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aprhys, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallia, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riggins, G. J.</span><span> </span><span class="NLM_article-title">Antiparasitic mebendazole shows survival benefit in 2 preclinical models of glioblastoma multiforme</span> <span class="citation_source-journal">Neuro-Oncology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">974</span><span class="NLM_x">–</span> <span class="NLM_lpage">982</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=974-982&issue=9&author=R.+Y.+Baiauthor=V.+Staedtkeauthor=C.+M.+Aprhysauthor=G.+L.+Galliaauthor=G.+J.+Riggins&title=Antiparasitic+mebendazole+shows+survival+benefit+in+2+preclinical+models+of+glioblastoma+multiforme"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DR.%2BY.%26aulast%3DStaedtke%26aufirst%3DV.%26aulast%3DAprhys%26aufirst%3DC.%2BM.%26aulast%3DGallia%26aufirst%3DG.%2BL.%26aulast%3DRiggins%26aufirst%3DG.%2BJ.%26atitle%3DAntiparasitic%2520mebendazole%2520shows%2520survival%2520benefit%2520in%25202%2520preclinical%2520models%2520of%2520glioblastoma%2520multiforme%26jtitle%3DNeuro-Oncology%26date%3D2011%26volume%3D13%26issue%3D9%26spage%3D974%26epage%3D982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Montesano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassalli, P.</span><span> </span><span class="NLM_article-title">In vitro rapid organization of endothelial cells into capillary-like networks is promoted by collagen matrices</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1648</span><span class="NLM_x">–</span> <span class="NLM_lpage">1652</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1983&pages=1648-1652&author=R.+Montesanoauthor=L.+Orciauthor=P.+Vassalli&title=In+vitro+rapid+organization+of+endothelial+cells+into+capillary-like+networks+is+promoted+by+collagen+matrices"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMontesano%26aufirst%3DR.%26aulast%3DOrci%26aufirst%3DL.%26aulast%3DVassalli%26aufirst%3DP.%26atitle%3DIn%2520vitro%2520rapid%2520organization%2520of%2520endothelial%2520cells%2520into%2520capillary-like%2520networks%2520is%2520promoted%2520by%2520collagen%2520matrices%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1983%26volume%3D97%26spage%3D1648%26epage%3D1652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Vailhe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feige, J.</span><span> </span><span class="NLM_article-title">In vitro models of vasculogenesis and angiogenesis</span> <span class="citation_source-journal">Lab. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2001&pages=439-452&author=B.+Vailheauthor=D.+Vittetauthor=J.+Feige&title=In+vitro+models+of+vasculogenesis+and+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVailhe%26aufirst%3DB.%26aulast%3DVittet%26aufirst%3DD.%26aulast%3DFeige%26aufirst%3DJ.%26atitle%3DIn%2520vitro%2520models%2520of%2520vasculogenesis%2520and%2520angiogenesis%26jtitle%3DLab.%2520Invest.%26date%3D2001%26volume%3D81%26spage%3D439%26epage%3D452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Pourgholami, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khachigian, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahmy, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, D. L.</span><span> </span><span class="NLM_article-title">Albendazole inhibits endothelial cell migration, tube formation, vasopermeability, VEGF receptor-2 expression and suppresses retinal neovascularization in ROP model of angiogenesis</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">397</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">729</span><span class="NLM_x">–</span> <span class="NLM_lpage">734</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=2010&pages=729-734&issue=4&author=M.+H.+Pourgholamiauthor=L.+M.+Khachigianauthor=R.+G.+Fahmyauthor=S.+Badarauthor=L.+Wangauthor=S.+W.+Chuauthor=D.+L.+Morris&title=Albendazole+inhibits+endothelial+cell+migration%2C+tube+formation%2C+vasopermeability%2C+VEGF+receptor-2+expression+and+suppresses+retinal+neovascularization+in+ROP+model+of+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPourgholami%26aufirst%3DM.%2BH.%26aulast%3DKhachigian%26aufirst%3DL.%2BM.%26aulast%3DFahmy%26aufirst%3DR.%2BG.%26aulast%3DBadar%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DS.%2BW.%26aulast%3DMorris%26aufirst%3DD.%2BL.%26atitle%3DAlbendazole%2520inhibits%2520endothelial%2520cell%2520migration%252C%2520tube%2520formation%252C%2520vasopermeability%252C%2520VEGF%2520receptor-2%2520expression%2520and%2520suppresses%2520retinal%2520neovascularization%2520in%2520ROP%2520model%2520of%2520angiogenesis%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D397%26issue%3D4%26spage%3D729%26epage%3D734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Sade, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuncay, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cimen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severcan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerjee, S.</span><span> </span><span class="NLM_article-title">Celecoxib reduces fluidity and decreases metastatic potential of colon cancer cell lines irrespective of COX-2 expression</span> <span class="citation_source-journal">Biosci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">44</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1042%2FBSR20100149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21401528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1yksr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=35-44&issue=1&author=A.+Sadeauthor=S.+Tuncayauthor=I.+Cimenauthor=F.+Severcanauthor=S.+Banerjee&title=Celecoxib+reduces+fluidity+and+decreases+metastatic+potential+of+colon+cancer+cell+lines+irrespective+of+COX-2+expression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib reduces fluidity and decreases metastatic potential of colon cancer cell lines irrespective of COX-2 expression</span></div><div class="casAuthors">Sade, Asli; Tuncay, Seda; Cimen, Ismail; Severcan, Feride; Banerjee, Sreeparna</div><div class="citationInfo"><span class="NLM_cas:title">Bioscience Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-44, S35/1-S35/2</span>CODEN:
                <span class="NLM_cas:coden">BRPTDT</span>;
        ISSN:<span class="NLM_cas:issn">0144-8463</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">CLX (celecoxib), a selective COX-2 (cyclo-oxygenase-2) inhibitor, has numerous pleiotropic effects on the body that may be independent of its COX-2 inhibitory activity.  The cancer chemopreventive ability of CLX, particularly in CRC (colorectal cancer), has been shown in epidemiol. studies.  Here we have, for the first time, examd. the biophys. effects of CLX on the cellular membranes of COX-2 expressing (HT29) and COX-2 non-expressing (SW620) cell lines using ATR-FTIR (attenuated total reflectance-Fourier transform IR) spectroscopy and SL-ESR (spin label-ESR) spectroscopy.  Our results show that CLX treatment decreased lipid fluidity in the cancer cell lines irresp. of COX-2 expression status.  As metastatic cells have higher membrane fluidity, we examd. the effect of CLX on the metastatic potential of these cells.  The CLX treatment efficiently decreased the proliferation, anchorage-independent growth, ability to close a scratch wound and migration and invasion of the CRC cell lines through Matrigel.  We propose that one of the ways by which CLX exerts its anti-tumorigenic effects is via alterations in cellular membrane fluidity which has a notable impact on the cells' metastatic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrezYIlBr4DMrVg90H21EOLACvtfcHk0lg5mUMpb3F42A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1yksr3I&md5=56859d5340ac8f8a5c45e893325292f9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1042%2FBSR20100149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20100149%26sid%3Dliteratum%253Aachs%26aulast%3DSade%26aufirst%3DA.%26aulast%3DTuncay%26aufirst%3DS.%26aulast%3DCimen%26aufirst%3DI.%26aulast%3DSevercan%26aufirst%3DF.%26aulast%3DBanerjee%26aufirst%3DS.%26atitle%3DCelecoxib%2520reduces%2520fluidity%2520and%2520decreases%2520metastatic%2520potential%2520of%2520colon%2520cancer%2520cell%2520lines%2520irrespective%2520of%2520COX-2%2520expression%26jtitle%3DBiosci.%2520Rep.%26date%3D2012%26volume%3D32%26issue%3D1%26spage%3D35%26epage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Zweers, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Boer, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Roon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bijlsma, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafeber, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mastbergen, S. C.</span><span> </span><span class="NLM_article-title">Celecoxib: considerations regarding its potential disease-modifying properties in osteoarthritis</span> <span class="citation_source-journal">Arthritis Res. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">239</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1186%2Far3437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21955617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlSnsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=239&issue=5&author=M.+C.+Zweersauthor=T.+N.+de+Boerauthor=J.+van+Roonauthor=J.+W.+Bijlsmaauthor=F.+P.+Lafeberauthor=S.+C.+Mastbergen&title=Celecoxib%3A+considerations+regarding+its+potential+disease-modifying+properties+in+osteoarthritis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib: considerations regarding its potential disease-modifying properties in osteoarthritis</span></div><div class="casAuthors">Zweers, Manon C.; de Boer, Tineke N.; van Roon, Joel; Bijlsma, Johannes W. J.; Lafeber, Floris P. J. G.; Mastbergen, Simon C.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis Research & Therapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">239</span>CODEN:
                <span class="NLM_cas:coden">ARTRCV</span>;
        ISSN:<span class="NLM_cas:issn">1478-6362</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Osteoarthritis is a degenerative joint disease and is characterized by progressive loss of articular cartilage, subchondral bone sclerosis, osteophyte formation, and synovial inflammation, causing substantial phys. disability, impaired quality of life, and significant health care utilization.  Traditionally, non-steroidal anti-infl ammatory drugs (NSAIDs), non-steroidal anti-inflammatory drugs (NSAIDs) including selective COX-2 inhibitors have been used.  Besides its anti-inflammatory properties, evidence is accumulating that celecoxib, one of the selective COX-2 inhibitors, has addnl. disease modifying effects.  Celecoxib was shown to affect all structures involved in OA pathogenesis: cartilage, bone, and synovium.  As well as COX-2 inhibition, evidence exists that celecoxib also modulates COX-2-independent signal transduction pathways.  These findings raise the question whether celecoxib and potentially other coxibs are more than just anti-inflammatory and analgesic drugs.  Can celecoxib be considered a disease modifying osteoarthritic drug (DMOAD).  In this review, these direct effects of celecoxib on cartilage, bone, and synoviocytes in OA treatment are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqD8ttKZiQi7Vg90H21EOLACvtfcHk0lg5mUMpb3F42A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlSnsL7J&md5=b38073dc4888b2502e2f160c58c42b26</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1186%2Far3437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Far3437%26sid%3Dliteratum%253Aachs%26aulast%3DZweers%26aufirst%3DM.%2BC.%26aulast%3Dde%2BBoer%26aufirst%3DT.%2BN.%26aulast%3Dvan%2BRoon%26aufirst%3DJ.%26aulast%3DBijlsma%26aufirst%3DJ.%2BW.%26aulast%3DLafeber%26aufirst%3DF.%2BP.%26aulast%3DMastbergen%26aufirst%3DS.%2BC.%26atitle%3DCelecoxib%253A%2520considerations%2520regarding%2520its%2520potential%2520disease-modifying%2520properties%2520in%2520osteoarthritis%26jtitle%3DArthritis%2520Res.%2520Ther.%26date%3D2011%26volume%3D13%26issue%3D5%26spage%3D239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Cervello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachvarov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cusimano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sardina, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzolina, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampiasi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannitrapani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCubrey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montalto, G.</span><span> </span><span class="NLM_article-title">COX-2-dependent and COX-2-independent mode of action of celecoxib in human liver cancer cells</span> <span class="citation_source-journal">OMICS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">392</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=383-392&issue=6&author=M.+Cervelloauthor=D.+Bachvarovauthor=A.+Cusimanoauthor=F.+Sardinaauthor=A.+Azzolinaauthor=N.+Lampiasiauthor=L.+Giannitrapaniauthor=J.+McCubreyauthor=G.+Montalto&title=COX-2-dependent+and+COX-2-independent+mode+of+action+of+celecoxib+in+human+liver+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCervello%26aufirst%3DM.%26aulast%3DBachvarov%26aufirst%3DD.%26aulast%3DCusimano%26aufirst%3DA.%26aulast%3DSardina%26aufirst%3DF.%26aulast%3DAzzolina%26aufirst%3DA.%26aulast%3DLampiasi%26aufirst%3DN.%26aulast%3DGiannitrapani%26aufirst%3DL.%26aulast%3DMcCubrey%26aufirst%3DJ.%26aulast%3DMontalto%26aufirst%3DG.%26atitle%3DCOX-2-dependent%2520and%2520COX-2-independent%2520mode%2520of%2520action%2520of%2520celecoxib%2520in%2520human%2520liver%2520cancer%2520cells%26jtitle%3DOMICS%26date%3D2011%26volume%3D15%26issue%3D6%26spage%3D383%26epage%3D392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Sareddy, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geeviman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramulu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babu, P. P.</span><span> </span><span class="NLM_article-title">The nonsteroidal anti-inflammatory drug celecoxib suppresses the growth and induces apoptosis of human glioblastoma cells via the NF-kappaB pathway</span> <span class="citation_source-journal">J. Neuro-Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs11060-011-0662-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21847707" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2011&pages=99-109&issue=1&author=G.+R.+Sareddyauthor=K.+Geevimanauthor=C.+Ramuluauthor=P.+P.+Babu&title=The+nonsteroidal+anti-inflammatory+drug+celecoxib+suppresses+the+growth+and+induces+apoptosis+of+human+glioblastoma+cells+via+the+NF-kappaB+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs11060-011-0662-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11060-011-0662-x%26sid%3Dliteratum%253Aachs%26aulast%3DSareddy%26aufirst%3DG.%2BR.%26aulast%3DGeeviman%26aufirst%3DK.%26aulast%3DRamulu%26aufirst%3DC.%26aulast%3DBabu%26aufirst%3DP.%2BP.%26atitle%3DThe%2520nonsteroidal%2520anti-inflammatory%2520drug%2520celecoxib%2520suppresses%2520the%2520growth%2520and%2520induces%2520apoptosis%2520of%2520human%2520glioblastoma%2520cells%2520via%2520the%2520NF-kappaB%2520pathway%26jtitle%3DJ.%2520Neuro-Oncol.%26date%3D2011%26volume%3D106%26issue%3D1%26spage%3D99%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Schonthal, A. H.</span><span> </span><span class="NLM_article-title">Exploiting cyclooxygenase-(in)dependent properties of COX-2 inhibitors for malignant glioma therapy</span> <span class="citation_source-journal">Anti-Cancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">450</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=450-461&issue=6&author=A.+H.+Schonthal&title=Exploiting+cyclooxygenase-%28in%29dependent+properties+of+COX-2+inhibitors+for+malignant+glioma+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchonthal%26aufirst%3DA.%2BH.%26atitle%3DExploiting%2520cyclooxygenase-%2528in%2529dependent%2520properties%2520of%2520COX-2%2520inhibitors%2520for%2520malignant%2520glioma%2520therapy%26jtitle%3DAnti-Cancer%2520Agents%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26issue%3D6%26spage%3D450%26epage%3D461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Bastos-Pereira, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugarini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira-Christoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avila, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teixeira, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pires Ado, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muscará, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cadena, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donatti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cristina da Silva de Assis., H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acco, A.</span><span> </span><span class="NLM_article-title">Celecoxib prevents tumor growth in an animal model by a COX-2 independent mechanism</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00280-009-1031-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19506872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2010&pages=267-276&issue=2&author=A.+L.+Bastos-Pereiraauthor=D.+Lugariniauthor=A.+Oliveira-Christoffauthor=T.+V.+Avilaauthor=S.+Teixeiraauthor=R.+Pires+Adoauthor=M.+N.+Muscar%C3%A1author=S.+M.+Cadenaauthor=L.+Donattiauthor=H.+Cristina+da+Silva+de+Assis.author=A.+Acco&title=Celecoxib+prevents+tumor+growth+in+an+animal+model+by+a+COX-2+independent+mechanism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib prevents tumor growth in an animal model by a COX-2 independent mechanism</span></div><div class="casAuthors">Bastos-Pereira, Amanda Leite; Lugarini, Daiana; Oliveira-Christoff, Adriana; Avila, Thiago Vinicius; Teixeira, Simone; do Rocio Andrade Pires, Amanda; Muscara, Marcelo Nicolas; Cadena, Silvia Maria Suter Correia; Donatti, Lucelia; da Silva de Assis, Helena Cristina; Acco, Alexandra</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-276</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Nonsteroidal antiinflammatory drugs (NSAIDs) have been shown to reduce cell growth in several tumors.  Among these possible antineoplastic drugs are cyclooxygenase-2 (COX-2)-selective drugs, such as celecoxib, in which antitumoral mechanisms were evaluated in rats bearing Walker-256 (W256) tumor.  W256 carcinosarcoma cells were inoculated s.c. (107 cells/rat) in rats submitted to treatment with celecoxib (25 mg/kg-1) or vehicle for 14 days.  Tumor growth, body-wt. gain, and survival data were evaluated.  The mechanisms, such as COX-2 expression and activity, oxidative stress, by means of enzymes and lipoperoxidn. levels, and apoptosis mediators were also investigated.  A redn. in tumor growth and an increased wt. gain were obsd.  Celecoxib provided a higher incidence of survival compared with the control group.  Cellular effects are probably COX-2 independent, because neither enzyme expression nor its activity, measured by tumoral PGE2, showed significant difference between groups.  It is probable that this antitumor action is dependent on an apoptotic way, which has been evaluated by the expression of the antiapoptotic protein Bcl-xL, in addn. to the cellular changes obsd. by electronic microscopy.  Celecoxib has also a possible involvement with redox homeostasis, because its administration caused significant changes in the activity of oxidative enzymes, such as catalase and superoxide dismutase.  These results confirm the antitumor effects of celecoxib in W256 cancer model, contributing to elucidating its antitumoral mechanism and corroborating scientific literature about its effect on other types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIBTD9P_P2orVg90H21EOLACvtfcHk0lh-NkLcOvQ6xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsL7I&md5=e6d10ea0b2f935b075240ad10a87c032</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2Fs00280-009-1031-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-009-1031-8%26sid%3Dliteratum%253Aachs%26aulast%3DBastos-Pereira%26aufirst%3DA.%2BL.%26aulast%3DLugarini%26aufirst%3DD.%26aulast%3DOliveira-Christoff%26aufirst%3DA.%26aulast%3DAvila%26aufirst%3DT.%2BV.%26aulast%3DTeixeira%26aufirst%3DS.%26aulast%3DPires%2BAdo%26aufirst%3DR.%26aulast%3DMuscar%25C3%25A1%26aufirst%3DM.%2BN.%26aulast%3DCadena%26aufirst%3DS.%2BM.%26aulast%3DDonatti%26aufirst%3DL.%26aulast%3DCristina%2Bda%2BSilva%2Bde%2BAssis.%26aufirst%3DH.%26aulast%3DAcco%26aufirst%3DA.%26atitle%3DCelecoxib%2520prevents%2520tumor%2520growth%2520in%2520an%2520animal%2520model%2520by%2520a%2520COX-2%2520independent%2520mechanism%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2010%26volume%3D65%26issue%3D2%26spage%3D267%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Tsutsumi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiramitsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishitani, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyoshi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitaori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span> </span><span class="NLM_article-title">Celecoxib inhibits production of MMP and NO via down-regulation of NF-kappaB and JNK in a PGE2 independent manner in human articular chondrocytes</span> <span class="citation_source-journal">Rheumatol. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">736</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00296-007-0511-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18080123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=727-736&issue=8&author=R.+Tsutsumiauthor=H.+Itoauthor=T.+Hiramitsuauthor=K.+Nishitaniauthor=M.+Akiyoshiauthor=T.+Kitaoriauthor=T.+Yasudaauthor=T.+Nakamura&title=Celecoxib+inhibits+production+of+MMP+and+NO+via+down-regulation+of+NF-kappaB+and+JNK+in+a+PGE2+independent+manner+in+human+articular+chondrocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib inhibits production of MMP and NO via down-regulation of NF-κB and JNK in a PGE2 independent manner in human articular chondrocytes</span></div><div class="casAuthors">Tsutsumi, R.; Ito, H.; Hiramitsu, T.; Nishitani, K.; Akiyoshi, M.; Kitaori, T.; Yasuda, T.; Nakamura, T.</div><div class="citationInfo"><span class="NLM_cas:title">Rheumatology International</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">727-736</span>CODEN:
                <span class="NLM_cas:coden">RHINDE</span>;
        ISSN:<span class="NLM_cas:issn">0172-8172</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">The purpose of this study was to examine the effects of celecoxib on matrix metalloproteinases (MMP-1 and MMP-3), nitric oxide (NO), and the phosphorylation of nuclear factor-κB (NF-κB) and three mitogen-activated protein kinases (MAPKs), (p38, JNK and ERK) in human articular chondrocytes from normal, osteoarthritis, and rheumatoid arthritis cartilages.  Celecoxib at 100 nM reduced the IL-1β-induced productions of MMP-1, MMP-3, iNOS, and NO, whereas indomethacin at 100 nM showed no effect.  The addnl. stimulation of prostaglandin E2 (PGE2) failed to restore those productions, while the prodn. of PGE2 were reduced by 1 and 10 μM but not 100 nM of celecoxib.  The inhibitors of NF-κB, JNK and p38, but not ERK, decreased IL-1β-enhanced MMP-1, MMP-3 and NO prodn., resp., and 100 nM celecoxib down-regulated the phosphorylation of NF-κB and JNK but has no effect on either p38 or ERK.  Celecoxib has inhibitory effects on MMP-1, MMP-3 and NO productions, suggesting the protective roles directly on articular chondrocytes.  Despite the COX-2 selectivity, celecoxib affects those productions via not PGE2 but NF-κB and JNK MAPK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR7m6RH7cRp7Vg90H21EOLACvtfcHk0lh-NkLcOvQ6xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D&md5=d5c48bf9b3fadbb5f018dc5de40a7f54</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs00296-007-0511-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00296-007-0511-6%26sid%3Dliteratum%253Aachs%26aulast%3DTsutsumi%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DH.%26aulast%3DHiramitsu%26aufirst%3DT.%26aulast%3DNishitani%26aufirst%3DK.%26aulast%3DAkiyoshi%26aufirst%3DM.%26aulast%3DKitaori%26aufirst%3DT.%26aulast%3DYasuda%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26atitle%3DCelecoxib%2520inhibits%2520production%2520of%2520MMP%2520and%2520NO%2520via%2520down-regulation%2520of%2520NF-kappaB%2520and%2520JNK%2520in%2520a%2520PGE2%2520independent%2520manner%2520in%2520human%2520articular%2520chondrocytes%26jtitle%3DRheumatol.%2520Int.%26date%3D2008%26volume%3D28%26issue%3D8%26spage%3D727%26epage%3D736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Miyamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyake, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusunoki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamura, T.</span><span> </span><span class="NLM_article-title">Selective COX-2 inhibitor celecoxib prevents experimental autoimmune encephalomyelitis through COX-2-independent pathway</span> <span class="citation_source-journal">Brain</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x"> (</span><span class="NLM_issue">Part 8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">1992</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2006&pages=1984-1992&issue=Part+8&author=K.+Miyamotoauthor=S.+Miyakeauthor=M.+Mizunoauthor=N.+Okaauthor=S.+Kusunokiauthor=T.+Yamamura&title=Selective+COX-2+inhibitor+celecoxib+prevents+experimental+autoimmune+encephalomyelitis+through+COX-2-independent+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DS.%26aulast%3DMizuno%26aufirst%3DM.%26aulast%3DOka%26aufirst%3DN.%26aulast%3DKusunoki%26aufirst%3DS.%26aulast%3DYamamura%26aufirst%3DT.%26atitle%3DSelective%2520COX-2%2520inhibitor%2520celecoxib%2520prevents%2520experimental%2520autoimmune%2520encephalomyelitis%2520through%2520COX-2-independent%2520pathway%26jtitle%3DBrain%26date%3D2006%26volume%3D129%26issue%3DPart%25208%26spage%3D1984%26epage%3D1992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Kusunoki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, S.</span><span> </span><span class="NLM_article-title">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span> <span class="citation_source-journal">Arthritis Rheum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">3159</span><span class="NLM_x">–</span> <span class="NLM_lpage">3167</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1002%2Fart.10692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12483719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=3159-3167&issue=12&author=N.+Kusunokiauthor=R.+Yamazakiauthor=S.+Kawai&title=Induction+of+apoptosis+in+rheumatoid+synovial+fibroblasts+by+celecoxib%2C+but+not+by+other+selective+cyclooxygenase+2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span></div><div class="casAuthors">Kusunoki, Natsuko; Yamazaki, Ryuta; Kawai, Shinichi</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3159-3167</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Selective cyclooxygenase 2 (COX-2) inhibitors are now being used as antiinflammatory agents that cause fewer gastrointestinal complications, compared with other antiinflammatory drugs, in patients with rheumatoid arthritis (RA).  This study was undertaken to investigate whether selective COX-2 inhibitors could induce apoptosis of RA synovial fibroblasts (RASFs).  RASFs were exposed to selective COX-2 inhibitors, i.e., celecoxib, etodolac, meloxicam, nimesulide, N-[2-(cyclohexyloxyl)-4-nitrophenyl]-methanesulfonamide, and rofecoxib, under various conditions.  Cell proliferation and cell viability were assessed by incorporation of 5-bromo-2'-deoxyuridine and by the 2-(4-iodophenyl)-3-(4-nitrophenyl)-5-(2,4-disulfophenyl)-2H-tetrazolium monosodium salt assay, resp.  Apoptosis was detected by identifying DNA fragmentation.  Activation of peroxisome proliferator-activated receptor γ (PPARγ) was measured by the luciferase reporter gene assay with a PPAR response element-driven luciferase reporter plasmid and a PPARγ expression plasmid.  Celecoxib strongly inhibited the proliferation of RASFs, whereas other selective COX-2 inhibitors had little or no effect.  In addn., celecoxib reduced the viability of RASFs by induction of apoptosis, in a concn.-dependent manner.  This action was abolished by addn. of caspase inhibitors.  Interleukin-1β had a weak enhancing effect on celecoxib-induced apoptosis in RASFs.  In contrast, other selective COX-2 inhibitors at concns. up to 100 μM did not induce apoptosis of RASFs.  Indomethacin, a nonselective COX inhibitor, activated PPARγ transcription, while celecoxib did not.  Celecoxib suppressed the proliferation of RASFs by COX-2-independent and PPARγ-independent induction of apoptosis.  Although the mechanism involved remains unclear, celecoxib may have not only antiinflammatory activity, but also a disease-modifying effect on rheumatoid synovial proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7T4nlS6hfVLVg90H21EOLACvtfcHk0lg6QOYm3RuycA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D&md5=8406153dd9e73542120554bc602dea0b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fart.10692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.10692%26sid%3Dliteratum%253Aachs%26aulast%3DKusunoki%26aufirst%3DN.%26aulast%3DYamazaki%26aufirst%3DR.%26aulast%3DKawai%26aufirst%3DS.%26atitle%3DInduction%2520of%2520apoptosis%2520in%2520rheumatoid%2520synovial%2520fibroblasts%2520by%2520celecoxib%252C%2520but%2520not%2520by%2520other%2520selective%2520cyclooxygenase%25202%2520inhibitors%26jtitle%3DArthritis%2520Rheum.%26date%3D2002%26volume%3D46%26issue%3D12%26spage%3D3159%26epage%3D3167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Tsutsumi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiramitsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishitani, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyoshi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitaori, T</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span> </span><span class="NLM_article-title">Celecoxib inhibits production of MMP and NO via down-regulation of NF-kappaB and JNK in a PGE2 independent manner in human articular chondrocytes</span> <span class="citation_source-journal">Rheumatol. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">736</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00296-007-0511-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18080123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=727-736&issue=8&author=R.+Tsutsumiauthor=H.+Itoauthor=T.+Hiramitsuauthor=K.+Nishitaniauthor=M.+Akiyoshiauthor=T+Kitaoriauthor=T.+Yasudaauthor=T.+Nakamura&title=Celecoxib+inhibits+production+of+MMP+and+NO+via+down-regulation+of+NF-kappaB+and+JNK+in+a+PGE2+independent+manner+in+human+articular+chondrocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib inhibits production of MMP and NO via down-regulation of NF-κB and JNK in a PGE2 independent manner in human articular chondrocytes</span></div><div class="casAuthors">Tsutsumi, R.; Ito, H.; Hiramitsu, T.; Nishitani, K.; Akiyoshi, M.; Kitaori, T.; Yasuda, T.; Nakamura, T.</div><div class="citationInfo"><span class="NLM_cas:title">Rheumatology International</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">727-736</span>CODEN:
                <span class="NLM_cas:coden">RHINDE</span>;
        ISSN:<span class="NLM_cas:issn">0172-8172</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">The purpose of this study was to examine the effects of celecoxib on matrix metalloproteinases (MMP-1 and MMP-3), nitric oxide (NO), and the phosphorylation of nuclear factor-κB (NF-κB) and three mitogen-activated protein kinases (MAPKs), (p38, JNK and ERK) in human articular chondrocytes from normal, osteoarthritis, and rheumatoid arthritis cartilages.  Celecoxib at 100 nM reduced the IL-1β-induced productions of MMP-1, MMP-3, iNOS, and NO, whereas indomethacin at 100 nM showed no effect.  The addnl. stimulation of prostaglandin E2 (PGE2) failed to restore those productions, while the prodn. of PGE2 were reduced by 1 and 10 μM but not 100 nM of celecoxib.  The inhibitors of NF-κB, JNK and p38, but not ERK, decreased IL-1β-enhanced MMP-1, MMP-3 and NO prodn., resp., and 100 nM celecoxib down-regulated the phosphorylation of NF-κB and JNK but has no effect on either p38 or ERK.  Celecoxib has inhibitory effects on MMP-1, MMP-3 and NO productions, suggesting the protective roles directly on articular chondrocytes.  Despite the COX-2 selectivity, celecoxib affects those productions via not PGE2 but NF-κB and JNK MAPK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR7m6RH7cRp7Vg90H21EOLACvtfcHk0lg6QOYm3RuycA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D&md5=d5c48bf9b3fadbb5f018dc5de40a7f54</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs00296-007-0511-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00296-007-0511-6%26sid%3Dliteratum%253Aachs%26aulast%3DTsutsumi%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DH.%26aulast%3DHiramitsu%26aufirst%3DT.%26aulast%3DNishitani%26aufirst%3DK.%26aulast%3DAkiyoshi%26aufirst%3DM.%26aulast%3DKitaori%26aufirst%3DT%26aulast%3DYasuda%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26atitle%3DCelecoxib%2520inhibits%2520production%2520of%2520MMP%2520and%2520NO%2520via%2520down-regulation%2520of%2520NF-kappaB%2520and%2520JNK%2520in%2520a%2520PGE2%2520independent%2520manner%2520in%2520human%2520articular%2520chondrocytes%26jtitle%3DRheumatol.%2520Int.%26date%3D2008%26volume%3D28%26issue%3D8%26spage%3D727%26epage%3D736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Miyamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyake, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusunoki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamura, T.</span><span> </span><span class="NLM_article-title">Selective COX-2 inhibitor celecoxib prevents experimental autoimmune encephalomyelitis through COX-2-independent pathway</span> <span class="citation_source-journal">Brain</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x"> (</span><span class="NLM_issue">Part 8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">1992</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2006&pages=1984-1992&issue=Part+8&author=K.+Miyamotoauthor=S.+Miyakeauthor=M.+Mizunoauthor=N.+Okaauthor=S.+Kusunokiauthor=T.+Yamamura&title=Selective+COX-2+inhibitor+celecoxib+prevents+experimental+autoimmune+encephalomyelitis+through+COX-2-independent+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DS.%26aulast%3DMizuno%26aufirst%3DM.%26aulast%3DOka%26aufirst%3DN.%26aulast%3DKusunoki%26aufirst%3DS.%26aulast%3DYamamura%26aufirst%3DT.%26atitle%3DSelective%2520COX-2%2520inhibitor%2520celecoxib%2520prevents%2520experimental%2520autoimmune%2520encephalomyelitis%2520through%2520COX-2-independent%2520pathway%26jtitle%3DBrain%26date%3D2006%26volume%3D129%26issue%3DPart%25208%26spage%3D1984%26epage%3D1992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Kusunoki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, S.</span><span> </span><span class="NLM_article-title">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span> <span class="citation_source-journal">Arthritis Rheum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">3159</span><span class="NLM_x">–</span> <span class="NLM_lpage">3167</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1002%2Fart.10692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12483719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=3159-3167&issue=12&author=N.+Kusunokiauthor=R.+Yamazakiauthor=S.+Kawai&title=Induction+of+apoptosis+in+rheumatoid+synovial+fibroblasts+by+celecoxib%2C+but+not+by+other+selective+cyclooxygenase+2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span></div><div class="casAuthors">Kusunoki, Natsuko; Yamazaki, Ryuta; Kawai, Shinichi</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3159-3167</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Selective cyclooxygenase 2 (COX-2) inhibitors are now being used as antiinflammatory agents that cause fewer gastrointestinal complications, compared with other antiinflammatory drugs, in patients with rheumatoid arthritis (RA).  This study was undertaken to investigate whether selective COX-2 inhibitors could induce apoptosis of RA synovial fibroblasts (RASFs).  RASFs were exposed to selective COX-2 inhibitors, i.e., celecoxib, etodolac, meloxicam, nimesulide, N-[2-(cyclohexyloxyl)-4-nitrophenyl]-methanesulfonamide, and rofecoxib, under various conditions.  Cell proliferation and cell viability were assessed by incorporation of 5-bromo-2'-deoxyuridine and by the 2-(4-iodophenyl)-3-(4-nitrophenyl)-5-(2,4-disulfophenyl)-2H-tetrazolium monosodium salt assay, resp.  Apoptosis was detected by identifying DNA fragmentation.  Activation of peroxisome proliferator-activated receptor γ (PPARγ) was measured by the luciferase reporter gene assay with a PPAR response element-driven luciferase reporter plasmid and a PPARγ expression plasmid.  Celecoxib strongly inhibited the proliferation of RASFs, whereas other selective COX-2 inhibitors had little or no effect.  In addn., celecoxib reduced the viability of RASFs by induction of apoptosis, in a concn.-dependent manner.  This action was abolished by addn. of caspase inhibitors.  Interleukin-1β had a weak enhancing effect on celecoxib-induced apoptosis in RASFs.  In contrast, other selective COX-2 inhibitors at concns. up to 100 μM did not induce apoptosis of RASFs.  Indomethacin, a nonselective COX inhibitor, activated PPARγ transcription, while celecoxib did not.  Celecoxib suppressed the proliferation of RASFs by COX-2-independent and PPARγ-independent induction of apoptosis.  Although the mechanism involved remains unclear, celecoxib may have not only antiinflammatory activity, but also a disease-modifying effect on rheumatoid synovial proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7T4nlS6hfVLVg90H21EOLACvtfcHk0liKTDVEvv-OZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D&md5=8406153dd9e73542120554bc602dea0b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fart.10692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.10692%26sid%3Dliteratum%253Aachs%26aulast%3DKusunoki%26aufirst%3DN.%26aulast%3DYamazaki%26aufirst%3DR.%26aulast%3DKawai%26aufirst%3DS.%26atitle%3DInduction%2520of%2520apoptosis%2520in%2520rheumatoid%2520synovial%2520fibroblasts%2520by%2520celecoxib%252C%2520but%2520not%2520by%2520other%2520selective%2520cyclooxygenase%25202%2520inhibitors%26jtitle%3DArthritis%2520Rheum.%26date%3D2002%26volume%3D46%26issue%3D12%26spage%3D3159%26epage%3D3167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 87 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guillaume  Médard</span>, <span class="hlFld-ContribAuthor ">Fiona  Pachl</span>, <span class="hlFld-ContribAuthor ">Benjamin  Ruprecht</span>, <span class="hlFld-ContribAuthor ">Susan  Klaeger</span>, <span class="hlFld-ContribAuthor ">Stephanie  Heinzlmeir</span>, <span class="hlFld-ContribAuthor ">Dominic  Helm</span>, <span class="hlFld-ContribAuthor ">Huichao  Qiao</span>, <span class="hlFld-ContribAuthor ">Xin  Ku</span>, <span class="hlFld-ContribAuthor ">Mathias  Wilhelm</span>, <span class="hlFld-ContribAuthor ">Thomas  Kuehne</span>, <span class="hlFld-ContribAuthor ">Zhixiang  Wu</span>, <span class="hlFld-ContribAuthor ">Antje  Dittmann</span>, <span class="hlFld-ContribAuthor ">Carsten  Hopf</span>, <span class="hlFld-ContribAuthor ">Karl  Kramer</span>, and <span class="hlFld-ContribAuthor ">Bernhard  Kuster</span>  . </span><span class="cited-content_cbyCitation_article-title">Optimized Chemical Proteomics Assay for Kinase Inhibitor Profiling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Proteome Research</span><span> <strong>2015,</strong> <em>14 </em>
                                    (3)
                                     , 1574-1586. <a href="https://doi.org/10.1021/pr5012608" title="DOI URL">https://doi.org/10.1021/pr5012608</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/pr5012608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fpr5012608%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Proteome%2520Research%26atitle%3DOptimized%252BChemical%252BProteomics%252BAssay%252Bfor%252BKinase%252BInhibitor%252BProfiling%26aulast%3DM%25C3%25A9dard%26aufirst%3DGuillaume%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D05122014%26date%3D20022015%26date%3D06032015%26date%3D08022015%26volume%3D14%26issue%3D3%26spage%3D1574%26epage%3D1586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anne Mai  Wassermann</span>, <span class="hlFld-ContribAuthor ">Eugen  Lounkine</span>, <span class="hlFld-ContribAuthor ">Laszlo  Urban</span>, <span class="hlFld-ContribAuthor ">Steven  Whitebread</span>, <span class="hlFld-ContribAuthor ">Shanni  Chen</span>, <span class="hlFld-ContribAuthor ">Kevin  Hughes</span>, <span class="hlFld-ContribAuthor ">Hongqiu  Guo</span>, <span class="hlFld-ContribAuthor ">Elena  Kutlina</span>, <span class="hlFld-ContribAuthor ">Alexander  Fekete</span>, <span class="hlFld-ContribAuthor ">Martin  Klumpp</span>, and <span class="hlFld-ContribAuthor ">Meir  Glick</span>  . </span><span class="cited-content_cbyCitation_article-title">A Screening Pattern Recognition Method Finds New and Divergent Targets for Drugs and Natural Products. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2014,</strong> <em>9 </em>
                                    (7)
                                     , 1622-1631. <a href="https://doi.org/10.1021/cb5001839" title="DOI URL">https://doi.org/10.1021/cb5001839</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb5001839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb5001839%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DA%252BScreening%252BPattern%252BRecognition%252BMethod%252BFinds%252BNew%252Band%252BDivergent%252BTargets%252Bfor%252BDrugs%252Band%252BNatural%252BProducts%26aulast%3DWassermann%26aufirst%3DAnne%2BMai%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D06012014%26date%3D06052014%26date%3D02062014%26date%3D18072014%26date%3D06052014%26volume%3D9%26issue%3D7%26spage%3D1622%26epage%3D1631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hailin  Chen</span>, <span class="hlFld-ContribAuthor ">Zuping  Zhang</span>, <span class="hlFld-ContribAuthor ">Jingpu  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">In silico drug repositioning based on the integration of chemical, genomic and pharmacological spaces. </span><span class="cited-content_cbyCitation_journal-name">BMC Bioinformatics</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12859-021-03988-x" title="DOI URL">https://doi.org/10.1186/s12859-021-03988-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12859-021-03988-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12859-021-03988-x%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Bioinformatics%26atitle%3DIn%252Bsilico%252Bdrug%252Brepositioning%252Bbased%252Bon%252Bthe%252Bintegration%252Bof%252Bchemical%25252C%252Bgenomic%252Band%252Bpharmacological%252Bspaces%26aulast%3DChen%26aufirst%3DHailin%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tamim  Ahsan</span>, <span class="hlFld-ContribAuthor ">Abu Ashfaqur  Sajib</span>. </span><span class="cited-content_cbyCitation_article-title">Repurposing of approved drugs with potential to interact with SARS-CoV-2 receptor. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry and Biophysics Reports</span><span> <strong>2021,</strong> <em>26 </em>, 100982. <a href="https://doi.org/10.1016/j.bbrep.2021.100982" title="DOI URL">https://doi.org/10.1016/j.bbrep.2021.100982</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrep.2021.100982&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrep.2021.100982%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%2520and%2520Biophysics%2520Reports%26atitle%3DRepurposing%252Bof%252Bapproved%252Bdrugs%252Bwith%252Bpotential%252Bto%252Binteract%252Bwith%252BSARS-CoV-2%252Breceptor%26aulast%3DAhsan%26aufirst%3DTamim%26date%3D2021%26volume%3D26%26spage%3D100982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqing  Ru</span>, <span class="hlFld-ContribAuthor ">Xiucai  Ye</span>, <span class="hlFld-ContribAuthor ">Tetsuya  Sakurai</span>, <span class="hlFld-ContribAuthor ">Quan  Zou</span>, <span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Chen  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Current status and future prospects of drug–target interaction prediction. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Functional Genomics</span><span> <strong>2021,</strong> <em>33 </em><a href="https://doi.org/10.1093/bfgp/elab031" title="DOI URL">https://doi.org/10.1093/bfgp/elab031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bfgp/elab031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbfgp%2Felab031%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Functional%2520Genomics%26atitle%3DCurrent%252Bstatus%252Band%252Bfuture%252Bprospects%252Bof%252Bdrug%2525E2%252580%252593target%252Binteraction%252Bprediction%26aulast%3DRu%26aufirst%3DXiaoqing%26date%3D2021%26date%3D2021%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shafi  Ullah</span>, <span class="hlFld-ContribAuthor ">Shafi Ullah  Khan</span>, <span class="hlFld-ContribAuthor ">Abbas  Khan</span>, <span class="hlFld-ContribAuthor ">Muhammad  Junaid</span>, <span class="hlFld-ContribAuthor ">Humaira  Rafiq</span>, <span class="hlFld-ContribAuthor ">Thet Thet  Htar</span>, <span class="hlFld-ContribAuthor ">Yaxue  Zhao</span>, <span class="hlFld-ContribAuthor ">Syed Adnan Ali  Shah</span>, <span class="hlFld-ContribAuthor ">Abdul  Wadood</span>. </span><span class="cited-content_cbyCitation_article-title">Prospect of Anterior Gradient 2 homodimer inhibition via repurposing FDA-approved drugs using structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2021,</strong> <em>106 </em><a href="https://doi.org/10.1007/s11030-021-10263-x" title="DOI URL">https://doi.org/10.1007/s11030-021-10263-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-021-10263-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-021-10263-x%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DProspect%252Bof%252BAnterior%252BGradient%252B2%252Bhomodimer%252Binhibition%252Bvia%252Brepurposing%252BFDA-approved%252Bdrugs%252Busing%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DUllah%26aufirst%3DShafi%26date%3D2021%26date%3D2021%26volume%3D106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shams  Tabrez</span>, <span class="hlFld-ContribAuthor ">Fazlur  Rahman</span>, <span class="hlFld-ContribAuthor ">Rahat  Ali</span>, <span class="hlFld-ContribAuthor ">Sajjadul Kadir  Akand</span>, <span class="hlFld-ContribAuthor ">Mohammed A.  Alaidarous</span>, <span class="hlFld-ContribAuthor ">Bader Mohammed  Alshehri</span>, <span class="hlFld-ContribAuthor ">Saeed  Banawas</span>, <span class="hlFld-ContribAuthor ">Abdul Aziz Bin  Dukhyil</span>, <span class="hlFld-ContribAuthor ">Abdur  Rub</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting sterol alpha‐14 demethylase of
              Leishmania donovani
              to fight against leishmaniasis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Biochemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.1002/jcb.29922" title="DOI URL">https://doi.org/10.1002/jcb.29922</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcb.29922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcb.29922%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Biochemistry%26atitle%3DTargeting%252Bsterol%252Balpha%2525E2%252580%25259014%252Bdemethylase%252Bof%252BLeishmania%252Bdonovani%252Bto%252Bfight%252Bagainst%252Bleishmaniasis%26aulast%3DTabrez%26aufirst%3DShams%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lance A.  Riley</span>, <span class="hlFld-ContribAuthor ">W. David  Merryman</span>. </span><span class="cited-content_cbyCitation_article-title">Cadherin-11 and cardiac fibrosis: A common target for a common pathology. </span><span class="cited-content_cbyCitation_journal-name">Cellular Signalling</span><span> <strong>2021,</strong> <em>78 </em>, 109876. <a href="https://doi.org/10.1016/j.cellsig.2020.109876" title="DOI URL">https://doi.org/10.1016/j.cellsig.2020.109876</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cellsig.2020.109876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cellsig.2020.109876%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520Signalling%26atitle%3DCadherin-11%252Band%252Bcardiac%252Bfibrosis%25253A%252BA%252Bcommon%252Btarget%252Bfor%252Ba%252Bcommon%252Bpathology%26aulast%3DRiley%26aufirst%3DLance%2BA.%26date%3D2021%26volume%3D78%26spage%3D109876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hussein  Hammoud</span>, <span class="hlFld-ContribAuthor ">Zahraa  Saker</span>, <span class="hlFld-ContribAuthor ">Hayat  Harati</span>, <span class="hlFld-ContribAuthor ">Youssef  Fares</span>, <span class="hlFld-ContribAuthor ">Hisham F.  Bahmad</span>, <span class="hlFld-ContribAuthor ">Sanaa  Nabha</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Repurposing in Medulloblastoma: Challenges and Recommendations. </span><span class="cited-content_cbyCitation_journal-name">Current Treatment Options in Oncology</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     <a href="https://doi.org/10.1007/s11864-020-00805-0" title="DOI URL">https://doi.org/10.1007/s11864-020-00805-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11864-020-00805-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11864-020-00805-0%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Treatment%2520Options%2520in%2520Oncology%26atitle%3DDrug%252BRepurposing%252Bin%252BMedulloblastoma%25253A%252BChallenges%252Band%252BRecommendations%26aulast%3DHammoud%26aufirst%3DHussein%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Imlimaong  Aier</span>, <span class="hlFld-ContribAuthor ">Pritish Kumar  Varadwaj</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Repositioning: Principles, Resources, and Application of Structure-Based Virtual Screening for the Identification of Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 313-336. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00006-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00006-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00006-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00006-0%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BRepositioning%25253A%252BPrinciples%25252C%252BResources%25252C%252Band%252BApplication%252Bof%252BStructure-Based%252BVirtual%252BScreening%252Bfor%252Bthe%252BIdentification%252Bof%252BAnticancer%252BAgents%26aulast%3DAier%26aufirst%3DImlimaong%26date%3D2021%26spage%3D313%26epage%3D336%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amutha  Ramaswamy</span>, <span class="hlFld-ContribAuthor ">Sangeetha  Balasubramanian</span>, <span class="hlFld-ContribAuthor ">Muthukumaran  Rajagopalan</span>. </span><span class="cited-content_cbyCitation_article-title">Biomolecular Talks—Part 2: Applications and Challenges of Molecular Docking Approaches. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 245-269. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00024-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00024-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00024-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00024-2%26sid%3Dliteratum%253Aachs%26atitle%3DBiomolecular%252BTalks%2525E2%252580%252594Part%252B2%25253A%252BApplications%252Band%252BChallenges%252Bof%252BMolecular%252BDocking%252BApproaches%26aulast%3DRamaswamy%26aufirst%3DAmutha%26date%3D2021%26spage%3D245%26epage%3D269%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Melissa F.  Adasme</span>, <span class="hlFld-ContribAuthor ">Daniele  Parisi</span>, <span class="hlFld-ContribAuthor ">Anastasia  Sveshnikova</span>, <span class="hlFld-ContribAuthor ">Michael  Schroeder</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based drug repositioning: Potential and limits. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2021,</strong> <em>68 </em>, 192-198. <a href="https://doi.org/10.1016/j.semcancer.2020.01.010" title="DOI URL">https://doi.org/10.1016/j.semcancer.2020.01.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2020.01.010%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DStructure-based%252Bdrug%252Brepositioning%25253A%252BPotential%252Band%252Blimits%26aulast%3DAdasme%26aufirst%3DMelissa%2BF.%26date%3D2021%26volume%3D68%26spage%3D192%26epage%3D198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruoqi  Liu</span>, <span class="hlFld-ContribAuthor ">Lai  Wei</span>, <span class="hlFld-ContribAuthor ">Ping  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">A deep learning framework for drug repurposing via emulating clinical trials on real-world patient data. </span><span class="cited-content_cbyCitation_journal-name">Nature Machine Intelligence</span><span> <strong>2021,</strong> <em>3 </em>
                                    (1)
                                     , 68-75. <a href="https://doi.org/10.1038/s42256-020-00276-w" title="DOI URL">https://doi.org/10.1038/s42256-020-00276-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42256-020-00276-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42256-020-00276-w%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Machine%2520Intelligence%26atitle%3DA%252Bdeep%252Blearning%252Bframework%252Bfor%252Bdrug%252Brepurposing%252Bvia%252Bemulating%252Bclinical%252Btrials%252Bon%252Breal-world%252Bpatient%252Bdata%26aulast%3DLiu%26aufirst%3DRuoqi%26date%3D2021%26date%3D2021%26volume%3D3%26issue%3D1%26spage%3D68%26epage%3D75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naiem T.  Issa</span>, <span class="hlFld-ContribAuthor ">Vasileios  Stathias</span>, <span class="hlFld-ContribAuthor ">Stephan  Schürer</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>. </span><span class="cited-content_cbyCitation_article-title">Machine and deep learning approaches for cancer drug repurposing. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2021,</strong> <em>68 </em>, 132-142. <a href="https://doi.org/10.1016/j.semcancer.2019.12.011" title="DOI URL">https://doi.org/10.1016/j.semcancer.2019.12.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2019.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2019.12.011%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DMachine%252Band%252Bdeep%252Blearning%252Bapproaches%252Bfor%252Bcancer%252Bdrug%252Brepurposing%26aulast%3DIssa%26aufirst%3DNaiem%2BT.%26date%3D2021%26volume%3D68%26spage%3D132%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeremy  Ariey‐Bonnet</span>, <span class="hlFld-ContribAuthor ">Kendall  Carrasco</span>, <span class="hlFld-ContribAuthor ">Marion  Le Grand</span>, <span class="hlFld-ContribAuthor ">Laurent  Hoffer</span>, <span class="hlFld-ContribAuthor ">Stéphane  Betzi</span>, <span class="hlFld-ContribAuthor ">Mikael  Feracci</span>, <span class="hlFld-ContribAuthor ">Philipp  Tsvetkov</span>, <span class="hlFld-ContribAuthor ">Francois  Devred</span>, <span class="hlFld-ContribAuthor ">Yves  Collette</span>, <span class="hlFld-ContribAuthor ">Xavier  Morelli</span>, <span class="hlFld-ContribAuthor ">Pedro  Ballester</span>, <span class="hlFld-ContribAuthor ">Eddy  Pasquier</span>. </span><span class="cited-content_cbyCitation_article-title">In silico
              molecular target prediction unveils mebendazole as a potent MAPK14 inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Molecular Oncology</span><span> <strong>2020,</strong> <em>14 </em>
                                    (12)
                                     , 3083-3099. <a href="https://doi.org/10.1002/1878-0261.12810" title="DOI URL">https://doi.org/10.1002/1878-0261.12810</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/1878-0261.12810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F1878-0261.12810%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Oncology%26atitle%3DIn%252Bsilico%252Bmolecular%252Btarget%252Bprediction%252Bunveils%252Bmebendazole%252Bas%252Ba%252Bpotent%252BMAPK14%252Binhibitor%26aulast%3DAriey%25E2%2580%2590Bonnet%26aufirst%3DJeremy%26date%3D2020%26date%3D2020%26volume%3D14%26issue%3D12%26spage%3D3083%26epage%3D3099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nalini  Schaduangrat</span>, <span class="hlFld-ContribAuthor ">Samuel  Lampa</span>, <span class="hlFld-ContribAuthor ">Saw  Simeon</span>, <span class="hlFld-ContribAuthor ">Matthew Paul  Gleeson</span>, <span class="hlFld-ContribAuthor ">Ola  Spjuth</span>, <span class="hlFld-ContribAuthor ">Chanin  Nantasenamat</span>. </span><span class="cited-content_cbyCitation_article-title">Towards reproducible computational drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-020-0408-x" title="DOI URL">https://doi.org/10.1186/s13321-020-0408-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-020-0408-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-020-0408-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DTowards%252Breproducible%252Bcomputational%252Bdrug%252Bdiscovery%26aulast%3DSchaduangrat%26aufirst%3DNalini%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kashyap K.  Dubey</span>, <span class="hlFld-ContribAuthor ">Indu</span>, <span class="hlFld-ContribAuthor ">Manisha  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Reprogramming of antibiotics to combat antimicrobial resistance. </span><span class="cited-content_cbyCitation_journal-name">Archiv der Pharmazie</span><span> <strong>2020,</strong> <em>353 </em>
                                    (11)
                                     , 2000168. <a href="https://doi.org/10.1002/ardp.202000168" title="DOI URL">https://doi.org/10.1002/ardp.202000168</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ardp.202000168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fardp.202000168%26sid%3Dliteratum%253Aachs%26jtitle%3DArchiv%2520der%2520Pharmazie%26atitle%3DReprogramming%252Bof%252Bantibiotics%252Bto%252Bcombat%252Bantimicrobial%252Bresistance%26aulast%3DDubey%26aufirst%3DKashyap%2BK.%26date%3D2020%26date%3D2020%26volume%3D353%26issue%3D11%26spage%3D2000168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dmitry  Karasev</span>, <span class="hlFld-ContribAuthor ">Boris  Sobolev</span>, <span class="hlFld-ContribAuthor ">Alexey  Lagunin</span>, <span class="hlFld-ContribAuthor ">Dmitry  Filimonov</span>, <span class="hlFld-ContribAuthor ">Vladimir  Poroikov</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Protein–ligand Interaction Based on Sequence Similarity and Ligand Structural Features. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (21)
                                     , 8152. <a href="https://doi.org/10.3390/ijms21218152" title="DOI URL">https://doi.org/10.3390/ijms21218152</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21218152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21218152%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DPrediction%252Bof%252BProtein%2525E2%252580%252593ligand%252BInteraction%252BBased%252Bon%252BSequence%252BSimilarity%252Band%252BLigand%252BStructural%252BFeatures%26aulast%3DKarasev%26aufirst%3DDmitry%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D21%26spage%3D8152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aisha  Naeem</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>, <span class="hlFld-ContribAuthor ">Henry  Walthieu</span>, <span class="hlFld-ContribAuthor ">Erika  Parasido</span>, <span class="hlFld-ContribAuthor ">Maria  Avantaggiati</span>, <span class="hlFld-ContribAuthor ">Lucas  Tricoli</span>, <span class="hlFld-ContribAuthor ">Deepak  Kumar</span>, <span class="hlFld-ContribAuthor ">Richard J.  Lee</span>, <span class="hlFld-ContribAuthor ">Adam  Feldman</span>, <span class="hlFld-ContribAuthor ">Muhammad S.  Noon</span>, <span class="hlFld-ContribAuthor ">Stephen  Byers</span>, <span class="hlFld-ContribAuthor ">Olga  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Chris  Albanese</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting new drug indications for prostate cancer: The integration of an in silico proteochemometric network pharmacology platform with patient‐derived primary prostate cells. </span><span class="cited-content_cbyCitation_journal-name">The Prostate</span><span> <strong>2020,</strong> <em>80 </em>
                                    (14)
                                     , 1233-1243. <a href="https://doi.org/10.1002/pros.24050" title="DOI URL">https://doi.org/10.1002/pros.24050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pros.24050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpros.24050%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Prostate%26atitle%3DPredicting%252Bnew%252Bdrug%252Bindications%252Bfor%252Bprostate%252Bcancer%25253A%252BThe%252Bintegration%252Bof%252Ban%252Bin%252Bsilico%252Bproteochemometric%252Bnetwork%252Bpharmacology%252Bplatform%252Bwith%252Bpatient%2525E2%252580%252590derived%252Bprimary%252Bprostate%252Bcells%26aulast%3DNaeem%26aufirst%3DAisha%26date%3D2020%26date%3D2020%26volume%3D80%26issue%3D14%26spage%3D1233%26epage%3D1243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jelena  Dinić</span>, <span class="hlFld-ContribAuthor ">Thomas  Efferth</span>, <span class="hlFld-ContribAuthor ">Alfonso T.  García-Sosa</span>, <span class="hlFld-ContribAuthor ">Jelena  Grahovac</span>, <span class="hlFld-ContribAuthor ">José M.  Padrón</span>, <span class="hlFld-ContribAuthor ">Ilza  Pajeva</span>, <span class="hlFld-ContribAuthor ">Flavio  Rizzolio</span>, <span class="hlFld-ContribAuthor ">Simona  Saponara</span>, <span class="hlFld-ContribAuthor ">Gabriella  Spengler</span>, <span class="hlFld-ContribAuthor ">Ivanka  Tsakovska</span>. </span><span class="cited-content_cbyCitation_article-title">Repurposing old drugs to fight multidrug resistant cancers. </span><span class="cited-content_cbyCitation_journal-name">Drug Resistance Updates</span><span> <strong>2020,</strong> <em>52 </em>, 100713. <a href="https://doi.org/10.1016/j.drup.2020.100713" title="DOI URL">https://doi.org/10.1016/j.drup.2020.100713</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drup.2020.100713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drup.2020.100713%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Resistance%2520Updates%26atitle%3DRepurposing%252Bold%252Bdrugs%252Bto%252Bfight%252Bmultidrug%252Bresistant%252Bcancers%26aulast%3DDini%25C4%2587%26aufirst%3DJelena%26date%3D2020%26volume%3D52%26spage%3D100713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng Yao  Low</span>, <span class="hlFld-ContribAuthor ">Isra Ahmad  Farouk</span>, <span class="hlFld-ContribAuthor ">Sunil Kumar  Lal</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Repositioning: New Approaches and Future Prospects for Life-Debilitating Diseases and the COVID-19 Pandemic Outbreak. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2020,</strong> <em>12 </em>
                                    (9)
                                     , 1058. <a href="https://doi.org/10.3390/v12091058" title="DOI URL">https://doi.org/10.3390/v12091058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v12091058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv12091058%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DDrug%252BRepositioning%25253A%252BNew%252BApproaches%252Band%252BFuture%252BProspects%252Bfor%252BLife-Debilitating%252BDiseases%252Band%252Bthe%252BCOVID-19%252BPandemic%252BOutbreak%26aulast%3DLow%26aufirst%3DZheng%2BYao%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D9%26spage%3D1058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Younhee  Ko</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Drug Repositioning: Current Progress and Challenges. </span><span class="cited-content_cbyCitation_journal-name">Applied Sciences</span><span> <strong>2020,</strong> <em>10 </em>
                                    (15)
                                     , 5076. <a href="https://doi.org/10.3390/app10155076" title="DOI URL">https://doi.org/10.3390/app10155076</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/app10155076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fapp10155076%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Sciences%26atitle%3DComputational%252BDrug%252BRepositioning%25253A%252BCurrent%252BProgress%252Band%252BChallenges%26aulast%3DKo%26aufirst%3DYounhee%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D15%26spage%3D5076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zichen  Wang</span>, <span class="hlFld-ContribAuthor ">Mu  Zhou</span>, <span class="hlFld-ContribAuthor ">Corey  Arnold</span>. </span><span class="cited-content_cbyCitation_article-title">Toward heterogeneous information fusion: bipartite graph convolutional networks for in silico drug repurposing. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2020,</strong> <em>36 </em>
                                    (Supplement_1)
                                     , i525-i533. <a href="https://doi.org/10.1093/bioinformatics/btaa437" title="DOI URL">https://doi.org/10.1093/bioinformatics/btaa437</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btaa437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtaa437%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DToward%252Bheterogeneous%252Binformation%252Bfusion%25253A%252Bbipartite%252Bgraph%252Bconvolutional%252Bnetworks%252Bfor%252Bin%252Bsilico%252Bdrug%252Brepurposing%26aulast%3DWang%26aufirst%3DZichen%26date%3D2020%26date%3D2020%26volume%3D36%26issue%3DSupplement_1%26spage%3Di525%26epage%3Di533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kiran A.  Vaidya</span>, <span class="hlFld-ContribAuthor ">Matthew P.  Donnelly</span>, <span class="hlFld-ContribAuthor ">Terence W.  Gee</span>, <span class="hlFld-ContribAuthor ">Marine-Ayan  Ibrahim Aibo</span>, <span class="hlFld-ContribAuthor ">Stephen  Byers</span>, <span class="hlFld-ContribAuthor ">Jonathan T.  Butcher</span>. </span><span class="cited-content_cbyCitation_article-title">Induction of aortic valve calcification by celecoxib and its COX-2 independent derivatives is glucocorticoid-dependent. </span><span class="cited-content_cbyCitation_journal-name">Cardiovascular Pathology</span><span> <strong>2020,</strong> <em>46 </em>, 107194. <a href="https://doi.org/10.1016/j.carpath.2019.107194" title="DOI URL">https://doi.org/10.1016/j.carpath.2019.107194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.carpath.2019.107194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.carpath.2019.107194%26sid%3Dliteratum%253Aachs%26jtitle%3DCardiovascular%2520Pathology%26atitle%3DInduction%252Bof%252Baortic%252Bvalve%252Bcalcification%252Bby%252Bcelecoxib%252Band%252Bits%252BCOX-2%252Bindependent%252Bderivatives%252Bis%252Bglucocorticoid-dependent%26aulast%3DVaidya%26aufirst%3DKiran%2BA.%26date%3D2020%26volume%3D46%26spage%3D107194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edita  Sarukhanyan</span>, <span class="hlFld-ContribAuthor ">Sergey  Shityakov</span>, <span class="hlFld-ContribAuthor ">Thomas  Dandekar</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Drug Design of Axl Tyrosine Kinase Type I Inhibitors as Promising Candidates Against Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00920" title="DOI URL">https://doi.org/10.3389/fchem.2019.00920</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00920%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DRational%252BDrug%252BDesign%252Bof%252BAxl%252BTyrosine%252BKinase%252BType%252BI%252BInhibitors%252Bas%252BPromising%252BCandidates%252BAgainst%252BCancer%26aulast%3DSarukhanyan%26aufirst%3DEdita%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pan  Pantziarka</span>, <span class="hlFld-ContribAuthor ">Lydie  Meheus</span>, <span class="hlFld-ContribAuthor ">Klara  Rombauts</span>, <span class="hlFld-ContribAuthor ">Liese  Vandeborne</span>, <span class="hlFld-ContribAuthor ">Gauthier  Bouche</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repurposing for cancer therapy—an introduction. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 1-14. <a href="https://doi.org/10.1016/B978-0-12-819668-7.00001-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-819668-7.00001-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-819668-7.00001-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-819668-7.00001-4%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252Brepurposing%252Bfor%252Bcancer%252Btherapy%2525E2%252580%252594an%252Bintroduction%26aulast%3DPantziarka%26aufirst%3DPan%26date%3D2020%26spage%3D1%26epage%3D14%26pub%3DElsevier%26atitle%3DDrug%252BRepurposing%252Bin%252BCancer%252BTherapy%26date%3D2020%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniele  Parisi</span>, <span class="hlFld-ContribAuthor ">Melissa F.  Adasme</span>, <span class="hlFld-ContribAuthor ">Anastasia  Sveshnikova</span>, <span class="hlFld-ContribAuthor ">Sarah Naomi  Bolz</span>, <span class="hlFld-ContribAuthor ">Yves  Moreau</span>, <span class="hlFld-ContribAuthor ">Michael  Schroeder</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repositioning or target repositioning: A structural perspective of drug-target-indication relationship for available repurposed drugs. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2020,</strong> <em>18 </em>, 1043-1055. <a href="https://doi.org/10.1016/j.csbj.2020.04.004" title="DOI URL">https://doi.org/10.1016/j.csbj.2020.04.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2020.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2020.04.004%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DDrug%252Brepositioning%252Bor%252Btarget%252Brepositioning%25253A%252BA%252Bstructural%252Bperspective%252Bof%252Bdrug-target-indication%252Brelationship%252Bfor%252Bavailable%252Brepurposed%252Bdrugs%26aulast%3DParisi%26aufirst%3DDaniele%26date%3D2020%26volume%3D18%26spage%3D1043%26epage%3D1055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Savíns  Puertas-Martín</span>, <span class="hlFld-ContribAuthor ">Juana  L. Redondo</span>, <span class="hlFld-ContribAuthor ">Horacio  Pérez-Sánchez</span>, <span class="hlFld-ContribAuthor ">Pilar  M. Ortigosa</span>. </span><span class="cited-content_cbyCitation_article-title">Optimizing Electrostatic Similarity for Virtual Screening: A New Methodology. </span><span class="cited-content_cbyCitation_journal-name">Informatica</span><span> <strong>2020,</strong> <em>56 </em>, 1-19. <a href="https://doi.org/10.15388/20-INFOR424" title="DOI URL">https://doi.org/10.15388/20-INFOR424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15388/20-INFOR424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15388%2F20-INFOR424%26sid%3Dliteratum%253Aachs%26jtitle%3DInformatica%26atitle%3DOptimizing%252BElectrostatic%252BSimilarity%252Bfor%252BVirtual%252BScreening%25253A%252BA%252BNew%252BMethodology%26aulast%3DPuertas-Mart%25C3%25ADn%26aufirst%3DSav%25C3%25ADns%26date%3D2020%26date%3D2020%26volume%3D56%26spage%3D1%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Justin F  Rousseau</span>, <span class="hlFld-ContribAuthor ">Ying  Ding</span>, <span class="hlFld-ContribAuthor ">Min  Song</span>, <span class="hlFld-ContribAuthor ">Wei  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Understanding Drug Repurposing From the Perspective of Biomedical Entities and Their Evolution: Bibliographic Research Using Aspirin. </span><span class="cited-content_cbyCitation_journal-name">JMIR Medical Informatics</span><span> <strong>2020,</strong> <em>8 </em>
                                    (6)
                                     , e16739. <a href="https://doi.org/10.2196/16739" title="DOI URL">https://doi.org/10.2196/16739</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2196/16739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2196%2F16739%26sid%3Dliteratum%253Aachs%26jtitle%3DJMIR%2520Medical%2520Informatics%26atitle%3DUnderstanding%252BDrug%252BRepurposing%252BFrom%252Bthe%252BPerspective%252Bof%252BBiomedical%252BEntities%252Band%252BTheir%252BEvolution%25253A%252BBibliographic%252BResearch%252BUsing%252BAspirin%26aulast%3DLi%26aufirst%3DXin%26date%3D2020%26date%3D2020%26volume%3D8%26issue%3D6%26spage%3De16739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dmitry  Karasev</span>, <span class="hlFld-ContribAuthor ">Boris  Sobolev</span>, <span class="hlFld-ContribAuthor ">Alexey  Lagunin</span>, <span class="hlFld-ContribAuthor ">Dmitry  Filimonov</span>, <span class="hlFld-ContribAuthor ">Vladimir  Poroikov</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Protein–Ligand Interaction Based on the Positional Similarity Scores Derived from Amino Acid Sequences. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (1)
                                     , 24. <a href="https://doi.org/10.3390/ijms21010024" title="DOI URL">https://doi.org/10.3390/ijms21010024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21010024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21010024%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DPrediction%252Bof%252BProtein%2525E2%252580%252593Ligand%252BInteraction%252BBased%252Bon%252Bthe%252BPositional%252BSimilarity%252BScores%252BDerived%252Bfrom%252BAmino%252BAcid%252BSequences%26aulast%3DKarasev%26aufirst%3DDmitry%26date%3D2020%26date%3D2019%26volume%3D21%26issue%3D1%26spage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leandro S.  Sangenito</span>, <span class="hlFld-ContribAuthor ">Rubem F.S.  Menna-Barreto</span>, <span class="hlFld-ContribAuthor ">Cláudia M.  d'Avila-Levy</span>, <span class="hlFld-ContribAuthor ">Marta H.  Branquinha</span>, <span class="hlFld-ContribAuthor ">André L.S.  Santos</span>. </span><span class="cited-content_cbyCitation_article-title">Repositioning of HIV Aspartyl Peptidase Inhibitors for Combating the Neglected Human Pathogen Trypanosoma cruzi. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2019,</strong> <em>26 </em>
                                    (36)
                                     , 6590-6613. <a href="https://doi.org/10.2174/0929867326666190610152934" title="DOI URL">https://doi.org/10.2174/0929867326666190610152934</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867326666190610152934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867326666190610152934%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DRepositioning%252Bof%252BHIV%252BAspartyl%252BPeptidase%252BInhibitors%252Bfor%252BCombating%252Bthe%252BNeglected%252BHuman%252BPathogen%252BTrypanosoma%252Bcruzi%26aulast%3DSangenito%26aufirst%3DLeandro%2BS.%26date%3D2019%26volume%3D26%26issue%3D36%26spage%3D6590%26epage%3D6613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xue-han  Chen</span>, <span class="hlFld-ContribAuthor ">Yu-jia  Xu</span>, <span class="hlFld-ContribAuthor ">Xiao-ge  Wang</span>, <span class="hlFld-ContribAuthor ">Peng  Lin</span>, <span class="hlFld-ContribAuthor ">Bi-yin  Cao</span>, <span class="hlFld-ContribAuthor ">Yuan-ying  Zeng</span>, <span class="hlFld-ContribAuthor ">Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Zu-bin  Zhang</span>, <span class="hlFld-ContribAuthor ">Xin-liang  Mao</span>, <span class="hlFld-ContribAuthor ">Tie  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Mebendazole elicits potent antimyeloma activity by inhibiting the USP5/c-Maf axis. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2019,</strong> <em>40 </em>
                                    (12)
                                     , 1568-1577. <a href="https://doi.org/10.1038/s41401-019-0249-1" title="DOI URL">https://doi.org/10.1038/s41401-019-0249-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41401-019-0249-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41401-019-0249-1%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DMebendazole%252Belicits%252Bpotent%252Bantimyeloma%252Bactivity%252Bby%252Binhibiting%252Bthe%252BUSP5%25252Fc-Maf%252Baxis%26aulast%3DChen%26aufirst%3DXue-han%26date%3D2019%26date%3D2019%26volume%3D40%26issue%3D12%26spage%3D1568%26epage%3D1577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ping  Xuan</span>, <span class="hlFld-ContribAuthor ">Hui  Cui</span>, <span class="hlFld-ContribAuthor ">Tonghui  Shen</span>, <span class="hlFld-ContribAuthor ">Nan  Sheng</span>, <span class="hlFld-ContribAuthor ">Tiangang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">HeteroDualNet: A Dual Convolutional Neural Network With Heterogeneous Layers for Drug-Disease Association Prediction via Chou’s Five-Step Rule. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2019,</strong> <em>10 </em><a href="https://doi.org/10.3389/fphar.2019.01301" title="DOI URL">https://doi.org/10.3389/fphar.2019.01301</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2019.01301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2019.01301%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DHeteroDualNet%25253A%252BA%252BDual%252BConvolutional%252BNeural%252BNetwork%252BWith%252BHeterogeneous%252BLayers%252Bfor%252BDrug-Disease%252BAssociation%252BPrediction%252Bvia%252BChou%2525E2%252580%252599s%252BFive-Step%252BRule%26aulast%3DXuan%26aufirst%3DPing%26date%3D2019%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Drug Repurposing: Current Trends. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2019,</strong> <em>26 </em>
                                    (28)
                                     , 5389-5409. <a href="https://doi.org/10.2174/0929867325666180530100332" title="DOI URL">https://doi.org/10.2174/0929867325666180530100332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666180530100332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666180530100332%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DComputational%252BDrug%252BRepurposing%25253A%252BCurrent%252BTrends%26aulast%3DKaraman%26aufirst%3DBerin%26date%3D2019%26volume%3D26%26issue%3D28%26spage%3D5389%26epage%3D5409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guerini</span>, <span class="hlFld-ContribAuthor ">Triggiani</span>, <span class="hlFld-ContribAuthor ">Maddalo</span>, <span class="hlFld-ContribAuthor ">Bonù</span>, <span class="hlFld-ContribAuthor ">Frassine</span>, <span class="hlFld-ContribAuthor ">Baiguini</span>, <span class="hlFld-ContribAuthor ">Alghisi</span>, <span class="hlFld-ContribAuthor ">Tomasini</span>, <span class="hlFld-ContribAuthor ">Borghetti</span>, <span class="hlFld-ContribAuthor ">Pasinetti</span>, <span class="hlFld-ContribAuthor ">Bresciani</span>, <span class="hlFld-ContribAuthor ">Magrini</span>, <span class="hlFld-ContribAuthor ">Buglione</span>. </span><span class="cited-content_cbyCitation_article-title">Mebendazole as a Candidate for Drug Repurposing in Oncology: An Extensive Review of Current Literature. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (9)
                                     , 1284. <a href="https://doi.org/10.3390/cancers11091284" title="DOI URL">https://doi.org/10.3390/cancers11091284</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11091284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11091284%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DMebendazole%252Bas%252Ba%252BCandidate%252Bfor%252BDrug%252BRepurposing%252Bin%252BOncology%25253A%252BAn%252BExtensive%252BReview%252Bof%252BCurrent%252BLiterature%26aulast%3DGuerini%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D9%26spage%3D1284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Pinzi</span>, <span class="hlFld-ContribAuthor ">Giulio  Rastelli</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking: Shifting Paradigms in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (18)
                                     , 4331. <a href="https://doi.org/10.3390/ijms20184331" title="DOI URL">https://doi.org/10.3390/ijms20184331</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20184331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20184331%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DMolecular%252BDocking%25253A%252BShifting%252BParadigms%252Bin%252BDrug%252BDiscovery%26aulast%3DPinzi%26aufirst%3DLuca%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D18%26spage%3D4331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard D.  Smith</span>, <span class="hlFld-ContribAuthor ">Jordan J.  Clark</span>, <span class="hlFld-ContribAuthor ">Aqeel  Ahmed</span>, <span class="hlFld-ContribAuthor ">Zachary J.  Orban</span>, <span class="hlFld-ContribAuthor ">James B.  Dunbar</span>, <span class="hlFld-ContribAuthor ">Heather A.  Carlson</span>. </span><span class="cited-content_cbyCitation_article-title">Updates to Binding MOAD (Mother of All Databases): Polypharmacology Tools and Their Utility in Drug Repurposing. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2019,</strong> <em>431 </em>
                                    (13)
                                     , 2423-2433. <a href="https://doi.org/10.1016/j.jmb.2019.05.024" title="DOI URL">https://doi.org/10.1016/j.jmb.2019.05.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2019.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2019.05.024%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DUpdates%252Bto%252BBinding%252BMOAD%252B%252528Mother%252Bof%252BAll%252BDatabases%252529%25253A%252BPolypharmacology%252BTools%252Band%252BTheir%252BUtility%252Bin%252BDrug%252BRepurposing%26aulast%3DSmith%26aufirst%3DRichard%2BD.%26date%3D2019%26volume%3D431%26issue%3D13%26spage%3D2423%26epage%3D2433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meghan A.  Bowler</span>, <span class="hlFld-ContribAuthor ">Michael A.  Raddatz</span>, <span class="hlFld-ContribAuthor ">Camryn L.  Johnson</span>, <span class="hlFld-ContribAuthor ">Brian R.  Lindman</span>, <span class="hlFld-ContribAuthor ">W. David  Merryman</span>. </span><span class="cited-content_cbyCitation_article-title">Celecoxib Is Associated With Dystrophic Calcification and Aortic Valve Stenosis. </span><span class="cited-content_cbyCitation_journal-name">JACC: Basic to Translational Science</span><span> <strong>2019,</strong> <em>4 </em>
                                    (2)
                                     , 135-143. <a href="https://doi.org/10.1016/j.jacbts.2018.12.003" title="DOI URL">https://doi.org/10.1016/j.jacbts.2018.12.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jacbts.2018.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jacbts.2018.12.003%26sid%3Dliteratum%253Aachs%26jtitle%3DJACC%253A%2520Basic%2520to%2520Translational%2520Science%26atitle%3DCelecoxib%252BIs%252BAssociated%252BWith%252BDystrophic%2525C2%2525A0Calcification%252Band%252BAortic%252BValve%2525C2%2525A0Stenosis%26aulast%3DBowler%26aufirst%3DMeghan%2BA.%26date%3D2019%26volume%3D4%26issue%3D2%26spage%3D135%26epage%3D143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dwight A.  Towler</span>. </span><span class="cited-content_cbyCitation_article-title">Off Target But on Track to New Strategies to Mitigate Calcific Aortic Valve Disease. </span><span class="cited-content_cbyCitation_journal-name">JACC: Basic to Translational Science</span><span> <strong>2019,</strong> <em>4 </em>
                                    (2)
                                     , 144-146. <a href="https://doi.org/10.1016/j.jacbts.2019.03.010" title="DOI URL">https://doi.org/10.1016/j.jacbts.2019.03.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jacbts.2019.03.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jacbts.2019.03.010%26sid%3Dliteratum%253Aachs%26jtitle%3DJACC%253A%2520Basic%2520to%2520Translational%2520Science%26atitle%3DOff%252BTarget%252BBut%252Bon%252BTrack%252Bto%252BNew%252BStrategies%252Bto%252BMitigate%252BCalcific%252BAortic%252BValve%252BDisease%26aulast%3DTowler%26aufirst%3DDwight%2BA.%26date%3D2019%26volume%3D4%26issue%3D2%26spage%3D144%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nalini  Schaduangrat</span>, <span class="hlFld-ContribAuthor ">Nuttapat  Anuwongcharoen</span>, <span class="hlFld-ContribAuthor ">Chuleeporn  Phanus-umporn</span>, <span class="hlFld-ContribAuthor ">Nagaya  Sriwanichpoom</span>, <span class="hlFld-ContribAuthor ">Jarl E.S.  Wikberg</span>, <span class="hlFld-ContribAuthor ">Chanin  Nantasenamat</span>. </span><span class="cited-content_cbyCitation_article-title">Proteochemometric Modeling for Drug Repositioning. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 281-302. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00010-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00010-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00010-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00010-9%26sid%3Dliteratum%253Aachs%26atitle%3DProteochemometric%252BModeling%252Bfor%252BDrug%252BRepositioning%26aulast%3DSchaduangrat%26aufirst%3DNalini%26date%3D2019%26spage%3D281%26epage%3D302%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mengzhu  Zheng</span>, <span class="hlFld-ContribAuthor ">Lixia  Chen</span>, <span class="hlFld-ContribAuthor ">Li  Hua</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Modeling of FDA-Approved Drugs for Discovery of Anticancer Agents: A Drug-Repurposing Approach. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 527-548. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00017-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00017-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00017-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00017-1%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BModeling%252Bof%252BFDA-Approved%252BDrugs%252Bfor%252BDiscovery%252Bof%252BAnticancer%252BAgents%25253A%252BA%252BDrug-Repurposing%252BApproach%26aulast%3DZheng%26aufirst%3DMengzhu%26date%3D2019%26spage%3D527%26epage%3D548%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anu R.  Melge</span>, <span class="hlFld-ContribAuthor ">K.  Manzoor</span>, <span class="hlFld-ContribAuthor ">Shantikumar V.  Nair</span>, <span class="hlFld-ContribAuthor ">C. Gopi  Mohan</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Modeling of FDA-Approved Drugs for Discovery of Anti-Cancer Agents: A Drug-Repurposing Approach. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 577-608. <a href="https://doi.org/10.1016/B978-0-12-816125-8.00019-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-816125-8.00019-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816125-8.00019-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816125-8.00019-5%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BModeling%252Bof%252BFDA-Approved%252BDrugs%252Bfor%252BDiscovery%252Bof%252BAnti-Cancer%252BAgents%25253A%252BA%252BDrug-Repurposing%252BApproach%26aulast%3DMelge%26aufirst%3DAnu%2BR.%26date%3D2019%26spage%3D577%26epage%3D608%26pub%3DElsevier%26atitle%3DIn%252BSilico%252BDrug%252BDesign%26date%3D2019%26volume%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sudeep  Pushpakom</span>, <span class="hlFld-ContribAuthor ">Francesco  Iorio</span>, <span class="hlFld-ContribAuthor ">Patrick A.  Eyers</span>, <span class="hlFld-ContribAuthor ">K. Jane  Escott</span>, <span class="hlFld-ContribAuthor ">Shirley  Hopper</span>, <span class="hlFld-ContribAuthor ">Andrew  Wells</span>, <span class="hlFld-ContribAuthor ">Andrew  Doig</span>, <span class="hlFld-ContribAuthor ">Tim  Guilliams</span>, <span class="hlFld-ContribAuthor ">Joanna  Latimer</span>, <span class="hlFld-ContribAuthor ">Christine  McNamee</span>, <span class="hlFld-ContribAuthor ">Alan  Norris</span>, <span class="hlFld-ContribAuthor ">Philippe  Sanseau</span>, <span class="hlFld-ContribAuthor ">David  Cavalla</span>, <span class="hlFld-ContribAuthor ">Munir  Pirmohamed</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repurposing: progress, challenges and recommendations. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2019,</strong> <em>18 </em>
                                    (1)
                                     , 41-58. <a href="https://doi.org/10.1038/nrd.2018.168" title="DOI URL">https://doi.org/10.1038/nrd.2018.168</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2018.168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2018.168%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DDrug%252Brepurposing%25253A%252Bprogress%25252C%252Bchallenges%252Band%252Brecommendations%26aulast%3DPushpakom%26aufirst%3DSudeep%26date%3D2019%26date%3D2018%26volume%3D18%26issue%3D1%26spage%3D41%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">So Jung  Sung</span>, <span class="hlFld-ContribAuthor ">Hyun-Kyung  Kim</span>, <span class="hlFld-ContribAuthor ">Yong-Kil  Hong</span>, <span class="hlFld-ContribAuthor ">Young Ae  Joe</span>. </span><span class="cited-content_cbyCitation_article-title">Autophagy Is a Potential Target for Enhancing the Anti-Angiogenic Effect of Mebendazole in Endothelial Cells. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules & Therapeutics</span><span> <strong>2019,</strong> <em>27 </em>
                                    (1)
                                     , 117-125. <a href="https://doi.org/10.4062/biomolther.2018.222" title="DOI URL">https://doi.org/10.4062/biomolther.2018.222</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4062/biomolther.2018.222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4062%2Fbiomolther.2018.222%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%2520%2526%2520Therapeutics%26atitle%3DAutophagy%252BIs%252Ba%252BPotential%252BTarget%252Bfor%252BEnhancing%252Bthe%252BAnti-Angiogenic%252BEffect%252Bof%252BMebendazole%252Bin%252BEndothelial%252BCells%26aulast%3DSung%26aufirst%3DSo%2BJung%26date%3D2019%26volume%3D27%26issue%3D1%26spage%3D117%26epage%3D125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nathalie  Lagarde</span>, <span class="hlFld-ContribAuthor ">Julien  Rey</span>, <span class="hlFld-ContribAuthor ">Aram  Gyulkhandanyan</span>, <span class="hlFld-ContribAuthor ">Pierre  Tufféry</span>, <span class="hlFld-ContribAuthor ">Maria A.  Miteva</span>, <span class="hlFld-ContribAuthor ">Bruno O.  Villoutreix</span>. </span><span class="cited-content_cbyCitation_article-title">Online structure-based screening of purchasable approved drugs and natural compounds: retrospective examples of drug repositioning on cancer targets. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2018,</strong> <em>9 </em>
                                    (64)
                                     , 32346-32361. <a href="https://doi.org/10.18632/oncotarget.25966" title="DOI URL">https://doi.org/10.18632/oncotarget.25966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.25966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.25966%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DOnline%252Bstructure-based%252Bscreening%252Bof%252Bpurchasable%252Bapproved%252Bdrugs%252Band%252Bnatural%252Bcompounds%25253A%252Bretrospective%252Bexamples%252Bof%252Bdrug%252Brepositioning%252Bon%252Bcancer%252Btargets%26aulast%3DLagarde%26aufirst%3DNathalie%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D64%26spage%3D32346%26epage%3D32361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catharina  Lippmann</span>, <span class="hlFld-ContribAuthor ">Dario  Kringel</span>, <span class="hlFld-ContribAuthor ">Alfred  Ultsch</span>, <span class="hlFld-ContribAuthor ">Jörn  Lötsch</span>. </span><span class="cited-content_cbyCitation_article-title">Computational functional genomics-based approaches in analgesic drug discovery and repurposing. </span><span class="cited-content_cbyCitation_journal-name">Pharmacogenomics</span><span> <strong>2018,</strong> <em>19 </em>
                                    (9)
                                     , 783-797. <a href="https://doi.org/10.2217/pgs-2018-0036" title="DOI URL">https://doi.org/10.2217/pgs-2018-0036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/pgs-2018-0036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fpgs-2018-0036%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacogenomics%26atitle%3DComputational%252Bfunctional%252Bgenomics-based%252Bapproaches%252Bin%252Banalgesic%252Bdrug%252Bdiscovery%252Band%252Brepurposing%26aulast%3DLippmann%26aufirst%3DCatharina%26date%3D2018%26volume%3D19%26issue%3D9%26spage%3D783%26epage%3D797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giuseppe  Aceto</span>, <span class="hlFld-ContribAuthor ">Valerio  Persico</span>, <span class="hlFld-ContribAuthor ">Antonio  Pescapé</span>. </span><span class="cited-content_cbyCitation_article-title">The role of Information and Communication Technologies in healthcare: taxonomies, perspectives, and challenges. </span><span class="cited-content_cbyCitation_journal-name">Journal of Network and Computer Applications</span><span> <strong>2018,</strong> <em>107 </em>, 125-154. <a href="https://doi.org/10.1016/j.jnca.2018.02.008" title="DOI URL">https://doi.org/10.1016/j.jnca.2018.02.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jnca.2018.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jnca.2018.02.008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Network%2520and%2520Computer%2520Applications%26atitle%3DThe%252Brole%252Bof%252BInformation%252Band%252BCommunication%252BTechnologies%252Bin%252Bhealthcare%25253A%252Btaxonomies%25252C%252Bperspectives%25252C%252Band%252Bchallenges%26aulast%3DAceto%26aufirst%3DGiuseppe%26date%3D2018%26volume%3D107%26spage%3D125%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ren  Kong</span>, <span class="hlFld-ContribAuthor ">Stephen T.  Wong</span>. </span><span class="cited-content_cbyCitation_article-title">Repositioning of Drug - New Indications for Marketed Drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 55-78. <a href="https://doi.org/10.1002/9783527674381.ch3" title="DOI URL">https://doi.org/10.1002/9783527674381.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527674381.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527674381.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DRepositioning%252Bof%252BDrug%252B-%252BNew%252BIndications%252Bfor%252BMarketed%252BDrugs%26aulast%3DKong%26aufirst%3DRen%26date%3D2017%26date%3D2017%26spage%3D55%26epage%3D78%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DDrug%252BSelectivity%26aulast%3DHandler%26aufirst%3DNorbert%26date%3D2017%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naiem T.  Issa</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Byers</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery Technologies for Drug Repurposing. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 79-100. <a href="https://doi.org/10.1002/9783527674381.ch4" title="DOI URL">https://doi.org/10.1002/9783527674381.ch4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527674381.ch4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527674381.ch4%26sid%3Dliteratum%253Aachs%26atitle%3DDiscovery%252BTechnologies%252Bfor%252BDrug%252BRepurposing%26aulast%3DIssa%26aufirst%3DNaiem%2BT.%26date%3D2017%26date%3D2017%26spage%3D79%26epage%3D100%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DDrug%252BSelectivity%26aulast%3DHandler%26aufirst%3DNorbert%26date%3D2017%26volume%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia  Zhang</span>, <span class="hlFld-ContribAuthor ">Candong  Li</span>, <span class="hlFld-ContribAuthor ">Yaojin  Lin</span>, <span class="hlFld-ContribAuthor ">Youwei  Shao</span>, <span class="hlFld-ContribAuthor ">Shaozi  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Computational drug repositioning using collaborative filtering via multi-source fusion. </span><span class="cited-content_cbyCitation_journal-name">Expert Systems with Applications</span><span> <strong>2017,</strong> <em>84 </em>, 281-289. <a href="https://doi.org/10.1016/j.eswa.2017.05.004" title="DOI URL">https://doi.org/10.1016/j.eswa.2017.05.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.eswa.2017.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.eswa.2017.05.004%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Systems%2520with%2520Applications%26atitle%3DComputational%252Bdrug%252Brepositioning%252Busing%252Bcollaborative%252Bfiltering%252Bvia%252Bmulti-source%252Bfusion%26aulast%3DZhang%26aufirst%3DJia%26date%3D2017%26volume%3D84%26spage%3D281%26epage%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Diego  Carvalho</span>, <span class="hlFld-ContribAuthor ">Margot  Paulino</span>, <span class="hlFld-ContribAuthor ">Fabio  Polticelli</span>, <span class="hlFld-ContribAuthor ">Florencia  Arredondo</span>, <span class="hlFld-ContribAuthor ">Robert J.  Williams</span>, <span class="hlFld-ContribAuthor ">Juan A.  Abin-Carriquiry</span>. </span><span class="cited-content_cbyCitation_article-title">Structural evidence of quercetin multi-target bioactivity: A reverse virtual screening strategy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmaceutical Sciences</span><span> <strong>2017,</strong> <em>106 </em>, 393-403. <a href="https://doi.org/10.1016/j.ejps.2017.06.028" title="DOI URL">https://doi.org/10.1016/j.ejps.2017.06.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejps.2017.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejps.2017.06.028%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DStructural%252Bevidence%252Bof%252Bquercetin%252Bmulti-target%252Bbioactivity%25253A%252BA%252Breverse%252Bvirtual%252Bscreening%252Bstrategy%26aulast%3DCarvalho%26aufirst%3DDiego%26date%3D2017%26volume%3D106%26spage%3D393%26epage%3D403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liang  Yu</span>, <span class="hlFld-ContribAuthor ">Ruidan  Su</span>, <span class="hlFld-ContribAuthor ">Bingbo  Wang</span>, <span class="hlFld-ContribAuthor ">Long  Zhang</span>, <span class="hlFld-ContribAuthor ">Yapeng  Zou</span>, <span class="hlFld-ContribAuthor ">Jing  Zhang</span>, <span class="hlFld-ContribAuthor ">Lin  Gao</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Novel Drugs for Hepatocellular Carcinoma Based on Multi-Source Random Walk. </span><span class="cited-content_cbyCitation_journal-name">IEEE/ACM Transactions on Computational Biology and Bioinformatics</span><span> <strong>2017,</strong> <em>14 </em>
                                    (4)
                                     , 966-977. <a href="https://doi.org/10.1109/TCBB.2016.2550453" title="DOI URL">https://doi.org/10.1109/TCBB.2016.2550453</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1109/TCBB.2016.2550453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1109%2FTCBB.2016.2550453%26sid%3Dliteratum%253Aachs%26jtitle%3DIEEE%252FACM%2520Transactions%2520on%2520Computational%2520Biology%2520and%2520Bioinformatics%26atitle%3DPrediction%252Bof%252BNovel%252BDrugs%252Bfor%252BHepatocellular%252BCarcinoma%252BBased%252Bon%252BMulti-Source%252BRandom%252BWalk%26aulast%3DYu%26aufirst%3DLiang%26date%3D2017%26volume%3D14%26issue%3D4%26spage%3D966%26epage%3D977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  March-Vila</span>, <span class="hlFld-ContribAuthor ">Luca  Pinzi</span>, <span class="hlFld-ContribAuthor ">Noé  Sturm</span>, <span class="hlFld-ContribAuthor ">Annachiara  Tinivella</span>, <span class="hlFld-ContribAuthor ">Ola  Engkvist</span>, <span class="hlFld-ContribAuthor ">Hongming  Chen</span>, <span class="hlFld-ContribAuthor ">Giulio  Rastelli</span>. </span><span class="cited-content_cbyCitation_article-title">On the Integration of In Silico Drug Design Methods for Drug Repurposing. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2017,</strong> <em>8 </em><a href="https://doi.org/10.3389/fphar.2017.00298" title="DOI URL">https://doi.org/10.3389/fphar.2017.00298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2017.00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2017.00298%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DOn%252Bthe%252BIntegration%252Bof%252BIn%252BSilico%252BDrug%252BDesign%252BMethods%252Bfor%252BDrug%252BRepurposing%26aulast%3DMarch-Vila%26aufirst%3DEric%26date%3D2017%26date%3D2017%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veda  Prachayasittikul</span>, <span class="hlFld-ContribAuthor ">Philip  Prathipati</span>, <span class="hlFld-ContribAuthor ">Reny  Pratiwi</span>, <span class="hlFld-ContribAuthor ">Chuleeporn  Phanus-umporn</span>, <span class="hlFld-ContribAuthor ">Aijaz Ahmad  Malik</span>, <span class="hlFld-ContribAuthor ">Nalini  Schaduangrat</span>, <span class="hlFld-ContribAuthor ">Kanokwan  Seenprachawong</span>, <span class="hlFld-ContribAuthor ">Prapimpun  Wongchitrat</span>, <span class="hlFld-ContribAuthor ">Aungkura  Supokawej</span>, <span class="hlFld-ContribAuthor ">Virapong  Prachayasittikul</span>, <span class="hlFld-ContribAuthor ">Jarl E. S.  Wikberg</span>, <span class="hlFld-ContribAuthor ">Chanin  Nantasenamat</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the epigenetic drug discovery landscape. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2017,</strong> <em>12 </em>
                                    (4)
                                     , 345-362. <a href="https://doi.org/10.1080/17460441.2017.1295954" title="DOI URL">https://doi.org/10.1080/17460441.2017.1295954</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2017.1295954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2017.1295954%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DExploring%252Bthe%252Bepigenetic%252Bdrug%252Bdiscovery%252Blandscape%26aulast%3DPrachayasittikul%26aufirst%3DVeda%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D4%26spage%3D345%26epage%3D362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James E.  Barrett</span>, <span class="hlFld-ContribAuthor ">Felix J.  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">The purpose of repurposing. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (8)
                                     , 12534-12535. <a href="https://doi.org/10.18632/oncotarget.14806" title="DOI URL">https://doi.org/10.18632/oncotarget.14806</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.14806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.14806%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DThe%252Bpurpose%252Bof%252Brepurposing%26aulast%3DBarrett%26aufirst%3DJames%2BE.%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D8%26spage%3D12534%26epage%3D12535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cynthia M.  Simbulan-Rosenthal</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>, <span class="hlFld-ContribAuthor ">Anirudh  Gaur</span>, <span class="hlFld-ContribAuthor ">You-Shin  Chen</span>, <span class="hlFld-ContribAuthor ">Hong-Bin  Fang</span>, <span class="hlFld-ContribAuthor ">Maryam  Abdussamad</span>, <span class="hlFld-ContribAuthor ">Hengbo  Zhou</span>, <span class="hlFld-ContribAuthor ">John  Zapas</span>, <span class="hlFld-ContribAuthor ">Valerie  Calvert</span>, <span class="hlFld-ContribAuthor ">Emanuel F.  Petricoin</span>, <span class="hlFld-ContribAuthor ">Michael B.  Atkins</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Byers</span>, <span class="hlFld-ContribAuthor ">Dean S.  Rosenthal</span>. </span><span class="cited-content_cbyCitation_article-title">The repurposed anthelmintic mebendazole in combination with trametinib suppresses refractory NRASQ61K melanoma. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (8)
                                     , 12576-12595. <a href="https://doi.org/10.18632/oncotarget.14990" title="DOI URL">https://doi.org/10.18632/oncotarget.14990</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.14990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.14990%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DThe%252Brepurposed%252Banthelmintic%252Bmebendazole%252Bin%252Bcombination%252Bwith%252Btrametinib%252Bsuppresses%252Brefractory%252BNRASQ61K%252Bmelanoma%26aulast%3DSimbulan-Rosenthal%26aufirst%3DCynthia%2BM.%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D8%26spage%3D12576%26epage%3D12595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xujun  Liang</span>, <span class="hlFld-ContribAuthor ">Pengfei  Zhang</span>, <span class="hlFld-ContribAuthor ">Lu  Yan</span>, <span class="hlFld-ContribAuthor ">Ying  Fu</span>, <span class="hlFld-ContribAuthor ">Fang  Peng</span>, <span class="hlFld-ContribAuthor ">Lingzhi  Qu</span>, <span class="hlFld-ContribAuthor ">Meiying  Shao</span>, <span class="hlFld-ContribAuthor ">Yongheng  Chen</span>, <span class="hlFld-ContribAuthor ">Zhuchu  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">LRSSL: predict and interpret drug–disease associations based on data integration using sparse subspace learning. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2017,</strong> <em>5 </em>, btw770. <a href="https://doi.org/10.1093/bioinformatics/btw770" title="DOI URL">https://doi.org/10.1093/bioinformatics/btw770</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btw770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtw770%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DLRSSL%25253A%252Bpredict%252Band%252Binterpret%252Bdrug%2525E2%252580%252593disease%252Bassociations%252Bbased%252Bon%252Bdata%252Bintegration%252Busing%252Bsparse%252Bsubspace%252Blearning%26aulast%3DLiang%26aufirst%3DXujun%26date%3D2017%26date%3D2017%26volume%3D5%26spage%3Dbtw770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tianyi  Qiu</span>, <span class="hlFld-ContribAuthor ">Jingxuan  Qiu</span>, <span class="hlFld-ContribAuthor ">Jun  Feng</span>, <span class="hlFld-ContribAuthor ">Dingfeng  Wu</span>, <span class="hlFld-ContribAuthor ">Yiyan  Yang</span>, <span class="hlFld-ContribAuthor ">Kailin  Tang</span>, <span class="hlFld-ContribAuthor ">Zhiwei  Cao</span>, <span class="hlFld-ContribAuthor ">Ruixin  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">The recent progress in proteochemometric modelling: focusing on target descriptors, cross-term descriptors and application scope. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2017,</strong> <em>18 </em>
                                    (1)
                                     , 125-136. <a href="https://doi.org/10.1093/bib/bbw004" title="DOI URL">https://doi.org/10.1093/bib/bbw004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbw004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbw004%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DThe%252Brecent%252Bprogress%252Bin%252Bproteochemometric%252Bmodelling%25253A%252Bfocusing%252Bon%252Btarget%252Bdescriptors%25252C%252Bcross-term%252Bdescriptors%252Band%252Bapplication%252Bscope%26aulast%3DQiu%26aufirst%3DTianyi%26date%3D2017%26date%3D2016%26volume%3D18%26issue%3D1%26spage%3D125%26epage%3D136" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naiem T.  Issa</span>, <span class="hlFld-ContribAuthor ">Jordan  Kruger</span>, <span class="hlFld-ContribAuthor ">Henri  Wathieu</span>, <span class="hlFld-ContribAuthor ">Rajarajan  Raja</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Byers</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>. </span><span class="cited-content_cbyCitation_article-title">DrugGenEx-Net: a novel computational platform for systems pharmacology and gene expression-based drug repurposing. </span><span class="cited-content_cbyCitation_journal-name">BMC Bioinformatics</span><span> <strong>2016,</strong> <em>17 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12859-016-1065-y" title="DOI URL">https://doi.org/10.1186/s12859-016-1065-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12859-016-1065-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12859-016-1065-y%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Bioinformatics%26atitle%3DDrugGenEx-Net%25253A%252Ba%252Bnovel%252Bcomputational%252Bplatform%252Bfor%252Bsystems%252Bpharmacology%252Band%252Bgene%252Bexpression-based%252Bdrug%252Brepurposing%26aulast%3DIssa%26aufirst%3DNaiem%2BT.%26date%3D2016%26date%3D2016%26volume%3D17%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhonghua  Wang</span>, <span class="hlFld-ContribAuthor ">Lu  Liang</span>, <span class="hlFld-ContribAuthor ">Zheng  Yin</span>, <span class="hlFld-ContribAuthor ">Jianping  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Improving chemical similarity ensemble approach in target prediction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-016-0130-x" title="DOI URL">https://doi.org/10.1186/s13321-016-0130-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-016-0130-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-016-0130-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DImproving%252Bchemical%252Bsimilarity%252Bensemble%252Bapproach%252Bin%252Btarget%252Bprediction%26aulast%3DWang%26aufirst%3DZhonghua%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamil  Pohlodek</span>, <span class="hlFld-ContribAuthor ">Yen Y.  Tan</span>, <span class="hlFld-ContribAuthor ">Christian F.  Singer</span>, <span class="hlFld-ContribAuthor ">Daphne  Gschwantler-Kaulich</span>. </span><span class="cited-content_cbyCitation_article-title">Cadherin-11 expression is upregulated in invasive human breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Oncology Letters</span><span> <strong>2016,</strong> <em>12 </em>
                                    (6)
                                     , 4393-4398. <a href="https://doi.org/10.3892/ol.2016.5236" title="DOI URL">https://doi.org/10.3892/ol.2016.5236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ol.2016.5236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fol.2016.5236%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Letters%26atitle%3DCadherin-11%252Bexpression%252Bis%252Bupregulated%252Bin%252Binvasive%252Bhuman%252Bbreast%252Bcancer%26aulast%3DPohlodek%26aufirst%3DKamil%26date%3D2016%26date%3D2016%26volume%3D12%26issue%3D6%26spage%3D4393%26epage%3D4398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Cao</span>, <span class="hlFld-ContribAuthor ">Yanli  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Molecular Targets for Small-Molecule Drugs with a Ligand-Based Interaction Fingerprint Approach. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (12)
                                     , 1352-1361. <a href="https://doi.org/10.1002/cmdc.201500228" title="DOI URL">https://doi.org/10.1002/cmdc.201500228</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500228%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DPredicting%252BMolecular%252BTargets%252Bfor%252BSmall-Molecule%252BDrugs%252Bwith%252Ba%252BLigand-Based%252BInteraction%252BFingerprint%252BApproach%26aulast%3DCao%26aufirst%3DRan%26date%3D2016%26date%3D2015%26volume%3D11%26issue%3D12%26spage%3D1352%26epage%3D1361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giridhar  Mudduluru</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Walther</span>, <span class="hlFld-ContribAuthor ">Dennis  Kobelt</span>, <span class="hlFld-ContribAuthor ">Mathias  Dahlmann</span>, <span class="hlFld-ContribAuthor ">Christoph  Treese</span>, <span class="hlFld-ContribAuthor ">Yehuda G.  Assaraf</span>, <span class="hlFld-ContribAuthor ">Ulrike  Stein</span>. </span><span class="cited-content_cbyCitation_article-title">Repositioning of drugs for intervention in tumor progression and metastasis: Old drugs for new targets. </span><span class="cited-content_cbyCitation_journal-name">Drug Resistance Updates</span><span> <strong>2016,</strong> <em>26 </em>, 10-27. <a href="https://doi.org/10.1016/j.drup.2016.03.002" title="DOI URL">https://doi.org/10.1016/j.drup.2016.03.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drup.2016.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drup.2016.03.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Resistance%2520Updates%26atitle%3DRepositioning%252Bof%252Bdrugs%252Bfor%252Bintervention%252Bin%252Btumor%252Bprogression%252Band%252Bmetastasis%25253A%252BOld%252Bdrugs%252Bfor%252Bnew%252Btargets%26aulast%3DMudduluru%26aufirst%3DGiridhar%26date%3D2016%26volume%3D26%26spage%3D10%26epage%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Krzysztof  Palczewski</span>. </span><span class="cited-content_cbyCitation_article-title">Systems Pharmacology Links GPCRs with Retinal Degenerative Disorders. </span><span class="cited-content_cbyCitation_journal-name">Annual Review of Pharmacology and Toxicology</span><span> <strong>2016,</strong> <em>56 </em>
                                    (1)
                                     , 273-298. <a href="https://doi.org/10.1146/annurev-pharmtox-010715-103033" title="DOI URL">https://doi.org/10.1146/annurev-pharmtox-010715-103033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1146/annurev-pharmtox-010715-103033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1146%2Fannurev-pharmtox-010715-103033%26sid%3Dliteratum%253Aachs%26jtitle%3DAnnual%2520Review%2520of%2520Pharmacology%2520and%2520Toxicology%26atitle%3DSystems%252BPharmacology%252BLinks%252BGPCRs%252Bwith%252BRetinal%252BDegenerative%252BDisorders%26aulast%3DChen%26aufirst%3DYu%26date%3D2016%26volume%3D56%26issue%3D1%26spage%3D273%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Dong</span>, <span class="hlFld-ContribAuthor ">Dong-Sheng  Cao</span>, <span class="hlFld-ContribAuthor ">Hong-Yu  Miao</span>, <span class="hlFld-ContribAuthor ">Shao  Liu</span>, <span class="hlFld-ContribAuthor ">Bai-Chuan  Deng</span>, <span class="hlFld-ContribAuthor ">Yong-Huan  Yun</span>, <span class="hlFld-ContribAuthor ">Ning-Ning  Wang</span>, <span class="hlFld-ContribAuthor ">Ai-Ping  Lu</span>, <span class="hlFld-ContribAuthor ">Wen-Bin  Zeng</span>, <span class="hlFld-ContribAuthor ">Alex F.  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">ChemDes: an integrated web-based platform for molecular descriptor and fingerprint computation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-015-0109-z" title="DOI URL">https://doi.org/10.1186/s13321-015-0109-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-015-0109-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-015-0109-z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DChemDes%25253A%252Ban%252Bintegrated%252Bweb-based%252Bplatform%252Bfor%252Bmolecular%252Bdescriptor%252Band%252Bfingerprint%252Bcomputation%26aulast%3DDong%26aufirst%3DJie%26date%3D2015%26date%3D2015%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei-Ling  Chang</span>, <span class="hlFld-ContribAuthor ">Lih-Ching  Hsu</span>, <span class="hlFld-ContribAuthor ">Wohn-Jenn  Leu</span>, <span class="hlFld-ContribAuthor ">Ching-Shih  Chen</span>, <span class="hlFld-ContribAuthor ">Jih-Hwa  Guh</span>. </span><span class="cited-content_cbyCitation_article-title">Repurposing of nitroxoline as a potential anticancer agent against human prostate cancer - a crucial role on AMPK/mTOR signaling pathway and the interplay with Chk2 activation. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2015,</strong> <em>6 </em>
                                    (37)
                                     , 39806-39820. <a href="https://doi.org/10.18632/oncotarget.5655" title="DOI URL">https://doi.org/10.18632/oncotarget.5655</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.5655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.5655%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DRepurposing%252Bof%252Bnitroxoline%252Bas%252Ba%252Bpotential%252Banticancer%252Bagent%252Bagainst%252Bhuman%252Bprostate%252Bcancer%252B-%252Ba%252Bcrucial%252Brole%252Bon%252BAMPK%25252FmTOR%252Bsignaling%252Bpathway%252Band%252Bthe%252Binterplay%252Bwith%252BChk2%252Bactivation%26aulast%3DChang%26aufirst%3DWei-Ling%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D37%26spage%3D39806%26epage%3D39820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ren-Yuan  Bai</span>, <span class="hlFld-ContribAuthor ">Verena  Staedtke</span>, <span class="hlFld-ContribAuthor ">Teresia  Wanjiku</span>, <span class="hlFld-ContribAuthor ">Michelle A.  Rudek</span>, <span class="hlFld-ContribAuthor ">Avadhut  Joshi</span>, <span class="hlFld-ContribAuthor ">Gary L.  Gallia</span>, <span class="hlFld-ContribAuthor ">Gregory J.  Riggins</span>. </span><span class="cited-content_cbyCitation_article-title">Brain Penetration and Efficacy of Different Mebendazole Polymorphs in a Mouse Brain Tumor Model. </span><span class="cited-content_cbyCitation_journal-name">Clinical Cancer Research</span><span> <strong>2015,</strong> <em>21 </em>
                                    (15)
                                     , 3462-3470. <a href="https://doi.org/10.1158/1078-0432.CCR-14-2681" title="DOI URL">https://doi.org/10.1158/1078-0432.CCR-14-2681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1078-0432.CCR-14-2681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1078-0432.CCR-14-2681%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Cancer%2520Research%26atitle%3DBrain%252BPenetration%252Band%252BEfficacy%252Bof%252BDifferent%252BMebendazole%252BPolymorphs%252Bin%252Ba%252BMouse%252BBrain%252BTumor%252BModel%26aulast%3DBai%26aufirst%3DRen-Yuan%26date%3D2015%26date%3D2015%26volume%3D21%26issue%3D15%26spage%3D3462%26epage%3D3470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Michaelis</span>, <span class="hlFld-ContribAuthor ">Bishr  Agha</span>, <span class="hlFld-ContribAuthor ">Florian  Rothweiler</span>, <span class="hlFld-ContribAuthor ">Nadine  Löschmann</span>, <span class="hlFld-ContribAuthor ">Yvonne  Voges</span>, <span class="hlFld-ContribAuthor ">Michel  Mittelbronn</span>, <span class="hlFld-ContribAuthor ">Tatjana  Starzetz</span>, <span class="hlFld-ContribAuthor ">Patrick N.  Harter</span>, <span class="hlFld-ContribAuthor ">Behnaz A.  Abhari</span>, <span class="hlFld-ContribAuthor ">Simone  Fulda</span>, <span class="hlFld-ContribAuthor ">Frank  Westermann</span>, <span class="hlFld-ContribAuthor ">Kristoffer  Riecken</span>, <span class="hlFld-ContribAuthor ">Silvia  Spek</span>, <span class="hlFld-ContribAuthor ">Klaus  Langer</span>, <span class="hlFld-ContribAuthor ">Michael  Wiese</span>, <span class="hlFld-ContribAuthor ">Wilhelm G.  Dirks</span>, <span class="hlFld-ContribAuthor ">Richard  Zehner</span>, <span class="hlFld-ContribAuthor ">Jaroslav  Cinatl</span>, <span class="hlFld-ContribAuthor ">Mark N.  Wass</span>, <span class="hlFld-ContribAuthor ">Jindrich  Cinatl</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of flubendazole as potential anti-neuroblastoma compound in a large cell line screen. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2015,</strong> <em>5 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/srep08202" title="DOI URL">https://doi.org/10.1038/srep08202</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/srep08202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsrep08202%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DIdentification%252Bof%252Bflubendazole%252Bas%252Bpotential%252Banti-neuroblastoma%252Bcompound%252Bin%252Ba%252Blarge%252Bcell%252Bline%252Bscreen%26aulast%3DMichaelis%26aufirst%3DMartin%26date%3D2015%26date%3D2015%26volume%3D5%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chanin  Nantasenamat</span>, <span class="hlFld-ContribAuthor ">Virapong  Prachayasittikul</span>. </span><span class="cited-content_cbyCitation_article-title">Maximizing computational tools for successful drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2015,</strong> <em>10 </em>
                                    (4)
                                     , 321-329. <a href="https://doi.org/10.1517/17460441.2015.1016497" title="DOI URL">https://doi.org/10.1517/17460441.2015.1016497</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2015.1016497&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2015.1016497%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DMaximizing%252Bcomputational%252Btools%252Bfor%252Bsuccessful%252Bdrug%252Bdiscovery%26aulast%3DNantasenamat%26aufirst%3DChanin%26date%3D2015%26date%3D2015%26volume%3D10%26issue%3D4%26spage%3D321%26epage%3D329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ren-Yuan  Bai</span>, <span class="hlFld-ContribAuthor ">Verena  Staedtke</span>, <span class="hlFld-ContribAuthor ">Charles M.  Rudin</span>, <span class="hlFld-ContribAuthor ">Fred  Bunz</span>, <span class="hlFld-ContribAuthor ">Gregory J.  Riggins</span>. </span><span class="cited-content_cbyCitation_article-title">Effective treatment of diverse medulloblastoma models with mebendazole and its impact on tumor angiogenesis. </span><span class="cited-content_cbyCitation_journal-name">Neuro-Oncology</span><span> <strong>2015,</strong> <em>17 </em>
                                    (4)
                                     , 545-554. <a href="https://doi.org/10.1093/neuonc/nou234" title="DOI URL">https://doi.org/10.1093/neuonc/nou234</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/neuonc/nou234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fneuonc%2Fnou234%26sid%3Dliteratum%253Aachs%26jtitle%3DNeuro-Oncology%26atitle%3DEffective%252Btreatment%252Bof%252Bdiverse%252Bmedulloblastoma%252Bmodels%252Bwith%252Bmebendazole%252Band%252Bits%252Bimpact%252Bon%252Btumor%252Bangiogenesis%26aulast%3DBai%26aufirst%3DRen-Yuan%26date%3D2015%26date%3D2014%26volume%3D17%26issue%3D4%26spage%3D545%26epage%3D554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Monica  Dixon</span>, <span class="hlFld-ContribAuthor ">Jordan  Woodrick</span>, <span class="hlFld-ContribAuthor ">Suhani  Gupta</span>, <span class="hlFld-ContribAuthor ">Soumendra Krishna  Karmahapatra</span>, <span class="hlFld-ContribAuthor ">Stephen  Devito</span>, <span class="hlFld-ContribAuthor ">Sona  Vasudevan</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>, <span class="hlFld-ContribAuthor ">Sanjay  Adhikari</span>, <span class="hlFld-ContribAuthor ">Venkata M.  Yenugonda</span>, <span class="hlFld-ContribAuthor ">Rabindra  Roy</span>. </span><span class="cited-content_cbyCitation_article-title">Naturally occurring polyphenol, morin hydrate, inhibits enzymatic activity of N-methylpurine DNA glycosylase, a DNA repair enzyme with various roles in human disease. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2015,</strong> <em>23 </em>
                                    (5)
                                     , 1102-1111. <a href="https://doi.org/10.1016/j.bmc.2014.12.067" title="DOI URL">https://doi.org/10.1016/j.bmc.2014.12.067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2014.12.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2014.12.067%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DNaturally%252Boccurring%252Bpolyphenol%25252C%252Bmorin%252Bhydrate%25252C%252Binhibits%252Benzymatic%252Bactivity%252Bof%252BN-methylpurine%252BDNA%252Bglycosylase%25252C%252Ba%252BDNA%252Brepair%252Benzyme%252Bwith%252Bvarious%252Broles%252Bin%252Bhuman%252Bdisease%26aulast%3DDixon%26aufirst%3DMonica%26date%3D2015%26volume%3D23%26issue%3D5%26spage%3D1102%26epage%3D1111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hiba Abi  Hussein</span>, <span class="hlFld-ContribAuthor ">Alexandre  Borrel</span>, <span class="hlFld-ContribAuthor ">Leslie  Regad</span>, <span class="hlFld-ContribAuthor ">Delphine  Flatters</span>, <span class="hlFld-ContribAuthor ">Anne  Badel</span>, <span class="hlFld-ContribAuthor ">Colette  Geneix</span>, <span class="hlFld-ContribAuthor ">Michel  Petitjean</span>, <span class="hlFld-ContribAuthor ">Anne-Claude  Camproux</span>, <span class="hlFld-ContribAuthor ">Olivier  Taboureau</span>. </span><span class="cited-content_cbyCitation_article-title">Systems Biology. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 409-425. <a href="https://doi.org/10.1016/B978-0-12-417205-0.00017-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-417205-0.00017-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-417205-0.00017-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-417205-0.00017-1%26sid%3Dliteratum%253Aachs%26atitle%3DSystems%252BBiology%26aulast%3DHussein%26aufirst%3DHiba%2BAbi%26date%3D2015%26spage%3D409%26epage%3D425%26pub%3DElsevier%26atitle%3DThe%252BPractice%252Bof%252BMedicinal%252BChemistry%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isidro  Cortés-Ciriano</span>, <span class="hlFld-ContribAuthor ">Qurrat Ul  Ain</span>, <span class="hlFld-ContribAuthor ">Vigneshwari  Subramanian</span>, <span class="hlFld-ContribAuthor ">Eelke B.  Lenselink</span>, <span class="hlFld-ContribAuthor ">Oscar  Méndez-Lucio</span>, <span class="hlFld-ContribAuthor ">Adriaan P.  IJzerman</span>, <span class="hlFld-ContribAuthor ">Gerd  Wohlfahrt</span>, <span class="hlFld-ContribAuthor ">Peteris  Prusis</span>, <span class="hlFld-ContribAuthor ">Thérèse E.  Malliavin</span>, <span class="hlFld-ContribAuthor ">Gerard J. P.  van Westen</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Polypharmacology modelling using proteochemometrics (PCM): recent methodological developments, applications to target families, and future prospects. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (1)
                                     , 24-50. <a href="https://doi.org/10.1039/C4MD00216D" title="DOI URL">https://doi.org/10.1039/C4MD00216D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00216D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00216D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DPolypharmacology%252Bmodelling%252Busing%252Bproteochemometrics%252B%252528PCM%252529%25253A%252Brecent%252Bmethodological%252Bdevelopments%25252C%252Bapplications%252Bto%252Btarget%252Bfamilies%25252C%252Band%252Bfuture%252Bprospects%26aulast%3DCort%25C3%25A9s-Ciriano%26aufirst%3DIsidro%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D1%26spage%3D24%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isidro  Cortes-Ciriano</span>, <span class="hlFld-ContribAuthor ">Gerard JP  van Westen</span>, <span class="hlFld-ContribAuthor ">Eelke Bart  Lenselink</span>, <span class="hlFld-ContribAuthor ">Daniel S  Murrell</span>, <span class="hlFld-ContribAuthor ">Andreas  Bender</span>, <span class="hlFld-ContribAuthor ">Thérèse  Malliavin</span>. </span><span class="cited-content_cbyCitation_article-title">Proteochemometric modeling in a Bayesian framework. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2014,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/1758-2946-6-35" title="DOI URL">https://doi.org/10.1186/1758-2946-6-35</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/1758-2946-6-35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2F1758-2946-6-35%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DProteochemometric%252Bmodeling%252Bin%252Ba%252BBayesian%252Bframework%26aulast%3DCortes-Ciriano%26aufirst%3DIsidro%26date%3D2014%26date%3D2014%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tiejun  Cheng</span>, <span class="hlFld-ContribAuthor ">Yongmei  Pan</span>, <span class="hlFld-ContribAuthor ">Ming  Hao</span>, <span class="hlFld-ContribAuthor ">Yanli  Wang</span>, <span class="hlFld-ContribAuthor ">Stephen H.  Bryant</span>. </span><span class="cited-content_cbyCitation_article-title">PubChem applications in drug discovery: a bibliometric analysis. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2014,</strong> <em>19 </em>
                                    (11)
                                     , 1751-1756. <a href="https://doi.org/10.1016/j.drudis.2014.08.008" title="DOI URL">https://doi.org/10.1016/j.drudis.2014.08.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2014.08.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2014.08.008%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DPubChem%252Bapplications%252Bin%252Bdrug%252Bdiscovery%25253A%252Ba%252Bbibliometric%252Banalysis%26aulast%3DCheng%26aufirst%3DTiejun%26date%3D2014%26volume%3D19%26issue%3D11%26spage%3D1751%26epage%3D1756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chanin  Nantasenamat</span>, <span class="hlFld-ContribAuthor ">Saw  Simeon</span>, <span class="hlFld-ContribAuthor ">Wiwat  Owasirikul</span>, <span class="hlFld-ContribAuthor ">Napat  Songtawee</span>, <span class="hlFld-ContribAuthor ">Maris  Lapins</span>, <span class="hlFld-ContribAuthor ">Virapong  Prachayasittikul</span>, <span class="hlFld-ContribAuthor ">Jarl E. S.  Wikberg</span>. </span><span class="cited-content_cbyCitation_article-title">Illuminating the origins of spectral properties of green fluorescent proteins via proteochemometric and molecular modeling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2014,</strong> <em>35 </em>
                                    (27)
                                     , 1951-1966. <a href="https://doi.org/10.1002/jcc.23708" title="DOI URL">https://doi.org/10.1002/jcc.23708</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.23708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.23708%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DIlluminating%252Bthe%252Borigins%252Bof%252Bspectral%252Bproperties%252Bof%252Bgreen%252Bfluorescent%252Bproteins%252Bvia%252Bproteochemometric%252Band%252Bmolecular%252Bmodeling%26aulast%3DNantasenamat%26aufirst%3DChanin%26date%3D2014%26date%3D2014%26volume%3D35%26issue%3D27%26spage%3D1951%26epage%3D1966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liang  Cheng</span>, <span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Peng  Ju</span>, <span class="hlFld-ContribAuthor ">Jiajie  Peng</span>, <span class="hlFld-ContribAuthor ">Yadong  Wang</span>, . </span><span class="cited-content_cbyCitation_article-title">SemFunSim: A New Method for Measuring Disease Similarity by Integrating Semantic and Gene Functional Association. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (6)
                                     , e99415. <a href="https://doi.org/10.1371/journal.pone.0099415" title="DOI URL">https://doi.org/10.1371/journal.pone.0099415</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0099415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0099415%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DSemFunSim%25253A%252BA%252BNew%252BMethod%252Bfor%252BMeasuring%252BDisease%252BSimilarity%252Bby%252BIntegrating%252BSemantic%252Band%252BGene%252BFunctional%252BAssociation%26aulast%3DCheng%26aufirst%3DLiang%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D6%26spage%3De99415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naiem T  Issa</span>, <span class="hlFld-ContribAuthor ">Stephen W  Byers</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>. </span><span class="cited-content_cbyCitation_article-title">Big data: the next frontier for innovation in therapeutics and healthcare. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Clinical Pharmacology</span><span> <strong>2014,</strong> <em>7 </em>
                                    (3)
                                     , 293-298. <a href="https://doi.org/10.1586/17512433.2014.905201" title="DOI URL">https://doi.org/10.1586/17512433.2014.905201</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1586/17512433.2014.905201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1586%2F17512433.2014.905201%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Clinical%2520Pharmacology%26atitle%3DBig%252Bdata%25253A%252Bthe%252Bnext%252Bfrontier%252Bfor%252Binnovation%252Bin%252Btherapeutics%252Band%252Bhealthcare%26aulast%3DIssa%26aufirst%3DNaiem%2BT%26date%3D2014%26date%3D2014%26volume%3D7%26issue%3D3%26spage%3D293%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shahin  Assefnia</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>, <span class="hlFld-ContribAuthor ">Jaime M.  Guidry Auvil</span>, <span class="hlFld-ContribAuthor ">Constanze  Hampel</span>, <span class="hlFld-ContribAuthor ">Panos Z.  Anastasiadis</span>, <span class="hlFld-ContribAuthor ">Bhaskar  Kallakury</span>, <span class="hlFld-ContribAuthor ">Aykut  Uren</span>, <span class="hlFld-ContribAuthor ">David W  Foley</span>, <span class="hlFld-ContribAuthor ">Milton L.  Brown</span>, <span class="hlFld-ContribAuthor ">Lawrence  Shapiro</span>, <span class="hlFld-ContribAuthor ">Michael  Brenner</span>, <span class="hlFld-ContribAuthor ">David  Haigh</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Byers</span>. </span><span class="cited-content_cbyCitation_article-title">Cadherin-11 in poor prognosis malignancies and rheumatoid arthritis: common target, common therapies. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2014,</strong> <em>5 </em>
                                    (6)
                                     , 1458-1474. <a href="https://doi.org/10.18632/oncotarget.1538" title="DOI URL">https://doi.org/10.18632/oncotarget.1538</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.1538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.1538%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DCadherin-11%252Bin%252Bpoor%252Bprognosis%252Bmalignancies%252Band%252Brheumatoid%252Barthritis%25253A%252Bcommon%252Btarget%25252C%252Bcommon%252Btherapies%26aulast%3DAssefnia%26aufirst%3DShahin%26date%3D2014%26date%3D2013%26volume%3D5%26issue%3D6%26spage%3D1458%26epage%3D1474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Ye</span>, <span class="hlFld-ContribAuthor ">Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Jia  Wei</span>, . </span><span class="cited-content_cbyCitation_article-title">Construction of Drug Network Based on Side Effects and Its Application for Drug Repositioning. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2014,</strong> <em>9 </em>
                                    (2)
                                     , e87864. <a href="https://doi.org/10.1371/journal.pone.0087864" title="DOI URL">https://doi.org/10.1371/journal.pone.0087864</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0087864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0087864%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DConstruction%252Bof%252BDrug%252BNetwork%252BBased%252Bon%252BSide%252BEffects%252Band%252BIts%252BApplication%252Bfor%252BDrug%252BRepositioning%26aulast%3DYe%26aufirst%3DHao%26date%3D2014%26date%3D2014%26volume%3D9%26issue%3D2%26spage%3De87864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna M.  Gallina</span>, <span class="hlFld-ContribAuthor ">Peer  Bork</span>, <span class="hlFld-ContribAuthor ">Domenico  Bordo</span>. </span><span class="cited-content_cbyCitation_article-title">Structural analysis of protein-ligand interactions: the binding of endogenous compounds and of synthetic drugs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Recognition</span><span> <strong>2014,</strong> <em>27 </em>
                                    (2)
                                     , 65-72. <a href="https://doi.org/10.1002/jmr.2332" title="DOI URL">https://doi.org/10.1002/jmr.2332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jmr.2332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjmr.2332%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Recognition%26atitle%3DStructural%252Banalysis%252Bof%252Bprotein-ligand%252Binteractions%25253A%252Bthe%252Bbinding%252Bof%252Bendogenous%252Bcompounds%252Band%252Bof%252Bsynthetic%252Bdrugs%26aulast%3DGallina%26aufirst%3DAnna%2BM.%26date%3D2014%26date%3D2014%26volume%3D27%26issue%3D2%26spage%3D65%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jochen  Schacht</span>. </span><span class="cited-content_cbyCitation_article-title">My Dull Deaf Ears: Four Millennia of Acquired Hearing Loss. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 551-567. <a href="https://doi.org/10.1007/978-1-4614-9102-6_29" title="DOI URL">https://doi.org/10.1007/978-1-4614-9102-6_29</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4614-9102-6_29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4614-9102-6_29%26sid%3Dliteratum%253Aachs%26atitle%3DMy%252BDull%252BDeaf%252BEars%25253A%252BFour%252BMillennia%252Bof%252BAcquired%252BHearing%252BLoss%26aulast%3DSchacht%26aufirst%3DJochen%26date%3D2014%26date%3D2014%26spage%3D551%26epage%3D567%26pub%3DSpringer%2520New%2520York%26atitle%3DPerspectives%252Bon%252BAuditory%252BResearch%26aulast%3DPopper%26aufirst%3DArthur%2BN.%26date%3D2014%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruben  Abagyan</span>, <span class="hlFld-ContribAuthor ">Clarisse Gravina  Ricci</span>. </span><span class="cited-content_cbyCitation_article-title">The Human Pocketome. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 79-96. <a href="https://doi.org/10.1002/9783527677016.ch3" title="DOI URL">https://doi.org/10.1002/9783527677016.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527677016.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527677016.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BHuman%252BPocketome%26aulast%3DAbagyan%26aufirst%3DRuben%26date%3D2013%26date%3D2013%26spage%3D79%26epage%3D96%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DDe%252Bnovo%252BMolecular%252BDesign%26aulast%3DSchneider%26aufirst%3DGisbert%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Men-Bao  Qian</span>, <span class="hlFld-ContribAuthor ">Peiling  Yap</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Yang</span>, <span class="hlFld-ContribAuthor ">Hai  Liang</span>, <span class="hlFld-ContribAuthor ">Zhi-Hua  Jiang</span>, <span class="hlFld-ContribAuthor ">Wei  Li</span>, <span class="hlFld-ContribAuthor ">Yu-Guang  Tan</span>, <span class="hlFld-ContribAuthor ">Hui  Zhou</span>, <span class="hlFld-ContribAuthor ">Jürg  Utzinger</span>, <span class="hlFld-ContribAuthor ">Xiao-Nong  Zhou</span>, <span class="hlFld-ContribAuthor ">Jennifer  Keiser</span>. </span><span class="cited-content_cbyCitation_article-title">Efficacy and Safety of Tribendimidine Against Clonorchis sinensis. </span><span class="cited-content_cbyCitation_journal-name">Clinical Infectious Diseases</span><span> <strong>2013,</strong> <em>56 </em>
                                    (7)
                                     , e76-e82. <a href="https://doi.org/10.1093/cid/cis1011" title="DOI URL">https://doi.org/10.1093/cid/cis1011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/cid/cis1011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fcid%2Fcis1011%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Infectious%2520Diseases%26atitle%3DEfficacy%252Band%252BSafety%252Bof%252BTribendimidine%252BAgainst%252BClonorchis%252Bsinensis%26aulast%3DQian%26aufirst%3DMen-Bao%26date%3D2013%26date%3D2012%26volume%3D56%26issue%3D7%26spage%3De76%26epage%3De82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naiem T  Issa</span>, <span class="hlFld-ContribAuthor ">Jordan  Kruger</span>, <span class="hlFld-ContribAuthor ">Stephen W  Byers</span>, <span class="hlFld-ContribAuthor ">Sivanesan  Dakshanamurthy</span>. </span><span class="cited-content_cbyCitation_article-title">Drug repurposing a reality: from computers to the clinic. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Clinical Pharmacology</span><span> <strong>2013,</strong> <em>6 </em>
                                    (2)
                                     , 95-97. <a href="https://doi.org/10.1586/ecp.12.79" title="DOI URL">https://doi.org/10.1586/ecp.12.79</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1586/ecp.12.79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1586%2Fecp.12.79%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Clinical%2520Pharmacology%26atitle%3DDrug%252Brepurposing%252Ba%252Breality%25253A%252Bfrom%252Bcomputers%252Bto%252Bthe%252Bclinic%26aulast%3DIssa%26aufirst%3DNaiem%2BT%26date%3D2013%26date%3D2014%26volume%3D6%26issue%3D2%26spage%3D95%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiyoun  Lee</span>, <span class="hlFld-ContribAuthor ">Matthew  Bogyo</span>. </span><span class="cited-content_cbyCitation_article-title">Target deconvolution techniques in modern phenotypic profiling. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2013,</strong> <em>17 </em>
                                    (1)
                                     , 118-126. <a href="https://doi.org/10.1016/j.cbpa.2012.12.022" title="DOI URL">https://doi.org/10.1016/j.cbpa.2012.12.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2012.12.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2012.12.022%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTarget%252Bdeconvolution%252Btechniques%252Bin%252Bmodern%252Bphenotypic%252Bprofiling%26aulast%3DLee%26aufirst%3DJiyoun%26date%3D2013%26volume%3D17%26issue%3D1%26spage%3D118%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_null" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0015.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0003.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Graphical summary of the TMFS method work flow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Accuracy of the TMFS method represented by ROC curves. We examined the TMFS method accuracy against the Glide docking scoring function. Here we report, in increasing order of enrichment of true bioactive compounds, the performance of each scoring method via their respective AUC: Glide score (0.3889, red), Glide score + atom pair (AP) similarity (0.3889, yellow), shape descriptors only (0.6905, teal), ligand-centric descriptors only (0.7500, blue), Glide score + AP similarity + post-shape (0.8167, green), and TMFS score (0.8810, purple). (B) Validation of predicted drug–target associations for FDA approved drugs. Predicted drug–target associations for each FDA drug in the top 1 through top 40 ranked hits were individually matched against the publically available experimental binding and functional data. Percent correctly predicted (PCP) targets were then calculated using eq <a class="ref internalNav" href="#eq9" aria-label="9">9</a> for each category of the top rank lists (i.e., top 1 to top 40). The histogram (filled bars) represents the percent correctly predicted (PCP) targets (<i>y</i>-axis) for each category of top rank lists. Error bars and percentages are highlighted on each histogram bar.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Principle component analysis (PCA) of individual protein- and ligand-based descriptor variables for determination of descriptor correlation with obtaining reliable predictions. Scree plot depicting the first three principal components accounting for the majority of the data variance. The first three principle components account for the majority of the data variance; hence, the transformed eigenvalue coefficients of the above descriptor variables were plotted against the first three principle components in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure 1">Supporting Information Figure 1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Analysis of FDA drug–target association. Frequency histogram depicting the number of protein target hits (<i>y</i>-axis) for each FDA drug (<i>x</i>-axis). Targets are considered hits for a particular molecule if the final ranking (<i>Z</i>-score) of the molecule places it in the top 1 position or somewhere in the top 40 positions. (A) Frequency histogram depicting the number of protein targets hit (<i>y</i>-aixs) for each FDA drugs (<i>x</i>-axis) in the top 1 position. The 2D structure of staurosporine, the drug with the most hits, is also displayed. (B) Frequency histogram depicting the number of protein target hits (<i>y</i>-axis) for each FDA drug (<i>x</i>-axis) in the top 40 position. The top 40 provides a more relaxed criterion for protein targets to be considered as hits. As such, for those molecules that survive the final cut-offs and are found in the top 40 rank list for a particular protein, we predict that they have a good potential to bind to that given target. DB02197, DB03376, and DB02916, drugs with the most predicted hits are depicted. (C) To further enrich our prediction paradigm, we included one more term corresponding to ligand shape. The value for this term is the rmsd of the docked ligand compared to the active conformation of the cocrystallized ligand for a particular protein crystal structure, which is derived from a set of 100 protein targets. The histogram portrays the frequency of hits of each FDA drug along with the 2D structure of the drug with most target hits (indomethacin).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Analysis of FDA blockbuster drug-target association. (A) Heat map depicting hit frequencies of the top 200 “blockbuster” FDA drugs across each top-rank category. Each box shows the number of occurrences, while the color scheme illustrates high frequencies as red and low frequencies as blue. (B) Heat map showing FDA approved drugs predicted to hit the greatest number of protein targets: Sutent, Alimta, Lescol, Celebrex, Premarin, Zetia, and Blopress. (C) Heat map portraying FDA approved drugs that have no hits in our protein data set: Prograf, Valcote, Concerta, Sifrol, Niaspan, Exelon, Evodart, Sevorane, and Klacid. (D) Histogram showing the percentage of total protein targets in our data set that have a FDA blockbuster drug in their top 1, 5, and 20 rank lists.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Analysis of drug promiscuity. The “value of promiscuity (nonspecificity) for each drug is represented as a numerical score from the combined sum of the number of unique folds and the number of unique families that a particular molecule is predicted to hit. The drug with the greatest “value of nonspecificity” is considered to be the most promiscuous molecule. The histogram depicts the “values of nonspecificity” (<i>y</i>-axis) for each drug (<i>x</i>-axis) that had been ranked in the top 1 position, along with the 2D structures of the three most promiscuous compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0009.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Similarly shaped protein binding pockets bind similar molecules. (A) Histogram where the left-most protein target on the <i>x</i>-axis corresponded to the protein target whose pocket was most similar to the template. If these histograms tapered off to the right, this indicates that protein target–ligand commonality is highly correlated to the three-dimensional spatial similarity of their binding pockets. (B) Commonality of the top-ranked drugs. The predicted top 5 ranked drugs were counted for each target. Commonality is defined as the number of times a molecule from the top-rank list for a reference protein target also shows up in the corresponding top-rank list for the rest of the targets. The histogram depicts the “commonality score” for molecules within the top 5 rank list for each protein target data set. The top 5 protein targets (with respect to commonality score), which were cocrystallized with a nucleotide (4 out of 5 are GDP, one is adenosine), are highlighted with their PDB codes and name. (C) Histogram depicting the number of molecules in common for all protein targets ordered from greatest to least with respect to pocket shape similarity to VEGFR2. (D) Histogram depicting the number of molecules in common for all protein targets ordered from greatest to least with respect to pocket shape similarity to ERα.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0010.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Mebendazole binds directly to VEGFR2 kinase assay and also inhibits angiogenesis. (A) Mebendazole binds directly to VEGFR2 and affects VEGFR2 kinase activity with an IC<sub>50</sub> of 3.6 μM. IC<sub>50</sub> curves were generated using GraphPad 5 and a standard four-parameter nonlinear regression model (log [inhibitor] vs response – variable slope). Data points correspond to the averages of duplicate wells, and error bars represent the mean ± replicate % activity. The graphical representation shows dashed lines at the IC<sub>50</sub> values, where the vertical line is at log <i>x</i> = −5.4437. Solving the equation log <i>x</i> = −5.4437 results in an IC<sub>50</sub> of 3.6 μM. (B) Control: Staurosporine binds to VEGFR2 and affects kinase activity with an IC<sub>50</sub> of 8 nM. (C) Mebendazole, predicted to act as a VEGFR2 inhibitor by TMFS, inhibits angiogenesis in a HUVEC cell based assay. Mebendazole significantly inhibited network formation with an IC<sub>50</sub> of 8.8 μM, which is implicated by the lack of cellular migration, alignment, and branching.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0011.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Celecoxib (CCB) and dimethyl celecoxib (DMC) bind directly to immobilized cadherin-11 (CDH11) in surface plasmon resonance (SPR) assay. (A) CCB and DMC bind to recombinant mouse extracellular domain (EC) 1-2 of CDH11 protein immobilized on the surface of the chip via similar patterns, as evident in the sensogram. CCB and DMC were separately injected three times on the CM5 chip at 25 μM. (B, C) CCB and DMC bind in a dose-dependent manner. (B) Lower magnification of the sensogram showing the signals generated from 200 and 100 μM DMC bound to cadherin-11. (C) Higher magnification of the compacted signals from panel A showing the binding levels of 50, 25, and 12 μM DMC to cadherin-11. Assays were performed in triplicate for each DMC concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/medium/jm-2012-00576q_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0012.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Growth inhibition of MDA-MB-231 invasive breast cancer cell line by celecoxib (CCB) and its COX-2 inactive analogue dimethyl celecoxib (DMC). MTS assays demonstrate concentration-dependent cell growth inhibition when MDA-MB-231 cells were exposed to increasing doses of CCB or DMC for 48 h. Data are presented as the mean ± SEM. (A) CCB causes inhibition with an IC<sub>50</sub> of 40 μM, and (B) DMC causes inhibition with an IC<sub>50</sub> of 36 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-15/jm300576q/production/images/large/jm-2012-00576q_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300576q&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i48">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 59 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Paul, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mytelka, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunwiddie, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Persinger, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munos, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindborg, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schacht, A. L.</span><span> </span><span class="NLM_article-title">How to improve R&D productivity: the pharmaceutical industry’s grand challenge</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">214</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnrd3078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=20168317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFemsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=203-214&author=S.+M.+Paulauthor=D.+S.+Mytelkaauthor=C.+T.+Dunwiddieauthor=C.+C.+Persingerauthor=B.+H.+Munosauthor=S.+R.+Lindborgauthor=A.+L.+Schacht&title=How+to+improve+R%26D+productivity%3A+the+pharmaceutical+industry%E2%80%99s+grand+challenge"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">How to improve R&D productivity: the pharmaceutical industry's grand challenge</span></div><div class="casAuthors">Paul, Steven M.; Mytelka, Daniel S.; Dunwiddie, Christopher T.; Persinger, Charles C.; Munos, Bernard H.; Lindborg, Stacy R.; Schacht, Aaron L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">203-214</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is under growing pressure from a range of environmental issues, including major losses of revenue owing to patent expirations, increasingly cost-constrained healthcare systems and more demanding regulatory requirements.  In our view, the key to tackling the challenges such issues pose to both the future viability of the pharmaceutical industry and advances in healthcare is to substantially increase the no. and quality of innovative, cost-effective new medicines, without incurring unsustainable R&D costs.  However, it is widely acknowledged that trends in industry R&D productivity have been moving in the opposite direction for a no. of years.  Here, we present a detailed anal. based on comprehensive, recent, industry-wide data to identify the relative contributions of each of the steps in the drug discovery and development process to overall R&D productivity.  We then propose specific strategies that could have the most substantial impact in improving R&D productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY67PCIhaBVLVg90H21EOLACvtfcHk0li3HzYvQY7RFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFemsbg%253D&md5=2f32bcc48c869290eef18ff9400afcc5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3078%26sid%3Dliteratum%253Aachs%26aulast%3DPaul%26aufirst%3DS.%2BM.%26aulast%3DMytelka%26aufirst%3DD.%2BS.%26aulast%3DDunwiddie%26aufirst%3DC.%2BT.%26aulast%3DPersinger%26aufirst%3DC.%2BC.%26aulast%3DMunos%26aufirst%3DB.%2BH.%26aulast%3DLindborg%26aufirst%3DS.%2BR.%26aulast%3DSchacht%26aufirst%3DA.%2BL.%26atitle%3DHow%2520to%2520improve%2520R%2526D%2520productivity%253A%2520the%2520pharmaceutical%2520industry%25E2%2580%2599s%2520grand%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D203%26epage%3D214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Lawrence, S.</span><span> </span><span class="NLM_article-title">Drug output slows in 2006</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1073</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1073&author=S.+Lawrence&title=Drug+output+slows+in+2006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLawrence%26aufirst%3DS.%26atitle%3DDrug%2520output%2520slows%2520in%25202006%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Southall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasgar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C.</span><span> </span><span class="NLM_article-title">The NCGC Pharmaceutical Collection: a comprehensive resource of clinically approved drugs enabling repurposing and chemical genomics</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">80</span><span class="NLM_x">) </span> <span class="NLM_fpage">80ps16</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscitranslmed.3001862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21525397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntleksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=80ps16&issue=80&author=R.+Huangauthor=N.+Southallauthor=Y.+Wangauthor=A.+Yasgarauthor=P.+Shinnauthor=A.+Jadhavauthor=D.+Nguyenauthor=C.+Austin&title=The+NCGC+Pharmaceutical+Collection%3A+a+comprehensive+resource+of+clinically+approved+drugs+enabling+repurposing+and+chemical+genomics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The NCGC Pharmaceutical Collection: a comprehensive resource of clinically approved drugs enabling repurposing and chemical genomics</span></div><div class="casAuthors">Huang, Ruili; Southall, Noel; Wang, Yuhong; Yasgar, Adam; Shinn, Paul; Jadhav, Ajit; Nguyen, Dac-Trung; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">80</span>),
    <span class="NLM_cas:pages">80ps16, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Small-mol. compds. approved for use as drugs may be "repurposed" for new indications and studied to det. the mechanisms of their beneficial and adverse effects.  A comprehensive collection of all small-mol. drugs approved for human use would be invaluable for systematic repurposing across human diseases, particularly for rare and neglected diseases, for which the cost and time required for development of a new chem. entity are often prohibitive.  Previous efforts to build such a comprehensive collection have been limited by the complexities, redundancies, and semantic inconsistencies of drug naming within and among regulatory agencies worldwide; a lack of clear conceptualization of what constitutes a drug; and a lack of access to phys. samples.  We report here the creation of a definitive, complete, and nonredundant list of all approved mol. entities as a freely available electronic resource and a phys. collection of small mols. amenable to high-throughput screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPSN4jSGue67Vg90H21EOLACvtfcHk0lgaIWSi0ova-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntleksr4%253D&md5=32b2767a0d1e74c000ca55853aec37e4</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3001862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3001862%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DSouthall%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DShinn%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DAustin%26aufirst%3DC.%26atitle%3DThe%2520NCGC%2520Pharmaceutical%2520Collection%253A%2520a%2520comprehensive%2520resource%2520of%2520clinically%2520approved%2520drugs%2520enabling%2520repurposing%2520and%2520chemical%2520genomics%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26issue%3D80%26spage%3D80ps16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Collins, F. S.</span><span> </span><span class="NLM_article-title">Mining for therapeutic gold</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">395</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=395&issue=6&author=F.+S.+Collins&title=Mining+for+therapeutic+gold"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DF.%2BS.%26atitle%3DMining%2520for%2520therapeutic%2520gold%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26issue%3D6%26spage%3D395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">DiMasi, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grabowski, H. G.</span><span> </span><span class="NLM_article-title">The price of innovation: new estimates of drug development costs</span> <span class="citation_source-journal">J. Health Econ.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS0167-6296%2802%2900126-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12606142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A280%3ADC%252BD3s%252FpsFSisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=151-185&issue=2&author=J.+A.+DiMasiauthor=R.+W.+Hansenauthor=H.+G.+Grabowski&title=The+price+of+innovation%3A+new+estimates+of+drug+development+costs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The price of innovation: new estimates of drug development costs</span></div><div class="casAuthors">DiMasi Joseph A; Hansen Ronald W; Grabowski Henry G</div><div class="citationInfo"><span class="NLM_cas:title">Journal of health economics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">151-85</span>
        ISSN:<span class="NLM_cas:issn">0167-6296</span>.
    </div><div class="casAbstract">The research and development costs of 68 randomly selected new drugs were obtained from a survey of 10 pharmaceutical firms.  These data were used to estimate the average pre-tax cost of new drug development.  The costs of compounds abandoned during testing were linked to the costs of compounds that obtained marketing approval.  The estimated average out-of-pocket cost per new drug is 403 million US dollars (2000 dollars).  Capitalizing out-of-pocket costs to the point of marketing approval at a real discount rate of 11% yields a total pre-approval cost estimate of 802 million US dollars (2000 dollars).  When compared to the results of an earlier study with a similar methodology, total capitalized costs were shown to have increased at an annual rate of 7.4% above general price inflation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcReRsWHw2uElYhLSh9fo81bfW6udTcc2ealTGpDhYc21bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s%252FpsFSisQ%253D%253D&md5=814fa4d06a749145df19ae7492bf323d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0167-6296%2802%2900126-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0167-6296%252802%252900126-1%26sid%3Dliteratum%253Aachs%26aulast%3DDiMasi%26aufirst%3DJ.%2BA.%26aulast%3DHansen%26aufirst%3DR.%2BW.%26aulast%3DGrabowski%26aufirst%3DH.%2BG.%26atitle%3DThe%2520price%2520of%2520innovation%253A%2520new%2520estimates%2520of%2520drug%2520development%2520costs%26jtitle%3DJ.%2520Health%2520Econ.%26date%3D2003%26volume%3D22%26issue%3D2%26spage%3D151%26epage%3D185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Buehler, L. K.</span><span> </span><span class="NLM_article-title">Advancing drug discovery—beyond design</span> <span class="citation_source-journal">Pharmagenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">24</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnslagu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=24-26&author=L.+K.+Buehler&title=Advancing+drug+discovery%E2%80%94beyond+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Advancing drug discovery - beyond design</span></div><div class="casAuthors">Buehler, Lukas K.</div><div class="citationInfo"><span class="NLM_cas:title">PharmaGenomics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">24, 26</span>CODEN:
                <span class="NLM_cas:coden">PHARCV</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Advanstar Communications, Inc.</span>)
        </div><div class="casAbstract">A review.  Drug discovery based upon high-throughput screening, combinatorial chem. and rational drug design faces not a practical but a conceptual hurdle: the redundancy inherent to biol. systems.  This conceptual hurdle is in the form of a structural and functional redundancy of metabolic pathways, signaling networks and protein complexes in living organisms.  All are based upon the combinatorial use of a limited set of structural motifs and domains in proteins.  The functional redundancy easily can be traced back to the way protein structures are formed.  At the mol. level, function follows form.  Single mutations often can change a similarity into a different one.  Often, proteins might not be highly specific and can bind many ligands and substrates, although with varying efficiency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRhLatpc6utLVg90H21EOLACvtfcHk0ljatjvGt_EpSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnslagu7s%253D&md5=56a1813f7a55420bfbd1d1de606a29b9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBuehler%26aufirst%3DL.%2BK.%26atitle%3DAdvancing%2520drug%2520discovery%25E2%2580%2594beyond%2520design%26jtitle%3DPharmagenomics%26date%3D2004%26volume%3D4%26spage%3D24%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Hughes, B.</span><span> </span><span class="NLM_article-title">2007 FDA drug approvals: a year of flux</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=107-109&issue=7&author=B.+Hughes&title=2007+FDA+drug+approvals%3A+a+year+of+flux"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DB.%26atitle%3D2007%2520FDA%2520drug%2520approvals%253A%2520a%2520year%2520of%2520flux%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26issue%3D7%26spage%3D107%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Ashburn, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thor, K. B.</span><span> </span><span class="NLM_article-title">Drug repositioning: identifying and developing new uses for existing drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">683</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnrd1468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=15286734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=673-683&author=T.+T.+Ashburnauthor=K.+B.+Thor&title=Drug+repositioning%3A+identifying+and+developing+new+uses+for+existing+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repositioning: identifying and developing new uses for existing drugs</span></div><div class="casAuthors">Ashburn, Ted T.; Thor, Karl B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">673-683</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biopharmaceutical companies attempting to increase productivity through novel discovery technologies have fallen short of achieving the desired results.  Repositioning existing drugs for new indications could deliver the productivity increases that the industry needs while shifting the locus of prodn. to biotechnol. companies.  More and more companies are scanning the existing pharmacopeia for repositioning candidates, and the no. of repositioning success stories is increasing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLkWl3DnePhrVg90H21EOLACvtfcHk0ljatjvGt_EpSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOhtL8%253D&md5=0145c02423ab81ace6e3b3d92585cccc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd1468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1468%26sid%3Dliteratum%253Aachs%26aulast%3DAshburn%26aufirst%3DT.%2BT.%26aulast%3DThor%26aufirst%3DK.%2BB.%26atitle%3DDrug%2520repositioning%253A%2520identifying%2520and%2520developing%2520new%2520uses%2520for%2520existing%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D673%26epage%3D683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Druker, B.</span><span> </span><span class="NLM_article-title">Imatinib as a paradigm of targeted therapies</span> <span class="citation_source-journal">Adv. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS0065-230X%2804%2991001-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=15327887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVent7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2004&pages=1-30&author=B.+Druker&title=Imatinib+as+a+paradigm+of+targeted+therapies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Imatinib as a paradigm of targeted therapies</span></div><div class="casAuthors">Druker, Brian J.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-30</span>CODEN:
                <span class="NLM_cas:coden">ACRSAJ</span>;
        ISSN:<span class="NLM_cas:issn">0065-230X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Imatinib (Gleevec) exemplifies the successful development of a rationally designed, molecularly targeted therapy for the treatment of a specific cancer.  This article reviews the identification of the BCR-ABL tyrosine kinase as a therapeutic target in chronic myeloid leukemia and the steps in the development of an agent to specifically inactivate this abnormality.  The clin. trials results are reviewed along with a description of resistance mechanisms.  As imatinib also inhibits the tyrosine kinase activity of KIT and the platelet-derived growth factor receptors, the extension of imatinib to malignancies driven by these kinases will be described.  Issues related to clin. trials of molecularly targeted agents are discussed, including patient and dose selection.  Last, the translation of this paradigm to other malignancies is explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorfaGCiPz14bVg90H21EOLACvtfcHk0ljTji2PtMcXsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVent7bP&md5=b9d2fd0bcc46afbf0dbe13e1c27100d8</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS0065-230X%2804%2991001-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0065-230X%252804%252991001-9%26sid%3Dliteratum%253Aachs%26aulast%3DDruker%26aufirst%3DB.%26atitle%3DImatinib%2520as%2520a%2520paradigm%2520of%2520targeted%2520therapies%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D2004%26volume%3D91%26spage%3D1%26epage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Young, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoyt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McWhinnie, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirn, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallarico, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchison, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span> </span><span class="NLM_article-title">Integrating high-content screening and ligand-target prediction to identify mechanism of action</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnchembio.2007.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18066055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVegtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=59-68&author=D.+Youngauthor=A.+Benderauthor=J.+Hoytauthor=E.+McWhinnieauthor=G.+Chirnauthor=C.+Taoauthor=J.+Tallaricoauthor=M.+Labowauthor=J.+Jenkinsauthor=T.+Mitchisonauthor=Y.+Feng&title=Integrating+high-content+screening+and+ligand-target+prediction+to+identify+mechanism+of+action"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating high-content screening and ligand-target prediction to identify mechanism of action</span></div><div class="casAuthors">Young, Daniel W.; Bender, Andreas; Hoyt, Jonathan; McWhinnie, Elizabeth; Chirn, Gung-Wei; Tao, Charles Y.; Tallarico, John A.; Labow, Mark; Jenkins, Jeremy L.; Mitchison, Timothy J.; Feng, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">High-content screening is transforming drug discovery by enabling simultaneous measurement of multiple features of cellular phenotype that are relevant to therapeutic and toxic activities of compds.  High-content screening studies typically generate immense datasets of image-based phenotypic information, and how best to mine relevant phenotypic data is an unsolved challenge.  Here, the authors introduce factor anal. as a data-driven tool for defining cell phenotypes and profiling compd. activities.  This method allows a large data redn. while retaining relevant information, and the data-derived factors used to quantify phenotype have discernable biol. meaning.  The authors used factor anal. of cells stained with fluorescent markers of cell cycle state to profile a compd. library and cluster the hits into seven phenotypic categories.  The authors then compared phenotypic profiles, chem. similarity and predicted protein binding activities of active compds.  By integrating these different descriptors of measured and potential biol. activity, the authors can effectively draw mechanism-of-action inferences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqIzrMvgKTaLVg90H21EOLACvtfcHk0ljTji2PtMcXsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVegtbnE&md5=0456c454ed40d4c52382e5bf786a8346</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2007.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2007.53%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DD.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DHoyt%26aufirst%3DJ.%26aulast%3DMcWhinnie%26aufirst%3DE.%26aulast%3DChirn%26aufirst%3DG.%26aulast%3DTao%26aufirst%3DC.%26aulast%3DTallarico%26aufirst%3DJ.%26aulast%3DLabow%26aufirst%3DM.%26aulast%3DJenkins%26aufirst%3DJ.%26aulast%3DMitchison%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DY.%26atitle%3DIntegrating%2520high-content%2520screening%2520and%2520ligand-target%2520prediction%2520to%2520identify%2520mechanism%2520of%2520action%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D59%26epage%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Wagner, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peck, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramanathan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mootha, V.</span><span> </span><span class="NLM_article-title">Large-scale chemical dissection of mitochondrial function</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=343-351&author=B.+Wagnerauthor=T.+Kitamiauthor=T.+Gilbertauthor=D.+Peckauthor=A.+Ramanathanauthor=S.+Schreiberauthor=T.+Golubauthor=V.+Mootha&title=Large-scale+chemical+dissection+of+mitochondrial+function"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DKitami%26aufirst%3DT.%26aulast%3DGilbert%26aufirst%3DT.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DRamanathan%26aufirst%3DA.%26aulast%3DSchreiber%26aufirst%3DS.%26aulast%3DGolub%26aufirst%3DT.%26aulast%3DMootha%26aufirst%3DV.%26atitle%3DLarge-scale%2520chemical%2520dissection%2520of%2520mitochondrial%2520function%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D343%26epage%3D351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Computational analysis of ligand relationships within target families</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">358</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2Fj.cbpa.2008.01.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18312862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ers7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=352-358&author=J.+Bajorath&title=Computational+analysis+of+ligand+relationships+within+target+families"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Computational analysis of ligand relationships within target families</span></div><div class="casAuthors">Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">352-358</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Computational tools for the large-scale anal. and prediction of ligand-target interactions and the identification of small mols. having different selectivity profiles within target protein families complement research in chem. genetics and chemogenomics.  For computational anal. and design, such tasks require a departure from the traditional focus on single targets, hit identification, and lead optimization.  Recently, studies were reported that profile compds. in silico against arrays of targets or systematically map ligand-target space.  To identify small mol. probes that are suitable for chem. genetics applications, mol. diversity needs to be viewed in a way that partly differs from principles guiding conventional library design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGFA5t6J8v3rVg90H21EOLACvtfcHk0lg1lyKPT-oQbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ers7k%253D&md5=1d645ef300d472939c2d062ebad9f86e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.01.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.01.044%26sid%3Dliteratum%253Aachs%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComputational%2520analysis%2520of%2520ligand%2520relationships%2520within%2520target%2520families%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D352%26epage%3D358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Oprea, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rintoul, M.</span><span> </span><span class="NLM_article-title">Systems chemical biology</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">447</span><span class="NLM_x">–</span> <span class="NLM_lpage">450</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnchembio0807-447" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17637771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvFaqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=447-450&author=T.+Opreaauthor=A.+Tropshaauthor=J.+Faulonauthor=M.+Rintoul&title=Systems+chemical+biology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Systems chemical biology</span></div><div class="casAuthors">Oprea, Tudor I.; Tropsha, Alexander; Faulon, Jean-Loup; Rintoul, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">447-450</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The increasing availability of data related to genes, proteins and their modulation by small mols. has provided a vast amt. of biol. information leading to the emergence of systems biol. and the broad use of simulation tools for data anal.  However, there is a crit. need to develop cheminformatics tools that can integrate chem. knowledge with these biol. databases and simulation approaches, with the goal of creating systems chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlQRRfUwh1ebVg90H21EOLACvtfcHk0lgiNEaEBymcug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvFaqs7Y%253D&md5=d47ca3c6aa344bfab403262e449645f2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnchembio0807-447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio0807-447%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DFaulon%26aufirst%3DJ.%26aulast%3DRintoul%26aufirst%3DM.%26atitle%3DSystems%2520chemical%2520biology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D447%26epage%3D450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Siegel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vieth, M.</span><span> </span><span class="NLM_article-title">Drugs in other drugs: a new look at drugs as fragments</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=71-79&author=M.+Siegelauthor=M.+Vieth&title=Drugs+in+other+drugs%3A+a+new+look+at+drugs+as+fragments"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSiegel%26aufirst%3DM.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DDrugs%2520in%2520other%2520drugs%253A%2520a%2520new%2520look%2520at%2520drugs%2520as%2520fragments%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D71%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Miller, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mochalkin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banotai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunkle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanna, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harwood, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huband, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karnovsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Limberakis, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehrens, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narasimhan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelly, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skerlos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sulavik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">VanderRoest, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stover, C.</span><span> </span><span class="NLM_article-title">A class of selective antibacterials derived from a protein kinase inhibitor pharmacophore</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">1737</span><span class="NLM_x">–</span> <span class="NLM_lpage">1742</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1073%2Fpnas.0811275106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19164768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1737-1742&author=J.+Millerauthor=S.+Dunhamauthor=I.+Mochalkinauthor=C.+Banotaiauthor=M.+Bowmanauthor=S.+Buistauthor=B.+Dunkleauthor=D.+Hannaauthor=H.+Harwoodauthor=M.+Hubandauthor=A.+Karnovskyauthor=M.+Kuhnauthor=C.+Limberakisauthor=J.+Liuauthor=S.+Mehrensauthor=W.+Muellerauthor=L.+Narasimhanauthor=A.+Ogdenauthor=J.+Ohrenauthor=J.+Prasadauthor=J.+Shellyauthor=L.+Skerlosauthor=M.+Sulavikauthor=V.+Thomasauthor=S.+VanderRoestauthor=L.+Wangauthor=Z.+Wangauthor=A.+Whittonauthor=T.+Zhuauthor=C.+Stover&title=A+class+of+selective+antibacterials+derived+from+a+protein+kinase+inhibitor+pharmacophore"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A class of selective antibacterials derived from a protein kinase inhibitor pharmacophore</span></div><div class="casAuthors">Miller, J. Richard; Dunham, Steve; Mochalkin, Igor; Banotai, Craig; Bowman, Matthew; Buist, Susan; Dunkle, Bill; Hanna, Debra; Harwood, James; Huband, Michael D.; Karnovsky, Alla; Kuhn, Michael; Limberakis, Chris; Liu, Jia Y.; Mehrens, Shawn; Mueller, W. Thomas; Narasimhan, Lakshmi; Ogden, Adam; Ohren, Jeff; Vara Prasad, J. V. N.; Shelly, John A.; Skerlos, Laura; Sulavik, Mark; Thomas, V. Hayden; VanderRoest, Steve; Wang, LiAnn; Wang, Zhigang; Whitton, Amy; Zhu, Tong; Stover, C. Kendall</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1737-1742</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the need for novel antibiotic classes to combat bacterial drug resistance increases, the paucity of leads resulting from target-based antibacterial screening of pharmaceutical compd. libraries is of major concern.  One explanation for this lack of success is that antibacterial screening efforts have not leveraged the eukaryotic bias resulting from more extensive chem. efforts targeting eukaryotic gene families such as G protein-coupled receptors and protein kinases.  Consistent with a focus on antibacterial target space resembling these eukaryotic targets, we used whole-cell screening to identify a series of antibacterial pyridopyrimidines derived from a protein kinase inhibitor pharmacophore.  In bacteria, the pyridopyrimidines target the ATP-binding site of biotin carboxylase (BC), which catalyzes the first enzymic step of fatty acid biosynthesis.  These inhibitors are effective in vitro and in vivo against fastidious Gram-neg. pathogens including Haemophilus influenzae.  Although the BC active site has architectural similarity to those of eukaryotic protein kinases, inhibitor binding to the BC ATP-binding site is distinct from the protein kinase-binding mode, such that the inhibitors are selective for bacterial BC.  In summary, the authors have discovered a promising class of potent antibacterials with a previously undescribed mechanism of action.  In consideration of the eukaryotic bias of pharmaceutical libraries, the findings also suggest that pursuit of a novel inhibitor leads for antibacterial targets with active-site structural similarity to known human targets will likely be more fruitful than the traditional focus on unique bacterial target space, particularly when structure-based and computational methodologies are applied to ensure bacterial selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8jpj8EOtRo7Vg90H21EOLACvtfcHk0lgiNEaEBymcug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D&md5=06758ee5befbe847825ec35adb177006</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811275106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811275106%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DDunham%26aufirst%3DS.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DBowman%26aufirst%3DM.%26aulast%3DBuist%26aufirst%3DS.%26aulast%3DDunkle%26aufirst%3DB.%26aulast%3DHanna%26aufirst%3DD.%26aulast%3DHarwood%26aufirst%3DH.%26aulast%3DHuband%26aufirst%3DM.%26aulast%3DKarnovsky%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DMehrens%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DW.%26aulast%3DNarasimhan%26aufirst%3DL.%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DOhren%26aufirst%3DJ.%26aulast%3DPrasad%26aufirst%3DJ.%26aulast%3DShelly%26aufirst%3DJ.%26aulast%3DSkerlos%26aufirst%3DL.%26aulast%3DSulavik%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DV.%26aulast%3DVanderRoest%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWhitton%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DStover%26aufirst%3DC.%26atitle%3DA%2520class%2520of%2520selective%2520antibacterials%2520derived%2520from%2520a%2520protein%2520kinase%2520inhibitor%2520pharmacophore%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D1737%26epage%3D1742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Walsh, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischbach, M. A.</span><span> </span><span class="NLM_article-title">Repurposing libraries of eukaryotic protein kinase inhibitors for antibiotic discovery</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1690</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1689-1690&author=C.+T.+Walshauthor=M.+A.+Fischbach&title=Repurposing+libraries+of+eukaryotic+protein+kinase+inhibitors+for+antibiotic+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%2BT.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26atitle%3DRepurposing%2520libraries%2520of%2520eukaryotic%2520protein%2520kinase%2520inhibitors%2520for%2520antibiotic%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D1689%26epage%3D1690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Keiser, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laggner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abbas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hufeisen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuijer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whaley, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glennon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B.</span><span> </span><span class="NLM_article-title">Predicting new molecular targets for known drugs</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">462</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnature08506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19881490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=175-182&author=M.+Keiserauthor=V.+Setolaauthor=J.+Irwinauthor=C.+Laggnerauthor=A.+Abbasauthor=S.+Hufeisenauthor=N.+Jensenauthor=M.+Kuijerauthor=R.+Matosauthor=T.+Tranauthor=R.+Whaleyauthor=R.+Glennonauthor=J.+Hertauthor=K.+Thomasauthor=D.+Edwardsauthor=B.+Shoichetauthor=B.+Roth&title=Predicting+new+molecular+targets+for+known+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting new molecular targets for known drugs</span></div><div class="casAuthors">Keiser, Michael J.; Setola, Vincent; Irwin, John J.; Laggner, Christian; Abbas, Atheir I.; Hufeisen, Sandra J.; Jensen, Niels H.; Kuijer, Michael B.; Matos, Roberto C.; Tran, Thuy B.; Whaley, Ryan; Glennon, Richard A.; Hert, Jerome; Thomas, Kelan L. H.; Edwards, Douglas D.; Shoichet, Brian K.; Roth, Bryan L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">175-181</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although drugs are intended to be selective, at least some bind to several physiol. targets, explaining side effects and efficacy.  Because many drug-target combinations exist, it would be useful to explore possible interactions computationally.  Here we compared 3665 US Food and Drug Administration (FDA)-approved and investigational drugs against hundreds of targets, defining each target by its ligands.  Chem. similarities between drugs and ligand sets predicted thousands of unanticipated assocns.  Thirty were tested exptl., including the antagonism of the β1 receptor by the transporter inhibitor Prozac, the inhibition of the 5-hydroxytryptamine (5-HT) transporter by the ion channel drug Vadilex, and antagonism of the histamine H4 receptor by the enzyme inhibitor Rescriptor.  Overall, 23 new drug-target assocns. were confirmed, five of which were potent (<100 nM).  The physiol. relevance of one, the drug N,N-dimethyltryptamine (DMT) on serotonergic receptors, was confirmed in a knockout mouse.  The chem. similarity approach is systematic and comprehensive, and may suggest side-effects and new indications for many drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozd-8cCXUskrVg90H21EOLACvtfcHk0ljzgNxa80g3UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlers7vO&md5=974e65f5d5b4dabde1a463c429215546</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnature08506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08506%26sid%3Dliteratum%253Aachs%26aulast%3DKeiser%26aufirst%3DM.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%26aulast%3DLaggner%26aufirst%3DC.%26aulast%3DAbbas%26aufirst%3DA.%26aulast%3DHufeisen%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DN.%26aulast%3DKuijer%26aufirst%3DM.%26aulast%3DMatos%26aufirst%3DR.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DWhaley%26aufirst%3DR.%26aulast%3DGlennon%26aufirst%3DR.%26aulast%3DHert%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DK.%26aulast%3DEdwards%26aufirst%3DD.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DRoth%26aufirst%3DB.%26atitle%3DPredicting%2520new%2520molecular%2520targets%2520for%2520known%2520drugs%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D175%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span> </span><span class="NLM_article-title">A computational approach to finding novel targets for existing drugs</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=1-13&author=Y.+Liauthor=J.+Anauthor=S.+Jones&title=A+computational+approach+to+finding+novel+targets+for+existing+drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DS.%26atitle%3DA%2520computational%2520approach%2520to%2520finding%2520novel%2520targets%2520for%2520existing%2520drugs%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D1%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Cho, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeire, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merkel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucala, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lolis, E.</span><span> </span><span class="NLM_article-title">Drug repositioning and pharmacophore identification in the discovery of hookworm MIF inhibitors</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1101</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2Fj.chembiol.2011.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21944748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1eju7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1089-1101&author=Y.+Choauthor=J.+Vermeireauthor=J.+Merkelauthor=L.+Lengauthor=X.+Duauthor=R.+Bucalaauthor=M.+Cappelloauthor=E.+Lolis&title=Drug+repositioning+and+pharmacophore+identification+in+the+discovery+of+hookworm+MIF+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Repositioning and Pharmacophore Identification in the Discovery of Hookworm MIF Inhibitors</span></div><div class="casAuthors">Cho, Yoonsang; Vermeire, Jon J.; Merkel, Jane S.; Leng, Lin; Du, Xin; Bucala, Richard; Cappello, Michael; Lolis, Elias</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1089-1101</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The screening of bioactive compd. libraries can be an effective approach for repositioning FDA-approved drugs or discovering new pharmacophores.  Hookworms are blood-feeding, intestinal nematode parasites that infect up to 600 million people worldwide.  Vaccination with recombinant Ancylostoma ceylanicum macrophage migration inhibitory factor (rAceMIF) provided partial protection from disease, thus establishing a "proof-of-concept" for targeting AceMIF to prevent or treat infection.  A high-throughput screen (HTS) against rAceMIF identified six AceMIF-specific inhibitors.  A nonsteroidal anti-inflammatory drug (NSAID), sodium meclofenamate, could be tested in an animal model to assess the therapeutic efficacy in treating hookworm disease.  Furosemide, an FDA-approved diuretic, exhibited submicromolar inhibition of rAceMIF tautomerase activity.  Structure-activity relationships of a pharmacophore based on furosemide included one analog that binds similarly to the active site, yet does not inhibit the Na-K-Cl symporter (NKCC1) responsible for diuretic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT30vfvK39a7Vg90H21EOLACvtfcHk0ljzgNxa80g3UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1eju7nL&md5=16e9c33e27c43f6f657c91e23f686114</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DY.%26aulast%3DVermeire%26aufirst%3DJ.%26aulast%3DMerkel%26aufirst%3DJ.%26aulast%3DLeng%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DBucala%26aufirst%3DR.%26aulast%3DCappello%26aufirst%3DM.%26aulast%3DLolis%26aufirst%3DE.%26atitle%3DDrug%2520repositioning%2520and%2520pharmacophore%2520identification%2520in%2520the%2520discovery%2520of%2520hookworm%2520MIF%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1089%26epage%3D1101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jou, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span> </span><span class="NLM_article-title">Fragment-based drug design and drug repositioning using multiple ligand simultaneous docking (MLSD): identifying celecoxib and template compounds as novel inhibitors of signal transducer and activator of transcription 3 (STAT3)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">5592</span><span class="NLM_x">–</span> <span class="NLM_lpage">5596</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm101330h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=5592-5596&author=H.+Liauthor=A.+Liuauthor=Z.+Zhaoauthor=Y.+Xuauthor=J.+Linauthor=D.+Jouauthor=C.+Li&title=Fragment-based+drug+design+and+drug+repositioning+using+multiple+ligand+simultaneous+docking+%28MLSD%29%3A+identifying+celecoxib+and+template+compounds+as+novel+inhibitors+of+signal+transducer+and+activator+of+transcription+3+%28STAT3%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm101330h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101330h%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DJou%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26atitle%3DFragment-based%2520drug%2520design%2520and%2520drug%2520repositioning%2520using%2520multiple%2520ligand%2520simultaneous%2520docking%2520%2528MLSD%2529%253A%2520identifying%2520celecoxib%2520and%2520template%2520compounds%2520as%2520novel%2520inhibitors%2520of%2520signal%2520transducer%2520and%2520activator%2520of%2520transcription%25203%2520%2528STAT3%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D5592%26epage%3D5596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Moriaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adcock, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chanas-Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surgand, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jelloul, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delfaud, F.</span><span> </span><span class="NLM_article-title">Identify drug repurposing candidates by mining the Protein Data Bank</span> <span class="citation_source-journal">Briefings Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">336</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1093%2Fbib%2Fbbr017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21768131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=336-340&author=F.+Moriaudauthor=S.+Richardauthor=S.+Adcockauthor=L.+Chanas-Martinauthor=J.+Surgandauthor=M.+Jelloulauthor=F.+Delfaud&title=Identify+drug+repurposing+candidates+by+mining+the+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Identify drug repurposing candidates by mining the Protein Data Bank</span></div><div class="casAuthors">Moriaud, Fabrice; Richard, Stephane B.; Adcock, Stewart A.; Chanas-Martin, Laetitia; Surgand, Jean-Sebastien; Ben Jelloul, Marouane; Delfaud, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">336-340</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1467-5463</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Predicting off-targets by computational methods is gaining increasing interest in early-stage drug discovery.  Here, we present a computational method based on full 3D comparisons of 3D structures.  When a similar binding site is detected in the Protein Data Bank (PDB) (or any protein structure database), it is possible that the corresponding ligand also binds to that similar site.  On one hand, this target hopping case is probably rare because it requires a high similarity between the binding sites.  On the other hand, it could be a strong rational evidence to highlight possible off-target reactions and possibly a potential undesired side effect.  This target-based drug repurposing can be extended a significant step further with the capability of searching the full surface of all proteins in the PDB, and therefore not relying on pocket detection.  Using this approach, we describe how MED-SuMo reproduces the repurposing of tadalafil from PDE5A to PDE4A and a structure of PDE4A with tadalafil.  Searching for local protein similarities generates more hits than for whole binding site similarities and therefore fragment repurposing is more likely to occur than for drug-sized compds.  In this work, we illustrate that by mining the PDB for proteins sharing similarities with the hinge region of protein kinases.  The exptl. validated examples, biotin carboxylase and synapsin, are retrieved.  Further to fragment repurposing, this approach can be applied to the detection of druggable sites from 3D structures.  This is illustrated with detection of the protein kinase hinge motif in the HIV-RT non-nucleosidic allosteric site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCPSI-y0H9RbVg90H21EOLACvtfcHk0ljTTzYvFUBo0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptl2jsrY%253D&md5=d171b0356ec3b9931d15ff1f7610a5d4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbr017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbr017%26sid%3Dliteratum%253Aachs%26aulast%3DMoriaud%26aufirst%3DF.%26aulast%3DRichard%26aufirst%3DS.%26aulast%3DAdcock%26aufirst%3DS.%26aulast%3DChanas-Martin%26aufirst%3DL.%26aulast%3DSurgand%26aufirst%3DJ.%26aulast%3DJelloul%26aufirst%3DM.%26aulast%3DDelfaud%26aufirst%3DF.%26atitle%3DIdentify%2520drug%2520repurposing%2520candidates%2520by%2520mining%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DBriefings%2520Bioinf.%26date%3D2011%26volume%3D12%26spage%3D336%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Kinnings, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, N</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchmeier, N</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P.</span><span> </span><span class="NLM_article-title">Drug discovery using chemical systems biology: repositioning the safe medicine comtan to treat multi-drug and extensively drug resistant tuberculosis</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1000423</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000423&issue=7&author=S.+Kinningsauthor=N+Liuauthor=N+Buchmeierauthor=P.+Tongeauthor=L.+Xieauthor=P.+Bourne&title=Drug+discovery+using+chemical+systems+biology%3A+repositioning+the+safe+medicine+comtan+to+treat+multi-drug+and+extensively+drug+resistant+tuberculosis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKinnings%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DN%26aulast%3DBuchmeier%26aufirst%3DN%26aulast%3DTonge%26aufirst%3DP.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%26atitle%3DDrug%2520discovery%2520using%2520chemical%2520systems%2520biology%253A%2520repositioning%2520the%2520safe%2520medicine%2520comtan%2520to%2520treat%2520multi-drug%2520and%2520extensively%2520drug%2520resistant%2520tuberculosis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26issue%3D7%26spage%3De1000423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Campillos, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gavin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span> </span><span class="NLM_article-title">Drug target identification using side-effect similarity</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">321</span><span class="NLM_x">, </span> <span class="NLM_fpage">263</span><span class="NLM_x">–</span> <span class="NLM_lpage">266</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscience.1158140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18621671" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2008&pages=263-266&author=M.+Campillosauthor=M.+Kuhnauthor=A.+Gavinauthor=L.+Jensenauthor=P.+Bork&title=Drug+target+identification+using+side-effect+similarity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Target Identification Using Side-Effect Similarity</span></div><div class="casAuthors">Campillos, Monica; Kuhn, Michael; Gavin, Anne-Claude; Jensen, Lars Juhl; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">5886</span>),
    <span class="NLM_cas:pages">263-266</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Targets for drugs have so far been predicted on the basis of mol. or cellular features, for example, by exploiting similarity in chem. structure or in activity across cell lines.  We used phenotypic side-effect similarities to infer whether two drugs share a target.  Applied to 746 marketed drugs, a network of 1018 side effect-driven drug-drug relations became apparent, 261 of which are formed by chem. dissimilar drugs from different therapeutic indications.  We exptl. tested 20 of these unexpected drug-drug relations and validated 13 implied drug-target relations by in vitro binding assays, of which 11 reveal inhibition consts. equal to less than 10 micromolar.  Nine of these were tested and confirmed in cell assays, documenting the feasibility of using phenotypic information to infer mol. interactions and hinting at new uses of marketed drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbiFcs_camL7Vg90H21EOLACvtfcHk0lhe-ubrLWlmpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXot1entbY%253D&md5=d8fe131adeb39750cb97ee4e147de355</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fscience.1158140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1158140%26sid%3Dliteratum%253Aachs%26aulast%3DCampillos%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DGavin%26aufirst%3DA.%26aulast%3DJensen%26aufirst%3DL.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DDrug%2520target%2520identification%2520using%2520side-effect%2520similarity%26jtitle%3DScience%26date%3D2008%26volume%3D321%26spage%3D263%26epage%3D266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Dudley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirota, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenoy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roedder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarwal, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasricha, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butte, A.</span><span> </span><span class="NLM_article-title">Computational repositioning of the anticonvulsant topiramate for inflammatory bowel disease</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">96</span><span class="NLM_x">) </span> <span class="NLM_fpage">96ra76</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscitranslmed.3002648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21849664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=96ra76&issue=96&author=J.+Dudleyauthor=M.+Sirotaauthor=M.+Shenoyauthor=R.+Paiauthor=S.+Roedderauthor=A.+Chiangauthor=A.+Morganauthor=M.+Sarwalauthor=P.+Pasrichaauthor=A.+Butte&title=Computational+repositioning+of+the+anticonvulsant+topiramate+for+inflammatory+bowel+disease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Computational repositioning of the anticonvulsant topiramate for inflammatory bowel disease</span></div><div class="casAuthors">Dudley, Joel T.; Sirota, Marina; Shenoy, Mohan; Pai, Reetesh K.; Roedder, Silke; Chiang, Annie P.; Morgan, Alex A.; Sarwal, Minnie M.; Pasricha, Pankaj Jay; Butte, Atul J.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">96</span>),
    <span class="NLM_cas:pages">96ra76</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Inflammatory bowel disease (IBD) is a chronic inflammatory disorder of the gastrointestinal tract for which there are few safe and effective therapeutic options for long-term treatment and disease maintenance.  Here, we applied a computational approach to discover new drug therapies for IBD in silico, using publicly available mol. data reporting gene expression in IBD samples and 164 small-mol. drug compds.  Among the top compds. predicted to be therapeutic for IBD by our approach were prednisolone, a corticosteroid used to treat IBD, and topiramate, an anticonvulsant drug not previously described to have efficacy for IBD or any related disorders of inflammation or the gastrointestinal tract.  Using a trinitrobenzenesulfonic acid (TNBS)-induced rodent model of IBD, we exptl. validated our topiramate prediction in vivo.  Oral administration of topiramate significantly reduced gross pathol. signs and microscopic damage in primary affected colon tissue in the TNBS-induced rodent model of IBD.  These findings suggest that topiramate might serve as a therapeutic option for IBD in humans and support the use of public mol. data and computational approaches to discover new therapeutic options for disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmF-DmOnq3dLVg90H21EOLACvtfcHk0ligL8BfCwotNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3J&md5=42dd418de306427e93735180e40288a9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3002648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3002648%26sid%3Dliteratum%253Aachs%26aulast%3DDudley%26aufirst%3DJ.%26aulast%3DSirota%26aufirst%3DM.%26aulast%3DShenoy%26aufirst%3DM.%26aulast%3DPai%26aufirst%3DR.%26aulast%3DRoedder%26aufirst%3DS.%26aulast%3DChiang%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DSarwal%26aufirst%3DM.%26aulast%3DPasricha%26aufirst%3DP.%26aulast%3DButte%26aufirst%3DA.%26atitle%3DComputational%2520repositioning%2520of%2520the%2520anticonvulsant%2520topiramate%2520for%2520inflammatory%2520bowel%2520disease%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26issue%3D96%26spage%3D96ra76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Zhao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iskar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeller, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noort, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bork, P.</span><span> </span><span class="NLM_article-title">Prediction of drug combinations by integrating molecular and pharmacological data</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">e1002323</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002323&issue=12&author=X.+Zhaoauthor=M.+Iskarauthor=G.+Zellerauthor=M.+Kuhnauthor=V.+Noortauthor=P.+Bork&title=Prediction+of+drug+combinations+by+integrating+molecular+and+pharmacological+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DIskar%26aufirst%3DM.%26aulast%3DZeller%26aufirst%3DG.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DNoort%26aufirst%3DV.%26aulast%3DBork%26aufirst%3DP.%26atitle%3DPrediction%2520of%2520drug%2520combinations%2520by%2520integrating%2520molecular%2520and%2520pharmacological%2520data%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26issue%3D12%26spage%3De1002323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Gottlieb, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruppin, E</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharan, R.</span><span> </span><span class="NLM_article-title">PREDICT: a method for inferring novel drug indications with application to personalized medicine</span> <span class="citation_source-journal">Mol. Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">496</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=496&author=A.+Gottliebauthor=G.+Steinauthor=E+Ruppinauthor=R.+Sharan&title=PREDICT%3A+a+method+for+inferring+novel+drug+indications+with+application+to+personalized+medicine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DG.%26aulast%3DRuppin%26aufirst%3DE%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DPREDICT%253A%2520a%2520method%2520for%2520inferring%2520novel%2520drug%2520indications%2520with%2520application%2520to%2520personalized%2520medicine%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Sirota, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dudley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweet-Cordero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sage, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butte, A.</span><span> </span><span class="NLM_article-title">Discovery and preclinical validation of drug indications using compendia of public gene expression data</span> <span class="citation_source-journal">Sci. Transl. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">96</span><span class="NLM_x">) </span> <span class="NLM_fpage">96ra77</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1126%2Fscitranslmed.3001318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21849665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=96ra77&issue=96&author=M.+Sirotaauthor=J.+Dudleyauthor=J.+Kimauthor=A.+Chiangauthor=A.+Morganauthor=A.+Sweet-Corderoauthor=J.+Sageauthor=A.+Butte&title=Discovery+and+preclinical+validation+of+drug+indications+using+compendia+of+public+gene+expression+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and preclinical validation of drug indications using compendia of public gene expression data</span></div><div class="casAuthors">Sirota, Marina; Dudley, Joel T.; Kim, Jeewon; Chiang, Annie P.; Morgan, Alex A.; Sweet-Cordero, Alejandro; Sage, Julien; Butte, Atul J.</div><div class="citationInfo"><span class="NLM_cas:title">Science Translational Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">96</span>),
    <span class="NLM_cas:pages">96ra77</span>CODEN:
                <span class="NLM_cas:coden">STMCBQ</span>;
        ISSN:<span class="NLM_cas:issn">1946-6242</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The application of established drug compds. to new therapeutic indications, known as drug repositioning, offers several advantages over traditional drug development, including reduced development costs and shorter paths to approval.  Recent approaches to drug repositioning use high-throughput exptl. approaches to assess a compd.'s potential therapeutic qualities.  Here, we present a systematic computational approach to predict novel therapeutic indications on the basis of comprehensive testing of mol. signatures in drug-disease pairs.  We integrated gene expression measurements from 100 diseases and gene expression measurements on 164 drug compds., yielding predicted therapeutic potentials for these drugs.  We recovered many known drug and disease relationships using computationally derived therapeutic potentials and also predict many new indications for these 164 drugs.  We exptl. validated a prediction for the antiulcer drug cimetidine as a candidate therapeutic in the treatment of lung adenocarcinoma, and demonstrate its efficacy both in vitro and in vivo using mouse xenograft models.  This computational method provides a systematic approach for repositioning established drugs to treat a wide range of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVAoZxuzFvS7Vg90H21EOLACvtfcHk0ljO4l-xCunj-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOisL3K&md5=73a95f1af7e6c2202eacb5e0f95eda9d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.3001318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.3001318%26sid%3Dliteratum%253Aachs%26aulast%3DSirota%26aufirst%3DM.%26aulast%3DDudley%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DChiang%26aufirst%3DA.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DSweet-Cordero%26aufirst%3DA.%26aulast%3DSage%26aufirst%3DJ.%26aulast%3DButte%26aufirst%3DA.%26atitle%3DDiscovery%2520and%2520preclinical%2520validation%2520of%2520drug%2520indications%2520using%2520compendia%2520of%2520public%2520gene%2520expression%2520data%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2011%26volume%3D3%26issue%3D96%26spage%3D96ra77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J.</span><span> </span><span class="NLM_article-title">Docking screens: right for the right reasons?</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">770</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.2174%2F156802609789207091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19754393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=755-770&author=P.+Kolbauthor=J.+Irwin&title=Docking+screens%3A+right+for+the+right+reasons%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Docking screens: right for the right reasons?</span></div><div class="casAuthors">Kolb, Peter; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">755-770</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Whereas docking screens have emerged as the most practical way to use protein structure for ligand discovery, an inconsistent track record raises questions about how well docking actually works.  In its favor, a growing no. of publications report the successful discovery of new ligands, often supported by exptl. affinity data and controls for artifacts.  Few reports, however, actually test the underlying structural hypotheses that docking makes.  To be successful and not just lucky, prospective docking must not only rank a true ligand among the top scoring compds., it must also correctly orient the ligand so the score it receives is biophys. sound.  If the correct binding pose is not predicted, a skeptic might well infer that the discovery was serendipitous.  Surveying over 15 years of the docking literature, we were surprised to discover how rarely sufficient evidence is presented to establish whether docking actually worked for the right reasons.  The paucity of exptl. tests of theor. predicted poses undermines confidence in a technique that has otherwise become widely accepted.  Of course, solving a crystal structure is not always possible, and even when it is, it can be a lot of work, and is not readily accessible to all groups.  Even when a structure can be detd., investigators may prefer to gloss over an erroneous structural prediction to better focus on their discovery.  Still, the absence of a direct test of theory by expt. is a loss for method developers seeking to understand and improve docking methods.  We hope this review will motivate investigators to solve structures and compare them with their predictions whenever possible, to advance the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXLK_ZZmntsrVg90H21EOLACvtfcHk0lh2K3MBF8i2qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSmt7vP&md5=3e6c0073795fae06c0d5a1175af8e8e5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2174%2F156802609789207091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802609789207091%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DIrwin%26aufirst%3DJ.%26atitle%3DDocking%2520screens%253A%2520right%2520for%2520the%2520right%2520reasons%253F%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2009%26volume%3D9%26spage%3D755%26epage%3D770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Taylor, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewsbury, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J.</span><span> </span><span class="NLM_article-title">A review of protein–small molecule docking methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=151-166&author=R.+Taylorauthor=P.+Jewsburyauthor=J.+Essex&title=A+review+of+protein%E2%80%93small+molecule+docking+methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DJewsbury%26aufirst%3DP.%26aulast%3DEssex%26aufirst%3DJ.%26atitle%3DA%2520review%2520of%2520protein%25E2%2580%2593small%2520molecule%2520docking%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D151%26epage%3D166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Abagyan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span> </span><span class="NLM_article-title">High-throughput docking for lead generation</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">375</span><span class="NLM_x">–</span> <span class="NLM_lpage">382</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS1367-5931%2800%2900217-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=11470599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=375-382&issue=4&author=R.+Abagyanauthor=M.+Totrov&title=High-throughput+docking+for+lead+generation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput docking for lead generation</span></div><div class="casAuthors">Abagyan, R.; Totrov, M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-382</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with refs.  Recent improvements in flexible docking technol. may lead to a bigger role for computational methods in lead discovery.  Although fast and accurate computational prediction of binding affinities for an arbitrary mol. is still beyond the limits of current methods, the docking and screening procedures can select small sets of likely lead candidates from large libraries of either com. or synthetically available compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9_FjizU9XdbVg90H21EOLACvtfcHk0lh2K3MBF8i2qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltlyqs7c%253D&md5=c9bfaa493becdadb4386522865b9e7fb</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2FS1367-5931%2800%2900217-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1367-5931%252800%252900217-9%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26atitle%3DHigh-throughput%2520docking%2520for%2520lead%2520generation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26issue%3D4%26spage%3D375%26epage%3D382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5912</span><span class="NLM_x">–</span> <span class="NLM_lpage">5931</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0liLCKJ3cY3zQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span> </span><span class="NLM_article-title">Prediction of protein–ligand interactions. Docking and scoring: successes and gaps</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5851</span><span class="NLM_x">–</span> <span class="NLM_lpage">5855</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060999m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5851-5855&author=A.+R.+Leachauthor=B.+K.+Shoichetauthor=C.+E.+Peishoff&title=Prediction+of+protein%E2%80%93ligand+interactions.+Docking+and+scoring%3A+successes+and+gaps"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm060999m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060999m%26sid%3Dliteratum%253Aachs%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26atitle%3DPrediction%2520of%2520protein%25E2%2580%2593ligand%2520interactions.%2520Docking%2520and%2520scoring%253A%2520successes%2520and%2520gaps%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5851%26epage%3D5855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Coupez, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. A.</span><span> </span><span class="NLM_article-title">Docking and scoring: theoretically easy, practically impossible?</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">2995</span><span class="NLM_x">–</span> <span class="NLM_lpage">3003</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.2174%2F092986706778521797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17073642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVamsrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=2995-3003&author=B.+Coupezauthor=R.+A.+Lewis&title=Docking+and+scoring%3A+theoretically+easy%2C+practically+impossible%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Docking and scoring - theoretically easy, practically impossible?</span></div><div class="casAuthors">Coupez, B.; Lewis, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">2995-3003</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based Drug Design (SBDD) is an essential part of the modern medicinal chem., and has led to the acceleration of many projects, and even to drugs on the market.  Programs that perform docking and scoring of ligands to receptors are powerful tools in the drug designer's armoury that enhance the process of SBDD.  They are even deployed on the desktop of many bench chemists.  It is timely to review the state of the art, to understand how good our docking programs are, and what are the issues.  In this review we would like to provide a guide around the reliable aspects of docking and scoring and the assocd. pitfalls aiming at an audience of medicinal chemists rather than modelers.  For convenience, we will divide the review into two parts: docking and scoring.  Docking concerns the prepn. of the receptor and the ligand(s), the sampling of conformational space and stereochem. (if appropriate).  Scoring concerns the evaluation of all of the ligand-receptor poses generated by docking.  The two processes are not truly independent, and this will be discussed here in detail.  The prepn. of the receptor and ligand(s) before docking requires great care.  For the receptor, issues of protonation, tautomerization and hydration are key, and we will discuss current approaches to these issues.  Even more important is the degree of sampling: can the algorithms reproduce what is obsd. exptl..  If they can, are the scoring algorithms good enough to recognize this pose as the best.  Do the scores correlate with obsd. binding affinity.  How does local knowledge of the target (for example hinge-binding to a kinase) affect the accuracy of the predictions.  We will review the key findings from several evaluation studies and present conclusions about when and how to interpret and trust the results of docking and scoring.  Finally, we will present an outline of some of the latest developments in the area of scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-ow8pkDVcLbVg90H21EOLACvtfcHk0liLCKJ3cY3zQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVamsrfF&md5=3ea8bd5273435881362163ca49f11d9e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.2174%2F092986706778521797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986706778521797%26sid%3Dliteratum%253Aachs%26aulast%3DCoupez%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DR.%2BA.%26atitle%3DDocking%2520and%2520scoring%253A%2520theoretically%2520easy%252C%2520practically%2520impossible%253F%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2006%26volume%3D13%26spage%3D2995%26epage%3D3003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Kroemer, R. T.</span><span> </span><span class="NLM_article-title">Structure-based drug design: docking and scoring</span> <span class="citation_source-journal">Curr. Protein Pept. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.2174%2F138920307781369382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17696866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=312-328&issue=4&author=R.+T.+Kroemer&title=Structure-based+drug+design%3A+docking+and+scoring"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design: docking and scoring</span></div><div class="casAuthors">Kroemer, Romano T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-328</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review gives an introduction into ligand - receptor docking and illustrates the basic underlying concepts.  An overview of different approaches and algorithms is provided.  Although the application of docking and scoring has led to some remarkable successes, there are still some major challenges ahead, which are outlined here as well.  Approaches to address some of these challenges and the latest developments in the area are presented.  Some aspects of the assessment of docking program performance are discussed.  A no. of successful applications of structure-based virtual screening are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbXWKN0_d9g7Vg90H21EOLACvtfcHk0ljF1Awo22Qu5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D&md5=5bfb61ca9c4b405a98f50ddc2bb00ed7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138920307781369382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369382%26sid%3Dliteratum%253Aachs%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26atitle%3DStructure-based%2520drug%2520design%253A%2520docking%2520and%2520scoring%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26issue%3D4%26spage%3D312%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Kahraman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laskowski, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J.</span><span> </span><span class="NLM_article-title">Shape variation in protein binding pockets and their ligands</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">368</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">301</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2Fj.jmb.2007.01.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17337005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVeqt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2007&pages=283-301&author=A.+Kahramanauthor=R.+Morrisauthor=R.+Laskowskiauthor=J.+Thornton&title=Shape+variation+in+protein+binding+pockets+and+their+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Shape Variation in Protein Binding Pockets and their Ligands</span></div><div class="casAuthors">Kahraman, Abdullah; Morris, Richard J.; Laskowski, Roman A.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">283-301</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A common assumption about the shape of protein binding pockets is that they are related to the shape of the small ligand mols. that can bind there.  But to what extent is that assumption true Here we use a recently developed shape matching method to compare the shapes of protein binding pockets to the shapes of their ligands.  We find that pockets binding the same ligand show greater variation in their shapes than can be accounted for by the conformational variability of the ligand.  This suggests that geometrical complementarity in general is not sufficient to drive mol. recognition.  Nevertheless, we show when considering only shape and size that a significant proportion of the recognition power of a binding pocket for its ligand resides in its shape.  Addnl., we observe a "buffer zone" or a region of free space between the ligand and protein, which results in binding pockets being on av. three times larger than the ligand that they bind.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoInd-ruAMm17Vg90H21EOLACvtfcHk0ljF1Awo22Qu5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVeqt78%253D&md5=3e1766868247fcc0baa99e55a21f8885</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.01.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.01.086%26sid%3Dliteratum%253Aachs%26aulast%3DKahraman%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DR.%26aulast%3DLaskowski%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%26atitle%3DShape%2520variation%2520in%2520protein%2520binding%2520pockets%2520and%2520their%2520ligands%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D368%26spage%3D283%26epage%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Patel, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciatto, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahna, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajebhosale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arkus, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schieren, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jessell, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, L.</span><span> </span><span class="NLM_article-title">Type II cadherin ectodomain structures: implications for classical cadherin specificity</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1268</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=1255-1268&issue=6&author=S.+D.+Patelauthor=C.+Ciattoauthor=C.+P.+Chenauthor=F.+Bahnaauthor=M.+Rajebhosaleauthor=N.+Arkusauthor=I.+Schierenauthor=T.+M.+Jessellauthor=B.+Honigauthor=S.+R.+Priceauthor=L.+Shapiro&title=Type+II+cadherin+ectodomain+structures%3A+implications+for+classical+cadherin+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DS.%2BD.%26aulast%3DCiatto%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DC.%2BP.%26aulast%3DBahna%26aufirst%3DF.%26aulast%3DRajebhosale%26aufirst%3DM.%26aulast%3DArkus%26aufirst%3DN.%26aulast%3DSchieren%26aufirst%3DI.%26aulast%3DJessell%26aufirst%3DT.%2BM.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DPrice%26aufirst%3DS.%2BR.%26aulast%3DShapiro%26aufirst%3DL.%26atitle%3DType%2520II%2520cadherin%2520ectodomain%2520structures%253A%2520implications%2520for%2520classical%2520cadherin%2520specificity%26jtitle%3DCell%26date%3D2006%26volume%3D124%26issue%3D6%26spage%3D1255%26epage%3D1268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span>Sc-PDB Home Page. <a href="http://cheminfo.u-strasbg.fr:8080/scPDB/2011/db_search/ acceuil.jsp&#xA;?uid=3807447192798425088" class="extLink">http://cheminfo.u-strasbg.fr:8080/scPDB/2011/db_search/ acceuil.jsp ?uid=3807447192798425088</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Sc-PDB+Home+Page.+http%3A%2F%2Fcheminfo.u-strasbg.fr%3A8080%2FscPDB%2F2011%2Fdb_search%2F+acceuil.jsp+%3Fuid%3D3807447192798425088."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span> <span class="citation_source-book">SYBYL X</span>; <span class="NLM_publisher-name">Tripos International</span>: <span class="NLM_publisher-loc">St. Louis, MO, U.S.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SYBYL+X%3B+Tripos+International%3A+St.+Louis%2C+MO%2C+U.S."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DSYBYL%2520X%26pub%3DTripos%2520International" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+Shoichetauthor=J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0ljF1Awo22Qu5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DIrwin%26aufirst%3DJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Karaman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herrgard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Treiber, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallant, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atteridge, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edeen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faraoni, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floyd, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockhart, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milanov, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrison, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pallares, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pritchard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wodicka, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zarrinkar, P.</span><span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&issue=1&author=M.+Karamanauthor=S.+Herrgardauthor=D.+Treiberauthor=P.+Gallantauthor=C.+Atteridgeauthor=B.+Campbellauthor=K.+Chanauthor=P.+Ciceriauthor=M.+Davisauthor=P.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+Huntauthor=D.+Lockhartauthor=Z.+Milanovauthor=M.+Morrisonauthor=G.+Pallaresauthor=H.+Patelauthor=S.+Pritchardauthor=L.+Wodickaauthor=P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lilLdOTDtu3mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DB.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%26aulast%3DEdeen%26aufirst%3DP.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%26aulast%3DLockhart%26aufirst%3DD.%26aulast%3DMilanov%26aufirst%3DZ.%26aulast%3DMorrison%26aufirst%3DM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%26aulast%3DZarrinkar%26aufirst%3DP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26issue%3D1%26spage%3D127%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Murzin, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chothia, C.</span><span> </span><span class="NLM_article-title">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">247</span><span class="NLM_x">, </span> <span class="NLM_fpage">536</span><span class="NLM_x">–</span> <span class="NLM_lpage">540</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1016%2FS0022-2836%2805%2980134-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=7723011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=247&publication_year=1995&pages=536-540&author=A.+G.+Murzinauthor=S.+E.+Brennerauthor=T.+Hubbardauthor=C.+Chothia&title=SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">SCOP: a structural classification of proteins database for the investigation of sequences and structures</span></div><div class="casAuthors">Murzin, Alexey G.; Brenner, Steven E.; Hubbard, Tim; Chothia, Cyrus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">247</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">536-40</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">To facilitate understanding of, and access to, the information available for protein structures, we have constructed the Structural Classification of Proteins (SCOP) database.  This database provides a detailed and comprehensive description of the structural and evolutionary relationships of the proteins of known structure.  It also provides for each entry links to coordinates, images of the structure, interactive viewers, sequence data and literature refs.  Two search facilitates are available.  The homol. search permits users to enter a sequence and obtain a list of any structures to which it has significant levels of sequence similarity.  The key word search finds, for a word entered by the user, matches from both the text of the SCOP database and the headers of Brookhaven Protein Databank structure files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-9_Oh4CusCbVg90H21EOLACvtfcHk0lilLdOTDtu3mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXltVGgsr4%253D&md5=65de3f2464aa75ba7efabbd183507913</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980134-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980134-2%26sid%3Dliteratum%253Aachs%26aulast%3DMurzin%26aufirst%3DA.%2BG.%26aulast%3DBrenner%26aufirst%3DS.%2BE.%26aulast%3DHubbard%26aufirst%3DT.%26aulast%3DChothia%26aufirst%3DC.%26atitle%3DSCOP%253A%2520a%2520structural%2520classification%2520of%2520proteins%2520database%2520for%2520the%2520investigation%2520of%2520sequences%2520and%2520structures%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1995%26volume%3D247%26spage%3D536%26epage%3D540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Cho, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katz, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chain, B. M.</span><span> </span><span class="NLM_article-title">Staurosporine induces rapid homotypic intercellular adhesion of U937 cells via multiple kinase activation</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">140</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1038%2Fsj.bjp.0705436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12970105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotFOmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2003&pages=269-276&author=J.+Y.+Choauthor=D.+R.+Katzauthor=B.+M.+Chain&title=Staurosporine+induces+rapid+homotypic+intercellular+adhesion+of+U937+cells+via+multiple+kinase+activation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Staurosporine induces rapid homotypic intercellular adhesion of U937 cells via multiple kinase activation</span></div><div class="casAuthors">Cho, Jae Youl; Katz, David R.; Chain, Benjamin M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-276</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">1 Staurosporine is a broad-specificity kinase inhibitor, which has acted as lead compd. for the development of some novel cytotoxic compds. for treatment of cancer.  This study investigates the unexpected observation that staurosporine can also induce homotypic cellular aggregation.  2 In this study, staurosporine is shown to activate rapid homotypic aggregation of U937 cells, at concns. below those required to induce cell death.  This activity is a particular feature of staurosporine, and is not shared by a no. of other kinase inhibitors.  The pro-aggregating activity of staurosporine is inhibited by deoxyglucose, cytochalasin B and colchicine.  Staurosporine-induced aggregation can be distinguished from that induced by the phorbol 12-myristate 13-acetate by faster kinetics and insensitivity to cycloheximide.  Staurosporine induces translocation of conventional and novel, but not atypical isoforms of protein kinase C (PKC).  Aggregation induced by staurosporine is inhibited by a no. of inhibitors of PKC isoforms, and by inhibitors of protein tyrosine kinases.  Staurosporine also induces rapid phosphorylation of ERK and p38, and inhibitors of both these enzymes block aggregation.  Staurosporine induces dis-regulated activation of multiple kinase signaling pathways in U937 cells, and the combined activity of several of these pathways is essential for the induction of aggregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokwzrAAO9guLVg90H21EOLACvtfcHk0liP9Q_xaRoh-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotFOmsbg%253D&md5=56377325fbb9c5947392ae3729223bda</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705436%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DJ.%2BY.%26aulast%3DKatz%26aufirst%3DD.%2BR.%26aulast%3DChain%26aufirst%3DB.%2BM.%26atitle%3DStaurosporine%2520induces%2520rapid%2520homotypic%2520intercellular%2520adhesion%2520of%2520U937%2520cells%2520via%2520multiple%2520kinase%2520activation%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D140%26spage%3D269%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Martarelli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompei, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazzoni, G.</span><span> </span><span class="NLM_article-title">Mebendazole inhibits growth of human adrenocortical carcinoma cell lines implanted in nude mice</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">809</span><span class="NLM_x">–</span> <span class="NLM_lpage">817</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00280-007-0538-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=17581752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitleru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2008&pages=809-817&issue=5&author=D.+Martarelliauthor=P.+Pompeiauthor=C.+Baldiauthor=G.+Mazzoni&title=Mebendazole+inhibits+growth+of+human+adrenocortical+carcinoma+cell+lines+implanted+in+nude+mice"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Mebendazole inhibits growth of human adrenocortical carcinoma cell lines implanted in nude mice</span></div><div class="casAuthors">Martarelli, Daniele; Pompei, Pierluigi; Baldi, Caterina; Mazzoni, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">809-817</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Adrenocortical carcinoma is a rare tumor of the adrenal gland which requires new therapeutic approaches as its early diagnosis is difficult and prognosis poor despite therapies used.  Recently, mebendazole has been proved to be effective against different cancers.  The aim of our study was to evaluate whether mebendazole may result therapeutically useful in the treatment of human adrenocortical carcinoma.  We analyzed the effect of mebendazole on human adrenocortical carcinoma cells in vitro and after implantation in nude mice.  In order to clarify mechanisms of mebendazole action, metastases formation, apoptosis and angiogenesis were also investigated.  Mebendazole significantly inhibited cancer cells growth, both in vitro and in vivo, the effects being due to the induction of apoptosis.  Moreover, mebendazole inhibited invasion and migration of cancer cells in vitro, and metastases formation in vivo.  Overall, these data suggest that treatment with mebendazole, also in combination with std. therapies, could provide a new protocol for the inhibition of adrenocortical carcinoma growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdwQ0oo1WtCLVg90H21EOLACvtfcHk0liP9Q_xaRoh-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitleru7c%253D&md5=723cfa4b786f346aa6a8789ee6a9b30e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2Fs00280-007-0538-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-007-0538-0%26sid%3Dliteratum%253Aachs%26aulast%3DMartarelli%26aufirst%3DD.%26aulast%3DPompei%26aufirst%3DP.%26aulast%3DBaldi%26aufirst%3DC.%26aulast%3DMazzoni%26aufirst%3DG.%26atitle%3DMebendazole%2520inhibits%2520growth%2520of%2520human%2520adrenocortical%2520carcinoma%2520cell%2520lines%2520implanted%2520in%2520nude%2520mice%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2008%26volume%3D61%26issue%3D5%26spage%3D809%26epage%3D817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Bai, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staedtke, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aprhys, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallia, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riggins, G. J.</span><span> </span><span class="NLM_article-title">Antiparasitic mebendazole shows survival benefit in 2 preclinical models of glioblastoma multiforme</span> <span class="citation_source-journal">Neuro-Oncology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">974</span><span class="NLM_x">–</span> <span class="NLM_lpage">982</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=974-982&issue=9&author=R.+Y.+Baiauthor=V.+Staedtkeauthor=C.+M.+Aprhysauthor=G.+L.+Galliaauthor=G.+J.+Riggins&title=Antiparasitic+mebendazole+shows+survival+benefit+in+2+preclinical+models+of+glioblastoma+multiforme"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DR.%2BY.%26aulast%3DStaedtke%26aufirst%3DV.%26aulast%3DAprhys%26aufirst%3DC.%2BM.%26aulast%3DGallia%26aufirst%3DG.%2BL.%26aulast%3DRiggins%26aufirst%3DG.%2BJ.%26atitle%3DAntiparasitic%2520mebendazole%2520shows%2520survival%2520benefit%2520in%25202%2520preclinical%2520models%2520of%2520glioblastoma%2520multiforme%26jtitle%3DNeuro-Oncology%26date%3D2011%26volume%3D13%26issue%3D9%26spage%3D974%26epage%3D982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Montesano, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vassalli, P.</span><span> </span><span class="NLM_article-title">In vitro rapid organization of endothelial cells into capillary-like networks is promoted by collagen matrices</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1648</span><span class="NLM_x">–</span> <span class="NLM_lpage">1652</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1983&pages=1648-1652&author=R.+Montesanoauthor=L.+Orciauthor=P.+Vassalli&title=In+vitro+rapid+organization+of+endothelial+cells+into+capillary-like+networks+is+promoted+by+collagen+matrices"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMontesano%26aufirst%3DR.%26aulast%3DOrci%26aufirst%3DL.%26aulast%3DVassalli%26aufirst%3DP.%26atitle%3DIn%2520vitro%2520rapid%2520organization%2520of%2520endothelial%2520cells%2520into%2520capillary-like%2520networks%2520is%2520promoted%2520by%2520collagen%2520matrices%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D1983%26volume%3D97%26spage%3D1648%26epage%3D1652" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Vailhe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittet, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feige, J.</span><span> </span><span class="NLM_article-title">In vitro models of vasculogenesis and angiogenesis</span> <span class="citation_source-journal">Lab. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">439</span><span class="NLM_x">–</span> <span class="NLM_lpage">452</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2001&pages=439-452&author=B.+Vailheauthor=D.+Vittetauthor=J.+Feige&title=In+vitro+models+of+vasculogenesis+and+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVailhe%26aufirst%3DB.%26aulast%3DVittet%26aufirst%3DD.%26aulast%3DFeige%26aufirst%3DJ.%26atitle%3DIn%2520vitro%2520models%2520of%2520vasculogenesis%2520and%2520angiogenesis%26jtitle%3DLab.%2520Invest.%26date%3D2001%26volume%3D81%26spage%3D439%26epage%3D452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Pourgholami, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khachigian, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahmy, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, D. L.</span><span> </span><span class="NLM_article-title">Albendazole inhibits endothelial cell migration, tube formation, vasopermeability, VEGF receptor-2 expression and suppresses retinal neovascularization in ROP model of angiogenesis</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">397</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">729</span><span class="NLM_x">–</span> <span class="NLM_lpage">734</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=2010&pages=729-734&issue=4&author=M.+H.+Pourgholamiauthor=L.+M.+Khachigianauthor=R.+G.+Fahmyauthor=S.+Badarauthor=L.+Wangauthor=S.+W.+Chuauthor=D.+L.+Morris&title=Albendazole+inhibits+endothelial+cell+migration%2C+tube+formation%2C+vasopermeability%2C+VEGF+receptor-2+expression+and+suppresses+retinal+neovascularization+in+ROP+model+of+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPourgholami%26aufirst%3DM.%2BH.%26aulast%3DKhachigian%26aufirst%3DL.%2BM.%26aulast%3DFahmy%26aufirst%3DR.%2BG.%26aulast%3DBadar%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DS.%2BW.%26aulast%3DMorris%26aufirst%3DD.%2BL.%26atitle%3DAlbendazole%2520inhibits%2520endothelial%2520cell%2520migration%252C%2520tube%2520formation%252C%2520vasopermeability%252C%2520VEGF%2520receptor-2%2520expression%2520and%2520suppresses%2520retinal%2520neovascularization%2520in%2520ROP%2520model%2520of%2520angiogenesis%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2010%26volume%3D397%26issue%3D4%26spage%3D729%26epage%3D734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Sade, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuncay, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cimen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severcan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerjee, S.</span><span> </span><span class="NLM_article-title">Celecoxib reduces fluidity and decreases metastatic potential of colon cancer cell lines irrespective of COX-2 expression</span> <span class="citation_source-journal">Biosci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">44</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1042%2FBSR20100149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21401528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1yksr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=35-44&issue=1&author=A.+Sadeauthor=S.+Tuncayauthor=I.+Cimenauthor=F.+Severcanauthor=S.+Banerjee&title=Celecoxib+reduces+fluidity+and+decreases+metastatic+potential+of+colon+cancer+cell+lines+irrespective+of+COX-2+expression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib reduces fluidity and decreases metastatic potential of colon cancer cell lines irrespective of COX-2 expression</span></div><div class="casAuthors">Sade, Asli; Tuncay, Seda; Cimen, Ismail; Severcan, Feride; Banerjee, Sreeparna</div><div class="citationInfo"><span class="NLM_cas:title">Bioscience Reports</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-44, S35/1-S35/2</span>CODEN:
                <span class="NLM_cas:coden">BRPTDT</span>;
        ISSN:<span class="NLM_cas:issn">0144-8463</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">CLX (celecoxib), a selective COX-2 (cyclo-oxygenase-2) inhibitor, has numerous pleiotropic effects on the body that may be independent of its COX-2 inhibitory activity.  The cancer chemopreventive ability of CLX, particularly in CRC (colorectal cancer), has been shown in epidemiol. studies.  Here we have, for the first time, examd. the biophys. effects of CLX on the cellular membranes of COX-2 expressing (HT29) and COX-2 non-expressing (SW620) cell lines using ATR-FTIR (attenuated total reflectance-Fourier transform IR) spectroscopy and SL-ESR (spin label-ESR) spectroscopy.  Our results show that CLX treatment decreased lipid fluidity in the cancer cell lines irresp. of COX-2 expression status.  As metastatic cells have higher membrane fluidity, we examd. the effect of CLX on the metastatic potential of these cells.  The CLX treatment efficiently decreased the proliferation, anchorage-independent growth, ability to close a scratch wound and migration and invasion of the CRC cell lines through Matrigel.  We propose that one of the ways by which CLX exerts its anti-tumorigenic effects is via alterations in cellular membrane fluidity which has a notable impact on the cells' metastatic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrezYIlBr4DMrVg90H21EOLACvtfcHk0lj6hNwwDa9eWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1yksr3I&md5=56859d5340ac8f8a5c45e893325292f9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1042%2FBSR20100149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20100149%26sid%3Dliteratum%253Aachs%26aulast%3DSade%26aufirst%3DA.%26aulast%3DTuncay%26aufirst%3DS.%26aulast%3DCimen%26aufirst%3DI.%26aulast%3DSevercan%26aufirst%3DF.%26aulast%3DBanerjee%26aufirst%3DS.%26atitle%3DCelecoxib%2520reduces%2520fluidity%2520and%2520decreases%2520metastatic%2520potential%2520of%2520colon%2520cancer%2520cell%2520lines%2520irrespective%2520of%2520COX-2%2520expression%26jtitle%3DBiosci.%2520Rep.%26date%3D2012%26volume%3D32%26issue%3D1%26spage%3D35%26epage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Zweers, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Boer, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Roon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bijlsma, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lafeber, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mastbergen, S. C.</span><span> </span><span class="NLM_article-title">Celecoxib: considerations regarding its potential disease-modifying properties in osteoarthritis</span> <span class="citation_source-journal">Arthritis Res. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">239</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1186%2Far3437" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21955617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlSnsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=239&issue=5&author=M.+C.+Zweersauthor=T.+N.+de+Boerauthor=J.+van+Roonauthor=J.+W.+Bijlsmaauthor=F.+P.+Lafeberauthor=S.+C.+Mastbergen&title=Celecoxib%3A+considerations+regarding+its+potential+disease-modifying+properties+in+osteoarthritis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib: considerations regarding its potential disease-modifying properties in osteoarthritis</span></div><div class="casAuthors">Zweers, Manon C.; de Boer, Tineke N.; van Roon, Joel; Bijlsma, Johannes W. J.; Lafeber, Floris P. J. G.; Mastbergen, Simon C.</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis Research & Therapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">239</span>CODEN:
                <span class="NLM_cas:coden">ARTRCV</span>;
        ISSN:<span class="NLM_cas:issn">1478-6362</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Osteoarthritis is a degenerative joint disease and is characterized by progressive loss of articular cartilage, subchondral bone sclerosis, osteophyte formation, and synovial inflammation, causing substantial phys. disability, impaired quality of life, and significant health care utilization.  Traditionally, non-steroidal anti-infl ammatory drugs (NSAIDs), non-steroidal anti-inflammatory drugs (NSAIDs) including selective COX-2 inhibitors have been used.  Besides its anti-inflammatory properties, evidence is accumulating that celecoxib, one of the selective COX-2 inhibitors, has addnl. disease modifying effects.  Celecoxib was shown to affect all structures involved in OA pathogenesis: cartilage, bone, and synovium.  As well as COX-2 inhibition, evidence exists that celecoxib also modulates COX-2-independent signal transduction pathways.  These findings raise the question whether celecoxib and potentially other coxibs are more than just anti-inflammatory and analgesic drugs.  Can celecoxib be considered a disease modifying osteoarthritic drug (DMOAD).  In this review, these direct effects of celecoxib on cartilage, bone, and synoviocytes in OA treatment are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqD8ttKZiQi7Vg90H21EOLACvtfcHk0lj6hNwwDa9eWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlSnsL7J&md5=b38073dc4888b2502e2f160c58c42b26</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1186%2Far3437&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Far3437%26sid%3Dliteratum%253Aachs%26aulast%3DZweers%26aufirst%3DM.%2BC.%26aulast%3Dde%2BBoer%26aufirst%3DT.%2BN.%26aulast%3Dvan%2BRoon%26aufirst%3DJ.%26aulast%3DBijlsma%26aufirst%3DJ.%2BW.%26aulast%3DLafeber%26aufirst%3DF.%2BP.%26aulast%3DMastbergen%26aufirst%3DS.%2BC.%26atitle%3DCelecoxib%253A%2520considerations%2520regarding%2520its%2520potential%2520disease-modifying%2520properties%2520in%2520osteoarthritis%26jtitle%3DArthritis%2520Res.%2520Ther.%26date%3D2011%26volume%3D13%26issue%3D5%26spage%3D239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Cervello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachvarov, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cusimano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sardina, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azzolina, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lampiasi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannitrapani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCubrey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montalto, G.</span><span> </span><span class="NLM_article-title">COX-2-dependent and COX-2-independent mode of action of celecoxib in human liver cancer cells</span> <span class="citation_source-journal">OMICS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">383</span><span class="NLM_x">–</span> <span class="NLM_lpage">392</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2011&pages=383-392&issue=6&author=M.+Cervelloauthor=D.+Bachvarovauthor=A.+Cusimanoauthor=F.+Sardinaauthor=A.+Azzolinaauthor=N.+Lampiasiauthor=L.+Giannitrapaniauthor=J.+McCubreyauthor=G.+Montalto&title=COX-2-dependent+and+COX-2-independent+mode+of+action+of+celecoxib+in+human+liver+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCervello%26aufirst%3DM.%26aulast%3DBachvarov%26aufirst%3DD.%26aulast%3DCusimano%26aufirst%3DA.%26aulast%3DSardina%26aufirst%3DF.%26aulast%3DAzzolina%26aufirst%3DA.%26aulast%3DLampiasi%26aufirst%3DN.%26aulast%3DGiannitrapani%26aufirst%3DL.%26aulast%3DMcCubrey%26aufirst%3DJ.%26aulast%3DMontalto%26aufirst%3DG.%26atitle%3DCOX-2-dependent%2520and%2520COX-2-independent%2520mode%2520of%2520action%2520of%2520celecoxib%2520in%2520human%2520liver%2520cancer%2520cells%26jtitle%3DOMICS%26date%3D2011%26volume%3D15%26issue%3D6%26spage%3D383%26epage%3D392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Sareddy, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geeviman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramulu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babu, P. P.</span><span> </span><span class="NLM_article-title">The nonsteroidal anti-inflammatory drug celecoxib suppresses the growth and induces apoptosis of human glioblastoma cells via the NF-kappaB pathway</span> <span class="citation_source-journal">J. Neuro-Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs11060-011-0662-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=21847707" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2011&pages=99-109&issue=1&author=G.+R.+Sareddyauthor=K.+Geevimanauthor=C.+Ramuluauthor=P.+P.+Babu&title=The+nonsteroidal+anti-inflammatory+drug+celecoxib+suppresses+the+growth+and+induces+apoptosis+of+human+glioblastoma+cells+via+the+NF-kappaB+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs11060-011-0662-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11060-011-0662-x%26sid%3Dliteratum%253Aachs%26aulast%3DSareddy%26aufirst%3DG.%2BR.%26aulast%3DGeeviman%26aufirst%3DK.%26aulast%3DRamulu%26aufirst%3DC.%26aulast%3DBabu%26aufirst%3DP.%2BP.%26atitle%3DThe%2520nonsteroidal%2520anti-inflammatory%2520drug%2520celecoxib%2520suppresses%2520the%2520growth%2520and%2520induces%2520apoptosis%2520of%2520human%2520glioblastoma%2520cells%2520via%2520the%2520NF-kappaB%2520pathway%26jtitle%3DJ.%2520Neuro-Oncol.%26date%3D2011%26volume%3D106%26issue%3D1%26spage%3D99%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Schonthal, A. H.</span><span> </span><span class="NLM_article-title">Exploiting cyclooxygenase-(in)dependent properties of COX-2 inhibitors for malignant glioma therapy</span> <span class="citation_source-journal">Anti-Cancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">450</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=450-461&issue=6&author=A.+H.+Schonthal&title=Exploiting+cyclooxygenase-%28in%29dependent+properties+of+COX-2+inhibitors+for+malignant+glioma+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchonthal%26aufirst%3DA.%2BH.%26atitle%3DExploiting%2520cyclooxygenase-%2528in%2529dependent%2520properties%2520of%2520COX-2%2520inhibitors%2520for%2520malignant%2520glioma%2520therapy%26jtitle%3DAnti-Cancer%2520Agents%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26issue%3D6%26spage%3D450%26epage%3D461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Bastos-Pereira, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugarini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira-Christoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avila, T. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teixeira, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pires Ado, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muscará, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cadena, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donatti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cristina da Silva de Assis., H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acco, A.</span><span> </span><span class="NLM_article-title">Celecoxib prevents tumor growth in an animal model by a COX-2 independent mechanism</span> <span class="citation_source-journal">Cancer Chemother. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">267</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00280-009-1031-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=19506872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsL7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2010&pages=267-276&issue=2&author=A.+L.+Bastos-Pereiraauthor=D.+Lugariniauthor=A.+Oliveira-Christoffauthor=T.+V.+Avilaauthor=S.+Teixeiraauthor=R.+Pires+Adoauthor=M.+N.+Muscar%C3%A1author=S.+M.+Cadenaauthor=L.+Donattiauthor=H.+Cristina+da+Silva+de+Assis.author=A.+Acco&title=Celecoxib+prevents+tumor+growth+in+an+animal+model+by+a+COX-2+independent+mechanism"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib prevents tumor growth in an animal model by a COX-2 independent mechanism</span></div><div class="casAuthors">Bastos-Pereira, Amanda Leite; Lugarini, Daiana; Oliveira-Christoff, Adriana; Avila, Thiago Vinicius; Teixeira, Simone; do Rocio Andrade Pires, Amanda; Muscara, Marcelo Nicolas; Cadena, Silvia Maria Suter Correia; Donatti, Lucelia; da Silva de Assis, Helena Cristina; Acco, Alexandra</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">267-276</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Nonsteroidal antiinflammatory drugs (NSAIDs) have been shown to reduce cell growth in several tumors.  Among these possible antineoplastic drugs are cyclooxygenase-2 (COX-2)-selective drugs, such as celecoxib, in which antitumoral mechanisms were evaluated in rats bearing Walker-256 (W256) tumor.  W256 carcinosarcoma cells were inoculated s.c. (107 cells/rat) in rats submitted to treatment with celecoxib (25 mg/kg-1) or vehicle for 14 days.  Tumor growth, body-wt. gain, and survival data were evaluated.  The mechanisms, such as COX-2 expression and activity, oxidative stress, by means of enzymes and lipoperoxidn. levels, and apoptosis mediators were also investigated.  A redn. in tumor growth and an increased wt. gain were obsd.  Celecoxib provided a higher incidence of survival compared with the control group.  Cellular effects are probably COX-2 independent, because neither enzyme expression nor its activity, measured by tumoral PGE2, showed significant difference between groups.  It is probable that this antitumor action is dependent on an apoptotic way, which has been evaluated by the expression of the antiapoptotic protein Bcl-xL, in addn. to the cellular changes obsd. by electronic microscopy.  Celecoxib has also a possible involvement with redox homeostasis, because its administration caused significant changes in the activity of oxidative enzymes, such as catalase and superoxide dismutase.  These results confirm the antitumor effects of celecoxib in W256 cancer model, contributing to elucidating its antitumoral mechanism and corroborating scientific literature about its effect on other types of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIBTD9P_P2orVg90H21EOLACvtfcHk0ljvxBs-EqFNGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVKnsL7I&md5=e6d10ea0b2f935b075240ad10a87c032</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2Fs00280-009-1031-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-009-1031-8%26sid%3Dliteratum%253Aachs%26aulast%3DBastos-Pereira%26aufirst%3DA.%2BL.%26aulast%3DLugarini%26aufirst%3DD.%26aulast%3DOliveira-Christoff%26aufirst%3DA.%26aulast%3DAvila%26aufirst%3DT.%2BV.%26aulast%3DTeixeira%26aufirst%3DS.%26aulast%3DPires%2BAdo%26aufirst%3DR.%26aulast%3DMuscar%25C3%25A1%26aufirst%3DM.%2BN.%26aulast%3DCadena%26aufirst%3DS.%2BM.%26aulast%3DDonatti%26aufirst%3DL.%26aulast%3DCristina%2Bda%2BSilva%2Bde%2BAssis.%26aufirst%3DH.%26aulast%3DAcco%26aufirst%3DA.%26atitle%3DCelecoxib%2520prevents%2520tumor%2520growth%2520in%2520an%2520animal%2520model%2520by%2520a%2520COX-2%2520independent%2520mechanism%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2010%26volume%3D65%26issue%3D2%26spage%3D267%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Tsutsumi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiramitsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishitani, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyoshi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitaori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span> </span><span class="NLM_article-title">Celecoxib inhibits production of MMP and NO via down-regulation of NF-kappaB and JNK in a PGE2 independent manner in human articular chondrocytes</span> <span class="citation_source-journal">Rheumatol. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">736</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00296-007-0511-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18080123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=727-736&issue=8&author=R.+Tsutsumiauthor=H.+Itoauthor=T.+Hiramitsuauthor=K.+Nishitaniauthor=M.+Akiyoshiauthor=T.+Kitaoriauthor=T.+Yasudaauthor=T.+Nakamura&title=Celecoxib+inhibits+production+of+MMP+and+NO+via+down-regulation+of+NF-kappaB+and+JNK+in+a+PGE2+independent+manner+in+human+articular+chondrocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib inhibits production of MMP and NO via down-regulation of NF-κB and JNK in a PGE2 independent manner in human articular chondrocytes</span></div><div class="casAuthors">Tsutsumi, R.; Ito, H.; Hiramitsu, T.; Nishitani, K.; Akiyoshi, M.; Kitaori, T.; Yasuda, T.; Nakamura, T.</div><div class="citationInfo"><span class="NLM_cas:title">Rheumatology International</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">727-736</span>CODEN:
                <span class="NLM_cas:coden">RHINDE</span>;
        ISSN:<span class="NLM_cas:issn">0172-8172</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">The purpose of this study was to examine the effects of celecoxib on matrix metalloproteinases (MMP-1 and MMP-3), nitric oxide (NO), and the phosphorylation of nuclear factor-κB (NF-κB) and three mitogen-activated protein kinases (MAPKs), (p38, JNK and ERK) in human articular chondrocytes from normal, osteoarthritis, and rheumatoid arthritis cartilages.  Celecoxib at 100 nM reduced the IL-1β-induced productions of MMP-1, MMP-3, iNOS, and NO, whereas indomethacin at 100 nM showed no effect.  The addnl. stimulation of prostaglandin E2 (PGE2) failed to restore those productions, while the prodn. of PGE2 were reduced by 1 and 10 μM but not 100 nM of celecoxib.  The inhibitors of NF-κB, JNK and p38, but not ERK, decreased IL-1β-enhanced MMP-1, MMP-3 and NO prodn., resp., and 100 nM celecoxib down-regulated the phosphorylation of NF-κB and JNK but has no effect on either p38 or ERK.  Celecoxib has inhibitory effects on MMP-1, MMP-3 and NO productions, suggesting the protective roles directly on articular chondrocytes.  Despite the COX-2 selectivity, celecoxib affects those productions via not PGE2 but NF-κB and JNK MAPK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR7m6RH7cRp7Vg90H21EOLACvtfcHk0ljvxBs-EqFNGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D&md5=d5c48bf9b3fadbb5f018dc5de40a7f54</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs00296-007-0511-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00296-007-0511-6%26sid%3Dliteratum%253Aachs%26aulast%3DTsutsumi%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DH.%26aulast%3DHiramitsu%26aufirst%3DT.%26aulast%3DNishitani%26aufirst%3DK.%26aulast%3DAkiyoshi%26aufirst%3DM.%26aulast%3DKitaori%26aufirst%3DT.%26aulast%3DYasuda%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26atitle%3DCelecoxib%2520inhibits%2520production%2520of%2520MMP%2520and%2520NO%2520via%2520down-regulation%2520of%2520NF-kappaB%2520and%2520JNK%2520in%2520a%2520PGE2%2520independent%2520manner%2520in%2520human%2520articular%2520chondrocytes%26jtitle%3DRheumatol.%2520Int.%26date%3D2008%26volume%3D28%26issue%3D8%26spage%3D727%26epage%3D736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Miyamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyake, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusunoki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamura, T.</span><span> </span><span class="NLM_article-title">Selective COX-2 inhibitor celecoxib prevents experimental autoimmune encephalomyelitis through COX-2-independent pathway</span> <span class="citation_source-journal">Brain</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x"> (</span><span class="NLM_issue">Part 8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">1992</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2006&pages=1984-1992&issue=Part+8&author=K.+Miyamotoauthor=S.+Miyakeauthor=M.+Mizunoauthor=N.+Okaauthor=S.+Kusunokiauthor=T.+Yamamura&title=Selective+COX-2+inhibitor+celecoxib+prevents+experimental+autoimmune+encephalomyelitis+through+COX-2-independent+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DS.%26aulast%3DMizuno%26aufirst%3DM.%26aulast%3DOka%26aufirst%3DN.%26aulast%3DKusunoki%26aufirst%3DS.%26aulast%3DYamamura%26aufirst%3DT.%26atitle%3DSelective%2520COX-2%2520inhibitor%2520celecoxib%2520prevents%2520experimental%2520autoimmune%2520encephalomyelitis%2520through%2520COX-2-independent%2520pathway%26jtitle%3DBrain%26date%3D2006%26volume%3D129%26issue%3DPart%25208%26spage%3D1984%26epage%3D1992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Kusunoki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, S.</span><span> </span><span class="NLM_article-title">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span> <span class="citation_source-journal">Arthritis Rheum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">3159</span><span class="NLM_x">–</span> <span class="NLM_lpage">3167</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1002%2Fart.10692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12483719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=3159-3167&issue=12&author=N.+Kusunokiauthor=R.+Yamazakiauthor=S.+Kawai&title=Induction+of+apoptosis+in+rheumatoid+synovial+fibroblasts+by+celecoxib%2C+but+not+by+other+selective+cyclooxygenase+2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span></div><div class="casAuthors">Kusunoki, Natsuko; Yamazaki, Ryuta; Kawai, Shinichi</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3159-3167</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Selective cyclooxygenase 2 (COX-2) inhibitors are now being used as antiinflammatory agents that cause fewer gastrointestinal complications, compared with other antiinflammatory drugs, in patients with rheumatoid arthritis (RA).  This study was undertaken to investigate whether selective COX-2 inhibitors could induce apoptosis of RA synovial fibroblasts (RASFs).  RASFs were exposed to selective COX-2 inhibitors, i.e., celecoxib, etodolac, meloxicam, nimesulide, N-[2-(cyclohexyloxyl)-4-nitrophenyl]-methanesulfonamide, and rofecoxib, under various conditions.  Cell proliferation and cell viability were assessed by incorporation of 5-bromo-2'-deoxyuridine and by the 2-(4-iodophenyl)-3-(4-nitrophenyl)-5-(2,4-disulfophenyl)-2H-tetrazolium monosodium salt assay, resp.  Apoptosis was detected by identifying DNA fragmentation.  Activation of peroxisome proliferator-activated receptor γ (PPARγ) was measured by the luciferase reporter gene assay with a PPAR response element-driven luciferase reporter plasmid and a PPARγ expression plasmid.  Celecoxib strongly inhibited the proliferation of RASFs, whereas other selective COX-2 inhibitors had little or no effect.  In addn., celecoxib reduced the viability of RASFs by induction of apoptosis, in a concn.-dependent manner.  This action was abolished by addn. of caspase inhibitors.  Interleukin-1β had a weak enhancing effect on celecoxib-induced apoptosis in RASFs.  In contrast, other selective COX-2 inhibitors at concns. up to 100 μM did not induce apoptosis of RASFs.  Indomethacin, a nonselective COX inhibitor, activated PPARγ transcription, while celecoxib did not.  Celecoxib suppressed the proliferation of RASFs by COX-2-independent and PPARγ-independent induction of apoptosis.  Although the mechanism involved remains unclear, celecoxib may have not only antiinflammatory activity, but also a disease-modifying effect on rheumatoid synovial proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7T4nlS6hfVLVg90H21EOLACvtfcHk0liTe2YlrfnjuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D&md5=8406153dd9e73542120554bc602dea0b</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fart.10692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.10692%26sid%3Dliteratum%253Aachs%26aulast%3DKusunoki%26aufirst%3DN.%26aulast%3DYamazaki%26aufirst%3DR.%26aulast%3DKawai%26aufirst%3DS.%26atitle%3DInduction%2520of%2520apoptosis%2520in%2520rheumatoid%2520synovial%2520fibroblasts%2520by%2520celecoxib%252C%2520but%2520not%2520by%2520other%2520selective%2520cyclooxygenase%25202%2520inhibitors%26jtitle%3DArthritis%2520Rheum.%26date%3D2002%26volume%3D46%26issue%3D12%26spage%3D3159%26epage%3D3167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Tsutsumi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiramitsu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishitani, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyoshi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kitaori, T</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yasuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span> </span><span class="NLM_article-title">Celecoxib inhibits production of MMP and NO via down-regulation of NF-kappaB and JNK in a PGE2 independent manner in human articular chondrocytes</span> <span class="citation_source-journal">Rheumatol. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">736</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1007%2Fs00296-007-0511-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=18080123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=727-736&issue=8&author=R.+Tsutsumiauthor=H.+Itoauthor=T.+Hiramitsuauthor=K.+Nishitaniauthor=M.+Akiyoshiauthor=T+Kitaoriauthor=T.+Yasudaauthor=T.+Nakamura&title=Celecoxib+inhibits+production+of+MMP+and+NO+via+down-regulation+of+NF-kappaB+and+JNK+in+a+PGE2+independent+manner+in+human+articular+chondrocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Celecoxib inhibits production of MMP and NO via down-regulation of NF-κB and JNK in a PGE2 independent manner in human articular chondrocytes</span></div><div class="casAuthors">Tsutsumi, R.; Ito, H.; Hiramitsu, T.; Nishitani, K.; Akiyoshi, M.; Kitaori, T.; Yasuda, T.; Nakamura, T.</div><div class="citationInfo"><span class="NLM_cas:title">Rheumatology International</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">727-736</span>CODEN:
                <span class="NLM_cas:coden">RHINDE</span>;
        ISSN:<span class="NLM_cas:issn">0172-8172</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">The purpose of this study was to examine the effects of celecoxib on matrix metalloproteinases (MMP-1 and MMP-3), nitric oxide (NO), and the phosphorylation of nuclear factor-κB (NF-κB) and three mitogen-activated protein kinases (MAPKs), (p38, JNK and ERK) in human articular chondrocytes from normal, osteoarthritis, and rheumatoid arthritis cartilages.  Celecoxib at 100 nM reduced the IL-1β-induced productions of MMP-1, MMP-3, iNOS, and NO, whereas indomethacin at 100 nM showed no effect.  The addnl. stimulation of prostaglandin E2 (PGE2) failed to restore those productions, while the prodn. of PGE2 were reduced by 1 and 10 μM but not 100 nM of celecoxib.  The inhibitors of NF-κB, JNK and p38, but not ERK, decreased IL-1β-enhanced MMP-1, MMP-3 and NO prodn., resp., and 100 nM celecoxib down-regulated the phosphorylation of NF-κB and JNK but has no effect on either p38 or ERK.  Celecoxib has inhibitory effects on MMP-1, MMP-3 and NO productions, suggesting the protective roles directly on articular chondrocytes.  Despite the COX-2 selectivity, celecoxib affects those productions via not PGE2 but NF-κB and JNK MAPK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR7m6RH7cRp7Vg90H21EOLACvtfcHk0liTe2YlrfnjuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyjsr0%253D&md5=d5c48bf9b3fadbb5f018dc5de40a7f54</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs00296-007-0511-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00296-007-0511-6%26sid%3Dliteratum%253Aachs%26aulast%3DTsutsumi%26aufirst%3DR.%26aulast%3DIto%26aufirst%3DH.%26aulast%3DHiramitsu%26aufirst%3DT.%26aulast%3DNishitani%26aufirst%3DK.%26aulast%3DAkiyoshi%26aufirst%3DM.%26aulast%3DKitaori%26aufirst%3DT%26aulast%3DYasuda%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DT.%26atitle%3DCelecoxib%2520inhibits%2520production%2520of%2520MMP%2520and%2520NO%2520via%2520down-regulation%2520of%2520NF-kappaB%2520and%2520JNK%2520in%2520a%2520PGE2%2520independent%2520manner%2520in%2520human%2520articular%2520chondrocytes%26jtitle%3DRheumatol.%2520Int.%26date%3D2008%26volume%3D28%26issue%3D8%26spage%3D727%26epage%3D736" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Miyamoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyake, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oka, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kusunoki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamamura, T.</span><span> </span><span class="NLM_article-title">Selective COX-2 inhibitor celecoxib prevents experimental autoimmune encephalomyelitis through COX-2-independent pathway</span> <span class="citation_source-journal">Brain</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x"> (</span><span class="NLM_issue">Part 8</span><span class="NLM_x">) </span> <span class="NLM_fpage">1984</span><span class="NLM_x">–</span> <span class="NLM_lpage">1992</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2006&pages=1984-1992&issue=Part+8&author=K.+Miyamotoauthor=S.+Miyakeauthor=M.+Mizunoauthor=N.+Okaauthor=S.+Kusunokiauthor=T.+Yamamura&title=Selective+COX-2+inhibitor+celecoxib+prevents+experimental+autoimmune+encephalomyelitis+through+COX-2-independent+pathway"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DS.%26aulast%3DMizuno%26aufirst%3DM.%26aulast%3DOka%26aufirst%3DN.%26aulast%3DKusunoki%26aufirst%3DS.%26aulast%3DYamamura%26aufirst%3DT.%26atitle%3DSelective%2520COX-2%2520inhibitor%2520celecoxib%2520prevents%2520experimental%2520autoimmune%2520encephalomyelitis%2520through%2520COX-2-independent%2520pathway%26jtitle%3DBrain%26date%3D2006%26volume%3D129%26issue%3DPart%25208%26spage%3D1984%26epage%3D1992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Kusunoki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, S.</span><span> </span><span class="NLM_article-title">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span> <span class="citation_source-journal">Arthritis Rheum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">3159</span><span class="NLM_x">–</span> <span class="NLM_lpage">3167</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm300576q&amp;key=10.1002%2Fart.10692" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm300576q&amp;key=12483719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm300576q&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=3159-3167&issue=12&author=N.+Kusunokiauthor=R.+Yamazakiauthor=S.+Kawai&title=Induction+of+apoptosis+in+rheumatoid+synovial+fibroblasts+by+celecoxib%2C+but+not+by+other+selective+cyclooxygenase+2+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of apoptosis in rheumatoid synovial fibroblasts by celecoxib, but not by other selective cyclooxygenase 2 inhibitors</span></div><div class="casAuthors">Kusunoki, Natsuko; Yamazaki, Ryuta; Kawai, Shinichi</div><div class="citationInfo"><span class="NLM_cas:title">Arthritis & Rheumatism</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3159-3167</span>CODEN:
                <span class="NLM_cas:coden">ARHEAW</span>;
        ISSN:<span class="NLM_cas:issn">0004-3591</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Selective cyclooxygenase 2 (COX-2) inhibitors are now being used as antiinflammatory agents that cause fewer gastrointestinal complications, compared with other antiinflammatory drugs, in patients with rheumatoid arthritis (RA).  This study was undertaken to investigate whether selective COX-2 inhibitors could induce apoptosis of RA synovial fibroblasts (RASFs).  RASFs were exposed to selective COX-2 inhibitors, i.e., celecoxib, etodolac, meloxicam, nimesulide, N-[2-(cyclohexyloxyl)-4-nitrophenyl]-methanesulfonamide, and rofecoxib, under various conditions.  Cell proliferation and cell viability were assessed by incorporation of 5-bromo-2'-deoxyuridine and by the 2-(4-iodophenyl)-3-(4-nitrophenyl)-5-(2,4-disulfophenyl)-2H-tetrazolium monosodium salt assay, resp.  Apoptosis was detected by identifying DNA fragmentation.  Activation of peroxisome proliferator-activated receptor γ (PPARγ) was measured by the luciferase reporter gene assay with a PPAR response element-driven luciferase reporter plasmid and a PPARγ expression plasmid.  Celecoxib strongly inhibited the proliferation of RASFs, whereas other selective COX-2 inhibitors had little or no effect.  In addn., celecoxib reduced the viability of RASFs by induction of apoptosis, in a concn.-dependent manner.  This action was abolished by addn. of caspase inhibitors.  Interleukin-1β had a weak enhancing effect on celecoxib-induced apoptosis in RASFs.  In contrast, other selective COX-2 inhibitors at concns. up to 100 μM did not induce apoptosis of RASFs.  Indomethacin, a nonselective COX inhibitor, activated PPARγ transcription, while celecoxib did not.  Celecoxib suppressed the proliferation of RASFs by COX-2-independent and PPARγ-independent induction of apoptosis.  Although the mechanism involved remains unclear, celecoxib may have not only antiinflammatory activity, but also a disease-modifying effect on rheumatoid synovial proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7T4nlS6hfVLVg90H21EOLACvtfcHk0lgz_n6kVfL1bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslGisw%253D%253D&md5=8406153dd9e73542120554bc602dea0b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fart.10692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fart.10692%26sid%3Dliteratum%253Aachs%26aulast%3DKusunoki%26aufirst%3DN.%26aulast%3DYamazaki%26aufirst%3DR.%26aulast%3DKawai%26aufirst%3DS.%26atitle%3DInduction%2520of%2520apoptosis%2520in%2520rheumatoid%2520synovial%2520fibroblasts%2520by%2520celecoxib%252C%2520but%2520not%2520by%2520other%2520selective%2520cyclooxygenase%25202%2520inhibitors%26jtitle%3DArthritis%2520Rheum.%26date%3D2002%26volume%3D46%26issue%3D12%26spage%3D3159%26epage%3D3167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ERD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ERD','PDB','3ERD'); return false;">PDB: 3ERD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KNU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KNU','PDB','1KNU'); return false;">PDB: 1KNU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NYX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NYX','PDB','1NYX'); return false;">PDB: 1NYX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS8','PDB','3CS8'); return false;">PDB: 3CS8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FUR','PDB','3FUR'); return false;">PDB: 3FUR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K8S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3K8S','PDB','3K8S'); return false;">PDB: 3K8S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P2H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P2H','PDB','2P2H'); return false;">PDB: 2P2H</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i46"><a href="/doi/suppl/10.1021/jm300576q">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_64011"></div></div></div></div></div><hr /></hr><p class="last">PCA plot, table of PDB codes, and table of raw score values.  This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300576q/suppl_file/jm300576q_si_001.pdf">jm300576q_si_001.pdf (297.56 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm300576q&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-15%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm300576q%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm300576q" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799899b18bd23e0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
