
<html lang="en"     class="pb-page"  data-request-id="2f2048ea-d773-48db-80d8-e31f8133c8cd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-18;article:article:10.1021/acs.jmedchem.0c00728;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Characterization of Aminobenzylphenols as Protein Disulfide Isomerase Inhibitors in Glioblastoma Cell Lines" /></meta><meta name="dc.Creator" content="Andrea  Shergalis" /></meta><meta name="dc.Creator" content="Ding  Xue" /></meta><meta name="dc.Creator" content="Fatma Z.  Gharbia" /></meta><meta name="dc.Creator" content="Hannah  Driks" /></meta><meta name="dc.Creator" content="Binita  Shrestha" /></meta><meta name="dc.Creator" content="Amina  Tanweer" /></meta><meta name="dc.Creator" content="Kirin  Cromer" /></meta><meta name="dc.Creator" content="Mats  Ljungman" /></meta><meta name="dc.Creator" content="Nouri  Neamati" /></meta><meta name="dc.Description" content="Disulfide bond formation is a critical post-translational modification of newly synthesized polypeptides in the oxidizing environment of the endoplasmic reticulum and is mediated by protein disulfi..." /></meta><meta name="Description" content="Disulfide bond formation is a critical post-translational modification of newly synthesized polypeptides in the oxidizing environment of the endoplasmic reticulum and is mediated by protein disulfi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 24, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00728" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00728" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00728" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00728" /></link>
        
    
    

<title>Characterization of Aminobenzylphenols as Protein Disulfide Isomerase Inhibitors in Glioblastoma Cell Lines | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00728" /></meta><meta property="og:title" content="Characterization of Aminobenzylphenols as Protein Disulfide Isomerase Inhibitors in Glioblastoma Cell Lines" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0028.jpeg" /></meta><meta property="og:description" content="Disulfide bond formation is a critical post-translational modification of newly synthesized polypeptides in the oxidizing environment of the endoplasmic reticulum and is mediated by protein disulfide isomerase (PDIA1). In this study, we report a series of α-aminobenzylphenol analogues as potent PDI inhibitors. The lead compound, AS15, is a covalent nanomolar inhibitor of PDI, and the combination of AS15 analogues with glutathione synthesis inhibitor buthionine sulfoximine (BSO) leads to synergistic cell growth inhibition. Using nascent RNA sequencing, we show that an AS15 analogue triggers the unfolded protein response in glioblastoma cells. A BODIPY-labeled analogue binds proteins including PDIA1, suggesting that the compounds are cell-permeable and reach the intended target. Taken together, these findings demonstrate an extensive biochemical characterization of a novel series of highly potent reactive small molecules that covalently bind to PDI." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00728"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00728">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00728&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00728&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00728&amp;href=/doi/10.1021/acs.jmedchem.0c00728" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 18</span><span class="cit-fg-pageRange">, 10263-10286</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/18" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00567" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00778" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Characterization of Aminobenzylphenols as Protein Disulfide Isomerase Inhibitors in Glioblastoma Cell Lines</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Andrea Shergalis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Shergalis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Shergalis">Andrea Shergalis</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1155-1583" title="Orcid link">http://orcid.org/0000-0002-1155-1583</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ding Xue</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ding Xue</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ding++Xue">Ding Xue</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fatma Z. Gharbia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fatma Z. Gharbia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fatma+Z.++Gharbia">Fatma Z. Gharbia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hannah Driks</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hannah Driks</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hannah++Driks">Hannah Driks</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Binita Shrestha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Binita Shrestha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Binita++Shrestha">Binita Shrestha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Amina Tanweer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amina Tanweer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amina++Tanweer">Amina Tanweer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kirin Cromer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kirin Cromer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kirin++Cromer">Kirin Cromer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mats Ljungman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mats Ljungman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Radiation Oncology, University of Michigan Medical School and Rogel Cancer Center, School of Public Health, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mats++Ljungman">Mats Ljungman</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nouri Neamati</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nouri Neamati</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#5e303b3f333f2a371e2b33373d36703b3a2b"><span class="__cf_email__" data-cfemail="83ede6e2eee2f7eac3f6eeeae0ebade6e7f6">[email protected]</span></a>. Phone: 734-647-2732.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nouri++Neamati">Nouri Neamati</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3291-7131" title="Orcid link">http://orcid.org/0000-0003-3291-7131</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00728&amp;href=/doi/10.1021%2Facs.jmedchem.0c00728" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 18</span><span class="cit-pageRange">, 10263–10286</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 24, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 April 2020</li><li><span class="item_label"><b>Published</b> online</span>24 August 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00728" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00728</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10263%26pageCount%3D24%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAndrea%2BShergalis%252C%2BDing%2BXue%252C%2BFatma%2BZ.%2BGharbia%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D18%26contentID%3Dacs.jmedchem.0c00728%26title%3DCharacterization%2Bof%2BAminobenzylphenols%2Bas%2BProtein%2BDisulfide%2BIsomerase%2BInhibitors%2Bin%2BGlioblastoma%2BCell%2BLines%26numPages%3D24%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10286%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00728"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">980</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00728" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Characterization of Aminobenzylphenols as Protein Disulfide Isomerase Inhibitors in Glioblastoma Cell Lines&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Shergalis&quot;},{&quot;first_name&quot;:&quot;Ding&quot;,&quot;last_name&quot;:&quot;Xue&quot;},{&quot;first_name&quot;:&quot;Fatma&quot;,&quot;last_name&quot;:&quot;Z. Gharbia&quot;},{&quot;first_name&quot;:&quot;Hannah&quot;,&quot;last_name&quot;:&quot;Driks&quot;},{&quot;first_name&quot;:&quot;Binita&quot;,&quot;last_name&quot;:&quot;Shrestha&quot;},{&quot;first_name&quot;:&quot;Amina&quot;,&quot;last_name&quot;:&quot;Tanweer&quot;},{&quot;first_name&quot;:&quot;Kirin&quot;,&quot;last_name&quot;:&quot;Cromer&quot;},{&quot;first_name&quot;:&quot;Mats&quot;,&quot;last_name&quot;:&quot;Ljungman&quot;},{&quot;first_name&quot;:&quot;Nouri&quot;,&quot;last_name&quot;:&quot;Neamati&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;18&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;10263-10286&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00728&quot;},&quot;abstract&quot;:&quot;Disulfide bond formation is a critical post-translational modification of newly synthesized polypeptides in the oxidizing environment of the endoplasmic reticulum and is mediated by protein disulfide isomerase (PDIA1). In this study, we report a series of α-aminobenzylphenol analogues as potent PDI inhibitors. The lead compound, AS15, is a covalent nanomolar inhibitor of PDI, and the combination of AS15 analogues with glutathione synthesis inhibitor buthionine sulfoximine (BSO) leads to synergistic cell growth inhibition. Using nascent RNA sequencing, we show that an AS15 analogue triggers the unfolded protein response in glioblastoma cells. A BODIPY-labeled analogue binds proteins including PDIA1, suggesting that the compounds are cell-permeable and reach the intended target. Taken together, these findings demonstrate an extensive biochemical characterization of a novel series of highly potent reactive small molecules that covalently bind to PDI.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00728&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00728" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00728&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00728" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00728&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00728" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00728&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00728&amp;href=/doi/10.1021/acs.jmedchem.0c00728" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00728" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00728" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00728%26sid%3Dliteratum%253Aachs%26pmid%3D32830969%26genre%3Darticle%26aulast%3DShergalis%26date%3D2020%26atitle%3DCharacterization%2Bof%2BAminobenzylphenols%2Bas%2BProtein%2BDisulfide%2BIsomerase%2BInhibitors%2Bin%2BGlioblastoma%2BCell%2BLines%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D18%26spage%3D10263%26epage%3D10286%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292169" title="Antioxidants">Antioxidants</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/18" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/jmcmar.2020.63.issue-18/20200924/jmcmar.2020.63.issue-18.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0028.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Disulfide bond formation is a critical post-translational modification of newly synthesized polypeptides in the oxidizing environment of the endoplasmic reticulum and is mediated by protein disulfide isomerase (PDIA1). In this study, we report a series of α-aminobenzylphenol analogues as potent PDI inhibitors. The lead compound, <b>AS15</b>, is a covalent nanomolar inhibitor of PDI, and the combination of <b>AS15</b> analogues with glutathione synthesis inhibitor buthionine sulfoximine (BSO) leads to synergistic cell growth inhibition. Using nascent RNA sequencing, we show that an <b>AS15</b> analogue triggers the unfolded protein response in glioblastoma cells. A BODIPY-labeled analogue binds proteins including PDIA1, suggesting that the compounds are cell-permeable and reach the intended target. Taken together, these findings demonstrate an extensive biochemical characterization of a novel series of highly potent reactive small molecules that covalently bind to PDI.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Glioblastoma (GBM) is the leading primary brain cancer diagnosed in adults.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> GBM is an aggressive, high-grade glioma with a low 5.6% 5-year survival rate. Current standard-of-care—radiation and temozolomide chemotherapy—prolongs survival by a few months, but even after remission, cancer recurs. Thus, innovative therapies are needed to effectively treat GBM.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> GBM cells survive by upregulating the protein folding oxidoreductase protein disulfide isomerase (PDIA1, also known as PDI). PDI reduces, oxidizes, and isomerizes disulfide bonds in nascent polypeptides and other substrates via two catalytic CGHC active sites that sit 15–30 Å apart in two homogeneous domains.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> PDI is overexpressed in several cancers to meet the increased demands in protein synthesis.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a></div><div class="NLM_p">The <b>a</b> and <b>a′</b> domains of PDI are connected by <b>b</b> and <b>b′</b> domains, which share identity with the <b>a</b> and <b>a′</b> domains, but do not contain the CGHC active sites. As evidenced by the p<i>K</i><sub>a</sub> values, the N-terminal cysteine in each <b>a</b> and <b>a′</b> domain active site (Cys53 and Cys397) is stabilized in the thiolate form, while the C-terminal cysteine thiolate is destabilized. This allows the nucleophilic N-terminal cysteines to attack substrates and form mixed disulfides. The C-terminal cysteines more selectively react with the N-terminal cysteines, mediated in the <b>a</b> domain by the p<i>K</i><sub>a</sub> of Cys56 that is lowered by the local environment, containing a conserved Arg120.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> This reaction generates a reduced substrate and disulfide in PDI.</div><div class="NLM_p">The thiolate form of the redox-active N-terminal cysteines reacts with electrophilic compounds. Thus, many electrophilic compounds have been identified as covalent PDI inhibitors, including <b>PACMA31</b>,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a><b>KSC–34</b>,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> 3,4-methylenedioxy-β-nitrostyrene (MNS),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and <b>16F16</b><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). <b>PACMA31</b>, an irreversible inhibitor of PDI, demonstrated <i>in vivo</i> efficacy in a mouse model of ovarian cancer.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Furthermore, <b>PACMA31</b> synergized with the multikinase inhibitor sorafenib in a mouse model of hepatocellular carcinoma.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> One of the main challenges of characterizing the many PDI inhibitors identified to date, recently emphasized by Foster et al., is competition with endogenous levels of glutathione.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Active site PDI inhibitors, especially reactive electrophiles such as <b>PACMA31</b>, may compete with glutathione for binding the cysteine thiols of PDI. This strategy is further confounded by the fact that several PDI family members share similar active site motifs, and although the CGHC active site is one of the most reactive of the thioredoxin superfamily, electrophiles that bind this site may also bind to other CxxC-containing PDI family members. One strategy to mitigate off-target toxicity of cysteine-reactive active site inhibitors is to develop more selective inhibitors.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Previously reported PDI inhibitors studied in the context of ovarian cancer (<b>PACMA31</b>), Huntington’s disease (<b>LOC14</b>), brain cancer (<b>BAP2</b> and <b>35G8</b>), thrombosis (<b>Bepristat 1a</b> and <b>isoquercetin</b>), and multiple myeloma (<b>CCF642</b>). <b>KSC-34</b> is an <b>a</b>-site selective probe and estradiol is an endogenous ligand of PDIA1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition to the increased PDI expression, GBM cells upregulate the antioxidant defense system and are increasingly dependent on glutathione as the tumor grows.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Glutathione is one of the most abundant molecules in the cell, with concentrations of reduced glutathione (GSH) estimated to reach up to 10 mM in cellular compartments. The tripeptide composed of glutamic acid, glycine, and cysteine is a key redox buffer and antioxidant molecule involved in many cellular processes including reactive oxygen species removal, signal transduction, and protein synthesis.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Although the total glutathione concentration is similar in the cytosol and endoplasmic reticulum (ER), the ratio of reduced to oxidized glutathione dictates its role in each compartment. In the cytosol, glutathione is present mainly in its reduced form, with a ratio of GSH:GSSG of ∼50 000:1; the ER is a more oxidizing compartment, with an estimated GSH:GSSG ratio of less than 7:1.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Because of this environment, PDIA1 is present in its disulfide form, poised to accept electrons from the reduced thiols in nascent polypeptides. The oxidizing environment of the ER aids in protein folding and disulfide bond formation.</div><div class="NLM_p last">The initial goal of this study was to characterize a new class of PDI inhibitors in the context of GBM. Via a medium-throughput biochemical screen, we identified a series of α-aminobenzylphenols as inhibitors of PDI that possibly target the active site cysteines of PDI via a retro-Michael addition reaction. Similar compounds have been reported for multiple targets, including HDACs,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> migration inhibitory factor (MIF) tautomerase,<a onclick="showRef(event, 'ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23">(20−23)</a> STAT3/5,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and frataxin,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> among others. This finding suggested that these inhibitors may not be selective for a single protein in the cells. However, we synthesized a BODIPY-labeled analogue that did not bind any of the previously published targets. Binding was enhanced when the cells were pretreated with buthionine sulfoximine (BSO), suggesting that the efficacy may be lowered by intracellular glutathione. Additionally, potency in cells increased with BSO treatment. Thus, the compounds are cell-permeable, and cytotoxicity is enhanced with glutathione depletion. In all, this study represents the importance of validating in-cell target engagement early in the drug discovery process and provides an extensive characterization of a class of thiol-reactive small molecules.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25555" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25555" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">Most of the compounds in the present study were synthesized via a well-established Betti reaction employing a phenolic compound, an aldehyde, and an amine in a one-pot fashion.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> Reactions were either heated in toluene at 100 °C or in EtOH under microwave irradiation at 120 °C for 30 min depending on the electronic nature of the phenolic starting material. Typically, simple phenols, hydroxy naphthalenes, and hydroxy quinolones can be used in such reactions, whereas the reactivity of other hydroxy heterocyclic structures remains uncertain. In our study, depending on the substitution pattern and electronic properties of the starting materials, three different types of products were generated (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). For starting materials represented by <b>3</b>, Type I compounds were obtained unambiguously, while with <b>4a</b>/<b>4b</b>, electrophilic replacement could occur at two distinct ortho positions of the hydroxy, leading to either Type II or Type III compounds. We found that regioselectivity was strongly related to the electronic properties of the bicyclic heterocycles: electron-efficient compounds tend to generate Type II compounds, and electron-deficient compounds generate Type III compounds.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0011.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Compounds with the Modified Betti Reaction<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) toluene, 100 °C and (b) EtOH, 120 °C microwave or conventional heating.</p></p></figure><div class="NLM_p">Compound <b>8</b> was prepared by direct methylation of <b>5g</b> with dimethyl sulfate (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Compound <b>13</b>, with a cyclohexane in place of the piperidine moiety, was synthesized by a four-step strategy (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Benzo[<i>d</i>][1,3]dioxol-5-ol (<b>9</b>) was reacted with benzoyl chloride by an ortho-selective electrophilic attack facilitated by a highly coordinating metal phenolate intermediate generated <i>in situ</i>.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> The diaryl ketone <b>10</b> was reacted with cyclohexylmagnesium bromide to give tertiary alcohol <b>11</b>. A TsOH-catalyzed elimination afforded <b>12</b>, and <b>12</b> was subjected to catalytic hydrogenation to give <b>13</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0012.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Me<sub>2</sub>SO<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, acetone.</p></p></figure><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0013.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a)(i), MeI, Mg, Et<sub>2</sub>O and (ii), benzoyl chloride; (b) cyclohexylmagnesium bromide, tetrahydrofuran (THF), rt −50 °C; (c) <i>p</i>-TsOH, toluene, rt; and (d) Pd/C, H<sub>2</sub>, EtOH, 60 °C.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Lead Compound AS15 Is a Nanomolar Inhibitor of PDI</h3><div class="NLM_p">A screening campaign of 1000 highly diverse compounds from the National Cancer Institute at 40 μM in the PDI reductase assay afforded almost 200 compounds that inhibited 50% of PDI activity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Those compounds were screened for cytotoxicity in U-87 MG and MIA PaCa-2 cell lines, and compounds with >40% inhibition of cell growth at 30 μM were subjected to dose–response assays in both the 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) and PDI reductase assays. <b>AS15</b> and <b>CD343</b> emerged as lead scaffolds with IC<sub>50</sub> values in the PDI reductase assay of 300 ± 90 and 170 ± 50 nM, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C). When a residue in the <b>b′</b> domain of PDI important for substrate binding, histidine 256, was mutated to an alanine residue, both <b>AS15</b> and <b>CD343</b> retained their activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S1A</a>). This result suggested that <b>AS15</b> and <b>CD343</b> activities were not dependent on the substrate-binding domain. Thus, we focused on determining whether the inhibition was based on the interaction between <b>AS15</b> and <b>CD343</b> and the active site cysteines. <b>AS15</b> and <b>CD343</b> inhibited the activity of PDIA2 and, to a lesser extent, PDIA3 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S1B</a>). The lead compounds decreased the viability of U-87 MG cells with IC<sub>50</sub> values of 18.3 ± 9.2 μM for <b>AS15</b> and 10.6 ± 0.7 μM for <b>CD343</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S1C</a>). Interaction with oxidized PDI was further probed with the thermal shift assay; however, <b>AS15</b> and <b>CD343</b> did not stabilize PDI to thermal degradation, similar to <b>PACMA31</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). These initial results suggested that the compounds inhibit PDI as thiol-reactive compounds and not via the substrate-binding domain like estradiol or <b>BAP2</b>.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Screening approach leading to the discovery of <b>AS15</b> and <b>CD343</b>. (b) Structures of <b>AS15</b> and <b>CD343</b>, and IC<sub>50</sub> values calculated in the PDI reductase assay. (c) Dose–response curves of <b>PACMA31</b>, <b>AS15</b>, and <b>CD343</b> in the PDI reductase assay. (d) Shifts in the melting temperatures of <b>PACMA31</b>, <b>AS15</b>, and <b>CD343</b> in the thermal shift assay. SAR: structure–activity relationship.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structure–Activity Relationships Reveal That AS15 Analogues Are Not Substrate-Binding Domain Inhibitors</h3><div class="NLM_p">To further assess the mechanism of inhibition of <b>AS15</b> and <b>CD343</b> and determine the structural requirements for their activity, we obtained 89 analogues from ChemDiv libraries and the NCI Developmental Therapeutics Program and tested their potency in the PDI reductase assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Compounds were tested for purity, and only compounds with purity >95% were used for structure–activity relationship analysis. Generally, all <b>AS15</b> and <b>CD343</b> analogues possess an α-aminobenzylphenol core that consists of phenolic, amino, and phenyl moieties. While the phenolic moiety is limited to 5-hydroxybenzo[<i>d</i>][1,3]dioxole (<b>AS15</b> analogues) or 8-hydroxyquinoline (<b>CD343</b> analogues), the amino and phenyl moieties are diversified. For compounds with the 5-hydroxybenzo[<i>d</i>][1,3]dioxole moiety, a variety of cyclic amines including morpholine, piperidine, piperazine, and pyrrolidine were incorporated. Most of the compounds inhibited PDI with IC<sub>50</sub> values below 1 μM, and different halogens and electron-donating groups such as chloro, fluoro, methoxy, amino, and hydroxy on the phenyl ring were well tolerated. Some of the compounds showed moderate cytotoxicity with IC<sub>50</sub> values of 10–30 μM; however, correlation with PDI inhibition was not observed (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Aromatic amines such as aminopyridine, aminopyrimidine, and imidazole were generally tolerated, and PDI inhibition was comparable to those with saturated cyclic amines. Ureas were not tolerated and led to a complete loss of activity, possibly because of their loss of basicity that contributed to PDI binding (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compounds without the bicyclic aromatic core or hydroxy group were inactive against PDI and nontoxic (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S2</a>). Furthermore, compounds without the tertiary amine were inactive against PDI and were nontoxic, with the exception of trimethoxy substituents on the aromatic ring (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Table S1</a>). <b>CD</b> analogues possess the 8-hydroxyquinoline core along with an aromatic amine such as 2-aminopyridine or aniline, and their ability to inhibit PDI activity was comparable to the <b>AS15</b> analogues, except that the 4-methyl substitution of the pyridine was not compatible when methyl or chloro was present on the phenyl moiety. Interestingly, many of these compounds showed stronger cytotoxicity with IC<sub>50</sub> values as low as 2.1 ± 0.1 μM (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Compounds with 5-Hydroxybenzo[<i>d</i>][1,3]dioxole and Saturated Cyclic Amine Moieties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0015.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0016.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of PDI was assessed by the PDI reductase assay. IC<sub>50</sub> values are indicated as the mean ± standard deviation (SD) of at least three independent experiments.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">MTT cytotoxicity IC<sub>50</sub> values were determined in U-87 MG cells.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of Compounds with 5-Hydroxybenzo[<i>d</i>][1,3]dioxole and Other Amine Moieties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0017.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0018.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of PDI was assessed by the PDI reductase assay. IC<sub>50</sub> values are indicated as the mean ± standard deviation (SD) of at least three independent experiments.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">MTT cytotoxicity IC<sub>50</sub> values were determined in U-87 MG cells.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of Compounds with 8-Hydroxyquinoline and 2-Aminopyridine Moieties<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0019.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0020.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of PDI was assessed by the PDI reductase assay. IC<sub>50</sub> values are indicated as the mean ± standard deviation (SD) of at least three independent experiments.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">MTT cytotoxicity IC<sub>50</sub> values were determined in U-87 MG cells.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of Compounds with 8-Hydroxyquinoline and Aniline Moieties</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0021.gif" alt="" id="fx7" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">NSC Number</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">PDI IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">MTT IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NC266</td><td class="colsep0 rowsep0" align="left">1008</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.11 ± 0.04</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NC268</td><td class="colsep0 rowsep0" align="left">1010</td><td class="colsep0 rowsep0" align="left">4-NO<sub>2</sub></td><td class="colsep0 rowsep0" align="left">0.15 ± 0.05</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NC269</td><td class="colsep0 rowsep0" align="left">1011</td><td class="colsep0 rowsep0" align="left">4-COOH</td><td class="colsep0 rowsep0" align="left">0.30 ± 0.05</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NC270</td><td class="colsep0 rowsep0" align="left">1012</td><td class="colsep0 rowsep0" align="left">2-COOH</td><td class="colsep0 rowsep0" align="left">1.48 ± 1.52</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NC273</td><td class="colsep0 rowsep0" align="left">1015</td><td class="colsep0 rowsep0" align="left">2-COOC<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left"><0.20</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NC282</td><td class="colsep0 rowsep0" align="left">84087</td><td class="colsep0 rowsep0" align="left">2-OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">0.39 ± 0.11</td><td class="colsep0 rowsep0" align="left">>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of PDI was assessed by the PDI reductase assay. IC<sub>50</sub> values are indicated as the mean ± standard deviation (SD) of at least three independent experiments.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">MTT cytotoxicity IC<sub>50</sub> values were determined in U-87 MG cells.</p></div></div></div><div class="NLM_p">Having explored the SAR around the amino and phenyl moieties, which generally had a nominal impact on PDI inhibition, we synthesized and tested an additional 38 compounds to diversify the phenolic moiety and probe the impact on PDI inhibition. Notably, this is also the first exploration of the reactivity and regioselectivity of a variety of hydroxy bicyclic heterocycles in Betti-style reactions and the biological activity of their products. Compounds were classified into three types based on the substitution pattern on the phenolic moiety. Type I compounds have a similar substitution pattern as <b>CD343</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The results indicated that 8-hydroxyquinazoline (<b>5c,d</b>), 5-hydroxyquinoxaline (<b>5e</b>), 4-hydroxybenzothiazole (<b>5f</b>), 7-hydroxybenzofuran (<b>5g</b>), 5-hydroxy-1,4-benzodioxane (<b>5h</b>), 4-hydroxybenzo[<i>d</i>][1,3]dioxole (<b>5i</b>), and 5-hydroxyisoquinoline (<b>5j</b>) are all well tolerated. <b>5h</b> has the highest potency with an IC<sub>50</sub> value of 290 ± 120 nM. The incorporation of 4-indanol (<b>5k</b>) led to a complete loss of the potency, indicating that a H-bond acceptor might be essential. The fact that <b>5l</b> lost activity despite containing a H-bond acceptor suggests that a bicyclic structure is necessary to fulfill the steric requirement around this moiety. Using <b>5g</b> as a model compound, different substituents were introduced. All substituents were well tolerated, and <b>5p</b> and <b>5q</b> exhibited IC<sub>50</sub> values below 0.2 μM. For Type II compounds with a similar substitution pattern as <b>AS15</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>), both 5-hydroxybenzo[<i>d</i>][1,3]dioxole and 6-hydroxy-1,4-benzodioxane are preferred, with 6-hydroxyindoline leading to a moderate loss of activity (<b>6d</b>). Similar to <b>5k</b> in the <b>CD</b> series, compounds <b>6f</b> and <b>6g</b> completely lost activity. Type III compounds with substitution patterns different from <b>AS15</b> and <b>CD343</b> were also synthesized by incorporating different phenolic moieties (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). The majority of the compounds lost their activity, suggesting an unfavorable binding mode. The exceptions included compounds containing a 5-hydroxyindole or 5-hydroxyindazole, with the latter leading to comparable inhibition of PDI at submicromolar IC<sub>50</sub> values for all of its analogues. It is possible that in this case the −NH group of indole/indazole forms additional interactions with PDI as a H-bond donor. Most of the synthesized α-aminobenzylphenols were not cytotoxic at 30 μM, which was similar to the trend observed with the <b>NC</b> and <b>CD</b> series (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Tables <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of Type I Compounds<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0022.gif" alt="" id="fx8" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0023.gif" alt="" id="fx9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of PDI was assessed by the PDI reductase assay. IC<sub>50</sub> values are indicated as the mean ± standard deviation (SD) of at least three independent experiments.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">MTT cytotoxicity determined in U-87 MG cells.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SAR of Type II Compounds<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0024.gif" alt="" id="fx10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0025.gif" alt="" id="fx11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of PDI was assessed by the PDI reductase assay. IC<sub>50</sub> values are indicated as the mean ± standard deviation (SD) of at least three independent experiments.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">MTT cytotoxicity determined in U-87 MG cells. <sup><i>c</i></sup>MTT cytotoxicity determined in A-172 cells.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">MTT cytotoxicity determined in A-172 cells.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. SAR of Type III Compounds<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0026.gif" alt="" id="fx12" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0027.gif" alt="" id="fx13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of PDI was assessed by the PDI reductase assay. IC<sub>50</sub> values are indicated as the mean ± standard deviation (SD) of at least three independent experiments.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">MTT cytotoxicity determined in U-87 MG cells.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> AS15 Analogues Covalently Bind to PDI</h3><div class="NLM_p">We addressed the importance of the phenolic hydroxy and amine moieties of the series with two compounds; <b>8</b> lost activity with the replacement of the phenolic hydroxy (<b>5g</b>) with a methoxy (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Similarly, replacing the piperidine in <b>6b</b> with a cyclohexane abolished its PDI inhibition (<b>13</b>) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Both compounds demonstrated the importance of the α-aminobenzylphenol core to the potency of this series of compounds.</div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure–activity relationship indicates that compounds are covalent PDI inhibitors. (a) Comparison of compounds with free hydroxy or methoxy on the bicyclic moiety. (b) Comparison of compounds with piperidine or cyclohexane moiety. (c) Proposed mechanism of inhibition via the retro-Michael addition reaction. (d) <b>6a</b> covalently binds to PDI. 100 μM <b>6a</b> was incubated with 10 μM PDI and 100 μM dithiothreitol (DTT) for 30 min prior to injection. (e) <b>6a</b> binds to PDI preferentially over GSTO1. 100 μM <b>6a</b> was incubated with a mixture of 10 μM PDI, 10 μM GSTO1, and 10 μM GRP78 as a nonspecific binding control for 30 min prior to injection. (Close-up of GRP78 trace in Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S3</a>.) (f) Concentration- and time-dependent PDI inhibition curves for <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> determination of <b>PACMA31</b>, <b>16F16</b>, and <b>AS15</b>. Activity was measured using the PDI reductase assay. Absorbance was monitored over time at various concentrations and preincubation times with indicated compounds. (g) Gel-based competition with recombinant PDI, PDI inhibitors, or inactive analogue <b>6f</b> at 100 μM and <b>PACMA57</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Due to the nature of disulfide bonds, we hypothesized that the <b>AS15</b> analogues may be attacked by the nucleophilic cysteine thiols in the active sites of reduced PDI, as is the case for other electrophilic PDI inhibitors. Upon a reaction that exposes the Michael acceptor, the free thiol on PDI could react and form a covalent adduct (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). This type of the retro-Michael addition to protein thiols has been observed with hydroxyquinolines like <b>CD343</b>.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> A zinc-dependent mechanism opened a quinone methide for selective reaction with HDAC5 and HDAC9;<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> another quinone methide intermediate was found to react with protein thiols over forming DNA adducts,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and cocrystallization confirmed pyridinylmethyl quinoline fragment binding to MIF tautomerase via a proline residue.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In the case of the pyridinylmethyl quinoline fragment, the compound bound via a retro-Michael addition reaction that formed the quinone methide intermediate. This intermediate was primed to undergo the aza-Michael addition to covalently link to a proline residue in MIF tautomerase.</div><div class="NLM_p">To confirm that <b>AS15</b> analogues covalently label PDI, we incubated 10 μM PDI with 100 μM DTT and 100 μM <b>6a</b> or <b>6f</b> and monitored adduct formation with quadrupole time-of-flight mass spectrometry (Q-TOF). We found that the fragment matching the proposed mechanism of action was apparent rapidly after the compound was added (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). <b>6a</b> bound reduced PDI at three sites. Recombinant PDI contains six cysteines, four of which form disulfide bonds when oxidized. When <b>6a</b> was incubated in a mixture of 1:1:1 PDI:GSTO1:GRP78, the mass of PDI increased by 2 equiv of the fragment, demonstrating that <b>6a</b> can bind PDI in the presence of competing proteins (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">S3A</a>). An inactive analogue of <b>6a</b> without the aromatic benzoxole (<b>6f</b>) did not demonstrate covalent binding to PDI under the same conditions (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S3B</a>).</div><div class="NLM_p">To assess the covalent binding nature, we measured the <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> of the lead compounds in the PDI reductase assay with reduced PDI. For covalent inhibitors, the <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> is the ratio of the observed rate of inactivation after a reversible reaction to form a protein–inhibitor (P–I) complex with all of the protein molecules (<i>k</i><sub>inact</sub>) to the concentration of inhibitor required to reach half of the maximum rate of covalent bond formation (<i>K</i><sub>I</sub>). The kinetics of covalent PDI inhibitors <b>16F16</b>, <b>PACMA31</b>, and <b>AS15</b> were measured by assessing activity in the PDI reductase assay at incubation times from 5 to 60 min. <b>AS15</b> inhibited PDI with a <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> of 2.6 × 10<sup>3</sup> M<sup>–1</sup> s<sup>–1</sup> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> for <b>PACMA31</b> was 2.0 × 10<sup>2</sup> M<sup>–1</sup> s<sup>–1</sup> and <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> for <b>16F16</b> was 1.7 × 10<sup>2</sup> M<sup>–1</sup> s<sup>–1</sup>. Thus, <b>AS15</b> was more efficient at inhibiting PDI than both covalent inhibitors <b>PACMA31</b> and <b>16F16</b>. Furthermore, a gel-based competition assay with the fluorescent probe of <b>PACMA31</b>, <b>PACMA57</b>, confirmed that <b>AS15</b> analogues could compete with <b>PACMA57</b> to bind PDI (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Because <b>PACMA57</b> binds PDI at either Cys397 or Cys400, this suggests that <b>AS15</b> can also bind one of these cysteines. Inactive analogue <b>6f</b> did not compete with <b>PACMA57</b> for binding to PDI.</div><div class="NLM_p">To address whether the <b>AS15</b> analogues targeted PDI irreversibly, like <b>PACMA31</b>, we performed washout experiments with the PDI reductase assay. <b>AS15</b> (50 μM) was incubated with 40 μM PDI for 3 h at room temperature. After 3 h, the PDI–<b>AS15</b> complex was diluted 100-fold into the reaction buffer, the reaction was incubated another hour at 37 °C, and insulin was added as a substrate to initiate the reaction. <b>PACMA31</b> (an irreversible inhibitor) at 1 and 100 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A) and <b>BAP2</b> (a reversible inhibitor) at 0.5 and 50 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) were used as controls. We found that both <b>AS15</b> and <b>CD343</b> did not maintain the characteristics of the high concentration of inhibitors after dilution to the low concentration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D). However, when we tested enzyme activity recovery with compound concentrations 10 times that of the protein to mimic the mass spectrometry conditions, <b>AS15</b> behaves like the expected irreversible inhibitor (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S4</a>). <b>CD343</b> and <b>6a</b> were too active at this ratio. We further tested whether the compounds were binding in the <b>b′</b> domain of oxidized PDI with the ANS (anilinonaphthalene sulfonic acid) spectral scan.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> ANS is a dye that fluoresces upon binding hydrophobic pockets and specifically targets the <b>b′</b> domain in PDI. <b>B′</b> domain-selective inhibitors of PDI such as estradiol and bepristat 1a compete with ANS.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a><b>AS15</b> and <b>CD343</b> did not lower the fluorescence of ANS (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). Our combined results from the thermal shift assay, washout experiment, and ANS spectral scan demonstrated that <b>AS15</b> and <b>CD343</b> are likely not substrate-binding domain inhibitors like estradiol and <b>BAP2</b>, but may be covalent inhibitors of PDI.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Recovery of PDI activity upon treatment with <b>PACMA31</b> (a), <b>BAP2</b> (b), <b>AS15</b> (c), or <b>CD343</b> (d). For the diluted samples (purple curves), 40 μM PDI was incubated with 100 μM <b>PACMA31</b>, 50 μM <b>BAP2</b>, 50 μM <b>AS15</b>, or 50 μM <b>CD343</b> for 3 h at rt before dilution. (f) ANS spectral scan with 5 μM PDI and 100 μM <b>PACMA31</b>, <b>BAP2</b>, <b>AS15</b>, or <b>CD343</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> AS15 Analogues Compete with Glutathione</h3><div class="NLM_p">Because the <b>AS15</b> analogues seemed highly reactive toward nucleophilic attack, we hypothesized that glutathione may also react with the compounds. Glutathione is present in high concentrations in the cytoplasm and is an important redox regulator in the ER.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> The oxidizing environment of the ER is maintained by the ratio of the reduced to oxidized glutathione, which is lower than the ratio in the cytoplasm. Incubating the compounds with <i>N</i>-acetyl cysteine (NAC) before adding them to the PDI reductase assay rendered the compounds inactive (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Although both the <b>AS15</b> series and <b>PACMA31</b> are irreversible PDI inhibitors, the differing characteristics in the NAC competition assay, as well as differing kinetics, suggest that the compounds may have different mechanisms of action. <b>AS15</b> analogues were also inactivated with competing glutathione at physiological concentrations (5 mM) in the PDI reductase assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The compounds retained activity when incubated with PDI before adding GSH or NAC, providing further evidence of potential thiol reactivity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S5</a>).</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <i>N</i>-Acetyl cysteine and glutathione inactivate <b>AS15</b> analogues. (a) <i>N</i>-Acetyl cysteine (NAC) competition in the PDI reductase assay. Compounds were incubated with NAC prior to the addition to a final concentration of 10 μM in the PDI reductase assay with PDIA1. (b) Glutathione competition in the PDI reductase assay. GSH was added to the PDI reaction buffer prior to a 10 μM compound addition. (c) Top two <b>AS15</b> analogues that are least sensitive to competition with 5 mM GSH in the PDI reductase assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To assess trends in the glutathione sensitivity of the <b>AS15</b> series, we screened all of the analogues in the PDI reductase assay in the presence of 5 mM GSH. The high concentrations of glutathione in the cytoplasm may inactivate the compounds <i>in vivo</i>; thus, the <i>in vitro</i> IC<sub>50</sub> value may not be a reliable indicator of <i>in vivo</i> activity.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Although the PDI reductase assay is performed at a relatively high concentration of DTT (500 μM), we added 5 mM glutathione to mimic a more physiological environment (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Table S2</a>). This issue is particularly relevant because the analogues behave as reversible thiol adducts. Substrate-binding domain inhibitors isoquercetin and <b>BAP2</b> remained active in the presence of high GSH concentrations. Most of the <b>AS15</b> analogues were inactive in the presence of competing GSH. These results provided further support that the <b>AS15</b> analogues may act via addition to PDI. We found two <b>AS15</b> analogues that maintained potency in the presence of competing glutathione: <b>14</b> and <b>5d</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C).</div><div class="NLM_p">After our GSH screen, we retested the purity of <b>14</b> and <b>5d</b> and observed that <b>5d</b> spontaneously formed a dimer after long-term storage. As a result, we repurified the monomeric and dimeric forms of <b>5d</b> and tested each form in the PDI reductase assay in the presence of glutathione (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). The dimeric form of <b>5d</b> was less sensitive to glutathione competition than the monomer (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Furthermore, we found that both the monomer and dimer of <b>5d</b> bound to PDI (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). Incubation with the <b>a′c</b> domain gave a species with two fragments of <b>5d</b> monomer bound (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S6A</a>). When we incubated the monomer and dimer of <b>5d</b> with a C53S mutant of PDI, we observed one and two species bound, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S6B</a>).</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Dimerized analogue is less sensitive to GSH treatment. (a) Structure of <b>5d</b> dimerization. (b) Percent inhibition of <b>5d</b> monomer and dimer in the PDI reductase assay in competition with 5 mM glutathione. (c) Protein mass spectrometry confirms that the fragment of <b>5d</b> binds to PDI. Toxicity of <b>PACMA31</b> (d), <b>AS15</b> (e), <b>CD343</b> (f), <b>5d</b> monomer (g), and <b>5d</b> dimer (h) in the colony formation assay in the absence or presence of BSO. U-118 MG cells were pretreated with BSO for 24 h prior to compound addition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Although the PDI disulfides are 500-fold more reactive than glutathione,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> we hypothesized that glutathione may be inactivating the <b>AS15</b> analogues and contributing to lower their potency in cells. Pretreatment of GBM cells with BSO for 24 h before adding the compounds increased potency in the colony formation assay (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D–H and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">S7</a>). While the monomer of <b>5d</b> was more sensitive to BSO addition, the <b>5d</b> dimer was more potent and its potency was not dependent on BSO addition. These results support the hypothesis that glutathione depletion sensitizes GBM cells to PDI inhibition.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> AS15 Analogue Target Identification</h3><div class="NLM_p">We sought to confirm whether the <b>AS15</b> analogues could target PDI in the cells by synthesizing two analogues of <b>6b</b> with a BODIPY fluorescent tag on the phenyl ring (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). The BODIPY-labeled compounds differed in the linker length between the parent compound and the tag. One compound was directly attached to BODIPY with an amide linkage on the phenyl ring (<b>15</b>), while the other (<b>16</b>) contained a two-carbon linker separating the amide groups on the BODIPY structure. <b>15</b> did not inhibit PDI activity in the PDI reductase assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S8</a>). However, it did covalently bind proteins around 55–70 kDa when added to cells (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). This prompted us to perform in-gel digestion and mass spectrometry of the proteins at this size (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). Albumin, the most abundant plasma protein, was identified as one of the proteins in the band, and incubation of both <b>15</b> and <b>16</b> with cell-free medium supplemented with fetal bovine serum demonstrated that the compounds bound to serum albumin (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Although <b>16</b> bound to serum albumin, it inhibited PDI activity with an IC<sub>50</sub> value comparable to the parent compound of 1.37 ± 0.23 μM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D). To verify that the band from <b>15</b> was not PDI, a Western blot was performed with cells treated with <b>15</b> and <b>16</b>. The GFP band from the <b>15</b>-treated cells runs closer to the molecular weight of albumin (69 kDa) than the PDI band around 55 kDa (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S9A</a>). Additionally, when the cells were treated with <b>15</b> after serum starvation, the band disappeared (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S9B</a>). With recombinant PDI, <b>16</b> covalently binds, and binding can be blocked with DTT, further supporting the mechanism of inhibition of this series (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S10</a>).</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0008.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <b>AS15</b> analogues bind serum albumin. (a) Structures of BODIPY-labeled <b>AS15</b> analogues. (b) U-118 MG cells treated with 40 μM <b>15</b> overnight. (c) Cell-free DMEM treated with 20 μM <b>15</b> or <b>16</b> overnight at 37 °C. (d) <b>16</b> dose–response curve in the PDI reductase assay. (e) Cell lysates incubated with 50 μM <b>6b</b> and 5 μM <b>15</b> or <b>16</b> for 24 h at room temperature after cell lysis. (f) Cell lysates treated with increasing concentrations of parent compound <b>6b</b> before the addition of BODIPY-labeled <b>16</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0014.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route for Compounds <b>15</b> and <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) toluene, 100 °C; (b) 4-(5,5-difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl)aniline (BODIPY aniline), EDCI, HOBt, dimethylformamide (DMF), rt; (c) <i>tert</i>-butyl (2-aminoethyl)carbamate, <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (HATU), <i>N</i>,<i>N-</i>diisopropylethylamine (DIEA), DMF, rt; (d) trifluoroacetic acid (TFA), dichloromethane (DCM), rt; and (e) 4-(5,5-difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl)benzoic acid, HATU, DIEA, DMF, rt.</p></p></figure><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Most Abundant Proteins Around 55–70 kDa in U-118 MG Cells</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">accession</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center"># cysteines</th><th class="colsep0 rowsep0" align="center">location</th><th class="colsep0 rowsep0" align="center" char=".">coverage [%]</th><th class="colsep0 rowsep0" align="center" char="."># peptides</th><th class="colsep0 rowsep0" align="center" char="."># PSMs</th><th class="colsep0 rowsep0" align="center" char="."># unique peptides</th><th class="colsep0 rowsep0" align="center" char=".">MW [kDa]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P14618</td><td class="colsep0 rowsep0" align="left">Pyruvate kinase PKM (PKM)</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">nucleus</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">181</td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="char" char=".">57.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P10809</td><td class="colsep0 rowsep0" align="left">60 kDa heat shock protein, mitochondrial (HSPD1)</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">mitochondria</td><td class="colsep0 rowsep0" align="char" char=".">72</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char=".">104</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char=".">61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P07237</td><td class="colsep0 rowsep0" align="left">Protein disulfide isomerase (P4HB)</td><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">ER/membrane</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">57.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P31948</td><td class="colsep0 rowsep0" align="left">Stress-induced-phosphoprotein 1 (STIP1)</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">nucleus</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">62.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P48643</td><td class="colsep0 rowsep0" align="left">T-complex protein 1 subunit epsilon (CCT5)</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">59.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P17987</td><td class="colsep0 rowsep0" align="left">T-complex protein 1 subunit α (TCP1)</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">60.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Q16555</td><td class="colsep0 rowsep0" align="left">Dihydropyrimidinase-related protein 2 (DPYSL)</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">70</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">62.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P49368</td><td class="colsep0 rowsep0" align="left">T-complex protein 1 subunit γ (CCT)</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">69</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">60.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P02768</td><td class="colsep0 rowsep0" align="left">Serum albumin (ALB)</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">extracellular</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">69.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P30153</td><td class="colsep0 rowsep0" align="left">Serine/threonine-protein phosphatase 2A 65 kDa regulatory subunit A α isoform (PPP2R1A)</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">nucleus</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">47</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">65.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P61978</td><td class="colsep0 rowsep0" align="left">Heterogeneous nuclear ribonucleoprotein K (HNRNPK)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">nucleus</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">50.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P50990</td><td class="colsep0 rowsep0" align="left">T-complex protein 1 subunit theta (CCT8)</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">45</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">59.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P54577</td><td class="colsep0 rowsep0" align="left">Tyrosine-tRNA ligase, cytoplasmic (YARS)</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">59.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">O43242</td><td class="colsep0 rowsep0" align="left">26S proteasome non-ATPase regulatory subunit 3 (PSMD3)</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">extracellular, nucleus, cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">60.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">O75083</td><td class="colsep0 rowsep0" align="left">WD repeat-containing protein 1 (WDR1)</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">cytoskeleton</td><td class="colsep0 rowsep0" align="char" char=".">53</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">66.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P36871</td><td class="colsep0 rowsep0" align="left">Phosphoglucomutase-1 (PGM1)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">extracellular, cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">61.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Q99832</td><td class="colsep0 rowsep0" align="left">T-complex protein 1 subunit eta (CCT7)</td><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">59.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P02545</td><td class="colsep0 rowsep0" align="left">Prelamin-A/C (LMNA)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">nucleus</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">74.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P40227</td><td class="colsep0 rowsep0" align="left">T-complex protein 1 subunit zeta (CCT6A)</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Q9H4M9</td><td class="colsep0 rowsep0" align="left">EH domain-containing protein 1 (EHD1)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">membrane, endosome</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">60.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">O43776</td><td class="colsep0 rowsep0" align="left">Asparagine-tRNA ligase, cytoplasmic (NARS)</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">62.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P29401</td><td class="colsep0 rowsep0" align="left">Transketolase (TKT)</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">extracellular, nucleus, cytoplasm</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">67.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P68363</td><td class="colsep0 rowsep0" align="left">Tubulin α-1B chain (TUBA1B)</td><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">cytoskeleton</td><td class="colsep0 rowsep0" align="char" char=".">65</td><td class="colsep0 rowsep0" align="char" char=".">19</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">50.1</td></tr></tbody></table></div></div><div class="NLM_p">To further investigate whether the BODIPY-labeled <b>AS15</b> analogues target PDI, we treated cell lysates with the compounds for 24 h. <b>15</b> did not covalently label proteins in the cell lysate; however, <b>16</b> bound in two major bands around 55 and 40 kDa, and a minor band below 55 kDa (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">S11</a>). In addition, the parent compound competed for labeling both bands in a dose-dependent manner but seemed to compete off the 55 kDa band at a lower concentration (5 × [probe]) than the 40 kDa band (20 × [probe]) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>F). In addition to inhibiting PDIA1 activity, <b>AS15</b> could also inhibit PDIA3 activity; thus, the band around 55 kDa could contain both PDIA1 and PDIA3. Compound <b>16</b> bound PDIA1, PDIA2, and PDIA3, in addition to BSA, and was competed off by the parent compound. Competition was not observed for binding to BSA, likely due to multiple binding sites for <b>16</b> on BSA (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S12</a>). <b>6b</b> dose-dependently competed off <b>16</b> for binding PDIA2 and PDIA3 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S13</a>). Combining BSA with PDIA1 for 24 h with <b>15</b> or <b>16</b> did not improve binding to PDIA1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S14</a>). Because glutathione inactivated the compounds <i>in vitro</i>, cells were preincubated with BSO prior to BODIPY-labeled compound treatment to determine whether depleting the cells of glutathione would improve on-target binding. BSO addition improved binding for <b>15</b>; however, binding was nonselective. Furthermore, the addition of 10% FBS decreased binding, further confirming the interaction between this series and serum albumin (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S15A</a>).</div><div class="NLM_p">Incubation of the cell lysate with <b>16</b> consistently resulted in multiple bands (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S16B</a>). Proteomic analysis of each of the bands revealed PDIA1 among other targets in the cell lysate (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Tables S3 and S4</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_001.xlsx" class="ext-link">Supplemental File 1</a>). Because of the reactivity of these compounds, it is possible that they bind to proteins besides PDIA1 in the cell lysate. Reported targets of similar scaffolds include MIF tautomerase, HDAC5/9, and BRAF<sup>V600E</sup>. However, we did not observe bands at the molecular weights of those three targets, suggesting that the <b>AS15</b> analogues do not bind these proteins or the proteins have low abundance in the cell lines tested. Furthermore, those known targets, as well as STAT3, STAT5, Mcl-1, frataxin, and P2Y12, were not found in the bands analyzed with the proteomics experiment. In addition, confocal microscopy revealed that the BODIPY-labeled analogues mainly reside in the cytoplasm and ER, not in the nucleus with HDAC5/9 and BRAF (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>).</div><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0009.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Confocal microscopy images at 60× magnification of A-172 cells treated with 10 μM <b>15</b> (a) or 2 μM <b>16</b> (b) for 24 h prior to fixation and staining for PDI.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> AS15 Analogue Activates the Unfolded Protein Response</h3><div class="NLM_p">We performed nascent RNA sequencing of one of the most potent analogues of <b>AS15</b>, <b>5p</b>, to analyze changes in gene transcription in U-87 MG cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Four hours after 20 μM <b>5p</b> treatment, 68 genes were upregulated at least twofold and 12 genes were downregulated at least twofold (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Tables S5–S12</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_002.xlsx" class="ext-link">Supplemental File 2</a>). We performed Gene Set Enrichment Analysis on the preranked gene list of 7907 genes and identified that <b>5p</b> upregulates the transcription of genes involved in the unfolded protein response (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">S16</a>). STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) interactions of significant genes in the <b>5p</b> Bru-seq data set also demonstrated that genes affected were involved in protein folding, ER stress, and response to ER stress (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Table S13</a>). Affected UPR genes included <i>CALR</i> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D), <i>HSPA5</i> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>E), <i>MYZAP</i> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>F), <i>NQO1</i> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>G), and <i>SLC7A11</i> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>H). Calreticulin (<i>CALR</i>) is an ER chaperonelike PDI, specifically folds glycoproteins to be secreted, and mediates calcium homeostasis in the organelle.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Calreticulin acts as a sensor of ER stress because ER Ca<sup>2+</sup> depletion triggers ER stress. Calreticulin has been demonstrated to bind to ERp57 and regulates glycoprotein isomerization.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> However, <b>5p</b> treatment did not increase the total cellular CALR expression in brain cancer cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>I). <i>HSPA5</i> encodes for GRP78/BiP, an important chaperone responsible for promoting tumor growth.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Nascent polypeptides enter the ER and interact with GRP78/BiP to initiate protein folding. Increased transcription of GRP78/BiP indicates that the cells are undergoing an unfolded protein stress response. NAD(P)H quinone dehydrogenase 1 (<i>NQO1</i>) is a cytosolic quinone reductase that promotes quinone–glutathione conjugation and removal from the cells. It is generally highly expressed in cancers and allows the tumor to cope with increased cytotoxic stress.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a><i>SLC7A11</i> encodes for a cystine/glutamate antiporter protein that resides on the cell membrane. SLC7A11 is part of the system x<sub>c</sub><sup>–</sup> antiporter system that uptakes extracellular cystine as a precursor for GSH biosynthesis in exchange for glutamate.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Interestingly, we observed the upregulated transcription of <i>SLC7A11</i> upon treatment with PDI inhibitor <b>35G8</b> as well.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Our results indicate that PDI inhibition may be synthetically lethal with system x<sub>c</sub><sup>–</sup> inhibition. Myocardial zonula adherens (<i>MYZAP</i>) is part of a transcriptional unit containing downstream gene <i>POLR2M</i> (polymerase (RNA) II (DNA-directed) polypeptide M). MYZAP protein is expressed in the cardiac tissue and is involved in signaling via Rho-related GTP-binding proteins. The Bru-seq RNA sequencing genes affected support <b>5p</b>-mediated PDI inhibition in U-87 MG cells.</div><figure id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0010.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. <b>5p</b> upregulates the transcription of genes involved in the unfolded protein response. (a) Structure of <b>5p</b>. (b) <b>5p</b> promoted gene set enrichment related to the unfolded protein response. NES: normalized enrichment score. FDR <i>q</i> val: false discovery rate <i>q</i> value. The criteria for GSEA were <i>p</i> < 0.05 and false discovery rate <25%. (c) STRING interactions of significant genes in <b>5p</b> Bru-seq data set. A 4 h treatment with <b>5p</b> increases the transcription of representative unfolded protein response genes including <i>CALR</i> (d), <i>HSPA5</i> (e), <i>MYZAP</i> (f), <i>NQO1</i> (g), and <i>SLC7A11</i> (h). (i) U-118 MG cells were treated with 20 μM <b>5p</b> for 12, 24, or 48 h and probed for calreticulin protein expression. Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) is used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We further analyzed the Bru-seq signature of <b>5p</b> with the Connectivity Map (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Tables S14–S20</a>).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Because less than 10 genes were significantly downregulated upon <b>5p</b> treatment, the Connectivity Map analysis included only upregulated genes. <b>5p</b> had a similar gene expression signature as the seleno-organic glutathione peroxidase mimetic ebselen.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Ebselen is an antioxidant that is known to react with cysteines, and it targets GTPase protein Rac1 in humans.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> Interestingly, ebselen inhibits MIF tautomerase activity as well.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> This indicates that the signature of <b>5p</b> may be an artifact of global cysteine reactivity rather than selective target inhibition. Furthermore, the signature of <b>5p</b> demonstrated similarity with the knockdown of <i>KDELR3</i> (KDEL endoplasmic reticulum protein retention receptor 3). KDELR3 contains four cysteines and is upregulated as part of the unfolded protein response.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The protein is in a family of three KDEL receptors localized to the ER and Golgi complex. These results confirm that <b>5p</b> exhibits a cysteine-reactive signature in brain cancer cells.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43064" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43064" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Target engagement in cells is a critical aspect of preclinical targeted drug development. It is important to understand and verify that the compound can reach the target and that the interaction causes the observed phenotype. There are multiple techniques used to assess target engagement, including direct assays such as the cellular thermal shift assay (CETSA), drug affinity responsive target stability (DARTS), the NanoLuc thermal shift assay, and bioluminescence resonance energy transfer (BRET), or indirect methods such as knockdown effects or biomarker expression.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> To determine on-target labeling of PDI in cells, we synthesized two variations of BODIPY-labeled <b>AS15</b> analogues. Our initial discovery upon treating cells with these compounds was that the compounds bind to serum albumin, an abundant protein containing 35 cysteine residues. When the <b>AS15</b> analogues were incubated with the cell lysates, we observed binding in three major bands, indicating that the compounds bound to proteins around 57 and 40 kDa. While plasma protein binding is a consideration for the improvement of potency of this series, proteomics analysis demonstrated that the BODIPY-labeled <b>AS15</b> analogues reach and bind covalently to PDIA1 in the cells. Several PDI family members have a molecular weight of around 40 kDa, including ERp44, PDIA6, and TXNDC5, which could be responsible for the lower band (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>).</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. PDI Family Members and Molecular Weights</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">gene name</th><th class="colsep0 rowsep0" align="center" char=".">size (kDa)</th><th class="colsep0 rowsep0" align="center">gene name</th><th class="colsep0 rowsep0" align="center">size (kDa)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">P4HB</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="left">PDIA12 (TMX2)</td><td class="colsep0 rowsep0" align="left">34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA2</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="left">PDIA13 (TMX3)</td><td class="colsep0 rowsep0" align="left">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA3 (ERp57)</td><td class="colsep0 rowsep0" align="char" char=".">54</td><td class="colsep0 rowsep0" align="left">PDIA14 (TMX4)</td><td class="colsep0 rowsep0" align="left">39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA4 (ERp72)</td><td class="colsep0 rowsep0" align="char" char=".">71</td><td class="colsep0 rowsep0" align="left">(TMX5)</td><td class="colsep0 rowsep0" align="left">not reported</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA5 (PDIR)</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="left">PDIA15 (ERp46)</td><td class="colsep0 rowsep0" align="left">48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA6 (P5)</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="left">PDIA16 (ERp19, AGR1)</td><td class="colsep0 rowsep0" align="left">18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA7 (PDILT)</td><td class="colsep0 rowsep0" align="char" char=".">67</td><td class="colsep0 rowsep0" align="left">PDIA17 (AGR2, HAG-2)</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA8 (ERp27)</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="left">PDIA18 (AGR3, HAG-3)</td><td class="colsep0 rowsep0" align="left">19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA9 (ERp29)</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="left">PDIA19 (ERdj5)</td><td class="colsep0 rowsep0" align="left">91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA10 (ERp44)</td><td class="colsep0 rowsep0" align="char" char=".">44</td><td class="colsep0 rowsep0" align="left">PDIB1 (CASQ1)</td><td class="colsep0 rowsep0" align="left">45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDIA11 (TMX1)</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="left">PDIB2 (CASQ2)</td><td class="colsep0 rowsep0" align="left">46</td></tr></tbody></table></div></div><div class="NLM_p">These results corroborate previous findings with this Mannich base series. Targets identified for this series of compounds are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Table S21</a>. The frataxin inhibitor, which differs from <b>AS15</b> by only an <i>ortho</i> methoxy in place of the para methoxy, was nontoxic up to 100 μM in cells and dose-dependently prevented the ubiquitination of frataxin. The authors did not perform selectivity experiments since they were measuring a cellular protein function; however, they did show that the compound did not bind denatured protein. In the same year, another group reported a series of hydroxyquinolines similar to <b>CD343</b> as selective Mcl-1 inhibitors. With an SAR campaign, they demonstrated that the hydroxyl group and nitrogen were important for Mcl-1 activity. While we did not test their reported compound <b>9</b>, <b>5a</b> and <b>5b</b> were similar compounds, with the piperazine replaced by a morpholino group or pyrazine group. These compounds had submicromolar IC<sub>50</sub> values in the PDI reductase assay, similar to the IC<sub>50</sub> value against Mcl-1 in the fluorescence polarization assay.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In terms of selectivity, the researchers were able to demonstrate a correlation between a downstream response to Mcl-1 inhibition, cytochrome c release, and the extent of mitochondrial priming in cells.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Further targets of this series include HDAC5/9, STAT3/5, BRAF<sup>V600E</sup>, and P2Y12. <b>NC124</b> was highly potent against leukemia cell lines (THP-1 and KASUMI-1), though it was much less potent against U-87 MG cells (IC<sub>50</sub> = 24.2 ± 7.1 μM), possibly because TET1 expression is relatively lower in the U-87 MG cells.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Interestingly, several groups reported that this series of compounds passed protein reactivity filters.</div><div class="NLM_p last">Because the analogues we tested were inactivated by GSH addition in the PDI reductase assay, we tested whether GSH depletion in a cell-based assay would influence potency. The compounds were more potent when cells were treated with nontoxic concentrations of BSO, the glutathione synthesis inhibitor. This result suggests that either the compounds are being sequestered by GSH in the cytoplasm and unable to reach the target protein or that GSH depletion prevents PDI from the rescue. Thus, a potential strategy for further modification would include decreasing GSH reactivity of the series. The glutathione-mediated antioxidant defense system is upregulated in cancer cells compared to that in normal cells to mitigate the harmful byproducts of increased cell metabolism.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Increased concentrations of GSH are responsible for resistance to anticancer therapy. Temozolomide-resistant GBM tumors rely on glutathione antioxidant signaling pathways for survival.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Glutathione promotes metastasis in liver cancer, and overexpression of glutathione synthesis enzymes has been linked with drug resistance.<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a> Thus, glutathione depletion, in particular, with the irreversible glutamate cysteine ligase (GCL) inhibitor buthionine sulfoximine (BSO), may be a promising combinatorial approach with this series. Additionally, it should be noted that PDI is present in a mixed population in the cells under physiological conditions. Therefore, cell-based experiments have been performed with the endogenous population of PDI.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60017" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60017" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Starting from a lead compound containing a benzobenzoxole scaffold and morpholine moiety, we investigated modifications around the core. The trends in the structure–activity relationships of the analogues that a tertiary amine and hydroxyl group were critical for activity suggest that the potency of the inhibitors likely relies on the thiol-reactive characteristics. Protein mass spectrometry analysis further identified that <b>AS15</b> analogues bound covalently to PDI after displacement of the amine group. Though the compounds were potent <i>in vitro</i> inhibitors of PDI, glutathione inactivated the compounds, and BODIPY-labeled analogues bound serum albumin. Because of the binding pattern of the <b>AS15</b> analogues and reactivity with glutathione, they have the potential to be optimized as <i>in vivo</i> thiol-reactive inhibitors. Notably, this study is the first to explore the reactivity and regioselectivity of a variety of hydroxy bicyclic heterocycles in Betti-style reactions and the biological activity of their products. Furthermore, our in-depth mechanistic studies demonstrate that their mechanism of action is directly related to PDI inhibition.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18243" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18243" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Methods</h3><div class="NLM_p last">All commercial reagents and anhydrous solvents were purchased and used without purification, unless specified. Column chromatography was performed on a Biotage Isolera flash chromatography system on Biotage normal phase Silica Gel columns. Analytical thin-layer chromatography was performed on Merck precoated plates (Silica Gel 60 F<sub>254</sub>). NMR spectra were recorded on a Bruker Ultrashield 300 MHz or Bruker Ascend 400 MHz spectrometer using deuterated CDCl<sub>3</sub> or CD<sub>3</sub>OD as solvents. Chemical shifts for proton magnetic resonance spectra (<sup>1</sup>H NMR) are listed in parts per million (ppm) referenced to the appropriate solvent peak or 0.0 ppm for tetramethylsilane (TMS). The following abbreviations are used to describe the peak-splitting patterns when appropriate: br, broad; s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; and dd, doublet of doublets. Coupling constants, <i>J</i>, are reported in hertz (Hz). Mass spectra were recorded on a Shimadzu LCMS-2020 system using the electrospray ionization (ESI) ion source. High-performance liquid chromatography (HPLC) was used to determine the purity of biologically tested compounds using a Shimadzu LC-2030C 3D system on a Kinetex XB-C18 column (2.6 μm, 4.6 mm × 75 mm) under the following gradient elution conditions: acetonitrile/water (10–95%) or methanol/water (10–95%). The purity was established by integration of the areas of major peaks detected at 254 nm, and all tested compounds including the <b>NC</b> and <b>CD</b> series compounds have >95% purity.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Synthesis Procedure A</h3><div class="NLM_p last">Phenolic compound, amine, and aldehyde were dissolved in toluene. The mixture was heated at 100 °C for a specified time and monitored by thin-layer chromatography (TLC). After completion of the reaction, the mixture was concentrated, and the residue was purified by flash chromatography to give the target compound.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Synthesis Procedure B</h3><div class="NLM_p">Phenolic compound, amine, and aldehyde were dissolved in EtOH. The mixture was heated at 120 °C for 0.5 h under a microwave or refluxed overnight. After completion of the reaction, the mixture was concentrated, and the residue was purified by flash chromatography or recrystallized from methanol/acetone to give the target compound.</div><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 7-((2,3-Dimethoxyphenyl)(morpholino)methyl)quinolin-8-ol (<b>5a</b>)</h4><div class="NLM_p last">Prepared from 2,3-dimethoxybenzaldehyde (85 mg, 0.51 mmol), 8-hydroxyquinoline (50 mg, 0.34 mmol), and morpholine (44 mg, 0.51 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (EtOAc/MeOH = 10:1) to give compound <b>5a</b> as a pale yellow solid (16% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): 12.13 (s, 1H), 8.89 (dd, <i>J</i> = 4.2, 1.7 Hz, 1H), 8.05 (dd, <i>J</i> = 8.3, 1.7 Hz, 1H), 7.38 (dd, <i>J</i> = 8.3, 4.2 Hz, 1H), 7.35–7.18 (m, 4H), 6.99 (t, <i>J</i> = 8.0 Hz, 1H), 6.81 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 5.33 (s, 1H), 3.94 (s, 3H), 3.87 (s, 4H), 3.86–3.72 (m, 4H), 2.82–2.53 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 381.2 [M + H]<sup>+</sup>. Purity 97.7%.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 7-((4-Fluorophenyl)(piperidin-1-yl)methyl)quinolin-8-ol (<b>5b</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (103 mg, 0.83 mmol), 8-hydroxyquinoline (80 mg, 0.55 mmol), and piperidine (70 mg, 0.83 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (DCM/EtOAc/NH<sub>4</sub>OH = 5:1:0.1) to give compound <b>5b</b> as a pale yellow solid (98 mg, 53% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): 12.82 (s, 1H), 8.90 (d, <i>J</i> = 4.1 Hz, 1H), 8.04 (d, <i>J</i> = 8.3 Hz, 1H), 7.47 (dd, <i>J</i> = 8.3, 5.3 Hz, 2H), 7.37 (dd, <i>J</i> = 8.3, 4.2 Hz, 1H), 7.18 (s, 2H), 6.98 (t, <i>J</i> = 8.5 Hz, 2H), 4.66 (s, 1H), 2.76–2.35 (m, 4H), 1.84–1.39 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 337.2 [M + H]<sup>+</sup>. Purity 96.6%.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 7-((4-Fluorophenyl)(piperidin-1-yl)methyl)quinazolin-8-ol (<b>5c</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (39 mg, 0.32 mmol), 8-hydroxyquinazoline (30 mg, 0.21 mmol), and piperidine (27 mg, 0.32 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (EtOAc/MeOH = 100:1) to give compound <b>5c</b> as a white solid (30 mg, 42% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 13.68 (s, 1H), 9.33 (s, 1H), 9.28 (s, 1H), 7.43 (t, <i>J</i> = 6.5 Hz, 2H), 7.32–7.18 (m, 2H), 7.01 (t, <i>J</i> = 8.5 Hz, 2H), 4.67 (s, 1H), 2.78–2.22 (s, 4H), 1.84–1.48 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 338.5 [M + H]<sup>+</sup>. Purity 98.4%.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 7-((4-Fluorophenyl)(piperazin-1-yl)methyl)quinazolin-8-ol (<b>5d</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (51 mg, 0.41 mmol), 8-hydroxyquinazoline (40 mg, 0.28 mmol), and piperazine (36 mg, 0.41 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (EtOAc/MeOH = 100:1) to give compound <b>5d</b> as a white solid (6 mg, 6% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.31 (d, <i>J</i> = 6.2 Hz, 2H), 7.52–7.31 (m, 4H), 7.01 (t, <i>J</i> = 8.4 Hz, 2H), 4.76 (s, 1H), 3.07 (d, <i>J</i> = 11.5 Hz, 4H), 2.62 (d, <i>J</i> = 23.5 Hz, 4H); MS (ESI) <i>m</i>/<i>z</i> = 339.0 [M + H]<sup>+</sup>. Purity 97.3%.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 6-((4-Fluorophenyl)(piperidin-1-yl)methyl)quinoxalin-5-ol (<b>5e</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (102 mg, 0.82 mmol), quinoxalin-5-ol (80 mg, 0.55 mmol), and piperidine (71 mg, 0.82 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (DCM/MeOH = 100:1) to give compound <b>5e</b> as a pale yellow solid (46 mg, 25% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.84–8.77 (m, 2H), 7.44 (q, <i>J</i> = 8.0, 7.0 Hz, 3H), 7.34–7.28 (m, 1H), 7.00 (t, <i>J</i> = 8.4 Hz, 2H), 4.69 (s, 1H), 2.97–2.13 (m, 4H), 1.86–1.41 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 338.1 [M + H]<sup>+</sup>. Purity 100.0%.</div></div><div id="sec5_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 5-((4-Fluorophenyl)(piperidin-1-yl)methyl)benzo[<i>d</i>]thiazol-4-ol (<b>5f</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (50 mg, 0.40 mmol), benzo[<i>d</i>]thiazol-4-ol (50 mg, 0.33 mmol), and piperidine (34 mg, 0.40 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 1:1) to give compound <b>5f</b> as a white solid (46% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.89 (s, 1H), 7.43 (t, <i>J</i> = 6.9 Hz, 2H), 7.27 (d, <i>J</i> = 8.3 Hz, 1H), 6.99 (q, <i>J</i> = 8.7 Hz, 3H), 4.63 (s, 1H), 2.80–2.32 (m, 4H), 1.79–1.40 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 343.5 [M + H]<sup>+</sup>. Purity 96.3%.</div></div><div id="sec5_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-((4-Fluorophenyl)(piperidin-1-yl)methyl)benzofuran-7-ol (<b>5g</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (32 mg, 0.26 mmol), benzofuran-7-ol (35 mg, 0.26 mmol), and piperidine (27 mg, 0.31 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 10:1) to give compound <b>5g</b> as a white solid (12 mg, 14% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.63 (s, 1H), 7.42 (s, 2H), 7.07–6.91 (m, 3H), 6.76 (d, <i>J</i> = 8.2 Hz, 1H), 6.69 (s, 1H), 4.62 (s, 1H), 3.01–2.24 (m, 4H), 1.83–1.40 (m, 7H); MS (ESI) <i>m</i>/<i>z</i> = 326.1 [M + H]<sup>+</sup>. Purity 95.6%.</div></div><div id="sec5_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 6-((4-Fluorophenyl)(piperidin-1-yl)methyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-5-ol (<b>5h</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (62 mg, 0.50 mmol), 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-5-ol (50 mg, 0.33 mmol), and piperidine (43 mg, 0.50 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5h</b> as a white solid (84 mg, 74% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.44–7.30 (m, 2H), 6.99 (t, <i>J</i> = 8.5 Hz, 2H), 6.39–6.22 (m, 2H), 4.44 (s, 1H), 4.29 (ddd, <i>J</i> = 26.8, 6.1, 3.3 Hz, 4H), 2.98–2.18 (m, 4H), 1.77–1.36 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 344.4 [M + H]<sup>+</sup>. Purity 99.4%.</div></div><div id="sec5_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 5-((4-Fluorophenyl)(piperidin-1-yl)methyl)benzo[<i>d</i>][1,3]dioxol-4-ol (<b>5i</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (67 mg, 0.54 mmol), benzo[<i>d</i>][1,3]dioxol-4-ol (50 mg, 0.36 mmol), and piperidine (47 mg, 0.54 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 1:1) to give compound <b>5i</b> as a white solid (82 mg, 69% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.38 (t, <i>J</i> = 6.6 Hz, 2H), 7.01 (t, <i>J</i> = 8.7 Hz, 2H), 6.41–6.21 (m, 2H), 5.97 (dd, <i>J</i> = 9.9, 1.4 Hz, 2H), 4.49 (s, 1H), 2.93–2.18 (m, 4H), 1.75–1.38 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 330.5 [M + H]<sup>+</sup>. Purity 97.8%.</div></div><div id="sec5_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 6-((4-Fluorophenyl)(piperidin-1-yl)methyl)isoquinolin-5-ol (<b>5j</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (64 mg, 0.52 mmol), 5-hydroxyisoquinoline (50 mg, 0.34 mmol), and piperidine (44 mg, 0.52 mmol) in EtOH under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5j</b> as a white solid (37 mg, 32% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.12 (s, 1H), 8.52 (d, <i>J</i> = 5.9 Hz, 1H), 8.08 (d, <i>J</i> = 5.8 Hz, 1H), 7.39 (t, <i>J</i> = 7.1 Hz, 2H), 7.33 (d, <i>J</i> = 8.4 Hz, 1H), 7.10 (d, <i>J</i> = 8.4 Hz, 1H), 7.01 (t, <i>J</i> = 8.7 Hz, 2H), 4.60 (s, 1H), 3.15–2.11 (m, 4H), 1.86–1.40 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 337.1 [M + H]<sup>+</sup>. Purity 95.7%.</div></div><div id="sec5_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 5-((4-Fluorophenyl)(piperidin-1-yl)methyl)-2,3-dihydro-1<i>H</i>-inden-4-ol (<b>5k</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (74 mg, 0.60 mmol), 2,3-dihydro-1<i>H</i>-inden-4-ol (80 mg, 0.60 mmol), and piperidine (74 mg, 0.60 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5k</b> as a white solid (49 mg, 17% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 12.49 (s, 1H), 7.40 (t, <i>J</i> = 6.7 Hz, 2H), 7.01 (t, <i>J</i> = 8.8 Hz, 2H), 6.74–6.59 (m, 2H), 4.48 (s, 1H), 2.91 (dt, <i>J</i> = 19.3, 7.4 Hz, 4H), 2.42 (s, 4H), 2.11 (p, <i>J</i> = 7.4 Hz, 2H), 1.75–1.39 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 326.6 [M + H]<sup>+</sup>. Purity 100.0%.</div></div><div id="sec5_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-(Dimethylamino)-6-((4-fluorophenyl)(piperidin-1-yl)methyl)phenol (<b>5l</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (68 mg, 0.55 mmol), 2-(dimethylamino)phenol (50 mg, 0.36 mmol), and piperidine (47 mg, 0.55 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5l</b> as a white solid (20 mg, 17% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 12.73 (s, 1H), 7.42 (t, <i>J</i> = 6.7 Hz, 2H), 6.99 (t, <i>J</i> = 8.5 Hz, 2H), 6.84 (d, <i>J</i> = 7.8 Hz, 1H), 6.68 (t, <i>J</i> = 7.7 Hz, 1H), 6.60 (s, 1H), 4.44 (s, 1H), 2.84 (s, 6H), 2.73–2.19 (m, 4H), 1.80–1.35 (m, 6H). MS (ESI) <i>m</i>/<i>z</i> = 329.0 [M + H]<sup>+</sup>. Purity 100.0%.</div></div><div id="sec5_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-(6-((4-Fluorophenyl)(piperidin-1-yl)methyl)-7-hydroxybenzofuran-2-yl)ethanone (<b>5m</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (112 mg, 0.90 mmol), 1-(7-hydroxybenzofuran-2-yl)ethanone (160 mg, 0.90 mmol), and piperidine (92 mg, 1.10 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5m</b> as a white solid (110 mg, 33% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.43 (s, 3H), 7.08–6.95 (m, 3H), 6.84 (s, 1H), 4.64 (s, 1H), 2.68 (s, 3H), 2.63–2.27 (m, 4H), 1.82–1.37 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 368.3 [M + H]<sup>+</sup>. Purity 96.1%.</div></div><div id="sec5_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 6-((4-Fluoro-2-hydroxyphenyl)(piperidin-1-yl)methyl)benzofuran-7-ol (<b>5n</b>)</h4><div class="NLM_p last">Prepared from 4-fluoro-2-hydroxybenzaldehyde (57 mg, 0.41 mmol), benzofuran-7-ol (50 mg, 0.37 mmol), and piperidine in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5n</b> as a white solid (26 mg, 21% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.62 (d, <i>J</i> = 2.0 Hz, 1H), 7.03 (d, <i>J</i> = 7.8 Hz, 3H), 6.73 (d, <i>J</i> = 2.0 Hz, 1H), 6.62 (dd, <i>J</i> = 10.5, 2.5 Hz, 1H), 6.44 (dd, <i>J</i> = 9.6, 7.3 Hz, 1H), 5.22 (s, 1H), 3.05–2.35 (s, 4H), 1.79–1.41 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 342.2 [M + H]<sup>+</sup>. Purity 95.6%.</div></div><div id="sec5_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 6-((2,3-Dihydro-1<i>H</i>-inden-5-yl)(piperidin-1-yl)methyl)benzofuran-7-ol (<b>5o</b>)</h4><div class="NLM_p last">Prepared from 2,3-dihydro-1<i>H</i>-indene-5-carbaldehyde (82 mg, 0.56 mmol), benzofuran-7-ol (50 mg, 0.37 mmol), and piperidine (47 mg, 0.56 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5o</b> as a white solid (52 mg, 41% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.61 (d, <i>J</i> = 1.9 Hz, 1H), 7.35 (s, 1H), 7.24 (d, <i>J</i> = 8.0 Hz, 1H), 7.16 (d, <i>J</i> = 7.6 Hz, 1H), 6.99–6.82 (m, 2H), 6.68 (d, <i>J</i> = 2.0 Hz, 1H), 4.69 (s, 1H), 3.04–2.24 (m, 8H), 2.13–1.98 (m, 2H), 1.86–1.36 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 348.2 [M + H]<sup>+</sup>. Purity 96.8%.</div></div><div id="sec5_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 6-(Piperidin-1-yl(pyridin-4-yl)methyl)benzofuran-7-ol (<b>5p</b>)</h4><div class="NLM_p last">Prepared from isonicotinaldehyde (36 mg, 0.34 mmol), benzofuran-7-ol (30 mg, 0.22 mmol), and piperidine (29 mg, 0.34 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 1:1) to give compound <b>5p</b> as a white solid (36 mg, 53% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.55 (d, <i>J</i> = 5.2 Hz, 2H), 7.63 (d, <i>J</i> = 2.0 Hz, 1H), 7.41 (d, <i>J</i> = 5.0 Hz, 2H), 6.96 (d, <i>J</i> = 8.0 Hz, 1H), 6.77 (d, <i>J</i> = 8.0 Hz, 1H), 6.69 (d, <i>J</i> = 2.0 Hz, 1H), 4.55 (s, 1H), 2.91–2.36 (m, 4H), 1.81–1.43 (m, 7H); MS (ESI) <i>m</i>/<i>z</i> = 309.2 [M + H]<sup>+</sup>. Purity 99.8%.</div></div><div id="sec5_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 6-((2,2-Difluorobenzo[<i>d</i>][1,3]dioxol-5-yl)(piperidin-1-yl)methyl)benzofuran-7-ol (<b>5q</b>)</h4><div class="NLM_p last">Prepared from 2,2-difluorobenzo[<i>d</i>][1,3]dioxole-5-carbaldehyde (49 mg, 0.26 mmol), benzofuran-7-ol (30 mg, 0.22 mmol), and piperidine (29 mg, 0.34 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane: DCM = 9:1) to give compound <b>5q</b> as a white solid (16 mg, 19% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.64 (d, <i>J</i> = 2.2 Hz, 1H), 7.27 (d, <i>J</i> = 4.5 Hz, 1H), 7.14 (d, <i>J</i> = 8.4 Hz, 1H), 7.03–6.91 (m, 2H), 6.76 (d, <i>J</i> = 7.9 Hz, 1H), 6.70 (d, <i>J</i> = 2.1 Hz, 1H), 4.58 (s, 1H), 2.95–2.22 (s, 4H), 1.81–1.43 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 388.2 [M + H]<sup>+</sup>. Purity 99.6%.</div></div><div id="sec5_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 6-(Piperidin-1-yl(4-(trifluoromethyl)phenyl)methyl)benzofuran-7-ol (<b>5r</b>)</h4><div class="NLM_p last">Prepared from 4-(trifluoromethyl)benzaldehyde (59 mg, 0.34 mmol), benzofuran-7-ol (30 mg, 0.22 mmol), and piperidine (29 mg, 0.34 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>5r</b> as a white solid (28 mg, 34% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.67–7.49 (m, 5H), 6.93 (d, <i>J</i> = 8.0 Hz, 1H), 6.74 (d, <i>J</i> = 8.0 Hz, 1H), 6.67 (d, <i>J</i> = 2.1 Hz, 1H), 4.63 (s, 1H), 3.08–2.18 (m, 4H), 1.78–1.39 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 376.1 [M + H]<sup>+</sup>. Purity 99.8%.</div></div><div id="sec5_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 6-((2,3-Dimethoxyphenyl)(morpholino)methyl)benzo[<i>d</i>][1,3]dioxol-5-ol (<b>6a</b>)</h4><div class="NLM_p last">Prepared from 2,3-dimethoxybenzaldehyde (5.00 g, 30.1 mmol), benzo[<i>d</i>][1,3]dioxol-5-ol (4.16 g, 30.1 mmol), and morpholine (2.62 g, 30.1 mmol) in EtOH reflux overnight and recrystallized from MeOH/acetone to give compound <b>6a</b> as a white solid (6.40 g, 57% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 11.75 (s, 1H), 7.13 (d, <i>J</i> = 8.0 Hz, 1H), 7.00 (t, <i>J</i> = 8.0 Hz, 1H), 6.81 (d, <i>J</i> = 8.1 Hz, 1H), 6.42 (d, <i>J</i> = 11.5 Hz, 2H), 5.81 (dd, <i>J</i> = 13.0, 1.3 Hz, 2H), 4.98 (s, 1H), 3.99–3.83 (m, 6H), 3.79–3.64 (m, 4H), 2.83–2.36 (m, 4H); MS (ESI) <i>m</i>/<i>z</i> = 374.0 [M + H]<sup>+</sup>. Purity 95.5%.</div></div><div id="sec5_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 6-(Phenyl(piperidin-1-yl)methyl)benzo[<i>d</i>][1,3]dioxol-5-ol (<b>6b</b>)</h4><div class="NLM_p last">Prepared from benzaldehyde (115 mg, 1.09 mmol), benzo[<i>d</i>][1,3]dioxol-5-ol (100 mg, 0.72 mmol), and piperidine (93 mg, 1.09 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>6b</b> as a white solid (120 mg, 54% yield). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.12 (s, 1H), 7.39 (d, <i>J</i> = 7.6 Hz, 2H), 7.30 (t, <i>J =</i> 7.5 Hz, 2H), 7.27–7.15 (m, 1H), 6.65 (s, 1H), 6.36 (s, 1H), 5.84 (d, <i>J</i> = 3.3 Hz, 2H), 4.50 (s, 1H), 2.47–2.21 (m, 4H), 1.60–1.33 (m, 6H). MS (ESI) <i>m</i>/<i>z</i> = 312.0 [M + H]<sup>+</sup>. Purity 99.6%.</div></div><div id="sec5_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 7-((4-Fluorophenyl)(piperidin-1-yl)methyl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-ol (<b>6c</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (45 mg, 0.36 mmol), 2,3-dihydrobenzo[<i>b</i>][1,4]dioxin-6-ol (50 mg, 0.33 mmol), and piperidine (31 mg, 0.36 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>6c</b> as a white solid (40 mg, 35% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 12.04 (s, 1H), 7.37 (t, <i>J</i> = 6.8 Hz, 2H), 7.01 (t, <i>J</i> = 8.5 Hz, 2H), 6.41 (s, 2H), 4.40 (s, 1H), 4.26–4.10 (m, 4H), 2.80–2.21 (m, 4H), 1.78–1.39 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 344.2 [M + H]<sup>+</sup>. Purity 100.0%.</div></div><div id="sec5_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 6-((4-Fluorophenyl)(piperidin-1-yl)methyl)indolin-5-ol (<b>6d</b>)</h4><div class="NLM_p last">Starting from 4-fluorobenzaldehyde (29 mg, 0.23 mmol), <i>tert</i>-butyl 5-hydroxyindoline-1-carboxylate (50 mg, 0.21 mmol), and piperidine (20 mg, 0.23 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 1:1) to give <i>tert</i>-butyl 6-((4-fluorophenyl)(piperidin-1-yl)methyl)-5-hydroxyindoline-1-carboxylate (34 mg, 38% yield), which was directly dissolved in DCM (3 mL), and a solution of HCl in dioxane (4N, 0.6 mL) was added. The mixture was stirred at room temperature for 3h, concentrated, and purified by silica column chromatography (DCM/MeOH = 30:1) to give <b>6d</b> as a white solid (12 mg, 46% yield). <sup>1</sup>H NMR (300 MHz, MeOD) δ 7.97 (s, 1H), 7.87 (t, <i>J</i> = 6.6 Hz, 2H), 7.20 (t, <i>J</i> = 8.3 Hz, 2H), 7.06 (s, 1H), 5.83 (s, 1H), 3.92–3.46 (m, 5H), 3.26–3.32 (m, 1H), 3.18–2.99 (m, 2H), 2.10–1.79 (m, 5H), 1.68–1.53 (m, 1H); MS (ESI) <i>m</i>/<i>z</i> = 327.1 [M + H]<sup>+</sup>. Purity 96.6%.</div></div><div id="sec5_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 6-((2,3-Dimethoxyphenyl)(morpholino)methyl)-2,2-difluorobenzo[<i>d</i>][1,3]dioxol-5-ol (<b>6e</b>)</h4><div class="NLM_p last">Prepared from 2,3-dimethoxybenzaldehyde (52 mg, 0.32 mmol), 2,2-difluorobenzo[<i>d</i>][1,3]dioxol-5-ol (55 mg, 0.32 mmol), and morpholine (27 mg, 0.32 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 1:1) to give compound <b>6e</b> as a white solid (18 mg, 14% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 12.23 (s, 1H), 7.15–7.08 (m, 1H), 7.04 (t, <i>J</i> = 7.9 Hz, 1H), 6.86 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 6.70 (s, 1H), 6.60 (s, 1H), 5.10 (s, 1H), 3.93 (s, 3H), 3.90 (s, 3H), 3.82–3.73 (m, 4H), 2.91–2.45 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 410.5 [M + H]<sup>+</sup>. Purity 95.9%.</div></div><div id="sec5_3_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 6-((4-Fluorophenyl)(piperidin-1-yl)methyl)-2,3-dihydro-1<i>H</i>-inden-5-ol (<b>6f</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (89 mg, 0.72 mmol), 2,3-dihydro-1<i>H</i>-inden-5-ol (80 mg, 0.60 mmol), and piperidine (61 mg, 0.72 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 10:1) to give compound <b>6f</b> as a white solid (94 mg, 48% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 12.23 (s, 1H), 7.42 (t, <i>J</i> = 6.6 Hz, 2H), 7.01 (t, <i>J</i> = 8.5 Hz, 2H), 6.76 (d, <i>J</i> = 8.2 Hz, 2H), 4.41 (s, 1H), 2.84 (t, <i>J</i> = 7.5 Hz, 2H), 2.74 (t, <i>J</i> = 7.7 Hz, 2H), 2.40 (s, 4H), 2.10–1.96 (m, 2H), 1.76–1.40 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 326.1 [M + H]<sup>+</sup>. Purity 98.3%.</div></div><div id="sec5_3_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 3-((4-Fluorophenyl)(piperidin-1-yl)methyl)-5,6,7,8-tetrahydronaphthalen-2-ol (<b>6g</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (63 mg, 0.51 mmol), 5,6,7,8-tetrahydronaphthalen-2-ol (50 mg, 0.34 mmol), and piperidine (44 mg, 0.51 mmol) in EtOH under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>6g</b> as a white solid (10 mg, 9% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.40 (s, 2H), 7.01 (t, <i>J</i> = 8.5 Hz, 2H), 6.59 (s, 2H), 4.39 (s, 1H), 2.80–2.26 (m, 8H), 1.78–1.42 (m, 10H); MS (ESI) <i>m</i>/<i>z</i> = 340.2 [M + H]<sup>+</sup>. Purity 96.4%.</div></div><div id="sec5_3_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 5-((4-Fluorophenyl)(piperidin-1-yl)methyl)quinolin-6-ol (<b>7a</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (53 mg, 0.62 mmol), 6-hydroxyquinoline (60 mg, 0.41 mmol), and piperidine (77 mg, 0.62 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>7a</b> as a white solid (30 mg, 22% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 13.99 (s, 1H), 8.64 (dd, <i>J</i> = 4.1, 1.4 Hz, 1H), 8.11 (d, <i>J</i> = 8.6 Hz, 1H), 7.94 (d, <i>J</i> = 9.1 Hz, 1H), 7.54–7.43 (m, 2H), 7.38 (d, <i>J</i> = 9.1 Hz, 1H), 7.24 (dd, <i>J</i> = 8.7, 4.2 Hz, 1H), 6.95 (t, <i>J</i> = 8.5 Hz, 2H), 5.00 (s, 1H), 3.33–1.48 (s, 10H); MS (ESI) <i>m</i>/<i>z</i> = 337.2 [M + H]<sup>+</sup>. Purity 97.4%.</div></div><div id="sec5_3_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-((4-Fluorophenyl)(piperidin-1-yl)methyl)benzo[<i>d</i>]thiazol-5-ol (<b>7b</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (50 mg, 0.40 mmol), 5-hydroxybenzothiazole (50 mg, 0.33 mmol), and piperidine (34 mg, 0.40 mmol) in EtOH under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>7b</b> as a white solid (60 mg, 53% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 12.77 (s, 1H), 8.68 (s, 1H), 7.88 (d, <i>J</i> = 8.8 Hz, 1H), 7.49 (t, <i>J</i> = 6.8 Hz, 2H), 7.11 (d, <i>J</i> = 8.8 Hz, 1H), 6.99 (t, <i>J</i> = 8.6 Hz, 2H), 4.45 (s, 1H), 2.78–2.19 (s, 4H), 1.80–1.32 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 343.1 [M + H]<sup>+</sup>. Purity 96.4%.</div></div><div id="sec5_3_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 5-((4-Fluorophenyl)(piperidin-1-yl)methyl)quinoxalin-6-ol (<b>7c</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (64 mg, 0.52 mmol), quinoxaline-6-ol (50 mg, 0.34 mmol), and piperidine (44 mg, 0.52 mmol) in EtOH under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>7c</b> as a white solid (32 mg, 28% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.66 (d, <i>J</i> = 1.9 Hz, 1H), 8.59 (d, <i>J</i> = 1.9 Hz, 1H), 7.90 (d, <i>J</i> = 9.1 Hz, 1H), 7.61 (dd, <i>J</i> = 8.5, 5.5 Hz, 2H), 7.48 (d, <i>J</i> = 9.2 Hz, 1H), 6.94 (t, <i>J</i> = 8.6 Hz, 2H), 5.84 (s, 1H), 3.61–1.21 (m, 10H); MS (ESI) <i>m</i>/<i>z</i> = 338.4 [M + H]<sup>+</sup>. Purity 98.6%.</div></div><div id="sec5_3_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 8-((4-Fluorophenyl)(piperidin-1-yl)methyl)isoquinolin-7-ol (<b>7d</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (89 mg, 0.72 mmol), isoquinolin-7-ol (80 mg, 0.48 mmol), and piperidine in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 3:1) to give compound <b>7d</b> as a white solid (94 mg, 58% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 14.24 (s, 1H), 9.31 (s, 1H), 8.33 (d, <i>J</i> = 5.4 Hz, 1H), 7.67 (d, <i>J</i> = 8.8 Hz, 1H), 7.53 (t, <i>J</i> = 8.0 Hz, 3H), 7.38 (d, <i>J</i> = 8.8 Hz, 1H), 6.98 (t, <i>J</i> = 8.1 Hz, 2H), 5.24 (s, 1H), 3.30 (s, 1H), 2.76–1.52 (m, 9H); MS (ESI) <i>m</i>/<i>z</i> = 337.1 [M + H]<sup>+</sup>. Purity 97.4%.</div></div><div id="sec5_3_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 5-((4-Fluorophenyl)(piperidin-1-yl)methyl)isoquinolin-6-ol (<b>7e</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (89 mg, 0.72 mmol), isoquinolin-6-ol (80 mg, 0.48 mmol), and piperidine (62 mg, 0.72 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 3:1) to give compound <b>7e</b> as a white solid (103 mg, 64% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 14.59 (s, 1H), 9.00 (s, 1H), 8.33 (d, <i>J</i> = 6.1 Hz, 1H), 7.80 (d, <i>J</i> = 8.9 Hz, 1H), 7.58–7.43 (m, 3H), 7.24 (d, <i>J</i> = 9.0 Hz, 1H), 6.99 (t, <i>J</i> = 8.5 Hz, 2H), 5.02 (s, 1H), 3.28 (s, 1H), 2.88–1.32 (m, 9H); MS (ESI) <i>m</i>/<i>z</i> = 337.1 [M + H]<sup>+</sup>. Purity 100.0%.</div></div><div id="sec5_3_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 4-((4-Fluorophenyl)(piperidin-1-yl)methyl)-1<i>H</i>-indol-5-ol (<b>7f</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (112 mg, 0.90 mmol), 1<i>H</i>-indol-5-ol (80 mg, 0.60 mmol), and piperidine (77 mg, 0.90 mmol) under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>7f</b> as a white solid (26 mg, 13% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.07 (s, 1H), 7.55 (s, 2H), 7.18 (d, <i>J</i> = 8.5 Hz, 1H), 7.11 (d, <i>J</i> = 2.8 Hz, 1H), 7.01–6.84 (m, 3H), 6.38 (s, 1H), 4.81– 4.71 (m, 1H), 3.12–2.11 (m, 4H), 1.78–1.46 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 325.1 [M + H]<sup>+</sup>. Purity 96.5%.</div></div><div id="sec5_3_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 4-((4-Fluorophenyl)(piperidin-1-yl)methyl)-1<i>H</i>-indazol-5-ol (<b>7g</b>)</h4><div class="NLM_p last">Prepared from 4-fluorobenzaldehyde (69 mg, 0.56 mmol), 1<i>H</i>-indazol-5-ol (50 mg, 0.37 mmol), and piperidine (48 mg, 0.56 mmol) in EtOH under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 3:1) to give compound <b>7g</b> as a white solid (32 mg, 27% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 12.24 (s, 1H), 7.85 (s, 1H), 7.49 (t, <i>J</i> = 6.7 Hz, 2H), 7.26 (d, <i>J</i> = 10.0 Hz, 1H), 6.99 (dt, <i>J</i> = 17.0, 8.7 Hz, 3H), 4.74 (s, 1H), 2.94–2.20 (m, 4H), 1.83–1.37 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 326.2 [M + H]<sup>+</sup>. Purity 95.2%.</div></div><div id="sec5_3_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-(Piperidin-1-yl(4-(trifluoromethyl)phenyl)methyl)-1<i>H</i>-indazol-5-ol (<b>7h</b>)</h4><div class="NLM_p last">Prepared from 4-(trifluoromethyl)benzaldehyde (97 mg, 0.56 mmol), 1<i>H</i>-indazol-5-ol (50 mg, 0.37 mmol), and piperidine (48 mg, 0.56 mmol) under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 3:1) to give compound <b>7h</b> as a white solid (90 mg, 65% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.88 (s, 1H), 7.68 (d, <i>J</i> = 7.7 Hz, 2H), 7.55 (d, <i>J</i> = 8.0 Hz, 2H), 7.32–7.23 (m, 1H), 7.11–6.99 (m, 1H), 4.81 (s, 1H), 2.78–2.11 (m, 4H), 1.83–1.41 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 376.1 [M + H]<sup>+</sup>. Purity 99.6%.</div></div><div id="sec5_3_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 4-((4-Nitrophenyl)(piperidin-1-yl)methyl)-1<i>H</i>-indazol-5-ol (<b>7i</b>)</h4><div class="NLM_p last">Prepared from 4-nitrobenzaldehyde (85 mg, 0.56 mmol), 1<i>H</i>-indazol-5-ol (50 mg, 0.37 mmol), and piperidine (48 mg, 0.56 mmol) under a microwave at 120 °C for 45 min. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 3:1) to give compound <b>7i</b> as a white solid (87 mg, 67% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.20–8.11 (m, 2H), 7.89 (s, 1H), 7.74 (d, <i>J</i> = 8.2 Hz, 2H), 7.31 (s, 2H), 7.05 (d, <i>J</i> = 9.1 Hz, 1H), 4.85 (s, 1H), 2.87–2.25 (m, 4H), 1.82–1.33 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 353.2 [M + H]<sup>+</sup>. Purity 96.5%.</div></div><div id="sec5_3_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 4-(Piperidin-1-yl(4-(thiazol-2-yl)phenyl)methyl)-1<i>H</i>-indazol-5-ol (<b>7j</b>)</h4><div class="NLM_p last">Prepared from 4-(thiazol-2-yl)benzaldehyde (84 mg, 0.44 mmol), 1<i>H</i>-indazol-5-ol (50 mg, 0.37 mmol), and piperidine (48 mg, 0.56 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>7j</b> as a white solid (102 mg, 70% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.93–7.79 (m, 4H), 7.60 (d, <i>J</i> = 8.0 Hz, 2H), 7.27 (d, <i>J</i> = 3.5 Hz, 1H), 7.23 (d, <i>J</i> = 8.8 Hz, 1H), 7.01 (d, <i>J</i> = 8.8 Hz, 1H), 4.79 (s, 1H), 3.12–2.08 (m, 4H), 1.83–1.39 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 391.5 [M + H]<sup>+</sup>. Purity 95.2%.</div></div><div id="sec5_3_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 4-((4-(1<i>H</i>-Pyrrol-1-yl)phenyl)(piperidin-1-yl)methyl)-1<i>H</i>-indazol-5-ol (<b>7k</b>)</h4><div class="NLM_p last">Prepared from 4-(1<i>H</i>-pyrrol-1-yl)benzaldehyde (96 mg, 0.56 mmol), 1<i>H</i>-indazol-5-ol (50 mg, 0.37 mmol), and piperidine (48 mg, 0.56 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>7k</b> as a white solid (86 mg, 62% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.90 (s, 1H), 7.58 (d, <i>J</i> = 7.9 Hz, 2H), 7.29 (t, <i>J</i> = 8.3 Hz, 3H), 7.10–6.99 (m, 3H), 6.32 (d, <i>J</i> = 2.3 Hz, 2H), 4.79 (s, 1H), 3.33–2.16 (m, 4H), 1.88–1.42 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 373.1 [M + H]<sup>+</sup>. Purity 98.9%.</div></div><div id="sec5_3_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 4-(Piperidin-1-yl(4-(pyrimidin-5-yl)phenyl)methyl)-1<i>H</i>-indazol-5-ol (<b>7l</b>)</h4><div class="NLM_p last">Prepared from 4-(pyrimidin-5-yl)benzaldehyde (103 mg, 0.56 mmol), 1<i>H</i>-indazol-5-ol (50 mg, 0.37 mmol), and piperidine (48 mg, 0.56 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give compound <b>7l</b> as a white solid (52 mg, 37% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.19 (d, <i>J</i> = 1.6 Hz, 1H), 8.89 (d, <i>J</i> = 1.6 Hz, 2H), 7.92 (s, 1H), 7.70 (d, <i>J</i> = 7.8 Hz, 2H), 7.49 (d, <i>J</i> = 7.9 Hz, 2H), 7.27 (d, <i>J</i> = 7.9 Hz, 1H), 7.05 (dd, <i>J</i> = 8.9, 1.6 Hz, 1H), 4.82 (s, 1H), 3.12–2.23 (m, 4H), 1.84–1.39 (m, 6H); MS (ESI) <i>m</i>/<i>z</i> = 386.1 [M + H]<sup>+</sup>. Purity 97.3%.</div></div><div id="sec5_3_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 6-((4-(Dimethylamino)phenyl)(pyrimidin-2-ylamino)methyl)benzo[<i>d</i>][1,3]dioxol-5-ol (<b>14</b>)</h4><div class="NLM_p last">Prepared from 4-(dimethylamino)benzaldehyde (149 mg, 1.00 mmol), benzo[<i>d</i>][1,3]dioxol-5-ol (138 mg, 1.00 mmol), and 2-aminopyrimidine (95 mg, 1.00 mmol) in toluene at 100 °C overnight. After concentration, the residue was purified by silica column chromatography (hexane/EtOAc = 2:1) to give compound <b>14</b> as a white solid (73 mg, 20% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 10.11 (s, 1H), 8.30 (d, <i>J</i> = 4.9 Hz, 2H), 7.25 (d, <i>J</i> = 8.5 Hz, 2H), 6.60 (t, <i>J</i> = 4.9 Hz, 1H), 6.50 (d, <i>J</i> = 4.3 Hz, 2H), 6.44–6.31 (m, 2H), 5.89–5.79 (m, 2H), 2.96 (s, 6H); MS (ESI) <i>m</i>/<i>z</i> = 365.2 [M + H]<sup>+</sup>. Purity 96.6%.</div></div><div id="sec5_3_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 1-((4-Fluorophenyl)(7-methoxybenzofuran-6-yl)methyl)piperidine (<b>8</b>)</h4><div class="NLM_p last"><b>5g</b> (32 mg, 0.098 mmol) was dissolved in acetone (3 mL), and K<sub>2</sub>CO<sub>3</sub> (41 mg, 0.294 mmol) and Me<sub>2</sub>SO<sub>4</sub> (19 mg, 0.148 mmol) were added. The mixture was stirred at room temperature for 6 h and concentrated. The residue was purified by silica column chromatography (hexane/EtOAc = 5:1) to give <b>8</b> as a white solid (10 mg, 30% yield). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.57 (d, <i>J</i> = 2.2 Hz, 1H), 7.51–7.40 (m, 3H), 7.24 (d, <i>J</i> = 8.1 Hz, 1H), 6.95 (t, <i>J</i> = 8.7 Hz, 2H), 6.71 (d, <i>J</i> = 2.2 Hz, 1H), 4.87 (s, 1H), 4.11 (s, 3H), 2.45–2.25 (m, 4H), 1.66–1.42 (m, 6H). MS (ESI) <i>m</i>/<i>z</i> = 340.3 [M + H]<sup>+</sup>. Purity 98.2%.</div></div><div id="sec5_3_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (6-Hydroxybenzo[<i>d</i>][1,3]dioxol-5-yl)(phenyl)methanone (<b>10</b>)</h4><div class="NLM_p last">MeI (1.29 g, 9.05 mmol) was added to a suspension of Mg beads (130 mg, 5.43 mmol) in dry Et<sub>2</sub>O (3 mL) at 35 °C under argon. When the Mg beads were dissolved, a solution of benzo[<i>d</i>][1,3]dioxol-5-ol (<b>9</b>, 511 mg, 3.62 mmol) in Et<sub>2</sub>O (2 mL) was added, and the resulting solution was stirred at room temperature for 1 h. The solvent was then evaporated, and toluene (3 mL) was added. Benzoyl chloride (511 mg, 3.62 mmol) in toluene was added dropwise at 0 °C. The mixture was stirred at room temperature for 12 h and quenched with sat. NH<sub>4</sub>Cl. The product was extracted with EtOAc, and the organic layer was washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The crude product was purified by flash chromatography (10% EtOAc in hexane) to give <b>10</b> as a white solid (570 mg, 65%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 13.03 (s, 1H), 7.67–7.47 (m, 5H), 6.97 (s, 1H), 6.57 (s, 1H), 6.01 (s, 2H).</div></div><div id="sec5_3_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 6-(Cyclohexyl(hydroxy)(phenyl)methyl)benzo[<i>d</i>][1,3]dioxol-5-ol (<b>11</b>)</h4><div class="NLM_p last">To an ice-cooled solution of <b>10</b> in Et<sub>2</sub>O was added cyclohexylmagnesium bromide (0.58 mL, 0.58 mmol, 1 M in THF) dropwise under argon. The mixture was heated at 50 °C for 3 h and cooled. Saturated NH<sub>4</sub>Cl was added to quench the reaction, and the mixture was extracted by EtOAc. The organic layer was washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The crude product was purified by flash chromatography (20% EtOAc in hexane) to give <b>11</b> as a white solid (30 mg, 54%). <sup>1</sup>H NMR (300 MHz, MeOD) δ 7.42 (d, <i>J</i> = 7.8 Hz, 2H), 7.27 (t, <i>J</i> = 7.6 Hz, 2H), 7.16 (t, <i>J</i> = 7.3 Hz, 1H), 6.71 (s, 1H), 6.23 (d, <i>J</i> = 1.0 Hz, 1H), 5.83 (d, <i>J</i> = 6.6 Hz, 2H), 2.33 (t, <i>J</i> = 10.3 Hz, 1H), 1.91–1.65 (m, 4H), 1.43–1.10 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.00, 147.24, 144.44, 140.46, 128.20, 126.95, 125.61, 119.53, 106.48, 101.00, 99.60, 84.04, 46.18, 27.47, 26.65, 26.59, 26.54, 26.42. MS (ESI) <i>m</i>/<i>z</i> = 325.3 [M – H]<sup>−</sup>.</div></div><div id="sec5_3_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 6-(Cyclohexylidene(phenyl)methyl)benzo[<i>d</i>][1,3]dioxol-5-ol (<b>12</b>)</h4><div class="NLM_p last">To a solution of <b>11</b> (12 mg, 0.037 mmol) in toluene was added TsOH (1 mg) and stirred at room temperature for 2 h. The mixture was concentrated and purified by flash chromatography (10% EtOAc in hexane) to give <b>12</b> as a light yellow solid (11 mg, 96%). <sup>1</sup>H NMR (300 MHz, MeOD) δ 7.30–7.09 (m, 5H), 6.38 (s, 2H), 5.82 (d, <i>J</i> = 5.5 Hz, 2H), 2.40–2.27 (m, 1H), 2.27–2.08 (m, 3H), 1.78–1.44 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 147.29, 147.12, 147.03, 143.89, 141.34, 141.13, 129.24, 128.22, 126.68, 120.30, 109.24, 100.95, 97.17, 32.64, 31.93, 28.79, 28.41, 26.55.</div></div><div id="sec5_3_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 6-(Cyclohexyl(phenyl)methyl)benzo[<i>d</i>][1,3]dioxol-5-ol (<b>13</b>)</h4><div class="NLM_p last">To a solution of <b>8</b> (46 mg, 0.15 mmol) in EtOH (5 mL) was added Pd/C (9.2 mg). The mixture was heated at 60 °C under H<sub>2</sub> for 3 h, cooled, and filtered. The filtrate was concentrated and purified by flash chromatography (20% EtOAc in hexane) to give <b>13</b> as a white solid (36 mg, 77%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.29 (d, <i>J</i> = 4.3 Hz, 4H), 7.18 (p, <i>J</i> = 4.4 Hz, 1H), 6.86 (s, 1H), 6.33 (s, 1H), 5.89 (dd, <i>J</i> = 16.8, 1.4 Hz, 2H), 4.59 (s, 1H), 3.87 (d, <i>J</i> = 10.8 Hz, 1H), 2.12–2.00 (m, 1H), 1.85–1.64 (m, 3H), 1.61–1.53 (m, 1H), 1.36–1.13 (m, 4H), 1.03–0.86 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> = 311.0 [M – H]<sup>−</sup>. Purity 97.3%.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Synthesis of BODIPY-Labeled Compounds</h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 4-((6-Hydroxybenzo[<i>d</i>][1,3]dioxol-5-yl)(piperidin-1-yl)methyl)benzoic Acid (<b>S4</b>)</h4><div class="NLM_p last">A mixture of benzo[<i>d</i>][1,3]dioxol-5-ol (1.00 g, 7.25 mmol), 4-formylbenzoic acid (1.09 g, 7.25 mmol), and piperidine (0.62 g, 7.25 mmol) was heated in toluene at 100 °C overnight. The precipitate generated was filtered, washed with toluene three times, and dried to give <b>S4</b> as a white solid (2.10 g, 82%). <sup>1</sup>H NMR (300 MHz, MeOD) δ 7.89 (d, <i>J</i> = 7.9 Hz, 2H), 7.42 (d, <i>J</i> = 7.9 Hz, 2H), 6.45 (d, <i>J</i> = 1.8 Hz, 1H), 6.34 (d, <i>J</i> = 1.9 Hz, 1H), 5.79 (d, <i>J</i> = 9.1 Hz, 2H), 4.49 (s, 1H), 2.73–2.31 (m, 4H), 1.68–1.39 (m, 6H). MS (ESI) <i>m</i>/<i>z</i> = 356.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>N</i>-(4-(5,5-Difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl)phenyl)-4-((6-hydroxybenzo[<i>d</i>][1,3]dioxol-5-yl)(piperidin-1-yl)methyl)benzamide (<b>15</b>)</h4><div class="NLM_p last">To a solution of <b>S4</b> (30 mg, 0.085 mmol) in DMF (2 mL) were added EDCI (13 mg, 0.085 mmol) and HOBt (11 mg, 0.085 mmol) followed by 4-(5,5-difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl)aniline (BODIPY aniline, 29 mg, 0.085 mmol). The mixture was stirred at room temperature overnight and partitioned between EtOAc and water. The organic layer was washed with water twice, brine once, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The mixture was then filtered and concentrated. The residue was purified by prep-HPLC (acetonitrile/water (10–95%) over 25 min at an 8 mL/min flow rate at room temperature) to give <b>15</b> as a red solid (8.6 mg, 15%). <sup>1</sup>H NMR (300 MHz, DMSO) δ 9.51 (s, 1H), 7.40 (d, <i>J</i> = 7.9 Hz, 2H), 7.31 (d, <i>J</i> = 7.8 Hz, 2H), 6.95 (d, <i>J</i> = 8.1 Hz, 2H), 6.83–6.71 (m, 3H), 6.63 (d, <i>J</i> = 7.5 Hz, 1H), 6.46 (s, 1H), 6.13 (s, 2H), 5.89 (dd, <i>J</i> = 15.7, 6.4 Hz, 3H), 3.31–3.19 (m, 4H), 2.42 (s, 6H), 1.66–1.46 (m, 6H), 1.41 (s, 6H). MS (ESI) <i>m</i>/<i>z</i> = 675.5 [M – H]<sup>−</sup>. Purity 98.1%.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>tert</i>-Butyl (2-(4-((6-hydroxybenzo[<i>d</i>][1,3]dioxol-5-yl)(piperidin-1-yl)methyl)benzamido)ethyl)carbamate (<b>S5</b>)</h4><div class="NLM_p last">To a solution of <b>S4</b> (200 mg, 0.56 mmol) and <i>tert</i>-butyl (2-aminoethyl)carbamate (108 mg, 0.68 mmol) in DMF (1 mL) were added HATU (319 mg, 0.84 mmol) and DIEA (217 mg, 1.68 mmol). The mixture was stirred at room temperature overnight and partitioned between EtOAc and water. The organic layer was washed with water twice, brine once, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The mixture was then filtered and concentrated. The residue was purified by flash chromatography (30% EtOAc in hexane) to give <b>S5</b> as a light yellow solid (168 mg, 60%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.82 (s, 1H), 8.38 (d, <i>J</i> = 5.5 Hz, 1H), 7.75 (d, <i>J</i> = 8.0 Hz, 2H), 7.45 (d, <i>J</i> = 8.1 Hz, 2H), 6.89 (t, <i>J</i> = 5.7 Hz, 1H), 6.68 (s, 1H), 6.37 (s, 1H), 5.84 (s, 2H), 4.59 (s, 1H), 3.26 (d, <i>J</i> = 6.1 Hz, 2H), 3.07 (d, <i>J</i> = 6.2 Hz, 2H), 2.46–2.18 (m, 4H), 1.59–1.41 (m, 6H), 1.36 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> = 498.4 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>N</i>-(4-(5,5-Difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl))-<i>N</i>-(2-(4-((6-hydroxybenzo[<i>d</i>][1,3]dioxol-5-yl)(piperidin-1-yl)methyl)benzamido)ethyl)benzamide (<b>16</b>)</h4><div class="NLM_p last">To a solution of <b>S5</b> (11 mg, 0.022 mmol) in DCM (0.5 mL) was added TFA (0.05 mL), and the mixture was stirred at room temperature for 3 h. The mixture was concentrated and dissolved in DMF (0.5 mL). 4-(5,5-Difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl)benzoic acid (8 mg, 0.022 mmol, synthesized by the reported method), HATU (12.5 mg, 0.033 mmol), and DIEA (8.5 mg, 0.066 mmol) were added subsequently. The mixture was stirred at room temperature overnight and partitioned between EtOAc and water. The organic layer was washed with water twice, brine once, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The mixture was then filtered and concentrated. The residue was purified by prep-HPLC (acetonitrile/water (10–95%) over 25 min at an 8 mL/min flow rate at room temperature) to give <b>16</b> as a red solid (3.0 mg, 18%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.75 (s, 1H), 8.55 (s, 1H), 8.02 (d, <i>J</i> = 8.0 Hz, 2H), 7.77 (d, <i>J</i> = 8.0 Hz, 2H), 7.48 (dd, <i>J</i> = 12.5, 8.1 Hz, 4H), 6.68 (s, 1H), 6.37 (s, 1H), 6.20 (s, 2H), 5.83 (d, <i>J</i> = 1.2 Hz, 2H), 4.59 (s, 1H), 2.70–2.66 (m, 2H), 2.46 (s, 3H), 2.35–2.32 (m, 2H), 2.28 (dd, <i>J</i> = 11.9, 5.2 Hz, 4H), 1.55–1.38 (m, 6H), 1.33 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> = 748.2 [M + H]<sup>+</sup>. Purity 96.1%.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> Cell Culture</h3><div class="NLM_p last">U-87 MG, U-118 MG, A-172, and MIA PaCa-2 were obtained from the ATCC (Manassas, VA), and U-118 MG, A-172, and MIA PaCa-2 were maintained in RPMI-1640 (Thermo Fisher Scientific, Waltham, MA) supplemented with 10% fetal bovine serum (FBS) (Thermo Fisher Scientific). Dulbecco′s phosphate-buffered saline (DPBS) was purchased from Thermo Fisher Scientific. U-87 MG cells were maintained in DMEM supplemented with 10% FBS. The cells were grown as monolayer cultures at 37 °C in a humidified atmosphere of 5% CO<sub>2</sub> and tested for <i>Mycoplasma</i> contamination with the <i>Mycoplasma</i> detection kit, PlasmoTest (InvivoGen, San Diego, California). All cell lines were authenticated with STR DNA profiling (University of Michigan, Michigan) and matched to reference profiles from the ATCC database. 3-(4,5-Dimethylthiazol-2-yl)-2, 5-diphenyltetrazolium bromide (MTT) was purchased from Amresco (Solon, OH). Small-molecule screening libraries were purchased from ChemDiv (San Diego, CA) or obtained from the National Cancer Institute through the Developmental Therapeutics Program.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> PDI Protein Purification</h3><div class="NLM_p last">The expression vector of the recombinant human wild-type PDI protein with a N-terminal His tag was a gift from Dr. Lloyd W. Ruddock (University of Oulu, Oulu, Finland). Full-length recombinant PDI and the a′c domain (residues V360–L499) were expressed and purified as previously described<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> with slight modifications. In brief, protein production was carried out in <i>Escherichia coli</i> strain BL21(DE3) grown in the LB medium with 200 μg/mL ampicillin (EMD Biosciences, La Jolla, CA) at 37 °C and incubated at an <i>A</i><sub>600</sub> of 0.5 for 4 h with 1 mM isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside (GoldBio, St. Louis, MO). The cells were harvested by centrifugation (2500<i>g</i> for 15 min) and were resuspended in 20 mM sodium phosphate, pH 7.3 buffer. The cells were lysed by sonication, and the cell debris was removed by centrifugation (17 000<i>g</i> for 45 min). The supernatant was applied to a bed of Ni-nitrilotriacetic acid in a histidine-binding column (Qiagen, Hilden, Germany), equilibrated with 10 mL of 20 mM sodium phosphate, pH 7.3 and incubated at 4 °C, overnight. After incubation, the column was washed in 20 mM sodium phosphate, pH 7.3 and then in 20 mM sodium phosphate, 0.5 M sodium chloride, and 20 mM imidazole, pH 7.3. His-tagged proteins were eluted with a buffer containing 20 mM sodium phosphate and 500 mM imidazole, pH 7.3, and the eluent was dialyzed in 100 mM sodium phosphate buffer (pH 7.0) with 2 mM EDTA. PDIA2 and PDIA3 were purified as described previously.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Site-Directed Mutagenesis</h3><div class="NLM_p last">H256A and C53S mutants of PDI were obtained using wild-type PDI as the DNA template with the QuikChange II XL Site-Directed Mutagenesis kit (Agilent Technologies, Santa Cruz, CA). The procedure was performed according to the manufacturer’s protocol. All constructs were sequenced for verification, and no additional mutations were observed. Mutant PDI constructs were transformed into BL21(DE3) cells and purified according to the wild-type PDI purification protocol.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> PDI Reductase Assay</h3><div class="NLM_p">PDI activity was assessed by measuring the PDI-catalyzed reduction of insulin as described previously.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In brief, the recombinant PDI protein (0.4 μM or 50 nM for PDIA1, 1.6 μM PDIA2 or PDIA3) was incubated with indicated compounds at 37 °C for 1 h in sodium phosphate buffer (100 mM sodium phosphate, 2 mM EDTA, 8 μM DTT, pH 7.0). A mixture of sodium phosphate buffer, DTT (500 μM or 125 μM for 50 nM PDI reaction), and bovine insulin (130 μM; Gemini BioProducts, West Sacramento, CA) was added to the incubated PDI protein. The reduction reaction was catalyzed by PDI at room temperature, and the resulting aggregation of reduced insulin B chains was measured at 620 nm. PDI activity was calculated with the formula, PDI activity (%) = [(OD<sub>T60[PDI+DTT+compound]</sub> – OD<sub>T0[PDI+DTT+compound]</sub>) – (OD<sub>T60[DTT]</sub> – OD<sub>T0[DTT]</sub>)] / [(OD<sub>T60[PDI+DTT]</sub> – OD<sub>T0[PDI+DTT]</sub>) – (OD<sub>T60[DTT]</sub> – OD<sub>T0[DTT]</sub>)] × 100 (OD<sub>T0</sub> and OD<sub>T60</sub> were the absorbance values at 0 and 60 min after the reduction reaction, respectively). For reactions containing 50 nM PDI, PDI activity was measured at T180 or 180 min after insulin was added.</div><div class="NLM_p last">To determine the <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> of covalent PDI inhibitors, the published procedure was adapted with the following modifications.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Compounds were incubated at 13.2, 19.8, 29.6, 44.4, 66.7, 100, and 150 μM for 5, 15, 30, 45, or 60 min before the addition of the insulin solution. The linear portions of the slopes of each kinetic curve obtained were used to calculate the <i>K</i><sub>obs</sub> in GraphPad Prism. The <i>k</i><sub>obs</sub> at each concentration was plotted to obtain the slope of the linear portion of the line as <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub>.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> Growth Inhibition Assay</h3><div class="NLM_p last">Cell growth inhibition was assessed by the MTT assay as previously described.<sup>61</sup> The cells were seeded in duplicate in 96-well plates at 3000–5000 cells/well. After overnight incubation at 37 °C and 5% CO<sub>2</sub>, the cells were treated with the indicated compounds for 72 h. For glutathione depletion experiments, the cells were pretreated for 24 h with buthionine sulfoximine (1 mM in A-172 or 4 μM in U-118 MG) before compound addition. MTT solution (20 μL, 3 mg/mL) was added to the wells, and the cells were incubated for 4 h at 37 °C. The supernatant was removed and dimethyl sulfoxide (DMSO, 100 μL) was added to each well. The plates were shaken for 15 min at room temperature, and absorbance of the formazan crystals was measured at 570 nm. Cell growth inhibition was assessed by the cell viability rate as [1–(<i>A</i><sub>t</sub> – <i>A</i><sub>b</sub>)/(<i>A</i><sub>c</sub> – <i>A</i><sub>b</sub>)] × 100 (<i>A</i><sub>t</sub>, <i>A</i><sub>c</sub>, and <i>A</i><sub>b</sub> were the absorbance values from the cells treated with the compound, cells not treated with the compound, and blank, respectively).</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Colony Formation Assay</h3><div class="NLM_p last">Briefly, 600 U-118 MG or A-172 cells were seeded per well in 96-well plates and allowed to attach. After 24 h, serial dilutions of buthionine sulfoximine (Sigma-Aldrich) were added. The cells were incubated overnight, and serial dilutions of experimental compounds were added to the culture medium. The cells were cultured until colonies formed (10–14 days), stained with crystal violet solution (0.25% crystal violet, 10% formaldehyde (37% v/v), 80% methanol) for 1 h, and thoroughly washed with water before imaging.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> Thermal Shift Assay</h3><div class="NLM_p last">Thermal shift of purified PDI (0.3 mg/mL in 100 mM NaPO<sub>4</sub>, pH 7.0) in the presence or absence of indicated compounds was determined as described.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Briefly, PDI, 100 μM compound or DMSO as a vehicle control, 1× ROX dye, and 5 μL Protein Thermal Shift Buffer were mixed to a 20 μL total volume in a 384-well microplate. Each reaction was repeated in quadruplicate, and the reactions were mixed before measurements were taken. The plate was heated from 25 to 90 °C at 0.05 °C/s with the ViiA 7 Real-Time PCR System (Thermo Fisher Scientific, Waltham, MO). Melt curves were analyzed with the Protein Thermal Shift software (Thermo Fisher Scientific), and Boltzmann melting temperatures were reported.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> Reversibility Assay</h3><div class="NLM_p last">PDI activity was assessed by measuring the PDI-catalyzed reduction of insulin as described previously.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In brief, 0.4 μM recombinant PDI was incubated with compounds at the indicated concentrations at 37 °C for 1 h in sodium phosphate buffer (100 mM sodium phosphate, 2 mM EDTA, 8 μM DTT, pH 7.0). For samples containing diluted protein–compound complexes, 40 μM PDI was incubated with 100 μM <b>PACMA31</b>, 50 μM <b>BAP2</b>, 50 μM <b>AS15</b>, or 50 μM <b>CD343</b> for 3 h at room temperature or was incubated with 400 μM <b>AS15</b>, 400 μM <b>CD343</b>, or 400 μM <b>6a</b> for 30 min at room temperature. The mixtures were diluted 100-fold into a buffer (100 mM sodium phosphate, 2 mM EDTA, 8 μM DTT, pH 7.0) and added to the 384-well, black, clear-bottom plate. A mixture of sodium phosphate buffer, DTT (500 μM), and bovine insulin (130 μM; Gemini BioProducts, West Sacramento, CA) was added to the incubated PDI–compound samples. The reduction reaction was catalyzed by PDI at room temperature, and the resulting aggregation of reduced insulin B chains was measured at 620 nm. Absorbance at 620 nm was measured in a 384-well black-walled, clear-bottom plate.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> 1-Anilinonaphthalene-8-sulfonic Acid (ANS) Spectral Scan</h3><div class="NLM_p last">The ANS spectral scan was performed as previously described.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Briefly, 5 μM PDI was incubated in the presence of 100 μM compounds or equivalent DMSO concentration in 50 μL of TBS at 37 °C for 1 h. Subsequently, ANS was added to a final concentration of 50 μM and the mixture was incubated in the dark at 25 °C for 20 min. Fluorescence spectrum (Ex: 370 nm, Em: 400–700 nm) was measured in a 384-well black-walled, clear-bottom plate.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Bromouridine RNA Sequencing (Bru-seq)</h3><div class="NLM_p">Bru-seq was performed as previously described.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> U-87 MG cells were treated with DMSO or <b>5p</b> (20 μM) for 4 h. Then, 2 mM Bru was added in the last 30 min of treatment. The cells were collected, and total RNA was isolated with a TRIzol reagent. Bru-labeled RNA was captured from the total RNA by incubation with anti-BrdU antibodies (BD Biosciences) conjugated to magnetic beads (Dynabeads, goat antimouse IgG; Invitrogen). Bru-containing RNA population was isolated and sequenced. The sequencing reads were mapped to the hg38 reference genome. Preranked gene lists were generated for each treatment ranking genes by fold change in transcription compared to those of control. Sequencing results were filtered using a cutoff value of gene size > 300 bp and mean RPKM > 0.5.</div><div class="NLM_p">The data sets were interrogated with Gene Set Enrichment Analysis (GSEA).<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> A log 2(fold change) preranked gene list of 7,908 genes was analyzed for gene enrichment using GSEA gene sets based on the Kolmogorov–Smirnov statistic. For each gene set, an enrichment score (ES) was normalized to account for the difference in gene set size, and the false discovery rate (FDR) was calculated based on the normalized enrichment score (NES) values.</div><div class="NLM_p last">The data sets were also interrogated with Connectivity Map (CMap, <a href="https://www.broadinstitute.org/connectivity-map-cmap" class="extLink">https://www.broadinstitute.org/connectivity-map-cmap</a>). Bru-seq gene sets were used with a cutoff of ≥2-fold change in transcription. Some genes were omitted from analysis because there was no connection in CMap.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Western Blot</h3><div class="NLM_p last">Cells were harvested with a lysis buffer (25 mM tris(hydroxymethyl)aminomethane, 150 mM NaCl, 17 mM Triton X-100, 3.5 mM SDS, pH 7.4), lysed via sonication, and spun in a centrifuge at 13 500<i>g</i> at 4 °C for 10 min. The supernatant was collected, and protein concentration was determined with the BCA assay (Thermo Fisher Scientific, Waltham, MO). The samples were prepared with 30 μg protein and loaded onto 10% acrylamide (Bio-Rad, Hercules, CA) gels. The protein from gels was electrotransferred to methanol-activated immobilon-FL PVDF membranes (EMD Millipore, La Jolla, CA). Membranes were blocked for 1 h with an Odyssey blocking buffer (LI-COR Biosciences, Lincoln, NE), probed for proteins using primary antibodies (PDI, Cell Signaling, Danvers, MA, 1:1000) overnight at 4 °C, and incubated with secondary antibodies (antirabbit, Cell Signaling, 1:7500, or antimouse, Cell Signaling, 1:7500), and fluorescence was imaged by an Odyssey imaging system (LI-COR Biosciences).</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Proteomics</h3><div class="NLM_p last">U-118 MG cells were seeded in a 6-well plate at 0.5 × 10<sup>6</sup> cells/well in RPMI supplemented with 10% FBS and allowed to attach overnight. The cells were treated with DMSO or 40 μM <b>15</b> overnight. The cells were washed with PBS and harvested with a CelLytic M buffer (Sigma-Aldrich). The cells were lysed by incubation for 1 h on ice and spun in a centrifuge at 13 500<i>g</i> at 4 °C for 10 min. The supernatant was collected, and protein concentration was determined with the BCA assay (Thermo Fisher Scientific, Waltham, MO). The samples were prepared with 50 μg protein boiled with the Laemmli sample buffer and loaded onto 10% acrylamide gels. The gel was immediately imaged on the iBright with the GFP channel and stained with Coomassie. The band containing the BODIPY-labeled compound was cut out, digested, and analyzed at the University of Michigan Proteomics Resource Facility in the Department of Pathology.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Confocal Imaging</h3><div class="NLM_p last">A-172 cells were treated with 10 μM <b>15</b> or 2 μM <b>16</b> overnight. The cells were fixed in 4% paraformaldehyde for 15 min at room temperature and washed with 1× PBS before blocking in 10% fetal bovine serum for 60 min. PDI antibody (Cell Signaling; 3501S) was applied at a 1:100 dilution overnight at 4 °C. ProLong Diamond with DAPI (Invitrogen) was used to prepare the slides for analysis on the ZEISS Laser Scanning Microscope.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Gel-Based Binding Assays</h3><div class="NLM_p last">Gel-based binding assays were performed with a recombinant protein and cell lysate, as indicated. Briefly, the cells were coated in 6-well or 12-well plates. After overnight attachment, the cells were either serum-starved, treated with BSO, or treated with test compounds at indicated concentrations overnight at 37 °C, 5% CO<sub>2</sub>. The cells were then washed with PBS and lysed using a CelLytic M buffer (Sigma-Aldrich) for 60 min on ice. A unit of 30–50 μg of whole-cell protein was boiled with the Laemmli sample buffer or nonreducing sample buffer (62.5 mM Tris–HCl, pH 6.8, 10% glycerol, 2% sodium dodecyl sulfate (SDS), 0.05% bromophenol blue) and resolved on a 10% polyacrylamide gel. Gels were immediately imaged on an iBright imaging system (Thermo Fisher Scientific). For cell lysates, the cells were harvested as above prior to compound treatment and then incubated with compounds overnight at room temperature before subjecting to SDS polyacrylamide gel electrophoresis (PAGE). Similarly, <i>in vitro</i> binding assays with recombinant PDIA1, PDIA2, PDIA3, and BSA were performed using 3.5 μM protein in a CelLytic M buffer incubated with compounds overnight at room temperature.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Statistical Analysis</h3><div class="NLM_p last">The IC<sub>50</sub> values were calculated using GraphPad Prism 7 software (GraphPad Software, Inc.). The error bars indicate the mean ± standard deviation. Bru-seq experiments were performed once.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00728" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60366" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60366" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00728?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00728</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplemental File 1: Proteomics (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_001.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Supplemental File 2: Bru-seq (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Additional computational analysis, biochemical data, activity of <b>AS15</b> and <b>CD343</b> in the PDI reductase assay, deconvoluted mass spectra, SAR of type XI compounds, and CMap structures (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings: jmcmar_mfs_AS15_analogues (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_001.xlsx">jm0c00728_si_001.xlsx (4.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_002.xlsx">jm0c00728_si_002.xlsx (1.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf">jm0c00728_si_003.pdf (9.5 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_004.csv">jm0c00728_si_004.csv (2.57 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00728" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nouri Neamati</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3291-7131" title="Orcid link">http://orcid.org/0000-0003-3291-7131</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e9878c8884889d80a99c84808a81c78c8d9c"><span class="__cf_email__" data-cfemail="472922262a26332e07322a2e242f69222332">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Shergalis</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1155-1583" title="Orcid link">http://orcid.org/0000-0002-1155-1583</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ding Xue</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fatma Z. Gharbia</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hannah Driks</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Binita Shrestha</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amina Tanweer</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kirin Cromer</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mats Ljungman</span> - <span class="hlFld-Affiliation affiliation">Department
of Radiation Oncology, University of Michigan
Medical School and Rogel Cancer Center, School of Public Health, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.S. and D.X. contributed equally to this work. The manuscript was written through the contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e5446-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i95">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75238" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75238" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The expression vectors of recombinant human PDI were generous gifts from Dr. Lloyd W. Ruddock (University of Oulu, Oulu, Finland). The <b>a′</b>c domain vector was a kind gift from Dr. C. C. Wang. We are grateful to Michelle Paulsen for performing the Bru-Seq experiments and to Jennifer Schmidt for her assistance with the Q-TOF mass spectrometry experiments. We would also like to thank the University of Michigan Proteomics Resource Facility at the Department of Pathology for performing the proteomics experiments and analyzing the results and the University of Michigan DNA Sequencing Core. This work was supported by a grant from the NIH (CA193690).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">DTT</td><td class="NLM_def"><p class="first last">dithiothreitol</p></td></tr><tr><td class="NLM_term">PDI</td><td class="NLM_def"><p class="first last">protein disulfide isomerase</p></td></tr><tr><td class="NLM_term">ROS</td><td class="NLM_def"><p class="first last">reactive oxygen species</p></td></tr><tr><td class="NLM_term">UPR</td><td class="NLM_def"><p class="first last">unfolded protein response</p></td></tr><tr><td class="NLM_term">ATCC</td><td class="NLM_def"><p class="first last">American Type Culture Collection</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">endoplasmic reticulum</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">GBM</td><td class="NLM_def"><p class="first last">glioblastoma</p></td></tr><tr><td class="NLM_term">GRP78</td><td class="NLM_def"><p class="first last">glucose-regulated protein, 78 kDa</p></td></tr><tr><td class="NLM_term">LB</td><td class="NLM_def"><p class="first last">lysogeny broth</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr><tr><td class="NLM_term">NCI</td><td class="NLM_def"><p class="first last">National Cancer Institute</p></td></tr><tr><td class="NLM_term">PVDF</td><td class="NLM_def"><p class="first last">poly(vinylidene difluoride)</p></td></tr><tr><td class="NLM_term">RPMI</td><td class="NLM_def"><p class="first last">Roswell Park Memorial Institute</p></td></tr><tr><td class="NLM_term">STR</td><td class="NLM_def"><p class="first last">short tandem repeats</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71807" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71807" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrom, Q. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cioffi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gittleman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waite, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruchko, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnholtz-Sloan, J. S.</span></span> <span> </span><span class="NLM_article-title">CBTRUS statistical report: Primary brain and other central nervous system tumors diagnosed in the United States in 2012-2016</span>. <i>Neuro-Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">v1</span>– <span class="NLM_lpage">v100</span>, <span class="refDoi"> DOI: 10.1093/neuonc/noz150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1093%2Fneuonc%2Fnoz150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=31675094" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=v1-v100&author=Q.+T.+Ostromauthor=G.+Cioffiauthor=H.+Gittlemanauthor=N.+Patilauthor=K.+Waiteauthor=C.+Kruchkoauthor=J.+S.+Barnholtz-Sloan&title=CBTRUS+statistical+report%3A+Primary+brain+and+other+central+nervous+system+tumors+diagnosed+in+the+United+States+in+2012-2016&doi=10.1093%2Fneuonc%2Fnoz150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fneuonc%2Fnoz150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fneuonc%252Fnoz150%26sid%3Dliteratum%253Aachs%26aulast%3DOstrom%26aufirst%3DQ.%2BT.%26aulast%3DCioffi%26aufirst%3DG.%26aulast%3DGittleman%26aufirst%3DH.%26aulast%3DPatil%26aufirst%3DN.%26aulast%3DWaite%26aufirst%3DK.%26aulast%3DKruchko%26aufirst%3DC.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%2BS.%26atitle%3DCBTRUS%2520statistical%2520report%253A%2520Primary%2520brain%2520and%2520other%2520central%2520nervous%2520system%2520tumors%2520diagnosed%2520in%2520the%2520United%2520States%2520in%25202012-2016%26jtitle%3DNeuro-Oncol.%26date%3D2019%26volume%3D21%26spage%3Dv1%26epage%3Dv100%26doi%3D10.1093%2Fneuonc%2Fnoz150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shergalis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luesakul, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muangsin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Current challenges and opportunities in treating glioblastoma</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1124/pr.117.014944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1124%2Fpr.117.014944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29669750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=412-445&author=A.+Shergalisauthor=A.+Bankheadauthor=U.+Luesakulauthor=N.+Muangsinauthor=N.+Neamati&title=Current+challenges+and+opportunities+in+treating+glioblastoma&doi=10.1124%2Fpr.117.014944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Current challenges and opportunities in treating glioblastoma</span></div><div class="casAuthors">Shergalis, Andrea; Bankhead, Armand; Luesakul, Urarika; Muangsin, Nongnuj; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">412-445</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Glioblastoma multiforme (GBM), the most common and aggressive primary brain tumor, has a high mortality rate despite extensive efforts to develop new treatments.  GBM exhibits both intra- and intertumor heterogeneity, lending to resistance and eventual tumor recurrence.  Large-scale genomic and proteomic anal. of GBM tumors has uncovered potential drug targets.  Effective and "druggable" targets must be validated to embark on a robust medicinal chem. campaign culminating in the discovery of clin. candidates.  Here, we review recent developments in GBM drug discovery and delivery.  To identify GBM drug targets, we performed extensive bioinformatics anal. using data from The Cancer Genome Atlas project.  We discovered 20 genes, BOC, CLEC4GP1, ELOVL6, EREG, ESR2, FDCSP, FURIN, FUT8-AS1, GZMB, IRX3, LITAF, NDEL1, NKX3-1, PODNL1, PTPRN, QSOX1, SEMA4F, TH, VEGFC, and C20orf166AS1 that are overexpressed in a subpopulation of GBM patients and correlate with poor survival outcomes.  Importantly, nine of these genes exhibit higher expression in GBM vs. low-grade glioma and may be involved in disease progression.  In this review, we discuss these proteins in the context of GBM disease progression.  We also conducted computational multi-parameter optimization to assess the blood-brain barrier (BBB) permeability of small mols. in clin. trials for GBM treatment.  Drug delivery in the context of GBM is particularly challenging because the BBB hinders small mol. transport.  Therefore, we discuss novel drug delivery methods, including nanoparticles and prodrugs.  Given the aggressive nature of GBM and the complexity of targeting the central nervous system, effective treatment options are a major unmet medical need.  Identification and validation of biomarkers and drug targets assocd. with GBM disease progression present an exciting opportunity to improve treatment of this devastating disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKoQ_eLuZhn7Vg90H21EOLACvtfcHk0liMGgA0yKSBqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsb8%253D&md5=80fb3912ca8df56db7b1c146ef45ffef</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1124%2Fpr.117.014944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.117.014944%26sid%3Dliteratum%253Aachs%26aulast%3DShergalis%26aufirst%3DA.%26aulast%3DBankhead%26aufirst%3DA.%26aulast%3DLuesakul%26aufirst%3DU.%26aulast%3DMuangsin%26aufirst%3DN.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DCurrent%2520challenges%2520and%2520opportunities%2520in%2520treating%2520glioblastoma%26jtitle%3DPharmacol.%2520Rev.%26date%3D2018%26volume%3D70%26spage%3D412%26epage%3D445%26doi%3D10.1124%2Fpr.117.014944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. C.</span></span> <span> </span><span class="NLM_article-title">Structural insights into the redox-regulated dynamic conformations of human protein disulfide isomerase</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1089/ars.2012.4630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1089%2Fars.2012.4630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22657537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC38not1Cnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=36-45&author=C.+Wangauthor=W.+Liauthor=J.+Renauthor=J.+Fangauthor=H.+Keauthor=W.+Gongauthor=W.+Fengauthor=C.+C.+Wang&title=Structural+insights+into+the+redox-regulated+dynamic+conformations+of+human+protein+disulfide+isomerase&doi=10.1089%2Fars.2012.4630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the redox-regulated dynamic conformations of human protein disulfide isomerase</span></div><div class="casAuthors">Wang Chao; Li Wei; Ren Jinqi; Fang Jingqi; Ke Huimin; Gong Weimin; Feng Wei; Wang Chih-Chen</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & redox signaling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-45</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AIM:  Human protein disulfide isomerase (hPDI) is a key enzyme and a redox-regulated chaperone responsible for oxidative protein folding in the endoplasmic reticulum.  This work aims to reveal the molecular mechanism underlying the redox-regulated functions of hPDI by determining the crystal structures of hPDI in different redox states.  RESULTS:  The structures of hPDI (abb'xa') in both the reduced and oxidized states showed that the four thioredoxin domains of a, b, b', and a' are arranged as a horseshoe shape with two CGHC active sites, respectively, in domains a and a' facing each other at the two ends.  In reduced hPDI, domains a, b, and b' line up in the same plane, whereas domain a' twists ∼45° out.  The two active sites are 27.6 ÅA apart.  In oxidized hPDI, the four domains are differently organized to stay in the same plane, and the distance between the active sites increases to 40.3 ÅA.  In contrast to the closed conformation of reduced hPDI, oxidized hPDI exists in an open state with more exposed hydrophobic areas and a larger cleft with potential for substrate binding.  INNOVATION:  This is the first report of the high-resolution structures of hPDI containing all four domains in both the reduced and the oxidized states.  It reveals the redox-regulated structural dynamic properties of the protein.  CONCLUSION:  The redox-regulated open/closed conformational switch of hPDI endows the protein with versatile target-binding capacities for its enzymatic and chaperone functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWKKds5MV7mi86Js-BCLiifW6udTcc2eZSGFfo0hqxNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38not1Cnug%253D%253D&md5=ff87e33dc426c3d2a8ad7c76d8a27635</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1089%2Fars.2012.4630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2012.4630%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DKe%26aufirst%3DH.%26aulast%3DGong%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DC.%2BC.%26atitle%3DStructural%2520insights%2520into%2520the%2520redox-regulated%2520dynamic%2520conformations%2520of%2520human%2520protein%2520disulfide%2520isomerase%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2013%26volume%3D19%26spage%3D36%26epage%3D45%26doi%3D10.1089%2Fars.2012.4630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. H.</span></span> <span> </span><span class="NLM_article-title">P4HB and PDIA3 are associated with tumor progression and therapeutic outcome of diffuse gliomas</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.3892/or.2017.6134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.3892%2For.2017.6134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29207176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKhsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=501-510&author=H.+Zouauthor=C.+Wenauthor=Z.+Pengauthor=Y.+Shaoauthor=L.+Huauthor=S.+Liauthor=C.+Liauthor=H.+H.+Zhou&title=P4HB+and+PDIA3+are+associated+with+tumor+progression+and+therapeutic+outcome+of+diffuse+gliomas&doi=10.3892%2For.2017.6134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">P4HB and PDIA3 are associated with tumor progression and therapeutic outcome of diffuse gliomas</span></div><div class="casAuthors">Zou, Hecun; Wen, Chunjie; Peng, Zhigang; Shao, Ying-Ying; Hu, Lei; Li, Shuang; Li, Cuilin; Zhou, Hong-Hao</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">501-510</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Diffuse gliomas are the most common type of primary brain and central nervous system (CNS) tumors.  Protein disulfide isomerases (PDIs) such as P4HB and PDIA3 act as mol. chaperones for reconstructing misfolded proteins, and are involved in endoplasmic reticulum stress and the unfolded protein response.  The present study focused on the role of P4HB and PDIA3 in diffuse gliomas.  Anal. of GEO and HPA data revealed that the expression levels of P4HB and PDIA3 were upregulated in glioma datasets. their increased expression was then validated in 99 glioma specimens compared with 11 non-tumor tissues.  High expression of P4HB and PDIA3 was significantly correlated with high Ki-67 and a high frequency of the TP53 mutation.  Kaplan-Meier survival curve and Cox regression analyses showed that glioma patients with high P4HB and PDIA3 expression had a poor survival outcome, P4HB and PDIA3 could be independent prognostic biomarkers for diffuse gliomas.  In vitro, knockdown of PDIA3 suppressed cell proliferation, induced cell apoptosis, and decreased the migration of glioma cells.  Furthermore, downregulation of P4HB and PDIA3 may contribute to improve the survival of patients who receive chemotherapy and radiotherapy.  The data suggest that high expression of P4HB and PDIA3 plays an important role in glioma progression, and could predict the survival outcome and therapeutic response of glioma patients.  Therefore, protein disulfide isomerases may be explored as prognostic biomarkers and therapeutic targets for diffuse gliomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdzdJGwdCF7bVg90H21EOLACvtfcHk0lhi0Cwa5qW63Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKhsLzF&md5=b7f85021f7bc0db18c2428f18a2fc156</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3892%2For.2017.6134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2017.6134%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DH.%2BH.%26atitle%3DP4HB%2520and%2520PDIA3%2520are%2520associated%2520with%2520tumor%2520progression%2520and%2520therapeutic%2520outcome%2520of%2520diffuse%2520gliomas%26jtitle%3DOncol.%2520Rep.%26date%3D2018%26volume%3D39%26spage%3D501%26epage%3D510%26doi%3D10.3892%2For.2017.6134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiang, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, G. K.</span></span> <span> </span><span class="NLM_article-title">Hyperoxia resensitizes chemoresistant glioblastoma cells to temozolomide through unfolded protein response</span>. <i>Anticancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">2957</span>– <span class="NLM_lpage">2966</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=24922660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=2957-2966&author=D.+Leeauthor=S.+Sunauthor=A.+S.+Hoauthor=K.+M.+Kiangauthor=X.+Q.+Zhangauthor=F.+F.+Xuauthor=G.+K.+Leung&title=Hyperoxia+resensitizes+chemoresistant+glioblastoma+cells+to+temozolomide+through+unfolded+protein+response"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Hyperoxia resensitizes chemoresistant glioblastoma cells to temozolomide through unfolded protein response</span></div><div class="casAuthors">Lee, Derek; Sun, Stella; Ho, Amy S. W.; Kiang, Karrie M. Y.; Zhang, Xiao Qin; Xu, Fei Fan; Leung, Gilberto K. K.</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2957-2966</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: Intratumoral hypoxia is assocd. with chemoresistance in glioblastoma multiforme (GBM), a highly malignant brain tumor.  Adaptive response to endoplasmic reticulum stress induced by temozolomide is a major obstacle in recurrent GBM.  We investigated whether hyperoxia resensitizes temozolomide-resistant GBM cells to temozolomide by abrogating the hypoxia-induced, unfolded protein response (UPR)-related protective mechanisms.  Materials and Methods: We examd. changes to key UPR modulators in temozolomide-sensitive and -resistant human GBM cells (D54 and U87) treated with/without temozolomide at different oxygen concns. using western blotting, and cytotoxic benefits of overexpressing key chaperone, P4HB, in GBM cells (U87 and U251) under normoxia and hyperoxia.  Results: Hyperoxia, alone or synergistically with temozolomide, activated the UPR in sensitive and resistant D54 and U87 cell lines.  Hyperoxia also reduced survival benefit of U87 and U251 cells with P4HB overexpression through the UPR.  Conclusion: Hyperoxia enhanced GBM cell sensitivity to temozolomide, likely through UPR, highlighting an important treatment modality targeting chemosensitive and -resistant GBM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG88WkoW_JFbVg90H21EOLACvtfcHk0lhi0Cwa5qW63Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtrrF&md5=0a0164306adb8a760cf6ae2cb14b0548</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DA.%2BS.%26aulast%3DKiang%26aufirst%3DK.%2BM.%26aulast%3DZhang%26aufirst%3DX.%2BQ.%26aulast%3DXu%26aufirst%3DF.%2BF.%26aulast%3DLeung%26aufirst%3DG.%2BK.%26atitle%3DHyperoxia%2520resensitizes%2520chemoresistant%2520glioblastoma%2520cells%2520to%2520temozolomide%2520through%2520unfolded%2520protein%2520response%26jtitle%3DAnticancer%2520Res.%26date%3D2014%26volume%3D34%26spage%3D2957%26epage%3D2966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Protein disulfide isomerase: A promising target for cancer therapy</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.drudis.2013.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=24184531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvValsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=222-240&author=S.+Xuauthor=S.+Sankarauthor=N.+Neamati&title=Protein+disulfide+isomerase%3A+A+promising+target+for+cancer+therapy&doi=10.1016%2Fj.drudis.2013.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Protein disulfide isomerase: a promising target for cancer therapy</span></div><div class="casAuthors">Xu, Shili; Sankar, Saranya; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">222-240</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein disulfide isomerase (PDI) has a key role in maintaining cellular homeostasis by mediating oxidative protein folding.  It catalyzes disulfide bond formation, breakage and rearrangement in the endoplasmic reticulum and has chaperone protein activity.  Increasing evidence suggests that PDI supports the survival and progression of several cancers.  During the past decade, robust PDI activity assays have been developed and several PDI inhibitors identified, but none has been approved for clin. use.  Herein, we review current knowledge of the role of PDI in cancer and discuss various assays for measuring the activities of PDI, highlighting their sensitivities and usefulness for high-throughput screening.  The previously reported PDI inhibitors require further validation to serve as bona fide leads and addnl. optimization to generate novel drug candidates for clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquANbcuJ6Bv7Vg90H21EOLACvtfcHk0lhi0Cwa5qW63Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvValsbrM&md5=cab1d37a444bf32f1974567b7622f7e0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DSankar%26aufirst%3DS.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DProtein%2520disulfide%2520isomerase%253A%2520A%2520promising%2520target%2520for%2520cancer%2520therapy%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D222%26epage%3D240%26doi%3D10.1016%2Fj.drudis.2013.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lappi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lensink, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alanen, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salo, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juffer, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddock, L. W.</span></span> <span> </span><span class="NLM_article-title">A conserved arginine plays a role in the catalytic cycle of the protein disulphide isomerases</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2003.10.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.jmb.2003.10.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=14659757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BD3srotVajug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2004&pages=283-295&author=A.+K.+Lappiauthor=M.+F.+Lensinkauthor=H.+I.+Alanenauthor=K.+E.+Saloauthor=M.+Lobellauthor=A.+H.+Jufferauthor=L.+W.+Ruddock&title=A+conserved+arginine+plays+a+role+in+the+catalytic+cycle+of+the+protein+disulphide+isomerases&doi=10.1016%2Fj.jmb.2003.10.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved arginine plays a role in the catalytic cycle of the protein disulphide isomerases</span></div><div class="casAuthors">Lappi A K; Lensink M F; Alanen H I; Salo K E H; Lobell M; Juffer A H; Ruddock L W</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">283-95</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The pK(a) values of the CXXC active-site cysteine residues play a critical role in determining the physiological function of the thioredoxin superfamily.  To act as an efficient thiol-disulphide oxidant the thiolate state of the N-terminal cysteine must be stabilised and the thiolate state of the C-terminal cysteine residue destabilised.  While increasing the pK(a) value of the C-terminal cysteine residue promotes oxidation of substrates, it has an inhibitory effect on the reoxidation of the enzyme, which is promoted by the formation of a thiolate at this position.  Since reoxidation is essential to complete the catalytic cycle, the differential requirement for a high and a low pK(a) value for the C-terminal cysteine residue for different steps in the reaction presents us with a paradox.  Here, we report the identification of a conserved arginine residue, located in the loop between beta5 and alpha4 of the catalytic domains of the human protein disulphide isomerase (PDI) family, which is critical for the catalytic function of PDI, ERp57, ERp72 and P5, specifically for reoxidation.  An examination of the published NMR structure for the a domain of PDI combined with molecular dynamic studies suggest that the side-chain of this arginine residue moves into and out of the active-site locale and that this has a very marked effect on the pK(a) value of the active-site cysteine residues.  This intra-domain motion resolves the apparent dichotomy of the pK(a) requirements for the C-terminal active-site cysteine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0DS9fXAJ2HWx8TEN9K18QfW6udTcc2eZVWZHeE4Tvnbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srotVajug%253D%253D&md5=546c07c515ab7212a38304a18c2cb4de</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.10.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.10.051%26sid%3Dliteratum%253Aachs%26aulast%3DLappi%26aufirst%3DA.%2BK.%26aulast%3DLensink%26aufirst%3DM.%2BF.%26aulast%3DAlanen%26aufirst%3DH.%2BI.%26aulast%3DSalo%26aufirst%3DK.%2BE.%26aulast%3DLobell%26aufirst%3DM.%26aulast%3DJuffer%26aufirst%3DA.%2BH.%26aulast%3DRuddock%26aufirst%3DL.%2BW.%26atitle%3DA%2520conserved%2520arginine%2520plays%2520a%2520role%2520in%2520the%2520catalytic%2520cycle%2520of%2520the%2520protein%2520disulphide%2520isomerases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D335%26spage%3D283%26epage%3D295%26doi%3D10.1016%2Fj.jmb.2003.10.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butkevich, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zandi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petasis, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16348</span>– <span class="NLM_lpage">16353</span>, <span class="refDoi"> DOI: 10.1073/pnas.1205226109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.1205226109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22988091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16348-16353&author=S.+Xuauthor=A.+N.+Butkevichauthor=R.+Yamadaauthor=Y.+Zhouauthor=B.+Debnathauthor=R.+Duncanauthor=E.+Zandiauthor=N.+A.+Petasisauthor=N.+Neamati&title=Discovery+of+an+orally+active+small-molecule+irreversible+inhibitor+of+protein+disulfide+isomerase+for+ovarian+cancer+treatment&doi=10.1073%2Fpnas.1205226109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span></div><div class="casAuthors">Xu, Shili; Butkevich, Alexey N.; Yamada, Roppei; Zhou, Yu; Debnath, Bikash; Duncan, Roger; Zandi, Ebrahim; Petasis, Nicos A.; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16348-16353, S16348/1-S16348/18</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI), an endoplasmic reticulum chaper-one protein, catalyzes disulfide bond breakage, formation, and rearrangement.  The effect of PDI inhibition on ovarian cancer progression is not yet clear, and there is a need for potent selective, and safe small-mol. inhibitors of PDI.  Here, we report a class of propynoic acid carbamoyl Me amides (PACMAs) that are active against a panel of human ovarian cancer cell lines.  Using fluorescent derivs., 2D gel electrophoresis, and MS, we established that one of the most active analogs, acts as an irreversible small-mol. inhibitor of PDI, forming a covalent bond with the active site cysteines of PDI.  We also showed that PDI activity is essential for the survival and proliferation of human ovarian cancer cells.  In vivo that compd. showed tumor targeting ability and significantly suppressed ovarian tumor growth without causing toxicity to normal tissues.  These irreversible small-mol. PDI inhibitors represent an important approach for the development of targeted anticancer agents for ovarian cancer therapy, and they can also serve as useful probes for investigating the biol. of PDI-implicated pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozZP4n4am-hLVg90H21EOLACvtfcHk0lgvr5hkkkBbQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE&md5=26801591b73d67710e06ef184ee9ddde</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1205226109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1205226109%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DButkevich%26aufirst%3DA.%2BN.%26aulast%3DYamada%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DDuncan%26aufirst%3DR.%26aulast%3DZandi%26aufirst%3DE.%26aulast%3DPetasis%26aufirst%3DN.%2BA.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520active%2520small-molecule%2520irreversible%2520inhibitor%2520of%2520protein%2520disulfide%2520isomerase%2520for%2520ovarian%2520cancer%2520treatment%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16348%26epage%3D16353%26doi%3D10.1073%2Fpnas.1205226109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandjean, J. M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witt, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Day, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoulders, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span> <span> </span><span class="NLM_article-title">Characterization of an A-site selective protein disulfide isomerase A1 inhibitor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2035</span>– <span class="NLM_lpage">2043</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFWgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2035-2043&author=K.+S.+Coleauthor=J.+M.+D.+Grandjeanauthor=K.+Chenauthor=C.+H.+Wittauthor=J.+O%E2%80%99Dayauthor=M.+D.+Shouldersauthor=R.+L.+Wisemanauthor=E.+Weerapana&title=Characterization+of+an+A-site+selective+protein+disulfide+isomerase+A1+inhibitor&doi=10.1021%2Facs.biochem.8b00178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of an a-Site Selective Protein Disulfide Isomerase A1 Inhibitor</span></div><div class="casAuthors">Cole, Kyle S.; Grandjean, Julia M. D.; Chen, Kenny; Witt, Collin H.; O'Day, Johanna; Shoulders, Matthew D.; Wiseman, R. Luke; Weerapana, Eranthie</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2035-2043</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein disulfide isomerase A1 (PDIA1) is an endoplasmic reticulum (ER)-localized thiol-disulfide oxidoreductase that is an important folding catalyst for secretory pathway proteins.  PDIA1 contains two active-site domains (a and a'), each contg. a Cys-Gly-His-Cys (CGHC) active-site motif.  The two active-site domains share 37% sequence identity and function independently to perform disulfide-bond redn., oxidn., and isomerization.  Numerous inhibitors for PDIA1 have been reported, yet the selectivity of these inhibitors toward the a and a' sites is poorly characterized.  Here, we identify a potent and selective PDIA1 inhibitor, KSC-34, with 30-fold selectivity for the a site over the a' site.  KSC-34 displays time-dependent inhibition of PDIA1 reductase activity in vitro with a kinact/KI of 9.66 × 103 M-1 s-1 and is selective for PDIA1 over other members of the PDI family, and other cellular cysteine-contg. proteins.  We provide the first cellular characterization of an a-site selective PDIA1 inhibitor and demonstrate that KSC-34 has minimal sustained effects on the cellular unfolded protein response (UPR), indicating that a-site inhibition does not induce global protein folding-assocd. ER stress.  KSC-34 treatment significantly decreases the rate of secretion of the destabilized amyloidogenic antibody light chain ALLC, thereby minimizing pathogenic amyloidogenic extracellular proteins that rely on high PDIA1 activity for proper folding and secretion.  Given the poor understanding of the contribution of each PDIA1 active site to the (patho)physiol. functions of PDIA1, site selective inhibitors like KSC-34 provide useful tools for delineating the pathol. role and therapeutic potential of PDIA1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMaqymP7LrLLVg90H21EOLACvtfcHk0lgvr5hkkkBbQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFWgsL8%253D&md5=95b6c4e046d4e8655ae0cfb45eed78a7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00178%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DK.%2BS.%26aulast%3DGrandjean%26aufirst%3DJ.%2BM.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWitt%26aufirst%3DC.%2BH.%26aulast%3DO%25E2%2580%2599Day%26aufirst%3DJ.%26aulast%3DShoulders%26aufirst%3DM.%2BD.%26aulast%3DWiseman%26aufirst%3DR.%2BL.%26aulast%3DWeerapana%26aufirst%3DE.%26atitle%3DCharacterization%2520of%2520an%2520A-site%2520selective%2520protein%2520disulfide%2520isomerase%2520A1%2520inhibitor%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D2035%26epage%3D2043%26doi%3D10.1021%2Facs.biochem.8b00178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. C.</span></span> <span> </span><span class="NLM_article-title">3,4-Methylenedioxy-beta-nitrostyrene inhibits adhesion and migration of human triple-negative breast cancer cells by suppressing beta1 integrin function and surface protein disulfide isomerase</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2015.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.biochi.2015.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=25593085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2htb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2015&pages=81-92&author=I.+H.+Chenauthor=F.+R.+Changauthor=Y.+C.+Wuauthor=P.+H.+Kungauthor=C.+C.+Wu&title=3%2C4-Methylenedioxy-beta-nitrostyrene+inhibits+adhesion+and+migration+of+human+triple-negative+breast+cancer+cells+by+suppressing+beta1+integrin+function+and+surface+protein+disulfide+isomerase&doi=10.1016%2Fj.biochi.2015.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">3,4-Methylenedioxy-β-nitrostyrene inhibits adhesion and migration of human triple-negative breast cancer cells by suppressing β1 integrin function and surface protein disulfide isomerase</span></div><div class="casAuthors">Chen, I-Hua; Chang, Fang-Rong; Wu, Yang-Chang; Kung, Po-Hsiung; Wu, Chin-Chung</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">81-92</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Triple neg. breast cancer (TNBC) exhibits an aggressive clin. course by high metastatic potential.  It is known that integrin-mediated cell adhesion and migration are important for cancer metastasis.  In the present study, a synthetic compd., 3, 4-methyenedioxy-β-nitrostyrene (MNS), significantly inhibited adhesion of TNBC cell lines to different extracellular matrix (ECM) components.  The antimetastatic capacity of MNS was also obsd. through reducing TNBC cells migration and invasion without affecting cell viability.  Confocal microscopy revealed that MNS disrupted the formation of focal adhesion complex and actin stress fiber networks.  Consistent with this finding, MNS inhibited phosphorylation of focal adhesion kinase (FAK) and paxillin as detected by Western blot anal.  In exploring the underlying mechanism, we found that MNS inhibited phosphorylation of FAK as a result of reducing β1 integrin activation and clustering.  A cell-impermeable dithiol reagent, 2, 3-dimercaptopropane-1-sulfonic acid abrogated all of MNS's actions, indicating that MNS may react with thiol groups of cell surface proteins that are involved in regulation of β1 integrin function as well as cell adhesion and migration.  Cell surface protein disulfide isomerase (PDI) has been reported to be essential for the affinity modulation of β integrins.  We also demonstrated that MNS inhibited PDI activity both in a pure enzyme system and in intact cancer cells.  Taken together, our results suggest that MNS inhibits in vitro metastatic properties of TNBC cells through suppression of β1 integrin activation and focal adhesion signaling.  Moreover, inhibition of surface PDI may contribute, at least in part, to the actions of MNS.  These results suggest that MNS has a potential to be developed as an anticancer agent for treatment of TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsh0eLDG0VdrVg90H21EOLACvtfcHk0ljlOMCKGHYJew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2htb4%253D&md5=2af854c64d414d515af7a8fe426652c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2015.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2015.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BH.%26aulast%3DChang%26aufirst%3DF.%2BR.%26aulast%3DWu%26aufirst%3DY.%2BC.%26aulast%3DKung%26aufirst%3DP.%2BH.%26aulast%3DWu%26aufirst%3DC.%2BC.%26atitle%3D3%252C4-Methylenedioxy-beta-nitrostyrene%2520inhibits%2520adhesion%2520and%2520migration%2520of%2520human%2520triple-negative%2520breast%2520cancer%2520cells%2520by%2520suppressing%2520beta1%2520integrin%2520function%2520and%2520surface%2520protein%2520disulfide%2520isomerase%26jtitle%3DBiochimie%26date%3D2015%26volume%3D110%26spage%3D81%26epage%3D92%26doi%3D10.1016%2Fj.biochi.2015.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffstrom, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turmel, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein disulfide isomerase suppress apoptosis induced by misfolded proteins</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">900</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1038/nchembio.467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1038%2Fnchembio.467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=21079601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVansLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=900-906&author=B.+G.+Hoffstromauthor=A.+Kaplanauthor=R.+Letsoauthor=R.+S.+Schmidauthor=G.+J.+Turmelauthor=D.+C.+Loauthor=B.+R.+Stockwell&title=Inhibitors+of+protein+disulfide+isomerase+suppress+apoptosis+induced+by+misfolded+proteins&doi=10.1038%2Fnchembio.467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein disulfide isomerase suppress apoptosis induced by misfolded proteins</span></div><div class="casAuthors">Hoffstrom, Benjamin G.; Kaplan, Anna; Letso, Reka; Schmid, Ralf S.; Turmel, Gregory J.; Lo, Donald C.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">900-906</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A hallmark of many neurodegenerative diseases is accumulation of misfolded proteins within neurons, leading to cellular dysfunction and cell death.  Although several mechanisms have been proposed to link protein misfolding to cellular toxicity, the connection remains enigmatic.  Here, we report a cell death pathway involving protein disulfide isomerase (PDI), a protein chaperone that catalyzes isomerization, redn. and oxidn. of disulfides.  Through a small mol. screening approach, we discovered five structurally distinct compds. that prevent apoptosis induced by mutant huntingtin protein.  Using modified Huisgen cycloaddn. chem., we then identified PDI as the mol. target of these small mols.  Expression of polyglutamine-expanded huntingtin exon 1 in PC12 cells caused PDI to accumulate at mitochondrial-assocd. ER membranes and trigger apoptotic cell death via mitochondrial outer-membrane permeabilization.  Inhibiting PDI in rat brain cells suppressed the toxicity of mutant huntingtin exon 1 and Aβ peptides processed from the amyloid precursor protein.  This pro-apoptotic function of PDI represents a new mechanism linking protein misfolding and apoptotic cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWvM36lxKwEbVg90H21EOLACvtfcHk0ljlOMCKGHYJew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVansLbN&md5=c8b685fcf565dfb37dcff4097e866481</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.467%26sid%3Dliteratum%253Aachs%26aulast%3DHoffstrom%26aufirst%3DB.%2BG.%26aulast%3DKaplan%26aufirst%3DA.%26aulast%3DLetso%26aufirst%3DR.%26aulast%3DSchmid%26aufirst%3DR.%2BS.%26aulast%3DTurmel%26aufirst%3DG.%2BJ.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DInhibitors%2520of%2520protein%2520disulfide%2520isomerase%2520suppress%2520apoptosis%2520induced%2520by%2520misfolded%2520proteins%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D900%26epage%3D906%26doi%3D10.1038%2Fnchembio.467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butkevich, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zandi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petasis, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16348</span>– <span class="NLM_lpage">16353</span>, <span class="refDoi"> DOI: 10.1073/pnas.1205226109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.1205226109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22988091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16348-16353&author=S.+Xuauthor=A.+N.+Butkevichauthor=R.+Yamadaauthor=Y.+Zhouauthor=B.+Debnathauthor=R.+Duncanauthor=E.+Zandiauthor=N.+A.+Petasisauthor=N.+Neamati&title=Discovery+of+an+orally+active+small-molecule+irreversible+inhibitor+of+protein+disulfide+isomerase+for+ovarian+cancer+treatment&doi=10.1073%2Fpnas.1205226109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span></div><div class="casAuthors">Xu, Shili; Butkevich, Alexey N.; Yamada, Roppei; Zhou, Yu; Debnath, Bikash; Duncan, Roger; Zandi, Ebrahim; Petasis, Nicos A.; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16348-16353, S16348/1-S16348/18</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI), an endoplasmic reticulum chaper-one protein, catalyzes disulfide bond breakage, formation, and rearrangement.  The effect of PDI inhibition on ovarian cancer progression is not yet clear, and there is a need for potent selective, and safe small-mol. inhibitors of PDI.  Here, we report a class of propynoic acid carbamoyl Me amides (PACMAs) that are active against a panel of human ovarian cancer cell lines.  Using fluorescent derivs., 2D gel electrophoresis, and MS, we established that one of the most active analogs, acts as an irreversible small-mol. inhibitor of PDI, forming a covalent bond with the active site cysteines of PDI.  We also showed that PDI activity is essential for the survival and proliferation of human ovarian cancer cells.  In vivo that compd. showed tumor targeting ability and significantly suppressed ovarian tumor growth without causing toxicity to normal tissues.  These irreversible small-mol. PDI inhibitors represent an important approach for the development of targeted anticancer agents for ovarian cancer therapy, and they can also serve as useful probes for investigating the biol. of PDI-implicated pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozZP4n4am-hLVg90H21EOLACvtfcHk0ljq6VwgXRiiVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE&md5=26801591b73d67710e06ef184ee9ddde</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1205226109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1205226109%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DButkevich%26aufirst%3DA.%2BN.%26aulast%3DYamada%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DDuncan%26aufirst%3DR.%26aulast%3DZandi%26aufirst%3DE.%26aulast%3DPetasis%26aufirst%3DN.%2BA.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520active%2520small-molecule%2520irreversible%2520inhibitor%2520of%2520protein%2520disulfide%2520isomerase%2520for%2520ovarian%2520cancer%2520treatment%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16348%26epage%3D16353%26doi%3D10.1073%2Fpnas.1205226109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Won, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span> <span> </span><span class="NLM_article-title">Protein disulfide isomerase inhibition synergistically enhances the efficacy of sorafenib for hepatocellular carcinoma</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">868</span>, <span class="refDoi"> DOI: 10.1002/hep.29237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1002%2Fhep.29237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28439950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOhsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=855-868&author=J.+K.+Wonauthor=S.+J.+Yuauthor=C.+Y.+Hwangauthor=S.+H.+Choauthor=S.+M.+Parkauthor=K.+Kimauthor=W.+M.+Choiauthor=H.+Choauthor=E.+J.+Choauthor=J.+H.+Leeauthor=K.+B.+Leeauthor=Y.+J.+Kimauthor=K.+S.+Suhauthor=J.+J.+Jangauthor=C.+Y.+Kimauthor=J.+H.+Yoonauthor=K.+H.+Cho&title=Protein+disulfide+isomerase+inhibition+synergistically+enhances+the+efficacy+of+sorafenib+for+hepatocellular+carcinoma&doi=10.1002%2Fhep.29237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protein disulfide isomerase inhibition synergistically enhances the efficacy of sorafenib for hepatocellular carcinoma</span></div><div class="casAuthors">Won, Jae-Kyung; Yu, Su Jong; Hwang, Chae Young; Cho, Sung-Hwan; Park, Sang-Min; Kim, Kwangsoo; Choi, Won-Mook; Cho, Hyeki; Cho, Eun Ju; Lee, Jeong-Hoon; Lee, Kyung Bun; Kim, Yoon Jun; Suh, Kyung-Suk; Jang, Ja-June; Kim, Chung Yong; Yoon, Jung-Hwan; Cho, Kwang-Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">855-868</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Sorafenib is the only approved targeted drug for hepatocellular carcinoma (HCC), but its effect on patients' survival gain is limited and varies over a wide range depending on pathogenetic conditions.  Thus, enhancing the efficacy of sorafenib and finding a reliable predictive biomarker are crucial to achieve efficient control of HCCs.  In this study, we utilized a systems approach by combining transcriptome anal. of the mRNA changes in HCC cell lines in response to sorafenib with network anal. to investigate the action and resistance mechanism of sorafenib.  Gene list functional enrichment anal. and gene set enrichment anal. revealed that proteotoxic stress and apoptosis modules are activated in the presence of sorafenib.  Further anal. of the endoplasmic reticulum stress network model, combined with in vitro expts., showed that introducing an addnl. stress by treating the orally active protein disulfide isomerase (PDI) inhibitor (PACMA 31) can synergistically increase the efficacy of sorafenib in vitro and in vivo, which was confirmed using a mouse xenograft model.  We also found that HCC patients with high PDI expression show resistance to sorafenib and poor clin. outcomes, compared to the low-PDI-expression group.  Conclusion: These results suggest that PDI is a promising therapeutic target for enhancing the efficacy of sorafenib and can also be a biomarker for predicting sorafenib responsiveness. (Hepatol. 2017;66:855-868).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI9MWIoMmE8rVg90H21EOLACvtfcHk0ljq6VwgXRiiVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOhsb3E&md5=321aa98f9948c3e949dbd9a445ef5821</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fhep.29237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.29237%26sid%3Dliteratum%253Aachs%26aulast%3DWon%26aufirst%3DJ.%2BK.%26aulast%3DYu%26aufirst%3DS.%2BJ.%26aulast%3DHwang%26aufirst%3DC.%2BY.%26aulast%3DCho%26aufirst%3DS.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BM.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DW.%2BM.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DE.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DK.%2BB.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DSuh%26aufirst%3DK.%2BS.%26aulast%3DJang%26aufirst%3DJ.%2BJ.%26aulast%3DKim%26aufirst%3DC.%2BY.%26aulast%3DYoon%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DK.%2BH.%26atitle%3DProtein%2520disulfide%2520isomerase%2520inhibition%2520synergistically%2520enhances%2520the%2520efficacy%2520of%2520sorafenib%2520for%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2017%26volume%3D66%26spage%3D855%26epage%3D868%26doi%3D10.1002%2Fhep.29237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, C.</span></span> <span> </span><span class="NLM_article-title">Challenges in the evaluation of thiol-reactive inhibitors of human protein disulfide Isomerase</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2017.04.367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.freeradbiomed.2017.04.367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28465261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFCisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2017&pages=741-749&author=C.+K.+Fosterauthor=C.+Thorpe&title=Challenges+in+the+evaluation+of+thiol-reactive+inhibitors+of+human+protein+disulfide+Isomerase&doi=10.1016%2Fj.freeradbiomed.2017.04.367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges in the evaluation of thiol-reactive inhibitors of human protein disulfide isomerase</span></div><div class="casAuthors">Foster, Celia K.; Thorpe, Colin</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">741-749</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This paper addresses how to evaluate the efficacy of the growing inventory of thiol-reactive inhibitors of mammalian protein disulfide isomerase (PDI) under realistic concns. of potentially competing thiol-contg. peptides and proteins.  For this purpose, we introduced a variant of the widely-used reductase assay by using a com.-available cysteine deriv., (BODIPY FL L-cystine; BD-SS), that yields a 55-fold increase in fluorescence (excitation/emission; 490/513 nm) on scission of the disulfide bond.  This plate reader-compatible method detects human PDI down to 5-10 nM, can utilize a range of thiol substrates (including 5 μM dithiothreitol, 10 μM reduced RNase thiols, and 5 mM glutathione; GSH), and can operate from pH 6-9.5 in a variety of buffers.  PDI assays often employ low micromolar levels of substrates leading to ambiguities when thiol-directed inhibitors are evaluated.  The present work utilized 5 mM GSH for both pre-incubation and assay phases to more realistically reflect the high concn. of thiols that an inhibitor would encounter intracellularly.  Extracellular PDI faced a much lower concn. of potentially competing thiols; to assess reductase activity under these conditions, the pre-reduced PDI was treated with an inhibitor and then the fluorescence increase upon redn. of BD-SS was followed in the absence of addnl. competing thiols.  Both assay modes were tested with 4 mechanistically diverse PDI inhibitors.  Two reversible reagents, 3,4-methylenedioxy-β-nitrostyrene (MNS) and the arsenical, APAO, were found to be strong inhibitors of PDI in the absence of competing thiols, but were ineffective in the presence of 5 mM GSH.  A further examn. of the nitrostyrene showed that MNS not only formed facile Michael adducts with GSH, but also with the thiols of unfolded proteins (Kd values of 7 and <0.1 μM, resp.) suggesting the existence of multiple potential intracellular targets for this membrane-permeant reagent.  The inhibition of PDI by the irreversible alkylating agent, the chloroacetamide, 16F16, was found to be only modestly attenuated by 5 mM GSH.  Finally, the thiol-independent flavonoid inhibitor, quercetin-3-O-rutinoside (rutin), was found to show equal efficacy in reoxidn. and turnover assay types.  Thus, this work provides a framework to evaluate inhibitors that may target the CxxC motifs of PDI and addresses some of the complexities in the interpretation of the behavior of thiol-directed reagents in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfLqAX92kPjLVg90H21EOLACvtfcHk0lh_7ih5NMwjpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFCisbc%253D&md5=0b4846f207b567e66275bb8f6ddec5a0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2017.04.367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2017.04.367%26sid%3Dliteratum%253Aachs%26aulast%3DFoster%26aufirst%3DC.%2BK.%26aulast%3DThorpe%26aufirst%3DC.%26atitle%3DChallenges%2520in%2520the%2520evaluation%2520of%2520thiol-reactive%2520inhibitors%2520of%2520human%2520protein%2520disulfide%2520Isomerase%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2017%26volume%3D108%26spage%3D741%26epage%3D749%26doi%3D10.1016%2Fj.freeradbiomed.2017.04.367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aye, Y.</span></span> <span> </span><span class="NLM_article-title">Privileged electrophile sensors: A resource for covalent drug development</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.chembiol.2017.05.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28648380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=787-800&author=M.+J.+C.+Longauthor=Y.+Aye&title=Privileged+electrophile+sensors%3A+A+resource+for+covalent+drug+development&doi=10.1016%2Fj.chembiol.2017.05.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged Electrophile Sensors: A Resource for Covalent Drug Development</span></div><div class="casAuthors">Long, Marcus John Curtis; Aye, Yimon</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">787-800</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">This Perspective delineates how redox signaling affects the activity of specific enzyme isoforms and how this property may be harnessed for rational drug design.  Covalent drugs have resurged in recent years and several reports have extolled the general virtues of developing irreversible inhibitors.  Indeed, many modern pharmaceuticals contain electrophilic appendages.  Several invoke a warhead that hijacks active-site nucleophiles whereas others take advantage of spectator nucleophilic side chains that do not participate in enzymic chem., but are poised to bind/react with electrophiles.  The latest data suggest that innate electrophile sensing-which enables rapid reaction with an endogenous signaling electrophile-is a quintessential resource for the development of covalent drugs.  For instance, based on recent work documenting isoform-specific electrophile sensing, isoenzyme non-specific drugs may be converted to isoenzyme-specific analogs by hijacking privileged first-responder electrophile-sensing cysteines.  Because this approach targets functionally relevant cysteines, we can simultaneously harness previously untapped moonlighting roles of enzymes linked to redox sensing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWQ9w0b7pay7Vg90H21EOLACvtfcHk0lh_7ih5NMwjpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E&md5=b7edc384ac48df7465303f33f7b1cca8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.023%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DAye%26aufirst%3DY.%26atitle%3DPrivileged%2520electrophile%2520sensors%253A%2520A%2520resource%2520for%2520covalent%2520drug%2520development%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D787%26epage%3D800%26doi%3D10.1016%2Fj.chembiol.2017.05.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogunrinu, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontheimer, H.</span></span> <span> </span><span class="NLM_article-title">Hypoxia increases the dependence of glioma cells on glutathione</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">37716</span>– <span class="NLM_lpage">37724</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.161190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M110.161190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=20858898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=37716-37724&author=T.+A.+Ogunrinuauthor=H.+Sontheimer&title=Hypoxia+increases+the+dependence+of+glioma+cells+on+glutathione&doi=10.1074%2Fjbc.M110.161190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia increases the dependence of glioma cells on glutathione</span></div><div class="casAuthors">Ogunrinu, Toyin Adeyemi; Sontheimer, Harald</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">37716-37724</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Glutathione (GSH) is an essential antioxidant responsible for the maintenance of intracellular redox homeostasis.  As tumors outgrow their blood supply and become hypoxic, their redox homeostasis is challenged by the prodn. of nitric oxide and reactive oxygen species (ROS).  In gliomas, the sustained import of L-cystine via the L-cystine/L-glutamate exchanger, system x[Formula: see text], is rate-limiting for the synthesis of GSH.  We show that hypoxia causes a significant increase in NO and ROS but without affecting glioma cell growth.  This is explained by a concomitant increase in the utilization of GSH, which is accompanied by an increase in the cell-surface expression of xCT, the catalytic subunit of system x[Formula: see text], and L-cystine uptake.  Growth was inhibited when GSH synthesis was blocked by buthionine sulfoximine (BSO), an inhibitor of the enzyme required for GSH synthesis, or when cells were deprived of L-cystine.  These findings suggest that glioma cells show an increased requirement for GSH to maintain growth under hypoxic conditions.  Therefore, approaches that limit GSH synthesis such as blocking system may be considered as an adjuvant to radiation or chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvUfkv7LQy2rVg90H21EOLACvtfcHk0lh_7ih5NMwjpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt73E&md5=15779fe20696b06e00499cb5899710e8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.161190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.161190%26sid%3Dliteratum%253Aachs%26aulast%3DOgunrinu%26aufirst%3DT.%2BA.%26aulast%3DSontheimer%26aufirst%3DH.%26atitle%3DHypoxia%2520increases%2520the%2520dependence%2520of%2520glioma%2520cells%2520on%2520glutathione%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D37716%26epage%3D37724%26doi%3D10.1074%2Fjbc.M110.161190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, P.</span></span> <span> </span><span class="NLM_article-title">Unraveling the potential role of glutathione in multiple forms of cell death in cancer therapy</span>. <i>Oxid. Med. Cell. Longevity</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2019</i></span>, <span class="NLM_elocation-id">3150145</span> <span class="refDoi"> DOI: 10.1155/2019/3150145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1155%2F2019%2F3150145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=31281572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BB3MzkvFKltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2019&publication_year=2019&author=H.+Lvauthor=C.+Zhenauthor=J.+Liuauthor=P.+Yangauthor=L.+Huauthor=P.+Shang&title=Unraveling+the+potential+role+of+glutathione+in+multiple+forms+of+cell+death+in+cancer+therapy&doi=10.1155%2F2019%2F3150145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling the Potential Role of Glutathione in Multiple Forms of Cell Death in Cancer Therapy</span></div><div class="casAuthors">Lv Huanhuan; Zhen Chenxiao; Liu Junyu; Yang Pengfei; Lv Huanhuan; Zhen Chenxiao; Liu Junyu; Yang Pengfei; Shang Peng; Lv Huanhuan; Hu Lijiang; Lv Huanhuan; Yang Pengfei; Shang Peng; Lv Huanhuan; Zhen Chenxiao; Liu Junyu; Yang Pengfei; Shang Peng</div><div class="citationInfo"><span class="NLM_cas:title">Oxidative medicine and cellular longevity</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">2019</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3150145</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Glutathione is the principal intracellular antioxidant buffer against oxidative stress and mainly exists in the forms of reduced glutathione (GSH) and oxidized glutathione (GSSG).  The processes of glutathione synthesis, transport, utilization, and metabolism are tightly controlled to maintain intracellular glutathione homeostasis and redox balance.  As for cancer cells, they exhibit a greater ROS level than normal cells in order to meet the enhanced metabolism and vicious proliferation; meanwhile, they also have to develop an increased antioxidant defense system to cope with the higher oxidant state.  Growing numbers of studies have implicated that altering the glutathione antioxidant system is associated with multiple forms of programmed cell death in cancer cells.  In this review, we firstly focus on glutathione homeostasis from the perspectives of glutathione synthesis, distribution, transportation, and metabolism.  Then, we discuss the function of glutathione in the antioxidant process.  Afterwards, we also summarize the recent advance in the understanding of the mechanism by which glutathione plays a key role in multiple forms of programmed cell death, including apoptosis, necroptosis, ferroptosis, and autophagy.  Finally, we highlight the glutathione-targeting therapeutic approaches toward cancers.  A comprehensive review on the glutathione homeostasis and the role of glutathione depletion in programmed cell death provide insight into the redox-based research concerning cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2QtG6nKnyx8zK395ZObgVfW6udTcc2eYW6-lv9g9bd7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzkvFKltA%253D%253D&md5=d9433b9d3229d62df295289a94505636</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1155%2F2019%2F3150145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2019%252F3150145%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DH.%26aulast%3DZhen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DShang%26aufirst%3DP.%26atitle%3DUnraveling%2520the%2520potential%2520role%2520of%2520glutathione%2520in%2520multiple%2520forms%2520of%2520cell%2520death%2520in%2520cancer%2520therapy%26jtitle%3DOxid.%2520Med.%2520Cell.%2520Longevity%26date%3D2019%26volume%3D2019%26doi%3D10.1155%2F2019%2F3150145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tachibana, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahr Winther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appenzeller-Herzog, C.</span></span> <span> </span><span class="NLM_article-title">Intracellular glutathione pools are heterogeneously concentrated</span>. <i>Redox Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1016/j.redox.2013.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.redox.2013.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=24251119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=508-513&author=D.+Monteroauthor=C.+Tachibanaauthor=J.+Rahr+Wintherauthor=C.+Appenzeller-Herzog&title=Intracellular+glutathione+pools+are+heterogeneously+concentrated&doi=10.1016%2Fj.redox.2013.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular glutathione pools are heterogeneously concentrated</span></div><div class="casAuthors">Montero, Davide; Tachibana, Christine; Rahr Winther, Jakob; Appenzeller-Herzog, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Redox Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">508-513</span>CODEN:
                <span class="NLM_cas:coden">RBEIB3</span>;
        ISSN:<span class="NLM_cas:issn">2213-2317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Glutathione is present in millimolar concns. in the cell, but its relative distribution among cellular compartments remains elusive.  We have chosen the endoplasmic reticulum (ER) as an example organelle to study compartment-specific glutathione levels.  Using a glutaredoxin sensor (sCGrx1pER), which rapidly and specifically equilibrates with the reduced glutathione (GSH)-glutathione disulfide (GSSG) redox couple with known equil. const., we showed that the [GSH]:[GSSG] ratio in the ER of intact HeLa cells is less than 7:1.  Taking into consideration the previously detd. value for [GSH]2:[GSSG] in the ER of 83 mM, this translates into a total glutathione concn. in the ER ([GStot]=[GSH]+2[GSSG]) of greater than 15 mM.  Since the integrated, intracellular [GStot] was measured as ∼7 mM, we conclude the existence of a [GStot] gradient across the ER membrane.  A possible homeostatic mechanism by which cytosol-derived glutathione is trapped in the ER is discussed.  We propose a high [GStot] as a distinguishing feature of the ER environment compared to the extracellular space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtUlaQYPjRKLVg90H21EOLACvtfcHk0lgbT6TJ7WjE3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFGlsg%253D%253D&md5=cb40f4e69e51f8b6f09f133f7b1b87f3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.redox.2013.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.redox.2013.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DMontero%26aufirst%3DD.%26aulast%3DTachibana%26aufirst%3DC.%26aulast%3DRahr%2BWinther%26aufirst%3DJ.%26aulast%3DAppenzeller-Herzog%26aufirst%3DC.%26atitle%3DIntracellular%2520glutathione%2520pools%2520are%2520heterogeneously%2520concentrated%26jtitle%3DRedox%2520Biol.%26date%3D2013%26volume%3D1%26spage%3D508%26epage%3D513%26doi%3D10.1016%2Fj.redox.2013.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boskovic, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freedy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanathan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa
Lazu, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calarese, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Love, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span> <span> </span><span class="NLM_article-title">Inhibition of zinc-dependent histone deacetylases with a chemically triggered electrophile</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1844</span>– <span class="NLM_lpage">1851</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslyqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1844-1851&author=Z.+V.+Boskovicauthor=M.+M.+Kempauthor=A.+M.+Freedyauthor=V.+S.+Viswanathanauthor=M.+S.+Popauthor=J.+H.+Fullerauthor=N.+M.+Martinezauthor=S.+O.+Figueroa%0ALazuauthor=J.+A.+Hongauthor=T.+A.+Lewisauthor=D.+Calareseauthor=J.+D.+Loveauthor=A.+Vetereauthor=S.+C.+Almoauthor=S.+L.+Schreiberauthor=A.+N.+Koehler&title=Inhibition+of+zinc-dependent+histone+deacetylases+with+a+chemically+triggered+electrophile&doi=10.1021%2Facschembio.6b00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Zinc-Dependent Histone Deacetylases with a Chemically Triggered Electrophile</span></div><div class="casAuthors">Boskovic, Zarko V.; Kemp, Melissa M.; Freedy, Allyson M.; Viswanathan, Vasanthi S.; Pop, Marius S.; Fuller, Jason H.; Martinez, Nicole M.; Figueroa Lazu, Samuel O.; Hong, Jiyoung A.; Lewis, Timothy A.; Calarese, Daniel; Love, James D.; Vetere, Amedeo; Almo, Steven C.; Schreiber, Stuart L.; Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1844-1851</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Unbiased binding assays involving small-mol. microarrays were used to identify compds. that display unique patterns of selectivity among members of the zinc-dependent histone deacetylase family of enzymes.  A novel, hydroxyquinoline-contg. compd., BRD4354, was shown to preferentially inhibit activity of HDAC5 and HDAC9 in vitro.  Inhibition of deacetylase activity appears to be time-dependent and reversible.  Mechanistic studies suggest that the compd. undergoes zinc-catalyzed decompn. to an ortho-quinone methide, which covalently modifies nucleophilic cysteines within the proteins.  The covalent nature of the compd.-enzyme interaction has been demonstrated in expts. with biotinylated probe compd. and with electrospray ionization-mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGhTWSlnioHbVg90H21EOLACvtfcHk0ljjDMu249uVUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslyqsr8%253D&md5=f12972109eb4daed651f1984ed30382d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00012%26sid%3Dliteratum%253Aachs%26aulast%3DBoskovic%26aufirst%3DZ.%2BV.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DFreedy%26aufirst%3DA.%2BM.%26aulast%3DViswanathan%26aufirst%3DV.%2BS.%26aulast%3DPop%26aufirst%3DM.%2BS.%26aulast%3DFuller%26aufirst%3DJ.%2BH.%26aulast%3DMartinez%26aufirst%3DN.%2BM.%26aulast%3DFigueroa%2BLazu%26aufirst%3DS.%2BO.%26aulast%3DHong%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DCalarese%26aufirst%3DD.%26aulast%3DLove%26aufirst%3DJ.%2BD.%26aulast%3DVetere%26aufirst%3DA.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DInhibition%2520of%2520zinc-dependent%2520histone%2520deacetylases%2520with%2520a%2520chemically%2520triggered%2520electrophile%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1844%26epage%3D1851%26doi%3D10.1021%2Facschembio.6b00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLean, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukasczyk, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fordham, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsay, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of covalent inhibitors for MIF tautomerase via cocrystal structures with phantom hits from virtual screening</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6717</span>– <span class="NLM_lpage">6720</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.09.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bmcl.2009.09.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19836948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6717-6720&author=L.+R.+McLeanauthor=Y.+Zhangauthor=H.+Liauthor=Z.+Liauthor=U.+Lukasczykauthor=Y.+M.+Choiauthor=Z.+Hanauthor=J.+Priscoauthor=J.+Fordhamauthor=J.+T.+Tsayauthor=S.+Reilingauthor=R.+J.+Vazauthor=Y.+Li&title=Discovery+of+covalent+inhibitors+for+MIF+tautomerase+via+cocrystal+structures+with+phantom+hits+from+virtual+screening&doi=10.1016%2Fj.bmcl.2009.09.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of covalent inhibitors for MIF tautomerase via cocrystal structures with phantom hits from virtual screening</span></div><div class="casAuthors">McLean, Larry R.; Zhang, Ying; Li, Hua; Li, Ziyu; Lukasczyk, Ulrike; Choi, Yong-Mi; Han, Zuoning; Prisco, Joy; Fordham, Jeremy; Tsay, Joseph T.; Reiling, Stephan; Vaz, Roy J.; Li, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6717-6720</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Biochem. and X-ray crystallog. studies confirmed that hydroxyquinoline derivs. identified by virtual screening were actually covalent inhibitors of the MIF tautomerase.  Adducts were formed by N-alkylation of the Pro-1 at the catalytic site with a loss of an amino group of the inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry3QrnMeARVbVg90H21EOLACvtfcHk0ljSzo_7quojtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7O&md5=899b261a6b7f85dbd93fdf32e0565ed1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.09.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.09.106%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DL.%2BR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLukasczyk%26aufirst%3DU.%26aulast%3DChoi%26aufirst%3DY.%2BM.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DPrisco%26aufirst%3DJ.%26aulast%3DFordham%26aufirst%3DJ.%26aulast%3DTsay%26aufirst%3DJ.%2BT.%26aulast%3DReiling%26aufirst%3DS.%26aulast%3DVaz%26aufirst%3DR.%2BJ.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520covalent%2520inhibitors%2520for%2520MIF%2520tautomerase%2520via%2520cocrystal%2520structures%2520with%2520phantom%2520hits%2520from%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6717%26epage%3D6720%26doi%3D10.1016%2Fj.bmcl.2009.09.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zapatero, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colmenarejo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Los Frailes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramon, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel inhibitors of the tautomerase activity of macrophage migration inhibitory factor (MIF)</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1177/1087057116633997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1177%2F1087057116633997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26933127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=446-458&author=M.+C.+Zapateroauthor=P.+Perezauthor=M.+J.+Vazquezauthor=G.+Colmenarejoauthor=M.+de+Los+Frailesauthor=F.+Ramon&title=Discovery+of+novel+inhibitors+of+the+tautomerase+activity+of+macrophage+migration+inhibitory+factor+%28MIF%29&doi=10.1177%2F1087057116633997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel inhibitors of the tautomerase activity of macrophage migration inhibitory factor (MIF)</span></div><div class="casAuthors">Zapatero, Maria Cleofe; Perez, Paloma; Vazquez, Maria Jesus; Colmenarejo, Gonzalo; de los Frailes, Maite; Ramon, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">446-458</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Macrophage migration inhibitory factor (MIF) is a proinflammatory cytokine assocd. with multiple diseases, including neurodegenerative disorders.  With the ultimate goal of providing novel chemotypes as starting points for development of disease-modifying therapeutics for neurodegeneration, we endeavored to screen the GSK compd. collection for MIF inhibitors using a miniaturized, activity-based kinetic assay.  The assay monitors the increase in absorbance at 320 nm resulting from keto-to-enol tautomerization of 4-hydroxyphenylpyruvate, a reaction catalyzed by MIF.  We ran a full- diversity screen evaluating the inhibitory activity of 1.6 million compds.  Primary hits were confirmed and retested in an orthogonal assay measuring tautomerization of L-dopachrome Me ester by the decrease in absorbance at 475 nm in kinetic mode.  Selected compds. were progressed to medium-throughput mode-of-inhibition studies, which included time dependence, enzyme concn. dependence, and reversibility of their inhibitory effect.  With these results and after inspection of the physicochem. properties of compds., 17 chemotypes were prioritized and progressed to further stages of validation and characterization to better assess their therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5fx7eEUwg47Vg90H21EOLACvtfcHk0ljSzo_7quojtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWrsbnF&md5=7e256ec74bd95d9d6f8763f3134f41fd</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1087057116633997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116633997%26sid%3Dliteratum%253Aachs%26aulast%3DZapatero%26aufirst%3DM.%2BC.%26aulast%3DPerez%26aufirst%3DP.%26aulast%3DVazquez%26aufirst%3DM.%2BJ.%26aulast%3DColmenarejo%26aufirst%3DG.%26aulast%3Dde%2BLos%2BFrailes%26aufirst%3DM.%26aulast%3DRamon%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520novel%2520inhibitors%2520of%2520the%2520tautomerase%2520activity%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D446%26epage%3D458%26doi%3D10.1177%2F1087057116633997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouertatani-Sakouhi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Turk, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauvet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinar
Karpinar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Roy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhagen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calandra, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweckstetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lashuel, H. A.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of novel classes of macrophage migration inhibitory factor (MIF) inhibitors with distinct mechanisms of action</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">26581</span>– <span class="NLM_lpage">26598</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.113951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M110.113951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=20516071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVahtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=26581-26598&author=H.+Ouertatani-Sakouhiauthor=F.+El-Turkauthor=B.+Fauvetauthor=M.+K.+Choauthor=D.+Pinar%0AKarpinarauthor=D.+Le+Royauthor=M.+Deworauthor=T.+Rogerauthor=J.+Bernhagenauthor=T.+Calandraauthor=M.+Zweckstetterauthor=H.+A.+Lashuel&title=Identification+and+characterization+of+novel+classes+of+macrophage+migration+inhibitory+factor+%28MIF%29+inhibitors+with+distinct+mechanisms+of+action&doi=10.1074%2Fjbc.M110.113951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Novel Classes of Macrophage Migration Inhibitory Factor (MIF) Inhibitors with Distinct Mechanisms of Action</span></div><div class="casAuthors">Ouertatani-Sakouhi, Hajer; El-Turk, Farah; Fauvet, Bruno; Cho, Min-Kyu; Pinar Karpinar, Damla; Le Roy, Didier; Dewor, Manfred; Roger, Thierry; Bernhagen, Juergen; Calandra, Thierry; Zweckstetter, Markus; Lashuel, Hilal A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">26581-26598</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Macrophage migration inhibitory factor (MIF), a proinflammatory cytokine, is considered an attractive therapeutic target in multiple inflammatory and autoimmune disorders.  In addn. to its known biol. activities, MIF can also function as a tautomerase.  Several small mols. have been reported to be effective inhibitors of MIF tautomerase activity in vitro.  Herein we employed a robust activity-based assay to identify different classes of novel inhibitors of the catalytic and biol. activities of MIF.  Several novel chem. classes of inhibitors of the catalytic activity of MIF with IC50 values in the range of 0.2-15.5 μM were identified and validated.  The interaction site and mechanism of action of these inhibitors were defined using structure-activity studies and a battery of biochem. and biophys. methods.  MIF inhibitors emerging from these studies could be divided into three categories based on their mechanism of action: (1) mols. that covalently modify the catalytic site at the N-terminal proline residue, Pro1; (2) a novel class of catalytic site inhibitors; and finally (3) mols. that disrupt the trimeric structure of MIF.  Importantly, all inhibitors demonstrated total inhibition of MIF-mediated glucocorticoid overriding and AKT phosphorylation, whereas ebselen, a trimer-disrupting inhibitor, addnl. acted as a potent hyperagonist in MIF-mediated chemotactic migration.  The identification of biol. active compds. with known toxicity, pharmacokinetic properties, and biol. activities in vivo should accelerate the development of clin. relevant MIF inhibitors.  Furthermore, the diversity of chem. structures and mechanisms of action of our inhibitors makes them ideal mechanistic probes for elucidating the structure-function relationships of MIF and to further det. the role of the oligomerization state and catalytic activity of MIF in regulating the function(s) of MIF in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqje3LpgpMop7Vg90H21EOLACvtfcHk0lj2DWokmsCe6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVahtbfK&md5=e25d6c06b2cf0b6454c00d23f9b7fbf4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.113951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.113951%26sid%3Dliteratum%253Aachs%26aulast%3DOuertatani-Sakouhi%26aufirst%3DH.%26aulast%3DEl-Turk%26aufirst%3DF.%26aulast%3DFauvet%26aufirst%3DB.%26aulast%3DCho%26aufirst%3DM.%2BK.%26aulast%3DPinar%2BKarpinar%26aufirst%3DD.%26aulast%3DLe%2BRoy%26aufirst%3DD.%26aulast%3DDewor%26aufirst%3DM.%26aulast%3DRoger%26aufirst%3DT.%26aulast%3DBernhagen%26aufirst%3DJ.%26aulast%3DCalandra%26aufirst%3DT.%26aulast%3DZweckstetter%26aufirst%3DM.%26aulast%3DLashuel%26aufirst%3DH.%2BA.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520novel%2520classes%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529%2520inhibitors%2520with%2520distinct%2520mechanisms%2520of%2520action%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D26581%26epage%3D26598%26doi%3D10.1074%2Fjbc.M110.113951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valhondo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Irregularities in enzyme assays: The case of macrophage migration inhibitory factor</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2764</span>– <span class="NLM_lpage">2767</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bmcl.2016.04.074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=27156768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2764-2767&author=J.+A.+Cisnerosauthor=M.+J.+Robertsonauthor=M.+Valhondoauthor=W.+L.+Jorgensen&title=Irregularities+in+enzyme+assays%3A+The+case+of+macrophage+migration+inhibitory+factor&doi=10.1016%2Fj.bmcl.2016.04.074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Irregularities in enzyme assays: The case of macrophage migration inhibitory factor</span></div><div class="casAuthors">Cisneros, Jose A.; Robertson, Michael J.; Valhondo, Margarita; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2764-2767</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Inhibitors of human macrophage migration inhibitory factor (MIF) previously reported in the literature have been reexamd. by synthesis, assaying for tautomerase activity, and protein crystallog.  Substantial inconsistencies between prior and current assay results are noted.  They appear to arise from difficulties with the tautomerase substrates, soly. issues, and esp. covalent inhibition.  Incubation time variation shows that several compds. are covalent or slow-binding inhibitors.  Two protein crystal structures are provided; one confirms that a twice-discovered compd. is a covalent inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgbtvPyvrjS7Vg90H21EOLACvtfcHk0lj2DWokmsCe6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCqtLs%253D&md5=a232a355d251396f4d19e79954bf469b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.074%26sid%3Dliteratum%253Aachs%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DValhondo%26aufirst%3DM.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DIrregularities%2520in%2520enzyme%2520assays%253A%2520The%2520case%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2764%26epage%3D2767%26doi%3D10.1016%2Fj.bmcl.2016.04.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferchen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skibbe, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhold, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnovitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibition of STAT/TET1 axis as a therapeutic strategy for acute myeloid leukemia</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">2099</span> <span class="refDoi"> DOI: 10.1038/s41467-017-02290-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1038%2Fs41467-017-02290-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29235481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC1MzgvFKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=X.+Jiangauthor=C.+Huauthor=K.+Ferchenauthor=J.+Nieauthor=X.+Cuiauthor=C.+H.+Chenauthor=L.+Chengauthor=Z.+Zuoauthor=W.+Seibelauthor=C.+Heauthor=Y.+Tangauthor=J.+R.+Skibbeauthor=M.+Wunderlichauthor=W.+C.+Reinholdauthor=L.+Dongauthor=C.+Shenauthor=S.+Arnovitzauthor=B.+Ulrichauthor=J.+Luauthor=H.+Wengauthor=R.+Suauthor=H.+Huangauthor=Y.+Wangauthor=C.+Liauthor=X.+Qinauthor=J.+C.+Mulloyauthor=Y.+Zhengauthor=J.+Diaoauthor=J.+Jinauthor=C.+Liauthor=P.+P.+Liuauthor=C.+Heauthor=Y.+Chenauthor=J.+Chen&title=Targeted+inhibition+of+STAT%2FTET1+axis+as+a+therapeutic+strategy+for+acute+myeloid+leukemia&doi=10.1038%2Fs41467-017-02290-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibition of STAT/TET1 axis as a therapeutic strategy for acute myeloid leukemia</span></div><div class="casAuthors">Jiang Xi; Hu Chao; Ferchen Kyle; Zuo Zhixiang; Skibbe Jennifer R; Dong Lei; Shen Chao; Lu Jiuwei; Weng Hengyou; Su Rui; Huang Huilin; Wang Yungui; Li Chenying; Qin Xi; Diao Jiajie; Chen Jianjun; Jiang Xi; Hu Chao; Arnovitz Stephen; Ulrich Bryan; Weng Hengyou; Wang Yungui; Chen Jianjun; Jiang Xi; Dong Lei; Shen Chao; Weng Hengyou; Su Rui; Huang Huilin; Li Chenying; Qin Xi; Chen Jianjun; Hu Chao; Wang Yungui; Li Chenying; Jin Jie; Nie Ji; Cui Xiaolong; He Chuan; Chen Chih-Hong; Chen Yuan; Cheng Liting; Tang Yixuan; Li Chong; Zuo Zhixiang; Seibel William; He Chunjiang; Wunderlich Mark; Mulloy James C; Zheng Yi; Reinhold William C; Liu Paul P</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2099</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Effective therapy of acute myeloid leukemia (AML) remains an unmet need.  DNA methylcytosine dioxygenase Ten-eleven translocation 1 (TET1) is a critical oncoprotein in AML.  Through a series of data analysis and drug screening, we identified two compounds (i.e., NSC-311068 and NSC-370284) that selectively suppress TET1 transcription and 5-hydroxymethylcytosine (5hmC) modification, and effectively inhibit cell viability in AML with high expression of TET1 (i.e., TET1-high AML), including AML carrying t(11q23)/MLL-rearrangements and t(8;21) AML.  NSC-311068 and especially NSC-370284 significantly repressed TET1-high AML progression in vivo.  UC-514321, a structural analog of NSC-370284, exhibited a more potent therapeutic effect and prolonged the median survival of TET1-high AML mice over three fold.  NSC-370284 and UC-514321 both directly target STAT3/5, transcriptional activators of TET1, and thus repress TET1 expression.  They also exhibit strong synergistic effects with standard chemotherapy.  Our results highlight the therapeutic potential of targeting the STAT/TET1 axis by selective inhibitors in AML treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRIMEzBUzrv5OR_TJajKxJDfW6udTcc2eZ15RJw9rT9dbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzgvFKrtA%253D%253D&md5=968bc63017ea5738ed944ec0b702622c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02290-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02290-w%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DFerchen%26aufirst%3DK.%26aulast%3DNie%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DZuo%26aufirst%3DZ.%26aulast%3DSeibel%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DSkibbe%26aufirst%3DJ.%2BR.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DReinhold%26aufirst%3DW.%2BC.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DArnovitz%26aufirst%3DS.%26aulast%3DUlrich%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DWeng%26aufirst%3DH.%26aulast%3DSu%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DP.%2BP.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DTargeted%2520inhibition%2520of%2520STAT%252FTET1%2520axis%2520as%2520a%2520therapeutic%2520strategy%2520for%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-02290-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavecchia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giovanni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel small molecule inhibitor targeting the frataxin/ubiquitin interaction via structure-based virtual screening and bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2861</span>– <span class="NLM_lpage">2873</span>, <span class="refDoi"> DOI: 10.1021/jm3017199</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3017199" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFKgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2861-2873&author=A.+Lavecchiaauthor=C.+Di+Giovanniauthor=C.+Cerchiaauthor=A.+Russoauthor=G.+Russoauthor=E.+Novellino&title=Discovery+of+a+novel+small+molecule+inhibitor+targeting+the+frataxin%2Fubiquitin+interaction+via+structure-based+virtual+screening+and+bioassays&doi=10.1021%2Fjm3017199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Small Molecule Inhibitor Targeting the Frataxin/Ubiquitin Interaction via Structure-Based Virtual Screening and Bioassays</span></div><div class="casAuthors">Lavecchia, Antonio; Di Giovanni, Carmen; Cerchia, Carmen; Russo, Annapina; Russo, Giulia; Novellino, Ettore</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2861-2873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Friedreich's ataxia (FRDA) is an autosomal recessive neuro- and cardiodegenerative disorder for which there are no proven effective treatments.  FRDA is caused by decreased expression and/or function of the mitochondrial protein frataxin.  Here, we report findings that frataxin is degraded via the ubiquitin-proteasomal pathway and that it is ubiquitinated at residue K147 in Calu-6 cells.  A theor. model of the frataxin-K147/Ub complex, constructed by combining bioinformatics interface predictions with information-driven docking, revealed a hitherto unnoticed, potential ubiquitin-binding domain in frataxin.  Through structure-based virtual screening and cell-based assays, we discovered a novel small mol. (compd. (+)-11) able to prevent frataxin ubiquitination and degrdn. (+)-11 was synthesized and tested for specific binding to frataxin by an UF-LC/MS based ligand-binding assay.  Follow-up scaffold-based searches resulted in the identification of a lead series with micromolar activity in disrupting the frataxin/Ub interaction.  This study also suggests that frataxin could be a potential target for FRDA drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhNCbVlCSkMbVg90H21EOLACvtfcHk0lh4zIMNdRKXAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFKgt7w%253D&md5=669158b27b2aa93345fd32563a85145c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm3017199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3017199%26sid%3Dliteratum%253Aachs%26aulast%3DLavecchia%26aufirst%3DA.%26aulast%3DDi%2BGiovanni%26aufirst%3DC.%26aulast%3DCerchia%26aufirst%3DC.%26aulast%3DRusso%26aufirst%3DA.%26aulast%3DRusso%26aufirst%3DG.%26aulast%3DNovellino%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520small%2520molecule%2520inhibitor%2520targeting%2520the%2520frataxin%252Fubiquitin%2520interaction%2520via%2520structure-based%2520virtual%2520screening%2520and%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2861%26epage%3D2873%26doi%3D10.1021%2Fjm3017199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thinnes, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scozzafava, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarhonskaya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walport, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkins, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Betti reaction enables efficient synthesis of 8-hydroxyquinoline inhibitors of 2-oxoglutarate oxygenases</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">15458</span>– <span class="NLM_lpage">15461</span>, <span class="refDoi"> DOI: 10.1039/C5CC06095H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1039%2FC5CC06095H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26345662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWqu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=15458-15461&author=C.+C.+Thinnesauthor=A.+Tumberauthor=C.+Yappauthor=G.+Scozzafavaauthor=T.+Yehauthor=M.+C.+Chanauthor=T.+A.+Tranauthor=K.+Hsuauthor=H.+Tarhonskayaauthor=L.+J.+Walportauthor=S.+E.+Wilkinsauthor=E.+D.+Martinezauthor=S.+Mullerauthor=C.+W.+Pughauthor=P.+J.+Ratcliffeauthor=P.+E.+Brennanauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Betti+reaction+enables+efficient+synthesis+of+8-hydroxyquinoline+inhibitors+of+2-oxoglutarate+oxygenases&doi=10.1039%2FC5CC06095H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Betti reaction enables efficient synthesis of 8-hydroxyquinoline inhibitors of 2-oxoglutarate oxygenases</span></div><div class="casAuthors">Thinnes, C. C.; Tumber, A.; Yapp, C.; Scozzafava, G.; Yeh, T.; Chan, M. C.; Tran, T. A.; Hsu, K.; Tarhonskaya, H.; Walport, L. J.; Wilkins, S. E.; Martinez, E. D.; Muller, S.; Pugh, C. W.; Ratcliffe, P. J.; Brennan, P. E.; Kawamura, A.; Schofield, C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">84</span>),
    <span class="NLM_cas:pages">15458-15461</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">There is interest in developing potent, selective, and cell-permeable inhibitors of human ferrous iron and 2-oxoglutarate (2OG) oxygenases for use in functional and target validation studies.  The 3-component Betti reaction enables efficient one-step C-7 functionalization of modified 8-hydroxyquinolines (8HQs) to produce cell-active inhibitors of KDM4 histone demethylases and other 2OG oxygenases; the work exemplifies how a template-based metallo-enzyme inhibitor approach can be used to give biol. active compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEdns6DiYknbVg90H21EOLACvtfcHk0liCWVp_FBinvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWqu73P&md5=02d64e182c29f6270ef714389d27261d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC5CC06095H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC06095H%26sid%3Dliteratum%253Aachs%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DScozzafava%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BA.%26aulast%3DHsu%26aufirst%3DK.%26aulast%3DTarhonskaya%26aufirst%3DH.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DWilkins%26aufirst%3DS.%2BE.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPugh%26aufirst%3DC.%2BW.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DBetti%2520reaction%2520enables%2520efficient%2520synthesis%2520of%25208-hydroxyquinoline%2520inhibitors%2520of%25202-oxoglutarate%2520oxygenases%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D15458%26epage%3D15461%26doi%3D10.1039%2FC5CC06095H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olyaei, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadeghpour, M.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the synthesis and synthetic applications of Betti base (aminoalkylnaphthol) and bis-Betti base derivatives</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18467</span>– <span class="NLM_lpage">18497</span>, <span class="refDoi"> DOI: 10.1039/C9RA02813G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1039%2FC9RA02813G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGitbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=18467-18497&author=A.+Olyaeiauthor=M.+Sadeghpour&title=Recent+advances+in+the+synthesis+and+synthetic+applications+of+Betti+base+%28aminoalkylnaphthol%29+and+bis-Betti+base+derivatives&doi=10.1039%2FC9RA02813G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the synthesis and synthetic applications of Betti base (aminoalkylnaphthol) and bis-Betti base derivatives</span></div><div class="casAuthors">Olyaei, Abolfazl; Sadeghpour, Mahdieh</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">18467-18497</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The multicomponent reaction between 2-naphthol, arylaldehydes and ammonia yields aminobenzylnaphthols in a process known as the Betti reaction, which was first uncovered at the beginning of the 20th century.  Various methods have been reported for the synthesis of aminobenzylnaphthol (Betti base) and bis-Betti base derivs. using various types of naphthols, arom. amines, heteroarom. amines, and aliph. and cyclic amines instead of ammonia or diamines and aliph. and arom. aldehydes or dialdehyde compds. under various conditions in recent years.  The Betti reaction produces racemic and non-racemic aminobenzylnaphthol ligands.  It is also clear that the most important area of application of the non-racemic aminonaphthols prepd. in this manner is their use in asym. synthesis, either as chiral ligands or as chiral auxiliaries.  The functional groups in these Mannich products offer many ring closure possibilities.  Some of these products or the starting bifunctional compds. possess biol. activity.  Herein, we present a selection of the relevant studies on this topic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv4slvgnhlYLVg90H21EOLACvtfcHk0liCWVp_FBinvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGitbjE&md5=91c60f2b0271ca60bef664b6c456caf0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC9RA02813G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA02813G%26sid%3Dliteratum%253Aachs%26aulast%3DOlyaei%26aufirst%3DA.%26aulast%3DSadeghpour%26aufirst%3DM.%26atitle%3DRecent%2520advances%2520in%2520the%2520synthesis%2520and%2520synthetic%2520applications%2520of%2520Betti%2520base%2520%2528aminoalkylnaphthol%2529%2520and%2520bis-Betti%2520base%2520derivatives%26jtitle%3DRSC%2520Adv.%26date%3D2019%26volume%3D9%26spage%3D18467%26epage%3D18497%26doi%3D10.1039%2FC9RA02813G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y. G.</span></span> <span> </span><span class="NLM_article-title">C-H oxidation/Michael addition/cyclization cascade for enantioselective synthesis of functionalized 2-amino-4H-chromenes</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">6134</span>– <span class="NLM_lpage">6137</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b03148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b03148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2qsrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=6134-6137&author=B.+Wuauthor=X.+Gaoauthor=Z.+Yanauthor=M.+W.+Chenauthor=Y.+G.+Zhou&title=C-H+oxidation%2FMichael+addition%2Fcyclization+cascade+for+enantioselective+synthesis+of+functionalized+2-amino-4H-chromenes&doi=10.1021%2Facs.orglett.5b03148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">C-H Oxidation/Michael Addition/Cyclization Cascade for Enantioselective Synthesis of Functionalized 2-Amino-4H-chromenes</span></div><div class="casAuthors">Wu, Bo; Gao, Xiang; Yan, Zhong; Chen, Mu-Wang; Zhou, Yong-Gui</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6134-6137</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A streamlined method for the enantioselective synthesis of 2-amino-4H-chromenes from readily available 2-alkyl-substituted phenols and active methylene compds. bearing a cyano group with up to 97% ee is presented.  This reaction is a cascade procedure including manganese dioxide mediated C-H oxidn. for the generation of o-quinone methides and bifunctional squaramide-catalyzed Michael addn./cyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9lNopOZNPq7Vg90H21EOLACvtfcHk0liCWVp_FBinvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2qsrfE&md5=440afb3b780de4e03f7ab484d519805b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b03148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b03148%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DM.%2BW.%26aulast%3DZhou%26aufirst%3DY.%2BG.%26atitle%3DC-H%2520oxidation%252FMichael%2520addition%252Fcyclization%2520cascade%2520for%2520enantioselective%2520synthesis%2520of%2520functionalized%25202-amino-4H-chromenes%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D6134%26epage%3D6137%26doi%3D10.1021%2Facs.orglett.5b03148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casnati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Predieri, G.</span></span> <span> </span><span class="NLM_article-title">Ortho-coordinated acylation of phenol systems</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">4371</span>– <span class="NLM_lpage">4377</span>, <span class="refDoi"> DOI: 10.1021/jo00301a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00301a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADyaK3cXksVSjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1990&pages=4371-4377&author=G.+Sartoriauthor=G.+Casnatiauthor=F.+Bigiauthor=G.+Predieri&title=Ortho-coordinated+acylation+of+phenol+systems&doi=10.1021%2Fjo00301a031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Ortho-coordinated acylation of phenol systems</span></div><div class="casAuthors">Sartori, Giovanni; Casnati, Giuseppe; Bigi, Franca; Predieri, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4371-7</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The ortho-coordinated acylation of phenol salts reported in recent synthetic applications has been extensively investigated.  The data obtained support the hypothesis that formation of an organized complex between the phenol salts and the acylating agents may strongly influence the reaction pathway, depending on the nature of the specific cation, phenol, and acyl chloride involved in the process.  The structural factors which control the formation of the reacting complex have been extensively investigated; the results obtained allow us to discuss the possibilities and limitations of this methodol. in selective o-acylphenol synthesis.  The present methodol. is the procedure of choice for ortho-functionalization of phenols with electrophilic reagents such as phosgene, oxalyl chlorides, polyunsatd. acid chlorides, phthalic dichlorides, and, in general, acid chlorides α-functionalized with an electron-withdrawing group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTXi-I_dtRHbVg90H21EOLACvtfcHk0lgtU9c1SrfgAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXksVSjur0%253D&md5=e956bf5a8f779047daa70a3aac9c7187</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo00301a031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00301a031%26sid%3Dliteratum%253Aachs%26aulast%3DSartori%26aufirst%3DG.%26aulast%3DCasnati%26aufirst%3DG.%26aulast%3DBigi%26aufirst%3DF.%26aulast%3DPredieri%26aufirst%3DG.%26atitle%3DOrtho-coordinated%2520acylation%2520of%2520phenol%2520systems%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1990%26volume%3D55%26spage%3D4371%26epage%3D4377%26doi%3D10.1021%2Fjo00301a031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shergalis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Inhibition of protein disulfide isomerase in glioblastoma causes marked downregulation of DNA repair and DNA damage response genes</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2282</span>– <span class="NLM_lpage">2298</span>, <span class="refDoi"> DOI: 10.7150/thno.30621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.7150%2Fthno.30621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=31149044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFKqtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=2282-2298&author=S.+Xuauthor=Y.+Liuauthor=K.+Yangauthor=H.+Wangauthor=A.+Shergalisauthor=A.+Kyaniauthor=A.+Bankheadauthor=S.+Tamuraauthor=S.+Yangauthor=X.+Wangauthor=C.+C.+Wangauthor=A.+Rehemtullaauthor=M.+Ljungmanauthor=N.+Neamati&title=Inhibition+of+protein+disulfide+isomerase+in+glioblastoma+causes+marked+downregulation+of+DNA+repair+and+DNA+damage+response+genes&doi=10.7150%2Fthno.30621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of protein disulfide isomerase in glioblastoma causes marked downregulation of DNA repair and DNA damage response genes</span></div><div class="casAuthors">Xu, Shili; Liu, Yajing; Yang, Kai; Wang, Hanxiao; Shergalis, Andrea; Kyani, Anahita; Bankhead, Armand., III; Tamura, Shuzo; Yang, Suhui; Wang, Xi; Wang, Chih-Chen; Rehemtulla, Alnawaz; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2282-2298</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Aberrant overexpression of endoplasmic reticulum (ER)-resident oxidoreductase protein disulfide isomerase (PDI) plays an important role in cancer progression.  In this study, we demonstrate that PDI promotes glioblastoma (GBM) cell growth and describe a class of allosteric PDI inhibitors that are selective for PDI over other PDI family members.  Methods: We performed a phenotypic screening triage campaign of over 20,000 diverse compds. to identify PDI inhibitors cytotoxic to cancer cells.  From this screen, BAP2 emerged as a lead compd., and we assessed BAP2-PDI interactions with gel filtration, thiol-competition assays, and site-directed mutagenesis studies.  To assess selectivity, we compared BAP2 activity across several PDI family members in the PDI reductase assay.  Finally, we performed in vivo studies with a mouse xenograft model of GBM combining BAP2 and the std. of care (temozolomide and radiation), and identified affected gene pathways with nascent RNA sequencing (Bru-seq).  Results:BAP2 and related analogs are novel PDI inhibitors that selectively inhibit PDIA1 and PDIp.  Though BAP2 contains a weak Michael acceptor, interaction with PDI relies on Histidine 256 in the b' domain of PDI, suggesting allosteric binding.  Furthermore, both in vitro and in vivo, BAP2 reduces cell and tumor growth.  BAP2 alters the transcription of genes involved in the unfolded protein response, ER stress, apoptosis and DNA repair response.  Conclusion: These results indicate that BAP2 has anti-tumor activity and the suppressive effect on DNA repair gene expression warrants combination with DNA damaging agents to treat GBM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvxA-gjkJrT7Vg90H21EOLACvtfcHk0lgtU9c1SrfgAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFKqtLfL&md5=084ba1e1a0f4294573c01c9c4f194cc1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.7150%2Fthno.30621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.30621%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DShergalis%26aufirst%3DA.%26aulast%3DKyani%26aufirst%3DA.%26aulast%3DBankhead%26aufirst%3DA.%26aulast%3DTamura%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DRehemtulla%26aufirst%3DA.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DInhibition%2520of%2520protein%2520disulfide%2520isomerase%2520in%2520glioblastoma%2520causes%2520marked%2520downregulation%2520of%2520DNA%2520repair%2520and%2520DNA%2520damage%2520response%2520genes%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26spage%3D2282%26epage%3D2298%26doi%3D10.7150%2Fthno.30621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madonna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beclin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laras, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcowycz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamoral-Theys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barthelemy-Requin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenglet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cresteil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, J. L.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships and mechanism of action of antitumor bis 8-hydroxyquinoline substituted benzylamines</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">638</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2009.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.ejmech.2009.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19931949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2010&pages=623-638&author=S.+Madonnaauthor=C.+Beclinauthor=Y.+Larasauthor=V.+Moretauthor=A.+Marcowyczauthor=D.+Lamoral-Theysauthor=J.+Duboisauthor=M.+Barthelemy-Requinauthor=G.+Lengletauthor=S.+Depauwauthor=T.+Cresteilauthor=G.+Aubertauthor=V.+Monnierauthor=R.+Kissauthor=M.+H.+David-Cordonnierauthor=J.+L.+Kraus&title=Structure-activity+relationships+and+mechanism+of+action+of+antitumor+bis+8-hydroxyquinoline+substituted+benzylamines&doi=10.1016%2Fj.ejmech.2009.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships and mechanism of action of antitumor bis 8-hydroxyquinoline substituted benzylamines</span></div><div class="casAuthors">Madonna, Sebastien; Beclin, Christophe; Laras, Younes; Moret, Vincent; Marcowycz, Aline; Lamoral-Theys, Delphine; Dubois, Jacques; Barthelemy-Requin, Magali; Lenglet, Gaelle; Depauw, Sabine; Cresteil, Thierry; Aubert, Genevieve; Monnier, Valerie; Kiss, Robert; David-Cordonnier, Marie-Helene; Kraus, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">623-638</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of twenty six 8-hydroxyquinoline-substituted amines, e.g. I (R = Ph, 4-F3COC6H4, 3-IC6H4, 2-thienyl, 2-pyridylmethyl, etc.), structurally related to compds. I (R = 4-MeC6H4, 4-F3CC6H4), were synthesized to evaluate the effects of structural changes on antitumor activity and understand their mechanism of action.  The studies were performed on a wide variety of cancer cell lines within glioma and carcinoma models.  The results obtained from chem. models and biol. techniques such as microarrays suggest the following hypothesis that a quinone methide intermediate which does not react with DNA but which gives covalent protein thiol adducts.  Micro-array anal. showed that the drugs induce the expression of a variety of stress related genes responsible for the cytotoxic and cytostatic effects in carcinoma and glioblastoma cells resp.  The described analogs could represent new promising anti-cancer candidates with specific action mechanisms, targeting accessible thiols from specific proteins and inducing potent anti-cancer effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS_V5g0nY2A7Vg90H21EOLACvtfcHk0lh5kJ6tvcLsDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1SgsQ%253D%253D&md5=ce938a7a00b47560c985eb7f03bbc955</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2009.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2009.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DMadonna%26aufirst%3DS.%26aulast%3DBeclin%26aufirst%3DC.%26aulast%3DLaras%26aufirst%3DY.%26aulast%3DMoret%26aufirst%3DV.%26aulast%3DMarcowycz%26aufirst%3DA.%26aulast%3DLamoral-Theys%26aufirst%3DD.%26aulast%3DDubois%26aufirst%3DJ.%26aulast%3DBarthelemy-Requin%26aufirst%3DM.%26aulast%3DLenglet%26aufirst%3DG.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DCresteil%26aufirst%3DT.%26aulast%3DAubert%26aufirst%3DG.%26aulast%3DMonnier%26aufirst%3DV.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.%2BH.%26aulast%3DKraus%26aufirst%3DJ.%2BL.%26atitle%3DStructure-activity%2520relationships%2520and%2520mechanism%2520of%2520action%2520of%2520antitumor%2520bis%25208-hydroxyquinoline%2520substituted%2520benzylamines%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2010%26volume%3D45%26spage%3D623%26epage%3D638%26doi%3D10.1016%2Fj.ejmech.2009.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekendam, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendapudi, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nag, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldenzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogg, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaumenhaft, R.</span></span> <span> </span><span class="NLM_article-title">A substrate-driven allosteric switch that enhances PDI catalytic activity</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">12579</span> <span class="refDoi"> DOI: 10.1038/ncomms12579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1038%2Fncomms12579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=27573496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=R.+H.+Bekendamauthor=P.+K.+Bendapudiauthor=L.+Linauthor=P.+P.+Nagauthor=J.+Puauthor=D.+R.+Kennedyauthor=A.+Feldenzerauthor=J.+Chiuauthor=K.+M.+Cookauthor=B.+Furieauthor=M.+Huangauthor=P.+J.+Hoggauthor=R.+Flaumenhaft&title=A+substrate-driven+allosteric+switch+that+enhances+PDI+catalytic+activity&doi=10.1038%2Fncomms12579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A substrate-driven allosteric switch that enhances PDI catalytic activity</span></div><div class="casAuthors">Bekendam, Roelof H.; Bendapudi, Pavan K.; Lin, Lin; Nag, Partha P.; Pu, Jun; Kennedy, Daniel R.; Feldenzer, Alexandra; Chiu, Joyce; Cook, Kristina M.; Furie, Bruce; Huang, Mingdong; Hogg, Philip J.; Flaumenhaft, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12579</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI) is an oxidoreductase essential for folding proteins in the endoplasmic reticulum.  The domain structure of PDI is a-b-b'-x-a', wherein the thioredoxin-like a and a' domains mediate disulfide bond shuffling and b and b' domains are substrate binding.  The b' and a' domains are connected via the x-linker, a 19-amino-acid flexible peptide.  Here we identify a class of compds., termed bepristats, that target the substrate-binding pocket of b'.  Bepristats reversibly block substrate binding and inhibit platelet aggregation and thrombus formation in vivo.  Ligation of the substrate-binding pocket by bepristats paradoxically enhances catalytic activity of a and a' by displacing the x-linker, which acts as an allosteric switch to augment reductase activity in the catalytic domains.  This substrate-driven allosteric switch is also activated by peptides and proteins and is present in other thiol isomerases.  Our results demonstrate a mechanism whereby binding of a substrate to thiol isomerases enhances catalytic activity of remote domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkUYTT5B9w97Vg90H21EOLACvtfcHk0lh5kJ6tvcLsDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbjL&md5=a138a269279f7bf35c2adc80332facad</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncomms12579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12579%26sid%3Dliteratum%253Aachs%26aulast%3DBekendam%26aufirst%3DR.%2BH.%26aulast%3DBendapudi%26aufirst%3DP.%2BK.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DNag%26aufirst%3DP.%2BP.%26aulast%3DPu%26aufirst%3DJ.%26aulast%3DKennedy%26aufirst%3DD.%2BR.%26aulast%3DFeldenzer%26aufirst%3DA.%26aulast%3DChiu%26aufirst%3DJ.%26aulast%3DCook%26aufirst%3DK.%2BM.%26aulast%3DFurie%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DHogg%26aufirst%3DP.%2BJ.%26aulast%3DFlaumenhaft%26aufirst%3DR.%26atitle%3DA%2520substrate-driven%2520allosteric%2520switch%2520that%2520enhances%2520PDI%2520catalytic%2520activity%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms12579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Go, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. P.</span></span> <span> </span><span class="NLM_article-title">Redox compartmentalization in eukaryotic cells</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1780</i></span>,  <span class="NLM_fpage">1273</span>– <span class="NLM_lpage">1290</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2008.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bbagen.2008.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=18267127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVarsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1780&publication_year=2008&pages=1273-1290&author=Y.+M.+Goauthor=D.+P.+Jones&title=Redox+compartmentalization+in+eukaryotic+cells&doi=10.1016%2Fj.bbagen.2008.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Redox compartmentalization in eukaryotic cells</span></div><div class="casAuthors">Go, Young-Mi; Jones, Dean P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1780</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1273-1290</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Diverse functions of eukaryotic cells are optimized by organization of compatible chemistries into distinct compartments defined by the structures of lipid-contg. membranes, multiprotein complexes and oligomeric structures of saccharides and nucleic acids.  This structural and chem. organization is coordinated, in part, through cysteine residues of proteins which undergo reversible oxidn.-redn. and serve as chem./structural transducing elements.  The central thiol/disulfide redox couples, thioredoxin-1, thioredoxin-2, GSH/GSSG and cysteine/cystine (Cys/CySS), are not in equil. with each other and are maintained at distinct, non-equil. potentials in mitochondria, nuclei, the secretory pathway and the extracellular space.  Mitochondria contain the most reducing compartment, have the highest rates of electron transfer and are highly sensitive to oxidn.  Nuclei also have more reduced redox potentials but are relatively resistant to oxidn.  The secretory pathway contains oxidative systems which introduce disulfides into proteins for export.  The cytoplasm contains few metabolic oxidases and this maintains an environment for redox signaling dependent upon NADPH oxidases and NO synthases.  Extracellular compartments are maintained at stable oxidizing potentials.  Controlled changes in cytoplasmic GSH/GSSG redox potential are assocd. with functional state, varying with proliferation, differentiation and apoptosis.  Variation in extracellular Cys/CySS redox potential is also assocd. with proliferation, cell adhesion and apoptosis.  Thus, cellular redox biol. is inseparable from redox compartmentalization.  Further elucidation of the redox control networks within compartments will improve the mechanistic understanding of cell functions and their disruption in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7q9nSxTPBAbVg90H21EOLACvtfcHk0lh5kJ6tvcLsDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVarsbfJ&md5=5a772255f00733cd606e3d3c47d57728</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2008.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2008.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DGo%26aufirst%3DY.%2BM.%26aulast%3DJones%26aufirst%3DD.%2BP.%26atitle%3DRedox%2520compartmentalization%2520in%2520eukaryotic%2520cells%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2008%26volume%3D1780%26spage%3D1273%26epage%3D1290%26doi%3D10.1016%2Fj.bbagen.2008.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walker, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, H. F.</span></span> <span> </span><span class="NLM_article-title">Oxidation of kinetically trapped thiols by protein disulfide isomerase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">13642</span>– <span class="NLM_lpage">13650</span>, <span class="refDoi"> DOI: 10.1021/bi00041a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00041a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADyaK2MXot1ylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=13642-13650&author=K.+W.+Walkerauthor=H.+F.+Gilbert&title=Oxidation+of+kinetically+trapped+thiols+by+protein+disulfide+isomerase&doi=10.1021%2Fbi00041a045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidation of Kinetically Trapped Thiols by Protein Disulfide Isomerase</span></div><div class="casAuthors">Walker, Kenneth W.; Gilbert, Hiram F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13642-50</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of a stabilized structure during oxidative protein folding can severely retard disulfide formation if the structure must be disrupted to gain access to buried cysteines.  These kinetic traps can slow protein folding and disulfide bond formation to the extent that unassisted folding is too slow to be kinetically competent in the cell.  Protein disulfide isomerase (PDI) facilitates the oxidn. of a kinetically trapped state of RTEM-1 β-lactamase in which two cysteines that form the single disulfide bond in the native protein are buried and approx. 500-fold less reactive than exposed cysteines.  Under second-order conditions, PDI-dependent oxidn. of reduced, folded β-lactamase is 500-fold faster than GSSG-dependent oxidn.  The rate difference obsd. between PDI and GSSG can be accounted for by the 520-fold higher kinetic reactivity of PDI as an oxidant.  Noncovalent interactions between PDI (35 μM) and β-lactamase increase the reactivity or unfolding of β-lactamase in the steady-state by less than 3-fold.  At high concns. of PDI or alkylating agents, the reaction of β-lactamase cysteines approaches a const. rate, limited by the spontaneous unfolding of the protein (kunfold = 0.024 min-1).  PDI does not substantially increase the rate of β-lactamase unfolding; however, once β-lactamase spontaneously unfolds, PDI at concns. greater than 44 μM, oxidizes the unfolded substrate before it can refold (kfold = 1.5 min-1).  PDI also facilitates the glutathione disulfide-dependent oxidn. of β-lactamase by approx. 1.7-3-fold, even at satg. GSSG concns.; however, oxidn. rates never exceed the rate of spontaneous β-lactamase unfolding.  The high kinetic reactivity of PDI as an oxidant and the almost millimolar concns. of PDI found in the endoplasmic reticulum provide one mechanism to avoid kinetic traps by making oxidn. kinetically competitive with the formation of kinetic traps.  In addn., PDI's high reactivity and its noncovalent interactions with unfolded proteins may minimize the significance of even relatively stable kinetic traps, as long as unfolding of the kinetic trap does not become rate-limiting.  Chaperones, including the hsp70 chaperone of the endoplasmic reticulum, BiP, and the Escherichia coli chaperonin, GroEL, do not appear to increase the rate of β-lactamase unfolding.  This suggests that the unfolding rate may help define those structures that the endoplasmic reticulum quality control system recognizes as native.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTv3EXMh_GpLVg90H21EOLACvtfcHk0ljlnrtWZTQiMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXot1ylu7o%253D&md5=859635a24742cb442ab46dfd7827bc27</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fbi00041a045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00041a045%26sid%3Dliteratum%253Aachs%26aulast%3DWalker%26aufirst%3DK.%2BW.%26aulast%3DGilbert%26aufirst%3DH.%2BF.%26atitle%3DOxidation%2520of%2520kinetically%2520trapped%2520thiols%2520by%2520protein%2520disulfide%2520isomerase%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D13642%26epage%3D13650%26doi%3D10.1021%2Fbi00041a045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veloso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span> <span> </span><span class="NLM_article-title">Use of Bru-Seq and BruChase-Seq for genome-wide assessment of the synthesis and stability of RNA</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2013.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.ymeth.2013.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=23973811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFeksbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2014&pages=45-54&author=M.+T.+Paulsenauthor=A.+Velosoauthor=J.+Prasadauthor=K.+Bediauthor=E.+A.+Ljungmanauthor=B.+Magnusonauthor=T.+E.+Wilsonauthor=M.+Ljungman&title=Use+of+Bru-Seq+and+BruChase-Seq+for+genome-wide+assessment+of+the+synthesis+and+stability+of+RNA&doi=10.1016%2Fj.ymeth.2013.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Bru-Seq and BruChase-Seq for genome-wide assessment of the synthesis and stability of RNA</span></div><div class="casAuthors">Paulsen, Michelle T.; Veloso, Artur; Prasad, Jayendra; Bedi, Karan; Ljungman, Emily A.; Magnuson, Brian; Wilson, Thomas E.; Ljungman, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-54</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Gene expression studies commonly examine total cellular RNA, which only provides information about its steady-state pool of RNA.  It remains unclear whether differences in the steady-state reflects variable rates of transcription or RNA degrdn.  To specifically monitor RNA synthesis and degrdn. genome-wide, we developed Bru-Seq and BruChase-Seq.  These assays are based on metabolic pulse-chase labeling of RNA using bromouridine (Bru).  In Bru-Seq, recently labeled RNAs are sequenced to reveal spans of nascent transcription in the genome.  In BruChase-Seq, cells are chased in uridine for different periods of time following Bru-labeling, allowing for the isolation of RNA populations of specific ages.  Here we describe these methodologies in detail and highlight their usefulness in assessing RNA synthesis and stability as well as splicing kinetics with examples of specific genes from different human cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkhkyBLXrRWrVg90H21EOLACvtfcHk0ljlnrtWZTQiMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFeksbvN&md5=a18ef08d32830fe03bf5024aa7246fb9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2013.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2013.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DM.%2BT.%26aulast%3DVeloso%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DJ.%26aulast%3DBedi%26aufirst%3DK.%26aulast%3DLjungman%26aufirst%3DE.%2BA.%26aulast%3DMagnuson%26aufirst%3DB.%26aulast%3DWilson%26aufirst%3DT.%2BE.%26aulast%3DLjungman%26aufirst%3DM.%26atitle%3DUse%2520of%2520Bru-Seq%2520and%2520BruChase-Seq%2520for%2520genome-wide%2520assessment%2520of%2520the%2520synthesis%2520and%2520stability%2520of%2520RNA%26jtitle%3DMethods%26date%3D2014%26volume%3D67%26spage%3D45%26epage%3D54%26doi%3D10.1016%2Fj.ymeth.2013.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenendyk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, M.</span></span> <span> </span><span class="NLM_article-title">Calreticulin signaling in health and disease</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">842</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2012.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.biocel.2012.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22373697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Glsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=842-846&author=W.+A.+Wangauthor=J.+Groenendykauthor=M.+Michalak&title=Calreticulin+signaling+in+health+and+disease&doi=10.1016%2Fj.biocel.2012.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Calreticulin signaling in health and disease</span></div><div class="casAuthors">Wang, Wen-An; Groenendyk, Jody; Michalak, Marek</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">842-846</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Calreticulin is an endoplasmic reticulum Ca2+ binding chaperone that has multiple functions inside and outside of the endoplasmic reticulum.  It is involved in the quality control of newly synthesized proteins and glycoproteins, interacting with various other endoplasmic reticulum chaperones, specifically calnexin and ER protein of 57-kDa in the calreticulin/calnexin cycle.  Calreticulin also plays a crucial role in regulating intracellular Ca2+ homeostasis, assocg. calreticulin with a wide variety of signaling processes, such as cardiogenesis, adipocyte differentiation and cellular stress responses.  The role of calreticulin outside of the endoplasmic reticulum is also extensive, including functions in wound healing and immunity.  Therefore, calreticulin has important implications in health and disease.Signaling factsCalreticulin controls calcineurin activity and NF-AT- and MEF2C-dependent expression of cardiac genes through InsP3R mediated ER Ca2+ release.Calreticulin deficiency up-regulates the CAMKII-dependent signaling pathway.Calreticulin over-expression decreases adipogenic factors (lipoprotein lipase, αP2, PPAR and C/EBPα).Cell surface calreticulin signals pre-apoptotic cellular clearance by binding to LDL-receptor-related protein (LRP) on phagocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvqHSSZettOrVg90H21EOLACvtfcHk0ljy3BTMXOUnbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Glsb0%253D&md5=3cd5893f9e77b4d9d408b0198c1d34f0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2012.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2012.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%2BA.%26aulast%3DGroenendyk%26aufirst%3DJ.%26aulast%3DMichalak%26aufirst%3DM.%26atitle%3DCalreticulin%2520signaling%2520in%2520health%2520and%2520disease%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2012%26volume%3D44%26spage%3D842%26epage%3D846%26doi%3D10.1016%2Fj.biocel.2012.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jessop, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavender, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulleid, N. J.</span></span> <span> </span><span class="NLM_article-title">Substrate specificity of the oxidoreductase ERp57 is determined primarily by its interaction with calnexin and calreticulin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">2194</span>– <span class="NLM_lpage">2202</span>, <span class="refDoi"> DOI: 10.1074/jbc.M808054200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M808054200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19054761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFaqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=2194-2202&author=C.+E.+Jessopauthor=T.+J.+Tavenderauthor=R.+H.+Watkinsauthor=J.+E.+Chambersauthor=N.+J.+Bulleid&title=Substrate+specificity+of+the+oxidoreductase+ERp57+is+determined+primarily+by+its+interaction+with+calnexin+and+calreticulin&doi=10.1074%2Fjbc.M808054200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Specificity of the Oxidoreductase ERp57 Is Determined Primarily by Its Interaction with Calnexin and Calreticulin</span></div><div class="casAuthors">Jessop, Catherine E.; Tavender, Timothy J.; Watkins, Rachel H.; Chambers, Joseph E.; Bulleid, Neil J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2194-2202</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The formation of disulfides within proteins entering the secretory pathway is catalyzed by the protein disulfide isomerase family of endoplasmic reticulum localized oxidoreductases.  One such enzyme, ERp57, is thought to catalyze the isomerization of non-native disulfide bonds formed in glycoproteins with unstructured disulfide-rich domains.  Here we investigated the mechanism underlying ERp57 specificity toward glycoprotein substrates and the interdependence of ERp57 and the calnexin cycle for their correct folding.  Our results clearly show that ERp57 must be phys. assocd. with the calnexin cycle to catalyze isomerization reactions with most of its substrates.  In addn., some glycoproteins only require ERp57 for correct disulfide formation if they enter the calnexin cycle.  Hence, the specificity of ER oxidoreductases is not only detd. by the phys. assocn. of enzyme and substrate but also by accessory factors, such as calnexin and calreticulin in the case of ERp57.  These conclusions suggest that the calnexin cycle has evolved with a specialized oxidoreductase to facilitate native disulfide formation in complex glycoproteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiwiNUERR-Z7Vg90H21EOLACvtfcHk0ljy3BTMXOUnbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFaqtA%253D%253D&md5=781f153089970a97fa54bf101c381e13</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M808054200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M808054200%26sid%3Dliteratum%253Aachs%26aulast%3DJessop%26aufirst%3DC.%2BE.%26aulast%3DTavender%26aufirst%3DT.%2BJ.%26aulast%3DWatkins%26aufirst%3DR.%2BH.%26aulast%3DChambers%26aufirst%3DJ.%2BE.%26aulast%3DBulleid%26aufirst%3DN.%2BJ.%26atitle%3DSubstrate%2520specificity%2520of%2520the%2520oxidoreductase%2520ERp57%2520is%2520determined%2520primarily%2520by%2520its%2520interaction%2520with%2520calnexin%2520and%2520calreticulin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D2194%26epage%3D2202%26doi%3D10.1074%2Fjbc.M808054200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. S.</span></span> <span> </span><span class="NLM_article-title">Role of the unfolded protein response regulator GRP78/BiP in development, cancer, and neurological disorders</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2307</span>– <span class="NLM_lpage">2316</span>, <span class="refDoi"> DOI: 10.1089/ars.2009.2485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1089%2Fars.2009.2485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19309259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFaru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=2307-2316&author=M.+Wangauthor=S.+Weyauthor=Y.+Zhangauthor=R.+Yeauthor=A.+S.+Lee&title=Role+of+the+unfolded+protein+response+regulator+GRP78%2FBiP+in+development%2C+cancer%2C+and+neurological+disorders&doi=10.1089%2Fars.2009.2485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Unfolded Protein Response Regulator GRP78/BiP in Development, Cancer, and Neurological Disorders</span></div><div class="casAuthors">Wang, Miao; Wey, Shiuan; Zhang, Yi; Ye, Risheng; Lee, Amy S.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2307-2316</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  GRP78/BiP is a major endoplasmic reticulum (ER) chaperone protein crit. for protein quality control of the ER, as well as controlling the activation of the ER-transmembrane signaling mols.  Through creation of mouse models targeting the Grp78 allele, the function of GRP78 in development and disease has been investigated.  These led to the discovery that GRP78 function is obligatory for early embryonic development.  However, in adult animals, GRP78 is preferably required for cancer cell survival under pathol. conditions such as tumor progression and drug resistance.  The discovery of surface localization of GRP78 in cancer cells reveals potential novel function, interaction with cell-surface receptors, and possible therapeutic implications.  Mouse models also reveal that GRP78 controls maturation and secretion of neuronal factors for proper neural migration and offers neuroprotection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozuyAwoRxPp7Vg90H21EOLACvtfcHk0ljy3BTMXOUnbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFaru70%253D&md5=6b06910b584fd9c164a1c415aec04ac5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1089%2Fars.2009.2485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2009.2485%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWey%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DA.%2BS.%26atitle%3DRole%2520of%2520the%2520unfolded%2520protein%2520response%2520regulator%2520GRP78%252FBiP%2520in%2520development%252C%2520cancer%252C%2520and%2520neurological%2520disorders%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2009%26volume%3D11%26spage%3D2307%26epage%3D2316%26doi%3D10.1089%2Fars.2009.2485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. J.</span></span> <span> </span><span class="NLM_article-title">Implications of NQO1 in cancer therapy</span>. <i>BMB Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.5483/BMBRep.2015.48.11.190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.5483%2FBMBRep.2015.48.11.190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26424559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvF2gsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=609-617&author=E.+T.+Ohauthor=H.+J.+Park&title=Implications+of+NQO1+in+cancer+therapy&doi=10.5483%2FBMBRep.2015.48.11.190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of NQO1 in cancer therapy</span></div><div class="casAuthors">Oh, Eun-Taex; Park, Heon Joo</div><div class="citationInfo"><span class="NLM_cas:title">BMB Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">609-617</span>CODEN:
                <span class="NLM_cas:coden">BRMEC2</span>;
        ISSN:<span class="NLM_cas:issn">1976-670X</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">NAD(P)H:quinone oxidoreductase (NQO1), an obligatory two-electron reductase, is a ubiquitous cytosolic enzyme that catalyzes the redn. of quinone substrates.  The NQO1- mediated two-electron redn. of quinones can be either chemoprotection/detoxification or a chemotherapeutic response, depending on the target quinones.  When toxic quinones are reduced by NQO1, they are conjugated with glutathione or glucuronic acid and excreted from the cells.  Based on this protective effect of NQO1, the use of dietary compds. to induce the expression of NQO1 has emerged as a promising strategy for cancer prevention.  On the other hand, NQO1-mediated two-electron redn. converts certain quinone compds. (such as mitomycin C, E09, RH1 and .ovrddot.ic-lapachone) to cytotoxic agents, leading to cell death.  It has been known that NQO1 is expressed at high levels in numerous human cancers, including breast, colon, cervix, lung, and pancreas, as compared with normal tissues.  This implies that tumors can be preferentially damaged relative to normal tissue by cytotoxic quinone drugs.  Importantly, NQO1 has been shown to stabilize many proteins, including p53 and p33ING1b, by inhibiting their proteasomal degrdn.  This review will summarize the biol. roles of NQO1 in cancer, with emphasis on recent findings and the potential of NQO1 as a therapeutic target for the cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnUeEnF9n8Y7Vg90H21EOLACvtfcHk0li_XL9f1xEWdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvF2gsbo%253D&md5=0291a80078ec971717eef27d267feb49</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.5483%2FBMBRep.2015.48.11.190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5483%252FBMBRep.2015.48.11.190%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DE.%2BT.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DImplications%2520of%2520NQO1%2520in%2520cancer%2520therapy%26jtitle%3DBMB%2520Rep.%26date%3D2015%26volume%3D48%26spage%3D609%26epage%3D617%26doi%3D10.5483%2FBMBRep.2015.48.11.190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koppula, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, B.</span></span> <span> </span><span class="NLM_article-title">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span>. <i>Cancer Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">12</span>, <span class="refDoi"> DOI: 10.1186/s40880-018-0288-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1186%2Fs40880-018-0288-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=12&author=P.+Koppulaauthor=Y.+Zhangauthor=L.+Zhuangauthor=B.+Gan&title=Amino+acid+transporter+SLC7A11%2FxCT+at+the+crossroads+of+regulating+redox+homeostasis+and+nutrient+dependency+of+cancer&doi=10.1186%2Fs40880-018-0288-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span></div><div class="casAuthors">Koppula Pranavi; Zhang Yilei; Zhuang Li; Gan Boyi; Koppula Pranavi; Gan Boyi; Gan Boyi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer communications (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells often upregulate nutrient transporters to fulfill their increased biosynthetic and bioenergetic needs, and to maintain redox homeostasis.  One nutrient transporter frequently overexpressed in human cancers is the cystine/glutamate antiporter solute carrier family 7 member 11 (SLC7A11; also known as xCT).  SLC7A11 promotes cystine uptake and glutathione biosynthesis, resulting in protection from oxidative stress and ferroptotic cell death.  Recent studies have unexpectedly revealed that SLC7A11 also plays critical roles in glutamine metabolism and regulates the glucose and glutamine dependency of cancer cells.  This review discusses the roles of SLC7A11 in regulating the antioxidant response and nutrient dependency of cancer cells, explores our current understanding of SLC7A11 regulation in cancer metabolism, and highlights key open questions for future studies in this emerging research area.  A deeper understanding of SLC7A11 in cancer metabolism may identify new therapeutic opportunities to target this important amino acid transporter for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSc0x1bKxSR5v69nzq6467DfW6udTcc2eaqAg8GajgYk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D&md5=22acaa68350c28038eb70f6974151c9f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs40880-018-0288-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40880-018-0288-x%26sid%3Dliteratum%253Aachs%26aulast%3DKoppula%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhuang%26aufirst%3DL.%26aulast%3DGan%26aufirst%3DB.%26atitle%3DAmino%2520acid%2520transporter%2520SLC7A11%252FxCT%2520at%2520the%2520crossroads%2520of%2520regulating%2520redox%2520homeostasis%2520and%2520nutrient%2520dependency%2520of%2520cancer%26jtitle%3DCancer%2520Commun.%26date%3D2018%26volume%3D38%26spage%3D12%26doi%3D10.1186%2Fs40880-018-0288-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kyani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shergalis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Discovery and mechanistic elucidation of a class of protein disulfide isomerase inhibitors for the treatment of glioblastoma</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1002%2Fcmdc.201700629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29235250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1ehuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=164-177&author=A.+Kyaniauthor=S.+Tamuraauthor=S.+Yangauthor=A.+Shergalisauthor=S.+Samantaauthor=Y.+Kuangauthor=M.+Ljungmanauthor=N.+Neamati&title=Discovery+and+mechanistic+elucidation+of+a+class+of+protein+disulfide+isomerase+inhibitors+for+the+treatment+of+glioblastoma&doi=10.1002%2Fcmdc.201700629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Mechanistic Elucidation of a Class of Protein Disulfide Isomerase Inhibitors for the Treatment of Glioblastoma</span></div><div class="casAuthors">Kyani, Anahita; Tamura, Shuzo; Yang, Suhui; Shergalis, Andrea; Samanta, Soma; Kuang, Yuting; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">164-177</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI) is overexpressed in glioblastoma, the most aggressive form of brain cancer, and folds nascent proteins responsible for the progression and spread of the disease.  Herein we describe a novel nanomolar PDI inhibitor, pyrimidotriazinedione 35G8, that is toxic in a panel of human glioblastoma cell lines.  We performed a medium-throughput 20 000-compd. screen of a diverse subset of 1 000 000 compds. to identify cytotoxic small mols.  Cytotoxic compds. were screened for PDI inhibition, and, from the screen, 35G8 emerged as the most cytotoxic inhibitor of PDI.  Bromouridine labeling and sequencing (Bru-seq) of nascent RNA revealed that 35G8 induces nuclear factor-like 2 (Nrf2) antioxidant response, endoplasmic reticulum (ER) stress response, and autophagy.  Specifically, 35G8 upregulated heme oxygenase 1 and solute carrier family 7 member 11 (SLC7A11) transcription and protein expression and repressed PDI target genes such as thioredoxin-interacting protein 1 (TXNIP) and early growth response 1 (EGR1).  Interestingly, 35G8-induced cell death did not proceed via apoptosis or necrosis, but by a mixt. of autophagy and ferroptosis.  Cumulatively, our data demonstrate a mechanism for a novel PDI inhibitor as a chem. probe to validate PDI as a target for brain cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKe8TT4OBZ6LVg90H21EOLACvtfcHk0li_XL9f1xEWdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1ehuw%253D%253D&md5=0f28e215d9b02792f2438f2dbc8b05e9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700629%26sid%3Dliteratum%253Aachs%26aulast%3DKyani%26aufirst%3DA.%26aulast%3DTamura%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DShergalis%26aufirst%3DA.%26aulast%3DSamanta%26aufirst%3DS.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DDiscovery%2520and%2520mechanistic%2520elucidation%2520of%2520a%2520class%2520of%2520protein%2520disulfide%2520isomerase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520glioblastoma%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D164%26epage%3D177%26doi%3D10.1002%2Fcmdc.201700629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hieronymus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: Using gene-expression signatures to connect small molecules, genes, and disease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1935</span>, <span class="refDoi"> DOI: 10.1126/science.1132939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1126%2Fscience.1132939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=17008526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&author=J.+Lambauthor=E.+D.+Crawfordauthor=D.+Peckauthor=J.+W.+Modellauthor=I.+C.+Blatauthor=M.+J.+Wrobelauthor=J.+Lernerauthor=J.+P.+Brunetauthor=A.+Subramanianauthor=K.+N.+Rossauthor=M.+Reichauthor=H.+Hieronymusauthor=G.+Weiauthor=S.+A.+Armstrongauthor=S.+J.+Haggartyauthor=P.+A.+Clemonsauthor=R.+Weiauthor=S.+A.+Carrauthor=E.+S.+Landerauthor=T.+R.+Golub&title=The+Connectivity+Map%3A+Using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease&doi=10.1126%2Fscience.1132939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The connectivity map: Using gene-expression signatures to connect small molecules, genes, and disease</span></div><div class="casAuthors">Lamb, Justin; Crawford, Emily D.; Peck, David; Modell, Joshua W.; Blat, Irene C.; Wrobel, Matthew J.; Lerner, Jim; Brunet, Jean-Philippe; Subramanian, Aravind; Ross, Kenneth N.; Reich, Michael; Hieronymus, Haley; Wei, Guo; Armstrong, Scott A.; Haggarty, Stephen J.; Clemons, Paul A.; Wei, Ru; Carr, Steven A.; Lander, Eric S.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5795</span>),
    <span class="NLM_cas:pages">1929-1935</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To pursue a systematic approach to the discovery of functional connections among diseases, genetic perturbation, and drug action, we have created the first installment of a ref. collection of gene-expression profiles from cultured human cells treated with bioactive small mols., together with pattern-matching software to mine these data.  We demonstrate that this "Connectivity Map" resource can be used to find connections among small mols. sharing a mechanism of action, chems. and physiol. processes, and diseases and drugs.  These results indicate the feasibility of the approach and suggest the value of a large-scale community Connectivity Map project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtu0M2BUO2w7Vg90H21EOLACvtfcHk0liPqoDMlh2fZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN&md5=1fad887ad515ffdd8064e526ff25a4db</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1132939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1132939%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DModell%26aufirst%3DJ.%2BW.%26aulast%3DBlat%26aufirst%3DI.%2BC.%26aulast%3DWrobel%26aufirst%3DM.%2BJ.%26aulast%3DLerner%26aufirst%3DJ.%26aulast%3DBrunet%26aufirst%3DJ.%2BP.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DHieronymus%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520Using%2520gene-expression%2520signatures%2520to%2520connect%2520small%2520molecules%252C%2520genes%252C%2520and%2520disease%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D1929%26epage%3D1935%26doi%3D10.1126%2Fscience.1132939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torikai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohigashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, K.</span></span> <span> </span><span class="NLM_article-title">Ebselen, a glutathione peroxidase mimetic seleno-organic compound, as a multifunctional antioxidant. Implication for inflammation-associated carcinogenesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">2687</span>– <span class="NLM_lpage">2694</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109641200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M109641200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=11714717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVSgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=2687-2694&author=Y.+Nakamuraauthor=Q.+Fengauthor=T.+Kumagaiauthor=K.+Torikaiauthor=H.+Ohigashiauthor=T.+Osawaauthor=N.+Noguchiauthor=E.+Nikiauthor=K.+Uchida&title=Ebselen%2C+a+glutathione+peroxidase+mimetic+seleno-organic+compound%2C+as+a+multifunctional+antioxidant.+Implication+for+inflammation-associated+carcinogenesis&doi=10.1074%2Fjbc.M109641200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ebselen, a glutathione peroxidase mimetic seleno-organic compound, as a multifunctional antioxidant: implication for inflammation-associated carcinogenesis</span></div><div class="casAuthors">Nakamura, Yoshimasa; Feng, Qing; Kumagai, Takeshi; Torikai, Koji; Ohigashi, Hajime; Osawa, Toshihiko; Noguchi, Noriko; Niki, Etsuo; Uchida, Koji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2687-2694</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Ebselen, a seleno-org. compd. showing glutathione peroxidase-like activity, is one of the promising synthetic antioxidants.  In the present study, we investigated the antioxidant activities of ebselen using a 12-O-tetradecanoylphorbol-13-acetate (TPA)-treated mouse skin model.  Double pretreatments of mouse skin with ebselen significantly inhibited TPA-induced formation of thiobarbituric acid-reacting substance, known as an overall oxidative damage biomarker, in mouse epidermis, suggesting that ebselen indeed acts as an antioxidant in mouse skin.  The antioxidative effect of ebselen is attributed to its selective blockade of leukocyte infiltration and activation leading to attenuation of the H2O2 level.  In in vitro studies, ebselen inhibited TPA-induced superoxide generation in differentiated HL-60 cells and lipopolysaccharide-induced cyclooxygenase-2 protein expression in RAW 264.7 cells.  In addn., we demonstrated for the first time that ebselen potentiated phase II enzyme activities, including NAD(P)H:(quinone-acceptor) oxidoreductase 1 and glutathione S-transferase in cultured hepatocytes and in mouse skin.  These results strongly suggest that ebselen, a multifunctional antioxidant, is a potential chemopreventive agent in inflammation-assocd. carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGeGWFoXCUYLVg90H21EOLACvtfcHk0lg18b7NhtGg9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVSgsb8%253D&md5=11107ceb27808fdfda33e7aa4c31c446</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109641200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109641200%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DKumagai%26aufirst%3DT.%26aulast%3DTorikai%26aufirst%3DK.%26aulast%3DOhigashi%26aufirst%3DH.%26aulast%3DOsawa%26aufirst%3DT.%26aulast%3DNoguchi%26aufirst%3DN.%26aulast%3DNiki%26aufirst%3DE.%26aulast%3DUchida%26aufirst%3DK.%26atitle%3DEbselen%252C%2520a%2520glutathione%2520peroxidase%2520mimetic%2520seleno-organic%2520compound%252C%2520as%2520a%2520multifunctional%2520antioxidant.%2520Implication%2520for%2520inflammation-associated%2520carcinogenesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D2687%26epage%3D2694%26doi%3D10.1074%2Fjbc.M109641200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreyra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hryckowian, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Child, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higginbottom, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banaei, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnenburg, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">A small-molecule antivirulence agent for treating Clostridium difficile infection</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">306ra148</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aac9103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1126%2Fscitranslmed.aac9103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26400909" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&author=K.+O.+Benderauthor=M.+Garlandauthor=J.+A.+Ferreyraauthor=A.+J.+Hryckowianauthor=M.+A.+Childauthor=A.+W.+Puriauthor=D.+E.+Solow-Corderoauthor=S.+K.+Higginbottomauthor=E.+Segalauthor=N.+Banaeiauthor=A.+Shenauthor=J.+L.+Sonnenburgauthor=M.+Bogyo&title=A+small-molecule+antivirulence+agent+for+treating+Clostridium+difficile+infection&doi=10.1126%2Fscitranslmed.aac9103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aac9103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aac9103%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DK.%2BO.%26aulast%3DGarland%26aufirst%3DM.%26aulast%3DFerreyra%26aufirst%3DJ.%2BA.%26aulast%3DHryckowian%26aufirst%3DA.%2BJ.%26aulast%3DChild%26aufirst%3DM.%2BA.%26aulast%3DPuri%26aufirst%3DA.%2BW.%26aulast%3DSolow-Cordero%26aufirst%3DD.%2BE.%26aulast%3DHigginbottom%26aufirst%3DS.%2BK.%26aulast%3DSegal%26aufirst%3DE.%26aulast%3DBanaei%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DSonnenburg%26aufirst%3DJ.%2BL.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DA%2520small-molecule%2520antivirulence%2520agent%2520for%2520treating%2520Clostridium%2520difficile%2520infection%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26doi%3D10.1126%2Fscitranslmed.aac9103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beilhartz, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, R. A.</span></span> <span> </span><span class="NLM_article-title">Comment on “A small-molecule antivirulence agent for treating Clostridium difficile infection”</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">370tc2</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aad8926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1126%2Fscitranslmed.aad8926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28003550" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&author=G.+L.+Beilhartzauthor=J.+Tamauthor=Z.+Zhangauthor=R.+A.+Melnyk&title=Comment+on+%E2%80%9CA+small-molecule+antivirulence+agent+for+treating+Clostridium+difficile+infection%E2%80%9D&doi=10.1126%2Fscitranslmed.aad8926"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aad8926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aad8926%26sid%3Dliteratum%253Aachs%26aulast%3DBeilhartz%26aufirst%3DG.%2BL.%26aulast%3DTam%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMelnyk%26aufirst%3DR.%2BA.%26atitle%3DComment%2520on%2520%25E2%2580%259CA%2520small-molecule%2520antivirulence%2520agent%2520for%2520treating%2520Clostridium%2520difficile%2520infection%25E2%2580%259D%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2016%26volume%3D8%26doi%3D10.1126%2Fscitranslmed.aad8926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trychta, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Back, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, B. K.</span></span> <span> </span><span class="NLM_article-title">KDEL receptors are differentially regulated to maintain the ER proteome under calcium deficiency</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1829</span>– <span class="NLM_lpage">1840.e6</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2018.10.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.celrep.2018.10.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=30428351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1829-1840.e6&author=K.+A.+Trychtaauthor=S.+Backauthor=M.+J.+Hendersonauthor=B.+K.+Harvey&title=KDEL+receptors+are+differentially+regulated+to+maintain+the+ER+proteome+under+calcium+deficiency&doi=10.1016%2Fj.celrep.2018.10.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">KDEL Receptors Are Differentially Regulated to Maintain the ER Proteome under Calcium Deficiency</span></div><div class="casAuthors">Trychta, Kathleen A.; Back, Susanne; Henderson, Mark J.; Harvey, Brandon K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1829-1840.e6</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Retention of crit. endoplasmic reticulum (ER) luminal proteins needed to carry out diverse functions (e.g., protein synthesis and folding, lipid metab.) is mediated through a carboxy-terminal ER retention sequence (ERS) and its interaction with KDEL receptors.  Here, we demonstrate that depleting ER calcium causes mass departure of ERS-contg. proteins from cells by overwhelming KDEL receptors.  In addn., we provide evidence that KDELR2 and KDELR3, but not KDELR1, are unfolded protein response (UPR) genes upregulated as an adaptive response to counteract the loss of ERS-contg. proteins, suggesting previously unknown isoform-specific functions of the KDEL receptors.  Overall, our findings establish that decreases in ER calcium change the compn. of the ER luminal proteome and secretome, which can impact cellular functions and cell viability.  The redistribution of the ER proteome from inside the cell to the outside has implications for dissecting the complex relationship of ER homeostasis with diverse disease pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzB6TodoZ_oLVg90H21EOLACvtfcHk0lg18b7NhtGg9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtbvP&md5=fdec1c46459cde29eed85c6421a21b1a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.10.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.10.055%26sid%3Dliteratum%253Aachs%26aulast%3DTrychta%26aufirst%3DK.%2BA.%26aulast%3DBack%26aufirst%3DS.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DHarvey%26aufirst%3DB.%2BK.%26atitle%3DKDEL%2520receptors%2520are%2520differentially%2520regulated%2520to%2520maintain%2520the%2520ER%2520proteome%2520under%2520calcium%2520deficiency%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D25%26spage%3D1829%26epage%3D1840.e6%26doi%3D10.1016%2Fj.celrep.2018.10.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schurmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule target engagement in cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.chembiol.2016.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=27049669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC28fntVKqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=435-441&author=M.+Schurmannauthor=P.+Janningauthor=S.+Zieglerauthor=H.+Waldmann&title=Small-molecule+target+engagement+in+cells&doi=10.1016%2Fj.chembiol.2016.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Target Engagement in Cells</span></div><div class="casAuthors">Schurmann Marc; Janning Petra; Ziegler Slava; Waldmann Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Cell chemical biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Monitoring how, when, and where small molecules engage their targets inside living cells is a critical step in chemical biology and pharmacological research, because it enables compound efficacy and confirmation of mode of action to be assessed.  In this mini-review we summarize the currently available methodologies to detect and prove direct target engagement in cells and offer a critical view of their key advantages and disadvantages.  As the interest of the field shifts toward discovery and validation of high-quality agents, we expect that efforts to develop and refine these types of methodologies will also intensify in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRSou8rC8Pn5252AMpI4GK2fW6udTcc2eZTPtZzDB3GHrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fntVKqsw%253D%253D&md5=49bd3dc012e84676f2939f189896c254</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DSchurmann%26aufirst%3DM.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall-molecule%2520target%2520engagement%2520in%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D435%26epage%3D441%26doi%3D10.1016%2Fj.chembiol.2016.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richard, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lena, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierceall, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minond, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardone, M. H.</span></span> <span> </span><span class="NLM_article-title">Hydroxyquinoline-derived compounds and analoguing of selective Mcl-1 inhibitors using a functional biomarker</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6642</span>– <span class="NLM_lpage">6649</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bmc.2013.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=23993674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGrurbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6642-6649&author=D.+J.+Richardauthor=R.+Lenaauthor=T.+Bannisterauthor=N.+Blakeauthor=W.+E.+Pierceallauthor=N.+E.+Carlsonauthor=C.+E.+Kellerauthor=M.+Koenigauthor=Y.+Heauthor=D.+Minondauthor=J.+Mishraauthor=M.+Cameronauthor=T.+Spicerauthor=P.+Hodderauthor=M.+H.+Cardone&title=Hydroxyquinoline-derived+compounds+and+analoguing+of+selective+Mcl-1+inhibitors+using+a+functional+biomarker&doi=10.1016%2Fj.bmc.2013.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxyquinoline-derived compounds and analoguing of selective Mcl-1 inhibitors using a functional biomarker</span></div><div class="casAuthors">Richard, David J.; Lena, Ryan; Bannister, Thomas; Blake, Noel; Pierceall, William E.; Carlson, Nicole E.; Keller, Christina Eberhart; Koenig, Marcel; He, Yuanjun; Minond, Dmitriy; Mishra, Jitendra; Cameron, Michael; Spicer, Timothy; Hodder, Peter; Cardone, Michael H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6642-6649</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Anti-apoptotic Bcl-2 family proteins are important oncol. therapeutic targets.  To date, BH3 mimetics that abrogate anti-apoptotic activity have largely been directed at Bcl-2 and/or Bcl-xL.  One obsd. mechanism of resistance to these inhibitors is increased Mcl-1 levels in cells exposed to such therapeutics.  For this reason, and because Mcl-1 is important in the onset of lymphoid, myeloid, and other cancers, it has become a target of great interest.  However, small mol. inhibitors displaying potency and selectivity for Mcl-1 are lacking.  Identifying such compds. has been challenging due to difficulties in translating the target selectivity obsd. at the biochem. level to the cellular level.  Herein we report the results of an HTS strategy coupled with directed hit optimization.  Compds. identified have selective Mcl-1 inhibitory activity with greater than 100-fold reduced affinity for Bcl-xL.  The selectivity of these compds. at the cellular level was validated using BH3 profiling, a novel personalized diagnostic approach.  This assay provides an important functional biomarker that allows for the characterization of cells based upon their dependencies on various anti-apoptotic Bcl-2 proteins.  We demonstrate that cells dependent on Mcl-1 or Bcl-2/Bcl-xL for survival are commensurately responsive to compds. that genuinely target those proteins.  The identification of compd. 9 with uniquely validated and selective Mcl-1 inhibitory activity provides a valuable tool to those studying the intrinsic apoptosis pathway and highlights an important approach in the development of a first-in-class cancer therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwGeLQPRnawrVg90H21EOLACvtfcHk0lgy7UkWU9lenA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGrurbJ&md5=a2875b8bc8fce034d17def9920dd31a4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DRichard%26aufirst%3DD.%2BJ.%26aulast%3DLena%26aufirst%3DR.%26aulast%3DBannister%26aufirst%3DT.%26aulast%3DBlake%26aufirst%3DN.%26aulast%3DPierceall%26aufirst%3DW.%2BE.%26aulast%3DCarlson%26aufirst%3DN.%2BE.%26aulast%3DKeller%26aufirst%3DC.%2BE.%26aulast%3DKoenig%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DMinond%26aufirst%3DD.%26aulast%3DMishra%26aufirst%3DJ.%26aulast%3DCameron%26aufirst%3DM.%26aulast%3DSpicer%26aufirst%3DT.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DCardone%26aufirst%3DM.%2BH.%26atitle%3DHydroxyquinoline-derived%2520compounds%2520and%2520analoguing%2520of%2520selective%2520Mcl-1%2520inhibitors%2520using%2520a%2520functional%2520biomarker%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6642%26epage%3D6649%26doi%3D10.1016%2Fj.bmc.2013.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S. W.</span></span> <span> </span><span class="NLM_article-title">TET1 exerts its tumour suppressor function by regulating autophagy in glioma cells</span>. <i>Biosci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>, <span class="NLM_elocation-id">BSR20160523</span> <span class="refDoi"> DOI: 10.1042/BSR20160523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1042%2FBSR20160523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28341638" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&author=R.+Fuauthor=Y.+Dingauthor=J.+Luoauthor=L.+Yuauthor=C.+L.+Liauthor=D.+S.+Liauthor=S.+W.+Guo&title=TET1+exerts+its+tumour+suppressor+function+by+regulating+autophagy+in+glioma+cells&doi=10.1042%2FBSR20160523"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1042%2FBSR20160523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20160523%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DR.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DLi%26aufirst%3DD.%2BS.%26aulast%3DGuo%26aufirst%3DS.%2BW.%26atitle%3DTET1%2520exerts%2520its%2520tumour%2520suppressor%2520function%2520by%2520regulating%2520autophagy%2520in%2520glioma%2520cells%26jtitle%3DBiosci.%2520Rep.%26date%3D2017%26volume%3D37%26doi%3D10.1042%2FBSR20160523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span> <span> </span><span class="NLM_article-title">Glutathione metabolism in cancer progression and treatment resistance</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>217</i></span>,  <span class="NLM_fpage">2291</span>– <span class="NLM_lpage">2298</span>, <span class="refDoi"> DOI: 10.1083/jcb.201804161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1083%2Fjcb.201804161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29915025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGntb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2018&pages=2291-2298&author=A.+Bansalauthor=M.+C.+Simon&title=Glutathione+metabolism+in+cancer+progression+and+treatment+resistance&doi=10.1083%2Fjcb.201804161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione metabolism in cancer progression and treatment resistance</span></div><div class="casAuthors">Bansal, Ankita; Simon, M. Celeste</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">217</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2291-2298</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">1540-8140</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Glutathione (GSH) is the most abundant antioxidant found in living organisms and has multiple functions, most of which maintain cellular redox homeostasis.  GSH preserves sufficient levels of cysteine and detoxifies xenobiotics while also conferring therapeutic resistance to cancer cells.  However, GSH metab. plays both beneficial and pathogenic roles in a variety of malignancies.  It is crucial to the removal and detoxification of carcinogens, and alterations in this pathway can have a profound effect on cell survival.  Excess GSH promotes tumor progression, where elevated levels correlate with increased metastasis.  In this review, we discuss recent studies that focus on deciphering the role of GSH in tumor initiation and progression as well as mechanisms underlying how GSH imparts treatment resistance to growing cancers.  Targeting GSH synthesis/utilization therefore represents a potential means of rendering tumor cells more susceptible to different treatment options such as chemotherapy and radiotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeFe4G21ytbVg90H21EOLACvtfcHk0lhdvquIZKEOqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGntb3N&md5=2d071bd5b51fa8628dd56a2ee683931d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201804161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201804161%26sid%3Dliteratum%253Aachs%26aulast%3DBansal%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26atitle%3DGlutathione%2520metabolism%2520in%2520cancer%2520progression%2520and%2520treatment%2520resistance%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2018%26volume%3D217%26spage%3D2291%26epage%3D2298%26doi%3D10.1083%2Fjcb.201804161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span> <span> </span><span class="NLM_article-title">Glutathione reductase mediates drug resistance in glioblastoma cells by regulating redox homeostasis</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1111/jnc.14250</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1111%2Fjnc.14250" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29105080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2018&pages=93-104&author=Z.+Zhuauthor=S.+Duauthor=Y.+Duauthor=J.+Renauthor=G.+Yingauthor=Z.+Yan&title=Glutathione+reductase+mediates+drug+resistance+in+glioblastoma+cells+by+regulating+redox+homeostasis&doi=10.1111%2Fjnc.14250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione reductase mediates drug resistance in glioblastoma cells by regulating redox homeostasis</span></div><div class="casAuthors">Zhu, Zhongling; Du, Shuangshuang; Du, Yibo; Ren, Jing; Ying, Guoguang; Yan, Zhao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-104</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Glutathione (GSH) and GSH-related enzymes constitute the most important defense system that protects cells from free radical, radiotherapy, and chemotherapy attacks.  In this study, we aim to explore the potential role and regulatory mechanism of the GSH redox cycle in drug resistance in glioblastoma multiforme (GBM) cells.  We found that temozolomide (TMZ)-resistant glioma cells displayed lower levels of endogenous reactive oxygen species and higher levels of total antioxidant capacity and GSH than sensitive cells.  Moreover, the expression of glutathione reductase (GSR), the key enzyme of the GSH redox cycle, was higher in TMZ-resistant cells than in sensitive cells.  Furthermore, silencing GSR in drug-resistant cells improved the sensitivity of cells to TMZ or cisplatin.  Conversely, the over-expression of GSR in sensitive cells resulted in resistance to chemotherapy.  In addn., the GSR enzyme partially prevented the oxidative stress caused by pro-oxidant L-buthionine -sulfoximine.  The modulation of redox state by GSH or L-buthionine -sulfoximine regulated GSR-mediated drug resistance, suggesting that the action of GSR in drug resistance is assocd. with the modulation of redox homeostasis.  Intriguingly, a trend toward shorter progress-free survival was obsd. among GBM patients with high GSR expression.  These results indicated that GSR is involved in mediating drug resistance and is a potential target for improving GBM treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwxSPa6vBqGLVg90H21EOLACvtfcHk0liVgm_fSgFbiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsLvI&md5=968215abecae08b2419389b9220b6741</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fjnc.14250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjnc.14250%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DYing%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DZ.%26atitle%3DGlutathione%2520reductase%2520mediates%2520drug%2520resistance%2520in%2520glioblastoma%2520cells%2520by%2520regulating%2520redox%2520homeostasis%26jtitle%3DJ.%2520Neurochem.%26date%3D2018%26volume%3D144%26spage%3D93%26epage%3D104%26doi%3D10.1111%2Fjnc.14250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span> <span> </span><span class="NLM_article-title">Mechanism and significance of increased glutathione level in human hepatocellular carcinoma and liver regeneration</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1096/fj.00-0445fje</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1096%2Ffj.00-0445fje" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=11099488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1Sns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=19-21&author=Z.+Z.+Huangauthor=C.+Chenauthor=Z.+Zengauthor=H.+Yangauthor=J.+Ohauthor=L.+Chenauthor=S.+C.+Lu&title=Mechanism+and+significance+of+increased+glutathione+level+in+human+hepatocellular+carcinoma+and+liver+regeneration&doi=10.1096%2Ffj.00-0445fje"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and significance of increased glutathione level in human hepatocellular carcinoma and liver regeneration</span></div><div class="casAuthors">Huang, Zong-Zhi; Chen, Changjin; Zeng, Zhaohui; Yang, Heping; Oh, Jenny; Chen, Lixin; Lu, Shelly C.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-21</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Compared to normal human liver, glutathione (GSH) levels were increased in human hepatocellular carcinomas; GSH synthetase (GS) and γ-glutamylcysteine synthetase (GCS) heavy subunit (GCS-HS) transcripts were also increased.  In the hepatocellular carcinomas, there was an increased nuclear binding activity to antioxidant response element 4 (ARE-4), nuclear factor kappa B (NF-κB), and activator protein 1 (AP-1) elements of the GCS heavy subunit (GCS-HS) promoter.  In vitro expts. with HepG2 cells indicated that increased GSH facilitates the growth of HepG2 cells.  Both a normal baseline hepatic GSH level and a subsequent increase in GSH were required for normal liver regeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK0hhu4kMw8LVg90H21EOLACvtfcHk0liVgm_fSgFbiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1Sns7w%253D&md5=91db39bacdd7319bbd94305715226ba1</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1096%2Ffj.00-0445fje&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.00-0445fje%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%2BZ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DOh%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DS.%2BC.%26atitle%3DMechanism%2520and%2520significance%2520of%2520increased%2520glutathione%2520level%2520in%2520human%2520hepatocellular%2520carcinoma%2520and%2520liver%2520regeneration%26jtitle%3DFASEB%2520J.%26date%3D2001%26volume%3D15%26spage%3D19%26epage%3D21%26doi%3D10.1096%2Ffj.00-0445fje" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span> <span> </span><span class="NLM_article-title">Regulation of glutathione synthesis</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.mam.2008.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=18601945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFelu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=42-59&author=S.+C.+Lu&title=Regulation+of+glutathione+synthesis&doi=10.1016%2Fj.mam.2008.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of glutathione synthesis</span></div><div class="casAuthors">Lu, Shelly C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">42-59</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Glutathione (GSH) is a ubiquitous intracellular peptide with diverse functions that include detoxification, antioxidant defense, maintenance of thiol status, and modulation of cell proliferation.  GSH is synthesized in the cytosol of all mammalian cells in a tightly regulated manner.  The major determinants of GSH synthesis are the availability of cysteine, the sulfur amino acid precursor, and the activity of the rate-limiting enzyme, glutamate cysteine ligase (GCL).  GCL is composed for a catalytic (GCLC) and modifier (GCLM) subunit and they are regulated at multiple levels and at times differentially.  The second enzyme of GSH synthesis, GSH synthase (GS) is also regulated in a coordinated manner as GCL subunits and its up-regulation can further enhance the capacity of the cell to synthesize GSH.  Oxidative stress is well known to induce the expression of GSH synthetic enzymes.  Key transcription factors identified thus far include Nrf2/Nrf1 via the antioxidant response element (ARE), activator protein-1 (AP-1) and nuclear factor κ B (NFκB).  Dysregulation of GSH synthesis is increasingly being recognized as contributing to the pathogenesis of many pathol. conditions.  These include diabetes mellitus, pulmonary fibrosis, cholestatic liver injury, endotoxemia and drug-resistant tumor cells.  Manipulation of the GSH synthetic capacity is an important target in the treatment of many of these disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXr29hUN4_i7Vg90H21EOLACvtfcHk0liVgm_fSgFbiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFelu78%253D&md5=691c8d8898facb97f10dd52eac5072b8</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%2BC.%26atitle%3DRegulation%2520of%2520glutathione%2520synthesis%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2009%26volume%3D30%26spage%3D42%26epage%3D59%26doi%3D10.1016%2Fj.mam.2008.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veloso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washburn, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar-Sinha, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span> <span> </span><span class="NLM_article-title">Coordinated regulation of synthesis and stability of RNA during the acute TNF-induced proinflammatory response</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">2240</span>– <span class="NLM_lpage">2245</span>, <span class="refDoi"> DOI: 10.1073/pnas.1219192110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.1219192110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=23345452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVersLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=2240-2245&author=M.+T.+Paulsenauthor=A.+Velosoauthor=J.+Prasadauthor=K.+Bediauthor=E.+A.+Ljungmanauthor=Y.+C.+Tsanauthor=C.+W.+Changauthor=B.+Tarrierauthor=J.+G.+Washburnauthor=R.+Lyonsauthor=D.+R.+Robinsonauthor=C.+Kumar-Sinhaauthor=T.+E.+Wilsonauthor=M.+Ljungman&title=Coordinated+regulation+of+synthesis+and+stability+of+RNA+during+the+acute+TNF-induced+proinflammatory+response&doi=10.1073%2Fpnas.1219192110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Coordinated regulation of synthesis and stability of RNA during the acute TNF-induced proinflammatory response</span></div><div class="casAuthors">Paulsen, Michelle T.; Veloso, Artur; Prasad, Jayendra; Bedi, Karan; Ljungman, Emily A.; Tsan, Ya-Chun; Chang, Ching-Wei; Tarrier, Brendan; Washburn, Joseph G.; Lyons, Robert; Robinson, Daniel R.; Kumar-Sinha, Chandan; Wilson, Thomas E.; Ljungman, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2240-2245, S2240/1-S2240/347</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Steady-state gene expression is a coordination of synthesis and decay of RNA through epigenetic regulation, transcription factors, micro RNAs (miRNAs), and RNA-binding proteins.  Here, we present bromouride labeling and sequencing (Bru-Seq) and bromouridine pulse-chase and sequencing (BruChase-Seq) to assess genome-wide changes to RNA synthesis and stability in human fibroblasts at homeostasis and after exposure to the proinflammatory tumor necrosis factor (TNF).  The inflammatory response in human cells involves rapid and dramatic changes in gene expression, and the Bru-Seq and BruChase-Seq techniques revealed a coordinated and complex regulation of gene expression both at the transcriptional and posttranscriptional levels.  The combinatory anal. of both RNA synthesis and stability using Bru-Seq and BruChase-Seq allows for a much deeper understanding of mechanisms of gene regulation than afforded by the anal. of steady-state total RNA and should be useful in many biol. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfuRwriMUd77Vg90H21EOLACvtfcHk0ljyQL8BZRC59A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVersLY%253D&md5=74e41864b9379692b3265cb7a806e7ca</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1219192110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1219192110%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DM.%2BT.%26aulast%3DVeloso%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DJ.%26aulast%3DBedi%26aufirst%3DK.%26aulast%3DLjungman%26aufirst%3DE.%2BA.%26aulast%3DTsan%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BW.%26aulast%3DTarrier%26aufirst%3DB.%26aulast%3DWashburn%26aufirst%3DJ.%2BG.%26aulast%3DLyons%26aufirst%3DR.%26aulast%3DRobinson%26aufirst%3DD.%2BR.%26aulast%3DKumar-Sinha%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DT.%2BE.%26aulast%3DLjungman%26aufirst%3DM.%26atitle%3DCoordinated%2520regulation%2520of%2520synthesis%2520and%2520stability%2520of%2520RNA%2520during%2520the%2520acute%2520TNF-induced%2520proinflammatory%2520response%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D2240%26epage%3D2245%26doi%3D10.1073%2Fpnas.1219192110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mootha, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomeroy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span> <span> </span><span class="NLM_article-title">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">15545</span>– <span class="NLM_lpage">15550</span>, <span class="refDoi"> DOI: 10.1073/pnas.0506580102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.0506580102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=16199517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&author=A.+Subramanianauthor=P.+Tamayoauthor=V.+K.+Moothaauthor=S.+Mukherjeeauthor=B.+L.+Ebertauthor=M.+A.+Gilletteauthor=A.+Paulovichauthor=S.+L.+Pomeroyauthor=T.+R.+Golubauthor=E.+S.+Landerauthor=J.+P.+Mesirov&title=Gene+set+enrichment+analysis%3A+A+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles&doi=10.1073%2Fpnas.0506580102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span></div><div class="casAuthors">Subramanian, Aravind; Tamayo, Pablo; Mootha, Vamsi K.; Mukherjee, Sayan; Ebert, Benjamin L.; Gillette, Michael A.; Paulovich, Amanda; Pomeroy, Scott L.; Golub, Todd R.; Lander, Eric S.; Mesirov, Jill P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15545-15550</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Although genomewide RNA expression anal. has become a routine tool in biomedical research, extg. biol. insight from such information remains a major challenge.  Here, we describe a powerful anal. method called Gene Set Enrichment Anal. (GSEA) for interpreting gene expression data.  The method derives its power by focusing on gene sets, i.e., groups of genes that share common biol. function, chromosomal location, or regulation.  We demonstrate how GSEA yields insights into several cancer-related data sets, including leukemia and lung cancer.  Notably, where single-gene anal. finds little similarity between two independent studies of patients survival in lung cancer, GSEA reveals many biol. pathways in common.  The GSEA method is embodied in a freely available software package, together with an initial database of 1,325 biol. defined gene sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH2_E6gWd0yrVg90H21EOLACvtfcHk0ljyQL8BZRC59A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO&md5=ca2eb221010f20379199e6442c65fc2e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506580102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506580102%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DMootha%26aufirst%3DV.%2BK.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DGillette%26aufirst%3DM.%2BA.%26aulast%3DPaulovich%26aufirst%3DA.%26aulast%3DPomeroy%26aufirst%3DS.%2BL.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26atitle%3DGene%2520set%2520enrichment%2520analysis%253A%2520A%2520knowledge-based%2520approach%2520for%2520interpreting%2520genome-wide%2520expression%2520profiles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D15545%26epage%3D15550%26doi%3D10.1073%2Fpnas.0506580102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao-Jie  Wang</span>, <span class="hlFld-ContribAuthor ">Xinxin  Guo</span>, <span class="hlFld-ContribAuthor ">Rui-Qin  Zhai</span>, <span class="hlFld-ContribAuthor ">Changning  Sun</span>, <span class="hlFld-ContribAuthor ">Guokai  Xiao</span>, <span class="hlFld-ContribAuthor ">Jin  Chen</span>, <span class="hlFld-ContribAuthor ">Mei-Yan  Wei</span>, <span class="hlFld-ContribAuthor ">Chang-Lun  Shao</span>, <span class="hlFld-ContribAuthor ">Yuchao  Gu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of penipanoid C-inspired 2-(3,4,5-trimethoxybenzoyl)quinazolin-4(3H)-one derivatives as potential anticancer agents by inhibiting cell proliferation and inducing apoptosis in hepatocellular carcinoma cells. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>224 </em>, 113671. <a href="https://doi.org/10.1016/j.ejmech.2021.113671" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113671%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bpenipanoid%252BC-inspired%252B2-%2525283%25252C4%25252C5-trimethoxybenzoyl%252529quinazolin-4%2525283H%252529-one%252Bderivatives%252Bas%252Bpotential%252Banticancer%252Bagents%252Bby%252Binhibiting%252Bcell%252Bproliferation%252Band%252Binducing%252Bapoptosis%252Bin%252Bhepatocellular%252Bcarcinoma%252Bcells%26aulast%3DWang%26aufirst%3DChao-Jie%26date%3D2021%26volume%3D224%26spage%3D113671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vishwanath  Jha</span>, <span class="hlFld-ContribAuthor ">Tripti  Kumari</span>, <span class="hlFld-ContribAuthor ">Vijayprakash  Manickam</span>, <span class="hlFld-ContribAuthor ">Zahra  Assar</span>, <span class="hlFld-ContribAuthor ">Kirk L.  Olson</span>, <span class="hlFld-ContribAuthor ">Jeong-Ki  Min</span>, <span class="hlFld-ContribAuthor ">Jaehyung  Cho</span>. </span><span class="cited-content_cbyCitation_article-title">ERO1-PDI Redox Signaling in Health and Disease. </span><span class="cited-content_cbyCitation_journal-name">Antioxidants & Redox Signaling</span><span> <strong>2021,</strong> <em>52 </em><a href="https://doi.org/10.1089/ars.2021.0018" title="DOI URL">https://doi.org/10.1089/ars.2021.0018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/ars.2021.0018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fars.2021.0018%26sid%3Dliteratum%253Aachs%26jtitle%3DAntioxidants%2520%2526%2520Redox%2520Signaling%26atitle%3DERO1-PDI%252BRedox%252BSignaling%252Bin%252BHealth%252Band%252BDisease%26aulast%3DJha%26aufirst%3DVishwanath%26date%3D2021%26date%3D2021%26volume%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renato Simões  Gaspar</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Gibbins</span>. </span><span class="cited-content_cbyCitation_article-title">Thiol Isomerases Orchestrate Thrombosis and Hemostasis. </span><span class="cited-content_cbyCitation_journal-name">Antioxidants & Redox Signaling</span><span> <strong>2021,</strong> <em>118 </em><a href="https://doi.org/10.1089/ars.2021.0086" title="DOI URL">https://doi.org/10.1089/ars.2021.0086</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/ars.2021.0086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fars.2021.0086%26sid%3Dliteratum%253Aachs%26jtitle%3DAntioxidants%2520%2526%2520Redox%2520Signaling%26atitle%3DThiol%252BIsomerases%252BOrchestrate%252BThrombosis%252Band%252BHemostasis%26aulast%3DGaspar%26aufirst%3DRenato%2BSim%25C3%25B5es%26date%3D2021%26date%3D2021%26volume%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul  Victor</span>, <span class="hlFld-ContribAuthor ">Dronamraju  Sarada</span>, <span class="hlFld-ContribAuthor ">Kunka Mohanram  Ramkumar</span>. </span><span class="cited-content_cbyCitation_article-title">Crosstalk between endoplasmic reticulum stress and oxidative stress: Focus on protein disulfide isomerase and endoplasmic reticulum oxidase 1. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2021,</strong> <em>892 </em>, 173749. <a href="https://doi.org/10.1016/j.ejphar.2020.173749" title="DOI URL">https://doi.org/10.1016/j.ejphar.2020.173749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2020.173749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2020.173749%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DCrosstalk%252Bbetween%252Bendoplasmic%252Breticulum%252Bstress%252Band%252Boxidative%252Bstress%25253A%252BFocus%252Bon%252Bprotein%252Bdisulfide%252Bisomerase%252Band%252Bendoplasmic%252Breticulum%252Boxidase%252B1%26aulast%3DVictor%26aufirst%3DPaul%26date%3D2021%26volume%3D892%26spage%3D173749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0028.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0028.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Previously reported PDI inhibitors studied in the context of ovarian cancer (<b>PACMA31</b>), Huntington’s disease (<b>LOC14</b>), brain cancer (<b>BAP2</b> and <b>35G8</b>), thrombosis (<b>Bepristat 1a</b> and <b>isoquercetin</b>), and multiple myeloma (<b>CCF642</b>). <b>KSC-34</b> is an <b>a</b>-site selective probe and estradiol is an endogenous ligand of PDIA1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0011.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Compounds with the Modified Betti Reaction<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) toluene, 100 °C and (b) EtOH, 120 °C microwave or conventional heating.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0012.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>8</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Me<sub>2</sub>SO<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, acetone.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0013.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a)(i), MeI, Mg, Et<sub>2</sub>O and (ii), benzoyl chloride; (b) cyclohexylmagnesium bromide, tetrahydrofuran (THF), rt −50 °C; (c) <i>p</i>-TsOH, toluene, rt; and (d) Pd/C, H<sub>2</sub>, EtOH, 60 °C.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Screening approach leading to the discovery of <b>AS15</b> and <b>CD343</b>. (b) Structures of <b>AS15</b> and <b>CD343</b>, and IC<sub>50</sub> values calculated in the PDI reductase assay. (c) Dose–response curves of <b>PACMA31</b>, <b>AS15</b>, and <b>CD343</b> in the PDI reductase assay. (d) Shifts in the melting temperatures of <b>PACMA31</b>, <b>AS15</b>, and <b>CD343</b> in the thermal shift assay. SAR: structure–activity relationship.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structure–activity relationship indicates that compounds are covalent PDI inhibitors. (a) Comparison of compounds with free hydroxy or methoxy on the bicyclic moiety. (b) Comparison of compounds with piperidine or cyclohexane moiety. (c) Proposed mechanism of inhibition via the retro-Michael addition reaction. (d) <b>6a</b> covalently binds to PDI. 100 μM <b>6a</b> was incubated with 10 μM PDI and 100 μM dithiothreitol (DTT) for 30 min prior to injection. (e) <b>6a</b> binds to PDI preferentially over GSTO1. 100 μM <b>6a</b> was incubated with a mixture of 10 μM PDI, 10 μM GSTO1, and 10 μM GRP78 as a nonspecific binding control for 30 min prior to injection. (Close-up of GRP78 trace in Supplementary <a href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf" class="ext-link">Figure S3</a>.) (f) Concentration- and time-dependent PDI inhibition curves for <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> determination of <b>PACMA31</b>, <b>16F16</b>, and <b>AS15</b>. Activity was measured using the PDI reductase assay. Absorbance was monitored over time at various concentrations and preincubation times with indicated compounds. (g) Gel-based competition with recombinant PDI, PDI inhibitors, or inactive analogue <b>6f</b> at 100 μM and <b>PACMA57</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Recovery of PDI activity upon treatment with <b>PACMA31</b> (a), <b>BAP2</b> (b), <b>AS15</b> (c), or <b>CD343</b> (d). For the diluted samples (purple curves), 40 μM PDI was incubated with 100 μM <b>PACMA31</b>, 50 μM <b>BAP2</b>, 50 μM <b>AS15</b>, or 50 μM <b>CD343</b> for 3 h at rt before dilution. (f) ANS spectral scan with 5 μM PDI and 100 μM <b>PACMA31</b>, <b>BAP2</b>, <b>AS15</b>, or <b>CD343</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <i>N</i>-Acetyl cysteine and glutathione inactivate <b>AS15</b> analogues. (a) <i>N</i>-Acetyl cysteine (NAC) competition in the PDI reductase assay. Compounds were incubated with NAC prior to the addition to a final concentration of 10 μM in the PDI reductase assay with PDIA1. (b) Glutathione competition in the PDI reductase assay. GSH was added to the PDI reaction buffer prior to a 10 μM compound addition. (c) Top two <b>AS15</b> analogues that are least sensitive to competition with 5 mM GSH in the PDI reductase assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Dimerized analogue is less sensitive to GSH treatment. (a) Structure of <b>5d</b> dimerization. (b) Percent inhibition of <b>5d</b> monomer and dimer in the PDI reductase assay in competition with 5 mM glutathione. (c) Protein mass spectrometry confirms that the fragment of <b>5d</b> binds to PDI. Toxicity of <b>PACMA31</b> (d), <b>AS15</b> (e), <b>CD343</b> (f), <b>5d</b> monomer (g), and <b>5d</b> dimer (h) in the colony formation assay in the absence or presence of BSO. U-118 MG cells were pretreated with BSO for 24 h prior to compound addition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0008.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. <b>AS15</b> analogues bind serum albumin. (a) Structures of BODIPY-labeled <b>AS15</b> analogues. (b) U-118 MG cells treated with 40 μM <b>15</b> overnight. (c) Cell-free DMEM treated with 20 μM <b>15</b> or <b>16</b> overnight at 37 °C. (d) <b>16</b> dose–response curve in the PDI reductase assay. (e) Cell lysates incubated with 50 μM <b>6b</b> and 5 μM <b>15</b> or <b>16</b> for 24 h at room temperature after cell lysis. (f) Cell lysates treated with increasing concentrations of parent compound <b>6b</b> before the addition of BODIPY-labeled <b>16</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0014.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthetic Route for Compounds <b>15</b> and <b>16</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) toluene, 100 °C; (b) 4-(5,5-difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl)aniline (BODIPY aniline), EDCI, HOBt, dimethylformamide (DMF), rt; (c) <i>tert</i>-butyl (2-aminoethyl)carbamate, <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (HATU), <i>N</i>,<i>N-</i>diisopropylethylamine (DIEA), DMF, rt; (d) trifluoroacetic acid (TFA), dichloromethane (DCM), rt; and (e) 4-(5,5-difluoro-1,3,7,9-tetramethyl-5<i>H</i>-4λ<sup>4</sup>,5λ<sup>4</sup>-dipyrrolo[1,2-<i>c</i>:2′,1′-<i>f</i>][1,3,2]diazaborinin-10-yl)benzoic acid, HATU, DIEA, DMF, rt.</p></p></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0009.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Confocal microscopy images at 60× magnification of A-172 cells treated with 10 μM <b>15</b> (a) or 2 μM <b>16</b> (b) for 24 h prior to fixation and staining for PDI.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/medium/jm0c00728_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0010.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. <b>5p</b> upregulates the transcription of genes involved in the unfolded protein response. (a) Structure of <b>5p</b>. (b) <b>5p</b> promoted gene set enrichment related to the unfolded protein response. NES: normalized enrichment score. FDR <i>q</i> val: false discovery rate <i>q</i> value. The criteria for GSEA were <i>p</i> < 0.05 and false discovery rate <25%. (c) STRING interactions of significant genes in <b>5p</b> Bru-seq data set. A 4 h treatment with <b>5p</b> increases the transcription of representative unfolded protein response genes including <i>CALR</i> (d), <i>HSPA5</i> (e), <i>MYZAP</i> (f), <i>NQO1</i> (g), and <i>SLC7A11</i> (h). (i) U-118 MG cells were treated with 20 μM <b>5p</b> for 12, 24, or 48 h and probed for calreticulin protein expression. Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) is used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-18/acs.jmedchem.0c00728/20200918/images/large/jm0c00728_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00728&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrom, Q. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cioffi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gittleman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waite, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruchko, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnholtz-Sloan, J. S.</span></span> <span> </span><span class="NLM_article-title">CBTRUS statistical report: Primary brain and other central nervous system tumors diagnosed in the United States in 2012-2016</span>. <i>Neuro-Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">v1</span>– <span class="NLM_lpage">v100</span>, <span class="refDoi"> DOI: 10.1093/neuonc/noz150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1093%2Fneuonc%2Fnoz150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=31675094" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=v1-v100&author=Q.+T.+Ostromauthor=G.+Cioffiauthor=H.+Gittlemanauthor=N.+Patilauthor=K.+Waiteauthor=C.+Kruchkoauthor=J.+S.+Barnholtz-Sloan&title=CBTRUS+statistical+report%3A+Primary+brain+and+other+central+nervous+system+tumors+diagnosed+in+the+United+States+in+2012-2016&doi=10.1093%2Fneuonc%2Fnoz150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fneuonc%2Fnoz150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fneuonc%252Fnoz150%26sid%3Dliteratum%253Aachs%26aulast%3DOstrom%26aufirst%3DQ.%2BT.%26aulast%3DCioffi%26aufirst%3DG.%26aulast%3DGittleman%26aufirst%3DH.%26aulast%3DPatil%26aufirst%3DN.%26aulast%3DWaite%26aufirst%3DK.%26aulast%3DKruchko%26aufirst%3DC.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%2BS.%26atitle%3DCBTRUS%2520statistical%2520report%253A%2520Primary%2520brain%2520and%2520other%2520central%2520nervous%2520system%2520tumors%2520diagnosed%2520in%2520the%2520United%2520States%2520in%25202012-2016%26jtitle%3DNeuro-Oncol.%26date%3D2019%26volume%3D21%26spage%3Dv1%26epage%3Dv100%26doi%3D10.1093%2Fneuonc%2Fnoz150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shergalis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luesakul, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muangsin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Current challenges and opportunities in treating glioblastoma</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1124/pr.117.014944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1124%2Fpr.117.014944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29669750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2018&pages=412-445&author=A.+Shergalisauthor=A.+Bankheadauthor=U.+Luesakulauthor=N.+Muangsinauthor=N.+Neamati&title=Current+challenges+and+opportunities+in+treating+glioblastoma&doi=10.1124%2Fpr.117.014944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Current challenges and opportunities in treating glioblastoma</span></div><div class="casAuthors">Shergalis, Andrea; Bankhead, Armand; Luesakul, Urarika; Muangsin, Nongnuj; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">412-445</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Glioblastoma multiforme (GBM), the most common and aggressive primary brain tumor, has a high mortality rate despite extensive efforts to develop new treatments.  GBM exhibits both intra- and intertumor heterogeneity, lending to resistance and eventual tumor recurrence.  Large-scale genomic and proteomic anal. of GBM tumors has uncovered potential drug targets.  Effective and "druggable" targets must be validated to embark on a robust medicinal chem. campaign culminating in the discovery of clin. candidates.  Here, we review recent developments in GBM drug discovery and delivery.  To identify GBM drug targets, we performed extensive bioinformatics anal. using data from The Cancer Genome Atlas project.  We discovered 20 genes, BOC, CLEC4GP1, ELOVL6, EREG, ESR2, FDCSP, FURIN, FUT8-AS1, GZMB, IRX3, LITAF, NDEL1, NKX3-1, PODNL1, PTPRN, QSOX1, SEMA4F, TH, VEGFC, and C20orf166AS1 that are overexpressed in a subpopulation of GBM patients and correlate with poor survival outcomes.  Importantly, nine of these genes exhibit higher expression in GBM vs. low-grade glioma and may be involved in disease progression.  In this review, we discuss these proteins in the context of GBM disease progression.  We also conducted computational multi-parameter optimization to assess the blood-brain barrier (BBB) permeability of small mols. in clin. trials for GBM treatment.  Drug delivery in the context of GBM is particularly challenging because the BBB hinders small mol. transport.  Therefore, we discuss novel drug delivery methods, including nanoparticles and prodrugs.  Given the aggressive nature of GBM and the complexity of targeting the central nervous system, effective treatment options are a major unmet medical need.  Identification and validation of biomarkers and drug targets assocd. with GBM disease progression present an exciting opportunity to improve treatment of this devastating disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKoQ_eLuZhn7Vg90H21EOLACvtfcHk0lgYtgcI6CIk6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKnsb8%253D&md5=80fb3912ca8df56db7b1c146ef45ffef</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1124%2Fpr.117.014944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.117.014944%26sid%3Dliteratum%253Aachs%26aulast%3DShergalis%26aufirst%3DA.%26aulast%3DBankhead%26aufirst%3DA.%26aulast%3DLuesakul%26aufirst%3DU.%26aulast%3DMuangsin%26aufirst%3DN.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DCurrent%2520challenges%2520and%2520opportunities%2520in%2520treating%2520glioblastoma%26jtitle%3DPharmacol.%2520Rev.%26date%3D2018%26volume%3D70%26spage%3D412%26epage%3D445%26doi%3D10.1124%2Fpr.117.014944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. C.</span></span> <span> </span><span class="NLM_article-title">Structural insights into the redox-regulated dynamic conformations of human protein disulfide isomerase</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1089/ars.2012.4630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1089%2Fars.2012.4630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22657537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC38not1Cnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=36-45&author=C.+Wangauthor=W.+Liauthor=J.+Renauthor=J.+Fangauthor=H.+Keauthor=W.+Gongauthor=W.+Fengauthor=C.+C.+Wang&title=Structural+insights+into+the+redox-regulated+dynamic+conformations+of+human+protein+disulfide+isomerase&doi=10.1089%2Fars.2012.4630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the redox-regulated dynamic conformations of human protein disulfide isomerase</span></div><div class="casAuthors">Wang Chao; Li Wei; Ren Jinqi; Fang Jingqi; Ke Huimin; Gong Weimin; Feng Wei; Wang Chih-Chen</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & redox signaling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-45</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AIM:  Human protein disulfide isomerase (hPDI) is a key enzyme and a redox-regulated chaperone responsible for oxidative protein folding in the endoplasmic reticulum.  This work aims to reveal the molecular mechanism underlying the redox-regulated functions of hPDI by determining the crystal structures of hPDI in different redox states.  RESULTS:  The structures of hPDI (abb'xa') in both the reduced and oxidized states showed that the four thioredoxin domains of a, b, b', and a' are arranged as a horseshoe shape with two CGHC active sites, respectively, in domains a and a' facing each other at the two ends.  In reduced hPDI, domains a, b, and b' line up in the same plane, whereas domain a' twists ∼45° out.  The two active sites are 27.6 ÅA apart.  In oxidized hPDI, the four domains are differently organized to stay in the same plane, and the distance between the active sites increases to 40.3 ÅA.  In contrast to the closed conformation of reduced hPDI, oxidized hPDI exists in an open state with more exposed hydrophobic areas and a larger cleft with potential for substrate binding.  INNOVATION:  This is the first report of the high-resolution structures of hPDI containing all four domains in both the reduced and the oxidized states.  It reveals the redox-regulated structural dynamic properties of the protein.  CONCLUSION:  The redox-regulated open/closed conformational switch of hPDI endows the protein with versatile target-binding capacities for its enzymatic and chaperone functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWKKds5MV7mi86Js-BCLiifW6udTcc2ebCDGr7MLgwLbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38not1Cnug%253D%253D&md5=ff87e33dc426c3d2a8ad7c76d8a27635</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1089%2Fars.2012.4630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2012.4630%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DKe%26aufirst%3DH.%26aulast%3DGong%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DC.%2BC.%26atitle%3DStructural%2520insights%2520into%2520the%2520redox-regulated%2520dynamic%2520conformations%2520of%2520human%2520protein%2520disulfide%2520isomerase%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2013%26volume%3D19%26spage%3D36%26epage%3D45%26doi%3D10.1089%2Fars.2012.4630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. H.</span></span> <span> </span><span class="NLM_article-title">P4HB and PDIA3 are associated with tumor progression and therapeutic outcome of diffuse gliomas</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.3892/or.2017.6134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.3892%2For.2017.6134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29207176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKhsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=501-510&author=H.+Zouauthor=C.+Wenauthor=Z.+Pengauthor=Y.+Shaoauthor=L.+Huauthor=S.+Liauthor=C.+Liauthor=H.+H.+Zhou&title=P4HB+and+PDIA3+are+associated+with+tumor+progression+and+therapeutic+outcome+of+diffuse+gliomas&doi=10.3892%2For.2017.6134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">P4HB and PDIA3 are associated with tumor progression and therapeutic outcome of diffuse gliomas</span></div><div class="casAuthors">Zou, Hecun; Wen, Chunjie; Peng, Zhigang; Shao, Ying-Ying; Hu, Lei; Li, Shuang; Li, Cuilin; Zhou, Hong-Hao</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">501-510</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Diffuse gliomas are the most common type of primary brain and central nervous system (CNS) tumors.  Protein disulfide isomerases (PDIs) such as P4HB and PDIA3 act as mol. chaperones for reconstructing misfolded proteins, and are involved in endoplasmic reticulum stress and the unfolded protein response.  The present study focused on the role of P4HB and PDIA3 in diffuse gliomas.  Anal. of GEO and HPA data revealed that the expression levels of P4HB and PDIA3 were upregulated in glioma datasets. their increased expression was then validated in 99 glioma specimens compared with 11 non-tumor tissues.  High expression of P4HB and PDIA3 was significantly correlated with high Ki-67 and a high frequency of the TP53 mutation.  Kaplan-Meier survival curve and Cox regression analyses showed that glioma patients with high P4HB and PDIA3 expression had a poor survival outcome, P4HB and PDIA3 could be independent prognostic biomarkers for diffuse gliomas.  In vitro, knockdown of PDIA3 suppressed cell proliferation, induced cell apoptosis, and decreased the migration of glioma cells.  Furthermore, downregulation of P4HB and PDIA3 may contribute to improve the survival of patients who receive chemotherapy and radiotherapy.  The data suggest that high expression of P4HB and PDIA3 plays an important role in glioma progression, and could predict the survival outcome and therapeutic response of glioma patients.  Therefore, protein disulfide isomerases may be explored as prognostic biomarkers and therapeutic targets for diffuse gliomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdzdJGwdCF7bVg90H21EOLACvtfcHk0lhQ6CK7hMkRNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKhsLzF&md5=b7f85021f7bc0db18c2428f18a2fc156</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3892%2For.2017.6134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2017.6134%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DZ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DH.%2BH.%26atitle%3DP4HB%2520and%2520PDIA3%2520are%2520associated%2520with%2520tumor%2520progression%2520and%2520therapeutic%2520outcome%2520of%2520diffuse%2520gliomas%26jtitle%3DOncol.%2520Rep.%26date%3D2018%26volume%3D39%26spage%3D501%26epage%3D510%26doi%3D10.3892%2For.2017.6134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiang, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, G. K.</span></span> <span> </span><span class="NLM_article-title">Hyperoxia resensitizes chemoresistant glioblastoma cells to temozolomide through unfolded protein response</span>. <i>Anticancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">2957</span>– <span class="NLM_lpage">2966</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=24922660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=2957-2966&author=D.+Leeauthor=S.+Sunauthor=A.+S.+Hoauthor=K.+M.+Kiangauthor=X.+Q.+Zhangauthor=F.+F.+Xuauthor=G.+K.+Leung&title=Hyperoxia+resensitizes+chemoresistant+glioblastoma+cells+to+temozolomide+through+unfolded+protein+response"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Hyperoxia resensitizes chemoresistant glioblastoma cells to temozolomide through unfolded protein response</span></div><div class="casAuthors">Lee, Derek; Sun, Stella; Ho, Amy S. W.; Kiang, Karrie M. Y.; Zhang, Xiao Qin; Xu, Fei Fan; Leung, Gilberto K. K.</div><div class="citationInfo"><span class="NLM_cas:title">Anticancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2957-2966</span>CODEN:
                <span class="NLM_cas:coden">ANTRD4</span>;
        ISSN:<span class="NLM_cas:issn">0250-7005</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: Intratumoral hypoxia is assocd. with chemoresistance in glioblastoma multiforme (GBM), a highly malignant brain tumor.  Adaptive response to endoplasmic reticulum stress induced by temozolomide is a major obstacle in recurrent GBM.  We investigated whether hyperoxia resensitizes temozolomide-resistant GBM cells to temozolomide by abrogating the hypoxia-induced, unfolded protein response (UPR)-related protective mechanisms.  Materials and Methods: We examd. changes to key UPR modulators in temozolomide-sensitive and -resistant human GBM cells (D54 and U87) treated with/without temozolomide at different oxygen concns. using western blotting, and cytotoxic benefits of overexpressing key chaperone, P4HB, in GBM cells (U87 and U251) under normoxia and hyperoxia.  Results: Hyperoxia, alone or synergistically with temozolomide, activated the UPR in sensitive and resistant D54 and U87 cell lines.  Hyperoxia also reduced survival benefit of U87 and U251 cells with P4HB overexpression through the UPR.  Conclusion: Hyperoxia enhanced GBM cell sensitivity to temozolomide, likely through UPR, highlighting an important treatment modality targeting chemosensitive and -resistant GBM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG88WkoW_JFbVg90H21EOLACvtfcHk0lhQ6CK7hMkRNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFGrtrrF&md5=0a0164306adb8a760cf6ae2cb14b0548</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DHo%26aufirst%3DA.%2BS.%26aulast%3DKiang%26aufirst%3DK.%2BM.%26aulast%3DZhang%26aufirst%3DX.%2BQ.%26aulast%3DXu%26aufirst%3DF.%2BF.%26aulast%3DLeung%26aufirst%3DG.%2BK.%26atitle%3DHyperoxia%2520resensitizes%2520chemoresistant%2520glioblastoma%2520cells%2520to%2520temozolomide%2520through%2520unfolded%2520protein%2520response%26jtitle%3DAnticancer%2520Res.%26date%3D2014%26volume%3D34%26spage%3D2957%26epage%3D2966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Protein disulfide isomerase: A promising target for cancer therapy</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.drudis.2013.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=24184531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvValsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=222-240&author=S.+Xuauthor=S.+Sankarauthor=N.+Neamati&title=Protein+disulfide+isomerase%3A+A+promising+target+for+cancer+therapy&doi=10.1016%2Fj.drudis.2013.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Protein disulfide isomerase: a promising target for cancer therapy</span></div><div class="casAuthors">Xu, Shili; Sankar, Saranya; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">222-240</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein disulfide isomerase (PDI) has a key role in maintaining cellular homeostasis by mediating oxidative protein folding.  It catalyzes disulfide bond formation, breakage and rearrangement in the endoplasmic reticulum and has chaperone protein activity.  Increasing evidence suggests that PDI supports the survival and progression of several cancers.  During the past decade, robust PDI activity assays have been developed and several PDI inhibitors identified, but none has been approved for clin. use.  Herein, we review current knowledge of the role of PDI in cancer and discuss various assays for measuring the activities of PDI, highlighting their sensitivities and usefulness for high-throughput screening.  The previously reported PDI inhibitors require further validation to serve as bona fide leads and addnl. optimization to generate novel drug candidates for clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquANbcuJ6Bv7Vg90H21EOLACvtfcHk0lgEq0TXDqk0vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvValsbrM&md5=cab1d37a444bf32f1974567b7622f7e0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DSankar%26aufirst%3DS.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DProtein%2520disulfide%2520isomerase%253A%2520A%2520promising%2520target%2520for%2520cancer%2520therapy%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D222%26epage%3D240%26doi%3D10.1016%2Fj.drudis.2013.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lappi, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lensink, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alanen, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salo, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juffer, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddock, L. W.</span></span> <span> </span><span class="NLM_article-title">A conserved arginine plays a role in the catalytic cycle of the protein disulphide isomerases</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2003.10.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.jmb.2003.10.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=14659757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BD3srotVajug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2004&pages=283-295&author=A.+K.+Lappiauthor=M.+F.+Lensinkauthor=H.+I.+Alanenauthor=K.+E.+Saloauthor=M.+Lobellauthor=A.+H.+Jufferauthor=L.+W.+Ruddock&title=A+conserved+arginine+plays+a+role+in+the+catalytic+cycle+of+the+protein+disulphide+isomerases&doi=10.1016%2Fj.jmb.2003.10.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A conserved arginine plays a role in the catalytic cycle of the protein disulphide isomerases</span></div><div class="casAuthors">Lappi A K; Lensink M F; Alanen H I; Salo K E H; Lobell M; Juffer A H; Ruddock L W</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">283-95</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The pK(a) values of the CXXC active-site cysteine residues play a critical role in determining the physiological function of the thioredoxin superfamily.  To act as an efficient thiol-disulphide oxidant the thiolate state of the N-terminal cysteine must be stabilised and the thiolate state of the C-terminal cysteine residue destabilised.  While increasing the pK(a) value of the C-terminal cysteine residue promotes oxidation of substrates, it has an inhibitory effect on the reoxidation of the enzyme, which is promoted by the formation of a thiolate at this position.  Since reoxidation is essential to complete the catalytic cycle, the differential requirement for a high and a low pK(a) value for the C-terminal cysteine residue for different steps in the reaction presents us with a paradox.  Here, we report the identification of a conserved arginine residue, located in the loop between beta5 and alpha4 of the catalytic domains of the human protein disulphide isomerase (PDI) family, which is critical for the catalytic function of PDI, ERp57, ERp72 and P5, specifically for reoxidation.  An examination of the published NMR structure for the a domain of PDI combined with molecular dynamic studies suggest that the side-chain of this arginine residue moves into and out of the active-site locale and that this has a very marked effect on the pK(a) value of the active-site cysteine residues.  This intra-domain motion resolves the apparent dichotomy of the pK(a) requirements for the C-terminal active-site cysteine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0DS9fXAJ2HWx8TEN9K18QfW6udTcc2eY-l34x8om50bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srotVajug%253D%253D&md5=546c07c515ab7212a38304a18c2cb4de</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2003.10.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2003.10.051%26sid%3Dliteratum%253Aachs%26aulast%3DLappi%26aufirst%3DA.%2BK.%26aulast%3DLensink%26aufirst%3DM.%2BF.%26aulast%3DAlanen%26aufirst%3DH.%2BI.%26aulast%3DSalo%26aufirst%3DK.%2BE.%26aulast%3DLobell%26aufirst%3DM.%26aulast%3DJuffer%26aufirst%3DA.%2BH.%26aulast%3DRuddock%26aufirst%3DL.%2BW.%26atitle%3DA%2520conserved%2520arginine%2520plays%2520a%2520role%2520in%2520the%2520catalytic%2520cycle%2520of%2520the%2520protein%2520disulphide%2520isomerases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D335%26spage%3D283%26epage%3D295%26doi%3D10.1016%2Fj.jmb.2003.10.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butkevich, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zandi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petasis, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16348</span>– <span class="NLM_lpage">16353</span>, <span class="refDoi"> DOI: 10.1073/pnas.1205226109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.1205226109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22988091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16348-16353&author=S.+Xuauthor=A.+N.+Butkevichauthor=R.+Yamadaauthor=Y.+Zhouauthor=B.+Debnathauthor=R.+Duncanauthor=E.+Zandiauthor=N.+A.+Petasisauthor=N.+Neamati&title=Discovery+of+an+orally+active+small-molecule+irreversible+inhibitor+of+protein+disulfide+isomerase+for+ovarian+cancer+treatment&doi=10.1073%2Fpnas.1205226109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span></div><div class="casAuthors">Xu, Shili; Butkevich, Alexey N.; Yamada, Roppei; Zhou, Yu; Debnath, Bikash; Duncan, Roger; Zandi, Ebrahim; Petasis, Nicos A.; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16348-16353, S16348/1-S16348/18</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI), an endoplasmic reticulum chaper-one protein, catalyzes disulfide bond breakage, formation, and rearrangement.  The effect of PDI inhibition on ovarian cancer progression is not yet clear, and there is a need for potent selective, and safe small-mol. inhibitors of PDI.  Here, we report a class of propynoic acid carbamoyl Me amides (PACMAs) that are active against a panel of human ovarian cancer cell lines.  Using fluorescent derivs., 2D gel electrophoresis, and MS, we established that one of the most active analogs, acts as an irreversible small-mol. inhibitor of PDI, forming a covalent bond with the active site cysteines of PDI.  We also showed that PDI activity is essential for the survival and proliferation of human ovarian cancer cells.  In vivo that compd. showed tumor targeting ability and significantly suppressed ovarian tumor growth without causing toxicity to normal tissues.  These irreversible small-mol. PDI inhibitors represent an important approach for the development of targeted anticancer agents for ovarian cancer therapy, and they can also serve as useful probes for investigating the biol. of PDI-implicated pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozZP4n4am-hLVg90H21EOLACvtfcHk0ljX0WtWImGCGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE&md5=26801591b73d67710e06ef184ee9ddde</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1205226109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1205226109%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DButkevich%26aufirst%3DA.%2BN.%26aulast%3DYamada%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DDuncan%26aufirst%3DR.%26aulast%3DZandi%26aufirst%3DE.%26aulast%3DPetasis%26aufirst%3DN.%2BA.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520active%2520small-molecule%2520irreversible%2520inhibitor%2520of%2520protein%2520disulfide%2520isomerase%2520for%2520ovarian%2520cancer%2520treatment%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16348%26epage%3D16353%26doi%3D10.1073%2Fpnas.1205226109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandjean, J. M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witt, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Day, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoulders, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerapana, E.</span></span> <span> </span><span class="NLM_article-title">Characterization of an A-site selective protein disulfide isomerase A1 inhibitor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2035</span>– <span class="NLM_lpage">2043</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFWgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2035-2043&author=K.+S.+Coleauthor=J.+M.+D.+Grandjeanauthor=K.+Chenauthor=C.+H.+Wittauthor=J.+O%E2%80%99Dayauthor=M.+D.+Shouldersauthor=R.+L.+Wisemanauthor=E.+Weerapana&title=Characterization+of+an+A-site+selective+protein+disulfide+isomerase+A1+inhibitor&doi=10.1021%2Facs.biochem.8b00178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of an a-Site Selective Protein Disulfide Isomerase A1 Inhibitor</span></div><div class="casAuthors">Cole, Kyle S.; Grandjean, Julia M. D.; Chen, Kenny; Witt, Collin H.; O'Day, Johanna; Shoulders, Matthew D.; Wiseman, R. Luke; Weerapana, Eranthie</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2035-2043</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein disulfide isomerase A1 (PDIA1) is an endoplasmic reticulum (ER)-localized thiol-disulfide oxidoreductase that is an important folding catalyst for secretory pathway proteins.  PDIA1 contains two active-site domains (a and a'), each contg. a Cys-Gly-His-Cys (CGHC) active-site motif.  The two active-site domains share 37% sequence identity and function independently to perform disulfide-bond redn., oxidn., and isomerization.  Numerous inhibitors for PDIA1 have been reported, yet the selectivity of these inhibitors toward the a and a' sites is poorly characterized.  Here, we identify a potent and selective PDIA1 inhibitor, KSC-34, with 30-fold selectivity for the a site over the a' site.  KSC-34 displays time-dependent inhibition of PDIA1 reductase activity in vitro with a kinact/KI of 9.66 × 103 M-1 s-1 and is selective for PDIA1 over other members of the PDI family, and other cellular cysteine-contg. proteins.  We provide the first cellular characterization of an a-site selective PDIA1 inhibitor and demonstrate that KSC-34 has minimal sustained effects on the cellular unfolded protein response (UPR), indicating that a-site inhibition does not induce global protein folding-assocd. ER stress.  KSC-34 treatment significantly decreases the rate of secretion of the destabilized amyloidogenic antibody light chain ALLC, thereby minimizing pathogenic amyloidogenic extracellular proteins that rely on high PDIA1 activity for proper folding and secretion.  Given the poor understanding of the contribution of each PDIA1 active site to the (patho)physiol. functions of PDIA1, site selective inhibitors like KSC-34 provide useful tools for delineating the pathol. role and therapeutic potential of PDIA1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMaqymP7LrLLVg90H21EOLACvtfcHk0ljX0WtWImGCGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFWgsL8%253D&md5=95b6c4e046d4e8655ae0cfb45eed78a7</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00178%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DK.%2BS.%26aulast%3DGrandjean%26aufirst%3DJ.%2BM.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DWitt%26aufirst%3DC.%2BH.%26aulast%3DO%25E2%2580%2599Day%26aufirst%3DJ.%26aulast%3DShoulders%26aufirst%3DM.%2BD.%26aulast%3DWiseman%26aufirst%3DR.%2BL.%26aulast%3DWeerapana%26aufirst%3DE.%26atitle%3DCharacterization%2520of%2520an%2520A-site%2520selective%2520protein%2520disulfide%2520isomerase%2520A1%2520inhibitor%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26spage%3D2035%26epage%3D2043%26doi%3D10.1021%2Facs.biochem.8b00178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. C.</span></span> <span> </span><span class="NLM_article-title">3,4-Methylenedioxy-beta-nitrostyrene inhibits adhesion and migration of human triple-negative breast cancer cells by suppressing beta1 integrin function and surface protein disulfide isomerase</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2015.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.biochi.2015.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=25593085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2htb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2015&pages=81-92&author=I.+H.+Chenauthor=F.+R.+Changauthor=Y.+C.+Wuauthor=P.+H.+Kungauthor=C.+C.+Wu&title=3%2C4-Methylenedioxy-beta-nitrostyrene+inhibits+adhesion+and+migration+of+human+triple-negative+breast+cancer+cells+by+suppressing+beta1+integrin+function+and+surface+protein+disulfide+isomerase&doi=10.1016%2Fj.biochi.2015.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">3,4-Methylenedioxy-β-nitrostyrene inhibits adhesion and migration of human triple-negative breast cancer cells by suppressing β1 integrin function and surface protein disulfide isomerase</span></div><div class="casAuthors">Chen, I-Hua; Chang, Fang-Rong; Wu, Yang-Chang; Kung, Po-Hsiung; Wu, Chin-Chung</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">81-92</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Triple neg. breast cancer (TNBC) exhibits an aggressive clin. course by high metastatic potential.  It is known that integrin-mediated cell adhesion and migration are important for cancer metastasis.  In the present study, a synthetic compd., 3, 4-methyenedioxy-β-nitrostyrene (MNS), significantly inhibited adhesion of TNBC cell lines to different extracellular matrix (ECM) components.  The antimetastatic capacity of MNS was also obsd. through reducing TNBC cells migration and invasion without affecting cell viability.  Confocal microscopy revealed that MNS disrupted the formation of focal adhesion complex and actin stress fiber networks.  Consistent with this finding, MNS inhibited phosphorylation of focal adhesion kinase (FAK) and paxillin as detected by Western blot anal.  In exploring the underlying mechanism, we found that MNS inhibited phosphorylation of FAK as a result of reducing β1 integrin activation and clustering.  A cell-impermeable dithiol reagent, 2, 3-dimercaptopropane-1-sulfonic acid abrogated all of MNS's actions, indicating that MNS may react with thiol groups of cell surface proteins that are involved in regulation of β1 integrin function as well as cell adhesion and migration.  Cell surface protein disulfide isomerase (PDI) has been reported to be essential for the affinity modulation of β integrins.  We also demonstrated that MNS inhibited PDI activity both in a pure enzyme system and in intact cancer cells.  Taken together, our results suggest that MNS inhibits in vitro metastatic properties of TNBC cells through suppression of β1 integrin activation and focal adhesion signaling.  Moreover, inhibition of surface PDI may contribute, at least in part, to the actions of MNS.  These results suggest that MNS has a potential to be developed as an anticancer agent for treatment of TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsh0eLDG0VdrVg90H21EOLACvtfcHk0ljX0WtWImGCGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2htb4%253D&md5=2af854c64d414d515af7a8fe426652c2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2015.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2015.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BH.%26aulast%3DChang%26aufirst%3DF.%2BR.%26aulast%3DWu%26aufirst%3DY.%2BC.%26aulast%3DKung%26aufirst%3DP.%2BH.%26aulast%3DWu%26aufirst%3DC.%2BC.%26atitle%3D3%252C4-Methylenedioxy-beta-nitrostyrene%2520inhibits%2520adhesion%2520and%2520migration%2520of%2520human%2520triple-negative%2520breast%2520cancer%2520cells%2520by%2520suppressing%2520beta1%2520integrin%2520function%2520and%2520surface%2520protein%2520disulfide%2520isomerase%26jtitle%3DBiochimie%26date%3D2015%26volume%3D110%26spage%3D81%26epage%3D92%26doi%3D10.1016%2Fj.biochi.2015.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffstrom, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turmel, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein disulfide isomerase suppress apoptosis induced by misfolded proteins</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">900</span>– <span class="NLM_lpage">906</span>, <span class="refDoi"> DOI: 10.1038/nchembio.467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1038%2Fnchembio.467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=21079601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVansLbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=900-906&author=B.+G.+Hoffstromauthor=A.+Kaplanauthor=R.+Letsoauthor=R.+S.+Schmidauthor=G.+J.+Turmelauthor=D.+C.+Loauthor=B.+R.+Stockwell&title=Inhibitors+of+protein+disulfide+isomerase+suppress+apoptosis+induced+by+misfolded+proteins&doi=10.1038%2Fnchembio.467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of protein disulfide isomerase suppress apoptosis induced by misfolded proteins</span></div><div class="casAuthors">Hoffstrom, Benjamin G.; Kaplan, Anna; Letso, Reka; Schmid, Ralf S.; Turmel, Gregory J.; Lo, Donald C.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">900-906</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A hallmark of many neurodegenerative diseases is accumulation of misfolded proteins within neurons, leading to cellular dysfunction and cell death.  Although several mechanisms have been proposed to link protein misfolding to cellular toxicity, the connection remains enigmatic.  Here, we report a cell death pathway involving protein disulfide isomerase (PDI), a protein chaperone that catalyzes isomerization, redn. and oxidn. of disulfides.  Through a small mol. screening approach, we discovered five structurally distinct compds. that prevent apoptosis induced by mutant huntingtin protein.  Using modified Huisgen cycloaddn. chem., we then identified PDI as the mol. target of these small mols.  Expression of polyglutamine-expanded huntingtin exon 1 in PC12 cells caused PDI to accumulate at mitochondrial-assocd. ER membranes and trigger apoptotic cell death via mitochondrial outer-membrane permeabilization.  Inhibiting PDI in rat brain cells suppressed the toxicity of mutant huntingtin exon 1 and Aβ peptides processed from the amyloid precursor protein.  This pro-apoptotic function of PDI represents a new mechanism linking protein misfolding and apoptotic cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWvM36lxKwEbVg90H21EOLACvtfcHk0lil5yY-Dqf9fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVansLbN&md5=c8b685fcf565dfb37dcff4097e866481</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.467%26sid%3Dliteratum%253Aachs%26aulast%3DHoffstrom%26aufirst%3DB.%2BG.%26aulast%3DKaplan%26aufirst%3DA.%26aulast%3DLetso%26aufirst%3DR.%26aulast%3DSchmid%26aufirst%3DR.%2BS.%26aulast%3DTurmel%26aufirst%3DG.%2BJ.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DInhibitors%2520of%2520protein%2520disulfide%2520isomerase%2520suppress%2520apoptosis%2520induced%2520by%2520misfolded%2520proteins%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D900%26epage%3D906%26doi%3D10.1038%2Fnchembio.467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butkevich, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zandi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petasis, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16348</span>– <span class="NLM_lpage">16353</span>, <span class="refDoi"> DOI: 10.1073/pnas.1205226109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.1205226109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22988091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16348-16353&author=S.+Xuauthor=A.+N.+Butkevichauthor=R.+Yamadaauthor=Y.+Zhouauthor=B.+Debnathauthor=R.+Duncanauthor=E.+Zandiauthor=N.+A.+Petasisauthor=N.+Neamati&title=Discovery+of+an+orally+active+small-molecule+irreversible+inhibitor+of+protein+disulfide+isomerase+for+ovarian+cancer+treatment&doi=10.1073%2Fpnas.1205226109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an orally active small-molecule irreversible inhibitor of protein disulfide isomerase for ovarian cancer treatment</span></div><div class="casAuthors">Xu, Shili; Butkevich, Alexey N.; Yamada, Roppei; Zhou, Yu; Debnath, Bikash; Duncan, Roger; Zandi, Ebrahim; Petasis, Nicos A.; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16348-16353, S16348/1-S16348/18</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI), an endoplasmic reticulum chaper-one protein, catalyzes disulfide bond breakage, formation, and rearrangement.  The effect of PDI inhibition on ovarian cancer progression is not yet clear, and there is a need for potent selective, and safe small-mol. inhibitors of PDI.  Here, we report a class of propynoic acid carbamoyl Me amides (PACMAs) that are active against a panel of human ovarian cancer cell lines.  Using fluorescent derivs., 2D gel electrophoresis, and MS, we established that one of the most active analogs, acts as an irreversible small-mol. inhibitor of PDI, forming a covalent bond with the active site cysteines of PDI.  We also showed that PDI activity is essential for the survival and proliferation of human ovarian cancer cells.  In vivo that compd. showed tumor targeting ability and significantly suppressed ovarian tumor growth without causing toxicity to normal tissues.  These irreversible small-mol. PDI inhibitors represent an important approach for the development of targeted anticancer agents for ovarian cancer therapy, and they can also serve as useful probes for investigating the biol. of PDI-implicated pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozZP4n4am-hLVg90H21EOLACvtfcHk0lil5yY-Dqf9fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFGktLzE&md5=26801591b73d67710e06ef184ee9ddde</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1205226109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1205226109%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DButkevich%26aufirst%3DA.%2BN.%26aulast%3DYamada%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DDuncan%26aufirst%3DR.%26aulast%3DZandi%26aufirst%3DE.%26aulast%3DPetasis%26aufirst%3DN.%2BA.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DDiscovery%2520of%2520an%2520orally%2520active%2520small-molecule%2520irreversible%2520inhibitor%2520of%2520protein%2520disulfide%2520isomerase%2520for%2520ovarian%2520cancer%2520treatment%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16348%26epage%3D16353%26doi%3D10.1073%2Fpnas.1205226109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Won, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span> <span> </span><span class="NLM_article-title">Protein disulfide isomerase inhibition synergistically enhances the efficacy of sorafenib for hepatocellular carcinoma</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">855</span>– <span class="NLM_lpage">868</span>, <span class="refDoi"> DOI: 10.1002/hep.29237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1002%2Fhep.29237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28439950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOhsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=855-868&author=J.+K.+Wonauthor=S.+J.+Yuauthor=C.+Y.+Hwangauthor=S.+H.+Choauthor=S.+M.+Parkauthor=K.+Kimauthor=W.+M.+Choiauthor=H.+Choauthor=E.+J.+Choauthor=J.+H.+Leeauthor=K.+B.+Leeauthor=Y.+J.+Kimauthor=K.+S.+Suhauthor=J.+J.+Jangauthor=C.+Y.+Kimauthor=J.+H.+Yoonauthor=K.+H.+Cho&title=Protein+disulfide+isomerase+inhibition+synergistically+enhances+the+efficacy+of+sorafenib+for+hepatocellular+carcinoma&doi=10.1002%2Fhep.29237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protein disulfide isomerase inhibition synergistically enhances the efficacy of sorafenib for hepatocellular carcinoma</span></div><div class="casAuthors">Won, Jae-Kyung; Yu, Su Jong; Hwang, Chae Young; Cho, Sung-Hwan; Park, Sang-Min; Kim, Kwangsoo; Choi, Won-Mook; Cho, Hyeki; Cho, Eun Ju; Lee, Jeong-Hoon; Lee, Kyung Bun; Kim, Yoon Jun; Suh, Kyung-Suk; Jang, Ja-June; Kim, Chung Yong; Yoon, Jung-Hwan; Cho, Kwang-Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">855-868</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Sorafenib is the only approved targeted drug for hepatocellular carcinoma (HCC), but its effect on patients' survival gain is limited and varies over a wide range depending on pathogenetic conditions.  Thus, enhancing the efficacy of sorafenib and finding a reliable predictive biomarker are crucial to achieve efficient control of HCCs.  In this study, we utilized a systems approach by combining transcriptome anal. of the mRNA changes in HCC cell lines in response to sorafenib with network anal. to investigate the action and resistance mechanism of sorafenib.  Gene list functional enrichment anal. and gene set enrichment anal. revealed that proteotoxic stress and apoptosis modules are activated in the presence of sorafenib.  Further anal. of the endoplasmic reticulum stress network model, combined with in vitro expts., showed that introducing an addnl. stress by treating the orally active protein disulfide isomerase (PDI) inhibitor (PACMA 31) can synergistically increase the efficacy of sorafenib in vitro and in vivo, which was confirmed using a mouse xenograft model.  We also found that HCC patients with high PDI expression show resistance to sorafenib and poor clin. outcomes, compared to the low-PDI-expression group.  Conclusion: These results suggest that PDI is a promising therapeutic target for enhancing the efficacy of sorafenib and can also be a biomarker for predicting sorafenib responsiveness. (Hepatol. 2017;66:855-868).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI9MWIoMmE8rVg90H21EOLACvtfcHk0lil5yY-Dqf9fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOhsb3E&md5=321aa98f9948c3e949dbd9a445ef5821</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fhep.29237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.29237%26sid%3Dliteratum%253Aachs%26aulast%3DWon%26aufirst%3DJ.%2BK.%26aulast%3DYu%26aufirst%3DS.%2BJ.%26aulast%3DHwang%26aufirst%3DC.%2BY.%26aulast%3DCho%26aufirst%3DS.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BM.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DW.%2BM.%26aulast%3DCho%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DE.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLee%26aufirst%3DK.%2BB.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DSuh%26aufirst%3DK.%2BS.%26aulast%3DJang%26aufirst%3DJ.%2BJ.%26aulast%3DKim%26aufirst%3DC.%2BY.%26aulast%3DYoon%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DK.%2BH.%26atitle%3DProtein%2520disulfide%2520isomerase%2520inhibition%2520synergistically%2520enhances%2520the%2520efficacy%2520of%2520sorafenib%2520for%2520hepatocellular%2520carcinoma%26jtitle%3DHepatology%26date%3D2017%26volume%3D66%26spage%3D855%26epage%3D868%26doi%3D10.1002%2Fhep.29237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, C.</span></span> <span> </span><span class="NLM_article-title">Challenges in the evaluation of thiol-reactive inhibitors of human protein disulfide Isomerase</span>. <i>Free Radical Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1016/j.freeradbiomed.2017.04.367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.freeradbiomed.2017.04.367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28465261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFCisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2017&pages=741-749&author=C.+K.+Fosterauthor=C.+Thorpe&title=Challenges+in+the+evaluation+of+thiol-reactive+inhibitors+of+human+protein+disulfide+Isomerase&doi=10.1016%2Fj.freeradbiomed.2017.04.367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges in the evaluation of thiol-reactive inhibitors of human protein disulfide isomerase</span></div><div class="casAuthors">Foster, Celia K.; Thorpe, Colin</div><div class="citationInfo"><span class="NLM_cas:title">Free Radical Biology & Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">741-749</span>CODEN:
                <span class="NLM_cas:coden">FRBMEH</span>;
        ISSN:<span class="NLM_cas:issn">0891-5849</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This paper addresses how to evaluate the efficacy of the growing inventory of thiol-reactive inhibitors of mammalian protein disulfide isomerase (PDI) under realistic concns. of potentially competing thiol-contg. peptides and proteins.  For this purpose, we introduced a variant of the widely-used reductase assay by using a com.-available cysteine deriv., (BODIPY FL L-cystine; BD-SS), that yields a 55-fold increase in fluorescence (excitation/emission; 490/513 nm) on scission of the disulfide bond.  This plate reader-compatible method detects human PDI down to 5-10 nM, can utilize a range of thiol substrates (including 5 μM dithiothreitol, 10 μM reduced RNase thiols, and 5 mM glutathione; GSH), and can operate from pH 6-9.5 in a variety of buffers.  PDI assays often employ low micromolar levels of substrates leading to ambiguities when thiol-directed inhibitors are evaluated.  The present work utilized 5 mM GSH for both pre-incubation and assay phases to more realistically reflect the high concn. of thiols that an inhibitor would encounter intracellularly.  Extracellular PDI faced a much lower concn. of potentially competing thiols; to assess reductase activity under these conditions, the pre-reduced PDI was treated with an inhibitor and then the fluorescence increase upon redn. of BD-SS was followed in the absence of addnl. competing thiols.  Both assay modes were tested with 4 mechanistically diverse PDI inhibitors.  Two reversible reagents, 3,4-methylenedioxy-β-nitrostyrene (MNS) and the arsenical, APAO, were found to be strong inhibitors of PDI in the absence of competing thiols, but were ineffective in the presence of 5 mM GSH.  A further examn. of the nitrostyrene showed that MNS not only formed facile Michael adducts with GSH, but also with the thiols of unfolded proteins (Kd values of 7 and <0.1 μM, resp.) suggesting the existence of multiple potential intracellular targets for this membrane-permeant reagent.  The inhibition of PDI by the irreversible alkylating agent, the chloroacetamide, 16F16, was found to be only modestly attenuated by 5 mM GSH.  Finally, the thiol-independent flavonoid inhibitor, quercetin-3-O-rutinoside (rutin), was found to show equal efficacy in reoxidn. and turnover assay types.  Thus, this work provides a framework to evaluate inhibitors that may target the CxxC motifs of PDI and addresses some of the complexities in the interpretation of the behavior of thiol-directed reagents in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfLqAX92kPjLVg90H21EOLACvtfcHk0liIwVClTkLN6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFCisbc%253D&md5=0b4846f207b567e66275bb8f6ddec5a0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.freeradbiomed.2017.04.367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.freeradbiomed.2017.04.367%26sid%3Dliteratum%253Aachs%26aulast%3DFoster%26aufirst%3DC.%2BK.%26aulast%3DThorpe%26aufirst%3DC.%26atitle%3DChallenges%2520in%2520the%2520evaluation%2520of%2520thiol-reactive%2520inhibitors%2520of%2520human%2520protein%2520disulfide%2520Isomerase%26jtitle%3DFree%2520Radical%2520Biol.%2520Med.%26date%3D2017%26volume%3D108%26spage%3D741%26epage%3D749%26doi%3D10.1016%2Fj.freeradbiomed.2017.04.367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aye, Y.</span></span> <span> </span><span class="NLM_article-title">Privileged electrophile sensors: A resource for covalent drug development</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">787</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.chembiol.2017.05.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28648380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=787-800&author=M.+J.+C.+Longauthor=Y.+Aye&title=Privileged+electrophile+sensors%3A+A+resource+for+covalent+drug+development&doi=10.1016%2Fj.chembiol.2017.05.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Privileged Electrophile Sensors: A Resource for Covalent Drug Development</span></div><div class="casAuthors">Long, Marcus John Curtis; Aye, Yimon</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">787-800</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">This Perspective delineates how redox signaling affects the activity of specific enzyme isoforms and how this property may be harnessed for rational drug design.  Covalent drugs have resurged in recent years and several reports have extolled the general virtues of developing irreversible inhibitors.  Indeed, many modern pharmaceuticals contain electrophilic appendages.  Several invoke a warhead that hijacks active-site nucleophiles whereas others take advantage of spectator nucleophilic side chains that do not participate in enzymic chem., but are poised to bind/react with electrophiles.  The latest data suggest that innate electrophile sensing-which enables rapid reaction with an endogenous signaling electrophile-is a quintessential resource for the development of covalent drugs.  For instance, based on recent work documenting isoform-specific electrophile sensing, isoenzyme non-specific drugs may be converted to isoenzyme-specific analogs by hijacking privileged first-responder electrophile-sensing cysteines.  Because this approach targets functionally relevant cysteines, we can simultaneously harness previously untapped moonlighting roles of enzymes linked to redox sensing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWQ9w0b7pay7Vg90H21EOLACvtfcHk0liIwVClTkLN6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtr3E&md5=b7edc384ac48df7465303f33f7b1cca8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.023%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BJ.%2BC.%26aulast%3DAye%26aufirst%3DY.%26atitle%3DPrivileged%2520electrophile%2520sensors%253A%2520A%2520resource%2520for%2520covalent%2520drug%2520development%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D787%26epage%3D800%26doi%3D10.1016%2Fj.chembiol.2017.05.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogunrinu, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sontheimer, H.</span></span> <span> </span><span class="NLM_article-title">Hypoxia increases the dependence of glioma cells on glutathione</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">37716</span>– <span class="NLM_lpage">37724</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.161190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M110.161190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=20858898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=37716-37724&author=T.+A.+Ogunrinuauthor=H.+Sontheimer&title=Hypoxia+increases+the+dependence+of+glioma+cells+on+glutathione&doi=10.1074%2Fjbc.M110.161190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Hypoxia increases the dependence of glioma cells on glutathione</span></div><div class="casAuthors">Ogunrinu, Toyin Adeyemi; Sontheimer, Harald</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">37716-37724</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Glutathione (GSH) is an essential antioxidant responsible for the maintenance of intracellular redox homeostasis.  As tumors outgrow their blood supply and become hypoxic, their redox homeostasis is challenged by the prodn. of nitric oxide and reactive oxygen species (ROS).  In gliomas, the sustained import of L-cystine via the L-cystine/L-glutamate exchanger, system x[Formula: see text], is rate-limiting for the synthesis of GSH.  We show that hypoxia causes a significant increase in NO and ROS but without affecting glioma cell growth.  This is explained by a concomitant increase in the utilization of GSH, which is accompanied by an increase in the cell-surface expression of xCT, the catalytic subunit of system x[Formula: see text], and L-cystine uptake.  Growth was inhibited when GSH synthesis was blocked by buthionine sulfoximine (BSO), an inhibitor of the enzyme required for GSH synthesis, or when cells were deprived of L-cystine.  These findings suggest that glioma cells show an increased requirement for GSH to maintain growth under hypoxic conditions.  Therefore, approaches that limit GSH synthesis such as blocking system may be considered as an adjuvant to radiation or chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvUfkv7LQy2rVg90H21EOLACvtfcHk0lhE3FzL2ZSiUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVCrt73E&md5=15779fe20696b06e00499cb5899710e8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.161190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.161190%26sid%3Dliteratum%253Aachs%26aulast%3DOgunrinu%26aufirst%3DT.%2BA.%26aulast%3DSontheimer%26aufirst%3DH.%26atitle%3DHypoxia%2520increases%2520the%2520dependence%2520of%2520glioma%2520cells%2520on%2520glutathione%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D37716%26epage%3D37724%26doi%3D10.1074%2Fjbc.M110.161190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, P.</span></span> <span> </span><span class="NLM_article-title">Unraveling the potential role of glutathione in multiple forms of cell death in cancer therapy</span>. <i>Oxid. Med. Cell. Longevity</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2019</i></span>, <span class="NLM_elocation-id">3150145</span> <span class="refDoi"> DOI: 10.1155/2019/3150145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1155%2F2019%2F3150145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=31281572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BB3MzkvFKltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2019&publication_year=2019&author=H.+Lvauthor=C.+Zhenauthor=J.+Liuauthor=P.+Yangauthor=L.+Huauthor=P.+Shang&title=Unraveling+the+potential+role+of+glutathione+in+multiple+forms+of+cell+death+in+cancer+therapy&doi=10.1155%2F2019%2F3150145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Unraveling the Potential Role of Glutathione in Multiple Forms of Cell Death in Cancer Therapy</span></div><div class="casAuthors">Lv Huanhuan; Zhen Chenxiao; Liu Junyu; Yang Pengfei; Lv Huanhuan; Zhen Chenxiao; Liu Junyu; Yang Pengfei; Shang Peng; Lv Huanhuan; Hu Lijiang; Lv Huanhuan; Yang Pengfei; Shang Peng; Lv Huanhuan; Zhen Chenxiao; Liu Junyu; Yang Pengfei; Shang Peng</div><div class="citationInfo"><span class="NLM_cas:title">Oxidative medicine and cellular longevity</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">2019</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3150145</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Glutathione is the principal intracellular antioxidant buffer against oxidative stress and mainly exists in the forms of reduced glutathione (GSH) and oxidized glutathione (GSSG).  The processes of glutathione synthesis, transport, utilization, and metabolism are tightly controlled to maintain intracellular glutathione homeostasis and redox balance.  As for cancer cells, they exhibit a greater ROS level than normal cells in order to meet the enhanced metabolism and vicious proliferation; meanwhile, they also have to develop an increased antioxidant defense system to cope with the higher oxidant state.  Growing numbers of studies have implicated that altering the glutathione antioxidant system is associated with multiple forms of programmed cell death in cancer cells.  In this review, we firstly focus on glutathione homeostasis from the perspectives of glutathione synthesis, distribution, transportation, and metabolism.  Then, we discuss the function of glutathione in the antioxidant process.  Afterwards, we also summarize the recent advance in the understanding of the mechanism by which glutathione plays a key role in multiple forms of programmed cell death, including apoptosis, necroptosis, ferroptosis, and autophagy.  Finally, we highlight the glutathione-targeting therapeutic approaches toward cancers.  A comprehensive review on the glutathione homeostasis and the role of glutathione depletion in programmed cell death provide insight into the redox-based research concerning cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2QtG6nKnyx8zK395ZObgVfW6udTcc2ebo7M-g0nfmTbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MzkvFKltA%253D%253D&md5=d9433b9d3229d62df295289a94505636</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1155%2F2019%2F3150145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2019%252F3150145%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DH.%26aulast%3DZhen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DShang%26aufirst%3DP.%26atitle%3DUnraveling%2520the%2520potential%2520role%2520of%2520glutathione%2520in%2520multiple%2520forms%2520of%2520cell%2520death%2520in%2520cancer%2520therapy%26jtitle%3DOxid.%2520Med.%2520Cell.%2520Longevity%26date%3D2019%26volume%3D2019%26doi%3D10.1155%2F2019%2F3150145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tachibana, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahr Winther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appenzeller-Herzog, C.</span></span> <span> </span><span class="NLM_article-title">Intracellular glutathione pools are heterogeneously concentrated</span>. <i>Redox Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">508</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1016/j.redox.2013.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.redox.2013.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=24251119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=508-513&author=D.+Monteroauthor=C.+Tachibanaauthor=J.+Rahr+Wintherauthor=C.+Appenzeller-Herzog&title=Intracellular+glutathione+pools+are+heterogeneously+concentrated&doi=10.1016%2Fj.redox.2013.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular glutathione pools are heterogeneously concentrated</span></div><div class="casAuthors">Montero, Davide; Tachibana, Christine; Rahr Winther, Jakob; Appenzeller-Herzog, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Redox Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">508-513</span>CODEN:
                <span class="NLM_cas:coden">RBEIB3</span>;
        ISSN:<span class="NLM_cas:issn">2213-2317</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Glutathione is present in millimolar concns. in the cell, but its relative distribution among cellular compartments remains elusive.  We have chosen the endoplasmic reticulum (ER) as an example organelle to study compartment-specific glutathione levels.  Using a glutaredoxin sensor (sCGrx1pER), which rapidly and specifically equilibrates with the reduced glutathione (GSH)-glutathione disulfide (GSSG) redox couple with known equil. const., we showed that the [GSH]:[GSSG] ratio in the ER of intact HeLa cells is less than 7:1.  Taking into consideration the previously detd. value for [GSH]2:[GSSG] in the ER of 83 mM, this translates into a total glutathione concn. in the ER ([GStot]=[GSH]+2[GSSG]) of greater than 15 mM.  Since the integrated, intracellular [GStot] was measured as ∼7 mM, we conclude the existence of a [GStot] gradient across the ER membrane.  A possible homeostatic mechanism by which cytosol-derived glutathione is trapped in the ER is discussed.  We propose a high [GStot] as a distinguishing feature of the ER environment compared to the extracellular space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtUlaQYPjRKLVg90H21EOLACvtfcHk0lhE3FzL2ZSiUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFGlsg%253D%253D&md5=cb40f4e69e51f8b6f09f133f7b1b87f3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.redox.2013.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.redox.2013.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DMontero%26aufirst%3DD.%26aulast%3DTachibana%26aufirst%3DC.%26aulast%3DRahr%2BWinther%26aufirst%3DJ.%26aulast%3DAppenzeller-Herzog%26aufirst%3DC.%26atitle%3DIntracellular%2520glutathione%2520pools%2520are%2520heterogeneously%2520concentrated%26jtitle%3DRedox%2520Biol.%26date%3D2013%26volume%3D1%26spage%3D508%26epage%3D513%26doi%3D10.1016%2Fj.redox.2013.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boskovic, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freedy, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanathan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pop, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa
Lazu, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calarese, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Love, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, A. N.</span></span> <span> </span><span class="NLM_article-title">Inhibition of zinc-dependent histone deacetylases with a chemically triggered electrophile</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1844</span>– <span class="NLM_lpage">1851</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslyqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1844-1851&author=Z.+V.+Boskovicauthor=M.+M.+Kempauthor=A.+M.+Freedyauthor=V.+S.+Viswanathanauthor=M.+S.+Popauthor=J.+H.+Fullerauthor=N.+M.+Martinezauthor=S.+O.+Figueroa%0ALazuauthor=J.+A.+Hongauthor=T.+A.+Lewisauthor=D.+Calareseauthor=J.+D.+Loveauthor=A.+Vetereauthor=S.+C.+Almoauthor=S.+L.+Schreiberauthor=A.+N.+Koehler&title=Inhibition+of+zinc-dependent+histone+deacetylases+with+a+chemically+triggered+electrophile&doi=10.1021%2Facschembio.6b00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Zinc-Dependent Histone Deacetylases with a Chemically Triggered Electrophile</span></div><div class="casAuthors">Boskovic, Zarko V.; Kemp, Melissa M.; Freedy, Allyson M.; Viswanathan, Vasanthi S.; Pop, Marius S.; Fuller, Jason H.; Martinez, Nicole M.; Figueroa Lazu, Samuel O.; Hong, Jiyoung A.; Lewis, Timothy A.; Calarese, Daniel; Love, James D.; Vetere, Amedeo; Almo, Steven C.; Schreiber, Stuart L.; Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1844-1851</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Unbiased binding assays involving small-mol. microarrays were used to identify compds. that display unique patterns of selectivity among members of the zinc-dependent histone deacetylase family of enzymes.  A novel, hydroxyquinoline-contg. compd., BRD4354, was shown to preferentially inhibit activity of HDAC5 and HDAC9 in vitro.  Inhibition of deacetylase activity appears to be time-dependent and reversible.  Mechanistic studies suggest that the compd. undergoes zinc-catalyzed decompn. to an ortho-quinone methide, which covalently modifies nucleophilic cysteines within the proteins.  The covalent nature of the compd.-enzyme interaction has been demonstrated in expts. with biotinylated probe compd. and with electrospray ionization-mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGhTWSlnioHbVg90H21EOLACvtfcHk0liShfCMV3o34Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslyqsr8%253D&md5=f12972109eb4daed651f1984ed30382d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00012%26sid%3Dliteratum%253Aachs%26aulast%3DBoskovic%26aufirst%3DZ.%2BV.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DFreedy%26aufirst%3DA.%2BM.%26aulast%3DViswanathan%26aufirst%3DV.%2BS.%26aulast%3DPop%26aufirst%3DM.%2BS.%26aulast%3DFuller%26aufirst%3DJ.%2BH.%26aulast%3DMartinez%26aufirst%3DN.%2BM.%26aulast%3DFigueroa%2BLazu%26aufirst%3DS.%2BO.%26aulast%3DHong%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DCalarese%26aufirst%3DD.%26aulast%3DLove%26aufirst%3DJ.%2BD.%26aulast%3DVetere%26aufirst%3DA.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DInhibition%2520of%2520zinc-dependent%2520histone%2520deacetylases%2520with%2520a%2520chemically%2520triggered%2520electrophile%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1844%26epage%3D1851%26doi%3D10.1021%2Facschembio.6b00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLean, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukasczyk, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fordham, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsay, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiling, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of covalent inhibitors for MIF tautomerase via cocrystal structures with phantom hits from virtual screening</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6717</span>– <span class="NLM_lpage">6720</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.09.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bmcl.2009.09.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19836948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6717-6720&author=L.+R.+McLeanauthor=Y.+Zhangauthor=H.+Liauthor=Z.+Liauthor=U.+Lukasczykauthor=Y.+M.+Choiauthor=Z.+Hanauthor=J.+Priscoauthor=J.+Fordhamauthor=J.+T.+Tsayauthor=S.+Reilingauthor=R.+J.+Vazauthor=Y.+Li&title=Discovery+of+covalent+inhibitors+for+MIF+tautomerase+via+cocrystal+structures+with+phantom+hits+from+virtual+screening&doi=10.1016%2Fj.bmcl.2009.09.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of covalent inhibitors for MIF tautomerase via cocrystal structures with phantom hits from virtual screening</span></div><div class="casAuthors">McLean, Larry R.; Zhang, Ying; Li, Hua; Li, Ziyu; Lukasczyk, Ulrike; Choi, Yong-Mi; Han, Zuoning; Prisco, Joy; Fordham, Jeremy; Tsay, Joseph T.; Reiling, Stephan; Vaz, Roy J.; Li, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6717-6720</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Biochem. and X-ray crystallog. studies confirmed that hydroxyquinoline derivs. identified by virtual screening were actually covalent inhibitors of the MIF tautomerase.  Adducts were formed by N-alkylation of the Pro-1 at the catalytic site with a loss of an amino group of the inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry3QrnMeARVbVg90H21EOLACvtfcHk0liShfCMV3o34Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisb7O&md5=899b261a6b7f85dbd93fdf32e0565ed1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.09.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.09.106%26sid%3Dliteratum%253Aachs%26aulast%3DMcLean%26aufirst%3DL.%2BR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLukasczyk%26aufirst%3DU.%26aulast%3DChoi%26aufirst%3DY.%2BM.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DPrisco%26aufirst%3DJ.%26aulast%3DFordham%26aufirst%3DJ.%26aulast%3DTsay%26aufirst%3DJ.%2BT.%26aulast%3DReiling%26aufirst%3DS.%26aulast%3DVaz%26aufirst%3DR.%2BJ.%26aulast%3DLi%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520covalent%2520inhibitors%2520for%2520MIF%2520tautomerase%2520via%2520cocrystal%2520structures%2520with%2520phantom%2520hits%2520from%2520virtual%2520screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6717%26epage%3D6720%26doi%3D10.1016%2Fj.bmcl.2009.09.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zapatero, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colmenarejo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Los Frailes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramon, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel inhibitors of the tautomerase activity of macrophage migration inhibitory factor (MIF)</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1177/1087057116633997</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1177%2F1087057116633997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26933127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=446-458&author=M.+C.+Zapateroauthor=P.+Perezauthor=M.+J.+Vazquezauthor=G.+Colmenarejoauthor=M.+de+Los+Frailesauthor=F.+Ramon&title=Discovery+of+novel+inhibitors+of+the+tautomerase+activity+of+macrophage+migration+inhibitory+factor+%28MIF%29&doi=10.1177%2F1087057116633997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel inhibitors of the tautomerase activity of macrophage migration inhibitory factor (MIF)</span></div><div class="casAuthors">Zapatero, Maria Cleofe; Perez, Paloma; Vazquez, Maria Jesus; Colmenarejo, Gonzalo; de los Frailes, Maite; Ramon, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">446-458</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Macrophage migration inhibitory factor (MIF) is a proinflammatory cytokine assocd. with multiple diseases, including neurodegenerative disorders.  With the ultimate goal of providing novel chemotypes as starting points for development of disease-modifying therapeutics for neurodegeneration, we endeavored to screen the GSK compd. collection for MIF inhibitors using a miniaturized, activity-based kinetic assay.  The assay monitors the increase in absorbance at 320 nm resulting from keto-to-enol tautomerization of 4-hydroxyphenylpyruvate, a reaction catalyzed by MIF.  We ran a full- diversity screen evaluating the inhibitory activity of 1.6 million compds.  Primary hits were confirmed and retested in an orthogonal assay measuring tautomerization of L-dopachrome Me ester by the decrease in absorbance at 475 nm in kinetic mode.  Selected compds. were progressed to medium-throughput mode-of-inhibition studies, which included time dependence, enzyme concn. dependence, and reversibility of their inhibitory effect.  With these results and after inspection of the physicochem. properties of compds., 17 chemotypes were prioritized and progressed to further stages of validation and characterization to better assess their therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5fx7eEUwg47Vg90H21EOLACvtfcHk0liShfCMV3o34Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWrsbnF&md5=7e256ec74bd95d9d6f8763f3134f41fd</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1087057116633997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057116633997%26sid%3Dliteratum%253Aachs%26aulast%3DZapatero%26aufirst%3DM.%2BC.%26aulast%3DPerez%26aufirst%3DP.%26aulast%3DVazquez%26aufirst%3DM.%2BJ.%26aulast%3DColmenarejo%26aufirst%3DG.%26aulast%3Dde%2BLos%2BFrailes%26aufirst%3DM.%26aulast%3DRamon%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520novel%2520inhibitors%2520of%2520the%2520tautomerase%2520activity%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2016%26volume%3D21%26spage%3D446%26epage%3D458%26doi%3D10.1177%2F1087057116633997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ouertatani-Sakouhi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Turk, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fauvet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinar
Karpinar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Roy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhagen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calandra, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweckstetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lashuel, H. A.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of novel classes of macrophage migration inhibitory factor (MIF) inhibitors with distinct mechanisms of action</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">26581</span>– <span class="NLM_lpage">26598</span>, <span class="refDoi"> DOI: 10.1074/jbc.M110.113951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M110.113951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=20516071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVahtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=26581-26598&author=H.+Ouertatani-Sakouhiauthor=F.+El-Turkauthor=B.+Fauvetauthor=M.+K.+Choauthor=D.+Pinar%0AKarpinarauthor=D.+Le+Royauthor=M.+Deworauthor=T.+Rogerauthor=J.+Bernhagenauthor=T.+Calandraauthor=M.+Zweckstetterauthor=H.+A.+Lashuel&title=Identification+and+characterization+of+novel+classes+of+macrophage+migration+inhibitory+factor+%28MIF%29+inhibitors+with+distinct+mechanisms+of+action&doi=10.1074%2Fjbc.M110.113951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Novel Classes of Macrophage Migration Inhibitory Factor (MIF) Inhibitors with Distinct Mechanisms of Action</span></div><div class="casAuthors">Ouertatani-Sakouhi, Hajer; El-Turk, Farah; Fauvet, Bruno; Cho, Min-Kyu; Pinar Karpinar, Damla; Le Roy, Didier; Dewor, Manfred; Roger, Thierry; Bernhagen, Juergen; Calandra, Thierry; Zweckstetter, Markus; Lashuel, Hilal A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">26581-26598</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Macrophage migration inhibitory factor (MIF), a proinflammatory cytokine, is considered an attractive therapeutic target in multiple inflammatory and autoimmune disorders.  In addn. to its known biol. activities, MIF can also function as a tautomerase.  Several small mols. have been reported to be effective inhibitors of MIF tautomerase activity in vitro.  Herein we employed a robust activity-based assay to identify different classes of novel inhibitors of the catalytic and biol. activities of MIF.  Several novel chem. classes of inhibitors of the catalytic activity of MIF with IC50 values in the range of 0.2-15.5 μM were identified and validated.  The interaction site and mechanism of action of these inhibitors were defined using structure-activity studies and a battery of biochem. and biophys. methods.  MIF inhibitors emerging from these studies could be divided into three categories based on their mechanism of action: (1) mols. that covalently modify the catalytic site at the N-terminal proline residue, Pro1; (2) a novel class of catalytic site inhibitors; and finally (3) mols. that disrupt the trimeric structure of MIF.  Importantly, all inhibitors demonstrated total inhibition of MIF-mediated glucocorticoid overriding and AKT phosphorylation, whereas ebselen, a trimer-disrupting inhibitor, addnl. acted as a potent hyperagonist in MIF-mediated chemotactic migration.  The identification of biol. active compds. with known toxicity, pharmacokinetic properties, and biol. activities in vivo should accelerate the development of clin. relevant MIF inhibitors.  Furthermore, the diversity of chem. structures and mechanisms of action of our inhibitors makes them ideal mechanistic probes for elucidating the structure-function relationships of MIF and to further det. the role of the oligomerization state and catalytic activity of MIF in regulating the function(s) of MIF in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqje3LpgpMop7Vg90H21EOLACvtfcHk0liSjHaoP5xeCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVahtbfK&md5=e25d6c06b2cf0b6454c00d23f9b7fbf4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.113951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.113951%26sid%3Dliteratum%253Aachs%26aulast%3DOuertatani-Sakouhi%26aufirst%3DH.%26aulast%3DEl-Turk%26aufirst%3DF.%26aulast%3DFauvet%26aufirst%3DB.%26aulast%3DCho%26aufirst%3DM.%2BK.%26aulast%3DPinar%2BKarpinar%26aufirst%3DD.%26aulast%3DLe%2BRoy%26aufirst%3DD.%26aulast%3DDewor%26aufirst%3DM.%26aulast%3DRoger%26aufirst%3DT.%26aulast%3DBernhagen%26aufirst%3DJ.%26aulast%3DCalandra%26aufirst%3DT.%26aulast%3DZweckstetter%26aufirst%3DM.%26aulast%3DLashuel%26aufirst%3DH.%2BA.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520novel%2520classes%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%2520%2528MIF%2529%2520inhibitors%2520with%2520distinct%2520mechanisms%2520of%2520action%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D26581%26epage%3D26598%26doi%3D10.1074%2Fjbc.M110.113951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valhondo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span> <span> </span><span class="NLM_article-title">Irregularities in enzyme assays: The case of macrophage migration inhibitory factor</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2764</span>– <span class="NLM_lpage">2767</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bmcl.2016.04.074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=27156768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2764-2767&author=J.+A.+Cisnerosauthor=M.+J.+Robertsonauthor=M.+Valhondoauthor=W.+L.+Jorgensen&title=Irregularities+in+enzyme+assays%3A+The+case+of+macrophage+migration+inhibitory+factor&doi=10.1016%2Fj.bmcl.2016.04.074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Irregularities in enzyme assays: The case of macrophage migration inhibitory factor</span></div><div class="casAuthors">Cisneros, Jose A.; Robertson, Michael J.; Valhondo, Margarita; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2764-2767</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Inhibitors of human macrophage migration inhibitory factor (MIF) previously reported in the literature have been reexamd. by synthesis, assaying for tautomerase activity, and protein crystallog.  Substantial inconsistencies between prior and current assay results are noted.  They appear to arise from difficulties with the tautomerase substrates, soly. issues, and esp. covalent inhibition.  Incubation time variation shows that several compds. are covalent or slow-binding inhibitors.  Two protein crystal structures are provided; one confirms that a twice-discovered compd. is a covalent inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgbtvPyvrjS7Vg90H21EOLACvtfcHk0liSjHaoP5xeCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCqtLs%253D&md5=a232a355d251396f4d19e79954bf469b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.074%26sid%3Dliteratum%253Aachs%26aulast%3DCisneros%26aufirst%3DJ.%2BA.%26aulast%3DRobertson%26aufirst%3DM.%2BJ.%26aulast%3DValhondo%26aufirst%3DM.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DIrregularities%2520in%2520enzyme%2520assays%253A%2520The%2520case%2520of%2520macrophage%2520migration%2520inhibitory%2520factor%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2764%26epage%3D2767%26doi%3D10.1016%2Fj.bmcl.2016.04.074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferchen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skibbe, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhold, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnovitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Targeted inhibition of STAT/TET1 axis as a therapeutic strategy for acute myeloid leukemia</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">2099</span> <span class="refDoi"> DOI: 10.1038/s41467-017-02290-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1038%2Fs41467-017-02290-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29235481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC1MzgvFKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=X.+Jiangauthor=C.+Huauthor=K.+Ferchenauthor=J.+Nieauthor=X.+Cuiauthor=C.+H.+Chenauthor=L.+Chengauthor=Z.+Zuoauthor=W.+Seibelauthor=C.+Heauthor=Y.+Tangauthor=J.+R.+Skibbeauthor=M.+Wunderlichauthor=W.+C.+Reinholdauthor=L.+Dongauthor=C.+Shenauthor=S.+Arnovitzauthor=B.+Ulrichauthor=J.+Luauthor=H.+Wengauthor=R.+Suauthor=H.+Huangauthor=Y.+Wangauthor=C.+Liauthor=X.+Qinauthor=J.+C.+Mulloyauthor=Y.+Zhengauthor=J.+Diaoauthor=J.+Jinauthor=C.+Liauthor=P.+P.+Liuauthor=C.+Heauthor=Y.+Chenauthor=J.+Chen&title=Targeted+inhibition+of+STAT%2FTET1+axis+as+a+therapeutic+strategy+for+acute+myeloid+leukemia&doi=10.1038%2Fs41467-017-02290-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted inhibition of STAT/TET1 axis as a therapeutic strategy for acute myeloid leukemia</span></div><div class="casAuthors">Jiang Xi; Hu Chao; Ferchen Kyle; Zuo Zhixiang; Skibbe Jennifer R; Dong Lei; Shen Chao; Lu Jiuwei; Weng Hengyou; Su Rui; Huang Huilin; Wang Yungui; Li Chenying; Qin Xi; Diao Jiajie; Chen Jianjun; Jiang Xi; Hu Chao; Arnovitz Stephen; Ulrich Bryan; Weng Hengyou; Wang Yungui; Chen Jianjun; Jiang Xi; Dong Lei; Shen Chao; Weng Hengyou; Su Rui; Huang Huilin; Li Chenying; Qin Xi; Chen Jianjun; Hu Chao; Wang Yungui; Li Chenying; Jin Jie; Nie Ji; Cui Xiaolong; He Chuan; Chen Chih-Hong; Chen Yuan; Cheng Liting; Tang Yixuan; Li Chong; Zuo Zhixiang; Seibel William; He Chunjiang; Wunderlich Mark; Mulloy James C; Zheng Yi; Reinhold William C; Liu Paul P</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2099</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Effective therapy of acute myeloid leukemia (AML) remains an unmet need.  DNA methylcytosine dioxygenase Ten-eleven translocation 1 (TET1) is a critical oncoprotein in AML.  Through a series of data analysis and drug screening, we identified two compounds (i.e., NSC-311068 and NSC-370284) that selectively suppress TET1 transcription and 5-hydroxymethylcytosine (5hmC) modification, and effectively inhibit cell viability in AML with high expression of TET1 (i.e., TET1-high AML), including AML carrying t(11q23)/MLL-rearrangements and t(8;21) AML.  NSC-311068 and especially NSC-370284 significantly repressed TET1-high AML progression in vivo.  UC-514321, a structural analog of NSC-370284, exhibited a more potent therapeutic effect and prolonged the median survival of TET1-high AML mice over three fold.  NSC-370284 and UC-514321 both directly target STAT3/5, transcriptional activators of TET1, and thus repress TET1 expression.  They also exhibit strong synergistic effects with standard chemotherapy.  Our results highlight the therapeutic potential of targeting the STAT/TET1 axis by selective inhibitors in AML treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRIMEzBUzrv5OR_TJajKxJDfW6udTcc2eaKr8wf-iHmzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzgvFKrtA%253D%253D&md5=968bc63017ea5738ed944ec0b702622c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02290-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02290-w%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DFerchen%26aufirst%3DK.%26aulast%3DNie%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DCheng%26aufirst%3DL.%26aulast%3DZuo%26aufirst%3DZ.%26aulast%3DSeibel%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DSkibbe%26aufirst%3DJ.%2BR.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DReinhold%26aufirst%3DW.%2BC.%26aulast%3DDong%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DArnovitz%26aufirst%3DS.%26aulast%3DUlrich%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DWeng%26aufirst%3DH.%26aulast%3DSu%26aufirst%3DR.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DP.%2BP.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DTargeted%2520inhibition%2520of%2520STAT%252FTET1%2520axis%2520as%2520a%2520therapeutic%2520strategy%2520for%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-02290-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavecchia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Giovanni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a novel small molecule inhibitor targeting the frataxin/ubiquitin interaction via structure-based virtual screening and bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2861</span>– <span class="NLM_lpage">2873</span>, <span class="refDoi"> DOI: 10.1021/jm3017199</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3017199" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFKgt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2861-2873&author=A.+Lavecchiaauthor=C.+Di+Giovanniauthor=C.+Cerchiaauthor=A.+Russoauthor=G.+Russoauthor=E.+Novellino&title=Discovery+of+a+novel+small+molecule+inhibitor+targeting+the+frataxin%2Fubiquitin+interaction+via+structure-based+virtual+screening+and+bioassays&doi=10.1021%2Fjm3017199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Small Molecule Inhibitor Targeting the Frataxin/Ubiquitin Interaction via Structure-Based Virtual Screening and Bioassays</span></div><div class="casAuthors">Lavecchia, Antonio; Di Giovanni, Carmen; Cerchia, Carmen; Russo, Annapina; Russo, Giulia; Novellino, Ettore</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2861-2873</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Friedreich's ataxia (FRDA) is an autosomal recessive neuro- and cardiodegenerative disorder for which there are no proven effective treatments.  FRDA is caused by decreased expression and/or function of the mitochondrial protein frataxin.  Here, we report findings that frataxin is degraded via the ubiquitin-proteasomal pathway and that it is ubiquitinated at residue K147 in Calu-6 cells.  A theor. model of the frataxin-K147/Ub complex, constructed by combining bioinformatics interface predictions with information-driven docking, revealed a hitherto unnoticed, potential ubiquitin-binding domain in frataxin.  Through structure-based virtual screening and cell-based assays, we discovered a novel small mol. (compd. (+)-11) able to prevent frataxin ubiquitination and degrdn. (+)-11 was synthesized and tested for specific binding to frataxin by an UF-LC/MS based ligand-binding assay.  Follow-up scaffold-based searches resulted in the identification of a lead series with micromolar activity in disrupting the frataxin/Ub interaction.  This study also suggests that frataxin could be a potential target for FRDA drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhNCbVlCSkMbVg90H21EOLACvtfcHk0lgYMZxuwemcIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFKgt7w%253D&md5=669158b27b2aa93345fd32563a85145c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm3017199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3017199%26sid%3Dliteratum%253Aachs%26aulast%3DLavecchia%26aufirst%3DA.%26aulast%3DDi%2BGiovanni%26aufirst%3DC.%26aulast%3DCerchia%26aufirst%3DC.%26aulast%3DRusso%26aufirst%3DA.%26aulast%3DRusso%26aufirst%3DG.%26aulast%3DNovellino%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520small%2520molecule%2520inhibitor%2520targeting%2520the%2520frataxin%252Fubiquitin%2520interaction%2520via%2520structure-based%2520virtual%2520screening%2520and%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2861%26epage%3D2873%26doi%3D10.1021%2Fjm3017199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thinnes, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scozzafava, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarhonskaya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walport, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkins, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pugh, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Betti reaction enables efficient synthesis of 8-hydroxyquinoline inhibitors of 2-oxoglutarate oxygenases</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">15458</span>– <span class="NLM_lpage">15461</span>, <span class="refDoi"> DOI: 10.1039/C5CC06095H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1039%2FC5CC06095H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26345662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWqu73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=15458-15461&author=C.+C.+Thinnesauthor=A.+Tumberauthor=C.+Yappauthor=G.+Scozzafavaauthor=T.+Yehauthor=M.+C.+Chanauthor=T.+A.+Tranauthor=K.+Hsuauthor=H.+Tarhonskayaauthor=L.+J.+Walportauthor=S.+E.+Wilkinsauthor=E.+D.+Martinezauthor=S.+Mullerauthor=C.+W.+Pughauthor=P.+J.+Ratcliffeauthor=P.+E.+Brennanauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Betti+reaction+enables+efficient+synthesis+of+8-hydroxyquinoline+inhibitors+of+2-oxoglutarate+oxygenases&doi=10.1039%2FC5CC06095H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Betti reaction enables efficient synthesis of 8-hydroxyquinoline inhibitors of 2-oxoglutarate oxygenases</span></div><div class="casAuthors">Thinnes, C. C.; Tumber, A.; Yapp, C.; Scozzafava, G.; Yeh, T.; Chan, M. C.; Tran, T. A.; Hsu, K.; Tarhonskaya, H.; Walport, L. J.; Wilkins, S. E.; Martinez, E. D.; Muller, S.; Pugh, C. W.; Ratcliffe, P. J.; Brennan, P. E.; Kawamura, A.; Schofield, C. J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">84</span>),
    <span class="NLM_cas:pages">15458-15461</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">There is interest in developing potent, selective, and cell-permeable inhibitors of human ferrous iron and 2-oxoglutarate (2OG) oxygenases for use in functional and target validation studies.  The 3-component Betti reaction enables efficient one-step C-7 functionalization of modified 8-hydroxyquinolines (8HQs) to produce cell-active inhibitors of KDM4 histone demethylases and other 2OG oxygenases; the work exemplifies how a template-based metallo-enzyme inhibitor approach can be used to give biol. active compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEdns6DiYknbVg90H21EOLACvtfcHk0lgYMZxuwemcIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWqu73P&md5=02d64e182c29f6270ef714389d27261d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC5CC06095H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC06095H%26sid%3Dliteratum%253Aachs%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DScozzafava%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BA.%26aulast%3DHsu%26aufirst%3DK.%26aulast%3DTarhonskaya%26aufirst%3DH.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DWilkins%26aufirst%3DS.%2BE.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPugh%26aufirst%3DC.%2BW.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DBetti%2520reaction%2520enables%2520efficient%2520synthesis%2520of%25208-hydroxyquinoline%2520inhibitors%2520of%25202-oxoglutarate%2520oxygenases%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D15458%26epage%3D15461%26doi%3D10.1039%2FC5CC06095H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olyaei, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadeghpour, M.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the synthesis and synthetic applications of Betti base (aminoalkylnaphthol) and bis-Betti base derivatives</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18467</span>– <span class="NLM_lpage">18497</span>, <span class="refDoi"> DOI: 10.1039/C9RA02813G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1039%2FC9RA02813G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGitbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=18467-18497&author=A.+Olyaeiauthor=M.+Sadeghpour&title=Recent+advances+in+the+synthesis+and+synthetic+applications+of+Betti+base+%28aminoalkylnaphthol%29+and+bis-Betti+base+derivatives&doi=10.1039%2FC9RA02813G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the synthesis and synthetic applications of Betti base (aminoalkylnaphthol) and bis-Betti base derivatives</span></div><div class="casAuthors">Olyaei, Abolfazl; Sadeghpour, Mahdieh</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">18467-18497</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The multicomponent reaction between 2-naphthol, arylaldehydes and ammonia yields aminobenzylnaphthols in a process known as the Betti reaction, which was first uncovered at the beginning of the 20th century.  Various methods have been reported for the synthesis of aminobenzylnaphthol (Betti base) and bis-Betti base derivs. using various types of naphthols, arom. amines, heteroarom. amines, and aliph. and cyclic amines instead of ammonia or diamines and aliph. and arom. aldehydes or dialdehyde compds. under various conditions in recent years.  The Betti reaction produces racemic and non-racemic aminobenzylnaphthol ligands.  It is also clear that the most important area of application of the non-racemic aminonaphthols prepd. in this manner is their use in asym. synthesis, either as chiral ligands or as chiral auxiliaries.  The functional groups in these Mannich products offer many ring closure possibilities.  Some of these products or the starting bifunctional compds. possess biol. activity.  Herein, we present a selection of the relevant studies on this topic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv4slvgnhlYLVg90H21EOLACvtfcHk0lhwL1tsoLSRow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGitbjE&md5=91c60f2b0271ca60bef664b6c456caf0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC9RA02813G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA02813G%26sid%3Dliteratum%253Aachs%26aulast%3DOlyaei%26aufirst%3DA.%26aulast%3DSadeghpour%26aufirst%3DM.%26atitle%3DRecent%2520advances%2520in%2520the%2520synthesis%2520and%2520synthetic%2520applications%2520of%2520Betti%2520base%2520%2528aminoalkylnaphthol%2529%2520and%2520bis-Betti%2520base%2520derivatives%26jtitle%3DRSC%2520Adv.%26date%3D2019%26volume%3D9%26spage%3D18467%26epage%3D18497%26doi%3D10.1039%2FC9RA02813G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y. G.</span></span> <span> </span><span class="NLM_article-title">C-H oxidation/Michael addition/cyclization cascade for enantioselective synthesis of functionalized 2-amino-4H-chromenes</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">6134</span>– <span class="NLM_lpage">6137</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b03148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b03148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2qsrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=6134-6137&author=B.+Wuauthor=X.+Gaoauthor=Z.+Yanauthor=M.+W.+Chenauthor=Y.+G.+Zhou&title=C-H+oxidation%2FMichael+addition%2Fcyclization+cascade+for+enantioselective+synthesis+of+functionalized+2-amino-4H-chromenes&doi=10.1021%2Facs.orglett.5b03148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">C-H Oxidation/Michael Addition/Cyclization Cascade for Enantioselective Synthesis of Functionalized 2-Amino-4H-chromenes</span></div><div class="casAuthors">Wu, Bo; Gao, Xiang; Yan, Zhong; Chen, Mu-Wang; Zhou, Yong-Gui</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6134-6137</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A streamlined method for the enantioselective synthesis of 2-amino-4H-chromenes from readily available 2-alkyl-substituted phenols and active methylene compds. bearing a cyano group with up to 97% ee is presented.  This reaction is a cascade procedure including manganese dioxide mediated C-H oxidn. for the generation of o-quinone methides and bifunctional squaramide-catalyzed Michael addn./cyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9lNopOZNPq7Vg90H21EOLACvtfcHk0lhwL1tsoLSRow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2qsrfE&md5=440afb3b780de4e03f7ab484d519805b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b03148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b03148%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DB.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DM.%2BW.%26aulast%3DZhou%26aufirst%3DY.%2BG.%26atitle%3DC-H%2520oxidation%252FMichael%2520addition%252Fcyclization%2520cascade%2520for%2520enantioselective%2520synthesis%2520of%2520functionalized%25202-amino-4H-chromenes%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D6134%26epage%3D6137%26doi%3D10.1021%2Facs.orglett.5b03148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casnati, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Predieri, G.</span></span> <span> </span><span class="NLM_article-title">Ortho-coordinated acylation of phenol systems</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">4371</span>– <span class="NLM_lpage">4377</span>, <span class="refDoi"> DOI: 10.1021/jo00301a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00301a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADyaK3cXksVSjur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1990&pages=4371-4377&author=G.+Sartoriauthor=G.+Casnatiauthor=F.+Bigiauthor=G.+Predieri&title=Ortho-coordinated+acylation+of+phenol+systems&doi=10.1021%2Fjo00301a031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Ortho-coordinated acylation of phenol systems</span></div><div class="casAuthors">Sartori, Giovanni; Casnati, Giuseppe; Bigi, Franca; Predieri, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4371-7</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The ortho-coordinated acylation of phenol salts reported in recent synthetic applications has been extensively investigated.  The data obtained support the hypothesis that formation of an organized complex between the phenol salts and the acylating agents may strongly influence the reaction pathway, depending on the nature of the specific cation, phenol, and acyl chloride involved in the process.  The structural factors which control the formation of the reacting complex have been extensively investigated; the results obtained allow us to discuss the possibilities and limitations of this methodol. in selective o-acylphenol synthesis.  The present methodol. is the procedure of choice for ortho-functionalization of phenols with electrophilic reagents such as phosgene, oxalyl chlorides, polyunsatd. acid chlorides, phthalic dichlorides, and, in general, acid chlorides α-functionalized with an electron-withdrawing group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTXi-I_dtRHbVg90H21EOLACvtfcHk0lhwL1tsoLSRow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXksVSjur0%253D&md5=e956bf5a8f779047daa70a3aac9c7187</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjo00301a031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00301a031%26sid%3Dliteratum%253Aachs%26aulast%3DSartori%26aufirst%3DG.%26aulast%3DCasnati%26aufirst%3DG.%26aulast%3DBigi%26aufirst%3DF.%26aulast%3DPredieri%26aufirst%3DG.%26atitle%3DOrtho-coordinated%2520acylation%2520of%2520phenol%2520systems%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1990%26volume%3D55%26spage%3D4371%26epage%3D4377%26doi%3D10.1021%2Fjo00301a031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shergalis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankhead, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Inhibition of protein disulfide isomerase in glioblastoma causes marked downregulation of DNA repair and DNA damage response genes</span>. <i>Theranostics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2282</span>– <span class="NLM_lpage">2298</span>, <span class="refDoi"> DOI: 10.7150/thno.30621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.7150%2Fthno.30621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=31149044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFKqtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=2282-2298&author=S.+Xuauthor=Y.+Liuauthor=K.+Yangauthor=H.+Wangauthor=A.+Shergalisauthor=A.+Kyaniauthor=A.+Bankheadauthor=S.+Tamuraauthor=S.+Yangauthor=X.+Wangauthor=C.+C.+Wangauthor=A.+Rehemtullaauthor=M.+Ljungmanauthor=N.+Neamati&title=Inhibition+of+protein+disulfide+isomerase+in+glioblastoma+causes+marked+downregulation+of+DNA+repair+and+DNA+damage+response+genes&doi=10.7150%2Fthno.30621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of protein disulfide isomerase in glioblastoma causes marked downregulation of DNA repair and DNA damage response genes</span></div><div class="casAuthors">Xu, Shili; Liu, Yajing; Yang, Kai; Wang, Hanxiao; Shergalis, Andrea; Kyani, Anahita; Bankhead, Armand., III; Tamura, Shuzo; Yang, Suhui; Wang, Xi; Wang, Chih-Chen; Rehemtulla, Alnawaz; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Theranostics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2282-2298</span>CODEN:
                <span class="NLM_cas:coden">THERDS</span>;
        ISSN:<span class="NLM_cas:issn">1838-7640</span>.
    
            (<span class="NLM_cas:orgname">Ivyspring International Publisher</span>)
        </div><div class="casAbstract">Aberrant overexpression of endoplasmic reticulum (ER)-resident oxidoreductase protein disulfide isomerase (PDI) plays an important role in cancer progression.  In this study, we demonstrate that PDI promotes glioblastoma (GBM) cell growth and describe a class of allosteric PDI inhibitors that are selective for PDI over other PDI family members.  Methods: We performed a phenotypic screening triage campaign of over 20,000 diverse compds. to identify PDI inhibitors cytotoxic to cancer cells.  From this screen, BAP2 emerged as a lead compd., and we assessed BAP2-PDI interactions with gel filtration, thiol-competition assays, and site-directed mutagenesis studies.  To assess selectivity, we compared BAP2 activity across several PDI family members in the PDI reductase assay.  Finally, we performed in vivo studies with a mouse xenograft model of GBM combining BAP2 and the std. of care (temozolomide and radiation), and identified affected gene pathways with nascent RNA sequencing (Bru-seq).  Results:BAP2 and related analogs are novel PDI inhibitors that selectively inhibit PDIA1 and PDIp.  Though BAP2 contains a weak Michael acceptor, interaction with PDI relies on Histidine 256 in the b' domain of PDI, suggesting allosteric binding.  Furthermore, both in vitro and in vivo, BAP2 reduces cell and tumor growth.  BAP2 alters the transcription of genes involved in the unfolded protein response, ER stress, apoptosis and DNA repair response.  Conclusion: These results indicate that BAP2 has anti-tumor activity and the suppressive effect on DNA repair gene expression warrants combination with DNA damaging agents to treat GBM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvxA-gjkJrT7Vg90H21EOLACvtfcHk0liIuaZ9GoB5rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFKqtLfL&md5=084ba1e1a0f4294573c01c9c4f194cc1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.7150%2Fthno.30621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fthno.30621%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DShergalis%26aufirst%3DA.%26aulast%3DKyani%26aufirst%3DA.%26aulast%3DBankhead%26aufirst%3DA.%26aulast%3DTamura%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%2BC.%26aulast%3DRehemtulla%26aufirst%3DA.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DInhibition%2520of%2520protein%2520disulfide%2520isomerase%2520in%2520glioblastoma%2520causes%2520marked%2520downregulation%2520of%2520DNA%2520repair%2520and%2520DNA%2520damage%2520response%2520genes%26jtitle%3DTheranostics%26date%3D2019%26volume%3D9%26spage%3D2282%26epage%3D2298%26doi%3D10.7150%2Fthno.30621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madonna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beclin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laras, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcowycz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamoral-Theys, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barthelemy-Requin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenglet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cresteil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnier, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, J. L.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships and mechanism of action of antitumor bis 8-hydroxyquinoline substituted benzylamines</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">638</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2009.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.ejmech.2009.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19931949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2010&pages=623-638&author=S.+Madonnaauthor=C.+Beclinauthor=Y.+Larasauthor=V.+Moretauthor=A.+Marcowyczauthor=D.+Lamoral-Theysauthor=J.+Duboisauthor=M.+Barthelemy-Requinauthor=G.+Lengletauthor=S.+Depauwauthor=T.+Cresteilauthor=G.+Aubertauthor=V.+Monnierauthor=R.+Kissauthor=M.+H.+David-Cordonnierauthor=J.+L.+Kraus&title=Structure-activity+relationships+and+mechanism+of+action+of+antitumor+bis+8-hydroxyquinoline+substituted+benzylamines&doi=10.1016%2Fj.ejmech.2009.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships and mechanism of action of antitumor bis 8-hydroxyquinoline substituted benzylamines</span></div><div class="casAuthors">Madonna, Sebastien; Beclin, Christophe; Laras, Younes; Moret, Vincent; Marcowycz, Aline; Lamoral-Theys, Delphine; Dubois, Jacques; Barthelemy-Requin, Magali; Lenglet, Gaelle; Depauw, Sabine; Cresteil, Thierry; Aubert, Genevieve; Monnier, Valerie; Kiss, Robert; David-Cordonnier, Marie-Helene; Kraus, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">623-638</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of twenty six 8-hydroxyquinoline-substituted amines, e.g. I (R = Ph, 4-F3COC6H4, 3-IC6H4, 2-thienyl, 2-pyridylmethyl, etc.), structurally related to compds. I (R = 4-MeC6H4, 4-F3CC6H4), were synthesized to evaluate the effects of structural changes on antitumor activity and understand their mechanism of action.  The studies were performed on a wide variety of cancer cell lines within glioma and carcinoma models.  The results obtained from chem. models and biol. techniques such as microarrays suggest the following hypothesis that a quinone methide intermediate which does not react with DNA but which gives covalent protein thiol adducts.  Micro-array anal. showed that the drugs induce the expression of a variety of stress related genes responsible for the cytotoxic and cytostatic effects in carcinoma and glioblastoma cells resp.  The described analogs could represent new promising anti-cancer candidates with specific action mechanisms, targeting accessible thiols from specific proteins and inducing potent anti-cancer effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS_V5g0nY2A7Vg90H21EOLACvtfcHk0liIuaZ9GoB5rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1SgsQ%253D%253D&md5=ce938a7a00b47560c985eb7f03bbc955</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2009.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2009.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DMadonna%26aufirst%3DS.%26aulast%3DBeclin%26aufirst%3DC.%26aulast%3DLaras%26aufirst%3DY.%26aulast%3DMoret%26aufirst%3DV.%26aulast%3DMarcowycz%26aufirst%3DA.%26aulast%3DLamoral-Theys%26aufirst%3DD.%26aulast%3DDubois%26aufirst%3DJ.%26aulast%3DBarthelemy-Requin%26aufirst%3DM.%26aulast%3DLenglet%26aufirst%3DG.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DCresteil%26aufirst%3DT.%26aulast%3DAubert%26aufirst%3DG.%26aulast%3DMonnier%26aufirst%3DV.%26aulast%3DKiss%26aufirst%3DR.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.%2BH.%26aulast%3DKraus%26aufirst%3DJ.%2BL.%26atitle%3DStructure-activity%2520relationships%2520and%2520mechanism%2520of%2520action%2520of%2520antitumor%2520bis%25208-hydroxyquinoline%2520substituted%2520benzylamines%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2010%26volume%3D45%26spage%3D623%26epage%3D638%26doi%3D10.1016%2Fj.ejmech.2009.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bekendam, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendapudi, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nag, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldenzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogg, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaumenhaft, R.</span></span> <span> </span><span class="NLM_article-title">A substrate-driven allosteric switch that enhances PDI catalytic activity</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">12579</span> <span class="refDoi"> DOI: 10.1038/ncomms12579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1038%2Fncomms12579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=27573496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=R.+H.+Bekendamauthor=P.+K.+Bendapudiauthor=L.+Linauthor=P.+P.+Nagauthor=J.+Puauthor=D.+R.+Kennedyauthor=A.+Feldenzerauthor=J.+Chiuauthor=K.+M.+Cookauthor=B.+Furieauthor=M.+Huangauthor=P.+J.+Hoggauthor=R.+Flaumenhaft&title=A+substrate-driven+allosteric+switch+that+enhances+PDI+catalytic+activity&doi=10.1038%2Fncomms12579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A substrate-driven allosteric switch that enhances PDI catalytic activity</span></div><div class="casAuthors">Bekendam, Roelof H.; Bendapudi, Pavan K.; Lin, Lin; Nag, Partha P.; Pu, Jun; Kennedy, Daniel R.; Feldenzer, Alexandra; Chiu, Joyce; Cook, Kristina M.; Furie, Bruce; Huang, Mingdong; Hogg, Philip J.; Flaumenhaft, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12579</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI) is an oxidoreductase essential for folding proteins in the endoplasmic reticulum.  The domain structure of PDI is a-b-b'-x-a', wherein the thioredoxin-like a and a' domains mediate disulfide bond shuffling and b and b' domains are substrate binding.  The b' and a' domains are connected via the x-linker, a 19-amino-acid flexible peptide.  Here we identify a class of compds., termed bepristats, that target the substrate-binding pocket of b'.  Bepristats reversibly block substrate binding and inhibit platelet aggregation and thrombus formation in vivo.  Ligation of the substrate-binding pocket by bepristats paradoxically enhances catalytic activity of a and a' by displacing the x-linker, which acts as an allosteric switch to augment reductase activity in the catalytic domains.  This substrate-driven allosteric switch is also activated by peptides and proteins and is present in other thiol isomerases.  Our results demonstrate a mechanism whereby binding of a substrate to thiol isomerases enhances catalytic activity of remote domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkUYTT5B9w97Vg90H21EOLACvtfcHk0liIuaZ9GoB5rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKjsbjL&md5=a138a269279f7bf35c2adc80332facad</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncomms12579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12579%26sid%3Dliteratum%253Aachs%26aulast%3DBekendam%26aufirst%3DR.%2BH.%26aulast%3DBendapudi%26aufirst%3DP.%2BK.%26aulast%3DLin%26aufirst%3DL.%26aulast%3DNag%26aufirst%3DP.%2BP.%26aulast%3DPu%26aufirst%3DJ.%26aulast%3DKennedy%26aufirst%3DD.%2BR.%26aulast%3DFeldenzer%26aufirst%3DA.%26aulast%3DChiu%26aufirst%3DJ.%26aulast%3DCook%26aufirst%3DK.%2BM.%26aulast%3DFurie%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DHogg%26aufirst%3DP.%2BJ.%26aulast%3DFlaumenhaft%26aufirst%3DR.%26atitle%3DA%2520substrate-driven%2520allosteric%2520switch%2520that%2520enhances%2520PDI%2520catalytic%2520activity%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms12579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Go, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D. P.</span></span> <span> </span><span class="NLM_article-title">Redox compartmentalization in eukaryotic cells</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1780</i></span>,  <span class="NLM_fpage">1273</span>– <span class="NLM_lpage">1290</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2008.01.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bbagen.2008.01.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=18267127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVarsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1780&publication_year=2008&pages=1273-1290&author=Y.+M.+Goauthor=D.+P.+Jones&title=Redox+compartmentalization+in+eukaryotic+cells&doi=10.1016%2Fj.bbagen.2008.01.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Redox compartmentalization in eukaryotic cells</span></div><div class="casAuthors">Go, Young-Mi; Jones, Dean P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1780</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1273-1290</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Diverse functions of eukaryotic cells are optimized by organization of compatible chemistries into distinct compartments defined by the structures of lipid-contg. membranes, multiprotein complexes and oligomeric structures of saccharides and nucleic acids.  This structural and chem. organization is coordinated, in part, through cysteine residues of proteins which undergo reversible oxidn.-redn. and serve as chem./structural transducing elements.  The central thiol/disulfide redox couples, thioredoxin-1, thioredoxin-2, GSH/GSSG and cysteine/cystine (Cys/CySS), are not in equil. with each other and are maintained at distinct, non-equil. potentials in mitochondria, nuclei, the secretory pathway and the extracellular space.  Mitochondria contain the most reducing compartment, have the highest rates of electron transfer and are highly sensitive to oxidn.  Nuclei also have more reduced redox potentials but are relatively resistant to oxidn.  The secretory pathway contains oxidative systems which introduce disulfides into proteins for export.  The cytoplasm contains few metabolic oxidases and this maintains an environment for redox signaling dependent upon NADPH oxidases and NO synthases.  Extracellular compartments are maintained at stable oxidizing potentials.  Controlled changes in cytoplasmic GSH/GSSG redox potential are assocd. with functional state, varying with proliferation, differentiation and apoptosis.  Variation in extracellular Cys/CySS redox potential is also assocd. with proliferation, cell adhesion and apoptosis.  Thus, cellular redox biol. is inseparable from redox compartmentalization.  Further elucidation of the redox control networks within compartments will improve the mechanistic understanding of cell functions and their disruption in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7q9nSxTPBAbVg90H21EOLACvtfcHk0lgusZolAQyLyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVarsbfJ&md5=5a772255f00733cd606e3d3c47d57728</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2008.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2008.01.011%26sid%3Dliteratum%253Aachs%26aulast%3DGo%26aufirst%3DY.%2BM.%26aulast%3DJones%26aufirst%3DD.%2BP.%26atitle%3DRedox%2520compartmentalization%2520in%2520eukaryotic%2520cells%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2008%26volume%3D1780%26spage%3D1273%26epage%3D1290%26doi%3D10.1016%2Fj.bbagen.2008.01.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walker, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, H. F.</span></span> <span> </span><span class="NLM_article-title">Oxidation of kinetically trapped thiols by protein disulfide isomerase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">13642</span>– <span class="NLM_lpage">13650</span>, <span class="refDoi"> DOI: 10.1021/bi00041a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00041a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADyaK2MXot1ylu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=13642-13650&author=K.+W.+Walkerauthor=H.+F.+Gilbert&title=Oxidation+of+kinetically+trapped+thiols+by+protein+disulfide+isomerase&doi=10.1021%2Fbi00041a045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Oxidation of Kinetically Trapped Thiols by Protein Disulfide Isomerase</span></div><div class="casAuthors">Walker, Kenneth W.; Gilbert, Hiram F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">13642-50</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The formation of a stabilized structure during oxidative protein folding can severely retard disulfide formation if the structure must be disrupted to gain access to buried cysteines.  These kinetic traps can slow protein folding and disulfide bond formation to the extent that unassisted folding is too slow to be kinetically competent in the cell.  Protein disulfide isomerase (PDI) facilitates the oxidn. of a kinetically trapped state of RTEM-1 β-lactamase in which two cysteines that form the single disulfide bond in the native protein are buried and approx. 500-fold less reactive than exposed cysteines.  Under second-order conditions, PDI-dependent oxidn. of reduced, folded β-lactamase is 500-fold faster than GSSG-dependent oxidn.  The rate difference obsd. between PDI and GSSG can be accounted for by the 520-fold higher kinetic reactivity of PDI as an oxidant.  Noncovalent interactions between PDI (35 μM) and β-lactamase increase the reactivity or unfolding of β-lactamase in the steady-state by less than 3-fold.  At high concns. of PDI or alkylating agents, the reaction of β-lactamase cysteines approaches a const. rate, limited by the spontaneous unfolding of the protein (kunfold = 0.024 min-1).  PDI does not substantially increase the rate of β-lactamase unfolding; however, once β-lactamase spontaneously unfolds, PDI at concns. greater than 44 μM, oxidizes the unfolded substrate before it can refold (kfold = 1.5 min-1).  PDI also facilitates the glutathione disulfide-dependent oxidn. of β-lactamase by approx. 1.7-3-fold, even at satg. GSSG concns.; however, oxidn. rates never exceed the rate of spontaneous β-lactamase unfolding.  The high kinetic reactivity of PDI as an oxidant and the almost millimolar concns. of PDI found in the endoplasmic reticulum provide one mechanism to avoid kinetic traps by making oxidn. kinetically competitive with the formation of kinetic traps.  In addn., PDI's high reactivity and its noncovalent interactions with unfolded proteins may minimize the significance of even relatively stable kinetic traps, as long as unfolding of the kinetic trap does not become rate-limiting.  Chaperones, including the hsp70 chaperone of the endoplasmic reticulum, BiP, and the Escherichia coli chaperonin, GroEL, do not appear to increase the rate of β-lactamase unfolding.  This suggests that the unfolding rate may help define those structures that the endoplasmic reticulum quality control system recognizes as native.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTv3EXMh_GpLVg90H21EOLACvtfcHk0lgusZolAQyLyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXot1ylu7o%253D&md5=859635a24742cb442ab46dfd7827bc27</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fbi00041a045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00041a045%26sid%3Dliteratum%253Aachs%26aulast%3DWalker%26aufirst%3DK.%2BW.%26aulast%3DGilbert%26aufirst%3DH.%2BF.%26atitle%3DOxidation%2520of%2520kinetically%2520trapped%2520thiols%2520by%2520protein%2520disulfide%2520isomerase%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D13642%26epage%3D13650%26doi%3D10.1021%2Fbi00041a045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veloso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span> <span> </span><span class="NLM_article-title">Use of Bru-Seq and BruChase-Seq for genome-wide assessment of the synthesis and stability of RNA</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2013.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.ymeth.2013.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=23973811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFeksbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2014&pages=45-54&author=M.+T.+Paulsenauthor=A.+Velosoauthor=J.+Prasadauthor=K.+Bediauthor=E.+A.+Ljungmanauthor=B.+Magnusonauthor=T.+E.+Wilsonauthor=M.+Ljungman&title=Use+of+Bru-Seq+and+BruChase-Seq+for+genome-wide+assessment+of+the+synthesis+and+stability+of+RNA&doi=10.1016%2Fj.ymeth.2013.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Bru-Seq and BruChase-Seq for genome-wide assessment of the synthesis and stability of RNA</span></div><div class="casAuthors">Paulsen, Michelle T.; Veloso, Artur; Prasad, Jayendra; Bedi, Karan; Ljungman, Emily A.; Magnuson, Brian; Wilson, Thomas E.; Ljungman, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-54</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Gene expression studies commonly examine total cellular RNA, which only provides information about its steady-state pool of RNA.  It remains unclear whether differences in the steady-state reflects variable rates of transcription or RNA degrdn.  To specifically monitor RNA synthesis and degrdn. genome-wide, we developed Bru-Seq and BruChase-Seq.  These assays are based on metabolic pulse-chase labeling of RNA using bromouridine (Bru).  In Bru-Seq, recently labeled RNAs are sequenced to reveal spans of nascent transcription in the genome.  In BruChase-Seq, cells are chased in uridine for different periods of time following Bru-labeling, allowing for the isolation of RNA populations of specific ages.  Here we describe these methodologies in detail and highlight their usefulness in assessing RNA synthesis and stability as well as splicing kinetics with examples of specific genes from different human cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkhkyBLXrRWrVg90H21EOLACvtfcHk0lgusZolAQyLyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFeksbvN&md5=a18ef08d32830fe03bf5024aa7246fb9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2013.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2013.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DM.%2BT.%26aulast%3DVeloso%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DJ.%26aulast%3DBedi%26aufirst%3DK.%26aulast%3DLjungman%26aufirst%3DE.%2BA.%26aulast%3DMagnuson%26aufirst%3DB.%26aulast%3DWilson%26aufirst%3DT.%2BE.%26aulast%3DLjungman%26aufirst%3DM.%26atitle%3DUse%2520of%2520Bru-Seq%2520and%2520BruChase-Seq%2520for%2520genome-wide%2520assessment%2520of%2520the%2520synthesis%2520and%2520stability%2520of%2520RNA%26jtitle%3DMethods%26date%3D2014%26volume%3D67%26spage%3D45%26epage%3D54%26doi%3D10.1016%2Fj.ymeth.2013.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groenendyk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, M.</span></span> <span> </span><span class="NLM_article-title">Calreticulin signaling in health and disease</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">842</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2012.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.biocel.2012.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=22373697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Glsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=842-846&author=W.+A.+Wangauthor=J.+Groenendykauthor=M.+Michalak&title=Calreticulin+signaling+in+health+and+disease&doi=10.1016%2Fj.biocel.2012.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Calreticulin signaling in health and disease</span></div><div class="casAuthors">Wang, Wen-An; Groenendyk, Jody; Michalak, Marek</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">842-846</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Calreticulin is an endoplasmic reticulum Ca2+ binding chaperone that has multiple functions inside and outside of the endoplasmic reticulum.  It is involved in the quality control of newly synthesized proteins and glycoproteins, interacting with various other endoplasmic reticulum chaperones, specifically calnexin and ER protein of 57-kDa in the calreticulin/calnexin cycle.  Calreticulin also plays a crucial role in regulating intracellular Ca2+ homeostasis, assocg. calreticulin with a wide variety of signaling processes, such as cardiogenesis, adipocyte differentiation and cellular stress responses.  The role of calreticulin outside of the endoplasmic reticulum is also extensive, including functions in wound healing and immunity.  Therefore, calreticulin has important implications in health and disease.Signaling factsCalreticulin controls calcineurin activity and NF-AT- and MEF2C-dependent expression of cardiac genes through InsP3R mediated ER Ca2+ release.Calreticulin deficiency up-regulates the CAMKII-dependent signaling pathway.Calreticulin over-expression decreases adipogenic factors (lipoprotein lipase, αP2, PPAR and C/EBPα).Cell surface calreticulin signals pre-apoptotic cellular clearance by binding to LDL-receptor-related protein (LRP) on phagocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvqHSSZettOrVg90H21EOLACvtfcHk0lg6XQbxPuKPyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Glsb0%253D&md5=3cd5893f9e77b4d9d408b0198c1d34f0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2012.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2012.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%2BA.%26aulast%3DGroenendyk%26aufirst%3DJ.%26aulast%3DMichalak%26aufirst%3DM.%26atitle%3DCalreticulin%2520signaling%2520in%2520health%2520and%2520disease%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2012%26volume%3D44%26spage%3D842%26epage%3D846%26doi%3D10.1016%2Fj.biocel.2012.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jessop, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavender, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulleid, N. J.</span></span> <span> </span><span class="NLM_article-title">Substrate specificity of the oxidoreductase ERp57 is determined primarily by its interaction with calnexin and calreticulin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">2194</span>– <span class="NLM_lpage">2202</span>, <span class="refDoi"> DOI: 10.1074/jbc.M808054200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M808054200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19054761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFaqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=2194-2202&author=C.+E.+Jessopauthor=T.+J.+Tavenderauthor=R.+H.+Watkinsauthor=J.+E.+Chambersauthor=N.+J.+Bulleid&title=Substrate+specificity+of+the+oxidoreductase+ERp57+is+determined+primarily+by+its+interaction+with+calnexin+and+calreticulin&doi=10.1074%2Fjbc.M808054200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Specificity of the Oxidoreductase ERp57 Is Determined Primarily by Its Interaction with Calnexin and Calreticulin</span></div><div class="casAuthors">Jessop, Catherine E.; Tavender, Timothy J.; Watkins, Rachel H.; Chambers, Joseph E.; Bulleid, Neil J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2194-2202</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The formation of disulfides within proteins entering the secretory pathway is catalyzed by the protein disulfide isomerase family of endoplasmic reticulum localized oxidoreductases.  One such enzyme, ERp57, is thought to catalyze the isomerization of non-native disulfide bonds formed in glycoproteins with unstructured disulfide-rich domains.  Here we investigated the mechanism underlying ERp57 specificity toward glycoprotein substrates and the interdependence of ERp57 and the calnexin cycle for their correct folding.  Our results clearly show that ERp57 must be phys. assocd. with the calnexin cycle to catalyze isomerization reactions with most of its substrates.  In addn., some glycoproteins only require ERp57 for correct disulfide formation if they enter the calnexin cycle.  Hence, the specificity of ER oxidoreductases is not only detd. by the phys. assocn. of enzyme and substrate but also by accessory factors, such as calnexin and calreticulin in the case of ERp57.  These conclusions suggest that the calnexin cycle has evolved with a specialized oxidoreductase to facilitate native disulfide formation in complex glycoproteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiwiNUERR-Z7Vg90H21EOLACvtfcHk0lg6XQbxPuKPyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFaqtA%253D%253D&md5=781f153089970a97fa54bf101c381e13</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M808054200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M808054200%26sid%3Dliteratum%253Aachs%26aulast%3DJessop%26aufirst%3DC.%2BE.%26aulast%3DTavender%26aufirst%3DT.%2BJ.%26aulast%3DWatkins%26aufirst%3DR.%2BH.%26aulast%3DChambers%26aufirst%3DJ.%2BE.%26aulast%3DBulleid%26aufirst%3DN.%2BJ.%26atitle%3DSubstrate%2520specificity%2520of%2520the%2520oxidoreductase%2520ERp57%2520is%2520determined%2520primarily%2520by%2520its%2520interaction%2520with%2520calnexin%2520and%2520calreticulin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D2194%26epage%3D2202%26doi%3D10.1074%2Fjbc.M808054200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. S.</span></span> <span> </span><span class="NLM_article-title">Role of the unfolded protein response regulator GRP78/BiP in development, cancer, and neurological disorders</span>. <i>Antioxid. Redox Signaling</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2307</span>– <span class="NLM_lpage">2316</span>, <span class="refDoi"> DOI: 10.1089/ars.2009.2485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1089%2Fars.2009.2485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=19309259" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFaru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=2307-2316&author=M.+Wangauthor=S.+Weyauthor=Y.+Zhangauthor=R.+Yeauthor=A.+S.+Lee&title=Role+of+the+unfolded+protein+response+regulator+GRP78%2FBiP+in+development%2C+cancer%2C+and+neurological+disorders&doi=10.1089%2Fars.2009.2485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Unfolded Protein Response Regulator GRP78/BiP in Development, Cancer, and Neurological Disorders</span></div><div class="casAuthors">Wang, Miao; Wey, Shiuan; Zhang, Yi; Ye, Risheng; Lee, Amy S.</div><div class="citationInfo"><span class="NLM_cas:title">Antioxidants & Redox Signaling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2307-2316</span>CODEN:
                <span class="NLM_cas:coden">ARSIF2</span>;
        ISSN:<span class="NLM_cas:issn">1523-0864</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  GRP78/BiP is a major endoplasmic reticulum (ER) chaperone protein crit. for protein quality control of the ER, as well as controlling the activation of the ER-transmembrane signaling mols.  Through creation of mouse models targeting the Grp78 allele, the function of GRP78 in development and disease has been investigated.  These led to the discovery that GRP78 function is obligatory for early embryonic development.  However, in adult animals, GRP78 is preferably required for cancer cell survival under pathol. conditions such as tumor progression and drug resistance.  The discovery of surface localization of GRP78 in cancer cells reveals potential novel function, interaction with cell-surface receptors, and possible therapeutic implications.  Mouse models also reveal that GRP78 controls maturation and secretion of neuronal factors for proper neural migration and offers neuroprotection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozuyAwoRxPp7Vg90H21EOLACvtfcHk0lg6XQbxPuKPyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFaru70%253D&md5=6b06910b584fd9c164a1c415aec04ac5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1089%2Fars.2009.2485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fars.2009.2485%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWey%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DA.%2BS.%26atitle%3DRole%2520of%2520the%2520unfolded%2520protein%2520response%2520regulator%2520GRP78%252FBiP%2520in%2520development%252C%2520cancer%252C%2520and%2520neurological%2520disorders%26jtitle%3DAntioxid.%2520Redox%2520Signaling%26date%3D2009%26volume%3D11%26spage%3D2307%26epage%3D2316%26doi%3D10.1089%2Fars.2009.2485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. J.</span></span> <span> </span><span class="NLM_article-title">Implications of NQO1 in cancer therapy</span>. <i>BMB Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.5483/BMBRep.2015.48.11.190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.5483%2FBMBRep.2015.48.11.190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26424559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvF2gsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2015&pages=609-617&author=E.+T.+Ohauthor=H.+J.+Park&title=Implications+of+NQO1+in+cancer+therapy&doi=10.5483%2FBMBRep.2015.48.11.190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Implications of NQO1 in cancer therapy</span></div><div class="casAuthors">Oh, Eun-Taex; Park, Heon Joo</div><div class="citationInfo"><span class="NLM_cas:title">BMB Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">609-617</span>CODEN:
                <span class="NLM_cas:coden">BRMEC2</span>;
        ISSN:<span class="NLM_cas:issn">1976-670X</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">NAD(P)H:quinone oxidoreductase (NQO1), an obligatory two-electron reductase, is a ubiquitous cytosolic enzyme that catalyzes the redn. of quinone substrates.  The NQO1- mediated two-electron redn. of quinones can be either chemoprotection/detoxification or a chemotherapeutic response, depending on the target quinones.  When toxic quinones are reduced by NQO1, they are conjugated with glutathione or glucuronic acid and excreted from the cells.  Based on this protective effect of NQO1, the use of dietary compds. to induce the expression of NQO1 has emerged as a promising strategy for cancer prevention.  On the other hand, NQO1-mediated two-electron redn. converts certain quinone compds. (such as mitomycin C, E09, RH1 and .ovrddot.ic-lapachone) to cytotoxic agents, leading to cell death.  It has been known that NQO1 is expressed at high levels in numerous human cancers, including breast, colon, cervix, lung, and pancreas, as compared with normal tissues.  This implies that tumors can be preferentially damaged relative to normal tissue by cytotoxic quinone drugs.  Importantly, NQO1 has been shown to stabilize many proteins, including p53 and p33ING1b, by inhibiting their proteasomal degrdn.  This review will summarize the biol. roles of NQO1 in cancer, with emphasis on recent findings and the potential of NQO1 as a therapeutic target for the cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnUeEnF9n8Y7Vg90H21EOLACvtfcHk0lj9BnJGKfCqcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvF2gsbo%253D&md5=0291a80078ec971717eef27d267feb49</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.5483%2FBMBRep.2015.48.11.190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5483%252FBMBRep.2015.48.11.190%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DE.%2BT.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DImplications%2520of%2520NQO1%2520in%2520cancer%2520therapy%26jtitle%3DBMB%2520Rep.%26date%3D2015%26volume%3D48%26spage%3D609%26epage%3D617%26doi%3D10.5483%2FBMBRep.2015.48.11.190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koppula, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, B.</span></span> <span> </span><span class="NLM_article-title">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span>. <i>Cancer Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">12</span>, <span class="refDoi"> DOI: 10.1186/s40880-018-0288-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1186%2Fs40880-018-0288-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=12&author=P.+Koppulaauthor=Y.+Zhangauthor=L.+Zhuangauthor=B.+Gan&title=Amino+acid+transporter+SLC7A11%2FxCT+at+the+crossroads+of+regulating+redox+homeostasis+and+nutrient+dependency+of+cancer&doi=10.1186%2Fs40880-018-0288-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid transporter SLC7A11/xCT at the crossroads of regulating redox homeostasis and nutrient dependency of cancer</span></div><div class="casAuthors">Koppula Pranavi; Zhang Yilei; Zhuang Li; Gan Boyi; Koppula Pranavi; Gan Boyi; Gan Boyi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer communications (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells often upregulate nutrient transporters to fulfill their increased biosynthetic and bioenergetic needs, and to maintain redox homeostasis.  One nutrient transporter frequently overexpressed in human cancers is the cystine/glutamate antiporter solute carrier family 7 member 11 (SLC7A11; also known as xCT).  SLC7A11 promotes cystine uptake and glutathione biosynthesis, resulting in protection from oxidative stress and ferroptotic cell death.  Recent studies have unexpectedly revealed that SLC7A11 also plays critical roles in glutamine metabolism and regulates the glucose and glutamine dependency of cancer cells.  This review discusses the roles of SLC7A11 in regulating the antioxidant response and nutrient dependency of cancer cells, explores our current understanding of SLC7A11 regulation in cancer metabolism, and highlights key open questions for future studies in this emerging research area.  A deeper understanding of SLC7A11 in cancer metabolism may identify new therapeutic opportunities to target this important amino acid transporter for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSc0x1bKxSR5v69nzq6467DfW6udTcc2eaHE9-tq0iQm7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MfjsFOgug%253D%253D&md5=22acaa68350c28038eb70f6974151c9f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1186%2Fs40880-018-0288-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40880-018-0288-x%26sid%3Dliteratum%253Aachs%26aulast%3DKoppula%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhuang%26aufirst%3DL.%26aulast%3DGan%26aufirst%3DB.%26atitle%3DAmino%2520acid%2520transporter%2520SLC7A11%252FxCT%2520at%2520the%2520crossroads%2520of%2520regulating%2520redox%2520homeostasis%2520and%2520nutrient%2520dependency%2520of%2520cancer%26jtitle%3DCancer%2520Commun.%26date%3D2018%26volume%3D38%26spage%3D12%26doi%3D10.1186%2Fs40880-018-0288-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kyani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shergalis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samanta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Discovery and mechanistic elucidation of a class of protein disulfide isomerase inhibitors for the treatment of glioblastoma</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1002%2Fcmdc.201700629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29235250" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1ehuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=164-177&author=A.+Kyaniauthor=S.+Tamuraauthor=S.+Yangauthor=A.+Shergalisauthor=S.+Samantaauthor=Y.+Kuangauthor=M.+Ljungmanauthor=N.+Neamati&title=Discovery+and+mechanistic+elucidation+of+a+class+of+protein+disulfide+isomerase+inhibitors+for+the+treatment+of+glioblastoma&doi=10.1002%2Fcmdc.201700629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Mechanistic Elucidation of a Class of Protein Disulfide Isomerase Inhibitors for the Treatment of Glioblastoma</span></div><div class="casAuthors">Kyani, Anahita; Tamura, Shuzo; Yang, Suhui; Shergalis, Andrea; Samanta, Soma; Kuang, Yuting; Ljungman, Mats; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">164-177</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein disulfide isomerase (PDI) is overexpressed in glioblastoma, the most aggressive form of brain cancer, and folds nascent proteins responsible for the progression and spread of the disease.  Herein we describe a novel nanomolar PDI inhibitor, pyrimidotriazinedione 35G8, that is toxic in a panel of human glioblastoma cell lines.  We performed a medium-throughput 20 000-compd. screen of a diverse subset of 1 000 000 compds. to identify cytotoxic small mols.  Cytotoxic compds. were screened for PDI inhibition, and, from the screen, 35G8 emerged as the most cytotoxic inhibitor of PDI.  Bromouridine labeling and sequencing (Bru-seq) of nascent RNA revealed that 35G8 induces nuclear factor-like 2 (Nrf2) antioxidant response, endoplasmic reticulum (ER) stress response, and autophagy.  Specifically, 35G8 upregulated heme oxygenase 1 and solute carrier family 7 member 11 (SLC7A11) transcription and protein expression and repressed PDI target genes such as thioredoxin-interacting protein 1 (TXNIP) and early growth response 1 (EGR1).  Interestingly, 35G8-induced cell death did not proceed via apoptosis or necrosis, but by a mixt. of autophagy and ferroptosis.  Cumulatively, our data demonstrate a mechanism for a novel PDI inhibitor as a chem. probe to validate PDI as a target for brain cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKe8TT4OBZ6LVg90H21EOLACvtfcHk0lj9BnJGKfCqcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1ehuw%253D%253D&md5=0f28e215d9b02792f2438f2dbc8b05e9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700629%26sid%3Dliteratum%253Aachs%26aulast%3DKyani%26aufirst%3DA.%26aulast%3DTamura%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DShergalis%26aufirst%3DA.%26aulast%3DSamanta%26aufirst%3DS.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DLjungman%26aufirst%3DM.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DDiscovery%2520and%2520mechanistic%2520elucidation%2520of%2520a%2520class%2520of%2520protein%2520disulfide%2520isomerase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520glioblastoma%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D164%26epage%3D177%26doi%3D10.1002%2Fcmdc.201700629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blat, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrobel, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hieronymus, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">The Connectivity Map: Using gene-expression signatures to connect small molecules, genes, and disease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1935</span>, <span class="refDoi"> DOI: 10.1126/science.1132939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1126%2Fscience.1132939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=17008526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2006&pages=1929-1935&author=J.+Lambauthor=E.+D.+Crawfordauthor=D.+Peckauthor=J.+W.+Modellauthor=I.+C.+Blatauthor=M.+J.+Wrobelauthor=J.+Lernerauthor=J.+P.+Brunetauthor=A.+Subramanianauthor=K.+N.+Rossauthor=M.+Reichauthor=H.+Hieronymusauthor=G.+Weiauthor=S.+A.+Armstrongauthor=S.+J.+Haggartyauthor=P.+A.+Clemonsauthor=R.+Weiauthor=S.+A.+Carrauthor=E.+S.+Landerauthor=T.+R.+Golub&title=The+Connectivity+Map%3A+Using+gene-expression+signatures+to+connect+small+molecules%2C+genes%2C+and+disease&doi=10.1126%2Fscience.1132939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The connectivity map: Using gene-expression signatures to connect small molecules, genes, and disease</span></div><div class="casAuthors">Lamb, Justin; Crawford, Emily D.; Peck, David; Modell, Joshua W.; Blat, Irene C.; Wrobel, Matthew J.; Lerner, Jim; Brunet, Jean-Philippe; Subramanian, Aravind; Ross, Kenneth N.; Reich, Michael; Hieronymus, Haley; Wei, Guo; Armstrong, Scott A.; Haggarty, Stephen J.; Clemons, Paul A.; Wei, Ru; Carr, Steven A.; Lander, Eric S.; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">5795</span>),
    <span class="NLM_cas:pages">1929-1935</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">To pursue a systematic approach to the discovery of functional connections among diseases, genetic perturbation, and drug action, we have created the first installment of a ref. collection of gene-expression profiles from cultured human cells treated with bioactive small mols., together with pattern-matching software to mine these data.  We demonstrate that this "Connectivity Map" resource can be used to find connections among small mols. sharing a mechanism of action, chems. and physiol. processes, and diseases and drugs.  These results indicate the feasibility of the approach and suggest the value of a large-scale community Connectivity Map project.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtu0M2BUO2w7Vg90H21EOLACvtfcHk0lj9BnJGKfCqcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVSnsrbN&md5=1fad887ad515ffdd8064e526ff25a4db</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1132939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1132939%26sid%3Dliteratum%253Aachs%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DCrawford%26aufirst%3DE.%2BD.%26aulast%3DPeck%26aufirst%3DD.%26aulast%3DModell%26aufirst%3DJ.%2BW.%26aulast%3DBlat%26aufirst%3DI.%2BC.%26aulast%3DWrobel%26aufirst%3DM.%2BJ.%26aulast%3DLerner%26aufirst%3DJ.%26aulast%3DBrunet%26aufirst%3DJ.%2BP.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DRoss%26aufirst%3DK.%2BN.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DHieronymus%26aufirst%3DH.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DThe%2520Connectivity%2520Map%253A%2520Using%2520gene-expression%2520signatures%2520to%2520connect%2520small%2520molecules%252C%2520genes%252C%2520and%2520disease%26jtitle%3DScience%26date%3D2006%26volume%3D313%26spage%3D1929%26epage%3D1935%26doi%3D10.1126%2Fscience.1132939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torikai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohigashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, K.</span></span> <span> </span><span class="NLM_article-title">Ebselen, a glutathione peroxidase mimetic seleno-organic compound, as a multifunctional antioxidant. Implication for inflammation-associated carcinogenesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">2687</span>– <span class="NLM_lpage">2694</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109641200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1074%2Fjbc.M109641200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=11714717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVSgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=2687-2694&author=Y.+Nakamuraauthor=Q.+Fengauthor=T.+Kumagaiauthor=K.+Torikaiauthor=H.+Ohigashiauthor=T.+Osawaauthor=N.+Noguchiauthor=E.+Nikiauthor=K.+Uchida&title=Ebselen%2C+a+glutathione+peroxidase+mimetic+seleno-organic+compound%2C+as+a+multifunctional+antioxidant.+Implication+for+inflammation-associated+carcinogenesis&doi=10.1074%2Fjbc.M109641200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Ebselen, a glutathione peroxidase mimetic seleno-organic compound, as a multifunctional antioxidant: implication for inflammation-associated carcinogenesis</span></div><div class="casAuthors">Nakamura, Yoshimasa; Feng, Qing; Kumagai, Takeshi; Torikai, Koji; Ohigashi, Hajime; Osawa, Toshihiko; Noguchi, Noriko; Niki, Etsuo; Uchida, Koji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2687-2694</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Ebselen, a seleno-org. compd. showing glutathione peroxidase-like activity, is one of the promising synthetic antioxidants.  In the present study, we investigated the antioxidant activities of ebselen using a 12-O-tetradecanoylphorbol-13-acetate (TPA)-treated mouse skin model.  Double pretreatments of mouse skin with ebselen significantly inhibited TPA-induced formation of thiobarbituric acid-reacting substance, known as an overall oxidative damage biomarker, in mouse epidermis, suggesting that ebselen indeed acts as an antioxidant in mouse skin.  The antioxidative effect of ebselen is attributed to its selective blockade of leukocyte infiltration and activation leading to attenuation of the H2O2 level.  In in vitro studies, ebselen inhibited TPA-induced superoxide generation in differentiated HL-60 cells and lipopolysaccharide-induced cyclooxygenase-2 protein expression in RAW 264.7 cells.  In addn., we demonstrated for the first time that ebselen potentiated phase II enzyme activities, including NAD(P)H:(quinone-acceptor) oxidoreductase 1 and glutathione S-transferase in cultured hepatocytes and in mouse skin.  These results strongly suggest that ebselen, a multifunctional antioxidant, is a potential chemopreventive agent in inflammation-assocd. carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGeGWFoXCUYLVg90H21EOLACvtfcHk0lifkhW-ML23JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVSgsb8%253D&md5=11107ceb27808fdfda33e7aa4c31c446</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109641200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109641200%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DKumagai%26aufirst%3DT.%26aulast%3DTorikai%26aufirst%3DK.%26aulast%3DOhigashi%26aufirst%3DH.%26aulast%3DOsawa%26aufirst%3DT.%26aulast%3DNoguchi%26aufirst%3DN.%26aulast%3DNiki%26aufirst%3DE.%26aulast%3DUchida%26aufirst%3DK.%26atitle%3DEbselen%252C%2520a%2520glutathione%2520peroxidase%2520mimetic%2520seleno-organic%2520compound%252C%2520as%2520a%2520multifunctional%2520antioxidant.%2520Implication%2520for%2520inflammation-associated%2520carcinogenesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D2687%26epage%3D2694%26doi%3D10.1074%2Fjbc.M109641200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreyra, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hryckowian, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Child, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solow-Cordero, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higginbottom, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segal, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banaei, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnenburg, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span> <span> </span><span class="NLM_article-title">A small-molecule antivirulence agent for treating Clostridium difficile infection</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">306ra148</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aac9103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1126%2Fscitranslmed.aac9103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=26400909" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&author=K.+O.+Benderauthor=M.+Garlandauthor=J.+A.+Ferreyraauthor=A.+J.+Hryckowianauthor=M.+A.+Childauthor=A.+W.+Puriauthor=D.+E.+Solow-Corderoauthor=S.+K.+Higginbottomauthor=E.+Segalauthor=N.+Banaeiauthor=A.+Shenauthor=J.+L.+Sonnenburgauthor=M.+Bogyo&title=A+small-molecule+antivirulence+agent+for+treating+Clostridium+difficile+infection&doi=10.1126%2Fscitranslmed.aac9103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aac9103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aac9103%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DK.%2BO.%26aulast%3DGarland%26aufirst%3DM.%26aulast%3DFerreyra%26aufirst%3DJ.%2BA.%26aulast%3DHryckowian%26aufirst%3DA.%2BJ.%26aulast%3DChild%26aufirst%3DM.%2BA.%26aulast%3DPuri%26aufirst%3DA.%2BW.%26aulast%3DSolow-Cordero%26aufirst%3DD.%2BE.%26aulast%3DHigginbottom%26aufirst%3DS.%2BK.%26aulast%3DSegal%26aufirst%3DE.%26aulast%3DBanaei%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DSonnenburg%26aufirst%3DJ.%2BL.%26aulast%3DBogyo%26aufirst%3DM.%26atitle%3DA%2520small-molecule%2520antivirulence%2520agent%2520for%2520treating%2520Clostridium%2520difficile%2520infection%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2015%26volume%3D7%26doi%3D10.1126%2Fscitranslmed.aac9103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beilhartz, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, R. A.</span></span> <span> </span><span class="NLM_article-title">Comment on “A small-molecule antivirulence agent for treating Clostridium difficile infection”</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">370tc2</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aad8926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1126%2Fscitranslmed.aad8926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28003550" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&author=G.+L.+Beilhartzauthor=J.+Tamauthor=Z.+Zhangauthor=R.+A.+Melnyk&title=Comment+on+%E2%80%9CA+small-molecule+antivirulence+agent+for+treating+Clostridium+difficile+infection%E2%80%9D&doi=10.1126%2Fscitranslmed.aad8926"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aad8926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aad8926%26sid%3Dliteratum%253Aachs%26aulast%3DBeilhartz%26aufirst%3DG.%2BL.%26aulast%3DTam%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMelnyk%26aufirst%3DR.%2BA.%26atitle%3DComment%2520on%2520%25E2%2580%259CA%2520small-molecule%2520antivirulence%2520agent%2520for%2520treating%2520Clostridium%2520difficile%2520infection%25E2%2580%259D%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2016%26volume%3D8%26doi%3D10.1126%2Fscitranslmed.aad8926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trychta, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Back, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, B. K.</span></span> <span> </span><span class="NLM_article-title">KDEL receptors are differentially regulated to maintain the ER proteome under calcium deficiency</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1829</span>– <span class="NLM_lpage">1840.e6</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2018.10.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.celrep.2018.10.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=30428351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2qtbvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=1829-1840.e6&author=K.+A.+Trychtaauthor=S.+Backauthor=M.+J.+Hendersonauthor=B.+K.+Harvey&title=KDEL+receptors+are+differentially+regulated+to+maintain+the+ER+proteome+under+calcium+deficiency&doi=10.1016%2Fj.celrep.2018.10.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">KDEL Receptors Are Differentially Regulated to Maintain the ER Proteome under Calcium Deficiency</span></div><div class="casAuthors">Trychta, Kathleen A.; Back, Susanne; Henderson, Mark J.; Harvey, Brandon K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1829-1840.e6</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Retention of crit. endoplasmic reticulum (ER) luminal proteins needed to carry out diverse functions (e.g., protein synthesis and folding, lipid metab.) is mediated through a carboxy-terminal ER retention sequence (ERS) and its interaction with KDEL receptors.  Here, we demonstrate that depleting ER calcium causes mass departure of ERS-contg. proteins from cells by overwhelming KDEL receptors.  In addn., we provide evidence that KDELR2 and KDELR3, but not KDELR1, are unfolded protein response (UPR) genes upregulated as an adaptive response to counteract the loss of ERS-contg. proteins, suggesting previously unknown isoform-specific functions of the KDEL receptors.  Overall, our findings establish that decreases in ER calcium change the compn. of the ER luminal proteome and secretome, which can impact cellular functions and cell viability.  The redistribution of the ER proteome from inside the cell to the outside has implications for dissecting the complex relationship of ER homeostasis with diverse disease pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzB6TodoZ_oLVg90H21EOLACvtfcHk0ljlqjgQWNic1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2qtbvP&md5=fdec1c46459cde29eed85c6421a21b1a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.10.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.10.055%26sid%3Dliteratum%253Aachs%26aulast%3DTrychta%26aufirst%3DK.%2BA.%26aulast%3DBack%26aufirst%3DS.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DHarvey%26aufirst%3DB.%2BK.%26atitle%3DKDEL%2520receptors%2520are%2520differentially%2520regulated%2520to%2520maintain%2520the%2520ER%2520proteome%2520under%2520calcium%2520deficiency%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D25%26spage%3D1829%26epage%3D1840.e6%26doi%3D10.1016%2Fj.celrep.2018.10.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schurmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule target engagement in cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.chembiol.2016.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=27049669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A280%3ADC%252BC28fntVKqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=435-441&author=M.+Schurmannauthor=P.+Janningauthor=S.+Zieglerauthor=H.+Waldmann&title=Small-molecule+target+engagement+in+cells&doi=10.1016%2Fj.chembiol.2016.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Target Engagement in Cells</span></div><div class="casAuthors">Schurmann Marc; Janning Petra; Ziegler Slava; Waldmann Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Cell chemical biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Monitoring how, when, and where small molecules engage their targets inside living cells is a critical step in chemical biology and pharmacological research, because it enables compound efficacy and confirmation of mode of action to be assessed.  In this mini-review we summarize the currently available methodologies to detect and prove direct target engagement in cells and offer a critical view of their key advantages and disadvantages.  As the interest of the field shifts toward discovery and validation of high-quality agents, we expect that efforts to develop and refine these types of methodologies will also intensify in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRSou8rC8Pn5252AMpI4GK2fW6udTcc2eYL_ELhBYwm3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fntVKqsw%253D%253D&md5=49bd3dc012e84676f2939f189896c254</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DSchurmann%26aufirst%3DM.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DZiegler%26aufirst%3DS.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DSmall-molecule%2520target%2520engagement%2520in%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D435%26epage%3D441%26doi%3D10.1016%2Fj.chembiol.2016.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richard, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lena, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierceall, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minond, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodder, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardone, M. H.</span></span> <span> </span><span class="NLM_article-title">Hydroxyquinoline-derived compounds and analoguing of selective Mcl-1 inhibitors using a functional biomarker</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6642</span>– <span class="NLM_lpage">6649</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.08.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.bmc.2013.08.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=23993674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGrurbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6642-6649&author=D.+J.+Richardauthor=R.+Lenaauthor=T.+Bannisterauthor=N.+Blakeauthor=W.+E.+Pierceallauthor=N.+E.+Carlsonauthor=C.+E.+Kellerauthor=M.+Koenigauthor=Y.+Heauthor=D.+Minondauthor=J.+Mishraauthor=M.+Cameronauthor=T.+Spicerauthor=P.+Hodderauthor=M.+H.+Cardone&title=Hydroxyquinoline-derived+compounds+and+analoguing+of+selective+Mcl-1+inhibitors+using+a+functional+biomarker&doi=10.1016%2Fj.bmc.2013.08.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Hydroxyquinoline-derived compounds and analoguing of selective Mcl-1 inhibitors using a functional biomarker</span></div><div class="casAuthors">Richard, David J.; Lena, Ryan; Bannister, Thomas; Blake, Noel; Pierceall, William E.; Carlson, Nicole E.; Keller, Christina Eberhart; Koenig, Marcel; He, Yuanjun; Minond, Dmitriy; Mishra, Jitendra; Cameron, Michael; Spicer, Timothy; Hodder, Peter; Cardone, Michael H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6642-6649</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Anti-apoptotic Bcl-2 family proteins are important oncol. therapeutic targets.  To date, BH3 mimetics that abrogate anti-apoptotic activity have largely been directed at Bcl-2 and/or Bcl-xL.  One obsd. mechanism of resistance to these inhibitors is increased Mcl-1 levels in cells exposed to such therapeutics.  For this reason, and because Mcl-1 is important in the onset of lymphoid, myeloid, and other cancers, it has become a target of great interest.  However, small mol. inhibitors displaying potency and selectivity for Mcl-1 are lacking.  Identifying such compds. has been challenging due to difficulties in translating the target selectivity obsd. at the biochem. level to the cellular level.  Herein we report the results of an HTS strategy coupled with directed hit optimization.  Compds. identified have selective Mcl-1 inhibitory activity with greater than 100-fold reduced affinity for Bcl-xL.  The selectivity of these compds. at the cellular level was validated using BH3 profiling, a novel personalized diagnostic approach.  This assay provides an important functional biomarker that allows for the characterization of cells based upon their dependencies on various anti-apoptotic Bcl-2 proteins.  We demonstrate that cells dependent on Mcl-1 or Bcl-2/Bcl-xL for survival are commensurately responsive to compds. that genuinely target those proteins.  The identification of compd. 9 with uniquely validated and selective Mcl-1 inhibitory activity provides a valuable tool to those studying the intrinsic apoptosis pathway and highlights an important approach in the development of a first-in-class cancer therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwGeLQPRnawrVg90H21EOLACvtfcHk0ljlqjgQWNic1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGrurbJ&md5=a2875b8bc8fce034d17def9920dd31a4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.08.017%26sid%3Dliteratum%253Aachs%26aulast%3DRichard%26aufirst%3DD.%2BJ.%26aulast%3DLena%26aufirst%3DR.%26aulast%3DBannister%26aufirst%3DT.%26aulast%3DBlake%26aufirst%3DN.%26aulast%3DPierceall%26aufirst%3DW.%2BE.%26aulast%3DCarlson%26aufirst%3DN.%2BE.%26aulast%3DKeller%26aufirst%3DC.%2BE.%26aulast%3DKoenig%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DMinond%26aufirst%3DD.%26aulast%3DMishra%26aufirst%3DJ.%26aulast%3DCameron%26aufirst%3DM.%26aulast%3DSpicer%26aufirst%3DT.%26aulast%3DHodder%26aufirst%3DP.%26aulast%3DCardone%26aufirst%3DM.%2BH.%26atitle%3DHydroxyquinoline-derived%2520compounds%2520and%2520analoguing%2520of%2520selective%2520Mcl-1%2520inhibitors%2520using%2520a%2520functional%2520biomarker%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6642%26epage%3D6649%26doi%3D10.1016%2Fj.bmc.2013.08.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S. W.</span></span> <span> </span><span class="NLM_article-title">TET1 exerts its tumour suppressor function by regulating autophagy in glioma cells</span>. <i>Biosci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span>, <span class="NLM_elocation-id">BSR20160523</span> <span class="refDoi"> DOI: 10.1042/BSR20160523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1042%2FBSR20160523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=28341638" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&author=R.+Fuauthor=Y.+Dingauthor=J.+Luoauthor=L.+Yuauthor=C.+L.+Liauthor=D.+S.+Liauthor=S.+W.+Guo&title=TET1+exerts+its+tumour+suppressor+function+by+regulating+autophagy+in+glioma+cells&doi=10.1042%2FBSR20160523"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1042%2FBSR20160523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20160523%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DR.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DLi%26aufirst%3DD.%2BS.%26aulast%3DGuo%26aufirst%3DS.%2BW.%26atitle%3DTET1%2520exerts%2520its%2520tumour%2520suppressor%2520function%2520by%2520regulating%2520autophagy%2520in%2520glioma%2520cells%26jtitle%3DBiosci.%2520Rep.%26date%3D2017%26volume%3D37%26doi%3D10.1042%2FBSR20160523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span> <span> </span><span class="NLM_article-title">Glutathione metabolism in cancer progression and treatment resistance</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>217</i></span>,  <span class="NLM_fpage">2291</span>– <span class="NLM_lpage">2298</span>, <span class="refDoi"> DOI: 10.1083/jcb.201804161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1083%2Fjcb.201804161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29915025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFGntb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=217&publication_year=2018&pages=2291-2298&author=A.+Bansalauthor=M.+C.+Simon&title=Glutathione+metabolism+in+cancer+progression+and+treatment+resistance&doi=10.1083%2Fjcb.201804161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione metabolism in cancer progression and treatment resistance</span></div><div class="casAuthors">Bansal, Ankita; Simon, M. Celeste</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">217</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2291-2298</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">1540-8140</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Glutathione (GSH) is the most abundant antioxidant found in living organisms and has multiple functions, most of which maintain cellular redox homeostasis.  GSH preserves sufficient levels of cysteine and detoxifies xenobiotics while also conferring therapeutic resistance to cancer cells.  However, GSH metab. plays both beneficial and pathogenic roles in a variety of malignancies.  It is crucial to the removal and detoxification of carcinogens, and alterations in this pathway can have a profound effect on cell survival.  Excess GSH promotes tumor progression, where elevated levels correlate with increased metastasis.  In this review, we discuss recent studies that focus on deciphering the role of GSH in tumor initiation and progression as well as mechanisms underlying how GSH imparts treatment resistance to growing cancers.  Targeting GSH synthesis/utilization therefore represents a potential means of rendering tumor cells more susceptible to different treatment options such as chemotherapy and radiotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeFe4G21ytbVg90H21EOLACvtfcHk0lhQwCnhBIvCUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFGntb3N&md5=2d071bd5b51fa8628dd56a2ee683931d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201804161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201804161%26sid%3Dliteratum%253Aachs%26aulast%3DBansal%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26atitle%3DGlutathione%2520metabolism%2520in%2520cancer%2520progression%2520and%2520treatment%2520resistance%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2018%26volume%3D217%26spage%3D2291%26epage%3D2298%26doi%3D10.1083%2Fjcb.201804161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span> <span> </span><span class="NLM_article-title">Glutathione reductase mediates drug resistance in glioblastoma cells by regulating redox homeostasis</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1111/jnc.14250</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1111%2Fjnc.14250" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=29105080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2018&pages=93-104&author=Z.+Zhuauthor=S.+Duauthor=Y.+Duauthor=J.+Renauthor=G.+Yingauthor=Z.+Yan&title=Glutathione+reductase+mediates+drug+resistance+in+glioblastoma+cells+by+regulating+redox+homeostasis&doi=10.1111%2Fjnc.14250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Glutathione reductase mediates drug resistance in glioblastoma cells by regulating redox homeostasis</span></div><div class="casAuthors">Zhu, Zhongling; Du, Shuangshuang; Du, Yibo; Ren, Jing; Ying, Guoguang; Yan, Zhao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-104</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Glutathione (GSH) and GSH-related enzymes constitute the most important defense system that protects cells from free radical, radiotherapy, and chemotherapy attacks.  In this study, we aim to explore the potential role and regulatory mechanism of the GSH redox cycle in drug resistance in glioblastoma multiforme (GBM) cells.  We found that temozolomide (TMZ)-resistant glioma cells displayed lower levels of endogenous reactive oxygen species and higher levels of total antioxidant capacity and GSH than sensitive cells.  Moreover, the expression of glutathione reductase (GSR), the key enzyme of the GSH redox cycle, was higher in TMZ-resistant cells than in sensitive cells.  Furthermore, silencing GSR in drug-resistant cells improved the sensitivity of cells to TMZ or cisplatin.  Conversely, the over-expression of GSR in sensitive cells resulted in resistance to chemotherapy.  In addn., the GSR enzyme partially prevented the oxidative stress caused by pro-oxidant L-buthionine -sulfoximine.  The modulation of redox state by GSH or L-buthionine -sulfoximine regulated GSR-mediated drug resistance, suggesting that the action of GSR in drug resistance is assocd. with the modulation of redox homeostasis.  Intriguingly, a trend toward shorter progress-free survival was obsd. among GBM patients with high GSR expression.  These results indicated that GSR is involved in mediating drug resistance and is a potential target for improving GBM treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwxSPa6vBqGLVg90H21EOLACvtfcHk0lhQwCnhBIvCUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgsLvI&md5=968215abecae08b2419389b9220b6741</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fjnc.14250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjnc.14250%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DS.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DYing%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DZ.%26atitle%3DGlutathione%2520reductase%2520mediates%2520drug%2520resistance%2520in%2520glioblastoma%2520cells%2520by%2520regulating%2520redox%2520homeostasis%26jtitle%3DJ.%2520Neurochem.%26date%3D2018%26volume%3D144%26spage%3D93%26epage%3D104%26doi%3D10.1111%2Fjnc.14250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span> <span> </span><span class="NLM_article-title">Mechanism and significance of increased glutathione level in human hepatocellular carcinoma and liver regeneration</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1096/fj.00-0445fje</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1096%2Ffj.00-0445fje" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=11099488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1Sns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=19-21&author=Z.+Z.+Huangauthor=C.+Chenauthor=Z.+Zengauthor=H.+Yangauthor=J.+Ohauthor=L.+Chenauthor=S.+C.+Lu&title=Mechanism+and+significance+of+increased+glutathione+level+in+human+hepatocellular+carcinoma+and+liver+regeneration&doi=10.1096%2Ffj.00-0445fje"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and significance of increased glutathione level in human hepatocellular carcinoma and liver regeneration</span></div><div class="casAuthors">Huang, Zong-Zhi; Chen, Changjin; Zeng, Zhaohui; Yang, Heping; Oh, Jenny; Chen, Lixin; Lu, Shelly C.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-21</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Compared to normal human liver, glutathione (GSH) levels were increased in human hepatocellular carcinomas; GSH synthetase (GS) and γ-glutamylcysteine synthetase (GCS) heavy subunit (GCS-HS) transcripts were also increased.  In the hepatocellular carcinomas, there was an increased nuclear binding activity to antioxidant response element 4 (ARE-4), nuclear factor kappa B (NF-κB), and activator protein 1 (AP-1) elements of the GCS heavy subunit (GCS-HS) promoter.  In vitro expts. with HepG2 cells indicated that increased GSH facilitates the growth of HepG2 cells.  Both a normal baseline hepatic GSH level and a subsequent increase in GSH were required for normal liver regeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK0hhu4kMw8LVg90H21EOLACvtfcHk0lhQwCnhBIvCUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1Sns7w%253D&md5=91db39bacdd7319bbd94305715226ba1</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1096%2Ffj.00-0445fje&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.00-0445fje%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%2BZ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DOh%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DS.%2BC.%26atitle%3DMechanism%2520and%2520significance%2520of%2520increased%2520glutathione%2520level%2520in%2520human%2520hepatocellular%2520carcinoma%2520and%2520liver%2520regeneration%26jtitle%3DFASEB%2520J.%26date%3D2001%26volume%3D15%26spage%3D19%26epage%3D21%26doi%3D10.1096%2Ffj.00-0445fje" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span> <span> </span><span class="NLM_article-title">Regulation of glutathione synthesis</span>. <i>Mol. Aspects Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.mam.2008.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1016%2Fj.mam.2008.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=18601945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltFelu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=42-59&author=S.+C.+Lu&title=Regulation+of+glutathione+synthesis&doi=10.1016%2Fj.mam.2008.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of glutathione synthesis</span></div><div class="casAuthors">Lu, Shelly C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Aspects of Medicine</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">42-59</span>CODEN:
                <span class="NLM_cas:coden">MAMED5</span>;
        ISSN:<span class="NLM_cas:issn">0098-2997</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Glutathione (GSH) is a ubiquitous intracellular peptide with diverse functions that include detoxification, antioxidant defense, maintenance of thiol status, and modulation of cell proliferation.  GSH is synthesized in the cytosol of all mammalian cells in a tightly regulated manner.  The major determinants of GSH synthesis are the availability of cysteine, the sulfur amino acid precursor, and the activity of the rate-limiting enzyme, glutamate cysteine ligase (GCL).  GCL is composed for a catalytic (GCLC) and modifier (GCLM) subunit and they are regulated at multiple levels and at times differentially.  The second enzyme of GSH synthesis, GSH synthase (GS) is also regulated in a coordinated manner as GCL subunits and its up-regulation can further enhance the capacity of the cell to synthesize GSH.  Oxidative stress is well known to induce the expression of GSH synthetic enzymes.  Key transcription factors identified thus far include Nrf2/Nrf1 via the antioxidant response element (ARE), activator protein-1 (AP-1) and nuclear factor κ B (NFκB).  Dysregulation of GSH synthesis is increasingly being recognized as contributing to the pathogenesis of many pathol. conditions.  These include diabetes mellitus, pulmonary fibrosis, cholestatic liver injury, endotoxemia and drug-resistant tumor cells.  Manipulation of the GSH synthetic capacity is an important target in the treatment of many of these disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXr29hUN4_i7Vg90H21EOLACvtfcHk0lh8WcS89scc-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltFelu78%253D&md5=691c8d8898facb97f10dd52eac5072b8</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.mam.2008.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mam.2008.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%2BC.%26atitle%3DRegulation%2520of%2520glutathione%2520synthesis%26jtitle%3DMol.%2520Aspects%2520Med.%26date%3D2009%26volume%3D30%26spage%3D42%26epage%3D59%26doi%3D10.1016%2Fj.mam.2008.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veloso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washburn, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar-Sinha, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ljungman, M.</span></span> <span> </span><span class="NLM_article-title">Coordinated regulation of synthesis and stability of RNA during the acute TNF-induced proinflammatory response</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">2240</span>– <span class="NLM_lpage">2245</span>, <span class="refDoi"> DOI: 10.1073/pnas.1219192110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.1219192110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=23345452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVersLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=2240-2245&author=M.+T.+Paulsenauthor=A.+Velosoauthor=J.+Prasadauthor=K.+Bediauthor=E.+A.+Ljungmanauthor=Y.+C.+Tsanauthor=C.+W.+Changauthor=B.+Tarrierauthor=J.+G.+Washburnauthor=R.+Lyonsauthor=D.+R.+Robinsonauthor=C.+Kumar-Sinhaauthor=T.+E.+Wilsonauthor=M.+Ljungman&title=Coordinated+regulation+of+synthesis+and+stability+of+RNA+during+the+acute+TNF-induced+proinflammatory+response&doi=10.1073%2Fpnas.1219192110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Coordinated regulation of synthesis and stability of RNA during the acute TNF-induced proinflammatory response</span></div><div class="casAuthors">Paulsen, Michelle T.; Veloso, Artur; Prasad, Jayendra; Bedi, Karan; Ljungman, Emily A.; Tsan, Ya-Chun; Chang, Ching-Wei; Tarrier, Brendan; Washburn, Joseph G.; Lyons, Robert; Robinson, Daniel R.; Kumar-Sinha, Chandan; Wilson, Thomas E.; Ljungman, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2240-2245, S2240/1-S2240/347</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Steady-state gene expression is a coordination of synthesis and decay of RNA through epigenetic regulation, transcription factors, micro RNAs (miRNAs), and RNA-binding proteins.  Here, we present bromouride labeling and sequencing (Bru-Seq) and bromouridine pulse-chase and sequencing (BruChase-Seq) to assess genome-wide changes to RNA synthesis and stability in human fibroblasts at homeostasis and after exposure to the proinflammatory tumor necrosis factor (TNF).  The inflammatory response in human cells involves rapid and dramatic changes in gene expression, and the Bru-Seq and BruChase-Seq techniques revealed a coordinated and complex regulation of gene expression both at the transcriptional and posttranscriptional levels.  The combinatory anal. of both RNA synthesis and stability using Bru-Seq and BruChase-Seq allows for a much deeper understanding of mechanisms of gene regulation than afforded by the anal. of steady-state total RNA and should be useful in many biol. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfuRwriMUd77Vg90H21EOLACvtfcHk0lh8WcS89scc-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVersLY%253D&md5=74e41864b9379692b3265cb7a806e7ca</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1219192110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1219192110%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DM.%2BT.%26aulast%3DVeloso%26aufirst%3DA.%26aulast%3DPrasad%26aufirst%3DJ.%26aulast%3DBedi%26aufirst%3DK.%26aulast%3DLjungman%26aufirst%3DE.%2BA.%26aulast%3DTsan%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BW.%26aulast%3DTarrier%26aufirst%3DB.%26aulast%3DWashburn%26aufirst%3DJ.%2BG.%26aulast%3DLyons%26aufirst%3DR.%26aulast%3DRobinson%26aufirst%3DD.%2BR.%26aulast%3DKumar-Sinha%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DT.%2BE.%26aulast%3DLjungman%26aufirst%3DM.%26atitle%3DCoordinated%2520regulation%2520of%2520synthesis%2520and%2520stability%2520of%2520RNA%2520during%2520the%2520acute%2520TNF-induced%2520proinflammatory%2520response%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D2240%26epage%3D2245%26doi%3D10.1073%2Fpnas.1219192110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mootha, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebert, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillette, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomeroy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span> <span> </span><span class="NLM_article-title">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">15545</span>– <span class="NLM_lpage">15550</span>, <span class="refDoi"> DOI: 10.1073/pnas.0506580102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=10.1073%2Fpnas.0506580102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=16199517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=15545-15550&author=A.+Subramanianauthor=P.+Tamayoauthor=V.+K.+Moothaauthor=S.+Mukherjeeauthor=B.+L.+Ebertauthor=M.+A.+Gilletteauthor=A.+Paulovichauthor=S.+L.+Pomeroyauthor=T.+R.+Golubauthor=E.+S.+Landerauthor=J.+P.+Mesirov&title=Gene+set+enrichment+analysis%3A+A+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles&doi=10.1073%2Fpnas.0506580102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</span></div><div class="casAuthors">Subramanian, Aravind; Tamayo, Pablo; Mootha, Vamsi K.; Mukherjee, Sayan; Ebert, Benjamin L.; Gillette, Michael A.; Paulovich, Amanda; Pomeroy, Scott L.; Golub, Todd R.; Lander, Eric S.; Mesirov, Jill P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15545-15550</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Although genomewide RNA expression anal. has become a routine tool in biomedical research, extg. biol. insight from such information remains a major challenge.  Here, we describe a powerful anal. method called Gene Set Enrichment Anal. (GSEA) for interpreting gene expression data.  The method derives its power by focusing on gene sets, i.e., groups of genes that share common biol. function, chromosomal location, or regulation.  We demonstrate how GSEA yields insights into several cancer-related data sets, including leukemia and lung cancer.  Notably, where single-gene anal. finds little similarity between two independent studies of patients survival in lung cancer, GSEA reveals many biol. pathways in common.  The GSEA method is embodied in a freely available software package, together with an initial database of 1,325 biol. defined gene sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH2_E6gWd0yrVg90H21EOLACvtfcHk0lh8WcS89scc-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ShtrnO&md5=ca2eb221010f20379199e6442c65fc2e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0506580102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0506580102%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DTamayo%26aufirst%3DP.%26aulast%3DMootha%26aufirst%3DV.%2BK.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DEbert%26aufirst%3DB.%2BL.%26aulast%3DGillette%26aufirst%3DM.%2BA.%26aulast%3DPaulovich%26aufirst%3DA.%26aulast%3DPomeroy%26aufirst%3DS.%2BL.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DLander%26aufirst%3DE.%2BS.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26atitle%3DGene%2520set%2520enrichment%2520analysis%253A%2520A%2520knowledge-based%2520approach%2520for%2520interpreting%2520genome-wide%2520expression%2520profiles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D15545%26epage%3D15550%26doi%3D10.1073%2Fpnas.0506580102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i93"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00728">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_62273"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00728?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00728</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplemental File 1: Proteomics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_001.xlsx">XLSX</a>)</p></li><li><p class="inline">Supplemental File 2: Bru-seq (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Additional computational analysis, biochemical data, activity of <b>AS15</b> and <b>CD343</b> in the PDI reductase assay, deconvoluted mass spectra, SAR of type XI compounds, and CMap structures (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings: jmcmar_mfs_AS15_analogues (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_001.xlsx">jm0c00728_si_001.xlsx (4.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_002.xlsx">jm0c00728_si_002.xlsx (1.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_003.pdf">jm0c00728_si_003.pdf (9.5 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00728/suppl_file/jm0c00728_si_004.csv">jm0c00728_si_004.csv (2.57 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00728&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-18%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00728%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00728" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67995acb481a3dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
