
<html lang="en"     class="pb-page"  data-request-id="615c67de-4141-4e99-9d56-d98029623679"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c02255;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2021.64.issue-6;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design and Synthesis of Novel Epigenetic Inhibitors Targeting Histone Deacetylases, DNA Methyltransferase 1, and Lysine Methyltransferase G9a with In Vivo Efficacy in Multiple Myeloma" /></meta><meta name="dc.Creator" content="Obdulia  Rabal" /></meta><meta name="dc.Creator" content="Edurne  San José-Enériz" /></meta><meta name="dc.Creator" content="Xabier  Agirre" /></meta><meta name="dc.Creator" content="Juan Antonio  Sánchez-Arias" /></meta><meta name="dc.Creator" content="Irene  de Miguel" /></meta><meta name="dc.Creator" content="Raquel  Ordoñez" /></meta><meta name="dc.Creator" content="Leire  Garate" /></meta><meta name="dc.Creator" content="Estíbaliz  Miranda" /></meta><meta name="dc.Creator" content="Elena  Sáez" /></meta><meta name="dc.Creator" content="Amaia  Vilas-Zornoza" /></meta><meta name="dc.Creator" content="Antonio  Pineda-Lucena" /></meta><meta name="dc.Creator" content="Ander  Estella" /></meta><meta name="dc.Creator" content="Feifei  Zhang" /></meta><meta name="dc.Creator" content="Wei  Wu" /></meta><meta name="dc.Creator" content="Musheng  Xu" /></meta><meta name="dc.Creator" content="Felipe  Prosper" /></meta><meta name="dc.Creator" content="Julen  Oyarzabal" /></meta><meta name="dc.Description" content="Concomitant inhibition of key epigenetic pathways involved in silencing tumor suppressor genes has been recognized as a promising strategy for cancer therapy. Herein, we report a first-in-class ser..." /></meta><meta name="Description" content="Concomitant inhibition of key epigenetic pathways involved in silencing tumor suppressor genes has been recognized as a promising strategy for cancer therapy. Herein, we report a first-in-class ser..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 4, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02255" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02255" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02255" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02255" /></link>
        
    
    

<title>Design and Synthesis of Novel Epigenetic Inhibitors Targeting Histone Deacetylases, DNA Methyltransferase 1, and Lysine Methyltransferase G9a with In Vivo Efficacy in Multiple Myeloma | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02255" /></meta><meta property="og:title" content="Design and Synthesis of Novel Epigenetic Inhibitors Targeting Histone Deacetylases, DNA Methyltransferase 1, and Lysine Methyltransferase G9a with In Vivo Efficacy in Multiple Myeloma" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0021.jpeg" /></meta><meta property="og:description" content="Concomitant inhibition of key epigenetic pathways involved in silencing tumor suppressor genes has been recognized as a promising strategy for cancer therapy. Herein, we report a first-in-class series of quinoline-based analogues that simultaneously inhibit histone deacetylases (from a low nanomolar range) and DNA methyltransferase-1 (from a mid-nanomolar range, IC50 &lt; 200 nM). Additionally, lysine methyltransferase G9a inhibitory activity is achieved (from a low nanomolar range) by introduction of a key lysine mimic group at the 7-position of the quinoline ring. The corresponding epigenetic functional cellular responses are observed: histone-3 acetylation, DNA hypomethylation, and decreased histone-3 methylation at lysine-9. These chemical probes, multitarget epigenetic inhibitors, were validated against the multiple myeloma cell line MM1.S, demonstrating promising in vitro activity of 12a (CM-444) with GI50 of 32 nM, an adequate therapeutic window (&gt;1 log unit), and a suitable pharmacokinetic profile. In vivo, 12a achieved significant antitumor efficacy in a xenograft mouse model of human multiple myeloma." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02255"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02255">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02255&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02255&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02255&amp;href=/doi/10.1021/acs.jmedchem.0c02255" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3392-3426</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02250" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01396" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Synthesis of Novel Epigenetic Inhibitors Targeting Histone Deacetylases, DNA Methyltransferase 1, and Lysine Methyltransferase G9a with <i>In Vivo</i> Efficacy in Multiple Myeloma</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Obdulia Rabal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Obdulia Rabal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Obdulia++Rabal">Obdulia Rabal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Edurne San José-Enériz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edurne San José-Enériz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edurne++San+Jos%C3%A9-En%C3%A9riz">Edurne San José-Enériz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xabier Agirre</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xabier Agirre</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xabier++Agirre">Xabier Agirre</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Juan Antonio Sánchez-Arias</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Juan Antonio Sánchez-Arias</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Juan+Antonio++S%C3%A1nchez-Arias">Juan Antonio Sánchez-Arias</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Irene de Miguel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Irene de Miguel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Irene++de+Miguel">Irene de Miguel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Raquel Ordoñez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Raquel Ordoñez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Raquel++Ordo%C3%B1ez">Raquel Ordoñez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leire Garate</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leire Garate</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leire++Garate">Leire Garate</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Estíbaliz Miranda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Estíbaliz Miranda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Est%C3%ADbaliz++Miranda">Estíbaliz Miranda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elena Sáez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elena Sáez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elena++S%C3%A1ez">Elena Sáez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Amaia Vilas-Zornoza</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amaia Vilas-Zornoza</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amaia++Vilas-Zornoza">Amaia Vilas-Zornoza</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Antonio Pineda-Lucena</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antonio Pineda-Lucena</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antonio++Pineda-Lucena">Antonio Pineda-Lucena</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ander Estella</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ander Estella</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ander++Estella">Ander Estella</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Feifei Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Feifei Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi Apptec (Tianjin) Company Ltd., TEDA, No. 168 Nanhai Road, 10th Avenue, 300456 Tianjin, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Feifei++Zhang">Feifei Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi Apptec (Tianjin) Company Ltd., TEDA, No. 168 Nanhai Road, 10th Avenue, 300456 Tianjin, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Wu">Wei Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Musheng Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Musheng Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi Apptec (Tianjin) Company Ltd., TEDA, No. 168 Nanhai Road, 10th Avenue, 300456 Tianjin, PR China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Musheng++Xu">Musheng Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Felipe Prosper</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Felipe Prosper</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Area de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div><div class="loa-info-affiliations-info">Departmento de Hematología, Clínica Universidad de Navarra, University of Navarra, Avenida Pio XII 36, E-31008 Pamplona, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#d7b1a7a5b8a4a7b2a597a2b9b6a1f9b2a4"><span class="__cf_email__" data-cfemail="234553514c5053465163564d42550d4650">[email protected]</span></a>. Phone: +34 948 194700 ext. 825807.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Felipe++Prosper">Felipe Prosper</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Julen Oyarzabal</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julen Oyarzabal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Small Molecule Discovery Platform, Molecular Therapeutics Program, Center for Applied Medical Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#1a706f767f7475637b68607b787b765a7f626e7f68747b76346f747b6c347f69"><span class="__cf_email__" data-cfemail="d0baa5bcb5bebfa9b1a2aab1b2b1bc90b5a8a4b5a2beb1bcfea5beb1a6feb5a3">[email protected]</span></a>, <a href="/cdn-cgi/l/email-protection#aec4c1d7cfdcd4cfcccfc2eecdc1c2dbc3ccdbddd8de80cdc1c3"><span class="__cf_email__" data-cfemail="a0cacfd9c1d2dac1c2c1cce0c3cfccd5cdc2d5d3d6d08ec3cfcd">[email protected]</span></a>. Phone: +34 948 19 47 00 ext. 2044.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julen++Oyarzabal">Julen Oyarzabal</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1941-7255" title="Orcid link">http://orcid.org/0000-0003-1941-7255</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02255&amp;href=/doi/10.1021%2Facs.jmedchem.0c02255" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3392–3426</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 4, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 December 2020</li><li><span class="item_label"><b>Published</b> online</span>4 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02255" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02255</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3392%26pageCount%3D35%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DObdulia%2BRabal%252C%2BEdurne%2BSan%2BJos%25C3%25A9-En%25C3%25A9riz%252C%2BXabier%2BAgirre%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D6%26contentID%3Dacs.jmedchem.0c02255%26title%3DDesign%2Band%2BSynthesis%2Bof%2BNovel%2BEpigenetic%2BInhibitors%2BTargeting%2BHistone%2BDeacetylases%252C%2BDNA%2BMethyltransferase%2B1%252C%2Band%2BLysine%2BMethyltransferase%2BG9a%2Bwith%2BIn%2BVivo%2BEfficacy%2Bin%2BMultiple%2BMyeloma%26numPages%3D35%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3426%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02255"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1186</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02255" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Synthesis of Novel Epigenetic Inhibitors Targeting Histone Deacetylases, DNA Methyltransferase 1, and Lysine Methyltransferase G9a with In Vivo Efficacy in Multiple Myeloma&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Obdulia&quot;,&quot;last_name&quot;:&quot;Rabal&quot;},{&quot;first_name&quot;:&quot;Edurne&quot;,&quot;last_name&quot;:&quot;San José-Enériz&quot;},{&quot;first_name&quot;:&quot;Xabier&quot;,&quot;last_name&quot;:&quot;Agirre&quot;},{&quot;first_name&quot;:&quot;Juan&quot;,&quot;last_name&quot;:&quot;Antonio Sánchez-Arias&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;de Miguel&quot;},{&quot;first_name&quot;:&quot;Raquel&quot;,&quot;last_name&quot;:&quot;Ordoñez&quot;},{&quot;first_name&quot;:&quot;Leire&quot;,&quot;last_name&quot;:&quot;Garate&quot;},{&quot;first_name&quot;:&quot;Estíbaliz&quot;,&quot;last_name&quot;:&quot;Miranda&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Sáez&quot;},{&quot;first_name&quot;:&quot;Amaia&quot;,&quot;last_name&quot;:&quot;Vilas-Zornoza&quot;},{&quot;first_name&quot;:&quot;Antonio&quot;,&quot;last_name&quot;:&quot;Pineda-Lucena&quot;},{&quot;first_name&quot;:&quot;Ander&quot;,&quot;last_name&quot;:&quot;Estella&quot;},{&quot;first_name&quot;:&quot;Feifei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Musheng&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Felipe&quot;,&quot;last_name&quot;:&quot;Prosper&quot;},{&quot;first_name&quot;:&quot;Julen&quot;,&quot;last_name&quot;:&quot;Oyarzabal&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;3392-3426&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02255&quot;},&quot;abstract&quot;:&quot;Concomitant inhibition of key epigenetic pathways involved in silencing tumor suppressor genes has been recognized as a promising strategy for cancer therapy. Herein, we report a first-in-class series of quinoline-based analogues that simultaneously inhibit histone deacetylases (from a low nanomolar range) and DNA methyltransferase-1 (from a mid-nanomolar range, IC50 &lt; 200 nM). Additionally, lysine methyltransferase G9a inhibitory activity is achieved (from a low nanomolar range) by introduction of a key lysine mimic group at the 7-position of the quinoline ring. The corresponding epigenetic functional cellular responses are observed: histone-3 acetylation, DNA hypomethylation, and decreased histone-3 methylation at lysine-9. These chemical probes, multitarget epigenetic inhibitors, were validated against the multiple myeloma cell line MM1.S, demonstrating promising in vitro activity of 12a (CM-444) with GI50 of 32 nM, an adequate therapeutic window (&gt;1 log unit), and a suitable pharmacokinetic profile. In vi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02255&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02255" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02255&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02255" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02255&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02255" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02255&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02255&amp;href=/doi/10.1021/acs.jmedchem.0c02255" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02255" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02255" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02255%26sid%3Dliteratum%253Aachs%26pmid%3D33661013%26genre%3Darticle%26aulast%3DRabal%26date%3D2021%26atitle%3DDesign%2Band%2BSynthesis%2Bof%2BNovel%2BEpigenetic%2BInhibitors%2BTargeting%2BHistone%2BDeacetylases%252C%2BDNA%2BMethyltransferase%2B1%252C%2Band%2BLysine%2BMethyltransferase%2BG9a%2Bwith%2BIn%2BVivo%2BEfficacy%2Bin%2BMultiple%2BMyeloma%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D6%26spage%3D3392%26epage%3D3426%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (6)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290795" title="Quinolines">Quinolines</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/jmcmar.2021.64.issue-6/20210325/jmcmar.2021.64.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Concomitant inhibition of key epigenetic pathways involved in silencing tumor suppressor genes has been recognized as a promising strategy for cancer therapy. Herein, we report a first-in-class series of quinoline-based analogues that simultaneously inhibit histone deacetylases (from a low nanomolar range) and DNA methyltransferase-1 (from a mid-nanomolar range, IC<sub>50</sub> < 200 nM). Additionally, lysine methyltransferase G9a inhibitory activity is achieved (from a low nanomolar range) by introduction of a key lysine mimic group at the 7-position of the quinoline ring. The corresponding epigenetic functional cellular responses are observed: histone-3 acetylation, DNA hypomethylation, and decreased histone-3 methylation at lysine-9. These chemical probes, multitarget epigenetic inhibitors, were validated against the multiple myeloma cell line MM1.S, demonstrating promising <i>in vitro</i> activity of <b>12a</b> (CM-444) with GI<sub>50</sub> of 32 nM, an adequate therapeutic window (>1 log unit), and a suitable pharmacokinetic profile. <i>In vivo</i>, <b>12a</b> achieved significant antitumor efficacy in a xenograft mouse model of human multiple myeloma.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetic modifications play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases. Due to the reversibility of epigenetic alterations, there has consequently been a focus upon directing probe and drug discovery efforts toward the identification of novel antitumor targets. Among the many epigenetic protein families, DNA methyltransferases (DNMTs) and histone deacetylases (HDACs), enzymes that respectively add methylation marks to DNA and erase acetylation marks from histones, were strongly implicated in cancer through extensive studies of their biological mechanisms and target validation via epigenetic medicinal chemistry. As a result, both are molecular targets of FDA-approved drugs for the treatment of hematologic malignancies, with a considerable number of active clinical trials and many inhibitors under development.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Approved DNMT inhibitors (DNMTi), Azacitidine and Decitabine, are nucleoside analogues that, following conversion to the triphosphate, incorporate into the DNA and covalently bind to DNMTs. Such non-specific mechanism of action translates into significant toxicity limiting the dose and efficacy. Despite recent progress in the development of non-nucleoside-based DNMTi binding to the catalytic site of DNMTs, none has entered into clinical phases.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> For HDACs, the pipeline is more advanced: a considerable number of potent HDAC inhibitors (HDACi) have been discovered; the pharmacophore pattern to inhibit them is well-established and five HDACi have been approved: SAHA (Vorinostat, <b>1</b>), Belinostat, Panobinostat (LBH-589, <b>2</b>), Romidepsin (Depsipeptide), and Chidamide,<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> the latter in China only. The clinical development of HDACi is hampered by dose-limiting toxicity, which can lead to poor single agent efficacy. Given that combination regimens are the mainstay of modern cancer therapy to achieve optimal clinical efficacy and the interplay between DNA methylation and histone deacetylation,<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> combination studies of HDACi and DNMTi have been pursued, which demonstrate a synergistic <i>in vitro</i> and <i>in vivo</i> antitumor activity and the induction of tumor suppressor genes (TSGs).<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11">(6−11)</a> As additive toxicity is often observed with combination therapy,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> dual DNMT and HDACi may overcome this hurdle.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">Only a few compounds have been reported having this dual DNMT/HDAC inhibitory profile: psammaplins<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (although its DNMT inhibitory activity is controversial<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>) and related analogues,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (−)-epigallocatechin-3-gallate,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> DC-517,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and NSC-319745-based hydroxamic acid derivatives,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> these last being claimed as the first <i>de novo</i>-designed dual DNMT–HDAC molecules. However, as exemplified with the advanced compound <b>4</b> in <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>, the NSC-319745-based chemical series still lacks potent DNMT1 inhibitory activity (e.g., <70% inhibition at 100 μM). In the same line, DC-517-based analogues (e.g., C02S) have IC<sub>50</sub> values in the low micromolar range against DNMTs and HDAC1.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0002.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. pan-HDACi (<b>1</b>,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a><b>2</b>,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><b>3</b><a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a>), Dual DNMT, and HDACi (<b>4</b>),<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Dual G9a and DNMT1 Inhibitors (<b>5</b>, <b>6</b>),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> G9a Inhibitor (<b>7</b>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and Dual G9a and HDACi (<b>8</b>);<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> IC<sub>50</sub> Values for <b>4</b> Were Determined Internally (See Footnote in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and the Rest of the Biochemical IC<sub>50</sub> Values Extracted from the Corresponding References; for <b>8</b>, the Enzymatic Activity of HDAC Was Measured in Intact Cells by the Homogeneous Cellular HDAC Assay Method Using the K562 Cell Line and the Ability to Block G9a by H3K9me2 Cell Immunofluorescence In-Cell Western (ICW) Assays Using the MDA-MB-231 Cell Line<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Here, considering polypharmacology as an attractive strategy to overcome the main limitations of the single target therapy leading to a superior therapeutic effect and a reduction of potential mechanism(s) of drug resistance,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> we set out with the goal of designing novel molecules able to inhibit DNMT and HDAC simultaneously. The chemical structures of our proprietary non-nucleoside epigenetic inhibitors CM-272 (<b>5</b>) and CM-579 (<b>6</b>) served as starting points because of their good potency against DNMT1 (IC<sub>50</sub> of 382 nM and 32 nM, respectively, <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) and because they reversibly bind at the substrate-binding site.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Of note, these compounds display also nanomolar inhibitory activity against G9a (EHMT2), a histone methyltransferase responsible for the mono- and di-methylation of the lysine 9 of the histone 3 (H3K9).<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> We postulated that combined H3K9 hypomethylation and hyperacetylation as a result of concurrent inhibition of G9a and HDAC should additionally contribute to relieve transcriptional repression of TSG in cancer. In fact, the anticancer response of dual G9a/HDAC inhibitory profile has also been examined for a series of quinazoline derivatives of BIX-01294 (<b>7</b>), exemplified by compound <b>8</b> in <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Therefore, achieving first-in-class molecules as DNMT1, HDAC and, optionally, G9a inhibitors targeting two or three different epigenetic marks simultaneously should expectedly result in chemical probes to validate the feasibility of the proposed multitarget epigenetic inhibition (dual and triple inhibition, IC<sub>50</sub> < 10 μM against each target of interest) as well as to study epigenetic targets from a mechanistic perspective (e.g., role in gene transcription) and to analyze their phenotypic and antiproliferative responses.</div><div class="NLM_p">Based on the well-established pharmacophore of HDACi and structural information available, incorporation of HDAC inhibitory activity to our chemical series seemed initially achievable considering the predicted binding mode of compounds <b>5</b> and <b>6</b> at the substrate-binding site of both methyltransferases; then, to achieve this goal, we explored around R2 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Moreover, owing to the structural knowledge gained from the SAR exploration of our lead compounds <b>5</b> and <b>6</b> and the optimization of <b>7</b> toward more potent G9a inhibitors,<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> we hypothesized that the inhibitory activity against G9a might be compatible with DNMT1 and HDAC inhibition to yield a triple epigenetic inhibitor; therefore, an exploration around R3 was also performed (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0004.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Focused exploration around three diversity points (R1, R2, and R3) of the quinoline ring to achieve first-in-class multitarget epigenetic inhibitors targeting DNMT1, HDAC, and, optionally, G9a—some examples are illustrated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Thus, using knowledge- and structure-based approaches, we designed a synthetically feasible focused exploration around the quinoline scaffold; the synthetic approach enables to explore three diversity points (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) that are key for primary activities. This exploration led to first-in-class multitarget epigenetic inhibitors targeting G9a, DNMT1, and HDAC. Herein, we report the discovery of chemical probes (e.g., <b>9a</b> and <b>12a</b>) that confirm the feasibility of the proposed multitarget epigenetic inhibition, which was not only validated by biochemical assays but also by their corresponding functional cellular responses: activities <i>versus</i> these epigenetic targets, in both validation scenarios, range from nM to low μM (up to ∼1–2 μM). In addition, this initial exploration led to basic SAR guidelines to achieve these two target compound profiles, (a) G9a, DNMT1, and HDAC inhibition as well as (b) DNMT1 and HDAC inhibition, and to identify absorption, distribution, metabolism, and excretion (ADME) properties that may require an optimization (e.g., permeability and solubility). From a drug discovery perspective, these chemical probes are key starting points to become optimized lead compounds according to project requirements (e.g., specific inhibition profile for selected targets, therapeutic window, ADME properties, etc.), fulfilling the corresponding target product profile.</div><div class="NLM_p">As mentioned above, DNMT1 and HDAC are molecular targets of FDA-approved drugs for the treatment of hematologic malignancies; however, multiple myeloma (MM) still remains incurable.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Despite approval by the FDA and the EMA of the pan-HDACi <b>2</b> in combination with bortezomib and deixamethasone for the treatment of relapsed or refractory MM,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> this is still an unmet medical need. Taking into account that not only histone 3 (H3) acetylation is altered in MM patients but also the DNA methylation pattern,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> a combination of HDAC and DNMT1 inhibition may be an adequate therapeutic strategy. In fact, very recently it was shown that a combination of azacitine and BG45 (HDACi) exhibited synergistic cell growth inhibition of MM tumor cells and induced cell growth inhibition in <i>in vivo</i> models.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p last">Thus, the antiproliferative response of these chemical probes was tested against MM tumor cells MM1.S. Those molecules with a potent antiproliferative response (GI<sub>50</sub>), <b>9a</b> as a triple inhibitor and <b>12a</b> as a dual inhibitor, were selected to monitor their corresponding functional cellular responses: histone-3 acetylation, DNA hypomethylation, and decreased histone-3 methylation at lysine-9. This exploration also led to the identification of compound <b>12a</b> (CM-444) as a pharmacological tool compound: suboptimal but adequate ADME and pharmacokinetic (PK) profiles as well as therapeutic window to perform an <i>in vivo</i> proof of concept. <i>In vivo</i> efficacy of <b>12a</b> was assessed in a xenograft mouse model of human MM.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05447" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05447" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Rational Design of Hybrid Compounds with HDAC, DNMT1, and G9a Inhibitory Activity</h3><div class="NLM_p">The classical pharmacophore for HDAC inhibition consists of a (i) a zinc-binding group (ZBG) that chelates the zinc ion in the active site, (ii) a recognition capping group that interacts with the rim of the catalytic tunnel, and (iii) a hydrophobic linker connecting the ZBG and the cap group. This pharmacophore pattern has been successfully exploited to derive a plethora of novel multifunctional HDACi compounds, most of them bearing a hydroxamic group as the ZBG.<a onclick="showRef(event, 'ref10 ref33'); return false;" href="javascript:void(0);" class="ref ref10 ref33">(10,33)</a> Compounds <b>5</b> and <b>6</b> are substrate competitive inhibitors against G9a and DNMTs, with predicted binding modes established by docking and consistent with SAR.<a onclick="showRef(event, 'ref21 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref21 ref34 ref35">(21,34,35)</a> Examination of these protein–ligand complex models suggested that linking a hydroxamic moiety to the methylpiperidine (position 4 of the quinoline ring; R2 in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) would project this ZBG substituent toward the solvent exposed area of both, G9a and DNMT, and be well tolerated from a potency perspective. From the viewpoint of HDAC inhibitory activity, the quinoline core of <b>5</b> and <b>6</b> would serve as a cap group. Following this rationale, and owing to its chemical similarity, the pyrimidylhydroxamic acid of quisinostat (<b>3</b>) was incorporated into <b>6</b> to yield compound <b>9a</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and synthesis in the <a class="ref internalNav" href="#sec4" aria-label="Chemistry">Chemistry</a> Section). As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, this compound retains the binding mode of the parent compound into G9a and DNMT1 and fits predictably well into the binding site cavities of HDAC1 and HDAC6. These two HDAC isoforms were chosen for routinely biochemical screening as representative of class I (HDAC1) and class IIb (HDAC6) HDAC isoforms. In fact, compound <b>9a</b> is a potent HDACi, with a preference for class I (HDAC1, HDAC2, and HDAC3 IC<sub>50</sub> of 3, 21, and 5 nM) over class IIb isoform (HDAC6 IC<sub>50</sub> of 152 nM). For the G9a and DNMT1 profiles, a significant drop in activity was observed compared to the parent compound <b>6</b> although <b>9a</b> satisfies our initial requirement of targeting the three epigenetic families (respective G9a and DNMT1 IC<sub>50</sub> values of 269 and 1160 nM, compared to 16 and 32 nM for <b>6</b>). On this basis, various analogues with different linker rings with varying lengths, hydrophobicities, and flexibilities were synthesized (compounds <b>9b–9k</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In general, flexible alkyl rings (<b>9c</b>) and methylene-homologated rings (<b>9d</b>, <b>9e</b>) yielded compounds with improved G9a (IC<sub>50</sub> < 100 nM), DNMT1 (IC<sub>50</sub> < 500 nM) and HDAC6 (IC<sub>50</sub> < 100 nM) potencies compared to that of compound <b>9a</b>, although at the cost of losing at least one log unit of HDAC1 activity. Conserving the basic nitrogen of the piperidine ring confers the highest potency against G9a (<b>9d</b>, <b>9e</b>, and <b>9g</b>),<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> with the striking exception of the cyclohexyl moiety of <b>9c</b> (G9a IC<sub>50</sub> of 55 nM), the triple inhibitor compound in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> with the most promising well-balanced biochemical profile against all targets. Substitution of the 5-methyl-2-furyl group of the initial hit <b>9a</b> by methyl to yield <b>10a</b> was beneficial for G9a and DNMT activities while retaining a similar HDAC inhibitory profile to that of <b>9a</b>. Finally, compound <b>11</b> was designed in an attempt to increase the DNMT1 potency of <b>9a</b> by incorporation of a 4-piperidyl moiety at the 7 position.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As this replacement of the 7-(3-pyrrolidin-1-ylpropoxy) group did not meet the expected improvement, SAR exploration of the 7 position (R3, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) with alternative groups with basic nitrogens was no longer continued.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0005.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Predicted complex of compound <b>9a</b> with G9a [(A) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>],<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> DNMT1 [(B) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>],<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> HDAC1 [(C) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKX">4BKX</a>]<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and HDAC6 [(D) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>]<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>—experimental validation of proposed binding modes will be experimentally performed by biophysical methods in due course.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Hybrid Compounds with Triple HDAC, DNMT1, and G9a Inhibitory Activity: Exploring Around R2 and R3 Positions<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0015.gif" alt="" id="GRAPHIC-d7e833-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0016.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">For data in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> and <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>, all IC<sub>50</sub> values are the average of at least two independent replicates performed at different days. If the absolute pIC<sub>50</sub> difference was higher than 0.3 log units, additional measurements were performed until satisfying the experimental error (by discarding individual results with values outside 2 MADs of the mean value). Standard Deviations are reported together with the corresponding IC<sub>50</sub> values in brackets and <i>in italics</i>. N.D. = not determined.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last"><b>1</b> (reference pan-HDACi),<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a><b>5</b> (dual G9a and DNMT1 inhibitor, potent G9a inhibition),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and <b>6</b> (dual G9a and DNMT1 inhibitor, potent DNMT1 inhibition)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> are utilized as positive controls for all biochemical assays reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. Synthetic approaches for all new molecules are shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> (compounds <b>9a–k</b>), <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> (<b>10a–b</b>) and <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> (<b>11</b>).</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Hybrid Compounds with Dual HDAC and DNMT1 Inhibitory Activity</h3><div class="NLM_p">During the course of our SAR exploration around compounds <b>5</b> and <b>6</b>, it was noticed that removal of the basic nitrogen at position 7 of the quinoline scaffold (R3, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was detrimental for G9a activity.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> This is consistent with the predicted role of the 7-(3-pyrrolidin-1-ylpropoxy) group mimicking the lysine side chain (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Similar conclusions were drawn during the optimization of <b>7</b><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> that can be explained on the basis of missing key interactions with Leu1086 (hydrogen-bond) and Tyr1154 (cation−π interaction) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The impact of replacing this lysine mimic group by a methoxy group was initially less impacting on DNMT1 activity for the quinoline analogues.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> With the goal of blocking G9a inhibitory activity, a handful of 7-methoxyquinolines with the optimal linkers from previous SAR explorations were prepared (<b>12a</b>, <b>12d</b>, <b>12e</b>, <b>12g</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), as well as some additional linkers designed to constrain the conformation of the hydroxamic moiety (<b>12b</b> and <b>12j</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). As expected, all of them were inactive against G9a (IC<sub>50</sub> > 10 μM). DNMT1 activity was less affected, spanning mid-nanomolar (again, for the flexible methylene-homologated rings of <b>12g</b> and <b>12h</b>) to low-micromolar ranges. For HDACs, inhibitory profiles between corresponding matched pairs in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> are in general consistent (e.g., <b>12a</b> vs <b>9a</b>, <b>12d</b> vs <b>9b</b> and <b>12e</b> vs <b>9c</b>), highlighting the minimal impact of the methoxy group moiety at the rim surface of the HDACs.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Hybrid Compounds with Dual DNMT1 and HDAC Activity: Exploration of the 4 Position<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0017.gif" alt="" id="GRAPHIC-d7e1011-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0018.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">N.D. = not determined. A corresponding synthetic approach is shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> (compounds <b>12a–k</b>).</p></div></div><div></div></div><div class="NLM_p">Considering a well-balanced contribution of the primary targets, compounds <b>12g</b> and <b>12h</b> had an adequate profile (IC<sub>50</sub> ∼ 300 nM vs DNMT1 and HDAC1; and, ∼100 nM against HDAC6). On the other hand, giving special emphasis to HDAC1 inhibition (IC<sub>50</sub> < 50 nM), the pyrimidylhydroxamic group of <b>12a</b>, its azabicyclo analogue <b>12c</b>, and the cyclohexyl linker (<b>12e</b>) were among the best compounds in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> (their IC<sub>50</sub> vs DNMT1 are ∼1–3 μM).</div><div class="NLM_p">Finally, a small exploration of the 2-position of the quinoline (R1, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was carried out by keeping constant the initial pyrimidylhydroxamic group of <b>12a</b> and replacing its heteroaryl ring, 5-methyl-2-furyl, by a methyl (<b>13a</b>), cyclohexyl (<b>13b</b>), phenyl (<b>13c</b>), N-linked piperidine (<b>13d</b>), 2,5-dimethyl-3-furyl (<b>13e</b>), and 5-methylthiophen-2-yl (<b>13f</b>) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, compared to <b>12a</b>, this exploration had a minor impact on the HDAC profile (HDAC1 IC<sub>50</sub> values are ∼10 nM); all these substitutions were detrimental to DNMT1 inhibition: IC<sub>50</sub> values > 10 μM for all these new molecules are reported in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, resulting in HDAC inhibition alone.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Hybrid Compounds: Exploration of the 2 Position (R1)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0019.gif" alt="" id="GRAPHIC-d7e1110-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0020.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Corresponding synthetic approach is shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a> (compounds <b>13a–f</b>).</p></div></div><div></div></div><div class="NLM_p">These initial explorations around these three diversity points (R1, R2, and R3) borne by the quinoline scaffold, represented in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and exemplified in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, led to some key general conclusions; a preliminary SAR guideline:<ul class="NLM_list-list_type-bullet"><li><p class="inline">Substituents at R1, position 2 of the quinoline ring, have a huge impact on DNMT1 activity; in fact, among all explored analogues (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), only the 5-methy-2-furyl (<b>12a</b>) was able to maintain a capacity of inhibition (IC<sub>50</sub> < 10 μM).</p></li><li><p class="inline">The classical pharmacophore for HDAC inhibition (containing a ZBG, e.g., hydroxamic acid<a onclick="showRef(event, 'ref33 ref39'); return false;" href="javascript:void(0);" class="ref ref33 ref39">(33,39)</a> or <i>ortho</i>-aminoanilides<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a>) should be located at R2, position 4 of the quinoline ring, thus causing minimal impact on G9a and DNMT1 inhibition (solvent exposed according to the proposed binding modes) and, on the other hand, quinoline serves as the cap group for binding to HDAC—as described in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. This substitution pattern led to potent inhibition of HDAC enzymatic activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>)</p></li><li><p class="inline">The 7-(3-pyrrolidin-1-ylpropoxy) group mimics the lysine side chain; then, as previously reported by our group,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> removal of the basic nitrogen at position 7 of the quinoline scaffold (located at R3) was detrimental for G9a activity. Thus, as illustrated in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a><i>versus</i><a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, we can easily modulate the G9a inhibitory activity.</p></li></ul></div><div class="NLM_p">Finally, we also explored an alternative scaffold to the quinoline ring: quinazoline. Interestingly, the corresponding quinazoline-based pair of the selected compound <b>12a</b> (details below), compound <b>14</b> (<a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a> and synthesis in the <named-content content-type="anchor" rid="sch1" type="simple"></named-content><named-content content-type="anchor" rid="sch2" type="simple"></named-content><named-content content-type="anchor" rid="sch3" type="simple"></named-content><named-content content-type="anchor" rid="sch4" type="simple"></named-content><named-content content-type="anchor" rid="sch5" type="simple"></named-content><a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>), does not exhibit DNMT1 activity (IC<sub>50</sub> > 10 μM), thereby indicating the impact of the quinoline scaffold on the DNMT1 activity—as previously reported for G9a and DNMT1 inhibitors.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0008.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>9a–k</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (15:1), 110 °C, MW, 4 h; (ii) corresponding amine, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 110–120 °C, 16 h; (iii) HCl/EtOAc(2.0 M) or HCl/MeOH (2.0 M), 20 °C, 30 min to 16 h; (iv) ethyl 4-oxocyclohexanecarboxylate, ZnCl<sub>2</sub>/diethyl ether, NaBH<sub>3</sub>CN, MeOH, 40 °C, 15.5 h; (v) LiOH·H<sub>2</sub>O, EtOH/H<sub>2</sub>O (2:1) or THF/MeOH/H<sub>2</sub>O (5:1:3 or 3:1:1), 20–25 °C, 1–16 h; (vi) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 20 °C, 2–16 h.</p></p></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0009.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>10a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) methyl boronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (10:1), 110 °C, MW, 3 h; (ii) corresponding amine, <i>t</i>-BuONa, xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, toluene, 110 °C, MW, 1–2 h; (iii) HCl/1,4-dioxane (4.0 M), rt, 1–3 h; (iv) ethyl 2-chloropyrimidine-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, rt, 3 h; (v) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (10:1:3), rt, overnight; (vi) THPONH<sub>2</sub>, HOBt, NMM, EDC·HCl, DMF, rt, overnight; (vii) HCl/1,4-dioxane (4.0 M), rt, 1 h.</p></p></figure><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0010.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, 95 °C, 12 h; (ii) <i>tert</i>-butyl 4-(bromomethyl)piperidine-1-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 16 h; (iii) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O (4:1), 90 °C, 16 h; (iv) ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 110 °C, 16 h; (v) LiOH·H<sub>2</sub>O, THF/H<sub>2</sub>O (2:1), 15 °C, 16 h; (vi) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 15 °C, 16 h; (vii) HCl/EtOAc (1.0 M), 15 °C, 16 h.</p></p></figure><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0011.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>12a–k</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, 100 °C, 16 h; (ii) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O (10:1), 90 °C, 16 h; (iii) corresponding amine, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120–140 °C, 12–16 h; (iv) HCl/MeOH (2.0 M) or HCl/EtOAc (1.0 or 2.0 M), 20–25 °C, 20 min −16 h; (v) ethyl 2-chloropyrimidine-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMF or CH<sub>3</sub>CN, 40–50 °C, 12–16 h; (vi) corresponding aldehyde or ketone, NaBH<sub>3</sub>CN, CH<sub>3</sub>COOH or ZnCl<sub>2</sub>, MeOH, 40–70 °C, 2–15.5 h; (vii) LiOH·H<sub>2</sub>O, THF/H<sub>2</sub>O (2:1) or THF/MeOH/H<sub>2</sub>O (6:1:1 or 3:1:1), 15–25 °C, 2–16 h; (viii) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 15–25 °C, 2–16 h.</p></p></figure><figure id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0012.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>13a–f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) corresponding boronic ester, Pd(PPh<sub>3</sub>)<sub>4</sub> or Pd(dppf)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O (10:1 or 5:1), 90–100 °C, 6–16 h; (ii) piperidine or ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane,100–140 °C, 12–48 h; (iii) Pd/C, H<sub>2</sub> (15 Psi), MeOH, 20 °C, 16 h; (iv) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (3:1:1) or THF/H<sub>2</sub>O (2:1), 15–25 °C, 16 h; (v) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 15–20 °C, 12–16 h; (vi) HCl/EtOAc (1.0 or 2.0 M), 15–25 °C, 4–16 h.</p></p></figure><figure id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0013.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 12 h; (ii) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 100 °C, 12 h; (iii) LiOH·H<sub>2</sub>O, THF/H<sub>2</sub>O (2:1), 25 °C, 12 h; (iv) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 25 °C, 12 h; (v) TFA, CH<sub>3</sub>CN/H<sub>2</sub>O (1:1), 60 °C, 5 min.</p></p></figure><figure id="cht2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0003.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Selected compound <b>12a</b> and its corresponding quinazoline-matched pair <b>14</b>.</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Cellular Response: Antiproliferative Effect and Functional Hallmarks</h3><div class="NLM_p">New molecules fulfilling target compound profiles, (a) G9a, DNMT1, and HDAC inhibition or (b) DNMT1 and HDAC inhibition (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), and covering diversity in terms of chemical space and biological responses were selected to test their antiproliferative activity using human cancer cells of MM, cell line MM1.S. From <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, only four triple inhibitors did not progress to the cellular assay against MM1.S: those chemical probes which IC<sub>50</sub> values are >1 μM for more than one target of interest (compounds <b>9g</b>, <b>9h</b>, <b>9j-<i>trans</i></b> and <b>9j-<i>cis</i></b>); <b>9i</b> and <b>11</b> were exceptions, diversity in chemical space, and were tested. From <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, we also discarded four double inhibitors: compounds the IC<sub>50</sub> values of which are >8 μM <i>versus</i> DNMT1 (<b>12f</b> and <b>12k</b>) and those molecules the IC<sub>50</sub> values of which are >1 μM for more than one target of interest (compounds <b>12j-<i>trans</i></b> and <b>12j-<i>cis</i></b>). Exploration around the R1 position (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) only led to HDACi lack of DNMT1 inhibition (IC<sub>50</sub> values > 10 μM). Then, for comparison purposes, only the most potent compound from <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> progressed to the cellular assay against MM1.S: <b>13b</b>.</div><div class="NLM_p">As seen in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, triple inhibitors exhibited GI<sub>50</sub> values in the low micromolar range (<b>9a–i</b>) comparable to that achieved by G9a-DNMT1 inhibitors <b>5</b> and <b>6</b> or were inactive below 10 μM (<b>10a–b</b>,<b>11</b>) possibly due to their poor PAMPA permeability (Pe ∼ <1 nm/s). Dual HDAC-DNMT1 compounds (<b>12a–12g</b>) with reduced molecular weight and slightly higher permeability (although still poor permeators, with Pe < 10 nm/s) mostly tended to show GI<sub>50</sub> < 1 μM. Among them, compound <b>12a</b> achieved the most potent antiproliferative response (GI<sub>50</sub> = 32 nM) close to the reference compound <b>2</b> (panobinostat, GI<sub>50</sub> = ∼ 9 nM), ∼0.5 log units difference, and with decreased toxicity against the healthy hepatic cell line THLE-2 (LC<sub>50</sub> 794 vs 22 nM); in fact, the therapeutic window of compound <b>12a</b> (1.4 log units) is 3 times larger than the corresponding therapeutic window of <b>2</b> (0.4 log units)—<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Furthermore, a pairwise comparison between <b>12a</b> and the most potent HDAC selective inhibitor of this chemical series (<b>13b</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) shows that dual inhibition leads to a more potent antiproliferative activity against MM1.S (>0.75 log units difference) and a better therapeutic window (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). On the other hand, we should also highlight that <b>12a</b> is >1.5 log units more potent against MM1.S than reference dual G9a and DNMT1 inhibitors (compounds <b>5</b> and <b>6</b>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Antiproliferative Response of Selected Hybrid Compounds against the MM1.S MM Cancer Cell Line and the Healthy Hepatic Cell Line THLE-2</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center">MM1.S GI<sub>50</sub> nM<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">MM1.S pGI<sub>50</sub><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">THLE-2 LC<sub>50</sub> nM 72 h<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">THLE-2 pLC<sub>50</sub><a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">PAMPA Pe (nm/s)<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">therapeutic window<a class="ref internalNav" href="#t4fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">8.9 (<i>0.1</i>)</td><td class="colsep0 rowsep0" align="left">8.05</td><td class="colsep0 rowsep0" align="left">22 (<i>14.5</i>)</td><td class="colsep0 rowsep0" align="left">7.66</td><td class="colsep0 rowsep0" align="left">14.2 (<i>0.7</i>)</td><td class="colsep0 rowsep0" align="left">0.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b><a onclick="showRef(event, 'ref21 ref34'); return false;" href="javascript:void(0);" class="ref ref21 ref34">(21,34)</a></td><td class="colsep0 rowsep0" align="left">1041 (<i>62.1</i>)</td><td class="colsep0 rowsep0" align="left">5.97</td><td class="colsep0 rowsep0" align="left">1780 (<i>596</i>)</td><td class="colsep0 rowsep0" align="left">5.75</td><td class="colsep0 rowsep0" align="left">12.9 (<i>1.4</i>)</td><td class="colsep0 rowsep0" align="left">0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b><a onclick="showRef(event, 'ref21 ref35'); return false;" href="javascript:void(0);" class="ref ref21 ref35">(21,35)</a></td><td class="colsep0 rowsep0" align="left">3842 (<i>92.1</i>)</td><td class="colsep0 rowsep0" align="left">5.42</td><td class="colsep0 rowsep0" align="left">1300 (<i>590</i>)</td><td class="colsep0 rowsep0" align="left">5.89</td><td class="colsep0 rowsep0" align="left">11.0 (<i>0.7</i>)</td><td class="colsep0 rowsep0" align="left">toxic<a class="ref internalNav" href="#t4fn8" aria-label="h">h</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="left">2077 (<i>409.5</i>)</td><td class="colsep0 rowsep0" align="left">5.68</td><td class="colsep0 rowsep0" align="left">10,900 (<i>1980</i>)</td><td class="colsep0 rowsep0" align="left">4.96</td><td class="colsep0 rowsep0" align="left">0 (<i>0</i>)</td><td class="colsep0 rowsep0" align="left">0.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9b</b></td><td class="colsep0 rowsep0" align="left">4505 (<i>2181</i>)</td><td class="colsep0 rowsep0" align="left">5.35</td><td class="colsep0 rowsep0" align="left">11,300 (<i>2496</i>)</td><td class="colsep0 rowsep0" align="left">4.95</td><td class="colsep0 rowsep0" align="left">1.0 (<i>0.2</i>)</td><td class="colsep0 rowsep0" align="left">0.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9c</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="left"><5</td><td class="colsep0 rowsep0" align="left">26,100 (<i>5940</i>)</td><td class="colsep0 rowsep0" align="left">4.58</td><td class="colsep0 rowsep0" align="left">0 (<i>0</i>)</td><td class="colsep0 rowsep0" align="left">N.A.<a class="ref internalNav" href="#t4fn9" aria-label="i">i</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9d</b></td><td class="colsep0 rowsep0" align="left">3571 (<i>1417</i>)</td><td class="colsep0 rowsep0" align="left">5.45</td><td class="colsep0 rowsep0" align="left">3890 (<i>516.2</i>)</td><td class="colsep0 rowsep0" align="left">5.41</td><td class="colsep0 rowsep0" align="left">0 (<i>0</i>)</td><td class="colsep0 rowsep0" align="left">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9e</b></td><td class="colsep0 rowsep0" align="left">4207 (<i>97</i>)</td><td class="colsep0 rowsep0" align="left">5.38</td><td class="colsep0 rowsep0" align="left">22,050 (<i>9970</i>)</td><td class="colsep0 rowsep0" align="left">4.66</td><td class="colsep0 rowsep0" align="left">0 (<i>0</i>)</td><td class="colsep0 rowsep0" align="left">0.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9f</b></td><td class="colsep0 rowsep0" align="left">3763 (<i>42.5</i>)</td><td class="colsep0 rowsep0" align="left">5.43</td><td class="colsep0 rowsep0" align="left">9700 (<i>367.7</i>)</td><td class="colsep0 rowsep0" align="left">5.01</td><td class="colsep0 rowsep0" align="left">0.5 (<i>0.7</i>)</td><td class="colsep0 rowsep0" align="left">0.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9i</b></td><td class="colsep0 rowsep0" align="left">1811 (<i>199</i>)</td><td class="colsep0 rowsep0" align="left">5.74</td><td class="colsep0 rowsep0" align="left">13,500 (<i>0.5</i>)</td><td class="colsep0 rowsep0" align="left">4.87</td><td class="colsep0 rowsep0" align="left">1.1 (<i>0.9</i>)</td><td class="colsep0 rowsep0" align="left">0.87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9k</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="left"><5</td><td class="colsep0 rowsep0" align="left">6656 (<i>82</i>)</td><td class="colsep0 rowsep0" align="left">5.18</td><td class="colsep0 rowsep0" align="left">0.14 (<i>0.2</i>)</td><td class="colsep0 rowsep0" align="left">toxic<a class="ref internalNav" href="#t4fn8" aria-label="h">h</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="left"><5</td><td class="colsep0 rowsep0" align="left">>100,000</td><td class="colsep0 rowsep0" align="left"><4</td><td class="colsep0 rowsep0" align="left">0 (<i>0</i>)</td><td class="colsep0 rowsep0" align="left">N.A.<a class="ref internalNav" href="#t4fn9" aria-label="i">i</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10b</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="left"><5</td><td class="colsep0 rowsep0" align="left">33,250 (<i>7778</i>)</td><td class="colsep0 rowsep0" align="left">4.48</td><td class="colsep0 rowsep0" align="left">0.75 (<i>0.6</i>)</td><td class="colsep0 rowsep0" align="left">N.A.<a class="ref internalNav" href="#t4fn9" aria-label="i">i</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="left"><5</td><td class="colsep0 rowsep0" align="left">19,700 (<i>282</i>)</td><td class="colsep0 rowsep0" align="left">4.71</td><td class="colsep0 rowsep0" align="left">0.25 (<i>0.3</i>)</td><td class="colsep0 rowsep0" align="left">N.A.<a class="ref internalNav" href="#t4fn9" aria-label="i">i</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="left">32 (<i>27.3</i>)</td><td class="colsep0 rowsep0" align="left">7.50</td><td class="colsep0 rowsep0" align="left">794 (<i>17.0</i>)</td><td class="colsep0 rowsep0" align="left">6.10</td><td class="colsep0 rowsep0" align="left">5.04 (<i>0.5</i>)</td><td class="colsep0 rowsep0" align="left">1.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12b</b></td><td class="colsep0 rowsep0" align="left">1025 (<i>73.2</i>)</td><td class="colsep0 rowsep0" align="left">5.99</td><td class="colsep0 rowsep0" align="left">3090 (<i>438.4</i>)</td><td class="colsep0 rowsep0" align="left">5.51</td><td class="colsep0 rowsep0" align="left">5.96 (<i>0.58</i>)</td><td class="colsep0 rowsep0" align="left">0.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12c</b></td><td class="colsep0 rowsep0" align="left">459 (<i>19.9</i>)</td><td class="colsep0 rowsep0" align="left">6.34</td><td class="colsep0 rowsep0" align="left">1350 (<i>21.2</i>)</td><td class="colsep0 rowsep0" align="left">5.87</td><td class="colsep0 rowsep0" align="left">9.59 (<i>0.66</i>)</td><td class="colsep0 rowsep0" align="left">0.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12d</b></td><td class="colsep0 rowsep0" align="left">520 (<i>29.6</i>)</td><td class="colsep0 rowsep0" align="left">6.28</td><td class="colsep0 rowsep0" align="left">2330 (<i>155.6</i>)</td><td class="colsep0 rowsep0" align="left">5.63</td><td class="colsep0 rowsep0" align="left">5.34 (<i>0.5</i>)</td><td class="colsep0 rowsep0" align="left">0.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12e</b></td><td class="colsep0 rowsep0" align="left">5826 (<i>920</i>)</td><td class="colsep0 rowsep0" align="left">5.24</td><td class="colsep0 rowsep0" align="left">9600 (<i>1590</i>)</td><td class="colsep0 rowsep0" align="left">5.02</td><td class="colsep0 rowsep0" align="left">3.93 (<i>0.76</i>)</td><td class="colsep0 rowsep0" align="left">0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12g</b></td><td class="colsep0 rowsep0" align="left">217 (<i>32.3</i>)</td><td class="colsep0 rowsep0" align="left">6.66</td><td class="colsep0 rowsep0" align="left">204 (<i>110.3</i>)</td><td class="colsep0 rowsep0" align="left">6.69</td><td class="colsep0 rowsep0" align="left">5.56 (<i>0.32</i>)</td><td class="colsep0 rowsep0" align="left">toxic<a class="ref internalNav" href="#t4fn8" aria-label="h">h</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12h</b></td><td class="colsep0 rowsep0" align="left">1938 (<i>272</i>)</td><td class="colsep0 rowsep0" align="left">5.71</td><td class="colsep0 rowsep0" align="left">2227 (<i>129</i>)</td><td class="colsep0 rowsep0" align="left">5.65</td><td class="colsep0 rowsep0" align="left">4.01 (<i>0.1</i>)</td><td class="colsep0 rowsep0" align="left">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12i</b></td><td class="colsep0 rowsep0" align="left">1985 (<i>458</i>)</td><td class="colsep0 rowsep0" align="left">5.70</td><td class="colsep0 rowsep0" align="left">10,410 (<i>1797</i>)</td><td class="colsep0 rowsep0" align="left">4.98</td><td class="colsep0 rowsep0" align="left">0.68 (<i>0.5</i>)</td><td class="colsep0 rowsep0" align="left">0.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13b</b></td><td class="colsep0 rowsep0" align="left">195 (<i>53.2</i>)</td><td class="colsep0 rowsep0" align="left">6.71</td><td class="colsep0 rowsep0" align="left">2298 (<i>277</i>)</td><td class="colsep0 rowsep0" align="left">5.64</td><td class="colsep0 rowsep0" align="left">2.1 (<i>0.7</i>)</td><td class="colsep0 rowsep0" align="left">1.07</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">N.D. = not determined.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">MM1.S proliferation assays are the average of three replicates on different days.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Antiproliferative efficacy <i>vs</i> MM1.S (GI<sub>50</sub> values) are also reported in log units, as pGI<sub>50</sub>.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">THLE-2 cytotoxicity results after 72 h of incubation are the average of at least two independent experiments performed on different days. If absolute pLC<sub>50</sub> difference was higher than 1 log unit, additional replicates were performed until satisfying the experimental error (by discarding individual results with values outside 3 MADs of the mean value).</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Cytotoxicity <i>vs</i> THLE-2 (LC<sub>50</sub> values) are also reported in log units, as pLC<sub>50</sub>.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">The PAMPA assay was performed in triplicate. Depending on permeability values (Pe, nm/s), compounds can be regarded as poor (Pe < 10 nm/s); moderate (10 < Pe < 30 nm/s), or good (>30 nm/s) permeators.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></p></div><div class="footnote" id="t4fn7"><sup><sup>g</sup></sup><p class="last">The therapeutic Window describes the difference between efficacy (pGI<sub>50</sub> values vs MM1.S) and toxicity (pLC<sub>50</sub> values vs THLE-2).</p></div><div class="footnote" id="t4fn8"><sup><sup>h</sup></sup><p class="last">Antiproliferative effects against the healthy cell line (THLE-2) are more potent than, or equal to, the tumor cell line (MM1.S): Toxicity.</p></div><div class="footnote" id="t4fn9"><sup><sup>i</sup></sup><p class="last">N.A. = not applicable (GI<sub>50</sub> or/and LC<sub>50</sub> are undetermined). Standard deviations are reported together with the corresponding experimental values (GI<sub>50</sub>, LC<sub>50</sub> and Pe) in brackets and <i>in italics</i>.</p></div></div></div><div class="NLM_p">When tested against other MM cell lines (JJN3, KMS28BM, and H929), compound <b>12a</b> also showed good antiproliferative activity with GI<sub>50</sub> values between 100 and 550 nM. <b>9a</b>, as representative of the triple inhibitors (described below), which was also tested <i>versus</i> these cell lines and showed more modest activities; in fact, GI<sub>50</sub> values are between 1 and 6 μM (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. GI<sub>50</sub> Values of Compounds <b>9a</b> and <b>12a</b><i>vs</i> Additional Cell Lines of MM<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">JJN3, GI<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">KMS28BM, GI<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">H929, GI<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="left">6035 (<i>856</i>)</td><td class="colsep0 rowsep0" align="left">1299 (<i>151</i>)</td><td class="colsep0 rowsep0" align="left">981 (<i>189</i>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="left">482 (<i>230</i>)</td><td class="colsep0 rowsep0" align="left">547 (<i>30.5</i>)</td><td class="colsep0 rowsep0" align="left">103 (<i>19.5</i>)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">These results are the average of three replicates on different days. Standard deviations are reported together with the corresponding experimental GI<sub>50</sub> values in brackets and <i>in italics</i>.</p></div></div></div><div class="NLM_p">Finally, in order to assess the functional cellular response of inhibiting these epigenetic targets (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), two chemical probes were selected as representatives: <b>9a</b>, as G9a, DNMT1 and HDACi; and, <b>12a</b>, as DNMT1 and HDACi. In fact, <b>12a</b> is the most potent molecule among dual inhibitors <i>versus</i> MM1.S. On the other hand, <b>9a</b> and <b>9i</b> are the most potent chemical probes among triple inhibitors; but, because <b>9a</b> is >0.9 log units more potent <i>versus</i> G9a than <b>9i</b>, <b>9a</b> was selected as the representative triple inhibitor to monitor its corresponding functional hallmarks. Further biochemical profiling <i>versus</i> two DNMT and seven HDAC additional isoforms was also performed (described in Table S1, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) for these two selected chemical probes, <b>9a</b> and <b>12a</b>. These results suggest that inhibition of these DNMT and HDAC additional isoforms may contribute to antiproliferative efficacy against MM1.S cell line as well as to monitor cellular functional responses.</div><div class="NLM_p">The global levels of histone-3 lysine-9 acetylation (H3K9Ac) and H3K9me2 hallmarks were monitored by Western Blot in MM1.S cells after 48 h of exposure of the selected compounds in a concentration-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) as well as in comparison with the reference compounds for each target: decitabine as DNMTi,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> panobinostat as HDACi<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> and A-366 as G9a inhibitor<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (Figure S1 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, both compounds led to an increase in H3K9Ac at concentrations above 500 nM (<b>12a</b>) and 1 μM (<b>9a</b>), such as the control compound panobinostat (Figure S1 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>). The higher dose required for compound <b>9a</b> might be due to its poorer permeability, given that both compounds are equally active against HDAC1, HDAC2, and HDAC3. In line with the lack of biochemical activity of <b>12a</b> against G9a (as reported in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), no alteration of H3K9me2 mark was detected, while this mark was significantly reduced after treatment with doses of compound <b>9a</b> above 1 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A), such as the reference compound A-366 (Figure S1 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>).</div><figure id="fig3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0006.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Changes in the epigenetic hallmarks following treatment of MM1.S cells with compounds <b>12a</b> and <b>9a</b>. (A) H3K9Ac and H3K9me2 dose–response western blots after incubation for 48 h with the indicated dose range of <b>12a</b> and <b>9a</b>. (B) <i>POU4F2</i> gene pyrosequencing after treatment with <b>12a</b> at 10 nM and <b>9a</b> at 2 μM for 5 days. All assays were performed in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">On the other hand, DNA methylation alteration was monitored in MM1.S cells by pyrosequencing analysis of several CpGs located in the promoter region of the <i>POU4F2</i> gene, which is differentially methylated in MM and involved in the TP53 pathway deregulation. As previously reported,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> doses below their IC<sub>50</sub> values and long incubation times of treating every day were used to avoid cell killing and thus to be able to monitor their impact on DNA methylation; in this case, <b>9a</b> was tested at 2 μM and <b>12a</b> at 10 nM were used for 5 days. Pyrosequencing analysis was focused on the promoter area <i>POU4F2</i> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) where treatment with compound <b>9a</b> clearly leads to hypomethylation, ∼15%, of the CpG position “3” and incubation with <b>12a</b> hypomethylates CpG positions “4” and “5” in ∼10% each. This DNA hypomethylating activity, monitored by <i>POU4F2</i> pyrosequencing, was also observed in the JJN.3 cell line treated with <b>9a</b> (at 1 μM) and <b>12a</b> (at 100 nM) for 5 days (Figure S2 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>In Vivo</i> Antitumoral Efficacy in MM</h3><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, compound <b>12a</b> exhibited a potent antiproliferative response against the MM1.S cell line (GI<sub>50</sub> = 32 nM) and an acceptable therapeutic window (>1 log unit); thus, this molecule was selected to test the efficacy in an <i>in vivo</i> xenogeneic mouse model. Compound <b>9a</b>, the selected triple inhibitor, exhibited a moderate antiproliferative response (GI<sub>50</sub> ∼ 2 μM) as well as therapeutic window (<1 log unit), an identical profile to <b>9i</b>. A preliminary PK study was performed for <b>9a</b>, but the mice (<i>n</i> = 5) died 10 min after its administration at 10 mg/kg (i.p.). Then, the dose was reduced to 5 mg/kg (i.p.); but, 15 min after administration of <b>9a</b>, during the blood extraction process, the mice (<i>n</i> = 5) also died (data not shown). Thus, any efforts for <i>in vivo</i> testing of <b>9a</b> were discontinued.</div><div class="NLM_p">Before the <i>in vivo</i> efficacy test, the corresponding PK study of the compound <b>12a</b> was done; in addition, a preliminary ADME profile was also performed (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). This ADME data show that <b>12a</b> inhibits two P450 isoforms (1A2 and 3A4) by more than 50% at 10 μM; then, P450 inhibition should be taken into account during the optimization process. Moreover, the reported ADME profiling highlights two key critical aspects: (i) <b>12a</b> metabolic stability, in human and mouse cryopreserved hepatocytes (≤50% remaining after 60 min, respectively), is not optimal and (ii) its solubility is poor (below the limit of quantification), <0.987 μg/mL.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. ADME Profile of Compound <b>12a</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center" char=".">1A2 (%)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">2C9 (%)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">2C19 (%)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">2D6 (%)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">3A4 (%)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">HH (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">HH (<i>t</i><sub>1/2</sub>)<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">HH (CLint)<a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">MH (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">MH (<i>t</i><sub>1/2</sub>)<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">MH (CLint)<a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">kinetic solubility (μg/mL)<a class="ref internalNav" href="#t6fn5" aria-label="e">e</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="char" char=".">58.8</td><td class="colsep0 rowsep0" align="char" char=".">0.0</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td><td class="colsep0 rowsep0" align="char" char=".">50.8</td><td class="colsep0 rowsep0" align="left">50.8 (<i>10</i>)</td><td class="colsep0 rowsep0" align="char" char=".">71.1</td><td class="colsep0 rowsep0" align="char" char=".">54.2</td><td class="colsep0 rowsep0" align="left">14.3 (<i>0.9</i>)</td><td class="colsep0 rowsep0" align="char" char=".">22.8</td><td class="colsep0 rowsep0" align="char" char=".">410.7</td><td class="colsep0 rowsep0" align="left"><0.987</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">% inhibition at 10 μM.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">% compound remaining after a 60 min incubation in pooled human or C57 mouse hepatocytes (HH and MH respectively); standard deviations are reported in brackets and <i>in italics</i>.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Half-life (min) in HH and MH.</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last"><i>In vivo</i> CLint (mL/min/kg) in HH and MH.</p></div><div class="footnote" id="t6fn5"><sup><sup>e</sup></sup><p class="last">Below the limit of quantification (BLQ), which is 0.987 μg/mL. All assays were performed in duplicate.</p></div></div></div><div class="NLM_p">These two factors led to suboptimal PKs. The low metabolic stability in hepatocytes translated in a high clearance and a reduced half-life when administered to BALB/c-RAG2<sup>-/-</sup>γc<sup>-/-</sup> mice for PK profiling (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>; details are described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>, Tables S2 and S3); in addition, the poor solubility of compound <b>12a</b> also precludes its use in an <i>in vivo</i> efficacy assay when resuspended in a 100% saline solution. However, if the vehicle is not only a saline solution but also DMSO and Tween 20 (10% each) then, <b>12a</b> revealed an acceptable profile with an adequate half-life (∼8.5 h) and an acceptable exposure (AUC<sub>0–24h</sub> = 916.15) to achieve <i>in vivo</i> efficacy <i>versus</i> the MM1.S cell line with GI<sub>50</sub> value of 32 nM. Compared with the PK profile of <b>12a</b> using the 100% saline solution, the formulation containing DMSO and Tween 20 leads to a half-life that is twice longer as well as to an exposure 1.5 times higher. Additional medicinal chemistry efforts are required to optimize this molecule and should focus, among others, on three critical factors: metabolic stability, solubility, and permeability (its PAMPA, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, is also poor).</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. PK Profile of Compound <b>12a</b><a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center">route</th><th class="colsep0 rowsep0" align="center">vehicle</th><th class="colsep0 rowsep0" align="center" char=".">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0–24h</sub> (nM h)</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">Cl/F (L/h)</th><th class="colsep0 rowsep0" align="center" char=".">Vz/F (L)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="left">i.p.</td><td class="colsep0 rowsep0" align="left">saline</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">591.1</td><td class="colsep0 rowsep0" align="char" char=".">3.8</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="left">i.p.</td><td class="colsep0 rowsep0" align="left">80% saline, 10% Tween20 and 10% DMSO</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">916.15</td><td class="colsep0 rowsep0" align="char" char=".">8.46</td><td class="colsep0 rowsep0" align="char" char=".">0.53</td><td class="colsep0 rowsep0" align="char" char=".">6.56</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Species: BALB/c-RAG2<sup>-/-</sup>γc<sup>-/-</sup> mice; i.p. means intraperitoneal administration, saline: NaCl 0.9%; <i>n</i> = 5 and time points: 0.25, 1, 2, 4, 8, and 24 h.</p></div></div></div><div class="NLM_p">Given the acceptable PK profile of <b>12a</b>, using saline together with 10% DMSO and 10% Tween 20 as vehicle, this compound was utilized as a pharmacological tool compound to test the <i>in vivo</i> efficacy and its mechanism of action (dual DNMT and HDAC inhibition) in terms of tumor growth in a mouse model using human MM xenografts, one of the most widely used models for the evaluation of <i>in vivo</i> efficacy of anticancer drugs.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">The mouse model showed that tumor growth is prevented using this inhibitor <b>12a</b> in tumors induced by subcutaneously injecting 10 × 10<sup>6</sup> MM1.S cells in BALB/c-RAG2<sup>-/-</sup>γc<sup>-/-</sup> mice, a model previously used in several studies.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The treatment with compound <b>12a</b> started when all mice presented tumors 12 days after myeloma cell inoculation. These mice were then treated with 10 mg/kg (i.p.) of compound <b>12a</b> administered daily during 5 consecutive days followed by 2 rest days and sacrificed at day 35 after cell inoculation. The mice were controlled for signs of morbidity (behavior and body weight loss), not observing differences in the body weight of the animals (Figure S3 in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>), and the tumor volume was monitored every 3–5 days. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, treatment with <b>12a</b> produced a significant (<i>p</i> value < 0.05) overall tumor growth inhibition, average tumor volumes at day 35 of 1788 ± 1164 and 3836 ± 1696 mm<sup>3</sup> for treated and control groups, respectively.</div><figure id="fig4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0007.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>12a</b> shows <i>in vivo</i> efficacy in an MM1.S tumor model. The tumor volume of MM1.S cells subcutaneously injected and treated with the vehicle (80% saline, 10% Tween 20, and 10% DMSO) or compound <b>12a</b> (10 mg/Kg for 5 consecutive days followed by 2 rest days) (<i>n</i> = 9). Statistical significance was calculated by a two-tailed Student’s <i>t</i>-test. *<i>p</i>-value ≤ 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47721" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47721" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We presented a detailed account of knowledge- and structure-based design of first-in-class multitarget epigenetic inhibitors targeting DNMT1, HDAC and, optionally, G9a. Synthetic approaches, schemes described below, enabled to explore three diversity points around the quinoline scaffold (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). These explorations not only identified the impact of each substitution pattern but also the role of each of the growing vectors (R1, R2, and R3) on different epigenetic targets (preliminary SAR described above). In addition, these focused explorations led to chemical probes fulfilling our initial objective (dual and triple inhibition, IC<sub>50</sub> < 10 μM against each target of interest) as well as to other compounds that only inhibit HDACs (IC<sub>50</sub> values for DNMT1 and G9a are >10 μM). Furthermore, replacement of the quinoline scaffold by quinazoline eliminates DNMT1 inhibitory activity (IC<sub>50</sub> > 10 μM), thereby validating the quinoline ring as the central core for the achievement of this multitarget epigenetic inhibition.</div><div class="NLM_p">Two chemical probes fulfilling the established target compound profiles, (i) <b>9a</b> as triple inhibitor (G9a, DNMT1 and HDAC) and (ii) <b>12a</b> as double inhibitor (DNMT1 and HDAC), were selected to validate their functional cellular responses and clearly showed the impact on their corresponding epigenetic marks: global levels of H3K9Ac and H3K9me2 as well as DNA methylation levels; assay concentrations, for both chemical probes, ranged from nanomolars to low micromolars (up to ∼1–2 μM).</div><div class="NLM_p">Taking into account that MM remains still incurable<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and that different epigenetic layers are altered in MM patients,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> a combination of HDAC and DNMT1 inhibition may be an adequate therapeutic strategy. Thus, those novel multitarget inhibitors showing potent double and triple inhibitory activities were tested <i>versus</i> the MM1.S cell line and <b>12a</b> showed a potent antiproliferative activity (GI<sub>50</sub> is 32 nM). Furthermore, pairwise comparisons between molecules of this chemical series, for example, reference G9a and DNMT1 inhibitors (<b>5</b> and <b>6</b>) as well as the most potent HDAC selective inhibitor of this chemical series (<b>13b</b>) <i>versus</i><b>12a</b>, show that dual DNMT and HDAC inhibition leads to more potent antiproliferative activity against MM1.S.</div><div class="NLM_p">Compound <b>12a</b> shows a potent antiproliferative activity, exhibits an adequate therapeutic window (>1 log unit) and, based on a special vehicle, a suitable PK was achieved. Thus, <b>12a</b> was assayed <i>in vivo</i> as a pharmacological tool compound to test its antitumor efficacy in a xenograft mouse model of human MM; a significant efficacy was achieved. However, <b>12a</b> is a chemical probe and requires an optimization process to evolve and become a new molecule with optimized drug-like properties: a lead compound. In fact, to overcome its poor ADME profile (special stress on solubility, permeability, and metabolic stability) and achieve an optimal PK, a focused exploration around R2 and R3 positions is currently <i>on-going.</i></div><div class="NLM_p last">The reported chemical probes are key tool compounds (i) to validate the feasibility of this proposal: multitarget epigenetic inhibition, (ii) to define a preliminary SAR guideline (focused on a double and triple inhibition: DNMT1, HDAC and, optionally, G9a), as well as to identify critical ADME issues (e.g., permeability and solubility). and (iii) to initiate a new drug discovery project that requires a multitarget epigenetic inhibitor as starting points (e.g., as <b>12a</b> for MM) to progress to an optimized lead molecule fulfilling the corresponding target product profile. Thus, these first-in-class molecules are paving the way for achieving multitarget epigenetic inhibition (DNMT1, HDAC, and, optionally, G9a) according to requirements that every drug discovery project may need (e.g., inhibition profile of selected targets, ADME properties, etc.)</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51448" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51448" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The preparation of target compounds is summarized in the following schemes. Compounds <b>9a–k</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) were synthesized from previously described 2,4-dichloroquinoline <b>15</b>,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> which was converted into the desired key intermediate <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> through Suzuki coupling. Then, different substitutions at the 4-position of the quinoline were installed using Buchwald–Hartwig amination conditions and esters <b>17a–k</b> were isolated. Hydrolysis of these esters led us to corresponding carboxylic acids <b>18a–k</b>. Finally, the desired hydroxamates <b>9a–k</b> were obtained by reaction with THPONH<sub>2</sub> and acidic cleavage of the protecting group.</div><div class="NLM_p">Synthesis of compounds <b>10a</b> and <b>10b</b> is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. In this case, compound <b>15</b> was reacted with methyl boronic acid and intermediate <b>20</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> was isolated. Then, amines <b>22a</b> and <b>22b</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> were prepared by reaction with <i>tert</i>-butyl 4-aminopiperidine-1-carboxylate or <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate respectively using Pd<sub>2</sub>(dba)<sub>3</sub> as the catalyst and subsequent deprotection of corresponding intermediates <b>21a</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and <b>21b</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> in acidic media. These amines were then substituted with a pyrimidyl ester to obtain compounds <b>23a</b> and <b>23b</b>. Finally, the desired hydroxamates were obtained by ester hydrolysis, synthesis of THP-protected intermediates <b>25a</b> and <b>25b</b>, and acidic deprotection.</div><div class="NLM_p">Next, compound <b>11</b> with a 4-piperidyl moiety at the 7-position was prepared as outlined in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. In this case, the synthesis started from commercially available aniline <b>26</b>, which was converted into 2,4-dichloroquinoline <b>27</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> by reaction with POCl<sub>3</sub>. Then, the piperidyl group at position 7 and 5-methyl-2-furyl group at position 2 of the quinoline were installed under standard conditions to obtain compound <b>29</b>.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Conversion of this intermediate into desired hydroxamate <b>11</b> was achieved using the similar synthetic procedure as described above (Buchwald–Hartwig amination, ester hydrolysis, synthesis of protected hydroxamic acid and removal of THP protecting group).</div><div class="NLM_p last">Finally, preparation of compounds <b>12a–k</b> and <b>13a–f</b> with a methoxy group at position 7 and following the same synthetic strategy as described for compounds above is outlined in <a class="ref internalNav" href="#sch4" aria-label="Schemes 4">Schemes 4</a> and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>. Synthetic strategy for intermediates <b>51</b>, <b>54</b>, <b>57</b>, <b>60</b>, <b>64</b>, <b>68</b>, <b>73</b> and <b>75</b> is described in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50566" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50566" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Chemistry</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure</h4><div class="NLM_p">Unless otherwise noted, all starting materials, reagents, and solvents were purchased from commercial suppliers and used without further purification. Air-sensitive reactions were conducted under N<sub>2</sub>. Flash column chromatography was performed on silica gel (230–400 mesh particle size) under standard techniques. Automated flash column chromatography was performed using ready-to-connect cartridges from Varian on irregular silica gel, particle size 15–40 μm (normal phase disposable flash columns) on a Biotage SPX flash purification system. Microwave-assisted reactions were obtained in a Biotage Smith Synthesis microwave reactor. The <sup>1</sup>H NMR spectroscopic data were recorded on a Bruker AV400 or VARIAN 400MR spectrometer with standard pulse sequences and <sup>13</sup>C NMR on a Bruker AVII-600 equipped with a 5 mm TCI cryoprobe and processed using MestreNova. <sup>1</sup>H NMR chemical shifts (δ) are reported in parts per million (ppm) and relative to the residual protons of deuterated reagents, which are corrected by tetramethylsilane (TMS). The abbreviations used to explain multiplicities are s = singlet, d = doublet, t = triplet, m = multiplet, br s = broad singlet. Coupling constants (<i>J</i>) are in hertz. HPLC-analysis was performed using a Shimadzu LC-20AB with a Luna-C18(2), 5 μm, 2.0 × 50 mm column at 40 °C and UV detection. Flow from the column was split to the MS detector (Agilent 1200, 6110MS or Agilent 1200, 6120MS Quadropole), configured with an electrospray source or API/APCI (N2 as the nebulizer gas, with the source temperature at 50 °C, ChemStation LC/MSD quad software). UHPLC-analysis was carried out using a BEH C18, 1.7 mm, 2.1 × 50 column at 40 °C and UV detection. The HPLC or UHPLC purity of all reported compounds, which were subjected to pharmacological evaluation, is ≥95% except for compound <b>42</b>, the purity of which is 94.53. No effort was put on yield optimization.</div><figure id="sch7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0014.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. (a–h) Synthesis of Intermediates: <b>51</b>, <b>54</b>, <b>57</b>, <b>60</b>, <b>64</b>, <b>68</b>, <b>73</b>, and <b>75</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Specific reaction conditions for each synthetic step are explicitly detailed in each scheme (a–h).</p></p></figure></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 2-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>9a</b>)</h4><div class="NLM_p last">A solution of compound <b>19a</b> (80 mg, 0.114 mmol) in HCl/MeOH (10 mL, 2.0 M) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 10 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9a</b> (29.6 mg, 42%) as a yellow solid; mp 126–127 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.64 (s, 2H), 7.74 (s, 1H), 7.53 (d, <i>J</i> = 3.6 Hz, 1H), 7.48 (s, 1H), 7.00 (s, 1H), 6.42 (t, <i>J</i> = 3.2 Hz, 1H), 4.35 (t, <i>J</i> = 5.6 Hz, 2H), 4.05 (s, 3H), 3.85–3.78 (m, 2H), 3.56–3.48 (m, 4H), 3.19–3.10 (m, 3H), 3.02–2.92 (m, 3H), 2.53 (s, 3H), 2.40–2.38 (m, 2H), 2.36–2.23 (m, 3H), 2.09–2.07 (m, 2H), 1.99–1.96 (m, 2H), 1.37–1.31 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. <sup>13</sup>C NMR (151 MHz, MeOD): δ 164.04, 161.50, 157.63, 156.94 (2C), 154.55, 153.66, 149.46, 144.11, 140.00, 134.50, 115.66, 113.43, 110.62, 109.59, 101.55, 100.51, 91.81, 66.67, 55.73, 54.08 (2C), 53.06, 48.38, 43.61 (2C), 35.78, 29.59 (2C), 25.22, 22.64 (2C), 12.41. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>41</sub>N<sub>7</sub>O<sub>5</sub>, 615.3; <i>m</i>/<i>z</i>: found, 616.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.01 min and purity is 98.97%. HRMS [M + H]<sup>+</sup>: calcd, 616.3242; found, 616.3235, Δ = 1.1 ppm.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]benzenecarbohydroxamic Acid (<b>9b</b>)</h4><div class="NLM_p last">A mixture of compound <b>18b</b> (80 mg, 0.155 mmol), THPONH<sub>2</sub>(27 mg, 0.232 mmol), HOBt (25 mg, 0.186 mmol), EDCI (36 mg, 0.186 mmol), and DIEA (40 mg, 0.310 mmol) in DMF (3 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 5 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 30 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9b</b> (22.4 mg, 27%) as a yellow solid; mp 100–101 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 13.36 (s, 1H), 11.22 (s, 1H), 9.43 (s, 1H), 7.89 (s, 1H), 7.77 (d, <i>J</i> = 8.4 Hz, 2H), 7.63 (s, 2H), 7.53 (d, <i>J</i> = 8.4 Hz, 2H), 6.99 (s, 1H), 6.52 (s, 1H), 4.93 (d, <i>J</i> = 6.0 Hz, 2H), 4.26–4.24 (m, 2H), 3.98 (s, 3H), 3.67–6.64 (m, 2H), 3.37–3.35 (m, 2H), 3.10–3.07 (m, 2H), 2.47 (s, 3H), 2.28–2.26 (m, 2H), 2.08–2.06 (m, 2H), 1.90–1.89 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub>, 530.2; <i>m</i>/<i>z</i>: found, 531.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.75 min and purity is 96.17%. HRMS [M + H]<sup>+</sup>: calcd, 531.2602; found, 531.2643, Δ = 7.7 ppm.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]cyclohexanecarbohydroxamic Acid (<b>9c</b>)</h4><div class="NLM_p last">A solution of compound <b>19c</b> (60 mg, 0.097 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 3 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 25 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9c</b> (9.6 mg, 18%) as a yellow solid; mp 127–128 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.73 (d, <i>J</i> = 4.4 Hz, 1H), 7.51 (d, <i>J</i> = 3.6 Hz, 1H), 7.46 (s, 1H), 6.93 (s, 1H), 6.43 (d, <i>J</i> = 3.2 Hz, 1H), 4.36–4.33 (m, 2H), 4.04 (s, 3H), 3.83 (m, 2H), 3.51–3.46 (m, 4H), 3.17 (m, 2H), 2.50 (s, 3H), 2.39–2.36 (m, 2H), 2.21 (m, 2H), 2.09–2.06 (m, 3H), 2.01–1.99 (m, 2H), 1.86–1.83 (m, 3H), 1.62–1.56 (m, 2H), 1.22–1.13 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>5</sub>, 536.3; <i>m</i>/<i>z</i>: found, 537.5 [M + H]<sup>+</sup>. HPLC (<i>method 2</i>): Rt is 3.02 min and purity is 97.84%. HRMS [M + H]<sup>+</sup>: calcd, 537.3071; found, 537.3126, Δ = 10.2 ppm.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-[[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]benzenecarbohydroxamic Acid (<b>9d</b>)</h4><div class="NLM_p last">A solution of compound <b>19d</b> (65 mg, 0.091 mmol) in HCl/EtOAc (5 mL, 2.0 M) was stirred at 20 °C for 0.5 h. Then, the solution was concentrated to give the residue, which was purified by prep-HPLC (method 31 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9d</b> (15.4 mg, 27%) as a yellow solid; mp 135–136 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.84 (d, <i>J</i> = 8.0 Hz, 2H), 7.71 (s, 1H), 7.60 (d, <i>J</i> = 8.0 Hz, 2H), 7.53 (d, <i>J</i> = 3.6 Hz, 1H), 7.47 (s, 1H), 6.98 (s, 1H), 6.42 (d, <i>J</i> = 3.6 Hz, 1H), 4.41–4.28 (m, 4H), 4.02 (s, 3H), 3.81 (s, 2H), 3.66–3.44 (m, 6H), 3.23–2.91 (m, 4H), 2.49 (s, 3H), 2.43–2.31 (m, 2H), 2.29–1.98 (m, 7H), 1.65 (d, <i>J</i> = 12.4 Hz, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>45</sub>N<sub>5</sub>O<sub>5</sub>, 627.3; <i>m</i>/<i>z</i>: found, 628.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.443 min and purity is 97.63%. HRMS [M + H]<sup>+</sup>: calcd, 628.3493; found, 628.3558, Δ = 10.3 ppm.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 4-[[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]methyl]benzenecarbohydroxamic Acid (<b>9e</b>)</h4><div class="NLM_p last">A solution of compound <b>19e</b> (80 mg, 0.115 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 2 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 32 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9e</b> (9.3 mg, 13%) as a yellow solid; mp 137–138 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.90 (d, <i>J</i> = 8.4 Hz, 2H), 7.81 (s, 1H), 7.67 (d, <i>J</i> = 8.4 Hz, 2H), 7.57 (d, <i>J</i> = 3.2 Hz, 1H), 7.50 (s, 1H), 7.07 (s, 1H), 6.44 (d, <i>J</i> = 2.8 Hz, 1H), 4.47 (s, 2H), 4.36–4.29 (m, 3H), 4.03 (s, 3H), 3.83 (s, 2H), 3.66 (s, 2H), 3.51–3.47 (m, 2H), 3.32–3.31 (m, 2H), 3.17–3.14 (m, 2H), 2.51 (s, 3H), 2.40–2.37 (m, 4H), 2.21–2.06 (m, 6H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>35</sub>H<sub>43</sub>N<sub>5</sub>O<sub>5</sub>, 613.3; <i>m</i>/<i>z</i>: found, 614.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.428 min and purity is 98.36%. HRMS [M + H]<sup>+</sup>: calcd, 614.3337; found, 614.3387, Δ = 8.1 ppm.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]benzenecarbohydroxamic Acid (<b>9f</b>)</h4><div class="NLM_p last">A solution of compound <b>19f</b> (85 mg, 0.122 mmol) in HCl/EtOAc (5 mL, 2.0 M) was stirred at 20 °C for 0.5 h. Then, the mixture was concentrated to give a residue, which was purified by prep-HPLC (method 25 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9f</b> (25.1 mg, 33%) as a yellow solid; mp 125–126 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.73 (s, 1H), 7.63 (d, <i>J</i> = 9.0 Hz, 2H), 7.53 (s, 1H), 7.47 (s, 1H), 7.02–6.95 (m, 3H), 6.42 (d, <i>J</i> = 2.0 Hz, 1H), 4.35 (s, 2H), 4.04 (s, 3H), 3.92 (d, <i>J</i> = 13.0 Hz, 2H), 3.85 (m, 2H), 3.58 (d, <i>J</i> = 6.4 Hz, 2H), 3.50 (t, <i>J</i> = 7.0 Hz, 2H), 3.18 (m, 2H), 2.88 (t, <i>J</i> = 12.0 Hz, 2H), 2.50 (s, 3H), 2.39 (d, <i>J</i> = 5.6 Hz, 2H), 2.22 (m, 2H), 2.09 (m, 3H), 1.98 (d, <i>J</i> = 12.0 Hz, 2H), 1.60–1.44 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>43</sub>N<sub>5</sub>O<sub>5</sub>, 613.3; <i>m</i>/<i>z</i>: found, 614.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.797 min and purity is 98.70%. HRMS [M + H]<sup>+</sup>: calcd, 614.3337; found, 614.3369, Δ = 5.2 ppm.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]cyclohexanecarbohydroxamic Acid (<b>9g</b>)</h4><div class="NLM_p last">A solution of compound <b>19g</b> (50 mg, 0.071 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 0.5 h. Then, the solution was concentrated to give a residue, which was purified by prep-HPLC (method 33 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9g</b> (5.3 mg, 12%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77 (s, 1H), 7.59 (d, <i>J</i> = 3.2 Hz, 1H), 7.51 (s, 1H), 6.99 (s, 1H), 6.43 (d, <i>J</i> = 3.2 Hz, 1H), 4.36 (t, <i>J</i> = 5.2 Hz, 2H), 4.04 (s, 3H), 3.83 (s, 2H), 3.64–3.60 (m, 5H), 3.58–3.49 (m, 2H), 3.18–3.09 (m, 5H), 2.50 (s, 3H), 2.40 (d, <i>J</i> = 6.0 Hz, 2H), 2.21–2.17 (m, 6H), 2.08 (d, <i>J</i> = 8.8 Hz, 5H), 1.95 (d, <i>J</i> = 8.8 Hz, 2H), 1.73–1.63 (m, 4H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>35</sub>H<sub>49</sub>N<sub>5</sub>O<sub>5</sub>, 619.4; <i>m</i>/<i>z</i>: found, 620.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.464 min and purity is 98.13%. HRMS [M + H]<sup>+</sup>: calcd, 620.3806; found, 620.3818, Δ = 1.9 ppm.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]cyclohexyl]benzenecarbohydroxamic Acid (<b>9h</b>)</h4><div class="NLM_p last">A solution of compound <b>19h</b> (15 mg, 0.022 mmol) in HCl/EtOAc (3 mL, 2.0 M) was stirred at 20 °C for 1 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 34 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9h</b> (8.5 mg, 64%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.75 (d, <i>J</i> = 6.4 Hz, 1H), 7.70–7.68 (m, 2H), 7.51 (d, <i>J</i> = 3.6 Hz, 1H), 7.47–7.46 (m, 2H), 7.43–7.33 (m, 1H), 7.00 (d, <i>J</i> = 8.0 Hz, 1H), 6.43 (d, <i>J</i> = 3.2 Hz, 1H), 4.37–4.34 (m, 2H), 4.05–4.04 (m, 3H), 3.84 (s, 2H), 3.76–3.74 (m, 1H), 3.56–3.54 (m, 1H), 3.50–3.48 (m, 2H), 3.31–3.18 (m, 2H), 2.77–2.62 (m, 1H), 2.51–2.48 (m, 3H), 2.40–2.37 (m, 2H), 2.22 (s, 3H), 2.09–2.07 (m, 2H), 1.94–1.92 (m, 2H), 1.84–1.82 (m, 4H), 1.59–1.57 (m, 1H), 1.36–1.33 (m, 1H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>36</sub>H<sub>44</sub>N<sub>4</sub>O<sub>5</sub>, 612.3; <i>m</i>/<i>z</i>: found, 613.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.127 min and purity is 99.19%. HRMS [M + H]<sup>+</sup>: calcd, 613.3384; found, 613.3408, Δ = 3.9 ppm.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]benzenecarbohydroxamic Acid (<b>9i</b>)</h4><div class="NLM_p last">A solution of compound <b>19i</b> (80 mg, 0.116 mmol) in HCl/EtOAc (5 mL, 2.0 M) was stirred at 20 °C for 2 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 10 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9i</b> (22.9 mg, 32%) as a yellow solid; mp 119–120 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77 (s, 3H), 7.67–7.65 (m, 4H), 7.55–7.53 (m, 2H), 7.46–7.38 (m, 2H), 6.91 (d, <i>J</i> = 3.6 Hz, 1H), 6.39 (s, 1H), 4.36 (s, 2H), 4.06 (s, 3H), 3.87 (s, 2H), 3.53–3.50 (m, 2H), 3.17 (s, 2H), 2.47–2.39 (m, 6H), 2.23–2.09 (m, 5H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>36</sub>H<sub>38</sub>N<sub>4</sub>O<sub>5</sub>, 606.3; <i>m</i>/<i>z</i>: found, 607.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.009 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 607.2915; found, 607.2961, Δ = 7.6 ppm.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> <i>rac-trans</i> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]cyclohexanecarbohydroxamic Acid (<b>9j-<i>trans</i></b>) and <i>rac-cis</i> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]cyclohexanecarbohydroxamic Acid (<b>9j-<i>cis</i></b>)</h4><div class="NLM_p last">A solution of compound <b>19j</b> (105 mg, 0.150 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 0.5 h. Then, the solution was concentrated and the residue was purified by prep-HPLC (method 10 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9j-<i>cis</i></b> (11.9 mg, 13%) as a yellow solid and pure compound <b>9j-<i>trans</i></b> (3.3 mg, 4%) as a yellow solid. <b>9j-<i>trans</i></b>: mp 93–94 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.78 (s, 1H), 7.46 (s, 1H), 7.40–7.34 (m, 3H), 7.31–7.29 (m, 2H), 6.95 (s, 1H), 6.40 (d, <i>J</i> = 3.4 Hz, 1H), 4.82 (s, 2H), 4.36 (t, <i>J</i> = 5.4 Hz, 2H), 4.04 (s, 3H), 3.83 (m, 2H), 3.52–3.49 (m, 2H), 3.19–3.14 (m, 2H), 2.67 (m, 1H), 2.47 (s, 3H), 2.46–2.37 (m, 3H), 2.23 (m, 3H), 2.10–1.85 (m, 6H), 1.75–1.69 (m, 4H); two exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>36</sub>H<sub>44</sub>N<sub>4</sub>O<sub>5</sub>, 612.3; <i>m</i>/<i>z</i>: found, 613.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.100 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 613.3384; found, 613.3451. <b>9j-<i>cis</i></b>: mp 134–135 °C.<sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77 (s, 1H), 7.46 (s, 1H), 7.39–7.37 (m, 3H), 7.27–7.25 (m, 2H), 6.93 (s, 1H), 6.39 (d, <i>J</i> = 3.4 Hz, 1H), 4.81 (s, 2H), 4.35 (m, 2H), 4.03 (s, 3H), 3.83 (s, 2H), 3.52–3.48 (m, 2H), 3.21–3.10 (m, 2H), 2.55 (m, 1H), 2.47 (s, 3H), 2.38 (m, 3H), 2.22 (s, 2H), 2.07 (m, 3H), 1.80–1.93 (m, 4H), 1.68 (m, 2H), 1.55–1.49 (m, 2H); two exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>36</sub>H<sub>44</sub>N<sub>4</sub>O<sub>5</sub>, 612.3; <i>m</i>/<i>z</i>: found, 613.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.040 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 613.3384; found, 613.3407, Δ = 3.8 ppm.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]benzenecarbohydroxamic Acid (<b>9k</b>)</h4><div class="NLM_p last">A solution of compound <b>19k</b> (100 mg, 0.146 mmol) in HCl/EtOAc (5 mL, 2.0 M) was stirred at 20 °C for 3 h. Then, the solution was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 5 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>9k</b> (23.4 mg, 27%) as a yellow solid; mp 138–139 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.79 (s, 1H), 7.67 (d, <i>J</i> = 8.8 Hz, 2H), 7.56 (s, 1H), 7.48 (br, 1H), 7.10 (s, 1H), 7.05 (d, <i>J</i> = 9.2 Hz, 2H), 6.43 (s, 1H), 4.35 (s, 2H), 4.24 (m, 1H), 4.07–4.02 (m, 5H), 3.83 (m, 2H), 3.51–3.48 (m, 2H), 3.18–3.12 (m, 4H), 2.51 (s, 3H), 2.38 (m, 2H), 2.21–2.18 (m, 4H), 2.07 (m, 2H), 1.95–1.92 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>34</sub>H<sub>41</sub>N<sub>5</sub>O<sub>5</sub>, 599.3; <i>m</i>/<i>z</i>: found, 600.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.862 min and purity is 98.07%. HRMS [M + H]<sup>+</sup>: calcd, 600.3180; found, 600.3245, Δ = 10.8 ppm.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-[4-[[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>10a</b>)</h4><div class="NLM_p last">A solution of compound <b>25a</b> (50 mg, 0.080 mmol) in HCl/1,4-dioxane (10 mL, 4.0 M) was stirred at room temperature for 1 h. Then, the reaction mixture was concentrated to give compound <b>10a</b> (20 mg, 46%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.67 (s, 2H), 7.69 (s, 1H), 7.19 (s, 1H), 6.66 (s, 1H), 4.30 (m, 2H), 4.04 (s, 3H), 3.47 (m, 2H), 3.09 (m, 8H), 2.65 (s, 3H), 2.26 (m, 3H), 1.96 (m, 8H), 1.34 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>39</sub>N<sub>7</sub>O<sub>4</sub>, 549.3; <i>m</i>/<i>z</i>: found, 550.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.85 min and purity is 95.59%. HRMS [M + H]<sup>+</sup>: calcd, 550.3136; found, 550.3161, Δ = 4.5 ppm.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-[4-[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>10b</b>)</h4><div class="NLM_p last">A solution of compound <b>25b</b> (60 mg, 0.097 mmol) in HCl/1,4-dioxane (10 mL, 4.0 M) was stirred at room temperature for 1 h. Then, the reaction mixture was concentrated to give the desired compound <b>10b</b> (31 mg, 60%) as a white solid; mp 192–193 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.73 (s, 2H), 7.83 (s, 1H), 7.23 (s, 1H), 6.89 (s, 1H), 4.37–4.28 (m, 3H), 4.04 (s, 3H), 3.86 (m, 2H), 3.52 (m, 2H), 3.33 (m, 4H), 3.19 (m, 2H), 2.71 (s, 3H), 2.41 (m, 2H), 2.23 (m, 4H), 2.11 (m, 2H), 1.89 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>37</sub>N<sub>7</sub>O<sub>4</sub>, 535.3; <i>m</i>/<i>z</i>: found, 536.3 [M + H]<sup>+</sup>. HPLC (<i>method 2</i>): Rt is 2.80 min and purity is 95.69%. HRMS [M + H]<sup>+</sup>: calcd, 536.2980; found, 536.3024, Δ = 8.2 ppm.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(4-piperidylmethoxy)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>11</b>)</h4><div class="NLM_p last">A solution of compound <b>32</b> (300 mg, 0.381 mmol) in HCl/EtOAc (1.0 M, 10 mL) was stirred at 15 °C for 16 h. Then, the reaction mixture was concentrated to give a residue, which was purified by prep-HPLC (method 35 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>11</b> (13 mg, 5%) as a yellow solid; mp 155–156 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.65 (s, 2H), 7.71 (s, 1H), 7.52 (d, <i>J</i> = 3.6 Hz, 1H), 7.45 (s, 1H), 6.99 (s, 1H), 6.42 (s, 1H), 4.95–4.93 (m, 1H), 4.13 (d, <i>J</i> = 5.6 Hz, 2H), 4.02 (s, 3H), 3.58–3.56 (m, 2H), 3.52–3.49 (m, 2H), 3.13–3.01 (m, 4H), 2.50 (s, 3H), 2.31–2.23 (br s, 2H), 2.18–2.14 (m, 2H), 1.99–1.96 (m, 2H), 1.79–1.73 (m, 2H), 1.40–1.30 (m, 3H); four exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>39</sub>N<sub>7</sub>O<sub>5</sub>, 601.3; <i>m</i>/<i>z</i>: found, 602.4 [M + H]<sup>+</sup>. UHPLC (<i>method 3</i>): Rt is 2.36 min and purity is 96.28%. HRMS [M + H]<sup>+</sup>: calcd, 602.3085; found, 602.3063, Δ = 3.6 ppm.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>12a</b>)</h4><div class="NLM_p last">A mixture of compound <b>38a</b> (80 mg, 0.133 mmol, 1.00 eq) in HCl/EtOAc (5.00 mL, 2M) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for 3 h under an N<sub>2</sub> atmosphere. The reaction mixture was concentrated in vacuo to give a residue. The residue was purified by prep-HPLC (method 5 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12a</b> (39.10 mg, 0.061 mmol, 45.91% yield, 98.74% purity) as a white solid; mp 136–137 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.65 (s, 2H), 7.69 (s, 1H), 7.54 (d, <i>J</i> = 3.54 Hz, 1H), 7.43 (s, 1H), 6.99 (s, 1H), 6.42 (dd, <i>J</i> = 3.54 Hz, 0.88 Hz, 1H), 4.95–4.89 (m, 2H), 4.03 (s, 3H), 4.02 (s, 3H), 3.56 (d, <i>J</i> = 7.06 Hz, 2H), 3.05–2.96 (m, 2H), 2.85–2.71 (m, 1H), 2.50 (s, 3H), 2.29–2.18 (m, 1H), 2.02–1.94 (m, 2H), 1.43–1.29 (m, 2H); two exchangeable protons were not observed. <sup>13</sup>C NMR (151 MHz, MeOD): δ 163.59, 161.07, 157.03, 156.49 (2C), 154.59, 153.89, 149.08, 143.55, 139.27, 134.07, 115.01, 113.04, 109.71, 109.03, 100.80, 98.99, 91.14, 55.08, 54.93, 47.93, 43.05 (2C), 35.14, 29.10 (2C), 11.88. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>30</sub>N<sub>6</sub>O<sub>5</sub>, 518.2; <i>m</i>/<i>z</i>: found, 519.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.37 min and purity is 98.74%. HRMS [M + H]<sup>+</sup>: calcd, 519.2361; found, 519.2335, Δ = 5.0 ppm.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-[4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>12b</b>)</h4><div class="NLM_p last">A mixture of compound <b>37b</b> (70 mg, 0.143 mmol), O-(tetrahydro-2H-pyran-2-yl) hydroxylamine hydrochloride (44 mg, 0.286 mmol), EDCI (41 mg, 0.214 mmol), DIEA (37 mg, 0.286 mmol), and HOBt (29 mg, 0.214 mmol) in DMF (5.00 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for 12 h. Then, the reaction mixture was concentrated in vacuum to give a residue, which was purified by prep-HPLC (method 15 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12b</b> (40.6 mg, 56%) as a yellow solid; mp 181–182 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.70 (s, 2H), 7.72 (s, 1H), 7.56–7.54 (m, 2H), 7.43 (s, 1H), 7.12 (s, 1H), 6.44 (d, <i>J</i> = 2.88 Hz, 1H), 5.07–4.97 (m, 2H), 4.38–4.29 (m, 1H), 4.02 (s, 3H), 3.99 (s, 3H), 3.27–3.20 (m, 2H), 2.52 (s, 3H), 2.25–2.16 (m, 2H), 1.85–1.72 (m, 2H); two exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>26</sub>H<sub>28</sub>N<sub>6</sub>O<sub>5</sub>, 504.2; <i>m</i>/<i>z</i>: found, 505.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.12 min and purity is 96.03%. HRMS [M + H]<sup>+</sup>: calcd, 505.2194; found, 505.2179, Δ = 3.0 ppm.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 2-[8-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-3-azabicyclo[3.2.1]octan-3-yl]pyrimidine-5-carbohydroxamic Acid (<b>12c</b>)</h4><div class="NLM_p last">A solution of compound <b>38c</b> (0.1 g, 0.159 mmol) in HCl/EtOAc (10 mL, 1.0 M) was stirred at 15 °C for 2 h. Then, the reaction mixture was concentrated to give a residue, which was purified by prep-HPLC (method 16 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12c</b> (15 mg, 17%) as a white solid; mp 180–181 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.69–8.65 (m, 2H), 7.76–7.74 (d, <i>J</i> = 6 Hz, 1H), 7.56–7.50 (m, 1H), 7.45 (s, 1H), 7.06–7.03 (m, 1H), 6.44–6.41 (m, 1H), 4.64–4.62 (m, 1H), 4.37–4.34 (m, 1H), 4.04–4.01 (m, 7H), 3.53–3.44 (m, 2H), 3.08–3.05 (m, 1H), 2.51–2.50 (d, <i>J</i> = 5.2 Hz, 3H), 2.46 (br s, 1H), 2.42–2.39 (m, 2H), 1.97–1.90 (m, 2H), 1.66–1.58 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>29</sub>H<sub>32</sub>N<sub>6</sub>O<sub>5</sub>, 544.2; <i>m</i>/<i>z</i>: found, 545.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.357 min and purity is 99.30%. HRMS [M + H]<sup>+</sup>: calcd, 545.2507; found, 545.2495, Δ = 2.2 ppm.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]benzenecarbohydroxamic Acid (<b>12d</b>)</h4><div class="NLM_p last">A mixture of compound <b>37d</b> (60 mg, 0.143 mmol), THPONH<sub>2</sub> (20 mg, 0.172 mmol), HOBt (23 mg, 0.172 mmol), EDCI (33 mg, 0.172 mmol), and DIEA (37 mg, 0.287 mmol) in DMF (3 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 20 °C for 3 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 10 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12d</b> (27.1 mg, 44%) as a yellow solid; mp 180–181 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 13.26 (s, 1H), 11.22 (s, 1H), 9.36 (s, 1H), 7.86 (s, 1H), 7.77 (d, <i>J</i> = 8.4 Hz, 2H), 7.59 (s, 2H), 7.53 (d, <i>J</i> = 8.0 Hz, 2H), 6.98 (s, 1H), 6.51 (d, <i>J</i> = 2.4 Hz, 1H), 4.92 (d, <i>J</i> = 5.2 Hz, 2H), 3.97 (s, 6H), 2.47 (s, 3H); one exchangeable proton was not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>23</sub>N<sub>3</sub>O<sub>5</sub>, 433.2; <i>m</i>/<i>z</i>: found, 434.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.01 min and purity is 97.66%. HRMS [M + H]<sup>+</sup>: calcd, 434.1710; found, 434.1746, Δ = 8.3 ppm.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]cyclohexanecarbohydroxamic Acid (<b>12e</b>)</h4><div class="NLM_p last">A mixture of compound <b>37e</b> (50 mg, 0.118 mmol), THPONH<sub>2</sub> (28 mg, 0.235 mmol), HOBt (19 mg, 0.141 mmol), EDCI (27 mg, 0.141 mmol), and DIEA (30 mg, 0.235 mmol) in DMF (20 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 3 h. Then, the mixture was concentrated in reduced pressure at 40 °C to give a residue, which was purified by prep-HPLC (method 10 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>12e</b> (17.2 mg, 33%) as a yellow solid; mp 148–149 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 13.20 (s, 1H), 10.39 (s, 1H), 8.75 (d, <i>J</i> = 5.6 Hz, 1H), 7.75 (s, 1H), 7.69 (d, <i>J</i> = 3.2 Hz, 1H), 7.55 (s, 1H), 6.86 (s, 1H), 6.50 (d, <i>J</i> = 3.2 Hz, 1H), 3.94 (s, 6H), 3.45–3.40 (m, 2H), 2.50–2.47 (m, 3H), 2.01–1.95 (m, 1H), 1.89–1.86 (m, 2H), 1.72–1.68 (m, 3H), 1.42–1.39 (m, 2H), 1.10–1.05 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>29</sub>N<sub>3</sub>O<sub>5</sub>, 439.2; <i>m</i>/<i>z</i>: found, 440.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.13 min and purity is 96.86%. HRMS [M + H]<sup>+</sup>: calcd, 440.2180; found, 440.2204, Δ = 5.5 ppm.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 6-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]pyridine-3-carbohydroxamic Acid (<b>12f</b>)</h4><div class="NLM_p last">A mixture of compound <b>37f</b> (50 mg, 0.119 mmol), THPONH<sub>2</sub>(28 mg, 0.238 mmol), EDCI (46 mg, 0.238 mmol), HOBt (32 mg, 0.238 mmol), and DIEA (46 mg, 0.357 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 20 °C for 16 h. Then, the reaction mixture was quenched with water (2 mL) and concentrated in vacuum to give a residue. The residue was dissolved in 0.5 M HCl aqueous solution and stirred for 30 min. Then, the reaction mixture was concentrated in vacuum to give a residue, which was purified by prep-HPLC (method 42 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12f</b> (3.5 mg, 6%) as a white solid.mp 138–139 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.92 (s, 1H), 8.17 (d, <i>J</i> = 8 Hz, 1H), 7.72 (s, 1H), 7.63 (d, <i>J</i> = 8.4 Hz, 1H), 7.45 (s, 1H), 7.38 (d, <i>J</i> = 3.6 Hz, 1H), 6.94 (s, 1H), 6.39 (d, <i>J</i> = 3.2 Hz, 1H), 5.02 (s, 2H), 4.04 (s, 6H), 2.46 (s, 3H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>22</sub>N<sub>4</sub>O<sub>5</sub>, 434.2; <i>m</i>/<i>z</i>: found, 435.2 [M + H]<sup>+</sup>. UHPLC (<i>method 3</i>): Rt is 2.35 min and purity is 96.76%. HRMS [M + H]<sup>+</sup>: calcd, 435.1663; found, 435.1639, Δ = 5.5 ppm.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 4-[[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]benzenecarbohydroxamic Acid (<b>12g</b>)</h4><div class="NLM_p last">A mixture of compound <b>37g</b> (50 mg, 0.097 mmol), THPONH<sub>2</sub> (13 mg, 0.116 mmol), HOBt (15 mg, 0.116 mmol), EDCI (22 mg, 0.116 mmol), and DIEA (25 mg, 0.194 mmol) in DMF (3 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 2 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 18 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12g</b> (21.5 mg, 41%) as a yellow solid; mp 162–163 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.85 (d, <i>J</i> = 7.6 Hz, 2H), 7.66 (s, 1H), 7.61 (d, <i>J</i> = 7.6 Hz, 2H), 7.52 (d, <i>J</i> = 3.6 Hz, 1H), 7.44 (s, 1H), 6.97 (s, 1H), 6.43 (d, <i>J</i> = 3.6 Hz, 1H), 4.37 (s, 2H), 4.03–4.02 (s, 3H), 4.02–4.01 (s, 3H), 3.63–3.50 (m, 4H), 3.09–3.03 (m, 2H), 2.50 (s, 3H), 2.18–2.14 (m, 3H), 1.66–1.63 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub>, 530.3; <i>m</i>/<i>z</i>: found, 531.4 [M + H]<sup>+</sup>. UHPLC (<i>method 3</i>): Rt is 2.04 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 531.2602; found, 531.2620, Δ = 3.4 ppm.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 5-[[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]thiophene-2-carbohydroxamic Acid (<b>12h</b>)</h4><div class="NLM_p last">A mixture of compound <b>37h</b> (60 mg, 0.115 mmol), THPONH<sub>2</sub> (16 mg, 0.138 mmol), HOBt (19 mg, 0.138 mmol), EDCI (26 mg, 0.138 mmol), and DIEA (18 mg, 0.138 mmol) in DMF (3 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 19 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12h</b> (5.0 mg, 8%) as a yellow solid; mp 165–166 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.68 (s, 1H), 7.55 (s, 1H), 7.52 (d, <i>J</i> = 3.2 Hz, 1H), 7.45 (s, 1H), 7.32 (d, <i>J</i> = 2.8 Hz, 1H), 6.97 (s, 1H), 6.42 (d, <i>J</i> = 3.2 Hz, 1H), 4.58 (s, 2H), 4.03–4.01 (m, 6H), 3.60 (m, 4H), 3.13–3.06 (m, 2H), 2.50 (s, 3H), 2.20–2.16 (m, 3H), 1.68 (s, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>32</sub>N<sub>4</sub>O<sub>5</sub>S, 536.2; <i>m</i>/<i>z</i>: found, 537.2 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.76 min and purity is 95.42%. HRMS [M + H]<sup>+</sup>: calcd, 537.2166; found, 537.2132, Δ = 6.3 ppm.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 3-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]cyclobutanecarbohydroxamic Acid (<b>12i</b>)</h4><div class="NLM_p last">A solution of compound <b>38i</b> (50 mg, 0.086 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 20 min. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 17 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12i</b> (14.7 mg, 34%) as a yellow solid; mp 150–151 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.67 (d, <i>J</i> = 3.6 Hz, 1H), 7.52 (d, <i>J</i> = 3.6 Hz, 1H), 7.44 (s, 1H), 6.97 (s, 1H), 6.42 (d, <i>J</i> = 3.6 Hz, 1H), 4.01 (d, <i>J</i> = 5.2 Hz, 6H), 3.61–3.55 (m, 4H), 2.86–2.77 (m, 3H), 2.59–2.42 (m, 8H), 2.21–2.18 (m, 3H), 1.66–1.55 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub>, 494.3; <i>m</i>/<i>z</i>: found, 495.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.644 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 495.2602; found, 495.2636, Δ = 6.9 ppm.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>rac-cis</i> 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]cyclohexanecarbohydroxamic Acid (<b>12j-<i>cis</i></b>)</h4><div class="NLM_p last">A solution of compound <b>38j-<i>cis</i></b> (30 mg, 0.059 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 3 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 20 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12j-<i>cis</i></b> (5.4 mg, 21%) as a yellow solid; mp 173–174 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.74 (s, 1H), 7.53 (d, <i>J</i> = 3.2 Hz, 1H), 7.42 (s, 1H), 7.01 (s, 1H), 6.42 (d, <i>J</i> = 2.8 Hz, 1H), 4.02 (d, <i>J</i> = 3.6 Hz, 6H), 3.97–3.94 (m, 1H), 2.51 (s, 3H), 2.23–2.16 (m, 3H), 1.96–1.80 (m, 4H), 1.68–1.63 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>23</sub>H<sub>27</sub>N<sub>3</sub>O<sub>5</sub>, 425.2; <i>m</i>/<i>z</i>: found, 426.2 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.069 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 426.2023; found, 426.2070, Δ = 11.0 ppm.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>rac-trans</i> 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]cyclohexanecarbohydroxamic Acid (<b>12j-<i>trans</i></b>)</h4><div class="NLM_p last">A solution of compound <b>38j-<i>trans</i></b> (20 mg, 0.039 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 3 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 21 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>12j-<i>trans</i></b> (3.6 mg, 21%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.80 (s, 1H), 7.49 (d, <i>J</i> = 3.2 Hz, 1H), 7.42 (s, 1H), 6.96 (s, 1H), 6.41 (d, <i>J</i> = 2.4 Hz, 1H), 4.04 (s, 1H), 4.03 (s, 6H), 2.50 (s, 3H), 2.42 (s, 1H), 2.15–2.06 (m, 4H), 1.95–1.93 (m, 2H), 1.80–1.79 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>27</sub>N<sub>3</sub>O<sub>5</sub>, 425.2; <i>m</i>/<i>z</i>: found, 426.2 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.131 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 426.2023; found, 426.2052, Δ = 6.8 ppm.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 6-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyridine-3-carbohydroxamic Acid (<b>12k</b>)</h4><div class="NLM_p last">A solution of compound <b>37k</b> (95 mg, 0.189 mmol, 1 equiv), <i>O</i>-tetrahydropyran-2-ylhydroxylamine (44.29 mg, 0.378 mmol, 2 equiv), HOBt (51.08 mg, 0.378 mmol, 2 equiv), EDCI (72.47 mg, 0.378 mmol, 2 equiv), and DIEA (73.29 mg, 0.567 mmol, 98.78 μL, 3 equiv) in DMF (5 mL) was degassed and purged with N<sub>2</sub> three times, and then the mixture was stirred at 25 °C for 12 h under an N<sub>2</sub> atmosphere. The reaction mixture was adjusted to pH ∼ 5 with aq HCl (1 M) and concentrated to give a residue, which was purified by prep-HPLC (method 17 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>). Compound <b>12k</b> was obtained as a light yellow solid: 19.4 mg, 0.038 mmol, 19.8% yield, 99.94% purity; mp 183–184 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.40 (s, 1H), 8.05 (d, <i>J</i> = 9.5 Hz, 1H), 7.71 (s, 1H), 7.52 (d, <i>J</i> = 3.4 Hz, 1H), 7.45 (s, 1H), 7.15 (d, <i>J</i> = 9.4 Hz, 1H), 6.99 (s, 1H), 6.42 (d, <i>J</i> = 2.6 Hz, 1H), 4.42 (d, <i>J</i> = 13.7 Hz, 2H), 4.03 (s, 3H), 4.02 (s, 3H), 3.59 (d, <i>J</i> = 7.0 Hz, 2H), 3.18 (t, <i>J</i> = 12.1 Hz, 2H), 2.50 (s, 3H), 2.28 (m, 1H), 2.12–2.01 (m, 2H), 1.54–1.42 (m, 2H); three exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>28</sub>H<sub>31</sub>N<sub>5</sub>O<sub>5</sub>, 517.2; <i>m</i>/<i>z</i>: found, 518.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.553 min and purity is 99.94%. HRMS [M + H]<sup>+</sup>: calcd, 518.2398; found, 518.2409, Δ = 2.1 ppm.</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 2-[4-[[(6,7-Dimethoxy-2-methyl-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>13a</b>)</h4><div class="NLM_p last">A solution of compound <b>42a</b> (300 mg, 0.559 mmol) in HCl/EtOAc (10 mL, 1.0 M) was stirred at 20 °C for 16 h. Then, the reaction mixture was concentrated in vacuum and the residue was purified by prep-HPLC (method 8 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>13a</b> (4 mg, 2%) as a yellow oil. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.65 (s, 2H), 7.68 (s, 1H), 7.13 (s, 1H), 6.68 (s, 1H), 4.95–4.93 (m, 2H), 4.01–4.00 (m, 6H), 3.49–3.47 (m, 2H), 3.02–2.96 (m, 2H), 2.64 (s, 3H), 2.21–2.17 (m, 1H), 1.96–1.93 (m, 2H), 1.37–1.30 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>28</sub>N<sub>6</sub>O<sub>4</sub>, 452.2; <i>m</i>/<i>z</i>: found, 453.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.528 min and purity is 98.28%. HRMS [M + H]<sup>+</sup>: calcd, 453.2245; found, 453.2209, Δ = 7.9 ppm.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 2-[4-[[(2-Cyclohexyl-6,7-dimethoxy-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>13b</b>)</h4><div class="NLM_p last">A solution of compound <b>42b</b> (50 mg, 0.083 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 4 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 6 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>13b</b> (9 mg, 21%) as a yellow solid; mp 155–156 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.65 (s, 2H), 7.67 (s, 1H), 7.24 (s, 1H), 6.62 (s, 1H), 4.91 (d, <i>J</i> = 13.2 Hz, 2H), 4.01 (d, <i>J</i> = 2.4 Hz, 6H), 3.51 (d, <i>J</i> = 6.8 Hz, 2H), 3.03–2.97 (m, 2H), 2.87–2.84 (m, 1H), 2.19 (m, 1H), 2.03–1.94 (m, 6H), 1.72–1.50 (m, 3H), 1.49–1.32 (m, 5H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>:calcd for C<sub>28</sub>H<sub>36</sub>N<sub>6</sub>O<sub>4</sub>, 520.3; <i>m</i>/<i>z</i>: found, 521.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.351 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 521.2871; found, 521.2837, Δ = 6.5 ppm.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2-[4-[[(6,7-Dimethoxy-2-phenyl-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>13c</b>)</h4><div class="NLM_p last">A solution of compound <b>42c</b> (60 mg, 0.100 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 4 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 7 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>13c</b> (23.3 mg, 45%) as a yellow solid; mp 165–166 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.64 (s, 2H), 7.91 (t, <i>J</i> = 5.2 Hz, 2H), 7.75 (s, 1H), 7.66–7.65 (m, 3H), 7.39 (s, 1H), 6.96 (s, 1H), 4.92–4.89 (m, 2H), 4.04 (s, 6H), 3.59 (d, <i>J</i> = 6.8 Hz, 2H), 3.00 (d, <i>J</i> = 12.0 Hz, 2H), 2.24 (s, 1H), 1.97 (d, <i>J</i> = 12.0 Hz, 2H), 1.39–1.31 (s, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>30</sub>N<sub>6</sub>O<sub>4</sub>, 514.3; <i>m</i>/<i>z</i>: found, 515.2 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.157 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 515.2401; found, 515.2363, Δ = 7.4 ppm.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-[4-[[[6,7-Dimethoxy-2-(1-piperidyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>13d</b>)</h4><div class="NLM_p last">A solution of compound <b>42d</b> (0.3 g, 0.495 mmol) in HCl/EtOAc (10 mL, 1.0 M) was stirred at 15 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue, which was purified by prep-HPLC (method 8 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>13d</b> (65.6 mg, 25%) as a white solid; mp 129–130 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.65 (m, 2H), 7.56 (s, 1H), 7.25 (s, 1H), 5.91 (s, 1H), 4.92 (s, 2H), 3.96 (s, 6H), 3.67 (s, 4H), 3.41–3.39 (m, 2H), 3.03–2.97 (m, 2H), 2.21–2.19 (br s, 1H), 1.96–1.94 (m, 2H), 1.77 (s, 6H), 1.36–1.30 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>35</sub>N<sub>7</sub>O<sub>4</sub>, 521.3; <i>m</i>/<i>z</i>: found, 522.4 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.233 min and purity is 98.77%. HRMS [M + H]<sup>+</sup>: calcd, 522.2823; found, 522.2834, Δ = 2.1 ppm.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-[4-[[[2-(2,5-dimethyl-3-furyl)-6,7-dimethoxy-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>13e</b>)</h4><div class="NLM_p last">To a solution of compound <b>42e</b> (0.6 g, 0.973 mmol, 1 equiv) in MeCN (5 mL) and water (10 mL) was added TFA (770.00 mg, 6.75 mmol, 0.5 mL, 6.94 equiv).The mixture was stirred at 60 °C for 1 h. LCMS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. MeCN was removed in vacuum, the residue was freeze-dried by the lyophilizer. Compound <b>13e</b> was obtained as a light yellow solid: 0.42 g, 0.789 mmol, 81.1% yield and 100% purity; its mp is 176–177 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.65 (s, 2H), 7.71 (s, 1H), 7.32 (s, 1H), 6.68 (s, 1H), 6.47 (s, 1H), 4.92 (br d, <i>J</i> = 13.2 Hz, 2H), 4.03 (s, 6H), 3.53 (br d, <i>J</i> = 7.1 Hz, 2H), 2.99 (br t, <i>J</i> = 11.9 Hz, 2H), 2.56 (s, 3H), 2.35 (s, 3H), 2.21 (br d, <i>J</i> = 4.2 Hz, 1H), 1.96 (br d, <i>J</i> = 11.0 Hz, 2H), 1.38–1.30 (m, 2H); three exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>32</sub>N<sub>6</sub>O<sub>5</sub>, 532.2; <i>m</i>/<i>z</i>: found, 533.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.288 min and purity is 100%. HRMS [M + H]<sup>+</sup>: calcd, 533.2518; found, 533.2549, Δ = 5.8 ppm.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-(4-(((6,7-Dimethoxy-2-(5-methylthiophen-2-yl)quinolin-4-yl)amino)methyl)piperidin-1-yl)-<i>N</i>-hydroxypyrimidine-5-carboxamide (<b>13f</b>)</h4><div class="NLM_p last">To a solution of compound <b>42f</b> (40 mg, 0.065 mmol, 1 equiv) in MeCN (5 mL) and water (5 mL) was added TFA (73.71 mg, 0.647 mmol, 47.86 μL, 10 equiv). The mixture was stirred at 60 °C for 5 min. HPLC showed that the starting material was consumed completely and one main peak was detected. The solvent was removed. The residue was purified by prep-HPLC (method 45 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>13f</b> (24.8 mg, 0.046 mmol, 71.06% yield and 98.54% purity) as an off-yellow solid; its mp is 177–178 °C. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.65 (s, 2H), 7.80 (d, <i>J</i> = 3.8 Hz, 1H), 7.70 (s, 1H), 7.39 (s, 1H), 7.03 (d, <i>J</i> = 2.8 Hz, 1H), 6.83 (s, 1H), 4.94 (br s, 2H), 4.04 (s, 3H), 4.03 (s, 3H), 3.56 (d, <i>J</i> = 6.9 Hz, 2H), 3.01 (t, <i>J</i> = 12.2 Hz, 2H), 2.62 (s, 3H), 2.22 (br s, 1H), 1.97 (d, <i>J</i> = 12.7 Hz, 2H), 1.37–1.34 (m, 2H); three exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>30</sub>N<sub>6</sub>O<sub>4</sub>S, 534.2; <i>m</i>/<i>z</i>: found, 535.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 2.186 min and purity is 98.543%. HRMS [M + H]<sup>+</sup>: calcd, 535.2122; found, 535.2138, Δ = 3.0 ppm.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)quinazolin-4-yl]amino]methyl]-1-piperidyl]pyrimidine-5-carbohydroxamic Acid (<b>14</b>)</h4><div class="NLM_p last">To a solution of <b>47</b> (130 mg, 0.215 mmol) in CH<sub>3</sub>CN/H<sub>2</sub>O (1:1, 10 mL) was added TFA (1.54 g, 13.51 mmol, 1 mL) and the mixture was stirred at 60 °C for 5 min. Then, the residue was purified by prep-HPLC (method 14 described below) to afford pure compound <b>42</b> (64.3 mg, 56%) as an off-white solid; mp 160–161 °C. <sup>1</sup>H NMR (MeOD, 400 MHz): δ 8.64 (s, 2H), 7.71 (s, 1H), 7.58 (d, <i>J</i> = 3.5 Hz, 1H), 7.35 (s, 1H), 6.47 (d, <i>J</i> = 2.5 Hz, 1H), 4.90 (br s, 2H), 4.04 (s, 3H), 4.01 (s, 3H), 3.78 (d, <i>J</i> = 6.8 Hz, 2H), 3.01 (br t, <i>J</i> = 11.5 Hz, 2H), 2.53 (s, 3H), 2.22 (br s, 1H), 1.94 (br d, <i>J</i> = 11.7 Hz, 2H), 1.39–1.31 (m, 2H); three exchangeable protons (NH from 4-aminoquinoline and NH–OH from hydroxamic group) were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>29</sub>N<sub>7</sub>O<sub>5</sub>, 519.2; <i>m</i>/<i>z</i>: found, 520.3 [M + H]<sup>+</sup>. HPLC (<i>method 1</i>): Rt is 1.897 min and purity is 94.528%. HRMS [M + H]<sup>+</sup>: calcd, 520.2303; found, 520.2309, Δ = 1.2 ppm.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Ethyl 2-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>17a</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (200 mg, 0.499 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 198 mg, 0.748 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (46 mg, 0.050 mmol), BINAP (31 mg, 0.050 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 0.998 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C to give a residue and this residue was poured into water (10 mL) and extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue. The residue was purified by prep-HPLC (method 23 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>17a</b> (80 mg, 25%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>44</sub>N<sub>6</sub>O<sub>5</sub>, 628.3; <i>m</i>/<i>z</i>: found, 629.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Ethyl 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]benzoate (<b>17b</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (100 mg, 0.249 mmol), ethyl 4-(aminomethyl)benzoate (58 mg, 0.324 mmol), Cs<sub>2</sub>CO<sub>3</sub> (162 mg, 0.499 mmol), BINAP (15 mg, 0.025 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (23 mg, 0.025 mmol) in 1,4-dioxane (3 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was filtrated and the filtrate was concentrated in reduced pressure at 40 °C to give compound <b>17b</b> (100 mg, 74%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>37</sub>N<sub>3</sub>O<sub>5</sub>, 543.3; <i>m</i>/<i>z</i>: found, 544.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Ethyl 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]cyclohexanecarboxylate (<b>17c</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (200 mg, 0.499 mmol), ethyl 4-(aminomethyl)cyclohexanecarboxylate (139 mg, 0.748 mmol), BINAP (31 mg, 0.050 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (46 mg, 0.050 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 0.998 mmol) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. The mixture was cooled to 20 °C and concentrated in reduced pressure at 40 °C. The residue was poured into water (20 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (20 mL × 3). The combined organic phase was washed with brine (20 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to afford compound <b>17c</b> (200 mg, 73%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>43</sub>N<sub>3</sub>O<sub>5</sub>, 549.3; <i>m</i>/<i>z</i>: found, 550.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Methyl 4-[[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]benzoate (<b>17d</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (140 mg, 0.349 mmol), methyl 4-[[4-(aminomethyl)-1-piperidyl]methyl]benzoate (<b>54</b>, 92 mg, 0.349 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (32 mg, 0.035 mmol), BINAP (22 mg, 0.035 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (228 mg, 0.698 mmol) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was filtered and the filtrate was concentrated. The residue was diluted with water (10 mL) and extracted with EtOAc (15 mL). The organic phase was acidized with aqueous HCl (1.0 N) to pH = 3 and the organic phase was separated. The aqueous phase was alkalized by saturated NaHCO<sub>3</sub> solution to pH = 8 and extracted with EtOAc (15 mL × 3). The combined organic layers were washed with brine (5 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford compound <b>17d</b> (220 mg, crude) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>, 626.4; <i>m</i>/<i>z</i>: found, 627.4 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Methyl 4-[[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]methyl]benzoate (<b>17e</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (200 mg, 0.499 mmol), methyl 4-[(4-amino-1-piperidyl)methyl]benzoate (<b>57</b>, 248 mg, 0.998 mmol), BINAP (31 mg, 0.050 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (46 mg, 0.050 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 0.998 mmol) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 110 °C for 16 h. The mixture was cooled to 20 °C and concentrated in reduced pressure at 40 °C. The residue was poured into water (20 mL) and extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue, which was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>17e</b> (150 mg, 49%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>44</sub>N<sub>4</sub>O<sub>5</sub>, 612.3; <i>m</i>/<i>z</i>: found, 613.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Methyl 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]benzoate (<b>17f</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (300 mg, 0.748 mmol), methyl 4-[4-(aminomethyl)-1-piperidyl]benzoate (<b>60</b>, 204 mg, 0.823 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (69 mg, 0.075 mmol), BINAP (47 mg, 0.075 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (488 mg, 1.50 mmol) in 1,4-dioxane (30 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was filtered and the filtrate was concentrated to give a residue, which was washed with a solution of PE/EtOAc = 10/1 to afford compound <b>17f</b> (220 mg, 48%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>44</sub>N<sub>4</sub>O<sub>5</sub>, 612.3; <i>m</i>/<i>z</i>: found, 613.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Ethyl 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]cyclohexanecarboxylate (<b>17g</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (500 mg, 1.25 mmol), <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (267 mg, 1.25 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (114 mg, 0.125 mmol), BINAP (78 mg, 0.125 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (813 mg, 2.49 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was filtered and the filtrate was concentrated to give a residue. The residue was washed with a solution of PE/EtOAc = 15/1 to afford intermediate <i>tert</i>-butyl 4-[[[6-methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]piperidine-1-carboxylate (250 mg, 34%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>46</sub>N<sub>4</sub>O<sub>5</sub>, 578.4; <i>m</i>/<i>z</i>: found, 579.4 [M + H]<sup>+</sup>. Then, a solution of this intermediate (250 mg, 0.432 mmol) in HCl/EtOAc (5 mL, 2.0 M) was stirred at 20 °C for 1 h. The mixture was concentrated to give the crude product. The crude product was diluted with MeOH (15 mL) and NaHCO<sub>3</sub> solid was added into the mixture to pH = 8. The mixture was filtered and the filtrate was concentrated to give intermediate 6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine(180 mg, 87%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>38</sub>N<sub>4</sub>O<sub>3</sub>, 478.3; <i>m</i>/<i>z</i>: found, 479.3 [M + H]<sup>+</sup>. Finally, to a solution of ZnCl<sub>2</sub>/diethyl ether (1.0 M, 20 μL) was added NaBH<sub>3</sub>CN (52 mg, 0.827 mmol) in MeOH (10 mL) and the mixture was stirred at 20 °C for 30 min. Then, a mixture of ethyl 4-oxocyclohexanecarboxylate (70 mg, 0.414 mmol) and intermediate 6-methoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (180 mg, 0.376 mmol) in MeOH (40 mL) was added into the reaction solution. The mixture was heated at 40 °C for 15.5 h. Then, the reaction was quenched with water (10 mL) and the mixture was filtered. The filtrate was concentrated and extracted with a solution of CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 3:1. The combined organic layers were washed with brine (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford compound <b>17g</b> (155 mg, 65%) as a white solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>52</sub>N<sub>4</sub>O<sub>5</sub>, 632.4; <i>m</i>/<i>z</i>: found, 633.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Methyl 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]cyclohexyl]benzoate (<b>17h</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (200 mg, 0.499 mmol), methyl 4-[4-(aminomethyl)cyclohexyl]benzoate (<b>64</b>, 185 mg, 0.748 mmol), BINAP (31 mg, 0.050 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (46 mg, 0.050 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 0.998 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C; the residue was poured into water (10 mL) and extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue, which was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 5:1) to afford pure compound <b>17h</b> (150 mg, 49%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>45</sub>N<sub>3</sub>O<sub>5</sub>, 609.3; <i>m</i>/<i>z</i>: found, 610.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Methyl 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]benzoate (<b>17i</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (200 mg, 0.499 mmol), methyl 4-[4-(aminomethyl)phenyl]benzoate (<b>68</b>, 180 mg, 0.748 mmol), BINAP (31 mg, 0.050 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (46 mg, 0.050 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (325 mg, 0.998 mmol) in 1,4-dioxane (15 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 110 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C; the residue was poured into water (10 mL) and extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue, which was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>17i</b> (200 mg, 66%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>39</sub>N<sub>3</sub>O<sub>5</sub>, 605.3; <i>m</i>/<i>z</i>: found, 606.2 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Ethyl 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]cyclohexanecarboxylate (<b>17j</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (400 mg, 0.998 mmol), ethyl 4-[4-(aminomethyl)phenyl]cyclohexanecarboxylate (<b>73</b>, 261 mg, 0.998 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (92 mg, 0.100 mmol), BINAP (62 mg, 0.100 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (650 mg, 2.00 mmol) in 1,4-dioxane (50 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was filtered and the filtrate was concentrated to give a residue, which was washed with a solution of PE/EtOAc = 10/1 to give compound <b>17j</b> (162 mg, 26%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>47</sub>N<sub>3</sub>O<sub>5</sub>, 625.3; <i>m</i>/<i>z</i>: found, 626.2 [M + H]<sup>+</sup>. This compound was used in the next step without further purification or characterization.</div></div><div id="sec5_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> Methyl 4-[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]benzoate (<b>17k</b>)</h4><div class="NLM_p last">A mixture of compound <b>16</b><a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> (300 mg, 0.748 mmol), methyl 4-(4-amino-1-piperidyl)benzoate (<b>75</b>, 210 mg, 0.898 mmol), BINAP (93 mg, 0.150 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (137 mg, 0.150 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (488 mg, 1.50 mmol) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 110 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C; the residue was poured into water (20 mL) and extracted with EtOAc (20 mL × 3). The combined organic phase was washed with brine (20 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue, which was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>17k</b> (200 mg, 44%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>42</sub>N<sub>4</sub>O<sub>5</sub>, 598.3; <i>m</i>/<i>z</i>: found, 599.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 2-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>18a</b>)</h4><div class="NLM_p last">To a solution of compound <b>17a</b> (100 mg, 0.155 mmol) in THF/MeOH/H<sub>2</sub>O (5:1:3, 9.0 mL) was added LiOH·H<sub>2</sub>O (10 mg, 0.233 mmol) and the mixture was stirred at 25 °C for 12 h. Then, the reaction mixture was concentrated and the pH was adjusted pH to 3–4 with 3.0 M aqueous HCl and the solid was precipitated. The solid was collected to give compound <b>18a</b> (90 mg, 94%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>40</sub>N<sub>6</sub>O<sub>5</sub>, 600.3; <i>m</i>/<i>z</i>: found, 601.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]benzoic Acid (<b>18b</b>)</h4><div class="NLM_p last">A mixture of compound <b>17b</b> (100 mg, 0.184 mmol) and LiOH·H<sub>2</sub>O (15 mg, 0.368 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 5.0 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was poured into water (2 mL) and extracted with EtOAc (3 mL). The mixture was adjusted to pH 3 with 2.0 M HCl. Then, the solution was filtered and the filter cake was concentrated to dryness to give compound <b>18b</b> (80 mg, 84%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>33</sub>N<sub>3</sub>O<sub>5</sub>, 515.2; <i>m</i>/<i>z</i>: found, 516.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]cyclohexanecarboxylic Acid (<b>18c</b>)</h4><div class="NLM_p last">A mixture of compound <b>17c</b> (200 mg, 0.364 mmol) and LiOH·H<sub>2</sub>O (45 mg, 1.09 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 5.0 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was poured into water (3 mL) and extracted with EtOAc (5 mL). The mixture was acidified to pH = 3–4 with 1 N HCl and filtered. The filter cake was concentrated to dryness to give compound <b>18c</b> (100 mg, 52%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>39</sub>N<sub>3</sub>O<sub>5</sub>, 521.3; <i>m</i>/<i>z</i>: found, 522.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 4-[[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]benzoic Acid (<b>18d</b>)</h4><div class="NLM_p last">To a mixture of compound <b>17d</b> (220 mg, 0.351 mmol) in EtOH/H<sub>2</sub>O (2:1, 6.0 mL) was added LiOH·H<sub>2</sub>O (29 mg, 0.702 mmol) and the mixture was stirred at 20 °C for 1 h. Then, the solution was concentrated to give a residue. The residue was acidized with 1.0 M aqueous HCl to pH = 3. Then, the mixture was concentrated to give compound <b>18d</b> (140 mg, 65%) as a white solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>44</sub>N<sub>4</sub>O<sub>5</sub>, 612.3; <i>m</i>/<i>z</i>: found, 613.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 4-[[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]methyl]benzoic Acid (<b>18e</b>)</h4><div class="NLM_p last">A mixture of compound <b>17e</b> (150 mg, 0.245 mmol) and LiOH·H<sub>2</sub>O (31 mg, 0.735 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 5.0 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (0.5 mL) and extracted with EtOAc (3 mL). The aqueous layer was acidified to pH = 3–4 with 1 N HCl and concentrated to dryness to afford compound <b>18e</b> (150 mg, crude) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>42</sub>N<sub>4</sub>O<sub>5</sub>, 598.3; <i>m</i>/<i>z</i>: found, 599.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]benzoic Acid (<b>18f</b>)</h4><div class="NLM_p last">To a solution of compound <b>17f</b> (220 mg, 0.359 mmol) in EtOH/H<sub>2</sub>O (2:1, 6.0 mL) was added LiOH·H<sub>2</sub>O (31 mg, 0.718 mmol) and the mixture was stirred at 20 °C for 3 h. Then, the solution was concentrated to give a residue. The residue was acidized with 1.0 M aqueous HCl to pH = 3 and the mixture was concentrated to give compound <b>18f</b> (155 mg, 72%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>42</sub>N<sub>4</sub>O<sub>5</sub>, 598.3; <i>m</i>/<i>z</i>: found, 599.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]cyclohexanecarboxylic Acid (<b>18g</b>)</h4><div class="NLM_p last">To a solution of compound <b>17g</b> (155 mg, 0.245 mmol) in EtOH/H<sub>2</sub>O (2:1, 12.0 mL) was added LiOH·H<sub>2</sub>O (21 mg, 0.490 mmol) and the mixture was stirred at 20 °C for 2 h. Then, the solution was concentrated to give a residue. The residue was acidized with 1.0 M aqueous HCl to pH = 3 and the mixture was concentrated to give compound <b>18g</b> (105 mg, 71%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>48</sub>N<sub>4</sub>O<sub>5</sub>, 604.4; <i>m</i>/<i>z</i>: found, 605.4 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]cyclohexyl]benzoic Acid (<b>18h</b>)</h4><div class="NLM_p last">A mixture of compound <b>17h</b> (150 mg, 0.245 mmol) and LiOH·H<sub>2</sub>O (31 mg, 0.735 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 5.0 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (2 mL) and extracted with EtOAc (3 mL). The aqueous layer was acidified to pH = 3–4 with 1.0 N HCl. The filter was concentrated to dryness to give a residue, which was purified by prep-HPLC (method 24 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>18h</b> (60 mg, 41%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>43</sub>N<sub>3</sub>O<sub>5</sub>, 597.3; <i>m</i>/<i>z</i>: found, 598.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]benzoic Acid (<b>18i</b>)</h4><div class="NLM_p last">A mixture of compound <b>17i</b> (200 mg, 0.330 mmol) and LiOH·H<sub>2</sub>O (41 mg, 0.990 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 5.0 mL) was stirred at 20 °C for 5 h. Then, the mixture was concentrated and the residue was diluted with water (0.5 mL) and extracted with EtOAc (3 mL). The aqueous phase was acidified to pH = 3–4 with 1.0 <i>N</i> HCl and concentrated to dryness to give compound <b>18i</b> (150 mg, 77%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>37</sub>N<sub>3</sub>O<sub>5</sub>, 591.3; <i>m</i>/<i>z</i>: found, 592.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]cyclohexanecarboxylic Acid (<b>18j</b>)</h4><div class="NLM_p last">To a solution of compound <b>17j</b> (160 mg, 0.256 mmol) in EtOH/H<sub>2</sub>O (2:1, 6.0 mL) was added LiOH·H<sub>2</sub>O (21 mg, 0.511 mmol), which was stirred at 20 °C for 2 h. Then, the mixture was acidized with 1.0 M HCl to pH = 3 and concentrated to give a residue. The residue was diluted with MeOH (5 mL), filtered, and the filtrate was concentrated to give compound <b>18j</b> (135 mg, 88%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>43</sub>N<sub>3</sub>O<sub>5</sub>, 597.3; <i>m</i>/<i>z</i>: found, 598.3 [M + H]<sup>+</sup>. This compound was used in the next step without further purification or characterization.</div></div><div id="sec5_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 4-[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]benzoic Acid (<b>18k</b>)</h4><div class="NLM_p last">A mixture of compound <b>17k</b> (200 mg, 0.334 mmol) and LiOH·H<sub>2</sub>O (42 mg, 1.00 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 5.0 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (0.5 mL) and extracted with EtOAc (3 mL). The aqueous phase was acidified to pH = 3–4 with 1.0 N HCl and concentrated to dryness to afford compound <b>18k</b> (150 mg, 76%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>40</sub>N<sub>4</sub>O<sub>5</sub>, 584.3; <i>m</i>/<i>z</i>: found, 585.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 2-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>19a</b>)</h4><div class="NLM_p last">A mixture of compound <b>18a</b> (80 mg, 0.133 mmol), DIEA (34 mg, 0.266 mmol), HOBt (21 mg, 0.160 mmol), THPONH<sub>2</sub> (23 mg, 0.200 mmol), and EDCI (30 mg, 0.160 mmol) in DMF (5.00 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C to give a residue that was poured into water (10 mL) and extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a residue. The residue was purified by prep-HPLC (method 25 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19a</b> (60 mg, 64%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>38</sub>H<sub>49</sub>N<sub>7</sub>O<sub>6</sub>, 699.4; <i>m</i>/<i>z</i>: found, 700.4 [M + H]<sup>+</sup>.This compound was used in the next step without further characterization.</div></div><div id="sec5_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> 4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-<i>N</i>-tetrahydropyran-2-yloxy-cyclohexanecarboxamide (<b>19c</b>)</h4><div class="NLM_p last">A mixture of compound <b>18c</b> (100 mg, 0.192 mmol), THPONH<sub>2</sub> (45 mg, 0.383 mmol), HOBt (31 mg, 0.230 mmol), EDCI (44 mg, 0.230 mmol), and DIEA (50 mg, 0.383 mmol) in DMF (3 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 2 h. Then, the mixture was concentrated in reduced pressure at 40 °C. The residue was poured into water (10 mL) and extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by prep-HPLC (method 26 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19c</b> (60 mg, 50%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>48</sub>N<sub>4</sub>O<sub>6</sub>, 620.4; <i>m</i>/<i>z</i>: found, 621.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 4-[[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]-<i>N</i>-tetrahydropyran-2-yloxy-benzamide (<b>19d</b>)</h4><div class="NLM_p last">A mixture of compound <b>18d</b> (140 mg, 0.229 mmol), THPONH<sub>2</sub> (30 mg, 0.251 mmol), EDCI (45 mg, 0.251 mmol), HOBt (44 mg, 0.321 mmol), and DIEA (56 mg, 0.457 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 2 h. Then, the mixture was concentrated and the residue was diluted with water (10 mL) and extracted with EtOAc (15 mL). The organic phase was concentrated to give a residue, which was purified by prep-HPLC (method 27 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19d</b> (65 mg, 40%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>53</sub>N<sub>5</sub>O<sub>6</sub>, 711.4; <i>m</i>/<i>z</i>: found, 712.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 4-[[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]methyl]-<i>N</i>-tetrahydropyran-2-yloxy-benzamide (<b>19e</b>)</h4><div class="NLM_p last">A mixture of compound <b>18e</b> (150 mg, 0.250 mmol), THPONH<sub>2</sub> (55 mg, 0.472 mmol), HOBt (38 mg, 0.283 mmol), EDCI (54 mg, 0.283 mmol), and DIEA (37 mg, 0.283 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 5 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 19 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19e</b> (80 mg, 46%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>51</sub>N<sub>5</sub>O<sub>6</sub>, 697.4; <i>m</i>/<i>z</i>: found, 698.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-benzamide (<b>19f</b>)</h4><div class="NLM_p last">A mixture of compound <b>18f</b> (120 mg, 0.200 mmol), THPONH<sub>2</sub> (26 mg, 0.220 mmol), EDCI (40 mg, 0.220 mmol), HOBt (38 mg, 0.300 mmol), and DIEA (49 mg, 0.400 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 2 h. Then, the mixture was diluted with water (10 mL) and extracted with EtOAc (15 mL). The organic phase was separated and the aqueous phase was concentrated to give the residue, which was purified by prep-HPLC (method 10 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19f</b> (85 mg, 61%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>51</sub>N<sub>5</sub>O<sub>6</sub>, 697.4; <i>m</i>/<i>z</i>: found, 698.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-cyclohexanecarboxamide (<b>19g</b>)</h4><div class="NLM_p last">A mixture of compound <b>18g</b> (105 mg, 0.174 mmol), THPONH<sub>2</sub> (22 mg, 0.191 mmol), EDCI (37 mg, 0.191 mmol), HOBt (35 mg, 0.260 mmol), and DIEA (45 mg, 0.347 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 2 h. Then, the mixture was concentrated and the residue was purified by prep-HPLC (method 18 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19g</b> (50 mg, 41%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>57</sub>N<sub>5</sub>O<sub>6</sub>, 703.4; <i>m</i>/<i>z</i>: found, 704.5 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]cyclohexyl]-<i>N</i>-tetrahydropyran-2-yloxy-benzamide (<b>19h</b>)</h4><div class="NLM_p last">A mixture of compound <b>18h</b> (60 mg, 0.1 mmol), THPONH<sub>2</sub> (20 mg, 0.168 mmol), HOBt (14 mg, 0.101 mmol), EDCI (19 mg, 0.101 mmol), and DIEA (22 mg, 0.101 mmol) in DMF (5 mL) was degassed and purged with N<sub>2</sub> 3 times and then the mixture was stirred at 20 °C for 2 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 28 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19h</b> (15 mg, 21%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>52</sub>N<sub>4</sub>O<sub>6</sub>, 696.4; <i>m</i>/<i>z</i>: found, 697.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]-<i>N</i>-tetrahydropyran-2-yloxy-benzamide (<b>19i</b>)</h4><div class="NLM_p last">A mixture of compound <b>18i</b> (150 mg, 0.253 mmol), THPONH<sub>2</sub> (56 mg, 0.477 mmol), HOBt (39 mg, 0.286 mmol), EDCI (55 mg, 0.286 mmol), and DIEA (37 mg, 0.286 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 3 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 5 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19i</b> (80 mg, 46%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>46</sub>N<sub>4</sub>O<sub>6</sub>, 690.4; <i>m</i>/<i>z</i>: found, 691.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 4-[4-[[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]phenyl]-<i>N</i>-tetrahydropyran-2-yloxy-cyclohexanecarboxamide (<b>19j</b>)</h4><div class="NLM_p last">A mixture of compound <b>18j</b> (135 mg, 0.226 mmol), THPONH<sub>2</sub> (27 mg, 0.234 mmol), EDCI (45 mg, 0.234 mmol), HOBt (43 mg, 0.319 mmol), and DIEA (55 mg, 0.425 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 2 h. Then, the mixture was concentrated to give a residue, which was purified by prep-HPLC (method 29 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19j</b> (105 mg, 67%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>41</sub>H<sub>52</sub>N<sub>4</sub>O<sub>6</sub>, 696.4; <i>m</i>/<i>z</i>: found, 697.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 4-[4-[[6-Methoxy-2-(5-methyl-2-furyl)-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-benzamide (<b>19k</b>)</h4><div class="NLM_p last">A mixture of compound <b>18k</b> (200 mg, 0.342 mmol), THPONH<sub>2</sub>(75 mg, 0.644 mmol), HOBt (52 mg, 0.387 mmol), EDCI (74 mg, 0.387 mmol), and DIEA (50 mg, 0.387 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 2 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 24 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>19k</b> (100 mg, 43%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>49</sub>N<sub>5</sub>O<sub>6</sub>, 683.4; <i>m</i>/<i>z</i>: found, 684.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 6-Methoxy-2-methyl-<i>N</i>-(4-piperidylmethyl)-7-(3-pyrrolidin-1-ylpropoxy)quinolin-4-amine (<b>22a</b>)</h4><div class="NLM_p last">A solution of compound <b>21a</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (250 mg, 0.49 mmol) in HCl/1,4-dioxane (10 mL, 4.0 M) was stirred at room temperature for 1 h. Then, the reaction mixture was concentrated to give compound <b>22a</b> (200 mg, 99%) as a white solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>36</sub>N<sub>4</sub>O<sub>2</sub>, 412.3; <i>m</i>/<i>z</i>: found, 413 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> Ethyl 2-[4-[[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>23a</b>)</h4><div class="NLM_p last">To a solution of compound <b>22a</b> (41 mg, 0.1 mmol) in acetonitrile (10 mL) were added K<sub>2</sub>CO<sub>3</sub> (69 mg, 0.5 mmol) and ethyl 2-chloropyrimidine-5-carboxylate(37 mg, 0.2 mmol) and the solution was stirred at room temperature for 3 h. Then, the mixture was concentrated to give compound <b>23a</b> (50 mg, 89%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>31</sub>H<sub>42</sub>N<sub>6</sub>O<sub>4</sub>, 562.3; <i>m</i>/<i>z</i>: found, 563 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> Ethyl 2-[4-[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]pyrimidine-5-carboxylate (<b>23b</b>)</h4><div class="NLM_p last">To a solution of compound <b>22b</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (120 mg, 0.3 mmol) in acetonitrile (15 mL) were added K<sub>2</sub>CO<sub>3</sub> (138 mg, 1 mmol) and ethyl 2-chloropyrimidine-5-carboxylate (88 mg, 0.45 mmol) and the mixture was stirred at room temperature for 3 h. Then, the mixture was concentrated to give compound <b>23b</b> (0.15 g, 91%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>40</sub>N<sub>6</sub>O<sub>4</sub>, 548.3; <i>m</i>/<i>z</i>: found, 549 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 2-[4-[[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>24a</b>)</h4><div class="NLM_p last">To a solution of compound <b>23a</b> (56 mg, 0.1 mmol) in THF/MeOH/H<sub>2</sub>O (10:1:3, 10 mL) was added LiOH·H<sub>2</sub>O (21 mg, 0.5 mmol) and the resulting mixture was stirred at room temperature overnight. Then, the mixture was diluted with water and the pH was adjusted to 2–3 with 1.0 N aqueous HCl. Then, the mixture was extracted with EtOAc and the combined organic layers were concentrated to give compound <b>24a</b> (45 mg, 83%). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>38</sub>N<sub>6</sub>O<sub>4</sub>, 534.3; <i>m</i>/<i>z</i>: found, 535 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 2-[4-[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>24b</b>)</h4><div class="NLM_p last">To a solution of compound <b>23b</b> (0.2 g, 0.365 mmol) in THF/MeOH/H<sub>2</sub>O (10:1:3, 10 mL) was added LiOH·H<sub>2</sub>O (78 mg, 1.82 mmol) and the resulting mixture was stirred at room temperature overnight. Then, the mixture was diluted with water and the pH was adjusted to 2–3 with 1.0 N aqueous HCl. Then, the mixture was extracted with EtOAc and the combined organic phase was concentrated to give compound <b>24b</b> (0.15 g, 79%). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>36</sub>N<sub>6</sub>O<sub>4</sub>, 520.3; <i>m</i>/<i>z</i>: found, 521 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 2-[4-[[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>25a</b>)</h4><div class="NLM_p last">To a solution of compound <b>24a</b> (45 mg, 0.084 mmol) in DMF (10 mL) were added EDC·HCl (29 mg, 0.17 mmol), HOBt (23 mg, 0.17 mmol), THPONH<sub>2</sub> (20 mg, 0.17 mmol), and NMM (34 mg, 0.34 mmol) and the mixture was stirred at room temperature overnight. Then, the mixture was extracted with EtOAc and the combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give a residue, which was purified by prep-HPLC (method 41 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>25a</b> (25 mg, 48%). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>47</sub>N<sub>7</sub>O<sub>5</sub>, 633.4; <i>m</i>/<i>z</i>: found, 634 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 2-[4-[[6-Methoxy-2-methyl-7-(3-pyrrolidin-1-ylpropoxy)-4-quinolyl]amino]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>25b</b>)</h4><div class="NLM_p last">To a solution of compound <b>24b</b> (104 mg, 0.2 mmol) in DMF (10 mL) were added EDC·HCl (69 mg, 0.4 mmol), HOBt (54 mg, 0.4 mmol), THPONH<sub>2</sub> (35 mg, 0.3 mmol), and NMM (61 mg, 0.6 mmol) and the mixture was stirred at room temperature overnight. Then, the mixture was extracted with EtOAc and the combined organic layers were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give a residue, which was purified by prep-HPLC (method 39 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>25b</b> (60 mg, 48%). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>45</sub>N<sub>7</sub>O<sub>5</sub>, 619.3; <i>m</i>/<i>z</i>: found, 620 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> Ethyl 2-[4-[[[7-[(1-<i>tert</i>-Butoxycarbonyl-4-piperidyl)methoxy]-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>30</b>)</h4><div class="NLM_p last">A mixture of compound <b>29</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (270 mg, 0.554 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 293 mg, 1.11 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (51 mg, 0.055 mmol), BINAP (34 mg, 0.055 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (451 mg, 1.39 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 110 °C for 16 h. Then, the reaction mixture was concentrated in vacuum and the residue was dissolved in EtOAc (50 mL) and filtered. The filtrate was concentrated in vacuum to give a residue, which was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>30</b> (200 mg, 50%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>50</sub>N<sub>6</sub>O<sub>7</sub>, 714.4; <i>m</i>/<i>z</i>: found, 715.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 2-[4-[[[7-[(1-<i>tert</i>-Butoxycarbonyl-4-piperidyl)methoxy]-6-methoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>31</b>)</h4><div class="NLM_p last">To a solution of compound <b>30</b> (200 mg, 0.280 mmol) in THF/H<sub>2</sub>O (2:1, 30.0 mL) was added LiOH·H<sub>2</sub>O (23 mg, 0.559 mmol) and the mixture was stirred at 15 °C for 16 h. Then, the reaction mixture was adjusted to pH 6 with 0.1 M HCl aqueous solution and concentrated in vacuum to obtain compound <b>31</b> (200 mg, crude) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>46</sub>N<sub>6</sub>O<sub>7</sub>, 686.3; <i>m</i>/<i>z</i>: found, 687.4 [M + H]<sup>+</sup>.This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> <i>tert</i>-Butyl 4-[[6-Methoxy-2-(5-methyl-2-furyl)-4-[[1-[5-(tetrahydropyran-2-yloxycarbamoyl)pyrimidin-2-yl]-4-piperidyl]methylamino]-7-quinolyl]oxymethyl]piperidine-1-carboxylate (<b>32</b>)</h4><div class="NLM_p last">A mixture of compound <b>31</b> (200 mg, 0.291 mmol), THPONH<sub>2</sub>(68 mg, 0.582 mmol), EDCI (112 mg, 0.582 mmol), HOBt (79 mg, 0.582 mmol), and DIEA (113 mg, 0.873 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 15 °C for 16 h. Then, the reaction mixture was quenched with water (10 mL) and concentrated in vacuum to give compound <b>32</b> (300 mg, crude) as a yellow oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>42</sub>H<sub>55</sub>N<sub>7</sub>O<sub>8</sub>, 785.5; <i>m</i>/<i>z</i>: found, 786.5 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> Ethyl 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>36a</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (100 mg, 0.329 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 104.43 mg, 0.395 mmol, 1.20 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (30.15 mg, 0.033 mmol, 0.10 equiv), BINAP (41.00 mg, 0.066 mmol, 0.20 equiv), and Cs<sub>2</sub>CO<sub>3</sub> (214.54 mg, 0.658 mmol, 2.00 equiv) in 1,4-dioxane (3.00 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 140 °C for 12 h under an N<sub>2</sub> atmosphere. The reaction mixture was filtrated and the filtrate was concentrated in vacuo to give the compound <b>36a</b> (120.00 mg, 0.225 mmol, 68.6% yield) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>33</sub>N<sub>5</sub>O<sub>5</sub>, 531.3; <i>m</i>/<i>z</i>: found, 532.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> Ethyl 2-[4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-1-piperidyl]pyrimidine-5-carboxylate (<b>36b</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (250 mg, 0.823 mmol), <i>tert</i>-butyl 4-aminopiperidine-1-carboxylate(247 mg, 1.23 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (75 mg, 0.082 mmol), BINAP (102 mg, 0.164 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (536 mg, 1.65 mmol) in 1,4-dioxane (10.0 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 12 h. Then, the reaction mixture was filtrated and the filtrate was concentrated under vacuum to give intermediate <i>tert</i>-butyl 4-[[6,7-dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]piperidine-1-carboxylate (350 mg, 91%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>33</sub>N<sub>3</sub>O<sub>5</sub>, 467.2; <i>m</i>/<i>z</i>: found, 468.3 [M + H]<sup>+</sup>. Then, a mixture of this intermediate (350 mg, 0.748 mmol) in HCl/MeOH (10.0 mL, 2.0 M) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 25 °C for 5 h. The reaction mixture was concentrated under vacuum and the residue was extracted with water (5 mL × 4) and washed with EtOAc (5 mL × 3). The combined aqueous phase was concentrated under reduced pressure to give intermediate 6,7-dimethoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidyl)quinolin-4-amine (220 mg, 80%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>25</sub>N<sub>3</sub>O<sub>3</sub>, 367.2; <i>m</i>/<i>z</i>: found, 368.3 [M + H]<sup>+</sup>. Finally, a mixture of this intermediate (100 mg, 0.272 mmol), ethyl 2-chloropyrimidine-5-carboxylate (66 mg, 0.354 mmol), and K<sub>2</sub>CO<sub>3</sub> (113 mg, 0.816 mmol) in DMF (10.0 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 50 °C for 12 h. Then, the reaction mixture was concentrated under vacuum and the residue was extracted with EtOAc (10 mL × 3). The combined organic layer was washed with water (10 mL × 1), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to afford compound <b>36b</b> (100 mg, 71%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>31</sub>N<sub>5</sub>O<sub>5</sub>, 517.2; <i>m</i>/<i>z</i>: found, 518.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> Ethyl 2-[8-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-3-azabicyclo[3.2.1]octan-3-yl]pyrimidine-5-carboxylate (<b>36c</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (200 mg, 0.658 mmol), <i>tert</i>-butyl 8-(aminomethyl)-3-azabicyclo[3.2.1]octane-3-carboxylate (190 mg, 0.790 mmol), Cs<sub>2</sub>CO<sub>3</sub> (429 mg, 1.32 mmol), BINAP (41 mg, 0.066 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (60 mg, 0.066 mmol) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C; the residue was poured into water (100 mL) and extracted with EtOAc (30 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford intermediate <i>tert</i>-butyl 8-[[[6,7-dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-3-azabicyclo[3.2.1]octane-3-carboxylate (250 mg, 75%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>37</sub>N<sub>3</sub>O<sub>5</sub>, 507.3; <i>m</i>/<i>z</i>: found, 508.3 [M + H]<sup>+</sup>. Then, a solution of this intermediate (250 mg, 0.492 mmol) in HCl/EtOAc (20 mL, 2.0 M) was stirred at 20 °C for 16 h and then, the mixture was concentrated in reduced pressure at 40 °C to afford <i>N</i>-(3-azabicyclo[3.2.1]octan-8-ylmethyl)-6,7-dimethoxy-2-(5-methyl-2-furyl)quinolin-4-amine (200 mg, 99%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>29</sub>N<sub>3</sub>O<sub>3</sub>, 407.3; <i>m</i>/<i>z</i>: found, 408.3 [M + H]<sup>+</sup>. Finally, to a solution of this amine (500 mg, 1.23 mmol) in CH<sub>3</sub>CN (20 mL) were added K<sub>2</sub>CO<sub>3</sub> (509 mg, 3.68 mmol) and ethyl 2-chloropyrimidine-5-carboxylate (343 mg, 1.84 mmol) and the mixture was stirred at 40 °C for 16 h. Then, the reaction mixture was filtered and concentrated in vacuum to give compound <b>36c</b> (0.4 g, 58%) as a yellow oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>31</sub>H<sub>35</sub>N<sub>5</sub>O<sub>5</sub>, 557.3; <i>m</i>/<i>z</i>: found, 558.2 [M + H]<sup>+</sup>.This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> Ethyl 4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]benzoate (<b>36d</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (100 mg, 0.329 mmol), ethyl 4-(aminomethyl)benzoate (77 mg, 0.428 mmol), Cs<sub>2</sub>CO<sub>3</sub> (214 mg, 0.658 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.033 mmol), and BINAP (20 mg, 0.033 mmol) in 1,4-dioxane (3 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C to afford compound <b>36d</b> (100 mg, 68%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>26</sub>N<sub>2</sub>O<sub>5</sub>, 446.2; <i>m</i>/<i>z</i>: found, 447.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Ethyl 4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]cyclohexanecarboxylate (<b>36e</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (100 mg, 0.329 mmol), ethyl 4-(aminomethyl)cyclohexanecarboxylate (122 mg, 0.659 mmol), BINAP (20 mg, 0.033 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.033 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (214 mg, 0.658 mmol) in 1,4-dioxane (5 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C to give compound <b>36e</b> (100 mg, 67%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>32</sub>N<sub>2</sub>O<sub>5</sub>, 452.2; <i>m</i>/<i>z</i>: found, 453.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> Methyl 6-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]pyridine-3-carboxylate (<b>36f</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (50 mg, 0.164 mmol), commercially available methyl 6-(aminomethyl)pyridine-3-carboxylate (55 mg, 0.329 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (30 mg, 0.033 mmol), BINAP (31 mg, 0.049 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (134 mg, 0.411 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the reaction mixture was filtered and concentrated in vacuum to give a residue. The residue was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>36f</b> (40 mg, 56%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>23</sub>N<sub>3</sub>O<sub>5</sub>, 433.2; <i>m</i>/<i>z</i>: found, 434.2 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> Ethyl 4-[[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]benzoate (<b>36g</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (500 mg, 1.65 mmol), <i>tert</i>-butyl 4-(aminomethyl)piperidine-1-carboxylate (530 mg, 2.47 mmol), Cs<sub>2</sub>CO<sub>3</sub> (1.08 g, 3.30 mmol), BINAP (102 mg, 0.165 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (151 mg, 0.165 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was poured into water (10 mL) and extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give the crude product, which was purified by silica gel column chromatography (PE/EtOAc = 1:0 to 0:1, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 1:0 to 1:1) to afford pure intermediate <i>tert</i>-butyl 4-[[[6,7-dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]piperidine-1-carboxylate (500 mg, 63%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>35</sub>N<sub>3</sub>O<sub>5</sub>, 481.3; <i>m</i>/<i>z</i>: found, 482.3 [M + H]<sup>+</sup>. Then, a solution of this intermediate (500 mg, 1.04 mmol) in HCl/MeOH (20 mL, 2.0 M) was degassed and purged with N<sub>2</sub> 3 times and then stirred at 20 °C for 5 h. Then, the mixture was concentrated in reduced pressure at 40 °C to give intermediate 6,7-dimethoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)quinolin-4-amine (400 mg, crude) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>, 381.2; <i>m</i>/<i>z</i>: found, 382.2 [M + H]<sup>+</sup>. Finally, a mixture of this amine (100 mg, 0.262 mmol), ethyl 4-formylbenzoate (93 mg, 0.524 mmol), CH<sub>3</sub>COOH (16 mg, 0.262 mmol), and NaBH<sub>3</sub>CN (49 mg, 0.786 mmol) in MeOH (3 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 70 °C for 2 h. Then, the solution was concentrated in reduced pressure at 40 °C to give a residue, which was purified by prep-HPLC (method 10 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>36g</b> (70 mg, 49%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>37</sub>N<sub>3</sub>O<sub>5</sub>, 543.3; <i>m</i>/<i>z</i>: found, 544.2 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> Ethyl 5-[[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]thiophene-2-carboxylate (<b>36h</b>)</h4><div class="NLM_p last">A mixture of 6,7-dimethoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)quinolin-4-amine (62 mg, 0.163 mmol, intermediate described in the synthesis of compound <b>36g</b>), ethyl 5-formylthiophene-2-carboxylate (60 mg, 0.325 mmol), CH<sub>3</sub>COOH (9 mg, 0.163 mmol), and NaBH<sub>3</sub>CN (30 mg, 0.488 mmol) in MeOH (3 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 70 °C for 2 h. Then, the solution was concentrated in reduced pressure at 40 °C to afford compound <b>36h</b> (60 mg, 76%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>35</sub>N<sub>3</sub>O<sub>5</sub>S, 549.2; <i>m</i>/<i>z</i>: found, 550.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> Methyl 3-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]cyclobutanecarboxylate (<b>36i</b>)</h4><div class="NLM_p last">To a solution of NaBH<sub>3</sub>CN (72 mg, 1.15 mmol) was added ZnCl<sub>2</sub> (78 mg, 0.576 mmol) in MeOH (5 mL) and the mixture was stirred at 20 °C for 30 min. Then, a mixture of 6,7-dimethoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)quinolin-4-amine(200 mg, 0.524 mmol, intermediate described in the synthesis of compound <b>36g</b>) and methyl 3-oxocyclobutanecarboxylate (73 mg, 0.576 mmol) in MeOH (10 mL) was added and the solution was stirred at 40 °C for 15.5 h. Then, the mixture was filtered and concentrated in vacuum to give compound <b>36i</b> (200 mg, 77%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>35</sub>N<sub>3</sub>O<sub>5</sub>, 493.3; <i>m</i>/<i>z</i>: found, 494.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> <i>rac-cis</i> Ethyl 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]cyclohexanecarboxylate (<b>36j-<i>cis</i></b>) and <i>rac-trans</i> Ethyl 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]cyclohexanecarboxylate (<b>36j-<i>trans</i></b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (200 mg, 0.658 mmol), ethyl 4-aminocyclohexanecarboxylate (169 mg, 0.987 mmol), Cs<sub>2</sub>CO<sub>3</sub> (429 mg, 1.32 mmol), BINAP (41 mg, 0.066 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (60 mg, 0.066 mmol) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 120 °C for 16 h. Then, the mixture was filtered and concentrated in vacuum. The residue was purified by prep-HPLC (method 11 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>36j-<i>cis</i></b> (100 mg, 34%) as a yellow solid and <b>36j-<i>trans</i></b> (50 mg, 17%) as a yellow solid. <b>36j-<i>cis</i></b>: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.74 (s, 1H), 7.52 (d, <i>J</i> = 3.2 Hz, 1H), 7.42 (s, 1H), 6.99 (s, 1H), 6.41 (d, <i>J</i> = 2.4 Hz, 1H), 4.18–4.13 (m, 2H), 4.01 (d, <i>J</i> = 2.0 Hz, 6H), 2.50 (s, 3H), 2.41–2.38 (m, 1H), 2.22–2.13 (m, 4H), 1.78–1.62 (m, 5H), 1.29–1.25 (m, 3H); one exchangeable proton was not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>30</sub>N<sub>2</sub>O<sub>5</sub>, 438.2; <i>m</i>/<i>z</i>: found, 439.2 [M + H]<sup>+</sup>. <b>36j-<i>trans</i></b>: <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.77–7.76 (m, 1H), 7.52 (d, <i>J</i> = 3.2 Hz, 1H), 7.44–7.43 (m, 1H), 7.02–7.00 (m, 1H), 6.43 (d, <i>J</i> = 2.8 Hz, 1H), 4.24–4.19 (m, 2H), 4.03 (d, <i>J</i> = 5.6 Hz, 6H), 2.76 (s, 1H), 2.52 (s, 3H), 2.32 (s, 2H), 2.29–2.24 (m, 1H), 2.05–2.00 (m, 2H), 1.85–1.74 (m, 4H), 1.33–1.29 (m, 3H); one exchangeable proton was not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>30</sub>N<sub>2</sub>O<sub>5</sub>, 438.2; <i>m</i>/<i>z</i>: found, 439.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> Methyl 6-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyridine-3-carboxylate (<b>36k</b>)</h4><div class="NLM_p last">A mixture of compound <b>35</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (0.2 g, 0.659 mmol, 1 equiv), tertbutyl 4-(aminomethyl)piperidine-1-carboxylate (282.22 mg, 1.32 mmol, 2 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (120.59 mg, 0.132 mmol, 0.2 equiv), BINAP (123.00 mg, 0.198 mmol, 0.3 equiv), and Cs<sub>2</sub>CO<sub>3</sub> (536.35 mg, 1.65 mmol, 2.5 equiv) in dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 16 h under an N<sub>2</sub> atmosphere. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The mixture was filtered and the filtrate was concentrated. The residue was purified by flash silica gel chromatography (ISCO, 20 g SepaFlash Silica Flash Column) using as eluent a 0–70% ethylacetate/petroleum ether gradient at 100 mL/min. Compound <i>tert</i>-butyl 4-[[[6,7-dimethoxy 2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]piperidine-1-carboxylate (0.2 g, 0.415 mmol, 63.07% yield) was obtained as a pure light yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>35</sub>N<sub>3</sub>O<sub>5</sub>, 481.3; <i>m</i>/<i>z</i>: found, 482.3 [M + H]<sup>+</sup>. Then, a solution of this intermediate (0.2 g, 0.415 mmol, 1 eq) in HCl/EtOAc (4.0 M, 10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for1 h under an N<sub>2</sub> atmosphere. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The solvent was removed and the pure intermediate, 6,7-dimethoxy-2-(5-methyl-2-furyl)-<i>N</i>-(4-piperidylmethyl)quinolin-4-amine (140 mg, 0.335 mmol, 80.66% yield), was obtained as a light yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>27</sub>N<sub>3</sub>O<sub>3</sub>, 381.2; <i>m</i>/<i>z</i>: found, 382.0 [M + H]<sup>+</sup>. Finally, to a mixture of this amine (140 mg, 0.335 mmol, 1 equiv) in DMF (10 mL) were added K<sub>2</sub>CO<sub>3</sub> (138.89 mg, 1.00 mmol, 3 equiv) and methyl 6-chloropyridine-3-carboxylate (68.97 mg, 0.402 mmol, 1.2 equiv). The mixture was stirred at 50 °C for 12 h. HPLC and LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The mixture was filtered and the filtrate was concentrated to give a residue, which was purified by prep-HPLC (method 43 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford the pure compound <b>36k</b> (0.1 g, 0.194 mmol, 57.8% yield) as a light yellow solid.<sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.66 (d, <i>J</i> = 1.9 Hz, 1H), 8.03 (dd, <i>J</i> = 2.4, 9.2 Hz, 1H), 7.69 (s, 1H), 7.51 (d, <i>J</i> = 3.3 Hz, 1H), 7.44 (s, 1H), 6.99 (s, 1H), 6.87 (d, <i>J</i> = 9.3 Hz, 1H), 6.42 (dd, <i>J</i> = 0.9, 3.5 Hz, 1H), 4.54 (d, <i>J</i> = 13.1 Hz, 2H), 4.04 (s, 3H), 4.03 (s, 3H), 3.86 (s, 3H), 3.57 (d, <i>J</i> = 7.2 Hz, 2H), 3.08–2.97 (m, 2H), 2.50 (s, 3H), 2.23 (m, 1H), 1.99 (br d, <i>J</i> = 10.9 Hz, 2H), 1.47–1.34 (m, 2H); one exchangeable proton was not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>32</sub>N<sub>4</sub>O<sub>5</sub>, 516.2; <i>m</i>/<i>z</i>: found, 517.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>37a</b>)</h4><div class="NLM_p last">A mixture of compound <b>36a</b> (120.00 mg, 0.226 mmol, 1.00 equiv) and LiOH·H<sub>2</sub>O (18.94 mg, 0.452 mmol, 2.00 equiv) in THF/MeOH/H<sub>2</sub>O (10.00/1.00/1.00 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for 12 h under an N<sub>2</sub> atmosphere. The reaction mixture was concentrated under reduced pressure to remove THF and MeOH to give a residue. The residue was diluted with water 3 mL and quenched by 2 M HCl to adjust the pH to 4. Then, the yellow solid was precipitated and collected. The yellow solid was concentrated in vacuo to give the compound <b>37a</b> (90.00 mg, 0.179 mmol, 79.2% yield) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>29</sub>N<sub>5</sub>O<sub>5</sub>, 503.2; <i>m</i>/<i>z</i>: found, 504.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_89" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> 2-[4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>37b</b>)</h4><div class="NLM_p last">A mixture of compound <b>36b</b> (100 mg, 0.193 mmol) and LiOH·H<sub>2</sub>O (24 mg, 0.579 mmol) in THF/MeOH/H<sub>2</sub>O (6:1:1, 4.00 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 25 °C for 12 h. Then, the reaction mixture was concentrated in vacuum and the residue was adjusted to pH 4 with 4.0 M HCl. The obtained solid was filtrated and collected to afford compound <b>37b</b> (70 mg, 74%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>27</sub>N<sub>5</sub>O<sub>5</sub>, 489.2; <i>m</i>/<i>z</i>: found, 490.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_90" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> 2-[8-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-3-azabicyclo[3.2.1]octan-3-yl]pyrimidine-5-carboxylic Acid (<b>37c</b>)</h4><div class="NLM_p last">To a solution of compound <b>36c</b> (0.4 g, 0.717 mmol) in THF/H<sub>2</sub>O (2:1, 30 mL) was added LiOH·H<sub>2</sub>O (45 mg, 1.08 mmol) and the mixture was stirred at 15 °C for 2 h. Then, the pH was adjusted to 6 with 1.0 M HCl aqueous solution and the mixture was concentrated in vacuum to afford compound <b>37c</b> (0.4 g, crude) as a brown solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>31</sub>N<sub>5</sub>O<sub>5</sub>, 529.2; <i>m</i>/<i>z</i>: found, 530.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_91" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> 4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]benzoic Acid (<b>37d</b>)</h4><div class="NLM_p last">A mixture of compound <b>36d</b> (100 mg, 0.224 mmol) and LiOH·H<sub>2</sub>O (19 mg, 0.448 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 2.5 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 20 °C for 5 h. Then, the mixture was concentrated, the residue was diluted with water (2 mL) and the residue was extracted with EtOAc (3 mL). The aqueous phase was adjusted pH to 3–4 with 2.0 M HCl and the obtained yellow solid was precipitated and filtrated. The filter cake was concentrated to dryness to give compound <b>37d</b> (60 mg, 64%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>22</sub>N<sub>2</sub>O<sub>5</sub>, 418.2; <i>m</i>/<i>z</i>: found, 419.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_92" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> 4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]cyclohexanecarboxylic Acid (<b>37e</b>)</h4><div class="NLM_p last">A mixture of compound <b>36e</b> (100 mg, 0.221 mmol) and LiOH·H<sub>2</sub>O (18 mg, 0.442 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 5.0 mL) was degassed and purged with N<sub>2</sub> for 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with H<sub>2</sub>O (2 mL) and extracted with EtOAc (5 mL). Then pH of the aqueous phase was adjusted to 3–4 with 2.0 M HCl. The obtained solid was filtered and concentrated to dryness to give compound <b>37e</b> (50 mg, 53%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>28</sub>N<sub>2</sub>O<sub>5</sub>, 424.2; <i>m</i>/<i>z</i>: found, 425.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_93" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> 6-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]pyridine-3-carboxylic Acid (<b>37f</b>)</h4><div class="NLM_p last">To a solution of compound <b>36f</b> (40 mg, 0.092 mmol) in THF/H<sub>2</sub>O (2:1, 15 mL) was added LiOH·H<sub>2</sub>O (8 mg, 0.184 mmol) and the mixture was stirred at 20 °C for 16 h. Then, the reaction mixture was adjust to pH 6 and concentrated in vacuum to give compound <b>37f</b> (50 mg, crude) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>21</sub>N<sub>3</sub>O<sub>5</sub>, 419.2; <i>m</i>/<i>z</i>: found, 420.2 [M + H]<sup>+</sup>.This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_94" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> 4-[[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]benzoic Acid (<b>37g</b>)</h4><div class="NLM_p last">A mixture of compound <b>36g</b> (70 mg, 0.129 mmol) and LiOH·H<sub>2</sub>O (11 mg, 0.257 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 2.5 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (0.5 mL) and extracted with EtOAc (3 mL). Then aqueous phase was acidified with 2.0 M HCl to adjust pH to 3–4. The solution was filtered and the filter cake was concentrated to dryness to give compound <b>37g</b> (50 mg, 75%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>33</sub>N<sub>3</sub>O<sub>5</sub>, 515.3; <i>m</i>/<i>z</i>: found, 516.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_95" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> 5-[[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]methyl]thiophene-2-carboxylic Acid (<b>37h</b>)</h4><div class="NLM_p last">A mixture of compound <b>36h</b> (100 mg, 0.182 mmol) and LiOH·H<sub>2</sub>O (23 mg, 0.545 mmol) in THF/MeOH/H<sub>2</sub>O (3:1:1, 2.5 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (2 mL) and extracted with EtOAc (3 mL). The aqueous layer was acidified to pH = 3–4 with 1 N HCl and filtered. The filter cake was concentrated to dryness to give compound <b>37h</b> (60 mg, 63%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>31</sub>N<sub>3</sub>O<sub>5</sub>S, 521.2; <i>m</i>/<i>z</i>: found, 522.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_96" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> 3-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]cyclobutanecarboxylic Acid (<b>37i</b>)</h4><div class="NLM_p last">A mixture of compound <b>36i</b> (200 mg, 0.405 mmol) and LiOH·H<sub>2</sub>O (51 mg, 1.22 mmol) in MeOH/THF/H<sub>2</sub>O (1:3:1, 15.0 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with H<sub>2</sub>O (2 mL) and extracted with EtOAc (3 mL). Then the aqueous layer was acidified to pH = 3–4 with 1 N HCl and filtered. The filter cake was concentrated to dryness to give compound <b>37i</b> (100 mg, 51%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>33</sub>N<sub>3</sub>O<sub>5</sub>, 479.2; <i>m</i>/<i>z</i>: found, 480.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_97" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> <i>rac-cis</i> 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]cyclohexanecarboxylic Acid (<b>37j-<i>cis</i></b>)</h4><div class="NLM_p last">A mixture of compound <b>36j-<i>cis</i></b> (70 mg, 0.160 mmol) and LiOH·H<sub>2</sub>O (20 mg, 0.479 mmol) in MeOH/THF/H<sub>2</sub>O (1:3.1, 10.0 mL) was stirred at 20 °C for 3 h. Then, the mixture was concentrated and the residue was diluted with water (3 mL) and extracted with EtOAc (5 mL). Then the aqueous layer was acidified to pH = 3–4 with 1 N HCl and filtered. The filter was concentrated to dryness to afford compound <b>37j-<i>cis</i></b> (60 mg, 91%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>26</sub>N<sub>2</sub>O<sub>5</sub>, 410.2; <i>m</i>/<i>z</i>: found, 411.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_98" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> <i>rac-trans</i> 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]cyclohexanecarboxylic Acid (<b>37j-<i>trans</i></b>)</h4><div class="NLM_p last">A mixture of compound <b>36j-<i>trans</i></b> (50 mg, 0.114 mmol) and LiOH·H<sub>2</sub>O (14 mg, 0.342 mmol) in MeOH/THF/H<sub>2</sub>O (1:3:1, 8.0 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (3 mL) and extracted with EtOAc (5 mL). Then the aqueous layer was acidified to pH = 3–4 with 1 N HCl and filtered. The filter was concentrated to dryness to give compound <b>37j-<i>trans</i></b> (50 mg, crude) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>26</sub>N<sub>2</sub>O<sub>5</sub>, 410.2; <i>m</i>/<i>z</i>: found, 411.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> 6-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyridine-3-carboxylic Acid (<b>37k</b>)</h4><div class="NLM_p last">To a solution of compound <b>36k</b> (0.1 g, 0.194 mmol, 1 equiv) in THF (5 mL) and water (5 mL) was added LiOH·H<sub>2</sub>O (24.37 mg, 0.581 mmol, 3 equiv). The mixture was stirred at 20 °C for 12 h. The mixture was adjusted pH ∼ 6 with HCl (2 M) and extracted with ethyl acetate (10 mL, three times). The combined organic phase was washed with brine (10 mL), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered and concentrated in vacuum. Compound <b>37k</b> (95 mg, 0.189 mmol, 97.7% yield) was obtained as a light yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>30</sub>N<sub>4</sub>O<sub>5</sub>, 502.2; <i>m</i>/<i>z</i>: found, 503.3 [M + H]<sup>+</sup>.This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_100" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>38a</b>)</h4><div class="NLM_p last">A mixture of compound <b>37a</b> (120.00 mg, 0.238 mmol, 1.00 equiv), <i>O</i>-tetrahydropyran-2-ylhydroxylamine (54.91 mg, 0.358 mmol, 1.50 equiv—as hydrochloride), HOBt (48.30 mg, 0.358 mmol, 1.50 equiv), EDCI (68.53 mg, 0.358 mmol, 1.50 equiv) and DIEA (61.60 mg, 0.477 mmol, 83.24 μL, 2.00 equiv) in DMF (10.00 mL) was degassed and purged with N<sub>2</sub> for 3 times, and then the mixture was stirred at 25 °C for 12 h under N<sub>2</sub> atmosphere. LC–MS showed 70% of desired compound was detected. The reaction mixture was filtrated and the filtrate was concentrated in vacuo to give a residue. The residue was extracted with EtOAc (20 mL, 3 times) and washed with water (10 mL, twice). The combined organic layers were washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue. The residue was purified by HPLC (method 2 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>38a</b> (80.00 mg, 0.133 mmol, 55.7% yield) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>38</sub>N<sub>6</sub>O<sub>6</sub>, 602.3; <i>m</i>/<i>z</i>: found, 603.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_101" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> 2-[8-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-3-azabicyclo[3.2.1]octan-3-yl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>38c</b>)</h4><div class="NLM_p last">A mixture of compound <b>37c</b> (200 mg, 0.378 mmol), THPONH<sub>2</sub> (88 mg, 0.755 mmol), EDCI (109 mg, 0.566 mmol), HOBt (76 mg, 0.566 mmol), and DIEA (146 mg, 1.13 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 15 °C for 16 h. Then, the reaction mixture was poured into H<sub>2</sub>O (100 mL) and extracted with EtOAc (20 mL × 3). The organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to obtain compound <b>38c</b> (0.1 g, 42%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>40</sub>N<sub>6</sub>O<sub>6</sub>, 628.3; <i>m</i>/<i>z</i>: found, 629.3 [M + H]<sup>+</sup>.This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_102" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> 3-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-cyclobutanecarboxamide (<b>38i</b>)</h4><div class="NLM_p last">A mixture of compound <b>37i</b> (90 mg, 0.188 mmol), THPONH<sub>2</sub> (44 mg, 0.375 mmol), DIEA (48 mg, 0.375 mmol), HOBt (30 mg, 0.225 mmol), and EDCI (43 mg, 0.225 mmol) in DMF (5 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C, the residue was poured into water (10 mL), and it was extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by prep-HPLC (method 12 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>38i</b> (50 mg, 46%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>42</sub>N<sub>4</sub>O<sub>6</sub>, 578.3; <i>m</i>/<i>z</i>: found, 579.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further characterization.</div></div><div id="sec5_1_103" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> <i>rac-cis</i> 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-<i>N</i>-tetrahydropyran-2-yloxy-cyclohexanecarboxamide (<b>38j-<i>cis</i></b>)</h4><div class="NLM_p last">A mixture of compound <b>37j-<i>cis</i></b> (60 mg, 0.146 mmol), THPONH<sub>2</sub>(34 mg, 0.292 mmol), DIEA (38 mg, 0.292 mmol), HOBt (24 mg, 0.175 mmol), and EDCI (34 mg, 0.175 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 13 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford compound <b>38j-<i>cis</i></b> (30 mg, 40%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>35</sub>N<sub>3</sub>O<sub>6</sub>, 509.3; <i>m</i>/<i>z</i>: found, 510.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further characterization.</div></div><div id="sec5_1_104" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> <i>rac-trans</i> 4-[[6,7-Dimethoxy-2-(5-methyl-2-furyl)-4-quinolyl]amino]-<i>N</i>-tetrahydropyran-2-yloxy-cyclohexanecarboxamide (<b>38j-<i>trans</i></b>)</h4><div class="NLM_p last">A mixture of compound <b>37j-<i>trans</i></b> (50 mg, 0.122 mmol), THPONH<sub>2</sub>(29 mg, 0.244 mmol), DIEA (31 mg, 0.244 mmol), HOBt (20 mg, 0.146 mmol), and EDCI (28 mg, 0.146 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times and the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 3 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>38j-<i>trans</i></b> (20 mg, 32%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>35</sub>N<sub>3</sub>O<sub>6</sub>, 509.3; <i>m</i>/<i>z</i>: found, 510.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further characterization.</div></div><div id="sec5_1_105" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> 4-Chloro-6,7-dimethoxy-2-(1-piperidyl)quinoline (<b>39d</b>)</h4><div class="NLM_p last">A mixture of compound <b>34</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (1.00 g, 3.87 mmol), piperidine (330 mg, 3.87 mmol), Cs<sub>2</sub>CO<sub>3</sub> (2.52 g, 7.75 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (355 mg, 0.387 mmol), and BINAP (241 mg, 0.387 mmol) in 1,4-dioxane (100 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 48 °C and the residue was poured into water (200 mL). The aqueous phase was extracted with EtOAc (100 mL × 3) and the combined organic phase was concentrated in reduced pressure at 48 °C to afford compound <b>39d</b> (300 mg, 25%) as a yellow solid, which was used in the next step without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.56 (s, 1H), 7.47 (s, 1H), 7.42 (s, 1H), 4.02 (s, 3H), 3.98 (s, 3H), 3.84 (s, 4H), 1.82 (s, 6H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>2</sub>, 306.1; <i>m</i>/<i>z</i>: found, 307.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_106" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> 4-Chloro-2-(2,5-dimethyl-3-furyl)-6,7-dimethoxy-quinoline (<b>39e</b>)</h4><div class="NLM_p last">A mixture of compound <b>34</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (2 g, 7.75 mmol, 1 equiv), 2-(2,5-dimethyl-3-furyl)-4,4,5,5 tetramethyl-1,3,2-dioxaborolane (1.72 g, 7.75 mmol, 1 equiv), K<sub>2</sub>CO<sub>3</sub> (2.14 g, 15.50 mmol, 2 equiv), Pd(PPh<sub>3</sub>)<sub>4</sub> (895.44 mg, 0.775 mmol, 0.1 equiv) in dioxane (20 mL), and water (5 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 80 °C for 2 h under an N<sub>2</sub> atmosphere. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The mixture was filtered and the filtrate was concentrated in vacuum. The residue was purified by flash silica gel chromatography (ISCO, 40 g SepaFlash Silica Flash Column) using as eluent a 0–15% ethylacetate/petroleum ether gradient at 100 mL/min. Compound <b>39e</b> was obtained as a light yellow solid: 1 g, 3.15 mmol, 40.61% yield. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>17</sub>H<sub>16</sub>ClNO<sub>3</sub>, 317.1; <i>m</i>/<i>z</i>: found, 318.1 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_107" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> Ethyl 2-[4-[[(6,7-Dimethoxy-2-methyl-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>40a</b>)</h4><div class="NLM_p last">A mixture of compound <b>39a</b><a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> (200 mg, 0.841 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 334 mg, 1.26 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (154 mg, 0.168 mmol), BINAP (157 mg, 0.252 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (548 mg, 1.68 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 48 h. Then, the reaction mixture was filtered and concentrated in vacuum to give a residue. The residue was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>40a</b> (230 mg, 59%) as a yellow oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>31</sub>N<sub>5</sub>O<sub>4</sub>, 465.2; <i>m</i>/<i>z</i>: found, 466.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_108" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> Ethyl 2-[4-[[(2-Cyclohexyl-6,7-dimethoxy-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>40b</b>)</h4><div class="NLM_p last">A mixture of compound <b>39g</b><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (300 mg, 0.987 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 313 mg, 1.19 mmol), Cs<sub>2</sub>CO<sub>3</sub> (643 mg, 1.98 mmol), BINAP (61 mg, 0.099 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (90 mg, 0.099 mmol) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the mixture was filtered and concentrated in vacuum. The residue was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure intermediate ethyl 2-[4-[[[2-(cyclohexen-1-yl)-6,7-dimethoxy-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (300 mg, 57%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>37</sub>N<sub>5</sub>O<sub>4</sub>, 531.3; <i>m</i>/<i>z</i>: found, 532.4 [M + H]<sup>+</sup>. To a solution of this intermediate (300 mg, 0.564 mmol) in MeOH (20 mL) was added Pd/C (10%, 30 mg) under an H<sub>2</sub> atmosphere. The suspension was degassed and purged with H<sub>2</sub> 3 times. The mixture was stirred under H<sub>2</sub> (15 Psi) at 20 °C for 16 h. Then, the mixture was filtered and concentrated in vacuum to afford compound <b>40b</b> (200 mg, 66%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>39</sub>N<sub>5</sub>O<sub>4</sub>, 533.3; <i>m</i>/<i>z</i>: found, 534.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_109" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> Ethyl 2-[4-[[(6,7-Dimethoxy-2-phenyl-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>40c</b>)</h4><div class="NLM_p last">A mixture of compound <b>39c</b><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (150 mg, 0.500 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 145 mg, 0.550 mmol), Cs<sub>2</sub>CO<sub>3</sub> (326 mg, 1.00 mmol), BINAP (31 mg, 0.050 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (45 mg, 0.050 mmol) in 1,4-dioxane (15 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 100 °C for 16 h. Then, the mixture was filtered and concentrated in vacuum. The residue was purified by prep-TLC (SiO<sub>2</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>40c</b> (200 mg, 76%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>33</sub>N<sub>5</sub>O<sub>4</sub>, 527.3; <i>m</i>/<i>z</i>: found, 528.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_110" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> Ethyl 2-[4-[[[6,7-Dimethoxy-2-(1-piperidyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>40d</b>)</h4><div class="NLM_p last">A mixture of compound <b>39d</b> (250 mg, 0.815 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 323 mg, 1.22 mmol), Pd<sub>2</sub>(dba)<sub>3</sub> (75 mg, 0.081 mmol), BINAP (51 mg, 0.081 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (531 mg, 1.63 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was purified by prep-TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10:1) to afford pure compound <b>40d</b> (270 mg, 62%) as a yellow solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.78 (s, 2H), 7.37 (s, 1H), 7.07 (s, 1H), 6.92 (s, 1H), 4.35–4.29 (m, 2H), 3.92 (s, 6H), 3.72 (s, 1H), 3.58 (br s, 4H), 3.30–3.27 (m, 2H), 3.04–2.98 (m, 2H), 2.22 (s, 2H), 1.98–1.95 (m, 2H), 1.87–1.86 (m, 1H), 1.70 (s, 6H), 1.36 (s, 3H), 1.31–1.27 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>38</sub>N<sub>6</sub>O<sub>4</sub>, 534.3; <i>m</i>/<i>z</i>: found, 535.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_111" class="NLM_sec NLM_sec_level_3"><div id="ac_i138" class="anchor-spacer"></div><h4 class="article-section__title" id="_i138"> Ethyl 2-[4-[[[2-(2,5-Dimethyl-3-furyl)-6,7-dimethoxy-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>40e</b>)</h4><div class="NLM_p last">A mixture of compound <b>39e</b> (1 g, 3.15 mmol, 1 equiv), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 1.23 g, 4.09 mmol, 1.3 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (576.35 mg, 0.629 mmol, 0.2 equiv), BINAP (587.86 mg, 0.944 mmol, 0.3 equiv), and Cs<sub>2</sub>CO<sub>3</sub> (3.08 g, 9.44 mmol, 3 equiv) in 1,4-dioxane (20 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 12 h under an N<sub>2</sub> atmosphere. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The mixture was filtered and the filtrate was concentrated in vacuum. The residue was purified by flash silica gel chromatography (ISCO, 40 g SepaFlash Silica Flash Column) using as eluent a 0–100% ethylacetate/petroleum ether gradient at 100 mL/min. Compound <b>40e</b> was obtained as a light yellow solid: 0.8 g, 1.47 mmol, 46.6% yield. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>30</sub>H<sub>35</sub>N<sub>5</sub>O<sub>5</sub>, 545.3; <i>m</i>/<i>z</i>: found, 546.2 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_112" class="NLM_sec NLM_sec_level_3"><div id="ac_i139" class="anchor-spacer"></div><h4 class="article-section__title" id="_i139"> Ethyl 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-thienyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>40f</b>)</h4><div class="NLM_p last">A mixture of compound <b>39f</b><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> (0.17 g, 0.532 mmol, 1 equiv), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 207.86 mg, 0.691 mmol, 1.3 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (97.35 mg, 0.106 mmol, 0.2 equiv), BINAP (99.3 mg, 0.160 mmol, 0.3 equiv), and Cs<sub>2</sub>CO<sub>3</sub> (519.6 mg, 1.59 mmol, 3 equiv) indioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 120 °C for 12 h under an N<sub>2</sub> atmosphere. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The mixture was filtered and the filtrate was concentrated. The residue was purified by flash silica gel chromatography (ISCO, 12 g SepaFlash Silica Flash Column) using as eluent a 0–80% ethylacetate/petroleum ether gradient at 50 mL/min. Compound <b>40f</b> was obtained as a light yellow solid (0.15 g, 0.274 mmol, 51.5% yield), which was used in the next step without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.85 (s, 2H), 7.44 (br s, 1H), 7.37 (s, 1H), 6.86 (s, 1H), 6.79–6.78 (m, 1H), 6.75 (s, 1H), 4.99 (d, <i>J</i> = 13.2 Hz, 2H), 4.75 (br s, 1H), 4.35 (q, <i>J</i> = 7.2 Hz, 2H), 4.02 (s, 6H), 3.34 (t, <i>J</i> = 6.2 Hz, 2H), 3.03–2.97 (m, 2H), 2.55 (s, 3H), 2.15–2.05 (m, 1H), 2.01 (d, <i>J</i> = 11.9 Hz, 2H), 1.38 (t, <i>J</i> = 7.2 Hz, 5H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>29</sub>H<sub>33</sub>N<sub>5</sub>O<sub>4</sub>S, 547.2; <i>m</i>/<i>z</i>: found, 548.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_113" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> 2-[4-[[(6,7-Dimethoxy-2-methyl-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>41a</b>)</h4><div class="NLM_p last">To a solution of compound <b>40a</b> (230 mg, 0.494 mmol) in THF/H<sub>2</sub>O (2:1, 30 mL) was added LiOH·H<sub>2</sub>O (41 mg, 0.988 mmol) and the mixture was stirred at 20 °C for 16 h. Then, the reaction mixture was adjusted to pH 6 and concentrated in vacuum to give compound <b>41a</b> (300 mg, crude) as a yellow solid that was used in the next step without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.91 (s, 2H), 7.76 (s, 1H), 7.20 (s, 1H), 6.69 (s, 1H), 4.82–4.80 (m, 2H), 4.02–4.00 (m, 6H), 3.53–3.51 (m, 2H), 3.28–3.21 (m, 2H), 2.66 (s, 3H), 2.35 (s, 1H), 2.08–2.05 (m, 2H), 1.50–1.48 (m, 2H); two exchangeable protons were not observed. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>27</sub>N<sub>5</sub>O<sub>4</sub>, 437.2; <i>m</i>/<i>z</i>: found, 438.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_114" class="NLM_sec NLM_sec_level_3"><div id="ac_i141" class="anchor-spacer"></div><h4 class="article-section__title" id="_i141"> 2-[4-[[(2-Cyclohexyl-6,7-dimethoxy-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>41b</b>)</h4><div class="NLM_p last">A mixture of compound <b>40b</b> (200 mg, 0.375 mmol) and LiOH·H<sub>2</sub>O (47 mg, 1.12 mmol) in MeOH/THF/H<sub>2</sub>O (1:3:1, 5 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (2 mL) and extracted with EtOAc (3 mL). Then, the aqueous phase was acidified to pH = 3–4 with 1 N HCl and filtered. The filter cake was concentrated to dryness to give compound <b>41b</b> (120 mg, 63%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>35</sub>N<sub>5</sub>O<sub>4</sub>, 505.3; <i>m</i>/<i>z</i>: found, 506.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_115" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> 2-[4-[[(6,7-Dimethoxy-2-phenyl-4-quinolyl)amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>41c</b>)</h4><div class="NLM_p last">A mixture of compound <b>40c</b> (200 mg, 0.379 mmol) and LiOH·H<sub>2</sub>O (47 mg, 1.14 mmol) in MeOH/THF/H<sub>2</sub>O (1:3:1, 8 mL) was stirred at 20 °C for 16 h. Then, the mixture was concentrated and the residue was diluted with water (5 mL) and extracted with EtOAc (10 mL). The aqueous phase was acidified to pH = 3–4 with 1 N HCl and filtered. The filter cake was concentrated to dryness to afford compound <b>41c</b> (150 mg, 79%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>29</sub>N<sub>5</sub>O<sub>4</sub>, 499.2; <i>m</i>/<i>z</i>: found, 500.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_116" class="NLM_sec NLM_sec_level_3"><div id="ac_i143" class="anchor-spacer"></div><h4 class="article-section__title" id="_i143"> 2-[4-[[[6,7-Dimethoxy-2-(1-piperidyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>41d</b>)</h4><div class="NLM_p last">To a solution of compound <b>40d</b> (270 mg, 0.505 mmol) in THF/H<sub>2</sub>O (2:1, 15 mL) was added LiOH·H<sub>2</sub>O (32 mg, 0.757 mmol) and the mixture was stirred at 15 °C for 16 h. Then, the pH was adjusted to 5 with 2.0 M HCl aqueous solution and the mixture was concentrated in vacuum to obtain crude compound <b>41d</b> (300 mg, crude) as a yellow oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>34</sub>N<sub>6</sub>O<sub>4</sub>, 506.3; <i>m</i>/<i>z</i>: found, 507.3 [M + H]<sup>+</sup>.This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_117" class="NLM_sec NLM_sec_level_3"><div id="ac_i144" class="anchor-spacer"></div><h4 class="article-section__title" id="_i144"> 2-[4-[[[2-(2,5-Dimethyl-3-furyl)-6,7-dimethoxy-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>41e</b>)</h4><div class="NLM_p last">To a solution of <b>40e</b> (0.8 g, 1.47 mmol, 1 equiv) in THF (10 mL)and water (5 mL) was added LiOH·H<sub>2</sub>O (307.61 mg, 7.33 mmol, 5 equiv).The mixture was stirred at 25 °C for 12 h. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The mixture was adjusted to pH ∼ 5 with aqueous HCl (2 M) at room temperature. Some precipitate was formed and, after filtration, the solid was collected to afford compound <b>41e</b> as a light yellow solid: 0.6 g, 1.16 mmol, 79.1% yield. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>31</sub>N<sub>5</sub>O<sub>5</sub>, 517.2; <i>m</i>/<i>z</i>: found, 518.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_118" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-thienyl)-4-quinolyl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>41f</b>)</h4><div class="NLM_p last">To a solution of <b>40f</b> (150 mg, 0.274 mmol, 1 equiv) in THF (8 mL) and water (4 mL) was added LiOH·H<sub>2</sub>O (57.46 mg, 1.37 mmol, 5 equiv). The mixture was stirred at 25 °C for 12 h. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The mixture was adjusted to pH ∼ 5 with aqueous HCl (2 M) at room temperature and then filtered. Compound <b>41f</b> was obtained as a yellow solid: 120 mg, 0.231 mmol, 84.3% yield. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>29</sub>N<sub>5</sub>O<sub>4</sub>S, 519.2; <i>m</i>/<i>z</i>: found, 520.2 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_119" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> 2-[4-[[(6,7-Dimethoxy-2-methyl-4-quinolyl)amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>42a</b>)</h4><div class="NLM_p last">A mixture of compound <b>41a</b> (300 mg, 0.685 mmol), THPONH<sub>2</sub> (161 mg, 1.37 mmol), EDCI (263 mg, 1.37 mmol), HOBt (185 mg, 1.37 mmol), and DIEA (265 mg, 2.06 mmol) in DMF (30 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 20 °C for 16 h. Then, the reaction mixture was quenched with water (10 mL) and concentrated in vacuum to give a residue. The residue was extracted with EtOAc (3 mL × 3) and the organic layers were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuum to give compound <b>42a</b> (300 mg, 82%) as a yellow oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>36</sub>N<sub>6</sub>O<sub>5</sub>, 536.3; <i>m</i>/<i>z</i>: found, 537.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_120" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> 2-[4-[[(2-Cyclohexyl-6,7-dimethoxy-4-quinolyl)amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>42b</b>)</h4><div class="NLM_p last">A mixture of compound <b>41b</b> (120 mg, 0.237 mmol), THPONH<sub>2</sub> (56 mg, 0.475 mmol), DIEA (61 mg, 0.475 mmol), HOBt (38 mg, 0.285 mmol), and EDCI (55 mg, 0.295 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 3 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>42b</b> (50 mg, 35%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>44</sub>N<sub>6</sub>O<sub>5</sub>, 604.3; <i>m</i>/<i>z</i>: found, 605.4 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_121" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> 2-[4-[[(6,7-Dimethoxy-2-phenyl-4-quinolyl)amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>42c</b>)</h4><div class="NLM_p last">A mixture of compound <b>41c</b> (150 mg, 0.300 mmol), THPONH<sub>2</sub> (70 mg, 0.600 mmol), DIEA (77 mg, 0.600 mmol), HOBt (48 mg, 0.360 mmol), and EDCI (69 mg, 0.360 mmol) in DMF (15 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C and the residue was purified by prep-HPLC (method 4 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure compound <b>42c</b> (60 mg, 33%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>38</sub>N<sub>6</sub>O<sub>5</sub>, 598.3; <i>m</i>/<i>z</i>: found, 599.3 [M + H]<sup>+</sup>. This compound was used in the next step without further characterization.</div></div><div id="sec5_1_122" class="NLM_sec NLM_sec_level_3"><div id="ac_i149" class="anchor-spacer"></div><h4 class="article-section__title" id="_i149"> 2-[4-[[[6,7-Dimethoxy-2-(1-piperidyl)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>42d</b>)</h4><div class="NLM_p last">A mixture of compound <b>41d</b> (300 mg, 0.592 mmol), THPONH<sub>2</sub> (139 mg, 1.18 mmol), EDCI (227 mg, 1.18 mmol), HOBt (160 mg, 1.18 mmol), and DIEA (229 mg, 1.78 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 15 °C for 16 h. Then, the reaction mixture was quenched with water (5 mL) and concentrated in vacuum to give a residue. The residue was dissolved in 20 mL of CH<sub>3</sub>CN (20 mL) and filtered. The filtrate was concentrated in vacuum to afford compound <b>42d</b> (500 mg, crude) as a brown oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>43</sub>N<sub>7</sub>O<sub>5</sub>, 605.3; <i>m</i>/<i>z</i>: found, 606.4 [M + H]<sup>+</sup>.This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_123" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> 2-[4-[[[2-(2,5-Dimethyl-3-furyl)-6,7-dimethoxy-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>42e</b>)</h4><div class="NLM_p last">A mixture of compound <b>41e</b> (0.6 g, 1.16 mmol, 1 equiv), <i>O</i>-tetrahydropyran-2-ylhydroxylamine (271.60 mg, 2.32 mmol, 2 equiv), HOBt (313.28 mg, 2.32 mmol, 2 equiv), EDCI (444.46 mg, 2.32 mmol, 2 equiv), and DIEA (749.11 mg, 5.80 mmol, 1.01 mL, 5 equiv) in DMF (8 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for 12 h under an N<sub>2</sub> atmosphere. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The residue was poured into ice-water (w/w = 1/1) (30 mL). The aqueous phase was extracted with ethyl acetate (20 mL, three times). The combined organic phase was washed with brine (30 mL), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, concentrated in vacuum, and purified by prep-HPLC (method 44 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford the pure compound <b>42e</b>, as a light yellow solid: 0.6 g, 0.973 mmol, 83.9% yield. ESI-MS <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>40</sub>N<sub>6</sub>O<sub>6</sub>, 616.3; <i>m</i>/<i>z</i>: found, 617.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_124" class="NLM_sec NLM_sec_level_3"><div id="ac_i151" class="anchor-spacer"></div><h4 class="article-section__title" id="_i151"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-thienyl)-4-quinolyl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>42f</b>)</h4><div class="NLM_p last">A mixture of compound <b>41f</b> (50 mg, 0.096 mmol, 1 equiv), <i>O</i>-tetrahydropyran-2-ylhydroxylamine (22.54 mg, 0.192 mmol, 2 equiv), HOBt (26.0 mg, 0.192 mmol, 2 equiv), EDCI (36.89 mg, 0.192 mmol, 2 equiv), and DIEA (62.18 mg, 0.481 mmol, 83.8 μL, 5 equiv) in DMF (5 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for 12 h under an N<sub>2</sub> atmosphere. LC–MS showed that the reaction was completed and one main peak with the desired <i>m</i>/<i>z</i> was detected. The residue was poured into ice-water (w/w = 1/1) (6 mL). The aqueous phase was extracted with ethyl acetate (5 mL, 3 times). The combined organic phase was washed with brine (10 mL), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. Compound <b>42f</b> was obtained as a light yellow solid (40mg). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>32</sub>H<sub>38</sub>N<sub>6</sub>O<sub>5</sub>S, 618.3; <i>m</i>/<i>z</i>: found, 619.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_125" class="NLM_sec NLM_sec_level_3"><div id="ac_i152" class="anchor-spacer"></div><h4 class="article-section__title" id="_i152"> Ethyl 2-[4-[[(2-Chloro-6,7-dimethoxy-quinazolin-4-yl)amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>44</b>)</h4><div class="NLM_p last">A mixture of commercially available 2,4-dichloro-6,7-dimethoxy-quinazoline (<b>43</b>) (0.2 g, 0.772 mmol), ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>, 232 mg, 0.772 mmol, HCl), and K<sub>2</sub>CO<sub>3</sub>(320 mg, 2.32 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 80 °C for 12 h under an N<sub>2</sub> atmosphere. Then, the mixture was filtered and the filtrate was concentrated to give a residue. The residue was purified by flash silica gel chromatography (ISCO; 12 g SepaFlash Silica FlashColumn, Eluent of 0–40% EtOAc/PE gradient at 100 mL/min) to obtain pure <b>44</b> (200 mg, 53%) as a light yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 8.83 (s, 2H), 7.15 (s, 1H), 6.85 (s, 1H), 5.73 (br s, 1H), 4.94 (br d, <i>J</i> = 13.2 Hz, 2H), 4.34 (q, <i>J</i> = 7.1 Hz, 2H), 4.00 (s, 3H), 3.98 (s, 3H), 3.63 (t, <i>J</i> = 6.4 Hz, 2H), 3.02–2.95 (m, 2H), 2.13–2.12 (m, 1H), 1.93 (br d, <i>J</i> = 11.2 Hz, 2H), 1.39–1.32 (m, 5H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>27</sub>ClN<sub>6</sub>O<sub>4</sub>, 486.2; <i>m</i>/<i>z</i>: found, 487.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_126" class="NLM_sec NLM_sec_level_3"><div id="ac_i153" class="anchor-spacer"></div><h4 class="article-section__title" id="_i153"> Ethyl 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)quinazolin-4-yl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>45</b>)</h4><div class="NLM_p last">A mixture of <b>44</b> (150 mg, 0.308 mmol),4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane (64 mg, 0.308), K<sub>2</sub>CO<sub>3</sub> (128 mg, 0.924 mmol) and Pd(PPh<sub>3</sub>)<sub>4</sub> (36 mg, 0.031 mmol) in 1,4-dioxane (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 100 °C for 12 h under an N<sub>2</sub> atmosphere. Then, the mixture was filtered and the filtrate was concentrated. The residue was purified by flash silica gel chromatography (ISCO; 12 g SepaFlash Silica FlashColumn, eluent of 0–40% EtOAc/PE gradient at 75 mL/min) to obtain pure <b>45</b> (140 mg, crude) as a light yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 8.83 (s, 2H), 7.36 (s, 1H), 7.16 (d, <i>J</i> = 3.3 Hz, 1H), 6.88 (s, 1H), 6.15 (dd, <i>J</i> = 0.9, 3.1 Hz, 1H), 5.58 (br s, 1H), 4.94 (br d, <i>J</i> = 13.5 Hz, 2H), 4.34 (q, <i>J</i> = 7.1 Hz, 2H), 4.00 (d, <i>J</i> = 0.9 Hz, 6H), 3.68 (t, <i>J</i> = 6.4 Hz, 2H), 3.02–2.95 (m, 2H), 2.46 (s, 3H), 2.20–2.17 (m, 1H), 1.96 (br d, <i>J</i> = 10.8 Hz, 2H), 1.41–1.33 (m, 5H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>32</sub>N<sub>6</sub>O<sub>5</sub>, 532.2; <i>m</i>/<i>z</i>: found, 533.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_1_127" class="NLM_sec NLM_sec_level_3"><div id="ac_i154" class="anchor-spacer"></div><h4 class="article-section__title" id="_i154"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)quinazolin-4-yl]amino]methyl]-1-piperidyl]pyrimidine-5-carboxylic Acid (<b>46</b>)</h4><div class="NLM_p last">To a solution of <b>45</b> (190 mg, 0.356 mmol) in THF/H<sub>2</sub>O (2:1, 12 mL) was added LiOH·H<sub>2</sub>O (45 mg, 1.07 mmol) and the mixture was stirred at 25 °C for 12 h. Then, the mixture was adjusted to pH ∼ 5 with aqueous HCl (2 M) and extracted with EtOAc (5 mL × 3). The combined organic phase was washed with brine (10 mL), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to afford <b>46</b> (150 mg, 83%) as a light yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>28</sub>N<sub>6</sub>O<sub>5</sub>, 504.2; <i>m</i>/<i>z</i>: found, 505.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div><div id="sec5_1_128" class="NLM_sec NLM_sec_level_3"><div id="ac_i155" class="anchor-spacer"></div><h4 class="article-section__title" id="_i155"> 2-[4-[[[6,7-Dimethoxy-2-(5-methyl-2-furyl)quinazolin-4-yl]amino]methyl]-1-piperidyl]-<i>N</i>-tetrahydropyran-2-yloxy-pyrimidine-5-carboxamide (<b>47</b>)</h4><div class="NLM_p last">A mixture of <b>46</b> (150 mg, 0.297 mmol), THPONH<sub>2</sub> (70 mg, 0.594 mmol), HOBt (80 mg, 0.594 mmol), EDCI (114 mg, 0.594 mmol), and DIEA (192 mg, 1.49 mmol) in DMF (10 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for 12 h under an N<sub>2</sub> atmosphere. Then, the residue was poured into ice-water (w/w = 1/1) (10 mL) and the aqueous phase was extracted with EtOAc (10 mL × 3). The combined organic phase was washed with brine (10 mL), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to afford <b>47</b> (130 mg, crude) as a light yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>31</sub>H<sub>37</sub>N<sub>7</sub>O<sub>6</sub>, 603.3; <i>m</i>/<i>z</i>: found, 604.3 [M + H]<sup>+</sup>. This intermediate was used in the next step without further purification or characterization.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i156" class="anchor-spacer"></div><h3 class="article-section__title" id="_i156"> Synthesis of Intermediates</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i157" class="anchor-spacer"></div><h4 class="article-section__title" id="_i157"> Ethyl 2-[4-[(<i>tert</i>-Butoxycarbonylamino)methyl]-1-piperidyl]pyrimidine-5-carboxylate (<b>50</b>)</h4><div class="NLM_p last">A mixture of <i>tert</i>-butyl <i>N</i>-(4-piperidylmethyl)carbamate (<b>48</b>, 1.00 g, 4.67 mmol), ethyl 2-chloropyrimidine-5-carboxylate (<b>49</b>, 870 mg, 4.67 mmol), BINAP (290 mg, 0.467 mmol), Cs<sub>2</sub>CO<sub>3</sub> (3.04 g, 9.34 mmol), and Pd<sub>2</sub>(dba)<sub>3</sub> (427 mg, 0.467 mmol) in 1,4-dioxane (100 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 110 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C, the residue was poured into water (50 mL), and it was extracted with EtOAc (30 mL × 3). The combined organic phase was washed with brine (30 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel column chromatography (PE/EtOAc = 1:0 to 2:1) to afford pure <b>50</b> (1.20 g, 70%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>18</sub>H<sub>28</sub>N<sub>4</sub>O<sub>4</sub>, 364.2; <i>m</i>/<i>z</i>: found, 365.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i158" class="anchor-spacer"></div><h4 class="article-section__title" id="_i158"> Ethyl 2-[4-(Aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate (<b>51</b>)</h4><div class="NLM_p last">A solution of <b>50</b> (1.20 g, 3.29 mmol) in HCl/EtOAc (30 mL, 2.0 M) was stirred at 20 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C, the residue was poured into water (20 mL), and it was extracted with EtOAc (10 mL × 3). The aqueous phase was concentrated in vacuum and then dissolved in MeOH (20 mL). NaHCO<sub>3</sub> (200 mg) was added and the mixture was stirred at 25 °C for 1 h. The mixture was filtered and concentrated in vacuum to afford <b>51</b> (700 mg, 80%) as a white solid, which was used for the synthesis of target compounds without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.77 (d, <i>J</i> = 8.0 Hz, 2H), 4.95–4.91 (m, 2H), 4.35–4.29 (m, 2H), 3.04–2.97 (m, 2H), 2.78–2.77 (m, 2H), 1.95–1.86 (m, 3H), 1.37–1.34 (m, 3H), 1.24–1.20 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>13</sub>H<sub>20</sub>N<sub>4</sub>O<sub>2</sub>, 264.2; <i>m</i>/<i>z</i>: found, 265.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i159" class="anchor-spacer"></div><h4 class="article-section__title" id="_i159"> Methyl 4-[[4-[(<i>tert</i>-Butoxycarbonylamino)methyl]-1-piperidyl]methyl]benzoate (<b>53</b>)</h4><div class="NLM_p last">To a solution of methyl 4-formylbenzoate (<b>52</b>, 200 mg, 1.22 mmol) in MeOH (20 mL) were added NaBH<sub>3</sub>CN (230 mg, 3.66 mmol), AcOH (77 mg, 1.28 mmol), and <i>tert</i>-butyl <i>N</i>-(4-piperidylmethyl)carbamate (<b>48</b>, 275 mg, 1.28 mmol) and the mixture was stirred at 80 °C for 3 h. Then, the reaction was quenched with water (5 mL) and MeOH was removed. The residue was extracted with EtOAc (30 mL × 3). The combined organic layers were washed with brine (10 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue, which was purified by prep-TLC (SiO<sub>2</sub>, PE/EtOAc = 1/1) to afford pure <b>53</b> (220 mg, 50%) as a white solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>30</sub>N<sub>2</sub>O<sub>4</sub>, 362.2; <i>m</i>/<i>z</i>: found, 363.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i160" class="anchor-spacer"></div><h4 class="article-section__title" id="_i160"> Methyl 4-[[4-(Aminomethyl)-1-piperidyl]methyl]benzoate (<b>54</b>)</h4><div class="NLM_p last">A solution of <b>53</b> (220 mg, 0.607 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 1 h. Then, the mixture was concentrated to give a residue. The residue was diluted with MeOH and the pH was adjusted to 7–8 with NaHCO<sub>3</sub>. Then, the mixture was filtered and the filtrate was concentrated to give <b>54</b> (140 mg, 88%) as a white solid, which was used for the synthesis of compound <b>17d</b> without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.98 (d, <i>J</i> = 8.0 Hz, 2H), 7.46 (d, <i>J</i> = 8.0 Hz, 2H), 3.91 (s, 3H), 3.58 (d, <i>J</i> = 6.4 Hz, 2H), 2.94–2.90 (m, 2H), 2.56–2.53 (m, 2H), 2.08–2.02 (m, 2H), 1.77–1.74 (m, 2H), 1.19–1.15 (m, 3H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>22</sub>N<sub>2</sub>O<sub>2</sub>, 262.2; <i>m</i>/<i>z</i>: found, 263.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i161" class="anchor-spacer"></div><h4 class="article-section__title" id="_i161"> Methyl 4-[[4-(<i>tert</i>-Butoxycarbonylamino)-1-piperidyl]methyl]benzoate (<b>56</b>)</h4><div class="NLM_p last">A mixture of <i>tert</i>-butyl <i>N</i>-(4-piperidyl)carbamate (<b>55</b>, 1.00 g, 4.99 mmol), methyl 4-formylbenzoate (<b>52</b>, 820 mg, 4.99 mmol), NaBH<sub>3</sub>CN (941 mg, 14.98 mmol), and CH<sub>3</sub>COOH (315 mg, 5.24 mmol) in MeOH (30 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 70 °C for 2 h. Then, the mixture was concentrated in reduced pressure at 40 °C, the residue was poured into water (50 mL), and it was extracted with EtOAc (50 mL × 3). The combined organic phase was washed with brine (50 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give <b>56</b> (1.00 g, 57%) as a white solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>19</sub>H<sub>28</sub>N<sub>2</sub>O<sub>4</sub>, 348.2; <i>m</i>/<i>z</i>: found, 349.2[M + H]<sup>+</sup>.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i162" class="anchor-spacer"></div><h4 class="article-section__title" id="_i162"> Methyl 4-[(4-Amino-1-piperidyl)methyl]benzoate (<b>57</b>)</h4><div class="NLM_p last">A solution of <b>56</b> (1.00 g, 2.87 mmol) in HCl/EtOAc (30 mL, 2.0 M) was stirred at 20 °C for 5 h under an N<sub>2</sub> atmosphere. Then, the reaction mixture was concentrated in vacuum to give a residue. The residue was dissolved in MeOH (50 mL) and NaHCO<sub>3</sub> (400 mg) was added. The reaction mixture was stirred at 25 °C for 1 h. The mixture was filtered and concentrated in vacuum to give <b>57</b> (300 mg, 42%) as a white solid, which was used for the synthesis of compound <b>17e</b> without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.02 (d, <i>J</i> = 8.0 Hz, 2H), 7.56 (d, <i>J</i> = 8.0 Hz, 2H), 3.91 (s, 3H), 3.90 (s, 2H), 3.19–3.14 (m, 3H), 2.56–2.50 (m, 2H), 2.11–2.08 (m, 2H), 1.87–1.80 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>, 248.2; <i>m</i>/<i>z</i>: found, 249.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i163" class="anchor-spacer"></div><h4 class="article-section__title" id="_i163"> Methyl 4-[4-[(<i>tert</i>-Butoxycarbonylamino)methyl]-1-piperidyl]benzoate (<b>59</b>)</h4><div class="NLM_p last">To a solution of methyl 4-fluorobenzoate (<b>58</b>, 500 mg, 3.24 mmol) in DMSO (35 mL) were added K<sub>2</sub>CO<sub>3</sub> (897 mg, 6.49 mmol) and <b>48</b> (695 mg, 3.24 mmol) and the mixture was stirred at 90 °C for 16 h. Then, the mixture was filtered and the filtrate was diluted with the water (10 mL) and it was extracted with EtOAc (20 mL × 3). The combined organic layers were washed with brine (50 mL × 3), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give a residue. The residue was purified by silica gel column chromatography (PE/EtOAc = 5:1 to 2:1) to afford pure <b>59</b> (430 mg, 38%) as a white solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>19</sub>H<sub>28</sub>N<sub>2</sub>O<sub>4</sub>, 348.2; <i>m</i>/<i>z</i>: found, 349.2[M + H]<sup>+</sup>.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i164" class="anchor-spacer"></div><h4 class="article-section__title" id="_i164"> Methyl 4-[4-(Aminomethyl)-1-piperidyl]benzoate (<b>60</b>)</h4><div class="NLM_p last">A solution of <b>59</b> (430 mg, 1.23 mmol) in HCl/EtOAc (15 mL, 2.0 M) was stirred at 20 °C for 1 h. Then, the mixture was concentrated to give a residue, which was diluted with MeOH (15 mL) and alkalized with a saturated aqueous NaHCO<sub>3</sub> solution. The mixture was filtered, and the filtrate was concentrated to give <b>60</b> (230 mg, 75%) as a white solid, which was used for the synthesis of compound <b>17f</b> without further purification. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>20</sub>N<sub>2</sub>O<sub>2</sub>, 248.2; <i>m</i>/<i>z</i>: found, 249.3 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i165" class="anchor-spacer"></div><h4 class="article-section__title" id="_i165"> (4-Cyanocyclohexen-1-yl)trifluoromethanesulfonate (<b>62</b>)</h4><div class="NLM_p last">To a stirred solution of the commercially available 4-oxocyclohexanecarbonitrile (4.00 g, 32.48 mmol) and KHMDS (1.0 M, 42.87 mL) in THF (200 mL) under N<sub>2</sub> at −78 °C was added PhNTf<sub>2</sub> (14.85 g, 41.57 mmol) and the mixture was stirred for 8 h at 20 °C. Then, the mixture was concentrated in reduced pressure at 40 °C, the residue was poured into water (100 mL), and it was extracted with EtOAc (100 mL × 3). The combined organic phase was washed with brine (100 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum. The residue was purified by silica gel column chromatography (PE/EtOAc = 50:1 to 1:1) to give <b>62</b> (3.00 g, 36%) as a colorless oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>8</sub>H<sub>8</sub>F<sub>3</sub>NO<sub>3</sub>S, 255.0; <i>m</i>/<i>z</i>: found, 256.1[M + H]<sup>+</sup>.</div></div><div id="sec5_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i166" class="anchor-spacer"></div><h4 class="article-section__title" id="_i166"> Methyl 4-(4-Cyanocyclohexen-1-yl)benzoate (<b>63</b>)</h4><div class="NLM_p last">A mixture of <b>62</b> (2.50 g, 9.80 mmol), (4-methoxycarbonylphenyl)boronic acid (1.47 g, 8.16 mmol), Pd(PPh<sub>3</sub>)<sub>4</sub> (943 mg, 0.816 mmol), and K<sub>2</sub>CO<sub>3</sub> (2.26 g, 16.33 mmol) in 1,4-dioxane/H<sub>2</sub>O (10:1, 55 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 90 °C for 1 h. Then, the mixture was concentrated in reduced pressure at 40 °C, the residue was poured into water (50 mL), and it was extracted with EtOAc (50 mL × 3). The combined organic phase was washed with brine (50 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give a crude product, which was purified by silica gel column chromatography (PE/EtOAc = 40:1 to 2:1) to afford pure <b>63</b> (2.00 g, crude) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>15</sub>NO<sub>2</sub>, 241.1; <i>m</i>/<i>z</i>: found, 242.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i167" class="anchor-spacer"></div><h4 class="article-section__title" id="_i167"> Methyl 4-[4-(Aminomethyl)cyclohexyl]benzoate (<b>64</b>)</h4><div class="NLM_p last">To a solution of <b>63</b> (1.50 g, 6.22 mmol) in HCl (1.23 mL, 36% purity) and MeOH (20 mL) was added Pd(OH)<sub>2</sub>/C (10%, 500 mg) under an H<sub>2</sub> atmosphere. The suspension was degassed and purged with H<sub>2</sub> 3 times. The mixture was stirred under H<sub>2</sub> (50 Psi) at 30 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue, which was purified by prep-HPLC (method 37 described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">Supporting Information</a>) to afford pure <b>64</b> (200 mg, 13%) as a white solid. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 7.94–7.92 (m, 2H), 7.37–7.32 (m, 2H), 3.88 (s, 3H), 2.81 (d, <i>J</i> = 7.2 Hz, 1H), 2.70 (s, 1H), 2.60 (d, <i>J</i> = 6.4 Hz, 1H), 1.96–1.91 (m, 2H), 1.76–1.69 (m, 5H), 1.55–1.52 (m, 1H), 1.15–1.12 (m, 1H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>21</sub>NO<sub>2</sub>, 247.2; <i>m</i>/<i>z</i>: found, 248.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i168" class="anchor-spacer"></div><h4 class="article-section__title" id="_i168"> Methyl 4-[4-[(<i>E</i>)-Hydroxyiminomethyl]phenyl]benzoate (<b>66</b>)</h4><div class="NLM_p last">A mixture of the commercially available methyl 4-(4-formylphenyl)benzoate (1.00 g, 4.16 mmol), NH<sub>2</sub>OH·HCl (289 mg, 4.16 mmol), and NaOAc (273 mg, 3.33 mmol) in EtOH (40 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 25 °C for 16 h. Then, the mixture was concentrated in reduced pressure at 40 °C, the residue was poured into water (10 mL), and it was extracted with CH<sub>2</sub>Cl (10 mL × 3). The combined organic phase was washed with brine (10 mL × 2), dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuum to give <b>66</b> (500 mg, 47%) as a yellow solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>13</sub>NO<sub>3</sub>, 255.1; <i>m</i>/<i>z</i>: found, 256.1[M + H]<sup>+</sup>.</div></div><div id="sec5_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i169" class="anchor-spacer"></div><h4 class="article-section__title" id="_i169"> Methyl 4-[4-[(<i>tert</i>-Butoxycarbonylamino)methyl]phenyl]benzoate (<b>67</b>)</h4><div class="NLM_p last">To a solution of <b>66</b> (500 mg, 1.96 mmol) and BOC<sub>2</sub>O (427 mg, 1.96 mmol) in EtOH (30 mL) was added Pd/C (10%, 200 mg) under an H<sub>2</sub> atmosphere. The suspension was degassed and purged with H<sub>2</sub> 3 times and then the mixture was stirred under H<sub>2</sub> (50 Psi) at 20 °C for 16 h. Then, the mixture was filtered and concentrated under vacuum to give a residue, which was purified by silica gel column chromatography (PE/EtOAc = 1:0 to 5:1) to afford pure <b>67</b> (400 mg, 59%) as a white solid. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>23</sub>NO<sub>4</sub>, 341.2; <i>m</i>/<i>z</i>: found, 342.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i170" class="anchor-spacer"></div><h4 class="article-section__title" id="_i170"> Methyl 4-[4-(Aminomethyl)phenyl]benzoate (<b>68</b>)</h4><div class="NLM_p last">A solution of <b>67</b> (400 mg, 1.17 mmol) in HCl/EtOAc (10 mL, 2.0 M) was stirred at 20 °C for 5 h under an N<sub>2</sub> atmosphere. Then, the reaction mixture was concentrated in vacuum to give a residue, which was dissolved in MeOH (20 mL). Then, NaHCO<sub>3</sub> (200 mg) was added and the solution was stirred at 25 °C for 1 h. The mixture was filtered and concentrated in vacuum to give <b>68</b> (250 mg, 88%) as a white solid, which was used for the synthesis of compound <b>17i</b> without further purification. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.12–8.10 (m, 2H), 7.79–7.76 (m 4H), 7.59–7.57 (m, 2H), 4.19 (s, 2H), 3.93 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>15</sub>H<sub>15</sub>NO<sub>2</sub>, 241.1; <i>m</i>/<i>z</i>: found, 242.1 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i171" class="anchor-spacer"></div><h4 class="article-section__title" id="_i171"> Ethyl 4-(4-Formylphenyl)cyclohex-3-ene-1-carboxylate (<b>70</b>)</h4><div class="NLM_p last">A mixture of (4-formylphenyl)boronic acid (1 g, 6.67 mmol), the commercially available ethyl 4-(trifluoromethylsulfonyloxy)cyclohex-3-ene-1-carboxylate (2.42 g, 8.00 mmol), Pd/C (100 mg, 10% purity), and PPh<sub>3</sub> (157 mg, 0.600 mmol) in DME (50 mL) and H<sub>2</sub>O (25 mL) was degassed and purged with N<sub>2</sub> 3 times, and then the mixture was stirred at 80 °C for 16 h. Then, the mixture was filtered and the filtrate was concentrated to give a residue, which was purified by silica gel column chromatography (PE/EtOAc = 20:1 to 5:1) to afford <b>70</b> (950 mg, 55%) as a yellow solid.<sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 9.99 (s, 1H), 7.83 (d, <i>J</i> = 8.0 Hz, 2H), 7.54 (d, <i>J</i> = 8.0 Hz, 2H), 6.30 (s, 1H), 4.19 (q, <i>J</i> = 7.4 Hz, 2H), 2.71–2.59 (m, 1H), 2.54–2.53 (m, 4H), 2.27–2.15 (m, 1H), 1.95–1.79 (m, 1H), 1.29 (t, <i>J</i> = 7.2 Hz, 3H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>18</sub>O<sub>3</sub>, 258.2; <i>m</i>/<i>z</i>: found, 259.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i172" class="anchor-spacer"></div><h4 class="article-section__title" id="_i172"> Ethyl 4-[4-[(<i>E</i>)-Hydroxyiminomethyl]phenyl]cyclohex-3-ene-1-carboxylate (<b>71</b>)</h4><div class="NLM_p last">To a solution of <b>70</b> (950 mg, 3.68 mmol) in EtOH (5 mL) and H<sub>2</sub>O (2.5 mL) were added NH<sub>2</sub>OH·HCl (256 mg, 3.68 mmol) and NaOAc (241 mg, 2.94 mmol) and the mixture was stirred at 20 °C for 16 h. Then, the reaction mixture was concentrated under reduced pressure to remove EtOH. The residue was diluted with water (30 mL) and it was extracted with EtOAc (50 mL × 3). The combined organic layers were washed with brine (10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give a residue, which was purified by silica gel column chromatography (PE/EtOAc = 10:1 to 5:1) to afford pure <b>71</b> (980 mg, 97%) as a yellow solid.<sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 8.13 (s, 1H), 7.52 (d, <i>J</i> = 8.0 Hz, 2H), 7.40 (d, <i>J</i> = 8.4 Hz, 2H), 6.19 (s, 1H), 4.18 (q, <i>J</i> = 7.0 Hz, 2H), 2.68–2.56 (m, 1H), 2.51 (m, 4H), 2.25–2.16 (m, 1H), 1.92–1.77 (m, 1H), 1.29 (t, <i>J</i> = 7.2 Hz, 3H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>19</sub>NO<sub>3</sub>, 273.1; <i>m</i>/<i>z</i>: found, 274.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i173" class="anchor-spacer"></div><h4 class="article-section__title" id="_i173"> Ethyl 4-[4-[(<i>tert</i>-Butoxycarbonylamino)methyl]phenyl]cyclohexanecarboxylate (<b>72</b>)</h4><div class="NLM_p last">To a solution of BOC<sub>2</sub>O (822 mg, 3.76 mmol) in EtOH (20 mL) were added <b>71</b> (980 mg, 3.59 mmol), Et<sub>3</sub>N (544 mg, 5.4 mmol), and Pd/C (10% purity, 100 mg) under an N<sub>2</sub> atmosphere. The suspension was degassed and purged with H<sub>2</sub> 3 times and then the mixture was stirred under H<sub>2</sub> (50 Psi) at 20 °C for 16 h. Then, the mixture was filtered and the filtrate was concentrated to give a residue, which was purified by silica gel column chromatography (PE/EtOAc = 5:1 to 2:1) to afford pure compound <b>72</b> (950 mg, 73%) as a yellow oil. ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>31</sub>NO<sub>4</sub>, 361.2; <i>m</i>/<i>z</i>: found, 262.2 [M – BOC]<sup>+</sup>.</div></div><div id="sec5_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i174" class="anchor-spacer"></div><h4 class="article-section__title" id="_i174"> Ethyl 4-[4-(Aminomethyl)phenyl]cyclohexanecarboxylate (<b>73</b>)</h4><div class="NLM_p last">A solution of <b>72</b> (950 mg, 2.63 mmol) in HCl/EtOAc (20 mL, 2.0 M) was stirred at 20 °C for 16 h. Then, the mixture was concentrated to give <b>73</b> (350 mg, 51%) as a light yellow oil, which was used for the synthesis of compound <b>17j</b> without further purification. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.30–7.23 (m, 2H), 7.22–7.17 (m, 2H), 4.25–4.12 (m, 2H), 3.88–3.81 (m, 2H), 2.71 (s, 1H), 2.62–2.48 (m, 1H), 2.27 (d, <i>J</i> = 9.4 Hz, 1H), 1.84–1.76 (m, 1H), 1.74–1.47 (m, 6H), 1.35–1.23 (m, 3H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>23</sub>NO<sub>2</sub>, 261.2; <i>m</i>/<i>z</i>: found, 262.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i175" class="anchor-spacer"></div><h4 class="article-section__title" id="_i175"> Methyl 4-[4-(<i>tert</i>-Butoxycarbonylamino)-1-piperidyl]benzoate (<b>74</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl <i>N</i>-(4-piperidyl)carbamate (<b>55</b>, 2.00 g, 9.99 mmol) in DMSO (50 mL) were added K<sub>2</sub>CO<sub>3</sub> (3.45 g, 24.98 mmol) and methyl 4-fluorobenzoate (<b>58</b>, 1.85 g, 11.99 mmol) and the mixture was stirred at 90 °C for 16 h. Then, the reaction mixture was poured into water (200 mL) and filtered. The solution was concentrated in vacuum to give <b>74</b> (1.40 g, 42%) as a white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.92–7.89 (m, 2H), 6.87–6.85 (m, 2H), 3.87 (s, 3H), 3.82–3.79 (m, 2H), 3.01–2.94 (s, 2H), 2.06–2.04 (m, 2H), 1.54–1.46 (m, 12H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>18</sub>H<sub>26</sub>N<sub>2</sub>O<sub>4</sub>, 334.2; <i>m</i>/<i>z</i>: found, 335.2 [M + H]<sup>+</sup>.</div></div><div id="sec5_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i176" class="anchor-spacer"></div><h4 class="article-section__title" id="_i176"> Methyl 4-(4-Amino-1-piperidyl)benzoate (<b>75</b>)</h4><div class="NLM_p last">Compound <b>74</b> (1.40 g, 4.19 mmol) was dissolved in HCl/EtOAc (20 mL, 2.0 M) and the mixture was stirred at 25 °C for 16 h. Then, the reaction mixture was concentrated in vacuum to give a residue, which was dissolved in MeOH (50 mL). Then, NaHCO<sub>3</sub> (400 mg) was added and the mixture solution was stirred at 25 °C for 1 h. The mixture was filtered and concentrated in vacuum to afford <b>75</b> (500 mg, 51%), which was used for the synthesis of compound <b>17k</b> without further purification.<sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz): δ 7.85 (d, <i>J</i> = 8.8 Hz, 2H), 6.99 (d, <i>J</i> = 9.2 Hz, 2H), 4.03–3.99 (m, 2H), 3.83 (s, 3H), 3.29–3.26 (m, 1H), 2.99–2.92 (m, 2H), 2.08–2.05 (m, 2H), 1.70–1.61 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>13</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub>, 234.1; <i>m</i>/<i>z</i>: found, 235.1 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i177" class="anchor-spacer"></div><h3 class="article-section__title" id="_i177"> Docking and Biological Assays</h3><div class="NLM_p last">Details on the following assays are reported: (a) docking set-up protocols into G9a and DNMT1,<a onclick="showRef(event, 'ref21 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref21 ref34 ref35">(21,34,35)</a> and HDAC1<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and HDAC6;<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (b) G9a,<a onclick="showRef(event, 'ref21 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref21 ref34 ref35">(21,34,35)</a> DNMT1,<a onclick="showRef(event, 'ref21 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref21 ref34 ref35">(21,34,35)</a> DNMT3A,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> DNMT3B,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> HDAC1,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> HDAC2,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> HDAC3,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and HDAC6<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> enzyme activity assays; (c) cytotoxicity in THLE-2 cells;<a onclick="showRef(event, 'ref21 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref21 ref34 ref35">(21,34,35)</a> (d) PAMPA permeability;<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (e) cytochrome P450s inhibition;<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (f) metabolic stability;<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (g) kinetic solubility;<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> (h) Western blot (WB) to monitor H3K9me2<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and H3K9Ac,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (i) DNA methylation analysis by pyrosequencing,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (j) cell proliferation assay<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (MM1.S cells were cultured at a density of 0.4 × 10<sup>6</sup> cells/mL), and (k) a PK study after approval from the Animal Care and Ethics Committee of the University of Navarra (protocol numbers 158-12 and 009-16) in a plasma sample.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Other tests are explicitly described (below).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i178" class="anchor-spacer"></div><h3 class="article-section__title" id="_i178"> <i>In Vivo</i> Experiments</h3><div class="NLM_p last">All animal studies had previous approval from the Animal Care and Ethics Committee of the University of Navarra (protocol number: 041-15). For the human subcutaneous MM1.S MM model, 10 × 10<sup>6</sup> MM1.S cells diluted in 100 μL of saline solution were subcutaneously inoculated in the back left flank of female BALB/cA-Rag2<sup>-/-</sup>γc<sup>-/-</sup> mice between 6 and 8 weeks of age (<i>n</i> = 18). When the tumors became palpable, the mice were randomized into two groups, control and compound <b>12a</b>, (9 animals/group). The treatment with 10 mg/kg of compound <b>12a</b> was started 12 days after cell inoculation when all mice presented subcutaneous tumors and was administered for 5 consecutive days followed by 2 rest days (<i>n</i> = 9) during 3 weeks. The control group (<i>n</i> = 9) received only 80% saline solution, 10% DMSO, and 10% Tween20 (diluents of compound <b>12a</b>). The tumor size was analyzed every 5 days using the following method: <i>V</i>1/4 <i>D</i>_<i>d</i>2/2, where <i>D</i> and <i>d</i> corresponded to the longest and shorter diameters, respectively. The mice were killed 35 days after cell inoculation.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i179" class="anchor-spacer"></div><h3 class="article-section__title" id="_i179"> Interference Compound Assessment</h3><div class="NLM_p last">No compound reported matches any of the structural filters for potential PAINS as defined by Baell & Holloway<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and implemented in a customized Pipeline Pilot protocol.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i180"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02255" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43582" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43582" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02255?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02255</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Protocols for preparative HPLC purification methods; method for high-resolution mass spectrometry (HRMS) of the final compounds; methods for analytical HPLC and UHPLC; HPLC or UHPLC traces of the final compounds; NMR spectra (<sup>1</sup>H & <sup>13</sup>C) of the final compounds; biochemical profiling of <b>9a</b> and <b>12a</b><i>versus</i> DNMT and HDAC isoforms; H3K9me2 and H3Ac marks after treatment with A-366, Panobinostat, and Decitabine in MM1.S cells; hypomethylating activity of compounds <b>9a</b> and <b>12a</b> in JJN3 cells; body weight of mice treated with compound <b>12a</b>; and plasmatic concentrations of <b>12a</b> after administration (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings together with their IC<sub>50</sub> values  (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Results from docking studies reported in Figure 2:</p></li><li><p class="inline">G9a compound <b>9a</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">DNMT1 compound <b>9a</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">HDAC1 compound <b>9a</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">HDAC6 compound <b>9a</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_006.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf">jm0c02255_si_001.pdf (3.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_002.csv">jm0c02255_si_002.csv (3.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_003.pdb">jm0c02255_si_003.pdb (349.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_004.pdb">jm0c02255_si_004.pdb (1.05 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_005.pdb">jm0c02255_si_005.pdb (253.74 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_006.pdb">jm0c02255_si_006.pdb (233.4 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID Codes: <b>1</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>; <b>2</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EF8">5EF8</a>; <b>3</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSH">6HSH</a>; G9a, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>; DNMT1, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>; HDAC1, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKX">4BKX</a>; HDAC6, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02255" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86055" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86055" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Felipe Prosper</span> - <span class="hlFld-Affiliation affiliation">Area
de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span>; 
    <span class="hlFld-Affiliation affiliation">Departmento
de Hematología, Clínica Universidad de Navarra, University of Navarra, Avenida Pio XII 36, E-31008 Pamplona, Spain</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5a3c2a2835292a3f281a2f343b2c743f29"><span class="__cf_email__" data-cfemail="137563617c6063766153667d72653d7660">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julen Oyarzabal</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, Center
for Applied Medical Research (CIMA), University
of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1941-7255" title="Orcid link">http://orcid.org/0000-0003-1941-7255</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#244e5148414a4b5d45565e4546454864415c5041564a45480a514a45520a4157044e4b5d45565e4546454864474b48514946515752540a474b49"><span class="__cf_email__" data-cfemail="c0aab5aca5aeafb9a1b2baa1a2a1ac80a5b8b4a5b2aea1aceeb5aea1b6eea5b3">[email protected]</span> <span class="__cf_email__" data-cfemail="e08a8f9981929a8182818ca0838f8c958d8295939690ce838f8d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Obdulia Rabal</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, Center
for Applied Medical Research (CIMA), University
of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edurne San José-Enériz</span> - <span class="hlFld-Affiliation affiliation">Area
de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xabier Agirre</span> - <span class="hlFld-Affiliation affiliation">Area
de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Juan Antonio Sánchez-Arias</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, Center
for Applied Medical Research (CIMA), University
of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene de Miguel</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, Center
for Applied Medical Research (CIMA), University
of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raquel Ordoñez</span> - <span class="hlFld-Affiliation affiliation">Area
de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leire Garate</span> - <span class="hlFld-Affiliation affiliation">Area
de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Estíbaliz Miranda</span> - <span class="hlFld-Affiliation affiliation">Area
de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Sáez</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, Center
for Applied Medical Research (CIMA), University
of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amaia Vilas-Zornoza</span> - <span class="hlFld-Affiliation affiliation">Area
de Hemato-Oncología, IDISNA, CIBERONC, Center for Applied Medical
Research (CIMA), University of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonio Pineda-Lucena</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, Center
for Applied Medical Research (CIMA), University
of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ander Estella</span> - <span class="hlFld-Affiliation affiliation">Small
Molecule Discovery Platform, Molecular Therapeutics Program, Center
for Applied Medical Research (CIMA), University
of Navarra, Avenida Pio XII 55, E-31008 Pamplona, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feifei Zhang</span> - <span class="hlFld-Affiliation affiliation">WuXi
Apptec (Tianjin) Company Ltd., TEDA, No. 168 Nanhai Road, 10th Avenue, 300456 Tianjin, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Wu</span> - <span class="hlFld-Affiliation affiliation">WuXi
Apptec (Tianjin) Company Ltd., TEDA, No. 168 Nanhai Road, 10th Avenue, 300456 Tianjin, PR China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Musheng Xu</span> - <span class="hlFld-Affiliation affiliation">WuXi
Apptec (Tianjin) Company Ltd., TEDA, No. 168 Nanhai Road, 10th Avenue, 300456 Tianjin, PR China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">O.R., E.S.J.-E., and X.A. share first authorship.</p><br /></br><p class="inlineNote">All animal experiments performed in the manuscript were conducted in compliance with institutional guidelines (as reported above)</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i184">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43006" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43006" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the Foundation for Applied Medical Research, University of Navarra (Pamplona, Spain), Fundación Fuentes Dutor, Paula and Rodger Riney Foundation, Fundación La Caixa Hepacare Project and Gobierno de Navarra (PI029 DIFF4LMA), Instituto de Salud Carlos III (ISCIII) PI16/02024, PI17/00701, PI19/01352 and PI20/01306, CIBERONC (CB16/12/00489), co-financed with FEDER funds, MINECO Explora SAF2017-92632-EXP (RTHALMY), Multiple Myeloma Research Foundation Networks of excellence, the International Myeloma Foundation (Brian van Novis), and the Qatar National Research Fund award 7-916-3-237 for financial support. We thank Pablo Garnica Calvo, Carmen Sanmartín Grijalba, Ana Romo Hualde and Ángel Irigoyen Barrio for their excellent technical assistance with compound characterization.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">BINAP</td><td class="NLM_def"><p class="first last">2,2′-bis(diphenylphosphino)-1,1′-binaphthyl</p></td></tr><tr><td class="NLM_term">BOC</td><td class="NLM_def"><p class="first last"><i>tert</i>-butoxycarbonyl</p></td></tr><tr><td class="NLM_term">Cpd</td><td class="NLM_def"><p class="first last">compound</p></td></tr><tr><td class="NLM_term">dba</td><td class="NLM_def"><p class="first last">dibenzylideneacetone</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethylsulfoxide</p></td></tr><tr><td class="NLM_term">DNA</td><td class="NLM_def"><p class="first last">deoxyribonucleic acid</p></td></tr><tr><td class="NLM_term">DNMT</td><td class="NLM_def"><p class="first last">DNA methyltransferase</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide</p></td></tr><tr><td class="NLM_term">EHMT2</td><td class="NLM_def"><p class="first last">euchromatic histone methyltransferase 2</p></td></tr><tr><td class="NLM_term">ESI-MS</td><td class="NLM_def"><p class="first last">electrospray ionization mass spectrometry</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">HDACi</td><td class="NLM_def"><p class="first last">histone deacetylase inhibitor</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">ICW</td><td class="NLM_def"><p class="first last">in-cell Western</p></td></tr><tr><td class="NLM_term">LCMS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">mp</td><td class="NLM_def"><p class="first last">melting point</p></td></tr><tr><td class="NLM_term">MTBE</td><td class="NLM_def"><p class="first last">methyl <i>tert</i>-butyl ether</p></td></tr><tr><td class="NLM_term">MM</td><td class="NLM_def"><p class="first last">multiple myeloma</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">microwave</p></td></tr><tr><td class="NLM_term">NMM</td><td class="NLM_def"><p class="first last"><i>N</i>-methylmorpholine</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assay</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">PE</td><td class="NLM_def"><p class="first last">petroleum ether</p></td></tr><tr><td class="NLM_term">Ph</td><td class="NLM_def"><p class="first last">phenyl</p></td></tr><tr><td class="NLM_term">HH</td><td class="NLM_def"><p class="first last">human hepatocytes</p></td></tr><tr><td class="NLM_term">MH</td><td class="NLM_def"><p class="first last">mouse hepatocytes</p></td></tr><tr><td class="NLM_term">prep.</td><td class="NLM_def"><p class="first last">preparative</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">Rt</td><td class="NLM_def"><p class="first last">retention time</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuONa</td><td class="NLM_def"><p class="first last">sodium <i>tert</i>-butoxide</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">THPONH<sub>2</sub></td><td class="NLM_def"><p class="first last"><i>O</i>-(tetrahydro-2<i>H</i>-pyran-2-yl)hydroxylamine</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">TMS</td><td class="NLM_def"><p class="first last">tetramethylsilane</p></td></tr><tr><td class="NLM_term">UHPLC</td><td class="NLM_def"><p class="first last">ultraperformance liquid chromatography</p></td></tr><tr><td class="NLM_term">UV</td><td class="NLM_def"><p class="first last">ultraviolet</p></td></tr><tr><td class="NLM_term">ZBG</td><td class="NLM_def"><p class="first last">zinc-binding group</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i186">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 49 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Marked for Death: Targeting Epigenetic Changes in Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnrd.2016.256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28280262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=241-263&author=S.+X.+Pfisterauthor=A.+Ashworth&title=Marked+for+Death%3A+Targeting+Epigenetic+Changes+in+Cancer&doi=10.1038%2Fnrd.2016.256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Marked for death: targeting epigenetic changes in cancer</span></div><div class="casAuthors">Pfister, Sophia Xiao; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">241-263</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past few years, it has become clear that mutations in epigenetic regulatory genes are common in human cancers.  Therapeutic strategies are now being developed to target cancers with mutations in these genes using specific chem. inhibitors.  In addn., a complementary approach based on the concept of synthetic lethality, which allows exploitation of loss-of-function mutations in cancers that are not targetable by conventional methods, has gained traction.  Both of these approaches are now being tested in several clin. trials.  In this Review, we present recent advances in epigenetic drug discovery and development, and suggest possible future avenues of investigation to drive progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP-s0n0CRO2rVg90H21EOLACvtfcHk0lif9yF7ovgbXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D&md5=f028b0c2e596b7c358e2158318a1303c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.256%26sid%3Dliteratum%253Aachs%26aulast%3DPfister%26aufirst%3DS.%2BX.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DMarked%2520for%2520Death%253A%2520Targeting%2520Epigenetic%2520Changes%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D241%26epage%3D263%26doi%3D10.1038%2Fnrd.2016.256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA Methylation with Small Molecules: What’s Next?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2569</span>– <span class="NLM_lpage">2583</span>, <span class="refDoi"> DOI: 10.1021/jm500843d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500843d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2569-2583&author=A.+Erdmannauthor=L.+Halbyauthor=J.+Fahyauthor=P.+B.+Arimondo&title=Targeting+DNA+Methylation+with+Small+Molecules%3A+What%E2%80%99s+Next%3F&doi=10.1021%2Fjm500843d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA Methylation with Small Molecules: What's Next?</span></div><div class="casAuthors">Erdmann, Alexandre; Halby, Ludovic; Fahy, Jacques; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2569-2583</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA methylation is a mammalian epigenetic mark that is involved in defining where and when genes are expressed, both in normal cells and in the context of diseases.  Like other epigenetic marks, it is reversible and can be modulated by chem. agents.  Because it plays an important role in cancer by silencing certain genes, such as tumor suppressor genes, and by reactivating other regions, such as repeated elements, it is a promising therapeutic target.  Two compds. are already approved to treat hematol. cancers.  Many efforts have been carried out to discover new mols. that are able to efficiently inhibit DNA methylation in cancer cells.  We will briefly overview the foremost of these efforts by focusing on what we have learned to this point on non-nucleoside inhibitors and on what we consider to be the features of an ideal inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9LcXQUubTNbVg90H21EOLACvtfcHk0lif9yF7ovgbXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L&md5=cc469a073e82dc2cbe2d822cd8c02cce</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm500843d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500843d%26sid%3Dliteratum%253Aachs%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DFahy%26aufirst%3DJ.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DTargeting%2520DNA%2520Methylation%2520with%2520Small%2520Molecules%253A%2520What%25E2%2580%2599s%2520Next%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2569%26epage%3D2583%26doi%3D10.1021%2Fjm500843d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chistiakov, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myasoedova, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orekhov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bobryshev, Y. V.</span></span> <span> </span><span class="NLM_article-title">Epigenetically Active Drugs Inhibiting DNA Methylation and Histone Deacetylation</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.2174/1381612822666161021110827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.2174%2F1381612822666161021110827" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27774908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFWht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1167-1174&author=D.+A.+Chistiakovauthor=V.+A.+Myasoedovaauthor=A.+N.+Orekhovauthor=Y.+V.+Bobryshev&title=Epigenetically+Active+Drugs+Inhibiting+DNA+Methylation+and+Histone+Deacetylation&doi=10.2174%2F1381612822666161021110827"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetically Active Drugs Inhibiting DNA Methylation and Histone Deacetylation</span></div><div class="casAuthors">Chistiakov, Dimitry A.; Myasoedova, Veronika A.; Orekhov, Alexander N.; Bobryshev, Yuri V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1167-1174</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetic mechanisms, which are involved in the regulation of gene expression, are tightly controlled.  Loss of a proper epigenetic control can lead to global epigenetic alterations frequently obsd. in various diseases including cancer.  Aberrant epigenetic changes induced in malignant cells lead to emergence of neoplastic properties, which inhibit cell differentiation and strict cell cycle control but greatly enhance stemness-related features.  However, abnormal epigenetic patterns can be reversed by action of epigenetically active agents.  Epigenetic machinery comprises a variety DNA/histone modifiers and chromatin remodelers.  Chem. substances able to influence on the activity of epigenetic factors such as inhibitors of DNA methyltransferases or histone deacetylase inhibitors can be used as therapeutic agents for improving aberrant epigenetic signatures in cancer cells.  Preclin. studies showed efficiency of such epigenetic drugs for the treatment of variety of cancers.  So far, several epigenetically active compds. were approved for therapy of hematol. malignancies.  However, many challenges should be resolved for efficient use of epidrugs in the treatment of non-hematol. solid tumors and advanced cancers assocd. with chemoresistance and higher risk of relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBp259_nzKoLVg90H21EOLACvtfcHk0lhqWO-zJiMzYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFWht78%253D&md5=7fe8fff0865fa2b91478f70aa8576364</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F1381612822666161021110827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612822666161021110827%26sid%3Dliteratum%253Aachs%26aulast%3DChistiakov%26aufirst%3DD.%2BA.%26aulast%3DMyasoedova%26aufirst%3DV.%2BA.%26aulast%3DOrekhov%26aufirst%3DA.%2BN.%26aulast%3DBobryshev%26aufirst%3DY.%2BV.%26atitle%3DEpigenetically%2520Active%2520Drugs%2520Inhibiting%2520DNA%2520Methylation%2520and%2520Histone%2520Deacetylation%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2017%26volume%3D23%26spage%3D1167%26epage%3D1174%26doi%3D10.2174%2F1381612822666161021110827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases and Their Inhibitors in Cancer, Neurological Diseases and Immune Disorders</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+Deacetylases+and+Their+Inhibitors+in+Cancer%2C+Neurological+Diseases+and+Immune+Disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0lhqWO-zJiMzYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520Deacetylases%2520and%2520Their%2520Inhibitors%2520in%2520Cancer%252C%2520Neurological%2520Diseases%2520and%2520Immune%2520Disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachman, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myöhänen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span> <span> </span><span class="NLM_article-title">Synergy of Demethylation and Histone Deacetylase Inhibition in the Re-Expression of Genes Silenced in Cancer</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1038/5047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2F5047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=9916800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1999&pages=103-107&author=E.+E.+Cameronauthor=K.+E.+Bachmanauthor=S.+My%C3%B6h%C3%A4nenauthor=J.+G.+Hermanauthor=S.+B.+Baylin&title=Synergy+of+Demethylation+and+Histone+Deacetylase+Inhibition+in+the+Re-Expression+of+Genes+Silenced+in+Cancer&doi=10.1038%2F5047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer</span></div><div class="casAuthors">Cameron, Elizabeth E.; Bachman, Kurtis E.; Myohanen, Sanna; Herman, James G.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Densely methylated DNA assocs. with transcriptionally repressive chromatin characterized by the presence of underacetylated histones.  Recently, these 2 epigenetic processes were dynamically linked.  The methyl-CpG-binding protein MeCP2 appears to reside in a complex with histone deacetylase activity.  MeCP2 can mediate formation of transcriptionally repressive chromatin on methylated promoter templates in vitro, and this process can be reversed by trichostatin A (TSA), a specific inhibitor of histone deacetylase.  Little is known about the relative roles of methylation and histone deacetylase activity in the stable inhibition of transcription on densely methylated endogenous promoters, such as those for silenced alleles of imprinted genes, genes on the female inactive X chromosome, and tumor-suppressor genes inactivated in cancer cells.  The authors show that the hypermethylated genes MLH1, TIMP3, CDKN2B (INK4B, p15), and CDKN2A (INK4, p16) cannot be transcriptionally reactivated with TSA alone in tumor cells in which the authors have shown that TSA alone can upregulate the expression of non-methylated genes.  Following minimal demethylation and slight gene reactivation in the presence of low dose 5-aza-2' deoxycytidine (5Aza-dC), however, TSA treatment results in robust re-expression of each gene.  TSA does not contribute to demethylation of the genes, and none of the treatments alter the chromatin structure assocd. with the hypermethylated promoters.  Thus, although DNA methylation and histone deacetylation appear to act as synergistic layers for the silencing of genes in cancer, dense CpG island methylation is dominant for the stable maintenance of a silent state at these loci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZCmJwXaY4o7Vg90H21EOLACvtfcHk0lhqWO-zJiMzYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D&md5=5f0666dac86ee091d5e06ab0b9403502</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F5047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F5047%26sid%3Dliteratum%253Aachs%26aulast%3DCameron%26aufirst%3DE.%2BE.%26aulast%3DBachman%26aufirst%3DK.%2BE.%26aulast%3DMy%25C3%25B6h%25C3%25A4nen%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DJ.%2BG.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSynergy%2520of%2520Demethylation%2520and%2520Histone%2520Deacetylase%2520Inhibition%2520in%2520the%2520Re-Expression%2520of%2520Genes%2520Silenced%2520in%2520Cancer%26jtitle%3DNat.%2520Genet.%26date%3D1999%26volume%3D21%26spage%3D103%26epage%3D107%26doi%3D10.1038%2F5047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grondin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikuchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span> <span> </span><span class="NLM_article-title">HDAC3 Regulates DNMT1 Expression in Multiple Myeloma: Therapeutic Implications</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2670</span>– <span class="NLM_lpage">2677</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fleu.2017.144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28490812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=2670-2677&author=T.+Haradaauthor=H.+Ohguchiauthor=Y.+Grondinauthor=S.+Kikuchiauthor=M.+Sagawaauthor=Y.-T.+Taiauthor=R.+Mazitschekauthor=T.+Hideshimaauthor=K.+C.+Anderson&title=HDAC3+Regulates+DNMT1+Expression+in+Multiple+Myeloma%3A+Therapeutic+Implications&doi=10.1038%2Fleu.2017.144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC3 regulates DNMT1 expression in multiple myeloma: therapeutic implications</span></div><div class="casAuthors">Harada, T.; Ohguchi, H.; Grondin, Y.; Kikuchi, S.; Sagawa, M.; Tai, Y.-T.; Mazitschek, R.; Hideshima, T.; Anderson, K. C.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2670-2677</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Epigenetic signaling pathways are implicated in tumorigenesis and therefore histone deacetylases (HDACs) represent novel therapeutic targets for cancers, including multiple myeloma (MM).  Although non-selective HDAC inhibitors show anti-MM activities, unfavorable side effects limit their clin. efficacy.  Isoform- and/or class-selective HDAC inhibition offers the possibility to maintain clin. activity while avoiding adverse events attendant to broad non-selective HDAC inhibition.  We have previously reported that HDAC3 inhibition, either by genetic knockdown or selective inhibitor BG45, abrogates MM cell proliferation.  Here we show that knockdown of HDAC3, but not HDAC1 or HDAC2, as well as BG45, downregulate expression of DNA methyltransferase 1 (DNMT1) mediating MM cell proliferation.  DNMT1 expression is regulated by c-Myc, and HDAC3 inhibition triggers degrdn. of c-Myc protein.  Moreover, HDAC3 inhibition results in hyperacetylation of DNMT1, thereby reducing the stability of DNMT1 protein.  Combined inhibition of HDAC3 and DNMT1 with BG45 and DNMT1 inhibitor 5-azacytidine (AZA), resp., triggers synergistic downregulation of DNMT1, growth inhibition and apoptosis in both MM cell lines and patient MM cells.  Efficacy of this combination treatment is confirmed in a murine xenograft MM model.  Our results therefore provide the rationale for combination treatment using HDAC3 inhibitor with DNMT1 inhibitor to improve patient outcome in MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogqlTnVwSxvLVg90H21EOLACvtfcHk0lgfTStMZ_1zWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurfJ&md5=1bd7e215aa417f78eb0b51ac0c482144</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.144%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DT.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DGrondin%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DS.%26aulast%3DSagawa%26aufirst%3DM.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26atitle%3DHDAC3%2520Regulates%2520DNMT1%2520Expression%2520in%2520Multiple%2520Myeloma%253A%2520Therapeutic%2520Implications%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D2670%26epage%3D2677%26doi%3D10.1038%2Fleu.2017.144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearstone, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quayle, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Duzer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarpe, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibitors of Histone Deacetylases 1 and 2 Synergize with Azacitidine in Acute Myeloid Leukemia</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0169128</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0169128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1371%2Fjournal.pone.0169128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28060870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvVegsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=C.+Minauthor=N.+Mooreauthor=J.+R.+Shearstoneauthor=S.+N.+Quayleauthor=P.+Huangauthor=J.+H.+van+Duzerauthor=M.+B.+Jarpeauthor=S.+S.+Jonesauthor=M.+Yang&title=Selective+Inhibitors+of+Histone+Deacetylases+1+and+2+Synergize+with+Azacitidine+in+Acute+Myeloid+Leukemia&doi=10.1371%2Fjournal.pone.0169128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of histone deacetylases 1 and 2 synergize with azacitidine in acute myeloid leukemia</span></div><div class="casAuthors">Min, Chengyin; Moore, Nathan; Shearstone, Jeffrey R.; Quayle, Steven N.; Huang, Pengyu; van Duzer, John H.; Jarpe, Matthew B.; Jones, Simon S.; Yang, Min</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0169128/1-e0169128/22</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a heterogeneous group of hematopoietic stem cell disorders characterized by defects in myeloid differentiation and increased proliferation of neoplastic hematopoietic precursor cells.  Outcomes for patients with AML remain poor, highlighting the need for novel treatment options.  Aberrant epigenetic regulation plays an important role in the pathogenesis of AML, and inhibitors of DNA methyltransferase or histone deacetylase (HDAC) enzymes have exhibited activity in preclin. AML models.  Combination studies with HDAC inhibitors plus DNA methyltransferase inhibitors have potential beneficial clin. activity in AML, however the toxicity profiles of non-selective HDAC inhibitors in the combination setting limit their clin. utility.  In this work, we describe the preclin. development of selective inhibitors of HDAC1 and HDAC2, which are hypothesized to have improved safety profiles, for combination therapy in AML.  We demonstrate that selective inhibition of HDAC1 and HDAC2 is sufficient to achieve efficacy both as a single agent and in combination with azacitidine in preclin. models of AML, including established AML cell lines, primary leukemia cells from AML patient bone marrow samples and in vivo xenograft models of human AML.  Gene expression profiling of AML cells treated with either an HDAC1/2 inhibitor, azacitidine, or the combination of both have identified a list of genes involved in transcription and cell cycle regulation as potential mediators of the combinatorial effects of HDAC1/2 inhibition with azacitidine.  Together, these findings support the clin. evaluation of selective HDAC1/2 inhibitors in combination with azacitidine in AML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkinwKH8HeabVg90H21EOLACvtfcHk0lgfTStMZ_1zWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvVegsbw%253D&md5=1f549b8d377869735e758ba914749eab</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0169128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0169128%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DC.%26aulast%3DMoore%26aufirst%3DN.%26aulast%3DShearstone%26aufirst%3DJ.%2BR.%26aulast%3DQuayle%26aufirst%3DS.%2BN.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3Dvan%2BDuzer%26aufirst%3DJ.%2BH.%26aulast%3DJarpe%26aufirst%3DM.%2BB.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DYang%26aufirst%3DM.%26atitle%3DSelective%2520Inhibitors%2520of%2520Histone%2520Deacetylases%25201%2520and%25202%2520Synergize%2520with%2520Azacitidine%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0169128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pathania, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariappan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manicassamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolhe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokeshwar, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganapathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaraju, M.</span></span> <span> </span><span class="NLM_article-title">Combined Inhibition of DNMT and HDAC Blocks the Tumorigenicity of Cancer Stem-like Cells and Attenuates Mammary Tumor Growth</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3224</span>– <span class="NLM_lpage">3235</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-15-2249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1158%2F0008-5472.CAN-15-2249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27197203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVykt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3224-3235&author=R.+Pathaniaauthor=S.+Ramachandranauthor=G.+Mariappanauthor=P.+Thakurauthor=H.+Shiauthor=J.-H.+Choiauthor=S.+Manicassamyauthor=R.+Kolheauthor=P.+D.+Prasadauthor=S.+Sharmaauthor=B.+L.+Lokeshwarauthor=V.+Ganapathyauthor=M.+Thangaraju&title=Combined+Inhibition+of+DNMT+and+HDAC+Blocks+the+Tumorigenicity+of+Cancer+Stem-like+Cells+and+Attenuates+Mammary+Tumor+Growth&doi=10.1158%2F0008-5472.can-15-2249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Combined Inhibition of DNMT and HDAC Blocks the Tumorigenicity of Cancer Stem-like Cells and Attenuates Mammary Tumor Growth</span></div><div class="casAuthors">Pathania, Rajneesh; Ramachandran, Sabarish; Mariappan, Gurusamy; Thakur, Priyanka; Shi, Huidong; Choi, Jeong-Hyeon; Manicassamy, Santhakumar; Kolhe, Ravindra; Prasad, Puttur D.; Sharma, Suash; Lokeshwar, Bal L.; Ganapathy, Vadivel; Thangaraju, Muthusamy</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3224-3235</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Recently, impressive tech. advancements have been made in the isolation and validation of mammary stem cells and cancer stem cells (CSC), but the signaling pathways that regulate stem cell self-renewal are largely unknown.  Furthermore, CSCs are believed to contribute to chemo- and radioresistance.  In this study, we used the MMTV-Neu-Tg mouse mammary tumor model to identify potential new strategies for eliminating CSCs.  We found that both luminal progenitor and basal stem cells are susceptible to genetic and epigenetic modifications, which facilitate oncogenic transformation and tumorigenic potential.  A combination of the DNMT inhibitor 5-azacytidine and the HDAC inhibitor butyrate markedly reduced CSC abundance and increased the overall survival in this mouse model.  RNA-seq anal. of CSCs treated with 5-azacytidine plus butyrate provided evidence that inhibition of chromatin modifiers blocks growth-promoting signaling mols. such as RAD51AP1 and SPC25, which play key roles in DNA damage repair and kinetochore assembly.  Moreover, RAD51AP1 and SPC25 were significantly overexpressed in human breast tumor tissues and were assocd. with reduced overall patient survival.  In conclusion, our studies suggest that breast CSCs are intrinsically sensitive to genetic and epigenetic modifications and can therefore be significantly affected by epigenetic-based therapies, warranting further investigation of combined DNMT and HDAC inhibition in refractory or drug-resistant breast cancer.  Cancer Res; 76(11); 3224-35. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplWN34n92rV7Vg90H21EOLACvtfcHk0lgfTStMZ_1zWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVykt7k%253D&md5=7b90bb0ed6a31755c5fd696ebc66bd3a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2249%26sid%3Dliteratum%253Aachs%26aulast%3DPathania%26aufirst%3DR.%26aulast%3DRamachandran%26aufirst%3DS.%26aulast%3DMariappan%26aufirst%3DG.%26aulast%3DThakur%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DJ.-H.%26aulast%3DManicassamy%26aufirst%3DS.%26aulast%3DKolhe%26aufirst%3DR.%26aulast%3DPrasad%26aufirst%3DP.%2BD.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DLokeshwar%26aufirst%3DB.%2BL.%26aulast%3DGanapathy%26aufirst%3DV.%26aulast%3DThangaraju%26aufirst%3DM.%26atitle%3DCombined%2520Inhibition%2520of%2520DNMT%2520and%2520HDAC%2520Blocks%2520the%2520Tumorigenicity%2520of%2520Cancer%2520Stem-like%2520Cells%2520and%2520Attenuates%2520Mammary%2520Tumor%2520Growth%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3224%26epage%3D3235%26doi%3D10.1158%2F0008-5472.can-15-2249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahnow, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudin, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span> <span> </span><span class="NLM_article-title">The Future of Epigenetic Therapy in Solid Tumours—lessons from the Past</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1038/nrclinonc.2013.42</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnrclinonc.2013.42" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=23546521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvFOrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=256-266&author=N.+Azadauthor=C.+A.+Zahnowauthor=C.+M.+Rudinauthor=S.+B.+Baylin&title=The+Future+of+Epigenetic+Therapy+in+Solid+Tumours%E2%80%94lessons+from+the+Past&doi=10.1038%2Fnrclinonc.2013.42"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The future of epigenetic therapy in solid tumours-lessons from the past</span></div><div class="casAuthors">Azad, Nilofer; Zahnow, Cynthia A.; Rudin, Charles M.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">256-266</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The promise of targeting epigenetic abnormalities for cancer therapy has not been realized for solid tumors, although increasing evidence is demonstrating its worth in haematol. malignancies.  In fact, true clin. efficacy in haematopoietic-related neoplasms has only become evident at low doses of epigenetic-targeting drugs (namely, inhibitors of histone deacetylase and DNA methyltransferases).  Describing data from preclin. studies and early clin. trial results, we hypothesize that in using low-dose epigenetic-modulating agents, tumor cells can be reprogrammed, which overrides any immediate cytotoxic and off-target effect obsd. at high dose.  We suggest that such optimization of drug dosing and scheduling of currently available agents could give these agents a prominent place in cancer management-when used alone or in combination with other therapies.  If so, optimal use of these known agents might also pave the way for the introduction of other agents that target the epigenome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcsvBnYhoB-rVg90H21EOLACvtfcHk0libl0yXCmfMRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvFOrsr0%253D&md5=f029612627ad09125aafe2f95a01115e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2013.42&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2013.42%26sid%3Dliteratum%253Aachs%26aulast%3DAzad%26aufirst%3DN.%26aulast%3DZahnow%26aufirst%3DC.%2BA.%26aulast%3DRudin%26aufirst%3DC.%2BM.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520Future%2520of%2520Epigenetic%2520Therapy%2520in%2520Solid%2520Tumours%25E2%2580%2594lessons%2520from%2520the%2520Past%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2013%26volume%3D10%26spage%3D256%26epage%3D266%26doi%3D10.1038%2Fnrclinonc.2013.42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Lera, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Polypharmacology: From Combination Therapy to Multitargeted Drugs</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">105</span>, <span class="refDoi"> DOI: 10.1186/s13148-016-0271-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1186%2Fs13148-016-0271-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27752293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=105&author=A.+R.+de%0ALeraauthor=A.+Ganesan&title=Epigenetic+Polypharmacology%3A+From+Combination+Therapy+to+Multitargeted+Drugs&doi=10.1186%2Fs13148-016-0271-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic polypharmacology: from combination therapy to multitargeted drugs</span></div><div class="casAuthors">de Lera, Angel R.; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Epigenetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105/1-105/21</span>CODEN:
                <span class="NLM_cas:coden">CELPCI</span>;
        ISSN:<span class="NLM_cas:issn">1868-7083</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The modern drug discovery process has largely focused its attention in the so-called magic bullets, single chem. entities that exhibit high selectivity and potency for a particular target.  This approach was based on the assumption that the deregulation of a protein was causally linked to a disease state, and the pharmacol. intervention through inhibition of the deregulated target was able to restore normal cell function.  However, the use of cocktails or multicomponent drugs to address several targets simultaneously is also popular to treat multifactorial diseases such as cancer and neurol. disorders.  We review the state of the art with such combinations that have an epigenetic target as one of their mechanisms of action.  Epigenetic drug discovery is a rapidly advancing field, and drugs targeting epigenetic enzymes are in the clinic for the treatment of hematol. cancers.  Approved and exptl. epigenetic drugs are undergoing clin. trials in combination with other therapeutic agents via fused or linked pharmacophores in order to benefit from synergistic effects of polypharmacol.  In addn., ligands are being discovered which, as single chem. entities, are able to modulate multiple epigenetic targets simultaneously (multitarget epigenetic drugs).  These multiple ligands should in principle have a lower risk of drug-drug interactions and drug resistance compared to cocktails or multicomponent drugs.  This new generation may rival the so-called magic bullets in the treatment of diseases that arise as a consequence of the deregulation of multiple signaling pathways provided the challenge of optimization of the activities shown by the pharmacophores with the different targets is addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphm-KIEGyYKrVg90H21EOLACvtfcHk0libl0yXCmfMRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D&md5=7cd89a93a22121cc9db2c7118e2e23f1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1186%2Fs13148-016-0271-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-016-0271-9%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLera%26aufirst%3DA.%2BR.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEpigenetic%2520Polypharmacology%253A%2520From%2520Combination%2520Therapy%2520to%2520Multitargeted%2520Drugs%26jtitle%3DClin.%2520Epigenet.%26date%3D2016%26volume%3D8%26spage%3D105%26doi%3D10.1186%2Fs13148-016-0271-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blagitko-Dorfs, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greve, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baude, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brocks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plass, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lübbert, M.</span></span> <span> </span><span class="NLM_article-title">Combination Treatment of Acute Myeloid Leukemia Cells with DNMT and HDAC Inhibitors: Predominant Synergistic Gene Downregulation Associated with Gene Body Demethylation</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">945</span>– <span class="NLM_lpage">956</span>, <span class="refDoi"> DOI: 10.1038/s41375-018-0293-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fs41375-018-0293-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=30470836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitl2isrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=945-956&author=N.+Blagitko-Dorfsauthor=P.+Schlosserauthor=G.+Greveauthor=D.+Pfeiferauthor=R.+Meierauthor=A.+Baudeauthor=D.+Brocksauthor=C.+Plassauthor=M.+L%C3%BCbbert&title=Combination+Treatment+of+Acute+Myeloid+Leukemia+Cells+with+DNMT+and+HDAC+Inhibitors%3A+Predominant+Synergistic+Gene+Downregulation+Associated+with+Gene+Body+Demethylation&doi=10.1038%2Fs41375-018-0293-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Combination treatment of acute myeloid leukemia cells with DNMT and HDAC inhibitors: predominant synergistic gene downregulation associated with gene body demethylation</span></div><div class="casAuthors">Blagitko-Dorfs, Nadja; Schlosser, Pascal; Greve, Gabriele; Pfeifer, Dietmar; Meier, Ruth; Baude, Annika; Brocks, David; Plass, Christoph; Luebbert, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">945-956</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">DNA methyltransferase inhibitors (DNMTi) approved for older AML patients are clin. tested in combination with histone deacetylase inhibitors (HDACi).  The mechanism of action of these drugs is still under debate.  In colon cancer cells, 5-aza-2'-deoxycytidine (DAC) can downregulate oncogenes and metabolic genes by reversing gene body DNA methylation, thus implicating gene body methylation as a novel drug target.  We asked whether DAC-induced gene body demethylation in AML cells is also assocd. with gene repression, and whether the latter is enhanced by HDACi.  Transcriptome analyses revealed that a combined treatment with DAC and the HDACi panobinostat or valproic acid affected significantly more transcripts than the sum of the genes regulated by either treatment alone, demonstrating a quant. synergistic effect on genome-wide expression in U937 cells.  This effect was particularly striking for downregulated genes.  Integrative methylome and transcriptome analyses showed that a massive downregulation of genes, including oncogenes (e.g., MYC) and epigenetic modifiers (e.g., KDM2B, SUV39H1) often overexpressed in cancer, was assocd. predominantly with gene body DNA demethylation and changes in acH3K9/27.  These findings have implications for the mechanism of action of combined epigenetic treatments, and for a better understanding of responses in trials where this approach is clin. tested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2VMMovamJgrVg90H21EOLACvtfcHk0libl0yXCmfMRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitl2isrfN&md5=b15ea653d78f9326895650bd557cf30f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41375-018-0293-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-018-0293-8%26sid%3Dliteratum%253Aachs%26aulast%3DBlagitko-Dorfs%26aufirst%3DN.%26aulast%3DSchlosser%26aufirst%3DP.%26aulast%3DGreve%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DD.%26aulast%3DMeier%26aufirst%3DR.%26aulast%3DBaude%26aufirst%3DA.%26aulast%3DBrocks%26aufirst%3DD.%26aulast%3DPlass%26aufirst%3DC.%26aulast%3DL%25C3%25BCbbert%26aufirst%3DM.%26atitle%3DCombination%2520Treatment%2520of%2520Acute%2520Myeloid%2520Leukemia%2520Cells%2520with%2520DNMT%2520and%2520HDAC%2520Inhibitors%253A%2520Predominant%2520Synergistic%2520Gene%2520Downregulation%2520Associated%2520with%2520Gene%2520Body%2520Demethylation%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26spage%3D945%26epage%3D956%26doi%3D10.1038%2Fs41375-018-0293-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehár, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, C. T.</span></span> <span> </span><span class="NLM_article-title">Multi-Target Therapeutics: When the Whole Is Greater than the Sum of the Parts</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.drudis.2006.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=17198971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ygtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=34-42&author=G.+R.+Zimmermannauthor=J.+Leh%C3%A1rauthor=C.+T.+Keith&title=Multi-Target+Therapeutics%3A+When+the+Whole+Is+Greater+than+the+Sum+of+the+Parts&doi=10.1016%2Fj.drudis.2006.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target therapeutics: when the whole is greater than the sum of the parts</span></div><div class="casAuthors">Zimmermann Grant R; Lehar Joseph; Keith Curtis T</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">34-42</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Drugs designed to act against individual molecular targets cannot usually combat multigenic diseases such as cancer, or diseases that affect multiple tissues or cell types such as diabetes and immunoinflammatory disorders.  Combination drugs that impact multiple targets simultaneously are better at controlling complex disease systems, are less prone to drug resistance and are the standard of care in many important therapeutic areas.  The combination drugs currently employed are primarily of rational design, but the increased efficacy they provide justifies in vitro discovery efforts for identifying novel multi-target mechanisms.  In this review, we discuss the biological rationale for combination therapeutics, review some existing combination drugs and present a systematic approach to identify interactions between molecular pathways that could be leveraged for therapeutic benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTI14vYK2Q1LENHe_gFldTbfW6udTcc2eYITeW79Nh3oLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ygtQ%253D%253D&md5=66715499ce21c2855dc6c477aa87f161</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DG.%2BR.%26aulast%3DLeh%25C3%25A1r%26aufirst%3DJ.%26aulast%3DKeith%26aufirst%3DC.%2BT.%26atitle%3DMulti-Target%2520Therapeutics%253A%2520When%2520the%2520Whole%2520Is%2520Greater%2520than%2520the%2520Sum%2520of%2520the%2520Parts%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D34%26epage%3D42%26doi%3D10.1016%2Fj.drudis.2006.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases as an Epigenetic Pillar for the Development of Hybrid Inhibitors in Cancer</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.cbpa.2019.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=30986654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsFSgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=89-100&author=G.+Staziauthor=R.+Fioravantiauthor=A.+Maiauthor=A.+Matteviauthor=S.+Valente&title=Histone+Deacetylases+as+an+Epigenetic+Pillar+for+the+Development+of+Hybrid+Inhibitors+in+Cancer&doi=10.1016%2Fj.cbpa.2019.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases as an epigenetic pillar for the development of hybrid inhibitors in cancer</span></div><div class="casAuthors">Stazi, Giulia; Fioravanti, Rossella; Mai, Antonello; Mattevi, Andrea; Valente, Sergio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The polypharmacol. strategy of multi-targeting drugs acting on different biol. pathways is capturing the researchers' attention, particularly in cancer.  The simultaneous inhibition of two or more targets by drug combination or by a single 'hybrid mol.' can provide improved therapeutic efficacy when compared to the one-target inhibitors.  In this regard, because of their multiple anticancer effects, histone deacetylase inhibitors have become a privileged tool for the development of hybrid drugs.  The clin. trials of two multi-acting chimeras, HDAC/EGFR/HER2 and HDAC/PI3K inhibitors, encouraged the design of novel hybrids, such as compds. 22a (LSD1/HDAC) and 16a (CDK4/JAK1/HDAC), which showed superior anticancer effects than single-targeting agents or their combination both in cellular and mouse models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDkcMKMsqtjLVg90H21EOLACvtfcHk0lhLAk_BvoOgYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsFSgsb4%253D&md5=8233a35ccdd5db5fd4c1d5dfdc3958b7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DStazi%26aufirst%3DG.%26aulast%3DFioravanti%26aufirst%3DR.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DValente%26aufirst%3DS.%26atitle%3DHistone%2520Deacetylases%2520as%2520an%2520Epigenetic%2520Pillar%2520for%2520the%2520Development%2520of%2520Hybrid%2520Inhibitors%2520in%2520Cancer%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.cbpa.2019.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piña, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautschi, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.-Y. -S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">France, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornell-Kennon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sambucetti, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remiszewski, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bair, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, P.</span></span> <span> </span><span class="NLM_article-title">Psammaplins from the Sponge Pseudoceratina Purpurea: Inhibition of Both Histone Deacetylase and DNA Methyltransferase</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">3866</span>– <span class="NLM_lpage">3873</span>, <span class="refDoi"> DOI: 10.1021/jo034248t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo034248t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFyhurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=3866-3873&author=I.+C.+Pi%C3%B1aauthor=J.+T.+Gautschiauthor=G.-Y.+-S.+Wangauthor=M.+L.+Sandersauthor=F.+J.+Schmitzauthor=D.+Franceauthor=S.+Cornell-Kennonauthor=L.+C.+Sambucettiauthor=S.+W.+Remiszewskiauthor=L.+B.+Perezauthor=K.+W.+Bairauthor=P.+Crews&title=Psammaplins+from+the+Sponge+Pseudoceratina+Purpurea%3A+Inhibition+of+Both+Histone+Deacetylase+and+DNA+Methyltransferase&doi=10.1021%2Fjo034248t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Psammaplins from the sponge Pseudoceratina purpurea: Inhibition of both histone deacetylase and DNA methyltransferase</span></div><div class="casAuthors">Pina, Ivette C.; Gautschi, Jeffrey T.; Wang, Gui-Yang-Sheng; Sanders, Miranda L.; Schmitz, Francis J.; France, Dennis; Cornell-Kennon, Susan; Sambucetti, Lidia C.; Remiszewski, Stacy W.; Perez, Larry B.; Bair, Kenneth W.; Crews, Phillip</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3866-3873</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four novel bisulfide bromotyrosine derivs., psammaplins E (e.g. I), F, G, and H , and two new bromotyrosine derivs., psammaplins I and J were isolated from the sponge Pseudoceratina purpurea, along with known psammaplins A, B, C, and D and bisaprasin.  The structures of psammaplins E and F, which each contain an oxalyl group rarely found in marine organisms, were detd. by spectroscopic anal.  Compds. psammaplin A, bisaprasin, and psammaplin F are potent histone deacetylase inhibitors and also show mild cytotoxicity.  Furthermore, compds. psammaplin A, bisaprasin, and psammaplin G are potent DNA methyltransferase inhibitors.  The biogenetic pathway previously proposed for the psammaplins class is also revisited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp80TzIEDoTYLVg90H21EOLACvtfcHk0lhLAk_BvoOgYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFyhurw%253D&md5=6e6344c8e1ef3eb33ca942f0e76506d9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjo034248t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034248t%26sid%3Dliteratum%253Aachs%26aulast%3DPi%25C3%25B1a%26aufirst%3DI.%2BC.%26aulast%3DGautschi%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DG.-Y.%2B-S.%26aulast%3DSanders%26aufirst%3DM.%2BL.%26aulast%3DSchmitz%26aufirst%3DF.%2BJ.%26aulast%3DFrance%26aufirst%3DD.%26aulast%3DCornell-Kennon%26aufirst%3DS.%26aulast%3DSambucetti%26aufirst%3DL.%2BC.%26aulast%3DRemiszewski%26aufirst%3DS.%2BW.%26aulast%3DPerez%26aufirst%3DL.%2BB.%26aulast%3DBair%26aufirst%3DK.%2BW.%26aulast%3DCrews%26aufirst%3DP.%26atitle%3DPsammaplins%2520from%2520the%2520Sponge%2520Pseudoceratina%2520Purpurea%253A%2520Inhibition%2520of%2520Both%2520Histone%2520Deacetylase%2520and%2520DNA%2520Methyltransferase%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D3866%26epage%3D3873%26doi%3D10.1021%2Fjo034248t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiser, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haus, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samlal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrucci, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaratnam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buluwela, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turlais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Almes, F.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchter, M. J.</span></span> <span> </span><span class="NLM_article-title">Defining the Mechanism of Action and Enzymatic Selectivity of Psammaplin A against Its Epigenetic Targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1731</span>– <span class="NLM_lpage">1750</span>, <span class="refDoi"> DOI: 10.1021/jm2016182</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2016182" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Ogur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1731-1750&author=M.+G.+J.+Baudauthor=T.+Leiserauthor=P.+Hausauthor=S.+Samlalauthor=A.+C.+Wongauthor=R.+J.+Woodauthor=V.+Petrucciauthor=M.+Gunaratnamauthor=S.+M.+Hughesauthor=L.+Buluwelaauthor=F.+Turlaisauthor=S.+Neidleauthor=F.-J.+Meyer-Almesauthor=A.+J.+P.+Whiteauthor=M.+J.+Fuchter&title=Defining+the+Mechanism+of+Action+and+Enzymatic+Selectivity+of+Psammaplin+A+against+Its+Epigenetic+Targets&doi=10.1021%2Fjm2016182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the Mechanism of Action and Enzymatic Selectivity of Psammaplin A against Its Epigenetic Targets</span></div><div class="casAuthors">Baud, Matthias G. J.; Leiser, Thomas; Haus, Patricia; Samlal, Sharon; Wong, Ai Ching; Wood, Robert J.; Petrucci, Vanessa; Gunaratnam, Mekala; Hughes, Siobhan M.; Buluwela, Lakjaya; Turlais, Fabrice; Neidle, Stephen; Meyer-Almes, Franz-Josef; White, Andrew J. P.; Fuchter, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1731-1750</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Psammaplin A (11c) is a marine metabolite previously reported to be a potent inhibitor of two classes of epigenetic enzymes: histone deacetylases and DNA methyltransferases.  The design and synthesis of a focused library based on the psammaplin A core has been carried out to probe the mol. features of this mol. responsible for its activity.  By direct in vitro assay of the free thiol generated upon redn. of the dimeric psammaplin scaffold, we have unambiguously demonstrated that 11c functions as a natural prodrug, with the reduced form being highly potent against HDAC1 in vitro (IC50 0.9 nM).  Furthermore, we have shown it to have high isoform selectivity, being 360-fold selective for HDAC1 over HDAC6 and more than 1000-fold less potent against HDAC7 and HDAC8.  SAR around our focused library revealed a no. of features, most notably the oxime functionality to be important to this selectivity.  Many of the compds. show significant cytotoxicity in A549, MCF7, and W138 cells, with the SAR of cytotoxicity correlating to HDAC inhibition.  Furthermore, compd. treatment causes upregulation of histone acetylation but little effect on tubulin acetylation.  Finally, we have found no evidence for 11c functioning as a DNMT inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplUTsYNBVg47Vg90H21EOLACvtfcHk0lhvDodBfqsYDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Ogur8%253D&md5=815f9aca36da2e0aa404a341909697e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm2016182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2016182%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLeiser%26aufirst%3DT.%26aulast%3DHaus%26aufirst%3DP.%26aulast%3DSamlal%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DA.%2BC.%26aulast%3DWood%26aufirst%3DR.%2BJ.%26aulast%3DPetrucci%26aufirst%3DV.%26aulast%3DGunaratnam%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DS.%2BM.%26aulast%3DBuluwela%26aufirst%3DL.%26aulast%3DTurlais%26aufirst%3DF.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DMeyer-Almes%26aufirst%3DF.-J.%26aulast%3DWhite%26aufirst%3DA.%2BJ.%2BP.%26aulast%3DFuchter%26aufirst%3DM.%2BJ.%26atitle%3DDefining%2520the%2520Mechanism%2520of%2520Action%2520and%2520Enzymatic%2520Selectivity%2520of%2520Psammaplin%2520A%2520against%2520Its%2520Epigenetic%2520Targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1731%26epage%3D1750%26doi%3D10.1021%2Fjm2016182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Rodríguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nebbioso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Rodríguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carafa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuhldreier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Barrios, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stunnenberg, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lera, Á. R.</span></span> <span> </span><span class="NLM_article-title">Indole-Derived Psammaplin A Analogues as Epigenetic Modulators with Multiple Inhibitory Activities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9467</span>– <span class="NLM_lpage">9491</span>, <span class="refDoi"> DOI: 10.1021/jm300618u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300618u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVGqtLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9467-9491&author=R.+Pereiraauthor=R.+Benedettiauthor=S.+P%C3%A9rez-Rodr%C3%ADguezauthor=A.+Nebbiosoauthor=J.+Garc%C3%ADa-Rodr%C3%ADguezauthor=V.+Carafaauthor=M.+Stuhldreierauthor=M.+Conteauthor=F.+Rodr%C3%ADguez-Barriosauthor=H.+G.+Stunnenbergauthor=H.+Gronemeyerauthor=L.+Altucciauthor=%C3%81.+R.+de+Lera&title=Indole-Derived+Psammaplin+A+Analogues+as+Epigenetic+Modulators+with+Multiple+Inhibitory+Activities&doi=10.1021%2Fjm300618u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Indole-Derived Psammaplin A Analogues as Epigenetic Modulators with Multiple Inhibitory Activities</span></div><div class="casAuthors">Pereira, Raquel; Benedetti, Rosaria; Perez-Rodriguez, Santiago; Nebbioso, Angela; Garcia-Rodriguez, Jose; Carafa, Vincenzo; Stuhldreier, Mayra; Conte, Mariarosaria; Rodriguez-Barrios, Fatima; Stunnenberg, Hendrik G.; Gronemeyer, Hinrich; Altucci, Lucia; de Lera, Angel R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9467-9491</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A SAR study has been carried out around a modified scaffold of the natural product psammaplin A obtained by replacing the o-bromophenol unit by an indole ring.  A series of indole psammaplin A constructs were generated in a short synthetic sequence that starts with the functionalization of the C3 indole position with in situ generated nitrosoacrylate, and this is followed by protection of the β-indole-α-oximino esters, sapon., condensation with sym. diamines, and deprotection.  Biochem. and cellular characterization using U937 and MCF-7 cells confirmed that many of these analogs displayed more potent activities than the parent natural product.  Moreover, in addn. to the reported HDAC and DNMT dual epigenetic inhibitory profile of the parent compd., some analogs, notably 4a (UVI5008, I), also inhibited the NAD+-dependent SIRT deacetylase enzymes.  The SAR study provides structural insights into the mechanism of action of these multiple epigenetic ligands and paves the way for addnl. structural exploration to optimize their pharmacol. profiles.  Because of their multi(epi)target features and their action in ex vivo samples, the indole-based psammaplin A derivs. are attractive mols. for the modulation of epigenetic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd30sExJo0P7Vg90H21EOLACvtfcHk0lhvDodBfqsYDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVGqtLvI&md5=4dd9ee7fd692b94464c1b51037e922ce</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm300618u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300618u%26sid%3Dliteratum%253Aachs%26aulast%3DPereira%26aufirst%3DR.%26aulast%3DBenedetti%26aufirst%3DR.%26aulast%3DP%25C3%25A9rez-Rodr%25C3%25ADguez%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DGarc%25C3%25ADa-Rodr%25C3%25ADguez%26aufirst%3DJ.%26aulast%3DCarafa%26aufirst%3DV.%26aulast%3DStuhldreier%26aufirst%3DM.%26aulast%3DConte%26aufirst%3DM.%26aulast%3DRodr%25C3%25ADguez-Barrios%26aufirst%3DF.%26aulast%3DStunnenberg%26aufirst%3DH.%2BG.%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3DAltucci%26aufirst%3DL.%26aulast%3Dde%2BLera%26aufirst%3D%25C3%2581.%2BR.%26atitle%3DIndole-Derived%2520Psammaplin%2520A%2520Analogues%2520as%2520Epigenetic%2520Modulators%2520with%2520Multiple%2520Inhibitory%2520Activities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9467%26epage%3D9491%26doi%3D10.1021%2Fjm300618u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaram, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alalami, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunasekera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quraishi, U.</span></span> <span> </span><span class="NLM_article-title">(-)-Epigallocatechin-3-Gallate Reverses the Expression of Various Tumor-Suppressor Genes by Inhibiting DNA Methyltransferases and Histone Deacetylases in Human Cervical Cancer Cells</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1976</span>– <span class="NLM_lpage">1984</span>, <span class="refDoi"> DOI: 10.3892/or.2015.3802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.3892%2For.2015.3802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=25682960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFyr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=1976-1984&author=M.+A.+Khanauthor=A.+Hussainauthor=M.+K.+Sundaramauthor=U.+Alalamiauthor=D.+Gunasekeraauthor=L.+Rameshauthor=A.+Hamzaauthor=U.+Quraishi&title=%28-%29-Epigallocatechin-3-Gallate+Reverses+the+Expression+of+Various+Tumor-Suppressor+Genes+by+Inhibiting+DNA+Methyltransferases+and+Histone+Deacetylases+in+Human+Cervical+Cancer+Cells&doi=10.3892%2For.2015.3802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">(-)-Epigallocatechin-3-gallate reverses the expression of various tumor-suppressor genes by inhibiting DNA methyltransferases and histone deacetylases in human cervical cancer cells</span></div><div class="casAuthors">Khan, Munawwar Ali; Hussain, Arif; Sundaram, Madhumitha Kedhari; Al Alami, Usama; Gunasekera, Dian; Ramesh, Laveena; Hamza, Amina; Quraishi, Uzma</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1976-1984</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">There has been increasing evidence that numerous bioactive dietary agents can hamper the process of carcinogenesis by targeting epigenetic alterations including DNA methylation.  This therapeutic approach is considered as a significant goal for cancer therapy due to the reversible nature of epigenetic-mediated gene silencing and warrants further attention.  One such dietary agent, green tea catechin, (-)-epigallocatechin-3-gallate (EGCG) has been shown to modulate many cancer-related pathways.  Thus, the present study was designed to investigate the role of EGCG as an epigenetic modifier in HeLa cells.  DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibition assays were conducted, and the transcription levels of DNMT3B and HDAC1 were assessed by enzymic activity assay and RT-PCR, resp.  Furthermore, we studied the binding interaction of EGCG with DNMT3B and HDAC1 by mol. modeling as well as promoter DNA methylation and expression of retinoic acid receptor-β (RARβ), cadherin 1 (CDH1) and death-assocd. protein kinase-1 (DAPK1) in EGCG-treated HeLa cells by RT-PCR and MS-PCR.  In the present study, time-dependent EGCG-treated HeLa cells were found to have a significant redn. in the enzymic activity of DNMT and HDAC.  However, the expression of DNMT3B was significantly decreased in a time-dependent manner whereas there was no significant change in HDAC1 expression.  Mol. modeling data also supported the EGCG-mediated DNMT3B and HDAC1 activity inhibition.  Furthermore, time-dependent exposure to EGCG resulted in reactivation of known tumor-suppressor genes (TSGs) in HeLa cells due to marked changes in the methylation of the promoter regions of these genes.  Overall, the present study suggests that EGCG may have a significant impact on the development of novel epigenetic-based therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbCuwOug7zBrVg90H21EOLACvtfcHk0lhvoWWs6Hp1HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFyr&md5=fe18eab228f3947354dffcec52b9997f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3892%2For.2015.3802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2015.3802%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BA.%26aulast%3DHussain%26aufirst%3DA.%26aulast%3DSundaram%26aufirst%3DM.%2BK.%26aulast%3DAlalami%26aufirst%3DU.%26aulast%3DGunasekera%26aufirst%3DD.%26aulast%3DRamesh%26aufirst%3DL.%26aulast%3DHamza%26aufirst%3DA.%26aulast%3DQuraishi%26aufirst%3DU.%26atitle%3D%2528-%2529-Epigallocatechin-3-Gallate%2520Reverses%2520the%2520Expression%2520of%2520Various%2520Tumor-Suppressor%2520Genes%2520by%2520Inhibiting%2520DNA%2520Methyltransferases%2520and%2520Histone%2520Deacetylases%2520in%2520Human%2520Cervical%2520Cancer%2520Cells%26jtitle%3DOncol.%2520Rep.%26date%3D2015%26volume%3D33%26spage%3D1976%26epage%3D1984%26doi%3D10.3892%2For.2015.3802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span> <span> </span><span class="NLM_article-title">Development of a Versatile DNMT and HDAC Inhibitor C02S Modulating Multiple Cancer Hallmarks for Breast Cancer Therapy</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2019.03.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.bioorg.2019.03.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=30901675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=200-208&author=Z.+Yuanauthor=S.+Chenauthor=C.+Gaoauthor=Q.+Daiauthor=C.+Zhangauthor=Q.+Sunauthor=J.-S.+Linauthor=C.+Guoauthor=Y.+Chenauthor=Y.+Jiang&title=Development+of+a+Versatile+DNMT+and+HDAC+Inhibitor+C02S+Modulating+Multiple+Cancer+Hallmarks+for+Breast+Cancer+Therapy&doi=10.1016%2Fj.bioorg.2019.03.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a versatile DNMT and HDAC inhibitor C02S modulating multiple cancer hallmarks for breast cancer therapy</span></div><div class="casAuthors">Yuan, Zigao; Chen, Shaopeng; Gao, Chunmei; Dai, Qiuzi; Zhang, Cunlong; Sun, Qinsheng; Lin, Jin-Shun; Guo, Chun; Chen, Yuzong; Jiang, Yuyang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">200-208</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNMT and HDAC are closely related to each other and involved in various human diseases esp. cancer.  These two enzymes have been widely recognized as antitumor targets for drug discovery.  Besides, research has indicated that combination therapy consisting of DNMT and HDAC inhibitors exhibited therapeutic advantages.  We have reported a DNMT and HDAC dual inhibitor 15a of which the DNMT enzymic inhibitory potency needs to be improved.  Herein we reported the development of a novel dual DNMT and HDAC inhibitor C02S which showed potent enzymic inhibitory activities against DNMT1, DNMT3A, DNMT3B and HDAC1 with IC50 values of 2.05, 0.93, 1.32, and 4.16 μM, resp.  Further evaluations indicated that C02S could inhibit DNMT and HDAC at cellular levels, thereby inversing mutated methylation and acetylation and increasing expression of tumor suppressor proteins.  Moreover, C02S regulated multiple biol. processes including inducing apoptosis and G0/G1 cell cycle arrest, inhibiting angiogenesis, blocking migration and invasion, and finally suppressing tumor cells proliferation in vitro and tumor growth in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkdrcp-eivG7Vg90H21EOLACvtfcHk0lhvoWWs6Hp1HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLg%253D&md5=55aa67a359816285488aac672f05a65f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2019.03.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2019.03.027%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DDai%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DJ.-S.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520a%2520Versatile%2520DNMT%2520and%2520HDAC%2520Inhibitor%2520C02S%2520Modulating%2520Multiple%2520Cancer%2520Hallmarks%2520for%2520Breast%2520Cancer%2520Therapy%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D87%26spage%3D200%26epage%3D208%26doi%3D10.1016%2Fj.bioorg.2019.03.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Anticancer Potential of NSC-319745 Hydroxamic Acid Derivatives as DNMT and HDAC Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.ejmech.2017.04.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28419930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2017&pages=281-292&author=Z.+Yuanauthor=Q.+Sunauthor=D.+Liauthor=S.+Miaoauthor=S.+Chenauthor=L.+Songauthor=C.+Gaoauthor=Y.+Chenauthor=C.+Tanauthor=Y.+Jiang&title=Design%2C+Synthesis+and+Anticancer+Potential+of+NSC-319745+Hydroxamic+Acid+Derivatives+as+DNMT+and+HDAC+Inhibitors&doi=10.1016%2Fj.ejmech.2017.04.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and anticancer potential of NSC-319745 hydroxamic acid derivatives as DNMT and HDAC inhibitors</span></div><div class="casAuthors">Yuan, Zigao; Sun, Qinsheng; Li, Dan; Miao, Shuangshuang; Chen, Shaopeng; Song, Lu; Gao, Chunmei; Chen, Yuzong; Tan, Chunyan; Jiang, Yuyang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">281-292</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMTs) and histone deacetylases (HDACs) are important epigenetic targets during anticancer drug development.  Recent study indicates that DNMT inhibitors and HDAC inhibitors display synergistic effects in certain cancers, therefore, development of mols. targeting both DNMT and HDAC is of therapeutic advantage against these cancers.  Based on the structure of DNMT inhibitor NSC-319745 and the pharmacophore characteristics of HDAC inhibitors, a series of hydroxamic acid derivs. of NSC-319745 were designed and synthesized as DNMT and HDAC multifunctional inhibitors.  Most compds. displayed potential DNMT inhibitory potency and potent HDAC inhibitory activity, esp. compd. (E)-2-chloro-N-(4-(3-(hydroxyamino)-3-oxoprop-1-en-1-yl) phenethyl)-5(trifluoromethyl)benzamide showed much better DNMT1 inhibitory potency than NSC-319745, and inhibited HDAC1, HDAC6 with IC50 values of 57, 17 nM, resp.  Furthermore, the synthesized compds. exhibited significant cytotoxicity against human cancer cells K562 and U937.  Further mechanistic studies demonstrated that (E)-2-chloro-N-(4-(3-(hydroxyamino)-3-oxoprop-1-en-1-yl) phenethyl)-5(trifluoromethyl)benzamide treatment in U937 increased histones H3K9 and H4K8 acetylation, prompted P16 CpG islands demethylation and upregulated P16 expression, regulated apoptosis-related protein expression on the cellular level and induced remarkable U937 apoptosis.  Moreover, genotoxicity of representative compds. was evaluated.  In summary, the authors' study provided a practical drug design strategy targeting multiple enzymes, and (E)-2-chloro-N-(4-(3-(hydroxyamino)-3-oxoprop-1-en-1-yl) phenethyl)-5(trifluoromethyl)benzamide represents a novel and promising lead compd. for the development of novel epigenetic inhibitors as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE4sLaet4XkrVg90H21EOLACvtfcHk0lhvoWWs6Hp1HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrtL4%253D&md5=79777adbb8f9c5ef8209889ce1f1dbf0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Anticancer%2520Potential%2520of%2520NSC-319745%2520Hydroxamic%2520Acid%2520Derivatives%2520as%2520DNMT%2520and%2520HDAC%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D134%26spage%3D281%26epage%3D292%26doi%3D10.1016%2Fj.ejmech.2017.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomaselli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic polypharmacology: A new frontier for epi-drug discovery</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1002/med.21600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1002%2Fmed.21600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=31218726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntFSjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2020&pages=190-244&author=D.+Tomaselliauthor=A.+Lucidiauthor=D.+Rotiliauthor=A.+Mai&title=Epigenetic+polypharmacology%3A+A+new+frontier+for+epi-drug+discovery&doi=10.1002%2Fmed.21600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic polypharmacology: A new frontier for epi-drug discovery</span></div><div class="casAuthors">Tomaselli, Daniela; Lucidi, Alessia; Rotili, Dante; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">190-244</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Recently, despite the great success achieved by the so-called "magic bullets" in the treatment of different diseases through a marked and specific interaction with the target of interest, the pharmacol. research is moving toward the development of "mol. network active compds.," embracing the related polypharmacol. approach.  This strategy was born to overcome the main limitations of the single target therapy leading to a superior therapeutic effect, a decrease of adverse reactions, and redn. of potential mechanism(s) of drug resistance caused by robustness and redundancy of biol. pathways.  It has become clear that multifactorial diseases such as cancer, neurol., and inflammatory disorders may require more complex therapeutic approaches hitting certain biol. system as a whole.  Concerning epigenetics, goal of multi-epi-target approach consists in the development of small mols. able to simultaneously and (often) reversibly bind different specific epi-targets.  To date, two dual histone deacetylase/kinase inhibitors (CUDC-101 and CUDC-907) are in an advanced stage of clin. trials.  Hence, to update the state-of-the-art of these therapeutic approaches avoiding redundancy, herein we focused only on multiple medication therapies and multitargeting compds. exploiting epigenetic plus nonepigenetic drugs reported in the literature in 2018.  In addn., all the multi-epi-target inhibitors known in literature so far, hitting two or more epigenetic targets, have been included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhhcr1_bhMHbVg90H21EOLACvtfcHk0lh3JS2c8irWAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntFSjug%253D%253D&md5=bd3f48877c4c40404b22d9278751fa8d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fmed.21600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21600%26sid%3Dliteratum%253Aachs%26aulast%3DTomaselli%26aufirst%3DD.%26aulast%3DLucidi%26aufirst%3DA.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DEpigenetic%2520polypharmacology%253A%2520A%2520new%2520frontier%2520for%2520epi-drug%2520discovery%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2020%26volume%3D40%26spage%3D190%26epage%3D244%26doi%3D10.1002%2Fmed.21600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">San
José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casares, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Subero, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogi, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soule, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiveri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos-Olivas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia Fernandez de Barrena, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barchino, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of First-in-Class Reversible Dual Small Molecule Inhibitors against G9a and DNMTs with in Vivo Activity in Hematological Malignancies</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15424</span>, <span class="refDoi"> DOI: 10.1038/ncomms15424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fncomms15424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28548080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15424&author=E.+San%0AJos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=O.+Rabalauthor=A.+Vilas-Zornozaauthor=J.+A.+S%C3%A1nchez-Ariasauthor=E.+Mirandaauthor=A.+Ugarteauthor=S.+Roaauthor=B.+Paivaauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=R.+M.+Alvarezauthor=N.+Casaresauthor=V.+Seguraauthor=J.+I.+Mart%C3%ADn-Suberoauthor=F.-X.+Ogiauthor=P.+Souleauthor=C.+M.+Santiveriauthor=R.+Campos-Olivasauthor=G.+Castellanoauthor=M.+Garcia+Fernandez+de+Barrenaauthor=J.+R.+Rodriguez-Madozauthor=M.+J.+Garc%C3%ADa-Barchinoauthor=J.+J.+Lasarteauthor=M.+A.+Avilaauthor=J.+A.+Martinez-Climentauthor=J.+Oyarzabalauthor=F.+Prosper&title=Discovery+of+First-in-Class+Reversible+Dual+Small+Molecule+Inhibitors+against+G9a+and+DNMTs+with+in+Vivo+Activity+in+Hematological+Malignancies&doi=10.1038%2Fncomms15424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of first-in-class reversible dual small molecule inhibitors against G9a and DNMTs in hematological malignancies</span></div><div class="casAuthors">San Jose-Eneriz, Edurne; Agirre, Xabier; Rabal, Obdulia; Vilas-Zornoza, Amaia; Sanchez-Arias, Juan A.; Miranda, Estibaliz; Ugarte, Ana; Roa, Sergio; Paiva, Bruno; Estella-Hermoso de Mendoza, Ander; Alvarez, Rosa Maria; Casares, Noelia; Segura, Victor; Martin-Subero, Jose I.; Ogi, Francois-Xavier; Soule, Pierre; Santiveri, Clara M.; Campos-Olivas, Ramon; Castellano, Giancarlo; Fernandez de Barrena, Maite Garcia; Rodriguez-Madoz, Juan Roberto; Garcia-Barchino, Maria Jose; Lasarte, Juan Jose; Avila, Matias A.; Martinez-Climent, Jose Angel; Oyarzabal, Julen; Prosper, Felipe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15424</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The indisputable role of epigenetics in cancer and the fact that epigenetic alterations can be reversed have favored development of epigenetic drugs.  In this study, we design and synthesize potent novel, selective and reversible chem. probes that simultaneously inhibit the G9a and DNMTs methyltransferase activity.  In vitro treatment of haematol. neoplasia (acute myeloid leukemia-AML, acute lymphoblastic leukemia-ALL and diffuse large B-cell lymphoma-DLBCL) with the lead compd. CM-272, inhibits cell proliferation and promotes apoptosis, inducing interferon-stimulated genes and immunogenic cell death.  CM-272 significantly prolongs survival of AML, ALL and DLBCL xenogeneic models.  Our results represent the discovery of first-in-class dual inhibitors of G9a/DNMTs and establish this chem. series as a promising therapeutic tool for unmet needs in haematol. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstz1QqrJd4rVg90H21EOLACvtfcHk0lh3JS2c8irWAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D&md5=6d37a6433cdb371bfb535b1fd8a7a803</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fncomms15424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15424%26sid%3Dliteratum%253Aachs%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DPaiva%26aufirst%3DB.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DAlvarez%26aufirst%3DR.%2BM.%26aulast%3DCasares%26aufirst%3DN.%26aulast%3DSegura%26aufirst%3DV.%26aulast%3DMart%25C3%25ADn-Subero%26aufirst%3DJ.%2BI.%26aulast%3DOgi%26aufirst%3DF.-X.%26aulast%3DSoule%26aufirst%3DP.%26aulast%3DSantiveri%26aufirst%3DC.%2BM.%26aulast%3DCampos-Olivas%26aufirst%3DR.%26aulast%3DCastellano%26aufirst%3DG.%26aulast%3DGarcia%2BFernandez%2Bde%2BBarrena%26aufirst%3DM.%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DGarc%25C3%25ADa-Barchino%26aufirst%3DM.%2BJ.%26aulast%3DLasarte%26aufirst%3DJ.%2BJ.%26aulast%3DAvila%26aufirst%3DM.%2BA.%26aulast%3DMartinez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DProsper%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520First-in-Class%2520Reversible%2520Dual%2520Small%2520Molecule%2520Inhibitors%2520against%2520G9a%2520and%2520DNMTs%2520with%2520in%2520Vivo%2520Activity%2520in%2520Hematological%2520Malignancies%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15424%26doi%3D10.1038%2Fncomms15424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segovia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munera-Maravilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Fernández, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lodewijk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segrelles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valcárcel, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casares, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Cabrera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Calderón, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casado, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dueñas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villacampa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero-Ramos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Velasco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prósper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paramio, J. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of a G9a/DNMT Network Triggers Immune-Mediated Bladder Cancer Regression</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1073</span>– <span class="NLM_lpage">1081</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0499-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fs41591-019-0499-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=31270502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlamtb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1073-1081&author=C.+Segoviaauthor=E.+San+Jos%C3%A9-En%C3%A9rizauthor=E.+Munera-Maravillaauthor=M.+Mart%C3%ADnez-Fern%C3%A1ndezauthor=L.+Garateauthor=E.+Mirandaauthor=A.+Vilas-Zornozaauthor=I.+Lodewijkauthor=C.+Rubioauthor=C.+Segrellesauthor=L.+V.+Valc%C3%A1rcelauthor=O.+Rabalauthor=N.+Casaresauthor=A.+Bernardiniauthor=C.+Suarez-Cabreraauthor=F.+F.+L%C3%B3pez-Calder%C3%B3nauthor=P.+Fortesauthor=J.+A.+Casadoauthor=M.+Due%C3%B1asauthor=F.+Villacampaauthor=J.+J.+Lasarteauthor=F.+Guerrero-Ramosauthor=G.+de+Velascoauthor=J.+Oyarzabalauthor=D.+Castellanoauthor=X.+Agirreauthor=F.+Pr%C3%B3sperauthor=J.+M.+Paramio&title=Inhibition+of+a+G9a%2FDNMT+Network+Triggers+Immune-Mediated+Bladder+Cancer+Regression&doi=10.1038%2Fs41591-019-0499-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of a G9a/DNMT network triggers immune-mediated bladder cancer regression</span></div><div class="casAuthors">Segovia, Cristina; San Jose-Eneriz, Edurne; Munera-Maravilla, Ester; Martinez-Fernandez, Monica; Garate, Leire; Miranda, Estibaliz; Vilas-Zornoza, Amaia; Lodewijk, Iris; Rubio, Carolina; Segrelles, Carmen; Valcarcel, Luis Vitores; Rabal, Obdulia; Casares, Noelia; Bernardini, Alejandra; Suarez-Cabrera, Cristian; Lopez-Calderon, Fernando F.; Fortes, Puri; Casado, Jose A.; Duenas, Marta; Villacampa, Felipe; Lasarte, Juan Jose; Guerrero-Ramos, Felix; de Velasco, Guillermo; Oyarzabal, Julen; Castellano, Daniel; Agirre, Xabier; Prosper, Felipe; Paramio, Jesus M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1073-1081</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bladder cancer is lethal in its advanced, muscle-invasive phase with very limited therapeutic advances1,2.  Recent mol. characterization has defined new (epi)genetic drivers and potential targets for bladder cancer3,4.  The immune checkpoint inhibitors have shown remarkable efficacy but only in a limited fraction of bladder cancer patients5-8.  Here, it show that high G9a (EHMT2) expression is assocd. with poor clin. outcome in bladder cancer and that targeting G9a/DNMT methyltransferase activity with a novel inhibitor (CM-272) induces apoptosis and immunogenic cell death.  Using an immunocompetent quadruple-knockout (PtenloxP/loxP; Trp53loxP/loxP; Rb1loxP/loxP; Rbl1-/-) transgenic mouse model of aggressive metastatic, muscle-invasive bladder cancer, we demonstrate that CM-272 + cisplatin treatment results in statistically significant regression of established tumors and metastases.  The antitumor effect is significantly improved when CM-272 is combined with anti-programmed cell death ligand 1, even in the absence of cisplatin.  These effects are assocd. with an endogenous antitumor immune response and immunogenic cell death with the conversion of a cold immune tumor into a hot tumor.  Finally, increased G9a expression was assocd. with resistance to programmed cell death protein 1 inhibition in a cohort of patients with bladder cancer.  In summary, these findings support new and promising opportunities for the treatment of bladder cancer using a combination of epigenetic inhibitors and immune checkpoint blockade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlqZc1td7xQbVg90H21EOLACvtfcHk0liyW7TmV5irOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlamtb3O&md5=bc41cbe9b10d641aa5330fbacf5ed3cb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0499-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0499-y%26sid%3Dliteratum%253Aachs%26aulast%3DSegovia%26aufirst%3DC.%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DMunera-Maravilla%26aufirst%3DE.%26aulast%3DMart%25C3%25ADnez-Fern%25C3%25A1ndez%26aufirst%3DM.%26aulast%3DGarate%26aufirst%3DL.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DLodewijk%26aufirst%3DI.%26aulast%3DRubio%26aufirst%3DC.%26aulast%3DSegrelles%26aufirst%3DC.%26aulast%3DValc%25C3%25A1rcel%26aufirst%3DL.%2BV.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DCasares%26aufirst%3DN.%26aulast%3DBernardini%26aufirst%3DA.%26aulast%3DSuarez-Cabrera%26aufirst%3DC.%26aulast%3DL%25C3%25B3pez-Calder%25C3%25B3n%26aufirst%3DF.%2BF.%26aulast%3DFortes%26aufirst%3DP.%26aulast%3DCasado%26aufirst%3DJ.%2BA.%26aulast%3DDue%25C3%25B1as%26aufirst%3DM.%26aulast%3DVillacampa%26aufirst%3DF.%26aulast%3DLasarte%26aufirst%3DJ.%2BJ.%26aulast%3DGuerrero-Ramos%26aufirst%3DF.%26aulast%3Dde%2BVelasco%26aufirst%3DG.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DCastellano%26aufirst%3DD.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DPr%25C3%25B3sper%26aufirst%3DF.%26aulast%3DParamio%26aufirst%3DJ.%2BM.%26atitle%3DInhibition%2520of%2520a%2520G9a%252FDNMT%2520Network%2520Triggers%2520Immune-Mediated%2520Bladder%2520Cancer%2520Regression%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1073%26epage%3D1081%26doi%3D10.1038%2Fs41591-019-0499-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondengaden, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigalapalli, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondengadan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. G.</span></span> <span> </span><span class="NLM_article-title">Structure Based Design, Synthesis and Activity Studies of Small Hybrid Molecules as HDAC and G9a Dual Inhibitors</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">63187</span>– <span class="NLM_lpage">63207</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.18730</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.18632%2Foncotarget.18730" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28968981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FksVSnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=63187-63207&author=L.+Zangauthor=S.+M.+Kondengadenauthor=Q.+Zhangauthor=X.+Liauthor=D.+K.+Sigalapalliauthor=S.+M.+Kondengadanauthor=K.+Huangauthor=K.+K.+Liauthor=S.+Liauthor=Z.+Xiaoauthor=L.+Wenauthor=H.+Zhuauthor=B.+N.+Babuauthor=L.+Wangauthor=F.+Cheauthor=P.+G.+Wang&title=Structure+Based+Design%2C+Synthesis+and+Activity+Studies+of+Small+Hybrid+Molecules+as+HDAC+and+G9a+Dual+Inhibitors&doi=10.18632%2Foncotarget.18730"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure based design, synthesis and activity studies of small hybrid molecules as HDAC and G9a dual inhibitors</span></div><div class="casAuthors">Zang Lanlan; Wang Lijuan; Che Fengyuan; Kondengaden Shukkoor M; Zhang Qing; Huang Kenneth; Li Keqin Kathy; Li Shanshan; Xiao Zhongying; Wen Liuqing; Zhu Hailiang; Wang Peng George; Li Xiaobo; Sigalapalli Dilep K; Babu Bathini N; Kondengadan Shameer M</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">63187-63207</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrant enzymatic activities or expression profiles of epigenetic regulations are therapeutic targets for cancers.  Among these, histone 3 lysine 9 methylation (H3K9Me2) and global de-acetylation on histone proteins are associated with multiple cancer phenotypes including leukemia, prostatic carcinoma, hepatocellular carcinoma and pulmonary carcinoma.  Here, we report the discovery of the first small molecule capable of acting as a dual inhibitor targeting both G9a and HDAC.  Our structure based design, synthesis, and screening for the dual activity of the small molecules led to the discovery of compound 14 which displays promising inhibition of both G9a and HDAC in low micro-molar range in cell based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLSKkSsKMGSJcXNb62YGmbfW6udTcc2eY2OQQNNzhGibntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FksVSnsA%253D%253D&md5=ccf8886c55eff785684b72d6bab4db44</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.18730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.18730%26sid%3Dliteratum%253Aachs%26aulast%3DZang%26aufirst%3DL.%26aulast%3DKondengaden%26aufirst%3DS.%2BM.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSigalapalli%26aufirst%3DD.%2BK.%26aulast%3DKondengadan%26aufirst%3DS.%2BM.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DBabu%26aufirst%3DB.%2BN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChe%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DP.%2BG.%26atitle%3DStructure%2520Based%2520Design%252C%2520Synthesis%2520and%2520Activity%2520Studies%2520of%2520Small%2520Hybrid%2520Molecules%2520as%2520HDAC%2520and%2520G9a%2520Dual%2520Inhibitors%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D63187%26epage%3D63207%26doi%3D10.18632%2Foncotarget.18730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">August, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teodoro, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechtler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span> <span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a Small-Molecule Inhibitor for the G9a Histone Methyltransferase</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.01.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+Small-Molecule+Inhibitor+for+the+G9a+Histone+Methyltransferase&doi=10.1016%2Fj.molcel.2007.01.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0ljzdJCHSUIPUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520Small-Molecule%2520Inhibitor%2520for%2520the%2520G9a%2520Histone%2520Methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481%26doi%3D10.1016%2Fj.molcel.2007.01.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival-Gervier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimaggio, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe Selectively Inhibits G9a and GLP Methyltransferase Activity in Cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+Chemical+Probe+Selectively+Inhibits+G9a+and+GLP+Methyltransferase+Activity+in+Cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0ljzdJCHSUIPUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520Chemical%2520Probe%2520Selectively%2520Inhibits%2520G9a%2520and%2520GLP%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Valckenborgh, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Riet, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderkerken, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bruyne, E.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Modulating Agents as a New Therapeutic Approach in Multiple Myeloma</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.3390/cancers5020430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.3390%2Fcancers5020430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=24216985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWisLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=430-461&author=K.+Maesauthor=E.+Menuauthor=E.+Van+Valckenborghauthor=I.+Van+Rietauthor=K.+Vanderkerkenauthor=E.+De+Bruyne&title=Epigenetic+Modulating+Agents+as+a+New+Therapeutic+Approach+in+Multiple+Myeloma&doi=10.3390%2Fcancers5020430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modulating agents as a new therapeutic approach in multiple myeloma</span></div><div class="casAuthors">Maes, Ken; Menu, Eline; Van Valckenborgh, Els; Van Riet, Ivan; Vanderkerken, Karin; De Bruyne, Elke</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">430-461</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Multiple myeloma (MM) is an incurable B-cell malignancy.  Therefore, new targets and drugs are urgently needed to improve patient outcome.  Epigenetic aberrations play a crucial role in development and progression in cancer, including MM.  To target these aberrations, epigenetic modulating agents, such as DNA methyltransferase inhibitors (DNMTi) and histone deacetylase inhibitors (HDACi), are under intense investigation in solid and hematol. cancers.  A clin. benefit of the use of these agents as single agents and in combination regimens has been suggested based on numerous studies in pre-clin. tumor models, including MM models.  The mechanisms of action are not yet fully understood but appear to involve a combination of true epigenetic changes and cytotoxic actions.  In addn., the interactions with the BM niche are also affected by epigenetic modulating agents that will further det. the in vivo efficacy and thus patient outcome.  A better understanding of the mol. events underlying the anti-tumor activity of the epigenetic drugs will lead to more rational drug combinations.  This review focuses on the involvement of epigenetic changes in MM pathogenesis and how the use of DNMTi and HDACi affect the myeloma tumor itself and its interactions with the microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK1Xdo9RpO2LVg90H21EOLACvtfcHk0lj0the-zMclJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWisLzM&md5=b861e9140f60af2be43de6ac9dfff3bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fcancers5020430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers5020430%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DK.%26aulast%3DMenu%26aufirst%3DE.%26aulast%3DVan%2BValckenborgh%26aufirst%3DE.%26aulast%3DVan%2BRiet%26aufirst%3DI.%26aulast%3DVanderkerken%26aufirst%3DK.%26aulast%3DDe%2BBruyne%26aufirst%3DE.%26atitle%3DEpigenetic%2520Modulating%2520Agents%2520as%2520a%2520New%2520Therapeutic%2520Approach%2520in%2520Multiple%2520Myeloma%26jtitle%3DCancers%26date%3D2013%26volume%3D5%26spage%3D430%26epage%3D461%26doi%3D10.3390%2Fcancers5020430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">San-Miguel, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einsele, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, P.</span></span> <span> </span><span class="NLM_article-title">The Role of Panobinostat Plus Bortezomib and Dexamethasone in Treating Relapsed or Relapsed and Refractory Multiple Myeloma: A European Perspective</span>. <i>Adv. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1896</span>– <span class="NLM_lpage">1920</span>, <span class="refDoi"> DOI: 10.1007/s12325-016-0413-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1007%2Fs12325-016-0413-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27677481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Srsb3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=1896-1920&author=J.+F.+San-Miguelauthor=H.+Einseleauthor=P.+Moreau&title=The+Role+of+Panobinostat+Plus+Bortezomib+and+Dexamethasone+in+Treating+Relapsed+or+Relapsed+and+Refractory+Multiple+Myeloma%3A+A+European+Perspective&doi=10.1007%2Fs12325-016-0413-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Panobinostat Plus Bortezomib and Dexamethasone in Treating Relapsed or Relapsed and Refractory Multiple Myeloma: A European Perspective</span></div><div class="casAuthors">San-Miguel, Jesus F.; Einsele, Hermann; Moreau, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1896-1920</span>CODEN:
                <span class="NLM_cas:coden">ADTHE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-238X</span>.
    
            (<span class="NLM_cas:orgname">Springer Healthcare Ltd.</span>)
        </div><div class="casAbstract">Panobinostat is an oral pan-histone deacetylase inhibitor developed by Novartis.  Panobinostat acts via epigenetic modification and inhibition of the aggresome pathway.  In August 2015, the European Commission authorized panobinostat for use in combination with bortezomib and dexamethasone for the treatment of relapsed or relapsed and refractory multiple myeloma (MM) in patients who have received ≥2 prior regimens including bortezomib and an immunomodulatory drug.  In Jan. 2016, the National Institute for Health and Care Excellence recommended panobinostat for use in the same combination and patient population.  The authorization and recommendation were based on results from the pivotal phase 3 PANORAMA 1 (NCT01023308) clin. trial, which demonstrated an improvement in median progression-free survival of 7.8 mo for the three-drug combination compared with placebo plus bortezomib and dexamethasone in this patient population.  This review will discuss the current treatment landscape for relapsed/refractory MM, the mechanism of action of panobinostat, clin. data supporting the European authorization, concerns about safety and strategies for mitigating toxicity, and how panobinostat fits into the current MM landscape in Europe.  Funding: Editorial support, funded by Novartis Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4VbHFN0ga1LVg90H21EOLACvtfcHk0lj0the-zMclJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Srsb3M&md5=7ebba36020f925d5ca137fa8cd7bd57b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2Fs12325-016-0413-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12325-016-0413-7%26sid%3Dliteratum%253Aachs%26aulast%3DSan-Miguel%26aufirst%3DJ.%2BF.%26aulast%3DEinsele%26aufirst%3DH.%26aulast%3DMoreau%26aufirst%3DP.%26atitle%3DThe%2520Role%2520of%2520Panobinostat%2520Plus%2520Bortezomib%2520and%2520Dexamethasone%2520in%2520Treating%2520Relapsed%2520or%2520Relapsed%2520and%2520Refractory%2520Multiple%2520Myeloma%253A%2520A%2520European%2520Perspective%26jtitle%3DAdv.%2520Ther.%26date%3D2016%26volume%3D33%26spage%3D1896%26epage%3D1920%26doi%3D10.1007%2Fs12325-016-0413-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteve, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raineri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agueda, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russiñol, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queirós, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beekman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">José-Enériz, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Verdugo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guruceaga, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calasanz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flicek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miguel, J. F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stunnenberg, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Subero, J. I.</span></span> <span> </span><span class="NLM_article-title">Whole-epigenome analysis in multiple myeloma reveals DNA hypermethylation of B cell-specific enhancers</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">478</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1101/gr.180240.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1101%2Fgr.180240.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=25644835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFCnsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=478-487&author=X.+Agirreauthor=G.+Castellanoauthor=M.+Pascualauthor=S.+Heathauthor=M.+Kulisauthor=V.+Seguraauthor=A.+Bergmannauthor=A.+Esteveauthor=A.+Merkelauthor=E.+Raineriauthor=L.+Aguedaauthor=J.+Blancauthor=D.+Richardsonauthor=L.+Clarkeauthor=A.+Dattaauthor=N.+Russi%C3%B1olauthor=A.+C.+Queir%C3%B3sauthor=R.+Beekmanauthor=J.+R.+Rodr%C3%ADguez-Madozauthor=E.+S.+Jos%C3%A9-En%C3%A9rizauthor=F.+Fangauthor=N.+C.+Guti%C3%A9rrezauthor=J.+M.+Garc%C3%ADa-Verdugoauthor=M.+I.+Robsonauthor=E.+C.+Schirmerauthor=E.+Guruceagaauthor=J.+H.+A.+Martensauthor=M.+Gutauthor=M.+J.+Calasanzauthor=P.+Flicekauthor=R.+Siebertauthor=E.+Campoauthor=J.+F.+S.+Miguelauthor=A.+Melnickauthor=H.+G.+Stunnenbergauthor=I.+G.+Gutauthor=F.+Prosperauthor=J.+I.+Mart%C3%ADn-Subero&title=Whole-epigenome+analysis+in+multiple+myeloma+reveals+DNA+hypermethylation+of+B+cell-specific+enhancers&doi=10.1101%2Fgr.180240.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Whole-epigenome analysis in multiple myeloma reveals DNA hypermethylation of B cell-specific enhancers</span></div><div class="casAuthors">Agirre, Xabier; Castellano, Giancarlo; Pascual, Marien; Heath, Simon; Kulis, Marta; Segura, Victor; Bergmann, Anke; Esteve, Anna; Merkel, Angelika; Raineri, Emanuele; Agueda, Lidia; Blanc, Julie; Richardson, David; Clarke, Laura; Datta, Avik; Russinol, Nuria; Queiros, Ana C.; Beekman, Renee; Rodriguez-Madoz, Juan R.; Jose-Eneriz, Edurne San; Fang, Fang; Gutierrez, Norma C.; Garcia-Verdugo, Jose M.; Robson, Michael I.; Schirmer, Eric C.; Guruceaga, Elisabeth; Martens, Joost H. A.; Gut, Marta; Calasanz, Maria J.; Flicek, Paul; Siebert, Reiner; Campo, Elias; San Miguel, Jesus F.; Melnick, Ari; Stunnenberg, Hendrik G.; Gut, Ivo G.; Prosper, Felipe; Martin-Subero, Jose I.</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">478-487</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">While analyzing the DNA methylome of multiple myeloma (MM), a plasma cell neoplasm, by whole-genome bisulfite sequencing and high-d. arrays, we obsd. a highly heterogeneous pattern globally characterized by regional DNA hypermethylation embedded in extensive hypomethylation.  In contrast to the widely reported DNA hypermethylation of promoter-assocd. CpG islands (CGIs) in cancer, hypermethylated sites in MM, as opposed to normal plasma cells, were located outside CpG islands and were unexpectedly assocd. with intronic enhancer regions defined in normal B cells and plasma cells.  Both RNA-seq and in vitro reporter assays indicated that enhancer hypermethylation is globally assocd. with down-regulation of its host genes.  ChIP-seq and DNase-seq further revealed that DNA hypermethylation in these regions is related to enhancer decommissioning.  Hypermethylated enhancer regions overlapped with binding sites of B cell-specific transcription factors (TFs) and the degree of enhancer methylation inversely correlated with expression levels of these TFs in MM.  Furthermore, hypermethylated regions in MM were methylated in stem cells and gradually became demethylated during normal B-cell differentiation, suggesting that MM cells either reacquire epigenetic features of undifferentiated cells or maintain an epigenetic signature of a putative myeloma stem cell progenitor.  Overall, we have identified DNA hypermethylation of developmentally regulated enhancers as a new type of epigenetic modification assocd. with the pathogenesis of MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopvhGrSOc8CLVg90H21EOLACvtfcHk0ljuVRj8p4qgqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFCnsrs%253D&md5=84b572c460997690cf42a4a7359dec22</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1101%2Fgr.180240.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.180240.114%26sid%3Dliteratum%253Aachs%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DCastellano%26aufirst%3DG.%26aulast%3DPascual%26aufirst%3DM.%26aulast%3DHeath%26aufirst%3DS.%26aulast%3DKulis%26aufirst%3DM.%26aulast%3DSegura%26aufirst%3DV.%26aulast%3DBergmann%26aufirst%3DA.%26aulast%3DEsteve%26aufirst%3DA.%26aulast%3DMerkel%26aufirst%3DA.%26aulast%3DRaineri%26aufirst%3DE.%26aulast%3DAgueda%26aufirst%3DL.%26aulast%3DBlanc%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DD.%26aulast%3DClarke%26aufirst%3DL.%26aulast%3DDatta%26aufirst%3DA.%26aulast%3DRussi%25C3%25B1ol%26aufirst%3DN.%26aulast%3DQueir%25C3%25B3s%26aufirst%3DA.%2BC.%26aulast%3DBeekman%26aufirst%3DR.%26aulast%3DRodr%25C3%25ADguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%2BS.%26aulast%3DFang%26aufirst%3DF.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DN.%2BC.%26aulast%3DGarc%25C3%25ADa-Verdugo%26aufirst%3DJ.%2BM.%26aulast%3DRobson%26aufirst%3DM.%2BI.%26aulast%3DSchirmer%26aufirst%3DE.%2BC.%26aulast%3DGuruceaga%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DJ.%2BH.%2BA.%26aulast%3DGut%26aufirst%3DM.%26aulast%3DCalasanz%26aufirst%3DM.%2BJ.%26aulast%3DFlicek%26aufirst%3DP.%26aulast%3DSiebert%26aufirst%3DR.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DMiguel%26aufirst%3DJ.%2BF.%2BS.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DStunnenberg%26aufirst%3DH.%2BG.%26aulast%3DGut%26aufirst%3DI.%2BG.%26aulast%3DProsper%26aufirst%3DF.%26aulast%3DMart%25C3%25ADn-Subero%26aufirst%3DJ.%2BI.%26atitle%3DWhole-epigenome%2520analysis%2520in%2520multiple%2520myeloma%2520reveals%2520DNA%2520hypermethylation%2520of%2520B%2520cell-specific%2520enhancers%26jtitle%3DGenome%2520Res.%26date%3D2015%26volume%3D25%26spage%3D478%26epage%3D487%26doi%3D10.1101%2Fgr.180240.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ordoñez, R.</span>; <span class="NLM_string-name">Kulis, M.</span>; <span class="NLM_string-name">Russiñol, N.</span>; <span class="NLM_string-name">Chapaprieta, V.</span>; <span class="NLM_string-name">Beekman, R.</span>; <span class="NLM_string-name">Meydan, C.</span>; <span class="NLM_string-name">Duran-Ferrer, M.</span>; <span class="NLM_string-name">Verdaguer-Dot, N.</span>; <span class="NLM_string-name">Clot, G.</span>; <span class="NLM_string-name">Vilarrasa-Blasi, R.</span>; <span class="NLM_string-name">Garate, L.</span>; <span class="NLM_string-name">Miranda, E.</span>; <span class="NLM_string-name">Carrasco, A.</span>; <span class="NLM_string-name">Ezponda, T.</span>; <span class="NLM_string-name">Vilas-Zornoza, A.</span>; <span class="NLM_string-name">Lara-Astiaso, D.</span>; <span class="NLM_string-name">Dupéré-Richer, D.</span>; <span class="NLM_string-name">Martens, J. H. A.</span>; <span class="NLM_string-name">Torrents, D.</span>; <span class="NLM_string-name">El-Omri, H.</span>; <span class="NLM_string-name">Taha, R. Y.</span>; <span class="NLM_string-name">Calasanz, M. J.</span>; <span class="NLM_string-name">Paiva, B.</span>; <span class="NLM_string-name">Miguel, J. S.</span>; <span class="NLM_string-name">Flicek, P.</span>; <span class="NLM_string-name">Gut, I.</span>; <span class="NLM_string-name">Melnick, A.</span>; <span class="NLM_string-name">Mitsiades, C. S.</span>; <span class="NLM_string-name">Licht, J. D.</span>; <span class="NLM_string-name">Campo, E.</span>; <span class="NLM_string-name">Stunnenberg, H. G.</span>; <span class="NLM_string-name">Agirre, X.</span>; <span class="NLM_string-name">Prosper, F.</span>; <span class="NLM_string-name">Martin-Subero, J. I.</span></span> <span> </span><span class="NLM_article-title">Chromatin activation as a unifying principle underlying pathogenic mechanisms in multiple myeloma</span>. <span class="NLM_year" style="font-weight: bold;">2019</span>, bioRxiv:740027.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=R.+Ordo%C3%B1ez&author=M.+Kulis&author=N.+Russi%C3%B1ol&author=V.+Chapaprieta&author=R.+Beekman&author=C.+Meydan&author=M.+Duran-Ferrer&author=N.+Verdaguer-Dot&author=G.+Clot&author=R.+Vilarrasa-Blasi&author=L.+Garate&author=E.+Miranda&author=A.+Carrasco&author=T.+Ezponda&author=A.+Vilas-Zornoza&author=D.+Lara-Astiaso&author=D.+Dup%C3%A9r%C3%A9-Richer&author=J.+H.+A.+Martens&author=D.+Torrents&author=H.+El-Omri&author=R.+Y.+Taha&author=M.+J.+Calasanz&author=B.+Paiva&author=J.+S.+Miguel&author=P.+Flicek&author=I.+Gut&author=A.+Melnick&author=C.+S.+Mitsiades&author=J.+D.+Licht&author=E.+Campo&author=H.+G.+Stunnenberg&author=X.+Agirre&author=F.+Prosper&author=J.+I.+Martin-Subero&title=Chromatin+activation+as+a+unifying+principle+underlying+pathogenic+mechanisms+in+multiple+myeloma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOrdo%25C3%25B1ez%26aufirst%3DR.%26atitle%3DChromatin%2520activation%2520as%2520a%2520unifying%2520principle%2520underlying%2520pathogenic%2520mechanisms%2520in%2520multiple%2520myeloma%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricobaraza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, R.</span></span> <span> </span><span class="NLM_article-title">Defining the Mechanism of Action of 4-Phenylbutyrate to Develop a Small-Molecule-Based Therapy for Alzheimer’s Disease</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5545</span>– <span class="NLM_lpage">5553</span>, <span class="refDoi"> DOI: 10.2174/092986711798347315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.2174%2F092986711798347315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=22172064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12ntbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=5545-5553&author=M.+Cuadrado-Tejedorauthor=A.+Garc%C3%ADa-Ostaauthor=A.+Ricobarazaauthor=J.+Oyarzabalauthor=R.+Franco&title=Defining+the+Mechanism+of+Action+of+4-Phenylbutyrate+to+Develop+a+Small-Molecule-Based+Therapy+for+Alzheimer%E2%80%99s+Disease&doi=10.2174%2F092986711798347315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the mechanism of action of 4-phenylbutyrate to develop a small-molecule-based therapy for Alzheimer's disease</span></div><div class="casAuthors">Cuadrado-Tejedor, M.; Garcia-Osta, A.; Ricobaraza, A.; Oyarzabal, J.; Franco, R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">5545-5553</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  4-Phenylbutyrate (PBA) is a small mol. that restores cognitive deficits in animal models of Alzheimer's disease (AD).  Although the mol. basis of the cognitive benefits of PBA remains unknown, a multi-modal/multi-target mechanism has been proposed.  Putative targets of this drug are different from those of drugs that are now used in clin. trials.  As PBA is already administered to patients with congenital defects affecting enzymes in the urea cycle, it can be rapidly tested in AD clin. trials.  However, the main drawback to its therapeutic use is the high dosage required (up to 15 g/day).  Thus, deciphering the precise mechanism(s) of action of this drug may enable novel drugs with similar therapeutic effects to PBA to be developed that can be used at more manageable doses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosAevZlfyhp7Vg90H21EOLACvtfcHk0lhOW8MuR6ZNDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12ntbbL&md5=0703163333de9afce25c352e4ef98df5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.2174%2F092986711798347315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711798347315%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Osta%26aufirst%3DA.%26aulast%3DRicobaraza%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DFranco%26aufirst%3DR.%26atitle%3DDefining%2520the%2520Mechanism%2520of%2520Action%2520of%25204-Phenylbutyrate%2520to%2520Develop%2520a%2520Small-Molecule-Based%2520Therapy%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D5545%26epage%3D5553%26doi%3D10.2174%2F092986711798347315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span> <span> </span><span class="NLM_article-title">Development of the Pan-DAC Inhibitor Panobinostat (LBH589): Successes and Challenges</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2009.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.canlet.2009.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=19344997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2009&pages=233-241&author=P.+Atadja&title=Development+of+the+Pan-DAC+Inhibitor+Panobinostat+%28LBH589%29%3A+Successes+and+Challenges&doi=10.1016%2Fj.canlet.2009.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the pan-DAC inhibitor panobinostat (LBH589): Successes and challenges</span></div><div class="casAuthors">Atadja, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-241</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  The histone deacetylase (HDAC) inhibitors are emerging as a highly useful class of anticancer agents that inhibit the enzyme HDAC involved in the deacetylation of histone and non-histone cellular proteins.  The HDAC inhibitor, panobinostat (LBH589, Novartis Pharmaceuticals), achieves potent inhibition of all HDAC enzymes implicated in cancer and has demonstrated potent anti-tumor activity in preclin. models and promising clin. efficacy in cancer patients.  In this review we discuss the successes and challenges surrounding the development of panobinostat, focusing on its proposed mechanism of action, preclin. anti-tumor activity, and early clin. efficacy in hematol. and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSTYxNKUMt7bVg90H21EOLACvtfcHk0lhOW8MuR6ZNDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D&md5=4c487a2f8dc13f72cecf105d9fae97d9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2009.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2009.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DAtadja%26aufirst%3DP.%26atitle%3DDevelopment%2520of%2520the%2520Pan-DAC%2520Inhibitor%2520Panobinostat%2520%2528LBH589%2529%253A%2520Successes%2520and%2520Challenges%26jtitle%3DCancer%2520Lett.%26date%3D2009%26volume%3D280%26spage%3D233%26epage%3D241%26doi%3D10.1016%2Fj.canlet.2009.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariën, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floren, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beliën, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilatte, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decrane, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilissen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vreys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bol, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talloen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goris, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andries, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du Jardin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janicot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Emelen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angibaud, P.</span></span> <span> </span><span class="NLM_article-title">JNJ-26481585, a Novel “second-Generation” oral Histone Deacetylase Inhibitor, Shows Broad-Spectrum Preclinical Antitumoral Activity</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6841</span>– <span class="NLM_lpage">6851</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-09-0547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1158%2F1078-0432.CCR-09-0547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=19861438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtl2ktLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=6841-6851&author=J.+Artsauthor=P.+Kingauthor=A.+Mari%C3%ABnauthor=W.+Florenauthor=A.+Beli%C3%ABnauthor=L.+Janssenauthor=I.+Pilatteauthor=B.+Rouxauthor=L.+Decraneauthor=R.+Gilissenauthor=I.+Hicksonauthor=V.+Vreysauthor=E.+Coxauthor=K.+Bolauthor=W.+Talloenauthor=I.+Gorisauthor=L.+Andriesauthor=M.+Du+Jardinauthor=M.+Janicotauthor=M.+Pageauthor=K.+van+Emelenauthor=P.+Angibaud&title=JNJ-26481585%2C+a+Novel+%E2%80%9Csecond-Generation%E2%80%9D+oral+Histone+Deacetylase+Inhibitor%2C+Shows+Broad-Spectrum+Preclinical+Antitumoral+Activity&doi=10.1158%2F1078-0432.ccr-09-0547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">JNJ-26481585, a Novel "Second-Generation" Oral Histone Deacetylase Inhibitor, Shows Broad-Spectrum Preclinical Antitumoral Activity</span></div><div class="casAuthors">Arts, Janine; King, Peter; Marien, Ann; Floren, Wim; Belien, Ann; Janssen, Lut; Pilatte, Isabelle; Roux, Bruno; Decrane, Laurence; Gilissen, Ron; Hickson, Ian; Vreys, Veronique; Cox, Eugene; Bol, Kees; Talloen, Willem; Goris, Ilse; Andries, Luc; Du Jardin, Marc; Janicot, Michel; Page, Martin; van Emelen, Kristof; Angibaud, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6841-6851</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Histone deacetylase (HDAC) inhibitors have shown promising clin. activity in the treatment of hematol. malignancies, but their activity in solid tumor indications has been limited.  Most HDAC inhibitors in clin. development only transiently induce histone acetylation in tumor tissue.  Here, we sought to identify a "second-generation" class I HDAC inhibitor with prolonged pharmacodynamic response in vivo, to assess whether this results in superior antitumoral efficacy.  Exptl. Design: To identify novel HDAC inhibitors with superior pharmacodynamic properties, we developed a preclin. in vivo tumor model, in which tumor cells have been engineered to express fluorescent protein dependent on HDAC1 inhibition, thereby allowing noninvasive real-time evaluation of the tumor response to HDAC inhibitors.  Results: In vivo pharmacodynamic anal. of 140 potent pyrimidyl-hydroxamic acid analogs resulted in the identification of JNJ-26481585.  Once daily oral administration of JNJ-26481585 induced continuous histone H3 acetylation.  The prolonged pharmacodynamic response translated into complete tumor growth inhibition in Ras mutant HCT116 colon carcinoma xenografts, whereas 5-fluorouracil was less active.  JNJ-26481585 also fully inhibited the growth of C170HM2 colorectal liver metastases, whereas again 5-fluorouracil/Leucovorin showed modest activity.  Further characterization revealed that JNJ-26481585 is a pan-HDAC inhibitor with marked potency toward HDAC1 (IC50, 0.16 nmol/L).  Conclusions: The potent antitumor activity as a single agent in preclin. models combined with its favorable pharmacodynamic profile makes JNJ-26481585 a promising "second-generation" HDAC inhibitor.  The compd. is currently in clin. studies, to evaluate its potential applicability in a broad spectrum of both solid and hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKc9hyMiADZrVg90H21EOLACvtfcHk0lhXXq16vvzB-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtl2ktLzE&md5=8f199c25c92c967a76f273d879d6c577</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-0547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-0547%26sid%3Dliteratum%253Aachs%26aulast%3DArts%26aufirst%3DJ.%26aulast%3DKing%26aufirst%3DP.%26aulast%3DMari%25C3%25ABn%26aufirst%3DA.%26aulast%3DFloren%26aufirst%3DW.%26aulast%3DBeli%25C3%25ABn%26aufirst%3DA.%26aulast%3DJanssen%26aufirst%3DL.%26aulast%3DPilatte%26aufirst%3DI.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DDecrane%26aufirst%3DL.%26aulast%3DGilissen%26aufirst%3DR.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DVreys%26aufirst%3DV.%26aulast%3DCox%26aufirst%3DE.%26aulast%3DBol%26aufirst%3DK.%26aulast%3DTalloen%26aufirst%3DW.%26aulast%3DGoris%26aufirst%3DI.%26aulast%3DAndries%26aufirst%3DL.%26aulast%3DDu%2BJardin%26aufirst%3DM.%26aulast%3DJanicot%26aufirst%3DM.%26aulast%3DPage%26aufirst%3DM.%26aulast%3Dvan%2BEmelen%26aufirst%3DK.%26aulast%3DAngibaud%26aufirst%3DP.%26atitle%3DJNJ-26481585%252C%2520a%2520Novel%2520%25E2%2580%259Csecond-Generation%25E2%2580%259D%2520oral%2520Histone%2520Deacetylase%2520Inhibitor%252C%2520Shows%2520Broad-Spectrum%2520Preclinical%2520Antitumoral%2520Activity%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D6841%26epage%3D6851%26doi%3D10.1158%2F1078-0432.ccr-09-0547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
De Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer’s Disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8967</span>– <span class="NLM_lpage">9004</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8967-9004&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0ADe+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+First-in-Class+Dual+Acting+Histone+Deacetylases+%28HDACs%29+and+Phosphodiesterase+5+%28PDE5%29+Inhibitors+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease&doi=10.1021%2Facs.jmedchem.6b00908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer's Disease</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Haizhong, Tan; Wei, Wu; Musheng, Xu; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8967-9004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Simultaneous inhibition of phosphodiesterase 5 (PDE5) and histone deacetylases (HDAC) has recently been validated as a potentially novel therapeutic approach for Alzheimer's Disease (AD).  To further extend this concept, the authors designed and synthesized the first chem. series of dual acting PDE5 and HDAC inhibitors, and the authors validated this systems therapeutics approach.  Following the implementation of structure- and knowledge-based approaches, initial hits were designed and were shown to validate the hypothesis of dual in vitro inhibition.  Then, an optimization strategy was pursued to obtain a proper tool compd. for in vivo testing in AD models.  Initial hits were translated into mols. with adequate cellular functional responses (histone acetylation and cAMP/cGMP response element-binding (CREB) phosphorylation in the nanomolar range), an acceptable therapeutic window (>1 log unit) and the ability to cross the blood-brain barrier, leading to the identification of 7 as a candidate for in vivo proof-of-concept testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbHBoJlefNEbVg90H21EOLACvtfcHk0lgeXxKorlC0kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O&md5=756a252809f05f5d7fc59e828736cfba</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00908%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2BDe%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520First-in-Class%2520Dual%2520Acting%2520Histone%2520Deacetylases%2520%2528HDACs%2529%2520and%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8967%26epage%3D9004%26doi%3D10.1021%2Facs.jmedchem.6b00908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamarina, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez Perez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordoñez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barchino, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Reversible DNA Methyltransferase and Lysine Methyltransferase G9a Inhibitors with Antitumoral in Vivo Efficacy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6518</span>– <span class="NLM_lpage">6545</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1altb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6518-6545&author=O.+Rabalauthor=E.+San+Jos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=J.+A.+S%C3%A1nchez-Ariasauthor=A.+Vilas-Zornozaauthor=A.+Ugarteauthor=I.+de+Miguelauthor=E.+Mirandaauthor=L.+Garateauthor=M.+Fragaauthor=P.+Santamarinaauthor=R.+Fernandez+Perezauthor=R.+Ordo%C3%B1ezauthor=E.+S%C3%A1ezauthor=S.+Roaauthor=M.+J.+Garc%C3%ADa-Barchinoauthor=J.+A.+Mart%C3%ADnez-Climentauthor=Y.+Liuauthor=W.+Wuauthor=M.+Xuauthor=F.+Prosperauthor=J.+Oyarzabal&title=Discovery+of+Reversible+DNA+Methyltransferase+and+Lysine+Methyltransferase+G9a+Inhibitors+with+Antitumoral+in+Vivo+Efficacy&doi=10.1021%2Facs.jmedchem.7b01926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Reversible DNA Methyltransferase and Lysine Methyltransferase G9a Inhibitors with Antitumoral in Vivo Efficacy</span></div><div class="casAuthors">Rabal, Obdulia; San Jose-Eneriz, Edurne; Agirre, Xabier; Sanchez-Arias, Juan Antonio; Vilas-Zornoza, Amaia; Ugarte, Ana; de Miguel, Irene; Miranda, Estibaliz; Garate, Leire; Fraga, Mario; Santamarina, Pablo; Fernandez Perez, Raul; Ordonez, Raquel; Saez, Elena; Roa, Sergio; Garcia-Barchino, Maria Jose; Martinez-Climent, Jose Angel; Liu, Yingying; Wu, Wei; Xu, Musheng; Prosper, Felipe; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6518-6545</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using knowledge- and structure-based approaches, we designed and synthesized reversible chem. probes that simultaneously inhibit the activity of two epigenetic targets, histone 3 lysine 9 methyltransferase (G9a) and DNA methyltransferases (DNMT), at nanomolar ranges.  Enzymic competition assays confirmed our design strategy: substrate competitive inhibitors.  Next, an initial exploration around our hit 11 was pursued to identify an adequate tool compd. for in vivo testing.  In vitro treatment of different hematol. neoplasia cell lines led to the identification of mols. with clear antiproliferative efficacies (GI50 values in the nanomolar range).  On the basis of epigenetic functional cellular responses (levels of lysine 9 methylation and 5-methylcytosine), an acceptable therapeutic window (around 1 log unit) and a suitable pharmacokinetic profile, 12 was selected for in vivo proof-of-concept.  Herein, 12 achieved a significant in vivo efficacy: 70% overall tumor growth inhibition of a human acute myeloid leukemia (AML) xenograft in a mouse model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf8aK3C-iah7Vg90H21EOLACvtfcHk0lgeXxKorlC0kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1altb%252FN&md5=41794ccc9d1b5662b796932b37b9efd3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01926%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DGarate%26aufirst%3DL.%26aulast%3DFraga%26aufirst%3DM.%26aulast%3DSantamarina%26aufirst%3DP.%26aulast%3DFernandez%2BPerez%26aufirst%3DR.%26aulast%3DOrdo%25C3%25B1ez%26aufirst%3DR.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DGarc%25C3%25ADa-Barchino%26aufirst%3DM.%2BJ.%26aulast%3DMart%25C3%25ADnez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DProsper%26aufirst%3DF.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Reversible%2520DNA%2520Methyltransferase%2520and%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%2520with%2520Antitumoral%2520in%2520Vivo%2520Efficacy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6518%26epage%3D6545%26doi%3D10.1021%2Facs.jmedchem.7b01926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6546</span>– <span class="NLM_lpage">6573</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSrtL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6546-6573&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=E.+San+Jos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=I.+de+Miguelauthor=L.+Garateauthor=E.+Mirandaauthor=E.+S%C3%A1ezauthor=S.+Roaauthor=J.+A.+Mart%C3%ADnez-Climentauthor=Y.+Liuauthor=W.+Wuauthor=M.+Xuauthor=F.+Prosperauthor=J.+Oyarzabal&title=Detailed+Exploration+around+4-Aminoquinolines+Chemical+Space+to+Navigate+the+Lysine+Methyltransferase+G9a+and+DNA+Methyltransferase+Biological+Spaces&doi=10.1021%2Facs.jmedchem.7b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan Antonio; San Jose-Eneriz, Edurne; Agirre, Xabier; de Miguel, Irene; Garate, Leire; Miranda, Estibaliz; Saez, Elena; Roa, Sergio; Martinez-Climent, Jose Angel; Liu, Yingying; Wu, Wei; Xu, Musheng; Prosper, Felipe; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6546-6573</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic regulators that exhibit aberrant enzymic activities or expression profiles are potential therapeutic targets for cancers.  Specifically, enzymes responsible for methylation at histone-3 lysine-9 (like G9a) and aberrant DNA hypermethylation (DNMTs) have been implicated in a no. of cancers.  Recently, mols. bearing a 4-aminoquinoline scaffold were reported as dual inhibitors of these targets and showed a significant in-vivo efficacy in animal models of hematol. malignancies.  Here, we report a detailed exploration around three growing vectors born by this chemotype.  Exploring this chem. space led to the identification of features to navigate G9a and DNMT1 biol. spaces; not only their corresponding exclusive areas, selective compds., but also common spaces.  Thus, we identified from selective G9a and first-in-class DNMT1 inhibitors, > 1 log unit between their IC50 values, with IC50 < 25nM (e.g. 43 and 26, resp.) to equipotent inhibitors with IC50 < 50nM for both targets (e.g. 13).  Their ADME/Tox profiling and antiproliferative efficacies, vs. some cancer cell lines, are also reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqysB6axnTb-LVg90H21EOLACvtfcHk0ljgSqlR57Pu9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSrtL3K&md5=b2579edfb0bb6a4bb9c104d30fd9a28a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01925%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DGarate%26aufirst%3DL.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DMart%25C3%25ADnez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DProsper%26aufirst%3DF.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDetailed%2520Exploration%2520around%25204-Aminoquinolines%2520Chemical%2520Space%2520to%2520Navigate%2520the%2520Lysine%2520Methyltransferase%2520G9a%2520and%2520DNA%2520Methyltransferase%2520Biological%2520Spaces%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6546%26epage%3D6573%26doi%3D10.1021%2Facs.jmedchem.7b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishibe-Murakami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Mechanistic Insights into DNMT1-Mediated Maintenance DNA Methylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">709</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1126/science.1214453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1126%2Fscience.1214453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=22323818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2012&pages=709-712&author=J.+Songauthor=M.+Teplovaauthor=S.+Ishibe-Murakamiauthor=D.+J.+Patel&title=Structure-Based+Mechanistic+Insights+into+DNMT1-Mediated+Maintenance+DNA+Methylation&doi=10.1126%2Fscience.1214453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Mechanistic Insights into DNMT1-Mediated Maintenance DNA Methylation</span></div><div class="casAuthors">Song, Jikui; Teplova, Marianna; Ishibe-Murakami, Satoko; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">6069</span>),
    <span class="NLM_cas:pages">709-712</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">DNMT1, the major maintenance DNA methyltransferase in animals, helps to regulate gene expression, genome imprinting, and X-chromosome inactivation.  We report on the crystal structure of a productive covalent mouse DNMT1(731-1602)-DNA complex contg. a central hemimethylated CpG site.  The Me group of methylcytosine is positioned within a shallow hydrophobic concave surface, whereas the cytosine on the target strand is looped out and covalently anchored within the catalytic pocket.  The DNA is distorted at the hemimethylated CpG step, with side chains from catalytic and recognition loops inserting through both grooves to fill an intercalation-type cavity assocd. with a dual base flip-out on partner strands.  Structural and biochem. data establish how a combination of active and autoinhibitory mechanisms ensures the high fidelity of DNMT1-mediated maintenance DNA methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOZ2gvW2FsqbVg90H21EOLACvtfcHk0ljgSqlR57Pu9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjsrY%253D&md5=d2f1071d7ef7d59efcff6e59b1c215d0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.1214453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1214453%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DTeplova%26aufirst%3DM.%26aulast%3DIshibe-Murakami%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DStructure-Based%2520Mechanistic%2520Insights%2520into%2520DNMT1-Mediated%2520Maintenance%2520DNA%2520Methylation%26jtitle%3DScience%26date%3D2012%26volume%3D335%26spage%3D709%26epage%3D712%26doi%3D10.1126%2Fscience.1214453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Millard, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Celardo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordiyenko, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairall, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwabe, J. W. R.</span></span> <span> </span><span class="NLM_article-title">Class I HDACs Share a Common Mechanism of Regulation by Inositol Phosphates</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.molcel.2013.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=23791785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVygu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2013&pages=57-67&author=C.+J.+Millardauthor=P.+J.+Watsonauthor=I.+Celardoauthor=Y.+Gordiyenkoauthor=S.+M.+Cowleyauthor=C.+V.+Robinsonauthor=L.+Fairallauthor=J.+W.+R.+Schwabe&title=Class+I+HDACs+Share+a+Common+Mechanism+of+Regulation+by+Inositol+Phosphates&doi=10.1016%2Fj.molcel.2013.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Class I HDACs Share a Common Mechanism of Regulation by Inositol Phosphates</span></div><div class="casAuthors">Millard, Christopher J.; Watson, Peter J.; Celardo, Ivana; Gordiyenko, Yuliya; Cowley, Shaun M.; Robinson, Carol V.; Fairall, Louise; Schwabe, John W. R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-67</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Class I histone deacetylases (HDAC1, HDAC2, and HDAC3) are recruited by cognate corepressor proteins into specific transcriptional repression complexes that target HDAC activity to chromatin resulting in chromatin condensation and transcriptional silencing.  The authors previously reported the structure of HDAC3 in complex with the SMRT corepressor.  This structure revealed inositol-tetraphosphate [Ins(1,4,5,6)P4] at the interface of the two proteins.  It was previously unclear whether the role of Ins(1,4,5,6)P4 is to act as a structural cofactor or a regulator of HDAC3 activity.  Here the authors report the structure of HDAC1 in complex with MTA1 from the NuRD complex.  The ELM2-SANT domains from MTA1 wrap completely around HDAC1 occupying both sides of the active site such that the adjacent BAH domain is ideally positioned to recruit nucleosomes to the active site of the enzyme.  Functional assays of both the HDAC1 and HDAC3 complexes reveal that Ins(1,4,5,6)P4 is a bona fide conserved regulator of class I HDAC complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYPhL0103LkLVg90H21EOLACvtfcHk0ljgSqlR57Pu9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVygu74%253D&md5=05a4267bf965b5a6217873217044ef2a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DMillard%26aufirst%3DC.%2BJ.%26aulast%3DWatson%26aufirst%3DP.%2BJ.%26aulast%3DCelardo%26aufirst%3DI.%26aulast%3DGordiyenko%26aufirst%3DY.%26aulast%3DCowley%26aufirst%3DS.%2BM.%26aulast%3DRobinson%26aufirst%3DC.%2BV.%26aulast%3DFairall%26aufirst%3DL.%26aulast%3DSchwabe%26aufirst%3DJ.%2BW.%2BR.%26atitle%3DClass%2520I%2520HDACs%2520Share%2520a%2520Common%2520Mechanism%2520of%2520Regulation%2520by%2520Inositol%2520Phosphates%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D51%26spage%3D57%26epage%3D67%26doi%3D10.1016%2Fj.molcel.2013.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase 6 Structure and Molecular Basis of Catalysis and Inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnchembio.2134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27454933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=741-747&author=Y.+Haiauthor=D.+W.+Christianson&title=Histone+Deacetylase+6+Structure+and+Molecular+Basis+of+Catalysis+and+Inhibition&doi=10.1038%2Fnchembio.2134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span></div><div class="casAuthors">Hai, Yang; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">741-747</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a crit. target for drug design because of its role in oncogenic transformation and cancer metastasis, and is unique among all histone deacetylases in that it contains tandem catalytic domains designated CD1 and CD2.  We now report the crystal structures of CD2 from Homo sapiens HDAC6 and of CD1 and CD2 from Danio rerio HDAC6.  We correlated these structures with activity measurements using 13 different substrates.  The catalytic activity of CD2 from both species exhibited broad substrate specificity, whereas that of CD1 was highly specific for substrates bearing C-terminal acetyllysine residues.  Crystal structures of substrate complexes yielded unprecedented snapshots of the catalytic mechanism.  Addnl., crystal structures of complexes with eight different inhibitors, including belinostat and panobinostat (currently used in cancer chemotherapy), the macrocyclic tetrapeptide HC toxin, and the HDAC6-specific inhibitor N-hydroxy-4-(2-((2-hydroxyethyl)(phenyl)amino)-2-oxoethyl)benzamide, revealed surprising new insight regarding changes in Zn2+ coordination and isoenzyme-specific inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCfziGBHBRLVg90H21EOLACvtfcHk0ljHUWpGfZXQtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ&md5=e167bc6f66e00923da4f2bced2b306b9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2134%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520Deacetylase%25206%2520Structure%2520and%2520Molecular%2520Basis%2520of%2520Catalysis%2520and%2520Inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D741%26epage%3D747%26doi%3D10.1038%2Fnchembio.2134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, Biological Evaluation and in Vivo Testing of Dual Phosphodiesterase 5 (PDE5) and Histone Deacetylase 6 (HDAC6)-Selective Inhibitors for the Treatment of Alzheimer’s Disease</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.ejmech.2018.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=29549837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslaiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2018&pages=506-524&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+Biological+Evaluation+and+in+Vivo+Testing+of+Dual+Phosphodiesterase+5+%28PDE5%29+and+Histone+Deacetylase+6+%28HDAC6%29-Selective+Inhibitors+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease&doi=10.1016%2Fj.ejmech.2018.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, biological evaluation and in vivo testing of dual phosphodiesterase 5 (PDE5) and histone deacetylase 6 (HDAC6)-selective inhibitors for the treatment of Alzheimer's disease</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Tan, Haizhong; Wu, Wei; Xu, Musheng; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">506-524</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The authors have identified chem. probes that act as dual phosphodiesterase 5 (PDE5) and histone deacetylase 6 (HDAC6)-selective inhibitors (>1 log unit difference vs. class I HDACs) to decipher the contribution of HDAC isoforms to the pos. impact of dual-acting PDE5 and HDAC inhibitors on mouse models of Alzheimer's disease (AD) and fine-tune this systems therapeutics approach.  Structure- and knowledge-based approaches led to the design of first-in-class mols. with the desired target compd. profile: dual PDE5 and HDAC6-selective inhibitors.  Compd. 44b (5-[[4-ethoxy-3-(1-methyl-7-oxo-3-propyl-6H-pyrazolo[4,3-d]pyrimidin-5-yl)phenyl]methyl]thiophene-2-carbohydroxamic acid), which fulfilled the biochem., functional and ADME-Tox profiling requirements and exhibited adequate pharmacokinetic properties, was selected as pharmacol. tool compd. and tested in a mouse model of AD (Tg2576) in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFqq9VoRla3bVg90H21EOLACvtfcHk0ljHUWpGfZXQtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslaiur4%253D&md5=1ced6634343dd8c0913ea3e6be07ff91</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520Biological%2520Evaluation%2520and%2520in%2520Vivo%2520Testing%2520of%2520Dual%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520and%2520Histone%2520Deacetylase%25206%2520%2528HDAC6%2529-Selective%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D150%26spage%3D506%26epage%3D524%26doi%3D10.1016%2Fj.ejmech.2018.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of in Vivo Chemical Probes for Treating Alzheimer’s Disease: Dual Phosphodiesterase 5 (PDE5) and Class I Histone Deacetylase Selective Inhibitors</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1765</span>– <span class="NLM_lpage">1782</span>, <span class="refDoi"> DOI: 10.1021/acschemneuro.8b00648</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.8b00648" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1765-1782&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Discovery+of+in+Vivo+Chemical+Probes+for+Treating+Alzheimer%E2%80%99s+Disease%3A+Dual+Phosphodiesterase+5+%28PDE5%29+and+Class+I+Histone+Deacetylase+Selective+Inhibitors&doi=10.1021%2Facschemneuro.8b00648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of in Vivo Chemical Probes for Treating Alzheimer's Disease: Dual Phosphodiesterase 5 (PDE5) and Class I Histone Deacetylase Selective Inhibitors</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Haizhong, Tan; Wei, Wu; Musheng, Xu; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1765-1782</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In order to det. the contributions of histone deacetylase (HDAC) isoforms to the beneficial effects of dual phosphodiesterase 5 (PDE5) and pan-HDAC inhibitors on in vivo models of Alzheimer's disease (AD), we have designed, synthesized, and tested novel chem. probes with the desired target compd. profile of PDE5 and class I HDAC selective inhibitors.  Compared to previous hydroxamate-based series, these mols. exhibit longer residence times on HDACs.  In this scenario, shorter or longer preincubation times may have a significant impact on the IC50 values of these compds. and therefore on their corresponding selectivity profiles on the different HDAC isoforms.  On the other hand, different chem. series have been explored and, as expected, some pairwise comparisons show a clear impact of the scaffold on biol. responses.  The lead identification process led to compd. I, which shows an adequate ADME-Tox profile and in vivo target engagement (histone acetylation and cAMP/cGMP response element-binding (CREB) phosphorylation) in the central nervous system (CNS), suggesting that this compd. represents an optimized chem. probe; thus, I has been assayed in a mouse model of AD (Tg2576).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpMx-DcLUAMrVg90H21EOLACvtfcHk0lhMKkDoH23SSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLvF&md5=d3d13a9121111b166ea69c09106eb3c5</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.8b00648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.8b00648%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520in%2520Vivo%2520Chemical%2520Probes%2520for%2520Treating%2520Alzheimer%25E2%2580%2599s%2520Disease%253A%2520Dual%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520and%2520Class%2520I%2520Histone%2520Deacetylase%2520Selective%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2019%26volume%3D10%26spage%3D1765%26epage%3D1782%26doi%3D10.1021%2Facschemneuro.8b00648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineda-Lucena, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Multitarget Approach for the Treatment of Alzheimer’s Disease: Inhibition of Phosphodiesterase 9 (PDE9) and Histone Deacetylases (HDACs) Covering Diverse Selectivity Profiles</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">4076</span>– <span class="NLM_lpage">4101</span>, <span class="refDoi"> DOI: 10.1021/acschemneuro.9b00303</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.9b00303" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1egu7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=4076-4101&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=H.+Tanauthor=W.+Wuauthor=M.+Xuauthor=A.+Pineda-Lucenaauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Multitarget+Approach+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease%3A+Inhibition+of+Phosphodiesterase+9+%28PDE9%29+and+Histone+Deacetylases+%28HDACs%29+Covering+Diverse+Selectivity+Profiles&doi=10.1021%2Facschemneuro.9b00303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Multitarget Approach for the Treatment of Alzheimer's Disease: Inhibition of Phosphodiesterase 9 (PDE9) and Histone Deacetylases (HDACs) Covering Diverse Selectivity Profiles</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Tan, Haizhong; Wu, Wei; Xu, Musheng; Pineda-Lucena, Antonio; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4076-4101</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present a series of dual-target phosphodiesterase 9 (PDE9) and histone deacetylase (HDAC) inhibitors devised as pharmacol. tool compds. for assessing the implications of these two targets in Alzheimer's disease (AD).  These novel inhibitors were designed taking into account the key pharmacophoric features of known selective PDE9 inhibitors as well as privileged chem. structures, bearing zinc binding groups (hydroxamic acids and ortho-amino anilides) that hit HDAC targets.  These substituents were selected according to rational criteria and previous knowledge from our group to explore diverse HDAC selectivity profiles (pan-HDAC, HDAC6 selective, and class I selective) that were confirmed in biochem. screens.  Their functional response in inducing acetylation of histone and tubulin and phosphorylation of cAMP response element binding (CREB) was measured as a requisite for further progression into complete in vitro absorption, distribution, metab. and excretion (ADME) and in vivo brain penetration profiling.  Compd. 31b, a selective HDAC6 inhibitor with acceptable brain permeability, was chosen for assessing in vivo efficacy of these first-in-class inhibitors, as well as studying their mode of action (MoA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9GkgLW75qELVg90H21EOLACvtfcHk0lgo4zKA1vWtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1egu7rF&md5=be57148d4b99a7e517c51142a1908fb8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.9b00303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.9b00303%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DPineda-Lucena%26aufirst%3DA.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DMultitarget%2520Approach%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%253A%2520Inhibition%2520of%2520Phosphodiesterase%25209%2520%2528PDE9%2529%2520and%2520Histone%2520Deacetylases%2520%2528HDACs%2529%2520Covering%2520Diverse%2520Selectivity%2520Profiles%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2019%26volume%3D10%26spage%3D4076%26epage%3D4101%26doi%3D10.1021%2Facschemneuro.9b00303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagadeeswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterling, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span> <span> </span><span class="NLM_article-title">The Histone Methyltransferase Inhibitor A-366 Uncovers a Role for G9a/GLP in the Epigenetics of Leukemia</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0131716</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0131716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1371%2Fjournal.pone.0131716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=26147105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWktL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=W.+N.+Pappanoauthor=J.+Guoauthor=Y.+Heauthor=D.+Fergusonauthor=S.+Jagadeeswaranauthor=D.+J.+Osterlingauthor=W.+Gaoauthor=J.+K.+Spenceauthor=M.+Pliushchevauthor=R.+F.+Sweisauthor=F.+G.+Buchananauthor=M.+R.+Michaelidesauthor=A.+R.+Shoemakerauthor=C.+Tseauthor=G.+G.+Chiang&title=The+Histone+Methyltransferase+Inhibitor+A-366+Uncovers+a+Role+for+G9a%2FGLP+in+the+Epigenetics+of+Leukemia&doi=10.1371%2Fjournal.pone.0131716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The histone methyltransferase inhibitor A-366 uncovers a role for G9a/GLP in the epigenetics of leukemia</span></div><div class="casAuthors">Pappano, William N.; Guo, Jun; He, Yupeng; Ferguson, Debra; Jagadeeswaran, Sujatha; Osterling, Donald J.; Gao, Wenqing; Spence, Julie K.; Pliushchev, Marina; Sweis, Ramzi F.; Buchanan, Fritz G.; Michaelides, Michael R.; Shoemaker, Alexander R.; Tse, Chris; Chiang, Gary G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0131716/1-e0131716/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Histone methyltransferases are epigenetic regulators that modify key lysine and arginine residues on histones and are believed to play an important role in cancer development and maintenance.  These epigenetic modifications are potentially reversible and as a result this class of enzymes has drawn great interest as potential therapeutic targets of small mol. inhibitors.  Previous studies have suggested that the histone lysine methyltransferase G9a (EHMT2) is required to perpetuate malignant phenotypes through multiple mechanisms in a variety of cancer types.  To further elucidate the enzymic role of G9a in cancer, we describe herein the biol. activities of a novel peptide-competitive histone methyltransferase inhibitor, A-366, that selectively inhibits G9a and the closely related GLP (EHMT1), but not other histone methyltransferases.  A-366 has significantly less cytotoxic effects on the growth of tumor cell lines compared to other known G9a/GLP small mol. inhibitors despite equiv. cellular activity on methylation of H3K9me2.  Addnl., the selectivity profile of A-366 has aided in the discovery of a potentially important role for G9a/GLP in maintenance of leukemia.  Treatment of various leukemia cell lines in vitro resulted in marked differentiation and morphol. changes of these tumor cell lines.  Furthermore, treatment of a flank xenograft leukemia model with A-366 resulted in growth inhibition in vivo consistent with the profile of H3K9me2 redn. obsd.  In summary, A-366 is a novel and highly selective inhibitor of G9a/GLP that has enabled the discovery of a role for G9a/GLP enzymic activity in the growth and differentiation status of leukemia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0f7uh1A8GjrVg90H21EOLACvtfcHk0lgo4zKA1vWtKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWktL7M&md5=33620fc152d0df292c5259ff4f6cf040</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0131716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0131716%26sid%3Dliteratum%253Aachs%26aulast%3DPappano%26aufirst%3DW.%2BN.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DJagadeeswaran%26aufirst%3DS.%26aulast%3DOsterling%26aufirst%3DD.%2BJ.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DSpence%26aufirst%3DJ.%2BK.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26atitle%3DThe%2520Histone%2520Methyltransferase%2520Inhibitor%2520A-366%2520Uncovers%2520a%2520Role%2520for%2520G9a%252FGLP%2520in%2520the%2520Epigenetics%2520of%2520Leukemia%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0131716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, H.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badeti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fritzky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.-g.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarrera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span> <span> </span><span class="NLM_article-title">Efficacy of anti-CD147 chimeric antigen receptors targeting hepatocellular carcinoma</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4810</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18444-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fs41467-020-18444-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=32968061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFOhsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4810&author=H.-c.+Tsengauthor=W.+Xiongauthor=S.+Badetiauthor=Y.+Yangauthor=M.+Maauthor=T.+Liuauthor=C.+A.+Ramosauthor=G.+Dottiauthor=L.+Fritzkyauthor=J.-g.+Jiangauthor=Q.+Yiauthor=J.+Guarreraauthor=W.-X.+Zongauthor=C.+Liuauthor=D.+Liu&title=Efficacy+of+anti-CD147+chimeric+antigen+receptors+targeting+hepatocellular+carcinoma&doi=10.1038%2Fs41467-020-18444-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of anti-CD147 chimeric antigen receptors targeting hepatocellular carcinoma</span></div><div class="casAuthors">Tseng, Hsiang-chi; Xiong, Wei; Badeti, Saiaditya; Yang, Yan; Ma, Minh; Liu, Ting; Ramos, Carlos A.; Dotti, Gianpietro; Fritzky, Luke; Jiang, Jie-gen; Yi, Qing; Guarrera, James; Zong, Wei-Xing; Liu, Chen; Liu, Dongfang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4810</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Chimeric antigen receptor (CAR) therapy is a promising immunotherapeutic strategy for treating multiple refractory blood cancers, but further advances are required for solid tumor CAR therapy.  One challenge is identifying a safe and effective tumor antigen.  Here, we devise a strategy for targeting hepatocellular carcinoma (HCC, one of the deadliest malignancies).  We report that T and NK cells transduced with a CAR that recognizes the surface marker, CD147, also known as Basigin, can effectively kill various malignant HCC cell lines in vitro, and HCC tumors in xenograft and patient-derived xenograft mouse models.  To minimize any on-target/off-tumor toxicity, we use logic-gated (log) GPC3-synNotch-inducible CD147-CAR to target HCC.  LogCD147-CAR selectively kills dual antigen (GPC3+CD147+), but not single antigen (GPC3-CD147+) pos. HCC cells and does not cause severe on-target/off-tumor toxicity in a human CD147 transgenic mouse model.  In conclusion, these findings support the therapeutic potential of CD147-CAR-modified immune cells for HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozkNcgFGZiSrVg90H21EOLACvtfcHk0ljJCCjyVQF2sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFOhsrbE&md5=9d14cd972275fe26618595303401e4a0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18444-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18444-2%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DH.-c.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DBadeti%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DRamos%26aufirst%3DC.%2BA.%26aulast%3DDotti%26aufirst%3DG.%26aulast%3DFritzky%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DJ.-g.%26aulast%3DYi%26aufirst%3DQ.%26aulast%3DGuarrera%26aufirst%3DJ.%26aulast%3DZong%26aufirst%3DW.-X.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DEfficacy%2520of%2520anti-CD147%2520chimeric%2520antigen%2520receptors%2520targeting%2520hepatocellular%2520carcinoma%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4810%26doi%3D10.1038%2Fs41467-020-18444-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1592</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1577-1592&author=Y.+Chenauthor=X.+Yuanauthor=W.+Zhangauthor=M.+Tangauthor=L.+Zhengauthor=F.+Wangauthor=W.+Yanauthor=S.+Yangauthor=Y.+Weiauthor=J.+Heauthor=L.+Chen&title=Discovery+of+Novel+Dual+Histone+Deacetylase+and+Mammalian+Target+of+Rapamycin+Target+Inhibitors+as+a+Promising+Strategy+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.8b01825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span></div><div class="casAuthors">Chen, Yong; Yuan, Xue; Zhang, Wanhua; Tang, Minghai; Zheng, Li; Wang, Fang; Yan, Wei; Yang, Shengyong; Wei, Yuquan; He, Jun; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1577-1592</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present study, a series of novel dual-target histone deacetylase (HDAC) and mammalian target of rapamycin (mTOR) inhibitors were designed and synthesized using pyrimidine-pyrazolyl pharmacophore to append HDAC recognition cap and hydroxamic acid as a zinc-binding motif.  Among them, I was the optimal lead compd. with potent inhibition activities against mTOR and HDAC1 with half-maximal inhibitory concn. of 1.2 and 0.19 nM, resp.  Western blot confirmed that I could upregulate acetylation of H3 and α-tubulin and downregulate mTOR-related downstream mediators.  I could also stimulate cell cycle arrest in G0/G1 phase and induce tumor cell apoptosis.  I showed comparable antitumor activity with the combination medication in MM1S xenograft model with a tumor growth inhibitory rate of 72.5%, without causing significant loss of body wt. and toxicity.  All of the results indicated that I could be a promising dual target inhibitor for treating hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ3AdehNv1ObVg90H21EOLACvtfcHk0ljJCCjyVQF2sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D&md5=7a2ff5020aedb0e3c302e65d993301e6</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01825%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520Novel%2520Dual%2520Histone%2520Deacetylase%2520and%2520Mammalian%2520Target%2520of%2520Rapamycin%2520Target%2520Inhibitors%2520as%2520a%2520Promising%2520Strategy%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1577%26epage%3D1592%26doi%3D10.1021%2Facs.jmedchem.8b01825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre Ena, X.</span>; <span class="NLM_string-name">Oyarzabal Santamarina, J.</span>; <span class="NLM_string-name">Prosper Cardoso, F.</span>; <span class="NLM_string-name">Rabal Gracia, M. O.</span>; <span class="NLM_string-name">Rodriguez Madoz, J. R.</span>; <span class="NLM_string-name">San Jose Eneriz, E.</span></span> <span> </span><span class="NLM_article-title">Novel compounds as dual inhibitors of histone methyltransferases and dna methyltransferases</span>. <span class="NLM_patent">WO 2015192981 A1</span>, March 30, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=X.+Aguirre+Ena&author=J.+Oyarzabal+Santamarina&author=F.+Prosper+Cardoso&author=M.+O.+Rabal+Gracia&author=J.+R.+Rodriguez+Madoz&author=E.+San+Jose+Eneriz&title=Novel+compounds+as+dual+inhibitors+of+histone+methyltransferases+and+dna+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAguirre%2BEna%26aufirst%3DX.%26atitle%3DNovel%2520compounds%2520as%2520dual%2520inhibitors%2520of%2520histone%2520methyltransferases%2520and%2520dna%2520methyltransferases%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre Ena, X.</span>; <span class="NLM_string-name">Oyarzabal Santamarina, J.</span>; <span class="NLM_string-name">Prosper Cardoso, F.</span>; <span class="NLM_string-name">Rabal Gracia, M. O.</span>; <span class="NLM_string-name">San Jose Eneriz, E.</span>; <span class="NLM_string-name">Sanchez Arias, J. A.</span>; <span class="NLM_string-name">Vilas Zornoza, A.</span></span> <span> </span><span class="NLM_article-title">Novel compounds for use in cancer</span>. <span class="NLM_patent">WO 2018229139 A1</span>, June 13, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=X.+Aguirre+Ena&author=J.+Oyarzabal+Santamarina&author=F.+Prosper+Cardoso&author=M.+O.+Rabal+Gracia&author=E.+San+Jose+Eneriz&author=J.+A.+Sanchez+Arias&author=A.+Vilas+Zornoza&title=Novel+compounds+for+use+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAguirre%2BEna%26aufirst%3DX.%26atitle%3DNovel%2520compounds%2520for%2520use%2520in%2520cancer%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orbe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belzunce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Páramo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of Novel Matrix Metalloproteinase Inhibitors As Potent Antihemorrhagic Agents: From Hit Identification to an Optimized Lead</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2488</span>, <span class="refDoi"> DOI: 10.1021/jm501940y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501940y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislGiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2465-2488&author=J.+Orbeauthor=J.+A.+S%C3%A1nchez-Ariasauthor=O.+Rabalauthor=J.+A.+Rodr%C3%ADguezauthor=A.+Salicioauthor=A.+Ugarteauthor=M.+Belzunceauthor=M.+Xuauthor=W.+Wuauthor=H.+Tanauthor=H.+Maauthor=J.+A.+P%C3%A1ramoauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+Novel+Matrix+Metalloproteinase+Inhibitors+As+Potent+Antihemorrhagic+Agents%3A+From+Hit+Identification+to+an+Optimized+Lead&doi=10.1021%2Fjm501940y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of Novel Matrix Metalloproteinase Inhibitors As Potent Antihemorrhagic Agents: From Hit Identification to an Optimized Lead</span></div><div class="casAuthors">Orbe, Josune; Sanchez-Arias, Juan A.; Rabal, Obdulia; Rodriguez, Jose A.; Salicio, Agustina; Ugarte, Ana; Belzunce, Miriam; Xu, Musheng; Wu, Wei; Tan, Haizhong; Ma, Hongyu; Paramo, Jose A.; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2465-2488</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Growing evidence suggests that matrix metalloproteinases (MMP) are involved in thrombus dissoln.; then, considering that new therapeutic strategies are required for controlling hemorrhage, we hypothesized that MMP inhibition may reduce bleeding by delaying fibrinolysis.  Thus, we designed and synthesized a novel series of MMP inhibitors to identify potential candidates for acute treatment of bleeding.  Structure-based and knowledge-based strategies were utilized to design this novel chem. series, α-spiropiperidine hydroxamates, of potent and sol. (>75 μg/mL) pan-MMP inhibitors.  The initial hit was progressed to an optimal lead I.  Racemic I showed a remarkable in vitro phenotypic response and outstanding in vivo efficacy; in fact, the mouse bleeding time at 1 mg/kg was 0.85 min compared to 29.28 min using saline.  In addn., I displayed an optimal ADME and safety profile (e.g., no thrombus formation).  Its corresponding enantiomers were sepd., leading to the preclin. candidate (R)-I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGretnPXVi3YibVg90H21EOLACvtfcHk0lhTkGDmOT85jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislGiurk%253D&md5=1d56002442eae50c83de14d1f217701b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm501940y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501940y%26sid%3Dliteratum%253Aachs%26aulast%3DOrbe%26aufirst%3DJ.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DJ.%2BA.%26aulast%3DSalicio%26aufirst%3DA.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DBelzunce%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DP%25C3%25A1ramo%26aufirst%3DJ.%2BA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520Novel%2520Matrix%2520Metalloproteinase%2520Inhibitors%2520As%2520Potent%2520Antihemorrhagic%2520Agents%253A%2520From%2520Hit%2520Identification%2520to%2520an%2520Optimized%2520Lead%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2465%26epage%3D2488%26doi%3D10.1021%2Fjm501940y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhTkGDmOT85jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Accelrys Software
Inc.</span> <i>Pipeline Pilot</i>, Version
9.5: <span class="NLM_publisher-loc">San Diego, CA</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Accelrys+Software%0AInc.&title=Pipeline+Pilot"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DPipeline%2520Pilot%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yudan  Cui</span>, <span class="hlFld-ContribAuthor ">Jingshan  Cai</span>, <span class="hlFld-ContribAuthor ">Wenxin  Wang</span>, <span class="hlFld-ContribAuthor ">Shengjun  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Regulatory Effects of Histone Deacetylase Inhibitors on Myeloid-Derived Suppressor Cells. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Immunology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fimmu.2021.690207" title="DOI URL">https://doi.org/10.3389/fimmu.2021.690207</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fimmu.2021.690207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffimmu.2021.690207%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Immunology%26atitle%3DRegulatory%252BEffects%252Bof%252BHistone%252BDeacetylase%252BInhibitors%252Bon%252BMyeloid-Derived%252BSuppressor%252BCells%26aulast%3DCui%26aufirst%3DYudan%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0002.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. pan-HDACi (<b>1</b>,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a><b>2</b>,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a><b>3</b><a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a>), Dual DNMT, and HDACi (<b>4</b>),<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Dual G9a and DNMT1 Inhibitors (<b>5</b>, <b>6</b>),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> G9a Inhibitor (<b>7</b>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and Dual G9a and HDACi (<b>8</b>);<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> IC<sub>50</sub> Values for <b>4</b> Were Determined Internally (See Footnote in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and the Rest of the Biochemical IC<sub>50</sub> Values Extracted from the Corresponding References; for <b>8</b>, the Enzymatic Activity of HDAC Was Measured in Intact Cells by the Homogeneous Cellular HDAC Assay Method Using the K562 Cell Line and the Ability to Block G9a by H3K9me2 Cell Immunofluorescence In-Cell Western (ICW) Assays Using the MDA-MB-231 Cell Line<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0004.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Focused exploration around three diversity points (R1, R2, and R3) of the quinoline ring to achieve first-in-class multitarget epigenetic inhibitors targeting DNMT1, HDAC, and, optionally, G9a—some examples are illustrated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0005.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Predicted complex of compound <b>9a</b> with G9a [(A) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>],<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> DNMT1 [(B) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>],<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> HDAC1 [(C) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKX">4BKX</a>]<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and HDAC6 [(D) PDB accession code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>]<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>—experimental validation of proposed binding modes will be experimentally performed by biophysical methods in due course.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0008.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>9a–k</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Na<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (15:1), 110 °C, MW, 4 h; (ii) corresponding amine, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 110–120 °C, 16 h; (iii) HCl/EtOAc(2.0 M) or HCl/MeOH (2.0 M), 20 °C, 30 min to 16 h; (iv) ethyl 4-oxocyclohexanecarboxylate, ZnCl<sub>2</sub>/diethyl ether, NaBH<sub>3</sub>CN, MeOH, 40 °C, 15.5 h; (v) LiOH·H<sub>2</sub>O, EtOH/H<sub>2</sub>O (2:1) or THF/MeOH/H<sub>2</sub>O (5:1:3 or 3:1:1), 20–25 °C, 1–16 h; (vi) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 20 °C, 2–16 h.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0009.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>10a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) methyl boronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-dioxane/H<sub>2</sub>O (10:1), 110 °C, MW, 3 h; (ii) corresponding amine, <i>t</i>-BuONa, xantphos, Pd<sub>2</sub>(dba)<sub>3</sub>, toluene, 110 °C, MW, 1–2 h; (iii) HCl/1,4-dioxane (4.0 M), rt, 1–3 h; (iv) ethyl 2-chloropyrimidine-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, rt, 3 h; (v) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (10:1:3), rt, overnight; (vi) THPONH<sub>2</sub>, HOBt, NMM, EDC·HCl, DMF, rt, overnight; (vii) HCl/1,4-dioxane (4.0 M), rt, 1 h.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0010.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, 95 °C, 12 h; (ii) <i>tert</i>-butyl 4-(bromomethyl)piperidine-1-carboxylate, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 16 h; (iii) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O (4:1), 90 °C, 16 h; (iv) ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 110 °C, 16 h; (v) LiOH·H<sub>2</sub>O, THF/H<sub>2</sub>O (2:1), 15 °C, 16 h; (vi) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 15 °C, 16 h; (vii) HCl/EtOAc (1.0 M), 15 °C, 16 h.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0011.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>12a–k</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) POCl<sub>3</sub>, malonic acid, 100 °C, 16 h; (ii) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O (10:1), 90 °C, 16 h; (iii) corresponding amine, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 120–140 °C, 12–16 h; (iv) HCl/MeOH (2.0 M) or HCl/EtOAc (1.0 or 2.0 M), 20–25 °C, 20 min −16 h; (v) ethyl 2-chloropyrimidine-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMF or CH<sub>3</sub>CN, 40–50 °C, 12–16 h; (vi) corresponding aldehyde or ketone, NaBH<sub>3</sub>CN, CH<sub>3</sub>COOH or ZnCl<sub>2</sub>, MeOH, 40–70 °C, 2–15.5 h; (vii) LiOH·H<sub>2</sub>O, THF/H<sub>2</sub>O (2:1) or THF/MeOH/H<sub>2</sub>O (6:1:1 or 3:1:1), 15–25 °C, 2–16 h; (viii) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 15–25 °C, 2–16 h.</p></p></figure><figure data-id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0012.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>13a–f</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) corresponding boronic ester, Pd(PPh<sub>3</sub>)<sub>4</sub> or Pd(dppf)Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/H<sub>2</sub>O (10:1 or 5:1), 90–100 °C, 6–16 h; (ii) piperidine or ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Cs<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane,100–140 °C, 12–48 h; (iii) Pd/C, H<sub>2</sub> (15 Psi), MeOH, 20 °C, 16 h; (iv) LiOH·H<sub>2</sub>O, THF/MeOH/H<sub>2</sub>O (3:1:1) or THF/H<sub>2</sub>O (2:1), 15–25 °C, 16 h; (v) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 15–20 °C, 12–16 h; (vi) HCl/EtOAc (1.0 or 2.0 M), 15–25 °C, 4–16 h.</p></p></figure><figure data-id="sch6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0013.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Conditions: (i) ethyl 2-[4-(aminomethyl)-1-piperidyl]pyrimidine-5-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 12 h; (ii) 4,4,5,5-tetramethyl-2-(5-methyl-2-furyl)-1,3,2-dioxaborolane, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane, 100 °C, 12 h; (iii) LiOH·H<sub>2</sub>O, THF/H<sub>2</sub>O (2:1), 25 °C, 12 h; (iv) THPONH<sub>2</sub>, HOBt, DIEA, EDCI, DMF, 25 °C, 12 h; (v) TFA, CH<sub>3</sub>CN/H<sub>2</sub>O (1:1), 60 °C, 5 min.</p></p></figure><figure data-id="cht2" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0003.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Selected compound <b>12a</b> and its corresponding quinazoline-matched pair <b>14</b>.</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0006.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Changes in the epigenetic hallmarks following treatment of MM1.S cells with compounds <b>12a</b> and <b>9a</b>. (A) H3K9Ac and H3K9me2 dose–response western blots after incubation for 48 h with the indicated dose range of <b>12a</b> and <b>9a</b>. (B) <i>POU4F2</i> gene pyrosequencing after treatment with <b>12a</b> at 10 nM and <b>9a</b> at 2 μM for 5 days. All assays were performed in duplicate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0007.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>12a</b> shows <i>in vivo</i> efficacy in an MM1.S tumor model. The tumor volume of MM1.S cells subcutaneously injected and treated with the vehicle (80% saline, 10% Tween 20, and 10% DMSO) or compound <b>12a</b> (10 mg/Kg for 5 consecutive days followed by 2 rest days) (<i>n</i> = 9). Statistical significance was calculated by a two-tailed Student’s <i>t</i>-test. *<i>p</i>-value ≤ 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/medium/jm0c02255_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0014.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. (a–h) Synthesis of Intermediates: <b>51</b>, <b>54</b>, <b>57</b>, <b>60</b>, <b>64</b>, <b>68</b>, <b>73</b>, and <b>75</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-6/acs.jmedchem.0c02255/20210319/images/large/jm0c02255_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02255&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Specific reaction conditions for each synthetic step are explicitly detailed in each scheme (a–h).</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i186">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21380" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21380" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 49 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Marked for Death: Targeting Epigenetic Changes in Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnrd.2016.256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28280262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=241-263&author=S.+X.+Pfisterauthor=A.+Ashworth&title=Marked+for+Death%3A+Targeting+Epigenetic+Changes+in+Cancer&doi=10.1038%2Fnrd.2016.256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Marked for death: targeting epigenetic changes in cancer</span></div><div class="casAuthors">Pfister, Sophia Xiao; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">241-263</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In the past few years, it has become clear that mutations in epigenetic regulatory genes are common in human cancers.  Therapeutic strategies are now being developed to target cancers with mutations in these genes using specific chem. inhibitors.  In addn., a complementary approach based on the concept of synthetic lethality, which allows exploitation of loss-of-function mutations in cancers that are not targetable by conventional methods, has gained traction.  Both of these approaches are now being tested in several clin. trials.  In this Review, we present recent advances in epigenetic drug discovery and development, and suggest possible future avenues of investigation to drive progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP-s0n0CRO2rVg90H21EOLACvtfcHk0lj8w89YdUPeRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1Ontr8%253D&md5=f028b0c2e596b7c358e2158318a1303c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.256%26sid%3Dliteratum%253Aachs%26aulast%3DPfister%26aufirst%3DS.%2BX.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DMarked%2520for%2520Death%253A%2520Targeting%2520Epigenetic%2520Changes%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D241%26epage%3D263%26doi%3D10.1038%2Fnrd.2016.256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">Targeting DNA Methylation with Small Molecules: What’s Next?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2569</span>– <span class="NLM_lpage">2583</span>, <span class="refDoi"> DOI: 10.1021/jm500843d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500843d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2569-2583&author=A.+Erdmannauthor=L.+Halbyauthor=J.+Fahyauthor=P.+B.+Arimondo&title=Targeting+DNA+Methylation+with+Small+Molecules%3A+What%E2%80%99s+Next%3F&doi=10.1021%2Fjm500843d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DNA Methylation with Small Molecules: What's Next?</span></div><div class="casAuthors">Erdmann, Alexandre; Halby, Ludovic; Fahy, Jacques; Arimondo, Paola B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2569-2583</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA methylation is a mammalian epigenetic mark that is involved in defining where and when genes are expressed, both in normal cells and in the context of diseases.  Like other epigenetic marks, it is reversible and can be modulated by chem. agents.  Because it plays an important role in cancer by silencing certain genes, such as tumor suppressor genes, and by reactivating other regions, such as repeated elements, it is a promising therapeutic target.  Two compds. are already approved to treat hematol. cancers.  Many efforts have been carried out to discover new mols. that are able to efficiently inhibit DNA methylation in cancer cells.  We will briefly overview the foremost of these efforts by focusing on what we have learned to this point on non-nucleoside inhibitors and on what we consider to be the features of an ideal inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9LcXQUubTNbVg90H21EOLACvtfcHk0li6RceC0skMGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOjsL3L&md5=cc469a073e82dc2cbe2d822cd8c02cce</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm500843d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500843d%26sid%3Dliteratum%253Aachs%26aulast%3DErdmann%26aufirst%3DA.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DFahy%26aufirst%3DJ.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DTargeting%2520DNA%2520Methylation%2520with%2520Small%2520Molecules%253A%2520What%25E2%2580%2599s%2520Next%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2569%26epage%3D2583%26doi%3D10.1021%2Fjm500843d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chistiakov, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myasoedova, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orekhov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bobryshev, Y. V.</span></span> <span> </span><span class="NLM_article-title">Epigenetically Active Drugs Inhibiting DNA Methylation and Histone Deacetylation</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.2174/1381612822666161021110827</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.2174%2F1381612822666161021110827" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27774908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFWht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1167-1174&author=D.+A.+Chistiakovauthor=V.+A.+Myasoedovaauthor=A.+N.+Orekhovauthor=Y.+V.+Bobryshev&title=Epigenetically+Active+Drugs+Inhibiting+DNA+Methylation+and+Histone+Deacetylation&doi=10.2174%2F1381612822666161021110827"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetically Active Drugs Inhibiting DNA Methylation and Histone Deacetylation</span></div><div class="casAuthors">Chistiakov, Dimitry A.; Myasoedova, Veronika A.; Orekhov, Alexander N.; Bobryshev, Yuri V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1167-1174</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetic mechanisms, which are involved in the regulation of gene expression, are tightly controlled.  Loss of a proper epigenetic control can lead to global epigenetic alterations frequently obsd. in various diseases including cancer.  Aberrant epigenetic changes induced in malignant cells lead to emergence of neoplastic properties, which inhibit cell differentiation and strict cell cycle control but greatly enhance stemness-related features.  However, abnormal epigenetic patterns can be reversed by action of epigenetically active agents.  Epigenetic machinery comprises a variety DNA/histone modifiers and chromatin remodelers.  Chem. substances able to influence on the activity of epigenetic factors such as inhibitors of DNA methyltransferases or histone deacetylase inhibitors can be used as therapeutic agents for improving aberrant epigenetic signatures in cancer cells.  Preclin. studies showed efficiency of such epigenetic drugs for the treatment of variety of cancers.  So far, several epigenetically active compds. were approved for therapy of hematol. malignancies.  However, many challenges should be resolved for efficient use of epidrugs in the treatment of non-hematol. solid tumors and advanced cancers assocd. with chemoresistance and higher risk of relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBp259_nzKoLVg90H21EOLACvtfcHk0li6RceC0skMGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFWht78%253D&md5=7fe8fff0865fa2b91478f70aa8576364</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F1381612822666161021110827&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612822666161021110827%26sid%3Dliteratum%253Aachs%26aulast%3DChistiakov%26aufirst%3DD.%2BA.%26aulast%3DMyasoedova%26aufirst%3DV.%2BA.%26aulast%3DOrekhov%26aufirst%3DA.%2BN.%26aulast%3DBobryshev%26aufirst%3DY.%2BV.%26atitle%3DEpigenetically%2520Active%2520Drugs%2520Inhibiting%2520DNA%2520Methylation%2520and%2520Histone%2520Deacetylation%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2017%26volume%3D23%26spage%3D1167%26epage%3D1174%26doi%3D10.2174%2F1381612822666161021110827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases and Their Inhibitors in Cancer, Neurological Diseases and Immune Disorders</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+Deacetylases+and+Their+Inhibitors+in+Cancer%2C+Neurological+Diseases+and+Immune+Disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0li9YGgjSz5otg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520Deacetylases%2520and%2520Their%2520Inhibitors%2520in%2520Cancer%252C%2520Neurological%2520Diseases%2520and%2520Immune%2520Disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachman, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myöhänen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span> <span> </span><span class="NLM_article-title">Synergy of Demethylation and Histone Deacetylase Inhibition in the Re-Expression of Genes Silenced in Cancer</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1038/5047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2F5047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=9916800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1999&pages=103-107&author=E.+E.+Cameronauthor=K.+E.+Bachmanauthor=S.+My%C3%B6h%C3%A4nenauthor=J.+G.+Hermanauthor=S.+B.+Baylin&title=Synergy+of+Demethylation+and+Histone+Deacetylase+Inhibition+in+the+Re-Expression+of+Genes+Silenced+in+Cancer&doi=10.1038%2F5047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer</span></div><div class="casAuthors">Cameron, Elizabeth E.; Bachman, Kurtis E.; Myohanen, Sanna; Herman, James G.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-107</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Densely methylated DNA assocs. with transcriptionally repressive chromatin characterized by the presence of underacetylated histones.  Recently, these 2 epigenetic processes were dynamically linked.  The methyl-CpG-binding protein MeCP2 appears to reside in a complex with histone deacetylase activity.  MeCP2 can mediate formation of transcriptionally repressive chromatin on methylated promoter templates in vitro, and this process can be reversed by trichostatin A (TSA), a specific inhibitor of histone deacetylase.  Little is known about the relative roles of methylation and histone deacetylase activity in the stable inhibition of transcription on densely methylated endogenous promoters, such as those for silenced alleles of imprinted genes, genes on the female inactive X chromosome, and tumor-suppressor genes inactivated in cancer cells.  The authors show that the hypermethylated genes MLH1, TIMP3, CDKN2B (INK4B, p15), and CDKN2A (INK4, p16) cannot be transcriptionally reactivated with TSA alone in tumor cells in which the authors have shown that TSA alone can upregulate the expression of non-methylated genes.  Following minimal demethylation and slight gene reactivation in the presence of low dose 5-aza-2' deoxycytidine (5Aza-dC), however, TSA treatment results in robust re-expression of each gene.  TSA does not contribute to demethylation of the genes, and none of the treatments alter the chromatin structure assocd. with the hypermethylated promoters.  Thus, although DNA methylation and histone deacetylation appear to act as synergistic layers for the silencing of genes in cancer, dense CpG island methylation is dominant for the stable maintenance of a silent state at these loci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZCmJwXaY4o7Vg90H21EOLACvtfcHk0li9YGgjSz5otg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltlWjtA%253D%253D&md5=5f0666dac86ee091d5e06ab0b9403502</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2F5047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F5047%26sid%3Dliteratum%253Aachs%26aulast%3DCameron%26aufirst%3DE.%2BE.%26aulast%3DBachman%26aufirst%3DK.%2BE.%26aulast%3DMy%25C3%25B6h%25C3%25A4nen%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DJ.%2BG.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DSynergy%2520of%2520Demethylation%2520and%2520Histone%2520Deacetylase%2520Inhibition%2520in%2520the%2520Re-Expression%2520of%2520Genes%2520Silenced%2520in%2520Cancer%26jtitle%3DNat.%2520Genet.%26date%3D1999%26volume%3D21%26spage%3D103%26epage%3D107%26doi%3D10.1038%2F5047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grondin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikuchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazitschek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hideshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, K. C.</span></span> <span> </span><span class="NLM_article-title">HDAC3 Regulates DNMT1 Expression in Multiple Myeloma: Therapeutic Implications</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2670</span>– <span class="NLM_lpage">2677</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fleu.2017.144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28490812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=2670-2677&author=T.+Haradaauthor=H.+Ohguchiauthor=Y.+Grondinauthor=S.+Kikuchiauthor=M.+Sagawaauthor=Y.-T.+Taiauthor=R.+Mazitschekauthor=T.+Hideshimaauthor=K.+C.+Anderson&title=HDAC3+Regulates+DNMT1+Expression+in+Multiple+Myeloma%3A+Therapeutic+Implications&doi=10.1038%2Fleu.2017.144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC3 regulates DNMT1 expression in multiple myeloma: therapeutic implications</span></div><div class="casAuthors">Harada, T.; Ohguchi, H.; Grondin, Y.; Kikuchi, S.; Sagawa, M.; Tai, Y.-T.; Mazitschek, R.; Hideshima, T.; Anderson, K. C.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2670-2677</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Epigenetic signaling pathways are implicated in tumorigenesis and therefore histone deacetylases (HDACs) represent novel therapeutic targets for cancers, including multiple myeloma (MM).  Although non-selective HDAC inhibitors show anti-MM activities, unfavorable side effects limit their clin. efficacy.  Isoform- and/or class-selective HDAC inhibition offers the possibility to maintain clin. activity while avoiding adverse events attendant to broad non-selective HDAC inhibition.  We have previously reported that HDAC3 inhibition, either by genetic knockdown or selective inhibitor BG45, abrogates MM cell proliferation.  Here we show that knockdown of HDAC3, but not HDAC1 or HDAC2, as well as BG45, downregulate expression of DNA methyltransferase 1 (DNMT1) mediating MM cell proliferation.  DNMT1 expression is regulated by c-Myc, and HDAC3 inhibition triggers degrdn. of c-Myc protein.  Moreover, HDAC3 inhibition results in hyperacetylation of DNMT1, thereby reducing the stability of DNMT1 protein.  Combined inhibition of HDAC3 and DNMT1 with BG45 and DNMT1 inhibitor 5-azacytidine (AZA), resp., triggers synergistic downregulation of DNMT1, growth inhibition and apoptosis in both MM cell lines and patient MM cells.  Efficacy of this combination treatment is confirmed in a murine xenograft MM model.  Our results therefore provide the rationale for combination treatment using HDAC3 inhibitor with DNMT1 inhibitor to improve patient outcome in MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogqlTnVwSxvLVg90H21EOLACvtfcHk0liqtyfLgAThjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurfJ&md5=1bd7e215aa417f78eb0b51ac0c482144</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.144%26sid%3Dliteratum%253Aachs%26aulast%3DHarada%26aufirst%3DT.%26aulast%3DOhguchi%26aufirst%3DH.%26aulast%3DGrondin%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DS.%26aulast%3DSagawa%26aufirst%3DM.%26aulast%3DTai%26aufirst%3DY.-T.%26aulast%3DMazitschek%26aufirst%3DR.%26aulast%3DHideshima%26aufirst%3DT.%26aulast%3DAnderson%26aufirst%3DK.%2BC.%26atitle%3DHDAC3%2520Regulates%2520DNMT1%2520Expression%2520in%2520Multiple%2520Myeloma%253A%2520Therapeutic%2520Implications%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D2670%26epage%3D2677%26doi%3D10.1038%2Fleu.2017.144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Min, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearstone, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quayle, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Duzer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarpe, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span> <span> </span><span class="NLM_article-title">Selective Inhibitors of Histone Deacetylases 1 and 2 Synergize with Azacitidine in Acute Myeloid Leukemia</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0169128</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0169128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1371%2Fjournal.pone.0169128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28060870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvVegsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=C.+Minauthor=N.+Mooreauthor=J.+R.+Shearstoneauthor=S.+N.+Quayleauthor=P.+Huangauthor=J.+H.+van+Duzerauthor=M.+B.+Jarpeauthor=S.+S.+Jonesauthor=M.+Yang&title=Selective+Inhibitors+of+Histone+Deacetylases+1+and+2+Synergize+with+Azacitidine+in+Acute+Myeloid+Leukemia&doi=10.1371%2Fjournal.pone.0169128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of histone deacetylases 1 and 2 synergize with azacitidine in acute myeloid leukemia</span></div><div class="casAuthors">Min, Chengyin; Moore, Nathan; Shearstone, Jeffrey R.; Quayle, Steven N.; Huang, Pengyu; van Duzer, John H.; Jarpe, Matthew B.; Jones, Simon S.; Yang, Min</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0169128/1-e0169128/22</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a heterogeneous group of hematopoietic stem cell disorders characterized by defects in myeloid differentiation and increased proliferation of neoplastic hematopoietic precursor cells.  Outcomes for patients with AML remain poor, highlighting the need for novel treatment options.  Aberrant epigenetic regulation plays an important role in the pathogenesis of AML, and inhibitors of DNA methyltransferase or histone deacetylase (HDAC) enzymes have exhibited activity in preclin. AML models.  Combination studies with HDAC inhibitors plus DNA methyltransferase inhibitors have potential beneficial clin. activity in AML, however the toxicity profiles of non-selective HDAC inhibitors in the combination setting limit their clin. utility.  In this work, we describe the preclin. development of selective inhibitors of HDAC1 and HDAC2, which are hypothesized to have improved safety profiles, for combination therapy in AML.  We demonstrate that selective inhibition of HDAC1 and HDAC2 is sufficient to achieve efficacy both as a single agent and in combination with azacitidine in preclin. models of AML, including established AML cell lines, primary leukemia cells from AML patient bone marrow samples and in vivo xenograft models of human AML.  Gene expression profiling of AML cells treated with either an HDAC1/2 inhibitor, azacitidine, or the combination of both have identified a list of genes involved in transcription and cell cycle regulation as potential mediators of the combinatorial effects of HDAC1/2 inhibition with azacitidine.  Together, these findings support the clin. evaluation of selective HDAC1/2 inhibitors in combination with azacitidine in AML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkinwKH8HeabVg90H21EOLACvtfcHk0liqtyfLgAThjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvVegsbw%253D&md5=1f549b8d377869735e758ba914749eab</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0169128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0169128%26sid%3Dliteratum%253Aachs%26aulast%3DMin%26aufirst%3DC.%26aulast%3DMoore%26aufirst%3DN.%26aulast%3DShearstone%26aufirst%3DJ.%2BR.%26aulast%3DQuayle%26aufirst%3DS.%2BN.%26aulast%3DHuang%26aufirst%3DP.%26aulast%3Dvan%2BDuzer%26aufirst%3DJ.%2BH.%26aulast%3DJarpe%26aufirst%3DM.%2BB.%26aulast%3DJones%26aufirst%3DS.%2BS.%26aulast%3DYang%26aufirst%3DM.%26atitle%3DSelective%2520Inhibitors%2520of%2520Histone%2520Deacetylases%25201%2520and%25202%2520Synergize%2520with%2520Azacitidine%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0169128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pathania, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariappan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manicassamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolhe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokeshwar, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganapathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thangaraju, M.</span></span> <span> </span><span class="NLM_article-title">Combined Inhibition of DNMT and HDAC Blocks the Tumorigenicity of Cancer Stem-like Cells and Attenuates Mammary Tumor Growth</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3224</span>– <span class="NLM_lpage">3235</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-15-2249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1158%2F0008-5472.CAN-15-2249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27197203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVykt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3224-3235&author=R.+Pathaniaauthor=S.+Ramachandranauthor=G.+Mariappanauthor=P.+Thakurauthor=H.+Shiauthor=J.-H.+Choiauthor=S.+Manicassamyauthor=R.+Kolheauthor=P.+D.+Prasadauthor=S.+Sharmaauthor=B.+L.+Lokeshwarauthor=V.+Ganapathyauthor=M.+Thangaraju&title=Combined+Inhibition+of+DNMT+and+HDAC+Blocks+the+Tumorigenicity+of+Cancer+Stem-like+Cells+and+Attenuates+Mammary+Tumor+Growth&doi=10.1158%2F0008-5472.can-15-2249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Combined Inhibition of DNMT and HDAC Blocks the Tumorigenicity of Cancer Stem-like Cells and Attenuates Mammary Tumor Growth</span></div><div class="casAuthors">Pathania, Rajneesh; Ramachandran, Sabarish; Mariappan, Gurusamy; Thakur, Priyanka; Shi, Huidong; Choi, Jeong-Hyeon; Manicassamy, Santhakumar; Kolhe, Ravindra; Prasad, Puttur D.; Sharma, Suash; Lokeshwar, Bal L.; Ganapathy, Vadivel; Thangaraju, Muthusamy</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3224-3235</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Recently, impressive tech. advancements have been made in the isolation and validation of mammary stem cells and cancer stem cells (CSC), but the signaling pathways that regulate stem cell self-renewal are largely unknown.  Furthermore, CSCs are believed to contribute to chemo- and radioresistance.  In this study, we used the MMTV-Neu-Tg mouse mammary tumor model to identify potential new strategies for eliminating CSCs.  We found that both luminal progenitor and basal stem cells are susceptible to genetic and epigenetic modifications, which facilitate oncogenic transformation and tumorigenic potential.  A combination of the DNMT inhibitor 5-azacytidine and the HDAC inhibitor butyrate markedly reduced CSC abundance and increased the overall survival in this mouse model.  RNA-seq anal. of CSCs treated with 5-azacytidine plus butyrate provided evidence that inhibition of chromatin modifiers blocks growth-promoting signaling mols. such as RAD51AP1 and SPC25, which play key roles in DNA damage repair and kinetochore assembly.  Moreover, RAD51AP1 and SPC25 were significantly overexpressed in human breast tumor tissues and were assocd. with reduced overall patient survival.  In conclusion, our studies suggest that breast CSCs are intrinsically sensitive to genetic and epigenetic modifications and can therefore be significantly affected by epigenetic-based therapies, warranting further investigation of combined DNMT and HDAC inhibition in refractory or drug-resistant breast cancer.  Cancer Res; 76(11); 3224-35. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplWN34n92rV7Vg90H21EOLACvtfcHk0lgDixSHVWZVtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVykt7k%253D&md5=7b90bb0ed6a31755c5fd696ebc66bd3a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2249%26sid%3Dliteratum%253Aachs%26aulast%3DPathania%26aufirst%3DR.%26aulast%3DRamachandran%26aufirst%3DS.%26aulast%3DMariappan%26aufirst%3DG.%26aulast%3DThakur%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DJ.-H.%26aulast%3DManicassamy%26aufirst%3DS.%26aulast%3DKolhe%26aufirst%3DR.%26aulast%3DPrasad%26aufirst%3DP.%2BD.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DLokeshwar%26aufirst%3DB.%2BL.%26aulast%3DGanapathy%26aufirst%3DV.%26aulast%3DThangaraju%26aufirst%3DM.%26atitle%3DCombined%2520Inhibition%2520of%2520DNMT%2520and%2520HDAC%2520Blocks%2520the%2520Tumorigenicity%2520of%2520Cancer%2520Stem-like%2520Cells%2520and%2520Attenuates%2520Mammary%2520Tumor%2520Growth%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3224%26epage%3D3235%26doi%3D10.1158%2F0008-5472.can-15-2249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahnow, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudin, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span> <span> </span><span class="NLM_article-title">The Future of Epigenetic Therapy in Solid Tumours—lessons from the Past</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1038/nrclinonc.2013.42</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnrclinonc.2013.42" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=23546521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvFOrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=256-266&author=N.+Azadauthor=C.+A.+Zahnowauthor=C.+M.+Rudinauthor=S.+B.+Baylin&title=The+Future+of+Epigenetic+Therapy+in+Solid+Tumours%E2%80%94lessons+from+the+Past&doi=10.1038%2Fnrclinonc.2013.42"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The future of epigenetic therapy in solid tumours-lessons from the past</span></div><div class="casAuthors">Azad, Nilofer; Zahnow, Cynthia A.; Rudin, Charles M.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">256-266</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The promise of targeting epigenetic abnormalities for cancer therapy has not been realized for solid tumors, although increasing evidence is demonstrating its worth in haematol. malignancies.  In fact, true clin. efficacy in haematopoietic-related neoplasms has only become evident at low doses of epigenetic-targeting drugs (namely, inhibitors of histone deacetylase and DNA methyltransferases).  Describing data from preclin. studies and early clin. trial results, we hypothesize that in using low-dose epigenetic-modulating agents, tumor cells can be reprogrammed, which overrides any immediate cytotoxic and off-target effect obsd. at high dose.  We suggest that such optimization of drug dosing and scheduling of currently available agents could give these agents a prominent place in cancer management-when used alone or in combination with other therapies.  If so, optimal use of these known agents might also pave the way for the introduction of other agents that target the epigenome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcsvBnYhoB-rVg90H21EOLACvtfcHk0lgDixSHVWZVtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvFOrsr0%253D&md5=f029612627ad09125aafe2f95a01115e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2013.42&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2013.42%26sid%3Dliteratum%253Aachs%26aulast%3DAzad%26aufirst%3DN.%26aulast%3DZahnow%26aufirst%3DC.%2BA.%26aulast%3DRudin%26aufirst%3DC.%2BM.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520Future%2520of%2520Epigenetic%2520Therapy%2520in%2520Solid%2520Tumours%25E2%2580%2594lessons%2520from%2520the%2520Past%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2013%26volume%3D10%26spage%3D256%26epage%3D266%26doi%3D10.1038%2Fnrclinonc.2013.42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Lera, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Polypharmacology: From Combination Therapy to Multitargeted Drugs</span>. <i>Clin. Epigenet.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">105</span>, <span class="refDoi"> DOI: 10.1186/s13148-016-0271-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1186%2Fs13148-016-0271-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27752293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=105&author=A.+R.+de%0ALeraauthor=A.+Ganesan&title=Epigenetic+Polypharmacology%3A+From+Combination+Therapy+to+Multitargeted+Drugs&doi=10.1186%2Fs13148-016-0271-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic polypharmacology: from combination therapy to multitargeted drugs</span></div><div class="casAuthors">de Lera, Angel R.; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Epigenetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105/1-105/21</span>CODEN:
                <span class="NLM_cas:coden">CELPCI</span>;
        ISSN:<span class="NLM_cas:issn">1868-7083</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The modern drug discovery process has largely focused its attention in the so-called magic bullets, single chem. entities that exhibit high selectivity and potency for a particular target.  This approach was based on the assumption that the deregulation of a protein was causally linked to a disease state, and the pharmacol. intervention through inhibition of the deregulated target was able to restore normal cell function.  However, the use of cocktails or multicomponent drugs to address several targets simultaneously is also popular to treat multifactorial diseases such as cancer and neurol. disorders.  We review the state of the art with such combinations that have an epigenetic target as one of their mechanisms of action.  Epigenetic drug discovery is a rapidly advancing field, and drugs targeting epigenetic enzymes are in the clinic for the treatment of hematol. cancers.  Approved and exptl. epigenetic drugs are undergoing clin. trials in combination with other therapeutic agents via fused or linked pharmacophores in order to benefit from synergistic effects of polypharmacol.  In addn., ligands are being discovered which, as single chem. entities, are able to modulate multiple epigenetic targets simultaneously (multitarget epigenetic drugs).  These multiple ligands should in principle have a lower risk of drug-drug interactions and drug resistance compared to cocktails or multicomponent drugs.  This new generation may rival the so-called magic bullets in the treatment of diseases that arise as a consequence of the deregulation of multiple signaling pathways provided the challenge of optimization of the activities shown by the pharmacophores with the different targets is addressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphm-KIEGyYKrVg90H21EOLACvtfcHk0liNZJhM8vd0SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslOgsbY%253D&md5=7cd89a93a22121cc9db2c7118e2e23f1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1186%2Fs13148-016-0271-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13148-016-0271-9%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLera%26aufirst%3DA.%2BR.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DEpigenetic%2520Polypharmacology%253A%2520From%2520Combination%2520Therapy%2520to%2520Multitargeted%2520Drugs%26jtitle%3DClin.%2520Epigenet.%26date%3D2016%26volume%3D8%26spage%3D105%26doi%3D10.1186%2Fs13148-016-0271-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blagitko-Dorfs, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greve, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baude, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brocks, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plass, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lübbert, M.</span></span> <span> </span><span class="NLM_article-title">Combination Treatment of Acute Myeloid Leukemia Cells with DNMT and HDAC Inhibitors: Predominant Synergistic Gene Downregulation Associated with Gene Body Demethylation</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">945</span>– <span class="NLM_lpage">956</span>, <span class="refDoi"> DOI: 10.1038/s41375-018-0293-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fs41375-018-0293-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=30470836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitl2isrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=945-956&author=N.+Blagitko-Dorfsauthor=P.+Schlosserauthor=G.+Greveauthor=D.+Pfeiferauthor=R.+Meierauthor=A.+Baudeauthor=D.+Brocksauthor=C.+Plassauthor=M.+L%C3%BCbbert&title=Combination+Treatment+of+Acute+Myeloid+Leukemia+Cells+with+DNMT+and+HDAC+Inhibitors%3A+Predominant+Synergistic+Gene+Downregulation+Associated+with+Gene+Body+Demethylation&doi=10.1038%2Fs41375-018-0293-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Combination treatment of acute myeloid leukemia cells with DNMT and HDAC inhibitors: predominant synergistic gene downregulation associated with gene body demethylation</span></div><div class="casAuthors">Blagitko-Dorfs, Nadja; Schlosser, Pascal; Greve, Gabriele; Pfeifer, Dietmar; Meier, Ruth; Baude, Annika; Brocks, David; Plass, Christoph; Luebbert, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">945-956</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">DNA methyltransferase inhibitors (DNMTi) approved for older AML patients are clin. tested in combination with histone deacetylase inhibitors (HDACi).  The mechanism of action of these drugs is still under debate.  In colon cancer cells, 5-aza-2'-deoxycytidine (DAC) can downregulate oncogenes and metabolic genes by reversing gene body DNA methylation, thus implicating gene body methylation as a novel drug target.  We asked whether DAC-induced gene body demethylation in AML cells is also assocd. with gene repression, and whether the latter is enhanced by HDACi.  Transcriptome analyses revealed that a combined treatment with DAC and the HDACi panobinostat or valproic acid affected significantly more transcripts than the sum of the genes regulated by either treatment alone, demonstrating a quant. synergistic effect on genome-wide expression in U937 cells.  This effect was particularly striking for downregulated genes.  Integrative methylome and transcriptome analyses showed that a massive downregulation of genes, including oncogenes (e.g., MYC) and epigenetic modifiers (e.g., KDM2B, SUV39H1) often overexpressed in cancer, was assocd. predominantly with gene body DNA demethylation and changes in acH3K9/27.  These findings have implications for the mechanism of action of combined epigenetic treatments, and for a better understanding of responses in trials where this approach is clin. tested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2VMMovamJgrVg90H21EOLACvtfcHk0liNZJhM8vd0SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitl2isrfN&md5=b15ea653d78f9326895650bd557cf30f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41375-018-0293-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-018-0293-8%26sid%3Dliteratum%253Aachs%26aulast%3DBlagitko-Dorfs%26aufirst%3DN.%26aulast%3DSchlosser%26aufirst%3DP.%26aulast%3DGreve%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DD.%26aulast%3DMeier%26aufirst%3DR.%26aulast%3DBaude%26aufirst%3DA.%26aulast%3DBrocks%26aufirst%3DD.%26aulast%3DPlass%26aufirst%3DC.%26aulast%3DL%25C3%25BCbbert%26aufirst%3DM.%26atitle%3DCombination%2520Treatment%2520of%2520Acute%2520Myeloid%2520Leukemia%2520Cells%2520with%2520DNMT%2520and%2520HDAC%2520Inhibitors%253A%2520Predominant%2520Synergistic%2520Gene%2520Downregulation%2520Associated%2520with%2520Gene%2520Body%2520Demethylation%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26spage%3D945%26epage%3D956%26doi%3D10.1038%2Fs41375-018-0293-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehár, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, C. T.</span></span> <span> </span><span class="NLM_article-title">Multi-Target Therapeutics: When the Whole Is Greater than the Sum of the Parts</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2006.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.drudis.2006.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=17198971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ygtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=34-42&author=G.+R.+Zimmermannauthor=J.+Leh%C3%A1rauthor=C.+T.+Keith&title=Multi-Target+Therapeutics%3A+When+the+Whole+Is+Greater+than+the+Sum+of+the+Parts&doi=10.1016%2Fj.drudis.2006.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target therapeutics: when the whole is greater than the sum of the parts</span></div><div class="casAuthors">Zimmermann Grant R; Lehar Joseph; Keith Curtis T</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">34-42</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Drugs designed to act against individual molecular targets cannot usually combat multigenic diseases such as cancer, or diseases that affect multiple tissues or cell types such as diabetes and immunoinflammatory disorders.  Combination drugs that impact multiple targets simultaneously are better at controlling complex disease systems, are less prone to drug resistance and are the standard of care in many important therapeutic areas.  The combination drugs currently employed are primarily of rational design, but the increased efficacy they provide justifies in vitro discovery efforts for identifying novel multi-target mechanisms.  In this review, we discuss the biological rationale for combination therapeutics, review some existing combination drugs and present a systematic approach to identify interactions between molecular pathways that could be leveraged for therapeutic benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTI14vYK2Q1LENHe_gFldTbfW6udTcc2eZkBP92LLCbfLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252Fgt1ygtQ%253D%253D&md5=66715499ce21c2855dc6c477aa87f161</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DZimmermann%26aufirst%3DG.%2BR.%26aulast%3DLeh%25C3%25A1r%26aufirst%3DJ.%26aulast%3DKeith%26aufirst%3DC.%2BT.%26atitle%3DMulti-Target%2520Therapeutics%253A%2520When%2520the%2520Whole%2520Is%2520Greater%2520than%2520the%2520Sum%2520of%2520the%2520Parts%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D34%26epage%3D42%26doi%3D10.1016%2Fj.drudis.2006.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylases as an Epigenetic Pillar for the Development of Hybrid Inhibitors in Cancer</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2019.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.cbpa.2019.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=30986654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsFSgsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2019&pages=89-100&author=G.+Staziauthor=R.+Fioravantiauthor=A.+Maiauthor=A.+Matteviauthor=S.+Valente&title=Histone+Deacetylases+as+an+Epigenetic+Pillar+for+the+Development+of+Hybrid+Inhibitors+in+Cancer&doi=10.1016%2Fj.cbpa.2019.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases as an epigenetic pillar for the development of hybrid inhibitors in cancer</span></div><div class="casAuthors">Stazi, Giulia; Fioravanti, Rossella; Mai, Antonello; Mattevi, Andrea; Valente, Sergio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">89-100</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The polypharmacol. strategy of multi-targeting drugs acting on different biol. pathways is capturing the researchers' attention, particularly in cancer.  The simultaneous inhibition of two or more targets by drug combination or by a single 'hybrid mol.' can provide improved therapeutic efficacy when compared to the one-target inhibitors.  In this regard, because of their multiple anticancer effects, histone deacetylase inhibitors have become a privileged tool for the development of hybrid drugs.  The clin. trials of two multi-acting chimeras, HDAC/EGFR/HER2 and HDAC/PI3K inhibitors, encouraged the design of novel hybrids, such as compds. 22a (LSD1/HDAC) and 16a (CDK4/JAK1/HDAC), which showed superior anticancer effects than single-targeting agents or their combination both in cellular and mouse models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDkcMKMsqtjLVg90H21EOLACvtfcHk0lgFEQwrqFhrzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsFSgsb4%253D&md5=8233a35ccdd5db5fd4c1d5dfdc3958b7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2019.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2019.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DStazi%26aufirst%3DG.%26aulast%3DFioravanti%26aufirst%3DR.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DValente%26aufirst%3DS.%26atitle%3DHistone%2520Deacetylases%2520as%2520an%2520Epigenetic%2520Pillar%2520for%2520the%2520Development%2520of%2520Hybrid%2520Inhibitors%2520in%2520Cancer%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D50%26spage%3D89%26epage%3D100%26doi%3D10.1016%2Fj.cbpa.2019.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piña, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautschi, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.-Y. -S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">France, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornell-Kennon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sambucetti, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remiszewski, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bair, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, P.</span></span> <span> </span><span class="NLM_article-title">Psammaplins from the Sponge Pseudoceratina Purpurea: Inhibition of Both Histone Deacetylase and DNA Methyltransferase</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">3866</span>– <span class="NLM_lpage">3873</span>, <span class="refDoi"> DOI: 10.1021/jo034248t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo034248t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFyhurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=3866-3873&author=I.+C.+Pi%C3%B1aauthor=J.+T.+Gautschiauthor=G.-Y.+-S.+Wangauthor=M.+L.+Sandersauthor=F.+J.+Schmitzauthor=D.+Franceauthor=S.+Cornell-Kennonauthor=L.+C.+Sambucettiauthor=S.+W.+Remiszewskiauthor=L.+B.+Perezauthor=K.+W.+Bairauthor=P.+Crews&title=Psammaplins+from+the+Sponge+Pseudoceratina+Purpurea%3A+Inhibition+of+Both+Histone+Deacetylase+and+DNA+Methyltransferase&doi=10.1021%2Fjo034248t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Psammaplins from the sponge Pseudoceratina purpurea: Inhibition of both histone deacetylase and DNA methyltransferase</span></div><div class="casAuthors">Pina, Ivette C.; Gautschi, Jeffrey T.; Wang, Gui-Yang-Sheng; Sanders, Miranda L.; Schmitz, Francis J.; France, Dennis; Cornell-Kennon, Susan; Sambucetti, Lidia C.; Remiszewski, Stacy W.; Perez, Larry B.; Bair, Kenneth W.; Crews, Phillip</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3866-3873</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four novel bisulfide bromotyrosine derivs., psammaplins E (e.g. I), F, G, and H , and two new bromotyrosine derivs., psammaplins I and J were isolated from the sponge Pseudoceratina purpurea, along with known psammaplins A, B, C, and D and bisaprasin.  The structures of psammaplins E and F, which each contain an oxalyl group rarely found in marine organisms, were detd. by spectroscopic anal.  Compds. psammaplin A, bisaprasin, and psammaplin F are potent histone deacetylase inhibitors and also show mild cytotoxicity.  Furthermore, compds. psammaplin A, bisaprasin, and psammaplin G are potent DNA methyltransferase inhibitors.  The biogenetic pathway previously proposed for the psammaplins class is also revisited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp80TzIEDoTYLVg90H21EOLACvtfcHk0libEk34TZ2P8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivFyhurw%253D&md5=6e6344c8e1ef3eb33ca942f0e76506d9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjo034248t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034248t%26sid%3Dliteratum%253Aachs%26aulast%3DPi%25C3%25B1a%26aufirst%3DI.%2BC.%26aulast%3DGautschi%26aufirst%3DJ.%2BT.%26aulast%3DWang%26aufirst%3DG.-Y.%2B-S.%26aulast%3DSanders%26aufirst%3DM.%2BL.%26aulast%3DSchmitz%26aufirst%3DF.%2BJ.%26aulast%3DFrance%26aufirst%3DD.%26aulast%3DCornell-Kennon%26aufirst%3DS.%26aulast%3DSambucetti%26aufirst%3DL.%2BC.%26aulast%3DRemiszewski%26aufirst%3DS.%2BW.%26aulast%3DPerez%26aufirst%3DL.%2BB.%26aulast%3DBair%26aufirst%3DK.%2BW.%26aulast%3DCrews%26aufirst%3DP.%26atitle%3DPsammaplins%2520from%2520the%2520Sponge%2520Pseudoceratina%2520Purpurea%253A%2520Inhibition%2520of%2520Both%2520Histone%2520Deacetylase%2520and%2520DNA%2520Methyltransferase%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D3866%26epage%3D3873%26doi%3D10.1021%2Fjo034248t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baud, M. G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leiser, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haus, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samlal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrucci, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaratnam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buluwela, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turlais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Almes, F.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchter, M. J.</span></span> <span> </span><span class="NLM_article-title">Defining the Mechanism of Action and Enzymatic Selectivity of Psammaplin A against Its Epigenetic Targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1731</span>– <span class="NLM_lpage">1750</span>, <span class="refDoi"> DOI: 10.1021/jm2016182</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2016182" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Ogur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1731-1750&author=M.+G.+J.+Baudauthor=T.+Leiserauthor=P.+Hausauthor=S.+Samlalauthor=A.+C.+Wongauthor=R.+J.+Woodauthor=V.+Petrucciauthor=M.+Gunaratnamauthor=S.+M.+Hughesauthor=L.+Buluwelaauthor=F.+Turlaisauthor=S.+Neidleauthor=F.-J.+Meyer-Almesauthor=A.+J.+P.+Whiteauthor=M.+J.+Fuchter&title=Defining+the+Mechanism+of+Action+and+Enzymatic+Selectivity+of+Psammaplin+A+against+Its+Epigenetic+Targets&doi=10.1021%2Fjm2016182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the Mechanism of Action and Enzymatic Selectivity of Psammaplin A against Its Epigenetic Targets</span></div><div class="casAuthors">Baud, Matthias G. J.; Leiser, Thomas; Haus, Patricia; Samlal, Sharon; Wong, Ai Ching; Wood, Robert J.; Petrucci, Vanessa; Gunaratnam, Mekala; Hughes, Siobhan M.; Buluwela, Lakjaya; Turlais, Fabrice; Neidle, Stephen; Meyer-Almes, Franz-Josef; White, Andrew J. P.; Fuchter, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1731-1750</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Psammaplin A (11c) is a marine metabolite previously reported to be a potent inhibitor of two classes of epigenetic enzymes: histone deacetylases and DNA methyltransferases.  The design and synthesis of a focused library based on the psammaplin A core has been carried out to probe the mol. features of this mol. responsible for its activity.  By direct in vitro assay of the free thiol generated upon redn. of the dimeric psammaplin scaffold, we have unambiguously demonstrated that 11c functions as a natural prodrug, with the reduced form being highly potent against HDAC1 in vitro (IC50 0.9 nM).  Furthermore, we have shown it to have high isoform selectivity, being 360-fold selective for HDAC1 over HDAC6 and more than 1000-fold less potent against HDAC7 and HDAC8.  SAR around our focused library revealed a no. of features, most notably the oxime functionality to be important to this selectivity.  Many of the compds. show significant cytotoxicity in A549, MCF7, and W138 cells, with the SAR of cytotoxicity correlating to HDAC inhibition.  Furthermore, compd. treatment causes upregulation of histone acetylation but little effect on tubulin acetylation.  Finally, we have found no evidence for 11c functioning as a DNMT inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplUTsYNBVg47Vg90H21EOLACvtfcHk0libEk34TZ2P8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Ogur8%253D&md5=815f9aca36da2e0aa404a341909697e0</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm2016182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2016182%26sid%3Dliteratum%253Aachs%26aulast%3DBaud%26aufirst%3DM.%2BG.%2BJ.%26aulast%3DLeiser%26aufirst%3DT.%26aulast%3DHaus%26aufirst%3DP.%26aulast%3DSamlal%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DA.%2BC.%26aulast%3DWood%26aufirst%3DR.%2BJ.%26aulast%3DPetrucci%26aufirst%3DV.%26aulast%3DGunaratnam%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DS.%2BM.%26aulast%3DBuluwela%26aufirst%3DL.%26aulast%3DTurlais%26aufirst%3DF.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DMeyer-Almes%26aufirst%3DF.-J.%26aulast%3DWhite%26aufirst%3DA.%2BJ.%2BP.%26aulast%3DFuchter%26aufirst%3DM.%2BJ.%26atitle%3DDefining%2520the%2520Mechanism%2520of%2520Action%2520and%2520Enzymatic%2520Selectivity%2520of%2520Psammaplin%2520A%2520against%2520Its%2520Epigenetic%2520Targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1731%26epage%3D1750%26doi%3D10.1021%2Fjm2016182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Rodríguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nebbioso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Rodríguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carafa, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuhldreier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Barrios, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stunnenberg, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gronemeyer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Lera, Á. R.</span></span> <span> </span><span class="NLM_article-title">Indole-Derived Psammaplin A Analogues as Epigenetic Modulators with Multiple Inhibitory Activities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9467</span>– <span class="NLM_lpage">9491</span>, <span class="refDoi"> DOI: 10.1021/jm300618u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300618u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVGqtLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9467-9491&author=R.+Pereiraauthor=R.+Benedettiauthor=S.+P%C3%A9rez-Rodr%C3%ADguezauthor=A.+Nebbiosoauthor=J.+Garc%C3%ADa-Rodr%C3%ADguezauthor=V.+Carafaauthor=M.+Stuhldreierauthor=M.+Conteauthor=F.+Rodr%C3%ADguez-Barriosauthor=H.+G.+Stunnenbergauthor=H.+Gronemeyerauthor=L.+Altucciauthor=%C3%81.+R.+de+Lera&title=Indole-Derived+Psammaplin+A+Analogues+as+Epigenetic+Modulators+with+Multiple+Inhibitory+Activities&doi=10.1021%2Fjm300618u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Indole-Derived Psammaplin A Analogues as Epigenetic Modulators with Multiple Inhibitory Activities</span></div><div class="casAuthors">Pereira, Raquel; Benedetti, Rosaria; Perez-Rodriguez, Santiago; Nebbioso, Angela; Garcia-Rodriguez, Jose; Carafa, Vincenzo; Stuhldreier, Mayra; Conte, Mariarosaria; Rodriguez-Barrios, Fatima; Stunnenberg, Hendrik G.; Gronemeyer, Hinrich; Altucci, Lucia; de Lera, Angel R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9467-9491</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A SAR study has been carried out around a modified scaffold of the natural product psammaplin A obtained by replacing the o-bromophenol unit by an indole ring.  A series of indole psammaplin A constructs were generated in a short synthetic sequence that starts with the functionalization of the C3 indole position with in situ generated nitrosoacrylate, and this is followed by protection of the β-indole-α-oximino esters, sapon., condensation with sym. diamines, and deprotection.  Biochem. and cellular characterization using U937 and MCF-7 cells confirmed that many of these analogs displayed more potent activities than the parent natural product.  Moreover, in addn. to the reported HDAC and DNMT dual epigenetic inhibitory profile of the parent compd., some analogs, notably 4a (UVI5008, I), also inhibited the NAD+-dependent SIRT deacetylase enzymes.  The SAR study provides structural insights into the mechanism of action of these multiple epigenetic ligands and paves the way for addnl. structural exploration to optimize their pharmacol. profiles.  Because of their multi(epi)target features and their action in ex vivo samples, the indole-based psammaplin A derivs. are attractive mols. for the modulation of epigenetic disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd30sExJo0P7Vg90H21EOLACvtfcHk0lgal2iXxMdRiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVGqtLvI&md5=4dd9ee7fd692b94464c1b51037e922ce</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm300618u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300618u%26sid%3Dliteratum%253Aachs%26aulast%3DPereira%26aufirst%3DR.%26aulast%3DBenedetti%26aufirst%3DR.%26aulast%3DP%25C3%25A9rez-Rodr%25C3%25ADguez%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DGarc%25C3%25ADa-Rodr%25C3%25ADguez%26aufirst%3DJ.%26aulast%3DCarafa%26aufirst%3DV.%26aulast%3DStuhldreier%26aufirst%3DM.%26aulast%3DConte%26aufirst%3DM.%26aulast%3DRodr%25C3%25ADguez-Barrios%26aufirst%3DF.%26aulast%3DStunnenberg%26aufirst%3DH.%2BG.%26aulast%3DGronemeyer%26aufirst%3DH.%26aulast%3DAltucci%26aufirst%3DL.%26aulast%3Dde%2BLera%26aufirst%3D%25C3%2581.%2BR.%26atitle%3DIndole-Derived%2520Psammaplin%2520A%2520Analogues%2520as%2520Epigenetic%2520Modulators%2520with%2520Multiple%2520Inhibitory%2520Activities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9467%26epage%3D9491%26doi%3D10.1021%2Fjm300618u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaram, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alalami, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunasekera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramesh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quraishi, U.</span></span> <span> </span><span class="NLM_article-title">(-)-Epigallocatechin-3-Gallate Reverses the Expression of Various Tumor-Suppressor Genes by Inhibiting DNA Methyltransferases and Histone Deacetylases in Human Cervical Cancer Cells</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1976</span>– <span class="NLM_lpage">1984</span>, <span class="refDoi"> DOI: 10.3892/or.2015.3802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.3892%2For.2015.3802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=25682960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFyr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=1976-1984&author=M.+A.+Khanauthor=A.+Hussainauthor=M.+K.+Sundaramauthor=U.+Alalamiauthor=D.+Gunasekeraauthor=L.+Rameshauthor=A.+Hamzaauthor=U.+Quraishi&title=%28-%29-Epigallocatechin-3-Gallate+Reverses+the+Expression+of+Various+Tumor-Suppressor+Genes+by+Inhibiting+DNA+Methyltransferases+and+Histone+Deacetylases+in+Human+Cervical+Cancer+Cells&doi=10.3892%2For.2015.3802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">(-)-Epigallocatechin-3-gallate reverses the expression of various tumor-suppressor genes by inhibiting DNA methyltransferases and histone deacetylases in human cervical cancer cells</span></div><div class="casAuthors">Khan, Munawwar Ali; Hussain, Arif; Sundaram, Madhumitha Kedhari; Al Alami, Usama; Gunasekera, Dian; Ramesh, Laveena; Hamza, Amina; Quraishi, Uzma</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1976-1984</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">There has been increasing evidence that numerous bioactive dietary agents can hamper the process of carcinogenesis by targeting epigenetic alterations including DNA methylation.  This therapeutic approach is considered as a significant goal for cancer therapy due to the reversible nature of epigenetic-mediated gene silencing and warrants further attention.  One such dietary agent, green tea catechin, (-)-epigallocatechin-3-gallate (EGCG) has been shown to modulate many cancer-related pathways.  Thus, the present study was designed to investigate the role of EGCG as an epigenetic modifier in HeLa cells.  DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibition assays were conducted, and the transcription levels of DNMT3B and HDAC1 were assessed by enzymic activity assay and RT-PCR, resp.  Furthermore, we studied the binding interaction of EGCG with DNMT3B and HDAC1 by mol. modeling as well as promoter DNA methylation and expression of retinoic acid receptor-β (RARβ), cadherin 1 (CDH1) and death-assocd. protein kinase-1 (DAPK1) in EGCG-treated HeLa cells by RT-PCR and MS-PCR.  In the present study, time-dependent EGCG-treated HeLa cells were found to have a significant redn. in the enzymic activity of DNMT and HDAC.  However, the expression of DNMT3B was significantly decreased in a time-dependent manner whereas there was no significant change in HDAC1 expression.  Mol. modeling data also supported the EGCG-mediated DNMT3B and HDAC1 activity inhibition.  Furthermore, time-dependent exposure to EGCG resulted in reactivation of known tumor-suppressor genes (TSGs) in HeLa cells due to marked changes in the methylation of the promoter regions of these genes.  Overall, the present study suggests that EGCG may have a significant impact on the development of novel epigenetic-based therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbCuwOug7zBrVg90H21EOLACvtfcHk0lgal2iXxMdRiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFyr&md5=fe18eab228f3947354dffcec52b9997f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3892%2For.2015.3802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2015.3802%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BA.%26aulast%3DHussain%26aufirst%3DA.%26aulast%3DSundaram%26aufirst%3DM.%2BK.%26aulast%3DAlalami%26aufirst%3DU.%26aulast%3DGunasekera%26aufirst%3DD.%26aulast%3DRamesh%26aufirst%3DL.%26aulast%3DHamza%26aufirst%3DA.%26aulast%3DQuraishi%26aufirst%3DU.%26atitle%3D%2528-%2529-Epigallocatechin-3-Gallate%2520Reverses%2520the%2520Expression%2520of%2520Various%2520Tumor-Suppressor%2520Genes%2520by%2520Inhibiting%2520DNA%2520Methyltransferases%2520and%2520Histone%2520Deacetylases%2520in%2520Human%2520Cervical%2520Cancer%2520Cells%26jtitle%3DOncol.%2520Rep.%26date%3D2015%26volume%3D33%26spage%3D1976%26epage%3D1984%26doi%3D10.3892%2For.2015.3802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span> <span> </span><span class="NLM_article-title">Development of a Versatile DNMT and HDAC Inhibitor C02S Modulating Multiple Cancer Hallmarks for Breast Cancer Therapy</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2019.03.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.bioorg.2019.03.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=30901675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2019&pages=200-208&author=Z.+Yuanauthor=S.+Chenauthor=C.+Gaoauthor=Q.+Daiauthor=C.+Zhangauthor=Q.+Sunauthor=J.-S.+Linauthor=C.+Guoauthor=Y.+Chenauthor=Y.+Jiang&title=Development+of+a+Versatile+DNMT+and+HDAC+Inhibitor+C02S+Modulating+Multiple+Cancer+Hallmarks+for+Breast+Cancer+Therapy&doi=10.1016%2Fj.bioorg.2019.03.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a versatile DNMT and HDAC inhibitor C02S modulating multiple cancer hallmarks for breast cancer therapy</span></div><div class="casAuthors">Yuan, Zigao; Chen, Shaopeng; Gao, Chunmei; Dai, Qiuzi; Zhang, Cunlong; Sun, Qinsheng; Lin, Jin-Shun; Guo, Chun; Chen, Yuzong; Jiang, Yuyang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">200-208</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">DNMT and HDAC are closely related to each other and involved in various human diseases esp. cancer.  These two enzymes have been widely recognized as antitumor targets for drug discovery.  Besides, research has indicated that combination therapy consisting of DNMT and HDAC inhibitors exhibited therapeutic advantages.  We have reported a DNMT and HDAC dual inhibitor 15a of which the DNMT enzymic inhibitory potency needs to be improved.  Herein we reported the development of a novel dual DNMT and HDAC inhibitor C02S which showed potent enzymic inhibitory activities against DNMT1, DNMT3A, DNMT3B and HDAC1 with IC50 values of 2.05, 0.93, 1.32, and 4.16 μM, resp.  Further evaluations indicated that C02S could inhibit DNMT and HDAC at cellular levels, thereby inversing mutated methylation and acetylation and increasing expression of tumor suppressor proteins.  Moreover, C02S regulated multiple biol. processes including inducing apoptosis and G0/G1 cell cycle arrest, inhibiting angiogenesis, blocking migration and invasion, and finally suppressing tumor cells proliferation in vitro and tumor growth in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkdrcp-eivG7Vg90H21EOLACvtfcHk0lgW7_rbk9CKbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1ehsLg%253D&md5=55aa67a359816285488aac672f05a65f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2019.03.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2019.03.027%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DDai%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DJ.-S.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520a%2520Versatile%2520DNMT%2520and%2520HDAC%2520Inhibitor%2520C02S%2520Modulating%2520Multiple%2520Cancer%2520Hallmarks%2520for%2520Breast%2520Cancer%2520Therapy%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D87%26spage%3D200%26epage%3D208%26doi%3D10.1016%2Fj.bioorg.2019.03.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Anticancer Potential of NSC-319745 Hydroxamic Acid Derivatives as DNMT and HDAC Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.ejmech.2017.04.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28419930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2017&pages=281-292&author=Z.+Yuanauthor=Q.+Sunauthor=D.+Liauthor=S.+Miaoauthor=S.+Chenauthor=L.+Songauthor=C.+Gaoauthor=Y.+Chenauthor=C.+Tanauthor=Y.+Jiang&title=Design%2C+Synthesis+and+Anticancer+Potential+of+NSC-319745+Hydroxamic+Acid+Derivatives+as+DNMT+and+HDAC+Inhibitors&doi=10.1016%2Fj.ejmech.2017.04.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and anticancer potential of NSC-319745 hydroxamic acid derivatives as DNMT and HDAC inhibitors</span></div><div class="casAuthors">Yuan, Zigao; Sun, Qinsheng; Li, Dan; Miao, Shuangshuang; Chen, Shaopeng; Song, Lu; Gao, Chunmei; Chen, Yuzong; Tan, Chunyan; Jiang, Yuyang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">281-292</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMTs) and histone deacetylases (HDACs) are important epigenetic targets during anticancer drug development.  Recent study indicates that DNMT inhibitors and HDAC inhibitors display synergistic effects in certain cancers, therefore, development of mols. targeting both DNMT and HDAC is of therapeutic advantage against these cancers.  Based on the structure of DNMT inhibitor NSC-319745 and the pharmacophore characteristics of HDAC inhibitors, a series of hydroxamic acid derivs. of NSC-319745 were designed and synthesized as DNMT and HDAC multifunctional inhibitors.  Most compds. displayed potential DNMT inhibitory potency and potent HDAC inhibitory activity, esp. compd. (E)-2-chloro-N-(4-(3-(hydroxyamino)-3-oxoprop-1-en-1-yl) phenethyl)-5(trifluoromethyl)benzamide showed much better DNMT1 inhibitory potency than NSC-319745, and inhibited HDAC1, HDAC6 with IC50 values of 57, 17 nM, resp.  Furthermore, the synthesized compds. exhibited significant cytotoxicity against human cancer cells K562 and U937.  Further mechanistic studies demonstrated that (E)-2-chloro-N-(4-(3-(hydroxyamino)-3-oxoprop-1-en-1-yl) phenethyl)-5(trifluoromethyl)benzamide treatment in U937 increased histones H3K9 and H4K8 acetylation, prompted P16 CpG islands demethylation and upregulated P16 expression, regulated apoptosis-related protein expression on the cellular level and induced remarkable U937 apoptosis.  Moreover, genotoxicity of representative compds. was evaluated.  In summary, the authors' study provided a practical drug design strategy targeting multiple enzymes, and (E)-2-chloro-N-(4-(3-(hydroxyamino)-3-oxoprop-1-en-1-yl) phenethyl)-5(trifluoromethyl)benzamide represents a novel and promising lead compd. for the development of novel epigenetic inhibitors as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE4sLaet4XkrVg90H21EOLACvtfcHk0lgW7_rbk9CKbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrtL4%253D&md5=79777adbb8f9c5ef8209889ce1f1dbf0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DY.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Anticancer%2520Potential%2520of%2520NSC-319745%2520Hydroxamic%2520Acid%2520Derivatives%2520as%2520DNMT%2520and%2520HDAC%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D134%26spage%3D281%26epage%3D292%26doi%3D10.1016%2Fj.ejmech.2017.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomaselli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">Epigenetic polypharmacology: A new frontier for epi-drug discovery</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1002/med.21600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1002%2Fmed.21600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=31218726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntFSjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2020&pages=190-244&author=D.+Tomaselliauthor=A.+Lucidiauthor=D.+Rotiliauthor=A.+Mai&title=Epigenetic+polypharmacology%3A+A+new+frontier+for+epi-drug+discovery&doi=10.1002%2Fmed.21600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic polypharmacology: A new frontier for epi-drug discovery</span></div><div class="casAuthors">Tomaselli, Daniela; Lucidi, Alessia; Rotili, Dante; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">190-244</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Recently, despite the great success achieved by the so-called "magic bullets" in the treatment of different diseases through a marked and specific interaction with the target of interest, the pharmacol. research is moving toward the development of "mol. network active compds.," embracing the related polypharmacol. approach.  This strategy was born to overcome the main limitations of the single target therapy leading to a superior therapeutic effect, a decrease of adverse reactions, and redn. of potential mechanism(s) of drug resistance caused by robustness and redundancy of biol. pathways.  It has become clear that multifactorial diseases such as cancer, neurol., and inflammatory disorders may require more complex therapeutic approaches hitting certain biol. system as a whole.  Concerning epigenetics, goal of multi-epi-target approach consists in the development of small mols. able to simultaneously and (often) reversibly bind different specific epi-targets.  To date, two dual histone deacetylase/kinase inhibitors (CUDC-101 and CUDC-907) are in an advanced stage of clin. trials.  Hence, to update the state-of-the-art of these therapeutic approaches avoiding redundancy, herein we focused only on multiple medication therapies and multitargeting compds. exploiting epigenetic plus nonepigenetic drugs reported in the literature in 2018.  In addn., all the multi-epi-target inhibitors known in literature so far, hitting two or more epigenetic targets, have been included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhhcr1_bhMHbVg90H21EOLACvtfcHk0lhHkCt1Y8usbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntFSjug%253D%253D&md5=bd3f48877c4c40404b22d9278751fa8d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fmed.21600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21600%26sid%3Dliteratum%253Aachs%26aulast%3DTomaselli%26aufirst%3DD.%26aulast%3DLucidi%26aufirst%3DA.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DEpigenetic%2520polypharmacology%253A%2520A%2520new%2520frontier%2520for%2520epi-drug%2520discovery%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2020%26volume%3D40%26spage%3D190%26epage%3D244%26doi%3D10.1002%2Fmed.21600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">San
José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casares, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Subero, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogi, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soule, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiveri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos-Olivas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia Fernandez de Barrena, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barchino, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avila, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span> <span> </span><span class="NLM_article-title">Discovery of First-in-Class Reversible Dual Small Molecule Inhibitors against G9a and DNMTs with in Vivo Activity in Hematological Malignancies</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">15424</span>, <span class="refDoi"> DOI: 10.1038/ncomms15424</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fncomms15424" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28548080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=15424&author=E.+San%0AJos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=O.+Rabalauthor=A.+Vilas-Zornozaauthor=J.+A.+S%C3%A1nchez-Ariasauthor=E.+Mirandaauthor=A.+Ugarteauthor=S.+Roaauthor=B.+Paivaauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=R.+M.+Alvarezauthor=N.+Casaresauthor=V.+Seguraauthor=J.+I.+Mart%C3%ADn-Suberoauthor=F.-X.+Ogiauthor=P.+Souleauthor=C.+M.+Santiveriauthor=R.+Campos-Olivasauthor=G.+Castellanoauthor=M.+Garcia+Fernandez+de+Barrenaauthor=J.+R.+Rodriguez-Madozauthor=M.+J.+Garc%C3%ADa-Barchinoauthor=J.+J.+Lasarteauthor=M.+A.+Avilaauthor=J.+A.+Martinez-Climentauthor=J.+Oyarzabalauthor=F.+Prosper&title=Discovery+of+First-in-Class+Reversible+Dual+Small+Molecule+Inhibitors+against+G9a+and+DNMTs+with+in+Vivo+Activity+in+Hematological+Malignancies&doi=10.1038%2Fncomms15424"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of first-in-class reversible dual small molecule inhibitors against G9a and DNMTs in hematological malignancies</span></div><div class="casAuthors">San Jose-Eneriz, Edurne; Agirre, Xabier; Rabal, Obdulia; Vilas-Zornoza, Amaia; Sanchez-Arias, Juan A.; Miranda, Estibaliz; Ugarte, Ana; Roa, Sergio; Paiva, Bruno; Estella-Hermoso de Mendoza, Ander; Alvarez, Rosa Maria; Casares, Noelia; Segura, Victor; Martin-Subero, Jose I.; Ogi, Francois-Xavier; Soule, Pierre; Santiveri, Clara M.; Campos-Olivas, Ramon; Castellano, Giancarlo; Fernandez de Barrena, Maite Garcia; Rodriguez-Madoz, Juan Roberto; Garcia-Barchino, Maria Jose; Lasarte, Juan Jose; Avila, Matias A.; Martinez-Climent, Jose Angel; Oyarzabal, Julen; Prosper, Felipe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15424</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The indisputable role of epigenetics in cancer and the fact that epigenetic alterations can be reversed have favored development of epigenetic drugs.  In this study, we design and synthesize potent novel, selective and reversible chem. probes that simultaneously inhibit the G9a and DNMTs methyltransferase activity.  In vitro treatment of haematol. neoplasia (acute myeloid leukemia-AML, acute lymphoblastic leukemia-ALL and diffuse large B-cell lymphoma-DLBCL) with the lead compd. CM-272, inhibits cell proliferation and promotes apoptosis, inducing interferon-stimulated genes and immunogenic cell death.  CM-272 significantly prolongs survival of AML, ALL and DLBCL xenogeneic models.  Our results represent the discovery of first-in-class dual inhibitors of G9a/DNMTs and establish this chem. series as a promising therapeutic tool for unmet needs in haematol. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstz1QqrJd4rVg90H21EOLACvtfcHk0lhHkCt1Y8usbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1egs7Y%253D&md5=6d37a6433cdb371bfb535b1fd8a7a803</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fncomms15424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15424%26sid%3Dliteratum%253Aachs%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DPaiva%26aufirst%3DB.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DAlvarez%26aufirst%3DR.%2BM.%26aulast%3DCasares%26aufirst%3DN.%26aulast%3DSegura%26aufirst%3DV.%26aulast%3DMart%25C3%25ADn-Subero%26aufirst%3DJ.%2BI.%26aulast%3DOgi%26aufirst%3DF.-X.%26aulast%3DSoule%26aufirst%3DP.%26aulast%3DSantiveri%26aufirst%3DC.%2BM.%26aulast%3DCampos-Olivas%26aufirst%3DR.%26aulast%3DCastellano%26aufirst%3DG.%26aulast%3DGarcia%2BFernandez%2Bde%2BBarrena%26aufirst%3DM.%26aulast%3DRodriguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DGarc%25C3%25ADa-Barchino%26aufirst%3DM.%2BJ.%26aulast%3DLasarte%26aufirst%3DJ.%2BJ.%26aulast%3DAvila%26aufirst%3DM.%2BA.%26aulast%3DMartinez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DProsper%26aufirst%3DF.%26atitle%3DDiscovery%2520of%2520First-in-Class%2520Reversible%2520Dual%2520Small%2520Molecule%2520Inhibitors%2520against%2520G9a%2520and%2520DNMTs%2520with%2520in%2520Vivo%2520Activity%2520in%2520Hematological%2520Malignancies%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D15424%26doi%3D10.1038%2Fncomms15424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segovia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munera-Maravilla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Fernández, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lodewijk, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segrelles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valcárcel, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casares, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Cabrera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Calderón, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casado, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dueñas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villacampa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasarte, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero-Ramos, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Velasco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prósper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paramio, J. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of a G9a/DNMT Network Triggers Immune-Mediated Bladder Cancer Regression</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1073</span>– <span class="NLM_lpage">1081</span>, <span class="refDoi"> DOI: 10.1038/s41591-019-0499-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fs41591-019-0499-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=31270502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlamtb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1073-1081&author=C.+Segoviaauthor=E.+San+Jos%C3%A9-En%C3%A9rizauthor=E.+Munera-Maravillaauthor=M.+Mart%C3%ADnez-Fern%C3%A1ndezauthor=L.+Garateauthor=E.+Mirandaauthor=A.+Vilas-Zornozaauthor=I.+Lodewijkauthor=C.+Rubioauthor=C.+Segrellesauthor=L.+V.+Valc%C3%A1rcelauthor=O.+Rabalauthor=N.+Casaresauthor=A.+Bernardiniauthor=C.+Suarez-Cabreraauthor=F.+F.+L%C3%B3pez-Calder%C3%B3nauthor=P.+Fortesauthor=J.+A.+Casadoauthor=M.+Due%C3%B1asauthor=F.+Villacampaauthor=J.+J.+Lasarteauthor=F.+Guerrero-Ramosauthor=G.+de+Velascoauthor=J.+Oyarzabalauthor=D.+Castellanoauthor=X.+Agirreauthor=F.+Pr%C3%B3sperauthor=J.+M.+Paramio&title=Inhibition+of+a+G9a%2FDNMT+Network+Triggers+Immune-Mediated+Bladder+Cancer+Regression&doi=10.1038%2Fs41591-019-0499-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of a G9a/DNMT network triggers immune-mediated bladder cancer regression</span></div><div class="casAuthors">Segovia, Cristina; San Jose-Eneriz, Edurne; Munera-Maravilla, Ester; Martinez-Fernandez, Monica; Garate, Leire; Miranda, Estibaliz; Vilas-Zornoza, Amaia; Lodewijk, Iris; Rubio, Carolina; Segrelles, Carmen; Valcarcel, Luis Vitores; Rabal, Obdulia; Casares, Noelia; Bernardini, Alejandra; Suarez-Cabrera, Cristian; Lopez-Calderon, Fernando F.; Fortes, Puri; Casado, Jose A.; Duenas, Marta; Villacampa, Felipe; Lasarte, Juan Jose; Guerrero-Ramos, Felix; de Velasco, Guillermo; Oyarzabal, Julen; Castellano, Daniel; Agirre, Xabier; Prosper, Felipe; Paramio, Jesus M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1073-1081</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bladder cancer is lethal in its advanced, muscle-invasive phase with very limited therapeutic advances1,2.  Recent mol. characterization has defined new (epi)genetic drivers and potential targets for bladder cancer3,4.  The immune checkpoint inhibitors have shown remarkable efficacy but only in a limited fraction of bladder cancer patients5-8.  Here, it show that high G9a (EHMT2) expression is assocd. with poor clin. outcome in bladder cancer and that targeting G9a/DNMT methyltransferase activity with a novel inhibitor (CM-272) induces apoptosis and immunogenic cell death.  Using an immunocompetent quadruple-knockout (PtenloxP/loxP; Trp53loxP/loxP; Rb1loxP/loxP; Rbl1-/-) transgenic mouse model of aggressive metastatic, muscle-invasive bladder cancer, we demonstrate that CM-272 + cisplatin treatment results in statistically significant regression of established tumors and metastases.  The antitumor effect is significantly improved when CM-272 is combined with anti-programmed cell death ligand 1, even in the absence of cisplatin.  These effects are assocd. with an endogenous antitumor immune response and immunogenic cell death with the conversion of a cold immune tumor into a hot tumor.  Finally, increased G9a expression was assocd. with resistance to programmed cell death protein 1 inhibition in a cohort of patients with bladder cancer.  In summary, these findings support new and promising opportunities for the treatment of bladder cancer using a combination of epigenetic inhibitors and immune checkpoint blockade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlqZc1td7xQbVg90H21EOLACvtfcHk0ljoo1DxncY_Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlamtb3O&md5=bc41cbe9b10d641aa5330fbacf5ed3cb</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41591-019-0499-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-019-0499-y%26sid%3Dliteratum%253Aachs%26aulast%3DSegovia%26aufirst%3DC.%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DMunera-Maravilla%26aufirst%3DE.%26aulast%3DMart%25C3%25ADnez-Fern%25C3%25A1ndez%26aufirst%3DM.%26aulast%3DGarate%26aufirst%3DL.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DLodewijk%26aufirst%3DI.%26aulast%3DRubio%26aufirst%3DC.%26aulast%3DSegrelles%26aufirst%3DC.%26aulast%3DValc%25C3%25A1rcel%26aufirst%3DL.%2BV.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DCasares%26aufirst%3DN.%26aulast%3DBernardini%26aufirst%3DA.%26aulast%3DSuarez-Cabrera%26aufirst%3DC.%26aulast%3DL%25C3%25B3pez-Calder%25C3%25B3n%26aufirst%3DF.%2BF.%26aulast%3DFortes%26aufirst%3DP.%26aulast%3DCasado%26aufirst%3DJ.%2BA.%26aulast%3DDue%25C3%25B1as%26aufirst%3DM.%26aulast%3DVillacampa%26aufirst%3DF.%26aulast%3DLasarte%26aufirst%3DJ.%2BJ.%26aulast%3DGuerrero-Ramos%26aufirst%3DF.%26aulast%3Dde%2BVelasco%26aufirst%3DG.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DCastellano%26aufirst%3DD.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DPr%25C3%25B3sper%26aufirst%3DF.%26aulast%3DParamio%26aufirst%3DJ.%2BM.%26atitle%3DInhibition%2520of%2520a%2520G9a%252FDNMT%2520Network%2520Triggers%2520Immune-Mediated%2520Bladder%2520Cancer%2520Regression%26jtitle%3DNat.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D1073%26epage%3D1081%26doi%3D10.1038%2Fs41591-019-0499-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondengaden, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigalapalli, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondengadan, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. G.</span></span> <span> </span><span class="NLM_article-title">Structure Based Design, Synthesis and Activity Studies of Small Hybrid Molecules as HDAC and G9a Dual Inhibitors</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">63187</span>– <span class="NLM_lpage">63207</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.18730</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.18632%2Foncotarget.18730" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=28968981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FksVSnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=63187-63207&author=L.+Zangauthor=S.+M.+Kondengadenauthor=Q.+Zhangauthor=X.+Liauthor=D.+K.+Sigalapalliauthor=S.+M.+Kondengadanauthor=K.+Huangauthor=K.+K.+Liauthor=S.+Liauthor=Z.+Xiaoauthor=L.+Wenauthor=H.+Zhuauthor=B.+N.+Babuauthor=L.+Wangauthor=F.+Cheauthor=P.+G.+Wang&title=Structure+Based+Design%2C+Synthesis+and+Activity+Studies+of+Small+Hybrid+Molecules+as+HDAC+and+G9a+Dual+Inhibitors&doi=10.18632%2Foncotarget.18730"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure based design, synthesis and activity studies of small hybrid molecules as HDAC and G9a dual inhibitors</span></div><div class="casAuthors">Zang Lanlan; Wang Lijuan; Che Fengyuan; Kondengaden Shukkoor M; Zhang Qing; Huang Kenneth; Li Keqin Kathy; Li Shanshan; Xiao Zhongying; Wen Liuqing; Zhu Hailiang; Wang Peng George; Li Xiaobo; Sigalapalli Dilep K; Babu Bathini N; Kondengadan Shameer M</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">63187-63207</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrant enzymatic activities or expression profiles of epigenetic regulations are therapeutic targets for cancers.  Among these, histone 3 lysine 9 methylation (H3K9Me2) and global de-acetylation on histone proteins are associated with multiple cancer phenotypes including leukemia, prostatic carcinoma, hepatocellular carcinoma and pulmonary carcinoma.  Here, we report the discovery of the first small molecule capable of acting as a dual inhibitor targeting both G9a and HDAC.  Our structure based design, synthesis, and screening for the dual activity of the small molecules led to the discovery of compound 14 which displays promising inhibition of both G9a and HDAC in low micro-molar range in cell based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLSKkSsKMGSJcXNb62YGmbfW6udTcc2eZJ8vuofhnOSrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FksVSnsA%253D%253D&md5=ccf8886c55eff785684b72d6bab4db44</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.18730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.18730%26sid%3Dliteratum%253Aachs%26aulast%3DZang%26aufirst%3DL.%26aulast%3DKondengaden%26aufirst%3DS.%2BM.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSigalapalli%26aufirst%3DD.%2BK.%26aulast%3DKondengadan%26aufirst%3DS.%2BM.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DBabu%26aufirst%3DB.%2BN.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChe%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DP.%2BG.%26atitle%3DStructure%2520Based%2520Design%252C%2520Synthesis%2520and%2520Activity%2520Studies%2520of%2520Small%2520Hybrid%2520Molecules%2520as%2520HDAC%2520and%2520G9a%2520Dual%2520Inhibitors%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D63187%26epage%3D63207%26doi%3D10.18632%2Foncotarget.18730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kubicek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">August, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teodoro, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mechtler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalski, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span> <span> </span><span class="NLM_article-title">Reversal of H3K9me2 by a Small-Molecule Inhibitor for the G9a Histone Methyltransferase</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2007.01.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.molcel.2007.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=17289593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=473-481&author=S.+Kubicekauthor=R.+J.+O%E2%80%99Sullivanauthor=E.+M.+Augustauthor=E.+R.+Hickeyauthor=Q.+Zhangauthor=M.+L.+Teodoroauthor=S.+Reaauthor=K.+Mechtlerauthor=J.+A.+Kowalskiauthor=C.+A.+Homonauthor=T.+A.+Kellyauthor=T.+Jenuwein&title=Reversal+of+H3K9me2+by+a+Small-Molecule+Inhibitor+for+the+G9a+Histone+Methyltransferase&doi=10.1016%2Fj.molcel.2007.01.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of H3K9me2 by a small-molecule inhibitor for the G9a histone methyltransferase</span></div><div class="casAuthors">Kubicek, Stefan; O'Sullivan, Roderick J.; August, E. Michael; Hickey, Eugene R.; Zhang, Qiang; Teodoro, Miguel L.; Rea, Stephen; Mechtler, Karl; Kowalski, Jennifer A.; Homon, Carol Ann; Kelly, Terence A.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">473-481</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone lysine methylation has important roles in the organization of chromatin domains and the regulation of gene expression.  To analyze its function and modulate its activity, we screened for specific inhibitors against histone lysine methyltransferases (HMTases) using recombinant G9a as the target enzyme.  From a chem. library comprising 125,000 preselected compds., seven hits were identified.  Of those, one inhibitor, BIX-01294 (diazepinquinazolin-amine deriv.), does not compete with the cofactor S-adenosyl-methionine, and selectively impairs the G9a HMTase and the generation of H3K9me2 in vitro.  In cellular assays, transient incubation of several cell lines with BIX-01294 lowers bulk H3K9me2 levels that are restored upon removal of the inhibitor.  Importantly, chromatin immunopptn. at several G9a target genes demonstrates reversible redn. of promoter-proximal H3K9me2 in inhibitor-treated mouse ES cells and fibroblasts.  Our data identify a biol. active HMTase inhibitor that allows for the transient modulation of H3K9me2 marks in mammalian chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3SW8czS4WjbVg90H21EOLACvtfcHk0liTSC5CKA5TyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXitl2jurs%253D&md5=d2f5290f1252e630a9b05489b0775c2c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DAugust%26aufirst%3DE.%2BM.%26aulast%3DHickey%26aufirst%3DE.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DTeodoro%26aufirst%3DM.%2BL.%26aulast%3DRea%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DKowalski%26aufirst%3DJ.%2BA.%26aulast%3DHomon%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DT.%2BA.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DReversal%2520of%2520H3K9me2%2520by%2520a%2520Small-Molecule%2520Inhibitor%2520for%2520the%2520G9a%2520Histone%2520Methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D25%26spage%3D473%26epage%3D481%26doi%3D10.1016%2Fj.molcel.2007.01.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rival-Gervier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labrie, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dimaggio, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasney, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siarheyeva, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangano, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pattenden, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kireev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tripathy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petronis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">A Chemical Probe Selectively Inhibits G9a and GLP Methyltransferase Activity in Cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">566</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.-C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.-P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+Chemical+Probe+Selectively+Inhibits+G9a+and+GLP+Methyltransferase+Activity+in+Cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0liTSC5CKA5TyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.-C.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520Chemical%2520Probe%2520Selectively%2520Inhibits%2520G9a%2520and%2520GLP%2520Methyltransferase%2520Activity%2520in%2520Cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Valckenborgh, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Riet, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderkerken, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bruyne, E.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Modulating Agents as a New Therapeutic Approach in Multiple Myeloma</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.3390/cancers5020430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.3390%2Fcancers5020430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=24216985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWisLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=430-461&author=K.+Maesauthor=E.+Menuauthor=E.+Van+Valckenborghauthor=I.+Van+Rietauthor=K.+Vanderkerkenauthor=E.+De+Bruyne&title=Epigenetic+Modulating+Agents+as+a+New+Therapeutic+Approach+in+Multiple+Myeloma&doi=10.3390%2Fcancers5020430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic modulating agents as a new therapeutic approach in multiple myeloma</span></div><div class="casAuthors">Maes, Ken; Menu, Eline; Van Valckenborgh, Els; Van Riet, Ivan; Vanderkerken, Karin; De Bruyne, Elke</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">430-461</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Multiple myeloma (MM) is an incurable B-cell malignancy.  Therefore, new targets and drugs are urgently needed to improve patient outcome.  Epigenetic aberrations play a crucial role in development and progression in cancer, including MM.  To target these aberrations, epigenetic modulating agents, such as DNA methyltransferase inhibitors (DNMTi) and histone deacetylase inhibitors (HDACi), are under intense investigation in solid and hematol. cancers.  A clin. benefit of the use of these agents as single agents and in combination regimens has been suggested based on numerous studies in pre-clin. tumor models, including MM models.  The mechanisms of action are not yet fully understood but appear to involve a combination of true epigenetic changes and cytotoxic actions.  In addn., the interactions with the BM niche are also affected by epigenetic modulating agents that will further det. the in vivo efficacy and thus patient outcome.  A better understanding of the mol. events underlying the anti-tumor activity of the epigenetic drugs will lead to more rational drug combinations.  This review focuses on the involvement of epigenetic changes in MM pathogenesis and how the use of DNMTi and HDACi affect the myeloma tumor itself and its interactions with the microenvironment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK1Xdo9RpO2LVg90H21EOLACvtfcHk0ljdkAhBtCKsjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWisLzM&md5=b861e9140f60af2be43de6ac9dfff3bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fcancers5020430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers5020430%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DK.%26aulast%3DMenu%26aufirst%3DE.%26aulast%3DVan%2BValckenborgh%26aufirst%3DE.%26aulast%3DVan%2BRiet%26aufirst%3DI.%26aulast%3DVanderkerken%26aufirst%3DK.%26aulast%3DDe%2BBruyne%26aufirst%3DE.%26atitle%3DEpigenetic%2520Modulating%2520Agents%2520as%2520a%2520New%2520Therapeutic%2520Approach%2520in%2520Multiple%2520Myeloma%26jtitle%3DCancers%26date%3D2013%26volume%3D5%26spage%3D430%26epage%3D461%26doi%3D10.3390%2Fcancers5020430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">San-Miguel, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Einsele, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, P.</span></span> <span> </span><span class="NLM_article-title">The Role of Panobinostat Plus Bortezomib and Dexamethasone in Treating Relapsed or Relapsed and Refractory Multiple Myeloma: A European Perspective</span>. <i>Adv. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1896</span>– <span class="NLM_lpage">1920</span>, <span class="refDoi"> DOI: 10.1007/s12325-016-0413-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1007%2Fs12325-016-0413-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27677481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Srsb3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=1896-1920&author=J.+F.+San-Miguelauthor=H.+Einseleauthor=P.+Moreau&title=The+Role+of+Panobinostat+Plus+Bortezomib+and+Dexamethasone+in+Treating+Relapsed+or+Relapsed+and+Refractory+Multiple+Myeloma%3A+A+European+Perspective&doi=10.1007%2Fs12325-016-0413-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Panobinostat Plus Bortezomib and Dexamethasone in Treating Relapsed or Relapsed and Refractory Multiple Myeloma: A European Perspective</span></div><div class="casAuthors">San-Miguel, Jesus F.; Einsele, Hermann; Moreau, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Therapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1896-1920</span>CODEN:
                <span class="NLM_cas:coden">ADTHE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-238X</span>.
    
            (<span class="NLM_cas:orgname">Springer Healthcare Ltd.</span>)
        </div><div class="casAbstract">Panobinostat is an oral pan-histone deacetylase inhibitor developed by Novartis.  Panobinostat acts via epigenetic modification and inhibition of the aggresome pathway.  In August 2015, the European Commission authorized panobinostat for use in combination with bortezomib and dexamethasone for the treatment of relapsed or relapsed and refractory multiple myeloma (MM) in patients who have received ≥2 prior regimens including bortezomib and an immunomodulatory drug.  In Jan. 2016, the National Institute for Health and Care Excellence recommended panobinostat for use in the same combination and patient population.  The authorization and recommendation were based on results from the pivotal phase 3 PANORAMA 1 (NCT01023308) clin. trial, which demonstrated an improvement in median progression-free survival of 7.8 mo for the three-drug combination compared with placebo plus bortezomib and dexamethasone in this patient population.  This review will discuss the current treatment landscape for relapsed/refractory MM, the mechanism of action of panobinostat, clin. data supporting the European authorization, concerns about safety and strategies for mitigating toxicity, and how panobinostat fits into the current MM landscape in Europe.  Funding: Editorial support, funded by Novartis Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4VbHFN0ga1LVg90H21EOLACvtfcHk0ljdkAhBtCKsjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Srsb3M&md5=7ebba36020f925d5ca137fa8cd7bd57b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2Fs12325-016-0413-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12325-016-0413-7%26sid%3Dliteratum%253Aachs%26aulast%3DSan-Miguel%26aufirst%3DJ.%2BF.%26aulast%3DEinsele%26aufirst%3DH.%26aulast%3DMoreau%26aufirst%3DP.%26atitle%3DThe%2520Role%2520of%2520Panobinostat%2520Plus%2520Bortezomib%2520and%2520Dexamethasone%2520in%2520Treating%2520Relapsed%2520or%2520Relapsed%2520and%2520Refractory%2520Multiple%2520Myeloma%253A%2520A%2520European%2520Perspective%26jtitle%3DAdv.%2520Ther.%26date%3D2016%26volume%3D33%26spage%3D1896%26epage%3D1920%26doi%3D10.1007%2Fs12325-016-0413-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascual, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteve, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raineri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agueda, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russiñol, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queirós, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beekman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Madoz, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">José-Enériz, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Verdugo, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guruceaga, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calasanz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flicek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miguel, J. F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stunnenberg, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gut, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Subero, J. I.</span></span> <span> </span><span class="NLM_article-title">Whole-epigenome analysis in multiple myeloma reveals DNA hypermethylation of B cell-specific enhancers</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">478</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1101/gr.180240.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1101%2Fgr.180240.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=25644835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFCnsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=478-487&author=X.+Agirreauthor=G.+Castellanoauthor=M.+Pascualauthor=S.+Heathauthor=M.+Kulisauthor=V.+Seguraauthor=A.+Bergmannauthor=A.+Esteveauthor=A.+Merkelauthor=E.+Raineriauthor=L.+Aguedaauthor=J.+Blancauthor=D.+Richardsonauthor=L.+Clarkeauthor=A.+Dattaauthor=N.+Russi%C3%B1olauthor=A.+C.+Queir%C3%B3sauthor=R.+Beekmanauthor=J.+R.+Rodr%C3%ADguez-Madozauthor=E.+S.+Jos%C3%A9-En%C3%A9rizauthor=F.+Fangauthor=N.+C.+Guti%C3%A9rrezauthor=J.+M.+Garc%C3%ADa-Verdugoauthor=M.+I.+Robsonauthor=E.+C.+Schirmerauthor=E.+Guruceagaauthor=J.+H.+A.+Martensauthor=M.+Gutauthor=M.+J.+Calasanzauthor=P.+Flicekauthor=R.+Siebertauthor=E.+Campoauthor=J.+F.+S.+Miguelauthor=A.+Melnickauthor=H.+G.+Stunnenbergauthor=I.+G.+Gutauthor=F.+Prosperauthor=J.+I.+Mart%C3%ADn-Subero&title=Whole-epigenome+analysis+in+multiple+myeloma+reveals+DNA+hypermethylation+of+B+cell-specific+enhancers&doi=10.1101%2Fgr.180240.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Whole-epigenome analysis in multiple myeloma reveals DNA hypermethylation of B cell-specific enhancers</span></div><div class="casAuthors">Agirre, Xabier; Castellano, Giancarlo; Pascual, Marien; Heath, Simon; Kulis, Marta; Segura, Victor; Bergmann, Anke; Esteve, Anna; Merkel, Angelika; Raineri, Emanuele; Agueda, Lidia; Blanc, Julie; Richardson, David; Clarke, Laura; Datta, Avik; Russinol, Nuria; Queiros, Ana C.; Beekman, Renee; Rodriguez-Madoz, Juan R.; Jose-Eneriz, Edurne San; Fang, Fang; Gutierrez, Norma C.; Garcia-Verdugo, Jose M.; Robson, Michael I.; Schirmer, Eric C.; Guruceaga, Elisabeth; Martens, Joost H. A.; Gut, Marta; Calasanz, Maria J.; Flicek, Paul; Siebert, Reiner; Campo, Elias; San Miguel, Jesus F.; Melnick, Ari; Stunnenberg, Hendrik G.; Gut, Ivo G.; Prosper, Felipe; Martin-Subero, Jose I.</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">478-487</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">While analyzing the DNA methylome of multiple myeloma (MM), a plasma cell neoplasm, by whole-genome bisulfite sequencing and high-d. arrays, we obsd. a highly heterogeneous pattern globally characterized by regional DNA hypermethylation embedded in extensive hypomethylation.  In contrast to the widely reported DNA hypermethylation of promoter-assocd. CpG islands (CGIs) in cancer, hypermethylated sites in MM, as opposed to normal plasma cells, were located outside CpG islands and were unexpectedly assocd. with intronic enhancer regions defined in normal B cells and plasma cells.  Both RNA-seq and in vitro reporter assays indicated that enhancer hypermethylation is globally assocd. with down-regulation of its host genes.  ChIP-seq and DNase-seq further revealed that DNA hypermethylation in these regions is related to enhancer decommissioning.  Hypermethylated enhancer regions overlapped with binding sites of B cell-specific transcription factors (TFs) and the degree of enhancer methylation inversely correlated with expression levels of these TFs in MM.  Furthermore, hypermethylated regions in MM were methylated in stem cells and gradually became demethylated during normal B-cell differentiation, suggesting that MM cells either reacquire epigenetic features of undifferentiated cells or maintain an epigenetic signature of a putative myeloma stem cell progenitor.  Overall, we have identified DNA hypermethylation of developmentally regulated enhancers as a new type of epigenetic modification assocd. with the pathogenesis of MM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopvhGrSOc8CLVg90H21EOLACvtfcHk0ljH29ORNCIBhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFCnsrs%253D&md5=84b572c460997690cf42a4a7359dec22</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1101%2Fgr.180240.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.180240.114%26sid%3Dliteratum%253Aachs%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DCastellano%26aufirst%3DG.%26aulast%3DPascual%26aufirst%3DM.%26aulast%3DHeath%26aufirst%3DS.%26aulast%3DKulis%26aufirst%3DM.%26aulast%3DSegura%26aufirst%3DV.%26aulast%3DBergmann%26aufirst%3DA.%26aulast%3DEsteve%26aufirst%3DA.%26aulast%3DMerkel%26aufirst%3DA.%26aulast%3DRaineri%26aufirst%3DE.%26aulast%3DAgueda%26aufirst%3DL.%26aulast%3DBlanc%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DD.%26aulast%3DClarke%26aufirst%3DL.%26aulast%3DDatta%26aufirst%3DA.%26aulast%3DRussi%25C3%25B1ol%26aufirst%3DN.%26aulast%3DQueir%25C3%25B3s%26aufirst%3DA.%2BC.%26aulast%3DBeekman%26aufirst%3DR.%26aulast%3DRodr%25C3%25ADguez-Madoz%26aufirst%3DJ.%2BR.%26aulast%3DJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%2BS.%26aulast%3DFang%26aufirst%3DF.%26aulast%3DGuti%25C3%25A9rrez%26aufirst%3DN.%2BC.%26aulast%3DGarc%25C3%25ADa-Verdugo%26aufirst%3DJ.%2BM.%26aulast%3DRobson%26aufirst%3DM.%2BI.%26aulast%3DSchirmer%26aufirst%3DE.%2BC.%26aulast%3DGuruceaga%26aufirst%3DE.%26aulast%3DMartens%26aufirst%3DJ.%2BH.%2BA.%26aulast%3DGut%26aufirst%3DM.%26aulast%3DCalasanz%26aufirst%3DM.%2BJ.%26aulast%3DFlicek%26aufirst%3DP.%26aulast%3DSiebert%26aufirst%3DR.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DMiguel%26aufirst%3DJ.%2BF.%2BS.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DStunnenberg%26aufirst%3DH.%2BG.%26aulast%3DGut%26aufirst%3DI.%2BG.%26aulast%3DProsper%26aufirst%3DF.%26aulast%3DMart%25C3%25ADn-Subero%26aufirst%3DJ.%2BI.%26atitle%3DWhole-epigenome%2520analysis%2520in%2520multiple%2520myeloma%2520reveals%2520DNA%2520hypermethylation%2520of%2520B%2520cell-specific%2520enhancers%26jtitle%3DGenome%2520Res.%26date%3D2015%26volume%3D25%26spage%3D478%26epage%3D487%26doi%3D10.1101%2Fgr.180240.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ordoñez, R.</span>; <span class="NLM_string-name">Kulis, M.</span>; <span class="NLM_string-name">Russiñol, N.</span>; <span class="NLM_string-name">Chapaprieta, V.</span>; <span class="NLM_string-name">Beekman, R.</span>; <span class="NLM_string-name">Meydan, C.</span>; <span class="NLM_string-name">Duran-Ferrer, M.</span>; <span class="NLM_string-name">Verdaguer-Dot, N.</span>; <span class="NLM_string-name">Clot, G.</span>; <span class="NLM_string-name">Vilarrasa-Blasi, R.</span>; <span class="NLM_string-name">Garate, L.</span>; <span class="NLM_string-name">Miranda, E.</span>; <span class="NLM_string-name">Carrasco, A.</span>; <span class="NLM_string-name">Ezponda, T.</span>; <span class="NLM_string-name">Vilas-Zornoza, A.</span>; <span class="NLM_string-name">Lara-Astiaso, D.</span>; <span class="NLM_string-name">Dupéré-Richer, D.</span>; <span class="NLM_string-name">Martens, J. H. A.</span>; <span class="NLM_string-name">Torrents, D.</span>; <span class="NLM_string-name">El-Omri, H.</span>; <span class="NLM_string-name">Taha, R. Y.</span>; <span class="NLM_string-name">Calasanz, M. J.</span>; <span class="NLM_string-name">Paiva, B.</span>; <span class="NLM_string-name">Miguel, J. S.</span>; <span class="NLM_string-name">Flicek, P.</span>; <span class="NLM_string-name">Gut, I.</span>; <span class="NLM_string-name">Melnick, A.</span>; <span class="NLM_string-name">Mitsiades, C. S.</span>; <span class="NLM_string-name">Licht, J. D.</span>; <span class="NLM_string-name">Campo, E.</span>; <span class="NLM_string-name">Stunnenberg, H. G.</span>; <span class="NLM_string-name">Agirre, X.</span>; <span class="NLM_string-name">Prosper, F.</span>; <span class="NLM_string-name">Martin-Subero, J. I.</span></span> <span> </span><span class="NLM_article-title">Chromatin activation as a unifying principle underlying pathogenic mechanisms in multiple myeloma</span>. <span class="NLM_year" style="font-weight: bold;">2019</span>, bioRxiv:740027.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=R.+Ordo%C3%B1ez&author=M.+Kulis&author=N.+Russi%C3%B1ol&author=V.+Chapaprieta&author=R.+Beekman&author=C.+Meydan&author=M.+Duran-Ferrer&author=N.+Verdaguer-Dot&author=G.+Clot&author=R.+Vilarrasa-Blasi&author=L.+Garate&author=E.+Miranda&author=A.+Carrasco&author=T.+Ezponda&author=A.+Vilas-Zornoza&author=D.+Lara-Astiaso&author=D.+Dup%C3%A9r%C3%A9-Richer&author=J.+H.+A.+Martens&author=D.+Torrents&author=H.+El-Omri&author=R.+Y.+Taha&author=M.+J.+Calasanz&author=B.+Paiva&author=J.+S.+Miguel&author=P.+Flicek&author=I.+Gut&author=A.+Melnick&author=C.+S.+Mitsiades&author=J.+D.+Licht&author=E.+Campo&author=H.+G.+Stunnenberg&author=X.+Agirre&author=F.+Prosper&author=J.+I.+Martin-Subero&title=Chromatin+activation+as+a+unifying+principle+underlying+pathogenic+mechanisms+in+multiple+myeloma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOrdo%25C3%25B1ez%26aufirst%3DR.%26atitle%3DChromatin%2520activation%2520as%2520a%2520unifying%2520principle%2520underlying%2520pathogenic%2520mechanisms%2520in%2520multiple%2520myeloma%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricobaraza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, R.</span></span> <span> </span><span class="NLM_article-title">Defining the Mechanism of Action of 4-Phenylbutyrate to Develop a Small-Molecule-Based Therapy for Alzheimer’s Disease</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5545</span>– <span class="NLM_lpage">5553</span>, <span class="refDoi"> DOI: 10.2174/092986711798347315</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.2174%2F092986711798347315" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=22172064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12ntbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=5545-5553&author=M.+Cuadrado-Tejedorauthor=A.+Garc%C3%ADa-Ostaauthor=A.+Ricobarazaauthor=J.+Oyarzabalauthor=R.+Franco&title=Defining+the+Mechanism+of+Action+of+4-Phenylbutyrate+to+Develop+a+Small-Molecule-Based+Therapy+for+Alzheimer%E2%80%99s+Disease&doi=10.2174%2F092986711798347315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Defining the mechanism of action of 4-phenylbutyrate to develop a small-molecule-based therapy for Alzheimer's disease</span></div><div class="casAuthors">Cuadrado-Tejedor, M.; Garcia-Osta, A.; Ricobaraza, A.; Oyarzabal, J.; Franco, R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">5545-5553</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  4-Phenylbutyrate (PBA) is a small mol. that restores cognitive deficits in animal models of Alzheimer's disease (AD).  Although the mol. basis of the cognitive benefits of PBA remains unknown, a multi-modal/multi-target mechanism has been proposed.  Putative targets of this drug are different from those of drugs that are now used in clin. trials.  As PBA is already administered to patients with congenital defects affecting enzymes in the urea cycle, it can be rapidly tested in AD clin. trials.  However, the main drawback to its therapeutic use is the high dosage required (up to 15 g/day).  Thus, deciphering the precise mechanism(s) of action of this drug may enable novel drugs with similar therapeutic effects to PBA to be developed that can be used at more manageable doses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosAevZlfyhp7Vg90H21EOLACvtfcHk0ljSUUrGOiTHqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12ntbbL&md5=0703163333de9afce25c352e4ef98df5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.2174%2F092986711798347315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711798347315%26sid%3Dliteratum%253Aachs%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Osta%26aufirst%3DA.%26aulast%3DRicobaraza%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26aulast%3DFranco%26aufirst%3DR.%26atitle%3DDefining%2520the%2520Mechanism%2520of%2520Action%2520of%25204-Phenylbutyrate%2520to%2520Develop%2520a%2520Small-Molecule-Based%2520Therapy%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D5545%26epage%3D5553%26doi%3D10.2174%2F092986711798347315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span> <span> </span><span class="NLM_article-title">Development of the Pan-DAC Inhibitor Panobinostat (LBH589): Successes and Challenges</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2009.02.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.canlet.2009.02.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=19344997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2009&pages=233-241&author=P.+Atadja&title=Development+of+the+Pan-DAC+Inhibitor+Panobinostat+%28LBH589%29%3A+Successes+and+Challenges&doi=10.1016%2Fj.canlet.2009.02.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the pan-DAC inhibitor panobinostat (LBH589): Successes and challenges</span></div><div class="casAuthors">Atadja, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Shannon, Ireland)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">233-241</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  The histone deacetylase (HDAC) inhibitors are emerging as a highly useful class of anticancer agents that inhibit the enzyme HDAC involved in the deacetylation of histone and non-histone cellular proteins.  The HDAC inhibitor, panobinostat (LBH589, Novartis Pharmaceuticals), achieves potent inhibition of all HDAC enzymes implicated in cancer and has demonstrated potent anti-tumor activity in preclin. models and promising clin. efficacy in cancer patients.  In this review we discuss the successes and challenges surrounding the development of panobinostat, focusing on its proposed mechanism of action, preclin. anti-tumor activity, and early clin. efficacy in hematol. and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSTYxNKUMt7bVg90H21EOLACvtfcHk0ljSUUrGOiTHqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsF2jtL4%253D&md5=4c487a2f8dc13f72cecf105d9fae97d9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2009.02.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2009.02.019%26sid%3Dliteratum%253Aachs%26aulast%3DAtadja%26aufirst%3DP.%26atitle%3DDevelopment%2520of%2520the%2520Pan-DAC%2520Inhibitor%2520Panobinostat%2520%2528LBH589%2529%253A%2520Successes%2520and%2520Challenges%26jtitle%3DCancer%2520Lett.%26date%3D2009%26volume%3D280%26spage%3D233%26epage%3D241%26doi%3D10.1016%2Fj.canlet.2009.02.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariën, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floren, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beliën, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilatte, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decrane, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilissen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vreys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bol, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talloen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goris, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andries, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du Jardin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janicot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Emelen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angibaud, P.</span></span> <span> </span><span class="NLM_article-title">JNJ-26481585, a Novel “second-Generation” oral Histone Deacetylase Inhibitor, Shows Broad-Spectrum Preclinical Antitumoral Activity</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6841</span>– <span class="NLM_lpage">6851</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-09-0547</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1158%2F1078-0432.CCR-09-0547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=19861438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtl2ktLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=6841-6851&author=J.+Artsauthor=P.+Kingauthor=A.+Mari%C3%ABnauthor=W.+Florenauthor=A.+Beli%C3%ABnauthor=L.+Janssenauthor=I.+Pilatteauthor=B.+Rouxauthor=L.+Decraneauthor=R.+Gilissenauthor=I.+Hicksonauthor=V.+Vreysauthor=E.+Coxauthor=K.+Bolauthor=W.+Talloenauthor=I.+Gorisauthor=L.+Andriesauthor=M.+Du+Jardinauthor=M.+Janicotauthor=M.+Pageauthor=K.+van+Emelenauthor=P.+Angibaud&title=JNJ-26481585%2C+a+Novel+%E2%80%9Csecond-Generation%E2%80%9D+oral+Histone+Deacetylase+Inhibitor%2C+Shows+Broad-Spectrum+Preclinical+Antitumoral+Activity&doi=10.1158%2F1078-0432.ccr-09-0547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">JNJ-26481585, a Novel "Second-Generation" Oral Histone Deacetylase Inhibitor, Shows Broad-Spectrum Preclinical Antitumoral Activity</span></div><div class="casAuthors">Arts, Janine; King, Peter; Marien, Ann; Floren, Wim; Belien, Ann; Janssen, Lut; Pilatte, Isabelle; Roux, Bruno; Decrane, Laurence; Gilissen, Ron; Hickson, Ian; Vreys, Veronique; Cox, Eugene; Bol, Kees; Talloen, Willem; Goris, Ilse; Andries, Luc; Du Jardin, Marc; Janicot, Michel; Page, Martin; van Emelen, Kristof; Angibaud, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6841-6851</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Histone deacetylase (HDAC) inhibitors have shown promising clin. activity in the treatment of hematol. malignancies, but their activity in solid tumor indications has been limited.  Most HDAC inhibitors in clin. development only transiently induce histone acetylation in tumor tissue.  Here, we sought to identify a "second-generation" class I HDAC inhibitor with prolonged pharmacodynamic response in vivo, to assess whether this results in superior antitumoral efficacy.  Exptl. Design: To identify novel HDAC inhibitors with superior pharmacodynamic properties, we developed a preclin. in vivo tumor model, in which tumor cells have been engineered to express fluorescent protein dependent on HDAC1 inhibition, thereby allowing noninvasive real-time evaluation of the tumor response to HDAC inhibitors.  Results: In vivo pharmacodynamic anal. of 140 potent pyrimidyl-hydroxamic acid analogs resulted in the identification of JNJ-26481585.  Once daily oral administration of JNJ-26481585 induced continuous histone H3 acetylation.  The prolonged pharmacodynamic response translated into complete tumor growth inhibition in Ras mutant HCT116 colon carcinoma xenografts, whereas 5-fluorouracil was less active.  JNJ-26481585 also fully inhibited the growth of C170HM2 colorectal liver metastases, whereas again 5-fluorouracil/Leucovorin showed modest activity.  Further characterization revealed that JNJ-26481585 is a pan-HDAC inhibitor with marked potency toward HDAC1 (IC50, 0.16 nmol/L).  Conclusions: The potent antitumor activity as a single agent in preclin. models combined with its favorable pharmacodynamic profile makes JNJ-26481585 a promising "second-generation" HDAC inhibitor.  The compd. is currently in clin. studies, to evaluate its potential applicability in a broad spectrum of both solid and hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKc9hyMiADZrVg90H21EOLACvtfcHk0lil_3QW_oQiBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtl2ktLzE&md5=8f199c25c92c967a76f273d879d6c577</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-0547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-0547%26sid%3Dliteratum%253Aachs%26aulast%3DArts%26aufirst%3DJ.%26aulast%3DKing%26aufirst%3DP.%26aulast%3DMari%25C3%25ABn%26aufirst%3DA.%26aulast%3DFloren%26aufirst%3DW.%26aulast%3DBeli%25C3%25ABn%26aufirst%3DA.%26aulast%3DJanssen%26aufirst%3DL.%26aulast%3DPilatte%26aufirst%3DI.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DDecrane%26aufirst%3DL.%26aulast%3DGilissen%26aufirst%3DR.%26aulast%3DHickson%26aufirst%3DI.%26aulast%3DVreys%26aufirst%3DV.%26aulast%3DCox%26aufirst%3DE.%26aulast%3DBol%26aufirst%3DK.%26aulast%3DTalloen%26aufirst%3DW.%26aulast%3DGoris%26aufirst%3DI.%26aulast%3DAndries%26aufirst%3DL.%26aulast%3DDu%2BJardin%26aufirst%3DM.%26aulast%3DJanicot%26aufirst%3DM.%26aulast%3DPage%26aufirst%3DM.%26aulast%3Dvan%2BEmelen%26aufirst%3DK.%26aulast%3DAngibaud%26aufirst%3DP.%26atitle%3DJNJ-26481585%252C%2520a%2520Novel%2520%25E2%2580%259Csecond-Generation%25E2%2580%259D%2520oral%2520Histone%2520Deacetylase%2520Inhibitor%252C%2520Shows%2520Broad-Spectrum%2520Preclinical%2520Antitumoral%2520Activity%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D6841%26epage%3D6851%26doi%3D10.1158%2F1078-0432.ccr-09-0547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
De Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer’s Disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8967</span>– <span class="NLM_lpage">9004</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8967-9004&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0ADe+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+First-in-Class+Dual+Acting+Histone+Deacetylases+%28HDACs%29+and+Phosphodiesterase+5+%28PDE5%29+Inhibitors+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease&doi=10.1021%2Facs.jmedchem.6b00908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of First-in-Class Dual Acting Histone Deacetylases (HDACs) and Phosphodiesterase 5 (PDE5) Inhibitors for the Treatment of Alzheimer's Disease</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Haizhong, Tan; Wei, Wu; Musheng, Xu; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8967-9004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Simultaneous inhibition of phosphodiesterase 5 (PDE5) and histone deacetylases (HDAC) has recently been validated as a potentially novel therapeutic approach for Alzheimer's Disease (AD).  To further extend this concept, the authors designed and synthesized the first chem. series of dual acting PDE5 and HDAC inhibitors, and the authors validated this systems therapeutics approach.  Following the implementation of structure- and knowledge-based approaches, initial hits were designed and were shown to validate the hypothesis of dual in vitro inhibition.  Then, an optimization strategy was pursued to obtain a proper tool compd. for in vivo testing in AD models.  Initial hits were translated into mols. with adequate cellular functional responses (histone acetylation and cAMP/cGMP response element-binding (CREB) phosphorylation in the nanomolar range), an acceptable therapeutic window (>1 log unit) and the ability to cross the blood-brain barrier, leading to the identification of 7 as a candidate for in vivo proof-of-concept testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbHBoJlefNEbVg90H21EOLACvtfcHk0ljk77pBC-hOKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL7O&md5=756a252809f05f5d7fc59e828736cfba</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00908%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2BDe%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520First-in-Class%2520Dual%2520Acting%2520Histone%2520Deacetylases%2520%2528HDACs%2529%2520and%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8967%26epage%3D9004%26doi%3D10.1021%2Facs.jmedchem.6b00908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilas-Zornoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santamarina, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez Perez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ordoñez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barchino, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Reversible DNA Methyltransferase and Lysine Methyltransferase G9a Inhibitors with Antitumoral in Vivo Efficacy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6518</span>– <span class="NLM_lpage">6545</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1altb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6518-6545&author=O.+Rabalauthor=E.+San+Jos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=J.+A.+S%C3%A1nchez-Ariasauthor=A.+Vilas-Zornozaauthor=A.+Ugarteauthor=I.+de+Miguelauthor=E.+Mirandaauthor=L.+Garateauthor=M.+Fragaauthor=P.+Santamarinaauthor=R.+Fernandez+Perezauthor=R.+Ordo%C3%B1ezauthor=E.+S%C3%A1ezauthor=S.+Roaauthor=M.+J.+Garc%C3%ADa-Barchinoauthor=J.+A.+Mart%C3%ADnez-Climentauthor=Y.+Liuauthor=W.+Wuauthor=M.+Xuauthor=F.+Prosperauthor=J.+Oyarzabal&title=Discovery+of+Reversible+DNA+Methyltransferase+and+Lysine+Methyltransferase+G9a+Inhibitors+with+Antitumoral+in+Vivo+Efficacy&doi=10.1021%2Facs.jmedchem.7b01926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Reversible DNA Methyltransferase and Lysine Methyltransferase G9a Inhibitors with Antitumoral in Vivo Efficacy</span></div><div class="casAuthors">Rabal, Obdulia; San Jose-Eneriz, Edurne; Agirre, Xabier; Sanchez-Arias, Juan Antonio; Vilas-Zornoza, Amaia; Ugarte, Ana; de Miguel, Irene; Miranda, Estibaliz; Garate, Leire; Fraga, Mario; Santamarina, Pablo; Fernandez Perez, Raul; Ordonez, Raquel; Saez, Elena; Roa, Sergio; Garcia-Barchino, Maria Jose; Martinez-Climent, Jose Angel; Liu, Yingying; Wu, Wei; Xu, Musheng; Prosper, Felipe; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6518-6545</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using knowledge- and structure-based approaches, we designed and synthesized reversible chem. probes that simultaneously inhibit the activity of two epigenetic targets, histone 3 lysine 9 methyltransferase (G9a) and DNA methyltransferases (DNMT), at nanomolar ranges.  Enzymic competition assays confirmed our design strategy: substrate competitive inhibitors.  Next, an initial exploration around our hit 11 was pursued to identify an adequate tool compd. for in vivo testing.  In vitro treatment of different hematol. neoplasia cell lines led to the identification of mols. with clear antiproliferative efficacies (GI50 values in the nanomolar range).  On the basis of epigenetic functional cellular responses (levels of lysine 9 methylation and 5-methylcytosine), an acceptable therapeutic window (around 1 log unit) and a suitable pharmacokinetic profile, 12 was selected for in vivo proof-of-concept.  Herein, 12 achieved a significant in vivo efficacy: 70% overall tumor growth inhibition of a human acute myeloid leukemia (AML) xenograft in a mouse model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf8aK3C-iah7Vg90H21EOLACvtfcHk0ljk77pBC-hOKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1altb%252FN&md5=41794ccc9d1b5662b796932b37b9efd3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01926%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DVilas-Zornoza%26aufirst%3DA.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DGarate%26aufirst%3DL.%26aulast%3DFraga%26aufirst%3DM.%26aulast%3DSantamarina%26aufirst%3DP.%26aulast%3DFernandez%2BPerez%26aufirst%3DR.%26aulast%3DOrdo%25C3%25B1ez%26aufirst%3DR.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DGarc%25C3%25ADa-Barchino%26aufirst%3DM.%2BJ.%26aulast%3DMart%25C3%25ADnez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DProsper%26aufirst%3DF.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Reversible%2520DNA%2520Methyltransferase%2520and%2520Lysine%2520Methyltransferase%2520G9a%2520Inhibitors%2520with%2520Antitumoral%2520in%2520Vivo%2520Efficacy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6518%26epage%3D6545%26doi%3D10.1021%2Facs.jmedchem.7b01926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San José-Enériz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agirre, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garate, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Climent, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prosper, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6546</span>– <span class="NLM_lpage">6573</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFSrtL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6546-6573&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=E.+San+Jos%C3%A9-En%C3%A9rizauthor=X.+Agirreauthor=I.+de+Miguelauthor=L.+Garateauthor=E.+Mirandaauthor=E.+S%C3%A1ezauthor=S.+Roaauthor=J.+A.+Mart%C3%ADnez-Climentauthor=Y.+Liuauthor=W.+Wuauthor=M.+Xuauthor=F.+Prosperauthor=J.+Oyarzabal&title=Detailed+Exploration+around+4-Aminoquinolines+Chemical+Space+to+Navigate+the+Lysine+Methyltransferase+G9a+and+DNA+Methyltransferase+Biological+Spaces&doi=10.1021%2Facs.jmedchem.7b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Detailed Exploration around 4-Aminoquinolines Chemical Space to Navigate the Lysine Methyltransferase G9a and DNA Methyltransferase Biological Spaces</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan Antonio; San Jose-Eneriz, Edurne; Agirre, Xabier; de Miguel, Irene; Garate, Leire; Miranda, Estibaliz; Saez, Elena; Roa, Sergio; Martinez-Climent, Jose Angel; Liu, Yingying; Wu, Wei; Xu, Musheng; Prosper, Felipe; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6546-6573</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic regulators that exhibit aberrant enzymic activities or expression profiles are potential therapeutic targets for cancers.  Specifically, enzymes responsible for methylation at histone-3 lysine-9 (like G9a) and aberrant DNA hypermethylation (DNMTs) have been implicated in a no. of cancers.  Recently, mols. bearing a 4-aminoquinoline scaffold were reported as dual inhibitors of these targets and showed a significant in-vivo efficacy in animal models of hematol. malignancies.  Here, we report a detailed exploration around three growing vectors born by this chemotype.  Exploring this chem. space led to the identification of features to navigate G9a and DNMT1 biol. spaces; not only their corresponding exclusive areas, selective compds., but also common spaces.  Thus, we identified from selective G9a and first-in-class DNMT1 inhibitors, > 1 log unit between their IC50 values, with IC50 < 25nM (e.g. 43 and 26, resp.) to equipotent inhibitors with IC50 < 50nM for both targets (e.g. 13).  Their ADME/Tox profiling and antiproliferative efficacies, vs. some cancer cell lines, are also reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqysB6axnTb-LVg90H21EOLACvtfcHk0liete4dXFrJHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFSrtL3K&md5=b2579edfb0bb6a4bb9c104d30fd9a28a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01925%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DSan%2BJos%25C3%25A9-En%25C3%25A9riz%26aufirst%3DE.%26aulast%3DAgirre%26aufirst%3DX.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DGarate%26aufirst%3DL.%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DRoa%26aufirst%3DS.%26aulast%3DMart%25C3%25ADnez-Climent%26aufirst%3DJ.%2BA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DProsper%26aufirst%3DF.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDetailed%2520Exploration%2520around%25204-Aminoquinolines%2520Chemical%2520Space%2520to%2520Navigate%2520the%2520Lysine%2520Methyltransferase%2520G9a%2520and%2520DNA%2520Methyltransferase%2520Biological%2520Spaces%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6546%26epage%3D6573%26doi%3D10.1021%2Facs.jmedchem.7b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishibe-Murakami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Mechanistic Insights into DNMT1-Mediated Maintenance DNA Methylation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">709</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1126/science.1214453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1126%2Fscience.1214453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=22323818" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=2012&pages=709-712&author=J.+Songauthor=M.+Teplovaauthor=S.+Ishibe-Murakamiauthor=D.+J.+Patel&title=Structure-Based+Mechanistic+Insights+into+DNMT1-Mediated+Maintenance+DNA+Methylation&doi=10.1126%2Fscience.1214453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Mechanistic Insights into DNMT1-Mediated Maintenance DNA Methylation</span></div><div class="casAuthors">Song, Jikui; Teplova, Marianna; Ishibe-Murakami, Satoko; Patel, Dinshaw J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">6069</span>),
    <span class="NLM_cas:pages">709-712</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">DNMT1, the major maintenance DNA methyltransferase in animals, helps to regulate gene expression, genome imprinting, and X-chromosome inactivation.  We report on the crystal structure of a productive covalent mouse DNMT1(731-1602)-DNA complex contg. a central hemimethylated CpG site.  The Me group of methylcytosine is positioned within a shallow hydrophobic concave surface, whereas the cytosine on the target strand is looped out and covalently anchored within the catalytic pocket.  The DNA is distorted at the hemimethylated CpG step, with side chains from catalytic and recognition loops inserting through both grooves to fill an intercalation-type cavity assocd. with a dual base flip-out on partner strands.  Structural and biochem. data establish how a combination of active and autoinhibitory mechanisms ensures the high fidelity of DNMT1-mediated maintenance DNA methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOZ2gvW2FsqbVg90H21EOLACvtfcHk0liWJa-En8XEDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjsrY%253D&md5=d2f1071d7ef7d59efcff6e59b1c215d0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.1214453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1214453%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DTeplova%26aufirst%3DM.%26aulast%3DIshibe-Murakami%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26atitle%3DStructure-Based%2520Mechanistic%2520Insights%2520into%2520DNMT1-Mediated%2520Maintenance%2520DNA%2520Methylation%26jtitle%3DScience%26date%3D2012%26volume%3D335%26spage%3D709%26epage%3D712%26doi%3D10.1126%2Fscience.1214453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Millard, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Celardo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordiyenko, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairall, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwabe, J. W. R.</span></span> <span> </span><span class="NLM_article-title">Class I HDACs Share a Common Mechanism of Regulation by Inositol Phosphates</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2013.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.molcel.2013.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=23791785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVygu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2013&pages=57-67&author=C.+J.+Millardauthor=P.+J.+Watsonauthor=I.+Celardoauthor=Y.+Gordiyenkoauthor=S.+M.+Cowleyauthor=C.+V.+Robinsonauthor=L.+Fairallauthor=J.+W.+R.+Schwabe&title=Class+I+HDACs+Share+a+Common+Mechanism+of+Regulation+by+Inositol+Phosphates&doi=10.1016%2Fj.molcel.2013.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Class I HDACs Share a Common Mechanism of Regulation by Inositol Phosphates</span></div><div class="casAuthors">Millard, Christopher J.; Watson, Peter J.; Celardo, Ivana; Gordiyenko, Yuliya; Cowley, Shaun M.; Robinson, Carol V.; Fairall, Louise; Schwabe, John W. R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-67</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Class I histone deacetylases (HDAC1, HDAC2, and HDAC3) are recruited by cognate corepressor proteins into specific transcriptional repression complexes that target HDAC activity to chromatin resulting in chromatin condensation and transcriptional silencing.  The authors previously reported the structure of HDAC3 in complex with the SMRT corepressor.  This structure revealed inositol-tetraphosphate [Ins(1,4,5,6)P4] at the interface of the two proteins.  It was previously unclear whether the role of Ins(1,4,5,6)P4 is to act as a structural cofactor or a regulator of HDAC3 activity.  Here the authors report the structure of HDAC1 in complex with MTA1 from the NuRD complex.  The ELM2-SANT domains from MTA1 wrap completely around HDAC1 occupying both sides of the active site such that the adjacent BAH domain is ideally positioned to recruit nucleosomes to the active site of the enzyme.  Functional assays of both the HDAC1 and HDAC3 complexes reveal that Ins(1,4,5,6)P4 is a bona fide conserved regulator of class I HDAC complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYPhL0103LkLVg90H21EOLACvtfcHk0liWJa-En8XEDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVygu74%253D&md5=05a4267bf965b5a6217873217044ef2a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DMillard%26aufirst%3DC.%2BJ.%26aulast%3DWatson%26aufirst%3DP.%2BJ.%26aulast%3DCelardo%26aufirst%3DI.%26aulast%3DGordiyenko%26aufirst%3DY.%26aulast%3DCowley%26aufirst%3DS.%2BM.%26aulast%3DRobinson%26aufirst%3DC.%2BV.%26aulast%3DFairall%26aufirst%3DL.%26aulast%3DSchwabe%26aufirst%3DJ.%2BW.%2BR.%26atitle%3DClass%2520I%2520HDACs%2520Share%2520a%2520Common%2520Mechanism%2520of%2520Regulation%2520by%2520Inositol%2520Phosphates%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D51%26spage%3D57%26epage%3D67%26doi%3D10.1016%2Fj.molcel.2013.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase 6 Structure and Molecular Basis of Catalysis and Inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fnchembio.2134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=27454933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=741-747&author=Y.+Haiauthor=D.+W.+Christianson&title=Histone+Deacetylase+6+Structure+and+Molecular+Basis+of+Catalysis+and+Inhibition&doi=10.1038%2Fnchembio.2134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span></div><div class="casAuthors">Hai, Yang; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">741-747</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a crit. target for drug design because of its role in oncogenic transformation and cancer metastasis, and is unique among all histone deacetylases in that it contains tandem catalytic domains designated CD1 and CD2.  We now report the crystal structures of CD2 from Homo sapiens HDAC6 and of CD1 and CD2 from Danio rerio HDAC6.  We correlated these structures with activity measurements using 13 different substrates.  The catalytic activity of CD2 from both species exhibited broad substrate specificity, whereas that of CD1 was highly specific for substrates bearing C-terminal acetyllysine residues.  Crystal structures of substrate complexes yielded unprecedented snapshots of the catalytic mechanism.  Addnl., crystal structures of complexes with eight different inhibitors, including belinostat and panobinostat (currently used in cancer chemotherapy), the macrocyclic tetrapeptide HC toxin, and the HDAC6-specific inhibitor N-hydroxy-4-(2-((2-hydroxyethyl)(phenyl)amino)-2-oxoethyl)benzamide, revealed surprising new insight regarding changes in Zn2+ coordination and isoenzyme-specific inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCfziGBHBRLVg90H21EOLACvtfcHk0liWJa-En8XEDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ&md5=e167bc6f66e00923da4f2bced2b306b9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2134%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520Deacetylase%25206%2520Structure%2520and%2520Molecular%2520Basis%2520of%2520Catalysis%2520and%2520Inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D741%26epage%3D747%26doi%3D10.1038%2Fnchembio.2134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, Biological Evaluation and in Vivo Testing of Dual Phosphodiesterase 5 (PDE5) and Histone Deacetylase 6 (HDAC6)-Selective Inhibitors for the Treatment of Alzheimer’s Disease</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1016%2Fj.ejmech.2018.03.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=29549837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslaiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2018&pages=506-524&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+Biological+Evaluation+and+in+Vivo+Testing+of+Dual+Phosphodiesterase+5+%28PDE5%29+and+Histone+Deacetylase+6+%28HDAC6%29-Selective+Inhibitors+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease&doi=10.1016%2Fj.ejmech.2018.03.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, biological evaluation and in vivo testing of dual phosphodiesterase 5 (PDE5) and histone deacetylase 6 (HDAC6)-selective inhibitors for the treatment of Alzheimer's disease</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Tan, Haizhong; Wu, Wei; Xu, Musheng; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">506-524</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The authors have identified chem. probes that act as dual phosphodiesterase 5 (PDE5) and histone deacetylase 6 (HDAC6)-selective inhibitors (>1 log unit difference vs. class I HDACs) to decipher the contribution of HDAC isoforms to the pos. impact of dual-acting PDE5 and HDAC inhibitors on mouse models of Alzheimer's disease (AD) and fine-tune this systems therapeutics approach.  Structure- and knowledge-based approaches led to the design of first-in-class mols. with the desired target compd. profile: dual PDE5 and HDAC6-selective inhibitors.  Compd. 44b (5-[[4-ethoxy-3-(1-methyl-7-oxo-3-propyl-6H-pyrazolo[4,3-d]pyrimidin-5-yl)phenyl]methyl]thiophene-2-carbohydroxamic acid), which fulfilled the biochem., functional and ADME-Tox profiling requirements and exhibited adequate pharmacokinetic properties, was selected as pharmacol. tool compd. and tested in a mouse model of AD (Tg2576) in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFqq9VoRla3bVg90H21EOLACvtfcHk0lh9OPEM2TaFjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslaiur4%253D&md5=1ced6634343dd8c0913ea3e6be07ff91</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.005%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520Biological%2520Evaluation%2520and%2520in%2520Vivo%2520Testing%2520of%2520Dual%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520and%2520Histone%2520Deacetylase%25206%2520%2528HDAC6%2529-Selective%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D150%26spage%3D506%26epage%3D524%26doi%3D10.1016%2Fj.ejmech.2018.03.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haizhong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of in Vivo Chemical Probes for Treating Alzheimer’s Disease: Dual Phosphodiesterase 5 (PDE5) and Class I Histone Deacetylase Selective Inhibitors</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1765</span>– <span class="NLM_lpage">1782</span>, <span class="refDoi"> DOI: 10.1021/acschemneuro.8b00648</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.8b00648" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1765-1782&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=T.+Haizhongauthor=W.+Weiauthor=X.+Mushengauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Discovery+of+in+Vivo+Chemical+Probes+for+Treating+Alzheimer%E2%80%99s+Disease%3A+Dual+Phosphodiesterase+5+%28PDE5%29+and+Class+I+Histone+Deacetylase+Selective+Inhibitors&doi=10.1021%2Facschemneuro.8b00648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of in Vivo Chemical Probes for Treating Alzheimer's Disease: Dual Phosphodiesterase 5 (PDE5) and Class I Histone Deacetylase Selective Inhibitors</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Haizhong, Tan; Wei, Wu; Musheng, Xu; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1765-1782</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In order to det. the contributions of histone deacetylase (HDAC) isoforms to the beneficial effects of dual phosphodiesterase 5 (PDE5) and pan-HDAC inhibitors on in vivo models of Alzheimer's disease (AD), we have designed, synthesized, and tested novel chem. probes with the desired target compd. profile of PDE5 and class I HDAC selective inhibitors.  Compared to previous hydroxamate-based series, these mols. exhibit longer residence times on HDACs.  In this scenario, shorter or longer preincubation times may have a significant impact on the IC50 values of these compds. and therefore on their corresponding selectivity profiles on the different HDAC isoforms.  On the other hand, different chem. series have been explored and, as expected, some pairwise comparisons show a clear impact of the scaffold on biol. responses.  The lead identification process led to compd. I, which shows an adequate ADME-Tox profile and in vivo target engagement (histone acetylation and cAMP/cGMP response element-binding (CREB) phosphorylation) in the central nervous system (CNS), suggesting that this compd. represents an optimized chem. probe; thus, I has been assayed in a mouse model of AD (Tg2576).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpMx-DcLUAMrVg90H21EOLACvtfcHk0lh2nitClB-Dcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLvF&md5=d3d13a9121111b166ea69c09106eb3c5</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.8b00648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.8b00648%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DHaizhong%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMusheng%26aufirst%3DX.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520in%2520Vivo%2520Chemical%2520Probes%2520for%2520Treating%2520Alzheimer%25E2%2580%2599s%2520Disease%253A%2520Dual%2520Phosphodiesterase%25205%2520%2528PDE5%2529%2520and%2520Class%2520I%2520Histone%2520Deacetylase%2520Selective%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2019%26volume%3D10%26spage%3D1765%26epage%3D1782%26doi%3D10.1021%2Facschemneuro.8b00648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuadrado-Tejedor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miguel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-González, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Barroso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estella-Hermoso
de Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sáez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espelosin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursua, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineda-Lucena, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Osta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Multitarget Approach for the Treatment of Alzheimer’s Disease: Inhibition of Phosphodiesterase 9 (PDE9) and Histone Deacetylases (HDACs) Covering Diverse Selectivity Profiles</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">4076</span>– <span class="NLM_lpage">4101</span>, <span class="refDoi"> DOI: 10.1021/acschemneuro.9b00303</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschemneuro.9b00303" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1egu7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=4076-4101&author=O.+Rabalauthor=J.+A.+S%C3%A1nchez-Ariasauthor=M.+Cuadrado-Tejedorauthor=I.+de+Miguelauthor=M.+P%C3%A9rez-Gonz%C3%A1lezauthor=C.+Garc%C3%ADa-Barrosoauthor=A.+Ugarteauthor=A.+Estella-Hermoso%0Ade+Mendozaauthor=E.+S%C3%A1ezauthor=M.+Espelosinauthor=S.+Ursuaauthor=H.+Tanauthor=W.+Wuauthor=M.+Xuauthor=A.+Pineda-Lucenaauthor=A.+Garcia-Ostaauthor=J.+Oyarzabal&title=Multitarget+Approach+for+the+Treatment+of+Alzheimer%E2%80%99s+Disease%3A+Inhibition+of+Phosphodiesterase+9+%28PDE9%29+and+Histone+Deacetylases+%28HDACs%29+Covering+Diverse+Selectivity+Profiles&doi=10.1021%2Facschemneuro.9b00303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Multitarget Approach for the Treatment of Alzheimer's Disease: Inhibition of Phosphodiesterase 9 (PDE9) and Histone Deacetylases (HDACs) Covering Diverse Selectivity Profiles</span></div><div class="casAuthors">Rabal, Obdulia; Sanchez-Arias, Juan A.; Cuadrado-Tejedor, Mar; de Miguel, Irene; Perez-Gonzalez, Marta; Garcia-Barroso, Carolina; Ugarte, Ana; Estella-Hermoso de Mendoza, Ander; Saez, Elena; Espelosin, Maria; Ursua, Susana; Tan, Haizhong; Wu, Wei; Xu, Musheng; Pineda-Lucena, Antonio; Garcia-Osta, Ana; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4076-4101</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present a series of dual-target phosphodiesterase 9 (PDE9) and histone deacetylase (HDAC) inhibitors devised as pharmacol. tool compds. for assessing the implications of these two targets in Alzheimer's disease (AD).  These novel inhibitors were designed taking into account the key pharmacophoric features of known selective PDE9 inhibitors as well as privileged chem. structures, bearing zinc binding groups (hydroxamic acids and ortho-amino anilides) that hit HDAC targets.  These substituents were selected according to rational criteria and previous knowledge from our group to explore diverse HDAC selectivity profiles (pan-HDAC, HDAC6 selective, and class I selective) that were confirmed in biochem. screens.  Their functional response in inducing acetylation of histone and tubulin and phosphorylation of cAMP response element binding (CREB) was measured as a requisite for further progression into complete in vitro absorption, distribution, metab. and excretion (ADME) and in vivo brain penetration profiling.  Compd. 31b, a selective HDAC6 inhibitor with acceptable brain permeability, was chosen for assessing in vivo efficacy of these first-in-class inhibitors, as well as studying their mode of action (MoA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9GkgLW75qELVg90H21EOLACvtfcHk0lh2nitClB-Dcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1egu7rF&md5=be57148d4b99a7e517c51142a1908fb8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facschemneuro.9b00303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschemneuro.9b00303%26sid%3Dliteratum%253Aachs%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DCuadrado-Tejedor%26aufirst%3DM.%26aulast%3Dde%2BMiguel%26aufirst%3DI.%26aulast%3DP%25C3%25A9rez-Gonz%25C3%25A1lez%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Barroso%26aufirst%3DC.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DEstella-Hermoso%2Bde%2BMendoza%26aufirst%3DA.%26aulast%3DS%25C3%25A1ez%26aufirst%3DE.%26aulast%3DEspelosin%26aufirst%3DM.%26aulast%3DUrsua%26aufirst%3DS.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DPineda-Lucena%26aufirst%3DA.%26aulast%3DGarcia-Osta%26aufirst%3DA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DMultitarget%2520Approach%2520for%2520the%2520Treatment%2520of%2520Alzheimer%25E2%2580%2599s%2520Disease%253A%2520Inhibition%2520of%2520Phosphodiesterase%25209%2520%2528PDE9%2529%2520and%2520Histone%2520Deacetylases%2520%2528HDACs%2529%2520Covering%2520Diverse%2520Selectivity%2520Profiles%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2019%26volume%3D10%26spage%3D4076%26epage%3D4101%26doi%3D10.1021%2Facschemneuro.9b00303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagadeeswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterling, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span> <span> </span><span class="NLM_article-title">The Histone Methyltransferase Inhibitor A-366 Uncovers a Role for G9a/GLP in the Epigenetics of Leukemia</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0131716</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0131716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1371%2Fjournal.pone.0131716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=26147105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWktL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=W.+N.+Pappanoauthor=J.+Guoauthor=Y.+Heauthor=D.+Fergusonauthor=S.+Jagadeeswaranauthor=D.+J.+Osterlingauthor=W.+Gaoauthor=J.+K.+Spenceauthor=M.+Pliushchevauthor=R.+F.+Sweisauthor=F.+G.+Buchananauthor=M.+R.+Michaelidesauthor=A.+R.+Shoemakerauthor=C.+Tseauthor=G.+G.+Chiang&title=The+Histone+Methyltransferase+Inhibitor+A-366+Uncovers+a+Role+for+G9a%2FGLP+in+the+Epigenetics+of+Leukemia&doi=10.1371%2Fjournal.pone.0131716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The histone methyltransferase inhibitor A-366 uncovers a role for G9a/GLP in the epigenetics of leukemia</span></div><div class="casAuthors">Pappano, William N.; Guo, Jun; He, Yupeng; Ferguson, Debra; Jagadeeswaran, Sujatha; Osterling, Donald J.; Gao, Wenqing; Spence, Julie K.; Pliushchev, Marina; Sweis, Ramzi F.; Buchanan, Fritz G.; Michaelides, Michael R.; Shoemaker, Alexander R.; Tse, Chris; Chiang, Gary G.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0131716/1-e0131716/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Histone methyltransferases are epigenetic regulators that modify key lysine and arginine residues on histones and are believed to play an important role in cancer development and maintenance.  These epigenetic modifications are potentially reversible and as a result this class of enzymes has drawn great interest as potential therapeutic targets of small mol. inhibitors.  Previous studies have suggested that the histone lysine methyltransferase G9a (EHMT2) is required to perpetuate malignant phenotypes through multiple mechanisms in a variety of cancer types.  To further elucidate the enzymic role of G9a in cancer, we describe herein the biol. activities of a novel peptide-competitive histone methyltransferase inhibitor, A-366, that selectively inhibits G9a and the closely related GLP (EHMT1), but not other histone methyltransferases.  A-366 has significantly less cytotoxic effects on the growth of tumor cell lines compared to other known G9a/GLP small mol. inhibitors despite equiv. cellular activity on methylation of H3K9me2.  Addnl., the selectivity profile of A-366 has aided in the discovery of a potentially important role for G9a/GLP in maintenance of leukemia.  Treatment of various leukemia cell lines in vitro resulted in marked differentiation and morphol. changes of these tumor cell lines.  Furthermore, treatment of a flank xenograft leukemia model with A-366 resulted in growth inhibition in vivo consistent with the profile of H3K9me2 redn. obsd.  In summary, A-366 is a novel and highly selective inhibitor of G9a/GLP that has enabled the discovery of a role for G9a/GLP enzymic activity in the growth and differentiation status of leukemia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0f7uh1A8GjrVg90H21EOLACvtfcHk0litoYR7oaueHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWktL7M&md5=33620fc152d0df292c5259ff4f6cf040</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0131716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0131716%26sid%3Dliteratum%253Aachs%26aulast%3DPappano%26aufirst%3DW.%2BN.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DFerguson%26aufirst%3DD.%26aulast%3DJagadeeswaran%26aufirst%3DS.%26aulast%3DOsterling%26aufirst%3DD.%2BJ.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DSpence%26aufirst%3DJ.%2BK.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DShoemaker%26aufirst%3DA.%2BR.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26atitle%3DThe%2520Histone%2520Methyltransferase%2520Inhibitor%2520A-366%2520Uncovers%2520a%2520Role%2520for%2520G9a%252FGLP%2520in%2520the%2520Epigenetics%2520of%2520Leukemia%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0131716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, H.-c.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badeti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fritzky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.-g.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarrera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, W.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span> <span> </span><span class="NLM_article-title">Efficacy of anti-CD147 chimeric antigen receptors targeting hepatocellular carcinoma</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4810</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18444-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=10.1038%2Fs41467-020-18444-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=32968061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFOhsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4810&author=H.-c.+Tsengauthor=W.+Xiongauthor=S.+Badetiauthor=Y.+Yangauthor=M.+Maauthor=T.+Liuauthor=C.+A.+Ramosauthor=G.+Dottiauthor=L.+Fritzkyauthor=J.-g.+Jiangauthor=Q.+Yiauthor=J.+Guarreraauthor=W.-X.+Zongauthor=C.+Liuauthor=D.+Liu&title=Efficacy+of+anti-CD147+chimeric+antigen+receptors+targeting+hepatocellular+carcinoma&doi=10.1038%2Fs41467-020-18444-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of anti-CD147 chimeric antigen receptors targeting hepatocellular carcinoma</span></div><div class="casAuthors">Tseng, Hsiang-chi; Xiong, Wei; Badeti, Saiaditya; Yang, Yan; Ma, Minh; Liu, Ting; Ramos, Carlos A.; Dotti, Gianpietro; Fritzky, Luke; Jiang, Jie-gen; Yi, Qing; Guarrera, James; Zong, Wei-Xing; Liu, Chen; Liu, Dongfang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4810</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Chimeric antigen receptor (CAR) therapy is a promising immunotherapeutic strategy for treating multiple refractory blood cancers, but further advances are required for solid tumor CAR therapy.  One challenge is identifying a safe and effective tumor antigen.  Here, we devise a strategy for targeting hepatocellular carcinoma (HCC, one of the deadliest malignancies).  We report that T and NK cells transduced with a CAR that recognizes the surface marker, CD147, also known as Basigin, can effectively kill various malignant HCC cell lines in vitro, and HCC tumors in xenograft and patient-derived xenograft mouse models.  To minimize any on-target/off-tumor toxicity, we use logic-gated (log) GPC3-synNotch-inducible CD147-CAR to target HCC.  LogCD147-CAR selectively kills dual antigen (GPC3+CD147+), but not single antigen (GPC3-CD147+) pos. HCC cells and does not cause severe on-target/off-tumor toxicity in a human CD147 transgenic mouse model.  In conclusion, these findings support the therapeutic potential of CD147-CAR-modified immune cells for HCC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozkNcgFGZiSrVg90H21EOLACvtfcHk0lg8mI9t02HhsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFOhsrbE&md5=9d14cd972275fe26618595303401e4a0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18444-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18444-2%26sid%3Dliteratum%253Aachs%26aulast%3DTseng%26aufirst%3DH.-c.%26aulast%3DXiong%26aufirst%3DW.%26aulast%3DBadeti%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DRamos%26aufirst%3DC.%2BA.%26aulast%3DDotti%26aufirst%3DG.%26aulast%3DFritzky%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DJ.-g.%26aulast%3DYi%26aufirst%3DQ.%26aulast%3DGuarrera%26aufirst%3DJ.%26aulast%3DZong%26aufirst%3DW.-X.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DEfficacy%2520of%2520anti-CD147%2520chimeric%2520antigen%2520receptors%2520targeting%2520hepatocellular%2520carcinoma%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4810%26doi%3D10.1038%2Fs41467-020-18444-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1592</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1577-1592&author=Y.+Chenauthor=X.+Yuanauthor=W.+Zhangauthor=M.+Tangauthor=L.+Zhengauthor=F.+Wangauthor=W.+Yanauthor=S.+Yangauthor=Y.+Weiauthor=J.+Heauthor=L.+Chen&title=Discovery+of+Novel+Dual+Histone+Deacetylase+and+Mammalian+Target+of+Rapamycin+Target+Inhibitors+as+a+Promising+Strategy+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.8b01825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span></div><div class="casAuthors">Chen, Yong; Yuan, Xue; Zhang, Wanhua; Tang, Minghai; Zheng, Li; Wang, Fang; Yan, Wei; Yang, Shengyong; Wei, Yuquan; He, Jun; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1577-1592</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present study, a series of novel dual-target histone deacetylase (HDAC) and mammalian target of rapamycin (mTOR) inhibitors were designed and synthesized using pyrimidine-pyrazolyl pharmacophore to append HDAC recognition cap and hydroxamic acid as a zinc-binding motif.  Among them, I was the optimal lead compd. with potent inhibition activities against mTOR and HDAC1 with half-maximal inhibitory concn. of 1.2 and 0.19 nM, resp.  Western blot confirmed that I could upregulate acetylation of H3 and α-tubulin and downregulate mTOR-related downstream mediators.  I could also stimulate cell cycle arrest in G0/G1 phase and induce tumor cell apoptosis.  I showed comparable antitumor activity with the combination medication in MM1S xenograft model with a tumor growth inhibitory rate of 72.5%, without causing significant loss of body wt. and toxicity.  All of the results indicated that I could be a promising dual target inhibitor for treating hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ3AdehNv1ObVg90H21EOLACvtfcHk0lg8mI9t02HhsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D&md5=7a2ff5020aedb0e3c302e65d993301e6</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01825%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520Novel%2520Dual%2520Histone%2520Deacetylase%2520and%2520Mammalian%2520Target%2520of%2520Rapamycin%2520Target%2520Inhibitors%2520as%2520a%2520Promising%2520Strategy%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1577%26epage%3D1592%26doi%3D10.1021%2Facs.jmedchem.8b01825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre Ena, X.</span>; <span class="NLM_string-name">Oyarzabal Santamarina, J.</span>; <span class="NLM_string-name">Prosper Cardoso, F.</span>; <span class="NLM_string-name">Rabal Gracia, M. O.</span>; <span class="NLM_string-name">Rodriguez Madoz, J. R.</span>; <span class="NLM_string-name">San Jose Eneriz, E.</span></span> <span> </span><span class="NLM_article-title">Novel compounds as dual inhibitors of histone methyltransferases and dna methyltransferases</span>. <span class="NLM_patent">WO 2015192981 A1</span>, March 30, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=X.+Aguirre+Ena&author=J.+Oyarzabal+Santamarina&author=F.+Prosper+Cardoso&author=M.+O.+Rabal+Gracia&author=J.+R.+Rodriguez+Madoz&author=E.+San+Jose+Eneriz&title=Novel+compounds+as+dual+inhibitors+of+histone+methyltransferases+and+dna+methyltransferases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAguirre%2BEna%26aufirst%3DX.%26atitle%3DNovel%2520compounds%2520as%2520dual%2520inhibitors%2520of%2520histone%2520methyltransferases%2520and%2520dna%2520methyltransferases%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aguirre Ena, X.</span>; <span class="NLM_string-name">Oyarzabal Santamarina, J.</span>; <span class="NLM_string-name">Prosper Cardoso, F.</span>; <span class="NLM_string-name">Rabal Gracia, M. O.</span>; <span class="NLM_string-name">San Jose Eneriz, E.</span>; <span class="NLM_string-name">Sanchez Arias, J. A.</span>; <span class="NLM_string-name">Vilas Zornoza, A.</span></span> <span> </span><span class="NLM_article-title">Novel compounds for use in cancer</span>. <span class="NLM_patent">WO 2018229139 A1</span>, June 13, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=X.+Aguirre+Ena&author=J.+Oyarzabal+Santamarina&author=F.+Prosper+Cardoso&author=M.+O.+Rabal+Gracia&author=E.+San+Jose+Eneriz&author=J.+A.+Sanchez+Arias&author=A.+Vilas+Zornoza&title=Novel+compounds+for+use+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAguirre%2BEna%26aufirst%3DX.%26atitle%3DNovel%2520compounds%2520for%2520use%2520in%2520cancer%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orbe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sánchez-Arias, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabal, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belzunce, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Páramo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyarzabal, J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of Novel Matrix Metalloproteinase Inhibitors As Potent Antihemorrhagic Agents: From Hit Identification to an Optimized Lead</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2488</span>, <span class="refDoi"> DOI: 10.1021/jm501940y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501940y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislGiurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2465-2488&author=J.+Orbeauthor=J.+A.+S%C3%A1nchez-Ariasauthor=O.+Rabalauthor=J.+A.+Rodr%C3%ADguezauthor=A.+Salicioauthor=A.+Ugarteauthor=M.+Belzunceauthor=M.+Xuauthor=W.+Wuauthor=H.+Tanauthor=H.+Maauthor=J.+A.+P%C3%A1ramoauthor=J.+Oyarzabal&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+Novel+Matrix+Metalloproteinase+Inhibitors+As+Potent+Antihemorrhagic+Agents%3A+From+Hit+Identification+to+an+Optimized+Lead&doi=10.1021%2Fjm501940y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of Novel Matrix Metalloproteinase Inhibitors As Potent Antihemorrhagic Agents: From Hit Identification to an Optimized Lead</span></div><div class="casAuthors">Orbe, Josune; Sanchez-Arias, Juan A.; Rabal, Obdulia; Rodriguez, Jose A.; Salicio, Agustina; Ugarte, Ana; Belzunce, Miriam; Xu, Musheng; Wu, Wei; Tan, Haizhong; Ma, Hongyu; Paramo, Jose A.; Oyarzabal, Julen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2465-2488</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Growing evidence suggests that matrix metalloproteinases (MMP) are involved in thrombus dissoln.; then, considering that new therapeutic strategies are required for controlling hemorrhage, we hypothesized that MMP inhibition may reduce bleeding by delaying fibrinolysis.  Thus, we designed and synthesized a novel series of MMP inhibitors to identify potential candidates for acute treatment of bleeding.  Structure-based and knowledge-based strategies were utilized to design this novel chem. series, α-spiropiperidine hydroxamates, of potent and sol. (>75 μg/mL) pan-MMP inhibitors.  The initial hit was progressed to an optimal lead I.  Racemic I showed a remarkable in vitro phenotypic response and outstanding in vivo efficacy; in fact, the mouse bleeding time at 1 mg/kg was 0.85 min compared to 29.28 min using saline.  In addn., I displayed an optimal ADME and safety profile (e.g., no thrombus formation).  Its corresponding enantiomers were sepd., leading to the preclin. candidate (R)-I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGretnPXVi3YibVg90H21EOLACvtfcHk0ljP-HOWXz1xkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislGiurk%253D&md5=1d56002442eae50c83de14d1f217701b</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm501940y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501940y%26sid%3Dliteratum%253Aachs%26aulast%3DOrbe%26aufirst%3DJ.%26aulast%3DS%25C3%25A1nchez-Arias%26aufirst%3DJ.%2BA.%26aulast%3DRabal%26aufirst%3DO.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DJ.%2BA.%26aulast%3DSalicio%26aufirst%3DA.%26aulast%3DUgarte%26aufirst%3DA.%26aulast%3DBelzunce%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DP%25C3%25A1ramo%26aufirst%3DJ.%2BA.%26aulast%3DOyarzabal%26aufirst%3DJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520Novel%2520Matrix%2520Metalloproteinase%2520Inhibitors%2520As%2520Potent%2520Antihemorrhagic%2520Agents%253A%2520From%2520Hit%2520Identification%2520to%2520an%2520Optimized%2520Lead%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2465%26epage%3D2488%26doi%3D10.1021%2Fjm501940y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0li5jYSZ9imsRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Accelrys Software
Inc.</span> <i>Pipeline Pilot</i>, Version
9.5: <span class="NLM_publisher-loc">San Diego, CA</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Accelrys+Software%0AInc.&title=Pipeline+Pilot"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DPipeline%2520Pilot%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW','PDB','3RJW'); return false;">PDB: 3RJW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4','PDB','4DA4'); return false;">PDB: 4DA4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKX','PDB','4BKX'); return false;">PDB: 4BKX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU','PDB','5EDU'); return false;">PDB: 5EDU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ','PDB','4LXZ'); return false;">PDB: 4LXZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EF8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EF8','PDB','5EF8'); return false;">PDB: 5EF8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSH','PDB','6HSH'); return false;">PDB: 6HSH</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i180"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02255">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34672"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02255?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02255</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Protocols for preparative HPLC purification methods; method for high-resolution mass spectrometry (HRMS) of the final compounds; methods for analytical HPLC and UHPLC; HPLC or UHPLC traces of the final compounds; NMR spectra (<sup>1</sup>H & <sup>13</sup>C) of the final compounds; biochemical profiling of <b>9a</b> and <b>12a</b><i>versus</i> DNMT and HDAC isoforms; H3K9me2 and H3Ac marks after treatment with A-366, Panobinostat, and Decitabine in MM1.S cells; hypomethylating activity of compounds <b>9a</b> and <b>12a</b> in JJN3 cells; body weight of mice treated with compound <b>12a</b>; and plasmatic concentrations of <b>12a</b> after administration (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings together with their IC<sub>50</sub> values  (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_002.csv">CSV</a>)</p></li><li><p class="inline">Results from docking studies reported in Figure 2:</p></li><li><p class="inline">G9a compound <b>9a</b> complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_003.pdb">PDB</a>)</p></li><li><p class="inline">DNMT1 compound <b>9a</b> complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_004.pdb">PDB</a>)</p></li><li><p class="inline">HDAC1 compound <b>9a</b> complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_005.pdb">PDB</a>)</p></li><li><p class="inline">HDAC6 compound <b>9a</b> complex (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_006.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_001.pdf">jm0c02255_si_001.pdf (3.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_002.csv">jm0c02255_si_002.csv (3.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_003.pdb">jm0c02255_si_003.pdb (349.51 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_004.pdb">jm0c02255_si_004.pdb (1.05 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_005.pdb">jm0c02255_si_005.pdb (253.74 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02255/suppl_file/jm0c02255_si_006.pdb">jm0c02255_si_006.pdb (233.4 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB ID Codes: <b>1</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>; <b>2</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EF8">5EF8</a>; <b>3</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSH">6HSH</a>; G9a, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RJW">3RJW</a>; DNMT1, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4DA4">4DA4</a>; HDAC1, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BKX">4BKX</a>; HDAC6, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02255&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02255%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-6%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02255" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799113abb99123e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
