
<html lang="en"     class="pb-page"  data-request-id="4e58aac7-c6c7-4ea2-aae6-1d1c90c2ffe9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.1c00412;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit" /></meta><meta name="dc.Creator" content="Francesco  Rianjongdee" /></meta><meta name="dc.Creator" content="Stephen J.  Atkinson" /></meta><meta name="dc.Creator" content="Chun-wa  Chung" /></meta><meta name="dc.Creator" content="Paola  Grandi" /></meta><meta name="dc.Creator" content="James R. J.  Gray" /></meta><meta name="dc.Creator" content="Laura J.  Kaushansky" /></meta><meta name="dc.Creator" content="Patricia  Medeiros" /></meta><meta name="dc.Creator" content="Cassie  Messenger" /></meta><meta name="dc.Creator" content="Alex  Phillipou" /></meta><meta name="dc.Creator" content="Alex  Preston" /></meta><meta name="dc.Creator" content="Rab K.  Prinjha" /></meta><meta name="dc.Creator" content="Inmaculada  Rioja" /></meta><meta name="dc.Creator" content="Alexander L.  Satz" /></meta><meta name="dc.Creator" content="Simon  Taylor" /></meta><meta name="dc.Creator" content="Ian D.  Wall" /></meta><meta name="dc.Creator" content="Robert J.  Watson" /></meta><meta name="dc.Creator" content="Gang  Yao" /></meta><meta name="dc.Creator" content="Emmanuel H.  Demont" /></meta><meta name="dc.Description" content="Second-generation bromodomain and extra terminal (BET) inhibitors, which selectively target one of the two bromodomains in the BET proteins, have begun to emerge in the literature. These inhibitors..." /></meta><meta name="Description" content="Second-generation bromodomain and extra terminal (BET) inhibitors, which selectively target one of the two bromodomains in the BET proteins, have begun to emerge in the literature. These inhibitors..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 12, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00412" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00412" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00412" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00412" /></link>
        
    
    

<title>Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00412" /></meta><meta property="og:title" content="Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0027.jpeg" /></meta><meta property="og:description" content="Second-generation bromodomain and extra terminal (BET) inhibitors, which selectively target one of the two bromodomains in the BET proteins, have begun to emerge in the literature. These inhibitors aim to help determine the roles and functions of each domain and assess whether they can demonstrate an improved safety profile in clinical settings compared to pan-BET inhibitors. Herein, we describe the discovery of a novel BET BD2-selective chemotype using a structure-based drug design from a hit identified by DNA-encoded library technologies, showing a structural differentiation from key previously reported greater than 100-fold BD2-selective chemotypes GSK620, GSK046, and ABBV-744. Following a structure-based hypothesis for the selectivity and optimization of the physicochemical properties of the series, we identified 60 (GSK040), an in vitro ready and in vivo capable BET BD2-inhibitor of unprecedented selectivity (5000-fold) against BET BD1, excellent selectivity against other bromodomains, and good physicochemical properties. This novel chemical probe can be added to the toolbox used in the advancement of epigenetics research." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00412"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00412">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00412&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00412&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00412&amp;href=/doi/10.1021/acs.jmedchem.1c00412" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00365" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00703" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Francesco Rianjongdee</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francesco Rianjongdee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong><a href="/cdn-cgi/l/email-protection#e28490838c818791818dcc9acc908b838c888d8c85868787a2859189cc818d8f"><span class="__cf_email__" data-cfemail="7d1b0f1c131e180e1e125305530f141c131712131a1918183d1a0e16531e1210">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francesco++Rianjongdee">Francesco Rianjongdee</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4432-3007" title="Orcid link">https://orcid.org/0000-0003-4432-3007</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen J. Atkinson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen J. Atkinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, Quantitative Pharmacology, Immunoinflammation Therapy Area Unit, and Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div><div class="loa-info-affiliations-info">Encoded Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</div><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr. 1, Heidelberg 69117, Germany</div><div class="loa-info-affiliations-info">WuXi AppTec (Shanghai) Co., Ltd., Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+J.++Atkinson">Stephen J. Atkinson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3636-3674" title="Orcid link">https://orcid.org/0000-0003-3636-3674</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chun-wa Chung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chun-wa Chung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chun-wa++Chung">Chun-wa Chung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2480-3110" title="Orcid link">https://orcid.org/0000-0002-2480-3110</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paola Grandi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Grandi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr. 1, Heidelberg 69117, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Grandi">Paola Grandi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James R. J. Gray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James R. J. Gray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Quantitative Pharmacology, Immunoinflammation Therapy Area Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+R.+J.++Gray">James R. J. Gray</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura J. Kaushansky</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura J. Kaushansky</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Encoded Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura+J.++Kaushansky">Laura J. Kaushansky</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Patricia Medeiros</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patricia Medeiros</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Encoded Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patricia++Medeiros">Patricia Medeiros</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cassie Messenger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cassie Messenger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cassie++Messenger">Cassie Messenger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alex Phillipou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex Phillipou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex++Phillipou">Alex Phillipou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alex Preston</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex Preston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex++Preston">Alex Preston</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0334-0679" title="Orcid link">https://orcid.org/0000-0003-0334-0679</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rab K. Prinjha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rab K. Prinjha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rab+K.++Prinjha">Rab K. Prinjha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Inmaculada Rioja</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Inmaculada Rioja</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Inmaculada++Rioja">Inmaculada Rioja</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexander L. Satz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexander L. Satz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec (Shanghai) Co., Ltd., Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexander+L.++Satz">Alexander L. Satz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1284-1977" title="Orcid link">https://orcid.org/0000-0003-1284-1977</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simon Taylor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon Taylor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, Quantitative Pharmacology, Immunoinflammation Therapy Area Unit, and Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div><div class="loa-info-affiliations-info">Encoded Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</div><div class="loa-info-affiliations-info">IVIVT Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr. 1, Heidelberg 69117, Germany</div><div class="loa-info-affiliations-info">WuXi AppTec (Shanghai) Co., Ltd., Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon++Taylor">Simon Taylor</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ian D. Wall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ian D. Wall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ian+D.++Wall">Ian D. Wall</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Watson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Watson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Watson">Robert J. Watson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gang Yao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gang Yao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Encoded Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gang++Yao">Gang Yao</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Emmanuel H. Demont</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Emmanuel H. Demont</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Epigenetics Discovery Performance Unit, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Emmanuel+H.++Demont">Emmanuel H. Demont</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7307-3129" title="Orcid link">https://orcid.org/0000-0001-7307-3129</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00412&amp;href=/doi/10.1021%2Facs.jmedchem.1c00412" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 12, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 March 2021</li><li><span class="item_label"><b>Published</b> online</span>12 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00412" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00412</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D28%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFrancesco%2BRianjongdee%252C%2BStephen%2BJ.%2BAtkinson%252C%2BChun-wa%2BChung%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00412%26title%3DDiscovery%2Bof%2Ba%2BHighly%2BSelective%2BBET%2BBD2%2BInhibitor%2Bfrom%2Ba%2BDNA-Encoded%2BLibrary%2BTechnology%2BScreening%2BHit%26numPages%3D28%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D28%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00412"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">997</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00412" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of a Highly Selective BET BD2 Inhibitor from a DNA-Encoded Library Technology Screening Hit&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Francesco&quot;,&quot;last_name&quot;:&quot;Rianjongdee&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;J. Atkinson&quot;},{&quot;first_name&quot;:&quot;Chun-wa&quot;,&quot;last_name&quot;:&quot;Chung&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Grandi&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;R. J. Gray&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;J. Kaushansky&quot;},{&quot;first_name&quot;:&quot;Patricia&quot;,&quot;last_name&quot;:&quot;Medeiros&quot;},{&quot;first_name&quot;:&quot;Cassie&quot;,&quot;last_name&quot;:&quot;Messenger&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;Phillipou&quot;},{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;Preston&quot;},{&quot;first_name&quot;:&quot;Rab&quot;,&quot;last_name&quot;:&quot;K. Prinjha&quot;},{&quot;first_name&quot;:&quot;Inmaculada&quot;,&quot;last_name&quot;:&quot;Rioja&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;L. Satz&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;Taylor&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;D. Wall&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Watson&quot;},{&quot;first_name&quot;:&quot;Gang&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Emmanuel&quot;,&quot;last_name&quot;:&quot;H. Demont&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00412&quot;},&quot;abstract&quot;:&quot;Second-generation bromodomain and extra terminal (BET) inhibitors, which selectively target one of the two bromodomains in the BET proteins, have begun to emerge in the literature. These inhibitors aim to help determine the roles and functions of each domain and assess whether they can demonstrate an improved safety profile in clinical settings compared to pan-BET inhibitors. Herein, we describe the discovery of a novel BET BD2-selective chemotype using a structure-based drug design from a hit identified by DNA-encoded library technologies, showing a structural differentiation from key previously reported greater than 100-fold BD2-selective chemotypes GSK620, GSK046, and ABBV-744. Following a structure-based hypothesis for the selectivity and optimization of the physicochemical properties of the series, we identified 60 (GSK040), an in vitro ready and in vivo capable BET BD2-inhibitor of unprecedented selectivity (5000-fold) against BET BD1, excellent selectivity against other bromodomains, and good physicoch&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00412&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00412" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00412&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00412" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00412&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00412" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00412&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00412&amp;href=/doi/10.1021/acs.jmedchem.1c00412" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00412" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00412" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00412%26sid%3Dliteratum%253Aachs%26pmid%3D34251219%26genre%3Darticle%26aulast%3DRianjongdee%26date%3D2021%26atitle%3DDiscovery%2Bof%2Ba%2BHighly%2BSelective%2BBET%2BBD2%2BInhibitor%2Bfrom%2Ba%2BDNA-Encoded%2BLibrary%2BTechnology%2BScreening%2BHit%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Second-generation bromodomain and extra terminal (BET) inhibitors, which selectively target one of the two bromodomains in the BET proteins, have begun to emerge in the literature. These inhibitors aim to help determine the roles and functions of each domain and assess whether they can demonstrate an improved safety profile in clinical settings compared to pan-BET inhibitors. Herein, we describe the discovery of a novel BET BD2-selective chemotype using a structure-based drug design from a hit identified by DNA-encoded library technologies, showing a structural differentiation from key previously reported greater than 100-fold BD2-selective chemotypes GSK620, GSK046, and ABBV-744. Following a structure-based hypothesis for the selectivity and optimization of the physicochemical properties of the series, we identified <b>60</b> (GSK040), an in vitro ready and in vivo capable BET BD2-inhibitor of unprecedented selectivity (5000-fold) against BET BD1, excellent selectivity against other bromodomains, and good physicochemical properties. This novel chemical probe can be added to the toolbox used in the advancement of epigenetics research.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span> Special Issue</h4><p class="last">Published as part of the <i>Journal of Medicinal Chemistry</i> special issue “<a href="/page/vi/epigenetics" class="ext-link">Epigenetics 2022</a>”.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23175" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23175" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The bromodomain and extra terminal (BET) proteins are a family of epigenetic readers that recognize acetylated lysines on histone tails.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> There are four isoforms in the BET family: BRD2, BRD3, BRD4, and the testis-specific BRDT. Each contains two bromodomains: BD1 at the N-terminus of the protein and BD2 closer to the C-terminus.</div><div class="NLM_p">The inhibition of all eight domains of the BET proteins (pan-BET inhibition) by small molecules has been been thoroughly investigated as a potential therapy for oncology<a onclick="showRef(event, 'ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4 ref5 ref6 ref7 ref8 ref9 ref10 ref11">(2−11)</a> and immunology<a onclick="showRef(event, 'ref12 ref13 ref22 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref22 ref14 ref15 ref16 ref17 ref18 ref19 ref20 ref21">(12,13,22,14−21)</a> diseases. While the therapeutic potential of pan-BET inhibitors has been well-characterized preclinically and is being investigated in a number of oncology trials, several safety signals, such as thrombocytopenia and gastro-intestinal toxicity, have been reported in patients.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> As such, the domain selectivity may help to improve the therapeutic margin of BET inhibitors.</div><div class="NLM_p">The BD1 and BD2 domains of each family member show a high level of homology (see the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information, Figure S2">Supporting Information, Figure S2</a>); however, there are some key differences in the binding site residues between these domains, which can be exploited to obtain selectivity.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Indeed a number of highly selective compounds that bind preferentially to the four BD1<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> or four BD2<a onclick="showRef(event, 'ref27 ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29 ref30 ref31">(27−31)</a> domains of this family have now been reported. It has been shown that the selective inhibition of the BD1 or BD2 domains of the BET proteins results in distinct biological phenotypes;<a onclick="showRef(event, 'ref24 ref32'); return false;" href="javascript:void(0);" class="ref ref24 ref32">(24,32)</a> BD1-selective inhibitors appear to mostly phenocopy pan-BET, especially in oncology, while BD2-selective inhibitors show a more nuanced phenotype appropriate for the treatment of immune-mediated inflammatory diseases.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><div class="NLM_p">ABBV-774 (<b>1</b>) was the first reported highly potent and selective (>100-fold) BD2 inhibitor and is currently in Phase I clinical trials for the treatment of acute myeloid leukemia.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> We also recently published a number of drug-like BD2-selective inhibitors,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> notably GSK046<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (<b>2</b>), GSK620<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (<b>3</b>), and GSK973<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> (<b>4</b>), which all show 100–1000-fold selectivity over the BD1 domains (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of ABBV-744 (<b>1</b>) and of BD2-selective inhibitors reported by GSK (<b>2</b>, <b>3</b>, and <b>4</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because of the similarity in the binding modes of compounds <b>2</b>, <b>3</b>, and <b>4</b>,<a onclick="showRef(event, 'ref28 ref29 ref34'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref34">(28,29,34)</a> we were interested in expanding our chemical equity and looked to identify a novel BD2-selective chemotype with a differentiated binding mode. Indeed, the identification of a differentiated chemotype would help to discharge any risk of toxicity due to an unknown off-target activity from our first template.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> In addition to a high-throughput screen (HTS) performed against BRD4 BD2 (representative of all BET BD2 domains),<a onclick="showRef(event, 'ref29 ref34'); return false;" href="javascript:void(0);" class="ref ref29 ref34">(29,34)</a> which ultimately yielded compounds <b>2</b>–<b>4</b>, we performed a screen using DNA-encoded library (DEL) technologies to identify novel chemotypes showing BD2 selectivity.</div><div class="NLM_p">A DEL is composed of chimeric molecules, each containing a small molecule covalently attached to a unique DNA tag that uniquely identifies the structure of the small molecule.<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> The screening of DELs via an affinity selection is becoming an important source for identifying novel small molecule hits for various therapeutic targets.<a onclick="showRef(event, 'ref39 ref40 ref41 ref42 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42 ref43 ref44 ref45">(39−45)</a> Because of the advancement of next-generation DNA-sequencing technology (NGS), billions of DEL molecules can be screened easily in a single tube and in a cost-effective manner.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The convenience of a DEL affinity selection also makes it possible to perform the DEL selection in a multiplex fashion, where multiple conditions are run simultaneously (e.g., different buffer conditions, different proteins as counter screens, with or without known ligands) in order to identify ligands of a desired mode of action.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><div class="NLM_p last">Our aim was to identify hits that could be optimized into leads having the following profile: negative log of the half-maximal inhibition concentration (pIC<sub>50</sub>) > 7 at BD2 and greater than 1000-fold selectivity for BD2 over BD1, using BRD4 as a representative example of the BET family;<a onclick="showRef(event, 'ref29 ref34'); return false;" href="javascript:void(0);" class="ref ref29 ref34">(29,34)</a> evidence of cellular potency; no liabilities in the hERG assay; and evidence of bioavailability in rodents.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To initiate a DEL screen in search of novel BET BD2-selective chemotypes, three constructs of BRD4 were screened in parallel against an ultralarge collection of diverse DELs. To screen for BD1 potency, a 6-His-BRD4 (1–477) (Y390A) mutant was used, which minimizes binding to the BD2 domain.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> For similar reasons, the BD2 affinity was assessed with a BRD4 (1–477)(Y97A) mutant.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Activities in these two constructs were also compared with the activity against a truncated native BRD4 dual domain (1–477) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representation of BRD4 (N to C terminus) showing the two bromodomains and the extra terminal (ET) domain and the three truncated constructs used in affinity selection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The DNA tags of enriched DEL molecules from the affinity screen were then submitted to a polymerase chain reaction (PCR) and subsequent next-generation sequencing (NGS) so that the corresponding hit molecules could be identified. By analyzing the signal strength of the enriched DEL molecules across the three BRD4 constructs and no target control, two hits of similar structures (hits <b>5</b> and <b>6</b>) were identified (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Both hits were enriched for the BRD4(1–477)(Y97A) mutant and the truncated native BRD4 dual domain (1–477), but not for the BRD4(1–477)(Y390A) mutant and no target control. This enrichment pattern fitted the profile of BRD4 BD2-selective ligands. Both the BRD4 BD2 hits were derived from the same two-cycle glycine-based library, the synthesis of which is being disclosed here for the first time, shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>a. This specific glycine-based library was constructed using a split-and-pool strategy,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> its synthesis started by functionalizing the free amine group on the headpiece with a protected glycine (<b>7</b>). The fluorenylmethoxycarbonyl protecting group (Fmoc group) was then removed to give a secondary amine, which underwent the reductive amination to install the cycle-1 building blocks. After an enzymatic ligation to install the cycle-1 DNA tags, the reaction wells were pooled together, and the allyl protecting group was removed. The pool products were then split into different reaction wells to install the cycle-2 DNA tags by an enzymatic ligation and cycle-2 building blocks by capping the free secondary amines with various electrophiles to afford a total of 1.3 million enumerated DEL molecules.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. (a) Synthesis of Glycine-Based DEL Library. (b) Result of the BRD4 Screen of Glycine-Based DEL Library. Signal Strength Versus the Affinity Selection Conditions. (c) Structures of Hits <b>5</b> and <b>6</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to confirm the potency and selectivity of the DNA-encoded compound, <b>8</b> was designed and synthesized as an analogous “off-DNA” compound to hit <b>5</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). As the DNA label was likely in a solvent-exposed region and not interacting with the protein, it was replaced with a methyl group. Compound <b>8</b> was tested in our in-house time-resolved fluorescence resonance energy transfer (TR-FRET) assay, showing a sub-micromolar potency for the BD2 domains across BRD4, BRD3, and BRD2 and greater than 30-fold selectivity against the BD1 domains. Compound <b>8</b> had an acceptable molecular weight (MW) of 430 Da for a hit, which translated to a reasonable ligand efficiency (LE) value of 0.28. It was, however, lipophilic (chromlogD of 5.7), which translated to a low lipophilic ligand efficiency (LLE) of 1.7. Compound <b>8</b> also had an excellent passive permeability of 925 nm/s in an artificial membrane permeability (AMP) assay and limited solubility as measured by a chemiluminescent nitrogen detection (CLND) assay (116 μg/mL). The three aromatic rings contained within this hit translated into a property forecast index (PFI) of 8.7, a value highlighting the suboptimal developability profile of this molecule (a PFI of 6 or below being considered as a cutoff for drug-like properties).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> This would need to be addressed by lowering the lipophilicity and/or reducing the number of aromatics present during a hit expansion.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Profile of Hit <b>8</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0013.gif" alt="" id="gr4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">compound</td><td class="colsep0 rowsep0" align="left">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 FRET BD1/BD2 (pIC<sub>50</sub> (n) (sel))</td><td class="colsep0 rowsep0" align="left">5.1 (9)/6.6 (8) (30-fold)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD3 FRET BD1/BD2 (pIC<sub>50</sub> (n) (sel))</td><td class="colsep0 rowsep0" align="left">4.6 (2)/7.0 (2) (250-fold)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD2 FRET BD1/BD2 (pIC<sub>50</sub> (n) (sel))</td><td class="colsep0 rowsep0" align="left">4.6 (2)/6.4 (1) (60-fold)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LE/LLE (BRD4 BD2)</td><td class="colsep0 rowsep0" align="left">0.28/1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chromlogD<sub>7.4</sub>/PFI</td><td class="colsep0 rowsep0" align="left">5.7/8.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMP (nM/s)</td><td class="colsep0 rowsep0" align="left">925</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLND (μg/mL)</td><td class="colsep0 rowsep0" align="left">114</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MW (Da)</td><td class="colsep0 rowsep0" align="left">430</td></tr></tbody></table></div></div><div class="NLM_p">The number of test occasions is given in (<i>n</i>), and the selectivity for BD2 over BD1 is given in (sel).</div><div class="NLM_p">A cocrystal X-ray structure of compound <b>8</b> was rapidly obtained in BRD4 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) and is shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a. The methoxy phenyl group sits in the acetyl lysine pocket, acting as an acetyl lysine (KAc) mimetic<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> with the ortho-methyl group mimicking the methyl group of acetyl lysine and the oxygen in the methoxy group mimicking the carbonyl, forming a hydrogen bond with the conserved asparagine, Asn433 (dashed red line). Unusually, the methyl of the methoxy group displaces one of the water molecules usually conserved upon ligand binding to bromodomains. Notably, the typical water-mediated interaction from an acetyl lysine mimetic with Tyr390 is not formed. The benzhydryl moiety interacts with His437 with both phenyl rings. One of these phenyls makes a staggered face-to-face interaction with His437, and the other makes an edge-to-face interaction with the same residue while sitting on the lipophilic region consisting of Trp374, Pro375, and Phe376 (i.e., the WPF shelf), and also making a hydrophobic interaction with Trp374. The glycine moiety occupies the ZA channel, with the terminal carboxamide projecting toward bulk solvent and does not make any hydrogen-bonding interactions with the protein. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b shows the overlay of compound <b>8</b> in BRD4 BD2 and compound <b>3</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1">6ZB1</a>). Here, a similar occupation of the WPF shelf is observed with the phenyl rings of <b>8</b> and <b>3</b>. Our previously reported compounds predominantly achieve their selectivity through an occupation of the region bound by the cyclopropyl amide moiety in compound <b>3</b>, which is facilitated by a second hydrogen bond to the conserved asparagine, shown as a dashed yellow line.<a onclick="showRef(event, 'ref28 ref29 ref30 ref34'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref34">(28−30,34)</a> The current series does not access this region, offering a structurally differentiated approach to achieving BD2 selectivity. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c shows the overlay of compound <b>8</b> in BRD4 BD2 and compound <b>1</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E6J">6E6J</a>). This overlay shows that the current series is similarly differentiated from <b>1</b>.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with van der Waals (VDW) surface and conserved waters shown in red, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>). The methoxyphenyl ring acts as the KAc-mimetic, forming a hydrogen bond with Asn433 (red dashed line). One of the phenyl rings occupies the WPF shelf and sandwiches His437 with the other phenyl ring. The glycine fragment of the molecule enters the ZA channel region. (b) Crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1">6ZB1</a>) overlain with <b>3</b> in BRD2 BD2 (blue ligand, protein not shown, BRD4 numbering used). The hydrogen-bonding interactions of <b>3</b> with the conserved asparagine are shown (dashed yellow line). (c) Crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface) overlain with the crystal structure of <b>1</b> in BRD2 BD2 (yellow ligand, protein not shown, BRD4 numbering used, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E6J">6E6J</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compound <b>8</b> was selective for the BD2 domains of the BET family with high BRDx BD2 potencies across the isoforms and served as an excellent starting point for further optimization.</div><div class="NLM_p">To identify inhibitors matching our lead criteria (vide supra), we focused our investigations in three distinct parts of the molecule, which are shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, namely, the acetyl-lysine mimetic (KAc-mimetic) substituent (shown in red), the WPF shelf substituent (shown in purple), and the ZA channel substituent (shown in green).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of compound <b>8</b> with the key areas for optimization highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We looked to replace the lipophilic aromatic KAc-mimetic in <b>8</b> with a group with better physicochemical properties and hopefully increase the binding efficiency against the target. An overlay of the crystal structure of <b>8</b> in BRD4 BD2 with a crystal structure of a published inhibitor I-BET-469<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (<b>9</b>, pink, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEP">7OEP</a>) in BRD2 BD2 showed  a remarkably close overlap of the two KAc-mimetic groups (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Overlay of crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface, dashed red hydrogen bonds, red waters, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) with <b>9</b> in BRD2 BD2 (magenta ligand, protein not shown, dashed yellow hydrogen bond, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEP">7OEP</a>), showing a close overlap of the KAc-mimetic groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This clear overlap suggested the possible replacement of the dimethyl methoxyphenyl acetyl-lysine mimetic with a dimethyl pyridone (DMP), and this was used in the design of compound <b>10</b>. Pleasingly, this replacement gave an increase in the BD2 potency of 1.5 log units while maintaining a 30-fold selectivity for BD2 over BD1 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Furthermore, this KAc-mimetic replacement resulted in a decrease in the chromlogD of our inhibitors from 5.7 to 3.4 along with an increase in LE and LLE to 0.36 and 5.9, respectively. The BD2 potency of <b>8</b> was close to the tight-binding limit of the FRET assay (pIC<sub>50</sub> ≈ 8.3, half-maximal inhibitory concentration (IC<sub>50</sub>) ≈ 5 nM) and therefore may have been higher than the quoted value. Encouragingly, the compound was able to inhibit the production of monocyte chemoattractant protein 1 (MCP-1) in a lipopolysaccharide (LPS)-stimulated human whole blood (hWB) assay with a high potency, reflecting cellular penetration, BD2 inhibition, and an anti-inflammatory phenotype. The reduction in lipophilicity from <b>8</b> appeared to affect the permeability of the compound, which decreased from 951 nm/s in <b>8</b> to 27 nm/s in <b>10</b>. This indicated that permeability would need to be monitored throughout the optimization. Overall, the DMP KAc-mimetic offered a good starting point to optimize other parts of the molecule; further exploration of the KAc-mimetic region was performed later in the project (vide infra).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Profile of Inhibitor <b>10</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0014.gif" alt="" id="gr8" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">compound</td><td class="colsep0 rowsep0" align="left"><b>10</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 FRET BD1/BD2 (pIC<sub>50</sub> (<i>n</i>) (sel))</td><td class="colsep0 rowsep0" align="left">6.6 (7)/8.1 (7) (32-fold)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCP1: hWB<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> (pIC<sub>50</sub> (<i>n</i>))</td><td class="colsep0 rowsep0" align="left">7.6 (3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LE/LLE (BRD4 BD2)</td><td class="colsep0 rowsep0" align="left">0.36/5.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chromlogD<sub>7.4</sub>/PFI</td><td class="colsep0 rowsep0" align="left">3.4/6.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMP (nm/s)</td><td class="colsep0 rowsep0" align="left">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MW (Da)</td><td class="colsep0 rowsep0" align="left">417</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLND (μg/mL)</td><td class="colsep0 rowsep0" align="left">163</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Measured using an MSD plate.</p></div></div></div><div class="NLM_p">Structural determination showed an extremely close overlay between the DMP and dimethoxyphenyl-containing compounds (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). With a change to the DMP KAc-mimetic, the full network of conserved water molecules was observed indicating that the increase in potency of 1.5 log units can be largely attributed to the binding of the W1 water molecule and the interactions therewith.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface, red waters, dashed red line for hydrogen bonds, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) and <b>10</b> in BRD2 BD2 (blue ligand, protein not shown, red and pink waters, dashed yellow lines for hydrogen bonds, BRD4 numbering used, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OER">7OER</a>). The hydrogen bond from each KAc-mimetic to Asn433 is shown, along with the hydrogen bonds with the returning water molecule in the crystal structure for <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to investigate opportunities to increase the selectivity of the series, we looked at modifying the nature of the group occupying the ZA channel. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> shows a crystal structure of <b>8</b> (yellow ligand) in BRD4 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) overlaid with a crystal structure of <b>3</b> in BRD4 BD1 (ligand not shown, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3">6ZB3</a>). It can be seen that the ZA channel in BRD4 BD2 between Leu385 and Trp374 is wider than the ZA channel between Leu92 and Trp81 in BRD4 BD1. We hypothesized that introducing bulky substituents in this region of the ZA channel may be less tolerated in BD1 due to this smaller channel, thereby increasing the BD2 selectivity. These crystallographic data also suggested that a substitution from the α-position of the glycine moiety of our inhibitors would provide the appropriate vector to do so, and a number of substituents were introduced in this position (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of crystal structure of <b>8</b> in BRD4 BD2 (purple protein, purple VDW surface, and yellow ligand, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) with an aligned crystal structure of <b>3</b> in BRD4 BD1 (ligand not shown, green protein, green VDW surface, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3">6ZB3</a>). Structures superposed with an in-house script using C-α atoms of key active-site residues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR for the α-Substituent from the Glycine Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0015.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0016.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Charged aerosol detection (CAD) solubility used instead of CLND.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Chirality not determined–single unknown enantiomers.</p></div></div><div></div></div><div class="NLM_p">In accordance with our hypothesis, increasing the size of the substituent in the α-position of the glycine moiety gave a stepwise increase in selectivity by a greater reduction of potency against BD1 compared to BD2.</div><div class="NLM_p">The two enantiomers (<b>12</b> and <b>13</b>) of the methylated analogue (<b>11</b>) were separated and found to show similar potency and selectivity. In the following structure–activity relation (SAR), we chose to only separate the enantiomers of the most interesting compounds, showing high potency (pIC<sub>50</sub> > 7.5) and selectivity (>100-fold) while being reasonably lipophilic (chromlogD < 4). Increasing the size of the ZA channel substituent from methyl to ethyl (<b>14</b>), isopropyl (<b>15</b>), and <i>tert</i>-butyl (<b>16</b>) had a minimal impact on BD2 potency but reduced the BD1 potency dramatically, with <b>16</b> being 1260-fold selective. We were surprised that substituents as large as <i>tert</i>-butyl could occupy this region, given that our initial assumption was that α-substituents of the glycine moiety would occupy the ZA channel. It was possible that such substituents adopted different binding conformations still providing potency and selectivity. Nonetheless, the data shown prove that larger α-substituents to the glycine moiety lead to selectivity, albeit with an unacceptable increase in chromlogD (5.4 in the case of <b>16</b>) leading to decreased solubility (<100 μg/mL) and LLE (3.6 for <b>16</b> vs 5.9 for <b>10</b>). We also looked at cyclic substituents in this α-position. The cyclopropyl group (<b>17</b>) gave a similar profile to the methyl substituent (<b>12</b>), while increasing the ring size to a cyclobutyl group (<b>18</b>) gave a boost to selectivity similar to that seen with the ethyl substituent (<b>14</b>) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_p">With these data in hand, we looked at cyclic ether derivatives to reduce lipophilicity: oxetane (<b>19</b>) led to a slight drop in the BD2 potency compared to the analogous cyclobutyl <b>18</b> (less than threefold) but maintained the same level of selectivity (500-fold). The drastic reduction in chromlogD from cyclobutyl <b>18</b> to oxetane <b>19</b> (from 4.9 down to 3.6) led to a sizable increase in LLE (from 3.9 for <b>18</b> to 6 for <b>19</b>). The potency of <b>19</b> was, however, considered too low to justify the separation of single enantiomers, and further analogues were considered. Increasing the ring size to a tetrahydrofuran (THF) introduced a second chiral center (<b>20</b>). Since the potency of <b>20</b> was similar to that of <b>19</b>, the diastereomers <b>21</b>, <b>22</b>, <b>23</b>, and <b>24</b> were separated and profiled in the hope that the potency will only reside in one of these four isomers. Gratifyingly, <b>23</b> showed a high BD2 potency (pIC<sub>50</sub> of 7.8) and a high selectivity (∼2000-fold) similar to that of the <i>tert</i>-butyl analogue <b>16</b> but with a reduced chromlogD of 4, an excellent LLE (6.0), and improved solubility (>181 μg/mL). Overall, while the cyclic ether analogues were slightly less potent than compound <b>10</b>, they provided a good balance of potency, selectivity, and chromlogD (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_p">We also assessed the opportunity to build selectivity without introducing chirality via a symmetric geminal substitution on this position. The dimethyl substituted analogue <b>25</b> was 10-fold less potent and selective than the most selective α-methyl isomer, compound <b>12</b>. However, the introduction of a geminal-cyclopropyl group (<b>26</b>) had a limited impact on the BD2 potency (pIC<sub>50</sub> of 7.5) but increased the selectivity (400-fold) thanks to a significant reduction in the BD1 potency to give a similar profile to the ethyl-substituted analogue <b>14</b>. Further increasing the size of the ring to cyclobutane <b>27</b> and cyclopentane <b>28</b> led to a decrease in potency and selectivity versus <b>26</b> while increasing the lipophilicity of the inhibitors, impacting both LLE and solubility. On the basis of these data, no analogous ether compounds were made. Overall, the best substituent from this exercise was the THF group seen in <b>23</b>, which showed a high BD2 potency (pIC<sub>50</sub> = 7.8), good LE (0.3) and LLE (6.0) values, a high selectivity (∼2000-fold) against BD1, and a moderate chromlogD (4.0), leading to good solubility and moderate permeability (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_p">In parallel to this effort, we also investigated the effect of changing the methyl glycine moiety to understand whether a compound with a similar biochemical profile could be obtained without the hydrogen-bond donor of the secondary amide, as this may have offered a route to a higher permeability at a similar or lower chromlogD (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). To begin with, we profiled the analogous primary amide <b>29</b> as a benchmark. This compound showed a similar potency and selectivity profile to those of <b>10</b> with a reduced chromlogD of 2.9. As expected, this also led to a slight reduction in AMP. The replacement of the entire glycine moiety with an ethyl group (<b>30</b>) maintained a similar potency to that of compound <b>10</b> with a 1.5 log increase in chromlogD and, hence, a parallel increase in AMP. As discussed previously, the amide was not thought to be making any beneficial interactions with the protein, and this was confirmed by this SAR. The removal of the amide resulted in an increase in the LE (0.36 to 0.41); however, the LLE was reduced (5.9 to 4.7), which reflected the increase in chromlogD coupled with a reduction in the heavy atom count.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the ZA Channel Region with Removal of the Amide</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0017.gif" alt="" id="gr13" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0018.gif" alt="" id="gr14" /></img><div></div></div><div class="NLM_p">With these data in hand, we looked at introducing small hydrophilic substituents devoid of hydrogen-bond donors in a bid to improve or maintain LE and LLE while increasing the AMP with respect to <b>10</b>: an α-substitution of the ethyl group with a nitrile (<b>31</b>) gave a boost in BD2 selectivity, however, without the desired reduction in chromlogD to give the desired reduction in the LE and LLE. The replacement of the nitrile in <b>31</b> with a tetrahydrofuran (<b>32</b>) gave a profile akin to the previous THF-containing glycine analogue <b>23</b> but with a higher lipophilicity. The addition of a moderately basic morpholine group to the ethyl analogue <b>30</b> (compound <b>33</b>) gave a 0.8 log decrease in chromlogD with a similar selectivity profile. An α-methyl substitution of the morpholine (compound <b>34</b>) was performed to assess whether the selectivity could be recovered, as this is the analogous position to the α-substituents to the glycine moiety. This gave inhibitor <b>34</b>, which had an improved selectivity (200-fold vs 30-fold), albeit with a high lipophilicity, and therefore we did not separate the enantiomers. A disubstitution with methoxy methyl groups (<b>35</b>) gave a similar profile in terms of the selectivity, potency, and chromlogD to the branched morpholine <b>34</b> but with the removal of chirality from the molecule. These findings provided further evidence that the amide was indeed not needed for potency or selectivity but acted as a useful functionality to lower the chromlogD of the series. Unfortunately it proved difficult to identify a substituent devoid of hydrogen-bond donors that was able to reduce the lipophilicity to within the same order as that of <b>10</b>.</div><div class="NLM_p">Overall these first rounds of SAR demonstrated the opportunity to increase potency and selectivity by a specific modification of the group accessing the ZA channel (compound <b>23</b>) and that it was possible to introduce a broad range of substituents in place of the methyl amide present in <b>10</b>, hence enabling tuning of the intrinsic properties of our inhibitors. In most cases, however, increasing the selectivity of the compounds while maintaining the BD2 potency led to inhibitors with increased MW and lipophilicity compared to those of <b>10</b>.</div><div class="NLM_p">We therefore revisited the KAc-mimetic region with the aim to reduce the chromlogD and/or the aromatic ring count of the series, both strategies likely to have a positive impact on the developability (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). For this exploration, both the methyl and ethyl terminal amides in the ZA channel region were used, which was shown to have a little impact on potency and selectivity in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> (<b>11</b> vs <b>36</b>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of KAc-Mimetic Replacements</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0019.gif" alt="" id="gr15" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0020.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">CAD solubility.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">All molecules are racemate with the exception of <b>41</b>, which is a single unknown enantiomer.</p></div></div><div></div></div><div class="NLM_p">In order to reduce the lipophilicity of <b>36</b>, we removed each of the methyl groups from the DMP to assess whether these were required for binding (compounds <b>37</b> and <b>38</b>). The lipophilicity was indeed reduced, but both changes led to a significant and unacceptable loss in the BD2 potency versus <b>36</b>. Introducing additional polarity into the DMP KAc-mimetic (dimethyl pyrazinone <b>39</b> as a representative example) was also detrimental to potency and gave a surprising increase in chromlogD. The replacement of a methyl group with a methoxy group (<b>40</b>) gave a slight reduction in chromlogD (3.7 in <b>11</b> vs 3.3 in <b>40</b>) coupled with a 10-fold reduction in the BD2 potency, resulting in similar LE and LLE values but with an unacceptable potency. Finally, the saturation of the ring with a <i>N</i>-methyl piperidone led to a large loss in the BD2 potency (see compound <b>41</b>, the only sub-micromolar compound of four separated diastereomers). Overall, these data suggested the initial DMP KAc-mimetic was optimal regarding potency, LE, and LLE, and therefore no further work was performed in this area.</div><div class="NLM_p">Our final iteration of SAR looked at the geminal diphenyl moiety. It was hypothesized that this region was crucial for activity (all known potent BET inhibitors interact with the WPF shelf) and selectivity (by interaction with His437). We aimed to introduce polarity into this region to reduce the overall lipophilicity in the series and to remove an aromatic to improve the overall physicochemical properties of the series (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SAR of the WPF Shelf Region</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0021.gif" alt="" id="gr17" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0022.gif" alt="" id="gr18" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Chirality not determined–single unknown enantiomer.</p></div></div><div></div></div><div class="NLM_p">To begin with, we looked at replacing a phenyl ring with a methyl group. This gave racemate <b>42</b>, which showed an almost 1000-fold drop in potency at BD2 and a significant drop in selectivity, LE, and LLE. A cocrystal structure of <b>42</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OES">7OES</a>) showed binding of the (<i>S</i>)-isomer with the phenyl ring interacting with His433 and Leu385 (not shown for clarity) rather than interacting with Trp370 on the WPF shelf (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Consequently, this binding mode results in a minimal occupation of the WPF shelf by the methyl group, which is unusual in BET ligands and probably explains the large decrease in potency.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0008.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cocrystal structure of <b>42</b> in BRD2 BD2 (green ligand, gray protein with VDW surface, red waters, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OES">7OES</a>) showing the methyl group occupying the WPF shelf.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The basis of the following SAR was the hypothesis that novel nonaryl substituents would interact preferentially with the WPF shelf, while the remaining phenyl ring would bind in a similar way to that seen with compound <b>42</b>. However, without crystal structures for each novel inhibitor, there remains the possibility that the novel phenyl replacement may have occupied the region between His437 and Leu385, placing the remaining phenyl ring on the WPF shelf. Growing the methyl group to an ethyl (<b>43</b>), isopropyl (<b>44</b>), and ethoxy (<b>45</b>) group led to slight increases in potency at both BD1 and BD2, however, remaining ∼100-fold less potent than the phenyl analogue (compound <b>10</b>). All of the modifications that explored the replacement of the aromatic ring led to a significant drop in the LE/LLE, emphasizing the speculation that the phenyl ring was making specific interactions with His433 and Trp370 rather than increasing binding through nonspecific lipophilic effects. The permeability increased in line with chromlogD. Substitution with a tetrahydropyran (THP) group (<b>46</b>) surprisingly led to a complete loss of selectivity, along with low BD2 and BD1 potencies. A substitution with a methylene-connected THP group (<b>47</b>) gave a small increase in BD2 potency; however, the potency remained below the pIC<sub>50</sub> value of 7, and the permeability was affected by the low chromlogD of this compound. The cyclohexyl derivative was not synthesized due its high calculated chromlogD (4.8 vs 2.2 calculated for THP <b>46</b>). These results suggested that two aromatic rings in this region were essential for potency and selectivity. Assuming that these nonaromatic substituents were indeed on the WPF shelf, this SAR is in accordance with our previously published work on BET BD2 inhibitors,<a onclick="showRef(event, 'ref29 ref34'); return false;" href="javascript:void(0);" class="ref ref29 ref34">(29,34)</a> where in the majority of cases nonaromatic substituents are poorly tolerated on the WPF shelf.</div><div class="NLM_p">On the basis of these findings, we then looked to replace one of the phenyl rings with a polar aromatic ring in order to reduce the lipophilicity of our inhibitors. The substitution of a phenyl ring with an imidazole (<b>48</b>) gave a compound with a similar selectivity to that of <b>10</b> albeit with a 32-fold lower potency at BD2, suggesting that a highly polar and electron-poor aromatic ring may not be tolerated in the WPF shelf region either. The significant reduction in chromlogD resulted in a permeability below the level of detection, and less polar/electron poor aromatics were then considered. Although the N-linked pyrazole group (<b>49</b>) had a measurable permeability thanks to an increased lipophilicity, its potency was not high enough to consider further analogues. All pyridine isomers were tested (inhibitors <b>50</b>, <b>53</b>, and <b>54</b>) and showed only a moderate drop-off in potency compared to the phenyl analogue <b>10</b> (pIC<sub>50</sub> values of ca. 7.2–7.4 vs 8.1) but a similar selectivity (25–50-fold). The enantiomers of <b>50</b> were separated (compounds <b>51</b> and <b>52</b>) and surprisingly showed a significant difference in potencies presumably due to placement of the pyridine ring either on the WPF shelf or against His437. Unfortunately the exact stereochemistry of the compounds could not be assigned. Overall, the permeability was low due to a reduced lipophilicity (chromlogD ≈ 1.7–2.3). A last attempt was made to lower the lipophilicity in this region with the introduction of a hydroxyl group in the dibenzylic position of <b>10</b> (compound <b>55</b>). This substituent indeed gave a reduction in chromlogD and also potency but had no effect on the selectivity and LLE compared to those of <b>10</b> while having a similar permeability. Overall, these data suggested that the removal of an aromatic ring in this region gave a dramatic deterioration in potency. Fortunately, polarity could still be introduced by the replacement of a phenyl ring with a pyridine (<b>52</b>) or by an addition of a benzylic hydroxyl substituent (<b>55</b>). While none of the analogues remained as potent and ligand efficient as the gem-diphenyl parent compound <b>10</b>, the pyridine WPF shelf group offered a handle to reduce lipophilicity.</div><div class="NLM_p">The SAR developed highlighted the importance of the DMP KAc-mimetic and of a geminal diaryl WPF shelf substituent for potency. We found that the ZA channel substituent had the most impact on selectivity, the installation of which generally led to an increase in lipophilicity. The distal methyl amide had a limited impact on the potency and selectivity but proved to significantly reduce the lipophilicity while maintaining the LE and improving the LLE. With all these data in hand, we looked to identify a set of molecules with an optimal potency and selectivity but with a chromlogD close to 3, as this was considered to provide the best opportunity for an acceptable solubility and permeability and possibly in vitro pharmacokinetics (PK). Therefore, using <b>52</b> as a starting point, we explored ZA channel substituents to increase selectivity as shown in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. SAR for Pyridyl-Containing Derivatives<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0024.gif" alt="" id="gr20" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0025.gif" alt="" id="gr21" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">All compounds are single enantiomers.</p></div><div class="footnote" id="t7fn2"><sup><sup>b</sup></sup><p class="last">Tested <4.3 on three occasions.</p></div><div class="footnote" id="t7fn3"><sup><sup>c</sup></sup><p class="last">Chirality not determined–single unknown enantiomer.</p></div><div class="footnote" id="t7fn4"><sup><sup>d</sup></sup><p class="last">Racemic mixture.</p></div><div class="footnote" id="t7fn5"><sup><sup>e</sup></sup><p class="last">(R) stereochemistry at benzylic center.</p></div><div class="footnote" id="t7fn6"><sup><sup>f</sup></sup><p class="last">(S) stereochemistry at benzylic center.</p></div></div><div></div></div><div class="NLM_p">The reintroduction of a methyl group in the α-position gave a mixture of four isomers, which were separated by chiral chromatography to give <b>56</b>, <b>57</b>, <b>58</b>, and <b>59</b>. As expected, a methyl substitution had a positive impact on the selectivity in all cases, although the potency against BD2 was reduced versus that of <b>52</b>. The lipophilicity was increased but was lower than our targeted value of chromlogD of 3. The most potent of these compounds, <b>56</b>, showed a reasonable potency but lacked the selectivity to match our probe criteria. We therefore looked to reintroduce our optimal α-substituent, which was the THF in <b>23</b>; however, we found that the separation by chiral chromatography of the eight resulting isomers was extremely challenging. We therefore made the THP analogues to remove the additional chiral center but maintain the steric bulk and ethereal cyclic system. The synthetic sequence separated the enantiomers of the pyridyl portion first, followed by a synthesis of the final compounds and the separation of the diastereomers giving two pairs of products with the same chirality at the benzylic center. These were the pairs <b>60</b>-<b>61</b> and <b>62</b>-<b>63</b>, shown in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>. The stereochemistry of the α-centers was deduced from the crystal structure of <b>60</b> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>) and the assignment of enantiomers by NMR (the enantiomeric pairs were <b>60</b>-<b>63</b> and <b>61</b>-<b>62</b>, deduced by the identity of assignments given in the experimental data). Pleasingly, <b>60</b> met our probe criteria with an unprecedented greater than 5000-fold level of selectivity for BD2 with a pIC<sub>50</sub> of 8.3 and chromlogD near the desired value of 3 (2.9).</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0009.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cocrystal structure of <b>60</b> in BRD2 BD2 (green ligand, gray protein with VDW surface, red waters, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OET">7OET</a>) of the protein at the binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A cocrystal structure of <b>60</b> in BRD2 BD2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OET">7OET</a>) was obtained and is shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>.</div><div class="NLM_p">Interestingly, a substitution with the THP results in the rotation of the terminal methyl amide to stack against Leu381, with the THP sitting in the solvent-exposed region beyond the ZA channel in the space occupied by the amide in the crystal structure of <b>10</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Indeed, this may suggest that this binding mode was also adopted by some of the bulkier compounds shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, such as the <i>tert</i>-butyl analogue <b>16</b>. The exact stereochemistry of the bis-aryl portion could not be unambiguously assigned, but the pyridyl was thought to sit on the WPF shelf, giving the (<i>S</i>) enantiomer, where the pyridyl nitrogen interacts with the bulk solvent rather than in the alternative position, where it would be adjacent to hydrophobic residues.</div><div class="NLM_p">Our approach of increasing the bulk on the α-carbon of the glycine to target the narrower ZA channel region in BD1 was successful in increasing the selectivity. Moreover, although bulkier substituents result in a flip in the orientation of the glycine and its α-substituent, both orientations yield the desired gain in selectivity. Note that there is also an amino acid change beyond the ZA channel (Lys378 in BRD4 BD2 is glutamine in BRD4 BD1), which could also be contributing to the high selectivity of these compounds either through direct interactions or changes in the water structure within the ZA channel as a result of the amino acid change. The gains in this region, coupled with the selectivity achieved by sandwiching His433 with the benzhydryl moiety, have resulted in achieving an extremely high selectivity with a novel BD2 pharmacophore.</div><div class="NLM_p">The solubility of an amorphous form of compound <b>60</b> in CLND and in the more relevant fasted state simulated intestinal fluid (FaSSIF) showed high kinetic and thermodynamic solubilities, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). This compound was also screened against the DiscoverX panel of bromodomain proteins and showed exquisite selectivity for the BD2-BET bromodomains (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>, <a class="ref internalNav" href="#notes1" aria-label="Supporting Information Table S2">Supporting Information Table S2</a>).</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0010.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Potency of compound <b>60</b> in the DiscoverX bromodomain panel (measured as % inhibition at a concentration of 10 uM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Data for Compound <b>60</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0026.gif" alt="" id="gr24" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD1/BD2 (pIC<sub>50</sub>, (<i>n</i>))</td><td class="colsep0 rowsep0" align="left">4.6 (4)/8.3 (4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4 BD2 selectivity</td><td class="colsep0 rowsep0" align="left">5012-fold</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LE/LLE/LLEat</td><td class="colsep0 rowsep0" align="left">0.31/6.5/0.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCP-1 hWB<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a> (pIC<sub>50</sub>, (<i>n</i>))</td><td class="colsep0 rowsep0" align="left">7.3 (3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chromlogD</td><td class="colsep0 rowsep0" align="left">2.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AMP (nm/s)</td><td class="colsep0 rowsep0" align="left">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CAD/FaSSIF (μg/mL)</td><td class="colsep0 rowsep0" align="left">>231/>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hepatocytes r/h/d (mL/min/g)</td><td class="colsep0 rowsep0" align="left">3.13, <0.45, <0.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat ivPK (dose)</td><td class="colsep0 rowsep0" align="left">(1 mg/kg)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CLb (mL/min/kg), <i>t</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">50, 0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>∞</sub> (ng·h/mL), Vss (L/kg)</td><td class="colsep0 rowsep0" align="left">340,0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rat poPK (dose)</td><td class="colsep0 rowsep0" align="left">(3 mg/kg)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub>(nM), <i>t</i><sub>max</sub>(h)</td><td class="colsep0 rowsep0" align="left">398, 0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>∞</sub> (ng.h/mL), <i>F</i>%</td><td class="colsep0 rowsep0" align="left">162, 16</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">Measured by flow cytometry.</p></div></div></div><div class="NLM_p">This compound satisfied our biochemical and physicochemical optimization goals and was further profiled to assess its use as an in vivo tool. Compound <b>60</b> was therefore screened in rat, dog, and human hepatocyte assays. Pleasingly, <b>60</b> showed a low turnover in the dog and human assays and a moderate turnover in the rat assay (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). This latter in vitro clearance value translated to a moderate blood clearance of 50 mL/min/kg in vivo in the rat, with a half-life of 0.2 h. A moderate oral bioavailability of 16% was observed, likely impacted by the permeability of the compound, showing that an exposure could be achieved by an oral administration.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Synthesis</h3><div class="NLM_p">The majority of the compounds detailed were synthesized in a single step via the Ugi reaction (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> In all cases, the yields on the first attempt were sufficient to deliver enough material for screening and were not optimized. Single enantiomers were separated at the last step with the exception of inhibitors <b>60</b>–<b>63</b>, where a chiral amine was used as the starting material. For the primary amide <b>29</b>, the Ugi reaction used α,α-dimethyl benzylamine isocyanate, and the intermediate benzyl amide was converted to the primary amide in acidic conditions (trifluoroacetic acid (TFA), 50 °C, 1.5 h, 62%). Inhibitors <b>30</b> and <b>33</b> were obtained by an alkylation of the secondary amide <b>64</b>, while compounds <b>8</b>, <b>31</b>, <b>32</b>, <b>34</b>, and <b>35</b> were obtained in a two-step procedure (a reductive amination to give secondary amine <b>65</b> followed by an amide coupling) in moderate yields.</div><figure id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0012.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route towards pan-BD2 Selective Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p><span class="fn-label"><sup>a</sup></span>Reagents and conditions: MeOH or CF<sub>3</sub>CH<sub>2</sub>OH, 70–100 °C, 30–90 min, microwave irradiation, 11–91%.</p><p><span class="fn-label"><sup>b</sup></span>Chiral separation.</p><p><span class="fn-label"><sup>c</sup></span>HATU, DIPEA, DMF, room temperature, 81%.</p><p><span class="fn-label"><sup>d</sup></span>EtI, <i>t</i>-BuOK, DMF, 100 °C, microwave irradiation, 43% or R<sub>1</sub>R<sub>2</sub>N(CH<sub>2</sub>)<sub>2</sub>Cl, NaH, DMF, room temperature, 18%.</p><p><span class="fn-label"><sup>e</sup></span>NaHB(OAc)<sub>3</sub>, AcOH, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 7–68%.</p><p><span class="fn-label"><sup>f</sup></span>SOCl<sub>2</sub>, 60 °C, 98% then CH<sub>3</sub>CN, DIPEA, room temperature, 34%.</p><p><span class="fn-label"><sup>g</sup></span>(Isocyanomethyl)trimethylsilane was used instead of isocyanomethane, followed by a TBAF-mediated deprotection: TBAF (1 M in THF), 50 °C, 1 h, 4%.</p><p><span class="fn-label"><sup>h</sup></span>(1-Ethoxycyclopropoxy)trimethylsilane was used as a cyclopropanone substitute.</p><p class="last"><span class="fn-label"><sup>i</sup></span>(2-Isocyanopropan-2-yl)benzene was used followed by a TFA-mediated deprotection: TFA, 50 °C, 1.5 h, 62%.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60131" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60131" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In search of a novel BD2-selective BET inhibitor chemotype, a DNA-encoded library screening was successfully employed to provide a valuable starting point, namely, compound <b>8</b>. This hit showed BD2-selectivity across the BET proteins and displaced W1 in BRD4. The replacement of the acetyl-lysine mimetic of <b>8</b> with a published BET KAc-mimetic gave a significant boost in potency and a more drug-like lead, <b>10</b>. By use of crystallography in BRD2 BD2 and a structure-based design, we sought to exploit the smaller ZA channel in BD1 compared to BD2 to increase the BD2-selectivity. An investigation of this hypothesis confirmed that the placement of the steric bulk in this region indeed led to an increase in the selectivity proportional to the size of the substituent; however, the exact placement of the bulky groups was later found to be able to switch between occupying the ZA channel or directed toward a bulk solvent. Branched substituents were found to be preferred for selectivity and potency, with cyclic ethereal substituents offering the greatest balance of potency, selectivity, and physicochemical properties. We performed extensive SAR investigations to the KAc-mimetic and bis-phenyl (WPF shelf) groups: The DMP KAc-mimetic was found to be the most optimal, and aromatic heterocycles in the WPF shelf region offered the best physicochemical properties with the maintenance of potency and selectivity at BD2. By combining these findings, we were able to design the highly potent (BRD4 BD2 pIC<sub>50</sub> = 8.3) inhibitor <b>60</b>, which showed an unprecedented greater than 5000-fold BD2 selectivity. Moreover, a pharmacokinetic profiling of <b>60</b> confirmed oral bioavailability in the rat. This compound represents a novel chemotype and can therefore be added to the tool box of potent and selective BD2-selective probes such as inhibitors <b>1</b>–<b>4</b>. Furthermore, the one-step synthesis of this compound followed by a chiral separation should enable groups to synthesize this compound with ease, helping the scientific community to further build its understanding of the impact of selective BD2 inhibition.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Experimental</h3><div class="NLM_p">Unless otherwise stated, all reactions were performed under an atmosphere of nitrogen in heat or oven-dried glassware and an anhydrous solvent. Solvents and reagents were purchased from commercial suppliers and used as received. Reactions were monitored by thin-layer chromatography (TLC) or liquid chromatography–mass spectrometry (LCMS). The TLC was performed on glass or aluminum-backed 60 silica plates coated with UV<sub>254</sub> fluorescent indicator. Spots were visualized using UV light (254 or 365 nm) or alkaline KMnO<sub>4</sub> solution, followed by a gentle heating. The LCMS analysis was performed on a Waters Acquity UPLC instrument equipped with a CSH C18 column (50 mm × 2.1 mm, 1.7 μm packing diameter) and a Waters micromass ZQ MS using alternate-scan positive and negative electrospray. Analytes were detected as a summed UV wavelength of 210–350 nm. Two liquid-phase methods were used: <i>Formic</i>—40 °C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) H<sub>2</sub>O containing 0.1% volume/volume (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. <i>High pH</i>—40 °C, 1 mL/min flow rate. Gradient elution with the mobile phases as (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia and (B) acetonitrile. Flash column chromatography was performed using Biotage SP4 or Isolera One apparatus with SNAP silica cartridges. A mass-directed automatic purification (MDAP) was performed using a Waters ZQ MS using alternate-scan positive and negative electrospray and a summed UV wavelength of 210–350 nm. Two liquid-phase methods were used: <i>Formic</i>—Sunfire C18 column (100 mm × 19 mm, 5 μm packing diameter, 20 mL/min flow rate) or Sunfire C18 column (150 mm × 30 mm, 5 μm packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases as (A) H<sub>2</sub>O containing 0.1% volume/volume (v/v) formic acid and (B) acetonitrile containing 0.1% (v/v) formic acid. <i>High pH</i>—Xbridge C18 column (100 mm × 19 mm, 5 μm packing diameter, 20 mL/min flow rate) or Xbridge C18 column (150 mm × 30 mm, 5 μm packing diameter, 40 mL/min flow rate). Gradient elution at ambient temperature with the mobile phases as (A) 10 mM aqueous ammonium bicarbonate solution, adjusted to pH 10 with 0.88 M aqueous ammonia, and (B) acetonitrile. NMR spectra were recorded at ambient temperature (unless otherwise stated) using standard pulse methods on any of the following spectrometers and signal frequencies: Bruker AV-400 (<sup>1</sup>H = 400 MHz, <sup>13</sup>C = 101 MHz,), Bruker AV-600 (<sup>1</sup>H = 600 MHz, <sup>13</sup>C = 150 MHz,), or Bruker AV4 700 MHz spectrometer (<sup>1</sup>H = 700 MHz, <sup>13</sup>C = 176 MHz). Chemical shifts are referenced to trimethylsilane (TMS) or the residual solvent peak and are reported in parts per million. Coupling constants are quoted to the nearest 0.1 Hz, and multiplicities are given by the following abbreviations and combinations thereof: s (singlet), δ (doublet), t (triplet), q (quartet), quin (quintet), sxt (sextet), m (multiplet), br. (broad). The purity of synthesized compounds was determined by an LCMS analysis. All compounds for biological testing were more than 95% pure.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> N-(2,2-Diphenylethyl)-4-methoxy-3,5-dimethyl-N-(2-(methylamino)-2-oxoethyl)benzamide, trifluoroacetic acid salt (8)</h4><div class="NLM_p last">2-Amino-<i>N</i>-methylacetamide (132 mg, 1.50 mmol) was dissolved in methanol (1 mL) and stirred with a basic anion exchange resin (hydroxide) for 1 h, after which the mixture was filtered, and the solvent was removed in vacuo to give a clear oil. This was dissolved in 1,2-dichloroethane (1 mL) with a few drops of methanol. To this was added 2,2-diphenylacetaldehyde (196 mg, 1.00 mmol) and sodium triacetoxyborohydride (212 mg, 1.00 mmol) along with acetic acid. The reaction was stirred at room temperature (rt) for 5 h, after which the reaction was dried in vacuo, and the resulting amine was purified by Gilson preparatory (prep) high-performance liquid chromatography (HPLC) to give amine A. To a solution of 4-methoxy-3,5-dimethylbenzoic acid in dichloromethane (DCM, 1 mL) was added a solution of hexafluorophosphate azabenzotriazole tetramethyl uronium (HATU) (456 mg, 1.20 mmol) in dimethylformamide (DMF) (0.5 mL), and the reaction was stirred at rt for 10 min, after which amine A was added along with <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) (0.524 mL, 3.00 mmol). The reaction was stirred at rt for 4 h, after which the reactions were concentrated and purified by Gilson prep HPLC to give <i>N</i>-(2,2-diphenylethyl)-4-methoxy-3,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)benzamide, trifluoroacetic acid salt as a colorless gum (13 mg, 2% yield).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Prep Conditions</h4><div class="NLM_p">Gilson Prep HPLC system, Sunfire C8 OBD 5 μM 30 × 50 mm size column. Flow rate 40 mL/min. Buffer Amine A—10–40% Water 0.01% TFA/MeCN 0.01% TFA, 1% H<sub>2</sub>O; Buffer product—25–65% Water 0.01% TFA/MeCN 0.01% TFA, 1% H<sub>2</sub>O.</div><div class="NLM_p"><sup>1</sup>H NMR (deuterated dimethyl sulfoxide (DMSO-<i>d</i><sub>6</sub>), 400 MHz) δ 7.2–7.4 (m, 11H), 6.77 (s, 2H), 4.43 (t, 1H, <i>J</i> = 7.7 Hz), 4.07 (d, 2H, <i>J</i> = 7.8 Hz), 3.7–3.7 (s, 2H), 3.70 (s, 3H), 2.61 (d, 3H, <i>J</i> = 4.6 Hz), 2.20 (s, 6H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.09 min, [M + H]<sup>+</sup> = 431.3.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (10)</h4><div class="NLM_p">A solution of paraformaldehyde (28.4 mg, 0.897 mmol), 2,2-diphenylethan-1-amine (61.5 mg, 0.299 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50.0 mg, 0.299 mmol), and isocyanomethane (0.024 mL, 0.450 mmol) in MeOH (2 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 1 h under microwave irradiation, then cooled to rt and diluted with water. The aqueous phase was extracted three times with ethyl acetate (EtOAc). The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, followed by a saturated NaHCO<sub>3</sub> aqueous solution, passed through a hydrophobic frit and concentrated in vacuo to a yellow oil. The purification of this residue by mass-directed auto preparatory HPLC (MDAP) (high pH method) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (96.6 mg, 77%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ 7.4–7.6 (m, 11H), 7.29 (br s, 1H), 4.5–4.7 (m, 1H), 4.38 (br d, 2H, <i>J</i> = 7.6 Hz), 4.1–4.3 (m, 2H), 3.72 (s, 3H), 2.96 (s, 3H), 2.29 (s, 3H). Exchangeable not observed.</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.86 min, [M + H]<sup>+</sup> = 418.1.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> N-(2,2-Diphenylethyl)-N-(1-(ethylamino)-1-oxopropan-2-yl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (11)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (64.9 mg, 0.329 mmol), acetaldehyde (0.050 mL, 0.897 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and isocyanomethane (0.027 mL, 0.449 mmol) in MeOH (0.5 mL) was stirred at 100 °C for 45 min under microwave irradiation. The mixture was diluted to 1 mL with DMSO and purified by MDAP (high-pH method). The residue was further purified by MDAP (high-pH method) to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(1-(methylamino)-1-oxopropan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (29 mg, 0.067 mmol, 22% yield<i>)</i> as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.35 (d, 1H, <i>J</i> = 2.5 Hz), 7.2–7.3 (m, 10H), 7.1–7.2 (m, 1H), 7.04 (dd, 1H, <i>J</i> = 1.3, 2.5 Hz), 4.43 (dd, 1H, <i>J</i> = 6.3, 8.3 Hz), 4.35 (q, 1H, <i>J</i> = 7.1 Hz), 4.18 (dd, 1H, <i>J</i> = 6.3, 14.3 Hz), 3.99 (dd, 1H, <i>J</i> = 8.3, 14.3 Hz), 2.83 (s, 3H), 2.62 (d, 3H, <i>J</i> = 4.8 Hz), 2.02 (s, 3H), 1.20 (d, 3H, <i>J</i> = 7.3 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): t<sub>R</sub> = 0.90 min, [M + H]<sup>+</sup> = 432.2.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> N-(2,2-Diphenylethyl)-N-(1-(ethylamino)-1-oxopropan-2-yl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide enantiomers (12, 13)</h4><div class="NLM_p"><i>N</i>-(2,2-Diphenylethyl)-<i>N</i>-(1-(ethylamino)-1-oxopropan-2-yl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide, 11 was purified by chiral stationary phase HPLC (column: Chiralcel OD-H (250 × 30 mm, 5 μm), flow rate: 30 mL/min, detection wavelength: 215 nm, solvents: 15% EtOH/heptane). Two enantiomers eluted, the first being the enantiomer compound <b>12</b> (87 mg, 35%) and the second being the enantiomer compound <b>13</b> (84 mg, 34%). The bulked mix fractions were concentrated in vacuo and reprocessed and combined with their respective products using the preparative method described above.</div><div id="sec4_1_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> (12)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.3–7.4 (m, 1H), 7.2–7.3 (m, 6H), 7.2–7.2 (m, 4H), 7.15 (br s, 1H), 7.03 (dd, 1H, <i>J</i> = 1.0, 2.5 Hz), 4.42 (dd, 1H, <i>J</i> = 6.0, 8.1 Hz), 4.34 (q, 1H, <i>J</i> = 7.1 Hz), 4.1–4.2 (m, 1H), 3.9–4.0 (m, 1H), 3.42 (s, 3H), 2.61 (d, 3H, <i>J</i> = 4.5 Hz), 2.01 (s, 3H), 1.18 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.91 min, [M + H]+ = 432..4</div></div><div id="sec4_1_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> (13)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.35 (d, 1H, <i>J</i> = 2.5 Hz), 7.2–7.3 (m, 6H), 7.2–7.2 (m, 4H), 7.15 (br s, 1H), 7.03 (dd, 1H, <i>J</i> = 1.0, 2.5 Hz), 4.42 (dd, 1H, <i>J</i> = 6.0, 8.1 Hz), 4.34 (q, 1H, <i>J</i> = 7.1 Hz), 4.1–4.2 (m, 1H), 3.98 (dd, 1H, <i>J</i> = 8.1, 14.1 Hz), 3.42 (s, 3H), 2.61 (d, 3H, <i>J</i> = 5.0 Hz), 2.02 (s, 3H), 1.19 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): t<sub>R</sub> = 0.91 min, [M + H]+ = 432.4.</div></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(1-(methylamino)-1-oxobutan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (14)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (78 mg, 0.37 mmol), isocyanomethane (0.030 mL, 0.510 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (60 mg, 0.34 mmol), and propionaldehyde (0.075 mL, 1.00 mmol) in MeOH (2 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiation. The reaction was cooled to rt and diluted with DMSO (to total volume of 1 mL). The solution was purified by MDAP (Formic method) to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(1-(methylamino)-1-oxobutan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (56 mg, 37%) as a yellow solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 7.2–7.3 (m, 12H), 7.01 (dd, 1H, <i>J</i> = 1.5, 2.5 Hz), 4.43 (t, 1H, <i>J</i> = 7.6 Hz), 4.17 (t, 1H, <i>J</i> = 7.1 Hz), 4.11 (dd, 2H, <i>J</i> = 2.0, 7.1 Hz), 3.41 (s, 3H), 2.62 (d, 3H, <i>J</i> = 5.0 Hz), 2.01 (s, 3H), 1.8–1.9 (m, 1H), 1.5–1.6 (m, 1H), 0.78 (t, 3H, <i>J</i> = 7.6 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.00 min, [M + H]<sup>+</sup> = 446.2.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(3-methyl-1-(methylamino)-1-oxobutan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (15)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (100 mg, 0.507 mmol), isobutyraldehyde (0.093 mL, 1.00 mmol), and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (67.8 mg, 0.406 mmol) in MeOH (1 mL) with a spatula of activated molecular sieves was treated with isocyanomethane (0.027 mL, 0.510 mmol), and the resulting mixture was stirred at 100 °C for 1 h under microwave irradiation. The reaction was then cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc. The combined organics were washed with water, passed through a hydrophobic frit, and concentrated in vacuo. The resulting solid was taken up in 1:1 MeOH/DMSO (2 mL). Heating was required to obtain a solution, and a solid crystallized on cooling, which was removed by filtration and dried to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(3-methyl-1-(methylamino)-1-oxobutan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (130 mg, 56%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 7.5–7.6 (m, 1H), 7.1–7.3 (m, 10H), 7.11 (d, 1H, <i>J</i> = 2.0 Hz), 6.95 (dd, 1H, <i>J</i> = 1.3, 2.3 Hz), 4.49 (t, 1H, <i>J</i> = 7.6 Hz), 4.20 (dd, 2H, <i>J</i> = 2.0, 7.6 Hz), 3.86 (d, 1H, <i>J</i> = 10.6 Hz), 3.4–3.4 (m, 3H), 2.66 (d, 3H, <i>J</i> = 4.5 Hz), 2.3–2.4 (m, 1H), 2.0–2.0 (m, 3H), 0.83 (dd, 6H, <i>J</i> = 6.5, 10.6 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.08 min, [M + H]<sup>+</sup> = 460.4.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> N-(3,3-Dimethyl-1-(methylamino)-1-oxobutan-2-yl)-N-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (16)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (100 mg, 0.507 mmol), pivalaldehyde (0.112 mL, 1.01 mmol), and 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (67.8 mg, 0.406 mmol) in MeOH (1 mL) with a spatula of activated molecular sieves was treated with isocyanomethane (0.027 mL, 0.510 mmol), and the resulting mixture was stirred at 100 °C for 1 h under microwave irradiation. The reaction was then cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc. The combined organics were washed with water, passed through a hydrophobic frit, and concentrated in vacuo to a give white solid that, after a trituration with MeOH, gave <i>N</i>-(3,3-dimethyl-1-(methylamino)-1-oxobutan-2-yl)-<i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (139 mg, 58%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.6–7.7 (m, 1H), 7.1–7.3 (m, 10H), 6.9–6.9 (m, 1H), 6.83 (d, 1H, <i>J</i> = 2.0 Hz), 4.7–4.7 (m, 1H), 4.4–4.5 (m, 1H), 4.2–4.3 (m, 2H), 3.35 (s, 3H), 2.6–2.7 (m, 3H), 2.0–2.0 (m, 3H), 1.0–1.0 (m, 9H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.14 min, [M + H]<sup>+</sup> = 474.4.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> N-(1-Cyclopropyl-2-(methylamino)-2-oxoethyl)-N-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (17)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (61.5 mg, 0.299 mmol), cyclopropanecarbaldehyde (0.07 mL, 0.90 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and isocyanomethane (0.024 mL, 0.450 mmol) in MeOH (1.7 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 1 h under microwave irradiation. The reaction was then cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, passed through a hydrophobic frit, and concentrated in vacuo. The purification by MDAP (high-pH method) gave <i>N</i>-(1-cyclopropyl-2-(methylamino)-2-oxoethyl)-<i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (65.7 mg, 48%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ 7.3–7.4 (m, 12H), 4.74 (t, 1H, <i>J</i> = 6.6 Hz), 4.4–4.7 (m, 2H), 3.86 (d, 1H, <i>J</i> = 10.6 Hz), 3.63 (s, 3H), 2.97 (s, 3H), 2.26 (s, 3H), 1.55 (br s, 1H), 0.8–1.0 (m, 2H), 0.5–0.7 (m, 2H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.96 min, [M + H]<sup>+</sup> = 458.1.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> N-(1-Cyclobutyl-2-(methylamino)-2-oxoethyl)-N-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (18)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (59.0 mg, 0.299 mmol), cyclobutanecarbaldehyde (0.07 mL, 0.90 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and isocyanomethane (0.027 mL, 0.450 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiation. The reaction was then cooled to rt and diluted with water (20 mL). The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. The purification of this residue by flash chromatography on silica gel (25 g column, gradient: 0–8% 2 M NH<sub>3</sub> in MeOH in DCM) gave a product that was further purified by MDAP (method high pH) to give <i>N</i>-(1-cyclobutyl-2-(methylamino)-2-oxoethyl)-<i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (32 mg, 23%) as a clear orange solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.1–7.3 (m, 12H), 6.98 (dd, 1H, <i>J</i> = 1.0, 2.5 Hz), 4.42 (t, 1H, <i>J</i> = 7.1 Hz), 4.34 (d, 1H, <i>J</i> = 10.6 Hz), 4.0–4.2 (m, 2H), 3.40 (s, 3H), 2.62 (d, 3H, <i>J</i> = 5.0 Hz), 2.01 (s, 3H), 1.6–1.9 (m, 7H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.98 min, [M + H]<sup>+</sup> = 472.3.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(2-(methylamino)-1-(oxetan-3-yl)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (19)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (59.0 mg, 0.299 mmol), oxetane-3-carbaldehyde (77 mg, 0.90 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50.0 mg, 0.299 mmol), and isocyanomethane (0.027 mL, 0.450 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiation. The reaction was then cooled to rt. Because of a low conversion, further 2,2-diphenylethan-1-amine (59.0 mg, 0.299 mmol), oxetane-3-carbaldehyde (77 mg, 0.90 mmol), and isocyanomethane (0.027 mL, 0.450 mmol) were added, and the reaction was stirred again at 100 °C for 45 min under microwave irradiation. The reaction was cooled to rt and diluted with water (30 mL). The aqueous phase was extracted three times with DCM. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue by MDAP (three injections, method high pH) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-(methylamino)-1-(oxetan-3-yl)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (53 mg, 37%) as an off white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.1–7.4 (m, 12H), 7.0–7.1 (m, 1H), 4.84 (d, 1H, <i>J</i> = 10.6 Hz), 4.60 (dd, 1H, <i>J</i> = 6.0, 7.6 Hz), 4.41 (dd, 1H, <i>J</i> = 6.5, 8.1 Hz), 0.00 (t, 1H, <i>J</i> = 7.1 Hz), 0.00 (t, 2H, <i>J</i> = 6.0 Hz), 3.9–4.1 (m, 2H), 3.5–3.6 (m, 1H), 3.4–3.4 (m, 3H), 2.6–2.6 (m, 3H), 2.0–2.0 (m, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.83 min, [M + H]<sup>+</sup> = 474.3.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(2-(methylamino)-2-oxo-1-(tetrahydrofuran-3-yl)ethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (20)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (59.0 mg, 0.299 mmol), tetrahydrofuran-3-carbaldehyde (50% w/w in water, 180 mg, 0.897 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and isocyanomethane (23.6 μL, 0.449 mmol) in MeOH (0.6 mL) was stirred at 80 °C for 1.5 h under microwave irradiation. The reaction was then cooled to rt and concentrated in vacuo. A purification by MDAP (high pH) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxo-1-(tetrahydrofuran-3-yl)ethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (71 mg, 49%) as a glassy solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 7.4–7.5 (m, 1H), 7.1–7.3 (m, 11H), 7.0–7.0 (m, 1H), 4.1–4.3 (m, 3H), 3.5–3.8 (m, 3H), 3.4–3.4 (m, 3H), 3.3–3.4 (m, 1H), 2.9–3.0 (m, 1H), 2.6–2.7 (m, 3H), 2.01 (d, 3H, <i>J</i> = 3.0 Hz), 1.8–2.0 (m, 2H), 1.4–1.6 (m, 1H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.92 min, [M + H]<sup>+</sup> = 488.2.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Chiral The purification of N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(2-(methylamino)-2-oxo-1-(tetrahydrofuran-3-yl)ethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (21, 22, 23, 24)</h4><div class="NLM_p">The four isomers were separated by chiral chromatography. Analytical Method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 5 uL of the solution was injected onto the column, eluting with 70% EtOH (+0.2% isopropylamine) in heptane; Flow rate 1.0 mL/min, wavelength 230 nm; Column 4.6 mm id × 25 cm Chiralpak IC. Preparative method: Approximately 71 mg of material was dissolved in EtOH (2 mL) and injected onto the column, eluting with 70% EtOH (+0.2% isopropylamine) in heptane (+0.2% isopropylamine); Flow rate: 30 mL/min; Wavelength 215 nm; Column 30 mm × 25 cm Chiralpak IC (5 uM). Fractions from 15.5 to 20 min were bulked and labeled “peak 1” (30 mg); Fractions from 36 to 41 min were bulked and purified by MDAP (high pH) to give compound <b>24</b> (6 mg, 8%); Fractions from 44 to 50 min were bulked and purified by MDAP (high pH) to give compound <b>23</b> (7 mg, 9%). In order to resolve the two isomers in peak 1 an alternative method was developed. Analytical Method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 5 uL of this was injected onto the column, eluting with 50% EtOH (+0.2% isopropylamine) in heptane; Flow rate 1.0 mL/min, wavelength 230 nm; Column 4.6 mm id × 25 cm Chiralpak ID. Preparative method: Approximately 30 mg of material was dissolved in EtOH (1 mL) and injected onto the column, eluting with 45% EtOH (+0.2% isopropylamine) in heptane (+0.2% isopropylamine); Flow rate: 30 mL/min; Wavelength 215 nm; Column 30 mm × 25 cm Chiralpak ID (5 uM). Fractions from 23 to 32 min were bulked to give compound <b>21</b> (10 mg, 14%); Fractions from 38 to 52 min were bulked to give compound <b>22</b> (8 mg, 11%). All compounds were obtained as orange gums.</div><div id="sec4_1_13_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> (21)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.1–7.5 (m, 12H), 7.00 (qd, 1H, <i>J</i> = 1.1, 2.4 Hz), 4.43 (t, 1H, <i>J</i> = 7.0 Hz), 4.1–4.2 (m, 3H), 3.7–3.7 (m, 1H), 3.62 (dd, 1H, <i>J</i> = 7.8, 14.8 Hz), 3.53 (dd, 1H, <i>J</i> = 6.8, 8.8 Hz), 3.40 (s, 3H), 3.34 (dd, 1H, <i>J</i> = 5.8, 9.0 Hz), 2.9–3.0 (m, 1H), 2.65 (d, 3H, <i>J</i> = 4.5 Hz), 2.01 (s, 3H), 1.9–2.0 (m, 1H), 1.4–1.5 (m, 1H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.92 min, [M + H]<sup>+</sup> = 488.2.</div></div><div id="sec4_1_13_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> (22)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.4–7.5 (m, 1H), 7.1–7.3 (m, 11H), 7.0–7.0 (m, 1H), 4.44 (t, 1H, <i>J</i> = 7.0 Hz), 4.1–4.2 (m, 3H), 3.7–3.7 (m, 1H), 3.62 (dd, 1H, <i>J</i> = 7.8, 14.6 Hz), 3.53 (dd, 1H, <i>J</i> = 6.8, 9.0 Hz), 3.34 (dd, 1H, <i>J</i> = 5.5, 9.0 Hz), 2.9–3.0 (m, 1H), 2.82 (s, 3H), 2.65 (d, 3H, <i>J</i> = 4.8 Hz), 2.01 (s, 3H), 1.9–2.0 (m, 1H), 1.4–1.5 (m, 1H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.93 min, [M + H]<sup>+</sup> = 488.3.</div></div><div id="sec4_1_13_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> (23)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.7–7.8 (m, 1H), 7.1–7.4 (m, 10H), 7.04 (dd, 1H, <i>J</i> = 1.3, 2.5 Hz), 4.4–4.5 (m, 1H), 4.2–4.3 (m, 1H), 4.1–4.2 (m, 1H), 3.7–3.8 (m, 1H), 3.6–3.7 (m, 1H), 3.5–3.5 (m, 1H), 3.3–3.3 (m, 1H), 2.9–3.0 (m, 1H), 2.8–2.9 (m, 3H), 2.6–2.7 (m, 3H), 2.0–2.0 (m, 3H), 1.9–2.0 (m, 1H), 1.8–1.9 (m, 1H), 1.7–1.8 (m, 1H), 1.5–1.6 (m, 1H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.92 min, [M + H]<sup>+</sup> = 488.2.</div></div><div id="sec4_1_13_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> (24)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.1–8.2 (m, 1H), 7.7–7.8 (m, 1H), 7.1–7.4 (m, 10H), 7.0–7.0 (m, 1H, <i>J</i> = 1.1, 2.4 Hz), 4.42 (d, 1H, <i>J</i> = 9.3 Hz), 4.1–4.3 (m, 1H), 3.7–3.8 (m, 2H), 3.6–3.7 (m, 2H), 3.52 (s, 3H), 3.51 (dd, 1H, <i>J</i> = 6.8, 8.8 Hz), 2.66 (br d, 2H, <i>J</i> = 2.0 Hz), 2.64 (s, 3H), 2.08 (s, 3H), 1.9–2.0 (m, 1H), 1.7–1.8 (m, 1H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): t<sub>R</sub> = 0.92 min, [M + H]<sup>+</sup> = 488.2.</div></div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(2-methyl-1-(methylamino)-1-oxopropan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (25)</h4><div id="sec4_1_14_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Step 1</h5><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (100 mg, 0.507 mmol) and acetone (29.4 mg, 0.507 mmol) in MeOH (1 mL) was stirred at rt for 5 min with activated molecular sieves. 1,5-Dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (85 mg, 0.51 mmol) and (isocyanomethyl)trimethylsilane (0.072 mL, 0.510 mmol) were then added, and the resulting mixture was stirred at rt for 16 h before being concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (10 g column, gradient 0–100% EtOAc in cyclohexane) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-methyl-1-oxo-1-(((trimethylsilyl)methyl)amino)propan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (160 mg, 61%) as a yellow gum.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.2–7.3 (m, 12H), 7.0–7.0 (m, 1H), 4.0–4.1 (m, 1H), 3.37 (s, 2H), 3.29 (s, 3H), 2.6–2.7 (m, 1H), 2.3–2.3 (m, 1H), 1.97 (s, 3H), 1.51 (s, 6H), −0.10 (s, 9H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.17 min, [M + H]<sup>+</sup> = 518.3.</div></div><div id="sec4_1_14_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Step 2</h5><div class="NLM_p">A solution of <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-methyl-1-oxo-1-(((trimethylsilyl)methyl)amino)propan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (155 mg, 0.300 mmol) in THF (1 mL) was treated at rt with tetra-<i>n</i>-butylammonium fluoride (TBAF) (1 M in THF, 0.6 mL, 0.6 mmol), and the resulting mixture was stirred at rt for 16 h. TBAF (1 M in THF, 0.6 mL, 0.6 mmol) was further added, and the mixture was stirred at rt for 4 h and then at 50 °C for 1 h, before being cooled to rt and diluted with water. The aqueous phase was extracted with EtOAc, and the organic phase was washed with brine, passed through a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (10 g column, gradient: 0–5% (2 N NH<sub>3</sub> in MeOH) in DCM) gave a residue that was further purified by MDAP (formic method) to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-methyl-1-(methylamino)-1-oxopropan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (5 mg, 4%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4,</sub> 400 MHz) δ 7.5–7.7 (m, 1H), 7.0–7.4 (m, 12H), 4.3–4.4 (m, 3H), 3.4–3.5 (m, 3H), 2.7–2.8 (m, 3H), 2.0–2.2 (m, 3H), 1.6–1.7 (m, 6H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): t<sub>R</sub> = 0.90 min, [M + H]<sup>+</sup> = 446.1.</div></div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(1-(methylcarbamoyl)cyclopropyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (26)</h4><div class="NLM_p">A solution of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50.0 mg, 0.299 mmol), 2,2-diphenylethan-1-amine (650 mg, 0.329 mmol), and (1-ethoxycyclopropoxy)trimethylsilane (180 μL, 0.897 mmol) in trifluoroethanol (0.6 mL) was treated with a spatula of activated molecular sieves followed by isocyanomethane (24 μL, 0.45 mmol). The resulting mixture was stirred at 75 °C for 1.5 h under microwave irradiation. The reaction was diluted to 1 mL with 50% MeOH/DMSO and purified by MDAP (high pH) to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(1-(methylcarbamoyl)cyclopropyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (170 mg, 43%) as a white foam.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.5–7.6 (m, 1H), 7.2–7.3 (m, 11H), 6.7–6.8 (m, 1H), 4.55 (t, 1H, <i>J</i> = 7.1 Hz), 4.11 (d, 2H, <i>J</i> = 7.1 Hz), 3.44 (s, 3H), 2.83 (s, 3H), 2.02 (s, 3H), 1.25 (q, 2H, <i>J</i> = 4.7 Hz), 0.79 (q, 2H, <i>J</i> = 4.5 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.96 min, [M + H]<sup>+</sup> = 444.2.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(1-(methylcarbamoyl)cyclobutyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (27)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (61.5 mg, 0.299 mmol), cyclobutanone (0.068 mL, 0.900 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and isocyanomethane (0.024 mL, 0.450 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 1 h under microwave irradiation. The reaction was then cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc, and the combined organics were washed with brine, passed through a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by MDAP (formic method) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(1-(methylcarbamoyl)cyclobutyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (28 mg, 20%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 6.7–6.7 (m, 1H), 6.3–6.5 (m, 8H), 6.2–6.3 (m, 4H), 3.2–3.4 (m, 3H), 2.74 (s, 3H), 1.9–2.0 (m, 2H), 1.85 (d, 3H, <i>J</i> = 5.0 Hz), 1.5–1.6 (m, 2H), 1.33 (s, 3H), 0.9–1.0 (m, 2H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.03 min, [M + H]<sup>+</sup> = 458.2.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(1-(methylcarbamoyl)cyclopentyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (28)</h4><div class="NLM_p">A solution of cyclopentanone (0.08 mL, 0.90 mmol), 2,2-diphenylethan-1-amine (61.5 mg, 0.299 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and isocyanomethane (0.024 mL, 0.450 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 1 h under microwave irradiation. The reaction was then cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. The purification of this residue by flash chromatography on silica gel (25 g column, gradient: 0–8% 2 M NH<sub>3</sub> in MeOH in DCM) gave a second residue, which was further purified by MDAP (method formic) to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(1-(methylcarbamoyl)cyclopentyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (57.5 mg, 36%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.1–7.3 (m, 12H), 7.0–7.0 (m, 1H), 4.31 (t, 1H, <i>J</i> = 7.1 Hz), 4.23 (d, 2H, <i>J</i> = 7.1 Hz), 2.83 (s, 3H), 2.60 (d, 3H, <i>J</i> = 4.5 Hz), 2.02 (s, 3H), 1.97 (br s, 3H), 1.6–1.7 (m, 5H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.02 min, [M + H]<sup>+</sup> = 472.2.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> N-(2-Amino-2-oxoethyl)-N-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (29)</h4><div id="sec4_1_18_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> Step 1</h5><div class="NLM_p">A solution of paraformaldehyde (64.7 mg, 2.15 mmol), 2,2-diphenylethan-1-amine (148 mg, 0.718 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (120 mg, 0.718 mmol), and (2-isocyanopropan-2-yl)benzene (0.206 mL, 1.08 mmol) in MeOH (3 mL) with a spatula of activated molecular sieves was stirred at 80 °C for 1 h under microwave irradiation. The reaction was then cooled to rt and diluted with water (20 mL). The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. The residue obtained was purified by flash chromatography on silica gel (25 g column, gradient: 0–5% 2 M NH<sub>3</sub> in MeOH in DCM) to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-<i>N</i>-(2-oxo-2-((2-phenylpropan-2-yl)amino)ethyl)-1,6-dihydropyridine-3-carboxamide (263 mg, 65%) as a colorless oil.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.6–7.6 (m, 1H), 7.1–7.4 (m, 16H), 6.9–6.9 (m, 1H), 4.44 (t, 1H, <i>J</i> = 7.6 Hz), 4.04 (d, 2H, <i>J</i> = 8.1 Hz), 3.91 (s, 2H), 3.38 (s, 3H), 1.98 (s, 3H), 1.58 (s, 6H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.15 min, [M + H]<sup>+</sup> = 522.4.</div></div><div id="sec4_1_18_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Step 2</h5><div class="NLM_p">A solution of <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-<i>N</i>-(2-oxo-2-((2-phenylpropan-2-yl)amino)ethyl)-1,6-dihydropyridine-3-carboxamide (263 mg, 0.469 mmol) in TFA (2 mL) was stirred at 50 °C for 1.5 h and then was cooled to rt and left overnight. The reaction mixture was diluted with water (10 mL) and neutralized with a saturated NaHCO<sub>3</sub> aqueous solution (10 mL) and then was extracted three times with EtOAc. The combined organics were dried using a hydrophobic frit and concentrated in vacuo. The purification of the residue obtained by MDAP (method high pH) gave <i>N</i>-(2-amino-2-oxoethyl)-<i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (117 mg, 62%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.2–7.3 (m, 11H), 6.96 (dd, 1H, <i>J</i> = 1.1, 2.3 Hz), 6.8–6.9 (m, 2H), 4.43 (t, 1H, <i>J</i> = 7.7 Hz), 4.08 (d, 2H, <i>J</i> = 7.8 Hz), 3.82 (s, 2H), 3.40 (s, 3H), 1.99 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.82 min, [M + H]<sup>+</sup> = 404.2.</div></div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> N-(2,2-Diphenylethyl)-N-ethyl-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (30)</h4><div id="sec4_1_19_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> Step 1</h5><div class="NLM_p">A suspension of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (1.00 g, 5.98 mmol) and HATU (2.50 g, 6.58 mmol) in DMF (20 mL) at rt was treated with DIPEA (1.04 mL, 5.98 mmol), and the resulting mixture was stirred at for 5 min before 2,2-diphenylethan-1-amine (1.30 g, 6.58 mmol) was added. The resulting mixture was stirred at rt for 1 h and then was diluted with a 10% w/w citric acid aqueous solution and extracted with EtOAc. The organic phase was washed with a 10% w/w citric acid aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (50 g column, gradient: 0–100% EtOAc in cyclohexane) gave a residue that was further purified by flash chromatography on silica gel (50 g column, gradient: 0–50% EtOAc in cyclohexane) to give <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (1.67 g, 81%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.20 (t, 1H, <i>J</i> = 5.5 Hz), 8.11 (d, 1H, <i>J</i> = 2.5 Hz), 7.66 (dd, 1H, <i>J</i> = 1.0, 2.5 Hz), 7.3–7.4 (m, 8H), 7.1–7.2 (m, 2H), 4.36 (t, 1H, <i>J</i> = 7.8 Hz), 3.86 (dd, 2H, <i>J</i> = 5.8, 7.8 Hz), 3.45 (s, 3H), 1.99 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.00 min, [M + H]<sup>+</sup> = 347.0.</div></div><div id="sec4_1_19_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> Step 2</h5><div class="NLM_p">A mixture of <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (50 mg, 0.14 mmol), iodoethane (0.023 mL, 0.290 mmol), and potassium <i>tert</i>-butoxide (32.4 mg, 0.289 mmol) in DMF (2 mL) was stirred at 100 °C for 1 h under microwave irradiations and then was cooled to rt. Further iodoethane (0.023 mL, 0.290 mmol) was added, and the mixture was again stirred at 100 °C for 1 h under microwave irradiations. Then it was cooled to rt and partitioned between water and EtOAc. The layers were separated, and the organic phase was washed with a 10% w/w LiCl aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. A purification of the residue obtained by MDAP (formic method) gave <i>N</i>-(2,2-diphenylethyl)-<i>N</i>-ethyl-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (23 mg, 43%) as a red solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.2–7.4 (m, 11H), 6.8–6.9 (m, 1H), 4.39 (t, 1H, <i>J</i> = 7.5 Hz), 4.01 (d, 2H, <i>J</i> = 7.8 Hz), 3.39 (s, 3H), 3.1–3.2 (m, 2H), 1.96 (s, 3H), 1.00 (t, 3H, <i>J</i> = 7.0 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.04 min, [M + H]<sup>+</sup> = 375.2.</div></div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> N-(1-Cyanoethyl)-N-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (31)</h4><div class="NLM_p">A suspension of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (1.00 g, 5.98 mmol) in thionyl chloride (10.0 mL, 137 mmol) was stirred at 60 °C for 2 h, during which time it became a solution. Then it was cooled to rt and concentrated in vacuo. The residue was coevaporated with toluene to give 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbonyl chloride (1.09 g, 98%) as a yellow solid, which was used in the next step without further purification.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.34 (d, 1H, <i>J</i> = 2.4 Hz), 7.68 (dd, 1H, <i>J</i> = 1.2, 2.7 Hz), 3.51 (s, 3H), 2.03 (s, 3H).</div><div class="NLM_p">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.65 min, [M + H]<sup>+</sup> = 182.3.</div><div id="sec4_1_20_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Step 2</h5><div class="NLM_p last">A solution of 2,2-diphenylacetaldehyde (0.152 mL, 0.856 mmol) and 2-aminopropanenitrile (60.0 mg, 0.856 mmol) in DCM was stirred for 5 min at rt, then a few drops of acetic acid were added, and the resulting mixture was stirred at rt for 1 h before being treated with sodium triacetoxyborohydride (363 mg, 1.71 mmol). The resulting mixture was stirred for 16 h at rt. The reaction was then diluted with water and extracted with DCM. The organic phase was dried using a hydrophobic frit and concentrated in vacuo. A purification of the residue obtained by flash chromatography on silica gel (gradient: 0–100% EtOAc in cyclohexane) gave 2-((2,2-diphenylethyl)amino)propanenitrile (102 mg, 48%) as a yellow gum. The crude product was used in the next step.</div></div><div id="sec4_1_20_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> Step 3</h5><div class="NLM_p">A suspension of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carbonyl chloride (80 mg, 0.43 mmol) and 2-((2,2-diphenylethyl)amino)propanenitrile (108 mg, 0.431 mmol) in MeCN (4 mL) was treated at rt with DIPEA (0.113 mL, 0.647 mmol); the reaction was stirred for 3 d and then was concentrated in vacuo. A purification of the residue by MDAP (method high pH) followed by a second purification by bespoke preparatory HPLC (Column: XBridge C18 150 × 30 mm, 5 μm; Solvent A: 10 mM ammonium bicarbonate in water adjusted to pH 10 with ammonia solution; Solvent B: acetonitrile; Flow Rate: 40 mL/min; Gradient: 30–100% A:B) gave <i>N</i>-(1-cyanoethyl)-<i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (59 mg, 0.148 mmol, 34.3% yield) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.41 (d, 1H, <i>J</i> = 2.4 Hz), 7.1–7.3 (m, 10H), 6.9–6.9 (m, 1H), 4.96 (q, 1H, <i>J</i> = 6.8 Hz), 4.33 (t, 1H, <i>J</i> = 7.3 Hz), 4.0–4.2 (m, 2H), 3.40 (s, 3H), 1.97 (s, 3H), 1.47 (d, 3H, <i>J</i> = 6.8 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.06 min, [M + H]<sup>+</sup> = 400.4.</div></div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> N-(2,2-Diphenylethyl)-1,5-dimethyl-6-oxo-N-(1-(tetrahydrofuran-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (32)</h4><div class="NLM_p">A solution of 1-(tetrahydrofuran-2-yl)ethan-1-amine (160 mg, 1.39 mmol) and 2,2-diphenylacetaldehyde (0.271 mL, 1.53 mmol) in DCM (3.5 mL) at rt was treated with acetic acid (0.088 mL, 1.5 mmol), and the resulting mixture was stirred for 5 min before being treated with sodium triacetoxyborohydride (884 mg, 4.17 mmol). The resulting mixture was stirred at rt for 3 h; then the reaction was quenched with the addition of a saturated NaHCO<sub>3</sub> aqueous solution (5 mL). The biphasic mixture was stirred for 1 h before being extracted with DCM (3 × 10 mL). The combined organics were dried using a phase separator and concentrated under a stream of nitrogen. The residue was dissolved in EtOAc (3 mL), and 1.5 mL of the solution was treated at rt with 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (93 mg, 0.56 mmol), HATU (238 mg, 0.625 mmol), and then DIPEA (0.109 mL, 0.625 mmol). The resulting mixture was stirred at rt for 3 h, after which DMF (1 mL) was added. The solution was then stirred at 40 °C for 1 h, cooled to rt, and left to stand for 3 d, after which it was partitioned between a saturated NaHCO<sub>3</sub> aqueous solution (10 mL) and EtOAc (10 mL). The layers were stirred and separated, and the aqueous phase was extracted with EtOAc. The combined organics were dried using a hydrophobic frit and concentrated under a stream of N<sub>2</sub>. The purification of the residue obtained by MDAP (method high pH) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-<i>N</i>-(1-(tetrahydrofuran-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (11 mg, 7%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.1–7.5 (m, 11H), 7.0–7.1 (m, 1H), 4.46 (dd, 1H, <i>J</i> = 5.9, 8.3 Hz), 4.10 (dd, 1H, <i>J</i> = 5.4, 13.7 Hz), 3.83 (dd, 1H, <i>J</i> = 8.3, 13.7 Hz), 3.5–3.7 (m, 4H), 3.44 (s, 3H), 3.22 (t, 1H, <i>J</i> = 7.3 Hz), 2.5–2.6 (m, 1H), 2.02 (s, 3H), 1.7–1.9 (m, 1H), 1.4–1.5 (m, 1H), 0.87 (d, 3H, <i>J</i> = 6.4 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.10 min, [M + H]<sup>+</sup> = 445.7.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(2-morpholinoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (33)</h4><div class="NLM_p">A solution of <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (compound <b>30</b>, step 1, 50 mg, 0.14 mmol) in DMF (4 mL) at rt was treated with sodium hydride (60% w/w in mineral oil, 11.5 mg, 0.289 mmol), and the resulting mixture was stirred for 5 min before being treated with 4-(2-chloroethyl)morpholine hydrochloride (29.5 mg, 0.159 mmol). The resulting mixture was stirred at rt for 2 h and then at 70 °C for 16 h before being cooled to rt and left to stand for 3 d. The reaction was carefully treated with water, and the aqueous phase was extracted with EtOAc. The organic phase was dried using a phase separator and concentrated in vacuo. A purification of the residue obtained by MDAP (formic method) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-morpholinoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (12 mg, 18%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ 8.2–8.3 (m, 1H), 7.2–7.4 (m, 10H), 7.0–7.1 (m, 1H), 4.4–4.5 (m, 1H), 4.22 (br d, 2H, <i>J</i> = 7.6 Hz), 3.68 (br d, 6H, <i>J</i> = 4.0 Hz), 3.54 (s, 3H), 2.4–2.7 (m, 6H), 2.1–2.2 (m, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.62 min, [M + H]<sup>+</sup> = 460.2.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> N-(2,2-Diphenylethyl)-1,5-dimethyl-N-(1-morpholinopropan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (34)</h4><div id="sec4_1_23_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> Step 1</h5><div class="NLM_p">A solution of the maleate salt of 1-morpholinopropan-2-one (180 mg, 0.694 mmol) and 2,2-diphenylethan-1-amine (137 mg, 0.694 mmol) in DCM (5 mL) was stirred at rt for 5 min before being treated with sodium triacetoxyborohydride (294 mg, 1.39 mmol). The resulting mixture was stirred at this temperature for 16 h and then was diluted with water. The aqueous phase was extracted with DCM, and the organic phase was then dried using a hydrophobic frit and concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (10 g column, gradient 0–10% (2 M NH<sub>3</sub> in MeOH) in DCM) gave <i>N</i>-(2,2-diphenylethyl)-1-morpholinopropan-2-amine (154 mg, 68%) as a colorless gum. The crude was taken to the next step.</div><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ 7.2–7.4 (m, 10H), 4.33 (dd, 1H, <i>J</i> = 6.8, 9.3 Hz), 3.5–3.8 (m, 2H), 3.3–3.4 (m, 4H), 3.2–3.3 (m, 1H), 2.3–2.4 (m, 4H), 2.2–2.3 (m, 2H), 1.23 (d, 3H, <i>J</i> = 6.5 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.59 min, [M + H]<sup>+</sup> = 325.2.</div></div><div id="sec4_1_23_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> Step 2</h5><div class="NLM_p">A mixture of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (75 mg, 0.45 mmol), HATU (205 mg, 0.538 mmol), and DIPEA (0.235 mL, 1.35 mmol) in DMF (4 mL) was stirred at rt for 5 min before <i>N</i>-(2,2-diphenylethyl)-1-morpholinopropan-2-amine (146 mg, 0.449 mmol) was added. The resulting mixture was stirred at rt for 1 h and then was diluted with water. The aqueous phase was extracted with EtOAc. The organic phase was washed with a 10% w/w LiCl aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. A purification of the residue obtained by MDAP (method formic) gave <i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(1-morpholinopropan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (130 mg, 61%) as a yellow solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.2–7.4 (m, 11H), 7.0–7.1 (m, 1H), 4.45 (dd, 1H, <i>J</i> = 6.5, 8.3 Hz), 3.8–3.9 (m, 1H), 3.7–4.0 (m, 2H), 3.51 (t, 4H, <i>J</i> = 4.5 Hz), 3.43 (s, 3H), 2.38 (dd, 1H, <i>J</i> = 8.0, 13.1 Hz), 2.2–2.3 (m, 4H), 2.16 (dd, 1H, <i>J</i> = 6.0, 13.1 Hz), 2.0–2.0 (m, 3H), 0.89 (d, 3H, <i>J</i> = 6.8 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.65 min, [M + H]<sup>+</sup> = 474.4.</div></div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> N-(1,3-Dimethoxypropan-2-yl)-N-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (35)</h4><div class="NLM_p">This compound was made in parallel to compound <b>32</b> using the same protocol, starting with 1,3-dimethoxypropan-2-amine (166 mg, 1.39 mmol) to give <i>N</i>-(1,3-dimethoxypropan-2-yl)-<i>N</i>-(2,2-diphenylethyl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (20 mg, 13%) as a white solid.</div><div class="NLM_p">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 1.14 min, [M + H]<sup>+</sup> = 449.6.</div><div class="NLM_p last"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ ppm 7.54 (br s, 1H), 7.25–7.32 (m, 8H), 7.18–7.24 (m, 2H), 7.05–7.07 (bs s, 1H), 4.37 (t, 1H), 3.89–4.02 (m, 3H), 3.41 (s, 3H), 3.21 (br s, 4H), 3.08 (s, 6H), 1.97 (s, 3H).</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> N-(2,2-Diphenylethyl)-N-(1-(ethylamino)-1-oxopropan-2-yl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (36)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (100 mg, 0.507 mmol), acetaldehyde (0.057 mL, 1.014 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (67.8 mg, 0.406 mmol), and ethyl isocyanide (28.4 mg, 0.507 mmol) in MeOH (1 mL) was stirred at 100 °C under microwave irradiation for 30 min. The reaction was cooled to rt, diluted with water, and extracted with EtOAc; the combined organics were washed with brine, passed through a hydrophobic frit, and concentrated in vacuo. This was purified by flash chromatography on silica gel (25 g column, gradient: 0–5% (2 M NH<sub>3</sub> in MeOH) in DCM, followed by a purification by MDAP (Formic Modifier) to give <i>N</i>-(2,2-diphenylethyl)-<i>N</i>-(1-(ethylamino)-1-oxopropan-2-yl)-1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxamide (86 mg, 0.193 mmol, 38% yield) as a cream solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.62 (t, 1H, <i>J</i> = 5.5 Hz), 7.44 (d, 1H, <i>J</i> = 2.0 Hz), 7.2–7.3 (m, 10H), 7.0–7.0 (m, 1H), 4.38 (dd, 1H, <i>J</i> = 6.0, 8.6 Hz), 4.26 (q, 1H, <i>J</i> = 7.1 Hz), 4.13 (dd, 1H, <i>J</i> = 6.0, 14.1 Hz), 3.94 (br dd, 1H, <i>J</i> = 8.6, 14.4 Hz), 3.41 (s, 3H), 3.0–3.1 (m, 2H), 1.98 (s, 3H), 1.07 (br d, 3H, <i>J</i> = 7.1 Hz), 0.97 (t, 3H, <i>J</i> = 7.3 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.96 min, [M + H]<sup>+</sup> = 446.1.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> N-(2,2-Diphenylethyl)-N-(1-(ethylamino)-1-oxopropan-2-yl)-5-methyl-6-oxo-1,6-dihydropyridine-3-carboxamide (37)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (64.4 mg, 0.327 mmol), acetaldehyde (0.055 mL, 0.980 mmol), 5-methyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.33 mmol), and isocyanoethane (0.022 mL, 0.33 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiations and then was cooled to rt and diluted with water (30 mL). The aqueous phase was extracted three times with EtOAc. The combined organics were washed with brine, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by MDAP (three injections, method high pH) gave <i>N</i>-(2,2-diphenylethyl)-<i>N</i>-(1-(ethylamino)-1-oxopropan-2-yl)-5-methyl-6-oxo-1,6-dihydropyridine-3-carboxamide (53 mg, 38%) as an off white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 11.58 (br s, 1H), 7.61 (t, 1H, <i>J</i> = 5.5 Hz), 7.1–7.3 (m, 11H), 7.03 (d, 1H, <i>J</i> = 1.0 Hz), 4.38 (dd, 1H, <i>J</i> = 5.5, 8.8 Hz), 4.22 (q, 1H, <i>J</i> = 6.7 Hz), 4.10 (dd, 1H, <i>J</i> = 5.5, 14.1 Hz), 3.8–3.9 (m, 1H), 3.0–3.1 (m, 2H), 1.94 (s, 3H), 0.9–1.0 (m, 6H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): t<sub>R</sub> = 0.91 min, [M + H]<sup>+</sup> = 432.2.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> N-(2,2-Diphenylethyl)-N-(1-(ethylamino)-1-oxopropan-2-yl)-1-methyl-6-oxo-1,6-dihydropyridine-3-carboxamide (38)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (62.5 mg, 0.317 mmol), acetaldehyde (0.036 mL, 0.630 mmol), 1-methyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.32 mmol), and isocyanoethane (0.022 mL, 0.320 mmol) in MeOH (1.7 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiations and then was cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc. The combined organics were washed with brine, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by MDAP (method formic) gave <i>N</i>-(2,2-diphenylethyl)-<i>N</i>-(1-(ethylamino)-1-oxopropan-2-yl)-1-methyl-6-oxo-1,6-dihydropyridine-3-carboxamide (81 mg, 59%) as a yellow gum.</div><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ 7.77 (br s, 1H), 7.2–7.4 (m, 13H), 4.4–4.5 (m, 2H), 4.25 (dd, 1H, <i>J</i> = 6.5, 14.6 Hz), 4.0–4.1 (m, 1H), 3.49 (s, 3H), 3.1–3.2 (m, 2H), 1.34 (br d, 3H, <i>J</i> = 6.0 Hz), 1.09 (t, 3H, <i>J</i> = 7.6 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.91 min, [M + H]<sup>+</sup> = 432.1.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> N-(2,2-Diphenylethyl)-4,6-dimethyl-N-(1-(methylamino)-1-oxopropan-2-yl)-5-oxo-4,5-dihydropyrazine-2-carboxamide (39)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (75 mg, 0.38 mmol), acetaldehyde (58.9 μL, 1.04 mmol), 4,6-dimethyl-5-oxo-4,5-dihydropyrazine-2-carboxylic acid (58.5 mg, 0.348 mmol), and isocyanomethane (27.4 μL, 0.522 mmol) in MeOH (0.7 mL) with a spatula of activated molecular sieves was stirred at 80 °C for 90 min under microwave irradiations and then was cooled to rt and diluted with DMSO to 0.9 mL. This solution was purified by MDAP (method high pH) to give <i>N</i>-(2,2-diphenylethyl)-4,6-dimethyl-<i>N</i>-(1-(methylamino)-1-oxopropan-2-yl)-5-oxo-4,5-dihydropyrazine-2-carboxamide (41 mg, 27%) as an off white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 7.61 (s, 1H), 7.2–7.3 (m, 4H), 7.2–7.2 (m, 3H), 7.1–7.2 (m, 3H), 7.0–7.1 (m, 1H), 4.4–4.6 (m, 2H), 4.29 (s, 1H), 4.3–4.3 (m, 1H), 3.45 (s, 3H), 2.58 (d, 3H, <i>J</i> = 5.0 Hz), 2.28 (s, 3H), 1.17 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.92 min, [M + H]<sup>+</sup> = no mass ion.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> N-(2,2-Diphenylethyl)-5-methoxy-1-methyl-N-(1-(methylamino)-1-oxopropan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (40)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (50.8 mg, 0.258 mmol), 5-methoxy-1-methyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (89% pure by LCMS, 53 mg, 0.26 mmol), isocyanomethane (0.023 mL, 0.39 mmol), and acetaldehyde (0.044 mL, 0.77 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiations then was cooled to rt. Further isocyanomethane (0.023 mL, 0.39 mmol), acetaldehyde (0.044 mL, 0.770 mmol), and 2,2-diphenylethan-1-amine (50.8 mg, 0.258 mmol) were added, and the reaction was heated at 100 °C for 45 min under microwave irradiations and then cooled to rt. This was repeated again twice before the mixture was diluted with water (30 mL) and extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution and then with a saturated NaHCO<sub>3</sub> aqueous solution, before being dried using a hydrophobic frit and concentrated in vacuo. The purification by MDAP (method high pH) gave <i>N</i>-(2,2-diphenylethyl)-5-methoxy-1-methyl-<i>N</i>-(1-(methylamino)-1-oxopropan-2-yl)-6-oxo-1,6-dihydropyridine-3-carboxamide (15 mg, 13%) as an off white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.2–7.3 (m, 12H), 6.53 (d, 1H, <i>J</i> = 2.0 Hz), 4.44 (dd, 1H, <i>J</i> = 6.0, 8.1 Hz), 4.34 (q, 1H, <i>J</i> = 7.1 Hz), 3.9–4.2 (m, 2H), 3.71 (s, 3H), 3.42 (s, 3H), 2.60 (d, 3H, <i>J</i> = 4.5 Hz), 1.16 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.86 min, [M + H]<sup>+</sup> = 448.2.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> N-(2,2-Diphenylethyl)-N-(1-(ethylamino)-1-oxopropan-2-yl)-1-methyl-6-oxopiperidine-3-carboxamide (41)</h4><div class="NLM_p">A solution of 2,2-diphenylethan-1-amine (62.8 mg, 0.318 mmol), acetaldehyde (0.054 mL, 0.950 mmol), 1-methyl-6-oxopiperidine-3-carboxylic acid (50 mg, 0.32 mmol), and isocyanoethane (0.022 mL, 0.32 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiations and then was cooled to rt and diluted with water (30 mL). The aqueous phase was extracted three times with EtOAc, and the combined organics were washed with brine, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue by MDAP (method high pH) gave a mixture of isomers that were separated by chiral chromatography.</div><div class="NLM_p">Analytical method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 20 uL of this was injected onto the column, eluting with 30% EtOH (+0.2% isopropylamine) in heptane. Flow rate 1 mL/min; Wavelength 215 nM. Column 4.6 mm id × 25 cm (R-R)-Whelk-O1. Preparative method: Approximately 35 mg of material was dissolved in EtOH (1 mL). This was injected onto the column, eluting with 25% EtOH (+0.2% isopropylamine) in heptane (+0.2% isopropylamine). Flow rate 30 mL/min; Wavelength 215 nM; Column 30 mm × 25 cm (R-R)-Whelk-O1 (5 μm). Fractions from 33 to 35 min were bulked to give the fastest eluting diastereoisomer (6 mg, 69%) as a white solid.</div><div class="NLM_p">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.94 min, [M + H]<sup>+</sup> = 436.3.</div><div class="NLM_p last"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.3–7.4 (m, 8H), 7.2–7.2 (m, 2H), 7.0–7.1 (m, 1H), 4.3–4.4 (m, 2H), 4.0–4.1 (m, 2H), 3.2–3.3 (m, 1H), 3.1–3.1 (m, 2H), 2.9–3.0 (m, 1H), 2.76 (s, 3H), 2.2–2.3 (m, 1H), 2.0–2.1 (m, 1H), 1.6–1.8 (m, 1H), 1.5–1.6 (m, 1H), 1.1–1.2 (m, 3H), 1.03 (t, 3H, <i>J</i> = 7.2 Hz).</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenylpropyl)-1,6-dihydropyridine-3-carboxamide (42)</h4><div class="NLM_p">A solution of paraformaldehyde (28.4 mg, 0.897 mmol), 2-phenylpropan-1-amine (0.044 mL, 0.300 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and isocyanomethane (0.024 mL, 0.450 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 1 h under microwave irradiation and then was cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, before being dried using a hydrophobic frit and concentrated in vacuo. The purification of the residue obtained by MDAP (method high pH) gave 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenylpropyl)-1,6-dihydropyridine-3-carboxamide (84 mg, 79%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 7.42 (d, 1H, <i>J</i> = 2.5 Hz), 7.37 (br d, 1H, <i>J</i> = 2.0 Hz), 7.3–7.3 (m, 2H), 7.2–7.2 (m, 3H), 7.1–7.1 (m, 1H), 3.90 (d, 1H, <i>J</i> = 16.1 Hz), 3.84 (d, 1H, <i>J</i> = 16.1 Hz), 3.61 (dd, 1H, <i>J</i> = 7.6, 14.1 Hz), 3.53 (dd, 1H, <i>J</i> = 8.1, 14.1 Hz), 3.43 (s, 3H), 3.15 (q, 1H, <i>J</i> = 7.1 Hz), 2.63 (d, 3H, <i>J</i> = 4.5 Hz), 2.01 (s, 3H), 1.22 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.71 min, [M + H]<sup>+</sup> = 356.0</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenylbutyl)-1,6-dihydropyridine-3-carboxamide (43)</h4><div class="NLM_p">A solution of 2-phenylbutan-1-amine (44.6 mg, 0.284 mmol), paraformaldehyde (26.9 mg, 0.852 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.28 mmol), and isocyanomethane (0.027 mL, 0.43 mmol) in MeOH (0.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiations and then was cooled to rt. Further isocyanomethane (0.027 mL, 0.43 mmol), 2-phenylbutan-1-amine (44.6 mg, 0.284 mmol), and paraformaldehyde (26.9 mg, 0.852 mmol) were added, and the reaction mixture was stirred at 100 °C for 45 min under microwave irradiations before being cooled to rt and diluted with water. The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, before being dried using a hydrophobic frit and concentrated in vacuo. The purification of the residue obtained by MDAP (method high pH) gave 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenylbutyl)-1,6-dihydropyridine-3-carboxamide (44 mg, 42%) as a yellow gum.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 7.98 (s, 1H), 7.37 (d, 1H, <i>J</i> = 2.5 Hz), 7.3–7.4 (m, 1H), 7.3–7.3 (m, 1H), 7.2–7.3 (m, 1H), 7.1–7.2 (m, 2H), 7.04 (dd, 1H, <i>J</i> = 1.3, 2.3 Hz), 3.7–3.9 (m, 2H), 3.7–3.7 (m, 1H), 3.5–3.6 (m, 1H), 3.43 (s, 3H), 2.8–3.0 (m, 2H), 2.63 (d, 2H, <i>J</i> = 4.5 Hz), 2.01 (d, 3H, <i>J</i> = 1.0 Hz), 1.6–1.8 (m, 1H), 1.5–1.6 (m, 1H), 0.77 (t, 3H, <i>J</i> = 7.3 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.80 min, [M + H]<sup>+</sup> = 370.4.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 1,5-Dimethyl-N-(3-methyl-2-phenylbutyl)-N-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (44)</h4><div class="NLM_p">A solution of 3-methyl-2-phenylbutan-1-amine (48.8 mg, 0.284 mmol), paraformaldehyde (26.9 mg, 0.852 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.28 mmol), and isocyanomethane (0.027 mL, 0.430 mmol) with a spatula of activated molecular sieves was stirred at 100 °C for 45 min under microwave irradiations and then was cooled to rt and diluted with water (15 mL). The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, before being dried using a hydrophobic frit and concentrated in vacuo. The purification of the residue obtained by MDAP (method formic) gave 1,5-dimethyl-<i>N</i>-(3-methyl-2-phenylbutyl)-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (71 mg, 65%) as an off white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.0–7.3 (m, 7H), 6.9–7.0 (m, 1H), 3.84 (dd, 1H, <i>J</i> = 5.0, 13.6 Hz), 3.6–3.8 (m, 3H), 3.40 (s, 3H), 2.7–2.8 (m, 1H), 2.61 (d, 3H, <i>J</i> = 4.5 Hz), 1.99 (s, 3H), 1.8–1.9 (m, 1H), 0.95 (d, 3H, <i>J</i> = 6.5 Hz), 0.69 (d, 3H, <i>J</i> = 6.5 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.86 min, [M + H]<sup>+</sup> = 384.3.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> N-(2-Ethoxy-2-phenylethyl)-1,5-dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (45)</h4><div class="NLM_p">A solution of 2-ethoxy-2-phenylethan-1-amine (109 mg, 0.658 mmol), paraformaldehyde (35.9 mg, 1.20 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (100 mg, 0.598 mmol), and isocyanomethane (0.063 mL, 1.20 mmol) in MeOH (1.5 mL) with a spatula of activated molecular sieves was stirred at 100 °C for 1 h under microwave irradiations and then was cooled to rt and diluted with water. The aqueous phase was extracted with EtOAc. The organic phase was washed with water, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (10 g column, gradient 0–10% (2 M NH<sub>3</sub> in MeOH) in DCM) gave <i>N</i>-(2-ethoxy-2-phenylethyl)-1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (66 mg, 29%) as a yellow solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 7.8–7.9 (m, 1H), 7.7–7.8 (m, 1H), 7.2–7.4 (m, 6H), 4.5–4.7 (m, 1H), 4.1–4.2 (m, 1H), 3.9–4.1 (m, 1H), 3.44 (s, 3H), 3.3–3.4 (m, 4H), 2.63 (d, 3H, <i>J</i> = 5.0 Hz), 2.01 (s, 3H), 1.11 (t, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.78 min, [M + H]<sup>+</sup> = 386.1.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (46)</h4><div class="NLM_p">A solution of 2-phenyl-2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethan-1-amine hydrochloride (101 mg, 0.419 mmol) and paraformaldehyde (18.0 mg, 0.598 mmol) in trifluoroethanol (2 mL) was stirred at rt for 15 min before being treated with DIPEA (0.052 mL, 0.30 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50.0 mg, 0.30 mmol), and isocyanomethane (0.031 mL, 0.600 mmol). The reaction mixture was stirred at 100 °C for 1 h under microwave irradiations and then was then was cooled to rt and diluted with water. The aqueous phase was extracted with EtOAc. The organic phase was dried using a hydrophobic frit and concentrated in vacuo. The purification of the residue obtained by MDAP (method formic) gave 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenyl-2-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (53 mg, 42%) as a pale yellow solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.1–7.3 (m, 7H), 6.9–7.0 (m, 1H), 3.8–3.9 (m, 2H), 3.6–3.8 (m, 5H), 3.41 (s, 3H), 2.82 (ddd, 2H, <i>J</i> = 5.1, 8.3, 9.9 Hz), 2.61 (d, 3H, <i>J</i> = 4.5 Hz), 2.00 (s, 3H), 1.7–1.9 (m, 2H), 1.1–1.3 (m, 2H), 1.0–1.1 (m, 1H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.70 min, [M + H]<sup>+</sup> = 426.3.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-3-(tetrahydro-2<i>H</i>-pyran-4-yl)propyl)-1,6-dihydropyridine-3-carboxamide (47)</h4><div class="NLM_p">A solution of paraformaldehyde (26.9 mg, 0.897 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), and 2-phenyl-3-(tetrahydro-2<i>H</i>-pyran-4-yl)propan-1-amine (72.2 mg, 0.329 mmol) in MeOH (0.5 mL) was stirred at rt for 15 min before being treated with isocyanomethane (0.027 mL, 0.45 mmol). The resulting mixture was stirred at 80 °C for 1 h under microwave irradiations and then was cooled to rt and diluted with water (15 mL). The aqueous phase was extracted three times with EtOAc. The combined organics were washed with a saturated NH<sub>4</sub>Cl aqueous solution, then with a saturated NaHCO<sub>3</sub> aqueous solution, before being dried using a hydrophobic frit and concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (10 g column, gradient 0–20% (2 M NH<sub>3</sub> in MeOH) in DCM) gave another residue, which was further purified by MDAP (method high pH) to give 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenyl-3-(tetrahydro-2<i>H</i>-pyran-4-yl)propyl)-1,6-dihydropyridine-3-carboxamide (43 mg, 11%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.1–7.4 (m, 7H), 7.03 (dd, 1H, <i>J</i> = 1.3, 2.5 Hz), 3.5–3.9 (m, 8H), 3.43 (s, 3H), 3.0–3.1 (m, 1H), 2.62 (d, 3H, <i>J</i> = 4.8 Hz), 2.01 (s, 3H), 1.5–1.6 (m, 2H), 1.3–1.5 (m, 3H), 1.1–1.2 (m, 2H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.75 min, [M + H]<sup>+</sup> = 440.3.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> N-(2-(1H-Imidazol-1-yl)-2-phenylethyl)-1,5-dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (48)</h4><div class="NLM_p">A solution of 2-(1<i>H</i>-imidazol-1-yl)-2-phenylethan-1-amine (44.8 mg, 0.239 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (40 mg, 0.24 mmol), and paraformaldehyde (14.4 mg, 0.479 mmol) in MeOH (500 μL) with a spatula of activated molecular sieves was stirred at rt for 20 min, before being treated with isocyanomethane (18.9 μL, 0.359 mmol). The resulting mixture was stirred at 70 °C for 1 h under microwave irradiation and then was cooled to rt and diluted to 1 mL with 1:1 MeOH/DMSO. This solution was purified by MDAP (method high pH) to give <i>N</i>-(2-(1<i>H</i>-imidazol-1-yl)-2-phenylethyl)-1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (50 mg, 51%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.73 (s, 1H), 7.3–7.4 (m, 7H), 7.20 (s, 1H), 7.0–7.1 (m, 1H), 6.94 (s, 1H), 5.73 (dd, 1H, <i>J</i> = 5.9, 8.8 Hz), 4.1–4.3 (m, 2H), 3.83 (dd, 2H, <i>J</i> = 16.9, 63.1 Hz), 3.43 (s, 3H), 2.63 (d, 3H, <i>J</i> = 4.6 Hz), 2.01 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.66 min, [M + H]<sup>+</sup> = 408.6.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-2-(1<i>H</i>-pyrazol-1-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (49)</h4><div class="NLM_p">A solution of 2-phenyl-2-(1<i>H</i>-pyrazol-1-yl)ethan-1-amine (17 mg, 0.09 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (15 mg, 0.09 mmol), and paraformaldehyde (5.4 mg, 0.18 mmol) in MeOH (500 μL) with a spatula of activated molecular sieves was stirred at rt for 20 min before being treated with <i>N</i>-methylidynemethanaminium (7.08 μL, 0.135 mmol). The resulting mixture was stirred at 70 °C for 1 h under microwave irradiations and then was cooled to rt and diluted to 1 mL in 1:1 MeOH/DMSO. This solution was purified by MDAP (method high pH) to give 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenyl-2-(1<i>H</i>-pyrazol-1-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (18 mg, 49%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.73 (d, 1H, <i>J</i> = 2.4 Hz), 7.51 (br s, 1H), 7.4–7.4 (m, 1H), 7.3–7.4 (m, 6H), 7.09 (br s, 1H), 6.2–6.3 (m, 1H), 5.82 (dd, 1H, <i>J</i> = 5.9, 8.8 Hz), 4.1–4.3 (m, 2H), 3.7–3.9 (m, 2H), 3.45 (s, 3H), 2.62 (d, 3H, <i>J</i> = 4.4 Hz), 2.03 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.74 min, [M + H]<sup>+</sup> = 408.6.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (50)</h4><div class="NLM_p">A solution of 2-phenyl-2-(pyridin-2-yl)ethan-1-amine (130 mg, 0.625 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (100 mg, 0.568 mmol), and paraformaldehyde (51.7 mg, 1.70 mmol) in MeOH (1 mL) with a spatula of activated molecular sieves was stirred at rt for 10 min before being treated with isocyanomethane (0.051 mL, 0.85 mmol). The resulting mixture was stirred at 80 °C for 1 h under microwave irradiation and then was cooled to rt and diluted to 1 mL with DMSO. This solution was injected into an MDAP (method high pH) to give 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (92 mg, 39%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.5–8.6 (m, 1H), 7.68 (dt, 1H, <i>J</i> = 2.0, 7.7 Hz), 7.39 (br d, 1H, <i>J</i> = 2.5 Hz), 7.2–7.3 (m, 8H), 7.04 (dd, 1H, <i>J</i> = 1.3, 2.5 Hz), 4.57 (t, 1H, <i>J</i> = 7.4 Hz), 4.25 (dd, 1H, <i>J</i> = 7.3, 14.1 Hz), 4.12 (dd, 1H, <i>J</i> = 7.3, 14.1 Hz), 3.84 (d, 2H, <i>J</i> = 4.0 Hz), 3.43 (s, 3H), 2.63 (d, 3H, <i>J</i> = 4.5 Hz), 2.0–2.0 (m, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.59 min, [M + H]<sup>+</sup> = 419.3.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> (R*)-1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (51) and (S*)-1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (52)</h4><div class="NLM_p">The two isomers were separated by chiral chromatography. Analytical method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 5 uL of the solution was injected onto the column, eluting with 60% EtOH (+0.2% isopropylamine) in heptane. Flow rate 1 mL/min; Wavelength 215 nM. Column 4.6 mm id × 25 cm Chiralpak AD-H. Preparative method: Approximately 92 mg of material was dissolved in EtOH (1.5 mL). This was injected onto the column, eluting with 60% EtOH in heptane. Flow 30 mL/min; Wavelength 215 nM; Column 30 mm × 25 cm Chiralpak AD-H (5 um). Fractions from 9.5 to 13 min were bulked to give the fastest eluting diastereoisomer, <b>51</b> (43 mg, 47%), as a yellow solid; Fractions from 15 to 21 min were bulked to give the slowest eluting diastereosiomer, <b>52</b> (41 mg, 45%), as a yellow solid.</div><div id="sec4_1_40_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 51</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.53 (br d, 1H, <i>J</i> = 4.5 Hz), 7.78 (br d, 1H, <i>J</i> = 4.3 Hz), 7.71 (dt, 1H, <i>J</i> = 1.8, 7.7 Hz), 7.45 (d, 1H, <i>J</i> = 2.5 Hz), 7.2–7.3 (m, 7H), 7.01 (br s, 1H), 4.5–4.6 (m, 1H), 4.0–4.2 (m, 2H), 3.8–3.9 (m, 2H), 3.41 (s, 3H), 2.59 (d, 3H, <i>J</i> = 4.5 Hz), 1.98 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.58 min, [M + H]<sup>+</sup> = 419.2.</div></div><div id="sec4_1_40_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 52</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 8.54 (ddd, 1H, <i>J</i> = 0.9, 1.8, 4.8 Hz), 7.68 (dt, 1H, <i>J</i> = 2.0, 7.7 Hz), 7.39 (d, 1H, <i>J</i> = 2.5 Hz), 7.2–7.4 (m, 8H), 7.0–7.1 (m, 1H), 4.58 (t, 1H, <i>J</i> = 7.3 Hz), 4.25 (dd, 1H, <i>J</i> = 7.3, 14.1 Hz), 4.12 (dd, 1H, <i>J</i> = 7.3, 14.1 Hz), 3.84 (d, 2H, <i>J</i> = 4.0 Hz), 3.43 (s, 3H), 2.63 (d, 3H, <i>J</i> = 4.8 Hz), 2.0–2.0 (m, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.58 min, [M + H]<sup>+</sup> = 419.3.</div></div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-2-(pyridin-3-yl)ethyl)-1,6-dihydropyridine-3-xarboxamide (53) and 1,5-Dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-N-(2-phenyl-2-(pyridin-4-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (54)</h4><div class="NLM_p">These two pyridyl isomers were done in parallel using the following procedure: A solution of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (50 mg, 0.30 mmol), 2-phenyl-2-(pyridin-3-yl)ethan-1-amine or 2-phenyl-2-(pyridin-4-yl)ethan-1-amine (65.2 mg, 0.329 mmol), and paraformaldehyde (18 mg, 0.60 mmol) in MeOH (1 mL) with a spatula of activated molecular sieves was treated at rt with isocyanomethane (31 μL, 0.60 mmol) and then was stirred at 100 °C for 30 min under microwave irradiation before being cooled to rt and diluted with DMSO (1 mL). These solutions were injected into an MDAP (two injections, method high pH) to give <b>53</b> 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenyl-2-(pyridin-3-yl)ethyl)-1,6-dihydropyridine-3-xarboxamide (57 mg, 41%) as a white solid and <b>54</b> 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-<i>N</i>-(2-phenyl-2-(pyridin-4-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (22 mg, 16%) as a white solid.</div><div id="sec4_1_41_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> (53)</h5><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ 8.5–8.5 (m, 1H), 8.43 (dd, 1H, <i>J</i> = 1.5, 5.0 Hz), 7.7–7.9 (m, 1H), 7.2–7.5 (m, 6H), 7.10 (dd, 1H, <i>J</i> = 1.0, 2.5 Hz), 4.4–4.6 (m, 1H), 4.1–4.3 (m, 2H), 3.9–4.1 (m, 2H), 3.53 (s, 3H), 2.7–2.8 (m, 3H), 2.09 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.70 min, [M + H]<sup>+</sup> = 419.4.</div></div><div id="sec4_1_41_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> (54)</h5><div class="NLM_p"><sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ 8.4–8.5 (m, 2H), 7.2–7.5 (m, 8H), 7.10 (d, 1H, <i>J</i> = 1.0 Hz), 4.50 (br s, 1H), 4.1–4.3 (m, 2H), 3.8–4.1 (m, 2H), 3.52 (s, 3H), 2.74 (s, 3H), 2.08 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.69 min, [M + H]<sup>+</sup> = 419.4.</div></div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> N-(2-Hydroxy-2,2-diphenylethyl)-1,5-dimethyl-N-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (55)</h4><div class="NLM_p">A solution of 2-amino-1,1-diphenylethan-1-ol (140 mg, 0.658 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (100 mg, 0.598 mmol), isocyanomethane (0.063 mL, 1.20 mmol), and paraformaldehyde (35.9 mg, 1.20 mmol) in MeOH (1.5 mL) was stirred at 100 °C for 60 min under microwave irradiation and then was cooled to rt and diluted with EtOAc. The organic phase was washed with saturated NH<sub>4</sub>Cl aqueous solution, then with water and brine, dried using a hydrophobic frit, and concentrated in vacuo to give a yellow solid. This solid was triturated with MeOH, with heating, to give <i>N</i>-(2-hydroxy-2,2-diphenylethyl)-1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxoethyl)-6-oxo-1,6-dihydropyridine-3-carboxamide (84 mg, 32%) as a white solid.</div><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 7.78 (br s, 1H), 7.40 (br d, 5H, <i>J</i> = 7.6 Hz), 7.2–7.3 (m, 7H), 4.31 (s, 2H), 3.94 (s, 2H), 3.34 (s, 3H), 3.30 (s, 4H), 1.92 (s, 3H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.82 min, [M + H]<sup>+</sup> = 434.1.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 1,5-Dimethyl-N-(1-(methylamino)-1-oxopropan-2-yl)-6-oxo-N-(2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (56, 57, 58, 59)</h4><div class="NLM_p">A solution of 2-phenyl-2-(pyridin-2-yl)ethan-1-amine (71.2 mg, 0.359 mmol), 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (60 mg, 0.36 mmol), and acetaldehyde (0.061 mL, 1.10 mmol) in MeOH (0.5 mL) with a spatula of activated molecular sieves was stirred at rt for 10 min before being treated with isocyanomethane (0.032 mL, 0.540 mmol), and the resulting mixture was stirred at 80 °C for 60 min under microwave irradiation and then was cooled to rt. Further acetaldehyde (0.061 mL, 1.10 mmol), isocyanomethane (0.032 mL, 0.540 mmol), and 2-phenyl-2-(pyridin-2-yl)ethan-1-amine (71.2 mg, 0.359 mmol) were added, and the resulting mixture was stirred at 80 °C for 60 min under microwave irradiation and then was cooled to rt. The mixture was diluted with DMSO (1.5 mL) and purified by MDAP (two injections, method formic) to give 1,5-dimethyl-<i>N</i>-(1-(methylamino)-1-oxopropan-2-yl)-6-oxo-<i>N</i>-(2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (84 mg, 54%) as a brown solid. The four diastereoisomers were separated by chiral chromatography. Analytical method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 20 uL of the solution was injected onto the column, eluting with 80% EtOH (+0.2% isopropylamine) in heptane. Flow rate 1 mL/min; Wavelength 230 nM. Column 4.6 mm id × 25 cm Chiralpak IC. Preparative method: Approximately 84 mg of material was dissolved in EtOH (1 mL). Two injections of 0.5 mL were done onto the column, eluting with 80% EtOH (+0.2% isopropylamine) in heptane (+0.2% isopropylamine). Flow rate 25 mL/min; Wavelength 215 nM; Column 30 mm × 25 cm Chiralpak IC (5 um). The four diastereoisomers were obtained as such (all as white solids): Fractions from 18 to 19.5 min were bulked to give diastereoisomer <b>59</b> (10 mg, 12%); Fractions from 20.5 to 22.5 min were bulked to give diastereoisomer <b>57</b> (12 mg, 14%); Fractions from 24 to 29 min were bulked to give diastereoisomer <b>56</b> (14 mg, 17%); Fractions from 45 to 56 min were bulked to give diastereoisomer <b>58</b> (15 mg, 18%). The mixed fractions between peak 1 and peak 2 (from 19.5 to 20 min) were concentrated in vacuo and reprocessed.</div><div id="sec4_1_43_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> (56)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.51 (dd, 1H, <i>J</i> = 2.0, 5.1 Hz), 7.64 (dt, 1H, <i>J</i> = 1.9, 7.6 Hz), 7.38 (d, 1H, <i>J</i> = 2.4 Hz), 7.2–7.3 (m, 8H), 7.0–7.1 (m, 1H), 4.54 (dd, 1H, <i>J</i> = 6.1, 7.6 Hz), 4.3–4.4 (m, 2H), 3.95 (dd, 1H, <i>J</i> = 7.6, 14.4 Hz, obs), 3.42 (s, 3H), 2.61 (d, 3H, <i>J</i> = 4.6 Hz), 2.00 (s, 3H), 1.15 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.63 min, [M + H]<sup>+</sup> = 433.3.</div></div><div id="sec4_1_43_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> (57)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 8.5–8.5 (m, 1H), 7.65 (dt, 1H, <i>J</i> = 1.8, 7.6 Hz), 7.41 (d, 1H, <i>J</i> = 2.4 Hz), 7.1–7.3 (m, 8H), 7.1–7.1 (m, 1H), 4.52 (dd, 1H, <i>J</i> = 5.6, 7.9 Hz), 4.33 (q, 1H, <i>J</i> = 7.1 Hz), 4.26 (dd, 1H, <i>J</i> = 7.9, 14.3 Hz), 4.05 (dd, 1H, <i>J</i> = 5.6, 14.3 Hz), 3.43 (s, 3H), 2.57 (d, 3H, <i>J</i> = 4.6 Hz), 2.01 (s, 3H), 1.21 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.65 min, [M + H]<sup>+</sup> = 433.3.</div></div><div id="sec4_1_43_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> (58)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 8.5–8.5 (m, 1H), 7.65 (dt, 1H, <i>J</i> = 1.8, 7.6 Hz), 7.41 (d, 1H, <i>J</i> = 2.4 Hz), 7.1–7.3 (m, 8H), 7.1–7.1 (m, 1H), 4.52 (dd, 1H, <i>J</i> = 5.6, 7.9 Hz), 4.33 (q, 1H, <i>J</i> = 7.1 Hz), 4.26 (dd, 1H, <i>J</i> = 7.9, 14.3 Hz), 4.05 (dd, 1H, <i>J</i> = 5.6, 14.3 Hz), 3.43 (s, 3H), 2.57 (d, 3H, <i>J</i> = 4.6 Hz), 2.01 (s, 3H), 1.21 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.64 min, [M + H]<sup>+</sup> = 433.4.</div></div><div id="sec4_1_43_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> (59)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 392 K) δ 8.51 (dd, 1H, <i>J</i> = 1.9, 5.1 Hz), 7.64 (dt, 1H, <i>J</i> = 1.9, 7.6 Hz), 7.38 (d, 1H, <i>J</i> = 2.4 Hz), 7.2–7.3 (m, 8H), 7.0–7.1 (m, 1H), 4.55 (dd, 1H, <i>J</i> = 6.1, 7.6 Hz), 4.3–4.4 (m, 2H), 3.95 (dd, 1H, <i>J</i> = 7.6, 14.4 Hz), 3.42 (s, 3H), 2.61 (d, 3H, <i>J</i> = 4.6 Hz), 2.00 (s, 3H), 1.15 (d, 3H, <i>J</i> = 7.1 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.65 min, [M + H]<sup>+</sup> = 433.3.</div></div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> Chiral Purification of 2-Phenyl-2-(pyridin-2-yl)ethan-1-amine</h4><div class="NLM_p last">Approximately 2 g of the two enantiomers of 2-phenyl-2-(pyridin-2-yl)ethan-1-amine hydrochloride was separated by chiral chromatography. Analytical method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 20 uL of the solution was injected onto the column, eluting with 20% EtOH (+0.2%isopropylamine) in heptane; Flow rate 1.0 mL/min, wavelength 215 nm; Column 4.6 mm id × 25 cm Chiralcel OJ-H. Preparative method (number of injections: 52): Approximately 40–50 mg was dissolved in 3:1 EtOH/isopropylamine (2 mL) and injected onto the column, eluting with 20% EtOH (+0.2% isopropylamine) in heptane (+0.2% isopropylamine); Flow rate: 30 mL/min; Wavelength 215 nm; Column 30 mm × 25 cm Chiralcel OJ-H (5 uM). Pure fractions were separated, while mixed fractions were concentrated in vacuo to be purified again via the same process. Fractions from 9.5 to 11.5 min were bulked to give the fastest-running enantiomer arbitrarily given the (<i>S</i>) conformation: (<i>S</i>*)-2-phenyl-2-(pyridin-2-yl)ethan-1-amine (567 mg, 57%), and fractions from 13 to 16 min were bulked to give the slowest-running enantiomer arbitrarily given the (<i>R</i>) conformation (<i>R</i>*)-2-phenyl-2-(pyridin-2-yl)ethan-1-amine (540 mg, 54%).</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> Enantiomer 1</h3><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.54 (br d, 1H, <i>J</i> = 4.4 Hz), 7.69 (dt, 1H, <i>J</i> = 1.5, 7.8 Hz), 7.2–7.3 (m, 5H), 7.1–7.2 (m, 2H), 4.12 (t, 1H, <i>J</i> = 7.3 Hz), 3.38 (dd, 1H, <i>J</i> = 8.1, 12.5 Hz), 3.11 (dd, 1H, <i>J</i> = 6.8, 12.7 Hz). NH<sub>2</sub> hidden in water peak.</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.77 min, [M + H]<sup>+</sup> = 199.3.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Enantiomer 2</h3><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.54 (br d, 1H, <i>J</i> = 4.4 Hz), 7.69 (dt, 1H, <i>J</i> = 1.5, 7.8 Hz), 7.2–7.3 (m, 5H), 7.1–7.2 (m, 2H), 4.12 (t, 1H, <i>J</i> = 7.3 Hz), 3.38 (dd, 1H, <i>J</i> = 8.1, 12.5 Hz), 3.11 (dd, 1H, <i>J</i> = 6.8, 12.7 Hz). NH<sub>2</sub> hidden in water peak.</div><div class="NLM_p">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.78 min, [M + H]<sup>+</sup> = 199.3.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxo-1-(tetrahydro-2H-pyran-4-yl)ethyl)-6-oxo-N-((S*)-2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (60, 61)</h4><div id="sec4_3_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> Step 1</h5><div class="NLM_p last">A solution of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (150 mg, 0.897 mmol), tetrahydro-2<i>H</i>-pyran-4-carbaldehyde (205 mg, 1.79 mmol), and (<i>R</i>*)-2-phenyl-2-(pyridin-2-yl)ethan-1-amine (249 mg, 1.26 mmol) in trifluoroethanol (4 mL) at rt under N<sub>2</sub> was treated with isocyanomethane (75 mg, 1.8 mmol), and the resulting mixture was stirred at 80 °C for 1 h under microwave irradiation and then was cooled to rt and partitioned between water and EtOAc. The layers were separated, and the organic phase was washed with a saturated NaHCO<sub>3</sub> aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (gradient 0–10% (2N NH<sub>3</sub> in MeOH) in DCM) gave 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxo-1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-6-oxo-<i>N</i>-((<i>R</i>*)-2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (304 mg, 67%) as a pale yellow solid.</div></div><div id="sec4_3_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> Step 2</h5><div class="NLM_p last">The two diastereoisomers were separated by chiral chromatography. Analytical method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 5 μL of this was injected onto the column, eluting with 50% EtOH (+0.2% isopropylamine) in heptane. Flow rate 1 mL/min; Wavelength 230 nM. Column 4.6 mm id × 25 cm Chiralpak IE. Preparative method: Approximately 304 mg of material was dissolved in EtOH (6 mL). Six injections of 1 mL were done onto the column, eluting with 50% EtOH (+0.2% isopropylamine) in heptane (+0.2% isopropylamine). Flow rate 25 mL/min; Wavelength 215 nM; Column 30 mm × 25 cm Chiralpak IE (5 μm). Fractions from 18 to 21 min were bulked to give the fastest-eluting diastereoisomer <b>61</b> (Enantiomer of <b>62</b>, 165 mg, 54%) as a white powder; Fractions from 22.5 to 27 min were bulked to give the slowest-eluting diastereosiomer <b>60</b> (Enantiomer of <b>63</b>, 135 mg, 44%) as a white powder.</div></div><div id="sec4_3_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> (60)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 393 K) δ 8.5–8.5 (m, 1H), 7.62 (dt, 1H, <i>J</i> = 2.0, 7.6 Hz), 7.4–7.5 (m, 1H), 7.28 (d, 1H, <i>J</i> = 2.4 Hz), 7.1–7.3 (m, 7H), 7.03 (dd, 1H, <i>J</i> = 1.0, 2.4 Hz), 4.57 (t, 1H, <i>J</i> = 6.8 Hz), 4.43 (dd, 1H, <i>J</i> = 6.8, 14.1 Hz), 4.20 (dd, 1H, <i>J</i> = 6.8, 14.1 Hz), 4.02 (d, 1H, <i>J</i> = 10.3 Hz), 3.8–3.8 (m, 2H), 3.42 (s, 3H), 3.28 (dt, 1H, <i>J</i> = 2.4, 11.5 Hz), 3.19 (dt, 1H, <i>J</i> = 2.4, 11.5 Hz), 2.63 (d, 3H, <i>J</i> = 4.9 Hz), 2.2–2.4 (m, 1H), 2.03 (s, 3H), 1.4–1.5 (m, 2H), 1.1–1.2 (m, 2H).</div><div class="NLM_p"><sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 151 MHz) δ 170.1, 169.2, 161.9, 161.1, 148.9, 141.7, 137.8, 136.6, 135.4, 128.4, 127.9, 127.1, 126.3, 123.6, 121.6, 113.5, 66.5, 66.3, 65.9, 50.9, 37.2, 34.1, 29.5, 29.3, 25.5, 16.8. Rotameric processes cause some signals to be broadened and/or split into separate chemical shifts.</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.68 min, [M + H]<sup>+</sup> = 503.4.</div></div><div id="sec4_3_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> (61)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz, 393 K) δ 8.48 (d, 1H, <i>J</i> = 3.9 Hz), 7.63 (dt, 1H, <i>J</i> = 2.0, 7.6 Hz), 7.6–7.6 (m, 1H), 7.32 (d, 1H, <i>J</i> = 2.0 Hz), 7.1–7.3 (m, 7H), 7.0–7.1 (m, 1H), 4.55 (t, 1H, <i>J</i> = 6.8 Hz), 4.38 (dd, 1H, <i>J</i> = 6.8, 14.1 Hz), 4.22 (dd, 1H, <i>J</i> = 6.8, 14.1 Hz), 4.12 (d, 1H, <i>J</i> = 10.3 Hz), 3.7–3.8 (m, 2H), 3.43 (s, 3H), 3.26 (dt, 1H, <i>J</i> = 2.5, 11.5 Hz), 3.19 (dt, 1H, <i>J</i> = 2.4, 11.5 Hz), 2.65 (d, 3H, <i>J</i> = 4.4 Hz), 2.2–2.4 (m, 1H), 2.02 (s, 3H), 1.4–1.6 (m, 2H), 1.1–1.3 (m, 2H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.69 min, [M + H]<sup>+</sup> = 503.4.</div></div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 1,5-Dimethyl-N-(2-(methylamino)-2-oxo-1-(tetrahydro-2H-pyran-4-yl)ethyl)-6-oxo-N-((R*)-2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (62, 63)</h4><div class="NLM_p">A solution of 1,5-dimethyl-6-oxo-1,6-dihydropyridine-3-carboxylic acid (150 mg, 0.897 mmol), tetrahydro-2<i>H</i>-pyran-4-carbaldehyde (205 mg, 1.79 mmol), and (<i>R</i>*)-2-phenyl-2-(pyridin-2-yl)ethan-1-amine (249 mg, 1.26 mmol) in trifluoroethanol (4 mL) at rt under N<sub>2</sub> was treated with isocyanomethane (75 mg, 1.8 mmol), and the resulting mixture was stirred at 80 °C for 1 h under microwave irradiation and then was cooled to rt and partitioned between water and EtOAc. The layers were separated, and the organic phase was washed with a saturated NaHCO<sub>3</sub> aqueous solution, dried using a hydrophobic frit, and concentrated in vacuo. The purification of the residue obtained by flash chromatography on silica gel (gradient 0–10% (2N NH<sub>3</sub> in MeOH) in DCM) gave a residue that was further purified by MDAP (method formic) to give 1,5-dimethyl-<i>N</i>-(2-(methylamino)-2-oxo-1-(tetrahydro-2<i>H</i>-pyran-4-yl)ethyl)-6-oxo-<i>N</i>-((<i>R</i>*)-2-phenyl-2-(pyridin-2-yl)ethyl)-1,6-dihydropyridine-3-carboxamide (127 mg, 28%) as a white solid.</div><div id="sec4_3_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> Step 2</h5><div class="NLM_p last">The two diastereoisomers were separated by chiral chromatography. Analytical method: Approximately 0.5 mg of material was dissolved in 1:1 EtOH/heptane (1 mL), and 5 uL of this was injected onto the column, eluting with 70% EtOH (+0.2% isopropylamine) in heptane. Flow rate 1 mL/min; Wavelength 215 nM. Column 4.6 mm id × 25 cm Chiralpak IC. Preparative method: Approximately 127 mg of material was dissolved in EtOH (5 mL) with heating. Two injections of 2.5 mL were done onto the column, eluting with 70% EtOH (+ 0.2% isopropylamine) in heptane. Flow 25 mL/min; Column 30 mm × 25 cm Chiralpak IC (5 μm). Fractions from 14.5–17 min were bulked to give the fastest-eluting diastereoisomer <b>63</b> (47 mg, 74%, enantiomer of <b>60</b>) as an off white powder; Fractions from 29 to 38 min were bulked to give the slowest-eluting diastereosiomer <b>62</b> (63 mg, 99%, enantiomer of <b>61</b>) as an off white powder.</div></div><div id="sec4_3_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> (62)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.48 (d, 1H, <i>J</i> = 3.9 Hz), 7.63 (dt, 1H, <i>J</i> = 2.0, 7.6 Hz), 7.58 (br d, 1H, <i>J</i> = 2.4 Hz), 7.32 (d, 1H, <i>J</i> = 2.0 Hz), 7.1–7.3 (m, 7H), 7.0–7.1 (m, 1H), 4.55 (t, 1H, <i>J</i> = 6.8 Hz), 4.38 (dd, 1H, <i>J</i> = 6.8, 14.2 Hz), 4.22 (dd, 1H, <i>J</i> = 6.8, 14.1 Hz), 4.12 (d, 1H, <i>J</i> = 10.8 Hz), 3.7–3.8 (m, 2H), 3.43 (s, 3H), 3.26 (dt, 1H, <i>J</i> = 2.5, 11.5 Hz), 3.19 (dt, 1H, <i>J</i> = 2.4, 11.5 Hz), 2.65 (d, 3H, <i>J</i> = 4.9 Hz), 2.2–2.4 (m, 1H), 2.02 (s, 3H), 1.4–1.6 (m, 2H), 1.19 (dt, 2H, <i>J</i> = 4.4, 12.7 Hz).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.67 min, [M + H]<sup>+</sup> = 503.4.</div></div><div id="sec4_3_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> (63)</h5><div class="NLM_p"><sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ 8.5–8.5 (m, 1H), 7.62 (dt, 1H, <i>J</i> = 2.0, 7.6 Hz), 7.4–7.5 (m, 1H), 7.28 (d, 1H, <i>J</i> = 2.4 Hz), 7.1–7.3 (m, 7H), 7.0–7.0 (m, 1H), 4.57 (t, 1H, <i>J</i> = 6.8 Hz), 4.43 (dd, 1H, <i>J</i> = 6.8, 14.1 Hz), 4.19 (dd, 1H, <i>J</i> = 6.8, 14.1 Hz), 4.02 (d, 1H, <i>J</i> = 10.3 Hz), 3.8–3.8 (m, 2H), 3.42 (s, 3H), 3.28 (dt, 1H, <i>J</i> = 2.4, 11.5 Hz), 3.19 (dt, 1H, <i>J</i> = 2.4, 11.5 Hz), 2.63 (d, 3H, <i>J</i> = 4.9 Hz), 2.2–2.4 (m, 1H), 2.03 (s, 3H), 1.4–1.5 (m, 2H), 1.1–1.3 (m, 2H).</div><div class="NLM_p last">LCMS (Formic, ES<sup>+</sup>): <i>t</i><sub>R</sub> = 0.68 min, [M + H]<sup>+</sup> = 503.4.</div></div></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43408" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43408" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> DEL Affinity Selection</h3><div class="NLM_p last">Prior to initiating target selections, 10 μg of Penta-His-Biotin conjugate (Qiagen) was captured on a PhyNexus tip packed with 5 μL of agarose streptavidin resin. This process involved the pipetting of 100 μL of solution up and down for 22 min using the PhyNexus ME200 at a rate of 250 μL/min. After this precapture of Penta-His-Biotin conjugate, the tip was washed five times with 100 μL of a selection buffer [20 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), pH 7.5; 100 mM NaCl; 0.5 mM 3-(3-cholamidopropyl)dimethylammonio-1-propanesulfonate hydrate (CHAPS); 1.0 mg/mL sheared salmon sperm DNA (sssDNA, heat denatured) (Ambion); 1.0 mg/mL bovine serum albumin (BSA) (Ambion)]. Then 7 μg of each of the Brd4 mutant proteins [6His-Thr BRD4 (1–477)(Y97A), 6His-Thr BRD4 (1–477)(Y390A), or 6His-Thr BRD4 (1–477)(Y97A)(Y390A)] was immobilized on the streptavidin tip with Penta-His-Biotin conjugate by pipetting 100 μL of 0.033 mg/mL protein solution up and down for 22 min. This was done for each of the BRD4 mutant constructs on separate tips. Each tip was then washed five times with 100 μL of selection buffer. Ten nanomoles of library pool (DEL34-DEL97) in 60 μL of selection buffer was incubated on the tip with an immobilized protein by pipetting up and down for 1 h. Following this incubation of protein and library pool, the tip was washed 10 times in 100 μL of selection buffer. In order to release and save the bound library molecules off of the tip, a heat elution at 72 °C was performed for 10 min in 60 μL of selection buffer (minus sssDNA). The collected eluant was then used for the second round of affinity selection with fresh immobilized BRD4 mutant protein on a streptavidin tip with a Penta-His-Biotin conjugate. A total of three rounds of affinity selections was performed with each of the three BRD4 mutant constructs. An equivalent No-Target-Control (NTC) selection was performed with all the same steps but without the introduction of protein. At the end of the selections, all samples were quantitated by quantitative polymerase chain reaction (qPCR), PCR amplified, cleaned by AMPure magnetic beads (Beckman Coulter), and sequenced on the Illumina sequencer.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> CLND/CAD Solubility</h3><div class="NLM_p last">The solubility was determined by a precipitation of 10 mM DMSO stock concentration to 5% DMSO pH 7.4 phosphate-buffered saline (PBS), with quantification by ChemiLuminescent Nitrogen Detection (CLND)<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> or Charged Aerosol Detection (CAD).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> FaSSIF Solubility</h3><div class="NLM_p last">Compounds were dissolved in DMSO at 2.5 mg/mL and then diluted in Fast State Simulated Intestinal Fluid (FaSSIF pH 6.5) at 125 μg/mL (final DMSO concentration is 5%). After 16 h of incubation at 25 °C, the suspension was filtered. The concentration of the compound was determined by a fast HPLC gradient. The ratio of the peak areas obtained from the standards and the sample filtrate was used to calculate the solubility of the compound.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> chromlogD<sub>7.4</sub></h3><div class="NLM_p last">The chromatographic hydrophobicity index (ChiLogD<sub>7.4</sub>) was determined using fast gradient HPLC, according to literature procedures<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> that were performed using a Waters Aquity UPLC System, Phenomenex Gemini NX 50 × 2 mm, 3 μm HPLC column, and 0–100% pH 7.40 ammonium acetate buffer/acetonitrile gradient. The retention time was compared to standards of known pH to derive the Chromatographic Hydrophobicity Index (CHI). chromlogD = 0.0857CHI – 2.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> Artificial Membrane Permeability</h3><div class="NLM_p last">The permeability across a lipid membrane was measured using the published protocol.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> hWB MCP-1 Assay</h3><div class="NLM_p last">Compounds to be tested were diluted in 100% DMSO to give a range of appropriate concentrations at 140 times the required final assay concentration, of which 1 μL was added to a 96-well tissue culture plate. 130 microliters of human whole blood, collected into a sodium heparin anticoagulant, (1 unit/mL final), was added to each well, and plates were incubated at 37 °C (5% CO<sub>2</sub>) for 30 min before the addition of 10 μL of 2.8 μg/mL LPS (Salmonella Typhosa), diluted in complete RPMI 1640 (final concentration 200 ng/mL), to give a total volume of 140 μL per well. After further incubation for 24 h at 37 °C, 140 μL of PBS was added to each well. The plates were sealed, shaken for 10 min, and then centrifuged (2500 rpm × 10 min). 100 microliters of the supernatant was removed, and MCP-1 levels were assayed immediately by an immunoassay (MesoScale Discovery technology).</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> BET Assays</h3><div class="NLM_p last">Protein expression and physicochemical property measurement. These were performed as described previously.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> BET BD1 and BD2 TR-FRET Assays</h3><div class="NLM_p">Tandem bromodomains of 6His-Thr-BRD4(1–477) were expressed, with an appropriate mutation in BD2(Y390A) to monitor compound binding to BD1, or in BD1(97A) to monitor compound binding to BD2. Analogous Y⃗A mutants were used to measure binding to the other BET bromodomains: 6His-Thr-BRD2(1–473 Y386A or Y113A), 6His-Thr-BRD3(1–435 Y348A or Y73A), and 6His-FLAG-Tev-BRDT(1–397 Y309A or Y66A). The AlexaFluor 647-labeled BET bromodomain ligand was prepared as follows: To a solution of Alexa Fluor 647 hydroxysuccinimide ester in DMF was added a 1.8-fold excess of <i>N</i>-(5-aminopentyl)-2-((4<i>S</i>)-6-(4-chlorophenyl)-8-methoxy-1-methyl-4<i>H</i>-benzo[f ][1,2,4]triazolo[4,3-<i>a</i>][1,4]-diazepin-4-yl)acetamide, also in DMF, and when thoroughly mixed, the solution was basified by the addition of a threefold excess of diisopropylethylamine. The reaction progress was followed by electrospray LC/MS, and when judged complete, the product was isolated and purified by reversed-phase C18 HPLC. The final compound was characterized by mass spectroscopy and analytical reversed-phase HPLC.</div><div class="NLM_p last">Compounds were titrated from 10 mM in 100% DMSO, and 50 nL was transferred to a low-volume black 384-well microtiter plate using a Labcyte Echo 555. A Thermo Scientific Multidrop Combi was used to dispense 5 μL of 20 nM protein in an assay buffer of 50 mM HEPES, 150 mM NaCl, 5% glycerol, 1 mM dithiothreitol (DTT), and 1 mM CHAPS, pH 7.4, and in the presence of 100 nM fluorescent ligand (∼<i>K</i><sub>d</sub> concentration for the interaction between BRD4 BD1 and ligand). After it had equilibrated for 30 min in the dark at rt, the bromodomain protein/fluorescent ligand interaction was detected using TR-FRET following a 5 μL addition of 3 nM europium chelate-labeled anti-6His antibody (PerkinElmer, W1024, AD0111) in an assay buffer. Time-resolved fluorescence (TRF) was then detected on a TRF laser-equipped PerkinElmer Envision multimode plate reader (excitation = 337 nm; emission 1 = 615 nm; emission 2 = 665 nm; dual wavelength bias dichroic = 400 nm, 630 nm). The TR-FRET ratio was calculated using the following equation: Ratio = ((Acceptor fluorescence at 665 nm)/(Donor fluorescence at 615 nm)) × 1000. The TR-FRET ratio data were normalized to high (DMSO) and low (compound control derivative of I-BET762) controls, and IC<sub>50</sub> values were determined for each of the compounds tested by fitting the fluorescence ratio data to a four-parameter model: <i>y</i> = <i>a</i> + ((<i>b</i> – <i>a</i>)/(1 + (10<sup><i>x</i></sup>/10<sup><i>c</i></sup>)<sup><i>d</i></sup>), where “<i>a</i>” is the minimum, “<i>b</i>” is the Hill slope, “<i>c</i>” is the IC<sub>50</sub>, and “<i>d</i>” is the maximum.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> In Vivo DMPK Studies</h3><div class="NLM_p last"><i>All animal studies were ethically reviewed and performed in accordance with Animals (Scientific Procedures) Act 1986 and the GSK Policy on the Care, Welfare and Treatment of Animals.</i> Rat studies were conducted through external CRO resource (Charles River Laboratories US). There were no known contaminants in the diet or water at concentrations that could interfere with the outcome of the studies.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> Externally Conducted Rat IV/PO n = 1 PK Studies</h3><div class="NLM_p">Male Wistar Han rats (supplied by Charles River US) were received from the supplier equipped with a surgically implanted femoral vein catheter (FVC) that terminated at a percutaneous vascular access port to facilitate intravenous infusion dosing. In addition, the animals were also equipped with a surgically implanted jugular vein catheter (JVC) for blood collections.</div><div class="NLM_p last">Rat PK studies were conducted as a crossover design over two dosing occasions, with 3 d between dose administrations. On the first dosing occasion, rats received a discrete 1 h intravenous (iv) infusion of the Compound of Interest formulated in DMSO and 10% (w/v) Kleptose HPB in saline aqueous (aq) (2%:98% (v/v)) at a concentration of 0.2 mg/mL to achieve a target dose of 1 mg/kg. On the second dosing occasion, the same animal was administered with the same Compound of Interest suspended in 1% (w/v) methylcellulose 400 aq at a concentration of 0.6 mg/mL orally, at a target dose of 3 mg/kg. Serial blood samples (∼100 μL) were collected predose, up to 24 h after the start of the iv infusion, and after oral dosing. Diluted blood samples were analyzed using a specific LC-MS/MS assay (LLQ = 2 ng/mL). At the end of the study the rats were euthanized by an intravenous administration of sodium pentobarbital (Euthatal).</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> Blood Sample Analysis</h3><div class="NLM_p last">Diluted blood samples (1:1 with water) were extracted using protein precipitation with acetonitrile containing an analytical internal standard. An aliquot of the supernatant was analyzed by reverse-phase LC-MS/MS using a heat-assisted electrospray interface in a positive ion mode. Samples were assayed against calibration standards prepared in control blood.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> PK Data Analysis from PK Studies</h3><div class="NLM_p last">Pharmacokinetic parameters were estimated from the blood concentration–time profiles using a noncompartmental analysis with Watson 7.4.2 Bioanalytical LIMS (Thermo Electron Corp).</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> Intrinsic Clearance (CL<sub>int</sub>) Measurements</h3><div class="NLM_p last"><i>The human biological samples were sourced ethically, and their research use was in accord with the terms of the informed consents under an IRB/EC approved protocol.</i> Microsome Intrinsic Clearance data were determined by Cyprotex UK. Hepatocyte Intrinsic Clearance data were determined by Cyprotex UK. The test compound (0.5 μM) was incubated with cryopreserved hepatocytes in suspension. Samples were removed at six time points over the course of a 60 min (rat) or 120 min (dog and human) experiment, and the test compound analyzed by LC-MS/MS. Cryopreserved pooled hepatocytes were purchased from a reputable commercial supplier and stored in liquid nitrogen prior to use. Williams E media supplemented with 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine and 25 mM HEPES and a test compound (final substrate concentration 0.5 μM; final DMSO concentration 0.25%) was preincubated at 37 °C prior to the addition of a suspension of cryopreserved hepatocytes (final cell density 0.5 × 10<sup>6</sup> viable cells/mL in Williams E media supplemented with 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine and 25 mM HEPES) to initiate the reaction. The final incubation volume was 500 μL. The reactions were stopped by transferring 50 μL of incubate to 100 μL of acetonitrile at the appropriate time points. The termination plates were centrifuged at 2500 rpm at 4 °C for 30 min to precipitate the protein. The remaining incubate (200 μL) was crashed with 400 μL of acetonitrile at the end of the incubation. Following the protein precipitation, the sample supernatants were combined in cassettes of up to four compounds and analyzed using Cyprotex generic LC-MS/MS conditions.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> Intrinsic Clearance (CL<sub>int</sub>) Data Analysis</h3><div class="NLM_p">From a plot of ln peak area ratio (compound peak area/internal standard peak area) against time, the gradient of the line was determined. Subsequently, the half-life (<i>t</i><sub>1/2</sub>) and the intrinsic clearance (CL<sub>int</sub>) were calculated using the equations below.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_m001.gif" alt="" id="_i113" /></img></span><span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_m002.gif" alt="" id="_i114" /></img></span><span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_m003.gif" alt="" id="_i115" /></img></span></div><div class="NLM_p last">where <i>V</i> = incubation volume (μL)/number of cells.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i116"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00412" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80138" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80138" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00412?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00412</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">SMILES data (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">DiscoverX BROMOscan bromodomain profiling of <b>60</b>, sequence alignment and differences of BET proteins, X-ray crystallographic data, molecular formula strings, and selected LCMS and NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_001.csv">jm1c00412_si_001.csv (6.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_002.pdf">jm1c00412_si_002.pdf (4.95 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00412" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48300" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48300" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesco Rianjongdee</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4432-3007" title="Orcid link">https://orcid.org/0000-0003-4432-3007</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#234551424d404650404c0d5b0d514a424d494c4d44474646634450480d404c4e"><span class="__cf_email__" data-cfemail="a7c1d5c6c9c4c2d4c4c889df89d5cec6c9cdc8c9c0c3c2c2e7c0d4cc89c4c8ca">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen J. Atkinson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, Quantitative Pharmacology, Immunoinflammation
Therapy Area Unit, and Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Encoded
Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</span>; 
    <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr.
1, Heidelberg 69117, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">WuXi AppTec
(Shanghai) Co., Ltd., Shanghai 200131, China</span>; 
    <span>Present Address:
                        (S.J.A.) Discovery Sciences, AstraZeneca, Cambridge, UK</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3636-3674" title="Orcid link">https://orcid.org/0000-0003-3636-3674</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chun-wa Chung</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2480-3110" title="Orcid link">https://orcid.org/0000-0002-2480-3110</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Grandi</span> - <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr.
1, Heidelberg 69117, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James R. J. Gray</span> - <span class="hlFld-Affiliation affiliation">Quantitative Pharmacology, Immunoinflammation
Therapy Area Unit, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura J. Kaushansky</span> - <span class="hlFld-Affiliation affiliation">Encoded
Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patricia Medeiros</span> - <span class="hlFld-Affiliation affiliation">Encoded
Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cassie Messenger</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex Phillipou</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex Preston</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0334-0679" title="Orcid link">https://orcid.org/0000-0003-0334-0679</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rab K. Prinjha</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Inmaculada Rioja</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander L. Satz</span> - <span class="hlFld-Affiliation affiliation">WuXi AppTec
(Shanghai) Co., Ltd., Shanghai 200131, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1284-1977" title="Orcid link">https://orcid.org/0000-0003-1284-1977</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon Taylor</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, Quantitative Pharmacology, Immunoinflammation
Therapy Area Unit, and Platform Technology and Science, GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Encoded
Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</span>; 
    <span class="hlFld-Affiliation affiliation">IVIVT
Cellzome, Platform Technology and Science, GlaxoSmithKline, Meyerhofstr.
1, Heidelberg 69117, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">WuXi AppTec
(Shanghai) Co., Ltd., Shanghai 200131, China</span>; 
    <span>Present Address:
                        (S.T.) Drug Discovery Services Europe, Pharmaron, Hertford Road, Hoddesdon, EN11 9BU, UK</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian D. Wall</span> - <span class="hlFld-Affiliation affiliation">Platform Technology and Science, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Watson</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gang Yao</span> - <span class="hlFld-Affiliation affiliation">Encoded
Library Technologies, R&D Medicinal Science and Technology, GSK, 200 Cambridge Park Drive, Cambridge 02140, Massachusetts, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emmanuel H. Demont</span> - <span class="hlFld-Affiliation affiliation">Epigenetics
Discovery Performance Unit, , and , GlaxoSmithKline, Medicines Research Centre, Stevenage, Hertfordshire, SG1 2NY, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7307-3129" title="Orcid link">https://orcid.org/0000-0001-7307-3129</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>GlaxoSmithKline.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors were GlaxoSmithKline full-time employees when this study was performed.<br /></br><p class="inlineNote">Authors will release the unpublished PDB ID, atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i121">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54437" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54437" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank E. Ellis and R. Andres for supporting this work during their industrial placement years. We also thank the members of Platform Technology Sciences group at GSK for protein reagent generation, assay, and crystallization support; E. Hortense, R. Briers, S. Jackson, and S. Hindley for analytical and purification support, and S. Lynn, R. Upton, and S. Richards for assistance with NMR analysis.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AMP</td><td class="NLM_def"><p class="first last">Artificial membrane permeability</p></td></tr><tr><td class="NLM_term">BD1</td><td class="NLM_def"><p class="first last">N-terminal Bromodomain</p></td></tr><tr><td class="NLM_term">BD2</td><td class="NLM_def"><p class="first last">C-terminal Bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">Bromodomain and Extra Terminal</p></td></tr><tr><td class="NLM_term">CAD</td><td class="NLM_def"><p class="first last">Charged Aerosol Detection</p></td></tr><tr><td class="NLM_term">chromlogD</td><td class="NLM_def"><p class="first last">Chromatographically determined LogD</p></td></tr><tr><td class="NLM_term">CLND</td><td class="NLM_def"><p class="first last">Chemiluminescent Nitrogen Detection</p></td></tr><tr><td class="NLM_term">DEL</td><td class="NLM_def"><p class="first last">DNA-encoded library</p></td></tr><tr><td class="NLM_term">DMP</td><td class="NLM_def"><p class="first last">Dimethylpyridine</p></td></tr><tr><td class="NLM_term">ET</td><td class="NLM_def"><p class="first last">Extra Terminal</p></td></tr><tr><td class="NLM_term">FaSSIF</td><td class="NLM_def"><p class="first last">Fasted State Simulated Intestinal Fluid</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">Time-Tesolved fluorescence Resonance Energy Transfer</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">Human Ether-à-go-go-Related Gene</p></td></tr><tr><td class="NLM_term">KAc-mimetic</td><td class="NLM_def"><p class="first last">Acetylated Lysine Mimetic</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">High-Throughput Screen</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">Ligand Efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">Lipophilic Ligand Efficiency</p></td></tr><tr><td class="NLM_term">MCP-1</td><td class="NLM_def"><p class="first last">Monocyte chemoattractant protein 1</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">Molecular Weight</p></td></tr><tr><td class="NLM_term">NGS</td><td class="NLM_def"><p class="first last">Next-generation DNA-sequencing technology</p></td></tr><tr><td class="NLM_term">PCR</td><td class="NLM_def"><p class="first last">Polymerase Chain Reaction</p></td></tr><tr><td class="NLM_term">PFI</td><td class="NLM_def"><p class="first last">Property forecast index</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">Pharmacokinetics</p></td></tr><tr><td class="NLM_term">pIC<sub>50</sub></td><td class="NLM_def"><p class="first last">Negative log of the half maximal inhibition concentration</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">Tetrahydrofuran</p></td></tr><tr><td class="NLM_term">THP</td><td class="NLM_def"><p class="first last">Tetrahydropyran</p></td></tr><tr><td class="NLM_term">VDW</td><td class="NLM_def"><p class="first last">van der Waals</p></td></tr><tr><td class="NLM_term">WPF</td><td class="NLM_def"><p class="first last">Tryptophan-proline-phenylalanine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i123">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44733" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44733" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: Epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+Epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0ljS0l42WiXw-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520Epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: The therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+The+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0ljVJPx81Du3tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520The%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0ljVJPx81Du3tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">697</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697-705&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+C.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0ljghSjxmYp7DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%2BC.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D697%26epage%3D705%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0ljghSjxmYp7DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET bromodomain targets genetically diverse glioblastoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1748</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-3066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F1078-0432.CCR-12-3066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23403638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1748-1759&author=Z.+Chengauthor=Y.+Gongauthor=Y.+Maauthor=K.+Luauthor=X.+Luauthor=L.+A.+Pierceauthor=R.+C.+Thompsonauthor=S.+Mullerauthor=S.+Knappauthor=J.+Wang&title=Inhibition+of+BET+bromodomain+targets+genetically+diverse+glioblastoma&doi=10.1158%2F1078-0432.CCR-12-3066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span></div><div class="casAuthors">Cheng, Zhixiang; Gong, Yuanying; Ma, Yufang; Lu, Kaihua; Lu, Xiang; Pierce, Larry A.; Thompson, Reid C.; Muller, Susanne; Knapp, Stefan; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1748-1759</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Glioblastoma is refractory to conventional therapies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic readers that selectively bind to acetylated lysine residues on histone tails.  These proteins recently emerged as important therapeutic targets in NUT midline carcinoma and several types of hematopoietic cancers.  In this study, the therapeutic potential of a novel BET bromodomain inhibitor, JQ1, was assessed in a panel of genetically heterogeneous glioblastoma samples.  Exptl. Design: The antineoplastic effects of JQ1 were shown using ex vivo cultures derived from primary glioblastoma xenograft lines and surgical specimens of different genetic background.  The in vivo efficacy was assessed in orthotopic glioblastoma tumors.  Results: We showed that JQ1 induced marked G1 cell-cycle arrest and apoptosis, which was phenocopied by knockdown of individual BET family members.  JQ1 treatment resulted in significant changes in expression of genes that play important roles in glioblastoma such as c-Myc, p21CIP1/WAF1, hTERT, Bcl-2, and Bcl-xL.  Unlike the observations in some hematopoietic cancer cell lines, exogenous c-Myc did not significantly protect glioblastoma cells against JQ1.  In contrast, ectopically expressed Bcl-xL partially rescued cells from JQ1-induced apoptosis, and knockdown of p21CIP1/WAF1 attenuated JQ1-induced cell-cycle arrest.  Cells genetically engineered for Akt hyperactivation or p53/Rb inactivation did not compromise JQ1 efficacy, suggesting that these frequently mutated signaling pathways may not confer resistance to JQ1.  Furthermore, JQ1 significantly repressed growth of orthotopic glioblastoma tumors.  Conclusion: Our results suggest potentially broad therapeutic use of BET bromodomain inhibitors for treating genetically diverse glioblastoma tumors.  Clin Cancer Res; 19(7); 1748-59. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OsFKqlwyDbVg90H21EOLACvtfcHk0ljKMQD-ps9MMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D&md5=1722189e5d950da233b785558f2c9f58</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3066%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPierce%26aufirst%3DL.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BC.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520BET%2520bromodomain%2520targets%2520genetically%2520diverse%2520glioblastoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D1748%26epage%3D1759%26doi%3D10.1158%2F1078-0432.CCR-12-3066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Da
Motta, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purshouse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bastiani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morotti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigfield, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, A.</span></span> <span> </span><span class="NLM_article-title">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fonc.2016.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27292261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=122-132&author=L.+L.+Da%0AMottaauthor=I.+Ledakiauthor=K.+Purshouseauthor=S.+Haiderauthor=M.+A.+De+Bastianiauthor=D.+Babanauthor=M.+Morottiauthor=G.+Steersauthor=S.+Wigfieldauthor=E.+Bridgesauthor=J.+L.+Liauthor=S.+Knappauthor=D.+Ebnerauthor=F.+Klamtauthor=A.+L.+Harrisauthor=A.+McIntyre&title=The+BET+inhibitor+JQ1+selectively+impairs+tumour+response+to+hypoxia+and+downregulates+CA9+and+angiogenesis+in+triple+negative+breast+cancer&doi=10.1038%2Fonc.2016.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span></div><div class="casAuthors">da Motta, L. L.; Ledaki, I.; Purshouse, K.; Haider, S.; De Bastiani, M. A.; Baban, D.; Morotti, M.; Steers, G.; Wigfield, S.; Bridges, E.; Li, J.-L.; Knapp, S.; Ebner, D.; Klamt, F.; Harris, A. L.; McIntyre, A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-132</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of bromodomain and extra-terminal inhibitors (BETi) has enabled translational epigenetic studies in cancer.  BET proteins regulate transcription by selectively recognizing acetylated lysine residues on chromatin.  BETi compete with this process leading to both downregulation and upregulation of gene expression.  Hypoxia enables progression of triple neg. breast cancer (TNBC), the most aggressive form of breast cancer, partly by driving metabolic adaptation, angiogenesis and metastasis through upregulation of hypoxia-regulated genes (for example, carbonic anhydrase 9 (CA9) and vascular endothelial growth factor A (VEGF-A).  Responses to hypoxia can be mediated epigenetically, thus we investigated whether BETi JQ1 could impair the TNBC response induced by hypoxia and exert anti-tumor effects.  JQ1 significantly modulated 44% of hypoxia-induced genes, of which two-thirds were downregulated including CA9 and VEGF-A.  JQ1 prevented HIF binding to the hypoxia response element in CA9 promoter, but did not alter HIF expression or activity, suggesting some HIF targets are BET-dependent.  JQ1 reduced TNBC growth in vitro and in vivo and inhibited xenograft vascularization.  These findings identify that BETi dually targets angiogenesis and the hypoxic response, an effective combination at reducing tumor growth in preclin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79-F0o2tp6bVg90H21EOLACvtfcHk0ljKMQD-ps9MMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D&md5=9f1592548db8e6306703bdab7e1e1c90</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.184%26sid%3Dliteratum%253Aachs%26aulast%3DDa%2BMotta%26aufirst%3DL.%2BL.%26aulast%3DLedaki%26aufirst%3DI.%26aulast%3DPurshouse%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DS.%26aulast%3DDe%2BBastiani%26aufirst%3DM.%2BA.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DMorotti%26aufirst%3DM.%26aulast%3DSteers%26aufirst%3DG.%26aulast%3DWigfield%26aufirst%3DS.%26aulast%3DBridges%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DMcIntyre%26aufirst%3DA.%26atitle%3DThe%2520BET%2520inhibitor%2520JQ1%2520selectively%2520impairs%2520tumour%2520response%2520to%2520hypoxia%2520and%2520downregulates%2520CA9%2520and%2520angiogenesis%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D122%26epage%3D132%26doi%3D10.1038%2Fonc.2016.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: A novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2563</span>– <span class="NLM_lpage">2574</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F1535-7163.MCT-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563-2574&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%E2%80%99Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+A+novel+bivalent+BET+bromodomain+inhibitor+highly+active+against+hematologic+malignancies&doi=10.1158%2F1535-7163.MCT-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0lj26u0u1D0mZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520A%2520novel%2520bivalent%2520BET%2520bromodomain%2520inhibitor%2520highly%2520active%2520against%2520hematologic%2520malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2563%26epage%3D2574%26doi%3D10.1158%2F1535-7163.MCT-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET bromodomain inhibitor OTX015 affects pathogenetic pathways in preclinical B-cell tumor models and synergizes with targeted drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1628</span>– <span class="NLM_lpage">1638</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628-1638&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+bromodomain+inhibitor+OTX015+affects+pathogenetic+pathways+in+preclinical+B-cell+tumor+models+and+synergizes+with+targeted+drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0lj26u0u1D0mZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520bromodomain%2520inhibitor%2520OTX015%2520affects%2520pathogenetic%2520pathways%2520in%2520preclinical%2520B-cell%2520tumor%2520models%2520and%2520synergizes%2520with%2520targeted%2520drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1628%26epage%3D1638%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">PFI-1, a highly selective protein interaction inhibitor, targeting BET bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3336</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F0008-5472.CAN-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+M%C3%BCllerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+highly+selective+protein+interaction+inhibitor%2C+targeting+BET+bromodomains&doi=10.1158%2F0008-5472.CAN-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0ljRp0pRaVkXcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520highly%2520selective%2520protein%2520interaction%2520inhibitor%252C%2520targeting%2520BET%2520bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segura, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanals-Cirera, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaziel-Sovran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanniford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Gomez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubierre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvishian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, E.</span></span> <span> </span><span class="NLM_article-title">BRD4 sustains melanoma proliferation and represents a new target for epigenetic therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6264</span>– <span class="NLM_lpage">6276</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0122-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F0008-5472.CAN-13-0122-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23950209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6264-6276&author=M.+F.+Seguraauthor=B.+Fontanals-Cireraauthor=A.+Gaziel-Sovranauthor=M.+V.+Guijarroauthor=D.+Hannifordauthor=G.+Zhangauthor=P.+Gonz%C3%A1lez-Gomezauthor=M.+Moranteauthor=L.+Jubierreauthor=W.+Zhangauthor=F.+Darvishianauthor=M.+Ohlmeyerauthor=I.+Osmanauthor=M.+M.+Zhouauthor=E.+Hernando&title=BRD4+sustains+melanoma+proliferation+and+represents+a+new+target+for+epigenetic+therapy&doi=10.1158%2F0008-5472.CAN-13-0122-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span></div><div class="casAuthors">Segura, Miguel F.; Fontanals-Cirera, Barbara; Gaziel-Sovran, Avital; Guijarro, Maria V.; Hanniford, Doug; Zhang, Guangtao; Gonzalez-Gomez, Pilar; Morante, Marta; Jubierre, Luz; Zhang, Weijia; Darvishian, Farbod; Ohlmeyer, Michael; Osman, Iman; Zhou, Ming-Ming; Hernando, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6264-6276</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Metastatic melanoma remains a mostly incurable disease.  Although newly approved targeted therapies are efficacious in a subset of patients, resistance and relapse rapidly ensue.  Alternative therapeutic strategies to manipulate epigenetic regulators and disrupt the transcriptional program that maintains tumor cell identity are emerging.  Bromodomain and extraterminal domain (BET) proteins are epigenome readers known to exert key roles at the interface between chromatin remodeling and transcriptional regulation.  Here, we report that BRD4, a BET family member, is significantly upregulated in primary and metastatic melanoma tissues compared with melanocytes and nevi.  Treatment with BET inhibitors impaired melanoma cell proliferation in vitro and tumor growth and metastatic behavior in vivo, effects that were mostly recapitulated by individual silencing of BRD4.  RNA sequencing of BET inhibitor-treated cells followed by Gene Ontol. anal. showed a striking impact on transcriptional programs controlling cell growth, proliferation, cell-cycle regulation, and differentiation.  In particular, we found that, rapidly after BET displacement, key cell-cycle genes (SKP2, ERK1, and c-MYC) were downregulated concomitantly with the accumulation of cyclin-dependent kinase (CDK) inhibitors (p21 and p27), followed by cell-cycle arrest.  Importantly, BET inhibitor efficacy was not influenced by BRAF or NRAS mutational status, opening the possibility of using these small-mol. compds. to treat patients for whom no effective targeted therapy exists.  Collectively, our study reveals a crit. role for BRD4 in melanoma tumor maintenance and renders it a legitimate and novel target for epigenetic therapy directed against the core transcriptional program of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80TZH2ueQFbVg90H21EOLACvtfcHk0ljRp0pRaVkXcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N&md5=a4a77f188de01c4c84f7293a32478fc7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0122-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0122-T%26sid%3Dliteratum%253Aachs%26aulast%3DSegura%26aufirst%3DM.%2BF.%26aulast%3DFontanals-Cirera%26aufirst%3DB.%26aulast%3DGaziel-Sovran%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%2BV.%26aulast%3DHanniford%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Gomez%26aufirst%3DP.%26aulast%3DMorante%26aufirst%3DM.%26aulast%3DJubierre%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDarvishian%26aufirst%3DF.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DHernando%26aufirst%3DE.%26atitle%3DBRD4%2520sustains%2520melanoma%2520proliferation%2520and%2520represents%2520a%2520new%2520target%2520for%2520epigenetic%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6264%26epage%3D6276%26doi%3D10.1158%2F0008-5472.CAN-13-0122-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: Breaking through the immune barrier</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">835</span>– <span class="NLM_lpage">853</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnrd.2016.185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27765940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=835-853&author=D.+F.+Toughauthor=P.+P.+Takauthor=A.+Tarakhovskyauthor=R.+K.+Prinjha&title=Epigenetic+drug+discovery%3A+Breaking+through+the+immune+barrier&doi=10.1038%2Fnrd.2016.185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: breaking through the immune barrier</span></div><div class="casAuthors">Tough, David F.; Tak, Paul P.; Tarakhovsky, Alexander; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">835-853</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immune-mediated diseases are clin. heterogeneous but they share genetic and pathogenic mechanisms.  These diseases may develop from the interplay of genetic factors and environmental or lifestyle factors.  Exposure to such factors, including infectious agents, is assocd. with coordinated changes in gene transcription owing to epigenetic alterations.  A growing understanding of how epigenetic mechanisms control gene expression patterns and cell function has been aided by the development of small-mol. inhibitors that target these processes.  These chem. tools have helped to reveal the importance of epigenetics in guiding cell fate decisions during immune responses and have also highlighted the potential for targeting epigenetic mechanisms for the treatment of inflammation and immune-mediated diseases.  In this Review, we discuss the most advanced areas of epigenetic drug development for autoimmune and inflammatory diseases and summarize the promising preclin. data in this exciting and evolving field.  These agents will inevitably begin to move into clin. trials for use in patients with immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplOyEVMzBi37Vg90H21EOLACvtfcHk0ljRp0pRaVkXcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE&md5=f939bcbbe3d74ce24718eef706aae3d1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.185%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520Breaking%2520through%2520the%2520immune%2520barrier%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D835%26epage%3D853%26doi%3D10.1038%2Fnrd.2016.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27925476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=573-584&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span></div><div class="casAuthors">Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Genome-wide assocn. studies have identified thousands of single nucleotide polymorphisms in the human genome that are statistically assocd. with particular disease traits.  In this perspective, we review emerging data suggesting that most single nucleotide polymorphisms assocd. with immune-mediated diseases are found in regulatory regions of the DNA - parts of the genome that control expression of the protein encoding genes - rather than causing mutations in proteins.  We discuss how the emerging understanding of particular gene regulatory regions, gene enhancers and the epigenetic mechanisms by which they are regulated is opening up new opportunities for the treatment of immune-mediated diseases, focusing particularly on the BET family of epigenetic reader proteins as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFU7Sh3r75ZbVg90H21EOLACvtfcHk0lgoqos-X5wZFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D&md5=8a2d44dd000c465b904d4ef2e6f37b77</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses TH17-mediated pathology</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>210</i></span>,  <span class="NLM_fpage">2181</span>– <span class="NLM_lpage">2190</span>, <span class="refDoi"> DOI: 10.1084/jem.20130376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1084%2Fjem.20130376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24101376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2013&pages=2181-2190&author=D.+A.+Meleauthor=A.+Salmeronauthor=S.+Ghoshauthor=H.+R.+Huangauthor=B.+M.+Bryantauthor=J.+M.+Lora&title=BET+bromodomain+inhibition+suppresses+TH17-mediated+pathology&doi=10.1084%2Fjem.20130376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses TH17-mediated pathology</span></div><div class="casAuthors">Mele, Deanna A.; Salmeron, Andres; Ghosh, Srimoyee; Huang, Hon-Ren; Bryant, Barbara M.; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin (IL) 17-producing T helper (TH17) cells have been selected through evolution for their ability to control fungal and bacterial infections.  It is also firmly established that their aberrant generation and activation results in autoimmune conditions.  Using a characterized potent and selective small mol. inhibitor, we show that the bromodomain and extra-terminal domain (BET) family of chromatin adaptors plays fundamental and selective roles in human and murine TH17 differentiation from naive CD4+ T cells, as well as in the activation of previously differentiated TH17 cells.  We provide evidence that BET controls TH17 differentiation in a bromodomain-dependent manner through a mechanism that includes the direct regulation of multiple effector TH17-assocd. cytokines, including IL17, IL21, and GMCSF.  We also demonstrate that BET family members Brd2 and Brd4 assoc. with the Il17 locus in TH17 cells, and that this assocn. requires bromodomains.  We recapitulate the crit. role of BET bromodomains in TH17 differentiation in vivo and show that therapeutic dosing of the BET inhibitor is efficacious in mouse models of autoimmunity.  Our results identify the BET family of proteins as a fundamental link between chromatin signaling and TH17 biol., and support the notion of BET inhibition as a point of therapeutic intervention in autoimmune conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBqVtggp2Fa7Vg90H21EOLACvtfcHk0lgoqos-X5wZFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF&md5=e6822f456cf42cdead9779afa92c05e0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1084%2Fjem.20130376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20130376%26sid%3Dliteratum%253Aachs%26aulast%3DMele%26aufirst%3DD.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DH.%2BR.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520TH17-mediated%2520pathology%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2013%26volume%3D210%26spage%3D2181%26epage%3D2190%26doi%3D10.1084%2Fjem.20130376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>– <span class="NLM_lpage">14537</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0lid6ac-2Tw4Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarilina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivashkiv, L. B.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1002/eji.201444862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1002%2Feji.201444862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25345375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=287-297&author=C.+H.+Chanauthor=C.+Fangauthor=Y.+Qiaoauthor=A.+Yarilinaauthor=R.+K.+Prinjhaauthor=L.+B.+Ivashkiv&title=BET+bromodomain+inhibition+suppresses+transcriptional+responses+to+cytokine-Jak-STAT+signaling+in+a+gene-specific+manner+in+human+monocytes&doi=10.1002%2Feji.201444862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span></div><div class="casAuthors">Chan, Chun Hin; Fang, Celestia; Qiao, Yu; Yarilina, Anna; Prinjha, Rab K.; Ivashkiv, Lionel B.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">EJIMAF</span>;
        ISSN:<span class="NLM_cas:issn">0014-2980</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Disruption of the interaction of bromo and extraterminal (BET) proteins with acetylated histones using small mol. inhibitors suppresses Myc-driven cancers and TLR-induced inflammation in mouse models.  The predominant mechanism of BET inhibitor action is to suppress BET-mediated recruitment of pos. transcription elongation factor b and, thus, transcription elongation.  We investigated the effects of BET inhibitor I-BET151 on transcriptional responses to TLR4 and TNF in primary human monocytes and also on responses to cytokines IFN-β, IFN-γ, IL-4, and IL-10, which activate the JAK-STAT signaling pathway and are important for monocyte polarization and inflammatory diseases.  I-BET151 suppressed TLR4- and TNF-induced IFN responses by diminishing both autocrine IFN-β expression and transcriptional responses to IFN-β.  I-BET151 inhibited cytokine-induced transcription of STAT targets in a gene-specific manner without affecting STAT activation or recruitment.  This inhibition was independent of Myc or other upstream activators.  IFN-stimulated gene transcription is regulated primarily at the level of transcription initiation.  Accordingly, we found that I-BET151 suppressed the recruitment of transcriptional machinery to the CXCL10 promoter and an upstream enhancer.  Our findings suggest that BET inhibition reduces inflammation partially through suppressing cytokine activity and expands the understanding of the inhibitory and potentially selective immunosuppressive effects of inhibiting BET proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbE0f71lXpp7Vg90H21EOLACvtfcHk0lid6ac-2Tw4Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP&md5=efa081bc6c79db9233ae1a99ebf49945</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Feji.201444862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201444862%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DYarilina%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DIvashkiv%26aufirst%3DL.%2BB.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520transcriptional%2520responses%2520to%2520cytokine-Jak-STAT%2520signaling%2520in%2520a%2520gene-specific%2520manner%2520in%2520human%2520monocytes%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2015%26volume%3D45%26spage%3D287%26epage%3D297%26doi%3D10.1002%2Feji.201444862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0lid6ac-2Tw4Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET protein function is required for inflammation: Brd2 genetic disruption and BET inhibitor JQ1 impair mouse macrophage inflammatory responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">3670</span>– <span class="NLM_lpage">3678</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+protein+function+is+required+for+inflammation%3A+Brd2+genetic+disruption+and+BET+inhibitor+JQ1+impair+mouse+macrophage+inflammatory+responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-α while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0lhaQKCB3NgF-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520protein%2520function%2520is%2520required%2520for%2520inflammation%253A%2520Brd2%2520genetic%2520disruption%2520and%2520BET%2520inhibitor%2520JQ1%2520impair%2520mouse%2520macrophage%2520inflammatory%2520responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26spage%3D3670%26epage%3D3678%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.+L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- α, interleukin (IL)-1β and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- α and IL-1β .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0lhaQKCB3NgF-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.%2BL.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0lgt7I6GaVhVCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORα/γ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0lgv0lJ_xKahUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCrescenzo, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devraj, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellsworth, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fobian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krieger-Burke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loesel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whiteley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Physiochemical drug properties associated with in vivo toxicological outcomes</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4872</span>– <span class="NLM_lpage">4875</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.07.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.bmcl.2008.07.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=18691886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4872-4875&author=J.+D.+Hughesauthor=J.+Blaggauthor=D.+A.+Priceauthor=S.+Baileyauthor=G.+A.+DeCrescenzoauthor=R.+V.+Devrajauthor=E.+Ellsworthauthor=Y.+M.+Fobianauthor=M.+E.+Gibbsauthor=R.+W.+Gillesauthor=N.+Greeneauthor=E.+Huangauthor=T.+Krieger-Burkeauthor=J.+Loeselauthor=T.+Wagerauthor=L.+Whiteleyauthor=Y.+Zhang&title=Physiochemical+drug+properties+associated+with+in+vivo+toxicological+outcomes&doi=10.1016%2Fj.bmcl.2008.07.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Physiochemical drug properties associated with in vivo toxicological outcomes</span></div><div class="casAuthors">Hughes, Jason D.; Blagg, Julian; Price, David A.; Bailey, Simon; DeCrescenzo, Gary A.; Devraj, Rajesh V.; Ellsworth, Edmund; Fobian, Yvette M.; Gibbs, Michael E.; Gilles, Richard W.; Greene, Nigel; Huang, Enoch; Krieger-Burke, Teresa; Loesel, Jens; Wager, Travis; Whiteley, Larry; Zhang, Yao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4872-4875</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Relationships between physicochem. drug properties and toxicity were inferred from a data set consisting of animal in vivo toleration (IVT) studies on 245 preclin. Pfizer compds.; an increased likelihood of toxic events was found for less polar, more lipophilic compds.  This trend held across a wide range of types of toxicity and across a broad swath of chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPMmGEpVF0mbVg90H21EOLACvtfcHk0lgv0lJ_xKahUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE&md5=80ea5ced359112a363f38b3545cd7646</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.071%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BD.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DBailey%26aufirst%3DS.%26aulast%3DDeCrescenzo%26aufirst%3DG.%2BA.%26aulast%3DDevraj%26aufirst%3DR.%2BV.%26aulast%3DEllsworth%26aufirst%3DE.%26aulast%3DFobian%26aufirst%3DY.%2BM.%26aulast%3DGibbs%26aufirst%3DM.%2BE.%26aulast%3DGilles%26aufirst%3DR.%2BW.%26aulast%3DGreene%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DE.%26aulast%3DKrieger-Burke%26aufirst%3DT.%26aulast%3DLoesel%26aufirst%3DJ.%26aulast%3DWager%26aufirst%3DT.%26aulast%3DWhiteley%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPhysiochemical%2520drug%2520properties%2520associated%2520with%2520in%2520vivo%2520toxicological%2520outcomes%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4872%26epage%3D4875%26doi%3D10.1016%2Fj.bmcl.2008.07.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: A novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+A+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2eb7aWYmzHJcdrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520A%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=32193360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=387-394&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.+M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.+J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immunoinflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span></div><div class="casAuthors">Gilan, Omer; Rioja, Inmaculada; Knezevic, Kathy; Bell, Matthew J.; Yeung, Miriam M.; Harker, Nicola R.; Lam, Enid Y. N.; Chung, Chun-wa; Bamborough, Paul; Petretich, Massimo; Urh, Marjeta; Atkinson, Stephen J.; Bassil, Anna K.; Roberts, Emma J.; Vassiliadis, Dane; Burr, Marian L.; Preston, Alex G. S.; Wellaway, Christopher; Werner, Thilo; Gray, James R.; Michon, Anne-Marie; Gobbetti, Thomas; Kumar, Vinod; Soden, Peter E.; Haynes, Andrea; Vappiani, Johanna; Tough, David F.; Taylor, Simon; Dawson, Sarah-Jane; Bantscheff, Marcus; Lindon, Matthew; Drewes, Gerard; Demont, Emmanuel H.; Daniels, Danette L.; Grandi, Paola; Prinjha, Rab K.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">387-394</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The two tandem bromodomains of the BET (bromodomain and extraterminal domain) proteins enable chromatin binding to facilitate transcription.  Drugs that inhibit both bromodomains equally have shown efficacy in certain malignant and inflammatory conditions.  To explore the individual functional contributions of the first (BD1) and second (BD2) bromodomains in biol. and therapy, we developed selective BD1 and BD2 inhibitors.  We found that steady-state gene expression primarily requires BD1, whereas the rapid increase of gene expression induced by inflammatory stimuli requires both BD1 and BD2 of all BET proteins.  BD1 inhibitors phenocopied the effects of pan-BET inhibitors in cancer models, whereas BD2 inhibitors were predominantly effective in models of inflammatory and autoimmune disease.  These insights into the differential requirement of BD1 and BD2 for the maintenance and induction of gene expression may guide future BET-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr078AuJubsqbVg90H21EOLACvtfcHk0li_H22G7Q3DdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D&md5=d78cfea091fcf5ee18735b02b2120dd7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immunoinflammation%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2020%26volume%3D368%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Benzo[ cd]indol-2(1 H)-ones and Pyrrolo[4,3,2- de]quinolin-2(1 H)-ones as bromodomain and extra-terminal domain (BET) Inhibitors with selectivity for the first bromodomain with potential high efficiency against acute gouty arthritis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11080</span>– <span class="NLM_lpage">11107</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11080-11107&author=F.+Jiangauthor=Q.+Huauthor=Z.+Zhangauthor=H.+Liauthor=H.+Liauthor=D.+Zhangauthor=H.+Liauthor=Y.+Maauthor=J.+Xuauthor=H.+Chenauthor=Y.+Cuiauthor=Y.+Zhiauthor=Y.+Zhangauthor=J.+Xuauthor=J.+Zhuauthor=T.+Luauthor=Y.+Chen&title=Discovery+of+Benzo%5B+cd%5Dindol-2%281+H%29-ones+and+Pyrrolo%5B4%2C3%2C2-+de%5Dquinolin-2%281+H%29-ones+as+bromodomain+and+extra-terminal+domain+%28BET%29+Inhibitors+with+selectivity+for+the+first+bromodomain+with+potential+high+efficiency+against+acute+gouty+arthritis&doi=10.1021%2Facs.jmedchem.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis</span></div><div class="casAuthors">Jiang, Fei; Hu, Qinghua; Zhang, Zhimin; Li, Hongmei; Li, Huili; Zhang, Dewei; Li, Hanwen; Ma, Yu; Xu, Jingjing; Chen, Haifang; Cui, Yong; Zhi, Yanle; Zhang, Yanmin; Xu, Junyu; Zhu, Jiapeng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11080-11107</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues.  Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2).  Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family.  Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1.  Through structure-based optimization, highly active and selective compds. are ultimately obtained.  The representative compds. are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2).  Among them, we further show that 68 (LT052) mediates BRD4/NF-κB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model.  Therefore, selective pharmacol. modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav2e3P7XQ1LVg90H21EOLACvtfcHk0lgnhIor_02MOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E&md5=5cce669e137812d20f8ba2a9785f4666</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Benzo%255B%2520cd%255Dindol-2%25281%2520H%2529-ones%2520and%2520Pyrrolo%255B4%252C3%252C2-%2520de%255Dquinolin-2%25281%2520H%2529-ones%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520Inhibitors%2520with%2520selectivity%2520for%2520the%2520first%2520bromodomain%2520with%2520potential%2520high%2520efficiency%2520against%2520acute%2520gouty%2520arthritis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11080%26epage%3D11107%26doi%3D10.1021%2Facs.jmedchem.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK789: A selective Inhibitor of the first bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>– <span class="NLM_lpage">9069</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.+W.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=GSK789%3A+A+selective+Inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+Bromo+and+Extra+Terminal+Domain+%28BET%29+Proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0lhMVQ0RW3B-ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK789%253A%2520A%2520selective%2520Inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520Bromo%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as Bromodomain and Extra-Terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+Bromodomain+and+Extra-Terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0lhMVQ0RW3B-ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520Bromodomain%2520and%2520Extra-Terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thesmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK973 is an inhibitor of the second bromodomains (BD2s) of the Bromodomain and Extra-Terminal (BET) family</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1581</span>– <span class="NLM_lpage">1587</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlegsrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1581-1587&author=A.+Prestonauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=L.+J.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=A.+J.+Lewisauthor=D.+Lugoauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=P.+Thesmarauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=GSK973+is+an+inhibitor+of+the+second+bromodomains+%28BD2s%29+of+the+Bromodomain+and+Extra-Terminal+%28BET%29+family&doi=10.1021%2Facsmedchemlett.0c00247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">GSK973 Is an Inhibitor of the Second Bromodomains (BD2s) of the Bromodomain and Extra-Terminal (BET) Family</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Gordon, Laurie J.; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Lewis, Antonia J.; Lugo, David; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rioja, Inmaculada; Seal, Jon; Taylor, Simon; Thesmar, Pierre; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1581-1587</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-BET inhibitors have shown profound efficacy in a no. of in vivo preclin. models and have entered the clinic in oncol. trials where adverse events have been reported.  These inhibitors interact equipotently with the eight bromodomains of the BET family of proteins.  To better understand the contribution of each domain to their efficacy and to improve from their safety profile, selective inhibitors are required.  This Letter discloses the profile of GSK973, a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive preclin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqErinCVgS7LVg90H21EOLACvtfcHk0li8bDQ53w1PQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlegsrfL&md5=93fbcba928a55167dae6825e87548a28</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00247%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DThesmar%26aufirst%3DP.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK973%2520is%2520an%2520inhibitor%2520of%2520the%2520second%2520bromodomains%2520%2528BD2s%2529%2520of%2520the%2520Bromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520family%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1581%26epage%3D1587%26doi%3D10.1021%2Facsmedchemlett.0c00247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak Bromo and Extra Terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>– <span class="NLM_lpage">9126</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+B.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+Bromo+and+Extra+Terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0li8bDQ53w1PQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCopley%26aufirst%3DR.%2BC.%2BB.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520Bromo%2520and%2520Extra%2520Terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Template-hopping approach leads to potent, selective, and highly soluble Bromo and Extraterminal Domain (BET) second bromodomain (BD2) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">24</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c02156</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c02156" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=24&author=H.+E.+Aylottauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=A.+Bassilauthor=C.+W.+Chungauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=C.+Messengerauthor=D.+Mitchellauthor=A.+Phillipouauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+T.+Sealauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=Template-hopping+approach+leads+to+potent%2C+selective%2C+and+highly+soluble+Bromo+and+Extraterminal+Domain+%28BET%29+second+bromodomain+%28BD2%29+inhibitors&doi=10.1021%2Facs.jmedchem.0c02156"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c02156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c02156%26sid%3Dliteratum%253Aachs%26aulast%3DAylott%26aufirst%3DH.%2BE.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBassil%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DTemplate-hopping%2520approach%2520leads%2520to%2520potent%252C%2520selective%252C%2520and%2520highly%2520soluble%2520Bromo%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D24%26doi%3D10.1021%2Facs.jmedchem.0c02156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anglin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ucisik, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faver, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyshadham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storl-Desmond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of bromodomain 2-specific inhibitors of BRDT</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">2021102118</span>, <span class="refDoi"> DOI: 10.1073/pnas.2021102118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.2021102118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXls12qur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2021&pages=2021102118&author=Z.+Yuauthor=A.+F.+Kuauthor=J.+L.+Anglinauthor=R.+Sharmaauthor=M.+N.+Ucisikauthor=J.+C.+Faverauthor=F.+Liauthor=P.+Nyshadhamauthor=N.+Simmonsauthor=K.+L.+Sharmaauthor=S.+Nagarajanauthor=K.+Riehleauthor=G.+Kaurauthor=B.+Sankaranauthor=M.+Storl-Desmondauthor=S.+S.+Palmerauthor=D.+W.+Youngauthor=C.+Kimauthor=M.+M.+Matzuk&title=Discovery+and+characterization+of+bromodomain+2-specific+inhibitors+of+BRDT&doi=10.1073%2Fpnas.2021102118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of bromodomain 2-specific inhibitors of BRDT</span></div><div class="casAuthors">Yu, Zhifeng; Ku, Angela F.; Anglin, Justin L.; Sharma, Rajesh; Ucisik, Melek Nihan; Faver, John C.; Li, Feng; Nyshadham, Pranavanand; Simmons, Nicholas; Sharma, Kiran L.; Nagarajan, Sureshbabu; Riehle, Kevin; Kaur, Gundeep; Sankaran, Banumathi; Storl-Desmond, Marta; Palmer, Stephen S.; Young, Damian W.; Kim, Choel; Matzuk, Martin M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e2021102118</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain testis (BRDT), a member of the bromodomain and extraterminal (BET) subfamily that includes the cancer targets BRD2, BRD3, and BRD4, is a validated contraceptive target.  All BET subfamily members have two tandem bromodomains (BD1 and BD2).  Knockout mice lacking BRDT-BD1 or both bromodomains are infertile.  Treatment of mice with JQ1, a BET BD1/BD2 nonselective inhibitor with the highest affinity for BRD4, disrupts spermatogenesis and reduces sperm no. and motility.  To assess the contribution of each BRDT bromodomain, we screened our collection of DNA-encoded chem. libraries for BRDT-BD1 and BRDT-BD2 binders.  High-enrichment hits were identified and resynthesized off-DNA and examd. for their ability to compete with JQ1 in BRDT and BRD4 bromodomain AlphaScreen assays.  These studies identified CDD-1102 as a selective BRDT-BD2 inhibitor with low nanomolar potency and &gt;1,000-fold selectivity over BRDT-BD1.  Structure-activity relationship studies of CDD-1102 produced a series of addnl. BRDT-BD2/BRD4-BD2 selective inhibitors, including CDD-1302, a truncated analog of CDD-1102 with similar activity, and CDD-1349, an analog with sixfold selectivity for BRDT-BD2 vs. BRD4-BD2.  BROMOscan bromodomain profiling confirmed the great affinity and selectivity of CDD-1102 and CDD-1302 on all BET BD2 vs. BD1 with the highest affinity for BRDT-BD2.  Cocrystals of BRDT-BD2 with CDD-1102 and CDD-1302 were detd. at 2.27 and 1.90 Å resoln., resp., and revealed BRDT-BD2 specific contacts that explain the high affinity and selectivity of these compds.  These BD2-specific compds. and their binding to BRDT-BD2 are unique compared with recent reports and enable further evaluation of their nonhormonal contraceptive potential in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe2-U6Ypwvm7Vg90H21EOLACvtfcHk0lhhhXDfgfTgtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXls12qur0%253D&md5=ff66b0f0b9ec50d33e8fafdc96303f8f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2021102118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2021102118%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DKu%26aufirst%3DA.%2BF.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DUcisik%26aufirst%3DM.%2BN.%26aulast%3DFaver%26aufirst%3DJ.%2BC.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DNyshadham%26aufirst%3DP.%26aulast%3DSimmons%26aufirst%3DN.%26aulast%3DSharma%26aufirst%3DK.%2BL.%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DRiehle%26aufirst%3DK.%26aulast%3DKaur%26aufirst%3DG.%26aulast%3DSankaran%26aufirst%3DB.%26aulast%3DStorl-Desmond%26aufirst%3DM.%26aulast%3DPalmer%26aufirst%3DS.%2BS.%26aulast%3DYoung%26aufirst%3DD.%2BW.%26aulast%3DKim%26aufirst%3DC.%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520bromodomain%25202-specific%2520inhibitors%2520of%2520BRDT%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2021%26volume%3D118%26spage%3D2021102118%26doi%3D10.1073%2Fpnas.2021102118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span> <span> </span><span class="NLM_article-title">Domain-selective targeting of BET proteins in cancer and immunological diseases</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2020.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.cbpa.2020.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=32741705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2020&pages=184-193&author=M.+Petretichauthor=E.+H.+Demontauthor=P.+Grandi&title=Domain-selective+targeting+of+BET+proteins+in+cancer+and+immunological+diseases&doi=10.1016%2Fj.cbpa.2020.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Domain-selective targeting of BET proteins in cancer and immunological diseases</span></div><div class="casAuthors">Petretich, Massimo; Demont, Emmanuel H.; Grandi, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">184-193</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review Cancer and inflammation are strongly interconnected processes.  Chronic inflammatory pathologies can be at the heart of tumor development; similarly, tumor-elicited inflammation is a consequence of many cancers.  The mechanistic interdependence between cancer and inflammatory pathologies points toward common protein effectors which represent potential shared targets for pharmacol. intervention.  Epigenetic mechanisms often drive resistance to cancer therapy and immunomodulatory strategies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic adapters which play a major role in controlling cell proliferation and the prodn. of inflammatory mediators.  A plethora of small mols. aimed at inhibiting BET protein function to treat cancer and inflammatory diseases have populated academic and industry efforts in the last 10 years.  In this review, we will discuss recent pharmacol. approaches aimed at targeting a single or a subset of the eight bromodomains within the BET family which have the potential to tease apart clin. efficacy and safety signals of BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKVLw12H8j5LVg90H21EOLACvtfcHk0lhhhXDfgfTgtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbjM&md5=82a99a0ebbcda749e42effa2820e232e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2020.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGrandi%26aufirst%3DP.%26atitle%3DDomain-selective%2520targeting%2520of%2520BET%2520proteins%2520in%2520cancer%2520and%2520immunological%2520diseases%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D57%26spage%3D184%26epage%3D193%26doi%3D10.1016%2Fj.cbpa.2020.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0lhkTpa54mM8zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the Bromodomain and Extra Terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>– <span class="NLM_lpage">9092</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+Bromodomain+and+Extra+Terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0lhkTpa54mM8zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target validation using chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0lisgWPfHJzXdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">Encoded combinatorial chemistry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">5381</span>– <span class="NLM_lpage">5383</span>, <span class="refDoi"> DOI: 10.1073/pnas.89.12.5381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.89.12.5381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1608946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=5381-5383&author=S.+Brennerauthor=R.+A.+Lerner&title=Encoded+combinatorial+chemistry&doi=10.1073%2Fpnas.89.12.5381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded combinatorial chemistry</span></div><div class="casAuthors">Brenner, Sydney; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5381-3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The diversity of chem. synthesis and the power of genetics are linked to provide a powerful, versatile method for drug screening.  A process of alternating parallel combinatorial synthesis is used to encode individual members of a large library of chems. with unique nucleotide sequences.  After the chem. entity is bound to a target, the genetic tag can be amplified by replication and utilized for enrichment of the bound mols. by serial hybridization to a subset of the library.  The library of the tagged chems. is termed an encoded combinatorial chem. library.  The nature of the chem. structure bound to the receptor is decoded by sequencing the nucleotide tag.  For example, with a library of peptides, each peptide was linked to a specifically designed oligodeoxynucleotide (genetic tag).  Each of these oligodeoxynucleotides had a signature (unique) sequence corresponding to the linked peptide and was flanked by sequences recognizable by designed DNA primers of the polymerase chain reaction (PCR).  After a desired member of this library of peptide-oligonucleotides was selected by screening (e.g., selected by binding to a target), this desired member was then identified by PCR amplification of the signature nucleotide sequence.  Detn. of the signature sequence from PCR amplified DNA gave the identity of the desired peptide.  Thus, this method can be used to identify a few members in a large chem. library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxFs-UqY_2JbVg90H21EOLACvtfcHk0lisgWPfHJzXdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D&md5=3f03fc103c039ce7985cd7b25142c7ed</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.12.5381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.12.5381%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DS.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DEncoded%2520combinatorial%2520chemistry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1992%26volume%3D89%26spage%3D5381%26epage%3D5383%26doi%3D10.1073%2Fpnas.89.12.5381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mannocci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimbacher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bellis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">High-throughput sequencing allows the identification of binding molecules isolated from DNA-encoded chemical libraries</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">17670</span>– <span class="NLM_lpage">17675</span>, <span class="refDoi"> DOI: 10.1073/pnas.0805130105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.0805130105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=19001273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVCmtbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=17670-17675&author=L.+Mannocciauthor=Y.+Zhangauthor=J.+Scheuermannauthor=M.+Leimbacherauthor=G.+De+Bellisauthor=E.+Rizziauthor=C.+Dumelinauthor=S.+Melkkoauthor=D.+Neri&title=High-throughput+sequencing+allows+the+identification+of+binding+molecules+isolated+from+DNA-encoded+chemical+libraries&doi=10.1073%2Fpnas.0805130105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput sequencing allows the identification of binding molecules isolated from DNA-encoded chemical libraries</span></div><div class="casAuthors">Mannocci, Luca; Zhang, Yixin; Scheuermann, Jorg; Leimbacher, Markus; De Bellis, Gianluca; Rizzi, Ermanno; Dumelin, Christoph; Melkko, Samu; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">17670-17675</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">DNA encoding facilitates the construction and screening of large chem. libraries.  Here, we describe general strategies for the stepwise coupling of coding DNA fragments to nascent org. mols. throughout individual reaction steps as well as the first implementation of high-throughput sequencing for the identification and relative quantification of the library members.  The methodol. was exemplified in the construction of a DNA-encoded chem. library contg. 4,000 compds. and in the discovery of binders to streptavidin, matrix metalloproteinase 3, and polyclonal human IgG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrsWWaqWxcEbVg90H21EOLACvtfcHk0ljjh_mAIzRJXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVCmtbbF&md5=d104229313e30f1df784a5d501a952c5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0805130105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0805130105%26sid%3Dliteratum%253Aachs%26aulast%3DMannocci%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DScheuermann%26aufirst%3DJ.%26aulast%3DLeimbacher%26aufirst%3DM.%26aulast%3DDe%2BBellis%26aufirst%3DG.%26aulast%3DRizzi%26aufirst%3DE.%26aulast%3DDumelin%26aufirst%3DC.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DHigh-throughput%2520sequencing%2520allows%2520the%2520identification%2520of%2520binding%2520molecules%2520isolated%2520from%2520DNA-encoded%2520chemical%2520libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D17670%26epage%3D17675%26doi%3D10.1073%2Fpnas.0805130105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span>; <span class="NLM_string-name">Lu, X.</span>; <span class="NLM_string-name">Phelps, C.</span>; <span class="NLM_string-name">Evindar, G.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded Library Technology (ELT)</span>. In  <i>Chemical and Biological Synthesis: Enabling Approaches for Understanding Biology</i><i>;</i> <span class="NLM_publisher-name">Royal Society of Chemistry</span>, <span class="NLM_year">2018</span>; pp  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">183</span><span class="refDoi"> DOI: 10.1039/9781788012805-00153</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1039%2F9781788012805-00153" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=153-183&author=G.+Yao&author=X.+Lu&author=C.+Phelps&author=G.+Evindar&title=Chemical+and+Biological+Synthesis%3A+Enabling+Approaches+for+Understanding+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2F9781788012805-00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781788012805-00153%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DG.%26atitle%3DDNA-encoded%2520Library%2520Technology%2520%2528ELT%2529%26btitle%3DChemical%2520and%2520Biological%2520Synthesis%253A%2520Enabling%2520Approaches%2520for%2520Understanding%2520Biology%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2018%26spage%3D153%26epage%3D183%26doi%3D10.1039%2F9781788012805-00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. wa</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dossang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddershaw, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimaszewska, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haq, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakhria, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jigorel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olszewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronzoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowedder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüdiger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomas, D. A.</span></span> <span> </span><span class="NLM_article-title">The development of highly potent and selective small molecule correctors of Z α1-antitrypsin misfolding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">127973</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2021.127973</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.bmcl.2021.127973" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=33753261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXotFahurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=127973&author=J.+Liddleauthor=A.+C.+Pearceauthor=C.+Arico-Muendelauthor=S.+Belyanskayaauthor=A.+Brewsterauthor=M.+Brownauthor=C.+wa+Chungauthor=A.+Denisauthor=N.+Dodicauthor=A.+Dossangauthor=P.+Eddershawauthor=D.+Klimaszewskaauthor=I.+Haqauthor=D.+S.+Holmesauthor=A.+Jaggerauthor=T.+Jakhriaauthor=E.+Jigorelauthor=K.+Lindauthor=J.+Messerauthor=M.+Neuauthor=A.+Olszewskiauthor=R.+Ronzoniauthor=J.+Rowedderauthor=M.+R%C3%BCdigerauthor=S.+Skinnerauthor=K.+J.+Smithauthor=L.+Trottetauthor=I.+Uingsauthor=Z.+Zhuauthor=J.+A.+Irvingauthor=D.+A.+Lomas&title=The+development+of+highly+potent+and+selective+small+molecule+correctors+of+Z+%CE%B11-antitrypsin+misfolding&doi=10.1016%2Fj.bmcl.2021.127973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The development of highly potent and selective small molecule correctors of Z α1-antitrypsin misfolding</span></div><div class="casAuthors">Liddle, John; Pearce, Andrew C.; Arico-Muendel, Christopher; Belyanskaya, Svetlana; Brewster, Andrew; Brown, Murray; Chung, Chun-wa; Denis, Alexis; Dodic, Nerina; Dossang, Anthony; Eddershaw, Peter; Klimaszewska, Diana; Haq, Imran; Holmes, Duncan S.; Jagger, Alistair; Jakhria, Toral; Jigorel, Emilie; Lind, Ken; Messer, Jeff; Neu, Margaret; Olszewski, Allison; Ronzoni, Riccardo; Rowedder, James; Rudiger, Martin; Skinner, Steve; Smith, Kathrine J.; Trottet, Lionel; Uings, Iain; Zhu, Zhengrong; Irving, James A.; Lomas, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127973</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">α1-antitrypsin deficiency is characterised by the misfolding and intracellular polymn. of mutant α1-antitrypsin protein within the endoplasmic reticulum (ER) of hepatocytes.  Small mols. that bind and stabilize Z α1-antitrypsin were identified via a DNA-encoded library screen.  A subsequent structure based optimization led to a series of highly potent, selective and cellular active α1-antitrypsin correctors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopH8iP7w6_wrVg90H21EOLACvtfcHk0ljjh_mAIzRJXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXotFahurw%253D&md5=3f5849489c9c53f681b58b7712123dd5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2021.127973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2021.127973%26sid%3Dliteratum%253Aachs%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DPearce%26aufirst%3DA.%2BC.%26aulast%3DArico-Muendel%26aufirst%3DC.%26aulast%3DBelyanskaya%26aufirst%3DS.%26aulast%3DBrewster%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2Bwa%26aulast%3DDenis%26aufirst%3DA.%26aulast%3DDodic%26aufirst%3DN.%26aulast%3DDossang%26aufirst%3DA.%26aulast%3DEddershaw%26aufirst%3DP.%26aulast%3DKlimaszewska%26aufirst%3DD.%26aulast%3DHaq%26aufirst%3DI.%26aulast%3DHolmes%26aufirst%3DD.%2BS.%26aulast%3DJagger%26aufirst%3DA.%26aulast%3DJakhria%26aufirst%3DT.%26aulast%3DJigorel%26aufirst%3DE.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMesser%26aufirst%3DJ.%26aulast%3DNeu%26aufirst%3DM.%26aulast%3DOlszewski%26aufirst%3DA.%26aulast%3DRonzoni%26aufirst%3DR.%26aulast%3DRowedder%26aufirst%3DJ.%26aulast%3DR%25C3%25BCdiger%26aufirst%3DM.%26aulast%3DSkinner%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DK.%2BJ.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DUings%26aufirst%3DI.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DIrving%26aufirst%3DJ.%2BA.%26aulast%3DLomas%26aufirst%3DD.%2BA.%26atitle%3DThe%2520development%2520of%2520highly%2520potent%2520and%2520selective%2520small%2520molecule%2520correctors%2520of%2520Z%2520%25CE%25B11-antitrypsin%2520misfolding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D41%26spage%3D127973%26doi%3D10.1016%2Fj.bmcl.2021.127973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Gall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, K. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vimal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Bromodomain and Extraterminal inhibitor with a low predicted human dose through synergistic use of encoded library technology and fragment screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01670</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01670" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFagtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=714-746&author=C.+R.+Wellawayauthor=D.+Amansauthor=P.+Bamboroughauthor=H.+Barnettauthor=R.+A.+Bitauthor=J.+A.+Brownauthor=N.+R.+Carlsonauthor=C.+W.+Chungauthor=A.+W.+J.+Cooperauthor=P.+D.+Craggsauthor=R.+P.+Davisauthor=T.+W.+Deanauthor=J.+P.+Evansauthor=L.+Gordonauthor=I.+L.+Haradaauthor=D.+J.+Hirstauthor=P.+G.+Humphreysauthor=K.+L.+Jonesauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+Lugoauthor=M.+Mahmoodauthor=S.+McClearyauthor=P.+Medeirosauthor=D.+J.+Mitchellauthor=M.+O%E2%80%99Sullivanauthor=A.+Le+Gallauthor=V.+K.+Patelauthor=C.+Pattenauthor=D.+L.+Pooleauthor=R.+R.+Shahauthor=J.+E.+Smithauthor=K.+A.+J.+Staffordauthor=P.+J.+Thomasauthor=M.+Vimalauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=N.+Wellawayauthor=G.+Yaoauthor=R.+K.+Prinjha&title=Discovery+of+a+Bromodomain+and+Extraterminal+inhibitor+with+a+low+predicted+human+dose+through+synergistic+use+of+encoded+library+technology+and+fragment+screening&doi=10.1021%2Facs.jmedchem.9b01670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening</span></div><div class="casAuthors">Wellaway, Christopher R.; Amans, Dominique; Bamborough, Paul; Barnett, Heather; Bit, Rino A.; Brown, Jack A.; Carlson, Neil R.; Chung, Chun-wa; Cooper, Anthony W. J.; Craggs, Peter D.; Davis, Robert P.; Dean, Tony W.; Evans, John P.; Gordon, Laurie; Harada, Isobel L.; Hirst, David J.; Humphreys, Philip G.; Jones, Katherine L.; Lewis, Antonia J.; Lindon, Matthew J.; Lugo, Dave; Mahmood, Mahnoor; McCleary, Scott; Medeiros, Patricia; Mitchell, Darren J.; O'Sullivan, Michael; Le Gall, Armelle; Patel, Vipulkumar K.; Patten, Chris; Poole, Darren L.; Shah, Rishi R.; Smith, Jane E.; Stafford, Kayleigh A. J.; Thomas, Pamela J.; Vimal, Mythily; Wall, Ian D.; Watson, Robert J.; Wellaway, Natalie; Yao, Gang; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">714-746</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) family of bromodomain-contg. proteins are important regulators of the epigenome through their ability to recognize N-acetyl lysine (KAc) post-translational modifications on histone tails.  These interactions have been implicated in various disease states and, consequently, disruption of BET-KAc binding has emerged as an attractive therapeutic strategy with a no. of small mol. inhibitors now under investigation in the clinic.  However, until the utility of these advanced candidates is fully assessed by these trials, there remains scope for the discovery of inhibitors from new chemotypes with alternative physicochem., pharmacokinetic, and pharmacodynamic profiles.  Herein, we describe the discovery of a candidate-quality dimethylpyridone benzimidazole compd. which originated from the hybridization of a dimethylphenol benzimidazole series, identified using encoded library technol., with an N-Me pyridone series identified through fragment screening.  Optimization via structure- and property-based design led to I-BET469, which possesses favorable oral pharmacokinetic properties, displays activity in vivo, and is projected to have a low human efficacious dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJtEmfPcCFtbVg90H21EOLACvtfcHk0lj-Ut0TuF7gBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFagtw%253D%253D&md5=826ebe53a3091893c9e6c6c4830ac388</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01670%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DAmans%26aufirst%3DD.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCooper%26aufirst%3DA.%2BW.%2BJ.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDavis%26aufirst%3DR.%2BP.%26aulast%3DDean%26aufirst%3DT.%2BW.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHarada%26aufirst%3DI.%2BL.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJones%26aufirst%3DK.%2BL.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DMahmood%26aufirst%3DM.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DM.%26aulast%3DLe%2BGall%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DV.%2BK.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DPoole%26aufirst%3DD.%2BL.%26aulast%3DShah%26aufirst%3DR.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BE.%26aulast%3DStafford%26aufirst%3DK.%2BA.%2BJ.%26aulast%3DThomas%26aufirst%3DP.%2BJ.%26aulast%3DVimal%26aufirst%3DM.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWellaway%26aufirst%3DN.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520a%2520Bromodomain%2520and%2520Extraterminal%2520inhibitor%2520with%2520a%2520low%2520predicted%2520human%2520dose%2520through%2520synergistic%2520use%2520of%2520encoded%2520library%2520technology%2520and%2520fragment%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D714%26epage%3D746%26doi%3D10.1021%2Facs.jmedchem.9b01670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kazmierski, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De La Rosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favre, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washio, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mebrahtu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basilla, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evindar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olszewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabhu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samano, V.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded library technology-based discovery, lead optimization, and prodrug strategy toward structurally unique Indoleamine 2,3-Dioxygenase-1 (IDO1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3552</span>– <span class="NLM_lpage">3562</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01799</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01799" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVOms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3552-3562&author=W.+M.+Kazmierskiauthor=B.+Xiaauthor=J.+Millerauthor=M.+De+La+Rosaauthor=D.+Favreauthor=R.+M.+Dunhamauthor=Y.+Washioauthor=Z.+Zhuauthor=F.+Wangauthor=M.+Mebrahtuauthor=H.+Dengauthor=J.+Basillaauthor=L.+Wangauthor=G.+Evindarauthor=L.+Fanauthor=A.+Olszewskiauthor=N.+Prabhuauthor=C.+Davieauthor=J.+A.+Messerauthor=V.+Samano&title=DNA-Encoded+library+technology-based+discovery%2C+lead+optimization%2C+and+prodrug+strategy+toward+structurally+unique+Indoleamine+2%2C3-Dioxygenase-1+%28IDO1%29+inhibitors&doi=10.1021%2Facs.jmedchem.9b01799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Technology-Based Discovery, Lead Optimization, and Prodrug Strategy toward Structurally Unique Indoleamine 2,3-Dioxygenase-1 (IDO1) Inhibitors</span></div><div class="casAuthors">Kazmierski, Wieslaw M.; Xia, Bing; Miller, John; De la Rosa, Martha; Favre, David; Dunham, Richard M.; Washio, Yoshiaki; Zhu, Zhengrong; Wang, Feng; Mebrahtu, Makda; Deng, Hongfeng; Basilla, Jonathan; Wang, Liping; Evindar, Ghotas; Fan, Lijun; Olszewski, Alison; Prabhu, Ninad; Davie, Christopher; Messer, Jeffrey A.; Samano, Vicente</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3552-3562</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the discovery of a novel indoleamine 2,3-dioxygenase-1 (IDO1) inhibitor class through the affinity selection of a previously unreported indole-based DNA-encoded library (DEL).  The DEL exemplar, spiro-chromane 1, had moderate IDO1 potency but high in vivo clearance.  Series optimization quickly afforded a potent, low in vivo clearance lead 11.  Although amorphous 11 was highly bio-available, cryst. 11 was poorly sol. and suffered disappointingly low bio-availability because of soly.-limited absorption.  A prodrug approach was deployed and proved effective in discovering the highly bio-available phosphonooxymethyl 31, which rapidly converted to 11 in vivo.  Obtaining cryst. 31 proved problematic, however; thus salt screening was performed in an attempt to circumvent this obstacle and successfully delivered greatly sol. and bio-available cryst. tris-salt 32.  IDO1 inhibitor 32 is characterized by a low calcd. human dose, best-in-class potential, and an unusual inhibition mode by binding the IDO1 heme-free (apo) form.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhSTj0UyMSxbVg90H21EOLACvtfcHk0lj_PQWT-8RvCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVOms7k%253D&md5=8cddeffb12cefd9b3e1c2e0641ad7226</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01799%26sid%3Dliteratum%253Aachs%26aulast%3DKazmierski%26aufirst%3DW.%2BM.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DDe%2BLa%2BRosa%26aufirst%3DM.%26aulast%3DFavre%26aufirst%3DD.%26aulast%3DDunham%26aufirst%3DR.%2BM.%26aulast%3DWashio%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMebrahtu%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DBasilla%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DEvindar%26aufirst%3DG.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DOlszewski%26aufirst%3DA.%26aulast%3DPrabhu%26aufirst%3DN.%26aulast%3DDavie%26aufirst%3DC.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DSamano%26aufirst%3DV.%26atitle%3DDNA-Encoded%2520library%2520technology-based%2520discovery%252C%2520lead%2520optimization%252C%2520and%2520prodrug%2520strategy%2520toward%2520structurally%2520unique%2520Indoleamine%25202%252C3-Dioxygenase-1%2520%2528IDO1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3552%26epage%3D3562%26doi%3D10.1021%2Facs.jmedchem.9b01799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhaus, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkämper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class Receptor Interacting Protein 1 (RIP1) kinase specific clinical candidate (GSK2982772) for the treatment of inflammatory diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1247-1261&author=P.+A.+Harrisauthor=S.+B.+Bergerauthor=J.+U.+Jeongauthor=R.+Nagillaauthor=D.+Bandyopadhyayauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=P.+M.+Eidamauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=J.+Kangauthor=V.+Kasparcovaauthor=B.+W.+Kingauthor=R.+Lehrauthor=Y.+Lanauthor=L.+K.+Leisterauthor=J.+D.+Lichauthor=T.+T.+MacDonaldauthor=N.+A.+Millerauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+Rahmanauthor=M.+A.+Reillyauthor=A.+R.+Rendinaauthor=E.+J.+Riveraauthor=M.+C.+Schaefferauthor=C.+A.+Sehonauthor=R.+R.+Singhausauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=A.+Vossenk%C3%A4mperauthor=P.+Wardauthor=D.+D.+Wisnoskiauthor=D.+Zhangauthor=R.+W.+Marquisauthor=P.+J.+Goughauthor=J.+Bertin&title=Discovery+of+a+first-in-class+Receptor+Interacting+Protein+1+%28RIP1%29+kinase+specific+clinical+candidate+%28GSK2982772%29+for+the+treatment+of+inflammatory+diseases&doi=10.1021%2Facs.jmedchem.6b01751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span></div><div class="casAuthors">Harris, Philip A.; Berger, Scott B.; Jeong, Jae U.; Nagilla, Rakesh; Bandyopadhyay, Deepak; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Eidam, Patrick M.; Finger, Joshua N.; Hoffman, Sandra J.; Kang, James; Kasparcova, Viera; King, Bryan W.; Lehr, Ruth; Lan, Yunfeng; Leister, Lara K.; Lich, John D.; MacDonald, Thomas T.; Miller, Nathan A.; Ouellette, Michael T.; Pao, Christina S.; Rahman, Attiq; Reilly, Michael A.; Rendina, Alan R.; Rivera, Elizabeth J.; Schaeffer, Michelle C.; Sehon, Clark A.; Singhaus, Robert R.; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Vossenkamper, Anna; Ward, Paris; Wisnoski, David D.; Zhang, Daohua; Marquis, Robert W.; Gough, Peter J.; Bertin, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RIP1 regulates necroptosis and inflammation and may play an important role in contributing to a variety of human pathologies, including immune-mediated inflammatory diseases.  Small-mol. inhibitors of RIP1 kinase that are suitable for advancement into the clinic have yet to be described.  Herein, the authors report the lead optimization of a benzoxazepinone hit from a DNA-encoded library and the discovery and profile of clin. candidate GSK2982772 (compd. I), currently in phase 2a clin. studies for psoriasis, rheumatoid arthritis, and ulcerative colitis.  Compd. I potently binds to RIP1 with exquisite kinase specificity and has excellent activity in blocking many TNF-dependent cellular responses.  Highlighting its potential as a novel anti-inflammatory agent, the inhibitor was also able to reduce spontaneous prodn. of cytokines from human ulcerative colitis explants.  The highly favorable physicochem. and ADMET properties of I, combined with high potency, led to a predicted low oral dose in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZGRDTQv-vs7Vg90H21EOLACvtfcHk0liN55okGb9NSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D&md5=ee0c3c60bb1351dff696490ac9edb1cc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01751%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DEidam%26aufirst%3DP.%2BM.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLan%26aufirst%3DY.%26aulast%3DLeister%26aufirst%3DL.%2BK.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DMiller%26aufirst%3DN.%2BA.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRahman%26aufirst%3DA.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DRivera%26aufirst%3DE.%2BJ.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSehon%26aufirst%3DC.%2BA.%26aulast%3DSinghaus%26aufirst%3DR.%2BR.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DVossenk%25C3%25A4mper%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520Receptor%2520Interacting%2520Protein%25201%2520%2528RIP1%2529%2520kinase%2520specific%2520clinical%2520candidate%2520%2528GSK2982772%2529%2520for%2520the%2520treatment%2520of%2520inflammatory%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Facs.jmedchem.6b01751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coote, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicker, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinod, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüdiger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slade, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of PAD4 activity is sufficient to disrupt mouse and human NET formation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnchembio.1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25622091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVehsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=189-191&author=H.+D.+Lewisauthor=J.+Liddleauthor=J.+E.+Cooteauthor=S.+J.+Atkinsonauthor=M.+D.+Barkerauthor=B.+D.+Baxauthor=K.+L.+Bickerauthor=R.+P.+Binghamauthor=M.+Campbellauthor=Y.+H.+Chenauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=R.+P.+Davisauthor=D.+Eberhardauthor=G.+Jobertyauthor=K.+E.+Lindauthor=K.+Lockeauthor=C.+Mallerauthor=K.+Martinodauthor=C.+Pattenauthor=O.+Polyakovaauthor=C.+E.+Riseauthor=M.+R%C3%BCdigerauthor=R.+J.+Sheppardauthor=D.+J.+Sladeauthor=P.+Thomasauthor=J.+Thorpeauthor=G.+Yaoauthor=G.+Drewesauthor=D.+D.+Wagnerauthor=P.+R.+Thompsonauthor=R.+K.+Prinjhaauthor=D.+M.+Wilson&title=Inhibition+of+PAD4+activity+is+sufficient+to+disrupt+mouse+and+human+NET+formation&doi=10.1038%2Fnchembio.1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of PAD4 activity is sufficient to disrupt mouse and human NET formation</span></div><div class="casAuthors">Lewis, Huw D.; Liddle, John; Coote, Jim E.; Atkinson, Stephen J.; Barker, Michael D.; Bax, Benjamin D.; Bicker, Kevin L.; Bingham, Ryan P.; Campbell, Matthew; Chen, Yu Hua; Chung, Chun-wa; Craggs, Peter D.; Davis, Rob P.; Eberhard, Dirk; Joberty, Gerard; Lind, Kenneth E.; Locke, Kelly; Maller, Claire; Martinod, Kimberly; Patten, Chris; Polyakova, Oxana; Rise, Cecil E.; Rudiger, Martin; Sheppard, Robert J.; Slade, Daniel J.; Thomas, Pamela; Thorpe, Jim; Yao, Gang; Drewes, Gerard; Wagner, Denisa D.; Thompson, Paul R.; Prinjha, Rab K.; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">189-191</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PAD4 has been strongly implicated in the pathogenesis of autoimmune, cardiovascular and oncol. diseases through clin. genetics and gene disruption in mice.  New selective PAD4 inhibitors binding a calcium-deficient form of the PAD4 enzyme have validated the crit. enzymic role of human and mouse PAD4 in both histone citrullination and neutrophil extracellular trap formation for, to our knowledge, the first time.  The therapeutic potential of PAD4 inhibitors can now be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLRDGrLt-nS7Vg90H21EOLACvtfcHk0liN55okGb9NSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVehsbo%253D&md5=24efcb7429535e5b2ec6e5cd48f0dd59</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1735%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DH.%2BD.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DCoote%26aufirst%3DJ.%2BE.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBax%26aufirst%3DB.%2BD.%26aulast%3DBicker%26aufirst%3DK.%2BL.%26aulast%3DBingham%26aufirst%3DR.%2BP.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%2BH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDavis%26aufirst%3DR.%2BP.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMaller%26aufirst%3DC.%26aulast%3DMartinod%26aufirst%3DK.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DPolyakova%26aufirst%3DO.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DR%25C3%25BCdiger%26aufirst%3DM.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSlade%26aufirst%3DD.%2BJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DThorpe%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DWagner%26aufirst%3DD.%2BD.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DInhibition%2520of%2520PAD4%2520activity%2520is%2520sufficient%2520to%2520disrupt%2520mouse%2520and%2520human%2520NET%2520formation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D189%26epage%3D191%26doi%3D10.1038%2Fnchembio.1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentile, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S.</span></span> <span> </span><span class="NLM_article-title">5-Aryl-4-carboxamide-1,3-oxazoles: Potent and selective GSK-3 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1989</span>– <span class="NLM_lpage">1994</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.bmcl.2012.01.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=22310227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVGnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1989-1994&author=G.+Gentileauthor=G.+Merloauthor=A.+Pozzanauthor=G.+Bernasconiauthor=B.+Baxauthor=P.+Bamboroughauthor=A.+Bridgesauthor=P.+Carterauthor=M.+Neuauthor=G.+Yaoauthor=C.+Broughauthor=G.+Cutlerauthor=A.+Coffinauthor=S.+Belyanskaya&title=5-Aryl-4-carboxamide-1%2C3-oxazoles%3A+Potent+and+selective+GSK-3+inhibitors&doi=10.1016%2Fj.bmcl.2012.01.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">5-Aryl-4-carboxamide-1,3-oxazoles: Potent and selective GSK-3 inhibitors</span></div><div class="casAuthors">Gentile, Gabriella; Merlo, Giancarlo; Pozzan, Alfonso; Bernasconi, Giovanni; Bax, Benjamin; Bamborough, Paul; Bridges, Angela; Carter, Paul; Neu, Margarete; Yao, Gang; Brough, Caroline; Cutler, Geoffrey; Coffin, Aaron; Belyanskaya, Svetlana</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1989-1994</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">5-Aryl-4-carboxamide-1,3-oxazoles are a novel, potent and selective series of GSK-3 inhibitors.  The optimization of the series to yield compds. with cell activity and brain permeability is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPQvQwnrcpa7Vg90H21EOLACvtfcHk0ljg9ky82opCxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVGnsbo%253D&md5=ec8383b7bceebdec334adef38b8a88b3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DGentile%26aufirst%3DG.%26aulast%3DMerlo%26aufirst%3DG.%26aulast%3DPozzan%26aufirst%3DA.%26aulast%3DBernasconi%26aufirst%3DG.%26aulast%3DBax%26aufirst%3DB.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBridges%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DNeu%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DBrough%26aufirst%3DC.%26aulast%3DCutler%26aufirst%3DG.%26aulast%3DCoffin%26aufirst%3DA.%26aulast%3DBelyanskaya%26aufirst%3DS.%26atitle%3D5-Aryl-4-carboxamide-1%252C3-oxazoles%253A%2520Potent%2520and%2520selective%2520GSK-3%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1989%26epage%3D1994%26doi%3D10.1016%2Fj.bmcl.2012.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veerman, J. J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruseker, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heijne, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bruggen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hekking, K. F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRiner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rijnsbergen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiss, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2,4–1 H-imidazole carboxamides as potent and selective TAK1 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">555</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=555&author=J.+J.+N.+Veermanauthor=Y.+B.+Brusekerauthor=E.+Damenauthor=E.+H.+Heijneauthor=W.+van+Bruggenauthor=K.+F.+W.+Hekkingauthor=R.+Winkelauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=J.+Liuauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+J.+McRinerauthor=M.+J.+Mulvihillauthor=P.+van+Rijnsbergenauthor=B.+Zechauthor=L.+M.+Renzettiauthor=L.+Babissauthor=G.+Muller&title=Discovery+of+2%2C4%E2%80%931+H-imidazole+carboxamides+as+potent+and+selective+TAK1+inhibitors&doi=10.1021%2Facsmedchemlett.0c00547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2,4-1H-Imidazole Carboxamides as Potent and Selective TAK1 Inhibitors</span></div><div class="casAuthors">Veerman, Johan J. N.; Bruseker, Yorik B.; Damen, Eddy; Heijne, Erik H.; van Bruggen, Wendy; Hekking, Koen F. W.; Winkel, Rob; Hupp, Christopher D.; Keefe, Anthony D.; Liu, Julie; Thomson, Heather A.; Zhang, Ying; Cuozzo, John W.; McRiner, Andrew J.; Mulvihill, Mark J.; van Rijnsbergen, Peter; Zech, Birgit; Renzetti, Louis M.; Babiss, Lee; Mueller, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">555-562</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein we report the discovery of 2,4-1H-imidazole carboxamides as novel, biochem. potent, and kinome selective inhibitors of transforming growth factor β-activated kinase 1 (TAK1).  The target was subjected to a DNA-encoded chem. library (DECL) screen.  After hit anal. a cluster of compds. was identified, which was based on a central pyrrole-2,4-1H-dicarboxamide scaffold, showing remarkable kinome selectivity.  A scaffold-hop to the corresponding imidazole resulted in increased biochem. potency.  Next, X-ray crystallog. revealed a distinct binding mode compared to other TAK1 inhibitors.  A benzylamide was found in a perpendicular orientation with respect to the core hinge-binding imidazole.  Addnl., an unusual amide flip was obsd. in the kinase hinge region.  Using structure-based drug design (SBDD), key substitutions at the pyrrolidine amide and the glycine resulted in a significant increase in biochem. potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsfdQbRX4C1LVg90H21EOLACvtfcHk0li1CHCCYeNg8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D&md5=26c25af441806342c2048ce49c6be192</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00547%26sid%3Dliteratum%253Aachs%26aulast%3DVeerman%26aufirst%3DJ.%2BJ.%2BN.%26aulast%3DBruseker%26aufirst%3DY.%2BB.%26aulast%3DDamen%26aufirst%3DE.%26aulast%3DHeijne%26aufirst%3DE.%2BH.%26aulast%3Dvan%2BBruggen%26aufirst%3DW.%26aulast%3DHekking%26aufirst%3DK.%2BF.%2BW.%26aulast%3DWinkel%26aufirst%3DR.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DMcRiner%26aufirst%3DA.%2BJ.%26aulast%3DMulvihill%26aufirst%3DM.%2BJ.%26aulast%3Dvan%2BRijnsbergen%26aufirst%3DP.%26aulast%3DZech%26aufirst%3DB.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DBabiss%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DG.%26atitle%3DDiscovery%2520of%25202%252C4%25E2%2580%25931%2520H-imidazole%2520carboxamides%2520as%2520potent%2520and%2520selective%2520TAK1%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D555%26doi%3D10.1021%2Facsmedchemlett.0c00547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foley, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burchett, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapinos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratnayake, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, M. C.</span></span> <span> </span><span class="NLM_article-title">Selecting approaches for hit identification and increasing options by building the efficient discovery of actionable chemical matter from DNA-encoded libraries</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1177/2472555220979589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1177%2F2472555220979589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=33412987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXit1els7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2021&pages=263-280&author=T.+L.+Foleyauthor=W.+Burchettauthor=Q.+Chenauthor=M.+E.+Flanaganauthor=B.+Kapinosauthor=X.+Liauthor=J.+I.+Montgomeryauthor=A.+S.+Ratnayakeauthor=H.+Zhuauthor=M.+C.+Peakman&title=Selecting+approaches+for+hit+identification+and+increasing+options+by+building+the+efficient+discovery+of+actionable+chemical+matter+from+DNA-encoded+libraries&doi=10.1177%2F2472555220979589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting Approaches for Hit Identification and Increasing Options by Building the Efficient Discovery of Actionable Chemical Matter from DNA-Encoded Libraries</span></div><div class="casAuthors">Foley, Timothy L.; Burchett, Woodrow; Chen, Qiuxia; Flanagan, Mark E.; Kapinos, Brendon; Li, Xianyang; Montgomery, Justin I.; Ratnayake, Anokha S.; Zhu, Hongyao; Peakman, Marie-Claire</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-280</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Over the past 20 years, the toolbox for discovering small-mol. therapeutic starting points has expanded considerably.  Pharmaceutical researchers can now choose from technologies that, in addn. to traditional high-throughput knowledge-based and diversity screening, now include the screening of fragment and fragment-like libraries, affinity selection mass spectrometry, and selection against DNA-encoded libraries (DELs).  Each of these techniques has its own unique combination of advantages and limitations that makes them more, or less, suitable for different target classes or discovery objectives, such as desired mechanism of action.  Layered on top of this are the constraints of the drug-hunters themselves, including budgets, timelines, and available platform capacity; each of these can play a part in dictating the hit identification strategy for a discovery program.  In this article, we discuss some of the factors that we use to govern our building of a hit identification roadmap for a program and describe the increasing role that DELs are playing in our discovery strategy.  Furthermore, we share our learning during our initial exploration of DEL and highlight the approaches we have evolved to maximize the value returned from DEL selections.  Topics addressed include the optimization of library design and prodn., reagent validation, data anal., and hit confirmation.  We describe how our thinking in these areas has led us to build a DEL platform that has begun to deliver tractable matter to our global discovery portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqqCnUXlyxTLVg90H21EOLACvtfcHk0li1CHCCYeNg8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXit1els7Y%253D&md5=c12b7655c625e163710f34cc7caf808b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1177%2F2472555220979589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555220979589%26sid%3Dliteratum%253Aachs%26aulast%3DFoley%26aufirst%3DT.%2BL.%26aulast%3DBurchett%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DKapinos%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DRatnayake%26aufirst%3DA.%2BS.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DPeakman%26aufirst%3DM.%2BC.%26atitle%3DSelecting%2520approaches%2520for%2520hit%2520identification%2520and%2520increasing%2520options%2520by%2520building%2520the%2520efficient%2520discovery%2520of%2520actionable%2520chemical%2520matter%2520from%2520DNA-encoded%2520libraries%26jtitle%3DSLAS%2520Discovery%26date%3D2021%26volume%3D26%26spage%3D263%26epage%3D280%26doi%3D10.1177%2F2472555220979589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">MacHutta, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besra, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barros-Aguirre, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhry, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosbenner, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingraham, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelievre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza-Losana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCloskey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Keefe, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Keeffe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelps, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafferty, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scavello, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiginga, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundersingh, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczuk, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toh, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evindar, G.</span></span> <span> </span><span class="NLM_article-title">Prioritizing multiple therapeutic targets in parallel using automated DNA-encoded library screening</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1038/ncomms16081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fncomms16081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=28232747" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-11&author=C.+A.+MacHuttaauthor=C.+S.+Kollmannauthor=K.+E.+Lindauthor=X.+Baiauthor=P.+F.+Chanauthor=J.+Huangauthor=L.+Ballellauthor=S.+Belyanskayaauthor=G.+S.+Besraauthor=D.+Barros-Aguirreauthor=R.+H.+Batesauthor=P.+A.+Centrellaauthor=S.+S.+Changauthor=J.+Chaiauthor=A.+E.+Choudhryauthor=A.+Coffinauthor=C.+P.+Davieauthor=H.+Dengauthor=J.+Dengauthor=Y.+Dingauthor=J.+W.+Dodsonauthor=D.+T.+Fosbennerauthor=E.+N.+Gaoauthor=T.+L.+Grahamauthor=T.+L.+Graybillauthor=K.+Ingrahamauthor=W.+P.+Johnsonauthor=B.+W.+Kingauthor=C.+R.+Kwiatkowskiauthor=J.+Lelievreauthor=Y.+Liauthor=X.+Liuauthor=Q.+Luauthor=R.+Lehrauthor=A.+Mendoza-Losanaauthor=J.+Martinauthor=L.+McCloskeyauthor=P.+McCormickauthor=H.+P.+O%27Keefeauthor=T.+O%27Keeffeauthor=C.+Paoauthor=C.+B.+Phelpsauthor=H.+Qiauthor=K.+Raffertyauthor=G.+S.+Scavelloauthor=M.+S.+Steigingaauthor=F.+S.+Sundersinghauthor=S.+M.+Sweitzerauthor=L.+M.+Szewczukauthor=A.+Taylorauthor=M.+F.+Tohauthor=J.+Wangauthor=M.+Wangauthor=D.+J.+Wilkinsauthor=B.+Xiaauthor=G.+Yaoauthor=J.+Zhangauthor=J.+Zhouauthor=C.+P.+Donahueauthor=J.+A.+Messerauthor=D.+Holmesauthor=C.+C.+Arico-Muendelauthor=A.+J.+Popeauthor=J.+W.+Grossauthor=G.+Evindar&title=Prioritizing+multiple+therapeutic+targets+in+parallel+using+automated+DNA-encoded+library+screening&doi=10.1038%2Fncomms16081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fncomms16081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms16081%26sid%3Dliteratum%253Aachs%26aulast%3DMacHutta%26aufirst%3DC.%2BA.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DP.%2BF.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DBallell%26aufirst%3DL.%26aulast%3DBelyanskaya%26aufirst%3DS.%26aulast%3DBesra%26aufirst%3DG.%2BS.%26aulast%3DBarros-Aguirre%26aufirst%3DD.%26aulast%3DBates%26aufirst%3DR.%2BH.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChang%26aufirst%3DS.%2BS.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DChoudhry%26aufirst%3DA.%2BE.%26aulast%3DCoffin%26aufirst%3DA.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DDodson%26aufirst%3DJ.%2BW.%26aulast%3DFosbenner%26aufirst%3DD.%2BT.%26aulast%3DGao%26aufirst%3DE.%2BN.%26aulast%3DGraham%26aufirst%3DT.%2BL.%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DIngraham%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DW.%2BP.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DKwiatkowski%26aufirst%3DC.%2BR.%26aulast%3DLelievre%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DQ.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DMendoza-Losana%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DJ.%26aulast%3DMcCloskey%26aufirst%3DL.%26aulast%3DMcCormick%26aufirst%3DP.%26aulast%3DO%2527Keefe%26aufirst%3DH.%2BP.%26aulast%3DO%2527Keeffe%26aufirst%3DT.%26aulast%3DPao%26aufirst%3DC.%26aulast%3DPhelps%26aufirst%3DC.%2BB.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DRafferty%26aufirst%3DK.%26aulast%3DScavello%26aufirst%3DG.%2BS.%26aulast%3DSteiginga%26aufirst%3DM.%2BS.%26aulast%3DSundersingh%26aufirst%3DF.%2BS.%26aulast%3DSweitzer%26aufirst%3DS.%2BM.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DToh%26aufirst%3DM.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWilkins%26aufirst%3DD.%2BJ.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DDonahue%26aufirst%3DC.%2BP.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DHolmes%26aufirst%3DD.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26aulast%3DGross%26aufirst%3DJ.%2BW.%26aulast%3DEvindar%26aufirst%3DG.%26atitle%3DPrioritizing%2520multiple%2520therapeutic%2520targets%2520in%2520parallel%2520using%2520automated%2520DNA-encoded%2520library%2520screening%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1%26epage%3D11%26doi%3D10.1038%2Fncomms16081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+Van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lhRndcjR2aIMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DVan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery II: The impact of chromatographic hydrophobicity measurements and aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+physical+in+drug+discovery+II%3A+The+impact+of+chromatographic+hydrophobicity+measurements+and+aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0liQy92HKaTicA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%2520II%253A%2520The%2520impact%2520of%2520chromatographic%2520hydrophobicity%2520measurements%2520and%2520aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Structural basis for acetylated histone H4 recognition by the human BRD2 bromodomain</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">7610</span>– <span class="NLM_lpage">7618</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.062422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1074%2Fjbc.M109.062422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=20048151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=7610-7618&author=T.+Umeharaauthor=Y.+Nakamuraauthor=M.+K.+Jangauthor=K.+Nakanoauthor=A.+Tanakaauthor=K.+Ozatoauthor=B.+Padmanabhanauthor=S.+Yokoyama&title=Structural+basis+for+acetylated+histone+H4+recognition+by+the+human+BRD2+bromodomain&doi=10.1074%2Fjbc.M109.062422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Acetylated Histone H4 Recognition by the Human BRD2 Bromodomain</span></div><div class="casAuthors">Umehara, Takashi; Nakamura, Yoshihiro; Jang, Moon-Kyoo; Nakano, Kazumi; Tanaka, Akiko; Ozato, Keiko; Padmanabhan, Balasundaram; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7610-7618</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Recognition of acetylated chromatin by the bromodomains and extra-terminal domain (BET) family proteins is a hallmark for transcriptional activation and anchoring viral genomes to mitotic chromosomes of the host.  One of the BET family proteins, BRD2, interacts with acetylated chromatin during mitosis and leads to transcriptional activation in culture cells.  Here, we report the crystal structures of the N-terminal bromodomain of human BRD2 (BRD2-BD1; residues 74-194) in complex with each of three different Lys-12-acetylated H4 peptides.  The BRD2-BD1 recognizes the H4 tail acetylated at Lys-12 (H4K12ac), whereas the side chain of hypoacetylated Lys-8 of H4 binds at the cavity of the dimer interface of BRD2-BD1.  From binding studies, we identified the BRD2-BD1 residues that are responsible for recognition of the Lys-12-acetylated H4 tail.  In addn., mutation to Lys-8 in the Lys-12-acetylated H4 tail decreased the binding to BRD2-BD1, implicating the crit. role of Lys-8 in the Lys-12-acetylated H4 tail for the recognition by BRD2-BD1.  Our findings provide a structural basis for deciphering the histone code by the BET bromodomain through the binding with a long segment of the histone H4 tail, which presumably prevents erasure of the histone code during the cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMfSa5EPdd57Vg90H21EOLACvtfcHk0liQy92HKaTicA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D&md5=88df449d778d37724b9295ad36e0e188</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062422%26sid%3Dliteratum%253Aachs%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DNakano%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520for%2520acetylated%2520histone%2520H4%2520recognition%2520by%2520the%2520human%2520BRD2%2520bromodomain%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D7610%26epage%3D7618%26doi%3D10.1074%2Fjbc.M109.062422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Placko, S.</span></span> <span> </span><span class="NLM_article-title">Chemiluminescent Nitrogen Detection (CLND) to measure kinetic aqueous solubility</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch130145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1002%2F9780470559277.ch130145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24391097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=269-280&author=A.+Kestranekauthor=A.+Chervenakauthor=J.+Longenbergerauthor=S.+Placko&title=Chemiluminescent+Nitrogen+Detection+%28CLND%29+to+measure+kinetic+aqueous+solubility&doi=10.1002%2F9780470559277.ch130145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Chemiluminescent nitrogen detection (CLND) to measure kinetic aqueous solubility</span></div><div class="casAuthors">Kestranek Aimee; Chervenak Andrew; Longenberger Justin; Placko Steven</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Solubility is the dose-limiting property for in vitro studies, and therefore is a critical physicochemical property to measure in drug discovery.  Solubility data can be used to guide lead optimization, troubleshoot erratic bioassay results, and identify potential downstream liabilities such as insufficient solubility for bioassays or oral bioavailability.  Typically, early in vitro studies are performed using library compounds prepared as dimethylsulfoxide (DMSO) stock solutions, resulting in in vitro test solutions containing DMSO at low concentration (<5% v/v).  Since DMSO can affect the apparent solubility, it is desirable to obtain solubility data under conditions mimicking the in vitro study.  Kinetic solubility (from DMSO stock solutions) is often preferred over thermodynamic solubility (from dry powder) in early drug discovery.  The protocols in this article describe a general procedure for assessing kinetic aqueous solubility of early drug discovery compounds using a miniaturized shake flask method with chemiluminescent nitrogen detection (CLND).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQa5zXGR-PJU_SAs_PE1QvifW6udTcc2eb0xWABMHHKU7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D&md5=d63d6d57c4ff0934379d3c37f82eb1a0</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch130145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch130145%26sid%3Dliteratum%253Aachs%26aulast%3DKestranek%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%26aulast%3DLongenberger%26aufirst%3DJ.%26aulast%3DPlacko%26aufirst%3DS.%26atitle%3DChemiluminescent%2520Nitrogen%2520Detection%2520%2528CLND%2529%2520to%2520measure%2520kinetic%2520aqueous%2520solubility%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3D269%26epage%3D280%26doi%3D10.1002%2F9780470559277.ch130145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boughtflower, B. J.</span></span> <span> </span><span class="NLM_article-title">Use of calculated physicochemical properties to enhance quantitative response when using charged aerosol detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b04060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b04060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=1772-1777&author=M.+W.+Robinsonauthor=A.+P.+Hillauthor=S.+A.+Readshawauthor=J.+C.+Hollertonauthor=R.+J.+Uptonauthor=S.+M.+Lynnauthor=S.+C.+Besleyauthor=B.+J.+Boughtflower&title=Use+of+calculated+physicochemical+properties+to+enhance+quantitative+response+when+using+charged+aerosol+detection&doi=10.1021%2Facs.analchem.6b04060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span></div><div class="casAuthors">Robinson, Max W.; Hill, Alan P.; Readshaw, Simon A.; Hollerton, John C.; Upton, Richard J.; Lynn, Sean M.; Besley, Steve C.; Boughtflower, Bob J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1772-1777</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Universal quant. detection without the need for analyte ref. stds. would offer substantial benefits in many areas of anal. science.  The quant. capability of high performance liq. chromatog. (HPLC) with charged aerosol detection (CAD) was investigated for 50 compds. with a wide range of phys. and chem. properties.  It is widely believed that CAD is a mass detector.  Quantification of the 50 compds. using a generic calibrant and mass calibration achieved an av. error of 11.4% relative to 1H NMR.  Correction factors are proposed that est. the relative surface area of particles in the detector, taking into account the effects of the d. and charge of analytes.  Performing these corrections and quantifying with surface area calibration, rather than mass, shows considerably improved linearity and uniformity of detection, reducing the av. error relative to 1H NMR to 7.1%.  The accuracy of CAD quantification was most significantly improved for highly dense compds., with traditional mass calibration showing an av. error of 34.7% and the newly proposed surface area calibration showing an av. error of 5.8%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWCD0M2yQRLVg90H21EOLACvtfcHk0lg03kJ4oI0U4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK&md5=46c11b7527157b76f1fbc8eaf5cef75a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b04060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b04060%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DReadshaw%26aufirst%3DS.%2BA.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DLynn%26aufirst%3DS.%2BM.%26aulast%3DBesley%26aufirst%3DS.%2BC.%26aulast%3DBoughtflower%26aufirst%3DB.%2BJ.%26atitle%3DUse%2520of%2520calculated%2520physicochemical%2520properties%2520to%2520enhance%2520quantitative%2520response%2520when%2520using%2520charged%2520aerosol%2520detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D1772%26epage%3D1777%26doi%3D10.1021%2Facs.analchem.6b04060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D.</span></span> <span> </span><span class="NLM_article-title">Chromatographic hydrophobicity index by fast-gradient RP-HPLC: A high-throughput alternative to log P/log D</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">2022</span>– <span class="NLM_lpage">2029</span>, <span class="refDoi"> DOI: 10.1021/ac961242d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac961242d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1997&pages=2022-2029&author=K.+Valkoauthor=C.+Bevanauthor=D.+Reynolds&title=Chromatographic+hydrophobicity+index+by+fast-gradient+RP-HPLC%3A+A+high-throughput+alternative+to+log+P%2Flog+D&doi=10.1021%2Fac961242d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatographic Hydrophobicity Index by Fast-Gradient RP-HPLC: A High-Throughput Alternative to Log P/log D</span></div><div class="casAuthors">Valko, Klara; Bevan, Chris; Reynolds, Derek</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2022-2029</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new chromatog. hydrophobicity index (CHI) is described which can be used as part of a protocol for high-throughput (50-100 compds./day) physicochem. property profiling for rational drug design.  The index is derived from retention times (tR) obsd. in a fast gradient reversed-phase HPLC method.  The isocratic retention factors (log k') were measured for a series of 76 structurally unrelated compds. by using various concns. of acetonitrile in the mobile phase.  By plotting the log k' as a function of the acetonitrile concn., the slope (S) and the intercept (log k'w) values were calcd.  The previously validated index of hydrophobicity φ0 was calcd. as -log k'w/S.  A good linear correlation was obtained between the gradient retention time values, tR and the isocratically detd. φ0 values for the 76 compds.  The consts. of this linear correlation can be used to calc. CHI.  For most compds., CHI is between 0 and 100 and in this range it approximates to the percentage (by vol.) of acetonitrile required to achieve an equal distribution of compd. between the mobile and the stationary phases.  CHI values can be measured using acidic, neutral, or slightly basic eluents.  Values corresponding to the neutral form of mols. could be measured for 52 of the compds. and showed good correlation (r = 0.851) to the calcd. octanol/water partition coeff. (c log P) values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozvQI1B1cnpbVg90H21EOLACvtfcHk0lg03kJ4oI0U4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D&md5=fcbb641105c12dc490d4a20db8fc5032</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fac961242d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac961242d%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DReynolds%26aufirst%3DD.%26atitle%3DChromatographic%2520hydrophobicity%2520index%2520by%2520fast-gradient%2520RP-HPLC%253A%2520A%2520high-throughput%2520alternative%2520to%2520log%2520P%252Flog%2520D%26jtitle%3DAnal.%2520Chem.%26date%3D1997%26volume%3D69%26spage%3D2022%26epage%3D2029%26doi%3D10.1021%2Fac961242d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of naphthyridones into potent ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-based+optimization+of+naphthyridones+into+potent+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0ljrDEWeN4E_Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-based%2520optimization%2520of%2520naphthyridones%2520into%2520potent%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of ABBV-744 (<b>1</b>) and of BD2-selective inhibitors reported by GSK (<b>2</b>, <b>3</b>, and <b>4</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Representation of BRD4 (N to C terminus) showing the two bromodomains and the extra terminal (ET) domain and the three truncated constructs used in affinity selection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. (a) Synthesis of Glycine-Based DEL Library. (b) Result of the BRD4 Screen of Glycine-Based DEL Library. Signal Strength Versus the Affinity Selection Conditions. (c) Structures of Hits <b>5</b> and <b>6</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with van der Waals (VDW) surface and conserved waters shown in red, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>). The methoxyphenyl ring acts as the KAc-mimetic, forming a hydrogen bond with Asn433 (red dashed line). One of the phenyl rings occupies the WPF shelf and sandwiches His437 with the other phenyl ring. The glycine fragment of the molecule enters the ZA channel region. (b) Crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1">6ZB1</a>) overlain with <b>3</b> in BRD2 BD2 (blue ligand, protein not shown, BRD4 numbering used). The hydrogen-bonding interactions of <b>3</b> with the conserved asparagine are shown (dashed yellow line). (c) Crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface) overlain with the crystal structure of <b>1</b> in BRD2 BD2 (yellow ligand, protein not shown, BRD4 numbering used, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E6J">6E6J</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure of compound <b>8</b> with the key areas for optimization highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Overlay of crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface, dashed red hydrogen bonds, red waters, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) with <b>9</b> in BRD2 BD2 (magenta ligand, protein not shown, dashed yellow hydrogen bond, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEP">7OEP</a>), showing a close overlap of the KAc-mimetic groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Overlay of crystal structure of <b>8</b> in BRD4 BD2 (green ligand, gray protein with VDW surface, red waters, dashed red line for hydrogen bonds, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) and <b>10</b> in BRD2 BD2 (blue ligand, protein not shown, red and pink waters, dashed yellow lines for hydrogen bonds, BRD4 numbering used, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OER">7OER</a>). The hydrogen bond from each KAc-mimetic to Asn433 is shown, along with the hydrogen bonds with the returning water molecule in the crystal structure for <b>10</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overlay of crystal structure of <b>8</b> in BRD4 BD2 (purple protein, purple VDW surface, and yellow ligand, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO">7OEO</a>) with an aligned crystal structure of <b>3</b> in BRD4 BD1 (ligand not shown, green protein, green VDW surface, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3">6ZB3</a>). Structures superposed with an in-house script using C-α atoms of key active-site residues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0008.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cocrystal structure of <b>42</b> in BRD2 BD2 (green ligand, gray protein with VDW surface, red waters, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OES">7OES</a>) showing the methyl group occupying the WPF shelf.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0009.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cocrystal structure of <b>60</b> in BRD2 BD2 (green ligand, gray protein with VDW surface, red waters, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OET">7OET</a>) of the protein at the binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0010.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Potency of compound <b>60</b> in the DiscoverX bromodomain panel (measured as % inhibition at a concentration of 10 uM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/medium/jm1c00412_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0012.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route towards pan-BD2 Selective Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00412/20210712/images/large/jm1c00412_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00412&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p><span class="fn-label"><sup>a</sup></span>Reagents and conditions: MeOH or CF<sub>3</sub>CH<sub>2</sub>OH, 70–100 °C, 30–90 min, microwave irradiation, 11–91%.</p><p><span class="fn-label"><sup>b</sup></span>Chiral separation.</p><p><span class="fn-label"><sup>c</sup></span>HATU, DIPEA, DMF, room temperature, 81%.</p><p><span class="fn-label"><sup>d</sup></span>EtI, <i>t</i>-BuOK, DMF, 100 °C, microwave irradiation, 43% or R<sub>1</sub>R<sub>2</sub>N(CH<sub>2</sub>)<sub>2</sub>Cl, NaH, DMF, room temperature, 18%.</p><p><span class="fn-label"><sup>e</sup></span>NaHB(OAc)<sub>3</sub>, AcOH, CH<sub>2</sub>Cl<sub>2</sub>, room temperature, 7–68%.</p><p><span class="fn-label"><sup>f</sup></span>SOCl<sub>2</sub>, 60 °C, 98% then CH<sub>3</sub>CN, DIPEA, room temperature, 34%.</p><p><span class="fn-label"><sup>g</sup></span>(Isocyanomethyl)trimethylsilane was used instead of isocyanomethane, followed by a TBAF-mediated deprotection: TBAF (1 M in THF), 50 °C, 1 h, 4%.</p><p><span class="fn-label"><sup>h</sup></span>(1-Ethoxycyclopropoxy)trimethylsilane was used as a cyclopropanone substitute.</p><p class="last"><span class="fn-label"><sup>i</sup></span>(2-Isocyanopropan-2-yl)benzene was used followed by a TFA-mediated deprotection: TFA, 50 °C, 1.5 h, 62%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i123">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42143" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42143" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: Epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+Epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lj_Eimkejh17g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520Epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span> <span> </span><span class="NLM_article-title">Place your BETs: The therapeutic potential of bromodomains</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.12.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.tips.2011.12.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=22277300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=146-153&author=R.+K.+Prinjhaauthor=J.+Witheringtonauthor=K.+Lee&title=Place+your+BETs%3A+The+therapeutic+potential+of+bromodomains&doi=10.1016%2Fj.tips.2011.12.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Place your BETs: the therapeutic potential of bromodomains</span></div><div class="casAuthors">Prinjha, R. K.; Witherington, J.; Lee, K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">146-153</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Therapeutic targeting of the processes that regulate histone modification is a growing area of scientific exploration.  Although most interest has concd. on the various families of enzymes that contribute to these processes, this review focuses on emerging data demonstrating the chem. tractability and therapeutic potential of a hitherto underexplored family of proteins, namely the bromodomain (BRD) family of reader proteins.  These proteins perform a crucial role in translating histone modifications with powerful transcriptional consequences.  We review current knowledge of the biol. of this emergent target class and highlight recent breakthroughs that now make the BRD family of reader proteins attractive for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_N9p7ru787rVg90H21EOLACvtfcHk0lgUUXAGphEHFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlOlt74%253D&md5=ea1e9e5740b1131239aa0407748d549b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.12.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.12.002%26sid%3Dliteratum%253Aachs%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26atitle%3DPlace%2520your%2520BETs%253A%2520The%2520therapeutic%2520potential%2520of%2520bromodomains%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2012%26volume%3D33%26spage%3D146%26epage%3D153%26doi%3D10.1016%2Fj.tips.2011.12.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgUUXAGphEHFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">697</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697-705&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+C.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0lgUUXAGphEHFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%2BC.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D697%26epage%3D705%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappaport, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chicas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogan, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nature10334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnature10334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=21814200" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=524-528&author=J.+Zuberauthor=J.+Shiauthor=E.+Wangauthor=A.+R.+Rappaportauthor=H.+Herrmannauthor=E.+A.+Sisonauthor=D.+Magoonauthor=J.+Qiauthor=K.+Blattauthor=M.+Wunderlichauthor=M.+J.+Taylorauthor=C.+Johnsauthor=A.+Chicasauthor=J.+C.+Mulloyauthor=S.+C.+Koganauthor=P.+Brownauthor=P.+Valentauthor=J.+E.+Bradnerauthor=S.+W.+Loweauthor=C.+R.+Vakoc&title=RNAi+screen+identifies+Brd4+as+a+therapeutic+target+in+acute+myeloid+leukaemia&doi=10.1038%2Fnature10334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia</span></div><div class="casAuthors">Zuber, Johannes; Shi, Junwei; Wang, Eric; Rappaport, Amy R.; Herrmann, Harald; Sison, Edward A.; Magoon, Daniel; Qi, Jun; Blatt, Katharina; Wunderlich, Mark; Taylor, Meredith J.; Johns, Christopher; Chicas, Agustin; Mulloy, James C.; Kogan, Scott C.; Brown, Patrick; Valent, Peter; Bradner, James E.; Lowe, Scott W.; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic pathways can regulate gene expression by controlling and interpreting chromatin modifications.  Cancer cells are characterized by altered epigenetic landscapes, and commonly exploit the chromatin regulatory machinery to enforce oncogenic gene expression programs.  Although chromatin alterations are, in principle, reversible and often amenable to drug intervention, the promise of targeting such pathways therapeutically has been limited by an incomplete understanding of cancer-specific dependencies on epigenetic regulators.  Here the authors describe a non-biased approach to probe epigenetic vulnerabilities in acute myeloid leukemia (AML), an aggressive hematopoietic malignancy that is often assocd. with aberrant chromatin states.  By screening a custom library of small hairpin RNAs (shRNAs) targeting known chromatin regulators in a genetically defined AML mouse model, the authors identify the protein bromodomain-contg. 4 (Brd4) as being critically required for disease maintenance.  Suppression of Brd4 using shRNAs or the small-mol. inhibitor JQ1 led to robust antileukemic effects in vitro and in vivo, accompanied by terminal myeloid differentiation and elimination of leukemia stem cells.  Similar sensitivities were obsd. in a variety of human AML cell lines and primary patient samples, revealing that JQ1 has broad activity in diverse AML subtypes.  The effects of Brd4 suppression are, at least in part, due to its role in sustaining Myc expression to promote aberrant self-renewal, which implicates JQ1 as a pharmacol. means to suppress MYC in cancer.  The authors' results establish small-mol. inhibition of Brd4 as a promising therapeutic strategy in AML and, potentially, other cancers, and highlight the utility of RNA interference (RNAi) screening for revealing epigenetic vulnerabilities that can be exploited for direct pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVyWFIqFkUVrVg90H21EOLACvtfcHk0lg7wzrr7LLcHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFers7%252FP&md5=618793a766f037bff5909524743fedb7</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature10334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10334%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DHerrmann%26aufirst%3DH.%26aulast%3DSison%26aufirst%3DE.%2BA.%26aulast%3DMagoon%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBlatt%26aufirst%3DK.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DM.%2BJ.%26aulast%3DJohns%26aufirst%3DC.%26aulast%3DChicas%26aufirst%3DA.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DBrown%26aufirst%3DP.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DRNAi%2520screen%2520identifies%2520Brd4%2520as%2520a%2520therapeutic%2520target%2520in%2520acute%2520myeloid%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D524%26epage%3D528%26doi%3D10.1038%2Fnature10334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierce, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET bromodomain targets genetically diverse glioblastoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1748</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-12-3066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F1078-0432.CCR-12-3066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23403638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1748-1759&author=Z.+Chengauthor=Y.+Gongauthor=Y.+Maauthor=K.+Luauthor=X.+Luauthor=L.+A.+Pierceauthor=R.+C.+Thompsonauthor=S.+Mullerauthor=S.+Knappauthor=J.+Wang&title=Inhibition+of+BET+bromodomain+targets+genetically+diverse+glioblastoma&doi=10.1158%2F1078-0432.CCR-12-3066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET Bromodomain Targets Genetically Diverse Glioblastoma</span></div><div class="casAuthors">Cheng, Zhixiang; Gong, Yuanying; Ma, Yufang; Lu, Kaihua; Lu, Xiang; Pierce, Larry A.; Thompson, Reid C.; Muller, Susanne; Knapp, Stefan; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1748-1759</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Glioblastoma is refractory to conventional therapies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic readers that selectively bind to acetylated lysine residues on histone tails.  These proteins recently emerged as important therapeutic targets in NUT midline carcinoma and several types of hematopoietic cancers.  In this study, the therapeutic potential of a novel BET bromodomain inhibitor, JQ1, was assessed in a panel of genetically heterogeneous glioblastoma samples.  Exptl. Design: The antineoplastic effects of JQ1 were shown using ex vivo cultures derived from primary glioblastoma xenograft lines and surgical specimens of different genetic background.  The in vivo efficacy was assessed in orthotopic glioblastoma tumors.  Results: We showed that JQ1 induced marked G1 cell-cycle arrest and apoptosis, which was phenocopied by knockdown of individual BET family members.  JQ1 treatment resulted in significant changes in expression of genes that play important roles in glioblastoma such as c-Myc, p21CIP1/WAF1, hTERT, Bcl-2, and Bcl-xL.  Unlike the observations in some hematopoietic cancer cell lines, exogenous c-Myc did not significantly protect glioblastoma cells against JQ1.  In contrast, ectopically expressed Bcl-xL partially rescued cells from JQ1-induced apoptosis, and knockdown of p21CIP1/WAF1 attenuated JQ1-induced cell-cycle arrest.  Cells genetically engineered for Akt hyperactivation or p53/Rb inactivation did not compromise JQ1 efficacy, suggesting that these frequently mutated signaling pathways may not confer resistance to JQ1.  Furthermore, JQ1 significantly repressed growth of orthotopic glioblastoma tumors.  Conclusion: Our results suggest potentially broad therapeutic use of BET bromodomain inhibitors for treating genetically diverse glioblastoma tumors.  Clin Cancer Res; 19(7); 1748-59. ©2013 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7OsFKqlwyDbVg90H21EOLACvtfcHk0lg7wzrr7LLcHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVCmsLo%253D&md5=1722189e5d950da233b785558f2c9f58</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-3066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-3066%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DPierce%26aufirst%3DL.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BC.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520BET%2520bromodomain%2520targets%2520genetically%2520diverse%2520glioblastoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2013%26volume%3D19%26spage%3D1748%26epage%3D1759%26doi%3D10.1158%2F1078-0432.CCR-12-3066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Da
Motta, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledaki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purshouse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bastiani, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baban, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morotti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steers, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigfield, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntyre, A.</span></span> <span> </span><span class="NLM_article-title">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/onc.2016.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fonc.2016.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27292261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=122-132&author=L.+L.+Da%0AMottaauthor=I.+Ledakiauthor=K.+Purshouseauthor=S.+Haiderauthor=M.+A.+De+Bastianiauthor=D.+Babanauthor=M.+Morottiauthor=G.+Steersauthor=S.+Wigfieldauthor=E.+Bridgesauthor=J.+L.+Liauthor=S.+Knappauthor=D.+Ebnerauthor=F.+Klamtauthor=A.+L.+Harrisauthor=A.+McIntyre&title=The+BET+inhibitor+JQ1+selectively+impairs+tumour+response+to+hypoxia+and+downregulates+CA9+and+angiogenesis+in+triple+negative+breast+cancer&doi=10.1038%2Fonc.2016.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The BET inhibitor JQ1 selectively impairs tumour response to hypoxia and downregulates CA9 and angiogenesis in triple negative breast cancer</span></div><div class="casAuthors">da Motta, L. L.; Ledaki, I.; Purshouse, K.; Haider, S.; De Bastiani, M. A.; Baban, D.; Morotti, M.; Steers, G.; Wigfield, S.; Bridges, E.; Li, J.-L.; Knapp, S.; Ebner, D.; Klamt, F.; Harris, A. L.; McIntyre, A.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-132</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The availability of bromodomain and extra-terminal inhibitors (BETi) has enabled translational epigenetic studies in cancer.  BET proteins regulate transcription by selectively recognizing acetylated lysine residues on chromatin.  BETi compete with this process leading to both downregulation and upregulation of gene expression.  Hypoxia enables progression of triple neg. breast cancer (TNBC), the most aggressive form of breast cancer, partly by driving metabolic adaptation, angiogenesis and metastasis through upregulation of hypoxia-regulated genes (for example, carbonic anhydrase 9 (CA9) and vascular endothelial growth factor A (VEGF-A).  Responses to hypoxia can be mediated epigenetically, thus we investigated whether BETi JQ1 could impair the TNBC response induced by hypoxia and exert anti-tumor effects.  JQ1 significantly modulated 44% of hypoxia-induced genes, of which two-thirds were downregulated including CA9 and VEGF-A.  JQ1 prevented HIF binding to the hypoxia response element in CA9 promoter, but did not alter HIF expression or activity, suggesting some HIF targets are BET-dependent.  JQ1 reduced TNBC growth in vitro and in vivo and inhibited xenograft vascularization.  These findings identify that BETi dually targets angiogenesis and the hypoxic response, an effective combination at reducing tumor growth in preclin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79-F0o2tp6bVg90H21EOLACvtfcHk0lgGhLOpogWjaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1ejtr4%253D&md5=9f1592548db8e6306703bdab7e1e1c90</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.184%26sid%3Dliteratum%253Aachs%26aulast%3DDa%2BMotta%26aufirst%3DL.%2BL.%26aulast%3DLedaki%26aufirst%3DI.%26aulast%3DPurshouse%26aufirst%3DK.%26aulast%3DHaider%26aufirst%3DS.%26aulast%3DDe%2BBastiani%26aufirst%3DM.%2BA.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DMorotti%26aufirst%3DM.%26aulast%3DSteers%26aufirst%3DG.%26aulast%3DWigfield%26aufirst%3DS.%26aulast%3DBridges%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DJ.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKlamt%26aufirst%3DF.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DMcIntyre%26aufirst%3DA.%26atitle%3DThe%2520BET%2520inhibitor%2520JQ1%2520selectively%2520impairs%2520tumour%2520response%2520to%2520hypoxia%2520and%2520downregulates%2520CA9%2520and%2520angiogenesis%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D122%26epage%3D132%26doi%3D10.1038%2Fonc.2016.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rhyasen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petteruti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boiko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castriotta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahdesmaki, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dry, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyne, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span> <span> </span><span class="NLM_article-title">AZD5153: A novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2563</span>– <span class="NLM_lpage">2574</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0141</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F1535-7163.MCT-16-0141" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27573426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2563-2574&author=G.+W.+Rhyasenauthor=M.+M.+Hattersleyauthor=Y.+Yaoauthor=A.+Dulakauthor=W.+Wangauthor=P.+Petterutiauthor=I.+L.+Daleauthor=S.+Boikoauthor=T.+Cheungauthor=J.+Zhangauthor=S.+Wenauthor=L.+Castriottaauthor=D.+Lawsonauthor=M.+Collinsauthor=L.+Baoauthor=M.+J.+Ahdesmakiauthor=G.+Walkerauthor=G.+O%E2%80%99Connorauthor=T.+C.+Yehauthor=A.+A.+Rabowauthor=J.+R.+Dryauthor=C.+Reimerauthor=P.+Lyneauthor=G.+B.+Millsauthor=S.+E.+Fawellauthor=M.+J.+Waringauthor=M.+Zindaauthor=E.+Clarkauthor=H.+Chen&title=AZD5153%3A+A+novel+bivalent+BET+bromodomain+inhibitor+highly+active+against+hematologic+malignancies&doi=10.1158%2F1535-7163.MCT-16-0141"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">AZD5153: a novel bivalent BET bromodomain inhibitor highly active against hematologic malignancies</span></div><div class="casAuthors">Rhyasen, Garrett W.; Hattersley, Maureen M.; Yao, Yi; Dulak, Austin; Wang, Wenxian; Petteruti, Philip; Dale, Ian L.; Boiko, Scott; Cheung, Tony; Zhang, Jingwen; Wen, Shenghua; Castriotta, Lillian; Lawson, Deborah; Collins, Michael; Bao, Larry; Ahdesmaki, Miika J.; Walker, Graeme; O'Connor, Greg; Yeh, Tammie C.; Rabow, Alfred A.; Dry, Jonathan R.; Reimer, Corinne; Lyne, Paul; Mills, Gordon B.; Fawell, Stephen E.; Waring, Michael J.; Zinda, Michael; Clark, Edwin; Chen, Huawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2563-2574</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD4 regulates gene expression via recruitment of transcriptional regulatory complexes to acetylated chromatin.  Pharmacol. targeting of BRD4 bromodomains by small mol. inhibitors has proven to be an effective means to disrupt aberrant transcriptional programs crit. for tumor growth and/or survival.  Herein, we report AZD5153, a potent, selective, and orally available BET/BRD4 bromodomain inhibitor possessing a bivalent binding mode.  Unlike previously described monovalent inhibitors, AZD5153 ligates two bromodomains in BRD4 simultaneously.  The enhanced avidity afforded through bivalent binding translates into increased cellular and antitumor activity in preclin. hematol. tumor models.  In vivo administration of AZD5153 led to tumor stasis or regression in multiple xenograft models of acute myeloid leukemia, multiple myeloma, and diffuse large B-cell lymphoma.  The relationship between AZD5153 exposure and efficacy suggests that prolonged BRD4 target coverage is a primary efficacy driver.  AZD5153 treatment markedly affects transcriptional programs of MYC, E2F, and mTOR.  Of note, mTOR pathway modulation is assocd. with cell line sensitivity to AZD5153.  Transcriptional modulation of MYC and HEXIM1 was confirmed in AZD5153-treated human whole blood, thus supporting their use as clin. pharmacodynamic biomarkers.  This study establishes AZD5153 as a highly potent, orally available BET/BRD4 inhibitor and provides a rationale for clin. development in hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAiKYOFhro1bVg90H21EOLACvtfcHk0lgGhLOpogWjaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVOqsrk%253D&md5=8d0ac899d081705aaa12d49fd90165a3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0141%26sid%3Dliteratum%253Aachs%26aulast%3DRhyasen%26aufirst%3DG.%2BW.%26aulast%3DHattersley%26aufirst%3DM.%2BM.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDulak%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPetteruti%26aufirst%3DP.%26aulast%3DDale%26aufirst%3DI.%2BL.%26aulast%3DBoiko%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCastriotta%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DM.%26aulast%3DBao%26aufirst%3DL.%26aulast%3DAhdesmaki%26aufirst%3DM.%2BJ.%26aulast%3DWalker%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DG.%26aulast%3DYeh%26aufirst%3DT.%2BC.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DDry%26aufirst%3DJ.%2BR.%26aulast%3DReimer%26aufirst%3DC.%26aulast%3DLyne%26aufirst%3DP.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DZinda%26aufirst%3DM.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DH.%26atitle%3DAZD5153%253A%2520A%2520novel%2520bivalent%2520BET%2520bromodomain%2520inhibitor%2520highly%2520active%2520against%2520hematologic%2520malignancies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2563%26epage%3D2574%26doi%3D10.1158%2F1535-7163.MCT-16-0141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET bromodomain inhibitor OTX015 affects pathogenetic pathways in preclinical B-cell tumor models and synergizes with targeted drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1628</span>– <span class="NLM_lpage">1638</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628-1638&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+bromodomain+inhibitor+OTX015+affects+pathogenetic+pathways+in+preclinical+B-cell+tumor+models+and+synergizes+with+targeted+drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0lh2LHpJ12usLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520bromodomain%2520inhibitor%2520OTX015%2520affects%2520pathogenetic%2520pathways%2520in%2520preclinical%2520B-cell%2520tumor%2520models%2520and%2520synergizes%2520with%2520targeted%2520drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1628%26epage%3D1638%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Costa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanasopoulou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniere, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stankovic, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">PFI-1, a highly selective protein interaction inhibitor, targeting BET bromodomains</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">3336</span>– <span class="NLM_lpage">3346</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-3292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F0008-5472.CAN-12-3292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23576556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=3336-3346&author=S.+Picaudauthor=D.+Da+Costaauthor=A.+Thanasopoulouauthor=P.+Filippakopoulosauthor=P.+V.+Fishauthor=M.+Philpottauthor=O.+Fedorovauthor=P.+Brennanauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=J.+E.+Bradnerauthor=P.+Taniereauthor=B.+O%E2%80%99Sullivanauthor=S.+M%C3%BCllerauthor=J.+Schwallerauthor=T.+Stankovicauthor=S.+Knapp&title=PFI-1%2C+a+highly+selective+protein+interaction+inhibitor%2C+targeting+BET+bromodomains&doi=10.1158%2F0008-5472.CAN-12-3292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">PFI-1, a Highly Selective Protein Interaction Inhibitor, Targeting BET Bromodomains</span></div><div class="casAuthors">Picaud, Sarah; Da Costa, David; Thanasopoulou, Angeliki; Filippakopoulos, Panagis; Fish, Paul V.; Philpott, Martin; Fedorov, Oleg; Brennan, Paul; Bunnage, Mark E.; Owen, Dafydd R.; Bradner, James E.; Taniere, Philippe; O'Sullivan, Brendan; Mueller, Susanne; Schwaller, Juerg; Stankovic, Tatjana; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3336-3346</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromo and extra terminal (BET) proteins (BRD2, BRD3, BRD4, and BRDT) are transcriptional regulators required for efficient expression of several growth promoting and antiapoptotic genes as well as for cell-cycle progression.  BET proteins are recruited on transcriptionally active chromatin via their two N-terminal bromo-domains (BRD), a protein interaction module that specifically recognizes acetylated lysine residues in histones H3 and H4.  Inhibition of the BET-histone interaction results in transcriptional downregulation of a no. of oncogenes, providing a novel pharmacol. strategy for the treatment of cancer.  Here, we present a potent and highly selective dihydroquinazoline-2-one inhibitor, PFFI-1, which efficiently blocks the interaction of BET BRDs with acetylated histone tails.  Cocrystal structures showed that PFI-1 acts as an acetyl-lysine (Kac) mimetic inhibitor efficiently occupying the Kac binding site in BRD4 and BRD2.  PFI-1 has antiproliferative effects on leukemic cell lines and efficiently abrogates their clonogenic growth.  Exposure of sensitive cell lines with PFI-I results in G1 cell-cycle arrest, downregulation of MYC expression, as well as induction of apoptosis and induces differentiation of primary leukemic blasts.  Intriguingly, cells exposed to PFI-1 showed significant downregulation of Aurora B kinase, thus attenuating phosphorylation of the Aurora substrate H3S10, providing an alternative strategy for the specific inhibition of this well-established oncol. target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGSqZBi1UELVg90H21EOLACvtfcHk0lh2LHpJ12usLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFSqsb0%253D&md5=4d35b56dc6e0cdbc0f3e6ccdabefca94</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-3292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-3292%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DDa%2BCosta%26aufirst%3DD.%26aulast%3DThanasopoulou%26aufirst%3DA.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTaniere%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DStankovic%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DPFI-1%252C%2520a%2520highly%2520selective%2520protein%2520interaction%2520inhibitor%252C%2520targeting%2520BET%2520bromodomains%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D3336%26epage%3D3346%26doi%3D10.1158%2F0008-5472.CAN-12-3292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segura, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanals-Cirera, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaziel-Sovran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guijarro, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanniford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">González-Gomez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morante, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jubierre, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvishian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernando, E.</span></span> <span> </span><span class="NLM_article-title">BRD4 sustains melanoma proliferation and represents a new target for epigenetic therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6264</span>– <span class="NLM_lpage">6276</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0122-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1158%2F0008-5472.CAN-13-0122-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23950209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6264-6276&author=M.+F.+Seguraauthor=B.+Fontanals-Cireraauthor=A.+Gaziel-Sovranauthor=M.+V.+Guijarroauthor=D.+Hannifordauthor=G.+Zhangauthor=P.+Gonz%C3%A1lez-Gomezauthor=M.+Moranteauthor=L.+Jubierreauthor=W.+Zhangauthor=F.+Darvishianauthor=M.+Ohlmeyerauthor=I.+Osmanauthor=M.+M.+Zhouauthor=E.+Hernando&title=BRD4+sustains+melanoma+proliferation+and+represents+a+new+target+for+epigenetic+therapy&doi=10.1158%2F0008-5472.CAN-13-0122-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Sustains Melanoma Proliferation and Represents a New Target for Epigenetic Therapy</span></div><div class="casAuthors">Segura, Miguel F.; Fontanals-Cirera, Barbara; Gaziel-Sovran, Avital; Guijarro, Maria V.; Hanniford, Doug; Zhang, Guangtao; Gonzalez-Gomez, Pilar; Morante, Marta; Jubierre, Luz; Zhang, Weijia; Darvishian, Farbod; Ohlmeyer, Michael; Osman, Iman; Zhou, Ming-Ming; Hernando, Eva</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6264-6276</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Metastatic melanoma remains a mostly incurable disease.  Although newly approved targeted therapies are efficacious in a subset of patients, resistance and relapse rapidly ensue.  Alternative therapeutic strategies to manipulate epigenetic regulators and disrupt the transcriptional program that maintains tumor cell identity are emerging.  Bromodomain and extraterminal domain (BET) proteins are epigenome readers known to exert key roles at the interface between chromatin remodeling and transcriptional regulation.  Here, we report that BRD4, a BET family member, is significantly upregulated in primary and metastatic melanoma tissues compared with melanocytes and nevi.  Treatment with BET inhibitors impaired melanoma cell proliferation in vitro and tumor growth and metastatic behavior in vivo, effects that were mostly recapitulated by individual silencing of BRD4.  RNA sequencing of BET inhibitor-treated cells followed by Gene Ontol. anal. showed a striking impact on transcriptional programs controlling cell growth, proliferation, cell-cycle regulation, and differentiation.  In particular, we found that, rapidly after BET displacement, key cell-cycle genes (SKP2, ERK1, and c-MYC) were downregulated concomitantly with the accumulation of cyclin-dependent kinase (CDK) inhibitors (p21 and p27), followed by cell-cycle arrest.  Importantly, BET inhibitor efficacy was not influenced by BRAF or NRAS mutational status, opening the possibility of using these small-mol. compds. to treat patients for whom no effective targeted therapy exists.  Collectively, our study reveals a crit. role for BRD4 in melanoma tumor maintenance and renders it a legitimate and novel target for epigenetic therapy directed against the core transcriptional program of melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80TZH2ueQFbVg90H21EOLACvtfcHk0lh2LHpJ12usLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ahur7N&md5=a4a77f188de01c4c84f7293a32478fc7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0122-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0122-T%26sid%3Dliteratum%253Aachs%26aulast%3DSegura%26aufirst%3DM.%2BF.%26aulast%3DFontanals-Cirera%26aufirst%3DB.%26aulast%3DGaziel-Sovran%26aufirst%3DA.%26aulast%3DGuijarro%26aufirst%3DM.%2BV.%26aulast%3DHanniford%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DGonz%25C3%25A1lez-Gomez%26aufirst%3DP.%26aulast%3DMorante%26aufirst%3DM.%26aulast%3DJubierre%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DDarvishian%26aufirst%3DF.%26aulast%3DOhlmeyer%26aufirst%3DM.%26aulast%3DOsman%26aufirst%3DI.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DHernando%26aufirst%3DE.%26atitle%3DBRD4%2520sustains%2520melanoma%2520proliferation%2520and%2520represents%2520a%2520new%2520target%2520for%2520epigenetic%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6264%26epage%3D6276%26doi%3D10.1158%2F0008-5472.CAN-13-0122-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Epigenetic drug discovery: Breaking through the immune barrier</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">835</span>– <span class="NLM_lpage">853</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnrd.2016.185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27765940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=835-853&author=D.+F.+Toughauthor=P.+P.+Takauthor=A.+Tarakhovskyauthor=R.+K.+Prinjha&title=Epigenetic+drug+discovery%3A+Breaking+through+the+immune+barrier&doi=10.1038%2Fnrd.2016.185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic drug discovery: breaking through the immune barrier</span></div><div class="casAuthors">Tough, David F.; Tak, Paul P.; Tarakhovsky, Alexander; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">835-853</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immune-mediated diseases are clin. heterogeneous but they share genetic and pathogenic mechanisms.  These diseases may develop from the interplay of genetic factors and environmental or lifestyle factors.  Exposure to such factors, including infectious agents, is assocd. with coordinated changes in gene transcription owing to epigenetic alterations.  A growing understanding of how epigenetic mechanisms control gene expression patterns and cell function has been aided by the development of small-mol. inhibitors that target these processes.  These chem. tools have helped to reveal the importance of epigenetics in guiding cell fate decisions during immune responses and have also highlighted the potential for targeting epigenetic mechanisms for the treatment of inflammation and immune-mediated diseases.  In this Review, we discuss the most advanced areas of epigenetic drug development for autoimmune and inflammatory diseases and summarize the promising preclin. data in this exciting and evolving field.  These agents will inevitably begin to move into clin. trials for use in patients with immune-mediated diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplOyEVMzBi37Vg90H21EOLACvtfcHk0lhtnvQbdrpPxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLnE&md5=f939bcbbe3d74ce24718eef706aae3d1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.185%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DTarakhovsky%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DEpigenetic%2520drug%2520discovery%253A%2520Breaking%2520through%2520the%2520immune%2520barrier%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D835%26epage%3D853%26doi%3D10.1038%2Fnrd.2016.185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.2217/epi-2016-0144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.2217%2Fepi-2016-0144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27925476" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=573-584&author=D.+F.+Toughauthor=R.+K.+Prinjha&title=Immune+disease-associated+variants+in+gene+enhancers+point+to+BET+epigenetic+mechanisms+for+therapeutic+intervention&doi=10.2217%2Fepi-2016-0144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Immune disease-associated variants in gene enhancers point to BET epigenetic mechanisms for therapeutic intervention</span></div><div class="casAuthors">Tough, David F.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">573-584</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">Genome-wide assocn. studies have identified thousands of single nucleotide polymorphisms in the human genome that are statistically assocd. with particular disease traits.  In this perspective, we review emerging data suggesting that most single nucleotide polymorphisms assocd. with immune-mediated diseases are found in regulatory regions of the DNA - parts of the genome that control expression of the protein encoding genes - rather than causing mutations in proteins.  We discuss how the emerging understanding of particular gene regulatory regions, gene enhancers and the epigenetic mechanisms by which they are regulated is opening up new opportunities for the treatment of immune-mediated diseases, focusing particularly on the BET family of epigenetic reader proteins as potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFU7Sh3r75ZbVg90H21EOLACvtfcHk0lhtnvQbdrpPxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Onsrk%253D&md5=8a2d44dd000c465b904d4ef2e6f37b77</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2217%2Fepi-2016-0144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2016-0144%26sid%3Dliteratum%253Aachs%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DImmune%2520disease-associated%2520variants%2520in%2520gene%2520enhancers%2520point%2520to%2520BET%2520epigenetic%2520mechanisms%2520for%2520therapeutic%2520intervention%26jtitle%3DEpigenomics%26date%3D2017%26volume%3D9%26spage%3D573%26epage%3D584%26doi%3D10.2217%2Fepi-2016-0144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses TH17-mediated pathology</span>. <i>J. Exp. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>210</i></span>,  <span class="NLM_fpage">2181</span>– <span class="NLM_lpage">2190</span>, <span class="refDoi"> DOI: 10.1084/jem.20130376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1084%2Fjem.20130376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24101376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=210&publication_year=2013&pages=2181-2190&author=D.+A.+Meleauthor=A.+Salmeronauthor=S.+Ghoshauthor=H.+R.+Huangauthor=B.+M.+Bryantauthor=J.+M.+Lora&title=BET+bromodomain+inhibition+suppresses+TH17-mediated+pathology&doi=10.1084%2Fjem.20130376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses TH17-mediated pathology</span></div><div class="casAuthors">Mele, Deanna A.; Salmeron, Andres; Ghosh, Srimoyee; Huang, Hon-Ren; Bryant, Barbara M.; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">210</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">JEMEAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-1007</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Interleukin (IL) 17-producing T helper (TH17) cells have been selected through evolution for their ability to control fungal and bacterial infections.  It is also firmly established that their aberrant generation and activation results in autoimmune conditions.  Using a characterized potent and selective small mol. inhibitor, we show that the bromodomain and extra-terminal domain (BET) family of chromatin adaptors plays fundamental and selective roles in human and murine TH17 differentiation from naive CD4+ T cells, as well as in the activation of previously differentiated TH17 cells.  We provide evidence that BET controls TH17 differentiation in a bromodomain-dependent manner through a mechanism that includes the direct regulation of multiple effector TH17-assocd. cytokines, including IL17, IL21, and GMCSF.  We also demonstrate that BET family members Brd2 and Brd4 assoc. with the Il17 locus in TH17 cells, and that this assocn. requires bromodomains.  We recapitulate the crit. role of BET bromodomains in TH17 differentiation in vivo and show that therapeutic dosing of the BET inhibitor is efficacious in mouse models of autoimmunity.  Our results identify the BET family of proteins as a fundamental link between chromatin signaling and TH17 biol., and support the notion of BET inhibition as a point of therapeutic intervention in autoimmune conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBqVtggp2Fa7Vg90H21EOLACvtfcHk0lhtnvQbdrpPxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ygsLbF&md5=e6822f456cf42cdead9779afa92c05e0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1084%2Fjem.20130376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1084%252Fjem.20130376%26sid%3Dliteratum%253Aachs%26aulast%3DMele%26aufirst%3DD.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DH.%2BR.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520TH17-mediated%2520pathology%26jtitle%3DJ.%2520Exp.%2520Med.%26date%3D2013%26volume%3D210%26spage%3D2181%26epage%3D2190%26doi%3D10.1084%2Fjem.20130376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bandukwala, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Togher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamperti, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molesworth, A. M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14532</span>– <span class="NLM_lpage">14537</span>, <span class="refDoi"> DOI: 10.1073/pnas.1212264109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.1212264109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=22912406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14532-14537&author=H.+S.+Bandukwalaauthor=J.+Gagnonauthor=S.+Togherauthor=J.+A.+Greenbaumauthor=E.+D.+Lampertiauthor=N.+J.+Parrauthor=A.+M.+H.+Molesworthauthor=N.+Smithersauthor=K.+Leeauthor=J.+Witheringtonauthor=D.+F.+Toughauthor=R.+K.+Prinjhaauthor=B.+Petersauthor=A.+Rao&title=Selective+inhibition+of+CD4%2B+T-cell+cytokine+production+and+autoimmunity+by+BET+protein+and+c-Myc+inhibitors&doi=10.1073%2Fpnas.1212264109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of CD4+ T-cell cytokine production and autoimmunity by BET protein and c-Myc inhibitors</span></div><div class="casAuthors">Bandukwala, Hozefa S.; Gagnon, John; Togher, Susan; Greenbaum, Jason A.; Lamperti, Edward D.; Parr, Nigel J.; Molesworth, Amy M. H.; Smithers, Nicholas; Lee, Kevin; Witherington, Jason; Tough, David F.; Prinjha, Rab K.; Peters, Bjoern; Rao, Anjana</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">14532-14537, S14532/1-S14532/21</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins bind acetylated lysine residues on histone tails and are involved in the recruitment of addnl. factors that mediate histone modifications and enable transcription.  A compd., I-BET-762, that inhibits binding of an acetylated histone peptide to proteins of the bromodomain and extra-terminal domain (BET) family, was previously shown to suppress the prodn. of proinflammatory proteins by macrophages and block acute inflammation in mice.  Here, we investigated the effect of short-term treatment with I-BET-762 on T-cell function.  Treatment of naive CD4+ T cells with I-BET-762 during the first 2 d of differentiation had long-lasting effects on subsequent gene expression and cytokine prodn.  Gene expression anal. revealed up-regulated expression of several antiinflammatory gene products, including IL-10, Lag3, and Egr2, and down-regulated expression of several proinflammatory cytokines including GM-CSF and IL-17.  The short 2-d treatment with I-BET-762 inhibited the ability of antigen-specific T cells, differentiated under Th1 but not Th17 conditions in vitro, to induce pathogenesis in an adoptive transfer model of exptl. autoimmune encephalomyelitis.  The suppressive effects of I-BET-762 on T-cell mediated inflammation in vivo were accompanied by decreased recruitment of macrophages, consistent with decreased GM-CSF prodn. by CNS-infiltrating T cells.  These effects were mimicked by an inhibitor of c-myc function, implicating reduced expression of c-myc and GM-CSF as one avenue by which I-BET-762 suppresses the inflammatory functions of T cells.  Our study demonstrates that inhibiting the functions of BET-family proteins during early T-cell differentiation causes long-lasting suppression of the proinflammatory functions of Th1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5akF95GW7K7Vg90H21EOLACvtfcHk0liits9SU72prw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqu7%252FO&md5=110cd83c79e452cb99d01ad902506a5b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1212264109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1212264109%26sid%3Dliteratum%253Aachs%26aulast%3DBandukwala%26aufirst%3DH.%2BS.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DTogher%26aufirst%3DS.%26aulast%3DGreenbaum%26aufirst%3DJ.%2BA.%26aulast%3DLamperti%26aufirst%3DE.%2BD.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DMolesworth%26aufirst%3DA.%2BM.%2BH.%26aulast%3DSmithers%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPeters%26aufirst%3DB.%26aulast%3DRao%26aufirst%3DA.%26atitle%3DSelective%2520inhibition%2520of%2520CD4%252B%2520T-cell%2520cytokine%2520production%2520and%2520autoimmunity%2520by%2520BET%2520protein%2520and%2520c-Myc%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14532%26epage%3D14537%26doi%3D10.1073%2Fpnas.1212264109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarilina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivashkiv, L. B.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span>. <i>Eur. J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1002/eji.201444862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1002%2Feji.201444862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25345375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2015&pages=287-297&author=C.+H.+Chanauthor=C.+Fangauthor=Y.+Qiaoauthor=A.+Yarilinaauthor=R.+K.+Prinjhaauthor=L.+B.+Ivashkiv&title=BET+bromodomain+inhibition+suppresses+transcriptional+responses+to+cytokine-Jak-STAT+signaling+in+a+gene-specific+manner+in+human+monocytes&doi=10.1002%2Feji.201444862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition suppresses transcriptional responses to cytokine-Jak-STAT signaling in a gene-specific manner in human monocytes</span></div><div class="casAuthors">Chan, Chun Hin; Fang, Celestia; Qiao, Yu; Yarilina, Anna; Prinjha, Rab K.; Ivashkiv, Lionel B.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">EJIMAF</span>;
        ISSN:<span class="NLM_cas:issn">0014-2980</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Disruption of the interaction of bromo and extraterminal (BET) proteins with acetylated histones using small mol. inhibitors suppresses Myc-driven cancers and TLR-induced inflammation in mouse models.  The predominant mechanism of BET inhibitor action is to suppress BET-mediated recruitment of pos. transcription elongation factor b and, thus, transcription elongation.  We investigated the effects of BET inhibitor I-BET151 on transcriptional responses to TLR4 and TNF in primary human monocytes and also on responses to cytokines IFN-β, IFN-γ, IL-4, and IL-10, which activate the JAK-STAT signaling pathway and are important for monocyte polarization and inflammatory diseases.  I-BET151 suppressed TLR4- and TNF-induced IFN responses by diminishing both autocrine IFN-β expression and transcriptional responses to IFN-β.  I-BET151 inhibited cytokine-induced transcription of STAT targets in a gene-specific manner without affecting STAT activation or recruitment.  This inhibition was independent of Myc or other upstream activators.  IFN-stimulated gene transcription is regulated primarily at the level of transcription initiation.  Accordingly, we found that I-BET151 suppressed the recruitment of transcriptional machinery to the CXCL10 promoter and an upstream enhancer.  Our findings suggest that BET inhibition reduces inflammation partially through suppressing cytokine activity and expands the understanding of the inhibitory and potentially selective immunosuppressive effects of inhibiting BET proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbE0f71lXpp7Vg90H21EOLACvtfcHk0liits9SU72prw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jurvP&md5=efa081bc6c79db9233ae1a99ebf49945</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Feji.201444862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Feji.201444862%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DC.%2BH.%26aulast%3DFang%26aufirst%3DC.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DYarilina%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DIvashkiv%26aufirst%3DL.%2BB.%26atitle%3DBET%2520bromodomain%2520inhibition%2520suppresses%2520transcriptional%2520responses%2520to%2520cytokine-Jak-STAT%2520signaling%2520in%2520a%2520gene-specific%2520manner%2520in%2520human%2520monocytes%26jtitle%3DEur.%2520J.%2520Immunol.%26date%3D2015%26volume%3D45%26spage%3D287%26epage%3D297%26doi%3D10.1002%2Feji.201444862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schilderink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reginato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilbers, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, W. J.</span></span> <span> </span><span class="NLM_article-title">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">66</span>– <span class="NLM_lpage">76</span>, <span class="refDoi"> DOI: 10.1016/j.molimm.2016.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.molimm.2016.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=27710838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2016&pages=66-76&author=R.+Schilderinkauthor=M.+Bellauthor=E.+Reginatoauthor=C.+Pattenauthor=I.+Riojaauthor=F.+W.+Hilbersauthor=P.+A.+Kabalaauthor=K.+A.+Reedquistauthor=D.+F.+Toughauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=W.+J.+de+Jonge&title=BET+bromodomain+inhibition+reduces+maturation+and+enhances+tolerogenic+properties+of+human+and+mouse+dendritic+cells&doi=10.1016%2Fj.molimm.2016.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">BET bromodomain inhibition reduces maturation and enhances tolerogenic properties of human and mouse dendritic cells</span></div><div class="casAuthors">Schilderink, Ronald; Bell, Matthew; Reginato, Eleonora; Patten, Chris; Rioja, Inmaculada; Hilbers, Francisca W.; Kabala, Pawel A.; Reedquist, Kris A.; Tough, David F.; Tak, Paul Peter; Prinjha, Rab K.; de Jonge, Wouter J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">66-76</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Transcription of inflammatory genes is tightly regulated by acetylation and deacetylation of histone tails.  An inhibitor of the acetylated-lysine reader bromodomain and extra-terminal domain (BET) proteins, I-BET151, is known to counteract the induction of expression of inflammatory genes in macrophages.  We have investigated the effects of I-BET151 on dendritic cell function, including expression of co-stimulatory mols. and cytokines, and capacity for T cell activation.  Treatment of mouse bone marrow derived dendritic cells (BMDC) and human monocyte derived DCs (mdDC) with I-BET151 reduced LPS-induced expression of co-stimulatory mols., as well as the prodn. of multiple cytokines and chemokines.  Most strikingly, secretion of IL-6, IL-12 and IL-10 was significantly reduced to 89.7%, 99.9% and 98.6% resp. of that produced by control cells.  I-BET151-treated mdDC showed a reduced ability to stimulate proliferation of autologous Revaxis-specific T cells.  Moreover, while I-BET151 treatment of BMDC did not affect their ability to polarize ovalbumin specific CD4+ CD62L+ naive T cells towards Th1, Th2, or Th17 phenotypes, an increase in Foxp3 expressing Tregs secreting higher IL-10 levels was obsd.  Suppression assays demonstrated that Tregs generated in response to I-BET151-treated BMDC displayed anti-proliferative capacity.  Finally, evidence that I-BET151 treatment can ameliorate inflammation in vivo in a T cell dependent colitis model is shown.  Overall, these results demonstrate marked effects of BET inhibition on DC maturation, reducing their capacity for pro-inflammatory cytokine secretion and T cell activation and enhancing the potential of DC to induce Foxp3 expressing Treg with suppressive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuoARRI77BrVg90H21EOLACvtfcHk0li6Ua9CdY0hfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Slu73I&md5=6a0da0ba320112023cb09df03c4802dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.molimm.2016.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molimm.2016.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DSchilderink%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DM.%26aulast%3DReginato%26aufirst%3DE.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DHilbers%26aufirst%3DF.%2BW.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3Dde%2BJonge%26aufirst%3DW.%2BJ.%26atitle%3DBET%2520bromodomain%2520inhibition%2520reduces%2520maturation%2520and%2520enhances%2520tolerogenic%2520properties%2520of%2520human%2520and%2520mouse%2520dendritic%2520cells%26jtitle%3DMol.%2520Immunol.%26date%3D2016%26volume%3D79%26spage%3D66%26epage%3D76%26doi%3D10.1016%2Fj.molimm.2016.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolajczyk, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET protein function is required for inflammation: Brd2 genetic disruption and BET inhibitor JQ1 impair mouse macrophage inflammatory responses</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>190</i></span>,  <span class="NLM_fpage">3670</span>– <span class="NLM_lpage">3678</span>, <span class="refDoi"> DOI: 10.4049/jimmunol.1202838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.4049%2Fjimmunol.1202838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23420887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=190&publication_year=2013&pages=3670-3678&author=A.+C.+Belkinaauthor=B.+S.+Nikolajczykauthor=G.+V.+Denis&title=BET+protein+function+is+required+for+inflammation%3A+Brd2+genetic+disruption+and+BET+inhibitor+JQ1+impair+mouse+macrophage+inflammatory+responses&doi=10.4049%2Fjimmunol.1202838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">BET Protein Function Is Required for Inflammation: Brd2 Genetic Disruption and BET Inhibitor JQ1 Impair Mouse Macrophage Inflammatory Responses</span></div><div class="casAuthors">Belkina, Anna C.; Nikolajczyk, Barbara S.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">190</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3670-3678</span>CODEN:
                <span class="NLM_cas:coden">JOIMA3</span>;
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    
            (<span class="NLM_cas:orgname">American Association of Immunologists</span>)
        </div><div class="casAbstract">Histone acetylation regulates activation and repression of multiple inflammatory genes known to play crit. roles in chronic inflammatory diseases.  However, proteins responsible for translating the histone acetylation code into an orchestrated proinflammatory cytokine response remain poorly characterized.  Bromodomain and extraterminal (BET) proteins are "readers" of histone acetylation marks, with demonstrated roles in gene transcription, but the ability of BET proteins to coordinate the response of inflammatory cytokine genes through translation of histone marks is unknown.  We hypothesize that members of the BET family of dual bromodomain-contg. transcriptional regulators directly control inflammatory genes.  We examd. the genetic model of brd2 lo mice, a BET protein hypomorph, to show that Brd2 is essential for proinflammatory cytokine prodn. in macrophages.  Studies that use small interfering RNA knockdown and a small-mol. inhibitor of BET protein binding, JQ1, independently demonstrate BET proteins are crit. for macrophage inflammatory responses.  Furthermore, we show that Brd2 and Brd4 phys. assoc. with the promoters of inflammatory cytokine genes in macrophages.  This assocn. is absent in the presence of BET inhibition by JQ1.  Finally, we demonstrate that JQ1 ablates cytokine prodn. in vitro and blunts the "cytokine storm" in endotoxemic mice by reducing levels of IL-6 and TNF-α while rescuing mice from LPS-induced death.  We propose that targeting BET proteins with small-mol. inhibitors will benefit hyperinflammatory conditions assocd. with high levels of cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrbebEe0G277Vg90H21EOLACvtfcHk0li6Ua9CdY0hfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVynsbk%253D&md5=8c4f69177b9ed814a7db339a8e2ab7ed</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1202838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1202838%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DNikolajczyk%26aufirst%3DB.%2BS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520protein%2520function%2520is%2520required%2520for%2520inflammation%253A%2520Brd2%2520genetic%2520disruption%2520and%2520BET%2520inhibitor%2520JQ1%2520impair%2520mouse%2520macrophage%2520inflammatory%2520responses%26jtitle%3DJ.%2520Immunol.%26date%3D2013%26volume%3D190%26spage%3D3670%26epage%3D3678%26doi%3D10.4049%2Fjimmunol.1202838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabala, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabiec, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tak, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ospelt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reedquist, K. A.</span></span> <span> </span><span class="NLM_article-title">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span>. <i>Ann. Rheum. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1136/annrheumdis-2014-205809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1136%2Fannrheumdis-2014-205809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25467295" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2016&pages=422-429&author=K.+Kleinauthor=P.+A.+Kabalaauthor=A.+M.+Grabiecauthor=R.+E.+Gayauthor=C.+Kollingauthor=L.+L.+Linauthor=S.+Gayauthor=P.+P.+Takauthor=R.+K.+Prinjhaauthor=C.+Ospeltauthor=K.+A.+Reedquist&title=The+bromodomain+protein+inhibitor+I-BET151+suppresses+expression+of+inflammatory+genes+and+matrix+degrading+enzymes+in+rheumatoid+arthritis+synovial+fibroblasts&doi=10.1136%2Fannrheumdis-2014-205809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The bromodomain protein inhibitor I-BET151 suppresses expression of inflammatory genes and matrix degrading enzymes in rheumatoid arthritis synovial fibroblasts</span></div><div class="casAuthors">Klein, Kerstin; Kabala, Pawel A.; Grabiec, Aleksander M.; Gay, Renate E.; Kolling, Christoph; Lin, Lih-Ling; Gay, Steffen; Tak, Paul P.; Prinjha, Rab K.; Ospelt, Caroline; Reedquist, Kris A.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the Rheumatic Diseases</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">422-429</span>CODEN:
                <span class="NLM_cas:coden">ARDIAO</span>;
        ISSN:<span class="NLM_cas:issn">0003-4967</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Objective To investigate the effects of BET bromodomain protein inhibition on inflammatory activation and functional properties of rheumatoid arthritis synovial fibroblasts (RASF).  The expression of the BET bromodomain proteins BRD2, BRD3 and BRD4 was analyzed in synovial tissue by immunohistochem.  RASF were stimulated with tumor necrosis factor (TNF)- α, interleukin (IL)-1β and toll-like receptor (TLR) ligands (Pam3, pIC and lipopolysaccharide (LPS)) in the presence or absence of the BET inhibitor I-BET151, or siRNA targeting BRD2, BRD3 and BRD4.  RASF expression of inflammatory mediators, including MMP1, MMP3, IL-6 and IL-8, was measured by q-PCR, q-PCR array and ELISA.  Cellular viability, apoptosis, proliferation and chemoattractive properties of RASF were investigated using MTT, cell apoptosis ELISA, BrdU-based proliferation and transwell migration assays.  BRD2, BRD3 and BRD4 proteins were detected in rheumatoid arthritis (RA) synovial tissue, expressed in both RASF and macrophages.  I-BET151 suppressed cytokine and TLR ligand-induced secretion of MMP1, MMP3, IL-6 and IL-8, and mRNA expression of more than 70% of genes induced by TNF- α and IL-1β .  Combined silencing of BRD2, BRD3 and BRD4 significantly reduced cytokine and TLR ligand-induced expression of a subset of gene products targeted by I-BET151, including MMP1, CXCL10 and CXCL11.  I-BET151 treatment of RASF reduced RASF proliferation, and the chemotactic potential for peripheral blood leukocytes of RASF conditioned medium.  Conclusions Inhibition of BET family proteins suppresses the inflammatory, matrix-degrading, proliferative and chemoattractive properties of RASF and suggests a therapeutic potential in the targeting of epigenetic reader proteins in RA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELw7WcQzbLLVg90H21EOLACvtfcHk0li9UTkagp0PKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGnur%252FO&md5=97f611195571a9772ff7668d06cdb335</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1136%2Fannrheumdis-2014-205809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fannrheumdis-2014-205809%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DK.%26aulast%3DKabala%26aufirst%3DP.%2BA.%26aulast%3DGrabiec%26aufirst%3DA.%2BM.%26aulast%3DGay%26aufirst%3DR.%2BE.%26aulast%3DKolling%26aufirst%3DC.%26aulast%3DLin%26aufirst%3DL.%2BL.%26aulast%3DGay%26aufirst%3DS.%26aulast%3DTak%26aufirst%3DP.%2BP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DOspelt%26aufirst%3DC.%26aulast%3DReedquist%26aufirst%3DK.%2BA.%26atitle%3DThe%2520bromodomain%2520protein%2520inhibitor%2520I-BET151%2520suppresses%2520expression%2520of%2520inflammatory%2520genes%2520and%2520matrix%2520degrading%2520enzymes%2520in%2520rheumatoid%2520arthritis%2520synovial%2520fibroblasts%26jtitle%3DAnn.%2520Rheum.%2520Dis.%26date%3D2016%26volume%3D75%26spage%3D422%26epage%3D429%26doi%3D10.1136%2Fannrheumdis-2014-205809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dewell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandwani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marazzi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarakhovsky, A.</span></span> <span> </span><span class="NLM_article-title">Suppression of inflammation by a synthetic histone mimic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1038/nature09589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnature09589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=21068722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1119-1123&author=E.+Nicodemeauthor=K.+L.+Jeffreyauthor=U.+Schaeferauthor=S.+Beinkeauthor=S.+Dewellauthor=C.+W.+Chungauthor=R.+Chandwaniauthor=I.+Marazziauthor=P.+Wilsonauthor=H.+Costeauthor=J.+Whiteauthor=J.+Kirilovskyauthor=C.+M.+Riceauthor=J.+M.+Loraauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=A.+Tarakhovsky&title=Suppression+of+inflammation+by+a+synthetic+histone+mimic&doi=10.1038%2Fnature09589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of inflammation by a synthetic histone mimic</span></div><div class="casAuthors">Nicodeme, Edwige; Jeffrey, Kate L.; Schaefer, Uwe; Beinke, Soren; Dewell, Scott; Chung, Chun-wa; Chandwani, Rohit; Marazzi, Ivan; Wilson, Paul; Coste, Herve; White, Julia; Kirilovsky, Jorge; Rice, Charles M.; Lora, Jose M.; Prinjha, Rab K.; Lee, Kevin; Tarakhovsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1119-1123</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Interaction of pathogens with cells of the immune system results in activation of inflammatory gene expression.  This response, although vital for immune defense, is frequently deleterious to the host due to the exaggerated prodn. of inflammatory proteins.  The scope of inflammatory responses reflects the activation state of signaling proteins upstream of inflammatory genes as well as signal-induced assembly of nuclear chromatin complexes that support mRNA expression.  Recognition of post-translationally modified histones by nuclear proteins that initiate mRNA transcription and support mRNA elongation is a crit. step in the regulation of gene expression.  Here we present a novel pharmacol. approach that targets inflammatory gene expression by interfering with the recognition of acetylated histones by the bromodomain and extra terminal domain (BET) family of proteins.  We describe a synthetic compd. (I-BET) that by mimicking' acetylated histones disrupts chromatin complexes responsible for the expression of key inflammatory genes in activated macrophages, and confers protection against lipopolysaccharide-induced endotoxic shock and bacteria-induced sepsis.  Our findings suggest that synthetic compds. specifically targeting proteins that recognize post-translationally modified histones can serve as a new generation of immunomodulatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxc3_PNaiuKLVg90H21EOLACvtfcHk0li9UTkagp0PKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2rtb%252FP&md5=0dfccd9d01b87859512bf591e1c5a93c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature09589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09589%26sid%3Dliteratum%253Aachs%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DJeffrey%26aufirst%3DK.%2BL.%26aulast%3DSchaefer%26aufirst%3DU.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DDewell%26aufirst%3DS.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DChandwani%26aufirst%3DR.%26aulast%3DMarazzi%26aufirst%3DI.%26aulast%3DWilson%26aufirst%3DP.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DRice%26aufirst%3DC.%2BM.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DTarakhovsky%26aufirst%3DA.%26atitle%3DSuppression%2520of%2520inflammation%2520by%2520a%2520synthetic%2520histone%2520mimic%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1119%26epage%3D1123%26doi%3D10.1038%2Fnature09589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nadeem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Harbi, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sherbeeny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoheir, K. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attia, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hosaini, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Sharary, S. D.</span></span> <span> </span><span class="NLM_article-title">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.phrs.2015.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=26149470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2015&pages=248-257&author=A.+Nadeemauthor=N.+O.+Al-Harbiauthor=M.+M.+Al-Harbiauthor=A.+M.+El-Sherbeenyauthor=S.+F.+Ahmadauthor=N.+Siddiquiauthor=M.+A.+Ansariauthor=K.+M.+A.+Zoheirauthor=S.+M.+Attiaauthor=K.+A.+Al-Hosainiauthor=S.+D.+Al-Sharary&title=Imiquimod-induced+psoriasis-like+skin+inflammation+is+suppressed+by+BET+bromodomain+inhibitor+in+mice+through+RORC%2FIL-17A+pathway+modulation&doi=10.1016%2Fj.phrs.2015.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Imiquimod-induced psoriasis-like skin inflammation is suppressed by BET bromodomain inhibitor in mice through RORC/IL-17A pathway modulation</span></div><div class="casAuthors">Nadeem, Ahmed; Al-Harbi, Naif O.; Al-Harbi, Mohamed M.; El-Sherbeeny, Ahmed M.; Ahmad, Sheikh F.; Siddiqui, Nahid; Ansari, Mushtaq A.; Zoheir, Khairy M. A.; Attia, Sabry M.; Al-Hosaini, Khaled A.; Al-Sharary, Shakir D.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">248-257</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Psoriasis is one of the most common skin disorders characterized by erythematous plaques that result from hyperproliferative keratinocytes and infiltration of inflammatory leukocytes into dermis and epidermis.  Recent studies suggest that IL-23/IL-17A/IL-22 cytokine axis plays an important role in the pathogenesis of psoriasis.  The small mol. bromodomain and extraterminal domain (BET) inhibitors, that disrupt interaction of BET proteins with acetylated histones have recently demonstrated efficacy in various models of inflammation through suppression of several pathways, one of them being synthesis of IL-17A/IL-22 which primarily depends on transcription factor, retinoic acid receptor-related orphan receptor C (RORC).  However, the efficacy and mechanistic aspect of a BET inhibitor in mouse model of skin inflammation has not been explored previously.  Therefore, this study investigated the role of BET inhibitor, JQ-1 in mouse model of psoriasis-like inflammation.  Mice were topically applied imiquimod (IMQ) to develop psoriasis-like inflammation on the shaved back and ear followed by assessment of skin inflammation (myeloperoxidase activity, ear thickness, and histopathol.), RORC and its signature cytokines (IL-17A/IL-22).  JQ-1 suppressed IMQ-induced skin inflammation as reflected by a decrease in ear thickness/myeloperoxidase activity, and RORC/IL-17A/IL-22 expression.  Addnl., a RORα/γ agonist SR1078 was utilized to investigate the role of RORC in BET-mediated skin inflammation.  SR1078 reversed the protective effect of JQ-1 on skin inflammation at both histol. and mol. levels in the IMQ model.  The current study suggests that BET bromodomains are involved in psoriasis-like inflammation through induction of RORC/IL-17A pathway.  Therefore, inhibition of BET bromodomains may provide a new therapy against skin inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOnC_sPUMSLVg90H21EOLACvtfcHk0lhODeoKUqjusw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2is77I&md5=3edb10526e30c18bce037dc0aca611a4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DNadeem%26aufirst%3DA.%26aulast%3DAl-Harbi%26aufirst%3DN.%2BO.%26aulast%3DAl-Harbi%26aufirst%3DM.%2BM.%26aulast%3DEl-Sherbeeny%26aufirst%3DA.%2BM.%26aulast%3DAhmad%26aufirst%3DS.%2BF.%26aulast%3DSiddiqui%26aufirst%3DN.%26aulast%3DAnsari%26aufirst%3DM.%2BA.%26aulast%3DZoheir%26aufirst%3DK.%2BM.%2BA.%26aulast%3DAttia%26aufirst%3DS.%2BM.%26aulast%3DAl-Hosaini%26aufirst%3DK.%2BA.%26aulast%3DAl-Sharary%26aufirst%3DS.%2BD.%26atitle%3DImiquimod-induced%2520psoriasis-like%2520skin%2520inflammation%2520is%2520suppressed%2520by%2520BET%2520bromodomain%2520inhibitor%2520in%2520mice%2520through%2520RORC%252FIL-17A%2520pathway%2520modulation%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D99%26spage%3D248%26epage%3D257%26doi%3D10.1016%2Fj.phrs.2015.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeCrescenzo, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devraj, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellsworth, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fobian, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krieger-Burke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loesel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whiteley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Physiochemical drug properties associated with in vivo toxicological outcomes</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4872</span>– <span class="NLM_lpage">4875</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.07.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.bmcl.2008.07.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=18691886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=4872-4875&author=J.+D.+Hughesauthor=J.+Blaggauthor=D.+A.+Priceauthor=S.+Baileyauthor=G.+A.+DeCrescenzoauthor=R.+V.+Devrajauthor=E.+Ellsworthauthor=Y.+M.+Fobianauthor=M.+E.+Gibbsauthor=R.+W.+Gillesauthor=N.+Greeneauthor=E.+Huangauthor=T.+Krieger-Burkeauthor=J.+Loeselauthor=T.+Wagerauthor=L.+Whiteleyauthor=Y.+Zhang&title=Physiochemical+drug+properties+associated+with+in+vivo+toxicological+outcomes&doi=10.1016%2Fj.bmcl.2008.07.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Physiochemical drug properties associated with in vivo toxicological outcomes</span></div><div class="casAuthors">Hughes, Jason D.; Blagg, Julian; Price, David A.; Bailey, Simon; DeCrescenzo, Gary A.; Devraj, Rajesh V.; Ellsworth, Edmund; Fobian, Yvette M.; Gibbs, Michael E.; Gilles, Richard W.; Greene, Nigel; Huang, Enoch; Krieger-Burke, Teresa; Loesel, Jens; Wager, Travis; Whiteley, Larry; Zhang, Yao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4872-4875</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Relationships between physicochem. drug properties and toxicity were inferred from a data set consisting of animal in vivo toleration (IVT) studies on 245 preclin. Pfizer compds.; an increased likelihood of toxic events was found for less polar, more lipophilic compds.  This trend held across a wide range of types of toxicity and across a broad swath of chem. space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPMmGEpVF0mbVg90H21EOLACvtfcHk0lhODeoKUqjusw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOnsbjE&md5=80ea5ced359112a363f38b3545cd7646</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.07.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.07.071%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DJ.%2BD.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DBailey%26aufirst%3DS.%26aulast%3DDeCrescenzo%26aufirst%3DG.%2BA.%26aulast%3DDevraj%26aufirst%3DR.%2BV.%26aulast%3DEllsworth%26aufirst%3DE.%26aulast%3DFobian%26aufirst%3DY.%2BM.%26aulast%3DGibbs%26aufirst%3DM.%2BE.%26aulast%3DGilles%26aufirst%3DR.%2BW.%26aulast%3DGreene%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DE.%26aulast%3DKrieger-Burke%26aufirst%3DT.%26aulast%3DLoesel%26aufirst%3DJ.%26aulast%3DWager%26aufirst%3DT.%26aulast%3DWhiteley%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DPhysiochemical%2520drug%2520properties%2520associated%2520with%2520in%2520vivo%2520toxicological%2520outcomes%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D4872%26epage%3D4875%26doi%3D10.1016%2Fj.bmcl.2008.07.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: A novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+A+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2earYJRxUKkfb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520A%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=32193360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=387-394&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Y.+N.+Lamauthor=C.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+G.+S.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.+M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.+J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immunoinflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span></div><div class="casAuthors">Gilan, Omer; Rioja, Inmaculada; Knezevic, Kathy; Bell, Matthew J.; Yeung, Miriam M.; Harker, Nicola R.; Lam, Enid Y. N.; Chung, Chun-wa; Bamborough, Paul; Petretich, Massimo; Urh, Marjeta; Atkinson, Stephen J.; Bassil, Anna K.; Roberts, Emma J.; Vassiliadis, Dane; Burr, Marian L.; Preston, Alex G. S.; Wellaway, Christopher; Werner, Thilo; Gray, James R.; Michon, Anne-Marie; Gobbetti, Thomas; Kumar, Vinod; Soden, Peter E.; Haynes, Andrea; Vappiani, Johanna; Tough, David F.; Taylor, Simon; Dawson, Sarah-Jane; Bantscheff, Marcus; Lindon, Matthew; Drewes, Gerard; Demont, Emmanuel H.; Daniels, Danette L.; Grandi, Paola; Prinjha, Rab K.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">387-394</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The two tandem bromodomains of the BET (bromodomain and extraterminal domain) proteins enable chromatin binding to facilitate transcription.  Drugs that inhibit both bromodomains equally have shown efficacy in certain malignant and inflammatory conditions.  To explore the individual functional contributions of the first (BD1) and second (BD2) bromodomains in biol. and therapy, we developed selective BD1 and BD2 inhibitors.  We found that steady-state gene expression primarily requires BD1, whereas the rapid increase of gene expression induced by inflammatory stimuli requires both BD1 and BD2 of all BET proteins.  BD1 inhibitors phenocopied the effects of pan-BET inhibitors in cancer models, whereas BD2 inhibitors were predominantly effective in models of inflammatory and autoimmune disease.  These insights into the differential requirement of BD1 and BD2 for the maintenance and induction of gene expression may guide future BET-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr078AuJubsqbVg90H21EOLACvtfcHk0ljzNaBkLAyKrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D&md5=d78cfea091fcf5ee18735b02b2120dd7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%2BY.%2BN.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%2BG.%2BS.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immunoinflammation%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2020%26volume%3D368%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Benzo[ cd]indol-2(1 H)-ones and Pyrrolo[4,3,2- de]quinolin-2(1 H)-ones as bromodomain and extra-terminal domain (BET) Inhibitors with selectivity for the first bromodomain with potential high efficiency against acute gouty arthritis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11080</span>– <span class="NLM_lpage">11107</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11080-11107&author=F.+Jiangauthor=Q.+Huauthor=Z.+Zhangauthor=H.+Liauthor=H.+Liauthor=D.+Zhangauthor=H.+Liauthor=Y.+Maauthor=J.+Xuauthor=H.+Chenauthor=Y.+Cuiauthor=Y.+Zhiauthor=Y.+Zhangauthor=J.+Xuauthor=J.+Zhuauthor=T.+Luauthor=Y.+Chen&title=Discovery+of+Benzo%5B+cd%5Dindol-2%281+H%29-ones+and+Pyrrolo%5B4%2C3%2C2-+de%5Dquinolin-2%281+H%29-ones+as+bromodomain+and+extra-terminal+domain+%28BET%29+Inhibitors+with+selectivity+for+the+first+bromodomain+with+potential+high+efficiency+against+acute+gouty+arthritis&doi=10.1021%2Facs.jmedchem.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis</span></div><div class="casAuthors">Jiang, Fei; Hu, Qinghua; Zhang, Zhimin; Li, Hongmei; Li, Huili; Zhang, Dewei; Li, Hanwen; Ma, Yu; Xu, Jingjing; Chen, Haifang; Cui, Yong; Zhi, Yanle; Zhang, Yanmin; Xu, Junyu; Zhu, Jiapeng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11080-11107</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues.  Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2).  Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family.  Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1.  Through structure-based optimization, highly active and selective compds. are ultimately obtained.  The representative compds. are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2).  Among them, we further show that 68 (LT052) mediates BRD4/NF-κB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model.  Therefore, selective pharmacol. modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav2e3P7XQ1LVg90H21EOLACvtfcHk0ljzNaBkLAyKrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E&md5=5cce669e137812d20f8ba2a9785f4666</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Benzo%255B%2520cd%255Dindol-2%25281%2520H%2529-ones%2520and%2520Pyrrolo%255B4%252C3%252C2-%2520de%255Dquinolin-2%25281%2520H%2529-ones%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520Inhibitors%2520with%2520selectivity%2520for%2520the%2520first%2520bromodomain%2520with%2520potential%2520high%2520efficiency%2520against%2520acute%2520gouty%2520arthritis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11080%26epage%3D11107%26doi%3D10.1021%2Facs.jmedchem.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK789: A selective Inhibitor of the first bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>– <span class="NLM_lpage">9069</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.+W.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=GSK789%3A+A+selective+Inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+Bromo+and+Extra+Terminal+Domain+%28BET%29+Proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0ljkU_qVpHApbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK789%253A%2520A%2520selective%2520Inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520Bromo%2520and%2520Extra%2520Terminal%2520Domain%2520%2528BET%2529%2520Proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as Bromodomain and Extra-Terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+J.+B.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+Bromodomain+and+Extra-Terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0ljkU_qVpHApbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%2BJ.%2BB.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520Bromodomain%2520and%2520Extra-Terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thesmar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">GSK973 is an inhibitor of the second bromodomains (BD2s) of the Bromodomain and Extra-Terminal (BET) family</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1581</span>– <span class="NLM_lpage">1587</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlegsrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1581-1587&author=A.+Prestonauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=L.+J.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=A.+J.+Lewisauthor=D.+Lugoauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=P.+Thesmarauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=GSK973+is+an+inhibitor+of+the+second+bromodomains+%28BD2s%29+of+the+Bromodomain+and+Extra-Terminal+%28BET%29+family&doi=10.1021%2Facsmedchemlett.0c00247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">GSK973 Is an Inhibitor of the Second Bromodomains (BD2s) of the Bromodomain and Extra-Terminal (BET) Family</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Gordon, Laurie J.; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Lewis, Antonia J.; Lugo, David; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rioja, Inmaculada; Seal, Jon; Taylor, Simon; Thesmar, Pierre; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1581-1587</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-BET inhibitors have shown profound efficacy in a no. of in vivo preclin. models and have entered the clinic in oncol. trials where adverse events have been reported.  These inhibitors interact equipotently with the eight bromodomains of the BET family of proteins.  To better understand the contribution of each domain to their efficacy and to improve from their safety profile, selective inhibitors are required.  This Letter discloses the profile of GSK973, a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive preclin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqErinCVgS7LVg90H21EOLACvtfcHk0lhKz46bllN3lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlegsrfL&md5=93fbcba928a55167dae6825e87548a28</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00247%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DThesmar%26aufirst%3DP.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DGSK973%2520is%2520an%2520inhibitor%2520of%2520the%2520second%2520bromodomains%2520%2528BD2s%2529%2520of%2520the%2520Bromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520family%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1581%26epage%3D1587%26doi%3D10.1021%2Facsmedchemlett.0c00247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak Bromo and Extra Terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>– <span class="NLM_lpage">9126</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+B.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+Bromo+and+Extra+Terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0lhKz46bllN3lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCopley%26aufirst%3DR.%2BC.%2BB.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520Bromo%2520and%2520Extra%2520Terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Template-hopping approach leads to potent, selective, and highly soluble Bromo and Extraterminal Domain (BET) second bromodomain (BD2) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">24</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c02156</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c02156" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=24&author=H.+E.+Aylottauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=A.+Bassilauthor=C.+W.+Chungauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=C.+Messengerauthor=D.+Mitchellauthor=A.+Phillipouauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+T.+Sealauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=Template-hopping+approach+leads+to+potent%2C+selective%2C+and+highly+soluble+Bromo+and+Extraterminal+Domain+%28BET%29+second+bromodomain+%28BD2%29+inhibitors&doi=10.1021%2Facs.jmedchem.0c02156"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c02156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c02156%26sid%3Dliteratum%253Aachs%26aulast%3DAylott%26aufirst%3DH.%2BE.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBassil%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DTemplate-hopping%2520approach%2520leads%2520to%2520potent%252C%2520selective%252C%2520and%2520highly%2520soluble%2520Bromo%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D24%26doi%3D10.1021%2Facs.jmedchem.0c02156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anglin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ucisik, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faver, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyshadham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmons, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storl-Desmond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of bromodomain 2-specific inhibitors of BRDT</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">2021102118</span>, <span class="refDoi"> DOI: 10.1073/pnas.2021102118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.2021102118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXls12qur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2021&pages=2021102118&author=Z.+Yuauthor=A.+F.+Kuauthor=J.+L.+Anglinauthor=R.+Sharmaauthor=M.+N.+Ucisikauthor=J.+C.+Faverauthor=F.+Liauthor=P.+Nyshadhamauthor=N.+Simmonsauthor=K.+L.+Sharmaauthor=S.+Nagarajanauthor=K.+Riehleauthor=G.+Kaurauthor=B.+Sankaranauthor=M.+Storl-Desmondauthor=S.+S.+Palmerauthor=D.+W.+Youngauthor=C.+Kimauthor=M.+M.+Matzuk&title=Discovery+and+characterization+of+bromodomain+2-specific+inhibitors+of+BRDT&doi=10.1073%2Fpnas.2021102118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of bromodomain 2-specific inhibitors of BRDT</span></div><div class="casAuthors">Yu, Zhifeng; Ku, Angela F.; Anglin, Justin L.; Sharma, Rajesh; Ucisik, Melek Nihan; Faver, John C.; Li, Feng; Nyshadham, Pranavanand; Simmons, Nicholas; Sharma, Kiran L.; Nagarajan, Sureshbabu; Riehle, Kevin; Kaur, Gundeep; Sankaran, Banumathi; Storl-Desmond, Marta; Palmer, Stephen S.; Young, Damian W.; Kim, Choel; Matzuk, Martin M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e2021102118</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomain testis (BRDT), a member of the bromodomain and extraterminal (BET) subfamily that includes the cancer targets BRD2, BRD3, and BRD4, is a validated contraceptive target.  All BET subfamily members have two tandem bromodomains (BD1 and BD2).  Knockout mice lacking BRDT-BD1 or both bromodomains are infertile.  Treatment of mice with JQ1, a BET BD1/BD2 nonselective inhibitor with the highest affinity for BRD4, disrupts spermatogenesis and reduces sperm no. and motility.  To assess the contribution of each BRDT bromodomain, we screened our collection of DNA-encoded chem. libraries for BRDT-BD1 and BRDT-BD2 binders.  High-enrichment hits were identified and resynthesized off-DNA and examd. for their ability to compete with JQ1 in BRDT and BRD4 bromodomain AlphaScreen assays.  These studies identified CDD-1102 as a selective BRDT-BD2 inhibitor with low nanomolar potency and &gt;1,000-fold selectivity over BRDT-BD1.  Structure-activity relationship studies of CDD-1102 produced a series of addnl. BRDT-BD2/BRD4-BD2 selective inhibitors, including CDD-1302, a truncated analog of CDD-1102 with similar activity, and CDD-1349, an analog with sixfold selectivity for BRDT-BD2 vs. BRD4-BD2.  BROMOscan bromodomain profiling confirmed the great affinity and selectivity of CDD-1102 and CDD-1302 on all BET BD2 vs. BD1 with the highest affinity for BRDT-BD2.  Cocrystals of BRDT-BD2 with CDD-1102 and CDD-1302 were detd. at 2.27 and 1.90 Å resoln., resp., and revealed BRDT-BD2 specific contacts that explain the high affinity and selectivity of these compds.  These BD2-specific compds. and their binding to BRDT-BD2 are unique compared with recent reports and enable further evaluation of their nonhormonal contraceptive potential in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe2-U6Ypwvm7Vg90H21EOLACvtfcHk0li5AsnbR8zUAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXls12qur0%253D&md5=ff66b0f0b9ec50d33e8fafdc96303f8f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2021102118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2021102118%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DKu%26aufirst%3DA.%2BF.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DUcisik%26aufirst%3DM.%2BN.%26aulast%3DFaver%26aufirst%3DJ.%2BC.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DNyshadham%26aufirst%3DP.%26aulast%3DSimmons%26aufirst%3DN.%26aulast%3DSharma%26aufirst%3DK.%2BL.%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DRiehle%26aufirst%3DK.%26aulast%3DKaur%26aufirst%3DG.%26aulast%3DSankaran%26aufirst%3DB.%26aulast%3DStorl-Desmond%26aufirst%3DM.%26aulast%3DPalmer%26aufirst%3DS.%2BS.%26aulast%3DYoung%26aufirst%3DD.%2BW.%26aulast%3DKim%26aufirst%3DC.%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520bromodomain%25202-specific%2520inhibitors%2520of%2520BRDT%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2021%26volume%3D118%26spage%3D2021102118%26doi%3D10.1073%2Fpnas.2021102118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span> <span> </span><span class="NLM_article-title">Domain-selective targeting of BET proteins in cancer and immunological diseases</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2020.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.cbpa.2020.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=32741705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2020&pages=184-193&author=M.+Petretichauthor=E.+H.+Demontauthor=P.+Grandi&title=Domain-selective+targeting+of+BET+proteins+in+cancer+and+immunological+diseases&doi=10.1016%2Fj.cbpa.2020.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Domain-selective targeting of BET proteins in cancer and immunological diseases</span></div><div class="casAuthors">Petretich, Massimo; Demont, Emmanuel H.; Grandi, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">184-193</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review Cancer and inflammation are strongly interconnected processes.  Chronic inflammatory pathologies can be at the heart of tumor development; similarly, tumor-elicited inflammation is a consequence of many cancers.  The mechanistic interdependence between cancer and inflammatory pathologies points toward common protein effectors which represent potential shared targets for pharmacol. intervention.  Epigenetic mechanisms often drive resistance to cancer therapy and immunomodulatory strategies.  The bromodomain and extraterminal domain (BET) proteins are epigenetic adapters which play a major role in controlling cell proliferation and the prodn. of inflammatory mediators.  A plethora of small mols. aimed at inhibiting BET protein function to treat cancer and inflammatory diseases have populated academic and industry efforts in the last 10 years.  In this review, we will discuss recent pharmacol. approaches aimed at targeting a single or a subset of the eight bromodomains within the BET family which have the potential to tease apart clin. efficacy and safety signals of BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKVLw12H8j5LVg90H21EOLACvtfcHk0lgJXL5wRgRN3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFajsbjM&md5=82a99a0ebbcda749e42effa2820e232e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2020.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2020.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGrandi%26aufirst%3DP.%26atitle%3DDomain-selective%2520targeting%2520of%2520BET%2520proteins%2520in%2520cancer%2520and%2520immunological%2520diseases%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D57%26spage%3D184%26epage%3D193%26doi%3D10.1016%2Fj.cbpa.2020.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0lgJXL5wRgRN3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the Bromodomain and Extra Terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>– <span class="NLM_lpage">9092</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+R.+J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+Bromodomain+and+Extra+Terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0ljdgDxopn-WUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Target validation using chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnchembio.1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=23508172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=195-199&author=M.+E.+Bunnageauthor=E.+L.+P.+Cheklerauthor=L.+H.+Jones&title=Target+validation+using+chemical+probes&doi=10.1038%2Fnchembio.1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Target validation using chemical probes</span></div><div class="casAuthors">Bunnage, Mark E.; Chekler, Eugene L. Piatnitski; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">195-199</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review on the fully profiled chem. probes essential to support the unbiased interpretation of biol. expts. necessary for rigorous preclin. target validation.  We believe that by developing a 'chem. probe tool kit', and a framework for its use, chem. biol. can have a more central role in identifying targets of potential relevance to disease, avoiding many of the biases that complicate target validation as practiced currently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLwRTT4pXGxbVg90H21EOLACvtfcHk0ljdgDxopn-WUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFagsr4%253D&md5=af0c362882191de96a8e8698ff2c9ce8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1197%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTarget%2520validation%2520using%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26spage%3D195%26epage%3D199%26doi%3D10.1038%2Fnchembio.1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">Encoded combinatorial chemistry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">5381</span>– <span class="NLM_lpage">5383</span>, <span class="refDoi"> DOI: 10.1073/pnas.89.12.5381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.89.12.5381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1608946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=5381-5383&author=S.+Brennerauthor=R.+A.+Lerner&title=Encoded+combinatorial+chemistry&doi=10.1073%2Fpnas.89.12.5381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded combinatorial chemistry</span></div><div class="casAuthors">Brenner, Sydney; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5381-3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The diversity of chem. synthesis and the power of genetics are linked to provide a powerful, versatile method for drug screening.  A process of alternating parallel combinatorial synthesis is used to encode individual members of a large library of chems. with unique nucleotide sequences.  After the chem. entity is bound to a target, the genetic tag can be amplified by replication and utilized for enrichment of the bound mols. by serial hybridization to a subset of the library.  The library of the tagged chems. is termed an encoded combinatorial chem. library.  The nature of the chem. structure bound to the receptor is decoded by sequencing the nucleotide tag.  For example, with a library of peptides, each peptide was linked to a specifically designed oligodeoxynucleotide (genetic tag).  Each of these oligodeoxynucleotides had a signature (unique) sequence corresponding to the linked peptide and was flanked by sequences recognizable by designed DNA primers of the polymerase chain reaction (PCR).  After a desired member of this library of peptide-oligonucleotides was selected by screening (e.g., selected by binding to a target), this desired member was then identified by PCR amplification of the signature nucleotide sequence.  Detn. of the signature sequence from PCR amplified DNA gave the identity of the desired peptide.  Thus, this method can be used to identify a few members in a large chem. library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxFs-UqY_2JbVg90H21EOLACvtfcHk0ljdgDxopn-WUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D&md5=3f03fc103c039ce7985cd7b25142c7ed</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.12.5381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.12.5381%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DS.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DEncoded%2520combinatorial%2520chemistry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1992%26volume%3D89%26spage%3D5381%26epage%3D5383%26doi%3D10.1073%2Fpnas.89.12.5381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mannocci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimbacher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Bellis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melkko, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">High-throughput sequencing allows the identification of binding molecules isolated from DNA-encoded chemical libraries</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">17670</span>– <span class="NLM_lpage">17675</span>, <span class="refDoi"> DOI: 10.1073/pnas.0805130105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1073%2Fpnas.0805130105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=19001273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVCmtbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=17670-17675&author=L.+Mannocciauthor=Y.+Zhangauthor=J.+Scheuermannauthor=M.+Leimbacherauthor=G.+De+Bellisauthor=E.+Rizziauthor=C.+Dumelinauthor=S.+Melkkoauthor=D.+Neri&title=High-throughput+sequencing+allows+the+identification+of+binding+molecules+isolated+from+DNA-encoded+chemical+libraries&doi=10.1073%2Fpnas.0805130105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput sequencing allows the identification of binding molecules isolated from DNA-encoded chemical libraries</span></div><div class="casAuthors">Mannocci, Luca; Zhang, Yixin; Scheuermann, Jorg; Leimbacher, Markus; De Bellis, Gianluca; Rizzi, Ermanno; Dumelin, Christoph; Melkko, Samu; Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">17670-17675</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">DNA encoding facilitates the construction and screening of large chem. libraries.  Here, we describe general strategies for the stepwise coupling of coding DNA fragments to nascent org. mols. throughout individual reaction steps as well as the first implementation of high-throughput sequencing for the identification and relative quantification of the library members.  The methodol. was exemplified in the construction of a DNA-encoded chem. library contg. 4,000 compds. and in the discovery of binders to streptavidin, matrix metalloproteinase 3, and polyclonal human IgG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrsWWaqWxcEbVg90H21EOLACvtfcHk0li-qZ9hCCR1Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVCmtbbF&md5=d104229313e30f1df784a5d501a952c5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0805130105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0805130105%26sid%3Dliteratum%253Aachs%26aulast%3DMannocci%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DScheuermann%26aufirst%3DJ.%26aulast%3DLeimbacher%26aufirst%3DM.%26aulast%3DDe%2BBellis%26aufirst%3DG.%26aulast%3DRizzi%26aufirst%3DE.%26aulast%3DDumelin%26aufirst%3DC.%26aulast%3DMelkko%26aufirst%3DS.%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DHigh-throughput%2520sequencing%2520allows%2520the%2520identification%2520of%2520binding%2520molecules%2520isolated%2520from%2520DNA-encoded%2520chemical%2520libraries%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D17670%26epage%3D17675%26doi%3D10.1073%2Fpnas.0805130105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span>; <span class="NLM_string-name">Lu, X.</span>; <span class="NLM_string-name">Phelps, C.</span>; <span class="NLM_string-name">Evindar, G.</span></span> <span> </span><span class="NLM_article-title">DNA-encoded Library Technology (ELT)</span>. In  <i>Chemical and Biological Synthesis: Enabling Approaches for Understanding Biology</i><i>;</i> <span class="NLM_publisher-name">Royal Society of Chemistry</span>, <span class="NLM_year">2018</span>; pp  <span class="NLM_fpage">153</span>– <span class="NLM_lpage">183</span><span class="refDoi"> DOI: 10.1039/9781788012805-00153</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1039%2F9781788012805-00153" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=153-183&author=G.+Yao&author=X.+Lu&author=C.+Phelps&author=G.+Evindar&title=Chemical+and+Biological+Synthesis%3A+Enabling+Approaches+for+Understanding+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2F9781788012805-00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781788012805-00153%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DG.%26atitle%3DDNA-encoded%2520Library%2520Technology%2520%2528ELT%2529%26btitle%3DChemical%2520and%2520Biological%2520Synthesis%253A%2520Enabling%2520Approaches%2520for%2520Understanding%2520Biology%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2018%26spage%3D153%26epage%3D183%26doi%3D10.1039%2F9781788012805-00153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. wa</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dossang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddershaw, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klimaszewska, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haq, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakhria, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jigorel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olszewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronzoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowedder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüdiger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lomas, D. A.</span></span> <span> </span><span class="NLM_article-title">The development of highly potent and selective small molecule correctors of Z α1-antitrypsin misfolding</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">127973</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2021.127973</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.bmcl.2021.127973" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=33753261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXotFahurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=127973&author=J.+Liddleauthor=A.+C.+Pearceauthor=C.+Arico-Muendelauthor=S.+Belyanskayaauthor=A.+Brewsterauthor=M.+Brownauthor=C.+wa+Chungauthor=A.+Denisauthor=N.+Dodicauthor=A.+Dossangauthor=P.+Eddershawauthor=D.+Klimaszewskaauthor=I.+Haqauthor=D.+S.+Holmesauthor=A.+Jaggerauthor=T.+Jakhriaauthor=E.+Jigorelauthor=K.+Lindauthor=J.+Messerauthor=M.+Neuauthor=A.+Olszewskiauthor=R.+Ronzoniauthor=J.+Rowedderauthor=M.+R%C3%BCdigerauthor=S.+Skinnerauthor=K.+J.+Smithauthor=L.+Trottetauthor=I.+Uingsauthor=Z.+Zhuauthor=J.+A.+Irvingauthor=D.+A.+Lomas&title=The+development+of+highly+potent+and+selective+small+molecule+correctors+of+Z+%CE%B11-antitrypsin+misfolding&doi=10.1016%2Fj.bmcl.2021.127973"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The development of highly potent and selective small molecule correctors of Z α1-antitrypsin misfolding</span></div><div class="casAuthors">Liddle, John; Pearce, Andrew C.; Arico-Muendel, Christopher; Belyanskaya, Svetlana; Brewster, Andrew; Brown, Murray; Chung, Chun-wa; Denis, Alexis; Dodic, Nerina; Dossang, Anthony; Eddershaw, Peter; Klimaszewska, Diana; Haq, Imran; Holmes, Duncan S.; Jagger, Alistair; Jakhria, Toral; Jigorel, Emilie; Lind, Ken; Messer, Jeff; Neu, Margaret; Olszewski, Allison; Ronzoni, Riccardo; Rowedder, James; Rudiger, Martin; Skinner, Steve; Smith, Kathrine J.; Trottet, Lionel; Uings, Iain; Zhu, Zhengrong; Irving, James A.; Lomas, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127973</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">α1-antitrypsin deficiency is characterised by the misfolding and intracellular polymn. of mutant α1-antitrypsin protein within the endoplasmic reticulum (ER) of hepatocytes.  Small mols. that bind and stabilize Z α1-antitrypsin were identified via a DNA-encoded library screen.  A subsequent structure based optimization led to a series of highly potent, selective and cellular active α1-antitrypsin correctors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopH8iP7w6_wrVg90H21EOLACvtfcHk0li4E3uBckEntA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXotFahurw%253D&md5=3f5849489c9c53f681b58b7712123dd5</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2021.127973&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2021.127973%26sid%3Dliteratum%253Aachs%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DPearce%26aufirst%3DA.%2BC.%26aulast%3DArico-Muendel%26aufirst%3DC.%26aulast%3DBelyanskaya%26aufirst%3DS.%26aulast%3DBrewster%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2Bwa%26aulast%3DDenis%26aufirst%3DA.%26aulast%3DDodic%26aufirst%3DN.%26aulast%3DDossang%26aufirst%3DA.%26aulast%3DEddershaw%26aufirst%3DP.%26aulast%3DKlimaszewska%26aufirst%3DD.%26aulast%3DHaq%26aufirst%3DI.%26aulast%3DHolmes%26aufirst%3DD.%2BS.%26aulast%3DJagger%26aufirst%3DA.%26aulast%3DJakhria%26aufirst%3DT.%26aulast%3DJigorel%26aufirst%3DE.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMesser%26aufirst%3DJ.%26aulast%3DNeu%26aufirst%3DM.%26aulast%3DOlszewski%26aufirst%3DA.%26aulast%3DRonzoni%26aufirst%3DR.%26aulast%3DRowedder%26aufirst%3DJ.%26aulast%3DR%25C3%25BCdiger%26aufirst%3DM.%26aulast%3DSkinner%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DK.%2BJ.%26aulast%3DTrottet%26aufirst%3DL.%26aulast%3DUings%26aufirst%3DI.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DIrving%26aufirst%3DJ.%2BA.%26aulast%3DLomas%26aufirst%3DD.%2BA.%26atitle%3DThe%2520development%2520of%2520highly%2520potent%2520and%2520selective%2520small%2520molecule%2520correctors%2520of%2520Z%2520%25CE%25B11-antitrypsin%2520misfolding%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D41%26spage%3D127973%26doi%3D10.1016%2Fj.bmcl.2021.127973" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bit, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Gall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poole, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, K. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vimal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Bromodomain and Extraterminal inhibitor with a low predicted human dose through synergistic use of encoded library technology and fragment screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">746</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01670</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01670" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFagtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=714-746&author=C.+R.+Wellawayauthor=D.+Amansauthor=P.+Bamboroughauthor=H.+Barnettauthor=R.+A.+Bitauthor=J.+A.+Brownauthor=N.+R.+Carlsonauthor=C.+W.+Chungauthor=A.+W.+J.+Cooperauthor=P.+D.+Craggsauthor=R.+P.+Davisauthor=T.+W.+Deanauthor=J.+P.+Evansauthor=L.+Gordonauthor=I.+L.+Haradaauthor=D.+J.+Hirstauthor=P.+G.+Humphreysauthor=K.+L.+Jonesauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+Lugoauthor=M.+Mahmoodauthor=S.+McClearyauthor=P.+Medeirosauthor=D.+J.+Mitchellauthor=M.+O%E2%80%99Sullivanauthor=A.+Le+Gallauthor=V.+K.+Patelauthor=C.+Pattenauthor=D.+L.+Pooleauthor=R.+R.+Shahauthor=J.+E.+Smithauthor=K.+A.+J.+Staffordauthor=P.+J.+Thomasauthor=M.+Vimalauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=N.+Wellawayauthor=G.+Yaoauthor=R.+K.+Prinjha&title=Discovery+of+a+Bromodomain+and+Extraterminal+inhibitor+with+a+low+predicted+human+dose+through+synergistic+use+of+encoded+library+technology+and+fragment+screening&doi=10.1021%2Facs.jmedchem.9b01670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Bromodomain and Extraterminal Inhibitor with a Low Predicted Human Dose through Synergistic Use of Encoded Library Technology and Fragment Screening</span></div><div class="casAuthors">Wellaway, Christopher R.; Amans, Dominique; Bamborough, Paul; Barnett, Heather; Bit, Rino A.; Brown, Jack A.; Carlson, Neil R.; Chung, Chun-wa; Cooper, Anthony W. J.; Craggs, Peter D.; Davis, Robert P.; Dean, Tony W.; Evans, John P.; Gordon, Laurie; Harada, Isobel L.; Hirst, David J.; Humphreys, Philip G.; Jones, Katherine L.; Lewis, Antonia J.; Lindon, Matthew J.; Lugo, Dave; Mahmood, Mahnoor; McCleary, Scott; Medeiros, Patricia; Mitchell, Darren J.; O'Sullivan, Michael; Le Gall, Armelle; Patel, Vipulkumar K.; Patten, Chris; Poole, Darren L.; Shah, Rishi R.; Smith, Jane E.; Stafford, Kayleigh A. J.; Thomas, Pamela J.; Vimal, Mythily; Wall, Ian D.; Watson, Robert J.; Wellaway, Natalie; Yao, Gang; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">714-746</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) family of bromodomain-contg. proteins are important regulators of the epigenome through their ability to recognize N-acetyl lysine (KAc) post-translational modifications on histone tails.  These interactions have been implicated in various disease states and, consequently, disruption of BET-KAc binding has emerged as an attractive therapeutic strategy with a no. of small mol. inhibitors now under investigation in the clinic.  However, until the utility of these advanced candidates is fully assessed by these trials, there remains scope for the discovery of inhibitors from new chemotypes with alternative physicochem., pharmacokinetic, and pharmacodynamic profiles.  Herein, we describe the discovery of a candidate-quality dimethylpyridone benzimidazole compd. which originated from the hybridization of a dimethylphenol benzimidazole series, identified using encoded library technol., with an N-Me pyridone series identified through fragment screening.  Optimization via structure- and property-based design led to I-BET469, which possesses favorable oral pharmacokinetic properties, displays activity in vivo, and is projected to have a low human efficacious dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJtEmfPcCFtbVg90H21EOLACvtfcHk0li4E3uBckEntA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFagtw%253D%253D&md5=826ebe53a3091893c9e6c6c4830ac388</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01670%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DAmans%26aufirst%3DD.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DBit%26aufirst%3DR.%2BA.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCooper%26aufirst%3DA.%2BW.%2BJ.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDavis%26aufirst%3DR.%2BP.%26aulast%3DDean%26aufirst%3DT.%2BW.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHarada%26aufirst%3DI.%2BL.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJones%26aufirst%3DK.%2BL.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DMahmood%26aufirst%3DM.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DM.%26aulast%3DLe%2BGall%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DV.%2BK.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DPoole%26aufirst%3DD.%2BL.%26aulast%3DShah%26aufirst%3DR.%2BR.%26aulast%3DSmith%26aufirst%3DJ.%2BE.%26aulast%3DStafford%26aufirst%3DK.%2BA.%2BJ.%26aulast%3DThomas%26aufirst%3DP.%2BJ.%26aulast%3DVimal%26aufirst%3DM.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWellaway%26aufirst%3DN.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DDiscovery%2520of%2520a%2520Bromodomain%2520and%2520Extraterminal%2520inhibitor%2520with%2520a%2520low%2520predicted%2520human%2520dose%2520through%2520synergistic%2520use%2520of%2520encoded%2520library%2520technology%2520and%2520fragment%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D714%26epage%3D746%26doi%3D10.1021%2Facs.jmedchem.9b01670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kazmierski, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De La Rosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favre, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Washio, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mebrahtu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basilla, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evindar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olszewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabhu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samano, V.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded library technology-based discovery, lead optimization, and prodrug strategy toward structurally unique Indoleamine 2,3-Dioxygenase-1 (IDO1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3552</span>– <span class="NLM_lpage">3562</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01799</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01799" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVOms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3552-3562&author=W.+M.+Kazmierskiauthor=B.+Xiaauthor=J.+Millerauthor=M.+De+La+Rosaauthor=D.+Favreauthor=R.+M.+Dunhamauthor=Y.+Washioauthor=Z.+Zhuauthor=F.+Wangauthor=M.+Mebrahtuauthor=H.+Dengauthor=J.+Basillaauthor=L.+Wangauthor=G.+Evindarauthor=L.+Fanauthor=A.+Olszewskiauthor=N.+Prabhuauthor=C.+Davieauthor=J.+A.+Messerauthor=V.+Samano&title=DNA-Encoded+library+technology-based+discovery%2C+lead+optimization%2C+and+prodrug+strategy+toward+structurally+unique+Indoleamine+2%2C3-Dioxygenase-1+%28IDO1%29+inhibitors&doi=10.1021%2Facs.jmedchem.9b01799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-Encoded Library Technology-Based Discovery, Lead Optimization, and Prodrug Strategy toward Structurally Unique Indoleamine 2,3-Dioxygenase-1 (IDO1) Inhibitors</span></div><div class="casAuthors">Kazmierski, Wieslaw M.; Xia, Bing; Miller, John; De la Rosa, Martha; Favre, David; Dunham, Richard M.; Washio, Yoshiaki; Zhu, Zhengrong; Wang, Feng; Mebrahtu, Makda; Deng, Hongfeng; Basilla, Jonathan; Wang, Liping; Evindar, Ghotas; Fan, Lijun; Olszewski, Alison; Prabhu, Ninad; Davie, Christopher; Messer, Jeffrey A.; Samano, Vicente</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3552-3562</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the discovery of a novel indoleamine 2,3-dioxygenase-1 (IDO1) inhibitor class through the affinity selection of a previously unreported indole-based DNA-encoded library (DEL).  The DEL exemplar, spiro-chromane 1, had moderate IDO1 potency but high in vivo clearance.  Series optimization quickly afforded a potent, low in vivo clearance lead 11.  Although amorphous 11 was highly bio-available, cryst. 11 was poorly sol. and suffered disappointingly low bio-availability because of soly.-limited absorption.  A prodrug approach was deployed and proved effective in discovering the highly bio-available phosphonooxymethyl 31, which rapidly converted to 11 in vivo.  Obtaining cryst. 31 proved problematic, however; thus salt screening was performed in an attempt to circumvent this obstacle and successfully delivered greatly sol. and bio-available cryst. tris-salt 32.  IDO1 inhibitor 32 is characterized by a low calcd. human dose, best-in-class potential, and an unusual inhibition mode by binding the IDO1 heme-free (apo) form.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhSTj0UyMSxbVg90H21EOLACvtfcHk0lhmNOb0JMLiSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVOms7k%253D&md5=8cddeffb12cefd9b3e1c2e0641ad7226</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01799%26sid%3Dliteratum%253Aachs%26aulast%3DKazmierski%26aufirst%3DW.%2BM.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DDe%2BLa%2BRosa%26aufirst%3DM.%26aulast%3DFavre%26aufirst%3DD.%26aulast%3DDunham%26aufirst%3DR.%2BM.%26aulast%3DWashio%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DMebrahtu%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DBasilla%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DEvindar%26aufirst%3DG.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DOlszewski%26aufirst%3DA.%26aulast%3DPrabhu%26aufirst%3DN.%26aulast%3DDavie%26aufirst%3DC.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DSamano%26aufirst%3DV.%26atitle%3DDNA-Encoded%2520library%2520technology-based%2520discovery%252C%2520lead%2520optimization%252C%2520and%2520prodrug%2520strategy%2520toward%2520structurally%2520unique%2520Indoleamine%25202%252C3-Dioxygenase-1%2520%2528IDO1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3552%26epage%3D3562%26doi%3D10.1021%2Facs.jmedchem.9b01799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhaus, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkämper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class Receptor Interacting Protein 1 (RIP1) kinase specific clinical candidate (GSK2982772) for the treatment of inflammatory diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1247-1261&author=P.+A.+Harrisauthor=S.+B.+Bergerauthor=J.+U.+Jeongauthor=R.+Nagillaauthor=D.+Bandyopadhyayauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=P.+M.+Eidamauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=J.+Kangauthor=V.+Kasparcovaauthor=B.+W.+Kingauthor=R.+Lehrauthor=Y.+Lanauthor=L.+K.+Leisterauthor=J.+D.+Lichauthor=T.+T.+MacDonaldauthor=N.+A.+Millerauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+Rahmanauthor=M.+A.+Reillyauthor=A.+R.+Rendinaauthor=E.+J.+Riveraauthor=M.+C.+Schaefferauthor=C.+A.+Sehonauthor=R.+R.+Singhausauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=A.+Vossenk%C3%A4mperauthor=P.+Wardauthor=D.+D.+Wisnoskiauthor=D.+Zhangauthor=R.+W.+Marquisauthor=P.+J.+Goughauthor=J.+Bertin&title=Discovery+of+a+first-in-class+Receptor+Interacting+Protein+1+%28RIP1%29+kinase+specific+clinical+candidate+%28GSK2982772%29+for+the+treatment+of+inflammatory+diseases&doi=10.1021%2Facs.jmedchem.6b01751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span></div><div class="casAuthors">Harris, Philip A.; Berger, Scott B.; Jeong, Jae U.; Nagilla, Rakesh; Bandyopadhyay, Deepak; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Eidam, Patrick M.; Finger, Joshua N.; Hoffman, Sandra J.; Kang, James; Kasparcova, Viera; King, Bryan W.; Lehr, Ruth; Lan, Yunfeng; Leister, Lara K.; Lich, John D.; MacDonald, Thomas T.; Miller, Nathan A.; Ouellette, Michael T.; Pao, Christina S.; Rahman, Attiq; Reilly, Michael A.; Rendina, Alan R.; Rivera, Elizabeth J.; Schaeffer, Michelle C.; Sehon, Clark A.; Singhaus, Robert R.; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Vossenkamper, Anna; Ward, Paris; Wisnoski, David D.; Zhang, Daohua; Marquis, Robert W.; Gough, Peter J.; Bertin, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RIP1 regulates necroptosis and inflammation and may play an important role in contributing to a variety of human pathologies, including immune-mediated inflammatory diseases.  Small-mol. inhibitors of RIP1 kinase that are suitable for advancement into the clinic have yet to be described.  Herein, the authors report the lead optimization of a benzoxazepinone hit from a DNA-encoded library and the discovery and profile of clin. candidate GSK2982772 (compd. I), currently in phase 2a clin. studies for psoriasis, rheumatoid arthritis, and ulcerative colitis.  Compd. I potently binds to RIP1 with exquisite kinase specificity and has excellent activity in blocking many TNF-dependent cellular responses.  Highlighting its potential as a novel anti-inflammatory agent, the inhibitor was also able to reduce spontaneous prodn. of cytokines from human ulcerative colitis explants.  The highly favorable physicochem. and ADMET properties of I, combined with high potency, led to a predicted low oral dose in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZGRDTQv-vs7Vg90H21EOLACvtfcHk0lj7aqvW9MGbQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D&md5=ee0c3c60bb1351dff696490ac9edb1cc</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01751%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DEidam%26aufirst%3DP.%2BM.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLan%26aufirst%3DY.%26aulast%3DLeister%26aufirst%3DL.%2BK.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DMiller%26aufirst%3DN.%2BA.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRahman%26aufirst%3DA.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DRivera%26aufirst%3DE.%2BJ.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSehon%26aufirst%3DC.%2BA.%26aulast%3DSinghaus%26aufirst%3DR.%2BR.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DVossenk%25C3%25A4mper%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520Receptor%2520Interacting%2520Protein%25201%2520%2528RIP1%2529%2520kinase%2520specific%2520clinical%2520candidate%2520%2528GSK2982772%2529%2520for%2520the%2520treatment%2520of%2520inflammatory%2520diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Facs.jmedchem.6b01751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coote, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicker, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinod, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüdiger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slade, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of PAD4 activity is sufficient to disrupt mouse and human NET formation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">191</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnchembio.1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=25622091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVehsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=189-191&author=H.+D.+Lewisauthor=J.+Liddleauthor=J.+E.+Cooteauthor=S.+J.+Atkinsonauthor=M.+D.+Barkerauthor=B.+D.+Baxauthor=K.+L.+Bickerauthor=R.+P.+Binghamauthor=M.+Campbellauthor=Y.+H.+Chenauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=R.+P.+Davisauthor=D.+Eberhardauthor=G.+Jobertyauthor=K.+E.+Lindauthor=K.+Lockeauthor=C.+Mallerauthor=K.+Martinodauthor=C.+Pattenauthor=O.+Polyakovaauthor=C.+E.+Riseauthor=M.+R%C3%BCdigerauthor=R.+J.+Sheppardauthor=D.+J.+Sladeauthor=P.+Thomasauthor=J.+Thorpeauthor=G.+Yaoauthor=G.+Drewesauthor=D.+D.+Wagnerauthor=P.+R.+Thompsonauthor=R.+K.+Prinjhaauthor=D.+M.+Wilson&title=Inhibition+of+PAD4+activity+is+sufficient+to+disrupt+mouse+and+human+NET+formation&doi=10.1038%2Fnchembio.1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of PAD4 activity is sufficient to disrupt mouse and human NET formation</span></div><div class="casAuthors">Lewis, Huw D.; Liddle, John; Coote, Jim E.; Atkinson, Stephen J.; Barker, Michael D.; Bax, Benjamin D.; Bicker, Kevin L.; Bingham, Ryan P.; Campbell, Matthew; Chen, Yu Hua; Chung, Chun-wa; Craggs, Peter D.; Davis, Rob P.; Eberhard, Dirk; Joberty, Gerard; Lind, Kenneth E.; Locke, Kelly; Maller, Claire; Martinod, Kimberly; Patten, Chris; Polyakova, Oxana; Rise, Cecil E.; Rudiger, Martin; Sheppard, Robert J.; Slade, Daniel J.; Thomas, Pamela; Thorpe, Jim; Yao, Gang; Drewes, Gerard; Wagner, Denisa D.; Thompson, Paul R.; Prinjha, Rab K.; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">189-191</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PAD4 has been strongly implicated in the pathogenesis of autoimmune, cardiovascular and oncol. diseases through clin. genetics and gene disruption in mice.  New selective PAD4 inhibitors binding a calcium-deficient form of the PAD4 enzyme have validated the crit. enzymic role of human and mouse PAD4 in both histone citrullination and neutrophil extracellular trap formation for, to our knowledge, the first time.  The therapeutic potential of PAD4 inhibitors can now be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLRDGrLt-nS7Vg90H21EOLACvtfcHk0lidGtSe7tRe0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVehsbo%253D&md5=24efcb7429535e5b2ec6e5cd48f0dd59</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1735%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DH.%2BD.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DCoote%26aufirst%3DJ.%2BE.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBax%26aufirst%3DB.%2BD.%26aulast%3DBicker%26aufirst%3DK.%2BL.%26aulast%3DBingham%26aufirst%3DR.%2BP.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%2BH.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DDavis%26aufirst%3DR.%2BP.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DLocke%26aufirst%3DK.%26aulast%3DMaller%26aufirst%3DC.%26aulast%3DMartinod%26aufirst%3DK.%26aulast%3DPatten%26aufirst%3DC.%26aulast%3DPolyakova%26aufirst%3DO.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DR%25C3%25BCdiger%26aufirst%3DM.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSlade%26aufirst%3DD.%2BJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DThorpe%26aufirst%3DJ.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DWagner%26aufirst%3DD.%2BD.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DInhibition%2520of%2520PAD4%2520activity%2520is%2520sufficient%2520to%2520disrupt%2520mouse%2520and%2520human%2520NET%2520formation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D189%26epage%3D191%26doi%3D10.1038%2Fnchembio.1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentile, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merlo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S.</span></span> <span> </span><span class="NLM_article-title">5-Aryl-4-carboxamide-1,3-oxazoles: Potent and selective GSK-3 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1989</span>– <span class="NLM_lpage">1994</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.01.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.bmcl.2012.01.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=22310227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVGnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=1989-1994&author=G.+Gentileauthor=G.+Merloauthor=A.+Pozzanauthor=G.+Bernasconiauthor=B.+Baxauthor=P.+Bamboroughauthor=A.+Bridgesauthor=P.+Carterauthor=M.+Neuauthor=G.+Yaoauthor=C.+Broughauthor=G.+Cutlerauthor=A.+Coffinauthor=S.+Belyanskaya&title=5-Aryl-4-carboxamide-1%2C3-oxazoles%3A+Potent+and+selective+GSK-3+inhibitors&doi=10.1016%2Fj.bmcl.2012.01.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">5-Aryl-4-carboxamide-1,3-oxazoles: Potent and selective GSK-3 inhibitors</span></div><div class="casAuthors">Gentile, Gabriella; Merlo, Giancarlo; Pozzan, Alfonso; Bernasconi, Giovanni; Bax, Benjamin; Bamborough, Paul; Bridges, Angela; Carter, Paul; Neu, Margarete; Yao, Gang; Brough, Caroline; Cutler, Geoffrey; Coffin, Aaron; Belyanskaya, Svetlana</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1989-1994</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">5-Aryl-4-carboxamide-1,3-oxazoles are a novel, potent and selective series of GSK-3 inhibitors.  The optimization of the series to yield compds. with cell activity and brain permeability is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPQvQwnrcpa7Vg90H21EOLACvtfcHk0lidGtSe7tRe0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVGnsbo%253D&md5=ec8383b7bceebdec334adef38b8a88b3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.01.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.01.034%26sid%3Dliteratum%253Aachs%26aulast%3DGentile%26aufirst%3DG.%26aulast%3DMerlo%26aufirst%3DG.%26aulast%3DPozzan%26aufirst%3DA.%26aulast%3DBernasconi%26aufirst%3DG.%26aulast%3DBax%26aufirst%3DB.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBridges%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DNeu%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DBrough%26aufirst%3DC.%26aulast%3DCutler%26aufirst%3DG.%26aulast%3DCoffin%26aufirst%3DA.%26aulast%3DBelyanskaya%26aufirst%3DS.%26atitle%3D5-Aryl-4-carboxamide-1%252C3-oxazoles%253A%2520Potent%2520and%2520selective%2520GSK-3%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D1989%26epage%3D1994%26doi%3D10.1016%2Fj.bmcl.2012.01.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veerman, J. J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruseker, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heijne, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bruggen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hekking, K. F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRiner, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulvihill, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Rijnsbergen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzetti, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiss, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2,4–1 H-imidazole carboxamides as potent and selective TAK1 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">555</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00547</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00547" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=555&author=J.+J.+N.+Veermanauthor=Y.+B.+Brusekerauthor=E.+Damenauthor=E.+H.+Heijneauthor=W.+van+Bruggenauthor=K.+F.+W.+Hekkingauthor=R.+Winkelauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=J.+Liuauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+J.+McRinerauthor=M.+J.+Mulvihillauthor=P.+van+Rijnsbergenauthor=B.+Zechauthor=L.+M.+Renzettiauthor=L.+Babissauthor=G.+Muller&title=Discovery+of+2%2C4%E2%80%931+H-imidazole+carboxamides+as+potent+and+selective+TAK1+inhibitors&doi=10.1021%2Facsmedchemlett.0c00547"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2,4-1H-Imidazole Carboxamides as Potent and Selective TAK1 Inhibitors</span></div><div class="casAuthors">Veerman, Johan J. N.; Bruseker, Yorik B.; Damen, Eddy; Heijne, Erik H.; van Bruggen, Wendy; Hekking, Koen F. W.; Winkel, Rob; Hupp, Christopher D.; Keefe, Anthony D.; Liu, Julie; Thomson, Heather A.; Zhang, Ying; Cuozzo, John W.; McRiner, Andrew J.; Mulvihill, Mark J.; van Rijnsbergen, Peter; Zech, Birgit; Renzetti, Louis M.; Babiss, Lee; Mueller, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">555-562</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein we report the discovery of 2,4-1H-imidazole carboxamides as novel, biochem. potent, and kinome selective inhibitors of transforming growth factor β-activated kinase 1 (TAK1).  The target was subjected to a DNA-encoded chem. library (DECL) screen.  After hit anal. a cluster of compds. was identified, which was based on a central pyrrole-2,4-1H-dicarboxamide scaffold, showing remarkable kinome selectivity.  A scaffold-hop to the corresponding imidazole resulted in increased biochem. potency.  Next, X-ray crystallog. revealed a distinct binding mode compared to other TAK1 inhibitors.  A benzylamide was found in a perpendicular orientation with respect to the core hinge-binding imidazole.  Addnl., an unusual amide flip was obsd. in the kinase hinge region.  Using structure-based drug design (SBDD), key substitutions at the pyrrolidine amide and the glycine resulted in a significant increase in biochem. potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsfdQbRX4C1LVg90H21EOLACvtfcHk0ljvifxS2cOaUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltl2lt7o%253D&md5=26c25af441806342c2048ce49c6be192</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00547%26sid%3Dliteratum%253Aachs%26aulast%3DVeerman%26aufirst%3DJ.%2BJ.%2BN.%26aulast%3DBruseker%26aufirst%3DY.%2BB.%26aulast%3DDamen%26aufirst%3DE.%26aulast%3DHeijne%26aufirst%3DE.%2BH.%26aulast%3Dvan%2BBruggen%26aufirst%3DW.%26aulast%3DHekking%26aufirst%3DK.%2BF.%2BW.%26aulast%3DWinkel%26aufirst%3DR.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DMcRiner%26aufirst%3DA.%2BJ.%26aulast%3DMulvihill%26aufirst%3DM.%2BJ.%26aulast%3Dvan%2BRijnsbergen%26aufirst%3DP.%26aulast%3DZech%26aufirst%3DB.%26aulast%3DRenzetti%26aufirst%3DL.%2BM.%26aulast%3DBabiss%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DG.%26atitle%3DDiscovery%2520of%25202%252C4%25E2%2580%25931%2520H-imidazole%2520carboxamides%2520as%2520potent%2520and%2520selective%2520TAK1%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D555%26doi%3D10.1021%2Facsmedchemlett.0c00547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foley, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burchett, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapinos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratnayake, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peakman, M. C.</span></span> <span> </span><span class="NLM_article-title">Selecting approaches for hit identification and increasing options by building the efficient discovery of actionable chemical matter from DNA-encoded libraries</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1177/2472555220979589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1177%2F2472555220979589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=33412987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BB3MXit1els7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2021&pages=263-280&author=T.+L.+Foleyauthor=W.+Burchettauthor=Q.+Chenauthor=M.+E.+Flanaganauthor=B.+Kapinosauthor=X.+Liauthor=J.+I.+Montgomeryauthor=A.+S.+Ratnayakeauthor=H.+Zhuauthor=M.+C.+Peakman&title=Selecting+approaches+for+hit+identification+and+increasing+options+by+building+the+efficient+discovery+of+actionable+chemical+matter+from+DNA-encoded+libraries&doi=10.1177%2F2472555220979589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting Approaches for Hit Identification and Increasing Options by Building the Efficient Discovery of Actionable Chemical Matter from DNA-Encoded Libraries</span></div><div class="casAuthors">Foley, Timothy L.; Burchett, Woodrow; Chen, Qiuxia; Flanagan, Mark E.; Kapinos, Brendon; Li, Xianyang; Montgomery, Justin I.; Ratnayake, Anokha S.; Zhu, Hongyao; Peakman, Marie-Claire</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-280</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Over the past 20 years, the toolbox for discovering small-mol. therapeutic starting points has expanded considerably.  Pharmaceutical researchers can now choose from technologies that, in addn. to traditional high-throughput knowledge-based and diversity screening, now include the screening of fragment and fragment-like libraries, affinity selection mass spectrometry, and selection against DNA-encoded libraries (DELs).  Each of these techniques has its own unique combination of advantages and limitations that makes them more, or less, suitable for different target classes or discovery objectives, such as desired mechanism of action.  Layered on top of this are the constraints of the drug-hunters themselves, including budgets, timelines, and available platform capacity; each of these can play a part in dictating the hit identification strategy for a discovery program.  In this article, we discuss some of the factors that we use to govern our building of a hit identification roadmap for a program and describe the increasing role that DELs are playing in our discovery strategy.  Furthermore, we share our learning during our initial exploration of DEL and highlight the approaches we have evolved to maximize the value returned from DEL selections.  Topics addressed include the optimization of library design and prodn., reagent validation, data anal., and hit confirmation.  We describe how our thinking in these areas has led us to build a DEL platform that has begun to deliver tractable matter to our global discovery portfolio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqqCnUXlyxTLVg90H21EOLACvtfcHk0ljvifxS2cOaUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXit1els7Y%253D&md5=c12b7655c625e163710f34cc7caf808b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1177%2F2472555220979589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555220979589%26sid%3Dliteratum%253Aachs%26aulast%3DFoley%26aufirst%3DT.%2BL.%26aulast%3DBurchett%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DFlanagan%26aufirst%3DM.%2BE.%26aulast%3DKapinos%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMontgomery%26aufirst%3DJ.%2BI.%26aulast%3DRatnayake%26aufirst%3DA.%2BS.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DPeakman%26aufirst%3DM.%2BC.%26atitle%3DSelecting%2520approaches%2520for%2520hit%2520identification%2520and%2520increasing%2520options%2520by%2520building%2520the%2520efficient%2520discovery%2520of%2520actionable%2520chemical%2520matter%2520from%2520DNA-encoded%2520libraries%26jtitle%3DSLAS%2520Discovery%26date%3D2021%26volume%3D26%26spage%3D263%26epage%3D280%26doi%3D10.1177%2F2472555220979589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">MacHutta, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besra, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barros-Aguirre, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhry, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fosbenner, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingraham, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelievre, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza-Losana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCloskey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Keefe, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Keeffe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelps, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafferty, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scavello, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiginga, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundersingh, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweitzer, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczuk, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toh, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evindar, G.</span></span> <span> </span><span class="NLM_article-title">Prioritizing multiple therapeutic targets in parallel using automated DNA-encoded library screening</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1038/ncomms16081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fncomms16081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=28232747" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-11&author=C.+A.+MacHuttaauthor=C.+S.+Kollmannauthor=K.+E.+Lindauthor=X.+Baiauthor=P.+F.+Chanauthor=J.+Huangauthor=L.+Ballellauthor=S.+Belyanskayaauthor=G.+S.+Besraauthor=D.+Barros-Aguirreauthor=R.+H.+Batesauthor=P.+A.+Centrellaauthor=S.+S.+Changauthor=J.+Chaiauthor=A.+E.+Choudhryauthor=A.+Coffinauthor=C.+P.+Davieauthor=H.+Dengauthor=J.+Dengauthor=Y.+Dingauthor=J.+W.+Dodsonauthor=D.+T.+Fosbennerauthor=E.+N.+Gaoauthor=T.+L.+Grahamauthor=T.+L.+Graybillauthor=K.+Ingrahamauthor=W.+P.+Johnsonauthor=B.+W.+Kingauthor=C.+R.+Kwiatkowskiauthor=J.+Lelievreauthor=Y.+Liauthor=X.+Liuauthor=Q.+Luauthor=R.+Lehrauthor=A.+Mendoza-Losanaauthor=J.+Martinauthor=L.+McCloskeyauthor=P.+McCormickauthor=H.+P.+O%27Keefeauthor=T.+O%27Keeffeauthor=C.+Paoauthor=C.+B.+Phelpsauthor=H.+Qiauthor=K.+Raffertyauthor=G.+S.+Scavelloauthor=M.+S.+Steigingaauthor=F.+S.+Sundersinghauthor=S.+M.+Sweitzerauthor=L.+M.+Szewczukauthor=A.+Taylorauthor=M.+F.+Tohauthor=J.+Wangauthor=M.+Wangauthor=D.+J.+Wilkinsauthor=B.+Xiaauthor=G.+Yaoauthor=J.+Zhangauthor=J.+Zhouauthor=C.+P.+Donahueauthor=J.+A.+Messerauthor=D.+Holmesauthor=C.+C.+Arico-Muendelauthor=A.+J.+Popeauthor=J.+W.+Grossauthor=G.+Evindar&title=Prioritizing+multiple+therapeutic+targets+in+parallel+using+automated+DNA-encoded+library+screening&doi=10.1038%2Fncomms16081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fncomms16081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms16081%26sid%3Dliteratum%253Aachs%26aulast%3DMacHutta%26aufirst%3DC.%2BA.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DP.%2BF.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DBallell%26aufirst%3DL.%26aulast%3DBelyanskaya%26aufirst%3DS.%26aulast%3DBesra%26aufirst%3DG.%2BS.%26aulast%3DBarros-Aguirre%26aufirst%3DD.%26aulast%3DBates%26aufirst%3DR.%2BH.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChang%26aufirst%3DS.%2BS.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DChoudhry%26aufirst%3DA.%2BE.%26aulast%3DCoffin%26aufirst%3DA.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DDodson%26aufirst%3DJ.%2BW.%26aulast%3DFosbenner%26aufirst%3DD.%2BT.%26aulast%3DGao%26aufirst%3DE.%2BN.%26aulast%3DGraham%26aufirst%3DT.%2BL.%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DIngraham%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DW.%2BP.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DKwiatkowski%26aufirst%3DC.%2BR.%26aulast%3DLelievre%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DQ.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DMendoza-Losana%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DJ.%26aulast%3DMcCloskey%26aufirst%3DL.%26aulast%3DMcCormick%26aufirst%3DP.%26aulast%3DO%2527Keefe%26aufirst%3DH.%2BP.%26aulast%3DO%2527Keeffe%26aufirst%3DT.%26aulast%3DPao%26aufirst%3DC.%26aulast%3DPhelps%26aufirst%3DC.%2BB.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DRafferty%26aufirst%3DK.%26aulast%3DScavello%26aufirst%3DG.%2BS.%26aulast%3DSteiginga%26aufirst%3DM.%2BS.%26aulast%3DSundersingh%26aufirst%3DF.%2BS.%26aulast%3DSweitzer%26aufirst%3DS.%2BM.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DToh%26aufirst%3DM.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWilkins%26aufirst%3DD.%2BJ.%26aulast%3DXia%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DDonahue%26aufirst%3DC.%2BP.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DHolmes%26aufirst%3DD.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26aulast%3DGross%26aufirst%3DJ.%2BW.%26aulast%3DEvindar%26aufirst%3DG.%26atitle%3DPrioritizing%2520multiple%2520therapeutic%2520targets%2520in%2520parallel%2520using%2520automated%2520DNA-encoded%2520library%2520screening%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1%26epage%3D11%26doi%3D10.1038%2Fncomms16081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belyanskaya, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjamin, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creaser, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davie, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzen, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gefter, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavarana, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollmann, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lind, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medeiros, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliff, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satz, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vloten, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, B. A.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and selection of DNA-encoded small-molecule libraries</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1038/nchembio.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1038%2Fnchembio.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=19648931" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=647-654&author=M.+A.+Clarkauthor=R.+A.+Acharyaauthor=C.+C.+Arico-Muendelauthor=S.+L.+Belyanskayaauthor=D.+R.+Benjaminauthor=N.+R.+Carlsonauthor=P.+A.+Centrellaauthor=C.+H.+Chiuauthor=S.+P.+Creaserauthor=J.+W.+Cuozzoauthor=C.+P.+Davieauthor=Y.+Dingauthor=G.+J.+Franklinauthor=K.+D.+Franzenauthor=M.+L.+Gefterauthor=S.+P.+Haleauthor=N.+J.+V.+Hansenauthor=D.+I.+Israelauthor=J.+Jiangauthor=M.+J.+Kavaranaauthor=M.+S.+Kelleyauthor=C.+S.+Kollmannauthor=F.+Liauthor=K.+Lindauthor=S.+Mataruseauthor=P.+F.+Medeirosauthor=J.+A.+Messerauthor=P.+Myersauthor=H.+O%E2%80%99Keefeauthor=M.+C.+Oliffauthor=C.+E.+Riseauthor=A.+L.+Satzauthor=S.+R.+Skinnerauthor=J.+L.+Svendsenauthor=L.+Tangauthor=K.+Van+Vlotenauthor=R.+W.+Wagnerauthor=G.+Yaoauthor=B.+Zhaoauthor=B.+A.+Morgan&title=Design%2C+synthesis+and+selection+of+DNA-encoded+small-molecule+libraries&doi=10.1038%2Fnchembio.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and selection of DNA-encoded small-molecule libraries</span></div><div class="casAuthors">Clark, Matthew A.; Acharya, Raksha A.; Arico-Muendel, Christopher C.; Belyanskaya, Svetlana L.; Benjamin, Dennis R.; Carlson, Neil R.; Centrella, Paolo A.; Chiu, Cynthia H.; Creaser, Steffen P.; Cuozzo, John W.; Davie, Christopher P.; Ding, Yun; Franklin, G. Joseph; Franzen, Kurt D.; Gefter, Malcolm L.; Hale, Steven P.; Hansen, Nils J. V.; Israel, David I.; Jiang, Jinwei; Kavarana, Malcolm J.; Kelley, Michael S.; Kollmann, Christopher S.; Li, Fan; Lind, Kenneth; Mataruse, Sibongile; Medeiros, Patricia F.; Messer, Jeffrey A.; Myers, Paul; O'Keefe, Heather; Oliff, Matthew C.; Rise, Cecil E.; Satz, Alexander L.; Skinner, Steven R.; Svendsen, Jennifer L.; Tang, Lujia; van Vloten, Kurt; Wagner, Richard W.; Yao, Gang; Zhao, Baoguang; Morgan, Barry A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">647-654</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Biochem. combinatorial techniques such as phage display, RNA display and oligonucleotide aptamers have proven to be reliable methods for generation of ligands to protein targets.  Adapting these techniques to small synthetic mols. has been a long-sought goal.  We report the synthesis and interrogation of an 800-million-member DNA-encoded library in which small mols. are covalently attached to an encoding oligonucleotide.  The library was assembled by a combination of chem. and enzymic synthesis, and interrogated by affinity selection.  We describe methods for the selection and deconvolution of the chem. display library, and the discovery of inhibitors for two enzymes: Aurora A kinase and p38 MAP kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp32j3MHOgNurVg90H21EOLACvtfcHk0lhTR1hXbpEv8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Ogtbo%253D&md5=8a40148f5fca24ecf912ed360802e19d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.211%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DAcharya%26aufirst%3DR.%2BA.%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26aulast%3DBelyanskaya%26aufirst%3DS.%2BL.%26aulast%3DBenjamin%26aufirst%3DD.%2BR.%26aulast%3DCarlson%26aufirst%3DN.%2BR.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DChiu%26aufirst%3DC.%2BH.%26aulast%3DCreaser%26aufirst%3DS.%2BP.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDavie%26aufirst%3DC.%2BP.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DFranzen%26aufirst%3DK.%2BD.%26aulast%3DGefter%26aufirst%3DM.%2BL.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DKavarana%26aufirst%3DM.%2BJ.%26aulast%3DKelley%26aufirst%3DM.%2BS.%26aulast%3DKollmann%26aufirst%3DC.%2BS.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLind%26aufirst%3DK.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DMedeiros%26aufirst%3DP.%2BF.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DMyers%26aufirst%3DP.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DH.%26aulast%3DOliff%26aufirst%3DM.%2BC.%26aulast%3DRise%26aufirst%3DC.%2BE.%26aulast%3DSatz%26aufirst%3DA.%2BL.%26aulast%3DSkinner%26aufirst%3DS.%2BR.%26aulast%3DSvendsen%26aufirst%3DJ.%2BL.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DVan%2BVloten%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DR.%2BW.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DB.%2BA.%26atitle%3DDesign%252C%2520synthesis%2520and%2520selection%2520of%2520DNA-encoded%2520small-molecule%2520libraries%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D647%26epage%3D654%26doi%3D10.1038%2Fnchembio.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luscombe, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span> <span> </span><span class="NLM_article-title">Getting physical in drug discovery II: The impact of chromatographic hydrophobicity measurements and aromaticity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">822</span>– <span class="NLM_lpage">830</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1016%2Fj.drudis.2011.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=21704184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=822-830&author=R.+J.+Youngauthor=D.+V.+S.+Greenauthor=C.+N.+Luscombeauthor=A.+P.+Hill&title=Getting+physical+in+drug+discovery+II%3A+The+impact+of+chromatographic+hydrophobicity+measurements+and+aromaticity&doi=10.1016%2Fj.drudis.2011.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Getting physical in drug discovery II: the impact of chromatographic hydrophobicity measurements and aromaticity</span></div><div class="casAuthors">Young, Robert J.; Green, Darren V. S.; Luscombe, Christopher N.; Hill, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17/18</span>),
    <span class="NLM_cas:pages">822-830</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Here, we review the performance of chromatog. hydrophobicity measurements in a data set of 100 000 GlaxoSmithKline compds., demonstrating the advantages of the method over octanol-water partitioning and highlighting new insights for drug discovery.  The value of chromatog. measurements, vs. other hydrophobicity ests., was supported by improved relationships with soly., permeation, cytochrome P450s, intrinsic clearance, hERG binding and promiscuity.  We also obsd. marked differentiation of the relative influence of intrinsic and effective hydrophobicity.  The summing of hydrophobicity values plus arom. ring count [log D pH7.4 (or log P) + #Ar], indicated a wide relevance for simplistic property forecast indexes' in developability assays, clearly enhanced by chromatog. values; therefore establishing new foundations for enriching property-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmIJyWi-DBW7Vg90H21EOLACvtfcHk0lhaYPjZHUl8oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFCru7vF&md5=22788603a45aa77537e8666433473d14</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DLuscombe%26aufirst%3DC.%2BN.%26aulast%3DHill%26aufirst%3DA.%2BP.%26atitle%3DGetting%2520physical%2520in%2520drug%2520discovery%2520II%253A%2520The%2520impact%2520of%2520chromatographic%2520hydrophobicity%2520measurements%2520and%2520aromaticity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D822%26epage%3D830%26doi%3D10.1016%2Fj.drudis.2011.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Structural basis for acetylated histone H4 recognition by the human BRD2 bromodomain</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">7610</span>– <span class="NLM_lpage">7618</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.062422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1074%2Fjbc.M109.062422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=20048151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=7610-7618&author=T.+Umeharaauthor=Y.+Nakamuraauthor=M.+K.+Jangauthor=K.+Nakanoauthor=A.+Tanakaauthor=K.+Ozatoauthor=B.+Padmanabhanauthor=S.+Yokoyama&title=Structural+basis+for+acetylated+histone+H4+recognition+by+the+human+BRD2+bromodomain&doi=10.1074%2Fjbc.M109.062422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Acetylated Histone H4 Recognition by the Human BRD2 Bromodomain</span></div><div class="casAuthors">Umehara, Takashi; Nakamura, Yoshihiro; Jang, Moon-Kyoo; Nakano, Kazumi; Tanaka, Akiko; Ozato, Keiko; Padmanabhan, Balasundaram; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">7610-7618</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Recognition of acetylated chromatin by the bromodomains and extra-terminal domain (BET) family proteins is a hallmark for transcriptional activation and anchoring viral genomes to mitotic chromosomes of the host.  One of the BET family proteins, BRD2, interacts with acetylated chromatin during mitosis and leads to transcriptional activation in culture cells.  Here, we report the crystal structures of the N-terminal bromodomain of human BRD2 (BRD2-BD1; residues 74-194) in complex with each of three different Lys-12-acetylated H4 peptides.  The BRD2-BD1 recognizes the H4 tail acetylated at Lys-12 (H4K12ac), whereas the side chain of hypoacetylated Lys-8 of H4 binds at the cavity of the dimer interface of BRD2-BD1.  From binding studies, we identified the BRD2-BD1 residues that are responsible for recognition of the Lys-12-acetylated H4 tail.  In addn., mutation to Lys-8 in the Lys-12-acetylated H4 tail decreased the binding to BRD2-BD1, implicating the crit. role of Lys-8 in the Lys-12-acetylated H4 tail for the recognition by BRD2-BD1.  Our findings provide a structural basis for deciphering the histone code by the BET bromodomain through the binding with a long segment of the histone H4 tail, which presumably prevents erasure of the histone code during the cell cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMfSa5EPdd57Vg90H21EOLACvtfcHk0lhaYPjZHUl8oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFaktro%253D&md5=88df449d778d37724b9295ad36e0e188</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.062422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.062422%26sid%3Dliteratum%253Aachs%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DJang%26aufirst%3DM.%2BK.%26aulast%3DNakano%26aufirst%3DK.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DPadmanabhan%26aufirst%3DB.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520for%2520acetylated%2520histone%2520H4%2520recognition%2520by%2520the%2520human%2520BRD2%2520bromodomain%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D7610%26epage%3D7618%26doi%3D10.1074%2Fjbc.M109.062422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kestranek, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chervenak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Placko, S.</span></span> <span> </span><span class="NLM_article-title">Chemiluminescent Nitrogen Detection (CLND) to measure kinetic aqueous solubility</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch130145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=10.1002%2F9780470559277.ch130145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=24391097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=269-280&author=A.+Kestranekauthor=A.+Chervenakauthor=J.+Longenbergerauthor=S.+Placko&title=Chemiluminescent+Nitrogen+Detection+%28CLND%29+to+measure+kinetic+aqueous+solubility&doi=10.1002%2F9780470559277.ch130145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Chemiluminescent nitrogen detection (CLND) to measure kinetic aqueous solubility</span></div><div class="casAuthors">Kestranek Aimee; Chervenak Andrew; Longenberger Justin; Placko Steven</div><div class="citationInfo"><span class="NLM_cas:title">Current protocols in chemical biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-80</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Solubility is the dose-limiting property for in vitro studies, and therefore is a critical physicochemical property to measure in drug discovery.  Solubility data can be used to guide lead optimization, troubleshoot erratic bioassay results, and identify potential downstream liabilities such as insufficient solubility for bioassays or oral bioavailability.  Typically, early in vitro studies are performed using library compounds prepared as dimethylsulfoxide (DMSO) stock solutions, resulting in in vitro test solutions containing DMSO at low concentration (<5% v/v).  Since DMSO can affect the apparent solubility, it is desirable to obtain solubility data under conditions mimicking the in vitro study.  Kinetic solubility (from DMSO stock solutions) is often preferred over thermodynamic solubility (from dry powder) in early drug discovery.  The protocols in this article describe a general procedure for assessing kinetic aqueous solubility of early drug discovery compounds using a miniaturized shake flask method with chemiluminescent nitrogen detection (CLND).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQa5zXGR-PJU_SAs_PE1QvifW6udTcc2eZCltNikWRDALntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czitF2itw%253D%253D&md5=d63d6d57c4ff0934379d3c37f82eb1a0</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch130145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch130145%26sid%3Dliteratum%253Aachs%26aulast%3DKestranek%26aufirst%3DA.%26aulast%3DChervenak%26aufirst%3DA.%26aulast%3DLongenberger%26aufirst%3DJ.%26aulast%3DPlacko%26aufirst%3DS.%26atitle%3DChemiluminescent%2520Nitrogen%2520Detection%2520%2528CLND%2529%2520to%2520measure%2520kinetic%2520aqueous%2520solubility%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3D269%26epage%3D280%26doi%3D10.1002%2F9780470559277.ch130145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Readshaw, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollerton, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynn, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boughtflower, B. J.</span></span> <span> </span><span class="NLM_article-title">Use of calculated physicochemical properties to enhance quantitative response when using charged aerosol detection</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1772</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acs.analchem.6b04060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b04060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2017&pages=1772-1777&author=M.+W.+Robinsonauthor=A.+P.+Hillauthor=S.+A.+Readshawauthor=J.+C.+Hollertonauthor=R.+J.+Uptonauthor=S.+M.+Lynnauthor=S.+C.+Besleyauthor=B.+J.+Boughtflower&title=Use+of+calculated+physicochemical+properties+to+enhance+quantitative+response+when+using+charged+aerosol+detection&doi=10.1021%2Facs.analchem.6b04060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Calculated Physicochemical Properties to Enhance Quantitative Response When Using Charged Aerosol Detection</span></div><div class="casAuthors">Robinson, Max W.; Hill, Alan P.; Readshaw, Simon A.; Hollerton, John C.; Upton, Richard J.; Lynn, Sean M.; Besley, Steve C.; Boughtflower, Bob J.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1772-1777</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Universal quant. detection without the need for analyte ref. stds. would offer substantial benefits in many areas of anal. science.  The quant. capability of high performance liq. chromatog. (HPLC) with charged aerosol detection (CAD) was investigated for 50 compds. with a wide range of phys. and chem. properties.  It is widely believed that CAD is a mass detector.  Quantification of the 50 compds. using a generic calibrant and mass calibration achieved an av. error of 11.4% relative to 1H NMR.  Correction factors are proposed that est. the relative surface area of particles in the detector, taking into account the effects of the d. and charge of analytes.  Performing these corrections and quantifying with surface area calibration, rather than mass, shows considerably improved linearity and uniformity of detection, reducing the av. error relative to 1H NMR to 7.1%.  The accuracy of CAD quantification was most significantly improved for highly dense compds., with traditional mass calibration showing an av. error of 34.7% and the newly proposed surface area calibration showing an av. error of 5.8%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWCD0M2yQRLVg90H21EOLACvtfcHk0liTeBK9ZKDfqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFams7%252FK&md5=46c11b7527157b76f1fbc8eaf5cef75a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b04060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b04060%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DM.%2BW.%26aulast%3DHill%26aufirst%3DA.%2BP.%26aulast%3DReadshaw%26aufirst%3DS.%2BA.%26aulast%3DHollerton%26aufirst%3DJ.%2BC.%26aulast%3DUpton%26aufirst%3DR.%2BJ.%26aulast%3DLynn%26aufirst%3DS.%2BM.%26aulast%3DBesley%26aufirst%3DS.%2BC.%26aulast%3DBoughtflower%26aufirst%3DB.%2BJ.%26atitle%3DUse%2520of%2520calculated%2520physicochemical%2520properties%2520to%2520enhance%2520quantitative%2520response%2520when%2520using%2520charged%2520aerosol%2520detection%26jtitle%3DAnal.%2520Chem.%26date%3D2017%26volume%3D89%26spage%3D1772%26epage%3D1777%26doi%3D10.1021%2Facs.analchem.6b04060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D.</span></span> <span> </span><span class="NLM_article-title">Chromatographic hydrophobicity index by fast-gradient RP-HPLC: A high-throughput alternative to log P/log D</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">2022</span>– <span class="NLM_lpage">2029</span>, <span class="refDoi"> DOI: 10.1021/ac961242d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac961242d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1997&pages=2022-2029&author=K.+Valkoauthor=C.+Bevanauthor=D.+Reynolds&title=Chromatographic+hydrophobicity+index+by+fast-gradient+RP-HPLC%3A+A+high-throughput+alternative+to+log+P%2Flog+D&doi=10.1021%2Fac961242d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatographic Hydrophobicity Index by Fast-Gradient RP-HPLC: A High-Throughput Alternative to Log P/log D</span></div><div class="casAuthors">Valko, Klara; Bevan, Chris; Reynolds, Derek</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2022-2029</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new chromatog. hydrophobicity index (CHI) is described which can be used as part of a protocol for high-throughput (50-100 compds./day) physicochem. property profiling for rational drug design.  The index is derived from retention times (tR) obsd. in a fast gradient reversed-phase HPLC method.  The isocratic retention factors (log k') were measured for a series of 76 structurally unrelated compds. by using various concns. of acetonitrile in the mobile phase.  By plotting the log k' as a function of the acetonitrile concn., the slope (S) and the intercept (log k'w) values were calcd.  The previously validated index of hydrophobicity φ0 was calcd. as -log k'w/S.  A good linear correlation was obtained between the gradient retention time values, tR and the isocratically detd. φ0 values for the 76 compds.  The consts. of this linear correlation can be used to calc. CHI.  For most compds., CHI is between 0 and 100 and in this range it approximates to the percentage (by vol.) of acetonitrile required to achieve an equal distribution of compd. between the mobile and the stationary phases.  CHI values can be measured using acidic, neutral, or slightly basic eluents.  Values corresponding to the neutral form of mols. could be measured for 52 of the compds. and showed good correlation (r = 0.851) to the calcd. octanol/water partition coeff. (c log P) values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozvQI1B1cnpbVg90H21EOLACvtfcHk0ljRDwd9hu0mHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D&md5=fcbb641105c12dc490d4a20db8fc5032</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fac961242d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac961242d%26sid%3Dliteratum%253Aachs%26aulast%3DValko%26aufirst%3DK.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DReynolds%26aufirst%3DD.%26atitle%3DChromatographic%2520hydrophobicity%2520index%2520by%2520fast-gradient%2520RP-HPLC%253A%2520A%2520high-throughput%2520alternative%2520to%2520log%2520P%252Flog%2520D%26jtitle%3DAnal.%2520Chem.%26date%3D1997%26volume%3D69%26spage%3D2022%26epage%3D2029%26doi%3D10.1021%2Fac961242d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furze, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Structure-based optimization of naphthyridones into potent ATAD2 bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6151</span>– <span class="NLM_lpage">6178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00773</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00773" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6151-6178&author=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+C.+Furzeauthor=P.+Grandiauthor=A.+M.+Michonauthor=R.+J.+Sheppardauthor=H.+Barnettauthor=H.+Dialloauthor=D.+P.+Dixonauthor=C.+Douaultauthor=E.+J.+Jonesauthor=B.+Karamshiauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=C.+Rauauthor=R.+J.+Watsonauthor=T.+Wernerauthor=E.+H.+Demont&title=Structure-based+optimization+of+naphthyridones+into+potent+ATAD2+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.5b00773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Optimization of Naphthyridones into Potent ATAD2 Bromodomain Inhibitors</span></div><div class="casAuthors">Bamborough, Paul; Chung, Chun-wa; Furze, Rebecca C.; Grandi, Paola; Michon, Anne-Marie; Sheppard, Robert J.; Barnett, Heather; Diallo, Hawa; Dixon, David P.; Douault, Clement; Jones, Emma J.; Karamshi, Bhumika; Mitchell, Darren J.; Prinjha, Rab K.; Rau, Christina; Watson, Robert J.; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6151-6178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 is a bromodomain-contg. protein whose overexpression is linked to poor outcomes in a no. of different cancer types.  To date, no potent and selective inhibitors of the bromodomain have been reported.  This article describes the structure-based optimization of a series of naphthyridones from micromolar leads with no selectivity over the BET bromodomains to inhibitors with sub-100 nM ATAD2 potency and 100-fold BET selectivity, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcCY9tFp5dGbVg90H21EOLACvtfcHk0ljRDwd9hu0mHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms7jK&md5=c7dcd72fdf4ab1c8557be45085cc68fa</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00773%26sid%3Dliteratum%253Aachs%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DFurze%26aufirst%3DR.%2BC.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDixon%26aufirst%3DD.%2BP.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DStructure-based%2520optimization%2520of%2520naphthyridones%2520into%2520potent%2520ATAD2%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6151%26epage%3D6178%26doi%3D10.1021%2Facs.jmedchem.5b00773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEO','PDB','7OEO'); return false;">PDB: 7OEO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB1','PDB','6ZB1'); return false;">PDB: 6ZB1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E6J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6E6J','PDB','6E6J'); return false;">PDB: 6E6J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OEP','PDB','7OEP'); return false;">PDB: 7OEP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OER" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OER','PDB','7OER'); return false;">PDB: 7OER</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ZB3','PDB','6ZB3'); return false;">PDB: 6ZB3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OES" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OES','PDB','7OES'); return false;">PDB: 7OES</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OET" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7OET','PDB','7OET'); return false;">PDB: 7OET</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i116"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00412">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_38588"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00412?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00412</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">SMILES data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_001.csv">CSV</a>)</p></li><li><p class="inline">DiscoverX BROMOscan bromodomain profiling of <b>60</b>, sequence alignment and differences of BET proteins, X-ray crystallographic data, molecular formula strings, and selected LCMS and NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_001.csv">jm1c00412_si_001.csv (6.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00412/suppl_file/jm1c00412_si_002.pdf">jm1c00412_si_002.pdf (4.95 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00412&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00412%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00412" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993be5ed822212","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
