
<html lang="en"     class="pb-page"  data-request-id="1b52da96-9864-4fcc-881b-51f5caeb7d2e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2018.61.issue-21;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b00736;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Tarantula Venom-Derived NaV1.7-Inhibitory JzTx-V Peptide 5-Br-Trp24 Analogue AM-6120 with Systemic Block of Histamine-Induced Pruritis" /></meta><meta name="dc.Creator" content="Bin  Wu" /></meta><meta name="dc.Creator" content="Justin K.  Murray" /></meta><meta name="dc.Creator" content="Kristin L.  Andrews" /></meta><meta name="dc.Creator" content="Kelvin  Sham" /></meta><meta name="dc.Creator" content="Jason  Long" /></meta><meta name="dc.Creator" content="Jennifer  Aral" /></meta><meta name="dc.Creator" content="Joseph  Ligutti" /></meta><meta name="dc.Creator" content="Shanti  Amagasu" /></meta><meta name="dc.Creator" content="Dong  Liu" /></meta><meta name="dc.Creator" content="Anruo  Zou" /></meta><meta name="dc.Creator" content="Xiaoshan  Min" /></meta><meta name="dc.Creator" content="Zhulun  Wang" /></meta><meta name="dc.Creator" content="Christopher P.  Ilch" /></meta><meta name="dc.Creator" content="Thomas J.  Kornecook" /></meta><meta name="dc.Creator" content="Min-Hwa Jasmine  Lin" /></meta><meta name="dc.Creator" content="Xuhai  Be" /></meta><meta name="dc.Creator" content="Les P.  Miranda" /></meta><meta name="dc.Creator" content="Bryan D.  Moyer" /></meta><meta name="dc.Creator" content="Kaustav  Biswas" /></meta><meta name="dc.Description" content="Inhibitors of the voltage-gated sodium channel NaV1.7 are being investigated as pain therapeutics due to compelling human genetics. We previously identified NaV1.7-inhibitory peptides GpTx-1 and Jz..." /></meta><meta name="Description" content="Inhibitors of the voltage-gated sodium channel NaV1.7 are being investigated as pain therapeutics due to compelling human genetics. We previously identified NaV1.7-inhibitory peptides GpTx-1 and Jz..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 22, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00736" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00736" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00736" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00736" /></link>
        
    
    

<title>Discovery of Tarantula Venom-Derived NaV1.7-Inhibitory JzTx-V Peptide 5-Br-Trp24 Analogue AM-6120 with Systemic Block of Histamine-Induced Pruritis | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00736" /></meta><meta property="og:title" content="Discovery of Tarantula Venom-Derived NaV1.7-Inhibitory JzTx-V Peptide 5-Br-Trp24 Analogue AM-6120 with Systemic Block of Histamine-Induced Pruritis" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0007.jpeg" /></meta><meta property="og:description" content="Inhibitors of the voltage-gated sodium channel NaV1.7 are being investigated as pain therapeutics due to compelling human genetics. We previously identified NaV1.7-inhibitory peptides GpTx-1 and JzTx-V from tarantula venom screens. Potency and selectivity were modulated through attribute-based positional scans of native residues via chemical synthesis. Herein, we report JzTx-V lead optimization to identify a pharmacodynamically active peptide variant. Molecular docking of peptide ensembles from NMR into a homology model-derived NaV1.7 structure supported prioritization of key residues clustered on a hydrophobic face of the disulfide-rich folded peptide for derivatization. Replacing Trp24 with 5-Br-Trp24 identified lead peptides with activity in electrophysiology assays in engineered and neuronal cells. 5-Br-Trp24 containing peptide AM-6120 was characterized in X-ray crystallography and pharmacokinetic studies and blocked histamine-induced pruritis in mice after subcutaneous administration, demonstrating systemic NaV1.7-dependent pharmacodynamics. Our data suggests a need for high target coverage based on plasma exposure for impacting in vivo end points with selectivity-optimized peptidic NaV1.7 inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00736"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00736">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00736&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00736&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00736&amp;href=/doi/10.1021/acs.jmedchem.8b00736" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9500-9512</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00802" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00800" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Tarantula Venom-Derived Na<sub>V</sub>1.7-Inhibitory JzTx-V Peptide 5-Br-Trp24 Analogue AM-6120 with Systemic Block of Histamine-Induced Pruritis</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Bin Wu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bin Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><span class="conrtib-corresp"><strong>*</strong>For B.W.: phone, (805)313-5661; E-mail, <a href="/cdn-cgi/l/email-protection#0d6f64637a4d6c606a6863236e6260"><span class="__cf_email__" data-cfemail="77151e190037161a1012195914181a">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bin++Wu">Bin Wu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5247-7332" title="Orcid link">http://orcid.org/0000-0002-5247-7332</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Justin K. Murray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justin K. Murray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justin+K.++Murray">Justin K. Murray</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kristin L. Andrews</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kristin L. Andrews</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kristin+L.++Andrews">Kristin L. Andrews</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kelvin Sham</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kelvin Sham</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kelvin++Sham">Kelvin Sham</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jason Long</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jason Long</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jason++Long">Jason Long</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jennifer Aral</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer Aral</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer++Aral">Jennifer Aral</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Joseph Ligutti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph Ligutti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, Amgen Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph++Ligutti">Joseph Ligutti</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Shanti Amagasu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shanti Amagasu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, Amgen Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shanti++Amagasu">Shanti Amagasu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Dong Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dong Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, Amgen Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dong++Liu">Dong Liu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Anruo Zou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anruo Zou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, Amgen Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anruo++Zou">Anruo Zou</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xiaoshan Min</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoshan Min</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Research, Amgen Inc., 1120 Veterans Blvd, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoshan++Min">Xiaoshan Min</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Zhulun Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhulun Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Research, Amgen Inc., 1120 Veterans Blvd, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhulun++Wang">Zhulun Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Christopher P. Ilch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher P. Ilch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, Amgen Research, Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+P.++Ilch">Christopher P. Ilch</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Thomas J. Kornecook</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas J. Kornecook</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, Amgen Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+J.++Kornecook">Thomas J. Kornecook</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Min-Hwa Jasmine Lin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Min-Hwa Jasmine Lin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmacokinetics and Drug Metabolism, Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Min-Hwa+Jasmine++Lin">Min-Hwa Jasmine Lin</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xuhai Be</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xuhai Be</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmacokinetics and Drug Metabolism, Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xuhai++Be">Xuhai Be</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Les P. Miranda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Les P. Miranda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Les+P.++Miranda">Les P. Miranda</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Bryan D. Moyer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bryan D. Moyer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neuroscience, Amgen Research, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bryan+D.++Moyer">Bryan D. Moyer</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Kaustav Biswas</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaustav Biswas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Therapeutic Discovery, Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</div></div><span class="conrtib-corresp"><strong>*</strong>For K.B.: <a href="/cdn-cgi/l/email-protection#42292337313623346c202b31352331022f273021296c212d2f"><span class="__cf_email__" data-cfemail="315a5044424550471f535842465042715c5443525a1f525e5c">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaustav++Biswas">Kaustav Biswas</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9971-1424" title="Orcid link">http://orcid.org/0000-0001-9971-1424</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00736&amp;href=/doi/10.1021%2Facs.jmedchem.8b00736" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9500â9512</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 22, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 May 2018</li><li><span class="item_label"><b>Published</b> online</span>22 October 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 November 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00736" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00736</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9500%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBin%2BWu%252C%2BJustin%2BK.%2BMurray%252C%2BKristin%2BL.%2BAndrews%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D21%26contentID%3Dacs.jmedchem.8b00736%26title%3DDiscovery%2Bof%2BTarantula%2BVenom-Derived%2BNaV1.7-Inhibitory%2BJzTx-V%2BPeptide%2B5-Br-Trp24%2BAnalogue%2BAM-6120%2Bwith%2BSystemic%2BBlock%2Bof%2BHistamine-Induced%2BPruritis%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9512%26publicationDate%3DNovember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00736"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1489</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00736" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Tarantula Venom-Derived NaV1.7-Inhibitory JzTx-V Peptide 5-Br-Trp24 Analogue AM-6120 with Systemic Block of Histamine-Induced Pruritis&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Bin&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Justin&quot;,&quot;last_name&quot;:&quot;K. Murray&quot;},{&quot;first_name&quot;:&quot;Kristin&quot;,&quot;last_name&quot;:&quot;L. Andrews&quot;},{&quot;first_name&quot;:&quot;Kelvin&quot;,&quot;last_name&quot;:&quot;Sham&quot;},{&quot;first_name&quot;:&quot;Jason&quot;,&quot;last_name&quot;:&quot;Long&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;Aral&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;Ligutti&quot;},{&quot;first_name&quot;:&quot;Shanti&quot;,&quot;last_name&quot;:&quot;Amagasu&quot;},{&quot;first_name&quot;:&quot;Dong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Anruo&quot;,&quot;last_name&quot;:&quot;Zou&quot;},{&quot;first_name&quot;:&quot;Xiaoshan&quot;,&quot;last_name&quot;:&quot;Min&quot;},{&quot;first_name&quot;:&quot;Zhulun&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;P. Ilch&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;J. Kornecook&quot;},{&quot;first_name&quot;:&quot;Min-Hwa&quot;,&quot;last_name&quot;:&quot;Jasmine Lin&quot;},{&quot;first_name&quot;:&quot;Xuhai&quot;,&quot;last_name&quot;:&quot;Be&quot;},{&quot;first_name&quot;:&quot;Les&quot;,&quot;last_name&quot;:&quot;P. Miranda&quot;},{&quot;first_name&quot;:&quot;Bryan&quot;,&quot;last_name&quot;:&quot;D. Moyer&quot;},{&quot;first_name&quot;:&quot;Kaustav&quot;,&quot;last_name&quot;:&quot;Biswas&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;9500-9512&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00736&quot;},&quot;abstract&quot;:&quot;Inhibitors of the voltage-gated sodium channel NaV1.7 are being investigated as pain therapeutics due to compelling human genetics. We previously identified NaV1.7-inhibitory peptides GpTx-1 and JzTx-V from tarantula venom screens. Potency and selectivity were modulated through attribute-based positional scans of native residues via chemical synthesis. Herein, we report JzTx-V lead optimization to identify a pharmacodynamically active peptide variant. Molecular docking of peptide ensembles from NMR into a homology model-derived NaV1.7 structure supported prioritization of key residues clustered on a hydrophobic face of the disulfide-rich folded peptide for derivatization. Replacing Trp24 with 5-Br-Trp24 identified lead peptides with activity in electrophysiology assays in engineered and neuronal cells. 5-Br-Trp24 containing peptide AM-6120 was characterized in X-ray crystallography and pharmacokinetic studies and blocked histamine-induced pruritis in mice after subcutaneous administration, demonstrating syste&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00736&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00736" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00736&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00736" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00736&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00736" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00736&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00736&amp;href=/doi/10.1021/acs.jmedchem.8b00736" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00736" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00736" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00736&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00736%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DWu%26date%3D2018%26atitle%3DDiscovery%2Bof%2BTarantula%2BVenom-Derived%2BNaV1.7-Inhibitory%2BJzTx-V%2BPeptide%2B5-Br-Trp24%2BAnalogue%2BAM-6120%2Bwith%2BSystemic%2BBlock%2Bof%2BHistamine-Induced%2BPruritis%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D21%26spage%3D9500%26epage%3D9512%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292161" title="Modification">Modification</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/jmcmar.2018.61.issue-21/20181108/jmcmar.2018.61.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inhibitors of the voltage-gated sodium channel Na<sub>V</sub>1.7 are being investigated as pain therapeutics due to compelling human genetics. We previously identified Na<sub>V</sub>1.7-inhibitory peptides GpTx-1 and JzTx-V from tarantula venom screens. Potency and selectivity were modulated through attribute-based positional scans of native residues via chemical synthesis. Herein, we report JzTx-V lead optimization to identify a pharmacodynamically active peptide variant. Molecular docking of peptide ensembles from NMR into a homology model-derived Na<sub>V</sub>1.7 structure supported prioritization of key residues clustered on a hydrophobic face of the disulfide-rich folded peptide for derivatization. Replacing Trp24 with 5-Br-Trp24 identified lead peptides with activity in electrophysiology assays in engineered and neuronal cells. 5-Br-Trp24 containing peptide AM-6120 was characterized in X-ray crystallography and pharmacokinetic studies and blocked histamine-induced pruritis in mice after subcutaneous administration, demonstrating systemic Na<sub>V</sub>1.7-dependent pharmacodynamics. Our data suggests a need for high target coverage based on plasma exposure for impacting in vivo end points with selectivity-optimized peptidic Na<sub>V</sub>1.7 inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17225" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17225" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The ongoing public health crisis stemming from opioid overuse and addiction has led to extensive investigation of alternative approaches for clinically managing the unmet burden of chronic pain.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> An appealing nonopioid pain target is the voltage-gated sodium channel Na<sub>V</sub>1.7.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Loss-of-function and gain-of-function mutations in the human gene encoding Na<sub>V</sub>1.7, SCN9A lead to channelopathies like congenital insensitivity to pain, erythromelalgia, paroxysmal extreme pain disorder, and small fiber neuropathy.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> This genetic validation data has led to multiple efforts to identify blockers of Na<sub>V</sub>1.7 function from which molecules have progressed into human clinical trials, although none are yet approved. One key challenge in developing a Na<sub>V</sub>1.7 blocker has been in engineering selectivity against related human sodium channels like Na<sub>V</sub>1.4 and Na<sub>V</sub>1.5, present in skeletal and cardiac muscle, respectively, to avoid unacceptable adverse effects.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a></div><div class="NLM_p">Venoms of poisonous species like spiders, cone snails, and scorpions contain natural peptidic blockers of ion channels like Na<sub>V</sub>1.7, with varying degrees of on-target potency and selectivity against off-target ion channels.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Tarantula venom-derived peptides like ProTx-II, HwTx-IV, and Pn3a and their variants are potent Na<sub>V</sub>1.7 inhibitors in electrophysiology assays. However, in vivo activity in Na<sub>V</sub>1.7-dependent pharmacodynamic models after peptide dosing via systemic parenteral administration like subcutaneous or intravenous injection have not been reported despite examples of successful target engagement via local or intrathecal delivery.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10">(7â10)</a> We previously screened venom collections using high throughput population patch clamp electrophysiology to identify two disulfide-rich Na<sub>V</sub>1.7-inhibitory peptides from tarantula venom, GpTx-1 and JzTx-V.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14">(11â14)</a> They fold into an inhibitory cystine knot (ICK) motif and contain a C-terminal carboxamide but have low overall sequence homology. Na<sub>V</sub>1.7 inhibitory peptides interact with the second voltage sensor domain of the Na<sub>V</sub>1.7 pore forming Î±-subunit and stabilize Na<sub>V</sub>1.7 in a closed conformation.<a onclick="showRef(event, 'ref12 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref14">(12,14)</a> As observed with many natural products, GpTx-1 and JzTx-V were potent blockers of Na<sub>V</sub>1.7 currents in electrophysiology assays in heterologous and neuronal cells but lacked selectivity, especially against the skeletal muscle channel Na<sub>V</sub>1.4. Employing solid-phase peptide synthesis (SPPS), we developed analogues of GpTx-1 with single-digit nanomolar Na<sub>V</sub>1.7 IC<sub>50</sub> values while increasing Na<sub>V</sub>1.4 selectivity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Systemic administration of native GpTx-1 caused adverse effects in mice at doses above 0.1 mg/kg,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and our analoguing campaign did not identify suitable GpTx-1 peptides for in vivo evaluation. In the JzTx-V series, a critical Ile28Glu modification identified via synthetic engineering afforded potent and Na<sub>V</sub>1.4-selective JzTx-V analogues that blocked capsaicin-induced dorsal root ganglion (DRG) neuron action potential firing and mechanically induced C-fiber spiking in a saphenous skinânerve preparation.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">Our structureâactivity relationship (SAR) studies employed multiattribute positional scan (MAPS) analoguing,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> which discretely interrogated single changes in amino acid side chain functionalities across the sequence and correlated them to potency and selectivity. We applied this empirical technique to gain a better understanding of the putative Na<sub>V</sub>1.7 binding face of the peptides in the absence of a high-resolution structure of peptide inhibitor-bound Na<sub>V</sub>1.7. This approach yielded a wealth of information regarding which specific residues on ICK-folded peptides were key to inhibiting channel function, as well as revealing novel substitutions that can impart Na<sub>V</sub> selectivity.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> However, these efforts did not provide peptides that blocked Na<sub>V</sub>1.7 function in preclinical in vivo studies (vide infra).</div><div class="NLM_p last">Herein, we report further optimization of the JzTx-V scaffold to identify peptides that could engage Na<sub>V</sub>1.7 in vivo to block pharmacodynamic end points. We used computational modeling to prioritize SAR studies to JzTx-V residues that potentially contact the channel in the binding pocket. The single substitution data from MAPS analoguing provided guidelines for building a binding model. Conformational ensembles from the NMR structure of JzTx-V analogue Pra-[Nle6]JzTx-V(1â29), peptide <b>1</b>,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> were docked into a homology model of the domain II voltage sensor domain (VSD) of Na<sub>V</sub>1.7. We selected residues Leu23 and Ile29 as well as Trp residues at positions 5, 7, and 24 for further derivatization. Specifically, substitution of Trp24 with 5-Br-Trp enhanced Na<sub>V</sub>1.7 potency. Further optimization led to the potent and selective peptide <b>41</b> (AM-6120), Glu-[Pra1;Nle6;Ala12;5-BrW24;Glu28]JzTx-V(1â29), which was characterized by electrophysiology, X-ray crystallography, and mouse pharmacokinetic studies. Evaluation of AM-6120 in a Na<sub>V</sub>1.7-dependent behavioral model showed robust reversal of histamine-induced pruritis in mice after subcutaneous administration. These studies demonstrate noteworthy systemic efficacy achieved with an optimized 30-residue tarantula-derived folded peptide containing noncanonical amino acids and identify new tools for in vivo target engagement for an ion channel of significant therapeutic interest.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54762" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54762" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Molecular Docking</h3><div class="NLM_p">To help inform SAR efforts, we carried out proteinâprotein docking calculations to generate a potential model of JzTx-V analogue peptide <b>1</b> bound to a homology model of the domain II VSD of human Na<sub>V</sub>1.7. As we had reported, this JzTx-V peptide analogue contains a norleucine (Nle) substitution to mitigate the potential metabolic liability of the native Met6 and a propargylglycine (Pra) extension of the N-terminus<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> that improved potency and provided a potential handle for future derivatization. As described previously,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> a set of homology models of the domain II VSD in the closed conformation were created based on a published Kv1.2 model as a template.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> We used ZDOCK<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> to rigidly dock all 10 members of the NMR ensemble of the peptide to each homology model conformation. Potential poses were filtered ensuring that peptide residues critical for activity were within 5 Ã of any residue from the receptor homology model and likewise that Na<sub>V</sub>1.7 residues 750, 753, 811, 816, and 818 had contact within 5 Ã of any atom from the peptide ligand.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Remaining poses were evaluated by eye to ensure that residues where substitution was tolerated (positions 11, 14, 17) were oriented away from the channel model and that residues critical for activity made contacts with the VSD. A final potential model (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) was chosen based on consistency with the Ala and Glu scanning data<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and further optimized using iterative minimizations with a molecular mechanics force-field (AMBER10:EHT with Generalized Born implicit solvation as implemented in MOE).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Selected binding model from docking of NMR structure ensemble of peptide <b>1</b>, Pra-[Nle6]JzTx-V(1â29) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CHC">6CHC</a>), into a homology model of human Na<sub>V</sub>1.7 domain II voltage sensor domain using ZDOCK. Peptide residues are labeled in black font, select Na<sub>V</sub>1.7 residues are labeled in white font. (B) Overlay of NMR structures of related JzTx-V peptides <b>1</b> (green/yellow) and AM-8145 (gray/magenta), showing alternate orientations of the Trp7 side chain indole moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the selected binding model, Trp24 and Arg26 of JzTx-V bind deeply into the domain II VSD cleft of Na<sub>V</sub>1.7. Arg26 is oriented to make a potential interaction with Glu811. The Trp-Arg pair reprises the hydrophobic-basic pairing frequently observed in peptide toxin binding.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Leu23 is similarly oriented toward the bottom of the VSD toward the channel pore domain near hydrophobic residues Met750, Leu820, and Ala1333. The C-terminal residue Ile29 is oriented into the cleft of the VSD and modeled toward the outside of the channel and lipid bilayer. Trp5 and Nle6 are modeled to contact the S1âS2 loop, while Lys22 is positioned to make electrostatic interactions with the S3âS4 loop including residue Glu818. Positions on the peptide scaffold where Ala/Glu substitution is tolerated (e.g., N-terminus, 11, 14, 17) are oriented away from the channel, as this was part of our model selection criteria.</div><div class="NLM_p">Interestingly, the docked model does not suggest specific contacts between Trp7 and the VSD in contrast to the single substitution scanning data. Comparing the NMR ensembles for peptide <b>1</b> and the potent and selective JzTx-V analogue AM-8145, Pra-[Nle6;Glu28]JzTx-V(1â29),<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> we observed that Trp7 shows conformational variability with the side chain orientation differing by approximately 90Â° between the two structures (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). This orientation would position Trp7 in proximity of the channel pore and the S1âS2 loop, suggesting inclusion of this residue in further SAR studies.</div><div class="NLM_p last">The docked model we generated as part of this SAR campaign was sufficient to support analoguing efforts described below, and it provided a post hoc visual rationalization of the existing SAR data. After this work was completed, the high resolution structures of eukaryotic voltage gated sodium channels were published; cryoelectron microscopy structures of the cockroach Na<sub>V</sub>PaS<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and the EeNa<sub>V</sub>1.4-Î²1 from electric eel.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> These new structures will inform future work with improved VSD models and may provide additional insights into the specific intermolecular interactions between the channel and toxins.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Peptide SAR Study</h3><div class="NLM_p">On the basis of the single residue scan data and binding model, we selected the nonpolar residues at positions 23 and 29, and the three Trp residues 5, 7, and 24 for initial SAR studies to modulate Na<sub>V</sub>1.7 potency (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Like JzTx-V, all new peptides contained C-terminal amides unless otherwise noted. Replacement of the Leu23 residue in AM-8145 with alkyl amino acids Ile (peptide <b>2</b>), norleucine (Nle, peptide <b>3</b>), or norvaline (Nva, peptide <b>4</b>) led to similar or modestly lower Na<sub>V</sub>1.7 potency in a whole cell patch clamp electrophysiology assay on an automated PatchXpress (PX) system using Na<sub>V</sub>1.7 stably transfected HEK293 cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Cyclic side chain modifications were explored next. Cyclohexylglycine (Chg) modification (peptide <b>5</b>) maintained Na<sub>V</sub>1.7 potency, while the slightly larger cyclohexylalanine (Cha, peptide <b>6</b>) resulted in 3-fold loss of activity on Na<sub>V</sub>1.7. We synthesized AM-8145 analogues with hydrophobic amino acid residues Phe (peptide <b>7</b>), homophenylalanine (hPhe, peptide <b>8</b>), Cha (peptide <b>9</b>), or Trp (peptide <b>10</b>) replacing the aliphatic Ile29 residue, and interestingly all of them showed excellent subnanomolar Na<sub>V</sub>1.7 inhibitory activity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Na<sub>V</sub>1.7 Inhibitory Activity of Single Residue Modified AM-8145 Analogues Determined by PatchXpress Electrophysiology<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="â" /></col><col align="center" /></col><col align="char" char="â" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">substitution</th><th class="colsep0 rowsep0" align="center">hNa<sub>V</sub>1.7Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="â">95% CI<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">hNa<sub>V</sub>1.4Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="â">95% CI<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AM-8145<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="center">none</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="char" char="â">0.4â0.68<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">145</td><td class="colsep0 rowsep0" align="char" char="â">90â200<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">Leu23 Substitution</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="center">Ile</td><td class="colsep0 rowsep0" align="center">1.1</td><td class="colsep0 rowsep0" align="char" char="â">0.61â1.84</td><td class="colsep0 rowsep0" align="center">193</td><td class="colsep0 rowsep0" align="char" char="â">146â256</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="center">Nle</td><td class="colsep0 rowsep0" align="center">1.7</td><td class="colsep0 rowsep0" align="char" char="â">1.1â2.7</td><td class="colsep0 rowsep0" align="center">102</td><td class="colsep0 rowsep0" align="char" char="â">87â120</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="center">Nva</td><td class="colsep0 rowsep0" align="center">3.0</td><td class="colsep0 rowsep0" align="char" char="â">1.7â5.3</td><td class="colsep0 rowsep0" align="center">nd<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="center">Chg</td><td class="colsep0 rowsep0" align="center">0.5</td><td class="colsep0 rowsep0" align="char" char="â">0.38â0.65</td><td class="colsep0 rowsep0" align="center">177</td><td class="colsep0 rowsep0" align="char" char="â">116â270</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="center">Cha</td><td class="colsep0 rowsep0" align="center">1.6</td><td class="colsep0 rowsep0" align="char" char="â">1.2â2.0</td><td class="colsep0 rowsep0" align="center">117</td><td class="colsep0 rowsep0" align="char" char="â">62â221</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">Ile29 Substitution</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="center">Phe</td><td class="colsep0 rowsep0" align="center">0.2</td><td class="colsep0 rowsep0" align="char" char="â">0.11â0.33</td><td class="colsep0 rowsep0" align="center">162</td><td class="colsep0 rowsep0" align="char" char="â">106â247</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="center">hPhe</td><td class="colsep0 rowsep0" align="center">0.3</td><td class="colsep0 rowsep0" align="char" char="â">0.25â0.47</td><td class="colsep0 rowsep0" align="center">23</td><td class="colsep0 rowsep0" align="char" char="â">14â38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="center">Cha</td><td class="colsep0 rowsep0" align="center">0.6</td><td class="colsep0 rowsep0" align="char" char="â">0.35â1.0</td><td class="colsep0 rowsep0" align="center">37</td><td class="colsep0 rowsep0" align="char" char="â">27â51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="center">Trp</td><td class="colsep0 rowsep0" align="center">0.2</td><td class="colsep0 rowsep0" align="char" char="â">0.11â0.37</td><td class="colsep0 rowsep0" align="center">92</td><td class="colsep0 rowsep0" align="char" char="â">71â120</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">Trp5 Substitution</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="center">1-Nal</td><td class="colsep0 rowsep0" align="center">71</td><td class="colsep0 rowsep0" align="char" char="â">56â90</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="center">2-Nal</td><td class="colsep0 rowsep0" align="center">22</td><td class="colsep0 rowsep0" align="char" char="â">15â32</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="center">5-Br-Trp</td><td class="colsep0 rowsep0" align="center">22</td><td class="colsep0 rowsep0" align="char" char="â">18â28</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="center">Phe</td><td class="colsep0 rowsep0" align="center">1.3</td><td class="colsep0 rowsep0" align="char" char="â">0.9â1.9</td><td class="colsep0 rowsep0" align="center">52</td><td class="colsep0 rowsep0" align="char" char="â">32â87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="center">hPhe</td><td class="colsep0 rowsep0" align="center">0.6</td><td class="colsep0 rowsep0" align="char" char="â">0.3â1.3</td><td class="colsep0 rowsep0" align="center">258</td><td class="colsep0 rowsep0" align="char" char="â">149â450</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">Trp7 Substitution</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="center">1-Nal</td><td class="colsep0 rowsep0" align="center">825</td><td class="colsep0 rowsep0" align="char" char="â">598â1138</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="center">2-Nal</td><td class="colsep0 rowsep0" align="center">471</td><td class="colsep0 rowsep0" align="char" char="â">351â631</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="center">5-Br-Trp</td><td class="colsep0 rowsep0" align="center">540</td><td class="colsep0 rowsep0" align="char" char="â">437â667</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="center">Phe</td><td class="colsep0 rowsep0" align="center">12.6</td><td class="colsep0 rowsep0" align="char" char="â">8.8â18.1</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="center">hPhe</td><td class="colsep0 rowsep0" align="center">6.8</td><td class="colsep0 rowsep0" align="char" char="â">4.2â11.2</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">Trp24 Substitution</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="center">1-Nal</td><td class="colsep0 rowsep0" align="center">32</td><td class="colsep0 rowsep0" align="char" char="â">18.1â56.5</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="center">2-Nal</td><td class="colsep0 rowsep0" align="center">182</td><td class="colsep0 rowsep0" align="char" char="â">110â301</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="center">Phe</td><td class="colsep0 rowsep0" align="center">15</td><td class="colsep0 rowsep0" align="char" char="â">9.6â24.4</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="center">hPhe</td><td class="colsep0 rowsep0" align="center">>1000</td><td class="colsep0 rowsep0" align="char" char="â">Â </td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b>Â (AM-0714)</td><td class="colsep0 rowsep0" align="center">5-Br-Trp</td><td class="colsep0 rowsep0" align="center">0.26</td><td class="colsep0 rowsep0" align="char" char="â">0.09â0.43<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">53</td><td class="colsep0 rowsep0" align="char" char="â">27â79<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="center">5-Cl-Trp</td><td class="colsep0 rowsep0" align="center">3.1</td><td class="colsep0 rowsep0" align="char" char="â">1.6â5.9</td><td class="colsep0 rowsep0" align="center">23</td><td class="colsep0 rowsep0" align="char" char="â">18â29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="center">6-Br-Trp</td><td class="colsep0 rowsep0" align="center">8.9</td><td class="colsep0 rowsep0" align="char" char="â">2.8â28</td><td class="colsep0 rowsep0" align="center">39</td><td class="colsep0 rowsep0" align="char" char="â">24â62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="center">6-Me-Trp</td><td class="colsep0 rowsep0" align="center">19</td><td class="colsep0 rowsep0" align="char" char="â">14â26</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="center">7-Br-Trp</td><td class="colsep0 rowsep0" align="center">24</td><td class="colsep0 rowsep0" align="char" char="â">17.7â32.5</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="char" char="â">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">At least four different concentrations of test compound spanning up to four log units were tested. IC<sub>50</sub> values were determined from at least 10 different cells with two to three data points per peptide concentration as previously described.<a onclick="showRef(event, 'ref11 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref13">(11,13)</a></p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">AM-8145: Pra-[Nle6;Glu28]JzTx-V(1â29);</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">To determine 95% confidence intervals, data were fit to a four parameter, variable slope Hill functions in GraphPad Prism.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">nd, not determined. Peptides with hNa<sub>V</sub>1.7 IC<sub>50</sub> < 5 nM were, in general, advanced to the hNa<sub>V</sub>1.4 assay.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last"><i>n</i> â¥ 3</p></div></div></div><div class="NLM_p">We explored Trp5, 7, and 24 modifications by substitution with different hydrophobic aromatic residues. Replacing Trp5 with amino acids containing bulkier side chains such as 1-naphthylalanine (1-Nal, peptide <b>11</b>), 2-naphthylalanine (2-Nal, peptide <b>12</b>), or Trp derivative 5-bromotryptophan (5-Br-Trp, peptide <b>13</b>) led to over 100-fold reduction in potency. Smaller aromatic side chains like Phe (peptide <b>14</b>) or especially the Phe derivative, hPhe5 analogue <b>15</b>, were better tolerated (hNa<sub>V</sub>1.7 IC<sub>50</sub> = 0.6 nM). Trp7 was much more sensitive to modifications, with larger losses in potency observed with both bicyclic (peptides <b>16</b>â<b>18</b>) and Phe-based side chain (peptides <b>19</b> and <b>20</b>) replacements. The hPhe7 analogue <b>20</b> displayed the best potency among this set (Na<sub>V</sub>1.7 IC<sub>50</sub> = 6.8 nM), although it blocked Na<sub>V</sub>1.7 currents an order of magnitude less potently than AM-8145 or hPhe5 peptide <b>15</b>.</div><div class="NLM_p">Trp24 modifications of AM-8145 gave rise to larger variations in Na<sub>V</sub>1.7 potency. While replacing Trp24 with 1-Nal (peptide <b>21</b>), 2-Nal (peptide <b>22</b>), Phe (peptide <b>23</b>), or hPhe (peptide <b>24</b>) all led to significant reduction in Na<sub>V</sub>1.7 potency, incorporation of 5-Br-Trp at position 24 led to analogue <b>25</b> (AM-0714), with IC<sub>50</sub> = 0.26 nM, a 2-fold improvement in Na<sub>V</sub>1.7 inhibition. We next explored the effect of replacing Trp24 with other indole-containing side chains with halogen or methyl moieties incorporated at different positions. Substitutions with 5-Cl-Trp (peptide <b>26</b>) or at positions 6 or 7 on the indole ring (peptides <b>27</b>â<b>29</b>) led to 10- to 100-fold less Na<sub>V</sub>1.7 potency than AM-0714, indicating a preference for the 5-Br-Trp residue in inhibiting Na<sub>V</sub>1.7.</div><div class="NLM_p">The SAR study at positions 5, 7, 23, 24, and 29 on AM-8145 identified a few favorable replacements, including hPhe5, Cha23, and 5-Br-Trp24. Next, we examined the effect of combination of two preferred substitutions on Na<sub>V</sub>1.7 inhibition. To that end, we synthesized a series of AM-8145 double-substitution peptides listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. Evaluation of peptides <b>30</b>â<b>33</b> in the Na<sub>V</sub>1.7 assay showed no improvement in potency by combining 5-Br-Trp24 and Phe29, hPhe29, or hPhe5. Interestingly, peptide <b>30</b>, with neighboring Chg23, 5-Br-Trp24 modifications was 75-fold less potent than AM-8145 (Na<sub>V</sub>1.7 IC<sub>50</sub> = 19.6 nM). We decided to focus further analoguing around the single 5-Br-Trp24 hydrophobic face modification of AM-8145.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Na<sub>V</sub>1.7 Inhibitory Activity of AM-8145 Analogues with Two Substitutions<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="â" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">AM-8145 substitutions</th><th class="colsep0 rowsep0" align="center" char=".">hNa<sub>V</sub>1.7Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="â">95% CI<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="center">Chg23,Â 5-Br-Trp24</td><td class="colsep0 rowsep0" align="char" char=".">19.6</td><td class="colsep0 rowsep0" align="char" char="â">11.1â34.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="center">5-Br-Trp24,Â Phe29</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char="â">0.8â1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="center">5-Br-Trp24,Â hPhe29</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char="â">0.7â1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="center">hPhe5,Â 5-Br-Trp24</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char="â">0.6â1.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">At least four different concentrations of test compound spanning up to four log units were tested. IC<sub>50</sub> values were determined from at least 10 different cells with two to three data points per peptide concentration as previously described.;<a onclick="showRef(event, 'ref11 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref13">(11,13)</a></p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">To determine 95% confidence intervals, data were fit to a four parameter, variable slope Hill functions in GraphPad Prism.</p></div></div></div><div class="NLM_p">We examined the effect of the 5-Br-Trp24 modification on selectivity against other representative sodium channels. AM-8145 and AM-0714 were tested on both tetrodotoxin-sensitive (TTX-S) channels like Na<sub>V</sub>1.4 (present in skeletal muscle) and Na<sub>V</sub>1.6 (present in motor neurons) as well as tetrodotoxin-resistant (TTX-R) sodium channels Na<sub>V</sub>1.5 (present in cardiac myocytes) and Na<sub>V</sub>1.8 (present in peripheral nociceptors).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> We observed increases in potency against other Na<sub>V</sub> isoforms with the 5-Br-Trp24 change, including Na<sub>V</sub>1.4, 1.5, and 1.6 while maintaining the overall selectivity profile of AM-8145 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Na<sub>V</sub> Isoform Selectivity of AM-8145 and 5-Br-Trp24 Analogue AM-0714<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center" char="Â±">Â </th><th class="rowsep1 colsep0" colspan="2" align="center" char="Â±">TTX-S sodium channels</th><th class="rowsep1 colsep0" colspan="2" align="center">TTX-R sodium channels</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="Â±">Na<sub>V</sub>1.7Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="Â±">Na<sub>V</sub>1.4Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="Â±">Na<sub>V</sub>1.6Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center">Na<sub>V</sub>1.5Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center">Na<sub>V</sub>1.8 IC<sub>50</sub>Â (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AM-8145</td><td class="colsep0 rowsep0" align="char" char="Â±">0.5Â Â±Â 0.07</td><td class="colsep0 rowsep0" align="char" char="Â±">145Â Â±Â 28</td><td class="colsep0 rowsep0" align="char" char="Â±">183Â Â±Â 32</td><td class="colsep0 rowsep0" align="center">3072Â Â±Â 72</td><td class="colsep0 rowsep0" align="center">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AM-0714</td><td class="colsep0 rowsep0" align="char" char="Â±">0.26Â Â±Â 0.08</td><td class="colsep0 rowsep0" align="char" char="Â±">53.4Â Â±Â 13</td><td class="colsep0 rowsep0" align="char" char="Â±">66Â Â±Â 17</td><td class="colsep0 rowsep0" align="center">260<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">509Â Â±Â 21</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">At least four different concentrations of test compound spanning up to four log units were tested. IC<sub>50</sub> values were determined from at least 10 different cells with two to three data points per peptide concentration as previously described.<a onclick="showRef(event, 'ref11 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref13">(11,13)</a></p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">MEAN Â± SEM, <i>n</i> â¥ 2.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> = 1.</p></div></div></div><div class="NLM_p">We next examined the effect of modifications at the termini of AM-8145. JzTx-V and the analogues prepared above contain a carboxamide moiety at the C-terminus. Replacing the terminal amide with a glycine-carboxylic acid moiety<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> in peptide <b>34</b> led to a 10-fold loss in potency (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Attribute-based N-terminal modifications were explored with Glu, Ala, and Lys (peptides <b>35</b>â<b>37</b>). The Pra residue was shifted from the N-terminus (in AM-8145) to position 1 in these three peptides, replacing Tyr1. The Glu-Pra1 peptide <b>35</b> was equipotent to AM-0714, and Ala- and Lys- additions led to 4-fold and >10-fold losses in potency, respectively.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Na<sub>V</sub>1.7 Inhibitory Activity of Terminal Extension Analogues of AM-8145<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="â" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">substitutions</th><th class="colsep0 rowsep0" align="center" char=".">hNa<sub>V</sub>1.7Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="â">95% CI<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="center">âGly-COOH</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char="â">6.2â11.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="center">Glu-Pra1,Â 5-BrW24</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char="â">0.25â0.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="center">Ala-Pra1,Â 5-BrW24</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char="â">0.5â2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="center">Lys-Pra1,Â 5-BrW24</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td><td class="colsep0 rowsep0" align="char" char="â">2.8â11.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">At least four different concentrations of test compound spanning up to four log units were tested. IC<sub>50</sub> values were determined from at least 10 different cells with two to three data points per peptide concentration as previously described.<a onclick="showRef(event, 'ref11 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref13">(11,13)</a></p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">To determine 95% confidence intervals, data were fit to a four parameter, variable slope Hill functions in GraphPad Prism.</p></div></div></div><div class="NLM_p">Ala and Glu scans of JzTx-V had showed that substitutions at positions 11, 12, and 14 were tolerated and showed modest improvements in Na<sub>V</sub>1.4 selectivity.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> With the N-terminus set as Glu-Pra1, we scanned the impact of substitutions at these locations (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Substitution of Ser11 and Ala14 (peptides <b>38</b> and <b>40</b>) led to slight lowering of Na<sub>V</sub>1.7 potency compared to AM-8145, but substitution of Lys12 with either Glu or Ala maintained subnanomolar Na<sub>V</sub>1.7 potency and also displayed over 100-fold selectivity against the skeletal muscle channel Na<sub>V</sub>1.4 (peptides <b>39</b> and <b>41</b>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Ala/Glu SAR at Positions 11/12/14 on Peptide Glu-[Pra1;Nle6;5-Br-Trp24;Glu28]JzTx-V(1-29)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="â" /></col><col align="char" char="." /></col><col align="char" char="â" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">substitution</th><th class="colsep0 rowsep0" align="center" char=".">Na<sub>V</sub>1.7Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="â">95% CI<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">Na<sub>V</sub>1.4Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="â">95% CI<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="center">Glu11</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char="â">1.9â3.5</td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="char" char="â">60â131</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="center">Glu12</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char="â">0.35â0.68</td><td class="colsep0 rowsep0" align="char" char=".">72</td><td class="colsep0 rowsep0" align="char" char="â">38â135</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="center">Glu14</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char="â">0.6â2.4</td><td class="colsep0 rowsep0" align="char" char=".">182</td><td class="colsep0 rowsep0" align="char" char="â">141â233</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b>Â (AM-6120)</td><td class="colsep0 rowsep0" align="center">Ala12</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char="â">0.64â1.1<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">104</td><td class="colsep0 rowsep0" align="char" char="â">44â164<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">At least four different concentrations of test compound spanning up to four log units were tested. IC<sub>50</sub> values were determined from at least 10 different cells with two to three data points per peptide concentration as previously described.<a onclick="showRef(event, 'ref11 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref13">(11,13)</a></p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">To determine 95% confidence intervals, data were fit to a four parameters, variable slope Hill functions in GraphPad Prism.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> â¥ 3.</p></div></div></div><div class="NLM_p">Peptide <b>41</b>, AM-6120, was selected for further profiling in the human Na<sub>V</sub> selectivity panel and exhibited excellent selectivity (>750-fold) against Na<sub>V</sub>1.5, 1.6, and 1.8 (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). In addition, it potently blocked the recombinant mouse Na<sub>V</sub>1.7 channel expressed in HEK293 cells and evaluated on the PatchXpress platform (mNa<sub>V</sub>1.7 IC<sub>50</sub> = 5.4 nM).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Na<sub>V</sub> Potency and Selectivity of AM-6120 Determined by PatchXpress Electrophysiology<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="Â±">hNa<sub>V</sub>1.7Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="Â±">mNa<sub>V</sub>1.7Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center" char="Â±">hNa<sub>V</sub>1.4Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center">hNa<sub>V</sub>1.5Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center">hNa<sub>V</sub>1.6Â PX IC<sub>50</sub>Â (nM)</th><th class="colsep0 rowsep0" align="center">hNa<sub>V</sub>1.8 IC<sub>50</sub>Â (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AM-6120</td><td class="colsep0 rowsep0" align="char" char="Â±">0.8Â Â±Â 0.12</td><td class="colsep0 rowsep0" align="char" char="Â±">5.4Â Â±Â 3.4</td><td class="colsep0 rowsep0" align="char" char="Â±">104Â Â±Â 30</td><td class="colsep0 rowsep0" align="center">6640<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">604<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">>1000</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">At least four different concentrations of test compound spanning up to four log units were tested. IC<sub>50</sub> values were determined from at least 10 different cells with two to three data points per peptide concentration as previously described.<a onclick="showRef(event, 'ref11 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref13">(11,13)</a></p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Mean Â± SEM, <i>n</i> â¥ 2.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> = 1.</p></div></div></div><div class="NLM_p">AM-6120 was evaluated in cultured dorsal root ganglia (DRG) neurons to evaluate the specificity of block of Na<sub>V</sub>1.7-dependent currents using manual whole cell patch clamp electrophysiology in a native setting. In mouse DRG neurons, the peptide was a potent inhibitor of TTX-sensitive (TTX-S) currents mediated in part by native Na<sub>V</sub>1.7 with an IC<sub>50</sub> of 5.1 nM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), while there was no effect observed on TTX-resistant (TTX-R) currents, mainly carried by Na<sub>V</sub>1.8, (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) at 0.5 Î¼M.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Potent inhibition of TTX-S Na<sub>V</sub> channel currents by AM-6120 but not TTX-R Na<sub>V</sub> currents in mouse DRG neurons. (A) Representative whole cell patch clamp recordings for AM-6120 showing block of TTX-S Na<sub>V</sub> channels. TTX was used as a positive control. (B) Representative whole cell patch clamp recordings for AM-6120 showing no block of TTX-R currents at 0.5 Î¼M. Recordings were performed in the presence of 500 nM TTX to block endogenous TTX-S currents. The positive control Na<sub>V</sub>1.8 blocker 5-(4-chlorophenyl)-<i>N</i>-((2-(2,2,2-trifluoroethoxy)pyridin-3-yl)methyl)nicotinamide (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf" class="ext-link">Figure S1</a>) fully blocked TTX-R currents at 0.5 Î¼M. (C) IC<sub>50</sub> values from above experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Racemic X-ray Crystallography</h3><div class="NLM_p">At the time of this work, there were no published reports of an X-ray crystal structure of a Na<sub>V</sub>1.7-inhibitory venom-derived peptide. Utilization of racemic and quasiracemic peptide mixtures has been widely used to overcome the challenges in crystallization of small peptides.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Using a similar approach, we synthesized and folded the selenomethionine analogue of AM-6120, peptide <b>42</b>, Glu-[Pra1;SeMet6;Ala12;5-BrW24;Glu28]JzTx-V(1â29), and the enantiomeric âall-Dâ version of AM-6120, peptide <b>43</b>, and solved the high resolution structure by the direct method. The quasiracemic mix of <b>42</b> and <b>43</b> crystallized in space group <i>P</i>312. One pair of each peptide was observed in the asymmetric unit (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Only the <span class="smallcaps smallerCapital">l</span>-peptide is used for description of the structure.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of JzTx-V-derived Na<sub>V</sub>1.7-inhibitory peptide AM-6120 solved at 1.05 Ã resolution using racemic crystallography. (A) Asymmetric unit showing one molecule of <span class="smallcaps smallerCapital">l</span>-peptide (blue) and one molecule of the <span class="smallcaps smallerCapital">d</span>-peptide (green). Disulfide bonds are in yellow; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CNU">6CNU</a>. (B) Only the <span class="smallcaps smallerCapital">l</span>-peptide is shown. Disulfide connectivity is labeled, and residues important for Na<sub>V</sub>1.7 potency are shown in salmon in stick representation. (C) Na<sub>V</sub>1.7 binding surface comparison of the SeMet6 analogue of AM-6120, peptide <b>42</b> (left) and NMR structure of peptide <b>1</b> (right). Key residues are shown as sticks and cluster on one side of the peptide structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Peptide <b>42</b> adopts a typical inhibitory cystine knot (ICK) motif. Three antiparallel Î²-strands form the core of the motif with three pairs of disulfide bonds stabilizing the loops (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Residues important for Na<sub>V</sub>1.7 binding cluster on one face of the peptide (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). While the overall fold is similar to the NMR solution structure obtained for peptide <b>1</b>, there are conformational differences at both N- and C- termini, as well as all three loops. Of note, the N-terminus and the C-terminus of the NMR structure and crystal structures show the greatest differences, probably due to variance from crystal packing effects. The conformational differences within all three loops reflect the flexible nature of these segments. Nevertheless, from a medicinal chemistry design perspective, it was pleasing to observe similar localization of the key binding residues of the two peptides, despite multiple changes to the primary sequence as part of the optimization process and different structure determination techniques (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Mouse Pharmacokinetics</h3><div class="NLM_p">We examined pharmacokinetic profiles of 5-Br-Trp24 analogues AM-0714 and AM-6120 in C57Bl/6 mice (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The peptides were administered subcutaneously, and systemic plasma levels were monitored by LC-MS/MS.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The overall profiles were similar, with maximal plasma concentrations observed at 30 min postdosing and half-lives of 2.4 h for AM-0714 and 5 h for AM-6120.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Mouse pharmacokinetic profiles of 5-Br-Trp24 containing JzTx-V peptides AM-0714 and AM-6120 in male C57Bl/6 mice (<i>n</i> = 3/time point). The peptides were dosed subcutaneously at 2 mg/kg and monitored by LC-MS/MS. Data are mean Â± SEM for each time point. (B) Mouse pharmacokinetic parameters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> In Vivo Pharmacology</h3><div class="NLM_p">Our initial SAR efforts had not identified pharmacodynamically active GpTx-1 analogues at well-tolerated systemic doses (data not shown). Optimization of JzTx-V peptides using a Na<sub>V</sub>1.4 selectivity-enhancing Ile28Glu modification afforded a series of potent (subnanomolar Na<sub>V</sub>1.7 IC<sub>50</sub>) and selective peptides, exemplified by AM-8145, Pra-[Nle6;Glu28]JzTx-V.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Unfortunately, AM-8145 had no effect on a Na<sub>V</sub>1.7-dependent histamine-induced pruritis model in mice<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> at 1 mg/kg after subcutaneous administration (data not shown), the highest dose with no locomotor effects in an open field assay. The plasma concentration at the time of assay was 6.5-fold over the in vitro mNa<sub>V</sub>1.7 IC<sub>50</sub> of 9 nM, below target coverage multiples shown to be desirable in this assay with small molecule Na<sub>V</sub>1.7 inhibitors.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> To assess if further optimized versions of tarantula-derived toxins could engage the target ion channel in vivo after systemic administration, peptide AM-6120 was evaluated in the histamine-induced pruritis model. AM-6120, administered subcutaneously to C57Bl/6 mice 2 h before an intradermal histamine challenge, produced a robust reduction in scratching at a 2 mg/kg dose, similar to that observed with the antihistamine diphenhydramine dosed orally at 30 mg/kg (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The dose response was steep, and analysis of peptide plasma concentration at the efficacious dose indicated the need for target coverage over 100-fold the in vitro mouse Na<sub>V</sub>1.7 IC<sub>50</sub>. This dose did not lead to any significant reduction in activity in a separate open field activity study in naive mice (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf" class="ext-link">Supporting Information</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Reduction of scratch bouts in a mouse histamine-induced pruritis model with vehicle, AM-6120, and DPH (diphenhydramine, 30 mg/kg po dosing) (male C57Bl/6 mice, <i>n</i> = 7â10/group). ****, <i>p</i> < 0.0001 versus vehicle group (One-Way ANOVA followed by Dunnettâs tests). (B) Plasma exposure levels (Â±SEM) of AM-6120 and calculated exposure multiples over the mouse Na<sub>V</sub>1.7 PX IC<sub>50</sub> (5.1 nM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46132" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46132" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We and others have reported the discovery of naturally occurring peptides from tarantula venom that inhibit the voltage-gated ion channel Na<sub>V</sub>1.7. These natural products offer the promise of pharmaceutical intervention in a setting of high unmet medical need, viz. non-opioid pain medications. However, it has been challenging to investigate these natural inhibitors in preclinical pharmacodynamic studies. First, challenges with peptide biodistribution and the ability of the peptide to access target tissue sites were revealed in an impactful ex vivo study by Priest and co-workers with the Na<sub>V</sub>1.7-inhibitory tarantula toxin peptide ProTx-II which did not block action potential firing unless the saphenous nerve was first desheathed to enable peptide access.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Second, a key liability of these molecules is target specificity and in vivo tolerability.<a onclick="showRef(event, 'ref7 ref15'); return false;" href="javascript:void(0);" class="ref ref7 ref15">(7,15)</a> Such Na<sub>V</sub>1.7 inhibitory peptides have been evolved to block insect sodium channels quickly and are not inherently specific for higher mammalian isoforms like human Na<sub>V</sub>1.7. It has been challenging to separate biodistribution challenges from systemic tolerability using natural product scaffolds, limiting progress toward drug development. Interestingly, it has been recently shown that local and intrathecal administration of toxin peptides and variants do impact pharmacodynamic end points preclinically, suggesting successful target engagement when peptides can access nerve tissue.<a onclick="showRef(event, 'ref7 ref8 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref10">(7,8,10)</a></div><div class="NLM_p">We strived to distinguish target access from tolerability by engineering selectivity enhancements on natural scaffolds by using peptide medicinal chemistry SAR to identify analogues suitable for validating pharmacodynamic studies via systemic administration. Selectivity was sought against related sodium channels including Na<sub>V</sub>1.4, Na<sub>V</sub>1.5, and Na<sub>V</sub>1.6 that play roles in skeletal muscle, heart, and peripheral nerve function, respectively. To identify a peptide that could be dosed in vivo to larger exposure multiples, we initiated SAR studies to modify AM-8145 while maintaining the sub-nM Na<sub>V</sub>1.7 IC<sub>50</sub>. We used molecular docking to visualize key peptide residues that potentially contact the target binding site on the Na<sub>V</sub>1.7 domain II voltage sensor. Docking studies using a NMR structural ensemble of a JzTx-V peptide and a Na<sub>V</sub>1.7 homology model, with attribute-based scan data of single substitution peptide analogues<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> as a guide, suggested that further investigation into nonpolar JzTx-V residues Leu23, Ile29, and Trp5, 7, and 24 in optimizing target binding was warranted.</div><div class="NLM_p">Electrophysiology testing of folded peptide analogues with nonpolar modifications at the indicated positions suggested more complex and subtle roles for each amino acid side chain in the interaction with Na<sub>V</sub>1.7. Leu and Ile have very similar hydrophobic accessible surface areas,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> but different effects were observed in SAR at Leu23 and Ile29. Position 23 preferred an alicyclic modification like Chg. However, position 29, the last amino acid in the JzTx-V sequence, was widely tolerant of alicyclic and aromatic replacements. This data is consistent with our binding model (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) where Leu23 is pointed deep into the Na<sub>V</sub>1.7 VSD pocket while Ile29 is oriented toward the lipid bilayer and could conceivably accommodate larger variations in side chain shape and size. SAR studies at the three Trp residues (5, 7 and 24) also suggested varying room for replacing the side chain indole moiety with Trp5 and more especially Trp7 being sensitive to modifications. Interestingly, Trp24 substitution with a bromine resulted in potency variations specific to the exact position of the substituent on the indole moiety, suggesting preferable interactions with the Na<sub>V</sub>1.7 VSD for the 5-Br-Trp24 analogue. Intriguingly, the optimal modifications did not have additive or synergistic effects on Na<sub>V</sub>1.7 inhibition, leading us to focus our attention on only 5-Br-Trp24 peptides. Encouragingly, the modest gain in Na<sub>V</sub>1.7 potency with this modification did not impact overall sodium channel selectivity, although there was a gain in absolute potency at both TTX-S and TTX-R sodium channels compared to AM-8145. Further analoguing was undertaken to examine the effect on Na<sub>V</sub> selectivity, at the termini and positions suggested by single mutant data, 11, 12, and 14. Introduction of a carboxylate functionality to the peptide termini had differing effects on Na<sub>V</sub>1.7 inhibition. Like toxin peptides GpTx-1<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and HwTx-IV,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> converting the C-terminus from carboxamide to carboxylic acid led to a 10-fold loss in AM-8145 potency. Conversely, the carboxylate side chain containing Glu residue was tolerated at the N-terminus. Our binding model suggests that the C-terminus is near the S3âS4 loop of domain II VSD of Na<sub>V</sub>1.7 and the surrounding lipid bilayer and therefore introducing a carboxylic acid functionality might be disfavored in that environment. The peptide N-terminus points away from the ion channel and may allow better compatibility with a carboxylate side chain. We tested Ala and Glu modifications in the residue 11â14 sector leaving Arg13 unchanged because it was important for activity and observed potent Na<sub>V</sub>1.7 inhibition especially with substitutions of Lys12. This process culminated in the identification of JzTx-V analogue AM-6120 with >100-fold Na<sub>V</sub> subtype selectivity, especially against Na<sub>V</sub>1.5, Na<sub>V</sub>1.6, and Na<sub>V</sub>1.8. In manual whole cell patch clamp electrophysiology studies with native mouse DRG neuronal cells, AM-6120 demonstrated similar potency and selectivity for blocking TTX-S over TTX-R currents that was observed in transfected HEK cells.</div><div class="NLM_p">AM-6120 has six modifications from the 29-residue native JzTx-V sequence including three non-natural amino acids, each selection emerging from different aspects of peptide engineering (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The two glutamic acid and alanine substitutions improved selectivity, propargylglycine afforded a chemical handle for potential derivatizations,<a onclick="showRef(event, 'ref12 ref26'); return false;" href="javascript:void(0);" class="ref ref12 ref26">(12,26)</a> the 5-Br-Trp residue improved potency and norleucine imparted chemical stability. To evaluate the impact of these extensive changes on peptide conformation, we solved the X-ray crystal structure of AM-6120 by the technique of racemic crystallization. The overall ICK folded structure and putative binding interface was comparable to the native toxin as suggested by the similar Na<sub>V</sub>1.7 potency. Our structural insights from the docking of peptide <b>1</b> to the Na<sub>V</sub>1.7 VSD homology model rationalized positions within the peptide that were amenable to empirical SAR for optimization of the interaction without disruption of the backbone conformation. Interestingly, the crystal structure of the peptide confirmed the relative orientation of the 5-Br-Trp24 side chain with the bromine potentially pointing in the direction of the bottom of the pocket in our putative binding model. In light of the recent high resolution Na<sub>V</sub>PaS<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and EeNa<sub>V</sub>1.4-Î²1<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> data, future refinement of these models is warranted and may provide additional insight into the mechanism of interaction between peptide and channel. Other than the recently published structure of ProTx-II by Wright et al.,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> this report is the only known crystal structure of a Na<sub>V</sub>1.7-inhibitory peptide derived from a natural toxin scaffold.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Linear amino acid sequences of peptides, highlighting variations from native JzTx-V in red font. The six cysteines are linked C<sub>1</sub>âC<sub>4</sub>, C<sub>2</sub>âC<sub>5</sub>, and C<sub>3</sub>âC<sub>6</sub> in an inhibitory cystine knot motif. (B) Chemical structure of lead peptide AM-6120 with the six residues different from native JzTx-V highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The 5-Br-Trp24 modified peptides AM-0714 and AM-6120 were evaluated in mouse pharmacokinetic studies. Following a subcutaneous 2 mg/kg dose, both peptides showed high initial exposures followed by clearance with half-lives in the 2â5 h range, a reasonable pharmacokinetic profile for peptides not subjected to rapid proteolysis.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> These data enabled the design of a study to evaluate effects of the more selective peptide AM-6120 in behavioral end-points impacted by Na<sub>V</sub>1.7 inhibition in vivo.</div><div class="NLM_p">Na<sub>V</sub>1.7 mediates pruriceptive sensory information transduction, as shown by human genetic evidence that SCN9A gain-of-function mutations led to paroxysmal itch, as well as the lack of scratching observed in Na<sub>V</sub>1.7 knockout mice in response to histamine.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Inhibition of scratching following an intradermal challenge with histamine has been utilized to test pharmacological effects of Na<sub>V</sub>1.7 inhibitors in mice<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and represents a translational model for clinical studies. AM-6120 robustly blocked histamine induced scratching in mice at a 2 mg/kg dose that was free of any locomotor effects in a separate open field assay. Interestingly, plasma exposures from this study indicated that a 100-fold coverage over the in vitro mouse Na<sub>V</sub>1.7 IC<sub>50</sub> was needed to elicit this response. The improved off-target selectivity of this peptide enabled in vivo testing at higher exposure multiples compared to 6â7-fold achieved with AM-8145 at its highest tolerated dose of 1 mg/kg or lower expected multiples from testing GpTx-1<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and ProTx-II<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> at their highest tolerated systemic dose of 0.1 mg/kg, all of which lacked in vivo pharmacodynamic efficacy. This data suggests that for successful Na<sub>V</sub>1.7 target engagement using systemic administration, peptidic inhibitors may need high plasma target coverage, e.g., â¥100-fold in vitro IC<sub>50</sub> values. This could be due to multiple reasons. Similar to small molecule Na<sub>V</sub>1.7 inhibitors,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> peptides probably need to robustly block Na<sub>V</sub>1.7 currents and action potential firing at peripheral nerve fibers to significantly impact behavior. Additionally, the possibly reduced ability of large molecules like peptides to distribute freely among in vivo compartments may also dictate the need for high exposures in plasma to access the target in peripheral nerve. High plasma coverage multiples have been reported for therapeutic doses of diverse peptidic modulators.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> The target coverage requirements can be reconciled with the ex vivo observations with ProTx-II in desheathed and intact nerves in that a high systemic exposure may partially overcome permeability challenges.</div><div class="NLM_p last">Medicinal chemistry optimization of a disulfide-rich 29-residue peptide, including employing non-natural amino acids, allowed us to probe Na<sub>V</sub>1.7 function in vivo with the tarantula-derived toxin class of Na<sub>V</sub>1.7 inhibitors. Parenthetically, we note that AM-6120 had no effect in a capsaicin-induced nociception model of pain at the 2 mg/kg sc dose (data not shown), suggesting a probable need for even higher target coverage for peptides in that setting. These results support further research into identifying Na<sub>V</sub>1.7 inhibitory peptides with improved potency and selectivity that can impact behavioral end points.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11814" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11814" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Toxin peptides from tarantula venom are a rich source of leads for inhibiting the voltage-gated sodium channel Na<sub>V</sub>1.7, but preclinical pharmacodynamic activity after systemic administration has not been reported. Potential challenges include suboptimal biodistribution and tolerability properties of these natural products. To identify leads that could be dosed in Na<sub>V</sub>1.7 target engagement studies at high plasma concentrations to access target tissues, an optimization campaign was initiated on the potent JzTx-V toxin-derived peptide AM-8145. Using positional scanning and molecular docking to focus SAR studies on peptide residues that contact the target binding site, we discovered a 5-Br-Trp24 modification which was optimized to deliver potent (sub-nM Na<sub>V</sub>1.7 IC<sub>50</sub>) and selective (>100Ã Na<sub>V</sub> subtype selectivity) lead peptide AM-6120 that was characterized further in neuronal cells, X-ray crystallography, and pharmacokinetic studies. In a Na<sub>V</sub>1.7-dependent behavioral model of target engagement, AM-6120 robustly blocked histamine-induced scratching in mice after subcutaneous administration. Lead optimization of JzTx-V to AM-6120 resulted in a 20% change in the native toxin sequence, including noncanonical amino acids. A plasma drug concentration of 100-fold over the in vitro IC<sub>50</sub> was required to block Na<sub>V</sub>1.7 dependent behavior following systemic subcutaneous dosing, suggesting high exposure thresholds may be needed to impact this ion channel in vivo using peptides and offering guidance for future inhibitor design.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37724" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37724" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Materials</h3><div class="NLM_p last"><i>N</i><sup>Î±</sup>-Fmoc protected amino acids were purchased from Novabiochem (San Diego, CA), Bachem (Torrance, CA), ChemPep (Wellington, FL), or GL Biochem (Shanghai, China). Rink Amide MBHA resin was purchased from Peptides International (Louisville, KY). SP Sepharose high performance resin was purchased from GE Healthcare Life Sciences. The following compounds were purchased: <i>N</i>,<i>N</i>-diisopropylethylamine (DIEA), trifluoroacetic acid (TFA), acetic acid, piperidine, 4-methylpiperidine, 3,6-dioxa-1,8-octanedithiol (DODT), triisopropylsilane, oxidized glutathione, and reduced glutathione (Sigma-Aldrich, Milwaukee, WI); dichloromethane (DCM, Mallinckrodt Baker, Inc.); <i>N</i>,<i>N</i>-dimethylforamide (DMF, Fisher Scientific); 6-chloro-1-hydroxy benzotriazole (6-Cl HOBt, Matrix Innovation); HPLC quality water and acetonitrile (Burdick and Jackson); 1.0 M Tris-HCl pH 7.5 (Teknova). Stable cell lines expressing human voltage-gated sodium (Na<sub>V</sub>) channels (HEK293-hNa<sub>V</sub>1.4, HEK293-hNa<sub>V</sub>1.5, HEK293-hNa<sub>V</sub>1.7, and CHO-hNa<sub>V</sub>1.8) and mouse voltage-gated sodium (Na<sub>V</sub>) channel (HEK293-mNa<sub>V</sub>1.7)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> were used for experiments. The purity of synthesized compounds was determined by LC-MS analysis: Elution from a Phenomenex Max-RP column (2.5 Î¼m, 2.0 mm Ã 50 mm) using a gradient of 5â50% acetonitrile in H<sub>2</sub>O (0.1%TFA) over 10 min at a flow rate of 750 Î¼L/min on an Agilent 1290 LC-MS system. All compounds for biological testing were â¥95% pure.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Parallel Peptide Synthesis</h3><div class="NLM_p last">Peptides were assembled using <i>N</i><sup>Î±</sup>-Fmoc solid-phase peptide synthesis methodologies with appropriate orthogonal protection and resin linker strategies as described previously.<a onclick="showRef(event, 'ref11 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref13">(11,13)</a> The peptides were synthesized on a 96 Ã 0.0125 mmol scale using Rink Amide MBHA resin (100â200 mesh, 1% DVB, 0.52 mol equiv/g initial loading, Peptides International, Louisville, KY). Dry resin (25 mg per well resin loader, 28 mg per well actual) was added to two Phenomenex deep-well protein precipitation plates (CEO-7565, 38710-1) using a resin loader (Radley). Amino acids were added to the growing peptide chain by stepwise addition using standard solid-phase methods on an automated peptide synthesizer (Intavis Multipep). Amino acids (5 mol equiv, 120 Î¼L) were preactivated (1 min) with 6-chloro-1-hydroxybenzotriazole (6-Cl-HOBt, Matrix Innovation, 5 mol equiv, 0.4 M, 150 Î¼L) in DMF and 1,3-diisopropylcarbodiimide (DIC, 5 mol equiv, 1M, 60 Î¼L). Preactivated amino acids (5 min) were transferred to the appropriate well. Resins were incubated for 60 min, drained, and the cycle repeated. Following the second amino acid incubation, the plates were drained and washed with DMF eight times (3 mL per column of eight wells). The Fmoc groups were then removed by two sequential incubations in 500 Î¼L of a 20% piperidine in DMF solution. The first incubation was 5 min, the resin was drained, and the second incubation was for 15 min. The resin was again drained and washed with DMF 10 times (3 mL per column). The Fmoc protecting group was removed from the N-terminus as described above and the resin was washed with DCM 5 times (3 mL per column) and allowed to air-dry.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Parallel Peptide Cleavage</h3><div class="NLM_p last">A drain port sealing mat (ArcticWhite, AWSM-1003DP) was affixed to the bottom of the filter plate. To the resin in each well were added triisopropylsilane (100 Î¼L), 3,6-dioxa-1,8-octane-dithiol (DODT, 100 Î¼L), and water (100 Î¼L). To the resin in each well was added TFA (1.2 mL). The top of the plate was covered with another drain port sealing mat. The mixture was agitated on a plate shaker for 2 h. The sealing mat was removed, and the cleavage solution was eluted into a solid bottom 24-well deep well plate. The bottom of the filter plate was affixed with a drain port sealing mat, and the resin in each well was washed with an additional 1 mL of TFA. The mixture was agitated on a plate shaker for 10 s, then the drain port sealing mat was removed and the TFA was drained into the solid bottom 24-well deep well plate. The solutions were transferred to 50 mL centrifuge tubes and concentrated using rotary evaporation (Genevac) for 2 h. To each tube was added 30 mL of cold diethyl ether, following which a white precipitate was observed. The mixture was centrifuged, and the ether was decanted. An additional 30 mL of cold ether was added to wash, and the mixture was vortexed. The mixture was centrifuged, and the ether was decanted. The tubes were allowed to air-dry overnight.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Individual Peptide Synthesis</h3><div class="NLM_p last">Rink Amide Chem Matrix resin (0.2 mmol, 0.45 mmol/g loading, 0.444 g, Matrix Innovation) was weighed into a CS BIO reaction vessel. The reaction vessel was connected to a channel of the CS BIO 336X automated peptide synthesizer, and the resin was washed 2Ã DMF and allowed to swell in DMF for 15 min. Fmoc-amino acids (1.0 mmol, Midwest Biotech or Novabiochem) were dissolved with 6-chloro-1-hydroxybenzotriazole (6-Cl-HOBt) in DMF (0.4 M, 2.5 mL). To the solution was added 1,3-diisopropylcarbodiimide (DIC, Sigma-Aldrich) in DMF (1.0 M, 1.0 mL). The solution was agitated with nitrogen bubbling for 15 min to accomplish preactivation and then added to the resin. The mixture was shaken for 2 h. The resin was filtered and washed with DMF (3Ã), DCM (2Ã), and DMF (3Ã). Fmoc-removal was accomplished by treatment with 20% piperdine in DMF (5 mL, 2 Ã 15 min, Fluka). The resin was filtered and washed with DMF (3Ã). All residues were single coupled through repetition of the Fmoc-amino acid coupling and Fmoc removal steps described above.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Individual Peptide Cleavage</h3><div class="NLM_p last">After final Fmoc-removal from the N-terminal residue, resin-bound linear peptide (0.2 mmol scale) was transferred to a 25 mL solid-phase extraction (SPE) filter tube, washed with DMF (3 times) and DCM (3 times), and dried under vacuum. To the resin was added triisopropylsilane (1.0 mL), 3,6-dioxa-1,8-octane-dithiol (DODT, 1.0 mL), water (1.0 mL), trifluoroacetic acid (TFA, 15 mL), and a stir bar, and the mixture was stirred for 3 h. The mixture was filtered into a 50 mL centrifuge tube. The resin was washed with TFA (5 mL), and the combined filtrate was concentrated by rotary evaporation in Genevac for 2 h. To the residue (5 mL) was added 40 mL of cold diethyl ether, and a white precipitate formed. The mixture was centrifuged, and the ether was decanted. Another 40 mL of cold ether was added, and the precipitate was stirred. The mixture was centrifuged, and the ether was decanted. The solid was dried under vacuum.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Linear Peptide Purification and Folding</h3><div class="NLM_p last">The crude linear peptide was purified by preparative LC-MS. The filtered sample (300 mg in 5 mL of DMSO) was purified with preparative HPLC column (Phenomenex Synergi 4 Î¼m MAX-RP 80 Ã AXIA, 250 mm Ã 30 mm). The fractions were analyzed by LC-MS, pooled, and lyophilized to afford pure linear peptide precursor. In a 1 L polypropylene bottle was prepared a folding buffer with water (800 mL), acetonitrile (100 mL), cysteine solution in water (1 M, 1 mL), and cystine dihydrochloride in water (0.15 M, 6.667 mL). To the pure linear peptide (100 mg) was added 5 mL of acetonitrile and 5 mL of water. The mixture was vortexed to complete dissolution of the peptide. The peptide solution was added to the buffer followed by Tris-HCl pH 8.0 (1M, 100 mL). The pH value was measured to be 8.0. The folding mixture was allowed to stand at 4 Â°C for 18â72 h. A small aliquot was removed, and the sample was analyzed by LC-MS to ensure that the folding was complete. The solution was quenched by the addition of TFA to pH 2.5, and the aqueous solution was filtered. The filtered solution (1 L, 100 mg peptide) was loaded onto a preparative HPLC column (Phenomenex Synergi 4 Î¼m MAX-RP 80 Ã AXIA, 250 mm Ã 30 mm) using an Agilent preparative loading pump. The column was attached to a prep HPLC, Agilent/LEAP prep LC-MS, and the peptide was eluted to afford pure folded peptide.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Crude Peptide Folding and Purification</h3><div class="NLM_p last">In a 1 L polypropylene bottle was prepared a folding buffer with water (800 mL), acetonitrile (100 mL), cysteine solution in water (1 M, 1 mL), and cystine dihydrochloride in water (0.15 M, 6.667 mL). To the crude linear peptide (100 mg) was added 5 mL of acetonitrile and 5 mL of water. The mixture was vortexed to complete dissolution of the peptide. The peptide solution was added to the buffer followed by Tris-HCl pH 8.0 (1M, 100 mL). The pH value was measured to be 8.0. The folding mixture was allowed to stand at 4 Â°C for 18â72 h. A small aliquot was removed, and the sample was analyzed by LC-MS to ensure that the folding was complete. The solution was quenched by the addition of TFA to pH 2.5, and the aqueous solution was filtered. The filtered solution (1000 mL, 100 mg peptide) was loaded onto a preparative HPLC column (Phenomenex Synergi 4 Î¼m MAX-RP 80 Ã AXIA, 250 mm Ã 30 mm) using an Agilent preparative loading pump. The column was attached to a prep HPLC, Agilent/LEAP prep LC-MS, and the peptide was eluted to afford pure folded peptide. The final yields varied with different sequences and generally range from 5 to 10%.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Electrophysiology</h3><div class="NLM_p last">All automated and manual patch clamp electrophysiology experiments were conducted as previously described, and IC<sub>50</sub> values were determined from at least 10 different cells with two to three data points per peptide concentration.<a onclick="showRef(event, 'ref11 ref27'); return false;" href="javascript:void(0);" class="ref ref11 ref27">(11,27)</a> All cell lines were validated by RT-PCR to express the indicated Na<sub>V</sub> isoform and mycoplasma free. Na<sub>V</sub> current inhibition was determined after 5 min of peptide addition, at which point inhibition reached near steady-state levels for concentrations at or above IC<sub>50</sub> values. For all PatchXpress experiments, cells were not reused for subsequent testing if currents did not return to >80% starting levels following peptide washout. In addition, steady-state inactivation curves were determined following washout of each concentration of peptide and preceding subsequent concentrations tested to ensure the same level of Na<sub>V</sub> channel inactivation prior to peptide addition.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> PatchXpress 7000A Electrophysiology</h3><div class="NLM_p last">Adherent cells were isolated from tissue culture flasks using 1:10 diluted 0.25% trypsinâEDTA treatment for 2â3 min and then were incubated in complete culture medium containing 10% fetal bovine serum for at least 15 min prior to resuspension in external solution consisting of 70 mM NaCl, 140 mM <span class="smallcaps smallerCapital">d</span>-mannitol, 10 mM HEPES, 2 mM CaCl<sub>2</sub>, and 1 mM MgCl<sub>2</sub>, pH 7.4, with NaOH. Internal solution consisted of 62.5 mM CsCl, 75 mM CsF, 10 mM HEPES, 5 mM EGTA, and 2.5 mM MgCl<sub>2</sub>, pH 7.25, with CsOH. Cells were voltage clamped using the whole-cell patch clamp configuration at room temperature (â¼22 Â°C) at a holding potential of â125 mV with test potentials to â10 mV (hNa<sub>V</sub> 1.4, hNa<sub>V</sub>1.6, and hNa<sub>V</sub>1.7), â20 mV (hNa<sub>V</sub>1.5). To record from partially inactivated channels, currents were recorded with a holding voltage that yielded â¼20% channel inactivation, calculated automatically for each individual cell. Test compounds were added, and Na<sub>V</sub> currents were monitored at 0.1 Hz at the appropriate test potential. All compound dilutions contained 0.1% bovine serum albumin to minimize nonspecific binding. Cells were used for additional compound testing if currents recovered to >80% of starting values following compound washout. At least four different concentrations of test compound at half log units were applied individually, with washout, recovery of current, and resetting of holding voltage between each individual concentration. Percent inhibition as a function of compound concentration was pooled from at least <i>n</i> = 10 different cells, with two to three data points per concentration, and fitting the resulting data set with a Hill (four-parameter logistic) fit in DataXpress 2.0 software to produce a single IC<sub>50</sub> curve.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Whole-Cell Patch Clamp Electrophysiology</h3><div class="NLM_p last">Cells were voltage clamped using the whole-cell patch clamp configuration at room temperature (â¼22 Â°C). Pipette resistances were between 1.5 and 2.0 MÎ©. Whole-cell capacitance and series resistance were uncompensated. Currents were digitized at 50 kHz and filtered (4-pole Bessel) at 10 kHz using pClamp10.2. Cells were lifted off the culture dish and positioned directly in front of a micropipette connected to a solution exchange manifold for compound perfusion. To record from partially inactivated channels, cells were held at â140 mV until currents stabilized and then switched to a voltage that yielded â¼20% channel inactivation. Then 10 ms pulses were delivered every 10 s and peak inward currents were recorded before and after compound addition. Compound dilutions contained 0.1% bovine serum albumin to minimize nonspecific binding. For hNa<sub>V</sub>1.8 channel recordings, tetrodotoxin (TTX, 0.5 uM) was added to inhibit endogenous TTX-sensitive voltage-gated sodium channels and record only Na<sub>V</sub>1.8-mediated TTX-resistant currents. External solution consisted of: 140 mM NaCl, 5.0 mM KCl, 2.0 mM CaCl<sub>2</sub>, 1.0 mM MgCl<sub>2</sub>, 10 mM HEPES, and 11 mM glucose, pH 7.4, by NaOH. Internal solution consisted of: 62.5 mM CsCl, 75 mM CsF, 2.5 mM MgCl<sub>2</sub>, 5 mM EGTA, and 10 mM HEPES, pH 7.25, by CsOH. Escalating compound concentrations were analyzed on the same cell, and IC<sub>50</sub> values were calculated with Clampfit 10.2 software and by fitting the resulting data set with a Hill (four-parameter logistic) fit in Origin Pro 8 software.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> DRG Neuron Isolation and Manual Patch Clamp Electrophysiology</h3><div class="NLM_p last">DRG from cervical, thoracic, and lumbar regions of adult male and female C57BL/6 mice were removed, placed in Ca<sup>2+</sup> and Mg<sup>2+</sup>-free Hanksâ Balanced Salt Solution (Invitrogen, Carlsbad, CA), and trimmed of attached fibers under a dissecting microscope. DRG were sequentially digested at 37 Â°C with papain (20 U/mL, Worthington Biochemical Corporation, Lakewood, NJ), <span class="smallcaps smallerCapital">l</span>-cysteine (25 Î¼M) in Ca<sup>2+</sup> and Mg<sup>2+</sup>-free Hanksâ (pH 7.4) for 20â30 min, and then with collagenase type 2 (0.9% w/v, Worthington Biochemical Corporation) for 20â30 min. Digestions were quenched with a 1:1 mixture of DMEM and Hamâs F-12 nutrient mixture (Invitrogen) supplemented with 10% calf serum (Invitrogen), and cells were triturated with a fire-polished Pasteur pipet prior to plating on poly-<span class="smallcaps smallerCapital">d</span>-lysine-coated glass coverslips (Cole-Parmer, Vernon Hills, IL). Cells were maintained in a humidified incubator at 28 Â°C with 5% CO<sub>2</sub> for 3â7 days in the presence of 1% NSF-1 (Lonza, Basel, Switzerland) to increase the expression of tetrodotoxin-sensitive sodium channel currents. DRG neurons were voltage clamped using the whole-cell patch clamp configuration at room temperature (21â24 Â°C) using an Axopatch 200 B or MultiClamp 700 B amplifier and DIGIDATA1322A with pCLAMP software (Molecular Devices, Sunnyvale, CA). Pipettes, pulled from borosilicate glass capillaries (World Precision Instruments, Sarasota, FL), had resistances between 1.0 and 3.0 MÎ©. Voltage errors were minimized using >80% series resistance compensation. A P/4 protocol was used for leak subtraction. Currents were digitized at 50 kHz and filtered (four-pole Bessel) at 10 kHz. Cells were lifted off the culture dish and positioned directly in front of a micropipette connected to a solution exchange manifold for compound perfusion. Cells were held at a voltage yielding approximately 20% inactivation and depolarized to â10 mV for 40 ms every 10 s. Tetrodotoxin (TTX, Sigma) was used following peptide addition to block any residual TTX-sensitive sodium currents. Pipette solution contained 62.5 mM CsCl, 75 mM CsF, 2.5 mM MgCl<sub>2</sub>, 5 mM EGTA, and 10 mM HEPES, pH 7.25, by CsOH. Bath solution contained 70 mM NaCl, 5.0 mM KCl, 2.0 mM CaCl<sub>2</sub>, 1.0 mM MgCl<sub>2</sub>, 10 mM HEPES, 11 mM glucose, and 140 mM <span class="smallcaps smallerCapital">d</span>-mannitol, pH 7.4, by NaOH. Data were analyzed with Clampfit and Origin Pro 8 (OriginLab Corp, Northampton, MA).</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> X-ray Crystallography</h3><div class="NLM_p">Equimolar ratios of racemic peptides (<b>42</b> and <b>43</b>) dissolved in water was mixed and centrifuged to remove any particulate matter before crystallizations. Crystallization screening was conducted at 20 Â°C by sitting drop vapor diffusion method using commercial crystallization screens. Large single crystals were grown under 0.1 mM sodium acetate, pH 3.6â5.6, and 1.5â2.0 M ammonium sulfate. For data collection, crystals were briefly transferred to the cryoprotectant (reservoir solution plus 20% (v/v) glycerol) and flash-frozen in liquid nitrogen. Data were collected at beamline 22ID at the Argonne National Laboratory (Chicago, IL).</div><div class="NLM_p last">The diffraction data were integrated using program XDS<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and scaled using the program Aimless<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> in CCP4 package.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The structure was solved by direct method using the ACORN program<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> in the CCP4 package. Model building was carried out in COOT,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and refinement was done in REFMAC5<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> in the CCP4 program suite. All structural figures were prepared using Pymol (Schrodinger Inc. San Diego, CA). The data collection and refinement statistics are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf" class="ext-link">Table S2</a> (Supporting Information).</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Animals and Ethics</h3><div class="NLM_p last">All studies involving the use of animals were approved by Amgenâs Institutional Animal Care and Use Committee in accordance with the National Institutes of Healthâs Guide for the Care and Use of Laboratory Animals and conducted at an AALAC-accredited facility.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Histamine-Induced Scratching in Mice.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></h3><div class="NLM_p last">Subjects were C57Bl/6 male mice (Charles River Laboratories, Franklin, NY) aged between 8 and 10 weeks at the beginning of the study. One day prior to behavioral testing, C57Bl/6 male mice were anesthetized under 3% isoflurane and the area at the nape of the neck was shaved. Immediately afterward, mice were transported to the testing room and acclimated to individual sound-attenuated chambers (12" l Ã 9.5" w Ã 8.25" h, Med Associates VFC-008, NIR-022MD, St. Albans, VT) for 15 min. Testing was performed the following day between the hours of 8 a.m. and 3 p.m. Two hours prior to histamine treatment, mice were subcutaneously administered either AM-6120 (0.5 (<i>n</i> = 8), 1.0 (<i>n</i> = 8), or 2.0 (<i>n</i> = 10) mg/kg body weight), or a vehicle control formulation (PBS with 1% MSA, <i>n</i> = 10). A separate group of animals was orally administered the antihistamine diphenhydramine (30 mg/kg in phosphate-buffered saline; Sigma D3630, <i>n</i> = 7) 90 min prior to testing to serve as a positive control. Histamine dichloride (8.15 mM in a volume of 100 Î¼L; Sigma, H7250, all subjects) was injected intradermally to the shaved area, mice were placed into the sound-attenuated testing chambers, and behavior was recorded on digital video files for a period of 15 min. Video recordings were later reviewed and individual scratching bouts were scored by trained experimenters. A scratching bout was defined as a rapid head tilt accompanied by a hind paw directed at the site of intradermal injection. Termination of a scratching bout was deemed to have occurred when the hind paw was placed back on the chamber floor or into the animalâs mouth. All dosing and scoring activities were conducted by experimenters who were fully blinded to treatment conditions.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00736" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80257" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80257" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00736" class="ext-link">10.1021/acs.jmedchem.8b00736</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Peptide characterization by ESI-MS, chemical structure of positive control Na<sub>V</sub>1.8 blocker, mouse pharmacokinetic study and mouse open field assay data with AM-6120, data collection and refinement statistics for X-ray crystal structures of peptides <b>42</b> and <b>43</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf">jm8b00736_si_005.pdf (991.49 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of the crystal structures of the mixture of peptides <b>42</b> and <b>43</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CNU">6CNU</a>) have been deposited in the Protein Data Bank (<a href="http://www.pdb.org" class="extLink">www.pdb.org</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b00736" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74035" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74035" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bin Wu</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5247-7332" title="Orcid link">http://orcid.org/0000-0002-5247-7332</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dcbeb5b2ab9cbdb1bbb9b2f2bfb3b1"><span class="__cf_email__" data-cfemail="d1b3b8bfa691b0bcb6b4bfffb2bebc">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaustav Biswas</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span>;Â 
    <span>Present Address:
                        For K.B.: Merck Research Laboratories, 33 Avenue Louis Pasteur, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9971-1424" title="Orcid link">http://orcid.org/0000-0001-9971-1424</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f59e948086819483db979c86829486b5989087969edb969a98"><span class="__cf_email__" data-cfemail="4d262c383e392c3b632f243e3a2c3e0d20283f2e26632e2220">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justin K. Murray</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kristin L. Andrews</span> - <span class="hlFld-Affiliation affiliation">Therapeutic Discovery and , and , Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kelvin Sham</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jason Long</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer Aral</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph Ligutti</span> - <span class="hlFld-Affiliation affiliation">Neuroscience, Amgen Research and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shanti Amagasu</span> - <span class="hlFld-Affiliation affiliation">Neuroscience, Amgen Research and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dong Liu</span> - <span class="hlFld-Affiliation affiliation">Neuroscience, Amgen Research and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anruo Zou</span> - <span class="hlFld-Affiliation affiliation">Neuroscience, Amgen Research and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoshan Min</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery, Amgen Research, Amgen Inc., 1120 Veterans Blvd, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhulun Wang</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery, Amgen Research, Amgen Inc., 1120 Veterans Blvd, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher P. Ilch</span> - <span class="hlFld-Affiliation affiliation">Neuroscience, Amgen Research and , and , Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas J. Kornecook</span> - <span class="hlFld-Affiliation affiliation">Neuroscience, Amgen Research and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Min-Hwa Jasmine Lin</span> - <span class="hlFld-Affiliation affiliation">Pharmacokinetics and Drug Metabolism and , and , Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xuhai Be</span> - <span class="hlFld-Affiliation affiliation">Pharmacokinetics and Drug Metabolism and , and , Amgen Inc., 360 Binney Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Les P. Miranda</span> - <span class="hlFld-Affiliation affiliation">Therapeutic
Discovery and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bryan D. Moyer</span> - <span class="hlFld-Affiliation affiliation">Neuroscience, Amgen Research and , Amgen Inc., One Amgen Center Drive, Thousand Oaks, California 91320, United States,</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors were employees of Amgen Inc. at the time this work was performed.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3258-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32017" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32017" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Rob Foti is acknowledged for help with figures and data analysis. X-ray diffraction data were collected at Southeast Regional Collaborative Access Team (SER-CAT) 22-ID (or 22-BM) beamline at the Advanced Photon Source, Argonne National Laboratory. Use of the Advanced Photon Source was supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences, under contract no. W-31-109-Eng-38.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">1-Nal</td><td class="NLM_def"><p class="first last">1-naphthylalanine</p></td></tr><tr><td class="NLM_term">2-Nal</td><td class="NLM_def"><p class="first last">2-naphthylalanine</p></td></tr><tr><td class="NLM_term">5-Br-Trp</td><td class="NLM_def"><p class="first last">5-bromotryptophan</p></td></tr><tr><td class="NLM_term">6-Cl HOBt</td><td class="NLM_def"><p class="first last">6-chloro-1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">Cha</td><td class="NLM_def"><p class="first last">cyclohexylalanine</p></td></tr><tr><td class="NLM_term">Chg</td><td class="NLM_def"><p class="first last">cyclohexylglycine</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DODT</td><td class="NLM_def"><p class="first last">3,6-dioxa-1,8-octanedithiol</p></td></tr><tr><td class="NLM_term">DRG</td><td class="NLM_def"><p class="first last">dorsal root ganglion</p></td></tr><tr><td class="NLM_term">ICK</td><td class="NLM_def"><p class="first last">inhibitory cystine knot</p></td></tr><tr><td class="NLM_term">MAPS</td><td class="NLM_def"><p class="first last">multiattribute positional scan</p></td></tr><tr><td class="NLM_term">Nle</td><td class="NLM_def"><p class="first last">norleucine</p></td></tr><tr><td class="NLM_term">Nva</td><td class="NLM_def"><p class="first last">norvaline</p></td></tr><tr><td class="NLM_term">Pra</td><td class="NLM_def"><p class="first last">propargylglycine</p></td></tr><tr><td class="NLM_term">PX</td><td class="NLM_def"><p class="first last">PatchXpress</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">SPPS</td><td class="NLM_def"><p class="first last">solid-phase peptide synthesis</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TTX-R</td><td class="NLM_def"><p class="first last">tetrodotoxin-resistant</p></td></tr><tr><td class="NLM_term">TTX-S</td><td class="NLM_def"><p class="first last">tetrodotoxin-sensitive</p></td></tr><tr><td class="NLM_term">VSD</td><td class="NLM_def"><p class="first last">voltage sensor domain</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28122" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28122" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volkow, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, F. S.</span></span> <span> </span><span class="NLM_article-title">The role of science in addressing the opioid crisis</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>377</i></span>,  <span class="NLM_fpage">391</span>â <span class="NLM_lpage">394</span>, <span class="refDoi">Â DOI: 10.1056/NEJMsr1706626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1056%2FNEJMsr1706626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28564549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A280%3ADC%252BC1cnjt1yitQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2017&pages=391-394&author=N.+D.+Volkowauthor=F.+S.+Collins&title=The+role+of+science+in+addressing+the+opioid+crisis&doi=10.1056%2FNEJMsr1706626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Science in Addressing the Opioid Crisis</span></div><div class="casAuthors">Volkow Nora D; Collins Francis S</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">391-394</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQwGjdZICWoPiEJCPKhJPuqfW6udTcc2ebKagvGrkrhArntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnjt1yitQ%253D%253D&md5=a6bb07206f42d3bbbcfadb92d2088e29</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMsr1706626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMsr1706626%26sid%3Dliteratum%253Aachs%26aulast%3DVolkow%26aufirst%3DN.%2BD.%26aulast%3DCollins%26aufirst%3DF.%2BS.%26atitle%3DThe%2520role%2520of%2520science%2520in%2520addressing%2520the%2520opioid%2520crisis%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2017%26volume%3D377%26spage%3D391%26epage%3D394%26doi%3D10.1056%2FNEJMsr1706626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D.</span></span> <span> </span><span class="NLM_article-title">The pain drain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">S2</span>â <span class="NLM_lpage">S3</span>, <span class="refDoi">Â DOI: 10.1038/535S2a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2F535S2a" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=S2-S3&author=D.+Holmes&title=The+pain+drain&doi=10.1038%2F535S2a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F535S2a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F535S2a%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DD.%26atitle%3DThe%2520pain%2520drain%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3DS2%26epage%3DS3%26doi%3D10.1038%2F535S2a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starobova, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobli, M.</span></span> <span> </span><span class="NLM_article-title">Na<sub>V</sub>1.7 as a pain target - from gene to pharmacology</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">73</span>â <span class="NLM_lpage">100</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2016.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.pharmthera.2016.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=27916648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislClsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2017&pages=73-100&author=I.+Vetterauthor=J.+R.+Deuisauthor=A.+Muellerauthor=M.+R.+Israelauthor=H.+Starobovaauthor=A.+Zhangauthor=L.+D.+Rashauthor=M.+Mobli&title=NaV1.7+as+a+pain+target+-+from+gene+to+pharmacology&doi=10.1016%2Fj.pharmthera.2016.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">NaV1.7 as a pain target - From gene to pharmacology</span></div><div class="casAuthors">Vetter, Irina; Deuis, Jennifer R.; Mueller, Alexander; Israel, Mathilde R.; Starobova, Hana; Zhang, Alan; Rash, Lachlan D.; Mobli, Mehdi</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73-100</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">NaV1.7, a subtype of the voltage-gated sodium channel family that is highly expressed in peripheral sensory neurons, remains one of the most promising targets for the treatment of pain.  However, despite compelling genetic evidence supporting a key role for NaV1.7 in regulating excitability of peripheral sensory neurons, the development of truly subtype-selective inhibitors has been challenging.  Here, we discuss complexities surrounding targeting NaV1.7 pharmacol. for treatment of pain and explore future opportunities for development of effective analgesic NaV1.7 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgpB1bBe31X7Vg90H21EOLACvtfcHk0lggJBOn4yu7GA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislClsg%253D%253D&md5=78a0e17dfccd84b511491b6c5d83fd94</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2016.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2016.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DMueller%26aufirst%3DA.%26aulast%3DIsrael%26aufirst%3DM.%2BR.%26aulast%3DStarobova%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DRash%26aufirst%3DL.%2BD.%26aulast%3DMobli%26aufirst%3DM.%26atitle%3DNaV1.7%2520as%2520a%2520pain%2520target%2520-%2520from%2520gene%2520to%2520pharmacology%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D172%26spage%3D73%26epage%3D100%26doi%3D10.1016%2Fj.pharmthera.2016.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dib-Hajj, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxman, S. G.</span></span> <span> </span><span class="NLM_article-title">The Na<sub>V</sub>1.7 sodium channel: from molecule to man</span>. <i>Nat. Rev. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">49</span>â <span class="NLM_lpage">62</span>, <span class="refDoi">Â DOI: 10.1038/nrn3404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2Fnrn3404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=23232607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaisrrJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=49-62&author=S.+D.+Dib-Hajjauthor=Y.+Yangauthor=J.+A.+Blackauthor=S.+G.+Waxman&title=The+NaV1.7+sodium+channel%3A+from+molecule+to+man&doi=10.1038%2Fnrn3404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The NaV1.7 sodium channel: from molecule to man</span></div><div class="casAuthors">Dib-Hajj, Sulayman D.; Yang, Yang; Black, Joel A.; Waxman, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-62</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The voltage-gated sodium channel NaV1.7 is preferentially expressed in peripheral somatic and visceral sensory neurons, olfactory sensory neurons and sympathetic ganglion neurons.  NaV1.7 accumulates at nerve fiber endings and amplifies small subthreshold depolarizations, poising it to act as a threshold channel that regulates excitability.  Genetic and functional studies have added to the evidence that NaV1.7 is a major contributor to pain signaling in humans, and homol. modeling based on crystal structures of ion channels suggests an at.-level structural basis for the altered gating of mutant NaV1.7 that causes pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC4lZH1L7XWLVg90H21EOLACvtfcHk0lhcWr4_vOU66A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaisrrJ&md5=0e075d2c82d98774da1d316f4409d3eb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrn3404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn3404%26sid%3Dliteratum%253Aachs%26aulast%3DDib-Hajj%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBlack%26aufirst%3DJ.%2BA.%26aulast%3DWaxman%26aufirst%3DS.%2BG.%26atitle%3DThe%2520NaV1.7%2520sodium%2520channel%253A%2520from%2520molecule%2520to%2520man%26jtitle%3DNat.%2520Rev.%2520Neurosci.%26date%3D2013%26volume%3D14%26spage%3D49%26epage%3D62%26doi%3D10.1038%2Fnrn3404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marron, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prime, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storer, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swain, N. A.</span></span> <span> </span><span class="NLM_article-title">Recent progress in sodium channel modulators for pain</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3690</span>â <span class="NLM_lpage">3699</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2014.06.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.bmcl.2014.06.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=25060923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Wls7vL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3690-3699&author=S.+K.+Bagalauthor=M.+L.+Chapmanauthor=B.+E.+Marronauthor=R.+Primeauthor=R.+I.+Storerauthor=N.+A.+Swain&title=Recent+progress+in+sodium+channel+modulators+for+pain&doi=10.1016%2Fj.bmcl.2014.06.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in sodium channel modulators for pain</span></div><div class="casAuthors">Bagal, Sharan K.; Chapman, Mark L.; Marron, Brian E.; Prime, Rebecca; Storer, R. Ian; Swain, Nigel A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3690-3699</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Voltage-gated sodium channels (Navs) are an important family of transmembrane ion channel proteins and Nav drug discovery is an exciting field.  Pharmaceutical investment in Navs for pain therapeutics has expanded exponentially due to genetic data such as SCN10A mutations and an improved ability to establish an effective screen sequence for example IonWorks Barracuda, Synchropatch and Qube.  Moreover, emerging clin. data (AZD-3161, XEN402, CNV1014802, PF-05089771, PF-04531083) combined with recent breakthroughs in Nav structural biol. pave the way for a future of fruitful prospective Nav drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbT6en4xkVxLVg90H21EOLACvtfcHk0lhcWr4_vOU66A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Wls7vL&md5=c8bae40b70509e2aa3dce106e2c41677</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.038%26sid%3Dliteratum%253Aachs%26aulast%3DBagal%26aufirst%3DS.%2BK.%26aulast%3DChapman%26aufirst%3DM.%2BL.%26aulast%3DMarron%26aufirst%3DB.%2BE.%26aulast%3DPrime%26aufirst%3DR.%26aulast%3DStorer%26aufirst%3DR.%2BI.%26aulast%3DSwain%26aufirst%3DN.%2BA.%26atitle%3DRecent%2520progress%2520in%2520sodium%2520channel%2520modulators%2520for%2520pain%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3690%26epage%3D3699%26doi%3D10.1016%2Fj.bmcl.2014.06.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Lera Ruiz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, R. L.</span></span> <span> </span><span class="NLM_article-title">Voltage-gated sodium channels: structure, function, pharmacology, and clinical Indications</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7093</span>â <span class="NLM_lpage">7118</span>, <span class="refDoi">Â DOI: 10.1021/jm501981g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501981g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFemu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7093-7118&author=M.+de%0ALera+Ruizauthor=R.+L.+Kraus&title=Voltage-gated+sodium+channels%3A+structure%2C+function%2C+pharmacology%2C+and+clinical+Indications&doi=10.1021%2Fjm501981g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Voltage-Gated Sodium Channels: Structure, Function, Pharmacology, and Clinical Indications</span></div><div class="casAuthors">de Lera Ruiz, Manuel; Kraus, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7093-7118</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The tremendous therapeutic potential of voltage-gated sodium channels (Navs) has been the subject of many studies in the past and is of intense interest today.  Nav1.7 channels in particular have received much attention recently because of strong genetic validation of their involvement in nociception.  Here we summarize the current status of research in the Nav field and present the most relevant recent developments with respect to the mol. structure, general physiol., and pharmacol. of distinct Nav channel subtypes.  We discuss Nav channel ligands such as small mols., toxins isolated from animal venoms, and the recently identified Nav1.7-selective antibody.  Furthermore, we review eight characterized ligand binding sites on the Nav channel Î± subunit.  Finally, we examine possible therapeutic applications of Nav ligands and provide an update on current clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn2idyYye8oLVg90H21EOLACvtfcHk0lhcWr4_vOU66A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFemu7c%253D&md5=a816fdf5b46e2e5727669bf17e2e4fd4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501981g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501981g%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLera%2BRuiz%26aufirst%3DM.%26aulast%3DKraus%26aufirst%3DR.%2BL.%26atitle%3DVoltage-gated%2520sodium%2520channels%253A%2520structure%252C%2520function%252C%2520pharmacology%252C%2520and%2520clinical%2520Indications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7093%26epage%3D7118%26doi%3D10.1021%2Fjm501981g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmalhofer, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calhoun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrows, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohler, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinglass, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaczorowski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koltzenburg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priest, B. T.</span></span> <span> </span><span class="NLM_article-title">ProTx-II, a selective inhibitor of Na<sub>V</sub>1.7 sodium channels, blocks action potential propagation in nociceptors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1476</span>â <span class="NLM_lpage">1484</span>, <span class="refDoi">Â DOI: 10.1124/mol.108.047670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1124%2Fmol.108.047670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=18728100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlCnt7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2008&pages=1476-1484&author=W.+A.+Schmalhoferauthor=J.+Calhounauthor=R.+Burrowsauthor=T.+Baileyauthor=M.+G.+Kohlerauthor=A.+B.+Weinglassauthor=G.+J.+Kaczorowskiauthor=M.+L.+Garciaauthor=M.+Koltzenburgauthor=B.+T.+Priest&title=ProTx-II%2C+a+selective+inhibitor+of+NaV1.7+sodium+channels%2C+blocks+action+potential+propagation+in+nociceptors&doi=10.1124%2Fmol.108.047670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">ProTx-II, a selective inhibitor of NaV1.7 sodium channels, blocks action potential propagation in nociceptors</span></div><div class="casAuthors">Schmalhofer, William A.; Calhoun, Jeffrey; Burrows, Rachel; Bailey, Timothy; Kohler, Martin G.; Weinglass, Adam B.; Kaczorowski, Gregory J.; Garcia, Maria L.; Koltzenburg, Martin; Priest, Birgit T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1476-1484</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Voltage-gated sodium (NaV1) channels play a crit. role in modulating the excitability of sensory neurons, and human genetic evidence points to NaV1.7 as an essential contributor to pain signaling.  Human loss-of-function mutations in SCN9A, the gene encoding NaV1.7, cause channelopathy-assocd. indifference to pain (CIP), whereas gain-of-function mutations are assocd. with two inherited painful neuropathies.  Although the human genetic data make NaV1.7 an attractive target for the development of analgesics, pharmacol. proof-of-concept in exptl. pain models requires NaV1.7-selective channel blockers.  Here, we show that the tarantula venom peptide ProTx-II selectively interacts with NaV1.7 channels, inhibiting NaV1.7 with an IC50 value of 0.3 nM, compared with IC50 values of 30 to 150 nM for other heterologously expressed NaV1 subtypes.  This subtype selectivity was abolished by a point mutation in DIIS3.  It is interesting that application of ProTx-II to desheathed cutaneous nerves completely blocked the C-fiber compd. action potential at concns. that had little effect on AÎ²-fiber conduction.  ProTx-II application had little effect on action potential propagation of the intact nerve, which may explain why ProTx-II was not efficacious in rodent models of acute and inflammatory pain.  Mono-iodo-ProTx-II (125I-ProTx-II) binds with high affinity (Kd = 0.3 nM) to recombinant hNaV1.7 channels.  Binding of 125I-ProTx-II is insensitive to the presence of other well characterized NaV1 channel modulators, suggesting that ProTx-II binds to a novel site, which may be more conducive to conferring subtype selectivity than the site occupied by traditional local anesthetics and anticonvulsants.  Thus, the 125I-ProTx-II binding assay, described here, offers a new tool in the search for novel NaV1.7-selective blockers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDrCjZWFExrVg90H21EOLACvtfcHk0liBGu6czT7d5Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlCnt7%252FN&md5=2d42ed47e6bcfcd04730120e1bc87eec</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1124%2Fmol.108.047670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.108.047670%26sid%3Dliteratum%253Aachs%26aulast%3DSchmalhofer%26aufirst%3DW.%2BA.%26aulast%3DCalhoun%26aufirst%3DJ.%26aulast%3DBurrows%26aufirst%3DR.%26aulast%3DBailey%26aufirst%3DT.%26aulast%3DKohler%26aufirst%3DM.%2BG.%26aulast%3DWeinglass%26aufirst%3DA.%2BB.%26aulast%3DKaczorowski%26aufirst%3DG.%2BJ.%26aulast%3DGarcia%26aufirst%3DM.%2BL.%26aulast%3DKoltzenburg%26aufirst%3DM.%26aulast%3DPriest%26aufirst%3DB.%2BT.%26atitle%3DProTx-II%252C%2520a%2520selective%2520inhibitor%2520of%2520NaV1.7%2520sodium%2520channels%252C%2520blocks%2520action%2520potential%2520propagation%2520in%2520nociceptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2008%26volume%3D74%26spage%3D1476%26epage%3D1484%26doi%3D10.1124%2Fmol.108.047670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rahnama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardoso, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanujam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobli, M.</span></span> <span> </span><span class="NLM_article-title">The structure, dynamics and selectivity profile of a Na<sub>V</sub>1.7 potency-optimized huwentoxin-IV variant</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e0173551</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0173551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1371%2Fjournal.pone.0173551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28301520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WjurjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0173551&author=S.+Rahnamaauthor=J.+R.+Deuisauthor=F.+C.+Cardosoauthor=V.+Ramanujamauthor=R.+J.+Lewisauthor=L.+D.+Rashauthor=G.+F.+Kingauthor=I.+Vetterauthor=M.+Mobli&title=The+structure%2C+dynamics+and+selectivity+profile+of+a+NaV1.7+potency-optimized+huwentoxin-IV+variant&doi=10.1371%2Fjournal.pone.0173551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The structure, dynamics and selectivity profile of a NaV1.7 potency-optimised huwentoxin-IV variant</span></div><div class="casAuthors">Rahnama, Sassan; Deuis, Jennifer R.; Cardoso, Fernanda C.; Ramanujam, Venkatraman; Lewis, Richard J.; Rash, Lachlan D.; King, Glenn F.; Vetter, Irina; Mobli, Mehdi</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0173551/1-e0173551/19</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Venom-derived peptides have attracted much attention as potential lead mols. for pharmaceutical development.  A well-known example is Huwentoxin-IV (HwTx-IV), a peptide toxin isolated from the venom of the Chinese bird-eating spider Haplopelma schmitdi.  HwTx-IV was identified as a potent blocker of a human voltage-gated sodium channel (hNaV1.7), which is a genetically validated analgesic target.  The peptide was promising as it showed high potency at NaV1.7 (IC50 â¼26 nM) and selectivity over the cardiac NaV subtype (Nav1.5).  Mutagenesis studies aimed at optimizing the potency of the peptide resulted in the development of a triple-mutant of HwTx-IV (E1G, E4G, Y33W, m3-HwTx-IV) with significantly increased potency against hNaV1.7 (IC50 = 0.4 Â± 0.1 nM) without increased potency against hNaV1.5.  The activity of m3-HwTx-IV against other NaV subtypes was, however, not investigated.  Similarly, the structure of the mutant peptide was not characterised, limiting the interpretation of the obsd. increase in potency.  In this study we produced isotopelabeled recombinant m3-HwTx-IV in E. coli, which enabled us to characterize the atomicresoln. structure and dynamics of the peptide by NMR spectroscopy.  The results show that the structure of the peptide is not perturbed by the mutations, while the relaxation studies reveal that residues in the active site of the peptide undergo conformational exchange.  Addnl., the NaV subtype selectivity of the recombinant peptide was characterised, revealing potent inhibition of neuronal NaV subtypes 1.1, 1.2, 1.3, 1.6 and 1.7.  In parallel to the in vitro studies, we investigated NaV1.7 target engagement of the peptide in vivo using a rodent pain model, where m3-HwTx-IV dose-dependently suppressed spontaneous pain induced by the NaV1.7 activator OD1.  Thus, our results provide further insight into the structure and dynamics of this class of peptides that may prove useful in guiding the development of inhibitors with improved selectivity for analgesic NaV subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMIVGMcrmA4rVg90H21EOLACvtfcHk0liBGu6czT7d5Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WjurjL&md5=a4d8353acf0b45c5310fdbdcf53d3462</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0173551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0173551%26sid%3Dliteratum%253Aachs%26aulast%3DRahnama%26aufirst%3DS.%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DCardoso%26aufirst%3DF.%2BC.%26aulast%3DRamanujam%26aufirst%3DV.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DRash%26aufirst%3DL.%2BD.%26aulast%3DKing%26aufirst%3DG.%2BF.%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DMobli%26aufirst%3DM.%26atitle%3DThe%2520structure%252C%2520dynamics%2520and%2520selectivity%2520profile%2520of%2520a%2520NaV1.7%2520potency-optimized%2520huwentoxin-IV%2520variant%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26spage%3De0173551%26doi%3D10.1371%2Fjournal.pone.0173551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingerd, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munasinghe, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhola, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imlach, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosengren, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosmans, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxman, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dib-Hajj, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoubas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minett, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span> <span> </span><span class="NLM_article-title">Pharmacological characterisation of the highly Na<sub>V</sub>1.7 selective spider venom peptide Pn3a</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">40883</span>, <span class="refDoi">Â DOI: 10.1038/srep40883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2Fsrep40883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28106092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFams74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=40883&author=J.+R.+Deuisauthor=Z.+Dekanauthor=J.+S.+Wingerdauthor=J.+J.+Smithauthor=N.+R.+Munasingheauthor=R.+F.+Bholaauthor=W.+L.+Imlachauthor=V.+Herzigauthor=D.+A.+Armstrongauthor=K.+J.+Rosengrenauthor=F.+Bosmansauthor=S.+G.+Waxmanauthor=S.+D.+Dib-Hajjauthor=P.+Escoubasauthor=M.+S.+Minettauthor=M.+J.+Christieauthor=G.+F.+Kingauthor=P.+F.+Alewoodauthor=R.+J.+Lewisauthor=J.+N.+Woodauthor=I.+Vetter&title=Pharmacological+characterisation+of+the+highly+NaV1.7+selective+spider+venom+peptide+Pn3a&doi=10.1038%2Fsrep40883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterisation of the highly NaV1.7 selective spider venom peptide Pn3a</span></div><div class="casAuthors">Deuis, Jennifer R.; Dekan, Zoltan; Wingerd, Joshua S.; Smith, Jennifer J.; Munasinghe, Nehan R.; Bhola, Rebecca F.; Imlach, Wendy L.; Herzig, Volker; Armstrong, David A.; Rosengren, K. Johan; Bosmans, Frank; Waxman, Stephen G.; Dib-Hajj, Sulayman D.; Escoubas, Pierre; Minett, Michael S.; Christie, Macdonald J.; King, Glenn F.; Alewood, Paul F.; Lewis, Richard J.; Wood, John N.; Vetter, Irina</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40883</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Human genetic studies have implicated the voltage-gated sodium channel NaV1.7 as a therapeutic target for the treatment of pain.  A novel peptide, Î¼-theraphotoxin-Pn3a, isolated from venom of the tarantula Pamphobeteus nigricolor, potently inhibits NaV1.7 (IC50 0.9 nM) with at least 40-1000-fold selectivity over all other NaV subtypes.  Despite on-target activity in small-diam. dorsal root ganglia, spinal slices, and in a mouse model of pain induced by NaV1.7 activation, Pn3a alone displayed no analgesic activity in formalin-, carrageenan- or FCA-induced pain in rodents when administered systemically.  A broad lack of analgesic activity was also found for the selective NaV1.7 inhibitors PF-04856264 and phlotoxin 1.  However, when administered with subtherapeutic doses of opioids or the enkephalinase inhibitor thiorphan, these subtype-selective NaV1.7 inhibitors produced profound analgesia.  Our results suggest that in these inflammatory models, acute administration of peripherally restricted NaV1.7 inhibitors can only produce analgesia when administered in combination with an opioid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOJrS6iZdTXLVg90H21EOLACvtfcHk0lhQ_pMpE4RI1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFams74%253D&md5=8b98535d8a9f5652f25391993c78d82e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsrep40883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep40883%26sid%3Dliteratum%253Aachs%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DDekan%26aufirst%3DZ.%26aulast%3DWingerd%26aufirst%3DJ.%2BS.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMunasinghe%26aufirst%3DN.%2BR.%26aulast%3DBhola%26aufirst%3DR.%2BF.%26aulast%3DImlach%26aufirst%3DW.%2BL.%26aulast%3DHerzig%26aufirst%3DV.%26aulast%3DArmstrong%26aufirst%3DD.%2BA.%26aulast%3DRosengren%26aufirst%3DK.%2BJ.%26aulast%3DBosmans%26aufirst%3DF.%26aulast%3DWaxman%26aufirst%3DS.%2BG.%26aulast%3DDib-Hajj%26aufirst%3DS.%2BD.%26aulast%3DEscoubas%26aufirst%3DP.%26aulast%3DMinett%26aufirst%3DM.%2BS.%26aulast%3DChristie%26aufirst%3DM.%2BJ.%26aulast%3DKing%26aufirst%3DG.%2BF.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DWood%26aufirst%3DJ.%2BN.%26aulast%3DVetter%26aufirst%3DI.%26atitle%3DPharmacological%2520characterisation%2520of%2520the%2520highly%2520NaV1.7%2520selective%2520spider%2520venom%2520peptide%2520Pn3a%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D40883%26doi%3D10.1038%2Fsrep40883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flinspach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piekarz, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellows, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neff, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonesteel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddinger, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaksh, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickenden, A. D.</span></span> <span> </span><span class="NLM_article-title">Insensitivity to pain induced by a potent selective closed-state Na<sub>V</sub>1.7 inhibitor</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">39662</span>, <span class="refDoi">Â DOI: 10.1038/srep39662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2Fsrep39662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28045073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslGquw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=39662&author=M.+Flinspachauthor=Q.+Xuauthor=A.+D.+Piekarzauthor=R.+Fellowsauthor=R.+Haganauthor=A.+Gibbsauthor=Y.+Liuauthor=R.+A.+Neffauthor=J.+Freedmanauthor=W.+A.+Eckertauthor=M.+Zhouauthor=R.+Bonesteelauthor=M.+W.+Penningtonauthor=K.+A.+Eddingerauthor=T.+L.+Yakshauthor=M.+Hunterauthor=R.+V.+Swansonauthor=A.+D.+Wickenden&title=Insensitivity+to+pain+induced+by+a+potent+selective+closed-state+NaV1.7+inhibitor&doi=10.1038%2Fsrep39662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Insensitivity to pain induced by a potent selective closed-state Nav1.7 inhibitor</span></div><div class="casAuthors">Flinspach, M.; Xu, Q.; Piekarz, A. D.; Fellows, R.; Hagan, R.; Gibbs, A.; Liu, Y.; Neff, R. A.; Freedman, J.; Eckert, W. A.; Zhou, M.; Bonesteel, R.; Pennington, M. W.; Eddinger, K. A.; Yaksh, T. L.; Hunter, M.; Swanson, R. V.; Wickenden, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39662</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pain places a devastating burden on patients and society and current pain therapeutics exhibit limitations in efficacy, unwanted side effects and the potential for drug abuse and diversion.  Although genetic evidence has clearly demonstrated that the voltage-gated sodium channel, Nav1.7, is crit. to pain sensation in mammals, pharmacol. inhibitors of Nav1.7 have not yet fully recapitulated the dramatic analgesia obsd. in Nav1.7-null subjects.  Using the tarantula venom-peptide ProTX-II as a scaffold, we engineered a library of over 1500 venom-derived peptides and identified JNJ63955918 as a potent, highly selective, closed-state Nav1.7 blocking peptide.  Here we show that JNJ63955918 induces a pharmacol. insensitivity to pain that closely recapitulates key features of the Nav1.7-null phenotype seen in mice and humans.  Our findings demonstrate that a high degree of selectivity, coupled with a closed-state dependent mechanism of action is required for strong efficacy and indicate that peptides such as JNJ63955918 and other suitably optimized Nav1.7 inhibitors may represent viable non-opioid alternatives for the pharmacol. treatment of severe pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH9Z1m5kOM1bVg90H21EOLACvtfcHk0lhQ_pMpE4RI1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslGquw%253D%253D&md5=1da5e9d488eec322b2f771e9b21e731b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsrep39662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep39662%26sid%3Dliteratum%253Aachs%26aulast%3DFlinspach%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DPiekarz%26aufirst%3DA.%2BD.%26aulast%3DFellows%26aufirst%3DR.%26aulast%3DHagan%26aufirst%3DR.%26aulast%3DGibbs%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNeff%26aufirst%3DR.%2BA.%26aulast%3DFreedman%26aufirst%3DJ.%26aulast%3DEckert%26aufirst%3DW.%2BA.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBonesteel%26aufirst%3DR.%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DEddinger%26aufirst%3DK.%2BA.%26aulast%3DYaksh%26aufirst%3DT.%2BL.%26aulast%3DHunter%26aufirst%3DM.%26aulast%3DSwanson%26aufirst%3DR.%2BV.%26aulast%3DWickenden%26aufirst%3DA.%2BD.%26atitle%3DInsensitivity%2520to%2520pain%2520induced%2520by%2520a%2520potent%2520selective%2520closed-state%2520NaV1.7%2520inhibitor%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D39662%26doi%3D10.1038%2Fsrep39662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">StÃ¶cklin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Engineering potent and selective analogues of GpTx-1, a tarantula venom peptide antagonist of the Na<sub>V</sub>1.7 sodium channel</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2299</span>â <span class="NLM_lpage">2314</span>, <span class="refDoi">Â DOI: 10.1021/jm501765v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501765v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1Snurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2299-2314&author=J.+K.+Murrayauthor=J.+Liguttiauthor=D.+Liuauthor=A.+Zouauthor=L.+Poppeauthor=H.+Liauthor=K.+L.+Andrewsauthor=B.+D.+Moyerauthor=S.+I.+McDonoughauthor=P.+Favreauauthor=R.+St%C3%B6cklinauthor=L.+P.+Miranda&title=Engineering+potent+and+selective+analogues+of+GpTx-1%2C+a+tarantula+venom+peptide+antagonist+of+the+NaV1.7+sodium+channel&doi=10.1021%2Fjm501765v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering Potent and Selective Analogues of GpTx-1, a Tarantula Venom Peptide Antagonist of the NaV1.7 Sodium Channel</span></div><div class="casAuthors">Murray, Justin K.; Ligutti, Joseph; Liu, Dong; Zou, Anruo; Poppe, Leszek; Li, Hongyan; Andrews, Kristin L.; Moyer, Bryan D.; McDonough, Stefan I.; Favreau, Philippe; Stocklin, Reto; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2299-2314</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NaV1.7 is a voltage-gated sodium ion channel implicated by human genetic evidence as a therapeutic target for the treatment of pain.  Screening fractionated venom from the tarantula Grammostola porteri led to the identification of a 34-residue peptide, termed GpTx-1, with potent activity on NaV1.7 (IC50 = 10 nM) and promising selectivity against key NaV subtypes (20Ã and 1000Ã over NaV1.4 and NaV1.5, resp.).  NMR structural anal. of the chem. synthesized three disulfide peptide was consistent with an inhibitory cystine knot motif.  Alanine scanning of GpTx-1 revealed that residues Trp29, Lys31, and Phe34 near the C-terminus are crit. for potent NaV1.7 antagonist activity.  Substitution of Ala for Phe at position 5 conferred 300-fold selectivity against NaV1.4.  A structure-guided campaign afforded additive improvements in potency and NaV subtype selectivity, culminating in the design of [Ala 5,Phe 6,Leu 26,Arg 28]GpTx-1 with a NaV1.7 IC50 value of 1.6 nM and >1000Ã selectivity against NaV1.4 and NaV1.5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryY8lf_XnPDbVg90H21EOLACvtfcHk0liulLXGuuL94A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1Snurs%253D&md5=28528454195c1d12d76a132dfa44273e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm501765v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501765v%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DSt%25C3%25B6cklin%26aufirst%3DR.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DEngineering%2520potent%2520and%2520selective%2520analogues%2520of%2520GpTx-1%252C%2520a%2520tarantula%2520venom%2520peptide%2520antagonist%2520of%2520the%2520NaV1.7%2520sodium%2520channel%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2299%26epage%3D2314%26doi%3D10.1021%2Fjm501765v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.
K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holder, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Sustained inhibition of the Na<sub>V</sub>1.7 sodium channel by engineered dimers of the domain II binding peptide GpTx-1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">4866</span>â <span class="NLM_lpage">4871</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.06.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.bmcl.2015.06.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=26112439" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVeiurzI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=4866-4871&author=J.%0AK.+Murrayauthor=K.+Biswasauthor=J.+R.+Holderauthor=A.+Zouauthor=J.+Liguttiauthor=D.+Liuauthor=L.+Poppeauthor=K.+L.+Andrewsauthor=F.+F.+Linauthor=S.+Y.+Mengauthor=B.+D.+Moyerauthor=S.+I.+McDonoughauthor=L.+P.+Miranda&title=Sustained+inhibition+of+the+NaV1.7+sodium+channel+by+engineered+dimers+of+the+domain+II+binding+peptide+GpTx-1&doi=10.1016%2Fj.bmcl.2015.06.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained inhibition of the NaV1.7 sodium channel by engineered dimers of the domain II binding peptide GpTx-1</span></div><div class="casAuthors">Murray, Justin K.; Biswas, Kaustav; Holder, J. Ryan; Zou, Anruo; Ligutti, Joseph; Liu, Dong; Poppe, Leszek; Andrews, Kristin L.; Lin, Fen-Fen; Meng, Shi-Yuan; Moyer, Bryan D.; McDonough, Stefan I.; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4866-4871</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Many efforts are underway to develop selective inhibitors of the voltage-gated sodium channel NaV1.7 as new analgesics.  Thus far, however, in vitro selectivity has proved difficult for small mols., and peptides generally lack appropriate pharmacokinetic properties.  The authors previously identified the NaV1.7 inhibitory peptide GpTx-1 from tarantula venom and optimized its potency and selectivity via structure-guided analoging.  To further understand GpTx-1 binding to NaV1.7, the authors have mapped the binding site to transmembrane segments 1-4 of the second pseudo-subunit internal repeat (commonly referred to as Site 4) using NaV1.5/NaV1.7 chimeric protein constructs.  The authors also report that select GpTx-1 amino acid residues apparently not contacting NaV1.7 can be derivatized with a hydrophilic polymer without adversely affecting peptide potency.  Homodimerization of GpTx-1 with a bifunctional polyethylene glycol (PEG) linker resulted in a compd. with increased potency and a significantly reduced off-rate, demonstrating the ability to modulate the function and properties of GpTx-1 by linking to addnl. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeHQHHy8plgbVg90H21EOLACvtfcHk0liulLXGuuL94A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVeiurzI&md5=d3f84f31eb02f132394811458242b646</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.06.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.06.033%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DHolder%26aufirst%3DJ.%2BR.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DLin%26aufirst%3DF.%2BF.%26aulast%3DMeng%26aufirst%3DS.%2BY.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DSustained%2520inhibition%2520of%2520the%2520NaV1.7%2520sodium%2520channel%2520by%2520engineered%2520dimers%2520of%2520the%2520domain%2520II%2520binding%2520peptide%2520GpTx-1%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D4866%26epage%3D4871%26doi%3D10.1016%2Fj.bmcl.2015.06.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Single residue substitutions that confer voltage-gated sodium ion channel subtype selectivity in the Na<sub>V</sub>1.7 inhibitory peptide GpTx-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2704</span>â <span class="NLM_lpage">2717</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC28XislOnsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2704-2717&author=J.+K.+Murrayauthor=J.+Longauthor=A.+Zouauthor=J.+Liguttiauthor=K.+L.+Andrewsauthor=L.+Poppeauthor=K.+Biswasauthor=B.+D.+Moyerauthor=S.+I.+McDonoughauthor=L.+P.+Miranda&title=Single+residue+substitutions+that+confer+voltage-gated+sodium+ion+channel+subtype+selectivity+in+the+NaV1.7+inhibitory+peptide+GpTx-1&doi=10.1021%2Facs.jmedchem.5b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Single Residue Substitutions That Confer Voltage-Gated Sodium Ion Channel Subtype Selectivity in the NaV1.7 Inhibitory Peptide GpTx-1</span></div><div class="casAuthors">Murray, Justin K.; Long, Jason; Zou, Anruo; Ligutti, Joseph; Andrews, Kristin L.; Poppe, Leszek; Biswas, Kaustav; Moyer, Bryan D.; McDonough, Stefan I.; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2704-2717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is interest in the identification and optimization of new mol. entities selectively targeting ion channels of therapeutic relevance.  Peptide toxins represent a rich source of pharmacol. for ion channels, and we recently reported GpTx-1 analogs that inhibit NaV1.7, a voltage-gated sodium ion channel that is a compelling target for improved treatment of pain.  Here we utilize multi-attribute positional scan (MAPS) analoging, combining high-throughput synthesis and electrophysiol., to interrogate the interaction of GpTx-1 with NaV1.7 and related NaV subtypes.  After one round of MAPS analoging, we found novel substitutions at multiple residue positions not previously identified, specifically glutamic acid at positions 10 or 11 or lysine at position 18, that produce peptides with single digit nanomolar potency on NaV1.7 and 500-fold selectivity against off-target sodium channels.  Docking studies with a NaV1.7 homol. model and peptide NMR structure generated a model consistent with the key potency and selectivity modifications mapped in this work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqcQfq19Y1YrVg90H21EOLACvtfcHk0lg0VwvJJHxOmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislOnsbg%253D&md5=cf317a857972d1d9897f8710504ccee0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01947%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DSingle%2520residue%2520substitutions%2520that%2520confer%2520voltage-gated%2520sodium%2520ion%2520channel%2520subtype%2520selectivity%2520in%2520the%2520NaV1.7%2520inhibitory%2520peptide%2520GpTx-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2704%26epage%3D2717%26doi%3D10.1021%2Facs.jmedchem.5b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">StÃ¶cklin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological characterization of potent and selective Na<sub>V</sub>1.7 inhibitors engineered from <i>Chilobrachys jingzhao</i> tarantula venom peptide JzTx-V</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e0196791</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0196791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1371%2Fjournal.pone.0196791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=29723257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsl2iurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0196791&author=B.+D.+Moyerauthor=J.+K.+Murrayauthor=J.+Liguttiauthor=K.+Andrewsauthor=P.+Favreauauthor=J.+B.+Jordanauthor=J.+H.+Leeauthor=D.+Liuauthor=J.+Longauthor=K.+Shamauthor=L.+Shiauthor=R.+St%C3%B6cklinauthor=B.+Wuauthor=R.+Yinauthor=V.+Yuauthor=A.+Zouauthor=K.+Biswasauthor=L.+P.+Miranda&title=Pharmacological+characterization+of+potent+and+selective+NaV1.7+inhibitors+engineered+from+Chilobrachys+jingzhao+tarantula+venom+peptide+JzTx-V&doi=10.1371%2Fjournal.pone.0196791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of potent and selective NaV1.7 inhibitors engineered from Chilobrachys jingzhao tarantula venom peptide JzTx-V</span></div><div class="casAuthors">Moyer, Bryan D.; Murray, Justin K.; Ligutti, Joseph; Andrews, Kristin; Favreau, Philippe; Jordan, John B.; Lee, Josie H.; Liu, Dong; Long, Jason; Sham, Kelvin; Shi, Licheng; Stocklin, Reto; Wu, Bin; Yin, Ruoyuan; Yu, Violeta; Zou, Anruo; Biswas, Kaustav; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e0196791/1-e0196791/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Identification of voltage-gated sodium channel NaV1.7 inhibitors for chronic pain therapeutic development is an area of vigorous pursuit.  In an effort to identify more potent leads compared to our previously reported GpTx-1 peptide series, electrophysiol. screening of fractionated tarantula venom discovered the NaV1.7 inhibitory peptide JzTx-V from the Chinese earth tiger tarantula Chilobrachys jingzhao.  The parent peptide displayed nominal selectivity over the skeletal muscle NaV1.4 channel.  Attribute-based positional scan analoging identified a key Ile28Glu mutation that improved NaV1.4 selectivity over 100-fold, and further optimization yielded the potent and selective peptide leads AM-8145 and AM-0422.  NMR analyses revealed that the Ile28Glu substitution changed peptide conformation, pointing to a structural rationale for the selectivity gains.  AM-8145 and AM-0422 as well as GpTx-1 and HwTx-IV competed for ProTx-II binding in HEK293 cells expressing human NaV1.7, suggesting that these NaV1.7 inhibitory peptides interact with a similar binding site.  AM-8145 potently blocked native tetrodotoxin-sensitive (TTX-S) channels in mouse dorsal root ganglia (DRG) neurons, exhibited 30- to 120-fold selectivity over other human TTX-S channels and exhibited over 1,000-fold selectivity over other human tetrodotoxin-resistant (TTX-R) channels.  Leveraging NaV1.7-NaV1.5 chimeras contg. various voltage-sensor and pore regions, AM-8145 mapped to the second voltage-sensor domain of NaV1.7.  AM-0422, but not the inactive peptide analog AM-8374, dose-dependently blocked capsaicin-induced DRG neuron action potential firing using a multi-electrode array readout and mech.-induced C-fiber spiking in a saphenous skin-nerve prepn.  Collectively, AM-8145 and AM-0422 represent potent, new engineered NaV1.7 inhibitory peptides derived from the JzTx-V scaffold with improved NaV selectivity and biol. activity in blocking action potential firing in both DRG neurons and C-fibers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr_v8GXWwSxLVg90H21EOLACvtfcHk0lg0VwvJJHxOmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsl2iurbK&md5=4478846639ebeda482156c38676e6ab3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0196791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0196791%26sid%3Dliteratum%253Aachs%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DK.%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DJordan%26aufirst%3DJ.%2BB.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DSham%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSt%25C3%25B6cklin%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DV.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DPharmacological%2520characterization%2520of%2520potent%2520and%2520selective%2520NaV1.7%2520inhibitors%2520engineered%2520from%2520Chilobrachys%2520jingzhao%2520tarantula%2520venom%2520peptide%2520JzTx-V%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26spage%3De0196791%26doi%3D10.1371%2Fjournal.pone.0196791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingerd, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weidner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nassar, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span> <span> </span><span class="NLM_article-title">Analgesic effects of GpTx-1, PF-04856264 and CNV1014802 in a mouse model of Na<sub>V</sub>1.7-mediated pain</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">78</span>, <span class="refDoi">Â DOI: 10.3390/toxins8030078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.3390%2Ftoxins8030078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCmurrM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=78&author=J.+R.+Deuisauthor=J.+S.+Wingerdauthor=Z.+Winterauthor=T.+Durekauthor=Z.+Dekanauthor=S.+R.+Sousaauthor=K.+Zimmermannauthor=T.+Hoffmannauthor=C.+Weidnerauthor=M.+A.+Nassarauthor=P.+F.+Alewoodauthor=R.+J.+Lewisauthor=I.+Vetter&title=Analgesic+effects+of+GpTx-1%2C+PF-04856264+and+CNV1014802+in+a+mouse+model+of+NaV1.7-mediated+pain&doi=10.3390%2Ftoxins8030078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Analgesic effects of GpTx-1, PF-04856264 and CNV1014802 in a mouse model of NaV1.7-mediated pain</span></div><div class="casAuthors">Deuis, Jennifer R.; Wingerd, Joshua S.; Winter, Zoltan; Durek, Thomas; Dekan, Zoltan; Sousa, Silmara R.; Zimmermann, Katharina; Hoffmann, Tali; Weidner, Christian; Nassar, Mohammed A.; Alewood, Paul F.; Lewis, Richard J.; Vetter, Irina</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">78/1-78/19</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Loss-of-function mutations of NaV1.7 lead to congenital insensitivity to pain, a rare condition resulting in individuals who are otherwise normal except for the inability to sense pain, making pharmacol. inhibition of NaV1.7 a promising therapeutic strategy for the treatment of pain.  We characterized a novel mouse model of NaV1.7-mediated pain based on intraplantar injection of the scorpion toxin OD1, which is suitable for rapid in vivo profiling of NaV1.7 inhibitors.  Intraplantar injection of OD1 caused spontaneous pain behaviors, which were reversed by co-injection with NaV1.7 inhibitors and significantly reduced in NaV1.7-/- mice.  To validate the use of the model for profiling NaV1.7 inhibitors, we detd. the NaV selectivity and tested the efficacy of the reported NaV1.7 inhibitors GpTx-1, PF-04856264 and CNV1014802 (raxatrigine).  GpTx-1 selectively inhibited NaV1.7 and was effective when co-administered with OD1, but lacked efficacy when delivered systemically.  PF-04856264 state-dependently and selectively inhibited NaV1.7 and significantly reduced OD1-induced spontaneous pain when delivered locally and systemically.  CNV1014802 state-dependently, but non-selectively, inhibited NaV channels and was only effective in the OD1 model when delivered systemically.  Our novel model of NaV1.7-mediated pain based on intraplantar injection of OD1 is thus suitable for the rapid in vivo characterization of the analgesic efficacy of NaV1.7 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn5aBTBFoLvbVg90H21EOLACvtfcHk0lg0VwvJJHxOmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCmurrM&md5=335b5e4c4747e94c1ac7750c47139fae</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3390%2Ftoxins8030078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins8030078%26sid%3Dliteratum%253Aachs%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DWingerd%26aufirst%3DJ.%2BS.%26aulast%3DWinter%26aufirst%3DZ.%26aulast%3DDurek%26aufirst%3DT.%26aulast%3DDekan%26aufirst%3DZ.%26aulast%3DSousa%26aufirst%3DS.%2BR.%26aulast%3DZimmermann%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DWeidner%26aufirst%3DC.%26aulast%3DNassar%26aufirst%3DM.%2BA.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DVetter%26aufirst%3DI.%26atitle%3DAnalgesic%2520effects%2520of%2520GpTx-1%252C%2520PF-04856264%2520and%2520CNV1014802%2520in%2520a%2520mouse%2520model%2520of%2520NaV1.7-mediated%2520pain%26jtitle%3DToxins%26date%3D2016%26volume%3D8%26spage%3D78%26doi%3D10.3390%2Ftoxins8030078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aral, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenkilsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salyers, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sickmier, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGivern, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gegg, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical optimization of peptide toxins for ion channel targets: potent, selective, and long-lived antagonists of K<sub>V</sub>1.3</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6784</span>â <span class="NLM_lpage">6802</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00495</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00495" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOjtbjK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6784-6802&author=J.+K.+Murrayauthor=Y.+X.+Qianauthor=B.+Liuauthor=R.+Elliottauthor=J.+Aralauthor=C.+Parkauthor=X.+Zhangauthor=M.+Stenkilssonauthor=K.+Salyersauthor=M.+Roseauthor=H.+Liauthor=S.+Yuauthor=K.+L.+Andrewsauthor=A.+Colomberoauthor=J.+Wernerauthor=K.+Gaidaauthor=E.+A.+Sickmierauthor=P.+Miuauthor=A.+Itanoauthor=J.+McGivernauthor=C.+V.+Geggauthor=J.+K.+Sullivanauthor=L.+P.+Miranda&title=Pharmaceutical+optimization+of+peptide+toxins+for+ion+channel+targets%3A+potent%2C+selective%2C+and+long-lived+antagonists+of+KV1.3&doi=10.1021%2Facs.jmedchem.5b00495"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical Optimization of Peptide Toxins for Ion Channel Targets: Potent, Selective, and Long-Lived Antagonists of Kv1.3</span></div><div class="casAuthors">Murray, Justin K.; Qian, Yi-Xin; Liu, Benxian; Elliott, Robin; Aral, Jennifer; Park, Cynthia; Zhang, Xuxia; Stenkilsson, Michael; Salyers, Kevin; Rose, Mark; Li, Hongyan; Yu, Steven; Andrews, Kristin L.; Colombero, Anne; Werner, Jonathan; Gaida, Kevin; Sickmier, E. Allen; Miu, Peter; Itano, Andrea; McGivern, Joseph; Gegg, Colin V.; Sullivan, John K.; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6784-6802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To realize the medicinal potential of peptide toxins, naturally occurring disulfide-rich peptides, as ion channel antagonists, more efficient pharmaceutical optimization technologies must be developed.  The therapeutic properties of multiple cysteine toxin peptides can be rapidly and substantially improved by combining direct chem. strategies with high-throughput electrophysiol.  The authors applied whole-mol., brute-force, structure-activity analoging to ShK, a peptide toxin from the sea anemone Stichodactyla helianthus that inhibits the voltage-gated potassium ion channel Kv1.3, to effectively discover crit. structural changes for 15Ã selectivity against the closely related neuronal ion channel Kv1.1.  Subsequent site-specific polymer conjugation resulted in an exquisitely selective Kv1.3 antagonist (>1000Ã over Kv1.1) with picomolar functional activity in whole blood and a pharmacokinetic profile suitable for weekly administration in primates.  The pharmacol. potential of the optimized toxin peptide was demonstrated by potent and sustained inhibition of cytokine secretion from T cells, a therapeutic target for autoimmune diseases, in cynomolgus monkeys.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEtF-Ugp8jFbVg90H21EOLACvtfcHk0licZuvx2zk4iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOjtbjK&md5=1cfb03e783bd953480c203dc9b95af84</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00495%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DQian%26aufirst%3DY.%2BX.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DElliott%26aufirst%3DR.%26aulast%3DAral%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DStenkilsson%26aufirst%3DM.%26aulast%3DSalyers%26aufirst%3DK.%26aulast%3DRose%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DColombero%26aufirst%3DA.%26aulast%3DWerner%26aufirst%3DJ.%26aulast%3DGaida%26aufirst%3DK.%26aulast%3DSickmier%26aufirst%3DE.%2BA.%26aulast%3DMiu%26aufirst%3DP.%26aulast%3DItano%26aufirst%3DA.%26aulast%3DMcGivern%26aufirst%3DJ.%26aulast%3DGegg%26aufirst%3DC.%2BV.%26aulast%3DSullivan%26aufirst%3DJ.%2BK.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DPharmaceutical%2520optimization%2520of%2520peptide%2520toxins%2520for%2520ion%2520channel%2520targets%253A%2520potent%252C%2520selective%252C%2520and%2520long-lived%2520antagonists%2520of%2520KV1.3%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6784%26epage%3D6802%26doi%3D10.1021%2Facs.jmedchem.5b00495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pathak, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarov-Yarovoy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohout, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tombola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacoff, E. Y.</span></span> <span> </span><span class="NLM_article-title">Closing in on the resting state of the Shaker K<sup>+</sup> channel</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">124</span>â <span class="NLM_lpage">140</span>, <span class="refDoi">Â DOI: 10.1016/j.neuron.2007.09.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.neuron.2007.09.023" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2007&pages=124-140&author=M.+M.+Pathakauthor=V.+Yarov-Yarovoyauthor=G.+Agarwalauthor=B.+Rouxauthor=P.+Barthauthor=S.+Kohoutauthor=F.+Tombolaauthor=E.+Y.+Isacoff&title=Closing+in+on+the+resting+state+of+the+Shaker+K%2B+channel&doi=10.1016%2Fj.neuron.2007.09.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2007.09.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2007.09.023%26sid%3Dliteratum%253Aachs%26aulast%3DPathak%26aufirst%3DM.%2BM.%26aulast%3DYarov-Yarovoy%26aufirst%3DV.%26aulast%3DAgarwal%26aufirst%3DG.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DBarth%26aufirst%3DP.%26aulast%3DKohout%26aufirst%3DS.%26aulast%3DTombola%26aufirst%3DF.%26aulast%3DIsacoff%26aufirst%3DE.%2BY.%26atitle%3DClosing%2520in%2520on%2520the%2520resting%2520state%2520of%2520the%2520Shaker%2520K%252B%2520channel%26jtitle%3DNeuron%26date%3D2007%26volume%3D56%26spage%3D124%26epage%3D140%26doi%3D10.1016%2Fj.neuron.2007.09.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Z. P.</span></span> <span> </span><span class="NLM_article-title">Docking unbound proteins using shape complementarity, desolvation, and electrostatics</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">281</span>â <span class="NLM_lpage">294</span>, <span class="refDoi">Â DOI: 10.1002/prot.10092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1002%2Fprot.10092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=11948782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFantLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2002&pages=281-294&author=R.+Chenauthor=Z.+P.+Weng&title=Docking+unbound+proteins+using+shape+complementarity%2C+desolvation%2C+and+electrostatics&doi=10.1002%2Fprot.10092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Docking unbound proteins using shape complementarity, desolvation, and electrostatics</span></div><div class="casAuthors">Chen, Rong; Weng, Zhiping</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">281-294</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A comprehensive docking study was performed on 27 distinct protein-protein complexes.  For 13 test systems, docking was performed with the unbound x-ray structures of both the receptor and the ligand.  For the remaining systems, the unbound x-ray structure of only mol. was available; therefore the bound structure for the other mol. was used.  Our method optimizes desolvation, shape complementarity, and electrostatics using a Fast Fourier Transform algorithm.  A global search in the rotational and translational space without any knowledge of the binding sites was performed for all proteins except nine antibodies recognizing antigens.  For these antibodies, we docked their well-characterized binding site-the complementarity-detg. region defined without information of the antigen-to the entire surface of the antigen.  For 24 systems, we were able to find near-native ligand orientations (interface CÎ± root mean square deviation less than 2.5 Ã from the crystal complex) among the top 2,000 choices.  For three systems, our algorithm could identify the correct complex structure unambiguously.  For 13 other complexes, we either ranked a near-native structure in the top 20 or obtained 20 or more near-native structures in the top 2000 or both.  The key feature of our algorithm is the use of target functions that are highly tolerant to conformational changes upon binding.  If combined with a post-processing method, our algorithm may provide a general soln. to the unbound docking problem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCxIONKClpN7Vg90H21EOLACvtfcHk0li1xlnotifZyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFantLg%253D&md5=7e7d07996a899a90ee71b758cdee429d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fprot.10092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10092%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWeng%26aufirst%3DZ.%2BP.%26atitle%3DDocking%2520unbound%2520proteins%2520using%2520shape%2520complementarity%252C%2520desolvation%252C%2520and%2520electrostatics%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2002%26volume%3D47%26spage%3D281%26epage%3D294%26doi%3D10.1002%2Fprot.10092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, J. O.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, T. R.</span></span> <span> </span><span class="NLM_article-title">Common molecular determinants of tarantula huwentoxin-IV inhibition of Na<sup>+</sup> channel voltage sensors in domains II and IV</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">27301</span>â <span class="NLM_lpage">27310</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.246876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1074%2Fjbc.M111.246876" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=27301-27310&author=Y.+Xiaoauthor=J.+O.+Jacksonauthor=S.+Liangauthor=T.+R.+Cummins&title=Common+molecular+determinants+of+tarantula+huwentoxin-IV+inhibition+of+Na%2B+channel+voltage+sensors+in+domains+II+and+IV&doi=10.1074%2Fjbc.M111.246876"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.246876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.246876%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJackson%26aufirst%3DJ.%2BO.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DCummins%26aufirst%3DT.%2BR.%26atitle%3DCommon%2520molecular%2520determinants%2520of%2520tarantula%2520huwentoxin-IV%2520inhibition%2520of%2520Na%252B%2520channel%2520voltage%2520sensors%2520in%2520domains%2520II%2520and%2520IV%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D27301%26epage%3D27310%26doi%3D10.1074%2Fjbc.M111.246876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klint, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupasinghe, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Er, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span> <span> </span><span class="NLM_article-title">Spider-venom peptides that target voltage-gated sodium channels: Pharmacological tools and potential therapeutic leads</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">478</span>â <span class="NLM_lpage">491</span>, <span class="refDoi">Â DOI: 10.1016/j.toxicon.2012.04.337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.toxicon.2012.04.337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=22543187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVyqu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=478-491&author=J.+K.+Klintauthor=S.+Senffauthor=D.+B.+Rupasingheauthor=S.+Y.+Erauthor=V.+Herzigauthor=G.+M.+Nicholsonauthor=G.+F.+King&title=Spider-venom+peptides+that+target+voltage-gated+sodium+channels%3A+Pharmacological+tools+and+potential+therapeutic+leads&doi=10.1016%2Fj.toxicon.2012.04.337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Spider-venom peptides that target voltage-gated sodium channels: Pharmacological tools and potential therapeutic leads</span></div><div class="casAuthors">Klint, Julie K.; Senff, Sebastian; Rupasinghe, Darshani B.; Er, Sing Yan; Herzig, Volker; Nicholson, Graham M.; King, Glenn F.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">478-491</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Voltage-gated Na (NaV) channels play a central role in the propagation of action potentials in excitable cells in both humans and insects.  Many venomous animals have therefore evolved toxins that modulate the activity of NaV channels to subdue their prey and deter predators.  Spider venoms in particular are rich in NaV channel modulators, with 1/3 of all known ion channel toxins from spider venoms acting on NaV channels.  Here we review the landscape of spider-venom peptides that have so far been described to target vertebrate or invertebrate NaV channels.  These peptides fall into 12 distinct families based on their primary structure and cysteine scaffold.  Some of these peptides have become useful pharmacol. tools, while others have potential as therapeutic leads because they target specific NaV channel subtypes that are considered to be important analgesic targets.  Spider venoms are conservatively predicted to contain more than 10 million bioactive peptides and so far only 0.01% of this diversity been characterized.  Thus, it is likely that future research will reveal addnl. structural classes of spider-venom peptides that target NaV channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrsU_ooS6kmbVg90H21EOLACvtfcHk0li1xlnotifZyA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVyqu7Y%253D&md5=35d3c33bd548ea6bf2d99b7702a7d623</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2012.04.337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2012.04.337%26sid%3Dliteratum%253Aachs%26aulast%3DKlint%26aufirst%3DJ.%2BK.%26aulast%3DSenff%26aufirst%3DS.%26aulast%3DRupasinghe%26aufirst%3DD.%2BB.%26aulast%3DEr%26aufirst%3DS.%2BY.%26aulast%3DHerzig%26aufirst%3DV.%26aulast%3DNicholson%26aufirst%3DG.%2BM.%26aulast%3DKing%26aufirst%3DG.%2BF.%26atitle%3DSpider-venom%2520peptides%2520that%2520target%2520voltage-gated%2520sodium%2520channels%253A%2520Pharmacological%2520tools%2520and%2520potential%2520therapeutic%2520leads%26jtitle%3DToxicon%26date%3D2012%26volume%3D60%26spage%3D478%26epage%3D491%26doi%3D10.1016%2Fj.toxicon.2012.04.337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, N.</span></span> <span> </span><span class="NLM_article-title">Structure of a eukaryotic voltage-gated sodium channel at near-atomic resolution</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">eaal4326</span>, <span class="refDoi">Â DOI: 10.1126/science.aal4326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1126%2Fscience.aal4326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28183995" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=eaal4326&author=H.+Shenauthor=Q.+Zhouauthor=X.+Panauthor=Z.+Liauthor=J.+Wuauthor=N.+Yan&title=Structure+of+a+eukaryotic+voltage-gated+sodium+channel+at+near-atomic+resolution&doi=10.1126%2Fscience.aal4326"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal4326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal4326%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DN.%26atitle%3DStructure%2520of%2520a%2520eukaryotic%2520voltage-gated%2520sodium%2520channel%2520at%2520near-atomic%2520resolution%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3Deaal4326%26doi%3D10.1126%2Fscience.aal4326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, N.</span></span> <span> </span><span class="NLM_article-title">Structure of the Na<sub>V</sub>1.4-Î²1 complex from electric eel</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">470</span>â <span class="NLM_lpage">482</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2017.06.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.cell.2017.06.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28735751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=470-482&author=Z.+Yanauthor=Q.+Zhouauthor=L.+Wangauthor=J.+Wuauthor=Y.+Zhaoauthor=G.+Huangauthor=W.+Pengauthor=H.+Shenauthor=J.+Leiauthor=N.+Yan&title=Structure+of+the+NaV1.4-%CE%B21+complex+from+electric+eel&doi=10.1016%2Fj.cell.2017.06.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Nav1.4-Î²1 complex from electric eel</span></div><div class="casAuthors">Yan, Zhen; Zhou, Qiang; Wang, Lin; Wu, Jianping; Zhao, Yanyu; Huang, Gaoxingyu; Peng, Wei; Shen, Huaizong; Lei, Jianlin; Yan, Nieng</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-482.e11</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Voltage-gated Na+ (Nav) channels initiate and propagate action potentials.  Here, we present the cryo-electron microscopic structure of EeNav1.4, the Nav channel from elec. eel (Electrophorus electricus; Ee), in complex with the Î²1 subunit at 4.0 Ã resoln.  The Ig domain of Î²1 docked onto the extracellular L5I and L6IV loops of EeNav1.4 via extensive polar interactions, and the single transmembrane helix interacted with the 3rd voltage-sensing domain (VSDIII).  The VSDs exhibited "up" conformations, while the intracellular gate of the pore domain was kept open by a digitonin-like mol.  Structural comparison with closed NavPaS showed that the outward transfer of gating charges was coupled to the iris-like pore domain dilation through intricate force transmissions involving multiple channel segments.  The IFM fast inactivation motif on the III-IV linker was plugged into the corner enclosed by the outer S4-S5 and inner S6 segments in repeats III and IV, suggesting a potential allosteric blocking mechanism for fast inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzkD2D5o-No7Vg90H21EOLACvtfcHk0lj6pCl_7Mb03w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73P&md5=02ae7b12b16bfe60df4aa019675fc769</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.06.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.06.039%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DN.%26atitle%3DStructure%2520of%2520the%2520NaV1.4-%25CE%25B21%2520complex%2520from%2520electric%2520eel%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D470%26epage%3D482%26doi%3D10.1016%2Fj.cell.2017.06.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruben, P. C.</span></span> <span> </span><span class="NLM_article-title">Interaction between voltage-gated sodium channels and the neurotoxin, tetrodotoxin</span>. <i>Channels</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">407</span>â <span class="NLM_lpage">412</span>, <span class="refDoi">Â DOI: 10.4161/chan.2.6.7429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.4161%2Fchan.2.6.7429" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2008&pages=407-412&author=C.+H.+Leeauthor=P.+C.+Ruben&title=Interaction+between+voltage-gated+sodium+channels+and+the+neurotoxin%2C+tetrodotoxin&doi=10.4161%2Fchan.2.6.7429"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.4161%2Fchan.2.6.7429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fchan.2.6.7429%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DRuben%26aufirst%3DP.%2BC.%26atitle%3DInteraction%2520between%2520voltage-gated%2520sodium%2520channels%2520and%2520the%2520neurotoxin%252C%2520tetrodotoxin%26jtitle%3DChannels%26date%3D2008%26volume%3D2%26spage%3D407%26epage%3D412%26doi%3D10.4161%2Fchan.2.6.7429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minassian, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neff, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellows, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husovsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinspach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickenden, A. D.</span></span> <span> </span><span class="NLM_article-title">Analysis of the structural and molecular basis of voltage-sensitive sodium channel inhibition by the spider toxin Huwentoxin-IV (Î¼-TRTX-Hh2a)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">22707</span>â <span class="NLM_lpage">22720</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M113.461392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1074%2Fjbc.M113.461392" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=22707-22720&author=N.+A.+Minassianauthor=A.+Gibbsauthor=A.+Y.+Shihauthor=Y.+Liuauthor=R.+A.+Neffauthor=S.+W.+Suttonauthor=T.+Mirzadeganauthor=J.+Connorauthor=R.+Fellowsauthor=M.+Husovskyauthor=S.+Nelsonauthor=M.+J.+Hunterauthor=M.+Flinspachauthor=A.+D.+Wickenden&title=Analysis+of+the+structural+and+molecular+basis+of+voltage-sensitive+sodium+channel+inhibition+by+the+spider+toxin+Huwentoxin-IV+%28%CE%BC-TRTX-Hh2a%29&doi=10.1074%2Fjbc.M113.461392"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.461392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.461392%26sid%3Dliteratum%253Aachs%26aulast%3DMinassian%26aufirst%3DN.%2BA.%26aulast%3DGibbs%26aufirst%3DA.%26aulast%3DShih%26aufirst%3DA.%2BY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNeff%26aufirst%3DR.%2BA.%26aulast%3DSutton%26aufirst%3DS.%2BW.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DConnor%26aufirst%3DJ.%26aulast%3DFellows%26aufirst%3DR.%26aulast%3DHusovsky%26aufirst%3DM.%26aulast%3DNelson%26aufirst%3DS.%26aulast%3DHunter%26aufirst%3DM.%2BJ.%26aulast%3DFlinspach%26aufirst%3DM.%26aulast%3DWickenden%26aufirst%3DA.%2BD.%26atitle%3DAnalysis%2520of%2520the%2520structural%2520and%2520molecular%2520basis%2520of%2520voltage-sensitive%2520sodium%2520channel%2520inhibition%2520by%2520the%2520spider%2520toxin%2520Huwentoxin-IV%2520%2528%25CE%25BC-TRTX-Hh2a%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D22707%26epage%3D22720%26doi%3D10.1074%2Fjbc.M113.461392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojeda, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Racemic and quasi-racemic X-ray structures of cyclic disulfide-rich peptide drug scaffolds</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">11236</span>â <span class="NLM_lpage">11241</span>, <span class="refDoi">Â DOI: 10.1002/anie.201406563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1002%2Fanie.201406563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGgtbnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=11236-11241&author=C.+K.+Wangauthor=G.+J.+Kingauthor=S.+E.+Northfieldauthor=P.+G.+Ojedaauthor=D.+J.+Craik&title=Racemic+and+quasi-racemic+X-ray+structures+of+cyclic+disulfide-rich+peptide+drug+scaffolds&doi=10.1002%2Fanie.201406563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Racemic and quasi-racemic X-ray structures of cyclic disulfide-rich peptide drug scaffolds</span></div><div class="casAuthors">Wang, Conan K.; King, Gordon J.; Northfield, Susan E.; Ojeda, Paola G.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11236-11241</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic disulfide-rich peptides have exceptional stability and are promising frameworks for drug design.  We were interested in obtaining X-ray structures of these peptides to assist in drug design applications, but disulfide-rich peptides can be notoriously difficult to crystallize.  To overcome this limitation, we chem. synthesized the L- and D-forms of three prototypic cyclic disulfide-rich peptides: SFTI-1 (14-mer with one disulfide bond), cVc1.1 (22-mer with two disulfide bonds), and kB1 (29-mer with three disulfide bonds) for racemic crystn. studies.  Facile crystal formation occurred from a racemic mixt. of each peptide, giving structures solved at resolns. from 1.25 Ã to 1.9 Ã.  Addnl., we obtained the quasi-racemic structures of two mutants of kB1, [G6A]kB1, and [V25A]kB1, which were solved at a resoln. of 1.25 Ã and 2.3 Ã, resp.  The racemic crystallog. approach appears to have broad utility in the structural biol. of cyclic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqliAlRrLTkj7Vg90H21EOLACvtfcHk0lhNir-kG1vxWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGgtbnO&md5=37a4997cdd96ead9514473b655c71148</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201406563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201406563%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DKing%26aufirst%3DG.%2BJ.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DOjeda%26aufirst%3DP.%2BG.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DRacemic%2520and%2520quasi-racemic%2520X-ray%2520structures%2520of%2520cyclic%2520disulfide-rich%2520peptide%2520drug%2520scaffolds%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D11236%26epage%3D11241%26doi%3D10.1002%2Fanie.201406563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixey, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falsey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herberich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holder, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunasekaran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Engineering antibody reactivity for efficient derivatization to generate Na<sub>V</sub>1.7 inhibitory GpTx-1 peptide-antibody conjugates</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2427</span>â <span class="NLM_lpage">2435</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.7b00542</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00542" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2427-2435&author=K.+Biswasauthor=T.+E.+Nixeyauthor=J.+K.+Murrayauthor=J.+R.+Falseyauthor=L.+Yinauthor=H.+Liuauthor=J.+Gingrasauthor=B.+E.+Hallauthor=B.+Herberichauthor=J.+R.+Holderauthor=H.+Liauthor=J.+Liguttiauthor=M.+J.+Linauthor=D.+Liuauthor=B.+D.+Sorianoauthor=M.+Sotoauthor=L.+Tranauthor=C.+M.+Tegleyauthor=A.+Zouauthor=K.+Gunasekaranauthor=B.+D.+Moyerauthor=L.+Dohertyauthor=L.+P.+Miranda&title=Engineering+antibody+reactivity+for+efficient+derivatization+to+generate+NaV1.7+inhibitory+GpTx-1+peptide-antibody+conjugates&doi=10.1021%2Facschembio.7b00542"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00542%26sid%3Dliteratum%253Aachs%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DNixey%26aufirst%3DT.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DFalsey%26aufirst%3DJ.%2BR.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGingras%26aufirst%3DJ.%26aulast%3DHall%26aufirst%3DB.%2BE.%26aulast%3DHerberich%26aufirst%3DB.%26aulast%3DHolder%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DSoriano%26aufirst%3DB.%2BD.%26aulast%3DSoto%26aufirst%3DM.%26aulast%3DTran%26aufirst%3DL.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DGunasekaran%26aufirst%3DK.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DDoherty%26aufirst%3DL.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DEngineering%2520antibody%2520reactivity%2520for%2520efficient%2520derivatization%2520to%2520generate%2520NaV1.7%2520inhibitory%2520GpTx-1%2520peptide-antibody%2520conjugates%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2427%26epage%3D2435%26doi%3D10.1021%2Facschembio.7b00542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kornecook, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Be, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilch, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarosh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehto, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortuno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taborn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, D. X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremeau, R. T.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic characterization of AMG8379, a potent and selective small molecule sulfonamide antagonist of the voltage-gated sodium channel Na<sub>V</sub>1.7</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>362</i></span>,  <span class="NLM_fpage">146</span>â <span class="NLM_lpage">160</span>, <span class="refDoi">Â DOI: 10.1124/jpet.116.239590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1124%2Fjpet.116.239590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28473457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOqsb3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2017&pages=146-160&author=T.+J.+Kornecookauthor=R.+Yinauthor=S.+Altmannauthor=X.+Beauthor=V.+Berryauthor=C.+P.+Ilchauthor=M.+Jaroshauthor=D.+Johnsonauthor=J.+H.+Leeauthor=S.+G.+Lehtoauthor=J.+Liguttiauthor=D.+Liuauthor=J.+Lutherauthor=D.+Matsonauthor=D.+Ortunoauthor=J.+Robertsauthor=K.+Tabornauthor=J.+Wangauthor=M.+M.+Weissauthor=V.+Yuauthor=D.+X.+D.+Zhuauthor=R.+T.+Fremeauauthor=B.+D.+Moyer&title=Pharmacologic+characterization+of+AMG8379%2C+a+potent+and+selective+small+molecule+sulfonamide+antagonist+of+the+voltage-gated+sodium+channel+NaV1.7&doi=10.1124%2Fjpet.116.239590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic characterization of AMG8379, a potent and selective small molecule sulfonamide antagonist of the voltage-gated sodium channel NaV1.7</span></div><div class="casAuthors">Kornecook, Thomas J.; Yin, Ruoyuan; Altmann, Stephen; Be, Xuhai; Berry, Virginia; Ilch, Christopher P.; Jarosh, Michael; Johnson, Danielle; Lee, Josie H.; Lehto, Sonya G.; Ligutti, Joseph; Liu, Dong; Luther, Jason; Matson, David; Ortuno, Danny; Roberts, John; Taborn, Kristin; Wang, Jinti; Weiss, Matthew M.; Yu, Violeta; Zhu, Dawn X. D.; Fremeau, Robert T.; Moyer, Bryan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">146-160</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Potent and selective antagonists of the voltage-gated sodium channel NaV1.7 represent a promising avenue for the development of new chronic pain therapies.  We generated a small mol. atropisomer quinolone sulfonamide antagonist AMG8379 and a less active enantiomer AMG8380.  Here we show that AMG8379 potently blocks human NaV1.7 channels with an IC50 of 8.5 nM and endogenous tetrodotoxin (TTX)-sensitive sodium channels in dorsal root ganglion (DRG) neurons with an IC50 of 3.1 nM in whole-cell patch clamp electrophysiol. assays using a voltage protocol that interrogates channels in a partially inactivated state.  AMG8379 was 100- to 1000-fold selective over other NaV family members, including NaV1.4 expressed in muscle and NaV1.5 expressed in the heart, as well as TTX-resistant NaV channels in DRG neurons.  Using an ex vivo mouse skin-nerve prepn., AMG8379 blocked mech. induced action potential firing in C-fibers in both a time-dependent and dose-dependent manner.  AMG8379 similarly reduced the frequency of thermally induced C-fiber spiking, whereas AMG8380 affected neither mech. nor thermal responses.  In vivo target engagement of AMG8379 in mice was evaluated in multiple NaV1.7-dependent behavioral endpoints.  AMG8379 dose-dependently inhibited intradermal histamine-induced scratching and intraplantar capsaicin-induced licking, and reversed UVB radiation skin burn-induced thermal hyperalgesia; notably, behavioral effects were not obsd. with AMG8380 at similar plasma exposure levels.  AMG8379 is a potent and selective NaV1.7 inhibitor that blocks sodium current in heterologous cells as well as DRG neurons, inhibits action potential firing in peripheral nerve fibers, and exhibits pharmacodynamic effects in translatable models of both itch and pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZr8oKonIYLVg90H21EOLACvtfcHk0lhNir-kG1vxWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOqsb3O&md5=97f6b3a928ac98aa59cf68859d008f19</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fjpet.116.239590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.116.239590%26sid%3Dliteratum%253Aachs%26aulast%3DKornecook%26aufirst%3DT.%2BJ.%26aulast%3DYin%26aufirst%3DR.%26aulast%3DAltmann%26aufirst%3DS.%26aulast%3DBe%26aufirst%3DX.%26aulast%3DBerry%26aufirst%3DV.%26aulast%3DIlch%26aufirst%3DC.%2BP.%26aulast%3DJarosh%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLehto%26aufirst%3DS.%2BG.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLuther%26aufirst%3DJ.%26aulast%3DMatson%26aufirst%3DD.%26aulast%3DOrtuno%26aufirst%3DD.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DTaborn%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DV.%26aulast%3DZhu%26aufirst%3DD.%2BX.%2BD.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26atitle%3DPharmacologic%2520characterization%2520of%2520AMG8379%252C%2520a%2520potent%2520and%2520selective%2520small%2520molecule%2520sulfonamide%2520antagonist%2520of%2520the%2520voltage-gated%2520sodium%2520channel%2520NaV1.7%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2017%26volume%3D362%26spage%3D146%26epage%3D160%26doi%3D10.1124%2Fjpet.116.239590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marx, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boezio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu-Moyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMauro, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feric
Bojic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foti, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graceffa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman-Perez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarosh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornecook, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreiman, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taborn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youngblood, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremeau, R. T.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Sulfonamides as selective Na<sub>V</sub>1.7 inhibitors: optimizing potency and pharmacokinetics while mitigating metabolic liabilities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5969</span>â <span class="NLM_lpage">5989</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01851</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01851" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5969-5989&author=M.+M.+Weissauthor=T.+A.+Dineenauthor=I.+E.+Marxauthor=S.+Altmannauthor=A.+Boezioauthor=H.+Bregmanauthor=M.+Chu-Moyerauthor=E.+F.+DiMauroauthor=E.+Feric%0ABojicauthor=R.+S.+Fotiauthor=H.+Gaoauthor=R.+Graceffaauthor=H.+Gunaydinauthor=A.+Guzman-Perezauthor=H.+Huangauthor=L.+Huangauthor=M.+Jaroshauthor=T.+Kornecookauthor=C.+R.+Kreimanauthor=J.+Liguttiauthor=D.+S.+Laauthor=M.+J.+Linauthor=D.+Liuauthor=B.+D.+Moyerauthor=H.+N.+Nguyenauthor=E.+A.+Petersonauthor=P.+E.+Roseauthor=K.+Tabornauthor=B.+D.+Youngbloodauthor=V.+Yuauthor=R.+T.+Fremeau&title=Sulfonamides+as+selective+NaV1.7+inhibitors%3A+optimizing+potency+and+pharmacokinetics+while+mitigating+metabolic+liabilities&doi=10.1021%2Facs.jmedchem.6b01851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfonamides as Selective NaV1.7 Inhibitors: Optimizing Potency and Pharmacokinetics While Mitigating Metabolic Liabilities</span></div><div class="casAuthors">Weiss, Matthew M.; Dineen, Thomas A.; Marx, Isaac E.; Altmann, Steven; Boezio, Alessandro; Bregman, Howard; Chu-Moyer, Margaret; DiMauro, Erin F.; Feric Bojic, Elma; Foti, Robert S.; Gao, Hua; Graceffa, Russell; Gunaydin, Hakan; Guzman-Perez, Angel; Huang, Hongbing; Huang, Liyue; Jarosh, Michael; Kornecook, Thomas; Kreiman, Charles R.; Ligutti, Joseph; La, Daniel S.; Lin, Min-Hwa Jasmine; Liu, Dong; Moyer, Bryan D.; Nguyen, Hanh N.; Peterson, Emily A.; Rose, Paul E.; Taborn, Kristin; Youngblood, Beth D.; Yu, Violeta; Fremeau, Robert T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5969-5989</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several reports have recently emerged regarding the identification of heteroarylsulfonamides as NaV1.7 inhibitors that demonstrate high levels of selectivity over other NaV isoforms.  The optimization of a series of internal NaV1.7 leads that address a no. of metabolic liabilities including bioactivation, PXR activation, as well as CYP3A4 induction and inhibition led to the identification of potent and selective inhibitors that demonstrated favorable pharmacokinetic profiles and were devoid of the aforementioned liabilities.  The key to achieving this within a series prone to transporter-mediated clearance was the identification of a small range of optimal cLogD values and the discovery of subtle PXR SAR that was not lipophilicity dependent.  This enabled the identification of compd. 20, which was advanced into a target engagement pharmacodynamic model where it exhibited robust reversal of histamine-induced scratching bouts in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovsPC2UZpxsbVg90H21EOLACvtfcHk0lgUhxLWq7_wBg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmtbw%253D&md5=6333a6b936e48f9d18cf9df10ea12853</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01851%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DM.%2BM.%26aulast%3DDineen%26aufirst%3DT.%2BA.%26aulast%3DMarx%26aufirst%3DI.%2BE.%26aulast%3DAltmann%26aufirst%3DS.%26aulast%3DBoezio%26aufirst%3DA.%26aulast%3DBregman%26aufirst%3DH.%26aulast%3DChu-Moyer%26aufirst%3DM.%26aulast%3DDiMauro%26aufirst%3DE.%2BF.%26aulast%3DFeric%2BBojic%26aufirst%3DE.%26aulast%3DFoti%26aufirst%3DR.%2BS.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGraceffa%26aufirst%3DR.%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DGuzman-Perez%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DJarosh%26aufirst%3DM.%26aulast%3DKornecook%26aufirst%3DT.%26aulast%3DKreiman%26aufirst%3DC.%2BR.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLa%26aufirst%3DD.%2BS.%26aulast%3DLin%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DNguyen%26aufirst%3DH.%2BN.%26aulast%3DPeterson%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DP.%2BE.%26aulast%3DTaborn%26aufirst%3DK.%26aulast%3DYoungblood%26aufirst%3DB.%2BD.%26aulast%3DYu%26aufirst%3DV.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26atitle%3DSulfonamides%2520as%2520selective%2520NaV1.7%2520inhibitors%253A%2520optimizing%2520potency%2520and%2520pharmacokinetics%2520while%2520mitigating%2520metabolic%2520liabilities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5969%26epage%3D5989%26doi%3D10.1021%2Facs.jmedchem.6b01851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasseur, R.</span></span> <span> </span><span class="NLM_article-title">Analysis of accessible surface of residues in proteins</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1406</span>â <span class="NLM_lpage">1417</span>, <span class="refDoi">Â DOI: 10.1110/ps.0304803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1110%2Fps.0304803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=12824487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltFekur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1406-1417&author=L.+Linsauthor=A.+Thomasauthor=R.+Brasseur&title=Analysis+of+accessible+surface+of+residues+in+proteins&doi=10.1110%2Fps.0304803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of accessible surface of residues in proteins</span></div><div class="casAuthors">Lins, Laurence; Thomas, Annick; Brasseur, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1406-1417</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We analyzed the total, hydrophobic, and hydrophilic accessible surfaces (ASAs) of residues from a nonredundant bank of 587 3D structure proteins.  In an extended fold, residues are classified into three families with respect to their hydrophobicity balance.  As expected, residues lose part of their solvent-accessible surface with folding but the three groups remain.  The decrease of accessibility is more pronounced for hydrophobic than hydrophilic residues.  Amazingly, Lysine is the residue with the largest hydrophobic accessible surface in folded structures.  Our anal. points out a clear difference between the mean (other studies) and median (this study) ASA values of hydrophobic residues, which should be taken into consideration for future investigations on a protein-accessible surface, in order to improve predictions requiring ASA values.  The different secondary structures correspond to different accessibility of residues.  Random coils, turns, and Î²-structures (outside Î²-sheets) are the most accessible folds, with an av. of 30% accessibility.  The helical residues are about 20% accessible, and the difference between the hydrophobic and the hydrophilic residues illustrates the amphipathy of many helixes.  Residues from Î²-sheets are the most inaccessible to solvent (10% accessible).  Hence, Î²-sheets are the most appropriate structures to shield the hydrophobic parts of residues from water.  We also show that there is an equal balance between the hydrophobic and the hydrophilic accessible surfaces of the 3D protein surfaces irresp. of the protein size.  This results in a patchwork surface of hydrophobic and hydrophilic arcas, which could be important for protein interactions and/or activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouA-fCE-awerVg90H21EOLACvtfcHk0lgUhxLWq7_wBg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltFekur0%253D&md5=1bc25f11ef043dbf08f1930c0c245421</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1110%2Fps.0304803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.0304803%26sid%3Dliteratum%253Aachs%26aulast%3DLins%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DA.%26aulast%3DBrasseur%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520accessible%2520surface%2520of%2520residues%2520in%2520proteins%26jtitle%3DProtein%2520Sci.%26date%3D2003%26volume%3D12%26spage%3D1406%26epage%3D1417%26doi%3D10.1110%2Fps.0304803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, Z. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cryar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilford, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takle, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thalassinos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabor, A. B.</span></span> <span> </span><span class="NLM_article-title">The role of disulfide bond replacements in analogues of the tarantula toxin ProTx-II and their effects on inhibition of the voltage-gated sodium ion channel Na<sub>V</sub>1.7</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">13063</span>â <span class="NLM_lpage">13075</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b06506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b06506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCqs7rN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=13063-13075&author=Z.+V.+F.+Wrightauthor=S.+McCarthyauthor=R.+Dickmanauthor=F.+E.+Reyesauthor=S.+Sanchez-Martinezauthor=A.+Cryarauthor=I.+Kilfordauthor=A.+Hallauthor=A.+K.+Takleauthor=M.+Topfauthor=T.+Gonenauthor=K.+Thalassinosauthor=A.+B.+Tabor&title=The+role+of+disulfide+bond+replacements+in+analogues+of+the+tarantula+toxin+ProTx-II+and+their+effects+on+inhibition+of+the+voltage-gated+sodium+ion+channel+NaV1.7&doi=10.1021%2Fjacs.7b06506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Disulfide Bond Replacements in Analogues of the Tarantula Toxin ProTx-II and Their Effects on Inhibition of the Voltage-Gated Sodium Ion Channel Nav1.7</span></div><div class="casAuthors">Wright, Zoe V. F.; McCarthy, Stephen; Dickman, Rachael; Reyes, Francis E.; Sanchez-Martinez, Silvia; Cryar, Adam; Kilford, Ian; Hall, Adrian; Takle, Andrew K.; Topf, Maya; Gonen, Tamir; Thalassinos, Konstantinos; Tabor, Alethea B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">13063-13075</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spider venom toxins, such as Protoxin-II (ProTx-II), have recently received much attention as selective Nav1.7 channel blockers, with potential to be developed as leads for the treatment of chronic nociceptive pain.  ProTx-II is a 30-amino acid peptide with three disulfide bonds that has been reported to adopt a well-defined inhibitory cystine knot (ICK) scaffold structure.  Potential drawbacks with such peptides include poor pharmacodynamics and potential scrambling of the disulfide bonds in vivo.  In order to address these issues, in the present study the authors report the solid-phase synthesis of lanthionine-bridged analogs of ProTx-II, in which one of the three disulfide bridges is replaced with a thioether linkage, and have evaluated the biol. properties of these analogs.  The authors have also investigated the folding and disulfide bridging patterns arising from different methods of oxidn. of the linear peptide precursor.  Finally, the authors report the x-ray crystal structure of ProTx-II to at. resoln.; to the knowledge this is the first crystal structure of an inhibitory cystine knot (ICK) spider venom peptide not bound to a substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn9ZD9HHB7jrVg90H21EOLACvtfcHk0lifnpbyMfkvFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCqs7rN&md5=26210eb421e73f0bd439147862a27b6e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b06506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b06506%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DZ.%2BV.%2BF.%26aulast%3DMcCarthy%26aufirst%3DS.%26aulast%3DDickman%26aufirst%3DR.%26aulast%3DReyes%26aufirst%3DF.%2BE.%26aulast%3DSanchez-Martinez%26aufirst%3DS.%26aulast%3DCryar%26aufirst%3DA.%26aulast%3DKilford%26aufirst%3DI.%26aulast%3DHall%26aufirst%3DA.%26aulast%3DTakle%26aufirst%3DA.%2BK.%26aulast%3DTopf%26aufirst%3DM.%26aulast%3DGonen%26aufirst%3DT.%26aulast%3DThalassinos%26aufirst%3DK.%26aulast%3DTabor%26aufirst%3DA.%2BB.%26atitle%3DThe%2520role%2520of%2520disulfide%2520bond%2520replacements%2520in%2520analogues%2520of%2520the%2520tarantula%2520toxin%2520ProTx-II%2520and%2520their%2520effects%2520on%2520inhibition%2520of%2520the%2520voltage-gated%2520sodium%2520ion%2520channel%2520NaV1.7%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D13063%26epage%3D13075%26doi%3D10.1021%2Fjacs.7b06506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couture, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feric, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottman, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beiler, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmberg, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span> <span> </span><span class="NLM_article-title">Global Na<sub>V</sub>1.7 knockout mice recapitulate the phenotype of human congenital indifference to pain</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e105895</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0105895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1371%2Fjournal.pone.0105895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=25188265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslaqt73O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e105895&author=J.+Gingrasauthor=S.+Smithauthor=D.+J.+Matsonauthor=D.+Johnsonauthor=K.+Nyeauthor=L.+Coutureauthor=E.+Fericauthor=R.+Yinauthor=B.+D.+Moyerauthor=M.+L.+Petersonauthor=J.+B.+Rottmanauthor=R.+J.+Beilerauthor=A.+B.+Malmbergauthor=S.+I.+McDonough&title=Global+NaV1.7+knockout+mice+recapitulate+the+phenotype+of+human+congenital+indifference+to+pain&doi=10.1371%2Fjournal.pone.0105895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Global Nav1.7 knockout mice recapitulate the phenotype of human congenital indifference to pain</span></div><div class="casAuthors">Gingras, Jacinthe; Smith, Sarah; Matson, David J.; Johnson, Danielle; Nye, Kim; Couture, Lauren; Feric, Elma; Yin, Ruoyuan; Moyer, Bryan D.; Peterson, Matthew L.; Rottman, James B.; Beiler, Rudolph J.; Malmberg, Annika B.; McDonough, Stefan I.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e105895/1-e105895/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Clin. genetic studies have shown that loss of Nav1.7 function leads to the complete loss of acute pain perception.  The global deletion is reported lethal in mice, however, and studies of mice with promoter-specific deletions of Nav1.7 have suggested that the role of Nav1.7 in pain transduction depends on the precise form of pain.  We developed genetic and animal husbandry strategies that overcame the neonatal-lethal phenotype and enabled construction of a global Nav1.7 knockout mouse.  Knockouts were anatomically normal, reached adulthood, and had phenotype wholly analogous to human congenital indifference to pain (CIP): compared to littermates, knockouts showed no defects in mech. sensitivity or overall movement yet were completely insensitive to painful tactile, thermal, and chem. stimuli and were anosmic.  Knockouts also showed no painful behaviors resulting from peripheral injection of nonselective sodium channel activators, did not develop complete Freund's adjuvant-induced thermal hyperalgesia, and were insensitive to intra-dermal histamine injection.  Tetrodotoxin-sensitive sodium current recorded from cell bodies of isolated sensory neurons and the mech.-evoked spiking of C-fibers in a skin-nerve prepn. each were reduced but not eliminated in tissue from knockouts compared to littermates.  Results support a role for Nav1.7 that is conserved between rodents and humans and suggest several possibly translatable biomarkers for the study of Nav1.7-targeted therapeutics.  Results further suggest that Nav1.7 may retain its key role in persistent as well as acute forms of pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAzxB508dD1LVg90H21EOLACvtfcHk0lifnpbyMfkvFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslaqt73O&md5=6a1acb0c6e4e38ac89b1ab38d67b538e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0105895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0105895%26sid%3Dliteratum%253Aachs%26aulast%3DGingras%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DMatson%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DNye%26aufirst%3DK.%26aulast%3DCouture%26aufirst%3DL.%26aulast%3DFeric%26aufirst%3DE.%26aulast%3DYin%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26aulast%3DRottman%26aufirst%3DJ.%2BB.%26aulast%3DBeiler%26aufirst%3DR.%2BJ.%26aulast%3DMalmberg%26aufirst%3DA.%2BB.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26atitle%3DGlobal%2520NaV1.7%2520knockout%2520mice%2520recapitulate%2520the%2520phenotype%2520of%2520human%2520congenital%2520indifference%2520to%2520pain%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De105895%26doi%3D10.1371%2Fjournal.pone.0105895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">ElbrÃ¸nd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakobsen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AgersÃ¸, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatorp, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zdravkovic, M.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetics, pharmacodynamics, safety, and tolerability of a single-dose of NN2211, a long-acting glucagon-like peptide 1 derivative, in healthy male subjects</span>. <i>Diabetes Care</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1398</span>â <span class="NLM_lpage">1404</span>, <span class="refDoi">Â DOI: 10.2337/diacare.25.8.1398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.2337%2Fdiacare.25.8.1398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=12145241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsFaitr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2002&pages=1398-1404&author=B.+Elbr%C3%B8ndauthor=G.+Jakobsenauthor=S.+Larsenauthor=H.+Agers%C3%B8author=L.+B.+Jensenauthor=P.+Rolanauthor=J.+Sturisauthor=V.+Hatorpauthor=M.+Zdravkovic&title=Pharmacokinetics%2C+pharmacodynamics%2C+safety%2C+and+tolerability+of+a+single-dose+of+NN2211%2C+a+long-acting+glucagon-like+peptide+1+derivative%2C+in+healthy+male+subjects&doi=10.2337%2Fdiacare.25.8.1398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetics, pharmacodynamics, safety, and tolerability of a single-dose of NN2211, a long-acting glucagon-like peptide 1 derivative, in healthy male subjects</span></div><div class="casAuthors">Elbrond, Bodil; Jakobsen, Grethe; Larsen, Soren; Agerso, Henrik; Jensen, Lisbeth Bjerring; Rolan, Paul; Sturis, Jeppe; Hatorp, Vibeke; Zdravkovic, Milan</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes Care</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1398-1404</span>CODEN:
                <span class="NLM_cas:coden">DICAD2</span>;
        ISSN:<span class="NLM_cas:issn">0149-5992</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association, Inc.</span>)
        </div><div class="casAbstract">OBJECTIVE - The primary objective of the present study was to investigate the safety, tolerability, and pharmacokinetics of a single dose of NN 2211, a long-acting glucagon-like peptide 1 (GLP-1) deriv., in healthy male subjects.  The secondary objective was to investigate the pharmacodynamics of NN 2211.  RESEARCH DESIGN AND METHODS - In a double-blind, randomized dose, escalation, placebo-controlled study, healthy male subjects were enrolled at eight consecutive dose levels (1.25, 2.5, 5.0, 10.0, 12.5, 15.0, 17.5, and 20.0 Î¼g/kg) with eight subjects per dose level at a 3:1 active:placebo randomization.  After s.c. dosing with NN 2211, 48-h pharmacokinetic, and 24-h glucose, insulin and glucagon profiles were assessed.  In addn., three subjects at each dose level were randomly assigned (one placebo/two active) to an i.v. glucose tolerance test (IVGTT) 9 h after the dose (corresponding to the time to maximal plasma concn. of NN 2211).  RESULTS - After s.c. administration, the half-life of NN 2211 was found to be 11-15 h.  Overall, although there were no statistically significant differences compared with placebo in the area under the curve (0-9 h for insulin or glucagon), there was a borderline-significant lowering of glucose levels.  During the IVGTT, there was a statistically significant increase in insulin secretion, but there was no significant effect on glucagon levels.  Although no significant effect was obsd. on glucose levels during the IVGTT, there was a dose-dependent increase in the glucose disappearance const.  Whereas no serious adverse events were obsd., there was a higher incidence of adverse events after active treatment compared with placebo treatment (notably headache, dizziness, nausea, and vomiting).  CONCLUSIONS - This study provides evidence that NN 2211 has a pharmacokinetic profile consistent with once-daily dosing in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCuouDtWjhLLVg90H21EOLACvtfcHk0lifnpbyMfkvFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsFaitr0%253D&md5=e91822e17b5ac8a3da0aa152aba558eb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.2337%2Fdiacare.25.8.1398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdiacare.25.8.1398%26sid%3Dliteratum%253Aachs%26aulast%3DElbr%25C3%25B8nd%26aufirst%3DB.%26aulast%3DJakobsen%26aufirst%3DG.%26aulast%3DLarsen%26aufirst%3DS.%26aulast%3DAgers%25C3%25B8%26aufirst%3DH.%26aulast%3DJensen%26aufirst%3DL.%2BB.%26aulast%3DRolan%26aufirst%3DP.%26aulast%3DSturis%26aufirst%3DJ.%26aulast%3DHatorp%26aufirst%3DV.%26aulast%3DZdravkovic%26aufirst%3DM.%26atitle%3DPharmacokinetics%252C%2520pharmacodynamics%252C%2520safety%252C%2520and%2520tolerability%2520of%2520a%2520single-dose%2520of%2520NN2211%252C%2520a%2520long-acting%2520glucagon-like%2520peptide%25201%2520derivative%252C%2520in%2520healthy%2520male%2520subjects%26jtitle%3DDiabetes%2520Care%26date%3D2002%26volume%3D25%26spage%3D1398%26epage%3D1404%26doi%3D10.2337%2Fdiacare.25.8.1398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ingwersen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khurana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madabushi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonker, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thi, T. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">TornÃ¸e, C. W.</span></span> <span> </span><span class="NLM_article-title">Dosing rationale for Liraglutide in type 2 diabetes mellitus: a pharmacometric assessment</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1815</span>â <span class="NLM_lpage">1823</span>, <span class="refDoi">Â DOI: 10.1177/0091270011430504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1177%2F0091270011430504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=22174428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ltrzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1815-1823&author=S.+H.+Ingwersenauthor=M.+Khuranaauthor=R.+Madabushiauthor=E.+Watsonauthor=D.+M.+Jonkerauthor=T.+D.+L.+Thiauthor=L.+V.+Jacobsenauthor=C.+W.+Torn%C3%B8e&title=Dosing+rationale+for+Liraglutide+in+type+2+diabetes+mellitus%3A+a+pharmacometric+assessment&doi=10.1177%2F0091270011430504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dosing rationale for liraglutide in type 2 diabetes mellitus: a pharmacometric assessment</span></div><div class="casAuthors">Ingwersen, Steen H.; Khurana, Manoj; Madabushi, Rajanikanth; Watson, Estelle; Jonker, Daniel M.; Le Thi, Tu Duyen; Jacobsen, Lisbeth V.; Tornoee, Christoffer W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1815-1823</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The glucagon-like peptide-1 (GLP-1) receptor agonist liraglutide was approved in 2010 by the US Food and Drug Administration (FDA) as an adjunct treatment to diet and exercise to improve glycemic control in adults with type 2 diabetes mellitus.  This article provides insights into the use of pharmacometric analyses for regulatory review with a focus on the dosing recommendations.  The assessment was based on the totality of exploratory and confirmatory anal. of dose-finding and pivotal clin. data and was structured around a set of key questions in accordance with current FDA review practice.  For the pharmacometric review of liraglutide, the key questions focused on exposure-response relationships for effects on fasting plasma glucose, Hb A1c, and calcitonin and on variability in exposure across demog. subgroups of patients.  The importance of conducting exploratory exposure-response anal. and population pharmacokinetic studies in clin. drug development to support dosing recommendations is highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLL90Sr-WPabVg90H21EOLACvtfcHk0lgGIb2UyQiwyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ltrzE&md5=3bc6e426ae9801f33c9aceb78c5fc247</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F0091270011430504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270011430504%26sid%3Dliteratum%253Aachs%26aulast%3DIngwersen%26aufirst%3DS.%2BH.%26aulast%3DKhurana%26aufirst%3DM.%26aulast%3DMadabushi%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DE.%26aulast%3DJonker%26aufirst%3DD.%2BM.%26aulast%3DThi%26aufirst%3DT.%2BD.%2BL.%26aulast%3DJacobsen%26aufirst%3DL.%2BV.%26aulast%3DTorn%25C3%25B8e%26aufirst%3DC.%2BW.%26atitle%3DDosing%2520rationale%2520for%2520Liraglutide%2520in%2520type%25202%2520diabetes%2520mellitus%253A%2520a%2520pharmacometric%2520assessment%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2012%26volume%3D52%26spage%3D1815%26epage%3D1823%26doi%3D10.1177%2F0091270011430504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgGIb2UyQiwyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>â <span class="NLM_lpage">1214</span>, <span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgGIb2UyQiwyw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">CCP4
(Collaborative Computational Project, Number 4)</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein
crystallography</span>. <i>Acta Crystallogr., Sect. D:
Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>â <span class="NLM_lpage">763</span>, <span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=CCP4%0A%28Collaborative+Computational+Project%2C+Number+4%29&title=The+CCP4+suite%3A+programs+for+protein%0Acrystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%250Acrystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%250ABiol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia-xing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfson, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">A modified ACORN to solve protein structures at resolutions of 1.7 Ã or better</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1465</span>â <span class="NLM_lpage">1475</span>, <span class="refDoi">Â DOI: 10.1107/S090744490502576X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS090744490502576X" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=1465-1475&author=Y.+Jia-xingauthor=M.+M.+Woolfsonauthor=K.+S.+Wilsonauthor=E.+J.+Dodson&title=A+modified+ACORN+to+solve+protein+structures+at+resolutions+of+1.7+%C3%85+or+better&doi=10.1107%2FS090744490502576X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS090744490502576X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744490502576X%26sid%3Dliteratum%253Aachs%26aulast%3DJia-xing%26aufirst%3DY.%26aulast%3DWoolfson%26aufirst%3DM.%2BM.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DA%2520modified%2520ACORN%2520to%2520solve%2520protein%2520structures%2520at%2520resolutions%2520of%25201.7%2520%25C3%2585%2520or%2520better%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D1465%26epage%3D1475%26doi%3D10.1107%2FS090744490502576X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljYVvtOmAUWUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljYVvtOmAUWUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Alexander Mueller, Zoltan Dekan, Quentin Kaas, Akello J. Agwa, Hana Starobova, Paul F. Alewood, Christina I. Schroeder, Mehdi Mobli, Jennifer R. Deuis, <span class="NLM_string-name hlFld-ContribAuthor">Irina Vetter</span>. </span><span class="cited-content_cbyCitation_article-title">Mapping the Molecular Surface of the Analgesic NaV1.7-Selective Peptide Pn3a Reveals Residues Essential for Membrane and Channel Interactions. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2020,</strong> <em>3 </em>
                                    (3)
                                     , 535-546. <a href="https://doi.org/10.1021/acsptsci.0c00002" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00002%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DMapping%252Bthe%252BMolecular%252BSurface%252Bof%252Bthe%252BAnalgesic%252BNaV1.7-Selective%252BPeptide%252BPn3a%252BReveals%252BResidues%252BEssential%252Bfor%252BMembrane%252Band%252BChannel%252BInteractions%26aulast%3DMueller%26aufirst%3DAlexander%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07012020%26date%3D05032020%26date%3D19022020%26volume%3D3%26issue%3D3%26spage%3D535%26epage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chun Yuen Chow, Yanni K.-Y. Chin, Andrew A. Walker, Shaodong Guo, Linda V. Blomster, Micaiah J. Ward, Volker Herzig, Darin R. Rokyta, <span class="NLM_string-name hlFld-ContribAuthor">Glenn F. King</span>. </span><span class="cited-content_cbyCitation_article-title">Venom Peptides with Dual Modulatory Activity on the Voltage-Gated Sodium Channel NaV1.1 Provide Novel Leads for Development of Antiepileptic Drugs. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2020,</strong> <em>3 </em>
                                    (1)
                                     , 119-134. <a href="https://doi.org/10.1021/acsptsci.9b00079" title="DOI URL">https://doi.org/10.1021/acsptsci.9b00079</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.9b00079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.9b00079%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DVenom%252BPeptides%252Bwith%252BDual%252BModulatory%252BActivity%252Bon%252Bthe%252BVoltage-Gated%252BSodium%252BChannel%252BNaV1.1%252BProvide%252BNovel%252BLeads%252Bfor%252BDevelopment%252Bof%252BAntiepileptic%252BDrugs%26aulast%3DChow%26aufirst%3DChun%2BYuen%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D01102019%26date%3D18122019%26date%3D25112019%26volume%3D3%26issue%3D1%26spage%3D119%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">John V. Mulcahy, Hassan Pajouhesh, Jacob T. Beckley, Anton Delwig, J. Du Bois, <span class="NLM_string-name hlFld-ContribAuthor">John C. Hunter</span>. </span><span class="cited-content_cbyCitation_article-title">Challenges and Opportunities for Therapeutics Targeting the Voltage-Gated Sodium Channel Isoform NaV1.7. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (19)
                                     , 8695-8710. <a href="https://doi.org/10.1021/acs.jmedchem.8b01906" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01906</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01906%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DChallenges%252Band%252BOpportunities%252Bfor%252BTherapeutics%252BTargeting%252Bthe%252BVoltage-Gated%252BSodium%252BChannel%252BIsoform%252BNaV1.7%26aulast%3DMulcahy%26aufirst%3DJohn%2BV.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D04122018%26date%3D07052019%26date%3D23042019%26volume%3D62%26issue%3D19%26spage%3D8695%26epage%3D8710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Justin K. Murray, Bin Wu, Christopher M. Tegley, Thomas E. Nixey, James R. Falsey, Brad Herberich, Li Yin, Kelvin Sham, Jason Long, Jennifer Aral, Yuan Cheng, Chawita Netirojjanakul, Liz Doherty, Charles Glaus, Tayo Ikotun, Hongyan Li, Linh Tran, Marcus Soto, Hossein Salimi-Moosavi, Joseph Ligutti, Shanti Amagasu, Kristin L. Andrews, Xuhai Be, Min-Hwa Jasmine Lin, Robert S. Foti, Christopher P. Ilch, Beth Youngblood, Thomas J. Kornecook, Margaret Karow, Kenneth W. Walker, Bryan D. Moyer, Kaustav Biswas, <span class="NLM_string-name hlFld-ContribAuthor">Les P. Miranda</span>. </span><span class="cited-content_cbyCitation_article-title">Engineering NaV1.7 Inhibitory JzTx-V Peptides with a Potency and Basicity Profile Suitable for Antibody Conjugation To Enhance Pharmacokinetics. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (4)
                                     , 806-818. <a href="https://doi.org/10.1021/acschembio.9b00183" title="DOI URL">https://doi.org/10.1021/acschembio.9b00183</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00183%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DEngineering%252BNaV1.7%252BInhibitory%252BJzTx-V%252BPeptides%252Bwith%252Ba%252BPotency%252Band%252BBasicity%252BProfile%252BSuitable%252Bfor%252BAntibody%252BConjugation%252BTo%252BEnhance%252BPharmacokinetics%26aulast%3DMurray%26aufirst%3DJustin%2BK.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07032019%26date%3D15032019%26date%3D27032019%26date%3D15032019%26volume%3D14%26issue%3D4%26spage%3D806%26epage%3D818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sigrid  Noreng</span>, <span class="hlFld-ContribAuthor ">Tianbo  Li</span>, <span class="hlFld-ContribAuthor ">Jian  Payandeh</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Pharmacology of Voltage-Gated Sodium Channels. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2021,</strong> <em>433 </em>
                                    (17)
                                     , 166967. <a href="https://doi.org/10.1016/j.jmb.2021.166967" title="DOI URL">https://doi.org/10.1016/j.jmb.2021.166967</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2021.166967&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2021.166967%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DStructural%252BPharmacology%252Bof%252BVoltage-Gated%252BSodium%252BChannels%26aulast%3DNoreng%26aufirst%3DSigrid%26date%3D2021%26volume%3D433%26issue%3D17%26spage%3D166967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert A.  Neff</span>, <span class="hlFld-ContribAuthor ">Alan D.  Wickenden</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Targeting of Nav1.7 with Engineered Spider Venom-Based Peptides. </span><span class="cited-content_cbyCitation_journal-name">Channels</span><span> <strong>2021,</strong> <em>15 </em>
                                    (1)
                                     , 179-193. <a href="https://doi.org/10.1080/19336950.2020.1860382" title="DOI URL">https://doi.org/10.1080/19336950.2020.1860382</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/19336950.2020.1860382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F19336950.2020.1860382%26sid%3Dliteratum%253Aachs%26jtitle%3DChannels%26atitle%3DSelective%252BTargeting%252Bof%252BNav1.7%252Bwith%252BEngineered%252BSpider%252BVenom-Based%252BPeptides%26aulast%3DNeff%26aufirst%3DRobert%2BA.%26date%3D2021%26date%3D2021%26volume%3D15%26issue%3D1%26spage%3D179%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronika  Temml</span>, <span class="hlFld-ContribAuthor ">Zsofia  Kutil</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based molecular modeling in SAR analysis and lead optimization. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2021,</strong> <em>19 </em>, 1431-1444. <a href="https://doi.org/10.1016/j.csbj.2021.02.018" title="DOI URL">https://doi.org/10.1016/j.csbj.2021.02.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2021.02.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2021.02.018%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DStructure-based%252Bmolecular%252Bmodeling%252Bin%252BSAR%252Banalysis%252Band%252Blead%252Boptimization%26aulast%3DTemml%26aufirst%3DVeronika%26date%3D2021%26volume%3D19%26spage%3D1431%26epage%3D1444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David A.  Eagles</span>, <span class="hlFld-ContribAuthor ">Chun Yuen  Chow</span>, <span class="hlFld-ContribAuthor ">Glenn F.  King</span>. </span><span class="cited-content_cbyCitation_article-title">Fifteen years of Na
              V
              1.7 channels as an analgesic target: Why has excellent in vitro pharmacology not translated into in vivo analgesic efficacy?. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2020,</strong> <em>4 </em><a href="https://doi.org/10.1111/bph.15327" title="DOI URL">https://doi.org/10.1111/bph.15327</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.15327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.15327%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DFifteen%252Byears%252Bof%252BNa%252BV%252B1.7%252Bchannels%252Bas%252Ban%252Banalgesic%252Btarget%25253A%252BWhy%252Bhas%252Bexcellent%252Bin%252Bvitro%252Bpharmacology%252Bnot%252Btranslated%252Binto%252Bin%252Bvivo%252Banalgesic%252Befficacy%25253F%26aulast%3DEagles%26aufirst%3DDavid%2BA.%26date%3D2020%26date%3D2020%26volume%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen  McCarthy</span>, <span class="hlFld-ContribAuthor ">Jenna  Robinson</span>, <span class="hlFld-ContribAuthor ">Konstantinos  Thalassinos</span>, <span class="hlFld-ContribAuthor ">Alethea B.  Tabor</span>. </span><span class="cited-content_cbyCitation_article-title">A Chemical Biology Approach to Probing the Folding Pathways of the Inhibitory Cystine Knot (ICK) Peptide ProTx-II. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.00228" title="DOI URL">https://doi.org/10.3389/fchem.2020.00228</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.00228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.00228%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DA%252BChemical%252BBiology%252BApproach%252Bto%252BProbing%252Bthe%252BFolding%252BPathways%252Bof%252Bthe%252BInhibitory%252BCystine%252BKnot%252B%252528ICK%252529%252BPeptide%252BProTx-II%26aulast%3DMcCarthy%26aufirst%3DStephen%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jannis  KÃ¶rner</span>, <span class="hlFld-ContribAuthor ">Angelika  Lampert</span>. </span><span class="cited-content_cbyCitation_article-title">Sodium Channels. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 120-141. <a href="https://doi.org/10.1016/B978-0-12-809324-5.24208-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-809324-5.24208-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-809324-5.24208-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-809324-5.24208-9%26sid%3Dliteratum%253Aachs%26atitle%3DSodium%252BChannels%26aulast%3DK%25C3%25B6rner%26aufirst%3DJannis%26date%3D2020%26spage%3D120%26epage%3D141%26pub%3DElsevier%26atitle%3DThe%252BSenses%25253A%252BA%252BComprehensive%252BReference%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert S.  Foti</span>, <span class="hlFld-ContribAuthor ">Kaustav  Biswas</span>, <span class="hlFld-ContribAuthor ">Jennifer  Aral</span>, <span class="hlFld-ContribAuthor ">Xuhai  Be</span>, <span class="hlFld-ContribAuthor ">Loren  Berry</span>, <span class="hlFld-ContribAuthor ">Yuan  Cheng</span>, <span class="hlFld-ContribAuthor ">Kip  Conner</span>, <span class="hlFld-ContribAuthor ">James R.  Falsey</span>, <span class="hlFld-ContribAuthor ">Charles  Glaus</span>, <span class="hlFld-ContribAuthor ">Brad  Herberich</span>, <span class="hlFld-ContribAuthor ">Dean  Hickman</span>, <span class="hlFld-ContribAuthor ">Tayo  Ikotun</span>, <span class="hlFld-ContribAuthor ">Hongyan  Li</span>, <span class="hlFld-ContribAuthor ">Jason  Long</span>, <span class="hlFld-ContribAuthor ">Liyue  Huang</span>, <span class="hlFld-ContribAuthor ">Les P.  Miranda</span>, <span class="hlFld-ContribAuthor ">Justin  Murray</span>, <span class="hlFld-ContribAuthor ">Bryan  Moyer</span>, <span class="hlFld-ContribAuthor ">Chawita  Netirojjanakul</span>, <span class="hlFld-ContribAuthor ">Thomas E.  Nixey</span>, <span class="hlFld-ContribAuthor ">Kelvin  Sham</span>, <span class="hlFld-ContribAuthor ">Marcus  Soto</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Tegley</span>, <span class="hlFld-ContribAuthor ">Linh  Tran</span>, <span class="hlFld-ContribAuthor ">Bin  Wu</span>, <span class="hlFld-ContribAuthor ">Lin  Yin</span>, <span class="hlFld-ContribAuthor ">Dan A.  Rock</span>. </span><span class="cited-content_cbyCitation_article-title">Use of Cryopreserved Hepatocytes as Part of an Integrated Strategy to Characterize In Vivo Clearance for Peptide-Antibody Conjugate Inhibitors of Nav1.7 in Preclinical Species. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism and Disposition</span><span> <strong>2019,</strong> <em>47 </em>
                                    (10)
                                     , 1111-1121. <a href="https://doi.org/10.1124/dmd.119.087742" title="DOI URL">https://doi.org/10.1124/dmd.119.087742</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1124/dmd.119.087742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1124%2Fdmd.119.087742%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520and%2520Disposition%26atitle%3DUse%252Bof%252BCryopreserved%252BHepatocytes%252Bas%252BPart%252Bof%252Ban%252BIntegrated%252BStrategy%252Bto%252BCharacterize%252BIn%252BVivo%252BClearance%252Bfor%252BPeptide-Antibody%252BConjugate%252BInhibitors%252Bof%252BNav1.7%252Bin%252BPreclinical%252BSpecies%26aulast%3DFoti%26aufirst%3DRobert%2BS.%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D10%26spage%3D1111%26epage%3D1121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Selected binding model from docking of NMR structure ensemble of peptide <b>1</b>, Pra-[Nle6]JzTx-V(1â29) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CHC">6CHC</a>), into a homology model of human Na<sub>V</sub>1.7 domain II voltage sensor domain using ZDOCK. Peptide residues are labeled in black font, select Na<sub>V</sub>1.7 residues are labeled in white font. (B) Overlay of NMR structures of related JzTx-V peptides <b>1</b> (green/yellow) and AM-8145 (gray/magenta), showing alternate orientations of the Trp7 side chain indole moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Potent inhibition of TTX-S Na<sub>V</sub> channel currents by AM-6120 but not TTX-R Na<sub>V</sub> currents in mouse DRG neurons. (A) Representative whole cell patch clamp recordings for AM-6120 showing block of TTX-S Na<sub>V</sub> channels. TTX was used as a positive control. (B) Representative whole cell patch clamp recordings for AM-6120 showing no block of TTX-R currents at 0.5 Î¼M. Recordings were performed in the presence of 500 nM TTX to block endogenous TTX-S currents. The positive control Na<sub>V</sub>1.8 blocker 5-(4-chlorophenyl)-<i>N</i>-((2-(2,2,2-trifluoroethoxy)pyridin-3-yl)methyl)nicotinamide (Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf" class="ext-link">Figure S1</a>) fully blocked TTX-R currents at 0.5 Î¼M. (C) IC<sub>50</sub> values from above experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of JzTx-V-derived Na<sub>V</sub>1.7-inhibitory peptide AM-6120 solved at 1.05 Ã resolution using racemic crystallography. (A) Asymmetric unit showing one molecule of <span class="smallcaps smallerCapital">l</span>-peptide (blue) and one molecule of the <span class="smallcaps smallerCapital">d</span>-peptide (green). Disulfide bonds are in yellow; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CNU">6CNU</a>. (B) Only the <span class="smallcaps smallerCapital">l</span>-peptide is shown. Disulfide connectivity is labeled, and residues important for Na<sub>V</sub>1.7 potency are shown in salmon in stick representation. (C) Na<sub>V</sub>1.7 binding surface comparison of the SeMet6 analogue of AM-6120, peptide <b>42</b> (left) and NMR structure of peptide <b>1</b> (right). Key residues are shown as sticks and cluster on one side of the peptide structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Mouse pharmacokinetic profiles of 5-Br-Trp24 containing JzTx-V peptides AM-0714 and AM-6120 in male C57Bl/6 mice (<i>n</i> = 3/time point). The peptides were dosed subcutaneously at 2 mg/kg and monitored by LC-MS/MS. Data are mean Â± SEM for each time point. (B) Mouse pharmacokinetic parameters.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Reduction of scratch bouts in a mouse histamine-induced pruritis model with vehicle, AM-6120, and DPH (diphenhydramine, 30 mg/kg po dosing) (male C57Bl/6 mice, <i>n</i> = 7â10/group). ****, <i>p</i> < 0.0001 versus vehicle group (One-Way ANOVA followed by Dunnettâs tests). (B) Plasma exposure levels (Â±SEM) of AM-6120 and calculated exposure multiples over the mouse Na<sub>V</sub>1.7 PX IC<sub>50</sub> (5.1 nM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/medium/jm-2018-00736c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Linear amino acid sequences of peptides, highlighting variations from native JzTx-V in red font. The six cysteines are linked C<sub>1</sub>âC<sub>4</sub>, C<sub>2</sub>âC<sub>5</sub>, and C<sub>3</sub>âC<sub>6</sub> in an inhibitory cystine knot motif. (B) Chemical structure of lead peptide AM-6120 with the six residues different from native JzTx-V highlighted in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-21/acs.jmedchem.8b00736/20181101/images/large/jm-2018-00736c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00736&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80657" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80657" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Volkow, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, F. S.</span></span> <span> </span><span class="NLM_article-title">The role of science in addressing the opioid crisis</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>377</i></span>,  <span class="NLM_fpage">391</span>â <span class="NLM_lpage">394</span>, <span class="refDoi">Â DOI: 10.1056/NEJMsr1706626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1056%2FNEJMsr1706626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28564549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A280%3ADC%252BC1cnjt1yitQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=377&publication_year=2017&pages=391-394&author=N.+D.+Volkowauthor=F.+S.+Collins&title=The+role+of+science+in+addressing+the+opioid+crisis&doi=10.1056%2FNEJMsr1706626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Science in Addressing the Opioid Crisis</span></div><div class="casAuthors">Volkow Nora D; Collins Francis S</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">377</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">391-394</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQwGjdZICWoPiEJCPKhJPuqfW6udTcc2easj3u9w7uDe7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnjt1yitQ%253D%253D&md5=a6bb07206f42d3bbbcfadb92d2088e29</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMsr1706626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMsr1706626%26sid%3Dliteratum%253Aachs%26aulast%3DVolkow%26aufirst%3DN.%2BD.%26aulast%3DCollins%26aufirst%3DF.%2BS.%26atitle%3DThe%2520role%2520of%2520science%2520in%2520addressing%2520the%2520opioid%2520crisis%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2017%26volume%3D377%26spage%3D391%26epage%3D394%26doi%3D10.1056%2FNEJMsr1706626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D.</span></span> <span> </span><span class="NLM_article-title">The pain drain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">S2</span>â <span class="NLM_lpage">S3</span>, <span class="refDoi">Â DOI: 10.1038/535S2a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2F535S2a" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=S2-S3&author=D.+Holmes&title=The+pain+drain&doi=10.1038%2F535S2a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F535S2a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F535S2a%26sid%3Dliteratum%253Aachs%26aulast%3DHolmes%26aufirst%3DD.%26atitle%3DThe%2520pain%2520drain%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3DS2%26epage%3DS3%26doi%3D10.1038%2F535S2a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starobova, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobli, M.</span></span> <span> </span><span class="NLM_article-title">Na<sub>V</sub>1.7 as a pain target - from gene to pharmacology</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">73</span>â <span class="NLM_lpage">100</span>, <span class="refDoi">Â DOI: 10.1016/j.pharmthera.2016.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.pharmthera.2016.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=27916648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislClsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2017&pages=73-100&author=I.+Vetterauthor=J.+R.+Deuisauthor=A.+Muellerauthor=M.+R.+Israelauthor=H.+Starobovaauthor=A.+Zhangauthor=L.+D.+Rashauthor=M.+Mobli&title=NaV1.7+as+a+pain+target+-+from+gene+to+pharmacology&doi=10.1016%2Fj.pharmthera.2016.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">NaV1.7 as a pain target - From gene to pharmacology</span></div><div class="casAuthors">Vetter, Irina; Deuis, Jennifer R.; Mueller, Alexander; Israel, Mathilde R.; Starobova, Hana; Zhang, Alan; Rash, Lachlan D.; Mobli, Mehdi</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73-100</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">NaV1.7, a subtype of the voltage-gated sodium channel family that is highly expressed in peripheral sensory neurons, remains one of the most promising targets for the treatment of pain.  However, despite compelling genetic evidence supporting a key role for NaV1.7 in regulating excitability of peripheral sensory neurons, the development of truly subtype-selective inhibitors has been challenging.  Here, we discuss complexities surrounding targeting NaV1.7 pharmacol. for treatment of pain and explore future opportunities for development of effective analgesic NaV1.7 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgpB1bBe31X7Vg90H21EOLACvtfcHk0lj4qbomfTyfWA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislClsg%253D%253D&md5=78a0e17dfccd84b511491b6c5d83fd94</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2016.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2016.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DMueller%26aufirst%3DA.%26aulast%3DIsrael%26aufirst%3DM.%2BR.%26aulast%3DStarobova%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DA.%26aulast%3DRash%26aufirst%3DL.%2BD.%26aulast%3DMobli%26aufirst%3DM.%26atitle%3DNaV1.7%2520as%2520a%2520pain%2520target%2520-%2520from%2520gene%2520to%2520pharmacology%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D172%26spage%3D73%26epage%3D100%26doi%3D10.1016%2Fj.pharmthera.2016.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dib-Hajj, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxman, S. G.</span></span> <span> </span><span class="NLM_article-title">The Na<sub>V</sub>1.7 sodium channel: from molecule to man</span>. <i>Nat. Rev. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">49</span>â <span class="NLM_lpage">62</span>, <span class="refDoi">Â DOI: 10.1038/nrn3404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2Fnrn3404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=23232607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaisrrJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=49-62&author=S.+D.+Dib-Hajjauthor=Y.+Yangauthor=J.+A.+Blackauthor=S.+G.+Waxman&title=The+NaV1.7+sodium+channel%3A+from+molecule+to+man&doi=10.1038%2Fnrn3404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The NaV1.7 sodium channel: from molecule to man</span></div><div class="casAuthors">Dib-Hajj, Sulayman D.; Yang, Yang; Black, Joel A.; Waxman, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-62</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The voltage-gated sodium channel NaV1.7 is preferentially expressed in peripheral somatic and visceral sensory neurons, olfactory sensory neurons and sympathetic ganglion neurons.  NaV1.7 accumulates at nerve fiber endings and amplifies small subthreshold depolarizations, poising it to act as a threshold channel that regulates excitability.  Genetic and functional studies have added to the evidence that NaV1.7 is a major contributor to pain signaling in humans, and homol. modeling based on crystal structures of ion channels suggests an at.-level structural basis for the altered gating of mutant NaV1.7 that causes pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC4lZH1L7XWLVg90H21EOLACvtfcHk0lhjTjWqJ2-Y0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaisrrJ&md5=0e075d2c82d98774da1d316f4409d3eb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrn3404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn3404%26sid%3Dliteratum%253Aachs%26aulast%3DDib-Hajj%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DBlack%26aufirst%3DJ.%2BA.%26aulast%3DWaxman%26aufirst%3DS.%2BG.%26atitle%3DThe%2520NaV1.7%2520sodium%2520channel%253A%2520from%2520molecule%2520to%2520man%26jtitle%3DNat.%2520Rev.%2520Neurosci.%26date%3D2013%26volume%3D14%26spage%3D49%26epage%3D62%26doi%3D10.1038%2Fnrn3404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marron, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prime, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storer, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swain, N. A.</span></span> <span> </span><span class="NLM_article-title">Recent progress in sodium channel modulators for pain</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3690</span>â <span class="NLM_lpage">3699</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2014.06.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.bmcl.2014.06.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=25060923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Wls7vL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=3690-3699&author=S.+K.+Bagalauthor=M.+L.+Chapmanauthor=B.+E.+Marronauthor=R.+Primeauthor=R.+I.+Storerauthor=N.+A.+Swain&title=Recent+progress+in+sodium+channel+modulators+for+pain&doi=10.1016%2Fj.bmcl.2014.06.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress in sodium channel modulators for pain</span></div><div class="casAuthors">Bagal, Sharan K.; Chapman, Mark L.; Marron, Brian E.; Prime, Rebecca; Storer, R. Ian; Swain, Nigel A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3690-3699</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Voltage-gated sodium channels (Navs) are an important family of transmembrane ion channel proteins and Nav drug discovery is an exciting field.  Pharmaceutical investment in Navs for pain therapeutics has expanded exponentially due to genetic data such as SCN10A mutations and an improved ability to establish an effective screen sequence for example IonWorks Barracuda, Synchropatch and Qube.  Moreover, emerging clin. data (AZD-3161, XEN402, CNV1014802, PF-05089771, PF-04531083) combined with recent breakthroughs in Nav structural biol. pave the way for a future of fruitful prospective Nav drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbT6en4xkVxLVg90H21EOLACvtfcHk0lhjTjWqJ2-Y0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Wls7vL&md5=c8bae40b70509e2aa3dce106e2c41677</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.06.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.06.038%26sid%3Dliteratum%253Aachs%26aulast%3DBagal%26aufirst%3DS.%2BK.%26aulast%3DChapman%26aufirst%3DM.%2BL.%26aulast%3DMarron%26aufirst%3DB.%2BE.%26aulast%3DPrime%26aufirst%3DR.%26aulast%3DStorer%26aufirst%3DR.%2BI.%26aulast%3DSwain%26aufirst%3DN.%2BA.%26atitle%3DRecent%2520progress%2520in%2520sodium%2520channel%2520modulators%2520for%2520pain%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D3690%26epage%3D3699%26doi%3D10.1016%2Fj.bmcl.2014.06.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Lera Ruiz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, R. L.</span></span> <span> </span><span class="NLM_article-title">Voltage-gated sodium channels: structure, function, pharmacology, and clinical Indications</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7093</span>â <span class="NLM_lpage">7118</span>, <span class="refDoi">Â DOI: 10.1021/jm501981g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501981g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFemu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7093-7118&author=M.+de%0ALera+Ruizauthor=R.+L.+Kraus&title=Voltage-gated+sodium+channels%3A+structure%2C+function%2C+pharmacology%2C+and+clinical+Indications&doi=10.1021%2Fjm501981g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Voltage-Gated Sodium Channels: Structure, Function, Pharmacology, and Clinical Indications</span></div><div class="casAuthors">de Lera Ruiz, Manuel; Kraus, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7093-7118</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The tremendous therapeutic potential of voltage-gated sodium channels (Navs) has been the subject of many studies in the past and is of intense interest today.  Nav1.7 channels in particular have received much attention recently because of strong genetic validation of their involvement in nociception.  Here we summarize the current status of research in the Nav field and present the most relevant recent developments with respect to the mol. structure, general physiol., and pharmacol. of distinct Nav channel subtypes.  We discuss Nav channel ligands such as small mols., toxins isolated from animal venoms, and the recently identified Nav1.7-selective antibody.  Furthermore, we review eight characterized ligand binding sites on the Nav channel Î± subunit.  Finally, we examine possible therapeutic applications of Nav ligands and provide an update on current clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn2idyYye8oLVg90H21EOLACvtfcHk0lgYYRdKrcgevQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFemu7c%253D&md5=a816fdf5b46e2e5727669bf17e2e4fd4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm501981g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501981g%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BLera%2BRuiz%26aufirst%3DM.%26aulast%3DKraus%26aufirst%3DR.%2BL.%26atitle%3DVoltage-gated%2520sodium%2520channels%253A%2520structure%252C%2520function%252C%2520pharmacology%252C%2520and%2520clinical%2520Indications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7093%26epage%3D7118%26doi%3D10.1021%2Fjm501981g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmalhofer, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calhoun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrows, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohler, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinglass, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaczorowski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koltzenburg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priest, B. T.</span></span> <span> </span><span class="NLM_article-title">ProTx-II, a selective inhibitor of Na<sub>V</sub>1.7 sodium channels, blocks action potential propagation in nociceptors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1476</span>â <span class="NLM_lpage">1484</span>, <span class="refDoi">Â DOI: 10.1124/mol.108.047670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1124%2Fmol.108.047670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=18728100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlCnt7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2008&pages=1476-1484&author=W.+A.+Schmalhoferauthor=J.+Calhounauthor=R.+Burrowsauthor=T.+Baileyauthor=M.+G.+Kohlerauthor=A.+B.+Weinglassauthor=G.+J.+Kaczorowskiauthor=M.+L.+Garciaauthor=M.+Koltzenburgauthor=B.+T.+Priest&title=ProTx-II%2C+a+selective+inhibitor+of+NaV1.7+sodium+channels%2C+blocks+action+potential+propagation+in+nociceptors&doi=10.1124%2Fmol.108.047670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">ProTx-II, a selective inhibitor of NaV1.7 sodium channels, blocks action potential propagation in nociceptors</span></div><div class="casAuthors">Schmalhofer, William A.; Calhoun, Jeffrey; Burrows, Rachel; Bailey, Timothy; Kohler, Martin G.; Weinglass, Adam B.; Kaczorowski, Gregory J.; Garcia, Maria L.; Koltzenburg, Martin; Priest, Birgit T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1476-1484</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Voltage-gated sodium (NaV1) channels play a crit. role in modulating the excitability of sensory neurons, and human genetic evidence points to NaV1.7 as an essential contributor to pain signaling.  Human loss-of-function mutations in SCN9A, the gene encoding NaV1.7, cause channelopathy-assocd. indifference to pain (CIP), whereas gain-of-function mutations are assocd. with two inherited painful neuropathies.  Although the human genetic data make NaV1.7 an attractive target for the development of analgesics, pharmacol. proof-of-concept in exptl. pain models requires NaV1.7-selective channel blockers.  Here, we show that the tarantula venom peptide ProTx-II selectively interacts with NaV1.7 channels, inhibiting NaV1.7 with an IC50 value of 0.3 nM, compared with IC50 values of 30 to 150 nM for other heterologously expressed NaV1 subtypes.  This subtype selectivity was abolished by a point mutation in DIIS3.  It is interesting that application of ProTx-II to desheathed cutaneous nerves completely blocked the C-fiber compd. action potential at concns. that had little effect on AÎ²-fiber conduction.  ProTx-II application had little effect on action potential propagation of the intact nerve, which may explain why ProTx-II was not efficacious in rodent models of acute and inflammatory pain.  Mono-iodo-ProTx-II (125I-ProTx-II) binds with high affinity (Kd = 0.3 nM) to recombinant hNaV1.7 channels.  Binding of 125I-ProTx-II is insensitive to the presence of other well characterized NaV1 channel modulators, suggesting that ProTx-II binds to a novel site, which may be more conducive to conferring subtype selectivity than the site occupied by traditional local anesthetics and anticonvulsants.  Thus, the 125I-ProTx-II binding assay, described here, offers a new tool in the search for novel NaV1.7-selective blockers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDrCjZWFExrVg90H21EOLACvtfcHk0lgYYRdKrcgevQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlCnt7%252FN&md5=2d42ed47e6bcfcd04730120e1bc87eec</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1124%2Fmol.108.047670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.108.047670%26sid%3Dliteratum%253Aachs%26aulast%3DSchmalhofer%26aufirst%3DW.%2BA.%26aulast%3DCalhoun%26aufirst%3DJ.%26aulast%3DBurrows%26aufirst%3DR.%26aulast%3DBailey%26aufirst%3DT.%26aulast%3DKohler%26aufirst%3DM.%2BG.%26aulast%3DWeinglass%26aufirst%3DA.%2BB.%26aulast%3DKaczorowski%26aufirst%3DG.%2BJ.%26aulast%3DGarcia%26aufirst%3DM.%2BL.%26aulast%3DKoltzenburg%26aufirst%3DM.%26aulast%3DPriest%26aufirst%3DB.%2BT.%26atitle%3DProTx-II%252C%2520a%2520selective%2520inhibitor%2520of%2520NaV1.7%2520sodium%2520channels%252C%2520blocks%2520action%2520potential%2520propagation%2520in%2520nociceptors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2008%26volume%3D74%26spage%3D1476%26epage%3D1484%26doi%3D10.1124%2Fmol.108.047670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rahnama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardoso, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanujam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobli, M.</span></span> <span> </span><span class="NLM_article-title">The structure, dynamics and selectivity profile of a Na<sub>V</sub>1.7 potency-optimized huwentoxin-IV variant</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e0173551</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0173551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1371%2Fjournal.pone.0173551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28301520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WjurjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0173551&author=S.+Rahnamaauthor=J.+R.+Deuisauthor=F.+C.+Cardosoauthor=V.+Ramanujamauthor=R.+J.+Lewisauthor=L.+D.+Rashauthor=G.+F.+Kingauthor=I.+Vetterauthor=M.+Mobli&title=The+structure%2C+dynamics+and+selectivity+profile+of+a+NaV1.7+potency-optimized+huwentoxin-IV+variant&doi=10.1371%2Fjournal.pone.0173551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The structure, dynamics and selectivity profile of a NaV1.7 potency-optimised huwentoxin-IV variant</span></div><div class="casAuthors">Rahnama, Sassan; Deuis, Jennifer R.; Cardoso, Fernanda C.; Ramanujam, Venkatraman; Lewis, Richard J.; Rash, Lachlan D.; King, Glenn F.; Vetter, Irina; Mobli, Mehdi</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0173551/1-e0173551/19</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Venom-derived peptides have attracted much attention as potential lead mols. for pharmaceutical development.  A well-known example is Huwentoxin-IV (HwTx-IV), a peptide toxin isolated from the venom of the Chinese bird-eating spider Haplopelma schmitdi.  HwTx-IV was identified as a potent blocker of a human voltage-gated sodium channel (hNaV1.7), which is a genetically validated analgesic target.  The peptide was promising as it showed high potency at NaV1.7 (IC50 â¼26 nM) and selectivity over the cardiac NaV subtype (Nav1.5).  Mutagenesis studies aimed at optimizing the potency of the peptide resulted in the development of a triple-mutant of HwTx-IV (E1G, E4G, Y33W, m3-HwTx-IV) with significantly increased potency against hNaV1.7 (IC50 = 0.4 Â± 0.1 nM) without increased potency against hNaV1.5.  The activity of m3-HwTx-IV against other NaV subtypes was, however, not investigated.  Similarly, the structure of the mutant peptide was not characterised, limiting the interpretation of the obsd. increase in potency.  In this study we produced isotopelabeled recombinant m3-HwTx-IV in E. coli, which enabled us to characterize the atomicresoln. structure and dynamics of the peptide by NMR spectroscopy.  The results show that the structure of the peptide is not perturbed by the mutations, while the relaxation studies reveal that residues in the active site of the peptide undergo conformational exchange.  Addnl., the NaV subtype selectivity of the recombinant peptide was characterised, revealing potent inhibition of neuronal NaV subtypes 1.1, 1.2, 1.3, 1.6 and 1.7.  In parallel to the in vitro studies, we investigated NaV1.7 target engagement of the peptide in vivo using a rodent pain model, where m3-HwTx-IV dose-dependently suppressed spontaneous pain induced by the NaV1.7 activator OD1.  Thus, our results provide further insight into the structure and dynamics of this class of peptides that may prove useful in guiding the development of inhibitors with improved selectivity for analgesic NaV subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMIVGMcrmA4rVg90H21EOLACvtfcHk0liEXpktaZRtuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WjurjL&md5=a4d8353acf0b45c5310fdbdcf53d3462</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0173551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0173551%26sid%3Dliteratum%253Aachs%26aulast%3DRahnama%26aufirst%3DS.%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DCardoso%26aufirst%3DF.%2BC.%26aulast%3DRamanujam%26aufirst%3DV.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DRash%26aufirst%3DL.%2BD.%26aulast%3DKing%26aufirst%3DG.%2BF.%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DMobli%26aufirst%3DM.%26atitle%3DThe%2520structure%252C%2520dynamics%2520and%2520selectivity%2520profile%2520of%2520a%2520NaV1.7%2520potency-optimized%2520huwentoxin-IV%2520variant%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26spage%3De0173551%26doi%3D10.1371%2Fjournal.pone.0173551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingerd, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munasinghe, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhola, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imlach, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosengren, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosmans, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waxman, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dib-Hajj, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoubas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minett, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span> <span> </span><span class="NLM_article-title">Pharmacological characterisation of the highly Na<sub>V</sub>1.7 selective spider venom peptide Pn3a</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">40883</span>, <span class="refDoi">Â DOI: 10.1038/srep40883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2Fsrep40883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28106092" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFams74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=40883&author=J.+R.+Deuisauthor=Z.+Dekanauthor=J.+S.+Wingerdauthor=J.+J.+Smithauthor=N.+R.+Munasingheauthor=R.+F.+Bholaauthor=W.+L.+Imlachauthor=V.+Herzigauthor=D.+A.+Armstrongauthor=K.+J.+Rosengrenauthor=F.+Bosmansauthor=S.+G.+Waxmanauthor=S.+D.+Dib-Hajjauthor=P.+Escoubasauthor=M.+S.+Minettauthor=M.+J.+Christieauthor=G.+F.+Kingauthor=P.+F.+Alewoodauthor=R.+J.+Lewisauthor=J.+N.+Woodauthor=I.+Vetter&title=Pharmacological+characterisation+of+the+highly+NaV1.7+selective+spider+venom+peptide+Pn3a&doi=10.1038%2Fsrep40883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterisation of the highly NaV1.7 selective spider venom peptide Pn3a</span></div><div class="casAuthors">Deuis, Jennifer R.; Dekan, Zoltan; Wingerd, Joshua S.; Smith, Jennifer J.; Munasinghe, Nehan R.; Bhola, Rebecca F.; Imlach, Wendy L.; Herzig, Volker; Armstrong, David A.; Rosengren, K. Johan; Bosmans, Frank; Waxman, Stephen G.; Dib-Hajj, Sulayman D.; Escoubas, Pierre; Minett, Michael S.; Christie, Macdonald J.; King, Glenn F.; Alewood, Paul F.; Lewis, Richard J.; Wood, John N.; Vetter, Irina</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40883</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Human genetic studies have implicated the voltage-gated sodium channel NaV1.7 as a therapeutic target for the treatment of pain.  A novel peptide, Î¼-theraphotoxin-Pn3a, isolated from venom of the tarantula Pamphobeteus nigricolor, potently inhibits NaV1.7 (IC50 0.9 nM) with at least 40-1000-fold selectivity over all other NaV subtypes.  Despite on-target activity in small-diam. dorsal root ganglia, spinal slices, and in a mouse model of pain induced by NaV1.7 activation, Pn3a alone displayed no analgesic activity in formalin-, carrageenan- or FCA-induced pain in rodents when administered systemically.  A broad lack of analgesic activity was also found for the selective NaV1.7 inhibitors PF-04856264 and phlotoxin 1.  However, when administered with subtherapeutic doses of opioids or the enkephalinase inhibitor thiorphan, these subtype-selective NaV1.7 inhibitors produced profound analgesia.  Our results suggest that in these inflammatory models, acute administration of peripherally restricted NaV1.7 inhibitors can only produce analgesia when administered in combination with an opioid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOJrS6iZdTXLVg90H21EOLACvtfcHk0liEXpktaZRtuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFams74%253D&md5=8b98535d8a9f5652f25391993c78d82e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fsrep40883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep40883%26sid%3Dliteratum%253Aachs%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DDekan%26aufirst%3DZ.%26aulast%3DWingerd%26aufirst%3DJ.%2BS.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DMunasinghe%26aufirst%3DN.%2BR.%26aulast%3DBhola%26aufirst%3DR.%2BF.%26aulast%3DImlach%26aufirst%3DW.%2BL.%26aulast%3DHerzig%26aufirst%3DV.%26aulast%3DArmstrong%26aufirst%3DD.%2BA.%26aulast%3DRosengren%26aufirst%3DK.%2BJ.%26aulast%3DBosmans%26aufirst%3DF.%26aulast%3DWaxman%26aufirst%3DS.%2BG.%26aulast%3DDib-Hajj%26aufirst%3DS.%2BD.%26aulast%3DEscoubas%26aufirst%3DP.%26aulast%3DMinett%26aufirst%3DM.%2BS.%26aulast%3DChristie%26aufirst%3DM.%2BJ.%26aulast%3DKing%26aufirst%3DG.%2BF.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DWood%26aufirst%3DJ.%2BN.%26aulast%3DVetter%26aufirst%3DI.%26atitle%3DPharmacological%2520characterisation%2520of%2520the%2520highly%2520NaV1.7%2520selective%2520spider%2520venom%2520peptide%2520Pn3a%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D40883%26doi%3D10.1038%2Fsrep40883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flinspach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piekarz, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellows, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neff, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freedman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonesteel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eddinger, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaksh, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickenden, A. D.</span></span> <span> </span><span class="NLM_article-title">Insensitivity to pain induced by a potent selective closed-state Na<sub>V</sub>1.7 inhibitor</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">39662</span>, <span class="refDoi">Â DOI: 10.1038/srep39662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1038%2Fsrep39662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28045073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslGquw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=39662&author=M.+Flinspachauthor=Q.+Xuauthor=A.+D.+Piekarzauthor=R.+Fellowsauthor=R.+Haganauthor=A.+Gibbsauthor=Y.+Liuauthor=R.+A.+Neffauthor=J.+Freedmanauthor=W.+A.+Eckertauthor=M.+Zhouauthor=R.+Bonesteelauthor=M.+W.+Penningtonauthor=K.+A.+Eddingerauthor=T.+L.+Yakshauthor=M.+Hunterauthor=R.+V.+Swansonauthor=A.+D.+Wickenden&title=Insensitivity+to+pain+induced+by+a+potent+selective+closed-state+NaV1.7+inhibitor&doi=10.1038%2Fsrep39662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Insensitivity to pain induced by a potent selective closed-state Nav1.7 inhibitor</span></div><div class="casAuthors">Flinspach, M.; Xu, Q.; Piekarz, A. D.; Fellows, R.; Hagan, R.; Gibbs, A.; Liu, Y.; Neff, R. A.; Freedman, J.; Eckert, W. A.; Zhou, M.; Bonesteel, R.; Pennington, M. W.; Eddinger, K. A.; Yaksh, T. L.; Hunter, M.; Swanson, R. V.; Wickenden, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">39662</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pain places a devastating burden on patients and society and current pain therapeutics exhibit limitations in efficacy, unwanted side effects and the potential for drug abuse and diversion.  Although genetic evidence has clearly demonstrated that the voltage-gated sodium channel, Nav1.7, is crit. to pain sensation in mammals, pharmacol. inhibitors of Nav1.7 have not yet fully recapitulated the dramatic analgesia obsd. in Nav1.7-null subjects.  Using the tarantula venom-peptide ProTX-II as a scaffold, we engineered a library of over 1500 venom-derived peptides and identified JNJ63955918 as a potent, highly selective, closed-state Nav1.7 blocking peptide.  Here we show that JNJ63955918 induces a pharmacol. insensitivity to pain that closely recapitulates key features of the Nav1.7-null phenotype seen in mice and humans.  Our findings demonstrate that a high degree of selectivity, coupled with a closed-state dependent mechanism of action is required for strong efficacy and indicate that peptides such as JNJ63955918 and other suitably optimized Nav1.7 inhibitors may represent viable non-opioid alternatives for the pharmacol. treatment of severe pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH9Z1m5kOM1bVg90H21EOLACvtfcHk0lgxf55TuDo9cg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslGquw%253D%253D&md5=1da5e9d488eec322b2f771e9b21e731b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsrep39662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep39662%26sid%3Dliteratum%253Aachs%26aulast%3DFlinspach%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DPiekarz%26aufirst%3DA.%2BD.%26aulast%3DFellows%26aufirst%3DR.%26aulast%3DHagan%26aufirst%3DR.%26aulast%3DGibbs%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNeff%26aufirst%3DR.%2BA.%26aulast%3DFreedman%26aufirst%3DJ.%26aulast%3DEckert%26aufirst%3DW.%2BA.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBonesteel%26aufirst%3DR.%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DEddinger%26aufirst%3DK.%2BA.%26aulast%3DYaksh%26aufirst%3DT.%2BL.%26aulast%3DHunter%26aufirst%3DM.%26aulast%3DSwanson%26aufirst%3DR.%2BV.%26aulast%3DWickenden%26aufirst%3DA.%2BD.%26atitle%3DInsensitivity%2520to%2520pain%2520induced%2520by%2520a%2520potent%2520selective%2520closed-state%2520NaV1.7%2520inhibitor%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D39662%26doi%3D10.1038%2Fsrep39662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">StÃ¶cklin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Engineering potent and selective analogues of GpTx-1, a tarantula venom peptide antagonist of the Na<sub>V</sub>1.7 sodium channel</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2299</span>â <span class="NLM_lpage">2314</span>, <span class="refDoi">Â DOI: 10.1021/jm501765v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501765v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1Snurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2299-2314&author=J.+K.+Murrayauthor=J.+Liguttiauthor=D.+Liuauthor=A.+Zouauthor=L.+Poppeauthor=H.+Liauthor=K.+L.+Andrewsauthor=B.+D.+Moyerauthor=S.+I.+McDonoughauthor=P.+Favreauauthor=R.+St%C3%B6cklinauthor=L.+P.+Miranda&title=Engineering+potent+and+selective+analogues+of+GpTx-1%2C+a+tarantula+venom+peptide+antagonist+of+the+NaV1.7+sodium+channel&doi=10.1021%2Fjm501765v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering Potent and Selective Analogues of GpTx-1, a Tarantula Venom Peptide Antagonist of the NaV1.7 Sodium Channel</span></div><div class="casAuthors">Murray, Justin K.; Ligutti, Joseph; Liu, Dong; Zou, Anruo; Poppe, Leszek; Li, Hongyan; Andrews, Kristin L.; Moyer, Bryan D.; McDonough, Stefan I.; Favreau, Philippe; Stocklin, Reto; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2299-2314</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NaV1.7 is a voltage-gated sodium ion channel implicated by human genetic evidence as a therapeutic target for the treatment of pain.  Screening fractionated venom from the tarantula Grammostola porteri led to the identification of a 34-residue peptide, termed GpTx-1, with potent activity on NaV1.7 (IC50 = 10 nM) and promising selectivity against key NaV subtypes (20Ã and 1000Ã over NaV1.4 and NaV1.5, resp.).  NMR structural anal. of the chem. synthesized three disulfide peptide was consistent with an inhibitory cystine knot motif.  Alanine scanning of GpTx-1 revealed that residues Trp29, Lys31, and Phe34 near the C-terminus are crit. for potent NaV1.7 antagonist activity.  Substitution of Ala for Phe at position 5 conferred 300-fold selectivity against NaV1.4.  A structure-guided campaign afforded additive improvements in potency and NaV subtype selectivity, culminating in the design of [Ala 5,Phe 6,Leu 26,Arg 28]GpTx-1 with a NaV1.7 IC50 value of 1.6 nM and >1000Ã selectivity against NaV1.4 and NaV1.5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryY8lf_XnPDbVg90H21EOLACvtfcHk0lgxf55TuDo9cg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1Snurs%253D&md5=28528454195c1d12d76a132dfa44273e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm501765v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501765v%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DSt%25C3%25B6cklin%26aufirst%3DR.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DEngineering%2520potent%2520and%2520selective%2520analogues%2520of%2520GpTx-1%252C%2520a%2520tarantula%2520venom%2520peptide%2520antagonist%2520of%2520the%2520NaV1.7%2520sodium%2520channel%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2299%26epage%3D2314%26doi%3D10.1021%2Fjm501765v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.
K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holder, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Sustained inhibition of the Na<sub>V</sub>1.7 sodium channel by engineered dimers of the domain II binding peptide GpTx-1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">4866</span>â <span class="NLM_lpage">4871</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.06.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.bmcl.2015.06.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=26112439" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVeiurzI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=4866-4871&author=J.%0AK.+Murrayauthor=K.+Biswasauthor=J.+R.+Holderauthor=A.+Zouauthor=J.+Liguttiauthor=D.+Liuauthor=L.+Poppeauthor=K.+L.+Andrewsauthor=F.+F.+Linauthor=S.+Y.+Mengauthor=B.+D.+Moyerauthor=S.+I.+McDonoughauthor=L.+P.+Miranda&title=Sustained+inhibition+of+the+NaV1.7+sodium+channel+by+engineered+dimers+of+the+domain+II+binding+peptide+GpTx-1&doi=10.1016%2Fj.bmcl.2015.06.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained inhibition of the NaV1.7 sodium channel by engineered dimers of the domain II binding peptide GpTx-1</span></div><div class="casAuthors">Murray, Justin K.; Biswas, Kaustav; Holder, J. Ryan; Zou, Anruo; Ligutti, Joseph; Liu, Dong; Poppe, Leszek; Andrews, Kristin L.; Lin, Fen-Fen; Meng, Shi-Yuan; Moyer, Bryan D.; McDonough, Stefan I.; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4866-4871</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Many efforts are underway to develop selective inhibitors of the voltage-gated sodium channel NaV1.7 as new analgesics.  Thus far, however, in vitro selectivity has proved difficult for small mols., and peptides generally lack appropriate pharmacokinetic properties.  The authors previously identified the NaV1.7 inhibitory peptide GpTx-1 from tarantula venom and optimized its potency and selectivity via structure-guided analoging.  To further understand GpTx-1 binding to NaV1.7, the authors have mapped the binding site to transmembrane segments 1-4 of the second pseudo-subunit internal repeat (commonly referred to as Site 4) using NaV1.5/NaV1.7 chimeric protein constructs.  The authors also report that select GpTx-1 amino acid residues apparently not contacting NaV1.7 can be derivatized with a hydrophilic polymer without adversely affecting peptide potency.  Homodimerization of GpTx-1 with a bifunctional polyethylene glycol (PEG) linker resulted in a compd. with increased potency and a significantly reduced off-rate, demonstrating the ability to modulate the function and properties of GpTx-1 by linking to addnl. mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeHQHHy8plgbVg90H21EOLACvtfcHk0lgxf55TuDo9cg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVeiurzI&md5=d3f84f31eb02f132394811458242b646</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.06.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.06.033%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DHolder%26aufirst%3DJ.%2BR.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DLin%26aufirst%3DF.%2BF.%26aulast%3DMeng%26aufirst%3DS.%2BY.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DSustained%2520inhibition%2520of%2520the%2520NaV1.7%2520sodium%2520channel%2520by%2520engineered%2520dimers%2520of%2520the%2520domain%2520II%2520binding%2520peptide%2520GpTx-1%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D4866%26epage%3D4871%26doi%3D10.1016%2Fj.bmcl.2015.06.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Single residue substitutions that confer voltage-gated sodium ion channel subtype selectivity in the Na<sub>V</sub>1.7 inhibitory peptide GpTx-1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2704</span>â <span class="NLM_lpage">2717</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC28XislOnsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2704-2717&author=J.+K.+Murrayauthor=J.+Longauthor=A.+Zouauthor=J.+Liguttiauthor=K.+L.+Andrewsauthor=L.+Poppeauthor=K.+Biswasauthor=B.+D.+Moyerauthor=S.+I.+McDonoughauthor=L.+P.+Miranda&title=Single+residue+substitutions+that+confer+voltage-gated+sodium+ion+channel+subtype+selectivity+in+the+NaV1.7+inhibitory+peptide+GpTx-1&doi=10.1021%2Facs.jmedchem.5b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Single Residue Substitutions That Confer Voltage-Gated Sodium Ion Channel Subtype Selectivity in the NaV1.7 Inhibitory Peptide GpTx-1</span></div><div class="casAuthors">Murray, Justin K.; Long, Jason; Zou, Anruo; Ligutti, Joseph; Andrews, Kristin L.; Poppe, Leszek; Biswas, Kaustav; Moyer, Bryan D.; McDonough, Stefan I.; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2704-2717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is interest in the identification and optimization of new mol. entities selectively targeting ion channels of therapeutic relevance.  Peptide toxins represent a rich source of pharmacol. for ion channels, and we recently reported GpTx-1 analogs that inhibit NaV1.7, a voltage-gated sodium ion channel that is a compelling target for improved treatment of pain.  Here we utilize multi-attribute positional scan (MAPS) analoging, combining high-throughput synthesis and electrophysiol., to interrogate the interaction of GpTx-1 with NaV1.7 and related NaV subtypes.  After one round of MAPS analoging, we found novel substitutions at multiple residue positions not previously identified, specifically glutamic acid at positions 10 or 11 or lysine at position 18, that produce peptides with single digit nanomolar potency on NaV1.7 and 500-fold selectivity against off-target sodium channels.  Docking studies with a NaV1.7 homol. model and peptide NMR structure generated a model consistent with the key potency and selectivity modifications mapped in this work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqcQfq19Y1YrVg90H21EOLACvtfcHk0lj46RSyzTDH6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XislOnsbg%253D&md5=cf317a857972d1d9897f8710504ccee0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01947%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DSingle%2520residue%2520substitutions%2520that%2520confer%2520voltage-gated%2520sodium%2520ion%2520channel%2520subtype%2520selectivity%2520in%2520the%2520NaV1.7%2520inhibitory%2520peptide%2520GpTx-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2704%26epage%3D2717%26doi%3D10.1021%2Facs.jmedchem.5b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sham, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">StÃ¶cklin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Pharmacological characterization of potent and selective Na<sub>V</sub>1.7 inhibitors engineered from <i>Chilobrachys jingzhao</i> tarantula venom peptide JzTx-V</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">e0196791</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0196791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1371%2Fjournal.pone.0196791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=29723257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsl2iurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=e0196791&author=B.+D.+Moyerauthor=J.+K.+Murrayauthor=J.+Liguttiauthor=K.+Andrewsauthor=P.+Favreauauthor=J.+B.+Jordanauthor=J.+H.+Leeauthor=D.+Liuauthor=J.+Longauthor=K.+Shamauthor=L.+Shiauthor=R.+St%C3%B6cklinauthor=B.+Wuauthor=R.+Yinauthor=V.+Yuauthor=A.+Zouauthor=K.+Biswasauthor=L.+P.+Miranda&title=Pharmacological+characterization+of+potent+and+selective+NaV1.7+inhibitors+engineered+from+Chilobrachys+jingzhao+tarantula+venom+peptide+JzTx-V&doi=10.1371%2Fjournal.pone.0196791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of potent and selective NaV1.7 inhibitors engineered from Chilobrachys jingzhao tarantula venom peptide JzTx-V</span></div><div class="casAuthors">Moyer, Bryan D.; Murray, Justin K.; Ligutti, Joseph; Andrews, Kristin; Favreau, Philippe; Jordan, John B.; Lee, Josie H.; Liu, Dong; Long, Jason; Sham, Kelvin; Shi, Licheng; Stocklin, Reto; Wu, Bin; Yin, Ruoyuan; Yu, Violeta; Zou, Anruo; Biswas, Kaustav; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e0196791/1-e0196791/23</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Identification of voltage-gated sodium channel NaV1.7 inhibitors for chronic pain therapeutic development is an area of vigorous pursuit.  In an effort to identify more potent leads compared to our previously reported GpTx-1 peptide series, electrophysiol. screening of fractionated tarantula venom discovered the NaV1.7 inhibitory peptide JzTx-V from the Chinese earth tiger tarantula Chilobrachys jingzhao.  The parent peptide displayed nominal selectivity over the skeletal muscle NaV1.4 channel.  Attribute-based positional scan analoging identified a key Ile28Glu mutation that improved NaV1.4 selectivity over 100-fold, and further optimization yielded the potent and selective peptide leads AM-8145 and AM-0422.  NMR analyses revealed that the Ile28Glu substitution changed peptide conformation, pointing to a structural rationale for the selectivity gains.  AM-8145 and AM-0422 as well as GpTx-1 and HwTx-IV competed for ProTx-II binding in HEK293 cells expressing human NaV1.7, suggesting that these NaV1.7 inhibitory peptides interact with a similar binding site.  AM-8145 potently blocked native tetrodotoxin-sensitive (TTX-S) channels in mouse dorsal root ganglia (DRG) neurons, exhibited 30- to 120-fold selectivity over other human TTX-S channels and exhibited over 1,000-fold selectivity over other human tetrodotoxin-resistant (TTX-R) channels.  Leveraging NaV1.7-NaV1.5 chimeras contg. various voltage-sensor and pore regions, AM-8145 mapped to the second voltage-sensor domain of NaV1.7.  AM-0422, but not the inactive peptide analog AM-8374, dose-dependently blocked capsaicin-induced DRG neuron action potential firing using a multi-electrode array readout and mech.-induced C-fiber spiking in a saphenous skin-nerve prepn.  Collectively, AM-8145 and AM-0422 represent potent, new engineered NaV1.7 inhibitory peptides derived from the JzTx-V scaffold with improved NaV selectivity and biol. activity in blocking action potential firing in both DRG neurons and C-fibers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr_v8GXWwSxLVg90H21EOLACvtfcHk0lj46RSyzTDH6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsl2iurbK&md5=4478846639ebeda482156c38676e6ab3</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0196791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0196791%26sid%3Dliteratum%253Aachs%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DK.%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DJordan%26aufirst%3DJ.%2BB.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DSham%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSt%25C3%25B6cklin%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DV.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DPharmacological%2520characterization%2520of%2520potent%2520and%2520selective%2520NaV1.7%2520inhibitors%2520engineered%2520from%2520Chilobrachys%2520jingzhao%2520tarantula%2520venom%2520peptide%2520JzTx-V%26jtitle%3DPLoS%2520One%26date%3D2018%26volume%3D13%26spage%3De0196791%26doi%3D10.1371%2Fjournal.pone.0196791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deuis, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingerd, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weidner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nassar, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span> <span> </span><span class="NLM_article-title">Analgesic effects of GpTx-1, PF-04856264 and CNV1014802 in a mouse model of Na<sub>V</sub>1.7-mediated pain</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">78</span>, <span class="refDoi">Â DOI: 10.3390/toxins8030078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.3390%2Ftoxins8030078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCmurrM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=78&author=J.+R.+Deuisauthor=J.+S.+Wingerdauthor=Z.+Winterauthor=T.+Durekauthor=Z.+Dekanauthor=S.+R.+Sousaauthor=K.+Zimmermannauthor=T.+Hoffmannauthor=C.+Weidnerauthor=M.+A.+Nassarauthor=P.+F.+Alewoodauthor=R.+J.+Lewisauthor=I.+Vetter&title=Analgesic+effects+of+GpTx-1%2C+PF-04856264+and+CNV1014802+in+a+mouse+model+of+NaV1.7-mediated+pain&doi=10.3390%2Ftoxins8030078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Analgesic effects of GpTx-1, PF-04856264 and CNV1014802 in a mouse model of NaV1.7-mediated pain</span></div><div class="casAuthors">Deuis, Jennifer R.; Wingerd, Joshua S.; Winter, Zoltan; Durek, Thomas; Dekan, Zoltan; Sousa, Silmara R.; Zimmermann, Katharina; Hoffmann, Tali; Weidner, Christian; Nassar, Mohammed A.; Alewood, Paul F.; Lewis, Richard J.; Vetter, Irina</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">78/1-78/19</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Loss-of-function mutations of NaV1.7 lead to congenital insensitivity to pain, a rare condition resulting in individuals who are otherwise normal except for the inability to sense pain, making pharmacol. inhibition of NaV1.7 a promising therapeutic strategy for the treatment of pain.  We characterized a novel mouse model of NaV1.7-mediated pain based on intraplantar injection of the scorpion toxin OD1, which is suitable for rapid in vivo profiling of NaV1.7 inhibitors.  Intraplantar injection of OD1 caused spontaneous pain behaviors, which were reversed by co-injection with NaV1.7 inhibitors and significantly reduced in NaV1.7-/- mice.  To validate the use of the model for profiling NaV1.7 inhibitors, we detd. the NaV selectivity and tested the efficacy of the reported NaV1.7 inhibitors GpTx-1, PF-04856264 and CNV1014802 (raxatrigine).  GpTx-1 selectively inhibited NaV1.7 and was effective when co-administered with OD1, but lacked efficacy when delivered systemically.  PF-04856264 state-dependently and selectively inhibited NaV1.7 and significantly reduced OD1-induced spontaneous pain when delivered locally and systemically.  CNV1014802 state-dependently, but non-selectively, inhibited NaV channels and was only effective in the OD1 model when delivered systemically.  Our novel model of NaV1.7-mediated pain based on intraplantar injection of OD1 is thus suitable for the rapid in vivo characterization of the analgesic efficacy of NaV1.7 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn5aBTBFoLvbVg90H21EOLACvtfcHk0libPbSCiCGAmA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCmurrM&md5=335b5e4c4747e94c1ac7750c47139fae</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3390%2Ftoxins8030078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins8030078%26sid%3Dliteratum%253Aachs%26aulast%3DDeuis%26aufirst%3DJ.%2BR.%26aulast%3DWingerd%26aufirst%3DJ.%2BS.%26aulast%3DWinter%26aufirst%3DZ.%26aulast%3DDurek%26aufirst%3DT.%26aulast%3DDekan%26aufirst%3DZ.%26aulast%3DSousa%26aufirst%3DS.%2BR.%26aulast%3DZimmermann%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DWeidner%26aufirst%3DC.%26aulast%3DNassar%26aufirst%3DM.%2BA.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DVetter%26aufirst%3DI.%26atitle%3DAnalgesic%2520effects%2520of%2520GpTx-1%252C%2520PF-04856264%2520and%2520CNV1014802%2520in%2520a%2520mouse%2520model%2520of%2520NaV1.7-mediated%2520pain%26jtitle%3DToxins%26date%3D2016%26volume%3D8%26spage%3D78%26doi%3D10.3390%2Ftoxins8030078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aral, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenkilsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salyers, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sickmier, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGivern, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gegg, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical optimization of peptide toxins for ion channel targets: potent, selective, and long-lived antagonists of K<sub>V</sub>1.3</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6784</span>â <span class="NLM_lpage">6802</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00495</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00495" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOjtbjK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6784-6802&author=J.+K.+Murrayauthor=Y.+X.+Qianauthor=B.+Liuauthor=R.+Elliottauthor=J.+Aralauthor=C.+Parkauthor=X.+Zhangauthor=M.+Stenkilssonauthor=K.+Salyersauthor=M.+Roseauthor=H.+Liauthor=S.+Yuauthor=K.+L.+Andrewsauthor=A.+Colomberoauthor=J.+Wernerauthor=K.+Gaidaauthor=E.+A.+Sickmierauthor=P.+Miuauthor=A.+Itanoauthor=J.+McGivernauthor=C.+V.+Geggauthor=J.+K.+Sullivanauthor=L.+P.+Miranda&title=Pharmaceutical+optimization+of+peptide+toxins+for+ion+channel+targets%3A+potent%2C+selective%2C+and+long-lived+antagonists+of+KV1.3&doi=10.1021%2Facs.jmedchem.5b00495"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical Optimization of Peptide Toxins for Ion Channel Targets: Potent, Selective, and Long-Lived Antagonists of Kv1.3</span></div><div class="casAuthors">Murray, Justin K.; Qian, Yi-Xin; Liu, Benxian; Elliott, Robin; Aral, Jennifer; Park, Cynthia; Zhang, Xuxia; Stenkilsson, Michael; Salyers, Kevin; Rose, Mark; Li, Hongyan; Yu, Steven; Andrews, Kristin L.; Colombero, Anne; Werner, Jonathan; Gaida, Kevin; Sickmier, E. Allen; Miu, Peter; Itano, Andrea; McGivern, Joseph; Gegg, Colin V.; Sullivan, John K.; Miranda, Les P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6784-6802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To realize the medicinal potential of peptide toxins, naturally occurring disulfide-rich peptides, as ion channel antagonists, more efficient pharmaceutical optimization technologies must be developed.  The therapeutic properties of multiple cysteine toxin peptides can be rapidly and substantially improved by combining direct chem. strategies with high-throughput electrophysiol.  The authors applied whole-mol., brute-force, structure-activity analoging to ShK, a peptide toxin from the sea anemone Stichodactyla helianthus that inhibits the voltage-gated potassium ion channel Kv1.3, to effectively discover crit. structural changes for 15Ã selectivity against the closely related neuronal ion channel Kv1.1.  Subsequent site-specific polymer conjugation resulted in an exquisitely selective Kv1.3 antagonist (>1000Ã over Kv1.1) with picomolar functional activity in whole blood and a pharmacokinetic profile suitable for weekly administration in primates.  The pharmacol. potential of the optimized toxin peptide was demonstrated by potent and sustained inhibition of cytokine secretion from T cells, a therapeutic target for autoimmune diseases, in cynomolgus monkeys.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEtF-Ugp8jFbVg90H21EOLACvtfcHk0libPbSCiCGAmA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOjtbjK&md5=1cfb03e783bd953480c203dc9b95af84</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00495%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DQian%26aufirst%3DY.%2BX.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DElliott%26aufirst%3DR.%26aulast%3DAral%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DStenkilsson%26aufirst%3DM.%26aulast%3DSalyers%26aufirst%3DK.%26aulast%3DRose%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DAndrews%26aufirst%3DK.%2BL.%26aulast%3DColombero%26aufirst%3DA.%26aulast%3DWerner%26aufirst%3DJ.%26aulast%3DGaida%26aufirst%3DK.%26aulast%3DSickmier%26aufirst%3DE.%2BA.%26aulast%3DMiu%26aufirst%3DP.%26aulast%3DItano%26aufirst%3DA.%26aulast%3DMcGivern%26aufirst%3DJ.%26aulast%3DGegg%26aufirst%3DC.%2BV.%26aulast%3DSullivan%26aufirst%3DJ.%2BK.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DPharmaceutical%2520optimization%2520of%2520peptide%2520toxins%2520for%2520ion%2520channel%2520targets%253A%2520potent%252C%2520selective%252C%2520and%2520long-lived%2520antagonists%2520of%2520KV1.3%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6784%26epage%3D6802%26doi%3D10.1021%2Facs.jmedchem.5b00495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pathak, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarov-Yarovoy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohout, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tombola, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacoff, E. Y.</span></span> <span> </span><span class="NLM_article-title">Closing in on the resting state of the Shaker K<sup>+</sup> channel</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">124</span>â <span class="NLM_lpage">140</span>, <span class="refDoi">Â DOI: 10.1016/j.neuron.2007.09.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.neuron.2007.09.023" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2007&pages=124-140&author=M.+M.+Pathakauthor=V.+Yarov-Yarovoyauthor=G.+Agarwalauthor=B.+Rouxauthor=P.+Barthauthor=S.+Kohoutauthor=F.+Tombolaauthor=E.+Y.+Isacoff&title=Closing+in+on+the+resting+state+of+the+Shaker+K%2B+channel&doi=10.1016%2Fj.neuron.2007.09.023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2007.09.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2007.09.023%26sid%3Dliteratum%253Aachs%26aulast%3DPathak%26aufirst%3DM.%2BM.%26aulast%3DYarov-Yarovoy%26aufirst%3DV.%26aulast%3DAgarwal%26aufirst%3DG.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DBarth%26aufirst%3DP.%26aulast%3DKohout%26aufirst%3DS.%26aulast%3DTombola%26aufirst%3DF.%26aulast%3DIsacoff%26aufirst%3DE.%2BY.%26atitle%3DClosing%2520in%2520on%2520the%2520resting%2520state%2520of%2520the%2520Shaker%2520K%252B%2520channel%26jtitle%3DNeuron%26date%3D2007%26volume%3D56%26spage%3D124%26epage%3D140%26doi%3D10.1016%2Fj.neuron.2007.09.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, Z. P.</span></span> <span> </span><span class="NLM_article-title">Docking unbound proteins using shape complementarity, desolvation, and electrostatics</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">281</span>â <span class="NLM_lpage">294</span>, <span class="refDoi">Â DOI: 10.1002/prot.10092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1002%2Fprot.10092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=11948782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFantLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2002&pages=281-294&author=R.+Chenauthor=Z.+P.+Weng&title=Docking+unbound+proteins+using+shape+complementarity%2C+desolvation%2C+and+electrostatics&doi=10.1002%2Fprot.10092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Docking unbound proteins using shape complementarity, desolvation, and electrostatics</span></div><div class="casAuthors">Chen, Rong; Weng, Zhiping</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">281-294</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A comprehensive docking study was performed on 27 distinct protein-protein complexes.  For 13 test systems, docking was performed with the unbound x-ray structures of both the receptor and the ligand.  For the remaining systems, the unbound x-ray structure of only mol. was available; therefore the bound structure for the other mol. was used.  Our method optimizes desolvation, shape complementarity, and electrostatics using a Fast Fourier Transform algorithm.  A global search in the rotational and translational space without any knowledge of the binding sites was performed for all proteins except nine antibodies recognizing antigens.  For these antibodies, we docked their well-characterized binding site-the complementarity-detg. region defined without information of the antigen-to the entire surface of the antigen.  For 24 systems, we were able to find near-native ligand orientations (interface CÎ± root mean square deviation less than 2.5 Ã from the crystal complex) among the top 2,000 choices.  For three systems, our algorithm could identify the correct complex structure unambiguously.  For 13 other complexes, we either ranked a near-native structure in the top 20 or obtained 20 or more near-native structures in the top 2000 or both.  The key feature of our algorithm is the use of target functions that are highly tolerant to conformational changes upon binding.  If combined with a post-processing method, our algorithm may provide a general soln. to the unbound docking problem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCxIONKClpN7Vg90H21EOLACvtfcHk0liMod_ZNphrJA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFantLg%253D&md5=7e7d07996a899a90ee71b758cdee429d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fprot.10092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10092%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DWeng%26aufirst%3DZ.%2BP.%26atitle%3DDocking%2520unbound%2520proteins%2520using%2520shape%2520complementarity%252C%2520desolvation%252C%2520and%2520electrostatics%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2002%26volume%3D47%26spage%3D281%26epage%3D294%26doi%3D10.1002%2Fprot.10092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, J. O.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummins, T. R.</span></span> <span> </span><span class="NLM_article-title">Common molecular determinants of tarantula huwentoxin-IV inhibition of Na<sup>+</sup> channel voltage sensors in domains II and IV</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">27301</span>â <span class="NLM_lpage">27310</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M111.246876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1074%2Fjbc.M111.246876" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=27301-27310&author=Y.+Xiaoauthor=J.+O.+Jacksonauthor=S.+Liangauthor=T.+R.+Cummins&title=Common+molecular+determinants+of+tarantula+huwentoxin-IV+inhibition+of+Na%2B+channel+voltage+sensors+in+domains+II+and+IV&doi=10.1074%2Fjbc.M111.246876"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.246876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.246876%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJackson%26aufirst%3DJ.%2BO.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DCummins%26aufirst%3DT.%2BR.%26atitle%3DCommon%2520molecular%2520determinants%2520of%2520tarantula%2520huwentoxin-IV%2520inhibition%2520of%2520Na%252B%2520channel%2520voltage%2520sensors%2520in%2520domains%2520II%2520and%2520IV%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D27301%26epage%3D27310%26doi%3D10.1074%2Fjbc.M111.246876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klint, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupasinghe, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Er, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span> <span> </span><span class="NLM_article-title">Spider-venom peptides that target voltage-gated sodium channels: Pharmacological tools and potential therapeutic leads</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">478</span>â <span class="NLM_lpage">491</span>, <span class="refDoi">Â DOI: 10.1016/j.toxicon.2012.04.337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.toxicon.2012.04.337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=22543187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVyqu7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=478-491&author=J.+K.+Klintauthor=S.+Senffauthor=D.+B.+Rupasingheauthor=S.+Y.+Erauthor=V.+Herzigauthor=G.+M.+Nicholsonauthor=G.+F.+King&title=Spider-venom+peptides+that+target+voltage-gated+sodium+channels%3A+Pharmacological+tools+and+potential+therapeutic+leads&doi=10.1016%2Fj.toxicon.2012.04.337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Spider-venom peptides that target voltage-gated sodium channels: Pharmacological tools and potential therapeutic leads</span></div><div class="casAuthors">Klint, Julie K.; Senff, Sebastian; Rupasinghe, Darshani B.; Er, Sing Yan; Herzig, Volker; Nicholson, Graham M.; King, Glenn F.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">478-491</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Voltage-gated Na (NaV) channels play a central role in the propagation of action potentials in excitable cells in both humans and insects.  Many venomous animals have therefore evolved toxins that modulate the activity of NaV channels to subdue their prey and deter predators.  Spider venoms in particular are rich in NaV channel modulators, with 1/3 of all known ion channel toxins from spider venoms acting on NaV channels.  Here we review the landscape of spider-venom peptides that have so far been described to target vertebrate or invertebrate NaV channels.  These peptides fall into 12 distinct families based on their primary structure and cysteine scaffold.  Some of these peptides have become useful pharmacol. tools, while others have potential as therapeutic leads because they target specific NaV channel subtypes that are considered to be important analgesic targets.  Spider venoms are conservatively predicted to contain more than 10 million bioactive peptides and so far only 0.01% of this diversity been characterized.  Thus, it is likely that future research will reveal addnl. structural classes of spider-venom peptides that target NaV channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrsU_ooS6kmbVg90H21EOLACvtfcHk0liMod_ZNphrJA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVyqu7Y%253D&md5=35d3c33bd548ea6bf2d99b7702a7d623</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2012.04.337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2012.04.337%26sid%3Dliteratum%253Aachs%26aulast%3DKlint%26aufirst%3DJ.%2BK.%26aulast%3DSenff%26aufirst%3DS.%26aulast%3DRupasinghe%26aufirst%3DD.%2BB.%26aulast%3DEr%26aufirst%3DS.%2BY.%26aulast%3DHerzig%26aufirst%3DV.%26aulast%3DNicholson%26aufirst%3DG.%2BM.%26aulast%3DKing%26aufirst%3DG.%2BF.%26atitle%3DSpider-venom%2520peptides%2520that%2520target%2520voltage-gated%2520sodium%2520channels%253A%2520Pharmacological%2520tools%2520and%2520potential%2520therapeutic%2520leads%26jtitle%3DToxicon%26date%3D2012%26volume%3D60%26spage%3D478%26epage%3D491%26doi%3D10.1016%2Fj.toxicon.2012.04.337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, N.</span></span> <span> </span><span class="NLM_article-title">Structure of a eukaryotic voltage-gated sodium channel at near-atomic resolution</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">eaal4326</span>, <span class="refDoi">Â DOI: 10.1126/science.aal4326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1126%2Fscience.aal4326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28183995" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=eaal4326&author=H.+Shenauthor=Q.+Zhouauthor=X.+Panauthor=Z.+Liauthor=J.+Wuauthor=N.+Yan&title=Structure+of+a+eukaryotic+voltage-gated+sodium+channel+at+near-atomic+resolution&doi=10.1126%2Fscience.aal4326"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.aal4326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aal4326%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DPan%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DN.%26atitle%3DStructure%2520of%2520a%2520eukaryotic%2520voltage-gated%2520sodium%2520channel%2520at%2520near-atomic%2520resolution%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3Deaal4326%26doi%3D10.1126%2Fscience.aal4326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, N.</span></span> <span> </span><span class="NLM_article-title">Structure of the Na<sub>V</sub>1.4-Î²1 complex from electric eel</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">470</span>â <span class="NLM_lpage">482</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2017.06.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1016%2Fj.cell.2017.06.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28735751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=470-482&author=Z.+Yanauthor=Q.+Zhouauthor=L.+Wangauthor=J.+Wuauthor=Y.+Zhaoauthor=G.+Huangauthor=W.+Pengauthor=H.+Shenauthor=J.+Leiauthor=N.+Yan&title=Structure+of+the+NaV1.4-%CE%B21+complex+from+electric+eel&doi=10.1016%2Fj.cell.2017.06.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Nav1.4-Î²1 complex from electric eel</span></div><div class="casAuthors">Yan, Zhen; Zhou, Qiang; Wang, Lin; Wu, Jianping; Zhao, Yanyu; Huang, Gaoxingyu; Peng, Wei; Shen, Huaizong; Lei, Jianlin; Yan, Nieng</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-482.e11</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Voltage-gated Na+ (Nav) channels initiate and propagate action potentials.  Here, we present the cryo-electron microscopic structure of EeNav1.4, the Nav channel from elec. eel (Electrophorus electricus; Ee), in complex with the Î²1 subunit at 4.0 Ã resoln.  The Ig domain of Î²1 docked onto the extracellular L5I and L6IV loops of EeNav1.4 via extensive polar interactions, and the single transmembrane helix interacted with the 3rd voltage-sensing domain (VSDIII).  The VSDs exhibited "up" conformations, while the intracellular gate of the pore domain was kept open by a digitonin-like mol.  Structural comparison with closed NavPaS showed that the outward transfer of gating charges was coupled to the iris-like pore domain dilation through intricate force transmissions involving multiple channel segments.  The IFM fast inactivation motif on the III-IV linker was plugged into the corner enclosed by the outer S4-S5 and inner S6 segments in repeats III and IV, suggesting a potential allosteric blocking mechanism for fast inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzkD2D5o-No7Vg90H21EOLACvtfcHk0liMod_ZNphrJA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Sls73P&md5=02ae7b12b16bfe60df4aa019675fc769</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.06.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.06.039%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DN.%26atitle%3DStructure%2520of%2520the%2520NaV1.4-%25CE%25B21%2520complex%2520from%2520electric%2520eel%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D470%26epage%3D482%26doi%3D10.1016%2Fj.cell.2017.06.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruben, P. C.</span></span> <span> </span><span class="NLM_article-title">Interaction between voltage-gated sodium channels and the neurotoxin, tetrodotoxin</span>. <i>Channels</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">407</span>â <span class="NLM_lpage">412</span>, <span class="refDoi">Â DOI: 10.4161/chan.2.6.7429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.4161%2Fchan.2.6.7429" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2008&pages=407-412&author=C.+H.+Leeauthor=P.+C.+Ruben&title=Interaction+between+voltage-gated+sodium+channels+and+the+neurotoxin%2C+tetrodotoxin&doi=10.4161%2Fchan.2.6.7429"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.4161%2Fchan.2.6.7429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fchan.2.6.7429%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DRuben%26aufirst%3DP.%2BC.%26atitle%3DInteraction%2520between%2520voltage-gated%2520sodium%2520channels%2520and%2520the%2520neurotoxin%252C%2520tetrodotoxin%26jtitle%3DChannels%26date%3D2008%26volume%3D2%26spage%3D407%26epage%3D412%26doi%3D10.4161%2Fchan.2.6.7429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minassian, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neff, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirzadegan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fellows, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husovsky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinspach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickenden, A. D.</span></span> <span> </span><span class="NLM_article-title">Analysis of the structural and molecular basis of voltage-sensitive sodium channel inhibition by the spider toxin Huwentoxin-IV (Î¼-TRTX-Hh2a)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">22707</span>â <span class="NLM_lpage">22720</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M113.461392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1074%2Fjbc.M113.461392" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=22707-22720&author=N.+A.+Minassianauthor=A.+Gibbsauthor=A.+Y.+Shihauthor=Y.+Liuauthor=R.+A.+Neffauthor=S.+W.+Suttonauthor=T.+Mirzadeganauthor=J.+Connorauthor=R.+Fellowsauthor=M.+Husovskyauthor=S.+Nelsonauthor=M.+J.+Hunterauthor=M.+Flinspachauthor=A.+D.+Wickenden&title=Analysis+of+the+structural+and+molecular+basis+of+voltage-sensitive+sodium+channel+inhibition+by+the+spider+toxin+Huwentoxin-IV+%28%CE%BC-TRTX-Hh2a%29&doi=10.1074%2Fjbc.M113.461392"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.461392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.461392%26sid%3Dliteratum%253Aachs%26aulast%3DMinassian%26aufirst%3DN.%2BA.%26aulast%3DGibbs%26aufirst%3DA.%26aulast%3DShih%26aufirst%3DA.%2BY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNeff%26aufirst%3DR.%2BA.%26aulast%3DSutton%26aufirst%3DS.%2BW.%26aulast%3DMirzadegan%26aufirst%3DT.%26aulast%3DConnor%26aufirst%3DJ.%26aulast%3DFellows%26aufirst%3DR.%26aulast%3DHusovsky%26aufirst%3DM.%26aulast%3DNelson%26aufirst%3DS.%26aulast%3DHunter%26aufirst%3DM.%2BJ.%26aulast%3DFlinspach%26aufirst%3DM.%26aulast%3DWickenden%26aufirst%3DA.%2BD.%26atitle%3DAnalysis%2520of%2520the%2520structural%2520and%2520molecular%2520basis%2520of%2520voltage-sensitive%2520sodium%2520channel%2520inhibition%2520by%2520the%2520spider%2520toxin%2520Huwentoxin-IV%2520%2528%25CE%25BC-TRTX-Hh2a%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D22707%26epage%3D22720%26doi%3D10.1074%2Fjbc.M113.461392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ojeda, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Racemic and quasi-racemic X-ray structures of cyclic disulfide-rich peptide drug scaffolds</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">11236</span>â <span class="NLM_lpage">11241</span>, <span class="refDoi">Â DOI: 10.1002/anie.201406563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1002%2Fanie.201406563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGgtbnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=11236-11241&author=C.+K.+Wangauthor=G.+J.+Kingauthor=S.+E.+Northfieldauthor=P.+G.+Ojedaauthor=D.+J.+Craik&title=Racemic+and+quasi-racemic+X-ray+structures+of+cyclic+disulfide-rich+peptide+drug+scaffolds&doi=10.1002%2Fanie.201406563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Racemic and quasi-racemic X-ray structures of cyclic disulfide-rich peptide drug scaffolds</span></div><div class="casAuthors">Wang, Conan K.; King, Gordon J.; Northfield, Susan E.; Ojeda, Paola G.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11236-11241</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic disulfide-rich peptides have exceptional stability and are promising frameworks for drug design.  We were interested in obtaining X-ray structures of these peptides to assist in drug design applications, but disulfide-rich peptides can be notoriously difficult to crystallize.  To overcome this limitation, we chem. synthesized the L- and D-forms of three prototypic cyclic disulfide-rich peptides: SFTI-1 (14-mer with one disulfide bond), cVc1.1 (22-mer with two disulfide bonds), and kB1 (29-mer with three disulfide bonds) for racemic crystn. studies.  Facile crystal formation occurred from a racemic mixt. of each peptide, giving structures solved at resolns. from 1.25 Ã to 1.9 Ã.  Addnl., we obtained the quasi-racemic structures of two mutants of kB1, [G6A]kB1, and [V25A]kB1, which were solved at a resoln. of 1.25 Ã and 2.3 Ã, resp.  The racemic crystallog. approach appears to have broad utility in the structural biol. of cyclic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqliAlRrLTkj7Vg90H21EOLACvtfcHk0li66KbgCMK2Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGgtbnO&md5=37a4997cdd96ead9514473b655c71148</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201406563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201406563%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DKing%26aufirst%3DG.%2BJ.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DOjeda%26aufirst%3DP.%2BG.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DRacemic%2520and%2520quasi-racemic%2520X-ray%2520structures%2520of%2520cyclic%2520disulfide-rich%2520peptide%2520drug%2520scaffolds%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D11236%26epage%3D11241%26doi%3D10.1002%2Fanie.201406563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biswas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nixey, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falsey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herberich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holder, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunasekaran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span> <span> </span><span class="NLM_article-title">Engineering antibody reactivity for efficient derivatization to generate Na<sub>V</sub>1.7 inhibitory GpTx-1 peptide-antibody conjugates</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2427</span>â <span class="NLM_lpage">2435</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.7b00542</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00542" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2427-2435&author=K.+Biswasauthor=T.+E.+Nixeyauthor=J.+K.+Murrayauthor=J.+R.+Falseyauthor=L.+Yinauthor=H.+Liuauthor=J.+Gingrasauthor=B.+E.+Hallauthor=B.+Herberichauthor=J.+R.+Holderauthor=H.+Liauthor=J.+Liguttiauthor=M.+J.+Linauthor=D.+Liuauthor=B.+D.+Sorianoauthor=M.+Sotoauthor=L.+Tranauthor=C.+M.+Tegleyauthor=A.+Zouauthor=K.+Gunasekaranauthor=B.+D.+Moyerauthor=L.+Dohertyauthor=L.+P.+Miranda&title=Engineering+antibody+reactivity+for+efficient+derivatization+to+generate+NaV1.7+inhibitory+GpTx-1+peptide-antibody+conjugates&doi=10.1021%2Facschembio.7b00542"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00542%26sid%3Dliteratum%253Aachs%26aulast%3DBiswas%26aufirst%3DK.%26aulast%3DNixey%26aufirst%3DT.%2BE.%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DFalsey%26aufirst%3DJ.%2BR.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGingras%26aufirst%3DJ.%26aulast%3DHall%26aufirst%3DB.%2BE.%26aulast%3DHerberich%26aufirst%3DB.%26aulast%3DHolder%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DSoriano%26aufirst%3DB.%2BD.%26aulast%3DSoto%26aufirst%3DM.%26aulast%3DTran%26aufirst%3DL.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DZou%26aufirst%3DA.%26aulast%3DGunasekaran%26aufirst%3DK.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DDoherty%26aufirst%3DL.%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26atitle%3DEngineering%2520antibody%2520reactivity%2520for%2520efficient%2520derivatization%2520to%2520generate%2520NaV1.7%2520inhibitory%2520GpTx-1%2520peptide-antibody%2520conjugates%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2427%26epage%3D2435%26doi%3D10.1021%2Facschembio.7b00542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kornecook, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Be, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilch, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarosh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehto, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortuno, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taborn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, D. X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremeau, R. T.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span> <span> </span><span class="NLM_article-title">Pharmacologic characterization of AMG8379, a potent and selective small molecule sulfonamide antagonist of the voltage-gated sodium channel Na<sub>V</sub>1.7</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>362</i></span>,  <span class="NLM_fpage">146</span>â <span class="NLM_lpage">160</span>, <span class="refDoi">Â DOI: 10.1124/jpet.116.239590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1124%2Fjpet.116.239590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=28473457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOqsb3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=2017&pages=146-160&author=T.+J.+Kornecookauthor=R.+Yinauthor=S.+Altmannauthor=X.+Beauthor=V.+Berryauthor=C.+P.+Ilchauthor=M.+Jaroshauthor=D.+Johnsonauthor=J.+H.+Leeauthor=S.+G.+Lehtoauthor=J.+Liguttiauthor=D.+Liuauthor=J.+Lutherauthor=D.+Matsonauthor=D.+Ortunoauthor=J.+Robertsauthor=K.+Tabornauthor=J.+Wangauthor=M.+M.+Weissauthor=V.+Yuauthor=D.+X.+D.+Zhuauthor=R.+T.+Fremeauauthor=B.+D.+Moyer&title=Pharmacologic+characterization+of+AMG8379%2C+a+potent+and+selective+small+molecule+sulfonamide+antagonist+of+the+voltage-gated+sodium+channel+NaV1.7&doi=10.1124%2Fjpet.116.239590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacologic characterization of AMG8379, a potent and selective small molecule sulfonamide antagonist of the voltage-gated sodium channel NaV1.7</span></div><div class="casAuthors">Kornecook, Thomas J.; Yin, Ruoyuan; Altmann, Stephen; Be, Xuhai; Berry, Virginia; Ilch, Christopher P.; Jarosh, Michael; Johnson, Danielle; Lee, Josie H.; Lehto, Sonya G.; Ligutti, Joseph; Liu, Dong; Luther, Jason; Matson, David; Ortuno, Danny; Roberts, John; Taborn, Kristin; Wang, Jinti; Weiss, Matthew M.; Yu, Violeta; Zhu, Dawn X. D.; Fremeau, Robert T.; Moyer, Bryan D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">362</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">146-160</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Potent and selective antagonists of the voltage-gated sodium channel NaV1.7 represent a promising avenue for the development of new chronic pain therapies.  We generated a small mol. atropisomer quinolone sulfonamide antagonist AMG8379 and a less active enantiomer AMG8380.  Here we show that AMG8379 potently blocks human NaV1.7 channels with an IC50 of 8.5 nM and endogenous tetrodotoxin (TTX)-sensitive sodium channels in dorsal root ganglion (DRG) neurons with an IC50 of 3.1 nM in whole-cell patch clamp electrophysiol. assays using a voltage protocol that interrogates channels in a partially inactivated state.  AMG8379 was 100- to 1000-fold selective over other NaV family members, including NaV1.4 expressed in muscle and NaV1.5 expressed in the heart, as well as TTX-resistant NaV channels in DRG neurons.  Using an ex vivo mouse skin-nerve prepn., AMG8379 blocked mech. induced action potential firing in C-fibers in both a time-dependent and dose-dependent manner.  AMG8379 similarly reduced the frequency of thermally induced C-fiber spiking, whereas AMG8380 affected neither mech. nor thermal responses.  In vivo target engagement of AMG8379 in mice was evaluated in multiple NaV1.7-dependent behavioral endpoints.  AMG8379 dose-dependently inhibited intradermal histamine-induced scratching and intraplantar capsaicin-induced licking, and reversed UVB radiation skin burn-induced thermal hyperalgesia; notably, behavioral effects were not obsd. with AMG8380 at similar plasma exposure levels.  AMG8379 is a potent and selective NaV1.7 inhibitor that blocks sodium current in heterologous cells as well as DRG neurons, inhibits action potential firing in peripheral nerve fibers, and exhibits pharmacodynamic effects in translatable models of both itch and pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZr8oKonIYLVg90H21EOLACvtfcHk0lh8NIIdsYNdiQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOqsb3O&md5=97f6b3a928ac98aa59cf68859d008f19</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1124%2Fjpet.116.239590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.116.239590%26sid%3Dliteratum%253Aachs%26aulast%3DKornecook%26aufirst%3DT.%2BJ.%26aulast%3DYin%26aufirst%3DR.%26aulast%3DAltmann%26aufirst%3DS.%26aulast%3DBe%26aufirst%3DX.%26aulast%3DBerry%26aufirst%3DV.%26aulast%3DIlch%26aufirst%3DC.%2BP.%26aulast%3DJarosh%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DLehto%26aufirst%3DS.%2BG.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DLuther%26aufirst%3DJ.%26aulast%3DMatson%26aufirst%3DD.%26aulast%3DOrtuno%26aufirst%3DD.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DTaborn%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DV.%26aulast%3DZhu%26aufirst%3DD.%2BX.%2BD.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26atitle%3DPharmacologic%2520characterization%2520of%2520AMG8379%252C%2520a%2520potent%2520and%2520selective%2520small%2520molecule%2520sulfonamide%2520antagonist%2520of%2520the%2520voltage-gated%2520sodium%2520channel%2520NaV1.7%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2017%26volume%3D362%26spage%3D146%26epage%3D160%26doi%3D10.1124%2Fjpet.116.239590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marx, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boezio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu-Moyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMauro, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feric
Bojic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foti, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graceffa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman-Perez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarosh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kornecook, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreiman, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligutti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taborn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youngblood, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremeau, R. T.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Sulfonamides as selective Na<sub>V</sub>1.7 inhibitors: optimizing potency and pharmacokinetics while mitigating metabolic liabilities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5969</span>â <span class="NLM_lpage">5989</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01851</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01851" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5969-5989&author=M.+M.+Weissauthor=T.+A.+Dineenauthor=I.+E.+Marxauthor=S.+Altmannauthor=A.+Boezioauthor=H.+Bregmanauthor=M.+Chu-Moyerauthor=E.+F.+DiMauroauthor=E.+Feric%0ABojicauthor=R.+S.+Fotiauthor=H.+Gaoauthor=R.+Graceffaauthor=H.+Gunaydinauthor=A.+Guzman-Perezauthor=H.+Huangauthor=L.+Huangauthor=M.+Jaroshauthor=T.+Kornecookauthor=C.+R.+Kreimanauthor=J.+Liguttiauthor=D.+S.+Laauthor=M.+J.+Linauthor=D.+Liuauthor=B.+D.+Moyerauthor=H.+N.+Nguyenauthor=E.+A.+Petersonauthor=P.+E.+Roseauthor=K.+Tabornauthor=B.+D.+Youngbloodauthor=V.+Yuauthor=R.+T.+Fremeau&title=Sulfonamides+as+selective+NaV1.7+inhibitors%3A+optimizing+potency+and+pharmacokinetics+while+mitigating+metabolic+liabilities&doi=10.1021%2Facs.jmedchem.6b01851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfonamides as Selective NaV1.7 Inhibitors: Optimizing Potency and Pharmacokinetics While Mitigating Metabolic Liabilities</span></div><div class="casAuthors">Weiss, Matthew M.; Dineen, Thomas A.; Marx, Isaac E.; Altmann, Steven; Boezio, Alessandro; Bregman, Howard; Chu-Moyer, Margaret; DiMauro, Erin F.; Feric Bojic, Elma; Foti, Robert S.; Gao, Hua; Graceffa, Russell; Gunaydin, Hakan; Guzman-Perez, Angel; Huang, Hongbing; Huang, Liyue; Jarosh, Michael; Kornecook, Thomas; Kreiman, Charles R.; Ligutti, Joseph; La, Daniel S.; Lin, Min-Hwa Jasmine; Liu, Dong; Moyer, Bryan D.; Nguyen, Hanh N.; Peterson, Emily A.; Rose, Paul E.; Taborn, Kristin; Youngblood, Beth D.; Yu, Violeta; Fremeau, Robert T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5969-5989</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several reports have recently emerged regarding the identification of heteroarylsulfonamides as NaV1.7 inhibitors that demonstrate high levels of selectivity over other NaV isoforms.  The optimization of a series of internal NaV1.7 leads that address a no. of metabolic liabilities including bioactivation, PXR activation, as well as CYP3A4 induction and inhibition led to the identification of potent and selective inhibitors that demonstrated favorable pharmacokinetic profiles and were devoid of the aforementioned liabilities.  The key to achieving this within a series prone to transporter-mediated clearance was the identification of a small range of optimal cLogD values and the discovery of subtle PXR SAR that was not lipophilicity dependent.  This enabled the identification of compd. 20, which was advanced into a target engagement pharmacodynamic model where it exhibited robust reversal of histamine-induced scratching bouts in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovsPC2UZpxsbVg90H21EOLACvtfcHk0lh8NIIdsYNdiQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmtbw%253D&md5=6333a6b936e48f9d18cf9df10ea12853</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01851%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DM.%2BM.%26aulast%3DDineen%26aufirst%3DT.%2BA.%26aulast%3DMarx%26aufirst%3DI.%2BE.%26aulast%3DAltmann%26aufirst%3DS.%26aulast%3DBoezio%26aufirst%3DA.%26aulast%3DBregman%26aufirst%3DH.%26aulast%3DChu-Moyer%26aufirst%3DM.%26aulast%3DDiMauro%26aufirst%3DE.%2BF.%26aulast%3DFeric%2BBojic%26aufirst%3DE.%26aulast%3DFoti%26aufirst%3DR.%2BS.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DGraceffa%26aufirst%3DR.%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DGuzman-Perez%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DJarosh%26aufirst%3DM.%26aulast%3DKornecook%26aufirst%3DT.%26aulast%3DKreiman%26aufirst%3DC.%2BR.%26aulast%3DLigutti%26aufirst%3DJ.%26aulast%3DLa%26aufirst%3DD.%2BS.%26aulast%3DLin%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DNguyen%26aufirst%3DH.%2BN.%26aulast%3DPeterson%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DP.%2BE.%26aulast%3DTaborn%26aufirst%3DK.%26aulast%3DYoungblood%26aufirst%3DB.%2BD.%26aulast%3DYu%26aufirst%3DV.%26aulast%3DFremeau%26aufirst%3DR.%2BT.%26atitle%3DSulfonamides%2520as%2520selective%2520NaV1.7%2520inhibitors%253A%2520optimizing%2520potency%2520and%2520pharmacokinetics%2520while%2520mitigating%2520metabolic%2520liabilities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5969%26epage%3D5989%26doi%3D10.1021%2Facs.jmedchem.6b01851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasseur, R.</span></span> <span> </span><span class="NLM_article-title">Analysis of accessible surface of residues in proteins</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1406</span>â <span class="NLM_lpage">1417</span>, <span class="refDoi">Â DOI: 10.1110/ps.0304803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1110%2Fps.0304803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=12824487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD3sXltFekur0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=1406-1417&author=L.+Linsauthor=A.+Thomasauthor=R.+Brasseur&title=Analysis+of+accessible+surface+of+residues+in+proteins&doi=10.1110%2Fps.0304803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of accessible surface of residues in proteins</span></div><div class="casAuthors">Lins, Laurence; Thomas, Annick; Brasseur, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1406-1417</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We analyzed the total, hydrophobic, and hydrophilic accessible surfaces (ASAs) of residues from a nonredundant bank of 587 3D structure proteins.  In an extended fold, residues are classified into three families with respect to their hydrophobicity balance.  As expected, residues lose part of their solvent-accessible surface with folding but the three groups remain.  The decrease of accessibility is more pronounced for hydrophobic than hydrophilic residues.  Amazingly, Lysine is the residue with the largest hydrophobic accessible surface in folded structures.  Our anal. points out a clear difference between the mean (other studies) and median (this study) ASA values of hydrophobic residues, which should be taken into consideration for future investigations on a protein-accessible surface, in order to improve predictions requiring ASA values.  The different secondary structures correspond to different accessibility of residues.  Random coils, turns, and Î²-structures (outside Î²-sheets) are the most accessible folds, with an av. of 30% accessibility.  The helical residues are about 20% accessible, and the difference between the hydrophobic and the hydrophilic residues illustrates the amphipathy of many helixes.  Residues from Î²-sheets are the most inaccessible to solvent (10% accessible).  Hence, Î²-sheets are the most appropriate structures to shield the hydrophobic parts of residues from water.  We also show that there is an equal balance between the hydrophobic and the hydrophilic accessible surfaces of the 3D protein surfaces irresp. of the protein size.  This results in a patchwork surface of hydrophobic and hydrophilic arcas, which could be important for protein interactions and/or activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouA-fCE-awerVg90H21EOLACvtfcHk0lhsum7cwGX5Kg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXltFekur0%253D&md5=1bc25f11ef043dbf08f1930c0c245421</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1110%2Fps.0304803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.0304803%26sid%3Dliteratum%253Aachs%26aulast%3DLins%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DA.%26aulast%3DBrasseur%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520accessible%2520surface%2520of%2520residues%2520in%2520proteins%26jtitle%3DProtein%2520Sci.%26date%3D2003%26volume%3D12%26spage%3D1406%26epage%3D1417%26doi%3D10.1110%2Fps.0304803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, Z. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Martinez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cryar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilford, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takle, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thalassinos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabor, A. B.</span></span> <span> </span><span class="NLM_article-title">The role of disulfide bond replacements in analogues of the tarantula toxin ProTx-II and their effects on inhibition of the voltage-gated sodium ion channel Na<sub>V</sub>1.7</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">13063</span>â <span class="NLM_lpage">13075</span>, <span class="refDoi">Â DOI: 10.1021/jacs.7b06506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b06506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCqs7rN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=13063-13075&author=Z.+V.+F.+Wrightauthor=S.+McCarthyauthor=R.+Dickmanauthor=F.+E.+Reyesauthor=S.+Sanchez-Martinezauthor=A.+Cryarauthor=I.+Kilfordauthor=A.+Hallauthor=A.+K.+Takleauthor=M.+Topfauthor=T.+Gonenauthor=K.+Thalassinosauthor=A.+B.+Tabor&title=The+role+of+disulfide+bond+replacements+in+analogues+of+the+tarantula+toxin+ProTx-II+and+their+effects+on+inhibition+of+the+voltage-gated+sodium+ion+channel+NaV1.7&doi=10.1021%2Fjacs.7b06506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Role of Disulfide Bond Replacements in Analogues of the Tarantula Toxin ProTx-II and Their Effects on Inhibition of the Voltage-Gated Sodium Ion Channel Nav1.7</span></div><div class="casAuthors">Wright, Zoe V. F.; McCarthy, Stephen; Dickman, Rachael; Reyes, Francis E.; Sanchez-Martinez, Silvia; Cryar, Adam; Kilford, Ian; Hall, Adrian; Takle, Andrew K.; Topf, Maya; Gonen, Tamir; Thalassinos, Konstantinos; Tabor, Alethea B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">13063-13075</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Spider venom toxins, such as Protoxin-II (ProTx-II), have recently received much attention as selective Nav1.7 channel blockers, with potential to be developed as leads for the treatment of chronic nociceptive pain.  ProTx-II is a 30-amino acid peptide with three disulfide bonds that has been reported to adopt a well-defined inhibitory cystine knot (ICK) scaffold structure.  Potential drawbacks with such peptides include poor pharmacodynamics and potential scrambling of the disulfide bonds in vivo.  In order to address these issues, in the present study the authors report the solid-phase synthesis of lanthionine-bridged analogs of ProTx-II, in which one of the three disulfide bridges is replaced with a thioether linkage, and have evaluated the biol. properties of these analogs.  The authors have also investigated the folding and disulfide bridging patterns arising from different methods of oxidn. of the linear peptide precursor.  Finally, the authors report the x-ray crystal structure of ProTx-II to at. resoln.; to the knowledge this is the first crystal structure of an inhibitory cystine knot (ICK) spider venom peptide not bound to a substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn9ZD9HHB7jrVg90H21EOLACvtfcHk0liu5RawzJV0Wg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCqs7rN&md5=26210eb421e73f0bd439147862a27b6e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b06506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b06506%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DZ.%2BV.%2BF.%26aulast%3DMcCarthy%26aufirst%3DS.%26aulast%3DDickman%26aufirst%3DR.%26aulast%3DReyes%26aufirst%3DF.%2BE.%26aulast%3DSanchez-Martinez%26aufirst%3DS.%26aulast%3DCryar%26aufirst%3DA.%26aulast%3DKilford%26aufirst%3DI.%26aulast%3DHall%26aufirst%3DA.%26aulast%3DTakle%26aufirst%3DA.%2BK.%26aulast%3DTopf%26aufirst%3DM.%26aulast%3DGonen%26aufirst%3DT.%26aulast%3DThalassinos%26aufirst%3DK.%26aulast%3DTabor%26aufirst%3DA.%2BB.%26atitle%3DThe%2520role%2520of%2520disulfide%2520bond%2520replacements%2520in%2520analogues%2520of%2520the%2520tarantula%2520toxin%2520ProTx-II%2520and%2520their%2520effects%2520on%2520inhibition%2520of%2520the%2520voltage-gated%2520sodium%2520ion%2520channel%2520NaV1.7%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D13063%26epage%3D13075%26doi%3D10.1021%2Fjacs.7b06506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nye, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couture, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feric, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rottman, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beiler, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmberg, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonough, S. I.</span></span> <span> </span><span class="NLM_article-title">Global Na<sub>V</sub>1.7 knockout mice recapitulate the phenotype of human congenital indifference to pain</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e105895</span>, <span class="refDoi">Â DOI: 10.1371/journal.pone.0105895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1371%2Fjournal.pone.0105895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=25188265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslaqt73O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e105895&author=J.+Gingrasauthor=S.+Smithauthor=D.+J.+Matsonauthor=D.+Johnsonauthor=K.+Nyeauthor=L.+Coutureauthor=E.+Fericauthor=R.+Yinauthor=B.+D.+Moyerauthor=M.+L.+Petersonauthor=J.+B.+Rottmanauthor=R.+J.+Beilerauthor=A.+B.+Malmbergauthor=S.+I.+McDonough&title=Global+NaV1.7+knockout+mice+recapitulate+the+phenotype+of+human+congenital+indifference+to+pain&doi=10.1371%2Fjournal.pone.0105895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Global Nav1.7 knockout mice recapitulate the phenotype of human congenital indifference to pain</span></div><div class="casAuthors">Gingras, Jacinthe; Smith, Sarah; Matson, David J.; Johnson, Danielle; Nye, Kim; Couture, Lauren; Feric, Elma; Yin, Ruoyuan; Moyer, Bryan D.; Peterson, Matthew L.; Rottman, James B.; Beiler, Rudolph J.; Malmberg, Annika B.; McDonough, Stefan I.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e105895/1-e105895/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Clin. genetic studies have shown that loss of Nav1.7 function leads to the complete loss of acute pain perception.  The global deletion is reported lethal in mice, however, and studies of mice with promoter-specific deletions of Nav1.7 have suggested that the role of Nav1.7 in pain transduction depends on the precise form of pain.  We developed genetic and animal husbandry strategies that overcame the neonatal-lethal phenotype and enabled construction of a global Nav1.7 knockout mouse.  Knockouts were anatomically normal, reached adulthood, and had phenotype wholly analogous to human congenital indifference to pain (CIP): compared to littermates, knockouts showed no defects in mech. sensitivity or overall movement yet were completely insensitive to painful tactile, thermal, and chem. stimuli and were anosmic.  Knockouts also showed no painful behaviors resulting from peripheral injection of nonselective sodium channel activators, did not develop complete Freund's adjuvant-induced thermal hyperalgesia, and were insensitive to intra-dermal histamine injection.  Tetrodotoxin-sensitive sodium current recorded from cell bodies of isolated sensory neurons and the mech.-evoked spiking of C-fibers in a skin-nerve prepn. each were reduced but not eliminated in tissue from knockouts compared to littermates.  Results support a role for Nav1.7 that is conserved between rodents and humans and suggest several possibly translatable biomarkers for the study of Nav1.7-targeted therapeutics.  Results further suggest that Nav1.7 may retain its key role in persistent as well as acute forms of pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAzxB508dD1LVg90H21EOLACvtfcHk0liu5RawzJV0Wg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslaqt73O&md5=6a1acb0c6e4e38ac89b1ab38d67b538e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0105895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0105895%26sid%3Dliteratum%253Aachs%26aulast%3DGingras%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DMatson%26aufirst%3DD.%2BJ.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DNye%26aufirst%3DK.%26aulast%3DCouture%26aufirst%3DL.%26aulast%3DFeric%26aufirst%3DE.%26aulast%3DYin%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DB.%2BD.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26aulast%3DRottman%26aufirst%3DJ.%2BB.%26aulast%3DBeiler%26aufirst%3DR.%2BJ.%26aulast%3DMalmberg%26aufirst%3DA.%2BB.%26aulast%3DMcDonough%26aufirst%3DS.%2BI.%26atitle%3DGlobal%2520NaV1.7%2520knockout%2520mice%2520recapitulate%2520the%2520phenotype%2520of%2520human%2520congenital%2520indifference%2520to%2520pain%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De105895%26doi%3D10.1371%2Fjournal.pone.0105895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">ElbrÃ¸nd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakobsen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">AgersÃ¸, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatorp, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zdravkovic, M.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetics, pharmacodynamics, safety, and tolerability of a single-dose of NN2211, a long-acting glucagon-like peptide 1 derivative, in healthy male subjects</span>. <i>Diabetes Care</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1398</span>â <span class="NLM_lpage">1404</span>, <span class="refDoi">Â DOI: 10.2337/diacare.25.8.1398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.2337%2Fdiacare.25.8.1398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=12145241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsFaitr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2002&pages=1398-1404&author=B.+Elbr%C3%B8ndauthor=G.+Jakobsenauthor=S.+Larsenauthor=H.+Agers%C3%B8author=L.+B.+Jensenauthor=P.+Rolanauthor=J.+Sturisauthor=V.+Hatorpauthor=M.+Zdravkovic&title=Pharmacokinetics%2C+pharmacodynamics%2C+safety%2C+and+tolerability+of+a+single-dose+of+NN2211%2C+a+long-acting+glucagon-like+peptide+1+derivative%2C+in+healthy+male+subjects&doi=10.2337%2Fdiacare.25.8.1398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetics, pharmacodynamics, safety, and tolerability of a single-dose of NN2211, a long-acting glucagon-like peptide 1 derivative, in healthy male subjects</span></div><div class="casAuthors">Elbrond, Bodil; Jakobsen, Grethe; Larsen, Soren; Agerso, Henrik; Jensen, Lisbeth Bjerring; Rolan, Paul; Sturis, Jeppe; Hatorp, Vibeke; Zdravkovic, Milan</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes Care</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1398-1404</span>CODEN:
                <span class="NLM_cas:coden">DICAD2</span>;
        ISSN:<span class="NLM_cas:issn">0149-5992</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association, Inc.</span>)
        </div><div class="casAbstract">OBJECTIVE - The primary objective of the present study was to investigate the safety, tolerability, and pharmacokinetics of a single dose of NN 2211, a long-acting glucagon-like peptide 1 (GLP-1) deriv., in healthy male subjects.  The secondary objective was to investigate the pharmacodynamics of NN 2211.  RESEARCH DESIGN AND METHODS - In a double-blind, randomized dose, escalation, placebo-controlled study, healthy male subjects were enrolled at eight consecutive dose levels (1.25, 2.5, 5.0, 10.0, 12.5, 15.0, 17.5, and 20.0 Î¼g/kg) with eight subjects per dose level at a 3:1 active:placebo randomization.  After s.c. dosing with NN 2211, 48-h pharmacokinetic, and 24-h glucose, insulin and glucagon profiles were assessed.  In addn., three subjects at each dose level were randomly assigned (one placebo/two active) to an i.v. glucose tolerance test (IVGTT) 9 h after the dose (corresponding to the time to maximal plasma concn. of NN 2211).  RESULTS - After s.c. administration, the half-life of NN 2211 was found to be 11-15 h.  Overall, although there were no statistically significant differences compared with placebo in the area under the curve (0-9 h for insulin or glucagon), there was a borderline-significant lowering of glucose levels.  During the IVGTT, there was a statistically significant increase in insulin secretion, but there was no significant effect on glucagon levels.  Although no significant effect was obsd. on glucose levels during the IVGTT, there was a dose-dependent increase in the glucose disappearance const.  Whereas no serious adverse events were obsd., there was a higher incidence of adverse events after active treatment compared with placebo treatment (notably headache, dizziness, nausea, and vomiting).  CONCLUSIONS - This study provides evidence that NN 2211 has a pharmacokinetic profile consistent with once-daily dosing in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCuouDtWjhLLVg90H21EOLACvtfcHk0liu5RawzJV0Wg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsFaitr0%253D&md5=e91822e17b5ac8a3da0aa152aba558eb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.2337%2Fdiacare.25.8.1398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdiacare.25.8.1398%26sid%3Dliteratum%253Aachs%26aulast%3DElbr%25C3%25B8nd%26aufirst%3DB.%26aulast%3DJakobsen%26aufirst%3DG.%26aulast%3DLarsen%26aufirst%3DS.%26aulast%3DAgers%25C3%25B8%26aufirst%3DH.%26aulast%3DJensen%26aufirst%3DL.%2BB.%26aulast%3DRolan%26aufirst%3DP.%26aulast%3DSturis%26aufirst%3DJ.%26aulast%3DHatorp%26aufirst%3DV.%26aulast%3DZdravkovic%26aufirst%3DM.%26atitle%3DPharmacokinetics%252C%2520pharmacodynamics%252C%2520safety%252C%2520and%2520tolerability%2520of%2520a%2520single-dose%2520of%2520NN2211%252C%2520a%2520long-acting%2520glucagon-like%2520peptide%25201%2520derivative%252C%2520in%2520healthy%2520male%2520subjects%26jtitle%3DDiabetes%2520Care%26date%3D2002%26volume%3D25%26spage%3D1398%26epage%3D1404%26doi%3D10.2337%2Fdiacare.25.8.1398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ingwersen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khurana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madabushi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonker, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thi, T. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">TornÃ¸e, C. W.</span></span> <span> </span><span class="NLM_article-title">Dosing rationale for Liraglutide in type 2 diabetes mellitus: a pharmacometric assessment</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1815</span>â <span class="NLM_lpage">1823</span>, <span class="refDoi">Â DOI: 10.1177/0091270011430504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1177%2F0091270011430504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=22174428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ltrzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1815-1823&author=S.+H.+Ingwersenauthor=M.+Khuranaauthor=R.+Madabushiauthor=E.+Watsonauthor=D.+M.+Jonkerauthor=T.+D.+L.+Thiauthor=L.+V.+Jacobsenauthor=C.+W.+Torn%C3%B8e&title=Dosing+rationale+for+Liraglutide+in+type+2+diabetes+mellitus%3A+a+pharmacometric+assessment&doi=10.1177%2F0091270011430504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dosing rationale for liraglutide in type 2 diabetes mellitus: a pharmacometric assessment</span></div><div class="casAuthors">Ingwersen, Steen H.; Khurana, Manoj; Madabushi, Rajanikanth; Watson, Estelle; Jonker, Daniel M.; Le Thi, Tu Duyen; Jacobsen, Lisbeth V.; Tornoee, Christoffer W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1815-1823</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The glucagon-like peptide-1 (GLP-1) receptor agonist liraglutide was approved in 2010 by the US Food and Drug Administration (FDA) as an adjunct treatment to diet and exercise to improve glycemic control in adults with type 2 diabetes mellitus.  This article provides insights into the use of pharmacometric analyses for regulatory review with a focus on the dosing recommendations.  The assessment was based on the totality of exploratory and confirmatory anal. of dose-finding and pivotal clin. data and was structured around a set of key questions in accordance with current FDA review practice.  For the pharmacometric review of liraglutide, the key questions focused on exposure-response relationships for effects on fasting plasma glucose, Hb A1c, and calcitonin and on variability in exposure across demog. subgroups of patients.  The importance of conducting exploratory exposure-response anal. and population pharmacokinetic studies in clin. drug development to support dosing recommendations is highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLL90Sr-WPabVg90H21EOLACvtfcHk0lhlg4cZYR5QQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ltrzE&md5=3bc6e426ae9801f33c9aceb78c5fc247</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1177%2F0091270011430504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0091270011430504%26sid%3Dliteratum%253Aachs%26aulast%3DIngwersen%26aufirst%3DS.%2BH.%26aulast%3DKhurana%26aufirst%3DM.%26aulast%3DMadabushi%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DE.%26aulast%3DJonker%26aufirst%3DD.%2BM.%26aulast%3DThi%26aufirst%3DT.%2BD.%2BL.%26aulast%3DJacobsen%26aufirst%3DL.%2BV.%26aulast%3DTorn%25C3%25B8e%26aufirst%3DC.%2BW.%26atitle%3DDosing%2520rationale%2520for%2520Liraglutide%2520in%2520type%25202%2520diabetes%2520mellitus%253A%2520a%2520pharmacometric%2520assessment%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2012%26volume%3D52%26spage%3D1815%26epage%3D1823%26doi%3D10.1177%2F0091270011430504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhlg4cZYR5QQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>â <span class="NLM_lpage">1214</span>, <span class="refDoi">Â DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lhlg4cZYR5QQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">CCP4
(Collaborative Computational Project, Number 4)</span> <span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein
crystallography</span>. <i>Acta Crystallogr., Sect. D:
Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>â <span class="NLM_lpage">763</span>, <span class="refDoi">Â DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=CCP4%0A%28Collaborative+Computational+Project%2C+Number+4%29&title=The+CCP4+suite%3A+programs+for+protein%0Acrystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%250Acrystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%250ABiol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia-xing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfson, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">A modified ACORN to solve protein structures at resolutions of 1.7 Ã or better</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1465</span>â <span class="NLM_lpage">1475</span>, <span class="refDoi">Â DOI: 10.1107/S090744490502576X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS090744490502576X" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=1465-1475&author=Y.+Jia-xingauthor=M.+M.+Woolfsonauthor=K.+S.+Wilsonauthor=E.+J.+Dodson&title=A+modified+ACORN+to+solve+protein+structures+at+resolutions+of+1.7+%C3%85+or+better&doi=10.1107%2FS090744490502576X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS090744490502576X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744490502576X%26sid%3Dliteratum%253Aachs%26aulast%3DJia-xing%26aufirst%3DY.%26aulast%3DWoolfson%26aufirst%3DM.%2BM.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DA%2520modified%2520ACORN%2520to%2520solve%2520protein%2520structures%2520at%2520resolutions%2520of%25201.7%2520%25C3%2585%2520or%2520better%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D1465%26epage%3D1475%26doi%3D10.1107%2FS090744490502576X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lggh4iv2NPBwA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lggh4iv2NPBwA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CHC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CHC','PDB','6CHC'); return false;">PDB: 6CHC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CNU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CNU','PDB','6CNU'); return false;">PDB: 6CNU</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i32"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00736">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34135"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00736">10.1021/acs.jmedchem.8b00736</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Peptide characterization by ESI-MS, chemical structure of positive control Na<sub>V</sub>1.8 blocker, mouse pharmacokinetic study and mouse open field assay data with AM-6120, data collection and refinement statistics for X-ray crystal structures of peptides <b>42</b> and <b>43</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00736/suppl_file/jm8b00736_si_005.pdf">jm8b00736_si_005.pdf (991.49 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates of the crystal structures of the mixture of peptides <b>42</b> and <b>43</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CNU">6CNU</a>) have been deposited in the Protein Data Bank (<a href="http://www.pdb.org" class="extLink">www.pdb.org</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00736&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-21%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00736%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00736" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992689dea5227f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
