
<html lang="en"     class="pb-page"  data-request-id="21a6bf8c-a829-404c-aff3-898f350dd2f7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-13;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01929;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of MFH290: A Potent and Highly Selective Covalent Inhibitor for Cyclin-Dependent Kinase 12/13" /></meta><meta name="dc.Creator" content="Yao  Liu" /></meta><meta name="dc.Creator" content="Mingfeng  Hao" /></meta><meta name="dc.Creator" content="Alan L.  Leggett" /></meta><meta name="dc.Creator" content="Yang  Gao" /></meta><meta name="dc.Creator" content="Scott B.  Ficarro" /></meta><meta name="dc.Creator" content="Jianwei  Che" /></meta><meta name="dc.Creator" content="Zhixiang  He" /></meta><meta name="dc.Creator" content="Calla M.  Olson" /></meta><meta name="dc.Creator" content="Jarrod A.  Marto" /></meta><meta name="dc.Creator" content="Nicholas P.  Kwiatkowski" /></meta><meta name="dc.Creator" content="Tinghu  Zhang" /></meta><meta name="dc.Creator" content="Nathanael S.  Gray" /></meta><meta name="dc.Description" content="Genetic depletion of cyclin-dependent kinase 12 (CDK12) or selective inhibition of an analog-sensitive CDK12 reduces DNA damage repair gene expression, but selective inhibition of endogenous CDK12 ..." /></meta><meta name="Description" content="Genetic depletion of cyclin-dependent kinase 12 (CDK12) or selective inhibition of an analog-sensitive CDK12 reduces DNA damage repair gene expression, but selective inhibition of endogenous CDK12 ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 5, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01929" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01929" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01929" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01929" /></link>
        
    
    

<title>Discovery of MFH290: A Potent and Highly Selective Covalent Inhibitor for Cyclin-Dependent Kinase 12/13 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01929" /></meta><meta property="og:title" content="Discovery of MFH290: A Potent and Highly Selective Covalent Inhibitor for Cyclin-Dependent Kinase 12/13" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0020.jpeg" /></meta><meta property="og:description" content="Genetic depletion of cyclin-dependent kinase 12 (CDK12) or selective inhibition of an analog-sensitive CDK12 reduces DNA damage repair gene expression, but selective inhibition of endogenous CDK12 is difficult. Here, we report the development of MFH290, a novel cysteine (Cys)-directed covalent inhibitor of CDK12/13. MFH290 forms a covalent bond with Cys-1039 of CDK12, exhibits excellent kinome selectivity, inhibits the phosphorylation of serine-2 in the C-terminal domain (CTD) of RNA-polymerase II (Pol II), and reduces the expression of key DNA damage repair genes. Importantly, these effects were demonstrated to be CDK12-dependent as mutation of Cys-1039 rendered the kinase refractory to MFH290 and restored Pol II CTD phosphorylation and DNA damage repair gene expression. Consistent with its effect on DNA damage repair gene expression, MFH290 augments the antiproliferative effect of the PARP inhibitor olaparib." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01929"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01929">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01929&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01929&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01929&amp;href=/doi/10.1021/acs.jmedchem.9b01929" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 6708-6726</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01942" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b02150" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of MFH290: A Potent and Highly Selective Covalent Inhibitor for Cyclin-Dependent Kinase 12/13</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yao Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yao Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yao++Liu">Yao Liu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Mingfeng Hao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingfeng Hao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingfeng++Hao">Mingfeng Hao</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Alan L. Leggett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alan L. Leggett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alan+L.++Leggett">Alan L. Leggett</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yang Gao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yang Gao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yang++Gao">Yang Gao</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Scott B. Ficarro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott B. Ficarro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Blais Proteomics Center, Dana Farber Cancer Institute, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott+B.++Ficarro">Scott B. Ficarro</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jianwei Che</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianwei Che</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianwei++Che">Jianwei Che</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Zhixiang He</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhixiang He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhixiang++He">Zhixiang He</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Calla M. Olson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Calla M. Olson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Calla+M.++Olson">Calla M. Olson</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Jarrod A. Marto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jarrod A. Marto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Blais Proteomics Center, Dana Farber Cancer Institute, Harvard Medical School, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jarrod+A.++Marto">Jarrod A. Marto</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Nicholas P. Kwiatkowski</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas P. Kwiatkowski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>N.P.K.: email, <a href="/cdn-cgi/l/email-protection#c08ea9a3a8afaca1b3909f8bb7a9a1b4abafb7b3aba980a4a6a3a9eea8a1b2b6a1b2a4eea5a4b5"><span class="__cf_email__" data-cfemail="733d1a101b1c1f1200232c38041a1207181c0400181a331715101a5d1b1201051201175d161706">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas+P.++Kwiatkowski">Nicholas P. Kwiatkowski</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Tinghu Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tinghu Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>T.Z.: email, <a href="/cdn-cgi/l/email-protection#aefac7c0c9c6dbf1f4c6cfc0c9eecac8cdc780c6cfdcd8cfdcca80cbcadb"><span class="__cf_email__" data-cfemail="5a0e33343d322f0500323b343d1a3e3c393374323b282c3b283e743f3e2f">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tinghu++Zhang">Tinghu Zhang</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Nathanael S. Gray</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nathanael S. Gray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115, United States</div><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><span class="conrtib-corresp"><strong>*</strong>N.S.G.: email, <a href="/cdn-cgi/l/email-protection#7b351a0f131a151a1e17243c091a023b1f1d181255131a090d1a091f551e1f0e"><span class="__cf_email__" data-cfemail="f2bc93869a939c93979eadb580938bb29694919bdc9a938084938096dc979687">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nathanael+S.++Gray">Nathanael S. Gray</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5354-7403" title="Orcid link">http://orcid.org/0000-0001-5354-7403</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01929&amp;href=/doi/10.1021%2Facs.jmedchem.9b01929" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 6708â6726</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 5, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 December 2019</li><li><span class="item_label"><b>Published</b> online</span>5 June 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01929" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01929</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6708%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYao%2BLiu%252C%2BMingfeng%2BHao%252C%2BAlan%2BL.%2BLeggett%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D13%26contentID%3Dacs.jmedchem.9b01929%26title%3DDiscovery%2Bof%2BMFH290%253A%2BA%2BPotent%2Band%2BHighly%2BSelective%2BCovalent%2BInhibitor%2Bfor%2BCyclin-Dependent%2BKinase%2B12%252F13%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6726%26publicationDate%3DJuly%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01929"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2573</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01929" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of MFH290: A Potent and Highly Selective Covalent Inhibitor for Cyclin-Dependent Kinase 12/13&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yao&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Mingfeng&quot;,&quot;last_name&quot;:&quot;Hao&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;L. Leggett&quot;},{&quot;first_name&quot;:&quot;Yang&quot;,&quot;last_name&quot;:&quot;Gao&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;B. Ficarro&quot;},{&quot;first_name&quot;:&quot;Jianwei&quot;,&quot;last_name&quot;:&quot;Che&quot;},{&quot;first_name&quot;:&quot;Zhixiang&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Calla&quot;,&quot;last_name&quot;:&quot;M. Olson&quot;},{&quot;first_name&quot;:&quot;Jarrod&quot;,&quot;last_name&quot;:&quot;A. Marto&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;P. Kwiatkowski&quot;},{&quot;first_name&quot;:&quot;Tinghu&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Nathanael&quot;,&quot;last_name&quot;:&quot;S. Gray&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;6708-6726&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01929&quot;},&quot;abstract&quot;:&quot;Genetic depletion of cyclin-dependent kinase 12 (CDK12) or selective inhibition of an analog-sensitive CDK12 reduces DNA damage repair gene expression, but selective inhibition of endogenous CDK12 is difficult. Here, we report the development of MFH290, a novel cysteine (Cys)-directed covalent inhibitor of CDK12/13. MFH290 forms a covalent bond with Cys-1039 of CDK12, exhibits excellent kinome selectivity, inhibits the phosphorylation of serine-2 in the C-terminal domain (CTD) of RNA-polymerase II (Pol II), and reduces the expression of key DNA damage repair genes. Importantly, these effects were demonstrated to be CDK12-dependent as mutation of Cys-1039 rendered the kinase refractory to MFH290 and restored Pol II CTD phosphorylation and DNA damage repair gene expression. Consistent with its effect on DNA damage repair gene expression, MFH290 augments the antiproliferative effect of the PARP inhibitor olaparib.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01929&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01929" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01929&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01929" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01929&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01929" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01929&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01929&amp;href=/doi/10.1021/acs.jmedchem.9b01929" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01929" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01929" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01929%26sid%3Dliteratum%253Aachs%26pmid%3D32502343%26genre%3Darticle%26aulast%3DLiu%26date%3D2020%26atitle%3DDiscovery%2Bof%2BMFH290%253A%2BA%2BPotent%2Band%2BHighly%2BSelective%2BCovalent%2BInhibitor%2Bfor%2BCyclin-Dependent%2BKinase%2B12%252F13%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D13%26spage%3D6708%26epage%3D6726%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/jmcmar.2020.63.issue-13/20200709/jmcmar.2020.63.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Genetic depletion of cyclin-dependent kinase 12 (CDK12) or selective inhibition of an analog-sensitive CDK12 reduces DNA damage repair gene expression, but selective inhibition of endogenous CDK12 is difficult. Here, we report the development of MFH290, a novel cysteine (Cys)-directed covalent inhibitor of CDK12/13. MFH290 forms a covalent bond with Cys-1039 of CDK12, exhibits excellent kinome selectivity, inhibits the phosphorylation of serine-2 in the C-terminal domain (CTD) of RNA-polymerase II (Pol II), and reduces the expression of key DNA damage repair genes. Importantly, these effects were demonstrated to be CDK12-dependent as mutation of Cys-1039 rendered the kinase refractory to MFH290 and restored Pol II CTD phosphorylation and DNA damage repair gene expression. Consistent with its effect on DNA damage repair gene expression, MFH290 augments the antiproliferative effect of the PARP inhibitor olaparib.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65121" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65121" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cyclin dependent kinases, or CDKs, are an intensively investigated family of evolutionarily conserved protein kinases that orchestrate the cell cycle and gene transcription.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Human cells encode 20 distinct CDKs which possess highly similar ATP-binding pockets, thereby making the development of selective CDK inhibitors challenging.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Despite these difficulties, some CDKs can be targeted quite selectively, such as CDK4/6 by the FDA approved drug palbociclib, based on recognition of unique combinations of residues in the ATP pocket. A more recently explored strategy has been to exploit uniquely positioned cysteine residues with reactive, electrophile-modified inhibitors as exemplified by the development of the CDK7/12/13 inhibitor THZ1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) which targets an unusual cysteine 312 located on a loop that traverses the inhibitor binding pocket.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Further structure-guided optimization of THZ1 resulted in the development of the selective CDK12/13 inhibitor THZ531<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which targets a spatially distinct cysteine residue (Cys1039 for CDK12 and Cys1017 for CDK13).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of THZ1, THZ531, dinaciclib, and reported CDK12 inhibitors <b>2</b>â<b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">CDK12 and the highly homologous protein CDK13 are both activated upon binding to cyclin K to regulate transcription through phosphorylating Ser2 residues located on the C-terminal domain (CTD) of RNA-polymerase II (RNA-Pol2) which contains 52 repeats of the YSPTSPS motif.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Selective inhibition of CDK12 kinase activity preferentially downregulates a small set of genes that respond to and mediate the repair of DNA-damage including homologous recombination (HR) repair genes BRCA1 and BRCA2.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Inhibitors of CDK12 increase DNA damage and have been demonstrated to potentiate the antiproliferative effects of poly (ADP-ribose) polymerase (PARP) inhibitors.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Recently, an <i>in vivo</i> combination treatment of dinaciclib (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), a clinical-stage nonselective CDK12 inhibitor, with olaparib, a poly (ADP-ribose) polymerase (PARP) inhibitor, has demonstrated a significant tumor growth inhibition in a PDX ovarian model,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> spurring a wave of medicinal chemistry efforts to discover more potent and selective CDK12 inhibitors.</div><div class="NLM_p">Among the reported inhibitors of CDK12 and CDK13 are the acrylamide-modified covalent inhibitors: THZ1 and THZ531 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) in cancer cells. THZ531 was shown to irreversibly inhibit phosphorylation of ser2 of RNA pol II CTD, represses transcription of DNA damage response (DDR) genes, and results in potent antiproliferative effects.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Although validated as a cellular probe, THZ531 is not suitable for <i>in vivo</i> murine efficacy studies due to its poor metabolic stability resulting in high clearance in mice.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In addition to the covalent inhibitors, dinaciclib,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> a clinical-stage pan-CDK inhibitor, exhibits activity in numerous models but is not suitable as a selective pharmacological probe of CDK12. Recently, 2,6,9-trisubstituted purines (<b>2</b> and <b>4</b>) and a trisubstituted urea (<b>3</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) have been reported as reversible CDK12 inhibitors that exhibit high kinase selectivity for CDK12.<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12â14)</a> Here we report our efforts that culminated in the development of MFH290, a highly selective covalent inhibitor of CDK12/13 using a structure-guided drug design strategy by combining structural features of THZ531 with a previously reported noncovalent, pan-CDK inhibitor SNS032<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Structure and biochemical IC<sub>50</sub> of SNS032. (B) Overlay of SNS032/CDK2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D1J">5D1J</a>, beige) and CDK12 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACB">5ACB</a>, blue) reveals that the piperidine ring could be extended toward Cys1039.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47737" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47737" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of compounds <b>1a</b>, <b>1b</b>, and <b>1c</b> is depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Commercially available intermediate <b>4a</b> reacts with carboxylic acid <b>5</b>, <b>6</b>, or <b>7</b> through a HATU-mediated condensation reaction under basic conditions to provide the final compound <b>1a</b> or the intermediates <b>8</b> and <b>9</b> after deprotection of the Boc group. While <b>8</b> was then subjected to a second amide condensation reaction with 4-acrylamidobenzoic acid to afford <b>1b</b>, compound <b>9</b> was treated with acryloyl chloride under basic conditions to introduce the acrylamide warhead to generate <b>1c</b>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>1a</b>â<b>1c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIEA, DMF, 8%; (b) (i) DMAP, EDCI, triethylamine, DCM/DMF; (ii) 4N HCl, dioxane DCM, 16â83%; (c) HATU, 4-acrylamidobenzoic acid, DIEA, DMF, 12%; (d) acryloyl chloride, DIEA, THF, 0 Â°C, 51%.</p></p></figure><div class="NLM_p">Further analogs including <b>1d</b>â<b>1i</b>, <b>1k</b>, and <b>1l</b> were synthesized by reacting 2-(((2-bromothiazol-5-yl)thio)methyl)-5-(<i>tert</i>-butyl)oxazole (<b>10</b>) with Boc protected 3-aminopiperidine or its variants or with 4-nitrophenylcyclohexane-1,3-diamine to provide the intermediate <b>11</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Free amino compound, derived from intermediate <b>11</b> after deprotection or reduction, was subjected to either a one-step amide coupling reaction with acryloyl chloride, compound <b>12</b> or a three-step reaction as amide coupling (compound <b>13</b>), reduction (compound <b>14</b>), and acrylamide formation with acryl chloride, leading to the final products <b>1d</b>â<b>1i</b>, <b>1k</b>, and <b>1l</b>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>1d</b>â<b>1i</b>, <b>1k</b>, and <b>1l</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NMP, DIEA, 140 Â°C, 6â67%; (b) (i) 4 N HCl/dioxane, MeOH, (ii) 4-nitrobenzoyl chloride, Py, 27â55%; (c) (i) 4 N HCl/dioxane, MeOH, (ii) T<sub>3</sub>P, DIEA, DCM, 19â60%; (d) SnCl<sub>2</sub>, ethyl acetate/MeOH, 80 Â°C, 24â76%; (e) acryloyl chloride, DIEA, MeCN, 0 Â°C, 16â41%.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> presents the strategy to synthesize compounds <b>1j</b>, <b>1m</b>â<b>1o</b>, <b>1u</b>, and <b>1v</b>. The commercially available starting material 2,4,5-trisubstituted thiazole or 2,5-disubstituted thiadiazole was reacted with KSCN to afford thiocyanate intermediate <b>16</b>. In a one-pot reaction thiocyanate <b>16</b> was reduced to the corresponding thiol using NaBH<sub>4</sub> and then alkylated with the appropriate halide. The intermediate <b>17</b> was subjected to the Sandmeyer reaction to convert the amine group to the bromide, which was then replaced by a Boc-protected 3-aminopiperidine. Deprotection of Boc from intermediate <b>19</b> led to compound <b>20</b>, which was then processed using the same procedure as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> to generate the final products. Compound <b>1t</b> was synthesized with a similar reaction route with 2,4,5-trichloropyrimidine as the starting material shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>1j</b>, <b>1m</b>â<b>1o</b>, <b>1u</b>, and <b>1v</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KSCN, MeOH, 41â93%; (b) (i) NaBH<sub>4</sub>, ethanol, 0 Â°C to room temperature, (ii) R<sub>2</sub>CH<sub>2</sub>X, ethanol, 65 Â°C, 41â57%; (c) CuBr<sub>2</sub>, <i>t</i>-BuONO, MeCN, 0 Â°C, 37â94%; (d) 1-Boc-(<i>R</i>)-3-aminopiperidine, NMP, DIEA, 140 Â°C, 53â67%; (e) 4 N HCl/dioxane, MeOH, 51â95% ; (f) 4-acrylamidobenzoic acid, T<sub>3</sub>P, DIEA, DCM, 53% or (i) 5-nitropicolinic acid, T<sub>3</sub>P, DIEA, DCM, (ii) SnCl<sub>2</sub>, ethyl acetate/MeOH, 80 Â°C, (iii) acryloyl chloride, DIEA, MeCN, 0 Â°C, 7â17%.</p></p></figure><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>1t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KSCN, MeOH, 55%; (b) (i) NaBH<sub>4</sub>, ethanol, 0 Â°C to rt, (ii) 5-(<i>tert</i>-butyl)-2-(chloromethyl)oxazole, ethanol, 65 Â°C, 17%; (c) 1-Boc-(<i>R</i>)-3-aminopiperidine, NMP, DIEA, 140 Â°C, 92%; (d) (i) 4 N HCl/dioxane, MeOH, (ii) 4-nitrobenzoyl chloride, Py, 89%; (e) SnCl<sub>2</sub>, ethyl acetate/MeOH, 80 Â°C, 20%; (f) acryloyl chloride, DIEA, MeCN, 0 Â°C, 65%.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>1p</b>, <b>1q</b>, and <b>1r</b> that contain different linkers between the oxazole and thiazolyl ring was carried out as shown in <a class="ref internalNav" href="#sch5" aria-label="Schemes 5">Schemes 5</a> and <a class="ref internalNav" href="#sch6" aria-label="6">6</a>. First, the Wittig reaction was performed using aldehyde <b>29</b> and phosphonate intermediate <b>28</b>, prepared from 5-(<i>tert</i>-butyl)-2-(chloromethyl)oxazole using an Arbuzov reaction. Reduction of the double bond with hydrogen/Pd provided intermediate <b>31</b>, which was converted to compound <b>32</b> by a Sandmeyer reaction. Installation of piperidine and acrylamide warhead was followed by the same procedure as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</div><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>1p</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) triethyl phosphite, 120 Â°C, 39%; (b) <i>t</i>-BuOK, THF, 84%; (c) H<sub>2</sub>, Pd/C, MeOH, then 4 N HCl/dioxane, 90%; (d) CuBr<sub>2</sub>, <i>t</i>-BuONO, MeCN, 0 Â°C, 53%; (e) NMP, DIEA, 140 Â°C, 63%; (f) 4 N HCl/dioxane, MeOH, 88%; (g) 4-acrylamidobenzoic acid, HOBT, EDCI, DIEA, DMF, 16%.</p></p></figure><figure id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0012.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>1q</b> and <b>1r</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOH, DMF, 120 Â°C, 45%; (b) NMP, DIEA, 140 Â°C, 22%; (c) THF, DIEA, 80 Â°C, 97%; (d) CoCl<sub>2</sub>â6H<sub>2</sub>O, NaBH<sub>4</sub>, ethanol, 28%; (e) NMP, DIEA, 80 Â°C, 58%; (f) (i) 4 N HCl/dioxane, MeOH, (ii) 4-acrylamidobenzoic acid, HOBT, EDCI, DIEA, DMF, 22â38%.</p></p></figure><div class="NLM_p last">For <b>1q</b> and <b>1r</b>, the synthesis of <b>36</b> is straightforward as <b>35</b> and <b>27</b> are all commercially available reagents, and thus one-step substitution reaction led to the key intermediate <b>41a</b>. Intermediate <b>41b</b> was synthesized using a three-step reaction sequence involving coupling 2-bromothiazole-5-carbonitrile with 3-aminopiperidine followed by reduction of cyano group to a primary amine and further reaction with 5-(<i>tert</i>-butyl)-2-(chloromethyl)oxazole. The rest of the reactions to install the acrylamide are the same as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26770" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26770" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Development of MFH290 as Covalent CDK12 Inhibitor</h3><div class="NLM_p">High-throughput biochemical screening of kinase-focused chemical libraries is a common method to find starting points for inhibitor optimization campaigns. However, our efforts to develop a kinase assay suitable for screening CDK12 reversible inhibitors were unsuccessful. Therefore, we changed our strategy to use a competition-based pulldown assay to screen for reversible CDK12 binders. Our aim was to identify compounds that bound CDK12 that could be optimized and converted to covalent inhibitors targeting the uniquely positioned cysteine 1039 (Cys1039) residue that lies in the C-terminal domain of CDK12. Competitive pulldown assays were performed by utilizing biotinylated (bio)THZ1, which has been shown to covalently bind and pull down both CDK12 and CDK13 (CDK12/13).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> As bioTHZ1 can be pulled down using streptavidin-coated beads, we gauged the ability of bioTHZ1 to bind free CDK12/13 by monitoring the extent to which CDK12, CDK13, or their obligate binding partner, cyclin K, were pulled down. We screened a focused collection of reported CDK inhibitors for their ability to protect CDK12 and -13 from being pulled down by bioTHZ1 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S1A in Supporting Information</a>). Among all known CDK inhibitors we screened, pretreatment with SNS032, a known inhibitor of CDK2, -7, and -9 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), resulted in a significant reduction of cyclin K pulldown from Jurkat cells by bioTHZ1 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S1B in Supporting Information</a>), providing a starting point for medicinal chemistry efforts.</div><div class="NLM_p">The cocrystal structure of SNS032 with CDK2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D1J">5D1J</a>) clearly illustrates the binding mode where the (1) 3-aminothiazole group forms two key hydrogen bonds with the hinge region, (2) 5-(<i>tert</i>-butyl)oxazole ring occupies the back hydrophobic pocket II, and (3) the piperidine faces the solvent exposed region. Structural overlay of CDK12 crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACB">5ACB</a>) and SNS032/CDK2 cocrystal structure indicated that the solvent-exposed piperidine ring would lie in close proximity to Cys1039 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Subsequent chemistry was performed to explore the ability to form a covalent bond with Cys1039 and to gain additional selectivity for CDK12 relative to other kinases. All newly synthesized compounds were assayed for CDK12 binding using the bioTHZ1 competition assay (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S2 in Supporting Information</a>). To quickly assess the selectivity for CDK12 versus the other CDKs targeted by SNS032, we used commercially available biochemical assays from Life Technologies for CDK2/cyclin A, CDK7/cyclin H/MNAT1, and CDK9/cyclin T1. In addition, we engineered HAP1 cells by mutating Cys1039 of CDK12 to a nonreactive serine to generate a chemicalâgenetic system to allow dissection of covalent-dependent inhibitor pharmacology.</div><div class="NLM_p">To explore advantageous modifications to SNS032, we started by installing the requisite acrylamide warhead on the solvent exposed region facing the cysteine residue. For example, compounds <b>1a</b>, <b>1b</b>, and <b>1c</b> were all designed with an acrylamide group attached to either phenyl or piperidine ring whose structural alignment suggested would position this warhead in close proximity to Cys1039. All warhead modifications conserved the ability to block cyclin K pulldown with bioTHZ1, indicating that CDK12/13 binding was preserved (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In regard to selectivity, compound <b>1b</b> maintained inhibitory activity against all the CDKs tested, whereas <b>1a</b> and <b>1c</b> exhibited dramatically reduced inhibition of CDK7 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S2 in Supporting Information</a>). In contrast, 3-aminopiperidine, which based on our work on THZ531 emerged as a privileged scaffold for selective CDK12 inhibition, led to a dramatic reduction in inhibition of CDK2/7/9 activity while CDK12/13 binding was retained (<b>1d</b>, hereafter named MFH290, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Moreover, the <i>S</i>-enantiomer of MFH290 (<b>1e</b>) lost the ability to target CDK12, meaning that the <i>R</i> configuration of 3-aminopiperidine is required to provide the proper trajectory to target Cys1039. Removing the carbonyl and installing an amino group through 3-aminocyclohexane (<b>1f</b>) slightly reduced CDK12 binding while also significantly restoring activity on CDK2 and CDK9. Therefore, further modifications focused on retaining the 3-aminopiperidine moiety.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Value or Pulldown Competition Percentage with bioTHZ1 of Compounds SNS032, <b>1a</b>â<b>1l</b> against CDK2, CDK7, CDK9, and CDK12</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0013.gif" alt="" id="fx1fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0014.gif" alt="" id="GRAPHIC-d7e939-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Tested using Zâ²-Lyte assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Tested using Adapta assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Tested using CDK12 pulldown assay. The remaining CDK12 protein in the presence of each compound was pulled down using bioTHZ1 and quantified by densitometry of the bands of CDK12 (normalized with DMSO-treated).</p></div><div class="footnote" id="t1fn4"><sup><sup>e</sup></sup><p class="last">â<b>1d</b>â is MFH290.</p></div></div><div></div></div><div class="NLM_p">Introduction of either a methyl group to the 2-position (<b>1g</b>) or two fluorines to the 5-postion of piperidine ring (<b>1h</b>) abrogated the inhibitory activity on all CDKs, whereas introduction of a methyl group to the 6-position (<b>1i</b>) resulted in comparable, but less selective, activity to MFH290. Moreover, a phenyl ring extending from the piperidine was found to be more favored than a pyridine ring (<b>1j</b>) and a small group such as a fluorine (<b>1k</b>) was better tolerated compared to a larger CF<sub>3</sub> group at the 3-position (<b>1l</b>) (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Replacement of the <i>tert</i>-butyloxazole group with pyridine (<b>1m</b>), pyrimidine (<b>1n</b>), or tetrahydropyran (<b>1o</b>) rings resulted in compete loss of activity. Additionally, the thioether linkage to the core thiazole was necessary for CDK12/13 activity as all non-thioether linked compounds such as <b>1p</b>, <b>1q</b> and <b>1r</b> were inactive (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Likewise, oxidation of the thiol to sulfoxide (<b>1s</b>) diminished inhibition and binding to CDK2/7/9 and CDK12. Finally, a nonsubstituted thiazole core was most favored as replacement with a pyrimidine (<b>1t</b>), thiadiazole (<b>1u</b>), or even methylated thiazole (<b>1v</b>) significantly reduced CDK12 binding activity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Taken all together, MFH290 emerged as our best-in-class inhibitor from this series.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub> Value or Pulldown Competition Percentage with bioTHZ1 of <b>1m</b>â<b>1s</b> against CDK2, CDK7, CDK9, and CDK12</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0015.gif" alt="" id="fx3fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0016.gif" alt="" id="GRAPHIC-d7e1061-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Tested using Zâ²-Lyte assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Tested using Adapta assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Tested using CDK12 pulldown assay. The remaining CDK12 protein in the presence of each compound was pulled down using bioTHZ1 and quantified by densitometry of the bands of CDK12 (normalized with DMSO-treated).</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub> Value or Pulldown Competition Percentage with bioTHZ1 of Compounds <b>1t</b>â<b>1v</b> against CDK2, CDK7, CDK9, and CDK12</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0017.gif" alt="" id="fx5fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0018.gif" alt="" id="GRAPHIC-d7e1104-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Tested using Zâ²-Lyte assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Tested using Adapta assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Tested using CDK12 pulldown assay. The remaining CDK12 protein in the presence of each compound was pulled down using bioTHZ1 and quantified by densitometry of the bands of CDK12 (normalized with DMSO-treated).</p></div></div><div></div></div><div class="NLM_p">To confirm the covalent bond formation of MFH290 with Cys1039 on CDK12, we analyzed purified CDK12 protein by LCâMS after incubation with a 10-fold molar excess of MFH290 for 2 h at 37 Â°C and observed a mass shift consistent with addition of a single molecule of MFH290 to CDK12 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). To confirm the site of modification, labeled protein was digested with trypsin and peptides were analyzed by capillary electrophoresis-MS/MS. Database search revealed exclusive modification of CDK12 by MFH290 at Cys1039 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Taken together, our mass-spectrometry-based analysis showed that MFH290 forms a covalent adduct with CDK12 by reacting with a single cysteine, Cys1039. We therefore carried MFH290 forward into all subsequent cellular experiments to characterize the inhibitor and evaluate CDK12/13 function.</div><figure id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CDK12 is exclusively labeled by MFH290 at Cys1039. (A) Mass spectra and (B) zero-charge mass spectra of CDK12/cyclin K complex treated with (top) DMSO or (bottom) a 10-fold excess of MFH290 for 1 h at room temperature. (A) Mass spectral peaks corresponding to cyclin K, unmodified CDK12, and CDK12 covalently labeled with MFH290 are indicated with black, green, and red circles, respectively. (B, left) Cyclin K is unaffected by compound treatment, while (B, right) CDK12 protein exhibits an increase in mass consistent with modification by a single molecule of MFH290. (C) MS/MS spectrum of CDK12 peptide <sup>1025</sup>LSKMAPPDLPHWQDC HE<sub>1041</sub> modified with MFH290 indicates covalent labeling of Cys1039. Ions of types b and y are indicated with blue and red glyphs, respectively. Inhibitor related ions 1<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> are marked with green glyphs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To characterize the kinome-wide selectivity of MFH290, we employed DiscoverXâs KinomeScan binding assays against a panel of 468 kinases at a concentration of 1 Î¼M.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Table S1 in Supporting Information</a>, MFH290 exhibits exquisite selectivity with CDK13 as the only potently inhibited target (CDK12 is not included in the panel). We also tested MFH290 on an extensive enzymatic CDKs panel which included CDKs 1â9, 12â14, and 16. The measured IC<sub>50</sub> values across all other CDKs are higher than 100 nM, indicating that MFH290 did not potently inhibit any of these CDKs (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).</div><figure id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0004.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Kinase profiling of MFH290. The levels of binding were measured at 1 Î¼M against a panel of 468 human kinases. The results for the binding interactions are reported as blue circles in the KiNOME tree. (B) IC<sub>50</sub> values for MFH290 against CDK1â9, 12â14, and 16: <sup><i>a</i></sup>tested in Zâ²-Lyte assays with ATP concentrations of <i>K</i><sub>M</sub>; <sup><i>b</i></sup>tested in LanthaScreen Eu kinase binding assay with ATP concentrations of <i>K</i><sub>M</sub>; <sup><i>c</i></sup>tested using Adapta assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>; <sup><i>d</i></sup>tested in radioisotope filter binding assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To define the working concentration range of MFH290, we employed the same competition-based target engagement assay described above (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S1A in Supporting Information</a>). Concentrations of MFH290 that block bioTHZ1-mediated pulldown of CDK12/13/cyclin K identify doses at which MFH290 covalently binds CDK12/13 in cells. Experiments in both Jurkat T-ALL and HAP1 (a near haploid) CML lines indicate that MFH290 saturates CDK12/13 binding at a concentration of approximately 40 nM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S3 in Supporting Information</a>).</div><figure id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) MFH290 targets CDK12 and CDK13 in cells. Treatment of HAP1 cells with MFH290 for 6 h blocks pulldown of CDK12 and associated cyclin K by biotinylated THZ1. (B) Antiproliferation assay using WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells. Output is CellTiterGlo luminescent signal following 72 h treatment with MFH290. (C) MFH290 reduces Pol II CTD phosphorylation in a concentration-dependent manner. Shown are effects of MFH290 on CTD phosphorylation in WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells after 6 h of treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To determine the cellular selectivity of MFH290, we used CRISPR/Cas9-mediated allelic replacement<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> to establish isogenic cell lines expressing covalent inhibitor refractory cysteine-to-serine mutant forms of CDK12 (C1039S), CDK13 (C1017S), and CDK12/CDK13 double mutants (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S4A in Supporting Information</a>). These isogenic lines enable us to assess the relative contribution of CDK12, CDK13, and CDK12/13 inhibition to MFH290 pharmacology. Western blotting demonstrated that WT and mutant forms of CDK12 and CDK13 were all expressed at similar levels (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Figure S4B in Supporting Information</a>).</div><div class="NLM_p">We previously reported that THZ531, our first-generation CDK12/13 inhibitor, led to a pronounced and rapid decrease in cell proliferation;<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> therefore we tested MFH290 across its effective dose range for antiproliferative activity in the HAP1 isogenic lines. In WT HAP1 cells, MFH290 displayed low nanomolar antiproliferative activity. Mutation of the reactive cysteine within CDK12, but not CDK13, significantly blunted these effects, with CDK12 mutant cells exhibiting 11-fold resistance to MFH290 to WT CDK12 cells, which was further enhanced to 25-fold as compared to WT when both CDK12 and CDK13 were mutated (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).</div><div class="NLM_p">CDK12 and CDK13 have been implicated in the regulation of RNA polymerase II (Pol II)-mediated gene transcription by phosphorylating Pol II at conserved residues within the CTD.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Therefore, we asked whether MFH290 affected Pol II CTD phosphorylation levels in our HAP1 isogenic lines. Indeed, MFH290 treatment reduced Pol II CTD phosphorylation at serine residues 2, 5, 7 (Ser2, Ser5, Ser7), as well as overall Pol II levels (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Mutation of CDK12 greatly restored levels of both total and phosphorylated Pol II to near WT levels in the presence of MFH290, while mutation of CDK13 exhibited a significantly reduced rescue. Complete rescue of Pol II CTD phosphorylation was achieved by expressing both mutant CDK12 and CDK13 in HAP1 cells (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The difference in rescue demonstrated by mutation of CDK12 compared to that of CDK13 likely stems from the fact that CDK12 is the more highly expressed and predominant homolog in HAP1 cells.</div><div class="NLM_p">As CDK12 has been shown to specifically regulate the expression of DNA damage response (DDR) genes,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> we treated HAP1 cells with MFH290 at 40 and 200 nM and analyzed expression of DDR RNA and protein expression by RT qPCR and Western blot. We found that treatment with MFH290 reduced expression of <i>RAD51</i> and <i>RAD51C</i> at both the RNA and protein level (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). These effects were almost completely rescued at the protein level and largely rescued at the RNA level by expression of mutant CDK12 alone or in combination with C1017S CDK13.</div><figure id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0006.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) MFH290 treatment leads to a reduction in DNA damage response proteins. Shown are Western blots of WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells following treatment with MFH290 for 24 h. (B) MFH290 treatment leads to reduced expression of DNA damage response genes. Shown are RT-qPCR of WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells following treatment with MFH290 for 6 h. (C) Excess over Bliss synergy plots for serial dilutions of MFH290 in combination with olaparib in HAP1 cells. Excess over Bliss scores >0 indicates drug synergy, whereas negative scores indicate antagonism.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Impaired expression of RAD51 and other homologous recombination (HR)-regulating proteins has been shown to sensitize cancer cells to PARP inhibitors.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> To ask if MFH290 could do the same, we treated HAP1 cells with increasing concentrations of both olaparib, an FDA-approved PARP inhibitor, and MFH290. We examined the proliferation data for synergistic effects as evaluated by extrapolated excess over Bliss score (0â1).<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Indeed MFH290 (25 nM) exhibited maximum synergy with olaparib (1.2 Î¼M) with excess over Bliss scores of 0.75 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).</div><div class="NLM_p">These data indicate that CDK12 is the primary target of MFH290 in HAP1 cells, that covalent targeting of CDK12 is obligatory for MFH290 activity, and that CDK12 inhibition is largely responsible for the observed antiproliferative effects. Consistent with biochemical kinase profiling, these results indicate that MFH290 has exceptional cellular selectivity for CDK12/13 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>â<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>), supporting the notion that CDK12/13 inhibition is responsible for inhibitor-induced effects on DDR gene expression.</div><div class="NLM_p">In order to know whether MFH290 is suitable for an <i>in vivo</i> efficacy study, we then took the compound for a pharmacokinetic study. As shown in the <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, MFH290 showed a short half-life in mouse plasma and a high clearance, indicating further optimization will be needed to improve its <i>in vivo</i> stability.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Pharmacokinetic Parameters of MFH290</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">route</th><th class="colsep0 rowsep0" align="center" char=".">dose</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>0</sub>/<i>C</i><sub>max</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>last</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub></th><th class="colsep0 rowsep0" align="center" char=".">CL</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub></th><th class="colsep0 rowsep0" align="center" char="."><i>F</i><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center" char=".">(mg/kg)</th><th class="colsep0 rowsep0" align="center" char=".">(h)</th><th class="colsep0 rowsep0" align="center" char=".">(ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">(hÂ·ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">(h)</th><th class="colsep0 rowsep0" align="center" char=".">(mLÂ min<sup>â1</sup>Â kg<sup>â1</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">(mL/kg)</th><th class="colsep0 rowsep0" align="center" char=".">(%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iv</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">1312.89</td><td class="colsep0 rowsep0" align="char" char=".">162.22</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td><td class="colsep0 rowsep0" align="char" char=".">102.03</td><td class="colsep0 rowsep0" align="char" char=".">730</td><td class="colsep0 rowsep0" align="char" char=".">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ip</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">0.08</td><td class="colsep0 rowsep0" align="char" char=".">885.05</td><td class="colsep0 rowsep0" align="char" char=".">393.63</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">po</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">556.46</td><td class="colsep0 rowsep0" align="char" char=".">412.44</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">25</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last"><i>C</i><sub>0</sub>, back extrapolated concentration for iv group.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">AUC<sub>last</sub> was considered for calculating bioavailability.</p></div></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81635" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81635" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have converted a pan-CDK inhibitor SNS032 to a selective CDK12/13 inhibitor MFH290 by hybridizing it with structural elements of the first selective CDK12/13 covalent inhibitor THZ531. Extensive structure-guided medicinal chemistry modifications delivered a narrow SAR, with (<i>R</i>)-3-aminopiperidine emerging as a privileged scaffold, enabling high binding affinity for CDK12/13 to the exclusion of other CDKs. MFH290 formed a covalent bond with Cys1039 on CDK12 and thus demonstrated a sustained CDK12 inhibition-induced phenotype including strong antiproliferative effect on cancer cells, a transcriptional defect for DDR genes, and a combinatorial effect with PARP inhibition. Compared to first generation covalent inhibitor THZ531, a higher degree of rescue from MFH290-induced phenotypes in HAP1 cells engineered with cysteine-to-serine CDK12 and CDK13 mutations implies that MFH290 possesses a superior selectivity for CDK12/13. Further optimization of this scaffold to improve <i>in vivo</i> properties will enable <i>in vivo</i> investigation of drug efficacy.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76243" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76243" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemistry</h3><div class="NLM_p last">Unless otherwise noted, reagents and solvents were obtained from commercial suppliers and were used without further purification. <sup>1</sup>H NMR spectra were recorded on 500 MHz (Bruker A500), and chemical shifts are reported in parts per million (ppm, Î´) downfield from tetramethylsilane (TMS). Coupling constants (<i>J</i>) are reported in Hz. Spin multiplicities are described as s (singlet), br (broad singlet), d (doublet), t (triplet), q (quartet), and m (multiplet). Mass spectra were obtained on a Waters Micromass ZQ instrument. Preparative HPLC was performed on a Waters Sunfire C18 column (19 mm Ã 50 mm, 5 Î¼m) using a gradient of 15â95% methanol in water or acetonitrile in water containing 0.05% trifluoroacetic acid (TFA) over 22 min (28 min run time) or 35 min (45 min run time) at a flow rate of 20 mL/min. All of the final compounds reported had purities greater than 95% based on <sup>1</sup>H nuclear magnetic resonance (NMR) and liquid chromatographyâmass spectrometry (LCâMS).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthesis of Compound <b>1a</b></h3><div class="NLM_p">To a solution of 5-(((5-(<i>tert</i>-butyl)oxazol-2-yl)methyl)thio)thiazol-2-amine (54 mg, 0.2 mmol) in DMF (1 mL) were added HATU (160 mg, 0.42 mmol), 3-acrylamidobenzoic acid (60 mg, 0.31 mmol), and DIEA (52 mg, 0.4 mmol). The resulting mixture was stirred at room temperature for 1 h and then diluted with ethyl acetate (100 mL). The mixture was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by silica gel column (MeOH/DCM, 0â15%) to afford the compound <b>1a</b> (7 mg, yield 8%) as a white solid.</div><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 3-Acrylamido-<i>N</i>-(5-(((5-(<i>tert</i>-butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)benzamide (<b>1a</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 12.36 (s, 1H), 9.08 (s, 1H), 8.24â7.99 (m, 1H), 7.90 (s, 1H), 7.63 (d, <i>J</i> = 7.6 Hz, 1H), 7.41 (t, <i>J</i> = 7.9 Hz, 1H), 6.95 (s, 1H), 6.55 (s, 1H), 6.44 (d, <i>J</i> = 16.8 Hz, 1H), 6.32 (dd, <i>J</i> = 16.8, 10.0 Hz, 1H), 5.76 (d, <i>J</i> = 10.0 Hz, 1H), 3.95 (s, 2H), 1.23 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 443.12 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Synthesis of Compounds <b>1b</b> and <b>1c</b></h3><div class="NLM_p">To a solution of 5-(((5-(<i>tert</i>-butyl)oxazol-2-yl)methyl)thio)thiazol-2-amine (<b>4</b>) (0.69 mmol, 1 equiv), appropriate acid (0.94 mmol, 1.4 equiv), DMAP (0.35 mmol, 0.5 equiv), and triethylamine (1.4 mmol, 2 equiv) in DMF (1 mL) and DCM (2 mL) was added EDCI (1.4 mmol, 2 equiv) at room temperature. The reaction mixture was stirred for 1.5 h and then diluted with ethyl acetate. The solution was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by silica gel column and then treated with 4 M HCl in dioxane (3 mL). The reaction was stirred at room temperature for 30 min. The mixture was concentrated under vacuum to afford the compounds <b>8</b> and <b>9</b> as HCl salt.</div><div class="NLM_p">To a solution of <b>8</b> (50 mg, 0.12 mmol) in DMF (1 mL) were added HATU (92 mg, 0.24 mmol), 4-acrylamidobenzoic acid (46 mg, 0.24 mmol), and DIEA (0.22 mL, 1.2 mmol). The resulting mixture was stirred at room temperature for 1 h and then diluted with ethyl acetate (100 mL). The solution was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by silica gel column (MeOH/DCM, 0â20%) to afford the compound <b>1b</b> as a white solid.</div><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 1-(4-Acrylamidobenzoyl)-<i>N</i>-(5-(((5-(<i>tert</i>-butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)piperidine-3-carboxamide (<b>1b</b>)</h4><div class="NLM_p"><sup>1</sup>H NMR (400 MHz, DMSO) Î´ 12.37 (s, 1H), 10.32 (s, 1H), 7.73 (d, <i>J</i> = 7.8 Hz, 2H), 7.55â7.22 (m, 3H), 6.71 (s, 1H), 6.44 (dd, <i>J</i> = 16.9, 10.3 Hz, 1H), 6.28 (d, <i>J</i> = 16.8 Hz, 1H), 5.78 (d, <i>J</i> = 9.9 Hz, 1H), 4.05 (s, 2H), 3.67â2.69 (m, 4H), 1.97â1.23 (m, 5H), 1.12 (s, 9H). MS (ESI) m/554.21 [M + H]<sup>+</sup>.</div><div class="NLM_p last">To a solution of <b>9</b> (20 mg, 0.05 mmol) in THF (5 mL) were added DIEA (17 mg, 0.15 mmol) and acryloyl chloride (43 mg, 0.05 mmol) at 0 Â°C for 1 h. The resulting mixture was stirred at room temperature for 2 h. The solution was concentrated and purified by silica gel column (MeOH/DCM, 0â20%) to afford the compound <b>1c</b> as a slightly white solid.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 1-Acryloyl-<i>N</i>-(5-(((5-(<i>tert</i>-butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)piperidine-4-carboxamide (<b>1c</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 11.64 (s, 1H), 7.29 (s, 1H), 6.61â6.55 (m, 2H), 6.28 (d, <i>J</i> = 16.7 Hz, 1H), 5.71 (d, <i>J</i> = 10.6 Hz, 1H), 4.60â4.57 (m, 1H), 4.12â3.96 (m, 1H), 3.96 (s, 2H), 3.21â2.90 (m, 2H), 2.68 (t, <i>J</i> = 10.4 Hz, 1H), 2.01â1.82 (m, 4H), 1.24 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 435.15 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Synthesis of Compounds <b>1d</b>â<b>1f</b></h3><div class="NLM_p">A mixture of 2-(((2-bromothiazol-5-yl)thio)methyl)-5-(<i>tert</i>-butyl)oxazole<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (<b>10</b>) (0.45 mmol, 1 equiv), appropriate amine (0.9 mmol, 2 equiv), and DIEA (1.35 mmol, 3 equiv) in NMP (3 mL) was heated to 140 Â°C overnight. The solution was diluted with water (20 mL) and extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by silica gel column (MeOH/DCM, 0â20%) to give compound <b>11a</b> and <b>11b</b>.</div><div class="NLM_p">To a mixture of <b>11b</b> (0.33 mmol, 1 equiv) in methanol (2 mL) was added 4 N HCl/dioxane (2 mL). The resulting solution was stirred at room temperature for 3 h. The mixture was concentrated under reduced pressure to give the corresponding HCl salt. To the solution of HCl salt (0.33 mmol, 1 equiv) in pyridine (2 mL) was added 4-nitrobenzoyl chloride (0.66 mmol, 2 equiv). The reaction mixture was stirred at room temperature overnight. The resulting mixture was concentrated under reduced pressure and the residue was purified by silica gel column (MeOH/DCM, 0â20%) to give compound <b>13</b>.</div><div class="NLM_p">To a solution of <b>13</b> (0.54 mmol, 1 equiv) in ethyl acetate and methanol (1:1) were added tin(II) chloride dihydrate (5.4 mmol, 10 equiv) and conc HCl (0.2 mL). After stirring for 3 h at 80 Â°C, the reaction mixture was diluted with chloroform and isopropanol (4:1), neutralized with saturated NaHCO<sub>3</sub>, and filtered. The filtrate was extracted with chloroform and isopropanol (4:1), concentrated under reduced pressure and the resulting residue was purified by silica gel column chromatography (MeOH/DCM = 0â20%) to give the compound <b>14</b>.</div><div class="NLM_p">To a solution of <b>14</b> (0.034 mmol, 1 equiv) and DIEA (0.1 mmol, 3 equiv) in MeCN (2 mL) was added acryloyl chloride (0.044 mmol, 1.3 equiv) dropwise at 0 Â°C. When the reaction was completed, Na<sub>2</sub>CO<sub>3</sub> aq (sat.) was added, and the resulting mixture was extracted with chloroform and isopropanol (4:1) (20 mL Ã 2). The combined organic layer was concentrated and purified with prep-HPLC (MeOH/H<sub>2</sub>O, 0.05% TFA) to give compounds <b>1d</b> and <b>1e</b>. <b>1f</b> was synthesized with same method.</div><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1d</b>, MFH290)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.27 (s, 1H), 7.98 (s, 1H), 7.85â7.57 (m, 2H), 7.49â7.27 (m, 2H), 7.15â6.81 (m, 1H), 6.71 (s, 1H), 6.44 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.28 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.78 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.94 (s, 2H), 3.86â3.61 (m, 2H), 3.53â2.77 (m, 3H), 2.02â1.40 (m, 4H), 1.18 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 526.50 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (<i>S</i>)-<i>N</i>-(4-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1e</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.31 (s, 1H), 8.08 (s, 1H), 7.80â7.60 (m, 2H), 7.46â7.28 (m, 2H), 7.00â6.82 (m, 1H), 6.71 (s, 1H), 6.43 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.27 (dd, <i>J</i> = 17.0, 1.9 Hz, 1H), 5.78 (dd, <i>J</i> = 10.1, 1.9 Hz, 1H), 3.94 (s, 2H), 3.73â3.63 (m, 2H), 3.34â3.21 (m, 1H), 3.15â2.70 (m, 2H), 2.03â1.41 (m, 4H), 1.17 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 526.50 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-(4-((3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)cyclohexyl)amino)phenyl)acrylamide (<b>1f</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.37â10.01 (m, 1H), 8.19â7.96 (m, 1H), 7.79â7.56 (m, 2H), 7.30â7.02 (m, 2H), 6.98â6.88 (m, 1H), 6.75â6.66 (m, 1H), 6.49â6.36 (m, 1H), 6.30â6.19 (m, 1H), 5.81â5.68 (m, 1H), 4.03 (s, 1H), 3.96â3.92 (m, 2H), 2.28â2.17 (m, 1H), 2.00â1.85 (m, 2H), 1.85â1.74 (m, 1H), 1.74â1.49 (m, 2H), 1.44â1.22 (m, 3H), 1.22â1.16 (m, 9H), 1.15â1.00 (m, 1H). MS (ESI) <i>m</i>/<i>z</i> 511.96 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Synthesis of Compounds <b>1g</b>â<b>1i</b>, <b>1k</b>, and <b>1l</b></h3><div class="NLM_p">To a mixture of compound <b>11b</b> (0.024 mmol, 1 equiv) in methanol (2 mL) was added 4 N HCl/dioxane (2 mL). The resulting solution was stirred at rt for 3 h. The mixture was concentrated under reduced pressure to give the corresponding HCl salt. To a mixture of the HCl salt (0.024 mmol, 1 equiv), <b>12</b> (0.03 mmol, 1.3 equiv), and DIEA (0.12 mmol, 5 equiv) in DCM (2 mL) was added T<sub>3</sub>P (50% solution in ethyl acetate, 46 mg, 0.07 mmol) dropwise at room temperature. The reaction mixture was stirred at room temperature overnight. The solution was diluted with DCM (20 mL) and washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by prep-HPLC (MeOH/H<sub>2</sub>O, 0.05% TFA) to give the compounds <b>1g</b>â<b>1i</b>, <b>1k</b>, and <b>1l</b>.</div><div id="sec5_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-(4-((2<i>R</i>,3<i>R</i>)-3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)-2-methylpiperidine-1-carbonyl)phenyl)acrylamide (<b>1g</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.30 (s, 1H), 8.02 (s, 1H), 7.72 (d, <i>J</i> = 8.7 Hz, 2H), 7.38 (d, <i>J</i> = 8.2 Hz, 2H), 7.00â6.62 (m, 2H), 6.45 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.28 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.79 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.94 (s, 2H), 3.86â3.70 (m, 2H), 3.14â2.57 (m, 2H), 1.81â1.44 (m, 4H), 1.17 (s, 9H), 1.04 (d, <i>J</i> = 6.9 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 540.58 [M + H]<sup>+</sup>.</div></div><div id="sec5_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i>-(4-(5-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)-3,3-difluoropiperidine-1-carbonyl)phenyl)acrylamide (<b>1h</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.35 (s, 1H), 8.20â8.04 (m, 1H), 7.74 (d, <i>J</i> = 8.3 Hz, 2H), 7.42 (d, <i>J</i> = 8.5 Hz, 2H), 7.05â6.90 (m, 1H), 6.76â6.66 (m, 1H), 6.45 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.29 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.80 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.96 (s, 2H), 3.78â3.50 (m, 4H), 2.16â2.06 (m, 1H), 1.90â1.65 (m, 2H), 1.22 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 562.54 [M + H]<sup>+</sup>.</div></div><div id="sec5_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(4-((2<i>S</i>,5<i>R</i>)-5-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)-2-methylpiperidine-1-carbonyl)phenyl)acrylamide (<b>1i</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.30 (s, 1H), 8.03 (s, 1H), 7.73 (d, <i>J</i> = 8.7 Hz, 2H), 7.39 (d, <i>J</i> = 8.2 Hz, 2H), 6.93 (s, 1H), 6.71 (s, 1H), 6.45 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.28 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.79 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.95 (s, 2H), 3.60â3.49 (m, 2H), 2.92â2.56 (m, 2H), 1.93â1.79 (m, 1H), 1.81â1.43 (m, 3H), 1.39â0.70 (m, 12H). MS (ESI) <i>m</i>/<i>z</i> 540.58 [M + H]<sup>+</sup>.</div></div><div id="sec5_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)-2-fluorophenyl)acrylamide (<b>1k</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.06 (s, 1H), 8.21â7.87 (m, 2H), 7.31 (s, 1H), 7.22â7.16 (m, 1H), 6.80 (s, 1H), 6.75â6.69 (m, 1H), 6.66 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.29 (dd, <i>J</i> = 16.9, 2.0 Hz, 1H), 5.80 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 3.94 (s, 2H), 3.73â3.65 (m, 2H), 3.36â3.00 (m, 3H), 2.01â1.92 (m, 1H), 1.80â1.52 (m, 3H), 1.18 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 544.56 [M + H]<sup>+</sup>.</div></div><div id="sec5_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)-2-(trifluoromethyl)phenyl)acrylamide (<b>1l</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 9.91 (s, 1H), 8.25â7.99 (m, 1H), 7.83â7.55 (m, 3H), 7.05â6.70 (m, 2H), 6.61â6.52 (m, 1H), 6.36â6.22 (m, 1H), 5.87â5.77 (m, 1H), 3.93 (s, 2H), 3.79â3.64 (m, 2H), 3.40â3.14 (m, 2H), 1.98â1.91 (m, 1H), 1.84â1.52 (m, 4H), 1.19 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 594.56 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Synthesis of Compound <b>1s</b></h3><div class="NLM_p">To a solution of (<i>R</i>)-(4-aminophenyl)(3-((5-(((5-(<i>tert</i>-butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)piperidin-1-yl)methanone (<b>14</b>, 30 mg, 0.064 mmol) in DCM (2 mL) was added <i>m</i>-CPBA (12 mg, 0.07 mmol). The reaction mixture was stirred at room temperature overnight. Then it was diluted with DCM, washed with NaHCO<sub>3</sub> aq (sat.), concentrated, and purified with silica gel column (eluted with MeOH in DCM 0% to 15%) to give the sulfoxide intermediate (20 mg, 64%) as a brown oil. To the solution of resulting sulfoxide (20 mg, 0.04 mmol) in MeCN (2 mL) was added DIEA (25 mg, 0.2 mmol). At 0 Â°C acryloyl chloride (4 mg, 0.044 mmol) was added dropwise. When the reaction was completed, Na<sub>2</sub>CO<sub>3</sub> aq (sat.) was added. The resulting mixture was extracted with chloroform and isopropanol (4:1) (20 mL Ã 2). The combined organic layer was concentrated and purified with prep-HPLC (MeOH/H<sub>2</sub>O, 0.05%TFA) to give compound <b>1s</b> (8 mg, 36.9%) as white solid.</div><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i>-(4-((3<i>R</i>)-3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)sulfinyl)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1s</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.18 (s, 1H), 8.45 (s, 1H), 7.59 (s, 2H), 7.47â7.00 (m, 3H), 6.68 (s, 1H), 6.43â6.27 (m, 1H), 6.25â6.08 (m, 1H), 5.76â5.58 (m, 1H), 4.53â4.32 (m, 2H), 3.97â3.38 (m, 3H), 3.34â2.81 (m, 2H), 1.98â1.80 (m, 1H), 1.77â1.31 (m, 3H), 1.04 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 542.55 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Synthesis of Compounds <b>1m</b>â<b>1o</b>, <b>1u</b>, <b>1v</b></h3><div class="NLM_p">A mixture of <b>15</b> (3 mmol, 1 equiv) and potassium thiocyanate (30 mmol, 10 equiv) in methanol (20 mL) was stirred at room temperature for 48 h. Methanol was evaporated, and water (3 mL) was added. The pH of the aqueous solution was adjusted to pH = 12 with 10% NaOH aq, and a precipitate formed. The solid was collected by filtration to yield compound <b>16</b>.</div><div class="NLM_p">To a solution of compound <b>16</b> (1 mmol, 1 equiv) in absolute ethanol (3 mL) was added NaBH<sub>4</sub> (2 mmol, 2 equiv) portionwise at 0 Â°C. The mixture was stirred at room temperature for 1 h, and then acetone (2 mL) was slowly introduced. After 1 h, a solution of appropriate halide (1.1 mmol, 1.1 equiv) in ethanol (2 mL) was added. The resulting dark reaction mixture was heated to 65 Â°C for 1 h and was then cooled and concentrated in vacuo. The residue was partitioned between ethyl acetate and brine. The organic phase was separated, dried over MgSO<sub>4</sub>, concentrated, and purified with silica gel column (MeOH/DCM, 0â10%) to provide compound <b>17</b>.</div><div class="NLM_p">To a solution of CuBr<sub>2</sub> (1.9 mmol, 1.7 equiv) in acetonitrile (10 mL) at 0 Â°C was added <i>t</i>-BuONO (1.9 mmol, 1.7 equiv) followed by compound <b>17</b> (1.1 mmol, 1 equiv). The mixture was stirred at 0 Â°C for 1 h, then at room temperature for 1 h. Ethyl acetate was added, and the organic layer was washed with 1 N hydrochloric acid (2 Ã 50 mL), dried over magnesium sulfate, filtered through a pad of silica gel, and concentrated in vacuo. The residue was purified on silica gel column (ethyl acetate in hexane, 0â50%) to give compound <b>18</b>.</div><div class="NLM_p">A mixture of <b>18</b> (0.4 mmol, 1 equiv), <i>tert</i>-butyl (<i>R</i>)-3-aminopiperidine-1-carboxylate (0.8 mmol, 2 equiv), DIEA (1.2 mmol, 3 equiv) in NMP (3 mL) was heated to 140 Â°C overnight. The solution was diluted with water (20 mL) and extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by silica gel column (MeOH/DCM, 0â20%) to give compound <b>19</b>.</div><div class="NLM_p">To a mixture of compound <b>19</b> (0.47 mmol, 1 equiv) in methanol (2 mL) was added 4 N HCl/dioxane (2 mL), and the resulting solution was stirred at room temperature for 3 h. The mixture was concentrated, and the residue was purified by prep-HPLC (MeOH/H<sub>2</sub>O, 0.05% TFA) to give compound <b>20</b>.</div><div class="NLM_p">To a mixture of <b>20</b> (22 mg, 0.06 mmol), 4-acrylamidobenzoic acid (14 mg, 0.072 mmol), and DIEA (39 mg, 0.3 mmol) in DCM (2 mL) was added T<sub>3</sub>P (50% solution in ethyl acetate, 57 mg, 0.18 mmol) dropwise at rt. The reaction mixture was stirred at rt overnight. The solution was diluted with DCM (20 mL) and washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by prep-HPLC (MeOH/H<sub>2</sub>O, 0.05% TFA) to give the compounds <b>1m</b>â<b>1o</b>, <b>1u</b>, and <b>1v</b>.</div><div id="sec5_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>R</i>)-<i>N</i>-(4-(3-((5-((Pyridin-4-ylmethyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1m</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.30 (s, 1H), 8.41â7.57 (m, 5H), 7.36 (s, 2H), 7.25â7.02 (m, 2H), 6.81 (s, 1H), 6.43 (dd, <i>J</i> = 17.4, 9.7 Hz, 1H), 6.27 (d, <i>J</i> = 17.0 Hz, 1H), 5.78 (d, <i>J</i> = 10.3 Hz, 1H), 3.99 (s, 2H), 3.44â2.75 (m, 5H), 2.02â1.66 (m, 2H), 1.66â1.37 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 479.96 [M + H]<sup>+</sup>.</div></div><div id="sec5_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((2-(Trifluoromethyl)pyrimidin-5-yl)methyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1n</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.26 (s, 1H), 8.81 (s, 2H), 8.10 (s, 1H), 7.92â7.59 (m, 2H), 7.51â7.23 (m, 2H), 7.12â6.72 (m, 1H), 6.56â6.35 (m, 1H), 6.35â6.16 (m, 1H), 5.87â5.68 (m, 1H), 4.01 (s, 2H), 3.42â2.82 (m, 5H), 2.02â1.87 (m, 1H), 1.85â1.63 (m, 1H), 1.63â1.39 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 548.89 [M + H]<sup>+</sup>.</div></div><div id="sec5_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((Tetrahydro-2<i>H</i>-pyran-4-yl)methyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1o</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.29 (s, 1H), 8.07 (s, 1H), 7.86â7.57 (m, 2H), 7.52â7.31 (m, 2H), 7.19â6.88 (m, 1H), 6.45 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.28 (dd, <i>J</i> = 16.9, 2.0 Hz, 1H), 5.79 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.90â3.78 (m, 2H), 3.24 (t, <i>J</i> = 11.5 Hz, 2H), 3.15â2.95 (m, 1H), 2.64â2.54 (m, 2H), 2.03â1.46 (m, 6H), 1.40â1.03 (m, 3H). MS (ESI) <i>m</i>/<i>z</i> 487.31 [M + H]<sup>+</sup>.</div></div><div id="sec5_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)-1,3,4-thiadiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1u</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.29 (s, 1H), 8.02 (d, <i>J</i> = 6.9 Hz, 1H), 7.68 (s, 2H), 7.36 (s, 2H), 6.75 (s, 1H), 6.44 (dd, <i>J</i> = 16.9, 10.1 Hz, 1H), 6.27 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.78 (dd, <i>J</i> = 10.0, 2.0 Hz, 1H), 4.33 (s, 2H), 4.01â3.63 (m, 3H), 3.35â2.96 (m, 2H), 2.07â1.42 (m, 4H), 1.18 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 526.98 [M + H]<sup>+</sup>.</div></div><div id="sec5_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)-4-methylthiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1v</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.28 (s, 1H), 8.05 (s, 1H), 7.76â7.58 (m, 2H), 7.35 (s, 2H), 6.70 (s, 1H), 6.44 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.28 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.78 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 3.87 (s, 2H), 3.70â3.55 (m, 2H), 3.28â3.01 (m, 3H), 1.96â1.84 (m, 2H), 1.62â1.44 (m, 5H), 1.17 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 540.54 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Synthesis of Compound <b>1j</b></h3><div class="NLM_p">To the mixture of <b>20</b> (0.12 mmol, 1 equiv), 5-nitropicolinic acid (0.14 mmol, 1.2 equiv), and DIEA (0.6 mmol, 5 equiv) in DCM (2 mL) was added T<sub>3</sub>P (50% solution in ethyl acetate, 0.36 mmol, 3 equiv) dropwise at room temperature. The reaction mixture was stirred at room temperature overnight. The solution was diluted with DCM (20 mL) and washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by silica gel column (MeOH/DCM, 0â20%).</div><div class="NLM_p">To the solution of the resulting intermediate (0.54 mmol, 1 equiv) in ethyl acetate and methanol (1:1) were added tin(II) chloride dihydrate (5.4 mmol, 10 equiv) and conc HCl (0.2 mL). After stirring for 3 h at 80 Â°C, the reaction mixture was diluted with chloroform and isopropanol (4:1), neutralized with saturated NaHCO<sub>3</sub>, and filtered. The filtrate was extracted with chloroform and isopropanol (4:1), concentrated under reduced pressure and the resulting residue was purified by silica gel column chromatography (MeOH/DCM = 0â20%) to give the corresponding amine intermediate.</div><div class="NLM_p">To a solution of amine intermediate (0.05 mmol, 1 equiv) in THF (5 mL) were added DIEA (0.15 mmol, 3 equiv) and acryloyl chloride (0.05 mmol, 1 equiv) at 0 Â°C for 30 min. The resulting mixture was stirred at room temperature for 2 h. The solution was concentrated and purified by silica gel column (MeOH/DCM, 0â20%) to afford the compound <b>1j</b> as a slightly white solid.</div><div id="sec5_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>R</i>)-<i>N</i>-(6-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)thiazol-2-yl)amino)piperidine-1-carbonyl)pyridin-3-yl)acrylamide (<b>1j</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) Î´ 8.93â8.51 (m, 1H), 8.29â7.89 (m, 1H), 7.49 (s, 1H), 7.19â6.79 (m, 1H), 6.74â6.53 (m, 1H), 6.42â6.15 (m, 2H), 5.87â5.68 (m, 1H), 4.24â3.81 (m, 3H), 3.80â3.23 (m, 4H), 2.14â1.47 (m, 4H), 1.15 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 527.52 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Synthesis of Compound <b>1t</b></h3><div class="NLM_p">A mixture of 2,4,5-trichloropyrimidine (<b>21</b>) (500 mg, 2.7 mmol) and potassium thiocyanate (1.1 g, 10.9 mmol) in methanol (10 mL) was stirred at room temperature for 48 h. Methanol was evaporated, and water (3 mL) was added. The pH of the aqueous solution was adjusted to pH = 12 with 10% NaOH aq, and a precipitate formed. The solid was collected by filtration to yield compound <b>22</b>.</div><div class="NLM_p">To a solution of compound <b>22</b> (300 mg, 1.5 mmol) in absolute ethanol (3 mL) was added NaBH<sub>4</sub> (110 mg, 2.9 mmol) portionwise at 0 Â°C. The mixture was stirred at room temperature for 1 h, and then acetone (3 mL) was slowly introduced. After 1 h, a solution of 5-(<i>tert</i>-butyl)-2-(chloromethyl)oxazole (253 mg, 1.46 mmol) in ethanol (5 mL) was added. The resulting dark reaction mixture was heated to 65 Â°C for 1 h and was then cooled and concentrated in vacuo. The residue was partitioned between ethyl acetate and brine. The organic phase was separated, dried over MgSO<sub>4</sub>, concentrated, and purified with silica gel column (eluted with MeOH in DCM 0% to 10%) to provide the compound <b>23</b>.</div><div class="NLM_p">A mixture of <b>23</b> (78 mg, 0.25 mmol), <i>tert</i>-butyl (<i>R</i>)-3-aminopiperidine-1-carboxylate (74 mg, 0.37 mmol), DIEA (63 mg, 0.49 mmol) in NMP (1 mL) was heated to 140 Â°C overnight. The solution was diluted with water (20 mL) and extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was purified by silica gel column (MeOH/DCM, 0â20%) to give compound <b>24</b>.</div><div class="NLM_p">To a mixture of <b>24</b> (120 mg, 0.25 mmol) in methanol (4 mL) was added 4 N HCl/dioxane (4 mL). The resulting solution was stirred at room temperature for 3 h. The mixture was concentrated under reduced pressure to give the corresponding HCl salt. To the solution of HCl salt (95 mg, 0.25 mmol) in pyridine (2 mL) was added 4-nitrobenzoyl chloride (56 mg, 0.3 mmol). The reaction mixture was stirred at room temperature overnight. The resulting mixture was concentrated under reduced pressure and the residue was purified by silica gel column (MeOH/DCM, 0â20%) to give compound <b>25</b>.</div><div class="NLM_p">To a solution of <b>25</b> (133 mg, 0.25 mmol) in ethyl acetate and methanol (1:1, 3 mL/3 mL) were added tin(II) chloride dihydrate (339 mg, 1.5 mmol) and conc HCl (0.2 mL). After stirring for 3 h at 80 Â°C, the reaction mixture was diluted with chloroform and isopropanol (4:1), neutralized with saturated NaHCO<sub>3</sub>, and filtered. The filtrate was extracted with chloroform and isopropanol (4:1), concentrated under reduced pressure and the resulting residue was purified by silica gel column chromatography (MeOH/DCM = 0â20%) to give compound <b>26</b>.</div><div class="NLM_p">To a solution of compound <b>26</b> (25 mg, 0.05 mmol) and DIEA (19 mg, 0.15 mmol) in MeCN (2 mL) was added acryloyl chloride (5 mg, 0.06 mmol) dropwise at 0 Â°C. When the reaction was completed, Na<sub>2</sub>CO<sub>3</sub> aq (sat.) was added, and the resulting mixture was extracted with chloroform and isopropanol (4:1) (20 mL x 2). The combined organic layer was concentrated and purified with prep-HPLC (MeOH/H<sub>2</sub>O, 0.05% TFA) to give compound <b>1t</b>.</div><div id="sec5_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>R</i>)-<i>N</i>-(4-(3-((4-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)thio)-5-chloropyrimidin-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1t</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.26 (s, 1H), 8.09 (s, 1H), 7.86â6.99 (m, 5H), 6.72 (s, 1H), 6.44 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.28 (dd, <i>J</i> = 17.0, 2.0 Hz, 1H), 5.78 (dd, <i>J</i> = 10.1, 2.0 Hz, 1H), 4.86â4.32 (m, 2H), 3.95â3.64 (m, 2H), 3.17 (s, 2H), 2.94â2.63 (m, 1H), 2.02â1.71 (m, 2H), 1.66â1.39 (m, 2H), 1.20 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 554.93 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Synthesis of Compound <b>1p</b></h3><div class="NLM_p">A solution of 5-(<i>tert</i>-butyl)-2-(chloromethyl)oxazole (<b>27</b>) (800 mg, 4.61 mmol) in triethyl phosphite (3.83 g, 23.04 mmol) was heated at 120 Â°C overnight. The solvent was removed under reduced pressure to provide a crude <b>28</b> which was directly used in the next step.</div><div class="NLM_p">To a solution of <b>28</b> (500 mg, 1.82 mmol) in THF (5 mL) was added <i>t</i>-BuOK (370 mg, 3.3 mmol) portionwise at room temperature. The vial was sealed and stirred at room temperature for 10 min. Then, a solution of <i>tert</i>-butyl 5-formylthiazol-2-ylcarbamate (345 mg, 1.5 mmol) in THF (3 mL) was added in the reaction mixture. After completion, the reaction mixture was quenched with water. The obtained mixture was extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the residue was purified by silica gel (MeOH/DCM = 0â20%) to obtain compound <b>30.</b></div><div class="NLM_p">A mixture of compound <b>30</b> (250 mg, 1 mmol) and 10% Pd/C (20 mg) in MeOH (10 mL) was stirred for 5 h at room temperature under H<sub>2</sub> balloon. The mixture was filtered through Celite. To the filtrate was added 4 N HCl/dioxane (5 mL). The solution was then stirred for 3 h at room temperature and the solvent was removed under reduced pressure to provide compound <b>31</b> as HCl salt which was directly used in the next step.</div><div class="NLM_p">To a solution of CuBr<sub>2</sub> (53 mg, 0.24 mmol) in acetonitrile (2 mL) at 0 Â°C was added <i>t</i>-BuONO (25 mg, 0.24 mmol) followed by compound <b>31</b> (60 mg, 0.24 mmol). The mixture was stirred at 0 Â°C for 1 h, then at room temperature for 1 h. Ethyl acetate was added, and the organic mixture was washed with 1 N hydrochloric acid (2 Ã 20 mL), dried over magnesium sulfate, filtered through a pad of silica gel, and concentrated in vacuo. The residue was chromatographed on silica gel (ethyl acetate in hexane, 0â50%) to give compound <b>32</b>.</div><div class="NLM_p">The mixture of <b>32</b> (40 mg, 0.13 mmol), (<i>R</i>)-<i>tert</i>-butyl 3-aminopiperidine-1-carboxylate (41 mg, 0.2 mmol), and DIEA (33 mg, 0.25 mmol) in NMP (0.5 mL) was stirred at 140 Â°C overnight. The residue was extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (10 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the residue was purified by silica gel (MeOH/DCM = 0â20%) to obtain compound <b>33.</b></div><div class="NLM_p">To a solution of compound <b>33</b> (35 mg, 0.08 mmol) in methanol (2 mL) was added 4 N HCl/dioxane (2 mL). The solution was then stirred for 3 h at room temperature and the solvent was removed under reduced pressure to provide <b>34</b> as HCl salt which was directly used in the next step.</div><div class="NLM_p">The mixture of compound <b>34</b> (23 mg, 0.07 mmol), 4-acrylamidobenzoic acid (15 mg, 0.08 mmol), HOBT (12 mg, 0.09 mmol), and EDCI (17 mg, 0.09 mmol) in DMF (0.5 mL) was stirred at room temperature overnight. The reaction mixture was purified by prep-HPLC (MeOH/H<sub>2</sub>O, 0.05%TFA) to provide compound <b>1p</b>.</div><div id="sec5_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(2-(5-(<i>tert</i>-Butyl)oxazol-2-yl)ethyl)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1p</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.39 (s, 1H), 8.30 (d, <i>J</i> = 7.5 Hz, 1H), 7.83 (d, <i>J</i> = 8.8 Hz, 2H), 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 6.96 (s, 1H), 6.69 (s, 1H), 6.46 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.29 (dd, <i>J</i> = 17.0, 1.9 Hz, 1H), 5.80 (dd, <i>J</i> = 10.1, 1.9 Hz, 1H), 4.03â3.83 (m, 2H), 3.68 (d, <i>J</i> = 12.8 Hz, 1H), 3.14â2.91 (m, 6H), 2.01â1.79 (m, 2H), 1.68â1.54 (m, 2H), 1.22 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 508.07 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Synthesis of Compounds <b>1q</b> and <b>1r</b></h3><div class="NLM_p">The mixture of (2-bromothiazol-5-yl)methanol (<b>35</b>) (220 mg, 1.1 mmol), 5-<i>tert</i>-butyl-2-(chloromethyl)oxazole (<b>27</b>) (256 mg, 1.5 mmol), and NaOH (82 mg, 2.04 mmol) in DMF (2 mL) was stirred at 120 Â°C for 3 h. The residue was extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (20 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the residue was purified by silica gel (PE/EA = 0â50%) to obtain compound <b>36</b>.</div><div class="NLM_p">The mixture of <b>36</b> (170 mg, 0.51 mmol), (<i>R</i>)-<i>tert</i>-butyl 3-aminopiperidine-1-carboxylate (164 mg, 0.82 mmol), and DIEA (133 mg, 1 mmol) in NMP (1 mL) was stirred at 140 Â°C overnight. The residue was extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (20 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the residue was purified by silica gel (MeOH/DCM = 0â20%) to obtain compound <b>41a</b>.</div><div class="NLM_p">The mixture of compound <b>38</b> (800 mg, 4.23 mmol), (<i>R</i>)-<i>tert</i>-butyl 3-aminopiperidine-1-carboxylate (1.1 g, 5.5 mmol), and DIEA (820 mg, 6.35 mmol) in THF (15 mL) was stirred at 80 Â°C for 6 h. The residue was extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the residue was purified by silica gel (MeOH/DCM = 0â20%) to obtain compound <b>39</b>.</div><div class="NLM_p">To a solution of <b>39</b> (300 mg, 1 mmol) and CoCl<sub>2</sub>â6H<sub>2</sub>O (232 mg, 1 mmol) in ethanol (8 mL) was added NaBH<sub>4</sub> (110 mg, 3 mmol) portionwise at room temperature. The vial was sealed and stirred at room temperature for 3 h and then quenched with water. The obtained mixture was extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the residue was purified by silica gel (MeOH/DCM = 0â20%) to obtain compound <b>40</b>.</div><div class="NLM_p">The mixture of compound <b>40</b> (160 mg, 0.51 mmol), 5-<i>tert</i>-butyl-2-(chloromethyl)oxazole (<b>27</b>) (89 mg, 0.51 mmol) and DIEA (132 mg, 1 mmol) in NMP (1 mL) was stirred at 80 Â°C overnight. The residue was extracted with chloroform and isopropanol (4:1). The organic phase was washed with brine (50 mL Ã 2) and dried over Na<sub>2</sub>SO<sub>4</sub>. After removal of the solvent, the residue was purified by silica gel (MeOH/DCM = 0â20%) to obtain compound <b>41b</b>.</div><div class="NLM_p">The mixture of compound <b>41</b> (0.07 mmol, 1 equiv), 4-acrylamidobenzoic acid (0.08 mmol, 1.1 equiv), HOBT (0.09 mmol, 1.3 equiv), and EDCI (0.09 mmol, 1.3 equiv) in DMF (0.5 mL) was stirred at room temperature overnight. The reaction mixture was purified by prep-HPLC (MeOH/H<sub>2</sub>O, 0.05% TFA) to provide compounds <b>1q</b> and <b>1r</b>.</div><div id="sec5_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>R</i>)-<i>N</i>-(4-(3-((5-(((5-(<i>tert</i>-Butyl)oxazol-2-yl)methoxy)methyl)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1q</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.38 (s, 1H), 8.51â8.23 (m, 1H), 7.84 (d, <i>J</i> = 8.5 Hz, 2H), 7.75 (d, <i>J</i> = 8.4 Hz, 2H), 7.49â6.91 (m, 1H), 6.74 (s, 1H), 6.46 (dd, <i>J</i> = 17.0, 10.1 Hz, 1H), 6.29 (dd, <i>J</i> = 16.9, 2.1 Hz, 1H), 5.80 (d, <i>J</i> = 10.0 Hz, 1H), 4.43 (s, 2H), 4.17â3.84 (m, 3H), 3.17 (s, 2H), 3.11â2.73 (m, 2H), 2.17â1.42 (m, 4H), 1.25 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 524.03 [M + H]<sup>+</sup>.</div></div><div id="sec5_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>R</i>)-<i>N</i>-(4-(3-((5-((((5-(<i>tert</i>-Butyl)oxazol-2-yl)methyl)amino)methyl)thiazol-2-yl)amino)piperidine-1-carbonyl)phenyl)acrylamide (<b>1r</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) Î´ 10.31 (s, 1H), 8.83â8.60 (m, 2H), 8.09â7.63 (m, 3H), 7.37 (s, 1H), 6.96 (d, <i>J</i> = 10.8 Hz, 1H), 6.78 (d, <i>J</i> = 1.4 Hz, 1H), 6.57â6.36 (m, 1H), 6.35â6.19 (m, 1H), 5.86â5.71 (m, 1H), 4.38â4.27 (m, 4H), 3.79â3.68 (m, 3H), 2.95â2.77 (m, 2H), 2.03â1.85 (m, 2H), 1.69â1.51 (m, 2H), 1.26 (s, 9H). MS (ESI) <i>m</i>/<i>z</i> 523.21 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Cell Culture</h3><div class="NLM_p last">In this study, Jurkat cell lines (A.T. Look Laboratory, DFCI) were grown in RPMI 1640, Glutamax (Life Technologies) supplemented with 10% FBS (Life Technologies). HAP1 cells (Horizon Discovery Group, cat # C859) were grown in IMDM medium supplemented (Life Technologies) with 10% FBS and 100 U/mL penicillin, 100 mg/mL (Life Technologies). All cell lines were cultured at 37 Â°C in a humidified chamber in the presence of 5% CO<sub>2</sub>, unless otherwise noted. All cells were mycoplasma tested upon initial receipt.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Constructs</h3><div class="NLM_p last">For CRISPR studies, guide RNAs (sgRNA) targeting CDK12 and CDK13 were cloned into pX330 (Addgene, 42230). pUC57-AMP was used as storage vector for CDK12 and CDK13 genome reference sequence.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Genome Editing</h3><div class="NLM_p">The CRISPR/Cas9 system was used to mutate the endogenous CDK12 and CDK13 WT loci to encode for CDK12 C1039S and CDK13 C1017S, respectively. Target-specific oligonucleotides were cloned into pX330 (Addgene, 42230), which carries a codon-optimized version of Cas9 and was further modified to express GFP for identifying transfectants. Cells were co-transfected (X-tremeGENE 9 (Roche)) with (1) pX330 expressing Cas9 and CDK12 (or CDK13) targeting sgRNA and (2) a pUC57-AMP construct bearing 1500 bp of modified CDK12 (or CDK13) reference genome that is centered around the CRISPR targeting site in CDK12 (or CDK13). Two days after transfection, cells were sorted using GFP as a marker of transfected cells and cells were replated for 5 days. Cells were then replated at low density to facilitate the isolation of individual clones. Individual clones were isolated, expanded, and PCR genotyped using mutant specific PCR primers. Following initial PCR screening, individual clones were Sanger sequenced to confirm the presence of the desired mutation. To generate a CDK12 C1039S/CDK13 C1017S double mutant clone, mutations were serially performed, sequenced verified, and resubjected to further mutagenesis. Western blot confirmed the presence of intact CDK12 (or CDK13) kinase. Subsequent experiments were conducted using a CDK12 C1039S clone, a CDK13 C1017S clone, a CDK12 C1039S/CDK13 C1017S double mutant clone, and a WT control clone that was carried through the entirety of the CRISPR protocol but that was verified by Sanger sequencing to be WT CDK12 and CDK13.</div><div id="sec5_14_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> For CDK12</h4><div class="NLM_p last">Guide RNA and repair template composition was previously described.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div></div><div id="sec5_14_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> For CDK13</h4><div class="NLM_p">The genomic sequence complementary to the CDK13-directed guide RNA that was cloned into pX330 and used in the genome editing experiments is GGCAAGAT<u class="uu">TGT</u>CATGAGTTA.</div><div class="NLM_p">The modified genomic sequence that was cloned into pUC57-AMP by Genewiz and used as the repair template for genome editing is shown in <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>. (1) Green highlighting indicates the introduced desired TCT mutation, which codes for serine (C1017S), replacing TGT which codes for cysteine (C1039, WT). (2) Yellow highlighting indicates wobble mutations introduced to (a) remove Cas9-targeting PAM sites to prevent cutting of repair template and (b) enable PCR-based screening to permit WT vs mutated allele discrimination. (3) Red highlighting indicates Sal I and EcoRI sites used for pUC57 cloning.</div><figure id="cht1" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0019.jpeg" id="GRAPHIC-d7e3205-autogenerated" class="inline-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Reagents and Antibodies</h3><div class="NLM_p last">The following antibodies were used for immunoblots: Pol II CTD Ser-2 (catalog no. 04-1571), Ser-5 (catalog no. 04-1572), and Ser-7 (catalog no. 04-1570), phospho-antibodies (Millipore); total Pol II (Santa Cruz catalog no. sc-899); CDK12 (Cell Signaling catalog no. 11973); cyclin K (Bethyl catalog no. A301-939A); cyclin H (Bethyl catalog no. A301-674A); RAD51 (Santa Cruz catalog no. sc-8349); RAD51C (Novus Biologicals catalog no. NB100-177); and BRCA2 (Millipore catalog no. OP95) and Î±-tubulin DM1Î± (Sigma catalog no. T9026).</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Immunoblotting</h3><div class="NLM_p last">Whole cell lysates for immunoblotting were prepared by pelleting cells from each cell line at 4 Â°C (1200 rpm) for 5 min using a Sorvall Legend centrifuge (ThermoFisher Scientific). The resulting cell pellets were washed 1Ã with ice-cold 1X PBS and then resuspended in lysis buffer containing 50 mM Tris-HCl, pH 8.0, 150 mM NaCl, 1% NP-40, 0.1% SDS, 5 mM EDTA with protease (Roche catalog no. 04 693 159 001) and phosphatase inhibitors (Roche catalog no. 04 906 837 001). Whole cell lysates were collected and snap-frozen in liquid nitrogen before being stored at â80 Â°C. Protein concentrations were determined by using the Biorad DC protein assay kit (Bio-Rad, no. 500-0111) or Pierce BCA protein assay kit (Life Technologies no. 23225). Whole cell lysates were loaded into Bolt 4â12% or 8% Bis-Tris gels (Invitrogen, Carlsbad, CA) and separated by electrophoresis at 90 V for 2 h. The gels were then transferred onto nitrocellulose membrane (Biorad catalog no. 162-0115) and blocked by incubation with 5% dry milk or BSA in TBST (TBS with 0.2% Tween-20). Membranes were probed using antibodies raised against the indicated proteins. Appropriate IR-labeled secondary antibodies were used, and Licor Odyssey CLx was used for detection.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Pulldown/Immunoprecipitation (IP) Experiments</h3><div class="NLM_p last">Cells were treated with MFH290 or DMSO for 6 h. Following treatment cells were washed 2-fold with cold PBS and then lysed in the following lysis buffer: 50 mM Hepes, pH 7.4, 150 mM NaCl, 1% Nonidet P40 substitute, 5 mM EDTA, 1 mM DTT, and protease/phosphatase cocktails. Following clearance, lysates were treated with bio-THZ1 for pulldown overnight at 4 Â°C. Lysates were further incubated at room temperature for 3 h to increase the efficiency of covalent bond formation. Lysates were then incubated with streptavidin agarose (ThermoFisher Scientific catalog no. 20349)) for pulldown for an additional 2â3 h at 4 Â°C. Agarose beads were washed 6 times with lysis buffer and then boiled in 2Ã SDS page loading buffer for 10 min at 95 Â°C. SDSâPAGE resolved precipitated proteins were probed for the indicated proteins.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Enzymatic Assays</h3><div class="NLM_p last">The enzymatic activities against CDK1, -2, and -5 were tested in Zâ²-Lyte assays with ATP concentrations of <i>K</i><sub>m</sub>. The activities against CDK3, -8, -14, and -16 were tested in LanthaScreen Eu kinase binding assays. The activities against CDK4, -6, -7, and -9 were tested in Adapta assays. The activity against CDK12 and -13 was tested in radioisotope filter binding assay. All the protocols are available from Life Technologies and Reaction Biology.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Kinome Profiling</h3><div class="NLM_p last">Kinome profiling was performed using KinomeScan ScanMAX at a compound concentration of 1 Î¼M. Data were reported in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">Supporting Information</a>. Protocols are available from DiscoverX.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Proliferation Assays</h3><div class="NLM_p last">Proliferation assays were conducted using CellTiter Glo  assay kit (Promega catalog no. G7571). For 72 h studies, HAP1 cells were plated in 96-well plates at 25 000 cells/well in fresh media. Twenty-four hours later, cells were treated with MFH290 at the indicated concentrations or DMSO for 72 h. For 8-day synergy studies, HAP1 cells were seeded in 96-well plates at 5000 cells/well in fresh media. Twenty-four hours later cells were treated with MFH290 Â± olaparib at the indicated concentrations for 8 days. Antiproliferative effects of compounds were assessed using CellTiter Glo as described in the product manual by luminescence measurements on an Envision plate reader.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Mass Spectrometry Analysis</h3><div class="NLM_p">CDK12/cyclin K complex (5 Î¼g) was treated with DMSO or a 10-fold molar excess of MFH-2-90 for 1 h at room temperature and analyzed by LCâMS using an HPLC system (Shimadzu, Marlborough, MA) interfaced to an LTQ ion trap mass spectrometer (ThermoFisher Scientific, San Jose, CA). After injection, proteins were desalted for 4 min on column with 100% A and then eluted with an HPLC gradient (0â100% B in 1 min; A = 0.2 M acetic acid in water; B = 0.2 M acetic acid in acetonitrile). The mass spectrometer was programmed to acquire full scan mass spectra (<i>m</i>/<i>z</i> 300â2000) in profile mode (spray voltage = 4.5 kV). Mass spectra were deconvoluted using MagTran software version 1.03b2.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p last">To identify the site of covalent modification, MFH290 treated CDK/cyclin K complex was reduced with 10 mM tris(2-carboxyethyl)phosphine for 10 min at room temperature, alkylated with 20 mM methyl methanethiosulfonate for 10 min at room temperature, and digested with Glu-C protease (Promega, Madison, WI) overnight at 37 Â°C. Peptides were desalted using C18 (SOLA, ThermoFisher Scientific, Madison, WI), dried by vacuum centrifugation, reconstituted in 50% acetonitrile, 1% formic acid, 100 mM ammonium acetate, and analyzed by CE-MS using a ZipChip CE-MS instrument and autosampler (908 devices, Boston, MA) interfaced to a QExactive HF mass spectrometer (ThermoFisher Scientific). Peptides were resolved at 500 V/cm using an HR chip with a background electrolyte consisting of 50% acetonitrile with 1% formic acid. The mass spectrometer was operated in data dependent mode and subjected the 5 most abundant ions in each MS scan (<i>m</i>/<i>z</i> 300â2000, 60K resolution, 3E6 target, 100 ms max fill time) to MS/MS (15K resolution, 1E5 target, 100 ms max fill time). Dynamic exclusion was enabled with a repeat count of 1 and an exclusion duration of 5 s. Raw mass spectrometry data files were converted to .mgf using Multiplierz software<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and searched against a forwardâreverse human refseq database using Mascot version 2.6.2. Search parameters specified fixed methylthio modification of cysteine, variable methionine oxidation, and variable MFH290 modification of cysteine. MFH290 modified spectra were examined and figures prepared using mzStudio software.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> MFH290 specific product ions were assigned as described.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> RNA Extraction and Synthetic RNA Spike-In</h3><div class="NLM_p last">Total RNA and sample preparation was performed as previously described.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Briefly, HAP1 cells were incubated in media containing MFH290 at the indicated concentrations or with DMSO for the specified duration of time. Cell numbers were determined by manually counting cells using C-chip disposable hemocytometers (Digital Bio, DHC-N01) prior to lysis and RNA extraction. Total RNA from biological replicates (equivalent to 5 million cells per replicate) was subsequently isolated using RNeasy Plus Mini kit (Qiagen, catalog no. 74134) following the manufacturerâs instructions and resuspended in 50 Î¼L of nuclease-free water (Ambion, AM9938).</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> cDNA Preparation and TaqMan Expression Analysis</h3><div class="NLM_p last">RNA utilized for RT-PCR was extracted as outlined above. One microgram of purified RNA was reverse transcribed using Superscript III First-Strand (Invitrogen, catalog no. 18080-051) with oligo dT primers to prime first-strand synthesis according to the manufacturerâs protocol. qPCR was carried out on the 7000 ABI detection system using the following Taqman probes according to the manufacturerâs protocol (Applied Biosystems). All experiments shown were performed in biological triplicate. Each individual biological sample was qPCR-amplified in technical duplicate. Error bars are Â±SD. Expression was normalized to GAPDH, and fold change in expression was calculated relative to the indicated conditions. The following Taqman probes from Life Technologies were used for qPCR-based gene expression analysis: GAPDH, Hs02758991_g1; BRCA2, Hs00609073_m1; RAD51, Hs00947967_m1; RAD51C, Hs00427442_m1.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> <i>In Vivo</i> Pharmacokinetic Studies</h3><div class="NLM_p last">Male Swiss albino mice were dosed with MFH290 solution formulation (intravenous, 5% NMP, 5% Solutol HS-15, 90% normal saline, dose 1 mg/kg) or (intraperitoneal 5% NMP, 5% Solutol, 90% normal saline, dose 5 mg/kg) or (oral, 5% NMP, 5% Solutol HS-15, 90% normal saline, dose 10 mg/kg). Blood samples were collected at predose, 0.08, 0.25, 0.5, 1, 2, 4, 8, and 24 h (iv and ip) and predose, 0.25, 0.5, 1, 2, 4, 6, 8, and 24 h (po). The blood samples were collected from sets of three mice at each time point in labeled microcentrifuge tubes containing K2EDTA as an anticoagulant. Plasma samples were separated by centrifugation and stored below â70 Â°C until bioanalysis. All samples were processed for analysis by precipitation using acetonitrile and analyzed with a partially validated LC/MS/MS method (LLOQ, 1.22 ng/mL for iv and po; LLOQ, 5.02 ng/mL for ip). Pharmacokinetic parameters were calculated using the noncompartmental analysis tool of WinNonlin Enterprise software (version 6.3).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01929" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01929?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01929</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Schematic of target engagement experiment (Figure S1A); known CDK inhibitors screened by target engagement assay (Figure S1B); structureâactivity relationship (SAR) screen of covalent compounds using target engagement assay (Figure S2); MFH290 targets CDK12 and CDK13 in cells (Figure S3); CRISPR technique mutates Cys-1039 to serine (C1039S) at CDK12 loci and Cys-1017 to serine (C1017S) at CDK13 loci (Figure S4A); Western blot of CDK12 and CDK13 WT and Cys-to-Ser mutants in HAP1 cells (Figure S4B); KinomeScan profiling data for MFH290 (Table S1); <sup>1</sup>H NMR spectrum of compound MFH290 (Figure S5A); HPLC spectrum of compound MFH290 (Figure S5B); LCâMS spectrum of compound MFH290 (Figure S5C) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Cocrystal structure of SNS032/CDK2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D1J">5D1J</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Crystal structure of CDK12 transformed into CDK2 structure coordinate frame (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACB">5ACB</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_004.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf">jm9b01929_si_001.pdf (674.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_002.csv">jm9b01929_si_002.csv (6.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_003.pdb">jm9b01929_si_003.pdb (182.1 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_004.pdb">jm9b01929_si_004.pdb (725.74 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01929" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11263" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11263" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas P. Kwiatkowski</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d19fb8b2b9bebdb0a2818e9aa6b8b0a5babea6a2bab891b5b7b2b8ffb9b0a3a7b0a3b5ffb4b5a4"><span class="__cf_email__" data-cfemail="307e5953585f5c5143606f7b475951445b5f47435b5970545653591e585142465142541e555445">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tinghu Zhang</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#63370a0d040b163c390b020d04230705000a4d0b0211150211074d060716"><span class="__cf_email__" data-cfemail="85d1ecebe2edf0dadfede4ebe2c5e1e3e6ecabede4f7f3e4f7e1abe0e1f0">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nathanael S. Gray</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5354-7403" title="Orcid link">http://orcid.org/0000-0001-5354-7403</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#216f405549404f40444d7e6653405861454742480f494053574053450f444554"><span class="__cf_email__" data-cfemail="4b052a3f232a252a2e27140c392a320b2f2d282265232a393d2a392f652e2f3e">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yao Liu</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingfeng Hao</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan L. Leggett</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yang Gao</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott B. Ficarro</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Blais
Proteomics Center, Dana Farber Cancer Institute, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianwei Che</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhixiang He</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Calla M. Olson</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jarrod A. Marto</span> - <span class="hlFld-Affiliation affiliation">Department of Biological Chemistry and
Molecular Pharmacology, Harvard Medical
School, Boston, Massachusetts 02115, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Blais
Proteomics Center, Dana Farber Cancer Institute, Harvard Medical School, Boston, Massachusetts 02215, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.L. and M.H. contributed equally. Author contributions are the following: conceptualization, N.S.G., T.Z.; chemistry, Y.L., M.H., Z.H.; biology, A.L.L., N.P.K. Y.G., C.M.O.; computational, J.C.; mass spectrum labeling, S.B.F., J.A.M.; writings, T.Z., Y.L., N.P.K., N.S.G. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): N.S.G. is a founder, SAB and equity holder in Gatekeeper, Syros, Petra, C4, B2S and Soltego. J.C. is a consultant for Soltego and equity holder of M3 bioinformatics & technology Inc; N.P.K. is a consultant to Epiphanes, Inc. The Gray lab receives/has received funding from Novartis, Takeda, Astellas, Taiho, Janssen, Kinogen, Voronoi, Her2llc, Deerfield, Epiphanes, and Sanofi. T.Z, N.S.G, N.P.K. and M.H. are inventors on MFH290 patent.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3295-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50284" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50284" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Jim Sun at the NMR facility of Dana-Farber Cancer Institute for his assistance on NMR data collection. Milka Kostic is greatly acknowledged for the editing and proofreading. This work was supported by Susan Smith Center for Womenâs CancerâPPG Award, NIH grant 1RO1 CA179483-01A1, and Dana-Farber Cancer Institute Accelerator Grant.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">BRCA</td><td class="NLM_def"><p class="first last">breast cancer gene</p></td></tr><tr><td class="NLM_term">PDX</td><td class="NLM_def"><p class="first last">patient-derived xenograft</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate, hexafluorophosphate azabenzotriazole tetramethyluronium</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyloxycarbonyl</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">m-CPBA</td><td class="NLM_def"><p class="first last">meta-chloroperbenzoic acid</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl-2-pyrrolidone</p></td></tr><tr><td class="NLM_term">HCl</td><td class="NLM_def"><p class="first last">hydrochloric acid</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">Py</td><td class="NLM_def"><p class="first last">pyridine</p></td></tr><tr><td class="NLM_term">T<sub>3</sub>P</td><td class="NLM_def"><p class="first last">propylphosphonic anhydride</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">KSCN</td><td class="NLM_def"><p class="first last">potassium thiocyanate</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOK</td><td class="NLM_def"><p class="first last">potassium <i>tert</i>-butoxide</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuONO</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl nitrite</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild type</p></td></tr><tr><td class="NLM_term">T-ALL</td><td class="NLM_def"><p class="first last">T-acute lymphoblastic leukemia</p></td></tr><tr><td class="NLM_term">CML</td><td class="NLM_def"><p class="first last">chronic myelogenous leukemia</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 28 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">Cell Cycle, CDKs and Cancer: a Changing Paradigm</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">153</span>â <span class="NLM_lpage">166</span>, <span class="refDoi">Â DOI: 10.1038/nrc2602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnrc2602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=19238148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1ans78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=153-166&author=M.+Malumbresauthor=M.+Barbacid&title=Cell+Cycle%2C+CDKs+and+Cancer%3A+a+Changing+Paradigm&doi=10.1038%2Fnrc2602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle, CDKs and cancer: a changing paradigm</span></div><div class="casAuthors">Malumbres, Marcos; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">153-166</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Tumor-assocd. cell cycle defects are often mediated by alterations in cyclin-dependent kinase (CDK) activity.  Misregulated CDKs induce unscheduled proliferation as well as genomic and chromosomal instability.  According to current models, mammalian CDKs are essential for driving each cell cycle phase, so therapeutic strategies that block CDK activity are unlikely to selectively target tumor cells.  However, recent genetic evidence has revealed that, whereas CDK1 is required for the cell cycle, interphase CDKs are only essential for proliferation of specialized cells.  Emerging evidence suggests that tumor cells may also require specific interphase CDKs for proliferation.  Thus, selective CDK inhibition may provide therapeutic benefit against certain human neoplasias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzlIKlbCPWnLVg90H21EOLACvtfcHk0lgRJMXBrryTFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1ans78%253D&md5=2a7eb34b2381e288b3f43ec1bffc0aec</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2602%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DCell%2520Cycle%252C%2520CDKs%2520and%2520Cancer%253A%2520a%2520Changing%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D153%26epage%3D166%26doi%3D10.1038%2Fnrc2602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hydbring, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span> <span> </span><span class="NLM_article-title">Non-Canonical Functions of Cell Cycle Cyclins and Cyclin-Dependent Kinases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">280</span>â <span class="NLM_lpage">292</span>, <span class="refDoi">Â DOI: 10.1038/nrm.2016.27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnrm.2016.27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=27033256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOksLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=280-292&author=P.+Hydbringauthor=M.+Malumbresauthor=P.+Sicinski&title=Non-Canonical+Functions+of+Cell+Cycle+Cyclins+and+Cyclin-Dependent+Kinases&doi=10.1038%2Fnrm.2016.27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Non-canonical functions of cell cycle cyclins and cyclin-dependent kinases</span></div><div class="casAuthors">Hydbring, Per; Malumbres, Marcos; Sicinski, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">280-292</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The roles of cyclins and their catalytic partners, the cyclin-dependent kinases (CDKs), as core components of the machinery that drives cell cycle progression are well established.  Increasing evidence indicates that mammalian cyclins and CDKs also carry out important functions in other cellular processes, such as transcription, DNA damage repair, control of cell death, differentiation, the immune response and metab.  Some of these non-canonical functions are performed by cyclins or CDKs, independently of their resp. cell cycle partners, suggesting that there was a substantial divergence in the functions of these proteins during evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqltFQjJuMXrbVg90H21EOLACvtfcHk0lgRJMXBrryTFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOksLw%253D&md5=4d8aaf10c0542828112c561f5311f7da</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.27%26sid%3Dliteratum%253Aachs%26aulast%3DHydbring%26aufirst%3DP.%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DSicinski%26aufirst%3DP.%26atitle%3DNon-Canonical%2520Functions%2520of%2520Cell%2520Cycle%2520Cyclins%2520and%2520Cyclin-Dependent%2520Kinases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D17%26spage%3D280%26epage%3D292%26doi%3D10.1038%2Fnrm.2016.27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asghar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span> <span> </span><span class="NLM_article-title">The History and Future of Targeting Cyclin-Dependent Kinases in Cancer Therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">130</span>â <span class="NLM_lpage">146</span>, <span class="refDoi">Â DOI: 10.1038/nrd4504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnrd4504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=25633797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=130-146&author=U.+Asgharauthor=A.+K.+Witkiewiczauthor=N.+C.+Turnerauthor=E.+S.+Knudsen&title=The+History+and+Future+of+Targeting+Cyclin-Dependent+Kinases+in+Cancer+Therapy&doi=10.1038%2Fnrd4504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The history and future of targeting cyclin-dependent kinases in cancer therapy</span></div><div class="casAuthors">Asghar, Uzma; Witkiewicz, Agnieszka K.; Turner, Nicholas C.; Knudsen, Erik S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-146</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer represents a pathol. manifestation of uncontrolled cell division; therefore, it has long been anticipated that our understanding of the basic principles of cell cycle control would result in effective cancer therapies.  In particular, cyclin-dependent kinases (CDKs) that promote transition through the cell cycle were expected to be key therapeutic targets because many tumorigenic events ultimately drive proliferation by impinging on CDK4 or CDK6 complexes in the G1 phase of the cell cycle.  Moreover, perturbations in chromosomal stability and aspects of S phase and G2/M control mediated by CDK2 and CDK1 are pivotal tumorigenic events.  Translating this knowledge into successful clin. development of CDK inhibitors has historically been challenging, and numerous CDK inhibitors have demonstrated disappointing results in clin. trials.  Here, we review the biol. of CDKs, the rationale for therapeutically targeting discrete kinase complexes and historical clin. results of CDK inhibitors.  We also discuss how CDK inhibitors with high selectivity (particularly for both CDK4 and CDK6), in combination with patient stratification, have resulted in more substantial clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_utd-yiEZLVg90H21EOLACvtfcHk0lgRJMXBrryTFQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D&md5=113bcd2e99a8267119d180670d3c7761</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd4504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4504%26sid%3Dliteratum%253Aachs%26aulast%3DAsghar%26aufirst%3DU.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26atitle%3DThe%2520History%2520and%2520Future%2520of%2520Targeting%2520Cyclin-Dependent%2520Kinases%2520in%2520Cancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D130%26epage%3D146%26doi%3D10.1038%2Fnrd4504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amzallag, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannett, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Regulation in Cancer with a Covalent CDK7 Inhibitor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>511</i></span>,  <span class="NLM_fpage">616</span>â <span class="NLM_lpage">620</span>, <span class="refDoi">Â DOI: 10.1038/nature13393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnature13393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=25043025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=616-620&author=N.+Kwiatkowskiauthor=T.+Zhangauthor=P.+B.+Rahlauthor=B.+J.+Abrahamauthor=J.+Reddyauthor=S.+B.+Ficarroauthor=A.+Dasturauthor=A.+Amzallagauthor=S.+Ramaswamyauthor=B.+Tesarauthor=C.+E.+Jenkinsauthor=N.+M.+Hannettauthor=D.+McMillinauthor=T.+Sandaauthor=T.+Simauthor=N.+D.+Kimauthor=T.+Lookauthor=C.+S.+Mitsiadesauthor=A.+P.+Wengauthor=J.+R.+Brownauthor=C.+H.+Benesauthor=J.+A.+Martoauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Targeting+Transcription+Regulation+in+Cancer+with+a+Covalent+CDK7+Inhibitor&doi=10.1038%2Fnature13393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span></div><div class="casAuthors">Kwiatkowski, Nicholas; Zhang, Tinghu; Rahl, Peter B.; Abraham, Brian J.; Reddy, Jessica; Ficarro, Scott B.; Dastur, Anahita; Amzallag, Arnaud; Ramaswamy, Sridhar; Tesar, Bethany; Jenkins, Catherine E.; Hannett, Nancy M.; McMillin, Douglas; Sanda, Takaomi; Sim, Taebo; Kim, Nam Doo; Look, Thomas; Mitsiades, Constantine S.; Weng, Andrew P.; Brown, Jennifer R.; Benes, Cyril H.; Marto, Jarrod A.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7511</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor oncogenes include transcription factors that co-opt the general transcriptional machinery to sustain the oncogenic state, but direct pharmacol. inhibition of transcription factors has so far proven difficult.  However, the transcriptional machinery contains various enzymic cofactors that can be targeted for the development of new therapeutic candidates, including cyclin-dependent kinases (CDKs).  Here the authors present the discovery and characterization of a covalent CDK7 inhibitor, THZ1, which has the unprecedented ability to target a remote cysteine residue located outside of the canonical kinase domain, providing an unanticipated means of achieving selectivity for CDK7.  Cancer cell-line profiling indicates that a subset of cancer cell lines, including human T-cell acute lymphoblastic leukemia (T-ALL), have exceptional sensitivity to THZ1.  Genome-wide anal. in Jurkat T-ALL cells shows that THZ1 disproportionally affects transcription of RUNX1 and suggests that sensitivity to THZ1 may be due to vulnerability conferred by the RUNX1 super-enhancer and the key role of RUNX1 in the core transcriptional regulatory circuitry of these tumor cells.  Pharmacol. modulation of CDK7 kinase activity may thus provide an approach to identify and treat tumor types that are dependent on transcription for maintenance of the oncogenic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVD-DeTAQCqrVg90H21EOLACvtfcHk0ljEblejkGCHJA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF&md5=895a28fb536663a31e7558fe79ce628b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature13393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13393%26sid%3Dliteratum%253Aachs%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DReddy%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DAmzallag%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DTesar%26aufirst%3DB.%26aulast%3DJenkins%26aufirst%3DC.%2BE.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DMcMillin%26aufirst%3DD.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLook%26aufirst%3DT.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWeng%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520Transcription%2520Regulation%2520in%2520Cancer%2520with%2520a%2520Covalent%2520CDK7%2520Inhibitor%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature13393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon-Clarke, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greifenberg, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannett, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkowiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Covalent Targeting of Remote Cysteine Residues to Develop CDK12 and CDK13 Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">876</span>â <span class="NLM_lpage">884</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnchembio.2166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=27571479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=876-884&author=T.+Zhangauthor=N.+Kwiatkowskiauthor=C.+M.+Olsonauthor=S.+E.+Dixon-Clarkeauthor=B.+J.+Abrahamauthor=A.+K.+Greifenbergauthor=S.+B.+Ficarroauthor=J.+M.+Elkinsauthor=Y.+Liangauthor=N.+M.+Hannettauthor=T.+Manzauthor=M.+Haoauthor=B.+Bartkowiakauthor=A.+L.+Greenleafauthor=J.+A.+Martoauthor=M.+Geyerauthor=A.+N.+Bullockauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Covalent+Targeting+of+Remote+Cysteine+Residues+to+Develop+CDK12+and+CDK13+Inhibitors&doi=10.1038%2Fnchembio.2166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent targeting of remote cysteine residues to develop CDK12 and CDK13 inhibitors</span></div><div class="casAuthors">Zhang, Tinghu; Kwiatkowski, Nicholas; Olson, Calla M.; Dixon-Clarke, Sarah E.; Abraham, Brian J.; Greifenberg, Ann K.; Ficarro, Scott B.; Elkins, Jonathan M.; Liang, Yanke; Hannett, Nancy M.; Manz, Theresa; Hao, Mingfeng; Bartkowiak, Bartlomiej; Greenleaf, Arno L.; Marto, Jarrod A.; Geyer, Matthias; Bullock, Alex N.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">876-884</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 12 and 13 (CDK12 and CDK13) play crit. roles in the regulation of gene transcription.  However, the absence of CDK12 and CDK13 inhibitors has hindered the ability to investigate the consequences of their inhibition in healthy cells and cancer cells.  Here we describe the rational design of a first-in-class CDK12 and CDK13 covalent inhibitor, THZ531.  Co-crystn. of THZ531 with CDK12-cyclin K indicates that THZ531 irreversibly targets a cysteine located outside the kinase domain.  THZ531 causes a loss of gene expression with concurrent loss of elongating and hyperphosphorylated RNA polymerase II.  In particular, THZ531 substantially decreases the expression of DNA damage response genes and key super-enhancer-assocd. transcription factor genes.  Coincident with transcriptional perturbation, THZ531 dramatically induced apoptotic cell death.  Small mols. capable of specifically targeting CDK12 and CDK13 may thus help identify cancer subtypes that are particularly dependent on their kinase activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXIyvy8vbjvrVg90H21EOLACvtfcHk0ljEblejkGCHJA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ&md5=5ac265105d1654e129c3c1325572f389</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2166%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DDixon-Clarke%26aufirst%3DS.%2BE.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DManz%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DCovalent%2520Targeting%2520of%2520Remote%2520Cysteine%2520Residues%2520to%2520Develop%2520CDK12%2520and%2520CDK13%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D876%26epage%3D884%26doi%3D10.1038%2Fnchembio.2166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzyk, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichu, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollett, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, G. B.</span></span> <span> </span><span class="NLM_article-title">Interaction of Cyclin-Dependent Kinase 12/CrkRS with Cyclin K1 Is Required for the Phosphorylation of the C-terminal Domain of RNA Polymerase II</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">4691</span>â <span class="NLM_lpage">4704</span>, <span class="refDoi">Â DOI: 10.1128/MCB.06267-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1128%2FMCB.06267-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=22988298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1CjurrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=4691-4704&author=S.+W.+Chengauthor=M.+A.+Kuzykauthor=A.+Moradianauthor=T.+A.+Ichuauthor=V.+C.+Changauthor=J.+F.+Tienauthor=S.+E.+Vollettauthor=M.+Griffithauthor=M.+A.+Marraauthor=G.+B.+Morin&title=Interaction+of+Cyclin-Dependent+Kinase+12%2FCrkRS+with+Cyclin+K1+Is+Required+for+the+Phosphorylation+of+the+C-terminal+Domain+of+RNA+Polymerase+II&doi=10.1128%2FMCB.06267-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of cyclin-dependent kinase 12/CrkRS with cyclin K1 is required for the phosphorylation of the C-terminal domain of RNA polymerase II</span></div><div class="casAuthors">Cheng, S.-W. Grace; Kuzyk, Michael A.; Moradian, Annie; Ichu, Taka-Aki; Chang, Vicky C.-D.; Tien, Jerry F.; Vollett, Sarah E.; Griffith, Malachi; Marra, Marco A.; Morin, Gregg B.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4691-4704</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">CrkRS (Cdc2-related kinase, Arg/Ser), or cyclin-dependent kinase 12 (CKD12), is a serine/threonine kinase believed to coordinate transcription and RNA splicing.  While CDK12/CrkRS complexes were known to phosphorylate the C-terminal domain (CTD) of RNA polymerase II (RNA Pol II), the cyclin regulating this activity was not known.  Using immunopptn. and mass spectrometry, we identified a 65-kDa isoform of cyclin K (cyclin K1) in endogenous CDK12/CrkRS protein complexes.  We show that cyclin K1 complexes isolated from mammalian cells contain CDK12/CrkRS but do not contain CDK9, a presumed partner of cyclin K.  Anal. of extensive RNA-Seq data shows that the 65-kDa cyclin K1 isoform is the predominantly expressed form across numerous tissue types.  We also demonstrate that CDK12/CrkRS is dependent on cyclin K1 for its kinase activity and that small interfering RNA (siRNA) knockdown of CDK12/CrkRS or cyclin K1 has similar effects on the expression of a luciferase reporter gene.  Our data suggest that cyclin K1 is the primary cyclin partner for CDK12/CrkRS and that cyclin K1 is required to activate CDK12/CrkRS to phosphorylate the CTD of RNA Pol II.  These properties are consistent with a role of CDK12/CrkRS in regulating gene expression through phosphorylation of RNA Pol II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolvJU3IDa9NrVg90H21EOLACvtfcHk0ljEblejkGCHJA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1CjurrF&md5=4b90352053b987a1857bf2e54e1e231a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FMCB.06267-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06267-11%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DS.%2BW.%26aulast%3DKuzyk%26aufirst%3DM.%2BA.%26aulast%3DMoradian%26aufirst%3DA.%26aulast%3DIchu%26aufirst%3DT.%2BA.%26aulast%3DChang%26aufirst%3DV.%2BC.%26aulast%3DTien%26aufirst%3DJ.%2BF.%26aulast%3DVollett%26aufirst%3DS.%2BE.%26aulast%3DGriffith%26aufirst%3DM.%26aulast%3DMarra%26aufirst%3DM.%2BA.%26aulast%3DMorin%26aufirst%3DG.%2BB.%26atitle%3DInteraction%2520of%2520Cyclin-Dependent%2520Kinase%252012%252FCrkRS%2520with%2520Cyclin%2520K1%2520Is%2520Required%2520for%2520the%2520Phosphorylation%2520of%2520the%2520C-terminal%2520Domain%2520of%2520RNA%2520Polymerase%2520II%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D4691%26epage%3D4704%26doi%3D10.1128%2FMCB.06267-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntoon, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karnitz, L. M.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer-Associated Mutations Disable Catalytic Activity of CDK12, a Kinase That Promotes Homologous Recombination Repair and Resistance to Cisplatin and Poly (ADP-ribose) polymerase Inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">9247</span>â <span class="NLM_lpage">9253</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M114.551143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1074%2Fjbc.M114.551143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=24554720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVSksrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9247-9253&author=P.+M.+Joshiauthor=S.+L.+Sutorauthor=C.+J.+Huntoonauthor=L.+M.+Karnitz&title=Ovarian+Cancer-Associated+Mutations+Disable+Catalytic+Activity+of+CDK12%2C+a+Kinase+That+Promotes+Homologous+Recombination+Repair+and+Resistance+to+Cisplatin+and+Poly+%28ADP-ribose%29+polymerase+Inhibitors&doi=10.1074%2Fjbc.M114.551143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian Cancer-associated Mutations Disable Catalytic Activity of CDK12, a Kinase That Promotes Homologous Recombination Repair and Resistance to Cisplatin and Poly(ADP-ribose) Polymerase Inhibitors</span></div><div class="casAuthors">Joshi, Poorval M.; Sutor, Shari L.; Huntoon, Catherine J.; Karnitz, Larry M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9247-9253</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mutations in the tumor suppressors BRCA1 and BRCA2, which encode proteins that are key participants in homologous recombination (HR) repair, occur in â¼20% of high grade serous ovarian cancers.  Although only 20% of these tumors have mutations in BRCA1 and BRCA2, nearly 50% of these tumors have defects in HR.  Notably, however, the underlying genetic defects that give rise to HR defects in the absence of BRCA1 and BRCA2 mutations have not been fully elucidated.  Here we show that the recurrent somatic CDK12 mutations identified in ovarian cancers impair the catalytic activity of this kinase, which is involved in the transcription of a subset of genes, including BRCA1 and other DNA repair genes.  Furthermore, we show that disabling CDK12 function in ovarian cancer cells reduces BRCA1 levels, disrupts HR repair, and sensitizes these cells to the crosslinking agents melphalan and cisplatin and to the poly(ADP-ribose) polymerase (PARP) inhibitor veliparib (ABT-888).  Taken together, these findings suggest that many CDK12 mutations are an unrecognized cause of HR defects in ovarian cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTj3CX_FsX_rVg90H21EOLACvtfcHk0lgowOyf0yB0TA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVSksrg%253D&md5=1ac9807318b6002c6134ef49751f3145</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.551143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.551143%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DP.%2BM.%26aulast%3DSutor%26aufirst%3DS.%2BL.%26aulast%3DHuntoon%26aufirst%3DC.%2BJ.%26aulast%3DKarnitz%26aufirst%3DL.%2BM.%26atitle%3DOvarian%2520Cancer-Associated%2520Mutations%2520Disable%2520Catalytic%2520Activity%2520of%2520CDK12%252C%2520a%2520Kinase%2520That%2520Promotes%2520Homologous%2520Recombination%2520Repair%2520and%2520Resistance%2520to%2520Cisplatin%2520and%2520Poly%2520%2528ADP-ribose%2529%2520polymerase%2520Inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D9247%26epage%3D9253%26doi%3D10.1074%2Fjbc.M114.551143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartkowiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, A. L.</span></span> <span> </span><span class="NLM_article-title">Engineering an Analog-Sensitive CDK12 Cell Line Using CRISPR/Cas</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1849</i></span>,  <span class="NLM_fpage">1179</span>â <span class="NLM_lpage">1187</span>, <span class="refDoi">Â DOI: 10.1016/j.bbagrm.2015.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.bbagrm.2015.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=26189575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1849&publication_year=2015&pages=1179-1187&author=B.+Bartkowiakauthor=C.+Yanauthor=A.+L.+Greenleaf&title=Engineering+an+Analog-Sensitive+CDK12+Cell+Line+Using+CRISPR%2FCas&doi=10.1016%2Fj.bbagrm.2015.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering an analog-sensitive CDK12 cell line using CRISPR/Cas</span></div><div class="casAuthors">Bartkowiak, Bartlomiej; Yan, Christopher; Greenleaf, Arno L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1849</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1179-1187</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The RNA Polymerase II C-terminal domain (CTD) kinase CDK12 has been implicated as a tumor suppressor and regulator of DNA damage response genes.  Although much has been learned about CDK12 and its activity, due to the lack of a specific inhibitor and the complications posed by long term RNAi depletion, much is still unknown about the particulars of CDK12 function.  Therefore gaining a better understanding of CDK12's roles at the mol. level will be challenging without the development of addnl. tools.  In order to address these issues we have used the CRISPR/Cas gene engineering system to create a mammalian cell line in which the only functional copy of CDK12 is selectively inhibitable by a cell-permeable adenine analog (analog-sensitive CDK12).  Inhibition of CDK12 results in a perturbation of the phosphorylation patterns on the CTD and an arrest in cellular proliferation.  This cell line should serve as a powerful tool for future studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWQ577ekAl_bVg90H21EOLACvtfcHk0lgowOyf0yB0TA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3I&md5=297f75a0a71c579a01811cc387fc78a6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2015.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2015.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26atitle%3DEngineering%2520an%2520Analog-Sensitive%2520CDK12%2520Cell%2520Line%2520Using%2520CRISPR%252FCas%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2015%26volume%3D1849%26spage%3D1179%26epage%3D1187%26doi%3D10.1016%2Fj.bbagrm.2015.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajrami, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankum, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafiq, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozarewa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assiotis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenwick, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natrajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Genome-Wide Profiling of Genetic Synthetic Lethality Identifies CDK12 as a Novel Determinant of PARP1/2 Inhibitor Sensitivity</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">287</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-2541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1158%2F0008-5472.CAN-13-2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=24240700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislKjuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=287-297&author=I.+Bajramiauthor=J.+R.+Frankumauthor=A.+Kondeauthor=R.+E.+Millerauthor=F.+L.+Rehmanauthor=R.+Broughauthor=J.+Campbellauthor=D.+Simsauthor=R.+Rafiqauthor=S.+Hooperauthor=L.+Chenauthor=I.+Kozarewaauthor=I.+Assiotisauthor=K.+Fenwickauthor=R.+Natrajanauthor=C.+J.+Lordauthor=A.+Ashworth&title=Genome-Wide+Profiling+of+Genetic+Synthetic+Lethality+Identifies+CDK12+as+a+Novel+Determinant+of+PARP1%2F2+Inhibitor+Sensitivity&doi=10.1158%2F0008-5472.CAN-13-2541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide Profiling of Genetic Synthetic Lethality Identifies CDK12 as a Novel Determinant of PARP1/2 Inhibitor Sensitivity</span></div><div class="casAuthors">Bajrami, Ilirjana; Frankum, Jessica R.; Konde, Asha; Miller, Rowan E.; Rehman, Farah L.; Brough, Rachel; Campbell, James; Sims, David; Rafiq, Rumana; Hooper, Sean; Chen, Lina; Kozarewa, Iwanka; Assiotis, Ioannis; Fenwick, Kerry; Natrajan, Rachael; Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of PARP1/2, such as olaparib, have been proposed to serve as a synthetic lethal therapy for cancers that harbor BRCA1 or BRCA2 mutations.  Indeed, in clin. trials, PARP1/2 inhibitors elicit sustained antitumor responses in patients with germline BRCA gene mutations.  In hypothesizing that addnl. genetic determinants might direct use of these drugs, we conducted a genome-wide synthetic lethal screen for candidate olaparib sensitivity genes.  In support of this hypothesis, the set of identified genes included known determinants of olaparib sensitivity, such as BRCA1, RAD51, and Fanconi's anemia susceptibility genes.  In addn., the set included genes implicated in established networks of DNA repair, DNA cohesion, and chromatin remodeling, none of which were known previously to confer sensitivity to PARP1/2 inhibition.  Notably, integration of the list of candidate sensitivity genes with data from tumor DNA sequencing studies identified CDK12 deficiency as a clin. relevant biomarker of PARP1/2 inhibitor sensitivity.  In models of high-grade serous ovarian cancer (HGS-OVCa), CDK12 attenuation was sufficient to confer sensitivity to PARP1/2 inhibition, suppression of DNA repair via homologous recombination, and reduced expression of BRCA1.  As one of only nine genes known to be significantly mutated in HGS-OVCa, CDK12 has properties that should confirm interest in its use as a biomarker, particularly in ongoing clin. trials of PARP1/2 inhibitors and other agents that trigger replication fork arrest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNCJR_Nh_WjbVg90H21EOLACvtfcHk0lgowOyf0yB0TA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislKjuw%253D%253D&md5=9e07aa538aed0fb3a5ac6be21659d051</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-2541%26sid%3Dliteratum%253Aachs%26aulast%3DBajrami%26aufirst%3DI.%26aulast%3DFrankum%26aufirst%3DJ.%2BR.%26aulast%3DKonde%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DR.%2BE.%26aulast%3DRehman%26aufirst%3DF.%2BL.%26aulast%3DBrough%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DD.%26aulast%3DRafiq%26aufirst%3DR.%26aulast%3DHooper%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DKozarewa%26aufirst%3DI.%26aulast%3DAssiotis%26aufirst%3DI.%26aulast%3DFenwick%26aufirst%3DK.%26aulast%3DNatrajan%26aufirst%3DR.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DGenome-Wide%2520Profiling%2520of%2520Genetic%2520Synthetic%2520Lethality%2520Identifies%2520CDK12%2520as%2520a%2520Novel%2520Determinant%2520of%2520PARP1%252F2%2520Inhibitor%2520Sensitivity%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D287%26epage%3D297%26doi%3D10.1158%2F0008-5472.CAN-13-2541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greifenberg, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dust, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primack, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazaro, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochupurakkal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unitt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarosiek, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaltriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juric, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DâAndrea, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BalmaÃ±a, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span> <span> </span><span class="NLM_article-title">CDK12 Inhibition Reverses De Novo and Acquired PARP Inhibitor Resistance in BRCA Wild-Type and Mutated Models of Triple-Negative Breast Cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2367</span>â <span class="NLM_lpage">2381</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2016.10.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.celrep.2016.10.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=27880910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGqt7%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=2367-2381&author=S.+F.+Johnsonauthor=C.+Cruzauthor=A.+K.+Greifenbergauthor=S.+Dustauthor=D.+G.+Stoverauthor=D.+Chiauthor=B.+Primackauthor=S.+Caoauthor=A.+J.+Bernhardyauthor=R.+Coulsonauthor=J.-B.+Lazaroauthor=B.+Kochupurakkalauthor=H.+Sunauthor=C.+Unittauthor=L.+A.+Moreauauthor=K.+A.+Sarosiekauthor=M.+Scaltritiauthor=D.+Juricauthor=J.+Baselgaauthor=A.+L.+Richardsonauthor=S.+J.+Rodigauthor=A.+D.+D%E2%80%99Andreaauthor=J.+Balma%C3%B1aauthor=N.+Johnsonauthor=M.+Geyerauthor=V.+Serraauthor=E.+Limauthor=G.+I.+Shapiro&title=CDK12+Inhibition+Reverses+De+Novo+and+Acquired+PARP+Inhibitor+Resistance+in+BRCA+Wild-Type+and+Mutated+Models+of+Triple-Negative+Breast+Cancer&doi=10.1016%2Fj.celrep.2016.10.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">CDK12 Inhibition Reverses De Novo and Acquired PARP Inhibitor Resistance in BRCA Wild-Type and Mutated Models of Triple-Negative Breast Cancer</span></div><div class="casAuthors">Johnson, Shawn F.; Cruz, Cristina; Greifenberg, Ann Katrin; Dust, Sofia; Stover, Daniel G.; Chi, David; Primack, Benjamin; Cao, Shiliang; Bernhardy, Andrea J.; Coulson, Rhiannon; Lazaro, Jean-Bernard; Kochupurakkal, Bose; Sun, Heather; Unitt, Christine; Moreau, Lisa A.; Sarosiek, Kristopher A.; Scaltriti, Maurizio; Juric, Dejan; Baselga, Jose; Richardson, Andrea L.; Rodig, Scott J.; D'Andrea, Alan D.; Balmana, Judith; Johnson, Neil; Geyer, Matthias; Serra, Violeta; Lim, Elgene; Shapiro, Geoffrey I.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2367-2381</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Although poly(ADP-ribose) polymerase (PARP) inhibitors are active in homologous recombination (HR)-deficient cancers, their utility is limited by acquired resistance after restoration of HR.  Here, we report that dinaciclib, an inhibitor of cyclin-dependent kinases (CDKs) 1, 2, 5, and 9, addnl. has potent activity against CDK12, a transcriptional regulator of HR.  In BRCA-mutated triple-neg. breast cancer (TNBC) cells and patient-derived xenografts (PDXs), dinaciclib ablates restored HR and reverses PARP inhibitor resistance.  Addnl., we show that de novo resistance to PARP inhibition in BRCA1-mutated cell lines and a PDX derived from a PARP-inhibitor-naive BRCA1 carrier is mediated by residual HR and is reversed by CDK12 inhibition.  Finally, dinaciclib augments the degree of response in a PARP-inhibitor-sensitive model, converting tumor growth inhibition to durable regression.  These results highlight the significance of HR disruption as a therapeutic strategy and support the broad use of combined CDK12 and PARP inhibition in TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyVKyKNc3G1bVg90H21EOLACvtfcHk0lh0LYX9uv4_wA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGqt7%252FL&md5=ae666e75b7a49d0c348789e2d59643fa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.10.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.10.077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BF.%26aulast%3DCruz%26aufirst%3DC.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DDust%26aufirst%3DS.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DChi%26aufirst%3DD.%26aulast%3DPrimack%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DS.%26aulast%3DBernhardy%26aufirst%3DA.%2BJ.%26aulast%3DCoulson%26aufirst%3DR.%26aulast%3DLazaro%26aufirst%3DJ.-B.%26aulast%3DKochupurakkal%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DUnitt%26aufirst%3DC.%26aulast%3DMoreau%26aufirst%3DL.%2BA.%26aulast%3DSarosiek%26aufirst%3DK.%2BA.%26aulast%3DScaltriti%26aufirst%3DM.%26aulast%3DJuric%26aufirst%3DD.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DA.%2BL.%26aulast%3DRodig%26aufirst%3DS.%2BJ.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DA.%2BD.%26aulast%3DBalma%25C3%25B1a%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DN.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DSerra%26aufirst%3DV.%26aulast%3DLim%26aufirst%3DE.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26atitle%3DCDK12%2520Inhibition%2520Reverses%2520De%2520Novo%2520and%2520Acquired%2520PARP%2520Inhibitor%2520Resistance%2520in%2520BRCA%2520Wild-Type%2520and%2520Mutated%2520Models%2520of%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D17%26spage%3D2367%26epage%3D2381%26doi%3D10.1016%2Fj.celrep.2016.10.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paruch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doll, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keertikar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKittrick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hruza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomeir, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschmeier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sgambellone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seghezzi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paradkar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokosz, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Dinaciclib (SCH 727965): a Potent and Selective Inhibitor of Cyclin-Dependent Kinases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">204</span>â <span class="NLM_lpage">208</span>, <span class="refDoi">Â DOI: 10.1021/ml100051d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100051d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=204-208&author=K.+Paruchauthor=M.+P.+Dwyerauthor=C.+Alvarezauthor=C.+Brownauthor=T.-Y.+Chanauthor=R.+J.+Dollauthor=K.+Keertikarauthor=C.+Knutsonauthor=B.+McKittrickauthor=J.+Riveraauthor=R.+Rossmanauthor=G.+Tuckerauthor=T.+Fischmannauthor=A.+Hruzaauthor=V.+Madisonauthor=A.+A.+Nomeirauthor=Y.+Wangauthor=P.+Kirschmeierauthor=E.+Leesauthor=D.+Parryauthor=N.+Sgambelloneauthor=W.+Seghezziauthor=L.+Schultzauthor=F.+Shanahanauthor=D.+Wiswellauthor=X.+Xuauthor=Q.+Zhouauthor=R.+A.+Jamesauthor=V.+M.+Paradkarauthor=H.+Parkauthor=L.+R.+Rokoszauthor=T.+M.+Staufferauthor=T.+J.+Guzi&title=Discovery+of+Dinaciclib+%28SCH+727965%29%3A+a+Potent+and+Selective+Inhibitor+of+Cyclin-Dependent+Kinases&doi=10.1021%2Fml100051d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Dinaciclib (SCH 727965): A Potent and Selective Inhibitor of Cyclin-Dependent Kinases</span></div><div class="casAuthors">Paruch, Kamil; Dwyer, Michael P.; Alvarez, Carmen; Brown, Courtney; Chan, Tin-Yau; Doll, Ronald J.; Keertikar, Kerry; Knutson, Chad; McKittrick, Brian; Rivera, Jocelyn; Rossman, Randall; Tucker, Greg; Fischmann, Thierry; Hruza, Alan; Madison, Vincent; Nomeir, Amin A.; Wang, Yaolin; Kirschmeier, Paul; Lees, Emma; Parry, David; Sgambellone, Nicole; Seghezzi, Wolfgang; Schultz, Lesley; Shanahan, Frances; Wiswell, Derek; Xu, Xiaoying; Zhou, Quiao; James, Ray A.; Paradkar, Vidyadhar M.; Park, Haengsoon; Rokosz, Laura R.; Stauffer, Tara M.; Guzi, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">204-208</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of cyclin-dependent kinases (CDKs) has emerged as an attractive strategy for the development of novel oncol. therapeutics.  Herein is described the utilization of an in vivo screening approach with integrated efficacy and tolerability parameters to identify candidate CDK inhibitors with a suitable balance of activity and tolerability.  This approach has resulted in the identification of SCH 727965, a potent and selective CDK inhibitor that is currently undergoing clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorrwrMq6CW6rVg90H21EOLACvtfcHk0lieeV3786h9eA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D&md5=3b982547e064f658748fe4887a208659</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fml100051d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100051d%26sid%3Dliteratum%253Aachs%26aulast%3DParuch%26aufirst%3DK.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DAlvarez%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DT.-Y.%26aulast%3DDoll%26aufirst%3DR.%2BJ.%26aulast%3DKeertikar%26aufirst%3DK.%26aulast%3DKnutson%26aufirst%3DC.%26aulast%3DMcKittrick%26aufirst%3DB.%26aulast%3DRivera%26aufirst%3DJ.%26aulast%3DRossman%26aufirst%3DR.%26aulast%3DTucker%26aufirst%3DG.%26aulast%3DFischmann%26aufirst%3DT.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DMadison%26aufirst%3DV.%26aulast%3DNomeir%26aufirst%3DA.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKirschmeier%26aufirst%3DP.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DParry%26aufirst%3DD.%26aulast%3DSgambellone%26aufirst%3DN.%26aulast%3DSeghezzi%26aufirst%3DW.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DWiswell%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DJames%26aufirst%3DR.%2BA.%26aulast%3DParadkar%26aufirst%3DV.%2BM.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DRokosz%26aufirst%3DL.%2BR.%26aulast%3DStauffer%26aufirst%3DT.%2BM.%26aulast%3DGuzi%26aufirst%3DT.%2BJ.%26atitle%3DDiscovery%2520of%2520Dinaciclib%2520%2528SCH%2520727965%2529%253A%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520Cyclin-Dependent%2520Kinases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D204%26epage%3D208%26doi%3D10.1021%2Fml100051d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impastato, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mlynarski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnes, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haddrick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Selective Noncovalent CDK12 Inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">231</span>â <span class="NLM_lpage">235</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201700695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1002%2Fcmdc.201700695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=29266803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSnt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=231-235&author=J.+W.+Johannesauthor=C.+R.+Denzauthor=N.+Suauthor=A.+Wuauthor=A.+C.+Impastatoauthor=S.+Mlynarskiauthor=J.+G.+Varnesauthor=D.+B.+Princeauthor=J.+Cidadoauthor=N.+Gaoauthor=M.+Haddrickauthor=N.+H.+Jonesauthor=S.+Liauthor=X.+Liauthor=Y.+Liuauthor=T.+B.+Nguyenauthor=N.+O%E2%80%99Connellauthor=E.+Riversauthor=D.+W.+Robbinsauthor=R.+Tomlinsonauthor=T.+Yaoauthor=X.+Zhuauthor=A.+D.+Fergusonauthor=M.+L.+Lambauthor=J.+I.+Manchesterauthor=S.+Guichard&title=Structure-Based+Design+of+Selective+Noncovalent+CDK12+Inhibitors&doi=10.1002%2Fcmdc.201700695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Selective Noncovalent CDK12 Inhibitors</span></div><div class="casAuthors">Johannes, Jeffrey W.; Denz, Christopher R.; Su, Nancy; Wu, Allan; Impastato, Anna C.; Mlynarski, Scott; Varnes, Jeffrey G.; Prince, D. Bryan; Cidado, Justin; Gao, Ning; Haddrick, Malcolm; Jones, Natalie H.; Li, Shaobin; Li, Xiuwei; Liu, Yang; Nguyen, Toan B.; O'Connell, Nichole; Rivers, Emma; Robbins, Daniel W.; Tomlinson, Ronald; Yao, Tieguang; Zhu, Xiahui; Ferguson, Andrew D.; Lamb, Michelle L.; Manchester, John I.; Guichard, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-235</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase (CDK) 12 knockdown via siRNA decreases the transcription of DNA-damage-response genes and sensitizes BRCA wild-type cells to poly(ADP-ribose) polymerase (PARP) inhibition.  To recapitulate this effect with a small mol., we sought a potent, selective CDK12 inhibitor.  Crystal structures and modeling informed hybridization between dinaciclib and SR-3029, resulting in lead compd. 5 [(S)-2-(1-(6-(((6,7-difluoro-1H-benzo[d]imidazol-2-yl)methyl)amino)-9-ethyl-9H-purin-2-yl)piperidin-2-yl)ethan-1-ol].  Further structure-guided optimization delivered a series of selective CDK12 inhibitors, including compd. 7 [(S)-2-(1-(6-(((6,7-difluoro-1H-benzo[d]imidazol-2-yl)methyl)amino)-9-isopropyl-9H-purin-2-yl)piperidin-2-yl)ethan-1-ol].  Profiling of this compd. across CDK9, 7, 2, and 1 at high ATP concn., single-point kinase panel screening against 352 targets at 0.1 Î¼M, and proteomics via kinase affinity matrix technol. demonstrated the selectivity.  This series of compds. inhibits phosphorylation of Ser2 on the C-terminal repeat domain of RNA polymerase II, consistent with CDK12 inhibition.  These selective compds. were also acutely toxic to OV90 as well as THP1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfuD5WxqOQa7Vg90H21EOLACvtfcHk0lieeV3786h9eA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSnt7k%253D&md5=02cb347d4e2b76347f531281e455449f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700695%26sid%3Dliteratum%253Aachs%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DDenz%26aufirst%3DC.%2BR.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DA.%26aulast%3DImpastato%26aufirst%3DA.%2BC.%26aulast%3DMlynarski%26aufirst%3DS.%26aulast%3DVarnes%26aufirst%3DJ.%2BG.%26aulast%3DPrince%26aufirst%3DD.%2BB.%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHaddrick%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DN.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DRivers%26aufirst%3DE.%26aulast%3DRobbins%26aufirst%3DD.%2BW.%26aulast%3DTomlinson%26aufirst%3DR.%26aulast%3DYao%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DGuichard%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520Selective%2520Noncovalent%2520CDK12%2520Inhibitors%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D231%26epage%3D235%26doi%3D10.1002%2Fcmdc.201700695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokubo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maezaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-Benzyl-1-(trans-4-((5-cyanopyridin-2-yl) amino) cyclohexyl)-1-arylurea Derivatives as Novel and Selective Cyclin-Dependent Kinase 12 (CDK12) Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7710</span>â <span class="NLM_lpage">7728</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVWrsbjI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7710-7728&author=M.+Itoauthor=T.+Tanakaauthor=A.+Toitaauthor=N.+Uchiyamaauthor=H.+Kokuboauthor=N.+Morishitaauthor=M.+G.+Kleinauthor=H.+Zouauthor=M.+Murakamiauthor=M.+Kondoauthor=T.+Sameshimaauthor=S.+Arakiauthor=S.+Endoauthor=T.+Kawamotoauthor=G.+B.+Morinauthor=S.+A.+Aparicioauthor=A.+Nakanishiauthor=H.+Maezakiauthor=Y.+Imaeda&title=Discovery+of+3-Benzyl-1-%28trans-4-%28%285-cyanopyridin-2-yl%29+amino%29+cyclohexyl%29-1-arylurea+Derivatives+as+Novel+and+Selective+Cyclin-Dependent+Kinase+12+%28CDK12%29+Inhibitors&doi=10.1021%2Facs.jmedchem.8b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-Benzyl-1-(trans-4-((5-cyanopyridin-2-yl)amino)cyclohexyl)-1-arylurea Derivatives as Novel and Selective Cyclin-Dependent Kinase 12 (CDK12) Inhibitors</span></div><div class="casAuthors">Ito, Masahiro; Tanaka, Toshio; Toita, Akinori; Uchiyama, Noriko; Kokubo, Hironori; Morishita, Nao; Klein, Michael G.; Zou, Hua; Murakami, Morio; Kondo, Mitsuyo; Sameshima, Tomoya; Araki, Shinsuke; Endo, Satoshi; Kawamoto, Tomohiro; Morin, Gregg B.; Aparicio, Samuel A.; Nakanishi, Atsushi; Maezaki, Hironobu; Imaeda, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7710-7728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 12 (CDK12) plays a key role in the coordination of transcription with elongation and mRNA processing.  CDK12 mutations found in tumors and CDK12 inhibition sensitize cancer cells to DNA-damaging reagents and DNA-repair inhibitors.  This suggests that CDK12 inhibitors are potential therapeutics for cancer that may cause synthetic lethality.  Here, we report the discovery of 3-benzyl-1-(trans-4-((5-cyanopyridin-2-yl)amino)cyclohexyl)-1-arylurea derivs. as novel and selective CDK12 inhibitors.  Structure-activity relationship studies of a HTS hit, structure-based drug design, and conformation-oriented design using the Cambridge Structural Database afforded the optimized compd. 2, which exhibited not only potent CDK12 (and CDK13) inhibitory activity and excellent selectivity but also good physicochem. properties.  Furthermore, 2 inhibited the phosphorylation of Ser2 in the C-terminal domain of RNA polymerase II and induced growth inhibition in SK-BR-3 cells.  Therefore, 2 represents an excellent chem. probe for functional studies of CDK12 and could be a promising lead compd. for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXLhQTcUpBfLVg90H21EOLACvtfcHk0lieeV3786h9eA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVWrsbjI&md5=6d22fd36823d459fb60113c1212aa84e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00683%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DToita%26aufirst%3DA.%26aulast%3DUchiyama%26aufirst%3DN.%26aulast%3DKokubo%26aufirst%3DH.%26aulast%3DMorishita%26aufirst%3DN.%26aulast%3DKlein%26aufirst%3DM.%2BG.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DKondo%26aufirst%3DM.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DAraki%26aufirst%3DS.%26aulast%3DEndo%26aufirst%3DS.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DMorin%26aufirst%3DG.%2BB.%26aulast%3DAparicio%26aufirst%3DS.%2BA.%26aulast%3DNakanishi%26aufirst%3DA.%26aulast%3DMaezaki%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%25203-Benzyl-1-%2528trans-4-%2528%25285-cyanopyridin-2-yl%2529%2520amino%2529%2520cyclohexyl%2529-1-arylurea%2520Derivatives%2520as%2520Novel%2520and%2520Selective%2520Cyclin-Dependent%2520Kinase%252012%2520%2528CDK12%2529%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7710%26epage%3D7728%26doi%3D10.1021%2Facs.jmedchem.8b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quereda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vena, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frydman, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monastyrskyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckett, D. R.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Targeting of CDK12/CDK13 in Triple-Negative Breast Cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">545</span>â <span class="NLM_lpage">558</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2019.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.ccell.2019.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=31668947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVGmtbvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=545-558&author=V.+Queredaauthor=S.+Bayleauthor=F.+Venaauthor=S.+M.+Frydmanauthor=A.+Monastyrskyiauthor=W.+R.+Roushauthor=D.+R.+Duckett&title=Therapeutic+Targeting+of+CDK12%2FCDK13+in+Triple-Negative+Breast+Cancer&doi=10.1016%2Fj.ccell.2019.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Targeting of CDK12/CDK13 in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Quereda, Victor; Bayle, Simon; Vena, Francesca; Frydman, Sylvia M.; Monastyrskyi, Andrii; Roush, William R.; Duckett, Derek R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">545-558.e7</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic regulation enables tumors to respond to changing environments during tumor progression and metastases and facilitates treatment resistance.  Targeting chromatin modifiers or catalytic effectors of transcription is an emerging anti-cancer strategy.  The cyclin-dependent kinases (CDKs) 12 and 13 phosphorylate the C-terminal domain of RNA polymerase II, regulating transcription and co-transcriptional processes.  Here we report the development of SR-4835, a highly selective dual inhibitor of CDK12 and CDK13, which disables triple-neg. breast cancer (TNBC) cells.  Mechanistically, inhibition or loss of CDK12/CDK13 triggers intronic polyadenylation site cleavage that suppresses the expression of core DNA damage response proteins.  This provokes a "BRCAness" phenotype that results in deficiencies in DNA damage repair, promoting synergy with DNA-damaging chemotherapy and PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd0hwftXX1YrVg90H21EOLACvtfcHk0ljMjYQwvkx-gA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVGmtbvM&md5=bd4a83717363acb4e6e33228727bfe01</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DQuereda%26aufirst%3DV.%26aulast%3DBayle%26aufirst%3DS.%26aulast%3DVena%26aufirst%3DF.%26aulast%3DFrydman%26aufirst%3DS.%2BM.%26aulast%3DMonastyrskyi%26aufirst%3DA.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DDuckett%26aufirst%3DD.%2BR.%26atitle%3DTherapeutic%2520Targeting%2520of%2520CDK12%252FCDK13%2520in%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D545%26epage%3D558%26doi%3D10.1016%2Fj.ccell.2019.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Misra, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarz, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellar, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulheron, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batorsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roongta, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marathe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranadive, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavletich, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, F. Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimball, S. D.</span></span> <span> </span><span class="NLM_article-title"><i>N</i>-(Cycloalkylamino) acyl-2-aminothiazole Inhibitors of Cyclin-Dependent Kinase 2. <i>N</i>-[5-[[[5-(1, 1-dimethylethyl)-2-oxazolyl] methyl] thio]-2-thiazolyl]-4-piperidinecarboxamide (BMS-387032), a Highly Efficacious and Selective Antitumor Agent</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1719</span>â <span class="NLM_lpage">1728</span>, <span class="refDoi">Â DOI: 10.1021/jm0305568</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0305568" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1yls7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1719-1728&author=R.+N.+Misraauthor=H.-y.+Xiaoauthor=K.+S.+Kimauthor=S.+Luauthor=W.-C.+Hanauthor=S.+A.+Barbosaauthor=J.+T.+Huntauthor=D.+B.+Rawlinsauthor=W.+Shanauthor=S.+Z.+Ahmedauthor=L.+Qianauthor=B.-C.+Chenauthor=R.+Zhaoauthor=M.+S.+Bednarzauthor=K.+A.+Kellarauthor=J.+G.+Mulheronauthor=R.+Batorskyauthor=U.+Roongtaauthor=A.+Kamathauthor=P.+Maratheauthor=S.+A.+Ranadiveauthor=J.+S.+Sackauthor=J.+S.+Tokarskiauthor=N.+P.+Pavletichauthor=F.+Y.+F.+Leeauthor=K.+R.+Websterauthor=S.+D.+Kimball&title=N-%28Cycloalkylamino%29+acyl-2-aminothiazole+Inhibitors+of+Cyclin-Dependent+Kinase+2.+N-%5B5-%5B%5B%5B5-%281%2C+1-dimethylethyl%29-2-oxazolyl%5D+methyl%5D+thio%5D-2-thiazolyl%5D-4-piperidinecarboxamide+%28BMS-387032%29%2C+a+Highly+Efficacious+and+Selective+Antitumor+Agent&doi=10.1021%2Fjm0305568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">N-(cycloalkylamino)acyl-2-aminothiazole inhibitors of cyclin-dependent kinase 2. N-[5-[[[5-(1,1-dimethylethyl)-2-oxazolyl]methyl]thio]-2-thiazolyl]-4- piperidinecarboxamide (BMS-387032), a highly efficacious and selective antitumor agent</span></div><div class="casAuthors">Misra, Raj N.; Xiao, Hai-yun; Kim, Kyoung S.; Lu, Songfeng; Han, Wen-Ching; Barbosa, Stephanie A.; Hunt, John T.; Rawlins, David B.; Shan, Weifang; Ahmed, Syed Z.; Qian, Ligang; Chen, Bang-Chi; Zhao, Rulin; Bednarz, Mark S.; Kellar, Kristen A.; Mulheron, Janet G.; Batorsky, Roberta; Roongta, Urvashi; Kamath, Amrita; Marathe, Punit; Ranadive, Sunanda A.; Sack, John S.; Tokarski, John S.; Pavletich, Nikola P.; Lee, Francis Y. F.; Webster, Kevin R.; Kimball, S. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1719-1728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-Acyl-2-aminothiazoles with nonarom. acyl side chains contg. a basic amine were found to be potent, selective inhibitors of CDK2/cycE which exhibit antitumor activity in mice.  In particular, compd. 21 {N-[5-[[[5-(1,1-dimethylethyl)-2-oxazolyl]methyl]thio]-2-thiazolyl]-4-piperidinecarboxamide, BMS-387032} has been identified as an ATP-competitive and CDK2-selective inhibitor which has been selected to enter Phase 1 human clin. trials as an antitumor agent.  In a cell-free enzyme assay, 21 showed a CDK2/cycE IC50 = 48 nM and was 10- and 20-fold selective over CDK1/cycB and CDK4/cycD, resp.  It was also highly selective over a panel of 12 unrelated kinases.  Antiproliferative activity was established in an A2780 cellular cytotoxicity assay in which 21 showed an IC50 = 95 nM.  Metab. and pharmacokinetic studies showed that 21 exhibited a plasma half-life of 5-7 h in three species and moderately low protein binding in both mouse (69%) and human (63%) serum.  Dosed orally to mouse, rat, and dog, 21 showed 100%, 31%, and 28% bioavailability, resp.  As an antitumor agent in mice, 21 administered at its max.-tolerated dose exhibited a clearly superior efficacy profile when compared to flavopiridol in both an i.p. P388 murine tumor model and in a s.c./i.p. A2780 human ovarian carcinoma xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHnukOh_KXGLVg90H21EOLACvtfcHk0ljMjYQwvkx-gA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1yls7k%253D&md5=28cafa472df3e6562023f7d24f70ed45</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm0305568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0305568%26sid%3Dliteratum%253Aachs%26aulast%3DMisra%26aufirst%3DR.%2BN.%26aulast%3DXiao%26aufirst%3DH.-y.%26aulast%3DKim%26aufirst%3DK.%2BS.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DW.-C.%26aulast%3DBarbosa%26aufirst%3DS.%2BA.%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DRawlins%26aufirst%3DD.%2BB.%26aulast%3DShan%26aufirst%3DW.%26aulast%3DAhmed%26aufirst%3DS.%2BZ.%26aulast%3DQian%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DB.-C.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DBednarz%26aufirst%3DM.%2BS.%26aulast%3DKellar%26aufirst%3DK.%2BA.%26aulast%3DMulheron%26aufirst%3DJ.%2BG.%26aulast%3DBatorsky%26aufirst%3DR.%26aulast%3DRoongta%26aufirst%3DU.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DMarathe%26aufirst%3DP.%26aulast%3DRanadive%26aufirst%3DS.%2BA.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DTokarski%26aufirst%3DJ.%2BS.%26aulast%3DPavletich%26aufirst%3DN.%2BP.%26aulast%3DLee%26aufirst%3DF.%2BY.%2BF.%26aulast%3DWebster%26aufirst%3DK.%2BR.%26aulast%3DKimball%26aufirst%3DS.%2BD.%26atitle%3DN-%2528Cycloalkylamino%2529%2520acyl-2-aminothiazole%2520Inhibitors%2520of%2520Cyclin-Dependent%2520Kinase%25202.%2520N-%255B5-%255B%255B%255B5-%25281%252C%25201-dimethylethyl%2529-2-oxazolyl%255D%2520methyl%255D%2520thio%255D-2-thiazolyl%255D-4-piperidinecarboxamide%2520%2528BMS-387032%2529%252C%2520a%2520Highly%2520Efficacious%2520and%2520Selective%2520Antitumor%2520Agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1719%26epage%3D1728%26doi%3D10.1021%2Fjm0305568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamberto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span> <span> </span><span class="NLM_article-title">Leveraging Gas-Phase Fragmentation Pathways for Improved Identification and Selective Detection of Targets Modified by Covalent Probes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">12248</span>â <span class="NLM_lpage">12254</span>, <span class="refDoi">Â DOI: 10.1021/acs.analchem.6b03394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b03394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2016&pages=12248-12254&author=S.+B.+Ficarroauthor=C.+M.+Browneauthor=J.+D.+Cardauthor=W.+M.+Alexanderauthor=T.+Zhangauthor=E.+Parkauthor=R.+McNallyauthor=S.+Dhe-Paganonauthor=H.-S.+Seoauthor=I.+Lambertoauthor=M.+J.+Eckauthor=S.+J.+Buhrlageauthor=N.+S.+Grayauthor=J.+A.+Marto&title=Leveraging+Gas-Phase+Fragmentation+Pathways+for+Improved+Identification+and+Selective+Detection+of+Targets+Modified+by+Covalent+Probes&doi=10.1021%2Facs.analchem.6b03394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging Gas-Phase Fragmentation Pathways for Improved Identification and Selective Detection of Targets Modified by Covalent Probes</span></div><div class="casAuthors">Ficarro, Scott B.; Browne, Christopher M.; Card, Joseph D.; Alexander, William M.; Zhang, Tinghu; Park, Eunyoung; McNally, Randall; Dhe-Paganon, Sirano; Seo, Hyuk-Soo; Lamberto, Ilaria; Eck, Michael J.; Buhrlage, Sara J.; Gray, Nathanael S.; Marto, Jarrod A.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">12248-12254</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent approval of covalent inhibitors for multiple clin. indications has reignited enthusiasm for this class of drugs.  As interest in covalent drugs has increased, so too has the need for anal. platforms that can leverage their mechanism-of-action to characterize modified protein targets.  Here the authors describe novel gas phase dissocn. pathways which yield predictable fragment ions during MS/MS of inhibitor-modified peptides.  The authors find that these dissocn. pathways are common to numerous cysteine-directed probes as well as the covalent drugs, Ibrutinib and Neratinib.  The authors leverage the predictable nature of these fragment ions to improve the confidence of peptide sequence assignment in proteomic analyses, and explore their potential use in selective mass spectrometry-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNu24Jo7-IWLVg90H21EOLACvtfcHk0lgSHVH-IsTNNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO&md5=7932392d2d2ac88def4cacecbda41a30</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b03394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b03394%26sid%3Dliteratum%253Aachs%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DCard%26aufirst%3DJ.%2BD.%26aulast%3DAlexander%26aufirst%3DW.%2BM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DMcNally%26aufirst%3DR.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DLamberto%26aufirst%3DI.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26atitle%3DLeveraging%2520Gas-Phase%2520Fragmentation%2520Pathways%2520for%2520Improved%2520Identification%2520and%2520Selective%2520Detection%2520of%2520Targets%2520Modified%2520by%2520Covalent%2520Probes%26jtitle%3DAnal.%2520Chem.%26date%3D2016%26volume%3D88%26spage%3D12248%26epage%3D12254%26doi%3D10.1021%2Facs.analchem.6b03394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ©lias, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A Small MoleculeâKinase Interaction Map for Clinical Kinase Inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>â <span class="NLM_lpage">336</span>, <span class="refDoi">Â DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+Small+Molecule%E2%80%93Kinase+Interaction+Map+for+Clinical+Kinase+Inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lgSHVH-IsTNNQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Small%2520Molecule%25E2%2580%2593Kinase%2520Interaction%2520Map%2520for%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A Quantitative Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+Quantitative+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lgXK3xpm8f8Hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520Quantitative%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartkowiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phatnani, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuda, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, A. L.</span></span> <span> </span><span class="NLM_article-title">CDK12 Is a Transcription Elongation-Associated CTD Kinase, the Metazoan Ortholog of Yeast Ctk1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2303</span>â <span class="NLM_lpage">2316</span>, <span class="refDoi">Â DOI: 10.1101/gad.1968210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1101%2Fgad.1968210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=20952539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlyqt7vM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2303-2316&author=B.+Bartkowiakauthor=P.+Liuauthor=H.+P.+Phatnaniauthor=N.+J.+Fudaauthor=J.+J.+Cooperauthor=D.+H.+Priceauthor=K.+Adelmanauthor=J.+T.+Lisauthor=A.+L.+Greenleaf&title=CDK12+Is+a+Transcription+Elongation-Associated+CTD+Kinase%2C+the+Metazoan+Ortholog+of+Yeast+Ctk1&doi=10.1101%2Fgad.1968210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CDK12 is a transcription elongation-associated CTD kinase, the metazoan ortholog of yeast Ctk1</span></div><div class="casAuthors">Bartkowiak, Bartlomiej; Liu, Pengda; Phatnani, Hemali P.; Fuda, Nicholas J.; Cooper, Jeffrey J.; Price, David H.; Adelman, Karen; Lis, John T.; Greenleaf, Arno L.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2303-2316</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Drosophila contains one (dCDK12) and humans contain two (hCDK12 and hCDK13) proteins that are the closest evolutionary relatives of yeast Ctk1, the catalytic subunit of the major elongation-phase C-terminal repeat domain (CTD) kinase in Saccharomyces cerevisiae, CTDK-I.  However, until now, neither CDK12 nor CDK13 has been demonstrated to be a bona fide CTD kinase.  Using Drosophila, the authors demonstrate that dCDK12 (CG7597) is a transcription-assocd. CTD kinase, the ortholog of yCtk1.  Fluorescence microscopy reveals that the distribution of dCDK12 on formaldehyde-fixed polytene chromosomes is virtually identical to that of hyperphosphorylated RNA polymerase II (RNAPII), but is distinct from that of P-TEFb (dCDK9 + dCyclin T).  Chromatin immunopptn. (ChIP) expts. confirm that dCDK12 is present on the transcribed regions of active Drosophila genes.  Compared with P-TEFb, dCDK12 amts. are lower at the 5' end and higher in the middle and at the 3' end of genes (both normalized to RNAPII).  Appropriately, Drosophila dCDK12 purified from nuclear exts. manifests CTD kinase activity in vitro.  Intriguingly, cyclin K is assocd. with purified dCDK12, implicating it as the cyclin subunit of this CTD kinase.  Most importantly, we demonstrate that RNAi knockdown of dCDK12 in S2 cells alters the phosphorylation state of the CTD, reducing its Ser2 phosphorylation levels.  Similarly, in human HeLa cells, hCDK13 purified from nuclear exts. displays CTD kinase activity in vitro, as anticipated.  Also, chimeric (yeast/human) versions of Ctk1 contg. the kinase homol. domains of hCDK12/13 (or hCDK9) are functional in yeast cells (and also in vitro).  This system was used to show that a bur1ts mutant is rescued more efficiently by a hCDK9 chimera than by a hCDK13 chimera, suggesting the following orthol. relationships: Bur1 â CDK9 and Ctk1 â CDK12/13.  Finally, siRNA knockdown of hCDK12 in HeLa cells results in alterations in the CTD phosphorylation state.  These findings demonstrate that metazoan CDK12 and CDK13 are CTD kinases, and that CDK12 is orthologous to yeast Ctk1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5PzvsoT0tWbVg90H21EOLACvtfcHk0lgXK3xpm8f8Hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlyqt7vM&md5=703994d04539e89a2efe59f8eb6c0457</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1101%2Fgad.1968210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1968210%26sid%3Dliteratum%253Aachs%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DPhatnani%26aufirst%3DH.%2BP.%26aulast%3DFuda%26aufirst%3DN.%2BJ.%26aulast%3DCooper%26aufirst%3DJ.%2BJ.%26aulast%3DPrice%26aufirst%3DD.%2BH.%26aulast%3DAdelman%26aufirst%3DK.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26atitle%3DCDK12%2520Is%2520a%2520Transcription%2520Elongation-Associated%2520CTD%2520Kinase%252C%2520the%2520Metazoan%2520Ortholog%2520of%2520Yeast%2520Ctk1%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2303%26epage%3D2316%26doi%3D10.1101%2Fgad.1968210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blazek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohoutek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholomeeusen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulinkova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimermancic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ule, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin, B. M.</span></span> <span> </span><span class="NLM_article-title">The Cyclin K/Cdk12 Complex Maintains Genomic Stability via Regulation of Expression of DNA Damage Response Genes. <i>Genes &</i></span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2158</span>â <span class="NLM_lpage">2172</span>, <span class="refDoi">Â DOI: 10.1101/gad.16962311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1101%2Fgad.16962311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=22012619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVant73E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=2158-2172&author=D.+Blazekauthor=J.+Kohoutekauthor=K.+Bartholomeeusenauthor=E.+Johansenauthor=P.+Hulinkovaauthor=Z.+Luoauthor=P.+Cimermancicauthor=J.+Uleauthor=B.+M.+Peterlin&title=The+Cyclin+K%2FCdk12+Complex+Maintains+Genomic+Stability+via+Regulation+of+Expression+of+DNA+Damage+Response+Genes.+Genes+%26&doi=10.1101%2Fgad.16962311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The cyclin K/Cdk12 complex maintains genomic stability via regulation of expression of DNA damage response genes</span></div><div class="casAuthors">Blazek, Dalibor; Kohoutek, Jiri; Bartholomeeusen, Koen; Johansen, Eric; Hulinkova, Petra; Luo, Zeping; Cimermancic, Peter; Ule, Jernej; Peterlin, B. Matija</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2158-2172</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Various cyclin-dependent kinase (Cdk) complexes have been implicated in the regulation of transcription.  In this study, we identified a 70-kDa Cyclin K (CycK) that binds Cdk12 and Cdk13 to form two different complexes (CycK/Cdk12 or CycK/Cdk13) in human cells.  The CycK/Cdk12 complex regulates phosphorylation of Ser2 in the C-terminal domain of RNA polymerase II and expression of a small subset of human genes, as revealed in expression microarrays.  Depletion of CycK/Cdk12 results in decreased expression of predominantly long genes with high nos. of exons.  The most prominent group of down-regulated genes are the DNA damage response genes, including the crit. regulators of genomic stability: BRCA1 (breast and ovarian cancer type 1 susceptibility protein 1), ATR (ataxia telangiectasia and Rad3-related), FANCI, and FANCD2.  We show that CycK/Cdk12, rather than CycK/Cdk13, is necessary for their expression.  Nuclear run-on assays and chromatin immunopptns. with RNA polymerase II on the BRCA1 and FANCI genes suggest a transcriptional defect in the absence of CycK/Cdk12.  Consistent with these findings, cells without CycK/Cdk12 induce spontaneous DNA damage and are sensitive to a variety of DNA damage agents.  We conclude that through regulation of expression of DNA damage response genes, CycK/Cdk12 protects cells from genomic instability.  The essential role of CycK for organisms in vivo is further supported by the result that genetic inactivation of CycK in mice causes early embryonic lethality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPlifbvX1LXrVg90H21EOLACvtfcHk0liujA0dnH6kYw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVant73E&md5=4e472d0f324bbfae1a9d0a8a7302fa1d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1101%2Fgad.16962311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.16962311%26sid%3Dliteratum%253Aachs%26aulast%3DBlazek%26aufirst%3DD.%26aulast%3DKohoutek%26aufirst%3DJ.%26aulast%3DBartholomeeusen%26aufirst%3DK.%26aulast%3DJohansen%26aufirst%3DE.%26aulast%3DHulinkova%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DCimermancic%26aufirst%3DP.%26aulast%3DUle%26aufirst%3DJ.%26aulast%3DPeterlin%26aufirst%3DB.%2BM.%26atitle%3DThe%2520Cyclin%2520K%252FCdk12%2520Complex%2520Maintains%2520Genomic%2520Stability%2520via%2520Regulation%2520of%2520Expression%2520of%2520DNA%2520Damage%2520Response%2520Genes.%2520Genes%2520%2526%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D2158%26epage%3D2172%26doi%3D10.1101%2Fgad.16962311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burness, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Trevino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harouaka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clouthier, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span> <span> </span><span class="NLM_article-title">RAD51 Mediates Resistance of Cancer Stem Cells to PARP Inhibition in Triple-Negative Breast Cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">514</span>â <span class="NLM_lpage">522</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-15-1348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1158%2F1078-0432.CCR-15-1348" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=28034904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVeitLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=514-522&author=Y.+Liuauthor=M.+L.+Burnessauthor=R.+Martin-Trevinoauthor=J.+Guyauthor=S.+Baiauthor=R.+Harouakaauthor=M.+D.+Brooksauthor=L.+Shangauthor=A.+Foxauthor=T.+K.+Lutherauthor=A.+Davisauthor=T.+L.+Bakerauthor=J.+Colacinoauthor=S.+G.+Clouthierauthor=Z.-M.+Shaoauthor=M.+S.+Wichaauthor=S.+Liu&title=RAD51+Mediates+Resistance+of+Cancer+Stem+Cells+to+PARP+Inhibition+in+Triple-Negative+Breast+Cancer&doi=10.1158%2F1078-0432.CCR-15-1348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">RAD51 Mediates Resistance of Cancer Stem Cells to PARP Inhibition in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Liu, Yajing; Burness, Monika L.; Martin-Trevino, Rachel; Guy, Joey; Bai, Shoumin; Harouaka, Ramdane; Brooks, Michael D.; Shang, Li; Fox, Alex; Luther, Tahra K.; Davis, April; Baker, Trenton L.; Colacino, Justin; Clouthier, Shawn G.; Shao, Zhi-ming; Wicha, Max S.; Liu, Suling</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">514-522</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Introduction: PARP inhibitors have shown promising results in early studies for treatment of breast cancer susceptibility gene (BRCA)-deficient breast cancers; however, resistance ultimately develops.  Furthermore, the benefit of PARP inhibitors (PARPi) in triple-neg. breast cancers (TNBC) remains unknown.  Recent evidence indicates that in TNBCs, cells that display "cancer stem cell" properties are resistant to conventional treatments, mediate tumor metastasis, and contribute to recurrence.  The sensitivity of breast cancer stem cells (CSC) to PARPi is unknown.  Exptl. Design: We detd. the sensitivity of breast CSCs to PARP inhibition in BRCA1-mutant and -wild-type TNBC cell lines and tumor xenografts.  We also investigated the role of RAD51 in mediating CSC resistance to PARPi in these in vitro and in vivo models.  Results: We demonstrated that the CSCs in BRCA1-mutant TNBCs were resistant to PARP inhibition, and that these cells had both elevated RAD51 protein levels and activity.  Downregulation of RAD51 by shRNA sensitized CSCs to PARP inhibition and reduced tumor growth.  BRCA1-wild-type cells were relatively resistant to PARP inhibition alone, but redn. of RAD51 sensitized both CSC and bulk cells in these tumors to PARPi treatment.  Conclusions: Our data suggest that in both BRCA1-mutant and BRCA1-wild-type TNBCs, CSCs are relatively resistant to PARP inhibition.  This resistance is mediated by RAD51, suggesting that strategies aimed at targeting RAD51 may increase the therapeutic efficacy of PARPi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooTDtZrEs0trVg90H21EOLACvtfcHk0liujA0dnH6kYw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVeitLs%253D&md5=56dce7989dcf9edb314d3927adf180df</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-15-1348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-15-1348%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBurness%26aufirst%3DM.%2BL.%26aulast%3DMartin-Trevino%26aufirst%3DR.%26aulast%3DGuy%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DS.%26aulast%3DHarouaka%26aufirst%3DR.%26aulast%3DBrooks%26aufirst%3DM.%2BD.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DFox%26aufirst%3DA.%26aulast%3DLuther%26aufirst%3DT.%2BK.%26aulast%3DDavis%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DT.%2BL.%26aulast%3DColacino%26aufirst%3DJ.%26aulast%3DClouthier%26aufirst%3DS.%2BG.%26aulast%3DShao%26aufirst%3DZ.-M.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26aulast%3DLiu%26aufirst%3DS.%26atitle%3DRAD51%2520Mediates%2520Resistance%2520of%2520Cancer%2520Stem%2520Cells%2520to%2520PARP%2520Inhibition%2520in%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D514%26epage%3D522%26doi%3D10.1158%2F1078-0432.CCR-15-1348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bliss, C.</span></span> <span> </span><span class="NLM_article-title">The Calculation of Microbial Assays</span>. <i>Bacteriol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1956</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">243</span>â <span class="NLM_lpage">258</span>, <span class="refDoi">Â DOI: 10.1128/MMBR.20.4.243-258.1956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1128%2FMMBR.20.4.243-258.1956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=13403845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A280%3ADyaG2s%252FltVWhtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1956&pages=243-258&author=C.+Bliss&title=The+Calculation+of+Microbial+Assays&doi=10.1128%2FMMBR.20.4.243-258.1956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The calculation of microbial assays</span></div><div class="casAuthors">BLISS C I</div><div class="citationInfo"><span class="NLM_cas:title">Bacteriological reviews</span>
        (<span class="NLM_cas:date">1956</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">243-58</span>
        ISSN:<span class="NLM_cas:issn">0005-3678</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-6-02IQzk_jW1zwg2yUK5fW6udTcc2ea4qb4-agcSl7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaG2s%252FltVWhtQ%253D%253D&md5=f0b119a1b363e5273d4932ed9ef488a0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FMMBR.20.4.243-258.1956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.20.4.243-258.1956%26sid%3Dliteratum%253Aachs%26aulast%3DBliss%26aufirst%3DC.%26atitle%3DThe%2520Calculation%2520of%2520Microbial%2520Assays%26jtitle%3DBacteriol.%2520Rev.%26date%3D1956%26volume%3D20%26spage%3D243%26epage%3D258%26doi%3D10.1128%2FMMBR.20.4.243-258.1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greco, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, J. C.</span></span> <span> </span><span class="NLM_article-title">The Search for Synergy: A Critical Review from a Response Surface Perspective</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">331</span>â <span class="NLM_lpage">385</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=7568331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A280%3ADyaK28%252FjsFSgtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1995&pages=331-385&author=W.+R.+Grecoauthor=G.+Bravoauthor=J.+C.+Parsons&title=The+Search+for+Synergy%3A+A+Critical+Review+from+a+Response+Surface+Perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The search for synergy: a critical review from a response surface perspective</span></div><div class="casAuthors">Greco W R; Bravo G; Parsons J C</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological reviews</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-85</span>
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRXywHk-SVAkdenUz0lpNqnfW6udTcc2eaPcVMSXaKpJ7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK28%252FjsFSgtQ%253D%253D&md5=80f1b0c8e20e587625a5d68339d30709</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreco%26aufirst%3DW.%2BR.%26aulast%3DBravo%26aufirst%3DG.%26aulast%3DParsons%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Search%2520for%2520Synergy%253A%2520A%2520Critical%2520Review%2520from%2520a%2520Response%2520Surface%2520Perspective%26jtitle%3DPharmacol.%2520Rev.%26date%3D1995%26volume%3D47%26spage%3D331%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choong, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafimova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockett, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahr, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, U.</span></span> <span> </span><span class="NLM_article-title">A Diaminocyclohexyl Analog of SNS-032 with Improved Permeability and Bioavailability Properties</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5763</span>â <span class="NLM_lpage">5765</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2008.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.bmcl.2008.09.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=18842409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht12rs77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5763-5765&author=I.+C.+Choongauthor=I.+Serafimovaauthor=J.+Fanauthor=D.+Stockettauthor=E.+Chanauthor=S.+Cheetiauthor=Y.+Luauthor=B.+Fahrauthor=P.+Phamauthor=M.+R.+Arkinauthor=D.+H.+Walkerauthor=U.+Hoch&title=A+Diaminocyclohexyl+Analog+of+SNS-032+with+Improved+Permeability+and+Bioavailability+Properties&doi=10.1016%2Fj.bmcl.2008.09.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A diaminocyclohexyl analog of SNS-032 with improved permeability and bioavailability properties</span></div><div class="casAuthors">Choong, Ingrid C.; Serafimova, Iana; Fan, Junfa; Stockett, David; Chan, Erica; Cheeti, Sravanthi; Lu, Yafan; Fahr, Bruce; Pham, Phuongly; Arkin, Michelle R.; Walker, Duncan H.; Hoch, Ute</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5763-5765</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The identification of a selective CDK2, 7, and 9 inhibitor (compd. 4) with improved permeability is described.  Compd. 4 exhibits comparable CDK selectivity profile to SNS-032, but shows improved permeability and higher bioavailability in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEqkByuQcxcbVg90H21EOLACvtfcHk0lj8n0Ty1rlkQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht12rs77M&md5=d956b4c2e3b15b3d52ddd76f2cef997e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DChoong%26aufirst%3DI.%2BC.%26aulast%3DSerafimova%26aufirst%3DI.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DStockett%26aufirst%3DD.%26aulast%3DChan%26aufirst%3DE.%26aulast%3DCheeti%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DFahr%26aufirst%3DB.%26aulast%3DPham%26aufirst%3DP.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWalker%26aufirst%3DD.%2BH.%26aulast%3DHoch%26aufirst%3DU.%26atitle%3DA%2520Diaminocyclohexyl%2520Analog%2520of%2520SNS-032%2520with%2520Improved%2520Permeability%2520and%2520Bioavailability%2520Properties%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5763%26epage%3D5765%26doi%3D10.1016%2Fj.bmcl.2008.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, A. G.</span></span> <span> </span><span class="NLM_article-title">A Universal Algorithm for Fast and Automated Charge State Deconvolution of Electrospray Mass-to-Charge Ratio Spectra</span>. <i>J. Am. Soc. Mass Spectrom.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">225</span>â <span class="NLM_lpage">233</span>, <span class="refDoi">Â DOI: 10.1016/S1044-0305(97)00284-5</span> </span><div class="citationLinks">[<a href="/doi/10.1016/S1044-0305%2897%2900284-5" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1998&pages=225-233&author=Z.+Zhangauthor=A.+G.+Marshall&title=A+Universal+Algorithm+for+Fast+and+Automated+Charge+State+Deconvolution+of+Electrospray+Mass-to-Charge+Ratio+Spectra&doi=10.1016%2FS1044-0305%2897%2900284-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A universal algorithm for fast and automated charge state deconvolution of electrospray mass-to-charge ratio spectra</span></div><div class="casAuthors">Zhang, Zhongqi; Marshall, Alan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">This article describes a new algorithm for charge state detn. and deconvolution of electrospray ionization (ESI) mass-to-charge ratio spectra.  The algorithm (ZSCORE) is based on a charge scoring scheme that incorporates all above-threshold members of a family of charge states or isotopic components, and deconvolves both low- and high-resoln. mass-to-charge ratio spectra, with or without a peak list (stick plot).  A scoring wt. factor, log (I/I0), in which I is the signal magnitude at a calcd. mass-to-charge ratio, and I0 is the signal threshold near that mass-to-charge ratio, was used in most cases.  For high-resoln. mass-to-charge ratio spectra in which all isotopic peaks are resolved, the algorithm can deconvolve overlapped isotopic multiplets of the same of different charge state.  Compared to other deconvolution techniques, the algorithm is robust, rapid, and fully automated (i.e., no user input during the deconvolution process).  It eliminates artifact peaks without introducing peak distortions.  Its performance is demonstrated for exptl. ESI Fourier transform ICR mass-to-charge ratio spectra (both low and high resoln.).  Charge state deconvolution to yield a zero-charge mass spectrum should prove particularly useful for interpreting spectra of complex mixts., identifying contaminants, noncovalent adducts, fragments (N-terminal, C-terminal, internal), and chem. modifications of electrosprayed biomacromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-7hFdAcR_BbVg90H21EOLACvtfcHk0lj8n0Ty1rlkQg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D&md5=5bee6ce6556beed4ee9004cfe6c7ecad</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS1044-0305%2897%2900284-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1044-0305%252897%252900284-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26atitle%3DA%2520Universal%2520Algorithm%2520for%2520Fast%2520and%2520Automated%2520Charge%2520State%2520Deconvolution%2520of%2520Electrospray%2520Mass-to-Charge%2520Ratio%2520Spectra%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D1998%26volume%3D9%26spage%3D225%26epage%3D233%26doi%3D10.1016%2FS1044-0305%2897%2900284-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelmant, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J.</span></span> <span> </span><span class="NLM_article-title">Multiplierz V2.0: A Python-Based Ecosystem for Shared Access and Analysis of Native Mass Spectrometry Data</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1700091</span>, <span class="refDoi">Â DOI: 10.1002/pmic.201700091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1002%2Fpmic.201700091" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=1700091&author=W.+M.+Alexanderauthor=S.+B.+Ficarroauthor=G.+Adelmantauthor=J.+Marto&title=Multiplierz+V2.0%3A+A+Python-Based+Ecosystem+for+Shared+Access+and+Analysis+of+Native+Mass+Spectrometry+Data&doi=10.1002%2Fpmic.201700091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201700091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201700091%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DW.%2BM.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DAdelmant%26aufirst%3DG.%26aulast%3DMarto%26aufirst%3DJ.%26atitle%3DMultiplierz%2520V2.0%253A%2520A%2520Python-Based%2520Ecosystem%2520for%2520Shared%2520Access%2520and%2520Analysis%2520of%2520Native%2520Mass%2520Spectrometry%2520Data%26jtitle%3DProteomics%26date%3D2017%26volume%3D17%26spage%3D1700091%26doi%3D10.1002%2Fpmic.201700091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J.</span></span> <span> </span><span class="NLM_article-title">MzStudio: a Dynamic Digital Canvas for User-Driven Interrogation of Mass Spectrometry Data</span>. <i>Proteomes</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">20</span>, <span class="refDoi">Â DOI: 10.3390/proteomes5030020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.3390%2Fproteomes5030020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkt1Sls7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=20&author=S.+Ficarroauthor=W.+Alexanderauthor=J.+Marto&title=MzStudio%3A+a+Dynamic+Digital+Canvas+for+User-Driven+Interrogation+of+Mass+Spectrometry+Data&doi=10.3390%2Fproteomes5030020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">mzStudio: a dynamic digital canvas for user-driven interrogation of mass spectrometry data</span></div><div class="casAuthors">Ficarro, Scott B.; Alexander, William M.; Marto, Jarrod A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteomes</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">20/1-20/8</span>CODEN:
                <span class="NLM_cas:coden">PROTHC</span>;
        ISSN:<span class="NLM_cas:issn">2227-7382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Although not yet truly 'comprehensive', modern mass spectrometry-based expts. can generate quant. data for a meaningful fraction of the human proteome.  Importantly for large-scale protein expression anal., robust data pipelines are in place for identification of un-modified peptide sequences and aggregation of these data to protein-level quantification.  However, interoperable software tools that enable scientists to computationally explore and document novel hypotheses for peptide sequence, modification status, or fragmentation behavior are not well-developed.  Here, we introduce mzStudio, an open-source Python module built on our multiplierz project.  This desktop application provides a highly-interactive graphical user interface (GUI) through which scientists can examine and annotate spectral features, re-search existing PSMs to test different modifications or new spectral matching algorithms, share results with colleagues, integrate other domain-specific software tools, and finally create publication-quality graphics. mzStudio leverages our common application programming interface (mzAPI) for access to native data files from multiple instrument platforms, including ion trap, quadrupole time-of-flight, Orbitrap, matrix-assisted laser desorption ionization, and triple quadrupole mass spectrometers and is compatible with several popular search engines including Mascot, Proteome Discoverer, X!Tandem, and Comet.  The mzStudio toolkit enables researchers to create a digital provenance of data analytics and other evidence that support specific peptide sequence assignments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquRNKhFy0rcLVg90H21EOLACvtfcHk0lhcHRdd0hytxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkt1Sls7w%253D&md5=9325e9aaf3183724de5f3f3e5ff4c9e0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3390%2Fproteomes5030020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fproteomes5030020%26sid%3Dliteratum%253Aachs%26aulast%3DFicarro%26aufirst%3DS.%26aulast%3DAlexander%26aufirst%3DW.%26aulast%3DMarto%26aufirst%3DJ.%26atitle%3DMzStudio%253A%2520a%2520Dynamic%2520Digital%2520Canvas%2520for%2520User-Driven%2520Interrogation%2520of%2520Mass%2520Spectrometry%2520Data%26jtitle%3DProteomes%26date%3D2017%26volume%3D5%26spage%3D20%26doi%3D10.3390%2Fproteomes5030020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LovÃ©n, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigova, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levens, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Revisiting Global Gene Expression Analysis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">476</span>â <span class="NLM_lpage">482</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.cell.2012.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=23101621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2htrzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=476-482&author=J.+Lov%C3%A9nauthor=D.+A.+Orlandoauthor=A.+A.+Sigovaauthor=C.+Y.+Linauthor=P.+B.+Rahlauthor=C.+B.+Burgeauthor=D.+L.+Levensauthor=T.+I.+Leeauthor=R.+A.+Young&title=Revisiting+Global+Gene+Expression+Analysis&doi=10.1016%2Fj.cell.2012.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Revisiting global gene expression analysis</span></div><div class="casAuthors">Loven, Jakob; Orlando, David A.; Sigova, Alla A.; Lin, Charles Y.; Rahl, Peter B.; Burge, Christopher B.; Levens, David L.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">476-482</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Gene expression anal. is a widely used and powerful method for investigating the transcriptional behavior of biol. systems, for classifying cell states in disease, and for many other purposes.  Recent studies indicate that common assumptions currently embedded in exptl. and anal. practices can lead to misinterpretation of global gene expression data.  We discuss these assumptions and describe solns. that should minimize erroneous interpretation of gene expression data from multiple anal. platforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9M9Rf1bUtyLVg90H21EOLACvtfcHk0lhcHRdd0hytxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2htrzJ&md5=ff82b942e83cf986cf8e678e31e622eb</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DLov%25C3%25A9n%26aufirst%3DJ.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DSigova%26aufirst%3DA.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DLevens%26aufirst%3DD.%2BL.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DRevisiting%2520Global%2520Gene%2520Expression%2520Analysis%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D476%26epage%3D482%26doi%3D10.1016%2Fj.cell.2012.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">AndrÃ¡s Gy.  NÃ©meth</span>, <span class="hlFld-ContribAuthor ">Bence  Marlok</span>, <span class="hlFld-ContribAuthor ">Attila  DomjÃ¡n</span>, <span class="hlFld-ContribAuthor ">Qinghe  Gao</span>, <span class="hlFld-ContribAuthor ">Xinya  Han</span>, <span class="hlFld-ContribAuthor ">GyÃ¶rgy M.  KeserÅ±</span>, <span class="hlFld-ContribAuthor ">PÃ©ter  ÃbrÃ¡nyiâBalogh</span>. </span><span class="cited-content_cbyCitation_article-title">Convenient Multicomponent OneâPot Synthesis of 2âIminothiazolines and 2âAminothiazoles Using Elemental Sulfur Under Aqueous Conditions. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>2021 </em>
                                    (25)
                                     , 3587-3597. <a href="https://doi.org/10.1002/ejoc.202100548" title="DOI URL">https://doi.org/10.1002/ejoc.202100548</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202100548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202100548%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DConvenient%252BMulticomponent%252BOne%2525E2%252580%252590Pot%252BSynthesis%252Bof%252B2%2525E2%252580%252590Iminothiazolines%252Band%252B2%2525E2%252580%252590Aminothiazoles%252BUsing%252BElemental%252BSulfur%252BUnder%252BAqueous%252BConditions%26aulast%3DN%25C3%25A9meth%26aufirst%3DAndr%25C3%25A1s%2BGy.%26date%3D2021%26date%3D2021%26volume%3D2021%26issue%3D25%26spage%3D3587%26epage%3D3597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Josef  Jansa</span>, <span class="hlFld-ContribAuthor ">Radek  Jorda</span>, <span class="hlFld-ContribAuthor ">Jana  Å kerlovÃ¡</span>, <span class="hlFld-ContribAuthor ">Petr  Pachl</span>, <span class="hlFld-ContribAuthor ">Miroslav  PeÅina</span>, <span class="hlFld-ContribAuthor ">Eva  ÅeznÃ­ÄkovÃ¡</span>, <span class="hlFld-ContribAuthor ">TomÃ¡Å¡  Heger</span>, <span class="hlFld-ContribAuthor ">TomÃ¡Å¡  GuckÃ½</span>, <span class="hlFld-ContribAuthor ">PavlÃ­na  ÅezÃ¡ÄovÃ¡</span>, <span class="hlFld-ContribAuthor ">AntonÃ­n  LyÄka</span>, <span class="hlFld-ContribAuthor ">VladimÃ­r  KryÅ¡tof</span>. </span><span class="cited-content_cbyCitation_article-title">Imidazo[1,2-c]pyrimidin-5(6H)-one inhibitors of CDK2: Synthesis, kinase inhibition and co-crystal structure. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>216 </em>, 113309. <a href="https://doi.org/10.1016/j.ejmech.2021.113309" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113309</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113309%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DImidazo%25255B1%25252C2-c%25255Dpyrimidin-5%2525286H%252529-one%252Binhibitors%252Bof%252BCDK2%25253A%252BSynthesis%25252C%252Bkinase%252Binhibition%252Band%252Bco-crystal%252Bstructure%26aulast%3DJansa%26aufirst%3DJosef%26date%3D2021%26volume%3D216%26spage%3D113309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Km  Kajol</span>, <span class="hlFld-ContribAuthor ">Chintakunta  Ramesh</span>. </span><span class="cited-content_cbyCitation_article-title">Sequential Oneâpot Method for the Synthesis of 4â(Hydroxymethyl)oxazoles and their Application in Phosphonates Synthesis. </span><span class="cited-content_cbyCitation_journal-name">Asian Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>10 </em>
                                    (3)
                                     , 649-654. <a href="https://doi.org/10.1002/ajoc.202000710" title="DOI URL">https://doi.org/10.1002/ajoc.202000710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ajoc.202000710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fajoc.202000710%26sid%3Dliteratum%253Aachs%26jtitle%3DAsian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSequential%252BOne%2525E2%252580%252590pot%252BMethod%252Bfor%252Bthe%252BSynthesis%252Bof%252B4%2525E2%252580%252590%252528Hydroxymethyl%252529oxazoles%252Band%252Btheir%252BApplication%252Bin%252BPhosphonates%252BSynthesis%26aulast%3DKajol%26aufirst%3DKm%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D3%26spage%3D649%26epage%3D654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Solomon  Tadesse</span>, <span class="hlFld-ContribAuthor ">Derek R  Duckett</span>, <span class="hlFld-ContribAuthor ">Andrii  Monastyrskyi</span>. </span><span class="cited-content_cbyCitation_article-title">The promise and current status of CDK12/13 inhibition for the treatment of cancer. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (2)
                                     , 117-141. <a href="https://doi.org/10.4155/fmc-2020-0240" title="DOI URL">https://doi.org/10.4155/fmc-2020-0240</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0240%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bpromise%252Band%252Bcurrent%252Bstatus%252Bof%252BCDK12%25252F13%252Binhibition%252Bfor%252Bthe%252Btreatment%252Bof%252Bcancer%26aulast%3DTadesse%26aufirst%3DSolomon%26date%3D2021%26volume%3D13%26issue%3D2%26spage%3D117%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of THZ1, THZ531, dinaciclib, and reported CDK12 inhibitors <b>2</b>â<b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Structure and biochemical IC<sub>50</sub> of SNS032. (B) Overlay of SNS032/CDK2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D1J">5D1J</a>, beige) and CDK12 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACB">5ACB</a>, blue) reveals that the piperidine ring could be extended toward Cys1039.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>1a</b>â<b>1c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIEA, DMF, 8%; (b) (i) DMAP, EDCI, triethylamine, DCM/DMF; (ii) 4N HCl, dioxane DCM, 16â83%; (c) HATU, 4-acrylamidobenzoic acid, DIEA, DMF, 12%; (d) acryloyl chloride, DIEA, THF, 0 Â°C, 51%.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>1d</b>â<b>1i</b>, <b>1k</b>, and <b>1l</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NMP, DIEA, 140 Â°C, 6â67%; (b) (i) 4 N HCl/dioxane, MeOH, (ii) 4-nitrobenzoyl chloride, Py, 27â55%; (c) (i) 4 N HCl/dioxane, MeOH, (ii) T<sub>3</sub>P, DIEA, DCM, 19â60%; (d) SnCl<sub>2</sub>, ethyl acetate/MeOH, 80 Â°C, 24â76%; (e) acryloyl chloride, DIEA, MeCN, 0 Â°C, 16â41%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>1j</b>, <b>1m</b>â<b>1o</b>, <b>1u</b>, and <b>1v</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KSCN, MeOH, 41â93%; (b) (i) NaBH<sub>4</sub>, ethanol, 0 Â°C to room temperature, (ii) R<sub>2</sub>CH<sub>2</sub>X, ethanol, 65 Â°C, 41â57%; (c) CuBr<sub>2</sub>, <i>t</i>-BuONO, MeCN, 0 Â°C, 37â94%; (d) 1-Boc-(<i>R</i>)-3-aminopiperidine, NMP, DIEA, 140 Â°C, 53â67%; (e) 4 N HCl/dioxane, MeOH, 51â95% ; (f) 4-acrylamidobenzoic acid, T<sub>3</sub>P, DIEA, DCM, 53% or (i) 5-nitropicolinic acid, T<sub>3</sub>P, DIEA, DCM, (ii) SnCl<sub>2</sub>, ethyl acetate/MeOH, 80 Â°C, (iii) acryloyl chloride, DIEA, MeCN, 0 Â°C, 7â17%.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>1t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KSCN, MeOH, 55%; (b) (i) NaBH<sub>4</sub>, ethanol, 0 Â°C to rt, (ii) 5-(<i>tert</i>-butyl)-2-(chloromethyl)oxazole, ethanol, 65 Â°C, 17%; (c) 1-Boc-(<i>R</i>)-3-aminopiperidine, NMP, DIEA, 140 Â°C, 92%; (d) (i) 4 N HCl/dioxane, MeOH, (ii) 4-nitrobenzoyl chloride, Py, 89%; (e) SnCl<sub>2</sub>, ethyl acetate/MeOH, 80 Â°C, 20%; (f) acryloyl chloride, DIEA, MeCN, 0 Â°C, 65%.</p></p></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>1p</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) triethyl phosphite, 120 Â°C, 39%; (b) <i>t</i>-BuOK, THF, 84%; (c) H<sub>2</sub>, Pd/C, MeOH, then 4 N HCl/dioxane, 90%; (d) CuBr<sub>2</sub>, <i>t</i>-BuONO, MeCN, 0 Â°C, 53%; (e) NMP, DIEA, 140 Â°C, 63%; (f) 4 N HCl/dioxane, MeOH, 88%; (g) 4-acrylamidobenzoic acid, HOBT, EDCI, DIEA, DMF, 16%.</p></p></figure><figure data-id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0012.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compounds <b>1q</b> and <b>1r</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOH, DMF, 120 Â°C, 45%; (b) NMP, DIEA, 140 Â°C, 22%; (c) THF, DIEA, 80 Â°C, 97%; (d) CoCl<sub>2</sub>â6H<sub>2</sub>O, NaBH<sub>4</sub>, ethanol, 28%; (e) NMP, DIEA, 80 Â°C, 58%; (f) (i) 4 N HCl/dioxane, MeOH, (ii) 4-acrylamidobenzoic acid, HOBT, EDCI, DIEA, DMF, 22â38%.</p></p></figure><figure data-id="fig3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. CDK12 is exclusively labeled by MFH290 at Cys1039. (A) Mass spectra and (B) zero-charge mass spectra of CDK12/cyclin K complex treated with (top) DMSO or (bottom) a 10-fold excess of MFH290 for 1 h at room temperature. (A) Mass spectral peaks corresponding to cyclin K, unmodified CDK12, and CDK12 covalently labeled with MFH290 are indicated with black, green, and red circles, respectively. (B, left) Cyclin K is unaffected by compound treatment, while (B, right) CDK12 protein exhibits an increase in mass consistent with modification by a single molecule of MFH290. (C) MS/MS spectrum of CDK12 peptide <sup>1025</sup>LSKMAPPDLPHWQDC HE<sub>1041</sub> modified with MFH290 indicates covalent labeling of Cys1039. Ions of types b and y are indicated with blue and red glyphs, respectively. Inhibitor related ions 1<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> are marked with green glyphs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0004.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Kinase profiling of MFH290. The levels of binding were measured at 1 Î¼M against a panel of 468 human kinases. The results for the binding interactions are reported as blue circles in the KiNOME tree. (B) IC<sub>50</sub> values for MFH290 against CDK1â9, 12â14, and 16: <sup><i>a</i></sup>tested in Zâ²-Lyte assays with ATP concentrations of <i>K</i><sub>M</sub>; <sup><i>b</i></sup>tested in LanthaScreen Eu kinase binding assay with ATP concentrations of <i>K</i><sub>M</sub>; <sup><i>c</i></sup>tested using Adapta assays with ATP concentrations at the apparent <i>K</i><sub>M</sub>; <sup><i>d</i></sup>tested in radioisotope filter binding assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) MFH290 targets CDK12 and CDK13 in cells. Treatment of HAP1 cells with MFH290 for 6 h blocks pulldown of CDK12 and associated cyclin K by biotinylated THZ1. (B) Antiproliferation assay using WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells. Output is CellTiterGlo luminescent signal following 72 h treatment with MFH290. (C) MFH290 reduces Pol II CTD phosphorylation in a concentration-dependent manner. Shown are effects of MFH290 on CTD phosphorylation in WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells after 6 h of treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0006.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) MFH290 treatment leads to a reduction in DNA damage response proteins. Shown are Western blots of WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells following treatment with MFH290 for 24 h. (B) MFH290 treatment leads to reduced expression of DNA damage response genes. Shown are RT-qPCR of WT, CDK12 C1039S, CDK13 C1017S, and CDK12 C1039S/CDK13 C1017S doubly mutated HAP1 cells following treatment with MFH290 for 6 h. (C) Excess over Bliss synergy plots for serial dilutions of MFH290 in combination with olaparib in HAP1 cells. Excess over Bliss scores >0 indicates drug synergy, whereas negative scores indicate antagonism.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/medium/jm9b01929_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0019.jpeg" id="rightTab-GRAPHIC-d7e3205-autogenerated" class="rightTab-fig internalNav" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.9b01929/20200701/images/large/jm9b01929_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01929&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i70">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 28 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">Cell Cycle, CDKs and Cancer: a Changing Paradigm</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">153</span>â <span class="NLM_lpage">166</span>, <span class="refDoi">Â DOI: 10.1038/nrc2602</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnrc2602" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=19238148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1ans78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=153-166&author=M.+Malumbresauthor=M.+Barbacid&title=Cell+Cycle%2C+CDKs+and+Cancer%3A+a+Changing+Paradigm&doi=10.1038%2Fnrc2602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cell cycle, CDKs and cancer: a changing paradigm</span></div><div class="casAuthors">Malumbres, Marcos; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">153-166</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Tumor-assocd. cell cycle defects are often mediated by alterations in cyclin-dependent kinase (CDK) activity.  Misregulated CDKs induce unscheduled proliferation as well as genomic and chromosomal instability.  According to current models, mammalian CDKs are essential for driving each cell cycle phase, so therapeutic strategies that block CDK activity are unlikely to selectively target tumor cells.  However, recent genetic evidence has revealed that, whereas CDK1 is required for the cell cycle, interphase CDKs are only essential for proliferation of specialized cells.  Emerging evidence suggests that tumor cells may also require specific interphase CDKs for proliferation.  Thus, selective CDK inhibition may provide therapeutic benefit against certain human neoplasias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzlIKlbCPWnLVg90H21EOLACvtfcHk0li8YBdx7ntQ5A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1ans78%253D&md5=2a7eb34b2381e288b3f43ec1bffc0aec</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrc2602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2602%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DCell%2520Cycle%252C%2520CDKs%2520and%2520Cancer%253A%2520a%2520Changing%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D153%26epage%3D166%26doi%3D10.1038%2Fnrc2602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hydbring, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span> <span> </span><span class="NLM_article-title">Non-Canonical Functions of Cell Cycle Cyclins and Cyclin-Dependent Kinases</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">280</span>â <span class="NLM_lpage">292</span>, <span class="refDoi">Â DOI: 10.1038/nrm.2016.27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnrm.2016.27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=27033256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XltlOksLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=280-292&author=P.+Hydbringauthor=M.+Malumbresauthor=P.+Sicinski&title=Non-Canonical+Functions+of+Cell+Cycle+Cyclins+and+Cyclin-Dependent+Kinases&doi=10.1038%2Fnrm.2016.27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Non-canonical functions of cell cycle cyclins and cyclin-dependent kinases</span></div><div class="casAuthors">Hydbring, Per; Malumbres, Marcos; Sicinski, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">280-292</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The roles of cyclins and their catalytic partners, the cyclin-dependent kinases (CDKs), as core components of the machinery that drives cell cycle progression are well established.  Increasing evidence indicates that mammalian cyclins and CDKs also carry out important functions in other cellular processes, such as transcription, DNA damage repair, control of cell death, differentiation, the immune response and metab.  Some of these non-canonical functions are performed by cyclins or CDKs, independently of their resp. cell cycle partners, suggesting that there was a substantial divergence in the functions of these proteins during evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqltFQjJuMXrbVg90H21EOLACvtfcHk0lhY9wHq4SOdmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltlOksLw%253D&md5=4d8aaf10c0542828112c561f5311f7da</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2016.27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2016.27%26sid%3Dliteratum%253Aachs%26aulast%3DHydbring%26aufirst%3DP.%26aulast%3DMalumbres%26aufirst%3DM.%26aulast%3DSicinski%26aufirst%3DP.%26atitle%3DNon-Canonical%2520Functions%2520of%2520Cell%2520Cycle%2520Cyclins%2520and%2520Cyclin-Dependent%2520Kinases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D17%26spage%3D280%26epage%3D292%26doi%3D10.1038%2Fnrm.2016.27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asghar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span> <span> </span><span class="NLM_article-title">The History and Future of Targeting Cyclin-Dependent Kinases in Cancer Therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">130</span>â <span class="NLM_lpage">146</span>, <span class="refDoi">Â DOI: 10.1038/nrd4504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnrd4504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=25633797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=130-146&author=U.+Asgharauthor=A.+K.+Witkiewiczauthor=N.+C.+Turnerauthor=E.+S.+Knudsen&title=The+History+and+Future+of+Targeting+Cyclin-Dependent+Kinases+in+Cancer+Therapy&doi=10.1038%2Fnrd4504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The history and future of targeting cyclin-dependent kinases in cancer therapy</span></div><div class="casAuthors">Asghar, Uzma; Witkiewicz, Agnieszka K.; Turner, Nicholas C.; Knudsen, Erik S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-146</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer represents a pathol. manifestation of uncontrolled cell division; therefore, it has long been anticipated that our understanding of the basic principles of cell cycle control would result in effective cancer therapies.  In particular, cyclin-dependent kinases (CDKs) that promote transition through the cell cycle were expected to be key therapeutic targets because many tumorigenic events ultimately drive proliferation by impinging on CDK4 or CDK6 complexes in the G1 phase of the cell cycle.  Moreover, perturbations in chromosomal stability and aspects of S phase and G2/M control mediated by CDK2 and CDK1 are pivotal tumorigenic events.  Translating this knowledge into successful clin. development of CDK inhibitors has historically been challenging, and numerous CDK inhibitors have demonstrated disappointing results in clin. trials.  Here, we review the biol. of CDKs, the rationale for therapeutically targeting discrete kinase complexes and historical clin. results of CDK inhibitors.  We also discuss how CDK inhibitors with high selectivity (particularly for both CDK4 and CDK6), in combination with patient stratification, have resulted in more substantial clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_utd-yiEZLVg90H21EOLACvtfcHk0lhY9wHq4SOdmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D&md5=113bcd2e99a8267119d180670d3c7761</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd4504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4504%26sid%3Dliteratum%253Aachs%26aulast%3DAsghar%26aufirst%3DU.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26atitle%3DThe%2520History%2520and%2520Future%2520of%2520Targeting%2520Cyclin-Dependent%2520Kinases%2520in%2520Cancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D130%26epage%3D146%26doi%3D10.1038%2Fnrd4504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amzallag, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannett, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Regulation in Cancer with a Covalent CDK7 Inhibitor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>511</i></span>,  <span class="NLM_fpage">616</span>â <span class="NLM_lpage">620</span>, <span class="refDoi">Â DOI: 10.1038/nature13393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnature13393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=25043025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=616-620&author=N.+Kwiatkowskiauthor=T.+Zhangauthor=P.+B.+Rahlauthor=B.+J.+Abrahamauthor=J.+Reddyauthor=S.+B.+Ficarroauthor=A.+Dasturauthor=A.+Amzallagauthor=S.+Ramaswamyauthor=B.+Tesarauthor=C.+E.+Jenkinsauthor=N.+M.+Hannettauthor=D.+McMillinauthor=T.+Sandaauthor=T.+Simauthor=N.+D.+Kimauthor=T.+Lookauthor=C.+S.+Mitsiadesauthor=A.+P.+Wengauthor=J.+R.+Brownauthor=C.+H.+Benesauthor=J.+A.+Martoauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Targeting+Transcription+Regulation+in+Cancer+with+a+Covalent+CDK7+Inhibitor&doi=10.1038%2Fnature13393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span></div><div class="casAuthors">Kwiatkowski, Nicholas; Zhang, Tinghu; Rahl, Peter B.; Abraham, Brian J.; Reddy, Jessica; Ficarro, Scott B.; Dastur, Anahita; Amzallag, Arnaud; Ramaswamy, Sridhar; Tesar, Bethany; Jenkins, Catherine E.; Hannett, Nancy M.; McMillin, Douglas; Sanda, Takaomi; Sim, Taebo; Kim, Nam Doo; Look, Thomas; Mitsiades, Constantine S.; Weng, Andrew P.; Brown, Jennifer R.; Benes, Cyril H.; Marto, Jarrod A.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7511</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor oncogenes include transcription factors that co-opt the general transcriptional machinery to sustain the oncogenic state, but direct pharmacol. inhibition of transcription factors has so far proven difficult.  However, the transcriptional machinery contains various enzymic cofactors that can be targeted for the development of new therapeutic candidates, including cyclin-dependent kinases (CDKs).  Here the authors present the discovery and characterization of a covalent CDK7 inhibitor, THZ1, which has the unprecedented ability to target a remote cysteine residue located outside of the canonical kinase domain, providing an unanticipated means of achieving selectivity for CDK7.  Cancer cell-line profiling indicates that a subset of cancer cell lines, including human T-cell acute lymphoblastic leukemia (T-ALL), have exceptional sensitivity to THZ1.  Genome-wide anal. in Jurkat T-ALL cells shows that THZ1 disproportionally affects transcription of RUNX1 and suggests that sensitivity to THZ1 may be due to vulnerability conferred by the RUNX1 super-enhancer and the key role of RUNX1 in the core transcriptional regulatory circuitry of these tumor cells.  Pharmacol. modulation of CDK7 kinase activity may thus provide an approach to identify and treat tumor types that are dependent on transcription for maintenance of the oncogenic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVD-DeTAQCqrVg90H21EOLACvtfcHk0lhY9wHq4SOdmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF&md5=895a28fb536663a31e7558fe79ce628b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature13393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13393%26sid%3Dliteratum%253Aachs%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DReddy%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DAmzallag%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DTesar%26aufirst%3DB.%26aulast%3DJenkins%26aufirst%3DC.%2BE.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DMcMillin%26aufirst%3DD.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLook%26aufirst%3DT.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWeng%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520Transcription%2520Regulation%2520in%2520Cancer%2520with%2520a%2520Covalent%2520CDK7%2520Inhibitor%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature13393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon-Clarke, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greifenberg, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannett, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkowiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Covalent Targeting of Remote Cysteine Residues to Develop CDK12 and CDK13 Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">876</span>â <span class="NLM_lpage">884</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnchembio.2166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=27571479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=876-884&author=T.+Zhangauthor=N.+Kwiatkowskiauthor=C.+M.+Olsonauthor=S.+E.+Dixon-Clarkeauthor=B.+J.+Abrahamauthor=A.+K.+Greifenbergauthor=S.+B.+Ficarroauthor=J.+M.+Elkinsauthor=Y.+Liangauthor=N.+M.+Hannettauthor=T.+Manzauthor=M.+Haoauthor=B.+Bartkowiakauthor=A.+L.+Greenleafauthor=J.+A.+Martoauthor=M.+Geyerauthor=A.+N.+Bullockauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Covalent+Targeting+of+Remote+Cysteine+Residues+to+Develop+CDK12+and+CDK13+Inhibitors&doi=10.1038%2Fnchembio.2166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent targeting of remote cysteine residues to develop CDK12 and CDK13 inhibitors</span></div><div class="casAuthors">Zhang, Tinghu; Kwiatkowski, Nicholas; Olson, Calla M.; Dixon-Clarke, Sarah E.; Abraham, Brian J.; Greifenberg, Ann K.; Ficarro, Scott B.; Elkins, Jonathan M.; Liang, Yanke; Hannett, Nancy M.; Manz, Theresa; Hao, Mingfeng; Bartkowiak, Bartlomiej; Greenleaf, Arno L.; Marto, Jarrod A.; Geyer, Matthias; Bullock, Alex N.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">876-884</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 12 and 13 (CDK12 and CDK13) play crit. roles in the regulation of gene transcription.  However, the absence of CDK12 and CDK13 inhibitors has hindered the ability to investigate the consequences of their inhibition in healthy cells and cancer cells.  Here we describe the rational design of a first-in-class CDK12 and CDK13 covalent inhibitor, THZ531.  Co-crystn. of THZ531 with CDK12-cyclin K indicates that THZ531 irreversibly targets a cysteine located outside the kinase domain.  THZ531 causes a loss of gene expression with concurrent loss of elongating and hyperphosphorylated RNA polymerase II.  In particular, THZ531 substantially decreases the expression of DNA damage response genes and key super-enhancer-assocd. transcription factor genes.  Coincident with transcriptional perturbation, THZ531 dramatically induced apoptotic cell death.  Small mols. capable of specifically targeting CDK12 and CDK13 may thus help identify cancer subtypes that are particularly dependent on their kinase activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXIyvy8vbjvrVg90H21EOLACvtfcHk0lgHTRi5grqJgw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ&md5=5ac265105d1654e129c3c1325572f389</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2166%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DDixon-Clarke%26aufirst%3DS.%2BE.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DManz%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DCovalent%2520Targeting%2520of%2520Remote%2520Cysteine%2520Residues%2520to%2520Develop%2520CDK12%2520and%2520CDK13%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D876%26epage%3D884%26doi%3D10.1038%2Fnchembio.2166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzyk, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichu, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollett, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, G. B.</span></span> <span> </span><span class="NLM_article-title">Interaction of Cyclin-Dependent Kinase 12/CrkRS with Cyclin K1 Is Required for the Phosphorylation of the C-terminal Domain of RNA Polymerase II</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">4691</span>â <span class="NLM_lpage">4704</span>, <span class="refDoi">Â DOI: 10.1128/MCB.06267-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1128%2FMCB.06267-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=22988298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1CjurrF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=4691-4704&author=S.+W.+Chengauthor=M.+A.+Kuzykauthor=A.+Moradianauthor=T.+A.+Ichuauthor=V.+C.+Changauthor=J.+F.+Tienauthor=S.+E.+Vollettauthor=M.+Griffithauthor=M.+A.+Marraauthor=G.+B.+Morin&title=Interaction+of+Cyclin-Dependent+Kinase+12%2FCrkRS+with+Cyclin+K1+Is+Required+for+the+Phosphorylation+of+the+C-terminal+Domain+of+RNA+Polymerase+II&doi=10.1128%2FMCB.06267-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of cyclin-dependent kinase 12/CrkRS with cyclin K1 is required for the phosphorylation of the C-terminal domain of RNA polymerase II</span></div><div class="casAuthors">Cheng, S.-W. Grace; Kuzyk, Michael A.; Moradian, Annie; Ichu, Taka-Aki; Chang, Vicky C.-D.; Tien, Jerry F.; Vollett, Sarah E.; Griffith, Malachi; Marra, Marco A.; Morin, Gregg B.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">4691-4704</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">CrkRS (Cdc2-related kinase, Arg/Ser), or cyclin-dependent kinase 12 (CKD12), is a serine/threonine kinase believed to coordinate transcription and RNA splicing.  While CDK12/CrkRS complexes were known to phosphorylate the C-terminal domain (CTD) of RNA polymerase II (RNA Pol II), the cyclin regulating this activity was not known.  Using immunopptn. and mass spectrometry, we identified a 65-kDa isoform of cyclin K (cyclin K1) in endogenous CDK12/CrkRS protein complexes.  We show that cyclin K1 complexes isolated from mammalian cells contain CDK12/CrkRS but do not contain CDK9, a presumed partner of cyclin K.  Anal. of extensive RNA-Seq data shows that the 65-kDa cyclin K1 isoform is the predominantly expressed form across numerous tissue types.  We also demonstrate that CDK12/CrkRS is dependent on cyclin K1 for its kinase activity and that small interfering RNA (siRNA) knockdown of CDK12/CrkRS or cyclin K1 has similar effects on the expression of a luciferase reporter gene.  Our data suggest that cyclin K1 is the primary cyclin partner for CDK12/CrkRS and that cyclin K1 is required to activate CDK12/CrkRS to phosphorylate the CTD of RNA Pol II.  These properties are consistent with a role of CDK12/CrkRS in regulating gene expression through phosphorylation of RNA Pol II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolvJU3IDa9NrVg90H21EOLACvtfcHk0lgHTRi5grqJgw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1CjurrF&md5=4b90352053b987a1857bf2e54e1e231a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1128%2FMCB.06267-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06267-11%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DS.%2BW.%26aulast%3DKuzyk%26aufirst%3DM.%2BA.%26aulast%3DMoradian%26aufirst%3DA.%26aulast%3DIchu%26aufirst%3DT.%2BA.%26aulast%3DChang%26aufirst%3DV.%2BC.%26aulast%3DTien%26aufirst%3DJ.%2BF.%26aulast%3DVollett%26aufirst%3DS.%2BE.%26aulast%3DGriffith%26aufirst%3DM.%26aulast%3DMarra%26aufirst%3DM.%2BA.%26aulast%3DMorin%26aufirst%3DG.%2BB.%26atitle%3DInteraction%2520of%2520Cyclin-Dependent%2520Kinase%252012%252FCrkRS%2520with%2520Cyclin%2520K1%2520Is%2520Required%2520for%2520the%2520Phosphorylation%2520of%2520the%2520C-terminal%2520Domain%2520of%2520RNA%2520Polymerase%2520II%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D4691%26epage%3D4704%26doi%3D10.1128%2FMCB.06267-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutor, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntoon, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karnitz, L. M.</span></span> <span> </span><span class="NLM_article-title">Ovarian Cancer-Associated Mutations Disable Catalytic Activity of CDK12, a Kinase That Promotes Homologous Recombination Repair and Resistance to Cisplatin and Poly (ADP-ribose) polymerase Inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">9247</span>â <span class="NLM_lpage">9253</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M114.551143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1074%2Fjbc.M114.551143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=24554720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVSksrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9247-9253&author=P.+M.+Joshiauthor=S.+L.+Sutorauthor=C.+J.+Huntoonauthor=L.+M.+Karnitz&title=Ovarian+Cancer-Associated+Mutations+Disable+Catalytic+Activity+of+CDK12%2C+a+Kinase+That+Promotes+Homologous+Recombination+Repair+and+Resistance+to+Cisplatin+and+Poly+%28ADP-ribose%29+polymerase+Inhibitors&doi=10.1074%2Fjbc.M114.551143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Ovarian Cancer-associated Mutations Disable Catalytic Activity of CDK12, a Kinase That Promotes Homologous Recombination Repair and Resistance to Cisplatin and Poly(ADP-ribose) Polymerase Inhibitors</span></div><div class="casAuthors">Joshi, Poorval M.; Sutor, Shari L.; Huntoon, Catherine J.; Karnitz, Larry M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9247-9253</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mutations in the tumor suppressors BRCA1 and BRCA2, which encode proteins that are key participants in homologous recombination (HR) repair, occur in â¼20% of high grade serous ovarian cancers.  Although only 20% of these tumors have mutations in BRCA1 and BRCA2, nearly 50% of these tumors have defects in HR.  Notably, however, the underlying genetic defects that give rise to HR defects in the absence of BRCA1 and BRCA2 mutations have not been fully elucidated.  Here we show that the recurrent somatic CDK12 mutations identified in ovarian cancers impair the catalytic activity of this kinase, which is involved in the transcription of a subset of genes, including BRCA1 and other DNA repair genes.  Furthermore, we show that disabling CDK12 function in ovarian cancer cells reduces BRCA1 levels, disrupts HR repair, and sensitizes these cells to the crosslinking agents melphalan and cisplatin and to the poly(ADP-ribose) polymerase (PARP) inhibitor veliparib (ABT-888).  Taken together, these findings suggest that many CDK12 mutations are an unrecognized cause of HR defects in ovarian cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTj3CX_FsX_rVg90H21EOLACvtfcHk0lhQK20aa8qOHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVSksrg%253D&md5=1ac9807318b6002c6134ef49751f3145</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.551143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.551143%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DP.%2BM.%26aulast%3DSutor%26aufirst%3DS.%2BL.%26aulast%3DHuntoon%26aufirst%3DC.%2BJ.%26aulast%3DKarnitz%26aufirst%3DL.%2BM.%26atitle%3DOvarian%2520Cancer-Associated%2520Mutations%2520Disable%2520Catalytic%2520Activity%2520of%2520CDK12%252C%2520a%2520Kinase%2520That%2520Promotes%2520Homologous%2520Recombination%2520Repair%2520and%2520Resistance%2520to%2520Cisplatin%2520and%2520Poly%2520%2528ADP-ribose%2529%2520polymerase%2520Inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D9247%26epage%3D9253%26doi%3D10.1074%2Fjbc.M114.551143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartkowiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, A. L.</span></span> <span> </span><span class="NLM_article-title">Engineering an Analog-Sensitive CDK12 Cell Line Using CRISPR/Cas</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1849</i></span>,  <span class="NLM_fpage">1179</span>â <span class="NLM_lpage">1187</span>, <span class="refDoi">Â DOI: 10.1016/j.bbagrm.2015.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.bbagrm.2015.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=26189575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1849&publication_year=2015&pages=1179-1187&author=B.+Bartkowiakauthor=C.+Yanauthor=A.+L.+Greenleaf&title=Engineering+an+Analog-Sensitive+CDK12+Cell+Line+Using+CRISPR%2FCas&doi=10.1016%2Fj.bbagrm.2015.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering an analog-sensitive CDK12 cell line using CRISPR/Cas</span></div><div class="casAuthors">Bartkowiak, Bartlomiej; Yan, Christopher; Greenleaf, Arno L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1849</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1179-1187</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The RNA Polymerase II C-terminal domain (CTD) kinase CDK12 has been implicated as a tumor suppressor and regulator of DNA damage response genes.  Although much has been learned about CDK12 and its activity, due to the lack of a specific inhibitor and the complications posed by long term RNAi depletion, much is still unknown about the particulars of CDK12 function.  Therefore gaining a better understanding of CDK12's roles at the mol. level will be challenging without the development of addnl. tools.  In order to address these issues we have used the CRISPR/Cas gene engineering system to create a mammalian cell line in which the only functional copy of CDK12 is selectively inhibitable by a cell-permeable adenine analog (analog-sensitive CDK12).  Inhibition of CDK12 results in a perturbation of the phosphorylation patterns on the CTD and an arrest in cellular proliferation.  This cell line should serve as a powerful tool for future studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWQ577ekAl_bVg90H21EOLACvtfcHk0lhQK20aa8qOHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Chsb3I&md5=297f75a0a71c579a01811cc387fc78a6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2015.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2015.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26atitle%3DEngineering%2520an%2520Analog-Sensitive%2520CDK12%2520Cell%2520Line%2520Using%2520CRISPR%252FCas%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2015%26volume%3D1849%26spage%3D1179%26epage%3D1187%26doi%3D10.1016%2Fj.bbagrm.2015.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajrami, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankum, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehman, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brough, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafiq, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozarewa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assiotis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenwick, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natrajan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Genome-Wide Profiling of Genetic Synthetic Lethality Identifies CDK12 as a Novel Determinant of PARP1/2 Inhibitor Sensitivity</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">287</span>â <span class="NLM_lpage">297</span>, <span class="refDoi">Â DOI: 10.1158/0008-5472.CAN-13-2541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1158%2F0008-5472.CAN-13-2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=24240700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2cXislKjuw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=287-297&author=I.+Bajramiauthor=J.+R.+Frankumauthor=A.+Kondeauthor=R.+E.+Millerauthor=F.+L.+Rehmanauthor=R.+Broughauthor=J.+Campbellauthor=D.+Simsauthor=R.+Rafiqauthor=S.+Hooperauthor=L.+Chenauthor=I.+Kozarewaauthor=I.+Assiotisauthor=K.+Fenwickauthor=R.+Natrajanauthor=C.+J.+Lordauthor=A.+Ashworth&title=Genome-Wide+Profiling+of+Genetic+Synthetic+Lethality+Identifies+CDK12+as+a+Novel+Determinant+of+PARP1%2F2+Inhibitor+Sensitivity&doi=10.1158%2F0008-5472.CAN-13-2541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide Profiling of Genetic Synthetic Lethality Identifies CDK12 as a Novel Determinant of PARP1/2 Inhibitor Sensitivity</span></div><div class="casAuthors">Bajrami, Ilirjana; Frankum, Jessica R.; Konde, Asha; Miller, Rowan E.; Rehman, Farah L.; Brough, Rachel; Campbell, James; Sims, David; Rafiq, Rumana; Hooper, Sean; Chen, Lina; Kozarewa, Iwanka; Assiotis, Ioannis; Fenwick, Kerry; Natrajan, Rachael; Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">287-297</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of PARP1/2, such as olaparib, have been proposed to serve as a synthetic lethal therapy for cancers that harbor BRCA1 or BRCA2 mutations.  Indeed, in clin. trials, PARP1/2 inhibitors elicit sustained antitumor responses in patients with germline BRCA gene mutations.  In hypothesizing that addnl. genetic determinants might direct use of these drugs, we conducted a genome-wide synthetic lethal screen for candidate olaparib sensitivity genes.  In support of this hypothesis, the set of identified genes included known determinants of olaparib sensitivity, such as BRCA1, RAD51, and Fanconi's anemia susceptibility genes.  In addn., the set included genes implicated in established networks of DNA repair, DNA cohesion, and chromatin remodeling, none of which were known previously to confer sensitivity to PARP1/2 inhibition.  Notably, integration of the list of candidate sensitivity genes with data from tumor DNA sequencing studies identified CDK12 deficiency as a clin. relevant biomarker of PARP1/2 inhibitor sensitivity.  In models of high-grade serous ovarian cancer (HGS-OVCa), CDK12 attenuation was sufficient to confer sensitivity to PARP1/2 inhibition, suppression of DNA repair via homologous recombination, and reduced expression of BRCA1.  As one of only nine genes known to be significantly mutated in HGS-OVCa, CDK12 has properties that should confirm interest in its use as a biomarker, particularly in ongoing clin. trials of PARP1/2 inhibitors and other agents that trigger replication fork arrest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNCJR_Nh_WjbVg90H21EOLACvtfcHk0lhQK20aa8qOHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXislKjuw%253D%253D&md5=9e07aa538aed0fb3a5ac6be21659d051</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-2541%26sid%3Dliteratum%253Aachs%26aulast%3DBajrami%26aufirst%3DI.%26aulast%3DFrankum%26aufirst%3DJ.%2BR.%26aulast%3DKonde%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DR.%2BE.%26aulast%3DRehman%26aufirst%3DF.%2BL.%26aulast%3DBrough%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DD.%26aulast%3DRafiq%26aufirst%3DR.%26aulast%3DHooper%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DKozarewa%26aufirst%3DI.%26aulast%3DAssiotis%26aufirst%3DI.%26aulast%3DFenwick%26aufirst%3DK.%26aulast%3DNatrajan%26aufirst%3DR.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DGenome-Wide%2520Profiling%2520of%2520Genetic%2520Synthetic%2520Lethality%2520Identifies%2520CDK12%2520as%2520a%2520Novel%2520Determinant%2520of%2520PARP1%252F2%2520Inhibitor%2520Sensitivity%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D287%26epage%3D297%26doi%3D10.1158%2F0008-5472.CAN-13-2541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greifenberg, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dust, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primack, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhardy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazaro, J.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochupurakkal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unitt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarosiek, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaltriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juric, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DâAndrea, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BalmaÃ±a, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span> <span> </span><span class="NLM_article-title">CDK12 Inhibition Reverses De Novo and Acquired PARP Inhibitor Resistance in BRCA Wild-Type and Mutated Models of Triple-Negative Breast Cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2367</span>â <span class="NLM_lpage">2381</span>, <span class="refDoi">Â DOI: 10.1016/j.celrep.2016.10.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.celrep.2016.10.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=27880910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGqt7%252FL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=2367-2381&author=S.+F.+Johnsonauthor=C.+Cruzauthor=A.+K.+Greifenbergauthor=S.+Dustauthor=D.+G.+Stoverauthor=D.+Chiauthor=B.+Primackauthor=S.+Caoauthor=A.+J.+Bernhardyauthor=R.+Coulsonauthor=J.-B.+Lazaroauthor=B.+Kochupurakkalauthor=H.+Sunauthor=C.+Unittauthor=L.+A.+Moreauauthor=K.+A.+Sarosiekauthor=M.+Scaltritiauthor=D.+Juricauthor=J.+Baselgaauthor=A.+L.+Richardsonauthor=S.+J.+Rodigauthor=A.+D.+D%E2%80%99Andreaauthor=J.+Balma%C3%B1aauthor=N.+Johnsonauthor=M.+Geyerauthor=V.+Serraauthor=E.+Limauthor=G.+I.+Shapiro&title=CDK12+Inhibition+Reverses+De+Novo+and+Acquired+PARP+Inhibitor+Resistance+in+BRCA+Wild-Type+and+Mutated+Models+of+Triple-Negative+Breast+Cancer&doi=10.1016%2Fj.celrep.2016.10.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">CDK12 Inhibition Reverses De Novo and Acquired PARP Inhibitor Resistance in BRCA Wild-Type and Mutated Models of Triple-Negative Breast Cancer</span></div><div class="casAuthors">Johnson, Shawn F.; Cruz, Cristina; Greifenberg, Ann Katrin; Dust, Sofia; Stover, Daniel G.; Chi, David; Primack, Benjamin; Cao, Shiliang; Bernhardy, Andrea J.; Coulson, Rhiannon; Lazaro, Jean-Bernard; Kochupurakkal, Bose; Sun, Heather; Unitt, Christine; Moreau, Lisa A.; Sarosiek, Kristopher A.; Scaltriti, Maurizio; Juric, Dejan; Baselga, Jose; Richardson, Andrea L.; Rodig, Scott J.; D'Andrea, Alan D.; Balmana, Judith; Johnson, Neil; Geyer, Matthias; Serra, Violeta; Lim, Elgene; Shapiro, Geoffrey I.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2367-2381</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Although poly(ADP-ribose) polymerase (PARP) inhibitors are active in homologous recombination (HR)-deficient cancers, their utility is limited by acquired resistance after restoration of HR.  Here, we report that dinaciclib, an inhibitor of cyclin-dependent kinases (CDKs) 1, 2, 5, and 9, addnl. has potent activity against CDK12, a transcriptional regulator of HR.  In BRCA-mutated triple-neg. breast cancer (TNBC) cells and patient-derived xenografts (PDXs), dinaciclib ablates restored HR and reverses PARP inhibitor resistance.  Addnl., we show that de novo resistance to PARP inhibition in BRCA1-mutated cell lines and a PDX derived from a PARP-inhibitor-naive BRCA1 carrier is mediated by residual HR and is reversed by CDK12 inhibition.  Finally, dinaciclib augments the degree of response in a PARP-inhibitor-sensitive model, converting tumor growth inhibition to durable regression.  These results highlight the significance of HR disruption as a therapeutic strategy and support the broad use of combined CDK12 and PARP inhibition in TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyVKyKNc3G1bVg90H21EOLACvtfcHk0li3vbEMeSqWQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGqt7%252FL&md5=ae666e75b7a49d0c348789e2d59643fa</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.10.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.10.077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BF.%26aulast%3DCruz%26aufirst%3DC.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DDust%26aufirst%3DS.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DChi%26aufirst%3DD.%26aulast%3DPrimack%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DS.%26aulast%3DBernhardy%26aufirst%3DA.%2BJ.%26aulast%3DCoulson%26aufirst%3DR.%26aulast%3DLazaro%26aufirst%3DJ.-B.%26aulast%3DKochupurakkal%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DUnitt%26aufirst%3DC.%26aulast%3DMoreau%26aufirst%3DL.%2BA.%26aulast%3DSarosiek%26aufirst%3DK.%2BA.%26aulast%3DScaltriti%26aufirst%3DM.%26aulast%3DJuric%26aufirst%3DD.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DRichardson%26aufirst%3DA.%2BL.%26aulast%3DRodig%26aufirst%3DS.%2BJ.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DA.%2BD.%26aulast%3DBalma%25C3%25B1a%26aufirst%3DJ.%26aulast%3DJohnson%26aufirst%3DN.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DSerra%26aufirst%3DV.%26aulast%3DLim%26aufirst%3DE.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26atitle%3DCDK12%2520Inhibition%2520Reverses%2520De%2520Novo%2520and%2520Acquired%2520PARP%2520Inhibitor%2520Resistance%2520in%2520BRCA%2520Wild-Type%2520and%2520Mutated%2520Models%2520of%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D17%26spage%3D2367%26epage%3D2381%26doi%3D10.1016%2Fj.celrep.2016.10.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paruch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doll, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keertikar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKittrick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hruza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madison, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomeir, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschmeier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sgambellone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seghezzi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiswell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paradkar, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokosz, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stauffer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzi, T. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Dinaciclib (SCH 727965): a Potent and Selective Inhibitor of Cyclin-Dependent Kinases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">204</span>â <span class="NLM_lpage">208</span>, <span class="refDoi">Â DOI: 10.1021/ml100051d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml100051d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=204-208&author=K.+Paruchauthor=M.+P.+Dwyerauthor=C.+Alvarezauthor=C.+Brownauthor=T.-Y.+Chanauthor=R.+J.+Dollauthor=K.+Keertikarauthor=C.+Knutsonauthor=B.+McKittrickauthor=J.+Riveraauthor=R.+Rossmanauthor=G.+Tuckerauthor=T.+Fischmannauthor=A.+Hruzaauthor=V.+Madisonauthor=A.+A.+Nomeirauthor=Y.+Wangauthor=P.+Kirschmeierauthor=E.+Leesauthor=D.+Parryauthor=N.+Sgambelloneauthor=W.+Seghezziauthor=L.+Schultzauthor=F.+Shanahanauthor=D.+Wiswellauthor=X.+Xuauthor=Q.+Zhouauthor=R.+A.+Jamesauthor=V.+M.+Paradkarauthor=H.+Parkauthor=L.+R.+Rokoszauthor=T.+M.+Staufferauthor=T.+J.+Guzi&title=Discovery+of+Dinaciclib+%28SCH+727965%29%3A+a+Potent+and+Selective+Inhibitor+of+Cyclin-Dependent+Kinases&doi=10.1021%2Fml100051d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Dinaciclib (SCH 727965): A Potent and Selective Inhibitor of Cyclin-Dependent Kinases</span></div><div class="casAuthors">Paruch, Kamil; Dwyer, Michael P.; Alvarez, Carmen; Brown, Courtney; Chan, Tin-Yau; Doll, Ronald J.; Keertikar, Kerry; Knutson, Chad; McKittrick, Brian; Rivera, Jocelyn; Rossman, Randall; Tucker, Greg; Fischmann, Thierry; Hruza, Alan; Madison, Vincent; Nomeir, Amin A.; Wang, Yaolin; Kirschmeier, Paul; Lees, Emma; Parry, David; Sgambellone, Nicole; Seghezzi, Wolfgang; Schultz, Lesley; Shanahan, Frances; Wiswell, Derek; Xu, Xiaoying; Zhou, Quiao; James, Ray A.; Paradkar, Vidyadhar M.; Park, Haengsoon; Rokosz, Laura R.; Stauffer, Tara M.; Guzi, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">204-208</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of cyclin-dependent kinases (CDKs) has emerged as an attractive strategy for the development of novel oncol. therapeutics.  Herein is described the utilization of an in vivo screening approach with integrated efficacy and tolerability parameters to identify candidate CDK inhibitors with a suitable balance of activity and tolerability.  This approach has resulted in the identification of SCH 727965, a potent and selective CDK inhibitor that is currently undergoing clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorrwrMq6CW6rVg90H21EOLACvtfcHk0li3vbEMeSqWQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtFWis78%253D&md5=3b982547e064f658748fe4887a208659</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fml100051d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml100051d%26sid%3Dliteratum%253Aachs%26aulast%3DParuch%26aufirst%3DK.%26aulast%3DDwyer%26aufirst%3DM.%2BP.%26aulast%3DAlvarez%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DT.-Y.%26aulast%3DDoll%26aufirst%3DR.%2BJ.%26aulast%3DKeertikar%26aufirst%3DK.%26aulast%3DKnutson%26aufirst%3DC.%26aulast%3DMcKittrick%26aufirst%3DB.%26aulast%3DRivera%26aufirst%3DJ.%26aulast%3DRossman%26aufirst%3DR.%26aulast%3DTucker%26aufirst%3DG.%26aulast%3DFischmann%26aufirst%3DT.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DMadison%26aufirst%3DV.%26aulast%3DNomeir%26aufirst%3DA.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DKirschmeier%26aufirst%3DP.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DParry%26aufirst%3DD.%26aulast%3DSgambellone%26aufirst%3DN.%26aulast%3DSeghezzi%26aufirst%3DW.%26aulast%3DSchultz%26aufirst%3DL.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DWiswell%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DQ.%26aulast%3DJames%26aufirst%3DR.%2BA.%26aulast%3DParadkar%26aufirst%3DV.%2BM.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DRokosz%26aufirst%3DL.%2BR.%26aulast%3DStauffer%26aufirst%3DT.%2BM.%26aulast%3DGuzi%26aufirst%3DT.%2BJ.%26atitle%3DDiscovery%2520of%2520Dinaciclib%2520%2528SCH%2520727965%2529%253A%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520Cyclin-Dependent%2520Kinases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D1%26spage%3D204%26epage%3D208%26doi%3D10.1021%2Fml100051d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denz, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impastato, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mlynarski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varnes, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haddrick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Selective Noncovalent CDK12 Inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">231</span>â <span class="NLM_lpage">235</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201700695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1002%2Fcmdc.201700695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=29266803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSnt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=231-235&author=J.+W.+Johannesauthor=C.+R.+Denzauthor=N.+Suauthor=A.+Wuauthor=A.+C.+Impastatoauthor=S.+Mlynarskiauthor=J.+G.+Varnesauthor=D.+B.+Princeauthor=J.+Cidadoauthor=N.+Gaoauthor=M.+Haddrickauthor=N.+H.+Jonesauthor=S.+Liauthor=X.+Liauthor=Y.+Liuauthor=T.+B.+Nguyenauthor=N.+O%E2%80%99Connellauthor=E.+Riversauthor=D.+W.+Robbinsauthor=R.+Tomlinsonauthor=T.+Yaoauthor=X.+Zhuauthor=A.+D.+Fergusonauthor=M.+L.+Lambauthor=J.+I.+Manchesterauthor=S.+Guichard&title=Structure-Based+Design+of+Selective+Noncovalent+CDK12+Inhibitors&doi=10.1002%2Fcmdc.201700695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Selective Noncovalent CDK12 Inhibitors</span></div><div class="casAuthors">Johannes, Jeffrey W.; Denz, Christopher R.; Su, Nancy; Wu, Allan; Impastato, Anna C.; Mlynarski, Scott; Varnes, Jeffrey G.; Prince, D. Bryan; Cidado, Justin; Gao, Ning; Haddrick, Malcolm; Jones, Natalie H.; Li, Shaobin; Li, Xiuwei; Liu, Yang; Nguyen, Toan B.; O'Connell, Nichole; Rivers, Emma; Robbins, Daniel W.; Tomlinson, Ronald; Yao, Tieguang; Zhu, Xiahui; Ferguson, Andrew D.; Lamb, Michelle L.; Manchester, John I.; Guichard, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-235</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase (CDK) 12 knockdown via siRNA decreases the transcription of DNA-damage-response genes and sensitizes BRCA wild-type cells to poly(ADP-ribose) polymerase (PARP) inhibition.  To recapitulate this effect with a small mol., we sought a potent, selective CDK12 inhibitor.  Crystal structures and modeling informed hybridization between dinaciclib and SR-3029, resulting in lead compd. 5 [(S)-2-(1-(6-(((6,7-difluoro-1H-benzo[d]imidazol-2-yl)methyl)amino)-9-ethyl-9H-purin-2-yl)piperidin-2-yl)ethan-1-ol].  Further structure-guided optimization delivered a series of selective CDK12 inhibitors, including compd. 7 [(S)-2-(1-(6-(((6,7-difluoro-1H-benzo[d]imidazol-2-yl)methyl)amino)-9-isopropyl-9H-purin-2-yl)piperidin-2-yl)ethan-1-ol].  Profiling of this compd. across CDK9, 7, 2, and 1 at high ATP concn., single-point kinase panel screening against 352 targets at 0.1 Î¼M, and proteomics via kinase affinity matrix technol. demonstrated the selectivity.  This series of compds. inhibits phosphorylation of Ser2 on the C-terminal repeat domain of RNA polymerase II, consistent with CDK12 inhibition.  These selective compds. were also acutely toxic to OV90 as well as THP1 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfuD5WxqOQa7Vg90H21EOLACvtfcHk0lhhkCs5uxi90A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSnt7k%253D&md5=02cb347d4e2b76347f531281e455449f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700695%26sid%3Dliteratum%253Aachs%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DDenz%26aufirst%3DC.%2BR.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DA.%26aulast%3DImpastato%26aufirst%3DA.%2BC.%26aulast%3DMlynarski%26aufirst%3DS.%26aulast%3DVarnes%26aufirst%3DJ.%2BG.%26aulast%3DPrince%26aufirst%3DD.%2BB.%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DHaddrick%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DN.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DRivers%26aufirst%3DE.%26aulast%3DRobbins%26aufirst%3DD.%2BW.%26aulast%3DTomlinson%26aufirst%3DR.%26aulast%3DYao%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DGuichard%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520Selective%2520Noncovalent%2520CDK12%2520Inhibitors%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D231%26epage%3D235%26doi%3D10.1002%2Fcmdc.201700695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokubo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morishita, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maezaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of 3-Benzyl-1-(trans-4-((5-cyanopyridin-2-yl) amino) cyclohexyl)-1-arylurea Derivatives as Novel and Selective Cyclin-Dependent Kinase 12 (CDK12) Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7710</span>â <span class="NLM_lpage">7728</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVWrsbjI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7710-7728&author=M.+Itoauthor=T.+Tanakaauthor=A.+Toitaauthor=N.+Uchiyamaauthor=H.+Kokuboauthor=N.+Morishitaauthor=M.+G.+Kleinauthor=H.+Zouauthor=M.+Murakamiauthor=M.+Kondoauthor=T.+Sameshimaauthor=S.+Arakiauthor=S.+Endoauthor=T.+Kawamotoauthor=G.+B.+Morinauthor=S.+A.+Aparicioauthor=A.+Nakanishiauthor=H.+Maezakiauthor=Y.+Imaeda&title=Discovery+of+3-Benzyl-1-%28trans-4-%28%285-cyanopyridin-2-yl%29+amino%29+cyclohexyl%29-1-arylurea+Derivatives+as+Novel+and+Selective+Cyclin-Dependent+Kinase+12+%28CDK12%29+Inhibitors&doi=10.1021%2Facs.jmedchem.8b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 3-Benzyl-1-(trans-4-((5-cyanopyridin-2-yl)amino)cyclohexyl)-1-arylurea Derivatives as Novel and Selective Cyclin-Dependent Kinase 12 (CDK12) Inhibitors</span></div><div class="casAuthors">Ito, Masahiro; Tanaka, Toshio; Toita, Akinori; Uchiyama, Noriko; Kokubo, Hironori; Morishita, Nao; Klein, Michael G.; Zou, Hua; Murakami, Morio; Kondo, Mitsuyo; Sameshima, Tomoya; Araki, Shinsuke; Endo, Satoshi; Kawamoto, Tomohiro; Morin, Gregg B.; Aparicio, Samuel A.; Nakanishi, Atsushi; Maezaki, Hironobu; Imaeda, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7710-7728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 12 (CDK12) plays a key role in the coordination of transcription with elongation and mRNA processing.  CDK12 mutations found in tumors and CDK12 inhibition sensitize cancer cells to DNA-damaging reagents and DNA-repair inhibitors.  This suggests that CDK12 inhibitors are potential therapeutics for cancer that may cause synthetic lethality.  Here, we report the discovery of 3-benzyl-1-(trans-4-((5-cyanopyridin-2-yl)amino)cyclohexyl)-1-arylurea derivs. as novel and selective CDK12 inhibitors.  Structure-activity relationship studies of a HTS hit, structure-based drug design, and conformation-oriented design using the Cambridge Structural Database afforded the optimized compd. 2, which exhibited not only potent CDK12 (and CDK13) inhibitory activity and excellent selectivity but also good physicochem. properties.  Furthermore, 2 inhibited the phosphorylation of Ser2 in the C-terminal domain of RNA polymerase II and induced growth inhibition in SK-BR-3 cells.  Therefore, 2 represents an excellent chem. probe for functional studies of CDK12 and could be a promising lead compd. for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXLhQTcUpBfLVg90H21EOLACvtfcHk0lhhkCs5uxi90A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVWrsbjI&md5=6d22fd36823d459fb60113c1212aa84e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00683%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DM.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DToita%26aufirst%3DA.%26aulast%3DUchiyama%26aufirst%3DN.%26aulast%3DKokubo%26aufirst%3DH.%26aulast%3DMorishita%26aufirst%3DN.%26aulast%3DKlein%26aufirst%3DM.%2BG.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DMurakami%26aufirst%3DM.%26aulast%3DKondo%26aufirst%3DM.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DAraki%26aufirst%3DS.%26aulast%3DEndo%26aufirst%3DS.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DMorin%26aufirst%3DG.%2BB.%26aulast%3DAparicio%26aufirst%3DS.%2BA.%26aulast%3DNakanishi%26aufirst%3DA.%26aulast%3DMaezaki%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%25203-Benzyl-1-%2528trans-4-%2528%25285-cyanopyridin-2-yl%2529%2520amino%2529%2520cyclohexyl%2529-1-arylurea%2520Derivatives%2520as%2520Novel%2520and%2520Selective%2520Cyclin-Dependent%2520Kinase%252012%2520%2528CDK12%2529%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7710%26epage%3D7728%26doi%3D10.1021%2Facs.jmedchem.8b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quereda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vena, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frydman, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monastyrskyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roush, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckett, D. R.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Targeting of CDK12/CDK13 in Triple-Negative Breast Cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">545</span>â <span class="NLM_lpage">558</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2019.09.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.ccell.2019.09.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=31668947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVGmtbvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=545-558&author=V.+Queredaauthor=S.+Bayleauthor=F.+Venaauthor=S.+M.+Frydmanauthor=A.+Monastyrskyiauthor=W.+R.+Roushauthor=D.+R.+Duckett&title=Therapeutic+Targeting+of+CDK12%2FCDK13+in+Triple-Negative+Breast+Cancer&doi=10.1016%2Fj.ccell.2019.09.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Targeting of CDK12/CDK13 in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Quereda, Victor; Bayle, Simon; Vena, Francesca; Frydman, Sylvia M.; Monastyrskyi, Andrii; Roush, William R.; Duckett, Derek R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">545-558.e7</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic regulation enables tumors to respond to changing environments during tumor progression and metastases and facilitates treatment resistance.  Targeting chromatin modifiers or catalytic effectors of transcription is an emerging anti-cancer strategy.  The cyclin-dependent kinases (CDKs) 12 and 13 phosphorylate the C-terminal domain of RNA polymerase II, regulating transcription and co-transcriptional processes.  Here we report the development of SR-4835, a highly selective dual inhibitor of CDK12 and CDK13, which disables triple-neg. breast cancer (TNBC) cells.  Mechanistically, inhibition or loss of CDK12/CDK13 triggers intronic polyadenylation site cleavage that suppresses the expression of core DNA damage response proteins.  This provokes a "BRCAness" phenotype that results in deficiencies in DNA damage repair, promoting synergy with DNA-damaging chemotherapy and PARP inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd0hwftXX1YrVg90H21EOLACvtfcHk0lhhkCs5uxi90A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVGmtbvM&md5=bd4a83717363acb4e6e33228727bfe01</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.09.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.09.004%26sid%3Dliteratum%253Aachs%26aulast%3DQuereda%26aufirst%3DV.%26aulast%3DBayle%26aufirst%3DS.%26aulast%3DVena%26aufirst%3DF.%26aulast%3DFrydman%26aufirst%3DS.%2BM.%26aulast%3DMonastyrskyi%26aufirst%3DA.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26aulast%3DDuckett%26aufirst%3DD.%2BR.%26atitle%3DTherapeutic%2520Targeting%2520of%2520CDK12%252FCDK13%2520in%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D545%26epage%3D558%26doi%3D10.1016%2Fj.ccell.2019.09.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Misra, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarz, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellar, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulheron, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batorsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roongta, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamath, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marathe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranadive, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sack, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavletich, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, F. Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimball, S. D.</span></span> <span> </span><span class="NLM_article-title"><i>N</i>-(Cycloalkylamino) acyl-2-aminothiazole Inhibitors of Cyclin-Dependent Kinase 2. <i>N</i>-[5-[[[5-(1, 1-dimethylethyl)-2-oxazolyl] methyl] thio]-2-thiazolyl]-4-piperidinecarboxamide (BMS-387032), a Highly Efficacious and Selective Antitumor Agent</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1719</span>â <span class="NLM_lpage">1728</span>, <span class="refDoi">Â DOI: 10.1021/jm0305568</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0305568" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1yls7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1719-1728&author=R.+N.+Misraauthor=H.-y.+Xiaoauthor=K.+S.+Kimauthor=S.+Luauthor=W.-C.+Hanauthor=S.+A.+Barbosaauthor=J.+T.+Huntauthor=D.+B.+Rawlinsauthor=W.+Shanauthor=S.+Z.+Ahmedauthor=L.+Qianauthor=B.-C.+Chenauthor=R.+Zhaoauthor=M.+S.+Bednarzauthor=K.+A.+Kellarauthor=J.+G.+Mulheronauthor=R.+Batorskyauthor=U.+Roongtaauthor=A.+Kamathauthor=P.+Maratheauthor=S.+A.+Ranadiveauthor=J.+S.+Sackauthor=J.+S.+Tokarskiauthor=N.+P.+Pavletichauthor=F.+Y.+F.+Leeauthor=K.+R.+Websterauthor=S.+D.+Kimball&title=N-%28Cycloalkylamino%29+acyl-2-aminothiazole+Inhibitors+of+Cyclin-Dependent+Kinase+2.+N-%5B5-%5B%5B%5B5-%281%2C+1-dimethylethyl%29-2-oxazolyl%5D+methyl%5D+thio%5D-2-thiazolyl%5D-4-piperidinecarboxamide+%28BMS-387032%29%2C+a+Highly+Efficacious+and+Selective+Antitumor+Agent&doi=10.1021%2Fjm0305568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">N-(cycloalkylamino)acyl-2-aminothiazole inhibitors of cyclin-dependent kinase 2. N-[5-[[[5-(1,1-dimethylethyl)-2-oxazolyl]methyl]thio]-2-thiazolyl]-4- piperidinecarboxamide (BMS-387032), a highly efficacious and selective antitumor agent</span></div><div class="casAuthors">Misra, Raj N.; Xiao, Hai-yun; Kim, Kyoung S.; Lu, Songfeng; Han, Wen-Ching; Barbosa, Stephanie A.; Hunt, John T.; Rawlins, David B.; Shan, Weifang; Ahmed, Syed Z.; Qian, Ligang; Chen, Bang-Chi; Zhao, Rulin; Bednarz, Mark S.; Kellar, Kristen A.; Mulheron, Janet G.; Batorsky, Roberta; Roongta, Urvashi; Kamath, Amrita; Marathe, Punit; Ranadive, Sunanda A.; Sack, John S.; Tokarski, John S.; Pavletich, Nikola P.; Lee, Francis Y. F.; Webster, Kevin R.; Kimball, S. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1719-1728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-Acyl-2-aminothiazoles with nonarom. acyl side chains contg. a basic amine were found to be potent, selective inhibitors of CDK2/cycE which exhibit antitumor activity in mice.  In particular, compd. 21 {N-[5-[[[5-(1,1-dimethylethyl)-2-oxazolyl]methyl]thio]-2-thiazolyl]-4-piperidinecarboxamide, BMS-387032} has been identified as an ATP-competitive and CDK2-selective inhibitor which has been selected to enter Phase 1 human clin. trials as an antitumor agent.  In a cell-free enzyme assay, 21 showed a CDK2/cycE IC50 = 48 nM and was 10- and 20-fold selective over CDK1/cycB and CDK4/cycD, resp.  It was also highly selective over a panel of 12 unrelated kinases.  Antiproliferative activity was established in an A2780 cellular cytotoxicity assay in which 21 showed an IC50 = 95 nM.  Metab. and pharmacokinetic studies showed that 21 exhibited a plasma half-life of 5-7 h in three species and moderately low protein binding in both mouse (69%) and human (63%) serum.  Dosed orally to mouse, rat, and dog, 21 showed 100%, 31%, and 28% bioavailability, resp.  As an antitumor agent in mice, 21 administered at its max.-tolerated dose exhibited a clearly superior efficacy profile when compared to flavopiridol in both an i.p. P388 murine tumor model and in a s.c./i.p. A2780 human ovarian carcinoma xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHnukOh_KXGLVg90H21EOLACvtfcHk0lhutkfEPWZ6rg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1yls7k%253D&md5=28cafa472df3e6562023f7d24f70ed45</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm0305568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0305568%26sid%3Dliteratum%253Aachs%26aulast%3DMisra%26aufirst%3DR.%2BN.%26aulast%3DXiao%26aufirst%3DH.-y.%26aulast%3DKim%26aufirst%3DK.%2BS.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DW.-C.%26aulast%3DBarbosa%26aufirst%3DS.%2BA.%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DRawlins%26aufirst%3DD.%2BB.%26aulast%3DShan%26aufirst%3DW.%26aulast%3DAhmed%26aufirst%3DS.%2BZ.%26aulast%3DQian%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DB.-C.%26aulast%3DZhao%26aufirst%3DR.%26aulast%3DBednarz%26aufirst%3DM.%2BS.%26aulast%3DKellar%26aufirst%3DK.%2BA.%26aulast%3DMulheron%26aufirst%3DJ.%2BG.%26aulast%3DBatorsky%26aufirst%3DR.%26aulast%3DRoongta%26aufirst%3DU.%26aulast%3DKamath%26aufirst%3DA.%26aulast%3DMarathe%26aufirst%3DP.%26aulast%3DRanadive%26aufirst%3DS.%2BA.%26aulast%3DSack%26aufirst%3DJ.%2BS.%26aulast%3DTokarski%26aufirst%3DJ.%2BS.%26aulast%3DPavletich%26aufirst%3DN.%2BP.%26aulast%3DLee%26aufirst%3DF.%2BY.%2BF.%26aulast%3DWebster%26aufirst%3DK.%2BR.%26aulast%3DKimball%26aufirst%3DS.%2BD.%26atitle%3DN-%2528Cycloalkylamino%2529%2520acyl-2-aminothiazole%2520Inhibitors%2520of%2520Cyclin-Dependent%2520Kinase%25202.%2520N-%255B5-%255B%255B%255B5-%25281%252C%25201-dimethylethyl%2529-2-oxazolyl%255D%2520methyl%255D%2520thio%255D-2-thiazolyl%255D-4-piperidinecarboxamide%2520%2528BMS-387032%2529%252C%2520a%2520Highly%2520Efficacious%2520and%2520Selective%2520Antitumor%2520Agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1719%26epage%3D1728%26doi%3D10.1021%2Fjm0305568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Card, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNally, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamberto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span> <span> </span><span class="NLM_article-title">Leveraging Gas-Phase Fragmentation Pathways for Improved Identification and Selective Detection of Targets Modified by Covalent Probes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">12248</span>â <span class="NLM_lpage">12254</span>, <span class="refDoi">Â DOI: 10.1021/acs.analchem.6b03394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.analchem.6b03394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2016&pages=12248-12254&author=S.+B.+Ficarroauthor=C.+M.+Browneauthor=J.+D.+Cardauthor=W.+M.+Alexanderauthor=T.+Zhangauthor=E.+Parkauthor=R.+McNallyauthor=S.+Dhe-Paganonauthor=H.-S.+Seoauthor=I.+Lambertoauthor=M.+J.+Eckauthor=S.+J.+Buhrlageauthor=N.+S.+Grayauthor=J.+A.+Marto&title=Leveraging+Gas-Phase+Fragmentation+Pathways+for+Improved+Identification+and+Selective+Detection+of+Targets+Modified+by+Covalent+Probes&doi=10.1021%2Facs.analchem.6b03394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Leveraging Gas-Phase Fragmentation Pathways for Improved Identification and Selective Detection of Targets Modified by Covalent Probes</span></div><div class="casAuthors">Ficarro, Scott B.; Browne, Christopher M.; Card, Joseph D.; Alexander, William M.; Zhang, Tinghu; Park, Eunyoung; McNally, Randall; Dhe-Paganon, Sirano; Seo, Hyuk-Soo; Lamberto, Ilaria; Eck, Michael J.; Buhrlage, Sara J.; Gray, Nathanael S.; Marto, Jarrod A.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">12248-12254</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent approval of covalent inhibitors for multiple clin. indications has reignited enthusiasm for this class of drugs.  As interest in covalent drugs has increased, so too has the need for anal. platforms that can leverage their mechanism-of-action to characterize modified protein targets.  Here the authors describe novel gas phase dissocn. pathways which yield predictable fragment ions during MS/MS of inhibitor-modified peptides.  The authors find that these dissocn. pathways are common to numerous cysteine-directed probes as well as the covalent drugs, Ibrutinib and Neratinib.  The authors leverage the predictable nature of these fragment ions to improve the confidence of peptide sequence assignment in proteomic analyses, and explore their potential use in selective mass spectrometry-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNu24Jo7-IWLVg90H21EOLACvtfcHk0lhutkfEPWZ6rg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGisrnO&md5=7932392d2d2ac88def4cacecbda41a30</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.analchem.6b03394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.analchem.6b03394%26sid%3Dliteratum%253Aachs%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DBrowne%26aufirst%3DC.%2BM.%26aulast%3DCard%26aufirst%3DJ.%2BD.%26aulast%3DAlexander%26aufirst%3DW.%2BM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DMcNally%26aufirst%3DR.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DSeo%26aufirst%3DH.-S.%26aulast%3DLamberto%26aufirst%3DI.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26atitle%3DLeveraging%2520Gas-Phase%2520Fragmentation%2520Pathways%2520for%2520Improved%2520Identification%2520and%2520Selective%2520Detection%2520of%2520Targets%2520Modified%2520by%2520Covalent%2520Probes%26jtitle%3DAnal.%2520Chem.%26date%3D2016%26volume%3D88%26spage%3D12248%26epage%3D12254%26doi%3D10.1021%2Facs.analchem.6b03394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ©lias, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A Small MoleculeâKinase Interaction Map for Clinical Kinase Inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>â <span class="NLM_lpage">336</span>, <span class="refDoi">Â DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.-M.+L%C3%A9liasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+Small+Molecule%E2%80%93Kinase+Interaction+Map+for+Clinical+Kinase+Inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lgoRbdjWlyLIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DL%25C3%25A9lias%26aufirst%3DJ.-M.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520Small%2520Molecule%25E2%2580%2593Kinase%2520Interaction%2520Map%2520for%2520Clinical%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A Quantitative Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+Quantitative+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lgoRbdjWlyLIg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520Quantitative%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartkowiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phatnani, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuda, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenleaf, A. L.</span></span> <span> </span><span class="NLM_article-title">CDK12 Is a Transcription Elongation-Associated CTD Kinase, the Metazoan Ortholog of Yeast Ctk1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2303</span>â <span class="NLM_lpage">2316</span>, <span class="refDoi">Â DOI: 10.1101/gad.1968210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1101%2Fgad.1968210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=20952539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlyqt7vM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2303-2316&author=B.+Bartkowiakauthor=P.+Liuauthor=H.+P.+Phatnaniauthor=N.+J.+Fudaauthor=J.+J.+Cooperauthor=D.+H.+Priceauthor=K.+Adelmanauthor=J.+T.+Lisauthor=A.+L.+Greenleaf&title=CDK12+Is+a+Transcription+Elongation-Associated+CTD+Kinase%2C+the+Metazoan+Ortholog+of+Yeast+Ctk1&doi=10.1101%2Fgad.1968210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CDK12 is a transcription elongation-associated CTD kinase, the metazoan ortholog of yeast Ctk1</span></div><div class="casAuthors">Bartkowiak, Bartlomiej; Liu, Pengda; Phatnani, Hemali P.; Fuda, Nicholas J.; Cooper, Jeffrey J.; Price, David H.; Adelman, Karen; Lis, John T.; Greenleaf, Arno L.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2303-2316</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Drosophila contains one (dCDK12) and humans contain two (hCDK12 and hCDK13) proteins that are the closest evolutionary relatives of yeast Ctk1, the catalytic subunit of the major elongation-phase C-terminal repeat domain (CTD) kinase in Saccharomyces cerevisiae, CTDK-I.  However, until now, neither CDK12 nor CDK13 has been demonstrated to be a bona fide CTD kinase.  Using Drosophila, the authors demonstrate that dCDK12 (CG7597) is a transcription-assocd. CTD kinase, the ortholog of yCtk1.  Fluorescence microscopy reveals that the distribution of dCDK12 on formaldehyde-fixed polytene chromosomes is virtually identical to that of hyperphosphorylated RNA polymerase II (RNAPII), but is distinct from that of P-TEFb (dCDK9 + dCyclin T).  Chromatin immunopptn. (ChIP) expts. confirm that dCDK12 is present on the transcribed regions of active Drosophila genes.  Compared with P-TEFb, dCDK12 amts. are lower at the 5' end and higher in the middle and at the 3' end of genes (both normalized to RNAPII).  Appropriately, Drosophila dCDK12 purified from nuclear exts. manifests CTD kinase activity in vitro.  Intriguingly, cyclin K is assocd. with purified dCDK12, implicating it as the cyclin subunit of this CTD kinase.  Most importantly, we demonstrate that RNAi knockdown of dCDK12 in S2 cells alters the phosphorylation state of the CTD, reducing its Ser2 phosphorylation levels.  Similarly, in human HeLa cells, hCDK13 purified from nuclear exts. displays CTD kinase activity in vitro, as anticipated.  Also, chimeric (yeast/human) versions of Ctk1 contg. the kinase homol. domains of hCDK12/13 (or hCDK9) are functional in yeast cells (and also in vitro).  This system was used to show that a bur1ts mutant is rescued more efficiently by a hCDK9 chimera than by a hCDK13 chimera, suggesting the following orthol. relationships: Bur1 â CDK9 and Ctk1 â CDK12/13.  Finally, siRNA knockdown of hCDK12 in HeLa cells results in alterations in the CTD phosphorylation state.  These findings demonstrate that metazoan CDK12 and CDK13 are CTD kinases, and that CDK12 is orthologous to yeast Ctk1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5PzvsoT0tWbVg90H21EOLACvtfcHk0ljT39c1b5D1Bw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlyqt7vM&md5=703994d04539e89a2efe59f8eb6c0457</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1101%2Fgad.1968210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1968210%26sid%3Dliteratum%253Aachs%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DPhatnani%26aufirst%3DH.%2BP.%26aulast%3DFuda%26aufirst%3DN.%2BJ.%26aulast%3DCooper%26aufirst%3DJ.%2BJ.%26aulast%3DPrice%26aufirst%3DD.%2BH.%26aulast%3DAdelman%26aufirst%3DK.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26atitle%3DCDK12%2520Is%2520a%2520Transcription%2520Elongation-Associated%2520CTD%2520Kinase%252C%2520the%2520Metazoan%2520Ortholog%2520of%2520Yeast%2520Ctk1%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2303%26epage%3D2316%26doi%3D10.1101%2Fgad.1968210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blazek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohoutek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholomeeusen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulinkova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimermancic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ule, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin, B. M.</span></span> <span> </span><span class="NLM_article-title">The Cyclin K/Cdk12 Complex Maintains Genomic Stability via Regulation of Expression of DNA Damage Response Genes. <i>Genes &</i></span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2158</span>â <span class="NLM_lpage">2172</span>, <span class="refDoi">Â DOI: 10.1101/gad.16962311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1101%2Fgad.16962311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=22012619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVant73E" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=2158-2172&author=D.+Blazekauthor=J.+Kohoutekauthor=K.+Bartholomeeusenauthor=E.+Johansenauthor=P.+Hulinkovaauthor=Z.+Luoauthor=P.+Cimermancicauthor=J.+Uleauthor=B.+M.+Peterlin&title=The+Cyclin+K%2FCdk12+Complex+Maintains+Genomic+Stability+via+Regulation+of+Expression+of+DNA+Damage+Response+Genes.+Genes+%26&doi=10.1101%2Fgad.16962311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The cyclin K/Cdk12 complex maintains genomic stability via regulation of expression of DNA damage response genes</span></div><div class="casAuthors">Blazek, Dalibor; Kohoutek, Jiri; Bartholomeeusen, Koen; Johansen, Eric; Hulinkova, Petra; Luo, Zeping; Cimermancic, Peter; Ule, Jernej; Peterlin, B. Matija</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2158-2172</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Various cyclin-dependent kinase (Cdk) complexes have been implicated in the regulation of transcription.  In this study, we identified a 70-kDa Cyclin K (CycK) that binds Cdk12 and Cdk13 to form two different complexes (CycK/Cdk12 or CycK/Cdk13) in human cells.  The CycK/Cdk12 complex regulates phosphorylation of Ser2 in the C-terminal domain of RNA polymerase II and expression of a small subset of human genes, as revealed in expression microarrays.  Depletion of CycK/Cdk12 results in decreased expression of predominantly long genes with high nos. of exons.  The most prominent group of down-regulated genes are the DNA damage response genes, including the crit. regulators of genomic stability: BRCA1 (breast and ovarian cancer type 1 susceptibility protein 1), ATR (ataxia telangiectasia and Rad3-related), FANCI, and FANCD2.  We show that CycK/Cdk12, rather than CycK/Cdk13, is necessary for their expression.  Nuclear run-on assays and chromatin immunopptns. with RNA polymerase II on the BRCA1 and FANCI genes suggest a transcriptional defect in the absence of CycK/Cdk12.  Consistent with these findings, cells without CycK/Cdk12 induce spontaneous DNA damage and are sensitive to a variety of DNA damage agents.  We conclude that through regulation of expression of DNA damage response genes, CycK/Cdk12 protects cells from genomic instability.  The essential role of CycK for organisms in vivo is further supported by the result that genetic inactivation of CycK in mice causes early embryonic lethality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPlifbvX1LXrVg90H21EOLACvtfcHk0ljT39c1b5D1Bw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVant73E&md5=4e472d0f324bbfae1a9d0a8a7302fa1d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1101%2Fgad.16962311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.16962311%26sid%3Dliteratum%253Aachs%26aulast%3DBlazek%26aufirst%3DD.%26aulast%3DKohoutek%26aufirst%3DJ.%26aulast%3DBartholomeeusen%26aufirst%3DK.%26aulast%3DJohansen%26aufirst%3DE.%26aulast%3DHulinkova%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DCimermancic%26aufirst%3DP.%26aulast%3DUle%26aufirst%3DJ.%26aulast%3DPeterlin%26aufirst%3DB.%2BM.%26atitle%3DThe%2520Cyclin%2520K%252FCdk12%2520Complex%2520Maintains%2520Genomic%2520Stability%2520via%2520Regulation%2520of%2520Expression%2520of%2520DNA%2520Damage%2520Response%2520Genes.%2520Genes%2520%2526%26jtitle%3DGenes%2520Dev.%26date%3D2011%26volume%3D25%26spage%3D2158%26epage%3D2172%26doi%3D10.1101%2Fgad.16962311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burness, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Trevino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harouaka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clouthier, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span> <span> </span><span class="NLM_article-title">RAD51 Mediates Resistance of Cancer Stem Cells to PARP Inhibition in Triple-Negative Breast Cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">514</span>â <span class="NLM_lpage">522</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-15-1348</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1158%2F1078-0432.CCR-15-1348" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=28034904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVeitLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=514-522&author=Y.+Liuauthor=M.+L.+Burnessauthor=R.+Martin-Trevinoauthor=J.+Guyauthor=S.+Baiauthor=R.+Harouakaauthor=M.+D.+Brooksauthor=L.+Shangauthor=A.+Foxauthor=T.+K.+Lutherauthor=A.+Davisauthor=T.+L.+Bakerauthor=J.+Colacinoauthor=S.+G.+Clouthierauthor=Z.-M.+Shaoauthor=M.+S.+Wichaauthor=S.+Liu&title=RAD51+Mediates+Resistance+of+Cancer+Stem+Cells+to+PARP+Inhibition+in+Triple-Negative+Breast+Cancer&doi=10.1158%2F1078-0432.CCR-15-1348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">RAD51 Mediates Resistance of Cancer Stem Cells to PARP Inhibition in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Liu, Yajing; Burness, Monika L.; Martin-Trevino, Rachel; Guy, Joey; Bai, Shoumin; Harouaka, Ramdane; Brooks, Michael D.; Shang, Li; Fox, Alex; Luther, Tahra K.; Davis, April; Baker, Trenton L.; Colacino, Justin; Clouthier, Shawn G.; Shao, Zhi-ming; Wicha, Max S.; Liu, Suling</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">514-522</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Introduction: PARP inhibitors have shown promising results in early studies for treatment of breast cancer susceptibility gene (BRCA)-deficient breast cancers; however, resistance ultimately develops.  Furthermore, the benefit of PARP inhibitors (PARPi) in triple-neg. breast cancers (TNBC) remains unknown.  Recent evidence indicates that in TNBCs, cells that display "cancer stem cell" properties are resistant to conventional treatments, mediate tumor metastasis, and contribute to recurrence.  The sensitivity of breast cancer stem cells (CSC) to PARPi is unknown.  Exptl. Design: We detd. the sensitivity of breast CSCs to PARP inhibition in BRCA1-mutant and -wild-type TNBC cell lines and tumor xenografts.  We also investigated the role of RAD51 in mediating CSC resistance to PARPi in these in vitro and in vivo models.  Results: We demonstrated that the CSCs in BRCA1-mutant TNBCs were resistant to PARP inhibition, and that these cells had both elevated RAD51 protein levels and activity.  Downregulation of RAD51 by shRNA sensitized CSCs to PARP inhibition and reduced tumor growth.  BRCA1-wild-type cells were relatively resistant to PARP inhibition alone, but redn. of RAD51 sensitized both CSC and bulk cells in these tumors to PARPi treatment.  Conclusions: Our data suggest that in both BRCA1-mutant and BRCA1-wild-type TNBCs, CSCs are relatively resistant to PARP inhibition.  This resistance is mediated by RAD51, suggesting that strategies aimed at targeting RAD51 may increase the therapeutic efficacy of PARPi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooTDtZrEs0trVg90H21EOLACvtfcHk0ljT39c1b5D1Bw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVeitLs%253D&md5=56dce7989dcf9edb314d3927adf180df</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-15-1348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-15-1348%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBurness%26aufirst%3DM.%2BL.%26aulast%3DMartin-Trevino%26aufirst%3DR.%26aulast%3DGuy%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DS.%26aulast%3DHarouaka%26aufirst%3DR.%26aulast%3DBrooks%26aufirst%3DM.%2BD.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DFox%26aufirst%3DA.%26aulast%3DLuther%26aufirst%3DT.%2BK.%26aulast%3DDavis%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DT.%2BL.%26aulast%3DColacino%26aufirst%3DJ.%26aulast%3DClouthier%26aufirst%3DS.%2BG.%26aulast%3DShao%26aufirst%3DZ.-M.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26aulast%3DLiu%26aufirst%3DS.%26atitle%3DRAD51%2520Mediates%2520Resistance%2520of%2520Cancer%2520Stem%2520Cells%2520to%2520PARP%2520Inhibition%2520in%2520Triple-Negative%2520Breast%2520Cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D514%26epage%3D522%26doi%3D10.1158%2F1078-0432.CCR-15-1348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bliss, C.</span></span> <span> </span><span class="NLM_article-title">The Calculation of Microbial Assays</span>. <i>Bacteriol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1956</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">243</span>â <span class="NLM_lpage">258</span>, <span class="refDoi">Â DOI: 10.1128/MMBR.20.4.243-258.1956</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1128%2FMMBR.20.4.243-258.1956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=13403845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A280%3ADyaG2s%252FltVWhtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1956&pages=243-258&author=C.+Bliss&title=The+Calculation+of+Microbial+Assays&doi=10.1128%2FMMBR.20.4.243-258.1956"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The calculation of microbial assays</span></div><div class="casAuthors">BLISS C I</div><div class="citationInfo"><span class="NLM_cas:title">Bacteriological reviews</span>
        (<span class="NLM_cas:date">1956</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">243-58</span>
        ISSN:<span class="NLM_cas:issn">0005-3678</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-6-02IQzk_jW1zwg2yUK5fW6udTcc2ebKEWqsOykWFrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaG2s%252FltVWhtQ%253D%253D&md5=f0b119a1b363e5273d4932ed9ef488a0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FMMBR.20.4.243-258.1956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.20.4.243-258.1956%26sid%3Dliteratum%253Aachs%26aulast%3DBliss%26aufirst%3DC.%26atitle%3DThe%2520Calculation%2520of%2520Microbial%2520Assays%26jtitle%3DBacteriol.%2520Rev.%26date%3D1956%26volume%3D20%26spage%3D243%26epage%3D258%26doi%3D10.1128%2FMMBR.20.4.243-258.1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greco, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, J. C.</span></span> <span> </span><span class="NLM_article-title">The Search for Synergy: A Critical Review from a Response Surface Perspective</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">331</span>â <span class="NLM_lpage">385</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=7568331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A280%3ADyaK28%252FjsFSgtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1995&pages=331-385&author=W.+R.+Grecoauthor=G.+Bravoauthor=J.+C.+Parsons&title=The+Search+for+Synergy%3A+A+Critical+Review+from+a+Response+Surface+Perspective"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The search for synergy: a critical review from a response surface perspective</span></div><div class="casAuthors">Greco W R; Bravo G; Parsons J C</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological reviews</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-85</span>
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRXywHk-SVAkdenUz0lpNqnfW6udTcc2ebKEWqsOykWFrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK28%252FjsFSgtQ%253D%253D&md5=80f1b0c8e20e587625a5d68339d30709</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreco%26aufirst%3DW.%2BR.%26aulast%3DBravo%26aufirst%3DG.%26aulast%3DParsons%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Search%2520for%2520Synergy%253A%2520A%2520Critical%2520Review%2520from%2520a%2520Response%2520Surface%2520Perspective%26jtitle%3DPharmacol.%2520Rev.%26date%3D1995%26volume%3D47%26spage%3D331%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choong, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafimova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockett, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheeti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fahr, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, U.</span></span> <span> </span><span class="NLM_article-title">A Diaminocyclohexyl Analog of SNS-032 with Improved Permeability and Bioavailability Properties</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5763</span>â <span class="NLM_lpage">5765</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2008.09.073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.bmcl.2008.09.073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=18842409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht12rs77M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=5763-5765&author=I.+C.+Choongauthor=I.+Serafimovaauthor=J.+Fanauthor=D.+Stockettauthor=E.+Chanauthor=S.+Cheetiauthor=Y.+Luauthor=B.+Fahrauthor=P.+Phamauthor=M.+R.+Arkinauthor=D.+H.+Walkerauthor=U.+Hoch&title=A+Diaminocyclohexyl+Analog+of+SNS-032+with+Improved+Permeability+and+Bioavailability+Properties&doi=10.1016%2Fj.bmcl.2008.09.073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A diaminocyclohexyl analog of SNS-032 with improved permeability and bioavailability properties</span></div><div class="casAuthors">Choong, Ingrid C.; Serafimova, Iana; Fan, Junfa; Stockett, David; Chan, Erica; Cheeti, Sravanthi; Lu, Yafan; Fahr, Bruce; Pham, Phuongly; Arkin, Michelle R.; Walker, Duncan H.; Hoch, Ute</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5763-5765</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The identification of a selective CDK2, 7, and 9 inhibitor (compd. 4) with improved permeability is described.  Compd. 4 exhibits comparable CDK selectivity profile to SNS-032, but shows improved permeability and higher bioavailability in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEqkByuQcxcbVg90H21EOLACvtfcHk0ljhgToY_oFxRw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht12rs77M&md5=d956b4c2e3b15b3d52ddd76f2cef997e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.09.073%26sid%3Dliteratum%253Aachs%26aulast%3DChoong%26aufirst%3DI.%2BC.%26aulast%3DSerafimova%26aufirst%3DI.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DStockett%26aufirst%3DD.%26aulast%3DChan%26aufirst%3DE.%26aulast%3DCheeti%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DFahr%26aufirst%3DB.%26aulast%3DPham%26aufirst%3DP.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWalker%26aufirst%3DD.%2BH.%26aulast%3DHoch%26aufirst%3DU.%26atitle%3DA%2520Diaminocyclohexyl%2520Analog%2520of%2520SNS-032%2520with%2520Improved%2520Permeability%2520and%2520Bioavailability%2520Properties%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D5763%26epage%3D5765%26doi%3D10.1016%2Fj.bmcl.2008.09.073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, A. G.</span></span> <span> </span><span class="NLM_article-title">A Universal Algorithm for Fast and Automated Charge State Deconvolution of Electrospray Mass-to-Charge Ratio Spectra</span>. <i>J. Am. Soc. Mass Spectrom.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">225</span>â <span class="NLM_lpage">233</span>, <span class="refDoi">Â DOI: 10.1016/S1044-0305(97)00284-5</span> </span><div class="citationLinks">[<a href="/doi/10.1016/S1044-0305%2897%2900284-5" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1998&pages=225-233&author=Z.+Zhangauthor=A.+G.+Marshall&title=A+Universal+Algorithm+for+Fast+and+Automated+Charge+State+Deconvolution+of+Electrospray+Mass-to-Charge+Ratio+Spectra&doi=10.1016%2FS1044-0305%2897%2900284-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A universal algorithm for fast and automated charge state deconvolution of electrospray mass-to-charge ratio spectra</span></div><div class="casAuthors">Zhang, Zhongqi; Marshall, Alan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">225-233</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">This article describes a new algorithm for charge state detn. and deconvolution of electrospray ionization (ESI) mass-to-charge ratio spectra.  The algorithm (ZSCORE) is based on a charge scoring scheme that incorporates all above-threshold members of a family of charge states or isotopic components, and deconvolves both low- and high-resoln. mass-to-charge ratio spectra, with or without a peak list (stick plot).  A scoring wt. factor, log (I/I0), in which I is the signal magnitude at a calcd. mass-to-charge ratio, and I0 is the signal threshold near that mass-to-charge ratio, was used in most cases.  For high-resoln. mass-to-charge ratio spectra in which all isotopic peaks are resolved, the algorithm can deconvolve overlapped isotopic multiplets of the same of different charge state.  Compared to other deconvolution techniques, the algorithm is robust, rapid, and fully automated (i.e., no user input during the deconvolution process).  It eliminates artifact peaks without introducing peak distortions.  Its performance is demonstrated for exptl. ESI Fourier transform ICR mass-to-charge ratio spectra (both low and high resoln.).  Charge state deconvolution to yield a zero-charge mass spectrum should prove particularly useful for interpreting spectra of complex mixts., identifying contaminants, noncovalent adducts, fragments (N-terminal, C-terminal, internal), and chem. modifications of electrosprayed biomacromols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-7hFdAcR_BbVg90H21EOLACvtfcHk0ljhgToY_oFxRw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtlynsbo%253D&md5=5bee6ce6556beed4ee9004cfe6c7ecad</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS1044-0305%2897%2900284-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1044-0305%252897%252900284-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DMarshall%26aufirst%3DA.%2BG.%26atitle%3DA%2520Universal%2520Algorithm%2520for%2520Fast%2520and%2520Automated%2520Charge%2520State%2520Deconvolution%2520of%2520Electrospray%2520Mass-to-Charge%2520Ratio%2520Spectra%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D1998%26volume%3D9%26spage%3D225%26epage%3D233%26doi%3D10.1016%2FS1044-0305%2897%2900284-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adelmant, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J.</span></span> <span> </span><span class="NLM_article-title">Multiplierz V2.0: A Python-Based Ecosystem for Shared Access and Analysis of Native Mass Spectrometry Data</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1700091</span>, <span class="refDoi">Â DOI: 10.1002/pmic.201700091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1002%2Fpmic.201700091" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=1700091&author=W.+M.+Alexanderauthor=S.+B.+Ficarroauthor=G.+Adelmantauthor=J.+Marto&title=Multiplierz+V2.0%3A+A+Python-Based+Ecosystem+for+Shared+Access+and+Analysis+of+Native+Mass+Spectrometry+Data&doi=10.1002%2Fpmic.201700091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201700091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201700091%26sid%3Dliteratum%253Aachs%26aulast%3DAlexander%26aufirst%3DW.%2BM.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DAdelmant%26aufirst%3DG.%26aulast%3DMarto%26aufirst%3DJ.%26atitle%3DMultiplierz%2520V2.0%253A%2520A%2520Python-Based%2520Ecosystem%2520for%2520Shared%2520Access%2520and%2520Analysis%2520of%2520Native%2520Mass%2520Spectrometry%2520Data%26jtitle%3DProteomics%26date%3D2017%26volume%3D17%26spage%3D1700091%26doi%3D10.1002%2Fpmic.201700091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J.</span></span> <span> </span><span class="NLM_article-title">MzStudio: a Dynamic Digital Canvas for User-Driven Interrogation of Mass Spectrometry Data</span>. <i>Proteomes</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">20</span>, <span class="refDoi">Â DOI: 10.3390/proteomes5030020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.3390%2Fproteomes5030020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkt1Sls7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=20&author=S.+Ficarroauthor=W.+Alexanderauthor=J.+Marto&title=MzStudio%3A+a+Dynamic+Digital+Canvas+for+User-Driven+Interrogation+of+Mass+Spectrometry+Data&doi=10.3390%2Fproteomes5030020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">mzStudio: a dynamic digital canvas for user-driven interrogation of mass spectrometry data</span></div><div class="casAuthors">Ficarro, Scott B.; Alexander, William M.; Marto, Jarrod A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteomes</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">20/1-20/8</span>CODEN:
                <span class="NLM_cas:coden">PROTHC</span>;
        ISSN:<span class="NLM_cas:issn">2227-7382</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Although not yet truly 'comprehensive', modern mass spectrometry-based expts. can generate quant. data for a meaningful fraction of the human proteome.  Importantly for large-scale protein expression anal., robust data pipelines are in place for identification of un-modified peptide sequences and aggregation of these data to protein-level quantification.  However, interoperable software tools that enable scientists to computationally explore and document novel hypotheses for peptide sequence, modification status, or fragmentation behavior are not well-developed.  Here, we introduce mzStudio, an open-source Python module built on our multiplierz project.  This desktop application provides a highly-interactive graphical user interface (GUI) through which scientists can examine and annotate spectral features, re-search existing PSMs to test different modifications or new spectral matching algorithms, share results with colleagues, integrate other domain-specific software tools, and finally create publication-quality graphics. mzStudio leverages our common application programming interface (mzAPI) for access to native data files from multiple instrument platforms, including ion trap, quadrupole time-of-flight, Orbitrap, matrix-assisted laser desorption ionization, and triple quadrupole mass spectrometers and is compatible with several popular search engines including Mascot, Proteome Discoverer, X!Tandem, and Comet.  The mzStudio toolkit enables researchers to create a digital provenance of data analytics and other evidence that support specific peptide sequence assignments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquRNKhFy0rcLVg90H21EOLACvtfcHk0ljOlfeht7Qb-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkt1Sls7w%253D&md5=9325e9aaf3183724de5f3f3e5ff4c9e0</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3390%2Fproteomes5030020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fproteomes5030020%26sid%3Dliteratum%253Aachs%26aulast%3DFicarro%26aufirst%3DS.%26aulast%3DAlexander%26aufirst%3DW.%26aulast%3DMarto%26aufirst%3DJ.%26atitle%3DMzStudio%253A%2520a%2520Dynamic%2520Digital%2520Canvas%2520for%2520User-Driven%2520Interrogation%2520of%2520Mass%2520Spectrometry%2520Data%26jtitle%3DProteomes%26date%3D2017%26volume%3D5%26spage%3D20%26doi%3D10.3390%2Fproteomes5030020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LovÃ©n, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigova, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahl, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burge, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levens, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Revisiting Global Gene Expression Analysis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">476</span>â <span class="NLM_lpage">482</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2012.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=10.1016%2Fj.cell.2012.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=23101621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2htrzJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=476-482&author=J.+Lov%C3%A9nauthor=D.+A.+Orlandoauthor=A.+A.+Sigovaauthor=C.+Y.+Linauthor=P.+B.+Rahlauthor=C.+B.+Burgeauthor=D.+L.+Levensauthor=T.+I.+Leeauthor=R.+A.+Young&title=Revisiting+Global+Gene+Expression+Analysis&doi=10.1016%2Fj.cell.2012.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Revisiting global gene expression analysis</span></div><div class="casAuthors">Loven, Jakob; Orlando, David A.; Sigova, Alla A.; Lin, Charles Y.; Rahl, Peter B.; Burge, Christopher B.; Levens, David L.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">476-482</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Gene expression anal. is a widely used and powerful method for investigating the transcriptional behavior of biol. systems, for classifying cell states in disease, and for many other purposes.  Recent studies indicate that common assumptions currently embedded in exptl. and anal. practices can lead to misinterpretation of global gene expression data.  We discuss these assumptions and describe solns. that should minimize erroneous interpretation of gene expression data from multiple anal. platforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9M9Rf1bUtyLVg90H21EOLACvtfcHk0ljOlfeht7Qb-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2htrzJ&md5=ff82b942e83cf986cf8e678e31e622eb</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DLov%25C3%25A9n%26aufirst%3DJ.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DSigova%26aufirst%3DA.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DBurge%26aufirst%3DC.%2BB.%26aulast%3DLevens%26aufirst%3DD.%2BL.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DRevisiting%2520Global%2520Gene%2520Expression%2520Analysis%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D476%26epage%3D482%26doi%3D10.1016%2Fj.cell.2012.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D1J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D1J','PDB','5D1J'); return false;">PDB: 5D1J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACB','PDB','5ACB'); return false;">PDB: 5ACB</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i66"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01929">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_36163"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01929?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01929</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Schematic of target engagement experiment (Figure S1A); known CDK inhibitors screened by target engagement assay (Figure S1B); structureâactivity relationship (SAR) screen of covalent compounds using target engagement assay (Figure S2); MFH290 targets CDK12 and CDK13 in cells (Figure S3); CRISPR technique mutates Cys-1039 to serine (C1039S) at CDK12 loci and Cys-1017 to serine (C1017S) at CDK13 loci (Figure S4A); Western blot of CDK12 and CDK13 WT and Cys-to-Ser mutants in HAP1 cells (Figure S4B); KinomeScan profiling data for MFH290 (Table S1); <sup>1</sup>H NMR spectrum of compound MFH290 (Figure S5A); HPLC spectrum of compound MFH290 (Figure S5B); LCâMS spectrum of compound MFH290 (Figure S5C) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_002.csv">CSV</a>)</p></li><li><p class="inline">Cocrystal structure of SNS032/CDK2 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D1J">5D1J</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Crystal structure of CDK12 transformed into CDK2 structure coordinate frame (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ACB">5ACB</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_004.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_001.pdf">jm9b01929_si_001.pdf (674.78 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_002.csv">jm9b01929_si_002.csv (6.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_003.pdb">jm9b01929_si_003.pdb (182.1 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01929/suppl_file/jm9b01929_si_004.pdb">jm9b01929_si_004.pdb (725.74 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01929&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-13%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01929%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01929" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679918894b323d7c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
