
<html lang="en"     class="pb-page"  data-request-id="519c7c4c-cd10-437b-b9f9-68c53b96b355"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.5b01632;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-4;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Design and Synthesis of Simplified Largazole Analogues as Isoform-Selective Human Lysine Deacetylase Inhibitors" /></meta><meta name="dc.Creator" content="Damodara N.  Reddy" /></meta><meta name="dc.Creator" content="Flavio  Ballante" /></meta><meta name="dc.Creator" content="Timothy  Chuang" /></meta><meta name="dc.Creator" content="Adele  Pirolli" /></meta><meta name="dc.Creator" content="Biagina  Marrocco" /></meta><meta name="dc.Creator" content="Garland R.  Marshall" /></meta><meta name="dc.Description" content="Selective inhibition of KDAC isoforms while maintaining potency remains a challenge. Using the largazole macrocyclic depsipeptide structure as a starting point for developing new KDACIs with increa..." /></meta><meta name="Description" content="Selective inhibition of KDAC isoforms while maintaining potency remains a challenge. Using the largazole macrocyclic depsipeptide structure as a starting point for developing new KDACIs with increa..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 7, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01632" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01632" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01632" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01632" /></link>
        
    
    

<title>Design and Synthesis of Simplified Largazole Analogues as Isoform-Selective Human Lysine Deacetylase Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01632" /></meta><meta property="og:title" content="Design and Synthesis of Simplified Largazole Analogues as Isoform-Selective Human Lysine Deacetylase Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0013.jpeg" /></meta><meta property="og:description" content="Selective inhibition of KDAC isoforms while maintaining potency remains a challenge. Using the largazole macrocyclic depsipeptide structure as a starting point for developing new KDACIs with increased selectivity, a combination of four different simplified largazole analogue (SLA) scaffolds with diverse zinc-binding groups (for a total of 60 compounds) were designed, synthesized, and evaluated against class I KDACs 1, 3, and 8, and class II KDAC6. Experimental evidence as well as molecular docking poses converged to establish the cyclic tetrapeptides (CTPs) as the primary determinant of both potency and selectivity by influencing the correct alignment of the zinc-binding group in the KDAC active site, providing a further basis for developing new KDACIs of higher isoform selectivity and potency." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01632"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01632">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01632&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01632&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01632&amp;href=/doi/10.1021/acs.jmedchem.5b01632" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1613-1633</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01517" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01708" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Synthesis of Simplified Largazole Analogues as Isoform-Selective Human Lysine Deacetylase Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Damodara+N.++Reddy">Damodara N. Reddy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Flavio++Ballante">Flavio Ballante</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Timothy++Chuang">Timothy Chuang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Adele++Pirolli">Adele Pirolli</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Biagina++Marrocco">Biagina Marrocco</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Garland+R.++Marshall">Garland R. Marshall</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Biochemistry and Molecular Biophysics, Washington University School of Medicine, 700 South Euclid Avenue, St. Louis, Missouri 63110, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Rome Center for Molecular Design, Dipartimento di Chimica e Tecnologie del Farmaco, Sapienza Università di Roma, P. le A. Moro 5, 00185 Roma, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Dipartimento di Chimica e Tecnologie del Farmaco, Sapienza Università di Roma, P. le A. Moro 5, 00185 Roma, Italy</span></div><div class="corresp-info"><strong>*</strong>Phone: +1-314-362-6039. Fax: 314-747-3330. E-mail: <a href="/cdn-cgi/l/email-protection#781f190a1419161c15381f15191114561b1715"><span class="__cf_email__" data-cfemail="4522243729242b2128052228242c296b262a28">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01632&amp;href=/doi/10.1021%2Facs.jmedchem.5b01632" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1613–1633</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 17, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 October 2015</li><li><span class="item_label"><b>Published</b> online</span>7 January 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 February 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01632" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01632</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1613%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDamodara%2BN.%2BReddy%252C%2BFlavio%2BBallante%252C%2BTimothy%2BChuang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D4%26contentID%3Dacs.jmedchem.5b01632%26title%3DDesign%2Band%2BSynthesis%2Bof%2BSimplified%2BLargazole%2BAnalogues%2Bas%2BIsoform-Selective%2BHuman%2BLysine%2BDeacetylase%2BInhibitors%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1633%26publicationDate%3DFebruary%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01632"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1916</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">20</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01632" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Synthesis of Simplified Largazole Analogues as Isoform-Selective Human Lysine Deacetylase Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Damodara&quot;,&quot;last_name&quot;:&quot;N. Reddy&quot;},{&quot;first_name&quot;:&quot;Flavio&quot;,&quot;last_name&quot;:&quot;Ballante&quot;},{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;Chuang&quot;},{&quot;first_name&quot;:&quot;Adele&quot;,&quot;last_name&quot;:&quot;Pirolli&quot;},{&quot;first_name&quot;:&quot;Biagina&quot;,&quot;last_name&quot;:&quot;Marrocco&quot;},{&quot;first_name&quot;:&quot;Garland&quot;,&quot;last_name&quot;:&quot;R. Marshall&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;1613-1633&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01632&quot;},&quot;abstract&quot;:&quot;Selective inhibition of KDAC isoforms while maintaining potency remains a challenge. Using the largazole macrocyclic depsipeptide structure as a starting point for developing new KDACIs with increased selectivity, a combination of four different simplified largazole analogue (SLA) scaffolds with diverse zinc-binding groups (for a total of 60 compounds) were designed, synthesized, and evaluated against class I KDACs 1, 3, and 8, and class II KDAC6. Experimental evidence as well as molecular docking poses converged to establish the cyclic tetrapeptides (CTPs) as the primary determinant of both potency and selectivity by influencing the correct alignment of the zinc-binding group in the KDAC active site, providing a further basis for developing new KDACIs of higher isoform selectivity and potency.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01632&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01632" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01632&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01632" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01632&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01632" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01632&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01632&amp;href=/doi/10.1021/acs.jmedchem.5b01632" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01632" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01632" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01632%26sid%3Dliteratum%253Aachs%26pmid%3D26681404%26genre%3Darticle%26aulast%3DReddy%26date%3D2016%26atitle%3DDesign%2Band%2BSynthesis%2Bof%2BSimplified%2BLargazole%2BAnalogues%2Bas%2BIsoform-Selective%2BHuman%2BLysine%2BDeacetylase%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D4%26spage%3D1613%26epage%3D1633%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/jmcmar.2016.59.issue-4/20160225/jmcmar.2016.59.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Selective inhibition of KDAC isoforms while maintaining potency remains a challenge. Using the largazole macrocyclic depsipeptide structure as a starting point for developing new KDACIs with increased selectivity, a combination of four different simplified largazole analogue (SLA) scaffolds with diverse zinc-binding groups (for a total of 60 compounds) were designed, synthesized, and evaluated against class I KDACs 1, 3, and 8, and class II KDAC6. Experimental evidence as well as molecular docking poses converged to establish the cyclic tetrapeptides (CTPs) as the primary determinant of both potency and selectivity by influencing the correct alignment of the zinc-binding group in the KDAC active site, providing a further basis for developing new KDACIs of higher isoform selectivity and potency.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/4"><issue-title>Epigenetics</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36280" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36280" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i2" class="anchor-spacer"></div><h3 class="article-section__title" id="_i2"> Lysine Deacetylases and Epigenetics</h3><div class="NLM_p">Lysine deacetylases (KDACs)<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> are a class of enzymes found in bacteria, fungi, plants, and animals that remove the acetyl group from the ε-amino groups of lysine residues in order to influence gene expression and modulate molecular recognition (epigenetics). Lysine acetyltransferases (KATs) catalyze the acetylation of histone tails causing localized relaxation of chromatin and transcriptional activation of nearby genes and many other proteins modulating signaling, while KDACs catalyze the deacetylation of acetylated histones and other proteins leading to transcriptional repression and modulation of functional responses. There are 18 known human lysine deacetylases. KDACs 1–11 are zinc-based enzymes and hydrolyze an amide bond similar to a variety of zinc proteases. KDAC inhibitors (KDACIs) are an emerging class of antitumor drugs. Three FDA-approved KDAC inhibitors: Vorinostat (suberoylanilide hydroxamic acid, SAHA),<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Romidepsin (FK228),<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and Belinostat (PXD101)<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> are indicated for treatment of peripheral T-cell lymphomas, in particular, cutaneous T-cell lymphoma, and other types of non-Hodgkin’s lymphoma.</div><div class="NLM_p last">The role of KDACs in pathology is ubiquitous. Besides oncology, KDACIs have shown potential therapeutic roles in addiction, asthma, cardiovascular disease, immunosuppression, neurodegenerative diseases, sepsis, sickle-cell disease, and termination of viral latency among others. It has become obvious that isoform-selective KDACIs are essential to help dissect the complex dynamics of epigenetic control of gene expression/signal transduction by lysine acetylation/deacetylation.</div></div><div id="sec1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> HIV-1 Latency</h3><div class="NLM_p last">As another example of potential therapeutic applications of lysine deacetylase inhibitors, KDACIs have been shown to reverse HIV-1 latency as a potential cure for patients infected with HIV to prevent subsequent development of AIDS.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> AIDS (acquired immunodeficiency syndrome) is a chronic, life-threatening condition due to infection by the human immunodeficiency virus (HIV-1). Highly active antiretroviral therapy (HAART) is an effective treatment to reduce HIV RNA in plasma to undetectable levels (generally defined as 50 copies/mL) which improves patients’ lives, prevents subsequent development of AIDS, and reduces the risk of transmission to others. Even after 15 years of viral suppression by HAART, however, HIV can still be harvested from resting T-cells,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and other cells<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> with latent HIV infection,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and viral rebound almost always occurs after HAART interruption.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Persistent proviral human immunodeficiency virus type 1 (HIV-1) infection, primarily within a small population of long-lived resting CD4+ T-cells,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> is a major obstacle to the eradication of HIV-1 infection. For this purpose, “shock and kill” strategies have been proposed by forcing HIV-infected T-cells to express their HIV genome<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> so that infected cells can be identified and eradicated to eliminate HIV infection. Such efforts will be aided by further understanding of epigenetic mechanisms that regulate transcription from the HIV-1 long terminal repeat (LTR) promoter and the role of Tat/TAR interactions.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> Evidence suggests that dynamic acetylation and methylation can regulate HIV-1 proviral expression.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> KDACs are recruited to the initiator and enhancer regions of the HIV-1 LTR by several transcription factors and corepressor complexes. More recently, it has been found that nonselective histone deacetylase inhibitors (pan KDACIs), such as SAHA,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Trichostatin A (TSA),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and FK228,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> have been shown to disrupt HIV latency and provides proof-of-principle,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> but potential toxicity remains a significant concern with any epigenetic therapeutic.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Furthermore, no current KDACI is completely specific for any of the 11 zinc-based HDAC isoforms, and there is no clinical experience to estimate the length of treatment necessary to eradicate the HIV reservoir. Recently, it has shown that KDAC inhibitors specific for a limited number of class I KDACs may offer a targeted approach to the disruption of persistent HIV-1 infection.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> Even if a vaccine was found to be effective, however, in preventing the spread of HIV, ∼34 million of HIV-positive patients worldwide remain to be cured of their infections. To address this potential therapeutic opportunity, however, isoform-selective KDACIs are essential to determine the mechanism of overcoming HIV latency and to minimize adverse side effects in the clinic. Furthermore, development of isoform-specific inhibitors would enable pharmacological dissection of the role of KDAC isoforms in both physiology as well as pathology.</div></div><div id="sec1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> KDAC Inhibitors</h3><div class="NLM_p">Many KDAC inhibitors have been reported, both naturally occurring such as TSA,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> apicidin,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> trapoxin,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> FK228,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> azumamide analogues,<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26-28)</a> largazole,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and a vast number of synthetic compounds, such as SAHA.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> These KDAC inhibitors (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) can be divided into categories according to their structural chemistry, such as hydroxamates, carboxylates, benzamides, and cyclic peptides, although most are hydroxamic acid derivatives.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> All classes of KDACIs contain three key structural elements: (i) a zinc-binding group (warhead) which coordinates the zinc ion at the bottom of the 12 Å narrow, active-site cavity, (ii) a capping or headgroup which interacts with the amino acids on the rim of the isoform’s binding cavity as the substrate–protein recognition surface, and (iii) a linker domain whose role is to ensure the correct positioning of the two former groups and to interact with the lipophilic binding tunnel (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The structures of naturally occurring, potent inhibitors of human KDACs often contain a cyclic tetrapeptide “headgroup” linked to a pendant aliphatic spacer with a terminal “warhead” functional group that interacts with zinc in the active site. Despite an intensive search for over a decade in both academia and industry and the discovery of selective KDAC8,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> KDAC3,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and KDAC6<a onclick="showRef(event, 'ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36">(34-36)</a> inhibitors, the development of new KDACIs endowed with higher potency and selectivity is still necessary to clarify the therapeutic effect of a single KDAC isoform inhibition.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of potent KDAC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic of KDAC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Design of Isoform-Selective KDAC Inhibitors</h3><div class="NLM_p last">Macrocyclic KDAC inhibitors (KDACIs) possess complex capping groups which interact with the KDAC enzyme’s outer rim and show excellent KDACI potency with some isoform selectivity.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The structure of largazole in complex with human and <i>Schistosoma mansoni</i> KDAC8s have been solved.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38, 39)</a> Comparison of the crystal structures of KDAC isoforms (2, 3, 4, 7, and 8) show that the length and conformation of the loops at the rim of the lysine-binding channel where the macrocyclic headgroups interact with the KDAC substrate are extremely variable and a likely source of enzyme/protein–substrate specificity. Overemphasis on targeting the zinc in the essentially identical active site of KDAC isoforms may explain the limited progress in obtaining isoform selectivity. More recently, KDACIs have been developed without a zinc-binding group to examine this strategy.<a onclick="showRef(event, 'ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42">(40-42)</a> Variability in the headgroup-binding region of KDAC isoforms provides one strategy for selectively targeting KDAC isoforms. KDACs target specific sites on substrate proteins; consequently, they must discriminate among the protein structures bearing the lysine residue to be modified. In naturally occurring KDACIs, macrocyclic tetrapeptides (CTPs) are located as capping groups at the recognition interface between the protein substrate and the KDAC. This strategy focuses on protein–substrate discrimination by different KDACs as seen in the example of the complex of largazole with human KDAC8<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and its homologue from <i>Schistosoma mansoni</i>.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Strategy for Isoform Selectivity in KDACIs</h3><div class="NLM_p">In this work, we have chosen to explore four different simplified largazole analogue (SLA) scaffolds with a wide variety of potential zinc-binding groups and directly compared their activities for a series of KDAC isoforms. Largazole thiol (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) is the most potent KDACI (picomolar against KDACs 1, 2, and 3) and selective to class I KDACs. Bowers et al. made one analogue ∼3–4-times as potent as largazole against KDAC1, 2, and 3 by substituting a pyridine group for one of the thiazole rings.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> In our search of isoform-selective KDACIs, largazole has been modified to simplify the synthesis of analogues with amino acid synthons as shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>. Replacing the thiazole ring with heterochiral proline dipeptides (Pro-<span class="smallcaps smallerCapital">d</span>-pro and <span class="smallcaps smallerCapital">d</span>-pro-Pro) that stabilize reverse turns<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45-47)</a> generated two CTP scaffolds. Preliminary conformational analysis of the simplified largazole analogues (SLAs) gave promising overlaps with the conformation of largazole bound to KDAC8<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). Further simplification replaced the unusual β-hydroxy acid bearing the thiol warhead with aspartic acid to which a variety of warheads could be linked by an amide bond when the <i>trans</i> olefin was replaced with an equivalent amide bond (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Replacement of the depsipeptide ester link in the largazole ring with an amide bond by Bowers et al. showed little differences in isoform selectivity and only a 4-fold reduction in affinity to 4 nM.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The proline rings of the SLA provide a semirigid structure for potential replacement with substituted prolines to further enhance specificity for KDAC isoforms. Varying the stereogenic center (Asp vs <span class="smallcaps smallerCapital">d</span>-asp) where the linker and warhead were attached might further optimize the orientation of the zinc-binding group. This proposed scaffold allowed combinatorial optimization of the CTP headgroup and the zinc-binding group by preassembly of the CTP headgroup followed by attachment of the linker-zinc-binding group through an amide bond. Optimization of both the linker and the zinc-binding group would hopefully generate maximal KDAC isoform specificity. Unfortunately, the simplified largazole analogues did not bind to KDACs in the same orientation as largazole itself based on the loss of inhibitory activity observed. A detectable KDAC6 inhibitory selectivity (among KDAC1, 3, 6, and 8) was measured for compound <b>25a</b> to help improve ligand–KDAC isoform recognition. Because of the loses of biological responses, an comprehensive docking assessment protocol has been developed and applied to the human lysine deacetylases (KDACs),<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> determining the best docking protocol to be used in rationalizing SLAs-KDACs interactions.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Original docking of largazole to KDAC8 (left) compared with crystal structure of largazole in complex; (right) docked structure of simplified largazole analogue overlapped with crystal structure of largazole when complexed to KDAC8.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Designing of selective KDAC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The Yoshida group has shown significant isoform selectivity enhancement against human KDACs with chlamydocin analogues by varying the chemical nature of the warhead interacting with the active-site zinc.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The choice of chemical functionality for binding to zinc at the active site is quite diverse.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> In addition to optimizing the headgroup for isoform selectivity, we varied the chemical nature of the warhead interacting with the zinc in the active site from the sulfhydryl, present in FK228 and largazole, to thioether, sulfone, carboxyl ester, carboxylic acid, amine, hydroxyl, and hydroxymate as well as methylsulfoxide functionality presents in sulforaphane, a naturally occurring KDAC inhibitor commonly found in cruciferous vegetables.<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52, 53)</a> Optimization of the warhead interacting with the zinc in the active site by transition-state analogues (TSAs) of amide hydrolysis may further improve the selectivity and potency.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> One can imagine extension of the zinc-binding headgroup to access the cavity associated with acetate release.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Thus, we plan to utilize experimental data from other classes of KDACIs to further optimize the isoform selectivity advantages for KDAC as seen for each of the three components; CTP, linker and warhead.</div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Results and Discussions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47861" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47861" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">First, we synthesized CTPs <b>1a</b>, <b>1b</b>, <b>2a</b>, and <b>2b</b> containing a pendant OBn-protected carboxylic acid on the α carbon of Asp, which was easily elongated with aminopropyl zinc-binding warheads. Synthesis of CTP shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Using a solution-phase strategy, we started with <i>O</i>-<sup>t</sup>butyl valine ester hydrochloride and coupled it with Fmoc-Pro/<span class="smallcaps smallerCapital">d</span>-pro-OH using standard EDC peptide coupling to give protected dipeptide <b>27</b> and <b>28</b>, respectively. The Fmoc group at <i>N</i>-terminal of dipeptide was deprotected using 20% piperidine in dichlomethane (DCM) and coupled with Fmoc-<span class="smallcaps smallerCapital">d</span>-pro/Pro-OH to yield the linear tripeptides <b>29</b> and <b>30</b>, and these tripeptides aggregated to form an insoluble material once solidified that decomposed on silica gel. To overcome this difficulty, we proceeded further without purification and characterization of the tripeptides. The Fmoc groups at the <i>N</i>-terminals of the tripeptides <b>28</b> and <b>29</b> were deprotected and coupled with Boc-Asp(OH)-OBn or with Boc-<span class="smallcaps smallerCapital">d</span>-asp(OH)-OBn to yield linear tetrapeptides <b>31a</b>, <b>31b</b>, <b>32a</b>, and <b>32b</b>, respectively. <i>N</i>-Terminal Boc and <i>C</i>-terminal O<sup><i>t</i></sup>Bu ester protection groups were then removed quantitatively using 30% trifluoroacetic acid in DCM to obtain the desired products (<b>33a</b>, <b>33b</b>, <b>34a</b>, and <b>34b</b>). Then macrolactamization was optimized at 2 mM in DMF using diphenylphosphoryl azide (DPPA) and diisopropylethylamine (DIPA); the CTPs were obtained in good yields as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. No epimerized or dimerized compound was observed during cyclization at 2 mM concentration in DMF. By synthesizing four CTP scaffolds with a pendant carboxyl group, a variety of warheads could be attached through an amide bond to determine the relative affinities of different warheads in the same structural context against different KDAC isoforms.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of CTP Reverse-Turn Scaffolds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) EDCl, HOBt, DIEA, DCM, 9 h; (ii) 20% piperidine/DCM, rt, 0.5 h, 100%; (iii) Fmoc-Xaa-OH, EDCl, HOBt, DIEA, DCM, 12 h; (iv) Boc-Zaa-OBn, EDCl, HOBt, DIEA, DCM, 24 h; (v) 30% TFA/DCM, 0 °C–rt, 3 h, 100%; (vi) DPPA, DIEA,DMF, 0 °C–rt, 72 h; (v) H<sub>2</sub>, Pd/C, EtOH, rt, 5–6 h.</p></p></figure><div class="NLM_p">Diastereomerically pure CTPs <b>3a</b>, <b>3b</b>, <b>4a</b>, and <b>4b</b> containing free pendant carboxyl groups were obtained quantitatively by hydrogenation of the OBn ester in the presence of Pd/C (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Sections S11–S14</a>). Epimerization at the Asp α-carbon stereocenter was observed, however, upon saponification of the OBn ester by lithium hydroxide (LiOH). The epimerized compound was separated by HPLC and confirmed as <b>3b</b> by comparing LC-MS data, <sup>1</sup>H and <sup>13</sup>CNMR spectra of <b>3b</b> (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Section S10</a>), which was authentically synthesized by Pd/C hydrogenation. CTP scaffolds with a pendant carboxyl group was coupled to 2,2′-diaminodiethyl disulfide and 3,3′-diaminodipropyl disulfide in the presence of EDCL yielded the desired CTPs having disulfide bridges of two and three carbons, respectively; phosphamide warhead CTP analogues <b>11a</b> and <b>11b</b> where obtained in the presence of DPPA. Upon treatment of disulfide containing CTPs (<b>35a</b>–<b>38a</b> and <b>35b</b>–<b>38b</b>) with (2<i>S</i>,3<i>S</i>)-1,4-bis(sulfanyl)butane-2,3-diol (DTT), sulfhydryl-warhead SLA analogues (<b>5a</b>, <b>6a</b>, <b>5b</b>, <b>6b</b>, <b>17a</b>, <b>17b</b>, <b>18a</b>, and <b>18b</b>) were obtained as shown in the <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>.</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Introduction of Sulfhydryl Zinc-Binding Functional Group on CTP Reverse-Turn Scaffolds</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Methylthioether, hydroxyl, and amine zinc-binding, three-carbon warheads were introduced using standard EDCL coupling to give a wide variety of SLA analogues. Sulfoxides and sulfones were introduced by borax-catalyzed, pH-controlled selective oxidation of organic sulfides by hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) according to a reported procedure.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Methylthioether was treated with hydrogen peroxide in the presence of borax in methanol (MeOH) and water mixture furnished the desired sulfoxide analogues (<b>8a</b>, <b>8b</b>, <b>20a</b>, and <b>20b</b>) at acidic pH and sulfones (<b>9a</b>, <b>9b</b>, <b>21a</b>, and <b>21b</b>) at basic pH (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).</div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Introduction of Thiol, Thioether, Sulfoxide, and Sulfone Zinc-Binding Groups</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Pendant carboxylic acids on CTPs were coupled with methyl 4-aminobutanoate to give methylester warheads (<b>12a</b>, <b>12b</b>, <b>23a</b>, and <b>23b</b>), which upon treatment with hydroxylamine in the presence of potassium hydroxide (KOH) furnished hydroxamic acid zinc-binding warheads (<b>14a</b>, <b>14b</b>, <b>25a</b>, and <b>25b</b>). SLA analogues <b>13a</b>, <b>13b</b>, <b>24a</b>, and <b>24b</b> having a carboxylic acid group were obtained quantitatively upon saponification of methyl esters by LiOH (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).</div><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Introduction of Carboxylic Acid, Ester, and Hydroxamate Zinc-Binding Groups</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Bioactivity Against Isolated KDAC Isoforms</h3><div class="NLM_p">A selection of synthesized SLA compounds (a total of 60, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Section S18</a> ) was assayed for their inhibitory activity against four human KDACs (herein KDACs 1, 3, 6, and 8), to clarify how the different CTP headgroups determine isoform selectivity (see <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). In vitro assays were performed on recombinant human KDACs 1, 3, and 8 (class I KDACs) and KDAC 6 (class II KDACs) using the Caliper EZ Reader II system (PerkinElmer Caliper Life Sciences, USA).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Most Active SLAs and Standard Compounds (Positive Controls) IC<sub>50</sub> (μM) Values</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">KDAC isoform, IC<sub>50</sub> μM</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center" char="."><b>1</b></th><th class="colsep0 rowsep0" align="center" char="."><b>3</b></th><th class="colsep0 rowsep0" align="center" char="."><b>6</b></th><th class="colsep0 rowsep0" align="center" char="."><b>8</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1 (12)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><b>14a</b></td><td class="colsep0 rowsep0" align="char" char=".">25.1</td><td class="colsep0 rowsep0" align="char" char=".">13.7</td><td class="colsep0 rowsep0" align="char" char=".">5.07</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2 (43)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left"><b>25a</b></td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char=".">7.66</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">MS-275<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></td><td class="colsep0 rowsep0" align="char" char=".">1.48</td><td class="colsep0 rowsep0" align="char" char=".">0.79</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">largazole<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a>,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></td><td class="colsep0 rowsep0" align="char" char=".">2.33</td><td class="colsep0 rowsep0" align="char" char=".">1.36</td><td class="colsep0 rowsep0" align="char" char=".">9.29</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">PCI-34051<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></td><td class="colsep0 rowsep0" align="char" char=".">14.41</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char=".">4.57</td><td class="colsep0 rowsep0" align="char" char=".">0.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">SAHA<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></td><td class="colsep0 rowsep0" align="char" char=".">0.0070</td><td class="colsep0 rowsep0" align="char" char=".">0.0014</td><td class="colsep0 rowsep0" align="char" char=".">0.0014</td><td class="colsep0 rowsep0" align="char" char=".">0.50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">SD-L-256<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></td><td class="colsep0 rowsep0" align="char" char=".">3.48</td><td class="colsep0 rowsep0" align="char" char=".">0.47</td><td class="colsep0 rowsep0" align="char" char=".">1.61</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">T247<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></td><td class="colsep0 rowsep0" align="char" char=".">1.11</td><td class="colsep0 rowsep0" align="char" char=".">3.94</td><td class="colsep0 rowsep0" align="char" char="."> >30</td><td class="colsep0 rowsep0" align="char" char="."> >30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">TSA<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></td><td class="colsep0 rowsep0" align="char" char=".">0.015</td><td class="colsep0 rowsep0" align="char" char=".">0.020</td><td class="colsep0 rowsep0" align="char" char=".">0.038</td><td class="colsep0 rowsep0" align="char" char=".">4.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">tubastatin A<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></td><td class="colsep0 rowsep0" align="char" char=".">2.87</td><td class="colsep0 rowsep0" align="char" char=".">0.77</td><td class="colsep0 rowsep0" align="char" char=".">0.014</td><td class="colsep0 rowsep0" align="char" char=".">2.34</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Number in the complete list of compounds (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">section S18</a>).</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">The thioester form of largazole<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (as depicted in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) was assayed.</p></div></div></div><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Isoform Selectivity</h4><div class="NLM_p last">Except for CTPs with hydroxymate groups (compounds <b>14a</b>, <b>14b</b>, <b>25a</b>, and <b>25b</b>), the SAR results clearly demonstrate the limited ability of the simplified largazole scaffolds to determine strong or selective interactions with the four used KDAC isoforms, showing generally weak activities (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Tables 1–2</a>). Conversely, SLAs with hydroxamate groups (compounds <b>14a</b>, <b>14b</b>, <b>25a</b>, and <b>25b</b>) show a general higher inhibitor potency compared to other SLAs (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Tables 1–2</a>), confirming a prominent role of the hydroxamate group in driving the ligand–protein recognition by interacting with the zinc ion. Compounds <b>14a</b> and <b>25a</b> show IC<sub>50</sub>s in a micromolar range with a detectable selectivity for KDAC6 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Tables 1–2</a>). These two compounds are <span class="smallcaps smallerCapital">d</span>-pro-Pro and Pro-<span class="smallcaps smallerCapital">d</span>-pro SLAs, and both are Asp derivatives. Thus, the presence of an Asp group instead of <span class="smallcaps smallerCapital">d</span>-asp consistently justifies the general increased selectivity vs KDAC6 (compare inhibition percentages of <b>14a</b> with <b>14b</b> and <b>25a</b> with <b>25b</b> in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Tables 1–2</a>) which is worth of consideration in designing new KDAC6 selective SLAs. This is consistent with the hypothesis that the selected SLA scaffolds oriented the linker, the warhead, and the capping group differently from largazole by binding in a different orientation, as confirmed by docking results (see molecular docking results of SLAs section and Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">8</a>). Nevertheless, some KDAC isoform selectivity between KDACs 1, 3, 6, and 8 was observed for non-hydroxamate derivatives (<a class="ref internalNav" href="#fig6" aria-label="Figures 5">Figures 5</a>–<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">6</a> and Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Tables 1–2</a>) that might be useful for future optimization. <span class="smallcaps smallerCapital">d</span>-pro-Pro CTP analogues with thiomethyl, sulfoxide, and sulfone warheads (compounds <b>8a</b>–<b>b</b> and <b>9a</b>–<b>b</b>; Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Table 1</a>) showed some selectivity for KDAC8 compared with KDACs 1, 3, and 6. The data further suggest:<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">The strong hydroxamic acid zinc-binding group generates, as reported above, the highest KDAC inhibitory activity (except in the case of the <span class="smallcaps smallerCapital">d</span>-asp-Pro-<span class="smallcaps smallerCapital">d</span>-pro scaffolds), in these cases, the strong zinc-binding group drives isoform recognition.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Thiol zinc-binding groups (along with the disulfides) showed the highest KDAC6 inhibitory activity next to hydroxamates.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Both the Asp α-carbon stereocenter and CTP type can modulate and change the activity profile and drive ligand–protein selectivity if a weaker zinc-binding group is present: in the case of sulfoxides, sulfones, hydroxyls, thiomethyl, and carboxylic acid zinc<b>-</b>binding groups, the interaction seems to be dominated by the CTP moiety that favors interactions with the KDAC8 isoform.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro derivatives with sulfoxides, sulfones, hydroxyls zinc<b>-</b>binding groups, and Asp-Pro-<span class="smallcaps smallerCapital">d</span>-pro derivatives with thiomethyl, sulfoxides, sulfones, and carboxylic acids, the zinc<b>-</b>binding group show the highest KDAC8 selectivity. About the <span class="smallcaps smallerCapital">d</span>-pro-Pro derivatives, with the above zinc-binding groups, the replacement of <span class="smallcaps smallerCapital">l</span>-Asp with <span class="smallcaps smallerCapital">d</span>-asp does not appear to affect selectivity on KDAC8 (except for the thiomethyl case, in fact <b>7b</b> is completely selective on KDAC8, but not <b>7a</b>). For Pro-<span class="smallcaps smallerCapital">d</span>-pro scaffolds with the above zinc-binding groups, the replacement of <span class="smallcaps smallerCapital">l</span>-Asp with <span class="smallcaps smallerCapital">d</span>-asp eliminates selectivity on KDAC8 (except for the thiomethyl case, in fact <b>19a</b> and <b>19b</b> have the same activity profile).</p></td></tr></table></div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Cases with No High Selectivity, but with Significant Activity Changes</h4><table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">As shown in <a class="ref internalNav" href="#fig6" aria-label="Figures 5">Figures 5</a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">6</a>, hydroxamic acid warheads (generally recognized as most potent, but with weak selectivity between different KDACs) provided, as in the case of the Asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro compound <b>14a</b> (<a class="ref internalNav" href="#fig6" aria-label="Figure 5">Figure 5</a>A and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Table 1</a>), the highest potency among the three different isoforms (KDACs 3, 6, and 8) with a detectable preference for KDACs 3 and 6 (2.16 and 2.65 times more active than with KDAC8, respectively) while the KDAC8 inhibitor activity was only average. The activity trend of compound <b>14a</b> was confirmed also for the <span class="smallcaps smallerCapital">d</span>-asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro analogue <b>14b</b> (<a class="ref internalNav" href="#fig6" aria-label="Figure 5">Figure 5</a>B), but interestingly without the large potency difference between KDAC3 and KDAC8 (1.29 times more active with respect to KDAC8, while the KDAC6 inhibitor potency was 2.52 times with respect to KDAC8). This clearly suggests a different KDAC3 conformational “availability and response” based on the Asp α-carbon stereocenter. Conversely, the Pro-<span class="smallcaps smallerCapital">d</span>-pro hydroxamic acid derivatives showed a different activity potency and selectivity trend (compare compound <b>25a</b> with <b>25b</b>, <a class="ref internalNav" href="#fig7" aria-label="Figures 6">Figures 6</a>A,B, Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Table 2</a>), with a loss of activity for KDAC3 and KDAC6 in the case of the <span class="smallcaps smallerCapital">d</span>-asp analogue (compound <b>25b</b>). The effect on activity from the Asp α-carbon stereocenter dominates considering the Pro-<span class="smallcaps smallerCapital">d</span>-pro derivative over the three KDAC isoforms even when a strong zinc-binding group (hydroxamic acid) was present.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Another interesting case was represented by CTPs characterized with thiol warheads: among the three-carbon linker-domain thiols, the Asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro and the <span class="smallcaps smallerCapital">d</span>-asp-Pro-<span class="smallcaps smallerCapital">d</span>-pro compounds <b>6a</b> and <b>18b</b>, respectively, showed the higher inhibition potency on KDAC6 suggesting, also in this case, a modulating role of the different CTPs. In fact, considering the <span class="smallcaps smallerCapital">d</span>-pro-Pro derivatives, the Asp derivative (compound <b>6a</b>) was generally more active compared with the <span class="smallcaps smallerCapital">d</span>-asp analogue, whereas for Pro-<span class="smallcaps smallerCapital">d</span>-pro derivatives, the Asp analogue (compound <b>18a</b>) was less active (but not for KDAC8) compared with the <span class="smallcaps smallerCapital">d</span>-asp derivative (compound <b>18b</b>).</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">A third interesting case was the disulfide compounds with a three-carbon linker (compounds <b>37a</b>, <b>37b</b>, <b>38a</b>, and <b>38b</b>): among the <span class="smallcaps smallerCapital">d</span>-pro-Pro analogues, it is appreciable how the <span class="smallcaps smallerCapital">d</span>-asp derivative (compound <b>37b</b>) maintained detectable KDAC3 inhibitory activity that was totally lost with the Asp analogue (compound <b>37a</b>).</p></td></tr></table><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Simplified largazole (<span class="smallcaps smallerCapital">d</span>-pro-Pro) analogues inhibition profiles plots. (A) Asp derivatives; (B) <span class="smallcaps smallerCapital">d</span>-asp derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Simplified largazole (Pro-<span class="smallcaps smallerCapital">d</span>-pro) analogues inhibition profiles plots. (A) Asp derivatives; (B) <span class="smallcaps smallerCapital">d</span>-asp derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Cases with High Selectivity</h4><table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Sulfoxides (compounds <b>8a</b>, <b>8b</b>, and <b>20a</b>) derivatives showed a clear selectivity for KDAC8 regardless of the CTP type. Only in the case of the <span class="smallcaps smallerCapital">d</span>-asp-Pro-<span class="smallcaps smallerCapital">d</span>-pro analogue (compound <b>20b</b>) was no selectivity and potency detected.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Sulfones (compounds <b>9a</b>, <b>9b</b>, and <b>21a</b>) derivatives showed clear selectivity for KDAC8 regardless the CTP type. Only in the case of the <span class="smallcaps smallerCapital">d</span>-asp-Pro-<span class="smallcaps smallerCapital">d</span>-pro analogue (compound <b>21b</b>) was no selectivity or potency detected.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Hydroxyls (compound <b>10a</b>, Asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro) showed selectivity for KDAC8, and other analogues (<b>10b</b>, <b>22a</b>, and <b>22b</b>) did not share that selectivity.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Carboxylic acids (only Pro-<span class="smallcaps smallerCapital">d</span>-pro derivatives, compounds <b>24a</b> and <b>24b</b>) showed selectivity on KDAC8.</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">Thiomethyl derivatives (<b>7b</b>, <b>19a</b>, and <b>19b</b>) showed selectivity on KDAC8, the presence of Asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro (as in the case of <b>7a</b>) neutralized the selectivity and decreased the potency.</p></td></tr></table><div class="NLM_p last">Our hypothesis that the warhead groups of the SLA scaffolds were not oriented correctly when bound to KDAC isoforms so that they did not interact optimally with the active-site zinc. An intensive effort to find docking protocols that demonstrates such a difference by developing a protocol<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> capable to detect a reliable structure-based procedure and applied to the lysine deacetylase (KDAC) inhibitors,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> providing a mechanism for optimizing novel CTP scaffolds. Molecular docking results confirmed this hypothesis (see Molecular Docking Results of SLAs paragraph and Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">8</a>).</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Molecular Modeling</h3><div class="NLM_p">Molecular docking was used to rationalize the SLA binding poses and help clarify any correlation with the obtained KDAC1, 3, 6, and 8 isoforms’ biological activities. A comprehensive molecular docking investigation on lysine deacetylases was performed by means of the recently released Clusterizer and DockAccessor software,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> with the purpose of detecting a best docking protocol to be applied, and used also to develop predictive COMBINEr<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> or 3-D QSAR<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> models. A comprehensive docking assessment proposed PLANTS<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> (using the PLP scoring function and considering the best docked poses, BD)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> eligible for predicting reliable SLAs binding poses. Because of the loss in activity of SLAs, a reliable binding pose analysis was not possible among the different KDAC isoforms, except for comparing their predicted binding poses with respect to the largazole thiol experimental one characterizing the KDAC8 cocrystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a>).</div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> KDAC8 Complex Structure Preparation</h4><div class="NLM_p last">KDAC8 cocrystal structures (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a>)<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> was prepared as previously reported,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> subsequently, compounds <b>14a</b> and <b>25a</b> were docked as described.</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Docking Assessment</h4><div class="NLM_p">As previously reported,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> an extensive docking assessment was applied to detect the best docking strategy using nine different docking programs: AutoDock4,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> AutoDock4(Zn),<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> AutoDock Vina,<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> DOCK6,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> MpSDockZn,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> PLANTS,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and Surflex-Dock<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a>). PLANTS<sub>PLP</sub><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> has been detected (using the Hungarian symmetry-corrected heavy-atom RMSD, HA_RMSD<i>h</i>, and the obtained best docked poses, BD)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> as the preferable docking program (showing the best compromise between performance and speed) to be used to dock largazole and its analogues, confirming its capability in predicting the binding pose of largazole thiol in the native protein (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a>, <a class="ref internalNav" href="#fig8" aria-label="Figure 7">Figure 7</a>), thus validating it for docking SLAs.</div><figure id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Docking of largazole in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a><a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> chain (A). Best Docked poses (BD): (A) experimental conformation redocking (ECRD); (B) random conformation redocking (RCRD). Experimental binding pose is green colored, BD predicted binding pose is magenta colored.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Molecular Docking Results of Compounds <b>14a</b> and <b>25a</b></h4><div class="NLM_p">PLANTS<sub>PLP</sub><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> was used to dock compounds <b>14a</b> and <b>25a</b> (the most active ones) in KDAC8 using a same strategy as previously reported.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> A first comparison between the predicted binding poses of <b>14a</b> and <b>25a</b> and the experimental crystal structure of largazole thiol in KDAC8 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a>) clearly show how the heteroproline dipeptide reverse-turn mimetics bind in a different manner compared to the largazole thiol (<a class="ref internalNav" href="#fig9" aria-label="Figure 8">Figure 8</a>), suggesting that the loss of planarity between the two heteroproline determines negative interactions and, as a consequence, low inhibition potency against KDAC8 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). The presence of a dihedral angle between the two heteroprolines leading to nonplanarity has a negative effect also on the inhibitor potency against other KDACs preventing the necessary interaction with the zinc ion.</div><figure id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison between largazole thiol (green) experimental binding pose and SLAs <b>14a</b> and <b>25a</b> BD poses in KDAC8 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a>). (A) Predicted BD poses of compound <b>14a</b> (Asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro, cyan); (B) predicted BD poses of compound <b>25a</b> (Asp-Pro-<span class="smallcaps smallerCapital">d</span>-pro, gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80404" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80404" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we have explored four different simplified largazole analogue (SLA) scaffolds by replacing the thiazole ring in largazole with heterochiral proline dipeptides (Pro-<span class="smallcaps smallerCapital">d</span>-pro and <span class="smallcaps smallerCapital">d</span>-pro-Pro). We have synthesized totally 60 SLA analogues by introducing a wide variety of potential zinc-binding groups on pendant carboxylic acids in CTPs. The effort to reduce the synthetic complexity of largazole by replacement of the bis-thiazole and unusual amino acid with readily accessible amino acid synthons did not give the anticipated results. Preliminary docking studies with KDAC8 of the simplified largazole analogue (SLA) indicated a plausible overlay that encouraged the synthetic effort described herein. The experimental results obtained, however, clearly:<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">indicate that binding of the SLA analogues reorients the linker and warhead in different positions that precluded their anticipated interaction with the zinc in the active site;</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">suggest the detrimental effect of a dihedral angle between the two heteroproline;</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">confirm the hydroxamate group capable of establishing stronger interactions with the zinc ion, partly reversing the depressing effect of the used CTP headgroups;</p></td></tr><tr class="li1"><td valign="top">•</td><td colspan="5" valign="top" class="paragraph"><p class="last">reveal the effect of the Asp group (where the linker and warhead are attached) which directs to a higher KDAC6 selectivity, suggesting its use when design new compounds with increase potency and selectivity against KDAC6.</p></td></tr></table>A recent docking protocol has been developed and assessed based on these results and will be used, together with the above-reported experimental evidencies, in the design of new SLA headgroups capable to properly interact with the zinc and endowed with higher activity and selectivity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27211" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27211" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemistry General</h3><div class="NLM_p last">All the reactions were performed in oven-dried apparatus and were stirred using magnetic stirbars. Starting materials, reagents, and solvents were purchased from commercial vendors unless otherwise noted. Chromatography grade ethyl acetate, dichloromethane, acetonitrile, and hexanes were obtained from Sigma-Aldrich. Column chromatography was performed on silica gel (100–200 mess) purchased from Sorbent Technologies. TLC was carried out on Analtech 200 μm silica-gel coated plastic-fiber sheets. All reactions were monitored by thin layer chromatography (TLC) carried out on Merck silica-gel plates (0.25 mm thick, 60F254), visualized by using UV (254 nm) or dyes such as ninhydrin, KMnO<sub>4</sub>, <i>p</i>-anisaldehyde, or CAM (ceric ammonium molybdate). High-performance liquid chromatography (HPLC) was carried out on GILSON GX-281 using Waters C18 5 μM, 4.6 mm × 50 mm and Waters Prep C18 5 μM, 19 mm × 150 mm reverse phase columns. The mobile phases used were A, H<sub>2</sub>O with 0.05% TFA; B, CH<sub>3</sub>CN with 0.05% TFA using a solvent gradient of A–B over 30 min with a flow rate of 14.8 mL/min, with detection at 220 and 254 nm UV detectors. Purity assessment and mass spectra (MS) data were obtained using a Hewlett-Packard HPLC/MSD using electrospray ionization (ESI) for detection. <sup>1</sup>H and <sup>13</sup>C NMR spectra were measured on a Varian 400 MHz NMR instrument. Chemical shifts are expressed in parts per million (ppm) from the residual of nondeuterated solvents as internal standard. (<sup>1</sup>H NMR: TMS <i>δ = </i> 0.00 ppm, CDCl<sub>3</sub> δ = 7.26 ppm, DMSO-<i>d</i><sub>6</sub> δ = 2.50 ppm, D<sub>2</sub>O: δ = 4.79 ppm. <sup>13</sup>C NMR (APT): TMS δ = 0.00 ppm, CDCl<sub>3</sub> δ = 77.16 ppm, DMSO-<i>d</i><sub>6</sub> δ = 39.52 ppm.) Coupling constants (<i>J</i>) are given in hertz (Hz). The following abbreviations were used to express the multiplicities: s = singlet; d = doublet; t = triplet; q = quartet; p = pentet; quin = quintet; sep = septet; hept = heptet; m = multiplet; dd = doublet of doublets; dt = doublet of triplet; td = triplet of doublet; m = multiplet; bs = broad singlet. All compounds used for biological assays are >95% pure based on NMR and LC-MS by UV absorbance at 210 and 254 nm wavelengths.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure for Synthesis of CTPs by Using Macrolactamization (<b>1a</b>, <b>1b</b>, <b>2a</b>, and <b>2b</b>) Protocol</h3><div class="NLM_p last">To a cold (0 °C) stirred solution of N- and C-terminal deprotected tetrapeptide (1 mmol) in dry DMF (500 mL) was added <i>N</i>,<i>N</i>-diisopropylethylamine (6 mmol) dropwise over 1 min followed by diphenyl phosphorazidate (DPPA) (4 mmol) then slowly brought it to room temperature and stirred until the complete consumption of starting material, then DMF was removed under reduced pressure. The resulting viscous solution was diluted with water (5 mL) and thoroughly extracted with ethyl acetate (15 mL). The combined organic extracts were washed with 1 N HCl (5 mL), saturated aqueous sodium bicarbonate (NaHCO<sub>3</sub>) (5 mL), and dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>) and concentrated to give a residue, which was purified by silica gel (100–200 mesh) flash column chromatograph.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Procedure for Catalytic Hydrogenation of Benzyl Ester (OBn) Protecting Group (<b>3a</b>, <b>3b</b>, <b>4a</b>, and <b>4b</b>)</h3><div class="NLM_p last">A stirred solution of the benzyl ester (1 mmol) and 10% Pd/C (0.1 mmol) in EtOH (10 mL) at 25 °C was placed under an atmosphere of hydrogen. After 4 h, the mixture was filtered through Celite using EtOH as eluent and concentrated under reduced pressure and proceeded further without purification.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Procedure for Introducing Warhead on Pendent Carboxylic Acid by Using EDCl Peptide Coupling (<b>7a</b>, <b>7b</b>, <b>10a</b>, <b>10b</b>, <b>12a</b>, <b>12b</b>, <b>16a</b>, <b>16b</b>, <b>19a</b>, <b>19b</b>, <b>22a</b>, <b>22b</b>, <b>23a</b>, and <b>23b</b>) Protocol</h3><div class="NLM_p last">To a solution (0 °C) of 1-(3-(dimethylamino)propyl)-3-ethylcarbodiimide hydrochloride (1.3 mmol), 1-hydroxybenzotriazole (1.3 mmol), and <i>cyclo</i>-Zaa(Xaa-Yaa-Val)-OH (1 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (10 mL) added <i>N</i>,<i>N</i>-diisopropylethylamine (3 mmol) was stirred for 15 min. Then added solution of amino propyl linker (1 mmol) in <i>N</i>,<i>N</i>-dimethylformamide (2 mL) flask, and the reaction mixture was stirred at room temperature until the starting material was consumed completely. <i>N</i>,<i>N</i>-Dimethylformamide and dichloromethane were removed under reduced pressure, and the resulting viscous solution was diluted with water (5 mL) and thoroughly extracted with ethyl acetate (15 mL). The combined organic extracts were washed with 1 N HCl (5 mL) and saturated aqueous sodium bicarbonate (NaHCO<sub>3</sub>) (5 mL) and dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>) and concentrated to give a residue, which was purified by HPLC.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedure for LiOH Base-Mediated Hydrolysis of Methyl Ester (<b>13a</b>, <b>13b</b>, <b>24a</b>, and <b>24b</b>)</h3><div class="NLM_p last">To a stirred solution of lithium hydroxide (1.5 mmol) in of distilled water (1 mL) was added to the methyl ester (1 mmol) in MeOH (3 mL) and then stirred at room temperature for 3.5 h, then MeOH was removed under reduced pressure and acidified with 1 N HCl until pH was between 2 and 3 at 0 °C. Then extracted with EtOAc, dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>), and concentrated to give the desired acid, which was used further without purification.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure for Synthesis of Hydroxamic Acids from Corresponding Methyl Esters (<b>14a</b>, <b>14b</b>, <b>25a</b>, and <b>25b</b>)</h3><div class="NLM_p last">To a stirred solution of methyl ester (1 mmol) in MeOH (3 mL), hydroxylamine hydrochloride (10 mmol) was added followed by 0.1 N potassium hydroxide (12 mmol) in distilled water (1 mL) and then stirred at room temperature for 30 min, then MeOH was removed under reduced pressure and acidified with 1 N HCl until pH was between 2 and 3 at 0 °C. The solution was extracted with EtOAc, dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>), and concentrated to give the desired hydroxamic acid in high yield, which was further purified by HPLC.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Procedure for Sulfoxide Synthesis from Organic Sulfides by H<sub>2</sub>O<sub>2</sub>/Borax (<b>8a</b>, <b>8b</b>, <b>20a</b>, and <b>20b</b>)</h3><div class="NLM_p last">In a typical experiment to a 25 mL flask equipped with a magnetic stirrer and 30% H<sub>2</sub>O<sub>2</sub> (6.0 mmol) in water was added borax (0.2 mmol) and MeOH (5 mL), followed by methyl thioether containing peptide (1 mmol). The reaction was monitored by TLC. After complete disappearance of the reactant added Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> to destroy the excess amount of H<sub>2</sub>O<sub>2</sub> and then removed MeOH, the product was extracted with EtOAc, dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>), and concentrated to give a desired sulfoxide, which was purified by reverse phase HPLC.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Procedure for Sulfones Synthesis from Organic Sulfides by H<sub>2</sub>O<sub>2</sub>/Borax (<b>9a</b>, <b>9b</b>, <b>21a</b>, and <b>21b</b>)</h3><div class="NLM_p last">In a typical experiment to a 25 mL flask equipped with a magnetic stirrer and 30% H<sub>2</sub>O<sub>2</sub> (6.0 mmol) was added borax (0.2 mmol) and MeOH (5 mL), followed by methyl thioether containing peptide (1 mmol). To the resulting solution was added 0.1 N NaOH to maintain the pH of the solution at 10, and the mixture was stirred at room temperature for 12–15 h and monitored by LC-MS. After complete disappearance of the reactant added Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub> to destroy the excess amount of H<sub>2</sub>O<sub>2</sub> and then removed MeOH, the product was extracted with EtOAc, dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>), and concentrated to give a desired sulfone, which was purified by reverse phase HPLC.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> General Procedure for Synthesis of Sulfhydryl from Corresponding Homodimer (<b>5a</b>, <b>5b</b>, <b>6a</b>, <b>6b</b>, <b>17a</b>, <b>17a</b>, <b>18a</b>, and <b>18b</b>)</h3><div class="NLM_p">To a stirred solution of disulfide containg homodimer of SLA analogue (1 mmol) in DMSO–water (1:1) (4 mL), dithiothreitol (DTT) (20 mmol) was added followed by 0.1 N sodium hydroxide (0.01 mL) in distilled water and then stirred at room temperature for 2 h. Then which was further purified by HPLC using 10–40 acetonitrile gradient over 30 min.</div><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-OBn (<b>1a</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide <b>1a</b> was synthesized by following the above general procedure for DPPA macrolactamization and purified by silical gel column chromatography (EtOAc) as a white solid (yield, 79%; purity by LC-MS, 99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.50–7.41 (m, 2 H), 7.41–7.28 (m, 3 H), 7.03 (d, <i>J</i> = 9.4 Hz, 1 H), 6.48 (d, <i>J</i> = 9.8 Hz, 1 H), 5.28 (d, <i>J</i> = 11.7 Hz, 1 H), 5.19 (d, <i>J</i> = 12.1 Hz, 1 H), 5.12 (t, <i>J</i> = 9.6 Hz, 1 H), 4.58 (dd, <i>J</i> = 3.6, 9.0 Hz, 1 H), 4.56 (t, <i>J</i> = 7.0 Hz, 1 H), 4.45 (dd, <i>J</i> = 3.9, 9.0 Hz, 1 H), 4.18–4.09 (m, 1 H), 3.63–3.54 (m, 1 H), 3.31–3.22 (m, 1 H), 3.12 (d, <i>J</i> = 12.9 Hz, 1 H), 3.01 (dt, <i>J</i> = 5.7, 9.7 Hz, 1 H), 2.79 (dd, <i>J</i> = 9.8, 12.5 Hz, 1 H), 2.53 (dt, <i>J</i> = 3.3, 6.7 Hz, 1 H), 2.41–2.29 (m, 1 H), 2.17–2.02 (m, 4 H), 1.93–1.79 (m, 2 H), 1.76–1.63 (m, 1 H), 0.93 (d, <i>J</i> = 7.0 Hz, 3 H), 0.89 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.7, 171.3, 171.1, 170.7, 169.9, 134.9, 129.2, 128.6, 128.5, 67.8, 62.6, 58.2, 56.0, 49.5, 47.8, 47.2, 35.8, 29.9, 29.1, 28.0, 26.1, 24.9, 20.2, 17.0. MS (ESI): found [M + H]<sup>+</sup>, 499.4.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-OBn (<b>1b</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide <b>1b</b> was synthesized by following the above general procedure for DPPA macrolactamisation and purified by silical gel column chromatography (EtOAc) as a white solid (yield, 78%; purity by LC-MS, 99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.42–7.28 (m, 5 H), 7.09 (d, <i>J</i> = 9.0 Hz, 1 H), 6.51 (d, <i>J</i> = 6.3 Hz, 1 H), 5.24 (d, <i>J</i> = 12.1 Hz, 1 H), 5.14 (d, <i>J</i> = 12.5 Hz, 1 H), 4.69 (t, <i>J</i> = 6.5 Hz, 1 H), 4.52 (dd, <i>J</i> = 4.1, 9.6 Hz, 1 H), 4.48 (dd, <i>J</i> = 4.3, 8.6 Hz, 1 H), 4.28–4.14 (m, 2 H), 3.99 (td, <i>J</i> = 5.7, 11.6 Hz, 1 H), 3.66–3.62 (m, 2 H), 3.07 (t, <i>J</i> = 12.3 Hz, 1 H), 2.95 (dd, <i>J</i> = 5.3, 12.7 Hz, 1 H), 2.45–2.36 (m, 2 H), 2.29–2.20 (m, 2 H), 2.14–2.02 (m, 5 H), 0.92 (d, <i>J</i> = 5.1 Hz, 3 H), 0.91 (d, <i>J</i> = 5.1 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.2, 172.4, 171.7, 171.5, 169.5, 135.4, 128.5, 128.4, 128.3, 67.5, 62.5, 58.2, 56.0, 50.4, 47.7, 47.5, 35.2, 29.7, 28.5, 28.0, 25.9, 24.9, 20.1, 17.0. MS (ESI): found [M + H]<sup>+</sup>, 499.4.</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-OH (<b>3a</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide pendent carboxylic acid <b>3a</b> was synthesized from <b>1a</b> by following the above general procedure for catalytic hydrogenation of benzyl ester, obtained the desired pendent carboxylic acid as a white solid (yield, 96%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.34 (bs, 1 H), 7.21 (d, <i>J</i> = 8.6 Hz, 1 H), 6.53 (d, <i>J</i> = 9.0 Hz, 1 H), 5.03 (t, <i>J</i> = 8.0 Hz, 1 H), 4.63 (t, <i>J</i> = 7.4 Hz, 1 H), 4.48 (dd, <i>J</i> = 4.7, 8.6 Hz, 1 H), 4.44 (dd, <i>J</i> = 4.7, 8.6 Hz, 1 H), 4.24–4.15 (m, 1 H), 3.71–3.52 (m, 3 H), 3.22 (dd, <i>J</i> = 2.1, 13.2 Hz, 1 H), 2.90 (dd, <i>J</i> = 9.0, 13.3 Hz, 1 H), 2.51–2.33 (m, 2 H), 2.24–1.96 (m, 6 H), 1.89 (d, <i>J</i> = 10.6 Hz, 1 H), 0.94 (d, <i>J</i> = 6.7 Hz, 3 H), 0.90 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.0, 172.7, 172.5, 170.7, 170.3, 62.6, 58.4, 56.7, 49.5, 48.2, 47.4, 35.5, 29.7, 29.1, 27.9, 26.2, 24.9, 20.0, 17.2. MS (ESI): found [M + H]<sup>+</sup>, 409.4.</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-OH (<b>3b</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide pendent carboxylic acid <b>3b</b> was synthesized from <b>1b</b> by following the above general procedure for catalytic hydrogenation of benzyl ester, obtained the desired pendent carboxylic acid as a white solid (yield, 94%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.13 (d, <i>J</i> = 9.4 Hz, 1 H), 6.64 (d, <i>J</i> = 5.9 Hz, 1 H), 6.35 (bs, 1 H), 4.69 (t, <i>J</i> = 6.5 Hz, 1 H), 4.56 (dd, <i>J</i> = 3.9, 9.0 Hz, 1 H), 4.48 (dd, <i>J</i> = 4.7, 8.2 Hz, 1 H), 4.24–4.08 (m, 2 H), 3.76–3.68 (m, 2 H), 3.67–3.60 (m, 1 H), 2.46–2.34 (m, 2 H), 2.34–2.18 (m, 2 H), 2.17–2.10 (m, 2 H), 2.10–1.99 (m, 3 H), 0.93 (d, <i>J</i> = 6.7 Hz, 3 H), 0.92 (d, <i>J</i> = 6.5 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.8, 174.1, 172.3, 170.9, 62.5, 58.4, 56.2, 53.2, 47.8, 47.5, 35.3, 29.7, 28.4, 28.0, 25.9, 25.0, 20.1, 17.0. MS (ESI): found [M + H]<sup>+</sup>, 409.4.</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Homodimer: [<i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> (<b>35a</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>35a</b> was synthesized from <b>3a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtaining the desired homodimer as a white solid (yield, 82%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.24 (t, <i>J</i> = 5.9 Hz, 1 H), 7.18 (d, <i>J</i> = 9.4 Hz, 1 H), 6.50 (d, <i>J</i> = 10.2 Hz, 1 H), 5.04 (t, <i>J</i> = 9.0 Hz, 1 H), 4.62 (t, <i>J</i> = 6.8 Hz, 1 H), 4.60 (dd, <i>J</i> = 3.7, 9.0 Hz, 1 H), 4.44 (dd, <i>J</i> = 5.5, 8.6 Hz, 1 H), 4.14 (td, <i>J</i> = 6.7, 9.8 Hz, 1 H), 3.70–3.56 (m, 4 H), 3.49 (dd, <i>J</i> = 2.0, 12.9 Hz, 1 H), 2.81 (dt, <i>J</i> = 1.6, 6.8 Hz, 2 H), 2.63 (dd, <i>J</i> = 9.4, 12.9 Hz, 1 H), 2.57 (dd, <i>J</i> = 3.7, 6.8 Hz, 1 H), 2.48–2.35 (m, 1 H), 2.22–2.09 (m, 4 H), 2.09–2.00 (m, 1 H), 2.00–1.83 (m, 2 H), 0.92 (d, <i>J</i> = 7.0 Hz, 3 H), 0.88 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: <sup>13</sup>C NMR (101 MHz, cdcl<sub>3</sub>) δ = 174.6, 172.8, 172.2, 171.1, 170.8, 77.3, 77.0, 76.7, 62.7, 58.5, 56.8, 50.8, 48.3, 47.4, 39.3, 37.0, 34.9, 29.8, 29.0, 28.0, 26.0, 25.0, 19.8, 16.8. MS (ESI): found [M + Na]<sup>+</sup>, 955.7.</div></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>SH (<b>5a</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>5a</b> was synthesized from <b>35a</b> by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer, obtaining the desired pendent thiol as a white solid (yield, 87%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.31 (d, <i>J</i> = 9.4 Hz, 1 H), 7.08 (t, <i>J</i> = 5.7 Hz, 1 H), 6.46 (d, <i>J</i> = 10.6 Hz, 1 H), 5.04 (t, <i>J</i> = 9.8 Hz, 1 H), 4.67 (dd, <i>J</i> = 3.1, 9.8 Hz, 1 H), 4.63 (t, <i>J</i> = 7.2 Hz, 1 H), 4.48 (dd, <i>J</i> = 5.3, 8.8 Hz, 1 H), 4.15 (td, <i>J</i> = 6.7, 9.8 Hz, 1 H), 3.74–3.56 (m, 5 H), 3.35 (qd, <i>J</i> = 6.6, 13.1 Hz, 1 H), 2.75–2.61 (m, 2 H), 2.61–2.54 (m, 1 H), 2.44 (td, <i>J</i> = 7.9, 12.8 Hz, 1 H), 2.24–2.01 (m, 6 H), 1.97–1.87 (m, 2 H), 1.84 (t, <i>J</i> = 8.8 Hz, 1 H), 0.93 (d, <i>J</i> = 7.0 Hz, 3 H), 0.89 (d, <i>J</i> = 7.0 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 468.4.</div></div><div id="sec4_9_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Homodimer: [<i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> (<b>35b</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>35b</b> was synthesized from <b>3b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtaining the desired homodimer as a white solid (yield, 78%; purity by LC-MS, 98%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.41 (t, <i>J</i> = 4.9 Hz, 1 H), 7.11 (d, <i>J</i> = 10.2 Hz, 1 H), 6.53 (d, <i>J</i> = 6.3 Hz, 1 H), 4.65 (t, <i>J</i> = 6.8 Hz, 1 H), 4.45 (t, <i>J</i> = 5.2 Hz, 1 H), 4.42 (t, <i>J</i> = 4.6 Hz, 1 H), 4.17 (td, <i>J</i> = 6.3, 9.7 Hz, 1 H), 4.10–4.01 (m, 1 H), 3.68 (t, <i>J</i> = 6.5 Hz, 2 H), 3.65–3.44 (m, 3 H), 3.05 (dd, <i>J</i> = 3.9, 12.5 Hz, 1 H), 2.92 (t, <i>J</i> = 12.5 Hz, 1 H), 2.85–2.65 (m, 2 H), 2.42–2.19 (m, 3 H), 2.19–1.90 (m, 6 H), 0.93 (d, <i>J</i> = 6.7 Hz, 3 H), 0.91 (d, <i>J</i> = 6.8 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.8, 172.7, 172.5, 172.1, 170.0, 62.5, 58.2, 57.0, 52.5, 47.8, 47.4, 38.6, 37.5, 34.5, 29.7, 28.7, 28.1, 25.9, 24.9, 20.1, 17.4. MS (ESI): found [M + Na]<sup>+</sup>, 955.7.</div></div><div id="sec4_9_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>SH (<b>5b</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>5b</b> was synthesized from <b>35b</b> by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer, obtaining the desired pendent thiol as a white solid (yield, 80%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.26 (bs, 1 H), 7.17 (d, <i>J</i> = 9.4 Hz, 1 H), 6.43 (d, <i>J</i> = 6.3 Hz, 1 H), 4.73–4.64 (m, 1 H), 4.54 (dd, <i>J</i> = 4.1, 9.6 Hz, 1 H), 4.46 (dd, <i>J</i> = 4.1, 8.4 Hz, 1 H), 4.15 (t, <i>J</i> = 6.5 Hz, 1 H), 4.02–3.95 (m, 1 H), 3.74–3.50 (m, 5 H), 3.32 (dd, <i>J</i> = 6.8, 13.1 Hz, 1 H), 3.08 (dd, <i>J</i> = 4.3, 12.5 Hz, 1 H), 2.95 (t, <i>J</i> = 12.3 Hz, 1 H), 2.74–2.60 (m, 2 H), 2.38 (dt, <i>J</i> = 3.5, 7.0 Hz, 2 H), 2.22–2.15 (m, 2 H), 2.14–2.05 (m 4 H), 1.50 (t, <i>J</i> = 8.6 Hz, 1 H), 0.94 (d, <i>J</i> = 3.1 Hz, 3 H), 0.92 (d, <i>J</i> = 3.1 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 468.4.</div></div><div id="sec4_9_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Homodimer: [<i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> Prodrug (<b>37a</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>37a</b> was synthesized <b>3a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtaining the desired homodimer as a white solid (yield, 65%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.09 (bs, 1 H), 7.00 (d, <i>J</i> = 8.6 Hz, 1 H), 6.38 (d, <i>J</i> = 5.0 Hz, 1 H), 4.58 (t, <i>J</i> = 6.7 Hz, 1 H), 4.37 (dd, <i>J</i> = 5.1, 8.2 Hz, 1 H), 4.34 (dd, <i>J</i> = 4.7, 8.2 Hz, 1 H), 4.14–4.05 (m,1 H), 4.03–3.91 (m, 1 H), 3.66–3.50 (m, 3 H), 3.39–3.20 (m, 2 H), 2.97 (dd, <i>J</i> = 3.9, 12.5 Hz, 1 H), 2.81 (t, <i>J</i> = 12.5 Hz, 1 H), 2.63 (t, <i>J</i> = 6.8 Hz, 2 H), 2.35–2.21 (m, 2 H), 2.17 (dd, <i>J</i> = 5.3, 10.8 Hz, 1 H), 2.13–2.06 (m, 1 H), 2.06–1.99 (m, 1 H), 1.99–1.87 (m, 2 H), 1.85–1.81 (m, 2 H), 1.83 (td, <i>J</i> = 6.6, 13.0 Hz, 2 H), 0.85 (d, <i>J</i> = 3.9 Hz, 3 H), 0.84 (d, <i>J</i> = 3.9 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.2, 173.2, 172.5, 172.2, 170.9, 65.5, 62.6, 58.4, 56.7, 53.0, 47.8, 47.5, 36.6, 34.4, 31.6, 29.7, 28.5, 28.1, 25.9, 25.0, 20.1, 17. MS (ESI): found [M + H]<sup>+</sup>, 961.8.</div></div><div id="sec4_9_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SH (<b>6a</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>6a</b> was synthesized from <b>37a</b> by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer, obtaining the desired pendent thiol as a white solid (yield, 79%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.31 (d, <i>J</i> = 9.0 Hz, 1 H), 7.15 (bs, 1 H), 6.58 (d, <i>J</i> = 10.2 Hz, 1 H), 5.11–5.00 (m, 1 H), 4.64 (t, <i>J</i> = 7.2 Hz, 1 H), 4.56 (dd, <i>J</i> = 3.7, 9.2 Hz, 1 H), 4.48 (dd, <i>J</i> = 5.7, 8.4 Hz, 1 H), 4.14 (td, <i>J</i> = 7.0, 9.9 Hz, 1 H), 3.73–3.55 (m, 3 H), 3.54–3.41 (m, 2 H), 3.36 (td, <i>J</i> = 6.5, 13.2 Hz, 1 H), 2.67 (dd, <i>J</i> = 8.8, 13.1 Hz, 1 H), 2.57 (q, <i>J</i> = 7.2 Hz, 3 H), 2.50–2.40 (m, 1 H), 2.25–2.12 (m, 4 H), 2.07 (td, <i>J</i> = 6.2, 12.7 Hz, 1 H), 1.99–1.92 (m, 2 H), 1.89–1.79 (m, 2 H), 1.43 (t, <i>J</i> = 8.0 Hz, 1 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H), 0.90–0.86 (d, <i>J</i> = 6.7 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 482.4.</div></div><div id="sec4_9_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Homodimer: [<i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> Prodrug (<b>37b</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>37b</b> was synthesized from <b>3b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtaining the desired homodimer as a white solid (yield, 67%; purity by LC-MS, 99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.07 (d, <i>J</i> = 9.4 Hz, 1 H), 6.72 (t, <i>J</i> = 5.3 Hz, 1 H), 6.34 (d, <i>J</i> = 11.2 Hz, 1 H), 4.91 (t, <i>J</i> = 9.6 Hz, 1 H), 4.58 (dd, <i>J</i> = 2.7, 9.8 Hz, 1 H), 4.53 (t, <i>J</i> = 6.8 Hz, 1 H), 4.34 (dd, <i>J</i> = 4.9, 8.8 Hz, 1 H), 4.11–4.02 (m, 1 H), 3.62–3.38 (m, 6 H), 3.26–3.14 (m, 1 H), 2.64 (dt, <i>J</i> = 2.3, 7.0 Hz, 2 H), 2.54 (dt, <i>J</i> = 2.9, 6.6 Hz, 1 H), 2.46 (dd, <i>J</i> = 9.4, 12.5 Hz, 1 H), 2.37–2.28 (m, 1 H), 2.08–2.02 (m, 4 H), 2.02–1.93 (m, 2 H), 1.84 (p, <i>J</i> = 6.8 Hz, 12 H), 1.82–1.75 (m, 1 H), 0.84 (d, <i>J</i> = 6.7 Hz, 3 H), 0.80 (d, <i>J</i> = 6.7 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 961.8.</div></div><div id="sec4_9_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SH (<b>6b</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>6b</b> was synthesized from <b>37b</b> by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer, obtaining the desired pendent thiol as a white solid (yield, 85%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.28 (d, <i>J</i> = 7.4 Hz, 1 H), 6.62 (bs, 1 H), 6.51 (d, <i>J</i> = 6.3 Hz, 1 H), 4.72–4.65 (m, 1 H), 4.51 (dd, <i>J</i> = 4.3, 8.2 Hz, 1 H), 4.47 (dd, <i>J</i> = 5.0, 9.2 Hz, 1 H), 4.17 (td, <i>J</i> = 6.6, 9.9 Hz, 2 H), 4.04–3.96 (m, 2 H), 3.72–3.61 (m, 3 H), 3.45 (td, <i>J</i> = 6.7, 13.3 Hz, 1 H), 3.37 (td, <i>J</i> = 6.6, 13.0 Hz, 1 H), 3.07 (dd, <i>J</i> = 4.7, 12.5 Hz, 1 H), 2.96 (t, <i>J</i> = 12.5 Hz, 1 H), 2.55 (d, <i>J</i> = 7.8 Hz, 1 H), 2.59–2.52 (m, 1 H), 2.47–2.18 (m, 4 H), 2.17–2.02 (m, 4 H), 1.87–1.78 (m, 2 H), 1.44 (t, <i>J</i> = 8.0 Hz, 1 H), 0.94 (d, <i>J</i> = 6.9 Hz, 3 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H); MS (ESI): found [M + H]<sup>+</sup>, 482.4.</div></div><div id="sec4_9_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SCH<sub>3</sub> (<b>7a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>7a</b> having the methylthioether warhead on pendent carboxylic acid was synthesized from <b>3a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 3-(methylthio)propan-1-amine and purified by HPLC using 5–55% acetonitrile gradient, obtaining the desired peptide as a white solid (yield, 68%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.17 (d, <i>J</i> = 9.4 Hz, 1 H), 6.78 (t, <i>J</i> = 6.4 Hz, 1 H), 6.41 (d, <i>J</i> = 10.6 Hz, 1 H), 5.00 (t, <i>J</i> = 10.0 Hz, 1 H), 4.66 (dd, <i>J</i> = 3.1, 9.4 Hz, 1 H), 4.61 (t, <i>J</i> = 6.7 Hz, 1 H), 4.44 (dd, <i>J</i> = 4.9, 8.4 Hz, 1 H), 4.20–4.11 (m, 1 H), 3.70–3.60 (m, 3 H), 3.57 (d, <i>J</i> = 12.5 Hz, 1 H), 3.48 (dt, <i>J</i> = 6.7, 13.3 Hz, 1 H), 3.33 (dt, <i>J</i> = 6.7, 12.9 Hz, 1 H), 2.63 (dt, <i>J</i> = 2.9, 6.9 Hz, 1 H), 2.60–2.49 (m, 2 H), 2.47–2.35 (m, 1 H), 2.21–2.11 (m, 3 H), 2.08 (s, 3 H), 2.07–1.99 (m, 1 H), 1.98–1.88 (m, 2 H), 1.83 (quin, <i>J</i> = 7.0 Hz, 2 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H), 0.89 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.8, 172.5, 172.2, 171.3, 170.3, 62.8, 58.6, 56.5, 51.1, 48.4, 47.4, 39.1, 34.8, 31.2, 29.9, 29.0, 28.3, 28.1, 26.1, 25.0, 20.0, 16.8, 15.3. MS (ESI): found: [M + Na]<sup>+</sup>, 518.8.</div></div><div id="sec4_9_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SCH<sub>3</sub> (<b>7b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>7b</b> having the methylthioether warhead on pendent carboxylic acid was synthesized from <b>3b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 3-(methylthio)propan-1-amine and purified by HPLC using 5–55% acetonitrile gradient, obtaining the desired peptide as a white solid (yield, 65%; purity by LC-MS, 98%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.15 (d, <i>J</i> = 9.4 Hz, 1 H), 7.08 (t, <i>J</i> = 6.4 Hz, 1 H), 6.41 (d, <i>J</i> = 6.3 Hz, 1 H), 4.67 (t, <i>J</i> = 9.8 Hz, 1 H), 4.49 (dd, <i>J</i> = 3.2, 9.2 Hz, 1 H), 4.45 (dd, <i>J</i> = 4.8, 8.7 Hz, 1 H), 4.17 (td, <i>J</i> = 6.3, 9.8 Hz, 1 H), 3.97 (td, <i>J</i> = 5.5, 11.2 Hz, 1 H), 3.68 (t, <i>J</i> = 6.4 Hz, 2 H), 3.67–3.59 (m, 1 H), 3.45–3.28 (m, 2 H), 3.06 (dd, <i>J</i> = 4.1, 12.7 Hz, 1 H), 2.92 (t, <i>J</i> = 12.5 Hz, 1 H), 2.51 (t, <i>J</i> = 7.2 Hz, 2 H), 2.43–2.30 (m, 2 H), 2.29–2.17 (m, 2 H), 2.16–2.10 (m, 2 H), 2.09 (s, 3 H), 2.06–1.95 (m, 3 H), 1.81 (dp, <i>J</i> = 2.2, 7.0 Hz, 2 H), 0.94 (d, <i>J</i> = 3.5 Hz, 3 H), 0.92 (d, <i>J</i> = 3.1 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.1, 173.2, 172.3, 172.1, 169.6, 62.6, 58.3, 56.6, 53.3, 47.8, 47.4, 38.7, 34.4, 31.4, 29.7, 28.6, 28.3, 28.1, 25.9, 24.9, 20.1, 17.2, 15.4. MS (ESI): found [M + H]<sup>+</sup>, 496.4.</div></div><div id="sec4_9_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)CH<sub>3</sub> (<b>8a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>8a</b> having the methylsulfoxide warhead on pendent carboxylic acid was synthesized from <b>7a</b> by following the above general procedure for sulfoxide synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/borax and purified by HPLC using 5–50% acetonitrile gradient, obtaining the desired sulfoxide as a white solid (yield, 69%; purity by LC-MS, >99%). Nonseparable diastereomeric mixture = 1.1:1. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.20 (d, <i>J</i> = 7.0 Hz, 1 H), 7.18 (d, <i>J</i> = 7.2 Hz, 1 H), 6.97 (t, <i>J</i> = 5.9 Hz, 1 H), 6.95 (t, <i>J</i> = 5.6 Hz, 1 H), 6.37 (d, <i>J</i> = 10.6 Hz, 1 H), 6.31 (d, <i>J</i> = 10.6 Hz, 1 H), 4.98 (t, <i>J</i> = 9.8 Hz, 2 H), 4.65 (dd, <i>J</i> = 3.1, 9.8 Hz, 2 H), 4.61 (t, <i>J</i> = 7.0 Hz, 2 H), 4.42 (dd, <i>J</i> = 4.7, 8.2 Hz, 1 H), 4.39 (dd, <i>J</i> = 4.5, 7.9 Hz, 1 H), 4.20–4.09 (m, 2 H), 3.73–3.54 (m, 8 H), 3.38–3.21 (m, 2 H), 2.91 (t, <i>J</i> = 8.1 Hz, 2 H), 2.81 (t, <i>J</i> = 8.1 Hz, 2 H), 2.67 (s, 3 H), 2.65 (s, 3 H), 2.64–2.59 (m, 2 H), 2.53 (ddd, <i>J</i> = 2.3, 9.6, 12.3 Hz, 2 H), 2.47–2.35 (m, 2 H), 2.20–2.09 (m, 9 H), 2.09–2.00 (m, 7 H), 1.98–1.83 (m, 4 H), 0.92 (d, <i>J</i> = 7.0 Hz, 6 H), 0.89 (d, <i>J</i> = 6.9 Hz, 6 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 175.0, 175.0, 172.4, 172.3, 172.3, 171.1, 170.6, 170.5, 62.8, 62.8, 58.6, 56.5, 53.4, 51.0, 50.7, 50.3, 48.5, 47.4, 38.4, 37.6, 34.4, 34.3, 31.6, 29.9, 28.8, 28.1, 28.0, 26.1, 25.0, 22.8, 22.6, 19.9, 16.7. MS (ESI): found [M + H]<sup>+</sup>, 512.4.</div></div><div id="sec4_9_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)CH<sub>3</sub> (<b>8b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>8b</b> having the methylsulfoxide warhead on pendent carboxylic acid was synthesized from <b>7b</b> by following the above general procedure for sulfoxide synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/borax and purified by HPLC using 5–50% acetonitrile gradient, obtaining the desired sulfoxide as a white solid (yield, 65%; purity by LC-MS, 98%). Nonseparable diastereomeric mixture = 1:1. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.12 (d, <i>J</i> = 7.0 Hz, 1 H), 6.87 (bs, 1 H), 6.41 (bs, 1 H), 4.69 (t, <i>J</i> = 6.7 Hz, 1 H), 4.53 (td, <i>J</i> = 3.2, 9.3 Hz, 1 H), 4.44 (dd, <i>J</i> = 4.5, 8.8 Hz, 1 H), 4.16 (td, <i>J</i> = 6.4, 9.6 Hz, 1 H), 3.93–3.84 (m, 1 H), 3.72–3.52 (m, 4 H), 3.38–3.22 (m, 1 H), 3.06 (dd, <i>J</i> = 3.9, 12.5 Hz, 1 H), 2.91 (t, <i>J</i> = 11.9 Hz, 1 H), 2.86–2.70 (m, 2 H), 2.62 (s, 1.5 H), 2.61 (s, 1.5 H), 2.46–2.30 (m, 2 H), 2.29–2.17 (m, 1 H), 2.16–1.91 (m, 6 H), 1.87–1.70 (m, 2 H), 0.93 (d, <i>J</i> = 6.7 Hz, 3 H), 0.92 (d, <i>J</i> = 6.6 Hz, 3 H). MS (ESI): found [M + Na]<sup>+</sup>, 512.4.</div></div><div id="sec4_9_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)<sub>2</sub>CH<sub>3</sub> (<b>9a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>9a</b> having the methylsulfone warhead on pendent carboxylic acid was synthesized from <b>7a</b> by following the above general procedure for sulfones synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/borax and purified by HPLC using 5–48% acetonitrile gradient over 30 min, obtaining the desired sulfone as a white solid (yield, 83%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.19 (d, <i>J</i> = 9.4 Hz, 1 H), 6.89 (t, <i>J</i> = 9.4 Hz, 1 H), 6.35 (d, <i>J</i> = 10.4 Hz, 1 H), 4.98 (t, <i>J</i> = 10.0 Hz, 1 H), 4.66 (dd, <i>J</i> = 2.9, 9.6 Hz, 1 H), 4.61 (t, <i>J</i> = 7.4 Hz, 1 H), 4.39 (dd, <i>J</i> = 5.1, 8.6 Hz, 1 H), 4.15 (td, <i>J</i> = 6.7, 10.0 Hz, 1 H), 3.76 (dd, <i>J</i> = 5.5, 13.7 Hz, 1 H), 3.69–3.58 (m, 2 H), 3.34–3.23 (m, 1 H), 3.23–3.11 (m, 2 H), 3.11–2.93 (m, 2 H), 2.90 (s, 3 H), 2.63 (dt, <i>J</i> = 3.1, 6.8 Hz, 1 H), 2.52 (dd, <i>J</i> = 9.6, 12.3 Hz, 1 H), 2.46–2.35 (m, 1 H), 2.21–2.10 (m, 3H), 2.10–1.95 (m, 2 H), 1.95–1.83 (m, 1 H), 0.92 (d, <i>J</i> = 7.0 Hz, 3 H), 0.89 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 175.2, 172.4, 172.2, 171.1, 170.5, 62.8, 58.7, 56.5, 51.6, 51.0, 48.6, 47.4, 40.4, 37.9, 34.2, 29.9, 28.8, 28.0, 26.1, 25.0, 22.5, 19.9, 16.7. MS (ESI): found [M + H]<sup>+</sup>, 528.4.</div></div><div id="sec4_9_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)<sub>2</sub>CH<sub>3</sub> (<b>9b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>9b</b> having the methylsulfone warhead on pendent carboxylic acid was synthesized from <b>7b</b> by following the above general procedure for sulfones synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/borax and purified by HPLC using 5–48% acetonitrile gradient over 30 min, obtaining the desired sulfone as a white solid (yield, 85%; purity by LC-MS, 97%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.11 (d, <i>J</i> = 9.4 Hz, 1 H), 6.78 (t, <i>J</i> = 4.7 Hz, 1 H), 6.44 (d, <i>J</i> = 6.3 Hz, 1 H), 4.69 (t, <i>J</i> = 11.2 Hz, 1 H), 4.54 (dd, <i>J</i> = 3.7, 9.6 Hz, 1 H), 4.43 (dd, <i>J</i> = 4.5, 8.8 Hz, 1 H), 4.15 (dd, <i>J</i> = 6.8, 10.0 Hz, 1 H), 3.93–3.83 (m, 1 H), 3.71–3.54 (m, 4 H), 3.31–3.19 (m, 2 H), 3.10–3.01 (m, 2 H), 2.93 (s, 3 H), 2.88 (d, <i>J</i> = 4.7 Hz, 1 H), 2.44–2.30 (m, 2 H), 2.26–1.97 (m, 6 H), 1.91 (d, <i>J</i> = 7.0 Hz, 1 H), 0.93 (d, <i>J</i> = 4.3 Hz, 3 H), 0.91 (d, <i>J</i> = 3.9 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.5, 173.7, 172.6, 171.5, 169.7, 62.7, 58.4, 56.0, 53.0, 51.5, 47.8, 47.4, 40.8, 37.6, 34.3, 29.7, 28.5, 28.1, 26.0, 25.0, 22.5, 20.3, 17.0. MS (ESI): found [M + Na]<sup>+</sup>, 528.4.</div></div><div id="sec4_9_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>OH (<b>10a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>10a</b> having the <i>n</i>-propylalcohol warhead on pendent carboxylic acid was synthesized from <b>3a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 3-aminopropan-1-ol and purified by HPLC using 11–40% acetonitrile gradient over 30 min, obtaining the desired alcohol as viscous oil (yield, 60%; purity by LC-MS, >97%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.18 (t, <i>J</i> = 5.7 Hz, 1 H), 7.04 (d, <i>J</i> = 9.6 Hz, 1 H), 6.28 (d, <i>J</i> = 10.4 Hz, 1 H), 4.98 (t, <i>J</i> = 10.0 Hz, 1 H), 4.68 (dd, <i>J</i> = 2.9, 9.8 Hz, 1 H), 4.63 (t, <i>J</i> = 6.7 Hz, 1 H), 4.42 (dd, <i>J</i> = 4.8, 8.7 Hz, 1 H), 4.16 (td, <i>J</i> = 6.6, 9.9 Hz, 1 H), 3.79–3.69 (m, 3 H), 3.67–3.56 (m, 4 H), 3.38–3.27 (m, 1 H), 2.69–2.59 (m, 1 H), 2.52 (dd, <i>J</i> = 9.5, 12.4 Hz, 1 H), 2.46–2.34 (m, 1 H), 2.17–2.02 (m, 4 H), 1.97–1.83 (m, 4 H), 1.73–1.71 (m, 1 H), 0.92 (d, <i>J</i> = 6.8 Hz, 3 H), 0.89 (d, <i>J</i> = 6.8 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 175.0, 172.5, 172.0, 171.1, 170.9, 62.8, 60.5, 58.6, 56.6, 50.7, 48.3, 47.4, 38.0, 34.5, 31.1, 29.8, 28.8, 28.1, 26.0, 25.0, 19.9, 16.7. MS (ESI): found [M + H]<sup>+</sup>, 466.4.</div></div><div id="sec4_9_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>OH (<b>10b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>10b</b> having the <i>n</i>-propylalcohol warhead on pendent carboxylic acid was synthesized from <b>3b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 3-aminopropan-1-ol and purified by HPLC using 11–40% acetonitrile gradient over 30 min, obtaining the desired alcohol as viscous oil (yield, 75%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.38–7.17 (m, 2 H), 6.51 (bs, 1H), 4.78–4.67 (m, 1 H), 4.62–4.39 (m, 2 H), 4.32–4.15 (m, 1 H), 4.15–3.96 (m, 1 H), 3.84–3.67 (m, 3 H), 3.67–3.47 (m, 1 H), 3.44–3.24 (m, 1 H), 3.24–3.05 (m, 1 H), 3.05–2.92 (m, 1 H), 2.54–2.35 (m, 2 H), 2.35–2.23 (m, 2 H), 2.20–2.13 (m, 2 H), 2.13–1.92 (m, 3 H), 1.91–1.64 (m, 2 H), 1.22–0.95 (m, 6 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.2, 173.2, 172.5, 172.1, 170.9, 170.0, 65.4, 62.5, 58.4, 56.6, 53.0, 47.8, 47.4, 34.4, 31.6, 29.7, 28.4, 28.1, 28.0, 25.9, 24.9, 20.1, 17.1. MS (ESI): found [M + H]<sup>+</sup>, 466.4.</div></div><div id="sec4_9_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S-SCH<sub>2</sub>CH<sub>2</sub>NHP(O)(OPh) (<b>11a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>11a</b> having the phosphate warhead on pendent carboxylic acid was synthesized by following from <b>3a</b> the above general procedure for DPPA peptide coupling of coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 52%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.39–7.30 (m, 4 H), 7.28–7.22 (m, 8 H), 7.22–7.14 (m, 2 H), 7.09 (d, <i>J</i> = 9.8 Hz, 1 H), 7.03 (t, <i>J</i> = 5.3 Hz, 1 H), 6.46 (d, <i>J</i> = 10.4 Hz, 1 H), 5.00 (t, <i>J</i> = 10.0 Hz, 1 H), 4.64 (dd, <i>J</i> = 3.1, 9.4 Hz, 1 H), 4.60 (t, <i>J</i> = 7.0 Hz, 1 H), 4.38 (dd, <i>J</i> = 5.1, 8.2 Hz, 1 H), 4.15–4.06 (m, 1 H), 3.70–3.48 (m, 5 H), 3.47–3.35 (m, 2 H), 2.83–2.72 (m, 4 H), 2.61 (dt, <i>J</i> = 3.5, 6.8 Hz, 1 H), 2.55 (dd, <i>J</i> = 9.6, 12.7 Hz, 1 H), 2.39–2.26 (m, 1 H), 2.17–2.03 (m, 4 H), 2.03–1.91 (m, 2 H), 1.87 (dd, <i>J</i> = 8.0, 12.3 Hz, 1 H), 0.91 (d, <i>J</i> = 7.0 Hz, 3 H), 0.88 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.9, 172.5, 172.1, 171.0, 170.6, 150.6, 129.8, 125.2, 120.8, 62.8, 58.5, 56.5, 50.9, 48.3, 47.5, 40.2, 39.9, 37.3, 34.7, 29.9, 29.7, 28.9, 28.0, 26.0, 25.0, 19.9, 16.8. MS (ESI): found [M + Na]<sup>+</sup>, 775.5.</div></div><div id="sec4_9_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S-SCH<sub>2</sub>CH<sub>2</sub>NHP(O)(OPh) (<b>11b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>11b</b> having the phosphate warhead on pendent carboxylic acid was synthesized from <b>3b</b> by following the above general procedure for DPPA peptide coupling of coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 56%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.35 (t, <i>J</i> = 8.0 Hz, 4 H), 7.28–7.23 (m, 4 H), 7.18 (t, <i>J</i> = 7.2 Hz, 3 H), 7.12 (d, <i>J</i> = 9.4 Hz, 1 H), 6.47 (d, <i>J</i> = 6.3 Hz, 1 H), 4.66 (t, <i>J</i> = 6.7 Hz, 1 H), 4.49 (dd, <i>J</i> = 4.7, 9.4 Hz, 1 H), 4.44 (dd, <i>J</i> = 4.7, 8.6 Hz, 1 H), 4.21–4.14 (m, 1 H), 4.14–4.05 (m, 1 H), 4.04–3.95 (m, 1 H), 3.64 (t, <i>J</i> = 6.8 Hz, 2 H), 3.62–3.56 (m, 1 H), 3.55–3.44 (m, 1 H), 3.39 (dd, <i>J</i> = 5.9, 12.1 Hz, 2 H), 3.03 (dd, <i>J</i> = 4.3, 12.5 Hz, 1 H), 2.94–2.85 (m, 1 H), 2.79–2.72 (m, 2 H), 2.71 (t, <i>J</i> = 6.6 Hz, 2 H), 2.43–2.26 (m, 2 H), 2.26–2.14 (m, 2 H), 2.14–2.04 (m, 3 H), 2.04–1.91 (m, 2 H), 0.91 (d, <i>J</i> = 6.7 Hz, 6 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.1, 173.3, 172.2, 169.8, 167.0, 150.5, 129.8, 125.2, 120.2, 120.2, 62.6, 58.3, 56.6, 53.0, 47.7, 47.4, 40.4, 39.7, 38.4, 37.2, 34.4, 29.7, 28.5, 28.1, 25.9, 24.9, 20.1, 17.1. MS (ESI): found: [M + Na]<sup>+</sup>, 775.5.</div></div><div id="sec4_9_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOMe (<b>12a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>12a</b> having the methylcarboxylate warhead on pendent carboxylic acid was synthesized from <b>3a</b> by following the above general procedure for EDCl peptide coupling of coupling of pendent carboxylic acid with methyl 4-aminobutanoate and purified by HPLC using 11–45% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 86%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.18 (d, <i>J</i> = 9.4 Hz, 1 H), 6.85 (t, <i>J</i> = 4.7 Hz, 1 H), 6.41 (d, <i>J</i> = 10.2 Hz, 1 H), 5.00 (t, <i>J</i> = 9.8 Hz, 1 H), 4.67 (dd, <i>J</i> = 2.3, 9.4 Hz, 1 H), 4.61 (t, <i>J</i> = 6.7 Hz, 1 H), 4.45 (dd, <i>J</i> = 5.1, 8.2 Hz, 1 H), 4.20–4.11 (m, 1 H), 3.67 (s, 3 H), 3.63 (t, <i>J</i> = 7.1 Hz, 2 H), 3.56 (d, <i>J</i> = 12.9 Hz, 1 H), 3.44–3.34 (m, 1 H), 3.34–3.23 (m, 1 H), 2.68–2.60 (m, 1 H), 2.56 (dd, <i>J</i> = 10.0, 12.3 Hz, 1 H), 2.47–2.36 (m, 3 H), 2.21–2.02 (m, 5 H), 1.99–1.81 (m, 4 H), 0.93 (d, <i>J</i> = 7.0 Hz, 3 H), 0.89 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.8, 173.6, 172.5, 172.3, 171.2, 170.4, 62.7, 58.6, 56.5, 51.7, 51.0, 48.3, 47.4, 39.4, 34.8, 31.2, 29.9, 28.9, 28.1, 26.0, 25.0, 24.4, 19.9, 16.8. MS (ESI): found [M + H]<sup>+</sup>, 508.5.</div></div><div id="sec4_9_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOMe (<b>12b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>12b</b> having the methylcarboxylate warhead on pendent carboxylic acid was synthesized from <b>3b</b> by following the above general procedure for EDCl peptide coupling of coupling of pendent carboxylic acid with methyl 4-aminobutanoate and purified by HPLC using 11–45% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 85%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.11 (d, <i>J</i> = 9.4 Hz, 1 H), 6.92 (t, <i>J</i> = 5.3 Hz, 1 H), 6.37 (d, <i>J</i> = 6.3 Hz, 1 H), 4.68 (t, <i>J</i> = 6.7 Hz, 1 H), 4.52 (dd, <i>J</i> = 4.3, 9.4 Hz, 1 H), 4.44 (dd, <i>J</i> = 4.3, 8.6 Hz, 1 H), 4.22–4.11 (m, 1 H), 3.94–3.85 (m, 1 H), 3.72–3.68 (m, 1 H), 3.67 (s, 3 H), 3.66–3.59 (m, 2 H), 3.39–3.22 (m, 2 H), 3.07 (dd, <i>J</i> = 4.1, 12.7 Hz, 1 H), 2.93 (t, <i>J</i> = 12.5 Hz, 1 H), 2.43–2.33 (m, 1 H), 2.36 (t, <i>J</i> = 6.9 Hz, 2 H), 2.29–2.19 (m, 1 H), 2.16–2.06 (m, 3 H), 2.06–1.98 (m, 2 H), 1.83 (p, <i>J</i> = 7.6 Hz, 2 H), 1.82–1.71 (m, 2 H), 0.93 (d, <i>J</i> = 6.9 Hz, 3 H), 0.93 (d, <i>J</i> = 6.8 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.3, 173.7, 173.5, 172.5, 171.6, 169.4, 62.7, 58.3, 56.2, 53.4, 51.7, 47.8, 47.4, 39.0, 34.3, 31.2, 29.8, 28.5, 28.1, 25.9, 25.0, 24.6, 20.2, 17.1. MS (ESI): found [M + H]<sup>+</sup>, 508.5.</div></div><div id="sec4_9_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOH (<b>13a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>13a</b> having the carboxylic acid warhead on pendent carboxylic acid was synthesized from <b>12a</b> by following the above general procedure for general procedure for LiOH base mediated hydrolysis of methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 94%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.68 (t, <i>J</i> = 4.9 Hz, 1 H), 7.16 (d, <i>J</i> = 9.4 Hz, 1 H), 6.21 (d, <i>J</i> = 10.2 Hz, 1 H), 5.08 (bs, 1 H), 4.98 (t, <i>J</i> = 10.0 Hz, 1 H), 4.76 (dd, <i>J</i> = 3.1, 9.4 Hz, 1 H), 4.60 (t, <i>J</i> = 6.8 Hz, 1 H), 4.40 (dd, <i>J</i> = 5.1, 8.6 Hz, 1 H), 4.21–4.05 (m, 1 H), 3.78–3.69 (m, 1 H), 3.68–3.57 (m, 2 H), 3.42–3.25 (m, 2 H), 2.67–2.58 (m, 1 H), 2.58–2.49 (m, 2 H), 2.42 (dd, <i>J</i> = 6.5, 13.1 Hz, 2 H), 2.22–1.99 (m, 5 H), 1.99–1.78 (m, 4 H), 0.93 (d, <i>J</i> = 6.7 Hz, 3 H), 0.89 (d, <i>J</i> = 7.0 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 177.0, 174.8, 173.6, 171.5, 171.2, 170.4, 63.0, 58.4, 56.7, 50.5, 48.4, 47.4, 40.9, 34.1, 31.8, 29.9, 28.8, 28.1, 26.0, 25.0, 22.5, 19.9, 16.8. MS (ESI): found [M + H]<sup>+</sup>, 494.4.</div></div><div id="sec4_9_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOH (<b>13b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>13b</b> having the carboxylic acid warhead on pendent carboxylic acid was synthesized from <b>12b</b> by following the above general procedure for general procedure for LiOH base mediated hydrolysis of methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 95%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.20 (bs, 1 H), 7.12 (d, <i>J</i> = 9.0 Hz, 1 H), 6.50 (d, <i>J</i> = 5.9 Hz, 1 H), 4.92 (bs, 1 H), 4.67 (t, <i>J</i> = 6.5 Hz, 1 H), 4.48 (dd, <i>J</i> = 4.9, 9.6 Hz, 1 H), 4.42 (dd, <i>J</i> = 4.5, 8.0 Hz, 1 H), 4.19 (q, <i>J</i> = 9.4 Hz, 1 H), 4.07–3.98 (m, 1 H), 3.69 (t, <i>J</i> = 6.1 Hz, 2 H), 3.64 (q, <i>J</i> = 8.5 Hz, 1 H), 3.39 (dd, <i>J</i> = 6.3, 13.3 Hz, 1 H), 3.30 (dd, <i>J</i> = 6.3, 12.9 Hz, 1 H), 3.06 (dd, <i>J</i> = 3.7, 12.3 Hz, 1 H), 2.92 (t, <i>J</i> = 12.4 Hz, 1 H), 2.40 (t, <i>J</i> = 6.7 Hz, 2 H), 2.38–2.29 (m, 2 H), 2.27 (d, <i>J</i> = 5.1 Hz, 1 H), 2.15–2.08 (m, 4 H), 2.04–1.95 (m, 4 H), 1.85 (quin, <i>J</i> = 6.6 Hz, 2 H), 0.94 (br. d, <i>J</i> = 6.3 Hz, 3 H), 0.92 (d, <i>J</i> = 6.3 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 176.1, 174.0, 172.9, 172.3, 172.2, 170.0, 62.6, 58.2, 56.9, 52.6, 47.8, 47.5, 39.2, 34.5, 31.4, 29.7, 28.6, 28.1, 25.9, 25.0, 24.4, 20.1, 17.3. MS (ESI): found [M + H]<sup>+</sup>, 494.4.</div></div><div id="sec4_9_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CONHOH (<b>14a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>14a</b> having the hydroxamic acid warhead on pendent carboxylic acid was synthesized from <b>12a</b> by following the above general procedure for general procedure for hydroxamic acid synthesis from the methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired hydroxamic acid peptide as a white solid (yield, 87%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm: 10.30 (bs, 1 H), 8.00 (t, <i>J</i> = 4.7 Hz, 1 H), 6.91 (d, <i>J</i> = 8.2 Hz, 1 H), 6.50 (d, <i>J</i> = 9.8 Hz, 1 H), 4.68 (t, <i>J</i> = 7.0 Hz, 1 H), 4.59 (t, <i>J</i> = 6.3 Hz, 1 H), 4.24 (d, <i>J</i> = 3.5 Hz, 1 H), 4.16 (dd, <i>J</i> = 5.3, 8.4 Hz, 1 H), 3.96 (d, <i>J</i> = 8.6 Hz, 1 H), 3.74–3.64 (m, 1 H), 3.63–3.53 (m, 4 H), 3.16–3.06 (m, 2 H), 3.06–2.97 (m, 1 H), 2.51–2.40 (m, 1 H), 2.36–2.30 (m, 1 H), 2.29–2.25 (m, 1 H), 2.19–2.08 (m, 1 H), 2.06–1.99 (m, 2 H), 1.95 (t, <i>J</i> = 7.0 Hz, 2 H), 1.85–1.71 (m, 3 H), 1.67–1.59 (p, <i>J</i> = 6.8 Hz, 2 H), 0.84 (d, <i>J</i> = 6.7 Hz, 3 H), 0.79 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm: 173.7, 171.5, 171.0, 170.3, 170.2, 169.3, 62.5, 58.0, 56.4, 50.2, 47.9, 47.3, 39.0, 36.1, 30.2, 29.9, 29.1, 28.2, 25.8, 25.5, 24.9, 20.1, 17.6. MS (ESI): found [M + H]<sup>+</sup>, 509.5.</div></div><div id="sec4_9_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CONHOH (<b>14b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>14b</b> having the hydroxamic acid warhead on pendent carboxylic acid was synthesized from <b>12b</b> by following the above general procedure for general procedure for hydroxamic acid synthesis from the methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired hydroxamic acid peptide as a white solid (yield, 85%; purity by LC-MS, 99%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm: 10.29 (bs, 1 H), 7.73 (t, <i>J</i> = 4.9 Hz, 1 H), 6.89 (d, <i>J</i> = 5.5 Hz, 1 H), 6.37 (d, <i>J</i> = 9.8 Hz, 1 H), 4.68–4.62 (m, 1 H), 4.27–4.14 (m, 3 H), 4.01–3.86 (m, 2 H), 3.75–3.66 (m, 3 H), 3.61–3.58 (m, 2 H), 3.05–2.89 (m, 3 H), 2.30–2.04 (m, 6 H), 2.03–1.88 (m, 6 H), 1.88–1.75 (m, 4 H), 1.63–1.54 (m, 2 H), 0.84 (d, <i>J</i> = 6.8 Hz, 3 H), 0.78 (d, <i>J</i> = 6.9 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm: 173.9, 172.0, 171.6, 171.5, 169.4, 169.4, 62.5, 58.0, 56.3, 51.5, 47.6, 47.5, 38.8, 34.9, 30.2, 29.9, 28.6, 28.3, 25.9, 25.6, 24.9, 20.6, 17.6. MS (ESI): found [M + Na]<sup>+</sup>, 509.5.</div></div><div id="sec4_9_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S-SCH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> (<b>15a</b>)</h4><div class="NLM_p last">The amine-containing peptide <b>15a</b> was synthesized from <b>3a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired homodimer as a viscous oil (yield, 65%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 8.24 (br. s., 2 H), 7.76 (t, <i>J</i> = 6.1 Hz, 1 H), 7.19 (d, <i>J</i> = 9.0 Hz, 1 H), 6.30 (d, <i>J</i> = 10.6 Hz, 1 H), 5.04 (t, <i>J</i> = 10.0 Hz, 1 H), 4.59 (t, <i>J</i> = 7.2 Hz, 1 H), 4.50 (dd, <i>J</i> = 3.3, 9.6 Hz, 1 H), 4.43 (dd, <i>J</i> = 6.1, 8.0 Hz, 1 H), 4.20–4.12 (m, 1 H), 3.80–3.72 (m, 1 H), 3.70–3.52 (m, 4 H), 3.38–3.26 (m, 2 H), 3.19–3.10 (m, 2 H), 3.02–2.91 (m, 2 H), 2.62–2.53 (m, 2 H), 2.45–2.36 (m, 3 H), 2.20–2.09 (m, 4 H), 2.03–1.89 (m, 2 H), 0.86 (d, <i>J</i> = 6.9 Hz, 3 H), 0.85 (d, <i>J</i> = 7.0 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 543.5.</div></div><div id="sec4_9_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S-SCH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> (<b>15b</b>)</h4><div class="NLM_p last">The amine-containing peptide <b>15b</b> was synthesized from <b>3b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired homodimer as a viscous oil (yield, 67%; purity by LC-MS, 97%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 8.11 (t, <i>J</i> = 6.5 Hz, 1 H), 8.06 (bs, 2 H), 6.76 (d, <i>J</i> = 9.8 Hz, 1 H), 6.54 (d, <i>J</i> = 7.0 Hz, 1 H), 4.61 (dd, <i>J</i> = 6.1, 7.2 Hz, 1 H), 4.47–4.39 (m, 1 H), 4.35–4.29 (m, 1 H), 4.20 (dd, <i>J</i> = 7.0, 16.4 Hz, 1 H), 3.75–3.59 (m, 3 H), 3.39–3.28 (m, 3 H), 3.10–2.90 (m, 3 H), 2.76 (t, <i>J</i> = 12.7 Hz, 1 H), 2.41–2.31 (m, 1 H), 2.31–2.21 (m, 1 H), 2.20–2.08 (m, 3 H), 2.06–1.92 (m, 3 H), 1.82–1.72 (m, 3 H), 0.97 (d, <i>J</i> = 6.3 Hz, 3 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 543.5.</div></div><div id="sec4_9_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>Cyclo</i>-Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> (<b>16a</b>) and <i>Cyclo</i>-<sup>D</sup>Asp(<sup>D</sup>Pro-Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> (<b>16b</b>)</h4><div class="NLM_p last">The amine-containing unseparable peptides <b>16a</b> and <b>16b</b> were synthesized by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 1,3-diaminopropane and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtaining the desired diastereomeric mixture (3:1) of amino propyl amide as a viscous oil (yield, 81%; purity by LC-MS, 99%); Diastereomer <b>16a</b>: <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.04 (d, <i>J</i> = 9.0 Hz, 1 H), 6.57 (d, <i>J</i> = 10.2 Hz, 1 H), 5.07 (dt, <i>J</i> = 3.2, 9.8 Hz, 1 H), 4.63 (t, <i>J</i> = 7.0 Hz, 1 H), 4.48 (dd, <i>J</i> = 3.9, 8.6 Hz, 1 H), 4.36 (dd, <i>J</i> = 4.7, 8.6 Hz, 1 H), 4.21–4.11 (m, 1 H), 3.77 (dt, <i>J</i> = 6.7, 9.4 Hz, 1 H), 3.72–3.62 (m, 2 H), 3.62–3.54 (m, 1 H), 3.38–3.31 (m, 3 H), 2.66 (dd, <i>J</i> = 9.6, 13.5 Hz, 1 H), 2.55–2.44 (m, 1 H), 2.36 (qd, <i>J</i> = 8.0, 12.3 Hz, 1 H), 2.22–2.07 (m, 5 H), 2.07–1.82 (m, 6 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H), 0.88 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.2, 170.8, 170.7, 169.3, 168.3, 167.9, 61.9, 57.5, 55.5, 48.4, 47.4, 46.4, 34.8, 29.1, 28.4, 27.8, 27.3, 25.2, 24.2, 20.0, 19.2, 16.4. Diastereomer <b>16b</b>: <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 6.99 (d, <i>J</i> = 9.4 Hz, 1 H), 6.50 (d, <i>J</i> = 6.3 Hz, 1 H), 5.02–4.95 (m, 1 H), 4.69 (dd, <i>J</i> = 6.3, 7.4 Hz, 1 H), 4.52 (dd, <i>J</i> = 3.5, 8.6 Hz, 1 H), 4.42 (dd, <i>J</i> = 4.7, 9.4 Hz, 1 H), 4.12–4.07 (m,1 H), 3.72–3.62 (m, 2 H), 3.62–3.54 (m, 1 H), 3.38–3.31 (m, 2 H), 3.01 (dd, <i>J</i> = 4.9, 12.3 Hz, 17 H), 2.89 (t, <i>J</i> = 12.0 Hz, 16 H) 2.55–2.36 (m, 2 H), 2.22–2.07 (m, 5 H), 2.07–1.82 (m, 6 H), 0.91 (d, <i>J</i> = 6.7 Hz, 3 H), 0.88 (d, <i>J</i> = 6.7 Hz, 3 H). MS (ESI): found [M + Na]<sup>+</sup>, 465.4.</div></div><div id="sec4_9_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-OBn (<b>2a</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide <b>2b</b> was synthesized by following the above general procedure for DPPA macrolactamisation and purified by silical gel column chromatography (EtOAc) as a white solid (yield, 67%; purity by LC-MS, >97%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.74 (d, <i>J</i> = 10.0 Hz, 1 H), 7.41–7.27 (m, 5 H), 6.70 (d, <i>J</i> = 7.6 Hz, 1 H), 5.31 (d, <i>J</i> = 12.3 Hz, 1 H), 5.13 (d, <i>J</i> = 12.5 Hz, 1 H), 4.78 (dd, <i>J</i> = 4.7, 7.8 Hz, 1 H), 4.75 (d, <i>J</i> = 7.4 Hz, 3 H), 4.71 (dt, <i>J</i> = 2.7, 5.3 Hz, 3 H), 4.39–4.24 (m, 2 H), 3.71–3.55 (m, 1 H), 3.55–3.39 (m, 2 H), 3.12 (dd, <i>J</i> = 5.7, 15.5 Hz, 1 H), 3.01 (dd, <i>J</i> = 3.3, 15.5 Hz, 1 H), 2.90–2.84 (m, 1 H), 2.56 (dd, <i>J</i> = 6.8, 12.3 Hz, 1 H), 2.37–2.18 (m, 3 H), 2.18–2.05 (m, 2 H), 2.03–1.90 (m, 2 H), 1.83–1.73 (m, 1 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H), 0.89 (d, <i>J</i> = 6.8 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 172.5, 170.6, 170.2, 170.1, 168.9, 135.4, 128.5, 128.2, 128.0, 67.4, 59.0, 57.5, 57.3, 49.3, 47.3, 47.2, 36.2, 28.3, 27.3, 25.8, 25.5, 25.0, 19.8, 17.5. MS (ESI): found [M + H]<sup>+</sup>, 499.4.</div></div><div id="sec4_9_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-OBn (<b>2b</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide <b>2b</b> was synthesized by following the above general procedure for DPPA macrolactamisation and purified by silical gel column chromatography (EtOAc) as a white solid (yield, 73%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.41–7.29 (m, 5 H), 7.01 (d, <i>J</i> = 8.6 Hz, 1 H), 6.77 (d, <i>J</i> = 8.6 Hz, 1 H), 5.21 (s, 2 H), 4.73–4.66 (m, 1 H), 4.61 (dd, <i>J</i> = 2.2, 8.8 Hz, 1 H), 4.56 (dd, <i>J</i> = 2.3, 8.2 Hz, 1 H), 4.08 (dd, <i>J</i> = 6.3, 9.0 Hz, 1 H), 3.86–3.76 (m, 1 H), 3.76–3.68 (m, 1 H), 3.51 (td, <i>J</i> = 7.6, 11.8 Hz, 2 H), 3.04 (dd, <i>J</i> = 5.1, 14.9 Hz, 1 H), 2.83 (dd, <i>J</i> = 0.8, 5.1 Hz,1 H), 2.57–2.44 (m, 1 H), 2.42–2.37 (m, 1 H), 2.32 (dd, <i>J</i> = 3.3, 15.1 Hz, 1 H), 2.16–2.00 (m, 4 H), 1.99–1.91 (m, 2 H), 1.02 (d, <i>J</i> = 7.0 Hz, 3 H), 1.00 (d, <i>J</i> = 7.1 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.1, 171.2, 170.4, 170.2, 168.8, 135.8, 128.4, 128.3, 128.0, 67.1, 61.6, 60.4, 60.1, 49.2, 46.9, 46.9, 36.0, 30.2, 29.5, 26.6, 25.7, 22.4, 19.3, 18.3. MS (ESI): found [M + H]<sup>+</sup>, 499.4.</div></div><div id="sec4_9_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-OH (<b>4a</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide pendent carboxylic acid <b>4a</b> was synthesized from <b>2a</b> by following the above general procedure for catalytic hydrogenation of benzyl ester, obtaining the desired pendent carboxylic acid as a white solid (yield, 94%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.78 (d, <i>J</i> = 10.2 Hz, 1 H), 7.10 (d, <i>J</i> = 6.7 Hz, 1 H), 5.06 (bs, 1 H), 4.73 (t, <i>J</i> = 7.0 Hz, 2 H), 4.66 (q, <i>J</i> = 5.6 Hz, 1 H), 4.25 (dd, <i>J</i> = 8.8, 9.3 Hz, 1 H), 4.19 (dd, <i>J</i> = 2.9, 10.0 Hz, 1 H), 3.66 (q, <i>J</i> = 8.5 Hz, 1 H), 3.58–3.50 (m, 1 H), 3.46 (q, <i>J</i> = 8.5 Hz, 1 H), 3.26 (dd, <i>J</i> = 5.9, 15.7 Hz, 1 H), 2.95 (dd, <i>J</i> = 4.5, 15.5 Hz, 1 H), 2.48 (dd, <i>J</i> = 7.2, 12.3 Hz, 1 H), 2.39–2.24 (m, 2 H), 2.20 (q, <i>J</i> = 7.2 Hz, 1 H), 2.10 (q, <i>J</i> = 7.0 Hz, 2 H), 2.04–1.90 (m, 2 H), 1.87–1.74 (m, 1 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H), 0.89 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 172.5, 172.2, 170.9, 170.4, 169.2, 59.1, 58.6, 57.7, 49.3, 47.5, 47.2, 35.7, 28.7, 27.4, 25.7, 25.5, 24.9, 19.5, 17.9. MS (ESI): found [M + H]<sup>+</sup>, 409.4.</div></div><div id="sec4_9_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-OH (<b>4b</b>)</h4><div class="NLM_p last">Cyclic tetrapeptide pendent carboxylic acid <b>4b</b> was synthesized from <b>2b</b> by following the above general procedure for catalytic hydrogenation of benzyl ester, obtainimg the desired pendent carboxylic acid as a white solid (yield, 91%; purity by LC-MS, 97%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.20 (d, <i>J</i> = 8.2 Hz, 1 H), 6.94 (d, <i>J</i> = 8.2 Hz, 1 H), 6.55 (bs, 1 H), 4.74 (dd, <i>J</i> = 4.5, 7.4 Hz, 1 H), 4.65 (dd, <i>J</i> = 2.6, 8.2 Hz, 1 H), 4.60 (dd, <i>J</i> = 2.7, 8.0 Hz, 1 H), 4.22–4.14 (m, 1 H), 4.08 (dd, <i>J</i> = 6.7, 8.2 Hz, 1 H), 3.77–3.70 (m, 1 H), 3.57–3.45 (m, 2 H), 3.10 (dd, <i>J</i> = 4.9, 15.1 Hz, 1 H), 2.54–2.33 (m, 2 H), 2.35 (dd, <i>J</i> = 3.2, 15.2 Hz, 1 H), 2.19–1.71 (m, 7 H), 1.05 (d, <i>J</i> = 6.7 Hz, 3 H), 1.03 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.2, 173.1, 173.1, 173.0, 169.7, 61.8, 60.7, 60.3, 51.2, 47.1, 47.0, 39.8, 34.5, 30.1, 29.4, 27.6, 26.7, 25.7, 22.4, 19.3, 17.9. MS (ESI): found [M + H]<sup>+</sup>, 409.4.</div></div><div id="sec4_9_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Homdimer: [<i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> (<b>36a</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>36a</b> was synthesized <b>4a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtainimg the desired homodimer as a white solid (yield, 77%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.77 (d, <i>J</i> = 9.0 Hz, 1 H), 7.25 (bs, 1 H), 7.22 (bs, 1 H), 4.82–4.74 (m, 1 H), 4.73–4.66 (m, 2 H), 4.27 (t, <i>J</i> = 8.2 Hz, 1 H), 4.19 (t, <i>J</i> = 7.6 Hz, 1 H), 4.13–4.00 (m, 1 H), 3.72–3.60 (m, 2 H), 3.59–3.44 (m, 2 H), 3.01 (dd, <i>J</i> = 8.0, 14.0 Hz, 1 H), 2.92 (dd, <i>J</i> = 6.2, 14.4 Hz, 1 H), 2.78 (t, <i>J</i> = 6.5 Hz, 2 H), 2.50–2.39 (m, 1 H), 2.37–2.18 (m, 3 H), 2.12 (q, <i>J</i> = 6.8 Hz, 2 H), 2.06–1.90 (m, 2 H), 1.89–1.76 (m, 1 H), 0.92 (d, <i>J</i> = 6.7 Hz, 3 H), 0.89 (d, <i>J</i> = 7.0 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 172.5, 171.7, 170.8, 169.7, 59.3, 58.9, 57.9, 49.8, 47.5, 47.3, 47.2, 38.6, 37.6, 36.7, 29.3, 27.6, 25.9, 25.8, 25.0, 19.7, 17.6. MS (ESI): found [M + H]<sup>+</sup>, 933.7.</div></div><div id="sec4_9_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>SH (<b>17a</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>17a</b> was synthesized by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer (<b>36a</b>), obtaining the desired pendent thiol as a white solid (yield, 74%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 8.02 (bs, 1 H), 7.75 (d, <i>J</i> = 9.4 Hz, 1 H), 7.08 (t, <i>J</i> = 5.0 Hz, 1 H), 4.78 (dt, <i>J</i> = 5.1, 8.2 Hz, 1 H), 4.73–4.64 (m, 2 H), 4.25–4.07 (m, 2 H), 3.75–3.60 (m, 2 H), 3.57–3.46 (m, 2 H), 3.42 (quin, <i>J</i> = 6.5 Hz, 1 H), 3.04 (dd, <i>J</i> = 5.5, 14.5 Hz, 1 H), 2.89 (dd, <i>J</i> = 8.4, 14.7 Hz, 1 H), 2.68–2.59 (m, 2 H), 2.49–2.20 (m, 3 H), 2.20–2.11 (m, 2 H), 2.11–1.84 (m, 4 H), 1.52 (t, <i>J</i> = 8.6 Hz, 1 H), 0.95 (d, <i>J</i> = 7.0 Hz, 3 H), 0.91 (d, <i>J</i> = 6.7 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 468.4.</div></div><div id="sec4_9_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Homodimer: [<i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> (<b>36b</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>36b</b> was synthesized from <b>4b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtainimg the desired homodimer as a white solid (yield, 73%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.65 (d, <i>J</i> = 8.5 Hz, 2 H), 6.85 (bs, 1 H), 4.74–4.67 (m, 3 H), 4.59–4.41 (m, 1 H), 4.14–4.04 (m, 1 H), 3.78–3.65 (m, 2 H), 3.57–3.49 (m, 3 H), 3.21–3.15 (m, 1 H), 3.09–2.90 (m, 1 H), 2.79 (t, <i>J</i> = 6.3 Hz, 2 H), 2.48–2.40 (m, 1 H), 2.33–2.25 (m, 2 H), 2.20–2.14 (m, 1 H), 2.06–1.98 (m, 4 H), 1.91–1.78 (m, 1 H), 0.97 (d, <i>J</i> = 7.2 Hz, 3 H), 0.94 (d, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 933.7.</div></div><div id="sec4_9_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>SH (<b>17b</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>17b</b> was synthesized by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer (<b>36b</b>), obtaining the desired pendent thiol as a white solid (yield, 71%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.64 (d, <i>J</i> = 9.8 Hz, 1 H), 7.45 (t, <i>J</i> = 5.3 Hz, 1 H), 6.87 (d, <i>J</i> = 6.7 Hz, 1 H), 4.76–4.69 (m, 2 H), 4.68–4.57 (m, 1 H), 4.17–4.09 (m, 2 H), 3.85–3.65 (m, 1 H), 3.61–3.47 (m, 4 H), 3.27 (dd, <i>J</i> = 11.3, 16.0 Hz, 1 H), 2.87 (dd, <i>J</i> = 5.1, 16.0 Hz, 1 H), 2.74–2.68 (m, 1 H), 2.67–2.61 (m, 1 H), 2.50–2.42 (m, 1 H), 2.35–2.29 (m, 1 H), 2.22–1.99 (m, 7 H), 1.45 (t, <i>J</i> = 8.6 Hz, 1 H), 0.96 (d, <i>J</i> = 6.9 Hz, 3 H), 0.94 (d, <i>J</i> = 6.9 Hz, 6 H). MS (ESI): found [M + H]<sup>+</sup>, 468.4.</div></div><div id="sec4_9_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> Homodimer: [<i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> prodrug (<b>38a</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>38a</b> was synthesized from <b>4a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtaining the desired homodimer as a white solid (yield, 67%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.60 (d, <i>J</i> = 9.0 Hz, 1 H), 7.53 (t, <i>J</i> = 7.2 Hz, 1 H), 6.57 (bs, 1 H), 4.74–4.61 (m, 2 H), 4.68 (dd, <i>J</i> = 7.6, 14.7 Hz, 1 H), 4.12–4.08 (m, 1 H), 4.07 (t, <i>J</i> = 7.2 Hz, 1 H), 3.80–3.65 (m, 1 H), 3.60–3.39 (m, 2 H), 3.29–3.19 (m, 1 H), 3.15–3.06 (m, 1 H), 2.80 (dd, <i>J</i> = 5.0, 16.0 Hz, 1 H), 2.72 (t, <i>J</i> = 7.0 Hz, 2 H), 2.41–2.28 (m, 4 H), 2.15–1.89 (m, 5 H), 1.88 (quin, <i>J</i> = 6.8 Hz, 2 H), 0.96 (t, <i>J</i> = 6.7 Hz, 6 H). MS (ESI): found [M + H]<sup>+</sup>, 961.7.</div></div><div id="sec4_9_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SH (<b>18a</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>18a</b> was synthesized by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer (<b>38a</b>), obtainimg the desired pendent thiol as a white solid (yield, 88%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.62 (d, <i>J</i> = 9.8 Hz, 1 H), 7.21 (t, <i>J</i> = 5.5 Hz, 1 H), 6.61 (d, <i>J</i> = 6.7 Hz, 1 H), 4.80–4.62 (m, 3 H), 4.62–4.48 (m, 1 H), 4.16 (t, <i>J</i> = 9.4 Hz, 1 H), 3.69 (q, <i>J</i> = 8.1 Hz, 1 H), 3.61–3.44 (m, 2 H), 3.38 (tdd, <i>J</i> = 6.8, 13.2, 19.8 Hz, 2 H), 3.27 (dd, <i>J</i> = 11.0, 16.0 Hz, 1 H), 2.81 (dd, <i>J</i> = 5.3, 15.8 Hz, 1 H), 2.54 (q, <i>J</i> = 7.2 Hz, 2 H), 2.47 (dd, <i>J</i> = 7.0, 12.5 Hz, 1 H), 2.32 (td, <i>J</i> = 7.8, 11.3 Hz, 2 H), 2.16–2.14 (m, 1 H), 2.13–1.84 (m, 7 H), 1.80 (quin, <i>J</i> = 6.8 Hz, 2 H), 1.45 (t, <i>J</i> = 8.0 Hz, 1 H), 0.95 (d, <i>J</i> = 7.1 Hz, 3 H), 0.93 (d, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 482.4.</div></div><div id="sec4_9_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> Homodimer: [<i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S]<sub>2</sub> prodrug (<b>38b</b>)</h4><div class="NLM_p last">The homodimer prodrug thioether <b>38b</b> was synthesized from <b>4b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–47% acetonitrile gradient, obtaining the desired homodimer as a white solid (yield, 63%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.79 (d, <i>J</i> = 9.0 Hz, 1 H), 6.83 (bs, 1 H), 6.68 (bs, 1 H), 4.74–4.64 (m, 3 H), 4.54–4.48 (m, 1 H), 4.32–4.25 (m, 1 H), 4.24–4.17 (m, 1 H), 3.78–3.38 (m, 5 H), 3.34 (dd, <i>J</i> = 6.5, 13.5 Hz, 1 H), 2.94–2.68 (m, 3 H), 2.49 (d, <i>J</i> = 7.8 Hz, 1 H), 2.42–2.28 (m, 2H), 2.16–1.88 (m, 8 H), 0.94 (d, <i>J</i> = 6.9 Hz, 3 H), 0.89 (d, <i>J</i> = 7.0 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 961.7.</div></div><div id="sec4_9_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SH (<b>18b</b>)</h4><div class="NLM_p last">Thiol warhead having SLA analogue <b>18b</b> was synthesized by following the above general procedure for synthesis of sulfhydryl from corresponding homodimer (<b>38b</b>), obtaining the desired pendent thioether as a white solid (yield, 87%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.79 (d, <i>J</i> = 9.4 Hz, 1 H), 6.89 (d, <i>J</i> = 8.2 Hz, 1 H), 6.67 (t, <i>J</i> = 5.3 Hz, 1 H), 4.75–4.65 (m, 3 H), 4.62–4.55 (m, 1 H), 4.26 (dd, <i>J</i> = 5.9, 9.4 Hz, 1 H), 3.67–3.56 (m, 2 H), 3.56–3.41 (m, 2 H), 3.38–3.27 (m, 1 H), 3.09 (dd, <i>J</i> = 5.5, 14.9 Hz, 1 H), 2.84 (dd, <i>J</i> = 5.7, 15.1 Hz, 1 H), 2.57 (q, <i>J</i> = 7.0 Hz, 2 H), 2.54–2.45 (m, 1 H), 2.43–2.27 (m, 3 H), 2.18–2.09 (m, 3 H), 2.09–1.96 (m, 2 H), 1.86–1.79 (m, 2 H), 1.49 (t, <i>J</i> = 8.0 Hz, 1 H), 0.95 (d, <i>J</i> = 7.0 Hz, 3 H), 0.90 (d, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 482.4.</div></div><div id="sec4_9_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SCH<sub>3</sub> (<b>19a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>19a</b> having the methylthioether warhead on pendent carboxylic acid was synthesized from <b>4a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 3-(methylthio)propan-1-amine and purified by HPLC using 5–55% acetonitrile gradient, obtaining the desired peptide as a white solid (yield, 70%; purity by LC-MS, 97%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.72 (d, <i>J</i> = 9.4 Hz, 1 H), 7.01 (d, <i>J</i> = 8.6 Hz, 1 H), 6.82 (t, <i>J</i> = 4.8 Hz, 1 H), 4.74–4.62 (m, 2 H), 4.23 (dd, <i>J</i> = 6.8, 9.0 Hz, 2 H), 3.69–3.55 (m, 2 H), 3.54–3.46 (m, 2 H), 3.43–3.28 (m, 2 H), 2.98 (dd, <i>J</i> = 6.9, 14.8 Hz, 1 H), 2.89 (dd, <i>J</i> = 5.3, 14.5 Hz, 1 H), 2.52 (t, <i>J</i> = 7.1 Hz, 2 H), 2.39–2.23 (m, 3 H), 2.17–2.06 (m, 2 H), 2.09(s, 3H), 2.06–1.83 (m, 4 H), 1.81 (p, <i>J</i> = 7.2 Hz, 2 H), 0.93 (d, <i>J</i> = 6.8 Hz, 3 H), 0.90 (d, <i>J</i> = 6.8 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 172.5, 171.6, 170.7, 170.4, 169.9, 59.6, 59.5, 58.8, 58.1, 49.9, 47.6, 47.3, 38.8, 36.1, 31.3, 28.9, 28.3, 27.5, 26.1, 25.7, 25.0, 19.5, 17.4. MS (ESI): found [M + Na]<sup>+</sup>, 518.4.</div></div><div id="sec4_9_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SCH<sub>3</sub> (<b>19b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>19b</b> having the methylthioether warhead on pendent carboxylic acid was synthesized from <b>4b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 3-(methylthio)propan-1-amine and purified by HPLC using 5–55% acetonitrile gradient, obtaining the desired peptide as a white solid (yield, 70%; purity by LC-MS, 98%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.63 (d, <i>J</i> = 10.2 Hz, 1 H), 7.16 (t, <i>J</i> = 5.7 Hz, 2 H), 6.54 (d, <i>J</i> = 6.7 Hz, 2 H), 4.79–4.71 (m, 1 H), 4.72 (t, <i>J</i> = 6.7 Hz, 2 H), 4.58–4.48 (m, 2 H), 4.20–4.09 (m, 1 H), 3.90–3.77 (m, 1 H), 3.73–3.61 (m, 1 H), 3.59–3.50 (m, 2 H), 3.49–3.42 (m, 1 H), 3.41–3.31 (m, 2 H), 3.27 (dd, <i>J</i> = 10.6, 16.0 Hz, 1 H), 2.81 (dd, <i>J</i> = 5.1, 16.0 Hz, 1 H), 2.64–2.41 (m, 1 H), 2.51 (t, <i>J</i> = 6.7 Hz, 2 H), 2.31 (tdd, <i>J</i> = 4.3, 7.8, 11.5 Hz, 2 H), 2.16–2.07 (m, 2 H), 2.08 (s, 3 H), 2.05–1.95 (m, 2 H), 1.87–1.80 (m, 2 H), 1.79 (p, <i>J</i> = 2.7 Hz, 2 H), 0.95 (d, <i>J</i> = 2.7 Hz, 3 H), 0.93 (d, <i>J</i> = 2.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.4, 172.4, 171.0, 169.7, 169.3, 59.9, 59.2, 58.2, 52.5, 47.6, 47.1, 38.7, 34.3, 31.3, 29.7, 28.4, 27.6, 26.0, 25.6, 24.8, 22.8, 19.3, 18.5. MS (ESI): found [M + H]<sup>+</sup>, 496.4.</div></div><div id="sec4_9_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)CH<sub>3</sub> (<b>20a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>20a</b> having the methylsulfoxide warhead on pendent carboxylic acid was synthesized from <b>19a</b> by following the above general procedure for sulfoxide synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/Borax and purified by HPLC using 5–50% acetonitrile gradient, obtaining the desired sulfoxide as a white solid (yield, 71%; purity by LC-MS, 98%). Nonseparable diastereomeric mixture = 1.1:1. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.87 (d, <i>J</i> = 9.0 Hz, 1 H), 7.50 (t, <i>J</i> = 7.0 Hz, 1 H), 7.45 (d, <i>J</i> = 6.0 Hz, 1 H), 4.74–4.61 (m, 3 H), 4.22–4.16 (m, 1 H), 3.74–3.64 (m, 1 H), 3.64–3.55 (m, 1 H), 3.55–3.43 (m, 1 H), 3.28 (dd, <i>J</i> = 6.1, 12.7 Hz, 1 H), 3.14 (t, <i>J</i> = 7.8 Hz, 2 H), 2.94 (dd, <i>J</i> = 1.8, 4.0 Hz, 1 H), 2.92 (s, 3 H), 2.90–2.86 (m, 2 H), 2.39 (d, <i>J</i> = 9.8 Hz, 1 H), 2.36–2.20 (m, 3 H), 2.17–2.08 (m, 2 H), 2.08–1.96 (m, 5 H), 0.93 (d, <i>J</i> = 6.8 Hz, 3 H), 0.91 (d, <i>J</i> = 6.8 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 172.4, 171.0, 171.0, 170.7, 169.2, 59.3, 58.9, 57.7, 51.8, 50.1, 47.4, 47.1, 37.6, 37.0, 29.5, 28.8, 27.4, 26.1, 25.6, 25.0, 22.6, 19.5, 17.6. MS (ESI): found [M + H]<sup>+</sup>, 512.4.</div></div><div id="sec4_9_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)CH<sub>3</sub> (<b>20b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>20b</b> having the methylsulfoxide warhead on pendent carboxylic acid was synthesized from <b>19b</b> by following the above general procedure for sulfoxide synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/borax and purified by HPLC using 5–50% acetonitrile gradient, obtaining the desired sulfoxide as a white solid (yield, 67%; purity by LC-MS, >99%). Nonseparable diastereomeric mixture = 1.1:1. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.80 (t, <i>J</i> = 8.0 Hz, 1 H), 6.92 (bs, 1 H), 6.82 (t, <i>J</i> = 9.4 Hz, 1 H), 4.79–4.68 (m, 2 H), 4.68–4.61 (m, 1 H), 4.31–4.26 (m, 2 H), 3.65–3.47 (m, 2 H), 3.42–3.28 (m, 1 H), 3.25–3.11 (m, 2 H), 2.92–2.75 (m, 2 H), 2.63 (s, 3 H), 2.49–2.38 (m, 1 H), 2.40 (td, <i>J</i> = 6.8, 12.9 Hz, 1 H), 2.36–2.22 (m, 2 H), 2.18–2.10 (m, 2 H), 2.09–1.93 (m, 6 H), 1.91–1.82 (m, 1 H), 0.95 (d, <i>J</i> = 7.0 Hz, 3 H), 0.90 (d, <i>J</i> = 7.0 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 172.3, 171.1, 170.8, 170.8, 169.1, 59.3, 58.9, 57.7, 51.4, 50.0, 47.4, 47.0, 38.4, 38.1, 37.1, 28.9, 27.5, 26.1, 25.6, 25.0, 22.7, 19.5, 17.8. MS (ESI): found [M + H]<sup>+</sup>, 512.4.</div></div><div id="sec4_9_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)<sub>2</sub>CH<sub>3</sub> (<b>21a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>21a</b> having the methylsulfone warhead on pendent carboxylic acid was synthesized from <b>19a</b> by following the above general procedure for sulfones synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/borax and purified by HPLC using 5–48% acetonitrile gradient over 30 min, obtaining the desired sulfone as a white solid (yield, 89%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.81 (d, <i>J</i> = 9.2 Hz, 1 H), 6.70 (t, <i>J</i> = 6.0 Hz, 1 H), 6.57 (d, <i>J</i> = 8.8 Hz, 1 H), 4.79 (t, <i>J</i> = 6.7 Hz, 1 H), 4.72 (d, <i>J</i> = 7.4 Hz, 1 H), 4.61 (ddd, <i>J</i> = 3.3, 5.8, 8.9 Hz, 1 H), 4.37–4.33 (m, 1 H), 4.33–4.28 (m, 1 H), 3.68–3.47 (m, 4 H), 3.32–3.22 (m, 1 H), 3.22–3.11 (m, 2 H), 2.90 (s, 3 H), 2.73 (dd, <i>J</i> = 6.0, 15.0 Hz, 1 H), 2.55–2.43 (m, 2 H), 2.38–2.26 (m, 2 H), 2.17–2.13 (m, 1 H), 2.14 (p, <i>J</i> = 6.8 Hz, 2 H), 2.09–1.95 (m, 3 H), 1.92–1.82 (m, 1 H), 0.97 (d, <i>J</i> = 6.8 Hz, 3 H), 0.90 (d, <i>J</i> = 6.8 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.0, 171.1, 170.9, 170.8, 170.0, 59.4, 58.6, 58.1, 51.8, 50.3, 47.7, 47.6, 40.5, 37.8, 35.8, 29.7, 28.5, 27.4, 26.1, 25.8, 25.3, 22.9, 19.8, 16.9. MS (ESI): found [M + H]<sup>+</sup>, 528.4.</div></div><div id="sec4_9_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S(═O)<sub>2</sub>CH<sub>3</sub> (<b>21b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>21b</b> having the methylsulfone warhead on pendent carboxylic acid was synthesized from <b>19b</b> by following the above general procedure for sulfones synthesis from organic sulfides by H<sub>2</sub>O<sub>2</sub>/borax at 60 °C for 24 h and purified by HPLC using 5–48% acetonitrile gradient over 30 min, obtaining the desired sulfone as a white solid (yield, 83%; purity by LC-MS, 99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.82 (d, <i>J</i> = 8.9 Hz, 1 H), 6.83 (t, <i>J</i> = 5.7 Hz, 1 H), 6.75 (d, <i>J</i> = 8.6 Hz, 1 H), 4.76 (t, <i>J</i> = 6.7 Hz, 1 H), 4.71 (dd, <i>J</i> = 1.8, 7.9 Hz, 1 H), 4.67–4.61 (m, 1 H), 4.33–4.26 (m, 2 H), 3.65–3.50 (m, 2 H), 3.36–3.26 (m, 1 H), 3.19–3.11 (m, 2 H), 2.94 (d, <i>J</i> = 3.9 Hz, 1 H), 2.91 (s, 3 H), 2.80 (dd, <i>J</i> = 5.9, 14.9 Hz, 1 H), 2.52–2.40 (m, 2 H), 2.37–2.30 (m, 2 H), 2.14 (t, <i>J</i> = 7.0 Hz, 2 H), 2.10–1.99 (m, 3 H), 1.89–1.75 (m, 2 H), 1.64 (p, <i>J</i> = 6.8 Hz, 2 H), 0.96 (d, <i>J</i> = 7.0 Hz, 3 H), 0.90 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 171.5, 171.3, 170.9, 170.9, 165.6, 59.4, 58.7, 58.1, 51.8, 50.2, 47.6, 40.5, 37.8, 36.0, 31.9, 29.7, 28.6, 27.4, 26.1, 25.8, 25.3, 22.7, 19.7, 17.0. MS (ESI): found [M + H]<sup>+</sup>, 528.4.</div></div><div id="sec4_9_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>OH (<b>22a</b>) and <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>OH (<b>22b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>22a</b> and <b>22b</b> having the <i>n</i>-propylalcohol warhead on pendent carboxylic acid was synthesized by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with 3-aminopropan-1-ol and purified by HPLC using 11–40% acetonitrile gradient over 30 min, obtaining the desired alcohol as viscous oil (yield, 75%; purity by LC-MS, >96%). Nonseparable diastereomeric mixture (1:0.65). Diastereomer <b>22a</b>: <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.81 (d, <i>J</i> = 9.0 Hz, 1 H), 6.72 (d, <i>J</i> = 8.5 Hz, 1 H), 6.65 (t, <i>J</i> = 5.5 Hz, 1 H), 4.75 (t, <i>J</i> = 7.0 Hz, 2 H), 4.70–4.64 (m, 1 H), 4.42 (t, <i>J</i> = 6.5 Hz, 1 H), 4.39–4.28 (m, 1 H), 3.72–3.67 (m, 1 H), 3.65–3.39 (m, 5 H), 3.37–3.26 (m, 1 H), 3.19 (dd, <i>J</i> = 4.3, 14.9 Hz, 1 H), 2.75 (t, <i>J</i> = 6.5 Hz, 1 H), 2.57–2.40 (m, 2 H), 2.38–2.20 (m, 2 H), 2.18–2.10 (m, 2 H), 2.10–1.80 (m, 5 H), 0.96 (d, <i>J</i> = 6.9 Hz, 3 H), 0.94 (d, <i>J</i> = 7.0 Hz, 3 H). Diastereomer <b>22b</b>: <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.99 (d, <i>J</i> = 9.8 Hz, 1 H), 7.02 (t, <i>J</i> = 6.5 Hz, 1 H), 6.75 (d, <i>J</i> = 8.4 Hz, 1 H), 4.73 (t, <i>J</i> = 7.0 Hz, 2 H), 4.63–4.59 (m, 1 H), 4.39–4.28 (m, 2 H), 3.65–3.39 (m, 5 H), 3.37–3.26 (m, 2 H), 2.71 (t, <i>J</i> = 7.0 Hz, 1 H), 2.57–2.40 (m, 2 H), 2.38–2.20 (m, 2 H), 2.18–2.10 (m, 2 H), 2.10–1.80 (m, 3 H), 1.71 (quin, <i>J</i> = 5.7 Hz, 2 H), 0.90 (d, <i>J</i> = 6.9 Hz, 3 H), 0.88 (d, <i>J</i> = 7.0 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 466.4.</div></div><div id="sec4_9_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOMe (<b>23a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>23a</b> having the methylcarboxylate warhead on pendent carboxylic acid was synthesized from <b>4a</b> by following the above general procedure for EDCl peptide coupling of coupling of pendent carboxylic acid with methyl 4-aminobutanoate and purified by HPLC using 11–45% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 86%; purity by LC-MS, 98%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.79 (d, <i>J</i> = 9.0 Hz, 1 H), 6.91 (bs, 1 H), 6.78 (bs, 1 H), 4.72 (t, <i>J</i> = 6.5 Hz, 2 H), 4.69–4.63 (m, 1 H), 4.36–4.22 (m, 2 H), 3.68 (s, 3 H), 3.65–3.55 (m, 2 H), 3.50 (q, <i>J</i> = 8.6 Hz, 1 H), 3.34–3.25 (m, 2 H), 2.83 (dd, <i>J</i> = 5.1, 14.9 Hz, 1 H), 2.50 (dd, <i>J</i> = 7.0, 11.0 Hz, 1 H), 2.44–2.35 (m, 4 H), 2.35–2.24 (m, 2 H), 2.14 (q, <i>J</i> = 6.8 Hz, 2 H), 2.09–1.91 (m, 2 H), 1.84 (p, <i>J</i> = 6.3 Hz, 2 H), 0.94 (d, <i>J</i> = 6.7 Hz, 3 H), 0.90 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.7, 172.7, 171.6, 170.5, 170.4, 170.0, 59.4, 58.5, 58.1, 51.7, 49.8, 47.5, 47.3, 39.1, 35.9, 31.2, 28.8, 27.5, 26.0, 25.7, 25.0, 24.4, 19.6, 17.2. MS (ESI): found: [M + H]<sup>+</sup>, 508.5.</div></div><div id="sec4_9_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOMe (<b>23b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>23b</b> having the methylcarboxylate warhead on pendent carboxylic acid was synthesized from <b>4b</b> by following the above general procedure for EDCl peptide coupling of coupling of pendent carboxylic acid with methyl 4-aminobutanoate and purified by HPLC using 11–45% acetonitrile gradient over 30 min, obtaining the desired peptide as a white solid (yield, 85%; purity by LC-MS, >98%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.62 (d, <i>J</i> = 10.2 Hz, 1 H), 7.13 (t, <i>J</i> = 5.6 Hz, 1 H), 6.74 (d, <i>J</i> = 6.7 Hz, 1 H), 4.71 (t, <i>J</i> = 8.5 Hz, 1 H), 4.72 (dd, <i>J</i> = 3.1, 7.7 Hz, 1 H), 4.55–4.49 (m, 1 H), 4.23–4.12 (m, 2 H), 3.77–3.67 (m, 1 H), 3.67 (s, 3 H), 3.59–3.45 (m, 2 H), 3.31–3.25 (m, 4 H), 2.77 (dd, <i>J</i> = 5.1, 16.0 Hz, 1 H), 2.53–2.39 (m, 2 H), 2.35 (t, <i>J</i> = 7.2 Hz, 2 H), 2.33–2.22 (m, 2 H), 2.20–2.10 (m, 1 H), 2.09–1.90 (m, 6 H), 1.81 (quin, <i>J</i> = 7.2 Hz, 2 H), 0.94 (d, <i>J</i> = 4.7 Hz, 3 H), 0.93 (d, <i>J</i> = 4.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 173.6, 173.3, 172.3, 170.9, 169.7, 169.2, 77.3, 77.0, 76.7, 59.9, 59.1, 58.0, 52.4, 51.7, 47.5, 47.0, 38.8, 34.2, 31.2, 29.7, 27.7, 26.0, 25.6, 24.8, 24.5, 19.3, 18.5. MS (ESI): found [M + H]<sup>+</sup>, 508.5.</div></div><div id="sec4_9_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOH (<b>24a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>24a</b> having the carboxylic acid warhead on pendent carboxylic acid was synthesized from <b>23a</b> by following the above general procedure for general procedure for LiOH base mediated hydrolysis of methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtainimg the desired peptide as a white solid (yield, 93%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 8.28 (d, <i>J</i> = 10.2 Hz, 1 H), 7.01 (t, <i>J</i> = 5.2 Hz, 1 H), 6.84 (d, <i>J</i> = 9.0 Hz, 1 H), 4.82 (d, <i>J</i> = 7.8 Hz, 1 H), 4.73 (t, <i>J</i> = 6.8 Hz, 1 H), 4.68 (dd, <i>J</i> = 4.5, 10.0 Hz, 2 H), 4.35 (dt, <i>J</i> = 2.0, 9.4 Hz, 1 H), 3.65–3.57 (m, 1 H), 3.57–3.39 (m, 3 H), 3.32 (dd, <i>J</i> = 2.0, 16.0 Hz, 1 H), 3.29–3.20 (m, 1 H), 2.75 (dd, <i>J</i> = 6.3, 16.0 Hz, 1 H), 2.60–2.44 (m, 3 H), 2.40 (dd, <i>J</i> = 2.7, 10.2 Hz, 1 H), 2.33–2.21 (m, 2 H), 2.18–2.10 (m, 2 H), 2.10–1.91 (m, 3 H), 1.90–1.76 (m, 2 H), 0.95 (d, <i>J</i> = 6.7 Hz, 3 H), 0.88 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 174.1, 174.0, 171.1, 170.0, 168.3, 167.7, 58.1, 58.0, 56.5, 48.6, 46.4, 45.9, 37.8, 36.5, 30.4, 28.6, 26.6, 24.8, 24.7, 23.9, 23.5, 18.5, 17.1. MS (ESI): found [M + H]<sup>+</sup>, 494.4.</div></div><div id="sec4_9_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>COOH (<b>24b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>24b</b> having the carboxylic acid warhead on pendent carboxylic acid was synthesized from <b>23b</b> by following the above general procedure for general procedure for LiOH base mediated hydrolysis of methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtainimg the desired peptide as a white solid (yield, 95%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 7.49 (d, <i>J</i> = 10.2 Hz, 1 H), 6.95 (t, <i>J</i> = 5.7 Hz, 1 H), 6.70 (d, <i>J</i> = 7.4 Hz, 1 H), 5.57 (bs, 1 H), 4.76 (d, <i>J</i> = 7.0 Hz, 1 H), 4.73–4.60 (m, 2 H), 4.32 (dd, <i>J</i> = 8.4, 10.0 Hz, 1 H), 4.19 (dt, <i>J</i> = 3.0, 9.0 Hz, 1 H), 3.69–3.40 (m, 3 H), 3.32 (dd, <i>J</i> = 8.4, 15.1 Hz, 1 H), 3.27–3.17 (m, 1 H), 2.75 (dd, <i>J</i> = 6.5, 15.1 Hz, 1 H), 2.56–2.30 (m, 4 H), 2.25 (td, <i>J</i> = 3.9, 7.5 Hz, 1 H), 2.20–2.10 (m, 2 H), 2.08–1.90 (m, 4 H), 1.86 (quin, <i>J</i> = 7.0 Hz, 2 H), 0.95 (d, <i>J</i> = 7.0 Hz, 6 H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ ppm: 176.3, 173.7, 172.1, 170.9, 169.4, 169.3, 59.8, 59.5, 58.2, 51.2, 47.6, 47.0, 39.3, 34.0, 31.8, 29.7, 27.7, 26.1, 25.8, 24.7, 24.7, 19.4, 18.5. MS (ESI): found [M + H]<sup>+</sup>, 494.4.</div></div><div id="sec4_9_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CONHOH (<b>25a</b>)</h4><div class="NLM_p last">The cyclic peptide <b>25a</b> having the hydroxamic acid warhead on pendent carboxylic acid was synthesized from <b>23a</b> by following the above general procedure for general procedure for hydroxamic acid synthesis from the methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtainimg the desired hydroxamic acid peptide as a white solid (yield, 86%; purity by LC-MS, 99%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm: 10.08 (bs, 1 H), 8.27 (d, <i>J</i> = 8.2 Hz, 1 H), 7.51 (t, <i>J</i> = 5.5 Hz, 1 H), 6.94 (d, <i>J</i> = 5.5 Hz, 1 H), 4.83 (dd, <i>J</i> = 2.1, 8.1 Hz, 1 H), 4.63 (dd, <i>J</i> = 2.0, 7.7 Hz, 1H), 4.54 (t, <i>J</i> = 7.2 Hz, 1 H), 4.16 (t, <i>J</i> = 6.1 Hz, 1 H), 3.88 (t, <i>J</i> = 7.8 Hz, 1 H), 3.77–3.69 (m, 1 H), 3.53–3.43 (m, 3 H), 3.40–3.38 (m, 1 H), 3.26 (td, <i>J</i> = 7.6, 11.3 Hz, 1 H), 3.03–2.93 (m, 2 H), 2.89 (dd, <i>J</i> = 5.1, 14.5 Hz, 1 H), 2.41–2.05 (m, 4 H), 1.99–1.60 (m, 5 H), 1.54 (quin, <i>J</i> = 6.8 Hz, 2 H), 0.83 (d, <i>J</i> = 7.0 Hz, 3 H), 0.78 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm: 173.0, 172.0, 171.0, 170.4, 169.5, 168.0, 60.0, 59.7, 57.2, 52.2, 47.0, 46.8, 40.8, 38.7, 30.1, 29.7, 28.0, 25.6, 25.6, 24.8, 22.5, 19.9, 17.9. MS (ESI): found [M + H]<sup>+</sup>, 509.5.</div></div><div id="sec4_9_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CONHOH (<b>25b</b>)</h4><div class="NLM_p last">The cyclic peptide <b>25b</b> having the hydroxamic acid warhead on pendent carboxylic acid was synthesized from <b>23b</b> by following the above general procedure for general procedure for hydroxamic acid synthesis from the methyl ester and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtainimg the desired hydroxamic acid peptide as a white solid (yield, 87%; purity by LC-MS, >99%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 10.33 (s, 1 H), 7.79 (d, <i>J</i> = 9.8 Hz, 1 H), 7.77 (d, <i>J</i> = 7.4 Hz, 1 H), 7.72 (t, <i>J</i> = 5.3 Hz, 1 H), 4.73–4.60 (m, 2 H), 4.46–4.32 (m, 1 H), 3.92–3.87 (m, 1 H), 3.84 (t, <i>J</i> = 9.2 Hz, 1 H), 3.63 (t, <i>J</i> = 6.5 Hz, 1 H), 3.58–3.38 (m, 3 H), 3.16–2.89 (m, 3 H), 2.80 (dd, <i>J</i> = 4.0, 16.0 Hz, 1 H), 2.60 (dd, <i>J</i> = 4.3, 15.7 Hz, 1 H), 2.37–2.20 (m, 1 H), 2.16 (dd, <i>J</i> = 3.0, 16.0 Hz, 1 H), 2.08–1.55 (m, 9 H), 0.85 (d, <i>J</i> = 6.7 Hz, 3 H), 0.83 (d, <i>J</i> = 6.7 Hz, 3 H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm: 171.9, 171.1, 170.6, 170.3, 169.2, 169.2, 59.5, 59.1, 58.7, 49.3, 47.2, 46.6, 38.8, 38.3, 30.2, 29.7, 27.7, 26.3, 25.5, 24.8, 22.5, 19.7, 18.8. MS (ESI): found [M + H]<sup>+</sup>, 509.5.</div></div><div id="sec4_9_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> <i>Cyclo</i>-Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S-SCH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> (<b>26a</b>)</h4><div class="NLM_p last">The amine-containing peptide <b>26a</b> was synthesized from <b>4a</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtainimg the desired homodimer as a viscous oil (yield, 69%; purity by LC-MS, 96%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 8.39 (bs, 2 H), 7.87 (d, <i>J</i> = 9.0 Hz, 1 H), 7.72 (t, <i>J</i> = 5.7 Hz, 1 H), 7.45 (d, <i>J</i> = 8.6 Hz, 1 H), 4.72–4.67 (m, 3 H), 4.31–4.21 (m, 1 H), 4.17 (t, <i>J</i> = 8.2 Hz, 1 H), 3.69–3.61 (m, 1 H), 3.55–3.49 (m, 3 H), 3.35–3.15 (m, 5 H), 3.01–2.90 (m, 2 H), 2.89–2.76 (m, 1 H), 2.46–2.36 (m, 1 H), 2.36–2.16 (m, 3 H), 2.16–1.91 (m, 4 H), 1.90–1.77 (m, 1 H), 0.91 (d, <i>J</i> = 4.3 Hz, 3 H), 0.89 (d, <i>J</i> = 4.3 Hz, 3 H). MS (ESI): found [M + H]<sup>+</sup>, 543.5.</div></div><div id="sec4_9_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> <i>Cyclo</i>-<sup>D</sup>Asp(Pro-<sup>D</sup>Pro-Val)-NHCH<sub>2</sub>CH<sub>2</sub>S-SCH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> (<b>26b</b>)</h4><div class="NLM_p last">The amine-containing peptide <b>26b</b> was synthesized from <b>4b</b> by following the above general procedure for EDCl peptide coupling of pendent carboxylic acid with cysteamine diamine and purified by HPLC using 11–35% acetonitrile gradient over 30 min, obtainimg the desired homodimer as a viscous oil (yield, 65%; purity by LC-MS, 97%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ ppm: 8.27 (t, <i>J</i> = 4.7 Hz, 1 H), 7.77 (br. s., 2 H), 7.37 (d, <i>J</i> = 8.2 Hz, 1 H), 6.77 (d, <i>J</i> = 7.0 Hz, 1 H), 4.97–4.84 (m, 1 H), 4.70–4.53 (m, 2 H), 3.95 (t, <i>J</i> = 9.2 Hz, 1 H), 3.90–3.80 (m, 1 H), 3.79–3.69 (m, 1 H), 3.69–3.58 (m, 1 H), 3.58–3.45 (m, 2 H), 3.36–3.18 (m, 4 H), 3.14–3.04 (m, 1 H), 3.02–2.89 (m, 3 H), 2.73 (dd, <i>J</i> = 4.5, 16.2 Hz, 1 H), 2.37–2.17 (m, 4 H), 2.11–1.87 (m, 5 H), 0.98 (d, <i>J</i> = 6.3 Hz, 6 H). MS (ESI): found [M + H]<sup>+</sup>, 543.5.</div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Microfluidic Chip-Based KDAC Inhibition Assay</h3><div class="NLM_p">Substrate A, a (FAM)-labeled peptide purchasable from PerkinElmer (Broad Substrate A, product no. CLS960006), was synthesized in house (see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Section S7</a>) and used for KDAC1 assays, while the others were purchased from PerkinElmer: two (FITC)-labeled peptides (p53 acetylated peptide and Histone 4 acetylated peptide, product nos. 760512 and 760513, respectively) were used as substrates to test compounds against KDAC3 and 6, respectively, and a (FAM)-labeled peptide (Broad Substrate B, product no. CLS960007,) was employed as substrate for KDAC8 assays. All KDACs were purchased from BPS Bioscience.</div><div class="NLM_p last">Compounds were tested in duplicate in a 10-point dose curve with 3-fold serial dilution starting from 30 μM; in the case of SAHA, a 15-point dose curve with 3-fold serial dilution starting from 30 μM was performed when testing on KDACs 3 and 6. Purified KDACs were incubated with 1 μM of p53 acetylated peptide, Histone 4 acetylated peptide, Broad Substrates A or B, and SLAs for 60 min at room temperature, in KDAC assay buffer that contained 25 mM Tris-HCl (pH 8.0), 137 mM NaCl, 2.7 mM KCl, 1 mM MgCl<sub>2</sub>, and 0.01% BSA. Reactions (in duplicates) were terminated by the addition of a stop buffer containing 100 mM HEPES, 0.015% Brij-35, 10 mM EDTA, 0.1% CR-3, and 1.5 μM of the known pan-KDAC inhibitor Panobinostat<a onclick="showRef(event, 'ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref67 ref68">(67, 68)</a> (LBH589, purchased from ApexBio Technology). Fluorescence intensity of electrophoretically separated substrate and product was detected using the Labchip EZ Reader, and the data were analyzed by nonlinear regression using GraphPad Prism 6.01 software<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> to afford IC<sub>50</sub> values from dose–response experiments. The percentage of inhibition at 30 μM is reported when its value is less than 50% at that concentration. As standard compounds (positive controls), eight well-known KDAC inhibitors: Entinostat (MS-275),<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> TSA,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> tubastatin A,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> SAHA,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> purchased from Selleck Chemicals, PCI-34051<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> (purchased from Cayman Chemical Company), together with T247<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (synthesized in house, see Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Section S8</a>) as well as largazole<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> (thioester) and one of its analogue (herein designated SD-L-256,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> generously supplied by Prof. Robert Williams of the Department of Chemistry at Colorado State University), were used (see <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, <a class="ref internalNav" href="#fig6" aria-label="Figures 5">Figures 5</a><b>–</b><a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">6</a> and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">Figure S3</a> for dose–response curves).</div></div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Molecular Modeling</h3><div class="NLM_p">All calculations were performed on two MacPros (dual 2.67 GHz six core Intel Xeon X5650) cluster (24 CPU total) running GNU/Linux Mint 17.1 64-bit operating system.</div><div class="NLM_p last">KDAC8 cocrystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a>)<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> was retrieved from the Protein Data Bank (PDB).<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Solvent molecules, buffer, and non-interaction ions were removed, and hydrogens added considering a neutral pH (7.4). A minimization process was then performed by means of GROMACS 5.0.2:<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> (1) ligand topology was computed using the ACPYPE/Antechamber tool,<a onclick="showRef(event, 'ref73 ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref73 ref74 ref75">(73-75)</a> (2) periodic boundary conditions were applied in the <i>x</i>, <i>y</i>, and <i>z</i> directions using a cubic box, and (3) 5000 steepest descend minimization steps were performed using the AMBER99SB-ILDN force field<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> with TIP4P as the water model. Na<sup>+</sup> and Cl<sup>–</sup> were employed as counterions. Marvin software was used for drawing and characterizing the chemical structures of compounds <b>14a</b> and <b>25a</b>, Marvin 14.11.3.0, 2014, ChemAxon (<a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>). The protonation states were assigned considering a physiological pH. To generate random conformations of compounds <b>14a</b> and <b>25a</b>, to be used as input structures for docking calculations, the OpenBabel suite (The Open Babel package, version 2.3.2 <a href="http://openbabel.org" class="extLink">http://openbabel.org</a>)<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> was employed to (1) generate a best conformer (after 250 geometry optimization steps) out of 250 conformers using the obconformer tool, and (2) further optimize the obtained geometry by applying the obminimize tool using first the steepest descent algorithm followed by the conjugate gradients algorithm with a default number of steps and force-field (2500 and MMFF94, respectively). Molecular docking of compounds <b>14a</b> and <b>25a</b> has been performed using PLANTS<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> (v1.2, PLP scoring function) to generate 10 conformation for each docked ligand. Search speed, number of ants, evaporation factor, and iteration scaling factor were set as SPEED1, 20, 0.15, and 2, respectively. Because of the stochastic nature of PLANTS, each experiment has been repeated 10 times to release a total of 100 conformations to be analyzed.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01632">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74653" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74653" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01632" class="ext-link">10.1021/acs.jmedchem.5b01632</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Detailed experimental procedures, purity and spectral data of compounds, additional results, and dose–response curves (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf">jm5b01632_si_001.pdf (12.18 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01632" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Garland R. Marshall</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biophysics, Washington University School of Medicine, 700 South Euclid Avenue, St. Louis, Missouri 63110, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3c5b5d4e505d5258517c5b515d5550125f5351"><span class="__cf_email__" data-cfemail="fa9d9b88969b949e97ba9d979b9396d4999597">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Damodara N. Reddy</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biophysics, Washington University School of Medicine, 700 South Euclid Avenue, St. Louis, Missouri 63110, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Flavio Ballante</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biophysics, Washington University School of Medicine, 700 South Euclid Avenue, St. Louis, Missouri 63110, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy Chuang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biophysics, Washington University School of Medicine, 700 South Euclid Avenue, St. Louis, Missouri 63110, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adele Pirolli</span> - <span class="hlFld-Affiliation affiliation">Rome
Center for Molecular Design, Dipartimento di Chimica e Tecnologie
del Farmaco, Sapienza Università
di Roma, P. le A. Moro
5, 00185 Roma, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biagina Marrocco</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Chimica e Tecnologie del Farmaco, Sapienza
Università di Roma, P. le A. Moro 5, 00185 Roma, Italy</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2796e6958-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i93">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00873" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00873" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Support of this research by NIH 5R01GM106974 is gratefully acknowledged by the authors. G.R.M. thanks the Dipartimento di Chimica e Tecnologie del Farmaco, “Sapienza” Universitá di Roma, P.le A. Moro 5, 00185 Rome (Italy) for a Visiting Professorship during the fall semester of 2011, during which he was indoctrinated in epigenetics and KDACIs by Profs. Antonello Mai, Rino Ragno, and their associates. Support for the lab exchanges of Flavio Ballante, Biagina Marrocco, and Adele Pirolli from Sapienza Universitá di Roma as part of their graduate educations is also acknowledged.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i94" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i94"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i95" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i95"> Abbreviations Used</h2><tr><td class="NLM_term">BD</td><td class="NLM_def"><p class="first last">Best Docked poses</p></td></tr><tr><td class="NLM_term">COMBINEr</td><td class="NLM_def"><p class="first last">enhanced comparative binding energy analysis</p></td></tr><tr><td class="NLM_term">CTP</td><td class="NLM_def"><p class="first last">cyclic tetrapeptide</p></td></tr><tr><td class="NLM_term">DIPA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DPPA</td><td class="NLM_def"><p class="first last">diphenylphosphoryl azide</p></td></tr><tr><td class="NLM_term">ECRD</td><td class="NLM_def"><p class="first last">experimental conformation redocking</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluoresceine isothiocyanate</p></td></tr><tr><td class="NLM_term">HA_RMSD<i>h</i></td><td class="NLM_def"><p class="first last">Hungarian symmetry-corrected heavy-atom RMSD</p></td></tr><tr><td class="NLM_term">HAART</td><td class="NLM_def"><p class="first last">highly active antiretroviral therapy</p></td></tr><tr><td class="NLM_term">KAT</td><td class="NLM_def"><p class="first last">lysine acetyltransferase</p></td></tr><tr><td class="NLM_term">KDAC</td><td class="NLM_def"><p class="first last">lysine deacetylase</p></td></tr><tr><td class="NLM_term">KDACI</td><td class="NLM_def"><p class="first last">lysine deacetylase inhibitor</p></td></tr><tr><td class="NLM_term">LTR</td><td class="NLM_def"><p class="first last">long terminal repeat</p></td></tr><tr><td class="NLM_term">O<sup><i>t</i></sup>Bu</td><td class="NLM_def"><p class="first last"><i>ortho</i>-<i>tert</i>-butyl</p></td></tr><tr><td class="NLM_term">LiOH</td><td class="NLM_def"><p class="first last">lithium hydroxide</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-(dimethylamino)propyl)carbodiimide</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">DMSO-<i>d</i><sub>6</sub></td><td class="NLM_def"><p class="first last">deuterated dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HCl</td><td class="NLM_def"><p class="first last">hydrochloric acid</p></td></tr><tr><td class="NLM_term">RCRD</td><td class="NLM_def"><p class="first last">random conformation redocking</p></td></tr><tr><td class="NLM_term">SAHA</td><td class="NLM_def"><p class="first last">suberoylanilide hydroxamic acid</p></td></tr><tr><td class="NLM_term">SLA</td><td class="NLM_def"><p class="first last">simplified largazole analogue</p></td></tr><tr><td class="NLM_term">TSA</td><td class="NLM_def"><p class="first last">trichostatin A</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33866" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33866" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 77 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="note"><p class="first last">Lysine deacetylases are generally referred to as histone deacetylases (HDACs), a historical imperative as epigenetic modification of histones was described in 1964 by Allfrey et al.,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> thus, the terms histone acetylases (HATs), deacetylases(HDACs), methylases (HMTs), etc. It has been shown by proteomics, however, that over 1700 proteins in cells besides histones undergo dynamic acetylation<sup>3</sup>, thus, the current preference for the term lysine acetylase (KAT), deacetylase (KDAC), or methylase (KMT).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Allfrey, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulkner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirsky, A. E.</span><span> </span><span class="NLM_article-title">Acetylation and methylation of histones and their possible role in the regulation of RNA synthesis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1964</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">794</span><span class="refDoi"> DOI: 10.1073/pnas.51.5.786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.51.5.786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=14172992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1964&pages=786-794&author=V.+G.+Allfreyauthor=R.+Faulknerauthor=A.+E.+Mirsky&title=Acetylation+and+methylation+of+histones+and+their+possible+role+in+the+regulation+of+RNA+synthesis&doi=10.1073%2Fpnas.51.5.786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation and methylation of histones and their possible role in the regulation of ribonucleic acid (RNA) synthesis</span></div><div class="casAuthors">Allfrey, V. G.; Faulkner, R.; Mirsky, A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">786-94</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Isolated calf thymus nuclei was incubated with 14C-labeled Na acetate.  Evidence that acetate was incorporated as acetyl groups attached to histones included: (1) following chromatography the label was eluted together with basic proteins; (2) acetyl-labeled histones were sepd. by electrophoresis; (3) acetate-14C was not removed from histones by org. solvents; (4) over 75% of the label remained after histones were treated with trichloroacetic acid.  Most extensive acetate uptake occurred with arginine-rich histones.  Puromycin did not inhibit the acetylation, indicating histones were modified by acetylation after completion of the polypeptide chain.  Acetylation lowered the histone effectiveness as inhibitor of the RNA polymerase reaction, and a dynamic and reversible mechanism was suggested for activation as well as repression of RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL5QIm_JdO1bVg90H21EOLACvtfcHk0licisiWn2U4GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D&md5=7a80f7a8a12636d4bc47930ee5eff217</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.51.5.786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.51.5.786%26sid%3Dliteratum%253Aachs%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26aulast%3DFaulkner%26aufirst%3DR.%26aulast%3DMirsky%26aufirst%3DA.%2BE.%26atitle%3DAcetylation%2520and%2520methylation%2520of%2520histones%2520and%2520their%2520possible%2520role%2520in%2520the%2520regulation%2520of%2520RNA%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1964%26volume%3D51%26spage%3D786%26epage%3D794%26doi%3D10.1073%2Fpnas.51.5.786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Choudhary, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walther, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span> </span><span class="NLM_article-title">Lysine acetylation targets protein complexes and co-regulates major cellular functions</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">834</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span><span class="refDoi"> DOI: 10.1126/science.1175371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1126%2Fscience.1175371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19608861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&author=C.+Choudharyauthor=C.+Kumarauthor=F.+Gnadauthor=M.+L.+Nielsenauthor=M.+Rehmanauthor=T.+C.+Waltherauthor=J.+V.+Olsenauthor=M.+Mann&title=Lysine+acetylation+targets+protein+complexes+and+co-regulates+major+cellular+functions&doi=10.1126%2Fscience.1175371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span></div><div class="casAuthors">Choudhary, Chunaram; Kumar, Chanchal; Gnad, Florian; Nielsen, Michael L.; Rehman, Michael; Walther, Tobias C.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5942</span>),
    <span class="NLM_cas:pages">834-840</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation is a reversible posttranslational modification of proteins and plays a key role in regulating gene expression.  Technol. limitations have so far prevented a global anal. of lysine acetylation's cellular roles.  We used high-resoln. mass spectrometry to identify 3600 lysine acetylation sites on 1750 proteins and quantified acetylation changes in response to the deacetylase inhibitors suberoylanilide hydroxamic acid and MS-275.  Lysine acetylation preferentially targets large macromol. complexes involved in diverse cellular processes, such as chromatin remodeling, cell cycle, splicing, nuclear transport, and actin nucleation.  Acetylation impaired phosphorylation-dependent interactions of 14-3-3 and regulated the yeast cyclin-dependent kinase Cdc28.  Our data demonstrate that the regulatory scope of lysine acetylation is broad and comparable with that of other major posttranslational modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eICW58ngkrVg90H21EOLACvtfcHk0liFNRsaMyOuFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D&md5=8f643db5ba91c50f0b61763ff3095d2c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1175371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1175371%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DKumar%26aufirst%3DC.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26aulast%3DRehman%26aufirst%3DM.%26aulast%3DWalther%26aufirst%3DT.%2BC.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DLysine%2520acetylation%2520targets%2520protein%2520complexes%2520and%2520co-regulates%2520major%2520cellular%2520functions%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D834%26epage%3D840%26doi%3D10.1126%2Fscience.1175371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jursic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Civoli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breslow, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rifkind, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, P. A.</span><span> </span><span class="NLM_article-title">Second generation hybrid polar compounds are potent inducers of transformed cell differentiation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">5705</span><span class="NLM_x">–</span> <span class="NLM_lpage">5708</span><span class="refDoi"> DOI: 10.1073/pnas.93.12.5705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.93.12.5705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=8650156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK28Xjs1Kqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=5705-5708&author=V.+M.+Richonauthor=Y.+Webbauthor=R.+Mergerauthor=T.+Sheppardauthor=B.+Jursicauthor=L.+Ngoauthor=F.+Civoliauthor=R.+Breslowauthor=R.+A.+Rifkindauthor=P.+A.+Marks&title=Second+generation+hybrid+polar+compounds+are+potent+inducers+of+transformed+cell+differentiation&doi=10.1073%2Fpnas.93.12.5705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Second generation hybrid polar compounds are potent inducers of transformed cell differentiation</span></div><div class="casAuthors">Richon, V. M.; Webb, Y.; Merger, R.; Sheppard, T.; Jursic, B.; Ngo, L.; Civoli, F.; Breslow, R.; Rifkind, R. A.; Marks, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5705-5708</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hybrid polar compds., of which hexamethylenebisacetamide (HMBA) is the prototype, are potent inducers of differentiation of murine erythroleukemia (MEL) cells and a wide variety of other transformed cells.  HMBA has been shown to induce differentiation of neoplastic cells of dose-limiting toxicity.  The authors report on a group of three potent second generation hybrid polar compds., di-Et bis-(pentamethylene-N,N-dimethylcarboxamide)malonate (EMBA), suberoylanilide hydroxamic acid (SAHA), and m-carboxycinnamic acid bis-hydroxamide (CBHA) with optimal concns. for inducing MEL cells of 0.4 mM, 2 μM, and 4 μM, resp., compared to 5 mM for HMBA.  All three agents induce accumulation of underphosphorylated pRB; increased levels of p21 protein, a prolongation of the initial G1 phase of the cell cycle; and accumulation of Hb.  However, based upon their effective concns., the cross-resistance or sensitivity of an HMBA-resistant MEL cell variant, and differences in c-myb expression during induction, these differentiation-inducing hybrid polar compds. can be grouped into two subsets, HMBA/EMBA and SAHA/CBHA.  This classification may prove of value in selecting and planning prospective preclin. and clin. studies toward the treatment of cancer by differentiation therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrHTaRYYg617Vg90H21EOLACvtfcHk0liFNRsaMyOuFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xjs1Kqtbc%253D&md5=ffe9e6c95b25b85a50a89201f0f5e5e1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.12.5705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.12.5705%26sid%3Dliteratum%253Aachs%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DWebb%26aufirst%3DY.%26aulast%3DMerger%26aufirst%3DR.%26aulast%3DSheppard%26aufirst%3DT.%26aulast%3DJursic%26aufirst%3DB.%26aulast%3DNgo%26aufirst%3DL.%26aulast%3DCivoli%26aufirst%3DF.%26aulast%3DBreslow%26aufirst%3DR.%26aulast%3DRifkind%26aufirst%3DR.%2BA.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26atitle%3DSecond%2520generation%2520hybrid%2520polar%2520compounds%2520are%2520potent%2520inducers%2520of%2520transformed%2520cell%2520differentiation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D5705%26epage%3D5708%26doi%3D10.1073%2Fpnas.93.12.5705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Furumai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuyama, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobashi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakajima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horinouchi, S.</span><span> </span><span class="NLM_article-title">FK228 (depsipeptide) as a natural prodrug that inhibits class I histone deacetylases</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">4916</span><span class="NLM_x">–</span> <span class="NLM_lpage">4921</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=12208741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmslylu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=4916-4921&author=R.+Furumaiauthor=A.+Matsuyamaauthor=N.+Kobashiauthor=K.+H.+Leeauthor=M.+Nishiyamaauthor=H.+Nakajimaauthor=A.+Tanakaauthor=Y.+Komatsuauthor=N.+Nishinoauthor=M.+Yoshidaauthor=S.+Horinouchi&title=FK228+%28depsipeptide%29+as+a+natural+prodrug+that+inhibits+class+I+histone+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">FK228 (depsipeptide) as a natural prodrug that inhibits class I histone deacetylases</span></div><div class="casAuthors">Furumai, Ryohei; Matsuyama, Akihisa; Kobashi, Nobuyuki; Lee, Kun-Hyung; Nishiyama, Makoto; Nakajima, Hidenori; Tanaka, Akito; Komatsu, Yasuhiko; Nishino, Norikazu; Yoshida, Minoru; Horinouchi, Sueharu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4916-4921</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">FK228 is a histone deacetylase (HDAC) inhibitor, the mol. mechanism of inhibition of which has been unknown.  Here we show that redn. of an intramol. disulfide bond of FK228 greatly enhanced its inhibitory activity and that the disulfide bond was rapidly reduced in cells by cellular reducing activity involving glutathione.  Computer modeling suggests that one of the sulfhydryl groups of the reduced form of FK228 (redFK) interacts with the active-site zinc, preventing the access of the substrate.  HDAC1 and HDAC2 were more strongly inhibited by redFK than HDAC4 and HDAC6.  RedFK was less active than FK228 in inhibiting in vivo HDAC activity, due to rapid inactivation in medium and serum.  Thus, FK228 serves as a stable prodrug to inhibit class I enzymes and is activated by redn. after uptake into the cells.  The glutathione-mediated activation also implicates its clin. usefulness for counteracting glutathione-mediated drug resistance in chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohcl-TJJMBcbVg90H21EOLACvtfcHk0liFNRsaMyOuFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmslylu78%253D&md5=691ed84f8a35b1664b4bab770d69a194</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFurumai%26aufirst%3DR.%26aulast%3DMatsuyama%26aufirst%3DA.%26aulast%3DKobashi%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DNishiyama%26aufirst%3DM.%26aulast%3DNakajima%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DNishino%26aufirst%3DN.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DHorinouchi%26aufirst%3DS.%26atitle%3DFK228%2520%2528depsipeptide%2529%2520as%2520a%2520natural%2520prodrug%2520that%2520inhibits%2520class%2520I%2520histone%2520deacetylases%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D4916%26epage%3D4921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Plumb, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watkins, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R.</span><span> </span><span class="NLM_article-title">Pharmacodynamic response and inhibition of growth of human tumor xenografts by the novel histone deacetylase inhibitor PXD101</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=12939461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3sXms1GktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=721-728&author=J.+A.+Plumbauthor=P.+W.+Finnauthor=R.+J.+Williamsauthor=M.+J.+Bandaraauthor=M.+R.+Romeroauthor=C.+J.+Watkinsauthor=N.+B.+La+Thangueauthor=R.+Brown&title=Pharmacodynamic+response+and+inhibition+of+growth+of+human+tumor+xenografts+by+the+novel+histone+deacetylase+inhibitor+PXD101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacodynamic Response and Inhibition of Growth of Human Tumor Xenografts by the Novel Histone Deacetylase Inhibitor PXD101</span></div><div class="casAuthors">Plumb, Jane A.; Finn, Paul W.; Williams, Robert J.; Bandara, Morwenna J.; Romero, M. Rosario; Watkins, Claire J.; La Thangue, Nicholas B.; Brown, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">721-728</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone acetylation has a central role in the control of gene expression, influencing transcriptional control of many genes, including tumor suppressor genes.  PXD101 is a novel hydroxamate-type inhibitor of histone deacetylase activity that inhibits histone deacetylase activity in HeLa cell exts. with an IC50 of 27 nM and induces a concn.-dependent (0.2-5 μM) increase in acetylation of histone H4 in tumor cell lines.  PXD101 is cytotoxic in vitro in a no. of tumor cell lines with IC50s in the range 0.2-3.4 μM as detd. by a clonogenic assay and induces apoptosis.  Treatment of nude mice bearing human ovarian and colon tumor xenografts with PXD101 (10-40 mg/kg/day i.p.) daily for 7 days causes a significant dose-dependent growth delay with no obvious signs of toxicity to the mice.  Growth delay is also obsd. for xenografts of cisplatin-resistant ovarian tumor cells.  A marked increase in acetylation of H4 is detected in blood and tumor of mice 3 h after treatment with PXD101.  The inhibition of growth of human tumor xenografts in mice, with no apparent toxicity, suggests that PXD101 has potential as a novel antitumor agent.  Furthermore, the ability to measure histone acetylation in blood samples could provide a suitable pharmacodynamic end point to monitor drug activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqzC5_Q06W7Vg90H21EOLACvtfcHk0lgn3CtNlFO3JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXms1GktLo%253D&md5=7d25952434f0e544d8b61ab20ae5b0e4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPlumb%26aufirst%3DJ.%2BA.%26aulast%3DFinn%26aufirst%3DP.%2BW.%26aulast%3DWilliams%26aufirst%3DR.%2BJ.%26aulast%3DBandara%26aufirst%3DM.%2BJ.%26aulast%3DRomero%26aufirst%3DM.%2BR.%26aulast%3DWatkins%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DBrown%26aufirst%3DR.%26atitle%3DPharmacodynamic%2520response%2520and%2520inhibition%2520of%2520growth%2520of%2520human%2520tumor%2520xenografts%2520by%2520the%2520novel%2520histone%2520deacetylase%2520inhibitor%2520PXD101%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2003%26volume%3D2%26spage%3D721%26epage%3D728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Taube, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterlin, M.</span><span> </span><span class="NLM_article-title">Lost in transcription: molecular mechanisms that control HIV latency</span> <span class="citation_source-journal">Viruses</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">902</span><span class="NLM_x">–</span> <span class="NLM_lpage">927</span><span class="refDoi"> DOI: 10.3390/v5030902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.3390%2Fv5030902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=23518577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVSrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=902-927&author=R.+Taubeauthor=M.+Peterlin&title=Lost+in+transcription%3A+molecular+mechanisms+that+control+HIV+latency&doi=10.3390%2Fv5030902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lost in transcription: molecular mechanisms that control HIV latency</span></div><div class="casAuthors">Taube, Ran; Peterlin, Boris Matija</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">902-928</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Highly active antiretroviral therapy (HAART) has limited the replication and spread of the human immunodeficiency virus (HIV).  However, despite treatment, HIV infection persists in latently infected reservoirs, and once therapy is interrupted, viral replication rebounds quickly.  Extensive efforts are being directed at eliminating these cell reservoirs.  This feat can be achieved by reactivating latent HIV while administering drugs that prevent new rounds of infection and allow the immune system to clear the virus.  However, current approaches to HIV eradication have not been effective.  Moreover, as HIV latency is multifactorial, the significance of each of its mol. mechanisms is still under debate.  Among these, transcriptional repression as a result of reduced levels and activity of the pos. transcription elongation factor b (P-TEFb: CDK9/cyclin T) plays a significant role.  Therefore, increasing levels of P-TEFb expression and activity is an excellent strategy to stimulate viral gene expression.  This review summarizes the multiple steps that cause HIV to enter into latency.  It positions the interplay between transcriptionally active and inactive host transcriptional activators and their viral partner Tat as valid targets for the development of new strategies to reactivate latent viral gene expression and eradicate HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP3nHdi5AyOLVg90H21EOLACvtfcHk0lgn3CtNlFO3JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVSrtLk%253D&md5=a2ba0ed16a2b7cceef4a15e355fb63d2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3390%2Fv5030902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv5030902%26sid%3Dliteratum%253Aachs%26aulast%3DTaube%26aufirst%3DR.%26aulast%3DPeterlin%26aufirst%3DM.%26atitle%3DLost%2520in%2520transcription%253A%2520molecular%2520mechanisms%2520that%2520control%2520HIV%2520latency%26jtitle%3DViruses%26date%3D2013%26volume%3D5%26spage%3D902%26epage%3D927%26doi%3D10.3390%2Fv5030902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Richman, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delaney, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, R. J.</span><span> </span><span class="NLM_article-title">The challenge of finding a cure for HIV infection</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x">, </span> <span class="NLM_fpage">1304</span><span class="NLM_x">–</span> <span class="NLM_lpage">1307</span><span class="refDoi"> DOI: 10.1126/science.1165706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1126%2Fscience.1165706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19265012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFemur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2009&pages=1304-1307&author=D.+D.+Richmanauthor=D.+M.+Margolisauthor=M.+Delaneyauthor=W.+C.+Greeneauthor=D.+Hazudaauthor=R.+J.+Pomerantz&title=The+challenge+of+finding+a+cure+for+HIV+infection&doi=10.1126%2Fscience.1165706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The Challenge of Finding a Cure for HIV Infection</span></div><div class="casAuthors">Richman, Douglas D.; Margolis, David M.; Delaney, Martin; Greene, Warner C.; Hazuda, Daria; Pomerantz, Roger J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">323</span>
        (<span class="NLM_cas:issue">5919</span>),
    <span class="NLM_cas:pages">1304-1307</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Although combination therapy for HIV infection represents a triumph for modern medicine, chronic suppressive therapy is required to contain persistent infection in reservoirs such as latently infected CD4+ lymphocytes and cells of the macrophage-monocyte lineage.  Despite its success, chronic suppressive therapy is limited by its cost, the requirement of lifelong adherence, and the unknown effects of long-term treatment.  This review discusses our current understanding of suppressive antiretroviral therapy, the latent viral reservoir, and the needs for and challenges of attacking this reservoir to achieve a cure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Yfud1T9cxbVg90H21EOLACvtfcHk0lgn3CtNlFO3JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFemur0%253D&md5=855ce658e427fb8dd40925a51fdb7d20</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.1165706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1165706%26sid%3Dliteratum%253Aachs%26aulast%3DRichman%26aufirst%3DD.%2BD.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26aulast%3DDelaney%26aufirst%3DM.%26aulast%3DGreene%26aufirst%3DW.%2BC.%26aulast%3DHazuda%26aufirst%3DD.%26aulast%3DPomerantz%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520challenge%2520of%2520finding%2520a%2520cure%2520for%2520HIV%2520infection%26jtitle%3DScience%26date%3D2009%26volume%3D323%26spage%3D1304%26epage%3D1307%26doi%3D10.1126%2Fscience.1165706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Siliciano, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kajdas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finzi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadwick, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolick, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovacs, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gange, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siliciano, R. F.</span><span> </span><span class="NLM_article-title">Long-term follow-up studies confirm the stability of the latent reservoir for HIV-1 in resting CD4+ T cells</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span><span class="refDoi"> DOI: 10.1038/nm880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fnm880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=12754504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFOnu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=727-728&author=J.+D.+Silicianoauthor=J.+Kajdasauthor=D.+Finziauthor=T.+C.+Quinnauthor=K.+Chadwickauthor=J.+B.+Margolickauthor=C.+Kovacsauthor=S.+J.+Gangeauthor=R.+F.+Siliciano&title=Long-term+follow-up+studies+confirm+the+stability+of+the+latent+reservoir+for+HIV-1+in+resting+CD4%2B+T+cells&doi=10.1038%2Fnm880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Long-term follow-up studies confirm the stability of the latent reservoir for HIV-1 in resting CD4+ T cells</span></div><div class="casAuthors">Siliciano, Janet D.; Kajdas, Joleen; Finzi, Diana; Quinn, Thomas C.; Chadwick, Karen; Margolick, Joseph B.; Kovacs, Colin; Gange, Stephen J.; Siliciano, Robert F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">727-728</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Latent HIV-1 persists in resting memory CD4+ T cells, even in patients receiving highly active antiretroviral therapy (HAART).  It has been unclear how stable this latent reservoir is and whether its persistence reflects replenishment by low-level viremia.  Here we show that even in treated patients who have had no detectable viremia for as long as 7 yr, the reservoir decays so slowly (t1/2 = 44 mo) that eradication is unlikely.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZzVUrIkxch7Vg90H21EOLACvtfcHk0liFsBjC9s-N2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFOnu7k%253D&md5=c5f963cda784002f8282d3aa3b2c67a3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnm880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm880%26sid%3Dliteratum%253Aachs%26aulast%3DSiliciano%26aufirst%3DJ.%2BD.%26aulast%3DKajdas%26aufirst%3DJ.%26aulast%3DFinzi%26aufirst%3DD.%26aulast%3DQuinn%26aufirst%3DT.%2BC.%26aulast%3DChadwick%26aufirst%3DK.%26aulast%3DMargolick%26aufirst%3DJ.%2BB.%26aulast%3DKovacs%26aufirst%3DC.%26aulast%3DGange%26aufirst%3DS.%2BJ.%26aulast%3DSiliciano%26aufirst%3DR.%2BF.%26atitle%3DLong-term%2520follow-up%2520studies%2520confirm%2520the%2520stability%2520of%2520the%2520latent%2520reservoir%2520for%2520HIV-1%2520in%2520resting%2520CD4%252B%2520T%2520cells%26jtitle%3DNat.%2520Med.%26date%3D2003%26volume%3D9%26spage%3D727%26epage%3D728%26doi%3D10.1038%2Fnm880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Coiras, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Huertas, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Olmeda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcami, J.</span><span> </span><span class="NLM_article-title">Understanding HIV-1 latency provides clues for the eradication of long-term reservoirs</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">798</span><span class="NLM_x">–</span> <span class="NLM_lpage">812</span><span class="refDoi"> DOI: 10.1038/nrmicro2223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fnrmicro2223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19834480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kqsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=798-812&author=M.+Coirasauthor=M.+R.+Lopez-Huertasauthor=M.+Perez-Olmedaauthor=J.+Alcami&title=Understanding+HIV-1+latency+provides+clues+for+the+eradication+of+long-term+reservoirs&doi=10.1038%2Fnrmicro2223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding HIV-1 latency provides clues for the eradication of long-term reservoirs</span></div><div class="casAuthors">Coiras, Mayte; Lopez-Huertas, Maria Rosa; Perez-Olmeda, Mayte; Alcami, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">798-812</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  HIV-1 can infect both activated and resting, non-dividing cells, following which the viral genome can be permanently integrated into a host cell chromosome.  Latent HIV-1 reservoirs are established early during primary infection and constitute a major barrier to eradication, even in the presence of highly active antiretroviral therapy.  This Review analyses the mol. mechanisms that are necessary for the establishment of HIV-1 latency and their relationships with different cellular and anatomical reservoirs, and discusses the current treatment strategies for targeting viral persistence in reservoirs, their main limitations and future perspectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5tPDPAK2R5bVg90H21EOLACvtfcHk0liFsBjC9s-N2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kqsb7J&md5=4df3674826db5a343d695693c1aec575</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2223%26sid%3Dliteratum%253Aachs%26aulast%3DCoiras%26aufirst%3DM.%26aulast%3DLopez-Huertas%26aufirst%3DM.%2BR.%26aulast%3DPerez-Olmeda%26aufirst%3DM.%26aulast%3DAlcami%26aufirst%3DJ.%26atitle%3DUnderstanding%2520HIV-1%2520latency%2520provides%2520clues%2520for%2520the%2520eradication%2520of%2520long-term%2520reservoirs%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2009%26volume%3D7%26spage%3D798%26epage%3D812%26doi%3D10.1038%2Fnrmicro2223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Zack, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrigo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weitsman, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Go, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haislip, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I. S.</span><span> </span><span class="NLM_article-title">HIV-1 entry into quiescent primary lymphocytes: molecular analysis reveals a labile, latent viral structure</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1016/0092-8674(90)90802-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2F0092-8674%2890%2990802-L" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1990&pages=213-222&author=J.+A.+Zackauthor=S.+J.+Arrigoauthor=S.+R.+Weitsmanauthor=A.+S.+Goauthor=A.+Haislipauthor=I.+S.+Chen&title=HIV-1+entry+into+quiescent+primary+lymphocytes%3A+molecular+analysis+reveals+a+labile%2C+latent+viral+structure&doi=10.1016%2F0092-8674%2890%2990802-L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2890%2990802-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252890%252990802-L%26sid%3Dliteratum%253Aachs%26aulast%3DZack%26aufirst%3DJ.%2BA.%26aulast%3DArrigo%26aufirst%3DS.%2BJ.%26aulast%3DWeitsman%26aufirst%3DS.%2BR.%26aulast%3DGo%26aufirst%3DA.%2BS.%26aulast%3DHaislip%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DI.%2BS.%26atitle%3DHIV-1%2520entry%2520into%2520quiescent%2520primary%2520lymphocytes%253A%2520molecular%2520analysis%2520reveals%2520a%2520labile%252C%2520latent%2520viral%2520structure%26jtitle%3DCell%26date%3D1990%26volume%3D61%26spage%3D213%26epage%3D222%26doi%3D10.1016%2F0092-8674%2890%2990802-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Lewin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wightman, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, P. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purcell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaunders, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bloch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, A. D.</span><span> </span><span class="NLM_article-title">Virologic determinants of success after structured treatment interruptions of antiretrovirals in acute HIV-1 infection</span> <span class="citation_source-journal">JAIDS, J. Acquired Immune Defic. Syndr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1097/QAI.0b013e31815dbf7f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1097%2FQAI.0b013e31815dbf7f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=18300365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGmsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=140-147&author=S.+R.+Lewinauthor=J.+M.+Murrayauthor=A.+Solomonauthor=F.+Wightmanauthor=P.+U.+Cameronauthor=D.+J.+Purcellauthor=J.+J.+Zaundersauthor=P.+Greyauthor=M.+Blochauthor=D.+Smithauthor=D.+A.+Cooperauthor=A.+D.+Kelleher&title=Virologic+determinants+of+success+after+structured+treatment+interruptions+of+antiretrovirals+in+acute+HIV-1+infection&doi=10.1097%2FQAI.0b013e31815dbf7f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Virologic Determinants of Success After Structured Treatment Interruptions of Antiretrovirals in Acute HIV-1 Infection</span></div><div class="casAuthors">Lewin, Sharon R.; Murray, John M.; Solomon, Ajantha; Wightman, Fiona; Cameron, Paul U.; Purcell, Damian J.; Zaunders, John J.; Grey, Pat; Bloch, Mark; Smith, Don; Cooper, David A.; Kelleher, Anthony D.</div><div class="citationInfo"><span class="NLM_cas:title">JAIDS, Journal of Acquired Immune Deficiency Syndromes</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">140-147</span>CODEN:
                <span class="NLM_cas:coden">JJASFJ</span>;
        ISSN:<span class="NLM_cas:issn">1525-4135</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Background: Latently infected resting memory CD4 T cells are thought to be the major reservoir that contributes to rebound viremia after cessation of antiretrovirals (ARVs).  Commencing ARVs during primary HIV-1 infection (PHI) may limit the size of the latent pool and lead to improved control of viral replication during structured treatment interruption (STI).  Methods: Individuals with PHI (n = 59) were randomized to receive ARVs with or without hydroxyurea.  After STI, a good response was defined as maintenance of HIV-1 RNA <5000 copies/mL for 24 wk off therapy.  In a detailed prospective virol. substudy (n = 19), integrated HIV-1 DNA, total HIV-1 DNA, and cell-assocd. HIV-1 unspliced (US) RNA were quantified using a real-time polymerase chain reaction assay.  Results: The plasma HIV-1 RNA 12 wk after the initiation of ARVs was significantly lower in good responders (n = 7) compared with poor responders (n = 12) (P = 0.005).  There were no significant differences between good and poor responders in integrated HIV-1 DNA, HIV-1 DNA, and HIV-1 US RNA.  Integrated HIV-1 DNA before initiation of ARVs was strongly correlated with plasma HIV-1 RNA at week 12 (P = 0.006, r = 0.81).  Conclusion: HIV-1 RNA measured 12 wk after initiation of ARV was the only virol. variable assocd. with viral rebound after treatment interruption in PHI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNtLX8clo1o7Vg90H21EOLACvtfcHk0liFsBjC9s-N2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGmsb8%253D&md5=6bec484bb4aad2a8ea4f7c6f49c50410</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FQAI.0b013e31815dbf7f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQAI.0b013e31815dbf7f%26sid%3Dliteratum%253Aachs%26aulast%3DLewin%26aufirst%3DS.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%2BM.%26aulast%3DSolomon%26aufirst%3DA.%26aulast%3DWightman%26aufirst%3DF.%26aulast%3DCameron%26aufirst%3DP.%2BU.%26aulast%3DPurcell%26aufirst%3DD.%2BJ.%26aulast%3DZaunders%26aufirst%3DJ.%2BJ.%26aulast%3DGrey%26aufirst%3DP.%26aulast%3DBloch%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DCooper%26aufirst%3DD.%2BA.%26aulast%3DKelleher%26aufirst%3DA.%2BD.%26atitle%3DVirologic%2520determinants%2520of%2520success%2520after%2520structured%2520treatment%2520interruptions%2520of%2520antiretrovirals%2520in%2520acute%2520HIV-1%2520infection%26jtitle%3DJAIDS%252C%2520J.%2520Acquired%2520Immune%2520Defic.%2520Syndr.%26date%3D2008%26volume%3D47%26spage%3D140%26epage%3D147%26doi%3D10.1097%2FQAI.0b013e31815dbf7f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Hamer, D. H.</span><span> </span><span class="NLM_article-title">Can HIV be cured? Mechanisms of HIV persistence and strategies to combat it</span> <span class="citation_source-journal">Curr. HIV Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">111</span><span class="refDoi"> DOI: 10.2174/1570162043484915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.2174%2F1570162043484915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=15078175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVGqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=99-111&author=D.+H.+Hamer&title=Can+HIV+be+cured%3F+Mechanisms+of+HIV+persistence+and+strategies+to+combat+it&doi=10.2174%2F1570162043484915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Can HIV be cured? Mechanisms of HIV persistence and strategies to combat it</span></div><div class="casAuthors">Hamer, Dean H.</div><div class="citationInfo"><span class="NLM_cas:title">Current HIV Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-111</span>CODEN:
                <span class="NLM_cas:coden">CHRUBF</span>;
        ISSN:<span class="NLM_cas:issn">1570-162X</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Stable remission is the ultimate goal of HIV therapy.  A review of recent studies on the ability of HIV to persist despite highly active antiretroviral therapy (HAART) and immune stimulation suggests that achieving this goal will require four developments in basic and clin. science.  First, more effective antiretroviral therapies, targeted at proteins other than reverse transcriptase and protease, in order to eliminate the cryptic replication that continues despite best available HAART.  Second, agents that activate latent HIV gene expression in quiescent CD4 memory T cells, thereby exposing this viral reservoir to therapeutic intervention by a "shock and kill" strategy.  Third, mols. such as immunotoxins that specifically recognize HIV-encoded membrane proteins and thereby potentiate the destruction of infected cells.  Fourth, and still most distant, novel approaches such as genetically engineered cytotoxic T lymphocytes or anti-HIV microbes to suppress rekindling of infection by residual virus sequestered in anatomical and cellular reservoirs.  Although each of these steps will be difficult to achieve, the many benefits of a cure for HIV make this a worthwhile pursuit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmOGegrIx1CrVg90H21EOLACvtfcHk0lipoan3ofD9ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVGqsLk%253D&md5=b0d3ac7855dced12e419f8c380adcea8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F1570162043484915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1570162043484915%26sid%3Dliteratum%253Aachs%26aulast%3DHamer%26aufirst%3DD.%2BH.%26atitle%3DCan%2520HIV%2520be%2520cured%253F%2520Mechanisms%2520of%2520HIV%2520persistence%2520and%2520strategies%2520to%2520combat%2520it%26jtitle%3DCurr.%2520HIV%2520Res.%26date%3D2004%26volume%3D2%26spage%3D99%26epage%3D111%26doi%3D10.2174%2F1570162043484915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Ott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Q.</span><span> </span><span class="NLM_article-title">The control of HIV transcription: keeping RNA polymerase II on track</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">426</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1016/j.chom.2011.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.chom.2011.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=22100159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2msrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=426-435&author=M.+Ottauthor=M.+Geyerauthor=Q.+Zhou&title=The+control+of+HIV+transcription%3A+keeping+RNA+polymerase+II+on+track&doi=10.1016%2Fj.chom.2011.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The control of HIV transcription: Keeping RNA polymerase II on track</span></div><div class="casAuthors">Ott, Melanie; Geyer, Matthias; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">426-435</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Thirteen years ago, human cyclin T1 was identified as part of the pos. transcription elongation factor b (P-TEFb) and the long-sought host cofactor for the HIV-1 transactivator Tat.  Recent years have brought new insights into the intricate regulation of P-TEFb function and its relationship with Tat, revealing novel mechanisms for controlling HIV transcription and fueling new efforts to overcome the barrier of transcriptional latency in eradicating HIV.  Moreover, the improved understanding of HIV and Tat forms a basis for studying transcription elongation control in general.  Here, we review advances in HIV transcription research with a focus on the growing family of cellular P-TEFb complexes, structural insights into the interactions between Tat, P-TEFb, and TAR RNA, and the multifaceted regulation of these interactions by posttranscriptional modifications of Tat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIp4eyVG6sSLVg90H21EOLACvtfcHk0lipoan3ofD9ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2msrzM&md5=c6bb819316c7c9439d7971ccca5c3fab</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2011.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2011.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DOtt%26aufirst%3DM.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DThe%2520control%2520of%2520HIV%2520transcription%253A%2520keeping%2520RNA%2520polymerase%2520II%2520on%2520track%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2011%26volume%3D10%26spage%3D426%26epage%3D435%26doi%3D10.1016%2Fj.chom.2011.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sakane, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagans, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizusawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lassen, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnoelzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">Activation of HIV transcription by the viral Tat protein requires a demethylation step mediated by lysine-specific demethylase 1 (LSD1/KDM1)</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1002184</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1002184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.ppat.1002184" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002184&author=N.+Sakaneauthor=H.+S.+Kwonauthor=S.+Pagansauthor=K.+Kaehlckeauthor=Y.+Mizusawaauthor=M.+Kamadaauthor=K.+G.+Lassenauthor=J.+Chanauthor=W.+C.+Greeneauthor=M.+Schnoelzerauthor=M.+Ott&title=Activation+of+HIV+transcription+by+the+viral+Tat+protein+requires+a+demethylation+step+mediated+by+lysine-specific+demethylase+1+%28LSD1%2FKDM1%29&doi=10.1371%2Fjournal.ppat.1002184"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002184%26sid%3Dliteratum%253Aachs%26aulast%3DSakane%26aufirst%3DN.%26aulast%3DKwon%26aufirst%3DH.%2BS.%26aulast%3DPagans%26aufirst%3DS.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMizusawa%26aufirst%3DY.%26aulast%3DKamada%26aufirst%3DM.%26aulast%3DLassen%26aufirst%3DK.%2BG.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DGreene%26aufirst%3DW.%2BC.%26aulast%3DSchnoelzer%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DActivation%2520of%2520HIV%2520transcription%2520by%2520the%2520viral%2520Tat%2520protein%2520requires%2520a%2520demethylation%2520step%2520mediated%2520by%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%252FKDM1%2529%26jtitle%3DPLoS%2520Pathog.%26date%3D2011%26volume%3D7%26spage%3De1002184%26doi%3D10.1371%2Fjournal.ppat.1002184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Quivy, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Walque, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Lint, C.</span><span> </span><span class="NLM_article-title">Chromatin-associated regulation of HIV-1 transcription: implications for the development of therapeutic strategies</span> <span class="citation_source-journal">Subcell Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">371</span><span class="NLM_x">–</span> <span class="NLM_lpage">396</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=17484137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A280%3ADC%252BD2s3nsVOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2007&pages=371-396&author=V.+Quivyauthor=S.+De+Walqueauthor=C.+Van+Lint&title=Chromatin-associated+regulation+of+HIV-1+transcription%3A+implications+for+the+development+of+therapeutic+strategies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-associated regulation of HIV-1 transcription: implications for the development of therapeutic strategies</span></div><div class="casAuthors">Quivy Vincent; De Walque Stephane; Van Lint Carine</div><div class="citationInfo"><span class="NLM_cas:title">Sub-cellular biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">371-96</span>
        ISSN:<span class="NLM_cas:issn">0306-0225</span>.
    </div><div class="casAbstract">Human Immunodeficiency Virus type 1 (HIV-1) infection can now be treated effectively in many patients in the developed world, using combinations of antiretroviral therapeutics, called Highly Active Anti-Retroviral Therapy (HAART).  However, despite prolonged treatment with HAART, the persistence of latently HIV-1-infected cellular reservoirs harboring transcriptionally silent but replication-competent proviruses represents the major hurdle to virus eradication.  These latently infected cells are a permanent source for virus reactivation and lead to a rebound of the viral load after interruption of HAART.  Therefore, a greater understanding of the molecular mechanisms regulating proviral latency and reactivation should lead to rational strategies aimed at purging these cellular reservoirs of HIV-1.  This review summarizes our current knowledge and understanding of the elements involved in HIV-1 transcriptional reactivation: (1) the site of integration; (2) the transcription factor NF-kappaB, which is induced by proinflammatory cytokines (such as TNFalpha) and binds to two kappaB sites in the HIV-1 promoter region; (3) the specific remodeling of a single nucleosome (called nuc-1 and located immediately downstream of the HIV-1 transcription start site under latency conditions) upon activation of the HIV-1 promoter; (4) post-translational acetylation of histones and of non-histone proteins (following treatment with deacetylases inhibitors, which induce viral transcription and nuc-1 remodeling); and (5) the viral trans-activator Tat, which promotes transcription by mediating the recruitment to the HIV-1 promoter of histone-modifying enzymes and ATP-dependent chromatin remodeling complexes required for nucleosome disruption and transcriptional processivity.  Finally, this review highlights experimental therapies aimed at administrating HIV-1 gene expression activators (such as HDAC inhibitors) combined with an effective HAART in order to reactivate and decrease/eliminate the pool of latently HIV-1-infected cellular reservoirs</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRlyO3RExbSUr5XHngOXiV_fW6udTcc2ebjW9nxi8rZFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3nsVOgtw%253D%253D&md5=63f50592399c37deb891ad9619544b36</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQuivy%26aufirst%3DV.%26aulast%3DDe%2BWalque%26aufirst%3DS.%26aulast%3DVan%2BLint%26aufirst%3DC.%26atitle%3DChromatin-associated%2520regulation%2520of%2520HIV-1%2520transcription%253A%2520implications%2520for%2520the%2520development%2520of%2520therapeutic%2520strategies%26jtitle%3DSubcell%2520Biochem.%26date%3D2007%26volume%3D41%26spage%3D371%26epage%3D396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Vigushin, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirsaidi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adcock, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coombes, R. C.</span><span> </span><span class="NLM_article-title">Trichostatin A is a histone deacetylase inhibitor with potent antitumor activity against breast cancer in vivo</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">971</span><span class="NLM_x">–</span> <span class="NLM_lpage">976</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=11309348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1eltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=971-976&author=D.+M.+Vigushinauthor=S.+Aliauthor=P.+E.+Paceauthor=N.+Mirsaidiauthor=K.+Itoauthor=I.+Adcockauthor=R.+C.+Coombes&title=Trichostatin+A+is+a+histone+deacetylase+inhibitor+with+potent+antitumor+activity+against+breast+cancer+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Trichostatin a is a histone deacetylase inhibitor with potent antitumor activity against breast cancer in vivo</span></div><div class="casAuthors">Vigushin, David M.; Ali, Simak; Pace, Paul E.; Mirsaidi, Nina; Ito, Kazuhiro; Adcock, Ian; Coombes, R. Charles</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">971-976</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Trichostatin A (TSA), an antifungal antibiotic with cytostatic and differentiating properties in mammalian cell culture, is a potent and specific inhibitor of histone deacetylase (HDAC) activity.  The purpose of this study was to evaluate the antiproliferative and HDAC inhibitory activity of TSA in vitro in human breast cancer cell lines and to assess its antitumor efficacy and toxicity in vivo in a carcinogen-induced rat mammary cancer model.  TSA inhibited proliferation of eight breast carcinoma cell lines with mean ± SD IC50 of 124.4±120.4 nM (range, 26.4-308.1 nM).  HDAC inhibitory activity of TSA was similar in all cell lines with mean ± SD IC50 of 2.4±0.5 nM (range, 1.5-2.9 nM), and TSA treatment resulted in pronounced histone H4 hyperacetylation.  In randomized controlled efficacy studies using the N-methyl-N-nitrosourea carcinogen-induced rat mammary carcinoma model, TSA had pronounced antitumor activity in vivo when administered to 16 animals at a dose of 500 μg/kg by s.c. injection daily for 4 wk compared with 14 control animals.  Furthermore, TSA did not cause any measurable toxicity in doses of up to 5 mg/kg by s.c. injection.  Forty-one tumors from 26 animals were examd. by histol.  Six tumors from 3 rats treated with TSA and 14 tumors from 9 control animals were adenocarcinomas.  In contrast, 19 tumors from 12 TSA-treated rats had a benign phenotype, either fibroadenoma or tubular adenoma, suggesting that the antitumor activity of TSA may be attributable to induction of differentiation.  Two control rats each had tumors with benign histol.  The present studies confirm the potent dose-dependent antitumor activity of TSA against breast cancer in vitro and in vivo, strongly supporting HDAC as a mol. target for anticancer therapy in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokpxVxd2w5QbVg90H21EOLACvtfcHk0ljgWX4wI1bkAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1eltrw%253D&md5=c3853a7e52c3f87dbac938b996d8e57c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVigushin%26aufirst%3DD.%2BM.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DPace%26aufirst%3DP.%2BE.%26aulast%3DMirsaidi%26aufirst%3DN.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DAdcock%26aufirst%3DI.%26aulast%3DCoombes%26aufirst%3DR.%2BC.%26atitle%3DTrichostatin%2520A%2520is%2520a%2520histone%2520deacetylase%2520inhibitor%2520with%2520potent%2520antitumor%2520activity%2520against%2520breast%2520cancer%2520in%2520vivo%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2001%26volume%3D7%26spage%3D971%26epage%3D976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Dokmanovic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, P. A.</span><span> </span><span class="NLM_article-title">Histone deacetylase inhibitors: overview and perspectives</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">981</span><span class="NLM_x">–</span> <span class="NLM_lpage">989</span><span class="refDoi"> DOI: 10.1158/1541-7786.MCR-07-0324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1158%2F1541-7786.MCR-07-0324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=17951399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFygsLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=981-989&author=M.+Dokmanovicauthor=C.+Clarkeauthor=P.+A.+Marks&title=Histone+deacetylase+inhibitors%3A+overview+and+perspectives&doi=10.1158%2F1541-7786.MCR-07-0324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase Inhibitors: Overview and Perspectives</span></div><div class="casAuthors">Dokmanovic, Milos; Clarke, Cathy; Marks, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">981-989</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase inhibitors (HDACi) comprise structurally diverse compds. that are a group of targeted anticancer agents.  The first of these new HDACi, vorinostat (suberoylanilide hydroxamic acid), has received Food and Drug Administration approval for treating patients with cutaneous T-cell lymphoma.  This review focuses on the activities of the 11 zinc-contg. HDACs, their histone and nonhistone protein substrates, and the different pathways by which HDACi induce transformed cell death.  A hypothesis is presented to explain the relative resistance of normal cells to HDACi-induced cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_V1LFFdlQYbVg90H21EOLACvtfcHk0ljgWX4wI1bkAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFygsLvN&md5=8696ff3e0f8e910fd6fc6a7e38579b95</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-07-0324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-07-0324%26sid%3Dliteratum%253Aachs%26aulast%3DDokmanovic%26aufirst%3DM.%26aulast%3DClarke%26aufirst%3DC.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26atitle%3DHistone%2520deacetylase%2520inhibitors%253A%2520overview%2520and%2520perspectives%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2007%26volume%3D5%26spage%3D981%26epage%3D989%26doi%3D10.1158%2F1541-7786.MCR-07-0324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Bolden, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peart, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, R. W.</span><span> </span><span class="NLM_article-title">Anticancer activities of histone deacetylase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">784</span><span class="refDoi"> DOI: 10.1038/nrd2133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fnrd2133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=16955068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=769-784&author=J.+E.+Boldenauthor=M.+J.+Peartauthor=R.+W.+Johnstone&title=Anticancer+activities+of+histone+deacetylase+inhibitors&doi=10.1038%2Fnrd2133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer activities of histone deacetylase inhibitors</span></div><div class="casAuthors">Bolden, Jessica E.; Peart, Melissa J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">769-784</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) are enzymes involved in the remodelling of chromatin, and have a key role in the epigenetic regulation of gene expression.  In addn., the activity of non-histone proteins can be regulated through HDAC-mediated hypo-acetylation.  In recent years, inhibition of HDACs has emerged as a potential strategy to reverse aberrant epigenetic changes assocd. with cancer, and several classes of HDAC inhibitors have been found to have potent and specific anticancer activities in preclin. studies.  However, such studies have also indicated that the effects of HDAC inhibitors could be considerably broader and more complicated than originally understood.  Here we summarize recent advances in the understanding of the mol. events that underlie the anticancer effects of HDAC inhibitors, and discuss how such information could be used in optimizing the development and application of these agents in the clinic, either as monotherapies or in combination with other anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhfqS1awkOHbVg90H21EOLACvtfcHk0ljgWX4wI1bkAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltrY%253D&md5=91e55ee942fda3e4e2055ddf299fc866</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrd2133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2133%26sid%3Dliteratum%253Aachs%26aulast%3DBolden%26aufirst%3DJ.%2BE.%26aulast%3DPeart%26aufirst%3DM.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DAnticancer%2520activities%2520of%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D769%26epage%3D784%26doi%3D10.1038%2Fnrd2133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Duverger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bibollet-Ruche, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cron, R. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutsch, O.</span><span> </span><span class="NLM_article-title">Determinants of the establishment of human immunodeficiency virus type 1 latency</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">3078</span><span class="NLM_x">–</span> <span class="NLM_lpage">3093</span><span class="refDoi"> DOI: 10.1128/JVI.02058-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1128%2FJVI.02058-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19144703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjs1Wrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=3078-3093&author=A.+Duvergerauthor=J.+Jonesauthor=J.+Mayauthor=F.+Bibollet-Rucheauthor=F.+A.+Wagnerauthor=R.+Q.+Cronauthor=O.+Kutsch&title=Determinants+of+the+establishment+of+human+immunodeficiency+virus+type+1+latency&doi=10.1128%2FJVI.02058-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Determinants of the establishment of human immunodeficiency virus type 1 latency</span></div><div class="casAuthors">Duverger, Alexandra; Jones, Jennifer; May, Jori; Bibollet-Ruche, Frederic; Wagner, Frederic A.; Cron, Randall Q.; Kutsch, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3078-3093</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Recent research has emphasized the notion that human immunodeficiency virus type 1 (HIV-1) latency is controlled by a restrictive histone code at, or DNA methylation of, the integrated viral promoter (long terminal repeat [LTR]).  The present concept of HIV-1 latency has essentially been patterned from the principles of cellular gene regulation.  Here we introduce an exptl. system that allows for the qual. and quant. kinetic study of latency establishment and maintenance at the population level.  In this system, we find no evidence that HIV-1 latency establishment is the consequence of downregulation of initial active infection followed by the establishment of a restrictive histone code at the viral LTR.  Latent infection was established following integration of the virus in the absence of viral gene expression (silent integration) and was a function of the NF-κB activation level in the host cell at the time of infection.  In the absence of a role for epigenetic regulation, we demonstrate that transcriptional interference, a mechanism that has recently been suggested to add to the stabilization of HIV-1 latency, is the primary mechanism to govern latency maintenance.  These findings provide direct exptl. evidence that the high no. of viral integration events (>90%) found in actively expressed genes of CD4+ memory T cells from highly active antiretroviral therapy-suppressed patients represent indeed latent infection events and that transcriptional interference may be the primary mechanism to control HIV-1 latency in vivo.  HIV-1 latency may thus not be governed by the principles of cellular gene regulation, and therapeutic strategies to deplete the pool of latently HIV-1-infected cells should be reconsidered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKJiZWjbwaY7Vg90H21EOLACvtfcHk0lidx49p1GytSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjs1Wrtro%253D&md5=09b01dc317f538ac492dffd2741c5180</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FJVI.02058-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02058-08%26sid%3Dliteratum%253Aachs%26aulast%3DDuverger%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DJ.%26aulast%3DMay%26aufirst%3DJ.%26aulast%3DBibollet-Ruche%26aufirst%3DF.%26aulast%3DWagner%26aufirst%3DF.%2BA.%26aulast%3DCron%26aufirst%3DR.%2BQ.%26aulast%3DKutsch%26aufirst%3DO.%26atitle%3DDeterminants%2520of%2520the%2520establishment%2520of%2520human%2520immunodeficiency%2520virus%2520type%25201%2520latency%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D3078%26epage%3D3093%26doi%3D10.1128%2FJVI.02058-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Keedy, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archin, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gates, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espeseth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, D. M.</span><span> </span><span class="NLM_article-title">A limited group of class I histone deacetylases acts to repress human immunodeficiency virus type 1 expression</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">4749</span><span class="NLM_x">–</span> <span class="NLM_lpage">4756</span><span class="refDoi"> DOI: 10.1128/JVI.02585-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1128%2FJVI.02585-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19279091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvVGhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=4749-4756&author=K.+S.+Keedyauthor=N.+M.+Archinauthor=A.+T.+Gatesauthor=A.+Espesethauthor=D.+J.+Hazudaauthor=D.+M.+Margolis&title=A+limited+group+of+class+I+histone+deacetylases+acts+to+repress+human+immunodeficiency+virus+type+1+expression&doi=10.1128%2FJVI.02585-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A limited group of class I histone deacetylases acts to repress human immunodeficiency virus type 1 expression</span></div><div class="casAuthors">Keedy, Kara S.; Archin, Nancie M.; Gates, Adam T.; Espeseth, Amy; Hazuda, Daria J.; Margolis, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4749-4756</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Silencing of the integrated human immunodeficiency virus type 1 (HIV-1) genome in resting CD4+ T cells is a significant contributor to the persistence of infection, allowing the virus to evade both immune detection and pharmaceutical attack.  Nonselective histone deacetylase (HDAC) inhibitors are capable of inducing expression of quiescent HIV-1 in latently infected cells.  However, potent global HDAC inhibition can induce host toxicity.  To det. the specific HDACs that regulate HIV-1 transcription, we evaluated HDAC1 to HDAC11 RNA expression and protein expression and compartmentalization in the resting CD4+ T cells of HIV-1-pos., aviremic patients.  HDAC1, -3, and -7 had the highest mRNA expression levels in these cells.  Although all HDACs were detected in resting CD4+ T cells by Western blot anal., HDAC5, -8, and -11 were primarily sequestered in the cytoplasm.  Using chromatin immunopptn. assays, we detected HDAC1, -2, and -3 at the HIV-1 promoter in Jurkat J89GFP cells.  Targeted inhibition of HDACs by small interfering RNA demonstrated that HDAC2 and HDAC3 contribute to repression of HIV-1 long terminal repeat expression in the HeLa P4/R5 cell line model of latency.  Together, these results suggest that HDAC inhibitors specific for a limited no. of class I HDACs may offer a targeted approach to the disruption of persistent HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLmd8sCQLl4rVg90H21EOLACvtfcHk0lidx49p1GytSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvVGhsLk%253D&md5=bf4aac6fc7096a987b6d7698796ffe10</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FJVI.02585-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02585-08%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DK.%2BS.%26aulast%3DArchin%26aufirst%3DN.%2BM.%26aulast%3DGates%26aufirst%3DA.%2BT.%26aulast%3DEspeseth%26aufirst%3DA.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26atitle%3DA%2520limited%2520group%2520of%2520class%2520I%2520histone%2520deacetylases%2520acts%2520to%2520repress%2520human%2520immunodeficiency%2520virus%2520type%25201%2520expression%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D4749%26epage%3D4756%26doi%3D10.1128%2FJVI.02585-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Choudhary, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, D. M.</span><span> </span><span class="NLM_article-title">Curing HIV: pharmacologic approaches to target HIV-1 latency</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">418</span><span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010510-100237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1146%2Fannurev-pharmtox-010510-100237" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=397-418&author=S.+K.+Choudharyauthor=D.+M.+Margolis&title=Curing+HIV%3A+pharmacologic+approaches+to+target+HIV-1+latency&doi=10.1146%2Fannurev-pharmtox-010510-100237"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010510-100237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010510-100237%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DS.%2BK.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26atitle%3DCuring%2520HIV%253A%2520pharmacologic%2520approaches%2520to%2520target%2520HIV-1%2520latency%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2011%26volume%3D51%26spage%3D397%26epage%3D418%26doi%3D10.1146%2Fannurev-pharmtox-010510-100237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kijima, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beppu, T.</span><span> </span><span class="NLM_article-title">Potent and specific inhibition of mammalian histone deacetylase both in vivo and in vitro by trichostatin A</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">17174</span><span class="NLM_x">–</span> <span class="NLM_lpage">17179</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=2211619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK3cXlvVyqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=17174-17179&author=M.+Yoshidaauthor=M.+Kijimaauthor=M.+Akitaauthor=T.+Beppu&title=Potent+and+specific+inhibition+of+mammalian+histone+deacetylase+both+in+vivo+and+in+vitro+by+trichostatin+A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and specific inhibition of mammalian histone deacetylase both in vivo and in vitro by trichostatin A</span></div><div class="casAuthors">Yoshida, Minoru; Kijima, Masako; Akita, Mitsuru; Beppu, Teruhiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">17174-9</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">(R)-Trichostatin A (TSA) is a Streptomyces product which causes the induction of Friend cell differentiation and specific inhibition of the cell cycle of normal rat fibroblasts in the G1 and G2 phases at very low concns.  TSA caused an accumulation of acetylated histone species in a variety of mammalian cell lines.  Pulse-labeling expts. indicated that TSA markedly prolonged the in vivo half-life of the labile acetyl groups on histones in mouse mammary gland tumor cells, FM3A.  The partially purified histone deacetylase from wild-type FM3A cells was effectively inhibited by TSA in a noncompetitive manner with Ki = 3.4 nM.  A newly isolated mutant cell line of FM3A resistant to TSA did not show the accumulation of the acetylated histones in the presence of a higher concn. of TSA.  The histone deacetylase prepn. from the mutant showed decreased sensitivity to TSA (Ki = 31 nM, noncompetitive).  These results clearly indicate that TSA is a potent and specific inhibitor of histone deacetylase and that the in vivo effect of TSA on cell proliferation and differentiation can be attributed to the inhibition of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_EbVVcRXYbVg90H21EOLACvtfcHk0lidx49p1GytSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlvVyqsrY%253D&md5=f2bed4f3b2d1502fd3610dfdf376b2dc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DKijima%26aufirst%3DM.%26aulast%3DAkita%26aufirst%3DM.%26aulast%3DBeppu%26aufirst%3DT.%26atitle%3DPotent%2520and%2520specific%2520inhibition%2520of%2520mammalian%2520histone%2520deacetylase%2520both%2520in%2520vivo%2520and%2520in%2520vitro%2520by%2520trichostatin%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D17174%26epage%3D17179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Darkin-Rattray, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurnett, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dulski, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crumley, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allocco, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cannova, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meinke, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colletti, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bednarek, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goetz, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrowski, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polishook, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmatz, D. M.</span><span> </span><span class="NLM_article-title">Apicidin: a novel antiprotozoal agent that inhibits parasite histone deacetylase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">13143</span><span class="NLM_x">–</span> <span class="NLM_lpage">13147</span><span class="refDoi"> DOI: 10.1073/pnas.93.23.13143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.93.23.13143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=8917558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK28XmvV2qtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=13143-13147&author=S.+J.+Darkin-Rattrayauthor=A.+M.+Gurnettauthor=R.+W.+Myersauthor=P.+M.+Dulskiauthor=T.+M.+Crumleyauthor=J.+J.+Alloccoauthor=C.+Cannovaauthor=P.+T.+Meinkeauthor=S.+L.+Collettiauthor=M.+A.+Bednarekauthor=S.+B.+Singhauthor=M.+A.+Goetzauthor=A.+W.+Dombrowskiauthor=J.+D.+Polishookauthor=D.+M.+Schmatz&title=Apicidin%3A+a+novel+antiprotozoal+agent+that+inhibits+parasite+histone+deacetylase&doi=10.1073%2Fpnas.93.23.13143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Apicidin: a novel antiprotozoal agent that inhibits parasite histone deacetylase</span></div><div class="casAuthors">Darkin-Rattray, Sandra J.; Gurnett, Anne M.; Myers, Robert W.; Dulski, Paula M.; Crumley, Tami M.; Allocco, John J.; Cannova, Christine; Meinke, Peter T.; Colletti, Steven L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">13143-13147</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A novel fungal metabolite, apicidin [cyclo(N-O-methyl-L-tryptophanyl-L-isoleucinyl-D-pipecolinyl-L-2-amino-8-oxodecanoyl)], that exhibits potent, broad spectrum antiprotozoal activity in vitro against Apicomplexan parasites has been identified.  It is also orally and parenterally active in vivo against Plasmodium berghei malaria in mice.  Many Apicomplexan parasites cause serious, life-threatening human and animal diseases, such as malaria, cryptosporidiosis, toxoplasmosis, and coccidiosis, and new therapeutic agents are urgently needed.  Apicidin's antiparasitic activity appears to be due to low nanomolar inhibition of Apicomplexan histone deacetylase (HDA), which induces hyperacetylation of histones in treated parasites.  The acetylation-deacetylation of histones is thought to play a central role in transcriptional control in eukaryotic cells.  Other known HDA inhibitors were also evaluated and found to possess antiparasitic activity, suggesting that HDA is an attractive target for the development of novel antiparasitic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqub6fPyIvFQrVg90H21EOLACvtfcHk0ljazwEZL5hpPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmvV2qtrk%253D&md5=3738f88674b6e5856d55bab7a0701e22</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.23.13143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.23.13143%26sid%3Dliteratum%253Aachs%26aulast%3DDarkin-Rattray%26aufirst%3DS.%2BJ.%26aulast%3DGurnett%26aufirst%3DA.%2BM.%26aulast%3DMyers%26aufirst%3DR.%2BW.%26aulast%3DDulski%26aufirst%3DP.%2BM.%26aulast%3DCrumley%26aufirst%3DT.%2BM.%26aulast%3DAllocco%26aufirst%3DJ.%2BJ.%26aulast%3DCannova%26aufirst%3DC.%26aulast%3DMeinke%26aufirst%3DP.%2BT.%26aulast%3DColletti%26aufirst%3DS.%2BL.%26aulast%3DBednarek%26aufirst%3DM.%2BA.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DGoetz%26aufirst%3DM.%2BA.%26aulast%3DDombrowski%26aufirst%3DA.%2BW.%26aulast%3DPolishook%26aufirst%3DJ.%2BD.%26aulast%3DSchmatz%26aufirst%3DD.%2BM.%26atitle%3DApicidin%253A%2520a%2520novel%2520antiprotozoal%2520agent%2520that%2520inhibits%2520parasite%2520histone%2520deacetylase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D13143%26epage%3D13147%26doi%3D10.1073%2Fpnas.93.23.13143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Kijima, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horinouchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beppu, T.</span><span> </span><span class="NLM_article-title">Trapoxin, an antitumor cyclic tetrapeptide, is an irreversible inhibitor of mammalian histone deacetylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">268</span><span class="NLM_x">, </span> <span class="NLM_fpage">22429</span><span class="NLM_x">–</span> <span class="NLM_lpage">22435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=8226751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK3sXlsleks7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=22429-22435&author=M.+Kijimaauthor=M.+Yoshidaauthor=K.+Sugitaauthor=S.+Horinouchiauthor=T.+Beppu&title=Trapoxin%2C+an+antitumor+cyclic+tetrapeptide%2C+is+an+irreversible+inhibitor+of+mammalian+histone+deacetylase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Trapoxin, an antitumor cyclic tetrapeptide, is an irreversible inhibitor of mammalian histone deacetylase</span></div><div class="casAuthors">Kijima, Masako; Yoshida, Minoru; Sugita, Kenji; Horinouchi, Sueharu; Beppu, Teruhiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">22429-35</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Trapoxin A (I), is a fungal product that induces morphol. reversion from transformed to normal in sis-transformed NIH3T3 fibroblasts.  I was found to cause accumulation of highly acetylated core histones in a variety of mammalian cell lines.  In vitro expts. using partially purified mouse histone deacetylase showed that a low concn. of I irreversibly inhibited deacetylation of acetylated histone mols.  Chem. redn. of an epoxide group in I completely abolished the inhibitory activity, suggesting that I binds covalently to the histone deacetylase via the epoxide.  In contrast, inhibition by trichostatin A, a known potent inhibitor of histone deacetylase, was reversible.  Despite the different mode of inhibition, I and trichostatin A induced almost the same biol. effects on the cell cycle and differentiation.  These results strongly suggest that the in vivo effects commonly induced by these agents can be attributed to histone hyperacetylation resulting from the inhibition of histone deacetylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqscgc_V7MoWrVg90H21EOLACvtfcHk0ljazwEZL5hpPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsleks7Y%253D&md5=5252b79bf5604918fea6ecfdb64bc015</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKijima%26aufirst%3DM.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DSugita%26aufirst%3DK.%26aulast%3DHorinouchi%26aufirst%3DS.%26aulast%3DBeppu%26aufirst%3DT.%26atitle%3DTrapoxin%252C%2520an%2520antitumor%2520cyclic%2520tetrapeptide%252C%2520is%2520an%2520irreversible%2520inhibitor%2520of%2520mammalian%2520histone%2520deacetylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D22429%26epage%3D22435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Nakao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsunaga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindoh, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Soest, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fusetani, N.</span><span> </span><span class="NLM_article-title">Azumamides A-E: histone deacetylase inhibitory cyclic tetrapeptides from the marine sponge Mycale izuensis</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7553</span><span class="NLM_x">–</span> <span class="NLM_lpage">7557</span><span class="refDoi"> DOI: 10.1002/anie.200602047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fanie.200602047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7553-7557&author=Y.+Nakaoauthor=S.+Yoshidaauthor=S.+Matsunagaauthor=N.+Shindohauthor=Y.+Teradaauthor=K.+Nagaiauthor=J.+K.+Yamashitaauthor=A.+Ganesanauthor=R.+W.+van+Soestauthor=N.+Fusetani&title=Azumamides+A-E%3A+histone+deacetylase+inhibitory+cyclic+tetrapeptides+from+the+marine+sponge+Mycale+izuensis&doi=10.1002%2Fanie.200602047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.200602047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200602047%26sid%3Dliteratum%253Aachs%26aulast%3DNakao%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DS.%26aulast%3DShindoh%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DY.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DYamashita%26aufirst%3DJ.%2BK.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3Dvan%2BSoest%26aufirst%3DR.%2BW.%26aulast%3DFusetani%26aufirst%3DN.%26atitle%3DAzumamides%2520A-E%253A%2520histone%2520deacetylase%2520inhibitory%2520cyclic%2520tetrapeptides%2520from%2520the%2520marine%2520sponge%2520Mycale%2520izuensis%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D7553%26epage%3D7557%26doi%3D10.1002%2Fanie.200602047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Izzo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maulucci, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bifulco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Riccardis, F.</span><span> </span><span class="NLM_article-title">Total synthesis of azumamides A and E</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7557</span><span class="NLM_x">–</span> <span class="NLM_lpage">7560</span><span class="refDoi"> DOI: 10.1002/anie.200602033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fanie.200602033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1yqtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7557-7560&author=I.+Izzoauthor=N.+Maulucciauthor=G.+Bifulcoauthor=F.+De+Riccardis&title=Total+synthesis+of+azumamides+A+and+E&doi=10.1002%2Fanie.200602033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis of azumamides A and E</span></div><div class="casAuthors">Izzo, Irene; Maulucci, Nakia; Bifulco, Giuseppe; De Riccardis, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">7557-7560</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A zoom in on azumamides: An efficient synthetic route for two of the newly discovered marine natural products, namely azumamides A and E I (R1 = NH2, OH, resp.) , has been developed.  The present synthesis was followed by detn. of the stereochem. structure of the azumamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouhkxm-GSHPrVg90H21EOLACvtfcHk0lgDBzVoi3EH_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1yqtb3E&md5=2f2dfdc381d3e1de37531c6d1ed6695d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.200602033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200602033%26sid%3Dliteratum%253Aachs%26aulast%3DIzzo%26aufirst%3DI.%26aulast%3DMaulucci%26aufirst%3DN.%26aulast%3DBifulco%26aufirst%3DG.%26aulast%3DDe%2BRiccardis%26aufirst%3DF.%26atitle%3DTotal%2520synthesis%2520of%2520azumamides%2520A%2520and%2520E%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D7557%26epage%3D7560%26doi%3D10.1002%2Fanie.200602033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Wen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fusetani, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">Total synthesis of azumamide A and azumamide E, evaluation as histone deacetylase inhibitors, and design of a more potent analogue</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1105</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1021/ol070046y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol070046y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvFKru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=1105-1108&author=S.+Wenauthor=K.+L.+Careyauthor=Y.+Nakaoauthor=N.+Fusetaniauthor=G.+Packhamauthor=A.+Ganesan&title=Total+synthesis+of+azumamide+A+and+azumamide+E%2C+evaluation+as+histone+deacetylase+inhibitors%2C+and+design+of+a+more+potent+analogue&doi=10.1021%2Fol070046y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis of Azumamide A and Azumamide E, Evaluation as Histone Deacetylase Inhibitors, and Design of a More Potent Analog</span></div><div class="casAuthors">Wen, Shijun; Carey, Krystle L.; Nakao, Yoichi; Fusetani, Nobuhiro; Packham, Graham; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1105-1108</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The unprecedented diastereoselective Mannich reaction of a Z-allylsulfoximine was a key step in the total synthesis of the marine natural products azumamide A and E, and an unnatural analog.  Their relative potency as histone deacetylase inhibitors was evaluated and found to correlate with predicted zinc-binding affinity.  The stereoselective Mannich reaction of an unusual β,γ-unsatd. sulfinylimine is likely the most challenging transformation accomplished to date with the Ellman auxiliary, and enabled a concise route with only 15 steps in the longest linear sequence.  Within the azumamide series, histone deacetylase (HDAC) inhibitory activity is in the order carboxamide < carboxylic acid < hydroxamic acid, as predicted by the relative zinc binding of these compds.  The biol. activity of the azumamides is significantly weaker than FK228 (IC50 of 15 ± 9 nM in the same assay).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf820ZWbDDobVg90H21EOLACvtfcHk0lgDBzVoi3EH_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvFKru7Y%253D&md5=ff577d9a068da7ee5a3d16f1c7299525</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fol070046y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol070046y%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCarey%26aufirst%3DK.%2BL.%26aulast%3DNakao%26aufirst%3DY.%26aulast%3DFusetani%26aufirst%3DN.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DTotal%2520synthesis%2520of%2520azumamide%2520A%2520and%2520azumamide%2520E%252C%2520evaluation%2520as%2520histone%2520deacetylase%2520inhibitors%252C%2520and%2520design%2520of%2520a%2520more%2520potent%2520analogue%26jtitle%3DOrg.%2520Lett.%26date%3D2007%26volume%3D9%26spage%3D1105%26epage%3D1108%26doi%3D10.1021%2Fol070046y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Taori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luesch, H.</span><span> </span><span class="NLM_article-title">Structure and activity of largazole, a potent antiproliferative agent from the Floridian marine cyanobacterium Symploca sp</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">1806</span><span class="NLM_x">–</span> <span class="NLM_lpage">1807</span><span class="refDoi"> DOI: 10.1021/ja7110064</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja7110064" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFGhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=1806-1807&author=K.+Taoriauthor=V.+J.+Paulauthor=H.+Luesch&title=Structure+and+activity+of+largazole%2C+a+potent+antiproliferative+agent+from+the+Floridian+marine+cyanobacterium+Symploca+sp&doi=10.1021%2Fja7110064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Activity of Largazole, a Potent Antiproliferative Agent from the Floridian Marine Cyanobacterium Symploca sp.</span></div><div class="casAuthors">Taori, Kanchan; Paul, Valerie J.; Luesch, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1806-1807</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel cytotoxic cyclodepsipeptide, termed largazole, has been isolated from the marine cyanobacterium Symploca sp. collected in the Florida Keys.  Its planar structure was elucidated by 1D and 2D NMR spectroscopy in conjunction with mass spectrometry.  The abs. configuration of largazole was detd. by chem. degrdn. followed by chiral HPLC anal.  Largazole possesses densely assembled unusual structural features, including a rare 4-methylthiazoline linearly fused to a thiazole in its cyclic core and a hitherto undescribed 3-hydroxy-7-mercaptohept-4-enoic acid unit incorporated in an ester, thioester, and amide framework.  Largazole exhibits potent antiproliferative activity and preferentially targets cancer cells over nontransformed cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0JtehDfpXXLVg90H21EOLACvtfcHk0lgDBzVoi3EH_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFGhtw%253D%253D&md5=bcd18bbee431e273c1b646f3c9c1388b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fja7110064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja7110064%26sid%3Dliteratum%253Aachs%26aulast%3DTaori%26aufirst%3DK.%26aulast%3DPaul%26aufirst%3DV.%2BJ.%26aulast%3DLuesch%26aufirst%3DH.%26atitle%3DStructure%2520and%2520activity%2520of%2520largazole%252C%2520a%2520potent%2520antiproliferative%2520agent%2520from%2520the%2520Floridian%2520marine%2520cyanobacterium%2520Symploca%2520sp%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D1806%26epage%3D1807%26doi%3D10.1021%2Fja7110064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emiliani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breslow, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rifkind, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, P. A.</span><span> </span><span class="NLM_article-title">A class of hybrid polar inducers of transformed cell differentiation inhibits histone deacetylases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">3003</span><span class="NLM_x">–</span> <span class="NLM_lpage">3007</span><span class="refDoi"> DOI: 10.1073/pnas.95.6.3003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.95.6.3003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=9501205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK1cXitV2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=3003-3007&author=V.+M.+Richonauthor=S.+Emilianiauthor=E.+Verdinauthor=Y.+Webbauthor=R.+Breslowauthor=R.+A.+Rifkindauthor=P.+A.+Marks&title=A+class+of+hybrid+polar+inducers+of+transformed+cell+differentiation+inhibits+histone+deacetylases&doi=10.1073%2Fpnas.95.6.3003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A class of hybrid polar inducers of transformed cell differentiation inhibits histone deacetylases</span></div><div class="casAuthors">Richon, Vicotria M.; Emiliani, Stephane; Verdin, Eric; Webb, Yael; Breslow, Ronald; Rifkind, Richard A.; Marks, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3003-3007</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hybrid polar compds. (HPCs) have been synthesized that induce terminal differentiation and/or apoptosis in various transformed cells.  We have previously reported on the development of the second-generation HPCs suberoylanilide hydroxamic acid (SAHA) and m-carboxycinnamic acid bishydroxamide (CBHA) that are 2,000-fold more potent inducers on a molar basis than the prototype HPC hexamethylene bisacetamide (HMBA).  Herein we report that CBHA and SAHA inhibit histone deacetylase 1 (HDAC1) and histone deacetylase 3 (HDAC3) activity in vitro.  Treatment of cells in culture with SAHA results in a marked hyperacetylation of histone H4, but culture with HMBA does not.  Murine erythroleukemia cells developed for resistance to SAHA are cross-resistant to trichostatin A, a known deacetylase inhibitor and differentiation inducer, but are not cross-resistant to HMBA.  These studies show that the second-generation HPCs, unlike HMBA, are potent inhibitors of HDAC activity.  In this sense, HMBA and the second-generation HPCs appear to induce differentiation by different pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA08MImdI837Vg90H21EOLACvtfcHk0lisnhKMQNh-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXitV2itro%253D&md5=1e821e03d7c9522cbf3084c6f659fd72</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.6.3003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.6.3003%26sid%3Dliteratum%253Aachs%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DEmiliani%26aufirst%3DS.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DWebb%26aufirst%3DY.%26aulast%3DBreslow%26aufirst%3DR.%26aulast%3DRifkind%26aufirst%3DR.%2BA.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26atitle%3DA%2520class%2520of%2520hybrid%2520polar%2520inducers%2520of%2520transformed%2520cell%2520differentiation%2520inhibits%2520histone%2520deacetylases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D3003%26epage%3D3007%26doi%3D10.1073%2Fpnas.95.6.3003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Miller, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witter, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvedere, S.</span><span> </span><span class="NLM_article-title">Histone deacetylase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">5097</span><span class="NLM_x">–</span> <span class="NLM_lpage">5116</span><span class="refDoi"> DOI: 10.1021/jm0303094</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0303094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5097-5116&author=T.+A.+Millerauthor=D.+J.+Witterauthor=S.+Belvedere&title=Histone+deacetylase+inhibitors&doi=10.1021%2Fjm0303094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors</span></div><div class="casAuthors">Miller, Thomas A.; Witter, David J.; Belvedere, Sandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5097-5116</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the medicinal chem. and structure-activity relationships underlying advances in histone deacetylase (HDAC) class I/II inhibitor discovery, design, and optimization.  HDAC structural studies and HDAC inhibitory activity of small-mol. hydroxamic acids, carboxylates, benzamides, electrophilic ketones, and cyclic peptides are described.  Miscellaneousness and selectivity of the class I/II HDACs are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU5UlOsBUF7rVg90H21EOLACvtfcHk0lisnhKMQNh-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D&md5=d2d0a95d87d699334c1630df5deda38a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm0303094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303094%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DT.%2BA.%26aulast%3DWitter%26aufirst%3DD.%2BJ.%26aulast%3DBelvedere%26aufirst%3DS.%26atitle%3DHistone%2520deacetylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5097%26epage%3D5116%26doi%3D10.1021%2Fjm0303094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirisawad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buggy, J. J.</span><span> </span><span class="NLM_article-title">A novel histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 induces apoptosis in T-cell lymphomas</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1038/leu.2008.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fleu.2008.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=18256683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1026-1034&author=S.+Balasubramanianauthor=J.+Ramosauthor=W.+Luoauthor=M.+Sirisawadauthor=E.+Vernerauthor=J.+J.+Buggy&title=A+novel+histone+deacetylase+8+%28HDAC8%29-specific+inhibitor+PCI-34051+induces+apoptosis+in+T-cell+lymphomas&doi=10.1038%2Fleu.2008.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A novel histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 induces apoptosis in T-cell lymphomas</span></div><div class="casAuthors">Balasubramanian, S.; Ramos, J.; Luo, W.; Sirisawad, M.; Verner, E.; Buggy, J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1026-1034</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We have developed a potent, histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 with >200-fold selectivity over the other HDAC isoforms.  PCI-34051 induces caspase-dependent apoptosis in cell lines derived from T-cell lymphomas or leukemias, but not in other hematopoietic or solid tumor lines.  Unlike broad-spectrum HDAC inhibitors, PCI-34051 does not cause detectable histone or tubulin acetylation.  Cells defective in T-cell receptor signaling were still sensitive to PCI-34051-induced apoptosis, whereas a phospholipase C-γ1 (PLCγ1)-defective line was resistant.  Jurkat cells showed a dose-dependent decrease in PCI-34051-induced apoptosis upon treatment with a PLC inhibitor U73122, but not with an inactive analog.  We found that rapid intracellular calcium mobilization from endoplasmic reticulum (ER) and later cytochrome c release from mitochondria are essential for the apoptotic mechanism.  The rapid Ca2+ flux was dependent on PCI-34051 concn., and was blocked by the PLC inhibitor U73122.  Further, apoptosis was blocked by Ca2+ chelators (BAPTA) and enhanced by Ca2+ effectors (thapsigargin), supporting this model.  These studies show that HDAC8-selective inhibitors have a unique mechanism of action involving PLCγ1 activation and calcium-induced apoptosis, and could offer benefits including a greater therapeutic index for treating T-cell malignancies.  Leukemia (2008) 22, 1026-1034; doi:10.1038/leu.2008.9; published online 7 Feb. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW-VB4Wb3h1bVg90H21EOLACvtfcHk0lisnhKMQNh-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D&md5=03446a05b396ea9497e4f6f9254cae03</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.9%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DRamos%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSirisawad%26aufirst%3DM.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26atitle%3DA%2520novel%2520histone%2520deacetylase%25208%2520%2528HDAC8%2529-specific%2520inhibitor%2520PCI-34051%2520induces%2520apoptosis%2520in%2520T-cell%2520lymphomas%26jtitle%3DLeukemia%26date%3D2008%26volume%3D22%26spage%3D1026%26epage%3D1034%26doi%3D10.1038%2Fleu.2008.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasuya, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asamitsu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of highly selective and potent histone deacetylase 3 inhibitors using click chemistry-based combinatorial fragment assembly</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e68669</span><span class="refDoi"> DOI: 10.1371/journal.pone.0068669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.pone.0068669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=23874714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SgsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e68669&author=T.+Suzukiauthor=Y.+Kasuyaauthor=Y.+Itohauthor=Y.+Otaauthor=P.+Zhanauthor=K.+Asamitsuauthor=H.+Nakagawaauthor=T.+Okamotoauthor=N.+Miyata&title=Identification+of+highly+selective+and+potent+histone+deacetylase+3+inhibitors+using+click+chemistry-based+combinatorial+fragment+assembly&doi=10.1371%2Fjournal.pone.0068669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of highly selective and potent histone deacetylase 3 inhibitors using click chemistry-based combinatorial fragment assembly</span></div><div class="casAuthors">Suzuki, Takayoshi; Kasuya, Yuki; Itoh, Yukihiro; Ota, Yosuke; Zhan, Peng; Asamitsu, Kaori; Nakagawa, Hidehiko; Okamoto, Takashi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e68669</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">To find histone deacetylase 3 (HDAC3)-selective inhibitors, a series of 504 candidates was assembled using "click chem.", by reacting nine alkynes bearing a zinc-binding group with 56 azide building blocks in the presence of Cu(I) catalyst.  Screening of the 504-member triazole library against HDAC3 and other HDAC isoenzymes led to the identification of potent and selective HDAC3 inhibitors T247 and T326.  These compds. showed potent HDAC3 inhibition with submicromolar IC50s, whereas they did not strongly inhibit other isoenzymes.  Compds. T247 and T326 also induced a dose-dependent selective increase of NF-κB acetylation in human colon cancer HCT116 cells, indicating selective inhibition of HDAC3 in the cells.  In addn., these HDAC3-selective inhibitors induced growth inhibition of cancer cells, and activated HIV gene expression in latent HIV-infected cells.  These findings indicate that HDAC3-selective inhibitors are promising candidates for anticancer drugs and antiviral agents.  This work also suggests the usefulness of the click chem. approach to find isoenzyme-selective HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4CvS43ILxQLVg90H21EOLACvtfcHk0lisnhKMQNh-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SgsrjF&md5=74d12c8fb1f06f8e00691415434fd929</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0068669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0068669%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DKasuya%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DAsamitsu%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520highly%2520selective%2520and%2520potent%2520histone%2520deacetylase%25203%2520inhibitors%2520using%2520click%2520chemistry-based%2520combinatorial%2520fragment%2520assembly%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De68669%26doi%3D10.1371%2Fjournal.pone.0068669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brochier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langley, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozikowski, A. P.</span><span> </span><span class="NLM_article-title">Rational design and simple chemistry yield a superior, neuroprotective HDAC6 inhibitor, tubastatin A</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">10842</span><span class="NLM_x">–</span> <span class="NLM_lpage">10846</span><span class="refDoi"> DOI: 10.1021/ja102758v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja102758v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=10842-10846&author=K.+V.+Butlerauthor=J.+Kalinauthor=C.+Brochierauthor=G.+Vistoliauthor=B.+Langleyauthor=A.+P.+Kozikowski&title=Rational+design+and+simple+chemistry+yield+a+superior%2C+neuroprotective+HDAC6+inhibitor%2C+tubastatin+A&doi=10.1021%2Fja102758v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design and Simple Chemistry Yield a Superior, Neuroprotective HDAC6 Inhibitor, Tubastatin A</span></div><div class="casAuthors">Butler, Kyle V.; Kalin, Jay; Brochier, Camille; Vistoli, Guilio; Langley, Brett; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10842-10846</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design combined with homol. modeling techniques were used to develop potent inhibitors of HDAC6 that display superior selectivity for the HDAC6 isoenzyme compared to other inhibitors.  These inhibitors can be assembled in a few synthetic steps, and thus are readily scaled up for in vivo studies.  An optimized compd. from this series, designated Tubastatin A, was tested in primary cortical neuron cultures in which it was found to induce elevated levels of acetylated α-tubulin, but not histone, consistent with its HDAC6 selectivity.  Tubastatin A also conferred dose-dependent protection in primary cortical neuron cultures against glutathione depletion-induced oxidative stress.  Importantly, when given alone at all concns. tested, this hydroxamate-contg. HDAC6-selective compd. displayed no neuronal toxicity, thus, forecasting the potential application of this agent and its analogs to neurodegenerative conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruEaJPWQfjH7Vg90H21EOLACvtfcHk0lhmDmolhkU_Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D&md5=f0cc68bb68f94ad1ef48f88cc6b57129</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja102758v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja102758v%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKalin%26aufirst%3DJ.%26aulast%3DBrochier%26aufirst%3DC.%26aulast%3DVistoli%26aufirst%3DG.%26aulast%3DLangley%26aufirst%3DB.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DRational%2520design%2520and%2520simple%2520chemistry%2520yield%2520a%2520superior%252C%2520neuroprotective%2520HDAC6%2520inhibitor%252C%2520tubastatin%2520A%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D10842%26epage%3D10846%26doi%3D10.1021%2Fja102758v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span> </span><span class="NLM_article-title">Discovery of selective histone deacetylase 6 inhibitors using the quinazoline as the cap for the treatment of cancer</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01342</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01342" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Z.+Yangauthor=T.+Wangauthor=F.+Wangauthor=T.+Niuauthor=Z.+Liuauthor=X.+Chenauthor=C.+Longauthor=M.+Tangauthor=D.+Caoauthor=X.+Wangauthor=W.+Xiangauthor=Y.+Yiauthor=L.+Maauthor=J.+Youauthor=L.+Chen&title=Discovery+of+selective+histone+deacetylase+6+inhibitors+using+the+quinazoline+as+the+cap+for+the+treatment+of+cancer&doi=10.1021%2Facs.jmedchem.5b01342"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01342%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DNiu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLong%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DYi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%2520using%2520the%2520quinazoline%2520as%2520the%2520cap%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26doi%3D10.1021%2Facs.jmedchem.5b01342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Kozikowski, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapadar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luchini, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Billadeau, D. D.</span><span> </span><span class="NLM_article-title">Use of the nitrile oxide cycloaddition (NOC) reaction for molecular probe generation: a new class of enzyme selective histone deacetylase inhibitors (HDACIs) showing picomolar activity at HDAC6</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4370</span><span class="NLM_x">–</span> <span class="NLM_lpage">4373</span><span class="refDoi"> DOI: 10.1021/jm8002894</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8002894" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovVGmtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4370-4373&author=A.+P.+Kozikowskiauthor=S.+Tapadarauthor=D.+N.+Luchiniauthor=K.+H.+Kimauthor=D.+D.+Billadeau&title=Use+of+the+nitrile+oxide+cycloaddition+%28NOC%29+reaction+for+molecular+probe+generation%3A+a+new+class+of+enzyme+selective+histone+deacetylase+inhibitors+%28HDACIs%29+showing+picomolar+activity+at+HDAC6&doi=10.1021%2Fjm8002894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the Nitrile Oxide Cycloaddition (NOC) Reaction for Molecular Probe Generation: A New Class of Enzyme Selective Histone Deacetylase Inhibitors (HDACIs) Showing Picomolar Activity at HDAC6</span></div><div class="casAuthors">Kozikowski, Alan P.; Tapadar, Subhasish; Luchini, Doris N.; Kim, Ki Hwan; Billadeau, Daniel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4370-4373</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of hydroxamate based HDAC inhibitors contg. a phenylisoxazole as the CAP group has been synthesized using nitrile oxide cycloaddn. chem.  An HDAC6 selective inhibitor having a potency of ∼2 picomolar was identified.  Some of the compds. were examd. for their ability to block pancreatic cancer cell growth and found to be about 10-fold more potent than SAHA (suberoylanilidehydroxamic acid).  This research provides valuable, new mol. probes for use in exploring HDAC biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ7Ca9tXdBvbVg90H21EOLACvtfcHk0lhmDmolhkU_Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovVGmtbk%253D&md5=006c71fe6f2bb09c2c65196e134a695e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm8002894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8002894%26sid%3Dliteratum%253Aachs%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DTapadar%26aufirst%3DS.%26aulast%3DLuchini%26aufirst%3DD.%2BN.%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DBilladeau%26aufirst%3DD.%2BD.%26atitle%3DUse%2520of%2520the%2520nitrile%2520oxide%2520cycloaddition%2520%2528NOC%2529%2520reaction%2520for%2520molecular%2520probe%2520generation%253A%2520a%2520new%2520class%2520of%2520enzyme%2520selective%2520histone%2520deacetylase%2520inhibitors%2520%2528HDACIs%2529%2520showing%2520picomolar%2520activity%2520at%2520HDAC6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4370%26epage%3D4373%26doi%3D10.1021%2Fjm8002894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Mwakwari, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guerrant, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oyelere, A. K.</span><span> </span><span class="NLM_article-title">Macrocyclic histone deacetylase inhibitors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1423</span><span class="NLM_x">–</span> <span class="NLM_lpage">1440</span><span class="refDoi"> DOI: 10.2174/156802610792232079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.2174%2F156802610792232079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=20536416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=1423-1440&author=S.+C.+Mwakwariauthor=V.+Patilauthor=W.+Guerrantauthor=A.+K.+Oyelere&title=Macrocyclic+histone+deacetylase+inhibitors&doi=10.2174%2F156802610792232079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic histone deacetylase inhibitors</span></div><div class="casAuthors">Mwakwari, Sandra C.; Patil, Vishal; Guerrant, William; Oyelere, Adegboyega K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1423-1440</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase inhibitors (HDACi) are an emerging class of novel anti-cancer drugs that cause growth arrest, differentiation, and apoptosis of tumor cells.  In addn., they have shown promise as anti-parasitic, anti-neurodegenerative, anti-rheumatol. and immunosuppressant agents.  To date, several structurally distinct small mol. HDACi have been reported including aryl hydroxamates, benzamides, short-chain fatty acids, electrophilic ketones, and macrocyclic peptides.  Macrocyclic HDACi possess the most complex cap-groups which interact with HDAC enzyme's outer rim and have demonstrated excellent HDAC inhibition potency and isoform selectivity.  This review focuses on the recent progress and current state of macrocyclic HDACi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXzgRmfQBx27Vg90H21EOLACvtfcHk0ljo8nCvqjyHHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrF&md5=d027b0380e04f364fb0985a350e99c60</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F156802610792232079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610792232079%26sid%3Dliteratum%253Aachs%26aulast%3DMwakwari%26aufirst%3DS.%2BC.%26aulast%3DPatil%26aufirst%3DV.%26aulast%3DGuerrant%26aufirst%3DW.%26aulast%3DOyelere%26aufirst%3DA.%2BK.%26atitle%3DMacrocyclic%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D1423%26epage%3D1440%26doi%3D10.2174%2F156802610792232079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Cole, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boone, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christianson, D. W.</span><span> </span><span class="NLM_article-title">Structural basis of the antiproliferative activity of largazole, a depsipeptide inhibitor of the histone deacetylases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">12474</span><span class="NLM_x">–</span> <span class="NLM_lpage">12477</span><span class="refDoi"> DOI: 10.1021/ja205972n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja205972n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=12474-12477&author=K.+E.+Coleauthor=D.+P.+Dowlingauthor=M.+A.+Booneauthor=A.+J.+Phillipsauthor=D.+W.+Christianson&title=Structural+basis+of+the+antiproliferative+activity+of+largazole%2C+a+depsipeptide+inhibitor+of+the+histone+deacetylases&doi=10.1021%2Fja205972n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of the Antiproliferative Activity of Largazole, a Depsipeptide Inhibitor of the Histone Deacetylases</span></div><div class="casAuthors">Cole, Kathryn E.; Dowling, Daniel P.; Boone, Matthew A.; Phillips, Andrew J.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12474-12477</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Largazole is a macrocyclic depsipeptide originally isolated from the marine cyanobacterium Symploca sp., which is indigenous to the warm, blue-green waters of Key Largo, Florida (whence largazole derives its name).  Largazole contains an unusual thiazoline-thiazole ring system that rigidifies its macrocyclic skeleton, and it also contains a lipophilic thioester side chain.  Hydrolysis of the thioester in vivo yields largazole thiol, which exhibits remarkable antiproliferative effects and is believed to be the most potent inhibitor of the metal-dependent histone deacetylases (HDACs).  Here, the 2.14 Å-resoln. crystal structure of the HDAC8-largazole thiol complex is the first of an HDAC complexed with a macrocyclic inhibitor and reveals that ideal thiolate-zinc coordination geometry is the key chem. feature responsible for its exceptional affinity and biol. activity.  Notably, the core structure of largazole is conserved in romidepsin, a depsipeptide natural product formulated as the drug Istodax recently approved for cancer chemotherapy.  Accordingly, the structure of the HDAC8-largazole thiol complex is the first to illustrate the mode of action of a new class of therapeutically important HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-DXxueVrU6bVg90H21EOLACvtfcHk0ljo8nCvqjyHHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D&md5=9d05a3f1fd412f119ced61aaa771ef28</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja205972n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja205972n%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DK.%2BE.%26aulast%3DDowling%26aufirst%3DD.%2BP.%26aulast%3DBoone%26aufirst%3DM.%2BA.%26aulast%3DPhillips%26aufirst%3DA.%2BJ.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructural%2520basis%2520of%2520the%2520antiproliferative%2520activity%2520of%2520largazole%252C%2520a%2520depsipeptide%2520inhibitor%2520of%2520the%2520histone%2520deacetylases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D12474%26epage%3D12477%26doi%3D10.1021%2Fja205972n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Marek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moraes Mourao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caby, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stolfa, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidtkunz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancelot, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrade, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renaud, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavarelli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romier, C.</span><span> </span><span class="NLM_article-title">Structural basis for the inhibition of histone deacetylase 8 (HDAC8), a key epigenetic player in the blood fluke Schistosoma mansoni</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003645</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1003645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.ppat.1003645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=24086136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOnurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003645&author=M.+Marekauthor=S.+Kannanauthor=A.+T.+Hauserauthor=M.+Moraes+Mouraoauthor=S.+Cabyauthor=V.+Curaauthor=D.+A.+Stolfaauthor=K.+Schmidtkunzauthor=J.+Lancelotauthor=L.+Andradeauthor=J.+P.+Renaudauthor=G.+Oliveiraauthor=W.+Sipplauthor=M.+Jungauthor=J.+Cavarelliauthor=R.+J.+Pierceauthor=C.+Romier&title=Structural+basis+for+the+inhibition+of+histone+deacetylase+8+%28HDAC8%29%2C+a+key+epigenetic+player+in+the+blood+fluke+Schistosoma+mansoni&doi=10.1371%2Fjournal.ppat.1003645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the inhibition of histone deacetylase 8 (HDAC8), a key epigenetic player in the blood fluke Schistosoma mansoni</span></div><div class="casAuthors">Marek, Martin; Kannan, Srinivasaraghavan; Hauser, Alexander-Thomas; Mourao, Marina Moraes; Caby, Stephanie; Cura, Vincent; Stolfa, Diana A.; Schmidtkunz, Karin; Lancelot, Julien; Andrade, Luiza; Renaud, Jean-Paul; Oliveira, Guilherme; Sippl, Wolfgang; Jung, Manfred; Cavarelli, Jean; Pierce, Raymond J.; Romier, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1003645</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The treatment of schistosomiasis, a disease caused by blood flukes parasites of the Schistosoma genus, depends on the intensive use of a single drug, praziquantel, which increases the likelihood of the development of drug-resistant parasite strains and renders the search for new drugs a strategic priority.  Currently, inhibitors of human epigenetic enzymes are actively investigated as novel anti-cancer drugs and have the potential to be used as new anti-parasitic agents.  Here, we report that Schistosoma mansoni histone deacetylase 8 (smHDAC8), the most expressed class I HDAC isotype in this organism, is a functional acetyl--lysine deacetylase that plays an important role in parasite infectivity.  The crystal structure of smHDAC8 shows that this enzyme adopts a canonical α/β HDAC fold, with specific solvent exposed loops corresponding to insertions in the schistosome HDAC8 sequence.  Importantly, structures of smHDAC8 in complex with generic HDAC inhibitors revealed specific structural changes in the smHDAC8 active site that cannot be accommodated by human HDACs.  Using a structure-based approach, we identified several small-mol. inhibitors that build on these specificities.  These mols. exhibit an inhibitory effect on smHDAC8 but show reduced affinity for human HDACs.  Crucially, we show that a newly identified smHDAC8 inhibitor has the capacity to induce apoptosis and mortality in schistosomes.  Taken together, our biol. and structural findings define the framework for the rational design of small-mol. inhibitors specifically interfering with schistosome epigenetic mechanisms, and further support an anti-parasitic epigenome targeting strategy to treat neglected diseases caused by eukaryotic pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptIGu8lR_TxrVg90H21EOLACvtfcHk0ljo8nCvqjyHHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOnurzI&md5=9fef2938ff73df80fc60b739c878f9a6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003645%26sid%3Dliteratum%253Aachs%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DMoraes%2BMourao%26aufirst%3DM.%26aulast%3DCaby%26aufirst%3DS.%26aulast%3DCura%26aufirst%3DV.%26aulast%3DStolfa%26aufirst%3DD.%2BA.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DLancelot%26aufirst%3DJ.%26aulast%3DAndrade%26aufirst%3DL.%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DOliveira%26aufirst%3DG.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DCavarelli%26aufirst%3DJ.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520histone%2520deacetylase%25208%2520%2528HDAC8%2529%252C%2520a%2520key%2520epigenetic%2520player%2520in%2520the%2520blood%2520fluke%2520Schistosoma%2520mansoni%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26spage%3De1003645%26doi%3D10.1371%2Fjournal.ppat.1003645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Marcaurelle, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dandapani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duvall, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kesavan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. D. t.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marie, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulrooney, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandya, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryba, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suh, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akella, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fass, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reis, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, M. A.</span><span> </span><span class="NLM_article-title">An aldol-based build/couple/pair strategy for the synthesis of medium- and large-sized rings: discovery of macrocyclic histone deacetylase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">16962</span><span class="NLM_x">–</span> <span class="NLM_lpage">16976</span><span class="refDoi"> DOI: 10.1021/ja105119r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105119r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCrsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=16962-16976&author=L.+A.+Marcaurelleauthor=E.+Comerauthor=S.+Dandapaniauthor=J.+R.+Duvallauthor=B.+Gerardauthor=S.+Kesavanauthor=M.+D.+t.+Leeauthor=H.+Liuauthor=J.+T.+Loweauthor=J.+C.+Marieauthor=C.+A.+Mulrooneyauthor=B.+A.+Pandyaauthor=A.+Rowleyauthor=T.+D.+Rybaauthor=B.+C.+Suhauthor=J.+Weiauthor=D.+W.+Youngauthor=L.+B.+Akellaauthor=N.+T.+Rossauthor=Y.+L.+Zhangauthor=D.+M.+Fassauthor=S.+A.+Reisauthor=W.+N.+Zhaoauthor=S.+J.+Haggartyauthor=M.+Palmerauthor=M.+A.+Foley&title=An+aldol-based+build%2Fcouple%2Fpair+strategy+for+the+synthesis+of+medium-+and+large-sized+rings%3A+discovery+of+macrocyclic+histone+deacetylase+inhibitors&doi=10.1021%2Fja105119r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An Aldol-Based Build/Couple/Pair Strategy for the Synthesis of Medium- and Large-Sized Rings: Discovery of Macrocyclic Histone Deacetylase Inhibitors</span></div><div class="casAuthors">Marcaurelle, Lisa A.; Comer, Eamon; Dandapani, Sivaraman; Duvall, Jeremy R.; Gerard, Baudouin; Kesavan, Sarathy; Lee, Maurice D.; Liu, Haibo; Lowe, Jason T.; Marie, Jean-Charles; Mulrooney, Carol A.; Pandya, Bhaumik A.; Rowley, Ann; Ryba, Troy D.; Suh, Byung-Chul; Wei, Jingqiang; Young, Damian W.; Akella, Lakshmi B.; Ross, Nathan T.; Zhang, Yan-Ling; Fass, Daniel M.; Reis, Surya A.; Zhao, Wen-Ning; Haggarty, Stephen J.; Palmer, Michelle; Foley, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">16962-16976</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nonracemic macrocycles such as I (PMB = 4-MeOC6H4CH2; Boc = tert-butoxycarbonyl), II, and III were prepd. using diastereoselective aldol addn. reactions followed by coupling with nonracemic protected alaninols and subsequent nucleophilic arom. substitution, regioselective azide-alkyne cycloaddn., and ring-closing metathesis reactions to generate a set of macrocycles of differing ring sizes and stereochemistries.  Asym. syn- and anti-aldol reactions were performed to produce four stereoisomers of a Boc-protected γ-amino acid; prepn. of both enantiomers of O-PMB-protected alaninol followed by coupling and amide redn. yielded a set of eight substituted secondary amines.  Acylation of the secondary amines with nitrofluorophenyl acyl chlorides followed by desilylation and intramol. nucleophilic arom. substitution yielded macrocycles such as I.  Acylation of the secondary amines with 4-azidobutanoic acid, desilylation, O-alkylation with propargyl bromides, and ruthenium- or copper-catalyzed azide-alkyne cycloaddns. provided triazole-contg. macrocycles such as II regioselectively.  Acylation of the secondary amines with a nonracemic pentenyloxybenzoyl chloride, desilylation, O-alkylation with allyl bromide, and ring-closing metathesis yielded macrocycles which can be reduced and further reacted to give macrocycles such as III.  Redn. of the alkene and nitro groups of ring-closing metathesis-derived macrocycles followed by Fmoc protection, Boc deprotection, and reprotection with allyloxycarbonyl chloride yields a set of sixteen stereoisomeric macrocycles which are use to generate a 14 400-membered library of macrolactams.  Screening of the macrocycle library led to the discovery of a novel class of histone deacetylase inhibitors, which display mixed enzyme inhibition, and led to increased levels of acetylation in a primary mouse neuron culture.  Development of stereochem. structure/activity relationships was made possible by screening all 16 stereoisomers of the macrolactams produced through the aldol-based build/couple/pair strategy.  [This abstr. record is one of 3 records for this document necessitated by the large no. of index entries required to fully index the document and publication system constraints.].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU_CeBimobNLVg90H21EOLACvtfcHk0lhD60w0ijCgEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCrsbzN&md5=e1dea4e0e2accf8451f2b5e712a61612</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja105119r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105119r%26sid%3Dliteratum%253Aachs%26aulast%3DMarcaurelle%26aufirst%3DL.%2BA.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DGerard%26aufirst%3DB.%26aulast%3DKesavan%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BD.%2Bt.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLowe%26aufirst%3DJ.%2BT.%26aulast%3DMarie%26aufirst%3DJ.%2BC.%26aulast%3DMulrooney%26aufirst%3DC.%2BA.%26aulast%3DPandya%26aufirst%3DB.%2BA.%26aulast%3DRowley%26aufirst%3DA.%26aulast%3DRyba%26aufirst%3DT.%2BD.%26aulast%3DSuh%26aufirst%3DB.%2BC.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DYoung%26aufirst%3DD.%2BW.%26aulast%3DAkella%26aufirst%3DL.%2BB.%26aulast%3DRoss%26aufirst%3DN.%2BT.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DFass%26aufirst%3DD.%2BM.%26aulast%3DReis%26aufirst%3DS.%2BA.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26atitle%3DAn%2520aldol-based%2520build%252Fcouple%252Fpair%2520strategy%2520for%2520the%2520synthesis%2520of%2520medium-%2520and%2520large-sized%2520rings%253A%2520discovery%2520of%2520macrocyclic%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D16962%26epage%3D16976%26doi%3D10.1021%2Fja105119r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Vaidya, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neelarapu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madriaga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendonca, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdelkarim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Breemen, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blond, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petukhov, P. A.</span><span> </span><span class="NLM_article-title">Novel histone deacetylase 8 ligands without a zinc chelating group: exploring an ’upside-down’ binding pose</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">6621</span><span class="NLM_x">–</span> <span class="NLM_lpage">6627</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.08.104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.bmcl.2012.08.104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=6621-6627&author=A.+S.+Vaidyaauthor=R.+Neelarapuauthor=A.+Madriagaauthor=H.+Baiauthor=E.+Mendoncaauthor=H.+Abdelkarimauthor=R.+B.+van+Breemenauthor=S.+Y.+Blondauthor=P.+A.+Petukhov&title=Novel+histone+deacetylase+8+ligands+without+a+zinc+chelating+group%3A+exploring+an+%E2%80%99upside-down%E2%80%99+binding+pose&doi=10.1016%2Fj.bmcl.2012.08.104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.08.104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.08.104%26sid%3Dliteratum%253Aachs%26aulast%3DVaidya%26aufirst%3DA.%2BS.%26aulast%3DNeelarapu%26aufirst%3DR.%26aulast%3DMadriaga%26aufirst%3DA.%26aulast%3DBai%26aufirst%3DH.%26aulast%3DMendonca%26aufirst%3DE.%26aulast%3DAbdelkarim%26aufirst%3DH.%26aulast%3Dvan%2BBreemen%26aufirst%3DR.%2BB.%26aulast%3DBlond%26aufirst%3DS.%2BY.%26aulast%3DPetukhov%26aufirst%3DP.%2BA.%26atitle%3DNovel%2520histone%2520deacetylase%25208%2520ligands%2520without%2520a%2520zinc%2520chelating%2520group%253A%2520exploring%2520an%2520%25E2%2580%2599upside-down%25E2%2580%2599%2520binding%2520pose%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D6621%26epage%3D6627%26doi%3D10.1016%2Fj.bmcl.2012.08.104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Vickers, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leman, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghadiri, M. R.</span><span> </span><span class="NLM_article-title">Discovery of HDAC inhibitors that lack an active site Zn(2+)-binding functional group</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">508</span><span class="refDoi"> DOI: 10.1021/ml300081u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300081u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=505-508&author=C.+J.+Vickersauthor=C.+A.+Olsenauthor=L.+J.+Lemanauthor=M.+R.+Ghadiri&title=Discovery+of+HDAC+inhibitors+that+lack+an+active+site+Zn%282%2B%29-binding+functional+group&doi=10.1021%2Fml300081u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml300081u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300081u%26sid%3Dliteratum%253Aachs%26aulast%3DVickers%26aufirst%3DC.%2BJ.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26aulast%3DLeman%26aufirst%3DL.%2BJ.%26aulast%3DGhadiri%26aufirst%3DM.%2BR.%26atitle%3DDiscovery%2520of%2520HDAC%2520inhibitors%2520that%2520lack%2520an%2520active%2520site%2520Zn%25282%252B%2529-binding%2520functional%2520group%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D505%26epage%3D508%26doi%3D10.1021%2Fml300081u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Cole, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boone, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christianson, D. W.</span><span> </span><span class="NLM_article-title">Structural basis of the antiproliferative activity of largazole, a depsipeptide inhibitor of the histone deacetylases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">12474</span><span class="NLM_x">–</span> <span class="NLM_lpage">12477</span><span class="refDoi"> DOI: 10.1021/ja205972n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja205972n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=12474-12477&author=K.+E.+Coleauthor=D.+P.+Dowlingauthor=M.+A.+Booneauthor=A.+J.+Phillipsauthor=D.+W.+Christianson&title=Structural+basis+of+the+antiproliferative+activity+of+largazole%2C+a+depsipeptide+inhibitor+of+the+histone+deacetylases&doi=10.1021%2Fja205972n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of the Antiproliferative Activity of Largazole, a Depsipeptide Inhibitor of the Histone Deacetylases</span></div><div class="casAuthors">Cole, Kathryn E.; Dowling, Daniel P.; Boone, Matthew A.; Phillips, Andrew J.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12474-12477</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Largazole is a macrocyclic depsipeptide originally isolated from the marine cyanobacterium Symploca sp., which is indigenous to the warm, blue-green waters of Key Largo, Florida (whence largazole derives its name).  Largazole contains an unusual thiazoline-thiazole ring system that rigidifies its macrocyclic skeleton, and it also contains a lipophilic thioester side chain.  Hydrolysis of the thioester in vivo yields largazole thiol, which exhibits remarkable antiproliferative effects and is believed to be the most potent inhibitor of the metal-dependent histone deacetylases (HDACs).  Here, the 2.14 Å-resoln. crystal structure of the HDAC8-largazole thiol complex is the first of an HDAC complexed with a macrocyclic inhibitor and reveals that ideal thiolate-zinc coordination geometry is the key chem. feature responsible for its exceptional affinity and biol. activity.  Notably, the core structure of largazole is conserved in romidepsin, a depsipeptide natural product formulated as the drug Istodax recently approved for cancer chemotherapy.  Accordingly, the structure of the HDAC8-largazole thiol complex is the first to illustrate the mode of action of a new class of therapeutically important HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-DXxueVrU6bVg90H21EOLACvtfcHk0lhD60w0ijCgEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D&md5=9d05a3f1fd412f119ced61aaa771ef28</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja205972n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja205972n%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DK.%2BE.%26aulast%3DDowling%26aufirst%3DD.%2BP.%26aulast%3DBoone%26aufirst%3DM.%2BA.%26aulast%3DPhillips%26aufirst%3DA.%2BJ.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructural%2520basis%2520of%2520the%2520antiproliferative%2520activity%2520of%2520largazole%252C%2520a%2520depsipeptide%2520inhibitor%2520of%2520the%2520histone%2520deacetylases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D12474%26epage%3D12477%26doi%3D10.1021%2Fja205972n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Bowers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. M.</span><span> </span><span class="NLM_article-title">Total synthesis and biological mode of action of largazole: a potent class I histone deacetylase inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">11219</span><span class="NLM_x">–</span> <span class="NLM_lpage">11222</span><span class="refDoi"> DOI: 10.1021/ja8033763</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8033763" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos12ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=11219-11222&author=A.+Bowersauthor=N.+Westauthor=J.+Tauntonauthor=S.+L.+Schreiberauthor=J.+E.+Bradnerauthor=R.+M.+Williams&title=Total+synthesis+and+biological+mode+of+action+of+largazole%3A+a+potent+class+I+histone+deacetylase+inhibitor&doi=10.1021%2Fja8033763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis and biological mode of action of largazole: A potent Class I histone deacetylase inhibitor</span></div><div class="casAuthors">Bowers, Albert; West, Nathan; Taunton, Jack; Schreiber, Stuart L.; Bradner, James E.; Williams, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11219-11222</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The efficient total synthesis of the recently described natural substance largazole and its active metabolite largazole thiol is described.  The synthesis required eight linear steps and proceeded in 37% overall yield.  It is demonstrated that largazole is a pro-rug that is activated by removal of the octanoyl residue from the 3-hydroxy-7-mercaptohept-4-enoic acid moiety to generate the active metabolite largazole thiol, which is an extraordinarily potent Class I histone deacetylase inhibitor.  Synthetic largazole and largazole thiol have been evaluated side-by-side with FK228 and SAHA for inhibition of HDACs 1, 2, 3, and 6.  Largazole and largazole thiol were further assayed for cytotoxic activity against a panel of chemoresistant melanoma cell lines, and it was found that largazole is substantially more cytotoxic than largazole thiol; this difference is attributed to differences in the cell permeability of the two substances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdAVdDfScHCrVg90H21EOLACvtfcHk0li9UbHGMujiSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos12ru78%253D&md5=0063841a5d6a996484cdb329ff6baa64</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja8033763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8033763%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DA.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWilliams%26aufirst%3DR.%2BM.%26atitle%3DTotal%2520synthesis%2520and%2520biological%2520mode%2520of%2520action%2520of%2520largazole%253A%2520a%2520potent%2520class%2520I%2520histone%2520deacetylase%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D11219%26epage%3D11222%26doi%3D10.1021%2Fja8033763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Che, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Engineering cyclic tetrapeptides containing chimeric amino acids as preferred reverse-turn scaffolds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1021/jm0507072</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0507072" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1GktLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=111-124&author=Y.+Cheauthor=G.+R.+Marshall&title=Engineering+cyclic+tetrapeptides+containing+chimeric+amino+acids+as+preferred+reverse-turn+scaffolds&doi=10.1021%2Fjm0507072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering Cyclic Tetrapeptides Containing Chimeric Amino Acids as Preferred Reverse-Turn Scaffolds</span></div><div class="casAuthors">Che, Ye; Marshall, Garland R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four residues making almost a complete 180° turn in the direction of the peptide chain define a reverse turn, a common motif and recognition site in proteins.  Cyclization between residues i and i+3 and incorporation of heterochiral dipeptides (such as D-Pro-L-Pro) in the i+1 and i+2 positions are used to constrain a peptide to a reverse-turn conformation.  A combined approach, cyclic tetrapeptides (CTPs) based on heterochiral dipeptides of chimeric amino acids, is evaluated as minimalist scaffolds for reverse-turn conformations.  Cyclo(D-Pro-L-Pro-D-Pro-L-Pro) has been studied with d. functional theory (DFT) calcns. and mol. dynamics simulations.  The all-trans amide conformer was the most stable in vacuo, while the cis-trans-cis-trans (ctct) or trans-cis-trans-cis (tctc) amide conformer was more favored in water due to its large dipole moment.  Different conformations could be selectively stabilized by different substitutions on the proline rings.  Due to the small 12-membered ring and exocyclic constraints, conformational interconversions could only occur at high temp.  The presence of seven hydrogens on each ring that could be functionalized offers an overwhelming diversity to design mols. to probe receptors.  The spatial relationships of Cα-Cβ vectors of reverse turns in proteins were subjected to principal component anal. for detn. of the relative orientation of the Cα-Cβ vectors.  Most reverse-turn structures could be mimicked effectively with a subset of CTP scaffolds with an root-mean-square displacement (RMSD) of approx. 0.5 Å.  Structural diversity of CTP scaffolds could be enhanced by the incorporation of proline analogs, such as azaproline (azPro) or pipecolic (Pip), azapipecolic (azPip), nipecotic (Nip), and isonipecotic (Inp) acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoChaXRJzSdI7Vg90H21EOLACvtfcHk0li9UbHGMujiSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1GktLbE&md5=13a4a7881b1e7a5e0eb884d4aedfec06</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm0507072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0507072%26sid%3Dliteratum%253Aachs%26aulast%3DChe%26aufirst%3DY.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DEngineering%2520cyclic%2520tetrapeptides%2520containing%2520chimeric%2520amino%2520acids%2520as%2520preferred%2520reverse-turn%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D111%26epage%3D124%26doi%3D10.1021%2Fjm0507072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Chalmers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Pro-D-NMe-amino acid and D-Pro-NMe-amino acid: simple, efficient reverse-turn constraints</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">5927</span><span class="NLM_x">–</span> <span class="NLM_lpage">5937</span><span class="refDoi"> DOI: 10.1021/ja00127a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00127a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK2MXlvFahtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=5927-5937&author=D.+K.+Chalmersauthor=G.+R.+Marshall&title=Pro-D-NMe-amino+acid+and+D-Pro-NMe-amino+acid%3A+simple%2C+efficient+reverse-turn+constraints&doi=10.1021%2Fja00127a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Pro-D-NMe-Amino Acid and D-Pro-NMe-Amino Acid: Simple, Efficient Reverse-Turn Constraints</span></div><div class="casAuthors">Chalmers, David K.; Marshall, Garland R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5927-37</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of mimetics that, when incorporated into a peptide, constrain it to adopt a reverse turn is currently of considerable interest.  Numerous bicyclic dipeptide surrogates have been suggested which fix two of the four backbone dihedral angles of residues i + 1 and i + 2 of the turn.  In this paper, the authors report the conformational anal. of tetrapeptides contg. several bicyclic mimetics, sequences contg. proline and other N-methylamino acids in the i + 1 and i + 2 positions of the turn, and control peptide sequences using a Monte Carlo conformational search followed by mol. dynamics simulation in water as implicitly represented by a solvation model.  These studies indicate that the sequences D-Pro-N-methylamino acid (D-Pro-NMe-AA) and Pro-D-NMe-AA are effective at stabilizing the trans conformer of the amide bond between residues i + 1 and i + 2 and constraining three of the four backbone dihedral angles to those assocd. with structures which satisfy several criteria for reverse turns.  In many cases, these sequences were more effective than more complicated polycyclic mimetics.  D-Pro-NMe-AA and Pro-D-NMe-AA are compatible with conventional peptide synthesis methods and should provide a simple method to probe receptors for reverse-turn recognition through combinatorial libraries and structure-activity studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCnmTHPFnVXLVg90H21EOLACvtfcHk0li9UbHGMujiSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlvFahtr4%253D&md5=8d7eef279cf77c752898c4521d5bcd77</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fja00127a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00127a004%26sid%3Dliteratum%253Aachs%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DPro-D-NMe-amino%2520acid%2520and%2520D-Pro-NMe-amino%2520acid%253A%2520simple%252C%2520efficient%2520reverse-turn%2520constraints%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D5927%26epage%3D5937%26doi%3D10.1021%2Fja00127a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Takeuchi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Conformational analysis of reverse-turn constraints by N-methylation and N-hydroxylation of amide bonds in peptides and non-peptide mimetics</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">5363</span><span class="NLM_x">–</span> <span class="NLM_lpage">5372</span><span class="refDoi"> DOI: 10.1021/ja970855k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja970855k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK1cXjtlWku74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=5363-5372&author=Y.+Takeuchiauthor=G.+R.+Marshall&title=Conformational+analysis+of+reverse-turn+constraints+by+N-methylation+and+N-hydroxylation+of+amide+bonds+in+peptides+and+non-peptide+mimetics&doi=10.1021%2Fja970855k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Reverse-Turn Constraints by N-Methylation and N-Hydroxylation of Amide Bonds in Peptides and Non-Peptide Mimetics</span></div><div class="casAuthors">Takeuchi, Yasuo; Marshall, Garland R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5363-5372</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several non-peptide systems have been designed to mimic different types of reverse turns.  The incorporation of some of these mimetics into biol. active peptides has led to peptidomimetics with enhanced activity or metabolic stability.  This paper reports the conformational anal. of tetrapeptides contg. several bicyclic mimetics, sequences contg. Pro, other N-Me and N-hydroxy amino acids, and pipecolic acid at residue i + 2 of the turn, and control peptide sequences using the Monte Carlo/stochastic dynamics simulation with the new set of AMBER* parameters for proline-contg. peptides in water as implicitly represented by the GB/SA solvation model.  Simple N-methylation (Pro-D-NMeAA and D-Pro-NMeAA) and N-hydroxylation of the amide bond between residues i + 1 and i + 2 or inclusion of the larger ring homolog pipecolic acid (D-Pro-Pip) in the third position (i + 2) causes significant nucleation of reverse-turn structures.  Spirotricycle analogs restrict three of the four torsion angles that characterize the type II β-turn.  Spirolactam analogs also restrict two of the four torsion angles as effective β-turn constraints.  However, the geometry of a turn induced by indolizidinone and BTD differs significantly from that of an ideal β-turn and (S)-indolizidinone is more effective as a reverse turn than as a β-turn mimetic.  These systems provide useful conformational constraints when incorporated into the structure of selected bioactive peptides.  Such analogs can scan receptors for biol. recognition of β-turn scaffolds with oriented side chains through combinatorial libraries to efficiently develop three-dimensional structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXBY0-f_KpY7Vg90H21EOLACvtfcHk0li9UbHGMujiSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtlWku74%253D&md5=c92318c941c199b8871372d5789e91fe</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja970855k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja970855k%26sid%3Dliteratum%253Aachs%26aulast%3DTakeuchi%26aufirst%3DY.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DConformational%2520analysis%2520of%2520reverse-turn%2520constraints%2520by%2520N-methylation%2520and%2520N-hydroxylation%2520of%2520amide%2520bonds%2520in%2520peptides%2520and%2520non-peptide%2520mimetics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D5363%26epage%3D5372%26doi%3D10.1021%2Fja970855k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Bowers, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greshock, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Synthesis and conformation-activity relationships of the peptide isosteres of FK228 and largazole</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">2900</span><span class="NLM_x">–</span> <span class="NLM_lpage">2905</span><span class="refDoi"> DOI: 10.1021/ja807772w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja807772w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=2900-2905&author=A.+A.+Bowersauthor=T.+J.+Greshockauthor=N.+Westauthor=G.+Estiuauthor=S.+L.+Schreiberauthor=O.+Wiestauthor=R.+M.+Williamsauthor=J.+E.+Bradner&title=Synthesis+and+conformation-activity+relationships+of+the+peptide+isosteres+of+FK228+and+largazole&doi=10.1021%2Fja807772w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Conformation-Activity Relationships of the Peptide Isosteres of FK228 and Largazole</span></div><div class="casAuthors">Bowers, Albert A.; Greshock, Thomas J.; West, Nathan; Estiu, Guillermina; Schreiber, Stuart L.; Wiest, Olaf; Williams, Robert M.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2900-2905</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The peptide isosteres (10 and 11) of the naturally occurring and potent histone deacetylase (HDAC) inhibitors FK228 and largazole have been synthesized and evaluated side-by-side with FK228, largazole, and SAHA for inhibition of the class I HDACs 1, 2, 3 and 6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA7om4rJhoarVg90H21EOLACvtfcHk0ligfMsXxRBrnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSltbo%253D&md5=e89f48feda9a2d8c92a9120f1a8428f9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fja807772w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja807772w%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DA.%2BA.%26aulast%3DGreshock%26aufirst%3DT.%2BJ.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DWilliams%26aufirst%3DR.%2BM.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSynthesis%2520and%2520conformation-activity%2520relationships%2520of%2520the%2520peptide%2520isosteres%2520of%2520FK228%2520and%2520largazole%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D2900%26epage%3D2905%26doi%3D10.1021%2Fja807772w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Ballante, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">An Automated Strategy for Binding-Pose Selection and Docking Assessment in Structure-Based Drug Design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00603</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00603" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=F.+Ballanteauthor=G.+R.+Marshall&title=An+Automated+Strategy+for+Binding-Pose+Selection+and+Docking+Assessment+in+Structure-Based+Drug+Design&doi=10.1021%2Facs.jcim.5b00603"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00603%26sid%3Dliteratum%253Aachs%26aulast%3DBallante%26aufirst%3DF.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DAn%2520Automated%2520Strategy%2520for%2520Binding-Pose%2520Selection%2520and%2520Docking%2520Assessment%2520in%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26doi%3D10.1021%2Facs.jcim.5b00603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Nishino, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jose, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span> </span><span class="NLM_article-title">Chlamydocin-hydroxamic acid analogues as histone deacetylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5777</span><span class="NLM_x">–</span> <span class="NLM_lpage">5784</span><span class="refDoi"> DOI: 10.1016/j.bmc.2004.08.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.bmc.2004.08.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=5777-5784&author=N.+Nishinoauthor=B.+Joseauthor=R.+Shintaauthor=T.+Katoauthor=Y.+Komatsuauthor=M.+Yoshida&title=Chlamydocin-hydroxamic+acid+analogues+as+histone+deacetylase+inhibitors&doi=10.1016%2Fj.bmc.2004.08.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2004.08.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2004.08.041%26sid%3Dliteratum%253Aachs%26aulast%3DNishino%26aufirst%3DN.%26aulast%3DJose%26aufirst%3DB.%26aulast%3DShinta%26aufirst%3DR.%26aulast%3DKato%26aufirst%3DT.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DM.%26atitle%3DChlamydocin-hydroxamic%2520acid%2520analogues%2520as%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D5777%26epage%3D5784%26doi%3D10.1016%2Fj.bmc.2004.08.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Madsen, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristensen, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, C. A.</span><span> </span><span class="NLM_article-title">The effect of various zinc binding groups on inhibition of histone deacetylases 1–11</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">614</span><span class="NLM_x">–</span> <span class="NLM_lpage">626</span><span class="refDoi"> DOI: 10.1002/cmdc.201300433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fcmdc.201300433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=24375963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVWjur3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=614-626&author=A.+S.+Madsenauthor=H.+M.+Kristensenauthor=G.+Lanzauthor=C.+A.+Olsen&title=The+effect+of+various+zinc+binding+groups+on+inhibition+of+histone+deacetylases+1%E2%80%9311&doi=10.1002%2Fcmdc.201300433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">The Effect of Various Zinc Binding Groups on Inhibition of Histone Deacetylases 1-11</span></div><div class="casAuthors">Madsen, Andreas S.; Kristensen, Helle M. E.; Lanz, Gyrithe; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">614-626</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) have the ability to cleave the acetyl groups of ε-N-acetylated lysine residues in a variety of proteins.  Given that human cells contain thousands of different acetylated lysine residues, HDACS may regulate a wide variety of processes including some implicated in conditions such as cancer and neurodegenerative disorders.  Herein we report the synthesis and in vitro biochem. profiling of a series of compds., including known inhibitors as well as novel chemotypes, that incorporate putative new zinc binding domains.  By evaluating the compd. collection against all 11 recombinant human HDACs, we found that the trifluoromethyl ketone functionality provides potent inhibition of all four subclasses of the Zn2+-dependent HDACs.  Potent inhibition was obsd. with two different scaffolds, demonstrating the efficiency of the trifluoromethyl ketone moiety as a zinc binding motif.  Interestingly, we also identified silanediol as a zinc binding group with potential for future development of non-hydroxamate class I and class IIb HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGN-Ah_s0G9rVg90H21EOLACvtfcHk0ligfMsXxRBrnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVWjur3N&md5=4cc4f341ecb112034d26aeb668f3332f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300433%26sid%3Dliteratum%253Aachs%26aulast%3DMadsen%26aufirst%3DA.%2BS.%26aulast%3DKristensen%26aufirst%3DH.%2BM.%26aulast%3DLanz%26aufirst%3DG.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DThe%2520effect%2520of%2520various%2520zinc%2520binding%2520groups%2520on%2520inhibition%2520of%2520histone%2520deacetylases%25201%25E2%2580%259311%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D614%26epage%3D626%26doi%3D10.1002%2Fcmdc.201300433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Gibbs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartzman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alumkal, J.</span><span> </span><span class="NLM_article-title">Sulforaphane destabilizes the androgen receptor in prostate cancer cells by inactivating histone deacetylase 6</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">16663</span><span class="NLM_x">–</span> <span class="NLM_lpage">16668</span><span class="refDoi"> DOI: 10.1073/pnas.0908908106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.0908908106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=16663-16668&author=A.+Gibbsauthor=J.+Schwartzmanauthor=V.+Dengauthor=J.+Alumkal&title=Sulforaphane+destabilizes+the+androgen+receptor+in+prostate+cancer+cells+by+inactivating+histone+deacetylase+6&doi=10.1073%2Fpnas.0908908106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0908908106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0908908106%26sid%3Dliteratum%253Aachs%26aulast%3DGibbs%26aufirst%3DA.%26aulast%3DSchwartzman%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DV.%26aulast%3DAlumkal%26aufirst%3DJ.%26atitle%3DSulforaphane%2520destabilizes%2520the%2520androgen%2520receptor%2520in%2520prostate%2520cancer%2520cells%2520by%2520inactivating%2520histone%2520deacetylase%25206%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D16663%26epage%3D16668%26doi%3D10.1073%2Fpnas.0908908106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Singh, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powolny, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stan, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlotti, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahm, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marynowski, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bommareddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parise, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beumer, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, W. H.</span><span> </span><span class="NLM_article-title">Sulforaphane inhibits prostate carcinogenesis and pulmonary metastasis in TRAMP mice in association with increased cytotoxicity of natural killer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2117</span><span class="NLM_x">–</span> <span class="NLM_lpage">2125</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-3502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1158%2F0008-5472.CAN-08-3502" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2117-2125&author=S.+V.+Singhauthor=R.+Warinauthor=D.+Xiaoauthor=A.+A.+Powolnyauthor=S.+D.+Stanauthor=J.+A.+Arlottiauthor=Y.+Zengauthor=E.+R.+Hahmauthor=S.+W.+Marynowskiauthor=A.+Bommareddyauthor=D.+Desaiauthor=S.+Aminauthor=R.+A.+Pariseauthor=J.+H.+Beumerauthor=W.+H.+Chambers&title=Sulforaphane+inhibits+prostate+carcinogenesis+and+pulmonary+metastasis+in+TRAMP+mice+in+association+with+increased+cytotoxicity+of+natural+killer+cells&doi=10.1158%2F0008-5472.CAN-08-3502"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3502%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BV.%26aulast%3DWarin%26aufirst%3DR.%26aulast%3DXiao%26aufirst%3DD.%26aulast%3DPowolny%26aufirst%3DA.%2BA.%26aulast%3DStan%26aufirst%3DS.%2BD.%26aulast%3DArlotti%26aufirst%3DJ.%2BA.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DHahm%26aufirst%3DE.%2BR.%26aulast%3DMarynowski%26aufirst%3DS.%2BW.%26aulast%3DBommareddy%26aufirst%3DA.%26aulast%3DDesai%26aufirst%3DD.%26aulast%3DAmin%26aufirst%3DS.%26aulast%3DParise%26aufirst%3DR.%2BA.%26aulast%3DBeumer%26aufirst%3DJ.%2BH.%26aulast%3DChambers%26aufirst%3DW.%2BH.%26atitle%3DSulforaphane%2520inhibits%2520prostate%2520carcinogenesis%2520and%2520pulmonary%2520metastasis%2520in%2520TRAMP%2520mice%2520in%2520association%2520with%2520increased%2520cytotoxicity%2520of%2520natural%2520killer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2117%26epage%3D2125%26doi%3D10.1158%2F0008-5472.CAN-08-3502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Whitehead, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobler, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radetich, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claiborne, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grob, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McRiner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pancost, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patnaik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shultz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tichkule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tommasi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vash, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stams, T.</span><span> </span><span class="NLM_article-title">Human HDAC isoform selectivity achieved via exploitation of the acetate release channel with structurally unique small molecule inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4626</span><span class="NLM_x">–</span> <span class="NLM_lpage">4634</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.06.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.bmc.2011.06.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=21723733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovF2ju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4626-4634&author=L.+Whiteheadauthor=M.+R.+Doblerauthor=B.+Radetichauthor=Y.+Zhuauthor=P.+W.+Atadjaauthor=T.+Claiborneauthor=J.+E.+Grobauthor=A.+McRinerauthor=M.+R.+Pancostauthor=A.+Patnaikauthor=W.+Shaoauthor=M.+Shultzauthor=R.+Tichkuleauthor=R.+A.+Tommasiauthor=B.+Vashauthor=P.+Wangauthor=T.+Stams&title=Human+HDAC+isoform+selectivity+achieved+via+exploitation+of+the+acetate+release+channel+with+structurally+unique+small+molecule+inhibitors&doi=10.1016%2Fj.bmc.2011.06.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Human HDAC isoform selectivity achieved via exploitation of the acetate release channel with structurally unique small molecule inhibitors</span></div><div class="casAuthors">Whitehead, Lewis; Dobler, Markus R.; Radetich, Branko; Zhu, Yanyi; Atadja, Peter W.; Claiborne, Tavina; Grob, Jonathan E.; McRiner, Andrew; Pancost, Margaret R.; Patnaik, Anup; Shao, Wenlin; Shultz, Michael; Tichkule, Ritesh; Tommasi, Ruben A.; Vash, Brian; Wang, Ping; Stams, Travis</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4626-4634</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we report the discovery of a family of novel yet simple, amino-acid derived class I HDAC inhibitors that demonstrate isoform selectivity via access to the internal acetate release channel.  Isoform selectivity criteria is discussed on the basis of X-ray crystallog. and mol. modeling of these novel inhibitors bound to HDAC8, potentially revealing insights into the mechanism of enzymic function through novel structural features revealed at the at. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgV7MIGevIxrVg90H21EOLACvtfcHk0liboYnxbcn-mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovF2ju7w%253D&md5=c252ec223e909e92ec165f60a1a33b16</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.06.030%26sid%3Dliteratum%253Aachs%26aulast%3DWhitehead%26aufirst%3DL.%26aulast%3DDobler%26aufirst%3DM.%2BR.%26aulast%3DRadetich%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DAtadja%26aufirst%3DP.%2BW.%26aulast%3DClaiborne%26aufirst%3DT.%26aulast%3DGrob%26aufirst%3DJ.%2BE.%26aulast%3DMcRiner%26aufirst%3DA.%26aulast%3DPancost%26aufirst%3DM.%2BR.%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DShultz%26aufirst%3DM.%26aulast%3DTichkule%26aufirst%3DR.%26aulast%3DTommasi%26aufirst%3DR.%2BA.%26aulast%3DVash%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DStams%26aufirst%3DT.%26atitle%3DHuman%2520HDAC%2520isoform%2520selectivity%2520achieved%2520via%2520exploitation%2520of%2520the%2520acetate%2520release%2520channel%2520with%2520structurally%2520unique%2520small%2520molecule%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D4626%26epage%3D4634%26doi%3D10.1016%2Fj.bmc.2011.06.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Hussain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bharadwaj, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaudhuri, M. K.</span><span> </span><span class="NLM_article-title">Borax-catalyzed and pH-controlled selective oxidation of organic sulfides by H<sub>2</sub>O<sub>2</sub>: An environmentally clean protocol</span> <span class="citation_source-journal">Eur. J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">2009</span><span class="NLM_x">, </span> <span class="NLM_fpage">3319</span><span class="NLM_x">–</span> <span class="NLM_lpage">3322</span><span class="refDoi"> DOI: 10.1002/ejoc.200900265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fejoc.200900265" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2009&publication_year=2009&pages=3319-3322&author=S.+Hussainauthor=S.+K.+Bharadwajauthor=R.+Pandeyauthor=M.+K.+Chaudhuri&title=Borax-catalyzed+and+pH-controlled+selective+oxidation+of+organic+sulfides+by+H2O2%3A+An+environmentally+clean+protocol&doi=10.1002%2Fejoc.200900265"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fejoc.200900265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.200900265%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DS.%26aulast%3DBharadwaj%26aufirst%3DS.%2BK.%26aulast%3DPandey%26aufirst%3DR.%26aulast%3DChaudhuri%26aufirst%3DM.%2BK.%26atitle%3DBorax-catalyzed%2520and%2520pH-controlled%2520selective%2520oxidation%2520of%2520organic%2520sulfides%2520by%2520H2O2%253A%2520An%2520environmentally%2520clean%2520protocol%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2009%26volume%3D2009%26spage%3D3319%26epage%3D3322%26doi%3D10.1002%2Fejoc.200900265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Saito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mariko, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nosaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchiya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ando, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuruo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, O.</span><span> </span><span class="NLM_article-title">A synthetic inhibitor of histone deacetylase, MS-27–275, with marked in vivo antitumor activity against human tumors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">4592</span><span class="NLM_x">–</span> <span class="NLM_lpage">4597</span><span class="refDoi"> DOI: 10.1073/pnas.96.8.4592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.96.8.4592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=4592-4597&author=A.+Saitoauthor=T.+Yamashitaauthor=Y.+Marikoauthor=Y.+Nosakaauthor=K.+Tsuchiyaauthor=T.+Andoauthor=T.+Suzukiauthor=T.+Tsuruoauthor=O.+Nakanishi&title=A+synthetic+inhibitor+of+histone+deacetylase%2C+MS-27%E2%80%93275%2C+with+marked+in+vivo+antitumor+activity+against+human+tumors&doi=10.1073%2Fpnas.96.8.4592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.8.4592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.8.4592%26sid%3Dliteratum%253Aachs%26aulast%3DSaito%26aufirst%3DA.%26aulast%3DYamashita%26aufirst%3DT.%26aulast%3DMariko%26aufirst%3DY.%26aulast%3DNosaka%26aufirst%3DY.%26aulast%3DTsuchiya%26aufirst%3DK.%26aulast%3DAndo%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DTsuruo%26aufirst%3DT.%26aulast%3DNakanishi%26aufirst%3DO.%26atitle%3DA%2520synthetic%2520inhibitor%2520of%2520histone%2520deacetylase%252C%2520MS-27%25E2%2580%2593275%252C%2520with%2520marked%2520in%2520vivo%2520antitumor%2520activity%2520against%2520human%2520tumors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D4592%26epage%3D4597%26doi%3D10.1073%2Fpnas.96.8.4592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosa, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitreva, M.</span><span> </span><span class="NLM_article-title">Targeting lysine deacetylases (KDACs) in Parasites</span> <span class="citation_source-journal">PLoS Neglected Trop. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0004026</span><span class="refDoi"> DOI: 10.1371/journal.pntd.0004026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.pntd.0004026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=26402733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlCnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=e0004026&author=Q.+Wangauthor=B.+A.+Rosaauthor=B.+Nareauthor=K.+Powellauthor=S.+Valenteauthor=D.+Rotiliauthor=A.+Maiauthor=G.+R.+Marshallauthor=M.+Mitreva&title=Targeting+lysine+deacetylases+%28KDACs%29+in+Parasites&doi=10.1371%2Fjournal.pntd.0004026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="#"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=&md5=9874b665cc7a056b8e2f928dd3112440</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pntd.0004026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pntd.0004026%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DRosa%26aufirst%3DB.%2BA.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DPowell%26aufirst%3DK.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26aulast%3DMitreva%26aufirst%3DM.%26atitle%3DTargeting%2520lysine%2520deacetylases%2520%2528KDACs%2529%2520in%2520Parasites%26jtitle%3DPLoS%2520Neglected%2520Trop.%2520Dis.%26date%3D2015%26volume%3D9%26spage%3De0004026%26doi%3D10.1371%2Fjournal.pntd.0004026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Ballante, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musmuca, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span> </span><span class="NLM_article-title">Comprehensive model of wild-type and mutant HIV-1 reverse transciptases</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">907</span><span class="NLM_x">–</span> <span class="NLM_lpage">919</span><span class="refDoi"> DOI: 10.1007/s10822-012-9586-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1007%2Fs10822-012-9586-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=907-919&author=F.+Ballanteauthor=I.+Musmucaauthor=G.+R.+Marshallauthor=R.+Ragno&title=Comprehensive+model+of+wild-type+and+mutant+HIV-1+reverse+transciptases&doi=10.1007%2Fs10822-012-9586-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9586-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9586-6%26sid%3Dliteratum%253Aachs%26aulast%3DBallante%26aufirst%3DF.%26aulast%3DMusmuca%26aufirst%3DI.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26aulast%3DRagno%26aufirst%3DR.%26atitle%3DComprehensive%2520model%2520of%2520wild-type%2520and%2520mutant%2520HIV-1%2520reverse%2520transciptases%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D907%26epage%3D919%26doi%3D10.1007%2Fs10822-012-9586-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Ballante, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span> </span><span class="NLM_article-title">3-D QSAutogrid/R: an alternative procedure to build 3-D QSAR models. Methodologies and applications</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1674</span><span class="NLM_x">–</span> <span class="NLM_lpage">1685</span><span class="refDoi"> DOI: 10.1021/ci300123x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300123x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFOmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1674-1685&author=F.+Ballanteauthor=R.+Ragno&title=3-D+QSAutogrid%2FR%3A+an+alternative+procedure+to+build+3-D+QSAR+models.+Methodologies+and+applications&doi=10.1021%2Fci300123x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">3-D QSAutogrid/R: An Alternative Procedure To Build 3-D QSAR Models. Methodologies and Applications</span></div><div class="casAuthors">Ballante, Flavio; Ragno, Rino</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1674-1685</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since it first appeared in 1988 3-D QSAR has proved its potential in the field of drug design and activity prediction.  Although thousands of citations now exist in 3-D QSAR, its development was rather slow with the majority of new 3-D QSAR applications just extensions of CoMFA.  An alternative way to build 3-D QSAR models, based on an evolution of software, has been named 3-D QSAutogrid/R and has been developed to use only software freely available to academics. 3-D QSAutogrid/R covers all the main features of CoMFA and GRID/GOLPE with implementation by multiprobe/multiregion variable selection (MPGRS) that improves the simplification of interpretation of the 3-D QSAR map.  The methodol. is based on the integration of the mol. interaction fields as calcd. by AutoGrid and the R statistical environment that can be easily coupled with many free graphical mol. interfaces such as UCSF-Chimera, AutoDock Tools, JMol, and others.  The description of each R package is reported in detail, and, to assess its validity, 3-D QSAutogrid/R has been applied to three mol. data sets of which either CoMFA or GRID/GOLPE models were reported in order to compare the results. 3-D QSAutogrid/R has been used as the core engine to prep. more that 240 3-D QSAR models forming the very first 3-D QSAR server (www.3d-qsar.com) with its code freely available through R-Cran distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcTVzJ7Ee_q7Vg90H21EOLACvtfcHk0liaPeV4iCif1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFOmsbs%253D&md5=a65b741e5bd01070dccf19db522a6685</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fci300123x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300123x%26sid%3Dliteratum%253Aachs%26aulast%3DBallante%26aufirst%3DF.%26aulast%3DRagno%26aufirst%3DR.%26atitle%3D3-D%2520QSAutogrid%252FR%253A%2520an%2520alternative%2520procedure%2520to%2520build%25203-D%2520QSAR%2520models.%2520Methodologies%2520and%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1674%26epage%3D1685%26doi%3D10.1021%2Fci300123x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stutzle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Empirical scoring functions for advanced protein-ligand docking with PLANTS</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/ci800298z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800298z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=84-96&author=O.+Korbauthor=T.+Stutzleauthor=T.+E.+Exner&title=Empirical+scoring+functions+for+advanced+protein-ligand+docking+with+PLANTS&doi=10.1021%2Fci800298z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span></div><div class="casAuthors">Korb, Oliver; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we present two empirical scoring functions, PLANTSCHEMPLP and PLANTSPLP, designed for our docking algorithm PLANTS (Protein-Ligand ANT System), which is based on ant colony optimization (ACO).  They are related, regarding their functional form, to parts of already published scoring functions and force fields.  The parametrization procedure described here was able to identify several parameter settings showing an excellent performance for the task of pose prediction on two test sets comprising 298 complexes in total.  Up to 87% of the complexes of the Astex diverse set and 77% of the CCDC/Astex clean listnc (noncovalently bound complexes of the clean list) could be reproduced with root-mean-square deviations of less than 2 Å with respect to the exptl. detd. structures.  A comparison with the state-of-the-art docking tool GOLD clearly shows that this is, esp. for the druglike Astex diverse set, an improvement in pose prediction performance.  Addnl., optimized parameter settings for the search algorithm were identified, which can be used to balance pose prediction reliability and search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_kyR7CGk9mbVg90H21EOLACvtfcHk0liaPeV4iCif1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D&md5=fce960b75b9f8906354b7c97ff84c092</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci800298z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800298z%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DStutzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DEmpirical%2520scoring%2520functions%2520for%2520advanced%2520protein-ligand%2520docking%2520with%2520PLANTS%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D84%26epage%3D96%26doi%3D10.1021%2Fci800298z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span><span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhHJvXMcOfFzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Santos-Martins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4(Zn): an improved AutoDock force field for small-molecule docking to zinc metalloproteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2371</span><span class="NLM_x">–</span> <span class="NLM_lpage">2379</span><span class="refDoi"> DOI: 10.1021/ci500209e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500209e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslWmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2371-2379&author=D.+Santos-Martinsauthor=S.+Forliauthor=M.+J.+Ramosauthor=A.+J.+Olson&title=AutoDock4%28Zn%29%3A+an+improved+AutoDock+force+field+for+small-molecule+docking+to+zinc+metalloproteins&doi=10.1021%2Fci500209e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock4Zn: An Improved AutoDock Force Field for Small-Molecule Docking to Zinc Metalloproteins</span></div><div class="casAuthors">Santos-Martins, Diogo; Forli, Stefano; Ramos, Maria Joao; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2371-2379</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Zinc is present in a wide variety of proteins and is important in the metab. of most organisms.  Zinc metalloenzymes are therapeutically relevant targets in diseases such as cancer, heart disease, bacterial infection, and Alzheimer's disease.  In most cases a drug mol. targeting such enzymes establishes an interaction that coordinates with the zinc ion.  Thus, accurate prediction of the interaction of ligands with zinc is an important aspect of computational docking and virtual screening against zinc contg. proteins.  We have extended the AutoDock force field to include a specialized potential describing the interactions of zinc-coordinating ligands.  This potential describes both the energetic and geometric components of the interaction.  The new force field, named AutoDock4Zn, was calibrated on a data set of 292 crystal complexes contg. zinc.  Redocking expts. show that the force field provides significant improvement in performance in both free energy of binding estn. as well as in root-mean-square deviation from the crystal structure pose.  The new force field has been implemented in AutoDock without modification to the source code.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0MFHtEldlWLVg90H21EOLACvtfcHk0lhHJvXMcOfFzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslWmurg%253D&md5=b331b3a92315f7fd9a56d1e43a16990d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci500209e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500209e%26sid%3Dliteratum%253Aachs%26aulast%3DSantos-Martins%26aufirst%3DD.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2528Zn%2529%253A%2520an%2520improved%2520AutoDock%2520force%2520field%2520for%2520small-molecule%2520docking%2520to%2520zinc%2520metalloproteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2371%26epage%3D2379%26doi%3D10.1021%2Fci500209e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhHJvXMcOfFzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Allen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balius, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brozell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">DOCK 6: impact of new features and current docking performance</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1132</span><span class="NLM_x">–</span> <span class="NLM_lpage">1156</span><span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-1156&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+impact+of+new+features+and+current+docking+performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0liohQ4NcyPXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520impact%2520of%2520new%2520features%2520and%2520current%2520docking%2520performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D1132%26epage%3D1156%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Bai, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H. L.</span><span> </span><span class="NLM_article-title">An accurate metalloprotein-specific scoring function and molecular docking program devised by a dynamic sampling and iteration optimization strategy</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">833</span><span class="NLM_x">–</span> <span class="NLM_lpage">847</span><span class="refDoi"> DOI: 10.1021/ci500647f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500647f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFensLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=833-847&author=F.+Baiauthor=S.+Liaoauthor=J.+F.+Guauthor=H.+L.+Jiangauthor=X.+C.+Wangauthor=H.+L.+Li&title=An+accurate+metalloprotein-specific+scoring+function+and+molecular+docking+program+devised+by+a+dynamic+sampling+and+iteration+optimization+strategy&doi=10.1021%2Fci500647f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">An Accurate Metalloprotein-Specific Scoring Function and Molecular Docking Program Devised by a Dynamic Sampling and Iteration Optimization Strategy</span></div><div class="casAuthors">Bai, Fang; Liao, Sha; Gu, Junfeng; Jiang, Hualiang; Wang, Xicheng; Li, Honglin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">833-847</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalloproteins, particularly zinc metalloproteins, are promising therapeutic targets, and recent efforts have focused on the identification of potent and selective inhibitors of these proteins.  However, the ability of current drug discovery and design technologies, such as mol. docking and mol. dynamics simulations, to probe metal-ligand interactions remains limited because of their complicated coordination geometries and rough treatment in current force fields.  Herein the authors introduce a robust, multiobjective optimization algorithm-driven metalloprotein-specific docking program named MpSDock, which runs on a scheme similar to consensus scoring consisting of a force-field-based scoring function and a knowledge-based scoring function.  For this purpose, in this study, an effective knowledge-based zinc metalloprotein-specific scoring function based on the inverse Boltzmann law was designed and optimized using a dynamic sampling and iteration optimization strategy.  This optimization strategy can dynamically sample and regenerate decoy poses used in each iteration step of refining the scoring function, thus dramatically improving both the effectiveness of the exploration of the binding conformational space and the sensitivity of the ranking of the native binding poses.  To validate the zinc metalloprotein-specific scoring function and its special built-in docking program, denoted MpSDockZn, an extensive comparison was performed against six universal, popular docking programs: Glide XP mode, Glide SP mode, Gold, AutoDock, AutoDock4Zn, and EADock DSS.  The zinc metalloprotein-specific knowledge-based scoring function exhibited prominent performance in accurately describing the geometries and interactions of the coordination bonds between the zinc ions and chelating agents of the ligands.  In addn., MpSDockZn had a competitive ability to sample and identify native binding poses with a higher success rate than the other six docking programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo79bqIwX3mGLVg90H21EOLACvtfcHk0liohQ4NcyPXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFensLg%253D&md5=737110fbbc8e0318cf6c518f742c6d5c</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fci500647f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500647f%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiao%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DJ.%2BF.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DWang%26aufirst%3DX.%2BC.%26aulast%3DLi%26aufirst%3DH.%2BL.%26atitle%3DAn%2520accurate%2520metalloprotein-specific%2520scoring%2520function%2520and%2520molecular%2520docking%2520program%2520devised%2520by%2520a%2520dynamic%2520sampling%2520and%2520iteration%2520optimization%2520strategy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D833%26epage%3D847%26doi%3D10.1021%2Fci500647f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surflex-Dock 2.1: robust performance from ligand energetic modeling, ring flexibility, and knowledge-based search</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">281</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1007/s10822-007-9114-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1007%2Fs10822-007-9114-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=17387436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVOgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=281-306&author=A.+N.+Jain&title=Surflex-Dock+2.1%3A+robust+performance+from+ligand+energetic+modeling%2C+ring+flexibility%2C+and+knowledge-based+search&doi=10.1007%2Fs10822-007-9114-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex-Dock 2.1: Robust performance from ligand energetic modeling, ring flexibility, and knowledge-based search</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">281-306</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Surflex flexible mol. docking method has been generalized and extended in two primary areas related to the search component of docking.  First, incorporation of a small-mol. force-field extends the search into Cartesian coordinates constrained by internal ligand energetics.  Whereas previous versions searched only the alignment and acyclic torsional space of the ligand, the new approach supports dynamic ring flexibility and all-atom optimization of docked ligand poses.  Second, knowledge of well established mol. interactions between ligand fragments and a target protein can be directly exploited to guide the search process.  This offers advantages in some cases over the search strategy where ligand alignment is guided solely by a "protomol" (a pre-computed mol. representation of an idealized ligand).  Results are presented on both docking accuracy and screening utility using multiple publicly available benchmark data sets that place Surflex's performance in the context of other mol. docking methods.  In terms of docking accuracy, Surflex-Dock 2.1 performs as well as the best available methods.  In the area of screening utility, Surflex's performance is extremely robust, and it is clearly superior to other methods within the set of cases for which comparative data are available, with roughly double the screening enrichment performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQaowZ5Af2ybVg90H21EOLACvtfcHk0liohQ4NcyPXFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVOgt78%253D&md5=3cb4321457ff35914787a08acfca7b23</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9114-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9114-2%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex-Dock%25202.1%253A%2520robust%2520performance%2520from%2520ligand%2520energetic%2520modeling%252C%2520ring%2520flexibility%252C%2520and%2520knowledge-based%2520search%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D281%26epage%3D306%26doi%3D10.1007%2Fs10822-007-9114-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Scuto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschbaum, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowolik, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kretzner, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juhasz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pullarkat, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jove, R.</span><span> </span><span class="NLM_article-title">The novel histone deacetylase inhibitor, LBH589, induces expression of DNA damage response genes and apoptosis in Ph- acute lymphoblastic leukemia cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">5093</span><span class="NLM_x">–</span> <span class="NLM_lpage">5100</span><span class="refDoi"> DOI: 10.1182/blood-2007-10-117762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1182%2Fblood-2007-10-117762" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2008&pages=5093-5100&author=A.+Scutoauthor=M.+Kirschbaumauthor=C.+Kowolikauthor=L.+Kretznerauthor=A.+Juhaszauthor=P.+Atadjaauthor=V.+Pullarkatauthor=R.+Bhatiaauthor=S.+Formanauthor=Y.+Yenauthor=R.+Jove&title=The+novel+histone+deacetylase+inhibitor%2C+LBH589%2C+induces+expression+of+DNA+damage+response+genes+and+apoptosis+in+Ph-+acute+lymphoblastic+leukemia+cells&doi=10.1182%2Fblood-2007-10-117762"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-10-117762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-10-117762%26sid%3Dliteratum%253Aachs%26aulast%3DScuto%26aufirst%3DA.%26aulast%3DKirschbaum%26aufirst%3DM.%26aulast%3DKowolik%26aufirst%3DC.%26aulast%3DKretzner%26aufirst%3DL.%26aulast%3DJuhasz%26aufirst%3DA.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DPullarkat%26aufirst%3DV.%26aulast%3DBhatia%26aufirst%3DR.%26aulast%3DForman%26aufirst%3DS.%26aulast%3DYen%26aufirst%3DY.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DThe%2520novel%2520histone%2520deacetylase%2520inhibitor%252C%2520LBH589%252C%2520induces%2520expression%2520of%2520DNA%2520damage%2520response%2520genes%2520and%2520apoptosis%2520in%2520Ph-%2520acute%2520lymphoblastic%2520leukemia%2520cells%26jtitle%3DBlood%26date%3D2008%26volume%3D111%26spage%3D5093%26epage%3D5100%26doi%3D10.1182%2Fblood-2007-10-117762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Crisanti, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeers, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campling, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fridlender, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kao, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albelda, S. M.</span><span> </span><span class="NLM_article-title">The HDAC inhibitor panobinostat (LBH589) inhibits mesothelioma and lung cancer cells in vitro and in vivo with particular efficacy for small cell lung cancer</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2221</span><span class="NLM_x">–</span> <span class="NLM_lpage">2231</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1158%2F1535-7163.MCT-09-0138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19671764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=2221-2231&author=M.+C.+Crisantiauthor=A.+F.+Wallaceauthor=V.+Kapoorauthor=F.+Vandermeersauthor=M.+L.+Dowlingauthor=L.+P.+Pereiraauthor=K.+Colemanauthor=B.+G.+Camplingauthor=Z.+G.+Fridlenderauthor=G.+D.+Kaoauthor=S.+M.+Albelda&title=The+HDAC+inhibitor+panobinostat+%28LBH589%29+inhibits+mesothelioma+and+lung+cancer+cells+in+vitro+and+in+vivo+with+particular+efficacy+for+small+cell+lung+cancer&doi=10.1158%2F1535-7163.MCT-09-0138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">The HDAC inhibitor panobinostat (LBH589) inhibits mesothelioma and lung cancer cells in vitro and in vivo with particular efficacy for small cell lung cancer</span></div><div class="casAuthors">Crisanti, M. Cecilia; Wallace, Africa F.; Kapoor, Veena; Vandermeers, Fabian; Dowling, Melissa L.; Pereira, Luana P.; Coleman, Kara; Campling, Barbara G.; Fridlender, Zvi G.; Kao, Gary D.; Albelda, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2221-2231</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Lung cancer is the leading cause of cancer deaths in the United States.  Current therapies are inadequate.  Histone deacetylase inhibitors (HDACi) are a recently developed class of anticancer agents that cause increased acetylation of core histones and nonhistone proteins leading to modulation of gene expression and protein activity involved in cancer cell growth and survival pathways.  We examd. the efficacy of the HDACi panobinostat (LBH589) in a wide range of lung cancers and mesotheliomas.  Panobinostat was cytotoxic in almost all 37 cancer cell lines tested.  IC50 and LD50 values were in the low nmol/L range (4-470 nmol/L; median, 20 nmol/L).  Small cell lung cancer (SCLC) cell lines were among the most sensitive lines, with LD50 values consistently <25 nmol/L.  In lung cancer and mesothelioma animal models, panobinostat significantly decreased tumor growth by an av. of 62% when compared with vehicle control.  Panobinostat was equally effective in immunocompetent and severe combined immunodeficiency mice, indicating that the inhibition of tumor growth by panobinostat was not due to direct immunol. effects.  Panobinostat was, however, particularly effective in SCLC xenografts, and the addn. of the chemotherapy agent etoposide augmented antitumor effects.  Protein anal. of treated tumor biopsies revealed elevated amts. of cell cycle regulators such as p21 and proapoptosis factors, such as caspase 3 and 7 and cleaved poly[ADP-ribose] polymerase, coupled with decreased levels of antiapoptotic factors such as Bcl-2 and Bcl-XL.  These studies together suggest that panobinostat may be a useful adjunct in the treatment of thoracic malignancies, esp. SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhFRlQ0BB9B7Vg90H21EOLACvtfcHk0lj8XNg0ijCAaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7g%253D&md5=dc513f6d94913942f310061a9172229f</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0138%26sid%3Dliteratum%253Aachs%26aulast%3DCrisanti%26aufirst%3DM.%2BC.%26aulast%3DWallace%26aufirst%3DA.%2BF.%26aulast%3DKapoor%26aufirst%3DV.%26aulast%3DVandermeers%26aufirst%3DF.%26aulast%3DDowling%26aufirst%3DM.%2BL.%26aulast%3DPereira%26aufirst%3DL.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCampling%26aufirst%3DB.%2BG.%26aulast%3DFridlender%26aufirst%3DZ.%2BG.%26aulast%3DKao%26aufirst%3DG.%2BD.%26aulast%3DAlbelda%26aufirst%3DS.%2BM.%26atitle%3DThe%2520HDAC%2520inhibitor%2520panobinostat%2520%2528LBH589%2529%2520inhibits%2520mesothelioma%2520and%2520lung%2520cancer%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%2520with%2520particular%2520efficacy%2520for%2520small%2520cell%2520lung%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D2221%26epage%3D2231%26doi%3D10.1158%2F1535-7163.MCT-09-0138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span>GraphPad 6.01; <span class="NLM_publisher-name">GraphPad Software</span>: <span class="NLM_publisher-loc">La Jolla CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GraphPad+6.01%3B+GraphPad+Software%3A+La+Jolla+CA%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGraphPad%25206.01%26pub%3DGraphPad%2520Software%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Ying, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luesch, H.</span><span> </span><span class="NLM_article-title">Total synthesis and molecular target of largazole, a histone deacetylase inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">8455</span><span class="NLM_x">–</span> <span class="NLM_lpage">8459</span><span class="refDoi"> DOI: 10.1021/ja8013727</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8013727" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVeksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=8455-8459&author=Y.+Yingauthor=K.+Taoriauthor=H.+Kimauthor=J.+Hongauthor=H.+Luesch&title=Total+synthesis+and+molecular+target+of+largazole%2C+a+histone+deacetylase+inhibitor&doi=10.1021%2Fja8013727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis and Molecular Target of Largazole, a Histone Deacetylase Inhibitor</span></div><div class="casAuthors">Ying, Yongcheng; Taori, Kanchan; Kim, Hyoungsu; Hong, Jiyong; Luesch, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">8455-8459</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Full details of the concise and convergent synthesis (eight steps, 19% overall yield), its extension to the prepn. of a series of key analogs, and the mol. target and pharmacophore of largazole are described.  Central to the synthesis of largazole is a macrocyclization reaction for formation of the strained 16-membered depsipeptide core followed by an olefin cross-metathesis reaction for installation of the thioester.  The biol. evaluation of largazole and its key analogs, including an acetyl analog, a thiol analog, and a hydroxyl analog, suggested that histone deacetylases (HDACs) are mol. targets of largazole and largazole is a class I HDAC inhibitor.  In addn., structure-activity relationship (SAR) studies revealed that the thiol group is the pharmacophore of the natural product.  Largazole's HDAC inhibitory activity correlates with its antiproliferative activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3QNyBNo1EnbVg90H21EOLACvtfcHk0lj8XNg0ijCAaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVeksr4%253D&md5=d580c55dfc1c921346a8d79b2b96baa3</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fja8013727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8013727%26sid%3Dliteratum%253Aachs%26aulast%3DYing%26aufirst%3DY.%26aulast%3DTaori%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DLuesch%26aufirst%3DH.%26atitle%3DTotal%2520synthesis%2520and%2520molecular%2520target%2520of%2520largazole%252C%2520a%2520histone%2520deacetylase%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D8455%26epage%3D8459%26doi%3D10.1021%2Fja8013727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The protein data bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+protein+data+bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0liIqFWGF3OP9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520protein%2520data%2520bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0liIqFWGF3OP9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Sousa da Silva, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vranken, W. F.</span><span> </span><span class="NLM_article-title">ACPYPE - AnteChamber PYthon Parser interfacE</span> <span class="citation_source-journal">BMC Res. Notes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">367</span><span class="refDoi"> DOI: 10.1186/1756-0500-5-367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1186%2F1756-0500-5-367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=22824207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=367&author=A.+W.+Sousa+da+Silvaauthor=W.+F.+Vranken&title=ACPYPE+-+AnteChamber+PYthon+Parser+interfacE&doi=10.1186%2F1756-0500-5-367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">ACPYPE - AnteChamber PYthon Parser interfacE</span></div><div class="casAuthors">Sousa da Silva Alan W; Vranken Wim F</div><div class="citationInfo"><span class="NLM_cas:title">BMC research notes</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">367</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  ACPYPE (or AnteChamber PYthon Parser interfacE) is a wrapper script around the ANTECHAMBER software that simplifies the generation of small molecule topologies and parameters for a variety of molecular dynamics programmes like GROMACS, CHARMM and CNS.  It is written in the Python programming language and was developed as a tool for interfacing with other Python based applications such as the CCPN software suite (for NMR data analysis) and ARIA (for structure calculations from NMR data).  ACPYPE is open source code, under GNU GPL v3, and is available as a stand-alone application at http://www.ccpn.ac.uk/acpype and as a web portal application at http://webapps.ccpn.ac.uk/acpype.  FINDINGS:  We verified the topologies generated by ACPYPE in three ways: by comparing with default AMBER topologies for standard amino acids; by generating and verifying topologies for a large set of ligands from the PDB; and by recalculating the structures for 5 protein-ligand complexes from the PDB.  CONCLUSIONS:  ACPYPE is a tool that simplifies the automatic generation of topology and parameters in different formats for different molecular mechanics programmes, including calculation of partial charges, while being object oriented for integration with other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbj1v3HcWoNJZoeNcnHOQ9fW6udTcc2eYqINtoSdE6VLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D&md5=60f750ba98c975374b351a39a1fa0ec4</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1186%2F1756-0500-5-367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-0500-5-367%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%2Bda%2BSilva%26aufirst%3DA.%2BW.%26aulast%3DVranken%26aufirst%3DW.%2BF.%26atitle%3DACPYPE%2520-%2520AnteChamber%2520PYthon%2520Parser%2520interfacE%26jtitle%3DBMC%2520Res.%2520Notes%26date%3D2012%26volume%3D5%26spage%3D367%26doi%3D10.1186%2F1756-0500-5-367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lh3i-gVQSLDzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lh3i-gVQSLDzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Lindorff-Larsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1950</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1002/prot.22711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fprot.22711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=20408171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1950-1958&author=K.+Lindorff-Larsenauthor=S.+Pianaauthor=K.+Palmoauthor=P.+Maragakisauthor=J.+L.+Klepeisauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Improved+side-chain+torsion+potentials+for+the+Amber+ff99SB+protein+force+field&doi=10.1002%2Fprot.22711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span></div><div class="casAuthors">Lindorff-Larsen, Kresten; Piana, Stefano; Palmo, Kim; Maragakis, Paul; Klepeis, John L.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1950-1958</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Recent advances in hardware and software have enabled increasingly long mol. dynamics (MD) simulations of biomols., exposing certain limitations in the accuracy of the force fields used for such simulations and spurring efforts to refine these force fields.  Recent modifications to the Amber and CHARMM protein force fields, for example, have improved the backbone torsion potentials, remedying deficiencies in earlier versions.  Here, the authors further advance simulation accuracy by improving the amino acid side-chain torsion potentials of the Amber ff99SB force field.  First, the authors used simulations of model alpha-helical systems to identify the four residue types whose rotamer distribution differed the most from expectations based on Protein Data Bank statistics.  Second, the authors optimized the side-chain torsion potentials of these residues to match new, high-level quantum-mech. calcns.  Finally, the authors used microsecond-timescale MD simulations in explicit solvent to validate the resulting force field against a large set of exptl. NMR measurements that directly probe side-chain conformations.  The new force field, which the authors have termed Amber ff99SB-ILDN, exhibits considerably better agreement with the NMR data.  Proteins 2010. © 2010 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNFHrYs9024bVg90H21EOLACvtfcHk0lh3i-gVQSLDzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D&md5=447a9004026e2b93f0f7beff165daa09</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fprot.22711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22711%26sid%3Dliteratum%253Aachs%26aulast%3DLindorff-Larsen%26aufirst%3DK.%26aulast%3DPiana%26aufirst%3DS.%26aulast%3DPalmo%26aufirst%3DK.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DImproved%2520side-chain%2520torsion%2520potentials%2520for%2520the%2520Amber%2520ff99SB%2520protein%2520force%2520field%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2010%26volume%3D78%26spage%3D1950%26epage%3D1958%26doi%3D10.1002%2Fprot.22711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G. R.</span><span> </span><span class="NLM_article-title">Open Babel: an open chemical toolbox</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="refDoi"> DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+an+open+chemical+toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: an open chemical toolbox</span></div><div class="casAuthors">O'Boyle, Noel M.; Banck, Michael; James, Craig A.; Morley, Chris; Vandermeersch, Tim; Hutchison, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A frequent problem in computational modeling is the interconversion of chem. structures between different formats.  While std. interchange formats exist (for example, Chem. Markup Language) and de facto stds. have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chem. data, differences in the data stored by different formats (0D vs. 3D, for example), and competition between software along with a lack of vendor-neutral formats.  Results: We discuss, for the first time, Open Babel, an open-source chem. toolbox that speaks the many languages of chem. data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chem. and mol. data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  Conclusions: Open Babel presents a soln. to the proliferation of multiple chem. file formats.  In addn., it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chem. data in areas such as org. chem., drug design, materials science, and computational chem.  It is freely available under an open-source license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfQ1lvaS5767Vg90H21EOLACvtfcHk0liWOTZ0hwc2JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF&md5=74e4f19b7f87417f916d57f7abcfb761</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520an%2520open%2520chemical%2520toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':[],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 20 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yikun Chen, Yongxin Gao, Yujun He, Gang Zhang, Hui Wen, Yuchen Wang, Qin-Pei Wu, <span class="NLM_string-name hlFld-ContribAuthor">Huaqing Cui</span>. </span><span class="cited-content_cbyCitation_article-title">Determining Essential Requirements for Fluorophore Selection in Various Fluorescence Applications Taking Advantage of Diverse Structure–Fluorescence Information of Chromone Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (2)
                                     , 1001-1017. <a href="https://doi.org/10.1021/acs.jmedchem.0c01508" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01508</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01508%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDetermining%252BEssential%252BRequirements%252Bfor%252BFluorophore%252BSelection%252Bin%252BVarious%252BFluorescence%252BApplications%252BTaking%252BAdvantage%252Bof%252BDiverse%252BStructure%2525E2%252580%252593Fluorescence%252BInformation%252Bof%252BChromone%252BDerivatives%26aulast%3DChen%26aufirst%3DYikun%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D29082020%26date%3D14122020%26volume%3D64%26issue%3D2%26spage%3D1001%26epage%3D1017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qi-Yin  Chen</span>, <span class="hlFld-ContribAuthor ">Pravin R.  Chaturvedi</span>, and <span class="hlFld-ContribAuthor ">Hendrik  Luesch</span>  . </span><span class="cited-content_cbyCitation_article-title">Process Development and Scale-up Total Synthesis of Largazole, a Potent Class I Histone Deacetylase Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Organic Process Research & Development</span><span> <strong>2018,</strong> <em>22 </em>
                                    (2)
                                     , 190-199. <a href="https://doi.org/10.1021/acs.oprd.7b00352" title="DOI URL">https://doi.org/10.1021/acs.oprd.7b00352</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.oprd.7b00352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.oprd.7b00352%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Process%2520Research%2520%2526%2520Development%26atitle%3DProcess%252BDevelopment%252Band%252BScale-up%252BTotal%252BSynthesis%252Bof%252BLargazole%25252C%252Ba%252BPotent%252BClass%252BI%252BHistone%252BDeacetylase%252BInhibitor%26aulast%3DChen%26aufirst%3DQi-Yin%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D02112017%26date%3D30012018%26date%3D16022018%26volume%3D22%26issue%3D2%26spage%3D190%26epage%3D199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jehad  Almaliti</span>, <span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Ahmed T.  Negmeldin</span>, <span class="hlFld-ContribAuthor ">Christin L.  Hanigan</span>, <span class="hlFld-ContribAuthor ">Lalith  Perera</span>, <span class="hlFld-ContribAuthor ">Mary Kay H.  Pflum</span>, <span class="hlFld-ContribAuthor ">Robert A.  Casero, Jr.</span>, and <span class="hlFld-ContribAuthor ">L. M. Viranga  Tillekeratne</span>  . </span><span class="cited-content_cbyCitation_article-title">Largazole Analogues Embodying Radical Changes in the Depsipeptide Ring: Development of a More Selective and Highly Potent Analogue. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (23)
                                     , 10642-10660. <a href="https://doi.org/10.1021/acs.jmedchem.6b01271" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01271</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01271%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLargazole%252BAnalogues%252BEmbodying%252BRadical%252BChanges%252Bin%252Bthe%252BDepsipeptide%252BRing%25253A%252BDevelopment%252Bof%252Ba%252BMore%252BSelective%252Band%252BHighly%252BPotent%252BAnalogue%26aulast%3DAlmaliti%26aufirst%3DJehad%26date%3D2016%26date%3D2016%26date%3D2016%26date%3D12092016%26date%3D21112016%26date%3D08122016%26date%3D03112016%26volume%3D59%26issue%3D23%26spage%3D10642%26epage%3D10660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Flavio  Ballante</span> and <span class="hlFld-ContribAuthor ">Garland R.  Marshall</span>  . </span><span class="cited-content_cbyCitation_article-title">An Automated Strategy for Binding-Pose Selection and Docking Assessment in Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (1)
                                     , 54-72. <a href="https://doi.org/10.1021/acs.jcim.5b00603" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00603</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00603%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAn%252BAutomated%252BStrategy%252Bfor%252BBinding-Pose%252BSelection%252Band%252BDocking%252BAssessment%252Bin%252BStructure-Based%252BDrug%252BDesign%26aulast%3DBallante%26aufirst%3DFlavio%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D01102015%26date%3D06012016%26date%3D25012016%26date%3D18122015%26volume%3D56%26issue%3D1%26spage%3D54%26epage%3D72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bingbing  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhu-Wei  Ruan</span>, <span class="hlFld-ContribAuthor ">Dongdong  Luo</span>, <span class="hlFld-ContribAuthor ">Yueyue  Zhu</span>, <span class="hlFld-ContribAuthor ">Tingbo  Ding</span>, <span class="hlFld-ContribAuthor ">Qiang  Sui</span>, <span class="hlFld-ContribAuthor ">Xinsheng  Lei</span>. </span><span class="cited-content_cbyCitation_article-title">Unexpected Enhancement of HDACs Inhibition by MeS Substitution at C-2 Position of Fluoro Largazole. </span><span class="cited-content_cbyCitation_journal-name">Marine Drugs</span><span> <strong>2020,</strong> <em>18 </em>
                                    (7)
                                     , 344. <a href="https://doi.org/10.3390/md18070344" title="DOI URL">https://doi.org/10.3390/md18070344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/md18070344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmd18070344%26sid%3Dliteratum%253Aachs%26jtitle%3DMarine%2520Drugs%26atitle%3DUnexpected%252BEnhancement%252Bof%252BHDACs%252BInhibition%252Bby%252BMeS%252BSubstitution%252Bat%252BC-2%252BPosition%252Bof%252BFluoro%252BLargazole%26aulast%3DZhang%26aufirst%3DBingbing%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D7%26spage%3D344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Loredana  Cappellacci</span>, <span class="hlFld-ContribAuthor ">Diego R.  Perinelli</span>, <span class="hlFld-ContribAuthor ">Filippo  Maggi</span>, <span class="hlFld-ContribAuthor ">Mario  Grifantini</span>, <span class="hlFld-ContribAuthor ">Riccardo  Petrelli</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Progress in Histone Deacetylase Inhibitors as Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (15)
                                     , 2449-2493. <a href="https://doi.org/10.2174/0929867325666181016163110" title="DOI URL">https://doi.org/10.2174/0929867325666181016163110</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666181016163110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666181016163110%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BProgress%252Bin%252BHistone%252BDeacetylase%252BInhibitors%252Bas%252BAnticancer%252BAgents%26aulast%3DCappellacci%26aufirst%3DLoredana%26date%3D2020%26volume%3D27%26issue%3D15%26spage%3D2449%26epage%3D2493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bingbing  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiami  Liu</span>, <span class="hlFld-ContribAuthor ">Dingding  Gao</span>, <span class="hlFld-ContribAuthor ">Xiaolin  Yu</span>, <span class="hlFld-ContribAuthor ">Jinlei  Wang</span>, <span class="hlFld-ContribAuthor ">Xinsheng  Lei</span>. </span><span class="cited-content_cbyCitation_article-title">A fluorine scan on the Zn2+-binding thiolate side chain of HDAC inhibitor largazole: Synthesis, biological evaluation, and molecular modeling. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>182 </em>, 111672. <a href="https://doi.org/10.1016/j.ejmech.2019.111672" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111672</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111672%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252Bfluorine%252Bscan%252Bon%252Bthe%252BZn2%25252B-binding%252Bthiolate%252Bside%252Bchain%252Bof%252BHDAC%252Binhibitor%252Blargazole%25253A%252BSynthesis%25252C%252Bbiological%252Bevaluation%25252C%252Band%252Bmolecular%252Bmodeling%26aulast%3DZhang%26aufirst%3DBingbing%26date%3D2019%26volume%3D182%26spage%3D111672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bingbing  Zhang</span>, <span class="hlFld-ContribAuthor ">Guangsheng  Shan</span>, <span class="hlFld-ContribAuthor ">Yinying  Zheng</span>, <span class="hlFld-ContribAuthor ">Xiaolin  Yu</span>, <span class="hlFld-ContribAuthor ">Zhu-Wei  Ruan</span>, <span class="hlFld-ContribAuthor ">Yang  Li</span>, <span class="hlFld-ContribAuthor ">Xinsheng  Lei</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Preliminary Biological Evaluation of Two Fluoroolefin Analogs of Largazole Inspired by the Structural Similarity of the Side Chain Unit in Psammaplin A. </span><span class="cited-content_cbyCitation_journal-name">Marine Drugs</span><span> <strong>2019,</strong> <em>17 </em>
                                    (6)
                                     , 333. <a href="https://doi.org/10.3390/md17060333" title="DOI URL">https://doi.org/10.3390/md17060333</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/md17060333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmd17060333%26sid%3Dliteratum%253Aachs%26jtitle%3DMarine%2520Drugs%26atitle%3DSynthesis%252Band%252BPreliminary%252BBiological%252BEvaluation%252Bof%252BTwo%252BFluoroolefin%252BAnalogs%252Bof%252BLargazole%252BInspired%252Bby%252Bthe%252BStructural%252BSimilarity%252Bof%252Bthe%252BSide%252BChain%252BUnit%252Bin%252BPsammaplin%252BA%26aulast%3DZhang%26aufirst%3DBingbing%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D6%26spage%3D333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Philipp  Ermert</span>, <span class="hlFld-ContribAuthor ">Anatol  Luther</span>, <span class="hlFld-ContribAuthor ">Peter  Zbinden</span>, <span class="hlFld-ContribAuthor ">Daniel  Obrecht</span>. </span><span class="cited-content_cbyCitation_article-title">Frontier Between Cyclic Peptides and Macrocycles. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 147-202. <a href="https://doi.org/10.1007/978-1-4939-9504-2_9" title="DOI URL">https://doi.org/10.1007/978-1-4939-9504-2_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-9504-2_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-9504-2_9%26sid%3Dliteratum%253Aachs%26atitle%3DFrontier%252BBetween%252BCyclic%252BPeptides%252Band%252BMacrocycles%26aulast%3DErmert%26aufirst%3DPhilipp%26date%3D2019%26date%3D2019%26spage%3D147%26epage%3D202%26pub%3DSpringer%2520New%2520York%26atitle%3DCyclic%252BPeptide%252BDesign%26aulast%3DGoetz%26aufirst%3DGilles%26date%3D2019%26volume%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoling  Yu</span>, <span class="hlFld-ContribAuthor ">Bingbing  Zhang</span>, <span class="hlFld-ContribAuthor ">Guangsheng  Shan</span>, <span class="hlFld-ContribAuthor ">Yue  Wu</span>, <span class="hlFld-ContribAuthor ">Feng-Ling  Yang</span>, <span class="hlFld-ContribAuthor ">Xinsheng  Lei</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of the molecular hybrid inspired by Largazole and Psammaplin A. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2018,</strong> <em>74 </em>
                                    (5)
                                     , 549-555. <a href="https://doi.org/10.1016/j.tet.2017.12.025" title="DOI URL">https://doi.org/10.1016/j.tet.2017.12.025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2017.12.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2017.12.025%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DSynthesis%252Bof%252Bthe%252Bmolecular%252Bhybrid%252Binspired%252Bby%252BLargazole%252Band%252BPsammaplin%252BA%26aulast%3DYu%26aufirst%3DXiaoling%26date%3D2018%26volume%3D74%26issue%3D5%26spage%3D549%26epage%3D555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John W.  Blunt</span>, <span class="hlFld-ContribAuthor ">Anthony R.  Carroll</span>, <span class="hlFld-ContribAuthor ">Brent R.  Copp</span>, <span class="hlFld-ContribAuthor ">Rohan A.  Davis</span>, <span class="hlFld-ContribAuthor ">Robert A.  Keyzers</span>, <span class="hlFld-ContribAuthor ">Michèle R.  Prinsep</span>. </span><span class="cited-content_cbyCitation_article-title">Marine natural products. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2018,</strong> <em>35 </em>
                                    (1)
                                     , 8-53. <a href="https://doi.org/10.1039/C7NP00052A" title="DOI URL">https://doi.org/10.1039/C7NP00052A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7NP00052A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7NP00052A%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DMarine%252Bnatural%252Bproducts%26aulast%3DBlunt%26aufirst%3DJohn%2BW.%26date%3D2018%26date%3D2018%26volume%3D35%26issue%3D1%26spage%3D8%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulio  Poli</span>, <span class="hlFld-ContribAuthor ">Romano  Di Fabio</span>, <span class="hlFld-ContribAuthor ">Luca  Ferrante</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Summa</span>, <span class="hlFld-ContribAuthor ">Maurizio  Botta</span>. </span><span class="cited-content_cbyCitation_article-title">Largazole Analogues as Histone Deacetylase Inhibitors and Anticancer Agents: An Overview of Structure-Activity Relationships. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (23)
                                     , 1917-1926. <a href="https://doi.org/10.1002/cmdc.201700563" title="DOI URL">https://doi.org/10.1002/cmdc.201700563</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700563%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DLargazole%252BAnalogues%252Bas%252BHistone%252BDeacetylase%252BInhibitors%252Band%252BAnticancer%252BAgents%25253A%252BAn%252BOverview%252Bof%252BStructure-Activity%252BRelationships%26aulast%3DPoli%26aufirst%3DGiulio%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D23%26spage%3D1917%26epage%3D1926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brice J.  Albert</span>, <span class="hlFld-ContribAuthor ">Austin  Niu</span>, <span class="hlFld-ContribAuthor ">Rashmi  Ramani</span>, <span class="hlFld-ContribAuthor ">Garland R.  Marshall</span>, <span class="hlFld-ContribAuthor ">Paul A.  Wender</span>, <span class="hlFld-ContribAuthor ">Robert M.  Williams</span>, <span class="hlFld-ContribAuthor ">Lee  Ratner</span>, <span class="hlFld-ContribAuthor ">Alexander B.  Barnes</span>, <span class="hlFld-ContribAuthor ">George B.  Kyei</span>. </span><span class="cited-content_cbyCitation_article-title">Combinations of isoform-targeted histone deacetylase inhibitors and bryostatin analogues display remarkable potency to activate latent HIV without global T-cell activation. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-07814-4" title="DOI URL">https://doi.org/10.1038/s41598-017-07814-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-07814-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-07814-4%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DCombinations%252Bof%252Bisoform-targeted%252Bhistone%252Bdeacetylase%252Binhibitors%252Band%252Bbryostatin%252Banalogues%252Bdisplay%252Bremarkable%252Bpotency%252Bto%252Bactivate%252Blatent%252BHIV%252Bwithout%252Bglobal%252BT-cell%252Bactivation%26aulast%3DAlbert%26aufirst%3DBrice%2BJ.%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lik Tong  Tan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Targets of Clinically Relevant Natural Products from Filamentous Marine Cyanobacteria. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 19-43. <a href="https://doi.org/10.1002/9783527805921.ch2" title="DOI URL">https://doi.org/10.1002/9783527805921.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527805921.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527805921.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252BTargets%252Bof%252BClinically%252BRelevant%252BNatural%252BProducts%252Bfrom%252BFilamentous%252BMarine%252BCyanobacteria%26aulast%3DTan%26aufirst%3DLik%2BTong%26date%3D2017%26date%3D2017%26spage%3D19%26epage%3D43%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DNatural%252BProducts%252BTargeting%252BClinically%252BRelevant%252BEnzymes%26aulast%3DAndrade%26aufirst%3DPaula%2BB.%26date%3D2017%26volume%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Garland R.  Marshall</span>, <span class="hlFld-ContribAuthor ">Flavio  Ballante</span>. </span><span class="cited-content_cbyCitation_article-title">Limiting Assumptions in the Design of Peptidomimetics. </span><span class="cited-content_cbyCitation_journal-name">Drug Development Research</span><span> <strong>2017,</strong> <em>78 </em>
                                    (6)
                                     , 245-267. <a href="https://doi.org/10.1002/ddr.21406" title="DOI URL">https://doi.org/10.1002/ddr.21406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ddr.21406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fddr.21406%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Development%2520Research%26atitle%3DLimiting%252BAssumptions%252Bin%252Bthe%252BDesign%252Bof%252BPeptidomimetics%26aulast%3DMarshall%26aufirst%3DGarland%2BR.%26date%3D2017%26date%3D2017%26volume%3D78%26issue%3D6%26spage%3D245%26epage%3D267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Flavio  Ballante</span>, <span class="hlFld-ContribAuthor ">D. Rajasekhar  Reddy</span>, <span class="hlFld-ContribAuthor ">Nancy J.  Zhou</span>, <span class="hlFld-ContribAuthor ">Garland R.  Marshall</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insights of SmKDAC8 inhibitors: Targeting Schistosoma epigenetics through a combined structure-based 3D QSAR, in vitro and synthesis strategy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (7)
                                     , 2105-2132. <a href="https://doi.org/10.1016/j.bmc.2017.02.020" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.02.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.02.020%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructural%252Binsights%252Bof%252BSmKDAC8%252Binhibitors%25253A%252BTargeting%252BSchistosoma%252Bepigenetics%252Bthrough%252Ba%252Bcombined%252Bstructure-based%252B3D%252BQSAR%25252C%252Bin%252Bvitro%252Band%252Bsynthesis%252Bstrategy%26aulast%3DBallante%26aufirst%3DFlavio%26date%3D2017%26volume%3D25%26issue%3D7%26spage%3D2105%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D. Rajasekhar  Reddy</span>, <span class="hlFld-ContribAuthor ">Flavio  Ballante</span>, <span class="hlFld-ContribAuthor ">Nancy J.  Zhou</span>, <span class="hlFld-ContribAuthor ">Garland R.  Marshall</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of benzodiazepine analogs as isoform-selective human lysine deacetylase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>127 </em>, 531-553. <a href="https://doi.org/10.1016/j.ejmech.2016.12.032" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.12.032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.12.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.12.032%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bbenzodiazepine%252Banalogs%252Bas%252Bisoform-selective%252Bhuman%252Blysine%252Bdeacetylase%252Binhibitors%26aulast%3DReddy%26aufirst%3DD.%2BRajasekhar%26date%3D2017%26volume%3D127%26spage%3D531%26epage%3D553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alex R.  Maolanon</span>, <span class="hlFld-ContribAuthor ">Helle M. E.  Kristensen</span>, <span class="hlFld-ContribAuthor ">Luke J.  Leman</span>, <span class="hlFld-ContribAuthor ">M. Reza  Ghadiri</span>, <span class="hlFld-ContribAuthor ">Christian A.  Olsen</span>. </span><span class="cited-content_cbyCitation_article-title">Natural and Synthetic Macrocyclic Inhibitors of the Histone Deacetylase Enzymes. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2017,</strong> <em>18 </em>
                                    (1)
                                     , 5-49. <a href="https://doi.org/10.1002/cbic.201600519" title="DOI URL">https://doi.org/10.1002/cbic.201600519</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201600519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201600519%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DNatural%252Band%252BSynthetic%252BMacrocyclic%252BInhibitors%252Bof%252Bthe%252BHistone%252BDeacetylase%252BEnzymes%26aulast%3DMaolanon%26aufirst%3DAlex%2BR.%26date%3D2017%26date%3D2016%26volume%3D18%26issue%3D1%26spage%3D5%26epage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayuki  Shioiri</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Development of Diphenyl Phosphorazidate (DPPA) as a Synthetic Reagent. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 131-145. <a href="https://doi.org/10.1007/978-981-10-3421-3_10" title="DOI URL">https://doi.org/10.1007/978-981-10-3421-3_10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-10-3421-3_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-10-3421-3_10%26sid%3Dliteratum%253Aachs%26atitle%3DRecent%252BDevelopment%252Bof%252BDiphenyl%252BPhosphorazidate%252B%252528DPPA%252529%252Bas%252Ba%252BSynthetic%252BReagent%26aulast%3DShioiri%26aufirst%3DTakayuki%26date%3D2017%26date%3D2017%26spage%3D131%26epage%3D145%26pub%3DSpringer%2520Singapore%26atitle%3DNew%252BHorizons%252Bof%252BProcess%252BChemistry%26aulast%3DTomioka%26aufirst%3DKiyoshi%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John A.  Halsall</span>, <span class="hlFld-ContribAuthor ">Bryan M.  Turner</span>. </span><span class="cited-content_cbyCitation_article-title">Histone deacetylase inhibitors for cancer therapy: An evolutionarily ancient resistance response may explain their limited success. </span><span class="cited-content_cbyCitation_journal-name">BioEssays</span><span> <strong>2016,</strong> <em>38 </em>
                                    (11)
                                     , 1102-1110. <a href="https://doi.org/10.1002/bies.201600070" title="DOI URL">https://doi.org/10.1002/bies.201600070</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bies.201600070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbies.201600070%26sid%3Dliteratum%253Aachs%26jtitle%3DBioEssays%26atitle%3DHistone%252Bdeacetylase%252Binhibitors%252Bfor%252Bcancer%252Btherapy%25253A%252BAn%252Bevolutionarily%252Bancient%252Bresistance%252Bresponse%252Bmay%252Bexplain%252Btheir%252Blimited%252Bsuccess%26aulast%3DHalsall%26aufirst%3DJohn%2BA.%26date%3D2016%26date%3D2016%26volume%3D38%26issue%3D11%26spage%3D1102%26epage%3D1110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Examples of potent KDAC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Schematic of KDAC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Original docking of largazole to KDAC8 (left) compared with crystal structure of largazole in complex; (right) docked structure of simplified largazole analogue overlapped with crystal structure of largazole when complexed to KDAC8.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Designing of selective KDAC inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of CTP Reverse-Turn Scaffolds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) EDCl, HOBt, DIEA, DCM, 9 h; (ii) 20% piperidine/DCM, rt, 0.5 h, 100%; (iii) Fmoc-Xaa-OH, EDCl, HOBt, DIEA, DCM, 12 h; (iv) Boc-Zaa-OBn, EDCl, HOBt, DIEA, DCM, 24 h; (v) 30% TFA/DCM, 0 °C–rt, 3 h, 100%; (vi) DPPA, DIEA,DMF, 0 °C–rt, 72 h; (v) H<sub>2</sub>, Pd/C, EtOH, rt, 5–6 h.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Introduction of Sulfhydryl Zinc-Binding Functional Group on CTP Reverse-Turn Scaffolds</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Introduction of Thiol, Thioether, Sulfoxide, and Sulfone Zinc-Binding Groups</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Introduction of Carboxylic Acid, Ester, and Hydroxamate Zinc-Binding Groups</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Simplified largazole (<span class="smallcaps smallerCapital">d</span>-pro-Pro) analogues inhibition profiles plots. (A) Asp derivatives; (B) <span class="smallcaps smallerCapital">d</span>-asp derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Simplified largazole (Pro-<span class="smallcaps smallerCapital">d</span>-pro) analogues inhibition profiles plots. (A) Asp derivatives; (B) <span class="smallcaps smallerCapital">d</span>-asp derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Docking of largazole in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a><a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> chain (A). Best Docked poses (BD): (A) experimental conformation redocking (ECRD); (B) random conformation redocking (RCRD). Experimental binding pose is green colored, BD predicted binding pose is magenta colored.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/medium/jm-2015-01632f_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Comparison between largazole thiol (green) experimental binding pose and SLAs <b>14a</b> and <b>25a</b> BD poses in KDAC8 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD">3RQD</a>). (A) Predicted BD poses of compound <b>14a</b> (Asp-<span class="smallcaps smallerCapital">d</span>-pro-Pro, cyan); (B) predicted BD poses of compound <b>25a</b> (Asp-Pro-<span class="smallcaps smallerCapital">d</span>-pro, gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01632/20160219/images/large/jm-2015-01632f_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01632&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i96">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 77 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="note"><p class="first last">Lysine deacetylases are generally referred to as histone deacetylases (HDACs), a historical imperative as epigenetic modification of histones was described in 1964 by Allfrey et al.,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> thus, the terms histone acetylases (HATs), deacetylases(HDACs), methylases (HMTs), etc. It has been shown by proteomics, however, that over 1700 proteins in cells besides histones undergo dynamic acetylation<sup>3</sup>, thus, the current preference for the term lysine acetylase (KAT), deacetylase (KDAC), or methylase (KMT).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Allfrey, V. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faulkner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirsky, A. E.</span><span> </span><span class="NLM_article-title">Acetylation and methylation of histones and their possible role in the regulation of RNA synthesis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1964</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">794</span><span class="refDoi"> DOI: 10.1073/pnas.51.5.786</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.51.5.786" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=14172992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1964&pages=786-794&author=V.+G.+Allfreyauthor=R.+Faulknerauthor=A.+E.+Mirsky&title=Acetylation+and+methylation+of+histones+and+their+possible+role+in+the+regulation+of+RNA+synthesis&doi=10.1073%2Fpnas.51.5.786"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acetylation and methylation of histones and their possible role in the regulation of ribonucleic acid (RNA) synthesis</span></div><div class="casAuthors">Allfrey, V. G.; Faulkner, R.; Mirsky, A. E.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">786-94</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Isolated calf thymus nuclei was incubated with 14C-labeled Na acetate.  Evidence that acetate was incorporated as acetyl groups attached to histones included: (1) following chromatography the label was eluted together with basic proteins; (2) acetyl-labeled histones were sepd. by electrophoresis; (3) acetate-14C was not removed from histones by org. solvents; (4) over 75% of the label remained after histones were treated with trichloroacetic acid.  Most extensive acetate uptake occurred with arginine-rich histones.  Puromycin did not inhibit the acetylation, indicating histones were modified by acetylation after completion of the polypeptide chain.  Acetylation lowered the histone effectiveness as inhibitor of the RNA polymerase reaction, and a dynamic and reversible mechanism was suggested for activation as well as repression of RNA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpL5QIm_JdO1bVg90H21EOLACvtfcHk0ljK9OssgJ1eeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXktlekurg%253D&md5=7a80f7a8a12636d4bc47930ee5eff217</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.51.5.786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.51.5.786%26sid%3Dliteratum%253Aachs%26aulast%3DAllfrey%26aufirst%3DV.%2BG.%26aulast%3DFaulkner%26aufirst%3DR.%26aulast%3DMirsky%26aufirst%3DA.%2BE.%26atitle%3DAcetylation%2520and%2520methylation%2520of%2520histones%2520and%2520their%2520possible%2520role%2520in%2520the%2520regulation%2520of%2520RNA%2520synthesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1964%26volume%3D51%26spage%3D786%26epage%3D794%26doi%3D10.1073%2Fpnas.51.5.786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Choudhary, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnad, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rehman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walther, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, J. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span> </span><span class="NLM_article-title">Lysine acetylation targets protein complexes and co-regulates major cellular functions</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">834</span><span class="NLM_x">–</span> <span class="NLM_lpage">840</span><span class="refDoi"> DOI: 10.1126/science.1175371</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1126%2Fscience.1175371" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19608861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=834-840&author=C.+Choudharyauthor=C.+Kumarauthor=F.+Gnadauthor=M.+L.+Nielsenauthor=M.+Rehmanauthor=T.+C.+Waltherauthor=J.+V.+Olsenauthor=M.+Mann&title=Lysine+acetylation+targets+protein+complexes+and+co-regulates+major+cellular+functions&doi=10.1126%2Fscience.1175371"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Acetylation Targets Protein Complexes and Co-Regulates Major Cellular Functions</span></div><div class="casAuthors">Choudhary, Chunaram; Kumar, Chanchal; Gnad, Florian; Nielsen, Michael L.; Rehman, Michael; Walther, Tobias C.; Olsen, Jesper V.; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5942</span>),
    <span class="NLM_cas:pages">834-840</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Lysine acetylation is a reversible posttranslational modification of proteins and plays a key role in regulating gene expression.  Technol. limitations have so far prevented a global anal. of lysine acetylation's cellular roles.  We used high-resoln. mass spectrometry to identify 3600 lysine acetylation sites on 1750 proteins and quantified acetylation changes in response to the deacetylase inhibitors suberoylanilide hydroxamic acid and MS-275.  Lysine acetylation preferentially targets large macromol. complexes involved in diverse cellular processes, such as chromatin remodeling, cell cycle, splicing, nuclear transport, and actin nucleation.  Acetylation impaired phosphorylation-dependent interactions of 14-3-3 and regulated the yeast cyclin-dependent kinase Cdc28.  Our data demonstrate that the regulatory scope of lysine acetylation is broad and comparable with that of other major posttranslational modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5eICW58ngkrVg90H21EOLACvtfcHk0ljdRujaPSVW9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Ogt70%253D&md5=8f643db5ba91c50f0b61763ff3095d2c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1175371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1175371%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DC.%26aulast%3DKumar%26aufirst%3DC.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DNielsen%26aufirst%3DM.%2BL.%26aulast%3DRehman%26aufirst%3DM.%26aulast%3DWalther%26aufirst%3DT.%2BC.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DLysine%2520acetylation%2520targets%2520protein%2520complexes%2520and%2520co-regulates%2520major%2520cellular%2520functions%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D834%26epage%3D840%26doi%3D10.1126%2Fscience.1175371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jursic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Civoli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breslow, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rifkind, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, P. A.</span><span> </span><span class="NLM_article-title">Second generation hybrid polar compounds are potent inducers of transformed cell differentiation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">5705</span><span class="NLM_x">–</span> <span class="NLM_lpage">5708</span><span class="refDoi"> DOI: 10.1073/pnas.93.12.5705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.93.12.5705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=8650156" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK28Xjs1Kqtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=5705-5708&author=V.+M.+Richonauthor=Y.+Webbauthor=R.+Mergerauthor=T.+Sheppardauthor=B.+Jursicauthor=L.+Ngoauthor=F.+Civoliauthor=R.+Breslowauthor=R.+A.+Rifkindauthor=P.+A.+Marks&title=Second+generation+hybrid+polar+compounds+are+potent+inducers+of+transformed+cell+differentiation&doi=10.1073%2Fpnas.93.12.5705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Second generation hybrid polar compounds are potent inducers of transformed cell differentiation</span></div><div class="casAuthors">Richon, V. M.; Webb, Y.; Merger, R.; Sheppard, T.; Jursic, B.; Ngo, L.; Civoli, F.; Breslow, R.; Rifkind, R. A.; Marks, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5705-5708</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hybrid polar compds., of which hexamethylenebisacetamide (HMBA) is the prototype, are potent inducers of differentiation of murine erythroleukemia (MEL) cells and a wide variety of other transformed cells.  HMBA has been shown to induce differentiation of neoplastic cells of dose-limiting toxicity.  The authors report on a group of three potent second generation hybrid polar compds., di-Et bis-(pentamethylene-N,N-dimethylcarboxamide)malonate (EMBA), suberoylanilide hydroxamic acid (SAHA), and m-carboxycinnamic acid bis-hydroxamide (CBHA) with optimal concns. for inducing MEL cells of 0.4 mM, 2 μM, and 4 μM, resp., compared to 5 mM for HMBA.  All three agents induce accumulation of underphosphorylated pRB; increased levels of p21 protein, a prolongation of the initial G1 phase of the cell cycle; and accumulation of Hb.  However, based upon their effective concns., the cross-resistance or sensitivity of an HMBA-resistant MEL cell variant, and differences in c-myb expression during induction, these differentiation-inducing hybrid polar compds. can be grouped into two subsets, HMBA/EMBA and SAHA/CBHA.  This classification may prove of value in selecting and planning prospective preclin. and clin. studies toward the treatment of cancer by differentiation therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrHTaRYYg617Vg90H21EOLACvtfcHk0lgqb-R0NTK9Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xjs1Kqtbc%253D&md5=ffe9e6c95b25b85a50a89201f0f5e5e1</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.12.5705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.12.5705%26sid%3Dliteratum%253Aachs%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DWebb%26aufirst%3DY.%26aulast%3DMerger%26aufirst%3DR.%26aulast%3DSheppard%26aufirst%3DT.%26aulast%3DJursic%26aufirst%3DB.%26aulast%3DNgo%26aufirst%3DL.%26aulast%3DCivoli%26aufirst%3DF.%26aulast%3DBreslow%26aufirst%3DR.%26aulast%3DRifkind%26aufirst%3DR.%2BA.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26atitle%3DSecond%2520generation%2520hybrid%2520polar%2520compounds%2520are%2520potent%2520inducers%2520of%2520transformed%2520cell%2520differentiation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D5705%26epage%3D5708%26doi%3D10.1073%2Fpnas.93.12.5705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Furumai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuyama, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobashi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakajima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishino, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horinouchi, S.</span><span> </span><span class="NLM_article-title">FK228 (depsipeptide) as a natural prodrug that inhibits class I histone deacetylases</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">4916</span><span class="NLM_x">–</span> <span class="NLM_lpage">4921</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=12208741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD38Xmslylu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=4916-4921&author=R.+Furumaiauthor=A.+Matsuyamaauthor=N.+Kobashiauthor=K.+H.+Leeauthor=M.+Nishiyamaauthor=H.+Nakajimaauthor=A.+Tanakaauthor=Y.+Komatsuauthor=N.+Nishinoauthor=M.+Yoshidaauthor=S.+Horinouchi&title=FK228+%28depsipeptide%29+as+a+natural+prodrug+that+inhibits+class+I+histone+deacetylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">FK228 (depsipeptide) as a natural prodrug that inhibits class I histone deacetylases</span></div><div class="casAuthors">Furumai, Ryohei; Matsuyama, Akihisa; Kobashi, Nobuyuki; Lee, Kun-Hyung; Nishiyama, Makoto; Nakajima, Hidenori; Tanaka, Akito; Komatsu, Yasuhiko; Nishino, Norikazu; Yoshida, Minoru; Horinouchi, Sueharu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4916-4921</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">FK228 is a histone deacetylase (HDAC) inhibitor, the mol. mechanism of inhibition of which has been unknown.  Here we show that redn. of an intramol. disulfide bond of FK228 greatly enhanced its inhibitory activity and that the disulfide bond was rapidly reduced in cells by cellular reducing activity involving glutathione.  Computer modeling suggests that one of the sulfhydryl groups of the reduced form of FK228 (redFK) interacts with the active-site zinc, preventing the access of the substrate.  HDAC1 and HDAC2 were more strongly inhibited by redFK than HDAC4 and HDAC6.  RedFK was less active than FK228 in inhibiting in vivo HDAC activity, due to rapid inactivation in medium and serum.  Thus, FK228 serves as a stable prodrug to inhibit class I enzymes and is activated by redn. after uptake into the cells.  The glutathione-mediated activation also implicates its clin. usefulness for counteracting glutathione-mediated drug resistance in chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohcl-TJJMBcbVg90H21EOLACvtfcHk0lgqb-R0NTK9Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xmslylu78%253D&md5=691ed84f8a35b1664b4bab770d69a194</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFurumai%26aufirst%3DR.%26aulast%3DMatsuyama%26aufirst%3DA.%26aulast%3DKobashi%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DNishiyama%26aufirst%3DM.%26aulast%3DNakajima%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DA.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DNishino%26aufirst%3DN.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DHorinouchi%26aufirst%3DS.%26atitle%3DFK228%2520%2528depsipeptide%2529%2520as%2520a%2520natural%2520prodrug%2520that%2520inhibits%2520class%2520I%2520histone%2520deacetylases%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D4916%26epage%3D4921" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Plumb, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandara, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romero, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watkins, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R.</span><span> </span><span class="NLM_article-title">Pharmacodynamic response and inhibition of growth of human tumor xenografts by the novel histone deacetylase inhibitor PXD101</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">721</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=12939461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3sXms1GktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=721-728&author=J.+A.+Plumbauthor=P.+W.+Finnauthor=R.+J.+Williamsauthor=M.+J.+Bandaraauthor=M.+R.+Romeroauthor=C.+J.+Watkinsauthor=N.+B.+La+Thangueauthor=R.+Brown&title=Pharmacodynamic+response+and+inhibition+of+growth+of+human+tumor+xenografts+by+the+novel+histone+deacetylase+inhibitor+PXD101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacodynamic Response and Inhibition of Growth of Human Tumor Xenografts by the Novel Histone Deacetylase Inhibitor PXD101</span></div><div class="casAuthors">Plumb, Jane A.; Finn, Paul W.; Williams, Robert J.; Bandara, Morwenna J.; Romero, M. Rosario; Watkins, Claire J.; La Thangue, Nicholas B.; Brown, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">721-728</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone acetylation has a central role in the control of gene expression, influencing transcriptional control of many genes, including tumor suppressor genes.  PXD101 is a novel hydroxamate-type inhibitor of histone deacetylase activity that inhibits histone deacetylase activity in HeLa cell exts. with an IC50 of 27 nM and induces a concn.-dependent (0.2-5 μM) increase in acetylation of histone H4 in tumor cell lines.  PXD101 is cytotoxic in vitro in a no. of tumor cell lines with IC50s in the range 0.2-3.4 μM as detd. by a clonogenic assay and induces apoptosis.  Treatment of nude mice bearing human ovarian and colon tumor xenografts with PXD101 (10-40 mg/kg/day i.p.) daily for 7 days causes a significant dose-dependent growth delay with no obvious signs of toxicity to the mice.  Growth delay is also obsd. for xenografts of cisplatin-resistant ovarian tumor cells.  A marked increase in acetylation of H4 is detected in blood and tumor of mice 3 h after treatment with PXD101.  The inhibition of growth of human tumor xenografts in mice, with no apparent toxicity, suggests that PXD101 has potential as a novel antitumor agent.  Furthermore, the ability to measure histone acetylation in blood samples could provide a suitable pharmacodynamic end point to monitor drug activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCqzC5_Q06W7Vg90H21EOLACvtfcHk0lhFXaZkrqzxHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXms1GktLo%253D&md5=7d25952434f0e544d8b61ab20ae5b0e4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPlumb%26aufirst%3DJ.%2BA.%26aulast%3DFinn%26aufirst%3DP.%2BW.%26aulast%3DWilliams%26aufirst%3DR.%2BJ.%26aulast%3DBandara%26aufirst%3DM.%2BJ.%26aulast%3DRomero%26aufirst%3DM.%2BR.%26aulast%3DWatkins%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DBrown%26aufirst%3DR.%26atitle%3DPharmacodynamic%2520response%2520and%2520inhibition%2520of%2520growth%2520of%2520human%2520tumor%2520xenografts%2520by%2520the%2520novel%2520histone%2520deacetylase%2520inhibitor%2520PXD101%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2003%26volume%3D2%26spage%3D721%26epage%3D728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Taube, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterlin, M.</span><span> </span><span class="NLM_article-title">Lost in transcription: molecular mechanisms that control HIV latency</span> <span class="citation_source-journal">Viruses</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">902</span><span class="NLM_x">–</span> <span class="NLM_lpage">927</span><span class="refDoi"> DOI: 10.3390/v5030902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.3390%2Fv5030902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=23518577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVSrtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=902-927&author=R.+Taubeauthor=M.+Peterlin&title=Lost+in+transcription%3A+molecular+mechanisms+that+control+HIV+latency&doi=10.3390%2Fv5030902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lost in transcription: molecular mechanisms that control HIV latency</span></div><div class="casAuthors">Taube, Ran; Peterlin, Boris Matija</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">902-928</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Highly active antiretroviral therapy (HAART) has limited the replication and spread of the human immunodeficiency virus (HIV).  However, despite treatment, HIV infection persists in latently infected reservoirs, and once therapy is interrupted, viral replication rebounds quickly.  Extensive efforts are being directed at eliminating these cell reservoirs.  This feat can be achieved by reactivating latent HIV while administering drugs that prevent new rounds of infection and allow the immune system to clear the virus.  However, current approaches to HIV eradication have not been effective.  Moreover, as HIV latency is multifactorial, the significance of each of its mol. mechanisms is still under debate.  Among these, transcriptional repression as a result of reduced levels and activity of the pos. transcription elongation factor b (P-TEFb: CDK9/cyclin T) plays a significant role.  Therefore, increasing levels of P-TEFb expression and activity is an excellent strategy to stimulate viral gene expression.  This review summarizes the multiple steps that cause HIV to enter into latency.  It positions the interplay between transcriptionally active and inactive host transcriptional activators and their viral partner Tat as valid targets for the development of new strategies to reactivate latent viral gene expression and eradicate HIV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP3nHdi5AyOLVg90H21EOLACvtfcHk0lhFXaZkrqzxHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVSrtLk%253D&md5=a2ba0ed16a2b7cceef4a15e355fb63d2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3390%2Fv5030902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv5030902%26sid%3Dliteratum%253Aachs%26aulast%3DTaube%26aufirst%3DR.%26aulast%3DPeterlin%26aufirst%3DM.%26atitle%3DLost%2520in%2520transcription%253A%2520molecular%2520mechanisms%2520that%2520control%2520HIV%2520latency%26jtitle%3DViruses%26date%3D2013%26volume%3D5%26spage%3D902%26epage%3D927%26doi%3D10.3390%2Fv5030902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Richman, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delaney, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pomerantz, R. J.</span><span> </span><span class="NLM_article-title">The challenge of finding a cure for HIV infection</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">323</span><span class="NLM_x">, </span> <span class="NLM_fpage">1304</span><span class="NLM_x">–</span> <span class="NLM_lpage">1307</span><span class="refDoi"> DOI: 10.1126/science.1165706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1126%2Fscience.1165706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19265012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFemur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=2009&pages=1304-1307&author=D.+D.+Richmanauthor=D.+M.+Margolisauthor=M.+Delaneyauthor=W.+C.+Greeneauthor=D.+Hazudaauthor=R.+J.+Pomerantz&title=The+challenge+of+finding+a+cure+for+HIV+infection&doi=10.1126%2Fscience.1165706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The Challenge of Finding a Cure for HIV Infection</span></div><div class="casAuthors">Richman, Douglas D.; Margolis, David M.; Delaney, Martin; Greene, Warner C.; Hazuda, Daria; Pomerantz, Roger J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">323</span>
        (<span class="NLM_cas:issue">5919</span>),
    <span class="NLM_cas:pages">1304-1307</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Although combination therapy for HIV infection represents a triumph for modern medicine, chronic suppressive therapy is required to contain persistent infection in reservoirs such as latently infected CD4+ lymphocytes and cells of the macrophage-monocyte lineage.  Despite its success, chronic suppressive therapy is limited by its cost, the requirement of lifelong adherence, and the unknown effects of long-term treatment.  This review discusses our current understanding of suppressive antiretroviral therapy, the latent viral reservoir, and the needs for and challenges of attacking this reservoir to achieve a cure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6Yfud1T9cxbVg90H21EOLACvtfcHk0lhFXaZkrqzxHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFemur0%253D&md5=855ce658e427fb8dd40925a51fdb7d20</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.1165706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1165706%26sid%3Dliteratum%253Aachs%26aulast%3DRichman%26aufirst%3DD.%2BD.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26aulast%3DDelaney%26aufirst%3DM.%26aulast%3DGreene%26aufirst%3DW.%2BC.%26aulast%3DHazuda%26aufirst%3DD.%26aulast%3DPomerantz%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520challenge%2520of%2520finding%2520a%2520cure%2520for%2520HIV%2520infection%26jtitle%3DScience%26date%3D2009%26volume%3D323%26spage%3D1304%26epage%3D1307%26doi%3D10.1126%2Fscience.1165706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Siliciano, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kajdas, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finzi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadwick, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolick, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovacs, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gange, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siliciano, R. F.</span><span> </span><span class="NLM_article-title">Long-term follow-up studies confirm the stability of the latent reservoir for HIV-1 in resting CD4+ T cells</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">728</span><span class="refDoi"> DOI: 10.1038/nm880</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fnm880" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=12754504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFOnu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=727-728&author=J.+D.+Silicianoauthor=J.+Kajdasauthor=D.+Finziauthor=T.+C.+Quinnauthor=K.+Chadwickauthor=J.+B.+Margolickauthor=C.+Kovacsauthor=S.+J.+Gangeauthor=R.+F.+Siliciano&title=Long-term+follow-up+studies+confirm+the+stability+of+the+latent+reservoir+for+HIV-1+in+resting+CD4%2B+T+cells&doi=10.1038%2Fnm880"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Long-term follow-up studies confirm the stability of the latent reservoir for HIV-1 in resting CD4+ T cells</span></div><div class="casAuthors">Siliciano, Janet D.; Kajdas, Joleen; Finzi, Diana; Quinn, Thomas C.; Chadwick, Karen; Margolick, Joseph B.; Kovacs, Colin; Gange, Stephen J.; Siliciano, Robert F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">727-728</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Latent HIV-1 persists in resting memory CD4+ T cells, even in patients receiving highly active antiretroviral therapy (HAART).  It has been unclear how stable this latent reservoir is and whether its persistence reflects replenishment by low-level viremia.  Here we show that even in treated patients who have had no detectable viremia for as long as 7 yr, the reservoir decays so slowly (t1/2 = 44 mo) that eradication is unlikely.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZzVUrIkxch7Vg90H21EOLACvtfcHk0lh7M0BBa-hc3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFOnu7k%253D&md5=c5f963cda784002f8282d3aa3b2c67a3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnm880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm880%26sid%3Dliteratum%253Aachs%26aulast%3DSiliciano%26aufirst%3DJ.%2BD.%26aulast%3DKajdas%26aufirst%3DJ.%26aulast%3DFinzi%26aufirst%3DD.%26aulast%3DQuinn%26aufirst%3DT.%2BC.%26aulast%3DChadwick%26aufirst%3DK.%26aulast%3DMargolick%26aufirst%3DJ.%2BB.%26aulast%3DKovacs%26aufirst%3DC.%26aulast%3DGange%26aufirst%3DS.%2BJ.%26aulast%3DSiliciano%26aufirst%3DR.%2BF.%26atitle%3DLong-term%2520follow-up%2520studies%2520confirm%2520the%2520stability%2520of%2520the%2520latent%2520reservoir%2520for%2520HIV-1%2520in%2520resting%2520CD4%252B%2520T%2520cells%26jtitle%3DNat.%2520Med.%26date%3D2003%26volume%3D9%26spage%3D727%26epage%3D728%26doi%3D10.1038%2Fnm880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Coiras, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Huertas, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Olmeda, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcami, J.</span><span> </span><span class="NLM_article-title">Understanding HIV-1 latency provides clues for the eradication of long-term reservoirs</span> <span class="citation_source-journal">Nat. Rev. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">798</span><span class="NLM_x">–</span> <span class="NLM_lpage">812</span><span class="refDoi"> DOI: 10.1038/nrmicro2223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fnrmicro2223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19834480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kqsb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=798-812&author=M.+Coirasauthor=M.+R.+Lopez-Huertasauthor=M.+Perez-Olmedaauthor=J.+Alcami&title=Understanding+HIV-1+latency+provides+clues+for+the+eradication+of+long-term+reservoirs&doi=10.1038%2Fnrmicro2223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding HIV-1 latency provides clues for the eradication of long-term reservoirs</span></div><div class="casAuthors">Coiras, Mayte; Lopez-Huertas, Maria Rosa; Perez-Olmeda, Mayte; Alcami, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">798-812</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  HIV-1 can infect both activated and resting, non-dividing cells, following which the viral genome can be permanently integrated into a host cell chromosome.  Latent HIV-1 reservoirs are established early during primary infection and constitute a major barrier to eradication, even in the presence of highly active antiretroviral therapy.  This Review analyses the mol. mechanisms that are necessary for the establishment of HIV-1 latency and their relationships with different cellular and anatomical reservoirs, and discusses the current treatment strategies for targeting viral persistence in reservoirs, their main limitations and future perspectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5tPDPAK2R5bVg90H21EOLACvtfcHk0lh7M0BBa-hc3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kqsb7J&md5=4df3674826db5a343d695693c1aec575</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro2223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro2223%26sid%3Dliteratum%253Aachs%26aulast%3DCoiras%26aufirst%3DM.%26aulast%3DLopez-Huertas%26aufirst%3DM.%2BR.%26aulast%3DPerez-Olmeda%26aufirst%3DM.%26aulast%3DAlcami%26aufirst%3DJ.%26atitle%3DUnderstanding%2520HIV-1%2520latency%2520provides%2520clues%2520for%2520the%2520eradication%2520of%2520long-term%2520reservoirs%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2009%26volume%3D7%26spage%3D798%26epage%3D812%26doi%3D10.1038%2Fnrmicro2223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Zack, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrigo, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weitsman, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Go, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haislip, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I. S.</span><span> </span><span class="NLM_article-title">HIV-1 entry into quiescent primary lymphocytes: molecular analysis reveals a labile, latent viral structure</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">222</span><span class="refDoi"> DOI: 10.1016/0092-8674(90)90802-L</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2F0092-8674%2890%2990802-L" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1990&pages=213-222&author=J.+A.+Zackauthor=S.+J.+Arrigoauthor=S.+R.+Weitsmanauthor=A.+S.+Goauthor=A.+Haislipauthor=I.+S.+Chen&title=HIV-1+entry+into+quiescent+primary+lymphocytes%3A+molecular+analysis+reveals+a+labile%2C+latent+viral+structure&doi=10.1016%2F0092-8674%2890%2990802-L"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2890%2990802-L&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252890%252990802-L%26sid%3Dliteratum%253Aachs%26aulast%3DZack%26aufirst%3DJ.%2BA.%26aulast%3DArrigo%26aufirst%3DS.%2BJ.%26aulast%3DWeitsman%26aufirst%3DS.%2BR.%26aulast%3DGo%26aufirst%3DA.%2BS.%26aulast%3DHaislip%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DI.%2BS.%26atitle%3DHIV-1%2520entry%2520into%2520quiescent%2520primary%2520lymphocytes%253A%2520molecular%2520analysis%2520reveals%2520a%2520labile%252C%2520latent%2520viral%2520structure%26jtitle%3DCell%26date%3D1990%26volume%3D61%26spage%3D213%26epage%3D222%26doi%3D10.1016%2F0092-8674%2890%2990802-L" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Lewin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wightman, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cameron, P. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purcell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaunders, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bloch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, A. D.</span><span> </span><span class="NLM_article-title">Virologic determinants of success after structured treatment interruptions of antiretrovirals in acute HIV-1 infection</span> <span class="citation_source-journal">JAIDS, J. Acquired Immune Defic. Syndr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">140</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span><span class="refDoi"> DOI: 10.1097/QAI.0b013e31815dbf7f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1097%2FQAI.0b013e31815dbf7f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=18300365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVGmsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=140-147&author=S.+R.+Lewinauthor=J.+M.+Murrayauthor=A.+Solomonauthor=F.+Wightmanauthor=P.+U.+Cameronauthor=D.+J.+Purcellauthor=J.+J.+Zaundersauthor=P.+Greyauthor=M.+Blochauthor=D.+Smithauthor=D.+A.+Cooperauthor=A.+D.+Kelleher&title=Virologic+determinants+of+success+after+structured+treatment+interruptions+of+antiretrovirals+in+acute+HIV-1+infection&doi=10.1097%2FQAI.0b013e31815dbf7f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Virologic Determinants of Success After Structured Treatment Interruptions of Antiretrovirals in Acute HIV-1 Infection</span></div><div class="casAuthors">Lewin, Sharon R.; Murray, John M.; Solomon, Ajantha; Wightman, Fiona; Cameron, Paul U.; Purcell, Damian J.; Zaunders, John J.; Grey, Pat; Bloch, Mark; Smith, Don; Cooper, David A.; Kelleher, Anthony D.</div><div class="citationInfo"><span class="NLM_cas:title">JAIDS, Journal of Acquired Immune Deficiency Syndromes</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">140-147</span>CODEN:
                <span class="NLM_cas:coden">JJASFJ</span>;
        ISSN:<span class="NLM_cas:issn">1525-4135</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Background: Latently infected resting memory CD4 T cells are thought to be the major reservoir that contributes to rebound viremia after cessation of antiretrovirals (ARVs).  Commencing ARVs during primary HIV-1 infection (PHI) may limit the size of the latent pool and lead to improved control of viral replication during structured treatment interruption (STI).  Methods: Individuals with PHI (n = 59) were randomized to receive ARVs with or without hydroxyurea.  After STI, a good response was defined as maintenance of HIV-1 RNA <5000 copies/mL for 24 wk off therapy.  In a detailed prospective virol. substudy (n = 19), integrated HIV-1 DNA, total HIV-1 DNA, and cell-assocd. HIV-1 unspliced (US) RNA were quantified using a real-time polymerase chain reaction assay.  Results: The plasma HIV-1 RNA 12 wk after the initiation of ARVs was significantly lower in good responders (n = 7) compared with poor responders (n = 12) (P = 0.005).  There were no significant differences between good and poor responders in integrated HIV-1 DNA, HIV-1 DNA, and HIV-1 US RNA.  Integrated HIV-1 DNA before initiation of ARVs was strongly correlated with plasma HIV-1 RNA at week 12 (P = 0.006, r = 0.81).  Conclusion: HIV-1 RNA measured 12 wk after initiation of ARV was the only virol. variable assocd. with viral rebound after treatment interruption in PHI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNtLX8clo1o7Vg90H21EOLACvtfcHk0lgWutplmPq6Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVGmsb8%253D&md5=6bec484bb4aad2a8ea4f7c6f49c50410</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FQAI.0b013e31815dbf7f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQAI.0b013e31815dbf7f%26sid%3Dliteratum%253Aachs%26aulast%3DLewin%26aufirst%3DS.%2BR.%26aulast%3DMurray%26aufirst%3DJ.%2BM.%26aulast%3DSolomon%26aufirst%3DA.%26aulast%3DWightman%26aufirst%3DF.%26aulast%3DCameron%26aufirst%3DP.%2BU.%26aulast%3DPurcell%26aufirst%3DD.%2BJ.%26aulast%3DZaunders%26aufirst%3DJ.%2BJ.%26aulast%3DGrey%26aufirst%3DP.%26aulast%3DBloch%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DCooper%26aufirst%3DD.%2BA.%26aulast%3DKelleher%26aufirst%3DA.%2BD.%26atitle%3DVirologic%2520determinants%2520of%2520success%2520after%2520structured%2520treatment%2520interruptions%2520of%2520antiretrovirals%2520in%2520acute%2520HIV-1%2520infection%26jtitle%3DJAIDS%252C%2520J.%2520Acquired%2520Immune%2520Defic.%2520Syndr.%26date%3D2008%26volume%3D47%26spage%3D140%26epage%3D147%26doi%3D10.1097%2FQAI.0b013e31815dbf7f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Hamer, D. H.</span><span> </span><span class="NLM_article-title">Can HIV be cured? Mechanisms of HIV persistence and strategies to combat it</span> <span class="citation_source-journal">Curr. HIV Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">111</span><span class="refDoi"> DOI: 10.2174/1570162043484915</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.2174%2F1570162043484915" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=15078175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVGqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2004&pages=99-111&author=D.+H.+Hamer&title=Can+HIV+be+cured%3F+Mechanisms+of+HIV+persistence+and+strategies+to+combat+it&doi=10.2174%2F1570162043484915"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Can HIV be cured? Mechanisms of HIV persistence and strategies to combat it</span></div><div class="casAuthors">Hamer, Dean H.</div><div class="citationInfo"><span class="NLM_cas:title">Current HIV Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">99-111</span>CODEN:
                <span class="NLM_cas:coden">CHRUBF</span>;
        ISSN:<span class="NLM_cas:issn">1570-162X</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Stable remission is the ultimate goal of HIV therapy.  A review of recent studies on the ability of HIV to persist despite highly active antiretroviral therapy (HAART) and immune stimulation suggests that achieving this goal will require four developments in basic and clin. science.  First, more effective antiretroviral therapies, targeted at proteins other than reverse transcriptase and protease, in order to eliminate the cryptic replication that continues despite best available HAART.  Second, agents that activate latent HIV gene expression in quiescent CD4 memory T cells, thereby exposing this viral reservoir to therapeutic intervention by a "shock and kill" strategy.  Third, mols. such as immunotoxins that specifically recognize HIV-encoded membrane proteins and thereby potentiate the destruction of infected cells.  Fourth, and still most distant, novel approaches such as genetically engineered cytotoxic T lymphocytes or anti-HIV microbes to suppress rekindling of infection by residual virus sequestered in anatomical and cellular reservoirs.  Although each of these steps will be difficult to achieve, the many benefits of a cure for HIV make this a worthwhile pursuit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmOGegrIx1CrVg90H21EOLACvtfcHk0lgWutplmPq6Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVGqsLk%253D&md5=b0d3ac7855dced12e419f8c380adcea8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.2174%2F1570162043484915&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1570162043484915%26sid%3Dliteratum%253Aachs%26aulast%3DHamer%26aufirst%3DD.%2BH.%26atitle%3DCan%2520HIV%2520be%2520cured%253F%2520Mechanisms%2520of%2520HIV%2520persistence%2520and%2520strategies%2520to%2520combat%2520it%26jtitle%3DCurr.%2520HIV%2520Res.%26date%3D2004%26volume%3D2%26spage%3D99%26epage%3D111%26doi%3D10.2174%2F1570162043484915" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Ott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Q.</span><span> </span><span class="NLM_article-title">The control of HIV transcription: keeping RNA polymerase II on track</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">426</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1016/j.chom.2011.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.chom.2011.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=22100159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2msrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=426-435&author=M.+Ottauthor=M.+Geyerauthor=Q.+Zhou&title=The+control+of+HIV+transcription%3A+keeping+RNA+polymerase+II+on+track&doi=10.1016%2Fj.chom.2011.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The control of HIV transcription: Keeping RNA polymerase II on track</span></div><div class="casAuthors">Ott, Melanie; Geyer, Matthias; Zhou, Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">426-435</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Thirteen years ago, human cyclin T1 was identified as part of the pos. transcription elongation factor b (P-TEFb) and the long-sought host cofactor for the HIV-1 transactivator Tat.  Recent years have brought new insights into the intricate regulation of P-TEFb function and its relationship with Tat, revealing novel mechanisms for controlling HIV transcription and fueling new efforts to overcome the barrier of transcriptional latency in eradicating HIV.  Moreover, the improved understanding of HIV and Tat forms a basis for studying transcription elongation control in general.  Here, we review advances in HIV transcription research with a focus on the growing family of cellular P-TEFb complexes, structural insights into the interactions between Tat, P-TEFb, and TAR RNA, and the multifaceted regulation of these interactions by posttranscriptional modifications of Tat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIp4eyVG6sSLVg90H21EOLACvtfcHk0lipyLsHauvEjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2msrzM&md5=c6bb819316c7c9439d7971ccca5c3fab</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2011.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2011.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DOtt%26aufirst%3DM.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DQ.%26atitle%3DThe%2520control%2520of%2520HIV%2520transcription%253A%2520keeping%2520RNA%2520polymerase%2520II%2520on%2520track%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2011%26volume%3D10%26spage%3D426%26epage%3D435%26doi%3D10.1016%2Fj.chom.2011.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sakane, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pagans, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizusawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lassen, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnoelzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">Activation of HIV transcription by the viral Tat protein requires a demethylation step mediated by lysine-specific demethylase 1 (LSD1/KDM1)</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1002184</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1002184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.ppat.1002184" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002184&author=N.+Sakaneauthor=H.+S.+Kwonauthor=S.+Pagansauthor=K.+Kaehlckeauthor=Y.+Mizusawaauthor=M.+Kamadaauthor=K.+G.+Lassenauthor=J.+Chanauthor=W.+C.+Greeneauthor=M.+Schnoelzerauthor=M.+Ott&title=Activation+of+HIV+transcription+by+the+viral+Tat+protein+requires+a+demethylation+step+mediated+by+lysine-specific+demethylase+1+%28LSD1%2FKDM1%29&doi=10.1371%2Fjournal.ppat.1002184"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002184%26sid%3Dliteratum%253Aachs%26aulast%3DSakane%26aufirst%3DN.%26aulast%3DKwon%26aufirst%3DH.%2BS.%26aulast%3DPagans%26aufirst%3DS.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DMizusawa%26aufirst%3DY.%26aulast%3DKamada%26aufirst%3DM.%26aulast%3DLassen%26aufirst%3DK.%2BG.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DGreene%26aufirst%3DW.%2BC.%26aulast%3DSchnoelzer%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DActivation%2520of%2520HIV%2520transcription%2520by%2520the%2520viral%2520Tat%2520protein%2520requires%2520a%2520demethylation%2520step%2520mediated%2520by%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%252FKDM1%2529%26jtitle%3DPLoS%2520Pathog.%26date%3D2011%26volume%3D7%26spage%3De1002184%26doi%3D10.1371%2Fjournal.ppat.1002184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Quivy, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Walque, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Lint, C.</span><span> </span><span class="NLM_article-title">Chromatin-associated regulation of HIV-1 transcription: implications for the development of therapeutic strategies</span> <span class="citation_source-journal">Subcell Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">371</span><span class="NLM_x">–</span> <span class="NLM_lpage">396</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=17484137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A280%3ADC%252BD2s3nsVOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2007&pages=371-396&author=V.+Quivyauthor=S.+De+Walqueauthor=C.+Van+Lint&title=Chromatin-associated+regulation+of+HIV-1+transcription%3A+implications+for+the+development+of+therapeutic+strategies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin-associated regulation of HIV-1 transcription: implications for the development of therapeutic strategies</span></div><div class="casAuthors">Quivy Vincent; De Walque Stephane; Van Lint Carine</div><div class="citationInfo"><span class="NLM_cas:title">Sub-cellular biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">371-96</span>
        ISSN:<span class="NLM_cas:issn">0306-0225</span>.
    </div><div class="casAbstract">Human Immunodeficiency Virus type 1 (HIV-1) infection can now be treated effectively in many patients in the developed world, using combinations of antiretroviral therapeutics, called Highly Active Anti-Retroviral Therapy (HAART).  However, despite prolonged treatment with HAART, the persistence of latently HIV-1-infected cellular reservoirs harboring transcriptionally silent but replication-competent proviruses represents the major hurdle to virus eradication.  These latently infected cells are a permanent source for virus reactivation and lead to a rebound of the viral load after interruption of HAART.  Therefore, a greater understanding of the molecular mechanisms regulating proviral latency and reactivation should lead to rational strategies aimed at purging these cellular reservoirs of HIV-1.  This review summarizes our current knowledge and understanding of the elements involved in HIV-1 transcriptional reactivation: (1) the site of integration; (2) the transcription factor NF-kappaB, which is induced by proinflammatory cytokines (such as TNFalpha) and binds to two kappaB sites in the HIV-1 promoter region; (3) the specific remodeling of a single nucleosome (called nuc-1 and located immediately downstream of the HIV-1 transcription start site under latency conditions) upon activation of the HIV-1 promoter; (4) post-translational acetylation of histones and of non-histone proteins (following treatment with deacetylases inhibitors, which induce viral transcription and nuc-1 remodeling); and (5) the viral trans-activator Tat, which promotes transcription by mediating the recruitment to the HIV-1 promoter of histone-modifying enzymes and ATP-dependent chromatin remodeling complexes required for nucleosome disruption and transcriptional processivity.  Finally, this review highlights experimental therapies aimed at administrating HIV-1 gene expression activators (such as HDAC inhibitors) combined with an effective HAART in order to reactivate and decrease/eliminate the pool of latently HIV-1-infected cellular reservoirs</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRlyO3RExbSUr5XHngOXiV_fW6udTcc2eb2dnvoA0OebLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3nsVOgtw%253D%253D&md5=63f50592399c37deb891ad9619544b36</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQuivy%26aufirst%3DV.%26aulast%3DDe%2BWalque%26aufirst%3DS.%26aulast%3DVan%2BLint%26aufirst%3DC.%26atitle%3DChromatin-associated%2520regulation%2520of%2520HIV-1%2520transcription%253A%2520implications%2520for%2520the%2520development%2520of%2520therapeutic%2520strategies%26jtitle%3DSubcell%2520Biochem.%26date%3D2007%26volume%3D41%26spage%3D371%26epage%3D396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Vigushin, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirsaidi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adcock, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coombes, R. C.</span><span> </span><span class="NLM_article-title">Trichostatin A is a histone deacetylase inhibitor with potent antitumor activity against breast cancer in vivo</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">971</span><span class="NLM_x">–</span> <span class="NLM_lpage">976</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=11309348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1eltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=971-976&author=D.+M.+Vigushinauthor=S.+Aliauthor=P.+E.+Paceauthor=N.+Mirsaidiauthor=K.+Itoauthor=I.+Adcockauthor=R.+C.+Coombes&title=Trichostatin+A+is+a+histone+deacetylase+inhibitor+with+potent+antitumor+activity+against+breast+cancer+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Trichostatin a is a histone deacetylase inhibitor with potent antitumor activity against breast cancer in vivo</span></div><div class="casAuthors">Vigushin, David M.; Ali, Simak; Pace, Paul E.; Mirsaidi, Nina; Ito, Kazuhiro; Adcock, Ian; Coombes, R. Charles</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">971-976</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Trichostatin A (TSA), an antifungal antibiotic with cytostatic and differentiating properties in mammalian cell culture, is a potent and specific inhibitor of histone deacetylase (HDAC) activity.  The purpose of this study was to evaluate the antiproliferative and HDAC inhibitory activity of TSA in vitro in human breast cancer cell lines and to assess its antitumor efficacy and toxicity in vivo in a carcinogen-induced rat mammary cancer model.  TSA inhibited proliferation of eight breast carcinoma cell lines with mean ± SD IC50 of 124.4±120.4 nM (range, 26.4-308.1 nM).  HDAC inhibitory activity of TSA was similar in all cell lines with mean ± SD IC50 of 2.4±0.5 nM (range, 1.5-2.9 nM), and TSA treatment resulted in pronounced histone H4 hyperacetylation.  In randomized controlled efficacy studies using the N-methyl-N-nitrosourea carcinogen-induced rat mammary carcinoma model, TSA had pronounced antitumor activity in vivo when administered to 16 animals at a dose of 500 μg/kg by s.c. injection daily for 4 wk compared with 14 control animals.  Furthermore, TSA did not cause any measurable toxicity in doses of up to 5 mg/kg by s.c. injection.  Forty-one tumors from 26 animals were examd. by histol.  Six tumors from 3 rats treated with TSA and 14 tumors from 9 control animals were adenocarcinomas.  In contrast, 19 tumors from 12 TSA-treated rats had a benign phenotype, either fibroadenoma or tubular adenoma, suggesting that the antitumor activity of TSA may be attributable to induction of differentiation.  Two control rats each had tumors with benign histol.  The present studies confirm the potent dose-dependent antitumor activity of TSA against breast cancer in vitro and in vivo, strongly supporting HDAC as a mol. target for anticancer therapy in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokpxVxd2w5QbVg90H21EOLACvtfcHk0ljm-yjgnDagJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1eltrw%253D&md5=c3853a7e52c3f87dbac938b996d8e57c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVigushin%26aufirst%3DD.%2BM.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DPace%26aufirst%3DP.%2BE.%26aulast%3DMirsaidi%26aufirst%3DN.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DAdcock%26aufirst%3DI.%26aulast%3DCoombes%26aufirst%3DR.%2BC.%26atitle%3DTrichostatin%2520A%2520is%2520a%2520histone%2520deacetylase%2520inhibitor%2520with%2520potent%2520antitumor%2520activity%2520against%2520breast%2520cancer%2520in%2520vivo%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2001%26volume%3D7%26spage%3D971%26epage%3D976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Dokmanovic, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, P. A.</span><span> </span><span class="NLM_article-title">Histone deacetylase inhibitors: overview and perspectives</span> <span class="citation_source-journal">Mol. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">981</span><span class="NLM_x">–</span> <span class="NLM_lpage">989</span><span class="refDoi"> DOI: 10.1158/1541-7786.MCR-07-0324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1158%2F1541-7786.MCR-07-0324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=17951399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFygsLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=981-989&author=M.+Dokmanovicauthor=C.+Clarkeauthor=P.+A.+Marks&title=Histone+deacetylase+inhibitors%3A+overview+and+perspectives&doi=10.1158%2F1541-7786.MCR-07-0324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase Inhibitors: Overview and Perspectives</span></div><div class="casAuthors">Dokmanovic, Milos; Clarke, Cathy; Marks, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">981-989</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase inhibitors (HDACi) comprise structurally diverse compds. that are a group of targeted anticancer agents.  The first of these new HDACi, vorinostat (suberoylanilide hydroxamic acid), has received Food and Drug Administration approval for treating patients with cutaneous T-cell lymphoma.  This review focuses on the activities of the 11 zinc-contg. HDACs, their histone and nonhistone protein substrates, and the different pathways by which HDACi induce transformed cell death.  A hypothesis is presented to explain the relative resistance of normal cells to HDACi-induced cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_V1LFFdlQYbVg90H21EOLACvtfcHk0ljm-yjgnDagJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFygsLvN&md5=8696ff3e0f8e910fd6fc6a7e38579b95</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-07-0324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-07-0324%26sid%3Dliteratum%253Aachs%26aulast%3DDokmanovic%26aufirst%3DM.%26aulast%3DClarke%26aufirst%3DC.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26atitle%3DHistone%2520deacetylase%2520inhibitors%253A%2520overview%2520and%2520perspectives%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2007%26volume%3D5%26spage%3D981%26epage%3D989%26doi%3D10.1158%2F1541-7786.MCR-07-0324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Bolden, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peart, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, R. W.</span><span> </span><span class="NLM_article-title">Anticancer activities of histone deacetylase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">784</span><span class="refDoi"> DOI: 10.1038/nrd2133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fnrd2133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=16955068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=769-784&author=J.+E.+Boldenauthor=M.+J.+Peartauthor=R.+W.+Johnstone&title=Anticancer+activities+of+histone+deacetylase+inhibitors&doi=10.1038%2Fnrd2133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Anticancer activities of histone deacetylase inhibitors</span></div><div class="casAuthors">Bolden, Jessica E.; Peart, Melissa J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">769-784</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) are enzymes involved in the remodelling of chromatin, and have a key role in the epigenetic regulation of gene expression.  In addn., the activity of non-histone proteins can be regulated through HDAC-mediated hypo-acetylation.  In recent years, inhibition of HDACs has emerged as a potential strategy to reverse aberrant epigenetic changes assocd. with cancer, and several classes of HDAC inhibitors have been found to have potent and specific anticancer activities in preclin. studies.  However, such studies have also indicated that the effects of HDAC inhibitors could be considerably broader and more complicated than originally understood.  Here we summarize recent advances in the understanding of the mol. events that underlie the anticancer effects of HDAC inhibitors, and discuss how such information could be used in optimizing the development and application of these agents in the clinic, either as monotherapies or in combination with other anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhfqS1awkOHbVg90H21EOLACvtfcHk0ljm-yjgnDagJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltrY%253D&md5=91e55ee942fda3e4e2055ddf299fc866</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrd2133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2133%26sid%3Dliteratum%253Aachs%26aulast%3DBolden%26aufirst%3DJ.%2BE.%26aulast%3DPeart%26aufirst%3DM.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DAnticancer%2520activities%2520of%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D769%26epage%3D784%26doi%3D10.1038%2Fnrd2133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Duverger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bibollet-Ruche, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cron, R. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutsch, O.</span><span> </span><span class="NLM_article-title">Determinants of the establishment of human immunodeficiency virus type 1 latency</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">3078</span><span class="NLM_x">–</span> <span class="NLM_lpage">3093</span><span class="refDoi"> DOI: 10.1128/JVI.02058-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1128%2FJVI.02058-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19144703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjs1Wrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=3078-3093&author=A.+Duvergerauthor=J.+Jonesauthor=J.+Mayauthor=F.+Bibollet-Rucheauthor=F.+A.+Wagnerauthor=R.+Q.+Cronauthor=O.+Kutsch&title=Determinants+of+the+establishment+of+human+immunodeficiency+virus+type+1+latency&doi=10.1128%2FJVI.02058-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Determinants of the establishment of human immunodeficiency virus type 1 latency</span></div><div class="casAuthors">Duverger, Alexandra; Jones, Jennifer; May, Jori; Bibollet-Ruche, Frederic; Wagner, Frederic A.; Cron, Randall Q.; Kutsch, Olaf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3078-3093</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Recent research has emphasized the notion that human immunodeficiency virus type 1 (HIV-1) latency is controlled by a restrictive histone code at, or DNA methylation of, the integrated viral promoter (long terminal repeat [LTR]).  The present concept of HIV-1 latency has essentially been patterned from the principles of cellular gene regulation.  Here we introduce an exptl. system that allows for the qual. and quant. kinetic study of latency establishment and maintenance at the population level.  In this system, we find no evidence that HIV-1 latency establishment is the consequence of downregulation of initial active infection followed by the establishment of a restrictive histone code at the viral LTR.  Latent infection was established following integration of the virus in the absence of viral gene expression (silent integration) and was a function of the NF-κB activation level in the host cell at the time of infection.  In the absence of a role for epigenetic regulation, we demonstrate that transcriptional interference, a mechanism that has recently been suggested to add to the stabilization of HIV-1 latency, is the primary mechanism to govern latency maintenance.  These findings provide direct exptl. evidence that the high no. of viral integration events (>90%) found in actively expressed genes of CD4+ memory T cells from highly active antiretroviral therapy-suppressed patients represent indeed latent infection events and that transcriptional interference may be the primary mechanism to control HIV-1 latency in vivo.  HIV-1 latency may thus not be governed by the principles of cellular gene regulation, and therapeutic strategies to deplete the pool of latently HIV-1-infected cells should be reconsidered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKJiZWjbwaY7Vg90H21EOLACvtfcHk0li6XJwxAvtsWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjs1Wrtro%253D&md5=09b01dc317f538ac492dffd2741c5180</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FJVI.02058-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02058-08%26sid%3Dliteratum%253Aachs%26aulast%3DDuverger%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DJ.%26aulast%3DMay%26aufirst%3DJ.%26aulast%3DBibollet-Ruche%26aufirst%3DF.%26aulast%3DWagner%26aufirst%3DF.%2BA.%26aulast%3DCron%26aufirst%3DR.%2BQ.%26aulast%3DKutsch%26aufirst%3DO.%26atitle%3DDeterminants%2520of%2520the%2520establishment%2520of%2520human%2520immunodeficiency%2520virus%2520type%25201%2520latency%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D3078%26epage%3D3093%26doi%3D10.1128%2FJVI.02058-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Keedy, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Archin, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gates, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espeseth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, D. M.</span><span> </span><span class="NLM_article-title">A limited group of class I histone deacetylases acts to repress human immunodeficiency virus type 1 expression</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">4749</span><span class="NLM_x">–</span> <span class="NLM_lpage">4756</span><span class="refDoi"> DOI: 10.1128/JVI.02585-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1128%2FJVI.02585-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19279091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvVGhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2009&pages=4749-4756&author=K.+S.+Keedyauthor=N.+M.+Archinauthor=A.+T.+Gatesauthor=A.+Espesethauthor=D.+J.+Hazudaauthor=D.+M.+Margolis&title=A+limited+group+of+class+I+histone+deacetylases+acts+to+repress+human+immunodeficiency+virus+type+1+expression&doi=10.1128%2FJVI.02585-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A limited group of class I histone deacetylases acts to repress human immunodeficiency virus type 1 expression</span></div><div class="casAuthors">Keedy, Kara S.; Archin, Nancie M.; Gates, Adam T.; Espeseth, Amy; Hazuda, Daria J.; Margolis, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4749-4756</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Silencing of the integrated human immunodeficiency virus type 1 (HIV-1) genome in resting CD4+ T cells is a significant contributor to the persistence of infection, allowing the virus to evade both immune detection and pharmaceutical attack.  Nonselective histone deacetylase (HDAC) inhibitors are capable of inducing expression of quiescent HIV-1 in latently infected cells.  However, potent global HDAC inhibition can induce host toxicity.  To det. the specific HDACs that regulate HIV-1 transcription, we evaluated HDAC1 to HDAC11 RNA expression and protein expression and compartmentalization in the resting CD4+ T cells of HIV-1-pos., aviremic patients.  HDAC1, -3, and -7 had the highest mRNA expression levels in these cells.  Although all HDACs were detected in resting CD4+ T cells by Western blot anal., HDAC5, -8, and -11 were primarily sequestered in the cytoplasm.  Using chromatin immunopptn. assays, we detected HDAC1, -2, and -3 at the HIV-1 promoter in Jurkat J89GFP cells.  Targeted inhibition of HDACs by small interfering RNA demonstrated that HDAC2 and HDAC3 contribute to repression of HIV-1 long terminal repeat expression in the HeLa P4/R5 cell line model of latency.  Together, these results suggest that HDAC inhibitors specific for a limited no. of class I HDACs may offer a targeted approach to the disruption of persistent HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLmd8sCQLl4rVg90H21EOLACvtfcHk0li6XJwxAvtsWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvVGhsLk%253D&md5=bf4aac6fc7096a987b6d7698796ffe10</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FJVI.02585-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02585-08%26sid%3Dliteratum%253Aachs%26aulast%3DKeedy%26aufirst%3DK.%2BS.%26aulast%3DArchin%26aufirst%3DN.%2BM.%26aulast%3DGates%26aufirst%3DA.%2BT.%26aulast%3DEspeseth%26aufirst%3DA.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26atitle%3DA%2520limited%2520group%2520of%2520class%2520I%2520histone%2520deacetylases%2520acts%2520to%2520repress%2520human%2520immunodeficiency%2520virus%2520type%25201%2520expression%26jtitle%3DJ.%2520Virol.%26date%3D2009%26volume%3D83%26spage%3D4749%26epage%3D4756%26doi%3D10.1128%2FJVI.02585-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Choudhary, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, D. M.</span><span> </span><span class="NLM_article-title">Curing HIV: pharmacologic approaches to target HIV-1 latency</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">418</span><span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010510-100237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1146%2Fannurev-pharmtox-010510-100237" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=397-418&author=S.+K.+Choudharyauthor=D.+M.+Margolis&title=Curing+HIV%3A+pharmacologic+approaches+to+target+HIV-1+latency&doi=10.1146%2Fannurev-pharmtox-010510-100237"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010510-100237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010510-100237%26sid%3Dliteratum%253Aachs%26aulast%3DChoudhary%26aufirst%3DS.%2BK.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26atitle%3DCuring%2520HIV%253A%2520pharmacologic%2520approaches%2520to%2520target%2520HIV-1%2520latency%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2011%26volume%3D51%26spage%3D397%26epage%3D418%26doi%3D10.1146%2Fannurev-pharmtox-010510-100237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kijima, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beppu, T.</span><span> </span><span class="NLM_article-title">Potent and specific inhibition of mammalian histone deacetylase both in vivo and in vitro by trichostatin A</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">17174</span><span class="NLM_x">–</span> <span class="NLM_lpage">17179</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=2211619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK3cXlvVyqsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=17174-17179&author=M.+Yoshidaauthor=M.+Kijimaauthor=M.+Akitaauthor=T.+Beppu&title=Potent+and+specific+inhibition+of+mammalian+histone+deacetylase+both+in+vivo+and+in+vitro+by+trichostatin+A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and specific inhibition of mammalian histone deacetylase both in vivo and in vitro by trichostatin A</span></div><div class="casAuthors">Yoshida, Minoru; Kijima, Masako; Akita, Mitsuru; Beppu, Teruhiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">17174-9</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">(R)-Trichostatin A (TSA) is a Streptomyces product which causes the induction of Friend cell differentiation and specific inhibition of the cell cycle of normal rat fibroblasts in the G1 and G2 phases at very low concns.  TSA caused an accumulation of acetylated histone species in a variety of mammalian cell lines.  Pulse-labeling expts. indicated that TSA markedly prolonged the in vivo half-life of the labile acetyl groups on histones in mouse mammary gland tumor cells, FM3A.  The partially purified histone deacetylase from wild-type FM3A cells was effectively inhibited by TSA in a noncompetitive manner with Ki = 3.4 nM.  A newly isolated mutant cell line of FM3A resistant to TSA did not show the accumulation of the acetylated histones in the presence of a higher concn. of TSA.  The histone deacetylase prepn. from the mutant showed decreased sensitivity to TSA (Ki = 31 nM, noncompetitive).  These results clearly indicate that TSA is a potent and specific inhibitor of histone deacetylase and that the in vivo effect of TSA on cell proliferation and differentiation can be attributed to the inhibition of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM_EbVVcRXYbVg90H21EOLACvtfcHk0liOMst2tUrOCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlvVyqsrY%253D&md5=f2bed4f3b2d1502fd3610dfdf376b2dc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DKijima%26aufirst%3DM.%26aulast%3DAkita%26aufirst%3DM.%26aulast%3DBeppu%26aufirst%3DT.%26atitle%3DPotent%2520and%2520specific%2520inhibition%2520of%2520mammalian%2520histone%2520deacetylase%2520both%2520in%2520vivo%2520and%2520in%2520vitro%2520by%2520trichostatin%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D17174%26epage%3D17179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Darkin-Rattray, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurnett, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dulski, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crumley, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allocco, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cannova, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meinke, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colletti, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bednarek, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goetz, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrowski, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polishook, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmatz, D. M.</span><span> </span><span class="NLM_article-title">Apicidin: a novel antiprotozoal agent that inhibits parasite histone deacetylase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">13143</span><span class="NLM_x">–</span> <span class="NLM_lpage">13147</span><span class="refDoi"> DOI: 10.1073/pnas.93.23.13143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.93.23.13143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=8917558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK28XmvV2qtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=13143-13147&author=S.+J.+Darkin-Rattrayauthor=A.+M.+Gurnettauthor=R.+W.+Myersauthor=P.+M.+Dulskiauthor=T.+M.+Crumleyauthor=J.+J.+Alloccoauthor=C.+Cannovaauthor=P.+T.+Meinkeauthor=S.+L.+Collettiauthor=M.+A.+Bednarekauthor=S.+B.+Singhauthor=M.+A.+Goetzauthor=A.+W.+Dombrowskiauthor=J.+D.+Polishookauthor=D.+M.+Schmatz&title=Apicidin%3A+a+novel+antiprotozoal+agent+that+inhibits+parasite+histone+deacetylase&doi=10.1073%2Fpnas.93.23.13143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Apicidin: a novel antiprotozoal agent that inhibits parasite histone deacetylase</span></div><div class="casAuthors">Darkin-Rattray, Sandra J.; Gurnett, Anne M.; Myers, Robert W.; Dulski, Paula M.; Crumley, Tami M.; Allocco, John J.; Cannova, Christine; Meinke, Peter T.; Colletti, Steven L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">13143-13147</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A novel fungal metabolite, apicidin [cyclo(N-O-methyl-L-tryptophanyl-L-isoleucinyl-D-pipecolinyl-L-2-amino-8-oxodecanoyl)], that exhibits potent, broad spectrum antiprotozoal activity in vitro against Apicomplexan parasites has been identified.  It is also orally and parenterally active in vivo against Plasmodium berghei malaria in mice.  Many Apicomplexan parasites cause serious, life-threatening human and animal diseases, such as malaria, cryptosporidiosis, toxoplasmosis, and coccidiosis, and new therapeutic agents are urgently needed.  Apicidin's antiparasitic activity appears to be due to low nanomolar inhibition of Apicomplexan histone deacetylase (HDA), which induces hyperacetylation of histones in treated parasites.  The acetylation-deacetylation of histones is thought to play a central role in transcriptional control in eukaryotic cells.  Other known HDA inhibitors were also evaluated and found to possess antiparasitic activity, suggesting that HDA is an attractive target for the development of novel antiparasitic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqub6fPyIvFQrVg90H21EOLACvtfcHk0liOMst2tUrOCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmvV2qtrk%253D&md5=3738f88674b6e5856d55bab7a0701e22</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.23.13143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.23.13143%26sid%3Dliteratum%253Aachs%26aulast%3DDarkin-Rattray%26aufirst%3DS.%2BJ.%26aulast%3DGurnett%26aufirst%3DA.%2BM.%26aulast%3DMyers%26aufirst%3DR.%2BW.%26aulast%3DDulski%26aufirst%3DP.%2BM.%26aulast%3DCrumley%26aufirst%3DT.%2BM.%26aulast%3DAllocco%26aufirst%3DJ.%2BJ.%26aulast%3DCannova%26aufirst%3DC.%26aulast%3DMeinke%26aufirst%3DP.%2BT.%26aulast%3DColletti%26aufirst%3DS.%2BL.%26aulast%3DBednarek%26aufirst%3DM.%2BA.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DGoetz%26aufirst%3DM.%2BA.%26aulast%3DDombrowski%26aufirst%3DA.%2BW.%26aulast%3DPolishook%26aufirst%3DJ.%2BD.%26aulast%3DSchmatz%26aufirst%3DD.%2BM.%26atitle%3DApicidin%253A%2520a%2520novel%2520antiprotozoal%2520agent%2520that%2520inhibits%2520parasite%2520histone%2520deacetylase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D13143%26epage%3D13147%26doi%3D10.1073%2Fpnas.93.23.13143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Kijima, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugita, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horinouchi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beppu, T.</span><span> </span><span class="NLM_article-title">Trapoxin, an antitumor cyclic tetrapeptide, is an irreversible inhibitor of mammalian histone deacetylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">268</span><span class="NLM_x">, </span> <span class="NLM_fpage">22429</span><span class="NLM_x">–</span> <span class="NLM_lpage">22435</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=8226751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK3sXlsleks7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=268&publication_year=1993&pages=22429-22435&author=M.+Kijimaauthor=M.+Yoshidaauthor=K.+Sugitaauthor=S.+Horinouchiauthor=T.+Beppu&title=Trapoxin%2C+an+antitumor+cyclic+tetrapeptide%2C+is+an+irreversible+inhibitor+of+mammalian+histone+deacetylase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Trapoxin, an antitumor cyclic tetrapeptide, is an irreversible inhibitor of mammalian histone deacetylase</span></div><div class="casAuthors">Kijima, Masako; Yoshida, Minoru; Sugita, Kenji; Horinouchi, Sueharu; Beppu, Teruhiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">268</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">22429-35</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Trapoxin A (I), is a fungal product that induces morphol. reversion from transformed to normal in sis-transformed NIH3T3 fibroblasts.  I was found to cause accumulation of highly acetylated core histones in a variety of mammalian cell lines.  In vitro expts. using partially purified mouse histone deacetylase showed that a low concn. of I irreversibly inhibited deacetylation of acetylated histone mols.  Chem. redn. of an epoxide group in I completely abolished the inhibitory activity, suggesting that I binds covalently to the histone deacetylase via the epoxide.  In contrast, inhibition by trichostatin A, a known potent inhibitor of histone deacetylase, was reversible.  Despite the different mode of inhibition, I and trichostatin A induced almost the same biol. effects on the cell cycle and differentiation.  These results strongly suggest that the in vivo effects commonly induced by these agents can be attributed to histone hyperacetylation resulting from the inhibition of histone deacetylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqscgc_V7MoWrVg90H21EOLACvtfcHk0lie0LiUbtab5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsleks7Y%253D&md5=5252b79bf5604918fea6ecfdb64bc015</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKijima%26aufirst%3DM.%26aulast%3DYoshida%26aufirst%3DM.%26aulast%3DSugita%26aufirst%3DK.%26aulast%3DHorinouchi%26aufirst%3DS.%26aulast%3DBeppu%26aufirst%3DT.%26atitle%3DTrapoxin%252C%2520an%2520antitumor%2520cyclic%2520tetrapeptide%252C%2520is%2520an%2520irreversible%2520inhibitor%2520of%2520mammalian%2520histone%2520deacetylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1993%26volume%3D268%26spage%3D22429%26epage%3D22435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Nakao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsunaga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindoh, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagai, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Soest, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fusetani, N.</span><span> </span><span class="NLM_article-title">Azumamides A-E: histone deacetylase inhibitory cyclic tetrapeptides from the marine sponge Mycale izuensis</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7553</span><span class="NLM_x">–</span> <span class="NLM_lpage">7557</span><span class="refDoi"> DOI: 10.1002/anie.200602047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fanie.200602047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7553-7557&author=Y.+Nakaoauthor=S.+Yoshidaauthor=S.+Matsunagaauthor=N.+Shindohauthor=Y.+Teradaauthor=K.+Nagaiauthor=J.+K.+Yamashitaauthor=A.+Ganesanauthor=R.+W.+van+Soestauthor=N.+Fusetani&title=Azumamides+A-E%3A+histone+deacetylase+inhibitory+cyclic+tetrapeptides+from+the+marine+sponge+Mycale+izuensis&doi=10.1002%2Fanie.200602047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fanie.200602047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200602047%26sid%3Dliteratum%253Aachs%26aulast%3DNakao%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DS.%26aulast%3DShindoh%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DY.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DYamashita%26aufirst%3DJ.%2BK.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3Dvan%2BSoest%26aufirst%3DR.%2BW.%26aulast%3DFusetani%26aufirst%3DN.%26atitle%3DAzumamides%2520A-E%253A%2520histone%2520deacetylase%2520inhibitory%2520cyclic%2520tetrapeptides%2520from%2520the%2520marine%2520sponge%2520Mycale%2520izuensis%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D7553%26epage%3D7557%26doi%3D10.1002%2Fanie.200602047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Izzo, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maulucci, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bifulco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Riccardis, F.</span><span> </span><span class="NLM_article-title">Total synthesis of azumamides A and E</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7557</span><span class="NLM_x">–</span> <span class="NLM_lpage">7560</span><span class="refDoi"> DOI: 10.1002/anie.200602033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fanie.200602033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1yqtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7557-7560&author=I.+Izzoauthor=N.+Maulucciauthor=G.+Bifulcoauthor=F.+De+Riccardis&title=Total+synthesis+of+azumamides+A+and+E&doi=10.1002%2Fanie.200602033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis of azumamides A and E</span></div><div class="casAuthors">Izzo, Irene; Maulucci, Nakia; Bifulco, Giuseppe; De Riccardis, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">7557-7560</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A zoom in on azumamides: An efficient synthetic route for two of the newly discovered marine natural products, namely azumamides A and E I (R1 = NH2, OH, resp.) , has been developed.  The present synthesis was followed by detn. of the stereochem. structure of the azumamides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouhkxm-GSHPrVg90H21EOLACvtfcHk0lie0LiUbtab5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1yqtb3E&md5=2f2dfdc381d3e1de37531c6d1ed6695d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fanie.200602033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200602033%26sid%3Dliteratum%253Aachs%26aulast%3DIzzo%26aufirst%3DI.%26aulast%3DMaulucci%26aufirst%3DN.%26aulast%3DBifulco%26aufirst%3DG.%26aulast%3DDe%2BRiccardis%26aufirst%3DF.%26atitle%3DTotal%2520synthesis%2520of%2520azumamides%2520A%2520and%2520E%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D7557%26epage%3D7560%26doi%3D10.1002%2Fanie.200602033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Wen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carey, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fusetani, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span> </span><span class="NLM_article-title">Total synthesis of azumamide A and azumamide E, evaluation as histone deacetylase inhibitors, and design of a more potent analogue</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1105</span><span class="NLM_x">–</span> <span class="NLM_lpage">1108</span><span class="refDoi"> DOI: 10.1021/ol070046y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol070046y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvFKru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=1105-1108&author=S.+Wenauthor=K.+L.+Careyauthor=Y.+Nakaoauthor=N.+Fusetaniauthor=G.+Packhamauthor=A.+Ganesan&title=Total+synthesis+of+azumamide+A+and+azumamide+E%2C+evaluation+as+histone+deacetylase+inhibitors%2C+and+design+of+a+more+potent+analogue&doi=10.1021%2Fol070046y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis of Azumamide A and Azumamide E, Evaluation as Histone Deacetylase Inhibitors, and Design of a More Potent Analog</span></div><div class="casAuthors">Wen, Shijun; Carey, Krystle L.; Nakao, Yoichi; Fusetani, Nobuhiro; Packham, Graham; Ganesan, A.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1105-1108</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The unprecedented diastereoselective Mannich reaction of a Z-allylsulfoximine was a key step in the total synthesis of the marine natural products azumamide A and E, and an unnatural analog.  Their relative potency as histone deacetylase inhibitors was evaluated and found to correlate with predicted zinc-binding affinity.  The stereoselective Mannich reaction of an unusual β,γ-unsatd. sulfinylimine is likely the most challenging transformation accomplished to date with the Ellman auxiliary, and enabled a concise route with only 15 steps in the longest linear sequence.  Within the azumamide series, histone deacetylase (HDAC) inhibitory activity is in the order carboxamide < carboxylic acid < hydroxamic acid, as predicted by the relative zinc binding of these compds.  The biol. activity of the azumamides is significantly weaker than FK228 (IC50 of 15 ± 9 nM in the same assay).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf820ZWbDDobVg90H21EOLACvtfcHk0lih4Acy8l1THA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvFKru7Y%253D&md5=ff577d9a068da7ee5a3d16f1c7299525</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fol070046y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol070046y%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DS.%26aulast%3DCarey%26aufirst%3DK.%2BL.%26aulast%3DNakao%26aufirst%3DY.%26aulast%3DFusetani%26aufirst%3DN.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DGanesan%26aufirst%3DA.%26atitle%3DTotal%2520synthesis%2520of%2520azumamide%2520A%2520and%2520azumamide%2520E%252C%2520evaluation%2520as%2520histone%2520deacetylase%2520inhibitors%252C%2520and%2520design%2520of%2520a%2520more%2520potent%2520analogue%26jtitle%3DOrg.%2520Lett.%26date%3D2007%26volume%3D9%26spage%3D1105%26epage%3D1108%26doi%3D10.1021%2Fol070046y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Taori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, V. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luesch, H.</span><span> </span><span class="NLM_article-title">Structure and activity of largazole, a potent antiproliferative agent from the Floridian marine cyanobacterium Symploca sp</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">1806</span><span class="NLM_x">–</span> <span class="NLM_lpage">1807</span><span class="refDoi"> DOI: 10.1021/ja7110064</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja7110064" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFGhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=1806-1807&author=K.+Taoriauthor=V.+J.+Paulauthor=H.+Luesch&title=Structure+and+activity+of+largazole%2C+a+potent+antiproliferative+agent+from+the+Floridian+marine+cyanobacterium+Symploca+sp&doi=10.1021%2Fja7110064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Activity of Largazole, a Potent Antiproliferative Agent from the Floridian Marine Cyanobacterium Symploca sp.</span></div><div class="casAuthors">Taori, Kanchan; Paul, Valerie J.; Luesch, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1806-1807</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel cytotoxic cyclodepsipeptide, termed largazole, has been isolated from the marine cyanobacterium Symploca sp. collected in the Florida Keys.  Its planar structure was elucidated by 1D and 2D NMR spectroscopy in conjunction with mass spectrometry.  The abs. configuration of largazole was detd. by chem. degrdn. followed by chiral HPLC anal.  Largazole possesses densely assembled unusual structural features, including a rare 4-methylthiazoline linearly fused to a thiazole in its cyclic core and a hitherto undescribed 3-hydroxy-7-mercaptohept-4-enoic acid unit incorporated in an ester, thioester, and amide framework.  Largazole exhibits potent antiproliferative activity and preferentially targets cancer cells over nontransformed cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0JtehDfpXXLVg90H21EOLACvtfcHk0lih4Acy8l1THA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFGhtw%253D%253D&md5=bcd18bbee431e273c1b646f3c9c1388b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fja7110064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja7110064%26sid%3Dliteratum%253Aachs%26aulast%3DTaori%26aufirst%3DK.%26aulast%3DPaul%26aufirst%3DV.%2BJ.%26aulast%3DLuesch%26aufirst%3DH.%26atitle%3DStructure%2520and%2520activity%2520of%2520largazole%252C%2520a%2520potent%2520antiproliferative%2520agent%2520from%2520the%2520Floridian%2520marine%2520cyanobacterium%2520Symploca%2520sp%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D1806%26epage%3D1807%26doi%3D10.1021%2Fja7110064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emiliani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breslow, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rifkind, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marks, P. A.</span><span> </span><span class="NLM_article-title">A class of hybrid polar inducers of transformed cell differentiation inhibits histone deacetylases</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">3003</span><span class="NLM_x">–</span> <span class="NLM_lpage">3007</span><span class="refDoi"> DOI: 10.1073/pnas.95.6.3003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.95.6.3003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=9501205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK1cXitV2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1998&pages=3003-3007&author=V.+M.+Richonauthor=S.+Emilianiauthor=E.+Verdinauthor=Y.+Webbauthor=R.+Breslowauthor=R.+A.+Rifkindauthor=P.+A.+Marks&title=A+class+of+hybrid+polar+inducers+of+transformed+cell+differentiation+inhibits+histone+deacetylases&doi=10.1073%2Fpnas.95.6.3003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A class of hybrid polar inducers of transformed cell differentiation inhibits histone deacetylases</span></div><div class="casAuthors">Richon, Vicotria M.; Emiliani, Stephane; Verdin, Eric; Webb, Yael; Breslow, Ronald; Rifkind, Richard A.; Marks, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3003-3007</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hybrid polar compds. (HPCs) have been synthesized that induce terminal differentiation and/or apoptosis in various transformed cells.  We have previously reported on the development of the second-generation HPCs suberoylanilide hydroxamic acid (SAHA) and m-carboxycinnamic acid bishydroxamide (CBHA) that are 2,000-fold more potent inducers on a molar basis than the prototype HPC hexamethylene bisacetamide (HMBA).  Herein we report that CBHA and SAHA inhibit histone deacetylase 1 (HDAC1) and histone deacetylase 3 (HDAC3) activity in vitro.  Treatment of cells in culture with SAHA results in a marked hyperacetylation of histone H4, but culture with HMBA does not.  Murine erythroleukemia cells developed for resistance to SAHA are cross-resistant to trichostatin A, a known deacetylase inhibitor and differentiation inducer, but are not cross-resistant to HMBA.  These studies show that the second-generation HPCs, unlike HMBA, are potent inhibitors of HDAC activity.  In this sense, HMBA and the second-generation HPCs appear to induce differentiation by different pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA08MImdI837Vg90H21EOLACvtfcHk0lgOyrLPyUoVgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXitV2itro%253D&md5=1e821e03d7c9522cbf3084c6f659fd72</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.95.6.3003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.95.6.3003%26sid%3Dliteratum%253Aachs%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DEmiliani%26aufirst%3DS.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DWebb%26aufirst%3DY.%26aulast%3DBreslow%26aufirst%3DR.%26aulast%3DRifkind%26aufirst%3DR.%2BA.%26aulast%3DMarks%26aufirst%3DP.%2BA.%26atitle%3DA%2520class%2520of%2520hybrid%2520polar%2520inducers%2520of%2520transformed%2520cell%2520differentiation%2520inhibits%2520histone%2520deacetylases%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1998%26volume%3D95%26spage%3D3003%26epage%3D3007%26doi%3D10.1073%2Fpnas.95.6.3003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Miller, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witter, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belvedere, S.</span><span> </span><span class="NLM_article-title">Histone deacetylase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">5097</span><span class="NLM_x">–</span> <span class="NLM_lpage">5116</span><span class="refDoi"> DOI: 10.1021/jm0303094</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0303094" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5097-5116&author=T.+A.+Millerauthor=D.+J.+Witterauthor=S.+Belvedere&title=Histone+deacetylase+inhibitors&doi=10.1021%2Fjm0303094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors</span></div><div class="casAuthors">Miller, Thomas A.; Witter, David J.; Belvedere, Sandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5097-5116</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the medicinal chem. and structure-activity relationships underlying advances in histone deacetylase (HDAC) class I/II inhibitor discovery, design, and optimization.  HDAC structural studies and HDAC inhibitory activity of small-mol. hydroxamic acids, carboxylates, benzamides, electrophilic ketones, and cyclic peptides are described.  Miscellaneousness and selectivity of the class I/II HDACs are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU5UlOsBUF7rVg90H21EOLACvtfcHk0lgOyrLPyUoVgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXosVCktbg%253D&md5=d2d0a95d87d699334c1630df5deda38a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm0303094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303094%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DT.%2BA.%26aulast%3DWitter%26aufirst%3DD.%2BJ.%26aulast%3DBelvedere%26aufirst%3DS.%26atitle%3DHistone%2520deacetylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5097%26epage%3D5116%26doi%3D10.1021%2Fjm0303094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirisawad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buggy, J. J.</span><span> </span><span class="NLM_article-title">A novel histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 induces apoptosis in T-cell lymphomas</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1034</span><span class="refDoi"> DOI: 10.1038/leu.2008.9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1038%2Fleu.2008.9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=18256683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=1026-1034&author=S.+Balasubramanianauthor=J.+Ramosauthor=W.+Luoauthor=M.+Sirisawadauthor=E.+Vernerauthor=J.+J.+Buggy&title=A+novel+histone+deacetylase+8+%28HDAC8%29-specific+inhibitor+PCI-34051+induces+apoptosis+in+T-cell+lymphomas&doi=10.1038%2Fleu.2008.9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A novel histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 induces apoptosis in T-cell lymphomas</span></div><div class="casAuthors">Balasubramanian, S.; Ramos, J.; Luo, W.; Sirisawad, M.; Verner, E.; Buggy, J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1026-1034</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We have developed a potent, histone deacetylase 8 (HDAC8)-specific inhibitor PCI-34051 with >200-fold selectivity over the other HDAC isoforms.  PCI-34051 induces caspase-dependent apoptosis in cell lines derived from T-cell lymphomas or leukemias, but not in other hematopoietic or solid tumor lines.  Unlike broad-spectrum HDAC inhibitors, PCI-34051 does not cause detectable histone or tubulin acetylation.  Cells defective in T-cell receptor signaling were still sensitive to PCI-34051-induced apoptosis, whereas a phospholipase C-γ1 (PLCγ1)-defective line was resistant.  Jurkat cells showed a dose-dependent decrease in PCI-34051-induced apoptosis upon treatment with a PLC inhibitor U73122, but not with an inactive analog.  We found that rapid intracellular calcium mobilization from endoplasmic reticulum (ER) and later cytochrome c release from mitochondria are essential for the apoptotic mechanism.  The rapid Ca2+ flux was dependent on PCI-34051 concn., and was blocked by the PLC inhibitor U73122.  Further, apoptosis was blocked by Ca2+ chelators (BAPTA) and enhanced by Ca2+ effectors (thapsigargin), supporting this model.  These studies show that HDAC8-selective inhibitors have a unique mechanism of action involving PLCγ1 activation and calcium-induced apoptosis, and could offer benefits including a greater therapeutic index for treating T-cell malignancies.  Leukemia (2008) 22, 1026-1034; doi:10.1038/leu.2008.9; published online 7 Feb. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW-VB4Wb3h1bVg90H21EOLACvtfcHk0lg6_75thD9kZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvVSrtr0%253D&md5=03446a05b396ea9497e4f6f9254cae03</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fleu.2008.9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2008.9%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DRamos%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DSirisawad%26aufirst%3DM.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26atitle%3DA%2520novel%2520histone%2520deacetylase%25208%2520%2528HDAC8%2529-specific%2520inhibitor%2520PCI-34051%2520induces%2520apoptosis%2520in%2520T-cell%2520lymphomas%26jtitle%3DLeukemia%26date%3D2008%26volume%3D22%26spage%3D1026%26epage%3D1034%26doi%3D10.1038%2Fleu.2008.9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasuya, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ota, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asamitsu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of highly selective and potent histone deacetylase 3 inhibitors using click chemistry-based combinatorial fragment assembly</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e68669</span><span class="refDoi"> DOI: 10.1371/journal.pone.0068669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.pone.0068669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=23874714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SgsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e68669&author=T.+Suzukiauthor=Y.+Kasuyaauthor=Y.+Itohauthor=Y.+Otaauthor=P.+Zhanauthor=K.+Asamitsuauthor=H.+Nakagawaauthor=T.+Okamotoauthor=N.+Miyata&title=Identification+of+highly+selective+and+potent+histone+deacetylase+3+inhibitors+using+click+chemistry-based+combinatorial+fragment+assembly&doi=10.1371%2Fjournal.pone.0068669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of highly selective and potent histone deacetylase 3 inhibitors using click chemistry-based combinatorial fragment assembly</span></div><div class="casAuthors">Suzuki, Takayoshi; Kasuya, Yuki; Itoh, Yukihiro; Ota, Yosuke; Zhan, Peng; Asamitsu, Kaori; Nakagawa, Hidehiko; Okamoto, Takashi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e68669</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">To find histone deacetylase 3 (HDAC3)-selective inhibitors, a series of 504 candidates was assembled using "click chem.", by reacting nine alkynes bearing a zinc-binding group with 56 azide building blocks in the presence of Cu(I) catalyst.  Screening of the 504-member triazole library against HDAC3 and other HDAC isoenzymes led to the identification of potent and selective HDAC3 inhibitors T247 and T326.  These compds. showed potent HDAC3 inhibition with submicromolar IC50s, whereas they did not strongly inhibit other isoenzymes.  Compds. T247 and T326 also induced a dose-dependent selective increase of NF-κB acetylation in human colon cancer HCT116 cells, indicating selective inhibition of HDAC3 in the cells.  In addn., these HDAC3-selective inhibitors induced growth inhibition of cancer cells, and activated HIV gene expression in latent HIV-infected cells.  These findings indicate that HDAC3-selective inhibitors are promising candidates for anticancer drugs and antiviral agents.  This work also suggests the usefulness of the click chem. approach to find isoenzyme-selective HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4CvS43ILxQLVg90H21EOLACvtfcHk0lg6_75thD9kZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SgsrjF&md5=74d12c8fb1f06f8e00691415434fd929</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0068669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0068669%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DKasuya%26aufirst%3DY.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOta%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DAsamitsu%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520highly%2520selective%2520and%2520potent%2520histone%2520deacetylase%25203%2520inhibitors%2520using%2520click%2520chemistry-based%2520combinatorial%2520fragment%2520assembly%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De68669%26doi%3D10.1371%2Fjournal.pone.0068669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brochier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langley, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozikowski, A. P.</span><span> </span><span class="NLM_article-title">Rational design and simple chemistry yield a superior, neuroprotective HDAC6 inhibitor, tubastatin A</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">10842</span><span class="NLM_x">–</span> <span class="NLM_lpage">10846</span><span class="refDoi"> DOI: 10.1021/ja102758v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja102758v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=10842-10846&author=K.+V.+Butlerauthor=J.+Kalinauthor=C.+Brochierauthor=G.+Vistoliauthor=B.+Langleyauthor=A.+P.+Kozikowski&title=Rational+design+and+simple+chemistry+yield+a+superior%2C+neuroprotective+HDAC6+inhibitor%2C+tubastatin+A&doi=10.1021%2Fja102758v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design and Simple Chemistry Yield a Superior, Neuroprotective HDAC6 Inhibitor, Tubastatin A</span></div><div class="casAuthors">Butler, Kyle V.; Kalin, Jay; Brochier, Camille; Vistoli, Guilio; Langley, Brett; Kozikowski, Alan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10842-10846</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design combined with homol. modeling techniques were used to develop potent inhibitors of HDAC6 that display superior selectivity for the HDAC6 isoenzyme compared to other inhibitors.  These inhibitors can be assembled in a few synthetic steps, and thus are readily scaled up for in vivo studies.  An optimized compd. from this series, designated Tubastatin A, was tested in primary cortical neuron cultures in which it was found to induce elevated levels of acetylated α-tubulin, but not histone, consistent with its HDAC6 selectivity.  Tubastatin A also conferred dose-dependent protection in primary cortical neuron cultures against glutathione depletion-induced oxidative stress.  Importantly, when given alone at all concns. tested, this hydroxamate-contg. HDAC6-selective compd. displayed no neuronal toxicity, thus, forecasting the potential application of this agent and its analogs to neurodegenerative conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruEaJPWQfjH7Vg90H21EOLACvtfcHk0lg6_75thD9kZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ers7k%253D&md5=f0cc68bb68f94ad1ef48f88cc6b57129</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja102758v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja102758v%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKalin%26aufirst%3DJ.%26aulast%3DBrochier%26aufirst%3DC.%26aulast%3DVistoli%26aufirst%3DG.%26aulast%3DLangley%26aufirst%3DB.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DRational%2520design%2520and%2520simple%2520chemistry%2520yield%2520a%2520superior%252C%2520neuroprotective%2520HDAC6%2520inhibitor%252C%2520tubastatin%2520A%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D10842%26epage%3D10846%26doi%3D10.1021%2Fja102758v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Yang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span> </span><span class="NLM_article-title">Discovery of selective histone deacetylase 6 inhibitors using the quinazoline as the cap for the treatment of cancer</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01342</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01342" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Z.+Yangauthor=T.+Wangauthor=F.+Wangauthor=T.+Niuauthor=Z.+Liuauthor=X.+Chenauthor=C.+Longauthor=M.+Tangauthor=D.+Caoauthor=X.+Wangauthor=W.+Xiangauthor=Y.+Yiauthor=L.+Maauthor=J.+Youauthor=L.+Chen&title=Discovery+of+selective+histone+deacetylase+6+inhibitors+using+the+quinazoline+as+the+cap+for+the+treatment+of+cancer&doi=10.1021%2Facs.jmedchem.5b01342"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01342%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DNiu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLong%26aufirst%3DC.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DYi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520selective%2520histone%2520deacetylase%25206%2520inhibitors%2520using%2520the%2520quinazoline%2520as%2520the%2520cap%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26doi%3D10.1021%2Facs.jmedchem.5b01342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Kozikowski, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapadar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luchini, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Billadeau, D. D.</span><span> </span><span class="NLM_article-title">Use of the nitrile oxide cycloaddition (NOC) reaction for molecular probe generation: a new class of enzyme selective histone deacetylase inhibitors (HDACIs) showing picomolar activity at HDAC6</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4370</span><span class="NLM_x">–</span> <span class="NLM_lpage">4373</span><span class="refDoi"> DOI: 10.1021/jm8002894</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8002894" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovVGmtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=4370-4373&author=A.+P.+Kozikowskiauthor=S.+Tapadarauthor=D.+N.+Luchiniauthor=K.+H.+Kimauthor=D.+D.+Billadeau&title=Use+of+the+nitrile+oxide+cycloaddition+%28NOC%29+reaction+for+molecular+probe+generation%3A+a+new+class+of+enzyme+selective+histone+deacetylase+inhibitors+%28HDACIs%29+showing+picomolar+activity+at+HDAC6&doi=10.1021%2Fjm8002894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Use of the Nitrile Oxide Cycloaddition (NOC) Reaction for Molecular Probe Generation: A New Class of Enzyme Selective Histone Deacetylase Inhibitors (HDACIs) Showing Picomolar Activity at HDAC6</span></div><div class="casAuthors">Kozikowski, Alan P.; Tapadar, Subhasish; Luchini, Doris N.; Kim, Ki Hwan; Billadeau, Daniel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4370-4373</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of hydroxamate based HDAC inhibitors contg. a phenylisoxazole as the CAP group has been synthesized using nitrile oxide cycloaddn. chem.  An HDAC6 selective inhibitor having a potency of ∼2 picomolar was identified.  Some of the compds. were examd. for their ability to block pancreatic cancer cell growth and found to be about 10-fold more potent than SAHA (suberoylanilidehydroxamic acid).  This research provides valuable, new mol. probes for use in exploring HDAC biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ7Ca9tXdBvbVg90H21EOLACvtfcHk0lhmFs1kyQIeDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovVGmtbk%253D&md5=006c71fe6f2bb09c2c65196e134a695e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm8002894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8002894%26sid%3Dliteratum%253Aachs%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DTapadar%26aufirst%3DS.%26aulast%3DLuchini%26aufirst%3DD.%2BN.%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DBilladeau%26aufirst%3DD.%2BD.%26atitle%3DUse%2520of%2520the%2520nitrile%2520oxide%2520cycloaddition%2520%2528NOC%2529%2520reaction%2520for%2520molecular%2520probe%2520generation%253A%2520a%2520new%2520class%2520of%2520enzyme%2520selective%2520histone%2520deacetylase%2520inhibitors%2520%2528HDACIs%2529%2520showing%2520picomolar%2520activity%2520at%2520HDAC6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D4370%26epage%3D4373%26doi%3D10.1021%2Fjm8002894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Mwakwari, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guerrant, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oyelere, A. K.</span><span> </span><span class="NLM_article-title">Macrocyclic histone deacetylase inhibitors</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1423</span><span class="NLM_x">–</span> <span class="NLM_lpage">1440</span><span class="refDoi"> DOI: 10.2174/156802610792232079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.2174%2F156802610792232079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=20536416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=1423-1440&author=S.+C.+Mwakwariauthor=V.+Patilauthor=W.+Guerrantauthor=A.+K.+Oyelere&title=Macrocyclic+histone+deacetylase+inhibitors&doi=10.2174%2F156802610792232079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic histone deacetylase inhibitors</span></div><div class="casAuthors">Mwakwari, Sandra C.; Patil, Vishal; Guerrant, William; Oyelere, Adegboyega K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1423-1440</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone deacetylase inhibitors (HDACi) are an emerging class of novel anti-cancer drugs that cause growth arrest, differentiation, and apoptosis of tumor cells.  In addn., they have shown promise as anti-parasitic, anti-neurodegenerative, anti-rheumatol. and immunosuppressant agents.  To date, several structurally distinct small mol. HDACi have been reported including aryl hydroxamates, benzamides, short-chain fatty acids, electrophilic ketones, and macrocyclic peptides.  Macrocyclic HDACi possess the most complex cap-groups which interact with HDAC enzyme's outer rim and have demonstrated excellent HDAC inhibition potency and isoform selectivity.  This review focuses on the recent progress and current state of macrocyclic HDACi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXzgRmfQBx27Vg90H21EOLACvtfcHk0liFDNFPDWsGBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFajurrF&md5=d027b0380e04f364fb0985a350e99c60</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.2174%2F156802610792232079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802610792232079%26sid%3Dliteratum%253Aachs%26aulast%3DMwakwari%26aufirst%3DS.%2BC.%26aulast%3DPatil%26aufirst%3DV.%26aulast%3DGuerrant%26aufirst%3DW.%26aulast%3DOyelere%26aufirst%3DA.%2BK.%26atitle%3DMacrocyclic%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2010%26volume%3D10%26spage%3D1423%26epage%3D1440%26doi%3D10.2174%2F156802610792232079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Cole, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boone, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christianson, D. W.</span><span> </span><span class="NLM_article-title">Structural basis of the antiproliferative activity of largazole, a depsipeptide inhibitor of the histone deacetylases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">12474</span><span class="NLM_x">–</span> <span class="NLM_lpage">12477</span><span class="refDoi"> DOI: 10.1021/ja205972n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja205972n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=12474-12477&author=K.+E.+Coleauthor=D.+P.+Dowlingauthor=M.+A.+Booneauthor=A.+J.+Phillipsauthor=D.+W.+Christianson&title=Structural+basis+of+the+antiproliferative+activity+of+largazole%2C+a+depsipeptide+inhibitor+of+the+histone+deacetylases&doi=10.1021%2Fja205972n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of the Antiproliferative Activity of Largazole, a Depsipeptide Inhibitor of the Histone Deacetylases</span></div><div class="casAuthors">Cole, Kathryn E.; Dowling, Daniel P.; Boone, Matthew A.; Phillips, Andrew J.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12474-12477</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Largazole is a macrocyclic depsipeptide originally isolated from the marine cyanobacterium Symploca sp., which is indigenous to the warm, blue-green waters of Key Largo, Florida (whence largazole derives its name).  Largazole contains an unusual thiazoline-thiazole ring system that rigidifies its macrocyclic skeleton, and it also contains a lipophilic thioester side chain.  Hydrolysis of the thioester in vivo yields largazole thiol, which exhibits remarkable antiproliferative effects and is believed to be the most potent inhibitor of the metal-dependent histone deacetylases (HDACs).  Here, the 2.14 Å-resoln. crystal structure of the HDAC8-largazole thiol complex is the first of an HDAC complexed with a macrocyclic inhibitor and reveals that ideal thiolate-zinc coordination geometry is the key chem. feature responsible for its exceptional affinity and biol. activity.  Notably, the core structure of largazole is conserved in romidepsin, a depsipeptide natural product formulated as the drug Istodax recently approved for cancer chemotherapy.  Accordingly, the structure of the HDAC8-largazole thiol complex is the first to illustrate the mode of action of a new class of therapeutically important HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-DXxueVrU6bVg90H21EOLACvtfcHk0liFDNFPDWsGBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D&md5=9d05a3f1fd412f119ced61aaa771ef28</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja205972n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja205972n%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DK.%2BE.%26aulast%3DDowling%26aufirst%3DD.%2BP.%26aulast%3DBoone%26aufirst%3DM.%2BA.%26aulast%3DPhillips%26aufirst%3DA.%2BJ.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructural%2520basis%2520of%2520the%2520antiproliferative%2520activity%2520of%2520largazole%252C%2520a%2520depsipeptide%2520inhibitor%2520of%2520the%2520histone%2520deacetylases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D12474%26epage%3D12477%26doi%3D10.1021%2Fja205972n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Marek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moraes Mourao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caby, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stolfa, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidtkunz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancelot, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrade, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renaud, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavarelli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romier, C.</span><span> </span><span class="NLM_article-title">Structural basis for the inhibition of histone deacetylase 8 (HDAC8), a key epigenetic player in the blood fluke Schistosoma mansoni</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003645</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1003645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.ppat.1003645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=24086136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOnurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003645&author=M.+Marekauthor=S.+Kannanauthor=A.+T.+Hauserauthor=M.+Moraes+Mouraoauthor=S.+Cabyauthor=V.+Curaauthor=D.+A.+Stolfaauthor=K.+Schmidtkunzauthor=J.+Lancelotauthor=L.+Andradeauthor=J.+P.+Renaudauthor=G.+Oliveiraauthor=W.+Sipplauthor=M.+Jungauthor=J.+Cavarelliauthor=R.+J.+Pierceauthor=C.+Romier&title=Structural+basis+for+the+inhibition+of+histone+deacetylase+8+%28HDAC8%29%2C+a+key+epigenetic+player+in+the+blood+fluke+Schistosoma+mansoni&doi=10.1371%2Fjournal.ppat.1003645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the inhibition of histone deacetylase 8 (HDAC8), a key epigenetic player in the blood fluke Schistosoma mansoni</span></div><div class="casAuthors">Marek, Martin; Kannan, Srinivasaraghavan; Hauser, Alexander-Thomas; Mourao, Marina Moraes; Caby, Stephanie; Cura, Vincent; Stolfa, Diana A.; Schmidtkunz, Karin; Lancelot, Julien; Andrade, Luiza; Renaud, Jean-Paul; Oliveira, Guilherme; Sippl, Wolfgang; Jung, Manfred; Cavarelli, Jean; Pierce, Raymond J.; Romier, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1003645</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The treatment of schistosomiasis, a disease caused by blood flukes parasites of the Schistosoma genus, depends on the intensive use of a single drug, praziquantel, which increases the likelihood of the development of drug-resistant parasite strains and renders the search for new drugs a strategic priority.  Currently, inhibitors of human epigenetic enzymes are actively investigated as novel anti-cancer drugs and have the potential to be used as new anti-parasitic agents.  Here, we report that Schistosoma mansoni histone deacetylase 8 (smHDAC8), the most expressed class I HDAC isotype in this organism, is a functional acetyl--lysine deacetylase that plays an important role in parasite infectivity.  The crystal structure of smHDAC8 shows that this enzyme adopts a canonical α/β HDAC fold, with specific solvent exposed loops corresponding to insertions in the schistosome HDAC8 sequence.  Importantly, structures of smHDAC8 in complex with generic HDAC inhibitors revealed specific structural changes in the smHDAC8 active site that cannot be accommodated by human HDACs.  Using a structure-based approach, we identified several small-mol. inhibitors that build on these specificities.  These mols. exhibit an inhibitory effect on smHDAC8 but show reduced affinity for human HDACs.  Crucially, we show that a newly identified smHDAC8 inhibitor has the capacity to induce apoptosis and mortality in schistosomes.  Taken together, our biol. and structural findings define the framework for the rational design of small-mol. inhibitors specifically interfering with schistosome epigenetic mechanisms, and further support an anti-parasitic epigenome targeting strategy to treat neglected diseases caused by eukaryotic pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptIGu8lR_TxrVg90H21EOLACvtfcHk0liyC1GyTjrHFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOnurzI&md5=9fef2938ff73df80fc60b739c878f9a6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1003645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1003645%26sid%3Dliteratum%253Aachs%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DMoraes%2BMourao%26aufirst%3DM.%26aulast%3DCaby%26aufirst%3DS.%26aulast%3DCura%26aufirst%3DV.%26aulast%3DStolfa%26aufirst%3DD.%2BA.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DLancelot%26aufirst%3DJ.%26aulast%3DAndrade%26aufirst%3DL.%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DOliveira%26aufirst%3DG.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DCavarelli%26aufirst%3DJ.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520histone%2520deacetylase%25208%2520%2528HDAC8%2529%252C%2520a%2520key%2520epigenetic%2520player%2520in%2520the%2520blood%2520fluke%2520Schistosoma%2520mansoni%26jtitle%3DPLoS%2520Pathog.%26date%3D2013%26volume%3D9%26spage%3De1003645%26doi%3D10.1371%2Fjournal.ppat.1003645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Marcaurelle, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Comer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dandapani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duvall, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kesavan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. D. t.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marie, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulrooney, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandya, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryba, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suh, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akella, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ross, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fass, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reis, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, M. A.</span><span> </span><span class="NLM_article-title">An aldol-based build/couple/pair strategy for the synthesis of medium- and large-sized rings: discovery of macrocyclic histone deacetylase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">16962</span><span class="NLM_x">–</span> <span class="NLM_lpage">16976</span><span class="refDoi"> DOI: 10.1021/ja105119r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja105119r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCrsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=16962-16976&author=L.+A.+Marcaurelleauthor=E.+Comerauthor=S.+Dandapaniauthor=J.+R.+Duvallauthor=B.+Gerardauthor=S.+Kesavanauthor=M.+D.+t.+Leeauthor=H.+Liuauthor=J.+T.+Loweauthor=J.+C.+Marieauthor=C.+A.+Mulrooneyauthor=B.+A.+Pandyaauthor=A.+Rowleyauthor=T.+D.+Rybaauthor=B.+C.+Suhauthor=J.+Weiauthor=D.+W.+Youngauthor=L.+B.+Akellaauthor=N.+T.+Rossauthor=Y.+L.+Zhangauthor=D.+M.+Fassauthor=S.+A.+Reisauthor=W.+N.+Zhaoauthor=S.+J.+Haggartyauthor=M.+Palmerauthor=M.+A.+Foley&title=An+aldol-based+build%2Fcouple%2Fpair+strategy+for+the+synthesis+of+medium-+and+large-sized+rings%3A+discovery+of+macrocyclic+histone+deacetylase+inhibitors&doi=10.1021%2Fja105119r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An Aldol-Based Build/Couple/Pair Strategy for the Synthesis of Medium- and Large-Sized Rings: Discovery of Macrocyclic Histone Deacetylase Inhibitors</span></div><div class="casAuthors">Marcaurelle, Lisa A.; Comer, Eamon; Dandapani, Sivaraman; Duvall, Jeremy R.; Gerard, Baudouin; Kesavan, Sarathy; Lee, Maurice D.; Liu, Haibo; Lowe, Jason T.; Marie, Jean-Charles; Mulrooney, Carol A.; Pandya, Bhaumik A.; Rowley, Ann; Ryba, Troy D.; Suh, Byung-Chul; Wei, Jingqiang; Young, Damian W.; Akella, Lakshmi B.; Ross, Nathan T.; Zhang, Yan-Ling; Fass, Daniel M.; Reis, Surya A.; Zhao, Wen-Ning; Haggarty, Stephen J.; Palmer, Michelle; Foley, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">16962-16976</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nonracemic macrocycles such as I (PMB = 4-MeOC6H4CH2; Boc = tert-butoxycarbonyl), II, and III were prepd. using diastereoselective aldol addn. reactions followed by coupling with nonracemic protected alaninols and subsequent nucleophilic arom. substitution, regioselective azide-alkyne cycloaddn., and ring-closing metathesis reactions to generate a set of macrocycles of differing ring sizes and stereochemistries.  Asym. syn- and anti-aldol reactions were performed to produce four stereoisomers of a Boc-protected γ-amino acid; prepn. of both enantiomers of O-PMB-protected alaninol followed by coupling and amide redn. yielded a set of eight substituted secondary amines.  Acylation of the secondary amines with nitrofluorophenyl acyl chlorides followed by desilylation and intramol. nucleophilic arom. substitution yielded macrocycles such as I.  Acylation of the secondary amines with 4-azidobutanoic acid, desilylation, O-alkylation with propargyl bromides, and ruthenium- or copper-catalyzed azide-alkyne cycloaddns. provided triazole-contg. macrocycles such as II regioselectively.  Acylation of the secondary amines with a nonracemic pentenyloxybenzoyl chloride, desilylation, O-alkylation with allyl bromide, and ring-closing metathesis yielded macrocycles which can be reduced and further reacted to give macrocycles such as III.  Redn. of the alkene and nitro groups of ring-closing metathesis-derived macrocycles followed by Fmoc protection, Boc deprotection, and reprotection with allyloxycarbonyl chloride yields a set of sixteen stereoisomeric macrocycles which are use to generate a 14 400-membered library of macrolactams.  Screening of the macrocycle library led to the discovery of a novel class of histone deacetylase inhibitors, which display mixed enzyme inhibition, and led to increased levels of acetylation in a primary mouse neuron culture.  Development of stereochem. structure/activity relationships was made possible by screening all 16 stereoisomers of the macrolactams produced through the aldol-based build/couple/pair strategy.  [This abstr. record is one of 3 records for this document necessitated by the large no. of index entries required to fully index the document and publication system constraints.].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU_CeBimobNLVg90H21EOLACvtfcHk0liyC1GyTjrHFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCrsbzN&md5=e1dea4e0e2accf8451f2b5e712a61612</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja105119r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja105119r%26sid%3Dliteratum%253Aachs%26aulast%3DMarcaurelle%26aufirst%3DL.%2BA.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DDandapani%26aufirst%3DS.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DGerard%26aufirst%3DB.%26aulast%3DKesavan%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BD.%2Bt.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLowe%26aufirst%3DJ.%2BT.%26aulast%3DMarie%26aufirst%3DJ.%2BC.%26aulast%3DMulrooney%26aufirst%3DC.%2BA.%26aulast%3DPandya%26aufirst%3DB.%2BA.%26aulast%3DRowley%26aufirst%3DA.%26aulast%3DRyba%26aufirst%3DT.%2BD.%26aulast%3DSuh%26aufirst%3DB.%2BC.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DYoung%26aufirst%3DD.%2BW.%26aulast%3DAkella%26aufirst%3DL.%2BB.%26aulast%3DRoss%26aufirst%3DN.%2BT.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DFass%26aufirst%3DD.%2BM.%26aulast%3DReis%26aufirst%3DS.%2BA.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26atitle%3DAn%2520aldol-based%2520build%252Fcouple%252Fpair%2520strategy%2520for%2520the%2520synthesis%2520of%2520medium-%2520and%2520large-sized%2520rings%253A%2520discovery%2520of%2520macrocyclic%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D16962%26epage%3D16976%26doi%3D10.1021%2Fja105119r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Vaidya, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neelarapu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madriaga, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendonca, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdelkarim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Breemen, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blond, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petukhov, P. A.</span><span> </span><span class="NLM_article-title">Novel histone deacetylase 8 ligands without a zinc chelating group: exploring an ’upside-down’ binding pose</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">6621</span><span class="NLM_x">–</span> <span class="NLM_lpage">6627</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2012.08.104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.bmcl.2012.08.104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=6621-6627&author=A.+S.+Vaidyaauthor=R.+Neelarapuauthor=A.+Madriagaauthor=H.+Baiauthor=E.+Mendoncaauthor=H.+Abdelkarimauthor=R.+B.+van+Breemenauthor=S.+Y.+Blondauthor=P.+A.+Petukhov&title=Novel+histone+deacetylase+8+ligands+without+a+zinc+chelating+group%3A+exploring+an+%E2%80%99upside-down%E2%80%99+binding+pose&doi=10.1016%2Fj.bmcl.2012.08.104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.08.104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.08.104%26sid%3Dliteratum%253Aachs%26aulast%3DVaidya%26aufirst%3DA.%2BS.%26aulast%3DNeelarapu%26aufirst%3DR.%26aulast%3DMadriaga%26aufirst%3DA.%26aulast%3DBai%26aufirst%3DH.%26aulast%3DMendonca%26aufirst%3DE.%26aulast%3DAbdelkarim%26aufirst%3DH.%26aulast%3Dvan%2BBreemen%26aufirst%3DR.%2BB.%26aulast%3DBlond%26aufirst%3DS.%2BY.%26aulast%3DPetukhov%26aufirst%3DP.%2BA.%26atitle%3DNovel%2520histone%2520deacetylase%25208%2520ligands%2520without%2520a%2520zinc%2520chelating%2520group%253A%2520exploring%2520an%2520%25E2%2580%2599upside-down%25E2%2580%2599%2520binding%2520pose%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D6621%26epage%3D6627%26doi%3D10.1016%2Fj.bmcl.2012.08.104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Vickers, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leman, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghadiri, M. R.</span><span> </span><span class="NLM_article-title">Discovery of HDAC inhibitors that lack an active site Zn(2+)-binding functional group</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">508</span><span class="refDoi"> DOI: 10.1021/ml300081u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300081u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=505-508&author=C.+J.+Vickersauthor=C.+A.+Olsenauthor=L.+J.+Lemanauthor=M.+R.+Ghadiri&title=Discovery+of+HDAC+inhibitors+that+lack+an+active+site+Zn%282%2B%29-binding+functional+group&doi=10.1021%2Fml300081u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml300081u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300081u%26sid%3Dliteratum%253Aachs%26aulast%3DVickers%26aufirst%3DC.%2BJ.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26aulast%3DLeman%26aufirst%3DL.%2BJ.%26aulast%3DGhadiri%26aufirst%3DM.%2BR.%26atitle%3DDiscovery%2520of%2520HDAC%2520inhibitors%2520that%2520lack%2520an%2520active%2520site%2520Zn%25282%252B%2529-binding%2520functional%2520group%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D505%26epage%3D508%26doi%3D10.1021%2Fml300081u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Cole, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boone, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christianson, D. W.</span><span> </span><span class="NLM_article-title">Structural basis of the antiproliferative activity of largazole, a depsipeptide inhibitor of the histone deacetylases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">12474</span><span class="NLM_x">–</span> <span class="NLM_lpage">12477</span><span class="refDoi"> DOI: 10.1021/ja205972n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja205972n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=12474-12477&author=K.+E.+Coleauthor=D.+P.+Dowlingauthor=M.+A.+Booneauthor=A.+J.+Phillipsauthor=D.+W.+Christianson&title=Structural+basis+of+the+antiproliferative+activity+of+largazole%2C+a+depsipeptide+inhibitor+of+the+histone+deacetylases&doi=10.1021%2Fja205972n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of the Antiproliferative Activity of Largazole, a Depsipeptide Inhibitor of the Histone Deacetylases</span></div><div class="casAuthors">Cole, Kathryn E.; Dowling, Daniel P.; Boone, Matthew A.; Phillips, Andrew J.; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12474-12477</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Largazole is a macrocyclic depsipeptide originally isolated from the marine cyanobacterium Symploca sp., which is indigenous to the warm, blue-green waters of Key Largo, Florida (whence largazole derives its name).  Largazole contains an unusual thiazoline-thiazole ring system that rigidifies its macrocyclic skeleton, and it also contains a lipophilic thioester side chain.  Hydrolysis of the thioester in vivo yields largazole thiol, which exhibits remarkable antiproliferative effects and is believed to be the most potent inhibitor of the metal-dependent histone deacetylases (HDACs).  Here, the 2.14 Å-resoln. crystal structure of the HDAC8-largazole thiol complex is the first of an HDAC complexed with a macrocyclic inhibitor and reveals that ideal thiolate-zinc coordination geometry is the key chem. feature responsible for its exceptional affinity and biol. activity.  Notably, the core structure of largazole is conserved in romidepsin, a depsipeptide natural product formulated as the drug Istodax recently approved for cancer chemotherapy.  Accordingly, the structure of the HDAC8-largazole thiol complex is the first to illustrate the mode of action of a new class of therapeutically important HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-DXxueVrU6bVg90H21EOLACvtfcHk0lg8pGPkabhnWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFSrtb0%253D&md5=9d05a3f1fd412f119ced61aaa771ef28</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja205972n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja205972n%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DK.%2BE.%26aulast%3DDowling%26aufirst%3DD.%2BP.%26aulast%3DBoone%26aufirst%3DM.%2BA.%26aulast%3DPhillips%26aufirst%3DA.%2BJ.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DStructural%2520basis%2520of%2520the%2520antiproliferative%2520activity%2520of%2520largazole%252C%2520a%2520depsipeptide%2520inhibitor%2520of%2520the%2520histone%2520deacetylases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D12474%26epage%3D12477%26doi%3D10.1021%2Fja205972n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Bowers, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. M.</span><span> </span><span class="NLM_article-title">Total synthesis and biological mode of action of largazole: a potent class I histone deacetylase inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">11219</span><span class="NLM_x">–</span> <span class="NLM_lpage">11222</span><span class="refDoi"> DOI: 10.1021/ja8033763</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8033763" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXos12ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=11219-11222&author=A.+Bowersauthor=N.+Westauthor=J.+Tauntonauthor=S.+L.+Schreiberauthor=J.+E.+Bradnerauthor=R.+M.+Williams&title=Total+synthesis+and+biological+mode+of+action+of+largazole%3A+a+potent+class+I+histone+deacetylase+inhibitor&doi=10.1021%2Fja8033763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Total synthesis and biological mode of action of largazole: A potent Class I histone deacetylase inhibitor</span></div><div class="casAuthors">Bowers, Albert; West, Nathan; Taunton, Jack; Schreiber, Stuart L.; Bradner, James E.; Williams, Robert M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11219-11222</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The efficient total synthesis of the recently described natural substance largazole and its active metabolite largazole thiol is described.  The synthesis required eight linear steps and proceeded in 37% overall yield.  It is demonstrated that largazole is a pro-rug that is activated by removal of the octanoyl residue from the 3-hydroxy-7-mercaptohept-4-enoic acid moiety to generate the active metabolite largazole thiol, which is an extraordinarily potent Class I histone deacetylase inhibitor.  Synthetic largazole and largazole thiol have been evaluated side-by-side with FK228 and SAHA for inhibition of HDACs 1, 2, 3, and 6.  Largazole and largazole thiol were further assayed for cytotoxic activity against a panel of chemoresistant melanoma cell lines, and it was found that largazole is substantially more cytotoxic than largazole thiol; this difference is attributed to differences in the cell permeability of the two substances.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdAVdDfScHCrVg90H21EOLACvtfcHk0lg8pGPkabhnWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXos12ru78%253D&md5=0063841a5d6a996484cdb329ff6baa64</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja8033763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8033763%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DA.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWilliams%26aufirst%3DR.%2BM.%26atitle%3DTotal%2520synthesis%2520and%2520biological%2520mode%2520of%2520action%2520of%2520largazole%253A%2520a%2520potent%2520class%2520I%2520histone%2520deacetylase%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D11219%26epage%3D11222%26doi%3D10.1021%2Fja8033763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Che, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Engineering cyclic tetrapeptides containing chimeric amino acids as preferred reverse-turn scaffolds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1021/jm0507072</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0507072" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1GktLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=111-124&author=Y.+Cheauthor=G.+R.+Marshall&title=Engineering+cyclic+tetrapeptides+containing+chimeric+amino+acids+as+preferred+reverse-turn+scaffolds&doi=10.1021%2Fjm0507072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering Cyclic Tetrapeptides Containing Chimeric Amino Acids as Preferred Reverse-Turn Scaffolds</span></div><div class="casAuthors">Che, Ye; Marshall, Garland R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four residues making almost a complete 180° turn in the direction of the peptide chain define a reverse turn, a common motif and recognition site in proteins.  Cyclization between residues i and i+3 and incorporation of heterochiral dipeptides (such as D-Pro-L-Pro) in the i+1 and i+2 positions are used to constrain a peptide to a reverse-turn conformation.  A combined approach, cyclic tetrapeptides (CTPs) based on heterochiral dipeptides of chimeric amino acids, is evaluated as minimalist scaffolds for reverse-turn conformations.  Cyclo(D-Pro-L-Pro-D-Pro-L-Pro) has been studied with d. functional theory (DFT) calcns. and mol. dynamics simulations.  The all-trans amide conformer was the most stable in vacuo, while the cis-trans-cis-trans (ctct) or trans-cis-trans-cis (tctc) amide conformer was more favored in water due to its large dipole moment.  Different conformations could be selectively stabilized by different substitutions on the proline rings.  Due to the small 12-membered ring and exocyclic constraints, conformational interconversions could only occur at high temp.  The presence of seven hydrogens on each ring that could be functionalized offers an overwhelming diversity to design mols. to probe receptors.  The spatial relationships of Cα-Cβ vectors of reverse turns in proteins were subjected to principal component anal. for detn. of the relative orientation of the Cα-Cβ vectors.  Most reverse-turn structures could be mimicked effectively with a subset of CTP scaffolds with an root-mean-square displacement (RMSD) of approx. 0.5 Å.  Structural diversity of CTP scaffolds could be enhanced by the incorporation of proline analogs, such as azaproline (azPro) or pipecolic (Pip), azapipecolic (azPip), nipecotic (Nip), and isonipecotic (Inp) acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoChaXRJzSdI7Vg90H21EOLACvtfcHk0lj0DIaLg5Lecg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1GktLbE&md5=13a4a7881b1e7a5e0eb884d4aedfec06</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm0507072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0507072%26sid%3Dliteratum%253Aachs%26aulast%3DChe%26aufirst%3DY.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DEngineering%2520cyclic%2520tetrapeptides%2520containing%2520chimeric%2520amino%2520acids%2520as%2520preferred%2520reverse-turn%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D111%26epage%3D124%26doi%3D10.1021%2Fjm0507072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Chalmers, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Pro-D-NMe-amino acid and D-Pro-NMe-amino acid: simple, efficient reverse-turn constraints</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">117</span><span class="NLM_x">, </span> <span class="NLM_fpage">5927</span><span class="NLM_x">–</span> <span class="NLM_lpage">5937</span><span class="refDoi"> DOI: 10.1021/ja00127a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00127a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK2MXlvFahtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=1995&pages=5927-5937&author=D.+K.+Chalmersauthor=G.+R.+Marshall&title=Pro-D-NMe-amino+acid+and+D-Pro-NMe-amino+acid%3A+simple%2C+efficient+reverse-turn+constraints&doi=10.1021%2Fja00127a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Pro-D-NMe-Amino Acid and D-Pro-NMe-Amino Acid: Simple, Efficient Reverse-Turn Constraints</span></div><div class="casAuthors">Chalmers, David K.; Marshall, Garland R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5927-37</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of mimetics that, when incorporated into a peptide, constrain it to adopt a reverse turn is currently of considerable interest.  Numerous bicyclic dipeptide surrogates have been suggested which fix two of the four backbone dihedral angles of residues i + 1 and i + 2 of the turn.  In this paper, the authors report the conformational anal. of tetrapeptides contg. several bicyclic mimetics, sequences contg. proline and other N-methylamino acids in the i + 1 and i + 2 positions of the turn, and control peptide sequences using a Monte Carlo conformational search followed by mol. dynamics simulation in water as implicitly represented by a solvation model.  These studies indicate that the sequences D-Pro-N-methylamino acid (D-Pro-NMe-AA) and Pro-D-NMe-AA are effective at stabilizing the trans conformer of the amide bond between residues i + 1 and i + 2 and constraining three of the four backbone dihedral angles to those assocd. with structures which satisfy several criteria for reverse turns.  In many cases, these sequences were more effective than more complicated polycyclic mimetics.  D-Pro-NMe-AA and Pro-D-NMe-AA are compatible with conventional peptide synthesis methods and should provide a simple method to probe receptors for reverse-turn recognition through combinatorial libraries and structure-activity studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCnmTHPFnVXLVg90H21EOLACvtfcHk0lj0DIaLg5Lecg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlvFahtr4%253D&md5=8d7eef279cf77c752898c4521d5bcd77</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fja00127a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00127a004%26sid%3Dliteratum%253Aachs%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DPro-D-NMe-amino%2520acid%2520and%2520D-Pro-NMe-amino%2520acid%253A%2520simple%252C%2520efficient%2520reverse-turn%2520constraints%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D5927%26epage%3D5937%26doi%3D10.1021%2Fja00127a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Takeuchi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Conformational analysis of reverse-turn constraints by N-methylation and N-hydroxylation of amide bonds in peptides and non-peptide mimetics</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">5363</span><span class="NLM_x">–</span> <span class="NLM_lpage">5372</span><span class="refDoi"> DOI: 10.1021/ja970855k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja970855k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADyaK1cXjtlWku74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=5363-5372&author=Y.+Takeuchiauthor=G.+R.+Marshall&title=Conformational+analysis+of+reverse-turn+constraints+by+N-methylation+and+N-hydroxylation+of+amide+bonds+in+peptides+and+non-peptide+mimetics&doi=10.1021%2Fja970855k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Reverse-Turn Constraints by N-Methylation and N-Hydroxylation of Amide Bonds in Peptides and Non-Peptide Mimetics</span></div><div class="casAuthors">Takeuchi, Yasuo; Marshall, Garland R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5363-5372</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several non-peptide systems have been designed to mimic different types of reverse turns.  The incorporation of some of these mimetics into biol. active peptides has led to peptidomimetics with enhanced activity or metabolic stability.  This paper reports the conformational anal. of tetrapeptides contg. several bicyclic mimetics, sequences contg. Pro, other N-Me and N-hydroxy amino acids, and pipecolic acid at residue i + 2 of the turn, and control peptide sequences using the Monte Carlo/stochastic dynamics simulation with the new set of AMBER* parameters for proline-contg. peptides in water as implicitly represented by the GB/SA solvation model.  Simple N-methylation (Pro-D-NMeAA and D-Pro-NMeAA) and N-hydroxylation of the amide bond between residues i + 1 and i + 2 or inclusion of the larger ring homolog pipecolic acid (D-Pro-Pip) in the third position (i + 2) causes significant nucleation of reverse-turn structures.  Spirotricycle analogs restrict three of the four torsion angles that characterize the type II β-turn.  Spirolactam analogs also restrict two of the four torsion angles as effective β-turn constraints.  However, the geometry of a turn induced by indolizidinone and BTD differs significantly from that of an ideal β-turn and (S)-indolizidinone is more effective as a reverse turn than as a β-turn mimetic.  These systems provide useful conformational constraints when incorporated into the structure of selected bioactive peptides.  Such analogs can scan receptors for biol. recognition of β-turn scaffolds with oriented side chains through combinatorial libraries to efficiently develop three-dimensional structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXBY0-f_KpY7Vg90H21EOLACvtfcHk0lj0DIaLg5Lecg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtlWku74%253D&md5=c92318c941c199b8871372d5789e91fe</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja970855k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja970855k%26sid%3Dliteratum%253Aachs%26aulast%3DTakeuchi%26aufirst%3DY.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DConformational%2520analysis%2520of%2520reverse-turn%2520constraints%2520by%2520N-methylation%2520and%2520N-hydroxylation%2520of%2520amide%2520bonds%2520in%2520peptides%2520and%2520non-peptide%2520mimetics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D5363%26epage%3D5372%26doi%3D10.1021%2Fja970855k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Bowers, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greshock, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiest, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span> </span><span class="NLM_article-title">Synthesis and conformation-activity relationships of the peptide isosteres of FK228 and largazole</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">2900</span><span class="NLM_x">–</span> <span class="NLM_lpage">2905</span><span class="refDoi"> DOI: 10.1021/ja807772w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja807772w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=2900-2905&author=A.+A.+Bowersauthor=T.+J.+Greshockauthor=N.+Westauthor=G.+Estiuauthor=S.+L.+Schreiberauthor=O.+Wiestauthor=R.+M.+Williamsauthor=J.+E.+Bradner&title=Synthesis+and+conformation-activity+relationships+of+the+peptide+isosteres+of+FK228+and+largazole&doi=10.1021%2Fja807772w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Conformation-Activity Relationships of the Peptide Isosteres of FK228 and Largazole</span></div><div class="casAuthors">Bowers, Albert A.; Greshock, Thomas J.; West, Nathan; Estiu, Guillermina; Schreiber, Stuart L.; Wiest, Olaf; Williams, Robert M.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2900-2905</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The peptide isosteres (10 and 11) of the naturally occurring and potent histone deacetylase (HDAC) inhibitors FK228 and largazole have been synthesized and evaluated side-by-side with FK228, largazole, and SAHA for inhibition of the class I HDACs 1, 2, 3 and 6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA7om4rJhoarVg90H21EOLACvtfcHk0lj0DIaLg5Lecg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSltbo%253D&md5=e89f48feda9a2d8c92a9120f1a8428f9</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fja807772w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja807772w%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DA.%2BA.%26aulast%3DGreshock%26aufirst%3DT.%2BJ.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DWilliams%26aufirst%3DR.%2BM.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSynthesis%2520and%2520conformation-activity%2520relationships%2520of%2520the%2520peptide%2520isosteres%2520of%2520FK228%2520and%2520largazole%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D2900%26epage%3D2905%26doi%3D10.1021%2Fja807772w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Ballante, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">An Automated Strategy for Binding-Pose Selection and Docking Assessment in Structure-Based Drug Design</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00603</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00603" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=F.+Ballanteauthor=G.+R.+Marshall&title=An+Automated+Strategy+for+Binding-Pose+Selection+and+Docking+Assessment+in+Structure-Based+Drug+Design&doi=10.1021%2Facs.jcim.5b00603"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00603%26sid%3Dliteratum%253Aachs%26aulast%3DBallante%26aufirst%3DF.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DAn%2520Automated%2520Strategy%2520for%2520Binding-Pose%2520Selection%2520and%2520Docking%2520Assessment%2520in%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26doi%3D10.1021%2Facs.jcim.5b00603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Nishino, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jose, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komatsu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, M.</span><span> </span><span class="NLM_article-title">Chlamydocin-hydroxamic acid analogues as histone deacetylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5777</span><span class="NLM_x">–</span> <span class="NLM_lpage">5784</span><span class="refDoi"> DOI: 10.1016/j.bmc.2004.08.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.bmc.2004.08.041" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=5777-5784&author=N.+Nishinoauthor=B.+Joseauthor=R.+Shintaauthor=T.+Katoauthor=Y.+Komatsuauthor=M.+Yoshida&title=Chlamydocin-hydroxamic+acid+analogues+as+histone+deacetylase+inhibitors&doi=10.1016%2Fj.bmc.2004.08.041"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2004.08.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2004.08.041%26sid%3Dliteratum%253Aachs%26aulast%3DNishino%26aufirst%3DN.%26aulast%3DJose%26aufirst%3DB.%26aulast%3DShinta%26aufirst%3DR.%26aulast%3DKato%26aufirst%3DT.%26aulast%3DKomatsu%26aufirst%3DY.%26aulast%3DYoshida%26aufirst%3DM.%26atitle%3DChlamydocin-hydroxamic%2520acid%2520analogues%2520as%2520histone%2520deacetylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D5777%26epage%3D5784%26doi%3D10.1016%2Fj.bmc.2004.08.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Madsen, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristensen, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, C. A.</span><span> </span><span class="NLM_article-title">The effect of various zinc binding groups on inhibition of histone deacetylases 1–11</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">614</span><span class="NLM_x">–</span> <span class="NLM_lpage">626</span><span class="refDoi"> DOI: 10.1002/cmdc.201300433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fcmdc.201300433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=24375963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitVWjur3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=614-626&author=A.+S.+Madsenauthor=H.+M.+Kristensenauthor=G.+Lanzauthor=C.+A.+Olsen&title=The+effect+of+various+zinc+binding+groups+on+inhibition+of+histone+deacetylases+1%E2%80%9311&doi=10.1002%2Fcmdc.201300433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">The Effect of Various Zinc Binding Groups on Inhibition of Histone Deacetylases 1-11</span></div><div class="casAuthors">Madsen, Andreas S.; Kristensen, Helle M. E.; Lanz, Gyrithe; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">614-626</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) have the ability to cleave the acetyl groups of ε-N-acetylated lysine residues in a variety of proteins.  Given that human cells contain thousands of different acetylated lysine residues, HDACS may regulate a wide variety of processes including some implicated in conditions such as cancer and neurodegenerative disorders.  Herein we report the synthesis and in vitro biochem. profiling of a series of compds., including known inhibitors as well as novel chemotypes, that incorporate putative new zinc binding domains.  By evaluating the compd. collection against all 11 recombinant human HDACs, we found that the trifluoromethyl ketone functionality provides potent inhibition of all four subclasses of the Zn2+-dependent HDACs.  Potent inhibition was obsd. with two different scaffolds, demonstrating the efficiency of the trifluoromethyl ketone moiety as a zinc binding motif.  Interestingly, we also identified silanediol as a zinc binding group with potential for future development of non-hydroxamate class I and class IIb HDAC inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGN-Ah_s0G9rVg90H21EOLACvtfcHk0lj9f3ZL3e_2ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitVWjur3N&md5=4cc4f341ecb112034d26aeb668f3332f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300433%26sid%3Dliteratum%253Aachs%26aulast%3DMadsen%26aufirst%3DA.%2BS.%26aulast%3DKristensen%26aufirst%3DH.%2BM.%26aulast%3DLanz%26aufirst%3DG.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DThe%2520effect%2520of%2520various%2520zinc%2520binding%2520groups%2520on%2520inhibition%2520of%2520histone%2520deacetylases%25201%25E2%2580%259311%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D614%26epage%3D626%26doi%3D10.1002%2Fcmdc.201300433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Gibbs, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartzman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alumkal, J.</span><span> </span><span class="NLM_article-title">Sulforaphane destabilizes the androgen receptor in prostate cancer cells by inactivating histone deacetylase 6</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">16663</span><span class="NLM_x">–</span> <span class="NLM_lpage">16668</span><span class="refDoi"> DOI: 10.1073/pnas.0908908106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.0908908106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=16663-16668&author=A.+Gibbsauthor=J.+Schwartzmanauthor=V.+Dengauthor=J.+Alumkal&title=Sulforaphane+destabilizes+the+androgen+receptor+in+prostate+cancer+cells+by+inactivating+histone+deacetylase+6&doi=10.1073%2Fpnas.0908908106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0908908106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0908908106%26sid%3Dliteratum%253Aachs%26aulast%3DGibbs%26aufirst%3DA.%26aulast%3DSchwartzman%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DV.%26aulast%3DAlumkal%26aufirst%3DJ.%26atitle%3DSulforaphane%2520destabilizes%2520the%2520androgen%2520receptor%2520in%2520prostate%2520cancer%2520cells%2520by%2520inactivating%2520histone%2520deacetylase%25206%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D16663%26epage%3D16668%26doi%3D10.1073%2Fpnas.0908908106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Singh, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powolny, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stan, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arlotti, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahm, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marynowski, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bommareddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parise, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beumer, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, W. H.</span><span> </span><span class="NLM_article-title">Sulforaphane inhibits prostate carcinogenesis and pulmonary metastasis in TRAMP mice in association with increased cytotoxicity of natural killer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2117</span><span class="NLM_x">–</span> <span class="NLM_lpage">2125</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-3502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1158%2F0008-5472.CAN-08-3502" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2117-2125&author=S.+V.+Singhauthor=R.+Warinauthor=D.+Xiaoauthor=A.+A.+Powolnyauthor=S.+D.+Stanauthor=J.+A.+Arlottiauthor=Y.+Zengauthor=E.+R.+Hahmauthor=S.+W.+Marynowskiauthor=A.+Bommareddyauthor=D.+Desaiauthor=S.+Aminauthor=R.+A.+Pariseauthor=J.+H.+Beumerauthor=W.+H.+Chambers&title=Sulforaphane+inhibits+prostate+carcinogenesis+and+pulmonary+metastasis+in+TRAMP+mice+in+association+with+increased+cytotoxicity+of+natural+killer+cells&doi=10.1158%2F0008-5472.CAN-08-3502"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-3502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-3502%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BV.%26aulast%3DWarin%26aufirst%3DR.%26aulast%3DXiao%26aufirst%3DD.%26aulast%3DPowolny%26aufirst%3DA.%2BA.%26aulast%3DStan%26aufirst%3DS.%2BD.%26aulast%3DArlotti%26aufirst%3DJ.%2BA.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DHahm%26aufirst%3DE.%2BR.%26aulast%3DMarynowski%26aufirst%3DS.%2BW.%26aulast%3DBommareddy%26aufirst%3DA.%26aulast%3DDesai%26aufirst%3DD.%26aulast%3DAmin%26aufirst%3DS.%26aulast%3DParise%26aufirst%3DR.%2BA.%26aulast%3DBeumer%26aufirst%3DJ.%2BH.%26aulast%3DChambers%26aufirst%3DW.%2BH.%26atitle%3DSulforaphane%2520inhibits%2520prostate%2520carcinogenesis%2520and%2520pulmonary%2520metastasis%2520in%2520TRAMP%2520mice%2520in%2520association%2520with%2520increased%2520cytotoxicity%2520of%2520natural%2520killer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2117%26epage%3D2125%26doi%3D10.1158%2F0008-5472.CAN-08-3502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Whitehead, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobler, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radetich, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claiborne, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grob, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McRiner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pancost, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patnaik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shultz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tichkule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tommasi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vash, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stams, T.</span><span> </span><span class="NLM_article-title">Human HDAC isoform selectivity achieved via exploitation of the acetate release channel with structurally unique small molecule inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">4626</span><span class="NLM_x">–</span> <span class="NLM_lpage">4634</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.06.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.bmc.2011.06.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=21723733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovF2ju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=4626-4634&author=L.+Whiteheadauthor=M.+R.+Doblerauthor=B.+Radetichauthor=Y.+Zhuauthor=P.+W.+Atadjaauthor=T.+Claiborneauthor=J.+E.+Grobauthor=A.+McRinerauthor=M.+R.+Pancostauthor=A.+Patnaikauthor=W.+Shaoauthor=M.+Shultzauthor=R.+Tichkuleauthor=R.+A.+Tommasiauthor=B.+Vashauthor=P.+Wangauthor=T.+Stams&title=Human+HDAC+isoform+selectivity+achieved+via+exploitation+of+the+acetate+release+channel+with+structurally+unique+small+molecule+inhibitors&doi=10.1016%2Fj.bmc.2011.06.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Human HDAC isoform selectivity achieved via exploitation of the acetate release channel with structurally unique small molecule inhibitors</span></div><div class="casAuthors">Whitehead, Lewis; Dobler, Markus R.; Radetich, Branko; Zhu, Yanyi; Atadja, Peter W.; Claiborne, Tavina; Grob, Jonathan E.; McRiner, Andrew; Pancost, Margaret R.; Patnaik, Anup; Shao, Wenlin; Shultz, Michael; Tichkule, Ritesh; Tommasi, Ruben A.; Vash, Brian; Wang, Ping; Stams, Travis</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4626-4634</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we report the discovery of a family of novel yet simple, amino-acid derived class I HDAC inhibitors that demonstrate isoform selectivity via access to the internal acetate release channel.  Isoform selectivity criteria is discussed on the basis of X-ray crystallog. and mol. modeling of these novel inhibitors bound to HDAC8, potentially revealing insights into the mechanism of enzymic function through novel structural features revealed at the at. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgV7MIGevIxrVg90H21EOLACvtfcHk0lgkD0dz2otuvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovF2ju7w%253D&md5=c252ec223e909e92ec165f60a1a33b16</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.06.030%26sid%3Dliteratum%253Aachs%26aulast%3DWhitehead%26aufirst%3DL.%26aulast%3DDobler%26aufirst%3DM.%2BR.%26aulast%3DRadetich%26aufirst%3DB.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DAtadja%26aufirst%3DP.%2BW.%26aulast%3DClaiborne%26aufirst%3DT.%26aulast%3DGrob%26aufirst%3DJ.%2BE.%26aulast%3DMcRiner%26aufirst%3DA.%26aulast%3DPancost%26aufirst%3DM.%2BR.%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DShultz%26aufirst%3DM.%26aulast%3DTichkule%26aufirst%3DR.%26aulast%3DTommasi%26aufirst%3DR.%2BA.%26aulast%3DVash%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DStams%26aufirst%3DT.%26atitle%3DHuman%2520HDAC%2520isoform%2520selectivity%2520achieved%2520via%2520exploitation%2520of%2520the%2520acetate%2520release%2520channel%2520with%2520structurally%2520unique%2520small%2520molecule%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D4626%26epage%3D4634%26doi%3D10.1016%2Fj.bmc.2011.06.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Hussain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bharadwaj, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaudhuri, M. K.</span><span> </span><span class="NLM_article-title">Borax-catalyzed and pH-controlled selective oxidation of organic sulfides by H<sub>2</sub>O<sub>2</sub>: An environmentally clean protocol</span> <span class="citation_source-journal">Eur. J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">2009</span><span class="NLM_x">, </span> <span class="NLM_fpage">3319</span><span class="NLM_x">–</span> <span class="NLM_lpage">3322</span><span class="refDoi"> DOI: 10.1002/ejoc.200900265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fejoc.200900265" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2009&publication_year=2009&pages=3319-3322&author=S.+Hussainauthor=S.+K.+Bharadwajauthor=R.+Pandeyauthor=M.+K.+Chaudhuri&title=Borax-catalyzed+and+pH-controlled+selective+oxidation+of+organic+sulfides+by+H2O2%3A+An+environmentally+clean+protocol&doi=10.1002%2Fejoc.200900265"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fejoc.200900265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.200900265%26sid%3Dliteratum%253Aachs%26aulast%3DHussain%26aufirst%3DS.%26aulast%3DBharadwaj%26aufirst%3DS.%2BK.%26aulast%3DPandey%26aufirst%3DR.%26aulast%3DChaudhuri%26aufirst%3DM.%2BK.%26atitle%3DBorax-catalyzed%2520and%2520pH-controlled%2520selective%2520oxidation%2520of%2520organic%2520sulfides%2520by%2520H2O2%253A%2520An%2520environmentally%2520clean%2520protocol%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2009%26volume%3D2009%26spage%3D3319%26epage%3D3322%26doi%3D10.1002%2Fejoc.200900265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Saito, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mariko, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nosaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchiya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ando, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuruo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, O.</span><span> </span><span class="NLM_article-title">A synthetic inhibitor of histone deacetylase, MS-27–275, with marked in vivo antitumor activity against human tumors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">4592</span><span class="NLM_x">–</span> <span class="NLM_lpage">4597</span><span class="refDoi"> DOI: 10.1073/pnas.96.8.4592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1073%2Fpnas.96.8.4592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=4592-4597&author=A.+Saitoauthor=T.+Yamashitaauthor=Y.+Marikoauthor=Y.+Nosakaauthor=K.+Tsuchiyaauthor=T.+Andoauthor=T.+Suzukiauthor=T.+Tsuruoauthor=O.+Nakanishi&title=A+synthetic+inhibitor+of+histone+deacetylase%2C+MS-27%E2%80%93275%2C+with+marked+in+vivo+antitumor+activity+against+human+tumors&doi=10.1073%2Fpnas.96.8.4592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.8.4592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.8.4592%26sid%3Dliteratum%253Aachs%26aulast%3DSaito%26aufirst%3DA.%26aulast%3DYamashita%26aufirst%3DT.%26aulast%3DMariko%26aufirst%3DY.%26aulast%3DNosaka%26aufirst%3DY.%26aulast%3DTsuchiya%26aufirst%3DK.%26aulast%3DAndo%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DTsuruo%26aufirst%3DT.%26aulast%3DNakanishi%26aufirst%3DO.%26atitle%3DA%2520synthetic%2520inhibitor%2520of%2520histone%2520deacetylase%252C%2520MS-27%25E2%2580%2593275%252C%2520with%2520marked%2520in%2520vivo%2520antitumor%2520activity%2520against%2520human%2520tumors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1999%26volume%3D96%26spage%3D4592%26epage%3D4597%26doi%3D10.1073%2Fpnas.96.8.4592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosa, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitreva, M.</span><span> </span><span class="NLM_article-title">Targeting lysine deacetylases (KDACs) in Parasites</span> <span class="citation_source-journal">PLoS Neglected Trop. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e0004026</span><span class="refDoi"> DOI: 10.1371/journal.pntd.0004026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1371%2Fjournal.pntd.0004026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=26402733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlCnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=e0004026&author=Q.+Wangauthor=B.+A.+Rosaauthor=B.+Nareauthor=K.+Powellauthor=S.+Valenteauthor=D.+Rotiliauthor=A.+Maiauthor=G.+R.+Marshallauthor=M.+Mitreva&title=Targeting+lysine+deacetylases+%28KDACs%29+in+Parasites&doi=10.1371%2Fjournal.pntd.0004026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="#"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=&md5=9874b665cc7a056b8e2f928dd3112440</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pntd.0004026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pntd.0004026%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DRosa%26aufirst%3DB.%2BA.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DPowell%26aufirst%3DK.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26aulast%3DMitreva%26aufirst%3DM.%26atitle%3DTargeting%2520lysine%2520deacetylases%2520%2528KDACs%2529%2520in%2520Parasites%26jtitle%3DPLoS%2520Neglected%2520Trop.%2520Dis.%26date%3D2015%26volume%3D9%26spage%3De0004026%26doi%3D10.1371%2Fjournal.pntd.0004026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Ballante, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musmuca, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span> </span><span class="NLM_article-title">Comprehensive model of wild-type and mutant HIV-1 reverse transciptases</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">907</span><span class="NLM_x">–</span> <span class="NLM_lpage">919</span><span class="refDoi"> DOI: 10.1007/s10822-012-9586-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1007%2Fs10822-012-9586-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=907-919&author=F.+Ballanteauthor=I.+Musmucaauthor=G.+R.+Marshallauthor=R.+Ragno&title=Comprehensive+model+of+wild-type+and+mutant+HIV-1+reverse+transciptases&doi=10.1007%2Fs10822-012-9586-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9586-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9586-6%26sid%3Dliteratum%253Aachs%26aulast%3DBallante%26aufirst%3DF.%26aulast%3DMusmuca%26aufirst%3DI.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26aulast%3DRagno%26aufirst%3DR.%26atitle%3DComprehensive%2520model%2520of%2520wild-type%2520and%2520mutant%2520HIV-1%2520reverse%2520transciptases%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26volume%3D26%26spage%3D907%26epage%3D919%26doi%3D10.1007%2Fs10822-012-9586-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Ballante, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span> </span><span class="NLM_article-title">3-D QSAutogrid/R: an alternative procedure to build 3-D QSAR models. Methodologies and applications</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1674</span><span class="NLM_x">–</span> <span class="NLM_lpage">1685</span><span class="refDoi"> DOI: 10.1021/ci300123x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300123x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsFOmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1674-1685&author=F.+Ballanteauthor=R.+Ragno&title=3-D+QSAutogrid%2FR%3A+an+alternative+procedure+to+build+3-D+QSAR+models.+Methodologies+and+applications&doi=10.1021%2Fci300123x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">3-D QSAutogrid/R: An Alternative Procedure To Build 3-D QSAR Models. Methodologies and Applications</span></div><div class="casAuthors">Ballante, Flavio; Ragno, Rino</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1674-1685</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Since it first appeared in 1988 3-D QSAR has proved its potential in the field of drug design and activity prediction.  Although thousands of citations now exist in 3-D QSAR, its development was rather slow with the majority of new 3-D QSAR applications just extensions of CoMFA.  An alternative way to build 3-D QSAR models, based on an evolution of software, has been named 3-D QSAutogrid/R and has been developed to use only software freely available to academics. 3-D QSAutogrid/R covers all the main features of CoMFA and GRID/GOLPE with implementation by multiprobe/multiregion variable selection (MPGRS) that improves the simplification of interpretation of the 3-D QSAR map.  The methodol. is based on the integration of the mol. interaction fields as calcd. by AutoGrid and the R statistical environment that can be easily coupled with many free graphical mol. interfaces such as UCSF-Chimera, AutoDock Tools, JMol, and others.  The description of each R package is reported in detail, and, to assess its validity, 3-D QSAutogrid/R has been applied to three mol. data sets of which either CoMFA or GRID/GOLPE models were reported in order to compare the results. 3-D QSAutogrid/R has been used as the core engine to prep. more that 240 3-D QSAR models forming the very first 3-D QSAR server (www.3d-qsar.com) with its code freely available through R-Cran distribution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcTVzJ7Ee_q7Vg90H21EOLACvtfcHk0liAa6dIyzCwPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsFOmsbs%253D&md5=a65b741e5bd01070dccf19db522a6685</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fci300123x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300123x%26sid%3Dliteratum%253Aachs%26aulast%3DBallante%26aufirst%3DF.%26aulast%3DRagno%26aufirst%3DR.%26atitle%3D3-D%2520QSAutogrid%252FR%253A%2520an%2520alternative%2520procedure%2520to%2520build%25203-D%2520QSAR%2520models.%2520Methodologies%2520and%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1674%26epage%3D1685%26doi%3D10.1021%2Fci300123x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stutzle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Empirical scoring functions for advanced protein-ligand docking with PLANTS</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/ci800298z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800298z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=84-96&author=O.+Korbauthor=T.+Stutzleauthor=T.+E.+Exner&title=Empirical+scoring+functions+for+advanced+protein-ligand+docking+with+PLANTS&doi=10.1021%2Fci800298z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span></div><div class="casAuthors">Korb, Oliver; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we present two empirical scoring functions, PLANTSCHEMPLP and PLANTSPLP, designed for our docking algorithm PLANTS (Protein-Ligand ANT System), which is based on ant colony optimization (ACO).  They are related, regarding their functional form, to parts of already published scoring functions and force fields.  The parametrization procedure described here was able to identify several parameter settings showing an excellent performance for the task of pose prediction on two test sets comprising 298 complexes in total.  Up to 87% of the complexes of the Astex diverse set and 77% of the CCDC/Astex clean listnc (noncovalently bound complexes of the clean list) could be reproduced with root-mean-square deviations of less than 2 Å with respect to the exptl. detd. structures.  A comparison with the state-of-the-art docking tool GOLD clearly shows that this is, esp. for the druglike Astex diverse set, an improvement in pose prediction performance.  Addnl., optimized parameter settings for the search algorithm were identified, which can be used to balance pose prediction reliability and search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_kyR7CGk9mbVg90H21EOLACvtfcHk0liAa6dIyzCwPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D&md5=fce960b75b9f8906354b7c97ff84c092</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci800298z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800298z%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DStutzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DEmpirical%2520scoring%2520functions%2520for%2520advanced%2520protein-ligand%2520docking%2520with%2520PLANTS%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D84%26epage%3D96%26doi%3D10.1021%2Fci800298z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span><span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0liAa6dIyzCwPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Santos-Martins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4(Zn): an improved AutoDock force field for small-molecule docking to zinc metalloproteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2371</span><span class="NLM_x">–</span> <span class="NLM_lpage">2379</span><span class="refDoi"> DOI: 10.1021/ci500209e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500209e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslWmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2371-2379&author=D.+Santos-Martinsauthor=S.+Forliauthor=M.+J.+Ramosauthor=A.+J.+Olson&title=AutoDock4%28Zn%29%3A+an+improved+AutoDock+force+field+for+small-molecule+docking+to+zinc+metalloproteins&doi=10.1021%2Fci500209e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock4Zn: An Improved AutoDock Force Field for Small-Molecule Docking to Zinc Metalloproteins</span></div><div class="casAuthors">Santos-Martins, Diogo; Forli, Stefano; Ramos, Maria Joao; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2371-2379</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Zinc is present in a wide variety of proteins and is important in the metab. of most organisms.  Zinc metalloenzymes are therapeutically relevant targets in diseases such as cancer, heart disease, bacterial infection, and Alzheimer's disease.  In most cases a drug mol. targeting such enzymes establishes an interaction that coordinates with the zinc ion.  Thus, accurate prediction of the interaction of ligands with zinc is an important aspect of computational docking and virtual screening against zinc contg. proteins.  We have extended the AutoDock force field to include a specialized potential describing the interactions of zinc-coordinating ligands.  This potential describes both the energetic and geometric components of the interaction.  The new force field, named AutoDock4Zn, was calibrated on a data set of 292 crystal complexes contg. zinc.  Redocking expts. show that the force field provides significant improvement in performance in both free energy of binding estn. as well as in root-mean-square deviation from the crystal structure pose.  The new force field has been implemented in AutoDock without modification to the source code.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0MFHtEldlWLVg90H21EOLACvtfcHk0ljUZt7AcR6CFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslWmurg%253D&md5=b331b3a92315f7fd9a56d1e43a16990d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci500209e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500209e%26sid%3Dliteratum%253Aachs%26aulast%3DSantos-Martins%26aufirst%3DD.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2528Zn%2529%253A%2520an%2520improved%2520AutoDock%2520force%2520field%2520for%2520small-molecule%2520docking%2520to%2520zinc%2520metalloproteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2371%26epage%3D2379%26doi%3D10.1021%2Fci500209e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0ljUZt7AcR6CFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Allen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balius, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brozell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">DOCK 6: impact of new features and current docking performance</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1132</span><span class="NLM_x">–</span> <span class="NLM_lpage">1156</span><span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-1156&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+impact+of+new+features+and+current+docking+performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0ljUZt7AcR6CFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520impact%2520of%2520new%2520features%2520and%2520current%2520docking%2520performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D1132%26epage%3D1156%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Bai, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H. L.</span><span> </span><span class="NLM_article-title">An accurate metalloprotein-specific scoring function and molecular docking program devised by a dynamic sampling and iteration optimization strategy</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">833</span><span class="NLM_x">–</span> <span class="NLM_lpage">847</span><span class="refDoi"> DOI: 10.1021/ci500647f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500647f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFensLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=833-847&author=F.+Baiauthor=S.+Liaoauthor=J.+F.+Guauthor=H.+L.+Jiangauthor=X.+C.+Wangauthor=H.+L.+Li&title=An+accurate+metalloprotein-specific+scoring+function+and+molecular+docking+program+devised+by+a+dynamic+sampling+and+iteration+optimization+strategy&doi=10.1021%2Fci500647f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">An Accurate Metalloprotein-Specific Scoring Function and Molecular Docking Program Devised by a Dynamic Sampling and Iteration Optimization Strategy</span></div><div class="casAuthors">Bai, Fang; Liao, Sha; Gu, Junfeng; Jiang, Hualiang; Wang, Xicheng; Li, Honglin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">833-847</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalloproteins, particularly zinc metalloproteins, are promising therapeutic targets, and recent efforts have focused on the identification of potent and selective inhibitors of these proteins.  However, the ability of current drug discovery and design technologies, such as mol. docking and mol. dynamics simulations, to probe metal-ligand interactions remains limited because of their complicated coordination geometries and rough treatment in current force fields.  Herein the authors introduce a robust, multiobjective optimization algorithm-driven metalloprotein-specific docking program named MpSDock, which runs on a scheme similar to consensus scoring consisting of a force-field-based scoring function and a knowledge-based scoring function.  For this purpose, in this study, an effective knowledge-based zinc metalloprotein-specific scoring function based on the inverse Boltzmann law was designed and optimized using a dynamic sampling and iteration optimization strategy.  This optimization strategy can dynamically sample and regenerate decoy poses used in each iteration step of refining the scoring function, thus dramatically improving both the effectiveness of the exploration of the binding conformational space and the sensitivity of the ranking of the native binding poses.  To validate the zinc metalloprotein-specific scoring function and its special built-in docking program, denoted MpSDockZn, an extensive comparison was performed against six universal, popular docking programs: Glide XP mode, Glide SP mode, Gold, AutoDock, AutoDock4Zn, and EADock DSS.  The zinc metalloprotein-specific knowledge-based scoring function exhibited prominent performance in accurately describing the geometries and interactions of the coordination bonds between the zinc ions and chelating agents of the ligands.  In addn., MpSDockZn had a competitive ability to sample and identify native binding poses with a higher success rate than the other six docking programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo79bqIwX3mGLVg90H21EOLACvtfcHk0lj4I9TSM20iyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFensLg%253D&md5=737110fbbc8e0318cf6c518f742c6d5c</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Fci500647f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500647f%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiao%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DJ.%2BF.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DWang%26aufirst%3DX.%2BC.%26aulast%3DLi%26aufirst%3DH.%2BL.%26atitle%3DAn%2520accurate%2520metalloprotein-specific%2520scoring%2520function%2520and%2520molecular%2520docking%2520program%2520devised%2520by%2520a%2520dynamic%2520sampling%2520and%2520iteration%2520optimization%2520strategy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D833%26epage%3D847%26doi%3D10.1021%2Fci500647f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surflex-Dock 2.1: robust performance from ligand energetic modeling, ring flexibility, and knowledge-based search</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">281</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.1007/s10822-007-9114-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1007%2Fs10822-007-9114-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=17387436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVOgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=281-306&author=A.+N.+Jain&title=Surflex-Dock+2.1%3A+robust+performance+from+ligand+energetic+modeling%2C+ring+flexibility%2C+and+knowledge-based+search&doi=10.1007%2Fs10822-007-9114-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex-Dock 2.1: Robust performance from ligand energetic modeling, ring flexibility, and knowledge-based search</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">281-306</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The Surflex flexible mol. docking method has been generalized and extended in two primary areas related to the search component of docking.  First, incorporation of a small-mol. force-field extends the search into Cartesian coordinates constrained by internal ligand energetics.  Whereas previous versions searched only the alignment and acyclic torsional space of the ligand, the new approach supports dynamic ring flexibility and all-atom optimization of docked ligand poses.  Second, knowledge of well established mol. interactions between ligand fragments and a target protein can be directly exploited to guide the search process.  This offers advantages in some cases over the search strategy where ligand alignment is guided solely by a "protomol" (a pre-computed mol. representation of an idealized ligand).  Results are presented on both docking accuracy and screening utility using multiple publicly available benchmark data sets that place Surflex's performance in the context of other mol. docking methods.  In terms of docking accuracy, Surflex-Dock 2.1 performs as well as the best available methods.  In the area of screening utility, Surflex's performance is extremely robust, and it is clearly superior to other methods within the set of cases for which comparative data are available, with roughly double the screening enrichment performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQaowZ5Af2ybVg90H21EOLACvtfcHk0lj4I9TSM20iyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVOgt78%253D&md5=3cb4321457ff35914787a08acfca7b23</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9114-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9114-2%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex-Dock%25202.1%253A%2520robust%2520performance%2520from%2520ligand%2520energetic%2520modeling%252C%2520ring%2520flexibility%252C%2520and%2520knowledge-based%2520search%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D281%26epage%3D306%26doi%3D10.1007%2Fs10822-007-9114-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Scuto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirschbaum, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kowolik, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kretzner, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juhasz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pullarkat, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhatia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jove, R.</span><span> </span><span class="NLM_article-title">The novel histone deacetylase inhibitor, LBH589, induces expression of DNA damage response genes and apoptosis in Ph- acute lymphoblastic leukemia cells</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">5093</span><span class="NLM_x">–</span> <span class="NLM_lpage">5100</span><span class="refDoi"> DOI: 10.1182/blood-2007-10-117762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1182%2Fblood-2007-10-117762" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2008&pages=5093-5100&author=A.+Scutoauthor=M.+Kirschbaumauthor=C.+Kowolikauthor=L.+Kretznerauthor=A.+Juhaszauthor=P.+Atadjaauthor=V.+Pullarkatauthor=R.+Bhatiaauthor=S.+Formanauthor=Y.+Yenauthor=R.+Jove&title=The+novel+histone+deacetylase+inhibitor%2C+LBH589%2C+induces+expression+of+DNA+damage+response+genes+and+apoptosis+in+Ph-+acute+lymphoblastic+leukemia+cells&doi=10.1182%2Fblood-2007-10-117762"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-10-117762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-10-117762%26sid%3Dliteratum%253Aachs%26aulast%3DScuto%26aufirst%3DA.%26aulast%3DKirschbaum%26aufirst%3DM.%26aulast%3DKowolik%26aufirst%3DC.%26aulast%3DKretzner%26aufirst%3DL.%26aulast%3DJuhasz%26aufirst%3DA.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DPullarkat%26aufirst%3DV.%26aulast%3DBhatia%26aufirst%3DR.%26aulast%3DForman%26aufirst%3DS.%26aulast%3DYen%26aufirst%3DY.%26aulast%3DJove%26aufirst%3DR.%26atitle%3DThe%2520novel%2520histone%2520deacetylase%2520inhibitor%252C%2520LBH589%252C%2520induces%2520expression%2520of%2520DNA%2520damage%2520response%2520genes%2520and%2520apoptosis%2520in%2520Ph-%2520acute%2520lymphoblastic%2520leukemia%2520cells%26jtitle%3DBlood%26date%3D2008%26volume%3D111%26spage%3D5093%26epage%3D5100%26doi%3D10.1182%2Fblood-2007-10-117762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Crisanti, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallace, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeers, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowling, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pereira, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campling, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fridlender, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kao, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albelda, S. M.</span><span> </span><span class="NLM_article-title">The HDAC inhibitor panobinostat (LBH589) inhibits mesothelioma and lung cancer cells in vitro and in vivo with particular efficacy for small cell lung cancer</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2221</span><span class="NLM_x">–</span> <span class="NLM_lpage">2231</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1158%2F1535-7163.MCT-09-0138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=19671764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=2221-2231&author=M.+C.+Crisantiauthor=A.+F.+Wallaceauthor=V.+Kapoorauthor=F.+Vandermeersauthor=M.+L.+Dowlingauthor=L.+P.+Pereiraauthor=K.+Colemanauthor=B.+G.+Camplingauthor=Z.+G.+Fridlenderauthor=G.+D.+Kaoauthor=S.+M.+Albelda&title=The+HDAC+inhibitor+panobinostat+%28LBH589%29+inhibits+mesothelioma+and+lung+cancer+cells+in+vitro+and+in+vivo+with+particular+efficacy+for+small+cell+lung+cancer&doi=10.1158%2F1535-7163.MCT-09-0138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">The HDAC inhibitor panobinostat (LBH589) inhibits mesothelioma and lung cancer cells in vitro and in vivo with particular efficacy for small cell lung cancer</span></div><div class="casAuthors">Crisanti, M. Cecilia; Wallace, Africa F.; Kapoor, Veena; Vandermeers, Fabian; Dowling, Melissa L.; Pereira, Luana P.; Coleman, Kara; Campling, Barbara G.; Fridlender, Zvi G.; Kao, Gary D.; Albelda, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2221-2231</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Lung cancer is the leading cause of cancer deaths in the United States.  Current therapies are inadequate.  Histone deacetylase inhibitors (HDACi) are a recently developed class of anticancer agents that cause increased acetylation of core histones and nonhistone proteins leading to modulation of gene expression and protein activity involved in cancer cell growth and survival pathways.  We examd. the efficacy of the HDACi panobinostat (LBH589) in a wide range of lung cancers and mesotheliomas.  Panobinostat was cytotoxic in almost all 37 cancer cell lines tested.  IC50 and LD50 values were in the low nmol/L range (4-470 nmol/L; median, 20 nmol/L).  Small cell lung cancer (SCLC) cell lines were among the most sensitive lines, with LD50 values consistently <25 nmol/L.  In lung cancer and mesothelioma animal models, panobinostat significantly decreased tumor growth by an av. of 62% when compared with vehicle control.  Panobinostat was equally effective in immunocompetent and severe combined immunodeficiency mice, indicating that the inhibition of tumor growth by panobinostat was not due to direct immunol. effects.  Panobinostat was, however, particularly effective in SCLC xenografts, and the addn. of the chemotherapy agent etoposide augmented antitumor effects.  Protein anal. of treated tumor biopsies revealed elevated amts. of cell cycle regulators such as p21 and proapoptosis factors, such as caspase 3 and 7 and cleaved poly[ADP-ribose] polymerase, coupled with decreased levels of antiapoptotic factors such as Bcl-2 and Bcl-XL.  These studies together suggest that panobinostat may be a useful adjunct in the treatment of thoracic malignancies, esp. SCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhFRlQ0BB9B7Vg90H21EOLACvtfcHk0lj4I9TSM20iyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpvFCnu7g%253D&md5=dc513f6d94913942f310061a9172229f</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0138%26sid%3Dliteratum%253Aachs%26aulast%3DCrisanti%26aufirst%3DM.%2BC.%26aulast%3DWallace%26aufirst%3DA.%2BF.%26aulast%3DKapoor%26aufirst%3DV.%26aulast%3DVandermeers%26aufirst%3DF.%26aulast%3DDowling%26aufirst%3DM.%2BL.%26aulast%3DPereira%26aufirst%3DL.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCampling%26aufirst%3DB.%2BG.%26aulast%3DFridlender%26aufirst%3DZ.%2BG.%26aulast%3DKao%26aufirst%3DG.%2BD.%26aulast%3DAlbelda%26aufirst%3DS.%2BM.%26atitle%3DThe%2520HDAC%2520inhibitor%2520panobinostat%2520%2528LBH589%2529%2520inhibits%2520mesothelioma%2520and%2520lung%2520cancer%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%2520with%2520particular%2520efficacy%2520for%2520small%2520cell%2520lung%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2009%26volume%3D8%26spage%3D2221%26epage%3D2231%26doi%3D10.1158%2F1535-7163.MCT-09-0138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span>GraphPad 6.01; <span class="NLM_publisher-name">GraphPad Software</span>: <span class="NLM_publisher-loc">La Jolla CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=GraphPad+6.01%3B+GraphPad+Software%3A+La+Jolla+CA%2C+2012."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGraphPad%25206.01%26pub%3DGraphPad%2520Software%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Ying, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taori, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luesch, H.</span><span> </span><span class="NLM_article-title">Total synthesis and molecular target of largazole, a histone deacetylase inhibitor</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x">, </span> <span class="NLM_fpage">8455</span><span class="NLM_x">–</span> <span class="NLM_lpage">8459</span><span class="refDoi"> DOI: 10.1021/ja8013727</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja8013727" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVeksr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=8455-8459&author=Y.+Yingauthor=K.+Taoriauthor=H.+Kimauthor=J.+Hongauthor=H.+Luesch&title=Total+synthesis+and+molecular+target+of+largazole%2C+a+histone+deacetylase+inhibitor&doi=10.1021%2Fja8013727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Total Synthesis and Molecular Target of Largazole, a Histone Deacetylase Inhibitor</span></div><div class="casAuthors">Ying, Yongcheng; Taori, Kanchan; Kim, Hyoungsu; Hong, Jiyong; Luesch, Hendrik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">8455-8459</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Full details of the concise and convergent synthesis (eight steps, 19% overall yield), its extension to the prepn. of a series of key analogs, and the mol. target and pharmacophore of largazole are described.  Central to the synthesis of largazole is a macrocyclization reaction for formation of the strained 16-membered depsipeptide core followed by an olefin cross-metathesis reaction for installation of the thioester.  The biol. evaluation of largazole and its key analogs, including an acetyl analog, a thiol analog, and a hydroxyl analog, suggested that histone deacetylases (HDACs) are mol. targets of largazole and largazole is a class I HDAC inhibitor.  In addn., structure-activity relationship (SAR) studies revealed that the thiol group is the pharmacophore of the natural product.  Largazole's HDAC inhibitory activity correlates with its antiproliferative activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3QNyBNo1EnbVg90H21EOLACvtfcHk0lhkv5GKT8v6lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVeksr4%253D&md5=d580c55dfc1c921346a8d79b2b96baa3</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fja8013727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja8013727%26sid%3Dliteratum%253Aachs%26aulast%3DYing%26aufirst%3DY.%26aulast%3DTaori%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DLuesch%26aufirst%3DH.%26atitle%3DTotal%2520synthesis%2520and%2520molecular%2520target%2520of%2520largazole%252C%2520a%2520histone%2520deacetylase%2520inhibitor%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D8455%26epage%3D8459%26doi%3D10.1021%2Fja8013727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The protein data bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+protein+data+bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhkv5GKT8v6lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520protein%2520data%2520bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Van Der Spoel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindahl, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groenhof, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mark, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J.</span><span> </span><span class="NLM_article-title">GROMACS: fast, flexible, and free</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">1718</span><span class="refDoi"> DOI: 10.1002/jcc.20291</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.20291" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=16211538" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1701-1718&author=D.+Van+Der+Spoelauthor=E.+Lindahlauthor=B.+Hessauthor=G.+Groenhofauthor=A.+E.+Markauthor=H.+J.+Berendsen&title=GROMACS%3A+fast%2C+flexible%2C+and+free&doi=10.1002%2Fjcc.20291"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS: Fast, flexible, and free</span></div><div class="casAuthors">Van Der Spoel, David; Lindahl, Erik; Hess, Berk; Groenhof, Gerrit; Mark, Alan E.; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1701-1718</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This article describes the software suite GROMACS (Groningen MAchine for Chem. Simulation) that was developed at the University of Groningen, The Netherlands, in the early 1990s.  The software, written in ANSI C, originates from a parallel hardware project, and is well suited for parallelization on processor clusters.  By careful optimization of neighbor searching and of inner loop performance, GROMACS is a very fast program for mol. dynamics simulation.  It does not have a force field of its own, but is compatible with GROMOS, OPLS, AMBER, and ENCAD force fields.  In addn., it can handle polarizable shell models and flexible constraints.  The program is versatile, as force routines can be added by the user, tabulated functions can be specified, and analyses can be easily customized.  Nonequil. dynamics and free energy detns. are incorporated.  Interfaces with popular quantum-chem. packages (MOPAC, GAMES-UK, GAUSSIAN) are provided to perform mixed MM/QM simulations.  The package includes about 100 utility and anal. programs.  GROMACS is in the public domain and distributed (with source code and documentation) under the GNU General Public License.  It is maintained by a group of developers from the Universities of Groningen, Uppsala, and Stockholm, and the Max Planck Institute for Polymer Research in Mainz.  Its Web site is http://www.gromacs.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxInIDyt9Y8bVg90H21EOLACvtfcHk0lhkv5GKT8v6lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbO&md5=4aa71b1dc0b5b4978a88b0568245a265</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20291%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26aulast%3DHess%26aufirst%3DB.%26aulast%3DGroenhof%26aufirst%3DG.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%26atitle%3DGROMACS%253A%2520fast%252C%2520flexible%252C%2520and%2520free%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1701%26epage%3D1718%26doi%3D10.1002%2Fjcc.20291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Sousa da Silva, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vranken, W. F.</span><span> </span><span class="NLM_article-title">ACPYPE - AnteChamber PYthon Parser interfacE</span> <span class="citation_source-journal">BMC Res. Notes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">367</span><span class="refDoi"> DOI: 10.1186/1756-0500-5-367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1186%2F1756-0500-5-367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=22824207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=367&author=A.+W.+Sousa+da+Silvaauthor=W.+F.+Vranken&title=ACPYPE+-+AnteChamber+PYthon+Parser+interfacE&doi=10.1186%2F1756-0500-5-367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">ACPYPE - AnteChamber PYthon Parser interfacE</span></div><div class="casAuthors">Sousa da Silva Alan W; Vranken Wim F</div><div class="citationInfo"><span class="NLM_cas:title">BMC research notes</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">367</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  ACPYPE (or AnteChamber PYthon Parser interfacE) is a wrapper script around the ANTECHAMBER software that simplifies the generation of small molecule topologies and parameters for a variety of molecular dynamics programmes like GROMACS, CHARMM and CNS.  It is written in the Python programming language and was developed as a tool for interfacing with other Python based applications such as the CCPN software suite (for NMR data analysis) and ARIA (for structure calculations from NMR data).  ACPYPE is open source code, under GNU GPL v3, and is available as a stand-alone application at http://www.ccpn.ac.uk/acpype and as a web portal application at http://webapps.ccpn.ac.uk/acpype.  FINDINGS:  We verified the topologies generated by ACPYPE in three ways: by comparing with default AMBER topologies for standard amino acids; by generating and verifying topologies for a large set of ligands from the PDB; and by recalculating the structures for 5 protein-ligand complexes from the PDB.  CONCLUSIONS:  ACPYPE is a tool that simplifies the automatic generation of topology and parameters in different formats for different molecular mechanics programmes, including calculation of partial charges, while being object oriented for integration with other applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbj1v3HcWoNJZoeNcnHOQ9fW6udTcc2ea6bFENTkIiPrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fitlWjtg%253D%253D&md5=60f750ba98c975374b351a39a1fa0ec4</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1186%2F1756-0500-5-367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-0500-5-367%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%2Bda%2BSilva%26aufirst%3DA.%2BW.%26aulast%3DVranken%26aufirst%3DW.%2BF.%26atitle%3DACPYPE%2520-%2520AnteChamber%2520PYthon%2520Parser%2520interfacE%26jtitle%3DBMC%2520Res.%2520Notes%26date%3D2012%26volume%3D5%26spage%3D367%26doi%3D10.1186%2F1756-0500-5-367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lhgcXyzeaOlyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lhgcXyzeaOlyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Lindorff-Larsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maragakis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klepeis, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dror, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span> </span><span class="NLM_article-title">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">78</span><span class="NLM_x">, </span> <span class="NLM_fpage">1950</span><span class="NLM_x">–</span> <span class="NLM_lpage">1958</span><span class="refDoi"> DOI: 10.1002/prot.22711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1002%2Fprot.22711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=20408171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2010&pages=1950-1958&author=K.+Lindorff-Larsenauthor=S.+Pianaauthor=K.+Palmoauthor=P.+Maragakisauthor=J.+L.+Klepeisauthor=R.+O.+Drorauthor=D.+E.+Shaw&title=Improved+side-chain+torsion+potentials+for+the+Amber+ff99SB+protein+force+field&doi=10.1002%2Fprot.22711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Improved side-chain torsion potentials for the Amber ff99SB protein force field</span></div><div class="casAuthors">Lindorff-Larsen, Kresten; Piana, Stefano; Palmo, Kim; Maragakis, Paul; Klepeis, John L.; Dror, Ron O.; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1950-1958</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Recent advances in hardware and software have enabled increasingly long mol. dynamics (MD) simulations of biomols., exposing certain limitations in the accuracy of the force fields used for such simulations and spurring efforts to refine these force fields.  Recent modifications to the Amber and CHARMM protein force fields, for example, have improved the backbone torsion potentials, remedying deficiencies in earlier versions.  Here, the authors further advance simulation accuracy by improving the amino acid side-chain torsion potentials of the Amber ff99SB force field.  First, the authors used simulations of model alpha-helical systems to identify the four residue types whose rotamer distribution differed the most from expectations based on Protein Data Bank statistics.  Second, the authors optimized the side-chain torsion potentials of these residues to match new, high-level quantum-mech. calcns.  Finally, the authors used microsecond-timescale MD simulations in explicit solvent to validate the resulting force field against a large set of exptl. NMR measurements that directly probe side-chain conformations.  The new force field, which the authors have termed Amber ff99SB-ILDN, exhibits considerably better agreement with the NMR data.  Proteins 2010. © 2010 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNFHrYs9024bVg90H21EOLACvtfcHk0ljyooy-RFk3Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFegtLo%253D&md5=447a9004026e2b93f0f7beff165daa09</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fprot.22711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22711%26sid%3Dliteratum%253Aachs%26aulast%3DLindorff-Larsen%26aufirst%3DK.%26aulast%3DPiana%26aufirst%3DS.%26aulast%3DPalmo%26aufirst%3DK.%26aulast%3DMaragakis%26aufirst%3DP.%26aulast%3DKlepeis%26aufirst%3DJ.%2BL.%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DImproved%2520side-chain%2520torsion%2520potentials%2520for%2520the%2520Amber%2520ff99SB%2520protein%2520force%2520field%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2010%26volume%3D78%26spage%3D1950%26epage%3D1958%26doi%3D10.1002%2Fprot.22711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">O’Boyle, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G. R.</span><span> </span><span class="NLM_article-title">Open Babel: an open chemical toolbox</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="refDoi"> DOI: 10.1186/1758-2946-3-33</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=10.1186%2F1758-2946-3-33" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=21982300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=33&author=N.+M.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+A.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+R.+Hutchison&title=Open+Babel%3A+an+open+chemical+toolbox&doi=10.1186%2F1758-2946-3-33"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Open Babel: an open chemical toolbox</span></div><div class="casAuthors">O'Boyle, Noel M.; Banck, Michael; James, Craig A.; Morley, Chris; Vandermeersch, Tim; Hutchison, Geoffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: A frequent problem in computational modeling is the interconversion of chem. structures between different formats.  While std. interchange formats exist (for example, Chem. Markup Language) and de facto stds. have arisen (for example, SMILES format), the need to interconvert formats is a continuing problem due to the multitude of different application areas for chem. data, differences in the data stored by different formats (0D vs. 3D, for example), and competition between software along with a lack of vendor-neutral formats.  Results: We discuss, for the first time, Open Babel, an open-source chem. toolbox that speaks the many languages of chem. data.  Open Babel version 2.3 interconverts over 110 formats.  The need to represent such a wide variety of chem. and mol. data requires a library that implements a wide range of cheminformatics algorithms, from partial charge assignment and aromaticity detection, to bond order perception and canonicalization.  We detail the implementation of Open Babel, describe key advances in the 2.3 release, and outline a variety of uses both in terms of software products and scientific research, including applications far beyond simple format interconversion.  Conclusions: Open Babel presents a soln. to the proliferation of multiple chem. file formats.  In addn., it provides a variety of useful utilities from conformer searching and 2D depiction, to filtering, batch conversion, and substructure and similarity searching.  For developers, it can be used as a programming library to handle chem. data in areas such as org. chem., drug design, materials science, and computational chem.  It is freely available under an open-source license.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfQ1lvaS5767Vg90H21EOLACvtfcHk0ljyooy-RFk3Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWjurbF&md5=74e4f19b7f87417f916d57f7abcfb761</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-33&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-33%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%2BA.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DOpen%2520Babel%253A%2520an%2520open%2520chemical%2520toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26spage%3D33%26doi%3D10.1186%2F1758-2946-3-33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RQD','PDB','3RQD'); return false;">PDB: 3RQD</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01632">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80522"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01632">10.1021/acs.jmedchem.5b01632</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Detailed experimental procedures, purity and spectral data of compounds, additional results, and dose–response curves (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01632/suppl_file/jm5b01632_si_001.pdf">jm5b01632_si_001.pdf (12.18 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01632&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01632%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-4%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01632" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c0ba7f523c2e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
